{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "code",
      "source": [
        "import os\n",
        "import json\n",
        "import logging\n",
        "import numpy as np"
      ],
      "metadata": {
        "id": "DTtgAV8RtWiO"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "def load_json_from_lib(nome_file, local = False):\n",
        "    # Usa __file__ per ottenere il percorso della directory del file corrente\n",
        "    if not local:\n",
        "        file_path = os.path.join(os.path.dirname(__file__), nome_file)\n",
        "    else:\n",
        "        file_path = nome_file\n",
        "    with open(file_path, 'r') as file:\n",
        "        return json.load(file)"
      ],
      "metadata": {
        "id": "Qns5kcqhtSK4"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "def load_api_keys():\n",
        "    root = os.path.dirname(os.path.abspath(__file__))\n",
        "    # if not api_keys.json in cwd, save it in pkg dir\n",
        "    print(os.path.join(root, \"api_keys.json\"))\n",
        "\n",
        "    if not os.path.exists(os.path.join(root, \"api_keys.json\")):\n",
        "        file_path = os.path.join(os.path.dirname(__file__), \"api_keys.json\")\n",
        "        if os.path.exists(file_path):\n",
        "            # load api keys from pkg\n",
        "            api_keys = load_json_from_lib(\"api_keys.json\")\n",
        "        else:\n",
        "            \"\"\"\n",
        "            Please, provide API keys to the system running function:\n",
        "            in the file \"api_keys.json\"\n",
        "\n",
        "            \"\"\"\n",
        "            api_keys = {\n",
        "                \"openai\":   \"miss\",\n",
        "                \"deepseek\": \"miss\",\n",
        "                #\"grok\":     \"miss\",\n",
        "                #\"gemini\":   \"miss\",\n",
        "                #\"aimlapi\":   \"miss\",\n",
        "                \"groq\":   \"miss\",\n",
        "                \"huggingface\": \"miss\",\n",
        "            }\n",
        "    else:\n",
        "        # if api_keys.json in cwd, take them from here\n",
        "        api_keys = load_json_from_lib(\"api_keys.json\", local=True)\n",
        "\n",
        "    return api_keys"
      ],
      "metadata": {
        "id": "6d_plv7wtOdK"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "api_keys = load_api_keys()"
      ],
      "metadata": {
        "id": "eK14DtqHtBNa"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "bFiydk4rsvf2"
      },
      "outputs": [],
      "source": [
        "print(\"\\n\\n<<<<<<<<<<<<< No-RAG Reply >>>>>>>>>>>>>>>\")\n",
        "GROQ_API_KEY = api_keys[\"groq\"]\n",
        "MODEL = \"meta-llama/llama-4-scout-17b-16e-instruct\"\n",
        "# MODEL = \"llama-3.1-8b-instant\"\n",
        "\n",
        "# Test Native LLM response\n",
        "from groq import Groq\n",
        "\n",
        "client = Groq(api_key=GROQ_API_KEY)\n",
        "\n",
        "response = client.chat.completions.create(\n",
        "    model=MODEL,\n",
        "    messages=[\n",
        "        {\"role\": \"system\", \"content\": \"You are a helpful assistant\"},\n",
        "        {\"role\": \"user\", \"content\": question},\n",
        "    ],\n",
        "    stream=False\n",
        ")\n",
        "\n",
        "print(response.choices[0].message.content)\n",
        "print(\"<<< ----------------- >>>\")"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Cose mie da qua"
      ],
      "metadata": {
        "id": "NYhnzH9MwJIQ"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install groq"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "jXwQjuGgwMt_",
        "outputId": "bb8c5a54-93a4-42de-fbf2-cf5e60fdd2f4"
      },
      "execution_count": 1,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting groq\n",
            "  Downloading groq-0.26.0-py3-none-any.whl.metadata (15 kB)\n",
            "Requirement already satisfied: anyio<5,>=3.5.0 in /usr/local/lib/python3.11/dist-packages (from groq) (4.9.0)\n",
            "Requirement already satisfied: distro<2,>=1.7.0 in /usr/local/lib/python3.11/dist-packages (from groq) (1.9.0)\n",
            "Requirement already satisfied: httpx<1,>=0.23.0 in /usr/local/lib/python3.11/dist-packages (from groq) (0.28.1)\n",
            "Requirement already satisfied: pydantic<3,>=1.9.0 in /usr/local/lib/python3.11/dist-packages (from groq) (2.11.4)\n",
            "Requirement already satisfied: sniffio in /usr/local/lib/python3.11/dist-packages (from groq) (1.3.1)\n",
            "Requirement already satisfied: typing-extensions<5,>=4.10 in /usr/local/lib/python3.11/dist-packages (from groq) (4.13.2)\n",
            "Requirement already satisfied: idna>=2.8 in /usr/local/lib/python3.11/dist-packages (from anyio<5,>=3.5.0->groq) (3.10)\n",
            "Requirement already satisfied: certifi in /usr/local/lib/python3.11/dist-packages (from httpx<1,>=0.23.0->groq) (2025.4.26)\n",
            "Requirement already satisfied: httpcore==1.* in /usr/local/lib/python3.11/dist-packages (from httpx<1,>=0.23.0->groq) (1.0.9)\n",
            "Requirement already satisfied: h11>=0.16 in /usr/local/lib/python3.11/dist-packages (from httpcore==1.*->httpx<1,>=0.23.0->groq) (0.16.0)\n",
            "Requirement already satisfied: annotated-types>=0.6.0 in /usr/local/lib/python3.11/dist-packages (from pydantic<3,>=1.9.0->groq) (0.7.0)\n",
            "Requirement already satisfied: pydantic-core==2.33.2 in /usr/local/lib/python3.11/dist-packages (from pydantic<3,>=1.9.0->groq) (2.33.2)\n",
            "Requirement already satisfied: typing-inspection>=0.4.0 in /usr/local/lib/python3.11/dist-packages (from pydantic<3,>=1.9.0->groq) (0.4.1)\n",
            "Downloading groq-0.26.0-py3-none-any.whl (129 kB)\n",
            "\u001b[?25l   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m0.0/129.6 kB\u001b[0m \u001b[31m?\u001b[0m eta \u001b[36m-:--:--\u001b[0m\r\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m129.6/129.6 kB\u001b[0m \u001b[31m7.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: groq\n",
            "Successfully installed groq-0.26.0\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import pandas as pd\n",
        "\n",
        "dataset_df = pd.read_csv(\"/content/drive/MyDrive/workshop-ECAI/working_dataset_1000.csv\")\n",
        "dataset_df"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 736
        },
        "id": "HdcXYTT9xL9d",
        "outputId": "bab24612-2c89-43cb-bfce-6f6a847a3312"
      },
      "execution_count": 2,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "     pubmed_id                                              INTRO  \\\n",
              "0     30419927  Background<SEP>The Mediterranean diet (MeD) is...   \n",
              "1     30032228  Introduction<SEP>Overweight and obesity have b...   \n",
              "2     21799482  INTRODUCTION<SEP>In recent years, the prevalen...   \n",
              "3     24516669  Introduction<SEP>Osteoporosis and obesity are ...   \n",
              "4     19079261  Obesity is a major public health problem, resu...   \n",
              "..         ...                                                ...   \n",
              "995   24852944  Introduction<SEP>Adoptive immunotherapy is a p...   \n",
              "996   28231819  Background<SEP>Targeted cancer immunotherapies...   \n",
              "997   31676868  In type 1 diabetes (T1D), early inflammation o...   \n",
              "998   31506281  Introduction<SEP>Germinal centers (GCs) form i...   \n",
              "999   28415571  INTRODUCTION<SEP>Essential thrombocythemia (ET...   \n",
              "\n",
              "                                               METHODS  \\\n",
              "0    Methods<SEP>Study design and subjects<SEP>The ...   \n",
              "1    Methods<SEP>This systematic review was perform...   \n",
              "2    METHODS AND PROCEDURES<SEP>Study population<SE...   \n",
              "3    Materials and Methods<SEP>Subjects<SEP>Two pop...   \n",
              "4    METHODS<SEP>Study design<SEP>This study is com...   \n",
              "..                                                 ...   \n",
              "995  Materials and Methods<SEP>T cell culture<SEP>A...   \n",
              "996  Methods<SEP>Tumor samples and sequencing<SEP>S...   \n",
              "997  Methods:<SEP>Human Islets and EndoC-βH1<SEP>Hu...   \n",
              "998  Materials and methods<SEP>Animals and treatmen...   \n",
              "999  PATIENTS AND METHODS<SEP>Patient enrollment<SE...   \n",
              "\n",
              "                                               RESULTS  \\\n",
              "0    Results<SEP>Population characteristics<SEP>The...   \n",
              "1    Results<SEP>Articles Identified<SEP>The initia...   \n",
              "2    RESULTS<SEP>Characteristics of the study parti...   \n",
              "3    Results<SEP>The general and clinical character...   \n",
              "4    RESULTS<SEP>Initial meta-analysis of GWAS stud...   \n",
              "..                                                 ...   \n",
              "995  Results and Discussions<SEP>The coupling of pr...   \n",
              "996  Results<SEP>Tumor mutanomes are unique<SEP>We ...   \n",
              "997  Results<SEP>Pro-inflammatory cytokines impact ...   \n",
              "998  Results<SEP>TGFβ signaling occurs in GCBs and ...   \n",
              "999  RESULTS<SEP>Mutational analysis<SEP>Among 54 E...   \n",
              "\n",
              "                                               DISCUSS  text_length  \\\n",
              "0    Discussion<SEP>Among the different dietetic pa...        21979   \n",
              "1    Discussion<SEP>The purpose of this review was ...        32452   \n",
              "2    DISCUSSION<SEP>In our study, we found that var...        22444   \n",
              "3    Discussion<SEP>Obesity is an established risk ...        21427   \n",
              "4    DISCUSSION<SEP>Through meta-analysis of GWA da...        35778   \n",
              "..                                                 ...          ...   \n",
              "995                                                NaN        19075   \n",
              "996  Discussion<SEP>Our data reveal that inter-indi...        22078   \n",
              "997  Discussion:<SEP>Our work illustrates the human...        43718   \n",
              "998  Discussion<SEP>In this study, we showed that T...        42774   \n",
              "999  DISCUSSION<SEP>CALR mutations have been found ...        27830   \n",
              "\n",
              "                        TOPIC_LABEL  \n",
              "0    ['OBWCCE', 'EATBTS', 'FOODIN']  \n",
              "1              ['OBWCCE', 'EATBTS']  \n",
              "2              ['OBWCCE', 'EATBTS']  \n",
              "3                        ['OBWCCE']  \n",
              "4                        ['OBWCCE']  \n",
              "..                              ...  \n",
              "995                      ['FOODAL']  \n",
              "996                      ['FOODAL']  \n",
              "997                      ['FOODAL']  \n",
              "998                      ['FOODAL']  \n",
              "999                      ['FOODAL']  \n",
              "\n",
              "[1000 rows x 7 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-3b5d2bf1-8188-4423-a984-ea267b4cb11f\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pubmed_id</th>\n",
              "      <th>INTRO</th>\n",
              "      <th>METHODS</th>\n",
              "      <th>RESULTS</th>\n",
              "      <th>DISCUSS</th>\n",
              "      <th>text_length</th>\n",
              "      <th>TOPIC_LABEL</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>30419927</td>\n",
              "      <td>Background&lt;SEP&gt;The Mediterranean diet (MeD) is...</td>\n",
              "      <td>Methods&lt;SEP&gt;Study design and subjects&lt;SEP&gt;The ...</td>\n",
              "      <td>Results&lt;SEP&gt;Population characteristics&lt;SEP&gt;The...</td>\n",
              "      <td>Discussion&lt;SEP&gt;Among the different dietetic pa...</td>\n",
              "      <td>21979</td>\n",
              "      <td>['OBWCCE', 'EATBTS', 'FOODIN']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>30032228</td>\n",
              "      <td>Introduction&lt;SEP&gt;Overweight and obesity have b...</td>\n",
              "      <td>Methods&lt;SEP&gt;This systematic review was perform...</td>\n",
              "      <td>Results&lt;SEP&gt;Articles Identified&lt;SEP&gt;The initia...</td>\n",
              "      <td>Discussion&lt;SEP&gt;The purpose of this review was ...</td>\n",
              "      <td>32452</td>\n",
              "      <td>['OBWCCE', 'EATBTS']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>21799482</td>\n",
              "      <td>INTRODUCTION&lt;SEP&gt;In recent years, the prevalen...</td>\n",
              "      <td>METHODS AND PROCEDURES&lt;SEP&gt;Study population&lt;SE...</td>\n",
              "      <td>RESULTS&lt;SEP&gt;Characteristics of the study parti...</td>\n",
              "      <td>DISCUSSION&lt;SEP&gt;In our study, we found that var...</td>\n",
              "      <td>22444</td>\n",
              "      <td>['OBWCCE', 'EATBTS']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>24516669</td>\n",
              "      <td>Introduction&lt;SEP&gt;Osteoporosis and obesity are ...</td>\n",
              "      <td>Materials and Methods&lt;SEP&gt;Subjects&lt;SEP&gt;Two pop...</td>\n",
              "      <td>Results&lt;SEP&gt;The general and clinical character...</td>\n",
              "      <td>Discussion&lt;SEP&gt;Obesity is an established risk ...</td>\n",
              "      <td>21427</td>\n",
              "      <td>['OBWCCE']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>19079261</td>\n",
              "      <td>Obesity is a major public health problem, resu...</td>\n",
              "      <td>METHODS&lt;SEP&gt;Study design&lt;SEP&gt;This study is com...</td>\n",
              "      <td>RESULTS&lt;SEP&gt;Initial meta-analysis of GWAS stud...</td>\n",
              "      <td>DISCUSSION&lt;SEP&gt;Through meta-analysis of GWA da...</td>\n",
              "      <td>35778</td>\n",
              "      <td>['OBWCCE']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>995</th>\n",
              "      <td>24852944</td>\n",
              "      <td>Introduction&lt;SEP&gt;Adoptive immunotherapy is a p...</td>\n",
              "      <td>Materials and Methods&lt;SEP&gt;T cell culture&lt;SEP&gt;A...</td>\n",
              "      <td>Results and Discussions&lt;SEP&gt;The coupling of pr...</td>\n",
              "      <td>NaN</td>\n",
              "      <td>19075</td>\n",
              "      <td>['FOODAL']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>996</th>\n",
              "      <td>28231819</td>\n",
              "      <td>Background&lt;SEP&gt;Targeted cancer immunotherapies...</td>\n",
              "      <td>Methods&lt;SEP&gt;Tumor samples and sequencing&lt;SEP&gt;S...</td>\n",
              "      <td>Results&lt;SEP&gt;Tumor mutanomes are unique&lt;SEP&gt;We ...</td>\n",
              "      <td>Discussion&lt;SEP&gt;Our data reveal that inter-indi...</td>\n",
              "      <td>22078</td>\n",
              "      <td>['FOODAL']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>997</th>\n",
              "      <td>31676868</td>\n",
              "      <td>In type 1 diabetes (T1D), early inflammation o...</td>\n",
              "      <td>Methods:&lt;SEP&gt;Human Islets and EndoC-βH1&lt;SEP&gt;Hu...</td>\n",
              "      <td>Results&lt;SEP&gt;Pro-inflammatory cytokines impact ...</td>\n",
              "      <td>Discussion:&lt;SEP&gt;Our work illustrates the human...</td>\n",
              "      <td>43718</td>\n",
              "      <td>['FOODAL']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>998</th>\n",
              "      <td>31506281</td>\n",
              "      <td>Introduction&lt;SEP&gt;Germinal centers (GCs) form i...</td>\n",
              "      <td>Materials and methods&lt;SEP&gt;Animals and treatmen...</td>\n",
              "      <td>Results&lt;SEP&gt;TGFβ signaling occurs in GCBs and ...</td>\n",
              "      <td>Discussion&lt;SEP&gt;In this study, we showed that T...</td>\n",
              "      <td>42774</td>\n",
              "      <td>['FOODAL']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>999</th>\n",
              "      <td>28415571</td>\n",
              "      <td>INTRODUCTION&lt;SEP&gt;Essential thrombocythemia (ET...</td>\n",
              "      <td>PATIENTS AND METHODS&lt;SEP&gt;Patient enrollment&lt;SE...</td>\n",
              "      <td>RESULTS&lt;SEP&gt;Mutational analysis&lt;SEP&gt;Among 54 E...</td>\n",
              "      <td>DISCUSSION&lt;SEP&gt;CALR mutations have been found ...</td>\n",
              "      <td>27830</td>\n",
              "      <td>['FOODAL']</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>1000 rows × 7 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-3b5d2bf1-8188-4423-a984-ea267b4cb11f')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-3b5d2bf1-8188-4423-a984-ea267b4cb11f button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-3b5d2bf1-8188-4423-a984-ea267b4cb11f');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "    <div id=\"df-4009ad99-5cb6-49a0-af95-b010c7247343\">\n",
              "      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-4009ad99-5cb6-49a0-af95-b010c7247343')\"\n",
              "                title=\"Suggest charts\"\n",
              "                style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "      </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "      <script>\n",
              "        async function quickchart(key) {\n",
              "          const quickchartButtonEl =\n",
              "            document.querySelector('#' + key + ' button');\n",
              "          quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "          quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "          try {\n",
              "            const charts = await google.colab.kernel.invokeFunction(\n",
              "                'suggestCharts', [key], {});\n",
              "          } catch (error) {\n",
              "            console.error('Error during call to suggestCharts:', error);\n",
              "          }\n",
              "          quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "          quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "        }\n",
              "        (() => {\n",
              "          let quickchartButtonEl =\n",
              "            document.querySelector('#df-4009ad99-5cb6-49a0-af95-b010c7247343 button');\n",
              "          quickchartButtonEl.style.display =\n",
              "            google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "        })();\n",
              "      </script>\n",
              "    </div>\n",
              "\n",
              "  <div id=\"id_a21f3fe9-c514-438b-847f-863b4e72d058\">\n",
              "    <style>\n",
              "      .colab-df-generate {\n",
              "        background-color: #E8F0FE;\n",
              "        border: none;\n",
              "        border-radius: 50%;\n",
              "        cursor: pointer;\n",
              "        display: none;\n",
              "        fill: #1967D2;\n",
              "        height: 32px;\n",
              "        padding: 0 0 0 0;\n",
              "        width: 32px;\n",
              "      }\n",
              "\n",
              "      .colab-df-generate:hover {\n",
              "        background-color: #E2EBFA;\n",
              "        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "        fill: #174EA6;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate {\n",
              "        background-color: #3B4455;\n",
              "        fill: #D2E3FC;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate:hover {\n",
              "        background-color: #434B5C;\n",
              "        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "        fill: #FFFFFF;\n",
              "      }\n",
              "    </style>\n",
              "    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('dataset_df')\"\n",
              "            title=\"Generate code using this dataframe.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "    <script>\n",
              "      (() => {\n",
              "      const buttonEl =\n",
              "        document.querySelector('#id_a21f3fe9-c514-438b-847f-863b4e72d058 button.colab-df-generate');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      buttonEl.onclick = () => {\n",
              "        google.colab.notebook.generateWithVariable('dataset_df');\n",
              "      }\n",
              "      })();\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "dataset_df",
              "summary": "{\n  \"name\": \"dataset_df\",\n  \"rows\": 1000,\n  \"fields\": [\n    {\n      \"column\": \"pubmed_id\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 4602737,\n        \"min\": 9547335,\n        \"max\": 34960035,\n        \"num_unique_values\": 1000,\n        \"samples\": [\n          26739746,\n          19541292,\n          19165485\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"INTRO\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 997,\n        \"samples\": [\n          \"Introduction<SEP>Vitamin D is a well-known determinant of health. Apart from its classical functions in bone homeostasis, a number of physiological mechanisms have been attributed to vitamin D, including several immune-regulatory actions. Moreover, vitamin D deficiency has been related to a wide range of disorders, from autoimmunity to cardiovascular disease (CVD).<SEP>In the field of rheumatic disorders, vitamin D deficiency is a common laboratory finding. Based on its immune-regulatory properties, several studies have been conducted to explore the potential role of vitamin D in rheumatic conditions, such as rheumatoid arthritis (RA). Currently, its deficiency has been associated with worse disease outcomes, however, contradictory results have been published thereafter. A similar controversy exists regarding the effect of vitamin D supplementation in RA. The circulating levels of vitamin D metabolites are influenced by several factors ranging from seasonality to a number of bio-activating enzymes. Circulating vitamin D levels, measured by 25(OH)-vitamin D levels, are influenced by seasonality and two bioactivating cytochrome P450 isoforms, CYP27A1 and mainly CYP2R1. The next bioactivation step rendering the vitamin D hormone is catalyzed by CYP27B1, an enzyme present in numerous cell types. Moreover, the vitamin D hormone exerts its actions via Vitamin D Receptor (VDR), whose activity can be also modulated. Genetic polymorphisms have been described for all these genes, linked to altered serum levels of vitamin D metabolites (CYP2R1, CYP27B1, CYP24A1 and others) and actions (like those in VDR). Recent evidence also discloses an association between vitamin D-related polymorphisms and RA susceptibility but not severity. Whether in RA these polymorphisms influence the association between vitamin D levels and clinical features and account for the heterogeneous results previously reported remains unknown.<SEP>Cohort studies confirmed a protective effect of vitamin D on CVD, which was attributed to a favorable effect on serum lipids. However, further studies and meta-analyses have challenged this hypothesis. Again, additional mediators of the vitamin D or cholesterol pathways need to be considered to gain some insight into this situation. It is important to consider that both vitamin D and cholesterol synthesis share a common metabolic substrate, 7-dehydrocholesterol (7-DHC). Recently, the enzyme 7-dehydrocholesterol reductase (DHCR7), essential for the Kandutsch-Russell cholesterol synthesis, has been identified as a critical determinant of vitamin D levels in a sunlight-dependent manner. In RA, where both lipid profile and vitamin D are usually altered, this enzyme could play an important role and may represent a promising therapeutic target.<SEP>We hypothesize that in RA, vitamin D-related polymorphisms and DHCR7 levels influence the relationship between vitamin D levels, lipid profile and clinical outcomes. Therefore, the main aims of the present study are (i) to evaluate the associations between vitamin D levels and clinical outcomes and lipid profile in RA patients depending on their genetic status of VDR (rs2228570), CYP27A1 (rs933994), CYP2R1 (rs10741657) and DHCR7 (rs12785878) polymorphisms, (ii) to analyze the levels of DHCR7 and its associations with the lipid profile and (iii) to prospectively evaluate the effect of RA activity on the former associations in a group of patients followed-up upon Tumor Necrosis Factor alpha (TNF\\u03b1)-blockade.\",\n          \"Introduction<SEP>The annual global incidence of clinical malaria is estimated at about 216 million cases leading to approximately 445,000 deaths in 2016, a slight increase in morbidity over what was previously reported. The mortality rate of malaria was estimated to be 90% in Africa, and 80% of these deaths occur in sub-Saharan Africa and mostly in the under five years\\u2013of-age group due to severe, and often-overlapping syndromes such as Severe Malarial Anemia (SMA) and Cerebral Malaria (CM). A quarter of the global cases and a third of malaria-attributable childhood deaths occur in Nigeria, the most populous country of Africa.<SEP>Malaria is characterized by increased inflammation and intravascular hemolysis due to schizogony leading to the release of free Hemoglobin (fHb) into the plasma. Heme, the molecule bearing the iron moiety of fHb, can be oxidized by nitric oxide (NO) leading to oxidative stress-mediated damage through mediators such as lipid and hydrogen peroxides. To avoid this, fHb is scavenged by plasma Haptoglobin (HP) forming a complex, thereby preventing fHb to be oxidized and potentially preventing further heme-mediated nitric oxide damage, renal excretion of iron and further oxidative damage.<SEP>The HP gene is comprised of two co-dominant alleles, HP1 and HP2, located on the long arm of chromosome 16. The HP1 allele has five exons and the HP2 allele has seven exons (Supplementary Fig.\\u00a01). The HP2 allele contains a 1.7 Kb intragenic duplication. HP is synthesized mainly by hepatocytes in response to increased levels of interleukins such as IL-6 and IL-1 and Tumor Necrosis Factor (TNF-\\u03b1) during an acute inflammatory response. The HP-fHb complex, formed by the binding of HP to fHb, is removed from circulation by CD163 receptors expressed on the surface of macrophages followed by internalization, where fHb is degraded and heme catabolized by heme oxygenase-1.<SEP>A high level of fHb during severe hemolysis has been reported to be responsible for low HP plasma levels and therefore the circulatory level of HP has been used as a surrogate marker of intravascular hemolysis. Similarly, Lactate Dehydrogenase (LDH) is known to be abundant in erythrocytes and its plasma level is elevated in the event of erythrocyte lysis. An elevated plasma level of LDH is also a surrogate marker of hemolysis in vascular pathophysiology.<SEP>In some individuals, a complete lack of HP (ahaptoglobinemia) or low circulating HP (hypo-haptoglobinemia) can be observed. Circulatory HP level shows ethnic and geographic variation with a high prevalence of hypo-haptoglobinemia in Africans compared with Europeans. Recent studies report that the average frequency of hypo-haptoglobinemia in Africa might be as high as 40% although with different and unknown etiologies.<SEP>Studies on hypo-haptoglobinemia have provided conflicting views regarding its etiology in childhood malaria. On one hand, the low level of plasma HP found in malaria infections has been proposed to have a genetic basis, while on the other hand the low level has been attributed to the malaria infection causing intravascular hemolysis and other immune pathological processes. HP levels have been reported to recover in 75% in a study population following anti-malarial treatment. While malaria infection induced hypo-haptoglobinemia should be reversed with anti-malarial treatment, that of genetic etiology should not. Differentiating between these two etiological scenarios should provide insights into the role of HP in the establishment of mild or severe malaria syndromes.<SEP>We hypothesized that if low HP levels are exclusively a consequence of malaria infection they would return to normal on patients\\u2019 recovery. To test this hypothesis, we carried out a prospective case-control study of children (with well-defined follow-up through convalescence to recovery) living in the sub-Saharan 3-million inhabitants densely-populated urban metropolis of Ibadan, Nigeria, that is under hyper-endemic and holoendemic malaria burden. We assayed for levels of plasma HP and the markers of intravascular hemolysis fHb and LDH from admission through convalescence to recovery. Furthermore, we performed high throughput amplicon-based sequencing of the HP genes to identify genotypes associated with both circulatory HP levels and the malarial disease syndromes studied.<SEP>Here we show, by longitudinal follow-up of malaria-positive pediatric cohorts, that hypo-haptoglobinemia is a risk factor for acute onset childhood SMA, rather than the unique consequence of this syndrome, and it is likely of genetic etiology. We report for the first time a variant in the distal upstream region of the HP gene rs12162087 to be associated with circulatory HP level. We provide evidence that the variant is also associated with SMA.\",\n          \"Introduction<SEP>Genetic variation in COMT, the locus that encodes catechol-O-methyltransferase, an enzyme which metabolizes norepinephrine, epinephrine, dopamine, and catechol estrogens, has been implicated in a broad cross section of disease and conditions from hypertension, preeclampsia, cardiovascular disease, psychiatric disorders, cancer and chronic fatigue syndrome (CFS). Furthermore, COMT has also been shown to modify clinical response to both active drugs and placebo treatments in randomized clinical trials. Thus COMT is emerging as an important hub in pharmacogenomics and the development of precision medicines.<SEP>Chronic fatigue syndrome (CFS) is a disabling condition characterized by unexplained, long-lasting, fatigue and exertion intolerance, accompanied by pain, cognitive impairments and orthostatic intolerance. Elevated norepineprhine and epinephrine levels, as well as attenuated heart rate variability in adolescents with CFS, support the hypothesis that heightened sympathetic and decreased parasympathetic nervous system activity contribute to the disease process. This disabling condition affects 0.1\\u20131.0% of children world-wide, but the pathophysiology is poorly understood and there are few treatment options.<SEP>The most well-studied single nucleotide polymorphism (SNP) in COMT, rs4680 (val158met), encodes a three- to four-fold difference in enzymatic activity between the valine high-activity and methionine low-activity forms of the enzyme. COMT enzymatic activity has been shown to be inversely related to catecholamine levels. Thus, individuals homozygous for the high-activity valine (val/val) form of the enzyme tend to have lower catecholamine levels compared to those homozygous for the low-activity methionine (met/met) form. Consistent with the observation that adolescents diagnosed with CFS have elevated levels of norepinephrine and epinephrine, the COMT low-activity met/met genotype appears to be more prevalent amongst adolescents with CFS.<SEP>Clonidine is a centrally acting \\u03b12-adrenergic receptor agonist which attenuates sympathetic nervous activity, decreasing circulating norepinephrine and epinephrine levels while increasing parasympathetic nervous activity. Clonidine has also been shown to inhibit dopamine activation in the brain. The Norwegian Study of Chronic Fatigue Syndrome in Adolescents: Pathophysiology and Intervention Trial (NorCAPITAL), was a randomized placebo controlled trial (RCT) designed to study the effects of short-term low-dose clonidine treatment for CFS among adolescents. In NorCAPITAL low-dose clonidine was found to reduce catecholamine levels and attenuate markers of sympathetic outflow and systemic inflammation; however clonidine treatment failed to provide greater benefit than placebo.<SEP>To date, the effect of genetic variation in COMT on treatment response to clonidine has not been examined. Based on current evidence, we hypothesized that individuals homozygous for the low-activity met-allele form of COMT rs4680 might benefit more from the catecholamine reduction induced by clonidine treatment than individuals homozygous for the high-activity val allele form. Further, given the COMT rs4680 association with response to placebo treatment, and clonidine effects on dopamine activation in brain regions implicated in placebo response, the association between COMT and outcomes in individuals randomized to placebo treatment was examined.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"METHODS\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 986,\n        \"samples\": [\n          \"Materials and methods<SEP>Ethics statement<SEP>Subjects were informed about the procedure and the aim of the study and signed an informed consent form. The Ethical Committee of the University Hospital of Cagliari approved the study procedures, which were performed in accordance with the latest revision of the Declaration of Helsinki.<SEP>Subjects<SEP>Forty-six non-smoking healthy young Caucasian volunteers (8 men and 38 women, age 26.6 \\u00b1 0.79 years) were recruited through public advertisements at the University of Cagliari (Italy). All were originally from Sardinia, Italy. They were normal weight with a body mass index (BMI) ranging from 18.6 to 25.3 kg/m2 and experienced no change in body weight larger than 5 kg over the previous 3 months. None were dieting or taking medications that might interfere with oral sensory perception. None had food allergies, or scored high on eating behavior scales (assessed by using the Three-Factor Eating Questionnaire). Normogeusia for four basic tastes (sweet, sour, salty, and bitter) was verified in all participants by a taste strip test (Burghart Messtechnik, Wedel, Germany). This trial was registered at ClinicalTrials.gov (identifier number: UNICADBSITB-1).<SEP>Experimental protocol<SEP>Subjects were tested in three sessions. In the first two (on two consecutive days) subjects were classified for PROP-taster status. In the third session, 1-month later, their oral perception for oleic acid, and its changes due to L-Arg administration were assessed. Subjects were requested to refrain from eating, drinking (except water) and using oral care products for at least 2 hours prior to testing. In women, the assessments were done on the sixth/seventh day of their menstrual cycle to avoid changes of oral sensitivity due to the estrogen phase. The testing room was kept virtually free from odors, was lit with standard solar lighting (15,000 lux) and noise level was kept at a minimum. Subjects had to be in the testing room 15 min before the beginning of trials in order to adapt to the environmental conditions (23\\u201324\\u00b0C, 40\\u201350% relative humidity), which were kept constant during the experimental session. Solutions were prepared in spring water the day before each session and stored in the refrigerator until 1 hour before testing.<SEP>A sample (2-mL) of whole mixed saliva was collected from each subject into an acid-washed polypropylene test tube. Samples of saliva were stored at \\u201380\\u00b0C until molecular analyses were completed, as described below.<SEP>PROP-taster status<SEP>In order to classify each subject for PROP-taster status (as PROP super-taster, medium taster, or non-taster), taste intensity ratings were collected, in 2 successive days, by using two different psychophysical procedures: the three-solution test, and the impregnated paper screening test, which have been validated in several studies. Both procedures are highly reliable as they strongly correlate with the degree of activation of peripheral taste function. In the three-solution test, the taste-intensity ratings were collected for three suprathreshold solutions (in 10-mL samples) of PROP (0.032, 0.32, and 3.2 mmol/L) (Sigma-Aldrich, Milan, Italy) and NaCl (0.01, 0.1, 1.0 mol/L) (Sigma-Aldrich, Milan, Italy). Instead, the impregnated paper screening test is based on the ratings of 2 paper disks, one impregnated with PROP solution (50 mmol/L) and the other with NaCl (1.0 mol/L). Stimuli were presented at room temperature. In both tests, taste-intensity ratings for PROP and sodium chloride (NaCl) were collected from each subject by using the Labeled Magnitude Scale (LMS). LMS gives subjects the freedom to rate the perceived taste intensity of PROP and NaCl, in relation to the \\u2018strongest imaginable\\u2019 oral stimulus they had ever experienced in their life. Each subject was trained in the use of the LMS before testing. In both procedures, PROP and NaCl were presented in a blind and counterbalanced order. Subjects who gave intensity ratings higher to PROP solutions than to NaCl, or rated the PROP disk higher than 67 mm on the LMS, were classified as PROP super-tasters, while those who gave ratings to PROP solutions lower than to NaCl solutions, or rated the PROP disk lower than 13 mm on the LMS were classified as non-tasters. Finally, those who gave comparable ratings to the two stimuli, or rated PROP disk with intermediate ratings, were classified as medium-tasters. Only subjects likewise classified by the two procedures were included in the study. Ten subjects were classified as PROP super-tasters (21.74%); 19 as medium-tasters (41.30%) and 17 as non-tasters (36.96%). Three-way ANOVA was used to validate the presence of the three taster groups (see S1 Table).<SEP>Molecular analysis<SEP>DNA was extracted from saliva samples using the QIAamp\\u00ae DNA Mini Kit (QIAGEN S.r.l., Milan, Italy) according to the manufacturer\\u2019s instructions. Purified DNA concentration was estimated by measuring the optical density at 260 nm with an Agilent Cary 60 UV-Vis Spectrophotometer.<SEP>Subjects were genotyped for three single nucleotide polymorphisms (SNPs) rs713598, rs1726866, and rs10246939, respectively at base pairs 145 (C/G), 785 (C/T), and 886 (G/A) of the TAS2R38 locus, that consist of three amino acid substitutions (Pro49Ala, Ala262Val, and Val296Ile), which give rise to two major haplotypes, PAV (the dominant taster variant) and AVI (the non-taster recessive one) and three rare haplotypes (AAI, AAV, and PVI). A polymerase chain reaction (PCR) was employed to amplify the short region of the TAS2R38 locus, including the first polymorphism of interest (rs713598), followed by analysis with restriction enzyme (HaeIII) of the fragments obtained according to our previous work. The rs1726866 and rs10246939 SNPs were determined by using the TaqMan\\u00ae SNP Genotyping Assay (C_9506827_10 for the rs1726866 assay and C_9506826_10 for the rs10246939 assay; Applied Biosystems by Life-Technologies Italia, Europe BV) according to the manufacturer\\u2019s specifications. Replicates and positive and negative controls were included in all reactions.<SEP>Subjects were also genotyped for the single nucleotide polymorphism (SNP), rs1761667 (G/A), of CD36, located at the \\u221231118 promoter region of exon 1A. To genotype the CD36 rs1761667 polymorphism molecular analyses were carried out by PCR followed by analysis with restriction enzyme (HhaI) of the fragments obtained according to Banerjee et al. 2010. All digestion products were separated by electrophoresis on a 2% agarose gel and the DNA bands were visualized by ethidium bromide staining and ultraviolet light to score the deletion. PCR 50 bp Low Ladder DNA was used as a molecular mass marker (Gene Ruler\\u2122 -Thermo Scientific).<SEP>Finally, subjects were genotyped for the OBPIIa gene polymorphism rs2590498 (A/G) by using custom TaqMan\\u00ae SNP Genotyping Assay (Applied Biosystems by Life-Technologies Italia, Europe BV) according to Tomassini et al. 2017, and as briefly described below. The following primers sets were used: the forward GCCAGGCAGGGACAGA and the reverse CTACACCTGAGACCCCACAAG and two TaqMan probes were designed according to the OBPII gene (bold and underlined), probe/reporter 1: VIC-TCGGTGACATGAACC and probe/reporter 2: FAM\\u2013TCGGTGACGTGAACC. Replicates as well as positive and negative controls were included in all reactions.<SEP>Molecular analysis at the three SNPs of the TAS2R38 locus identified 8 subjects who were PAV homozygous, 20 were heterozygous, and 15 were AVI homozygous. Three subjects with rare haplotype were excluded. Molecular analysis at the SNP (rs1761667) of the CD36 identified: 15 subjects who were homozygous GG for CD36 locus, 23 who were heterozygous, and 8 who were homozygous AA. While, the observed genotype distribution for OBPIIa locus identified 9 subjects who were homozygous AA, 14 who were heterozygous, and 23 who were homozygous GG.<SEP>Quantum-mechanical calculations<SEP>Theoretical calculations were carried out at the Density Functional Theory (DFT) level with the aim of investigating the relative stabilities of oleic acid in its neutral protonated and anionic deprotonated form, L-Arg, L-argininium (L-ArgH+) and the neutral molecular adduct resulting from the interaction of L-Arg and oleic acid (oleic acid\\u00b7Arg). In DFT methods, the relationship between the electron density and the energy of the molecular system is described by a \\u201cfunctional\\u201d, i.e. a functions of another function. Based on the results of our previous calculations on Arg, and a variety of different molecular systems, also including the evaluation of intermolecular interactions, we adopted an hybrid functional (mPW1PW). Electron shells were mathematically represented by split-valence basis sets, including polarization functions. Atomic charges, bond orders, and interaction strengths were calculated at Natural Bond Orbital level (NBO). Calculations were carried out on isolated molecules (gas phase) and in the presence of solvents, namely water and n-pentadecane to mimic the differently polar environment within and outside oleic acid vesicles. More details on the adopted QM methods are available as S1 File.<SEP>Effect of L-Arg supplementation on oleic acid oral perception<SEP>The effect of supplementation with of L-Arg on oleic acid multimodal oral perception was assessed in each subject in the third session. In order to evaluate possible variations in oleic acid perception due to L-Arg administration, an amount (1 \\u03bcL) of oleic acid just above threshold was also presented supplemented with three concentrations of L-Arg (5, 10 and 15 mmol/L), which had previously been shown to be effective in modulating taste perception of five taste qualities.<SEP>Stimuli were presented to each subject, in the absence of nose clips, by means of filter paper disks (1.5 cm diameter) according to Melis et al. After rinsing mouth with spring water, each subject was presented, in a random order and a double blinded way, with 5 filter paper disks impregnated with 15 \\u03bcL of each stimulus. One contained only mineral oil (10 \\u03bcL) supplemented with spring water (5 \\u03bcL); one contained a mixture of oleic acid (1 \\u03bcL) and mineral oil (9 \\u03bcL) supplemented with spring water (5 \\u03bcL); three contained a mixture of oleic acid (1 \\u03bcL) and mineral oil (9 \\u03bcL) supplemented with L-Arg (5 \\u03bcL) at different concentrations. Subjects were instructed to place the paper disk on the center of their tongue, keep it in the mouth to savor it for 10 s in order to facilitate the release of the stimulus, then spit it out. Each stimulation was followed by oral rinsing with spring water. The interstimulus interval was set at 5 min. After 1 h, each subject was presented with three more paper disks as controls which contained mineral oil (10 \\u03bcL) supplemented with L-Arg (5 \\u03bcL) at different concentrations. Perceived intensity ratings for each stimulation were collected by having the subject place a mark on the LMS corresponding to his/her stimulus perception.<SEP>Statistical analyses<SEP>Fisher\\u2019s exact test was used to compare the percentages of subjects who showed an increase of perceived intensity ratings when the oleic acid was supplemented with the three concentrations of L-Arg (5, 10 and 15 mmol/L), also as function of PROP-taster status, or polymorphisms of TAS2R38, CD36 (rs1761667) and OBPIIa (rs2590498) genes. One-way ANOVA was used to compare the perceived intensity ratings according to PROP taster status, or polymorphisms of TAS2R38, CD36 (rs1761667) and OBPIIa (rs2590498) genes. Repeated-measures ANOVA was used to analyze the percentage increase of perceived intensity ratings by subjects in which the three concentration of L-Arg (5, 10 and 15 mmol/L) were effective, also as function of PROP-taster status, or polymorphisms of TAS2R38, CD36 (rs1761667) and OBPIIa (rs2590498) genes. Post hoc comparisons were conducted with the Fisher\\u2019s least significant difference (LSD) test, unless the assumption of homogeneity of variance was violated, in which case the Duncan\\u2019s test was used. Statistical analyses were conducted using STATISTICA for WINDOWS (version 7; StatSoft Inc, Tulsa, OK, USA) with 95% confidence interval. P values < 0.05 were considered significant.\",\n          \"Materials and Methods<SEP>Production and Purification of 20(OH)D3<SEP>20S(OH)D3 was generated by enzymatic hydroxylation of D3 catalyzed by CYP11A1 as previously described by our group. Briefly, 10 mM D3 in 45% 2-hydroxpropyl-\\u03b2-cyclodextrin was prepared. Buffer comprising 20 mM HEPES (pH 7.4), 100 mM NaCl, 0.1mM dithiothreitol, 2 \\u00b5M CYP11A1, 0.1 mM EDTA, 0.3 \\u00b5M adrenodoxin reductase, 10 \\u00b5M adrendoxin, 2\\u00a0mM glucose 6-phosphate, 2U/ml glucose 6-phosphate dehydrogenase, and 50 \\u00b5M NADPH was mixed with the D3 stock solution rendering a final D3 concentration of 200 \\u00b5M and 0.9% concentration of 2-hydroxypropyl-\\u03b2-cyclodexrtin. Following 8\\u00a0min pre-incubation, the reaction was initiated by adding NADPH, after which the samples were incubated for 3\\u00a0h at 37\\u00b0C with gentle shaking. Ice cold dichloromethane (20\\u00a0ml) was then added to stop the reactions after which the reaction products were extracted with dichloromethane as previously described. The 20(OH)D3 product was purified by preparative thin-layer chromatography, followed by reverse phase high performance liquid chromatography, as previously described. Routinely 0.3 mg of purified 20S(OH)D3 was recovered from 12.5\\u00a0ml of the starting incubation mixture. Aliquots of the purified 20S(OH)D3 were dried under nitrogen and stored at -80\\u00b0C until used.<SEP>Mice<SEP>Female DBA/1 Lac J mice, age 6 weeks old (Jackson Laboratories, Bar Harbour, ME) were housed in a pathogen-free AAALAC-approved animal care facility at the University of Tennessee Health Science Center (UTHSC) and Department of Veterans Affairs Medical Center (VAMC), Memphis, TN. Mice were fed regular laboratory chow and water ad libitam, and housed under a 12\\u00a0h light and 12\\u00a0h darkness cycle. Animal protocols for the study were approved by the Institutional Animal Care and Use Committees at UTHSC and VAMC Memphis.<SEP>Induction and Assessment of CIA<SEP>Mice were immunized with native bovine CII prepared (as previously described from fetal calf articular cartilage). Groups of 12 mice (for assessment of arthritis development) or groups of 5 or 6 mice (for other studies) were administered different amounts of 20S(OH)D3 dissolved in sterile sesame oil (S.O) or propylene glycol [(PG) (Sigma-Aldrich, St. Louis, MO)]. The S.O. or PG, with-or-without 20S(OH)D3, was administered daily intraperitoneally (i.p.) or via daily oral gavage in volumes of 50 \\u00b5l and 100 \\u00b5l, respectively. For studies using PG to solubilize 20S(OH)D3, the PG, with-or-without 20S(OH)D3, was diluted 1:5 by volume with sterile normal saline, and 100 \\u00b5l was administered daily by oral gavage. Rheumatrex tablets (DAVA Pharmaceuticals, Inc., Fort Lee, NJ) were used as a source of methotrexate sodium. Methotrexate 2.5 mg/kg was administered in 100 \\u00b5l normal saline weekly by oral gavage. The tablets were crushed and solubilized in sterile normal saline.<SEP>Arthritis severity was assessed in each paw every other day by two observers (one of whom was blinded to treatment) using the following scale: 0=no swelling or redness, 1=slight swelling and redness, 2=moderate swelling or redness, 3=marked swelling and redness, and 4=marked swelling and redness with some deformity. For histological assessment of joint tissue, mice were euthanized, all paws were removed, decalcified, processed and scored histologically, as previously described. Evaluation of each joint was done in a blinded manner using 4 parameters (0\\u20133 scale for each parameter): synovial inflammation and thickness, synovial leukocyte invasion into the joint, cartilage unevenness caused by inflammation related cartilage damage, and subchondral bone erosion. The total histologic score represented the sum of the 4 parameters. The maximal histologic score per mouse paw was 12 and 48 per mouse.<SEP>Quantitation of Anti-CII Antibodies in Sera<SEP>Specific murine IgG1, IgG2a, and Ig2b anti-CII antibodies were quantitated in sera using a commercial ELISA, according to the manufacturer\\u2019s instructions (Chondrex, Redwood, WA).<SEP>Flow Cytometric Assessment of Lymphoid Cells<SEP>Isolated cells from spleen or draining para aortic, popliteal or inguinal lymph nodes were labeled as follows: Alexa-700 labeled rat anti-mouse CD3, PE-Cy7-labeled rat anti-mouse CD25, Per CP-Cy5.5-labeled rat anti-mouse CD4, FoxP3 Tregs were detected by FoxP3 staining kit, and FITC-labeled rat anti-mouse CD-19 (BD Bioscience, San Diego, CA). Flow cytometry was performed on a SORP BD LSRII instrument and results analyzed by FlowJo.<SEP>Cytokine Quantitation<SEP>Draining lymph node cells were isolated and cultured at 2x106 cells/ml in RPMI 1640 medium containing 9% fetal calf serum, penicillin 100 u/ml, streptomycin 100 \\u00b5g/ml, 1% glutamax, 1% pyruvate, and 1% 2-mercaptoethanol. Supernatants were collected after 48\\u00a0h or 120\\u00a0h of culture and analyzed by multiplex sandwich immunoassay (Bio-plex mouse cytokine/chemokine kits, Bio Rad) for levels of different cytokines using a Luminex instrument according to the manufacturer\\u2019s protocol.<SEP>Quantitation of Total Serum Calcium<SEP>Levels of calcium in mice serum were quantitated by atomic absorption spectroscopy as previously described.<SEP>Statistical Analyses<SEP>Differences between groups were analyzed using 2 way RM ANOVA when multiple comparisons were made, Student\\u2019s 2-sample t-test was used for single comparisons between groups with normally distributed data, or by Mann-Whitney rank sum test for data not normally distributed. The level of significance was set at P < 0.05. \",\n          \"MATERIALS AND METHODS<SEP>Patients and controls<SEP>Peripheral blood (PB) samples were obtained from 279 unrelated consecutive patients diagnosed with AA (133 females and 146 males), according to the criteria of the International Agranulocytosis and AA Study. These AA patients were treated at a single institution (Hematology Branch, National Heart, Lung, and Blood Institute, National Institutes of Health), and composed of four ethnic groups (159 Caucasians (57.0%), 33African-Americans (11.8%), 49 Hispanics (17.6%), and 33Asians(13.6%)). Blood samples from ethnically matched 184 healthy donors (117 Caucasians (63.6%), 23 African-Americans (12.5%), 22 Hispanics (12.0%), and 22 Asians (12.0%) from SNP-500Cancer) served as controls. Patients or their guardians provided written informed consent for genetic testing, according to protocols approved by the institutional review board of the National Heart, Lung, and Blood Institute.<SEP>DNA amplification and sequencing<SEP>DNA was extracted from PB mononuclear cells using the Qiagen DNA extraction kit and subjected to PCR amplification using the LA Taq polymerase kit (Takara, Otsu, Japan) and adequate primer sets in order to obtain DNA fragments containing SNPs (rs11209026, rs41313262, and rs11465797). Primers for rs11209026 and rs41313262: 5\\u2032-GAGCAGAGTAAAGAGAATAGTAA-3\\u2032 (forward), 5\\u2032-TGGGCTGAGGACTTAGCCTCTTTAAGCCTC-3\\u2032 (reverse); primers for rs11465797: 5\\u2032-CAAGTAACTGGGATTACAGGCACATG-3\\u2032 (forward), 5\\u2032-CTTTACCTATATCATCCAGGT-3\\u2032 (reverse). Sequence analysis of the amplified DNA product was performed using the BigDye Terminator version 3.1 ready reactionkit and the ABI Prism 3100 Genetic Analyzer (Applied Biosystems, Foster City, CA). Sequencing primers are 5\\u2032-GACATTTGTAGAGAGTTTGGCATG -3\\u2032 for rs11209026 and rs41313262, and 5\\u2032-CTTCCATGCCTAGTGCGTTTGCTG-3\\u2032 for rs11465797. Sequencing data were aligned and compared using the SeqMan program (DYNASTER, Madison, WI).<SEP>Cytokine enzyme-linked immunosorbent assay (ELISA)<SEP>Sera were obtained from 23 AA patients with IL-23R polymorphisms, 37 AA patients without polymorphisms, and 21 healthy controls. IL-23 serum levels were measured using the Human Interleukin 23 (Hu IL-23) Heterodimer ELISA kit (BioSource, Camarillo, CA), following the manufacturer\\u2019s recommendations. All samples were assayed in duplicate.<SEP>Statistical analysis<SEP>Summary statistics, such as proportions and p-values were used to describe the related variables. Statistical analysis was performed by \\u03c72 tests with a significance level of P < 0.05, for SNPs and for ELISA data. All numerical results were computed using Splus 8 (Insightful Corp, Seattle, WA) and NCSS, PASS (Hintze. J, (2006). NCSS, PASS, and GESS. NCSS, Kaysville, Utah) for Windows statistical software.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"RESULTS\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1000,\n        \"samples\": [\n          \"Results<SEP>Demographics of patients<SEP>The mean age was 69 years (range: 19\\u201395 years) and 55.6 % were men. Patients were treated for atrial fibrillation (AF; 66.0 %), venous thromboembolism (VTE; 24.5 %), cardiovascular disease (comprising ischaemic heart disease, congenital cardiac failure, heart valve disease and other pathologies, total 7.6 %) or other conditions (comprising respiratory disease, cerebrovascular accident/transient ischaemic attack, gastrointestinal disease, neurological disease and other pathologies, total 2.1 %).<SEP>Clinical and biochemical factors<SEP>Seven patients failed the genotyping rate threshold, nine failed the heterozygosity criteria, 15 were excluded based on ethnicity using principal component analysis with HapMap3 samples as they did not cluster with samples of European ancestry (data not shown), three had ambiguous gender, three were excluded because of IBD, and four patients decided not to participate in the study, leaving 711 available genotypes for analysis. One hundred patients stopped treatment within the first 2 weeks of treatment, not allowing their inclusion for mean weekly dose calculation, though 13 with appropriate available data were included in INR >4 calculations.<SEP>Out of 612 patients with dose and INR information fitting our SMWD outcome, only 326 (53.3 %) reached stability over the follow-up period, which covered up to 277 days. One hundred and nineteen (16.6 %) patients were current smokers (N\\u2009=\\u2009711, mean number of cigarettes\\u2009=\\u200912.7, sd\\u2009=\\u20098.9). Out of 625, 116 (18.6 %) patients had an INR >4.0 during the first week of treatment. Out of the 711 patients retained for analysis, 160 had baseline data on full blood count, liver enzymes and coagulation factor levels. Weight and BMI were highly correlated (R2\\u2009=\\u20090.73), as were gender and height (R2\\u2009=\\u20090.83), current smoking status and number of cigarettes smoked (R2\\u2009=\\u20090.79); therefore, only the most significant variable of each pair was kept as a covariate in the regression analyses.<SEP>For MWD, age (P\\u2009=\\u20091.20\\u2009\\u00d7\\u200910\\u221217), height (P\\u2009=\\u20095.10\\u2009\\u00d7\\u200910\\u221207), weight (P\\u2009=\\u20095.02\\u2009\\u00d7\\u200910\\u221210), total follow-up time (P\\u2009=\\u20099.41\\u2009\\u00d7\\u200910\\u221203), number of cigarettes smoked per day (P\\u2009=\\u20095.34\\u2009\\u00d7\\u200910\\u221206), ex-smoker status (P\\u2009=\\u20093.01\\u2009\\u00d7\\u200910\\u221204), alcohol consumption (P\\u2009=\\u20092.00\\u2009\\u00d7\\u200910\\u221204) as well as the use of interacting co-medications (P\\u2009=\\u20096.30\\u2009\\u00d7\\u200910\\u221203), the sum of interactions (P\\u2009=\\u20091.38\\u2009\\u00d7\\u200910\\u221204), and the use of medications not classified as interacting (P\\u2009=\\u20092.94\\u2009\\u00d7\\u200910\\u221202) were found significant univariately and therefore adjusted for when testing for association with each SNP. MWD increased with height and weight, and decreased with age; it was higher for smokers and increased with the number of cigarettes smoked each day, while it appeared to be lower for ex-smokers in comparison to people who never smoked. MWD was lower in patients taking interacting co-medications, and decreased as the sum of interactions increased, but it also appeared to be lower in patients taking co-medications which are not on the interacting medications list.<SEP>In the subset of samples with extra baseline data, age (P\\u2009=\\u20092.06\\u2009\\u00d7\\u200910\\u221205), height (P\\u2009=\\u20092.43\\u2009\\u00d7\\u200910\\u221203), weight (P\\u2009=\\u20093.58\\u2009\\u00d7\\u200910\\u221204), haemoglobin (P\\u2009=\\u20093.64\\u2009\\u00d7\\u200910\\u221202), basophils (P\\u2009=\\u20099.20\\u2009\\u00d7\\u200910\\u221203), urea (P\\u2009=\\u20094.16\\u2009\\u00d7\\u200910\\u221202), ALT (P\\u2009=\\u20094.71\\u2009\\u00d7\\u200910\\u221202), Factor IX (P\\u2009=\\u20097.19\\u2009\\u00d7\\u200910\\u221203) and current smoking status (P\\u2009=\\u20093.36\\u2009\\u00d7\\u200910\\u221204) were found significant univariately and included as covariates in the linear regressions. Though not found univariately significant in this subset of patients, the use of interacting co-medications, the sum of interactions, and the use of medications not classified as interacting were also used as covariates in the linear regressions.<SEP>For SMWD, age (P\\u2009=\\u20096.20\\u2009\\u00d7\\u200910\\u221209), height (P\\u2009=\\u20091.72\\u2009\\u00d7\\u200910\\u221205), weight (P\\u2009=\\u20092.11\\u2009\\u00d7\\u200910\\u221207) and the use of interacting co-medications (P\\u2009=\\u20096.62\\u2009\\u00d7\\u200910\\u221203) were found significant univariately and adjusted for when testing for association with each SNP. SMWD increased with height and weight, and decreased with age, and was lower in patients taking interacting co-medications.<SEP>For INR >4.0 during first week of treatment (INR4), age (P\\u2009=\\u20092.77\\u2009\\u00d7\\u200910\\u221203), height (P\\u2009=\\u20091.53\\u2009\\u00d7\\u200910\\u221202), weight (P\\u2009=\\u20092.98\\u2009\\u00d7\\u200910\\u221204), loading dose (P\\u2009=\\u20097.28\\u2009\\u00d7\\u200910\\u221207), alcohol consumption (P\\u2009=\\u20097.76\\u2009\\u00d7\\u200910\\u221203) and use of medications (during first week of treatment) not classified as interacting (P\\u2009=\\u20094.28\\u2009\\u00d7\\u200910\\u221202) were found significant univariately and included as covariates in the logistic regressions. In addition to these, in the subset of patient with extra baseline data, triglycerides (P\\u2009=\\u20094.97\\u2009\\u00d7\\u200910\\u221202), albumin (P\\u2009=\\u20094.23\\u2009\\u00d7\\u200910\\u221202) and Factor VII (P\\u2009=\\u20093.82\\u2009\\u00d7\\u200910\\u221202) were also found significantly associated with INR4.<SEP>GWAS for MWD and SMWD<SEP>For warfarin MWD (Table\\u00a02 and Fig.\\u00a01a), genome-wide significant signals clustered on chromosome 16 around VKORC1, and on chromosome 10 around CYP2C9. The top signal in VKORC1 was rs9923231 (P\\u2009=\\u20095.00\\u2009\\u00d7\\u200910\\u221250). In CYP2C9, the strongest signal, rs4917639 (P\\u2009=\\u20092.05\\u2009\\u00d7\\u200910\\u221230) acts as a composite of CYP2C9*2 and *3; nevertheless, both *3 (rs1057910, P\\u2009=\\u20091.24\\u2009\\u00d7\\u200910\\u221223) and *2 (rs1799853, P\\u2009=\\u20091.65\\u2009\\u00d7\\u200910\\u221209) also reached genome-wide significance.<SEP>After adjustment for genetic and non-genetic covariates, no other SNP reached genome-wide significance (Fig.\\u00a01b). The previously described signal in CYP4F2, rs2108622 was absent from top hits after multiple regression (Table\\u00a02 and Fig.\\u00a01b; P\\u2009=\\u20097.70\\u2009\\u00d7\\u200910\\u221203), explaining only a further 0.5 % of the MWD variance. Examination of genes at proximity of other signals of interest did not yield any obvious candidates based on gene function. Canonical pathway analysis in IPA, however, using all signals with a P value below 10\\u221203, highlighted the thrombin signalling pathway, with 12 independent signals with P values ranging from 7.8\\u2009\\u00d7\\u200910\\u221204 to 2.2\\u2009\\u00d7\\u200910\\u221206 (Table\\u00a03), and a P value of 4.6\\u221204for the pathway as a whole. We performed univariate regressions to investigate if any of these 12 variants were linked to coagulation factor levels (data not shown). One variant, rs2298978, appeared significantly associated with Factor IX levels (P\\u2009=\\u20091.26\\u221203, Bonferroni threshold: 4.15\\u221203).<SEP>After stepwise regression including all significant non-genetic covariates as well as rs9923231, rs1057910 and rs1799853, ex-smoker status and alcohol consumption were not retained in the final model. Percentage of the MWD variance explained in the complete model was 57.4 % (breakdown was age 11.20 %, height 3.56 %, weight 5.98 %, follow-up time 1.20 %, number of cigarettes smoked/day 3.12 %, interacting co-medications 0.98 %, sum of effects of interacting medications 2.20 %, non-interacting medications 0.61 %, rs9923231 25.61 %, rs1057910 12.93 % and rs1799853 3.72 %). Inclusion of rs2108622 (CYP4F2) brought the total variation explained to 57.89 %.<SEP>In the 160 patients for which baseline blood count and coagulation factor levels were available, age, height, weight, haemoglobin, basophils, urea, ALT, factor IX and smoking status were found significant in the univariate regression on MWD, and included as covariates in the multiple regression models. As expected given the low power, only rs9923231 (VKORC1) reached genome-wide significance, as well as an imputed SNP in the CYP2C9 locus (rs1072753, R2\\u2009=\\u20090.61 with rs1057910, CYP2C9*3, in 1000 genome project pilot 1) (Additional file 3). CYP2C9*3, instead of the imputed rs1072753, was used in the stepwise regression. Only age (13.39 %), weight (7.58 %), Factor IX (4.37 %), smoking status (9.98 %), rs9923231 (18.15 %) and rs1057910 (14.97 %) were retained in the final model, explaining a total of 59.64 % of the MWD variation.<SEP>For warfarin SMWD (Additional file 1 Sheet 3), the univariate signals (Fig.\\u00a02a) were rs9923231 (P\\u2009=\\u20099.26\\u2009\\u00d7\\u200910\\u221232) for VKORC1, and rs4917639 (composite CYP2C9 *2/*3, P\\u2009=\\u20095.26\\u2009\\u00d7\\u200910\\u221215) and rs1057910 (CYP2C9*3, P\\u2009=\\u20093.95\\u2009\\u00d7\\u200910\\u221209) for CYP2C9, while rs1799853 (CYP2C9*2, P\\u2009=\\u20094.70\\u2009\\u00d7\\u200910\\u221207) did not reach genome-wide significance. Despite not reaching genome-wide significance, rs1799853 was included in the multiple regression as it influences warfarin dose, along with rs9923231 and rs1057910. Three signals reached genome-wide significance after adjusting for genetic and non-genetic factors (Fig.\\u00a02b): 6\\u2013106244024 (P\\u2009=\\u20096.15\\u2009\\u00d7\\u200910\\u221209), 11\\u201315383178 (P\\u2009=\\u20096.23\\u2009\\u00d7\\u200910\\u221209) and rs112942398 (P\\u2009=\\u20092.40\\u2009\\u00d7\\u200910\\u221208). All these SNPs are the result of imputation; there is close to no support from surrounding genotyped SNPs for 11\\u201315383178 and 6\\u2013106244024 (data not shown), unlike for rs112942398 (Additional file 4), which is gene-rich, but none of the genes had a link to warfarin metabolism or blood coagulation. To further assess the validity of rs112942398, 94 samples were genotyped using a KASPar custom assay; 10 of the aforementioned samples were given as homozygous for the minor allele by the imputation, and 42 were given as heterozygous, the remainder being given as homozygous for the major allele. Four samples failed genotyping, leaving 90 genotypes available for comparison. There were 12 discordant genotypes between imputation and KASPar genotyping, giving an approximate imputation error rate of 13.3 %.<SEP>All covariates from the multiple regression remained in the final model after stepwise regression, with age explaining 9.72 % of SMWD variation, height 4.45 %, weight 7.94 %, interacting co-medications 1.92 %, rs9923231 30.38 %, rs1057910 8.08 % and rs1799853 3.81 %. The final model explained 55.27 % of the variation. Inclusion of rs2108622, retained after stepwise regression, brought the total variation explained to 56.97 %.<SEP>GWAS analysis for INR >4.0 in the first week<SEP>Age, height, weight, loading dose, alcohol consumption and medications not known to interact with warfarin were significant in the univariate analysis and were adjusted for when testing for SNP associations. Two regions harbour signals reaching genome-wide significance (Additional file 1 Sheet 4 and Fig.\\u00a03a): the lowest P value was observed for rs2288004 (P\\u2009=\\u20098.76\\u2009\\u00d7\\u200910\\u221214) on chromosome 16, an imputed SNP in high LD with VKORC1\\u2019s rs9923231 (R2\\u2009=\\u20090.98), followed by rs1072753 (P\\u2009=\\u20092.85\\u2009\\u00d7\\u200910\\u221208), an imputed SNP on chromosome 10 in high LD with CYP2C9*3 (rs1057910, R2\\u2009=\\u20090.78). After accounting for rs2288004 and rs1072753 and non-genetic factors, no other signal reached genome-wide significance (Fig.\\u00a03b). Individuals homozygous for rs9923231 minor allele had an odds ratio of 8.04 (4.43\\u201314.90, N\\u2009=\\u200983) for having an INR >4.0 during the first week of treatment, while the odds ratio for heterozygous patients was 2.22 (1.34\\u20133.78, N\\u2009=\\u2009287), in comparison to patients homozygous for the major allele (N\\u2009=\\u2009255); similarly, the odds ratio for patients homozygous for rs1057910 minor allele was 24.17 (3.66\\u2013644.29, N\\u2009=\\u20096) and 2.93 (1.70\\u20134.96, N\\u2009=\\u200974) for heterozygous, reported against homozygous patients for the major allele (N\\u2009=\\u2009545). Two signals not quite reaching genome-wide significance appear in regions of biological relevance to clotting; rs747180 (P\\u2009=\\u20091.09\\u2009\\u00d7\\u200910\\u221206) is located in an intron in APLP2, which has been linked to haemostasis through its inhibitory effect on Factor XIa, and rs6809892 (P\\u2009=\\u20092.94\\u2009\\u00d7\\u200910\\u221206) is located near TFRC, a gene implicated in the development of erythrocytes.<SEP>Sensitivity analyses<SEP>There were 33 patients with at least one clinical value over three standard deviations from the cohort mean. We repeated all the regression analyses after excluding these outlier patients; this did not significantly alter any of our findings (data not shown).\",\n          \"Results<SEP>The numbers of genotyped individuals for the two independent samples used in the association analysis are shown in Table 2. The ascertainment of the CIDI sample means that both the cases and controls are likely to be more extreme that those of the SSAGA sample, and this is reflected in the summary statistics based on EPQ neuroticism and harm avoidance (Table 3) collected several years earlier.<SEP>Linkage disequilibrium around 5HTTLPR<SEP>None of the markers showed evidence for Hardy-Weinberg disequilibrium. Figure 2 shows the linkage disequilibrium as both |D\\u2019| and r2 measures; although the |D\\u2019| values are high, the corresponding r2 values are lower reflecting differences in allele frequencies. The best prediction of 5HTTLPR is provided by rs4251417 and rs2020934 (Figure 2b), where the CA haplotype is coupled with the S allele and the CG haplotype is coupled with the L allele of 5HTTLPR (r2 = 0.72). No useful prediction could be made of 5HTTLPR-rs25531 or of rs25531 within LL individuals. Genotype data are available from the corresponding author on request.<SEP>Association analysis<SEP>SNPs rs6354 and rs2020936 (markers 4 and 6) show association (p < 0.05) with all of the depression and anxiety phenotypes presented in Figure 1, with the T and A alleles of these SNPs respectively, being more frequent in cases than controls (0.82 vs 0.72, Odds ratio (OR) = 1.19 (1.03 \\u2013 1.38)). These SNPs are in high LD (r2 = 0.98) and so represent a single association signal. Although the associations for the two independent data sets are only significant for some analyses the direction of the association is the same, so that the association in the combined data set is significant. The association with Depression is marginally significant, OR = 1.17 (1.01 \\u2013 1.26). However, the association with the two subsets of cases thought to represent genetically more homogenous groups, both show higher association odds ratios: recurrent, early onset depression (Figures 3e and 4), OR = 1.55 (1.16\\u20132.06), and co-morbid group of Anxiety with Depression (Figure 3d and 4), OR = 1.31 (1.02\\u20131.69). Permutation analysis showed that this pattern of association occurred only 62 times in 10,000 permutations across all markers under the null hypothesis, resulting in an empirical p-value which accounts for multiple testing of 0.0062. Odds ratios of association with rs6354 for the different anxiety disorders are presented in Figure 4. These results must be interpreted recognising the constraints on possible diagnostic classes from the questionnaires. Necessarily these results should be considered as hypothesis generating for future studies. The associations for rs6354 and rs2020936 with neuroticism were 0.019 and 0.015 respectively (Figure 3f), but there is no evidence for association between either harm avoidance or extraversion which had phenotypic correlations of 0.66 and \\u22120.25, respectively, with neuroticism. Association (p < 0.05) is seen for rs4251417 and harm avoidance. Haplotype analysis of combinations of SNPs revealed no associations more significant than those of the individual genotyped markers.<SEP>For the association analysis in the CIDI sample we observed association (p <0.05) for the 5HTTLPR (L allele), most significantly with the Depression cases. We interpret this as a chance result as the association is not replicated in the SSAGA sample for which any trend in association is with the S allele. A detailed investigation is provided in Supplement 1. We find no evidence for association of 5HTTLPR (with or without rs25531) with panic/agoraphobia, OCD, neuroticism, extraversion or harm avoidance (Figure 3 e and f). As a consequence, our results provide no evidence, either way, for the usefulness of genotyping of SNP rs25531 to subclassify the L allele. A full report of allele and genotype frequencies and p-values is available online (Supplement 2).\",\n          \"RESULTS AND DISCUSSION<SEP>To assess the contribution of IL-23 SNPs to AA susceptibility and phenotype, we first targeted the non-synonymous rs11209026 SNP (c.1142G>A, p.Arg381Gln) because of its strong association withIBD. In addition, we examined allele frequencies of two non-synonymous SNPs, rs41313262 (c.1084G>A, p.Val362Ile) and rs11465797 (c.524C>A, p.Thr175Asn). Genotype frequencies of the three IL-23R polymorphisms examined in AA patients and healthy controls are shown in Table 1, together with demographic and clinical features. The p.Arg381Gln variant was present in 6.1% ofAA patients and in 6.0% ofethnically matched controls (P = 0.960; \\u03b2-error =0.16). The carrier frequencyof the p.Val362Ile (rs41313262) variant was low, but similar inAA patients (2.2%) and controls (1.6%) (P = 0.958); the p.Thr175Asn (rs11465797) variant was not observed in patients or controls. No homozygous carriers of the p.Arg381Gln or p.Val362Ile variant were found in either AA patients or controls. All carriers had only a single SNP, except one with two SNPs.<SEP>We performed ELISA analysis to measure IL-23 levels in sera of 23 AA patients with SNP polymorphisms and 37 AA patients without polymorphisms, and compared results with those of 21 healthy controls (Figure 1). There wereno statistical differences among thethree groups, even after stratifying AA patients according to treatment status.<SEP>Collectively, our dataindicatethat the Arg381Gln SNP did not correlate withsusceptibility ofAA.<SEP>The IL-23/IL-17-mediated inflammatory axis has been implicated in numerous autoimmune diseases and inflammatory conditions. A series of cytokines, including IL-6, IL-21, TGF-\\u03b2, and IL-23, sequentially or synergistically induce Th17 lineage cells, together with the cooperative action of transcription factors. Conversely, other cytokines, including IL-2, IL-4, IFN-\\u03b3, and IL-27, antagonize differentiation of the Th17 lineage and inhibit IL-17 and IL-23 production. In AA patients, increased productionof type 1 cytokines (IFN-\\u03b3, TNF, and IL-2) from T cells has seenobserved, implicating that the hematopoietic cell destruction bya Th1 T-cellresponse, which includes boththe Fas-mediated cell death and the inhibition of hematopoieticstem cell proliferation. Our previous work has suggestedthat T-bet mediated IFN-\\u03b3 elevation is responsible for the Th1 shift of CD4+ T cells in AA patients. As CD4+ effecter T cells were shifted to Th1 cells in AA patients, the IL-23/IL-17-mediated inflammatory axis might be suppressed in AA, due to antagonistic effect ofTh1 cytokines onTh17 differentiation and development. Therefore, it is consistent that IL-23R SNPs have no significant association with AA, although it cannot be excluded that other IL-23R SNPs as well as IL-23R isoforms (differentially expressed in distinct cell types) might play protective roles.<SEP>In other autoimmune diseases such as MS, RA, and SLE, similar results have beenobtained, IL23R polymorphisms had no significant correlation with susceptibility or severity of disease, in agreement withour results. It has beensuggestedthat the IL23R polymorphisms might play a more important role in regulating local inflammation, as in IBD and psoriasis, rather than in systemic inflammation in RA and SLE, supported by some work in mouse models. Further, the IL-23-pathway molecules have no association with SLE; type I cytokines, which are not induced withIL-23, are relevant in the development and maintenance of this disease. AA patients may share a common background of genetic risk affecting disease development and severity, which may be similar to some other autoimmune diseases and completely different from others. Further genetic studies are required to investigate as yet unidentified causal genetic alleles responsible for AA.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"DISCUSS\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 945,\n        \"samples\": [\n          \"DISCUSSION<SEP>T-ALL is an aggressive hematological cancer resulting from leukemic transformation of thymocytes. Although there has been a remarkable increase in our knowledge of T-ALL molecular pathogenesis, the identification and characterization of the players and mechanisms driving proliferation and survival of leukemia T cells remains relatively poor. IL-7 and its receptor are essential for normal T-cell development and have been suggested to play a role in T-ALL. In the present study, we showed that 9% of pediatric T-ALL cases have IL7R exon 6 mutations that are gain of function and have oncogenic ability. Thus, our findings expand the spectrum of disease-associated IL7R genetic alterations to cancer. Moreover, this is the first example of an oncogene in the \\u03b3c family of cytokine receptors, which is critically involved in numerous lymphoid cell functions.<SEP>Notably, IL7R mutations do not occur in the cytoplasmic tail, which recruits signaling effectors, but do occur at the extracellular juxtamembrane-transmembrane interface. The vast majority of IL7R mutations we identified create an unpaired cysteine residue, which is necessary for disulfide-bond\\u2013dependent IL-7R\\u03b1 homodimerization and bypasses the requirement for ligand binding and \\u03b3c heterodimerization to trigger downstream signaling. Moreover, all IL7R mutations insert additional amino acids rather than involving a single amino acid change to cysteine. This may indicate that these additional amino acids are required for the optimal conformation leading to maximal signaling, perhaps by allowing for the most adequate alignment and exposure of the unpaired cysteine and/or by maximizing the interactions between downstream effectors at the cytoplasmic tail of the receptor. The three remaining T-ALL cases resulted in the inclusion of either a tryptophan or an SxxxG motif in the transmembrane domain. Although we did not analyze the mechanisms by which these mutations might contribute to T-cell leukemia, tryptophan residues and SxxxG motifs have both been reported to promote association of transmembrane helices that could result in homodimer or heterodimer formation with possibly similar outcomes to cysteine mutations. However, preliminary analyses of mutant P5, which has the insertion of a tryptophan in the transmembrane domain (Table 1), suggest that it does not form dimers (data not shown) and suggest that the pro-survival effect of this mutation is relatively minor: P5 expression in D1 cells deprived of IL-7 for 48 h resulted in a 2.8-fold increase in viability relative to wild-type IL-7R, as compared to 7.4-, 9.1- and 6.0-fold increases for P1, P2 and P4, respectively. In accordance, P5 appears to be relatively inefficient at inducing constitutive signaling as compared to the other IL7R mutations (Supplementary Fig. 7). These results suggest that IL7R mutations not involving cysteine insertion are not as potent and probably require additional cooperating oncogenic events compared to those that result in the introduction of an unpaired cysteine, which constitute the vast majority of the cases we identified in childhood T-ALL and characterized in our study.<SEP>IL7R gene alterations appear to be highly predominant in T-cell compared to B-cell leukemia. We did not detect exon 6 mutations in any of the 50 childhood pre\\u2013B-ALL cases we analyzed, and a recent report indicated that IL7R mutations occur in only 0.6% pre\\u2013B-ALL cases. In contrast to T-ALL, half of the B-cell\\u2013associated mutations affect exon 5, rather than exclusively exon 6, and require cooperation with TSLPR (CRLF2) overexpression. TSLPR expression is rare in T-ALL and is not necessary for signaling driven by the IL7R mutations, as we showed here in 293T cells, which do not express TSLPR yet display constitutive signaling after expression of mutant IL7R. Notably, the fact that IL-7R\\u03b1 is apparently expressed in various carcinoma cell lines and breast cancer tissue raises the intriguing question of whether mutations in IL7R might also occur in solid tumors.<SEP>We found IL7R mutations in different T-ALL oncogenetic subgroups, but they tended to associate predominantly with HOXA aberrant expression. Although the exact biological importance of this link remains to be fully understood, it is noteworthy that Hoxa9\\u2212/\\u2212 mice show impaired early T-cell development with reduced Bcl-2 and IL-7R\\u03b1 expression. Curiously, IL7R gene alterations did not associate with T-ALL maturation stage or with most T-cell differentiation markers. These observations are reminiscent of the fact that primary T-ALL cells, in contrast to normal developing thymocytes, respond to IL-7 independent of their maturation stage.<SEP>We showed that pharmacological inhibition of the JAK-STAT pathway induces cell death of mutant IL-7R\\u03b1\\u2013expressing cells. The preliminary data on the effect of these inhibitors in one primary T-ALL case sample was evident but not as striking as in cell lines. This may relate to the early time point at which we assessed viability (which may have prevented the inhibitors from having the maximal effect), to the importance of other alternative downstream signaling pathways in the regulation of cell survival in primary leukemia and/or to higher dependence on other oncogenic defects in the leukemia cells of the case analyzed. Irrespective of these considerations, our results suggest that JAK-STAT pathway inhibitors are cytotoxic to mutated IL-7R\\u03b1\\u2013expressing T-ALL cells. Whether inhibitors of other signaling components activated by gain-of-function IL7R mutations, such as Akt, can be exploited on their own or in combination with JAK-STAT antagonists to target IL7R mutant T-ALL cells requires further investigation.<SEP>The extraordinary improvement in T-ALL treatment outcome in recent years is mitigated by the long-term side effects associated with current regimens and by the dismal prognosis of relapsed cases. Further improvement requires an in-depth understanding of T-ALL molecular genetics and leukemogenic pathways, which will ultimately lead to the identification of new molecular players and to the development of effective targeted therapies. This line of reasoning has led, for instance, to the identification of CREBBP (CBP) mutations that are associated with ALL relapse or CRLF2 rearrangements, which are particularly frequent in Down syndrome ALL. PTPN2 and PHF6 mutational loss are among the most recently characterized genetic lesions involved in T-ALL. Our present work, and that of a parallel study, indicates that IL7R mutational activation takes part in human T-cell leukemogenesis, thereby expanding the spectrum of genetic alterations in T-ALL to a long recognized major regulator of lymphoid biology. Notably, our findings provide a strong rationale for specific targeting of IL-7R\\u2013mediated signaling as a treatment option for T-ALL.<SEP>URLs.<SEP>LIMMA, http://bioconductor.org/packages/release/bioc/html/limma.html.\",\n          \"4. Discussion<SEP>In an era of highest rate of age standardized global incidence of prostate cancer recorded from New Zealand, it is inevitable that we should be looking at every possible source to ease this problem. Restoring general prostate health in New Zealand men could indirectly reduce this long-term public health burden. Meanwhile, New Zealand is known for soil deficiencies in selenium that affects the dietary content of this trace mineral. Our previous studies with New Zealand men have shown that benefits of 200 \\u03bcg/d selenium supplementation for six months in preserving DNA integrity is achieved only in stratified groups of men. We have also shown that DNA integrity benefits vary between serum selenium levels between 116 and 150 ng/mL depending on genetic variation. The current analysis looked into the utilization of supplemented selenium in changing serum selenium levels and subsequent changes in the stability of prostate glandular epithelium microarchitecture, using PSA as a surrogate biomarker.<SEP>We observed a significant positive dependence of post-supplementation serum selenium level, on the baseline serum selenium level. When stratified by genotypes, the GPX1 rs1050450 TT, AKR1C3 rs12529 CG and SEL15 rs5845 CT showed the best correlation between the baseline and post-supplementation serum selenium levels. This implies that the individual serum selenium carrying capacity could be varying with the genotypes.<SEP>Beyond the highest gain in serum selenium by supplementation of 126 ng/mL, the overall gain in serum selenium by supplementation declined at a rate of 0.828 ng/mL with each one ng/mL increase in baseline serum selenium level. This means that part of the supplemented selenium would have passed though different routes in addition to replenishing the serum component. Among such possible selenium channeling routes may include, excretion; metabolism by gut biota to forms that reduce absorption into circulation; used up for toxicity modulation; channeled towards other tissue optimization including that of the prostate; or the selenomethionine component in the selenized yeast getting non-specifically incorporated into a wide range of tissue proteins in place of methionine. Takata et al., 2009 have shown that serum selenium level is a reflection of the levels in the prostate. Our previous studies with the current study cohort have shown that selenium supplementation led to no gain in red blood cell GPX activity, indicating, this has been optimized with the baseline selenium levels; although the thioredoxin reductases activity was upregulated with selenium supplementation. This is in line with the whole body selenium regulation as discussed by Burk and Hill 2015.<SEP>There was no overall significant correlation between the difference in serum selenium after supplementation and subsequent difference in PSA level, although significant correlations were recorded among, men below the median age of 55 years as well as the group who were never-smokers. Our previous studies have shown that age based PSA increase in men with prostate cancer beyond 69 years could be due to the increase in extra-testicular androgen production from adrenal dehydroepiandrosterone (DHEA)-based androgen precursors catalyzed by the AKR1C3 enzyme, particularly among men carrying the AKR1C3 rs12529 CG and GG genotypes. Therefore, our current observation of a significant inverse correlation between the difference in serum selenium and difference in PSA level after selenium supplementation among men below the age of 55 years may indicate that increased selenium level does not support a significant reduction in PSA when the extra-testicular androgen production starts to increase with advancing age. A placebo-controlled study tested against 200\\u00b5g selenium daily for 8 weeks with women aged 10\\u201340 years with polycystic ovarian syndrome has recorded decreased serum DHEA levels with selenium supplementation. This further implies that at an age below 55 years, PSA decline by selenium supplementation may happen due to indirect DHEA suppression. Among ever-tobacco smokers, supplemented selenium may get triaged towards seleno-antioxidant enzyme activity over prostate glandular epithelium stability, restricting prostate health benefits to never-smokers.<SEP>When data were stratified by the SNP genotypes, we see a PSA reducing benefit by supplementation only with the GPX rs1050450 CT and TT genotypes. The EPIC-Heidelberg nested prostate cancer case-control study indicates that men with the variant T allele of the GPX1 rs1050450 polymorphism record a risk ratio (odds ratio OR) of 0.87 per 10 ng/mL increase in serum selenium. The authors also showed that the prostate cancer risk ratio was further reduced (OR, 0.64) per 10 ng/mL increase in serum selenium among men with high-grade prostate cancer carrying the rs1050450 variant T allele, while no such risk modifications were observed with increasing serum selenium with the GPX1 rs1050450 dominant allele C.<SEP>PSA is exclusively produced by the glandular prostate epithelium, and is abundant in the seminal fluid. PSA production is signaled by androgen activated androgen receptors. According to Prcic and co-workers, PSA reaches serum when the microarchitecture of prostate glandular tissue is disrupted. According to these authors increased levels in circulation is an indication of prostate disease or trauma in the prostate gland including prostate cancer. Therefore, the decline in PSA with increase in serum selenium among men carrying the GPX1 rs1050450 T allele in our study is an indirect reflection of prostate health benefits that may parallel prostate cancer risk reduction as recorded by the Steinbrecher et al. study.<SEP>In a previous study, an inverse trend in PSA change with increasing baseline serum selenium level was recorded by us for a group of New Zealand men with benign prostate issues but with negative biopsies, and carrying a PSA >4 ng/mL at recruitment. When these men were supplemented with either a 200 or 400 \\u00b5g/d selenium as selenized yeast or placebo for six months, two thirds of the cohort showed a reduction in the rate of PSA progression. Zhang et al., 2011 reported that a 3-month supplementation with 200 \\u03bcg/d selenium (in the form of glycinate) increased plasma and erythrocyte glutathione peroxidase (GPX) and lowered serum PSA levels. The Procomb study from Italy, where men with lower urinary tract infections (UTI) randomized to receive either a Serenoa repens plant extract containing 50 \\u00b5g of selenium and 5 mg of lycopene or a placebo for over one year showed no significant change in PSA between the test and the placebo groups. It is a possibility that this combination supplement with 50 \\u00b5g/d of selenium was not sufficient to influence a pharmacological benefit of selenium in PSA reduction in these men with UTI. A six-month 55 \\u00b5g/d of selenium supplementation with 35 mg/d of lycopene and 600 mg/d of polyphenol supplementation is reported in a group of men diagnosed with isolated high-grade prostatic intraepithelial neoplasia HGPIN on biopsy. These authors record increased incidence of prostate cancer and markers indicating prostate cancer progression in men receiving the supplements. A six-month supplementation of 100 \\u00b5g/d selenium, 30 mg/d vitamin E, and 50 mg/d soy isoflavonoids in men diagnosed with HGPIN has resulted in a stable PSA within three months in 4.2% of the supplemented group, while a decrease in PSA level was recorded in 57.8%. These authors also record that 34%, 48% and 18.3% of these men having either prostate cancer or remaining with HGPIN or no HGPIN or carcinoma respectively after six months. They further recorded that the prostate cancer risk remained at 25% among men having a stable or decreasing PSA due to supplementation (68%) while the risk reaching 52% in the group with increasing PSA after supplementation (32%). Richie et al., 2014 supplemented 200 \\u00b5g/d selenium as selenomethionine or 200 or 285 \\u00b5g/d selenized yeast or placebo for nine months in a group of 69 healthy men, in the age range of 20\\u201379 y who were non-smokers and carrying PSA levels (<4 ng/mL) at baseline. The authors report no change in PSA level compared to the baseline in any of the groups consisting of 15\\u201320 men. Contrary to 110 ng/mL baseline median serum selenium (or 113 ng/mL mean serum selenium) level in our cohort, their cohort recorded a higher mean baseline plasma selenium level ranging from 129 to 139 ng/mL. Our cohort gained a post-supplementation median serum selenium levels ranging from 157 ng/mL among the KLKrs17632542 CT genotype carriers to 181 ng/mL in SEL15 rs5845 TT genotype carriers within six months of supplementation, while the Richie et al., (2014) study recorded mean post-supplementation levels above 200 ng/mL within six months for all three types of supplements. Waters and Chiang (2018) discuss the influence of both inadequate as well as excess selenium in prostate health in dogs. The inverse correlation of serum PSA level with increasing serum selenium level in our cohort with GPX rs1050450 T alleles could be due to our men with lower baseline serum selenium levels showing a differential benefit compared to men from Ritchie et al. study. We have previously recorded that tobacco smoking is associated with increased levels of serum PSA levels in New Zealand men. The impact of tobacco smoking on prostate tissue blood perfusion and vascular injury have been reported in men with benign prostatic hyperplasia. It is a possibility that increased serum PSA due to tobacco smoking related prostatic glandular epithelium damage could not be circumvented by the selenium supplementation protocol followed by us.<SEP>The post-supplementation selenium level showed a significant difference among those carrying the SEL15 rs5845 genotypes, with the TT genotype showing the highest level. However, this is contrary to the findings in studies with functional gene constructs, where selenium supplementation response was shown superior with the SEL15 rs5845 C allele. A US study by Ekoue et al., 2018 have reported that AAs have a higher frequency (34%) of the TT genotype compared to EAs (3.4%), and this genotype distribution in our New Zealand cohort is similar (3.2%) to the latter group. These authors also record that AAs record a lower level of SEL15 protein in their prostate tissue samples and in the serum compared to the European Americans, and that SEL15 levels are generally lower in prostate cancer tissue compared to adjacent benign tissues. They also record that each ng/mL increase in serum selenium level increases the odds ratio for Gleason sum by 2% in men carrying the SEL15 rs5845 TT genotype, while in men carrying the CC genotype a five-fold decrease in the odds ratio of higher Gleason sum is recorded. Considering the highest retention of serum selenium by supplementation among men carrying the SEL15 rs5845 TT genotype in our cohort without an accompanying decline in PSA level, they may be a group that should consider serum selenium adjustments cautiously.<SEP>The interaction of a panel of dietary nutrients associated with DNA methylation, shows that intakes above the RDI for zinc, benefitting with a significant reduction in PSA levels with increasing selenium levels due to supplementation. Reduction in PSA with increasing serum selenium levels was more stringent with vitamin B12 levels below the RDI. If the selenium supplementation-related PSA reduction was at least partially a consequence of increasing the activity of the antioxidant seleno enzyme GPX, it is imperative that nutrient levels of both selenium and zinc be sufficient. The redox action of seleno enzyme GPX is preceded by prior production of H2O2 through the superoxide dismutases. Such superoxide dismutases include the mitochondrial MnSOD and cytosolic Cu/Zn SOD, which catalyze reduction of the superoxide radicals to H2O2. Therefore, besides being an important molecule in the DNA methylation process, zinc is also an important trace mineral in the Cu/Zn SOD metaloenzyme, supporting redox cycling of reactive oxygen species. Findings of the 2008/09 New Zealand Adult Nutrition Survey indicates that 39% of men had inadequate intakes of zinc (median daily intake 12.9 mg) while in the category of men >71 years, this inadequacy was common among 89.7% of men. The New Zealand Adult Nutrition Survey indicates that 32% of the New Zealand males also have inadequate intakes of selenium (67 \\u03bcg/d that is closer to our current estimate of 65 \\u03bcg/d). The comparative dietary intake of selenium recorded for US males >20 years was 135.9 \\u03bcg/d. The major source of selenium in New Zealand diets comes from breads (15%) followed by fish, and sea food (12%) and poultry (10%). An inadequate intake of vitamin B12 recorded in the 2008/09 New Zealand Adult Nutrition Survey was only 1.3% for men whereas our cohort had 4.4% men with this inadequacy. The above details indicate the importance of dietary nutrient adjustments in achieving optimum prostate glandular epithelial health benefits of selenium as shown through the surrogate serum PSA marker.<SEP>Our study implies that it could be beneficial for New Zealand men carrying the GPX1 rs1050450 T alleles, to get their serum selenium levels adjusted, at least to reach the level between 116 and 150 ng/mL as estimated by us as the optimum level for DNA integrity. Besides, our data indicate the importance of serum selenium level adjustments in men carrying the GPX1 rs1050450 T alleles for prostate glandular epithelial architecture stability that can be monitored with serum PSA levels, rather than using a set dose of selenium supplements. This study also highlights the importance of other nutritional requirements such as dietary zinc and vitamin B12 in maximizing selenium related benefits on prostate health.<SEP>Among the shortcomings of the current study are the duration of the study being limited to six months, the nutrient measures except serum selenium were assessed based on the four-day diet and activity diaries that may not provide an accurate nutrient profile in blood of the participants. Another shortcoming is the nutrient profiles were analysed only with a limited panel associated with DNA methylation. It will be beneficial to assess a wider range of serum/plasma based nutrient profiles for better understanding of the interactive nature of nutrients with selenium supplementation and PSA outcomes. Our study has also not recorded urinary selenium excretion, which would have further supported our claim on variable supplemented selenium retention levels based on baseline serum selenium level. As prostate cancer has a long aetiology, it is important that men from this New Zealand selenium supplementation study be prospectively monitored for any future prostate cancer recordings, and to understand the interactive genetic, nutrient and serum PSA measures in such prostate cancer outcomes.\",\n          \"4. Discussion<SEP>Determinants of vitamin D status in morbidly obese individuals were examined, and factors considered were dietary vitamin D intake, BMI, skin colour, and sun exposure. The key findings were (i) natural skin colour accounted for 13.5% of the variation in serum 25(OH)D concentrations; (ii) there was a significant positive association between age and serum 25(OH)D; (iii) weight and BMI were not significantly associated with serum 25(OH)D concentrations; (iv) there was no relationship between sun exposure time, or amount of skin exposed, with 25(OH)D concentrations in this group. <SEP>Natural skin colour (ITAC score) accounted for 13.5% of the variation in serum 25(OH)D concentrations. Results showed that as natural skin colour becomes darker, serum 25(OH)D concentrations increased, suggesting those with darker natural skin colour have higher 25(OH)D concentrations. ITAC score was also used to predict maximal mean serum 25(OH)D concentrations for each skin colour category. It is well documented that those with darker natural skin colour have lower serum 25(OH)D concentrations. As this study had only recruited participants with an intermediate skin colour or lighter, it is possible the increase in 25(OH)D concentration with increasing natural skin colour was due to sun protective behaviours from those with lighter skin. For similar reasons, the predictive model became unreliable when dealing with those in the tanned and brown skin colour categories. Although when comparing changes in ITA score i.e., degree of tanning, sun exposure times, and %BSA, there was no correlation between any of these measures and natural skin colour, potentially suggesting no differences in sun protective behaviours regardless of natural skin colour.<SEP>A trend towards significance was seen between tanned skin colour (ITAF score) and serum 25(OH)D concentrations. Tanned skin colour accounted for 7.2% of the variation in serum 25(OH)D concentrations in this group. As ITAF score increased (i.e., darker tan), 25(OH)D concentrations increased, suggesting that those with a darker tan (up to intermediate) will have higher 25(OH)D concentrations. Previous research has shown that tanned skin colour is a significant predictor of 25(OH)D. This result came from a larger group but with similar proportions of participants in each skin colour group. Our results confirm those of Rockell et al., showing that for each 10\\u00b0 decrease in ITAF score (i.e., darker tan) serum 25(OH)D concentrations increase by 5 nmol/L.<SEP>The Australian Health Survey (2011\\u201312) showed similar rates of deficiency between genders, and an increase in vitamin D status with age, concurrent with increases in supplement use. Population data in Australia show varying effects of age on serum 25(OH)D, with some showing an increase with age and others a decrease with age. There was a significant positive association between age and serum 25(OH)D concentrations in our study. Further investigation of the variables showed that age positively correlated with natural skin colour and tanned skin colour, suggesting that the older participants had darker natural and tanned skin colour. There was no relationship between age and degree of tan, or %BSA exposed to the sun, or sun exposure time.<SEP>Weight and BMI were not significantly associated with serum 25(OH)D concentrations. There are several meta-analyses that have shown a significant decrease in serum 25(OH)D concentrations with increasing weight or BMI, although not all included morbidly obese individuals. There are several studies in morbidly obese individuals that found a significant or borderline significant inverse relationship between BMI and 25(OH)D concentrations, with small to large effect sizes. It is possible that once a certain BMI threshold is reached, the dilution effects of obesity on vitamin D status plateau, and the effect becomes minimal, hence the trend of lower serum 25(OH)D in this study, but the lack of a significant difference. Sun exposure, or lack of it, appears to have a major influence on vitamin D status in Australian obese populations. A study into determinants of serum 25(OH)D in Australian Adults reported that the amount of skin exposed to the sun was the single largest contributor to serum 25(OH)D concentrations, followed by location, season, and personal UV radiation exposure. BMI only explained 4% of the variance in their population, which include a wide range of BMI.<SEP>As the majority of vitamin D3 is produced endogenously in the skin, it is logical to expect that a relationship would exist between sun exposure times, the %BSA exposed, and vitamin D status. There was no relationship between sun exposure times or %BSA, with serum 25(OH)D concentrations in our study. Previous research into determinants of vitamin D status in Australian adults found a significant association between vitamin D status and time spent outdoors (rs = 0.16, p < 0.0001), and vitamin D status and the percentage of clothing cover (rs = \\u22120.50, p < 0.001), in a population with a wide range of BMI, including individuals with obesity. There are a few possible reasons why no relationship was identified between these measures and vitamin D status in our study. This study reports behaviours of a specific group of morbidly obese individuals, and so represented only their sun exposure practices, whereas the AusD study included a wide range of BMI and a larger sample size. The data provided on sun exposure times and body surface area exposed was self-reported data, and may not have been accurate. There was a trend toward a positive correlation between change in ITA (degree of tan) and time in the sun. This would be expected, as longer sun exposure would generally lead to an increase in tan, providing some evidence that this measure of sun exposure is accurate. Vitamin D is stored in adipose tissue, and there is some evidence that muscle may store 25(OH)D, thereby reducing the amount of 25(OH)D available in the circulation for measurement. This may impact the ability to correlate vitamin D status with any measures of skin colour, or sun exposure, especially in those with high amounts of adipose tissue or large muscle mass, as seen in this group of morbidly obese individuals. <SEP>Three SNPs for the VDR and two for the DBP were analysed in this group. No significant differences in serum 25(OH)D concentrations were seen between genotypes of any gene. Although all homozygous rare genotypes had lower mean serum 25(OH)D concentrations than the common homozygous genotype. From a clinical perspective, VDR fok1 and DBP rs4588 both indicated insufficient vitamin D status for those with homozygous rare genotypes compared to homozygous common genotypes. Rare alleles for all four of the main variants for the VDR gene have been associated with lower serum 25(OH)D concentration and vitamin D deficiency in a range of populations. This suggests a possible method of personalised nutrition in this population, by considering specialised treatments for the patients with a genetic predisposition for lower serum 25(OH)D, based on the SNP genotype, particularly for fok1 and rs4588. This information could contribute to the overall assessment of vitamin D status, and identify those that may achieve more benefit from supplementation. <SEP>The VDR bsm1 rare allele has been associated with higher weight, BMI, and lower percentage of excess weight loss in bariatric patients. The taq1 rare allele has also been associated with higher BMI and weight. The fok1 rare allele has not been associated with weight and BMI previously. In this group of morbidly obese individuals, there was no statistically significant difference in weight, BMI or %EWL, between rare or common genotypes for bsm1, taq1, and fok1. Of clinical significance was the difference in mean serum 25(OH)D concentrations between common homozygous and rare homozygous individuals for fok1 and rs4588, where the common homozygous genotype was vitamin D sufficient, and the rare homozygous genotype was vitamin D deficient. <SEP>Both DBP SNP common alleles have been associated with higher BMI in females, but not males. We found no significant differences in weight or BMI between the genotypes for both DBP SNPs. The difference in results could be related to differences in sample populations, this study only included patients with BMI > 30, whereas the previous study included a wide range of BMI (16.93\\u201357.21 kg/m2). It is important to note that the SNP analysis from this study was under powered, and so results must be considered with caution. Particularly those that conflict with results from adequately powered studies. Post hoc power analysis using G*Power indicated that power ranged from 6 to 63% for these analyses, and required sample sizes from 69 to 957 for 80% power. <SEP>There are several limitations to this study. Serum 25(OH)D concentrations were measured using chemiluminescent competitive immunoassays, on two different platforms. The gold standard in measuring 25(OH)D concentration is liquid chromatography with tandem mass spectrometry. Issues with measuring 25(OH)D arise from the ability of these assays to release 25(OH)D from its binding protein or other carriers like albumin, the hydrophobic properties of 25(OH)D, and the differing antibody specificities to the metabolites. Although both pathology laboratories reported using quality assurance programs, both platforms have potential issues with under or over recovering metabolites, negative biases, and large deviations at lower concentrations of 25(OH)D (~50 nmo/L). This highlights an issue for clinicians in interpreting results for the 25(OH)D assay. Recruitment did not occur over a full year, this led to lower numbers of participants recruited post summer and autumn. This influences the results of differences in serum 25(OH)D between seasons, and potentially, the range of tanned skin colours. Using a small range of BMI has also minimised the effects of weight and BMI on serum 25(OH)D; use of a wider range, including normal, overweight, and obese individuals may have shown a more pronounced effect. A similar issue is found with the skin colour assessments, as individuals recruited only had natural skin colour from very light to intermediate. This limits the ability of results to be applied to those with darker natural skin colour. Our sample size was also under powered for SNP analysis, and so results should be considered with caution.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"text_length\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 10845,\n        \"min\": 5844,\n        \"max\": 104787,\n        \"num_unique_values\": 986,\n        \"samples\": [\n          27003,\n          27342,\n          21906\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"TOPIC_LABEL\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 24,\n        \"samples\": [\n          \"['OBWCCE', 'EATBTS', 'FOODIN', 'FOODAL']\",\n          \"['VITMIN', 'FOODAL']\",\n          \"['OBWCCE', 'EATBTS', 'FOODIN']\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 2
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "dataset_df['TOPIC_LABEL'].unique()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "lT7cgCqkzAlK",
        "outputId": "7d729827-73a2-4339-cb2f-8f00c8d9b3d4"
      },
      "execution_count": 9,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "array([\"['OBWCCE', 'EATBTS', 'FOODIN']\", \"['OBWCCE', 'EATBTS']\",\n",
              "       \"['OBWCCE']\", \"['OBWCCE', 'VITMIN', 'EATBTS', 'FOODIN']\",\n",
              "       \"['OBWCCE', 'VITMIN']\", \"['OBWCCE', 'FOODAL']\",\n",
              "       \"['OBWCCE', 'VITMIN', 'EATBTS']\", \"['OBWCCE', 'FOODIN', 'FOODAL']\",\n",
              "       \"['OBWCCE', 'EATBTS', 'FOODIN', 'FOODAL']\", \"['OBWCCE', 'FOODIN']\",\n",
              "       \"['OBWCCE', 'VITMIN', 'FOODIN']\", \"['VITMIN']\",\n",
              "       \"['VITMIN', 'FOODIN', 'FOODAL']\", \"['VITMIN', 'EATBTS']\",\n",
              "       \"['VITMIN', 'EATBTS', 'FOODAL']\", \"['VITMIN', 'FOODIN']\",\n",
              "       \"['VITMIN', 'FOODAL']\", \"['VITMIN', 'EATBTS', 'FOODIN']\",\n",
              "       \"['EATBTS']\", \"['EATBTS', 'FOODIN']\", \"['EATBTS', 'FOODAL']\",\n",
              "       \"['FOODIN']\", \"['FOODIN', 'FOODAL']\", \"['FOODAL']\"], dtype=object)"
            ]
          },
          "metadata": {},
          "execution_count": 9
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# df_FOODAL = dataset_df[dataset_df[\"TOPIC_LABEL\"] == [\"FOODAL\"]]\n",
        "# df_OBWCCE = dataset_df[dataset_df[\"TOPIC_LABEL\"] == [\"OBWCCE\"]]\n",
        "# df_EATBTS = dataset_df[dataset_df[\"TOPIC_LABEL\"] == [\"EATBTS\"]]\n",
        "\n",
        "df_FOODAL = dataset_df[dataset_df['TOPIC_LABEL'].apply(lambda x: 'FOODAL' in x)]\n",
        "df_OBWCCE = dataset_df[dataset_df['TOPIC_LABEL'].apply(lambda x: 'OBWCCE' in x)]\n",
        "df_EATBTS = dataset_df[dataset_df['TOPIC_LABEL'].apply(lambda x: 'EATBTS' in x)]\n",
        "df_VITMIN = dataset_df[dataset_df['TOPIC_LABEL'].apply(lambda x: 'VITMIN' in x)]\n",
        "df_FOODIN = dataset_df[dataset_df['TOPIC_LABEL'].apply(lambda x: 'FOODIN' in x)]"
      ],
      "metadata": {
        "id": "jf7YePG1xZF1"
      },
      "execution_count": 11,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "df_FOODAL.sample(5)['RESULTS'].to_json(\"/content/drive/MyDrive/workshop-ECAI/foodal.json\", orient=\"records\")\n",
        "df_OBWCCE.sample(5)['RESULTS'].to_json(\"/content/drive/MyDrive/workshop-ECAI/obwcce.json\", orient=\"records\")\n",
        "df_EATBTS.sample(5)['RESULTS'].to_json(\"/content/drive/MyDrive/workshop-ECAI/eatbts.json\", orient=\"records\")\n",
        "df_VITMIN.sample(5)['RESULTS'].to_json(\"/content/drive/MyDrive/workshop-ECAI/vitmin.json\", orient=\"records\")\n",
        "df_FOODIN.sample(5)['RESULTS'].to_json(\"/content/drive/MyDrive/workshop-ECAI/foodin.json\", orient=\"records\")"
      ],
      "metadata": {
        "id": "GB1f-m2LzsoI"
      },
      "execution_count": 12,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "# prompt: read a json file and count the element\n",
        "import json\n",
        "\n",
        "file_path = \"/content/drive/MyDrive/workshop-ECAI/foodal.json\"  # Replace with the actual path to your JSON file\n",
        "\n",
        "with open(file_path, 'r') as file:\n",
        "  data = json.load(file)\n",
        "\n",
        "# Assuming the JSON file contains a list or a dictionary\n",
        "if isinstance(data, list):\n",
        "  element_count = len(data)\n",
        "  print(f\"The JSON file contains {element_count} elements in the list.\")\n",
        "elif isinstance(data, dict):\n",
        "  element_count = len(data)\n",
        "  print(f\"The JSON file contains {element_count} key-value pairs in the dictionary.\")\n",
        "else:\n",
        "  print(\"The JSON file does not contain a list or a dictionary at the top level.\")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Hz3HMz7oz4qq",
        "outputId": "e27b5a45-8040-46b2-be44-bd2d52b2a96c"
      },
      "execution_count": 14,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "The JSON file contains 5 elements in the list.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "role_PROMPT = \"\"\"\n",
        "      You are a helpful assistant responsible for generating evaluation questions\n",
        "      from a given corpus using a structured procedure.\"\"\"\n",
        "goal_PROMPT = \"\"\"\n",
        "      Given a corpus description and the parameters K (number of users), N (number of tasks per user),\n",
        "      and M (number of high-level questions per (user, task) pair), your task is to generate high-level questions\n",
        "      that assess global understanding of the corpus.\n",
        "      \"\"\"\n",
        "generation_step_PROMPT = \"\"\"\n",
        "      Your generation should follow these steps:\n",
        "      1. Create K user personas based on the corpus, including their role/background and motivation for using the corpus.\n",
        "      2. For each user, define N realistic and relevant tasks they would perform using the corpus.\n",
        "      3. For each (user, task) pair, generate M high-level questions that:\n",
        "          - Require comprehensive understanding of the whole corpus.\n",
        "          - Do NOT depend on low-level facts or specific data points (e.g., names, dates, figures).\"\"\"\n",
        "\n",
        "output_PROMPT = \"\"\"\n",
        "      User 1: [Persona description]\n",
        "      Task 1:\n",
        "      Q1. ...\n",
        "      Q2. ...\n",
        "      ...\n",
        "      Task 2:\n",
        "      Q1. ...\n",
        "      ...\n",
        "      User 2: ...\n",
        "      ...\n",
        "      User K: ...\"\"\"\n",
        "\n",
        "output_format_PROMPT = \"\"\"\n",
        "      {{\n",
        "        \"corpus_description\": \"<your summary of the corpus>\",\n",
        "        \"generated_questions\": \"<your structured output as described above>\"\n",
        "      }}\"\"\""
      ],
      "metadata": {
        "id": "Pu8Y_gpD3PiI"
      },
      "execution_count": 44,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "def generate_questions(corpus_description, K, N, M):\n",
        "  PROMPT = f\"\"\"\n",
        "  ---Role--\n",
        "  {role_PROMPT}\n",
        "\n",
        "  ---Goal--\n",
        "  {goal_PROMPT}\n",
        "\n",
        "  {generation_step_PROMPT}\n",
        "\n",
        "  Your output should be structured as follows:\n",
        "  {output_PROMPT}\n",
        "\n",
        "  Format your response as a JSON object with the following structure:\n",
        "  {output_format_PROMPT}\n",
        "\n",
        "  ---Corpus Description--\n",
        "  {corpus_description}\n",
        "  ---Parameters--\n",
        "  Number of Users (K): {K}\n",
        "  Tasks per User (N): {N}\n",
        "  Questions per (User, Task) (M): {M}\n",
        "\n",
        "  Output:\n",
        "  \"\"\"\n",
        "  return PROMPT"
      ],
      "metadata": {
        "id": "GWOHvw762McB"
      },
      "execution_count": 43,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "import os\n",
        "from groq import Groq\n",
        "\n",
        "MODEL = \"meta-llama/llama-4-scout-17b-16e-instruct\"\n",
        "GROQ_API_KEY = \"***\"\n",
        "\n",
        "client = Groq(\n",
        "    api_key=GROQ_API_KEY,\n",
        ")"
      ],
      "metadata": {
        "id": "mrSlx6kx9r0k"
      },
      "execution_count": 20,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "with open('/content/drive/MyDrive/workshop-ECAI/groq_api.txt', 'w') as f:\n",
        "  f.write(GROQ_API_KEY)"
      ],
      "metadata": {
        "id": "ZX-4q8NbaIgO"
      },
      "execution_count": 82,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "foodal_docs = json.load(open(\"/content/drive/MyDrive/workshop-ECAI/foodal.json\"))\n",
        "obwcce_docs = json.load(open(\"/content/drive/MyDrive/workshop-ECAI/obwcce.json\"))\n",
        "eatbts_docs = json.load(open(\"/content/drive/MyDrive/workshop-ECAI/eatbts.json\"))\n",
        "vitmin_docs = json.load(open(\"/content/drive/MyDrive/workshop-ECAI/vitmin.json\"))\n",
        "foodin_docs = json.load(open(\"/content/drive/MyDrive/workshop-ECAI/foodin.json\"))"
      ],
      "metadata": {
        "id": "kuSS3c27Lx-P"
      },
      "execution_count": 23,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "corpus_description_PROMPT = f\"\"\"\n",
        "---Role--\n",
        "You are a helpful assistant tasked with generating a description of a corpus based on the results of 5 randomly selected academic papers.\n",
        "\n",
        "---Goal--\n",
        "Given a set of 5 academic papers, your task is to summarize the key findings from each paper and combine this information into a coherent corpus description. The description should highlight common themes, methodologies used, and major conclusions drawn across the papers.\n",
        "\n",
        "---Papers---\n",
        "1. {foodin_docs[0]}\n",
        "2. {foodin_docs[1]}\n",
        "3. {foodin_docs[2]}\n",
        "4. {foodin_docs[3]}\n",
        "5. {foodin_docs[4]}\n",
        "\n",
        "---Instructions---\n",
        "For each paper:\n",
        "- Briefly summarize the study's objective.\n",
        "- Describe the methodology used.\n",
        "- List the main findings or results.\n",
        "\n",
        "After analyzing all 5 papers:\n",
        "- Identify common themes across the papers.\n",
        "- Highlight any significant differences in methodologies or results.\n",
        "- Provide an overall description of the corpus that reflects the combined knowledge and insights from the papers.\n",
        "\n",
        "---Output Format---\n",
        "Return your response as a single text object with the following structure:\n",
        "\n",
        "<Detailed description of the corpus based on the 5 papers>\n",
        "\n",
        "---Output---\n",
        "\n",
        "\"\"\"\n",
        "\n",
        "\n",
        "chat_completion = client.chat.completions.create(\n",
        "    messages=[\n",
        "        {\n",
        "            \"role\": \"user\",\n",
        "            \"content\": corpus_description_PROMPT\n",
        "        }\n",
        "    ],\n",
        "    model=MODEL,\n",
        ")\n",
        "\n",
        "answer = chat_completion.choices[0].message.content\n",
        "\n",
        "print(answer)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "mcZLpQPYMAy_",
        "outputId": "9b998198-7c31-4e35-bfde-6885b91ebb12"
      },
      "execution_count": 40,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "The corpus consists of five academic papers that explore various genetic and molecular aspects of human and animal diseases. \n",
            "\n",
            "**Paper 1: Genetic analysis of telomere-related diseases**\n",
            "\n",
            "* Objective: To study the genetic basis of telomere-related diseases, such as dyskeratosis congenita (DC), aplastic anemia (AA), and pulmonary fibrosis (PF).\n",
            "* Methodology: The study used PCR amplification, DNA sequencing, and massive sequencing to analyze genes related to telomere biology in patients with DC/AA and PF.\n",
            "* Main findings: The study identified single nucleotide variants (SNVs) and indels in genes such as TERT, TERC, RTEL1, ACD, and CTC1 in patients with telomere-related diseases.\n",
            "\n",
            "**Paper 2: IL-6 and BMI in relation to a specific genotype**\n",
            "\n",
            "* Objective: To investigate the relationship between a specific genotype (Rs1800795) and IL-6 levels and BMI.\n",
            "* Methodology: The study used a mixed model to analyze the association between the genotype and IL-6 levels and BMI in a cohort of participants.\n",
            "* Main findings: The study found lower IL-6 levels in participants with the CC genotype compared to carriers of the G allele.\n",
            "\n",
            "**Paper 3: Exome sequencing and burden analysis in CAKUT**\n",
            "\n",
            "* Objective: To identify novel genetic causes of congenital anomalies of the kidney and urinary tract (CAKUT).\n",
            "* Methodology: The study used whole exome sequencing and burden analysis in patients with CAKUT.\n",
            "* Main findings: The study identified likely pathogenic variants in the SRGAP1 gene and found that SRGAP1 mutations cause augmented inhibition of RAC1.\n",
            "\n",
            "**Paper 4: Genetic associations with weight loss and regain**\n",
            "\n",
            "* Objective: To investigate the genetic associations with weight loss and regain in a cohort of participants.\n",
            "* Methodology: The study used genetic markers and statistical analysis to investigate the association between genetic variants and weight loss and regain.\n",
            "* Main findings: The study found that obesity risk alleles in FTO, SH2B1, and QPCTL/GIPR regions predicted baseline weight and that the FTO variant was associated with weight regain.\n",
            "\n",
            "**Paper 5: Genetic analysis of inherited gastritis syndrome**\n",
            "\n",
            "* Objective: To identify the genetic cause of inherited gastritis syndrome in Komondors.\n",
            "* Methodology: The study used whole genome sequencing, genetic exclusion analysis, and genotyping to investigate the genetic basis of the disease.\n",
            "* Main findings: The study identified a pathogenic variant in the CUBN gene that causes inherited gastritis syndrome.\n",
            "\n",
            "The common themes across the papers include:\n",
            "\n",
            "* The use of genetic analysis and molecular biology techniques to investigate the basis of human and animal diseases.\n",
            "* The identification of novel genetic variants and their association with specific diseases or traits.\n",
            "* The use of statistical analysis and bioinformatics tools to analyze large datasets.\n",
            "\n",
            "Significant differences in methodologies and results include:\n",
            "\n",
            "* The range of diseases and traits studied, from telomere-related diseases to CAKUT and inherited gastritis syndrome.\n",
            "* The use of different genetic analysis techniques, such as PCR, DNA sequencing, and whole exome sequencing.\n",
            "* The focus on different types of genetic variants, such as SNVs, indels, and copy number variants.\n",
            "\n",
            "Overall, the corpus provides insights into the genetic and molecular basis of various diseases and traits, and highlights the importance of genetic analysis in understanding human and animal health. The studies demonstrate the power of genetic analysis to identify novel disease-causing variants and to provide new insights into the biology of disease. The corpus also highlights the importance of using a range of methodologies, including bioinformatics and statistical analysis, to analyze large datasets and to identify significant associations between genetic variants and disease. \n",
            "\n",
            "The corpus description highlights common themes and insights that cut across several papers. Specifically, it focuses on major conclusions drawn across the papers. A common theme that runs through several papers is the application of genetic and molecular techniques to investigate and elucidate diseases’ causation. For instance, Papers 1, 3, and 5 employed genetic sequencing to unravel the genetic underpinnings of the studied conditions. Moreover, several studies highlight a complex interplay between genetic predispositions and phenotypic expressions, such as Papers 2 and 4, which found significant associations between certain genetic markers and variations in patient outcomes. Overall, the papers emphasize a detailed understanding of genetic diseases through comprehensive analysis.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# prompt: write the foodal_answer in a file text in my folder in drive\n",
        "\n",
        "corpus_description = answer.split(\"SNVs, indels, and copy number variants.\\n\\n\")[1]\n",
        "print(corpus_description)\n",
        "\n",
        "with open('/content/drive/MyDrive/workshop-ECAI/foodin_answer.txt', 'w') as f:\n",
        "  f.write(corpus_description)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "SPLKFSSNN7UB",
        "outputId": "d242af07-e6b7-4bd7-93c4-a35892f47cda"
      },
      "execution_count": 42,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Overall, the corpus provides insights into the genetic and molecular basis of various diseases and traits, and highlights the importance of genetic analysis in understanding human and animal health. The studies demonstrate the power of genetic analysis to identify novel disease-causing variants and to provide new insights into the biology of disease. The corpus also highlights the importance of using a range of methodologies, including bioinformatics and statistical analysis, to analyze large datasets and to identify significant associations between genetic variants and disease. \n",
            "\n",
            "The corpus description highlights common themes and insights that cut across several papers. Specifically, it focuses on major conclusions drawn across the papers. A common theme that runs through several papers is the application of genetic and molecular techniques to investigate and elucidate diseases’ causation. For instance, Papers 1, 3, and 5 employed genetic sequencing to unravel the genetic underpinnings of the studied conditions. Moreover, several studies highlight a complex interplay between genetic predispositions and phenotypic expressions, such as Papers 2 and 4, which found significant associations between certain genetic markers and variations in patient outcomes. Overall, the papers emphasize a detailed understanding of genetic diseases through comprehensive analysis.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Generiamo le domande"
      ],
      "metadata": {
        "id": "XbgL0ElyQyAP"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "K = 3\n",
        "N = 5\n",
        "M = 5\n",
        "with open('/content/drive/MyDrive/workshop-ECAI/foodin_answer.txt', 'r') as file:\n",
        "  corpus_description = file.read()\n",
        "\n",
        "prompt = generate_questions(corpus_description, K, N, M)\n",
        "\n",
        "chat_completion = client.chat.completions.create(\n",
        "    messages=[\n",
        "        {\n",
        "            \"role\": \"user\",\n",
        "            \"content\": prompt\n",
        "        }\n",
        "    ],\n",
        "    model=MODEL,\n",
        ")\n",
        "\n",
        "answer = chat_completion.choices[0].message.content\n",
        "\n",
        "print(answer)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "P2O_-ffVwK43",
        "outputId": "b631ebbc-d9e2-4ba7-cbb2-2c0f23f9d4bf"
      },
      "execution_count": 53,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Here is the output:\n",
            "\n",
            "```\n",
            "{\n",
            "  \"corpus_description\": \"The corpus provides insights into the genetic and molecular basis of various diseases and traits, highlighting the importance of genetic analysis in understanding human and animal health. It demonstrates the power of genetic analysis to identify novel disease-causing variants and provide new insights into disease biology, emphasizing the use of methodologies like bioinformatics and statistical analysis to analyze large datasets.\",\n",
            "  \n",
            "  \"generated_questions\": \"\n",
            "  \n",
            "  User1: A genetic researcher investigating the molecular basis of rare diseases.\n",
            "  Task1: Investigating the genetic basis of a newly discovered disease.\n",
            "  Q1. How do genetic sequencing techniques contribute to understanding disease causation?\n",
            "  Q2. What are the implications of identifying novel disease-causing variants for patient treatment?\n",
            "  Q3. How can comprehensive genetic analysis inform personalized medicine approaches?\n",
            "  Q4. What role do genetic predispositions play in phenotypic expressions of disease?\n",
            "  Q5. How can genetic analysis be used to develop targeted therapies?\n",
            "\n",
            "  Task2: Analyzing the genetic markers associated with patient outcomes.\n",
            "  Q1. How do genetic markers influence patient outcomes in complex diseases?\n",
            "  Q2. What are the challenges in identifying significant associations between genetic variants and disease?\n",
            "  Q3. How can bioinformatics tools be used to analyze large genetic datasets?\n",
            "  Q4. What are the implications of genetic analysis for disease diagnosis and treatment?\n",
            "  Q5. How can genetic research inform the development of novel therapeutics?\n",
            "\n",
            "  Task3: Exploring the genetic basis of disease using statistical analysis.\n",
            "  Q1. How can statistical analysis be used to identify genetic associations with disease?\n",
            "  Q2. What are the limitations of using statistical analysis in genetic research?\n",
            "  Q3. How can genetic analysis be integrated with other 'omics' fields to understand disease?\n",
            "  Q4. What are the implications of genetic research for understanding disease mechanisms?\n",
            "  Q5. How can genetic analysis inform the development of preventive measures?\n",
            "\n",
            "  Task4: Investigating the role of genetic predispositions in disease.\n",
            "  Q1. How do genetic predispositions influence disease susceptibility?\n",
            "  Q2. What are the challenges in identifying genetic predispositions associated with disease?\n",
            "  Q3. How can genetic analysis be used to develop predictive models of disease risk?\n",
            "  Q4. What are the implications of genetic research for disease prevention and treatment?\n",
            "  Q5. How can genetic analysis inform the development of targeted interventions?\n",
            "\n",
            "  Task5: Developing genetic testing strategies for disease diagnosis.\n",
            "  Q1. How can genetic testing be used to diagnose genetic diseases?\n",
            "  Q2. What are the challenges in developing genetic testing strategies?\n",
            "  Q3. How can genetic analysis be used to identify genetic variants associated with disease?\n",
            "  Q4. What are the implications of genetic testing for patient outcomes?\n",
            "  Q5. How can genetic testing inform personalized medicine approaches?\n",
            "\n",
            "  User2: A clinician seeking to apply genetic insights to improve patient care.\n",
            "  Task1: Applying genetic analysis to inform treatment decisions.\n",
            "  Q1. How can genetic analysis inform treatment decisions for patients with genetic diseases?\n",
            "  Q2. What are the challenges in integrating genetic analysis into clinical practice?\n",
            "  Q3. How can genetic research inform the development of targeted therapies?\n",
            "  Q4. What are the implications of genetic analysis for patient outcomes?\n",
            "  Q5. How can genetic testing be used to diagnose genetic diseases?\n",
            "\n",
            "  Task2: Investigating the genetic basis of complex diseases.\n",
            "  Q1. How do genetic variants contribute to complex diseases?\n",
            "  Q2. What are the challenges in identifying genetic associations with complex diseases?\n",
            "  Q3. How can genetic analysis be used to develop predictive models of disease risk?\n",
            "  Q4. What are the implications of genetic research for disease prevention and treatment?\n",
            "  Q5. How can genetic analysis inform the development of targeted interventions?\n",
            "\n",
            "  Task3: Exploring the role of genetic predispositions in disease.\n",
            "  Q1. How do genetic predispositions influence disease susceptibility?\n",
            "  Q2. What are the challenges in identifying genetic predispositions associated with disease?\n",
            "  Q3. How can genetic analysis be used to develop predictive models of disease risk?\n",
            "  Q4. What are the implications of genetic research for disease prevention and treatment?\n",
            "  Q5. How can genetic analysis inform the development of targeted interventions?\n",
            "\n",
            "  Task4: Developing genetic testing strategies for disease diagnosis.\n",
            "  Q1. How can genetic testing be used to diagnose genetic diseases?\n",
            "  Q2. What are the challenges in developing genetic testing strategies?\n",
            "  Q3. How can genetic analysis be used to identify genetic variants associated with disease?\n",
            "  Q4. What are the implications of genetic testing for patient outcomes?\n",
            "  Q5. How can genetic testing inform personalized medicine approaches?\n",
            "\n",
            "  Task5: Investigating the genetic basis of rare diseases.\n",
            "  Q1. How do genetic variants contribute to rare diseases?\n",
            "  Q2. What are the challenges in identifying genetic associations with rare diseases?\n",
            "  Q3. How can genetic analysis be used to develop predictive models of disease risk?\n",
            "  Q4. What are the implications of genetic research for disease prevention and treatment?\n",
            "  Q5. How can genetic analysis inform the development of targeted therapies?\n",
            "\n",
            "  User3: A bioinformatics specialist analyzing large genetic datasets.\n",
            "  Task1: Analyzing large genetic datasets to identify disease-associated variants.\n",
            "  Q1. How can bioinformatics tools be used to analyze large genetic datasets?\n",
            "  Q2. What are the challenges in identifying significant associations between genetic variants and disease?\n",
            "  Q3. How can genetic analysis be integrated with other 'omics' fields to understand disease?\n",
            "  Q4. What are the implications of genetic research for understanding disease mechanisms?\n",
            "  Q5. How can genetic analysis inform the development of novel therapeutics?\n",
            "\n",
            "  Task2: Investigating the genetic basis of disease using machine learning techniques.\n",
            "  Q1. How can machine learning techniques be used to identify genetic associations with disease?\n",
            "  Q2. What are the challenges in developing machine learning models for genetic analysis?\n",
            "  Q3. How can genetic analysis be used to develop predictive models of disease risk?\n",
            "  Q4. What are the implications of genetic research for disease prevention and treatment?\n",
            "  Q5. How can genetic analysis inform the development of targeted interventions?\n",
            "\n",
            "  Task3: Exploring the genetic basis of complex diseases.\n",
            "  Q1. How do genetic variants contribute to complex diseases?\n",
            "  Q2. What are the challenges in identifying genetic associations with complex diseases?\n",
            "  Q3. How can genetic analysis be used to develop predictive models of disease risk?\n",
            "  Q4. What are the implications of genetic research for disease prevention and treatment?\n",
            "  Q5. How can genetic analysis inform the development of targeted therapies?\n",
            "\n",
            "  Task4: Developing computational tools for genetic analysis.\n",
            "  Q1. How can computational tools be used to analyze large genetic datasets?\n",
            "  Q2. What are the challenges in developing computational tools for genetic analysis?\n",
            "  Q3. How can genetic analysis be integrated with other 'omics' fields to understand disease?\n",
            "  Q4. What are the implications of genetic research for understanding disease mechanisms?\n",
            "  Q5. How can genetic analysis inform the development of novel therapeutics?\n",
            "\n",
            "  Task5: Investigating the genetic basis of rare diseases.\n",
            "  Q1. How do genetic variants contribute to rare diseases?\n",
            "  Q2. What are the challenges in identifying genetic associations with rare diseases?\n",
            "  Q3. How can genetic analysis be used to develop predictive models of disease risk?\n",
            "  Q4. What are the implications of genetic research for disease prevention and treatment?\n",
            "  Q5. How can genetic analysis inform the development of targeted therapies?\n",
            "\"\n",
            "}\n",
            "```\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "with open('/content/drive/MyDrive/workshop-ECAI/foodin_generated_question.txt', 'w') as f:\n",
        "  f.write(answer)"
      ],
      "metadata": {
        "id": "aHJQboBD1UiL"
      },
      "execution_count": 54,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [],
      "metadata": {
        "id": "fK5-N_u6SuSk"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Ristrutturazione delle domande in un DataFrame\n"
      ],
      "metadata": {
        "id": "82eZDMxeSvn3"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "with open('/content/drive/MyDrive/workshop-ECAI/questions_files/foodal_generated_question.txt', 'r') as file:\n",
        "  foodal_questions = file.read()\n",
        "\n",
        "with open('/content/drive/MyDrive/workshop-ECAI/questions_files/obwcce_generated_question.txt', 'r') as file:\n",
        "  obwcce_questions = file.read()\n",
        "\n",
        "with open('/content/drive/MyDrive/workshop-ECAI/questions_files/eatbts_generated_question.txt', 'r') as file:\n",
        "  eatbts_questions = file.read()\n",
        "\n",
        "with open('/content/drive/MyDrive/workshop-ECAI/questions_files/vitmin_generated_question.txt', 'r') as file:\n",
        "  vitmin_questions = file.read()\n",
        "\n",
        "with open('/content/drive/MyDrive/workshop-ECAI/questions_files/foodin_generated_question.txt', 'r') as file:\n",
        "  foodin_questions = file.read()"
      ],
      "metadata": {
        "id": "ykxVRxq9S1Nc"
      },
      "execution_count": 65,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "print(foodal_questions)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "collapsed": true,
        "id": "toNX4nt_S9FK",
        "outputId": "89223218-1ff5-44a5-d5a1-4112724ef458"
      },
      "execution_count": 58,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Here is the output:\n",
            "\n",
            "```\n",
            "{\n",
            "  \"corpus_description\": \"The corpus consists of five papers that explore various aspects of medical research, including vaccine efficacy, genetic associations with disease susceptibility, and toxicological responses. The studies employed different designs and involved various populations, but shared common themes, including genetic associations, immunological responses, and interindividual variability. The corpus highlights the importance of understanding genetic and immunological factors in disease susceptibility and response to treatment. The findings from these studies have implications for the development of personalized medicine approaches and the identification of novel therapeutic targets. Overall, the corpus provides a comprehensive overview of the complex interactions between genetic and environmental factors that influence human health and disease.\",\n",
            "  \"generated_questions\": \"\n",
            "  User1: A clinical researcher interested in vaccine development and personalized medicine.\n",
            "  Task1: Designing a new vaccine trial\n",
            "  Q1. What are the key genetic and immunological factors that influence vaccine efficacy across different populations?\n",
            "  Q2. How do interindividual variability and environmental factors impact the response to vaccination?\n",
            "  Q3. What are the implications of the findings from these studies for the development of personalized vaccine approaches?\n",
            "  Q4. How can understanding genetic associations with disease susceptibility inform the design of vaccine trials?\n",
            "  Q5. What are the potential therapeutic targets for improving vaccine efficacy based on the insights from this corpus?\n",
            "\n",
            "  Task2: Analyzing genetic data from a new study\n",
            "  Q1. How do the genetic associations with disease susceptibility reported in the corpus inform the interpretation of genetic data from my study?\n",
            "  Q2. What are the limitations of using genetic data to predict disease susceptibility, and how can I account for these limitations?\n",
            "  Q3. How can I integrate the findings from this corpus into my analysis to better understand the role of genetic factors in disease susceptibility?\n",
            "  Q4. What are the potential applications of understanding genetic associations with disease susceptibility in clinical practice?\n",
            "  Q5. How do the studies in the corpus account for interindividual variability in genetic data analysis?\n",
            "\n",
            "  Task3: Developing a new therapeutic approach\n",
            "  Q1. What are the key insights from the corpus that can inform the development of novel therapeutic targets?\n",
            "  Q2. How do the findings from these studies on immunological responses impact the design of therapeutic approaches?\n",
            "  Q3. What are the potential benefits and challenges of targeting genetic and immunological factors in disease treatment?\n",
            "  Q4. How can understanding interindividual variability in response to treatment inform the development of personalized medicine approaches?\n",
            "  Q5. What are the implications of the corpus findings for the development of combination therapies?\n",
            "\n",
            "  Task4: Communicating with patients about genetic risks\n",
            "  Q1. How can I effectively communicate genetic risk information to patients, taking into account interindividual variability and environmental factors?\n",
            "  Q2. What are the key findings from the corpus that I should share with patients to help them understand their genetic risks?\n",
            "  Q3. How can I use the insights from this corpus to counsel patients about personalized medicine approaches?\n",
            "  Q4. What are the potential psychological and social implications of sharing genetic risk information with patients?\n",
            "  Q5. How can I balance the benefits and limitations of genetic testing in clinical practice?\n",
            "\n",
            "  Task5: Identifying research gaps\n",
            "  Q1. What are the key research gaps in our understanding of genetic and immunological factors in disease susceptibility and response to treatment?\n",
            "  Q2. How can the findings from this corpus inform the development of future research studies?\n",
            "  Q3. What are the potential areas of investigation that could lead to novel therapeutic targets?\n",
            "  Q4. How can we better integrate genetic and environmental factors in future research studies?\n",
            "  Q5. What are the implications of the corpus findings for the development of new research methodologies?\n",
            "\n",
            "  User2: A genetic epidemiologist interested in understanding the complex interactions between genetic and environmental factors.\n",
            "  Task1: Investigating genetic associations with disease susceptibility\n",
            "  Q1. What are the key genetic associations with disease susceptibility reported in the corpus, and how do they relate to my own research?\n",
            "  Q2. How can I use the findings from these studies to inform my analysis of genetic data?\n",
            "  Q3. What are the implications of interindividual variability in genetic associations with disease susceptibility for my research?\n",
            "  Q4. How can I integrate environmental factors into my analysis of genetic associations with disease susceptibility?\n",
            "  Q5. What are the potential applications of understanding genetic associations with disease susceptibility in public health?\n",
            "\n",
            "  Task2: Developing a new genetic testing approach\n",
            "  Q1. How can the insights from the corpus inform the development of genetic testing approaches for disease susceptibility?\n",
            "  Q2. What are the potential benefits and limitations of using genetic testing to predict disease susceptibility?\n",
            "  Q3. How can I use the findings from these studies to inform the interpretation of genetic test results?\n",
            "  Q4. What are the implications of the corpus findings for the development of genetic testing guidelines?\n",
            "  Q5. How can I balance the benefits and limitations of genetic testing in clinical practice?\n",
            "\n",
            "  Task3: Analyzing environmental influences on genetic associations\n",
            "  Q1. How do environmental factors influence genetic associations with disease susceptibility, according to the corpus?\n",
            "  Q2. What are the key insights from the corpus that can inform my analysis of environmental influences on genetic associations?\n",
            "  Q3. How can I integrate the findings from these studies into my analysis to better understand the role of environmental factors?\n",
            "  Q4. What are the potential applications of understanding environmental influences on genetic associations in public health?\n",
            "  Q5. How do the studies in the corpus account for interindividual variability in environmental influences on genetic associations?\n",
            "\n",
            "  Task4: Investigating interindividual variability\n",
            "  Q1. What are the key insights from the corpus about interindividual variability in genetic associations with disease susceptibility?\n",
            "  Q2. How can I use the findings from these studies to inform my analysis of interindividual variability?\n",
            "  Q3. What are the implications of interindividual variability in genetic associations with disease susceptibility for personalized medicine?\n",
            "  Q4. How can I integrate the findings from these studies into my analysis to better understand the role of interindividual variability?\n",
            "  Q5. What are the potential applications of understanding interindividual variability in genetic associations with disease susceptibility?\n",
            "\n",
            "  Task5: Communicating with policymakers about genetic research\n",
            "  Q1. How can I effectively communicate the findings from genetic research to policymakers, taking into account the complexities of genetic associations with disease susceptibility?\n",
            "  Q2. What are the key insights from the corpus that I should share with policymakers to inform public health policy?\n",
            "  Q3. How can I use the insights from this corpus to inform policy decisions about genetic testing and personalized medicine?\n",
            "  Q4. What are the potential implications of the corpus findings for healthcare policy and resource allocation?\n",
            "  Q5. How can I balance the benefits and limitations of genetic research in informing public health policy?\n",
            "\n",
            "  User3: A toxicologist interested in understanding the impact of environmental toxins on human health.\n",
            "  Task1: Investigating toxicological responses\n",
            "  Q1. What are the key insights from the corpus about toxicological responses and their relationship to genetic and immunological factors?\n",
            "  Q2. How can I use the findings from these studies to inform my analysis of toxicological responses?\n",
            "  Q3. What are the implications of the corpus findings for understanding the impact of environmental toxins on human health?\n",
            "  Q4. How can I integrate the findings from these studies into my analysis to better understand the role of genetic and immunological factors in toxicological responses?\n",
            "  Q5. What are the potential applications of understanding toxicological responses in risk assessment and regulatory policy?\n",
            "\n",
            "  Task2: Developing a new toxicological testing approach\n",
            "  Q1. How can the insights from the corpus inform the development of new toxicological testing approaches?\n",
            "  Q2. What are the potential benefits and limitations of using genetic and immunological factors to predict toxicological responses?\n",
            "  Q3. How can I use the findings from these studies to inform the interpretation of toxicological test results?\n",
            "  Q4. What are the implications of the corpus findings for the development of toxicological testing guidelines?\n",
            "  Q5. How can I balance the benefits and limitations of toxicological testing in regulatory policy?\n",
            "\n",
            "  Task3: Analyzing gene-environment interactions\n",
            "  Q1. How do genetic and environmental factors interact to influence toxicological responses, according to the corpus?\n",
            "  Q2. What are the key insights from the corpus that can inform my analysis of gene-environment interactions?\n",
            "  Q3. How can I integrate the findings from these studies into my analysis to better understand the role of gene-environment interactions?\n",
            "  Q4. What are the potential applications of understanding gene-environment interactions in toxicology?\n",
            "  Q5. How do the studies in the corpus account for interindividual variability in gene-environment interactions?\n",
            "\n",
            "  Task4: Investigating interindividual variability in toxicological responses\n",
            "  Q1. What are the key insights from the corpus about interindividual variability in toxicological responses?\n",
            "  Q2. How can I use the findings from these studies to inform my analysis of interindividual variability?\n",
            "  Q3. What are the implications of interindividual variability in toxicological responses for risk assessment and regulatory policy?\n",
            "  Q4. How can I integrate the findings from these studies into my analysis to better understand the role of interindividual variability?\n",
            "  Q5. What are the potential applications of understanding interindividual variability in toxicological responses?\n",
            "\n",
            "  Task5: Communicating with regulatory agencies about toxicological research\n",
            "  Q1. How can I effectively communicate the findings from toxicological research to regulatory agencies, taking into account the complexities of gene-environment interactions?\n",
            "  Q2. What are the key insights from the corpus that I should share with regulatory agencies to inform policy decisions?\n",
            "  Q3. How can I use the insights from this corpus to inform policy decisions about toxicological testing and risk assessment?\n",
            "  Q4. What are the potential implications of the corpus findings for regulatory policy and risk management?\n",
            "  Q5. How can I balance the benefits and limitations of toxicological research in informing regulatory policy?\"\n",
            "}\n",
            "```\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import pandas as pd\n",
        "import re\n",
        "\n",
        "# Step 1: Read file as plain text\n",
        "#-- Fatto nella cella sopra per tutti\n",
        "\n",
        "# Step 2: Manually extract the 'generated_questions' content\n",
        "match = re.search(r'\"generated_questions\"\\s*:\\s*\"(?P<content>.*)\"\\s*}', foodin_questions, re.DOTALL)\n",
        "if not match:\n",
        "    raise ValueError(\"Could not find 'generated_questions' content in the file.\")\n",
        "\n",
        "content = match.group(\"content\")\n",
        "\n",
        "# Step 3: Unescape escaped quotes and normalize the content\n",
        "content = content.replace('\\\\\"', '\"')\n",
        "\n",
        "# Step 4: Parse lines\n",
        "lines = content.splitlines()\n",
        "\n",
        "category = \"foodin\".upper()  # or derive from file name\n",
        "rows = []\n",
        "current_user = None\n",
        "current_task = None\n",
        "questions = []\n",
        "\n",
        "for line in lines:\n",
        "    line = line.strip()\n",
        "    if not line:\n",
        "        continue\n",
        "    if line.startswith(\"User\"):\n",
        "        current_user = line.split(\":\", 1)[1].strip()\n",
        "    elif line.startswith(\"Task\"):\n",
        "        if current_task and questions:\n",
        "            rows.append({\n",
        "                \"Category\": category,\n",
        "                \"USER Description\": current_user,\n",
        "                \"TASK Description\": current_task,\n",
        "                \"Questions\": questions\n",
        "            })\n",
        "            questions = []\n",
        "        current_task = line.split(\":\", 1)[1].strip()\n",
        "    elif re.match(r\"Q\\d+\\.\", line):\n",
        "        questions.append(line)\n",
        "\n",
        "# Append the last block\n",
        "if current_task and questions:\n",
        "    rows.append({\n",
        "        \"Category\": category,\n",
        "        \"USER Description\": current_user,\n",
        "        \"TASK Description\": current_task,\n",
        "        \"Questions\": questions\n",
        "    })\n",
        "\n",
        "# Step 5: Create DataFrame\n",
        "df_foodin = pd.DataFrame(rows)"
      ],
      "metadata": {
        "id": "G0eSkhwcTX0c"
      },
      "execution_count": 77,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "df_foodal\n",
        "df_obwcce\n",
        "df_eatbts\n",
        "df_vitmin\n",
        "df_foodin"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 519
        },
        "id": "bDD92uF_WmmB",
        "outputId": "a41b1874-02fd-46fa-f306-b7a29c263de1"
      },
      "execution_count": 78,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "   Category                                   USER Description  \\\n",
              "0    FOODIN  A genetic researcher investigating the molecul...   \n",
              "1    FOODIN  A genetic researcher investigating the molecul...   \n",
              "2    FOODIN  A genetic researcher investigating the molecul...   \n",
              "3    FOODIN  A genetic researcher investigating the molecul...   \n",
              "4    FOODIN  A clinician seeking to apply genetic insights ...   \n",
              "5    FOODIN  A clinician seeking to apply genetic insights ...   \n",
              "6    FOODIN  A clinician seeking to apply genetic insights ...   \n",
              "7    FOODIN  A clinician seeking to apply genetic insights ...   \n",
              "8    FOODIN  A clinician seeking to apply genetic insights ...   \n",
              "9    FOODIN  A bioinformatics specialist analyzing large ge...   \n",
              "10   FOODIN  A bioinformatics specialist analyzing large ge...   \n",
              "11   FOODIN  A bioinformatics specialist analyzing large ge...   \n",
              "12   FOODIN  A bioinformatics specialist analyzing large ge...   \n",
              "13   FOODIN  A bioinformatics specialist analyzing large ge...   \n",
              "14   FOODIN  A bioinformatics specialist analyzing large ge...   \n",
              "\n",
              "                                     TASK Description  \\\n",
              "0   Investigating the genetic basis of a newly dis...   \n",
              "1   Analyzing the genetic markers associated with ...   \n",
              "2   Exploring the genetic basis of disease using s...   \n",
              "3   Investigating the role of genetic predispositi...   \n",
              "4   Developing genetic testing strategies for dise...   \n",
              "5   Applying genetic analysis to inform treatment ...   \n",
              "6   Investigating the genetic basis of complex dis...   \n",
              "7   Exploring the role of genetic predispositions ...   \n",
              "8   Developing genetic testing strategies for dise...   \n",
              "9   Investigating the genetic basis of rare diseases.   \n",
              "10  Analyzing large genetic datasets to identify d...   \n",
              "11  Investigating the genetic basis of disease usi...   \n",
              "12   Exploring the genetic basis of complex diseases.   \n",
              "13  Developing computational tools for genetic ana...   \n",
              "14  Investigating the genetic basis of rare diseases.   \n",
              "\n",
              "                                            Questions  \n",
              "0   [Q1. How do genetic sequencing techniques cont...  \n",
              "1   [Q1. How do genetic markers influence patient ...  \n",
              "2   [Q1. How can statistical analysis be used to i...  \n",
              "3   [Q1. How do genetic predispositions influence ...  \n",
              "4   [Q1. How can genetic testing be used to diagno...  \n",
              "5   [Q1. How can genetic analysis inform treatment...  \n",
              "6   [Q1. How do genetic variants contribute to com...  \n",
              "7   [Q1. How do genetic predispositions influence ...  \n",
              "8   [Q1. How can genetic testing be used to diagno...  \n",
              "9   [Q1. How do genetic variants contribute to rar...  \n",
              "10  [Q1. How can bioinformatics tools be used to a...  \n",
              "11  [Q1. How can machine learning techniques be us...  \n",
              "12  [Q1. How do genetic variants contribute to com...  \n",
              "13  [Q1. How can computational tools be used to an...  \n",
              "14  [Q1. How do genetic variants contribute to rar...  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-1f1f8569-bc65-4705-995c-83b0b573878e\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Category</th>\n",
              "      <th>USER Description</th>\n",
              "      <th>TASK Description</th>\n",
              "      <th>Questions</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A genetic researcher investigating the molecul...</td>\n",
              "      <td>Investigating the genetic basis of a newly dis...</td>\n",
              "      <td>[Q1. How do genetic sequencing techniques cont...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A genetic researcher investigating the molecul...</td>\n",
              "      <td>Analyzing the genetic markers associated with ...</td>\n",
              "      <td>[Q1. How do genetic markers influence patient ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A genetic researcher investigating the molecul...</td>\n",
              "      <td>Exploring the genetic basis of disease using s...</td>\n",
              "      <td>[Q1. How can statistical analysis be used to i...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A genetic researcher investigating the molecul...</td>\n",
              "      <td>Investigating the role of genetic predispositi...</td>\n",
              "      <td>[Q1. How do genetic predispositions influence ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A clinician seeking to apply genetic insights ...</td>\n",
              "      <td>Developing genetic testing strategies for dise...</td>\n",
              "      <td>[Q1. How can genetic testing be used to diagno...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A clinician seeking to apply genetic insights ...</td>\n",
              "      <td>Applying genetic analysis to inform treatment ...</td>\n",
              "      <td>[Q1. How can genetic analysis inform treatment...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A clinician seeking to apply genetic insights ...</td>\n",
              "      <td>Investigating the genetic basis of complex dis...</td>\n",
              "      <td>[Q1. How do genetic variants contribute to com...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A clinician seeking to apply genetic insights ...</td>\n",
              "      <td>Exploring the role of genetic predispositions ...</td>\n",
              "      <td>[Q1. How do genetic predispositions influence ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A clinician seeking to apply genetic insights ...</td>\n",
              "      <td>Developing genetic testing strategies for dise...</td>\n",
              "      <td>[Q1. How can genetic testing be used to diagno...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A bioinformatics specialist analyzing large ge...</td>\n",
              "      <td>Investigating the genetic basis of rare diseases.</td>\n",
              "      <td>[Q1. How do genetic variants contribute to rar...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>10</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A bioinformatics specialist analyzing large ge...</td>\n",
              "      <td>Analyzing large genetic datasets to identify d...</td>\n",
              "      <td>[Q1. How can bioinformatics tools be used to a...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>11</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A bioinformatics specialist analyzing large ge...</td>\n",
              "      <td>Investigating the genetic basis of disease usi...</td>\n",
              "      <td>[Q1. How can machine learning techniques be us...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>12</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A bioinformatics specialist analyzing large ge...</td>\n",
              "      <td>Exploring the genetic basis of complex diseases.</td>\n",
              "      <td>[Q1. How do genetic variants contribute to com...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>13</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A bioinformatics specialist analyzing large ge...</td>\n",
              "      <td>Developing computational tools for genetic ana...</td>\n",
              "      <td>[Q1. How can computational tools be used to an...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>14</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A bioinformatics specialist analyzing large ge...</td>\n",
              "      <td>Investigating the genetic basis of rare diseases.</td>\n",
              "      <td>[Q1. How do genetic variants contribute to rar...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-1f1f8569-bc65-4705-995c-83b0b573878e')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-1f1f8569-bc65-4705-995c-83b0b573878e button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-1f1f8569-bc65-4705-995c-83b0b573878e');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "    <div id=\"df-4a75590d-35cf-4f1b-99e8-37b6c34f5f6f\">\n",
              "      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-4a75590d-35cf-4f1b-99e8-37b6c34f5f6f')\"\n",
              "                title=\"Suggest charts\"\n",
              "                style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "      </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "      <script>\n",
              "        async function quickchart(key) {\n",
              "          const quickchartButtonEl =\n",
              "            document.querySelector('#' + key + ' button');\n",
              "          quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "          quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "          try {\n",
              "            const charts = await google.colab.kernel.invokeFunction(\n",
              "                'suggestCharts', [key], {});\n",
              "          } catch (error) {\n",
              "            console.error('Error during call to suggestCharts:', error);\n",
              "          }\n",
              "          quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "          quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "        }\n",
              "        (() => {\n",
              "          let quickchartButtonEl =\n",
              "            document.querySelector('#df-4a75590d-35cf-4f1b-99e8-37b6c34f5f6f button');\n",
              "          quickchartButtonEl.style.display =\n",
              "            google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "        })();\n",
              "      </script>\n",
              "    </div>\n",
              "\n",
              "  <div id=\"id_612ca49f-8ee7-4642-b383-c26b87baf257\">\n",
              "    <style>\n",
              "      .colab-df-generate {\n",
              "        background-color: #E8F0FE;\n",
              "        border: none;\n",
              "        border-radius: 50%;\n",
              "        cursor: pointer;\n",
              "        display: none;\n",
              "        fill: #1967D2;\n",
              "        height: 32px;\n",
              "        padding: 0 0 0 0;\n",
              "        width: 32px;\n",
              "      }\n",
              "\n",
              "      .colab-df-generate:hover {\n",
              "        background-color: #E2EBFA;\n",
              "        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "        fill: #174EA6;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate {\n",
              "        background-color: #3B4455;\n",
              "        fill: #D2E3FC;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate:hover {\n",
              "        background-color: #434B5C;\n",
              "        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "        fill: #FFFFFF;\n",
              "      }\n",
              "    </style>\n",
              "    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('df_foodin')\"\n",
              "            title=\"Generate code using this dataframe.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "    <script>\n",
              "      (() => {\n",
              "      const buttonEl =\n",
              "        document.querySelector('#id_612ca49f-8ee7-4642-b383-c26b87baf257 button.colab-df-generate');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      buttonEl.onclick = () => {\n",
              "        google.colab.notebook.generateWithVariable('df_foodin');\n",
              "      }\n",
              "      })();\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "df_foodin",
              "summary": "{\n  \"name\": \"df_foodin\",\n  \"rows\": 15,\n  \"fields\": [\n    {\n      \"column\": \"Category\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 1,\n        \"samples\": [\n          \"FOODIN\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"USER Description\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 3,\n        \"samples\": [\n          \"A genetic researcher investigating the molecular basis of rare diseases.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"TASK Description\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 13,\n        \"samples\": [\n          \"Exploring the genetic basis of complex diseases.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"Questions\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 78
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# prompt: append all these dataframe together: df_foodal\n",
        "# df_obwcce\n",
        "# df_eatbts\n",
        "# df_vitmin\n",
        "# df_foodin\n",
        "\n",
        "df_combined = pd.concat([df_foodal, df_obwcce, df_eatbts, df_vitmin, df_foodin], ignore_index=True)"
      ],
      "metadata": {
        "id": "RW-UDhrIWnfD"
      },
      "execution_count": 79,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "df_combined.to_pickle(\"/content/drive/MyDrive/workshop-ECAI/questions_files/questions_dataframe.pkl\")"
      ],
      "metadata": {
        "id": "n3RjtCnBXdco"
      },
      "execution_count": 81,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "gDz9WfO9Xo3Q"
      },
      "execution_count": null,
      "outputs": []
    }
  ]
}