{
  "cells": [
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "id": "DTtgAV8RtWiO"
      },
      "outputs": [],
      "source": [
        "import os\n",
        "import json\n",
        "import logging\n",
        "import numpy as np"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "id": "Qns5kcqhtSK4"
      },
      "outputs": [],
      "source": [
        "def load_json_from_lib(nome_file, local = False):\n",
        "    # Usa __file__ per ottenere il percorso della directory del file corrente\n",
        "    if not local:\n",
        "        file_path = os.path.join(os.path.dirname(__file__), nome_file)\n",
        "    else:\n",
        "        file_path = nome_file\n",
        "    with open(file_path, 'r') as file:\n",
        "        return json.load(file)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "metadata": {},
      "outputs": [
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:datasets:PyTorch version 2.7.0 available.\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Using default model: meta-llama/llama-4-maverick-17b-128e-instruct\n",
            "/root/projects/nano-graphrag/biomedical/api_keys.json\n",
            "Ollama base url http://0.0.0.0:11434\n"
          ]
        }
      ],
      "source": [
        "import sys\n",
        "import os\n",
        "\n",
        "# Append the parent directory to sys.path\n",
        "sys.path.append(os.path.abspath('..'))\n",
        "\n",
        "# Now you can import the module\n",
        "from llm_utils import api_keys\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 4,
      "metadata": {
        "id": "bFiydk4rsvf2"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\n",
            "\n",
            "<<<<<<<<<<<<< No-RAG Reply >>>>>>>>>>>>>>>\n",
            "GROQ_API_KEY: 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\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "The capital of France is Paris.\n",
            "<<< ----------------- >>>\n"
          ]
        }
      ],
      "source": [
        "print(\"\\n\\n<<<<<<<<<<<<< No-RAG Reply >>>>>>>>>>>>>>>\")\n",
        "GROQ_API_KEY = api_keys[\"groq\"]\n",
        "print(\"GROQ_API_KEY:\", GROQ_API_KEY)\n",
        "MODEL = \"meta-llama/llama-4-scout-17b-16e-instruct\"\n",
        "# MODEL = \"llama-3.1-8b-instant\"\n",
        "\n",
        "# Test Native LLM response\n",
        "from groq import Groq\n",
        "\n",
        "client = Groq(api_key=GROQ_API_KEY)\n",
        "question = \"What is the capital of France?\"\n",
        "\n",
        "response = client.chat.completions.create(\n",
        "    model=MODEL,\n",
        "    messages=[\n",
        "        {\"role\": \"system\", \"content\": \"You are a helpful assistant\"},\n",
        "        {\"role\": \"user\", \"content\": question},\n",
        "    ],\n",
        "    stream=False\n",
        ")\n",
        "\n",
        "print(response.choices[0].message.content)\n",
        "print(\"<<< ----------------- >>>\")"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "NYhnzH9MwJIQ"
      },
      "source": [
        "# Data Preparation"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "jXwQjuGgwMt_",
        "outputId": "bb8c5a54-93a4-42de-fbf2-cf5e60fdd2f4"
      },
      "outputs": [],
      "source": [
        "!pip install groq"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 49,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 736
        },
        "id": "HdcXYTT9xL9d",
        "outputId": "bab24612-2c89-43cb-bfce-6f6a847a3312"
      },
      "outputs": [
        {
          "data": {
            "application/vnd.microsoft.datawrangler.viewer.v0+json": {
              "columns": [
                {
                  "name": "index",
                  "rawType": "int64",
                  "type": "integer"
                },
                {
                  "name": "pubmed_id",
                  "rawType": "int64",
                  "type": "integer"
                },
                {
                  "name": "INTRO",
                  "rawType": "object",
                  "type": "unknown"
                },
                {
                  "name": "METHODS",
                  "rawType": "object",
                  "type": "unknown"
                },
                {
                  "name": "RESULTS",
                  "rawType": "object",
                  "type": "string"
                },
                {
                  "name": "DISCUSS",
                  "rawType": "object",
                  "type": "unknown"
                },
                {
                  "name": "text_length",
                  "rawType": "int64",
                  "type": "integer"
                },
                {
                  "name": "TOPIC_LABEL",
                  "rawType": "object",
                  "type": "string"
                }
              ],
              "ref": "07e48dcb-c65d-49b6-b236-11b66903ec9f",
              "rows": [
                [
                  "0",
                  "30419927",
                  "Background<SEP>The Mediterranean diet (MeD) is considered one of the healthiest dietetic pattern in the world. It is characterized by a high consumption of olive oil, fish, fruits, legumes, vegetables, unrefined cereals, a moderate consumption of wine and dairy products, and a low consumption of non-fish meat products. According to several studies, thanks to its peculiar distribution of macro and micronutrients, this dietary model plays a key role in the prevention of a wide number of chronic diseases, such as cardiovascular diseases (CDVs), diabetes and metabolic syndrome, making an important contribution to the weight loss, especially if it is associated with energy restriction and constant physical activity. In view of above, it is worth recalling that this dietary models can also prevent and treat obesity, a disease, usually classified through that provides alterations in body composition like abnormal or excessive fat accumulation. This medical condition is caused by a several numbers of co-factors, such as psychological problems, sedentary life and incorrect dietetic habits, and it is an important risk factor for all those aforementioned diseases fought by the Mediterranean diet. Therefore, in this context, it is understandable why this dietary model plays a key role in the health protection.<SEP>Nevertheless, independently from the dietetic habits, also genetics play a key role in the development of obesity. In the human genome, a several number of genes are involved in the obesity, and Fat mass and obesity-associated gene (FTO) is surely one of the most important. This gene, widely expressed in several fetal and adult tissues, is located on chromosome 16 (16q12.2), and encodes for the enzyme alpha-ketoglutarate-dependent dioxygenase. This enzyme is implied in the regulation of both the control of adipocyte thermogenesis and differentiation, contributing considerably to the body fat accumulation. Furthermore, it contributes to the regulation of energy homeostasis and metabolic rate, increasing also the food intake. Finally, among the other activities, is implicated in the repairs of alkylated DNA and RNA via oxidative demethylation. Among the several variants of this gene, FTO rs9939609 is one of the best-know. It is located in the first intron of the gene and, in 2007, it was for the first time associated with the body mass. Subsequently, further studies demonstrated that the allele A of FTO rs9939609 is related to both a higher body mass index and body circumferences, and, among different ethnicities, also Italian population demonstrated the same results. Furthermore, this SNP is included among the genetic etiological factors in the development of both metabolic syndrome and Type-2 diabetes, and allele A was also associated with a higher fat and lean mass. In a PREDIMED substudy, it was observed that homozygous subjects for A allele had the highest baseline body weight, but also the lowest body weight gain after 3 years of Mediterranean-style intervention, compared to TT genotype. However, no interaction between nutritional intervention and the polymorphism was found. On the other hand, Ortega-Azorín et al. demonstrated consistent gene-diet interactions between FTO rs9939609 and Melanocortin-4 Receptor (MC4R) rs17782313 genes and to the Mediterranean diet adherence in the type 2 diabetes risk. The same effect was observed in obese phenotypes in Iranian population. Subjects with minor allele carriers of FTO variants rs9939973, rs8050136, rs1781749, and rs3751812 had a lower risk of obesity when they had a higher Mediterranean dietary score compared to wild-type homozygote genotype carriers.<SEP>In the light of these observations the objective of this study was to investigate about the possible influence of FTO rs9939609 variant on both weight loss and modification of body composition in Italian subjects, after having followed a 4-weeks dietetic intervention based on the Mediterranean model, in order to see if this polymorphism could influence the response to a specific dietary treatment.",
                  "Methods<SEP>Study design and subjects<SEP>The study was carried out between January 2017 and March 2018 at the Section of Clinical Nutrition and Nutrigenomics, Department of Biomedicine and Prevention of the University of Rome Tor Vergata. The initial sample recruited was composed of 300 subjects, who came up for the first time for a nutritional-medical checkup at the Clinical Nutrition and Nutrigenomic Section at the University of Rome Tor Vergata. To be eligible, each individual had to belong to the Caucasian race, to be Italian and older than 16 years old. Furthermore, for each subject, the medical assessment was evaluated before and after the application of prescribed nutritional treatments. We have pooled the sample in two groups, 150 subjects followed, for a period of 4 weeks, a Mediterranean diet (MeD), whereas the remaining 150 were allocated, within the same period, in the control group (CTR). The randomization and allocation of the study participants was performed using the IBM SPSS 21.0 for Windows (IBM Corp., Armonk, NY, USA). All the individuals included in the study approved their participation by learning and signing the informed consent, which was drawn up in accordance with the commissariat of the Ethics Committee of Medicine, University of Rome Tor Vergata and with the Helsinki Declaration of 1975 as revised in 1983. Trial Registration: this protocol has been registered by ClinicalTrials.gov, ID: NCT01890070.<SEP>Anthropometric, bioimpendance analysis, and dual-energy X-ray absorptiometry<SEP>After a 12-h overnight fasting, all subjects underwent anthropometric evaluation. All the individuals were instructed to take off their clothes and shoes before undergoing the measurements. Body weight was assessed with balance scale to the nearest 0.1 kg (Invernizzi, Rome, Italy). Height was evaluated using a stadiometer (Invernizzi, Rome, Italy) to the nearest 0.1 cm. BMI was calculated using the formula . Waist, hip, neck and abdomen circumferences were assessed using a flexible steel metric tape to the nearest 0.5 cm, according to the International Society for the Advancement of Kin anthropometry protocol and National Institute of Health Guidelines.<SEP>Body composition analysis was performed using both dual-energy X-ray absorptiometry (DXA) (I-DXA, GE Medical Systems, Milwaukee, WI, USA) and bioelectrical impedance analysis (BIA 101S, Akern/RJL Systems, Pontassieve, Florence, Italy). DXA was carried out to evaluate total, android and gynoid, of fat mass percentage (FM%), fat mass (FM) and lean mass (LM) in Kg. Total fat mass percentage (Total FM%) was calculated as Total body fat mass (Total FM) divided by the total mass of all tissues (Total LM), including the total body bone (TBBone), as the following: . Bioelectrical impedance analysis was carried out to evaluate resistance (R), reactance (Xc), phase angle (PA), hydration, exchange Na/K, total body water (TBW), extracellular water (ECW), intracellular water (ICW), body cell mass (BCM), body cell mass index (BCMI). Finally, waist/hip ratio (WHR) was analyzed and evaluated according to the clinical risk thresholds, equivalent to WHR > 0.9 for men and WHR > 0.85 for women.<SEP>According to De Lorenzo et al., we categorized our population based on phenotype classification through BMI and TBFat% as follows: underweight (UW) (BMI < 18.50); normal-weight (NW) (18.50 ≤ BMI < 25 or BMI ≥ 25 but Total TBFat% lower than 30% for women and 25% for men); normal weight obese (NWO) (18.50 ≤ BMI < 25 and TBFat% higher than 30% for women and 25% for men); Preobese (PreOb) (25 ≤ BMI < 30 and TBFat% higher than 30% for women and 25% for men); Obesity I (30 ≤ BMI < 35); Obesity II (35 ≤ BMI < 40); Obesity III (BMI ≥ 40).<SEP>DNA isolation and RTq-PCR analysis<SEP>The phenol–chloroform extraction described by Barker et al. was used to extract the Genomic DNA, which was collected, in turn, via saliva swab. To prepare the gDNA for the genotyping, a two allele-specific fluorescent probe including a PCR primer pair (TaqMan SNP Genotyping Assays, Life Technologies, CA, USA) and a master mix including dNTPs and Taq DNA Polymerase (TaqPath ProAmp Master mix Life Technologies, CA, USA) were used. The FTO rs9939609 context sequence was the following: GGTTCCTTGCGACTGCTGTGAATTT [A/T] GTGATGCACTTGGATAGTCTCTGTT. Finally, SNP genotyping assessment was executed using a Real-Time PCR analysis (Applied Biosystems StepOnePLus Real-Time PCR, Life Technologies, CA, USA), according to the manufacturer’s instructions.<SEP>Dietary assessments<SEP>At baseline subjects food intake was assessed with a 3-day diet records completed for 2 weekdays and 1 weekend day. Participants were instructed to record weight and/or measures of foods and beverages consumed. Diet records were reviewed to clarify the amounts of foods ingested. The estimated intake of macronutrients was calculated by using Dietosystem dietary software (DS Medica S.r.l., Milan, Italy).<SEP>Dietary intervention<SEP>In MeD intervention, an isocaloric Mediterranean diet in which the daily macronutrients intake was distributed as follows: 55% of carbohydrates, 20% of protein (> 50% of vegetable derivation), < 25% of lipids (on total daily energy intake: saturated fat < 10%, 6–10% polyunsaturated fatty acids (PUFA), n-6/n-3 PUFA ratio of 3:1, 15% of monounsaturated fatty acids (MUFA); < 1% trans-fatty acids) and 25 g of fiber.<SEP>For each subject, the energy intake was calculate according to the estimation of the rest energy expenditure (REE), which was determined using the Weir Formula:, in which VO2 and VCO2 were calculated as follows:<SEP>VO2 = Total LM DXA (kg) × 5 for females, VO2 = Total LM DXA (kg) × 4.5 for males and VCO2 = VO2 × 0.85.<SEP>The REE was multiplied for the proper physical activity level (PAL) according to the WHO and Food and Agriculture Organization of the United Nations (FAO) recommendations.Conversely, in the control group, the subjects did not follow any specific diet, but they have received general recommendations on healthy nutritional habit and were only monitored, even though also for these subjects REE and the energy intake were evaluated as aforementioned.<SEP>Analysis of blood sample<SEP>Blood samples, taken after a 12-h overnight fast, were collected in sterile tubes containing EDTA (Vacutainer®) and put on ice. Plasma, after being separated by centrifugation (1600 rpm, at 4 °C for 10 min), was removed, aliquoted and stored at − 80 °C. All clinical chemistry analyses, except serum lipid plasma and glucose analysis, were performed using an ADVIA®1800 Chemistry System (Siemens Healthcare), following standard procedures. Plasma glucose concentrations were measured through the glucose oxidase method and automated glucose analyzer (COBAS INTEGRA 400, Roche Diagnostics, Indianapolis, IN, USA); serum lipid profile constituents were evaluated by standard enzymatic colorimetric techniques (Roche143 Modular P800, Roche Diagnostics, Indianapolis, IN, USA).<SEP>Statistical analysis<SEP>The SNP-HWE program was used to calculate the Hardy-Weinberg equilibrium (HWE) for FTO rs9939609, and the result was tested using the χ2 analysis. To analyze the sample, the subjects were divided into carrier/non-carrier (carrier for A allele vs homozygous T) and into MeD and CTR (Mediterranean diet vs Control group). The Kolmogorov–Smirnov test was used to analyze the distribution of variables, and data were normalized according to Z-score transformation. T-test analysis was performed between A carriers and TT genotype at baseline, as well as ANOVA one way analysis, adjusted with Bonferroni post hoc analysis, between the genotype/treatment groups in order to see significant differences among groups (p < 0.05). Change (Δ) in body weight (kg), BMI, neck, waist, abdomen and hip circumferences (cm), WHR, R, Xc, BCM (kg), Na/K, TBW (L), ECW (L), ICWb (L), BCMI, Android Bfat (%), Gynoid Bfat (%), TBFat (%), Android Bfat (Kg), Gynoid Bfat (Kg), TBFat (Kg), Android BLean (kg), Gynoid BLean (kg), TBLean (kg) and REE were calculated by subtracting measurements recorded before the beginning of the nutritional treatments, from the measurement recorded after the completion of the nutritional intervention. In order to compare the differences in mean of the aforementioned values, respectively for A carriers and TT genotype, gene, diet and gene–diet interaction analyses were carried out using a Generalized Linear Model (linear GLM), adjusted for sex and age. TBFat (kg) was selected as a parameter to calculate minimum sample size. The minimum sample size was calculated using a two-tailed one-sample Student’s t-test, considering:(i) TBFat to be detected between baseline and MED|δ| = − 3.86 kg; (ii) SD of the paired differences SD = 6.38 kg, (iii) type I error probability α = 0.05 and power 1 − β = 0.95. The result was a minimum sample size of 36 for MED and 12 for CTR group. Significance was set as p < 0.05 and the statistical analysis was performed using IBM SPSS 21.0 for Windows (IBM Corp., Armonk, NY, USA).",
                  "Results<SEP>Population characteristics<SEP>The enrolled 300 subjects met the inclusion criteria and nobody declined to participate. Subjects were equally randomized allocated in MeD group and CTR group. During this clinical trial, 11 subjects from MeD group and 101 subjects from CTR group abandoned the study as specified in Fig. 1. In fact, 5 subjects form the MeD group abandoned the study for poor weight loss results and 6 subjects for poor adherence to the diet therapy. On the other hand, in the CTR group 101 subjects abandoned the study since they did not see any kind of health benefits or improvement. The final sample analyzed consisted of 188 patients, divided as follow: 139 subjects in MeD group and 49 for CTR group. These patients successfully participated and completed the study protocol. <SEP>In our sample, the HWE was respected (p > 0.05). The comprehensive description of the whole sample population at baseline can be seen in Table 1. All results were stated as mean and standard deviation. The average age of the individuals was 46.83 ± 15.01 years, 62.2% females and 37.8% males (Table 1). According to the statistical analysis, at baseline, carriers and no carriers differs significantly only for age, diastolic blood pressure, R, basophils count and unsaturated fatty acids and polyunsaturated fatty acids intake (p < 0.05) (Table 1). Genotype frequencies of our individuals (TT: 0.310; AA: 0.190; AT: 0.500) are similar to the ones shown in TSI population (TT: 0.327, AA: 0.252, AT: 0.421), and the same result was demonstrated about the allele frequencies, similar between TSI (T: 0.537; A: 0.463) and our sample (T: 0.550; A: 0.450) (Table 2). The average BMI of the subject was 29.39 ± 6.99, and the average Total FM (kg) was 37.3 ± 9.76. Finally, in order to have a detailed description of the sample, in Table 3 the individuals, at the baseline, were divided by genotype and categorized according to phenotype classification.   <SEP>At baseline, differences were highlighted between TT genotype and A carriers for age, DBP, BCM (kg), basophils (1000/μL), unsaturated fatty acids (g) and polyunsaturated fatty acids (g) intake (p < 0.05).<SEP>Among the genotype/treatment groups significances were found for SBP between control group TT genotype and Med group TT genotype, DBP between control group TT genotype and the other groups (control group A carriers, Med group TT genotype and Med group A carriers) and neck circumference between control group TT genotype and Med group A carriers, control group A carriers and Med group TT genotype and Med group A carriers (p < 0.05). No other statistical significance were found at baseline between groups.<SEP>Influences of FTO rs9939609 and nutritional intervention on BMI, body composition and metabolism<SEP>In this study, the GLM analysis was used to demonstrate the statistical significance between FTO rs9939609 carriers A and TT genotype together with the nutritional intervention.<SEP>According to the results, TBFat (kg) decreases dependently both of the nutritional intervention (pD = 0.00) and the interaction gene-diet (pGD = 0.04), showing a significant difference between MeD and CTR, but also suggesting a potential role, even though not statistically significant, of FTO rs9939609 (pG = 0.06), since A carrier CTR gained weight sensibly more than TT genotype CTR, and A carrier MeD lost more weight than TT genotype MeD (MeD/TT genotype = − 3.59 ± 4.78; MeD/A carriers = − 3.97 ± 6.97; CTR/TT genotype = 1.93 ± 5.54; CTR/A carriers = 6.11 ± 9.60) (Table 4). Furthermore, MeD group, regardless of FTO rs9939609 (pG = 0.70; pGD = 0.32) (Table 4), lost a higher amount of Gynoid BFat (%) compared to CTR (p = 0.04) (MeD/TT genotype = − 2.94 ± 5.14; MeD/A carriers = − 1.67 ± 3.51; CTR/TT genotype = 0.03 ± 1.26; CTR/A carriers = − 0.42 ± 1.74) (Table 4). <SEP>The bioelectrical impedance analysis highlighted that TBW, was significantly reduced in A carriers compared to TT genotype (pG = 0.02), independently of nutritional intervention (pD = 0.48; pGD = 0.47) (MeD/TT genotype = − 0.83 ± 1.63; MeD/A carriers = − 1.88 ± 4.35; CTR/TT genotype = 0.70 ± 3.04; CTR/A carriers = − 1.75 ± 3.28) (Table 4). All the other results were not statistically significant (p > 0.05).",
                  "Discussion<SEP>Among the different dietetic patterns, the Mediterranean diet is surely one of the healthiest in the world. In fact, several studies demonstrated its capability to prevent different cardiovascular and metabolic diseases, such as metabolic syndrome and type-2 diabetes. Moreover, this type of dietetic model turns out to be fundamental both in preventing and treating the obesity, which is becoming one of the most widespread medical condition in the world, since recent statistics demonstrated that, by 2030, the 20% of the global adult population will be obese, and the 38% will be overweight. Nevertheless, also genetics plays an important role in the development of the obesity. So far, a wide number of genes are associated with the body composition, and one of the most studies is certainly FTO. Several variants of this gene were associated with the BMI, and FTO rs9939609 is certainly one of the most known. In fact, its allele A is highly related with higher BMI, fat and lean mass. On the contrary, for the time being, several studies demonstrated that, during a dietetic treatment, FTO rs9939609 do not seem to influence the weight loss. In view of above, we have conducted this study in order to analyze whether and how FTO rs9939609 variant influence both weight loss and body composition in Italian patients, within a Mediterranean dietetic treatment.<SEP>According to our results, TBW is highly influenced by FTO. In fact, analyzing the alteration of this value in each group (Table 4), it is possible to see that A carriers lost a higher quantity of body water compared to TT genotype (MeD/TT genotype = − 0.83 ± 1.63; CTR/TT genotype = 0.70 ± 3.04; MeD/A carriers = − 1.88 ± 4.35; CTR/A carriers = − 1.75 ± 3.28) (Table 4), whilst no difference due to MeD intervention was found. This result may be a confounding factor in the weight loss, since it influences the decreasing of both BMI and body weight during nutritional treatments, overestimating the effective fat mass loss. Analyzing the results of body composition evaluated via DXA, both dietetic treatment and interaction diet-gene influence the total body fat mass, whilst FTO alone did not demonstrate the same effect. Moreover, Gynoid BFat% declining was highly related to the dietetic therapy, confirming that MeD treatment had an effect on body composition, confirming its capabilities, already demonstrated in several studies, to reduce body fat, giving its important contribution against all the diseases related to the body fat mass. In our sample, the difference between the two nutritional treatments is highly considerable, since both MeD groups lost weight in a similar manner. It is also important to highlight that, in our sample, FTO did not influence the outcome of the nutritional treatment, thus confirming the results of previous studies. These results might mean that, during a dietetic therapy, this SNP should not be taken into consideration, since it does not influence the outcome of both body composition and anthropometric measurements. However, the fact that also the interaction gene-diet had an effect on the TBFat suggests that, even though slightly, FTO might influence the outcome of the Mediterranean diet on this specific value, as previously proposed by another study. Nonetheless, analyzing our results, it can be assumed that this statistical significance may be given by the incredible amount of TBFat gained by the A carriers/CTR group, thus suggesting an interesting role of this SNP in the development and preservation of body fat mass outside periods of nutritional therapies, as previously assumed by studies on murine models. Considering these results, we suggest that understanding whether and how FTO, along with other genes or alone, influence the fat mass loss during nutritional treatments could help us to draft better clinical pictures of patients, and prevent difficulties of weight loss due to genetic factors. Moreover, both nutritional treatment and FTO alleles did not influence the lean mass, which, in contrast to the fat mass, remained almost steady within the studied period in all the analyzed groups. Finally, according to the statistical significance, in our sample, FTO rs9939609 did not influence body circumferences, weight, and consequently BMI. Nevertheless, even though in a non-significant manner, analyzing the data in Table 4, it can be seen how A carriers subjects lost less weight compared to TT genotype, following a trend already noted, and leaving some doubt about the effective activities of this SNP during nutritional treatments.",
                  "21979",
                  "['OBWCCE', 'EATBTS', 'FOODIN']"
                ],
                [
                  "1",
                  "30032228",
                  "Introduction<SEP>Overweight and obesity have become a worldwide health problem. Between 1975 and 2014, the prevalence of obesity has more than doubled. According to the WHO, 39% of adults were overweight and 13% were obese in 2014. The obesity epidemic is mainly due to the modern lifestyle, which is characterized by low physical activity and a high consumption of energy-dense food. However, genetic factors also play a substantial role in the pathogenesis of obesity. To date, >100 loci have been identified for an association with BMI, but the causal genetic variants and their underlying biological mechanisms are largely unknown. Furthermore, a modification of eating behaviors by genetic variants has been described.<SEP>In addition, studies have shown a considerable interindividual variation in metabolic responses to defined meal challenges. This variability may be partly explained by genetic influences, and there is growing interest to better understand the gene-diet associations. The identification of associations as well as interactions between loci and dietary intake may help to elucidate the molecular pathways that link them with body weight. More research on these interactions has been recently promoted by an NIH Working Group. There are currently major efforts to investigate the association between genetic factors and dietary intake. For instance, loci associated with obesity are expressed in the brain, assuming a potential role in eating behavior and food preferences. A recent genomewide association study (GWAS) suggested that genetic variants are associated with macronutrient consumption in observational studies. In addition, in 3 independent populations a gene-diet interaction on obesity has been shown for an APOA2 polymorphism and saturated fat intake. In that context, most of the literature tackles the fat mass and obesity–associated (FTO) gene or consists of single findings from candidate gene studies without any replication. Further challenges include imprecise assessment of dietary intake, a high heterogeneity in study design, as well as the loss of standardized statistical models.<SEP>The general vision of these research activities is that genotype-based dietary recommendations may become a more effective approach for weight management and disease prevention. So far, the Food4Me project provided evidence that the personalized intervention groups lost more weight than the control group. However, integrating the information on different genetic variants into the personalized dietary recommendations had no benefit for weight loss. Against this background, several commercially available genetic tests (direct-to-consumer tests) are currently offered with the promise to provide reliable information for better prevention or treatment of obesity and related metabolic disturbances. However, a strong evidence base for these tests is currently lacking.<SEP>Therefore, the aim of this study was to perform a systematic literature search to study potential associations between genetic variants and total energy, carbohydrate, and fat intakes and to provide a better knowledge base for future direct-to-consumer tests. The results will be beneficial for hypotheses of clinical trials on gene-diet interactions. They should also serve to develop more robust personalized dietary recommendations and, finally, to improve the prevention and treatment of obesity and metabolic diseases.",
                  "Methods<SEP>This systematic review was performed according to the guidelines on systematic reviewing methodology and the Preferred Reporting Items for Systematic Reviews (PRISMA) have been considered. This review was registered in the International Prospective Register of Systematic Reviews (PROSPERO; registration no. CRD42015025738).<SEP>Search Strategy<SEP>The 4 electronic databases the Cochrane Library, Web of Science, PubMed, and Embase were searched for articles published between 1994 and September 2017. The search terms used were “genetic variant,” “gene variant,” “genotype,” “single nucleotide polymorphism,” “SNP,” “FTO,” “FABP,” “PPARG,” “ADRB,” “APOA2,” and “APOA5.” Due to biological knowledge [e.g., PPAR γ (PPARG) ] as well as due to application in direct-to-consumer tests (e.g., bodykey by NUTRILITE, Amway GmbH, Puchheim, Germany), the search terms included some selected gene names. In particular, the FTO locus as the genomic region with the strongest effect on body weight was specifically included in the literature search. Furthermore, due to gene-diet interactions in clinical research studies, APOA2 and APOA5 were specifically included. For the search strategy, the genetic terms were combined with the Boolean operator “OR.” The second search terms related to nutrition included “diet,” “energy intake,” “macronutrient intake,” “carbohydrate intake,” and “fat intake.” The nutritional search terms were combined with the Boolean operator “OR.” The genetic and nutritional search terms were combined with the Boolean operator “AND.” Depending on the database, plural forms of the search terms as well as quotation marks were used (Supplemental Material). Additional articles were identified through a hand-search of the reference lists of relevant publications.<SEP>Article Screening and Selection<SEP>All of the records identified through the electronic database search were imported into the reference management software EndNote X7 (Thomsen Reuters). After the removal of duplicates, 2 independent reviewers (TD and JG) assessed titles, abstracts, and full-text articles for eligibility according to the following inclusion criteria: articles in English or German language, published between 1994 and September 2017, and in adults. Animal studies as well as studies focusing on specific populations such as pregnant or breastfeeding women or patients with cancer or other severe diseases were excluded. Furthermore, articles investigating the association between single nucleotide polymorphisms (SNPs) and the intake of food groups or the adherence to a specific diet without analyzing the association with total energy, carbohydrate, or fat intakes were not considered. Publications on copy number variations, dietary patterns, or interaction terms with no clear analysis of SNPs and energy, carbohydrate, or fat intakes were not included in this systematic review. Because protein intake accounts for fewer calories than carbohydrate and fat intakes, it has a minor contribution to energy balance. Furthermore, dietary recommendations for weight loss usually do not refer to protein intake. Thus, protein intake was not considered. Reasons for excluding articles were documented. Throughout the article screening, discrepancies between the 2 reviewers were discussed with a third reviewer (CH). The review team contacted authors if additional information was required.<SEP>Data Extraction<SEP>Two independent reviewers extracted the data from relevant articles into an Excel spreadsheet in order to synthesize results narratively and in a tabular form. Data extraction included the study design, study participants, intervention, primary and secondary outcomes, sample size, statistical methods, and assessment of total energy, carbohydrate, and fat intakes. Results for an association between SNPs and total energy, carbohydrate, and fat intakes were documented. Inconsistencies during data extraction were discussed with the third reviewer. Furthermore, the linkage disequilibrium (LD) based on the genotype data for the CEU population (i.e. Utah residents with Northern and Western European ancestry) was used in order to summarize and interpret findings. LD plots for the FTO and MC4R SNPs are shown in Supplemental Figures 1 and 2.<SEP>The results were reported in a tabular synthesis, separately for each genotype according to the number of publications per single locus. In the narrative synthesis, each SNP was discussed for an association with total energy, carbohydrate, and fat intakes without quality assessment of the articles. Loci<SEP>published in only 1 article are listed in Supplemental Table 1.",
                  "Results<SEP>Articles Identified<SEP>The initial database search identified 14,692 articles (Figure 1). Thirty-nine articles reported findings on the association between SNPs and total energy, carbohydrate, or fat intakes. These articles met the inclusion criteria according to the PICOS (Participants, Intervention, Comparator, Outcome, Study design) statement for systematic literature search (Participants: adults without severe diseases and nonpregnant or nonbreastfeeding women; Intervention: not described; Comparator: SNPs and loci; Outcome: total energy, carbohydrate, and fat intakes; Study design: all kind of studies). Twenty articles presented loci that were described only in a single study (Supplemental Table 1). SNPs that were investigated in >1 study were reviewed according to the number of articles and the analyzed SNPs.<SEP>Characteristics of Included Studies<SEP>The publication dates of articles ranged between 2000 and 2017. More than 80% of the articles (n = 32) represented cross-sectional, cohort, or case-control studies. Two postprandial studies and 5 meta-analyses were included in the current review. The sample sizes ranged from 20 to 29,480<SEP>subjects and ≤213,173 individual participants in the meta-analyses. FFQs, food records, or dietary recalls were applied for the assessment of dietary intake. However, most studies (n = 25) used FFQs. The included studies differed in terms of population characteristics such as BMI, sex distribution, disease status, and ethnicity, as well as statistical methods applied (Tables 1–3, Supplemental Table 1). Furthermore, FTO and MC4R SNPs differed in low and high LD values (Supplemental Figures 1 and 2).<SEP>Gene Loci and Dietary Intake<SEP>In the following, studies considering the association between the most commonly studied loci and dietary intake are presented. Figure 2 shows the overlap of significant associations between SNPs and total energy, carbohydrate, and fat intakes.<SEP> FTO Locus and Dietary Intake<SEP>In total, 13 studies as well as 4 meta-analyses reported on potential associations between the FTO locus and total energy, carbohydrate, and fat intakes. Approximately 40% of these articles (n = 7) reported results on populations of European ancestry. Six publications presented data for different populations (e.g., Asian, African American, American Indian, Hispanic, and Asian or Pacific Islanders). The sample sizes varied between 40 and 29,480 participants in observational and experimental studies and yielded 213,173 subjects in the meta-analyses. In total, 8 different FTO SNPs were investigated, whereas data for SNP rs9939609 were presented in >60% of the articles (n = 11) (Table 1).<SEP>Three publications, including 2 meta-analyses, reported a significant association between the A risk allele of rs9939609 and lower total energy intake (P < 0.01). Both meta-analyses estimated a lower total energy intake of 6.4 kcal/d (unadjusted) or 5.9 kcal/d (adjusted), respectively. In contrast, other studies reported evidence of a significant association between the A risk allele with a higher total energy intake. Four articles did not find evidence for a significant association between SNP rs9939609 and total energy intake (Table 1). In addition, an association with higher total energy intake was reported for risk allele carriers of the FTO SNPs rs1421085 and rs8050136, which are in a high LD to rs9939609 (r² > 0.90), whereas Park et al. observed findings in the opposite direction in African and European Americans. The SNPs rs3751812, rs9922708, and rs1121980 showed consistently nonsignificant associations with total energy intake (Table 1).<SEP>With regard to carbohydrate intake, FTO risk allele carriers of rs9939609 consumed fewer carbohydrates [ß = –0.002% of energy (%E); P = 0.005]. Lear et al. analyzed a sample of 706 individuals of different ethnicities for associations between rs9939609 and carbohydrate intake, resulting in nonsignificant findings. However, subanalyses suggested a lower carbohydrate intake per A allele change in Aboriginal Canadians (ß = –2.2%E; P = 0.049) and a higher intake per minor A allele in participants of European descent (ß = 2.3%E; P = 0.007). Park et al. observed a significantly lower carbohydrate intake (in %E) per A allele change for the SNP rs8050136, whereas Haupt et al. did not. In a joint analysis of samples of European ancestry (n = 71,326), A allele carriers of the SNP rs10163409, which is not in LD to rs8050136, showed a significantly positive association with carbohydrate intake (ß = 0.166%E; P = 0.001). Nonsignificant results were consistently reported for the SNPs rs1421085, rs3751812, rs9922708, rs9935401, and rs1121980 and carbohydrate intake (Table 1).<SEP>With regard to fat intake, significantly positive associations between obesity-risk alleles of SNP rs9939609, rs1421085, and rs8050136 and fat intake were observed. Steemburgo et al. showed a positive association between SNP rs9939609 and fat intake as percentage of energy (P = 0.019) only for females (n = 126). McCaffery et al. confirmed this result in a subanalysis of non-Hispanic white participants, showing a nominal association between the obesity-associated minor allele of rs1421085 and greater fat intake (ß = 0.37%E; P = 0.042). Moreover, Park et al. described a significantly positive association between the FTO rs8050136 genotype and higher fat intake (Table 1). In contrast, in the meta-analysis of Livingstone et al., the FTO risk allele carriers of rs9939609 consumed less fat (P = 0.004). Chu et al. showed a significant inverse association between the SNP rs10163409 and fat intake (ß = –0.22%E; P ≤ 0.001) in a subcohort of 33,531 individuals. With regard to fat intake, nonsignificant results were consistently reported for the SNPs rs3751812, rs9922708, rs9935401, and rs1121980 (Table 1).<SEP>Melanocortin 4 Receptor Locus and Dietary Intake<SEP>Eleven publications (8 observational studies, 1 experimental study, and 2 meta-analyses) studied associations between the melanocortin 4 receptor (MC4R) locus and total energy, carbohydrate, and fat intakes (Table 2). Almost one-third of the articles (n = 3) included data from European populations. The remaining articles included data from mixed populations (n = 4) or nonspecified individuals (n = 4). The sample sizes varied between 40 and 29,480 individuals for observational and experimental studies and between 36,973 and 177,330 for the 2 meta-analyses. The studies reported data on 5 MC4R SNPs: rs17782313, rs17700633, rs17700144, rs2229616, and rs571312 (Table 2).<SEP>Most articles (8 out of 11) investigated the MC4R SNP rs17782313, which is in high LD to rs571312 (r² = 0.96). Two of them reported a significant association between the CC genotype and a higher total energy intake (P < 0.01). For SNP rs571312, a similar significantly positive association with energy intake was reported (ß = 58.84 kcal/d; P = 0.02). Other studies, including 2 meta-analyses, did not find significant associations between MC4R SNPs and total energy intake, or for rs17782313, rs17700633, or rs2229616 (Table 2).<SEP>With regard to carbohydrate intake, a study in an Iranian population showed a significant association between SNP rs17782313 and a lower carbohydrate intake (P = 0.04 in lean subjects, P < 0.001 in overweight subjects). A study in persons with severe obesity showed that carriers of the rs2229616 variant 103I had a higher carbohydrate intake (P = 0.01). Other studies did not observe significant associations between SNPs rs17782313, rs17700633, rs571312, or rs17700144 and carbohydrate intake.<SEP>With regard to fat intake, results were nonsignificant, with 1 exception (Table 2). Only the analysis of 5724 white women, performed by Qi et al. showed a significant association between the CC genotype and a higher total fat intake in grams per day (P = 0.001). This association was no longer significant, when data were expressed as percentage of total energy intake.<SEP>Other Loci and Dietary Intake<SEP>The remaining 18 loci analyzed for an association with total energy, carbohydrate, or fat intakes in >1 article are listed in Table 3. Ten cross-sectional studies and 3 meta-analyses were identified. The sample sizes varied between 118 and 71,326 participants. Data on total energy intake were available in 10 articles and data on carbohydrate and fat intakes in 11 articles.<SEP>Two studies on the brain-derived neurotrophic factor (BDNF) locus showed a significantly positive association between the obesity-risk alleles and total energy intake. No significant association between BDNF SNPs and total energy intake was found in the other articles. With regard to carbohydrate and fat intakes, nonsignificant findings were reported for all 7 BDNF SNPs.<SEP>For SNP rs6548238 in the transmembrane protein 18 (TMEM18) locus, a significant association with fat intake (P = 0.030) in a sample of 12,462 German adults was reported for T allele carriers, which was not confirmed by other studies. Consistently, no significant associations were found between TMEM18 SNPs and total energy or carbohydrate intakes.<SEP>For the potassium channel tetramerization domain-containing 15 (KCTD15) locus, no significant associations with total energy intake were shown. However, SNP rs368794 showed a significant association with higher carbohydrate intake (P = 0.020). Studies on the SNP rs11084753, which is in LD to rs368794 (r² = 0.8) did not confirm this result. With regard to fat intake and rs11084753, Park et al. did not find any significant relation, whereas Holzapfel et al. described a significant association with a higher fat score (P = 0.03). Moreover, no significant associations were reported for rs29941 or for rs368794 and fat intake.<SEP>A significant association between the neuronal growth regulator 1 (NEGR1) SNP rs2815752 and lower total energy intake (P = 0.004) was described in a Swedish cross-sectional study (n = 29,480). However, significance was lost after correction for misreporting. McCaffery et al. confirmed the significant finding in non-Hispanic white participants. This association was not confirmed by the meta-analysis of Park et al.. In the Swedish study, a significant association between rs2815752 and higher carbohydrate intake (P ≤ 0.001) was reported. This significant result was not confirmed by the other articles. With regard to fat intake, no association with NEGR1 SNPs was described, with the exception of Rukh et al. who found a significantly inverse association between the rs2815752 T allele carriers and fat intake (ß = –0.21%E; P ≤ 0.001) (Table 3).<SEP>None of the studies investigating the SH2B adaptor protein 1 (SH2B1) locus reported significant associations with total energy, carbohydrate, or fat intakes, with one exception. Bauer et al. found a significantly higher fat intake for risk G allele carriers of SNP rs7498665 (P = 0.003). The SNPs rs7647305 and rs9816226 at the ETS variant 5 (ETV5/SFRS10) locus showed no significant associations with total energy, carbohydrate, or fat intakes, except for the subanalysis of McCaffery et al..<SEP>The glucosamine-6-phosphate deaminase 2 (GNPDA2) locus was not significantly associated with total energy, carbohydrate, or fat intakes (Table 3). Only Rukh et al. showed that the obesity-risk G allele was significantly associated (ß = –10.97 kcal/d; P = 0.039) with a lower energy intake, although this result was no longer significant after correction for multiple comparisons.<SEP>Two SNPs that were in high LD (r² = 0.81) at the fibroblast growth factor 21 (FGF21) locus were significantly associated with a higher carbohydrate intake as a percentage of energy (P < 0.001). Chu et al. presented data on fat intake and showed a significantly negative association with SNP rs838133 (ß = –0.21%E).<SEP>For the PPARG SNP rs1801282, there was no significant association with total energy or carbohydrate intakes (Table 3). However, a significantly higher fat intake was observed for Ala12 carriers compared with carriers of the Pro12 allele. Significantly positive associations were also found for the rs1761667 at the CD36 locus. Ramos-Lopez et al. presented data showing a higher intake of calories (P < 0.001) and a higher total fat intake (P < 0.001) for the AA genotype of overweight participants but not for subjects with obesity. This association was not confirmed by the study of Shen et al..<SEP> No significant associations between SNPs and total energy, carbohydrate, or fat intakes were found for the other loci described in Table 3.",
                  "Discussion<SEP>The purpose of this review was to systematically explore associations between SNPs and total energy, carbohydrate, or fat intakes. In total, 39 articles, including 86 different loci and 176 SNPs, were identified. Twenty loci were described in detail in this review, and the others are listed in Supplemental Table 1.<SEP>First, significant associations were reported between the FTO rs9939609 and rs8050136 risk alleles and total energy intake (Table 1). One study and 2 partly overlapping meta-analyses reported a significant association (P < 0.003) between rs9939609 and a lower total energy intake. Considering the estimated effect sizes per risk allele of −6.4 (unadjusted) and −5.9 (adjusted) kcal/d from the 2 partly overlapping meta-analyses, the results might be clinically irrelevant with respect to measurement errors in the collection of dietary data. In contrast, studies reporting a significantly higher energy intake presented unadjusted results, apart from Oyeyemi et al. and Lear et al. who adjusted for age and sex. Furthermore, the significant associations with higher energy intake were described in specific target groups and limited sample sizes.<SEP>It must be emphasized that results were mainly obtained in cohorts of European ancestry. Qi et al. replicated the significant association between rs9939609 and a lower energy intake in a subanalysis of whites, but Lear et al. could not replicate a significant association after dividing participants according to ethnicity. Therefore, the significant association between FTO rs9939609 and energy intake observed in white populations is possibly not applicable to other ethnicities.<SEP>Moreover, the reported association with a lower energy intake was unexpected due to the association between the FTO locus and a higher body mass. In addition, a major limitation is the measurement error inherent in collecting self-reported dietary intake data. Rukh et al. indicated that the significant association between the FTO locus and lower energy intake became nonsignificant after excluding misreporters. Sonestedt et al. analyzed the relation between FTO risk allele carriers and underreporting of dietary intake, showing a higher frequency of underreporters among AA carriers of the FTO SNP rs9939609 than among TT carriers.<SEP>However, since Frayling et al. identified the association between the FTO locus and BMI, many studies have tried to elucidate the molecular mechanisms underlying this relation. Recently, Claussnitzer et al. identified the FTO SNP rs1421085 as the causal variant whose risk allele leads to an enhanced fat storage and lower mitochondrial fat burning. This latter observation suggests that the modest increase in fat storage is due to lower thermogenesis and energy expenditure rather than due to a difference in energy intake. Taken together, the association between FTO SNPs and a lower total energy intake is weak and possibly of low clinical relevance despite large sample sizes and sufficient statistical power. In addition, there is little biological plausibility due to the heterogeneity of reported mechanisms and, in particular, evidence that gene variants may affect energy expenditure rather than intake.<SEP>The results for an association between FTO SNPs and carbohydrate intake were also inconsistent. Livingstone et al. and Park et al. showed a significantly lower carbohydrate intake in risk allele carriers of rs9939609 and rs8050136, respectively. Chu et al. showed a higher intake for rs10163409. No significant evidence for an association between SNPs and carbohydrate intake was shown in the remaining studies (n = 10). In addition, a subanalysis by Lear et al. described a significant association between rs9939609 and a higher carbohydrate intake in a cohort of Aboriginal Canadians and a lower carbohydrate intake in European persons. This ethnic difference was not confirmed by the meta-analysis of Qi et al.. Therefore, there is no consistent evidence to conclude that FTO SNPs are associated with carbohydrate intake.<SEP>Six out of 17 articles described a significant association between FTO SNPs and fat intake, whereas 1 study and 1 meta-analysis reported a significantly positive association between rs9939609 and fat intake. Consistently, a significant association between the risk C allele of rs1421085 (LD to rs9939609, r2 = 0.9) and a higher fat intake as a percentage of energy was reported in a mixed population and in a subgroup of non-Hispanic white participants. Furthermore, the meta-analysis of Park et al. suggests that carriers of the obesity-risk allele (C allele) of the SNP rs8050136 are characterized by a higher fat intake. Due to the high linkage between these SNPs, results suggest a positive association between the FTO risk allele and fat intake. However, the effect sizes were small and the changes in dietary intake across years were not considered. It is questionable whether this finding, which is largely based on epidemiologic studies, is of clinical relevance.<SEP>Second, results for an association between MC4R SNPs and dietary intake were inconsistent. The MC4R locus is known to be associated with BMI, eating behavior, and the regulation of food intake. Significantly positive associations with total energy intake have been shown for rs17782313 and rs571312. Khalilitehrani et al. showed that, after further adjustment for energy intake, the significant association remained only in the overweight group. The same study also presented significant evidence for a negative association between rs17782313 and carbohydrate intake. The results may be ascribed to the association between MC4R and BMI itself. However, no other studies (n = 6) confirmed this significant result. Furthermore, the association between a higher total fat intake and rs17782313 was no longer significant after adjustment for energy intake. Therefore, the genetic association studies identified in this review do not provide consistent evidence that the MC4R SNPs are significantly associated with total energy, carbohydrate, or fat intakes.<SEP>Third, associations between further loci and total energy, carbohydrate, and fat intakes were more consistent, especially in terms of nonsignificant findings. One reason could be that fewer articles, and therefore lower heterogeneity, have been published for these loci. Most of the loci (Table 3) are known to be associated with body weight management. Moreover, BDNF is involved in neuronal regulatory pathways of appetite and energy balance in animal studies but is also associated with BMI in humans. The latter might explain the consistent positive association between total energy intake and the obesity-risk genotype of BDNF SNPs. Significant associations between a lower energy intake and the obesity-risk allele of the SNP rs2815752 (NEGR1) were shown in 2 studies. It can be supposed that the NEGR1 as well as TMEM18 loci, which are also associated with BMI, are involved in food regulation due to their function in neural development. As investigated by Berglund et al., the FGF21 locus is involved in carbohydrate and lipid metabolism, which may drive the positive association between FGF21 SNPs and carbohydrate intake.<SEP>Strengths and Limitations<SEP>The major strength of this systematic review is that the inclusion of identified SNPs associated with total energy, carbohydrate, or fat intakes was not limited to a specific locus. This gives a wide overview of articles focusing on a direct association between SNPs and total energy, carbohydrate, and fat intakes, published between 1994 and September 2017. This strength is in direct contrast to the systematic review and meta-analysis performed by Livingstone et al. who focused only on associations between FTO SNPs and macronutrient intake. The other 3 meta-analyses were not based on a systematic review, but on original data. A further strength is that this systematic review was not restricted to a specific study type and included experimental as well as population-based studies. However, this does not allow pooling of data in a meta-analysis. The present review did not include articles describing copy number variations, mutation analyses, haplotypes, or studies investigating the association between genetic factors and food groups or dietary patterns as well as gene-diet interactions. Furthermore, potential associations between SNPs and protein intake, which plays a minor role in the treatment and prevention of overweight and obesity, were not considered. It must be mentioned that GWASs identified SNPs at the FTO or FGF21 loci, which may be relevant for protein intake. For instance Chu et al. showed no significant evidence for an association between rs10163409 at the FTO locus and protein intake (ß = − 0.05%E; P = 0.08), whereas Tanaka et al. showed a significantly positive association between rs1421085, which was not in LD to rs10163409, and protein intake (ß = 0.08; P ≤ 0.001). However, due to the exclusion of protein intake for this review, the article gained more focus and clarity.<SEP>The search strategy might be also biased due to the inclusion of selected gene names. It is assumed that this had no impact on the results, because the MC4R locus—not included in the list of search terms—was identified as the second most common locus published for the reviewed topic.<SEP>A limitation is the high heterogeneity of data due to sample size, nutritional assessment, and characteristics of participants. The FFQ is the most commonly used dietary assessment tool to represent energy intake in observational studies. As mentioned by Cade et al., FFQs may poorly represent dietary intake, which can lead to both overestimation and underestimation of macronutrient intake. It could be speculated that there is a relation between the assessment tool used for dietary intake as well as the study type and the significance level of the results. As shown in Tables 1–3, there was no tendency for such bias.<SEP>Studies also varied in the statistical analysis and in the definition of the primary endpoints. In nutrition research, the comparison of results across studies and the replication of valid data is a major area of concern. There is an urgent need for studies harmonizing the data on macronutrient intake between the cohorts and standardizing the applied statistical models.<SEP>Due to large differences in the primary outcomes, a formal quality assessment was not performed. In addition, data pooling and performing a meta-analysis were not considered to be appropriate, because the data were too heterogeneous for statistical pooling. Therefore, a narrative synthesis as indicated in PROSPERO was conducted. A general limitation, especially in the field of genetic association studies, was the high publication bias. There is strong evidence that negative results are less frequently published.<SEP>The “pleiotropic” effect of identified SNPs may show the complexity as well as the challenge of gene-based dietary recommendations. Most of the genetic loci identified in this systematic literature search represent candidate genes (e.g., PPARG) for biological phenotypes. Only a few SNPs investigated for an association with total energy, carbohydrate, or fat intakes have been identified as BMI-related SNPs in GWASs. Tanaka et al. (Supplemental Table 1) identified some loci being associated with macronutrient intake, whereas most of these associations were not confirmed by replication analysis. Another weakness is that prospective intervention studies are needed for confirmation before any conclusion on the clinical relevance can be drawn. In addition, the small effect size of the SNPs identified and potential gene-gene interactions may require studies focusing on genetic scores.",
                  "32452",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "2",
                  "21799482",
                  "INTRODUCTION<SEP>In recent years, the prevalence of central obesity, elevated BMI and diabetes has escalated rapidly throughout India. Although Westernization, including greater availability of calorie dense foods and increasingly sedentary lifestyles, bears much of the blame for these epidemics, some evidence also suggests that Asian-Indians are especially susceptible to metabolic abnormalities. Compared with individuals of European descent (Europeans), Asian-Indians are more predisposed to adiposity of the trunk, including the waist. This propensity toward central adiposity among Asian-Indians relative to Europeans is evident as early as age 10, and applies to both Asian-Indians resident in India and Asian-Indians who have emigrated to Western nations.<SEP>Unfortunately, few data are available that address the issue of genetic susceptibility in Asian-Indians to central adiposity, elevated BMI, or to diabetes risk factors such as elevated levels of blood glucose. Recent genome wide association studies (GWAS) in Europeans have identified a large number of singlenucleotide polymorphisms (SNPs) conclusively linked to elevated BMI, fasting plasma glucose levels, and diabetes. Yet, few studies have attempted to replicate these findings among Asian-Indians, and thus their generalizability to Asian-Indians remains unknown.<SEP>To explore the role of genetic variability in relation to adiposity and fasting plasma glucose in the Asian-Indian population, we investigated 16 SNPs that have been previously shown to be associated with waist circumference, BMI, or fasting plasma glucose among Europeans in a population of 1,129 Asian-Indians enrolled in the India Health Study.",
                  "METHODS AND PROCEDURES<SEP>Study population<SEP>The India Health Study was a multicenter pilot study conducted between December 2006 and July 2008 investigating diet and cancer in India. Participants were recruited from New Delhi and Trivandrum, and reflected a diversity of lifestyles, religions, and income levels. Human ethics committees from the two study centers, as well as the US National Cancer Institute, and the Indian Health Ministry Screening Committee reviewed and approved the study.<SEP>Within the study centers’ coverage area, households were sampled according to religion (Hindu/Muslim/Christian) and urban-rural residence using census data in New Delhi and the voter’s registration list in Trivandrum. Residents from slum and temporary housing were excluded. Within households, eligible individuals were 35–69 years of age, had resided in the study area for at least 1 year, and had no history of cancer, recent cardiac event, or blood disorders. Participants had to be proficient in English or a regional language, capable of informed consent, willing to provide biological samples, and without physical ailments or limitations preventing them from participating in the study.<SEP>From 6,213 households in Delhi and Trivandrum, 2,346 (38%) were successfully contacted at home by an interviewer and 3,845 eligible individuals were identified. Of the eligible individuals, 2,586 (67%) donated blood during a morning follow-up visit after an overnight fast. Up to 15 ml of blood were collected by a trained nurse or phlebotomist in one red top (serum) and two lavender top (plasma) tubes. Plasma, buffy coat, and red blood cells were separated, aliquoted, and stored at −80 °C.<SEP>Genotyping and SNP selection<SEP>For this genotyping study, we selected at random 1,405 participants, with the sample size based on the number needed to determine minor allele frequencies specific to each region and religion with statistical precision. We genotyped 16 SNPs confirmed by GWAS to be associated with adiposity (increased BMI and/or waist circumference) or fasting plasma glucose among Europeans as of August 31, 2008. The selected SNPs are located in or near the FTO, MC4R, G6PC2, GCKR, TCF7L2, and SLC30A8 genes. Because of data specifically implicating MC4R, in particular among Asian-Indians, we included three additional tagSNPs to represent allelic variation between the rs17782313 and rs12970134 SNPs. All SNPs conformed to Hardy– Weinberg equilibrium.<SEP>Genotyping was performed by Bioserve, a biomedical research company based in Laurel, MD, and Hyderabad, India. The genotyping was performed in Hyderabad using Sequenom Mass ARRAY (Sequenom, San Diego, CA). The samples were labeled with randomly generated codes to ensure blinding. Quality control was assessed using duplicate masked specimens for 162 participants. The genotype concordance rate between duplicate samples was 98%. The overall genotyping completion percentage was 99 (range of 0.1–2.0% for individual SNPs). Any sample, where more than two SNPs failed was removed from the dataset (N = 276; 20%). The final number of study participants was 1,129 persons.<SEP>Assessment of phenotype<SEP>Measures of waist circumference, BMI, and fasting plasma glucose were performed during the follow-up visit, which was conducted in the participant’s home or in mobile clinics within the local neighborhood. Waist circumference, height, and weight were measured by trained staff. Participants wore their normal clothing but removed their shoes for all measures. The waist measurement was taken at the midpoint between the lowest rib and the top of the hip bone (illiac crest). The waist measurement was taken on bare skin at the end of a normal expiration and recorded to the nearest 1 mm. Measures were taken in duplicate; if these did not agree to within 1 cm of tolerance, then a third measure was taken and all values were averaged. Height was measured by stadiometer and weight by digital scale. BMI was calculated as weight in kilograms divided by the square of height in meters. Fasting plasma glucose levels were determined with the glucose oxidase/peroxidase method.<SEP>Physical activity assessment<SEP>During the initial eligibility visit, interviewers administered the short form version of the International Physical Activity Questionnaire (IPAQ). This questionnaire asks respondents to report the number of days and the duration of vigorous, moderate, and walking activities done during the past week. Participants were specifically asked to include activities done at work, at home, to get from place to place, and in spare time for recreation, exercise, or sport. Example activities were modified to include culturally relevant items. We estimated energy expended for each activity type in metabolic equivalent (MET) hours per week and summed to obtain total physical activity (PA). The questionnaire has been validated in an international study (Spearman r = 0.33 with a Computer Science and Application, Inc. accelerometer, ref.), although Asian-Indians were not specifically included. In addition to the short IPAQ, participants were also asked about the ownership of labor-saving devices that could be related to PA levels.<SEP>Diet assessment<SEP>Diet history was assessed by interviewer at the initial eligibility visit using an instrument similar to a food frequency questionnaire, that was designed specifically to measure consumption of foods and meals prevalent across different regions of India. This questionnaire comprised 910 predefined foods and food items linked to a nutrient database.<SEP>Statistical analysis<SEP>We analyzed the relationships between genotypes and corresponding GWAS-identified phenotypes (waist circumference, BMI, or fasting plasma glucose) using linear regression, adjusting for age, gender, religion, and study region, assuming an additive genetic model. Overall, 12 SNPs were examined in relation to waist circumference and BMI (24 statistical tests), and four SNPs were examined in relation to fasting plasma glucose (four statistical tests) for a total of 28 statistical tests. In secondary analyses, we examined BMI and waist circumference SNPs in relation to fasting plasma glucose; we also examined fasting plasma glucose SNPs in relation to BMI and waist circumference. Genotypes were coded as ordinal variables (e.g., 0 = no effect allele, 1 = one effect allele, 2 = two effect alleles). Statistical power for each SNP was calculated in R, utilizing the allele frequencies and standard deviations observed in our study population. Assumed effect sizes for BMI were based on Speliotes et al., effect sizes for waist circumference were based on Ruiz et al., and Chambers et al., and effect sizes for fasting plasma glucose were based on Dupuis et al..<SEP>After determining the main effects, we conducted further exploratory analysis of gene–environment interactions for the FTO rs3751812 SNP. Based upon prior reports, we selected three factors for further exploration: (i) PA level, (ii) energy intake, and (iii) fat content of diet. Gene and environment interactions were modeled using the cross-product of genotypes and exposure level on a continuous scale (marginal effects were also included in these models).<SEP>We followed procedures in the IPAQ scoring guidance to normalize PA data. We deleted observations, where the sum of time spent in vigorous and moderate intensity activities and walking exceeded 960 mins (N = 6) and truncated values for vigorous intensity activity (N = 36) and walking (N = 50) that exceeded 3 h/day (e.g., 3.5 h = 3.0 h). We did not truncate moderate intensity activity at 3 h/day but instead at 8 h/ day (N = 70); truncating at 3 h would have affected data for fully half of Trivandrum residents. A higher cutpoint may be more appropriate for a developing nation, where labor-related moderate intensity PA may be highly prevalent. Log transformation of PA had little impact upon study findings; we present models without transformation.<SEP>Correction for multiple statistical tests<SEP>To limit the false positive discovery rate, we used a Bonferroni correction to set the α level (false positive level) for statistical significance. Given our 28 statistical tests with an established prior, we set the twosided α level at α = 0.05/28 = 0.0018. This threshold is conservative given (i) strong priors, (ii) linkage disequilibrium between SNPs, and (iii) correlation between phenotypes. Regarding the latter, the Pearson correlation between waist circumference and BMI was 0.72 among women and 0.69 among men, suggesting overlap in hypotheses tested.",
                  "RESULTS<SEP>Characteristics of the study participants are presented in Table 1. New Delhi participants were predominantly Hindu or Sikh, had typically graduated from secondary school or university, and earned a household monthly income of 10,000 or greater Indian rupees. They had low PA levels and many labor-saving conveniences, e.g., 79% had washing machines. Trivandrum participants included a plurality of Hindu, Muslims, and Christians and were less educated and had low incomes compared with New Delhi participants. Their PA level was high—three times that of New Delhi participants—and ownership of labor-saving devices was uncommon, e.g., 14% had washing machines.<SEP>The adiposity-related SNPs in our analysis and their relation to waist circumference and BMI are indicated in Table 2. One SNP in the FTO gene (rs3751812) was associated with waist circumference after correction for multiple testing (Padj = 0.04). The T allele of the FTO rs3751812 was associated with per allele increase in waist size of 1.58 cm (95% confidence interval (CI) = 0.60, 2.56; Ptrend = 0.0015). The A allele at rs1121980 had a similar effect that was not statistically significant at the multiple testing corrected threshold (1.32 cm; 95% CI = 0.38, 2.27; Ptrend = 0.0061). The two SNPs were highly correlated (r = 0.81, r2 = 0.66) in our study population; we advanced rs3751812 alone for subsequent analyses.<SEP>No other adiposity-related SNPs were significantly associated with outcome measures after correction for multiple testing. The T allele at the rs3751812 locus had a nominally significant (P < 0.05) association with increased BMI (0.55 kg/m2 increase per allele; 95% CI = 0.14, 0.96; Ptrend = 0.008). In exploratory analyses, there was no rs3751812 T allele association with waist–hip ratio (Ptrend = 0.92), waist–thigh ratio (Ptrend = 0.89), or with PA level (Ptrend = 0.49). An additional nominally significant finding was that the A allele at the MC4R locus rs17700633 was associated with increased waist size (1.07 cm; 95% CI = 0.10, 2.04; Ptrend = 0.03). For the adiposity-related SNPs overall, the direction of effect was consistent with that of prior reports for 11 out of 12 SNPs.<SEP>There was no association observed between the fasting plasma glucose SNPs and fasting plasma glucose, including at the nominal level of statistical significance (Table 3). Restriction of analyses to nondiabetics yielded nearly identical results.<SEP>On an exploratory basis, we examined whether the association of rs3751812 with waist circumference varied by PA level, and found a nominally statistically significant interaction (Pinteraction = 0.008, Table 4). Among participants with a PA level of <81 MET-h/wk, the T allele was associated with increased waist circumference (+2.68 cm; 95% CI = 1.24, 4.12). In contrast, among participants with a PA level of 212+ MET-h/wk, the T allele had no association with increased waist circumference (−1.79 cm; 95% CI = −4.17, 0.58). This interaction is primarily driven by the results among participants of Trivandrum (Pinteraction = 0.008), where there was a large range in PA levels. In New Delhi, there was no statistically significant interaction (Pinteraction = 0.63), but data was sparse at the high PA levels, which limits statistical power to detect interactions.<SEP>The overall association of the FTO variant with waist circumference seemed to vary by study region (Pinteraction = 0.10), with a more robust association in New Delhi than in Trivandrum. However, this effect modification appears to be an artifact of differing PA levels by region. When the FTO–PA interaction is added to the model, the FTO–region interaction is eliminated (Pinteraction = 0.50), suggesting that regional heterogeneity is a consequence of PA-related heterogeneity.<SEP>There was no evidence for effect modification by energy intake or by fat content of the diet (all Pinteraction > 0.05). We also examined whether PA modified the association between rs3751812 and BMI (data not shown). The direction of results was consistent with the waist circumference findings but the interaction was not statistically significant (Pinteraction = 0.14).",
                  "DISCUSSION<SEP>In our study, we found that variants in the FTO gene were associated with waist circumference and also nominally with BMI among Asian-Indians. The effect sizes were comparable to those previously found among Europeans. These results contradict prior studies among Asian-Indians, that had found null or weak associations between FTO and waist circumference and/or BMI. In addition, we found evidence of an FTO and PA interaction, so that the FTO effect was no longer statistically significant at high PA levels. This result underscores the importance of PA for the prevention of central adiposity, including among those with an inherited susceptibility to central adiposity. Also, as shown in our data, PA levels vary by region of India, perhaps due to variability in modernization, and the FTO–PA interaction could possibly explain discrepancies in existing literature for Asian-Indians.<SEP>In recent years, GWAS studies have identified at least 32 common genetic variants associated with BMI and/or waist circumference among Europeans, the most important of which are variants in the FTO gene. The FTO SNPs were the first to be linked to adiposity by GWAS, they explain the most variance in adiposity of any variants yet identified, and the associations have replicated in many major ethnic groups, including Africans, Chinese, and the Japanese. However, the three studies to date among Asian-Indians have yielded equivocal evidence for an FTO–adiposity association, with two reporting null findings and the third reporting a weak effect of 0.2 kg/m2, as compared to the 0.4–0.6 kg/m2 effect size among Europeans. These data do not strongly support a lack of association, however, as one null study had a small sample size and the other, a GWAS, had limited power because of strict control for multiple comparisons. The third study, the only one with at least 50% statistical power, did detect a weak association. Our results, when taken together with the other positive study, appear to favor an association between FTO variants and adiposity. Whether effect sizes are as large as in Europeans, which is what we found, or more modest requires further elaboration in larger study populations.<SEP>Similar to our findings, studies among Europeans have reported that the association between FTO gene variants, especially the rs9939609 locus, and adiposity varies by PA levels. The rs9939609 SNP is in high linkage disequilibrium with the rs3751812 locus, that we examined (pairwise r2 = 0.97 in Asian-Indians in Hapmap3, ref.). These prior studies assessed BMI as a marker of adiposity; two also examined waist circumference and found similar interactions. Three other studies found no FTO–PA interaction, although modest sample sizes or shortcomings in PA questionnaire design could explain these null results, as discussed by Ruiz et al.. Several studies have also examined, whether randomized assignment to PA results in differential weight loss according to FTO genotype, but the trials have been small and results have been conflicting.<SEP>Although our study was small, we observed a nominally statistically significant FTO–PA interaction. The unusually high PA levels of our Trivandrum participants, equal to three times that of persons from developed nations in a prior IPAQ-based study, may have amplified our statistical power to detect gene–environment interactions. A similar example—one of the first studies to find an FTO–PA interaction—is a study done among the Amish, an agrarian population known for eschewing modern labor-saving devices and for exceptionally high PA levels.<SEP>The FTO gene encodes a protein that is highly expressed in the hypothalamus and other regions of the brain, and is involved in the regulation of food intake and reward-driven behavior. Human studies suggest that FTO gene variants may increase adiposity by influencing eating behavior, particularly increased consumption of energy dense foods. The FTO effect does not appear to be mediated through reductions in PA level; on the contrary, several studies suggest that the FTO gene variants are associated with modest increases in PA level. We would anticipate the FTO variants would act through similar mechanisms among Asian-Indians, although we were unable to test this directly.<SEP>In addition to the FTO gene, we examined several other genetic variants in relation to waist circumference, BMI, and fasting plasma glucose. We found no statistically significant associations for these other loci after adjustment for multiple comparisons. This was not surprising because our statistical power was greatest for the analysis of FTO SNPs in relation to adiposity, i.e., ~50% at the nominal level of statistical significance and 12% at the multiple comparisons threshold, assuming an effect size similar to that of Europeans (observed effects were larger). For other SNPs and for analyses of fasting plasma glucose, our study had less than 40% statistical power to detect associations, even at the nominal level of statistical significance. For the intragenic adiposity SNPs (rs748192 and rs10498767) and the GCKR SNP, low statistical power is at least partly the result of lower minor allele frequencies in Asian-Indians than in Europeans. Failure to replicate prior results from Europeans may reflect low statistical power and not a lack of true associations.<SEP>One exception may be the G6PC2 variant. We had 76% statistical power at the nominal level of statistical significance to detect an association between this variant and fasting glucose levels but found no association. This is intriguing as the G6PC2 and fasting plasma glucose association has been previously replicated among Asian-Indians residing in London. Although speculative, the different results between our study and the London study suggest a possible gene–environment interaction. Alternately, our null finding could be due to chance.<SEP>Strengths of our study include the measurement of waist circumference, weight, and height by trained staff rather than by self-report, and a wide range of PA levels. One limitation is our cross-sectional study design. To evaluate whether PA can truly eliminate gains in adiposity associated with common FTO gene variants, participants should be followed prospectively, preferably within a large PA trial. Also, our PA assessment was based upon self-report and thus includes imprecision and error. Finally, our study lacked statistical power needed to rule out modest effect sizes for these genetic variants.<SEP>In conclusion, this study confirmed that an FTO gene variant previously linked to central adiposity in European populations is also associated with central adiposity among Asian-Indians. Our results also suggest that the FTO-related genetic susceptibility to central adiposity could be attenuated by increased PA. That we were able to identify such an effect owes much, we hypothesize, to the unusually high PA levels of our participants in Trivandrum, where labor-saving devices are uncommon. This highlights a sometimes overlooked issue in current genetic research—that populations of developed nations tend to lead relatively homogenous lifestyles, thus limiting the ability to detect gene–environment interactions with precision. Further study of diverse populations in developing countries may be an important strategy to further our understanding of gene–environment interactions.<SEP>The author’s contributions were as follows—S.C.M.: statistical analysis, interpretation of results, writing of the manuscript; N.D. and P.S.G.: study implementation; L.R.: laboratory analysis; M.J.G., C.R.D., N.C., S.J.C., and S.I.B.: interpretation of results; S.Y.: data management and study coordination; R.S., D.P., A.M., N.C., K.S.R.: study design and concept; and all authors: critical review and approval of the final manuscript.<SEP>           Disclosure         <SEP>The authors declared no conflict of interest.",
                  "22444",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "3",
                  "24516669",
                  "Introduction<SEP>Osteoporosis and obesity are both multifactorial disorders that in recent years have become major public health problems. At one time considered to be mutually exclusive, it is now recognized that these conditions share many genetic and environmental risk factors and are linked to each other through a number of complex regulatory pathways. Epidemiological studies have shown that increased body weight is positively associated with bone mass, while low body weight is a risk factor for bone loss and osteoporosis. The positive effect of body weight on bone mass may be attributable to a number of factors: increased mechanical load which has an anabolic effect on bone; conversion of steroid precursors to estrogen in peripheral adipose tissue or through the secretion of bone active hormones from β-cells in the pancreas and adipocytes themselves. Homocysteine (Hcy), vitamin B12, vitamin D and folate are biomarkers for a number of pathologies including cardiovascular disease and diabetes and a strong correlation between these biomarkers with BMI has been reported. Elevated levels of Hcy and low levels of vitamin D are strong and independent risk factors for osteoporotic fracture risk.<SEP>Studies have shown that variation in diet and life style modulate Hcy, folate and vitamin B12, all of which are important components in intermediary metabolism. Elevated Hcy levels have been associated with detrimental effects on bone metabolism, however, whether Hcy and other biochemical parameters play a causal role or act as markers for other mechanisms underlying these pathologies is unclear.<SEP>The complex relationship between fat cells and bone has been under intense scrutiny. Adipocytes and osteoblasts share a common progenitor, the pluripotent mesenchymal stem cell, and there is a degree of plasticity between the two cell types. Differentiation to a particular lineage is regulated by numerous transcription factors and with increasing age there is a shift away from osteoblast towards adipocyte production which in conjunction with increased osteoclast function may lead to osteoporosis.<SEP>At the genetic level a number of association studies have identified single nucleotide polymorphisms (SNPs) close to genes contributing to both osteoporosis and body composition. To date, few bivariate genome-wide association studies (GWAS) for osteoporosis and obesity have been performed although GWAS for obesity and its associated pathological outcomes have identified a number of SNPs close to genes expected to also play an important role in bone metabolism. In the current study, we selected five SNPs identified through GWAS: rs17782313_MC4R; rs1770633_MC4R; rs7566605_INSIG2; rs9939609_FTO and rs1121980_FTO.<SEP>The FTO gene is highly expressed in the hypothalamus and is involved in energy homeostasis through the control of energy expenditure. Located on chromosome 16, the FTO gene has nine exons and spans more than 400 kb. The SNPs associated with BMI in the GWAS lie in the first intron that harbors a region highly conserved across different species. Individuals with the rs9939609 variant allele were shown to have a 31% increased risk per variant allele, of developing obesity. Variation in the FTO gene has been analyzed for association with BMD in a number of populations including children and adults and in a mouse knockout model, BMD was lower in the FTO knockout compared to controls.<SEP>The MC4R gene on chromosome 18 encodes the MC4 protein, a G-protein coupled receptor that plays a major role in the central regulation of body weight through maintaining energy homeostasis and the suppression of food intake. Genetic variation in the MC4R gene has been identified as responsible for monogenic forms of obesity. In a GWAS by Loos et al., SNPs present in the intergenic region upstream of MC4R were associated with BMI. Patients deficient in MC4R have been reported to have increased BMD and decreased bone resorption, while genetic variation has been evaluated for association with bone mass, but only in children.<SEP>The INSIG2 gene has been reported to be associated with increased risk of obesity. SNP rs7566605, located 10 kb upstream of the INSIG2 gene transcription start site, was associated with fat mass. INSIG2 is a candidate gene for increased BMI; it binds to the sterol regulatory element-binding protein complex (SREBP) and reduces the activity of cholesterol and fatty acid synthesis in the endoplasmic reticulum. Although variation in the INSIG2 gene has recently identified in a GWAS for BMD, it has not been yet fully explored.<SEP>The rationale for our study was to comprehensively evaluate the association between selected obesity associated polymorphisms and aspects of bone strength, a complex trait not captured by BMD alone. Since the skeletal fragility associated with osteoporosis reflects reduced bone quality as well as quantity, we have assessed micro-architectural properties of bone, bone geometry and long-term fracture risk in addition to BMD and body composition. Furthermore, in order to understand the mechanisms underlying these associations, we have also evaluated the association between these polymorphisms and biomarkers for obesity and bone mass. By studying two differently aged cohorts we have evaluated age-related differences in the contribution of obesity associated polymorphisms with bone phenotypes.",
                  "Materials and Methods<SEP>Subjects<SEP>Two population based cohorts of Swedish women were studied; the OPRA cohort consisting of 1044 elderly women all aged exactly 75 and prospectively followed for 10 years and the PEAK-25 cohort consisting of 1061 women all aged exactly 25. Details of the two cohorts have been published. Participants gave written informed consent and the study was approved by the Regional Ethical Review Board in Lund according to the Helsinki agreement.<SEP>Measurement of Bone Phenotypes and Body Composition Using DXA<SEP>Bone mineral density (g/cm2) at the femoral neck (FN), lumbar spine (LS) and total body (TB) was measured using dual-energy x-ray absorptiometry (Lunar Prodigy: PEAK-25; Lunar DPX-L: OPRA (Lunar Corporation, Madison, WI, USA)). Fat and lean mass for total body (TB) and trunk were also measured using the same instrument. Calibrations were performed daily using a phantom supplied by the manufacturer. Precision error (coefficient of variation) for DXA scanning was 0.94%, 1.45%, 4.01% for TB, LS and FN respectively in the OPRA cohort and 0.90% and 0.65% for FN and LS respectively in PEAK-25. For the OPRA cohort, all measurements at baseline were performed using the same instrument, while analyses of scans were made with software versions 1.33 and 1.35.<SEP>Hip geometry was assessed only in the OPRA cohort by employing the software provided by Lunar® (Lunar Corporation, WI, USA) for the DPX-L scanner. The following phenotypes were analyzed: Hip Axis Length (mm); femoral neck width (mm); Cross Sectional Moment of Inertia [CSMI] (Cm4) and Section Modulus [SM] (Cm3). To minimize variability, all variables were analyzed by a single operator. The coefficient of variation for these measurements was between 0.6 and 3.7%.<SEP>Ultrasound measurements (Speed of Sound (SOS) (m/s)), Broadband Ultrasound Attenuation ((BUA) (dB/MHz)) and Stiffness Index (SI)) were performed on the right calcaneus using the Lunar Achilles (R) system (Lunar Corporation Madison, WI, USA) to assess bone quality in both cohorts. The precision was 1.5% for derivatives of BUA and SOS. Daily calibrations were made to control the long-term stability of the apparatus.<SEP>Fracture Ascertainment<SEP>In the PEAK-25 cohort the fracture incidence is low, therefore fracture data was analyzed only in the OPRA cohort. Self-reported fractures sustained between age 20 and 75 were recorded and verified from the radiological files. The majority of fractures (>99%) occurring in the elderly women were attributable to low energy trauma.<SEP>Blood Sample Collection and Biochemical Phenotypes Measurements<SEP>Non-fasting blood samples were collected before noon for DNA isolation (PEAK-25 and OPRA) and to assay serum concentration of biochemical markers (OPRA). Samples were stored at −80°C until analysis. Assays were performed at the Department of Clinical Chemistry, Malmö, Skåne University Hospital according to accredited methods. Biochemical phenotypes, available only in the OPRA cohort, were assayed using standardized analytical protocols. Total serum Hcy (µmol/L) was measured using HPLC. Serum vitamin B12 (pmol/L) and folate (nmol/L) were measured using Elecsys assays (Roche Diagnostics, Mannheim, Germany) and serum 25-hydroxy vitamin D (25OHD) (nmol/L) was assessed by liquid chromatography mass spectrophotometry (LC-MS).<SEP>Genotyping and Statistical Analysis<SEP>Five obesity associated SNPs from three genes were genotyped in both cohorts (Table S1). Sequenom’s iPlex Gold system (Sequenom, San Diego, CA) was employed to score the genotypes. A total of 993 women from the OPRA and 1001 women from the PEAK-25 cohort were genotyped successfully. Approximately 3% of the samples from each cohort were genotyped in duplicate with 100% concordance. Departures from Hardy-Weinberg equilibrium were tested using the χ2 test with one degree of freedom (HWE Program, Jurg Ott and Rockefeller University, New York). Linkage disequilibrium (LD) between SNPs from the same gene was tested using Haploview (http://www.broad.mit.edu/mpg/haploview/). Statistical analysis was performed using SPSS (version 20.0, SPSS Inc., Chicago, IL).<SEP>Using a co-dominant model (comparing the three genotypes, under the assumption that neither of the alleles is dominant), genotype specific differences between the phenotypes were analyzed with the Kruskal-Wallis test and to determine association adjusting for confounding factors (height, and smoking) regression analysis was performed. Gene interaction with Hcy was analyzed comparing the lowest and highest quartiles of serum Hcy levels, where quartile 1 was considered ‘Normal’ (<11.6 µmol/L) and quartile 4 ‘High’ (>17.5 µmol/L). The χ2 test was used to analyze association between genotypes and categorical variables. Multiple statistical tests were performed, however since most of the phenotypes are dependent, we report uncorrected p-values (two-tailed) and associations were considered nominally significant at the level p<0.05.",
                  "Results<SEP>The general and clinical characteristics of the women from the two differently aged cohorts are reported in  Table 1 . Genotype and allele frequencies did not differ between cohorts ( Table 2 ). All SNPs conformed to HWE (p>0.05). Both SNPs from the FTO gene were in strong LD (OPRA, D′ = 0.98, r2 = 0.84; PEAK-25, D′ = 0.99, r2 = 0.87) therefore only rs9939609 was used for further analysis. No LD was observed for the MC4R SNPs (D′ = 0.45, r2 = 0.13).<SEP>The PEAK-25 participants had lower BMI and fat mass and higher lean mass compared to the elderly individuals from OPRA ( Table 1 ). As previously reported, fat and lean mass were strongly positively associated with BMD, with lean mass making a greater contribution than fat mass to BMD in young women (data not shown). For QUS phenotypes, the positive association with fat and lean mass was very similar in the elderly women, while in the young women the contribution from lean mass was stronger ( Table 3  ).<SEP>Association between Obesity Associated Polymorphisms and Body Composition<SEP>SNP rs9939609_FTO was associated with a number of body composition measurements including weight, BMI, and fat mass in PEAK-25 ( Table 4 ). Individuals carrying the minor allele had higher BMI, fat mass (TB and trunk) but no association was found with lean mass ( Table 4 ). The association with rs9939609_FTO remained after adjusting for smoking status and height (p = 0.001 to p<0.0001) ( Table 4 ). We observed trends for BMI, TB fat mass and percentage trunk-fat in the same direction in the OPRA cohort, but these did not reach statistical significance. After adjustment for height and smoking status an association with percentage of trunk-fat was observed with rs9939609_FTO (p = 0.007). No association between MC4R or INSIG2 polymorphisms and body composition were observed in the OPRA and PEAK-25 cohorts (data not shown).<SEP>Obesity Associated Polymorphisms and Association with BMD, QUS and Geometry<SEP>SNP rs1121980 from the FTO gene was excluded from analysis. The remaining four SNPs were analyzed for association with bone density, but no significant genotype related differences in BMD at any skeletal site were observed in either cohort (data not shown).<SEP>Polymorphisms were also analyzed for association with bone quantitative ultrasound in both cohorts. The rs17782313_MC4R showed association with BUA, SOS and SI in OPRA (p = 0.007–0.001) ( Table 5  ) and individuals carrying the minor C-allele had higher values compared to homozygotes for the common allele. The association remained even after adjustment for height and smoking (p = 0.02–0.0004) ( Table 5 ) and additional adjustment for weight (p = 0.015 to 0.007). Similar trends were observed in the PEAK-25 cohort but were not statistically significant. Polymorphisms close to FTO and INSIG2 were not associated with ultrasound phenotypes in either cohort.<SEP>Femoral neck geometry is an important component of hip fracture risk and we evaluated SNP-phenotype associations in the OPRA cohort. The rs7566605_INSIG2 showed association with FN width (p = 0.03), with individuals carrying the minor allele having a lower mean value compared to subjects homozygous for the major allele (34.1 vs. 34.5 mm), but this did not withstand adjustment for height and weight. Polymorphisms from FTO and MC4R did not show any association with bone geometry (data not shown).<SEP>Obesity Associated Polymorphisms and Association with Biomarkers and Fracture<SEP>The association between obesity associated polymorphisms, biochemical risk factors and fracture was evaluated in the OPRA cohort. Carriers of rs17782313_MC4R C-allele had high Hcy (p = 0.06), low serum folate (p = 0.03) and low vitamin D (p = 0.03) ( Table 5 ). After adjustment for smoking and height, the association remained for folate (p = 0.01) and vitamin D (p = 0.02).<SEP>We wanted to determine whether the association between MC4R SNPs and QUS was influenced in relation to normal (<11.6 µmol/L) and high (>17.5 µmol/L) levels of Hcy. Only in the high Hcy group was rs17782313_MC4R associated with QUS (p = 0.03 to p = 0.005) and this association remained even after adjustment for height and smoking (p = 0.007 to p = 0.001) ( Table 6 ) and additionally adjusted for weight the association remained significant (p = 0.01 to 0.002). Interestingly, in the high Hcy group, vitamin D levels decreased with number of C-alleles, in direct contrast with the observation in the normal Hcy group ( Table 6 ). As expected, proportionally fewer women fractured prior to baseline in the lowest (BMI<23.4; 63.9%) compared to the highest BMI quartile (BMI >28.7; 52.3%); p = 0.009).<SEP>Variation in FTO and INSIG2 did not appear to make an important contribution to fracture risk, even when smoking, TB-BMD and any one of body weight, BMI or fat mass were included in the regression model. Women carrying the MC4R_rs17782313 C-allele showed a non-significant trend towards fewer fractures although there was no allele dose effect (52.1% vs. 58.5%). As expected, compared to women without a baseline fracture, QUS values were lower in those with a fracture regardless of genotype (data not shown), however, within the fracture category women with the rs17782313_MC4R C-allele had higher QUS values ((BUA: 97 vs. 100 vs. 103; p = 0.04); (SOS: 1516 vs. 1517 vs. 1517; p = 0.18); (SI: 67 vs. 70 vs. 72; p = 0.021)) consistent with a lower fracture incidence.",
                  "Discussion<SEP>Obesity is an established risk factor for a number of complex disorders including cardiovascular complications, diabetes mellitus and hypertension, but it has been suggested to be protective against osteoporosis. A complex, differential influence from lean and fat mass on bone strength is suggested and the current study supports the supposition that lean mass makes a larger contribution to BMD during young age while fat mass plays a major role for BMD in later stages of life.<SEP> FTO has been well described in relation to body composition and obesity phenotypes. In our study, we observed higher BMI and fat mass in relation to the rs9939609_FTO C-allele; however it is interesting that the association was only seen in the young women. This is consistent with suggestions that the effect size of obesity susceptibility genes varies with age. Although the underlying mechanisms are unclear, data from a mouse model has shown that mRNA expression of FTO is regulated by nutritional intake and expression levels vary according to feeding and fasting behavior and we might speculate that food intake patterns differ between young and elderly individuals. We did not find any association with BMD or other bone phenotypes in either the young or elderly cohort of women. An age-specific effect has been reported in at least one study alongside suggestions that FTO could be a genetic marker for peak bone mass due to the potential role of FTO in postnatal growth. Our results do not support this however since the women in the PEAK-25 cohort are at an age where peak bone mass is assumed to have been reached. Nonetheless this does not rule out the possibility that FTO variants could be associated with skeletal growth trajectory in childhood and adolescence. Although we found no direct association between variations in the vicinity of FTO and BMD or QUS parameters, it is likely that any effect of the gene on bone is indirect, through BMI and fat mass.<SEP> MC4R is crucial in the regulation of body weight and monogenic forms of obesity commonly result from mutations in its gene. Although we observed a trend for higher BMI and fat mass in both cohorts with the C-allele, the association with MC4R did not reach significance, which contrasts with the findings reported in GWAS and other association studies. In the current study, we have shown for the first time that variation in the MC4R gene is associated with QUS phenotypes. A trend towards better bone quality with carriage of the variant MC4R rs17782313 C-allele was observed in the young women, but was more pronounced in the elderly women. Furthermore this association appeared to be mediated by both direct and indirect mechanisms which may explain in part the age specific effect observed, since a higher BMI, as displayed by the older women, is positively associated with bone strength. Although a genetic association between MC4R gene polymorphism and bone mass has been reported, albeit in children, we found no association with BMD or bone structural traits (femoral neck geometry) in our study.<SEP>One of the novel findings of our study is that MC4R is associated with altered vitamin D, folate and Hcy levels, which are associated with obesity. Vitamin D deficiency associated with obesity has been shown at all ages and independent of sex in a recent meta-analysis. The results from our study indicates that a gene environment interaction has the potential to improve bone quality through increased fat mass in elderly women, demonstrated by the fact that the strongest association between MC4R and QUS was in the elevated Hcy group. This finding is in keeping with what is known about MC4R expression, i.e. that it is altered in response to environmental stimuli through hypothalamic neuronal networks, and recent studies suggest this has an important role in bone homeostasis.<SEP>Although in a meta-analysis of GWAS variation in INSIG2 was associated with femoral neck BMD, in our cohorts INSIG2 was not associated with BMD, body composition or bone quality, although this is unsurprising since we have analyzed a BMI associated SNP which is not in LD (r2 = 0.01; D′ = 0.39) with the SNP identified in the BMD GWAS. In our study, although width at the femoral neck was narrower in elderly women with the variant allele, the association was attenuated after adjustment for height and weight and furthermore indices of bone strength and hip fracture rates were not different. To date, none of the GWAS for bone geometry have shown evidence of association within or near the INSIG2 gene.<SEP>The strengths of this study include the extensive data collected on body composition, bone related phenotypes and biochemical risk factors for obesity and osteoporosis. By including two differently aged cohorts we have the possibility to distinguish age related effects of genetic variation on these phenotypes. The cohorts studied are well-characterized, large, of identical age within each cohort and the majority of women were of Swedish origin. Whether the findings are applicable to other ethnic groups requires replication in other populations. A limitation of the study is that biomarker data was not available in the PEAK-25 cohort, which would have enabled us to identify if there are age related effects associated with the homocysteine-MC4R-obesity relationship.<SEP>In summary, our data provides novel evidence that variation in the obesity associated gene MC4R is associated with improved quantitative ultrasound phenotypes, an important component of bone strength.",
                  "21427",
                  "['OBWCCE']"
                ],
                [
                  "4",
                  "19079261",
                  "Obesity is a major public health problem, resulting in increased morbidity and mortality and severe economic burdens on health-care systems. Excessive energy intake and diminished physical activity contribute to the increasing prevalence of obesity, but genetic factors strongly modulate the impact of the modern environment on each individual. Indeed, family and twin studies have shown that genetic factors account for 40–70% of the population variation in BMI. BMI is the most commonly used quantitative measure of adiposity, and adults with high values of BMI (>30 kg/m2) are termed obese.<SEP>Until recently, genetic variants known to influence BMI were largely restricted to mutations in several genes that cause rare, often severe monogenic syndromes with obesity as the main feature. Mutations in these genes are thought to act through the CNS, and in particular the hypothalamus, to influence energy balance and appetite, thereby leading to obesity. However, it is not known whether genetic variation in similar pathways is also relevant to the common form of obesity and population variation in BMI.<SEP>In the past year, large-scale searches for genetic determinants of BMI revealed previously unreported associations with common variants at two loci, FTO and MC4R. Common variants at these loci are associated with modest effects on BMI (0.2–0.4 kg/m2 per allele) that translate into odds ratios of 1.1–1.3 for obesity (defined as BMI ≥ 30 kg/m2). Common variation in PCSK1 has been strongly associated with the risk of extreme obesity, but this association has not yet been independently replicated.<SEP>Together, common variants at FTO and MC4R and rare variants known to cause obesity explain only a small fraction of the inherited contribution to population variation in BMI. To expedite the identification of alleles associated with variation in BMI, obesity and other anthropometric traits, we formed the GIANT (Genetic Investigation of ANthropometric Traits) consortium to facilitate large-scale meta-analysis of data from multiple genome-wide association studies (GWAS). Here, we report a meta-analysis of 15 GWAS totaling 32,387 individuals and test for association between BMI and ~2.4 million genotyped or imputed SNPs. We then follow up 35 SNPs drawn from the most significantly associated loci by a combination of de novo genotyping in up to 45,018 additional individuals and analysis of these SNPs in another 14,064 individuals already genotyped as part of other GWAS. These studies show that variants at six previously unreported loci in or near TMEM18, KCTD15, SH2B1, MTCH2, GNPDA2 and NEGR1 are reproducibly associated with BMI.",
                  "METHODS<SEP>Study design<SEP>This study is comprised of two stages. Stage 1 is a meta-analysis of GWA studies comprised of 32,387 individuals of European ancestry. This meta-analysis allowed us to select 35 loci for detailed examination in stage 2, which included direct genotyping in 45,018 European-origin individuals from nine studies and in silico comparisons with results from 14,064 European-origin individuals from five studies with GWA data (Supplementary Fig. 1 and Supplementary Table 1).<SEP>Stage 1 samples and genotyping<SEP>The GIANT consortium currently encompasses 15 study cohorts with 32,387 individuals of European ancestry informative for adult BMI (Supplementary Fig. 1 and Supplementary Table 1). The 15 study cohorts, including between 1,094 and 5,433 individuals each, were genotyped using the Affymetrix 500K Mapping Array Set (11 cohorts, n = 25,394), Illumina HumanHap300 BeadChip (2 cohorts, n = 2,385), Illumina HumanHap300+240 (1 cohort, n = 2,235) or Illumina HumanHap 550 BeadChip (1 cohort, n = 2,265) (Supplementary Tables 2 and 3). To allow for meta-analysis across different marker sets and to improve coverage of the genome, we performed imputation of polymorphic HapMap CEU SNPs (Supplementary Note and Supplementary Table 3) using either MACH (Y. Li, C.J.W., J. Ding, P.S. and G.R.A., unpublished data) or IMPUTE.<SEP>Association analysis with BMI and meta-analysis of association results<SEP>First, each study performed GWA analyses for BMI assuming an additive model implemented in either MACH2QTL (Y. Li, C.J.W., J. Ding, P.S. and G.R.A., unpublished data), Merlin or SNPTEST. Covariates, trait transformation and strategies for excluding outliers or accounting for family relatedness varied according to each study's original design (Supplementary Tables 2 and 3), but the main results were essentially unchanged when we repeated meta-analysis after imposing a uniform set of analyses and procedures across the 15 study cohorts. For those samples based around case-control designs (such as those from FUSION and from the type 2 diabetes, coronary artery disease, and hypertension components of the Wellcome Trust Case Control Consortium), cases were analyzed separately from controls. To allow for relatedness in the SardiNIA and FUSION samples, regression coefficients were estimated in the context of a variance component model that modeled background polygenic effects.<SEP>Next, we carried out meta-analysis using a weighted z-score method, which accounts for the direction of association relative to a consistent reference allele. In this method, P values for each study are first converted to z scores. Then, a weighted sum of z scores is calculated where each statistic is weighted by the square root of the sample size for each study. The resulting sum is divided by the square root of the total sample size to obtain an overall z statistic, which can be used to evaluate the overall evidence for association. The method takes direction of effect across studies into account by reversing the sign of the z score for a study if the effect is in the opposite direction. We obtained similar results when we analyzed each cohort using a uniform protocol (which involved a quantile transformation to approximate normality and adjusting for age and age in men and women separately) and combined the results using the regression coefficients and standard errors estimated from each study (Supplementary Fig. 2; Pearson correlation r = 0.91). Both meta-analysis procedures were implemented in the freely available METAL software package. The genomic control parameter λ was 1.10 in our initial meta-analysis without using genomic control correction in any study except SardiNIA, which, given our large sample size, suggests only a modest impact of unmodeled relatedness or population stratification in our results. The P values we report have all subsequently been corrected for this unmodeled relatedness or population stratification by application of a genomic control correction to all input studies as well as to the meta-analysis results.<SEP>Selection of SNPs for follow-up<SEP>For follow-up analyses (stage 2), we genotyped 35 SNPs drawn from the most significantly associated independent loci. We defined signals at two SNPs to be independent of each other if the SNPs were in low LD (r2 < 0.3) or if they were >1 Mb apart. In some cases, the SNP with the strongest signal of association at a locus could not be genotyped for technical reasons, and we substituted another SNP that was strongly correlated with the original SNP in the HapMap CEU sample (Supplementary Note). Because SNP selection was based on an earlier version of the meta-analysis and because some SNPs failed primer design, not all of the top signals were represented among the 35 SNPs. Among the SNPs that were followed up, the highest stage 1 P value was 6.9 × 10−4.<SEP>Stage 2 samples and genotyping<SEP>We genotyped 35 SNPs in a total of 45,018 individuals of European ancestry from nine study cohorts using Sequenom iPLEX or TaqMan (Supplementary Note). Individuals were eliminated from analysis if <80% of SNPs were called successfully. Among successfully typed individuals, genotype frequencies were in Hardy-Weinberg equilibrium (P > 10−6), call rates were >94%, and concordance of duplicate genotypes was >99% in each of the follow-up study cohorts.<SEP>For in silico replication, we also obtained association results for 35 SNPs from 14,064 individuals of European ancestry from five studies (Supplementary Table 1). The five study cohorts, each including 856 to 5,373 individuals, were genotyped using the Illumina HumanHap 550, 300 or Illumina Human CNV370 DUO (Supplementary Tables 2 and 3). To allow for meta-analysis across different marker sets and to improve coverage of the genome, we carried out imputation of polymorphic (minor allele frequency >1%) autosomal HapMap SNPs (Supplementary Note and Supplementary Table 3) using either MACH or IMPUTE with the HapMap CEU sample as a reference panel. We accounted for uncertainty in each genotype prediction in the analysis of imputed genotype data by using either the dosage information from MACH or the genotype probabilities from IMPUTE. Stage 1 and 2 results for FTO and MC4R are not presented directly in the main text but are shown for comparison in Supplementary Table 5 and Supplementary Figure 5 online.<SEP>Association analyses with BMI and secondary analyses<SEP>Association with BMI was tested as in stage 1, assuming an additive model. Logistic regression analysis was used to test for association with the risk of being overweight (defined as BMI ≥ 25 kg/m2) or obese (BMI ≥ 30 kg/m2), with adjustment for age, age, and sex, testing for SNP effects in an additive genetic model. Evidence for association between our replicating SNPs and type 2 diabetes, lipidlevels and coronary artery disease was extracted from publicly available datasets. The effect of the replicating SNPs on expression of nearby genes was determined from publicly available eQTL GWA studies from lymphocytes and brain tissue.<SEP>Gene expression studies<SEP>Adult human RNA samples were obtained from Clontech either as poly(A) purified RNA (hypothalamus and adipocyte) or as total RNA (cerebellum, cortex, spleen, pancreas, lung, kidney, liver, testes and total brain). The total RNAs were purified to poly-A RNA using the Micro-Poly(A)Purist kit (Ambion) according to manufacturer's instructions. We used 20 ng of poly(A) RNA in a random-primed first-strand cDNA synthesis using SuperScript II (Invitrogen) according to manufacturer's instructions. The resulting cDNAs were diluted fourfold, and 5 μl of each sample were used in 12 μl reaction with SYBR Green PCR Master Mix kit (Applied Biosystems). Quantitative PCR reactions were done in triplicate on an ABI 7900HT (Applied Biosystems). We calculated expression levels from their average crossing points and expressed relative to the control gene EEF2 (elongation factor 2) and normalized to levels of gene-specific expression in total brain.<SEP>CNV analyses<SEP>We previously typed 1,350 copy number polymorphisms (CNPs) in the HapMap analysis panels; 360 of these CNPs were found to be common (minor allele frequency >5%) in individuals with European ancestry (HapMap CEU), explaining more than 80% of the copy number differences between any two individuals. 323 common CNPs seemed to be diallelic, and of these 261 were in strong LD with HapMap SNPs that are close to, but do not overlap, the CNPs. For the current work, for each of these common, diallelic CNPs, we identified (from among the SNPs successfully typed or imputed in the GIANT meta-analysis) the SNP that best captured each CNP via LD in HapMap CEU. This formed the set of 261 ‘CNP-tagging SNPs’ that were used for analysis here; we used the GIANT meta-analysis P values for these SNPs.<SEP>At the NEGR1 locus, we found that the 10-kb deletion, the 45-kb deletion and the reference structural allele at NEGR1 each have perfect tagging SNPs (r2 = 1.0) in the HapMap CEU sample. In constructing Figure 4c, we colored each SNP according to which of these structural-allele-tagging SNPs it showed the strongest LD with in HapMap CEU. Locations of conserved elements were obtained from the phastConsElements17way track of the UCSC Genome Browser. A threshold score of 300 was set for inclusion in this figure.<SEP>URLs<SEP>MaCH, http://www.sph.umich.edu/csg/abecasis/mach/. METAL, http://www.sph.umich.edu/csg/abecasis/metal/.",
                  "RESULTS<SEP>Initial meta-analysis of GWAS studies of BMI (stage 1)<SEP>We carried out a GWA meta-analysis of a total of 32,387 individuals of European ancestry from 15 cohorts of 1,094 to 5,433 individuals using two parallel analytic strategies (Supplementary Fig. 1 and Supplementary Tables 1–3 online). First, we carried out a weighted z-score–based meta-analysis combining P values from cohort-specific analysis strategies. Second, we also performed an inverse-variance meta-analysis using regression coefficients and their standard errors obtained by applying a uniform analysis strategy across all studies. The results for these two strategies were highly congruent (Supplementary Fig. 2 online). Here we report results of the weighted P value analysis, as it was completed first and used to select SNPs for follow-up genotyping.<SEP>SNPs that reached P < 5 × 10−8 (a threshold that corresponds to P < 0.05 after adjusting for ~1 million independent tests) in this stage 1 analysis all mapped within the FTO gene (association peak at rs1421085, P = 2.6 × 10−19), were in linkage disequilibrium (LD) with each other (r2 > 0.51), and strongly confirm previous reports of association at this locus. A locus located near MC4R (rs17782313, P = 3.9 × 10−7) and recently associated with BMI was the fourth most significant region in the stage 1 data (Fig. 1). Even after excluding SNPs in these two established BMI loci, we observed an excess of SNPs with small P values compared to chance expectations, suggesting that some of the remaining loci with strong but not definitive evidence of association in stage 1 are truly associated with BMI (Fig. 1b).<SEP>Additional analysis of the strongest associations (stage 2)<SEP>To validate potential associations with BMI, we designed a pool of 35 variants for further genotyping, drawn from among the most strongly associated independent loci (for technical reasons, these SNPs do not correspond perfectly to the top 35 loci; see Methods). We genotyped these SNPs in up to 45,018 additional individuals of European ancestry from nine stage 2 samples (Supplementary Fig. 1, Supplementary Tables 1 and 4 and Supplementary Note online). We also obtained in silico association results for these SNPs from five BMI GWAS on 14,064 additional individuals of European ancestry (Supplementary Fig. 1, Supplementary Tables 1 and 4 and Supplementary Note). Meta-analysis of these stage 2 results combined with stage 1 data revealed SNPs from five previously unreported loci near TMEM18, KCTD15, SH2B1, MTCH2 and GNPDA2 that are strongly associated with BMI (P < 5 × 10−8; Table 1, Fig. 2 and Supplementary Table 5 online). Two additional loci, represented by rs2815752 (near NEGR1) and rs10769908 (near STK33) had supporting evidence in stage 2 samples but did not reach the P < 5 × 10−8 threshold (P = 6.0 × 10−8 and P = 1.3 × 10−6, respectively). Among these two, rs2815752 also showed a highly significant independent association with severe obesity in a pediatric cohort (P = 2.2 × 10−7; Supplementary Table 6 online), strongly suggesting that this variant represents a sixth newly discovered locus influencing BMI. For each of the six loci, multiple SNPs showed highly significant association in the stage 1 data (Fig. 2), and the associations were observed across multiple cohorts genotyped on different platforms (Supplementary Table 7 online), suggesting that idiosyncratic genotyping artifacts are unlikely to explain our results. Furthermore, the consistent association signals across different European-ancestry samples, each with low genomic control inflation factors (Supplementary Table 3), also suggest that population structure is unlikely to account for these associations. Finally, five of the six associated variants (near TMEM18, KCTD15, SH2B1, MTCH2 and NEGR1, but not GNPDA2) had Illumina proxies in high LD (r2 > 0.66) with our best SNPs that were included in an independent GWAS by Thorleifsson et al.; for all five, they observed confirmatory evidence of association with BMI (Table 1), providing strong validation of these newly discovered associations.<SEP>Of the variants showing strong association with BMI, only rs9939609 (in FTO) showed nominally significant evidence of heterogeneity across cohorts (P = 0.02, Supplementary Table 5), and none of the associations showed significantly different effects by sex (P > 0.16, Supplementary Table 5). We did not observe any significant evidence supporting the recently reported BMI associations with SNPs near INSIG2 (rs7566605, P = 0.98) and CTNNBL1 (rs6013029, P = 0.34). We did observe modest evidence for association between BMI and variation in PCSK1 (rs6232, P = 0.03 in the appropriate direction), which has previously been associated with severe obesity.<SEP>Impact on BMI, obesity, related traits and complications<SEP>The effects of the associated variants on BMI were estimated using data solely from genotyped stage 2 samples, to lessen the impact of the ‘winner's curse’; they ranged from 0.06 kg/m2 to 0.33 kg/m2 per allele, corresponding to a change of 173–954 g in weight per allele in adults who are 160–180 cm tall (Table 1). In our stage 2 samples, the six newly discovered loci together account for 0.40% of the variance of BMI, and in conjunction with the known associations at FTO and MC4R account for 0.84% of the variance (Table 1). We also estimated the allelic odds ratios for these six newly discovered variants on the risk of being overweight (BMI ≥ 25 kg/m2) or obese (BMI ≥ 30 kg/m2) compared to non-overweight controls (BMI < 25 kg/m2). According to data from the newly genotyped stage 2 samples, the allelic odds ratios for being overweight for each of the six variants ranged from 1.03 to 1.14, and for being obese from 1.03 to 1.25 (Supplementary Table 8 online).<SEP>To estimate the combined impact of these variants on BMI, we examined our largest population-based stage 2 sample (the EPIC–Norfolk cohort), analyzing the 14,409 individuals who had no missing genotypes for associated SNPs at any of the eight validated loci (TMEM18, KCTD15, SH2B1, MTCH2, NEGR1 and GNPDA2, plus FTO and MC4R). We calculated a genotype score for each individual, weighting the number of BMI-increasing alleles by their relative effect sizes (so that FTO alleles had the largest weight and MTCH2 alleles the smallest). In this cohort, the 1.2% (n = 178) of the sample with 13 or more ‘standardized’ BMI-increasing alleles across these eight loci is on average 1.46 kg/m2 (equivalent to 3.7–4.7 kg for an adult 160–180 cm in height) heavier than the 1.4% (n = 205) of the sample with ≤3 standardized BMI-increasing alleles, and 0.59 kg/m2 (1.5–1.9 kg for an adult 160–180 cm in height) heavier than the average individual in our study (Fig. 3).<SEP>Further follow-up of the confirmed SNPs in a large geographically based cohort of children (ALSPAC Study, n = 4,951 children with BMI information at age 11) showed significant and directionally consistent associations between BMI and the variants near TMEM18 (P = 3.4 × 10−5), KCTD15 (P = 0.0010) and GNPDA2 (P = 0.018) (Supplementary Table 6). Comparison of extreme childhood obesity cases (n = 1,308, SCOOP-UK) to all children in the ALSPAC cohort (n = 8,369 in the full cohort) revealed an increased risk of extreme childhood obesity for the BMI-increasing alleles near TMEM18 (OR = 1.41, P = 7.9 × 10−7), GNPDA2 (OR = 1.20, P = 1.5 × 10−4) and NEGR1 (OR = 1.29, P = 2.2 × 10−7). The absense of significant associations with childhood BMI or extreme childhood obesity for the variants near MTCH2 and SH2B1 could reflect the relatively smaller sample sizes and lower statistical power of our childhood cohorts, or perhaps a differential effect of these variants on the risk of childhood and adult-onset obesity.<SEP>Although BMI is a well accepted and commonly used measure of obesity, it is an indirect and approximate measure of adiposity. BMI has two components, weight and height, and can also be influenced by lean and/or fat mass. To determine which aspect(s) of BMI are influenced by the variants we identified, we analyzed their association with the different anthropometric components of BMI, and also with a more direct measure of adiposity, percentage fat mass. All of the variants had much stronger associations with weight than with height (Supplementary Tables 6 and 8), with the exception that for KCTD15 and MTCH2 the small effects on BMI in stage 2 samples limited our ability to dissect the effect on BMI into its constituent components. Variation at MC4R was significantly associated with adult height, as previously reported. To measure more directly the effects on adiposity, we tested these variants for association with percentage fat mass in a meta-analysis of three cohorts of adults in which percent fat mass was assessed (EPIC-Norfolk, Botnia PPP and METSIM; total n = 18,279), and also in the children from ALSPAC in whom percent body fat mass was measured at age 11 (n = 4,876). As was seen previously for FTO and MC4R, the BMI-increasing alleles at all new loci were also associated with or trended with increased fat mass in both the combined samples of adults and the childhood cohort (Supplementary Tables 6 and 8; each variant had a P value <0.1 in the appropriate direction in either adults, children or both). Thus, the associations with BMI are largely driven by effects on weight rather than height, and seem to act at least in part through an effect on adiposity.<SEP>We used publicly available results of GWAS for known obesity complications, including type 2 diabetes, lipid levels and coronary artery disease (CAD), to assess the impact of the newly discovered obesity loci on these traits. Two of the loci were associated with diabetes: GNPDA2 (P = 6.6 × 10−5) and TMEM18 (P = 7.5 × 10−4) (Supplementary Table 8). Most of the BMI-associated variants were not significantly associated with these BMI-related traits, most likely because of low power to detect very small effects in the public datasets and the incomplete correlation between BMI and these traits.<SEP>Impact of copy number polymorphism (CNP) on BMI<SEP>A large fraction of human copy number variation arises from common, diallellic polymorphisms. Most of these CNPs are in LD with adjacent SNPs, so their contribution to phenotypes can be assessed via these SNPs. We used these SNP–CNP LD relationships to assess the extent to which this subset of human copy number variation might influence BMI (see Methods). The distribution of BMI association P values in stage 1 samples for CNP-tagging SNPs conforms closely to the distribution expected under the null hypothesis, except for a single SNP (rs2815752, P = 9.3 × 10−6) (Fig. 4a).<SEP>We noticed that this SNP is the most strongly associated variant at one of our six validated loci, NEGR1. To understand better common patterns of structural variation at NEGR1, we analyzed hybridization data from 270 HapMap samples, finding that two distinct genomic segments upstream of NEGR1 are copy number variable (Fig. 4b). Haplotype analysis indicated that two deletion polymorphisms—a 10-kb deletion and a 45-kb deletion—are segregating at the locus on distinct haplotypes (Fig. 4c). The two most significantly BMI-associated SNPs immediately flank the 45-kb deletion and are in perfect LD with it (r2 = 1.0) across all HapMap analysis panels. Indeed, what initially seemed to be a long associated haplotype (the 47.3 kb spanned by these SNPs on the reference genome sequence) is in fact a short haplotype whose major feature is the absence of 45.6 kb of the reference sequence (Fig. 4c). The 45-kb deletion is therefore a strong candidate to explain the association signal at NEGR1. Although the deletion region consists entirely of noncoding sequence, the deletion allele lacks several conserved elements upstream of NEGR1 that are present on the other structural haplotypes at the locus (Fig. 4c).<SEP>Possible role in the CNS of genes near associated variants<SEP>The newly discovered variants showing strong associations with BMI lie in or downstream of KCTD15, SH2B1, TMEM18, MTCH2 and GNPDA2, and upstream of NEGR1 (Fig. 2). SH2B1 is a strong prior candidate for regulating body weight. SH2B1 is implicated in leptin signaling, and Sh2b1-null mice are obese. Notably, the obesity in Sh2b1-null mice can be reversed by targeted Sh2b1 expression in neurons, suggesting that the effects of this gene on obesity are mediated through the CNS. KCTD15​, TMEM18 and GNPDA2 have unknown functions, whereas MTCH2 encodes a putative mitochondrial carrier protein that may function in cellular apoptosis, and NEGR1 has a role in neuronal outgrowth. Although fine mapping and other experimental approaches will be required to identify and confirm the causal variant(s) and gene(s) for each locus, we note that, with the exception of SH2B1, our newly associated loci do not include obvious or previously studied candidate genes. Thus, a large sample size and an unbiased genome-wide approach has not only increased the number of known obesity loci, but also highlighted new aspects of the biology of body weight regulation.<SEP>To provide additional data on where these genes may function, we measured the expression of the genes nearest to our best SNP association signals in a panel of different human tissues. We found that, in our data, all genes except MTCH2 were highly expressed in the brain and/or hypothalamus (Supplementary Fig. 3 online). Additionally, MTCH2 mRNA expression is observed in the brain in publicly available expression data, and in these data, variant rs17788930 (r2 = 1 with lead SNP rs10838738) was associated (P = 1.3 × 10−6) with MTCH2 mRNA levels (Supplementary Table 8). These expression data suggest that, as is seen in monogenic forms of obesity, inherited variation influences common human obesity through effects in the CNS, although effects in other tissues for at least some of these genes remain possible.",
                  "DISCUSSION<SEP>Through meta-analysis of GWA data from >32,000 samples, followed by additional large-scale follow-up, we have identified six new loci that show compelling associations with adult BMI. Four of these loci (TMEM18, GNPDA2, SH2B1 and NEGR1) also show compelling evidence of association with obesity in adults or children. In general, definitive identification of the specific mechanisms through which these loci influence BMI and obesity will require detailed fine mapping and subsequent functional characterization. With the exception of SH2B1, the genes most strongly implicated on the basis of colocalization with the association signal have limited prior candidacy.<SEP>We compared our results with those obtained in another large GWAS of BMI, described in an accompanying manuscript by Thorleifsson et al.. For the five of our six newly identified loci where a comparison was possible (those that had strongly correlated proxies on the Illumina 317K genotyping platform at TMEM18, KCTD15, SH2B1, MTCH2 and NEGR1), the data of Thorleifsson et al. also showed strong evidence of association (Table 1); for GNPDA2, no adequate proxy was available. None of the other top SNPs for which we attempted replication and which had adequate proxies showed evidence of associations in the study by Thorleifsson et al. (Supplementary Table 5; results provided by U. Thorsteinsdottir, G. Thorleifsson and K. Stefansson on behalf of Thorleifsson et al.). After the six validated loci (and SNPs in LD with them) were removed from our analysis, we no longer observed a clear excess of P values smaller than expected by chance (Fig. 1c). One might conclude from this that few detectable BMI loci remain to be found. However, we are encouraged in further pursuit because among the remaining data are two additional loci reported by Thorleifsson et al. (BDNF and ETV5); both of these loci show strong confirming evidence for association in our stage 1 meta-analysis (P values of 0.00035 and 0.00043).<SEP>Many of our associated loci highlight genes that are highly expressed in the brain (and several particularly so in the hypothalamus), consistent with an important role for CNS processes in weight regulation. We found that TMEM18, KCTD15, SH2B1, GNPDA2 and NEGR1 are expressed at high levels in brain and hypothalamus (as are FTO and MC4R; Supplementary Fig. 3). The remaining gene, MTCH2, has evidence of expression in the brain in published data, as does BDNF, a locus identified by Thorleifsson et al.. These results extend and confirm previous observations with respect to FTO and MC4R, and are consistent with insights derived from monogenic forms of obesity and functional studies. Disruption in mice of Mc4r, Sh2b1 and Bdnf (all genes that seem to be involved in signaling in the brain) results in hyperphagia and/or obesity, and both Fto and Sh2b1 show diet- or obesity-related changes in expression in hypothalamus. Further general support for a neuronal basis for obesity comes from the observation that NEGR1 is thought to affect neuronal outgrowth. Finally, the effect of variants that map to a gene desert between GNPDA2 (Supplementary Fig. 3) and GABRA2 (ref.) might be mediated by GABRA2, which affects addiction behavior. Abundant evidence supports multiple possible roles of the CNS on body weight regulation, including on appetite, energy expenditure and other behavioral aspects. Determining the precise mechanism of action of these loci will require further experimentation.<SEP>Our analyses explicitly interrogate only a minority of common sequence variants in a given region; we expect therefore that the causal variant is, for some loci at least, yet to be examined. Although many variants are strongly correlated at each locus, precluding definitive identification of a causal variant, several loci have intriguing candidates. These include a large polymorphic deletion in the association interval upstream of NEGR1 (Fig. 4), and missense variants rs7498665 (A484T) at SH2B1 (r2 = 0.71 to best SNP) and rs1064608 (A290P) at MTCH2 (r2 = 1.0 to best SNP), which also disrupts a predicted SC35 exonic splicing enhancer site.<SEP>We cannot be sure which of the nearby genes are causally involved in influencing BMI. As a source of additional clues to likely causal mechanisms, we exploited publicly available eQTL data for lymphocytes and brain, and tested for association between the eight replicated variants and mRNA levels of the nearby genes (Supplementary Fig. 3 and Supplementary Table 9 online). Other than variants in the MTCH2 locus (associated with MTCH2 mRNA levels in brain and NDUFS3 levels in lymphocytes) and in the SH2B1 locus (associated with EIF3C levels in lymphocytes and brain and with TUFM levels in lymphocytes only), these studies did not yield indications of the likely causal gene(s). The SH2B1 result also illustrates some of the difficulties in interpreting associations with gene expression levels, as the presence of a missense SNP in SH2B1 and the strong prior candidacy of this gene would seem to implicate strongly alteration in SH2B1 function as the causal mechanism for influencing obesity. One possibility is that the SH2B1 variant has a causal role but happens to be in LD with a different variant that influences EIF3C and TUFM mRNA levels; alternatively, regulation of EIF3C or TUFM mRNA levels could have a causal role, instead of or in addition to variation in SH2B1.<SEP>Logistically, one important challenge in executing our study was coordinating analysis strategies and phenotype modeling across 15 different cohorts, each with specific genotyping, phenotyping, trait modeling and analytical strategies. Given this challenge, we decided to start by carrying out a meta-analysis of results from study-specific analyses, relying only on knowledge of the BMI-increasing allele and P value for each study, before completing a uniform analysis across all studies. Notably, we found very similar results between the study-specific analysis, in which different adjustments for covariates and analytical procedures were performed in different studies, and the uniform analysis, in which these procedures were harmonized across all studies (Supplementary Fig. 2). Thus, at least for this phenotype, the association analysis is robust enough to differences in phenotypic modeling so that differences in study design or analytic strategies do not preclude discovery of new loci using meta-analysis.<SEP>The effect sizes attributable to the associated variants range from 0.06 to 0.33 BMI units per allele, and each explains only a small proportion of the variance in adult BMI. As might be expected, given these modest effects and the smaller size of the relevant available datasets, we did not consistently observe measurable effects on the risk of diseases in which obesity is one of several contributing factors (such as type 2 diabetes). It is also possible that some of these variants influence BMI but have negligible effects on the downstream risk of obesity-related disease. Despite these small effects on BMI, when we combined information from the eight validated loci, we were able to identify small groups of individuals who differ appreciably with respect to mean BMI. However, at the population level, the value of these signals in predicting obesity remains quite limited (Supplementary Fig. 4 online).<SEP>These results raise the question as to why the variants detected in this large study only explain a small fraction of the inherited variability in BMI. There are several possible explanations, which require further experimentation to explore. First, there may be many more loci with common variants that influence BMI. We can predict that additional loci will be discovered by similarly sized studies in new samples: because we had only 5–10% power to detect variants such as those at KCTD15, MTCH2 and NEGR1, dozens of additional variants with comparable effect sizes likely remain unidentified. The number of common variants with smaller effects, and which might be detected with larger samples, is harder to predict, as this depends on the allelic architecture; if the number of causal variants increases as effect sizes decrease, then increasing sample size will be especially productive. Modifying effects such as interactions with environmental factors, other genetic variants, age, sex or other variables may, if substantial, also diminish apparent effect sizes, so detailed analyses of interaction with validated variants may be informative. Finally, other than the MC4R coding region, these loci have not yet been explored thoroughly for additional rare (or common) variants. As such, it is not known whether additional variants at these loci (those causal for the index association or those representing independent causal events) could explain a greater fraction of BMI variation. There are a growing number of examples, including at MC4R, where genes containing common variants associated with a particular phenotype also harbor lower-frequency, higher-penetrance variants with more severe phenotypic consequences. Comprehensive sequencing studies in these and other loci (perhaps in individuals with extreme obesity) may represent a path to finding such variants and beginning to explore the relative contributions of common and rare variation to BMI. Discovering additional variants will slowly increase predictive power. However, a greater immediate impact of these studies is the identification of previously unsuspected loci that participate in the biology of body weight regulation, and which may help guide the development of new therapies.",
                  "35778",
                  "['OBWCCE']"
                ],
                [
                  "5",
                  "27804228",
                  "Introduction<SEP>There are very few neural and genetic predictive biomarkers of weight loss. We explored two potential biomarkers of response to behavioral weight loss (BWL): neural response to palatable food receipt and a genotype found to predict better outcome in BWL interventions.<SEP>A weight loss of >5% body weight is associated with clinically significant improvements in cardiovascular and metabolic functioning and adiposity in individuals with overweight and obesity. BWL results in clinically significant weight loss of up to 10% of initial weight. Despite being the most commonly used intervention for obesity, little is known about the neural changes associated with BWL or whether these neural changes and baseline genetic factors predict BWL response.<SEP>The three fMRI studies examining the effects of BWL, have examined anticipatory neural responses using food picture paradigms (excluding cued learning tasks)(Table 1). Greater baseline activation in the putamen, insula, and hippocampus and in visual regions in response to high-calorie food pictures predicted less weight loss at 9-months. To our knowledge there are no studies examining neural responses to palatable food receipt. This is important as food receipt and food anticipation picture paradigms may be associated with different patterns of neural activation. This study is focused on understanding palatable taste receipt, without anticipatory visual responses to food pictures. In addition, this study controls for the potential confound of sex differences by only using female participants.<SEP>Prospective, observational studies examining neural response to palatable food receipt suggest that frontostriatal circuit activation is associated with weight gain over time. The frontostriatal circuit includes the dorsal and ventral striatum, midbrain, amygdala, and orbitofrontal cortex, putamen, insula, precuneus, and thalamus, which are involved in reward processing. The frontostriatal circuit is also comprised of regions implicated in cognitive control, including the ventrolateral prefrontal cortex and inferior frontal gyrus. These studies also suggest that the relationship between weight change and caudate activation response to food pictures may be moderated by the A1 allele of the TaqIA single nucleotide polymorphism (SNP) near the DRD2 gene. However, how changes in neural response to food stimuli and genotype predict weight loss in BWL are unclear.<SEP>The fat mass and obesity–associated (FTO) gene is among the most robustly associated with BMI, across different ancestries. FTO may regulate obesity risk through macronutrient intake and attenuated satiety response. A recent meta-analysis of 10 BWL trials (total n=6,951) found that those possessing the obesity-predisposing A allele (A/A or T/A genotype) of the FTO single nucleotide polymorphism (SNP) rs9939609 show almost twice the amount of weight loss as those with the T/T genotype. Although fMRI studies assessing FTO are limited, in European males with a range of BMIs, the A/A vs. T/T genotype at rs9939609 was associated with increased frontostriatal circuit activation to high- vs. low- calorie food images. Together, these data suggest the importance of considering FTO genotype within both fMRI studies assessing response to food stimuli and studies of BWL.<SEP>This pilot trial examines changes in frontostriatal circuit response to palatable food receipt among women with obesity before and after 12 weeks of BWL compared to a control group who did not receive BWL. All participants were scanned at baseline and at 12 weeks. We anticipated greater weight loss in the BWL group at 12 weeks and that BWL participants would show greater reductions in frontostriatal activation in response to palatable food receipt at 12 weeks compared to controls. Finally, we explored the contribution of both neural response to palatable food receipt and possession of the A allele of the FTO SNP rs9939609 within BWL participants on weight loss from baseline to 12, 36, and 60 weeks; 12 to 60 weeks, 12 to 36 weeks and 36 to 60 weeks.",
                  "Materials and methods<SEP>Participants<SEP>Participants were recruited using flyers and local newspaper advertisements. Exclusion criteria were: tobacco or drug use in the last year, current Axis I or II disorder; diabetes, blood pressure>140/90, kidney or liver disease, history of serious neurological event; use of psychotropic or asthma medication or appetite suppressant or weight loss medication within the last year; pregnancy, breastfeeding or specific dietary needs; or contraindications to the fMRI protocol such as claustrophobia and metal implants. No participants met criteria for any obesity-related endocrine disorders. Control participants were required to agree to not participate in any weight loss program during the study. Initially 22 women with obesity or overweight were recruited for BWL and 19 female controls. Three BWL participants dropped out of the study and two BWL participants and two control participants were excluded from analysis due to excessive head motion. Thus, 17 BWL and 17 control participants completed two scans. The average age of the sample was 41 years. The majority of participants were African-American, single, professional with at least a college education (Table 2). There was a significant difference in BMI between the BWL and the control group (m=32.2 vs. m=28.5, p=0.04). There was no difference between groups in race and ethnicity status or age.<SEP>Procedure<SEP>Participants were phone-screened for eligibility. After giving informed consent participants completed a clinical diagnostic interview, a medical health screen review and their weight and height was measured. At a second session, participants completed further interviews, underwent a mock scan and donated saliva for DNA testing. Participants were scheduled for an fMRI session within the two weeks before starting BWL and within two weeks after ending BWL. There were no differences in the days between scans for the BWL and control groups (t(32)=0.21, p<.83, control: m=85 sd=17; BWL m=81, sd=46). The weights of BWL and control participants were assessed at baseline, end of treatment or after 12-, 36- and 60-weeks post-baseline. All procedures were reviewed and approved by the Temple University Institutional Review Board.<SEP>Behavioral weight loss treatment (BWL)<SEP>BWL sessions were 90 minutes and led by a registered dietitian or a clinical psychologist. Groups of ten to twelve met weekly for 12 weeks. BWL involved goal setting, cognitive restructuring, self-monitoring, stimulus control, and physical activity. Dietary intake goals were 1200–1500 kcal/day. Physical activity goals were four 20-minute sessions per week starting at week 4, progressing to four 40-minute sessions by week 12.<SEP>Genetic assessment and analysis<SEP>Saliva specimens were collected at baseline using Oragene Discover collection kits. Samples were genotyped through Oragene GenoFind Services. The SNPs were analyzed using a TaqMan single tube genotyping assay. The TaqMan assay is an allele discrimination assay using PCR amplification and a pair of fluorescent dye detectors that target the SNP. One fluorescent dye is attached to the detector that is a perfect match to the first allele (e.g. an “A” nucleotide) and a different fluorescent dye is attached to the detector that is a perfect match to the second allele (e.g. a “C” nucleotide). During PCR, the polymerase releases the fluorescent probe into solution where it is detected using endpoint analysis in an Applied Biosystems, Inc. (Foster City, CA) 7900HT Real-Time instruments. Primers and probes were also obtained through Applied Biosystems. The triallelic haplotype for rs9939609, rs1421085, and rs17817449 were determined using the TaqMan single tube genotyping assay results in conjunction with “Haploview” and additional haplotyping bioinformatics software.<SEP>fMRI assessment<SEP>All participants fasted 8 hours prior to their scans at baseline and 12 weeks. Immediately prior to the scan, participants were screened for pregnancy (HCG Pregnancy Midstream Test) and breathalyzed for recent use of alcohol (BACtrack Special S70), and saliva-tested for use of cocaine, cannabis, opioids and amphetamines (Discover 4 Panel Oral Fluid Cassette). Fasting blood glucose levels were assessed to document fasting.<SEP>Stimuli Presentation<SEP>During the scan, participants were presented with a food receipt task. Chocolate milkshake, tasteless solution, and wash were administered via programmable syringe pumps (Braintree Scientific BS-8000) to participants while in the scanner using teflon tubing inserted through a wave guide into the scanning room. Stimuli consisted of the presentation of two runs of six conditions, each presented four times per run in a mixed event-related block design. The six conditions were (1) milkshake picture, (2) water picture, (3) milkshake picture concurrent to milkshake taste, (4) water picture concurrent to tasteless solution, (5) milkshake taste, and (6) tasteless solution. Participants received wash after each taste administration. Any liquid administration alone occurred at the same time a crosshairs was presented to avoid conflation of visual with food receipt response. During intervals between conditions, a crosshair was presented for a random period ranging from 2–10 sec.<SEP>MRI Data Acquisition<SEP>Structural and functional MRI data were collected using a Siemens 3-Tesla Verio (Siemens, A G, Erlangen, Germany) scanner using a standard 12-channel head coil, at Temple University Magnetic Resonance Imaging Center. Scanning parameters optimized the BOLD signal while maintaining a sufficient number of slices to acquire whole-brain data. Data was acquired using a two-shot z-shim EPI sequence1 that enables compensation of local magnetic field inhomogeneities in high susceptibility regions. Calibration of the z-shim settings were determined as described previously. Prior to functional scanning, a high-resolution T1-weighted 3D structural volume was collected from each participant (MPRAGE, TR=1,600ms, TE=2.46ms, flip angle= 9°, matrix size= 256 × 256 × 176 mm, field of view=252mm, slice thickness=1mm, 176 slices gathered in ascending order) for accurate slice prescription. Next, BOLD functional images were acquired with a T2* weighted double-shot z-shim echo planar imaging sequence and covered 52 descending axial slices (TR=3,020ms, TE=20ms; flip angle=90°, bandwidth=2,232 Hz/pixel, matrix size=80 × 80 × 52mm, field-of-view=240mm, slice thickness=2.5mm, distance factor=20%, 188 volumes).<SEP>fMRI Data Analysis<SEP>Data were preprocessed using SPM12 http://www.fil.ion.ucl.ac.uk/spm/software/spm12/. After using a field map, functional images were realigned and unwarped. Functional images were coregistered to the high-resolution T1-weighted images, then segmented and spatially normalized to standard Montreal Neurological Institute brain space and spatially smoothed using a three-dimensional Gaussian kernel of 6 × 6 × 5mm full-width at half-maximum, resulting in a resampled in-plane resolution of 3 × 3 × 3mm voxels. We utilized Artifact Detection Tools (ART; https://www.nitrc.org/projects/artifact_detect/) to create motion outlier and movement regressors for the first-level analyses. Any participant was removed if the data were revealed excess motion, defined as >20% of the data being outliers (i.e., global signal exceeded 9.0 and scan-to-scan movement exceeded 2mm and/or rotation exceeded 1 radians).<SEP>Participant-level analyses were performed using general linear modelling implemented in SPM12. For each participant, the regression model consisted of regressors representing the stimulus condition, convolved with the canonical hemodynamic response function six motion parameters to account for motion and regressors to account for outliers detected with ART.<SEP>This preliminary study tested if changes in frontostriatal activation to palatable food receipt from baseline to 12 weeks differed between the BWL vs. control group. Given this is the first fMRI study assessing neural changes in response to palatable taste receipt before and after BWL, we first assessed changes in all the regions of the brain (whole-brain analyses) rather than in specific brain regions (region-of-interest analyses). For the group-level analyses, whole-brain effects were examined using a 2 × 2 ANCOVA model controlling for baseline BMI to reveal any main effects for Treatment (BWL / control), Time (baseline / 12-weeks) and a Treatment group × Time interaction. To understand any interaction effects yielded, a mask was generated including the clusters where the interaction effect was significant. Post-hoc contrasts were estimated using this mask as a region of interest for evaluating specific contrasts for each Treatment group per Time point.<SEP>Given that this was a preliminary study (see Supplementary for the power yielded and the sample estimate for future trials), we used an uncorrected p<.001, with a cluster threshold of 3 for the 2 × 2 ANCOVA controlling for baseline BMI. For follow-up post-hoc contrasts, we used a family-wise error (FWE) rate corrected p< .01 to correct for multiple comparisons. Our contrast of interest was the milkshake taste vs. crosshairs contrast, which does not conflate the effects of anticipatory visual responses to food and was the most basic contrast assessing palatable food receipt available. Secondary contrasts examined were milkshake taste vs. tasteless; milkshake picture and milkshake taste vs crosshairs; and milkshake picture and milkshake taste vs tasteless.<SEP>To explore the prediction of weight loss within the BWL group, we first entered eigenvariates yielded from the 2 × 2 ANCOVA interaction into a principal components analysis (PCA) to derive fewer components to enter as predictors. Component scores were used as neural predictors of weight loss. For the FTO genotype predictor, A/A and T/A of the FTO rs9939609 were coded 0 while the T/T genotype was coded 1 because a meta-analysis of behavioral weight loss interventions showed that carriers of the FTO A/A and T/A genotypes have greater weight loss than those with the T/T. The neural and genotype predictors were simultaneously entered into separate hierarchical linear regressions predicting weight loss in BWL from baseline to: 12-, 36- and 60-weeks post-baseline, and from 12 to 36, 12 to 60 weeks and 36 to 60 weeks. Percentage weight loss was chosen as the dependent variable to allow for control of baseline weight.",
                  "Results<SEP>Weight loss at 12-, 36-, and 60-weeks<SEP>There was significantly greater weight loss in the BWL group compared to the control group after 12 weeks (Table 3). Among the BWL participants, 59% lost > 5% or more of their body weight by 12 weeks compared to 12% among controls (χ2(1)=8.24, p= .004). Although BWL participants had weight losses that exceeded 5% and were double that of controls at 36 and 60 weeks, the differences were not statistically significant possibly due to the small sample size (Table 3).<SEP>Neural response to milkshake taste from baseline to 12-weeks in BWL vs. control (N=34)<SEP>Controlling for baseline BMI, whole brain analyses of the milkshake taste vs crosshairs contrast showed a significant main effect of Time and a significant Treatment × Time interactions in the 2 × 2 ANCOVA (Table 4). When BMI was not controlled for in the same analysis, results were similar.<SEP>Post-hoc comparisons were then conducted to assess the direction of the significant interaction effects found in the 2 × 2 ANCOVA (Table 4). These comparisons showed that there was greater activation in the right caudate, right medial prefrontal cortex and right central sulcus and right orbitofrontal cortex in the BWL group at baseline compared to the control group. From the baseline fMRI scan to the 12-week scan, all these regions showed a reduction in activation in the BWL group relative to the control group, a reduction that occurred during significant weight loss in the BWL group relative to the control group. However, for these four regions, only the results for the right caudate met a more stringent thresholding criteria of p<.01 FWE-corrected. There was greater right caudate activation in the BWL group compared to the control group at baseline in response to milkshake receipt and reduced activation in this region in the BWL group from baseline to 12 weeks where p<.01 FWE-corrected (Figure 1, Table 5). While right medial prefrontal cortex, right central sulcus and right orbitofrontal cortex activation showed similar differences and changes, these were not significant at this threshold, which may have resulted from limited power.<SEP>We also examined if there were main effects for a milkshake picture and milkshake taste vs. crosshairs, milkshake picture and taste vs. tasteless and milkshake taste vs tasteless (p <.001 uncorrected, kE=3 voxels, Supplementary Tables 1–3). Follow-up contrasts for these analyses were not significant where p<.01, FWE-corrected.<SEP>Exploration of genetic and neural moderators of weight loss outcome in BWL (N=17)<SEP>The Treatment by Time interaction was associated with neural activation in seven regions. Bivariate correlations between these regions are presented in Table 6. The PCA identified two components: one which maps onto the frontostriatal circuit comprised of the right orbitofrontal cortex, the right caudate, right insula and the left insula, and the other with regions less understood but currently not associated with the frontostriatal circuit (left uvula, right central sulcus, right middle temporal gyrus, Table 6).<SEP>Within the BWL group, greater frontostriatal and other region activation to milkshake receipt and the A/A or T/A versus the T/T genotype for FTO rs9939609 predicted greater 12-week weight loss (Table 7). Similarly, greater frontostriatal activation to milkshake receipt and A/A or T/A versus the T/T genotype for FTO rs9939609 predicted greater 36-week weight loss. Only greater frontostriatal activation to milkshake receipt predicted greater 60-week weight loss. Neither neural activation nor FTO genotype predicted weight loss between 12 to 36 weeks, 12 and 60 weeks and 36 to 60 weeks in BWL.",
                  "Discussion<SEP>Our findings suggest that BWL treatment leads to reduced activation of the right caudate in response to palatable food receipt in women with obesity. Using the decoding analysis in Neurosynth(see Supplementary), the two latent topics most strongly linked to the right caudate are ‘reward’ and ‘eating’. The caudate has been implicated in the development and maintenance of habitual overeating without pleasure. A prospective non-intervention study also found activation of the caudate to palatable food receipt was predictive of long-term weight change. BWL may serve to interrupt patterns of overeating marked by a lack of enjoyment.<SEP>In addition to the right caudate findings, the right medial prefrontal cortex, right central sulcus and the right orbitofrontal cortex showed reduced activation in response to milkshake receipt in BWL relative to control from baseline to 12 weeks. Using food picture paradigms, a previous study67 showed that BWL is associated with greater reductions in right medial prefrontal cortex activation. Reduced orbitofrontal cortex response has been observed in examination of palatable taste receipt after gastric bypass suggesting that weight loss may alter the reward value of palatable foods.<SEP>Similar to findings with food picture paradigms, we found differential changes in the insula between the BWL group and control group over 12 weeks in response to combined milkshake picture and milkshake taste relative to crosshairs. However, these findings were not significant when we probed the direction of activation.<SEP>Unlike previous findings, changes in activation in the putamen or hippocampus did not predict follow-up weight loss. This may be due to methodological and sampling differences. Unlike previous studies, the current study enrolled only female participants. In addition, this study examined neural responses to food receipt rather than food pictures in BWL. Notably there may be differential neural responses to food pictures compared to food receipt have been reported previously.<SEP>The current study explored the predictive value of neural and FTO responses to BWL. Greater reductions in frontostriatal circuit activation (right medial prefrontal cortex, right orbitofrontal cortex, right caudate, and left insula) to palatable food receipt from baseline to 12 weeks among BWL participants predicted greater weight loss up to one year after BWL. We also found that possession of the A/A or T/A rather than the T/T genotype of the FTO rs993960 predicted greater weight loss at 12 weeks and at 36-weeks follow-up. This provides further evidence that this genotype may be important for BWL treatment success. A sizeable amount of the variance in both short- and long-term weight loss was accounted for by both baseline FTO and baseline to 12-week reduction in frontostriatal activation at 12 (73%) and 36 (60%) weeks, and by baseline to 12-week frontostriatal activation at 60 weeks (52%)(Table 7). However, the sample size was too small to test for interactions between genotype and neural response or covary for baseline variables like ancestry. Replication and testing of this interaction with more covariates is needed in larger samples.<SEP>Limitations of the current study include small sample size. The relatively small cluster sizes of the regions (e.g., right caudate), discovered to change differentially over time with treatment, may have resulted from the lack of power. The design would have been improved by randomizing participants to treatment or control to minimize any baseline differences between groups. Strengths of the study include a socio-economically diverse sample, eliminating confounding of sex, and examining long-term follow-up. The study is also notably diverse, including nearly 60% African American, a group disproportionately affected by obesity and for whom current interventions including BWL appear less effective.<SEP>To our knowledge, this is the first trial to examine neural response to food receipt in the context of BWL treatment and the first to examine both neural and genetic predictors of weight loss and maintenance. Additional long-term examination of the FTO gene and neural responses to palatable food receipt is needed in larger samples and across ancestries. Given the cluster sizes found in this preliminary study, there is a risk of Type 1 error and future studies may use whole-brain multiple comparisons correction based on cluster size such as AlphaSim or ClusterSim or FDR. Examination of predictors of weight loss at longer follow-up periods is also needed.<SEP>There are few biomarkers of weight loss for BWL interventions. However, the current study suggests BWL may be particularly effective for individuals with the FTO A allele and those with greater frontostriatal reduction in response to food receipt during treatment. This preliminary finding requires replication but may add to our understanding of how precision-based medicine strategies may be useful in the treatment of obesity.",
                  "23333",
                  "['OBWCCE', 'EATBTS', 'FOODIN']"
                ],
                [
                  "6",
                  "23849767",
                  "Introduction<SEP>Obesity is a well-known risk factor for hypertension and type-2 diabetes mellitus. Genetic factors play an important role in the development of obesity, but the underlying genetic pathways are poorly understood.<SEP>FTO (fat mass and obesity-associated) gene was first identified in mouse and it has been associated with energy balance. MC4R (melanocortin 4 receptor) gene was the first loci in which mutations were associated with morbid obesity in human beings. The most common genetic variant of the FTO gene (rs9939609 A/T) and a variant near the MC4R gene (rs17782313 C/T) were detected recently and both single nucleotide polymorphisms (SNPs) have been associated mainly with overall and, in some cases, abdominal obesity. Cardiovascular risk factors as type-2 diabetes mellitus, insulin resistance, and hypertension were associated with the risk allele A for FTO rs9939609 and the risk allele C for MC4R rs17782313, regardless body mass index.<SEP>FTO gene is strongly expressed in hypothalamus, a brain structure involved in the control of energy homeostasis and regulation of blood pressure. The association between the risk allele A for FTO rs9939609 polymorphism and elevated blood pressure was demonstrated in Caucasian populations. MC4R gene is also expressed in the hypothalamus, but when inactivated is associated with lower blood pressure, independently of obesity. However, this association has been shown only in individuals with mutations. In the overall population, individuals carrying the risk allele C for near MC4R rs17782313 did not have different blood pressure levels comparing to non-carriers. For type-2 diabetes mellitus risk, studies that evaluated both polymorphisms in the overall population are inconclusive.<SEP>Among individuals with hypertension, who are often obese and have type-2 diabetes, the contribution of both polymorphisms on obesity, blood pressure control and type-2 diabetes mellitus had not been investigated. The aim of this study was to investigate the association between FTO rs9939609, MC4R rs17782313 with anthropometric indicators, blood pressure, and type-2 diabetes in patients with hypertension.",
                  "Methods<SEP>This is a cross-sectional study, which enrolled men and women, aged 18 to 80 years, who had been diagnosed with hypertension, living in the urban area of Porto Alegre, southern Brazil. In total, a sub-sample of 217 patients was genotyped (38.8%) and examined in an outpatient clinic of a reference hospital. The study was approved by the Ethical Committee of our institution, and all patients accepted to participate and signed a consent form.<SEP>Genotyping<SEP>DNA was extracted from peripheral blood using NucleoSpin® Blood (Macherey-Nagel GmbH & Co. KG, Germany). StepOneTM Real-Time PCR Systems (Applied Biosystems Inc., Foster City, CA) was used to genotype the FTO locus rs9939609 and the MC4R locus rs17782313 using TaqMan-based assays (Applied Biosystems Inc., Foster City, CA).<SEP>Outcome variables<SEP>Trained research assistants carried out six standardized blood pressure measurements, obtained in three visits, using an aneroid sphygmomanometer with an adequate sized cuff to the arm circumference and the average was used to calculate mean blood pressure [(Systolic BP + 2*Diastolic BP)/3]. Hypertension was diagnosed by systolic blood pressure ≥140 or diastolic blood pressure ≥90 mmHg, or use of antihypertensive lowering agents. Type-2 diabetes mellitus was diagnosed by fasting blood glucose ≥ 126 mg /dl, glycated hemoglobin ≥ 6.5%, or use of anti-diabetic agents.<SEP>Other covariates<SEP>Demographic data and information regarding education (years at school), smoking (ex-smoker or current smoker), abusive alcohol consumption (≥30 g ethanol/day in men and ≥15 g/day in women), and physical activity (short version of the International Physical Activity Questionnaire, IPAQ, categorized as low/moderate or high intensity) were collected using standardized questionnaires by trained interviewers.<SEP>Weight (kg) was measured with patients in light clothes, barefoot, in a 100 g scale (Filizola®, model 31, IN Filizola – SA, Sao Paulo, Brazil) and height was obtained with patient in Frankfurt plane, using a Tonelli® stadiometer with 0.1 cm scale, model E120 A (IN Tonelli – SA, Santa Catarina, Brazil). Body mass index (BMI) was calculated by the weight (kg)/height (m)2.<SEP>Waist circumference was obtained with a plastic, flexible, inelastic measuring tape positioned in the middle point, between the last costal rib and the iliac crest, in a perpendicular plane, with patient standing in both feet, approximately 20 cm apart, and with both arms hanging freely at the side, with palms facing thighs. Neck circumference was measured with the head straight and eyes staring forward, horizontally, one inch below the laryngeal prominence, and hip circumference at the level of the higher diameter over the buttocks. The anthropometry was carried out in duplicate, with a one week break, by interviewers trained independently and periodically reassessed. Body Adiposity Index (BAI) was calculated using the formula: hip (cm)/[height (m)1.5]-18.<SEP>Serum triglycerides were measured by calorimetric enzymatic method using standardized techniques. Lipid Accumulation Product Index (LAP) was calculated from waist circumference, in centimeters, plus serum triglycerides, in mmol, for men [(WC-65)*TG] and women [(WC-58)*TG].<SEP>Analyses<SEP>Statistical analyses were performed using SPSS (Statistical Package for the Social Science, SPSS®, version 17.0, IL, USA). Means ± SD and percentages were compared using analysis of variance or Pearson’s chi-square test. We tested the potential relationship of genotype and blood pressure, using analysis of covariance models, with the adjustment for age and BMI. Modified Poisson’s regression (expressed as risk ratio and 95% CI) was used to analyze the association between genotype and type-2 diabetes, after adjustment for age and BMI. A P value between 0.05 and 0.1 was considered as a trend toward association and less than 0.05 as statistically significant.",
                  "Results<SEP>Characteristics of individuals included in the study for each polymorphism are shown in Table 1. Rare allele frequencies were 0.40 for A for FTO rs9939609 and 0.18 for C for MC4R rs17782313. Genotype frequencies did not differ from those predicted by the Hardy-Weinberg equilibrium (P = 0.8 for FTO and P = 0.8 for MC4R).<SEP>In total, 86 men and 131 women were assessed, aged 59.8 ± 11.2 years, systolic blood pressure of 152.1 ± 20.6 mmHg, and diastolic blood pressure of 87.7 ± 13.0 mmHg. The prevalence of type-2 diabetes mellitus was 44.7%; 4.2% of the sample reported abusive alcohol consumption and 43.8% were current or ex smokers.<SEP>Obesity<SEP>FTO rs9939609 individuals with AT and AA genotypes had higher BMI than those with TT, as well as those who had MC4R rs17782313 with CT and CC genotypes. In addition, neck circumference was higher among AT/AA carriers than TT carriers according to FTO risk allele. For other anthropometric indices, the P values remained as a trend toward association, as seen for waist circumference (FTO and MC4R) and Body Adiposity Index (BAI).<SEP>Analyses conducted according to gender (Table 2) showed that men and women who were carriers of AT/AA genotype had higher neck circumference than those homozygotes TT for FTO rs9939609, while a trend toward association was verified for body mass index. Women carrying risk alleles AA/AT also had higher Body Adiposity Index versus TT for FTO genotype.<SEP>According to MC4R rs17782313 genotype, there were no significant differences among anthropometric indexes for men with CT/CC genotype, in comparison to homozygotes TT (Table 3). Waist circumference and Lipid Accumulation Product Index had a trend to be higher in women with CT and CC genotypes and neck circumference was significantly higher among women carrying risk allele C.<SEP>Type-2 diabetes mellitus<SEP>There was no difference in prevalence of type-2 diabetes according to FTO rs9939609 genotype. The trend toward association detected for MC4R rs17782313 (Table 1), further analyzed by gender showed that women who had MC4R rs17782313 CT/CC genotype had higher prevalence of diabetes than those with TT allele (Table 3). The multivariate analysis taking into account age and BMI showed that the allele C for MC4R was associated with higher risk of type 2 diabetes mellitus in women (RR = 1.5 95%CI: 1.1 – 2.2, P = 0.03), but not among men (RR = 0.9 95%CI: 0.6 – 1.7, P = 0.9). FTO was not associated to diabetes among men (RR = 1.1 95%CI: 0.7 – 1.7, P = 0.9) and women (RR = 1.0 95%CI: 0.7 – 1.6, P = 0.9).<SEP>Blood Pressure<SEP>Carriers of risk alleles (A for FTO rs9939609 and C for MC4R rs17782313) had average diastolic blood pressure similar to non-carriers, but systolic blood pressure was significantly lower among MC4R rs17782313 individuals with CT/CC genotype and for FTO variant there was a trend toward association.<SEP>Table 4 shows systolic, diastolic, and mean blood pressure adjusted for age and BMI, among men and women according to the genotype. FTO was not associated with blood pressure for both genders, but among men with risk allele C for MC4R rs17782313, diastolic blood pressure was significantly lower (P = 0.03), and there was a trend for systolic blood pressure (P = 0.05). Therefore, mean blood pressure was markedly lower among those with allele C for MC4R, independently of age and BMI.",
                  "Discussion<SEP>As far as we know, this is the first study showing the association of a common genetic variant near MC4R gene with blood pressure in patients with hypertension. The negative association of MC4R common variant with blood pressure in men is a novelty, as it had been described only in morbidly obese individuals with rare mutations in the MC4R gene and in animal models. The association was further evaluated in analyses by gender, showing that men carrying the C allele for MC4R had lower diastolic and mean blood pressure compared with TT homozygotes.<SEP>Genome-wide association studies have been identified a number of genetic variants associated with obesity and their role in obesity-related vascular diseases. Frayling et al. showed for the first time the contribution of the SNP rs9939609 in FTO on excess of weight, and this result has been replicated in several populations. As expected, common variant of FTO was associated with BMI in our overall population, but the analysis by gender confirmed only a trend toward association. Body Adiposity Index, a simple index which detects the percentage of total fat was significantly higher among women carrying FTO risk allele, and the effect of the gene on women body weight could be essentially over body fat. Among men, the association between FTO and BMI has not been confirmed or it was weakened. Another simple measure correlated with total and abdominal obesity is neck circumference, which was higher in men and women with risk allele A. This association has also been confirmed in European adolescents.<SEP>For the overall population, common variant near MC4R rs17782313 was associated with BMI and there was a trend to have subjects with higher BAI among CT/CC carriers. Analyses conducted by gender showed that in addition to the association of neck circumference with MC4R genotype, the trend toward higher waist circumference and Lipid Accumulation Product suggest an effect of the risk allele C on visceral obesity in women with hypertension. Mutation carriers on MC4R had increased lean mass and the variant near MC4R could reflect this effect among in men, who had proportionally more free-fat mass comparing to women. Furthermore, men and women had differences regarding metabolism of free fatty acids, which increased accumulation on visceral adipose tissue in women when compared with men. Among Chinese, no association was found between MC4R rs17782313 and abdominal obesity detected by waist circumference and, in a European cohort, MC4R rs17782313 plus FTO rs1421085 were associated with abdominal obesity and metabolic syndrome. However, when analyzed separated, neither MC4R nor FTO were associated with any individual metabolic syndrome feature, including waist circumference.<SEP>In this study, type-2 diabetes mellitus was associated with women carriers of risk allele C for MC4R rs17782313, which remained significant after control for age and BMI. Studies that assessed the contribution of polymorphisms in FTO and MC4R genes on type-2 diabetes mellitus are controversial, probably due to ethnic differences regarding body composition and lifestyle among populations (significant differences about the frequencies of several common obesity susceptibility variants in or near FTO and MC4R were found in class III obese Hispanic patients, i.e.). Among hypertensive women, the effect of risk allele C for MC4R rs17782313 on type-2 diabetes mellitus could be mediated by increased visceral adipose tissue. MC4R showed higher expression in mesenteric fat among obese and diabetic rats when compared with lean rats. Besides, studies support that the association of FTO rs9939609 and the MC4R rs17782313 polymorphisms with type-2 diabetes and obesity could be modulated and depended on diet. It is also suggested that both type-2 diabetes and obesity had a common genetic etiology from the same 16.3 encoding region were FTO is located, and probably this is not the only susceptibility gene on this region which share the risk for both obesity and type-2 diabetes. Genetic predisposition to obesity and FTO common variants contributed to increased insulin resistance among Europeans, but weakly increased type-2 diabetes risk, which is mediated by BMI itself.<SEP>As seen in morbid obese - with no hypertension and with rare mutations in MC4R gene, this sample of hypertensive individuals carrier of risk allele C for common variant near MC4R had lower levels of blood pressure in comparison to homozygote non-carriers. After adjustment for age and BMI, the association remained significant in men. It has been widely demonstrated in animal models the contribution of melanocortin system on cardiovascular function, and, in human beings, it has been shown how centrally expressed melanocortin 4 receptor (MC4R) not only conveys hypothalamic signals suppressing food intake, but it is also involved in blood pressure control. Common variants near MC4R could reflect the same effect, but is unknown if anti-hypertensive drugs or changes in lifestyle could modify the expression of these genes or the effects of both polymorphisms on blood pressure, as it has been shown. Among different model species, it has been demonstrated signaling cascades and transcription factors for several obesity genes that could be useful for development of new drugs and for translational research.<SEP>This study has some limitations and strengths that should be taken into account when interpreting the results. The analysis of genotype by gender is exploratory and it is underpowered to detect small differences. Therefore, some associations that achieved statistical significance in the overall analysis remained only as a trend toward association (BMI and FTO genotype, for instance). In the other side, some associations remained significant, as neck circumference and FTO genotype, while others came up only for women (FTO and BAI). The associations of gene variants with anthropometric indexes should be confirmed in further studies, with statistical power to carry out analysis by gender.<SEP>In conclusion, this study shows a negative effect of MC4R rs17782313 on mean arterial pressure and diastolic blood pressure among men. Associations of some anthropometric indexes with polymorphisms are exploratory for both, men and women. The pathways for these associations are not completely understood, but genetic variants near the MC4R gene may reflect the complexity of such mechanisms.",
                  "15964",
                  "['OBWCCE']"
                ],
                [
                  "7",
                  "23628854",
                  "Introduction<SEP>Obesity is a major public health problem associated with increased risk of a number of diseases, including cardiovascular disease (CVD), type 2 diabetes and certain cancers. Behavioral weight loss is the treatment of choice for mild to moderate obesity as weight losses of 10% have repeatedly been documented to improve diabetes and cardiovascular disease risk factors. At the same time, the long-term maintenance of these losses remains a critical issue in obesity treatment.<SEP>Obesity susceptibility loci identified through genome-wide association studies (GWAS) and replicated in multiple independent cohorts have provided new insights into the genetic factors that contribute to the development of obesity. The fat mass and obesity associated gene (FTO) was one of the first genes to be identified by this approach and it has emerged as an important gene associated with obesity and body mass in numerous cohorts. With increasing samples sizes in GWAS studies, the number of confirmed loci continues to increase.<SEP>The treatment implications of these obesity susceptibility loci remain unclear. In particular, it is not known whether obesity-risk genetic markers predict success with weight loss or weight loss maintenance. Previously, in the Diabetes Prevention Program, FTO rs9939609 predicted a greater increase in subcutaneous adipose tissue in the placebo group compared to lifestyle intervention at year 1, but no significant genotype×treatment interaction was observed for overall weight loss. The obesity risk allele at rs6265 in BDNF was also associated with greater weight regain at two year follow-up among those who lost 3% or more of their initial weight at six-months.<SEP>The goal of the present study was to define the effects of obesity genetic risk markers available on the Illumina CARe iSelect IBC chip (within or in the region of FTO, SH2B1, MC4R, BDNF, TNNI3K, MTIF3, QPCTL/GIPR, and TFAP2B) on weight loss at 1 year in response to gold standard behavioral weight loss intervention, and weight regain from year 1 to 4 among those who lost 3% or more of their initial weight at year 1. The Look AHEAD trial, a randomized controlled trial designed to determine the effects of intensive lifestyle intervention, including diet and physical activity, on cardiovascular morbidity and mortality among overweight individuals with type 2 diabetes, provides a unique opportunity to conduct such analyses.",
                  "Material and Methods<SEP>Study cohort<SEP>The Look AHEAD study enrolled 5,145 ethnically diverse overweight and obese subjects with type 2 diabetes and aged 45 to 76 years. Of these, 1,038 did not provide genetic consent to be included in a genetic ancillary study, including all participants from three Southwest American Indian sites, 10 withdrew consent for genotyping and 60 were identified to have no or a low concentration of DNA. This left 4,037 individuals, of which 3,899 contributed genetic data on at least one of the 13 markers of interest that passed genotyping quality control procedures. These subjects form the basis for the present analyses. Overall, relative to those who provided genetic consent, those who did not were more frequently African-American, Hispanic, female, more highly educated and not dyslipidemic. Consent rates did not differ by BMI.<SEP>The design and methods of the Look AHEAD trial have been reported elsewhere, as have the baseline characteristics of the randomized cohort. Briefly, at baseline participants were randomized to either an Intensive Lifestyle Intervention (ILI) or a Diabetes Support and Education (DSE) arm. Both the ILI and DSE groups were provided one session of education on diabetes and cardiovascular risk factors. In addition, ILI patients received an intensive lifestyle program, combining diet modification and increased physical activity, designed to produce an average of 7% weight loss and maintain these weight losses. The ILI included one individual and three group meetings per month for six months, followed by one individual and two group meetings per month through one year. From years 2–4, participants were seen individually at least once a month, contacted another time each month by telephone or email, and offered a variety of ancillary classes. These sessions focused on behavioral weight loss strategies, such as self-monitoring, goal setting and stimulus control, to achieve and maintain weight loss. The DSE group received the option of attending three sessions per year on nutrition, physical activity and social support with no explicit weight loss goals. In the full trial, maximal difference in average weight loss across intervention arm occurred at 1 year follow-up (8.6% in ILI vs. 0.7% in DSE, p < 0.001), with an average weight loss of 4.7% in ILI and 1.1% in DSE at year 4 follow-up.<SEP>The Look AHEAD trial was approved by local Institutional Review Boards, including genetic analyses.<SEP>Anthropometric Measures<SEP>Weight was measured in duplicate at baseline and year 1 and 4 follow-ups using a digital scale and height was measured at baseline and year 4 using a standard wall-mounted stadiometer. Weight regain was defined as weight change from year 1 – year 4 among individuals initially losing at least some weight (>=3%) at year 1 following methods used in the Diabetes Prevention Program. As can be seen in Table 1a, among those who lost 3% or more weight at year 1, women regained 3.7±8.2 and men regained 4.8±7.8 from year 1 – 4 on average. It is important to note, however, that only 72.5% of women and 78.5% of men in this subgroup regained weight, as defined by a weight at year 4 greater than their weight at year 1, while the remaining individuals either maintained or continued to lose weight.<SEP>Genotyping<SEP>The genomic DNA extraction was based on the use of FlexiGene DNA Kit (Qiagen Inc., Valencia, CA) as described by the manufacturer and DNA quantitation was performed using the PicoGreen dsDNA Quantitation Reagent (Invitrogen, Inc., Carlsbad, CA). Genotyping was carried out at the Children’s Hospital of Philadelphia using the Illumina CARe iSelect (IBC) chip, a gene-centric 50,000 single nucleotide polymorphism (SNP) array designed to assess relevant loci across a range of cardiovascular, metabolic and inflammatory syndromes. Taqman Applied Biosystems (ABI) Assays-On-Demand were used to genotype the MC4R polymorphism rs17782313 (ABI catalogue number C_32667060_10)) using an Applied Biosystems 7900HT.<SEP>Gene and SNP Selection<SEP>We performed a search of published literature and selected SNPs that had been associated with obesity by GWAS and appeared on the IBC chip or, in the case of MC4R rs17782313, had been genotyped by Taqman. References for the selection of each SNP are provided in Table 2. As multiple markers have showed the strongest association with obesity in the FTO region and two distinct loci have been identified in the BDNF region, we retained multiple SNPs in each of these regions. FTO rs1421085, rs3751812 and rs9939609, BDNF rs6265 and rs10767664 and TFAP2B rs2272903 were assayed directly on the IBC chip. GWAS obesity SNPs not on the IBC chip were replaced by proxies where possible using the SNP Annotation and Proxy Search tool (SNAP) based on haplotype maps of individuals of European ancestry (CEU) and Yoruba people of Ibadan (YRI) as follows: FTO rs9930506 was replaced by rs9922708 (distance 681 bp r2=1.00, D’=1.00 in both CEU and YRI); BDNF rs925946 was replaced by rs1401635 (distance 26,789 bp r2=0.96, D’=1.00 in CEU; no proxy was available in YRI), SH2B1 rs7498665 was replaced by rs4788099 (distance 27,514 bp r2=1.00, D’=1.00 in CEU and D’=1.00 and r2=0.94 in YRI), TNNI3K rs1514175 was replaced by rs1514176 (distance 48 bp, r2=1.00, D’=1.00 in CEU and r2=1.00, D’=1.00 in YRI), MTIF3 rs4771122 was replaced by rs7988412 (distance 19898, r2=0.83, D’=1.00 in CEU; no proxy was available in YRI), and QPCTL/GIPR rs2287019 was replaced by rs11672660 (distance 21988 bp, r2=0.83, D’=1.00 in CEU, r2=0.89, D’=1.00 in YRI).<SEP>The four FTO SNPs selected for inclusion were in strong linkage disequilibrium in non-Hispanic Whites (r2=0.78–0.97), but differed in the degree of disequilibrium among African-Americans (rs3751812, rs1421085: r2=0.98; rs3751812, rs9922708: r2=0.70; rs1421085, rs9922708: r2=0.67; rs9939609 with other SNPs: r2<0.36). In contrast, two BDNF SNPs, rs6265 and rs10767664, were in strong linkage disequilibrium in both non-Hispanic Whites (r2=0.88) and African-Americans (r2=0.81).<SEP>Observed genotype frequencies were compared with those expected under Hardy Weinberg Equilibrium (HWE) using stratified Χ2 tests within the two largest racial/ethnic groups (non-Hispanic White and African-American). All SNPs under study conformed to HWE (p > 0.001).<SEP>Statistical Analysis<SEP>To control for admixed study population, all IBC SNPs were examined by principal component analysis (PCA) using the EIGENSTRAT algorithm as implemented in Golden Helix version 7.1 (Bozeman, Montana, USA). PCA results indicated that the majority of the variance among the Look AHEAD cohort was accounted for by the first two principal components, which agreed with self-reported ethnicity (Supplemental Figure 1). Accordingly, the first two principal components were included as covariates in our analyses to adjust for population stratification in the multi-ethnic Look AHEAD cohort.<SEP>As the primary adiposity outcome in the Look AHEAD clinical trial is weight (not body mass index), we focus on baseline weight (in kg) and change in weight (in kg) as primary outcomes. Longitudinal linear mixed models were used to model the effect of SNP on weight change by treatment arm over time. As baseline weight as well as treatment response can be associated with the SNPs, baseline was modeled as the first time point in longitudinal analyses as recommended by McArdle & Whitcomb, 2009. Within this model, differential SNP effects on year 1 weight change or by treatment arm are detected through SNP (0,1 or 2 copies of the minor allele) × time (baseline, year 1) × treatment arm (ILI, DSE) interaction. An additive genetic model was used for all markers, with genotype coded by the number of minor alleles. Therefore, all our SNP effects can be interpreted as the effect on the outcome of interest of one additional copy of the corresponding minor allele. Models were estimated in Splus 8.2 using restricted maximum likelihood. Longitudinal outcomes were additionally adjusted for age, gender, study site, and the first two ancestry informative marker principal components.<SEP>Next, we examined the extent to which the genetic markers predicted weight regain at year 4. As weight regain implies initial weight loss, we limited analyses to those who lost 3% or more of their initial weight at year 1, consistent with prior analyses focusing on weight regain in the Diabetes Prevention Program. Interest centered on whether SNP effects, if present, could be averaged across treatment arms or should be presented in a treatment-specific fashion (SNP × treatment arm interaction). The same covariates were employed as above, with the addition of year 1 weight.<SEP>To adjust for multiple comparisons, we calculated the effective number of independent genetic loci using principal component analysis as recommended by Li and Ji. Principal component analysis of the genotypic correlation matrix of the 13 markers of interest identified 10 independent loci in the full and non-Hispanic White samples. Therefore, one can maintain the family-wise error rate at 0.05 via Sidak’s adjustment for multiplicity by declaring as significant only those markers with a nominal significance level of 0.05/10=0.005. However, as the markers were selected a priori, we also discuss results with p values less than 0.05 not adjusted for multiple testing. All analyses were performed at Brown University.",
                  "Results<SEP>Descriptive statistics<SEP>Participant characteristics of the sub-cohort of Look AHEAD used in these analyses are shown in Table 1. Individuals were evenly distributed between the ILI and DSE intervention arms, and had comparable age, gender and ethnicity as in the entire cohort (data not shown). No baseline differences in demographic or clinical characteristics across ILI and DSE were observed. Similar to the larger Look AHEAD trial, participants assigned to ILI lost significantly more weight at year 1 and 4 than those assigned to DSE. SNP characteristics, including the obesity risk allele identified in the prior literature, are presented in Table 2.<SEP>Genetic associations with baseline weight<SEP>Genetic associations of SNP markers with baseline weight are listed in Table 3. Obesity risk alleles in FTO, SH2B1 and QPCTL/GIPR regions predicted baseline weight in directions consistent with prior research. Risk alleles for the markers in these 3 genes were associated with elevated baseline weight of 1.01–1.29 kg per copy. Similar associations were found for baseline BMI (Table 3), with BMI effects per risk allele in the 0.38–0.46 range.<SEP>Genetic associations with weight loss at year 1<SEP>Genetic associations of the full set of SNP markers with year 1 weight change in ILI and DSE are listed in Supplemental Table 1. No SNPs were significantly associated with the magnitude of weight change in either ILI or DSE or interacted with treatment arm in predicting the degree of weight change<SEP>Weight regain<SEP>Participant characteristics for those who lost 3% or more at year 1 of their weight at baseline are presented in Table 1a. Genetic associations of the full set of SNP markers with weight at year 4 in this subgroup is presented in Table 4. The obesity risk (A) allele at FTO rs3751812 was significantly associated with weight regain in DSE (1.559 kg per risk allele, p = 0.005), but not ILI (-0.092 kg per risk allele, p = 0.761), resulting in SNP×treatment arm interaction (p = 0.009). Similar effects were seen for FTO rs1421085 and rs9922708.<SEP>A regional plot of the association of FTO with differential change in weight regain across ILI and DSE is depicted in Supplemental Figure 2. Of interest, rs3751812 and SNPs in linkage disequilibrium do not show the strongest association with differential weight change across ILI and DSE. One SNP, rs8061397, in a distinct linkage disequilibrium block is associated with differential change in weight across ILI and DSE (p = 6.4 ×10−5), suggesting a possible additional signal in the region.<SEP>In a possible replication of prior research, the obesity risk (G) allele at BDNF rs6265 was associated with greater weight regain across treatment arms (0.773 kg per risk allele), although results were of borderline statistical significance (p=0.051). When combined, the FTO and BDNF SNPs accounted for R2 = 1.43% of year 4 weight across treatment arms.",
                  "Discussion<SEP>This paper presents the results of the largest study to date examining whether SNPs previously associated with obesity predict weight loss in response to behavioral treatment or weight regain after successful weight loss treatment. We found no significant SNP associations with magnitude of weight loss at year 1 or SNP × treatment arm interactions in predicting year 1 weight change, suggesting behavioral factors, such as adherence to weight loss recommendations, may predominate in predicting initial weight loss. However, the obesity risk region within FTO was significantly associated with weight regain in the control group, but not in the lifestyle intervention group, resulting in SNP × treatment arm interaction. Further, variation within BDNF was associated with weight regain across treatment arms in replication of prior results in the Diabetes Prevention Program. Overall, these results suggest that the obesity risk alleles do not appear to be strongly predictive of the magnitude of weight loss in response to behavioral intervention but may instead be associated with weight regain after weight loss.<SEP>FTO was the first gene to show replicated association with obesity in GWAS and continues to show the strongest association with obesity and body mass index across a variety of populations. This region also shows strong evidence for gene×behavior interaction as the interaction of obesity-risk alleles at FTO with physical activity in predicting body weight has been confirmed by replication and meta-analysis in epidemiologic studies. In the context of randomized, controlled trials, FTO rs9939609 predicted a greater increase in subcutaneous adipose tissue in the placebo group compared to lifestyle intervention at year 1, but no significant genotype × treatment interaction was observed for overall weight loss in the Diabetes Prevention Program. In the POUNDS LOST trial, the minor allele at FTO rs1558902 predicted greater free fatty mass in response to a low-protein diet but less free fatty mass in response to a high fat diet at two year follow-up, again with no effect on weight change. The present results extend this gene × behavior interaction to the context of a combined caloric restriction and physical activity intervention arm in a longitudinal randomized, controlled clinical trial with four year follow-up. Taken together, these results indicate that behavioral strategies may blunt FTO effects on weight gain and weight regain after weight loss. However, the detection of effects of the FTO region on weight loss may require detailed measurements of body fat, such as with dual energy X-ray absorptiometry scan.<SEP>Previously in the DPP, the obesity risk allele at rs6265 in BDNF was associated with greater weight regain over 2 years among those who had initially lost 3% at or more at six months. The effect occurred across all three treatment arms: a lifestyle intervention promoting weight loss and physical activity, 850 mg metformin twice daily, and placebo. In Look AHEAD, we provide some evidence, albeit of marginal significance (p = 0.051), for association of the obesity risk allele at rs6265 with weight regain at four-year follow-up among those who had lost 3% at one-year follow-up across two treatment arms. BDNF and its primary receptor TrkB are widely expressed in the brain, including key regions of the hypothalamus and dorsal vagal complex related to body weight and energy homeostasis. In these regions, infusion of BDNF produces appetite suppression and weight loss. Conversely, targeted disruption of BDNF in transgenic models results in hyperphagia and obesity. In a prior Look AHEAD study, BDNF rs6265 was associated with greater total caloric intake and more servings from the dairy and the meat, eggs, nuts and beans food groups. At least one case study also links rare mutations in BDNF to severe obesity in an 8-year-old girl.<SEP>This study has several strengths, including a randomized clinical trial design, a highly effective behavioral weight loss intervention and inclusion of multiple genetic markers previously associated with obesity. The sample size of the study is both a strength and limitation. While this is the largest study to examine genetic predictors of weight loss and weight maintenance, it is smaller in size than samples used to discover the obesity risk SNPs under consideration. It is plausible that the inclusion of more obesity risk polymorphisms would have identified additional associations with weight loss or regain. However, we did include markers reflecting several of the strongest associations with obesity, including those within FTO, MC4R and BDNF. Although we sought to define the role of obesity risk SNPs in weight loss and weight regain, it is possible that genetic factors associated with weight loss and regain may derive from different pathways than those influencing obesity per se. An agnostic approach, such as GWAS or exome sequencing, may be required to identify such pathways. Finally, this cohort was selected for type 2 diabetes and overweight and the generalization of these results to other populations remains to be determined.<SEP>Overall, our findings advance existing knowledge on the treatment implications of obesity susceptibility loci derived from GWAS. No significant SNP associations with magnitude of weight loss at year 1 or SNP×treatment arm interactions in predicting year 1 weight change were observed. However, we identify FTO and BDNF as possible predictors of weight regain after weight loss.",
                  "20397",
                  "['OBWCCE', 'EATBTS', 'FOODIN']"
                ],
                [
                  "8",
                  "21720422",
                  "INTRODUCTION<SEP>There is extensive evidence that the risk of chronic diseases such as obesity, diabetes, coronary heart disease, and cancer is developmentally programmed. That is, risk for disease is affected by the interaction of both prenatal and early postnatal environmental exposures with the fetal and infant genome. Despite the importance of early life gene-by-environment interaction on health and disease, there is relatively little genetic epidemiologic research conducted in infancy and early childhood. For example, while evidence of genetic effects on birth weight is strong, documentation of the heritability and measured genotypic contributions to growth and other markers of health during the early postnatal period remains fragmentary, despite the known importance of postnatal weight gain to future obesity risk. Previous work by our-selves and others have shown a high heritability (h2) for weight and length during infancy, but the genetic architecture of obesity-related measures such as relative weight or weight velocity, which are frequently used as markers of infant nutritional status and obesity risk, have not been examined.<SEP>BMI tracks strongly from adolescence into adulthood, but less strongly from early childhood. One explanation is that different genetic variants contribute to obesity in early life and in adulthood. Polymorphisms in the FTO (fat mass and obesity-related) gene are, to date, the most widely replicated common obesity-related genetic variants in adults, but evidence is mixed on their associations in infancy. Some studies have found a relationship between FTO variants and infant weight and adiposity, while others have not. We hypothesize that lack of replication of FTO effects in infancy may relate to the rapidly changing genetic and maternal environmental influences on growth traits that may increase the likelihood of finding effects at some ages more than others.<SEP>Using one of the largest existing family-based longitudinal studies of infant and childhood growth, the aims of the study were to: (i) provide a comprehensive analysis of the heritability of infant weight, relative weight, length and growth velocities, and the changes in their heritability during infancy (i.e., gene-by- age interactions), and (ii) test the hypothesis that a common variant in FTO is significantly associated with infant growth or growth velocities, particularly at ages when trait heritabilities are high and theoretically most likely to reveal the influence of measured genetic effects.",
                  "METHODS AND PROCEDURES<SEP>Study sample and measurements<SEP>Study subjects included 917 white participants (444 females, 473 males) in the Fels Longitudinal Study, a study of child growth and development and later cardiovascular disease risks conducted in the Dayton, OH area of the United States. The 917 participants were distributed among 121 nuclear and extended families and in total comprised of 5,393 relative pairs, of which 1,036 pairs were of first-degree relatives, 1,031 pairs were of second-degree relatives, 1,128 were of third-degree relatives, and 2,198 pairs were of less closely related relatives.<SEP>For inclusion in this analysis, participants were required to have at least three serial weight and length measurements taken between birth and age 42 months. Weight and recumbent length were measured by trained anthropometrists at target ages of 0, 1, 3, 6, 12, 18, 24, 30, 36, and 42 months. For measurements over 24 months of age, standing height was used in lieu of recumbent length.<SEP>Relative age of the infant was calculated as the age at measurement minus the exact target age and was included in all analyses as a covariate to account for small individual variations in the exact timing of the measurements, which varied no more than 2 weeks before or after the target age at birth and 1 month, no more than 6 weeks at 3, 6, 9, and 12 months, and no more than 3 months at 18, 24, 30, and 36 months. Gestational age, birth order, and maternal age were obtained from questionnaires administered by study staff to the mothers at their infant’s first exam. Means and ranges of the covariates used for analyses are listed in Table 1. Gestational age information was missing for ~30% of the subjects, and was imputed five times using PROC MI in SAS (Cary, NC). The imputation model contained all measurements of weight, length, ponderal index (PI), sex, birth year, age at measurement, birth order, maternal age, and interactions of birth year by gestational age, maternal age, and sex, in order to conform to the potential structural complexity of the data. Heritabilities and s.e. of the heritabilities differed minimally across the five gestational age imputation runs. As the increases in s.e. estimates after adjusting for multiple imputations were negligible relative to the magnitude of the h2 estimates, and using just one imputation did not influence the interpretation of results, we randomly chose one imputation set for all (univariate, gene-by-age and measured genotype) analyses.<SEP>A subset of 534 participants with available DNA was genotyped for single-nucleotide polymorphisms (SNPs) in the FTO gene (rs1421085, rs17817449, rs8050136, and rs9939609) using TaqMan assays (Applied Biosystems, Foster City, CA). These common variants were selected based upon their frequently observed association with adiposity traits in adults of European ancestry. At the phenotypic level, the 534 individuals did not have significantly different weights and lengths than the full sample, although on average, they were slightly older when measured at their visit at birth and one month (P < 0.02). Participants were also no different at the additive genetic level as tested in a gene-by-genotyped analysis. That is, the genetic variances of all weights and lengths for participants with SNP data were not significantly different from those without SNP data. All protocols were approved by the institutional review board of Wright State University and informed consent of the parents was obtained at each study visit.<SEP>Model fitting<SEP>Missing growth data (<5% of subjects at any given time point) occurred randomly due to missed appointments to the study center. That is, while one participant may be missing their 3-month visit, a different participant may have missed their 6-month visit. We modeled the serial weight and length data for all children in a mixed effects regression model using a fourth-degree polynomial function of age, implemented in SAS PROC MIXED, which was similar to a technique used by van Dommelen et al. to analyze longitudinal growth data in the Dutch Twin Study. Allowing each curve parameter to be estimated simultaneously in a mixed effects regression model ensures a consistent sample size for each time point (age) and a reduction in error compared to the use of individual regression models. All age terms were included as both fixed and random effects, and models also included sex, birth year, birth year2, age-by-sex and age-by-birth year interactions as fixed effects. Age3 and age4 random effects were not significant for length and were dropped out of the models for length. Residual and influence diagnostics indicated that model assumptions were not violated and estimates of fixed effects were not overly influenced by any single individual. Additionally, correlations between observed and predicted values for all growth metrics and at all ages were high (r > 0.88) indicating that these models accurately fit the observed patterns of individual infant growth.<SEP>Using the estimates for fixed effects and predictions for random effects from these models, predicted weight and length were obtained at each specific target age (e.g., birth, 1 month, etc.), and then used to calculate predicted BMI (weight (kg)/length (m2) and predicted PI (weight (kg)/ length (m3)) at each target age. We calculated the first derivative from each individual’s growth curve equation for weight and length to compute individual instantaneous velocities at each age except at beginning and end points (birth and 42 months). We computed velocities at age one day in lieu of at birth. The predicted static measurements and instantaneous velocities (Table 2) were then used in the following genetic analyses described below. Age 42 months was not included in the final analysis as it was predicted with less accuracy due to it being at the tail of the growth curve and there were fewer measurements available.<SEP>Quantitative genetic analysis<SEP>Growth traits and velocities were inverse normalized where necessary to handle residual kurtosis. Maximum-likelihood variance components methods in SOLAR were used to estimate narrow-sense heritabilities (h2) of each infant metric at ages 0, 1, 3, 6, 9, 12, 18, 24, 30, and 36 months adjusted for gestational age, sex, birth order, maternal age, birth year, and relative age. The age at maximum heritability was noted for each trait. Narrow-sense heritability, defined as the proportion of the phenotypic variance attributable to additive genetic effects, was estimated as h2 = σ2G/σ2P, where σ2G is the additive genetic variance and σ2P is the total phenotypic variance.<SEP>Likelihood ratio tests were used to test if h2 estimates were significantly different from zero. Gene-by-age interactions were also evaluated using likelihood ratio tests. Gene-by-age interaction effects occur when the genetic correlation (ρG), a measure of shared genetic influences (pleiotropy) between a trait measured at two different ages is significantly different from one, or when the genetic variances (σG) of a trait measured at two different ages is significantly different at the two ages. Trivariate quantitative genetic analysis was used to simultaneously estimate the genetic correlations and variances at three time points: birth, age at maximum trait h2, and at 36 months of age in order to minimize the number of statistical tests performed. In SOLAR, for a trivariate analysis, the covariance is given by a 3 × 3 covariance matrix with elements defined as Ωab = 2ΦρGσgaσgb + IρEσeaσeb, where 2Φ is the kinship matrix that structures σga and σgb, the variance due to the additive effects of genes for time points a and b; I is an identity matrix that structures σea and σeb, the variance due to unmeasured, non-genetic factors in a and b; ρG is the additive genetic correlation between time points a and b, and ρE is the unmeasured environmental correlation between the two time points.<SEP>Specifically, to test for gene-by-age interaction effects or continuity of genetic control, we tested whether ρG estimates for each trait measured at the three different ages were significantly different from one (i.e., complete pleiotropy). For each trait measured at different times (i.e., ages), complete pleiotropy occurs when the same gene (or set of genes) that influences a trait at one age also influences the trait at other ages. In this gene-by-age context, we tested for common genetic effects for each trait at the three selected age points. Thus, if ρG is different from one it is concluded that at least some unique genes are influencing the same trait at different ages, and a gene-by-age interaction effect exists. In situations of no pleiotropy, genetic correlations are not significantly different from zero, and the genes that influence a trait at one age are wholly different from the genes that influence the trait at the second age. Incomplete pleiotropy occurs when the genetic correlation is different from both zero and one. In this case, some, but not all, of the genes that influence a trait at one age, also influence the trait at other ages.<SEP>Similarly, differences in h2 by age were tested by examining differences in the decomposed variances of each trait at each age. We tested whether genetic variances (σG) or environmental variances (σE) were significantly different at the different ages. By modeling parameters of σG and σE, it could be determined if differences in h2 were due to changes in the effect of polygenes (σG), residual environmental effects (σE), or both. Differences in the genetic variances estimated at different ages can be interpreted as an indicator of differences in the expression of polygenes and are therefore indicative of gene-by-age interaction effects.<SEP>Measured genotype approach<SEP>Associations between FTO variants and infant growth metrics measured at birth, age at maximum h2, and age 36 months were tested using the measured genotype approach using a maximum-likelihood, variance components–based, procedure implemented in SOLAR that accounts for the nonindependence among family members and assumes additivity of allelic effects. Additive genetic effects and residual environmental effects were modeled as random effects, while sex, gestational age, maternal age, birth order and the FTO SNPs were modeled as fixed effects. The quantitative transmission dis-equilibrium test was used to examine potential hidden population stratification.",
                  "RESULTS<SEP>Heritability of infant growth traits<SEP>The heritability of infant growth metrics was significant (P < 3.9 × 10−10) and additive genetic effects explained from 39% of covariate-adjusted variance for PI at birth (h2 = 0.39, s.e. of h2 = 0.08) to 74% of the phenotypic variance for length at birth (Figure 1 and see Supplementary Table S1 online). Heritability estimates of all traits increased during early infancy to mid-infancy, explaining 75–92% of the phenotypic variance by 1 year. Maximum h2 occurred at age 9 months for weight, age 30 months for recumbent length, and age 6 months for PI and BMI (Figure 1).<SEP>Heritability estimates of the instantaneous velocities for weight and length are shown in Figure 2 (and see Supplementary Table S1 online). Heritabilities of weight velocity were 0.64 at birth (i.e., one day of age), 0.78 at 6 months (maximum h2), and 0.58 at 36 months (all P < 3.0 × 10−12). Heritability of length velocities were 0.60 at birth (i.e., one day of age), 0.99 at 18 months of age, and 0.52 at 36 months (all P < 2.2 × 10−13).<SEP>Cross-trait bivariate quantitative genetic analyses (see Supplementary Table S2 online) were conducted to determine if each trait was influenced by a set of unique (non-pleiotropic) or shared (pleiotropic) genes at each age. While little evidence for genetic pleiotropy existed between length and BMI at each age, incomplete pleiotropy existed between all of the other traits, demonstrating that, as expected, the proportion of shared genes is high among infant weight, BMI, and PI.<SEP>Gene-by-age interaction<SEP>Differences in heritability of the traits at different ages were formally tested using trivariate quantitative genetic analyses for each trait at birth, age at maximum trait h2, and age 36 months, and show a dynamic picture of genetic and environmental influences on growth during the infancy period (Table 3). The genetic correlations (ρG) among all traits at birth, maximum h2, and age 36 months were significantly greater than zero (all P < 0.01) and significantly less than 1.0 (all P < 0.000004), indicating that some unique and some shared genes control growth traits and growth velocities over time (incomplete pleiotropy). Thus, gene-by-age interaction effects are present for each growth trait and velocity examined.<SEP>Tests for differences in the genetic variance (σG) and the residual environmental variance (σE), at each age indicated that the changes in heritability from birth to maximum h2 were generally due to both significant increases in σG and significant decreases in σE. In the case of weight and weight velocity, however, only one of these components significantly changed (see Table 3). From age at maximum h2 to 36 months, σG and σE were relatively stable for weight, length, PI and BMI, while σE increased for weight and length velocity. The σG decreased for length velocity over the same period.<SEP>Association of FTO variants with infant growth traits<SEP>The four genotyped FTO SNPs were in Hardy–Weinberg equilibrium (P > 0.66). Given that they exhibited high linkage dis-equilibrium (r2 > 0.96 for all), we tested only the association with the rs9939609 SNP (T > A), the SNP most often reported in the literature as being associated with BMI. The minor allele frequency of the A allele was 39.9%. Trait means by SNP genotype, as well as the significance of the SNP for each metric at birth, maximum h2 and 36 months are presented in Table 4. There was no evidence of population stratification (P > 0.13). No association of rs9939609 with infant weight, length, relative weight, or instantaneous velocities (Table 4) was found at any time point (all P > 0.11). The maximum variance explained by the SNP marker was h2m = 0.009 (0.9%) for length velocity at 18 months.",
                  "DISCUSSION<SEP>Despite a strong and growing interest in understanding the interaction of genetic and prenatal influences on childhood obesity, as seen in the design of the upcoming National Children’s Study of 100,000 children born in the United States, understanding of the genetic contributions to commonly collected infant growth and health traits remains fragmentary. This information is useful for guiding investigations into the genetic and epigenetic influences on early obesity-related traits, as it may help to identify infant growth traits that have the greatest likelihood of revealing genetic effects and may also be used to target nutritional interventions to periods of growth when environmental factors play a greater role.<SEP>Our estimates of total additive h2 for birth weight and birth length are strong, and, like other family studies, our estimates are higher than those reported from twin studies (see review in ref.). There are a number of reasons why heritability estimates may differ by study design, but in the case of twin studies, the lower h2 of birth weight and length may reflect greater prenatal environmental stressors experienced by twins. Not only are twins frequently born preterm, they are also more frequently growth-restricted relative to their gestational age at birth than singleton infants, stemming from some degree of placental insufficiency in multiple pregnancies, which may affect the twins unequally. Although twin studies provide uniquely powerful evidence regarding the role of shared maternal and postnatal environment vs. shared genetic influences, early growth traits in twins are nonetheless markedly different from singletons. Longitudinal extended pedigree studies such as the Fels Longitudinal Study provide an important perspective for infant (singleton) growth.<SEP>Wardle et al. noted high heritability estimates for BMI and waist circumference in children born during the obesity epidemic that were similar to those from much earlier studies. Significant secular trends in childhood BMI have been previously reported in the Fels Longitudinal Study, however, the present data indicate that even after accounting for birth year, infant weight, BMI, and PI, are nonetheless highly heritable. It is sometimes assumed that because average weight and BMI have increased greatly during the past 30 years due to changes in the food and activity environment, genetic effects are relatively unimportant in explaining the risk of current human variation in obesity and its related traits. However, the present study, which followed families from 1929 to the present, clearly shows that even after accounting for secular trends towards increasing body size, the contribution of genetic factors to the individual differences in growth is very high.<SEP>As relative weight is rarely examined in studies of the genetics of infant growth, there are few studies with which to compare the heritability of relative weight (BMI and PI) against weight and length. Beardsall et al. found a h2 of 0.72 for BMI at birth, which was higher than their estimate for weight (h2 = 0.43) at birth. The only h2 reported for PI to date was very low (h2 = 0.06), and not significantly different from zero. To our knowledge, only one other study has examined weight and length velocities. In this twin study, heritability estimates of instantaneous weight (h2 = 0.57–0.63) and length (h2 = 0.36–0.44) growth velocities at 1 year of age were lower than our estimates. Our study is the first to report growth velocities at different ages for pedigreed data.<SEP>In our study sample, there was a significant increase in the heritability of growth traits during the first postnatal year, which was due to simultaneously increasing genetic variance (σG) and decreasing environmental variance (σE). Some indications of increases in the heritability of infant weight and relative weight have been previously reported. Together, these studies and our data point to the timing of the “canalization of development”, or when the genetically based developmental course is most strongly manifest. It has long been held that genetically based growth trajectories do not become completely established in humans until after 2 years of age. The increasing σG in our data suggests that canalization is established sooner than that; canalization appears to be established by 6–9 months for weight and relative weight, and by 30 months of age for length. Indeed, by 18 months, the heritabilities for length and length velocity are remarkably high, and close to one. It is important to remember, however, that heritabilities are estimates with associated errors and are population specific. The Fels Longitudinal Study population is generally well nourished and free from significant disease and therefore our results may demonstrate the upper limits of such heritability estimates.<SEP>Previous work suggests that rapid growth in early infancy is more deleterious for future obesity risk and insulin resistance than is rapid growth in later infancy. The rapidly increasing genetic variance for infant weight, length, and their respective velocities that we observed over the first postnatal months suggests why weight and relative weight might be more responsive to environmental perturbation (e.g., formula feeding, dietary intervention effects) earlier than later in infancy.<SEP>FTO is predominantly expressed in the hypothalamus and thought to play a role in energy homeostasis. FTO variants have been found to predict child appetite, satiety, and food intake behavior, as well as BMI and adiposity in childhood. These studies suggest that FTO does not exert its influence on obesity until later in childhood, somewhere between 6 and 11 years of age. Haworth et al. found an increasing influence of FTO on BMI as heritability increased, and we similarly hypothesized that the disputed effect of FTO on weight status could be explained by focusing on specific periods during infancy and early childhood when heritability was highest. However, we found no influence of rs9939609 on weight, length, relative weight, or instantaneous velocities in weight or length at any age during infancy. Similarly, some other studies also have been unable to identify a significant effect of FTO at this young age. For instance, in a large cohort (>20,000) of European American children 2–18 years of age, none of the FTO variants that were associated with BMI s.d. score at ages 6–18 were significant in children at 2–5 years of age. However, in a study of 225 Spanish infants, those carrying the T allele at rs9939609 had higher weight, PI, total fat, abdominal fat, and visfatin levels by 2 weeks of age, and Gemma et al. found that AA homozygotes for the rs9930506 FTO SNP had lower weights at birth compared to infants who carried the G allele. The latter study was conducted in a very small sample (total n = 82), however, that included large numbers of both small and large for gestational age infants. Additional work is required to understand how FTO expression relates to the FTO genotype and energy metabolism in different tissues and at different developmental periods in humans.<SEP>More generally, a comprehensive examination of the genetic, intra-uterine, and postnatal environmental determinants of infant growth is critical, given its relationship to later obesity and cardiometabolic risk. Recent evidence from the EDEN study showed that paternal BMI effects on infant growth velocity increase in the first 3 months, while maternal BMI effects were only important at birth. Genes involved in growth are often imprinted, and there is some evidence that paternally expressed genes (such as IGF2) may be more strongly expressed postnatally. Our present study did not examine maternal and paternal effects on prenatal and postnatal growth, but it is possible that part of the increasing genetic variance in infant size may reflect the postnatal manifestation of paternally expressed growth regulation, or other epigenetic changes in gene expression occurring dynamically over the period of infancy, as has been suggested recently.<SEP>Limitations of the study reported here are that the results cannot be generalized to nonwhite populations, or to the current US white population of infants because the average birth year in our study was 1962. However, our quantitative genetic results are consistent with the findings of more recently born cohorts of Hispanic and UK white European children. Although our study is among the largest of family studies of infant growth in singletons, we were somewhat underpowered to detect FTO effects. Given the pedigree structure of the 534 individuals with SNP data, we had ~60% power to detect a SNP explaining 1.0% of the variance of in a trait with an additive heritability of 0.73 (the approximate mean of all heritabilities listed in Supplementary Table S1 online). Nonetheless, our findings agree with those of much larger studies that have failed to find significant associations between obesity-related traits and FTO variants before age 5.<SEP>In summary, we found that the genetic and residual environmental influences on infant growth significantly change from birth to 3 years, with maximum heritability occurring at age 6–9 months for weight, relative weight, and weight velocity, and at 18 and 30 months for length and length velocity. Common FTO variants do not account for a significant proportion of this heritability. Due to the lower heritability estimates of weight, relative weight, and weight velocity at birth, environmental interventions to reduce rapid infant weight gain should theoretically have their greatest potential for effect before the age of 6–9 months, when heritabilities become high.",
                  "26606",
                  "['OBWCCE']"
                ],
                [
                  "9",
                  "30442880",
                  "Background<SEP>With improvement of economy and lifestyle, the incidence of obesity has been increasing year by year and has become a problem that seriously endangers human health, burdens people’s lives, and limits social development. Over 1 billion adults in the world are overweight and nearly 300 million adults are diagnosed as obese. Obesity can not only cause hypertension, diabetes, and other metabolic diseases, but also has a serious effect on psychology, resulting in low self-esteem, eccentricity, and other behavioral disorders.<SEP>Obesity is a disease caused by many factors, such as environment and genetics. Some studies found that FTO is involved in the pathogenesis of obesity. The human FTO gene is located on chromosome 16 (16q12.2) and its mRNA is mainly expressed in the hypothalamus, which controls energy balance. Therefore, the FTO gene may affect the occurrence of obesity through neuro-modulation, food-related genes, and eating behaviors. A number of studies demonstrated that there is a close correlation between FTO single-nucleotide polymorphism (SNP) and the occurrence of obesity. However, the polymorphic sites are different in different countries, nationalities, and age groups. The FTO gene rs9939609 and rs1421085 genotypes have been identified in numerous large-scale trials that were related to obesity, BMI, and diabetes. Therefore, the present study selected these 2 polymorphic sites for investigation.<SEP>As an ethnic minority area, Hohhot, Inner Mongolia has a large ethnic Mongolian population, as well as many ethnic Han people, with different lifestyles and diets. A previous study demonstrated that the age-standardized prevalence of obesity, denoted by the international BMI cutoff values, in men and women between 2005 and 2013 increased from 10.8% to 17.6% and from 18.9% to 26.4%, respectively. Using Asian-specific BMI cutoff values for men and women, the age-standardized prevalence of obesity between 2005 and 2013 increased from 20.0% to 32.8% and 33.4% to 43.7%, respectively, which was significantly higher than the prevalence of obesity worldwide (13% of the world’s adult population, including 11% of men and 15% of women were obese in 2016). However, the distribution of FTO alleles in ethnic Mongolians and its correlation with obesity and metabolic syndrome remains poorly understood. This study investigated the relationship of the FTO gene in Mongolians with obesity and related metabolic syndrome, aiming to more effectively manage Mongolian obesity.",
                  "Material and Methods<SEP>Subjects<SEP>According to the diagnostic criteria of the 2003 edition of “Guidelines for Prevention and Control of Overweight and Obese Adults in China”, 300 Mongolian overweight and obese patients from January 2017 to July 2017 at the Medical Examination Center of the Affiliated Hospital of Inner Mongolia Medical University were selected, including 150 males and 150 females, with a mean age of 35.82±6.54 years old. Another 200 non-overweight or obese subjects were enrolled as the control group, including 100 males and females, with an average age of 36.07±6.23 years.<SEP>General information<SEP>Data on gender, age, medical history, family history, smoking, drinking, diet, and exercise of all the participants were collected. Blood pressure, height, weight, WC, and HC were measured to calculate BMI=body weight (Kg)/height (m2); WHR=WC/HC. A total of 4 ml EDTA anticoagulation venous blood was collected from each subject. FPG was measured by glucose oxidase method. TG, TC, HDL-C, and LDL-C were detected by automatic biochemical analyzer (AU 600 Olympus, Japan).<SEP>DNA extraction<SEP>A total of 300 μL anticoagulant blood was used to extract DNA using the blood DNA extraction kit (Tiangen Biochemical Technology Co., Ltd., Beijing), to which we added 750 μL of cell lysate CLA and centrifuged the mixture at 12 000 rpm for 1 min. Then, we added 198.5 μL FGA and 1.5 μL Proteinase K. Next, we added 200 μL isopropanol and centrifuged the mixture at 12 000 rpm for 5 min. Then, we added 300 μL of 70% ethanol and centrifuged it at 12 000 rpm for 2 min. Finally, the dry DNA sediment was dissolved in 50 μL buffer TB. DNA concentration and purity were measured on a NanoDrop 2000 UV spectrophotometer (Thermo, USA) and the sample was stored at −20°C.<SEP>PCR amplification<SEP>PCR reaction with 2 primers was performed to distinguish different genotypes. The PCR reaction system contained 12.5 μL 2×KOD buffer, 4 μL dNTPs, 0.5 μL F, 0.5 μL DNA R, 2 μL DNA, and 5.5 μL ddH2O. PCR amplification was performed under the condition of pre-denaturation at 94°C for 2 min, followed by 16 cycles of 94°C for 30 s, 60–52°C for 30 s each cycle (−0.5°C), and 72°C for 30 s; followed by 24 cycles of 94°C for 30 s, 55°C for 30 s, and 72°C for 30 s, and 72°C for 10 min. Primer 5.0 was used to design primers. Primer sequences were:<SEP>609F: 5′CTGTATCTTTTGGCAGATCAG;<SEP>609aR: AGACTATCCAAGTGCATCACT3′;<SEP>609tR: 5′AGACTATCCAAGTGCATCACA3′;<SEP>085F: 5′ACTGTCTCTAAGCCCAACAAAC3′;<SEP>085tR: 5′ATTCTCATCAGACACTTAATCAATA3′;<SEP>085cR: 5′ATTCTCATCAGACACTTAATCAATG3′.<SEP>Agarose gel electrophoresis<SEP>A total of 5 μL of PCR product was mixed with 1 μL of 6×loading buffer, and then we added it into a 1.5% agarose gel and incubated it in a TBE electrophoresis buffer at 100 V for 40 min. The results were read by a gel electrophoresis imager and recorded radiographically. For different pairs of specific primers, different genotypes were analyzed according to the results of electrophoresis (Table 1).<SEP>Statistical analysis<SEP>All data analyses were performed using SPSS17.0 software. Measurement data are expressed as mean ± standard deviation and compared by t test. Enumeration data were depicted as the number of cases or rate and analyzed by χ2-test. Risk assessment was performed using single-factor logistic regression analysis. P<0.05 was considered as statistical significance.",
                  "Results<SEP>Comparison of general information and biochemical indicators<SEP>There was no significant difference in age or height distribution between the 2 groups. Body mass, BMI, WC, HC, WHR, systolic blood pressure (SBP), diastolic blood pressure (DBP), fasting plasma glucose (FPG), TG, TC, and LDL-C in the obese group were significantly higher than in the control group (Table 2).<SEP>FTO gene distribution frequency balance test<SEP>The frequency of FTO gene polymorphisms at 2 loci conformed to Hardy-Weinberg equilibrium (P > 0.05), indicating that the selected samples were representative (Table 3).<SEP>Comparison of allele and genotype frequencies<SEP>Rs9939609 genotype frequency and allele frequency distribution were significantly different between the 2 groups (P<0.05). TT genotype and T allele frequency in the obese group was markedly higher than in the control group. However, there was no significant difference in rs1421085 genotype frequency and allele frequency distribution between the 2 groups (P>0.05) (Tables 4, 5).<SEP>Risk analysis of FTO gene rs9939609 polymorphism in Mongolian obesity<SEP>The Mongolian FTO gene rs9939609 was analyzed by one-way logistic regression analysis with TT genotype as a control. As shown in Table 6, the risk of obesity in Mongolian individuals carrying the rs9939609 AT genotype and AA genotype was 1.312 and 1.896 times higher, respectively than that of the TT genotype.<SEP>The relationship of rs9939609 polymorphism with obesity and related metabolic indicators<SEP>The body weight, BMI, WC, HC, and WHR in individuals with rs9939609 AA and AT genotype were significantly higher than those with TT genotype. However, there was no significant difference in SBP, DBP, FPG, TG, TC, HDL-C, or LDL-C between genotype carriers and TT genotype carriers (P>0.05) (Table 7).",
                  "Discussion<SEP>Obesity is an important factor affecting physical and mental health. With increased incidence, obesity has become a global public health problem. Obesity is associated with several diseases, such as diabetes, hypertension, hyperlipidemia, and cardiovascular and cerebrovascular diseases. In this study, the body weight, BMI, WC, HC, WHR, SBP, DBP, FPG, TG, TC, and LDL-C were higher, while HDL-C was lower in the obesity group compared with controls, confirming its role in the pathogenesis of obesity. Individual genetic analysis has been demonstrated to be more effective for use in screening susceptible populations and developing individualized intervention programs.<SEP>FTO is the first candidate gene associated with obesity and was discovered by genome-wide association studies (GWAS), providing the basis for scientific management and early intervention for people at risk of obesity. Several studies found that multiple SNPs of FTO are related to metabolic syndromes, such as obesity, BMI, and diabetes. However, the functional SNPs loci in different ethnic and age groups are not consistent. Blauw et al. enrolled 6990 participants 45–65 years old in the Netherlands and observed that rs1421085-CC carriers had a higher prevalence of obesity than TT carriers, with a higher BMI of 0.56 (0.15, 0.98) kg/m2, WC of 1.25 (0.02, 2.49) cm, and overall weight of 1.21 (0.28, 2.14) kg. Wu et al. selected 401 Chinese adolescents from 14 to 18 years old and analyzed multiple SNPs of the FTO gene. They found that BMI in the rs9939609 TA+AA, rs8050136 AC+AA, rs1558902 TA+AA, and rs3751812 GT+TT carriers was significantly higher than in WT TT genotype (rs9939609: P=0.038; rs1558902: P=0.038), CC genotype (rs8050136: P=0.024), and GG genotype (rs3751812: P=0.024) carriers. However, these sites were independent of metabolic indicators, including blood glucose and lipid. Albuquerque et al. investigated Turkish children and found that rs9939609 and rs1421085 were closely related to weight, BMI, WC, and HC. However, Mustafa Solak et al. showed no significant effects of rs1421085 and rs9939609 on Turkish obesity. In addition to multiple large-scale studies demonstrating that rs1421085 and rs9939609 are associated with obesity, it was also confirmed that rs1421085 and rs9939609 can influence BMI through various factors, such as dietary behaviors, physical activity, food intake, and mental health. Przeliorz-Pyszczek et al. suggested that rs1421085 can regulate the differential expression of IRX3 and IRX5 to affect the occurrence of obesity. Therefore, the present study selected rs1421085 and rs9939609 to investigate their relationship with obesity in Mongolia.<SEP>In this study, we found that 3 genotype frequencies and allele frequencies distribution of rs9939609 in the obesity group were significantly different from in controls. The frequencies of TT genotype and T allele in the obesity group were higher than in the controls. However, the genotype frequencies of 3 genotypes and allele frequencies of Rs1421085 were comparable between the 2 groups, suggesting that rs9939609 may be related to the occurrence of obesity in ethnic Mongolians. Furthermore, we assessed the risk of rs99396093 genotype frequency and allele frequency on obesity. The risk of obesity in Mongolian individuals carrying the rs9939609 AT genotype and AA genotype was 1.312 and 1.896 times higher, respectively, than that of TT genotype. The obese risk of A allele was 1.904 times higher than that of T allele. In the UAE population, it was observed that rs9939609 was significantly associated with BMI (P=0.028), and A allele had a significant additive effect on BMI. The effect of AA genotype on BMI can be reduced by physical activity (P=0.027). Body weight, BMI, WC, HC, and WHR of rs9939609 AA and AT genotype carriers were significantly higher than those of TT genotype carriers, which was consistent with previous reports. Kim et al. found that rs9939609 polymorphism can affect dietary intake of fatty acids, of which the percentage of saturated fatty acids in total dietary energy was 12.6% higher in AA genotype than the TT genotype, and this may be one of the main causes of increasing risk of obesity in people with the AA genotype. In addition, many studies also confirmed that rs9939609 polymorphism is associated with physical activity and dietary intake and may influence the incidence of obesity through these behaviors. However, the exact mechanism by which allele A is associated with the risk of obesity in Mongolians remains unclear and requires further investigations.<SEP>This study found that rs9939609 polymorphism was significantly associated with the occurrence of obesity in ethnic Mongolians, which may occur through regulation of energy metabolism. In addition, this study only selected a small part of the population in Hohhot, but not Mongolian people in other regions, and this is a certain bias in the sample selection. Therefore, large-cohort clinical studies are required to confirm this finding.",
                  "12810",
                  "['OBWCCE']"
                ],
                [
                  "10",
                  "34836030",
                  "1. Introduction<SEP>Asians with a high body mass index (BMI) have a higher body fat mass than Caucasians. Asians with higher adiposity have a greater risk of metabolic diseases than Caucasians with the same adiposity level. Accordingly, the World Health Organization defines obesity among Asians as a BMI of 25 kg/m2 or more for both genders and as a body fat of 25% and 30% for men and women, respectively.<SEP>Obesity is a multifactorial disorder including genetic, gut microbial, and environmental factors that influence energy balance. Although monogenic variants are involved in severe obesity, they are rare, and approximately 500 genetic variants of approximately 400 genes have been revealed to be associated with obesity risk at p < 5 × 10−8. Most studies have explored the effect of an individual genetic variant on obesity risk. A polygenic model provides the cumulative genetic impact on obesity risk compared to individual genetic variants when the selected genes have gene−gene and SNP−SNP interactions with the pathways of biological relevance to obesity. Polygenetic risk scores (PRS) calculated by a sum of the number of risk alleles of each SNP is informative for examining the polygenetic impact on obesity risk. Fat mass and obesity-associated protein (FTO), melanocortin 4 receptor (MC4R), and brain-derived neurotrophic factor (BDNF) genetic variants have been reported to be strongly associated with obesity risk in a genome-wide association study (GWAS) and meta-analysis studies of children and adults of different ethnicities including Europeans, Americans, Asians, and Africans. <SEP>In the protein−protein interaction network analysis, MC4R, one of the hub genes related to obesity, is highly interconnected with gastric inhibitory polypeptide receptor (GIPR), luteinizing hormone/choriogonadotropin receptor (LHCGR), calcitonin receptor (CALCR), adenylate cyclase (ADCY)3, ADCY9, FTO, transmembrane protein 18 (TMEM18), BDNF, and potassium channel tetramerization domain containing 15 (KCTD15). These genes are related to the obesity-related signaling pathways such as the AMP kinase (AMPK), neurotrophin, and phosphoinositide 3-kinases àAkt signaling pathway. Their variants affect the obesity risk by modulating the binding affinity of obesity-related transcription factors, such as STAT3, CEBPB, TCF7L2, FTO, and GATA2, and changing the phosphorylation of proteins, such as BDNF with the rs6265 risk allele. The interaction of the genetic variants may account for a more significant proportion of the genetic impact on obesity. However, the genetic variant−genetic variant interaction of the obesity-related genes has not been highly studied.<SEP>Furthermore, obesity-related genetic variants do not always equally influence obesity risk in different people because there are complex interactions between genetic and environmental factors, behavior, and the gut microbiota, influencing the actual obesity phenotypes. Recent studies have demonstrated that an individual genetic variant and lifestyle interactions influence the risk of obesity and metabolic diseases in Europeans and Asians. For example, FTO genetic variants interact with the genetic and environmental factors that influence obesity risk. Adult Asians with the FTO_rs1421085 risk allele have a higher BMI than those with the nonrisk allele when they have no regular exercise. On the other hand, adults with the risk allele have a lower BMI when they engage in regular exercise. These observations suggest that the interaction of regular exercise with FTO_rs1421085 modulates obesity risk, indicating that genetic variants interact with lifestyles. However, there is no study to determine the interaction between the PRS of the genetic variant−genetic variant interaction and lifestyles. This study hypothesized that the PRS of the interacting genetic variants associated with obesity risk interacts with lifestyle factors in middle-aged and elderly adults. We aimed to examine the hypothesis in a large city hospital-based cohort in Korea.",
                  "2. Methods<SEP>2.1. Participants<SEP>During 2004–2013, 58,630 Korean adults aged >40 years volunteered to participate in a hospital-based city cohort study called the Korean Genome and Epidemiology Study (KoGES) organized by the Korean Centers for Disease Control and Prevention. The replicate study, used only to validate obesity-related genetic variants, was conducted in 5493 adults, aged 40–79 years, for whom Korean Chip data was available among the Ansan/Ansung cohort. People with a history of cancers, thyroid diseases, chronic kidney disease, and brain-related diseases were excluded from the participants. The urban hospital-based cohort and Ansan/Ansung cohort excluded 4802 and 691 patients, respectively. All procedures of the KoGES conformed with the Declaration of Helsinki and were approved by the Institutional Review Boards of the Korean National Institute of Health (KBP-2019-055) and Hoseo University (1041231-150811-HR-034-01). All participants provided written informed consent.<SEP>2.2. Demographic, Anthropometric, and Biochemical Information<SEP>Participants were interviewed to obtain their demographic information (e.g., age, residence area, education, income, current occupation, smoking status, alcohol intake, and regular exercise). The residence area was where the participant had lived for over 6 months when he/she enrolled in the city-based cohort study. It was categorized into six provinces, including Gyeonggi-do plus Seoul, Chungcheong-do, Gangwon-do, Jeolla-do, Gyeongsang-do plus Busan, and Jeju-Do. Household income was stratified as very low (<USD 1000/month), low (USD 1000–2000/month), intermediate (USD 2000–4000/month), or high (>USD 4000/month). Educational status was classified as less than high school, high school, or college or higher. Current job status was defined as the occupation where the person had been employed for the longest period if the job was recently changed. Current job status was classified as unemployed when a participant reported being unemployed or a housewife and employed as professional, director, office worker, service staff, salesperson, agricultural or fishery worker, labor worker. Alcohol intake was quantified from the questionnaires about alcohol drinking frequencies and amounts according to alcohol beverage types. The participants were classified as light (<100 g/week) or heavy (≥100 g/week) drinkers based on their average daily alcohol intake (Table 1). The smoking status was categorized as current, past, or never, and defined as current when >100 cigarettes had been smoked over the previous six months.<SEP>Anthropometric measurements, including body weight, height, and waist circumferences, were assessed according to standardized procedures. After an overnight fast (no food for over 12 h), a participant visited the hospital, and they switched clothes into a light gown with no shoes. Body weight, and height were measured in a standing position using a well-calibrated digital weight and height scale (Inbody, Cheonan, Korea). Waist circumference was measured taken around the abdomen at the position of two finger-widths above the umbilicus in a relaxed state with a tape measure (Stanley, New Britain, CT, USA). The BMI was calculated by dividing weight (kilograms) by height2 (meters). Blood pressure was measured using a sphygmomanometer (W.A. Baum Co., New York, NY, USA), on the right arm, in a sitting position, at heart level. The biochemical parameters were determined using plasma and serum from blood drawn after an overnight fast. Skeletal muscle mass was predicted using a machine learning approach from the Ansan/Ansung cohort data measured with an Inbody scanner (Cheonan, Korea). The skeletal muscle index was defined as skeletal muscle mass divided by the BMI. The lipid profiles and the glucose and creatinine in plasma and serum concentrations were measured using a Hitachi 7600 Automatic Analyzer (Hitachi LTD., Tokyo, Japan). WBC counts were obtained from heparin-treated blood. The blood hemoglobin A1c (HbA1c; glycated hemoglobin) concentrations were determined using a Hitachi 7600 Automatic Analyzer (Hitachi, Tokyo, Japan), and the plasma lipid profiles (total cholesterol, HDL, and triglyceride) were examined using a Hitachi 7600 Automatic Analyzer. The serum high-sensitive C-reactive protein (hs-CRP) concentrations were measured using an ELISA kit. The estimated glomerular filtration rate (eGFR) was calculated using the equation of 175 × (serum creatinine concentrations)−1.154 × (age)−0.203. In females, the eGFR was multiplied by 0.742.<SEP>2.3. Definition of Obesity and MetS<SEP>Obesity for Asians is defined as ≥25 kg/m2. In the urban hospital-based and Ansan/Ansung cohorts, 17,545 and 2756 participants belonged to high-BMI (case), respectively, and 36,283 and 5395 participants in low-BMI (control), respectively.<SEP>MetS was defined according to the 2005 revised National Cholesterol Education Program−Adult Treatment Panel III criteria for Asia. Participants with three or more of the following criteria were considered as having MetS: (1) elevated blood pressure (average systolic blood pressure ≥130 mmHg or diastolic blood pressure ≥85 mmHg) or current blood pressure medication use; (2) low HDL-C concentration (<40 mg/dL for men and <50 mg/dL for women); (3) elevated serum triglyceride concentration (≥150 mmol/L) or current anti-dyslipidemic medication use; (4) elevated fasting blood glucose concentration (≥100 mmol/L) or current use of antidiabetic medication; (5) abdominal obesity (waist circumference ≥ 90 cm for men and ≥85 cm for women).<SEP>2.4. Food and Nutrient Intake and Dietary Patterns<SEP>The food and nutrient intakes were assessed using a semi-quantitative food frequency questionnaire (SQFFQ) developed and validated during the KoGES. The usual dietary intake was estimated over the previous six months. The questionnaire requested information regarding the intake of 106 food items. Each participant completed the SQFFQ, and 23 nutrient intakes were calculated from the SQFFQ data using the Computer-Aided Nutritional Analysis Program (CAN Pro) 3.0, a nutrient database program developed by the Korean Nutrition Society.<SEP>The 106 food items included were categorized into 29 food groups. These 29 food groups were used as independent variables during factor analysis to determine the dietary patterns using the FACTOR procedure. The number of factors retained in principle component analysis was determined using the eigenvalues >1.5, and the orthogonal rotation procedure (Varimax) was applied. Dietary factor-loading values of ≥0.40 indicated significant contributions to the dietary patterns. Four distinct dietary factors were selected for the Korean dietary patterns. As integrating food intake estimated by SQFFQ, the participants were categorized into a balanced Korean diet, plant-based diet, Western-style diet, and rice-based diet. The plant-based diet was high in beans, eggs, milk, beverage, fruits, and nuts, while the Western-style diet was rich in noodles, soups, meats, processed meats, and fast foods.<SEP>2.5. Dietary Inflammatory Index (DII)<SEP>DII represents an index of dietary inflammatory potentials from individual food and nutrient intake using their dietary inflammatory weights for certain foods and nutrients (energy, 32 nutrients, four food products, four spices, and caffeine), as previously described. Since the SQFFQ did not include the intake of spices, we excluded intake of garlic, ginger, saffron, and turmeric from DII calculations. DII was calculated by multiplying the dietary inflammatory scores of the 38 food and nutrient components by daily intake and dividing the sum of the scores of 38 items by 100.<SEP>2.6. Genotyping Using a Korean Chip and Quality Control<SEP>The genotype data were provided by the Center for Genome Science at the Korea National Institute of Health. Genomic DNA was isolated from whole blood, and genotypes were determined using a Korean Chip (Affymetrix, Santa Clara, CA, USA), which was designed to examine the Korean genetic variants and included the disease-related single nucleotide polymorphisms (SNPs). The genotyping accuracy was checked by Bayesian Robust Linear Modeling using the Mahalanobis Distance Genotyping Algorithm. For genotyping analysis using this Korean chip, the genotyping accuracy, missing genotype call rate, and heterozygosity were ≥98%, <4%, and <30%, respectively; the data showed no gender bias. In addition, genetic variants that met the Hardy−Weinberg equilibrium (HWE, p > 0.05) and minor allele frequency (MAF) > 1% criterion were included.<SEP>2.7. Genetic Variants Influencing the Obesity Risk and the Best Model with SNP−SNP Interactions<SEP>The steps to select genetic variants for the best model of SNP−SNP interactions are shown in Figure 1. Genetic variants for obesity risk were explored by conducting GWAS with low-BMI (n = 36,283) and high-BMI groups (n = 17,545) in the urban hospital-based cohort (p < 0.00001). From the GWAS, 9335 genetic variants were selected at p < 5 × 10−5. Gene names of the 9335 SNPs were identified using g:Profiler (, accessed on 29 March 2021), and 367 genes (3858 SNPs) were identified. The 3415 SNPs without identified gene names were discarded since their association with obesity cannot be explained by associations with obesity-related pathways. Among the 367 genes, the 36 obesity-related genes (453 SNPs) based on the literature were selected using HuGE Navigator (, accessed on 8 April 2021). The corresponding linkage disequilibrium (LD) analyses were performed on the SNPs of the selected genes in the same chromosomes using Haploview 4.2 in PLINK. Because SNPs with high r2 values provided the same information on the genetic impact, they were not included in the GMDR. The potential genetic variants in the same chromosome were not strongly correlated in an LD analysis (r2 < 0.3).<SEP>Of the 45 potential genetic variants in the 36 obesity-related genes, ten SNPs exhibiting SNP−SNP interactions strongly associated with an obesity risk were automatically selected by generalized multifactor dimensionality reduction (GMDR) with 1–10 marker count range, 10 CVC, and exhaustive search type. GMDR provided ten potential models among different combinations of 45 genetic variants. The best SNP−SNP interaction model was selected in a sign rank test of trained balanced accuracy (TRBA) and testing balanced accuracy (TEBA) with or without adjusting for the covariates using a GMDR program and a p-value threshold of 0.05. The covariates used were age, gender, residence area, education, income, occupation, energy intake, alcohol intake, regular exercise, and smoking status. Ten-fold cross-validation was also checked for cross-validation consistency (CVC) because the sample size was larger than 1000, and 10 out of 10 in CVC met the perfect cross-validation criteria. Using the best model determined by GMDR analysis, the risk allele of each SNP in the selected best model was counted as 1. For example, when people had AA, AG, and GG of one SNP, and the A allele was the risk allele, the genetic score for the SNP was 2, 1, and 0, respectively. The PRS for the best gene−gene-interaction model was assessed by summating the number of the risk alleles (genetic score) from each selected SNP in the best gene−gene-interaction model. The PRS in the five and seven SNP models was divided into three categories according to the number of risk alleles; they were classified as low PRS, middle PRS, and high PRS when the number of risk alleles in the PRS was 0–3, 4–5, and ≥6 in the 5-SNP model and 0–5, 6–7, and ≥8 in the 7-SNP model, respectively.<SEP>2.8. Statistical Analyses<SEP>Statistical analysis was conducted using PLINK version 2.0 (, accessed on 9 March 2021) and SAS version 9.3 (SAS Institute, Cary, NC, USA). Descriptive statistics of the categorical variables (e.g., gender and smoking status, etc.) were analyzed using the frequency distributions in the low-, middle-, and high-PRS groups. The significant differences between their frequency distributions were assessed using a chi-squared test. The descriptive values of continuous variables were expressed as the means and standard deviations according to the PRS categories. The significance of the differences among the PRS groups was analyzed using a one-way analysis of variance (ANOVA) to adjust covariates. Finally, multiple comparisons among the PRS groups were performed using a Tukey’s test.<SEP>The associations among the PRSs were obtained using the best model, and the obesity risk was examined using multivariate logistic regression analysis with an adjustment for covariates. The odds ratios (ORs) and 95% confidence intervals (CI) were calculated against an index reference: low PRS. Multivariate logistic regression analysis was performed using two adjusted models. The covariates of model 1 included gender, age, residence area, education, and income, while those of model 2 contained the covariates in model 1 plus smoking status, drinking amount, daily energy intake, and regular exercise. The adjusted ORs and 95% CI were calculated for obesity risk according to PRS.<SEP>The participants were categorized into higher and lower intake groups using the classification criterion described above. A multivariate interaction model was used to examine the interactions between PRS and lifestyles and demographic parameters after adjusting for the covariates. p values < 0.05 were considered significant. ",
                  "3. Results<SEP>3.1. General and Demographic Characteristics of the Participants According to Gender and Obesity<SEP>The men were older than the women, while the obese women were older than the nonobese women (p < 0.001). Obese women were shorter than nonobese women, but there was no height difference linked to obesity in men (p < 0.001). The average BMIs of the nonobese and obese groups were approximately 22.5 and 27.1 kg/m2, respectively, in both gender groups (Table 1). The waist circumferences were higher in the obese group than the nonobese group in both genders, but men in the obese and nonobese groups had greater waist circumferences than women in the corresponding weight groups. Interestingly, the skeletal muscle and fat mass were higher in the obese group than the nonobese group for both genders (p < 0.05; Table 1). On the other hand, the adjusted ORs were much higher for waist circumferences (OR = 18.3 and 95% CI: 16.5–20.3) and fat mass (OR = 20.71 and 95% CI = 17.83–24.16) for obesity risk than the skeletal muscle mass. The initial menstrual age was younger in the obese group than in the nonobese group, while there was no significant difference in menopausal age in the obese and nonobese groups (Table 1).<SEP>The incidence of metabolic syndrome was much higher in the obese groups in both genders, and the risk of metabolic syndrome increased 5.86-fold in the obese group (p < 0.001; Table 1). The components of metabolic syndrome were higher in the obese group than the nonobese group in both genders. Serum total cholesterol and LDL concentrations were much higher in women than men, but the serum triglyceride concentrations were higher in men than women (Table 1). The serum HDL concentrations were higher in women than men. In the combined lipid profiles, women had as many lipid disturbances as men. Furthermore, obese women exhibited higher serum hs-CRP concentrations than nonobese women, but this was not observed in men (Table 1). Blood pressure, including systolic blood pressure (SBP) and diastolic blood pressure (DBP), was higher in the obese group than the nonobese group, and men showed higher blood pressure than women. Women had much higher eGFR than men, and eGFR was lower in the obese group than in the nonobese group. The serum alanine aminotransferase (ALT) and aspartate aminotransferase (AST) concentrations were higher in the obese groups than the nonobese groups in both genders. Obesity increased the odds ratio for higher AST and ALT concentrations by 2.0 and 2.7-fold, respectively (Table 1). <SEP>3.2. Lifestyle Characteristics of the Participants According to Genders and Obesity<SEP>The daily energy intake based on the estimated energy intake was higher in the obese groups than the nonobese group in both genders, and it increased the obesity risk by 1.24-fold. The protein intake also elevated the obesity risk (p < 0.05; Table 2) based on the daily energy intake. In women, the carbohydrate intake was higher in the obesity group than the nonobese group, and the association of the carbohydrate intake with the obesity risk was marginally significant (p = 0.0508). Fat, including saturated, monounsaturated, polyunsaturated fatty acids, and cholesterol intake, were not associated with the obesity risk (Table 2). Their intake was slightly higher in the obese group than the nonobese group in men, but their intakes showed an opposite tendency in women. DII was similar in the obese and nonobese groups, but DII was higher in men than women (Table 2). Interestingly, the fried-food intake was higher in the obese group than the nonobese group in both genders, but the intake of sugar-containing foods was higher in the nonobese groups than the obese group in women, but not men (Table 2). Obese men had a higher intake of a Korean balanced diet and Western-style diet than the nonobese men, while there was no difference in the plant-based diet and rice-based diet intake in obese and nonobese men (Table 2). Obese women had a lower plant-based diet and a higher Korean balanced diet than nonobese women. There was no significant association of dietary patterns with obesity risk (Table 2).<SEP>Alcohol intake was much higher in the obese group than the nonobese group in both genders, particularly in men (Table 2). Alcohol intake was positively associated with the obesity risk. Fewer current smokers belonged to the obese group than the nonobese group in men but not women (Table 2). Regular exercise lowered the incidence of obesity in women but not men. Regular exercise was inversely associated with the risk of obesity (Table 2). <SEP>3.3. Genetic Variants Related to the Obesity Risk and the Best Model with SNP-SNP Interaction<SEP>Ten genes with SNPs that affect the obesity risk were chosen to examine genetic variant−genetic variant interactions in the GMDR in the urban hospital-based cohort (Table 3). The selected genetic variants involved in the obesity risk were SEC16 Homolog B (SEC16B, endoplasmic reticulum export factor) rs543874, DnaJ heat shock protein family (Hsp40) member C27 (DNAJC27)_rs713586, CDK5 regulatory subunit-associated protein 1-like 1 (CDKAL1)_rs9356744, transcription factor AP-2 Beta (TFAP2B)_rs2206277, BDNF_rs6265, myosin light chain-2 (MYL2)_rs3782889, olfactomedin-4 (OLFM4)_rs9568856, FTO_rs1421085, MC4R_rs6567160, and GIPR_rs1444988703 (Table 3). These genes were strongly connected, and FTO and MC4R acted as hub genes that affect the risk of obesity. All of the genes were involved in either energy intake or energy expenditure by modulating transcription factors. These SNPs satisfied the MAF (>0.01) and HWE (p > 0.05) criteria (Table 2). Some SNPs had an inverse association with obesity risk, while the other SNPs were positively associated with the obesity risk. The p values of most selected SNPs were higher than 5×10−7, but rs3782889_MYL2 and rs9356744_CDKAL1 did not meet the statistical criteria (Table 3). These SNPs had similar associations with the obesity risk, and the statistical significance was higher in the Ansan/Ansung cohort (Table 3), but they were not ultimately included in the best model.<SEP>Among the genetic variants selected, the models including 5, 7, 8, 9, and 10 SNPs met the criteria of TREB and CVC for the best model to explain the SNP−SNP interactions that contribute to obesity risk. The five models exhibited significant interactions among the genetic variants that influenced the obesity risk (Table 4). The 5-SNP model included SEC16B_rs543874, DNAJC27_rs713586, BDNF_rs6265, MC4R_rs6567160, and GIPR_rs1444988703. The genetic variants in model 5 plus CDKAL1_rs9356744 and OLFM4_rs9568856 (model 7) also exhibited significant interactions (Table 4). The models that added the remainder of the SNPs to model 7 also had significant interactions for obesity risk. After adjusting for covariates, the models included five, seven, eight, nine, and ten genetic variants that met the TRBA, TEBA, and CVC criteria, including gender, age, resident area, regular exercise, and smoking status. However, since all of these models have similar statistical significance; therefore model 5, which included the smallest number of variables, was selected.<SEP>Model 5 resulted in participants having PRS scores ranging from 0–9 risk alleles. The results show that BMI showed an increasing trend with PRS, and the participants with PRS scores of 8 and 9 had a sharp increment in BMI (Figure 2A). Although there was a positive association between PRS and BMI, some severely obese and underweight participants were distributed among the top deciles of PRS (8 and 9) to the bottom decile (0). However, obese (25–30 kg/m2) and severely obese (>30 kg/m2) persons had much higher representations in the middle deciles (2–7) than normal (18.5–23 kg/m2 for men and 18.5–22 kg/m2 for women), overweight (23–25 kg/m2 for men and 22–25 kg/m2 for women), and underweight (Figure 2B). In the other stratification, participants were assigned to low PRS (0–3), medium PRS (4–5), and high PRS (≥6) (Figure 2C). The PRS stratification showed a similar pattern of the top, middle, and bottom deciles, but the percentage of overweight participants in the three PRS groups was not significantly different.<SEP>The high PRS of the 5-SNP model was positively associated with the obesity risk: by 1.626 (1.474–1.801) and 1.629 (1.475–1.798) times after adjusting for covariates 1 and 2, respectively (Figure 2D). The high PRS with seven SNPs also showed a positive association with the obesity risk, but the ORs of the 7-SNP model were slightly smaller than those in the 5-SNP model (Figure 2D).<SEP>The PRS was significantly associated with the BMI: 1.43-fold and 1.55-fold in men and women, respectively, after adjusting for the covariates (Table 5). On the other hand, PRS was not related to waist circumference and SMI after adjusting for the covariates, including BMI. Interestingly, the fat mass was positively associated with fat by 1.46-fold only in men but not women after adjusting for covariates, including the BMI (Table 5). In addition, PRS was not related to menarche and menopausal age in women after adjusting for the covariates. <SEP>3.4. Interaction of PRS and Nutrient Intake in Obesity Risk<SEP>Interestingly, initial menstruation age interacted with PRS for the obesity risk. In participants with an early menarche age, the BMI was higher than those with late menarche age. The impact of PRS was much higher in the participants with early menstruation than those with late menstruation (p = 0.027; Figure 3A). The adjusted ORs and 95% CI were 2.38 and 1.608–2.841 in the participants with early menstruation and 1.562 and 1.405–1.738 in those with late menstruation, respectively (Table 6).<SEP>A plant-based diet intake showed an interaction with the PRS to influence the obesity risk (Table 6). In a low plant-based diet intake, the genetic impact was lower for BMI, whereas the participants with a high PRS had a higher BMI than those with a low PRS with a high plant-based dietary intake (Figure 3B). Adults with a high PRS were protected against the obesity risk in a high plant-based diet (Table 6). The nutrient intake and other dietary patterns did not show any interactions with the PRS to affect the obesity risk.<SEP>Fried-food intake exhibited an interaction with the PRS (Table 6). The high fried-food intake participants had a higher BMI, but the PRS impact was offset in the participants with a high fried-food intake (Figure 3C). Thus, fried-food intake should be avoided to protect against obesity, even though a high PRS did not significantly exacerbate the obesogenic effects of fried-food intake (p = 0.035).",
                  "4. Discussion<SEP>GWAS studies have previously identified genetic variants associated with obesity risk, but few studies have been conducted to identify the genetic variant−genetic variants and genetic variant−lifestyle interactions. The present study showed the participants with high PRS of appetite-related genes, mainly MC4R pathway-related genes, including SEC16B_ rs543874, BDNF_ rs6265 DNAJC27_ rs713586, MC4R_rs17782313, GIPR_ rs1444988703, increased the obesity risk by 1.6-fold. The participants with early menarche age, low plant-based diet, and high fried-food intake had a higher PRS impact on the obesity risk than those with late menarche age, high plant-based diet, and low fried-food intake. Therefore, the participants with high PRS, especially with early menstruation, could be at obesity risk, and they should consume a plant-based diet with fewer fried foods to prevent obesity risk.<SEP>The arcuate nucleus of the hypothalamus, a central organ for regulating energy balance, has a range of receptors for appetite-regulating hormones and neurotransmitters. Leptin, insulin, ghrelin, cholecystokinin, and glucagon-like peptide-1 modulate the activation of orexigenic neurons and anorexigenic neurons of the hypothalamus. When the hunger center is activated, food intake increases and energy expenditure decreases. The leptin−melanocortin signaling system is the critical appetite regulatory system: leptin activates proopiomelanocortin, an α-melanocyte-stimulating hormone that activates the melanocortin type 3 (MC3R) and MC4R. MC4R stimulates the anorexigenic effectors and suppresses the orexigenic effectors. Therefore, the MC4R-related pathway is involved in obesity risk.<SEP>The present study shows that MC4R mutations are reported to develop obesity ranging from 0.5–5.8%. A GWAS study in Danish, Icelandic, Dutch, European Americans, and African American adults has shown that SNPs in the FTO, MC4R, BDNF, and SH2B1 genes are highly associated with obesity risk via modulating leptin secretion to induce leptin resistance in different ethnicities. In Chinese people, SEC16B is involved in the serum leptin concentration connected with FTO, MC4R, and BDNF involved in the appetite regulation in the hypothalamus. SEC16B is expressed in various tissues and is also involved in transporting appetite-regulatory peptides, including neuropeptide Y and proopiomelanocortin. It acts in the adipose tissues to secrete obesity-related molecules that regulate fat mass. Adults with the risk alleles of MC4R_rs17782313 have higher ghrelin concentrations, and more than 50% have severe binge eating issues. The genes and genetic variants were associated with high BMI in middle-aged adults and the elderly. Furthermore, afamelanotide stimulates BDNF expression in the brain by activating MC4R to mediate neurogenesis and cognitive function in an Alzheimer’s disease animal model. The present study showed that MC4R_rs17782313 and BDNF_rs6265 were included in the best genetic model for obesity. The two SNPs might interact with each other to influence the obesity risk through modulating appetite.<SEP>This study demonstrates that genetic factors can interact with environmental factors in obesity development, and obesity can be prevented with high compliance using genetic and environmental interaction results. The association of fat and carbohydrate intake with obesity remains controversial, although the daily energy intake was positively associated with obesity risk. Furthermore, food intake has shown controversial results for obesity risk. The present study reported that DII, a dietary inflammatory index, and sugar-containing food intake were not involved in the obesity risk, while fried-food intake was positively associated. In contrast, the serum hs-CRP concentrations have a positive association with obesity. Therefore, obesity may be associated with inflammation, but the DII itself may not fully represent the inflammation status in the present study.<SEP>Dietary patterns have been reported to influence obesity risk. A plant-based diet was inversely associated with the obesity risk, whereas a Western-style diet had a positive association in the present study. In the present study, the participants with a high intake of plant-based foods had a higher energy intake with a high intake of fat and low intakes of carbohydrates. Thus, the intake of a plant-based diet might reduce the obesity risk despite the higher energy intake. The previous study has also shown that a plant-based diet is inversely related to liver fat deposition in children aged 6 to 15. These results suggest that the reduced obesity risk may be related to the gut microbiome because a high plant-based diet had a higher dietary fiber intake, but a Western diet had a lower fiber intake. Previous studies reported that the participants with vegetarian-type diets have more of the Prevotella enterotype in the Western countries, whereas among Koreans, those with a rice-based diet have a higher level of Prevotella. Therefore, a plant-based diet may be intermediate between a rice-based diet and a balanced Korean diet. Thus, a plant-based diet improves obesity risk potentially by promoting the gut microbiota balance.<SEP>A few studies have shown the genetic variants−diet interaction for obesity risk. Previous studies have not shown that MC4R, BDNF, SEC16B, and FTO genetic variants interacted with the food and nutrient intake, even though nutrient interactions with the genetic variants of other genes, such as GRB14, LYPLAL1, LRRN6C, and MTIF3, have been reported. In the present study, a plant-based diet and fried-food intake interacted with a high PRS, but other lifestyle-related factors did not interact with the PRS. The fried-food intake increased BMI, but the PRS impact was attenuated in the high intake group of fried foods, even though BMI was higher in the high intake group of fried food in the adults regardless of PRS. A high intake of fried foods has been shown to be related to hypertension, cardiovascular disease, and cancer risk. Therefore, people with high PRS may have a higher risk of obesity and other metabolic disorders, and they need to consume less fried foods to reduce obesity risk.<SEP>The merit of this study was to show that the PRS of the appetite-related genetic variants increased obesity risk, and a high PRS had a higher impact on obesity with an early menarche age, high fried-food intake, and low plant-based diet intake. This result can be used as a basis of personalized nutrition for obesity risk. This study had some limitations. First, this was a case-control study that did not show a causal relationship. Second, although the results were analyzed from a large cohort, this study was not conducted with a survey design. Lifestyles and nutrient intake were self-reported, and they might include some biases. Finally, the PRS model employed in this study probably underestimates the true genetic impact on obesity since a limited number of genes were included. This model was clinically used as a tool in personalized nutrition and only included well-described SNPs associated with known metabolic pathways. Future research should be conducted with a total PRS that evaluates the complete impact of genetics on obesity among Koreans.<SEP>In conclusion, a PRS model using SEC16B_rs543874, DNAJC27_rs713586, BDNF_rs6265, MC4R_ rs6567160, and GIPR_rs1444988703 had gene−gene interactions which elevated the obesity risk. In addition, the PRS interacted with the menarche age in adult women with high PRS: those who had an early menarche age showed an elevated obesity susceptibility in later life. PRS also interacted with a plant-based diet and fried-food intake. Therefore, people with high PRS, especially women with early menarche, need to consume a more plant-based diet with fewer fried foods to decrease their obesity risk. These results can be applied to personalized nutrition for protecting against obesity from childhood.<SEP>Publisher’s Note: MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations.",
                  "36342",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "11",
                  "31929364",
                  "Introduction<SEP>There is a significant relationship between overweight/obesity and glycemic status, so that weight loss can improve fasting glucose. The hypocaloric diet regardless of its severity and macronutrient distribution through weight loss can improve insulin resistance and glycemic status. However, insulin resistance has multiple causes and is determined by the interaction between genetic and environmental factors. The individual responses to diet therapy can be influenced by individual genetic background.<SEP>Common polymorphisms of the fat mass and obesity-associated gene (FTO) have been linked obesity and glycemic traits. In addition, replicable evidence has shown that genetic variants of FTO may interact with effective low-calorie dietary interventions on glycemic status. Mainly, studies’ results indicated that overweight and obese individuals with TA/AA genotype of FTO have greater improvement in response to diet/lifestyle interventions.<SEP>To the best of our knowledge, few studies have been published trying to summarize the gene-diet interaction of FTO gene on glycemic status. Therefore, in the present study, we conducted this systematic review and meta-analysis to investigate the gene-diet interaction of FTO-rs9939609 gene variant and hypocaloric diets on insulin resistance and glycemic control.",
                  "Methods<SEP>This research was conducted following the guidelines and the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) statement for reporting systematic reviews and meta-analysis. Due to the study type, ethical approval was not necessary according to local legislation.<SEP>Search strategy<SEP>Two independent reviewers (HR and AAR) searched databases including the Cochrane Central Register of Controlled Trials, PubMed, Scopus, ISI Web of Science, and Google scholar for clinical trials that investigated the gene-diet interaction of FTO-rs9939609 gene variant and hypocaloric diet on glycemic control in overweight and obese adults. The search included all studies published as original full-text articles covering a period up to December 2018. The literature search was conducted using the following keywords and medical subject heading (MeSH) terms in any possible combination: FTO, obesity-associated gene, obesity-risk allele, fasting blood glucose, fasting blood sugar (FBS), glucose, sugar, insulin, homeostasis model assessment (HOMA), glucose intolerance, diabetes, hemoglobin A1c, type 2 diabetes mellitus, glycemic control, and clinical trial. No restriction was applied to publication year, and all studies published in English were included. The reference lists of included studies were investigated to identify any additional relevant studies. The papers were selected by title and abstract, then full texts of all potentially relevant studies were retrieved by two reviewers separately, and potentially relevant studies were identified.<SEP>Study selection<SEP>After the relevance of a study was confirmed, publication's full texts were reviewed, and those that fulfilled the eligibility criteria were included. The following eligibility criteria were applied: (1) published in English; (2) using hypocaloric diets; and (3) reporting FBS, insulin and HOMA of insulin resistance (HOMA-IR) as the outcome of the study. Following studies were excluded: (1) articles without full-text availability, non-English, ecological study, qualitative study, opinion pieces, conference abstracts, review articles, and editorials; and (2) reporting unrelated data.<SEP>Data extraction<SEP>The data were extracted independently by two reviewers (HR and AAR), and in the event of disagreement, a decision being made after HT cross-examined doubtful data. Studies characteristics including first author's name, publication year, country, study design, quality score, sample size, type of diet, study duration, participant's gender, age, and health status were extracted.<SEP>Assessment of risk of bias<SEP>The quality of the studies was evaluated by two separate reviewers according to the Jadad score. The Jadad score considers randomization, blinding, description of withdrawals and dropouts. Each study was scored between 0 and 5; higher numbers represent better quality.<SEP>Statistical analysis<SEP>The effect size, estimated as the mean difference (MD), was used to perform the fixed method meta-analysis. In case of significant heterogeneity between studies, a random-effects meta-analysis was carried out. Heterogeneity was evaluated using the I2 index and Cochrane's Q test. Heterogeneity was considered low if I2 < 30%, moderate if I2 = 30% to 75%, and high if I2 > 75%. Subgroup analyses were performed according to study quality (low or high), and randomization (yes or no) to identify the potential sources of heterogeneity. In addition, sensitivity analysis and meta-regression were performed to further investigation on heterogeneity sources. Begg's rank correlation, Egger's linear regression, and funnel plots were used to examine for the presence of publication bias. All analyses were carried out using Stata, version 14 SE (Stata Crop, College Station, TX, USA). P-values <0.05 were considered statistically significant.",
                  "Results<SEP>Characteristics of the studies<SEP>As shown in Figure 1, the early electronic search resulted in 1434 studies, after duplicate removal. Following the title and abstract screen, 1381 studies were excluded due to reporting unrelated data, being review articles, and not being written in English. Overall, 53 studies were evaluated for eligibility, and 44 studies were excluded for the following reasons: did not report FBS, serum insulin or HOMA-IR as study outcomes, did not provide enough data, was not conducted in clinical trial design, or was published as study protocol. Nine studies met the inclusion criteria for the meta-analysis.Table 1 summarizes the characteristics of all studies that were included in the systematic review. Studies were conducted in Spain, Japan, and the United States. Also one of the trials was a multi-center study including eight clinical centers in seven European countries (Sweden, Denmark, United Kingdom, The Netherlands, Czech Republic, France, and Spain). Most of the studies recruited both male and female genders, while two studies were conducted only in females. Duration of the intervention ranged from 12 to 104 weeks. All studies recruited overweight or obese subjects into the study. According to Jadad scores, only one study was classified as high-quality papers (score ≥3) and the other studies were classified as low quality due to issues regarding blinding in prescribing diets.<SEP>Meta-analysis of the FBS<SEP>Overall, nine studies provided 13 effect sizes, including 1799 subjects with AA/AT genotype and 921 subjects with TT genotype, regarding the effect of hypocaloric diet on the FBS. According to the meta-analysis, there is no difference between AA/AT and TT genotypes in the effect of hypocaloric diet on FBS (weighted MD [WMD] = 0.01, 95% confidence interval [CI]: –1.08, 1.10; P = 0.984) [Figure 2]. A moderate level of heterogeneity was observed among the studies (I2 = 42.3%, P = 0.054). The subgroup analysis showed no heterogeneity among studies with randomization (n = 7, I2 = 0.0%, P = 0.637); however, in studies without randomization there was a moderate heterogeneity (n = 6, I2 = 47.8%, P = 0.088). Subgroup analysis did not suggest study quality as a source of heterogeneity [Table 2]. Sensitivity analysis did not provide further information. According to meta-regression, randomization, study duration, participants’ age, and study quality were not found to be associated with the relationship between A/T genotype and FBS following a hypocaloric diet [Table 3]. Visual inspection of the funnel plot [Figure 3A] demonstrated no publication bias of trials that investigate the effect of hypocaloric diet on the FBS according to the rs9939609 genotype (Egger's test, P = 0.073; Begg's test, P = 0.127).<SEP>Meta-analysis of the serum insulin<SEP>A total of 11 effect sizes (n = 699 AA/AT and 299 TT) were reported considering the effect of the rs9939609 genotype on serum levels of insulin following hypocaloric diet. As shown in Figure 4, there was no significant difference between AA/AT and TT genotype in serum insulin after intervention (WMD = 0.20, 95% CI: –0.86, 1.26; P = 0.707) [Figure 4]. Moderate level of heterogeneity was observed among the studies (I2 = 71.7%, P < 0.001). The subgroup analysis according to the study quality failed to identify source of heterogeneity. The subgroup analysis of randomization (yes or no) suggested that heterogeneity is significant in trials that performed randomization (n = 7, I2 = 67.8%, P = 0.005). Also, after hypocaloric diet, there was a significant increase in serum insulin in the AA/AT group compared to the TT group in the studies with randomization (WMD = 1.17, 95% CI: 0.02, 2.33; P = 0.046), while in the studies without randomization lower serum insulin was observed in the AA/AT group (WMD = –1.71, 95% CI: –2.81, –0.62; P = 0.002). However, meta-regression did not find randomization, study duration, participants’ age, and study quality as sources of heterogeneity [Table 3]. Sensitivity analysis suggests no significant change in the results following the exclusion of any study. The funnel plot [Figure 3B] demonstrated no publication bias of trials in evaluating the impact of rs9939609 genotype on serum insulin following hypocaloric diet intervention (Egger's test, P = 0.952; Begg's test, P = 0.755).<SEP>Meta-analysis of the HOMA-IR<SEP>In total, seven studies (11 effect sizes; n = 699 AA/AT and 299 TT) investigated the role of rs9939609 genotype in the effect of hypocaloric diet on insulin resistance, assessed by HOMA-IR. Although subjects in AA/AT group had lower insulin resistance compared to TT genotype following intervention, this association was not statistically significant (WMD = –0.39, 95% CI: –0.94, 0.16, P = 0.167) [Figure 5]. There was a high level of heterogeneity among the studies (I2 = 91.8%, P < 0.001). According to the subgroup analysis, studies with randomization had a high heterogeneity level (I2 = 94.7%, P < 0.001), while the heterogeneity was not evident among studies without randomization (I2 = 0.0%, P = 0.394). Study quality was not identified as a source of heterogeneity in the subgroup analysis. One-by-one exclusion of the studies in the sensitivity analysis did not change the results. The heterogeneity among studies was not explained by randomization, study duration, participants’ age, and study quality in the meta-regression [Table 3]. Publication bias was not obvious according to the funnel plot [Figure 3C], Begg's test (P = 0.533), and Egger's test (P = 0.687).",
                  "Discussion<SEP>In this systematic review and meta-analysis of trials, we found that overweight and obese individuals carrying the homozygous FTO obesity-predisposing allele (AA-AT genotype) did not have greater improvement of glycemic status than non-carriers (TT genotype) after adherence to hypocaloric diets.<SEP>The previous meta-analyses evaluated the role of FTO-rs9939609 gene variants in response to combine of weight loss intervention (diet/exercise) on anthropometric status. In this regard, the results of meta-analyses were not consistent. For example, a meta-analysis of ten studies reported that individuals with the FTO TA genotype and AA genotype may lose more weight through diet/lifestyle interventions than TT genotype. Inconsistently, Livingstone et al indicated that carriage of the FTO (rs9939609 or a proxy) minor allele respond equally to weight loss interventions (dietary, physical activity, or drug) on anthropometric indexes such as body mass index, body weight, or waist circumference.<SEP>However, few meta-analyses have been conducted to investigate interaction of FTO diet on glycemic status, and previous trials have shown inconsistent results. There are several reasons for these contradictory results. First, the difference in macronutrients (fat, protein, and carbohydrates) ratio and the type of dietary fat in the hypocaloric diets are potential reasons for discrepant findings, which can affect glycemic responses. De Luis et al reported that in obese subjects with FTO-rs9939609 gene variant, glycemic improvement was better in A carriers with a low fat hypocaloric diet than low carbohydrate hypocaloric diet. Other study showed that A allele carriers have a greater response to high polyunsaturated fat hypocaloric diet on insulin levels and HOMA index, but similar results were not reported for high monounsaturated fat hypocaloric diet. Secondly, existence of polymorphism diversity in FTO gene can cause a difference in the results of studies. Zheng et al have demonstrated that in overweight or obese adults, FTO variants rs1558902 play a role in improving insulin sensitivity by consuming high-fat weight-loss diets; on the other hand their results showed that the association between FTO-rs9939609 and changes in insulin resistance was not affected by dietary fat modifies. In the present meta-analysis to remove this confounding factor, exclusively one of the FTO gene variants (rs9939609) was evaluated.<SEP>The present study showed that different randomization schemes caused contradict on the results of serum insulin. The randomization in clinical trials is useful for minimize selection and accidental bias, balance treatment assignment in order to maximize the power of the comparison and to obtain the basis for a correct statistical interpret. In present study, due to the nature of the intervention (diet therapy) in clinical trials, the design of more studies (55%) was without randomization.<SEP>There are several limitations in our study. First, studies included in the meta-analysis varied in intervention duration, race/ethnicity, sample size, and other individuals characteristics. Second, although in our meta-analysis, studies with combined weight loss interventions (diet plus exercise/lifestyle changes) were excluded. However, due to the limited number of published articles, studies related to weight loss diets were included in the study regardless of calorie and the macronutrient ratio of diets. Thus, more studies are needed to examine gene-diet interaction of FTO-rs9939609 gene polymorphism and standard weight loss diets.<SEP>In conclusion, our results showed there was no significant difference between AA/AT and TT genotype of FTO-rs9939609 on FBS, serum insulin level and insulin resistance in response to hypocaloric diets for overweight and obese subjects. Nevertheless, Long-term randomized clinical trials with same distribution of macronutrients are highly recommended.",
                  "14735",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "12",
                  "26193063",
                  "Introduction<SEP>Binge-eating disorder (BED) is the most common eating disorder (ED) in the general population, with a lifetime prevalence of about 1.5% in adolescents and adults, and is associated with adverse physical and psychological outcomes. BED is characterized by episodes of overeating with loss of control (occurring on average once a week over 3 months) and accompanied by distress. BED has recently been recognized as a diagnostic category in DSM5. The prevalence of engaging in binge eating behaviors, both in the context of a full-criteria ED diagnosis or in the absence of other ED features, is about 10% in adults and adolescents. Binge eating is most common in individuals who are overweight/obese.<SEP>It is widely accepted that ED are partly explained by genetic factors. However, since BED was only recently recognized as an official disorder, few studies have investigated genetic risk for BED. Despite the paucity of research there is initial evidence that genetic factors influence risk for BED. Heritability estimates range between 0.39 in a mixed-gender sample of Norwegian twins and 0.45 in a female-only twin sample. Evidence from twin studies shows a moderate correlation between obesity and binge eating, which suggests that some of the same genetic factors might influence both obesity and binge eating.<SEP>Genetic association studies have investigated risk-conferring genes particularly in the dopamine, serotonin, and appetitive systems. Two studies found an association between MC4R (melanocortin 4 receptor) variants and BED, but another study failed to find an association. A more recent study of 289 youth aged between 6 and 19 years found an association between a polymorphism in the FTO locus and loss-of-control eating, which is one of the main characteristics of BED/binge eating behavior. Similarly a recent study showed an association between a weighted allelic risk score, obtained from combining the effect of 32 SNPs robustly associated with body mass index (BMI), and emotional and uncontrolled eating, behaviors highly correlated with binge eating.<SEP>Based on these studies and evidence from our studies that BED predicts overweight, we aimed to: investigate whether variants previously identified as associated with BMI were also associated with adolescent binge eating (overeating with loss of control) and test the role of a polygenic weighted allelic score [obtained by combining the 32 independent genetic variants associated with BMI from a genome-wide association study ]. We also aimed to explore whether associations varied by gender and frequency of binge eating at each time-point.",
                  "Methods<SEP>Participants<SEP>The Avon Longitudinal Study of Parents and Children (ALSPAC) is a longitudinal, population-based, prospective study of women and their children. All pregnant women living in the geographical area of Avon, UK, who were expected to deliver their baby between 1st April 1991 and 31st December 1992 were invited to take part in the study. All women gave informed and written consent. The study website contains details of all the data that are available through a fully searchable data dictionary (http://www.bris.ac.uk/alspac/researchers/data-access/data-dictionary).<SEP>The children from 14,541 pregnancies were enrolled; 13,988 children were alive at 1 year. At age 14 years 10,303 adolescents (singletons) were eligible for follow-up and were sent questionnaires; 5,958 adolescents (57.8% of those eligible) returned completed questionnaires. At age 16 years 9,660 adolescents (singletons) were sent questionnaires and 4,948 returned them (51.2%).<SEP>Ethical approval for the study was obtained from the ALSPAC Ethics and Law Committee and the Local Research Ethics Committees.<SEP>Binge eating<SEP>Binge eating was assessed using a two-part question (21). Participants were first asked about the frequency during the past year of eating a very large amount of food. Those who answered yes were directed to a follow-up question that asked whether they felt out of control during these episodes, i.e., whether they could not stop eating even if they wanted to stop. Adolescents who answered yes to both questions were classified as engaging in binge eating.<SEP>For the purpose of the study we used binge eating either at 14 or 16 years of age as an outcome. We also created an ordinal variable for frequency of bingeing reported in the year prior to assessment (never, < once a month, one to three times per month, once a week, >once a week), in order to determine whether increasing frequency of binge eating was associated with genetic risk.<SEP>Genotyping<SEP>In total, 9,912 participants were genotyped using the Illumina HumanHap550 quad genome-wide SNP genotyping platform by the Wellcome Trust Sanger Institute, Cambridge, UK, and the Laboratory Corporation of America, Burlington, NC. Individuals were excluded from analyses on the basis of excessive or minimal heterozygosity, gender mismatch, individual missingness (>3%), cryptic relatedness as measured by identity by descent (genome-wide IBD >10%) and sample duplication. Individuals were assessed for population stratification using multi-dimensional scaling modeling seeded with HapMap Phase II release 22 reference populations. Individuals of non-European ancestry were removed from further analyses. SNPs with a final call rate of <95%, minor allele frequency (MAF) <1% and evidence of departure from Hardy-Weinberg equilibrium (HWE) (P < 5 × 10−7) were also excluded from analyses. Individuals were imputed to HapMap Phase II (Build 36, release 22) using the Markov Chain Haplotyping software (MACH v.1.0.16).<SEP>From these genome-wide data the 32 SNPs previously identified in a large GWAS meta-analysis as being robustly and independently associated with BMI were selected. We also generated weighted polygenic allelic scores using these 32 SNPs. The estimated dosage of each effect allele was weighted by its regression coefficient from Speliotes et al. We calculated allelic scores using all 32 SNPs and using 31 SNPs excluding the FTO variant.<SEP>Covariates<SEP>BMI, weight (kg)/height (m)2, was obtained from measured weight and height during a face to face assessment at mean age 13.5 and mean age 15.5 years. BMI at 13.5 was used as the main covariate, if this was missing, BMI at mean age 15.5 was used (correlations between BMI at the two time-points was very high r = 0.98). Data on BMI were available on 4,048 adolescents (82.3% of those with available data on binge eating and genotype).<SEP>Data on binge eating and genotype were available on 4,360 adolescents (2,406 girls and 1,954 boys) at 14 years, and 3,663 (2,151 girls and 1,512 boys) at 16 years. Overall 4916 adolescents had genotype data and information on binge eating at either age 14 or 16 (2725 girls and 2191 boys).<SEP>Data analyses<SEP>We tested the effect of BMI associated SNPs in unadjusted and adjusted logistic regression models where binge eating was the outcome. Adjusted models included age, BMI and gender as covariates. Unadjusted logistic regression models were also conducted stratifying by gender to investigate whether the pattern of association differed between males and females. In order to increase the power of our analyses, we examined the relationship between a weighted allelic risk score and binge eating at either age in an unadjusted logistic regression model. We used effect size coefficients as reported in Speliotes et al. to weight individual SNPs. In order to assess the contribution of the FTO variant to the allelic score, two models were tested, one including all 32 SNPs and one comprising 31 SNPs and excluding the FTO variant.<SEP>Finally, we investigated the relationship between BMI associated SNPs and an ordinal indicator of binge eating (frequency-based) in age stratified adjusted (for gender and BMI) ordinal logistic regression models.<SEP>Analyses were run using STATA 12 and the R software package 2.15.2.",
                  "Results<SEP>The prevalence of binge eating in adolescence was 5.6% at age 14 years and 11.2% at age 16 years amongst adolescents included in the study (See Table1). Binge eating was more common in girls at both age 14 (7.4% of girls, 3.5% of boys) and 16 (16.1% of girls and 4.4% of boys).<SEP>Relationship between BMI variants and binge eating at either 14 or 16 years of age<SEP>Minimally adjusted analyses (adjusted for gender and age) revealed an association in the hypothesised direction between the A allele of rs1558902 (FTO) and increased risk of binge eating [odds ratio (OR) = 1.25 (95% CI: 1.10-1.41), P = 5.1 × 10−4]. This association showed little attenuation after adjusting for BMI (OR = 1.20 (95% CI: 1.05-1.38), P = 8.0 × 10−3) (Table2). There was also nominal evidence of a protective association (direction opposite to that expected) between rs10150332 (NRXN3) and binge eating in unadjusted (OR = 0.85 (95% CI: 0.72-0.99), P = 0.038), but not adjusted analyses [OR = 0.88 (95% CI: 0.74-1.04), P = 0.13].<SEP>Females (OR = 1.30, P = 3.3 × 10−4) displayed slightly stronger evidence for association at the FTO locus than males (OR = 1.23, P = 0.51), but the difference was not significant (P = 0.26) (Table3).<SEP>Amongst girls there was nominal association between rs2112347 (FLJ35779/HMGCR) and binge eating in the expected direction (OR=1.21 (95%CI: 1.04-1.40), P = 0.012) (see Table3), whilst in boys there was nominal evidence of association between rs2815752 (NEGR1) and binge eating in the expected direction (OR = 1.32 (95%CI: 1.00-1.73, P = 0.047) and rs10150332 (NRXN3) and bingeing in the direction opposite to expected (OR = 0.70 (95%CI: 0.49-1.01), P = 0.047).<SEP>Frequency of binge eating<SEP>Ordinal regression models confirmed the association between the A allele of rs1558902 (FTO) and higher binge eating frequency at age 14 years [OR = 1.21 (95%CI: 1.01-1.46), P = 0.04] and at 16 years [OR = 1.28 (95% CI: 1.10-1.48), P = 1.0 × 10−3]. In other words, the presence of the A allele increased the odds of higher binge eating frequency (from never to less than monthly, monthly, weekly, and more than weekly) by 28% for each increase in level at age 16 (see Table4).<SEP>Two SNPs showed nominal association with frequency of binge eating at either age 14 [rs3817334 (MTCH2): OR = 1.21 (95%CI: 1.01-1.46), P = 0.03] or 16 years [rs1514175 (TNNI3K): OR=1.17 (95% CI: 1.01-1.36), P = 0.04] in the expected direction; but these did not replicate across ages.<SEP>Polygenic risk score<SEP>In unadjusted analyses, the weighted polygenic risk score was strongly positively associated with adolescent binge eating at either age 14 or 16 (P = 7.9 × 10−4). This association weakened dramatically once FTO was removed from the score (P = 0.08). The same pattern was seen in females with the 32 allelic variant weighted polygenic risk score being strongly positively associated with binge eating (P = 8.6 × 10−4), but not after FTO was removed from the risk score (P = 0.08). The risk score was not strongly associated with binge eating in males (all P > 0.05).",
                  "Discussion<SEP>This is the first study to investigate the association between binge eating in adolescence and 32 SNPs that have been robustly associated with BMI, and to include the effect of a weighted allelic score of 32 SNPs associated with BMI. We investigated this association across genders, and explored the association between 32 SNPs and increasing frequency of binge eating (as a more sensitive measure of binge eating and a severity indicator).<SEP>We found a significant positive (risk-conferring) association between the A allele of the FTO SNP (rs1558902) and adolescent binge eating, independent of BMI. This association was also found when investigating binge eating as a frequency-based ordinal outcome (engaging in binge eating weekly, monthly, or less than monthly).<SEP>Analyses stratified by gender showed some evidence for gender differences suggesting the FTO rs1558902 A allele might confer increased risk of binge eating amongst girls compared to boys, although the absence of an effect on boys might also be due to lower power due to a reduced prevalence of binge eating in boys (a formal test for interaction was not significant).<SEP>A weighted allelic risk score derived from 32 variants associated with BMI showed a positive association with binge eating in unadjusted analyses, with a seemingly stronger effect in girls. Removing the rs1558902 (FTO) variant from the allelic risk score attenuated the association between the risk score and binge eating, suggesting that: (i) the FTO variant is the major variant driving the association with binge eating, and (ii) the association with bingeing is unlikely to be mediated by BMI (in which case we would also expect an allelic score of BMI SNPs to show strong association with bingeing).<SEP>The association between SNPs at the FTO locus and other EDs [anorexia (AN) and bulimia nervosa (BN)] has been examined in the literature and resulted in conflicting findings. In particular a recent study found an association between the obesity predisposing allele of the FTO variant rs9939609 and AN but not with BN (once adjusted for BMI). In contrast an earlier study did not find an association between the same SNP and AN in a smaller sample of 225 patients with AN and 1,351 controls.<SEP>The rs1558902 polymorphism is in high linkage disequilibrium with the rs9939609 FTO SNP (r2 = 0.93 in CEU HapMap 2) and a recent meta-analysis has shown similarly strong associations between each of these SNPs and overweight/obesity in children and adolescents. There is reason to believe that FTO locus SNPs may be related to binge eating. Both rs1558902 and rs9939609 have previously been associated with eating behavior, such as loss of control in adolescents, food choice and higher intake of energy-dense food in children, food responsiveness, and decreased satiety. A recent study also showed a positive association between rs1558902 (FTO) and the cognitive restraint subscale from the three factor eating questionnaire (a measure of restrained eating). Taken together these findings suggest that polymorphisms at the FTO locus might increase the risk for both binge eating and overweight/obesity via higher palatable food and energy intake and decreased satiety; or that binge eating might be a mediator on the pathway between genotype and increased BMI/obesity.<SEP>Our findings of: (a) an association between an FTO locus SNP and binge eating, that showed little reduction in strength after conditioning on BMI, (b) the lack of association with other BMI-increasing SNPS, and (c) an association between the allelic risk score and binge eating when FTO was included that attenuated markedly after excluding FTO from the allelic risk score suggests that the effect of FTO on binge eating is not mediated by BMI. Therefore our findings suggest either pleiotropic effects of variants in or near the FTO locus, or that binge eating might be on the causal pathway between genetic variants and obesity. We have shown in two large population-based cohorts (including the one under study) that BED significantly increases the odds of overweight/obesity even when taking into account baseline weight (Micali et al., submitted).<SEP>Recent findings have highlighted differences in prefrontal cortex activation in the processing of food stimuli associated with FTO polymorphisms, with lower activity (hence lower inhibitory control of eating) in this brain region post-prandially in subjects carrying the risk allele. Evidence also suggests that FTO rs9939609 AA carriers have a different brain response to food and ghrelin expression compared to TT carriers.<SEP>Thus, evidence from behavioral genetics and neuroimaging studies supports our findings that genetic variation at the FTO locus might confer risk of binge eating. In our study, the strongest associations between the FTO polymorphism and binge eating were most evident in girls; although this might be due to a lower prevalence of binge eating in boys and hence lower power to detect differences, or a true differential gender effect (although the formal test for interaction was negative).<SEP>In gender-stratified analyses some polymorphisms were nominally associated with binge eating: (e.g., a polymorphism in FLJ35779/HMGCR in girls, and a polymorphism in NEGR1 with binge eating in boys). A variant at MTCH2 was associated with frequency of binge eating at age 14 years, and a polymorphism in TNNI3K with frequency of binge eating at age 16 years. These findings might reflect type I error given the large number of statistical comparisons performed in this paper, and the rather weak P values. Future studies should aim to replicate these findings. FLJ35779/HMGCR has not, to our knowledge, been previously implicated in appetite regulation. In contrast the TNNI3K and MTCH2 obesity risk-conferring SNPs have both been shown to be positively associated with emotional eating and uncontrolled eating. Emotional eating and uncontrolled eating are traits strongly associated with binge eating, and confer risk for binge eating.<SEP>Amongst the SNPs studied we did not find an association between binge eating and variants within MC4R or RBJ/ADCY3/POMC, despite evidence from previous studies that mutations in the melanocortin 4 receptor (MC4R) and the Pro Opio Melanocortin (POMC) genes might be implicated in appetite control. Similarly, polymorphisms in the BDNF gene have been shown to be associated with ED phenotypes but were not strongly associated with binge eating in this study. This might be due to low power to detect associations in our study, or to the selection of more severe phenotypes in previous studies (mostly reliant on clinical populations).<SEP>Strengths of our study include a relatively large sample size in comparison to previous studies on genetic risk for binge eating, an investigation of binge eating as a dichotomous outcome and of as an ordinal outcome (i.e., frequency of binge eating), and a candidate gene approach based on 32 SNPs robustly associated with BMI. We also investigated a possible relationship between a weighted allelic risk score and binge eating. Limitations include the fact that data on binge eating were collected using self-report, however our questions were validated in a population-based sample and showed high reliability and validity. Secondly, this sample mostly includes subjects of European ancestry, which might limit generalizability of findings. Thirdly, despite being one of the larger genetic studies of binge eating, our study has low statistical power in some analyses, partially due to the lower prevalence of binge eating amongst boys, which might have resulted in false negatives in the associations reported. Our study has limited power to detect genetic variants associated with binge eating in adolescence. This is because of our limited sample size and the fact that genetic variants that influence complex traits like binge eating are likely to exhibit relatively small effects. For example in the case of rs1558902 (FTO) which has a risk allele frequency of 0.4, assuming 586 cases and 4331 controls (a prevalence of 13%), a heterozygote relative risk of 1.1, a multiplicative disease model (on the risk scale), we only have ∼40% power to detect an association at α = 0.05 using an allelic test of association. This calculation illustrates that our study has low to moderate power to detect association at individual variants if the effect size of these variants is small (as is likely to be the case). However, assuming that one or more variants jointly contribute to binge eating, combining the variants into an allelic score should increase our power to detect association. Whilst an allelic score comprised of BMI associated variants did show some association with binge eating, our results suggest that most of this association was due to the FTO variant, rather than the other BMI related SNPs. We were unable to carry out a replication of our findings in an independent sample within the current study; therefore our findings need replication. However, it should be noted that Tanofsky-Kraff et al. observed an association between a SNP at the FTO locus and loss-of-control eating; thus, our finding could be considered a replication.",
                  "20317",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "13",
                  "22891219",
                  "The prevalence of overweight and obesity has increased substantially in the U.S. and worldwide, and the health burden of obesity-related complications has grown accordingly. Obesity is primarily determined by both genetic and lifestyle factors, including diet, as well as their interactions. In the past few years, genome-wide association studies (GWASs) have identified a group of genetic loci associated with BMI and obesity risk. Among them, the fat mass and obesity-associated gene (FTO) locus shows the strongest effect. Accumulating evidence has suggested that this locus is involved in the hypothalamic regulation of appetite and dietary energy intake.<SEP>Recently, several studies have examined the effect of the FTO-diet interaction on body weight, but the results are not entirely consistent. Several cross-sectional studies showed that dietary factors such as low fat intake might modify the genetic effect of FTO on BMI or fat distribution. However, the gene by diet interaction was not univocally observed in randomized intervention trials, although one study found that a Mediterranean diet intervention modified the association between the FTO variant and weight changes in a population with high cardiovascular risk. These intervention trials, however, largely are limited by relatively small sample size or the short term of follow-up. In addition, animal studies have suggested that FTO might differentially affect various body compositions and fat distribution at different depots. Few studies have evaluated systematically the effect of the FTO variant on these measurements.<SEP>The POUNDS LOST Trial thus far is the largest 2-year randomized intervention trial that tested the effect of four diets varying in proportions of fat, protein, and carbohydrate on weight loss in overweight or obese subjects. By use of the data from this trial, we evaluated whether various weight-loss diets might modify the effect of the FTO variant on weight loss and long-term changes in body composition and fat distribution.",
                  "RESEARCH DESIGN AND METHODS<SEP>Study population.<SEP>The POUNDS LOST Trial was conducted from October 2004 through December 2007 at two sites as follows: Harvard School of Public Health and Brigham & Women’s Hospital in Boston, Massachusetts; and the Pennington Biomedical Research Center of Louisiana State University System, Baton Rouge, Louisiana. The design and sample collection have been described previously in detail. In brief, the study population was composed of 811 overweight or obese (BMI ranged from 25 to 40 kg/m2) participants aged 30–70 years. Major criteria for exclusion were the presence of diabetes or unstable cardiovascular disease, the use of medications that affect body weight, and insufficient motivation as assessed by interview and questionnaire. Among the 742 participants who were genotyped successfully, 61% were women, 80% were white, 15% were black, 3% were Hispanic, and 2% were Asian or other ethnic groups by self-report. The participants were assigned randomly to one of four diets constituting a two-by-two factorial design; the target percentages of energy derived from fat, protein, and carbohydrate in the four diets were 20, 15, and 65%; 20, 25, and 55%; 40, 15, and 45%; and 40, 25, and 35%. After 2 years, 645 participants (80% of total population) completed the trial. The study was approved by the human subjects committee at each institution and by a data and safety monitoring board appointed by the National Heart, Lung, and Blood Institute. All participants provided written informed consent.<SEP>Measurements.<SEP>In the morning before breakfast, body weight and waist circumference (WC) were measured on 2 days at baseline; 6, 12, and 18 months; and 2 years. BMI was calculated as weight by height squared (kg/m2). A dual-energy X-ray absorptiometry (DEXA) scan was performed on 50% of a random sample from the total study participants (n = 424), including 242 (57.1%) women, using a Hologic QDR 4500A (Waltham, MA). Total fat mass (kg), total fat-free mass (FFM; kg), whole body total percentage of fat mass (FM%) and percentage of trunk fat were obtained once at baseline, 6 months, and 2 years. Computed tomography (CT) was used in 50% of a random sample from those participants who had DEXA scans, resulting in a sample of 25% of the total participants (n = 195), including 113 (58.2%) women. Total adipose tissue (TAT) mass, visceral adipose tissue (VAT) mass, deep subcutaneous adipose tissue (DSAT) mass, and superficial adipose tissue (SAT) mass within the abdomen were measured by standard methods, once at baseline, 6 months, and 2 years. Because of radiation exposure, premenopausal women would not subject themselves to CT scans. A series of eight single-slice images were obtained every 10 cm from 2 below and 5 above the fourth and fifth lumbar vertebrae interspaces. These contiguous cross-sectional images were analyzed, and then the total volume was calculated from the individual slices. In this analysis, we only included data at baseline, 6 months, and 2 years with all the outcomes because the DEXA and CT scans were only performed at these three time points.<SEP>Genotyping.<SEP>DNA was extracted from the buffy coat fraction of centrifuged blood using the QIAmp Blood Kit (Qiagen, Chatsworth, CA). Single nucleotide polymorphism (SNP) rs1558902 was selected because it had emerged as the top variant of FTO locus for BMI and WC in recent obesity-related GWAS. The SNP was genotyped successfully in 742 of 811 total participants and 603 of 645 participants who completed the trial using the OpenArray SNP Genotyping System (BioTrove, Woburn, MA). Of the 424 participants who received DEXA scans, 391 were genotyped at baseline, and 224 participants who completed the trial were genotyped. Of the 195 participants who received CT scans, 175 were genotyped at baseline and 105 participants who completed the trial were genotyped. The genotype success rate was 99% in available DNA samples. Replicated quality control samples (10%) were included in every genotyping plate with greater than 99% concordance. The allele frequency in two major ethnic groups (white and black) was compatible with Hardy-Weinberg equilibrium (P > 0.05).<SEP>Statistical analysis.<SEP>The primary outcomes were changes in body weight and WC. Secondary outcomes were changes in body composition including total fat mass, FFM, FM% and percentage of trunk fat, and fat distribution (TAT, VAT, SAT, and DSAT). Data were pooled from the diets for the two factorial comparisons: low protein versus high protein and low fat versus high fat. Because the majority of the study population were white (80%), we also analyzed the main effects and interactions among white participants separately. The Hardy-Weinberg equilibrium and comparison of categorical variables at baseline were assessed with χ2 test. Differences in continuous variables at baseline were tested using ANCOVA, with adjustment for age, sex, and ethnicity. The main effects of genotype and diet intervention on outcome changes at 6 months and 2 years were analyzed using general linear regression models, with adjustment for covariates including age, sex, ethnicity, carbohydrate, the baseline value for the respective outcome, and baseline BMI. We excluded individuals with missing measures at each time point in the analysis. Moreover, to analyze the potential interactions between genotype and diet intervention, an interaction product term of genotype-diet was included in the models. In a secondary analysis, we used linear mixed models, with time as a repeated measurement factor, to test genetic associations with the trajectory of changes in outcomes in the participants who provided measurements at baseline, 6 months, and 2 years in each of four diet groups over the 2-year intervention by including genotype-time interaction terms. Because an additive genetic effect was reported in the original large-scale GWAS in which the SNP was identified, additive models were analyzed for genotype. All reported P values were two-sided and a P value of 0.05 was considered statistically significant. All data were analyzed with SAS version 9.1 (SAS Institute, Inc., Cary, NC).",
                  "RESULTS<SEP>Characteristics of study population.<SEP>Baseline characteristics of participants according to the FTO rs1558902 genotype are presented in Table 1. The minor allele frequency (MAF; A allele) was 0.402 in the total population. The genotype frequencies were significantly different among the sexes and ethnicities. After adjustment for age, sex, and ethnicity, all variables such as weight, BMI, WC, body composition, and fat distribution had no association with genotype at baseline. Baseline characteristics were similar among participants in the four diet groups (Supplementary Table 1). Likewise, no associations between the FTO genotypes and these measures were observed in the white participants (data not shown).<SEP>Effects of FTO rs1558902 genotype on weight and waist: overall and two-factorial analysis.<SEP>After adjustment for age, sex, ethnicity, baseline BMI, and diet groups, no main effects of the FTO rs1558902 genotype on changes in weight or WC were found in any participants at 6 months and 2 years (data not shown). We next examined the genetic effects on changes in weight and WC following a two-factorial design: low versus high fat and low versus high protein. We found that the risk allele (A) was significantly associated with a 1.51-kg greater weight loss in the high-protein group (P = 0.010), but not in the low-protein group, by the end of intervention (2 years). The changes in weight and WC were less significant at 6 months (Table 2). In subgroups treated by different proportions of dietary fat, we did not find significant genetic effects on changes in weight and WC (all P > 0.05; Supplementary Table 2).Similarly, in the white participants, we found the risk allele was associated with a 1.38-kg greater weight loss in the high-protein group at 2 years (P = 0.028), but not in other subgroups (data not shown).<SEP>The FTO rs1558902 genotype and changes in body composition by diet intervention.<SEP>Consistent with the observations of change in body weight, we found that the rs15589002 risk allele (A) was associated with greater loss of total fat, FFM, FM%, and percentage of trunk fat at 2 years in the high-protein group, but not in the low-protein group (Table 2). Tests for genotype-diet protein interaction were significant on changes in FFM and FM% (for interactions, P = 0.034 and 0.049, respectively) adjusted for age, sex, ethnicity, carbohydrate, baseline BMI, and the baseline value of body composition (Fig. 1). At 6 months, we only observed gene by protein diet interaction on changes in FFM (P = 0.008 for interaction; Fig. 1). The risk allele carriers in the high-protein group had greater loss of FFM than noncarriers, but those in the low-protein group had less loss of FFM compared with noncarriers (Table 2).<SEP>We did not find significant genetic effect and interactions between the FTO variant and dietary fat intake on changes in body composition in total participants (Supplementary Table 2 and Supplementary Fig. 1). The results in the white participants were similar (data not shown).<SEP>The FTO rs1558902 genotype and changes in fat distribution by diet intervention.<SEP>We further analyzed body fat distribution measured by CT. At 2 years, we found significant interactions between the FTO rs1558902 genotype and protein diet intervention on changes in TAT, VAT, and SAT (for interactions, P = 0.001, 0.012, and 0.002, respectively; Fig. 2). The risk allele (A) was associated with greater loss of TAT and VAT in the high-protein group but with less loss of TAT and SAT in the low-protein group (Table 2). At 6 months, gene-protein interactions were observed on changes in TAT and SAT (for interactions, P = 0.026 and 0.050, respectively; Fig. 2), and the risk allele carriers in the high-protein group had greater loss of TAT and VAT than noncarriers, but those in the low-protein group had less loss of SAT than noncarriers (Table 2).<SEP>We did not find significant genetic effects and interactions on changes in fat distribution in subgroups treated by different dietary fat components (all P > 0.05; Supplementary Table 2 and Supplementary Fig. 2).Similar results were found in the white participants (data not shown).<SEP>The trajectory of changes in body composition and fat distribution by FTO rs1558902 in response to protein diet.<SEP>In a secondary analysis, we used linear mixed models to assess the genotype by time effect over the 2-year trial in those treated by the four dietary compositions. We observed significant genotype-time interactions on changes in total fat, FM%, and percentage of trunk fat in response to the high-protein diet. When assigned to the high-protein diet, participants who carried the AA genotype had greater loss in total fat, FM%, and percentage of trunk fat than those without this genotype. No genotype-time interaction on changes in body composition was found in the low-protein group (Fig. 3).<SEP>We also observed significant genotype-time interactions on changes in fat distribution in response to the low- and high-protein diets. Participants with the AA genotype had greater decrease in fat distribution in response to the high-protein diet compared with those without this genotype. In contrast, participants with the AA genotype were associated with less loss in TAT and SAT in response to the low-protein diet (Fig. 4).<SEP>We found genotype-time interactions on changes in total fat, FM%, and percentage of trunk fat in response to the low-fat diet (Supplementary Fig. 3), but no genotype-time interactions on changes in fat distribution were found in response to the low- or high-fat diets (Supplementary Fig. 4).A similar trend was observed in the white population (data not shown).",
                  "DISCUSSION<SEP>In the POUNDS LOST Trial, a 2-year, randomized weight-loss intervention, we found that dietary protein intake significantly modified the effect of an FTO variant on changes in body composition and fat distribution. Carriers of the risk allele (A allele) of the rs1558902 genotype had a greater loss of weight and regional fat in response to a high-protein diet compared with noncarriers, whereas an opposite genetic effect was observed regarding changes in fat distribution in response to a low-protein diet. Our data indicate that the modification effects of dietary treatment were more evident with prolonged intervention. We did not observe significant modification of dietary fat intake on the genotype effects.<SEP>The rs1558902 genotype was reported to show the strongest association with obesity in the European and other ethnic populations, and it has strong linkage disequilibrium with other obesity-associated FTO variants such as the rs9939609 genotype. In this study, the MAF of the polymorphism in all participants was similar to those in the HapMap CEU population (0.45). At baseline, no significant difference was observed in anthropometrics and metabolic estimates, body composition, or fat distribution across genotypes. The lack of association with baseline BMI is probably largely due to the fact that the participants were all overweight or obese, so that the groups had relatively smaller variances in BMI than the general population.<SEP>Several cross-sectional studies showed that diets might modify the effect of the FTO variant on obesity, but the data from randomized diet intervention trials are conflicting and limited by small sample size or short term of follow-up (Supplementary Table 3). Two lifestyle intervention studies with follow-up periods of 9 and 12 months did not find significant influence of FTO polymorphisms (rs8050136 and rs9939609) on changes in body weight or fat distribution related to diet among 200 overweight and obese individuals treated by diets with reduced fat and increased fiber or reduced fat and sugar. In another 10-week, hypo-energetic diet intervention with either low fat or high fat content, the FTO variant had an effect on only changes in resting energy expenditure, insulin release, and sensitivity, not on weight loss. Similarly, in the Finnish Diabetes Prevention Study, the FTO variant did not modify weight change by individualized diet intervention with reduced fat and increased fiber during the 4-year follow-up of 255 individuals with impaired glucose tolerance. In our study, when macronutrient components of diets were not considered, we found no main effects of the FTO variant on changes in weight and body composition during the intervention.<SEP>Grau et al. reported that dietary fat/carbohydrate content interacted with some genetic variants including the FTO variant on weight reduction or change in obesity-related phenotypes. In our study, we also found significant gene-diet interactions on changes in body composition and fat distribution. However, our data indicate that it is the dietary protein component, rather than dietary fat, that might drive the observed interactions. In previous studies, high-protein intervention has been found to result in a greater weight loss and abdominal fat mass. Our results suggest that individuals with a certain genetic background may benefit more in weight loss by following a high-protein diet. The mechanism of how protein intake interacts with FTO genotype is unclear.<SEP>Our data indicate that the genetic effects on certain fat compositions or depots may be more evident than the effects on overall adiposity. Functional studies have shown that the loss or overexpression of FTO in mice led to different changes in fat distribution at different dissected sites. FTO mRNA expression was fat depot–specific and was found to differ significantly in subcutaneous fat and in visceral fat. Epidemiological studies also have shown that FTO variants are significantly associated with distribution of fat depots. Taken together, these data suggest that genetic effects of FTO on the change of fat mass at various sites may be different, and changes in anthropometrics may not adequately reflect the effects of an FTO variant.<SEP>The genetic effect in our study seemed to be more evident at 2 years than that at 6 months. The results were in line with a recent study by Razquin et al. in which it was found that FTO risk allele carriers had the highest weight reduction after 3 years of intervention with a Mediterranean diet compared with several short-term diet interventions (less than 1 year) in which no influence of an FTO variant on weight loss or change in fat distribution was found. Of note, between 6 months and 2 years of intervention in our trial, the participants regained weight. Therefore, it seems that the genetic variant might affect both the reduction and regain of the adiposity measures. These data suggest that the modification effects of diet treatment on an FTO genotype effect are more likely to be identified in long-term interventions.<SEP>Several limitations need to be considered when interpreting our findings. Even though our study is thus far the largest and longest diet intervention weight-loss trial, the relatively small sample size of the subgroups may limit the power to detect very moderate genetic effects or interactions. We did not adjust for multiple testing according to the recommendation by Rothman and Lai et al. because outcomes and the repeated measurements at 6 months and 2 years were highly correlated in our study. Overadjustment for multiple comparisons may increase the type II error and reduce power to detect significant differences. In addition, the majority of the total participants were white, and further studies are needed to determine whether our findings are generalizable to other ethnic groups. Even though the randomized clinical trial is thought to be the best model to test gene-environment interactions, we acknowledge that replication in diverse populations is needed to verify our findings.<SEP>In summary, we found that dietary protein intake might modify the FTO variant’s effect on changes in body composition and fat distribution. A high-protein diet may be beneficial for weight loss in individuals with the risk allele of an FTO variant. Further studies are warranted to verify our findings and explore the potential mechanisms.<SEP>This article contains Supplementary Data online at http://diabetes.diabetesjournals.org/lookup/suppl/doi:10.2337/db11-1799/-/DC1.",
                  "20526",
                  "['OBWCCE', 'EATBTS', 'FOODIN']"
                ],
                [
                  "14",
                  "20712903",
                  "Background<SEP>Small for gestational age (SGA) babies (typically defined as birthweight below the 10th centile according to gestational age) are not only at increased risk of neonatal morbidity and mortality, but are also at increased risk of rapid postnatal weight gain, later obesity and diseases in adulthood such as type 2 diabetes, hypertension and ischemic heart disease which are major causes of adult morbidity and mortality worldwide. Although the cause of this association is unknown, several hypotheses have been proposed. The fetal insulin hypothesis proposes that common genes inherited by the fetus affect both birth size and predisposition to adult diseases. In contrast, the Barker hypothesis suggests the association to be the result of fetal programming - permanent changes in physiology and metabolism in response to adverse maternal uterine environment in pregnancy that result in increased metabolic disease risk in adulthood. The increased risk of adult metabolic diseases in those who are born small at birth is further amplified by an accelerated pattern of growth during childhood. The thrifty phenotype hypothesis explains this phenomenon by suggesting that the fetal nutrient conserving adaptations in response to intrauterine under nutrition is overwhelmed by nutrient abundance post-natally and manifests in adult metabolic diseases. Singhal and Lucas propose that it is not low birth weight per se, but this rapid postnatal growth that is responsible for the increased risk for disease in later life.<SEP>Environmental risk factors for SGA are well established and include smoking, low pregnancy weight, maternal short stature, maternal diet, ethnic origin of mother and hypertension. These risk factors have been studied and confirmed in the Auckland Birthweight Collaborative (ABC) study. However, in a large proportion of SGA, no underlying cause is evident, and these individuals may have a larger genetic contribution. The existence of such genetic factors is supported by the observation that SGA births tend to cluster in families and to recur in successive generations. However, genetic association publications for SGA to date have been rare and inconsistent. Infante-Rivard et al., 2007 provides a summary overview of genetic association studies for SGA. In brief, only the thrombophilia pathway including genetic variants in MTHFR, FV, and FII , and xenobiotic-metabolizing pathways with variants in CYP1A1, GSTT and GSTM  have been studied with any frequency. Other pathways that have been studied with less frequency include the vascular dysfunction or atherosclerosis pathway with variants in APOE, PON and ACE  and the insulin resistance pathway with variants in IGF-1 . However, none of these pathways have shown robust associations with susceptibility to adult diseases with which SGA has been linked.<SEP>As children born SGA often become obese in later life and/or develop type 2 diabetes we tested the association between SGA and polymorphisms in genes that have previously been associated with obesity and/or type 2 diabetes under the hypothesis that a common genetic denominator might predispose to both SGA and obesity and/or type 2 diabetes. The hypothesis was tested in subjects from the Auckland Birthweight Collaborative (ABC) study. Since we began this study there have been several publications examining the relationship between type 2 diabetes susceptibility genes and birthweight (these studies are described in the discussion), however we have included both obesity and type 2 diabetes susceptibility genes and we have focused on SGA status.",
                  "Methods<SEP>Samples<SEP>The ABC study has been fully described previously.<SEP>In summary, between 16 October 1995 and 12 August 1996, babies born and resident in the Waitemata Health or Auckland Healthcare regions were eligible for inclusion and from 12 August 1996 to 30 November 1997, babies born and resident in the Auckland Healthcare region were eligible for inclusion. Preterm infants (< 37 completed weeks of gestation), multiple births and those with congenital abnormalities were excluded. The sample was selected disproportionately to include all infants born at term and SGA (≤10th percentile for sex and gestation for New Zealand) and a random sample of appropriate for gestational age (AGA) infants (> 10th percentile for sex and gestation). Gestational age was estimated using the date of the last menstrual period where it was available and was within 2 weeks of the best clinical estimate of gestational age at birth; otherwise the best clinical estimate was used.<SEP>Data have been collected at birth, 1, 3.5, 7 and most recently 11 years of age. The original sample at birth resulted in a sample of 1714 subjects, of which 871 mothers were identified in the obstetric data to be of European ethnicity. At the age of 1 and 3.5 follow up of non-European ethnicities was poor resulting in a lack of ability to generalise the results from these children to their particular populations. As a result follow-up from the age of 7 has only been carried out on those children whose mothers were identified as European ethnicity at birth. At 11 years 546 participants consented to collection of peripheral blood (n = 397) or a buccal swab (n = 149) for DNA extraction and genotyping. 227 samples were from children born small for gestational age (SGA) and 319 were from children born appropriate for gestational age (AGA).<SEP>DNA was extracted from the blood/buccal samples using Qiagen's DNA extraction kit and following the manufacturer's instructions.<SEP>The study received ethical approval from the Northern Regional Ethics Committee. Signed consent for the study and extraction of DNA was given by the parents of the children and assent also given by the child.<SEP>SNP selection<SEP>The SNPs were selected from a systematic literature search to identify genetic variants demonstrating association with obesity and/or type 2 diabetes. We included 18 diabetes SNPs and 46 obesity SNPs identified from published candidate gene or genome-wide association studies, totalling 64 SNPs which were located in 42 genes. See table 1 for the list of genes/SNPs selected for investigation.<SEP>Genotyping<SEP>Genotyping was performed with the MassARRAY and iPlex systems of the Sequenom genotyping platform (Sequenom, San Diego, CA), which uses the MALDI-TOF primer extension assay according to manufacturers' recommendations.<SEP>Assays were optimized in 24 samples consisting of 20 reference Centre d'Etude du Polymorphisme Humain (CEPH) samples and 4 blanks.<SEP>All sample plates contained cases, controls, blanks, CEPH and duplicate samples. Quality control measures included independent double genotyping, blind to sample identity and blind to the other caller, and where available comparison of our CEPH genotypes to those in the HapMap http://www.hapmap.org.<SEP>Statistical analysis<SEP>SNPs were tested for deviation from Hardy Weinberg Equilibrium (HWE) in both cases (SGA) and controls (AGA), and for the weighted population using a chi-square goodness-of-fit test.<SEP>To determine if there were differences between children born SGA and those born AGA, genotype and allele frequencies for each SNP were analyzed by logistic regression using the major allele as the reference for allele analyses and the major homozygote group as the reference for the genotype analysis. Odds ratios show the increased risk (OR > 1) or decreased risk (OR < 1) for the minor allele, or genotype group, of being SGA in relation to the reference group. Univariable logistic regression was carried out to assess the relationship of each SNP with SGA and those found to be significant at the 10% level were carried through to multivariable analyses. The multivariable model controlled for the previously published model for SGA in this population namely gestational age, gender, socio-economic status, age mother left school, marital status, attendance at antenatal classes, primiparity, maternal smoking during pregnancy, marijuana use during pregnancy, maternal height and weight, maternal age at index pregnancy and maternal hypertension.<SEP>Statistical analyses were carried out using R and SAS (V9.1 SAS Institute., Cary, NC, USA).",
                  "Results<SEP>We do not have genotypic data for 10 SNPs: rs1387153 (MTNR1B), rs6020339 (CTNNBL1), rs2388399 (PFKP), rs477181 (MC4R), rs3820152 (ADIPOR1), rs7561317 (TMEM18), rs7647305 (ETV5 - DGKG), rs2844479 (NCR3 - AIF1), rs8050136 (FTO) and rs10487818 (NAMPT). These SNPs either could not be multiplexed into our sequenom assays, failed or did not pass our quality control measure for inclusion in the analysis. The remaining 54 SNPs produced genotypic data for analysis. Each of these SNPs had a genotyping call rate greater than 90% and the genotyping calls did not differ significantly from HWE criteria.<SEP>Nine SNPs were significant at the 10% level in the univariable analysis at either the genotypic or allelic level and each of these SNPs was then taken forward to multivariable analysis (see tables 2 and 3 for results).<SEP>Six SNPs demonstrated statistical significance at p < 0.05 in either the univariable or multivariable analysis: rs9939609 which is an intronic SNP in FTO, rs5219 which is a missense (Lys-Glu) SNP in KCNJ11, rs925946 which is located 9,240 bp from BDNF, rs6602024 which is an intronic SNP in PFKP, rs10508503 which is located 179,016 bp from PTER and rs10913469 which is an intronic SNP in SEC16B.<SEP>As we have two SNPs in BDNF and these two SNPs are in LD (D' = 1, r2 = 0.116) and exist within the same haplotype block (confirmed using haploview 4.2) we conducted a haplotype analysis to determine if the haplotype would give stronger results than the individual SNPs (data not shown). The haplotype analysis of rs6265 and rs925946 found an overall effect (Global Stat = 5.92, p = 0.05). In univariable analysis compared to the GG haplotype the AG haplotype did not show a statistically increased risk (OR = 1.22, 95%CI = 0.86-1.68), whilst those with the GT haplotype had a borderline decreased risk (OR = 0.78, 95%CI = 0.59-1.04). In multivariable analyses these odds ratios moved towards unity and were not statistically significant suggesting that the haplotype analysis does not add anything further to the analysis of each SNP individually.<SEP>We have examined anthropometric characteristics of the children at 11 years of age (table 4) and found that the children born SGA remain significantly lighter, shorter and still have lower BMI than the AGA children. Hence not all of the SGA children have shown catch up growth.",
                  "Discussion<SEP>We have found associations (with p values less than 0.05) for SGA with the diabetes related SNP in KCNJ11 and the obesity related SNPs in FTO, PFKP, PTER, SEC16B and BDNF. After controlling for potential confounders the association with the FTO SNP did not remain significant, whilst the other 5 SNPs were positively associated with SGA in the multivariable model.<SEP>The T allele of KCNJ11 SNP rs5219 is associated with type 2 diabetes in adults and has been shown to be associated with reduced insulin secretion (see table 1 for references) so our study result finding that this risk allele is associated with being SGA is compatible with the fetal insulin hypothesis where genetically mediated reduced insulin secretion beginning in-utero results in reduced birthweight, and later increases the risk of developing T2 D. Two previous studies evaluated the diabetes related KCNJ11 variant with birthweight and found no association. It is possible that this variant interacts with other factors, either genetic or environmental, that exist within the ABC cohort but are not present in the other two studies.<SEP>Our study found that the high risk allele for obesity in the PTER SNP (C allele at rs10508503) was associated with being SGA. This finding would fit with the fetal insulin hypothesis only if this allele had a direct effect on increasing insulin resistance prior to manifesting as increased BMI later in life, and thus manifests as low birth weight and later leads to obesity. Alternatively, the association of this obesity gene in SGA babies may be due to some survival advantage of being a \"thin-fat\" baby in terms of inappropriate fat mass for body size, not discerned by the simple measure of birth weight. It may be that most of these SGA babies grow into genetically predisposed obese children and adults; hence we are observing the association with post-natal obesity in our SGA cohort.<SEP>Conversely, our study found an association of SGA with the low risk alleles for obesity in the BDNF and SEC16B genes suggesting that these alleles may confer a propensity to small size beginning in-utero, since the same SNP in BDNF has been associated with thinness in women. It would be interesting to examine whether this sub-group of SGA babies go on to have improved metabolic outcomes later in life by having a lower risk of obesity.<SEP>Since we began this study associations between common variants in type 2 diabetes susceptibility genes have been tested in several large birthweight cohorts. Freathy et al, 2009 looked at five type 2 diabetes susceptibility genes and found that the CDKAL1 and HHEX-IDE loci were associated with reduced birth weight. They did not detect an association with CDKN2A/B, IGF2BP2, and SLC30A8. All 5 of these loci were included in our study and we did not detect an association for any of them with SGA. Zhao et al, 2009 also observed an association between lower birth weight and the CDKAL1 locus. However, no association was found with 19 other diabetes genes examined, including KCNJ11 for which we found an association with SGA. Pulizzi et al, 2009 investigated 9 diabetes genes, all of which are included in our study but were not found to be associated with SGA. Of the tested variants, the risk variant in HHEX showed a trend towards a low birthweight and the risk variant in the CDKN2A/2B locus was associated with high birthweight. The three studies described above investigated birthweight. Only TCF7L2 has been studied for association with SGA. The gene was not associated with SGA in these two cohorts or our own. However, an association has been described between TCF7L2 and birthweight, although the effect was strongest with maternal genotype and after adjustment for maternal genotype fetal TCF7L2 genotype was not associated with birth weight.<SEP>We examined the publically available British 1958 birth cohort database http://www.b58cgene.sgul.ac.uk/ for our significant genes. SNPs in BDNF and FTO were associated with birthweight but KCNJ11, PTER, PFKP and SEC16B did not show any associations.<SEP>The failure to replicate the associations reported by Freathy et al, Pulizzi et al and Zhao et al and our reporting of significant results for different genes may be due to the different phenotype used (birthweight vs. SGA) and/or due to different study populations with different environmental and genetic influences.<SEP>To summarise, we have identified five SNPs/genes which are associated with SGA. While noting that replication in independent samples is essential, our data provides evidence that genetic variation in type 2 diabetes and obesity susceptibility genes such as KCNJ11, BDNF, PFKP, PTER and SEC16B have a possible role in SGA as well as their established roles in obesity and/or diabetes.<SEP>We recognise that the association observed with these SNPs are unlikely to survive any adjustment for multiple testing and could thus be false positives. But it is possible that we may be seeing small genetic effects here and as our sample size is small compared to the majority of genetic association studies today we have low power to detect these associations with a high level of statistical significance. Calculations of statistical power using PS 2.1.31 show that for the ABC study we have 31.61% power to detect an odds ratio of 1.2, 56.84% power to detect an odds ratio of 1.3 and 78.01% power to detect an odds ratio of 1.4 for a SNP with a minor allele frequency of 0.48 (such as rs864745 in JAZF11). For SNPs with a lower allele frequency the power would be less. For example for a SNP with a minor allele frequency of 0.12 (such as rs6602024 in PFKP) we have 17.36% power to detect an odds ratio of 1.2, 31.42% power to detect an odds ratio of 1.3 and 48.05% power to detect an odds ratio of 1.4.<SEP>It is also possible that these genes may have more subtle effects and could affect a related phenotype, rather than be directly associated with SGA. Although beyond the scope of this paper it would be interesting to look at these SNPs/genes in relation such phenotypes e.g., catch up growth. Alternatively, the associations could reflect underlying LD with other markers in these genes. Further analysis in these genes with which we demonstrate an association with SGA is therefore required. Also, further investigation of the 36 genes for which we found no association should not be ruled out. The lack of association of these genes with SGA in our sample could be explained by a lack of power and we cannot rule out that we were unable to detect smaller effects of these variants. It is also possible that these obesity and/or diabetes genes may lead to small decreases in birthweight but do not result in the more severe SGA phenotype. Alternatively, it may be possible that any direct effects of susceptibility genes resulting in an individual being born SGA (by reduced insulin secretion) may be offset by an opposing effect from the maternal genotype (through the effects of the same variants on maternal glucose levels). Unfortunately, maternal DNA samples are not presently available from the ABC cohort and so we are unable to test this.<SEP>During revision of this manuscript Freathy and colleagues reported a meta-analysis of genome-wide association studies and followed up the top hits in 13 replication studies. They identified two loci, in ADCY5 and near CCNL1, that are associated with birth weight and explain 0.3% and 0.1% of the variance in birth weight, respectively. Both loci were also associated with smallness for gestational age. SNPs in ADCY5 have recently been implicated in regulation of glucose levels and susceptibility to type 2 diabetes, providing further evidence that the association between lower birth weight and/or SGA and subsequent type 2 diabetes does indeed have a genetic component.",
                  "18440",
                  "['OBWCCE']"
                ],
                [
                  "15",
                  "26873362",
                  "Background<SEP>The International Diabetes Federation (IDF) has reported that there were 382 million people living with diabetes worldwide in 2013, and this number is expected to rise to 592 million by 2035. Most people with diabetes live in low- and middle-income countries, where rapid changes in lifestyle have increased the prevalence of diabetes, cardiovascular diseases, and cancer, and these countries are expected to experience the greatest increase in cases of diabetes in the next 20 years. In 2010, the Chinese Center for Disease Control and Prevention (CDC) and the Endocrinology Branch of the Chinese Medical Association conducted a cross-sectional survey in a nationally representative sample of 98,658 Chinese adults with diabetes. The prevalence of diabetes in Chinese adults was estimated to be 9.7% using the 1999 World Health Organization criteria. However, when glycosylated hemoglobin (HbA1c) ≥6.5% was included in the diagnostic criteria, the prevalence of diabetes in China was estimated to be approximately 11.6%, of which only one-quarter (25.8%) received treatment for diabetes, and only roughly one-third of those receiving treatment (39.7%) achieved adequate glycemic control.<SEP>Xinjiang is located in the middle of the Eurasian continent occupying one sixth of the mainland. It is a multi-ethnic region where the Uyghur nationality comprises about 46% of the total population. In 2012, Yang et al. screened an Uyghur population of 4620 adults aged over 35 years and found that 6.23% had type 2 diabetes (T2D). Only 35% of these participants were aware of their diabetes, 7.3% were taking hypoglycemic agents, and 3.13% achieved blood glucose control. These data demonstrate an urgent need to develop effective strategies to improve prevention, detection, and treatment of diabetes in the Uyghur population.<SEP>Development of T2D can be attributed to the combined effect of genetic and environmental factors. Establishment of single-nucleotide polymorphism (SNP) databases, development and improvement of cost-effective high-throughput genotyping technology, and multi-center consortium large-scale genome-wide association studies (GWAS) is an effective method to investigate genetic susceptibility to T2D. By searching susceptibility variants across the entire genome in an unbiased, hypothesis-free manner, GWAS have successfully identified genetic loci robustly associated with T2D, the number of which has risen from just three in 2006 to approximately 70 today. The first GWAS analysis for T2D was conducted in a European cohort in 2007. The study was initially conducted in 600 T2D patients and 600 control patients, and was followed by a related study of nearly 3,000 T2D patients and 3000 controls from an independent European cohort. This study identified 3 novel susceptibility genes associated with T2D including TCF7L2, HHEX/IDE and SLC30A8. Subsequently, a large number of T2D associated susceptibility genes have been characterized by GWAS, including the cyclin-dependent kinase inhibitor 2A/2B (CDKN2A/2B) gene, the CDK5 regulatory subunit associated protein 1-like 1 (CDKAL1) gene, and the fat mass and obesity-associated (FTO) gene.<SEP>The HapMap Project has provided data for genome-wide SNPs in diverse populations such as Han Chinese and European residents. Nevertheless, the association between CDKN2A/2B, CDKAL1 and FTO and T2D has not been verified in an Uyghur population. This study aims to investigate the genotype distribution of SNPs and to assess the association between gene polymorphisms and T2D in Uyghur adults. Furthermore, using multifactor dimensionality reduction (MDR), a nonparametric and genetic model-free method for detecting and characterizing nonlinear interactions between discrete genetic and environmental attributes, the capacity of gene-gene and gene-environment interactions to predict T2D will be investigated. MDR has been successfully applied in studies investigating the etiology of diseases such as multiple sclerosis and T2D. Lou et al. proposed a modified Generalized MDR (GMDR) method applicable to both dichotomous and continuous variables in population-based study designs, which improves predictive ability. We will apply GMDR to provide information about the molecular pathogenesis of T2D in the Uyghur population.",
                  "Material and Methods<SEP>Study design<SEP>A case-controlled study of Uyghur patients with T2D and non-diabetic controls was conducted at the First Affiliated Hospital of Xinjiang Medical University between March 2012 and September 2013. Written informed consent was obtained from each participant prior to the study, and the protocol was approved by the Ethics Committee of the First Affiliated Hospital of Xinjiang Medical University.<SEP>Patients with T2D were recruited according to the following inclusion criteria: newly diagnosed T2D according to WHO criteria, fasting plasma glucose (FPG) ≥7.0 mmol/L, and/or oral glucose tolerance test 2 hour sample (OGTT-2h) ≥11.0 mmol/L, or previously diagnosed T2D treated with hypoglycemic drugs where FPG and OGTT-2h levels were below the diagnostic levels. The exclusion criteria for the patient group included: patients with type 1 diabetes, or any wasting diseases such as viral infections, tuberculosis, cancer, mental illness, epilepsy, or acute inflammation.<SEP>Non-diabetic control patients were enrolled during health examination at the hospital during according to the following inclusion criteria: fasting plasma glucose <7.0 mmol/L, and/or OGTT-2h <11.1 mmol/L, and without a history of treatment with hypoglycemic agents. The exclusion criteria included: any wasting diseases such as viral infections, tuberculosis, cancer, mental illness, epilepsy, and acute inflammation.<SEP>Subject clinical characteristics<SEP>Any previous diagnoses, current clinical characteristics and family medical history of all diabetic and non-diabetic control patients were recorded. Clinical characteristics included height, weight, body mass index (BMI), waist circumference at the midpoint between the lateral iliac crest and the lower edge of the ribs in the midaxillary line, and systolic and diastolic blood pressures. Peripheral blood samples were drawn from all subjects after fasting for 8 hours, with 4 ml of blood mixed with or without anticoagulants. The blood samples were centrifuged at 3500 r/min for 10 min within 2 hours of collection. Serum creatinine (Scr), uric acid (UA), fasting plasma glucose (FPG), triglycerides (TG), total cholesterol (TC), high density lipoproteins (HDL), low density lipoproteins (LDL), aspartate aminotransferase (AST), and alanine aminotransferase (ALT) were measured with qualified controls using a Hitachi 7600 automatic biochemical analyzer in the Clinical Laboratory of the First Affiliated Hospital of Xinjiang Medical University.<SEP>Hypertension was diagnosed according to the 2010 Guidelines for prevention and treatment of hypertension in China. Coronary heart disease (CHD) was diagnosed by a cardiology specialist according to the 1979 WHO diagnostic criteria, or according to a previous medical history of angina or myocardial infarction previously confirmed as CHD. Non-alcoholic fatty liver disease (NAFLD) was diagnosed according to the nonalcoholic fatty liver disease diagnostic criteria established in 2010 by the Chinese Society of Hepatology. Dyslipidemia was diagnosed according to the Guidelines on Prevention and Treatment of Blood Lipid Abnormality in Chinese Adults established in 2007, or previous diagnosis of dyslipidemia and treatment with a lipid-lowering drug. Stroke was diagnosed according to the criteria established by the Fourth National Conference on cerebrovascular diseases in 1995.<SEP>Detection of SNPs within CDKAL1, CDKN2A/2B, and FTO<SEP>Anticoagulated blood samples were stored at −80°C for DNA extraction using the iAUTOMAG DNA extraction system (BioTeke Corp., Beijing, China) following the manufacturer’s instructions. The concentration and quality of extracted DNA was assessed by 1% agarose gel electrophoresis. Samples containing greater than 50 ng/μl DNA, an A260/A280 ratio (OD) between 1.8–2.0, and a clear band in gel electrophoresis were used for further analysis and transferred to 96-well plates and stored at −20ºC for later use.<SEP>Eight hotspot SNPs within the CDKAL1, CDKN2A/2B, and FTO genes were selected, including loci rs10946398, rs7754840 (CDKAL1), rs10811661, rs3088440, rs11515 (CDKN2A/2B), rs7195539, rs8050136, and rs9939609 (FTO). SNPs were detected by CapitalBio Corporation (Beijing, China) using the Sequenom MassARRAY®SNP technology with a genotyping success rate and accuracy >98%.<SEP>Sequenom MassARRAY®SNP detection<SEP>PCR reactions were carried out in 5 μl per well of a 384-well plate using 4 μl of PCR master mix, 20–50 ng template DNA, 0.5 U of Hotstar Taq, 0.5 pmol of each primer, and 0.1 μl of 25 mM dNTPs incubated at 94°C for 4 min; 94°C for 20 sec; 56°C for 30 sec; 72°C for 1 min; for 45 cycles and 72°C for 3 min. PCR products were treated with shrimp alkaline phosphatase (SAP) to remove free dNTPs and PCR was performed with 2 μl of SAP mix (SAP 0.5 U, buffer 0.17 μl) and 5 μl of PCR product at 37°C for 40 min and 85°C for 5 min. After SAP treatment, a single-base extension reaction was carried out with 7 μl of SAP-treated PCR product and 2 μl of EXTEND Mix (primer mix 0.94 μl, iPLEX enzyme 0.041 μl, and extending mixture 0.2 μl) at 94°C for 30 sec; 94°C for 5 sec; 52°C for 5 sec; 80°C for 5 sec; GOTO III, 4 more times; GOTO II, 39 more times; and 72°C for 3 min.<SEP>Purified extension products were loaded onto the 384 SpectroCHIP (Sequenom) chip using a MassARRAY Nanodispenser RS1000 spotter. The spotted SpectroCHIP chips were analyzed using matrix-assisted laser desorption/ionization-time of flight (MALDI-TOF) mass spectrometry. All results were obtained using TYPER 4.0 software (Sequenom) genotyping.<SEP>Statistical analysis<SEP>All statistical analyses were performed using IBM SPSS 21.0 software (IBM Corp., Armonk, NY, USA). The Hardy-Weinberg equilibrium (HWE) of the control group was calculated to evaluate whether our samples were representative of a population. All normally distributed qualitative data are expressed as mean ± standard deviation, and comparisons between two groups were carried out using the t test. Abnormally distributed data are presented as median and quartiles, and group-group comparisons were conducted using the Wilcoxon rank test. Differences in qualitative data, SNP genotype or allele frequency were analyzed using the χ2 test. Logistic regression analysis was applied to evaluate risk factors associated with T2D, using co-dominant, dominant, recessive, additive, and over-dominant models for each SNP.<SEP>For gene-environment interaction analysis, clinical characteristics were categorized accordingly. According to the Guidelines for prevention and control of overweight and obesity in Chinese adults established in 2004, patients were categorized as underweight if BMI <18.5 kg/m2 (value=0); normal weight if 18.5 kg/m2 ≤BMI <24 kg/m2 (value=1); overweight if 24 kg/m2 ≤BMI <28 kg/m2 (value=2); or obese if BMI ≥28 kg/m2 (value=3). The waistline (WL) values were categorized as 0 where male WL <85 cm or female WL <80 cm, and 1 where male WL ≥85 cm or female WL ≥80 cm. According the Guidelines on Prevention and Treatment of Blood Lipid Abnormality in Chinese Adults established in 2007, the scores associated with TG, TC, HDL, and LDL levels were categorized as 0 where TG <1.7 mmol/L, TC <5.18 mmol/L, HDL ≥1.04 mmol/L, and LDL <3.37 mmol/L, and 1 where TG ≥1.7 mmol/L, TC ≥5.18 mmol/L, HDL <1.04 mmol/L, and LDL ≥3.37 mmol/L.<SEP>Gene-gene interactions within each SNP were analyzed using Generalized MDR (GMDR) 0.9 software (http://www.ssg.uab.edu/gmdr/) using the best model with a relatively high cross-validation count and balanced testing accuracy. Statistical significance was considered at P<0.05.",
                  "Results<SEP>General characteristics of the study population<SEP>In total 879 Uyghur patients with T2D hospitalized in the First Affiliated Hospital of Xinjiang Medical University were enrolled between March 2012 and September 2013, including 543 males and 336 females aged 21–83 years. 895 non-diabetic, unrelated Uyghur adults were included as controls, including 571 males and 324 females aged 21–79 years.<SEP>There were no significant differences in age or sex between the diabetes and control groups (P=0.302, and 0.378, respectively; Table 1). The levels of BMI, WL, SBP, FPG, and ALT in the diabetes group were higher than those in the control group (P<0.05), whereas the UA, SCR, HDL, and LDL levels were lower in diabetic patients than in controls (P<0.05). The proportion of patients with a family history of diabetes, or hypertension, CHD, NAFLD, dyslipidemia, stroke, smoking, alcohol abuse or high pressure work environment was higher in the diabetes group than in the control group (P<0.05). However, the levels of DBP, TG, TC, and AST did not differ significantly between these groups (P>0.05) (Table 1).<SEP>Analysis of risk factors associated with type 2 diabetes in the Uyghur population<SEP>Univariate logistic regression analyses were conducted to analyze the variables associated with T2D, and multivariate logistic regression analyses were conducted to analyze the variables showing statistically significant differences in the univariate logistic regression analyses (Table 2). Hypertension, CHD, family history of diabetes, FPG, TC, and HDL were independent risk factors for T2D (P<0.001).<SEP>Distribution of SNP genotypes in the Uyghur population<SEP>The Hardy-Weinberg equilibrium (HWE) test was performed to assess the genotype distributions of SNPs in the control group. The genotype distributions of 7 of the 8 sites in the control group were within HWE (P>0.05), indicating that these samples were representative of the Uyghur nationality. However, the genotype distribution of rs11515 did not fit the HWE in the control group (P=0.041), and was thus excluded from further analysis. Logistic regression analyses of genetic models of SNP genotype and allele frequency revealed statistically significant differences in the genotype distributions and allele frequencies between the diabetes and control groups within loci rs10811661 (CDKN2A/2B), rs7195539 (FTO), rs8050136 (FTO), and rs9939609 (FTO) (P<0.05), but not in loci rs10946398 (CDKAL1), rs7754840 (CDKAL1), or rs3088440 (CDKN2A/2B), (P>0.05; Table 3). The allele distributions in loci rs8050136 (FTO) and rs9939609 (FTO) exhibited significant divergence between the diabetes and control groups (P<0.05), and allele A was a common risk allele for both loci. The odds ratio (OR) values were 1.198 (95% confidence interval [CI]: 1.037–1.384) for rs8050136 (FTO) and 1.238 (95% CI: 1.075–14.427) for rs9939609 (FTO).<SEP>The genotype distribution of rs10811661 (CDKN2A/2B) differed between the diabetes and control groups in co-dominant, additive and recessive models, with P values of 0.045, 0.007, and 0.011, respectively, and OR values of 1.484 (95% CI: 1.010–2.183), 1.230 (95% CI: 1.057–1.431), and 1.276 (95% CI: 1.057–1.541), respectively. Similarly, the rs8050136 (FTO) genotype distributions differed significantly between the two groups in co-dominant, additive and recessive models with P values of 0.027, 0.015, and 0.037, respectively, and OR values of 1.472 (95% CI: 1.045–2.072), 1.196 (95% CI: 1.035–1.382), and 1.220 (95% CI: 1.012–1.470), respectively. In addition, the genotype distributions of rs9939609 (FTO) in co-dominant, additive, dominant, and recessive models differed significantly between the two groups (P=0.046, 0.010, 0.004, 0.010, and 0.036, respectively), with OR values of 1.224 (95% CI: 1.004–1.492), 1.513 (95% CI: 1.106–2.070), 1.228 (95% CI: 1.069–1.411), 1.279 (95% CI: 1.061–1.542), and 1.377 (95% CI: 1.021–1.858), respectively.<SEP>However, all the differences discussed above were eliminated after adjusting for sex, age, BMI, hypertension, CHD, family history of diabetes, FPG, TC, and HDL (P>0.05). Nonetheless, the variation in rs7195539 (FTO) genotype distribution between the diabetes and control groups remained significant in co-dominant and dominant models after adjusting for covariates, with P values of 0.009, 0.012, and 0.031, respectively, and OR values of 0.013 (95%CI: 0.001–0.334), 0.017 (95% CI: 0.001–0.404), and 0.026 (95% CI: 0.001–0.713).<SEP>Gene-gene and gene-environment interactions between SNPs and environmental factors<SEP>The SNP loci were assigned according to co-dominant markers and gene-gene interactions were evaluated by GMDR software (Table 4). After adjusting for covariates including age, sex, BMI, hypertension, CHD, family history of diabetes, FPG, TC, and HDL, the two-locus interaction model rs10811661–rs7195539 appeared to be the best model, with a cross-validation consistency of 10/10, the highest balanced testing accuracy of 0.5483, and a P value of 0.011, which increased to only 0.014 after 1000 permutation tests. The gene-gene interactions in the best model are shown in Figure 1.<SEP>Candidate variables for gene-environment interaction analyses were set as each SNP, family history of diabetes, smoking history, alcohol history and work pressure, whereas covariates were set as age, gender, FPG, hypertension, and CHD (Table 5, Figure 2). The three-locus interaction model TG-HDL-family history of diabetes appeared to be the best model with a cross-validation consistency of 10/10, the highest balanced testing accuracy of 0.7072, and a P value of 0.001, which was reduced to <0.001 after 1000 permutation tests. The second-best model was the four-locus interaction model of rs7195539-TG-HDL-family history of diabetes, with a cross-validation consistency of 8/10, the highest balanced testing accuracy of 0.7002, and a P value of 0.001, which was reduced to <0.001 after 1000 permutation tests.",
                  "Discussion<SEP>Identifying individuals at risk for T2D is extremely important, as this allows the implementation of appropriate strategies to potentially prevent the occurrence of T2D and its systemic complications, including atherosclerotic cardiovascular diseases. It is notable that prediabetes and a family history of diabetes are both associated with impairments of the cardiovascular system and increased cardiovascular risk in the absence of overt T2D, highlighting the importance of identifying susceptible individuals before the disease develops. It has been suggested that systemic inflammation and elevated blood glucose levels in prediabetes lead to microvascular and macrovascular disease that manifest as increased intima-media thickness of the common carotid artery, reduced flow-mediated dilatation, increased arterial stiffness and impaired coronary endothelial function. The insulin resistance associated with a family history of diabetes is though to cause endothelial dysfunction through activation of the Ras/Raf/MAPK pathway, up-regulation of endothelin and its receptors, and reduction of NO levels, as well as increased intima-media thickness of the common carotid artery. Although several risk factors for T2D have already been determined, the discovery of new genetic markers will improve the identification of individuals predisposed to the development of this disease. This might help to reduce the burden of T2D on individuals and society through the use of screening and appropriate interventions in those at risk of developing the T2D.<SEP>Novel susceptibility genes that have been reported to be associated with T2D include TCF7L2, HHEX/IDE and SLC30A8, CDKN2A/2B, CDKAL1 and FTO. This study aimed to analyze the association of CDKN2A/2B, CDKAL1 and FTO with T2D in a Uyghur population.<SEP>In a case-control study of 879 Uyghur patients with T2D and 895 age- and sex-matched healthy controls, we found that the genotype distributions of the SNPs rs10811661 (CDKN2A/2B) and rs7195539, rs8050136 and rs9939609 (FTO) differed significantly between the T2D and control groups. The allele frequencies of rs8050136 and rs9939609 also differed significantly between the two groups, and allele A appeared to be a risk allele for both SNPs. Genotype distributions of rs10811661, rs7195539, rs8050136, and rs9939609 differed significantly between genetic models, but rs10811661, rs8050136, and rs9939609 were eliminated after adjusting for covariates, and only rs7195539 differed significantly in both co-dominant and dominant models.<SEP>The physiological role of CDKAL1 remains unknown. The amino acid sequence of CDKAL1 is similar to that of CDK5-associated protein 1, which inhibits CDK5 by interacting with its p35 and p39 subunits. Due to the regulatory effects of CDK5 on insulin secretion, CDKAL1 is suspected to influence insulin secretion by regulating the activity of CDK5. Currently, five non-coding SNPs within the CDKAL1 gene (rs4712523, rs10946398, rs7754840, rs7756992, and rs9465871) have been found to associate with T2D; rs7754840 and rs10946398 exhibited the strongest association with T2D, with complete linkage disequilibrium (LD) between the two loci (r2=1.0). European carriers of the GC and CC genotypes of rs7754840 had lower first-phase insulin release than carriers of the GG genotype. On the other hand, a study of Finnish men demonstrated that rs7754840 was significantly associated with T2D; hence CDKAL1 is thought to increase the risk of T2D by impairing insulin secretion. CDKAL1 locus rs10946398 was also found to be associated with T2D in a British population, and rs10946398 was associated with T2D in a Han population in China. Nevertheless, we did not detect a significant difference in these genotype and allele frequencies between T2D and control groups in a Uyghur population (P>0.05), indicating that the common susceptibility loci rs10946398 and rs7754840 in CDKAL1 were not associated with T2D in Uyghurs.<SEP>CDKN2A/2B is located on chromosome 9p21. Both are highly expressed in islet and fat cells, and CDKN2A is also expressed in brain cells. CDKN2A and B are tumor suppressor genes involved in cell proliferation, apoptosis and tumorigenesis, and encode CDK inhibitors p16INK4a and p15INK4b, respectively, which inhibit CDK4 and CDK6. In a murine model, CDK4 influenced proliferation and quality of β cells, yet its activity was reduced by overexpression of CDKN2A, which in turn caused islet hypoplasia and diabetes. CDKN2A/2B has been reported to be associated with the risk of T2D in a GWAS. However, additional studies in Europe and Asia have both confirmed and failed to detect an association between the rs10811661 polymorphism upstream of the CDKN2A/2B gene and T2D. In our study, the genotype distribution of rs10811661 differed significantly between patients in the diabetes group and those in the control group, suggesting that rs1088661 polymorphism may be associated with T2D risk in Uyghurs. The distribution of the rs1088661 genotype also differed significantly in co-dominant, additive and recessive models, indicating an increased risk of T2D in TT genotype carriers. Nevertheless, after adjusting for covariates, such as age, sex, BMI, hypertension, CHD, family history of diabetes, FPG, TC, and HDL, these differences were eliminated suggesting that the locus polymorphism might act via one of the covariates to impact the pathogenesis of T2D in Uyghurs. Several studies have found a correlation between the rs3088440 polymorphism in the CDKN2A/2B gene and the pathogenesis of malignant tumors, including elevated risk of salivary gland carcinoma and melanoma, but not the onset of cervical cancer or differentiated thyroid cancer. We found no significant difference between the genotype distributions and allele frequencies of locus rs3088440 between diabetic patients and controls in Uyghurs, suggesting that in this population rs3088440 is not associated with T2D.<SEP>The nucleic acid demethylation activity of FTO was hypothesized to regulate the expression of genes involved in metabolism, and dysregulation of this process may cause obesity; however, the precise molecular mechanism responsible for this activity remains unclear. Loss of FTO function appeared to reduce appetite control, ultimately leading to obesity. FTO mRNA was found to be abundantly expressed in the brain tissue of wild-type mice, particularly in hypothalamic nuclei governing energy balance. Thus, it is conceivable that the FTO gene may regulate the pathogenesis and development of obesity primarily through its impact on the energy center in the hypothalamus. Few studies have investigated the rs7195539 locus in FTO. Loci rs8050136 and rs9939609 have been reported to associate with T2D in multiple studies, yet there is some inconsistency. This association has been reported to be both mediated by BMI and independent of BMI. In this study, the genotype distributions of loci rs7195539, rs8050136, and rs9939609 in the FTO gene differed significantly between the diabetes and control groups, implying that these polymorphisms are associated with T2D in Uyghurs. In addition, the allele frequencies of loci rs8050136 and rs9939609 differed significantly between the two groups with a common allele A. This indicates that Uyghurs with loci rs8050136 and rs9939609 containing allele A have an escalated risk of T2D. The rs8050136 genotype distributions in co-dominant, additive and recessive models also differed significantly, and the genotype distributions of rs9939609 in co-dominant, additive, dominant, and recessive models differed significantly between the two groups. These results suggest that the risk of T2D is higher in Uyghurs carrying CA or AA genotypes at rs8050136 and/or TA or AA genotypes at rs9939609 than in those carrying the CC or TT genotype, respectively. Nevertheless, these variations were eliminated after adjusting for sex, age, BMI, hypertension, CHD, family history of diabetes, FPG, TC, and HDL, indicating that these SNPs are likely to influence the risk of T2D in Uyghurs by affecting BMI.<SEP>To date, few GMDR analyses of gene-gene and gene-environment interactions between the CDKAL1, CDKN2A/2B, and FTO genes and T2D have been reported. We found that, after adjusting for demographic and environmental factors, the success of the two-locus model rs10811661–rs7195539 indicated a potential association between these two loci and T2D in Uyghurs. The three-locus interaction model TG-HDL-family history of diabetes appeared to be the best model for analyzing gene-environment interactions after assigning SNPs, biochemical characteristics, family history of diabetes, smoking and alcohol history and work pressure as candidate variables, and age, gender, FPG, hypertension, and CHD as covariates. No gene-environment interactions were observed. The next best model was the four-locus interaction model of rs7195539-TG-HDL-family history of diabetes, suggesting that interactions among these factors may be associated with T2D in Uyghurs.",
                  "26989",
                  "['OBWCCE']"
                ],
                [
                  "16",
                  "23990951",
                  "Introduction<SEP>The number of people with diabetes grows worldwide. In the past few decades, China has experienced a dramatic increase in diabetes incidence. According to the recent Chinese National Diabetes and Metabolic Disorders Study (CNDMDS) performed in 2007–2008, about 92.4 million adults (9.7% of the adult population) in China have diabetes. Type 2 diabetes is the most common type of diabetes, caused by an interaction of multiple genetic and environmental factors.<SEP>Newly diagnosed diabetes generally includes IFH, IPH or combined fasting and postprandial hyperglycaemia (FH/PH). According to the DECODE study, IFH and IPH accounted for 40.4% and 28.4% of newly diagnosed diabetes in Caucasians, respectively. In a study by Yang et al. , the prevalence of IPH was reported to be 47%, and that of IFH was 17% in newly diagnosed cases of diabetes, indicating IPH was the major type of diabetes in Han Chinese, which is a markedly different profile compared to Caucasians.<SEP>It was reported that IFH and IPH are derived from isolated impaired fasting glucose and isolated impaired glucose tolerance respectively. Though the etiology of IFH and IPH is unclear, their heterogeneity in clinical manifestation has been well described. A higher level of insulin resistance within the liver is associated with IFH; whereas, impairment of early phase insulin secretion and total insulin secretion in response to glucose were worse in subjects with IPH. Also, there were significant differences in duration, drug therapy, chronic complications and mortality between IFH and IPH subjects. According to the DECODE study, postprandial glucose concentrations of subjects with type 2 diabetes were found to be positively associated with incidence of cardiovascular disease, independent of fasting glucose. However, fasting hyperglycaemia was not found to be predictive of the incidence of cardiovascular disease in patients with type 2 diabetes. Therefore, we speculate that IFH and IPH have different genetic etiology.<SEP>It has been reported that all subjects with newly diagnosed IFH or IPH ultimately develop fasting hyperglycaemia combined with postprandial hyperglycaemia over the course of decades. In addition, type 2 diabetes patients recruited in hospital-based studies cannot be distinguished to have IFH or IPH when they are first diagnosed. Over the past decade, many studies have attempted to elucidate susceptibility genes for type 2 diabetes. To date, more than 50 genes have been found to be involved in the pathogenesis of type 2 diabetes, mostly in Caucasian populations. However, no susceptible genetic loci of IFH or IPH have been investigated due to difficulties in sampling. It is worthwhile to determine whether common genetic variations play a role in their pathogenesis, as well as to distinguish the common and different genetic basis of IFH and IPH.<SEP>In this study, we examined the association of 27 SNPs in GWAS-validated type 2 diabetes susceptible variants in patients with IPH or IFH. The study population was a nationally representative cohort of newly diagnosed patients of Han ancestry recruited during CNDMDS conducted in 2007–2008. Further, we compared the difference of genetic basis of IFH and IPH.",
                  "Methods<SEP>Ethics Statement<SEP>The study protocol was approved by the Ethics Committee of China-Japan Friendship Hospital in Beijing and was in accordance with Helsinki Declaration II. Written informed consent was obtained from all participants before data collection.<SEP>Participants<SEP>All samples were recruited from CNDMDS. After the exclusion of subjects with type 1 diabetes and special type diabetes, a total of 6,663 unrelated Han people from 13 provinces and municipalities participated in the final analysis, including 341 with IFH, 865 with IPH, 1,203 with combined FH/PH, and 4,254 control subjects. Each participant received a standard 75 g oral glucose tolerance test (OGTT). Type 2 diabetes was defined by 1999 WHO criteria. IFH was defined as fasting plasma glucose (FPG) ≥7 mmol/l (126 mg/dl) and 2-hour plasma glucose in OGTT (2 h PG) <11.1 mmol/l (200 mg/dl). IPH was defined as FPG <7.0 mmol/l and 2 h PG ≥11.1 mmol/l. Combined FH/PH was defined as FPG ≥7 mmol/l and 2 h PG ≥11.1 mmol/l. The inclusion criteria for normoglycaemic control were: 1) age over 40 years; 2) normal glucose regulation in OGTT according to WHO criteria (FPG <6.1 mmol/l (110 mg/dl) and 2 h PG <7.8 mmol/l (140 mg/dl)); 3) no diabetes history or family history of diabetes; 4) BMI <28 kg/m2; 5) normal blood pressure (systolic blood pressure <140 mmHg and diastolic blood pressure <90 mmHg); 6) normal blood lipid levels (triglycerides (TG) <1.7 mmol/l and high density lipoprotein-cholesterol ≥1.0 mmol/l). Clinical characteristics of the study groups are shown in Table 1. Normally distributed data are shown as mean ± SD while non-Gaussian data are shown as median and interquartile range. Non-fatal cardiovascular diseases were determined as previously reported.<SEP>Clinical examinations<SEP>Anthropometric and biochemical characteristics were carefully examined. Height, weight, waist circumference (WC) and hip circumference were measured with subjects lightly clothed, from which BMI and waist-hip-ratio were calculated. Without glucose-lowering treatment, subjects were requested to fast from food for more than 10 hours and given a standard 75 g OGTT the next day. Blood samples were drawn at 0, 30 and 120 minutes after OGTT to measure plasma glucose and serum insulin concentrations. Serum insulin was measured by double-antibody radioimmunoassay. HOMA was used to estimate insulin resistance (HOMA-IR: fasting serum insulin (mU/l) × FPG (mmol/l)/22.5) and beta cell function (HOMA-B: fasting serum insulin (mU/l) ×20/(FPG (mmol/l) –3.5)). Insulinogenic index (ΔI30/ΔG30) was calculated from fasting and 30-min serum insulin (mU/l) and plasma glucose (mmol/l) during OGTT using (Ins30 – Ins0)/(Glu30 – Glu0). The compensation of beta cells to insulin resistance was calculated using ΔI30/ΔG30/HOMA-IR. Matsuda index (ISIm) was calculated as 10,000/(Glu0 (mg/dl) × Ins0 (mU/l) × mean glucose OGTT (mg/dl) × mean insulin (mU/l))1/2.<SEP>Total cholesterol, TG, high density lipoprotein-cholesterol and low density lipoprotein-cholesterol in fasting serum were tested using an automatic biochemical analyzer (Olympus, Tokyo, Japan).<SEP>Joint study of IPH associated SNPs, including TCF7L2, CDKAL1, KCNQ1, PRC1, TP53INP1 and GCKR, shows that individuals with IPH carry more risk alleles from the above loci than controls (ptrend <0.0001; Figure 2A). Risk of IPH increased 19.0% for each additional risk allele carried (1.190 [1.120–1.263] per allele, p<0.0001; Figure 2B). In SNPs associated with IPH, the numbers of risk alleles carried in IFH subjects were not significantly more than that in the control group (ptrend  = 0.2752; Figure 2A); and the number of risk alleles did not increase the risk of IFH (1.053 [0.963–1.150] per allele, p = 0.2564; Figure 2C).<SEP>Genotyping<SEP>Genomic DNA samples were isolated from the peripheral blood using a DNA extraction kit. Genotyping was accomplished with Illumina GoldenGate Indexing assay (Illumina Inc., San Diego, USA), used according to the manufacturer's instructions. We selected 31 common SNPs from 29 type 2 diabetes susceptible loci in the Caucasian population, including KCNQ1 (rs2237895, rs231362), CDKAL1 (rs7756992), CDKN2BAS (rs10811661), TCF7L2 (rs7903146), HHEX (rs1111875), TCF2 (rs7501939), WFS1 (rs10010131), CDC123/CAMK1D (rs12779790), MTNR1B (rs10830963), FTO (rs8050136, rs9939609), ADAMTS9 (rs4607103), JAZF1 (rs864745), TSPAN8/LGR5 (rs7961581), TP53INP1 (rs896854), PRC1 (rs8042680), GCKR (rs780094), ZFAND6 (rs11634397), BCL11A (rs243021), KLF14 (rs972283), ZBED3 (rs4457053), HNF1A (rs7957197), CENTD2 (rs1552224), NOTCH2 (rs10923931), CHCHD9 (rs13292136), PPARG (rs1801282), THADA (rs7578597), SLC30A8 (rs13266634), HMGA2 (rs1531343), DUSP9 (rs5945326). Only the SNPs with Genotyping success rate >90% were investigated (rs231362, rs13266634, rs1531343 and rs5945326 were excluded from the study for their genotyping success rate was lower than 90%). The overall mean calling rate of 27 SNPs was 99.47%. The concordance rate based on 229 genotyping duplication was 100%.<SEP>Statistical analyses<SEP>Hardy Weinberg equilibrium test was done for each SNP by χ2 test before further analysis. Difference in allele frequency was determined by χ2 test, with OR and 95% CI estimated.<SEP>The association study was performed assuming an additive model. Genotype distributions between case and control were compared using logistic regression adjusted for geographical region, age and gender. OR and 95% CI were calculated before and after adjustment for BMI. Quantitative characteristics of normoglycaemic or newly diagnosed type 2 diabetes individuals were analyzed by linear regression adjusted for geographical region, age and gender, while BMI was adjusted when appropriate (apart from anthropometric traits including height, weight, WC, hip circumference, waist-hip-ratio and BMI), and regression coefficients (βs) were determined. All non-Gaussian distributed quantitative traits were subjected to natural logarithm transformation to approximately approach normal distribution. The association studies were adjusted for multiple testing by a permutation step of 1,000 times using the PLINK analysis software. χ2 test and Cochran-Armitage test was used to analyze differences of risk allele frequency and genotype distribution between IFH and IPH.<SEP>In order to determine the joint effects of associated SNPs (IFH associated SNPs included KCNQ1, CDKN2BAS, TCF7L2, FTO (rs8050136), GCKR, except FTO (rs9939609) which was from the same linkage disequilibrium block with rs8050136; IPH associated SNPs included KCNQ1, CDKAL1, TCF7L2, TP53INP1, PRC1, GCKR) on the risk of IFH and IPH, we categorized the individuals based on risk alleles they carried, as previous reported. The analysis included individuals for whom genotypes of involved SNPs were available (sample size for IFH SNPs was 291/767/4060 (IFH/IPH/Control); sample size for IPH SNPs was 293/771/4053). Considering the association results, we defined the G allele of rs780094 as a risk allele of IFH SNPs, while A allele was the risk allele for IPH SNPs. Individuals were grouped based on the amount of risk alleles they carried. For the joint study of IFH associated SNPs, individuals were divided into 5 groups (≤1, 2, 3, 4, ≥5), while there were 6 groups for IPH SNP analysis (≤3, 4, 5, 6, 7, ≥8). The group carrying fewest risk alleles was considered as reference (OR = 1). Effective sizes for every unit increase in the number of risk alleles on IFH or IPH were calculated after adjustment for geographical region, age and gender using a logistic regression model.<SEP>All statistical analyses were performed using either SAS for Windows (version 9.2; SAS Institute, Cary, NC) or PLINK software (v1.05). All p values <0.05 (two-tailed) were considered statistically significant.<SEP>Power calculations were performed using Quanto software (available at http://hydra.usc.edu/gxe/) and shown in Table S7. Power was calculated using the ORs reported in the original studies as well as sample size and minor allele frequencies (MAF) in the present study. An overall diabetes risk of 9.7% among Chinese was used in power analysis of newly diagnosed type 2 diabetes. In addition, for power calculation of IFH and IPH, overall IFH and IPH risk (1.65%, 4.56%) were calculated from 9.7% multiplying by the proportion of IFH and IPH in newly diagnosed type 2 diabetes (17%, 47%).",
                  "Results<SEP>Association study of 27 SNPs and newly diagnosed type 2 diabetes in Han Chinese<SEP>All SNPs were within Hardy-Weinberg equilibrium in controls (p>0.05; Table S1). TCF7L2, CDKN2BAS, CDKAL1, KCNQ1, HHEX, TCF2, CENTD2 and TP53INP1 showed associations with type 2 diabetes (ORs ranged between 1.089–1.385, p value ranged between 4.112×10−7–0.0311; Table S2). Three of the eight SNPs remained significant after adjustment for multiple comparisons (p value ranged between 0.0020–0.0080; Table S2).<SEP>The association study of SNPs and type 2 diabetes related quantitative traits showed that subjects carrying the CDKAL1 risk allele had higher plasma glucose 30 min in OGTT (β = 0.01669 per G allele for lnFPG; p = 0.0006). The correlation remained after multiple test correction (Empirical p = 0.0140). In cases of newly diagnosed type 2 diabetes, subjects carrying the KCNQ1 risk allele had lower BMI (β = −0.01738 per for C allele for lnBMI; p = 0.0001), and this correlation remained after multiple test correction (Empirical p = 0.0070) (Table S3).<SEP>Clinical characteristics of individuals with IFH and IPH<SEP>As shown in Table 1, individuals with IFH had higher FPG, liver insulin resistance (evaluated by HOMA-IR and ISIm) and better early phase insulin secretion ability (evaluated by ΔI30/ΔG30) compared to subjects with IPH. 2h PG of individuals with IPH was higher, as well as higher prevalence of cardiovascular disease. HOMA-B was lower in IFH than IPH, which was calculated from FPG and fasting insulin level.<SEP>Association study of SNPs and IFH in Hans<SEP> Table 2 shows that TCF7L2, CDKN2BAS, KCNQ1, FTO and GCKR are significantly associated with IFH (OR ranged between 1.171–1.524; p value ranged between 0.0023–0.0476). When adjusted for region, age and gender, TCF7L2 and FTO (rs8050136, rs9939609) risk alleles conferred 1.454-, 1.403- and 1.398-fold increased risk of IFH. After further adjustment for BMI, the significant association between IFH with TCF7L2 or CDKN2BAS remained (rs7903146: 1.503 [1.023–2.208], p = 0.0379; rs10811661: 1.220 [1.034–1.441], p = 0.0188), while KCNQ1 was no longer associated with IFH (1.194 [0.991–1.439], p = 0.0617). SNPs in FTO were not associated with IFH after adjustment for BMI (rs8050136: 1.263 [0.995–1.604], p = 0.0554; rs9939609: 1.253 [0.987–1.590], p = 0.0642) with a reduction in OR values.<SEP>Joint study of IFH associated SNPs, including TCF7L2, CDKN2BAS, KCNQ1, FTO and GCKR, shows that individuals with IFH carry more risk alleles from the above loci than controls (ptrend <0.0001; Figure 1A). Risk of IFH increased 24.6% for each additional risk allele carried (1.246 [1.129–1.376] per allele, p<0.0001; Figure 1B). However, individuals with IPH did not carry more risk alleles than controls (ptrend  = 0.2208; Figure 1A). The number of risk alleles did not increase the risk of IPH (1.037 [0.971–1.107] per allele, p = 0.2776; Figure 1C).<SEP>Association study of SNPs and IPH in Han Chinese<SEP> Table 3 shows that TCF7L2, CDKAL1, KCNQ1, and PRC1 are significantly associated with IPH (OR ranged between 1.154–1.709, p value ranged between 0.00038–0.03482). Another two SNPs, TP53INP1 and GCKR, were also found to be associated with IPH after adjustment for region, age and gender (rs896854: 1.129 [1.009–1.263], p = 0.0348; rs780094: 0.891 [0.801–0.991], p = 0.0331). The PRC1 and TCF7L2 risk alleles conferred a 1.758- and 1.485-fold of increased risk of IPH, respectively. Risk alleles in CDKAL1, KCNQ1 and TP53INP1 conferred a 1.217-, 1.153- and 1.129-fold of increased risk of IPH, respectively. Noticeably, carriers of the validated type 2 diabetes risk allele G of GCKR showed lower risk of IPH; therefore the A allele of GCKR was defined as the risk allele in further joint study. TCF7L2, CDKAL1 and KCNQ1 remained associated with IPH after adjustment for BMI (rs7903146: 1.438 [1.080–1.888], p = 0.0123; rs7756992: 1.233 [1.095–1.389], p = 0.0005; rs2237895: 1.181 [1.038–1.343], p = 0.0115). GCKR, TP53INP1 and PRC1 were not associated with IPH after adjustment for BMI (rs780094: 0.895 [0.796–1.006], p = 0.0627; rs896854: 1.099 [0.971–1.245], p = 0.1360; rs8042680: 1.555 [0.917–2.639], p = 0.1016).<SEP>Difference of allelic frequencies and genotype distributions between IFH and IPH<SEP>Among all tested SNPs, risk allele frequencies of SNPs in FTO and GCKR in IFH were much higher than those in IPH (A allele of rs8050136, p = 0.0026; A allele of rs9939609, p = 0.0053; G allele of rs780094, p = 0.0043; Table 4). However risk allele frequencies of SNPs in CDKAL1 were much lower in IFH subjects than IPH subjects (G allele of rs7756992, p = 0.0103). Genotype distributions of these SNPs between IFH and IPH were significantly different (rs7756992 ptrend  = 0.0122; rs8050136 ptrend  = 0.0024; rs9939609 ptrend  = 0.0049; rs780094 ptrend  = 0.0037). Allele frequencies or genotype distributions of CDKN2BAS, TP53INP1 and PRC1 were statistically comparable between IFH and IPH (Table S6).",
                  "Discussion<SEP>Current understanding on the genetic etiology of type 2 diabetes has been greatly improved over past decades. Considering the high heterogeneity of IFH and IPH, we proposed the hypothesis that genetic bases of IFH and IPH are different. Further, we speculated that genetic variants in loci modulating insulin sensitivity promoted IFH, while variants in loci affecting beta cell function promoted IPH.<SEP>The present study indicated that gene variants from FTO and GCKR were specifically associated with increased risk of IFH, while gene variant from CDKAL1 was associated with increased risk of IPH. It also showed that risk allele frequencies of FTO and GCRK in IFH were higher than that in IPH. However, risk allele frequency of CDKAL1 in IFH was lower than that in IPH.<SEP> FTO was initially found to be associated with obesity in Caucasians. In this study, we also confirmed the association of FTO with obesity (shown as WC and BMI) in newly diagnosed type 2 diabetes. It was reported that the fat mass and obesity associated protein, which is the genetic product of FTO, plays a role in the regulation of insulin sensitivity and insulin secretion from pancreatic beta cells. Berulava et al.  found that SNP rs9939609 influenced the transcription of FTO. FTO SNPs rs8050136 and rs9939609, located in the same linkage disequilibrium block, were first found to be associated with type 2 diabetes in Caucasians. According to Hapmap, risk allele (A) frequency of rs8050136 and rs9939609 are much higher in Caucasians (0.450), compared to East Asians (0.144). Studies in the Han Chinese population failed to establish consistency of association between FTO variants and type 2 diabetes. Further, no association remained between FTO and type 2 diabetes after BMI adjustment in some studies, indicating that FTO variants were associated with increased risk of type 2 diabetes through modulating the process of obesity development. However, some reports do not support this viewpoint. It had been well accepted that these contradictory findings result from the low MAF in Han Chinese, which results in insufficient statistical power.<SEP>In the present study, we found that SNPs in FTO are associated with a 40% increased risk for IFH. Furthermore, we found OR decreased to 25–26% after adjustment for BMI (p>0.05, as the sample size of IFH was not sufficient large). On the other hand, SNPs in FTO were not associated with IPH (Table S5). We postulate that risk alleles of SNPs in FTO (rs8050136, rs9939609) or causal variants genetically linked with them may specifically promote risk of IFH in Han Chinese, which is partially independent of general obesity (evaluated by BMI).<SEP>Based on these observations, we speculate that previous findings of the association between FTO and type 2 diabetes may have resulted from the different proportions of IFH and IPH in those type 2 diabetes populations. According to DECODE and DECODA, the ratios of IFH/IPH were 613/473 (40.4%/28.4%) in Caucasians and 220/546 (18.1%/44.9%) in Asians. In the CNDMDS performed in 2007–2008, the ratio of IFH/IPH was 17%/47% (unpublished data) in Han Chinese. Previous studies have demonstrated that insulin resistance in Caucasians is much higher than Han Chinese, while Han Chinese typically have worse beta cell function than Caucasians. Therefore, FTO is well established as a susceptibility gene of type 2 diabetes in Caucasians, but not in Chinese. No association between FTO and type 2 diabetes was observed in our population of newly diagnosed patients.<SEP>Glucokinase regulatory protein is the product of GCKR genetic encoding, whose overexpression could ameliorate insulin sensitivity and glucose tolerance in mice, while leading to higher serum TG. GCKR gene polymorphisms were initially reported to be associated with TG levels. In the present study, we confirm that the G allele of rs780094 in GCKR is associated with increased FPG and lower TG levels in newly diagnosed type 2 diabetes of Han ancestry (Table S3).<SEP> GCKR (rs780094) was first found to be associated with type 2 diabetes in Caucasians. A nonsynonymous variant of GCKR (rs1260326, P446L) has strong linkage disequilibrium with rs780094 according to HapMap II data (CEU: r2 = 0.93; CHB/JPT: r2 = 0.83). This variant could regulate insulin secretion and blood TG levels through regulating the activity of glucokinase in the liver, while it was also correlated with type 2 diabetes. In Han Chinese, the association of rs780094 with type 2 diabetes did not allow for consistent conclusions to be drawn. Notably, we found that the G allele was associated with increased risk of IFH, but decreased risk of IPH. Therefore, it's not surprising that we cannot confirm a correlation between rs780094 and newly diagnosed type 2 diabetes. We suppose that risk alleles of causal variants at the same linkage disequilibrium region with rs780094 specifically promote the progression of IFH. Therefore, the proportion of IFH and IPH adds to the debate over whether GCKR is a susceptibility gene for type 2 diabetes in Han Chinese.<SEP>Genetic product of the CDKAL1 gene belongs to the mammalian methylthyiotransferase family, and is involved in the process of synthesis and cleavage of proinsulin by ensuring the accurate translation of Lys codons. It has been reported that the proinsulin/insulin conversion rate of CDKAL1 (rs7754840) risk allele carriers were lower. Variations in CDKAL1 have been reported to be associated with impaired insulin secretion and the increased risk of type 2 diabetes. The association of CDKAL1 (rs7756992) with type 2 diabetes has been well established in Han Chinese. In this study, we found that G allele of rs7756992 was associated with increased risk of IPH, rather than IFH (Table S4). Though the sample size of IFH cases was smaller than that of IPH, allele frequencies and genotype distribution of rs7756992 were significantly different and the power to detect association of IFH and rs7756992 reached 79.49% (Table S7). Moreover, it was reported that rs7756992, a common variant in CDKAL1 loci (MAF  = 0.479), contributed to type 2 diabetes risk in Han Chinese in a study with a similar sample size. Therefore, we speculate that the risk allele of rs7756992 was specifically associated with increased risk of IPH. Association of rs7756992 with newly diagnosed type 2 diabetes was also confirmed in present study, which likely resulted from the high proportion of IPH in the Han Chinese population.<SEP>We also observed that IFH and IPH share common susceptible SNPs including TCF7L2 and KCNQ1, both of which played approximately equal effect sizes in IFH and IPH subjects, respectively. TCF7L2 and KCNQ1 were previously considered to be the strongest type 2 diabetes susceptible genes in Caucansians and East Asians, both of which were well replicated in Han Chinese. Both TCF7L2 and KCNQ1 have been shown to play essential roles in beta cell survival and function, and they have been suggested to be involved in the modulation of insulin sensitivity and obesity. Based on our observations, causal variants in linkage disequilibrium with rs7903146 and rs2237895 were the common hereditary bases of IFH and IPH, resulting in their similar clinical features. In addition, Yu, et al. demonstrated that the C allele of rs2237895 showed a greater effect on diabetes risk in participants with lower BMI, as well as a correlation between BMI and rs2237895. Our results, and other studies, support the idea that subjects carrying risk alleles at rs2237895 have lower BMI in newly diagnosed type 2 diabetes patients.<SEP>Though we confirm that CDKN2BAS is associated with IFH, while PRC1 and TP53INP1 are associated with IPH, the allele frequencies of the three SNPs were not found to be different between IFH and IPH (Table S6). However, it should not be ignored that only 341 IFH individuals were recruited, resulting in a lower or insufficient statistical power for IFH (Table S7). Further investigation of large samples at these loci is warranted to elucidate their association with IPH or IFH.<SEP>We observed that the five variants associated with IFH were additively associated with the increased risk of IFH. Moreover, six variants associated with IPH were additively associated with the increased risk of IPH. Despite of the common hereditary bases, different joint effects of SNPs on IFH and IPH groups also reflect hereditary differences.<SEP>The present study has some limitations to be addressed. First, the study could not achieve enough statistical power for several SNPs because of the limited sample size (in particular, the sample size of incident IFH was relatively small). Moreover, all SNPs involved in the study have been validated to be type 2 diabetes susceptible SNPs previously; thus novel loci could not be established by our work. However, this is the first study to investigate the susceptibility genes of IFH and IPH based on a general population sample which gave us samples of IFH and IPH. It also provides clues regarding targeted regions for future genetic research on IFH and IPH.<SEP>In conclusion, our study indicates that both distinct (FTO, GCKR, CDKAL1) and common genetic bases (TCF7L2, KCNQ1) exist for IPH and IFH in Han Chinese, suggesting specific underlying pathogenic mechanisms resulting in the heterogeneity of their clinic manifestation. Given the differences in risk allele frequencies, it will be valuable to further examine these genes thoroughly to search for the culprit disease loci of IPH and IFH. It also provides strong evidence for prediction, prevention, diagnosis, personalized medicine and drug development for Han Chinese with type 2 diabetes, among which most of the newly diagnosed type 2 diabetes patients have IPH.",
                  "26431",
                  "['OBWCCE']"
                ],
                [
                  "17",
                  "33910581",
                  "Background<SEP>Body fat percentage (BF%) is a proxy for adiposity that is genetically regulated through a leptin or melanocortin pathway in the central nervous system (CNS). Recent studies found that high BF%, especially with excess visceral adipose tissue, is associated with increased risk of cardiovascular diseases (CVDs), independent of body mass index (BMI). BF% is biologically important and distinct from other proxies of adiposity such as BMI and waist-to-hip ratio for its capacity to differentiate between fat-free mass (i.e., lean mass, bone mass and fluid mass) and fat mass. One hypothesis to account for the link between high BF% and CVDs is shared genetic etiology. BF% and CVDs may share common genetic variants that influence metabolism or response to environmental risk factors. While the genetic basis for BF% and CVDs is poorly understood, large-scale genome-wide association studies (GWAS) may provide novel insight into specific biological processes underlying their comorbidity.<SEP>Genetic correlation analysis estimates the correlation of genetic effects between two clinically related traits and highlights the shared etiologies behind such an association. With methodological advances in molecular genetics and the increased number of available GWAS results, it is now feasible for us to investigate genome-wide genetic correlation and identify significant expression-trait associations for complex traits by using genomics resources (i.e., summary-level statistics from large GWAS and the Genotype-Tissue Expression (GTEx) project) and state-of-the-art statistical analysis methods (i.e., linkage disequilibrium score regression (LDSC), multi-trait analysis of GWAS (MTAG), and transcriptome-wide association studies (TWAS)). Furthermore, previous twin and family studies have shown that BF% and CVDs are heritable traits, with heritability estimates ranging from 25 to 40%. Large-scale GWAS have enabled detection of more than 20 susceptibility loci for BF%. In particular, the identified BF%-related loci near FTO predicts long-term CVD risk (OR = 1.895), suggesting that BF% and CVDs might share genetic architecture. Previous post-GWAS analyses conducted in a large European population found that genetically predicted BMI was strongly associated with several CVD outcomes, including heart failure (HF) and coronary artery disease (CAD). Further, there was suggestive evidence of associations between genetically predicted fat mass index and some CVDs. However, these studies are limited because BMI is an imperfect measure of adiposity which does not directly measure body fat. To our knowledge, no large-scale genome-wide study has systematically reported the shared genetic loci between excess adiposity and CVDs, which is not fully accounted for through BMI evaluation.<SEP>Although observational studies have reported associations between BF% and adverse cardiovascular outcomes, some of the findings have been inconsistent, which may be due to biases such as unmeasured confounding. Mendelian randomization (MR) is a form of instrumental variable analysis that can be used to estimate the causal association under certain assumptions, even in the presence of unmeasured confounders. HF is a catabolic state that can lead to weight reduction, which indicates that the association between body weight and HF could be bidirectional. Given that inherited genetic variants are unlikely to be influenced by reverse causation or environmental confounders after accounting for population stratification, investigating the directions of these associations is crucially important.<SEP>Using genome-wide association study (GWAS) summary-level data from several international consortia (n = 6968-977,323), we investigated the genetic correlation and causality between BF% and 10 CVD-related traits with the overarching goal of characterizing the specific shared genetic loci and biological pathways. Further, we conducted a large-scale, genome-wide cross-trait analysis to explore novel genetic components among these diseases. The biological effects reflected by shared loci may play important roles in the co-occurrence of high BF% and CVDs.",
                  "Methods<SEP>Study design, data summary and quality control<SEP>The overall study design is shown in Fig. 1. We retrieved summary statistics from the Genetic Investigation of ANthropometric Traits (GIANT) consortium for BF% (n = 100,716); the Heart Failure Molecular Epidemiology for Therapeutic Targets (HERMES) for HF (47,309 cases and 930,014 controls); the Coronary ARtery Disease Genome wide Replication and Meta-analysis (CARDIoGRAM) plus the Coronary Artery Disease (C4D) Genetics (CARDIoGRAMplusC4D) consortium for CAD (60,801 cases and 123,504 controls); and myocardial infarction (MI) (43,676 cases and 128,197 controls), respectively; from the NINDS Stroke Genetics Network (SiGN) and International Stroke Genetics Consortium (ISGC) for ischemic stroke (IS) (37,792 cases and 397,209 controls); from the AFGen Consortium for atrial fibrillation (AF) (65,446 cases and 522,744 controls); from the Genetics of Cerebral Hemorrhage with Anticoagulation (GOCHA) study and the Genetic and Environmental Risk Factors for Hemorrhagic Stroke (GERFHS) studies for intracerebral hemorrhage (ICH) (3226 cases and 3742 controls); and from the Global Lipids Genetics Consortium (GLGC) consortium (n = 188,578) for lipids such as high-density lipoprotein (HDL), low-density lipoprotein (LDL), total cholesterol (TC), and triglycerides (TG). Each study has done the study-specific quality control to ensure the criteria of BF% measurement and CVDs similar among the different studies. Baseline characteristics for each study are also provided in previous studies. For example, BF% in each cohort was measured either with bioimpedance analysis or dual energy X-ray absorptiometry as described in detail before. In addition, CAD status was defined by an inclusive CAD diagnosis, including MI, chronic stable angina, acute coronary syndrome, or coronary stenosis > 50%. HF cases included participants with a clinical diagnosis of HF of any etiology with no inclusion criteria based on LV ejection fraction. Details of each of the GWAS studies are present in Table S1. <SEP>The ethical approval and quality control procedures of each consortium have been described in previous studies. Informed consent was obtained from all participants of contributing studies. In addition, we restricted the chromosome region to autosomal chromosomes and excluded single-nucleotide polymorphisms (SNPs) in MHC region (chr6:25 Mb–34 Mb).<SEP>LDSC analysis<SEP>We performed a post-GWAS genome-wide genetic correlation analysis for BF% and CVD-related traits using the LDSC software by assuming that the effect size for each SNP in GWASs represents all SNPs in LD with it. The genetic correlation of two traits (ranging from − 1 to 1) can be computed on LDSC software using the known LD structure of European ancestry reference data from the 1000 Genomes Project. Since low imputation quality may yield lower test statistics, we restricted our analyses to HapMap3 SNPs which seem to be well-imputed in most studies to minimize the bias in our analysis. The mean chi-squared statistics in LDSC is higher in high-LD region compared with low-LD region and cannot be further analyzed when it is below 1.02. The analysis also provides a self-estimated intercept to show the sample overlap between single-trait GWASs.<SEP>Partitioned genetic correlation analysis<SEP>The genetic correlation between BF% and CVDs was partitioned by 13 functional category using partitioned LDSC, including conserved region, DNaseI digital genomic footprinting region (DGF), DNase I hypersensitivity sites (DHSs), fetal DHSs, intron region, repressed region, super enhancers, transcription factor binding site (TFBS), transcribed region, and histone marks H3K4me1, H3K4me3, H3K9ac, and H3K27ac from Roadmap Epigenomics Project. We recalculated the LD scores of SNPs assigned to specific annotation which was used to evaluate the BF%-CVDs genetic correlation for each functional category. Additionally, we conducted another genetic correlation analysis partitioned by SNPs groups at different significant level. The P value of a single SNP was defined a larger one in two GWASs, and then sorted and divided it into five groups in quartile, to further test the proportion of observed genetic correlation explained by each group and the robustness of our findings.<SEP>Cross-trait meta-analysis<SEP>We applied MTAG, a novel approach for conducting meta-analysis of summary statistics from GWAS of multiple traits robust to sample overlap, to identify genome-wide significant loci between BF% and CVD-related traits, and detect novel genes by combining GWASs of two correlated diseases. The key assumption of MTAG is that all SNPs share the same variance-covariance matrix of effect sizes among traits. As Turley et al. initially described in 2018, MTAG is a consistent estimator whose effect estimates always have a lower genome-wide mean-squared error than the corresponding single-trait GWAS, even if the assumption is not satisfied. In addition, association statistics from MTAG also yield more statistical power and little inflation of the false discovery rate for each trait analyzed with high correlation, matching theoretical expectations.<SEP>Fine-mapping credible set analysis<SEP>For each of the shared loci between BF% and HF or CAD that meet the cross-trait meta-analysis significance criteria, we extracted variants within 500 kb of the index SNP and then identified a 99% credible set of causal SNPs using the Bayesian likelihood fine-mapping algorithm. This algorithm only maps the primary signal and uses flat prior with steepest descent approximation to identify causal variants, which may reveal molecular mechanisms behind the associations. Details of the method have been described in previous studies.<SEP>Co-localization analysis<SEP>We first extracted summary statistics for variants within 500 kb of the index SNP at each of the shared loci between BF% and HF or CAD and then used R “coloc” package to perform genetic co-localization analysis to calculate the probability that the two traits shared a common genetic causal variant (H4). We conducted fully Bayesian co-localization analysis using the function “coloc.abf” in the R “coloc” package, which requires regression coefficients for each SNP and variance of these regression coefficients for each trait when only summary data are available. In the present study, we considered loci with probability greater than 0.5 to be co-localized.<SEP>GTEx TSEA<SEP>To test if shared gene sets were highly enriched or specific expressed in a tissue, we conducted a tissue-specific expression analysis (TSEA). The analysis was based on the gene lists that were identified from cross-trait meta-analysis with a matching HUGO Gene Nomenclature Committee (HGNC) name. The gene expression data used in TSEA was collected using published RNA-Seq data from the GTEx project. The raw GTEx data was derived from 189 post-mortem subjects consisting of 1839 samples from 45 different tissues. Considering the small sample size of GTEx data, we selected suggestive significant loci for this analysis (Pmeta < 1 × 10− 4) to ensure the robustness of TSEA results. In addition, we used Benjamini–Hochberg correction to account for multiple testing.<SEP>Over-representation enrichment analysis<SEP>To obtain biological insights for identified shared genes (Pmeta < 5 × 10− 08) from cross-trait meta-analysis, we used the PANTHER tool to access enrichment of the gene sets in the Gene Ontology (GO) biological process and Reactome pathway. Benjamini–Hochberg procedure was used to account for multiple testing (false discovery rate < 0.05).<SEP>Bidirectional MR<SEP>We performed a bidirectional MR analysis between BF% and HF and CAD since they are genetically correlated, using inverse-variance weighted (IVW) as the primary method. Median-based methods (simple and weighted), MR-Egger, MR-Robust Adjusted Profile Scores (MR-RAPS), and MR-Pleiotropy Residual Sum and Outlier (MR-PRESSO) methods were used as sensitivity analyses. The intercept of MR-Egger can be explained as a test of overall unbalanced horizontal pleiotropy. In addition, we also performed single-SNP and leave-one-out analysis to determine whether there was any single SNP that might drive the IVW point estimate. For instrumental variables, we have used the largest and latest GWASs for these traits. We only selected independent genetic variants which are not in linkage disequilibrium (LD) (defined as r2 < 0.1) with other genetic variants based on European ancestry reference data from the 1000 Genomes Project. We chose the variant with the lowest P value for association with the exposure when genetic variants were in LD. For SNPs that were not available in outcome GWASs, we used the LD proxy search on the online platform (https://snipa.helmholtz-muenchen.de/snipa3/index.php/) to replace them with the proxy SNPs identified in high-LD (r2 > 0.8) or discard them if the proxies were not available.<SEP>TWAS<SEP>To identify associations of BF% and CVDs with gene expressions in specific tissues, we conducted a TWAS using FUSION software package based on 44 post-mortal GTEx (version 6) tissue expression weights. We applied Benjamini-Hochberg correction on TWAS P values of all gene-tissue pairs for each trait, and false discovery rate < 0.05 was considered significant.<SEP>Availability of data and materials<SEP>All data used in the present study were obtained from genome-wide association study summary statistics which were publicly released by genetic consortia.<SEP>Declarations<SEP>Consent to publication<SEP>Not applicable.<SEP>Ethics approval and consent to participate<SEP>Not applicable.",
                  "Results<SEP>Genome-wide genetic correlation<SEP>We found strong positive genetic correlations with BF% for both HF (Rg = 0.47, P = 1.27 × 10− 22) and CAD (Rg = 0.22, P = 3.26 × 10− 07) in large study populations that were predominantly of European ancestry (more than 75%) (Table 1, Table S1). Additionally, we found nominally significant genetic correlation with BF% for ICH (Rg = 0.29; P = 0.021) and HDL (Rg = − 0.326; P = 0.048). However, we did not find evidence of genetic correlation between BF% and IS, MI, AF, TC, TG, and LDL (all P > 0.05) (Table 1). Estimates of SNP-based heritability on the observed scale using GWAS summary statistics are shown in Table S2. <SEP>Since high positive genetic correlations between BF% and CVDs was only observed with HF and CAD, we further used 13 functional annotations to evaluate genetic correlations between BF% and HF or CAD in the partitioned genetic correlation analysis by specific functional category. The highest magnitude of significant genetic correlation was in the repressed region (Rg = 0.88; P = 1.51 × 10− 05) for BF% and HF, which could restore the phenotypic effects of a mutant gene. The correlation estimate for BF% and CAD was highest in the conserved region (Rg = 0.20; P = 5.00 × 10− 04), where this region remained almost unchanged during evolution (Fig. 2; Table S3). The shared genetic etiology for BF% and CVDs encourages the exploration of a common pathophysiology, especially in specific functional categories. Subsequently, we conducted a partitioned genetic correlation analysis by SNP groups with different P values and found that the first two groups remained significant (P < 1.00 × 10− 05), suggesting that our findings were robust (Figure S1-S2). <SEP>MTAG for single traits<SEP>Manhattan plots from the GWAS and MTAG analyses for each trait are shown in Fig. 3. From GWAS to MTAG, the total number of lead SNPs increased from 11 to 16 for BF% and from 42 to 51 for CAD, while no changes were found for HF. We confirmed most of the previously identified loci and found novel associations between BF% and HF and CAD. Out of the 16 independent loci reported for BF% using MTAG, 6 were novel associations and 2 of these fell within protein-coding gene bodies (PLA2G6, RPTOR) (Table S4). Among the 51 loci associated with CAD, 13 were novel loci, 10 of which were mapped to protein-coding genes (i.e., HNRNPUL1, NAA25, FES, TNS1, CYP46A1, ABCG8, IGF2BP1, BCAS3, SMG6, APOE) (Table S4). The proteins encoded by these gene and gene-related pathways are shown in Table S4. Out of the 10 independent loci identified for HF, 8 were novel, 3 of which were mapped to protein-coding genes (i.e., FTO, NPC1, IGF2BP1) (Table S4). <SEP>Cross-trait meta-analysis<SEP>Given the strong genetic correlation between BF% and HF or CAD, we further used MTAG to perform genome-wide meta-analysis to improve our power to identify shared significant genetic loci (meta-analysis P < 5 × 10− 08 and single trait P < 0.05). The genomic control parameter (λ) was 1.16 in the cross-trait meta-analysis for BF% and HF, while λ = 1.20 for BF% and CAD (Figure S3-S4). Illustrative calculations in the two-trait setting were 0.0012 for BF% and CAD and 0.0035 for BF% and HF. The Manhattan plot of these results indicated that shared genetic loci drove the overall significance of the meta-analysis (Fig. 4). <SEP>We identified 5 genome-wide significant independent loci for BF% and HF (Table 2). Out of the 5 independent loci, 3 were mapped to protein-coding genes (i.e., FTO, NPC1, LPA). The strongest signal was observed on chromosome 16 at the FTO region (index SNP rs9937053, Pmeta = 1.09 × 10− 18), the first GWAS-identified susceptibility gene for obesity. This locus was not only significant after meta-analysis, but also reached genome-wide significance in both single-trait GWAS of BF% (P = 9.86 × 10− 26) and HF (P = 2.99 × 10− 08). The second strongest signal was in close proximity to an intergenic region closest to the GNPDA2 gene on chromosome 4 (index SNP rs10938397, Pmeta = 4.36 × 10− 10), a critical gene involved in lipid and glucose metabolism. Previous studies found that the expression level of GNPDA2 alters the transcriptome profile of human adipose-derived mesenchymal stem cells. In addition, we found that genetic loci represented by rs10455872 (Pmeta = 3.88 × 10− 09) on LPA are associated with both BF% and HF after meta-analysis. Variation within LPA has shown strong association with Lp(a)-cholesterol levels, which is an independent risk factor for cardiovascular-related events. <SEP>The cross-trait meta-analysis between BF% and CAD identified 16 genome-wide significant loci (Table 3), 10 of which were in protein-coding gene bodies (i.e., LPA, HORMAD1, MIA3, WDR12, DGKH, CYP46A1, FES, FTO, IGF2BP1, BCAS3). The most significant locus is characterized by the LPA gene (index SNP rs10455872, Pmeta = 3.14 × 10− 28). The second locus (index SNP rs8050136, Pmeta = 4.02 × 10− 19) was mapped to the FTO gene. These loci were also found to be significant in the meta-analysis for BF% and HF, showing genetic overlaps between BF% and CVD-related traits. The third strongest signal was observed closest to the MC4R gene (index SNP rs663129, Pmeta = 3.10 × 10− 18), which is involved in the leptin signaling pathway and its disruption is a causal factor of obesity. <SEP>GTEx TSEA<SEP>In order to assess whether shared genes between BF% and CVDs are enriched for expression in the disease-relevant tissue, we conducted the TSEA using the GTEx pilot data. We found that shared genes of BF% and HF had three significantly enriched tissues, including blood vessel, brain, and fallopian tube (Fig. 5). The shared genes of BF% and CAD were enriched in six tissues including blood vessel, brain, fallopian tube, heart, nerve, and uterus (Fig. 5). The most strongly enriched tissue for both BF% and HF or CAD was part of the cardiovascular and nervous system (Fig. 5). <SEP>Over-representation enrichment analysis<SEP>The GO analysis indicated several significant shared biological processes between BF% and HF or CAD (false discovery rate < 0.05) such as glutathione derivative metabolic, glutathione derivative biosynthetic, and nitrobenzene metabolic processes (Table S5-S6). In additional analyses of Reactome pathways, we found that shared association signals for BF% and HF or CAD were significantly enriched in glutathione conjugation and plasma lipoprotein-related pathways (Table S7-S8). In general, the identified shared genes between BF and CVDs showed common significant enrichment in expression for glutathione metabolic-related pathways.<SEP>Fine-mapping and co-localization analysis<SEP>Lists of credible set SNPs in each shared locus for BF% and HF or CAD from fine mapping are shown in Table S9-S10. The co-localization analysis showed that 4 out of 5 loci and 16 out of 16 share causal variants between BF% and HF or CAD, respectively (Table S11-S12). The results of co-localization analysis were consistent with the cross-trait meta-analysis.<SEP>Single-trait TWAS<SEP>We conducted a TWAS analysis to examine if there are genes whose expression are related with BF%, CAD, and HF, and to determine if these genes are common among these traits. A total of 270 gene-tissue pairs were found across 44 GTEx tissues to be significantly associated with BF% after Benjamini-Hochberg correction (false discovery rate < 0.05), in addition to 786 gene-tissue pairs with CAD, and 270 gene-tissue pairs with HF (Fig. 6). Among these gene-tissue pairs, 32 overlapped between BF% and HF, most of which were observed in nervous and cardiovascular system (Table S17). Notably, C18orf8 and NPC1 are expressed in multiple tissues including brain, nerve, artery, and adipose tissue. Consistent with a previous study, NPC1 was validated as a shared genetic component in lipid metabolism and cardiovascular health from our TWAS results. In addition, the methylation levels of C18orf8 were considered as an epigenetic mechanism in maternal obesity and early life origins of CVD and cancers. Furthermore, we identified 28 gene-tissue pairs that were shared by BF% and CAD (Table S17). Most of the associations were found in exo-/endocrine and digestive system. Some genes are expressed among multiple tissues such as ATP5G1, SLC22A3, SNF8, and UBE2Z. ATP5G1 has been shown to encode a subunit of mitochondrial ATP synthase and SLC22A3 have been reported to be expressed at the blood-brain barrier and alters neuronal excitability. By integrating human and mouse results, previous studies have predicted that SNF8 and UBE2Z play a causal role in the development of CAD through a role in the vasculature. <SEP>Bidirectional MR<SEP>We conducted a bidirectional MR analysis to evaluate potential causal associations between BF% and HF and CAD. To assess the causal effect of BF% on HF and CAD, we used 10 previously identified genetic variants for BF% (Table S13). To assess the causal effect of HF and CAD on BF%, 5 and 24 previously identified genetic variants were used as instruments for HF and CAD, respectively (Table S14). We found that genetically predicted BF% was causally associated with an increased risk of HF (OR 1.63 per 1-SD increase in BF%, P = 4.16 × 10− 04) (Table 4) after Bonferroni correction. However, HF was not causally associated with BF%. In addition, there were no causal relationships between BF% and the risk of CAD. Sensitivity analysis showed that our results were reliable and were not affected by horizontal pleiotropy or outliers (Table 4, Table S15-S16). ",
                  "Discussion<SEP>Using large-scale GWAS data from international consortia, we analyzed shared genetic etiology and causal associations between BF% and CVD-related traits. First, we found strong positive genetic correlations and further identified novel shared genetic loci between BF% and HF or CAD. Second, we found that the shared genetic loci were enriched mainly in blood vessels and brain tissues and glutathione-related metabolic pathway using functional analysis. Third, we found that the shared genes between BF% and HF or CAD are mostly from nervous, cardiovascular and exo-/endocrine system using TWAS analysis. Finally, we found that BF% was causally associated with HF using bidirectional MR analysis results. Taken together, evaluation of the genetic correlation and causality between BF% and CVDs furthers the understanding of the shared loci and biological mechanisms underlying this comorbidity.<SEP>BF% is becoming recognized as a more predictive measurement in cardiovascular risk assessment than BMI or waist circumference, and increased BF% was often related to higher CVD risks with the presence of the metabolic abnormalities. Genetic studies have suggested that most of the BF% loci also affect cardiometabolic traits and comorbidities. Compared to GWAS results for single traits, MTAG discovered 6 novel loci for BF%, 8 novel loci for HF, and 13 novel loci for CAD. Importantly, we further conducted genome-wide cross-trait meta-analysis to improve our power to identify specific shared loci, including 3 novel loci between BF% and HF and 8 novel loci between BF% and CAD which did not reach genome-wide significance in previous GWAS. We highlight the potentially interesting functions of the novel associations for NPC1, PMAIP1, and GNPDA2 between BF% and HF, along with NEGR1, HORMAD1, GNPDA2, DGKH, CYP46A1, FES, POC1B, and BCAS3 between BF% and CAD. Most of these loci have significant single-tissue expression quantitative trait loci (eQTL), mainly in the cardiovascular, nervous, and exo-/endocrine system (Table S18-S19).<SEP>The only top loci common to the BF%-HF and BF%-CAD meta-analysis was rs10938397 near GNPDA2, which encodes an enzyme that catalyzes the deamination of the glucosamine-6-phosphate involved in the hexosamine signaling pathway. An animal study also suggested that GNPDA2 was involved in the regulation of body weight, fat and energy metabolism in the development of cardiovascular disease. Other top associations for BF%-HF were NPC1 gene variations, which might lead to metabolic diseases by modulating steroid hormone synthesis and lipid homeostasis. NPC1 is known to play a critical role in the atherosclerotic progression, and loss-of-function mutations in NPC1 can cause adiposity in humans. In particular, the NPC1 gene was also confirmed to be shared between BF% and HF using TWAS (Table S17). BCAS3 is a cytoskeletal protein that promotes directional cell migration and angiogenesis in vitro and is implicated in CAD. Additionally, other causal genes are highly expressed or known to act in human brain function (i.e., NEGR1, GNPDA2, CYP46A1, DGKH) and various carcinomas (i.e., PMAIP1, HORMAD1, FES, BCAS3), indicating the potential role of metabolic dysregulation in the pathogenesis of obesity and cardiac events.<SEP>We found that the shared loci for BF% with HF and CAD were highly expressed in blood vessel/heart tissues, indicating that these traits might be caused by dysfunction of the cardiovascular system. We found that the FTO gene on chromosome 16 was associated with both BF%/HF and BF%/CAD. FTO is an essential regulator in the development of obesity-induced metabolic and vascular changes, which is independent of its known function in regulation of obesity. Previous studies found that FTO plays a critical role in cardiac contractile function during homeostasis, remodeling, and regeneration for HF and CAD. Moreover, from the perspective of gene function, overlaps in genetic and molecular pathways advance our understanding of the potential role of brain/nerve tissue diseases in the association between BF% and CVDs. For example, the NEGR1 gene, which is strongly expressed in the brain, has been reported to affect neuronal control of food intake and promote obesity. People who have mutations in the NEGR1 gene may have abnormal fat deposition in various peripheral cells, suggesting a potential molecular target for regulating body fat. In the over-representation enrichment analysis, glutathione-related metabolic pathway was found to be common in shared genes of both BF%/HF and BF%/CAD meta-analyses. Experimental studies have reported that glutathione, which is the most abundant antioxidant in the heart, plays a key role in preventing the damage of redox homeostasis to cause obesity-related cardiovascular complications. More importantly, the protective effect of glutathione on blood vessel and brain tissue against oxidative stress has been previously reported and partially explains our findings. However, the nonsignificant genetic correlations between BF% and the other CVD-related traits suggested heterogeneous genetic architecture among CVD-related traits, which cannot be determined without further study.<SEP>In addition to the cardiovascular and nervous system, our TWAS also reported tissues enrichment from the exo-/endocrine and digestive system suggesting that the shared pathway between BF% and CVDs might have significant functions extending beyond brain and blood vessels. Besides a variety of adaptations/alterations in cardiac structure and function, body fat may affect the risk of HF with an altered metabolic profile in exo-/endocrine system. The incidence of CVDs are known to be closely related to endocrine factors such as hormone levels in vivo, which may be altered by body fat distribution and then affect the whole body organs such as ovary, thyroid, colon, and liver. For example, adolescent girls with obesity and polycystic ovary syndrome have increased fasting and postprandial plasma triglycerides and ApoB-lipoprotein remnants, and these indices are highly associated with early subclinical CVD risk. Indeed, an essential cause of CVDs is excessive body fat, which can be distributed in various organs of the body and cause pathological changes there. Thus, TWAS provided additional evidence of the enrichment of BF% and CVD genes expressed are not specific to a certain tissue; rather, it seems to be generalizable across metabolic organs of the whole body.<SEP>The bidirectional MR analysis showed a significant association between genetically predicted BF% and increased risk of HF (1.63-fold risk per 1-SD increase in BF%), which was stronger than that observed in a community-based cohort study (n = 100,280) at 1.32-fold per 1-SD increase in BF%. Our findings were also consistent with a large cohort study which was conducted in 5520 community-based, elderly individuals, showing that higher BF% might contribute to the development of HF through increasing ventricular-arterial stiffness. However, clinically, the association between BF% and HF could be bidirectional. In addition, previous studies suggested that BMI or body weight was positively or negatively correlated with HF or clinical outcomes. Although various associations were observed between BF% and CVDs, they were inconsistent possibly due to reverse causation and confounding. Therefore, this MR design is warranted to obtain causality which largely avoided bias such as reverse causation and confounding under MR key assumptions. Our MR findings provided reliable evidence of a causal role of BF% in HF based on large GWAS consortia regardless of unknown confounders existing in observational studies, highlighting the benefit of targeting BF% in the prevention of HF. The potential mechanisms underlying the causal association between BF% and HF require investigation, but the shared loci and related pathway could provide new directions in revealing shared etiologies.<SEP>In addition, we observed nominal positive correlation of BF% with intracerebral hemorrhage and negative correlation with HDL from genetic correlation analysis, but not with any other lipid profiles. Previous studies have suggested that low HDL may contribute directly to establishing or maintaining the obese condition due to the role of HDL in cellular lipid transport. HDL contribute to modulating body fat content by controlling the extent of lipolysis in mice model, which appears to be key components of lipid metabolism in adipose tissue and constitute new therapeutic targets in obesity. In cell culture studies, HDL specifically increased catecholamine-induced lipolysis possibly through modulating the adipocyte plasma membrane cholesterol content. In addition, the HERITAGE Family Study has shown that high HDL cholesterol levels are good correlates of the metabolic profile, representing a better index in CVD risk assessment.<SEP>This study had limitations. First, the study is limited by the quality of data collected including the BF% measurement, CAD diagnosis, medication administration, and so on. However, each previous study has conducted the study-specific quality control to ensure the data quality. In addition, considering the natural and random assortment of genetic variants during meiosis yielding the random distribution of genetic variants in populations, the result and conclusion of this genetic analysis are less likely to be influenced by confounding factors. Second, while BF% accurately reflects the proportion of fat content in the human body, it does not give full insight to distinguish between visceral and subcutaneous fat or fat content in each anatomical part. Further studies are required to explore the genetic association of body fat distribution with CVD risk. Third, we used limited number of SNPs as instrumental variables in the MR analysis, so we cannot exclude the possibility that our findings might have been affected by weak instrument bias, although all genetic instruments were associated with the exposure (F-statistic> 10). Fourth, this study did not assess sex-specific genetic effects using LDSC and MR analysis. Since we conducted the analysis using summary-level GWASs without individual data, it is difficult for us to verify the findings of this study for any variation with sex-specific analysis. Fifth, we could not re-test our hypothesis using additional or alternative large cohort without individual information in the present study. However, the data we used are the largest and latest GWASs for these traits, and the sample sizes of other cohorts are small from which we cannot yield more reliable results. Finally, the current study was limited to assessing shared genetic factors between BF% and CVD-related traits, which could only explain a small proportion of these traits. The effect of gene-environment interaction, in which the genetic variants lead to the occurrence of diseases in high-risk environments, may explain part of the rest variance. Although CAD and MI or stroke vest in similar pathophysiologic mechanisms, the complex roles of interactions between genes and environmental factors may be different. More large-scaled, well-conducted studies on gene-environment interaction in the development of CVDs are warranted.",
                  "34689",
                  "['OBWCCE']"
                ],
                [
                  "18",
                  "30089489",
                  "Background<SEP>Gestational Diabetes Mellitus (GDM) has been officially defined as “carbohydrate intolerance” of variable severity with onset or first recognition during pregnancy irrespective of treatment and whether or not the condition persists after pregnancy. GDM represents almost 90% of all pregnancies complicated by diabetes. The prevalence of GDM is rapidly increasing, ranging from 2 to 14% depending upon diagnostic criteria. In a study of South Indian women, GDM prevalence varied between 12 and 21% while another study of North Indian women reported a prevalence of 10% using WHO criteria. The hallmark of GDM is increased insulin resistance accompanied by decreased compensatory insulin secretory response. Type 2 diabetes (T2D) is also caused by increased insulin resistance and decreased insulin secretion to compensate for the former. Thus, both T2D and GDM share the same pathophysiology which is influenced by similar risk factors like high body mass index (BMI), history of abnormal glucose intolerance, family history of diabetes, age, and ethnicity.<SEP>A family history of both T2D and GDM is known to increase GDM risk, indicative of a common genetic component underlying both T2D and GDM. Till date, more than 120 T2D risk loci have been confirmed to be associated with T2D. A large proportion of them have also shown association with GDM. T2D risk variants at the MTNR1B, FTO, TLE1, G6PC2, GCKR, TCF7L2, ADCY5, CDKAL1, TCF2, HNF1B, PPARG, KCNJ11, SLC30A8 loci have previously been associated with GDM in European populations whereas variants in the CDKAL1, CDKN2A/2B, MTNR1B and KCNQ1 loci were associated with GDM in Korean women.<SEP>Some genetic variants are more unique to Indian T2D patients e.g. the SGCG (rs9552911) and TMEM163 (rs998451) variants. However, genetic studies of GDM in India are scarce. The SNPs rs7754840 and rs7756992 in the CDKAL1 gene were associated with GDM in South Indian women, while variants in the HMG20A (rs7178572) and HNF4A (rs4812829) genes were associated with both GDM and T2D. The aim of the present study was to investigate whether a panel of known variants previously associated with GDM and T2D in Indian and European populations are associated with GDM in Punjabi women.",
                  "Methods<SEP>Study population and phenotyping<SEP>Five thousand one hundred pregnant women were recruited by applying a multistage random screening in the State of Punjab in North India for GDM. Pregnant women at gestational week 24–28 were randomly selected and recruited. This was part of a WDF supported project titled “Gestational diabetes in Punjab” with the goal to create and implement sustainable awareness, education, screening, intervention and treatment capacities of diabetes in pregnancy (GDM) within the public and private health care system, as well as in the general population in Punjab. The team included a chief research coordinator, an assistant coordinator, doctors, nurses, lab technicians from all selected sites both in private hospitals and public healthcare system. Approval for screening was obtained from DRME, Chandigarh, India. The recruitment sites included Recruitment sites:, Deep Hospital, Model Town, Ludhiana as the epicenter, Shri Rama Charitable Hospital, Ludhiana, Chawla Hospital, Ludhiana, Iqbal Hospital, Ludhiana, Government Medical Colleges and Hospital, Patiala, Amritsar and Faridkot, PHC Verka, Amritsar, Health Centre Bhadsoan, Patiala, Health Centre Faridkot. The project was approved by Independent ethics committee, Ludhiana in 2009. The ethics committee is registered with Office of Drugs Controller General (India) Directorate General of Health Services with Registration no. ECR/525/Inst/PB/2014.<SEP>Information was obtained on age, BMI, family history of diabetes, diet, habitat (urban or rural), education and religion. All information material and written consent forms were provided in 3 languages (Hindi, Punjabi & English) and duly signed by the participants. The study protocol was approved by local Ethical Committees. Glucose was measured in venous plasma samples at fasting and at 2 h after a 75 g glucose challenge using glucometers (Accucheck-Roche Diagnostics). Fasting insulin concentrations were determined with ELISA (Diametra, Milan, Italy; intra- and inter-assay variation of < 5.0 and < 10.0%, respectively). The homeostatic model assessment (HOMA2) was used to quantify insulin resistance (HOMA2-IR) and beta-cell function (HOMA2-B) from fasting insulin and glucose values using the HOMA2 calculator v2.2.3 (http://https://www.dtu.ox.ac.uk/homacalculator/). GDM was diagnosed according to the WHO1999 (FPG ≥7.0 mmol/l and/or 2-h glucose ≥7.8 mmol/l) and the adapted WHO2013 (FPG ≥5.1 and/or 2-h glucose ≥8.5 mmol/l) criteria (ref). The clinical characteristics of subjects are shown in Table 1.<SEP>Genotyping<SEP>DNA was extracted from frozen and stored buffy coats using (QIAGEN Autopure LS kits. Six SNPs previously associated with GDM or T2D in India (Additional file 2: Table S1) and 79 SNPs previously associated with T2D in Europe and elsewhere from GWAS studies up to 2012 (some of these also with GDM risk from candidate gene studies in GDM populations) were genotyped in the present study (Additional file 2: Table S1) on a Sequenom Mass ARRAY Platform (Sequenom San Diego, CA, USA) PLEX using MALDI-TOF mass spectrometer or Taqman allelic discrimination assays using an ABI Prism 7900 sequence detection system (Applied Biosystems, Foster City, CA, USA). Genotyping was performed at the Lund University Diabetes Centre, Sweden after obtaining permission from ICMR (dated 21 october 2010 and Office of Drugs Controller General (India)(dated 14/12/2010).<SEP>Replication genotyping of 6% of the samples showed > 98% concordance. rs6467136, and rs7202877 had a Hardy-Weinberg equilibrium (HWE) p-value of < 0.001 in unaffected women based on WHO1999 criteria and < 0.05 in unaffected women based on WHO2013 criteria and were hence removed from the analysis.<SEP>Statistical analyses<SEP>Association of selected SNPs with risk of GDM was assessed by logistic regression analysis adjusted for maternal age and BMI and results presented as ORs with 95% confidence intervals (CI). We also tested for associations with fasting and 2-h glucose values as well as with fasting insulin and HOMA2-B and HOMA2-IR (Additional file 2: Table S1) using linear regression analysis with maternal age and BMI as covariates. Individuals with missing data were excluded. Data were logarithmically transformed before analysis. The power to detect association with GDM2013 including 1386 GDM women and 2632 controls at p < 0.0006 (0.05/79) (after Bonferroni correction) for a SNP allele frequency of 0.3 and effect size 1.3 was 0.97, which decreased to 0.64 for effect size 1.2 under an additive model. For GDM1999, with 346 GDM and 3672 controls, the corresponding figures were 0.39 and 0.12 respectively. For association with quantitative glucose traits, power to detect association was 1 at alpha 0.05 for and allele frequency of 0.3. A p-value of ≤0.05 was considered statistically significant on account of the current analyses being replication of previously published associations.<SEP>Genetic risk scores for insulin secretion (HOMA-2B) and insulin resistance (HOMA-2IR) were calculated using SNPs previously associated with insulin secretion and insulin resistance. SNPs were assessed for linkage disequilibrium (LD) and for those in high LD (r2), only one representative SNP was retained. Individual scores were calculated based on number of risk alleles weighed by their effect sizes reported in previous GWAS studies and logistic regression was performed against normalized measures of insulin secretion and insulin resistance.<SEP>All calculations were implemented in STATA, plink 1.09 and SPSS v22.0.",
                  "Results<SEP>Among the 4018 genotyped women, applying the WHO2013 criteria resulted in a total of 1386 women with GDM (34.5%) whereas the number was reduced to 346 (8.6%) when WHO1999 criteria were used. Notably, only 283 (7.0%) women were diagnosed using both GDM 2013 and GDM 1999 criteria (Additional file 1: Figure S1). This is concordant with our previously published reports on the larger subset of the same population comprising 5100 women. HOMA2-B was lower in GDM women defined by both criteria compared to pregnant normal glucose tolerant women (PNGT). HOMA2-IR was also higher in women with GDM2013 who thereby were more insulin resistant than PNGT (Table 1).<SEP>SNPs previously associated with GDM/T2D in India<SEP>None of the 8 SNPs previously associated with GDM or T2D in Indian populations was here associated with GDM (Table 2). However, analysis for association with GDM1999 or GDM 2013 against controls who did not satisfy either criterion revealed the nominal association of rs7756992 in CDKAL1 while rs689 in INS showed a trend towards association with GDM2013 (Table 3).<SEP>Previously reported GDM risk loci<SEP>Out of 12 selected previously reported GDM risk loci, the T allele of the missense SNP rs5219 in the KCNJ11 gene was nominally associated with GDM1999 (p = 0.019) (Table 4). Contrary to previous reports, the risk allele A of SNP rs11708067 in the ADCY5 gene showed reduced risk for GDM defined by 2013 (p = 0.037) (Table 4) but not by 1999 criteria. The SNP rs2796441 in the TLE1 gene was associated with decreased insulin secretion (p = 0.013) (Additional file 2: Table S2). The rs13266634 at SLC30A8 locus associated with GDM1999 while SNPs rs5219 in KCNJ11 and rs11708067 in ADCY5 associated with GDM2013 nominally when controls satisfying neither GDM diagnosis criteria were considered (Table 3).<SEP>Previously reported T2D loci<SEP>The risk allele C of SNP rs13389219 in the GRB14 gene was associated with GDM1999 (p = 0.022) (Table 5) but not with GDM2013 (p = 0.058) (Table 5). The T2D risk allele T of SNP rs11920090 in the intron of the SLC2A2 gene was associated with GDM2013 (p = 0.030) (Table 5).<SEP>Surprisingly, the T2D risk allele A of SNP rs11605924 in the CRY2 gene was associated with reduced risk of GDM1999 (p = 0.025) (Table 5). The same variant associated with GDM1999 in a sensitivity analysis when controls meeting neither GDM diagnosis criteria were considered (Table 3). In support of this, the same allele was also associated with lower 2-h glucose levels (p = 0.038) (Additional file 2: Table S3).<SEP>The risk allele A of SNP rs1552224 in the CENTD2 locus was associated with decreased risk of GDM2013 (p = 0.001) (Table 5).<SEP>Association with insulin secretion and insulin resistance<SEP>Twelve SNPs previously associated with insulin secretion were here tested for association with HOMA2-B. The T2D risk allele A of rs11071657 at the FAM148B locus was nominally associated with increased insulin secretion (p = 0.044) (Table 6). A GRS comprising of 3 previously reported insulin secretion loci with the lowest p-values for insulin secretion in the present study associated with insulin secretion in the present study (p = 0.008, beta = 0.25, SE = 0.098). GRS for insulin secretion did not associate with either GDM2013 (p = 0.15, beta = − 0.06, SE = 0.045) or GDM1999 (p = 0.73, beta = − 0.009, SE = 0.026).<SEP>Of 6 SNPs previously associated with measures of insulin resistance, 3 SNPs here associated with HOMA2-IR. The C allele of rs7607980 in the COBLL1 gene was associated with decreased HOMA2-IR (p = 0.0001). The C allele of rs13389219 near GRB14 (p = 0.026) and A allele of rs10423928 in the intron of the GIPR gene (p = 0.012) showed worse insulin resistance (increased HOMA2-IR; Table 7). Genetic risk scores (GRS) calculated based on the 3 SNPs associated with insulin resistance showed an increase of insulin resistance by 0.07 (SE = 0.145, p = 0.006) per allele. GRS for insulin resistance showed a trend towards GDM2013 (p = 0.065, beta = 0.076, SE = 0.04) but not GDM1999 (p = 0.14, beta = 0.023, SE = 0.025).",
                  "Discussion<SEP>In this large study, we investigated the genetic basis of gestational diabetes mellitus in Punjabi Indian women.<SEP>Surprisingly, the genetic variants in the HMG20A and HNF4A genes which previously have been associated with risk of T2D and GDM in South India were not associated with GDM or T2D in Punjabi pregnant women. This could be due to differences in allele frequencies between the North and South Indian populations, which are ethnically quite distinctive populations. The Punjabi Indian population belongs to the “Ancestral North Indians” group and shares genetic similarities with populations from Middle East, Central Asia and to some degree Europe whereas the South Indian population genetically belongs to the distinct “Ancestral South Indian” group. Notably the CDKAL1 variant associated with GDM only when a sensitivity analysis was performed using controls that had no GDM diagnosis using either GDM1999 or GDM2013 criteria, thus replicating a previous association.<SEP>Neither did we observe associations with loci associated with GDM elsewhere including variants in the CDKAL1 and MTNR1B loci, which have been reported to be associated with GDM in South Korea. A sensitivity analysis using controls that had no GDM diagnosis using either criterion revealed the nominal association of variants in SLC30A8, KCNJ11 and ADCY5. These largely negative findings could be attributed to population-based differences. Previous studies have indicated differences in anthropometry between Indian and European populations, with the former manifesting a “thin-fat” phenotype. Subsequently, it is possible that since most T2D loci were identified in European ancestry cohorts, the negative findings could reflect differences in tagging SNPs due to differences haplotypes between populations. On the other hand, the underlying etiology of GDM could also be different genetically. While the study population is the largest GDM study till date, this might lack sufficient power to detect genome-wide significance levels of association with an unstable phenotype. The effect sizes of previously reported T2D loci were low, generally under odds ratios of 1.2, therefore the study was not sufficiently powered to demonstrate association of SNPs with such low effect sizes. Alternately, considering the lack of consensus for GDM diagnosis criteria worldwide, it is plausible that this could be due to different thresholds that might apply for the Indian population.<SEP>Notably, T2D risk variants in the CRY2 (WHO1999), CENTD2 (WHO2013) and the ADCY5 (WHO2013) genes were here protective for GDM. CRY2 encodes for the cryptochrome protein involved in the regulation of the circadian clock. Risk allele carriers of the rs11708067 SNP in ADCY5 has previously been shown to reduce ADCY5 expression in pancreatic beta cells and important for coupling glucose to insulin secretion in human islets. It has been previously shown that T2D risk alleles show extreme directional differentiation across various populations, with T2D risk alleles decreasing in frequency along human migration into East Asia. Such flip-flops of risk alleles may be explained by population differences, possibly due to genetics or environment. Alternately, such “flip-flop” associations have also been attributed to multi-locus effects as shown from theoretical modeling studies demonstrating that the direction of allelic effect may flip when tested allele is inversely correlated with another risk allele at another locus, or positively correlated with a protective allele at another locus.<SEP>A HWE threshold of < 0.001 in unaffected individuals based in either criteria was set as a cut-off; SNPs showing significant deviations from HWE should be interpreted with caution, since these could be indicative of population substructures, inbreeding or selection. The current study only comprises genotyping data from candidate SNPs which do not provide sufficient coverage of the genome to detail population stratification or inbreeding. HWE could also be indicative of actual association. A serious problem in the study of the genetics of GDM is the implementation of different criteria, since some women could be classified as controls based on different criteria. For SNP rs5219 in KCNJ11 (HWE p = 0.004, WHO1999; HWE p = 0.01, WHO2013) and rs11605924 in CRY2 (HWE p = 0.007 WHO1999 and HWE p = 0.06, WHO2013), HWE values were nominally significant for the same criteria where an association was observed; these findings need to be replicated in independent cohorts.<SEP>Of 6 loci previously associated with insulin resistance, here 3 also showed an association with HOMA2-IR and a trend towards significance for GDM2013 but not GDM1999 including SNPs rs7607980 in the COBLL1 gene, rs13389219 near GRB14 and rs10423928 in the GIPR gene indicating that some of the genetic basis seem to be driven by previously reported insulin resistance loci. Similarly, a GRS with the 3 variants with the lowest p-values for insulin secretion associated with insulin secretion but not GDM2013 or GDM1999.<SEP>Taken together, the results demonstrate that GDM in women from Punjab in Northern India shows a genetic component, partially shared with GDM in other parts of the world, and seems to be driven by both insulin resistance and secretion. However, the direction of the effect can differ; some T2D risk variants were indicative of being protective for GDM in these Indian women.",
                  "17370",
                  "['OBWCCE', 'EATBTS', 'FOODIN']"
                ],
                [
                  "19",
                  "33849843",
                  "Introduction<SEP>Background<SEP>The global death toll of COVID-19 has eclipsed 1 million cases. Obesity, following age, has emerged as the most critical risk factor for morbidity, hospitalizations, and complications. The prevalence of obesity in the United States and in other Western countries has increased sharply in the last 2 decades. Since the early 1960s, when more than 10% of Americans were obese, that proportion has grown to 42.4% of adults. Moreover, the prevalence of obesity is higher in minority communities: 49.6% of non-Hispanic Black individuals and 44.8% of Hispanic Americans are obese, compared with 42.2% of non-Hispanic White individuals. These same minority communities are experiencing disproportionate COVID-19–driven mortality, likely linked, at least in part, to the heightened prevalence of obesity. Although the precise cause of obesity is yet to be discovered, several factors have been linked to its development. In particular, biology interacts with behavior and demographics (such as socioeconomic status or ethnic and cultural cuisine) to influence obesity risk. Obesity-associated biological factors include, but are far from limited to, genetics and epigenetics, microbiome composition, age, circadian rhythm disruption, pharmaceutical interactions, and comorbidities and their management.<SEP>The rapid increase in obesity prevalence has coincided with sociological factors, such as generally reduced physical activity alongside a rise in the consumption of highly processed, high-calorie, but nutrient-poor foodstuffs. However, these obesogenic conditions did not affect the population uniformly. Instead, a notable proportion of the population is still able to remain at a healthy weight, indicating that the heterogeneous response to obesogenic conditions may result, in part, from individual innate protection from these conditions, possibly conferred by the genetic makeup.<SEP>Most current clinical interventions for obesity management focus on lifestyle and dietary adaptation with varying levels of professional guidance and involvement, short- or long-term pharmaceutical therapies, and bariatric surgery. Individual responses to these therapeutic interventions are confoundingly (for clinicians and participants alike) heterogeneous for multifactorial reasons, making the need for personalized, precision medicine courses of treatment imperative. Most Americans (63%) have made serious efforts toward weight loss over the course of their lives, and almost one-third are trying to lose weight. In 2014, commercial weight loss services were a US $2.5 billion market consisting primarily of the following market shares—Weight Watchers (45%), NutriSystem (14%), and Jenny Craig (13%)—but the long-term effectiveness of various commercial calorie restriction–based weight loss programs is unclear (Table 1).<SEP>Personalizing Weight Loss Interventions<SEP>Recent research has elucidated the mechanisms of food-derived biomarkers, allowing for stratification based on a participant’s unique metabolism of given food products. This permits the targeting of personalized nutrition to groups that are better characterized. For example, given that low-grade inflammation has been implicated in insulin resistance, mediating inflammation via targeted dietary approaches is a precision nutrition intervention.<SEP>Advances have already been made in the early intervention and risk assessment of participants who are obese by designing therapies based on unique genetic predisposition and risk. Environmental interventions such as diet and exercise can trigger epigenetic changes, altering gene expression in metabolic pathways. Recent research indicates that physical activity and high-fat diets may alter DNA methylation patterns in skeletal muscle and adipose tissue, influencing weight management. Eventually, researchers hope to elucidate the genetic patterns that influence individual obesity and concomitant illness susceptibility, risk of progression, and response to therapy, to provide participants with optimal treatment.<SEP>Epigenetics and Their Role in Obesity<SEP>Even as science illuminates many genetic risk factors of complex metabolic diseases such as obesity and type 2 diabetes, these genetic variants account for only a fraction of BMI variation. The missing heritability of obesity might be at least partially explained by interactions between genetics and environmental factors. In particular, specific gene variants may influence sensitivity to certain environmental factors so that exposure to these factors in susceptible individuals can contribute to disease. As individuals who are obese are characterized by considerable heterogeneity within the spectrum of clinical obesity, supporting gene-diet interaction and precision nutrition in different subtypes of obesity is imperative.<SEP>Bariatric surgery is a weight loss option for participants with severe and complex obesity, for whom dietary interventions or digital therapeutics have been less than successful. Genetics may be a significant predictor of weight loss following Roux-en-Y gastric bypass surgery, but few genetic variants have been characterized to date.<SEP>The Role of Diet in Obesity<SEP>Although obesity can, in some cases, be linked to excessive appetite and food consumption, these behaviors may have a genetic component, and even food preferences themselves may have a genetic basis. For example, the alpha-ketoglutarate-dependent dioxygenase (FTO) locus rs9939609 has been associated with reduced satiety, increased caloric and fat intake, and a propensity to consume calorie-dense foods. The TAS2R38 genotype differentiates potential super-, medium-, and nontasters of bitter-tasting thiourea compounds. These different bitter-tasting profiles appear to be predictive of differential dietary preferences, and in particular, nontasters were observed to have higher BMIs. Considered together and alongside other evidence, this research implies that body weight and BMI may be affected by genetic variations in food preferences, tendencies, and eating behaviors. Elucidating how food intake and body metrics are mediated by genetics is challenging because of the difficulty of reproducing results across varying populations and the complexity of identifying causal interactions. Research using randomized controlled trials (RCTs) and large sample–sized biobanks with electronic health records will better characterize how diet and genetics interact to mediate health outcomes.<SEP>The Role of Physical Activity in Obesity<SEP>Exercise that can prevent weight gain and promote weight maintenance has been well established through research. Evidence suggests that body weight, waist-to-hip ratio, and BMI are significantly associated with adherence to an aerobic exercise intervention. Interestingly, the propensity for exercise appears to be heritable, at least in part, with studies estimating heritability ranging from 9% to almost 80%. MC4R genes appear to be associated with physical inactivity; however, other genes may be associated with adherence and tolerance to physical activity regimens.<SEP>Gut Microbiome and Its Role in Obesity<SEP>The human gastrointestinal tract hosts millions of commensal micro-organisms comprising the gut microbiome, which acts as a virtual endocrine organ regulating nutrient production and metabolism, satiety, and even energy homeostasis. These microbes are intrinsically linked to host health, as they are implicated in nutrient processing and metabolism, pathogen displacement, vitamin synthesis, and body weight regulation. Researchers and clinicians have been studying alterations of the gut microbiome in individuals, as perturbations in the gut microbiome appear to underlie the pathophysiology of obesity and associated comorbidities, such as type 2 diabetes and metabolic syndrome. Microbiome profiling for nutritional intervention is gaining prominence as a key feature of precision nutrition.<SEP>Research on the impact of specific dietary factors on microbiome diversity can guide interventions focused on optimizing gut microbial composition. For example, variation in the lactase (LCT) gene region, associated with response to dairy intake, appears to be associated with the abundance of the gut microbiome Bifidobacterium. In particular, variations in LCT were found to be predictive of the obesity-based modulation of dairy lactose and milk intake, indicating that shifts in gut microbiota across LCT genotypes could be tied to the caloric extraction of ingested food. Similar to specific genes, specific bacterial species have also been directly implicated in the etiology of obesity. Methanobrevibacter smithii, for example, can itself metabolize dietary substrates or metabolic byproducts of other bacteria, thereby promoting weight gain.<SEP>Further evidence ties both an individual’s genetics and diet to microbiome composition because lower microbial diversity appears to be associated with excess weight gain. Even in early childhood, disruptions in the gut microbiome can have a long-lasting influence on adult body weight. Moreover, nutritional interventions such as administering prebiotics and probiotics to manipulate gut microbiota that promote or are refractory to weight loss show potential as obesity interventions but require further study. Weight loss, whether mediated by diet or via bariatric surgery, can alter the gut microbiome in ways that affect the efficacy of various weight loss strategies. An interesting feature of bariatric surgery is that it appears to induce obesity-associated gut microbiota to shift toward lean microbiome phenotypes.<SEP>Behavioral and Digital Interventions in Obesity<SEP>As the obesity epidemic continues to proliferate, new digital programs available on websites or as smartphone apps are being leveraged to promote weight loss. Digital programs are agile in that they can easily be modified to reflect the latest research and best practices in a rapidly changing field; they are more cost-effective than traditional in-person programs and are also more easily scalable, increasing their reach. Resources can include activity trackers, videos, logs, device-to-device communication, and third-party app compatibility. In addition, research indicates that remotely administered programs can result in significant weight loss.<SEP>Digital programs have the ability to provide personalization to address the plethora of needs presented by participants. Individuals partaking in such programs are still able to leverage interpersonal relationships. Digital health coaching, for example, allows participants to discuss their weight loss journey via any number of communication platforms. According to research, both in-person and telehealth coaching relationships are effective in motivating overweight individuals to work toward weight loss. In a recent study of more than 600 participants in a smartphone-based weight loss program with a coaching component, participants lost, on average, more than 7% of their body weight, successfully passing the 5% weight loss marker that many in-person programs set.<SEP>The multifactorial nature of obesity is reflected in the myriad heritable, behavioral, and environmental factors that can lead to obesity risk. The most successful interventions are likely to be those that leverage current findings across the full spectrum of obesity-related risk factors: dietary interventions accounting for both genetic markers of food sensitivity, metabolic predispositions, and behavioral risk as well as those geared toward optimizing gut microbial diversity and composition; physical activity measures taken in consideration of genetic risk profiles; and behavioral modifications undertaken via digital care. The precision nutrition therapy offered by Digbi Health aims to account for these various factors in delivering a personalized course of obesity intervention. In this study, we describe the weight loss of a cohort of participants enrolled in the Digbi Health program, modeling and analyzing genomic, lifestyle and engagement factors that were found to be influential in this cohort. It was hypothesized that weight loss models would exhibit a better fit when incorporating genomic data than when using demographic and engagement variables alone.",
                  "Methods<SEP>Recruitment<SEP>For this study, we identified all Digbi Health participants who were enrolled between June 2019 and June 2020, had been in the program for at least 120 days, and had been genotyped by Digbi Health. Sample collection kits were shipped to 443 participants, of which 393 mailed back their samples for processing, thereby yielding a cohort size of 393. Among these participants, 315 individuals self-identified as female, 77 individuals self-identified as male, and one individual declined to state. All participants self-enrolled for the Digbi Health therapy via a large California-based insurance payor wellness program. The qualifying criteria to join the program were BMI >25 kg/m2 with a comorbidity (eg, prediabetes, diabetes, cardiovascular disease, or hypertension) or BMI >30 kg/m2, regardless of comorbidities. Participants were advised to remain under the care and supervision of their existing physicians and were further advised to notify physicians and other health care providers of their participation in the Digbi Health program. The data set included data from each participant’s first 120 days in the program. This Digbi Health anonymized, retrospective research study was exempted from full review by the Ethical and Independent Review Services West Coast Board, Corte Madera, California, reference 20149-01. All participants agreed to the Digbi Health terms and conditions and privacy policy when enrolling in the therapy.<SEP>Intervention<SEP>Digbi Health is a next-generation, prescription-grade, digital therapeutic platform that uses artificial intelligence to analyze genetics, gut bacteria, lifestyle habits, and socioeconomic and behavioral risk patterns to create evidence-based personalized nutrition, fitness, sleep, and stress management programs to reduce weight and reverse weight-related inflammatory gut, musculoskeletal, cardiovascular, and insulin-related diseases. Digital precision care interventions are delivered via web-based or mobile apps to expand the accessibility, safety, and effectiveness of health care. Digbi Health’s individualized program is geared primarily toward individuals who are overweight or obese, with or without a comorbidity, and functions as a weight loss management tool. The therapy is currently covered by a large California-based health insurance payor for their qualifying members through its obesity management wellness platform.<SEP>On enrolling in the Digbi Health program, participants were provided with web-based log-in credentials and were mailed a Bluetooth-compatible digital weighing scale and saliva and stool biosampling kits. App usage consisted of daily tracking of weight (via the Bluetooth scale), tracking of dietary intake (uploading photographs of all food items consumed), and tracking wellness-associated metrics (sleep quality and quantity, exercise type and duration, stress and meditation, energy levels, cravings, and recommended foods consumed or avoided).<SEP>Sample Collection<SEP>The individual’s DNA was self-collected using a buccal swab (Mawi Technologies iSwab DNA collection kit, model no. ISWAB-DNA-1200). Saliva DNA extraction, purification, and genotyping using Affymetrix Direct to Consumer Array version 2.0 on the Affymetrix GeneTitan was all performed by the AKESOgen laboratory. The results presented in the genetics section of the report were determined by the number of markers and risk genotypes present in the genomic raw data, the Digbi Health reports were loaded into the app, and coaching was individualized based on participants’ genomic risk factors. Individuals’ gut microbiomes were self-collected via a fecal swab (Mawi Technologies iSWAB Microbiome collection kit, model no. ISWAB-MBF-1200). Sample processing and 16SrRNA-targeted next-generation sequencing were performed at the AKESOgen laboratory. Although the app and coaching are personalized based on participants’ microbiome data, these data were not analyzed in this study and are the subject of a forthcoming research article. The personalized Digbi Health plan was systematically reviewed with the participants in individualized sessions with the health coach over a 4-month period at regular, predetermined, weekly, and biweekly intervals.<SEP>Genetic Report<SEP>The Digbi Health genetic report consisted of two sections: gene nutrition and gene fitness. The gene nutrition report analyzed participants’ genotypes that have been shown to influence nutritional traits, such as diet and weight management, micronutrient requirements, food intolerance and sensitivity, and several other attributes relevant to nutritional well-being. For each of these traits, participants were assigned a high-, medium-, or low-risk score based on the number of risk alleles detected, and health coaches guided interventions based on these potential risks (eg, suggesting someone with high risk for gluten intolerance eliminates dietary gluten or someone with medium risk reduces consumption). The degree of risk associated with any specific single-nucleotide polymorphism (SNP) was determined by the presence of 0, 1, or 2 risk alleles. Several individual SNPs may have contributed to a single trait or function, and some of these SNPs might have increased the risk for a trait, whereas others may have decreased it. In Digbi Health gene reports, as many SNPs as possible were considered when determining the risk of a particular trait. Although Digbi Health coaching is individualized based on several different traits, a number of notable traits and associated SNPs and how risk factors for these traits inform individualized health coaching have been highlighted in the Results and Discussion sections of this paper.<SEP>The gene fitness report analyzed SNPs studied in conjunction with fitness regimes, exercise motivation, and the ability to develop various types of muscle fibers. This section of the report also analyzed the potential inflammatory response to exercise, including endurance, strength, and flexibility training. As in the gene nutrition section, each trait was assigned a high-, medium-, or low-risk score based on SNP data, and health coaches guided participants through recommendations for healthy exercise.<SEP>Gut Microbiome Report<SEP>In addition to using genetic risk profiles to guide the course of participants’ precision care, the Digbi Health program also analyzed gut microbiome profiles (collected from stool swab sampling) to guide the course of care. However, in this study, we aimed to analyze only the effect of demographics and lifestyle and genomic factors on weight loss; the incorporation of the related microbiome data and how they inform individualized health coaching is the subject of forthcoming analyses from our group, currently in preparation.<SEP>Lifestyle<SEP>The Digbi Health therapy is a 120-day program that uses body metrics, gut microbiome and genetic profiles, and personalized health coaching to manage weight loss. Participants use the Digbi Health app to track 10 key lifestyle and wellness markers (weight, sleep, hunger, cravings, stress, meditation, superfoods, morning energy, foods to avoid, and exercise) on a daily basis and take photos of the food they consume. Each participant is assigned a health coach who works personally with the participant through 12 guided sessions at various intervals to interpret the personalized wellness reports generated from sampling participants’ DNA and gut microbiota. The reports also provide a breakdown of obesity risk based on individuals’ genetic and gut microbiome profiles. The program is geared toward participants losing at least 5% of their baseline body weight by day 90 of the 120-day program. To achieve this goal, the program seeks to nudge participants toward making incremental lifestyle changes focused on reducing sugar consumption, timing meals to optimize insulin sensitivity, reducing systemic inflammation by identifying possible inflammatory and anti-inflammatory nutrients via genetic testing, and establishing a base level of physical activity. The personalized incremental behavioral modifications suggested by the program are designed to reduce inflammation, optimize gut health based on microbiome testing, and most importantly are supported by health coaching and the app to integrate into the participant's lifestyle so as to be sustainable long term. The genetic profile of Digbi Health users identifies several nutrient and food risk factors that have associations with obesity, comorbidity, or inflammatory risk (eg, gluten sensitivity, lactose tolerance, caffeine sensitivity, fatty acid metabolism, blood pressure response to salt or riboflavin intake, or reduced insulin resistance with exercise), and health coaching guidance is tailored specifically to incorporate participants’ risk profiles.<SEP>Statistical Analysis<SEP>The data from our cohort of 393 participants over their first 120 days in the Digbi Health personalized digital weight loss program were analyzed retrospectively. Interpretable regression models (linear and logistic) were built, and visualizations generated using R software (R Core Team). Modeling of demographic and behavioral engagement was conducted by fitting 2 linear regression models of weight loss (pounds lost and percentage lost) in this cohort as a function of the variables listed in Table 2.<SEP>Genomic-enhanced models were built by incorporating 197 SNPs from participant genomic data as predictors, using Lasso regression on SNPs for variable selection, and then fitting a model to the data set after adding the selected SNPs to the previous engagement variables. The 197 genomic variables were from Digbi-curated panels of SNPs associated with obesity, fitness, nutrient metabolism, and inflammatory markers (Table S1 in Multimedia Appendix 1). Each SNP value was encoded for each participant as their number of risk alleles (0, 1, or 2). One participant did not identify gender, therefore was excluded from all models, resulting in 392 observations included in each of the 4 linear regression models.<SEP>Success or failure logistic regression models were also fit, with and without genomic data. Genomic variables were similarly selected from the full panel of 197 SNPs using Lasso logistic regression. Success was defined as ≥5% weight loss, failure as weight gain or negligible change of <2 lb (0.9 kg). Removed from this model were observations of participants who were only partially successful, having lost weight but without reaching the milestone of 5% weight loss. This resulted in the inclusion of 251 cohort participants in the logistic models, both genomic-enhanced and demographic and engagement only.<SEP>Insignificant variables were removed from each model, resulting in 6 final interpretable models, half containing demographic and behavioral engagement variables only, whereas the remaining 3 were genomic enhanced.<SEP>Demographic variables included gender, age, and baseline BMI. Behavioral engagement variables included the number of coaching sessions completed, number of weight entries, and number of food posts (Table 2). As the number of food posts and number of weight entries were highly correlated (Pearson correlation 0.98), each regression model could include one but not both. To incorporate both variables in modeling, the number of food posts was retained as a predictor for the linear models, whereas the number of weight entries was kept as a predictor for the logistic models.<SEP>For genomic-enhanced models, SNP variables were imputed to the most frequent value (mode). SNPs with >10% missing information, high (≥0.8) Pearson correlation with another variable, or zero variance were removed, resulting in 124 SNPs remaining for linear and 122 SNPs remaining for logistic model variable selection by Lasso regression. The SNPs with nonzero coefficients after Lasso regularization for that particular outcome variable (pounds lost, percentage weight loss, and successful weight loss) then served as predictors, along with the three demographic variables and two engagement variables (number of coaching sessions completed along with either number of weight entries or number of food posts).",
                  "Results<SEP>Weight Loss<SEP>A total of 393 participants were included in this study to describe and model the weight loss of participants enrolled in the Digbi Health program for 120 days. Of these, 80.4% (315/392) were female and 19,6% (77/392) were male, and one participant declined to state. Tables S1 and S2 in Multimedia Appendix 1 provide a full distribution of the baseline variables. A total of 72% (283/393) participants lost weight compared with 10.7% (42/393) who gained weight, whereas for 17.3% (68/393) participants, the weight remained within normal fluctuations (Table S3, Multimedia Appendix 1). A total of 142 participants lost ≥5% of their baseline body weight within the first 120 days. Improvement in both BMI measures and BMI class over 120 days of treatment is evident in Figure S1, with 25.0% of participants having lost enough weight to move to a lower BMI class (Table S4 in Multimedia Appendix 1). BMI class was defined as presented in Table S5 of Multimedia Appendix 1. The distribution of engagement variables, overall and by gender, is presented in Table 2. In our cohort, no significant difference was found in these variables between males and females (Welch two-sided two-sample t tests; results not shown). End points by the obesity class are presented in Table S4 of Multimedia Appendix 1.<SEP>As hypothesized, the addition of genomic predictors substantially improved the fit of weight loss models. For linear regression weight loss models (pounds lost and percent), the addition of genomic data improved the mean squared error from 70 to 60 and 16 to 13, respectively, whereas the logistic success or fail model improved pseudo R2 from 0.193 to 0.285.<SEP>Figure 1 depicts the distribution by gender of percent weight loss. The difference in percent weight loss for males and females was found to be statistically significant (Welch two-sided two-sample t test, P=.02). At an average of 4.8 (SD 4.2) percent of body weight lost, the difference in weight loss between males and females was 1.3 (SD 0.5) percent of body weight. Gender was significant to all linear regression models (Tables S6-S9 in Multimedia Appendix 1) but not to the logistic success or fail models (Tables S10 and S11 in Multimedia Appendix 1), as both women and men succeeded in 5% weight loss within 120 days.<SEP>Significant Variables<SEP>Unsurprisingly, baseline BMI was significant to both pounds lost linear models (Tables S6 and S7 in Multimedia Appendix 1) but not for any other model. The participant’s age was not significant in any of the models. Increased completion of coaching sessions was significantly associated with increased weight loss in all regression models (Tables S6 to S11 in Multimedia Appendix 1). The two highly correlated engagement variables, number of weight entries and number of food posts, were significant to all models in which they were considered (as described earlier, weight entries were in logistic models, whereas food posts were in linear models; Tables S6 to S11 in Multimedia Appendix 1).<SEP>Significant SNPs<SEP>In addition to the demographic and engagement variables described earlier, the genomic-enhanced models identified 10 SNPs that were significant to the linear pounds lost model (Table S7 in Multimedia Appendix 1), 11 SNPs to the linear weight loss percentage model (Table S9 in Multimedia Appendix 1), and 6 SNPs to the logistic model (Table S11 in Multimedia Appendix 1). Of the SNPs found significant to the linear models, 8 SNPs were common in both genomic-enhanced linear models (Tables S7 and S9 in Multimedia Appendix 1). In total, 3 notable SNPs that were found to be strongly associated with changes in body weight for this cohort, rs17300539_G, rs2016520_C, and rs4074995_A, were further explored. The literature suggests explanatory metabolomic factors and findings from recent studies that provide context and explanation for these associations in our descriptive study.<SEP>Rs17300539 is located in the promoter region of the ADIPOZ gene, which encodes adiponectin. The high-risk allele has been associated with insulin resistance, whereas the low-risk allele may be associated with protection from weight regain postweight loss intervention. Moreover, the high-risk allele has been associated with higher weight, BMI, and waist and hip circumferences. However, genotype-related differences in BMI became undetectable in the interaction with a diet that is low, below the median (ie, <13% of energy intake) in monounsaturated fats (MUFAs). This led researchers to propose the possibility of moderating high risk with dietary interventions to reduce MUFAs for those with the risk alleles. Of the 392 participants, 334 were homozygous for the high-risk allele (G), 54 were heterozygous for the risk allele, and 4 were homozygous for the low-risk allele (Table 3).<SEP>The regression models are interpretable models describing weight loss in this cohort and may be visualized to gain insight into variables found to be significant. Figure S3 in Multimedia Appendix 1 and Figure 2 depict the relationships of engagement variables and rs17300539 to weight loss in the genomic-enhanced weight loss percent model. These plots reveal the least squares fit of weight loss percent for females (panel A) and males (panel B), as the two visualized predictors are varied while holding all other model variables constant (SNPs were held constant at their most frequent [mode] values, whereas engagement variables were held constant at their gender-specific means, except for coaching sessions completed, which was fixed at its gender-specific median). The visualizations permit us to see the model relationships of particular predictors, as they impact the outcome variable. For example, the weight loss (%) model fit to this cohort describes the average male having 2 risk alleles who posts no food photos as losing 3.5% of body weight, whereas the average male with the same genomic risk who posts 975 food photos loses 8.75% of body weight (Figure 2). The coefficients of the fitted predictors reveal that in this model, for every 100 additional food posts, participants lose, on average, an additional 0.60% weight while holding all other model predictors constant (Table S9 in Multimedia Appendix 1).<SEP>In this model, for each additional risk allele (G) of rs17300539, participants lose, on average, an additional 1.09% weight while holding all other model predictors constant (Table S9 in Multimedia Appendix 1). Similarly, as the number of risk alleles of rs17300539 increases from 0 to 1 to 2, so does percentage weight loss as a function of greater behavioral engagement measured both in the number of completed coaching sessions (Figure S3 in Multimedia Appendix 1) and the number of food photos posted (Figure 2). In essence, participants in this cohort who were at higher risk lost a greater percentage of weight compared with their lower risk counterparts. Moreover, the percentage of weight loss increased in proportion to greater behavioral engagement.<SEP>Rs2016520 is a variant of the PPARD gene, which encodes a protein implicated in fat metabolism and baseline cholesterol levels. In women, this SNP has been shown to be associated with muscle development and blood cholesterol reduction after a 12-week exercise regime. High-risk alleles predisposed women to less weight loss on exercise. Of the 392 participants, 20 were homozygous for the high-risk allele (C) of SNP rs2016520, 127 were heterozygous for the risk allele, 244 were homozygous for the low-risk allele, and 1 had no available data (Table 3).<SEP>Similar to Figure 2 and Figure S3 in Multimedia Appendix 1, Figure 3 and Figure 4 and Figure S4 in Multimedia Appendix 1 reveal the least squares fit of weight loss in pounds for females (panel A) and males (panel B) as the two visualized predictors are varied while holding all other model variables constant. The weight loss pounds model fit to this cohort describes the average female having 2 risk alleles who posts 975 food photos as losing 21 lb (9.5 kg), but only 8 lb (3.6 kg) if no food photos are posted (Figure S4A in Multimedia Appendix 1). Similarly, this model describes the average male having 2 risk alleles as losing 25 lb (11.3 kg) with 975 food posts but only 13 lb (5.9 kg) with no food posts (Figure S4B in Multimedia Appendix 1). As visualized in Figures 3 and 4 and Figure S4 in Multimedia Appendix 1 the effect in this descriptive model of an increase in the number of risk alleles from 0 to 2 is that pounds of weight loss with respect to engagement increases when engagement is measured either as the number of coaching sessions or as the number of food photos posted in the Digbi Health app. Moreover, as seen in Figure 4, a higher baseline BMI was associated with more pounds lost. Males lost more weight than females in each risk group of this SNP.<SEP>The rs4074995 SNP has been implicated in calcium-potassium regulation; it is located within the RGS14 gene and is associated with both serum phosphate and serum calcium levels. In particular, each copy of the A allele is correlated with an increase in serum calcium concentration. For the rs4074995_A SNP, of the 251 participants, 25 were homozygous for the high-risk allele (A), 67 were heterozygous for the risk allele, and 159 were homozygous for the low-risk allele (Table 3). The sample size of 251 was smaller than for the abovementioned linear models because rs4074995 was chosen to be highlighted as a predictor of the genomic-enhanced logistic regression (success vs failure) model, which was fit to a subset of the cohort that experienced success, defined as ≥5% weight loss, or failure, defined as weight gain or negligible change of <2 lb (0.9 kg).<SEP>As seen in Figure 5 and Figure S5 in Multimedia Appendix 1, as the risk alleles of SNP rs4074995 increase from 0 to 1 to 2, there is an increase in the success score, which is the likelihood of this model assigning a particular observation to the success class. Similar to the other models, we found that an increased number of coaching sessions completed is associated with a sharp increase in the success score. However, on highest engagement, the effect of risk status diminishes (those with 0, 1, or 2 risk alleles were approximately equally likely to achieve weight loss on highest engagement).",
                  "Discussion<SEP>Principal Findings<SEP>A group of 393 participants underwent lifestyle changes over 120 days through the Digbi Health program—a precision digital care program applying machine learning analytics to genetic and microbiome profiles, demographics, and self-reported lifestyle habits—delivering care through the app and weekly health coaching check-ins. Over the duration of the program, patients’ genomic and gut microbiome data pertinent to weight loss (from Digbi Health–curated panels) were provided and translated into lifestyle recommendations and recipes. Of the participants, 72% (283/393) lost weight, whereas 10.7% (42/393) gained ≥2 lb. Of those who lost weight, 50.2% (142/283) were able to lose 5% or more over 120 days.<SEP>Interpretable linear regression models of weight loss in this cohort (pounds lost and percentage lost) as a function of demographic and behavioral engagement variables were fit to describe the weight loss of this cohort. Genomic-enhanced models were also built by adding participant genomic data as predictors. Interpretable success or failure logistic regression models were also fit, with and without genomic data. The addition of genomic predictors substantially improved the fit of all models.<SEP>The fitted models were examined to gain insights into the weight loss journey of this cohort. Gender, engagement, and specific SNP risk alleles were significantly associated with successful weight loss. The models described greater average weight loss in our cohort for participants having more of certain risk alleles. Here, we consider how successful weight loss may be obtained in the face of greater genetic risk factors. Notably, Digbi Health precision coaching for lifestyle modification is personalized to these genetic risks, and patients reported realizing success that was previously unattainable after being empowered by the knowledge of their genetic and microbiome risk factors, accompanied by advice on lifestyle modifications to address these risks.<SEP>We profiled three of these genetic markers (see Results section) to elucidate their relationships with explanatory metabolomic processes and weight gain and loss. Here, we connected these relationships with personalized recommendations delivered by the Digbi Health app and coaching staff. The profiled SNPs were associated with circulating adiponectin and response to dietary MUFA consumption, fat metabolism, and baseline cholesterol levels, and serum calcium levels and calcium-potassium metabolism were strongly associated with weight loss success.<SEP>As an example of personalized dietary advice delivered by both the app and coach for program participants who are at genetic risk of weight gain, we considered the advice delivered to participants with a different genetic outlook with regard to rs17300539, a risk allele for weight gain with high MUFA intake. This SNP is depicted in the visualizations of our linear model for weight loss percentage (Figure 2 and Figure S3 in Multimedia Appendix 1). As reported earlier, participants in this cohort who were at higher risk lost a greater percentage of weight compared with their lower risk counterparts, and percentages of weight loss correlated with greater behavioral engagement. This finding can be explained by the fact that those with high risk for this trait were advised by both the app and human coaching to avoid MUFA consumption as much as possible (contrary to the conventional wisdom that these—olive oils, almond oils, etc—are comparatively healthy fats). Instead, they were advised to shift to the consumption of polyunsaturated or saturated fats, depending on their genotypes. Moreover, this SNP is associated with insulin resistance, and parts of the Digbi Health Nutrition Plan (eg, intermittent fasting and reducing processed carbohydrate consumption) would be expected to reduce insulin resistance, addressing the risk associated with this SNP, thereby helping with weight loss.<SEP>The linear regression pounds lost models found an association between higher baseline BMI and increased weight loss in this cohort (Figure 4). For each one-unit increase in baseline BMI, participants lost an additional 0.2 lb (0.09 kg) on average while holding the other variables in the model constant. This finding could be encouraging to new participants with higher BMI, who may have attempted weight loss with other programs but without much success. Adding genomic information (Figure 3), the model describes the average man of this cohort at the highest baseline BMI, having 2 risk alleles for rs2016520_C and completing five coaching sessions as losing 21 lb (9.5 kg), but women having the same risk outlook and behavioral engagement as losing, on average, 16.5 lb (7.5 kg). When compared with participants of the same gender, baseline BMI, number of coaching sessions, and genomic outlook for all SNPs except rs2016520_C, participants in this data set lost 2.4 lb (1.1 kg), on average, over their treatment for each additional risk allele they had of rs2016520_C (Table S7 in Multimedia Appendix 1).<SEP>This SNP is poorly characterized in the general population, but studies associate it with BMI and waist circumference among Han Chinese as well as with cholesterol metabolism. This latter association drives the recommendation by Digbi Health that participants presenting with the high-risk allele limit cholesterol consumption. The association between the number of risk alleles of rs2016520_C and increased weight loss in this data set may indicate the efficacy of data-driven coaching by Digbi Health. A visualization in Figure 3 of this descriptive model depicts a male of average baseline BMI and the most frequent genomic outlook for all except rs2016520_C as losing 20 lb (9.0 kg), on average, if he completed 12 coaching sessions, but only 11 lb (5.0 kg) on average with only one coaching session over the course of treatment.<SEP>Calcium is an essential mineral critical for vascular function, muscle function, neurotransmission, cell signaling, and hormone secretion. Serum calcium levels tend not to respond directly to dietary calcium intake, and instead, the body relies on reservoirs in bone tissue to maintain consistent calcium concentrations. Recent research has emerged tying higher serum calcium levels to the development of insulin resistance and cardiovascular hypertension. High serum calcium levels have long been correlated with obesity. Figure 5 depicts the success or fail logistic regression model of the associations between the number of coaching sessions completed and rs4074995_A with successful weight loss while holding all other variables in the model constant at their most frequent number of risk alleles. As with the linear model, in this Digbi Health treatment cohort, increasing total coaching sessions was associated with higher success in losing weight. Those at high risk for excess serum calcium levels were especially encouraged to embrace intermittent fasting and carbohydrate avoidance to combat insulin resistance. This may explain their higher success in achieving ≥5% weight loss (Figure S5 in Multimedia Appendix 1). We found that for participants with more risk alleles of the rs4074995 SNP, success in weight loss increased with more coaching, although it was not as pronounced in those with minimum (0) risk alleles. It may be that success for those with more risk alleles was not as heavily dependent on more coaching sessions, as the app itself conveys pertinent dietary advice.<SEP>In addition, our data strongly indicate that behavioral engagement, particularly coaching, contributed to weight loss success. Participants experienced, on average, 0.37% more weight loss with each additional coaching session while holding all other model variables constant (Table S9 in Multimedia Appendix 1). All models found weight loss to be significantly associated with behavioral engagement with the program and app (number of coaching sessions completed, weight entries logged, and food photos logged as predictors). Food photos and weight tracking showed more than 98% correlation with each other and both were significantly associated with weight loss success (Tables S6 to S11 in Multimedia Appendix 1). Prior research has shown that regular engagement with digital weight loss platforms and regular weight tracking is associated with greater weight loss success.<SEP>We hypothesized that successful weight loss was achieved by adhering to data-driven dietary recommendations that depart from conventional nutritional weight loss advice. Of those 10.7% (42/393) of the participant population who gained weight, there was a notable lack of engagement in the program. Those who gained weight, compared with their counterparts who lost weight, tended to neither engage in coaching nor regularly use the Digbi Health app to log body weight and post food photos of meals. Those who checked in with the coach regularly and logged into the app frequently to post weight and food photos were more likely to lose weight than those who did not.<SEP>Coaching sessions completed, along with other behavioral engagement variables, differed between participants who lost weight and those who gained weight, whereas baseline weight and BMI did not. The density plots in Figure 6 fairly compare distributions of the two groups: although many more people lost weight than gained, the area under the curve of each group is uniform at 1. Figure 6A illustrates the distributions of completed coaching sessions for those who lost weight (blue) versus those who gained weight (red). The difference is striking: only a fraction of those who failed to lose weight completed at least five (the mean and median) coaching sessions, whereas those who succeeded generally completed five or more. All three measures of engagement were significantly higher in participants who lost weight (blue) versus those who gained weight (red). These distributions are visualized in Figure 6. In contrast, however, Figure S6 in Multimedia Appendix 1 shows no statistical difference in means in (A) baseline weight and (B) baseline BMI, confirmed by the Welch two-sided two-sample t test (P=.64 and P=.42, respectively) between participants who lost (blue) versus gained (red) weight.<SEP>Table 2 and Table S1 in Multimedia Appendix 1 show the distributions of variables based on gender. A notable feature of this cohort is that women are grossly overrepresented—a feature that is not specific to the demographics of obesity. Although globally, more women are obese than men, the disparity is driven in large part by demographics, particularly in Africa and the Middle East. In Western countries, men are more likely to be obese, which is not reflected in our sample. Instead, our participant demographics may be more reflective of individual self-image. Women appear more likely to perceive themselves as overweight and are more likely to attempt weight loss. Exploration of gender differences in weight loss maintenance reported that men comprised only 27% of participants in behavioral weight loss programs.<SEP>Additionally pertinent to the gender composition of this cohort is that patients self-select and continue with the weight management program of Digbi Health based on the approach they feel works for them. One study indicated that women were more than twice as likely to report having used an organized weight loss program, whereas men were more self-directed in their weight loss. In a single-blinded, randomized clinical trial, the efficacies of three e-coaching approaches were compared: no coaching, nondirected coaching, and directed coaching. Women achieved most success (weight loss, reduction of waist circumference, and improvement of physical activity) in the first 12 weeks with directive e-coaching (similar to Digbi Health), whereas men lost more weight with nondirected e-coaching. Men who chose the Digbi Health program succeeded in weight loss. In this cohort, men lost more weight than women over 120 days, consistent with previous findings that men lose weight faster than women. However, the low participation in behavioral weight loss programs, such as that examined here, points to the importance of identifying the underlying factors that impact early engagement and success in weight loss. We are currently undertaking such a study, with the aim of innovating for a level of personalization that will empower and drive success with different subgroups of clients, including men.<SEP>Recent research has explored the incorporation of genotypic information into nutritional advice. Researchers have tested the hypothesis that dietary interventions using personalized genotype information have greater efficacy than the same interventions without genomic data, achieving mixed findings. However, these studies tested only a few SNPs. Some groups found no benefit from genomic information, whereas others found specific SNPs (eg, APOE and ACE genes) to be associated with improved dietary changes. Some studies observed an increase in positive dietary behaviors on the part of participants who carried a risk allele pertinent to a dietary factor (eg, sodium and fat) with whose recommendations they had not been adhering. This aligns with our anecdotal experience of Digbi Health clients, including those in this cohort who had risk alleles for rs17300539, rs2016520, and rs4074995, as described earlier.<SEP>Food4Me conducted an RCT to investigate the effectiveness of internet-based personalized nutrition interventions on weight loss and dietary intake with 1269 participants randomized into four groups: one control without personalization and three personalized groups, each with an additional level of personalization: personalized by individual baseline diet, by baseline diet and phenotype (anthropometry and blood markers), and by baseline diet, phenotype, and genotype (5 SNPs). The group comprising participants combined from all three levels of personalized nutrition experienced significantly better improvements in body weight and BMI at month 3 and a more positive behavioral change than the control group. The authors found no evidence that the addition of either phenotype or phenotype plus genotype information to individual baseline diet data enhanced the efficacy of the personalized intervention. However, none of the three personalization interventions, including individual baseline diet, were reportedly tested in isolation for differences in outcomes from the control protocol, and furthermore only five variants were used in genomic personalization.<SEP>The five variants used in the Food4Me genomic personalization were from the MTHFR, FTO, TCF7L2, APOEε4, and FADS1 genes. Of these five variants, three were present in the Digbi Health panel of 197 curated SNPs, and of these 3 SNPs, only rs7903146 from the TCF7L2 gene was found to be significant to any of our 3 genomic-enhanced models (linear percent weight loss, linear pounds lost, and logistic success or fail). Rs9939609 (FTO gene) and rs1801133 (MTHFR) were the 2 SNPs that were a part of our gene panel, and hence included as variables, but were found to not significantly contribute to any of the three weight loss models. Rs7903146 was found to be significant to the linear percent weight loss model, and although it was selected as a variable by Lasso for the pounds lost model, it was not statistically significant. Research associates rs7903146 with a higher risk of gestational and type 2 diabetes as well as reduced insulin levels. These medical implications are especially relevant to postmenopausal women as well as those in childbearing years. Unlike the pounds lost model, the percent weight loss model captures weight loss independently of start weight and more aptly models women’s weight loss alongside that of men. Digbi Health has a robust protocol to address diabetes risk and insulin resistance, which differs substantially from the Food4Me personalization based on rs7903146 as reflected in the Food4Me article (including the decision tree for TCF7L2-based information delivered to level 3 “Diet plus phenotype plus genotype” in Figure S3 in Multimedia Appendix 1); thus, we would not expect the outcomes of our personalization regarding this SNP to be the same. We noted that Digbi Health personalization arises from the much broader set of traits in our curated panels and a greater number of SNPs associated with those traits, enabling personalization that is more fine-grained than that informed by a few SNPs. We are not aware of any RCT to test the added value of genomic data that uses the breadth of genotypic markers considered in the Digbi Health program.<SEP>Conclusions<SEP>Over the last two decades, the obesity epidemic has coincided with a dramatic change in unhealthy eating habits, a sedentary lifestyle, and physical inactivity. In the United States, more than 40% of the adult population is now overweight or obese. Hereditary predisposition to obesity may have interacted with the obesogenic environment and contributed even further toward the epidemic. The recent accumulation of genomic and lifestyle data has led to the demonstration of possible effects of gene-environmental interactions on obesity. Data from dietary intervention trials indicate that genetic variants, particularly those linked to obesity, metabolism, and nutrient consumption, may significantly alter changes in adiposity and metabolic responses to nutritional interventions and promote effective weight loss.<SEP>In the foreseeable future, the incorporation of data on genes, eating patterns, metabolites, and gut microbiome into weight loss interventions will be one of the most promising fields of precision care and may allow for the generation of predictable weight loss models based on individual genomic, microbiomic, and metabolomic factors. The goal is precision nutrition, individually tailored to enable effective weight loss and prevent chronic diseases on the basis of genomic history; habitual consumption of food and drink; intake of nutrients (especially those that contribute to disease risks); and metabolomics, microbiome, and other omics profiles of a person.<SEP>Although using precision medicine to target heterogeneous conditions may seem counter-intuitive, it is the heterogeneous nature of conditions, such as obesity and metabolic illness, that make them such potent targets for intervention, impacting the greatest number of people. Obese subpopulations identified as genetically predisposed to favorably or unfavorably respond to a given weight loss intervention could be targeted accordingly.<SEP>To date, few studies have investigated metabolomic functioning, lifestyle and behavioral mechanisms, and gut microbiome, which can affect obesity and health at the interface between genetic variation and the environment. The Digbi Health digital precision weight loss program operates at this interface. This study was limited by its retrospective and descriptive nature. The field of precision nutrition would benefit from additional prospective randomized controlled studies on a larger scale. Although such studies will be needed to validate these findings, the analysis and modeling presented here appear to support dietary precision interventions considering genetic predisposition to disease and genetic variants defining dietary preference and metabolic risk. In addition, our results point to the efficacy of coaching that empowers and actively engages participants in their own success.<SEP>Future studies should explore the synergistic effects of genomic variables in interactions with other genome, microbiome, and lifestyle and behavior variables. A follow-up to the work presented here, exploring not only the effect of incorporating genomic data but also including the microbiome data used in Digbi Health precision care, is currently in preparation. Personalized protocols that incorporate data on genes, eating patterns, metabolites, and gut microbiome into weight loss interventions may well be a promising field of precision care, allowing for the generation of predictable weight loss models that account for the synergistic effect of these influential factors.<SEP>Conflicts of Interest: Digbi Health is sponsoring this study, and the principal investigator and study staff have a financial interest in the company.<SEP>RS is the founder and CEO of Digbi Health. Authors DK, RRR, SS-R, KMM, VS, CI, CR-S, GS, and PAF-L are employees at Digbi Health. IJ is the Chief Medical Officer employed by Digbi Health.",
                  "55470",
                  "['OBWCCE', 'EATBTS', 'FOODIN']"
                ],
                [
                  "20",
                  "32826294",
                  "Introduction<SEP>Genome-wide association studies (GWAS) have provided a detailed inventory of genetic loci conferring susceptibility to type 2 diabetes (T2D). A study of ∼900,000 individuals of European descent identified >400 association signals. To date, most studies of T2D predisposition have focused on the detection of main effects attributable to individual genetic variants. However, there has been longstanding interest in understanding the contributions of gene-gene interactions (GGIs) to individual predisposition to T2D. This interest was initially driven by the possibility that non–log-additive interactions might explain the apparent failure of main effects to account for the observed heritability of T2D. More recently, the search for GGIs has been motivated by the desire to establish whether second-order genetic effects could improve disease prediction models on the basis of genotype data and by the potential for statistical interactions to provide clues to underlying disease mechanisms. To date, there has been little evidence to indicate that GGIs have any appreciable impact on T2D risk.<SEP>However, it is clear that detection of all but the largest GGI effects require sample sizes substantially larger than those used to identify the main effects. Furthermore, the sample sizes necessary to detect GGI scale exponentially (precisely, to the fourth power of the correlation coefficient) when the variants being tested are only partially correlated with the (often unknown) causal variant.<SEP>The UK Biobank study, which provides genetic data for ∼500,000 individuals, offers a singular opportunity to advance the exploration of GGIs in T2D. We sought to capitalize on recent advances in the characterization of T2D risk loci resulting from fine-mapping efforts that have improved localization of the causal variant at many risk loci.",
                  "Research Design and Methods<SEP>UK Biobank Study Population<SEP>The UK Biobank is a prospective cohort study of ∼500,000 individuals from across the U.K. We used imputed genetic data from the March 2018 release (version 3); details regarding quality control and imputation are provided elsewhere. We generated discrete genotypes using a genotype probability threshold of 0.8 and excluded variants with info score <0.5 and Hardy-Weinberg disequilibrium (P < 10−6). Individuals with discordant sex, putative sex chromosome aneuploidy, withdrawal of consent, and diagnosis of another form of diabetes, such as type 1 or gestational diabetes mellitus, were excluded. We further selected a subset of 375,736 individuals (16,430 T2D cases, 359,306 controls) who were 1) unrelated (up to second-degree relatives determined using the KING toolset [https://people.virginia.edu/∼wc9c/KING/manual.html]) and 2) of European ancestry (determined using a combination of 15 principal components [PCs] provided by UK Biobank and self-reported white British ancestry). Variance-weighted PC scores were used to calculate the “genetic distance” with a hypothetical median white British participant to identify European individuals (genetic distance <60 units). Prevalent T2D status was defined using self-reported medical history and medication information.<SEP>Prioritization of Variants for GGI Analysis<SEP>We used the following two approaches. The first approach was to select variants associated with T2D risk (the T2D risk set). We selected the index variant at each of the 403 conditionally independent association signals reported in the largest T2D GWAS in Europeans.<SEP>The second approach was to select variants associated with heterogeneity in T2D variance (the T2D variance set): variants that demonstrate marked heterogeneity in phenotype variance across genotypes represent potential candidates for GGI effects. To detect such variants, we selected a random subset of 100,000 unrelated European subjects (4,284 T2D cases, 95,716 controls), adjusted their T2D status for age, sex, genotyping batch, and 10 PCs; standardized the residuals; and used Levene test to assess equality of variance across genotypes (including only variants with minor allele frequency [MAF] >5%). For each 1-Mb block of the genome, we identified the variant most significantly associated with T2D variance (index variant). Finally, we selected the 100 most significant index variants that did not overlap with variants in the T2D risk set (Supplementary Table 1). The union of T2D risk and T2D variance sets provided a T2D joint set of 503 variants.<SEP>Analysis of GGIs for T2D<SEP>GGI effects were sought using two complementary strategies (Supplementary Fig. 1). The first strategy was single variant-based GGI analysis. We tested for interactions between individual variants using the –epistasis function in PLINK (https://www.cog-genomics.org/plink2), which fits a logistic regression model (assuming a log-additive model of disease risk), as follows: ln(P [T2D] / P [control]) = β0 +β1SNP1 +β2SNP2 +β3SNP1SNP2, where SNP1 and SNP2 refer to the variants being tested, β1 and β2 refer to their main effects, and β3 refers to their interaction effect on T2D.<SEP>We deployed two analytical approaches: in the T2D joint set pairwise analysis, interaction was tested between each pair of variants in the T2D joint set; and, in the T2D joint set-versus-genome set analysis, interaction testing for the T2D joint set variants was extended to variants in the remainder of the genome (the genome set). Power calculations were performed to estimate the MAF threshold (10%) for a variant in the genome set above which there was adequate power (>75%) to detect a substantial interaction effect (which we defined as an odds ratio [OR] >1.5) with a variant of MAF = 5% from the T2D joint set on the basis of the following parameters: 1) main effect OR on T2D for T2D joint set variant = 1.1; 2) main effect OR on T2D for genome set variant = 1.0; 3) interaction effect OR between the variants = 1.5; 4) sample size = 375,736, with a T2D case:control ratio (per UK Biobank) of 0.044; and 5) single test α = 0.05, with Bonferroni adjustment for 503 genome-wide analyses. Since the presence of linkage disequilibrium (LD) can confound interaction tests, we removed variants in the genome set in LD (r2 > 0.1 within 1 Mb) with a given T2D joint set index variant.<SEP>The second strategy was polygenic score (PS)–based GGI analysis. Compared with the single variant–based approach, the PS-based approach for testing GGIs offers greater statistical power if the underlying interaction effects for multiple T2D risk alleles are shared across alleles. We aggregated effects of the 403 variants in the T2D risk set to construct an overall T2D PS using the –score function in PLINK. The risk allele dosage for each variant was weighed by the effect size obtained from a T2D meta-analysis of 455,302 European individuals that included all studies from Mahajan et al. except UK Biobank.<SEP>Similarly, a set of 93 T2D variants, which were either members of the T2D risk set or proxies (median r2 = 0.91) thereof, was used to construct six (weighted) partitioned PS (pPS) that captured biological processes relevant to T2D pathophysiology (Supplementary Table 2). These pPS are referred to as pPSIS1, pPSIS2 (both reflecting processes involved in insulin secretion), pPSIA (insulin action), pPSadiposity (overall adiposity), pPSdyslipidemia (predominantly affecting liver metabolism), and pPSmix (a mixture of the above). The overall PS and the pPS were standardized to represent SD units. For the PS-based interaction analysis, we tested interactions between each PS (overall PS and six pPS) and genome-wide variants (MAF ≥1%; ∼6 million) variants.<SEP>Data Resource and Availability<SEP>The summary statistics of genome-wide analyses performed in this study are available at https://zenodo.org/record/3978776#.XzIAzC3MzRY.",
                  "Results<SEP>The approaches deployed to characterize GGIs in T2D are summarized in Supplementary Fig. 1. These analyses were performed in 375,736 unrelated European individuals from UK Biobank (16,430 T2D cases, 359,306 controls) (research design and methods).<SEP>Analysis of GGIs for T2D<SEP>Since testing all genome-wide variants for interaction presents computational and statistical challenges, we prioritized two sets of variants for single variant analysis: the T2D risk set of 403 index variants, representing conditionally independent association signals for T2D risk in Europeans, and the T2D variance set, a nonoverlapping set of 100 index variants selected for the most extreme effects on T2D variance within UK Biobank (indicative of possible interaction effects). Our primary analyses considered the combined T2D joint set of 503 variants (research design and methods).<SEP>We first sought pairwise interactions between variants in the T2D joint set. None of the pairwise interactions crossed the Bonferroni-corrected significance threshold (α = 4 × 10−7, correcting for 126,253 pairwise tests) (Supplementary Table 3). We estimate >70% power to detect an interaction effect of OR ≥1.5 between two variants in the T2D joint set, when both have an MAF at the lower end of the common variant range (5%) and main effect T2D OR = 1.1 (Supplementary Fig. 2). For more common variants (MAF = 50%), we were powered for interaction effects OR >1.2. The quantile-quantile plot provided no evidence that the distribution of GGI effects departed from the null (Fig. 1A). The strongest signal in this pairwise GGI analysis, involving rs629137 (near UVRAG, T2D variance set) and rs76011118 (near CDKN2A/B, T2D risk set), had an interaction OR of 1.24 (95% CI 1.21–1.27; P = 1 × 10−5). No significant pairwise interactions were observed when analyses were restricted to just the T2D risk set (81,003 tests, α = 6 × 10−7) or the T2D variance set (4,950 tests, α = 1 × 10−5) (Fig. 1A).<SEP>Next, we tested interactions between T2D joint set variants and the remainder of the genome. For the latter, we focused on ∼3.2 million variants with MAF >10%, since power calculations indicated >75% power to detect an interaction effect of OR ≥1.5 for variants above this MAF threshold, given a joint set MAF at the lower end of the common variant range (5%) (Supplementary Fig. 3). Again, we found no evidence for significant interactions at α = 1 × 10−10 (accounting for 503 genome-wide analyses) (Figs. 1B and 2 and Supplementary Table 4). The strongest signal involved two variants near TCF7L2: rs184509201, a T2D risk set variant corresponding to one of seven secondary signals at this locus, and rs10885397, a variant from the genome set (interaction OR 1.55 [1.51–1.59]; P = 2 × 10−9). These two variants, located ∼28 kb apart, are not in LD (r2 = 0.0). Genome-wide interaction analyses with variants exclusive to the T2D risk set (α = 1 × 10−10) or the T2D variance set (α = 5 × 10−10) were similarly negative.<SEP>Since the power to detect GGI effects for individual variants can be limited even with large sample sizes, we sought to bolster GGI detection by aggregating T2D risk variants into PS. In addition to an overall PS generated from the 403 variants in the T2D risk set, we constructed six additional pPS using a subset of 93 T2D risk variants that were stratified into physiological clusters capturing biological processes relevant to T2D pathophysiology (research design and methods).<SEP>Analyses evaluating GGIs using a PS-based approach failed to detect significant interactions between these PS and variants in the genome set (α = 7 × 10−9, accounting for seven GWAS) (Supplementary Table 5 and Supplementary Figs. 4 and 5). The strongest signal from this analysis was between variants at the CHN2 locus and pPSdyslipidemia (interaction OR 1.22; P = 1 × 10−7); this locus has previously been associated with diabetic retinopathy and development of severe insulin resistance.",
                  "Discussion<SEP>By addressing several limitations of previous analyses, the current study offers substantially improved power for detecting GGI effects influencing T2D risk. First, we conducted GGI analyses in a much larger sample size than earlier efforts. Second, we prioritized sets of genetic variants likely to be enriched for interaction effects. Third, we leveraged improved fine-mapping of association signals, avoiding the attrition of power for signals only partially correlated with the causal variant. Fourth, we implemented a PS-based approach to complement the single variant–based interaction analysis. Despite using all these measures, we found no credible evidence either in the overall distribution of interaction effects or in individual signals meeting study-wide significance for GGI effects influencing T2D risk.<SEP>There are some clear limitations to this analysis. For computational and statistical reasons, we chose not to test all genome-wide variants for interaction (requiring ∼1013 tests). Nevertheless, the sets of variants prioritized were those most likely to demonstrate interaction effects, and the lack of significant associations within this subset should be indicative of patterns seen more broadly. Additionally, our study did not rule out the possibility of higher-order interactions and interactions detectable on a scale of disease-risk other than log-additive.<SEP>We are unable to determine the extent to which more subtle or infrequent GGI effects, singly or in combination, contribute to the residual heritability not attributable to known T2D risk variants. Nevertheless, the absence of detectable major GGI effects on T2D risk implies that for many practical clinical and epidemiological purposes, the joint effects of multiple genetic risk factors for T2D can be derived purely from the combination of main effects. Although it has been suggested that statistical GGIs might be indicative of functional interactions and, thereby, provide mechanistic insights into disease biology, the extent to which this holds for common variants influencing complex human diseases remains to be established.<SEP>M.I.M. and A.M. are currently affiliated with Genentech, South San Francisco, CA.<SEP>This article contains supplementary material online at https://doi.org/10.2337/figshare.12827219.",
                  "14162",
                  "['OBWCCE']"
                ],
                [
                  "21",
                  "23667675",
                  "Introduction<SEP>Age at menarche (AAM), the onset of the first menstrual period in girls, is considered as a landmark of female pubertal development. Menarche generally occurs after a series of complex neuroendocrine events leading to full activation of the hypothalamic-pituitary-gonadal axis. Menarche is associated with physical, emotional, and social development. In addition, AAM was shown to be associated with the risk of various diseases. Early AAM is reported to be one of the significant risk factors for depression, eating disorders, obesity, diabetes, breast cancer, and coronary heart disease. On the other hand, late AAM has been associated with osteoporosis and taller adult stature. Therefore, the identification of loci contributing to variation in AAM could lead to a better understanding of a wide range of phenotypes.<SEP>AAM is known to be a complex trait determined by an array of genetic and environmental variables. Twin and family studies suggest a significant genetic contribution to AAM with a heritability of more than 50%. Several genetic variations within candidate genes such as the estrogen receptor genes (ESR1 and ESR2), CYP19A1 , and the SHBG gene were shown to be associated with AAM. To date, a number of genome-wide linkage analyses and genome-wide association studies (GWAS) for genes underlying variation in AAM have been performed. In 2009, the association of genetic variations in LIN28B with AAM was identified by four independent groups. Currently, more than 30 loci have been shown to be significantly associated with AAM. However, most of these studies were conducted using women of European ancestry. Here we performed a large scale meta-analysis of GWAS using more than 15,000 Japanese female samples.",
                  "Methods<SEP>Samples<SEP>The subjects enrolled in the GWAS meta-analysis for age at menarche (AAM) (n = 15,495) consisted of female patients that were classified into 33 disease groups; cancer (lung, esophagus, stomach, colon, liver, cholangiocarcinoma, pancreas, breast, uterine cervix, uterine body, ovary, hematopoietic organ), diabetes, myocardial infarction, brain infarction, arteriosclerosis, arrhythmia, drug eruption, liver cirrhosis, amyotrophic lateral sclerosis, osteoporosis, fibroid, and drug response, rheumatoid arthritis. chronic hepatitis B, pulmonary tuberculosis, keloid, heat cramp, brain aneurysm, chronic obstractive lung disease, glaucoma, and endometriosis ( Tables 1   and   2 ). All subjects were collected under the support of the BioBank Japan Project, in which the individuals with any one of the 47 common diseases were enrolled between 2003 and 2008. Subjects under 17 years of age were not included in this study. Various life style information such as AAM was obtained through a face-to-face interview by trained medical coordinators using questionnaire sheets at each hospital. All participants provided written informed consent as approved by the ethical committees of each institute. For participants between the age of 17 and 20 years old, we obtained written informed consent from both the participant and her parents. AAM was collected by self-report on the questionnaire. The subjects with AAM between the ages of 10 and 17 were enrolled for the study. This project was approved by the ethical committees of the BioBank Japan Project and the University of Tokyo.<SEP>Genotyping and quality control<SEP>In the four GWAS enrolled in the meta-analysis of AAM, all samples were genotyped at more than 500,000 loci using one of the following platforms: Illumina HumanHap550v3 Genotyping BeadChip, Illumina HumanHap610-Quad Genotyping BeadChip, or Illumina Omni express Genotyping BeadChip ( Table 1 ). Then we applied the following quality control for each GWAS separately: exclusion criteria; subjects with call rates<0.98, SNPs with call rates<0.99 or with ambiguous clustering of the intensity plots, or non-autosomal SNPs. We then excluded subjects whose ancestries were estimated to be distinct from East-Asian populations using principle component analysis performed by EIGENSTRAT version 2.0. Subsequently, the SNPs with MAF<0.01 or P-value of the Hardy-Weinberg equilibrium test<1.0×10−7 were excluded.<SEP>After the quality control criteria mentioned above were applied, genotype imputation was performed by MACH 1.0.16 using the genotype data of Phase II HapMap JPT and CHB individuals (release 24) as references, in a two-step procedure as described elsewhere. In the first step of the imputation, recombination and error rate maps were estimated using 500 subjects randomly selected from the GWAS data. In the second step, imputation of the genotypes of all subjects was performed using the rate maps estimated in the first step. Quality control filters of MAF≥0.01 and Rsq values≥0.7 were applied for the imputed SNPs.<SEP>Statistical analysis<SEP>Associations of the SNPs with AAM were assessed by linear regression assuming the additive effects of the allele dosages on AAM, using mach2qtl software. Years of birth and affection statuses of the diseases were used as covariates. Meta-analysis of all four GWAS was performed using an inverse-variance method from the summary statistics of beta and standard error (SE), using the Java source code implemented by the authors. Genomic control correction was applied for each GWAS separately, and applied again for the results of GWAS meta-analysis. We set the P-values of 5.0×10−8 and 0.0015 ( = 0.05/33, Bonferroni's correction based on the numbers of the evaluated loci) as significance thresholds in GWAS and candidate gene analysis, respectively. Heterogeneity of the effect sizes among the studies was evaluated using Cochran's Q statistics.<SEP>Body mass index and Height QTL analysis<SEP>The associations of 42 SNPs with BMI and height were evaluated using previously published results, in which a total of 26,620 subjects with 32 diseases from Biobank Japan were enrolled. Of the 26,620 subjects, 12,350 were also included in this AAM study. Genotyping was performed using the Illumina HumanHap610-Quad Genotyping BeadChip. Genotype imputation was performed using MACH 1.0. BMI was calculated based on self-reported body weight and height data. A rank-based inverse-normal transformation was applied to the BMI values of the subjects. Associations of the SNPs with transformed values of BMI were assessed by linear regression assuming additive effects of allele dosages (bound between 0.0 and 2.0) using mach2qtl software using gender, age, smoking history, the affection statuses of the diseases and the demographic classifications of the medical institutes in Japan where the subjects were enrolled were used as covariates.<SEP>Web resources<SEP>The URLs for data presented herein are as follows. BioBank Japan Project, http://biobankjp.org <SEP>MACH and mach2qtl software, http://www.sph.umich.edu/csg/abecasis/MACH/index.html International HapMap Project, http://www.hapmap.org PLINK software, http://pngu.mgh.harvard.edu/~purcell/plink/index.shtml EIGENSTRAT software, http://genepath.med.harvard.edu/~reich/Software.htm R statistical software, http://cran.r-project.org.",
                  "Results<SEP>A total of 15,495 Japanese female subjects from four GWAS using different SNP genotyping systems were enrolled in this analysis. Characteristics of samples and genotyping methods are summarized in  Table 1 . All the subjects were of Japanese origin and obtained from the Biobank Japan Project. Samples consist of patients that were classified into 33 disease groups. The average and S.D. of AAM in each disease cohort is shown in Table S1. Some diseases such as breast cancer and osteoporosis are likely to be associated with early or late AAM, as reported previously. We also found that AAM was negatively associated with birth year (p<0.0001). Thus we used disease status and birth year as covariates in this study. Genotyping was performed with over 500,000 SNP markers using Illumina HumanHap 550 Genotyping BeadChip, Illumina610-Quad Genotyping BeadChip, or Illumina Omni Express (Illumina, CA, USA). We applied stringent quality control criteria as mentioned in the methods section. We also conducted principal component analysis to evaluate potential population stratification. To extend the genomic coverage and conduct meta-analysis, we subsequently performed a whole-genome imputation of the SNPs, using HapMap Phase II genotype data. After the imputation, we performed SNP quality control (minor allele frequency≥0.01 and an imputation score (Rsq value by MACH software)≥0.7) and found that more than two million autosomal SNPs satisfied these criteria.<SEP>The associations of these imputed SNPs with AAM were evaluated using a linear regression model and meta-analysis. Quantile-Quantile plots of P-values indicated the Inflation factors to be as low as 1.039 ( Figure 1 ), suggesting no substantial population stratification existed in our study population. Although we could not identify significant association that satisfied the genome-wide significant threshold (P<5×10−8), SNP rs364663 which is located within intron 2 of the LIN28B gene at 6q21 indicated the strongest association with a P-value of 5.49×10−7 ( Table 2 ,  Figure 2 ). SNP rs364663 exhibited the association with AAM in all four cohorts without significant heterogeneity in both effect sizes and directions (P-value for heterogeneity = 0.29;  Table 3 ), suggesting that the observed associations at LIN28B are not the result of false-positives due to study-specific bias. Regional p-value plots indicated that all of the AMM-associated SNPs were clustered around the LIN28B locus ( Figure 2 ). Previously reported SNPs rs314263 and rs7759938 near the LIN28B locus also associated with AAM (P = 1.03×10−6 and 1.12×10−6, respectively) in the same direction. In addition to the 6q21 locus, 17 SNPs in seven genomic regions exhibited suggestive association with a p-value of<1×10−5 ( Table 2 ). To further investigate the physiological role of these loci, the associations of these variations with body mass index (BMI) and height were evaluated using previously published results in 26,620 Japanese subjects. SNP rs9404590 near the LIN28B locus associated with height with p-value of 0.0003 (Table S2), but we did not find significant association (P<0.01) between these loci and BMI (Table S2).<SEP>Since AAM is associated with various disease risks, we conducted separate analyses for each disease and then performed meta-analysis for the top 42 loci. As a result, some SNPs showed slightly stronger association, but none cleared the genome wide significant threshold (Table S3). Similar to the current study, our group had previously conducted QTL analyses using disease status as a covariate and successfully identified many QTL loci. Therefore, different background due to disease status was unlikely to significantly affect the result of association analysis.<SEP>Additionally, we examined the loci already known to show significant association with AAM in women of European ancestry. We selected 37 SNPs for this candidate analysis and successfully obtained the genotyping results of 33 SNPs ( Table 4 ), and the risk allele was consistent with previous reports for 31 SNPs. In addition, eight SNPs in or near RXRG, CCDC85A, LOC100421670, TMEM38B, ZNF483, ARNTL, and CA10 indicated possible associations with AAM (P<0.05). Among them, rs4452860 and rs7028916 at the TMEM38B locus exhibited significant association even after Bonferroni's correction (P<0.0015 = 0.05/33). Taken together, these variations as well as LIN28B are likely to be common AAM loci, although their effect sizes are different between women of European ancestry and those of Japanese.",
                  "Discussion<SEP>AAM is a complex trait that is influenced by both genetic and environmental factors. In recent years, genome wide association analyses have become a standard method to identify genetic factors related with various diseases and phenotypes. In 2002 our group performed the first GWAS for myocardial infarction and successfully identified LTA as a disease susceptibility gene. Using this method, we have identified a number of loci associated with various phenotypes and common diseases.<SEP>Through the analysis of more than 15,000 Japanese female subjects from the Biobank Japan Project, we here report the association of LIN28B with AAM. In our analysis, SNP rs364663 within intron 2 of the LIN28B gene exhibited the strongest association. Previously reported SNPs rs314263, rs7759938, rs314280, and rs314276 near or in the LIN28B gene also showed equivalent association with p-values of 1.03−1.57×10−6. Although the directions of associations were consistent between women of European ancestry and those of Japanese, the effect sizes among Japanese (0.085–0.087) were less than those among European ancestry (0.09–0.14). Thus LIN28B is a common AAM loci, but its effects are relatively small among the Japanese population.<SEP>The lin-28 gene was originally identified through C. elegans mutants showing abnormality in developmental timing. Deleterious mutations in lin-28 produce an abnormal rapid tempo of development through larval stages to adult cuticle formation. Two mammalian homologs, Lin28a and Lin28b, possess similar biochemical activities. Transgenic mice expressing Lin28a exhibited increased body size, crown-rump length and delayed onset of puberty due to increased glucose metabolism and insulin sensitivity. In humans, both LIN28B and LIN28A control preprocessing of the let7 microRNA family. Lin28B is highly expressed in the majority of human hepatocellular carcinomas and embryonic stem cells, and regulate cell pluripotency as well as cancer growth. However the role of genetic variations in the LIN28B locus should be investigated in the future study.<SEP>Among the seven suggestive loci indentified in our GWAS, ZFHX4 and NKX2.1 loci were previously shown to be associated with height in the Caucasian population. These loci also exhibited association with height among the Japanese population (p = 0.053 and 0.023, respectively). The NKX2.1 gene encodes a thyroid-specific transcription factor that was shown to be associated with hypothyroidism. Since delayed pubertal development is a common manifestation of hypothyroidism, variation at NKX2.1 locus might affect AAM through the regulation of serum thyroid hormone level. The ZFHX4 gene encodes a homeodomain-zinc finger protein. ZFHX4 mRNA is highly expressed in brain, liver, and muscle, however the molecular mechanism whereby variations within this genetic locus affect AAM is not clear. In addition, KIRREL3 locus was shown to be associated with breast size in women of European ancestry. The KIRREL3 gene encode a member of the nephrin-like protein family. KIRREL3 was highly expressed in brain and kidney, and shown to be associated with mental retardation autosomal dominant type 4 and neurocognitive delay associated with Jacobsen Syndrome. Taken together, these three loci seem to be common development traits in women of both European ancestry and Japanese.<SEP>In our candidate analysis, the association of SNPs rs4452860 and rs7028916 on 9q31.2 with AAM was also validated. This locus was repeatedly shown to be associated with AAM as well as serum ALT (alanine transaminase) level. SNPs rs4452860 and rs7028916 were located more than 300 kb away from the TMEM38B gene. In mice, TMEM38B is strongly expressed in brain, and TMEM38B deficient mice is neonatal lethal. In humans, a homozygous mutation of TMEM38B was associated with autosomal recessive osteosgenesis imperfecta. These findings also suggest an important role of TMEM38B in developmental processes.<SEP>In this study, we observed early AAM among breast cancer patients compared with overall samples (12.99 vs 13.44, Table S1). On the other hand, patients with osteoporosis exhibited late AAM (14.33 y.o.). These findings are concordant with previous reports. Since we do not have age-matched controls, the association of these diseases and AAM should be examined using more appropriate sample sets.<SEP>Although more than 15,000 samples were employed in this study, no SNP cleared the genome wide significance threshold (p<5×10−8). This could be explained by some limitations in our study. Firstly, the information about AAM was self-reported and might not be very accurate. Although this distinct event is often well recalled many years later, possible error due to age-associated memory impairment would increase the risk of false negative association. Secondly, we did not have a replication sample set, and the total sample size may be inadequate to detect variations with modest effects. At present, we unfortunately do not have sufficient samples for replication, since more than 2,900 additional samples are necessary to obtain a genome wide significant association for the top SNP rs364663. Previous studies comprising up to 17,510 women detected only one or two genome-wide significant signals. However, 30 significant loci were identified by using 87,802 subjects in the screening stage. Although no SNPs reached the genome wide significance in our study, analyzing an increased number of subjects in a relatively younger generation would improve statistical power as well as data accuracy, and subsequently enable us to identify other genetic factors with modest effects.",
                  "17343",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "22",
                  "32564308",
                  "Background<SEP>The prevalence of obesity is consistent with more than one third of adults having the disease of obesity. Obesity is the focus for many public health efforts in the USA with one treatment option being bariatric surgery. Achieving weight loss is a benefit from bariatric surgery; however, micronutrient deficiencies can occur. Micronutrient deficiencies are associated with serious consequences due to the negative effects on metabolic and cellular signaling pathways. Possible causes of micronutrient deficiencies after bariatric surgery are decreased food intake, food intolerance, reduced gastric secretions, bypass of intestinal surface area for absorption, as well as failure to comply with recommended vitamin regimens. Multiple case series have reported postoperative, malabsorptive procedures to increase prevalence of iron deficiency to 20–49%, calcium and vitamin D deficiency 25–50%, vitamin B12 deficiency about 33%, and folate deficiency as high as 45%. For malabsorptive procedures, patients are recommended to take at least double the recommended daily dose of a multivitamin plus mineral supplement and additional 1200–2400 mg calcium, 3000 IU vitamin D to reach levels > 30 ng/mL, and vitamin B12 as needed for normal levels. In a cohort of adults who underwent bariatric surgery, 73% of the patients had at least one nutritional deficiency 5 years later even though they reported taking a dietary supplement. However, there are some patients, up to 47%, that may be non-responders to supplements even with compliance rates of about 86–93%. Indeed, individuals respond differently to dietary interventions. Genetic variation among individuals could be the root cause for varying responses to the same regimen and explains why some individuals respond better to a certain regimen than others in the same environmental conditions.<SEP>Introduction<SEP>Genetic testing can be a critical tool for health and medical diagnosis, treatment, and prevention. Predictive testing may be among the most useful tests regarding medical nutrition therapy (MNT). Genetics along with environment and behavior are the key to providing the best assessment, intervention, and tailored changes for an individual. MNT should follow an appropriate paradigm that encompasses prediction (early diagnosis), prevention (intervention on healthy persons), and a tailored therapy for patients. Identifying ways for early intervention may help develop strategies for preventing poor nutritional status and maximizing surgery-induced metabolic benefits later.<SEP>Sequencing of the human genome and identifying gene-nutrient interactions are the underlying concept of PN. Nutrigenomics is the study of the effect of specific nutrients on gene expression, while nutrigenetics refers to the study of genetic variations of an individual that can provide some prediction to help prevent as well as contribute to personalized dietary management. Both nutrigenomics and nutrigenetics may be a strategy to improve understanding of the gene-diet interaction and deliver individualized MNT to prevent chronic nutrition-related diseases [(], [)]. The usefulness and validity of this type of PN are in their infancy, although some studies have shown that individuals find dietary recommendations based on genetics more beneficial than general dietary advice. A survey conducted by the publisher Nature showed that 27% of respondents who had their genomes analyzed changed their diet, lifestyle, or medication based on their genetic information. However, another study reported that genetic testing led to no short-term changes in specific dietary or exercise behaviors. Thus, increased understanding and awareness of these tests is required to effectively use them among public and healthcare providers.<SEP>Since many micronutrients control energy metabolism, their deficiencies can result in an array of symptoms, ranging from anemia to neurological dysfunction. Additionally, subclinical micronutrient deficiency can lead to increased risks for coronary artery disease, infections, age-related macular degeneration, and oxidative damage. Therefore, measuring nutritional status in the context of pathophysiology is critical, but this is a major challenge because it is influenced by a number of factors including dietary consumption, physical/social stressors, and infections. Furthermore, the impact of nutrition could vary among individuals and specific population subgroups based on their molecular and genetic make-up. Studying this complex nutrient-gene relationship to understand the metabolic networks in context of health and disease should be a focus. It can provide information on potential biomarkers of nutritional status, disease progression, and response to interventions. This literature review aims to summarize data from studies of genes involved in micronutrient metabolism. Identifying these nutrient-gene pathways and their variants can help predict those at risk for deficiencies. This may recognize the need for increasing consumption of essential nutrients to intervene prior to bariatric surgery and develop strategies to prevent micronutrient deficiency postoperatively.",
                  "Methods<SEP>Due to the limited amount of literature published on micronutrient deficiencies, micronutrient genetic variants, and bariatric surgery, the authors conducted a narrative review. A comprehensive search of the literature from 1975 to 2020 was conducted to identify articles examining the association between genetic variants of micronutrient metabolic pathways and serum levels of micronutrients. Searches were conducted in databases that contain research related to health and metabolic outcomes, including PubMed, Medline, and Google Scholar. The search terms that were used included genetic variants, micronutrient metabolism, treatment of genetic defects of micronutrients, precision nutrition, nutrigenomics, and bariatric surgery. Additionally, review articles produced through the database searches were examined for further articles that fit within the inclusion criteria and thus were included in the results.<SEP>Inclusion/Exclusion Criteria<SEP>Criteria for inclusion in the review were peer-reviewed articles, articles that included empirical data, articles published or available in English, articles that included people with low micronutrient levels, articles that included people that had micronutrient genetic variants, supplementation/treatment regimens for people with genetic variants and low micronutrient levels, and articles that included genetic testing among obesity and bariatric surgery populations and outcomes that were examined. Only two articles involving micronutrient genetic variants among bariatric surgery patients were found. Due to this limitation, studies involving non-surgical patients and micronutrient genetic variants were included. Other studies involving genetic testing and bariatric surgery were included to demonstrate its potential as a tool for this patient population. There were also limitations on studies involving treatment and supplementation according to genetic variants. Ideally, inclusion criteria would comprise of studies with a high number of participants, a control group, and that used similar measures and procedures across studies for comparison; however, using the search methods and criteria described above, 80 articles met the inclusion criteria. All the authors confirmed that the articles met the inclusion criteria and were appropriate for the review. The articles that met the inclusion criteria focused on micronutrient genetic variants and genetic testing in diverse populations shifting to describe the findings.",
                  "Results<SEP>Of the 80 articles included, 2 were published between 1975 and 1990, 5 were published between 1991 and 2000, 25 were published between 2001 and 2010, and 48 were published between 2011 and 2020. Twenty-two of these studies took place in the USA, and the rest were conducted in other countries. The articles were divided into four categories which are used to organize the results: [1] micronutrient deficiencies prevalent among bariatric surgery patients (19 articles), [2] micronutrient genetic variants prevalence among different populations (29 articles), [3] clinical trials involving supplementation for micronutrient genetic variants (26 articles), and [4] genetic testing studies in persons with obesity and bariatric surgery populations (6 articles).<SEP>Genetic Variants and Their Effect on Vitamin and Mineral Pathways and Response to Supplementation<SEP>Genetic variations in specific genes among vitamin and mineral metabolic pathways are associated with altered nutrient homeostasis and adverse health outcomes. SNPs are the most common type of genetic variations among people. In the human genome, SNPs may occur at every 1000 nucleotides, which means that a person may have 4–5 million SNPs. SNPs are known to impact micronutrient status or chronic diseases related to micronutrient metabolism. The ability to identify a person having genetic variants involved in vitamin and mineral metabolism may reduce the chance of developing micronutrient deficiencies that can lead to various diseases. GWAS have shown that several genetic variants associated with vitamin metabolism can affect circulating vitamin levels, which could lead to abnormal vitamin function. Most GWAS have been conducted among healthy, Caucasian populations, which is a limitation in this research. Table 1 demonstrates recent studies that associate genetic variants and micronutrient metabolism.<SEP>Vitamin D<SEP>Vitamin D is essential for many functions of the body. Deficiency of vitamin D is associated with many cancers, autoimmune disorders, and cardiovascular disease as well as significantly affects musculoskeletal function. Obesity has been identified as a risk factor of vitamin D deficiency, and those seeking bariatric surgery for obesity treatment have an additional risk for low vitamin D levels post-op. One study showed that 57.4% of patients seeking bariatric surgery were vitamin D-deficient preoperatively. In 51 observational studies assessing vitamin D status in patients undergoing bariatric surgery, the mean (25(OH)D) level was less than 30 ng/ml (which is the minimum recommended level for optimal long-term health), before and after bariatric surgery, despite various vitamin D supplementation regimens. Another review of 30 studies showed vitamin D deficiency prevalence to range from 13 to 90% preoperatively which was maintained after surgery.<SEP>The heritability of vitamin D status is estimated to be 30% and common variants group-specific component (GC) (also known as vitamin D–binding protein); 7-dehydrocholesterol reductase (DHCR7) and CYP2R1 (involved in 25-hydroxylase production) are associated with fasting plasma 25(OH) D concentrations. Nissen and colleagues have shown that 7 prominent variants in CYP2R1 and GC genes were significantly associated with low serum 25(OH) D concentrations. People who have these common genetic variations could be treated on a more individualized basis to correct deficiencies that occur.<SEP>One randomized controlled trial looked at older Australians randomly assigned to monthly doses of 30,000 IU or 60,000 IU vitamin D3 for 12 months and found that genetic variability is associated with response to supplementation, perhaps suggesting that some people might need a higher dose to reach optimal 25(OH) D levels. Another study investigated 41 candidate single nucleotide polymorphisms (SNPs) in vitamin D and calcium pathway genes among healthy non-Hispanic white participants and stated that the increase in [25(OH)D] attributable to vitamin D3 supplementation may vary according to common genetic differences in CYP2R1, 24-hydroxylase (CYP24A1), and vitamin D receptor (VDR) genes. There is evidence from three randomized controlled trials that indicate a strong association between genetic polymorphisms and levels of serum 25(OH) D in response to 40,000 IU vitamin D/week given for 6 months. However, there is a wide variation in the response of blood 25(OH) D to vitamin D supplementation that is associated with genetic variants in vitamin D metabolism.<SEP>Vitamin B12<SEP>Vitamin B12 is a coenzyme, cofactor, and essential component in vitamin B complex. It is essential for cardiac health and cognitive function. Deficiency of vitamin B12 can lead to deleterious consequences including macrocytic anemia, neuropsychiatric symptoms, cardiovascular diseases, and onset of different forms of cancer. The most common cause of vitamin B12 deficiency is loss of intrinsic factor (IF) as absorption depends on it. People who have bariatric surgery, short gut syndrome, long-term vegetarian, or vegan diets can potentially develop vitamin B12 deficiency. While vitamin B12 level can be normal at baseline, it is often found to be lower in individuals after bariatric surgery. Nutritional parameters were compared preoperatively and at similar periods postoperatively among patients undergoing malabsorptive procedures. Vitamin B12 abnormalities prior to surgery ranged from 3.2–8.3% to 24–25% at 1 year post-op. In a study of gastric bypass surgery subjects, vitamin B12 deficiency was observed in 33.3% at 2 years and in 27.2% at 3 years postoperatively.<SEP>Genetic variants may impact the proteins involved in vitamin B12 absorption, cellular uptake, and intracellular metabolism. Genetic influence for B12 levels is estimated to be 59% in a study using monozygotic and dizygotic twins and 27% in another study among Icelandic sibling pairs. Variants of the transcobalamin 1 (TCN1) gene (vitamin B12 binding protein, transcobalamin I (TCI)) have been associated with circulating B12 concentrations. Genetic variants of fucosyltransferase 2 (FUT2 gene) that codes for an enzyme in the vitamin B12 pathway are associated with B12 levels. Transcobalamin 2 (TCN2) gene is responsible for making a B12-binding protein called transcobolamin II (TC) that carries B12 from the intestine to blood and liver. Although TC represents approximately 10–20% of circulating B12, the most common variant of this gene among Caucasian populations has been associated with B12 levels. In a study among Irish men, having this SNP and homozygous CC genotype had lower vitamin B12 levels than those with GG genotype. This demonstrates that different genotypes of transcobalamin impact the distribution of vitamin B12 and shows an association between this genetic variant and B12 levels.<SEP>Vitamin B12 along with folate influences one-carbon metabolism. Cubulin (CUBN) is the intestinal (IF) and polymorphisms of this gene have been associated with chronic diseases in individuals with low B12 status. A study involving a Canadian population found that many SNPs in genes related to folate, B12, and homocysteine metabolism—CUBN, TCN1, TCN2, methylenetetrahydrofolate reductase (MTHFR), MUT (methylmalonyl coenzyme A mutase), and FUT2—are possibly correlated with B vitamin-related diseases. Genetic polymorphisms of MTHFR, MTR, MTRR, MMAA (methylmalonic aciduria (cobalamin deficiency) cb1A type), MMACHC (methylmalonic aciduria and homocystinuria, cblC type), and MUT have been analyzed. This research has failed to show an association between MTHFR gene polymorphisms and B12 concentrations. However, a study using a classic twin model found that common gene variants—MMAA, MMACHC, MTRR, and MUT—were significantly associated with B12 levels and could explain the variation in B12 levels, which might facilitate the prevention and treatment of B12 insufficiency/deficiency in individuals at a higher risk of associated diseases. A cross sectional study looking at 56 SNPs of the B12 pathway among an older female population and found TCN2 to be significantly associated with elevated serum methylmalonic acid (MMA) levels, a marker for available B12. When using MMA levels as a marker for B12, it is suggested that TCN2 gene variants may lead to decreased vitamin B12 availability. This review spotlights the complex nature of nutrigenomics and vitamin B12. Identifying these gene variants among people having bariatric surgery could contribute to a more personalized nutrition plan.<SEP>Folate<SEP>Folate plays a role in one-carbon metabolism, methylation and DNA synthesis, and methionine regeneration. Folate deficiency is associated with elevated homocysteine, cardiovascular diseases, neural tube defects, cleft lip and palate, late pregnancy complications, neurodegenerative and psychiatric disorders. Elevated homocysteine levels are a risk marker for dementia, Alzheimer’s disease, bone fractures, cancers, and cardiovascular diseases. Many studies show folate deficiency to be low due to food fortification in America. Although preoperative deficiencies are not alarming, prevalence of folate deficiency and elevated homocysteine have shown to persist or worsen after bariatric surgery despite supplementation. The prevalence of abnormalities 1 year after gastric bypass were higher compared to preoperative levels in 232 patients with elevated homocysteine as high as 29% and low RBC folate in 12% of 149 postoperative subjects. Another study found similar results among patients undergoing bariatric surgery with 13% having folate deficiency postoperatively.<SEP>Several studies have shown an association between SNPs related to folate metabolism, folate deficiency, and elevated homocysteine. A common genetic variant in MTHFR is known to influence blood folate and prevalent in 10% of the population worldwide. Steluti and colleagues studied polymorphism frequencies and differences in homocysteine concentrations even in the presence of folic acid fortification and found that homocysteine levels increased in those carrying genetic variants in folate metabolism, specifically in the MTHFR gene. The prevalence of variant MTHFR TT has been found in 25% of Americans of Hispanic origin, 10–15% among white Americans, and only 0–1% for African Americans. A review examining the nutritional deficiencies, bariatric surgery, and serum homocysteine levels found that the mutations of the MTHFR gene can be one of the reasons for persistent elevated serum homocysteine after surgery despite supplementation with B-group vitamins. Knowing the presence of genetic variants of folate metabolism would provide a critical personalized care to those that might benefit from the methylated form of folic acid to prevent elevated homocysteine levels.<SEP>Thiamine<SEP>Thiamine is essential for glucose, amino acid, and energy metabolisms. Deficiency of thiamine can cause complications including cardiovascular and neurological diseases, including Wernicke-Korsakoff syndrome. Preoperative thiamine deficiency is prevalent in about 29% of patients undergoing bariatric surgery. Studies have found that preexisting thiamine deficiency can be present in 15.5% and as high as 47% of patients; however, race plays a role showing Hispanic patients with the highest level of prevalence followed by African Americans (31%) and Caucasians (7%). Similarly, a retrospective study showed 33.6% of patients having thiamine deficiency pre-operatively, suggesting that people with obesity, especially those with many weight loss attempts, may have different needs to maintain adequate thiamine levels.<SEP>Mutations in thiamine transporter genes, SLC19A2 and SLC19A3, have been observed in cases of thiamine deficiency due to decreased absorption of thiamine that leads to neurological dysfunction. SLC35F3 is another thiamine transporter gene that plays a role in cardiac health and blood pressure. Genetic variants have been associated with thiamine deficiency as well as hypertension. Prevalence of mutations in these genes is largely unaccounted for despite recent advances in GWA studies. However, studies show that thiamine deficiency and cardiac dysfunction associated with these genetic variants are alleviated with thiamine supplementation. Literature reviews have shown that treatment for thiamine deficiency vary according to the genetic defect of thiamine metabolism and that supplementation results in adequate thiamine levels and improved clinical outcomes. The best responses to thiamine therapy were associated with early referral for genetic testing and early initiation of thiamine treatment. This evidence demonstrates that early diagnosis of these mutations can be beneficial. It may also implicate the hereditability of thiamine deficiency and that therapeutic doses of thiamine vary according to the genetic defect.<SEP>Iron<SEP>Iron is essential for metabolic processes like oxygen transport, deoxyribonucleic acid (DNA) synthesis, electron transport, as well as cellular functions can affect one’s well-being. In individuals with obesity, the chronic inflammatory state related to obesity might be a possible risk factor for iron deficiency, which is also called the anemia of inflammation. Studies have shown that the prevalence of iron deficiency in adults with obesity is remarkable, and a decrease in serum iron and transferrin saturation levels is inversely associated with an increase in body mass index. A study involving bariatric surgery candidates showed 86.2% of females and 80% of males to be iron deficient prior to surgery. A retrospective analysis of patients undergoing RYGB surgery showed that 43.9% were iron-deficient pre-operatively, which may be associated with higher complication rates as well as worsening of iron deficiency after surgery. These findings reaffirm the need to assess and possibly intervene to manage deficiency in bariatric surgery candidates preoperatively.<SEP>Considering the results of several GWAS, there is strong evidence of genetic regulation of iron metabolism, and mutations in transmembrane serine protease 6 (TMPRSS6) gene that encodes for an enzyme that regulates hepcidin involved in iron homeostasis, iron carrier transferrin (TF), and transferrin receptor-2 (TFR2) genes have been associated with iron deficiency. A GWAS concluded that identifying mutations in the TMPRSS6 gene has broad applications in understanding clinical disorders of iron metabolism, and polymorphisms in TMPRSS6 gene may contribute to iron deficiency anemia (IDA) in individuals even in absence of other predisposing factors for IDA. Studies have shown a common TMPRSS6 gene variant to be prevalent in 45% of the individuals without iron deficiency and clinically relevant inflammatory conditions and 36.5–41.7% in a group of non-pregnant women. TF and human hemochromatosis (HFE) genes are involved in genetic regulation of maintenance of iron homeostasis. Mutations in the HFE gene can lead to hereditary hemochromatosis, an iron overload disorder. These factors should be considered to possibly affect iron absorption and thus response to treatment.<SEP>TMPRSS6 mutations have been associated with refractoriness to oral iron and studies confirm the role of TMPRSS6 in predicting oral iron response. One study evaluated subjects with persistent IDA to poorly respond to oral iron, indicating that TMPRSS6 polymorphisms are more frequent in subjects with persistent IDA. Identifying mutations of these iron-related genes can help with providing personalized iron supplementation for a common deficiency post bariatric surgery.<SEP>Association Between Genetic Defects and Micronutrient Supplementation<SEP>The vitamin and mineral supplementation studies that focus on treating genetic disorders are mainly case studies. Supplementation studies for vitamin D-related genetic variants have been conducted in populations that are overweight and have obesity. Limited data is available on micronutrient supplementation according to genetic variants in bariatric surgery populations. Table 2 shows studies involving micronutrient supplementation according to genetic defect in diverse populations.<SEP>Bariatric Surgery, Genetic Testing, and Gene Expression Profiles<SEP>Genetic expression patterns can be a predictive tool for responsiveness to nutritional treatments. Some studies have indicated that surgery-induced weight loss was associated with remodeling of the epigenome that helps regulate metabolic gene expression. One study found that 1366 genes were differentially expressed after bariatric surgery and subsequent weight loss, which are associated with gene transcription and energy metabolism. Knowing the impact of bariatric surgery on the vitamin/mineral metabolic pathways can lead to successful prevention and treatment of micronutrient deficiencies. A study that specifically assessed the mRNA of genes within B12 degradation pathway after gastric bypass found that the intestine reprogrammed its genetic phenotype to compensate for the changes in B12 metabolism. The authors also found decreased expression of TCN1 but an increased production of CUBN, which reflects adaptive genetic reprogramming. However, research on the role of vitamin metabolism genes and their adaptation after bariatric surgery is scarce. We do know that healthy individuals and people with obesity have different gene expression profiles and bariatric surgery further modifies the epigenome. Genetic testing is a useful tool for applying personalized medicine in bariatric surgery patients as demonstrated by Bandstein et al. that showed presurgery vitamin D levels may impact the size of genotype effects of FTO rs9939609 on weight loss among gastric bypass surgery patients. Nutritional genomics may provide the path for precise nutrition recommendations to provide high-risk individuals with personalized treatment and to prevent micronutrient deficiencies.",
                  "Discussion<SEP>From the review, it is evident that the deficiencies of the studied micronutrients are influenced by genetic mutations. Postbariatric surgery, patients frequently have these deficiencies and knowledge of these mutations may have bearing on its management. Additional research is needed to establish this association. This review confirmed the scarcity of research that has been conducted in the area of bariatric surgery and micronutrient genetic variants, with only two articles being found in this search. This limitation should be considered when interpreting the findings in this discussion. Furthermore, the treatment regimen for those who have micronutrient genetic variants and undergoing bariatric surgery should be an area of future research.<SEP>Personalized dietary and supplement advice derived from genetic testing should be based on appropriately designed studies. Utility of data from GWAS in providing dietary advice is limited because it is not known what diet and supplement intakes are required to prevent and treat the deficiencies that might be caused by micronutrient genetic variants. Identifying how a genetic variant modifies the response to supplementation on the micronutrient status and possibly identify responders and non-responders will be required to understand this population and area of research. Genotype along with micronutrient blood levels would be the initial step in applying PN among bariatric surgery patients. Genetic marker is only one factor that influences improvements related to micronutrient status.<SEP>Future work should focus on genotyping for multiple variants in the micronutrient metabolic pathways and their additive and interactive effects to get a complete understanding of the influence of genetic factors on micronutrient metabolism. Then, utilizing genomic technology to understand this influence on the responses to micronutrient supplementation is also important. This would involve micronutrient status, genetic variations, and genetic interactions within metabolic pathways involving the micronutrient, its molecular targets, and environmental stressors.<SEP>Furthermore, studies should focus to understand the role of the gut microbiome and its influence on metabolism and physiology. The human gut microbiota (which has its own genome) can modulate signaling pathways and regulate gene expression. Diet, lifestyle, medications, and environmental exposure can increase inflammation within the gut, causing dysbiosis, which can contribute to chronic diseases and other illnesses. Interestingly, gut microbial contribution to vitamin metabolism has been recognized in whole-genome metagenomic studies, suggesting microbe-mediated vitamin metabolism. Pre- and probiotics as well as diet can alter the gut microbiome in a manner that improves human health. Investigating how the gut microbes can positively influence vitamin metabolism is warranted.<SEP>Techniques used for genetic testing will determine the cost. Methods being used in healthcare and research to identify genetic variations are known as next-generation sequencing (NGS). Sequencing costs have rapidly decreased, which has increased NGS applications in the clinical setting. There are 3 NGS approaches used: targeted gene panels (TGP), whole-exome sequencing (WES), and whole-genome sequencing (WGS). WGS implies the determination of the sequence of the entire genome of an individual; WES is a component of the genome; and TGP analyze specific mutations in a given set of genes or gene regions. Per sample costs of diagnostic NGS applications, that include the total cost (in euros) of processing and analysis are estimated at €333 (TGP), €792 (WES), and €1669 (WGS). Another study showed cost analysis for cancer diagnosis using NGS. From the pre-analytical phase to delivery of results, cost per patient for TGP ranged from €376 to €968. Costs for NGS have declined and will continue to decline with innovations in genome-sequencing technologies and strategies. In the area of bariatric surgery research, TGP could be utilized with the specific genes involved in micronutrient metabolic pathways, thus facilitating a more cost-effective and easy-to-interpret analysis.<SEP>Concerns and limitations involved in genetic testing include the rapid growth of direct-to-consumer (DTC) genetic testing services with non-evidenced based testing, potential ethical dilemmas, and applications to the broader community. Clinicians will be critical in providing genetic counseling regarding health decisions based on genomic information. Many issues must be addressed prior to genetic testing such as informed consent, sample and data storage, return of results, and privacy and confidentiality to minimize these risks that can be introduced with such testing. As mentioned previously, the majority of GWAS are conducted among Caucasian/European subjects. However, it is difficult to extrapolate from these studies to other populations. Sex differences also must be considered to improve the application of genetic tests to the broader community. Addressing these areas and concerns are critical to implement genome sequencing in clinical practice. This type of analysis is likely to be cost-effective, especially in specific populations such as patients with bariatric surgery.",
                  "30962",
                  "['OBWCCE', 'VITMIN', 'EATBTS', 'FOODIN']"
                ],
                [
                  "23",
                  "30157214",
                  "Introduction<SEP>Body Mass index (BMI) as well as other body mass and composition parameters are multifactorial traits that have genetic and environmental components. Today we know that post-training changes of body mass parameters are also controlled by a large number of genes with mild to moderate individual effects–the latest gene-based association analysis generated gene networks associated with the BMI-related sub-phenotypes, including 93 genes associated with BMI phenotypic variability. Many attempts looking for independent BMI associations lead to the suggestion that genes encoding the Peroxisome Proliferator-Activated Receptors (PPARs) are in the group of markers important for the effectiveness of weight-loss strategies. PPARs are nuclear transcriptional factors induced by lipid ligands. The main molecular activity of these receptors is control of the gene expression among PPRE (PPAR response elements) sequences in their regulatory regions. The systemic effect of PPAR induction is transcriptional stimulation (or less often repression) of specific genes, which results in changes of levels of the factors engaged in metabolism of energy substrates (like lipids and carbohydrates). As a consequence of the activity of such proteins, the availability of energy substrates decreases and the ligand-dependent induction of PPARs is reduced–in this way the feedback loop regulates the molecular factors and physiological interaction.<SEP>So far, three PPAR isoforms have been described, one of them is PPARδ, which has been proposed as a key regulator of energy metabolism due to its ability to enhance fatty acid catabolism, energy uncoupling, and insulin sensitivity in multiple tissues. In humans, PPARδ is encoded by the PPARD gene which is located on chromosome 6. The structure of the human PPARD gene was described by Skogsberg et al.. PPARD differs from the classical eukaryotic gene model: it has been reported to encompass 9 exons, of which exons 1–3, the 5'-end of exon 4 and the 3'-end of exon 9, are untranslated. In 2007, five new 5'-untranslated exons (designated exons 2a – 2e) in alternatively spliced transcripts of human PPARδ mRNA were identified.<SEP>Recent studies have indicated the potential roles for allelic variants in the PPARD gene in modulating its mRNA and protein levels which, in turn, affect PPARδ target genes. The studies of PPARD polymorphisms have been mainly focused on three single nucleotide polymorphisms (SNPs): rs2267668 with its locus in intron 3, rs2016520 with its locus in the 5’ untranslated region of exon 4, and rs1053049 with its locus in the 3’ untranslated region of exon 9. One of them, rs2016520, a T/C transition in nucleotide 15 of exon 4 located 87 base pairs before the start codon, is mostly studied due to its pivotal role in controlling lipid metabolism. The functional relevance of rs2016520 SNP comes from its influence on PPARD promoter, which in the presence of the rs2016520 C allele is more prone to bind the Sp-1 transcription factor–consequently, the transcription level of the PPARD gene rises. The impact of the rs1053049 polymorphic site is connected with control of the plasma low-density lipoprotein cholesterol (LDL-C) levels as well as with regulation of glucose disposal in the skeletal muscles. In studies of the rs2267668 SNP during a 9-month period, Tuebingen Lifestyle Intervention Program revealed that the changes of individual anaerobic threshold as well as insulin sensitivity are dependent on rs2267668 alleles. The rs2267668 G allele was recognized as one of the genetic factors responsible for decreasing aerobic physical fitness. Moreover, in vitro analyses showed an effect of lowering the skeletal muscle mitochondrial function in G allele carriers.<SEP>Taken together, the aforementioned findings suggest that PPARD polymorphisms are engaged in developing specific training-induced physiological reactions. In the present study, we have decided to correlate the presence of different PPARD alleles with post-training changes of body mass measurements as well as with biochemical parameters of energy metabolism. Looking for any associations, we have examined the genotype and alleles frequencies (described in PPARD rs2267668, rs2016520, and rs1053049 polymorphic sites) in female participants engaged in a 12-week training program. Body mass/composition and biochemical parameters were measured before and after the completion of a whole training program.",
                  "Materials and methods<SEP>Ethics statement<SEP>The procedures followed in the study were conducted ethically according to the principles of the World Medical Association Declaration of Helsinki and ethical standards in sport and exercise science research. The study was approved by the Ethics Committee of the Regional Medical Chamber in Szczecin (Approval number 09/KB/IV/2011). All participants were given a consent form and a written information sheet concerning the study, providing all pertinent information (purpose, procedures, risks, and benefits of participation). The experimental procedures were conducted in accordance with the set of guiding principles for reporting the results of genetic association studies defined by the Strengthening the Reporting of Genetic Association studies (STREGA) Statement.<SEP>Participants<SEP>162 Polish Caucasian women aged 21 ± 1 years (range 19–24) were included in the study. None of these individuals had engaged in regular physical activity in the previous 6 months. They had no history of any metabolic or cardiovascular diseases. Participants were nonsmokers and refrained from taking any medications or supplements known to affect metabolism. Participants were included in a dietary program and on the basis of an individual dietary plan, were asked to keep a balanced diet of approximately 2000 kcal/day. The participants were asked to keep a food diary every day. Weekly consultations were held on which the quality and quantity of meals were analyzed and, if necessary, minor adjustments were made.<SEP>Body composition measurements<SEP>All participants were measured for selected body mass and body composition variables before and after the completion of a 12-week training period. Body mass and body composition were assessed using the bioimpedance method as described by Leońska-Duniec et al..<SEP>Biochemical and hematological analyses<SEP>Fasting blood samples were obtained in the morning from the elbow vein before the start of the aerobic fitness training program and repeated at the 12th week of this training program (after the 36th training unit). The analyses were performed immediately after the blood collection, as it described earlier.<SEP>Training phase<SEP>The training stage was preceded by a week-long familiarization stage, when the examined women exercised 3 times a week for 30 minutes, at an intensity of about 50% of their maximum heart rate (HRmax). After the week-long familiarization stage, the proper training started. Each training unit consisted of a warm-up routine (10 minutes), the main aerobic routine (43 minutes), and stretching and breathing exercises (7 minutes). The main aerobic routine was a combination of two alternating styles—low and high impact, as described by Leońska-Duniec et al..<SEP>Genetic analyses<SEP>DNA was extracted from the buccal cells using a GenElute Mammalian Genomic DNA Miniprep Kit (Sigma, Germany) according to the manufacturer’s protocol. All samples were genotyped in duplicate using an allelic discrimination assay on a C1000 Touch Thermal Cycler (Bio-Rad, Germany) instrument with TaqMan® probes. To discriminate PPARD rs2267668, rs2016520, and rs1053049 alleles, TaqMan® Pre-Designed SNP Genotyping Assays were used (Applied Biosystems, USA) (assay ID: C__15872729_10, C___8851952_30, and C___8851955_60, respectively), including primers and fluorescently labelled (FAM and VIC) MGBTM probes to detect alleles.<SEP>Statistical analyses<SEP>Allele frequencies were determined by gene counting. An 𝜒2 test was used to test the Hardy-Weinberg equilibrium. To test the influence of PPARD rs2267668, rs2016520, and rs1053049 polymorphisms on training response, the mixed 2 × 2 ANOVA with one between-subject factor (genotype) and one within-subject factor (time: before training versus after training) was employed. Additionally, normality Kolmogorov-Smirnov test was used to check for data normality, and post hocTukey test was applied when interaction was significant and was used to perform pair-wise comparisons. Haplotype analysis was conducted with R (https://cran-r.project.org, version 3.1.0) using haplo.stats package and haplo.glm regression function. Percentage change over training was used as the dependent variable, while the PPARD haplotypes were used as the independent variables. The level of statistical significance was set at p<0.05.",
                  "Results<SEP>PPARD genotypes described for rs2267668, rs2016520, and rs1053049 conformed to Hardy-Weinberg equilibrium (p>0.05 in all groups, tested separately). The genotyping error was assessed as 1%, while the call rate (the proportion of samples in which the genotyping provided unambiguous reading) exceeded 95%.<SEP>To examine the hypothesis that the PPARD rs2267668, rs2016520, and rs1053049 polymorphisms modulate training response, we conducted a mixed 2 × 2 ANOVA for eleven dependent variables (Tables 1–3). There was no main effect of PPARD rs2267668 genotypes on dependent variables, although two significant genotype × training interactions, for Chol and triglycerides (TGL) and five significant main effects of training: for fat mass (p = 0.014), FFM (p<0.001), Chol (p = 0.008), HDL-C (p< 0.001), and glucose (p = 0.006) were observed. Carriers of the rs2267668 G allele exhibited a 4.6% decrease in Chol in the course of training (173±24 vs 165±26, p = 0.018 [LSD, Least Significant Difference]), compared with elevation in the AA homozygotes (169±25 vs 170±28, p = 0.633 [LSD]). Similarly, there was a decrease, although not significant, in TGL over the period of training (87±29 vs 77±30, p = 0.075). In addition, TGL rose significantly in the AA homozygotes (78±33 vs 85±37, p = 0.009) (Table 1).<SEP>Likewise, there was no main effect of PPARD rs2016520 genotypes on dependent variables, however, we observed four significant genotype × training interactions (for fat mass, FFM, Chol, and TGL), as well as the five significant main effects of training: for fat mass (p = 0.011), FFM (p<0.001), Chol (p = 0.010), HDL-C (p = 0.001), and glucose (p = 0.011). Carriers of the rs2016520 C allele demonstrated a 4.6% decrease in Chol in the course of the training (174±24 vs 167±28, p = 0.032 [LSD]), compared with an elevation in the TT homozygotes (168±25 vs 169±28, p = 0.681 [LSD]). As in rs2267668, there were opposite effects of training with regard to TGL. Specifically, carriers of the rs2016520 C allele demonstrated a decrease in TGL (88±29 vs 79±30, p = 0.068), while in the TT homozygotes, it increased significantly in response to training (77±33 vs 85±37, p = 0.006) (Table 2).<SEP>There was no main effect of PPARD rs1053049 genotypes on dependent variables. We found four significant main effects of training: for BMR (p = 0.034), FFM (p<0.001), Chol (p = 0.034), and HDL-C (p = 0.002). In addition, there were two significant genotype × training interactions (for fat mass and TGL) observed. TGL decreased (not significantly) in carriers of the rs1053049 C allele (81±27 vs 74±24, p = 0.124) and increased significantly in the TT homozygotes (79±34 vs 88±39, p = 0.005) (Table 3).<SEP>Analysis of association with haplotypes is presented in Table 4. Carriers of the PPARD (rs2267668/rs2016520/rs1053049) G/C/T haplotype had smaller decreases in body mass and BMI (by 2.5% per haplotype copy, p = 0.009 and by 2.3% per haplotype copy, p = 0.008, respectively) as compared with individuals homozygous for the reference A/T/T (frequency 81.7%) haplotype. Carriers of the G/C/C haplotype had significantly greater increase in FFM from training (by 1.0% per haplotype copy, p = 0.041) compared with reference haplotype. Conversely, the G/C/C haplotype was significantly associated with decreased Chol (by 4.3% per copy, p = 0.042) and TGL (by 19.7% per copy, p = 0.003) in response to training. The A/C/C haplotype was associated with an increase in HDL-C from training (by 21.5% per copy, p = 0.015) (Table 4).",
                  "Discussion<SEP>To address the question of whether the common PPARD polymorphisms influence selected body composition measurements as well as lipid and lipoprotein phenotypes, we chose to correlate the distribution of genotype, alleles, and haplotypes described in PPARD rs2267668, rs2016520, and rs1053049 polymorphisms in female participants engaged in a 12-week training program. The changes in body mass/composition and biochemical parameters measured before and after training have been analyzed in the context of carrying specific PPARD alleles or its haplotype combinations. The results provide some further evidence that PPARD plays a role in human lipid and carbohydrate metabolism and, in consequence, in weight control. Our major concern was whether the PPARD polymorphisms are involved in the modulation of lipid profile, and if so, to what extent.<SEP>When tested individually, our statistical analyses revealed that harboring a specific PPARD genotype may be associated with different post-training changes of measured biochemical parameters. With reference to PPARD rs2267668 genotypes, there were two significant genotype × training interactions (for Chol and TGL) in which a greater decrease of Chol over training was observed, in the rs2267668 G allele carriers and a significant increase of TGL levels in the AA homozygotes. Moreover, carriers of an rs2016520 PPARD C allele exhibited a significant decrease in Chol from training with accompanying decreases in TGL. Another finding in the current study is the overrepresentation of PPARD rs1053049 TT homozygotes in the group characterized by significantly higher post-training TGL levels.<SEP>Little is known about the molecular mechanism underlying PPARD polymorphisms influence on human metabolism and most information comes from animal studies. Increased PPARδ expression in adipose tissue in mice has a protective effect against elevated adiposity and serum lipid levels. PPARD transcription is activated in cardiac and skeletal muscles, as well as in adipocytes, providing energy from fatty acid catabolism during starvation and ensuring energy supplies for working muscles, while in a well-fed state the PPARδ level decreases. Animal studies suggest that PPARδ is involved in Chol metabolism, as treatment with PPARδ agonists has been shown to increase plasma HDL-C levels in db/db mice and in obese rhesus monkeys. Furthermore, it was proven that GW501516 (the potent selective PPARδ agonist) causes a significant decrease in fasting plasma TGL and LDL-C in obese rhesus monkeys. Moreover, it has been observed that one of the molecular effects of endurance training is activation of PPARδ with an accompanying development and maintenance of the “slow” (type I) phenotype of muscle fibers at the expense of “fast” fibers (type IIB). The intense endurance activities were also recognized as factors stimulating the PPARD expression. On this basis the conclusion that PPARD polymorphisms may be of relevance in shaping athletic performance and proper physiological reaction associated with physical activity seems to be reasonable. Polymorphic forms of the PPARD gene have also been associated with genetic proclivity toward obesity, however, subsequent attempts have failed to replicate this finding. Other reports have associated PPARD polymorphisms with the effectiveness of cardiovascular fitness, demonstrating the crucial role of PPARD gene variants in mitochondrial function and, in consequence, in weight control.<SEP>Our previous study on the same PPARD genetic markers conducted in four different subgroups of elite athletes and a group of sedentary controls revealed that PPARD rs2016520 and rs1053049 were individually associated with elite athletic performance. This was in accordance with several reports that examined the association between one of these PPARD polymorphisms and athletic performance and/or health-related phenotypes. In Russian athlete studies, it has been suggested that endurance athletes may benefit from carrying an rs2016520 C allele. However, in a group of Israeli endurance athletes as well as in Mount Olympus marathon runners these results were not confirmed. On the other hand, when tested together with the PPARGC1A Gly482/Gly482 genotype, the PPARD rs2016520 CC genotype seemed to be positively correlated with elite endurance athletes status. Taking into account the results from in vitro analyses, showing increased PPARD expression in the presence of an rs2016520 C allele, it may be assumed that (nevertheless inconsistent results of some studies) the rs2016520 SNP is a very promising candidate for further athlete performance association studies. The molecular evidence suggested that this SNP interferes with the binding of Sp-1 and results in the PPARD transcriptional activity changes. In vitro studies have shown higher transcriptional activity for the C allele in comparison with the T allele. Described modification of PPARδ protein function induced by the polymorphism in the PPARD would decrease the ability of fat oxidation upregulation in working skeletal muscle which affects physical performance. Indeed, the HERITAGE Family Study revealed that rs2016520 CC homozygotes showed the smallest increases in cardiorespiratory fitness measured by maximal oxygen consumption and a lower training response in maximal power output after 20 weeks of endurance training. Furthermore, CC homozygotes were characterized by higher LDL-C level and a propensity towards a higher risk of coronary heart disease (CHD) than subjects homogenous for the T allele although observations still exist in diverse populations which are inconsistent with these findings.<SEP>Regarding the subject of the next PPARD polymorphic site analyzed in the current study, it was confirmed that among other PPARD SNPs, rs1053049 is also associated with maintenance of glucose homoeostasis in skeletal muscles. Moreover, in Tuebingen Lifestyle Intervention Program it was observed that the polymorphism could affect diet and physical activity induced changes in human body composition. The presence of the rs1053049 C allele was associated with changes in overall adiposity, hepatic fat storage, and relative muscle mass. Finally, regarding the final PPARD gene SNP rs2267668 analyzed, little is known about the molecular mechanism underlying its influence on human metabolism and function. Nevertheless, it was shown that increases in anaerobic threshold and insulin sensitivity after a long-term aerobic training program were expressed to a lesser degree in carriers of the rs2267668 G allele compared with AA homozygotes, suggesting that PPARD rs2267668 genotypes may influence the effectiveness of aerobic exercise training in increasing aerobic physical fitness. Moreover, the role of PPARD rs2267668 in the modulation of aerobic fitness was supported by the study of Thamer et al. revealing that the presence of the G allele in an individual’s genotype is associated with a lower increase in relative muscle volume and a smaller decrease in adipose tissue mass and hepatic fat storage.<SEP>Collectively with the result obtained in the current study, all these studies support our initial hypothesis and suggest that these variants play a role not only in athletic performance, but also in post-training body mass changes as well as modulation of glucose levels and lipid profile. Moreover, when the results obtained in the presented study were incorporated in the complex haplotype analysis, the novel finding was that carriers of the rare PPARD (rs2267668/rs2016520/rs1053049) G/C/T haplotype displayed smaller post-training effects, in terms of body mass decrease, in comparison with individuals homozygous for the most common A/T/T haplotype, suggesting that harboring this specific G/C/T haplotype is unfavorable for achieving the desired training-induced body mass measurement changes. On the other hand, the G/C/C haplotype (when compared with reference haplotype) was significantly associated with post-training increase in FFM as well as with lower levels of Chol and TGL observed in the blood of the participants in response to applied training. This observation constitutes the second important finding of the current study, implying that when specific training-induced biochemical changes are taken into account, some individuals may benefit from carrying the G/C/C haplotype. To the best of our knowledge, our research group is the first team to explore the association of PPARD rs2267668, rs2016520, and rs1053049 polymorphisms in haplotype combination with post-training body mass and biochemical parameter changes in physically active participants. Therefore, the current results cannot be discussed with direct comparisons to relevant studies from other authors. However, when considering the combined effects of the analyzed PPARD polymorphisms, our previous study conducted among athletes and inactive controls revealed that the PPARD A/C/C haplotype (rs2267668/rs2016520/rs1053049) was significantly underrepresented in athletes compared to controls, suggesting that harboring this specific haplotype is unfavorable for becoming an elite athlete.<SEP>The first limitation of almost every genetic association study is a proper number of participants in a study group–we are aware that in our case it could also be a problem and we see the need of replicating our results in another, preferably larger, population. The second issue is the question of whether the analyzed PPARD polymorphisms are true causative factors or perhaps only in linkage disequilibrium with variants directly engaged in developing a specific physical trait. Even if the analyzed PPARD polymorphisms actually influence the post-training changes observed in active participants, it should be underlined that PPARD diversity probably accounts for only a small portion of phenotypic variability, due to the polygenic character of the traits connected with body mass and biochemical parameters measured in our experiment.",
                  "22402",
                  "['OBWCCE']"
                ],
                [
                  "24",
                  "28644415",
                  "1. Introduction<SEP>Millions of people in both developed and developing countries are affected by cardiovascular diseases (CVDs), one of the world’s leading causes of morbidity and mortality. Obesity is known to increase the risk of CVD and a combination of decreased levels of physical activity and an increase in adverse eating behaviors contributes to the obesity pandemic. There is also clear evidence of individual variability in response, suggesting that genetic susceptibility may have an important contribution to individual risk.<SEP>Single nutrients or food components have been studied to understand their impact on the development of chronic diseases. Accordingly, individual dietary components have been associated with increased or decreased risk of diseases without consideration of the cumulative or synergistic effects of the consumption of multiple nutrients within a diet, a concept extensively discussed. An alternative method for estimating diet may be to measure global food supply, thus taking into account the potential synergistic effects of multiple components within the diet. One of the methods used to regroup foods that are consumed together involves factor analysis. This ‘a posteriori’ hypothesis-free derivation method uses observed/reported dietary data in order to extract dietary patterns. Despite an ongoing debate on the validity of memory-based dietary assessment methods (M-BMs; e.g., 24-h dietary recalls and food frequency questionnaires (FFQs)), especially concerning their use in the formulation of national dietary guidelines, dietary patterns have been demonstrated to be concurrently valid and reproducible in comparison to other M-BMs, and are associated with CVD mortality and risk factors such as diabetes, blood pressure, obesity and dyslipidemia. Multiple studies summarized in a meta-analysis and in systematic reviews identified the Prudent dietary pattern as a protective factor for CVD and reported an opposite relationship for the Western dietary pattern. The Prudent dietary pattern is mostly characterized by the consumption of vegetables, fruits, whole-grain products, fish and non-hydrogenated fats, whereas the Western dietary pattern is characterized by higher intakes of red meats, processed meats, refined grains, French fries and sweets/desserts.<SEP>Genetic variations in several genes were associated with macronutrient intakes such as protein, fat and carbohydrate. Expanding single nutrients or food components, food preferences and dietary patterns were shown to be influenced by genetic variations. Greater desirability for “unhealthy” food items was associated with gene variation in the dopamine-related COMT gene and the rare allele of the rs9939609 single nucleotide polymorphism (SNP) in the fat mass and obesity-associated (FTO) gene has been associated with food preference; carriers of the rare allele consumed more biscuits and pastry and less soft drinks compared with TT carriers.<SEP>In line with abovementioned association of Prudent and Western dietary patterns with CVD and CVD risk factors, potential contribution of genetic susceptibility to CVD risk, and taking into account the debate on the validity of M-BM combined with the lack of error-free, practical, and affordable method to assess whole dietary pattern data, our group previously demonstrated that gene expression profiles differed in individuals with high vs. low scores for both Prudent and Western dietary patterns, and that expression profiles may potentially modulate the risk of chronic diseases including CVD. The current study aimed to assess the association of SNPs with the reported Prudent and Western dietary patterns scores. We conducted unbiased genome-wide approach and identified reported dietary pattern-related genetic variations. Further assessment of SNPs from associations identified was carried out through gene expression level and in silico analyses, and suggested interplay of genes involved in the metabolic response to dietary patterns in the adoption of dietary patterns.",
                  "2. Materials and Methods<SEP>2.1. Subjects<SEP>One hundred and forty-one individuals were selected among the 210 participants who completed the Fatty Acid Sensor (FAS) study, primarily aiming to understand how genes and environment act together to define CVD risk profile. Individuals recruited in the FAS study had to be non-smokers and be free of any thyroid or metabolic disorders requiring treatment such as diabetes, hypertension, severe dyslipidemia, and coronary heart disease. A concurrently validated FFQ was administered by a registered dietician before omega-3 fatty acid supplementation. Dietary patterns were derived by factor analysis from dietary intakes reported in FFQ. Further details on FAS study participants and recruitment criteria were published elsewhere. This trial was registered at  as NCT01343342. The subset of 141 individuals was originally selected among the FAS study participants based on DNA material availability and response to an n-3 polyunsaturated fatty acid supplementation. The experimental protocol was approved by the Ethics Committees of Laval University Hospital Research Center and Laval University. The study was conducted in accordance with the Declaration of Helsinki and all participants provided written informed consent before their inclusion.<SEP>2.2. Anthropometric Measurements and Biochemical Profiling<SEP>Body weight (kg), height (m) and waist circumference (cm) were measured according to standardized methods. Resting blood pressure (mm Hg) was measured in triplicate after a 10-min rest in a sitting position, phases I and V of Korotkoff sounds being respectively used for systolic (SBP) and diastolic (DBP) blood pressures. Blood samples were collected prior the supplementation period from an antecubital vein into Vacutainer tubes (Becton, Dickinson and Company, Franklin Lakes, NJ, USA) containing ethylenediaminetetraacetic acid after a 12-h overnight fast and 48-h alcohol abstinence. Blood buffy coat and plasma were separated by centrifugation. Plasma total cholesterol (total-C, mmol/L) and triglyceride (TG, mmol/L) concentrations were measured using enzymatic assays on an Olympus AU400e analyzer (Olympus America Inc., Melville, NY, USA). The high-density lipoprotein cholesterol (HDL-C; mmol/L) fraction was obtained after precipitation of very low-density lipoprotein cholesterol and low-density lipoprotein cholesterol (LDL-C) particles. LDL-C (mmol/L) was calculated with the Friedewald formula. Fasting insulinemia (pmol/L) was measured by radioimmunoassay with polyethylene glycol separation and fasting glucose concentrations (mmol/L) were enzymatically measured.<SEP>2.3. Dietary Assessment and Food Pattern Derivation<SEP>Habitual dietary intake for the month preceding the study was determined by a 91-item FFQ including 27 items with 1 to 3 sub-questions and specifically based on food habits of Quebecers. This FFQ was previously shown to be reproducible and concurrently valid based on comparisons with a 3-day dietary record. Participants had to answer to each question during a face-to-face interview with a registered dietician and were asked to report how often they consumed each type of food: daily, weekly, monthly or none at all during the last month. Examples of portion size were provided to ensure that each participant estimated correctly the proportion eaten. Information was compiled and the Nutrition Data System for Research software version 4.03 with Nutrient Database v2011 (Nutrition Coordination Center, University of Minnesota, Minneapolis, MN, USA) was used to analyze FFQ data. This database includes more than 16,000 food items with complete nutritional values for 112 nutrients. Similar food items from the FFQ were grouped, as previously described, and based on similarity of nutrient profiles, culinary usage and groups used in other studies. Twenty-seven food groups were then formed and used for factor analyses to generate reported dietary patterns. The FACTOR procedure from Statistical Analysis Software (SAS) was used to derive factors from all participants considering eigenvalue >1, values at Scree test and interpretability to determine the number of factors to retain. Briefly, two main reported dietary patterns were derived. These patterns were similar to Prudent and Western dietary patterns from the literature. Each individual was given a score for both reported dietary patterns. The SCORE procedure of SAS was used to calculate scores from the sum of food groups multiplied by their respective factor loading. These scores reflect the degree of each participant´s reported dietary intake conformance to a dietary pattern. Further details on reported dietary assessment, food grouping, food pattern derivation and factor loadings were provided elsewhere.<SEP>2.4. Genome-Wide Genotyping and Quality Control<SEP>DNA was isolated from blood buffy coats using the GenElute™ Blood Genomic DNA kit (Sigma, St. Louis, MO, USA). Quantification and verification of DNA quality were conducted via NanoDrop spectrophotometer (Thermo Scientific, Wilmington, DE, USA) and PicoGreen DNA methods. Illumina HumanOmni-5-Quad BeadChip® (Illumina Inc., San Diego, CA, USA) were used to genotype more than 4,300,000 SNPs at the genome-wide level in the 141 individuals. Samples were tested for call rate (>95%) and gender mismatch based on genotyping data. All 141 samples were used in further analysis. Genotyping arrays were processed at the McGill University/Génome Québec Innovation Center (Montreal, QC, Canada) according to manufacturer’s recommendations. SNP allele frequencies and tests for Hardy–Weinberg equilibrium (HWE) were performed using PLINK (version 1.07). SNP quality control was conducted and SNPs failing one of the criteria were excluded from analyses. Specifically, SNPs with insufficient call rate (<95%) or genotype distribution deviating from Hardy–Weinberg equilibrium (p values < 1.87 × 10−8) were excluded. In addition, monomorphic (non-variable) SNPs or with a minor allele frequency (MAF) < 0.01 were removed from analyses. Thus, a total of 1,632,526 SNPs were excluded, leaving 2,668,805 SNPs for statistical analyses.<SEP>2.5. Gene Expression Analyses<SEP>Pre-supplementation gene expression data were retrieved from previously published data for 30 of the 141 individuals. Briefly, gene expression profiling was performed on RNA extracted from peripheral blood mononuclear cells using the Illumina Human-6 v3 Expression BeadChip and carried out at the McGill University/Génome Québec Innovation Center (Montreal, QC, Canada). The microarray data re-analysis was performed using the FlexArray software and the Lumi algorithm. Robust multiarray average background adjustment was applied followed by log2 variance stabilization and quartile normalization. Transcripts were considered as expressed if they were detected in 25% of the samples.<SEP>2.6. Functional Analyses<SEP>Potential impacts of reported dietary patterns associated-SNPs herein identified on amino acid (aa) sequence and at protein level were analyzed using Variant Effect Predictor (VEP). Potential impacts of these SNPs on transcription factor (TF) binding sites and prediction of TF binding affinities based on DNA sequences were conducted using TRAP an online tool comparing SNP surrounding sequences with known TF recognition sequences. TRAP has the capacity to identify TF binding sites among a SNP-surrounding sequence and to estimate TF affinity to the common and rare alleles. It also offers the possibility to identify overrepresented/enriched TFs among a group of sequences submitted, thus highlighting potential disruption of global regulators of biological mechanisms and providing biological insights for the associations identified. Sequences overlapping SNPs of interest (30 bp upstream and downstream) were submitted for analysis as input sequences. The Transfac vertebrates 2010.1 database was used as TF matrix file and human promoter sequences were introduced as background model.<SEP>2.7. Statistical Analysis<SEP>Clinical data were expressed as mean ± standard deviation for the full cohort and according to sex. Differences in clinical data between men and women were tested using Student’s t-test for continuous variables and Chi-square test for categorical variables. The general linear model (type III sum of squares) with adjustments for the effects of age, sex and body mass index (BMI) was used to test the associations of SNPs with CVD risk factors (fasting plasma lipids, glucose, insulin, SBP and DBP) as well as the associations of prudent and Western reported dietary pattern scores with these CVD risk factors. Transformations were applied for TG (logarithmic transformation; log10) and insulin levels (negative inverse transformation; 1/(-1*X)) to meet the criteria for normality. Partial Pearson correlations were computed to assess the relation between reported dietary pattern scores and associated CVD risk factors. Associations between SNPs and scores for prudent and Western reported dietary patterns were tested under linear regression using PLINK including age, sex and BMI as covariates. Nominal genome-wide association threshold of p < 1.0 × 10−5 was used to identify SNPs associated to reported dietary patterns. This significance threshold was used in order to avoid discounting true positive association based on the fact that statistical tests in genome-wide association studies (GWASs) are not independent due to linkage disequilibrium (LD) between SNPs and therefore the traditional method to adjust significance thresholds for multiple testing overcorrects when used in GWASs. To evaluate the contribution of SNPs to the variance of reported dietary pattern scores, stepwise regression analysis was conducted. Differences in gene expression levels between genotype groups for reported dietary pattern-associated SNPs were tested using analysis of variance (general linear model, type III sum of squares) with adjustments for the effects of age, sex and BMI. LD (r2) between SNPs demonstrating significant associations was calculated from our data and from the 1000 Genomes Project phase 1v3 data using Haploview and LD calculator (), respectively. SAS software version 9.3 (SAS Institute Inc., Cary, NC, USA) was used to test for differences in clinical data, associations and correlation of reported dietary pattern scores with CVD risk factors, and differences in gene expression levels according to genotype groups.",
                  "3. Results<SEP>3.1. Subjects’ Description<SEP>The current study included 141 individuals from a previously described supplementation study aimed at assessing gene–environment interactions on CVD risk profile. Individuals included here were overweight, middle-aged men and women (68 men and 73 women; Table 1). Men and women had similar BMI, while men had higher SBP (p < 0.0001) and lower HDL-C levels than women (p < 0.0001). Comparing reported dietary pattern scores derived from dietary intakes reported for the month preceding the study, women were characterized by higher Prudent and lower Western scores than men (p = 0.04 and 0.01, respectively). When categorizing individuals with high (>0) vs. low (<0) scores for both reported dietary patterns, men were more prone to showing a high score for Western reported dietary pattern (p = 0.0006) while no difference between sex was identified for the Prudent reported dietary pattern score (p = 0.11).<SEP>3.2. Dietary Scores and CVD Risk Factors<SEP>Reported dietary pattern, characterized by high intakes of vegetables, fruits, whole grain products, non-hydrogenated fats for the Prudent and by high intakes of refined grain products, desserts, sweets and processed meats for the Western were tested for associations with CVD risk factors. Although limited by our sample size, the respective SNP frequency and their potential effect size, assessment of associations of Prudent score with CVD risk factors using correlation analysis revealed that DBP (r = −0.259, p = 0.002) and fasting insulin levels (r = −0.282, p = 0.0008), both showed inverse correlation with the Prudent score following adjustments for age, sex and BMI.<SEP>3.3. Association between SNPs and Reported Dietary Patterns<SEP>Associations were tested between 2,668,805 SNPs and each reported dietary pattern including age, sex and BMI as covariates. A total of 78 and 27 SNPs was associated with the Prudent and Western reported dietary pattern scores, respectively (p < 1 × 10−5; Figure 1, Tables S1 and S2). Associations identified were unique; none of the SNPs showed an association with both Prudent and Western scores. Low LD was generally observed in our study sample between Prudent-associated SNPs considering SNPs on the same chromosome, with few exceptions of large regions on chromosomes 2 (5 SNPs; 250 kb), 19 (3 SNPs; 118 kb) and 20 (12 SNPs; 476 kb) demonstrating strong LD (r2 ≥ 0.8). LD calculation from the 1000 Genomes Project data revealed moderate LD (r2 ≥ 0.6) between SNPs located within these regions. No such large region with strong LD was observed between SNPs associated with Western score with a mean LD of 0.23 in the present study sample.<SEP>SNPs associated with Prudent reported dietary pattern score were mainly located in gene regions, with 44 of the 78 Prudent score-associated-SNPs being located in gene regions. Most of these SNPs were intronic, while 5 were exonic, one was located in promoter and another in the 3’near gene region. Prudent-associated intergenic SNP rs13042507 is located near the CTCFL gene previously associated with type 2 diabetes (T2D). SNPs annotated to genes previously associated to obesity traits (LINGO2, NELL1) and neurological disorders (schizophrenia (ACSM1, KIF26B, NALCN)), and alcohol and nicotine dependence (LINGO, SH3BP5) were found among Prudent reported dietary pattern score associated-SNPs. SNPs associated to Western reported dietary pattern score were mostly intergenic; 19 of the 27 significant SNPs being intergenic while 7 were intronic and another was located in 3′ near gene region. SNPs from genes associated with alcohol dependence (ESR1) and obesity traits (RGS7, NRG3 and ESR1) were observed among Western reported dietary pattern score-associated SNPs.<SEP>To get further insights on the contribution of SNPs in variability of reported dietary patterns, and to identify potential leading SNPs for regions demonstrating multiple significant associations, stepwise regression was performed from Prudent- and Western-associated SNPs. Among Prudent-associated SNPs, 14 SNPs contributed to explaining 76.2% of the Prudent reported dietary pattern score variability, while sex and BMI explained 2.0% and 1.0% of variability, respectively. From the 27 Western-associated SNPs, 9 explained 63.6% of variability in the Western reported dietary pattern score while confounding factors (age, sex, BMI) did not seem to contribute to variability. Potential leading SNPs revealed by stepwise regression analysis are highlighted in Tables S1 and S2.<SEP>3.4. Impact of SNPs on CVD Risk Factors<SEP>In order to test the potential implication of reported dietary pattern-associated SNPs in the associations between reported dietary patterns and CVD risk factors, we further tested reported dietary pattern-associated SNPs for associations with CVD risk factors. In line with associations identified here between Prudent reported dietary pattern scores and CVD risk factors (DBP and insulin), a total of three significant associations were identified with insulin levels (Table 2). Among these, the rs6499924 SNP located within CNGB1, showed the most significant association with insulin levels (p = 0.0005). Significant associations between SNPs located in the gluconeogenesis-regulating PCK1 gene region and fasting glucose levels were also found although the Prudent reported dietary pattern score was not associated with fasting glucose levels in our previous analysis. Regarding Western-associated SNPs, five significant associations were identified between Western reported dietary pattern score-associated SNPs and total-C, including SNPs located within or near RGS7, TET2, ARID1B and PFKFB3.<SEP>3.5. Impact of SNPs on Gene Expression Level<SEP>To assess the physiological impact of reported dietary pattern-associated SNPs and to provide potential molecular mechanisms underlying associations identified, we tested the association of SNPs with gene expression levels using gene expression data retrieved from a previous study conducted on 30 individuals from our study sample (Table S3). Corresponding gene expression data were obtained for SNPs located in the gene region while expression levels of the nearest gene were retrieved for intergenic SNPs. Among genes annotated to diet associated-SNPs, 55 were found on gene expression array and 21 were detected in peripheral blood mononuclear cells. Despite few of the SNPs tested being associated with gene expression levels in this small study sample of 30 individuals, two intergenic SNPs associated with the Prudent reported dietary pattern (rs1454469, rs976145) were associated with expression levels (p = 0.02 for both) of BCKDHB (NM_183050). Rare allele carriers of these SNPs had higher expression levels (Figure 2A,B). These two SNPs demonstrated strong LD (r2 = 1.0) in our sample as well as in data from the 1000 Genomes Project. Testing Western reported dietary pattern-associated SNPs for association with expression levels, rs113152482 rare allele carriers showed higher PFKFB3 (NM_004566) expression levels following adjustments for the effect of age, sex and BMI (p = 0.0004; Figure 2C). It is interesting to note that this SNP was highlighted by stepwise regression analysis as it contributed to 1.3% of the variability of the Western reported dietary pattern score.<SEP>3.6. Functional Analysis of SNPs<SEP>To provide further mechanistic insights for associations identified between SNPs, Prudent and Western reported dietary pattern scores, CVD risk factors and expression levels, we conducted TF analysis from SNP-surrounding sequences. Considering all Prudent-associated SNPs, FOXM1, glucocorticoid receptor (GR), CEBP and CEBPB were found among the most overrepresented TF binding sites. STAT family members and HMGA1 TFs were overrepresented from SNP-surrounding sequences for SNP associated with either Prudent or Western reported dietary patterns (Tables S4 and S5). IRF8 and PDX1 TFs were also overrepresented among surrounding sequences from SNPs associated with either the Prudent or the Western reported dietary pattern. Focusing on SNPs associated to CVD risk factors identified here, SNP rs6499924 associated with fasting insulin levels showed creation of potential GABP-alpha and ATF5 binding sites in the presence of the rare allele. Among glucose level-associated SNPs located in the PCK1 gene region, rs6070157 resulted in aa change that was predicted to be tolerated or benign. For SNPs associated with gene expression, the rs976145 SNP associated to BCKDHB gene expression levels showed creation of HIF2A binding site while the presence of the rare allele of rs1454469 SNP, also located within the BCKDHB gene region, was predicted to disrupt IRX2 and IRX3 binding sites and to create a MEF2 binding site. Western reported dietary pattern-associated rs113152482 SNP, found to be associated with PFKFB3 gene expression, showed the creation of a potential NFAT1 binding site.",
                  "4. Discussion<SEP>Using factor analysis from reported dietary intakes obtained from a concurrently validated FFQ, we first derived dietary patterns corresponding to Prudent and Western dietary patterns previously reported to be associated respectively with protective and deleterious effects on CVD. We thereafter tested associations between SNPs and reported dietary pattern scores using a nominal threshold of p < 1.0 × 10−5. This genome-wide association threshold was used to account the non-independency of statistical tests conducted and combined with functional analyses to provide potential mechanistic insights for the associations identified. Although not reaching the conventional p < 5.0 × 10−8 GWAS significance threshold or Bonferroni corrected threshold, it provides clues for the discovery of biologically relevant associations. Identification of associations between reported dietary pattern-associated SNPs, CVD risk factors and gene expression levels argued for such biological importance of SNPs identified. Nonetheless, the most significant association observed here, between rs13212846 and the Western score (p = 4.16 × 10−8), reached a conventional p < 5.0 × 10−8 GWAS significance threshold. This SNP is located ~285 kb upstream the DEFB112 gene encoding an antimicrobial and cytotoxic peptides made by neutrophils. Another SNP located upstream of the DEFB112 gene (~259 kb) was previously associated with BMI. However, very low LD is observed between the BMI-associated rs17665162 SNP and the Western score-associated rs13212846 SNP herein identified. Globally, low LD observed between reported dietary pattern-associated SNPs and subsequent regression analysis demonstrated that a limited number of SNPs explains a large proportion of the variability in reported dietary pattern scores. These results suggest that: (1) some of the SNPs identified herein may act under an additive model; and (2) some other SNPs may act through common functional mechanisms with major SNPs potentially alleviating the impact of certain SNPs in common biological mechanism.<SEP>In line with the relationship between dietary patterns and CVD risk factors, the current study identified the rs13042507 SNP, near the CTCFL gene previously associated with T2D. This SNP, herein associated with the Prudent reported dietary pattern, shows very low LD (0.008; 1000 Genomes Project data) with the rs328506 SNP associated with decreased risk of T2D, thus not allowing a potential biological link between Prudent dietary pattern and T2D-associated risk previously reported. Nonetheless, association of reported dietary pattern-associated SNPs with CVD risk factors were also identified in the current study. Three SNPs (rs73180793, rs11552145, rs6070157) located within the gluconeogenesis-regulating PCK1 gene region were found to be associated with fasting glucose levels. Although subjects recruited had to be non-diabetics, these associations are coherent with a potential association between PCK1 SNPs and T2D, and between Prudent-like dietary patterns and decreased risk of T2D. Testing Western reported dietary pattern -associated SNPs with CVD risk factors, the most significant association found involved the rs1348307 SNP located within the long intergenic non-protein coding RNA 706 (LINC00706) and fasting insulin levels (p = 0.0008). Although association between Western reported dietary pattern score and insulin levels was not observed in our cohort of overweight/obese men and women, such association of Western reported dietary pattern-associated SNPs with insulin level is coherent with a correlation of the Western score with insulin levels, as previously reported in men.<SEP>Mechanistic insights for the associations identified are provided herein through analysis of gene expression levels in blood and TF analysis. Increased expression levels of the BCKDHB gene (NM_183050) were observed in the presence of rare allele of Prudent reported dietary pattern-associated SNPs rs1454469 and rs976145, both SNPs demonstrating perfect LD in our study sample. Mutations in the BCKDHB gene are known to be responsible for the maple syrup urine disease (Online Mendelian Inheritance in Man #248600) characterized by mental and physical retardation, feeding problems, and a maple syrup odor of the urine. Specifically, the presence of SNP rs1454469 was predicted to create an MEF2 binding site. In Caenorhabditis elegans chemosensory neurons, MEF2 TF was recently found to be involved in sensory neuron–gut interaction, linking feeding state conditions to the regulation of chemoreceptor genes via insulin signaling. An association between the Western diet associated-SNP rs113152482 and gene expression of PFKFB3 in blood was also identified. PFKFB3 encodes inducible 6-phosphofructo-2-kinase and is expressed in the brain. It was shown to act as an essential glucosensor in hypothalamic neurons, linking glycolysis, AMP-activated protein kinase signaling and neuropeptide expression in mouse. The rs113152482 SNP, highlighted by stepwise regression and explaining 1.3% of Western score variability, was predicted to disrupt the NFAT1 binding site. NFAT signaling plays critical roles in the development of multiple organ systems, including pancreas and nervous system, and was reported to play a role in glucose homeostasis in pancreatic β-cells cellular models.<SEP>Having a global look at TF overrepresentation from surrounding sequences of reported dietary pattern-associated SNPs, overrepresentation of FOXM1 and GR TF were observed from Prudent reported dietary pattern-associated SNPs. FOXM1 is involved in cell proliferation, is necessary for the maintenance of adult beta-cell mass, beta-cell proliferation and glucose homeostasis, and was shown to be up-regulated in obesity. Glucocorticoids (GCs) are known to mobilize the endocannabinoid system which is essential for negative feedback regulation of the hypothalamic–pituitary–adrenal axis. In addition, a recent study using Cushing’s syndrome patients as a unique model of chronic GCs exposure demonstrated a negative correlation of urine cortisone with food-related choice thus implying a potential role of GR in food-choice behavior. STAT family members and PDX1 TF were found to be overrepresented from SNP-surrounding sequences from both Prudent and Western reported dietary pattern-associated SNPs (Tables S5 and S6). STAT TFs were shown to be involved in energy homeostasis through an activation of the JAK-STAT pathway by leptin and their role in leptin-mediated satiety. Specifically, STAT5 TF herein overrepresented is recruited by many hormones and cytokines that regulate food intake whereas the PDX1 TF is involved in pancreatic development and glucose metabolism.<SEP>Results presented here tend to highlight a potential involvement of obesity-related and glucose metabolism genes in the adoption of dietary patterns concordant with a potential involvement of obesity genes in nutrient-specific food preference proposed following the analysis of obesity-associated loci revealed through genome-wide association study. Notably, variants associated with body weight and BMI were previously reported to be associated with appetite, energy intake and eating behaviors, and several obesity genes were reported to be expressed in the hypothalamus, a center for energy balance and regulation of food intake. Specifically, interplay exists between food-induced brain responses and eating behaviour, and hypothalamus is a brain area specifically involved in food reward thus potentially influencing food choice and the adoption of dietary patterns.<SEP>The current study used unbiased genome-wide approach to assess the genetics of the adoption of Prudent and Western reported dietary pattern scores. Results from the 91-items FFQ administered in the current study are based on reported data known to be biased by omissions, false memories, intentional misreporting and gross misestimation, and face-to-face interviews may have affected participants’ responses due to social desirability bias. While these biases cannot be measured in the current study, the use of a population-specific FFQ combined with an extensive database of food items with nutritional values available for 112 nutrients may partially alleviate the impact of self-reported nutritional assessment method on the derivation of reported dietary patterns. Despite subject to the imperfection of self-reported data and the ongoing debate on the validity of the memory-based dietary assessment methods, the concurrent validity and reproducibility of the FFQ used here were previously reported using a home- and self-completed 3-day food record, a dietary assessment method subject to recall bias, thus arguing for concurrent validity of the FFQ administered although validation was not performed in the current study and actual dietary intakes were not measured. Interactions between genetic and dietary factors as well as the impact of developmental processes on CVD risk factors were not analyzed, the main objective of the study being to identify associations between SNPs and reported dietary patterns to provide novel potential targets and biological mechanisms for CVD prevention. Since differences in reported dietary pattern scores between men and women have been identified herein from reported dietary intakes, sex has been included as a covariate in genome-wide analyses. However, analyses have not been conducted separately in men and women. BMI was also included as a covariate in our analysis, suggesting that association identified are BMI independent. However, we acknowledge that other CVD risk-associated confounding factors, e.g., developmental programming and physical activity, were not taken into account for testing associations between reported dietary pattern-associated SNPs and CVD risk factors. Further generalization of conclusions at the population level merits further validation in general population, our cohort being composed of overweight individuals. An impact of SNPs on blood cell expression levels was observed here for a limited number SNPs. Nonetheless, we cannot rule out the possibility that they may exert their effect in other tissues.<SEP>Collectively, the association of SNPs with reported dietary pattern scores, CVD risk factors and expression levels argues for an impact of genetic variations on the determination of the adoption of Prudent and Western dietary patterns. Integration of association, expression and transcription factor data tends to reveal the involvement of obesity, glucose metabolism and neurological genes in the adoption of dietary patterns. As proposed herein, reported dietary pattern-associated SNPs may potentially act through an impact on glucose metabolism and food- and energy-sensing pathways.",
                  "34311",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "25",
                  "31970519",
                  "Introduction<SEP>Iron is an important component of the oxygen-binding proteins, such as hemoglobin and myoglobin. Whereas hemoglobin transports oxygen (via erythrocytes), myoglobin’s function is to store oxygen in working skeletal muscles and to facilitate its transport to mitochondria. Approximately 65% of iron is stored in hemoglobin (Wallace), thus there is a positive correlation between serum iron concentrations and hemoglobin (Ofojekwu et al.; Baart et al.). Iron can affect many physiological processes, and its deficiency is associated with fatigue, anemia, and decreased exercise performance (DellaValle; Abbaspour et al.). There is a balance between iron loss, iron absorption, and iron storage to maintain iron homeostasis (DellaValle; Wallace; Rubeor et al.). Endurance athletes have an increased risk for iron loss because of the insufficient dietary intake and training intensity, which leads to increased risk for suboptimal iron status (Hinton).<SEP>Serum iron measures and hematological parameters have significant heritability components. The heritability estimates are 23% for iron, 29–37% for ferritin, and 28% for transferrin saturation (Njajou et al.; McLaren et al.), and 84% for hemoglobin (Evans et al.). Genetic variation plays a significant role in interindividual differences in serum iron parameters. More specifically, previous studies have suggested that the missense mutations of the hemochromatosis (HFE) gene are associated with iron indexes (Burt et al.; Wallace). The proportion of variance explained by HFE gene mutations was reported to be 2.1% for serum iron level, 5.6% for ferritin, and 3.5% for transferrin saturation (Njajou et al.).<SEP>The HFE gene (full name—homeostatic iron regulator) is a protein coding gene located on chromosome 6. The protein regulates iron absorption by regulating the interaction of the transferrin receptor with transferrin. The HFE protein interacts with TFRC, the transferrin receptor, so its primary mode of action is through regulation of the iron storage hormone hepcidin. Individuals with one (C/G or H63D genotype) or two (G/G or D63D genotype) missense mutations of the H63D (also known as His63Asp or rs1799945 C/G) polymorphism, show higher circulating iron concentrations than people without mutations (Burt et al.). In the H63D carrier group, a positive correlation between iron and hemoglobin was noted (Barbara et al.). The H63D mutation is commonly found in European (17%) and American (12%) populations, and is rarer in East Asian (3%), South Asian (7%), and African (1%) populations.<SEP>The H63D mutation accounts for a mild form of hereditary hemochromatosis (HH), a condition with increased intestinal iron absorption which may lead to liver fibrosis and cirrhosis, hepatocellular carcinoma, diabetes mellitus, cardiomyopathy, and hypogonadotropic hypogonadism (Wallace). Given the importance of iron and hemoglobin in athletic performance, one might suggest that the HFE gene H63D may give some advantage in endurance sports. Indeed, Deugnier et al. have identified an increased frequency of the G allele in 77 elite French road male cyclists when compared to controls (24.7 vs 17.1%, P = 0.04). In accordance with this data, the frequency of the C/G genotype was significantly higher in 65 professional Spanish endurance athletes (50 road cyclists and 15 endurance runners) in comparison with controls (41.5 vs 24.6%, P = 0.01) (Chicharro et al.). Finally, Hermine et al. have found that the frequency of CG/GG genotypes was significantly higher in the group of French elite athletes compared to controls (38% vs 21.9%, P = 0.0019). Although these findings in West European populations support the hypothesis that the iron-increasing HFE rs1799945 G allele is favorable for endurance performance, replication studies in different ethnic groups using different designs are warranted. This approach leads to the exclusion of false-positive genetic associations (Eynon et al.; Zarebska et al.; Papadimitriou et al.; Yvert et al.; Guilherme et al.; Pickering et al.).<SEP>The aim of the study was to investigate the association between the HFE gene H63D polymorphism and endurance athlete status in Japanese and Russian populations, aerobic capacity, and to perform a meta-analysis using current findings and three previous studies.",
                  "Methods<SEP>Ethical approval<SEP>The study was approved by the Ethics Committee of the Physiological Section of the Russian National Committee for Biological Ethics, Ethics Committees of the Juntendo University and National Institutes of Biomedical Innovation, Health and Nutrition (Japan) and by the Institutional Research Board of Anti-Doping Laboratory Qatar (ADLQ) (F2014000009). Written informed consent was obtained from each participant. The study complied with the guidelines set out in the World Medical Association Declaration of Helsinki and ethical standards in sport and exercise science research. The experimental procedures were conducted in accordance with the set of guiding principles for reporting the results of genetic association studies defined by the Strengthening the reporting of genetic association studies (STREGA) statement.<SEP>Study participants<SEP>The study involved 315 international-level endurance athletes (255 Russian and 60 Japanese) and 809 healthy controls (405 Russian and 404 Japanese). The first group comprised 255 international-level Russian endurance athletes tested negative for doping substances and involved in biathlon, kayaking, cross-country skiing, cycling, rowing, running ≥ 800 m, speed skating ≥ 1.5 km, swimming ≥ 400 m, and triathlon. Controls were 405 healthy, unrelated citizens of Russia without any competitive sport experience. Of those, 46 male endurance athletes (middle-distance athletes (n = 31): rowers, kayakers, speed skaters; long-distance athletes (n = 15): biathletes and cross-country skiers) participated in the study of aerobic performance. The second group involved 60 Japanese international-level endurance athletes (800 m to marathon runners) tested negative for doping substances, including several world record holders and medallist in Olympic Games. Controls were (n = 406) healthy, unrelated citizens of Japanese.<SEP>Genotyping<SEP>DNA samples of the Russian cohorts were majorly genotyped using micro-array analysis, as described previously (Pickering et al.). In part, some DNA samples of Russian athletes and controls were genotyped for the HFE rs1799945 polymorphism with a TaqMan® SNP Genotyping Assay (Thermo Fisher Scientific Inc, Waltham, Massachusetts, USA) with a StepOne TM Real-Time PCR System (Thermo Fisher Scientific Inc, Waltham, Massachusetts, USA) or using PCR-restriction fragment length polymorphism (RFLP) method, according to a previously described method (Merryweather-Clarke et al.).<SEP>Japanese cohort: total DNA was extracted from saliva or venous blood using Oragene DNA Collection Kit (DNA genotek, Ontario, Canada) or QIAamp DNA blood Maxi Kit (QIAGEN, Hilden, Germany), respectively. Illumina® HumanOmniExpress Beadchip (Illumina Inc, Hayward, California, USA) were used for genotyping of more than 700,000 SNPs in athletes and controls. The genotype calls were performed with Illumina® GenomeStudio Software. Genotype data of the HFE rs1799945 polymorphism were obtained from the genotyping results of the Illumina® HumanOmniExpress Beadchip.<SEP>VO2max measurement<SEP>Maximal oxygen consumption rate (V̇O2max) in rowers was determined using an incremental test to exhaustion on a PM 3 rowing ergometer (Concept II, Morrisville, Vermont, USA). The initial workload was 150 W. The duration of exercise at each workload was 3 min, with a 30 s rest period between increments of 50 W. VO2 and VCO2 was determined breath by breath by a MetaMax 3B gas analysis system (Cortex, Leipzig, Germany) using an electro-chemical cell and non-dispersive infrared sensor, respectively; air flow was measured using a turbine transducer (Triple V). Two-point gas calibrations (first gas—15% O2, 5% CO2; second gas—ambient air) were performed daily. A one-point gas calibration with ambient air was performed before each test as well as a flow transducer calibration using a 3 L syringe (Hans Rudolph, Kansas City, USA). The criteria used to confirm a maximal test were a decrease in power of more than 30 W from the target power despite strong verbal encouragement and a respiratory exchange ratio greater than 1.1 before cessation of exercise. V̇O2max was recorded as the highest mean value observed over a 30 s period.<SEP>V̇O2max in kayakers was determined using an incremental test to exhaustion on a kayaking ergometer (Efremov, Moscow, Russia). The initial workload was 8 kg for men and 5 kg for women. The duration of exercise at each workload was 2 min, with a 30 s rest period between increments of 1 kg. V̇O2max was determined breath by breath using a MetaLyzer II gas analysis system (Cortex Biophysik, Leipzig, Germany). VO2max was recorded as the highest mean value observed over a 30 s period.<SEP>V̇O2max in speed skaters was determined using a ramp test to exhaustion on an electromagnetic cycle ergometer Ergoselect 200 K (Ergoline, Bitz, Germany). The initial workload was 60 W, the increment was 15 W/min, and the target cadence was 60–70 rpm. V̇O2max was determined breath by breath using a MetaMax 3B gas analysis system (Cortex Biophysik, Leipzig, Germany). The criteria used to confirm a maximal test were a decrease in cadence of less than 50 rpm despite strong verbal encouragement and a respiratory exchange ratio greater than 1.1 before cessation of exercise. V̇O2max was recorded as the highest mean value observed over a 30 s period.<SEP>V̇O2max in biathletes and cross-country skiers was determined using an incremental test to exhaustion on a treadmill HP Cosmos (h/p/cosmos sports & medical gmbh, Nussdorf, Germany). The initial speed was 7 km/h, the increment was 0.1 km/h every 10 s. V̇O2max was determined breath by breath using a MetaMax 3B-R2 gas analysis system (Cortex Biophysik, Leipzig, Germany). V̇O2max was recorded as the highest mean value observed over a 30 s period.<SEP>Selection of studies for the meta-analysis<SEP>Databases of PubMed, Web of Science, Science Direct and Google Scholar were searched for association studies as of July 19, 2019. The terms used were “HFE” and “athletes” restricted to English. The exclusion criteria were: (1) review; (2) non-English; (3) studies did not involve endurance athletes; (4) ethnically mixed group of athletes were analysed (given that allelic frequencies vary significantly across different ethnicities; for example, we did not include the study of Grealy et al. because the mixed group of athletes from North America, Europe, Oceania, South America, Asia, and Africa was studied); and (5) duplicates. The inclusion criteria were: (1) case–control study design evaluating the association between HFE gene H63D polymorphism and endurance athlete status; (2) sufficient genotype frequency data to calculate the odds ratios (ORs) and 95% confidence intervals (CIs) and (3) athletes and controls in studies comply with the Hardy–Weinberg equilibrium (HWE). Overall, seven articles published between 1998 and 2015 were identified of which three were found as eligible including a total number of 271 endurance athletes and 607 controls.<SEP>Statistical analysis<SEP>Genotype distribution and allele frequencies between athletes and controls were compared using χ2 tests. Differences in phenotype between groups were analysed using unpaired t tests. Data are presented as mean (standard deviation). Statistical analyses were conducted using GraphPad InStat software (GraphPad Software, Inc., California, USA) and PLINK software program (Purcell et al.). To perform the meta-analysis with obtained data and all published studies the Cochrane Review Manager (RevMan) (London, UK) version 5.3 was used. Random and fixed effect models were applied. Odds ratio with 95% confidence intervals (CI) was estimated using the Mantel–Haenszel method. The heterogeneity degree between the studies was assessed with the I2 statistics. P values < 0.05 were considered statistically significant.",
                  "Results<SEP>Case–control study<SEP>In Japanese and Russian groups of athletes and controls, the HFE gene rs1799945 polymorphism met Hardy–Weinberg expectations (P > 0.05 in both groups tested separately). The frequencies of the rs1799945 G allele were significantly higher in Russian (21.0 vs 13.2%; P = 0.0002) and Japanese (7.5 vs 2.5%; P = 0.0032) endurance athletes compared to ethnically matched controls (Table 1). Furthermore, the rs1799945 CG/GG genotypes were significantly over- represented in Russian (38.0 vs 24.9%; OR 1.85, P = 0.0003) and Japanese (13.3 vs 5.0%; OR 2.95, P = 0.011) endurance athletes compared to ethnically matched controls (Table 2). These results remained statistically significant after correction for multiple testing.<SEP>Meta-analysis<SEP>Multi-database literature search yielded three eligible studies involving endurance athletes that were genotyped for the HFE gene H63D polymorphism. These involved 77 French elite road cyclists and 254 controls (Deugnier et al.); 65 Spanish highly trained athletes (50 professional road cyclists and 15 Olympic class endurance runners) and 134 controls (sedentary men from Spain) (Chicharro et al.) and 129 French elite athletes (Nordic skiing, rowing, fighting) and 219 controls (Hermine et. al.). The genotypic frequencies for both the cases and the controls in all studies were in Hardy–Weinberg equilibrium.<SEP>The frequencies of the rs1799945 CG/GG genotypes were significantly higher in three groups of French and Spanish endurance athletes compared to controls (Table 2). Overall, five case–control studies (two current and three previous) including a total number of 586 endurance athletes and 1416 controls were used for the meta-analysis. The pooled OR for the CG/GG genotypes compared to the CC genotype was 1.95 (95% CI 1.57–2.43, P = 2.5 × 10–9 for the fixed effect model) and 1.96 (95% CI 1.58–2.45, P = 1.7 × 10–9 for the random effect model) (Fig. 1). No heterogeneity between studies (I2 = 0%; P = 0.83) was observed. These results indicate that the carriage of the HFE mutation (i.e. CG/GG genotypes) is strongly associated with endurance athlete status.<SEP>Aerobic study<SEP>We identified that the HFE gene rs1799945 G allele was significantly associated with increased V̇O2max in the whole group of Russian male endurance athletes (CC [n = 29]: 61.8 (6.1), CG/GG [n = 17]: 66.3 (7.8) ml/min/kg; P = 0.036), as well as in long-distance athletes only (CC [n = 8]: 68.1 (3.4), CG/GG [n = 7]: 73.0 (4.6) ml/min/kg; P = 0.038).",
                  "Discussion<SEP>This is the first study to demonstrate that H63D variation in the HFE is associated with elite endurance athlete status in Russian and Japanese populations. More specifically, we found that the frequencies of the iron-increasing genotypes (i.e. CG/GG) were significantly higher in Russian and Japanese elite endurance athletes compared to ethnically matched controls. We also confirmed the observation that the H63D mutation is commonly found in East Europeans (13.2%) and is rarer in East Asian (2.5%) populations. In addition, the meta-analysis using five cohorts (two French, Japanese, Spanish, and Russian) including a total number of 586 endurance athletes and 1416 controls showed significantly higher prevalence of the CG/GG genotypes in endurance athletes compared with controls.<SEP>The H63D polymorphism is functional given that the rare G allele has been shown to reduce the ability of the HFE protein to bind to its ligand, thereby preventing the inhibition of transferrin–TFRC binding and resulting in increased transport of iron into circulation and cells (Feder et al.). The hypothesis that the iron-increasing HFE rs1799945 G (63D) allele is favorable for endurance performance was confirmed in our functional study, where we identified that the G allele was associated with increased V̇O2max in Russian male endurance athletes. One might suggest that the favorable effect of the HFE G allele on aerobic capacity and ability to become an endurance athlete is mediated through its impact on hematological parameters, as was shown in the study of French endurance athletes (Hermine et al.). Furthermore, in the genome-wide association study (GWAS) of 173,480 European-ancestry participants, the HFE rs1799945 G allele was shown to be significantly (P < 5 × 10–8) associated with increased values of hematological parameters, such as hematocrit, mean corpuscular hemoglobin concentration, hemoglobin, and reticulocyte count (Astle et al.). Previous studies in athletes have also shown that variations in genes, which regulate hematological traits, are associated with aerobic capacity and endurance athlete status (Ahmetov et al.; Malczewska-Lenczowska et al.).<SEP>Our findings seem reasonable given the importance of iron metabolism for endurance athletes (Abbaspour et al.). The leading role of iron is to transport oxygen into the red blood cells and tissues, and it does so mainly through hemoglobin. Furthermore, iron is present in myoglobin and cytochromes of skeletal muscle mostly in oxidative (slow-twitch) muscle fibers. The normal level of iron is crucial to maintain redox balance in muscle and produce mitochondrial energy production, which are primary factors determining exercise performance (Buratti et al.). Iron deficiency without anemia and/or sports anemia are a common issue in athletic populations (at 15–35% of female and 3–11% of male athletes) (Fallon; Malczewska et al.; Parks et al.) with higher frequency in endurance athletes, e.g., distance runners and triathletes (Rietjens et al.; Lukaski; Sinclair and Hinton), and physically active individuals compared with sedentary controls (Milic et al.; Eliakim et al.; Gropper et al.; Woolf et al.). Factors, which could affect both male and female athletes’ iron stores, is low energy intake, inadequate dietary iron intake, vegetarian diets, exercise-associated iron losses, reduced iron recycling (Hinton; Castell et al.; Sim et al.), and menstrual blood losses in female athletes (Pedlar et al.). Iron deficiency, which is accompanied by a reduction of oxygen transport to the working skeletal muscle, can lead to lower blood pH, depletion of muscle glycogen, which may negatively affect the endurance performance and exercise economy (Sim et al.).<SEP>Humans with experimentally induced anemia showed reduced VO2max, which is proportional to hemoglobin concentrations (Woodson et al.; Celsing et al.). Iron supplementation of anemic women improved iron status and performance during a standardized, multi-stage treadmill test and reduced exercise heart rate and blood lactate concentrations (Gardner et al.). Although iron supplementation does not necessarily improve VO2max (Klingshirn et al.; Zhu and Haas), this strategy is useful for iron-deficient nonanemic athletes in the improvement of athletic performance in endurance sports (Burden et al.; Rubeor et al.). Therefore, the iron status of athletes should be monitored systematically throughout the training and competition season to early detection or prevention of iron deficiency.<SEP>The limitation of our study is the small sample sizes of Japanese athletes, as well as sub-group of Russian athletes with VO2max data, which may lead to potential type I errors. As in all such studies, extension to, and replication within other racial groups is proposed.<SEP>In conclusion, we have shown that the HFE gene H63D polymorphism is strongly associated with endurance athlete status across East Asian, East and West European populations and with aerobic capacity in Russian athletes.",
                  "19768",
                  "['OBWCCE']"
                ],
                [
                  "26",
                  "30111348",
                  "Background<SEP>In our previous study, conducted from 1994 to 2000 in the Late Effects Outpatient Clinic of the Department of Oncology and Hematology, Institute of Pediatrics, Jagiellonian University, Krakow, Poland, we studied 255 patients, aged 4 to 28 years (median 12 years), who have completed acute lymphoblastic leukemia (ALL) treatment. The median time from completion of the treatment to enrollment in the study was 3 years. More than 50% of ALL survivors had late effects (LE) of treatment. The most frequent LEs were cognitive impairment and abnormalities of growth and body weight. Second neoplasms were diagnosed in 2% of patients. Endocrine LEs of any type were observed in 40% of ALL survivors. Other LEs were thyroid disorders, hypogonadism, premature puberty, seen in 11, 3 and 1% of patients, respectively. Menstruation disturbances were observed in 2% of female patients. Results of psychological tests of the ALL survivors were below the level of their healthy peers, mainly in mathematical skills, and the rate of test processing was slower. Girls had worse results than boys, and cranial radiotherapy (CRT) was implicated as the main cause of the impairments. Neurological LEs were seen in 18% of patients, and behavioral problems in 20% of patients. Serological markers of hepatitis B virus infection were positive in more than 30% of patients, and markers of hepatitis C virus infection in almost 50% of patients. Patients were followed-up in the clinics of the Institute of Pediatrics until they were 18 years old, and then they were referred to an adult outpatient center at the University Hospital in Krakow. The aim of our study was to assess the health status of childhood ALL survivors after more than 10 years of follow-up.<SEP>Polymorphisms of the genes responsible for regulation of metabolism can be associated with various LEs. As the regulation of the insulin pathway is responsible for proliferation of cell lineages, the role of polymorphisms of the genes involved in this pathway in the emergence of LEs of ALL treatment is interesting. In vitro, recombinant FTO takes part in catalyzing demethylation of certain methyl nucleotides in single-stranded DNA and RNA, which suggests a potential role of FTO in nucleic acid repair or modification. The FTO gene polymorphism is associated with an increase in the risk of obesity, as well as type 2 diabetes, heart failure, coronary heart disease, lifetime all-cause and ischemic stroke, hypertension, dyslipidemia, metabolic syndrome, and mortality. Moreover, it is also associated with increased fasting glucose and insulin levels, 2-h OGTT test results, HbA1c, blood pressure, lipid levels, liver function tests, and inflammatory markers, which are potential risk factors of the disorders mentioned above. Obesity is also considered a risk factor for certain cancers. Leptin receptor gene (LEPR) encodes the leptin receptor that is activated by leptin and has effects on food intake in humans. Mutations of the leptin gene and leptin receptor gene in obese patients suggest that leptin takes part in regulation of energy balance. LEPR gene polymorphisms, K109R and Q223R, located in exons 4 and 6, are implicated in the regulation of lipid metabolism and insulin resistance. Genetic associations between these two polymorphisms and obesity parameters, including insulin resistance, glucose levels and serum lipid profile, have been reported. Moreover, Q223R polymorphism of the LEPR gene is associated with metabolic syndrome.<SEP>Based on this theoretical background we investigated associations between the selected gene polymorphisms and prevalence of LEs in pediatric ALL survivors.",
                  "Methods<SEP>The study cohort included 125 patients who have previously completed ALL treatment and were available for a telephone survey conducted by a physician working at the Outpatient Clinic of the Department of Oncology and Hematology, Institute of Pediatrics, Jagiellonian University. Majority of them (65 out of 125) were included in our previous study. ALL therapy was conducted from 16/01/1984 to 8/11/2004 according to a modified BFM regimen with subsequent revisions (100 patients) or a New York regimen (25 patients). Details concerning treatment regimens were published elsewhere. Age at ALL diagnosis was 1–18.5 years (median 4.4, mean 4.2 years.) Details of the patients are provided in Table 1.<SEP>All patients included in the survey were interviewed by the same physician and the questions concerned: education/employment, health problems, established diagnoses, recent outpatient visits or hospital admissions, medications in use, marital status and offspring.<SEP>Data regarding the polymorphisms studied in the patients were previously published elsewhere.<SEP>Correlations between gene polymorphisms and the numbers and types of LEs were analyzed with respect to the intensity of chemotherapy (less intensive BFM regimens for standard/ intermediate-risk patients vs. more intensive New York regimens for high-risk patients, and BFM regimen for relapsed ALL) and to CRT. Descriptive statistics, odds ratios (OR) with 95% confidence interval, and Fisher’s exact test were used.<SEP>Local bioethics committee approved the study protocol. All parents and adolescent patients signed an informed consent before blood sample collection.",
                  "Results<SEP>The study cohort included 125 survivors of childhood ALL aged from 10.6 to 33 years (median 20.3 years); 58 patients (46%) were male and 67 patients (54%) were female. In 61 patients CRT was administered (doses: 14 to 24 Gy; median 18.2 Gy) according to the treatment regimens. A second CRT was performed in 3 patients (15 Gy, 18 Gy and 18.2 Gy), and spinal radiotherapy in one patient. Testicular radiotherapy was used in 2 patients (18 Gy and 21 Gy). In 95 (76%) patients less intensive ALL treatment regimens were used, and 30 (24%) patients were treated according to the more intensive regimens. Two patients completed a second course of treatment for relapsed ALL. One of them was treated with hematopoietic cell transplantation from a matched sibling donor. The duration of ALL treatment was from 1.7 to 4.2 years (median 3.2 years).<SEP>LEs were seen in 65 patients (52%), including one LE in 35 patients (28%), and > 1 LE in 30 patients (24%). Two LEs were seen in 20 patients (16%), 3 LEs in 8 patients (6%), and 5 and 7 LEs in 1 patient each (about 1%).<SEP>The most frequent LEs (Table 2) were endocrine disturbances, hepatitis, psychological and neurological abnormalities, which were seen in 22, 18, 10 and 6% patients, respectively. Further analysis revealed increase in the prevalence of hepatitis in patients treated with CRT (p = 0.0001).<SEP>Sixty four percent of the patients continued their education. About 51% of the patients who have completed their education had no permanent employment, including 4 mothers of children below 1 year of age and 3 persons who have qualified for a disability living allowance (Table 3).<SEP>The offspring of ALL survivors included 11 children (age from 0.3 years to 6 years; 7 girls and 4 boys). All of them were healthy.<SEP>Genetic studies revealed higher prevalence of hepatitis in individuals homozygous for the rs9939609A variant of the FTO gene compared with other patients (p = 0.03). Moreover, psychosocial complications were seen more frequently in patients with wild-type rs1137101 polymorphism (Q223R) of LEPR gene (p = 0.03). Numbers of patients in whom the studied polymorphisms were found are presented in Tables 4 and 5.<SEP>In the analysis of the entire study cohort, no statistically significant differences in the studied gene variants and ALL treatment intensity were found.",
                  "Discussion<SEP>Currently, a combined multimodal therapy (multi-agent intensive chemotherapy, radiotherapy and stem cell transplantation) allows for achieving a complete cure in approximately 90% of children with ALL. Growing numbers of young and middle-aged (15–45 years) childhood cancer survivors require skilled professionals to take care of this category of patients. In 1990, one in 1000 young adults was a survivor of childhood cancer, whereas in 2010 the rate was one in 250. In 1994 Childhood Cancer Survivor Study (CCSS) was started in the USA. It is still ongoing and it currently includes follow-up data of more than 20,000 childhood cancer survivors who have completed therapy more than 5 years ago. It was shown that approximately half of the survivors have symptomatic LEs, including severe and life threatening LEs in 21 and 14% of them, respectively. The LEs include: premature mortality, a second malignant neoplasm, organ dysfunction (e.g. heart, lung, gonads), growth impairment, delayed puberty, infertility, impairment of cognitive function. These may cause employment and insurance problems and impaired quality of life.<SEP>Use of radiotherapy and the type of cancer are the key risk factors for LE development. Prevalence of LEs was higher in patients treated for solid tumors compared with those treated for leukemia or lymphoma.<SEP>It is estimated that 1 in 715 young adults in the UK are survivors of childhood cancer and ALL was the diagnosis in 15% of the survivors aged from 20 to 39 years. About 28% of ALL survivors have psychological sequelae. Chronic conditions are found in 66 to 88% of childhood cancer survivors, and their prevalence increases with age. The prevalence of LEs in our cohort was within the ranges reported in the literature. However, given the methodology of our study, we may have underestimated the prevalence of LEs in our study population.<SEP>In the present study we assessed the prevalence of LEs using a telephone survey conducted in patients who have previously been treated at our center. This is an important limitation of the study. However, the calls were performed by the same physician who has previously taken care of the patients at the Late Effects Outpatient Clinic, which in turn improves the quality of the study. We were able to reach approximately 50% of the patients included in our previous study in the late 1990s. The difference in a median follow-up between both studies was approximately 9 years (3 vs 12 years). Only a few patients remained in a continued care of an outpatient clinic for adult survivors of childhood cancer. Almost all other patients were in a care of local healthcare providers. Prevalence of LEs in both studies was the same (50%). More than one LE was found in almost 25% of the patients. Endocrine LEs were less frequently seen in our present study (44% vs 22%). This was probably due to underdiagnosis. The prevalence of hepatitis B/hepatitis C decreased from 30%/50 to 18% (counted together), and prevalence of neurological LEs decreased from 18 to 6%. The increase in the rates of second malignant neoplasms was not significant (2% vs. 3%). The study revealed higher prevalence of hepatitis in patients treated with CRT (p = 0.0001) and in patients homozygous for the rs9939609A variant of the FTO gene (which in our previous study was associated with higher BMI) compared with other patients. This may be explained by the use of CRT in the more intensive treatment regimens, which increased the risk of infection, whereas obesity, which was more prevalent in patients homozygous for rs9939609A variant of the FTO gene, causes impaired liver function due to activation of the insulin pathway and may be associated with higher susceptibility to infections. However, we are unable to explain why wild-type Q223R polymorphism of LEPR gene was more frequently found in patients with psychological LEs (p = 0.03). Interestingly, as in our previous study, we were unable to establish correlations between the tested polymorphisms and the development of LEs in the context of intensity of chemotherapy. It was probably due to a limited number of patients included in the study. In the future, genetic studies (assessment of gene expression profile, GWAS) might be useful in stratification of patients, personalization of therapy, outcome prediction and estimation of the risk of LEs, as well as in identification of new therapeutic targets.<SEP>In our previous study, quality of life was assessed using a score (1–5) defined by Skoczen. The average score was 4.4 points. Unfortunately, the score was not included in the current telephone survey questionnaire.<SEP>Sixty four percent of childhood ALL survivors continued their education. Approximately 51% of patients who completed the education had no permanent employment, including 4 mothers of children below 1 year of age and 3 persons who have qualified for a disability living allowance. The offspring of the ALL survivors included 11 children (age from 0.3 years to 6 years; 7 girls and 4 boys). All of them were healthy.<SEP>As it was shown in other studies, cognitive impairment may be the reason for employment problems. The unemployment rate in the study cohort was significantly higher than in age-matched general population in Poland (11% unemployed, 15% temporarily unemployed). The levels of education of survivors of childhood malignancies are generally lower than in the healthy population of the same age. In our study, a university degree was achieved by 20% of patients who completed their education, compared with 31% in the age-matched Polish population. One of the most important issues in many pediatric oncology centers is the transition of patients > 18 years of age to adult oncology centers. Our procedure is to refer the survivors to the University Outpatient Clinic. Unfortunately, most of our patients are not satisfied with the care offered by the adult clinic. Therefore, they are most frequently followed-up by local healthcare providers, who have no skills necessary for comprehensive care of childhood cancer survivors. This may cause frequent underdiagnosis of LEs due to the limited awareness. This is the first study of survivors of childhood ALL that describes the current situation of this group of patients in Poland. We found shortages of appropriate medical care of these patients due to a lack of awareness in patients, physicians and health care providers. Because of inefficient organization and limited resources, the diagnosis and treatment of LEs were delayed.<SEP>Well-designed protocols regulating the transition of patients from pediatric to adult outpatient care are necessary to improve this situation. Outpatient clinics should include multidisciplinary care covering most common LEs. Appropriate communication and data exchange between pediatric and adult healthcare providers should be ensured, with joint discussion on specific recommendations in particular patients, where necessary. It would be beneficial for the childhood cancer survivors, particularly those who have completed their therapy long time ago, to participate in conferences and use internet resources to provide them with access to comprehensive information on their health, quality of life, education and employment.",
                  "14982",
                  "['OBWCCE']"
                ],
                [
                  "27",
                  "25474695",
                  "Introduction<SEP>As obesity represents a substantial and growing threat to public health, efforts to identify the determinants of obesity are of considerable scientific and societal importance. Genome-wide association studies (GWAS) have identified numerous variants associated with body mass index (BMI), but a substantial proportion of the estimated heritability remains to be accounted for. At the same time, a number of modifiable environmental factors have been identified that influence BMI, with cigarette smoking a strong lifestyle influence on BMI. In a previous Mendelian randomisation analysis, we used a single nucleotide polymorphism in the CHRNA5-A3–B4 gene cluster associated with heaviness of smoking within smokers to confirm the causal effect of smoking in reducing BMI.<SEP>We sought to extend these findings in a larger sample drawn from the Causal Analysis Research in Tobacco and Alcohol (CARTA) consortium (http://www.bris.ac.uk/expsych/research/brain/targ/research/collaborations/carta/). We used the same genetic variant, characterised by two SNPs (rs16969968 and rs1051730) which are in perfect linkage disequilibrium (LD) in samples of European ancestry, and therefore reflect the same genetic signal (hereafter rs16969968-rs1051730). This variant is associated with approximately 1% phenotypic variance in cigarettes per day and approximately 4% variance in cotinine levels (the primary metabolite of nicotine, and a more precise measure of exposure). Mendelian randomisation analyses of the causal effects of smoking heaviness require stratification according to smoking status – any causal effects of the exposure (i.e., smoking heaviness) should be reflected in an association of the instrument (i.e., genotype) among current smokers only, and not never smokers (former smokers might be expected to be intermediate between current and never smokers). The never smoking group therefore enables a test of the specificity of the instrument (i.e., that the variant only affects the outcome through the exposure of interest). Critically, the rs16969968-rs1051730 variant has not been shown to be associated with smoking initiation (i.e., it does not influence risk of being an ever versus a never smoker) in previous GWAS of smoking behaviour, which reduces the risk of introducing collider bias when stratifying on smoking status.<SEP>In the course of these analyses, we observed an unexpected finding, which we report here. Specifically, we observed an association of rs16969968-rs1051730 with higher BMI in never smokers. This association has not previously been reported in GWAS of BMI published to date. We therefore focus on the implications of this novel finding, and not the Mendelian randomisation analysis of the causal effects of smoking on BMI.",
                  "Methods<SEP>Study populations<SEP>We used data on individuals (≥16 years) of European ancestry (ascertained via self report, or based on the genome-wide genotype data where available) from 29 studies in the Causal Analysis Research in Tobacco and Alcohol (CARTA) consortium (http://www.bris.ac.uk/expsych/research/brain/targ/research/collaborations/carta/): the 1958 Birth Cohort (1958 BC), the Avon Longitudinal Study of Parents and Children (ALSPAC, including both mothers and children), the British Regional Heart Study (BRHS), the British Women's Heart and Health Study (BWHHS), the Caerphilly Prospective Study (CaPS), the Christchurch Health and Development Study (CHDS), the Cohorte Lausannoise (CoLaus) study, the Exeter Family Study of Child Health (EFSOCH), the English Longitudinal Study of Ageing (ELSA), FINRISK, the Danish GEMINAKAR twin study, Generation Scotland, the Genomics of Overweight Young Adults (GOYA) females, GOYA males, the Helsinki Birth Cohort Study (HBCS), Health2006, Health2008, the Nord-Trøndelag health study (HUNT), Inter99, the Northern Finland Birth Cohorts (NFBC 1966 and NFBC 1986), MIDSPAN, the Danish MONICA study, the National Health and Nutrition Examination Survey (NHANES), the MRC National Survey of Health & Development (NSHD), the Netherlands Twin Registry (NTR), the Prospective Study of Pravastatin in the Elderly at Risk (PROSPER) and Whitehall II. References to these individual studies are available on request. All studies received ethics approval from local research ethics committees (see Text S1 for full details).<SEP>Genotyping<SEP>Within each study, individuals were genotyped for one of two single nucleotide polymorphisms (SNPs) in the CHRNA5-A3-B4 nicotinic receptor subunit gene cluster, rs16969968 or rs1051730. These single nucleotide polymorphisms are in perfect linkage disequilibrium with each other in Europeans (R2 = 1.00 in HapMap 3, http://hapmap.ncbi.nlm.nih.gov/) and therefore represent the same genetic signal. Where studies had data available for both SNPs, we used the SNP that was genotyped in the largest number of individuals.<SEP>Body mass index<SEP>Height (m), weight (kg) and waist circumference (cm) were assessed within each study, directly measured for 99% of participants, and self-reported for GOYA females (N = 1,015) and a sub-set of NTR (N = 602). Body mass index (BMI) was calculated as weight/height2.<SEP>Smoking status<SEP>Smoking status was self-reported (either by questionnaire or interview). Individuals were classified as current, former, or never cigarette smokers. Where information on smoking frequency was available, current smokers were restricted to individuals who smoked regularly (typically at least one cigarette per day). Where information on pipe and cigar smoking was available, individuals reporting being current or former smokers of pipes or cigars but not cigarettes were excluded from all analyses. For studies with adolescent populations (ALSPAC children and NFBC 1986), analyses were restricted to current daily smokers who reported smoking at least one cigarette per day (current smokers) and individuals who had never tried smoking (never smokers). Descriptive characteristics of smoking frequency data are provided in Text S2.<SEP>Statistical analysis<SEP>Analyses were conducted within each contributing study using Stata and R software, following the same analysis plan. Analyses were restricted to individuals with full data on smoking status and rs16969968-rs1051730 genotype. Within each study, genotype frequencies were tested for deviation from Hardy Weinberg Equilibrium (HWE) using a chi-squared test. Mendelian randomisation analyses of the association between rs16969968-rs1051730 and BMI were performed using linear regression, stratified by smoking status (never, former and current) and sex, and adjusted for age. BMI was log transformed prior to analysis. An additive genetic model was assumed on log values, so that each effect size could be exponentiated to represent the percentage increase in BMI per minor (risk) allele.<SEP>For NHANES, which has a survey design, Taylor series linearization was implemented to estimate variances. For studies including related family members appropriate methods were used to adjust standard errors: in GEMINAKAR, twin pair identity was included as a cluster variable in the model, in MIDSPAN linear mixed effects regression models fitted using restricted maximum likelihood were used to account for related individuals. ALSPAC mothers and children were analysed as separate samples; as there are related individuals across these samples, sensitivity analyses were performed excluding each of these studies in turn.<SEP>Results from individual studies were meta-analysed in Stata (version 13) using the “metan” command. As I2 values were all equal to or below 36% (indicating low to moderate heterogeneity), fixed effects meta-analyses were performed. The “metareg” command was used to examine whether SNP effects varied by sex and estimates were combined as there was no evidence for effect modification by sex. Evidence for interaction between genotype and smoking status was assessed using the Cochran Q statistic. Data are available from the Institutional Data Access/Ethics Committees of the individual studies that contributed to this analysis, for researchers who meet the criteria for access to confidential data. Full details are provided in Text S3.<SEP>Sample size calculations<SEP>Sample size calculations were performed using Quanto software (http://biostats.usc.edu/Quanto.html). The following parameters were used: 80% power to detect associations, minor allele frequency of 0.33, mean and standard deviation for BMI of 25 kg/m2 and 3.8 kg/m2 respectively, alpha values of 0.05 and 5×10−8.",
                  "Results<SEP>Our total sample size comprised 148,730 never smokers, former smokers and current smokers. In the 66,809 never smokers, we observed positive association of rs16969968-rs1051730 with BMI (Table 1), indicating an association operating via pathways other than smoking (percentage change per minor allele +0.35, 95% CI +0.18 to +0.52, P = 6.38×10−5). We also confirmed the expected inverse association of rs16969968-rs1051730 with BMI in the 38,913 current smokers (percentage change −0.74, 95% CI −0.97 to −0.51, P = 2.00×10−10), consistent with a causal, weight-reducing effect of cigarette smoking on BMI. There was no evidence of association in the 43,009 former smokers (percentage change −0.14, 95% CI −0.34 to +0.07, P = 0.19). An interaction test indicated that these estimates differed from each other (P = 4.95×10−13). Similar associations were observed for weight (Table 1) and waist circumference (data available on request), but not height (Ps ≥0.27 for all smoking categories). Between-study heterogeneity was low (I2 values ≤36%), and there was no evidence for effect modification by sex. Critically, when data were examined without stratification by smoking status no clear evidence of association with BMI was observed (P = 0.22), indicating that a conventional GWAS would have failed to detect this signal.<SEP>The 0.35% per minor allele BMI increase in never smokers represents a change of approximately 0.09 kg/m2. This is smaller than the effect of rs9939609 in FTO (∼0.4 kg/m2) but is comparable in terms of variance explained to the other variants identified by Speliotes and colleagues. As noted above, the rs16969968-rs1051730 variant has not been shown to be associated with smoking initiation in previous GWAS of smoking behaviour. This is also true in our data (ever smoker versus never smoker: OR per minor allele 1.01, 95% CI 0.99 to 1.03, P = 0.50), although we observed an association with smoking cessation (current smoker versus former smoker: OR per minor allele 1.08, 95% CI 1.06 to 1.10, P = 1.44×10−12), consistent with previous studies. Therefore, we do not believe that these findings are due to collider bias, whereby stratifying on the exposure measure can induce associations between instrument and outcome.",
                  "Discussion<SEP>Our results indicate that rs16969968-rs1051730 may be associated with BMI in never smokers, via pathways other than smoking, as well as with heaviness of smoking among current smokers. At this stage we can only speculate as to the mechanism through which rs16969968-rs1051730 may exert a positive effect on BMI in never smokers. In GWAS, the CHRNA5-A3-B4 gene cluster was confirmed to be associated with heaviness of smoking, and downstream health outcomes including lung cancer and peripheral arterial disease. It has been shown that the rs16969968 variant is functional and leads to an amino acid change (D398N) in the α5 nicotinic acetylcholine receptor (nAChR) subunit protein. Animal models indicate that this subunit modulates tolerance to high doses of nicotine. Candidate gene studies have suggested an association of rs16969968-rs1051730 with other substance use phenotypes, such as cocaine use, while other variants in this region have been reported to be associated with alcohol consumption, although the evidence for these associations is currently weak. Therefore, one possibility is that nAChRs play a role in central mechanisms mediating responding to rewarding stimuli in general, which could include natural rewards such as food.<SEP>It is also notable that rs3743075, located within the CHRNA3 gene and correlated with rs16969968-rs1051730 (r2 = 0.34, D′ = 1.00), shows association (N = 974, P = 9.06×10−5) with BMI (defined as <30 kg/m2 vs ≥30 kg/m2) (dbGaP Study Accession: pha003015.1). There is evidence from animal models that activation of hypothalamic α3β4 nAChRs leads to activation of pro-opiomelanocortin neurons, and subsequent activation of melanocortin 4 receptors, which have been shown to be critical for nicotine-induced decreases in food intake. Therefore, another possibility is that nAChR sub-units play a role specifically in mediating food intake, through as yet undescribed mechanisms. In other words, the effects we have observed operate via other nAChRs, and other genes in this region (namely CHRNA3 and CHRNB4) may contribute to our finding. Clearly further work is required to explore this possibility. The use of more detailed body composition measures such as percent body fat and its distribution may also serve to refine the nature of the association.<SEP>Our results, if confirmed, have important implications for the design of future GWAS. The association we observed in never smokers would essentially be undetectable in an unstratified sample, since the effect size observed in the combined sample would require approximately 791,000 participants to detect even at an uncorrected P-value of 0.05, and even then would indicate an inaccurate effect size. This is essentially because the effect of rs16969968-rs1051730 on BMI that operates via pathways other than smoking is countered by the weight-reducing effect of smoking. Therefore, since there are roughly twice as many never smokers as current smokers on average across our sample, these two effects negate each other. On the other hand, a sample of approximately 160,000 never smokers would be required to detect the effect we observed with genome-wide significance. Assuming the proportions of never, former and current smokers in our sample, this would imply a total sample size of around 350,000. While this is larger than published GWAS of BMI, it is achievable. Therefore, although we cannot say how frequent a scenario such as the one we observed here will be, additional variants may be identified in GWAS stratified by environmental exposures known to have pronounced effects on the phenotype of interest, such as cigarette smoking or physical activity on BMI.<SEP>The pleiotropic effect of rs16969968-rs1051730 (or LD of this variant with another variant causally influencing BMI), if shown to be robust via replication, has important implications for Mendelian randomisation studies assessing the causal effects of smoking. In this case, we can be reasonably confident that the BMI-reducing effect of the variant operates through smoking because the association with BMI in current smokers is in the opposite direction to the association in never smokers. Furthermore, if the effects on BMI that operate via pathways other than smoking and the effects that operate via the weight-reducing effects of smoking are independent, then the true causal estimate of the magnitude of effect of smoking in reducing BMI is likely to be larger than estimated with this variant. However, some caution must be exercised in conducting and interpreting the results of other Mendelian randomisation analyses using this variant because rs16969968-rs1051730 may influence outcomes through its effects on BMI, instead of or in addition to smoking heaviness. One possible solution is to use genetic variants for BMI as a method of reciprocal randomization to determine the direction of causation within inter-correlated networks of mechanistic pathways (i.e., network Mendelian randomisation).<SEP>A limitation to our analysis is that we were only able to control for potential population stratification indirectly in most samples, by restricting analyses to participants of self-reported European ancestry. We were not able to use other methods, such as adjustment for principal components, given that not all contributing studies hold the necessary genetic data. However, we note that the minor allele frequency of the rs16969968-rs1051730 differed only slightly across studies (between 0.30 and 0.36).<SEP>Testing for gene-environment interaction in GWAS is not novel, and examples exist which incorporate smoking status as an environmental factor. However, this remains relatively uncommon, due to methodological challenges (e.g., introducing collider bias) and sample size constraints. A key challenge is the identification of suitable environmental variables on which to stratify GWAS analyses, from the multitude available. We suggest that focusing on environmental factors that are most strongly associated with the phenotype of interest, are likely to have profound biological effects, and which can be characterised in a relatively consistent way across studies, is likely to be the best strategy. Smoking status meets all of these criteria, and the data presented here demonstrate how stratification on well-characterized environmental factors known to impact on health outcomes (such as smoking status) may reveal novel genetic associations with health outcomes. As our data indicate, these associations may operate through genetic influences on the environmental factors themselves, or through new pathways which are masked by the environmental factors.",
                  "17509",
                  "['OBWCCE', 'EATBTS', 'FOODIN']"
                ],
                [
                  "28",
                  "23173044",
                  "Introduction<SEP>Genome-wide association (GWA) studies have facilitated the identification of genetic regions involved in the development of type 2 diabetes. Robust evidence of disease association in different populations has been obtained for several novel susceptibility gene loci identified by GWA studies; CDKAL1 is among the best-replicated susceptibility loci. The CDKAL1 gene encodes a 65 kDa protein– cyclin-dependent kinase 5 regulatory subunit associated protein 1-like 1 (CDKAL1). A cluster of single nucleotide polymorphisms (SNPs) in intron 5 of the CDKAL1 gene were associated with type 2 diabetes in populations of European and Asian descent. This association was further tested with phenotypes of β-cell dysfunction, in particular, impaired insulin secretion as assessed by the oral or intravenous glucose tolerance test or a hyperglycemic clamp, showing reproducible association of the same variants with reduced first-phase insulin secretion. In addition, it was reported that the type 2 diabetes risk–conferring alleles of CDKAL1 were associated with lower birth weight, which is known to be associated with an increased risk of type 2 diabetes, presumably due to reduced insulin secretion or insulin sensitivity, i.e., the fetal insulin hypothesis. Together, despite a lack of direct biological evidence reported to date, these association data support the possible contribution of causal variants at CDKAL1 to the pathogenesis of type 2 diabetes.<SEP>In the present study, we first examined the effect of a global Cdkal1-null mutation in a mouse model to clarify the physiological importance of CDKAL1. Further, to see whether the observations of altered fat accumulation and reduced body weight in Cdkal1-deficient mice, not at birth but adulthood, could be pertinent to the human situation, we tested an association of CDKAL1 with adult body mass index (BMI) in 2 independent Japanese populations and detected reproducible associations.",
                  "Materials and Methods<SEP>Ethics Statement<SEP>All animal experiments were approved by the Animal Care and Use Committee of the National Center for Global Health and Medicine (NCGM) Research Institute (permit number: 23-Tg-31), and conducted in accordance with institutional procedures. All human participants provided written informed consent, and the ethics committees of NCGM, Kyushu University, Osaka University, and Amagasaki Health Medical Foundation approved the protocols.<SEP>Generation of Cdkal1-knockout Mice<SEP> Cdkal1-knockout mice were generated by the gene trapping method at TransGenic Inc. (Kobe, Japan) and thereafter established as an experimental model at NCGM. An ES cell line TT2, which had a mixed genetic background of CBA/JNCrj and C57BL/6J, was used for gene trapping. 5′-rapid amplification of cDNA ends (RACE) and sequence analysis showed that the gene trap vector pU17 was successfully inserted into intron 3 of the Cdkal1 gene (Figure 1A). Germ-line transmitting chimaeric mice were generated and mated with C57BL/6 females. The F1 heterozygous mice (Cdkal1 +/−) were examined for the presence of the trap gene by PCR. The F1 heterozygous mice were backcrossed at least twice onto a C57BL/6 background and then interbred to generate Cdkal1 −/− mice. Their wild-type littermates (Cdkal1 +/+) were used as the controls. All mice were housed in air-conditioned animal rooms at an ambient air temperature of 22±2°C and relative humidity of 50±15%, under specific pathogen-free conditions with a 12-h light/dark cycle. Male mice were used for the experiments; they were weaned at 4 weeks of age, given free access to drinking water, and were fed a standard diet (CE-2, 12 kcal%, CLEA Japan, Inc., Tokyo, Japan). Some mice were switched to a high-fat diet (D12451, 45 kcal%, Research diet, New Brunswick, NJ, USA) at 8 weeks of age (Table S1).<SEP>Metabolic Studies<SEP>Blood samples were obtained after 15-h fast. Plasma glucose levels were measured with a Glucose C-II kit (Wako Pure Chemical Industries, Osaka, Japan). Plasma insulin concentrations were determined by an ultra-high sensitivity mouse insulin enzyme-linked immunosorbent assay kit (Morinaga, Yokohama, Japan). Total cholesterol (TC) and triglyceride (TG) levels in plasma, the liver and muscle were measured with a cholesterol C-II kit and a triglyceride E kit (Wako Pure Chemical Industries, Osaka, Japan), respectively. Insulin contents were measured in the whole pancreas according to the protocols described previously.<SEP>Assessment of Locomotor Activity<SEP>Locomotor activity was assessed by scoring the number of photobeam breaks in an open field test chamber (40 cm×40 cm×35 cm) (Panlab S. I., Barcelona, Spain). The 16-weeks-old male mice fed on a high-fat diet for 8 weeks were used for the open field test. Each mouse was placed in the chamber and habituated for 2 h and its spontaneous locomotor activities were recorded for the following 60 min.<SEP>Oral Glucose Tolerance Test (OGTT) and Insulin Tolerance Test (ITT)<SEP>Glucose tolerance was assessed by oral glucose administration. After a 15-h fast, the mice were weighed and glucose (1 g/kg or 2 g/kg) was administered orally. Glucose and insulin concentrations in the blood of tail vein and retro-orbital venous plexus, respectively, were measured. For ITT, mice fasted for 3 h, and human insulin (1 IU/kg or 0.75 IU/kg; Novolin R, Novo Nordisk, Denmark) was administered via intraperitoneal injection. Blood samples were drawn from the tail vein at different time points. The glucose value at each time point was expressed as a percentage of the value at time 0.<SEP>Perfusion Experiments in Mouse Pancreata<SEP>Overnight-fasted mice (Cdkal1 +/+and Cdkal1 −/−) at 12 weeks of age, which were fed either standard diet or high fat diet, were used in perfusion experiments as previously reported with slight modifications. Briefly, after anesthesia, the superior mesenteric and renal arteries were ligated, and the aorta was tied off just below the diaphragm. The perfusate was infused from a catheter placed in the aorta and collected from the portal vein. The perfusion protocols began with a 20-min equilibration period with Krebs-Ringer bicarbonate (KRB) buffer containing 2.8 mmol/L glucose. At 3 min after sampling initiation, the glucose concentration of the perfusate was shifted from 2.8 mmol/L to 16.7 mmol/L during the subsequent period of 17 min. The flow rate of the perfusate was 1 ml/min. All samples were readily collected on ice and stored at −80°C. Insulin concentrations in the perfusate were measured as described above.<SEP>CT Scan<SEP>Intraabdominal and subcutaneous fat of mice was examined radiographically using LaTheta LCT-200 (ALOKA, Mitaka, Japan) according to the manufacturer’s protocol. CT scanning was performed at 1.5-mm intervals from the diaphragm to the bottom of the abdominal cavity.<SEP>Microarray Analysis<SEP>Microarray gene expression analysis was performed on the muscle from Cdkal1 −/− mice and wild-type littermates (n = 4 each) using a mouse whole-genome microarray kit ver2.0 (Agilent Technologies, Palo Alto, CA, USA) according to the manufacturer’s protocol. The data analysis was done with the Bioconductor software (http://www.bioconductor.org/).<SEP>Quantitative Real-time PCR<SEP>For the preparation of RNA, Isogen (Nippon Gene, Tokyo, Japan) was used. cDNA synthesis was performed with random hexamer-oligonucleotides. Quantitative PCR was performed on the 7900HT Fast Real-Time PCR system (Applied Biosystems, Foster City, CA, USA) with FastStart Universal SYBR Green Master (Roche Diagnostics, Mannheim, Germany). The primers used for the PCR amplification are shown in Table S2.<SEP>Immunoblot Analysis<SEP>For western blotting, 5 units of human regular insulin was injected into the inferior vena cava of anesthetized mice after overnight fast, and the livers were removed at 2 min, the hind limb muscles and white adipose tissues (WAT) at 5 min after injection. The tissues were then homogenized in the lysis buffer and protease inhibitor cocktail (Roche Diagnostics, Mannheim, Germany). Fifty micrograms of protein was loaded to a 12% SDS/PAGE gel, and transferred onto PVDF membrane (BIO-Rad, Hercules CA, USA). We purchased total-Akt (#4691) and phospho-Akt (pAkt, #9271) antibodies (Cell Signaling, Beverly MA, USA) and Cdkal1 (#ab68045) antibody (Abcam, Cambridge MA, USA). The proteins were visualized by using ECL (Thermo Fisher Scientific, Waltham MA, USA) and quantified by densitometry.<SEP>Human Study Sample<SEP>An association study of 7 previously identified variants for type 2 diabetes and/or BMI was performed in Japanese subjects (Text S1); 5,695 samples of the Amagasaki panel and 12,569 samples of the Fukuoka panel; a case-control study panel comprising 6,369 cases and 6,406 controls, where type 2 diabetes was diagnosed according to the 1999 WHO criteria as described elsewhere. Brief descriptions of the assessment of biological parameters and lifestyle are available elsewhere.<SEP>SNP Genotyping<SEP>Samples were genotyped using the TaqMan assay (Life Technologies Japan, Tokyo, Japan) for 7 SNPs from 7 unique loci. These included CDKAL1 (rs4712523), IGF2BP2 (rs4402960), SLC30A8 (rs13266634), CDKN2A/B (rs2383208), HHEX (rs1111875), TCF7L2 (rs7903146), and KCNQ1 (rs2237892). The genotype distribution of all tested SNPs was in Hardy-Weinberg equilibrium (P>0.01). We obtained successful genotyping call rates of>99% for the whole characterized sample.<SEP>Statistical Analysis<SEP>Comparative analysis in mice<SEP>The results are expressed as means ± SEM unless otherwise indicated. Differences were analyzed using an unpaired Student’s t-test when comparing two groups means. Data for body weight and insulin secretion during OGTT were subjected to repeated measure ANOVA. P<0.05 was considered to be statistically significant.<SEP>SNP association analysis<SEP>We standardized BMI to the z-score in each panel before association analysis. SNPs were tested for association with BMI using linear regression analysis in the additive genotype model after adjustment for age classes separately by sex. Age classes were defined according to the age distribution in the individual panels; they were ≤40, 41–50, 51–60, and>60 (Amagasaki panel); and ≤55, 56–60, 61–65, 66–70, and>70 (Fukuoka panel). Association results for the two Japanese panels were combined using the inverse variance method. We used PLINK (http://pngu.mgh.harvard.edu/~purcell/plink/), R software (version 2.8.1; www.r-project.org), and the rmeta package (http://cran.r-project.org) for association tests and meta-analysis. A significance level was set at P<0.007 according to Bonferroni correction for multiple testing.",
                  "Results<SEP>Phenotypic Characterization of Cdkal1-knockout Mice<SEP>Expression of Cdkal1 was found to be relatively ubiquitous in mice. Cdkal1 mRNA was prominently expressed in the skeletal muscle, pancreatic islets, and heart (Figures 1B and 1C), as previously reported in humans. Here, CDKAL1 protein was confirmed to be detectable in Cdkal1 +/+but not in Cdkal1 −/− mice (Figure S1C) as previously reported. Genotype analysis of 91 pups generated by cross-breeding heterozygous Cdkal1 +/− mice demonstrated that 21 mice were Cdkal1 +/+, 46 were Cdkal1 +/− and 24 were Cdkal1 −/−, a distribution consistent with Mendelian inheritance. The locomotor activities of Cdkal1 −/− mice were indistinguishable from their wild-type littermates (Figure S2) and no gross anatomical changes were observed either externally or in the major organs.<SEP> Table 1 summarizes the phenotypes in these animals. On a standard diet, although the effects on body weight were relatively modest (i.e., reduction by 2–5%) in Cdkal1 −/− mice compared to wild-type littermates (see Figure 2A), we observed significant, symmetrical increases of body weight in Cdkal1 transgenic mice, which we further generated (Text S1 and Figure S1). This supported the possible involvement of Cdkal1 in the regulation of body weight. No significant changes in plasma lipids, glucose, or insulin were observed in Cdkal1 −/− mice relative to wild-type littermates. On a high-fat (high-saturated fat/low-carbohydrate) diet, Cdkal1 −/− mice also tended to be lighter throughout the follow-up period. In particular, Cdkal1 −/− mice showed a significant reduction in fat accumulation, monitored by %intraabdominal and %subcutaneous fat, after 8 weeks on the high-fat diet, i.e., 16 weeks of age (see Figures 2B and 2C). These findings were further supported by reduced lipid accumulation (as estimated by triglyceride content) in the liver and muscle at the corresponding time point (Figure S3). Inversely, at a later stage (20 weeks) of high-fat feeding, lipid accumulation became more prominent in Cdkal1 −/− mice than wild-type littermates (Figure S3). Food intake tended to be reduced in Cdkal1 −/−mice (Table 1).<SEP>During the follow-up period, Cdkal1 −/−mice caught up with wild-type littermates in %intraabdominal fat after 16 weeks on the high-fat diet (Figure 2C). Plasma glucose and insulin levels remained lower in Cdkal1 −/−mice over almost the entire follow-up period, except that plasma insulin eventually increased in Cdkal1 −/−mice after 20 weeks on the high-fat diet. At this time point, despite almost equivalent levels of insulin, and %intraabdominal and %subcutaneous fat between the strains, plasma leptin levels tended to be higher in Cdkal1 −/−mice than wild-type littermates (28.1±1.1 ng/ml vs. 31.2±1.2 ng/ml for wild-type littermates vs. Cdkal1 −/− mice, P = 0.078). They did not accompany the expected increases in TNF-α (Table 1), whose circulatory levels are known to generally increase with the degree of obesity. On a high-fat diet, Cdkal1 −/−mice were more insulin-sensitive than wild-type littermates until a certain time point; that is, a lower glucose level was attainable with a significantly (P = 0.015) lower insulin level despite almost equivalent fat accumulation after 16 weeks on the high-fat diet (Figure 2C).<SEP>Glucose Homeostasis and Insulin Release in vivo <SEP>On a standard diet, no apparent impairment of glucose tolerance was observed in Cdkal1 −/−mice at 12 and 20 weeks of age (Figure S4A and S4B). While a modest (but not significant) decrease in plasma glucose was seen at 60–120 min after insulin injection at both ages in Cdkal1 −/−mice relative to wild-type littermates, insulin sensitivity was almost indistinguishable between the two strains on a standard diet (Figure S4C and S4D). On a high-fat diet, elevated fasting glucose levels and glucose intolerance during OGTT were seen in both strains at an early stage (after 4 and 8 weeks) of high-fat feeding, which was more prominent in wild-type littermates than Cdkal1 −/− mice (Figure S4E and S4F). Also, in wild-type littermates on a high-fat diet, insulin sensitivity assessed by ITT deteriorated more markedly (Figure S4G and S4H). However, at a later stage (after 20 weeks) of high-fat feeding (Figure 3), the impairment of glucose tolerance and insulin sensitivity became more prominent in Cdkal1 −/− mice than wild-type littermates (P<0.05; Figures 3A and 3C), contrary to the findings at the early stage. Insulin secretion during OGTT was reduced in Cdkal1 −/− mice on a high-fat diet (in particular, at an early stage of high-fat feeding) but not on a standard diet (Figure 3B and Figure S4I and S4J).<SEP>Insulin Secretion in Perfused Pancreata<SEP>To examine the time course of the insulin secretory response to high glucose in Cdkal1 −/− mice, perfusion experiments were performed in the standard (Figure 4A) and high-fat (Figure 4B) diet groups. In wild-type littermates (Cdkal1 +/+), 16.7 mmol/L glucose elicited insulin secretion [the amount of secreted insulin (AUCinsulin) after glucose stimulation (from 3 to 20 min); 130±18 ng in 17 min, n = 6], which was further potentiated by 4 weeks of high-fat feeding [AUCinsulin; 212±18 ng, n = 6, P<0.01 vs. standard diet] (Figure 4C). In Cdkal1 −/− mice, 16.7 mmol/L glucose elicited insulin secretion almost equivalently to Cdkal1 +/+mice on a standard diet (AUCinsulin; 151±19 ng, n = 8), whereas there was no apparent potentiation of insulin secretion after 4 weeks of high-fat feeding (AUCinsulin; 147±14 ng, n = 6). Thus, based on an assessment of AUCinsulin, high-fat feeding did not potentiate insulin secretion in Cdkal1 −/− mice (1.0-fold), contrary to the findings in Cdkal1 +/+mice (1.6-fold), although glucose tolerance was impaired in both strains as compared to the situation on a standard diet (Figure 4C and Figure S4A and S4E). It remains to be determined whether such inter-strain differences in impaired insulin secretion are reproducible in mice high-fat fed for 20 weeks, at which time points insulin sensitivity in Cdkal1 −/− mice became worse and almost equivalent to that in Cdkal1 +/+mice (Figure 2C). Nevertheless, the present findings in perfusion experiments after 4 weeks of high-fat diet appeared to be consistent with the results for in vitro experiments (using batch incubated β cells) in the previous study.<SEP>Changes Associated with Protection Against Diet-induced Obesity and Insulin Resistance in Cdkal1 −/− Mice<SEP>To understand biological mechanisms, by which Cdkal1 −/− mice could show protection against high fat diet-induced obesity and insulin resistance, we performed microarray gene expression analysis on the muscle, along with comparing mRNA expression of target genes (including several physiological candidate genes) in brown adipose tissues (BAT) and WAT, liver and muscle between Cdkal1 +/+and Cdkal1 −/− mice (Figure 5). Although there were significant (P<0.05) inter-strain differences in mRNA expression of a few candidate genes, e.g., Adrb3 in BAT and Ucp2 in the liver, they did not appear to be causal, considering the pattern of expression differences. In the microarray analysis, we identified an Scd1 gene among a list of most significant genes differentially expressed (Table S3). Scd1 was confirmed to be down-regulated in muscle and WAT of Cdkal1 −/− mice (Figure 5).<SEP>Next, to test whether protection against high fat diet-induced insulin resistance is mediated by enhanced insulin signaling, we examined insulin-dependent Akt activation. We found that the levels of phosphorylated Akt protein were comparable between the strains when fed on a high-fat diet for 4 weeks (Figure S5).<SEP>Correlation of type 2 Diabetes and BMI Associations at Candidate Loci<SEP>In the Japanese general population samples (Table 2), we found significant (P<0.05/7 ≈ 0.007) associations with BMI at CDKAL1 rs4712523 and KCNQ1 rs2237892, and nominal (P<0.05) associations at 2 other loci–CDKN2A/B rs2383208 and TCF7L2 rs7903146 (Table 3). At CDKAL1, we evaluated the association of rs4712523, which showed the strongest association in our previous GWA study of type 2 diabetes, with BMI and found reproducible evidence for association in 2 independent panels (Table 3), where the risk allele (G of rs4712523) for type 2 diabetes showed nominal association with lower BMI [P = 0.024 in the Amagasaki panel (n = 5,695) and P = 0.02 in the Fukuoka panel (n = 12,569)]. Even when restricted to non-diabetic individuals, the effect sizes were almost unchanged as compared to those calculated in the whole panel (Table S4). When we reanalyzed the BMI association at CDKAL1 by arbitrarily categorizing the samples into two age groups (age<60 years and age≥60 years) without adjusting for age in the linear regression, there was no statistically significant inter-age-group difference (P heterogeneity = 0.56, Table 3). In addition, the “inverse” correlation between disease risk and lower BMI was replicated for 6 other type 2 diabetes risk loci, which had been all suggested to affect insulin secretion (Figure 6). There was, on the other hand, a positive correlation between type 2 diabetes risk and higher BMI for 2 obesity (or insulin resistance)-associated loci–FTO and MC4R–as previously reported.<SEP>Meta-analysis of Effect Size for type 2 Diabetes at the CDKAL1 Locus<SEP>In the longitudinal follow-up, Cdkal1 −/− mice gained adipose tissues and showed the deterioration of insulin sensitivity after a certain period of extreme high-fat feeding (Figure 2C). Analogously in the human, as dietary habits substantially influence BMI in the population at large, genetic effects on type 2 diabetes, attributable to CDKAL1, may differ between populations (or ethnic groups) with distinct dietary habits and the resultant cross-population diversity in BMI. To evaluate this possibility, we plotted effect sizes for type 2 diabetes against BMI in the control group (as the population mean data) for the individual case-control studies that investigated disease association at CDKAL1 (Figure 7). There was a tendency toward inverse correlation between the 2 tested variables. As a consequence, we identified significant cross-population heterogeneity in effect sizes (I2 = 96.4%, P<1.4×10−7) between East Asians (a relatively thin population, mean BMI = 23.3 kg/m2; OR = 1.27, 95% CI 1.23–1.30) and Europeans (a relatively stout population, mean BMI = 26.4 kg/m2; OR = 1.14, 95% CI 1.10–1.17) at CDKAL1 (Table S5).",
                  "Discussion<SEP>In Cdkal1-deficient mice, we tested the hypothesis that changes in the activity of Cdkal1 result in islet dysfunction and/or insulin resistance, thereby contributing to the pathogenesis of type 2 diabetes. Our results indicate that deletion of the Cdkal1 gene results in a rather mild metabolic phenotype except a modest decrease in weight (Figure 2A) and a tendency toward enhanced insulin sensitivity (Figure S4C and S4D) on a standard diet. Of note is the fact that, whereas Cdkal1-deficient mice gain less fat (Figure 2C and Figure S3) and have less impaired glucose tolerance and insulin sensitivity (Figure 2C and Figure S4G and S4H) during the first 8 weeks of high-fat diet, their glucose tolerance and insulin sensitivity deteriorate after 20 weeks of fat enriched-diet compared to wild-type littermates (Figures 2C and 3A-3C). Together with the independent observations in human association studies, we speculate that Cdkal1 modulates whole-body glucose metabolism in a bidirectional manner; that is, a lack of Cdkal1 enhances insulin sensitivity presumably in the skeletal muscle, adipose tissue and/or liver, and impairs insulin secretion in the pancreatic islets. These compensatory mechanisms become evident with dietary intervention and may protect the mice against glucose intolerance or overt diabetes in the first place. However, after a certain period of extreme high-fat feeding, Cdkal1 −/− mice come to gain fat, to show deterioration of insulin sensitivity, and eventually to exhibit apparent glucose intolerance.<SEP>Molecular variants that linked the human CDKAL1 gene to increased type 2 diabetes susceptibility are located in the middle of a large intron of CDKAL1. It remains unknown whether the causal variant(s) can activate or attenuate the function of CDKAL1. Thus far, several human studies have indicated that the risk variant of CDKAL1 is associated with reduced insulin secretion. Besides the circumstantial evidence for reduced insulin secretion after a certain period of high-fat feeding in Cdkal1 −/− mice (Figures 4B and 4C), two lines of evidence have supported our hypothesis that deletion of Cdkal1 in mice impairs glucose tolerance and/or insulin secretion. First, by using β cells derived from Cdkal1 −/− mice, we have verified that the number of fusion events during the first-phase insulin release is reduced in vitro, thereby leading to impairment of insulin exocytosis in Cdkal1 −/− mice. Also, it has been recently reported that pancreatic β cell-specific knockout mice show a decrease in insulin secretion and impaired blood glucose control, resulting from a reduction of glucose-stimulated proinsulin synthesis.<SEP>The most notable characteristic of Cdkal1 −/− mice is the reduced fat accumulation with high-fat-fed intervention, accompanied by protection against insulin resistance (or in other words, enhanced insulin sensitivity). Although body weight tended to be lower in Cdkal1 −/− mice on a standard diet throughout the follow-up period (Figure 2A), an inter-strain difference in %intraabdominal and %subcutaneous fat and lipid accumulation in the liver and muscle became most prominent after 8 weeks on a high-fat diet (Figure 2C and Figure S3A-S3C). After 16 weeks on a high-fat diet, when the inter-strain difference in %intraabdominal and %subcutaneous fat was no longer noticeable, insulin sensitivity assessed by a homeostasis model assessment of insulin resistance (HOMA-IR) was still significantly better in Cdkal1 −/− mice (P<0.05, n = 5 for each strain) (Figure 2C). This enhanced insulin sensitivity is likely to surpass the potential impairment of insulin secretion and to result in a reduced impairment of glucose disposal during OGTT in Cdkal1 −/− mice as compared to wild-type littermates in the early phase (Figure S4E, S4F and S4J).<SEP>To discuss a hypothetical molecular mechanism underlying the metabolic phenotypes of Cdkal1 −/− mice, we performed microarray gene expression analysis on the muscle, which we chose as a primary target tissue because of the abundant expression of Cdkal1 (Figure 1B) and its physiological importance in energy expenditure. It has turned out that a relatively small proportion of genes are differentially expressed in the muscle between Cdkal1 +/+and Cdkal1 −/− mice at a fair cut-off level; e.g., 32 probesets were detectable by a filtering condition of |fold change|>1.5 and P<0.05 (Table S3), where we could not find strong evidence of enriched functionally-related genes. Among the 32 probesets, Scd1 is remarkable in that Scd1-deficient mice show the metabolic phenotypes in accordance with those observed in Cdkal1 −/− mice; e.g., decreased susceptibility to diet-induced obesity, decreased plasma insulin level, and improved glucose tolerance. Scd1 in adipose tissue is assumed to play a key role in the development of obesity. We therefore validated by quantitative reverse-transcription PCR a significant reduction in Scd1 mRNA expression in the muscle and WAT of Cdkal1 −/− mice at an early stage but not at a later stage of high-fat feeding (Figures 5A and 5B). In Cdkal1 transgenic mice, Scd1 mRNA expression tended to be elevated in a reciprocal manner (Figure S1). Moreover, while a previous study showed that Klf15 could suppress the expression of Scd1 in adipose tissues, we have not obtained the results supporting it, considering the direction of Klf15 expression changes in Cdkal1 −/− mice and Cdkal1 transgenic mice (Figure S6). Taken together, our data indicate that Scd1 might function as a critical mediator of the altered metabolism in Cdkal1 −/− mice. Further investigation is warranted to elucidate the network triggered by Cdkal1 deletion, including the potential mechanisms beneath the insulin-independent glucose uptake (Figure S5).<SEP>In terms of genetic correlation between heritable quantitative traits, body weight, BMI, and waist circumference (all of which could represent intraabdominal fat) were found to cluster near serum insulin levels by a hierarchical clustering method, suggesting the presence of an etiological relationship between the traits. This supports the bidirectional genetic effects of Cdkal1–reduced adiposity and increased susceptibility to type 2 diabetes. The complex interplay between insulin secretion, insulin sensitivity, and dietary habits in relation to glucose metabolism led us to hypothesize that type 2 diabetes risk loci including CDKAL1 could exert some protective effects on fat accumulation, which was represented by BMI in the present study. As genetic effects of Cdkal1 deletion on body weight reduction and protection against diet-induced obesity appeared to be rather modest in mice, we should be careful in extrapolating the findings in mice to humans. We identified a nominally significant (P<0.05) association between CDKAL1 rs4712523 and BMI in 2 Japanese general populations; the effect size (in β) was almost comparable to that at MC4R rs12970134, one of the most robustly confirmed BMI-associated loci, while the direction of BMI association was reversed (β = −3.38 and 3.08 for CDKAL1 and MC4R, respectively; Table S4). Among the loci thus tested, the largest effect size for BMI reduction was found at TCF7L2, where the OR for type 2 diabetes was also greatest in Japanese. In agreement with these findings, several previous studies in European populations have reported that the risk variant(s) of TCF7L2 can be associated with lower BMI, whereas a few studies (involving<5,000 participants) have not detected significant association between measures of fatness and genetic variants of type 2 diabetes risk loci except FTO and MC4R; here, the risk variants of these 2 loci increase both type 2 diabetes susceptibility and BMI (Figure 6A). Large-scale studies of general populations are needed in multiple ethnic groups to gain more insight.<SEP>Another issue of note is the adiposity (or fat accumulation)-related heterogeneity in patterns of type 2 diabetes susceptibility. As a whole, there appears to be a tendency of inverse correlation between the OR for type 2 diabetes at CDKAL1 and BMI (Figure 7). Along this line, a few previous studies in Europeans have shown that some genetic variants acting on insulin secretion (e.g., TCF7L2) have a greater impact on type 2 diabetes in non-obese subjects than in obese subjects. Dietary habits (i.e., non-genetic or environmental factors) and obesity susceptibility can collectively and interactively exert influences on BMI (or degree of obesity). In Cdkal1 −/− mice, reduced fat accumulation is evident at the early stage but becomes inconspicuous after a certain period of high-fat feeding. Accordingly, it is possible that excess dietary fat intake, e.g., the Western diet, results in the attenuation of the genetic effect of Cdkal1 on reduced fat accumulation (or BMI) within the population. Type 2 diabetes risk conferred by CDKAL1 variant(s) can be appreciably modified by BMI (Figure 7) as a consequence of a gene–environment (or diet) interaction, in relation to ethnic-group-specific dietary habits, e.g., a Western diet vs. an East Asian diet.<SEP>It has been shown that a high fat–fed C57BL/6J mouse model is suitable for studies on islet dysfunction in combination with insulin resistance. Compensatory adaptations of insulin secretion for insulin resistance have been reported to change over time in the high fat–fed mouse model during long-term (∼40 weeks after initiation of high-fat diet) studies. That is, while there was no significant compensatory increase in the early insulin response to glucose in the high-fat diet after 4 weeks, the compensation (although insufficient for the total requirement) appeared after 12 weeks and persisted throughout the study period. Thus, considering the coexistent, enhanced insulin sensitivity in addition to the assumed compensation in Cdkal1 −/− mice, it is reasonable that long-term high fat–fed intervention, i.e., 20 weeks of high-fat feeding in the present study, was required to demonstrate the exaggerated glucose intolerance by OGTT, which could result from the eventual imbalance of compensatory mechanisms.<SEP>There are several limitations in the present study. Above all, detailed mechanisms linking a global Cdkal1-null mutation to fluctuations in fat accumulation remain to be defined. One plausible explanation is that the decreased food intake, resulting from the suppression of appetite, can drive the metabolic changes observed in Cdkal1 −/− mice. However, the decreased food intake alone cannot account for fat and body weight increases in a later stage of high-fat feeding. It is therefore possible that the effect on fat accumulation is partly independent of the food intake, although a pair feeding experiment is required to verify this. Also, further investigations of the network involving Scd1 are warranted to gain mechanistic insights into the altered insulin sensitivity in Cdkal1 −/− mice.<SEP>In summary, the present data show that Cdkal1 gene deletion is accompanied by modestly impaired insulin secretion and longitudinal fluctuations in insulin sensitivity during high-fat feeding in mice. As observed in the early phase, the compensatory mechanisms are appreciably modified by diet. In this context, data presented from two Japanese populations support the opposing effects of CDKAL1 variants on BMI and type 2 diabetes.<SEP> Note added in proof: Since this manuscript was submitted, two BMI meta-analyses in East Asians were published. In these studies, a highly significant (P<1×10−10) association was detected for CDKAL1, in accordance with the present study in Japanese.",
                  "32809",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "29",
                  "21379325",
                  "Introduction<SEP>Genome-wide association studies on body mass index (BMI) and adiposity have reliably identified its association with variation at the fat mass and obesity related locus (FTO) in adult and child populations. In meta-analyses, the addition of each minor (A) allele at the single nucleotide polymorphism (SNP) rs9939609 within the first intron of FTO has been shown to be associated with a higher BMI of up to 0.33 kg/m2 or approximately 0.1 standard deviations. The biological mechanisms behind this association are yet to be fully determined, however evidence from both population based analyses and functional investigations have suggested that this locus is likely involved in the hypothalamic regulation of appetite or energy expenditure and metabolic rate. Indeed, following a series of investigations noting the correlation between differential Fto expression, fat mass, food consumption and of raised FTO mRNA levels in the subcutaneous adipose tissue of obese individuals, ubiquitous overexpression of Fto has been shown recently to lead to a dose-dependent increase in body and fat mass irrespective of diet type. Coincident observation of increases in dietary consumption, reduced leptin levels and further studies showing high expression levels in cerebellum, hippocampus and hypothalamus, point towards an important role for Fto/FTO in the regulation of dietary intake.<SEP>Until recently, most replication efforts concentrating on variation in FTO have employed singlepoint analyses of cross-sectional data. These have been conducted from ages as low as 2 weeks to old age (>70 years) and have demonstrated, with differing degrees of reliability, associations between variation at FTO, BMI and related traits. Although limited, available evidence suggests that the cross-sectional FTO/BMI association varies by age. Specifically, at early ages up to and around 7 years, the association between common variation at FTO and BMI appears to be reduced in magnitude, with smaller studies being unable to detect association. This pattern then changes in early adulthood with peak effect sizes (approximately 0.3kg/m2/minor allele) occurring by age 20. Following this peak, this association appears to diminish absolutely (not relatively) in a manner one would expect for coincident reduction in adiposity levels with later age.<SEP>Haworth et al simultaneously examined the FTO/BMI association and the heritability of BMI in a longitudinal twin collection with data at ages 4, 7, 10 and 11 years. They found that the association between BMI and variation at FTO was age dependent, that the heritability for BMI increased with age and that the proportion of variance in BMI explained by shared environment diminished over the same period. Consistent with the idea that BMI and adiposity related traits may be determined by a complex interplay between genetic and environmental features, these findings suggest that with age and dietary autonomy, loci such as FTO may be able to exert a greater effect on BMI. A Finnish twin study has since suggested an increase in the heritability of BMI throughout adolescent years.<SEP>Individuals vary considerably in their rate of growth and differ in their rate of development so that some children mature faster and reach milestones such as puberty earlier. These two processes can be distinguished from each other. For example most inter-individual variability in pubertal height velocity can be explained in terms of developmental age/timing and adjusting for this allows their growth velocity curves to be superimposed. So if developmental age explains variability in pubertal height, might it also explain BMI variability in childhood?<SEP>The timing of BMI adiposity rebound (AR) is inversely related to the risk of later obesity, and it is also positively correlated with the timing of puberty. In terms of adiposity, the age of AR can be viewed as a developmental marker and an early AR implies an advanced developmental age. This suggests that variation in the pattern of BMI development may arise from differences on the developmental age scale that are both independent of and in addition to differences on the BMI scale. In fact, there is good evidence to suggest that different processes are involved in the pattern of high BMI throughout life as opposed to high adult BMI preceded by average or low BMI in infancy followed by early AR.<SEP>Despite the published work on longitudinal differences in the associations between the variation at FTO and BMI or related traits, lack of dense lifecourse data and inadequate statistical power has made interpretation of findings difficult. In this investigation, we aimed to assess the relationship between variation at rs9939609 and changes in BMI from after birth until 13 years of age, by meta-analysing data from eight cohorts. We also aimed to explore the possibility of differences in developmental age between FTO genotypes by scaling the ages for each genotype appropriately, so as to minimise the differences in the pattern of BMI development through childhood. Lastly, we aimed to fit individual growth curves in order to explore directly the relationship of this variant to critical change points in BMI; all features known to be related to health in later life.",
                  "Materials and Methods<SEP>To examine the association between rs9939609 genotype and BMI from birth to 13 years of age we used the growth measurements from eight studies (Table 1). All subjects were unrelated children of white European ancestry, with multiple births excluded. When multiple siblings were present, only data from the oldest sibling were used. All studies have previously been described in detail, but brief descriptions are given below.<SEP>The Avon Longitudinal Study of Parents and Children (ALSPAC) is a prospective birth cohort in Bristol, UK, which recruited pregnant women with expected delivery dates in 1991–1992 (present analysis: 7,482 subjects). The Barry Caerphilly Growth Study (BCG) is a longitudinal study of infants born in the towns of Barry and Caerphilly in South Wales between 1972 and 1974 (569 subjects). The Christ's Hospital Cohort (CH) is a retrospective follow-up study comprised of former male students between the ages of 10 and 18 years of Christ's Hospital School born between 1927 and 1956 (812 subjects). As part of the Energy Balance Study (EBS), data were collected in 2002 and 2003 on pre-pubertal schoolchildren, ages 4 through 10 years, from north-eastern Scotland (2,604 subjects). The Generation R Study (GENR) is a prospective birth cohort from early foetal life onwards based in Rotterdam, the Netherlands; subjects born between 2002 and 2006 (2,851 subjects). The Northern Finland Birth Cohort 1966 (NFBC1966) is a prospective pregnancy/birth cohort with expected deliveries in 1966 in the two northernmost provinces of Finland (3,707 subjects). The Raine Study (RAINE) is a prospective pregnancy cohort set up in 1989, which recruited pregnant women from Perth, Western Australia for ultrasound imaging (1,106 subjects). The Uppsala Family Study Study (UFS) is a multigenerational study set up in 1995 in Uppsala, Sweden (594 subjects).<SEP>In all studies, weight and height were measured during routine visits at community health centres or research centres. All subjects (or their parents/guardians) gave informed consent and each study obtained ethical approval from the local ethical review board. For further details please see Text S1.<SEP>Genotyping and quality control<SEP>Genotyping of the rs9939609 was performed directly in all eight cohorts. DNA was isolated either from buccal swabs, blood or cord blood. Further details of the studies and of genotyping undertaken in them can be seen in the Text S1.<SEP>Analytical strategy and statistical analysis<SEP>Cross-sectional analyses<SEP>BMI was defined as weight [kg]/(height [m])2. For the cross-sectional meta-analysis, growth measurements were grouped into ten strata: 0.01 to <0.5 years (i.e. excluding birth); 0.5-; 1.5-; 2.5-; 3.5-; 4.5-; 5.5-; 7-; 9-; and 11 to <13 years. To approximate normality in each stratum BMI was natural log transformed before analysis, and effects may be expressed as percentage changes through multiplication by 100. To remove outliers stratum-specific Z-scores were created using the “zscore” package in STATA (version 11, Stata Corp. Texas, USA) and values exceeding ±3 were excluded (height and weight were cleaned similarly). To examine the association between FTO genotype and BMI within each stratum multivariable linear regression was used. Study-specific effect estimates within each stratum were created assuming an additive genetic model. These models were adjusted for age because age varied within each age-stratum and sex. No further adjustment was done as variation at rs9939609 is unrelated to birth weight or gestational age and the distribution of genotypes is assumed to be unrelated to possible environmental confounders. To take into account known correlations between BMI and height at early ages, a sensitivity analysis was done using weight/heightp (p ranging from 1.7 to 2.8), with estimated stratum- and sex-specific powers p for each study. Basic analyses were performed in Stata 11 (Stata corp.).<SEP>Cross-sectional cohort-specific results were meta-analyzed within each stratum using a random effects model to account for the existence of heterogeneity between studies. Analyses were performed using the “metan” package in STATA (version 11, Stata Corp. Texas, USA).<SEP>Curves of median BMI by age were estimated for each of the three genotypes, using the LMS method to adjust for age-specific heteroscedasticity and skewness. This was implemented with the “gamlss” package in R (version 2.11.1). The median (or M) curve was modelled as a penalized B-spline in log age with 9 degrees of freedom, with corresponding curves for the coefficient of variation (S) with 5 d.f. and skewness (L) with 3. Study (as an eight-level factor) and sex were also adjusted for. Data points with age <0.1 years were excluded to improve model convergence. To illustrate the effect on the curves of a k% age scaling per A allele, scaled age was defined as , where n(A) is the number of A minor alleles (0, 1 or 2), and C is such that mean age(k) = mean age. This ensured that the age scale was foreshortened by k% in the AA group and stretched by k% in the TT group, each relative to TA, while maintaining the same mean age.<SEP>It is not possible to estimate the optimal age scaling  explicitly by linear regression. Instead a series of six LMS models was fitted with age scaled by k = 0, 1 … 5%, each with a single median BMI curve fitted as before as a penalized B-spline curve in log age(k) with 9 d.f., adjusted for study and sex, and S and L curves with respectively 5 and 3 d.f. The fitted deviance dev(k) for each model was plotted against k and a quadratic in k fitted such that . By differentiation the optimal scaling  was given by  with 95% confidence interval  where  and  . The model was refitted at  for confirmation. As a further stage this age-scaled analysis was repeated including an additive log age(k) by minor allele interaction, and the value of  re-estimated.<SEP>Longitudinal analyses<SEP>Modelling BMI longitudinally is complex due to the adiposity peak in infancy and the increasing population variance in BMI throughout childhood. For this reason the data were split into two age windows: 2 weeks to 18 months (infancy) and 18 months to 13 years (childhood), using the studies with the most data in these age windows (BCG, GENR, NFBC1966 and UFS in the first and ALSPAC, RAINE, NFBC1966 and Uppsala in the second). BMI measurements in the first two weeks of life were excluded to avoid the period of weight loss after birth. The change point between infancy and childhood was set at 18 months primarily on statistical grounds to lie roughly mid-way between AP and AR.<SEP>Derivation of age and BMI at adiposity peak (AP) and adiposity rebound (AR)<SEP>Age and BMI at AP and AR were derived from cubic models in age for the two age groups separately, with random effects for the intercept and slope terms. Sex was also adjusted for, but rs9939609 genotype was ignored in order to estimate the outcomes for individuals and to then relate them to FTO. Subsequently, sex interactions with linear and quadratic age were added to the childhood model (both interactions p<0.01 in ALSPAC and NFBC1966). The models are written as:<SEP>Infancy model:<SEP>Childhood model:<SEP>For each participant, predicted BMI at AP and AR (on a grid of every 0.05 years in infancy and every 0.1 years in childhood) was calculated using the estimated fixed and random coefficients. Age at AP was defined as the age at maximum BMI between 0.25 and 1.25 years, and age at AR as the age at minimum BMI between 2.5 and 8.5 years. These cut-off points were chosen based on descriptive analysis of growth curves in the NFBC1966. The associations between rs9939609 genotype and these growth parameters were analyzed using both general and additive genetic models. To account for uncertainty in the derived parameters, each person's data were weighted by the number of measurements within the age window, and those with fewer than three were excluded. Sensitivity analyses with gestational age as a further adjustment in the AP models made no substantive difference to the results (performed in the NFBC1966). Age at AP and age at AR were analysed without transformation, but are presented as percentages for comparison with the cross-sectional results. BMI at AP and AR was log-transformed due to right skewness, and association results are reported as percentage differences in BMI between genotypes by multiplying the log differences by 100. Study-specific association results between each growth parameter and FTO were meta-analyzed using random effects models to account for the existence of heterogeneity between studies using the “metan” package in STATA (version 11, Stata Corp. Texas, USA).",
                  "Results<SEP>Cross-sectional results<SEP>For the eight studies in the cross-sectional meta-analysis of the association between rs9939609 and BMI, the average sample size per age stratum was 9143. Table 1 gives the stratum and study-specific subject characteristics. Genotypic frequencies at rs9939609 were broadly consistent across studies and in accordance with expectations for population samples of European origin. All genotypic sampling adhered to Hardy Weinberg equilibrium (Table S1).<SEP>In meta-analyses above the age of 5.5 years (childhood) the minor allele (A) was associated additively with a higher BMI, though this was not detectable in the age stratum 11 to 13 years where the sample was small and where age associated increase in variance compromises analytical power. Expressed as a percentage change, the additive effect of each minor allele (A) was 0.7% (95% CI: 0.3, 1.1), 1.0% (95% CI: 0.6, 1.3), and 1.3% (95% CI: 0.6, 2.0) at 5.5–7, 7–9 and 9–11 years respectively. Maximum heterogeneity was high with I2 = 69.6% (95% CI: 22, 88). In contrast to this, each minor allele was associated with a lower BMI before the age of 2.5 years. The additive effect of each minor allele was −0.4% (95% CI: −0.6, −0.1), −0.3% (95% C%I: −0.6, −0.1) and −0.3% (95% CI: −0.5, 0.0) at age 0–0.5, 0.5–1.5 and 1.5–2.5 years respectively. Between 2.5 and 5.5 years there was no association between rs9939609 genotype and BMI. For these periods, maximum heterogeneity was high with I2 = 44.1% (95% CI: 0, 81). Figure 1 shows meta-analysis results representing major observations throughout the age range. Similar results (not shown) were found for weight/heightp.<SEP>To clarify the age trends in BMI for each genotype, median curves were estimated using the LMS method to adjust for age-specific heteroscedasticity and skewness. Figure 2 shows the median curves by genotype, where comparison shows three distinct features: (i) BMI is higher for A carriers later in childhood, but lower early in childhood; (ii) AR is earlier for A carriers; (iii) the A allele effects are additive in that the TA group is consistently midway between AA and TT. Curves for weight and height (Figure 3) show genotype differences for weight that emerge only after age 4, and height differences that are small at all ages. The differences in age at AR (Figure 2) can be removed by estimating a developmental age scaling effect per A allele of 2.79% (95% CI: 2.35, 3.23), such that the age scale in the AA group is shrunk, and in the TT group stretched, by 2.79% relative to TA. However there remains a rising BMI trend in AA relative to TT which is confirmed by fitting a log age by genotype interaction (coefficient 0.039 kg/m2 (0.031, 0.046) per log age unit per A allele, p = 10−23). Fitting the interaction reduces the optimal age scaling slightly to 2.37% (95% CI: 1.87, 2.87, p = 10−20), but provides evidence suggesting that the two processes together, developmental differences on the age scale and on the BMI scale, explain the complex age-related genotype effects on BMI in childhood. Figure 4 shows the estimation of this optimal scaling (top) and the curves of Figure 2 scaled by this amount (bottom), where now AR occurs at the same developmental ‘age’ for all groups. For comparison the optimal age scale to adjust for genotype differences in height (Figure 3) is 0.3% (0.2, 0.4), a value attaining statistical confidence, but far smaller than for BMI. Thus FTO appears to affect BMI developmental age much more than it does height developmental age.<SEP>Longitudinal results<SEP>We analysed the richest data sets longitudinally for age and BMI at adiposity peak (AP) and adiposity rebound (AR). Mean ages at AP and AR were 0.75 (0.73, 0.78) and 5.70 years (4.56, 6.84) and mean BMIs 17.87 (17.11, 18.62) and 15.73 kg/m2 (14.66, 16.80) respectively. There was weak evidence for a lower BMI at AP only in the carriers of two minor alleles (AA) compared to the reference group (TT): −0.40% (95%CI: −0.74, −0.06), p = 0.02 (Table 2). In contrast, there was evidence a per minor allele difference in BMI of 0.47% (95%CI: 0.17, 0.77) at AR. This was realised as carriers of two minor alleles (AA) having a 0.93% (95%CI: 0.22, 1.64), p = 0.01 higher BMI at AR than those in the reference group (TT) (Table 3). There was no evidence for genotypic association with age at AP, but there was evidence for an additive relationship between carriage of minor alleles (A) at rs9939609 and earlier AR. Per minor allele, there was a −2.31% (95%CI: −3.05, −1.57) difference in age at AR. Indeed, carriers of one minor allele showed −2.28% earlier AR (95%CI: −3.90, −0.65), p = 0.006 and carriers of two minor alleles −4.72% earlier AR (−5.81, −3.63), p = 10−17 versus baseline genotype TT). There was evidence of heterogeneity between cohorts, lower for AP (maximum I2 30% (0, 74)) than for AR (I2 67% (2, 89)). The estimated effect on age at AR, and hence the scaling of developmental age, was very similar with the cross-sectional and longitudinal data; 2.4% and 2.3% age shrinkage per allele respectively.",
                  "Discussion<SEP>We present an investigation into the association of rs9939609 with BMI in infancy and childhood. Results here not only confirm the increasing magnitude of associations between this variant and adiposity from the end of infancy through to childhood, but also suggest an inverse association at early ages—an event intimately linked with the timing of AR. With resolution afforded by a large sample size and dense anthropometry data, we have been able to demonstrate that rather than a null association between the rs9939609 adult adiposity associated variant and BMI before the age of 2.5 years, each extra minor (A) allele at this locus is associated with lower BMI. After the age of 5.5 years a positive association between rs9939609 and BMI emerges as seen in previous work. However this study reveals further details of the association by examining the genotype-specific trends. Thus the two additive effects of the A allele are to accelerate developmental age by around 2.4% per allele, corresponding to an early AR, and at the same time to increase BMI accretion by about 0.1 kg/m2 per allele from 1 to 13 years. This is confirmed in longitudinal modelling of our richest data. High levels of adult BMI may be preceded by either raised BMI throughout childhood, or alternatively normal or low BMI during infancy with an early AR and steep climb to high adult BMI. Results from this work suggest that, rather than predisposing to elevated adipose levels across the lifecourse, variation at FTO is associated with a shift in the timing of AR, and is entirely consistent with both the contrast between patterns of association between infancy and childhood and the later life BMI associations by which this locus was discovered.<SEP>The changing associations between rs9939609 and BMI over the course of infancy and childhood are consistent with what is known about the biology of this locus. It has been suggested previously that energy balance through early life may exert an influence on the timing of AR. If FTO is operating through an influence on appetite and the amount of food consumed, then it may be that as individuals are able to autonomously regulate dietary intake (and other relevant behavioural traits such as activity), realised differences in appetite will have an impact on the age at AR. Alternatively, before this period, it may be that differential metabolic activity according to genotype exerts influence on the patterns and timing of BMI change. These possibilities reflect an anticipated interplay between genetic and environmental factors supported also by the only simultaneous study of the heritability and genetics of BMI to date. Beyond this and without further functional understanding as to the action of the FTO locus, it is difficult to speculate as to the mechanism by which variation at this locus might lead to earlier AR.<SEP>Considering the implications of these findings, rapid early weight gain is a known risk factor for later obesity and since weight gain is calculated as weight increment divided by time interval, shortening the developmental time interval has the same impact on gain as increasing the increment. More directly, earlier AR has been consistently associated with higher BMI and the risk of obesity in later life. Furthermore, this relationship has been shown to be both incremental and predictive of downstream risk of diseases such as T2D and coronary heart disease. Overall, whilst the impact of AR being ∼5% earlier for AA carriers may be relatively small, the ultimate impact on BMI trajectory may have important lifecourse effects which go some way to explain the known associations between this genotype and the binary phenotype “obesity”. Secondary effects of such relationships may also be seen in features such as early puberty which is known to be associated with greater adiposity and corresponds to an advanced developmental age. This is in accordance with our second finding, that in addition to the earlier AR associated with minor allele carriage, BMI in this group is low early on, but subsequently rises faster. Thus we predict that suitably powered studies of BMI around puberty will show the minor allele at rs9939609 associated both with earlier puberty and greater adiposity.<SEP>Although FTO impacts on developmental age it is important to stress that it applies to BMI and not to height. There are only minor differences in height growth between the three genotype groups (Figure 3), and they correspond to an age scaling per A allele of just 0.3%, which is only an eighth of the BMI age scaling effect. Thus it is not a generalised maturation effect but specific to adiposity accretion.<SEP>Despite the strengths of this study, it has limitations. Firstly, the definition of age windows in longitudinal analysis may not be optimal. There are still age groups for which we have limited data and this is reflected in the sampling error associated with these periods of growth and development. Furthermore, although the ability to examine the influence of this locus at different ages is aided by an analysis of many samples, the stratum specific estimates and their error terms are subject to the different measurement techniques and ages. The availability of further collections with broad age ranges and dense growth data and genotyping would increase the precision of findings documented here.<SEP>Secondly, a possible complication to the patterns of association seen between common variation at the FTO gene and BMI relates to the interplay between maternal and offspring genotypes. Whilst not within the bounds of this paper, the observation that mothers with greater BMI have, on average, offspring with greater birthweight who go on to be larger may be relevant in our interpretation of results. Owing to the correlation between maternal and foetal genotypes, one may hypothesise that, on average, the elevation of adiposity in mothers carrying minor alleles at rs9939609 may translate to increased levels of birthweight or differential growth and development in early ages as shown observationally. This would theoretically counter the inverse association between minor (A) alleles at rs9939609 and BMI in offspring at very young ages that we have documented. Whilst this does not appear to be occurring, proper examination of this requires further large collections with available maternal genotypes (Table S2).<SEP>Lastly, the value of BMI as an assessment of adiposity at early ages has been questioned, although BMI is still commonly used. In this investigation, we performed sensitivity analyses using the derived measure weight/heightp  to account for this limitation and found that results were largely consistent with weight/height2. For this reason and for consistency with later ages in childhood, we adopted the use of BMI throughout.<SEP>Overall, we conducted a large analysis of the association between common variation at FTO locus and BMI. We have noted that the effect of this locus appears to be age dependent. Importantly our results suggest an inversion of the known adult association between this locus and BMI at ages below 2.5 years, an observation which is consistent with relationships between variation in FTO and the timing of AR and which may help develop understanding of the biological mechanisms behind the association between common variation at this locus, adiposity related traits and disease risk. Further, specific, analyses will be required to confirm the age dependent associations and to investigate the clinical implications of associations between common genetic variation BMI and the timing of AR.",
                  "26780",
                  "['OBWCCE']"
                ],
                [
                  "30",
                  "34208143",
                  "1. Introduction<SEP>Obesity is a highly prevalent chronic health problem associated with an increased risk of type 2 diabetes mellitus, heart disease, metabolic syndrome, hypertension, stroke, and certain forms of cancer. This problem affects over 340 million children and adolescents. The prevalence of obesity has increased approximately from 4% to 18% in the last 40 years worldwide. Furthermore, it is estimated that the number of obese individuals will reach 1.12 billion or 20% of the world’s adult population in 2030.<SEP>The childhood obesity period represents a priority public health problem associated with developing chronic metabolic diseases. The prevalence increased in both developed and developing countries. In Chile, childhood obesity has increased steadily since the late 1980s. In 2019, the National Board of School Aid and Scholarships of the Government of Chile (JUNAEB) reported that the prevalence of overweight and obesity at different school levels ranges from 47.6% to 60%. In the development of obesity, it is essential to keep in mind modifiable and non-modifiable risk factors. Non-modifiable factors include sex, age, ethnicity, Tanner stage, and genetic polymorphisms. Indeed, Frayling et al. showed that the rs9939609 polymorphism of the FTO gene (Fat Mass Obesity-associated gene) was associated with the body mass index (BMI). After this research, more than 97 genetic polymorphisms associated with a higher BMI had been reported, and a meta-analysis summarized the existence of 738 SNPs associated with various adiposity markers. Despite the large number of genetic variants related to obesity, the rs9939609 polymorphism of the FTO gene has been more studied due to its effect on increasing BMI. This association has been confirmed in different populations of adults and children worldwide.<SEP>The prevalence of metabolic syndrome (MetS) in childhood is increasing; likewise there is a rise in obesity and metabolic diseases. According to the Joint Task Force statement, MetS has been diagnosed with three of these five risk factors: abdominal obesity, high blood pressure, hypertriglyceridaemia, low HDL, and fasting hyperglycemia. Currently, there are no consensus guidelines for metabolic syndrome in the pediatric population. Concerning the association of metabolic syndrome with polymorphisms of the FTO gene, a meta-analysis concluded that the FTO gene might play a critical role in leading to MetS.<SEP>Previously, our group reported that the rs9939609 variant of the FTO gene was associated with a higher risk of increased BMI in Chilean children, and recently we reported the association of this same variant of the FTO gene with increased general and central adiposity. However, we did not analyze whether the risk of metabolic syndrome is modified with this variant of the FTO gene in this sample. Consequently, the objective of this study was to investigate the association between the rs9939609 in FTO gene and metabolic syndrome and to evaluate the components of metabolic syndrome in this same Chilean child population.",
                  "2. Materials and Methods <SEP>2.1. Design, Population, Sample<SEP>We used data from a cross-sectional study in children sample from an urban area in the Biobío Region of Chile. For an expected MetS prevalence of 22.7%, an accuracy of 6%, and a 95% confidence interval, the calculated minimum sample size was 188. Finally, 220 children were studied. Children from an age range of 6 to 11 years were included. Children suffering from any chronic pathology and those whose parents or legal guardian did not sign the informed consent were excluded.<SEP>2.2. Study Variables and Measurements<SEP>MetS was diagnosed following the criteria suggested by Cook. These define MetS by the appearance of three or more of the following alterations: waist circumference (WC) ≥ 90th percentile, systolic blood pressure (SBP) or diastolic (DBP) ≥ 90th percentile, high-density lipoprotein cholesterol (HDL-C) ≤ 40 mg/dL, triglycerides (TG) ≥ 110 mg/dL, but at fasting blood glucose levels ≥ 100 mg /dL, the value defined by the American Diabetes Association was considered. <SEP>In addition, a set of independent variables were collected: age (years), sex (woman/man), weight (kg), height (cm), BMI (kg/m2), waist–height index (WHtR), fat-free mass (FFM, kg), fat mass (FM, kg), body fat percentage (BF%), systolic (SBP, mmHg) and diastolic blood pressure (DBP, mmHg), glucose (mg/dL), high-density lipoproteins cholesterol (HDL-C, mg/dL), low-density lipoproteins cholesterol (LDL-C, mg/dL), TG (mg/dL), and total cholesterol (mg/dL).<SEP>Anthropometric were measured following the criteria of the standardized anthropometric manual. Height was assessed without shoes, using 0.1 cm precision wall-mounted stadiometers (Seca, model 208). Bodyweight was measured with light clothing and without shoes on a Tanita scale (TANITA TBF-300, TANITA, Tokyo, Japan) with an accuracy of 1 g. The WC was taken at the midpoint between the last rib and the upper border of the iliac crest with a non-elastic flexible tape (Seca, model 201) with an accuracy of 0.1 cm. Body composition was evaluated by bioelectrical impedance analysis (TANITA TBF-300, TANITA, Tokyo, Japan). Blood pressure was taken following established recommendations, using a calibrated digital sphygmomanometer (OMRON M3, OMRON, Kyoto, Japan). Blood pressure was measured three times, on the right arm, with the child seated, with an interval of 5 min between each measurement and using the mean of the last two measures. Trained nutritionists took all measurements. Each measurement was repeated three times, and the mean was registered. According to the Tanner criteria, the pubertal stage was established by a pediatrician.<SEP>BMI was calculated as body weight divided by height in meters squared. The BMI z score, based on age and sex, was calculated according to the WHO definitions. The children were classified following the WHO criteria as normal weight: Z-Score BMI> − 2DE and < + 1SD; overweight: Z-Score IMC> + 1SD or < + 2SD; Obesity: Z-Score BMI> + 2SD. Determinations of abdominal obesity and arterial hypertension were made using the reference tables according to age and sex, considering height in blood pressure.<SEP>2.3. Plasma Determinations<SEP>We collected 4 mL of fasting venous blood after an overnight fast, between 8–12 h of fasting. We used colourimetric methods to measure total cholesterol, HDL-C, LDL-C, TG, and glucose levels with analysis kits (Cobas C111 Roche, Indianapolis, IN, USA). Plasma insulin was measured using a commercial ELISA kit (Linco Research, St. Charles, MO, USA) using a multi-reader, Synergy 2 (Biotek, Winooski, VT, USA).<SEP>2.4. Determination of Allelic Variants of the FTO Gene<SEP>The single nucleotide polymorphism (SNP) (rs9939609) of the FTO gene was determined using genomic DNA extracted from leukocytes, using the Mini Kit QIAamp DNA Blood (Qiagen GmbH, Hilden, Germany), according to the manufacturer’s protocol. Polymerase chain reaction (PCR) amplifications were performed on a real-time PCR thermocycler, Rotor-Gene 6500 (Corbett Research, Sydney, Australia). For this, previously described primers were used (direct: 5′d AACTG GCTCTTGAATGAAATAGGATTCAGA 3′ and inverse: 5′d AGAGTAACAGAGACTATCCAAGTGCATCAC 3′), under a standardized protocol. Genotype identification was performed by comparison (confidence intervals, 95% CI) of the fusion data with standard genotypes identified by sequencing analysis in the Department of Ecology, Faculty of Biological Sciences at Pontificia Universidad Católica de Chile. To confirm the existence of a single PCR product, 3% agarose gel electrophoresis was performed. All analyses of the samples were performed in duplicates, with 98% success in determining the genotype. <SEP>2.5. Ethical and Legal Aspects<SEP>We conducted the study following the Declaration of Helsinki (1964), in the Convention of the Council of Europe relating to human rights and biomedicine (1997), in the Universal Declaration on the human genome and human rights (UNESCO, 1997), as well as fulfilling the requirements established in the Chilean legislation in the field of biomedical research, the protection of personal data and bioethics, according to Decree No. 114 of 2010, which approves the regulation of the Law No. 20,120, and which was modified and updated in Decree 30 January of 14 January 2013. The Bioethics Committee of the Vice-Rectory of Research of the University of Concepción approved the study protocol (352-2019).<SEP>2.6. Statistical Analyses<SEP>The continuous variables were presented as mean and standard deviation (SD), and categorical variables as absolute frequency and percentages. The goodness of fit of continuous variables to a normal distribution was tested using the Kolmogorov–Smirnov test with Lilliefors’ correction. For comparing more than two means, the analysis of variance (ANOVA) with Bonferroni for the post hoc comparison was performed. In case the data did not fit a normal distribution, the non-parametric Kruskal–Wallis version was computed. For categorical variables, the Chi-square test (2) was used. Prevalence ratios were calculated. The linear trend test was performed to determine significant differences in the outcome across the FTO genotype (the TT genotype was coded as 0 and used as the reference group, TA was coded as 1, and the AA was coded as 2).<SEP>The association between the FTO genotype and continuous outcomes included in or directly or indirectly related to MetS (Weight, WC, WHtR, BMI, FM, BF%, Insulin, HOMA-IR, TG, and HDL) was estimated through multiple linear regressions. Also, standard error, adjusted coefficient of determination, F statistic, linearity analysis, and residues were analyzed to determine the goodness of fit.<SEP>For all statistical analyses, an alpha error probability of less than 5% (p < 0.05) was accepted and the confidence interval was calculated at 95%. The software programs IBM SPSS Statistics 25.0 (IBM, Chicago, IL, USA), EPIDAT 4.2 (Department of Sanidade, Xunta de Galicia, Galicia, Spain) were used for the statistical analysis.",
                  "3. Results<SEP>3.1. Sample Description<SEP>The cohort characteristic is presented in Table 1. In summary, of the 220 children included in the study, 50% was females. The mean age was 9.1 (SD 1.2) years. The prevalence of obesity was 33.3%, whereas the prevalence of central obesity based on WHtR was 57.7%. The average BMI (z-score) was 1.52 (SD 0.83), and the body fat percentage was 30.0% (SD 9.4). The fasting glucose and insulin concentrations were 88.3 mg/dL (SD 8.56) and 8.6 μU/mL (SD 6.85), respectively. For lipids profile, the average total cholesterol concentration was 181.84 mg/dL (SD 33.1) and 108.26 mg/dL (SD 28.9) for LDL-C. For TG, the average concentration was 123.1 mg/dL (SD 73.87). Based on the Cook criteria for MetS, the prevalence of high blood pressure was 28.5% (95%CI, 22.654–34.947), abdominal obesity (WC > 90th percentile) was 59.1% (95%CI, 52.280–65.653), higher TG (TG > 110 mg/dL) was 47.7% (95%IC, 40.970–54.547), low HDL-C (HDL < 40 mg/dL) was 20% (95%IC, 14.924–25.903), and high glycemia (glucose > 100 m/dL) was 9.5% (95%IC, 5.979–14.158). Finally, the prevalence of metabolic syndrome was 26.8% (95%CI, 21.085–33.187).<SEP>3.2. FTO Genotype and Metabolic Syndrome<SEP>Furthermore, the presence of the AA variant of the FTO gene was related to a worse anthropometric and cardiometabolic condition. The children who presented it showed a higher average weight (p = 0.004), WC (p = 0.015), WHtR (p = 0.041), BMI (z-score) (p = 0.001) and BF% (p = 0.004). Concerning to metabolic state, those with two risk alleles of rs9939609 polymorphism had higher mean values of fasting insulin (p < 0.001), HOMA-IR (p = 0.004) and TC (p = 0.051). Also, this group showed significantly lower means of HDL-C (p = 0.001). Finally, this situation turned into a higher prevalence of MetS in this group (p = 0.006), with 44.7%, compared to 23% and 15% present in children with TT (Wild type) and TA (Heterozygous) genotypes, respectively. A more detailed description of these results can be observed in Table 2.<SEP>The presence of MetS components seems to be related to the type of alleles present in the rs9939609 polymorphism of the FTO gene of the participants (Table 2). Significantly different prevalence ratios were obtained for the HBP component (p < 0.001), WC (p = 0.006), fasting glucose (p = 0.004), and HDL-C (p < 0.001). Besides, significant linear trends were found in the prevalence ratios (Level 1: TT (reference), Level 2: TA and Level 3: AA), with WC (p = 0.001) increasing and the presence of the HDL-C component decreasing (p = 0.048) (Table 3).<SEP>Significant linear trends have also been found between the number of components and the genotype of the FTO gene. This trend is decreasing for children with none (p = 0.046) and increasing for those showing three or four components (p = 0.045 and p = 0.048, respectively). As a result, we found that this linear trend is also observed in the diagnosis of MetS, which is 2.21 (p = 0.006) higher in children with genotype AA (Table 4).<SEP>Furthermore, according to Tanner stage, adjusted multiple linear regression analysis by age, gender, and physical development showed that the presence of the risk allele of rs9939609 of the FTO gene significantly increases the probability of suffering from metabolic syndrome through its influence on its different components. In the case of AA genotype, results have shown a clear association with increased WC (β = 6.934; p < 0.001), TG (β = 42.629; p < 0.001), insulin resistance estimated through HOMA-IR (β = 1.010; p < 0.001) and a decrease in HDL-C levels (β = –6.912; p < 0.001). Harmful effects on the nutritional status of children have also been detected, with a significant increase in weight (β = 7.289; p < 0.001), BMI (β = 2.934; p < 0.001) and body fat percentage (β = 6.178; p < 0.001) of subjects with this genotype. Finally, children with the TA genotype were also at higher risk than those with the TT genotype for increased weight, WC, WHtR, BMI, BF%, and insulin resistance (Table 5).<SEP>The relationship between the different components of the metabolic syndrome, the presence or absence of this syndrome, and the FTO gene variant is shown in Figure 1. The means of TG are higher in children with MetS and with genotype AA, while those of HDL-C are lower in this group. Overall, glucose is lower in participants with MetS and even more so in those with the genotype AA. However, in this group, insulin levels are much higher, suggesting the activation of a compensation system that seems to be effective in this studied group.",
                  "4. Discussion<SEP>This research aimed to determine whether the different genotypes associated with the polymorphism rs9939609 correlate to an increased susceptibility to MetS in children. <SEP>Previous reports have shown the relationship between the A allele rs9939609 of the FTO gene and the increased risk of overweight and obesity in children, regardless of ethnicity, gender, or developmental stage. This risk, which is higher for homozygous (AA), is demonstrated in both general and central adiposity parameters. In other words, the presence of allele A is associated with a more significant deposit of fat, which is related to a higher number of cardio-metabolic complications. This fact is explained by the increase in adipose tissue, which is associated with higher insulin resistance (IR) and represents the primary physiopathological mechanism of MetS. Our results have shown that IR increases significantly between the different genotypes, being higher in AA (β = 1.010, p < 0.001). This trend leads to high insulinemia as a compensatory mechanism that prevents hyperglycemia in children. For this reason, high basal blood glucose is the least prevalent component of MetS (9.5%). Despite this, this criterion was more present among carriers of the genotype TA and AA (p = 0.004).<SEP>Although the increase of insulinemia limits the development of hyperglycemia, IR is involved in the rest of the MetS-linked disorders. Several authors have shown that IR affects lipid metabolism, altering HDL-C levels. Furthermore, it is involved in the genesis of hypertension. These alterations are more frequent in our sample than hyperglycemia, with 20% and 28.5%, respectively. In terms of distribution among genotypes, all MetS criteria were significantly more prevalent in AA, except in hyper-TG. Each copy of the A allele represented a significant linear increase in the prevalence ratios of abdominal obesity criteria and HDL-C alteration. This influence of the genotype on the variables involved in MetS was also observed in the adjusted multivariate analysis. <SEP>The proportion of children with 0–4 components was not significantly different between the different genotypes. However, the prevalence ratios of those with three and four components increased significantly with each copy of the A allele. This situation resulted in the prevalence of MetS being distributed differently between the different genotypes (p = 0.004), growing linearly in the TA (1.26 95% CI 0.71–2.23) and the AA (2.21 (95%CI 1.36–3.59), and taking TT as reference (p = 0.002). These results are similar to those found by Kelishadi et al. in their meta-analysis, in which they stated the relationship between the FTO gene and MetS. Other authors have concluded that this association is not present in the pediatric population but in the adults. However, considering that the genotype does not change throughout life, it can be thought that these results are influenced by the low methodological quality of the studies evaluated. Finally, other SNP interactions associated with MetS and its components, such as WC, should also be considered because they could increase the risk of MetS.<SEP>Although the association found between the rs9939609 SNP and MetS is in line with evidence from other authors, and remains constant in different ethnicities, the mechanism by which the polymorphism influences the development of MetS and the alteration of its components is unclear. One hypothesis is that its role in lipogenesis, widely described, derives from the associated metabolic alterations, but this has not been demonstrated. In this regard, our results show that for each risk allele (A), there is a significant increase in the amount of fatty tissue and abdominal obesity measured by WC. Its relation with the impairment of lipid metabolism, especially in TG and HDL-C, has also been suggested, which has been observed in our results. On the other hand, it has been evidenced that being a carrier of the risk allele of the rs9939609 SNP is associated with an alteration of glucose metabolism, which could be a precursor of changes in the regular brain control of adiposity and, therefore, derive in MetS. However, one aspect on which most authors agree is that further studies are needed to identify the mechanism that defines this association.<SEP>However, although the underlying mechanism has not yet been clarified, the relationship between this polymorphism and MetS seems clear. Hence authors consider that it is essential to use this knowledge for preventing the development of MetS. Although genetic load increases the risk of developing MetS, it has been shown that lifestyle factors, such as feeding and physical activity, can modulate its expression. Concerning diet, high saturated fatty acid (SFA) intake and a low ratio between polyunsaturated fatty acid (PUFA) and SFA intake have been found to increase the risk of obesity, especially in patients with MetS. Furthermore, substituting SFA for PUFA in patients carrying the risk allele (A) decreases the risk of cardiovascular disease. Controlling fat intake would modulate the rs9939609 polymorphism and could also help to reduce the effect of this macronutrient on other genetic variants related to metabolic alterations, such as the rs5082 polymorphism of the APOA2 gene and its influence on insulin resistance. In addition, it has recently been found that adapting the intake of different macronutrients to the other genotypes linked to several FTO gene polymorphisms (rs9939609, among them) modifies the risk of obesity and metabolic disorders. This fact means that carriers of the risk allele of the rs9939609 polymorphism, or other FTO gene polymorphisms, could be helped through individualized nutritional interventions.<SEP>On the other hand, physical activity is another lifestyle component that should be taken into account to improve patients’ health. It has been observed that the association between the rs9939609 SNP and MetS is higher in subjects with a low level of physical activity. However, evidence shows that performing physical activity can significantly attenuate the rs9939609 SNP of the FTO gene and other genes associated with obesity and metabolic alterations.<SEP>5. Limitations<SEP>A larger sample size would reinforce the findings, clarifying some results that might be affected by a limited number of cases. In addition, no variables related to the dietary pattern of the subjects were collected, so the effect of the diet on the participants could not be evaluated. Future research should develop prospective methodologies to determine the risk of MetS based on the genotype determined by the rs9939609 SNP. The literature on this topic is not recent, which makes it difficult to compare to our results.",
                  "21500",
                  "['OBWCCE']"
                ],
                [
                  "31",
                  "24935819",
                  "Background<SEP>Indicators of socioeconomic status (SES) are strongly related to health conditions with groups of low SES showing higher prevalence and incidence for almost all diseases. Despite the evidence that differences in working and living conditions, health behaviors and psychosocial factors are important determinants underlying these health inequalities, twin and other types of family studies suggest a contribution of genetic factors due to selection effects. However, on the molecular level the relevance of disease-related genetic variants for the explanation of social inequalities in health is still unclear and theoretical approaches as well as empirical analyses incorporating genetic data are largely missing. Learning more about the possible role of genetic factors in health inequalities is considered to further improve the understanding of population health in general and of health inequalities in particular.<SEP>If differences in genetic predisposition of a certain disease had an impact on health inequalities, it would be expected that the disease itself would have an influence on SES through intra- and intergenerational processes of social mobility, allowing for a higher frequency of disease-related risk alleles in lower SES groups. This assumption is described by the hypothesis of direct health selection supposing that individuals with good health are more likely to move upward in SES than individuals with poor health and vice versa. Incorporating genetic factors in the hypothesis of direct health selection would give disease-related risk alleles an impact on SES through a mediating effect of disease. However, direct health selection cannot be regarded as the main explanation for health inequalities because processes of social mobility are scarce in older ages when most diseases arise. Particularly, if focusing on late onset diseases and measures of SES representing early life conditions or conditions across the life span (e.g., parental SES, education), reverse causation is unlikely.<SEP>With regard to common complex diseases it is supposed that next to environmental factors a large number of genetic variants contribute to disease etiology. In the recent past, disease-related risk alleles were primarily identified by genomewide-association studies (GWAS) following the common disease – common variants hypothesis. These GWAS-based gene variants usually have small to modest individual effects, however strong relationships between common variants and complex traits would be necessary to gain a further prerequisite for supposed differences in allele frequencies between SES groups.<SEP>Hence, there is little reason to assume that differences in genetic predisposition to complex diseases play a role in health inequalities. To reconsider this argument by empirical analysis on the molecular level, the aim of this study was to examine whether there are associations between different SES indicators and genetic variants predisposing to diabetes mellitus as an example for a late onset complex disease. The inverse relationship between indicators of SES and diabetes is well reported across different populations. In addition, a large number of genomic loci robustly associated with diabetes were identified through GWAS in the recent past. To date, no studies have examined the association of SES and diabetes-related genetic variants to explore whether SES differences in genetic predisposition contribute to health inequalities in diabetes.",
                  "Methods<SEP>Study population<SEP>Data was used from the baseline examination of the Heinz Nixdorf Recall (Risk Factors, Evaluation of Coronary Calcium, and Lifestyle) Study, a prospective population-based cohort study. The rationale and design of the study were described elsewhere. A random sample derived from mandatory citizen registries of three large cities (Bochum, Essen, Mülheim/Ruhr) in an urban region in the western part of Germany was used to recruit 4814 women and men aged 45–75 years. Baseline examination took place from 2000 to 2003 and the baseline response proportion was 55.8%. Written informed consent was obtained from all participants. The study was approved by the institutional ethics committee of the University Hospital Essen and comprises extended quality management procedures, including a certification according to DIN ISO 9001:2000.<SEP>Data assessment<SEP>Diabetes was defined as either of the following criteria: reported history of diabetes, taking glucose-lowering drugs, having fasting blood glucose levels of greater than 125 mg/dL, or having nonfasting glucose levels of 200 mg/dL or greater. Overall, 23 participants reported early disease onset indicating rather Type 1 than Type 2 diabetes. In analyses excluding these 23 participants virtually identical results were obtained as those presented in the following. To be consistent with a previous study using the same data we decided to include the 23 participants in the analyses.<SEP>Based on literature research the following 11 single nucleotide polymorphisms (SNPs) related to 8 genetic loci (in parentheses) derived from GWAS for diabetes in European-origin populations were selected: rs4402960 (insulin-like growth factor-binding protein 2 [IGF2BP2]), rs1801282 (peroxisome proliferator-activated receptor gamma [PPARG]), rs7754840, rs7756992, rs10946398 (CDK5 regulatory subunit associated protein 1-like 1 [CDKAL1]), rs13266634 (solute carrier family 30 (zinc transporter), member 8 [SLC30A8]), rs10811661, rs564398 (cyclin-dependent kinase inhibitor 2A/2B [CDKN2A/2B]), rs7903146 (transcription factor 7-like2 [TCF7L27]), rs1111875 (hematopoietically-expressed homeobox [HHEX]) and rs8050136 (fat mass and obesity associated [FTO]). These SNPs include common variants with some of the highest effects on diabetes risk reported to date. The literature research took place in January 2009 and was previously described in detail. Genotyping was performed by matrix-assisted laser desorption ionization-time of flight mass spectrometry-based iPLEX Gold assay at the Department of Genomics, Life and Brain Center, Bonn, Germany.<SEP>Education, income and paternal occupation were collected as SES indicators by standardized interviews. Paternal occupation was classified referring to the International Standard Classification of Occupations (ISCO-88) and categorized into four groups (unskilled employees/workers; qualified (skilled) employees/workers; technicians and associate professionals; managers and professionals).<SEP>Education was defined by combining school and vocational training as total years of formal education according to the International Standard Classification of Education and categorized into three groups with the lowest educational group of 10 and less years (equivalent to a basic school degree with no vocational training), the medium educational group of 11 to 13 years (equivalent to upper secondary educational degrees or a combination of lower secondary education and vocational training) and the highest educational group of 14 and more years of education (equivalent to a vocational training including additional qualification or a university degree). In previous analyses of the same study population no further differentiation between university degrees and other types of higher education has been made because of the small number of diabetes cases in the respective group. This small sample size would have caused problems in conducting multivariate analyses.<SEP>Income was measured as the monthly household equivalent income calculated by dividing the total household net income by a weighting factor for each household member. Income was included into analyses either as a continuous variable or divided into four groups using sex-specific quartiles.<SEP>Statistical analyses<SEP>The analyses were conducted with 4655 participants who had information on diabetes status, indicators of SES and genetic data. Some observations on paternal occupation (n = 258), education (n = 14) and income (n = 296) were missing. As the SES indicators were analyzed separately, these participants were excluded only from the respective analyses. No correlations between missing SES measures and diabetes status were observed. All analyses were performed using the R statistical package version 2.14.0 and PLINK (v1.07) for Windows.<SEP>First, log-binomial regression models were fitted to assess the association of SES indicators and diabetes status by estimating sex-specific and age-adjusted prevalence ratios (PR) and their corresponding 95%-confidence intervals (95%-CIs). Education, income and paternal occupation were entered separately as categorical predictors by coding dummy variables with the highest category as reference.<SEP>Second, logistic regression models were fitted to check the association of the GWAS-based SNP alleles to diabetes status. Therefore, sex- and age-adjusted odds ratios (OR) and 95%-CIs were estimated under a (log-)additive genetic model for each SNP, as suggested in previous studies. In addition, a genetic risk score was developed by adding the risk alleles (0/1/2) of the diabetes-related SNPs for each participant to explore the association between the sum of risk alleles and diabetes status. For missing genotype information expected values were imputed based on the risk allele frequency of the respective SNP in the study population. SNP pruning for the genetic risk score was performed using pairwise linkage disequilibrium of r 2  > 0.8 as cut off to account for correlated effects, resulting in the exclusion of rs1094639. The calculated effect size estimators are to be interpreted as average effects for one additional risk allele.<SEP>Third, for the primary research question the relationship between each SNP and SES was explored by computing sex- and age-adjusted ORs and 95%-CIs under a (log-)additive, dominant and recessive genetic model. Again, categorical education, income and paternal occupation were used separately as indicators of SES. For each SES category a binary outcome variable with the highest category as reference was entered in a logistic regression model. Income was also used as a continuous variable in a linear regression model to estimate standardized effect sizes and 95%-CI for each SNP. For this analysis income was loge-transformed to normalize the distribution. According to our knowledge, no association of the selected SNPs to SES has been demonstrated previously. Hence, it was decided to control the family-wise error rate of the primary research question at α = 0.05. We adjusted the alpha-level for multiple testing of 11 SNPs using Bonferroni’s method (αBF ~ 0.005). In addition, the association between the genetic risk score and SES was explored using the SES indicators as outcome.",
                  "Results<SEP>Characteristics of the study population are shown in Table 1. Of the 4655 participants 13.6% (n = 634) had diabetes with women having a lower prevalence (9.8%) than men (17.4%). Differences between women and men in the distribution of two SES indicators were observed: Women reported less years of formal education and showed a lower median income.Inequalities in all SES indicators were related to diabetes status for both women and men (Figures 1, 2 and 3). The comparison of men in the lowest category of paternal occupation (unskilled employees/workers) to those in the highest category (managers and professionals) showed an age-adjusted PR of 1.5 (95%-CI: 1.0-2.3) for the occurrence of diabetes in the study population. The respective PR for women was 1.6 (95%-CI: 0.9-3.0). The analyses for education (≤10 years of education vs. ≥ 14 years; women: PR 2.3, 95%-CI: 1.4-3.9; men: PR 1.4, 95%-CI: 1.0-2.0) and income (lowest sex-specific quartile vs. highest sex-specific quartile; women: PR 2.0, 95%-CI: 1.3-3.2; men: PR 1.2, 95%-CI: 0.9-1.5) showed similar results. There are gender differences with women revealing stronger associations with diabetes status across all SES indicators.<SEP>Table 2 shows the estimated ORs of the logistic regression models for the 11 selected SNPs and diabetes status. All effect size estimators are small to modest with ORs ranging from 1.01 to 1.28 for each risk allele. The SNPs rs4402960 (IGF2BP2), rs7756992 (CDKAL1), rs13266634 (SLC30A8), rs1111875 (HHEX), and rs7903146 (TCF7L2) showed statistically significant associations at a nominal α level of 0.05. For the genetic risk score a clear association with diabetes status could be observed (OR: 1.12, 95%-CI: 1.07-1.16, p: 5.95 × 10-8).<SEP>The results for the logistic regression models for the 11 SNPs and paternal occupation under a (log-)additive genetic model are shown in Table 3. No statistically significant associations at αBF ~ 0.005 were observed. The estimated ORs were small to modest ranging from 0.87 to 1.23 for each respective diabetes risk allele. The logistic regression models for the outcomes education (Table 4) and income (Table 5) revealed similar results with no statistically significant associations at αBF ~ 0.005 and ORs ranging from 0.88 to 1.16 (education) and 0.87 to 1.14 (income). Under a dominant and recessive genetic model no deviant results to those under a (log-)additive model were obtained (results not shown). Furthermore, no statistically significant associations were observed for the analysis using loge-transformed income (results not shown). This was consistent with the observation for income as a categorized outcome.<SEP>Table 6 shows the results using the genetic risk score in logistic regression models for all SES indicators. The estimated ORs are close to 1.0 and statistically significant results at a nominal α level of 0.05 were observed only for income comparing the 3rd quartile with the highest quartile (OR 0.95, 95%-CI: 0.91-0.99).",
                  "Discussion<SEP>The presented data showed an association between all indicators of SES and diabetes status. Magnitude and trend of the associations across different SES groups are similar to those reported in the literature. Stronger associations of SES differences in diabetes for women have been reported before as well. With the selected SES indicators different but related aspects of social inequalities are measured representing different stages in the life course. As associations for all of the explored SES indicators were found, our results give supporting evidence that health inequalities in diabetes are affected by diverse social conditions during different stages in life.<SEP>As expected, considering the given sample size and the relatively small number of diabetes cases in the study population, only 5 of the 11 selected diabetes-related SNPs (5 of the 8 genetic loci) were replicated with nominal statistically significant results for their association with diabetes status. However, all SNP alleles showed directionally consistent effects when compared with those reported in the literature.<SEP>The main finding of the study is the lack of evidence for the contribution of 11 selected diabetes-related SNPs representing 8 genetic loci to the observed health inequalities in diabetes. There were no statistically significant associations between the individual SNPs and SES indicators after conservatively controlling for multiple testing by the Bonferroni method. Even with an uncorrected level of significance (α = 0.05) the number of estimators with p < α does not exceed the number expected by chance.<SEP>In general, no clear inverse trend of the calculated ORs could be found across the different SES groups. There may be a few exceptions for markers related to FTO (for paternal occupation), PPARG, CDKAL1 and CDKN2A/2B (for education). As the differences between the respective ORs are small and statistically significant results were missing, they have to be interpreted as random trends. Furthermore, we observed many ORs < 1.0 which we would not expect if the risk alleles of the selected SNPs generally have an impact on the observed health inequalities in diabetes. In addition, the highest ORs – which are only of small to modest magnitude – were not exclusively present for the lowest compared with the highest SES group. The results for using the genetic risk score in the analyses support the observations for the individual SNPs: The sum of selected diabetes risk alleles is not increasing with a decreasing in SES.<SEP>To our knowledge, this is the first study investigating social inequalities in diabetes and simultaneously exploring the impact of selected SNPs robustly associated with diabetes. There are just a few studies that have investigated supposed differences in risk allele frequencies to estimate their contribution to health inequalities with varying results. Only Holzapfel et al. analyzed the relationship between SES, body mass index (BMI) and the SNP rs9935401 within FTO, which is also associated with diabetes through its effect on BMI. They reported no association with education and income for rs9935401. This is in line with our observations for the FTO marker rs8050136, which is strongly correlated to rs9935401 (r 2  = 1.0) within the HapMap CEU population.<SEP>Our study suggests that there is no contribution of diabetes-related SNPs to health inequalities in terms of differences in risk allele frequencies between SES groups. The question remains, how genetic factors could adequately be integrated in explanations of health inequalities. One possible approach is offered by the life course perspective, which describes the interplay of biological, environmental and social factors and their impact on health over the life course. Thus, the life course perspective is not tied to the assumption of a single causal direction as described by, e.g., the hypothesis of direct health selection. Following this approach, a more plausible picture of genetic risk factors interacting with environmental and social factors can be drawn to give consideration to the complex chain of risks that produces health inequalities.<SEP>The following limitations of the study need to be considered: First, due to the given sample size the statistical power to confirm the reported genetic associations with diabetes and to detect associations between the SNPs and the SES indicators – especially in the analyses with categorized SES indicators – is limited. To address this limitation and to increase the detection power we also used a genetic risk score for our analyses.<SEP>Second, over 66 genomic loci related to diabetes are already described which account for approximately 6% of variance in diabetes susceptibility suggesting that there still may be some unexplained genetic variance. Thus, the present investigation of the relationship between diabetes-related genomic loci and SES indicators is far from being comprehensive. Given that the more recently discovered loci yield smaller effects on diabetes than the SNPs explored here, our analysis should be regarded as a first step to address the relationship of diabetes-related genomic loci with SES indicators.<SEP>Third, as we do not have information on diabetes status of earlier life stages it was not possible to check for the causal direction of the association between the SES indicators and diabetes. However, especially for education and paternal occupation reverse causation is unlikely as the former is a stable indicator of socioeconomic status across the life course and the latter of the participants’ childhood. For income, reverse causation cannot be ruled out.<SEP>Fourth, the validity of the SES indicators is restricted. For education and income this is due to their age dependency. The distribution of educational degrees varies by age groups with the higher groups showing lower variance and income generally declines in relation to retirement. Therefore, these indicators of SES may be more valid in younger age groups.",
                  "19859",
                  "['OBWCCE']"
                ],
                [
                  "32",
                  "29327327",
                  "Introduction<SEP>Osteoporosis (OP) is the most common metabolic bone disease, mainly characterized by low bone mineral density (BMD) and deterioration of bone microstructure. It is a major cause of osteoporotic fracture (OF) in the elderly. OF may occur with slight collision or even happen spontaneously and lead to severe decrease of life quality, with increased morbidity, mortality, and disability. Only 33% of elderly women who suffered hip OF will be able to return to independence. OP is highly prevalent worldwide, over 200 million people suffer from OP. In USA, about 8 million women and 2 million men had OP in 2010. In China, the prevalence of OP ranged from 6.6% to 19.3%.<SEP>Dyslipidemia (DL) is a condition with abnormal levels of plasma lipid (PL), including abnormally elevated levels of triglyceride (TG), total cholesterol (TC) and low density lipoprotein cholesterol (LDL), and descended levels of high density lipoprotein cholesterol (HDL). Presence of DL increases the risk of suffering coronary heart disease (CHD) and future cardiovascular (CV) diseases, which are ranked as the top two causes of premature death and decreased disability-adjusted life-years (DALYs) worldwide. In US, 36.7% adults were on or suit for lipid-lowing therapy during 2005–2012. In China, the prevalence of DL was 41.9% among adults.<SEP>Experimental and clinical studies have demonstrated that DL is closely related to OP. In clinic, patients with DL are often diagnosed with OP. A study with 279 either pre- or post-menopausal women demonstrated that high TC level was associated with low BMD. High cholesterol might influence cellular functions of bone tissue, such as increased osteoclast activity and decreased osteoblast function. Bone metabolism and lipid metabolism might interact with each other by some molecules, such as osteoprotegerin (OPG), apolipoprotein E (APOE), peroxisome proliferators-activated receptor γ (PPARγ) and vitamin D receptor (VDR). For example, OPG can reduce the production of osteoclasts and regulate the resorption of osteoclasts. Meanwhile, elevated OPG levels have been reported in heart diseases, which may be a link between bone and atherosclerosis - a main complication of DL. Furthermore, medication for DL therapy, such as statins, can decrease PL and increase BMD simultaneously.<SEP>OP and DL both are complex human diseases, which are affected by multiple genes. Both BMD and PL are highly heritable. For BMD, its heritability was about 50%–80%, and for PL it was 40–60%. Previous GWASs (Genome wide Association Studies) have identified about 200 loci associated with BMD and about 500 loci associated with PL (https://www.ebi.ac.uk/gwas, Aug 1, 2017). However, these identified SNPs (single nucleotide polymorphisms) only explained a small part of the heritability. Further explorations were required for the missing heritability. Considering the multiple genes which might affect OP and DL as well as the high missing heritability, we would like to carry out more studies to explore the potentially novel genetic mechanisms and the potentially shared genetic relationship for OP and/or DL.<SEP>Novel pleiotropy informed analytical methods have emerged in recent years, such as conditional false discovery rate (cFDR), ccFDR (conjunction cFDR) and genetic analysis incorporating pleiotropy and annotation (GPA). These methods could incorporate summary statistics from independent GWASs to capture polygenic effects and identify more potentially novel loci for the interested phenotypes. Recently, Andreassen et al. have identified pleiotropy between BMD and PL and found some novel BMD-associated SNPs conditioned on PL by cFDR method. Encouraged by their initial study, in this study, we will apply not only the cFDR method but also the complementary GPA method to larger and newer GWAS datasets of PL and BMD. We hope to validate the pleiotropy between BMD and PL and identify potential common genes as well as potentially novel DL-associated genetic variants, with the results to be obtained from newer and larger datasets and with different analytical approaches for robustness. In our previous studies, we have successfully implemented cFDR analyses on GWAS datasets of femoral neck (FNK) BMD, height, birth weight, type 2 diabetes and coronary artery disease (CAD) and identified potentially novel and pleiotropic genetic variants for these phenotypes simultaneously and respectively. These experiences would contribute to better implementation of this study.",
                  "Materials and Methods<SEP>GWAS datasets<SEP>We obtained GWAS results of FNK BMD and PL (HDL, LDL, TC, TG) in the form of summary statistic p-values. FNK BMD GWAS dataset contained more than 10 million SNPs from 49,988 subjects, and PL GWAS recruited 188,577 subjects and used GWAS array and Metabochip array to genotype/impute more than 2 million SNPs. Both datasets are the largest for FNK BMD and PL so far. FNK BMD dataset was published online by Genetic Factors for Osteoporosis (GEFOS) (http://www.gefos.org/), and PL dataset was published by Global Lipids Genetics Consortium (GLGC) (http://csg.sph.umich.edu/abecasis/public/lipids2013/). Detailed inclusion criteria and phenotype characteristics for the two GWASs were demonstrated in the original respective papers.<SEP>SNP pruning<SEP>We performed SNP pruning on FNK BMD and PL datasets respectively before further analysis. Since most of the individuals of the original GWASs were European ancestry, we used the CEU HapMap 3 genotype data for the SNP pruning. First, we merged common SNPs between FNK BMD and each kind of PL. After merging, there were about 2 million SNPs remained in each PL type. According to HapMap3 information, we calculated LD (linkage disequilibrium) values between each pair of SNPs by PLINK 1.9 software. We selected default value of the software (50, 5, 0.2) as parameters, which meant the calculation of LD was performed in a window containing 50 SNPs. And the SNP in each pair with smaller minor allele frequency (MAF) was removed when the LD value (r2) was greater than 0.2. Then the calculation window slid forward with 5 SNPs and repeated the above process until no pairs of SNPs that were in high LD. After SNP merging and pruning, the remaining SNPs were prepared for subsequent analysis.<SEP>Statistical analysis<SEP>Pleiotropic enrichment estimation<SEP>We used “ggplot2” package in R software to construct fold-enrichment plots to estimate the pleiotropic enrichment between FNK BMD and PL. The plots were formed by nominal −log10(p) values at different stratifications. Stratification was divided by p-value of conditional phenotype with the cut-offs as p < 1 (expected base line, all SNPs), p < 0.1, p < 0.01 and p < 0.001. Nominal p-values (−log10 (p)) were plotted on x-axis and fold enrichments were plotted on y-axis. In each cutoff group, for all possible -log10(p-values) on the x axis (between 0 and 10), we compute the fold enrichment values,<SEP>  where En is the enrichment values. Ni is the proportion of SNPs with −log10(p-values) ≧ x. N0 is the number of all SNPs in each cutoff group, and the i is from 1 to N0. Presence of pleiotropy can be visually observed as an upward shift from the expected base line. And there would be separation between different stratification. The greater separation indicated the stronger pleiotropy.<SEP>As a complement for pleiotropic enrichment estimation, we performed a hypothesis testing procedure by GPA. In this testing, likelihood ratio test (LRT) was used to assess statistical significance and show statistical evidence for pleiotropic enrichment. Here we set threshold as p value = 0.05. We downloaded and ran “GPA” package in R software (http://dongjunchung.github.io/GPA/), then we fit GPA model to test the hypothesis for pleiotropy.<SEP>Calculation of cFDR, ccFDR and GPA<SEP>We set PL as the principal phenotype and FNK BMD as the conditional phenotype. Summary statistic p-values of PL and FNK BMD GWAS datasets were incorporated by common SNPs for calculation of cFDR, ccFDR and GPA. Detailed procedures and formulas were described by Greenbaum and Chung D respectively.<SEP>The cFDR method was developed from standard FDR framework. The cFDR value represented a random SNP associated with PL conditioned on FNK BMD. And ccFDR value referred to the possibility of a given SNP that had association with both FNK BMD and PL simultaneously. Both cFDR and ccFDR thresholds were set to 0.05, which meant that the SNP reached this threshold achieved significant association with the corresponding phenotype and the false discovery rate was 5%.<SEP>GPA method was performed by “GPA” package in R software. We fit a GPA model with summary statistic p-values of SNPs and performed the analysis. fdr.GPA was used to represent the SNP that was associated with one of the two phenotypes, and fdr11.GPA was used to represent the SNP that was associated with both phenotypes. According to the criterion set in the original paper, we adopted the same significance threshold of GPA to 0.2. Over this threshold meant that a SNP might not be associated with the corresponding phenotype.<SEP>Annotation of potentially novel SNPs for PL<SEP>If a SNP met the thresholds (cFDR < 0.05 and fdr.GPA < 0.2) for PL, we conferred it as a significant one. To verify whether the identified PL significant SNPs with p-values > 5E-08 were potentially novel ones, we compared these SNPs to the other previous GWAS results by querying GWAS web site (https://www.ebi.ac.uk/gwas, Aug 1, 2017). After comparison, we input non-repeat SNPs (including the significant SNPs with cFDR < 0.05, fdr.GPA < 0.2 and p values > 5E-8 in the original PL GWAS dataset, and the previously confirmed SNPs which were reported in the other PL GWASs) into the SNP Annotation and Proxy Search (SNAP, http://archive.broadinstitute.org/mpg/snap/) for LD analysis. In the search options of the website, we chose suitable SNP data set and population panel based on the original GWASs (SNP data set was HapMap3 and population panel was CEU) for pairwise LD calculation. Then we used the default values (r2 threshold = 0.8, distance limit = 500, where distance means the maximum number of kilobase between query and proxy SNP) as the criteria to determine whether the identified novel SNPs were in same LD block with previous GWAS signals. Among these SNPs, only the SNP that had not been identified in other previous GWASs and was not clustered in the same LD block with those previously GWAS confirmed SNPs would be regarded as a potentially novel SNP, otherwise it was regarded as a replication of the previous GWAS results.<SEP>Conditional and conjunction Manhattan plots<SEP>We used information including SNP number, chromosome position and cFDR or ccFDR value to construct Manhattan plots by R software to visualize the localization of the significant SNPs (Fig. 2 and Fig. 4). The Manhattan plots consist of conditional and conjunction Manhattan plots, marking the significant SNPs and their chromosomal locations. In the conjunction Manhattan plot (Fig. 2), the SNPs with significant −log10(ccFDR) values more than 1.3 (corresponding to a ccFDR value less than 0.05) were shown in the plot. Only those SNPs with fdr11.GPA less than 0.2 were highlighted with SNP numbers. These SNPs were defined as being associated with both phenotypes in the conjunction Manhattan plot. In the conditional Manhattan plot (Fig. 4), the SNPs with −log10(cFDR) values more than 1.3 were determined as being associated with the principal phenotype.<SEP>Gene ontology enrichment analysis and protein-protein interaction analysis of the identified SNPs and genes<SEP>We mapped the identified SNPs to corresponding genes by the online tool SNP and CNV Annotation Database (SCAN, http://scandb.org/newinterface/about.html). To evaluate the function of these genes and to partially validate our results, we performed Gene Ontology (GO) Enrichment analysis (http://geneontology.org/page/go-enrichment-analysis). Meanwhile, using the online tool STRING 10.0 (http://string-db.org/), we further explored the functional interaction between the proteins produced by the corresponding genes. Protein-protein interaction analysis enabled us to identify the potential associations between the corresponding genes.",
                  "Results<SEP>Pleiotropy between FNK BMD and PL<SEP>As shown in fold-enrichment plots (Fig. 1), each type of PL was the principal trait while FNK BMD was the conditional trait. When restricting the subset of SNPs with a stronger level of association in the conditional trait, we observed an obvious upward shift from the expected base line. It well demonstrated the pleiotropy between FNK BMD and each type of PL (HDL | FNK BMD, LDL | FNK BMD, TC | FNK BMD and TG | FNK BMD). Among the four plots, HDL conditioned on FNK BMD (HDL | FNK BMD) achieved the most significant pleiotropic enrichment, as a 9.5 fold enrichment was observed in Y axis while comparing the most stringent subset to the all SNPs subset (Fig. 1 a). LRT-p values acted as statistical evidence for pleiotropic enrichment through hypothesis testing of GPA and the results were presented in Table 1. The pleiotropy between FNK BMD and HDL with a LRT-pvalue of 5.92E-03 was still the strongest, suggesting the highest level of pleiotropy between FNK BMD and HDL among all the PL traits.<SEP>Potentially pleiotropic SNPs/genes for both FNK BMD and PL<SEP>According to the thresholds of ccFDR < 0.05 and fdr11.GPA<0.2, three SNPs (rs2178950, rs9939318 and rs9368716) were identified to be associated with both FNK BMD and PL (Fig. 2). All of the three SNPs’ p-values were higher than 5E-08 in the original FNK BMD and PL GWAS datasets. They were mapped to chromosomes 6, 8, 16 and corresponded to C6orf10/LOC101929163, TRPS1 and NLRC5 genes respectively (Table 2). TRPS1 (rs2178950) gene was involved in autosomal dominant skeletal disorder, encoding a GATA-type transcription factor. TRPS1 was associated with HDL and CAD in a meta-analysis of 46 GWASs of lipids. NLRC5 (rs9939318) was a member of the NLR family. It acted as a transcriptional activator of MHC class I genes. Meanwhile, it contributed to inflammatory and type I interferon responses in vitro. According to protein-protein interaction information from “STRING Interaction Network” (Fig. 3), we found these pleiotropic genes interacted with many other bone and/or PL associated genes indirectly. For example, NLRC5 interacted with IKBKB, CHUK, DDX58, IFIH1 and RNF135, while TRPS1 interacted with JUN family and WDR family.<SEP>Potentially novel PL loci identified by cFDR and GPA<SEP>According to the thresholds of cFDR < 0.05 and fdr.GPA < 0.2, we identified totally 395 significant SNPs associated with PL variation conditioned on FNK BMD (Supplemental Table 1). Among the 395 SNPs, 144 SNPs achieved p values lower than 5E-8 in the original PL GWAS datasets, 1 SNP (rs12708980) in HDL was confirmed before to be GWAS-associated with PL, and 5 SNPs had high LD values (according to the default value of the SNAP website, r2 > 0.8) with previous PL GWAS findings, including 2 SNPs (rs5754467 and rs9930506) in HDL, 1 SNP (rs2699429) in LDL, 1 SNP (rs2178950) in TC and 2 SNPs (rs12751742 and rs9930506) in TG (Table 3). These SNPs were regarded as replications of the original PL GWAS. The remaining 245 SNPs with p-values higher than genome-wide significance threshold of 5E-08 were potentially novel SNPs for PL, among which 71 SNPs were identified for HDL, 67 SNPs for LDL, 92 SNPs for TC and 54 SNPs for TG (Table 3). These significant SNPs were mapped to 21 chromosomes (1–20, 22) and the positions were showed in the conditional Manhattan plots (Fig. 4).<SEP>Gene annotation and function enrichment analysis for potentially novel PL SNPs<SEP>We mapped the 245 potentially novel SNPs to their corresponding genes by SCAN and performed the GO term enrichment analysis. The strongest enriched GO term and associated genes/SNPs for PL were listed in Table 4. For HDL, the most enriched GO term was “phospholipid homeostasis”, with a fold enrichment value over 100 (p-value = 6.53E-05). The significant genes in this GO term included HNF4A (rs2071197), LIPG (rs2097055, rs883218 and rs4556888), CETP (rs17369163) and GPAM (rs10787429). For TC, the most enriched GO term was “chylomicron remnant clearance“, with fold enrichment as over 100 (p-value = 2.12E-02). The significant genes in this GO term included LDLR (rs2738456), APOB (rs6733447) and LIPC (rs792902, rs1652519, rs6494007, rs4775046 and rs12324517). Both of the most enriched GO terms play key roles in PL metabolism. No enriched GO terms were found for LDL and TG. The other enriched GO terms were presented in Supplemental Table 2.",
                  "Discussion<SEP>In this study, we combined the summary statistics from two independent GWAS datasets and jointly implemented cFDR and GPA methods to validate the pleiotropy between FNK BMD and PL (HDL, LDL, TC, and TG). Potentially common genes were identified between FNK BMD and PL, which could lead to a further and novel understanding of the shared genetic mechanisms for both OP and DL, and potentially have a positive impact on future clinical treatment and prevention. A number of potentially novel SNPs associated with PL were also identified, which provided new directions for future studies of molecular pathogenesis for DL.<SEP>To explore more missing heritability, traditional ideas included recruiting more participants and genotyping larger samples, but these were difficult and impractical in many cases and costly. The advantages of cFDR and GPA methods were that they could leverage the power of pleiotropy by using current GWAS datasets, they virtually increased the existing sample size and enhanced statistical power to explore more potential genetic variants. They also lessened the burden of multiple testing by controlling the false discovery rate, which meant that the cFDR and GPA methods could leverage pleiotropy information and also provide less conservative control of type I errors compared to Bonferroni correction. cFDR method was the first approach which statistically addressed the issue of pleiotropy between GWASs of two different traits based on Bayesian formula. GPA method could systematically integrate pleiotropy and annotate information based on LRT. In this study, if a SNP was identified by the two methods simultaneously, it meant that the SNP was validated virtually by two different approaches, which might further reduce false positive findings for more robust results. It should be noted that GPA method could be used to analyze GWAS results with or without annotation information. Meanwhile, summary statistic p-values of GWAS was more important than functional annotation information in GPA. Therefore, in this study we chose not to use annotation information, which not only rendered efficient analysis in terms of computation but more importantly also matched the cFDR results better since cFDR method could use only summary statistics information.<SEP>Andreassen et al. had identified pleiotropy between FNK BMD and PL by cFDR method in a previous study. They had identified 65 novel BMD loci by conditioning on cardiovascular disease (CVD) related phenotypes, including PL (HDL, LDL, TC and TG). Unlike their initial study, we hope to explore more relationships between FNK BMD and PL from different/novel aspects, such as to identify common genes to both PL and BMD, and novel PL-associated SNPs. These study aspects were not covered in the Andreassen’s findings. Hence, on one hand, we validated the pleiotropy first. In our pleiotropic enrichment analysis, we used GPA method together with cFDR method based on larger and newer GWAS datasets and successfully validated the pleiotropy. We further quantified the degree of pleiotropy by enrichment plot and LRT. On the other hand, based on the validated pleiotropy, we focused on identifying potentially novel SNPs for PL conditioned on FNK BMD and identified common genes for FNK BMD and PL. These findings were novel compared to the Andreassen’s results.<SEP>By leveraging the pleiotropy, we could integrate PL and FNK BMD GWAS datasets and virtually increase the existing sample size. Then we could use cFDR method and GPA method to enhance the statistical power to explore potentially novel PL associated SNPs. In this study, we identified 395 significant SNPs for PL, of which 150 SNPs had reached p < 5E-8 in the original GWAS or were identified in other previous PL GWASs. These results reflected the reliability of the cFDR method and GPA method. Meanwhile, we identified 245 potentially novel SNPs for PL. Several genes corresponding to these potentially novel SNPs were enriched in plasma metabolism related GO terms, such as “phospholipid homeostasis” and “chylomicron remnant clearance“. This functional enrichment analysis result suggested that these potentially novel SNPs might be associated with PL metabolism.<SEP>We identified 3 pleiotropic SNPs (rs2178950, rs9939318 and rs9368716) that were associated with both FNK BMD and PL. rs2178950 was located in TRPS1. TRPS1 gene was confirmed to be associated with HDL in a previous GWAS meta-analysis. Meanwhile, TRPS1 was associated with BMD and Wnt signaling pathway. Protein produced by TRPS1 was a transcription factor and played an important role in skeletal development by influencing osteoblast cell differentiation and osteocalcin expression. Deletion of TPRS1 could cause skeletal abnormalities called skeletal abnormalities of tricho-rhino-phalangeal syndrome type I, characterized by craniofacial abnormalities and disturbances in formation and maturation of bone matrix.<SEP>The pleiotropic SNP rs9939318 was located in NLRC5, which was related to inflammation and immunity. Interestingly, PL was also involved in the pathological procedure of inflammation and immunity. Cholesterol could induce immune response and inflammation during the progression of atherosclerosis, and impact on the circulating monocytes and bone marrow. So we inferred that NLRC5 might be involved in DL through the process of inflammation and immunity. Meanwhile, NLRC5 could regulate bone mineralization by stimulating Wnt signal pathway to promote osteoblast differentiation, and it could also inhibit nuclear factor kappa B (NF-κB) to decrease osteoclast. Wnt signaling could stimulate generation of osteoblasts by promoting mesenchymal stem cells (MSCs) towards osteoblast lineage, and decrease osteoclast differentiation by inducing OPG secretion and production. NF-κB is a transcriptional factor which regulates the bone remodeling processes and inflammatory response in both bone resorption cells and bone forming. As mentioned above, NLRC5 was associated with OP through Wnt signal pathway and NF-κB.<SEP>“STRING Interaction Network Analysis” offered information about protein-protein interaction and allowed us to explore the indirect interactions between genes. As shown in Fig. 3, TRPS1 and NLRC5 were found to interact with many genes, such as IKBKB, CHUK, DDX58, IFIH1, RNF135, JUN family and WDR family. Among them, IKBKB and CHUK were related to NF-κB, which was a regulating factor to the inflammatory response and bone-remodeling processes. CHUK was an important paralog of IKBKB. The protein encoded by IKBKB phosphorylated the inhibitor in the NF-κB complex, causing activation of NF-κB. The encoded protein of CHUK was an inhibitor of the essential transcription factor NF-κB complex. DDX58, IFIH1 and RNF135 were involved in pro-inflammatory cytokines and/or immune system. These information indicated that TRPS1 and NLRC5 might be the potential common genes associated with both OP and DL. NF-κB signaling pathway, Wnt signaling pathway, inflammation and innate immune might be the shared mechanisms underlying the relationship between OP and DL.<SEP>Although we successfully improved the identification of potentially novel SNPs for PL and pleiotropic SNPs for PL and FNK BMD, there were still some limitations in our study. First, the contribution of our findings to the proportion of the phenotypes’ variability could not be evaluated, since we only had access to summary statistics of GWAS without raw data to analyse. If we could get the raw genotype data in future, we are willing to perform linear regression with the novel SNPs and those earlier identified SNPs so that we could analyse how much our findings would contribute to the whole heritability. Second, neither cFDR method nor GPA method had the ability to identify causal variants for the interested phenotype. The aim of our study was to identify more potentially novel SNPs and provide a new direction for further functional studies to be performed by molecular and cellular biologists, thus follow-up studies should be conducted for replication and biological functional validation, and further elucidate the overall genetic mechanisms. However, under the experimental conditions of our group and time restriction, we could not carry out the functional validation experiments at present. We hope that this limitation could be partially addressed in the future by follow-up fine mapping studies and functional mechanistic studies of the GWAS associated regions by our own and/or other groups which are more specialized in functional studies. Meanwhile, considering the racial and population differences of heritability, in the future we could study other populations in the same way to identify the relationship between FNK BMD and PL among different populations.<SEP>In summary, by performing cFDR and GPA methods on current GWAS datasets, we validated pleiotropy between PL and FNK BMD and identified more potentially novel SNPs for PL. NLRC5 and TRPS1 might be the potentially common genes for PL and FNK BMD. NF-κB, Wnt, inflammation and immune may be involved in the common pathogenesis of DL and OP.",
                  "25885",
                  "['OBWCCE']"
                ],
                [
                  "33",
                  "29795460",
                  "Introduction<SEP>The human fatty acid desaturase gene (FADS) cluster on chromosome 11 harbours multiple genes including FADS1, FADS2 and FADS3. FADS1 and FADS2 encode the Δ-5 and Δ-6 desaturases, respectively. These desaturases introduce cis double bonds at the 5th and 6th positions from the carboxyl end in the biosynthesis of long-chain polyunsaturated fatty acids (LC-PUFAs) from shorter chain polyunsaturated fatty acid (PUFA) precursors during fatty acid metabolism. Genetic variants in FADS1/FADS2 are known to correlate with the variability of LC-PUFAs levels in serum/plasma, erythrocyte membranes and adipocytes, and are associated with inflammation, cardiovascular disease and type 2 diabetes. Genome-wide association studies have established associations between FADS1/FADS2 genetic variants and blood lipid traits (e.g., low-density lipoprotein cholesterol (LDL-C), high-density lipoprotein cholesterol (HDL-C), triglycerides, and total cholesterol) and fasting glucose levels. FADS3 shares high sequence homology with FADS1 (62%) and FADS2 (70%), and has similar intron and exon structures. However, the function of FADS3 is yet to be determined.<SEP>The FADS region has been the focus of numerous gene-diet interaction studies. Fumagalli et al. recently detected a strong positive selection signal in the FADS region in a Greenlandic Inuit cohort. The authors identified six single nucleotide polymorphisms (SNPs) associated with multiple quantitative traits, with the strongest association observed for body weight and height. These findings were validated in an additional Inuit cohort and the association with weight was confirmed in further European cohorts. The Greenlandic Inuit is a population isolate that has been residing in Greenland for more than 1,000 years, subsisting predominately on marine animals that are rich in monounsaturated and n-3 polyunsaturated fatty acids; these specific dietary constraints are thought to have resulted in enrichment for genetic variants involved in fatty acid desaturation. We hypothesize that the genetic adaptation that drove the observed changes in allele frequencies at certain FADS loci in the Inuits might be partially due to gene × PUFA interactions on traits, such as height, weight, and blood lipid and glucose concentrations, factors directly related to evolutionary fitness. Thus, it would be reasonable to speculate that the enhanced genetic effects of FADS variants on height, weight and other metabolic traits shown in Inuit may be driven by high dietary PUFA concentrations in this population. Interactions may be evident in other Arctic and sub-Arctic populations whose diets vary in fatty acid composition.<SEP>The primary aim of this study was to test whether dietary PUFA intake modifies the associations of the six candidate FADS SNPs reported by Fumagalli et al. and eight obesity-related traits in the GLACIER Study, a population-based cohort from the far north of Sweden. Our secondary aim was to conduct gene- and haplotype-based interaction tests focused on the FADS locus and functionally annotate top-ranking variants from relevant interactions using state of the art in silico tools.",
                  "Materials and Methods<SEP>Study participants<SEP>The GLACIER Study (cohort registration number: ISRCTN35275922) is a prospective population-based cohort study of 19,547 Swedish adults from a sub-Arctic population in northern Sweden; roughly half the background population have ancestral lineages within the region (Västerbotten), dating back to the 1700s. Clinical and lifestyle data in GLACIER participants were collected within the on-going Västerbottens Hälsoundersökning (Västerbotten’s Health Survey, VHU; also known as the Västerbotten Intervention Programme) where residents in the county are invited to their primary care centre to complete a detailed health and lifestyle survey and clinical examination at ages of 40, 50, and 60 years. The Regional Ethical Review Board in Umeå approved the study protocol and all study participants provided written informed consent as part of VHU. The study sample is a sub-cohort of 5,160 participants, who received health examinations between 1985 and 2004 and have complete genotype and phenotype data for the analyses.<SEP>Clinical measurements<SEP>The clinical protocol is described in detail elsewhere. Anthropometric data were obtained by trained nurses: body weight was measured to the nearest 0.1 kg and height was measured to the nearest 1 cm. Capillary blood was drawn following an overnight fast and a second sample was drawn 2 hours after a 75-gram oral glucose load. Blood glucose and serum lipid levels were measured with a Reflotron bench-top analyzer (Roche Diagnostics Scandinavia AB). Approximately 5% of the participants reported being fasted for less than 8-hours and 15% of the participants did not report fasting duration. LDL-C levels were calculated using the Friedewald formula. For people on lipid lowering medications, lipid levels were corrected by adding a constant for statins (+0.208 mmol/l for triglycerids, +1.347 mmol/l for total cholesterol, -0.060 mmol/l for HDL-C and +1.290 mmol/l for LDL-C), as previously described. Data outside the following trait-specific intervals were treated as missing: BMI, 15-70 kg/m2; weight, ≥ 35 kg; height, 130-210 cm; total cholesterol, 0.5-15 mmol/l; triglycerids, 0.15-20 mmol/l; HDL-C, 0.15-7 mmol/l; fasting glucose 1-25 mmol/l; 2-h glucose, 1-35 mmol/l. Food intake level (total energy intake divided by estimated basal metabolic rate, 0.63-2.3) was used to exclude participants with improbable self-reported total energy intake. Participants with >10% genotype data missing for the whole genotyping array were excluded.<SEP>Dietary intake assessment<SEP>A validated self-administered food frequency questionnaire (FFQ), designed to quantify habitual diet during the past year, was administered in connection with the health examination. The FFQ initially consisted of 84 foods, food groups and beverage items, but it was shortened to 66 items in 1996 by collapsing similar items and removing others. Daily n-3 (calculated as α-linolenic acid + eicosapentaenoic acid [EPA] + docosahexaenoic acid [DHA]), n-6 (calculated as linoleic acid + arachidonic acid) and total PUFA intake (calculated as n-3 + n-6 PUFA) and total energy intake were then calculated using the Swedish Food Composition Database.<SEP>Variant selection and genotyping<SEP>Genomic DNA was extracted from peripheral leukocytes and diluted to 4 ng/μl. Genotyping was performed using Cardio-MetaboChip array at the Wellcome Trust Sanger Institute (Hinxton, UK). Genotypes for all available variants in the FADS1-FADS2-FADS3 region (Chr.11: 61317028-61416099, build 36; n=290) were extracted and, under the hypothesis that the regulatory regions of these genes may be proximal to the exonic sites, genotypes for variants within 200 kb upstream and downstream of the FADS region (n=436) were also extracted (total n=726). Rare variants with minor allele count <10 were excluded (n=284). Variants with a Hardy-Weinberg equilibrium P-value <0.0001 (n=3) were flagged but not excluded from the analyses, as potential deviations from expected genotype frequencies might reflect evolutionary processes, which are of special interest for the FADS region. Haplotype blocks were constructed based on the algorithm developed by Gabriel et al. using Haploview Java 1.0.<SEP>Statistical analyses<SEP>Step 1: generalised linear regression models were fitted to assess the pairwise interaction effects of six candidate SNPs (rs174570, rs174602, rs74771917, rs3168072, rs12577276, rs7115739) and dietary PUFA intake in relation to eight quantitative traits (weight, BMI, HDL-C, LDL-C, triglycerides, total cholesterol, fasting glucose and 2-h glucose);<SEP>Step 2: gene-centric joint analyses were performed to assess gene × dietary PUFA interaction effects in the whole FADS region, thereby targeting the quantitative traits and PUFA type to be studied in the next step;<SEP>Step 3: to study the statistically significant results from Step 2, haplotype-based gene × dietary PUFA interaction analyses were performed to narrow down haplotype blocks that are likely to interact with PUFA intake in the FADS region;<SEP>Step 4: functional annotation of candidate SNPs from Steps 1 and 3 was undertaken.<SEP>Statistical analyses were performed using R software v3.2.2, PLINK v1.07, gPLINK v2.050. Statistical analyses were carried out in four steps. These steps represent distinct analytic procedures that generally build on the previous steps. The study steps are described below and illustrated in the project flowchart (Figure 1):  <SEP>All models were adjusted for age, age2, sex, FFQ version, and the first four genomic principal components to control for population stratification. To account for potential confounding by total energy intake, we added dietary fatty acid residuals (regressed on total energy intake) along with total energy intake as covariates. The models with body weight as the outcome were adjusted for height. We added fasting status known (1/0 for yes vs. no) and fasting hours (1/0 for >8 h vs. 4–8 h) as dummy covariates in the interaction tests with blood lipid and glucose levels as outcomes. In models where 2-h glucose was the outcome, fasting glucose was included as a covariate. In Step 1, where the six candidate SNPs were analysed, P<0.05 was considered statistically significant as there is a strong prior for these tests, given previous published analyses.<SEP>For the gene/haplotype based analyses in Steps 2 and 3, we applied the method proposed by Chen et al., which is an extension of the sequence kernel association test (SKAT), but further incorporates a joint test of main and interaction effects to maximise the power to detect associations/interactions. For each haplotype block we calculated P values for the interactions with fixed genetic main effects (INT-FIX), interactions with random genetic main effects (INT-RAN), and joint main effects and interaction (JOINT), whilst adjusting for covariates. Bonferroni correction was applied to correct for multiple testing in gene-centric and haplotype-based analyses. As the dietary PUFA exposures and metabolic traits were not independent, we calculated the total number of effective environmental factors (1.37 instead of 3), and effective phenotypic traits (7.38 instead of 8) by accounting for the collective correlation of the two sets. In Step 2, P<0.005 [0.05/(7.38 × 1.37)] was considered statistically significant. In Step 3, P<0.002 [0.05/30] for testing 30 haplotype blocks was considered statistically significant (only one metabolic trait and outcome used here).<SEP>Functional annotation<SEP>In Step 4, the six candidate SNPs from Step 1 and candidate variants located in the haplotypes showing the strongest statistical evidence for interactions from Step 3 were functionally annotated. ChromHMM chromatin state predictions were used to classify the variants into enhancers, repressors, promoters, insulators or others based on chromatin states in nine human cell lines using ANNOVAR. The web based database 3DSNP was used to narrow down the candidate variant list and identify potentially causal variants based on functionality. Initially, we included the six candidate SNPs (rs174570, rs174602, rs74771917, rs3168072, rs12577276, rs7115739) and all common variants (build 137) located in the haplotype blocks generated from the interaction analyses in Step 3. We then screened for variants in high LD (r2>0.8) with the candidate SNPs in the European population (imputed to 1000 Genomes Phase 3 data, using 3DSNP). If a variant was in high LD and had a higher functionality score than its original counterpart, then it was determined to be the leading variant and was prioritized over the original candidate variant in subsequent analyses. All variants were assigned functionality scores based on six functional parameters: i) evidence for disruption of transcription factor binding sites (TFBS); ii) evolutionary conservation, iii) ability to alter sequence motifs; iv) being located in a promoter region; v) being located in an enhancer region; and vi) number of topological interactions with distant genomic regions (genes and/or variants) via chromatin loops. The candidate variants and leading variants were then ranked based on the functionality score. The overall top-ranking leading variant and the most significant candidate SNP were visualized according to six functional parameters and topological features using radar charts and Circos plots, respectively. We searched for quantitative trait loci (QTL) related to lipid concentrations and expression QTL (eQTL) using HaploReg. Gene ontology enrichment analysis was performed on distal interacting genes using the PANTHER Overrepresentation Test to identify relevant biological processes and pathways.",
                  "Results<SEP>Characteristics<SEP>Characteristics for the 5,160 GLACIER participants (38.6 % males and 61.4 % females) are shown in Table 1. Allele frequencies of the six SNPs in the GLACIER cohort are shown in S1 Table. With the exception of rs174570 and rs174602, the derived alleles are of low frequency in this northern Swedish population (DAF<5%). Dietary n-3, n-6 and total PUFA intake were highly correlated with each other (r2 > 0.9 for all pairwise Pearson correlations).<SEP>Dietary PUFA × SNP interactions (Step 1)<SEP>Several gene × dietary PUFA interactions were detected (Pint<0.05) for the six candidate SNPs across the eight traits. The strongest interaction effects were observed between rs174602 and n–3 PUFA intake on total cholesterol (Pint=0.001, Figure 2) and between rs174570 and n-6 PUFA intake on fasting glucose (Pint=0.005, S1 Figure). While rs174602 showed a negative association with total cholesterol (β=-0.09 mmol/l per C allele, 95% CI -0.17; -0.01, P=0.02) among those with low n-3 PUFA intake, no association was observed (β = -0.03 mmol/l per C allele, 95% CI -0.11; 0.05, P=0.50) among those with high n-3 PUFA intake. While rs174570 showed a negative association with fasting glucose (β = -0.06 mmol/l per T allele, 95% CI -0.11; -0.01, P = 0.01) among participants with a high n-6 PUFA intake, no association (β = 0.02 mmol/l per T allele, 95% CI 0.03; 0.95, P = 0.34) was observed in participants who reported low n-6 PUFA intake. Repeating the analysis with inverse normalized traits as dependent variables did not materially change the results. Results for all interaction tests are presented in S2 Table.<SEP>Gene-centric analyses (Step 2)<SEP>As shown in Table 2, a significant interaction effect (PINT_RAN=0.005; qINT_RAN=0.05) was observed between the FADS gene cluster and n-3 PUFA intake on triglycerides with genetic main effects as random, and was followed up with haplotype-based analyses to refine the signal.<SEP>Haplotype block interaction analyses (Step 3)<SEP>From over 700 variants in the FADS gene cluster, 442 variants passed quality control and were used to reconstruct haplotypes (S3 Table). Haploview inferred 30 haplotype blocks across the FADS cluster, and the number of variants in the haplotype blocks ranged from 2 to 63. Focusing on the finding from Step 2, we performed haplotype block based n-3 PUFA interaction analyses in relation to triglyceride levels (Table 3). Although none of the haplotypes surpassed the pre-defined P<0.002 threshold, haplotype blocks 12 (PINT_RAN = 0.01; qINT_RAN = 0.32), 16 (PINT_RAN = 0.01; qINT_RAN = 0.30) and 21 (PINT_RAN = 0.01; qINT_RAN = 0.34) showed tentative signals for interactions; we thus functionally annotated variants within these three haplotype blocks.<SEP>Functional annotation (Step 4)<SEP>Variants from FADS haplotype blocks 12 (n=6), 16 (n=12) and 21 (n=19) and the six candidate SNPs (in total, 43 variants) were functionally annotated (S4 Table) using ANNOVAR. ChromHMM predictions based on nine human cell types demonstrated tissue-dependent functionality for the haplotype blocks and most variants. For instance, the overwhelming majority of variants (18 out of 19) in haplotype block 21 show an enhancer state in the K562 blood (leukemia) cell line, while in the other cell lines, the same variants demonstrate predominantly repressed or inactive chromatin states. As FADS1 and FADS2 are highly expressed in the liver, we assessed chromatin states in the HepG2 liver carcinoma cell line. Here a cluster of variants in haplotype block 21 was enriched with enhancer states, two neighbouring variants (rs187943834, rs117518711) showed promoter states and one variant (rs7115739; a previously reported variant) showed an insulator state. We observed fewer regulatory states among variants in haplotype blocks 12 and 16 in HepG2 cells. Among the other candidate variants, rs174570 variant showed weak promoter, active promoter, or strong enhancer states across multiple cell lines, while the rs174602 variant showed weak transcription elongation state across all but two cell lines.<SEP>To validate the performance of our 3DSNP prioritization pipeline, we first showed that the established causal variant (rs1421085) at FTO (as reported by Claussnitzer et al.) can be detected using the known tagging variant (rs9930506) (S1 Text). The 3DSNP functional annotation analyses of the candidate FADS SNPs is summarised in S5 Table and includes information about the functionality score, distal interacting genes, and the selected leading variants. From the list of tagging variants (prior to the selection of leading variants and ranking by functionality score), rs174570 ranked the highest (score=96.7, driven by promoter status, Figure 3). However, after re-ranking by the leading variant, the FADS2 intronic rs5792235 deletion (CA/C) showed the highest functionality score (218.8), which is 18x higher than that of its proxies, rs174599 (score=12.13; r2=0.83) and rs174601 (score=11.69; r2=0.82), driven by promoter and TFBS status (Figure 3). Eight variants in LD with at least one of the original variants ranked higher than rs174570 (score>96.7; range of 114.0-218.0), many of them driven by high promoter, TFBS and/or enhancer scores. The Circos plot based on rs5792235 (Figure 4) demonstrates 17 proximal (including the three FADS genes) and distal interacting genes (these genes are described in brief in S2 Text), many of which are known to be linked with cardiovascular traits. These findings are supported by HaploReg eQTL annotations (S6 Table), which demonstrate cis and trans eQTL evidence for rs5792235 and multiple other variants in the region. We curated the rs5792235 variant’s 65 distal interacting variants and their associated phenotypes in S7 Table; multiple variants show associations with glycaemic and lipid traits in previous GWAS, many of which are also QTLs for serum lipid traits (S6 Table). Gene ontology enrichment analysis of the 17 genes in distal interaction with rs5792235 revealed enrichment in the unsaturated fatty acid metabolic process (P=5.7×10-3); however, no pathways were identified when FADS1, FADS2 and FADS3 were removed from the analysis. The Circos plot of rs174570 was identical to the one generated for rs5792235, while the one based on rs174602 revealed less interacting nodes.",
                  "Discussion<SEP>We systematically assessed gene × dietary PUFA interactions at the FADS1-FADS2-FADS3 gene cluster across multiple obesity-related traits in a population isolate from the far north of Sweden. There is strong molecular and genetic data implicating FADS gene variation and action in lipid metabolism in several populations. In Greenlandic Inuit, the strongest associations were observed for FADS3 rs7115739 and FADS2 rs174570 and anthropometric traits. These signals persisted when conditioning on other variants in the FADS gene cluster, suggesting independent association signals and multiple causal variants in the region. This observation motivated us to study the entire FADS region in gene-centric and haplotype-centric analyses and undertake functional annotations. Indeed, results emanating from the genomic annotations reveal multiple potential causal variants in the FADS region.<SEP>Specifically, we investigated gene × environment interactions between PUFA intake (and its components, n-3 and n-6 PUFA intake) and the six candidate SNPs (rs74771917, rs3168072, rs12577276, rs7115739, rs174602, and rs174570); these candidate variants have been shown to be under selective pressure.<SEP>FADS1 and FADS2 represent rate limiting steps in the fatty acid metabolism pathway. Multiple genome-wide associations studies have consistently demonstrated that genetic variants at FADS1 and FADS2 are associated with plasma and tissue levels of arachidonic acid and EPA and commonly measured blood lipids, such as triglycerides, total cholesterol, LDL-C and HDL-C. Here we show that the variant rs174570 at FADS2 appears to modify the association between n-6 PUFA intake and fasting glucose levels. Previous studies have investigated the role of FADS2 variants and glycaemic traits; for example, Corpeleijn et al. reported that the increased activity of the Δ-5 desaturase and decreased activity of the Δ-6 desaturase are associated with reduced insulin resistance in a fasted state, and this association is modified by total fat intake. In a gene × environment interaction analysis focusing on 18 FADS SNPs, Cormier et al. observed an interaction between rs174570 and n-3 PUFA intake on insulin sensitivity, but no interaction was observed on fasting glucose levels. Fumagalli et al. reported a putative association between the rs174570 T allele and decreased fasting glucose. Consistently, we found here that the minor T allele of the same SNP was associated with decreased fasting glucose levels, but only when n-6 PUFA intake was high. Cormier et al. also reported a statistically significant interaction between the rs174602 variant and n-3 PUFA intake on insulin sensitivity.<SEP>In our study, statistical interactions were observed between rs174602 and n-3 PUFA intake in levels of total cholesterol and LDL-C, but there was no evidence of interactions in glycaemic traits. Buckley et al. have also reported tentative evidence of interactions between the FADS1 rs174594 and lipid traits. Notably, in the case of the two observed interactions, the associations between the FADS variants and the outcomes are only apparent in the subpopulations with lower n-3 and higher n-6 PUFA intakes (less healthy diets), respectively. In these “unhealthful” strata of GLACIER, the major (compared with minor) alleles are associated with higher total cholesterol and fasting glucose levels. The fact that the major alleles in GLACIER are the minor alleles in the Inuit suggests the presence of a cross-over interaction, where the major allele raises blood cholesterol and glucose in the setting of an unhealthy diet (in GLACIER), and lowers these metabolites in the presence of a healthy diet (in Inuits), relative to the minor allele. Interactions of this nature are consistent with the hypothesis described by Neel with regard to the storage and metabolism of metabolic substrate. Both rs174602 and rs174570 are reported with >10 within loop (proximal) or anchor-to-anchor (distal) interactions, which allows us to speculate on the putative functional background of the observed interactions. Although all these loci are curated in S2 Text, we highlight here that while some of these genes have yet unclear functions, some have putative associations with lipid (BEST1, FEN1, MIR1908, RAB3IL1), glycaemic (MIR1908, MYRF, SYT7), obesity traits (DAGLA, RPLP0P2) or basic cellular functions (INCENP). It is also possible, that following environmental, dietary triggers, different FADS variants interact with different target genes in the region, thereby exerting a complex metabolic phenotype.<SEP>Recently, results from two CHARGE Consortium reports indicated no interactions between FADS variants and dietary PUFAs in relation to two clinical endpoints, type 2 diabetes and coronary heart disease. While these large-scale meta-analyses include larger sample sizes than the GLACIER cohort, the reported lack of interactions might be explained by the analyzed disease outcomes having diverse etiology, using ancestrally different populations, and the dilution of real effects/interactions due to within and between study heterogeneity, as discussed elsewhere.<SEP>Although the Inuit study indicates that variation within the FADS locus may modulate the effects of dietary fats in energy metabolism, the specific variants may not correspond across Greenlandic and northern Swedish populations, owing to their diverse evolutionary backgrounds. Thus, we extended the single variant analyses to include variation across the entire FADS1-FADS2-FADS3 gene cluster using the MetaboChip array. Although sequencing the region would likely provide better coverage and allow analyses of structural elements, such as indels and copy number variations, in terms of MetaboChip, there is an above-average coverage of the FADS region, as this locus was chosen for fine-mapping purposes by the developers of the array.<SEP>Due to the high multiple testing burden, the abundance of low-frequency variants and the small magnitude of interaction effects that were anticipated, we would likely have been underpowered to perform a region-wide single variant interaction analysis, and so this was not attempted. To address this, we undertook a top-down, sequencial approach (described above), where we first assessed the whole region for gene × environment signals to identify exposures and subsequently narrowed down our results to haplotype blocks and single variants through haplotype-based analyses and functional annotation, respectively. We detected a gene cluster interaction with n-3 PUFA intake on triglycerides concentrations, which we scrutinized further by formally testing which haplotype blocks might drive the observed gene-centric interaction effects. Three haplotype blocks (, and) showed suggestive interactions with n-3 PUFA intake on triglyceride levels.<SEP>We chose to employ 3DSNP to undertake detailed in silico functional annotations of selected FADS variants, as this software utilizes publically available HiC datasets to visualize long-range 3D chromosomal interactions, which are able to reveal distal regulatory potential of genomic regions—a key functional feature that many other tools do not include. Inspired by the findings of Claussnitzer et al., we first tested our 3DSNP functional analysis pipeline on the GWAS-identified FTO rs9930506 variant and the subsequently reported causal FTO variant, rs1421085. Using 3DSNP, we successfully validated rs1421085 as the causal variant by demonstrating long-range interactions with IRX5 and a 15x higher functional score compared to rs9930506. We then applied the pipeline on 43 variants from FADS, selected from our previous analyses. Functional annotation of these variants revealed potentially important regulatory signals and topological interactions in the region. The variant showing the strongest evidence of functionality was rs5792235, an intronic FADS2 deletion, with 17 potential interacting genes and high evidence for localizing in a promoter region and a TFBS (Figures 3 and 4). Further annotations and visualizations suggested multiple regulatory variants in the region. Distal interactions of rs5792235 reveal multiple interacting loci with previously demonstrated associations with cardiometabolic traits, mainly lipids and glycaemia. Multiple annotated variants in the region demonstrate QTL evidence for serum lipid traits and eQTL evidence for genes in the proximity.<SEP>Functional annotation of the lead SNPs and haplotypes identified FADS2 rs5792235 as a probable causal SNP. Interestingly, the deletion rs5792235 is common in European (35%), East Asian (57%) and mixed American (60%) populations, while lower frequencies are seen in South Asian (14%) and African (10%) populations. rs5792235 is not captured by any genotyping array, which may explain why this has not previously been implicated in lipid variation.<SEP>A limitation of this study is that the dietary variables were obtained from FFQs. While validated FFQs are often used in large epidemiological studies such as the GLACIER Study, self-report methods are prone to recall bias. However, in the context of these gene-diet interaction analyses, where there is a strong a prior hypothesis, the most serious consequence of reporting bias may be the underestimation of the interaction effects. We also recognize that the in silico functional validations provide only suggestive evidence of functionality, and in vitro analyses, which go beyond the scope of this paper, would be necessary to determine function with very high certainty.<SEP>In conclusion, we provide evidence for gene × environment interactions on a gene variant level for fasting glucose and total cholesterol levels, and on a whole gene level for triglycerides. Through functional annotation and characterization of topological genomic interactions, we identified the intronic rs5792235 FADS2 variant as a potential causal variant (deletion) in the region, as well as multiple interacting genes proximal to FADS2. While contemporary studies often aim to identify single causal master regulators for GWAS-associated loci, our data suggest that it is more likely that multiple causative variants with various regulatory roles in the FADS1-FADS2-FADS3 gene cluster regulate the gene-diet interactions of relevance here.",
                  "29591",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "34",
                  "30373434",
                  "FTO gene polymorphisms are associated with BMI<SEP>FTO gene polymorphisms are associated with body composition<SEP> What is already known about this subject?  <SEP>This study is the first human study to examine the relationship between the changes of FTO gene expression in PBMCs with changes in anthropometric measurements.<SEP> What does this study add?  <SEP>The fat mass and obesity-associated (FTO) locus is one of the most important genetic risk factors for obesity across different populations. There is no general agreement about the mechanism of the effects of the FTO locus on obesity. FTO rs9939506 GG genotype is related to higher body mass index (BMI) compared to the people with AA and AG genotype (Wrzosek, Zakrzewska, Ruczko, Jabłonowska-Lietz, & Nowicka, 2008). FTO rs9930506 is reported to be significantly associated with higher BMI in a G allele dose-dependent manner (BMI + 1.4 kg/m2 per G allele). In other studies, a relationship between the levels of FTO gene expression in the hypothalamus and adipose tissue with body composition was reported. Suppression of the FTO gene in rats resulted in reduction of the ratio of white adipose tissue to brown adipose tissue (WAT/BAT).<SEP>The FTO gene is expressed ubiquitously, and the role of the FTO gene expression in other cells is not yet clear. For example, it is not clear whether the relationship between FTO gene expression and body weight and composition exists in all tissues or is exclusive to some tissues such as adipose and hypothalamus. Recent studies demonstrated a critical role of the FTO gene on cellular metabolic pathways associated with growth and proliferation. Overexpression of the FTO gene increases PI3K/AKT/mTOR pathway activity, which is important in regulating the cell cycle and consequently, reduces apoptosis and allows proliferation. The FTO gene might have a different role in cellular metabolism apart from its effect in adipocytes and the hypothalamus. For example, some recent human studies reported association of FTO gene expression level in peripheral blood mononuclear cells (PBMCs) with different metabolic pathways that affect BMI, ghrelin production, and the risk of type 2 diabetes.<SEP>Today, identification of factors affecting the expression of the FTO gene is especially interesting. It’s reported that knock-down of FTO expression with siRNAs in preadipocyte resulted in a decrease of differentiation into mature adipocytes. Recently, attention has been paid to the use of new drugs to selectively inhibit FTO activity. The effect of FTO genotype on the association between FTO expression and anthropometric indices is not still recognized. Identifying differences in the relationship between FTO gene expression and anthropometric indices in individuals with different FTO genotypes may be helpful in determining those who would benefit from drug therapies.<SEP>The result of previous studies suggested that the role of FTO gene can be varied at different ages. Associations between BMI and the FTO gene are age dependent and the effect of the FTO genotype on BMI becomes evident only after age 7. Another study reported that the effect of FTO single nucleotide polymorphisms (SNPs) may be, in some populations at least, restricted to males and postmenopausal females. Boys have a higher prevalence of obesity than girls in Iran. So, the aim of the present study was to investigate the role of FTO genotype in the relationship between changes in the FTO gene expression level in PBMCs with changes in body weight, BMI, and body composition in obese and overweight adolescent boys.",
                  "Methodology<SEP>This study was an ancillary study within a randomized, controlled, school-based trial that carried out a comprehensive weight-reduction program. Personalized diet and physical activity intervention was implemented for each participant. In addition, parents were provided an educational session on healthy diet and physical activity to creating a supportive environment at home. The personalized diet was adopted with free snacks offered in school days by researchers. Furthermore, a high-intensity interval training was carried out for improving the physical activity at the schools. In this method, students warmed up for 10 min and they were involved in high-intensity exercise for a minimum of 30 min. The details of the interventions were published previously. The study involved students in two high schools (7th, 8th, and 9th grade students) of a district of Tehran city that was chosen randomly (district 5). The original study had an intervention group (n = 44) and a control group (n = 40). Data from the control and intervention groups were combined for purposes of the present study (n = 84).<SEP>Study Population<SEP>In total, 96 students expressed interest in participating in the ancillary study. Twelve were excluded due to failure to collect blood samples and 84 students enrolled and provided both baseline and week 18 blood samples. The inclusion criteria were age between 12 and 16 years and BMI percentile reported as ⩾85th percentile for age and sex. Non-inclusion criteria included diagnosed weight-related diseases (including hypothyroidism, insulin resistance, and Cushing’s syndrome) and use of weight-related medications (including drugs for diabetes, such as insulin, thiazolidinediones, and sulfonylureas; antipsychotic drugs such as haloperidol, clozapine, and lithium; and antidepressant drugs like amitriptyline, paroxetine, and sertraline) determined by self-report by the participants.<SEP>The aims, methods, and the other details of the project were explained to the participants and their parents, and written consent forms from both parents and students were obtained. All measures were taken between morning and noon at the baseline and after 18 weeks of the study.<SEP>Anthropometric Measures<SEP>The measures of baseline and week 18 were obtained by trained personnel. The height of students was measured with a calibrated tape line fastened to a wall. A bioimpedance analysis scale (Omron BF511, Kyoto, Japan) was used to measure weight, BMI, %body fat, and %skeletal muscle after entering their age, gender, and height. This device is a digital, mobile, and non-invasive device that has eight electrodes that sends an extremely weak electrical current of 50 kHz and less than 500 πA through the body to determine the amount of fat tissue. The validity of this device has been confirmed in a previous study. All data were classified according to the z-score guidelines defined by WHO recommendations (for weight and BMI) and recently published papers (for %body fat and %skeletal muscle).<SEP>Gene Expression<SEP>At baseline and week 18, blood samples (5 ml) were collected from all students who participated in the study, and then transferred to EDTA tubes and stored at −80 °C. The PBMCs were isolated from anticoagulated peripheral blood by density centrifugation, and total RNA was subsequently isolated using the GeneAll RNA extraction kit (GeneAll, South Korea), cDNA synthesis was performed using the cDNA synthesis kit (Gene All, South Korea), and mRNA expression levels were determined using the Opticon real-time PCR detection system (Bio-Rad Laboratories, California). Reactions were carried out in duplicate using the SYBR Green Gene Expression Master Mix (Cat. no. 638317; Takara). The Hypoxanthine Phosphoribosyltransferase (HPRT) gene was chosen as a reference gene for its stable expression in PBMCs. Data were analyzed using the 2−∆∆Ct method and expressed as fold change.<SEP>Genotyping<SEP>The DNA extraction kit manufactured by GeneAll was used to extract and purify the DNA samples. The NanoDrop device (Thermo Scientific, Wilmington, DE, USA) was used to quantify the DNA concentration. The optical density (OD) of the samples was obtained in the absorption rate of 260–280, and it was confirmed if the OD was from 1.8 to 2. Moreover, to check the quality of the extracted DNA, electrophoresis using the agarose gel technique was used. In brief, genomic DNA was amplified by PCR using the Taq DNA Pol 2X Master Mix Red (Cat. no. A180301; Ampliqon, Denmark). The PCR products were sent to GeneAll for DNA sequencing. The quality and average length of the sequence library for each sample were assessed using the Chromas software (version 2.33, http://www.technelysium.com.au/chromas.html).<SEP>Assessment of Other Variables<SEP>Usual dietary intakes of participants were examined by a validated 168-item semi-quantitative food frequency questionnaire (FFQ). The FFQ consisted of 168 food items with standard portion sizes commonly consumed by Iranian people. A trained interviewer administered the FFQ through face-to-face interviews. All reported consumption frequencies were converted to grams per day by using household measures. Daily intakes of energy were measured for each person by using the U.S. Department of Agriculture food consumption database, which was modified for Iranian foods.<SEP>The International Physical Activity Questionnaire long form (IPAQ-LF) was used for measuring physical activity of participants through the face-to-face interview. All results of the IPAQ were expressed as metabolic equivalents per minute (MET-minutes per week).<SEP>Statistical Analysis<SEP>Given the age range of the participants, we expect weight and body composition changes in these individuals to be significant over a period of 18 weeks. We aimed to test the hypothesis that there is an association between changes of the level of FTO gene expression with anthropometric indices over a time period regardless of the effect of dietary intake and physical activity. All associations were evaluated under dominant models. Multiple linear regression was used to determine the relationship between changes in FTO gene expression with changes of anthropometric parameters of subjects with different FTO genotypes after adjusting for age, physical activity, and calorie, protein and carbohydrate intake. We confirmed that the assumptions of the linear regression model were met. Statistical analyses were performed using SPSS version 23.0 (IBM SPSS Statistics for Windows, IBM Corp., Armonk, NY, USA). The results were considered statistically significant at p < .05.",
                  "Results<SEP>At baseline, individuals with GG genotype of rs9930506 had higher BMI, %body fat, and lower %skeletal muscle compared to AA or AG genotype carriers (p = .02, .001, and .001, respectively) (Table 1). After 18 weeks, the participants with GG genotype experienced the following mean anthropometric parameter reductions: weight 6.8 kg, BMI 1.69 kg/m2, and %body fat 0.78%. Moreover, mean of %skeletal muscle increased by 0.61%, and the FTO expression was 0.15-fold higher after 18 weeks than baseline. The participants with AA or AG genotype experienced the following mean anthropometric parameter reductions: weight 0.21 kg, BMI 0.69 kg/m2, and %body fat 0.84%. Overall, mean of %skeletal muscle increased by 0.35%, and the FTO expression was 0.51-fold higher after 18 weeks than baseline. There were no outliers in the data, as assessed by box plotting.<SEP>At baseline, there was no significant association between FTO expression level and anthropometric measurements (Table 2). Under the dominant model, we observed a significant association between change of the level of gene expression and change of %skeletal muscle. The gene expression fold change was significantly associated with change in %skeletal muscle in AA or AG genotype carriers (β = 0.34, p = .02). No significant association was detected between the change in FTO gene expression with change in anthropometric indices in GG genotype carriers (Table 3).",
                  "Discussion<SEP>The results identified an association between up-regulation of FTO in PBMCs with increase in %skeletal muscle at 18 weeks in individuals with AA or AG genotype of FTO rs9930506. However, there was no association between changes in FTO gene expression and weight, BMI, and %body fat, regardless of their genotype. The association between FTO gene expression and anthropometric measures has been reported in several studies. Berulava and Horsthemke reported that increased expression of FTO in PBMCs is associated with increased body mass. Most of the studies on the interactions between FTO gene expression and body composition have focused on the expression level of the FTO gene in the hypothalamus and adipose tissue due to its hypothetical role in the regulation of energy intake and adipocyte differentiation. For example, identified that ubiquitous overexpression of FTO in the hypothalamus leads to a dose-dependent increase in body and fat mass. In the present study, no significant association was identified between the level of FTO gene expression in PBMCs and BMI and body fat which contrasts with the results of other studies on FTO gene expression and anthropometric parameters. The effect of FTO expression on anthropometric measurements may be tissue specific. It has been strongly suggested that the role of FTO in the tissues other than adipose tissue might be completely independent from its role in obesity. Moreover, the relationship between FTO expression and anthropometric measurements may be age-specific. It’s reported that the effects of rs9930506 variants on body size varied with age and the carriers of the risk allele develop an increase in body weight earlier in life. The associations of FTO variant with BMI is identified to be more significant during childhood and adolescence, peaked in early adulthood, and then weakened in adult age. It can be hypothesized that the effect of FTO on body composition may be more pronounced in adolescence and early adulthood when people have the most muscular mass in the entire lifetime.<SEP>The underlying mechanisms for the effect of FTO gene expression on body weight and body composition are not recognized. However, the role of FTO in the hypothalamus is likely different from its role in other tissues. Recent studies have reported that FTO may have an essential role in metabolic-related signaling in most cells. For example, it is suggested that the FTO gene regulates the proliferation and differentiation of cells via PI3K/Akt signaling, and FTO knockdown leads to suppression of cell proliferation.<SEP>As the results of the present study indicated, FTO gene polymorphisms can modify the expression level of FTO and other BMI-related genes such as IRX3 and the regulation of the expression of FTO gene (e.g., by drugs or dietary supplements) can induce desired changes in individuals with desired genotype. A separate study on the participants of the present study identified that the FTO gene polymorphism rs9930506 had a risk allele frequency of 40% and was strongly associated with higher %body fat and BMI. So, we should consider FTO genotyping in all studies focused on the association between FTO gene expression and its clinical manifestations and diseases including obesity.<SEP>However, some other factors can modify the effects of the level of FTO gene expression. reported that the association between the FTO gene and BF and BMI could be altered at different ages and FTO genotype is associated with BMI after the age of 7 years. In the present study, we investigated the changes in the expression of FTO gene in adolescents who are experiencing physical growth to identify the association between FTO genotype with changes in FTO expression and body composition. Environmental factors such as lifestyle can also play a key role in the interaction between FTO gene expression and anthropometric measures. For example, the level of FTO gene expression is influenced by dietary intake and the effect of the FTO gene expression on obesity may be influenced by dietary components. Although, we adjusted differences in dietary intake and physical activity. So, it can be inferred from the results of the present study that the level of FTO expression has a positive association with %skeletal muscle regardless of the effect of dietary intake and physical activity.<SEP>Overall, in the present study, we tried to identify differences between individuals with different FTO genotypes in the relationship between FTO gene expression and anthropometric indices. These results may be helpful in determining those who would benefit from drug therapies. However, it is also important to mention here that relationship between FTO gene expression and anthropometric indices might be tissue-specific and potential treatment related decisions may need to be made in the context of the tissues from which FTO expression is being assessed.<SEP>The present study had some limitations. This study was confined to male adolescents. Associations between BMI and the FTO gene may be age dependent and more evident in males. Future studies in both sexes, a broader age range, and in a longer period of time are needed to address these limitations. The significance level was not adjusted for the number of tests performed in order to not limit the power of the study to discover novel associations. Thus, the results should be reproduced in independent cohorts.",
                  "17129",
                  "['OBWCCE']"
                ],
                [
                  "35",
                  "27437873",
                  "INTRODUCTION<SEP>Multiple myeloma is an incurable and heterogeneous plasma cell neoplasm that affects about 6.3 per 100.000 people per year worldwide (i.e., 25.850 new cases only in 2015) and represents 1.6% of all cancers and 2% of all cancer deaths. In spite of the widespread use of proteasome inhibitors and immunomodulatory drugs, which has dramatically improved the life expectancy of MM patients over the last few decades, MM survival still remains poor with a 5-year survival of 46.6% (SEER Cancer Statistics Review, http://seer.cancer.gov/csr/1975_2012/).<SEP>Epidemiological and observational studies have consistently identified several factors that affect MM patient survival such as age at diagnosis, stage at diagnosis (coded by either the Durie-Salmon staging system (DSS) or the International Staging System (ISS)), Eastern Cooperative Oncology Group (ECOG) performance status, renal failure, high plasma cell proliferative rate, high lactate deshydrogenase (LDH) levels and chromosomal abnormalities. Increasing evidences point towards a positive correlation of pre-existing type 2 diabetes (T2D) with MM risk but also with the appearance of severe clinical complications and patient survival. In this regard, Chiu et al. (2006) reported that high level of postload glucose was associated with increased risk of mortality in hematological malignancies whereas Chou et al. (2012) reported that MM patients with pre-existing T2D have 50% higher all-cause mortality compared with non-diabetic patients. These observations might be explained, at least in part, by the stimulatory effects of T2D-associated hyperglycaemia, insulin resistance and resulting hyperinsulinemia on MM cell growth but also by the deregulation of tumour-suppressor genes linked to T2D (such as CDKN2A-2B, KCNQ1, HNF1B) that might lead to uncontrolled cell proliferation, cell differentiation and disease progression and, consequently, to shorter survival periods. In support of this notion, CDKN2A-2B genes have been found to be frequently hypermethylated in MM whereas loss of expression of KCNQ1 has been associated with poor overall survival in cancer patients. Furthermore, emerging evidences also suggest that the activation of certain T2D-related genes (such as NOTCH2) may induce MM cell migration from the infiltrated site to different bone marrow districts and promote osteoclast formation, which is a process intimately related to proliferation and long-term survival of MM cells.<SEP>Although germline variants may influence the susceptibility of MM and survival, the knowledge regarding the role of diabetogenic variants in modulating the risk of MM and survival remains scarce. We have recently reported that diabetogenic variants influence MM risk and recent genome-wide association studies (GWAS) have also suggested the involvement of genetic variants within the MTHFD1L, AKAP12 and FOPNL loci in determining MM patient survival but also an indirect implication of diabetogenic genes such as TCF7L2. Johnson et al. (2016) reported in their GWAS a strong association of rs12374648, which maps to a binding site for the transcription factor TCF7L2, with MM overall survival (OS) and proposed a functional mechanism of this variant to modulate the synthesis of purines and the regulation of cell cycle.<SEP>Based on these findings, we explored for the first time the relationship between diabetogenic variants and OS of MM patients in a study developed in the context of the International Multiple Myeloma rESEarch (IMMENSE) consortium. We attempted to confirm our findings by analysing GWAS data on an independent set of German MM patients (Heidelberg cohort).",
                  "MATERIALS AND METHODS<SEP>Patients, clinical data collection and survival endpoint definition<SEP>A total of 1420 Caucasian MM patients were ascertained through the IMMEnSE consortium. Full details of this consortium have been published elsewhere. In brief, inclusion criteria were newly diagnosed MM with Salmon & Durie stage I, II and III, age 18–90 years inclusive and Caucasian origin. DNA was purified from blood specimens using the QIAamp DNA Blood Mini Kit (Qiagen) and clinicophathological characteristics including age, gender, country of origin and disease stage (Durie-Salmon) were retrospectively gathered from medical records in each participant institution (Table 1). Diagnosis of patients with symptomatic MM was carried out by hematologists according to the International Myeloma Working Group (IMWG) criteria. All patients within the IMMENSE consortium for whom survival information was available were included in the study (936 MM cases, 454 women and 482 men) (Table 1). All participants gave their written informed consent to participate in the study.<SEP>SNP selection and genotyping<SEP>Fifty-eight variants were selected based on the GWAS for T2D and were genotyped in the IMMEnSE consortium population (Table 4). We considered only SNPs that were replicated in large and independent populations or which came up in several GWAS or their meta-analyses. Additional criteria were potential functionality and linkage disequilibrium (LD) between the reported SNPs. The genotyping of the selected polymorphisms was carried out at GENYO (Centre for Genomics and Oncological Research: Pfizer/University of Granada/Andalusian Regional Government, Granada, Spain) using KASPar® assays (LGC Genomics, Hoddesdon, UK) according to manufacturer's instructions. For internal quality control, 5% of samples were randomly selected and included as duplicates. Concordance between the original and the duplicate samples for the 58 SNPs was ≥ 99.0%. Call rates for all SNPs were ≥ 90.0% with the exception of the WFS1rs734312 SNP that was excluded from further analyses.<SEP>Replication<SEP>For replication purposes, seven hundred MM patients (296 women and 404 men) were provided by the University Clinic of Heidelberg (Germany). This cohort consists of 98 GMMG-HD3 trial patients, 291 GMMG-HD4 trial patients and 311 patients transplanted in Heidelberg but not enrolled in clinical trials (Table 1). Ethical approval for these patients and written informed consent of trial patients was also obtained. Clinical and survival data were prospectively collected for trial patients on case report forms and retrospectively gathered from medical records for none-trial patients. Genetic information of 53 SNPs (36 genotyped SNPs and 17 imputed SNPs) was extracted from the GWAS conducted in the Heidelberg cohort. After imputation, no information was available for 5 SNPs.<SEP>In silico functional analysis<SEP>Haploreg (http://www.broadinstitute.org/mammals/haploreg/haploreg.php) and ENCODE annotation data (https://genome.ucsc.edu/ENCODE/) were used to predict the functional role of potentially interesting SNPs.<SEP>eQTL analysis<SEP>We also assessed whether selected SNPs correlated with mRNA expression levels in a public eQTL browser for peripheral blood mononuclear cells (http://genenetwork.nl/bloodeqtlbrowser/). Expression quantitative trait loci (eQTL) data on malignant plasma cells of 658 patients from the University Clinic of Heidelberg (Germany) were also available for this study. Detailed information on sample collection and clinico-pathological characteristics of MM patients as well as technical details of gene expression analysis have been published elsewhere.<SEP>Statistical analysis<SEP>We used chi-square tests to assess Hardy–Weinberg Equilibrium (HWE) for each SNP among IMMEnSE patients. The primary outcome was OS and the endpoint was defined as death from any cause. Survival time was calculated as the time from MM diagnosis (discovery population) or the first stem cell transplantation (replication population) until the occurrence of the study endpoint, censoring at the date of death or the last observed follow-up time. Association with OS defined as hazard ratio (HR) was calculated for each SNP using Cox regression multivariate analysis adjusted for age, gender, country of origin and Durie-Salmon stage (IMMENSE cohort) or for age, gender and clinical trial (Heidelberg cohort). Association estimates were calculated according to dominant, recessive and log-additive models of inheritance with the major allele as reference for regression analyses (Table 4). We also performed gene-gender interaction analyses to determine whether the association between SNPs and MM OS was of similar magnitude in men and women. Survival function was displayed using the Kaplan-Meier method and survival differences across genotypic groups were analysed using the log-rank test.<SEP>In order to account for multiple comparisons, we used the Meff/MeffLi method, which calculates the effective number of independent genetic markers analysed (N = 54) on the basis of the spectral decomposition (SpD) of matrices of pairwise LD between SNPs (http://neurogenetics.qimrberghofer.edu.au/SNPSpDlite). In addition, we also considered the number of genetic inheritance models tested (dominant, recessive and log-additive). This resulted in a study-wide significance threshold of 0.00031 ([0.05/54]/3) to keep type I error rate at 5%.<SEP>Finally, in order to confirm significant associations, a meta-analysis combining genetic data obtained in the IMMENSE population with those extracted from the GWAS conducted in the Heidelberg cohort was also performed following dominant, recessive and additive models of inheritance. The I2 statistic was used to assess heterogeneity between both studies and the pooled HR was computed using the random-effect model (assuming that between-study variation might depend on chance or random variation and an individual study effect). Random-effects models are more conservative than fixed-effects models and give rise to wider confidence intervals (CI), which ensures the reliability of the results even though the data come from studies with a relatively different design. All statistics were calculated using SPSS (v.20) and STATA (v.12) for MAC.",
                  "RESULTS<SEP>The demographic and clinical characteristics of the MM patients included in the IMMENSE (n = 939) and Heidelberg (n = 700) cohorts are listed in Table 1. The median age at diagnosis was similar in both populations (59.73 ± 10.08 vs. 55.85 ± 8.33) but the male/female ratio was higher in the Heidelberg cohort (1.36 vs. 1.06). Durie-Salmon stage was available for IMMENSE and Heidelberg cohorts and included patients at stages I, II and III (11.83%, 23.54% and 64.63% vs. 0.8%, 12.4% and 86.8%, respectively).<SEP>All SNPs tested showed genotype frequencies consistent with the HWE (P > 0.001) and the observed allele frequency for all selected SNPs was in accordance with Hapmap data. When we evaluated the effect of selected polymorphisms on MM OS in the IMMENSE population, we found that 6 SNPs showed a noteworthy association with OS. The most relevant effect was observed for the HNF1Brs7501939 SNP that was associated with poor OS when recessive and log-additive models of inheritance were assumed (HRREC = 1.49, 95% CI 1.11–2.00, P = 0.008 and HRADD = 1.34 95% CI 1.13–1.59, P = 0.001, respectively; Table 2). Patients harbouring the non-diabetogenic HNF1Brs7501939T/T genotype showed a median survival time (MST) significantly shorter than those carrying the C allele (MSTT/T = 81.91 months vs. MSTC/C+C/T = 101.42 months; Figure 1A). This result was confirmed with the 700 MM patients recruited from the University Clinic of Heidelberg (HRREC = 1.40, 95% CI 1.06–1.84 and MSTT/T = 74.4 months vs. MSTC/C+C/T = 97.2 months; Table 3 and Figure 1B). The result of the meta-analysis for this SNP remained significant after correction for multiple testing (HRMeta-Rec = 1.44, 95% CI 1.18–1.76, P = 0.0001, I2 = 0.0%, PHet = 0.74; Table 3). According to publicly available eQTL data for human peripheral blood mononuclear cells, the risk allele (T) was associated with higher HNF1B mRNA expression levels (Z score = 3.31, P = 9.23·10–4 and FDR = 0.23). However, we could not validate this finding using eQTL data on plasma cells from 665 German MM patients (P = 0.60; Supplementary Table S1). Nonetheless, according to Haploreg and ENCODE annotation data, the rs7501939 SNP resides near of a poised promoter in many cell lines including a lymphoblastoid and human stem cell lines (GM12878 and H1-HSCs) that might be rapidly activated upon specific stimuli. In addition, this SNP was predicted to change binding motifs for 2 regulatory transcription factors (CEBPB and p300) and mapped among enhancer histone marks in primary naïve and memory forms of cytotoxic T cells (CD8+) and helper T cells (CD4+) from peripheral blood.<SEP>We also observed significant associations at P < 0.05 for SNPs within CDKN2A-2B, GCKR, KCNQ1 and SLC30A8 genes with OS in the IMMENSE population. Thus, patients carrying the KCNQ1rs2074196T and SLC30A8rs13266634T alleles or the GCKRrs1260326T/T genotype had an increased risk of death whereas subjects bearing the CDKN2A-2Brs564398C/C genotype showed longer OS (Table 2 and Supplementary Tables S2–S4). The association of the SLC30A8rs13266634T allele with OS was confirmed in the Heidelberg population and the meta-analysis showed that the presence of each additional copy of the SLC30A8rs13266634T allele was associated with poor OS (HRMeta-Add = 1.22, 95% CI 1.09-1.37; Table 3). Although the association of the SLC2A2rs11920090 SNP with OS was not significant in the IMMENSE population, we observed a significant association of this variant with MM survival in the Heidelberg population that remained significant in the pooled analysis. Patients harbouring the SLC2A2rs11920090T allele showed a better survival compared with those carrying the A/A genotype (HRMeta-Dom = 0.80, 95% CI 0.66-0.97; Table 3). The meta-analysis also showed a weak association of the TCF7L2rs7903146T allele with better survival that was neither significant in the IMMENSE population nor in the Heidelberg cohort (HRMeta-Dom = 0.84, 95% CI 0.71–0.98). Based on Haploreg data, the missense rs13266634 SNP was predicted to change binding motifs for transcription factors implicated in tumorigenesis (AP1 and PAX5) and mapped on enhancer histone marks in several human embrionic stem cell lines. In addition, this polymorphism affects binding to 5 proteins implicated in cancer development (CCNT2, GATA2, TAL1, KAP1 and CTCF). On the other hand, the rs11920090 and rs7903146 SNPs mapped among enhancer and promoter histone marks in bone marrow- and/or adipose-derived mesenchymal stem cells. In addition, the rs7903146 SNP was predicted to alter the binding site of 7 transcription factors. However, despite the consistency and the potential interest of these findings, none of the associations of the SLC2A2, SLC30A8, and TCF7L2 SNPs with OS remained significant after correction for multiple testing and, therefore, require further confirmation. Given the lack of genetic information in the GWAS conducted in the Heidelberg population for SNPs within MADD and KCNQ1 genes, we imputed genotypes to test whether the preliminary associations observed in the IMMENSE population could be validated. Although there was no imputed data available for the KCNQ1 SNP, the meta-analysis of IMMENSE data with imputed genotypes of MADD variants in the Heidelberg cohort suggested a link between this locus and MM survival (HRMeta-Rec = 0.75, 95% CI 0.57–0.99; Table 3).<SEP>Based on the evidences that point toward the existence of gender-associated differences in survival for patients with MM, we decided to carry out a gender-stratified analysis. This analysis revealed gender-specific associations for SNPs within or near the ADAMTS9, KCNJ11, PROX1 and SLC30A8 genes with OS. We found that men carrying the KCNJ11rs5215C or SLC30A8rs13266634T alleles had poorer OS compared with those harbouring the wild type genotype whereas an opposite but not significant effect was seen in women (PInteraction= 0.022 and PInteraction = 0.057, respectively; Table 2 and Supplementary Table S2–S4). We also observed that women carrying the PROX1rs340874G allele or the ADAMTS9rs4607103T/T genotype experienced a poorer survival with an opposite but not significant effect in men (PInteraction = 0.016 and PInteraction = 0.024). In order to confirm these gender-specific associations, we performed a meta-analysis with available GWAS data of the Heidelberg population. Although there was a partial overlapping of SNPs between both populations that limited our ability to validate some potentially interesting gender-associated effects on OS, we could confirm the strong association of the SLC30A8rs13266634 SNP with OS in men that could not be detected in women (per-allele HRMen = 1.32, 95% CI 1.13-1.54; Supplementary Table S5). This gender-specific association remained significant at the experiment-wide significance threshold. On the other hand, although it was not statistically significant in the analysis of the IMMENSE population, the pooled analysis also showed that men carrying the BCL11Ars10490072C allele had a poorer OS compared with those carrying the wild type genotype whereas no effect was seen in women (HRMen = 1.37, 95% CI 1.10-1.70). Finally, we observed in the pooled analysis that women bearing the PRC1rs8042680A allele or men carrying the PROX1rs340874G allele or the MADDrs7944584T/T genotype showed a significantly better OS when compared with those patients carrying the corresponding wild type allele or genotype (HRWomen = 0.62, 95% CI 0.39–0.98; HRMen = 0.74, 95% CI 0.59–0.94 and HRMen = 0.59, 95% CI 0.39–0.87; Supplementary Table S5). The regulatory characteristics of the rs10490072, rs8042680 and rs7944584 SNPs were changes in transcription binding motifs for transcription factors involved in tumorigenesis and T- and B-cell malignancies. The rs10490072 changed sites for HNF4B, Pou2f2 and Pou5f1 whereas the rs8042680 altered sites for GR, PAX5 and TAL1. The rs7944584 was found to modify regulatory motifs for AP1, AP4, IRF and KAP1. Finally, the rs340874 mapped among promoter and enhancer histone marks in primary naïve and memory forms of helper T cells (CD4+) and regulatory T cells from peripheral blood.",
                  "DISCUSSION<SEP>Previous population-based studies have demonstrated the impact of GWAS-identified variants for T2D on cancer susceptibility and patient survival. However, despite these important research advances, there is still a noticeable lack of information regarding the role of T2D-related variants in modulating patient survival especially in hematological malignancies. In this scenario, we decided to investigate for the first time to our knowledge the relationship between 58 genetic variants associated with T2D identified by GWAS and OS of MM patients.<SEP>The analysis of the IMMENSE consortium data revealed a significant association of the intronic HNF1Brs7501939 SNP with poor OS. We successfully replicated this association in a large and independent population recruited by the University Clinic of Heidelberg. However, although a positive correlation between this variant and eQTL data on PBMCs has been reported, we failed to find correlation between the risk allele and HNF1B mRNA expression levels on plasma cells from a large cohort of MM patients. This suggested that the effect of this variant on overall survival is not mediated by changes in transcriptional activity of the gene. Nonetheless, given that HFN1B contains multiple independent SNPs or haplotypes that have been associated with HNF1B mRNA expression and methylation levels but also with the risk of developing several types of cancer, it seems to be reasonable to consider the possibility that other SNPs within this locus and showing a stronger association with OS could explain better the link between the HNF1B and clinical outcome. However, when we analysed imputed common SNPs from the GWAS conducted in the Heidelberg cohort, we could not find any stronger association signals with OS in the region, which suggested that the HNF1Brs7501939 SNP or perhaps a rare SNP in LD with it might be responsible of the observed effect. Future fine-mapping studies encompassing common but also rare variants within or near the HNF1B locus are needed to elucidate whether a rare variant or haplotype might account for the observed effect.<SEP>HNF1B contains 9 exons and expands over 58 kb on chromosome 17p21. It encodes for a transcription factor that has been associated with multiple clinical features including early-onset of T2D. In line with this, it has been also suggested that HNF1B may induce impaired glucose tolerance and attenuated insulin sensitivity in a miRNA-dependent manner, which might lead to an enhanced insulin secretion and the activation of the IGF1 pathway, an important factor mediating myeloma cell growth, proliferation and cell maturation. Alternatively, it has been postulated that HNF1B is able to influence cancer cell survival by promoting the activation of NFkB pathway or through the inhibition of mitochondria-associated apoptotic signals. In support of the tumorogenic effect of HFN1B, it has also been reported that it may act as an oncogene and that the HNF1B gene is amplified in 23% of all cancers and in about 5% of all haematological malignancies (http://broadinstitute.org/tumorscape). On the contrary, it has also been reported that HFN1B may act as tumour suppressor gene and that its expression may largely vary depending on the target tissue. Whereas HNF1B has been found to be overexpressed in ovarian clear cell carcinomas and prostate or endometrial cancers and its silencing induces apoptosis of cancer cells, it has been found to be down-regulated in serous epithelial ovarian cancer and colorectal, gastric and pancreatic cancer cell lines. In addition, it has been reported that the down-regulation of HFN1B gene is associated with progression in hepatocarcinoma and poor prognosis in renal and prostate cancers. Considering all the above but also the fact that the association of the HNF1Brs7501939 SNP with OS was driven by a non-diabetogenic (T) allele that does not affect HNF1B mRNA expression, we hypothesize that the effect of this variant to contribute to tumour progression in MM might be mediated by a non-insulin-dependent mechanism. There was a reasonable amount of regulatory data for the HNF1Brs7501939 SNP that supported evidence of the active role of the HNF1B locus. However, whether elevated HNF1B levels lead to tumour transformation and disease progression is not yet understood and functional studies to examine whether HNF1B variants influence cancer prognosis are lacking.<SEP>Another interesting finding of this study was the association of the BCL11A, MADD, PRC1, PROX1, SCL30A8, SLC2A2 and TCF7L2 SNPs with OS. We found an overall association of the SLC2A2rs11920090T and TCF7L2rs7903146T alleles with better OS whereas the association of the SLC30A8rs13266634T, BCL11Ars10490072C, PRC1rs8042680A and PROX1rs340874G alleles or the MADDrs7944584T/T genotype with OS was restricted to male or female genders. Despite the potential interest of the associations observed for these SNPs with OS, only the association of the SLC30A8rs13266634 SNP with poor OS in men reached significance at experiment-wide significance threshold. This result suggested a key role of the SLC30A8 locus in the modulation of overall survival. However, given the consistency of the overall or gender-specific associations observed for BCL11A, MADD, PRC1, PROX1, SLC2A2 and TCF7L2 SNPs with OS across the populations tested and considering that gender-specific genetic alterations might influence MM survival, we suggest that these variants might also exert a modest effect to modulate patient survival.<SEP>SCL30A8 gene encodes a zinc transporter involved in the control of insulin processing and secretion. Although no previous studies have reported a link between this locus and MM, there are evidences that suggest that zinc transporters might contribute to cancinogenesis through a gender-dependent mechanism. The association of the coding SLC30A8rs13266634 SNP with OS was due to a non-diabetogenic allele suggesting that, rather than modulating glucose homeostasis and insulin secretion, the effect attributed to the SLC30A8 locus on MM survival might be driven by a direct effect of Zinc in biological processes such as DNA and RNA stabilization, binding of protooncogenes to DNA and the activation of IGF1 or telomerase. The SLC30A8rs13266634C allele has been consistently associated with decreased rates of Zinc transport activitiy and reduced intragranular Zinc levels. However, eQTL data on plasma cells from MM patients did not reveal correlation between this variant and SLC30A8 mRNA levels suggesting that, rather than regulating gene expression, the T allele affect transporter activity in an allele-dose-dependent manner causing increased Zinc concentration and thereby promoting unlimited proliferation of MM cells, disease progression and poor survival. In addition, regulatory data suggest that the SLC30A8 locus might play a role in survival through the modulation of specific transcription factors implicated in tumour promotion and dissemination.<SEP>As for the HNF1B and SLC30A8 SNPs, the association of the BCL11Ars10490072 and MADDrs7944584 SNPs with OS was determined by non-diabetogenic alleles. BCL11A functions as a myeloid and B-cell proto-oncogen and has been associated with the development of B-cell malignancies whereas MADD encodes for a MAP-kinase activating cell domain involved in the control of physiological cell death through TNF- and caspase-dependent apoptosis. In contrast to these associations, the association of the TCF7L2rs7903146,SLC2A2rs11920090 PRC1rs8042680, and PROX1rs340874 SNPs with OS was driven by diabetogenic alleles, which suggested that the effect of these variants on OS might be explained by their regulatory effect on insulin secretion and, consequently, on cell proliferation and tumour cell growth. Whereas SLC2A2 encodes a highly efficient glucose transport that is expressed in pancreatic cells and regulates insulin secretion by modulating entry of glucose into the pancreatic cell, TCF7L2, PRC1 and PROX1 are proteins that have been involved in β-cell survival and function and in glucose and nonesterified fatty acids or branched-chain amino acids metabolism in liver. Despite these interesting results and the regulatory data observed for all these SNPs, the lack of information regarding T2D status among MM patients did not allow us to ensure that the observed effect of the TCF7L2, SLC2A2, PRC1, and PROX1 SNPs on OS could not be due to a different distribution of diabetic patients when grouping by genotype or gender.<SEP>This study had both strengths and limitations. Strengths include the use of relative large discovery and replication populations that allowed us to validate the most interesting associations. Limitations include lack of information regarding the classical genetic prognostic factors (chromosomal abnormalities, etc.), T2D status and a relatively small statistical power to detect modest associations with OS, especially when gender-stratified analysis were performed. Another limitation was the partial overlapping of genetic information between studies and the use of imputed genotypes that did not allow to perform a reliable validation of the association observed for genetic variants within ADAMTS9, BCL11A, KCNQ1, MADD and PROX1 genes with overall survival.<SEP>In conclusion, this study reports the first evidence of an association between the HNF1Brs7501939 SNP and OS for MM and suggests that the HNF1B locus might, likely through a non-insulin-dependent mechanism, play an important role in modulating MM prognosis. Likewise, this study shows a strong association of the SLC30A8rs13266634 SNP with poor OS in men that might, at least in part, account for gender differences in OS. Additional studies using larger and well-characterized populations are needed to further replicate these findings but also those involving the BCL11A, MADD, PRC1, PROX1, SLC2A2 and TCF7L2 loci on OS.",
                  "28046",
                  "['OBWCCE']"
                ],
                [
                  "36",
                  "18694974",
                  "Type 2 diabetes is a multifactorial disease caused by a complex interplay of multiple genetic variants and many environmental factors. With the recent genome-wide association (GWA) studies, the number of replicated common genetic variants associated with type 2 diabetes has rapidly increased. A total of 18 polymorphisms have been firmly replicated. It is unclear whether and how the currently known genetic variants can be used in practice, because the combined effect of these variants has not been investigated in a population-based study. Particularly, because most GWA studies were enriched for patients with a positive family history and early onset of the disease, association of these variants to type 2 diabetes risk in the general population, including elderly individuals, remains to be determined.<SEP>Because complex diseases are caused by multiple genetic variants, predictive testing based on a single genetic marker will be of limited value. Simulation studies suggest that the predictive value could be improved by combining multiple common low-risk variants. Several empirical studies on the predictive value of genetic polymorphisms have been conducted before the recent GWA data were available. In a case-control study, Weedon et al. showed that combining the information of three polymorphisms improved disease prediction, albeit to a limited extent. Vaxillaire et al. investigated 19 polymorphisms and found that the predictive value was low compared with clinical characteristics.<SEP>Genetic variants associated with risk of type 2 diabetes could potentially be useful for the prediction, prevention, and early treatment of the disease. We investigated whether combining the currently known and well-replicated genetic variants predicts type 2 diabetes in the Rotterdam Study, a prospective population-based follow-up study. We investigated whether these genetic variants improve prediction beyond clinical characteristics.",
                  "RESEARCH DESIGN AND METHODS<SEP>The design and data collection of the Rotterdam Study have been described previously. In short, the Rotterdam Study is a prospective, population-based, cohort study among 7,983 inhabitants of a Rotterdam suburb, designed to investigate determinants of chronic diseases. Participants were aged 55 years and older. Baseline examinations took place from 1990 until 1993. Follow-up examinations were performed in 1993–1994, 1997–1999, and 2002–2004. Between these exams, continuous surveillance on major disease outcomes was conducted. Information on vital status was obtained from municipal health authorities. The medical ethics committee of the Erasmus Medical Center approved the study protocol, and all participants gave their written informed consent.<SEP>Data collection.<SEP>At baseline, prevalent cases of diabetes were diagnosed by a nonfasting or postload glucose level (after oral glucose tolerance testing) ≥11.1 mmol/l and/or treatment with antidiabetic medication (oral medication or insulin) and the diagnosis of diabetes as registered by a general practitioner. During follow-up, diabetes was diagnosed at fasting plasma glucose levels ≥7.0 mmol/l and/or a nonfasting plasma glucose levels ≥11.0 mmol/l and/or treatment with antidiabetic medication (oral medication or insulin) and the diagnosis of diabetes as registered by a general practitioner. Patients registered in general practitioners’ records as type 1 diabetic were excluded from the present analyses (n = 15).<SEP>The CDKAL1 rs7754840, CDKN2AB rs10811661, FTO rs8050136, HHEX rs1111875, IGF2BP2 rs4402960, KCNJ11 rs5219, PPARG rs1801282, SLC30A8 rs13266634, and TCF7L2 rs7903146 polymorphisms were genotyped by means of TaqMan allelic discrimination assays. DNA material was available for 6,544 of the 7,983 participants for the TaqMan analyses. The assays were designed and optimalized by Applied Biosystems (Foster City, CA; http://store.appliedbiosystems.com). Genotypes were determined in 2-ng genomic DNA. Reactions were performed on the TaqMan Prism 7900HT platform. The analyses were performed as described previously. Assays were run on 90 blood bank samples to test for adequate cluster separation. A total of 325 samples were genotyped in duplo. Success rates for TaqMan genotyping ranged from 93.2 to 96.7%, with the exception of 86.1% for IGF2BP2 and 87.4% for HHEX. TaqMan duplicate error rates for the HHEX and IGF2BP2 polymorphisms were 1.2 and 0.6%<SEP>The ADAMTS9 rs4411878 (proxy for rs4607103, r2 = 0.95), CDC123-CAMK1D rs11257622 (proxy for rs12779790, r2 = 0.83), CDKN2A/B rs1412829 (proxy for rs564398, r2 = 0.97), JAZF1 rs1635852 (proxy for rs864745, r2 = 0.97), NOTCH2 rs1493694 (proxy for rs10923931, r2 = 1.0), TCF2 rs4430796, THADA rs7578597, TSPAN8-LGR5 rs1353362 (proxy for rs7961581, r2 = 0.96), and WFS1 rs10012946 (proxy for rs10010131, r2 = 1.0) genotypes were derived from the genotype data of the version 3 Illumina Infinium II HumanHap550 SNP chip array. From a total of 6,449 subjects, there was sufficient DNA material for the array. Samples with a call rate <97.5% (n = 209), excess autosomal heterozygosity >0.336 (approximate false discovery rate [FDR] <0.1% [n = 21]), or mismatch between called and phenotypic sex (n = 36) or with outliers identified by the identity-by-state (IBS) clustering analysis with >3 SDs from population mean (n = 102) or IBS probabilities >97% (n = 129) were excluded from the analysis; in total, 5,974 samples remained for analyses.<SEP>The availability of Illumina 550K array data enabled us to compare genotype calls between TaqMan and Illumina data for the FTO, HHEX, IGF2BP2, SLC30A8, TCF7L2, and CDKAL1 polymorphisms as well. Concordance rates ranged between 98.6 and 99.7%. To increase success rates, we merged the data and deleted pairs that were not concordant. The success rates for the polymorphisms increased to 98.4–99.4%.<SEP>Statistical analyses.<SEP>Associations of individual polymorphisms were investigated using Cox proportional hazards models for the prediction of incident type 2 diabetes and logistic regression analyses for the prediction of prevalent and incident type 2 diabetes together. Analyses were performed crude and adjusted for age, sex, and BMI. We also applied Cox proportional hazards models and logistic regression analyses to investigate the combined predictive value of 1) the 18 polymorphisms (all polymorphisms included as separate independent categorical variables); 2) the risk allele score based on the 18 polymorphisms (assuming all effect sizes of equal weight); 3) age, sex, and BMI; and 4) age, sex, and BMI and all polymorphisms on type 2 diabetes risk. The risk allele score was calculated by summing up the number of risk alleles for each participant with complete genotype information, with risk alleles being the alleles associated with increased risk of type 2 diabetes. The risk allele score assumes that all genetic variants have the same effect, i.e., minor differences in effects size are ignored. The association between the risk allele score and the predicted probabilities was quantified by the Spearman correlation coefficient.<SEP>The discriminative accuracy was evaluated by the area under the receiver operating characteristic (ROC) curves (AUCs). The AUC can range from 0.5 (total lack of discrimination) to 1.0 (perfect discrimination). AUCs were calculated for the predicted risks of the logistic regression model, the risk allele score, and the linear predictor values of the Cox proportional hazards models. AUCs were compared with Analyze-it version 2.11 (www.analyze-it.com), which uses the method of Hanley and McNeil for ROC curve analyses.<SEP>The analyses were repeated for subgroups for age (cutoff 70 years of age) and BMI (cutoff 26 kg/m2). All analyses were performed with SPSS software version 12.0.1.<SEP>Simulation analyses.<SEP>A simulation analysis was performed to quantify the expected AUC for prediction of incident type 2 diabetes based on the odds ratios (ORs) of the investigated polymorphisms in literature (OR 1.09 for ADAMTS9, 1.11 for CDC123-CAMK1D, 1.12 for CDKAL1, 1.12 for CDKN2A/B rs1412829, 1.20 for CDKN2A/B rs10811661, 1.17 for FTO, 1.13 for HHEX, 1.14 for IGF2BP2, 1.10 for JAZF1, 1.14 for KCNJ11, 1.13 for NOTCH2, 1.14 for PPARG, 1.12 for SLC30A8, 1.10 for TCF2, 1.37 for TCF7L2, 1.15 for THADA, 1.09 for TSPAN8, and 1.11 for WFS1). The method of simulation has been described in detail previously. In brief, we simulated genetic profiles and type 2 diabetes status for 100,000 individuals, of whom 10.3% were supposed to have incident type 2 diabetes, as observed in our population. Genetic profiles were constructed from the polymorphisms based on observed allele frequencies. Under the assumption that each polymorphism has two alleles and that allele proportions were in Hardy-Weinberg equilibrium, genotype frequencies for the single polymorphisms were calculated. Assuming that the polymorphisms segregate independently, for each individual, a genotype was randomly assigned. Disease risks associated with the genetic profiles were modeled using Bayes’ theorem. The likelihood ratio of the genetic profile was calculated by multiplying the likelihood ratios of the single genotypes. The OR of the heterozygous genotypes compared with the homozygous nonrisk genotypes were derived from the three large GWA studies. Finally, disease status was modeled by a procedure that compares disease risk of each subject to a randomly drawn value between 0 and 1 from a uniform distribution. This procedure ensures that for each genomic profile, the percentage of people who will develop the disease equals the disease risk associated with that profile, when the subgroup of individuals with that profile is sufficiently large. The simulation was repeated 10 times to obtain a robust estimate of the AUC. The AUC was obtained as the c-statistic by the function somers2, which is available in the Hmisc library of R software (version 2.5.1; www.R-project.org, accessed December 2007).",
                  "RESULTS<SEP>Baseline characteristics.<SEP>A total of 6,544 participants were successfully genotyped for at least one polymorphism. Complete genotype information on all polymorphisms was present in 5,297 subjects (of whom 490 were incident cases and 545 were prevalent cases). Age (P = 0.11), sex (P = 0.22), BMI (P = 0.30), and presence of type 2 diabetes (P = 0.20) were not significantly different between successfully genotyped individuals or individuals with one or more missing genotypes. General characteristics of the population are shown in Table 1. Individuals with type 2 diabetes had higher BMI, higher waist circumference, more often hypertension, and lower HDL cholesterol compared with individuals without type 2 diabetes. All polymorphisms were in Hardy Weinberg equilibrium in the total population and in individuals without type 2 diabetes (highest χ2 3.58, 2 d.f., P = 0.06 for PPARG rs1801282).<SEP>Individual effects of clinical characteristics and polymorphisms on type 2 diabetes risk.<SEP>Table 2 shows the effect of each polymorphism on type 2 diabetes risk (prevalent and incident type 2 diabetes). The minor alleles of the CDKAL1, FTO, IGF2BP2, and TCF7L2 variants were associated with increased risk of type 2 diabetes. The ADAMTS9, CDKN2A/B rs1412829, JAZF, SLC30A8, and WFS1 minor alleles decreased type 2 diabetes risk. Cox regression analyses restricted to incident type 2 diabetes results gave similar results. Adjustment for age, sex, and BMI did not materially change the results, except for the FTO polymorphism, for which the effect on type 2 diabetes risk disappeared after adjustment for BMI.<SEP>In a Cox regression analysis, age (hazard ratio [HR] 1.02 [95% CI 1.01–1.03]), sex (0.67 [0.57–0.79]), and BMI (1.14 [1.12–1.16]) affected prospective type 2 diabetes risk.<SEP>Risk allele score and risk of type 2 diabetes.<SEP>Figure 1 shows the ORs associated with increasing risk allele scores compared with the reference group (0–12 risk alleles) in a logistic regression model. Individuals carrying 21 risk alleles or more (14.4% of the population) had significantly higher type 2 diabetes risk (7.2% of the population carried 21 alleles, OR 1.90 [1.07–3.40]; 4.4% had 22 alleles, 2.11 [1.15–3.86]; 2.0% had 23 alleles, 2.11 [1.07–4.18]; and 1.8% had 24–32 alleles, 2.10 [1.04–4.22]) compared with the reference group of 0–12 alleles (n = 109; 2.0% of the population). In a Cox regression analysis on incident cases of diabetes, this figure was similar (data not shown). The per-allele HR was 1.04 (95% CI 1.02–1.07) (P = 0.001).<SEP>Risk allele score.<SEP>Figure 2 shows the predicted type 2 diabetes risks from the logistic regression model that included all 18 genetic polymorphisms in relation to the risk allele score. The Spearman correlation coefficient was 0.60, indicating a wide range of predicted risks for each value of the risk allele scores. When analyzing only incident type 2 diabetes, this figure was similar (Spearmen rho 0.59; figure not shown).<SEP>Analyses of discriminative accuracy.<SEP>Figure 3 shows the ROC curves for the prediction of incident type 2 diabetes based on the genetic polymorphisms, clinical characteristics, and both. The AUC was 0.60 (95% CI 0.57–0.63) for prediction based on the genetic polymorphisms; 0.66 (0.63–0.68) for age, sex, and BMI; and 0.68 (0.66–0.71) for the genetic polymorphisms and clinical characteristics combined. The difference between the AUCs for clinical characteristics with and without the genetic polymorphisms was significant (P < 0.0001). The AUC of the risk allele score was 0.56 (0.53–0.59). When including only the significantly associated genetic variants of the current study (ADAMTS9, CDKAL1, CDKN1412829, FTO, IGFBP2, JAZF1, TCF7L2, SLC30A8, and WFS1), the AUC was 0.58 (0.56–0.61). Combining the KCNJ11, PPARG, and TCF7L2 variants resulted in an AUC of 0.53 (0.50–0.55). Based on the simulation study, the expected AUC for all genetic polymorphisms using the effect sizes described in literature was 0.57. When combining incident and prevalent type 2 diabetes cases, the AUC of all polymorphisms was 0.60 (0.58–0.62).<SEP>In subgroup analyses, the AUC of the all polymorphisms was 0.62 (95% CI 0.58–0.67) in the low BMI group and 0.59 (0.56–0.62) in the high BMI subgroup. The AUC was 0.61 (0.59–0.65) in the low age-group and 0.63 (0.58–0.67) in the high age-groups.",
                  "DISCUSSION<SEP>We investigated the predictive value of 18 type 2 diabetes risk polymorphisms from the recent GWA studies for the prediction of type 2 diabetes in a large, prospective, population-based study of elderly individuals. Our study shows that combining information of these 18 well-replicated variants has relatively low discriminative accuracy for the prediction of type 2 diabetes in a general population (AUC 0.60). The 18 genetic variants identified to date did not substantially improve the discriminative accuracy of disease prediction based on clinical characteristics.<SEP>In line with the results of the GWA studies, ADAMTS9, CDKAL1, CDKN2A/B, FTO, IGFBP2, JAZF1, SLC30A8, TCF7L2, and WFS1 were associated type 2 diabetes risk in our population. Some of these effects were slightly stronger than the effects described previously. In contrast to most previous studies, the KCNJ11 polymorphism was not associated. The ORs of the other polymorphisms were similar to previously published results but not statistically significant, which may be explained by the smaller number of type 2 diabetes cases and therefore smaller power to reach significance in our prospective study.<SEP>A risk allele score, obtained by counting the number of risk alleles, can be used as a simple proxy of the combined effect of multiple polymorphisms. Risk allele scores ignore the effect sizes of the individual genetic variants, but a previous simulation study has shown that this has limited impact on the discriminative accuracy. In contrast, we found a wide range of predicted risks for each value of the risk allele score, suggesting that differences in the variants carried result in substantial differences in actual disease risks. The risk allele score associated with modest increases in disease risk and the AUC for prediction was 0.56. When taking effect size differences between polymorphisms into account, the AUC was 0.60, showing that the risk allele score predicted less accurately. Other empirical studies have not investigated the differences in diagnostic accuracy between simple risk allele scores and predicted risks obtained from regression models.<SEP>The discriminative accuracy of predictive genetic testing in complex diseases depends on the number of genes involved, the risk allele frequencies, and the size of the associated risks. The maximum discriminative accuracy is determined by the heritability of the disease. Based on previously published effect sizes for the 18 polymorphisms, we predicted that the AUC would be 0.57; and based on our empirical data, we found that it was 0.60. This difference is explained by the fact that some polymorphisms had a slightly larger effect in our population than described in the literature. Nonetheless, the discriminative accuracy of all known replicated type 2 diabetes susceptibility variants to date was rather low. However, our analysis was based on 18 common variants with relatively small effects. The heritability of type 2 diabetes is estimated to range from 30 to 70% depending on the population investigated, and many common variants with small effects or fewer rare variants with stronger effects are still to be discovered. These may further improve the discriminative accuracy of predictive genetic testing for type 2 diabetes.<SEP>Several previous studies have investigated the predictive value of multiple genetic variants in type 2 diabetes, either alone or in addition to clinical characteristics. An overview of the studies performed so far in Caucasian populations and the number of genes investigated is provided in Table 3. Weedon et al. investigated three variants that were also in our study and reported an AUC of 0.58, whereas we found an AUC of 0.53 for the same polymorphisms. The population of Weedon et al. consisted in large part of patients who had early onset of type 2 diabetes or a positive family history of type 2 diabetes, whereas our population included elderly subjects from the general population. The percentage of variance of the disease explained by genetic factors is expected to be higher in populations with a positive family history for the disease, and this may lead to a higher diagnostic accuracy for genetic variants. Recently, Cauchi et al. investigated 15 genetic variants that were associated in GWA analyses in a French case-control study. The AUC of the genetic variants together with age, sex, and BMI was 0.86. Unfortunately, the AUC was not calculated for genes and clinical characteristics separately to assess the additive value of genetic information. Some of the included variants were specifically identified in this case-control study and had relatively large effects on type 2 diabetes risk compared with effects found in meta-analyzed GWA studies. It is therefore difficult to generalize these findings to other populations, and we expect that our population-based prospective study yielded more realistic estimates. Two other studies investigated the improvement of the discriminative accuracy by adding genetic test results to clinical characteristics. Even though we included the 18 firmly replicated polymorphisms to date and they predominantly tested other genetic variants, our findings were similar, demonstrating no substantial added value of genetic information beyond clinical characteristics.<SEP>We can only speculate on the reasons why these genetic variants have little added value beyond clinical characteristics. First, the effects of the genetic variants on type 2 diabetes risk could be exerted through clinical characteristics such as BMI, which implies that including both genes and intermediate factors in the regression model will reduce the effect of the gene. However, adjustment for clinical characteristics did not substantially change the effect of the genetic variants on type 2 diabetes risk (Table 2). Second, the effects of age, sex, and BMI on type 2 diabetes risk in our population may outweigh the contribution of the genetic variants. Such an effect was illustrated in an earlier study, which showed that a genetic predisposition became apparent in subjects with less other risk factors. In our elderly population, one may expect that nongenetic risk factors are more prevalent compared with younger populations. However, the AUC for the genetic variants was higher than expected from the simulation analyses. This makes an underestimation of the contribution of the genetic variants in our population unlikely.<SEP>Obvious strengths of our study are the large size of the study population, the population-based design, and the long period of follow-up. Despite these advantages, the number of incident type 2 diabetes cases was still relatively low to demonstrate statistically significant effects of low-risk susceptibility genes. Furthermore, we had insufficient statistical power to formally investigate gene-gene and gene-environment interactions. In age and BMI subgroup analyses, the estimates for prediction based on the genetic variants were similar and showed overlapping CIs. However, because of smaller numbers of cases in the subgroups, these results should be interpreted with caution. Cauchi et al. reported gene-gene interactions of recently discovered loci but did not report on the effects of these interactions on the AUC. Earlier studies found no evidence for strong gene-gene interactions. Taking into account these interactions may further improve the discriminative accuracy.<SEP>In conclusion, we showed that 9 of 18 currently well-established genetic risk variants were associated with type 2 diabetes in a population-based study. The currently known and replicated genetic variants found in GWA studies contributed modestly to the prediction of type 2 diabetes in a population-based setting and marginally improved the risk prediction when added to clinical characteristics. Future research should aim at identifying and replicating new genetic susceptibility variants and gene-gene and gene-environment interactions to approach levels of discriminative accuracy that enable the identification of at-risk groups.",
                  "22506",
                  "['OBWCCE']"
                ],
                [
                  "37",
                  "32066663",
                  "Introduction<SEP>Schizophrenia is a chronic mental disorder with typical onset in early adulthood and a lifetime risk of approximately 0.7–0.9%. Affected individuals have a life expectancy that is reduced by an average of 14.5 years relative to the general population. The primary factor contributing to increased mortality is cardiovascular disease (CVD). Weight gain and obesity, which are common in schizophrenia, are important risk factors for CVD. Notably, evidence of shared genetic factors between schizophrenia and obesity has been reported, but not in the direction expected from epidemiological data. Genetic correlations estimated from genome-wide association study (GWAS) results from independently collected schizophrenia case-control samples and other traits show a significant negative genetic correlation (rg) of schizophrenia risk with body mass index (BMI) (rg = −0.10, s.e. = 0.03, p = 0.0002). There is no evidence for a genetic relationship between schizophrenia and Type 2 diabetes (rg = −0.028, s.e. = 0.06, p = 0.62) or coronary artery disease (rg = −0.0, s.e. = 0.05, p = 1.0). These results imply that if genetic factors also contribute to the associations between metabolic syndrome and schizophrenia, this is a likely complex relationship.<SEP>Dietary intake has a causal association with obesity and people with schizophrenia tend to have an unhealthy diet, higher in fat and refined sugar and low in fruit and vegetables. We hypothesised that there might be an underlying genetic susceptibility to the self-selected dietary composition in individuals with schizophrenia and that this would be manifest as a significant genetic correlation between schizophrenia and self-selected diet measured in a community sample. A twin study of 18–19-year-olds (N = 2865) reported heritability (h2) estimates for vegetable eating of 54% (95% CI: 47–59%) and for meat or fish-eating of 44% (95% CI: 38–51%). These estimates may be inflated by shared family environment. Meta-analyses of GWASs for macronutrient intake (i.e., protein, carbohydrate and fat intake) have confirmed associations between consumption of carbohydrates, fat and protein with the fibroblast growth factor 21 (FGF21) gene and associations of consumption of protein intake with the fat mass and an obesity-associated locus (FTO). Significant genetic correlations between protein intake and BMI (rg = 0.23) have been reported, but no significant evidence for genetic correlations between any macronutrient types and schizophrenia (rg = < 0.07). Larger samples are needed to replicate these findings and to elucidate further how diet correlates with other traits at the genetic level.<SEP>Our study aimed to investigate (1) genetic influences on dietary intake using GWAS data from the UK Biobank; (2) whether there is shared genetic susceptibility between dietary intake and schizophrenia and (3) if so, whether there was any statistical evidence consistent with a causal relationship between SNPs are bi-directional using Mendelian Randomisation. We also explored genetic correlations of dietary intake with a number of other traits with available GWAS summary statistics.",
                  "Materials/subjects and methods<SEP>Study sample<SEP>The United Kingdom Biobank (UKB) is a major community-based longitudinal study with extensive genetic and phenotypic information of over 500,000 participants aged 40–69 years from across the UK during 2006–2010. The study design and sample characteristics have been extensively described elsewhere.<SEP>Ethics statement<SEP>This research has been conducted using the UK Biobank resource under application number 12505 and follows UK Biobank’s Ethics and Governance Framework.<SEP>Generic diet questionnaire<SEP>All participants completed a generic diet questionnaire (UKB, category:100052) that was used to estimate the average consumption of fruit, vegetables (raw and cooked), fish (oily and non-oily), meat (processed, beef, lamb, pork), bread, cheese, cereal, tea, coffee and drinking water.<SEP>We only included responses from individuals at the questionnaire at the first time-point as only a small proportion had completed the questionnaire twice. We standardised the diet questionnaire responses for each item, and we set values that were >3.5 standard deviations from the mean to 3.5 standard deviations. Given the high correlation between question responses, we summarised the questionnaire information by conducting a principal component analysis (PCA). Since questions about bread, cheese, cereal, tea, coffee and drinking water had low loadings on the components (<0.08), we excluded these questions from the PCA and repeated the PCA using only the questions about fruit, vegetables, fish, and meat consumption. Three eigenvalues were greater than 1. We selected the first two factors, factor 1 (Diet Component 1, DC1), explaining 23% of the variance of the included questionnaire items and representing a meat-related diet (high intake of processed meat, poultry, beef, lamb and pork), and factor 2 (Diet Component 2, DC2), explaining 18% of the variance and representing a fish and plant-related diet (high intake of raw and cooked vegetables, fruit, oily and non-oily fish) (Fig. 1). The third factor, accounted for only 12% of the variance and was not as interpretable as DC1 and DC2 and, therefore, we did not include it in our analysis (Supplementary Fig. 1). A schematic diagram with the number of individuals excluded at each stage is provided in Supplementary Fig. 2 and the distributions of the anthropometric traits of the final sample are provided in Supplementary Fig. 3. By design the phenotypic correlation between DC1 and DC2 is zero; the phenotypic correlations between diet components are in Supplementary Table 1. We excluded from the sample individuals with a BMI that was more or <3 standard deviations from the mean based on their sex and individuals with a diagnosis of anorexia nervosa (ICD-10 code: F50 and ICD-9 code: 307.1) and/or schizophrenia, schizotypal and delusional disorders (ICD-10 codes: F20-F29 and ICD-9 codes: 290–299). Taking into account that data on individuals who follow special diets was only available for 58,985 participants, we did not include this information in our analyses.<SEP>Genotypes<SEP>The genotype measures and quality control (QC) of the UKB data have been described extensively by the UKB group (also see Supplementary Note for more information). We utilised the latest July 2018 genotype release of imputed data from UKB. We only included individuals of White European descent with genetic data. Ancestry was defined using a combination of self-report information on ethnic background and genetic information as described. The total number of markers included was 25,921,788. Principal components were calculated with genotyped variants used by the ukb (identified from the ukb_snp_qc.txt file) and passing additional QC filters (as applied in to unrelated white European set; geno 0.05, pHWE 10-6, MAF > 0.01). Genotyped SNP used by the UKB had already been LD pruned (r2 < 0.1) and had long-range LD regions removed (Table S12 UKB QC documentation). There were 137,102 SNP included in the analysis. Genetic principal components were calculated for the unrelated white European set using flashPCA then projected onto the related individuals.<SEP>The number of individuals with complete phenotypic and genotypic data was 335,576.<SEP>Statistical analyses<SEP>The UK Biobank provided KING kinship coefficients and the fraction of markers, which share no alleles identical-by-descent. We identified likely 100 twin, 12,957 full-sibling, 3354 parent–offspring, 6092 second- and 37,947 third-degree relationship pairs in our European individuals following the procedure outlined in Bycroft et al.. We then estimated the phenotypic correlation (rp) for each group or relatives and approximated the heritability of DC1 and DC2 (residuals after regression on covariates) within each class as rp/aR; where aR is the average coefficients of relationship (i.e., monozygotic twins aR = 1, full siblings aR = 0.5, parent–offspring aR = 0.5, second-degree aR = 0.25 and third-degree aR = 0.125) relatives). This approximation assumes that the rp reflects only shared additive genetic contributions. Since close family member share non-additive genetic and family/social networks, contributions from such factors would generate higher rp/aR for close relatives compared to more distant relatives. We also estimated rp/aR for BMI, as a benchmark.<SEP>For our GWA analyses, we used the DC1 and DC2 residuals after regressing on covariates of (1) year of birth, (2) sex, (3) month of assessment, (4) assessment centre, (5) batch and (6) 100 genetic PCs. To obtain a better understanding of DC1 and DC2, we also compared their geographical distribution using the whole sample vs. the sample on unrelated Europeans after regressing out the covariates (Supplementary Figs. 4 and 5). We decided to adjust for 100 genetic PCs given that diet is a trait that is likely to vary within subpopulations.<SEP>We used the BOLT-LMM software package to model the associations between SNPs and the residuals of the two phenotypes, DC1 and DC2. BOLT-LMM uses a mixed model that uses genetic data to account for population structure and relatedness between individuals. The threshold for significance of associations was a p-value < 5 × 10−8.<SEP>We used the FUMA web application to identify independent significant SNPs with a genome-wide significant p-value (<5 × 10−08) that are in approximate linkage disequilibrium with each other at r2 < 0.1 and to generate Manhattan and Quintile–Quintile plots and achieve SNP functional annotations.<SEP>We performed gene analysis and gene-set analysis with MAGMA v1.6 using FUMA using the association analysis summary statistics. Taking into account that the UK Biobank imputation used both 1000 genomes and Haplotype Reference Consortium (HRC) reference panels while FUMA only uses 10,000 genomes as a reference panel, it is likely that our gene-set analyses may be based on an incomplete set of variants. Gene expression analysis was obtained from GTEx v6 (https://www.gtexportal.org/home/) integrated by FUMA (Supplementary Note).<SEP>We estimated genetic correlations between schizophrenia and other complex traits using linkage disequilibrium (LD) score regression through LD Hub v1.9.0 (http://ldsc.broadinstitute.org/centers/) and GWAS summary statistics. 235 traits were examined using LD hub, and the Bonferroni corrected p-value threshold for significance is 0.05/470 = 1 × 10−4. For traits that had estimated genetic correlations significantly different from 0 with DC1 or DC2, we used the gsmr R-package to implement Generalised Summary-data-based Mendelian Randomisation to test for bi-directional genetic associations. Heterogeneity in dependent instrument (HEIDI) outlier analyses were implemented to exclude SNPs that have significant pleiotropic effects.<SEP>We used the summary-data-based Mendelian randomisation (SMR) software to examine if the association of an SNP with the phenotype is mediated through gene expression, a tool to help prioritise GWAS results for follow-up functional studies. We used the following summary data expressed: (1) brain-expressed: expression quantitative trait loci (eQTL) meta-analysis data where we tested 7324 probes and DNA methylation quantitative trait loci (mQTL) summary data where we tested 92,867 probes from. The Bonferroni adjusted p-value threshold for eQTL analyses was 0.05/7324 = 6.8 × 10−6 and for the mQTL was 0.05/92,867 = 5.4 × 10−7; (2) blood-expressed: eQTL summary data from the CAGE data set, where we tested 8468 probes and mQTL data from the Brisbane Systems Genetics Study and the Lothian Birth Cohorts of 1921 and 1936 from where we tested 92,867 probes. The Bonferroni adjusted p-value threshold for eQTL analyses was 0.05/8468 = 5.9 × 10−6 and for mQTL was 0.05/91,578 = 5.4 × 10−7. We conducted a number of sensitivity analyses (Supplementary Note).",
                  "Results<SEP>Diet components (DCs)<SEP>In all, 335,576 individuals (46% males) had complete genotypic and phenotypic data and were included in a PCA to generate two independent diet components (DCs) (see Methods, Supplementary Table 2). DC1 represented a meat-related diet and DC2 a fish and plant-related diet (Fig. 1 and Supplementary Table 1). DC1 was associated with younger age (b = −0.01, p < 0.001) (i.e., a 1-year increase in year of birth was associated with a decrease in DC1 by 0.01 standard deviations) and females were more likely to have a lower DC1 score (b = −0.38, p < 0.001) (Supplementary Table 3). DC2 was also associated with younger age (b = −0.03, p < 0.001), but in contrast to DC1, females were more likely to have a higher DC2 score (b = 0.28, p < 0.001) (Supplementary Table 3). Month of questionnaire administration was also significantly associated with DC1 (less meat eating in summer) and DC2 (more fish and plant eating in summer). We report analyses of standardised DC1 and DC2 residuals after regression on covariates, including age, sex, month of questionnaire, assessment centre, genotyping batch and 100 genetic principal components (PCs).<SEP>To determine if genetic factors contribute to these DCs, we identified pairs of 1st, 2nd and 3rd-degree relatives and estimated phenotypic correlations (rp). As expected, rp increased with the coefficient of relationship (aR), and the rp/aR estimates were higher for close relatives. For 3rd-degree relatives, coefficients of non-additive genetic relationship and influences of a shared common environment are expected to be small, so rp/aR provides estimates of the trait heritability (h2). We estimate h2 of 0.16 for DC1 and 0.16 for DC2 (Table 1).<SEP>Genome-wide association study<SEP>For DC1, 29 independent SNPs reached genome-wide significance (p < 5 × 10−8; Fig. 2, Supplementary Table 4, Supplementary Data File, Supplementary Fig. 6). The proportion of variance explained by genome-wide common genetic variants () is 0.055 (s.e. = 0.003), i.e., 31% of the h2 estimated from 3rd-degree relatives. Among the top-associated loci was the chromosome 19 apolipoprotein E gene (APOE, rs429358, p = 4.5 × 10−13, C allele b = −0.02), with the APOE protein a significant cholesterol transporter that has been directly related to low-density lipoprotein cholesterol and mostly known for its associations with Alzheimer’s disease. Here, the C allele, associated with higher risk of Alzheimer’s disease was negatively correlated with the meat-related diet. Another strongly associated locus was the chromosome 22, rs429358 SNP that maps to the FGF21 gene, replicating previous GWASs on macronutrient intake. This locus was also associated with the individual questionnaire items (Supplementary Table 5) providing further validity on their associations with food consumption. In gene-based analyses there were 41 genes significantly associated with DC1 (Supplementary Table 6), including the neuronal growth regulator 1 (NEGR1) gene (p = 5.7 × 10−15) a BMI-related gene. Seven gene-sets reached statistical significance (Supplementary Table 7). The top three gene-sets were related to synaptic plasticity, a process related to memory and learning that is also found disrupted in people with schizophrenia.<SEP>For DC2  was 0.078 (0.004), or 44% of the estimated h2 estimated from 3rd-degree relatives. There were 63 independent genome-wide significant SNPs (p < 5.0 × 10−8; Fig. 2, Supplementary Table 8, Supplementary Data File, Supplementary Fig. 6), and 260 genes (Supplementary Table 6) that achieved statistical significance in their respective association analysis tests.<SEP>Fat mass and the obesity-associated protein (FTO) gene (p = 4.4 × 10−17), one of the most extensively studied genes in the field of food consumption and obesity, was associated with DC2 at the gene analysis further supporting previous candidate gene studies.<SEP>Two gene-sets achieved statistical significance (Supplementary Table 7). The top-associated gene-set was the Nikolsky breast cancer 20q11 amplicon gene-set (i.e., genes within amplicon 20q11 that were identified in a copy number alterations study of 191 breast tumour samples) was associated with DC2 (p = 1.4 × 10−08).<SEP>In silico functional analyses<SEP>Integration of association results with GTEx gene expression across 30 tissue types showed that SNP associations were significantly enriched in the brain tissue gene-set for both DC1 and DC2 while DC1 was also significantly enriched in the pituitary gland tissue gene-set (Supplementary Figs. 7 and 8). To investigate in silico putative functional relevance of loci, we integrated eQTL, i.e., SNP gene expression associations) and mQTL summary statistics expressed in the brain and blood tissues using the SMR method. The analyses highlight >200 genes for which association with self-selected diet have the strongest evidence-base (Supplementary Table 9). Results include associations putatively mediated through gene expression for both DC1 and DC2 with neuronal growth regulator 1 (NEGR1) and the ribosomal protein L31 pseudogene 12 (RPL31P12), both of which have been consistently associated with BMI, educational attainment, intelligence and major depressive disorder. In addition SMR association analyses link DC1 and Histone Cluster 1 H2B Family Member F (HIST1H2BF) that has been associated with hip circumference and body height, and DC2 with IKAROS family zinc finger 3 (IKZF3) gene that has been associated with inflammatory bowel disease and family with sequence similarity 167 member A (FAM167A) gene that has been associated with systemic lupus erythematosus.<SEP>Cross-trait analyses<SEP>Genetic correlations between DC1 or DC2 with other traits estimated from publicly available GWAS summary statistics (Figs. 3 and 4, and Supplementary Table 10, Supplementary Table 11) show significant estimates with schizophrenia and a number of other traits. In particular, there was a negative genetic correlation between schizophrenia and DC1 (rg = −0.13, s.e. = 0.03, p = 2.0 × 10−6) and positive genetic correlation between schizophrenia and DC2 (rg = 0.16, s.e. = 0.03, p = 9.4 × 10−8). Since DC1 and DC2 are orthogonal (i.e., phenotypically uncorrelated), they were also genetically uncorrelated (rg = −0.03, s.e. = 0.03, p = 0.30).<SEP>DC1 showed a significant negative genetic correlation with anorexia nervosa (rg = −0.18, s.e. = 0.03, p = 5.02 × 10−08) and positive genetic correlations with waist circumference (rg = 0.14, s.e. = 0.03, p = 3.8 × 10−06) and waist-to-hip-ratio (rg = 0.14, s.e. = 0.03, p = 5.13 × 10−06) (Fig. 3).<SEP>DC2 was positively correlated to bipolar disorder (rg = 0.23, s.e. = 0.04, p = 6.9 × 10−08) , chronotype (rg = 0.13, s.e. = 0.03, p = 4.2 × 10−05), BMI (rg  =  0.26, s.e. = 0.03, p  =  5.16  × 10−19) and other BMI-related traits while negative genetic correlations were found with age at menarche (rg = −0.16, s.e. = 0.03, p = 4.9 × 10−09) and intelligence (rg = −0.13, s.e. = 0.03, p = 4.2 × 10−05). We also conducted genetic correlations analyses for the DC1 per sex separately but the results were similar for both sexes (Supplementary Tables 16 and 17).<SEP>Mendelian randomisation (MR) analyses<SEP>Given the significant genetic correlations between the DCs and other traits we used MR analyses (via the GSMR method) to investigate if there is statistical evidence consistent with uni- or bi-directional relationships between the correlated traits.<SEP>MR analyses between DC1 and schizophrenia were weakly significant when DC1 was an outcome, where for 1 standard deviation liability to schizophrenia there was a reduction of 0.03 standard deviations of DC1 (bDC1|SCZ = −0.03, s.e. = 0.003, p = 2 × 10−02) but were not significant when DC1 was the exposure (Table 2).<SEP>MR analyses between DC2 and schizophrenia were bi-directionally significant (Table 2). Particularly, when DC2 was an outcome, for 1 standard deviation liability to schizophrenia there was an increase of 0.05 standard deviations of DC2 (bDC2|SCZ = 0.05; s.e. = 0.00, p = 3.2 × 10−06, number of independent SNP instruments (NSNP) = 118). When DC2 was an exposure, for 1 standard deviation change in DC2 the odds of liability to schizophrenia increased 1.43 fold (bSCZ|DC2 = 1.43, s.e. = 0.10, p = 1 × 10−03, NSNP = 46).<SEP>MR analyses between DC1 and waist hip ratio and DC1 and anorexia nervosa were not significant.<SEP>Interestingly, BMI was associated with DC2 when DC2 was an outcome (bDC2|BMI = 0.15, s.e. = 0.01, p = 4.9 × 10−39, NSNP = 82), but not when DC2 was an exposure (bBMI|DC2 = 0.07, s.e. = 0.04, p = 0.06, NSNP = 44) (Supplementary Fig. 9). Thus, there is no evidence for a high consumption of fruit, vegetables and fish being causally related to an increase in BMI, consistent with observational studies and dietary population guidelines rather the observed correlations are a result of tendency to have increased consumption of fruit, vegetables and fish as a (direct or indirect) consequence of high BMI, which may reflect that, in general, individuals with high BMI consume larger quantities of all types of food.",
                  "Discussion<SEP>Dietary intake<SEP>Our study is the largest to investigate the genome-wide associations of dietary intake. We applied PCA to diet questionnaire item responses and identified two independent diet components, with high DC1 representing high-meat consumption and high DC2 reflecting high consumption of fish and plant-related products. This analysis enabled us to undertake GWAS analyses of quantitative measures of self-reported dietary intake.<SEP>Variation between people in dietary intake as represented by DC1 and DC2 is predominantly driven by non-genetic factors, consistent with it being primarily influenced by a variety of socioeconomic and psychological factors, including lifestyle, culture and health beliefs. Nonetheless, both DC1 and DC2 were moderately heritable (both 16% Table 1) in line with reports from twin studies and consistent with animal studies that imply biological driving forces underpinning self-selection of diet. The proportion of variance explained by genome-wide SNPs was 6% and 8% for DC1 and DC2. Hence this common variation explains 31% and 44% of the estimated heritability, respectively. Twenty-nine independent loci passed the GWA threshold for DC1 and sixty three for DC2. The FGF21 gene reached GWA significance in the gene-wide analyses for both DC1 and DC2 (which were phenotypically and genetically uncorrelated), replicating previous GWA studies on macronutrient intake. Moreover, 4 out of 7 brain mQTL SNPs and 6 out of 12 blood-expressed mQTL SNPs were associated with DC1. Five mQTL SNPs and one eQTL SNP expressed in the brain and eight mQTL SNPs expressed in the blood were associated with DC2 were at the 19q13.3 locus, including the izumo sperm-egg fusion 1 (IZUMO1) gene, the MEF2 activating motif, and SAP domain containing transcriptional regulator (MAMSTR) gene and the RAS-interacting protein 1 (RASIP1) gene supporting a role of the 19q13.3 locus in diet intake.<SEP>Dietary intake and schizophrenia<SEP>Our primary hypothesis that there would be a genetic correlation between dietary intake and schizophrenia was supported. It is important to note that we chose to examine food type consumption rather than macronutrient intake, a decision informed by a published GWA meta-analysis (n = 91,114) that did not find evidence of a phenotypic relationship of macronutrient intake and schizophrenia. Based on the diet questionnaire data available, our DCs reflect self-selected diet composition and quantity.<SEP>We found that genetic factors contributing to high DC2 values, indicating higher consumption, mainly, of fish (oily and non-oily) and cooked vegetables were positively and significantly correlated with genetic factors associated with schizophrenia (regardless of whether BMI was included as a covariate in analyses) (Supplementary table 12). On first consideration, taking into account wealth of evidence indicating that higher consumption of fruit and vegetables is related to a variety of positive health and psychological outcomes, including decreasing risk for cancer and heart disease, lower incidence of depression, better emotional health, greater happiness and life-satisfaction, the direction of our association of dietary intake and schizophrenia seems unexpected. However, our results suggest a more complex relationship between self-selected diet and schizophrenia risk, and that the reported negative health consequences of schizophrenia are likely to be consequences of factors associated with the illness (e.g., lack of access to care, economic disadvantage) or drug treatment (where drugs targeting schizophrenia pathways could impact DC2 pathways). Mendelian Randomisation analyses indicated that while results were consistent with schizophrenia being related with a reduction of DC1, that the relationship between schizophrenia and DC2 is more likely to reflect pleiotropy.<SEP>Dietary intake and BMI<SEP>Given the wording of the dietary questions used to generate the DCs, we expect that the DCs reflect both dietary compositions and quantity. Hence, a relationship with BMI and other obesity-related traits is expected. Indeed, we found a strong genetic correlation between DC1 and waist circumference and genetic correlations between DC2 and many BMI-related traits, including obesity and being overweight. We decided not to include these traits as covariates to our analysis to avoid potential biases arising from using residuals and from conditioning on heritable covariates. However, as a sensitivity analysis, we re-ran our analyses adjusting DC1 and DC2 for BMI and then, as predicted observed a significant genetic correlation between DC2 and BMI (Supplementary Tables 13 & 14), and an induced correlation between the DC1 and BMI; these results affirmed our decision not to adjust for BMI at the phenotypic level. We also used Genome-Wide Inferred Study (GWIS), where we conditioned DC1 on waist circumference and DC2 on BMI; this analysis forces the genetic correlation between the conditioned trait (e.g., DC2 conditioned on BMI) and the conditioning trait (e.g., BMI) to be zero. In that case, all the previously significant genetic correlations between DC1, DC2 and the other traits became non-significant (Supplementary Table 15), even though the overlap in liberally associated SNPs was low (Supplementary Fig. 9). Our results illustrate the complex relationship between the DCs and obesity-related traits, with DC1 and DC2 independent dimensions; the former related to waist circumference and waist/hip ratio, and the latter associated with BMI. The relationship between DC2 and BMI is particularly complex. The genetic correlation is in the direction of genetic factors associated with eating more fish, fruit and vegetables as associated with higher BMI.<SEP>Implications<SEP>The low-heritability estimates of DC1 and DC2 attest to dietary intake being a mostly environmental, and therefore potentially modifiable, behavioural trait. Although we observed genetic correlations between genetic liability to dietary choice, schizophrenia, and bipolar disorder, the patterns of results from the MR tests suggested this is likely to be primarily the result of genetic pleiotropy. In other words, some of the alleles that influence dietary choice also influence liability to psychiatric disorder, but those alleles do not do so by their effects on diet itself. Nevertheless, given that diet and nutrition affect biological processes potentially involved in psychiatric (and definitely in some physical disorders), such as inflammation, oxidative processes, and brain plasticity, it is important to further examine potentially shared pathways between dietary intake and disease to gain a better understanding of the underlying biology. Although the direction of the association results between the DCs and schizophrenia was unexpected, similar seemingly paradoxical results have been type found for 2 diabetes (T2D). Zhu et al. recently used GSMR Mendelian randomisation to report a negative association between BMI and type 2 diabetes (T2D), despite BMI being a known risk factor (confirmed through randomised control trial). They interpret the relationship between BMI and T2D as a complex mixture of causality, reverse causality and pleiotropy. It seems that this broad interpretation is likely to represent the relationship between dietary intake, BMI and schizophrenia.<SEP>Limitations<SEP>We used a self-report questionnaire to assess dietary intake. Although this is the typical assessment method used in large population-based studies, there is evidence that higher BMI is associated with under-reporting of the quantity of food consumption and that individuals tend to under-report unhealthy foods and over-report consumption of fruit and vegetables. The assessment we used is subjective and reflects a significant limitation of population and community studies of nutritional science in general. Another limitation is that diet changes over time. The UK biobank participants are not representative of the population in certain sociodemographic characteristics. Finally, ascertainment bias could lead to collider bias (i.e., spurious associations between two unrelated traits, which is driven by each being associated with a trait that influences participation in the study). Therefore, our findings need to be tested in other cohorts.",
                  "29526",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "38",
                  "26789123",
                  "Introduction<SEP>New onset diabetes after transplantation (NODAT), also known as post transplantation diabetes mellitus (PTDM), is a serious complication of solid organ transplantation. It affects 2–50% of organ transplant recipients and is associated with greater healthcare costs and an increased risk of graft failure, cardiovascular complications and death. The wide variation in reported prevalence of NODAT in part reflects the varying clinical definitions of this disorder. In different clinical studies the NODAT phenotype has been defined by various criteria including elevated fasting blood glucose; abnormal oral glucose tolerance tests; elevated glycated haemoglobin (HbA1c) or absolute requirement for hypoglycaemic therapies following solid organ transplantation. A number of modifiable and non-modifiable risk factors have been identified which may predict NODAT. Modifiable risk factors include obesity and choice of anti-rejection immunosuppression medication. Patients receiving tacrolimus-based immunosuppressive regimens are at greater risk of developing NODAT compared to those prescribed ciclosporin-based immunosuppressive treatment. However, choosing an immunosuppressive regimen to specifically avoid NODAT may have a damaging effect on the graft itself. Non-modifiable risk factors include family history of diabetes mellitus, polycystic kidney disease, hepatitis C infection, female gender and older recipient age. There is an established genetic component to NODAT, however the identification of genetic risk factors has proved challenging. It is well documented that ethnicity is an important risk factor; people of African American, Hispanic, or South Asian background are at a significantly increased risk of developing the disease. Low plasma adiponectin concentration, a factor which is under significant genetic control, has also been demonstrated to be predictive for NODAT. Genome-wide association studies (GWAS) are revealing SNPs associated with diabetes, which are replicated across multiple populations, but such robust multi-centre GWAS have not yet been published for NODAT. However, multiple publications have reported genetic associations with NODAT in the literature, often with inconsistent results; this report describes an inclusive review and meta-analysis of existing data.",
                  "Materials and Methods<SEP>Selection Criteria<SEP>Review of the literature was performed to identify all published genetic variants associated with NODAT in a kidney transplant population. Studies carried out in NODAT populations following other forms of organ transplant (such as liver transplant) were not included. PubMed, Web of Science and Google Scholar were searched from their inception until May 2015 with no language restrictions, using the following keywords: ‘New Onset Diabetes’, ‘Diabetes Mellitus’, ‘Gene’, ‘Genetic’, ‘Genotype’, ‘Transplantation’, ‘Transplant’, ‘Polymorphism’, ‘Mutation’, ‘NODAT’ and ‘PTDM’ (Post-Transplantation Diabetes Mellitus). Bibliographies for all identified articles and reviews were examined to identify further publications not found in the original search.<SEP>Inclusion Criteria<SEP>Studies were included when there was a minimum of three studies investigating the association of a specific variant with NODAT. Studies were deemed eligible if they fulfilled the following criteria: (a) published in a peer reviewed journal article or conference abstract using original data; (b) were conducted in a kidney transplant population in a case-control manner for NODAT; (c) included patients diagnosed with NODAT; (d) included controls who had undergone kidney transplantation but did not develop NODAT during follow-up. Authors were contacted if further essential information was required or if there was a query regarding eligibility. If sufficient information could not be obtained, the study was excluded, as were studies that duplicated data.<SEP>Statistical Analysis<SEP>Data was manually extracted from the studies. Information was gathered on study size, numbers of cases versus controls, ethnicity, genotyping methods, recorded odds ratios and p values. If ethnicity was not explicitly stated, this was inferred from the geographical location of the recruitment site and/or contact with authors. Deviation from Hardy-Weinberg Equilibrium was measured using genotype counts with a threshold of p <0.0001. Funnel plots of standard error of the log-odds-ratio against the log-odds-ratio were produced to estimate publication bias. Power calculations were conducted using StatCalc version 6.<SEP>Heterogeneity was calculated using a Cochrane Q test for heterogeneity with the I2 statistic used to describe percentage variation across studies. Meta-analysis was performed using a random effects model for variants replicated in three or more eligible studies, with significance value set at p <0.05.<SEP>All meta-analyses were performed using Review Manager software version 5.3.5 (RevMan 5.3) (http://tech.cochrane.org/revman).<SEP>Funnel plots of standard error of the log-odds-ratio against the log-odds-ratio were produced to estimate publication bias. These were assessed by visual inspection. Funnel plots are capable of detecting publication bias which would be undetected by more formal statistical tests. Statistical tests such as Egger’s test were not conducted in this review due to the small number of studies in the meta-analyses, which were not sufficient to distinguish chance from asymmetry.",
                  "Results<SEP>Included Studies<SEP>The preliminary literature search yielded 4,147 citations, 40 of which were relevant studies investigating NODAT and 36 of which had all the required information to allow the extraction of variant information (Fig 1). Data was extracted from these articles for all investigated SNPs.<SEP>Of the 36 studies deemed eligible for inclusion, 16 studies were carried out in Asian populations, 16 in Caucasian populations and 4 in populations of mixed ethnicity. Table 1 outlines the characteristics of each of the eligible studies.<SEP>The literature review revealed 18 genetic variants considered for association with NODAT that were reported in a minimum of three studies across 12 genes (Table 2 and Fig 2).<SEP>Of these analysed variants, three were significantly associated with NODAT based on meta-analysis; CDKAL1 rs10946398 p = 0.006 OR = 1.43, 95% CI = 1.11–1.85 (n = 696 individuals), KCNQ1 rs2237892 p = 0.007 OR = 1.43, 95% CI = 1.10–1.86 (n = 1,270 individuals), and TCF7L2 rs7903146 p = 0.01 OR = 1.41, 95% CI = 1.07–1.85 (n = 2,967 individuals) (Fig 3).<SEP>Power calculations (S1 Table) suggest this study is adequately powered to identify a risk variant; for example, considering 2360 cases and 607 controls there was >80% power to identify a risk variant with odds ratio 1.5 and minor allele frequency of 5%.",
                  "Discussion<SEP>Main Findings<SEP>Thorough investigation of genetic variants reportedly associated with NODAT in kidney transplant patients has revealed significant associations by combined analysis. Three SNPs were significantly associated with NODAT–TCF7L2 rs7903146, CDKAL1 rs10946398 and KCNQ1 rs2237892 at the significance level p<0.05.<SEP>Many of the studies used in this investigation focused on genes previously associated with type 2 diabetes (T2D). NODAT and T2D have a number of important similarities. Both are characterised by insulin resistance and insulin hypo-secretion and share similar risk factors including increased age, family history of diabetes and non-white ethnicity.<SEP>TCF7L2 (transcription factor 7-like-2) has been previously linked to T2D, and has been cited as one of the most important signals associated with T2D. The T allele was identified as a diabetes risk factor in the pre-GWAS era and was later replicated across a number of groups with different ethnic ancestry. It is not yet completely understood how TCF7L2 influences risk of T2D but a number of theories have been put forward. It may affect blood glucose homeostasis by altering levels of glucagon-like peptide 1 in the gut, or it may decrease insulin secretion via the pancreatic beta, adipose or liver cells. rs7903146 is located in an intron; a non-protein coding region of the gene. There is no obvious mechanism by which a mutation at this locus could affect NODAT or T2D development, however the variant rs7903146 may either be in linkage disequilibrium with a causal allele or may itself influence gene expression through regulatory mechanisms.<SEP>CDKAL1 (cyclin dependent kinase 5 regulatory subunit associated protein 1 like 1) has been associated with impaired insulin secretion and the development of T2D in both European and Han Chinese populations by GWAS and the variant rs10946398 has been found to be significantly associated with T2D by meta-analysis. CDKAL1 encodes a methylthiotransferase which is thought to regulate the CDK5 protein which stimulates production of insulin as well as other processes in the pancreatic beta cells. In this manner, by impairing insulin production via over-expression of CDK5, CDKAL1 may increase risk of T2D and NODAT. The rs10946398 variant is found in exon 5 of the CDKAL1 gene. An alternative splicing product of CDKAL1 (CDKAL1v1) is increased in individuals homozygous for the minor C allele at this locus. It has therefore been suggested that this particular variant influences splicing of the gene.<SEP>KCNQ1 is also an established T2D risk factor and has been associated with gestational diabetes. Variants of KCNQ1 cause a variety of disorders including hereditary long QT syndrome (Romano-Ward syndrome). It is expressed in the pancreatic islet cells as well as the heart and encodes a protein which combines with KCNE proteins to form voltage charged potassium channels found in the membranes. The KCNQ1 proteins form the structure of the channel while the KCNE proteins regulate the activity of the channel. Pancreatic beta cell survival rate is thought to be affected by these potassium channels. It is thought that dysfunction of these potassium channels could alter cell membrane potential and contribute to development of T2D or NODAT. A specific KCNQ1 blocker 293B has been shown to increase insulin production. The variant rs2237892 C risk allele has been shown to be associated with fasting plasma glucose concentration, suggesting that C homozygous individuals have impaired baseline insulin secretion. The gene is also under the control of tissue specific imprinting.<SEP>These genetic variants are all established T2D risk factors and several variants have been implicated in potential mechanisms contributing to diabetes. Therefore, it is not surprising that these variants are linked to NODAT, another form of diabetes, since the mechanisms controlling insulin production and maintenance of stable glucose levels will both be similar in T2D and NODAT. The meta-analyses conducted on the other variants identified in the literature did not reach statistical significance. This may have been for several reasons, including the small number of studies, small numbers of study participants, or varying phenotypic definitions. Of note, our meta-analysis incorporates data from both candidate gene and genome-wide association studies.<SEP>A number of variants which were associated with NODAT in previous studies were not found to be associated with NODAT following meta-analysis. Notable variants which did not reach significance after meta-analysis include KCNJ11 rs5219 and ADIPOQ rs1501299. KCNJ11 rs5219 is an established T2D risk factor in a gene encoding a voltage gated potassium channel. ADIPOQ rs1501299 has been previously associated with NODAT as well as breast cancer, prostate cancer and T2D complications such as heart disease. Adiponectin encoded by ADIPOQ is involved in lipid metabolism and insulin sensitivity and making this an attractive candidate for association with NODAT. Neither of these particular variants reached the p<0.05 significance threshold following meta-analysis which could mean they are not associated with NODAT or that the association is only present in certain populations.<SEP>Limitations of the Review<SEP>This study does have a number of limitations. The definition of NODAT differs from centre to centre as highlighted by the varying prevalence of NODAT reported in Table 1 which ranges from 5–51%. A potential explanation for this large variation in reported prevalence is the differences in how NODAT is diagnosed i.e. heterogeneity of the clinical phenotype. Some authors employed diagnostic criteria for diabetes as defined by the World Health Organisation and American Diabetic Association, although the final interpretation of these standards did vary in published studies of NODAT. Others authors used pragmatic clinical criteria for NODAT diagnosis defining the affected patients as requiring the de novo prolonged use of insulin and/or oral hypoglycaemic medication following transplant. A more rigorously defined NODAT phenotype may facilitate more reproducible results between studies. There were a small number of studies available for many of the variants; larger, carefully phenotyped studies would provide better power to identify alleles robustly associated with NODAT. There was a varying degree of heterogeneity noted between studies, some of which was likely due to different ethnicities considered. In addition, the variations in prescribed immunosuppressive regimens, and their differential effects on NODAT incidence, were not accounted for in many of the studies. It is of note that the Belfast derived data was from the single transplant centre for Northern Ireland. TCF7L2 rs7903146 variant was only nominally associated with NODAT (p = 0.01), but this association was replicated with the same direction of effect across five independent collections. Possible interactions among the genetic variants identified have not been investigated and this is a further limitation of the study.",
                  "13956",
                  "['OBWCCE']"
                ],
                [
                  "39",
                  "23334806",
                  "Introduction<SEP>Complex diseases like type 2 diabetes (T2D) have multifactorial etiologies, with genetic and environmental factors playing roles (Schwartz and Collins). Genome-wide association studies (GWAS) have identified many common single nucleotide polymorphisms (SNPs) associated with disease (Hindorff et al.; Visscher et al.). However, individual SNPs confer modest risks, and cumulatively, they account for only a limited portion of missing heritability (Manolio et al.), and they have little prognostic utility (Meigs et al.). Furthermore, it has been reported that the availability of entire genomes may not be clinically informative (Roberts et al.). As a result, many models have been proposed or re-introduced to describe the genetic basis of complex disease (Gibson; McClellan and King).<SEP>Gene–environment interactions have been an important concept in evolutionary biology. For example, gene–environment interactions result in phenotypic plasticity, where a given genotype produces different phenotypes in response to different environmental conditions. The spectrum of possible phenotypes, such as variation in disease risk, is known as the “reaction norm” (Pigliucci). Thus, interactions may account for some of unexplained disease risk and/or improve our understanding of genetic basis of risk. In human disease epidemiology, gene–environment interactions describe the effect size of the combination of genetic and environmental factors as different than the effects of each factor alone (Hunter; Thomas). In the following, we consider this type of statistical gene–environment interaction.<SEP>Statistical interaction may offer hints about biological interaction, where genetic and environmental factors jointly determine physiological effects on a molecular or cellular level (Wang et al.). Biological interactions may be informative of disease etiology. For example, genetic risk for bladder cancer associated with variants of the NAT2 gene, a gene that plays a possible role in metabolism of tobacco smoke constituents, is known to be larger when considering smoking status of individuals (Rothman et al.). Thus, identification of robust statistical/epidemiological interactions can be seen as a first step toward creation of hypotheses relevant to disease pathophysiology.<SEP>We have created a method called environment-wide association study (EWAS). EWAS finds environmental factors associated with disease (Patel et al.; Tzoulaki et al.). EWAS is analogous to GWAS in which it evaluates multiple environmental factors and has proper adjustment for the multiplicity of comparisons. The associations that emerge are validated across different datasets. In sum, EWAS is a way to unify differences between genetic and environmental association studies and to accelerate our knowledge regarding potential environmental influences on health and disease (Ioannidis et al.).<SEP>Most studies in this area examine genetic or environmental factors. However, it is still rare for both types of factors to be measured in human tissue concurrently. A key challenge involves choosing which factors to examine. Analyzing all of them is not possible with current technology (Thomas), and factors are often selected for convenience, without sufficient documentation of the strength of their marginal associations. Given the complexity of gene–environment interaction analyses, there may be problems with selective analyses and selective reporting of results in a fragmented and possibly biased fashion (Ioannidis). For example, many studies do not account for all the interaction effects that they explore. Thus, there is a need to select common SNPs and exposures and systematically screen their interactions to avoid spurious results (Khoury and Wacholder; Patel et al.).<SEP>Here, we used a data-driven and systematic approach for selecting gene–environment interactions associated with a common disease such as T2D. Our specific goal was to test NHANES data (Centers for Disease Control and Prevention (CDC)) for interactions between robust factors found in GWAS and EWAS. NHANES includes 261 genotyped loci, 266 environmental factors measured in blood and urine, and clinical measures for the same individuals. We focused on the top GWAS and EWAS hits, and systematically investigated SNP–environment interactions associated with T2D. Top GWAS hits were defined as SNPs that had been associated with T2D in at least one study. Top EWAS hits had robust associations and low false discovery rates (FDRs) in multiple cohorts. In our statistical specification of interactions, we modeled disease risk due to the combination of genotypic and environmental factors as different than the sum of the risks of each factor alone (Khoury et al.; Thomas).<SEP>A source of major debate for the etiology of T2D is the thrifty genotype hypothesis, in which thrifty genes provided advantages to human populations during the hunter–gatherer era (Diamond; Neel; Zimmet et al.). People with the thrifty genotype stored food energy efficiently during times of feast, making more available to them during times of famine (Neel; Zimmet et al.). Thus, in modern societies, thrifty genotypes may have become risk genotypes. However, evidence to support this hypothesis is lacking, and competing hypotheses have emerged. A data-driven study of interactions between SNPs and common environmental exposures may shed light on this debate and to bring to fore its clinical implications.",
                  "Materials and methods<SEP>Data and selected genetic and environmental factors<SEP>Data came from National Health and Nutrition Examination Survey (NHANES) (Centers for Disease Control and Prevention (CDC)). All SNPs available had been chosen a priori by independent researchers investigating other topics (Matise et al.). These SNPs were assayed in two NHANES: 1999–2000 and 2001–2002. Genotypes were not collected in 2003–2004 and 2005–2006 and have yet to be released for 2007–2008. Of these, we used 18 SNPs with documented near-GWS associations in T2D (for examples and references, see Table S1). We computed allele frequencies of each SNP stratified on race to confirm their presence. For quality control, we estimated deviation from Hardy–Weinberg equilibrium (HWE) for each SNP by race. In NHANES, ethnicity was coded in five groups (Mexican-American, non-Hispanic black, non-Hispanic white, other Hispanic, other).<SEP>We previously used NHANES data from 1999–2000, 2001–2002, 2003–2004, and 2005–2006 to screen 266 environmental factors measured in blood or urine (Patel et al.). We identified and tentatively validated five environmental factors associated with T2D: trans-β-carotene, cis-β-carotene, γ-tocopherol, heptachlor epoxide, and PCB170. The FDR for each association was <10 % in at least 2 independent surveys. Trans- and cis-β-carotenes were measured in the 2001–2002, 2003–2004, and 2005–2006 surveys and had a significance threshold under FDR 10 % for all 3 surveys. γ-Tocopherol was measured in all four surveys. Its FDR was <10 % in 1999–2000, 2003–2004, and 2005–2006 and <20 % in the 2001–2002 survey. Heptachlor epoxide was measured in the 1999–2000, 2001–2002, and 2003–2004 surveys. Its FDR was <10 % in the 1999–2000 and 2003–2004 surveys. PCB170 was measured in the 1999–2000, 2001–2002, and 2003–2004 surveys and its FDR was <10 % in the 1999–2000 and 2003–2004 surveys. In the current investigation, we analyzed the 1999–2000 and 2001–2002 surveys.<SEP>T2D was defined in survey participants having 8.5-h fasting blood glucose (FBG) values ≥126 mg/dL, as advised by the American Diabetes Association. We acknowledge neither FBG nor the self-reported diabetes status distinguishes between Type 1 diabetes (T1D) and T2D, but given that T2D accounts for over 95 % of all diabetes cases, we assume most of our cases are T2D. To increase study power, we combined data from the 1999–2000 and 2001–2002 surveys. Depending on the genetic and environmental variables tested, there were 841–2,655 controls and 81–274 case subjects.<SEP>Each SNP was coded for the number of risk alleles as designated in the publications citing it (Chen et al.). Environmental factors were continuous and followed a long-tailed distribution; thus, they were log-transformed and standardized (expressed in standard deviation units) (Patel et al.).<SEP>We compared age, BMI, sex distribution, and race in T2D subjects and controls. Risk alleles of SNPs from the literature were found and their frequencies in NHANES subjects were computed by race. We computed baseline levels of environmental factors in both groups. We assessed the marginal effect between SNPs or environmental factors on T2D with survey-weighted logistic regression, adjusting for race, age, sex, and BMI. We also assessed marginal effects stratified by race.<SEP>Baseline characteristics for each subsample<SEP>We defined a subsample as a group of survey participants available for each interaction test. SNPs and environmental factors were not assayed equally among subjects, so each subsample contained a different number of participants.<SEP>We computed baseline characteristics for each subsample to assess overall differences between them. First, we computed risk allele frequency stratified by race in the HapMap as follows: CEU (Americans of European decent living in Utah), ASW (Americans of African ancestry living in the Southwest), and MEX (Americans of Mexican ancestry living in Southern California) (Frazer et al.).<SEP>We ascertained whether SNPs might be correlated with serum levels of environmental factors. We evaluated the correlation between genetic and environmental factors through survey-weighted linear regression, regressing log base 10 of the environmental exposure variable on each genetic variable, adjusted for race, sex, age, and BMI.<SEP>Systematic interaction screen between SNPs and environmental factors<SEP>Next, we screened the 90 possible pairs for interaction (18 genetic loci times 5 environmental factors; Fig. 1a, b). We utilized survey logistic regression to associate each pair of factors to T2D, incorporating a multiplicative interaction term and main effects of both factors. Each model was adjusted by age, sex, race, and BMI. <SEP>NHANES is a complex multi-staged survey and we accounted for its sampling methodologies as recommended by the National Centers for Health Statistics (NCHS) (CDC and National Center for Health Statistics (NCHS) 2003). Specifically, we used 4-year survey probability weights corresponding to the smallest subsample analyzed to accurately estimate point estimates of effects (Vittinghoff et al.). We also accounted for clustering and stratification of the survey to compute standard errors and p values accurately (Vittinghoff et al.). We used SAS version 9.2 and the “PROC SURVEY” suite of commands, as also recommended by the NCHS (CDC and National Center for Health Statistics (NCHS). Restricted data were accessed with permission through a Research Data Center (RDC) in Hyattsville, Maryland.<SEP>Multiplicity correction and false discovery rate estimation<SEP>Bonferroni multiplicity correction adjusts the threshold for statistical significance by the number of statistical tests conducted. Since our tests were not independent, we estimated the total number of effective genetic loci and environmental exposures tested jointly by accounting for the correlation between selected factors. This approach estimates the number of hypotheses for a group of correlated factors and has been applied to the study of SNPs (Nyholt). We expanded the method to environmental factors. For the 18 SNPs, we calculated the correlation between SNPs stratified by race, and concluded that there were 17.7 effective genetic factors. For the 5 environmental factors, we calculated 4.41 effective factors. Thus, the total number of effective tests was 78.1 (17.7 × 4.41). The adjusted level of significance for a single test with a threshold of p = 0.05 was 0.0006 (0.05/78.1).<SEP>We also calculated the FDR (Storey), the expected ratio of the number of false positives to the total number of positives, or the expected fraction of results drawn from the null distribution at a given significance level (Benjamini and Hochberg). To estimate the number of false positives, we generated a distribution of null test statistics corresponding to an interaction term, while preserving the main effects of the SNP and exposure terms, using a parametric bootstrap method (Bůžková et al.).<SEP>The parametric bootstrap methodology is a randomization technique that simulates the distribution of test statistics for the interaction term under the null hypothesis of no interaction. To create the null distribution of test statistics corresponding to the interaction term (β G×E), we fit a logistic regression model omitting the interaction term (β G×E = 0), while leaving parameters that modeled the main effects of the environmental factor, SNP, and age, sex, race, and BMI. We “bootstrapped” (randomized with replacement) fitted dependent values (predicted odds for diabetes) from the null model and refit the interaction model described above, adding the covariate corresponding to the interaction between SNP and environmental factor (β G×E). To simulate a null distribution of test statistics, this procedure was repeated 100 times. The FDR was estimated as the ratio of interaction terms deemed significant in the simulated distribution to all results deemed significant in the real data.<SEP>We screened all 90 interaction pairs and collected all p values corresponding to the coefficient of the interaction, β g×e. We called these p values P real. These models were specified as:<SEP>For each of the 90 interaction pairs, we computed the model corresponding to “no interaction,” or β g×e = 0:<SEP>From Model 2 corresponding to β g×e = 0, we bootstrapped sample (sample with replacement) Y. They were used as dependent values for Model 1, which corresponded to the interaction model. We collected p values corresponding to the interaction term β g×e.<SEP>Steps 2 and 3 were repeated 100 times, leading to 100 × 90 (9,000) models. The set of 9,000 p values collected in this way was called P null.<SEP>We estimated the FDR for a given significance level. For example, for a significance level of 0.05:<SEP>The specific bootstrapping procedure was as follows:  Here, “g” denotes the SNP coded by the number of risk alleles and “e” is the environmental factor, mean-centered and standardized by the standard deviation.  The predicted values for Logit(T2D) are called Y. FDR(0.05) = ((#P null < 0.05)/100)/(#P real < 0.05).<SEP>Power calculations for interactions<SEP>Sample sizes, genetic risk allele frequency and marginal OR, and environmental exposure marginal OR were used to compute power to detect moderate-to-high interaction effects (interaction OR = 1.5 and 2.0) at p value of 0.01 (FDR <18 %). We assumed marginal effects of genetic factors observed among NHANES cohorts (genetic marginal OR = 1.0) and environmental factors OR as observed in EWAS (exposure marginal OR = 1.5). We used Quanto software for these calculations (Gauderman and Morrison).<SEP>Analyses stratified by race and consideration of other T2D risk factors<SEP>Our main analysis scan included all participants from diverse ancestral groups, as reflected in NHANES. Given that the strongest evidence for T2D associations has come from studies of Caucasians, we also performed interaction analyses stratified by race.<SEP>We investigated whether eight other candidate T2D risk factors interact with 18 SNPs. These other factors included BMI and pulse rate (a proxy for physical fitness). They also included self-reported intake variables derived from a 24-h dietary questionnaire. In this questionnaire, NHANES participants were queried regarding foods they consumed in 24 h prior to the survey. Variables derived from this questionnaire include total energy (as calories), carbohydrate, monounsaturated fat, polyunsaturated fat, total saturated fat, and total fat consumed in 24 h. As above, all variables were standardized by the population mean and standard deviation. Interaction models were specified as above (main effects and multiplicative interaction term), adjusting for BMI, self-reported race, sex, and age. Sample sizes for these tests were greater than in our main analyses (N = 278–291 cases, 3,066–3,139 controls).",
                  "Results<SEP>Methodological overview<SEP>Figure 1 shows a schematic overview of our approach to search for interacting SNPs and environmental factors associated with T2D (defined as fasting blood glucose ≥126 mg/dL as recommended by the American Diabetes Association). We used a dataset containing measurements for SNPs and environmental factors and chose data with strong evidence of association of marginal effects in GWAS and EWAS.<SEP>GWAS provide a framework for assembling robustly replicated sets of common SNPs with genome-wide significance [GWS, p < 5 × 10−8 (Pearson and Manolio)]. As noted above, EWAS provides a way to search for and validate environmental factors associated with disease (Patel et al.). We selected environmental factors with significant associations in 2–4 independent surveys after accounting for the multiplicity of analyses and adjusting for demographic factors.<SEP>First, we examined the separate marginal effects of each SNP/genetic variant (“G”) or each environmental factor (“E”) on T2D. Next, we computed correlations between each environmental factor and SNP (total of E × G correlations) to ascertain the degree of their dependence.<SEP>Each environmental factor and SNP (total of E × G statistical tests) was tested for interaction while adjusting for age, sex, BMI, and race (self-reported) (Fig. 1b) in association to T2D. Finally, we accounted for multiplicity of analyses with both Bonferroni-adjusted p values and FDR estimation (Fig. 1c).<SEP>Baseline characteristics of cohorts<SEP>Table 1 shows that the mean ages of diabetics and non-diabetics who participated in the 1999–2002 surveys differed (56 years for diabetic vs. 40 years for non-diabetics), as did their sex (39 % female diabetic vs. 52 % female non-diabetic), and mean BMI (32 vs. 27 kg/m2). <SEP>Risk allele frequency and genetic marginal associations<SEP>Figure S1 shows estimated allele frequencies and main effects of the risk allele for each of the 18 SNPs in the two NHANES. Risk allele frequencies were >5 % (or <95 %) for all of the ethnicities, except for rs1801282 (97 % risk allele frequency/3 % minor allele frequency) in blacks (Figure S1). We estimated deviation from HWE per race through a Chi-squared test. Gross deviations from HWE were not found in the Mexican-American, non-Hispanic white, and non-Hispanic black races (p > 0.05).<SEP>Three of 18 SNPs were marginally associated with T2D at significance level of 0.05 after adjustment for age, sex, race, and BMI. They were rs10923931 (NOTCH2), rs7903146 (TCF7L2), and rs13266634 (SLC30A8) (Figure S1). These data are uncorrected for multiple hypotheses, given that these SNPs have been associated with T2D in populations of European descent (Table S1).<SEP>We computed the marginal effect sizes for each SNP in non-Hispanic whites, non-Hispanic blacks, and Mexican-Americans to assess the potential effects of race on the marginal estimates (Figure S1). We did not find any strong associations for most of the 18 loci (uncorrected p > 0.05 for most SNPs for all races). Notable exceptions were rs13266634 (SLC30A8, p = 0.05 for non-Hispanic white), rs2237895 (KCNQ1, p = 0.04 for non-Hispanic black), rs8050136 (FTO, p = 0.001 for Mexican-Americans), rs7903196 (TCF7L2, p = 0.02 for non-Hispanic white).<SEP>SNPs nominally (p < 0.05) or marginally (p < 0.15) associated with T2D had OR similar to that observed in the literature (Table S1). However, marginal effects of other SNPs were null (OR 1.0–1.1), to be expected given our smaller cohorts (NHANES) and small effects observed in GWAS (Table S1).<SEP>Environmental factor marginal associations<SEP>We had detected five environmental factors in our T2D EWAS (Patel et al.). Figure S2 shows marginal T2D associations between diabetics and non-diabetics and average serum levels of these factors. The pollutant factors PCB170 and heptachlor epoxide had adjusted odds ratio of 1.7 and 1.5, respectively (p = 0.01, adjusted for age, sex, race, and BMI) for a 1 SD change in logged exposure. The nutrient factor γ-tocopherol had an adjusted odds ratio of 1.5 (p < 0.001) and two β-carotene factors were associated with protection from T2D, with odds ratios of 0.6 (p = 0.003 and 0.006). While we observed some differences in mean serum levels of environmental factors between races, the directions of effect size estimates were consistent between them (Figure S2).<SEP>To study interactions between genotypes and environmental factors, we computed the sample sizes available for each pair of environmental factor and SNP, defined here as “subsamples.” Subsamples contained 81–274 diabetic case participants and 841–2,655 control participants. In spite of the differences in absolute numbers, the percentage of diabetics per subsample was constant at 9–10 % (Figure S3A).<SEP>Power calculations for genotype–environment interactions depend on minor allele frequency, environmental factor variability, the ratio of cases to controls, and marginal associations to disease (Gauderman and Morrison). We estimated the minor allele frequencies in our NHANES participants (5–44 %), the ratio of cases and controls for each genotype–environment factor pair (8–10 %), and assumed standardized environmental variables (SD = 1). Further, we assumed a genetic marginal OR of 1.0 and environmental factor OR of 1.5 as observed in these NHANES participants (Figure S1, S2). Under these assumptions, we determined power of 50–100 % (median 93 %) to detect an interaction OR of 2.0 for a significance threshold α = 0.01 (corresponding to FDR <20 %) (Gauderman and Morrison) (Figure S3B).<SEP>Subsample characteristics<SEP>Figure S4A, B shows our estimates of the per-subsample risk allele frequency and estimates of the genetic marginal effect size. Overall, we found few differences in risk allele frequency between subsamples by race (Figure S4A). Further, we found little difference between risk allele frequency between non-Hispanic whites, Mexican-Americans, and non-Hispanic blacks compared to the CEU, MEX, and ASW HapMap populations (Frazer et al.) (Figure S4A).<SEP>Correlations between SNPs and environmental factors<SEP>We found little correlation between the 18 SNPs and the 5 environmental factors (Figure S5A–E). There was a nominal negative association between rs10923931 and heptachlor epoxide (p = 0.02), where levels of heptachlor epoxide decreased 10 % per risk allele. We also observed a negative association between rs10923931 and cis-β-carotene (p = 0.04), where levels of heptachlor epoxide decreased 5 % per risk allele.<SEP>Correlations among environmental factors<SEP>We computed the pair-wise correlations between each of the environmental factors (Table S2) to assess their interdependency. There was moderate-to-low correlation between most factors (ρ ranging from −0.2 to 0.34). For example, there was a low inverse correlation between both versions of β-carotene and γ-tocopherol (ρ = −0.2); however, we observed high correlation between cis-β-carotene and trans-β-carotene (ρ = 0.96).<SEP>Screening for SNP by environment interactions<SEP>To study interactions between the 18 SNPs and the 5 environmental factors, we tested 90 interactions using survey-weighted logistic regression adjusted for age, sex, race, and BMI. Figure 2 is a Manhattan-style plot showing results for the 90 interaction terms. Seven results (5 SNPs and 4 environmental factors) had p values ≤0.05 (FDR ≤23 %). FDRs for these seven results were between 1.5 and 23 %. We discuss interactions that have reached FDR <25 % here. <SEP>Our top interaction was between trans-β-carotene (a nutrient marker) and the non-synonymous SNP rs13266634 (SLC30A8). Specifically, higher levels (defined as 1 SD above the mean) of this nutrient factor appeared to have a protective effect. The interaction was significant beyond the Bonferroni-adjusted cutoff level (interaction p = 5 × 10−5, Bonferroni adjusted p value 0.006, FDR = 1.5 %). At lower levels of trans-β-carotene (defined as 1 SD below the mean), the per-allele effect size/OR was 1.8 (95 % CI 1.3–2.6), which was 40 % greater than the marginal effect (Fig. 3a). We also estimated the environmental risk for the different genotypes. For example, the adjusted OR per change in trans-β-carotene levels (environmental effect size) was protective in subjects with two risk alleles for the SNP (adjusted OR 0.5, 95 % CI 0.4–0.7), while the effects were negligible in subjects with ≤1 risk alleles (Figure S6), suggesting protective effects of trans-β-carotene for individuals with two risk alleles. We observed similar effects for cis-β-carotene and rs13266634 (Fig. 3a). <SEP>We observed an opposite effect in subjects with the rs13266634 risk alleles as levels of γ-tocopherol rose (interaction p = 0.009, FDR = 18 %, Fig. 3a); γ-tocopherol had an adverse effect in combination with the rs13266634 risk genotype. When γ-tocopherol levels were 1 SD higher than the mean, the adjusted OR was 1.6 (adjusted 95 % CI 1.3–2), a 25 % increase in per-allele adjusted OR when compared to the marginal effect (Figure S1). Genetic risk in subjects below the mean levels of γ-tocopherol appeared mitigated.<SEP>We did not detect a marginal individual association between intergenic SNP rs12779790 and T2D, but we did observe an interaction with this locus and trans-β-carotene (Fig. 3a). The protective effect of trans-β-carotene increased 50 % in subjects with two risk alleles, compared to 0.6 for its marginal per-SD effect (Figure S6). This was an adjusted per-SD environmental factor OR of 0.3 (95 % CI 0.2–0.5).<SEP>Similarly, we did not detect a marginal association between rs2237895 (KCNQ) and T2D, but we saw environmental risk of PCB170 equal to 2.8 (95 % CI 1.7–4.5, Figure S6) for individuals with two risk alleles. This was twofold greater than the marginal estimate of 1.7 (Figure S2).<SEP>Interactions stratified by race<SEP>We computed interaction estimates separately for each race. As expected, estimates for non-Hispanic whites were comparable to those of all participants (Fig. 3a, b). However, the study was underpowered for observing interaction effect sizes in Mexican-Americans or non-Hispanic blacks (Figure S7A, B). We observed amplified effects for some per-race stratified interactions. For example, non-Hispanic blacks with low levels of cis-β-carotene and risk alleles for rs12779790 had a threefold increased risk for T2D compared to the overall sample (Figure S7A). Similarly, Mexican-Americans with low levels of trans-β-carotene and risk alleles for rs4712523 (CDKAL1) also had twofold increased risk for T2D compared to all participants (Figure S7B).<SEP>Interactions adjusted by cis-β-carotene<SEP>The interaction OR conferred by the combination of cis/trans-β-carotene and rs13266634 was similar and in the same direction (Fig. 3). Furthermore, the interaction OR between γ-tocopherol and rs13266634 ran in the opposite direction. To assess independence of interactions, we re-estimated the interaction OR for the top findings while adjusting for cis-β-carotene in addition to age, sex, BMI, and race (Figure S8). After adjusting for cis-β-carotene, our inferences were unchanged. For example, for low trans-β-carotene, the per-allele adjusted OR for rs13266634 was 2.0 (95 % CI 1.4–2.8; interaction p = 2 × 10−4) after adjustment for cis-β-carotene. At high levels of γ-tocopherol, the per-allele OR for rs13266634 was 1.5 (95 % CI 1.1, 1.8; interaction p = 0.06). The interaction OR for our top findings was independent of serum cis-β-carotene levels.<SEP>Untransformed environmental factor levels<SEP>Environmental factors were log-transformed to achieve linearity. To assess sensitivity of this transformation on our interaction OR, we recomputed models for our top findings without log transforming the environmental factors. We centered and standardized each environmental factor by their raw mean and SD, and recomputed the interaction OR while adjusting for the same covariates above. In conclusion, log transforming the environmental factors did not lead to sizable changes in interaction OR as compared to using the non-logged factors for our top findings (Figure S9). For example, the adjusted OR for participants with 1 SD lower than the mean for trans-β-carotene was 1.8 (95 % CI 1.4–2.5) per one allele change in rs13266634, comparable to the estimate computed using log-transformation of trans-β-carotene.<SEP>Limited evidence to support interactions with other risk factors<SEP>BMI, physical fitness, energy intake, carbohydrate intake, and fat intake are well-known risk factors for T2D. We assessed eight variables representing these factors in interaction with the 18 SNPs tested in this study. We were unable to uncover substantial interaction effects that would survive multiple comparison control (Figure S10). The lowest p value was 0.02, corresponding to FDR of 100 %. We did observe a modest interaction between BMI and rs8050136 (FTO) (uncorrected interaction p value = 0.03). rs8050136 is an obesity-related locus whose association with T2D is explained primarily through its effect on BMI (Zeggini et al.).",
                  "Discussion<SEP>We have shown that it is possible to screen for gene–environment interactions by integrating results from GWAS and EWAS. Our most promising results are candidates for prospective studies in additional independent cohorts.<SEP>We chose environmental factors and SNPs with strong evidence for marginal associations in EWAS and GWAS. However, it would also be possible to evaluate interactions that lack strong evidence. Given the small marginal effects for most common SNPs, many genuine associations do not reach GWS and remain false negatives (Ioannidis et al.). Some may have strong interactions with environmental factors (Khoury and Wacholder), and may only be discovered if appropriate joint environmental variables are considered. However, choosing them from millions of non-GWS SNPs would be a significant challenge. In addition, testing for interactions is power-intensive (Hunter), and testing a large number would impose a significant power and multiplicity burden (Thomas). It has been argued that strict Bonferroni multiplicity corrections need not be used when considering factors derived from previous observations (Rothman). However, we counter that interaction effects need not exist between factors that have robust evidence from EWAS and GWAS. Further, by estimating the FDR, we present a more powerful way to prioritize findings versus the Bonferroni correction.<SEP>Selecting environmental factors to test for interaction is even more difficult. In contrast to SNPs, there is no high-throughput platform that captures environmental factors with low measurement error. This lack of measuring capacity limits data.<SEP>We were able to use a prior EWAS to systematically screen 266 T2D-environmental factors measured in serum and urine. We selected five factors with the strongest support for further testing. An advantage of our approach is that it allows for hypothesis generation while keeping the total number of tests lower than testing all possible factor pairs. However, it is still very important to account for multiple hypothesis testing. We used multiplicity correction and FDR, but other approaches may also be employed (Ioannidis). Other alternatives exist to filter the hypothesis space of interactions, such as prioritizing interacting factors based on evidence of physical or toxicological interaction (Patel et al.).<SEP>There were other challenges in this study. First, we had low-to-moderate power to detect moderate interaction effects for some of the interactions we tested. Not surprisingly, the p values and effect sizes of results were modest and only one survived Bonferroni correction. We also obtained modest FDR estimates for the other highest-ranking interactions. However, we observed that the top interactions between these SNPs and EWAS factors were stronger than the interactions between the any of the same SNPs and other conventional risk factors for T2D, such as caloric intake, BMI, and physical fitness. We conclude that our top findings are ideal candidates for extensive validation through replication in higher-powered investigations.<SEP>Replication studies can investigate trends in SNP interactions with various environmental entities in populations of different ancestry. Population stratification (Smith et al.) is one type of bias for the phenotypic effect of SNPs. Although our analysis adjusted and stratified for race, to date, the SNPs identified by GWAS are best characterized in Caucasian populations. Genetic effects for GWAS-discovered markers may be different in other groups (Hayes et al.; Ioannidis; Shu et al.; Tsai et al.; Unoki et al.; Yamauchi et al.). For example, one study of African–American heart disease patients replicated 17 SNPs found in subjects of European descent. The study identified only one SNP (rs7903146 TCF7L2) associated with T2D in African–Americans from a list of 15 SNPs common to this study, including rs13266634 (SLC30A8) (Lettre et al.). Little is known about gene–environment interactions in populations of different ancestry and this idea should be investigated.<SEP>The potential imbalance of each interaction test was a limitation of this study. Ideally, each interaction pair should have the same participants. However, NHANES subjects did not all undergo the same tests. Our smallest subsamples were those with Heptachlor Epoxide and PCB170. These factors gave high marginal effects, but their analyses were lower powered relative to other subsamples. Our results may be biased and not as generalizable as tests with larger sample sizes.<SEP>There are few documented examples of interaction effects between T2D, GWS SNPs and diverse environmental or dietary factors (Cornelis et al.). We have been able to hypothesize about possible new ones. For example, the strongest evidence for interaction in our data was between rs13266634, a non-synonymous coding SNP in the SLC30A8 gene and three nutrient factors, trans- and cis-β-carotene, and γ-tocopherol. SLC30A8 is expressed in pancreatic islets and localized in insulin secretory granules of islet β cells. It appears to modulate insulin secretion and storage (Chimienti et al.). Several reports have found diet-dependent glucose intolerance and insulin secretion abnormalities in SLC30A8 knockout mice (Lemaire et al.; Nicolson et al.; Pound et al.). rs13266634 has been associated T2D in numerous GWAS [e.g., Sladek et al., Table S1], and can influence insulin secretion following glucose challenge (Staiger et al.). Thus, this SNP may be important in T2D pathogenesis. Our study enabled us to hypothesize that impaired insulin secretion driven by rs13266634 may increase T2D risk if combined with high or low levels of specific nutrients.<SEP>Alternatively, γ-tocopherol and β-carotene may be markers of other dietary components. β-Carotene is a lipid-soluble dietary factor correlated with fruit and vegetable consumption (Block et al.), components that are associated with T2D prevention (Carter et al.). In contrast, the richest sources for γ-tocopherol include soybean oils and margarine (Wagner et al.), components with higher fatty acid content. Fatty acids influence β-cell function and have been shown to even potentiate insulin secretion among individuals genetically predisposed to T2D (Ashcroft and Rorsman). Of interest, vitamin E appears to modify GWAS-identified SNPs associated with serum lipid levels, metabolic traits that are risk factors for T2D (Dumitrescu et al.).<SEP>One hypothesis under debate regarding the etiology of T2D is the thrifty genotype hypothesis, in which T2D risk genotypes provided advantages for indigenous human populations. Now, in times of more readily available nutrients and calories, a result of a different environment, these thrifty genotypes are now risk genotypes. However, evidence to support existence of such thrifty genes or interactions with these environmental factors and behaviors is lacking. To this end, competing hypotheses have emerged, including the “thrifty phenotype” (Hales and Barker) and “drifty genotype” (Speakman), whereby predisposition to metabolic diseases are a result of mismatch in nutrition environments between early (pre-childhood) and adult life or due to random genetic drift, respectively. Further, more recent events in human history, such as famine, may have played a role to enrich thrifty genes in certain populations (Diamond). Perhaps one reason behind lack of formal evidence to support these hypotheses may be that other constituents of the modern lifestyle, such as those indicated by EWAS (in addition to higher overall energy intake), may be interacting with genotypes that conferred advantages to early human populations. Future studies should examine the role of other indicators of modern lifestyle and environment on T2D as we have attempted here.<SEP>There was some unavoidable asymmetry in our selection of SNPs and environmental factors. We chose SNPs with documented robust associations with T2D and environmental factors with strong associations to T2D in NHANES. Only three variants were significantly associated with T2D overall, and only two were significantly associated with T2D in race-stratified analyses. This pattern was anticipated, given the small marginal effects of these genetic factors.<SEP>While interactions may be informative of causality (Davey Smith), these findings are subject to bias. For environmental factors, confounding and reverse causality are major issues (Ioannidis et al.). Little is known about the causal nature, if any, of these factors and T2D (Song et al.). Our findings must be confirmed in independent, larger populations. Prospective studies will be critical.<SEP>The SNPs we examined may have robust marginal associations to T2D, but could only tag the actual causal SNP. Our power is decreased for tagging SNPs that are not in complete linkage disequilibrium with the causal SNP. More importantly, etiological inference might be hindered if the causal SNP is unknown.<SEP>Nevertheless, these findings may have important implications for personalized medicine (Chan and Ginsburg) or the “missing heritability” debate (Manolio et al.). For example, Roberts et al. have recently quantified the difficulty in predicting disease risk using entire genomes of individuals. However, Roberts et al. only considered genetic or environmental main effects and interactions were not considered. On the other hand, Aschard et al. recently provided theoretical arguments that gene–environment interactions are unlikely to improve risk prediction. However, only a limited number of interactions (maximum of 10) were considered in these simulations. It is possible that inclusion of many interaction effects may increase prediction. We hypothesize that perhaps the lack of predictive capacity in the Roberts et al. investigations and predicted by the Aschard et al. simulations arises from not considering multiple interactions between environmental exposures and the genome. To test the hypothesis empirically that multiple interactions may influence heritability estimates, we would require relatedness information between participants currently unavailable in NHANES. Further, to test if multiple interactions influence risk prediction, we would require samples with same environmental and genetic measures for all participants. Nevertheless, we demonstrate one way of identifying multiple interactions to test in these contexts in future investigations.<SEP>Infrastructure-related challenges remain in this area (Hunter). First, unlike common SNPs (Hindorff et al.), we lack a complete list of candidate environmental factors. Screening and validating gene–environment interactions is power-intensive, and will require both environmental and genetic measures to be measured in multiple studies (Ioannidis et al.), augmentation of GWAS with environmental data (Khoury and Wacholder), and adoption of measurement standards (e.g., Hamilton et al.). A systematic approach to investigating the interactions of environment and the individual genome may help explain a substantial component of disease risk, lead to hypotheses regarding disease pathology, or help shed light on the debate on the genetic basis of disease (Gibson).",
                  "40777",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "40",
                  "27106561",
                  "Introduction<SEP>Evidence of parallels between addiction and feeding behavior continues to accumulate in neurobiological and behavioral research. The novel view of obesity as a neurobehavioral disorder caused by an interaction between a vulnerable brain and an obesogenic environment is reminiscent of models of drug addiction. Indeed, genome-wide association studies (GWAS) of adiposity and follow-up studies of confirmed loci yield evidence supporting a behavioral component to obesity. Early candidate and more recent GWAS of addictive behaviors have yielded a number of susceptibly loci that point to common higher-order genetic pathways underlying addiction and thus, provide an opportunity to elucidate whether specific genetic influences on drug addiction generalize to addictive eating behaviors.<SEP>The Yale Food Addiction Scale (YFAS) is a psychometric tool based on the Diagnostic and Statistical Manual for Mental Disorders (DSM)-IV codes for substance dependence criteria to assess food addiction in individuals. The YFAS has previously been associated with binge-eating episodes, hedonic eating, emotional eating, impulsivity, food and snack craving as well as patterns of neural response implicated in other addictive disorders. Food addiction was recently assessed in women participating in the Nurses’ Health Study (NHS) and Nurses’ Health Study II (NHS2) using a modified version of the YFAS, which has similar psychometric properties relative to the original YFAS. Among the subset of these women who have GW scan data our aim was to conduct the first comprehensive genetic analysis of food addiction in a population. If food addiction, as captured by mYFAS, shares a molecular pathophysiology to classical addictions, we hypothesized that a GWAS of food addiction would yield significant enrichment in genes and pathways linked to addiction.",
                  "Methods<SEP>Study populations<SEP>The NHS was established in 1976 with 121,700 female registered nurses aged 30–55 years and residing in 11 U.S. states. The NHS2 cohort was established in 1989 with enrollment of 116,609 female nurses aged 25–44 years and residing in 14 U.S. states. Every two years, women in both cohorts have been followed-up with mailed questionnaires on medical history and lifestyle characteristics. Study protocols were approved by the institutional review board of Brigham and Women’s Hospital.<SEP>Measures<SEP>Assessment of food addiction and covariates<SEP>The YFAS is composed of 25 questionnaire items that are used to assess diagnostic criteria for food addiction and provides both a count of food addiction symptoms and a diagnosis of food addiction as scoring options. We recently developed and validated a modified YFAS (mYFAS) for use in large epidemiologic cohorts by adapting the original YFAS to a core of 9 questionnaire items with 1 question from each of the symptom groups that compose the 7 diagnostic criteria plus 2 individual items that assess the presence of clinically significant impairment and distress, respectively (Table S1). For each of the diagnostic criteria, the frequency threshold from the original YFAS was used, and all questions were summed for a total score from 0 to 7 (mYscore). Reliability, convergent, discriminant, and incremental validity of the mYFAS have been reported in detail elsewhere. Briefly, in a study sample whose data were previously reported in the validation of the YFAS, the internal consistency of mYscore was adequate and identical to that of the full version of YFAS (Kuder-Richardson α = 0.75). Convergent and discriminant validity of mYscore and the original YFAS were also similar. The mYFAS diagnostic version and mYscore were also significantly associated with binge-eating scores above and beyond other measures of eating pathology compared with for the original YFAS diagnostic version and YFAS symptom count.<SEP>The mYFAS questionnaire was administered in NHS in 2008, at which time the participants were aged 62 to 87 years. In NHS2, mYFAS questionnaires were administered in 2009, when participants were aged 45 to 64 years. Response rates exceeded 80% for both cohorts and those who responded to the food addiction questions were not substantially different from those who did not, with respect to age, BMI, or smoking status. A subset of NHS2 responders were invited to complete the full YFAS in 2012 and a strong correlation in scores (Pearson’s r=0.98) and internal consistency (Cronbach’s coefficient α =0.84) was observed.<SEP>For the current GW analysis, we examined two food addiction traits: i) food addiction symptoms (mYscore) modeled as a continuous or dichotomous (mYscore ≥3) trait and ii) the presence/absence of clinically significant impairment and distress (Yclinical). mYscore and Yclinical are derived from a different set of questions as detailed in Table S1. The combined presence of ≥3 food addiction symptoms (mYscore) and significant impairment or distress (Yclinical) are proposed diagnostic criteria for food addiction (mYdiag).<SEP>All covariates were collected via self-administered questionnaire and concurrently with food addiction measures. BMI (kg/m2) was derived from self-reported weight and height, which were reported with high accuracy in our cohorts.<SEP>Genotyping, quality control and imputation<SEP>Blood was collected from 32,826 NHS members between 1989 and 1990 and from 29,611 NHS2 members between 1996 and 1999. DNA was extracted from white blood cells. Women contributing to the current genetic analysis were those previously selected for independent GWAS in nested case-control studies initially designed for a variety of chronic diseases (Table S2). To allow for maximum efficiency and power we pooled samples genotyped on the same platforms, which resulted in three datasets herein referred to as Affy (NHS), Illumina (NHS, NHS2) and Omni (NHS). Detailed methods and quality assurance pertaining to these genetic datasets have been reported elsewhere and relevant descriptive and quality control (QC) data are provided in Table S2. Any samples that had substantial genetic similarity to non-European reference samples were excluded. For each of the three datasets we used MACH (v.1.0.18.c) and Minimac (v.2012-08-15) to impute ~31 million SNPs based on the 1000G v3 ALL reference panel.<SEP>Statistical analysis<SEP>Genome-wide analysis of mYFAS<SEP>Each genetic dataset was examined separately for each cohort of women and the results were combined by meta-analysis. Within the Illumina dataset there are data from NHS and NHS2. Thus, four datasets were examined: Affy-NHS (N=3298), Illumina-NHS (N=2690), Omni-NHS (N=2520) and Illumina-NHS2 (N=806). For each dataset we performed GWA-testing for each trait across ~31 million SNPs (expressed as allele dosage), based on linear (mYscore) or logistic (mYscore ≥3, Yclinical) regression under an additive genetic model and adjusting for age, BMI, initial case-control dataset, and four principal components of population substructure. SNPs with minor allele frequency (MAF) <0.01 or with low imputation quality scores (MACH’s Rsq<0.3) were removed prior to meta-analysis (Table S2). In secondary analyses we removed BMI from the model or further adjusted for smoking status (current, past, never).<SEP>For both food addiction traits (mYscore, mYclinical), GW meta-analysis was conducted using a fixed effects model and inverse-variance weighting with a single genomic control (GC) correction as implemented in METAL. Between-dataset heterogeneity was investigated using the I2 statistic. Top loci associated with each trait were retained and formally presented if i) SNPs passed QC filters in all four datasets and ii) direction of effects were consistent across all datasets. GW significance was defined as P<2.5×10−8; the traditional threshold (P<5×10−8) corrected for the number of independent traits. Nominally significant loci were tabulated if the direction of effects was consistent across all datasets and food addiction traits. Top loci were examined for associations with mYdiag, BMI and smoking in the NHS (the larger of the two contributing cohorts). Our full summary level results were also investigated for associations with BMI based on a published large-scale GWAS.<SEP>Candidate SNP, gene-set and pathway analysis<SEP>We interrogated summary levels results from our GWAS of mYscore and Yclinical for evidence of overlap with SNPs, genes and pathways implicated in drug addiction. ‘Addiction SNPs’ included 44 independent (r2<0.8) common (MAF>0.01) SNPs achieving at least nominal significance (<1×10−6) in GWAS for addiction traits as reported in the NHGRI catalogue (August 2014) and unrelated to drug metabolism (Table S3). ‘Addiction Genes’ were also unrelated to drug metabolism and included 5 genes associated with at least one drug addiction that have been verified by meta-analysis and 60 genes associated with more than one drug addiction, as reviewed by Li et al, and genes within 300kb (or closest gene >300 kb) of each ‘Addiction SNP’. In total, 238 genes were considered ‘Addiction Genes’ (Table S4). Candidate gene-based analyses were performed using VErsatile Gene-based Association Study (VEGAS). We applied Bonferroni-corrected thresholds of P<0.001 [0.05/44 (number of SNPs tested)] and P<2.1×10−4 [0.05/238 (number of genes tested)] for SNP-level and gene-level significance, respectively. Multi-SNP linear kernel tests were additionally used to assess the joint relation between the 44 ‘Addiction SNPs’ and food addiction. These linear models allow associations of multiple SNPs to be tested simultaneously with one test and require no prespecification of risk allele direction.<SEP>Meta-Analysis Gene-set Enrichment of variaNT Associations (MAGENTA) was used to test our GW mYFAS results for addiction pathway enrichment. In preliminary pathway analyses of our ‘Addiction Genes’ (defined above) we observed consistent enrichment of genes related to the Kegg pathways: (1) ‘neuroactive ligand-receptor interaction’, (2)’tyrosine metabolism’, (3)’calcium signaling pathway’, (4)’amyotrophic lateral sclerosis’, (5)’tryptophan metabolism’, ’histidine metabolism’ and ’long-term potentiation’ (Table S5). Two of these pathways overlapped with those reported by Li et al; who further reported over enrichment of genes related to ’GnRH signally pathway’, ’MAPK signaling pathway’ and ’Gap junctions’. We additionally created an ‘Addiction’ gene-set that included the ‘Addiction Genes’ defined above (excluding 14 genes: hypothetical or pseudo genes or genes not annotated by MAGENTA). Taken together, 11 gene-sets or pathways were specifically tested and MAGENTA’s nominal significance threshold of 0.05 was applied. Hypothesis testing was supplemented with exploration: results were tested against 3,218 pathways from seven databases. For each pathway, enrichment of highly ranked gene scores above the 95th percentile of all gene scores in the meta-analyses was evaluated compared to 10,000 randomly sampled gene-sets.",
                  "Results<SEP>Characteristics of NHS and NHS2<SEP>General characteristics of the 9,314 women included in the current analysis are presented in Table S2. The mean (standard deviation, SD) mYscore for NHS and NHS2 was 0.60 (1.05) and 1.04 (1.6), respectively. The prevalence of Yclinical and mYdiag were 4.9% and 2.6% in NHS. Corresponding prevalences in NHS2 were 11.2% and 8.7%. mYscore was correlated with BMI in both cohorts (NHS Pearson’s r=0.27, NHS2 Pearson’s r=0.42; P<0.001). These results are similar to those reported for the full NHS and NHS2 cohorts, showing a higher presence of food addiction in the younger cohort.<SEP>GWAS of mYFAS<SEP>Two loci met criteria for GW-significance in GWAS of food addiction traits (Table 1, Figure S1–S3). SNPs at 17q21.31 mapping to the intronic region of PRKCA were significantly associated with mYscore (P=2.0× 10−8) and also associated with Yclinical (P=6.4× 10−4. In the NHS, the variant of the 17q21.31 index SNP rs74902201 linked to higher mYscore was also associated with a positive mYdiag (P=3.4× 10−5). SNPs at 11q13.4 mapping to the intronic region of NTM were significantly associated with mYscore but not with Yclinical. In the NHS, the variant of the 11q13.4 index SNP rs75038630 linked to higher mYscore was also associated with a positive mYdiag(P=1.7× 10−4) but with a lower BMI (P=0.02). A borderline GW-significant association was observed between an intergenic 6q22.32 locus near CENPW and mYscore (P=3.1× 10−8). In the NHS, the variant of the 6q22.32 index SNP rs139878170 linked to higher mYscore was associated with a positive mYdiag (P=1.4× 10−5). Removing BMI from the regression model strengthened the association between rs139878170 and mYscore (P=6.9 × 10−9, Table S6).<SEP>Loci (or proxies) reported in Table 1 were not associated with BMI according to a published large-scale GWAS of BMI. Of the 32 BMI loci reported in the latter, rs1558902 (FTO, P=0.04), rs206936 (NUDT3, P=0.01), and rs10150332 (NRXN3, P=0.05) were associated with mYscore when modeled dichotomously.<SEP>Candidate addiction SNPs<SEP>No association between candidate SNPs and food addiction traits met our pre-specified significance threshold (P<0.001). Among the 44 SNPs previously associated with addiction traits in GWAS, only a nominally significant association between the intergenic SNP rs1868152 (previously associated with illicit drug use) was observed for mYscore and Yclinical (P<0.004, Table S7). Results from multi-SNP linear kernel tests were not consistent across Affy-NHS, Omni-NHS, Illumina-NHS and Illumina-NHS2. Tests were only significant for dichotomously modeled mYscore in NHS-Affy (P=0.03) and Yclinical in NHS2-illumina (P=0.01). Tests performed on the combined datasets were not significant (P>0.07).<SEP>Candidate addiction genes<SEP>No candidate gene-based test for associations with food addiction traits met our pre-specified significance threshold (P<2.1×10−4) (Table S7). The most statistically significant gene was LOC100130673 (P<5.0×10−4 for mYscore), a pseudogene on chromosome 7 selected for its proximity to a smoking behavior SNP (rs215614). The latter is ~135kb away from our top SNP in this pseudogene (rs61436781, P<9.7×10−6) and is not in LD (r2<0.2). Four ‘addiction genes’ were nominally significant (P<0.05) for both mYscore and Yclinical including HOMER1, ZHX2, DRD2 and SURF6.<SEP>Candidate addiction pathways<SEP>Yclinical results were significantly enriched for MAPK signaling pathway genes (P=0.02, Table 2) and this same gene-set reached borderline significance for mYscore (P=0.07). Enrichment for tyrosine, histidine and tryptophan metabolism genes was observed for the continuously modeled mYscore (P<0.03). Our mYscore and Yclinical results were not significantly enriched for gene members of our custom addiction gene-set (Table 2). An exploratory GW pathway analysis of food addiction, yielded significant enrichment for genes in GO’s interleukin-1 receptor (IL1R) binding pathway [mYscore (binary), FDR=0.003]. Larger but similar gene sets from Ingenuity (IL-10 signaling) and Biocarta (IL1R pathway) were also among the top pathways.",
                  "Discussion<SEP>The concept of ‘food addiction’ (or ‘eating addiction’) is an ongoing debate of growing scientific interest. The most compelling evidence supporting such a condition is the overlapping neurobiological systems reportedly activated by both drugs of abuse and highly palatable foods in experimental and clinical models. In the current study, we examined whether genetic determinants of food addiction overlap with those of drug addictions. To this end, we conducted the first GWAS of mYFAS and identified suggestive loci worthy of further follow-up but provide limited support of shared genetics with drug addictions based on comprehensive candidate SNP, gene and pathway analysis.<SEP>We identified novel GW significant loci in PRKCA and NTM in two populations of U.S. women of European ancestry. Each index SNP variant conferring a higher mYscore occurs in ~6% of European populations and in 0 to 16% in non-European populations. PRKCA encodes a calcium-activated, phospholipid- and diacylglycerol-dependent serine/threonine-protein kinase that is involved in regulating numerous biological processes such as insulin signaling, inflammation and mitogen-activated protein kinase (MAPK) activity. MAPK signaling is highly implicated in brain function and also a candidate drug addiction pathway. The index SNP in PRKCA resides in regulatory regions for multiple tissues, notably enhancer regions for the brain, and alters a binding site of NRSF, a transcriptional repressor of neuronal genes in non-neuronal tissues. PRKCA has previously been associated with both BMI and asthma via linkage and follow-up SNP analysis by Murphy et al. SNPs examined by Murphy et al were not associated with mYFAS in the current study (P>0.32) nor with BMI in GWAS. NTM at 11q25 encodes neurotrimin and is highly expressed in human brain tissue and closely linked to opioid binding protein/cell adhesion molecule-like (OPCML), also on chromosome 11. This SNP region binds NR2C2, which serves as an important repressor of several nuclear receptor signaling pathways and is required for normal cerebellum development. Significant to nominal associations in the 11q25 region have also been reported for alcohol dependence (OPCML), body fat distribution (OPCML), and various other traits but none of the index SNPs in these reports were associated with food addiction in the current study. Only one (rs4937665) of the top SNPs near NTM previously associated with IQ in GWAS was also associated with food addiction traits. Food addiction has not been measured in any other large population-based study and thus we were unable to replicate our two novel and promising loci in an independent study.<SEP>Our GW analysis of food addiction provides limited support for shared pathways among food addiction and drug addiction. Literature informed SNPs and genes linked to addiction traits were not associated with mYFAS. Of the eleven addiction related pathways tested, only the MAPK signaling pathway met our threshold for significance. The same pathway was nominally significant (nominal GSEA P=0.02) in the published gene-set enrichment analysis of BMI; BDNK, NFKB1 and MAP2K5 were this pathway’s gene-members mapping to established BMI loci. PRKCA and its neighboring calcium channel genes are also members of this pathway which may have induced a degree of chance or, alternatively, lends support for our novel loci and a link between drug addiction and food addiction. Interestingly, interleukin-1 receptor binding, a significant pathway in our global pathway analysis and also recently implicated in addiction behavior, does not include PRKCA or NTM, suggesting additional and novel loci of sub-GW significance in the current study await discovery in future efforts.<SEP>Davis et al developed a genetic risk score for dopamine signaling and among 121 overweight adults reported a higher score (conferring elevated dopamine signaling) in those with YFAS diagnosed food addiction and a positive correlation with binge eating, food cravings, and emotional eating. Their score included 6 SNPs near DRD2, SLC6A3 and COMT. Four (rs1800497, rs6277, rs12364283, rs4680) of the 6 SNPs were genotyped/imputed in the current study. High quality proxies for the remaining two SNPs were not available. rs1800497 was associated with Yclinical (P=0.04) and in the expected direction. rs12364283 was associated with mYscore when modeled dichotomously (P=0.03) but the direction of effect was opposite to that expected. All three genes were ‘addiction candidate genes’ that were examined in the current study but none met our significance criteria. Differences in study population might explain the discrepancies between the current study and that of Davis et al.<SEP>GWAS of BMI have identified several loci and some of these are implicated in hedonic rather than homeostatic pathways for obesity. BMI is not a direct measure of food addiction but purportedly has a behavioral component which forms the basis of the obesity- addiction hypothesis. Thorgeirsson et al recently reported a significant positive correlation between a genetic risk score for higher BMI and smoking behaviors (smoking initiation, and dose). Less than 10% of our cohort participants were current smokers and this limited our ability to conduct a similar analysis. Nevertheless, none of our GW-significant food addiction SNPs were associated with BMI in a large GWAS and only nominal associations between 4 of 32 validated obesity loci and food addiction were observed. These results together with limited evidence of overlap with addiction pathways conflict with conclusions drawn by Thorgeirsson et al. In the NHS, the associations between our GW-loci and mYFAS were largely independent of BMI. Nevertheless, individuals genetically predisposed to food addiction might be more susceptible to obesity in the presence of an environment that fosters availability of palatable foods; a concept analogous to that described for illicit drug addiction.<SEP>The current study marks the first comprehensive genetic analysis of food addiction in a population. We discovered novel loci associated with food addiction traits warranting independent replication. We additionally gained insight to potential genetic overlaps between food addiction and drug addiction by integrating our GW food addiction results with extant genetic knowledge of drug addiction. Although results suggested that the MAPK signaling pathway may be a shared determinant, our results taken together suggest the genetic underpinnings of food addiction and drug addiction are largely distinct. However, our study may have had modest power to detect novel loci and significant overlap with genetic variation of other addictions, given the sample size, measurement error in food addiction assessment and the narrow range and low prevalence of food addiction symptomology. The latter might be addressed in future studies by employing a case-enrichment design. The ability to identify shared genetic determinants of food and drug addiction may also be limited by incomplete knowledge of genetic determinants of drug addiction.<SEP>Disclosure: The authors declare no conflict of interest.<SEP>Author Contributions: MCC designed the study, performed the analysis and wrote the manuscript. All authors contributed to discussion and reviewed/edited the manuscript.<SEP>What is already known about this subject?<SEP>The concept of ‘food addiction’ is supported by the overlapping neurobiological systems reportedly activated by both drugs of abuse and highly palatable foods in experimental and clinical models<SEP>Early candidate and more recent GWAS of addictive substances have yielded a number of susceptibly loci that point to common higher-order genetic pathways underlying addiction<SEP>The Yale Food Addiction Scale (YFAS) is a psychometric tool used to assess food addiction and has previously been associated with disordered eating and patterns of neural response implicated in other addictions<SEP>What does this study add?<SEP>The authors conducted a conducted a GWAS of food addiction, determined by a modified version of the YFAS among women and examined results for enrichment of single-nucleotide polymorphisms, genes and pathways implicated in drug addiction<SEP>Two loci met GW-significance mapping to regions that harbor genes with no obvious roles in eating behavior. GW results were significantly enriched for gene members of the MAPK signaling pathway but no candidate SNP or gene for drug addiction was significantly associated with food addiction<SEP>The authors identified suggestive loci worthy of further follow-up, but provide limited support for shared genetic underpinnings of food addiction and drug addiction",
                  "23893",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "41",
                  "22589738",
                  "Introduction<SEP>Obesity is an important risk factor for cardiometabolic outcomes. Heterogeneity in the regional deposition of fat, particularly, visceral adipose tissue (VAT), may be more deleterious than total body obesity. Numerous epidemiologic studies have demonstrated that central obesity, measured by simple anthropometric measures including waist circumference or waist-hip-ratio (WHR), is associated with cardiovascular disease (CVD) and glucose, insulin, and lipid metabolism, independent of overall obesity as measured by body mass index (BMI). However, waist circumference is limited due to its inability to discriminate between VAT and subcutaneous adipose tissue (SAT). Computed tomography (CT) provides a more direct and precise assessment of adipose tissue compartments. In many studies, the associations between CVD risk factors and directly-measured VAT are stronger than the associations observed with other typical anthropometric measures.<SEP>Prior studies have shown that indices of body fat distribution, including waist circumference, VAT, and SAT are heritable. A recent large-scale genome-wide association study (GWAS) identified 14 loci in association with waist-hip-ratio, providing proof-of-principle for the concept that genetic variants are associated with body fat distribution above and beyond generalized adiposity. However, there are currently no large-scale GWAS for directly-measured VAT and SAT. Thus, the purpose of the present study was to perform GWAS for VAT and SAT in 4 large population-based cohorts. We analyzed SAT, VAT, VAT adjusted for BMI, and the VAT/SAT ratio, a metric of the propensity to store fat viscerally as compared to subcutaneously. Given the known sex differences in body fat distribution, we additionally performed sex-specific analyses.",
                  "Methods<SEP>Phenotype Definition<SEP>VAT and SAT were measured on CT following protocols established by each study as detailed in the Study-Specific Methods. We created sex-specific residuals adjusting for age, age-squared, smoking and principal components derived from genotypes denoting population stratification. The following traits were created in the overall sample and in women and men separately for each participating study: VAT, SAT, VAT-adjusted-for-BMI, and VAT/SAT ratio. VAT-adjusted-for-BMI provides insight into the relative amount of VAT controlling for the degree of generalized adiposity, whereas the VAT/SAT ratio is a metric of the propensity to deposit fat viscerally as compared to subcutaneously. The VAT/SAT ratio has been previously shown to be associated with cardiometabolic risk factors. The correlation between VAT-adjusted-for-BMI and the VAT/SAT ratio in the Family Heart Study is 0.76 (p<0.0001; N = 2658).<SEP>Heritability Analyses<SEP>We created sex-and-cohort specific residuals, which were then pooled and analyzed using variance components analysis (SOLAR).<SEP>Genotyping<SEP>Genotyping was conducted as specified in the Table S1 and the Study Specific Methods. We applied quality-control filters in order to exclude low-quality SNPs or samples. Based on CEU samples, each study imputed ∼2.5 million Phase 2 HapMap SNPs. We used imputed allelic dosage in the analysis. Additional details can be found in Table S1.<SEP>Primary Association Analyses and Meta-Analysis<SEP>The primary analysis was conducted in each cohort separately using regression analysis, assuming additive genetic effects and accounting for dependence among family member when appropriate. We accounted for principal components where necessary in order to prevent population stratification as well as the assumption of homogeneity within samples of European ancestry. These results were gathered together and used to conduct a fixed effects weighted Z meta-analysis (to allow for differences in phenotype scaling across the participating studies) using METAL. We have previously shown that the association between single-slice VAT and SAT measurements at the L4/L5 level is highly correlated with volumetric measurements (r = 0.95–0.99). This approach weights the signed Z statistics from each study by its sample size to obtain a weighted sum, from which a p value is obtained. We applied the genomic control correction to control type I error rates. SNPs that reached a meta-analysis P value≤5×10−8 were considered to be genome-wide significant.<SEP>Validation Analysis<SEP>Stage 2 validation was conducted for our novel lead SNP, rs1659258, using data from studies not part of the present meta-analysis in the GIANT consortium data in sex-specific meta-analyses for BMI and waist circumference (personal communication, Heid I et al). P-values for SNPs discovered in the present effort in association with adiposity phenotypes reported by the GIANT consortium were queried. We concluded significant results when a direction-consistent p-value was at least p<0.05.<SEP>Analyses of Related Phenotypes<SEP>For each validated SNP, we obtained sex-specific association results for lipid and glycemic phenotypes from the MAGIC and GLGC consortia.<SEP>Interaction Testing<SEP>We performed formal tests of interaction of rs1659258 by sex (women as compared to men) and age (stratified above/below age 60). Briefly, each study generated the interaction regression coefficient, its standard error and its p value through regression. For the sex interaction, we included, age, age-squared, smoking (yes/no), sex, and any principal components (and study center) that were used in the original discovery analysis. We additionally added rs1659258 and the cross-product rs1659258*sex. The age interaction analysis was performed in women only. Because the AGES and Health ABC studies only included participants older than 65 years, they did not contribute to this analysis. Model covariates included age, age-squared, smoking (yes/no), and any principal components (and study center) with the addition of rs1659258 and the cross-product of rs1659258 and the dichotomized age term. We meta-analyzed the interaction terms across all studies using the weighted z-score approach.<SEP>eSNP Analysis<SEP>Using publically available datasets, we tested for the association of rs1659258 in expression SNPs (eSNP) datasets comprised of the following tissue types: lymphocytes, leukocytes, leukocytes from patients with Celiac disease, lymphoblastoid cell lines (LCL) from children with asthma, HapMap LCL from 3 populations, a separate study on HapMap CEU LCL, peripheral blood monocytes, adipose tissue, and blood samples, 2 studies on brain cortex, three large studies of brain regions including prefrontal cortex, visual cortex and cerebellum (Emilsson, personal communication), liver, osteoblasts, skin, and additional fibroblast, T cell and LCL samples. We considered significance to be the association with gene transcript levels as originally described.<SEP>For sex-specific expression data, we queried data from patients who underwent a Roux-en-Y gastric bypass. Briefly, we queried a 1 MB region surrounding our lead SNP for VAT in women and men separately. Altogether, 31 genes were included in the query. Statistical significance was considered to be a p-value<0.05 after correction for multiple testing.<SEP>Study-Specific Information: Framingham Heart Study<SEP>In 1948, the Framingham Heart Study began when the Original Cohort was enrolled. Beginning in 1971, the Offspring Cohort was enrolled (5,124 participants); the methodology and design has been described. In 2002, the Third Generation cohort was enrolled (n = 4095). Participants for this study were drawn from the Framingham Heart Study Multi-detector Computed Tomography (MDCT) Study, a population-based sub-study of the community-based Framingham Heart Study Offspring and Third Generation cohorts. Participants for the current study were drawn from the MDCT sub-study. Between June 2002 to April 2005, 3529 participants (2111 Third Generation, 1418 Offspring participants) underwent MDCT assessment of coronary and aortic calcium. Inclusion in this study was weighted towards participants from larger Framingham Heart Study families and those who resided in the Greater New England area. Men had to be at least 35 years of age, women had to be at least 40 years of age and non-pregnant, and all participants had to weigh less than 350 pounds. Of the total of 3529 subjects imaged, 3394 had interpretable CT measures, 3329 of whom had both SAT and VAT measured, and 3158 participated in the present GWAS study.<SEP>We observed association with the first principal component estimated using Eigenstrat; this component was included in our regression models.<SEP>Volumetric adipose tissue imaging<SEP>Subjects underwent eight-slice MDCT imaging of the chest and abdomen in a supine position as previously described (LightSpeed Ultra, General Electric, Milwaukee, WI). Briefly, twenty-five contiguous five mm thick slices (120 kVp, 400 mA, gantry rotation time 500 ms, table feed 3∶1) were acquired covering 125 mm above the level of S1.<SEP>Abdominal adipose tissue measurements<SEP>Subcutaneous and visceral adipose tissue volumes (SAT and VAT) were assessed (Aquarius 3D Workstation, TeraRecon Inc., San Mateo, CA). In order to identify pixels containing fat, an image display window width of −195 to −45 Hounsfield Units (HU) and a window center of −120 HU were used. The abdominal muscular wall separating the visceral from the subcutaneous compartment was manually traced. Inter-reader reproducibility was assessed by two independent readers measuring VAT and SAT on a subset of 100 randomly selected participants. Inter-class correlations for inter-reader comparisons were 0.992 for VAT and 0.997 for SAT. Similar high correlations were noted for intra-reader comparisons.<SEP>Imputation<SEP>As a reference panel for imputation, we used Phase II CEU HapMap individuals; we imputed genotypes to nearly 2.5 million HapMap SNPs; further details are presented in Table S1. We used MACH v1.0.15/16 (http://www.sph.umich.edu/csg/abecasis/MACH/) in conjunction with 200 (101 Men and 99 Women) biologically independent individuals to establish parameter estimates and then used the parameter estimates to infer gene dosage for all study participants. We expressed imputed genotypes as allelic dosage (which is a decimal value ranging from 0–2).<SEP>Statistical analysis<SEP>We performed linear mixed effects regression modeling for SAT, VAT, and the VAT/SAT ratio to account for pedigree structure (using the R lme and kinship package).<SEP>Study-Specific Information: Family Heart Study<SEP>The Family Heart Study (FamHS) is a multicenter, population-based, family study designed to investigate the determinants of cardiovascular disease. Families in the FamHS were selected at random (588 families) or ascertained for family history of CHD (656 families) using information collected in the parent studies—Framingham Heart Study (Framingham, MA, USA), the Utah Health Family Tree Study (Salt Lake City, UT, USA) or the Atherosclerosis Risk in Communities Study (Minneapolis Suburbs, MN, USA and Forsyth County, NC,USA). Between 2002 and 2003 about two-thirds of the largest families were invited to participate in a follow-up clinical examination that included measurement of the liver and abdomen with cardiac CT using standardized procedures and quality control methods developed in NHLBI's MESA and CARDIA studies. Informed consent was obtained from all participants, and this project was approved by the Institutional Review Boards of all participating institutions.<SEP>CT scan–related phenotypes<SEP>Participants underwent a cardiac MDCT exam with four detectors using a standardized protocol as described previously. For participants weighing 100 kg (220 lbs) or greater, the mAs were increased by 25%. The effective radiation exposure for the average participant of each coronary scan was 1.5 mSv for men and 1.9 mSv for women. Participants received two sequential scans. CT images from all study centers were sent electronically to the central CT reading center located at Wake Forest University Health Sciences, Winston Salem, NC, USA.<SEP>CT scans of the abdomen were reconstructed into 5 mm slices with the maximum 50 cm field-of-view to include the whole abdomen for body composition. Total and adipose tissues were measured volumetrically from two 5 mm contiguous slices located at the level of the lumbar disk between the 4th and 5th vertebra so as to be comparable to a single 10 mm slice used historically. Tissues with an attenuation between −190 to −30 Hounsfield units was define as adipose tissue. The MIPAV application (http://mipav.cit.nih.gov/index.php) was used by experienced analysts to segment the images based on anatomic boundaries (skin, subcutaneous fat-muscle interface and peritoneum) into the entire abdomen, abdominal wall and intra-abdominal compartments. In each compartment the total volume and fat volumes were determined allowing calculation of the total abdominal adipose tissue, subcutaneous adipose tissue and visceral adipose tissue contained with the 10 mm slice located at L4–5.<SEP>Statistical analysis<SEP>We performed linear mixed effects regression modeling for SAT, VAT, and the VAT/SAT ratio, accounting for dependency among family member as a function of their kinship correlation (R kinship package).<SEP>Study-Specific Information: HABC Study<SEP>The Health ABC study is a prospective cohort study investigating the associations between body composition, weight-related health conditions, and incident functional limitation in older adults. Health ABC enrolled well-functioning, community-dwelling black (n = 1281) and white (n = 1794) men and women aged 70–79 years between April 1997 and June 1998. Participants were recruited from a random sample of white and all black Medicare eligible residents in the Pittsburgh, PA, and Memphis, TN, metropolitan areas. Participants have undergone annual exams and semi-annual phone interviews. The current study sample consists of 1559 white participants who attended the second exam in 1998–1999 with available genotyping and SAT/VAT data.<SEP>Regional fat depots were assessed from CT scans obtained in Pittsburgh on a General Electric 9800 Advantage (General Electric, Milwaukee, WI) and in Memphis on a Siemens Somatron Plus 4 (Siemens, Erlangen, Germany) or Picker PQ2000S (Marconi Medical Systems, Cleveland, OH). A single axial scan (140 kVp, 300 to 360 mAs, 10-mm thickness) was taken at the disk space between the fourth and fifth lumbar vertebrae. Images were transferred to the Reading Center at the University of Colorado Health Sciences Center on optical disc or magnetic tape. Analyses were performed on a SPARC station II (Sun Microsystems, Mountain View, CA) using IDL development software (RSI Systems, Boulder, CO). An outline was traced surrounding the abdominal cavity. The adipose tissue density range was determined with a bimodal image distribution histogram for each participant. Visceral fat was defined as the area of all adipose tissue within the abdominal cavity with exclusion of the muscle region, calculated by multiplying the number of pixels within this range by a single pixel area. Abdominal subcutaneous fat was defined as the difference in the area between the entire adipose tissue in the scan and visceral fat. To assess the reproducibility of these measurements, 5% of the data was re-read in a blinded fashion. The intra-class correlation coefficients of reliability ranged from 0.93 to 1.000.<SEP>Genotyping and imputation<SEP>Genomic DNA was extracted from buffy coat collected using PUREGENE DNA Purification Kit during the baseline exam. Genotyping was performed by the Center for Inherited Disease Research (CIDR) using the Illumina Human1M-Duo BeadChip system. Samples were excluded from the dataset for the reasons of sample failure, genotypic sex mismatch, and first-degree relative of an included individual based on genotype data. Genotyping was successful for 1,151,215 SNPs in 2,802 unrelated individuals (1663 Caucasians and 1139 African Americans). Imputation was done for the autosomes using the MACH software version 1.0.16. SNPs with minor allele frequency ≥1%, call rate ≥97% and HWE p≥10-6 were used for imputation. HapMap II phased haplotypes were used as reference panels. For EAs, genotypes were available on 914,263 high quality SNPs for imputation based on the HapMap CEPH reference panel (release 22, build 36). A total of 2,543,887 in EAs are available for analysis.<SEP>Statistical analysis<SEP>We performed linear regression modeling for SAT, VAT, and the VAT/SAT ratio. We observed association with the first principal components estimated using Eigenstrat; this was accounted for in our analyses.<SEP>Study-Specific Information: Age, Gene/Environment Susceptibility–Reykjavik Study<SEP>The AGES-Reykjavik study is an ongoing study of the effects of gene-environment interactions and other risk factors on disease in old age. AGES-Reykjavik is a subset of a larger population based cohort study called the Reykjavik-study. The aim of the original study was to prospectively investigate risk factors for cardiovascular disease in the Icelandic population. The original Reykjavik-cohort was established in 1967 with a random sample of 30,795 individuals born in the years 1907–1935, and residing in Reykjavik, the capital of Iceland. A total of 18,045 individuals entered the study as participants and attended examinations. The AGES-Reykjavik sample was constructed in 2002 by randomly drawing 8,030 individuals who were still alive from the original Reykjavik-cohort (n = 11,459). A total of 5,764 individuals (58% women) entered the AGES-Reykjavik study as participants. All cohort members were European Caucasians.<SEP>The AGES-Reykjavik Study was designed to investigate aging using a multifaceted comprehensive approach. Physical, physiological and questionnaire examinations were conducted in three visits for each subject including detailed medical history, physical examination, laboratory and screening tests, and questionnaires on health-related behaviors such as alcohol consumption, smoking history, and physical activity. Pertinent to this study, these measures included anthropometric measurements and computerized tomography measures of adipose depots in the abdomen and thigh. The AGES–Reykjavik study was approved by the institutional review boards of the National Institute on Aging and the Icelandic National Bioethics Committee (VSN: 00-063), and written informed consent was obtained from all participants.<SEP>Analytic sample<SEP>Of the 5,764 individuals who agreed to participate in the AGES-Reykjavik study, 5,427 individuals attended the research center for examinations while 337 received a home visit. A total of 204 individuals did not contribute CT-data at the research center. However, among those individuals with CT, only 3,664 had genotyping at the Laboratory of Neurogenetics, Intramural Research Program, NIA, Bethesda, Maryland, and 3,219 participants passed QC criteria for genotyping. Of these, 3172 had complete genotyping and complete CT data for abdominal subcutaneous and visceral adipose depots.<SEP>Image acquisition<SEP>Images for abdominal adipose depots were acquired in a single 10 mm thick trans-axial section of the abdomen at the level of the L4–L5 vertebrae using a Siemens Somatom Sensation 4 multi-detector CT scanner (Siemens Medical Solutions, Erlangen, Germany) (standard scan setting: slice thickness: 10 mm, tube voltage; 140 kilo-voltage, tube-current-time-product; 50 milli-ampere-seconds and scan time 0.361 sec). Study participants weighing more than 110 kg (kilo-grams) underwent CT with a tube current setting that was 25% higher than the standard scan setting. The images were reconstructed into a display field of view of 350 mm to include a calibration phantom (Image Analysis, Columbia, KY, USA) which was positioned under the abdomen of each subject. VAT area was estimated from all pixels in the abdominal cavity within the range of −50 to −200 Hounsfield units. Inter-observer variability based on the re-analysis of randomly selected 365 scans from the core study population by an expert observer showed an average correlation coefficient of 0.99. Intra-observer variability based on re-analysis of 45 scans by each of the four observers resulted in an average correlation coefficient of 0.99.<SEP>Imputation<SEP>As a reference panel for imputation, we used Phase II CEU HapMap individuals; we imputed genotypes to nearly 2.5 million HapMap SNPs; further details are presented in Table S1.<SEP>Statistical analysis<SEP>We performed linear regression modeling for SAT, VAT, and the VAT/SAT ratio.",
                  "Results<SEP>The characteristics of the study sample are presented in Table 1 and Table S1. Overall, 5560 women and 4997 men were available for analysis. Study participants ranged in age from their thirties to their mid seventies, and BMI ranged from 26.6 kg/m2 to 28.8 kg/m2.<SEP>Heritability Analyses<SEP>In order to document a genetic or familial component of directly-imaged CT adipose tissue traits, we previously performed heritability analyses of VAT (h2 36%) and SAT (h2 57%). For the present analysis, we additionally calculated the heritability of VAT and SAT in the Family Heart Study and the VAT/SAT ratio in both the Family Heart Study and the Framingham Heart Study. In the Family Heart Study, the heritability of VAT and SAT was 36% and 44%, respectively. The heritability of the VAT/SAT ratio was 43%; after adjustment for either BMI or VAT, the heritability was not materially different (44% and 47%, respectively). Similarly, we found that the heritability of the VAT/SAT ratio was 55% (p<0.0001) in the Framingham Heart Study, which was essentially unchanged after adjustment for BMI (h2 55%) or VAT (64%).<SEP>Stage 1 Discovery Results<SEP>After confirming a heritable component to directly imaged CT adipose tissue traits, we proceeded with GWAS. To assess for occult population stratification, we examined q-q plots for all traits (SAT, VAT, VAT-adjusted-for-BMI, and VAT/SAT ratio in the overall sample and in women and men separately), which can be found in Figure S1. All lambda values were <1.08, with little evidence to suggest unaccounted for population stratification. Manhattan plots for these traits can be found in Figure S2, with p-values<5.0*10E08 for the VAT/SAT ratio overall and VAT in women.<SEP>Our most significant finding was for rs11118316 at LYPLAL1 for the VAT/SAT ratio (Table S2). This SNP is in low to moderate LD with rs4846567 (r2 = 0.285, D′ 0.935), which was previously identified in the GIANT consortium in association with WHR-adjusted-for-BMI. It is notable that in the GIANT consortium, rs4846567 was only associated with WHR in women (p = 4.9*10E-33) but not men (p = 0.36), whereas rs11118316 was associated with both women (p = 4.5*10E-6) and men (p = 8.3*10E-5) in the present analysis. We further note that for rs4846567, the p-value is 4.4*10E-04 in women but p = 0.05 in men. Thus, it is possible that we have identified a slightly different locus with varying sex differences. Our next genome-wide significant finding was for rs1659258 at chromosome 2 for VAT in women (p = 1.58*10E-08; Table S2 and Figure 1). Imputation scores for this SNP ranged from 0.98 to 1.0. This region has not previously been identified in association with adiposity phenotypes. Table 2 shows the results for rs1659258 across the abdominal adiposity traits in our meta-analysis. We observed no association in men (p = 0.75) for VAT. In women, we observed a nominally-significant signal for SAT in women (p = 0.002), and for VAT-adjusted-for-BMI (p = 6.9*10E-05). Figure 2 shows the standardized beta coefficients in women as compared to men for all traits in each contributing study.<SEP>Given our finding of a stronger association in women as compared to men at rs1659258 for VAT, we formally tested a sex interaction term, and found a significant effect in the magnitude of the association between our lead SNP and VAT in women as compared to men (pinteraction = 0.0002).<SEP>All additional independent SNPs with p-values<9.9*10E-06 can be found in Table S2.<SEP>Stage 2 Validation<SEP>We focused on rs1659258 as a new locus for independent validation in the non-overlapping cohorts that are part of the GIANT consortium data. Results are displayed in Table 2, with evidence for replication at this locus in women (p = 0.006) but not men (p = 0.24) for BMI. Similar results were observed for waist circumference.<SEP>Metabolic Traits Association Studies with rs1659258<SEP>After confirming the association of rs1659258 with measures of adiposity in a sex-specific manner, we next sought to perform association studies with correlated metabolic traits. Because VAT is associated with glycemic and lipid traits, we requested sex-specific associations of rs1659258 in the Global Lipids Genetics Consortium for lipid traits and in the MAGIC consortium for traits related to glucose metabolism. Overall, we observed a nominal and direction-consistent association (i.e., same allele associated with higher VAT and lower HDL) with HDL (p = 0.019, n = 98,263), but no robust associations for LDL, triglycerides, or total cholesterol (all p>0.06), which is surprising given the phenotypic correlations between VAT and triglycerides. In sex-specific analyses, we observed no association for this SNP with HDL, triglycerides, or total cholesterol for either women (all p>0.09, personal communication) or men (all p>0.73, personal communication). In MAGIC, we observed a direction-consistent and borderline statistically significant result for this SNP in women (p = 0.048, n = 43,754) but not men (p = 0.39, n = 36,514) for fasting glucose (personal communication, MAGIC consortium).<SEP>Test of Age Interaction with rs1659258<SEP>Because fat distribution can vary by age, we performed a formal age interaction test between rs1659258 and VAT in women only. Stratifying above and below age 60 years, we did not observe a significant sex interaction (p = 0.79).<SEP>Interrogation of Published Loci for WHR and BMI Loci<SEP>In order to understand whether directly-imaged adipose traits are associated with previously-identified loci in a GWAS for fat distribution using anthropometric traits, we performed an association analysis of the 14 previously-published loci for WHR-adjusted-for-BMI for all 4 of our traits in the overall sample and by sex (Table 3) in the results from the GIANT consortium. Overall, we observed associations (defined as p<0.01) for 7/14 of the previously reported loci, most of which were direction-consistent (with the exception of loci associated with SAT, consistent with the fact that SAT and hip are highly correlated traits [r = 0.92 among 2656 individuals from the Family Heart Study]). We observed only one association with VAT at the NISCH-STAB1 locus.<SEP>We additionally performed an association analysis of all of the 32 published BMI loci identified via the GIANT consortium (Table 4). Overall, we observed associations (defined as p<0.01) at 7/32 loci, the majority of which overlapped for SAT and were all direction-consistent with the exception of VAT/SAT ratio at NEGR1. We observed very few associations with VAT, with the exception of FTO and NRXN3.<SEP>eQTL Results<SEP>In order to help identify the potential causal gene in the region surrounding our lead SNP, we first reviewed publically available databases, but did not identify any associations of rs1659258 with eQTLs (see methods for additional details). However, most of these databases consist of women and men in combined analyses; given that our GWAS finding was in women alone, we performed sex-specific eQTL analyses of rs1659258 in up to 848 patients (mean BMI 50.5 kg/m2) who underwent Roux-en-Y gastric bypass surgery who also underwent subcutaneous and visceral fat biopsy. We performed eQTL testing in the 1 MB region surrounding rs1659258 (n = 31 genes). Using a corrected p-value threshold of p<0.05, only THNSL2 expression in subcutaneous fat in women was associated with our lead SNP (p = 0.03) but not men (p = 0.96). We did not observe association with expression in VAT in either women or men.",
                  "Discussion<SEP>Principal Findings<SEP>We have uncovered a new locus for VAT at THNSL2 in women that reveals a striking sexual dimorphism, where we observed significance only in women, but not men. We also observed genome-wide significance for rs11118316 at LYPLAL1 for the VAT/SAT ratio; this region was previously identified in a GWAS for WHR in the GIANT consortium, although our lead SNP is only in moderate LD with the SNP identified by GIANT. Finally, we performed targeted SNP evaluations of 14 SNPs previously identified in association with central fat distribution and identified nominal associations at 7 loci.<SEP>In the Context of the Current Literature<SEP>Prior genome-wide association studies have focused on using anthropometric measures to define body fat distribution. The GIANT consortium identified 14 loci associated with WHR adjusted for BMI, the majority of which demonstrated stronger associations in women as compared to men. More recently, a SNP at the IRS1 locus was identified in association with body percent fat in men, but not women. We showed that the VAT/SAT ratio in men was associated with this same IRS1 SNP. Taken together, these prior data suggest that additional loci may exist in association with fat distribution above and beyond those associated with generalized adiposity. In the present study, we identified a new locus in association with VAT in women, highlighting the utility of more precise phenotyping of abdominal fat distribution.<SEP>We also extend the observations of a marked sexual dimorphism that have been made using primarily anthropometric measurements to CT-imaged fat depots. It is well-known that in women as compared with men, VAT levels are relatively lower, and SAT levels are higher. This gender difference underscores the need to consider women and men separately in assessing the genetic architecture of fat distribution. We also note that our most significant SNP at the LYPLAL1 locus was associated with the VAT/SAT ratio in both women and men as compared to only WHR in GIANT, while the lead SNP identified in GIANT demonstrated some evidence for heterogeneity by sex with the VAT/SAT ratio. Taken together, these findings highlight the potential differences in directly-imaged fat distribution traits as compared to anthropometric data.<SEP>An important consideration is whether our finding of rs1659258 at THNSL2 in association with VAT in women represents a variant that is specific to central adiposity. Indeed, our strongest result is derived from VAT itself. However, we observed associations with SAT in women, and our validation data suggests this SNP is also associated with BMI in women. Once we adjusted the VAT trait for BMI, our SNP did not completely lose its statistical significance (p = 6.9*10E-05), suggesting that there is some specificity of our finding to fat deposition in the central abdominal compartment. This result is in contrast to our prior results for a SNP in NRXN3 in association with waist circumference: upon adjustment for BMI, statistical significance was attenuated. Finally, our eQTL data reveals nominal expression in subcutaneous but not visceral adipose tissue.<SEP>Potential Underlying Biology<SEP>Our lead SNP for VAT in women, rs1659258, is located in an intergenic region upstream from THNSL2 and FABP1. However, rs1659258 is not in linkage disequilibrium (LD) with any coding SNPs within 88,200-88,700 kb. In addition, the correlations between rs1659258 with coding SNPs in FOXI3, C2orf51, THNSL2, EIF2AK3, FABP1, and SMYD1 genes are low (r2<0.15). Finally, there is no evidence that 2p11-p12, where rs1659258 is located, has been previously implicated in association with copy number variation in adipose-related human disease. Nonetheless, we explored the potential biology in this region. Fatty acid binding protein is produced in the liver and is involved with fatty acid metabolism. Free fatty acid flux has previously been shown to be more strongly associated with visceral as compared to subcutaneous fat. In addition, women have been shown to have a faster rate of non-oxidative free fatty acid disposal as compared to men, but without concomitant worsened metabolic risk factor profiles. While FABP1 represents an exciting potential candidate gene, rs1659258 resides in a neighboring linkage disequilibrium block that does not contain any genes. THNSL2 is just downstream of FABP1 and our lead SNP demonstrates nominal gene expression to THNSL2, which is part of the threonine synthase family. A recent analysis of RNA expression in 225 healthy Pima Indian skeletal muscle biopsies showed a bimodal (ie two discrete clusters) expression of THNSL2, thought to occur due to cis-acting polymorphisms.<SEP>It is particularly notable that we uncovered a genome-wide significant finding with rs11118316 at LYPLAL1 for the VAT/SAT ratio overall. The VAT/SAT ratio is a metric of propensity to store visceral as compared to subcutaneous fat and has been shown to be associated with cardiometabolic risk. We previously observed associations of a SNP in this gene in association with WHR-adjusted-for-BMI in the GIANT consortium, and LYPLAL1 has also been associated with cardiometabolic traits and fatty liver. LYPLAL1 encodes the lysophospholipase-like protein 1 and has been shown to be upregulated in the visceral and subcutaneous fat of obese subjects.<SEP>There are several potential explanations that could potentially account for our observed genetic association, particularly the marked sexual dimorphism. It has been previously shown that the familial contribution to fat distribution phenotypes is stronger in women as compared to men. This concept is further strengthened by findings in mice suggesting that gonadal hormones are important in the sex-specific expression of genes related to metabolic traits. In addition, known gender differences in fat distribution may also in part contribute to our findings, as women have been shown to have more subcutaneous fat but less visceral fat compared to men. The relatively smaller amount of visceral fat in women may have increased our ability to detect a genetic signal. Finally, we have consistently observed stronger associations among women as compared to men with respect to metabolic risk factors in association with VAT. While the reasons for this have not been fully elucidated, the stronger associations in women as compared to men is similar to what we have observed in the present analysis and in a prior GWAS of fat distribution phenotypes.<SEP>Strengths and Limitations<SEP>Strengths of our study include directly measured visceral and subcutaneous fat using CT imaging. Phenotyping using imaging is superior to typical anthropometric measures in the ability to partition the subcutaneous from visceral fat depots. Limitations include sample size: because of the limited number of studies with these imaging measurements and genome-wide association data, our discovery sample size was modest compared with other contemporary analyses. However, we note that performing sex-specific analyses actually enabled us to uncover a new locus, highlighting how heterogeneity can mask findings even when sample sizes are larger. Finally, the mean BMI in our gastric bypass eQTL dataset was substantially higher than the mean BMI in our discovery GWAS, which may affect generalizability of the eQTL data.<SEP>Conclusions<SEP>We have uncovered new loci for body fat distribution phenotypes, highlighting that loci exist for fat distribution that are largely independent of overall adiposity. More refined phenotypes for body composition and fat distribution can detect new loci not uncovered in large-scale GWAS of anthropometric traits.",
                  "35953",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "42",
                  "29375034",
                  "Background<SEP>Diabetes has become a worldwide epidemic, with an estimated more than 387 million people with diabetes in 2014. However, the etiology of diabetes is poorly understood. Identifying potentially causal risk factors could help guide prevention efforts.<SEP>Compelling evidence showed that excessive caloric intake is a major driving force behind escalating obesity and type 2 diabetes (T2DM) epidemics, Importantly, diet quality of fats, carbohydrates and protein, and unhealthy lifestyles have been also implicated in the rapid rise of T2DM. For example, cigarette smoking is a well-established risk factor for T2DM, and smoking cessation leads to higher short-term risk. In addition, both short and long sleep duration are associated with a significantly increased risk of T2DM. However, it remains uncertain whether these relationshipsare causal since it is difficult to fully protect observational studies from bias due to reverse causation or confounding.<SEP>In the absence of high-quality RCT data, the principles of Mendelian randomization (MR) can be applied to strengthen or refute the causality of diet/lifestyle in T2DM etiology. This approachis based on the principle that genetic variants are randomly allocated at meiosis, and consequently are independent of many factors that bias observational studies, such as confounding and reverse causation. Therefore, MR is conceptually similar to an RCT.<SEP>Recent advances in GWAS have substantially improved our understanding of genetic roles in diet/lifestyle such as macronutrients, cigarette smoking, smoking initiation, smoking cessation, sleep, allowing us to use MR to estimate their causal effects. Therefore, we undertook an MR analysis using summary statistics from DIAGRAM for T2DM and GAGIC consortium for glycemic traits, respectively to estimate the causal effects of diet/lifestyle factors on risk of T2DM and glycemic traits.",
                  "Methods<SEP>SNP selection and data sources<SEP>The design of our study had three key components: 1) the identification of genetic variants to serve as instrumental variable (IV) for diet/lifestyle; 2) the acquisition of summary datafor the genetic variants from the DIAGRAM and MAGIC consortium; and 3) the estimate of causal effect of diet/lifestyleon T2DM and glycemic traits.<SEP>We searched the GWAS catalog to identify single nucleotide polymorphisms (SNPs) associated with diet/lifestyle. We included all SNPs identified from original study reports for diet/lifestyle such as macronutrients, cigarette smoking, smoking initiation, smoking cessation, sleep in the GWAS catalog as potential IV. Effect estimates of these diet/lifestyle-associated SNPs on the risk of T2DM and glycemic traits such as HOMA-IR, HOMA-B, fasting insulin, fasting glucose, Hba1c, 2hGLU were assessed using the summary statistics from the DIAGRAM consortium and MAGIC consortium, respectively. Summary statistics from these consortia can be downloaded at the following links: DIAGRAM consortium, http://diagram-consortium.org/; MAGIC consortium, https://www.magicinvestigators.org/.Cohorts participating in the DIAGRAM consortium and MAGIC consortium received ethics approval from local institutional review boards and informed consent from all participants.<SEP>We harmonized the data from the summary statistics of consortium. First, we identified variants that do not share the same allele pair between datasets, and then correct this if possible. Second, we identified variants with unmatched effect and other alleles and then ‘flip’ their effect estimates and effect allele frequencies in only one of the datasets.<SEP>MR estimates and statistical analyses<SEP>Inference of causality in the estimated etiological associations between diet/lifestyle and risk of T2DM and glycemic traits is based on three MR assumptions. The selected genetic variants are valid instrumental variables if these three assumptions are satisfied (Fig. 1). Three assumptions are a) the genetic variant (instrumental variable) is associated with exposure (diet/lifestyle); b) the genetic variant is not associated with confounders; and c) the genetic variant is associated with outcomes (T2DM and glycemic traits) only through their effect on exposure, not through other pathways.<SEP>Here, we employed the previously described methods to examine the causal association of diet/lifestyle with T2DM and glycemic traits. We obtained the effect estimates of the selected SNPs on diet/lifestyle, and corresponding effect estimates for the selected SNPs on T2DM and glycemic traits were extracted from the DIAGRAM and MAGIC consortium. A two-sample MR approach were applied to examine causal effect. This two-sample approach has equivalent statistical power to one-sample approaches and is favorable in this setting since large GWAS consortium and are thus better powered than an MR study in a single cohort with a smaller sample size. This approach weighted the effect estimate of each SNP on T2DM and glycemic traits by its effect on diet/lifestyle. These estimates were then pooled using a fixed meta-analytic model to produce a summary measure of the effect of diet/lifestyle on T2DM and glycemic traits.<SEP>P values were two-sided and evidence of association was declared at P < 0.05. Where indicated, Bonferroni corrections were used to make allowance for multiple testing, although this is likely to be overly conservative given the non-independence of many of the outcomes tested. All analyses were performed in R version 3.1.2 and Stata release 13.1 (StataCorp, College Station, TX).",
                  "Results<SEP>SNP selection<SEP>Overall, we identified 2, 2, 1, 8, 3, 1, 1that achieved genome-wide significance for carbohydrate intake, protein intake, fat intake, smoking, smoking initiation, smoking cessation, respectively. The SNPs for carbohydrate intake, protein intake, smoking, smoking initiation, and sleep duration are LD-independent SNPs. Therefore, in total we used 15 SNPs for our MR analysis, as shown in Table 1.<SEP>Association of the selected SNPs with diet/lifestyle<SEP>Table 1 displays the SNPs that identified in GWAS for diet/lifestyle and describes their associations with carbohydrate intake, protein intake, fat intake, smoking (number), smoking initiation, smoking cessation, and sleep. Each of these SNPs explained an important proportion of the population-level variance in diet/lifestyle.<SEP>Association of the selected SNPs with T2DM and glycemic traits<SEP>We found that protein intake SNP rs1421085 at FTO was significantly associated risk of T2DM and glycemic traits (Table 2). The protein intake-increasing allele C of FTO was significant associated with higher risk of T2DM (Beta ± SE = 0.104 ± 0.014, P = 4.40 × 10− 11), higher level of HOMA-IR (Beta ± SE = 0.016 ± 0.004, P = 9.55 × 10− 5), HOMA-B (Beta ± SE = 0.008 ± 0.003, P = 0.020), Hba1c (Beta ± SE = 0.008 ± 0.004, P = 0.028), whereas lower level of fasting insulin (Beta ± SE = − 0.015 ± 0.004, P = 7.48 × 10− 5). In addition, the G allele of SNP rs3025343 at DBH forsmoking cessation was significantly associated with higher level of HOMA-IR (Beta ± SE = 0.020 ± 0.008, P = 0.010), fasting insulin (Beta ± SE = 0.016 ± 0.008, P = 0.032), and fasting glucose (Beta ± SE = 0.016 ± 0.007, P = 0.029).<SEP>MR estimates<SEP>Using MR analyses, we found that increased percentages of total energy consumption from protein was causally associated with an increased risk of T2DM (Beta ± SE = 0.806 ± 0.260, P = 0.002) (Table 3). TheI2estimate for heterogeneity was 0% (95% CI 0% -12%). However, increased percentages of total energy consumption from protein was not associated with glycemic traits (Table 3). In addition, our results also suggested that smoking cessation was causally associated with increased levels of glycemic traits such as HOMA-IR (Beta ± SE = 0.165 ± 0.072, P = 0.021), fasting insulin (Beta ± SE = 0.132 ± 0.066, P = 0.047) and fasting glucose (Beta ± SE = 0.132 ± 0.064, P = 0.039). However, we did no observe significant association between fat intake, carbohydrate intake, sleep duration, and smoking initiation and number of cigarettes and risk of T2DM and glycemic traits (Table 3).",
                  "Discussion<SEP>Using a MR study design, we found that genetically elevated protein intake and smoking cession are causally associated with an increased risk of T2DM and higher level of insulin resistance. These results provide evidence supporting a causal role for higher protein intake and smoking cession in diabetes, and suggest that individuals at risk for diabetes to keep healthy lifestyle.<SEP>Previous research efforts have focused on macronutrient intake in relation to type 2 diabetes risk, but mainly on relative carbohydrate and fat content. Meanwhile, high-protein diets may contribute to disturbance of glucose metabolism. Previous studies addressing dietary protein and diabetes risk focused mainly on high-protein food groups, such as meat and soy. For example, red processed meat intake was related to increased diabetes risk. In addition, another study found an increased diabetes risk with higher intake of animal protein and no association with vegetable protein intake, whereas intake of legumes and soy decreased diabetes risk in Asians. However, the Nurses Health Study II did not find such an association, suggesting divergent effects of animal and vegetable protein. The contradictory findings may reflect reverse causation bias and confounding effects. Interestingly, the European Prospective Investigation into Cancer and Nutrition (EPIC)-NL study found that high total protein intake was associated with increased diabetes risk, and suggested that this relation was not explained by specific protein sources such as meat. However, the causality between protein intake and risk of diabetes is not documented. Furthermore, several trials showed beneficial effect of high protein intake on glycemic traits, whereas, high protein diet did not decrease hemoglobin A1c and fasting plasma glucose, and increase insulin sensitivity, making the conclusion difficulty. In the present MR analysis, we found that protein intake was causally associated with increased risk of diabetes and insulin resistance. Our results were generated by using the large scale GWAS summary results, which suggest the robustness of our findings. Our findings for protein intake are generally consistent with those based on prospective observational studies, which tend to report increased risk of diabetes. In addition, reliable GWAS identified genetic variants for protein intake were used as instrumental variables to infer the causality, therefore, our findings were protected from bias such as confounding and reverse causation.<SEP>Cigarette smoking is an established predictor of incident T2DM. Therefore, smoking cessation should be associated with a decrease in the risk of T2DM. However, smoking cessation is associated with substantial weight gain, which could increase diabetes risk. Several studies have found an increased diabetes risk after smoking cessation. However, residual confounding is possible even with meticulous adjustment for established diabetes risk factors in the observational studies. Interestingly, a systematic review and meta-analysis of data from randomised controlled trials of smoking cessation in adults with diabetes found that pooled results did not provide evidence of efficacy for smoking cessation interventions in people with diabetes. Therefore, the causality between smoking cessation and risk of diabetes are still unknown. Using GWAS identified genetic variants for smoking cessation as instrumental variable, our MR analysis provided robust evidence to support that smoking cessation might cause increased risk of T2DM. These findings may carry important public health implications. Smokers at risk for diabetes who quit should receive advice about avoiding weight gain and about diabetes prevention and early detection.<SEP>However, our MR did not observe causal association of other lifestyle factors such as fat and carbohydrate intakes and number of smoking cigarettes, smoking initiation, sleep duration, and morning person with risk of T2DM. These findingsare generally contradictory to those based on previous prospective studies, which tend to report increased risk for diabetes in individuals with smoking, both shorter and longer sleep. The contradictory findings may reflect confounding effects, e.g. due to cases being slightly older than controls, or reverse causation bias in the retrospective studies, whereby lifestyle changes arise as a result of disease.<SEP>Given the random distribution of genotypes in the general population with respect to lifestyle and other environmental factors, the results of the MR analysis may offer some of the best evidence to assess the causal role of protein and smoking cession in T2DM etiology since the results are less likely to be biased by confounding or reverse causation than traditional observational epidemiological study designs. By employing the two-sample MR approach, we were able to increase statistical power by selecting summary statistics from the largest GWAS studies for T2DM (DIAGRAM, n = 149,821) and glycemic traits (MAGIC, n = 133,010).<SEP>Our study is subject to some limitations. First, our results assume that the samples used to define the instrumental variable for diet/lifestyle and the samples from consortium used to estimate the genetic association with disease/traits are from the same population. Second, our results mightbeconfoundedby pleiotropic pathways.35We cannot entirely rule out this possibility. Third, our study assumed a linear relationship between diet/lifestyle and T2DM and glycemic traits. In this study, we could not investigate nonlinear effects of diet/lifestyle. Fourth, we cannot exclude the possibility of sample overlap since both DIAGRAM and the MAGIC consortiums used samples from the lifestyle GWASstudy. Therefore, this might introduce bias into our results. Finally, we cannot rule out chance of violation of any of the three MR assumptions, which would potentially bias the magnitude of the estimated causal association.",
                  "14149",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "43",
                  "34290091",
                  "INTRODUCTION<SEP>The FTO locus has been reproducibly associated with body mass index (BMI) in humans across diverse ethnicities. Like the vast majority of trait-associated variants identified by genome-wide efforts, BMI-associated variants at the FTO locus map to the noncoding genome, which is still poorly characterized. In addition, genome-wide association study (GWAS) loci characteristically contain dozens of statistically significant variants in high linkage disequilibrium (LD), with regulatory variants often acting at distances up to a megabase (Mb), thus implicating multiple potential effector genes. These confounding factors pose a major challenge when translating genetic associations to mechanisms and therapeutic hypotheses. It has recently been shown that FTO intronic regions form functional connections to IRX3 and IRX5, an Mb away, suggesting that variants within FTO could also affect gene regulatory mechanisms at a long distance. The risk allele at the rs1421085 variant in the FTO locus has been connected to cellular consequences in adipocytes and to cellular phenotypes that are relevant for obesity. Furthermore, multiple variants and tissues, including both adipose and brain, have been implicated in mediating risk at the FTO locus, pointing toward multiorgan and multivariant mechanisms.<SEP>To date, it is unclear whether rs1421085—independently of correlated neighboring variants—affects an organismal or metabolic phenotype. Because gene expression is context specific, variants often depend on specific in vivo stimuli, such as nutritional challenges, to exert their individual effects. Model organisms have the potential to allow investigation of the spatiotemporal functional effects of variants and their target genes in relevant cell types and tissues in the whole-body context, linking genetic variation with in vivo metabolic physiology. Although the potential for modeling human regulatory variation in mice strongly depends on the conservation of functional elements between human and mouse, there are many good examples of modeling human enhancers and their function in the mouse, such as the recent mechanistic analysis of the distal enhancer at a human autoimmune and allergic disease risk locus (11q13.5), involved in immune regulatory T (Treg) cell function in the suppression of colitis. Noncoding regions of the FTO-IRX locus show a notable degree of evolutionary sequence conservation between mouse and human that places them among the top 2% of comparably sized genomic regions. Furthermore, studies comparing the human regulatory landscape to their murine ortholog have yielded new insight into mechanisms involved in diabetes and obesity, and cross-species conservation-based computational models have successfully been exploited in prioritizing functional regulatory elements that associate with metabolic diseases.<SEP>The aim of this study was to recapitulate the FTO rs1421085 regulatory circuitry at the whole organismal level using a mouse model that harbors a mutation at the murine orthologous region flanking the human rs1421085 C/T variant. Building on conservation patterns of the regulatory landscape between human and mouse, we confirm previous results reporting Irx3 and Irx5 to be target genes of the rs1421085 cis-regulatory module (CRM) in adipocyte progenitors. In addition, we also reveal a complex context-dependent effect on Irx3 and Irx5 target gene regulation, on fat mass and skin fat thickness, and on metabolic consequences under high-fat diet (HFD) conditions. For the latter, we test the effects of the rs1421085 circuitry on metabolic murine phenotypes under HFD conditions using untargeted, ultrahigh-resolution metabolome analysis and show an rs1421085-specific disruption of steroidal metabolic homeostasis. We recover an rs1421085 steroid metabotype (metabolic phenotype) in human blood plasma samples following an oral glucose tolerance test (OGTT) in 379 healthy, normal-weight individuals.",
                  "MATERIALS AND METHODS<SEP>Metabolome analysis of mouse tissue<SEP>DI-FT-ICR MS measurements of mouse tissue<SEP>sc-iWAT and vc-gWAT were excised at 20 weeks of age and pretreated for the metabotype analysis. Mouse adipose tissue samples (50 mg) were placed in tubes containing ceramic beads (Tissue homogenizing CKMix, 2 ml, P000918-LYSKO-A, Precellys, Bertin Corp., Rockville, MD). One milliliter of cold methanol (−20°C) [LiChrosolv, hypergrade for liquid chromatography–mass spectrometry (LC-MS); Merck KGaA, Darmstadt, Germany] was added to the tissue samples followed by homogenization using the Precellys Evolution Homogenizer (Bertin Corp., Rockville, MD; 5800 rpm, 2 × 15 s, 30-s pause time). The samples were centrifuged (11,292 rcf, 10 min at 4°C), and the supernatants were collected into Eppendorf tubes and stored at −80°C. The extracts, diluted in methanol-water (50:50, v/v) by a factor of 1000, were analyzed in positive electrospray ionization (ESI) mode via DI-FT-ICR MS, using a Bruker SolariX instrument equipped with a 12-T magnet (Bruker Daltonik GmbH, Bremen, Germany). The instrument was externally calibrated by injecting a solution (10 μg/ml) of arginine and observing corresponding peaks with m/z values equal to 175.11895 [M+H]+, 349.23062 [2M+H]+, 523.34230 [3M+H]+, and 697.45397 [4M+H]+. In the experiment, the flow rate of infusion was set to 120 μl/hour. Four hundred scans, each corresponding to 4 mega words (MWs) in the interval from 147.4 to 1000.0 m/z, were acquired and averaged. The ion accumulation time was set to 0.7 s, and the time of flight to the detector was set to 1 ms. The voltages of capillary and spray shields were set to 3800 and −500 V, respectively. The flow rate of nebulizer gas was kept at 2.2 bar, and the drying gas flow rate was set to 4 liters/min (at a temperature of 180°C).<SEP>Ultrahigh resolution (UHR)–MS data preprocessing<SEP>FT-ICR MS spectra were collected and exported using the vendor software’s ftms Control 2.2.0 (Bruker Daltonik, GmbH, Bremen, Germany) and Data Analysis 4.4 (Bruker Daltonik, GmbH, Bremen, Germany), respectively. m/z peaks were picked at Signal-to-noise ratio (S/N) ≥ 4, and a minimum intensity of 1.5 × 106 counts was set. Gibbs peaks were removed by Data Analysis automatically. All spectra were exported as tab-separated asc files and loaded into the Kernel Calibrator. A mouse calibrant list was generated manually after first precalibrating representative mouse tissue spectra against fatty acids, followed by isotopic pattern simulation of the most abundant MS features. Molecular formulas that could be confirmed by means of isotopic fine structure were listed as appropriate. Mouse spectra were then subjected to the Kernel Calibrator. Kernel Calibrator visualizes a mass spectrum’s innate mass error distribution as a density surface, and a calibration function can be optimized under visual inspection until a representative calibration curve is built for each spectrum. The SD of mass error after calibration is generally found to be <100 ppb (parts per billion) at m/z < 500 and <300 ppb at m/z < 1000. Spectra were further cleaned by removing m/z features whose mass defect could not be realized within the acquired m/z range given Carbon (C), Hydrogen (H), Nitrogen (N), Oxygen (O), Phosphorus (P), Sulfur (S), Sodium (Na), and z = 1.<SEP>Matrix generation<SEP>To perform peak alignment and generate data matrices for vc-gWAT and sc-iWAT cases, all the corresponding mass spectra were subjected to an in-house written matrix generator algorithm that was set to align peaks within a 1-ppm error window.<SEP>Feature selection<SEP>To investigate how the presence or absence of the rs1421085-DEL82 genotype affects WAT (vc-gWAT or sc-iWAT) and to select the most relevant features, we used PLS-DA. Given 7797 features for the vc-gWAT tissue type and 2338 features for the sc-iWAT tissue type, distinct gender-specific PLS-DA models on DEL82 effect were computed. The computation involved an iterative procedure described by van Velzen et al., where each step represents the selection of a smaller feature subset and building a model on this subset. Before building initial PLS-DA models, the matrices were subjected to a filtering procedure that removed features whose intensity values were present in less than 10% of the included samples. Each missing value was substituted by a minimal intensity value of the corresponding feature multiplied by 0.9. We also applied a logarithmic log2 transformation that approximates the distribution of the data to be normal. To assess the quality of PLS-DA models obtained at each step of the aforementioned workflow of van Velzen et al., we used a sevenfold cross-validation scheme to estimate the values of Q2 (the cross-validated coefficient of determination R2). The PLS-DA models were decided to be final if the further model refinement via selecting a smaller subset of features did not improve the values R2 and Q2 (R2 = 0.94 and Q2 = 0.79 for males from vc-gWAT samples, R2 = 0.84 and Q2 = 0.32 for females from vc-gWAT samples, R2 = 0.98 and Q2 = 0.88 for males from sc-iWAT samples, and R2 = 0.97 and Q2 = 0.82 for females from sc-iWAT samples). All the calculations were done in MATLAB 2018b.<SEP>MDiN computation and network visualization<SEP>MDiN reconstruction<SEP>MDiNs are reconstructed as published in. Briefly, MDiN reconstruction requires a list of mass spectrometric m/z values (metabolic features) as nodes and a list of mass differences for the assignment of edges that connect nodes. Mass differences are constructed from known biochemical reactions by subtracting the neutral monoisotopic mass of a substrate from the neutral monoisotopic mass of the corresponding reaction’s product. The mass difference itself is then indicative of the incorporation of the reaction cosubstrate. Expressed in equations: given reaction 1: m(A) + m(B) → m(C), Δmass = m(B) = m(C) − m(A). A pair of nodes xi and xj is connected by an edge corresponding to m(B) if |(m(xj) − m(xi)) − m(B)| < ppm(xi,xj), where ppm(xi,xj) is the mean of the mass measurement errors at m(xj) and m(xi). If the mass difference between the masses of xj and xi is equal to the mass difference m(b) up to a specified error, xi and xj receive an edge corresponding to reaction 1. Each assignment of a mass difference edge to a pair of nodes places a reaction hypothesis on the nodes. Herein, the mass differences from were augmented with steroid conjugations (condensations and additions thereof), yielding a set of 487 mass differences. Constructed MDiNs are used for both molecular formula assignment and data visualization.<SEP>MDiN visualization<SEP>Visualizations of MDiNs were performed using Gephi 0.9.2. Networks too dense for visual interpretation are pruned by omitting edges that are not of statistical relevance. MDEA selects mass differences associated with node sets of interest, such as biomarker candidates or compound classes. Herein, MDiNs were visualized using a set of mass differences that is optimal for clustering chemical compound classes as published by Moritz et al. and Schmitt-Kopplin et al.. The respective mass differences occur within compound classes at a much higher frequency than they do between compound classes. These mass differences separate compound classes optimally, enabling visual detection of biomarker compound class associations on an MDiN.<SEP>ORA of HMDB compound classes in mouse tissue<SEP>Given a number of instances with nonunique properties and a subset from this set, the ORA discloses whether the observed subset, encompassing certain properties, can be obtained by a random sampling rather than some defined process. We used this notion to understand the general behavior of different compound classes, i.e., whether some compound classes have higher or lower prevalence in mice with the rs1421085-DEL82 genotype. For each of the considered adipocyte tissue types (vc-gWAT or sc-iWAT), the set of all features, used to build an initial PLS-DA model, represented the total number of instances for ORA and therewith the reference distribution of properties. The property of each instance/feature represented a compound class assigned by the HMDB database as well as the sign of the associated regression coefficient. A positive sign of the regression coefficient relates to the higher abundance of the feature in the rs1421085-DEL82 mice, and the negative sign relates to the lower abundance. If no assignment was found, then a generic class, denoting the absence thereof, was given to the feature. The subset of features chosen for the final PLS-DA model represented the subset of instances whose distribution of properties was to be compared to the reference distribution introduced above. For each of the considered classes and its behavior, we used the hypergeometric distribution to estimate the probability (P value) whether the corresponding number of features (or less) can be observed in the subset if it was sampled randomly. P values for compound class enrichment were computed for the subset of features chosen for the final PLS-DA model as well as for the corresponding subsets with positive and negative regression coefficients. For further investigations, we have considered only those classes whose total number of instances in the reference distribution was higher than 40. All the calculations were done in MATLAB 2018b.<SEP>ORA of compound classes driven by mass difference analysis<SEP>For each of the considered adipocyte tissue types (vc-gWAT or sc-iWAT), the ORA was extended to show which compound classes prevail for each mass difference type selected from the MDEA. Every source and target node was given the corresponding regression coefficient obtained from the initial PLS-DA model. Because source and target nodes correspond to specific features, they were assigned to the respective compound classes obtained from the HMDB database. If no assignment was present, then a generic class, denoting the absence thereof, was given to the source or target node. For each of the considered mass difference types, the total set of instances referred to all the corresponding mass differences/edges with constituent source and target nodes. In turn, we considered two subsets, where, for each mass difference type, only those mass differences were considered that were characterized (i) by the negative regression coefficients for the source nodes and the positive regression coefficients for the target nodes or (ii) by the positive regression coefficients for the source nodes and the negative regression coefficients for the target nodes. Because a mass difference approximates a real biochemical reaction, these scenarios reflect the transformations happening during this process. Afterward, depending on the use of either the source or target nodes, we performed the ORA to calculate the probability (P value) to observe a specific number of features belonging to a certain compound class within the given subset. These calculations were done using the hypergeometric distribution. All the calculations were done in MATLAB 2018b and R using the ggplot2 package.<SEP>KORA F4 cohort: Population characteristics<SEP>Study samples<SEP>Subjects were recruited from the population-based KORA F4 cohort in the region of Augsburg, southern Germany. The population sample encompassed 50 male individuals of Caucasian origin, aged between 35 and 67 years. The subjects were selected on the basis of the FTO rs1421085 SNP genotype: homozygous controls (TT; n = 29), heterozygous risk allele carriers (CT; n = 4), and homozygous risk allele carriers (CC; n = 17). All individuals underwent OGTT, and the blood samples collected at baseline (fasting state, t0), 60 min (t1), and 120 min (t2) after glucose load were used for the metabolome analysis. Last, a set of 48 subjects sampled at three time points and 2 individuals sampled at two time points was analyzed via DI-FT-ICR MS. Anthropometric and clinical data as well as blood samples were available from a standard 2-hour OGTT (75 g of glucose; Dextro O.G.T., Roche Diagnostics, Mannheim, Germany). Details of the oral metabolic challenge including anthropometric and biochemical measurements, sample collection, and storage are described by Wahl et al.. The study was conducted in accordance with the Declaration of Helsinki. All the participants gave written informed consent, and the Ethics Committee of the Bavarian Medical Association (Bayerische Landesärztekammer) approved the study.<SEP>Genotyping<SEP>KORA F4 samples were genotyped for the SNP rs1421085 with a concordance rate of >99.5% using the MassARRAY system with iPLEX chemistry (Sequenom, USA), as previously described by Holzapfel et al.. All subjects′ FTO rs1421085 genotype was validated by Sanger sequencing. The following primers were used: polymerase chain reaction (PCR) primers: 5′-accatcaaagaggctgttgt-3′ (rs1421085_PCRfwd) and 5′-gcacccattaactcgtcatt-3′ (rs1421085_PCRrev); sequencing primers: 5′-tgtctctaagcccaacaaac-3′ (rs1421085_Seqfwd) and 5′-attgagccatccatcaggtt-3′ (rs1421085_Seqrev).<SEP>The PCR was performed with around 50 ng of input genomic DNA in a thermocycler (Biometra, Jena, Germany) as follows: 12 min at 95°C, 50 cycles of 20 s at 95°C, 40 s at 56°C, and 90 s at 72°C, and finally 2 min at 72°C before cooling.<SEP>Replication cohort: Population characteristics<SEP>Study samples<SEP>For the validation of the findings from the KORA F4 population, we investigated 326 volunteers (males = 100; females = 226) aged between 22 and 75 years who attended a diabetes screening between September 2013 and May 2016 at the Institute for Nutritional Medicine, University Hospital Klinikum rechts der Isar of the Technical University of Munich (prediabetes clinical cohort). The screening addressed individuals with positive family history of type 2 diabetes (first- and/or second-degree relatives), overweight (BMI ≥ 25 kg/m2) or obesity (BMI ≥ 30 kg/m2), already elevated blood glucose levels in previous examinations, former gestational diabetes, or an increased German Diabetes Risk Score. The screening was performed in accordance with the Declaration of Helsinki. All the participants gave written informed consent, and the Ethics Committee at the Faculty of Medicine of the Technical University of Munich approved the study. The subjects were selected on the basis of the FTO rs1421085 SNP genotype: homozygous controls (TT; n = 83), heterozygous risk allele carriers (CT; n = 162), and homozygous risk allele carriers (CC; n = 81).<SEP>Oral metabolic challenge<SEP>All individuals underwent a standard 2-hour OGTT (75 g of glucose; Dextro O.G.T., Roche Diagnostics, Mannheim, Germany) starting between 8:00 a.m. and 9:00 a.m. after a 10- to 12-hour overnight fast. Blood samples for standard laboratory measurements and DNA isolation were taken at baseline (fasting values, t0), 30 min, 60 min (t1), 90 min, and 120 min (t2) after glucose load. Samples taken at t0, t1, and t2 were kept for metabolome analyses.<SEP>Anthropometric and clinical parameters<SEP>The standard examination program comprised anthropometric measurements of height (seca 213 portable stadiometer, seca GmbH, Hamburg, Germany), weight, and body composition (Tanita BC-418 MA segmental body composition analyzer, Sindelfingen, Germany), waist and hip circumference, as well as measurement of blood pressure following established protocols.<SEP>Laboratory measurements<SEP>Measurement of standard laboratory/biochemical parameters was performed by SYNLAB clinical laboratory service (Munich, Germany). Hemoglobin A1C (HBA1c), Glutamic Oxaloacetic Transaminase (GOT), Glutamic-Pyruvate Transaminase (GPT), Gamma-Glutamyl Transpeptidase (GGT), cholesterol, Triglyceride (TG), High Density Lipoprotein (HDL), Low Density Lipoprotein (LDL), High-sensitivity C-reactive protein (hsCRP), and Thyroid Stimulating Hormone (TSH) were determined in fasting blood samples, whereas glucose values were determined at every sampling point. Determination of insulin, proinsulin, C-peptide, and nonesterified free fatty acid levels at all sampling points was taken over by IDM (Tübingen Germany).<SEP>Genotyping<SEP>Double-stranded DNA was isolated from EDTA full blood or buffy coat using the DNeasy Blood & Tissue Kit with DNeasy Mini spin-columns (QIAGEN, Hilden, Germany) following the manufacturer’s protocol. The genotyping was partially performed by GENEWIZ UK (Takeley, Essex, UK) and at the Broad Institute of Massachusetts Institute of Technology and Harvard (Cambridge, MA, USA). Genotyping revealed n = 243 carriers of the FTO rs1421085 risk allele. In accordance with the discovery study, this group combined/unifies n = 81 homozygous (CC) and n = 162 heterozygous (CT) carriers. Eighty-three subjects homozygous for the nonrisk allele (TT, n = 83) were considered as controls.<SEP>Metabolome analysis of human blood plasma<SEP>DI-FT-ICR MS measurements of human blood plasma<SEP>Plasma samples from a total of 376 subjects (see above for details and ethical approvals), from two independent cohorts, were analyzed via DI-FT-ICR MS. Before analyses, the metabolites (from 50 μl of blood plasma) were extracted by C18 solid-phase extraction (SPE) technology, using Omix C18 100 μl tips (Varian) and following the protocol described by Forcisi et al.. The extracts, diluted in methanol by a factor of 50, were analyzed in positive ESI mode via DI-FT-ICR MS, using a Bruker SolariX instrument equipped with a 12-T magnet (Bruker Daltonik GmbH, Bremen, Germany). The instrument was externally calibrated by injecting a solution of arginine (10 μg/ml) and observing corresponding peaks with m/z values equal to 175.11895 [M+H]+, 349.23062 [2M+H]+, 523.34230 [3M+H]+, and 697.45397 [4M+H]+. In the experiment, the flow rate of infusion was set to 120 μl/hour. Four hundred scans, each corresponding to 4 MWs in the interval from 147.4 to 1000.0 m/z, were acquired and averaged. The time of accumulation ion was set to 0.7 s, and the time of flight to the detector was set to 1 ms. The voltages of capillary and spray shields were set to 3800 and −500 V, respectively. The flow rate of nebulizer gas was kept at 2.2 bar, and the drying gas flow rate was set to 4 liters/min (at a temperature of 180°C).<SEP>UHR-MS data preprocessing<SEP>FT-ICR MS spectra were collected and exported using the vendor software ftms Control 2.2.0 (Bruker Daltonik GmbH, Bremen, Germany) and Data Analysis 4.4 (Bruker Daltonik GmbH, Bremen, Germany), respectively. m/z peaks were picked at S/N ≥ 4, and a minimum intensity of 1.5 × 106 counts was set. Gibbs peaks were removed by Data Analysis automatically. All spectra were exported as tab-separated asc files and loaded into the Kernel Calibrator. m/z calibration was then performed against an in-house repository of human blood plasma metabolites for both cohorts. Human blood spectra were then subjected to the Kernel Calibrator. Kernel Calibrator visualizes a mass spectrum’s innate mass error distribution as a density surface, and a calibration function can be optimized under visual inspection until a representative calibration curve is built for each spectrum. The SD of mass error after calibration is generally found to be <100 ppb at m/z < 500 and <300 ppb at m/z < 1000. Spectra were further cleaned by removing m/z features whose mass defect could not be realized within the acquired m/z range given CHNOPS, Na, and z = 1.<SEP>Matrix generation<SEP>To perform peak alignment and generate a data matrix for the KORA F4 cohort, all mass spectra were subjected to an in-house written matrix generator algorithm that was set to align peaks within a 1 ppm error window. Heavy isotopologues were detected and eliminated by searching for m/z features at mass spacings corresponding to those of typical heavy isotopes within an error of 3 ppm in conjunction to a correlation coefficient, with their corresponding mono-isotopic peak being r ≥ 0.9.<SEP>In the replication cohort, to target the steroid-like compounds, the mass spectra were filtered to match the [M+H]+ and [M+Na]+ adducts of any compound from the HMDB database, belonging to the class of “steroid and steroid derivatives.” The search window was set to 1 ppm. The filtered mass spectra were subjected to an in-house written matrix generator algorithm that was set to align peaks within a 1-ppm error window.<SEP>MDiN-based molecular formula annotation and network visualization<SEP>MDiN-based molecular formula assignment was performed following Tziotis et al.. An MDiN was reconstructed on the data matrix from the KORA F4 cohort, using 0.1-ppm edge error (the error for finding a positively matching m/z peak). Molecular formulas to be used as starting points were assigned manually following isotopic fine structure confirmation as described above. Molecular formulas were propagated from assigned starters to nonassigned m/z features by propagating the molecular formula difference contained in Mass differences (MDs) to nonassigned neighbors. Newly assigned formulas were tested against the seven golden rules and a modified version of the senior rule. Each newly assigned molecular formula was checked for consistency with all other adjacent m/z features of the valid formula. Formulas that did not pass these tests were reset to zero and allowed to be reassigned. m/z features with more than 10 unsuccessful assignment events were discarded. The mass error of the resulting molecular formula assignments was found to be on the same scale as reported above for calibration. Unassigned features were discarded. MDiNs were visualized as described in the “MDiN computation and network visualization” section describing the metabolome analysis of mouse data.<SEP>Feature selection<SEP>To investigate how the presence or absence of the FTO rs1421085 SNP genotype in the KORA F4 cohort affects the outcome of the metabolic challenge separated by time intervals as well as to select the most relevant features, we used ML-PLS-DA. The method is capable of handling longitudinal data as described by van Velzen et al.. The original matrix was divided into one containing the samples from the times t0 and t1, and another containing the samples from the times t1 and t2. Such a division was used to investigate the effect of the metabolic challenge within the corresponding time frame (further referred to as immediate OGTT response and short-term OGTT response, respectively). For feature selection and the construction of final ML-PLS-DA models, we followed the same workflow as in metabolomics analysis of mouse tissue using PLS-DA. All the calculations were done in MATLAB 2018b.<SEP>To select the most relevant steroid-like compounds, associated with the effect of FTO rs1421085 SNP genotype over OGTT challenge in the replication cohort, we targeted only those features that showed similar behavioral patterns as in the case of the KORA F4 cohort. The missing values in the matrix were substituted by a minimal intensity value of the corresponding feature multiplied by 0.9. We also applied a logarithmic log2 transformation that approximates the distribution of the data to be normal. For each gender, the immediate and short-term response matrices were constructed as the difference between the corresponding time points (t1-t0 and t2-t1, respectively). The criterion to select a feature encompassed all of the following rules: (i) The median value of a feature in the t1-t0 matrix should be higher in FTO rs1421085 SNP carriers; (ii) the median value of a feature in the t2-t1 matrix should be lower in FTO rs1421085 SNP carriers; and (iii) the P value of a two-sample Student’s t test (assuming equal variance), describing the effect of the presence of FTO rs1421085 SNP genotype on the immediate or short-term OGTT response, should be lower than 0.1.<SEP>ORA of HMDB compound classes in humans<SEP>Given a number of instances with nonunique properties and a subset from this set, the ORA discloses whether the observed subset can be obtained by a random sampling rather than some defined process, according to the distribution of the properties in the subset. We used this notion to understand the general behavior of different compound classes, i.e., they increase or decrease during the OGTT responses. The set of all features, used to build an initial ML-PLS-DA model, represented the total number of instances for ORA at each of the considered time frames. The property of each instance/feature represented a compound class assigned by the HMDB database as well as the sign of the associated regression coefficient. The latter defines the behavior of the feature: The positive sign relates to the increase, and the negative sign relates to the decrease. If no assignment was found, then a generic class, denoting the absence of thereof, was given to the feature. The subset of features chosen for the final ML-PLS-DA model represented the subset of instances whose distribution of properties was to be compared to the reference distribution introduced above. For each of the considered compound classes and its behavior, we used the hypergeometric distribution to estimate the probability (P value) whether the corresponding number of features (or less) can be observed in the subset if it was sampled randomly. Therefore, a low probability indicates that the compound class is strongly associated with its increase or decrease during the OGTT responses. P values for compound class enrichment were computed for the subset of features chosen for the final PLS-DA model as well as for the corresponding subsets with positive and negative regression coefficients. All the calculations were done in MATLAB 2018b.<SEP>Creating boxplots for selected steroids<SEP>The selected set of features corresponding to both the KORA F4 and the replication cohorts was represented as five box plots depicting the behavior of these features at specific time points (t0, t1, and t2) as well as during the challenge (t1-t0 and t2-t1). Before creating boxplots, the original matrix (characterized only by the selected set of features) was divided into three matrices containing the samples from the corresponding time point. They were logarithmically log2-transformed, whereas each missing value was imputed by a minimal intensity value of the corresponding feature multiplied by 0.9. To create matrices representing the behavior during the challenge t1-t0 and t2-t1, we performed the subtraction of the matrices of the corresponding time points. Using only the selected set of features, the boxplots were created using R and its ggplot2 package. For each pair describing the presence or absence of the FTO rs1421085 SNP genotype, we used the two-sample Student’s t test with equal variance to calculate the associated P value.<SEP>Adipocyte precursor isolation, culture, and differentiation<SEP>Mouse adipocyte mesenchymal stem cell isolation and differentiation<SEP>Mouse primary adipocytes were isolated essentially as previously described by Church et al.. Briefly, individual adipose tissue depots were excised from 6- to 10-week-old mice and placed in phosphate-buffered saline (PBS). Tissues were minced and digested in digestion buffer [sterile Hanks’ balanced salt solution (H8264), collagenase type 2 (0.8 mg/ml) (Worthington Biochemical Corporation, NJ, USA; LS004174), 3% bovine serum albumin (BSA) (with fatty acids)] and incubated in a 37°C water bath for 60 to 75 min with shaking every 10 min by hand. When digestion was complete, tubes were centrifuged for 3 min at 300g to separate floating mature adipocytes from the supravascular fraction (SVF), containing adipocyte precursors. The supernatant containing the floating adipocyte fraction was removed, and the cell pellet was resuspended in prewarmed growth medium consisting of Dulbecco’s modified Eagle’s medium (DMEM) GlutaMAX (#10569010 DMEM, high glucose, GlutaMAX Supplement, pyruvate) supplemented with 10% fetal bovine serum (FBS) (Gibco, 10082-147) and 1% penicillin-streptomycin (5000 U/ml) (Gibco, 15070063) and grown at 37°C and 5% CO2. The cell solution was subsequently filtered through a 40-μm nylon mesh and plated. Growth medium was replaced the next day, and every 2 days from then.<SEP>For differentiation experiments, attached preadipocytes were trypsin-treated using TrypLE Express Enzyme (1X) (Gibco 12605-010), inactivated with growth medium, and counted using either the Scepter 2.0 Handheld Automated Cell Counter (Millipore) with 60-μm tips (Millipore) or the Countess Automated Cell Counter (Thermo Fisher Scientific). Cells (100,000/ml) were plated and grown to confluence. Two days after confluence, preadipocytes were induced to differentiate using induction medium containing growth medium supplemented with 0.5 mM 3-isobutyl-1-methylxathine (IBMX) (Sigma-Aldrich, I5879), 1 μM dexamethasone (Sigma-Aldrich, D2915), and human insulin (5 μg/ml) (Sigma-Aldrich, I9278). On day 4 of differentiation, medium was changed to maintenance medium containing growth medium supplemented with insulin only. Medium was changed every 2 days, and differentiation was completed on days 7 to 9 after adipogenic induction.<SEP>Human subcutaneous white adipose tissue culture and differentiation<SEP>Human primary SVF cells were received from Y.-H. Tseng (Harvard Medical School, Joslin Diabetes Center, One Joslin Place, Boston, MA). The cells were previously isolated and immortalized from human subcutaneous white adipose tissue of a female subject, aged 56 with a BMI of 30.8. Culture and differentiation were performed following the protocol from the originating laboratory as described by Xue et al.. Briefly, preadipocytes were cultured in DMEM GlutaMAX (Gibco, 10569010) supplemented with 10% FBS (Gibco, 10082-147) and 1% penicillin-streptomycin (5000 U/ml) (Gibco, 15070063) at 37°C and 5% CO2. For differentiation, cells were treated with 0.25% trypsin (Gibco) and counted using an automatic cell counter, and 100,000 cells per well were seeded in a 12-well plate. Once cells reached confluency, differentiation was induced by adding freshly prepared adipogenic induction medium to cells [DMEM/high glucose supplemented with 10% FBS, 1% penicillin-streptomycin, 33 μM biotin, 0.5 μM human insulin, 17 μM pantothenate, 0.1 μM dexamethasone, 2 nM 3,3′,5-triiodo-l-thyronine (T3), 500 μM IBMX, and 30 μM indomethacin]. Induction medium was replaced with fresh induction medium every 3 days for 24 days, until fully differentiated.<SEP>Oxygen consumption measurements using seahorse Bioflux analyzer<SEP>Oxygen Consumption and Bioenergetics Profile was measured using the XF24 extracellular flux analyzer from Seahorse Bioscience. The protocol used in this assay was adapted from Gesta et al.. For this assay, preadipocytes were counted using the Countess (Thermo Fisher Scientific) and 20,000 cells per well were seeded onto seahorse 24-well plates in 100 μl of growth medium and left to adhere overnight. The next morning, 100 μl of growth medium was added to each well. Two or 3 days later (depending on the depot of origin/cell type), cells were induced to differentiate within the seahorse plate following the adipogenic differentiation protocol as described previously. Each cell type was run in 10 replicates, and four wells evenly distributed within the plate were left without cells to use for correction of temperature variation. When the cells were terminally differentiated at days 7 to 9 after adipogenic induction, the assay was performed. For isoproterenol stimulation experiments, cells were treated with 1 μM isoproterenol (Sigma-Aldrich, I6504) overnight (approximately 16 to 20 hours depending on time of assay) before the assay was performed. The evening before the assay, the seahorse XF-24 instrument cartridge was loaded with seahorse calibrant and placed in a CO2-free incubator at 37°C overnight. On the day of the assay, cells were washed in XF Assay Media, 2 mM l-glutamine, 2 mM sodium pyruvate, and 10 mM glucose (pH was measured and adjusted to pH 7.4 at 37°C). The Seahorse plate containing the differentiated adipocytes was then incubated for at least 1 hour at 37°C in a CO2-free incubator to allow CO2 to diffuse out of solution. According to the manufacturer’s protocol, the ports of the seahorse XF-24 analyzer cartridge were then loaded with the following compounds: port A: oligomycin (complex 1 inhibitor); port B: FCCP (carbonyl cyanide p-trifluoromethoxyphenylhydrazone; mitochondrial uncoupler); port C: rotenone and antimycin (inhibitors of electron transfer).<SEP>Before running the assay, the XF-24 instrument cartridge was calibrated. For total OCR measurements, the minimum OCR reading after rotenone/antimycin A treatment was subtracted from the initial untreated level following the manufacturer’s protocol. To directly measure mitochondrial thermogenesis, uncoupled respiration (proton leak) was measured by subtracting the minimum OCR level after rotenone/antimycin from the minimum level after oligomycin treatment. Oxygen concentrations were measured over time periods of 4 min with 2-min waiting and 2-min mixing. The protocol for a standard bioenergetics profile is composed of basal mitochondrial respiration, adenosine triphosphate (ATP) turnover, proton leak, and mitochondrial respiratory capacity. First, OCR in basal conditions was determined and used to calculate the basal mitochondrial respiration. After this, 2 μM oligomycin was injected from the first port to inhibit ATP synthase, resulting in an accumulation of protons in the mitochondrial intermembrane space and a reduced activity of the electron transport chain (ETC). The resulting decrease in OCR reveals the respiration driving ATP synthesis in the cells, indicating ATP turnover. Residual oxygen consumption capacity can be attributed to the proton leak maintaining a minimal ETC and nonmitochondrial respiration. Next, 2 μM of the mitochondrial uncoupler FCCP was injected, which results in an increase in OCR as the proton gradient across the IMM is dissipated and ETC resumes. This measurement reflects the maximal mitochondrial respiratory capacity. Last, 2 μM rotenone/antimycin A is injected to completely stop ETC activity and the OCR reading at this phase reflects nonmitochondrial respiration.<SEP>Gene expression<SEP>RNA isolation<SEP>Total RNA from all cells and tissues was extracted using TRIzol reagent (Thermo Fisher Scientific, #15596026) following the manufacturer’s instructions. For tissues, 1 ml of TRIzol Reagent was added to 50 to 100 mg of tissue and homogenized using ceramic beads (Precellys) in a Precellys-24 automated homogenizer (Precellys). For RNA extractions, either the RNeasy Mini Kit (Qiagen, #74106) or the Direct-zol RNA MiniPrep Plus Kit (Zymo Research, #R2072) was used. For the RNeasy Mini Kit, 200 μl of chloroform per 1 ml of TRIzol was added and tubes were vigorously shaken for 15 s, followed by 3-min incubation at room temperature. Tubes were then centrifuged at 13,000 rpm for 15 min at 4°C. Following phase separation, the colorless upper phase containing RNA was transferred to a clean tube and an equal volume of 70% EtOH was added. After thorough mixing to precipitate the RNA, the sample was transferred into a RNeasy spin column and consecutive steps were following the manufacturer’s protocol (RNeasy Mini Kit, Qiagen, #74106). For deoxyribonuclease (DNase) digestion, each sample was treated with 10 μl of DNaseI stock plus 70 μl of Buffer RDD (1500 Kunitz units) for 15 min at room temperature (RNase-Free DNase Set, Qiagen, #79254). Contaminant DNA removal was performed between Buffer RW1 washes (350 μl before and 350 μl added just after genomic DNA digestion). Alternatively, 100% EtOH was directly added to the samples in TRIzol following a Direct-zol RNA MiniPrep Plus kit protocol (Zymo Research, #R2072). Purified RNA was quantified using the Epoch Microplate Spectrophotometer (BioTek).<SEP>cDNA synthesis<SEP>Complementary DNA (cDNA) synthesis was performed using the High-Capacity cDNA Reverse Transcription Kit (Thermo Fisher Scientific, #4368814) according to the manufacturer’s protocol.<SEP>Quantitative reverse transcriptase–PCR<SEP>Gene expression was measured using quantitative real-time PCR real-time fluorescence detection. TaqMan Gene Expression Assay reagents and TaqMan FAM dye-labeled probes (Thermo Fisher Scientific) were used to set up appropriate reactions according to the manufacturer’s protocol, using PCR Master Mix (2×) (Thermo Fisher Scientific K0171) and the ABI PRISM 7500 Fast Real-Time PCR System (Applied Biosystems). To determine the most suitable housekeeping gene to use, we performed GeNORM analysis (kit from PrimerDesign) of 12 housekeeping genes in advance. Canx was determined as the most stable gene between samples and conditions and used as a housekeeping gene. The following TaqMan probes were used (gene name, TaqMan assay, and TaqMan assay ID): Canx: Mm00500330_m1, Rpgrip1l: Mm00452421_m1, Fto: Mm00488755_m1, Irx3: Mm00500463_m1, Irx5: Mm00502107_m1, Irx6: Mm01253620_m1, Ppargc1a: Mm01208835_m1, Ucp1: Mm01244861_m1, Cox8b: Mm00432648_m1, Prdm16: Mm00712556_m1, Dio2: Mm00515664_m1, Elovl3: Mm00468164_m1, Pparg: Mm00440940_m1, Cebpa: Mm00514283_s1, Fabp4: Mm00445878_m1, Plin1: Mm00558672_m1, Fasn: Mm00662319_m1, Adrb3: Mm02601819_g1, and Cox7a1: Mm00438297_g1.<SEP>Assay for Transposase-Accessible Chromatin (ATAC) sequencing<SEP>Assay for Transposase-Accessible Chromatin sequencing (ATAC-seq) was performed by adapting the protocol from Buenrostro et al.. Differentiating cells were lysed directly in the cell culture plate. Lysis buffer was added directly onto cells grown in a 12-well plate. Plates were incubated on ice for 10 min until cells were permeabilized and nuclei were released. Lysis buffer was gently pipetted up and down to wash nuclei off the well and transferred into a chilled 1.5-ml tube to create crude nuclei. Nuclei were spun down at 600g for 10 min at 4°C, nuclei pellets were then resuspended in 40 μl of Tagment DNA (TD) Buffer, nuclei were counted using trypan blue, and a volume of 50,000 nuclei was determined. Transposition reaction was performed following a previously published protocol. Tagmented DNA was PCR-amplified for 11. Quality was assessed using the DNA1000 Chip (Applied Biosystems) and run on a Bioanalyzer (Applied Biosystems). The profiles showed that all libraries had a mean fragment size of ~190 base pairs (bp) and characteristic nucleosome patterning, indicating good quality of the libraries. Libraries were sequenced at the Wellcome Trust Centre for Human Genetics in Oxford on the HiSeq4000 Illumina generating 50 Mio reads/sample, 75-bp paired end. To reduce bias due to PCR amplification of libraries, duplicate reads were removed. Sequencing reads were aligned to mm10. BWA-MEM was used for mapping.<SEP>ATAC-seq statistical evaluation<SEP>Trimmed reads were aligned to the mm10 mouse reference genome using Bowtie 2. Reads mapping to the mitochondrial genome and alternative contigs were excluded from further analysis. After removing reads with a mapping quality score < 30, peaks were filtered to exclude those with duplicates or that overlapped with a blacklisted region. We used MACS2 with “--nomodel --shift -100 --extsize 200 -q 0.05” for the detection of open chromatin. The resulting bed files were converted into coverage tracks (i.e., bigWig format) for visualization and are deposited in the Gene Expression Omnibus (GEO) database under accession number GSE169032.<SEP>3D FISH<SEP>Preadipocytes were isolated as described in the “Adipocyte precursor isolation, culture, and differentiation” section. Adipocytes were plated on Superfrost Plus slides (Thermo Fisher Scientific, REF J1800AMNZ) placed in quadriPERM dishes. Cells were either fixed 1 day after they attached (D0) or induced to differentiate for 2 days (D2). For fixation, cells were washed three times in PBS and fixed in 4% paraformaldehyde for 10 min. After three washes in PBS, cells were permeabilized in 0.5% Triton X for 10 min, followed by three washes in PBS. Slides were then air-dried for approximately 30 min and frozen at −80°C. Slides were then shipped to Iain Williamson (Wendy Bickmore group) at The MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, University of Edinburgh, where in situ hybridization was performed with the following fosmids (http://bacpacresources.org): (mm9 coordinates) rs1421085-en (WIBR1-0590 M09) chr8: 93911541-93950923, Irx3 (WIBR1-1206E19) chr8: 94307305-94341470, and Irx5 (WIBR1-1060D04) chr8: 94965335-94904882.<SEP>Briefly, between 160 and 240 ng of biotin- and digoxigenin-labeled and direct-labeled (Green 496 dUTP, Enzo Life Sciences) fosmid probes were used per slide, with 16 to 24 mg of mouse Cot1 DNA (Invitrogen) and 10 mg of salmon sperm DNA. EtOH was added, and the probe was air-dried. Hybridization mix containing deionized formamide, 20× SSC, 50% dextran sulfate, and Tween 20 was added to the probes for ~1 hour at room temperature. Slides were treated with ribonuclease (RNase) for 1 hour at 37°C in 2× SSC (100 μg/ml) followed by a series of alcohol washes (70, 90, and 100%). Denaturing was performed by heating the samples at 70°C for 5 min and then for 15 min at 80°C before a second series of alcohol washes (70% ice cold), and slides were then left to air dry. The hybridization mix containing the probes was added to the slides, and the probes were hybridized to the target DNA overnight at 37°C. Following a series of washes in 2× SSC (45°C) and 0.1× SSC (60°C), slides were blocked in blocking buffer (4× SSC, 5% Marvel) for ~5 min. The following antibody dilutions were made: rhodamine anti-dig FAB fragments (Roche) 1:20, Texas Red anti-sheep 1:100, biotinylated anti-avidin (Vector) 1:100, and streptavidin Cy5 1:10.<SEP>The slides were incubated with antibody in a moistened chamber at 37°C for 30 to 60 min in the following order with 4× SSC/0.1% Tween 20 washes in between: rhodamine anti-dig (Roche, catalog no. 11207750910, lot 35710000), Texas Red anti-sheep (Abcam, catalog no. ab6745, lot GR29419-7)/streptavidin Cy5, biotinylated anti-avidin (Vector, catalog no. BA-0300, lot ZF-0415), and streptavidin Cy5 (Amersham, catalog no. PA45001, lot 17037668). For nuclear staining, slides were treated with 1:1000 dilution of 4′,6-diamidino-2-phenylindole (DAPI) (stock 50 μg/ml) for 5 min before mounting in Vectashield under a coverslip.<SEP>Image analysis<SEP>Following procedures previously described by Williamson et al., slides were imaged using a Photometrics CoolSNAP HQ2 CCD camera and a Zeiss AxioImager A1 fluorescence microscope with a Plan Apochromat 100× 1.4 numerical aperture (NA) objective, a Nikon Intensilight Mercury–based light source (Nikon UK Ltd., Kingston-on-Thames, UK), and either Chroma #89014ET (three-color) or #89000ET (four-color) single excitation and emission filters (Chroma Technology Corp., Rockingham, VT) with the excitation and emission filters installed in prior motorized filter wheels. A piezoelectrically driven objective mount (PIFOC model P-721, Physik Instrumente GmbH & Co., Karlsruhe) was used to control movement in the z dimension. Step size for z stacks was set at 0.2 mm. Hardware control, image capture, and analysis were performed using Nikon Nis-Elements software (Nikon UK Ltd., Kingston-on-Thames, UK). Images were deconvolved using a calculated point spread function with the constrained iterative algorithm of Volocity (PerkinElmer Inc., Waltham, MA). Image analysis was carried out using the Quantitation module of Volocity (PerkinElmer Inc., Waltham, MA). For DNA FISH, only alleles with single probe signals were analyzed to eliminate the possibility of measuring sister chromatids.<SEP>Mouse methods<SEP>Animal husbandry<SEP>All animal studies were approved by the Medical Research Council Harwell Institute Animal Welfare and Ethical Review Board, and all procedures were carried out within project license restrictions (PPL 30/2642 and 30/3146) under the UK Animals (Scientific Procedures) Act 1986 and following the ARRIVE guidelines for animal research. Mice were housed according to UK Home Office welfare guidelines in a 12-hour light/dark cycle at a temperature of 21 ± 2°C and humidity of 55 ± 10%. Mice were fed ad libitum and had free access to water (25 ppm chlorine). The research diets used in this study are indicated in the figures. Mice that were used for primary preadipocyte isolation were fed SDS maintenance chow (RM3, 3.6 kcal/g). The high-fat and low-fat research diets used for the phenotyping cohorts were the following matched diets from Research Diets Inc.: D12492 and D122450J, respectively. Cohort size was estimated from previous experiments using the same tests and power calculations using G*Power 2. For the phenotyping cohorts, mice were weaned and randomized onto their respective diets at 3 weeks of age. All in vivo phenotyping was performed blind as was primary cell culture work, with the exception of the selected genotypes used in FISH and Seahorse analysis.<SEP>Genetic mouse model generation<SEP>Using CRISPR/Cas9 technology, the mouse models described in this study were generated by the Molecular & Cellular Biology group at MRC Harwell essentially as published by Mianné et al.. The aim was to create a point mutation at the mouse ortholog of human rs1421085, namely, to alter the T to a C, mimicking the human BMI risk allele, and the rs1421085-DEL82 allele was generated in the process as a result of nonhomologous end joining (NHEJ).<SEP>Design tool and choice of guide<SEP>Single-guide RNA (sgRNA) sequences were designed using http://tefor.net/ with the guide sequence most proximal to the intended change applied (5′-3′: TAATCAATACGATGCCTT, PAM sequence: AGG). The sgRNA applied is truncated with the intention of increasing specificity for the target site.<SEP>Donor oligonucleotide template<SEP>Donor sequence templates were designed with homology arms of 60 nt in size flanking the intended point mutation. The donor template sequence was ordered as an Ultramer DNA oligonucleotide from IDT (5′-3′: TGCCCTGTGGCTGCAGCTCAGAAGGCTGCCCTACAAATTCTCACTAGACGCTTAATCAATGCGATGCCTTAGGACTCGAACTGCTACCGTAAAATCAATATTACCTTTATTTTAAGTAGCA). The sequences of oligonucleotides, protospacers, and donor DNAs used within the examples presented in this study are shown as follows: reagent name 5′-3′ sequence: sgRNA_#1 sequence TAATCAATACGATGCCTTAGG, sgRNA_#2 sequence AGCGTCTAGTGAGAATTTGTAGG.<SEP>Pronuclear microinjections of zygotes<SEP>Pronuclear microinjection into C57BL/6NTac embryos was performed as described by Gardiner and Teboul. Cas9 mRNA (Tebu-Bio), sgRNAs, and single-stranded oligo DNA nucleotides (ssODN) were diluted in microinjection buffer [10 mM tris-HCl, 0.1 mM EDTA, and 100 mM NaCl (pH7.5)] and injected using a FemtoJet system (Eppendorf) at concentrations of 200 or 100 ng/μl, 100 or 50 ng/μl each, and 50 or 20 ng/μl, respectively. Injected embryos were reimplanted in CD1 pseudo-pregnant female hosts, which were allowed to litter and rear F0 progeny. DNA for genotyping was derived from mouse ear clips.<SEP>Establishment of colony<SEP>Founder animals with evidence of a NHEJ allele were crossed to WT C57BL/6NTac animals. The rs1421085-DEL82 allele was characterized at the F1 generation by PCR and Sanger sequencing using the following primers (5′-3′): Geno_FTO_rs_F1, TTCCTGAGCTAGTGTGTGTACC; Geno_FTO_rs_R1, GTCAGATTAAGGTGACGGGC.<SEP>Genotyping<SEP>Samples were genotyped with both WT and loss of allele mutant assays together with an internal control for copy counting in a qPCR TaqMan assay. In this assay, there were (i) universal probe and universal primer designed near the CRISPR deletion for both alleles, (ii) a WT-specific primer in the deletion designed for the WT allele, and (iii) a mutant-specific primer that bridges the junction designed for the CRISPR mutant allele.<SEP>The following Fto-DEL82 WT1 assay (FAM-labeled) primers were used: Fto-DEL82-Univ-Probe (5 nmol), CAGGAGCCAGATTGTCCACAGCA; Fto-DEL82-WT-R located in deletion (15 nmol), GGCTGCCCTACAAATTCTCACTAG; Fto-DEL82-Univ-F (15 nmol), CAGGCAAAAGCAAAAGGTGACATAC.<SEP>The following Fto-DEL82 MUT1 assay (FAM-labeled) primers were used: Fto-DEL82-Univ-Probe (5 nmol), CAGGAGCCAGATTGTCCACAGCA; Fto-DEL82-MUT-R (15 nmol), AGGGTCAGCACAGAGATGC; Fto-DEL82-Univ-F (15 nmol), CAGGCAAAAGCAAAAGGTGACATAC.<SEP>See the Supplementary Materials (“Fto-DEL82 qPCR genotyping strategy”) for sequence alignments showing the position of the probe and primers in these assays.<SEP>The following Dot1l internal control (VIC-labeled) primers were used: primer 1, GCCCCAGCACGACCATT; primer 2, TAGTTGGCATCCTTATGCTTCATC; probe, CCAGCTCTCAAGTCG.<SEP>Mouse phenotyping techniques<SEP>Body composition<SEP>Total body weight was measured every 2 weeks on a scale calibrated to 0.01 g. Body composition measurements were performed every 2 weeks using an echo MRI (Echo-MRI-100, EchoMRI, Texas, USA). The readouts were total fat mass (g), lean mass (g), and free water in live, nonanesthetized mice.<SEP>Intraperitoneal glucose tolerance test<SEP>Intraperitoneal glucose tolerance test was performed in the morning after an overnight (up to 16 hours) fast. Body weight of the animals was measured, and a local anesthetic was administered to the mouse tail (EMLA cream, Eutectic mixture of Local Anaesthetics Lidocaine/Prilocaine, AstraZeneca, UK). To establish the baseline glucose level at time point zero, blood glucose levels were measured using the handheld Alphatrak (Abbott) glucose monitor with Alphatrak strips (Abbott). Subsequently, 2 g of glucose per kilogram of body weight (20% glucose in 0.9% NaCl) was injected intraperitoneally and blood glucose levels were measured at 30, 60, and 120 min after injection. A fresh strip was used for each reading.<SEP>Retro-orbital bleed<SEP>Animals were euthanized by intraperitoneal injection of an overdose of anesthetic (0.2 ml of pentobarbitone) in accordance with Home Office procedures. Once fully anesthetized, a glass capillary was inserted into the anterior corner of the mouse eye to perforate the membrane of the retro-orbital sinus. Blood was collected from the capillary in Lithium-Heparin microvette tubes (CB30, Sarstedt, Nümbrecht, Germany). After cervical dislocation, animals were dissected and tissues were taken for weight, histology, and gene expression analysis.<SEP>Statistical analysis<SEP>Mouse data were analyzed using GraphPad Prism 8 Software and tests as indicated in the figure legends.<SEP>Human research participants<SEP>We have complied with all relevant ethical regulations.",
                  "RESULTS<SEP>rs1421085 is within a regulatory region conserved across human and mouse<SEP>To develop a murine model system to recapitulate and validate the rs1421085 regulatory circuitry at the organismal level, we first assessed the evolutionary conservation of chromatin structure across humans and mice at the FTO/Fto locus. In human adipocyte progenitor cells, the BMI association signal in FTO, and more specifically the regulatory variant rs1421085, is within a region decorated by enhancer-associated histone marks (H3K4me1 and H3K27ac; Fig. 1). Cross-species analysis revealed that the peaks for histone modifications associated with regulatory activity in human preadipocytes were also present in mouse preadipocytes [Fig. 1 and fig. S1 (FTO/Fto gene zoomed inset)], suggesting a functionally conserved regulatory region surrounding rs1421085.<SEP>To test whether the physical environment of proximal and distal target genes are conserved across human and mouse at the FTO locus, we examined HiC data surrounding FTO/Fto. In human and mouse embryonic stem cells (ESCs), we observed topologically associated domains (TADs) that encompassed the same five candidate target genes Rpgrip1l, Fto, Irx3, Irx5, and Irx6 in both human and mouse (Fig. 1). Last, we performed sequence-level analysis of the rs1421085 surrounding region and observed that transcription factor binding site (TFBS) motifs are well conserved at the nucleotide level, including the ARID motif, which was previously shown to be disrupted by rs1421085. Together, these data suggest that rs1421085, and more generally the human FTO risk locus, localizes within a regulatory element that is conserved at the sequence and chromatin structure level.<SEP>Deletion of the conserved rs1421085 CRM in mice affects body composition<SEP>To dissect the mechanism and physiological role of rs1421085, we used CRISPR/Cas9 genome editing to delete the rs1421085 orthologous CRM (rs1421085-DEL82) in the mouse. Specifically, rs1421085-DEL82 harbors a homozygous mutation in the orthologous region between human and mouse, deleting 2 nucleotides (nt) upstream and 79 nt downstream of rs1421085 (Fig. 2A), spanning the ARID motif. None of the other correlated variants of the FTO risk haplotype localizes within these 82 nt. To test whether rs1421085-DEL82 affects whole-body composition, we examined mutants versus controls under HFD and low-fat diet (LFD) conditions. We observed that homozygous male rs1421085-DEL82 mice on an HFD showed a subtle reduction in body weight and fat mass from 14 weeks of age (Fig. 2, B and C), which was statistically significant between 14 and 18 weeks of age for body weight (P < 0.05; Fig. 2B) and 14 and 16 weeks of age (P < 0.05; Fig. 2C) for fat mass. Lean mass was not altered between genotypes (Fig. 2D). Heterozygous rs1421085-DEL82 had an intermediate effect on body weight (fig. S2A) and fat mass gain (fig. S2B) on HFD compared to homozygous alleles, suggesting a dose-dependent effect of the rs1421085-DEL82 allele on body weight regulation and obesity development in male animals. Mice on an LFD did not present with altered body weight or fat mass (Fig. 2, B and C). The rs1421085-DEL82 mutation did not have a detectable effect on body weight or fat mass in female animals (fig. S3, A to C, E and F), and rs1421085-DEL82 had no effect on glucose tolerance in any of the groups (figs. S2, C and D, and S3D).<SEP>To further investigate the effect on obesity-related phenotypes, we examined the effect of rs1421085-DEL82 on subcutaneous dermal adipose tissue (sc-dWAT). We quantified sc-dWAT by measuring adipose layer thickness at two anatomical locations, namely, in the ventral-abdominal area and the dorsal area. Subcutaneous adipose tissue below the skin is the major site of fat deposition, accounting for approximately 50% of total body fat mass in mice. The data revealed that the subcutaneous adipose layer thickness was reduced by 35% at the ventral-abdominal site (P = 0.046) and 32% at the dorsal site (P = 0.005) in homozygous rs1421085-DEL82 relative to their homozygous controls (Fig. 2, E and F), which was in line with the reduction in body weight and overall fat mass measured by echo magnetic resonance imaging (echoMRI).<SEP>rs1421085-DEL82 is within a regulatory element for Irx3 and Irx5 in mouse adipocyte progenitors<SEP>To identify the target genes under the genetic control of the rs1421085 CRM, we examined gene expression levels of five candidate target genes contained within the TAD, namely, Rpgrip1l, Fto, Irx3, Irx5, and Irx6. Obesogenic WAT expansion is mainly regulated by hypothalamic signals in the brain and WAT-intrinsic mechanisms, and both tissues have been previously implicated in the FTO obesity risk locus. We tested target gene expression in mouse adipose tissue (mostly consisting of mature adipocytes) derived from subcutaneous inguinal WAT (sc-iWAT; a proxy for gluteofemoral adipose tissue in human), visceral perigonadal WAT (vc-gWAT; a proxy for visceral adipose tissue in human), isolated adipocyte progenitors from both these adipose depots, and the hypothalamus.<SEP>We observed a decrease of Irx3 (0.65-fold, P = 0.028) and Irx5 expression (0.71-fold, P = 0.029) in sc-iWAT–derived mouse preadipocytes homozygous for rs1421085-DEL82 (Fig. 3A). Heterozygosity for rs1421085-DEL82 had an intermediate effect in Irx3 (0.81-fold, P = 0.0336) and Irx5 expression levels (0.79-fold, P = 0.021) (fig. S4A), suggesting a dose-dependent effect of rs1421085-DEL82 on target gene expression in sc-iWAT preadipocytes.<SEP>In contrast to the effect in sc-iWAT progenitors, we observed an increase of Irx3 (1.5-fold, P = 0.026) mRNA levels in vc-gWAT–derived mouse preadipocytes (Fig. 3B). The data further revealed a lack of gene expression changes in female sc-iWAT– or vc-gWAT–derived preadipocytes at any time point (fig. S5, A and B), which was consistent with the lack of an organismal phenotype in female mutant mice (fig. S3). The expression of the Irx6 gene was below detection limits in both sc-iWAT and vc-gWAT preadipocytes. In addition, we did not observe any effect on Irx3 and Irx5 expression in the hypothalamus of male (Fig. 3C) or female animals (fig. S5C), suggesting a cell type–specific effect of rs1421085-DEL82 on gene expression. Last, we show that the rs1421085 CRM deletion did not affect Fto, Rpgrip1l, and Irx6 gene expression levels in any tissue examined (Fig. 3, A to C, and fig. S5, A to C). Expression analysis of gWAT and iWAT mature adipose depots from homozygous male rs1421085-DEL82 and control mice revealed that Irx3 and Irx5 mRNA levels were not altered (fig. S4B), indicating that target gene expression changes and regulatory activity are specific to early differentiating preadipocytes.<SEP>Together, we show that out of the five potential target genes measured in five relevant mouse cell/tissue types, only Irx3 and Irx5 were altered in preadipocytes of male rs1421085-DEL82 animals. We observed an opposite direction of target gene expression in vc-gWAT and sc-iWAT adipocyte progenitors, which was in line with fat mass alterations in opposing directions. Differences in gene expression across depots could be due to either a direct effect on regulatory activity or an indirect effect on gene expression mediated by secondary effects independent of the rs1421085 regulatory circuitry.<SEP>To examine depot-specific spatial enhancer-promoter proximity between rs1421085 CRM and the target genes Irx3 and Irx5, we performed three-dimensional (3D) fluorescence in situ hybridization (FISH) in iWAT- and gWAT-derived primary adipocyte progenitors of male wild-type (WT) and rs1421085-DEL82 animals. We performed the experiment in actively proliferating cells before confluency and before adipogenesis was induced (D0) and 1 day after induction of differentiation (D1). Fosmids that hybridize to rs1421085-en (enhancer element that overlaps rs1421085), Irx3, and Irx5 were used (Fig. 3, D and E).<SEP>In WT sc-iWAT, the Fto intron 1 rs1421085 enhancer (rs1421085-en) and Irx3 promoter probe pairs were significantly more colocalized (from 8 to 29%) and the average distance between them was significantly decreased (P = 0.0001), in cells induced for differentiation compared to proliferating cells (D1 versus D0, respectively) (Fig. 3, F and G). In contrast, in rs1421085-DEL82 cells, rs1421085-en and Irx3 colocalization frequencies were markedly decreased and interprobe distances increased at D1 compared to WT cells (P = 0.0072). These data indicate disrupted chromosome conformation upon induction of adipogenesis (Fig. 3G) that may be consistent with a decrease of Irx3 mRNA levels following rs1421085 CRM deletion in confluent sc-iWAT cells (see above, Fig. 3A). There was less probe colocalization (<5%) and the larger interprobe distances at both time points between rs1421085-en and Irx5 than was observed with Irx3, with no significant differences observed between genotypes (Fig. 3, F and G).<SEP>In vc-gWAT, probe colocalization frequencies were lower (<9%) and the interprobe distances were larger for rs1421085-en and Irx3 promoter in all samples compared to sc-iWAT (Fig. 3, F and G). Nevertheless, we could detect increased distances between rs1421085-en and Irx3 or Irx5 in rs1421085-DEL82 compared to WT at D0 (P = 0.0017, rs1421085-en/Irx3; P = 0.0065, rs1421085-en/Irx5), but not at D1 of differentiation (Fig. 3G). The functional significance of these changes in chromosome conformation is unclear, as there is increased Irx3 mRNA levels in vc-gWAT from rs1421085-DEL82 mice (Fig. 3B). It is possible that the increase in Irx3 expression in vc-gWAT could be a secondary effect rather than a direct effect of the genetic locus.<SEP>These data together provide evidence for dynamic chromatin conformation changes at the Fto/Irx3/Irx5 regulatory circuitry in sc-iWAT and vc-gWAT. In particular the strongly increased rs1421085-en and Irx3 promoter spatial colocalization occurring over D0 to D1 of differentiation is disrupted in sc-iWAT from rs1421085-DEL82 mice.<SEP>rs1421085 CRM controls mitochondrial copy number and mitochondrial function in human and mouse adipose-derived cells<SEP>The FTO obesity variant circuitry, mediated through IRX3 and IRX5, has been linked to adipocyte browning in humans and thus to mitochondrial function. Therefore, to assess the effect of rs1421085-DEL82 on mitochondrial function and activity in WAT, we excised adipose tissues from male mice on an HFD. We show that adipose tissues from rs1421085-DEL82 mice were characterized by a significantly increased number of mitochondria (fig. S6A) with 10.8-fold (P = 0.022) and 6.6-fold (P = 0.020) increase in vc-gWAT and sc-iWAT, respectively, compared to WT animals. We next examined the expression of marker genes involved in thermogenesis (Adrb3, Pgc1a, Ucp1, Cox7a, Cox7b, Prdm16, Dio2, and Elovl3) and adipogenesis (Pparg, Cebpa, Fabp4, Plin1, and Fasn). We found that rs1421085-DEL82 resulted in a significant increase of β-adrenergic receptor 3 (Adrb3) expression in gWAT (threefold; P < 0.0001) (fig. S6C) and on the browning marker Elovl3 in sc-iWAT (fig. S6D).<SEP>To assess whether gene expression changes in Adrb3 and Evolv3 in rs1421085-DEL82 vc-gWAT and sc-iWAT had functional consequences on cellular oxygen consumption rate (OCR), preadipocytes from WT and rs1421085-DEL82 male mice were isolated and differentiated to perform a mitochondrial stress test using the Seahorse Bioflux XF24 Analyser. Under basal conditions, rs1421085-DEL82 did not affect OCR profiles in vc-gWAT– or sc-iWAT–derived preadipocytes (fig. S6B). However, β-adrenergic stimulation resulted in significantly increasing OCR and proton leak (fig. S6B) in gWAT-derived rs1421085-DEL82 preadipocytes, indicative of uncoupled respiration in these cells. In contrast, β-adrenergic stimulation in preadipocytes isolated from rs1421085-DEL82 iWAT had no effect on OCR (fig. S6B). In summary, rs1421085-DEL82 had an effect on mitochondrial traits, affecting slightly different mitochondrial subtraits across the adipose depots.<SEP>Acyl steroids in adipose are under the genetic control of rs1421085-DEL82 in male mice<SEP>Obesity and fat mass–related traits are associated with a dysregulation in blood and tissue metabolite levels. To link the rs1421085 CRM to metabolic changes in adipose, we combined our rs1421085 CRM deletion model on HFD with untargeted metabolome analysis as an agnostic high-dimensional, high-resolution phenotypic readout (Fig. 4A). Untargeted metabolome analysis, which avoids prefiltering on selected compound features, allows to resolve mass to charge ratios (m/z) for a wide metabolite space, including metabolites of known and unknown chemical identities. We performed direct-infusion ion-cyclotron-resonance Fourier transform mass spectrometry (DI-FT-ICR MS), known for its ultrahigh resolution and accuracy, to systematically connect rs1421085 genetics with metabolic activity in vc-gWAT and sc-iWAT from rs1421085-DEL82 mice (Fig. 4A). We used mass difference network (MDiN) analysis to assign MS features to high-confidence molecular formulas encompassing known and unknown metabolites. Here, “known metabolic features” refers to molecular formulas that have a match in the Human Metabolome database (HMDB) database. The data revealed that 940 of 2338 differential features matched to known metabolic features in sc-iWAT and 1981 of 7797 features in vc-gWAT when comparing rs1421085-DEL82 mutants with their controls. While any assigned molecular formula can match to multiple isomeric compound structures and annotation is hence putative, molecular formulas are more specific for compound classes, particularly when distinct compound class labels are overrepresented within differential features. To understand the causal connections between fat mass phenotypes (driven by the rs1421085 CRM deletion) and variation in metabolite levels, we performed compound class overrepresentation analysis (ORA) of known metabolic features that differ between the rs1421085-DEL82 mutant and its control littermates. In sc-iWAT from male mice, we observed a decrease of steroids and their derivatives in rs1421085-DEL82 compared to WT controls (P = 0.0062) (Fig. 4B and table S1). We did not observe significant changes in sc-iWAT from females. We found an increase of glycerophospholipids (P = 0.0339) and a decrease in organooxygen compounds (P = 0.0255) in vc-gWAT from male rs1421085-DEL82 mice compared to their controls (Fig. 4C and table S2). Fatty acyls were strongly overrepresented among both increased features (P = 0.0011) and decreased features (P = 0.0053), indicating dynamic fatty acyl rearrangements. Steroid compounds did not exhibit significant enrichment within rs1421085-DEL82 vc-gWAT from males when taking all putatively annotated metabolic features into account. However, steroids on the vc-gWAT MDiN (Fig. 4C) showed two topologies: One subset was spread over a wide number of edges, indicating high chemical diversity, and another subset was located in a dense cluster, requiring more refined analyses. We observed a strong decrease of glycerolipids (P = 0.00001) in rs1421085-DEL82 female mice (table S3). To evaluate whether steroid homeostasis, which we observed to be altered in sc-iWAT, is perturbed in vc-gWAT as well (and might be missed in the ORA analysis because ORA tests for known metabolites only), we performed mass difference enrichment analysis (MDEA). MDEA screens all pairs of MS features for mass differences of known biochemical transformations. MS feature pairs connected by a biochemical transformation comprise a source node (reactant, substrate) of lower mass and a target node (product) of higher mass, irrespective of annotation status (known or unknown metabolic feature) (Fig. 4D). Last, MDEA tests whether a specific type of transformation, e.g., conjugation to fatty acyls or conjugation to steroid backbones, occurs with MS features of interest. MDEA on metabolic features allows to implicitly model the genotype effect on anabolic (i.e., substrate node is down-regulated, while product node is up-regulated) and catabolic reactions (i.e., substrate node is up-regulated, while product node is down-regulated). We observed a decrease of long fatty acyl steroid compositions in favor of elevated levels of free fatty acids in vc-gWAT from male rs1421085-DEL82 mice compared to their littermate controls. More specifically, we observed forward reactions implicating a consumption of fatty acyls, prenol lipids, and carboxylic acids related to the TCA (tricarboxylic acid cycle), carbohydrate units, and oxidative processes (Fig. 4E and table S4). Reverse reactions imply a production of prenol lipids associated with mass differences of fatty acids and steroids, and a consumption of (i) fatty acyls and glycerolipids associated with mass differences of long unsaturated fatty acids, and (ii) steroids and steroid relatives in association with fatty acyls, which is further substantiated by (iii) consumption of fatty acyls in association with steroids (Fig. 4F and table S5).<SEP>Together, these data indicate a stimulation of lipolysis, energy metabolism, and mobilization of fatty acids in adipose from rs1421085-DEL82 compared to their littermates and simultaneously an inhibition of acyl steroid production in rs1421085-DEL82 mice, pointing to a lower adipocyte lipidic esterification and lipogenesis state. These findings suggest that steroid and acyl steroid turnover in sc-iWAT and vc-gWAT is under the genetic control of rs1421085-DEL82 in male mice.<SEP>Connecting the FTO rs1421085 genotype to a steroid metabotype in humans<SEP>Individual predispositions toward diseases can be determined by establishing the effects of genetically determined variants on metabolic phenotypes (metabotypes). To validate the genotype-metabotype finding in rs1421085-DEL82 mice in humans and identify a metabolic pattern in blood plasma that differentiates rs1421085 risk versus nonrisk individuals, we performed high-resolution FT-ICR MS profiling in human blood plasma at baseline and following an OGTT. We chose a two-step discovery-replication scheme. For the discovery stage, we collected blood plasma samples from 50 healthy male individuals from the KORA F4 cohort (21 risk allele carriers and 29 nonrisk allele carriers) (table S6). We collected and profiled the samples before (Time 0h), 60 min (Time 1h) after, and 120 min (Time 2h) after an OGTT. To investigate genotype-dependent effects over the time course of the glucose tolerance test, we modeled the immediate OGTT response (which is the change in metabolic profiles between Time 0 and Time 1h) and the short-term OGTT response (which is the change between Time 1h and Time 2h) (Fig. 5A). Following FT-ICR MS, we performed MDiN analysis (Fig. 5B) and assigned high-confidence molecular formulas to 8382 metabolic blood plasma features and used multilevel partial least squares discriminant analysis (ML-PLS-DA) on both OGTT responses (fig. S7, A to D). The immediate OGTT response was built on a subset of 1045 features (211 known metabolic features), while the short-term OGTT response was built on 1031 features (281 known metabolic features). To identify compound classes that are under the genetic control of the rs1421085 genotype, we performed ORA, which revealed 11 different compound classes significantly contributing to both OGTT responses (fig. S8). These rs1421085-associated compound classes included steroids, amino acids, flavonoids, acyclic alkenes, carbonyl compounds, fatty amides, phenylethylamines, glycerolipids, disaccharides, peptides, and fatty alcohols (Fig. 5C). Notably, the class of steroids and steroid derivatives (table S7) showed the strongest effect among all for both OGTT responses and revealed an opposite behavior (fig. S8), with a notable increase of steroids and their derivatives in risk individuals compared to nonrisk individuals in the immediate OGTT response and a decrease in the short-term OGTT response (Fig. 5D).<SEP>When we looked at individual steroid feature effects, we observed a concerted strong up-regulation in the immediate OGTT response and down-regulation in the short-term OGTT response (Fig. 6A, fig. S9A, and table S7). No significant differences were observed at basal condition (Time 0), which is the standard fasted state most commonly sampled for clinical diagnosis of diseases. A substantial amount of features of high significance could be observed in the immediate OGTT response (Time 1h–Time 0), both visually and statistically. Consequently, Time 1h displayed the resulting overall disbalance of steroid compounds. This trend reverted noticeably in the short-term OGTT response (Time 2h–Time 1h). To replicate these results, we recruited 326 healthy subjects from an independent cohort, comprising 243 risk allele carriers (78 males/165 females) and 83 nonrisk allele carriers (22 males/61 females) (table S8). Following FT-ICR MS and matrix generation, for male subjects, we confirmed the strong increase in steroids in the immediate OGTT response (putatively annotated steroids and steroid derivatives, HMDB V3.6) and their decrease in the short-term OGTT response (Fig. 6B, fig. S9B, and table S9). We did not observe a substantial amount of significant rs1421085-dependent features neither at baseline (Fig. 6 and fig. S9) nor in female subjects (fig. S10 and table S10), supporting the context-dependent link of the rs1421085 genotype on the steroidal metabotype.",
                  "DISCUSSION<SEP>The FTO locus is linked to multiple obesity-related phenotypes through common variant association studies including BMI, fat mass–related traits, and basal metabolic rate, and is considered the strongest genetic risk locus for obesity in humans. We have previously implicated the noncoding variant rs1421085 as a causal variant at the FTO obesity risk locus linking rs1421085 to specific effector genes (IRX3 and IRX5) and their cellular functions (mitochondrial energy expenditure following β-adrenergic receptor activation) in human subcutaneous adipocyte progenitors. Here, we present a murine model system, rs1421085-DEL82, to study the organismal, metabolic, and cellular consequences of the highly conserved CRM flanking the genetic variant rs1421085. We validated previously reported results from human in vitro studies in the rs1421085-DEL82 mouse model and expanded on those results by providing evidence for a context-dependent role of the genetic variant in adipose depots and established an effect on obesity-related whole-body phenotypes. We then used untargeted high-resolution metabolome analysis in adipose and plasma across mice and humans to link the genetic variant to its physiological state and conclude that the rs1421085 genetic variant mediates cross-species conserved changes in steroid patterns following nutritional challenges.<SEP>Our data show that deleting the rs1421085-conserved CRM in mice results in decreased Irx3 and Irx5 gene expression in iWAT adipocyte progenitors to roughly 70% of WT levels, consistent with regulatory fine-tuning of gene expression by noncoding common genetic variation. We note that the regulatory effect was specific for adipocyte progenitor cells and not visible at the tissue level, which harbor mostly mature differentiated adipocytes. All these results are consistent with previous results from in vitro human subcutaneous adipocyte model systems in vitro, where the rs1421085 variant was edited (fig. S11). In those experiments, we established a chain of causation in human preadipocytes in vitro in which the rs1421085 C allele (associated with increased BMI) was an active enhancer resulting in increased IRX3/5, decreased mitochondrial function, and increased lipid accumulation. In contrast, the rs1421085 T allele (associated with decreased BMI) was a less active enhancer, resulting in decreased IRX3/5 expression, increased mitochondrial function, and decreased lipid accumulation. In line with these results, our rs1421085DEL82 mouse model recapitulates the effects of enhancer perturbation (removing the ARID5B TFBS motif rather than changing binding affinity). Consistent with the direction of effect that we observe in human preadipocytes in vitro, the directed perturbation of the rs1421085 flanking region in our rs1421085DEL82 mouse model reduced interaction of this enhancer with its downstream target genes Irx3 and Irx5 (see FISH data; Fig. 3, E to G). This loss of enhancer interaction was specific to induced adipocytes (day 1 of differentiation), potentially explaining the reduced Irx3 and Irx5 gene expression found in confluent iWAT preadipocytes. In contrast, rs1421085DEL82 had no effect on enhancer interactions with Irx3 and Irx5 on day 0 in the FISH experiments, which were actively proliferating cells at this stage. Thus, the effect of rs1421085DEL82 on enhancer interactions and gene expression of target genes seems to depend on the transition from proliferation to induction of the adipogenic program. Last, the rs1421085DEL82 perturbation of the variant flanking regions increased mitochondrial content and resulted in decreased HFD induced body weight gain, fat mass, and skin layer thickness, recapitulating the BMI association in humans.<SEP>While we found an expected decreasing effect of rs1421085-DEL82 on Irx3 and Irx5 gene expression in subcutaneous adipocyte progenitors from sc-iWAT, we observed an increasing effect on Irx3 gene expression in adipocyte progenitors from vc-gWAT. Two possible explanations are either that the rs1421085 CRM acts as an enhancer in subcutaneous adipocyte progenitors and a repressor in visceral adipocyte progenitors or, more likely, given the lower vc-gWAT enhancer-promoter colocalization we detect in vc-gWAT compared with iWAT, that Irx3 up-regulation in gWAT is not a direct consequence of the rs1421085 CRM but rather a secondary compensatory mechanism. While we observed an opposite direction of Irx3 gene regulation in visceral adipocyte progenitors, we note that our data indicate that cellular and steroidal metabolic changes were consistent across cellular and metabolic phenotypes in both subcutaneous and visceral adipose tissue depots, although we observed stronger genetically determined cellular and metabolic changes in subcutaneous compared to visceral adipose. It would have been interesting to examine expression in sc-dWAT; however, to maintain differentiation capacity in mouse preadipocytes, these cells have to be isolated from mice that are between 4 and 8 weeks of age, and at this age, there is very limited sc-dWAT accessible for dissection. The two depots used in this study are the most readily accessible and relatively well-defined depots, iWAT and gWAT, that are typically used to isolate primary preadipocytes.<SEP>Our rs1421085 CRM mouse model indicates a steroid metabotype under the genetic control of rs1421085 CRM in adipose. We confirmed this finding across two independent human cohorts, which provides evidence that the rs1421085 genetically determined steroid metabotype is evolutionarily conserved across mice and humans and detectable in both adipose and plasma. Steroid hormones (including but not limited to sex hormones) are small lipophilic molecules that circulate in the bloodstream controlling many physiological functions including the balance of energy, metabolism, and responses to stress and feeding. Adipocytes are steroidogenic and known to synthesize and metabolize steroid hormones. Mitochondria play important roles in cholesterol homeostasis and the subsequent biosynthesis of steroid hormones, which can, in turn, regulate mitochondrial function. The production of steroids starts at the inner mitochondrial membrane (IMM) with the conversion of cholesterol to pregnenolone by the cytochrome P450 cholesterol side-chain cleavage enzyme (CYP11A1). Increasing IRX3 has been associated with increasing UCP1, which is involved in white-to-beige adipocyte conversion along with mitochondrial fission. Mitochondrial fusion was, in turn, reported to be required for steroid biosynthesis, which links both steroid biosynthesis and thermogenesis to the same event within mitochondrial dynamics. Consistently, the rs1421085-DEL82 model showed increased IRX3 and OCR and inhibited acyl steroid production in vc-gWAT, while decreased IRX3 and no OCR response to β-adrenergic stimulation were accompanied with increasing steroid levels in sc-iWAT. These findings are consistent with the postprandial steroid pattern responses in blood plasma of human risk allele carriers observed herein.<SEP>Large-scale genetic association studies connecting genetic variants to obesity, as assessed by BMI, have implicated the brain as primary tissue of action, and the FTO haplotype has previously been suggested to affect gene expression in the brain. In this work, we provide confirmatory evidence that the rs1421085 regulatory element links to context-dependent Irx3 and Irx5 gene expression in early differentiating adipocytes and affects body weight–related phenotypes. Noncoding variants can regulate one or more genes across long genomic distances, and the same allele or multiple alleles within a haplotype might have context-specific functions or function in different cell types. Because of the possibility for pleiotropic regulation of multiple genes and multiple cell types at the FTO locus, evaluation of multiple variants, candidate genes, and cell types [i.e., adipose and central nervous system (CNS)] is critical to assess causality and to inform how those variants might converge on phenotypes that contribute to the progression of body weight gain. The FTO locus associates not only with BMI and fat mass–related traits but also with food preferences, which could be due to a direct effect of the genetic risk locus on gene expression in the CNS, an indirect effect mediated by a genetically determined metabotype as we described in our study, or a combination of both processes. The observed metabolic features have a high probability to constitute either derivatives of sex hormones or derivatives of progestagens such as cortisol. The mass differences involved with significantly perturbed acyl steroid generation in rs1421085-DEL82 vc-gWAT highlighted aldosterone/cortisol (isomeric) addition and condensation, pregnenolone derivatives, and androgens. Cholesterol, corticosterone, and chenodeoxycholic acid mass differences were not implied in fatty-acyl balance significantly. Cortisol is implied to counteract insulin action and to promote lipogenesis under chronic elevation in blood. A postprandial increase of cortisol derivatives conditioned on reduced mitochondrial activity could promote a vicious circle toward obesity and diabetes. At the same time, our data imply that androgens are affected by rs1421085, and we have found responses to rs1421085 in males only. Sex hormones were previously shown to be implicated in eating behavior, food preferences, and hypothalamic body weight regulation. The ratio of androgens to estrogens is tightly orchestrating the energy balance regulation, and a shift in this ratio leads to fat accumulation and body weight gain. Our data further indicate a sexual dimorphism of the steroid metabotype, which will need further exploration. Pronounced metabotypic patterns in blood plasma might facilitate risk assessment and planning of preventive strategies, such as paving the way for personalized diets.<SEP>Our work stresses the critical role of environment, i.e., nutrition, on the manifestation of a phenotype. We observed a resistance to weight gain conditional on HFD feeding of rs1421085-DEL82 mice, which is consistent with the notion that nutrition is considered to have the strongest environmental effect on obesity risk at the FTO locus. Our work implicates a strong nutritionally influenced steroid metabotype (metabolic phenotype) under the genetic control of rs1421085 CRM in mice on HFD and in humans following an OGTT. Notably, the link between genotype and metabotype was not significant at baseline and was characterized by a dynamic behavior as revealed by modeling different time points following the OGTT in humans. This is consistent with previous reports, as sex hormones have been shown to underlie a dynamic nutrition-dependent regulation characterized by a postprandial decrease in healthy individuals.<SEP>Modeling noncoding variation in the mouse is challenging given the approximate 90- to 100-million-year divergence between mouse and human. In contrast to coding sequences, which have retained high sequence conservation, regulatory noncoding sequence has undergone a degree of evolutionary rewiring that may confound some cross-species comparisons. However, there is evidence for significant enrichment of functional GWAS variants in species-conserved TFBSs, which may allow modeling of some functional single-nucleotide polymorphisms (SNPs) in the mouse. Furthermore, at least 40% of enhancer predictions in the mouse are also predicted to be enhancers in humans, although this is likely an underestimate given the need to match tissue, cell type, and developmental stage to make accurate cross-species comparisons; thus, there will be many opportunities to probe human enhancer function by genetic manipulation in the mouse. Consequently, for a proportion of species conserved regulatory GWAS variants in enhancers, it will likely be possible to model their physiological function in the mouse, as we demonstrate here. Engineering the rs1421085 CRM allowed us to establish organismal phenotypes under the genetic control of the variant, including HFD-induced weight gain and whole-body fat mass. The in vivo effect of the rs1421085 CRM deletion was more subtle than those reported for global and adipose-specific Irx3 or Fto knockout models, which is consistent with typically small effect sizes of common genetic variation and, in our study, a small deletion encompassing such a regulatory SNP. We observed a dose-dependent effect of the rs1421085-DEL82 genotype on body weight, fat mass gain, as well as Irx3 and Irx5 target gene expression, which is consistent with a dose-dependent effect of the FTO risk haplotype on body weight (~3 pounds per allele).<SEP>Together, our data establish an evolutionary conserved rs1421085 regulatory circuitry, which allowed us to characterize the genetic circuitry across multiple phenotypic scales: (i) adipocyte progenitor as cell types of action; (ii) Irx3 and Irx5 as effector genes; (iii) mitochondrial traits as cellular function; and (iv) diet-induced body weight gain and fat mass as organismal phenotypes. We link the rs1421085 circuitry to its genetically determined steroidal metabotype in males, which again was dependent on nutritional challenges.",
                  "89510",
                  "['OBWCCE', 'EATBTS']"
                ],
                [
                  "44",
                  "34702754",
                  "Observational epidemiology often examines the associations between exposures and health outcomes. However, such associations reported in epidemiological studies are often not reliable estimates of causal effects, and can be produced by confounding (that is, by another factor that affects both the outcome and exposure)   or by other forms of bias. For example, alcohol consumption might be related to many potential confounding factors, including smoking, an unhealthy diet, and limited exercise. In turn, ill health could be related to a reduction or cessation of alcohol consumption, introducing potential bias due to reverse causality, when interest is in studying the effect of alcohol consumption on subsequent health.  Several approaches have been developed with the aim of mitigating such biases. For example, instrumental variable methods rely on an external factor that determines the exposure of interest but is not associated with the outcome other than through its effect on the exposure.  <SEP>Over the past decade, advances in genetic technologies have enabled the identification of thousands of reproducible associations between genetic variation and relevant exposures, traits, and health outcomes. These genetic variations can be used as instrumental variables to analyse the effect of modifiable exposures on diseases through a study method termed mendelian randomisation (MR). MR studies use genetic variants robustly related to modifiable exposures to understand the influence of the exposure on various health, social, and economic outcomes. Genetic variation is essentially randomly inherited from parents to offspring at conception, and consequently, many factors that confound the association between the exposure and outcome cannot affect the genetic variants. Similarly, genetic variants are generally not influenced by the outcome and therefore, by reverse causation. MR thus provides an opportunity to study the association between exposures and outcomes while reducing potential bias from confounding and reverse causation. <SEP>These features make genetic variants suitable candidates as instrumental variables, which can help estimate the causal effects of modifiable exposures on outcomes. For example, the rs1229984 variant in the alcohol dehydrogenase 1B gene (ADH1B) has been used as an instrument to investigate the causal role of alcohol in cardiovascular disease. Given these advantages, MR studies have increased in popularity and have begun to inform understanding of disease causation. MR is not limited to studies using genetic variants to generate instrumental variable estimates (box 1, table 1); however, these studies dominate the literature. A glossary of terms commonly used in MR is given in table 2. Additional terms and explanations can be found in a comprehensive open access MR dictionary. <SEP>Scope of mendelian randomisation (MR) and the STROBE-MR checklist<SEP>While MR generally uses genetic variation as the instrumental variable, MR is not limited to such studies. Indeed, the term “mendelian randomisation” was introduced in 1991 for investigations of bone marrow transplantation in the treatment of childhood malignancies.  The basic notion was that if a child had an HLA compatible sibling, that child was more likely to receive a bone marrow transplant than a child with no compatible sibling. Analysing outcomes according to whether the child has such a sibling (optimally taking the number of siblings into account) is analogous to an intention-to-treat analysis in a randomised clinical trial.  Having an HLA compatible sibling (as a matter of chance) could also serve as a genetic instrument for bone marrow transplantation, and so might be used to infer effects of transplantation on cancer outcomes. This approach has continued to be used.   Initially, MR was defined as the use of germline genetic variation to strengthen causal inference for the influence of modifiable exposures on risk of disease or other outcomes. This wider definition includes, for example, studies of gene-by-covariate interaction (often with environment as the covariate), for which the interaction cannot be viewed as an instrument for the exposure of interest.  Other study designs, such as twin studies, also use the basic principles of mendelian genetics and so can be considered a form of MR. One such example used a male co-twin as an indicator of (on average) higher antenatal testosterone to appraise the effect of testosterone on neurodevelopmental traits. MR studies range from a simple test of an association between single nucleotide polymorphisms and outcome, which can provide evidence as to whether an exposure affects a disease, to a specific effect estimate from an instrumental variable analysis.<SEP>The STROBE-MR (strengthening the reporting of observational studies in epidemiology using mendelian randomisation) guidelines are aimed at the (currently) large majority of MR studies that are implemented within an instrumental variable framework. For MR studies that do not use an instrument for the exposure (such as those of gene-by-environment interaction) or MR studies that use genetic variants in an instrumental variable framework but do not report instrumental variable estimates (such as those of sibling compatibility for transplantation), some items of STROBE-MR will not apply, but the checklist still provides useful guidance. Table 1 gives an overview of study designs addressed and not addressed by STROBE-MR.<SEP>Summary points<SEP>In observational epidemiology, mendelian randomisation (MR) studies provide an opportunity to study the causal association between an exposure and an outcome while reducing the risk of certain biases<SEP>Little consensus exists around the reporting of MR studies, and the quality of reporting of these studies has been inconsistent; many MR study reports do not state or examine the various assumptions of MR and report insufficient details on the data sources<SEP>STROBE-MR (strengthening the reporting of observational studies in epidemiology using mendelian randomisation), a checklist of 20 reporting items, has been developed for the communication of MR studies<SEP>This article explains the rationale of these checklist items and provides examples of transparent reporting<SEP>MR study authors, reviewers, and journal editors are encouraged to use STROBE-MR to improve the reporting of these studies<SEP>Strengthening the reporting of MR studies<SEP>Despite the growth in MR applications and methods and the increasing relevance of MR findings, little consensus exists around the reporting of MR studies. As a result, the quality of reporting of these studies has been inconsistent. Empirical evidence   indicates that many reports of MR studies do not clearly state or examine the various assumptions of MR methods and report insufficient details on the data sources, which makes the quality and reliability of the results difficult to evaluate.<SEP>The STROBE (strengthening the reporting of observational studies in epidemiology) guidelines for observational research  were developed for the three main study designs in epidemiology (cohort, case-control, and cross sectional studies). Some STROBE items are either too general or do not apply to MR studies, while other items relevant to MR studies are missing. To improve reporting MR studies, we developed a separate checklist of items motivated by the STROBE guidelines but explicitly focused on the MR study design, resulting in the STROBE-MR statement (strengthening the reporting of observational studies in epidemiology using mendelian randomisation; table 3). Similar to the STROBE checklist, the STROBE-MR items relate to the title, abstract, introduction, methods, results, and discussion sections of articles. <SEP>Development, scope, and intended use of STROBE-MR<SEP>Described in detail elsewhere, we established this initiative in 2018, following guidance for the development of medical research reporting guidelines. We invited a group of experts, ranging from MR methodologists and authors of previous reporting guidelines to frequent MR study design users and scientific journal editors to participate in a workshop. The group met for a face-to-face meeting in Bristol, UK, over two days in May 2019 to discuss the empirical evidence on reporting quality of MR studies and draft the checklist items. The draft checklist was published as a preprint in July 2019, and debated on the preprint platform, social media, and a dedicated session at the 4th International Mendelian Randomisation Conference. We revised the checklist in the light of the comments received and produced an article presenting the STROBE-MR statement. <SEP>The STROBE-MR reporting guidelines are meant to apply to studies that use properties of germline genetic variants to strengthen causal inference regarding possible effects of potentially modifiable exposures on outcomes. The two principal types of MR studies are one sample MR and two sample MR. In a one sample MR study, the associations between the genetic variant and exposure and between the genetic variant and outcome are both measured in the same sample. In a two sample MR study, these two associations are measured in separate samples. MR studies can also use either individual level or summary level data to derive or apply the weights for each single nucleotide polymorphism (SNP). Two sample MR studies are most commonly conducted with summary level data, where the weights are derived from the first sample, which are then applied in the second sample to estimate the gene-outcome association. Summary sample weights for the association of genetic variants with an exposure can also be used in an individual level data analysis of the association of these variants with the outcome. Table 1 (in box 1) outlines the study designs covered and those not covered by the STROBE-MR guidelines.<SEP>Purpose of this article<SEP>This explanation and elaboration (E&E) document is intended to complement the STROBE-MR statement. The format follows that of previous reporting guidelines, such as the STROBE E&E document; it aims to provide readers with a detailed explanation supporting each of the 20 items in the checklist and examples of transparent reporting. Examples of quality reporting for each checklist item have been identified from published MR studies.<SEP>This document should be considered as a reference for authors to understand better what is meant by each item in the accompanying checklist. The examples do not necessarily represent the ideal statement for each checklist item. Rather, they highlight the intended issue meant to be covered in each item in the checklist. The boxes and tables in this article contain more theoretical background pertaining to MR study designs and complement recommendations on reporting. Additional guidance on performing MR studies can be found elsewhere. <SEP>Some examples were edited by removing citations and sections not related to the reported item. Items are divided into sections: title and abstract (item 1), introduction (items 2-3), methods (items 4-9), results (items 10-13), discussion (items 14-17), and other information (items 18-20; table 3). Some items have subparts that relate to the same topic (eg, item 10d only relates to a two sample MR study design). Additional examples are provided in web appendix 2. We advise authors to address all items in the checklist, even if some information is reported in their supplementary materials because of space restriction.<SEP>Title and abstract (item 1) <SEP>Indicate MR as the study’s design in the title and/or the abstract if that is a main purpose of the study.<SEP>Title<SEP>Examples<SEP>“BMI as a Modifiable Risk Factor for Type 2 Diabetes: Refining and Understanding Causal Estimates Using Mendelian Randomization.” <SEP>“Genome Wide Analyses of >200,000 Individuals Identify 58 Loci for Chronic Inflammation and Highlight Pathways that Link Inflammation and Complex Disorders.” <SEP>Explanation<SEP>When MR has played a crucial role in the study design, the term “mendelian randomisation” should be included in the title. In some situations, MR is used as a follow-on analytical technique, when the primary analysis is not MR. In this case, there might be no need to directly include MR in the title but retain focus on the manuscript’s main objectives.<SEP>Abstract<SEP>Example<SEP>“Importance: Human genetic studies have indicated that plasma lipoprotein(a) (Lp[a]) is causally associated with the risk of coronary heart disease (CHD), but randomized trials of several therapies that reduce Lp(a) levels by 25% to 35% have not provided any evidence that lowering Lp(a) level reduces CHD risk.<SEP>“Objective: To estimate the magnitude of the change in plasma Lp(a) levels needed to have the same evidence of an association with CHD risk as a 38.67-mg/dL (ie, 1-mmol/L) change in low-density lipoprotein cholesterol (LDL-C) level, a change that has been shown to produce a clinically meaningful reduction in the risk of CHD.<SEP>“Design, Setting, and Participants: A Mendelian randomisation analysis was conducted using individual participant data from 5 studies and with external validation using summarised data from 48 studies. Population-based prospective cohort and case-control studies featured 20 793 individuals with CHD and 27 540 controls with individual participant data, whereas summarised data included 62 240 patients with CHD and 127 299 controls. Data were analysed from November 2016 to March 2018.<SEP>“Exposures: Genetic LPA score and plasma Lp(a) mass concentration.<SEP>“Main Outcomes and Measures: Coronary heart disease.<SEP>“Results: Of the included study participants, 53% were men, all were of white European ancestry, and the mean age was 57.5 years. The association of genetically predicted Lp(a) with CHD risk was linearly proportional to the absolute change in Lp(a) concentration. A 10-mg/dL lower genetically predicted Lp(a) concentration was associated with a 5.8% lower CHD risk (odds ratio [OR], 0.942; 95% CI, 0.933-0.951; P=3×10−37), whereas a 10-mg/dL lower genetically predicted LDL-C level estimated using an LDL-C genetic score was associated with a 14.5% lower CHD risk (OR, 0.855; 95% CI, 0.818-0.893; P=2×10−12). Thus, a 101.5-mg/dL change (95% CI, 71.0-137.0) in Lp(a) concentration had the same association with CHD risk as a 38.67-mg/dL change in LDL-C level. The association of genetically predicted Lp(a) concentration with CHD risk appeared to be independent of changes in LDL-C level owing to genetic variants that mimic the relationship of statins, PCSK9 inhibitors, and ezetimibe with CHD risk.<SEP>“Conclusions and Relevance: The clinical benefit of lowering Lp(a) is likely to be proportional to the absolute reduction in Lp(a) concentration. Large absolute reductions in Lp(a) of approximately 100 mg/dL may be required to produce a clinically meaningful reduction in the risk of CHD similar in magnitude to what can be achieved by lowering LDL-C level by 38.67 mg/dL (ie, 1 mmol/L).” (Further examples are available in web appendix 2.)<SEP>Explanation<SEP>The abstract should provide an informative and balanced summary of what was done and what was found. This summary should be presented alongside critical issues in study design, including sources of data, exposures or outcomes, individual versus summary data and would (if possible) include the term “mendelian randomisation” to make the article discoverable as such. Results should be presented in a fully transparent manner and include both point estimates and their error (that is, not only P values) for the range of approaches applied. The word “causal” should be used carefully, because MR only provides estimates intended to inform our understanding of causal associations under specific assumptions. The abstract should be sufficiently detailed to act as a standalone part of the manuscript. When permitted by the journal, structured abstracts can provide clarity and help assure that all relevant information is included. Web appendix 2 includes additional examples of abstracts for one sample, two sample, and embedded MR studies.<SEP>Introduction<SEP>Background (item 2)<SEP>Explain the scientific background and rationale for the reported study. What is the exposure? Is causality between exposure and outcome plausible? Justify why MR is a helpful method to answer the study question.<SEP>Examples<SEP>“Epidemiologic studies have reported an increased risk of multiple sclerosis (MS) with earlier age at puberty, particularly among women. However, others failed to replicate this finding. Pubertal timing has complex interactions with weight status, whereby higher childhood adiposity leads to earlier puberty, which in turn is associated with higher adult body mass index (BMI). Because evidence supports a role for increased BMI in MS pathogenesis, at least part of the observed link between pubertal timing and MS might be explained by BMI. Some of the limitations faced by observational studies can be mitigated through instrumental variable methods, in which a variable is used as a proxy for an exposure to explore the effect of that exposure on an outcome. In Mendelian randomisation (MR), genetic variants are used as instrumental variables to test for a causal association between a risk factor and an outcome.” <SEP>“Nonlinear relationships exist between risk factors and outcomes across biomedical research. Methods have been developed that help to estimate non-linear relationships between exposures such as BMI and cardiovascular disease . . . We estimated the localized average causal effects of BMI on each risk factor within quintiles of the distribution of BMI after the genetic component is subtracted (the IV [instrumental variable]-free BMI) and performed heterogeneity and trend tests on these values.” <SEP>Explanation<SEP>While some authors have used MR to test the effect of exposures on many different outcomes without prior hypotheses, most MR studies are designed to assess a specific hypothesis that has arisen from previous studies. When using a specific hypothesis, the rationale for assessing the current hypothesis should be described, including the a priori expectation of the effect size. MR can be used to test causal null hypotheses or estimate point, period, or lifetime effects. The role of MR in assessing the study hypothesis should be delineated to orient readers to what specific gap in the literature can be addressed by applying MR methods to the study hypothesis.<SEP>Objectives (item 3)<SEP>State specific objectives clearly, including prespecified causal hypotheses (if any). State that MR is a method that, under specific assumptions, intends to estimate causal effects.<SEP>Example<SEP>“Objective: To evaluate the potential causal association between genetic variants related to elevated serum calcium levels and risk of coronary artery disease (CAD) and myocardial infarction using Mendelian randomization.” <SEP>Explanation<SEP>Authors should clearly state that the study aims to estimate a causal effect of the specified exposure on the specified outcome. This section should define the key exposure(s) and outcome(s) of interest to orient readers, and state the overall study objectives.",
                  "Methods<SEP>Study design and data sources (item 4)<SEP>Present key elements of the study design early in the article. Consider including a table listing sources of data for all phases of the study.<SEP>Example<SEP>“Details of the contributing GWAS consortiums are listed in table 1 [see table 4]. The studies were selected for investigating traits related to cardiovascular or metabolic health, having the largest sample sizes, and consisting of the most similar populations while minimising sample overlap. Percentage sample overlap is presented in supplementary table S1. Subjective wellbeing was measured using any items relating to happiness or positive affect and overall life satisfaction. GWAS of each component were meta-analysed to capture subjective wellbeing. For further information on the phenotype definitions and GWAS methods for all traits, see supplementary table S2. All phenotype scores were z scored apart from blood pressure.” <SEP>Explanation<SEP>As in STROBE, presenting critical elements of study design early in the article allows readers to orient themselves on the study basics. Authors should clarify whether the MR study used individual level participant data or SNP level summary data, and whether it uses a one sample or two sample MR design. In a two sample MR study, one stage can use summary data and another stage can use individual level data. Some MR studies draw on multiple sources of data (eg, different sources for the ascertainment of the association between the genetic variant and exposure, and for the association between the genetic variant and outcome). Furthermore, sources of data could be from meta-analyses of multiple samples. The general design and data sources should therefore be made clear.<SEP>We recommend a table to provide clear documentation of the sources of genetic-variant level information for the MR study (see table 4). For example, the genetic variants used to estimate the exposure could have been ascertained in one study, but the effect size (or weight) of these genetic variants on the exposure taken from a separate study. If so, we recommend reporting both sources of information. The table should be expanded as required. For example, if different MR studies with different outcomes are added to the study, then authors should add additional columns to the table. If additional exposures are studied, then additional rows can be added.<SEP>If data were extracted from pre-existing studies, describe how the data were obtained. If data are publicly available, provide a hyperlink to the data source, where possible. If using summary level data, ensure that all of these details are traceable and allow for a qualitative assessment of data sources’ heterogeneity. For each data source contributing to the analysis, describe the following factors in items 4a to 4e.<SEP>Setting (item 4a)<SEP>Describe the study design and the underlying population, if possible. Describe the setting, locations, and relevant dates, including periods of recruitment, exposure, follow-up, and data collection, when available.<SEP>Examples<SEP>“This study comprised a meta-analysis of directly genotyped and imputed SNPs from 21 cohorts totalling 42,024 individuals (Table 1). An expanded description of the participating studies is provided in the Text S2.” <SEP>“A total of 23 cohorts with genome wide genotyping and fracture data were recruited globally through the GEnetic Factors for OSteoporosis consortium (GEFOS; http://www.gefos.org/). These cohorts were predominantly of European descent and from Europe (n=13), North America (n=8), Australia (n=1), and east Asia (n=1; tables S1A and S2A), and included 20 439 fracture cases and 78 843 controls.” <SEP>Explanation<SEP>Readers need information on the population(s) studied, setting, and locations to assess the context and generalisability of the study results. Exposures such as environmental factors and therapies can change over time. Also, study methods could evolve over time. Knowing when a study took place and over what period participants were recruited and followed up puts the study in historical context, which is essential for interpreting results. Where such information has been described in previous publications, unambiguous reference to these is likely to be sufficient. Providing a description of the ancestry of the participants will help understand potential sources of heterogeneity and generalisability of results. If summary level data from existing studies are used, ensure that details are traceable to allow for a qualitative assessment of any heterogeneity of settings across data sources.<SEP>Participants (item 4b)<SEP>Give the eligibility criteria, and the sources and methods of selection of participants. Report the sample size, and whether any power or sample size calculations were carried out before the main analysis.<SEP>Example <SEP>“The UK Biobank recruited more than 500 000 people aged 37-73 years (99.5% were between 40 and 69 years) from across the country in 2006-10. Participants provided a range of information via questionnaires and interviews (such as demographics, health status, and lifestyle); anthropometric measurements, blood pressure readings, and blood, urine and saliva samples were taken for future analysis. This has been described in more detail elsewhere. We used 120 286 participants of white British descent from the initial UK Biobank dataset, of whom 119 669 had valid genetic data and both BMI and height measures available. We did not include other ethnic groups, because individually they were underpowered.” (Further examples are available in web appendix 2.)<SEP>Explanation<SEP>Detailed descriptions of the study participants and sampling frame help readers understand the applicability of the results. Authors should provide all the eligibility criteria, participant sources and methods for selecting participants, and methods of follow-up where applicable. Method of recruitment into the study should likewise be described. Where such information has been described in previous publications, unambiguous reference to these is likely to be sufficient. If summary level data from existing studies are used, ensure that details are traceable to allow for a qualitative assessment of participants’ heterogeneity across data sources.<SEP>In case-control studies, the choice of cases and controls is crucial to interpreting the results, and the method of their selection has major implications for study validity. In general, controls should reflect the population from which the cases arose.<SEP>If sample size or power calculations were conducted before the main analysis, this should also be reported either with the study design or the statistical methods. When planning a study or interpreting the analysis results, information about the sample size is essential. Power calculations provide information about sample sizes needed to obtain appropriate power for desired precision of the effect estimate,  and if they are performed, they should be performed before the study was conducted. Statistical power is best ascertained by examination of the 95% confidence intervals of the estimates, since they require fewer assumptions than pre-experiment power calculations.<SEP>Genetic variants (item 4c)<SEP>Describe measurement, quality control and selection of genetic variants.<SEP>Examples<SEP>“Genotyping was conducted using the Affymetrix UK Biobank Array. Autosomal analysis was restricted to up to 13,977,204 high-quality Haplotype Reference Consortium imputed variants with a MAF [minor allele frequency] >0.05%, minor allele count >5, info score >0.3, genotype hard call rate >0.95, and Hardy–Weinberg P>1×10−6.” <SEP>“Genetic markers for various obesity-related risk factors comprised SNPs that were associated with the risk factor of interest (P<5×10−8) based on study participants with European ancestry. Correlated SNPs were excluded based on measures of linkage disequilibrium (LD) R2 <0.1 . . . SNPs with ambiguous strand codification (A/T or C/G) were replaced by SNPs in genetic linkage (R2 >0.8) using the proxy snps R package (European populations) (R Project) or were removed from the analyses if the minor allele frequency was higher than 0.4.” <SEP>Explanation<SEP>Providing information on the ascertainment of genotypes and their quality control will enable readers to assess the quality of the genetic variants used in the study. For two sample MR, this information will often require referring to supplementary material presented in previously published articles.<SEP>The methods section should provide a clear explanation of the selection and inclusion of specific genetic variants in the analysis. This information would include a description of genetic variants allocated to the exposure of interest and in the case of reverse MR, the genetic variants allocated to the outcome. For each variant, the rsID or base position and chromosome should be disclosed, along with clear reasoning for the variant choice and the reference panel used. The reasoning could include the evidence of association with the exposure or outcome of interest or the characteristics that qualify the specific variant to be used, such as linkage disequilibrium in the case of proxy use. The inclusion of variants in high linkage disequilibrium might not contribute additional information in estimating the causal effect, and could even lead to biased estimates of standard errors if this correlation structure is not accounted for. Authors should define the threshold used to select independent variants (eg, r2), the reference panel, and the population under investigation. However, there are cases in which variants (although in linkage disequilibrium) from a specific gene region with biological relevance regarding the exposure of interest can also be included. In this situation, authors should describe the biological pathways in which these variants are implicated, the r2 threshold for inclusion, and which method was used to model the correlation structure.<SEP>Studies should also provide estimates of the quality control parameters for the SNPs used in analyses. This information includes the info score (metric of imputation quality), call rate (percentage of individuals with called alleles at a particular SNP), and P value from the Hardy-Weinberg equilibrium test (which could indicate imputation or genotyping problems, population stratification, or non-random mating).<SEP>Further information is required to chart the management of genetic variants and harmonisation of datasets in two sample MR analysis. This information includes the conditions used to identify proxy variants in the absence of the same variant being available in both datasets (eg, linkage disequilibrium threshold), the presence or absence and handling of strand alignment, and orientation of effect and non-effect alleles. Other aspects, such as temporal stability of association, population specificity, or biological plausability, could help understand the selected genetic variants’ validity to be used as instrumental variables.<SEP>Assessment and diagnostic criteria for diseases (item 4d)<SEP>For each exposure, outcome, and other relevant variables, describe methods of assessment and diagnostic criteria for diseases.<SEP>Example—continuous exposure or outcome<SEP>“Outcomes of the study were WHR [waist-to-hip ratio] (stages 1 and 2b), hip and waist circumference (stage 2a), compartmental body fat masses (stage 3) . . . WHR was defined as the ratio of the circumference of the waist to that of the hip, both of which were estimated in centimeters using a Seca 200-cm tape measure . . . Compartmental fat masses were measured in grams by DEXA, a whole-body, low-intensity x-ray scan that precisely quantifies fat mass in different body regions . . . using a Lunar Prodigy advanced fan beam scanner (GE Healthcare). Participants were scanned by trained operators using standard imaging and positioning protocols. All images were manually processed by one trained researcher, who corrected DEXA demarcations according to a standardised procedure.” (Further examples are available in web appendix 2.)<SEP>Explanation<SEP>This section provides details on the choice and definition of key exposures, outcomes, and confounders used in the analyses. Where several outcomes or hypothesis-free approaches are used, this information should be clearly indicated, together with any method that accounts for multiple testing. This section will ideally include definitions used in each study (for meta-analyses of different studies) or provide a brief summary with a clear reference if this has previously been described for the study sample. Readers can then consider the case definition’s sensitivity and specificity, and assess the relevance for their question or the generalisability to their population of interest.<SEP>Ethical approval and informed consent (item 4e)<SEP>Provide details of ethics committee approval and participant informed consent, if relevant.<SEP>Example<SEP>“Informed consent was obtained from all participants, and study protocols were approved by the local, regional, or institutional ethics committees.” <SEP>Explanation <SEP>All investigators must ensure that the planning, conduct, and reporting of human research are in accordance with the Helsinki Declaration as revised in 2013. Authors need to provide information on the approval from the responsible ethics committee and acquisition of the informed consent. This information should also be made available if the data were obtained from publicly available sources or previously conducted studies. Authors should make sure that their study falls into the scope of the original ethics committee approval and does not violate the original agreement.<SEP>Assumptions (item 5)<SEP>Explicitly state the three core instrumental variable assumptions for the main analysis (relevance, independence, and exclusion restriction) as well as assumptions for any additional or sensitivity analysis.<SEP>Examples<SEP>“As in any Mendelian randomisation analysis, several assumptions were made, including that the genetic instruments were associated with the risk factor of interest, were independent of potential confounders, and could only affect the outcome through the risk factor and not through alternative pathways (that is, through pleiotropy).” <SEP>“Additionally, the slope of MR-Egger regression can provide pleiotropy-corrected causal estimates . . . An important condition of approach is that a SNP’s association with the exposure variable must be independent of its direct effects upon the outcome (previously described as the InSIDE assumption).” <SEP>Explanation<SEP>Explicitly stating the three core instrumental variable assumptions, ideally in the methods sections, can help readers to understand the underlying premises of the MR method, and to allow them to judge their validity. Ideally, the assumptions would be stated in the text using intuitive language specific to the study setting and what they imply in the context of the question being asked. Articulating the assumptions also motivates sensitivity analyses and other additional analyses used to assess the assumptions or the robustness of conclusions to their violations.<SEP>When instrumental variable estimation is used to obtain effect estimates, a fourth assumption should then be stated: typically, an assumption of effect homogeneity or monotonicity. In many MR studies, other methods are used to augment traditional instrumental variables estimators (eg, two stage least squares or Wald estimators), and their assumptions should also be stated. For example, MR Egger regression or weighted median  are often used as a supplementary analyses to obtain the estimates when multiple genetic variants are included. Box 2, figure 1, and box 3 give more details on instrumental variable assumptions, common violations, and assessment.<SEP>Instrumental variable (IV) assumptions and mendelian randomisation (MR)<SEP>Core IV estimation assumptions and additional assumptions<SEP>Most MR studies rely on three core IV assumptions (relevance, independence, and exclusion restriction; fig 1, box 3) to carry out testing for causal effects of the exposure on the outcome. Estimating effect sizes through the instrumental variable approach imposes a fourth assumption, usually homogeneity of effects of the exposure on the outcome.    The homogeneity assumption can also be replaced by imposing a monotonicity assumption that an increase in the number of risk alleles does not lower the likelihood of exposure for any individuals, typically leading to estimating an effect in a subgroup of the study population. <SEP>Violations<SEP>The exclusion restriction is sometimes also referred to as an assumption of no horizontal pleiotropy (box 3), but it can be violated in several other ways (eg, by gene-exposure interaction, by having some form of time varying exposures, by measurement error in the exposure that is related to the instrument or by a multi-component exposure). Concerns about violations of the independence assumption usually focus on confounding by ancestry (or population stratification). However, it can also be violated by various forms of selection or collider bias, by dynastic effects, or by assortative mating.   When multiple variants are used in the analysis, these assumptions pertain to each of the variants. Other methods can relax these assumptions, as described below.<SEP>Assumptions for additional analyses<SEP>In many MR studies, instrumental variable methods have been extended in several ways. For example, when multiple genetic variants are used, MR Egger regression, weighted median, or weighted mode are often used as supplementary estimators. MR Egger regression relaxes the exclusion restriction assumption but imposes an InSIDE assumption that the size of the direct effects of the genetic variants on the outcome that do not operate through the exposure are independent of the size of the genetic variants’ effects on the exposure. Additionally, the two sample MR approach assumes that the association between the genetic variants and the exposure is the same in the two samples, which might not hold if samples are selected from different subpopulations (eg, by sex, age, ethnicity). <SEP>Assessment of assumptions of mendelian randomisation and sensitivity analyses<SEP>Relevance<SEP>For the relevance assumption, authors should report how they measured instrument strength. Reporting the F statistic, if individual level data are available, provides several advantages for understanding the risk of weak instrument bias. The F statistic can also be approximated using summary level data. If the proposed instrument strength is low, reporting should include whether approaches that are robust to weak instruments have been used.<SEP>Independence<SEP>The independence assumption cannot be directly verified, but it can be partially assessed in many research settings. Negative control outcomes or negative control populations can sometimes evaluate the reasonableness of the assumption. Reporting associations between measured covariates that might confound the variant-outcome association can also prove helpful, particularly if scaled by instrument strength  or presented alongside a related bias analytical approach. <SEP>Exclusion restriction<SEP>For the exclusion restriction assumption, MR Egger regression can be used to detect certain versions of pleiotropy and therefore provide evidence of certain violations of the exclusion restriction. However, the approach depends on an additional assumption (described above) and requires multiple independent variants. Additional approaches to test the exclusion restriction include weighted median and mode. The use of negative control outcomes or negative control populations might also allow evaluation of this assumption. The use of known biological effects of a SNP can also be leveraged to decrease the probability of violation of this assumption.<SEP>Homogeneity<SEP>The homogeneity assumption requires that the exposure has the same effects in everyone, which is not directly verifiable. One possibility for supporting its validity is to determine whether the effect estimate, or even the genetic variants’ effects on the exposure, is the same across subpopulations.  Authors can perform stratified or adjusted analyses to relax this assumption if effects with meaningful differences are estimated in different subpopulations. Furthermore, a global exploration of the homogeneity assumption can examine any differences in variance of a continuous outcome across the genetic instrument; the extent of such differences provides evidence as to extent of violation of the homogeneity assumption.<SEP>Joint falsification strategies<SEP>Some falsification strategies assess assumptions jointly. When using multiple genetic variants as proposed instruments, it is possible to test whether heterogeneity exists across the separate effect estimates (see Test for difference, table 2). Although this test is often interpreted as an assessment of the exclusion restriction, it is jointly testing the exclusion restriction, independence, and homogeneity assumptions. Another relatively straightforward joint test of all assumptions comes from comparing the effect estimate with that obtained by use of a more traditional confounding-adjustment approach. Assuming that the traditional approach is biased owing to unmeasured confounding, and that the direction of that confounding is suspected, the examination of whether the MR effect estimate aligns with the suspected direction of confounding can support the joint validity of the assumptions underlying the MR effect estimate.<SEP>Sensitivity analyses<SEP>Because several of the estimators using multiple genetic variants rely on different versions of relaxing or adapting the instrumental variable assumptions (eg, MR Egger, median based, or modal based estimators), a comparison of estimates obtained using each of these approaches can help understand the sensitivity of effect estimates to the non-overlapping assumptions of each. Researchers might also compare MR effect estimates with non-MR estimates, depending on the assumptions underlying alternative methods. Independent replication of MR findings in an independent dataset or with a different study design (eg, one sample v two sample MR) is typically advocated to assess the findings’ robustness. Many of the traditional bias analytical techniques in epidemiology can be adapted for MR readily, including the calculations from formulas for understanding the magnitude and direction of confounding bias     or for violations of the exclusion restriction.  When selection bias is a concern, researchers also frequently conduct simulations to understand the plausible size and direction of bias. Simulations might also help understand the plausible size and direction of bias induced by assortative mating, dynastic effects,  and time varying effects, if deemed relevant.<SEP>Statistical methods: main analysis (item 6)<SEP>Describe statistical methods and statistics used.<SEP>Quantitative variables (item 6a)<SEP>Describe how quantitative variables were handled in the analyses (that is, scale, units, model).<SEP>Example<SEP>“The effect size for each meta-analysis is reported in the main results as the effect of a one-standard-deviation (1-SD) change in natural-log-transformed 25OHD [25-hydroxyvitamin D] level, since this metric is more interpretable than an arbitrary difference . . . In order to provide a better clinical interpretation of a 1-SD change in natural-log-transformed 25OHD level, we selected three different clinically relevant 25OHD thresholds for vitamin D status (<25 nmol/l for vitamin D deficiency, <50 nmol/l for vitamin D insufficiency, and >75 nmol/l for vitamin D sufficiency).” <SEP>Explanation<SEP>Any transformations made in the quantitative variables (that is, exposure, outcome, or relevant covariates) should be explicitly mentioned, because they affect both interpretation of results and their comparability with other studies. Describing biological knowledge or previous evidence can help justify chosen groupings. When possible, authors should also back-transform estimates to report the units of measurement in common terms to enable future replication of findings. For example, if the effect size is reported in standard deviation change, we suggest reporting the magnitude of the standard deviation for clarification.<SEP>Genetic variants (item 6b)<SEP>Describe how genetic variants were handled in the analyses and, if applicable, how their weights were selected.<SEP>Example<SEP>“We created an allele score from 97 genetic variants previously found to be associated with BMI [body mass index], in a recent GWAS [genome wide association study] meta-analysis by the GIANT consortium. The score was calculated as a sum of the number of BMI-increasing alleles, weighted by the effect size as reported in the GIANT GWAS (reported as a SD [standard deviation] change of BMI per dosage increase such that a higher allele score corresponds to a higher BMI, and was standardised to have a mean of zero and SD of 1).” <SEP>Explanation<SEP>An allele score (also sometimes referred to as genetic risk score, polygenic risk scores, or genetic prediction scores) is a variable that summarises multiple genetic variants in one measure. When many variants are included in the score, bias and coverage probabilities of the instrumental variable estimates are improved compared with estimates from the two stage least squares approach. Authors should explicitly define the criteria for selecting variants included in the allele score and whether these criteria are based on external data. An allele score could be weighted or unweighted. If weighted, author should clarify whether the weights are derived from the data under analysis or from an independent data source. Authors should also report which genetic model of inheritance is implied in the calculation of genetic variant-exposure and variant-outcome associations (that is, additive or multiplicative). If the weights were derived from the same sample (eg, in a one sample MR), authors should report any efforts to mitigate potential overfitting, typically using methods such as the cross validation or jackknife approaches.<SEP>Mendelian randomisation estimator (item 6c)<SEP>Describe the MR estimator (eg, two stage least squares, Wald ratio) and related statistics. Detail the included covariates and, for two sample MR studies, whether the same covariate set was used for adjustment in the two samples.<SEP>Example<SEP>“Genetic associations with all exposures were taken from a large meta-analysis of GWAS, conducted in adults (n=108,557; mean age, 50.6 years; ~53% men) of European ancestry, without diabetes, adjusted for age, sex, study site and geographic covariates using an additive genetic model . . . Genetic associations with MI [myocardial infarction], angina and heart failure were obtained using logistic regression controlling for age, assay array and 10 principal components in sex-specific analysis and additionally adjusted for sex in the overall analysis, as the adjustment in our previous MR study in the UK Biobank . . . Specifically, we obtained SNP-specific Wald estimates (quotient of genetic association on outcome and genetic association on insulin) and then meta-analysed them using inverse variance weighting (IVW) with multiplicative random effects.” <SEP>Explanation<SEP>The authors should present all the analytical details on the calculation of the instrumental variable estimator. Further clarification on estimating the associated standard errors should also be provided (that is, if this estimator is based on a normal approximation, bootstrapping, or other approaches). Covariates used in the MR analysis should be detailed. For a two sample analysis, authors should state the covariates used to estimate the genetic variant-exposure and genetic variant-outcome associations to assess any differences in the use of covariates between the two associations, which could lead to bias.<SEP>Missing data (item 6d)<SEP>Explain how missing data were addressed.<SEP>Example<SEP>“We conduct our analyses in a Bayesian framework as this lends itself naturally to data imputation. We first introduce a Bayesian complete-case analysis method and then 4 methods for imputing data under the missing-at-random assumption that can be incorporated into the Bayesian model to include subjects with missing data . . . We use cross-sectional baseline data on 3,693 participants who have complete or partial data for C-reactive protein, fibrinogen, and the 3 SNPs. There is missingness in 2.1% of participants for C-reactive protein, 2.4% for fibrinogen, 10.8% for rs1205, 1.9% for rs1130864, and 2.6% for rs1800947.” <SEP>Explanation<SEP>The inclusion of multiple variants with missing data in the estimation of causal effects could decrease precision. Authors should present the percentage of missing data as well as whether any imputation was performed. If imputation was performed the authors should report details of the imputation panel and method of imputation.<SEP>Multiple testing (item 6e)<SEP>If applicable, indicate how multiple testing was addressed.<SEP>Example<SEP>“The significance threshold for all cancer risk and mortality is 0.004 (6 PUFAs times 2 outcomes (risk, mortality) require correcting 0.05 by 12 tests) . . . Given 6 individual cancers were considered, we set a significance threshold of 0.004/36=0.0001.” <SEP>Explanation<SEP>In an MR analysis that involves multiple exposures or multiple outcomes, authors should state whether and how they accounted for multiple testing and provide justification. They should state whether the correction was for the total number of statistically independent exposures or outcomes for all exposures and outcomes. Such a correction could involve reporting false discovery rates, Bonferroni correction, or other techniques, as outlined in the above example.<SEP>Assessment of assumptions (item 7)<SEP>Describe any methods or previous knowledge used to assess the assumptions or justify their validity.<SEP>Examples<SEP>“Mendelian randomisation was implemented using the two stage least squares method in the R package ivpack. We included age and sex as covariates. To assess the risk of weak instrument bias, we used F tests to determine the strength of association in the first stage regressions between allele score and exposure . . . We used confounding bias plots to assess relative bias in the instrumental variable estimate compared with standard multivariable regression . . . To investigate the degree of bias in the initial causal estimates due to pleiotropic effects, we used two sensitivity analyses (mendelian randomisation-Egger and weighted median mendelian randomisation) . . . Mendelian randomisation-Egger and weighted median methods were implemented using the R package TwoSampleMR.” <SEP>“There are reasons for considering MR analysis of a protein drug target to be a distinct category of MR analysis . . . Aside from mRNA expression, differences in protein expression or function are the most proximal consequence of natural genetic variation. This has two consequences: frequently, variants located in and around the encoding gene can be identified with a very substantial effect on protein expression in comparison to other traits; moreover such instruments may also be less prone to violating the ‘no horizontal pleiotropy’ assumption than variants located elsewhere in the genome . . . In the case of MR analysis of proteins, Crick’s ‘Central Dogma’ imposes an order on the direction of information flow from gene to mRNA to encoded protein, which does not extend beyond this to other biological traits that lie more distally in the causal chain that connects genetic variation to disease risk. Finally, cis-MR of a protein risk factor greatly reduces the risk of reverse causation, because Crick’s dogma indicates that the pathway gene → encoded protein → disease would always be favoured over the pathway gene → disease → encoded protein, especially given that the gene → encoded protein association is typically derived from population-based (disease-free) samples. Thus, from an MR perspective, proteins are in a privileged position compared with other categories of risk factor and the use of cis-MR represents an optimal approach to instrument their causal effect for disease.” <SEP>Explanation<SEP>For each of the assumptions underlying an MR analysis, authors should report any methods used to assess the assumptions or justify their validity. Generally, the subject related background can be used to support the reasonableness of each assumption. While many assumptions cannot be verified, there are methods available to attempt to falsify them. In line with the relevance assumption, the authors can report how they assessed instrument strength. If the proposed instrument strength is low, reporting might include whether approaches that are robust to weak instruments have been used. Although many possible methods are available, some are usable only in certain settings (eg, for dichotomous exposures). Box 3 describes some of the more common and useful approaches, and table 5 lists the commonly used statistics used when examining assumptions and performing sensitivity analyses. The first three core assumptions pertain to any MR analysis with a single instrument; the additional assumptions are needed for instrumental variable estimation. Exclusion restriction is relaxed in sensitivity analyses such as MR Egger regression. These assessments and sensitivity analyses do not represent an exhaustive list of possible strategies, and not all sensitivity analyses are relevant to all MR analyses. For example, F statistics are principally relevant for instrumental variable analyses, because they are approximately equivalent to variance explained in two sample MR approaches based on genome wide association study (GWAS) output. Associations with measured covariates (eg, age, sex, and race or ethnic origin) and effect estimates across subpopulations can be reported with one sample MR studies, but generally not with two sample MR studies, however, GWAS summary statistics are increasingly available for sex and ancestry specific analyses. More comprehensive reviews are provided by Glymour et al and Labrecque and Swanson. <SEP>Sensitivity analyses and additional analyses (item 8)<SEP>Describe any sensitivity analyses or additional analyses performed (eg, comparison of effect estimates from different approaches, independent replication, bias analytical techniques, validation of instruments, and simulations).<SEP>Examples<SEP>“Confounding: We used confounding bias plots to assess relative bias in the instrumental variable estimate compared with standard multivariable regression. Such analyses are designed to quantify the bias present in a mendelian randomisation analysis in a manner analogous to examining the effect of adjusting or not adjusting for a potential confounder in a standard regression analysis. Additionally, in supplementary analyses we included suspected confounding factors as covariates (see supplementary table 4). The confounding variables considered were the first 10 genetic principal components, Townsend deprivation index, birth weight, breast fed, and place of birth (northing and easting coordinates).<SEP>“Horizontal (genetic) pleiotropy: To investigate the degree of bias in the initial causal estimates due to pleiotropic effects, we used two sensitivity analyses (mendelian randomisation-Egger and weighted median mendelian randomisation). Mendelian randomisation-Egger is not valid for studies in which the instrumental variable-exposure and instrumental variable-outcome associations are calculated in the same sample (as was done for the main analyses in this study). Therefore, we ran the mendelian randomisation-Egger as a split sample analysis, by randomly splitting the sample in half (groups A and B). The supplementary data table shows the associations of the variants and time spent in education and refractive error for each group. Mendelian randomisation-Egger and weighted median methods were implemented using the R package TwoSampleMR (https://github.com/MRCIEU/TwoSampleMR).<SEP>“Measurement error: To ensure the association between time spent in education and myopia was not an artefact of the non-normal distribution of the variable for age when full time education was completed, we used two alternative methods to recode time spent in education: dichotomisation into age more than 16 years when education was completed and age 16 years or less when education was completed; and excluding those who attended college or university. We compared the results with the original analyses using the continuous variable for age when full time education was completed.” <SEP>“Tests of association for individual genetic variants were complemented with gene-based tests of association and S-PrediXcan analysis. The latter was used to identify genes with differential expression levels in cannabis users versus nonusers. We further estimated the genetic correlation of lifetime cannabis use with other traits, including use of other substances and mental health traits, such as schizophrenia. Lastly, we performed bidirectional two-sample Mendelian randomization analysis to examine whether there was evidence for a causal relationship from cannabis use to schizophrenia risk, and from liability to schizophrenia to cannabis use.” (Further examples are available in web appendix 2.) <SEP>Explanation<SEP>Sensitivity analyses can test the robustness of effect estimates to plausible violations of the underlying assumptions and help understand the plausible size or direction of bias. Authors should report on any such sensitivity analyses performed. Some common strategies are described in box 3, and further information is available elsewhere.   <SEP>Software and pre-registration (item 9)<SEP>Statistical software (item 9a)<SEP>Name statistical software and package(s), including version and setting used. <SEP>Example<SEP>“We performed the analysis by using Stata version 14 (StataCorp LP) and R version 3.4.3 (The R Foundation for Statistical Computing). We used the mrrobust package for Stata and the TwoSample MR package for R to facilitate MR analyses.” <SEP>Explanation <SEP>Statistical methods and software should ideally be described with enough detail to enable a knowledgeable user with access to the original data to verify the reported results. It is preferable to provide the statistical code used in an online repository.<SEP>Pre-registration (item 9b)<SEP>State whether the study protocol and details were pre-registered (as well as when and where).<SEP>Explanation<SEP>Authors should report if a study was pre-registered and provide a link to the study protocol. Examples of pre-registration in MR are rare at present, partly because it poses challenges for secondary data analysis. Potential solutions that protect against researcher bias have been proposed: “The pre-registration can be achieved by pre-specifying the rationale, hypotheses, methods, and analysis plans, and submitting these to either a third-party registry (e.g., the Open Science Framework [OSF]; https://osf.io/), or a journal in the form of a Registered Report.” Wider adoption of these methods should increase the accuracy, transparency, and robustness of MR studies.",
                  "Results<SEP>Descriptive data (item 10)<SEP>Number of participants (item 10a) <SEP>Report the numbers of individuals at each stage of included studies and reasons for exclusion. Consider the use of a flow diagram.<SEP>Example<SEP>“UK Biobank recruited 502 664 participants aged 40 to 69 years through 22 assessment centres across the UK . . . All participants completed sociodemographic questionnaires, which included questions on past educational and professional qualifications. In the latter stages of recruitment, an ophthalmic assessment was introduced, and this was completed by approximately 23% of participants . . . In total, 69 798 participants had valid education, refractive error, and genetic data available (fig 1).” <SEP>Explanation<SEP>Information on study participants will help readers understand the target population and assess the validity and generalisability of results. It also provides readers with the information needed to replicate the study and to assess whether the study is likely to show collider bias. If the data sources include individual level data, authors should report information on the participants in the study. Specifically, authors should report the number of individuals at each stage of the study and the reasons why individuals were excluded from further study. Examples of such reasons include loss to follow-up, removal for lack of data, and quality control. Including a STROBE flowchart for inclusion into the study can quickly provide information about how the study sample was selected. Where possible, report missing values for variables.<SEP>Summary statistics (item 10b)<SEP>Report summary statistics for phenotypic exposure(s), outcome(s), and other relevant variables (eg, means, standard deviations, proportions).<SEP>Example<SEP>“The UK Biobank sample comprised 53.7% women (table 1), and the median age at recruitment was 58.0 years (interquartile range 51.0-63.0). The distribution of adiposity [exposure] and smoking behaviour [outcome] variables in the UK Biobank sample are described in table 1 and table 2 [table 6 and table 7]. As observed in previous studies, current smokers had a lower body mass index than never smokers (−0.22 (95% confidence interval −0.27 to −0.16)). Conversely, former smokers had a higher body mass index than current smokers (1.04 (0.98 to 1.09)).” <SEP>Explanation<SEP>Information on the distribution of the exposure, outcomes, and other variables helps judge the comparability of groups and generalisability of the findings. Distributions of continuous variables are easily summarised by mean and standard deviation, or by median and percentile range (eg, 25th and 75th percentiles) in case of asymmetrical distribution. Numbers and percentages best describe categorical variables. Readers can assess group differences better if the descriptive statistics are provided for each category separately. Statistical inference regarding differences between groups should be reserved for the main analysis. <SEP>In cohort studies in which the outcome is an event, authors should report both the number of events and, if appropriate, the event rate (eg, number of events per person year). A summary measure of follow-up time—such as mean, median, or total follow-up—is also important to understand the period over which events were recorded.<SEP>For a time varying outcome, for which time-to-event data are available, the summary measures should be presented over time; a figure could help communicate this. In case-control studies, the summary measures are typically presented separately for cases and controls. A table of continuous exposures or outcomes by categories might also be helpful. <SEP>Heterogeneity assessment (item 10c)<SEP>If the data sources include meta-analyses of previous studies, provide the assessments of heterogeneity across these studies.<SEP>Example<SEP> Table 8 demonstrates the I2 test statistic, allowing for an assessment of heterogeneity of the effect of the genetic variants on the outcome across studies. <SEP>Explanation<SEP>Evidence on the consistency of the genetic variant’s association with the exposure or outcome helps to understand the degree of heterogeneity of effects. If the estimation is based on a meta-analysis, the number of included studies will also help determine if tests for heterogeneity are properly powered to detect its presence. Presenting 95% confidence intervals along with the I2 statistic is recommended.  <SEP>Two sample mendelian randomisation (item 10d)<SEP>For two sample MR, authors should (1) provide justification of the similarity of the genetic variant-exposure associations between the exposure and outcome samples, and (2) provide information on the number of individuals who were in both samples for the exposure and for the outcome.<SEP>Example 1<SEP>“The genetic variants used for MR were obtained from a GWAS of gallstones conducted in Europeans. A comparison between European and Indian populations with respect to allele frequencies, risk of developing gallstones and gallbladder cancer (GBC) for the genetic variants was made and results are shown in Supplementary Table 1. The allele frequencies between the two populations were generally similar, although with striking differences for some SNPs (e.g. for rs601338, rs1260326, rs174567, rs2469991, rs2290846, where the difference in minor allele frequency was >15%). The risk for developing gallstones and GBC were in broadly the same direction for the SNPs in the Indian population (consistently increased risk for 80% of SNPs in relation to gallstones and 70% SNPs in relation to GBC).” <SEP>Explanation 1<SEP>Two sample MR analyses assume that the SNP-exposure associations are similar in the two samples. For example, the analysis assumes that the two samples are drawn from the same underlying population. However, characteristics such as ethnic origin are not the only relevant factors. Similarity of the two samples can also be violated if, for example, genetic associations were estimated in pre-menopausal versus post-menopausal women, or in a population based sample versus a high risk sample. Where this assumption cannot be made, it should be evaluated by comparing the SNPs’ association with the exposure and the outcome in the two samples, whenever the data are available. If the associations are similar in the two samples, heterogeneity in the associations of the SNPs with the exposure and the outcome is less likely to cause bias. Authors should also provide information on the number of individuals who were in both samples for the exposure and for the outcome.<SEP>Example 2<SEP>“These genome-wide association study estimates were selected from studies that did not include UK Biobank participants, so as to avoid participant overlap, and therefore, in some cases, the genome-wide association study and subsequent instruments differed from the genome-wide association study studies used for the two-sample mendelian randomisation described previously.” <SEP>Explanation 2<SEP>If the authors used the same or similar individuals to estimate the SNP-exposure and the SNP-outcome associations, MR estimates could be biased by a form of the winner’s curse,  which occurs when the statistically strongest associations—usually using a P value threshold—are selected from the discovery sample. This bias can be overcome by using entirely separate samples to select SNPs and estimate SNP-outcome associations. The bias is a linear function of the number of individuals included in both samples, so the consequences of a small amount of overlap may not be severe. <SEP>Main results (item 11)<SEP>Genetic variant associations (item 11a)<SEP>Report the associations between genetic variant and exposure, and between genetic variant and outcome, preferably on an interpretable scale.<SEP>Example<SEP>“The BMI allele score created from the 12 BMI-related SNPs showed a positive dose-response association with BMI (per unit increase 0.14% [0.12%–0.16%], p = 6.30×10−62). The BMI allele score was also associated with 25(OH)D concentrations (per unit increase −0.06% [-0.20% to −0.02%], p=0.004).” <SEP>Explanation<SEP>Reporting the association between the genetic variant and the exposure is required to evaluate the relevance assumption (item 8b). Comparing levels of exposure across the genotype distribution can also indicate monotonicity and linearity of the genetic effect. Reporting on the association between the genetic variant and the outcome is useful because it can provide an initial indication about the possibility of a causal association between the exposure and outcome.<SEP>Mendelian randomisation estimates (item 11b)<SEP>Report MR estimates of the association between exposure and outcome, and the measures of uncertainty from the MR analysis, on an interpretable scale, such as odds ratio or relative risk per standard deviation difference.<SEP>Example<SEP>“The odds ratio of CAD [coronary artery disease] per 1-standard deviation increase in genetically predicted BMI was 1.49 (95% CI [confidence interval] 1.39 to 1.60).” <SEP>Explanation<SEP>If the instrumental variable assumptions are not apparently falsified and are generally supported, or sensitivity analyses are robust to violation of the assumptions (item 8b), then estimates from the MR analysis can be reported in a meaningful manner, preferably on an intuitive scale (eg, relative risk, risk difference). However, if the homogeneity and monotonicity assumptions do not hold, it might be preferable not to report the estimates and outline this situation. Instead, the estimation is replaced by testing for a non-null effect.<SEP>Calculating absolute risk (item 11c)<SEP>If relevant, consider translating estimates of relative risk into absolute risk for a meaningful time period.<SEP>Example<SEP>“LDL [low density lipoprotein] cholesterol lowering alleles at the NPC1L1 locus were inversely associated with coronary artery disease (OR [odds ratio] for a genetically predicted 1-mmol/L [38.7-mg/dL] reduction in LDL-C of 0.61 [95% CI, 0.42-0.88]; P=.008) and directly associated with type 2 diabetes, both individually and collectively (OR for a genetically predicted 1-mmol/L reduction in LDL-C of 2.42 [95% CI, 1.70-3.43], P<.001; estimated absolute risk difference, 5.3 incident cases per 1000 person-years for a 1-mmol/L genetically predicted reduction in LDL-C).” <SEP>Explanation<SEP>In some instances, the interpretation of estimates in terms of absolute risks or risk differences rather than relative risk differences might be more clinically meaningful, by taking into account the baseline risk. A measure of absolute risk can provide an estimate of the excess amount of disease that can be attributed to the exposure over a particular period, which can then be used to estimate the absolute benefit of an intervention aimed at reducing levels of the exposure.<SEP>Visualisation of results (item 11d)<SEP>Consider plots to visualise results (eg, forest plot, scatterplot of associations between genetic variants and outcome versus associations between genetic variants and exposure; see example in fig 3).<SEP>Example<SEP>Explanation<SEP>Plots can be useful for examining potential violations of the instrumental variable assumptions, especially for the exclusion restriction assumption. Authors should report the associations of the exposure and outcome with the genetic variants individually, which can be presented using a scatterplot or funnel plot. The scatterplot depicts the association of the genetic effects on the exposure versus the genetic effects on the outcome, with the slope of the line corresponding to the estimated causal effect, with an intercept that is fixed at the origin (except for MR Egger regression; item 8b). A funnel plot, in which causal effect estimates for variants are plotted against their precisions, can be used to perform a visual inspection for asymmetry, which might indicate horizontal pleiotropy. Forest plots, which plot the causal estimate obtained from each genetic variant, allow for a visual inspection of heterogeneity around the overall causal estimate. <SEP>Assessment of assumptions (item 12)<SEP>Validity of assumptions (item 12a)<SEP>Report the assessment of the validity of the assumptions.<SEP>Example—relevance assumption<SEP>“The myopia allele score explained 4.32% (F=3155) of the variance in average mean spherical equivalent refractive error of participants in UK Biobank and the education allele score explained 0.71% (F=464) of the variance in time spent in education. We selected these genetic variants to use as instrumental variables because of their robust association with time spent in education and myopia, allowing us to construct strong aggregate instrumental variables for making mendelian randomisation inferences. The large F statistics suggested that these analyses would not be affected by weak instrument bias.” <SEP>Example—independence assumption<SEP>“In tests of the association between the allele scores for time spent in education and myopia with potential confounders, there was evidence that the geographical coordinate, northing (measured northward distance in UK) was negatively associated with time spent in education (β=−1.6e-6, 95% confidence interval −1.8e-6 to −1.5e-6) and positively with refractive error (β=1.2e-6, 9.8e-7 to 1.3e-6). Northing was also associated with the time spent in education (P=7e-5) and myopia (P=6e-3) allele scores (see supplementary table 2). Compared with standard regression, the confounding bias plot suggested that inclusion of the northing variable in the instrumental variable analysis would result in a greater degree of bias for the education allele score but not for the myopia allele score.” <SEP>Example—exclusion restriction assumption<SEP>“MR-Egger, weighted mode, and weighted median methods . . . yielded similar causal estimates in magnitude and direction, such that increasing time spent in education led to a more myopic refractive error (by −0.17 to −0.40 dioptres/y), whereas there was little evidence that a more myopic refractive error led to more time spent in education . . . There was little evidence that the Egger intercept deviated from zero either for more time in education causing refractive error (intercept=0.007, SE=0.006, P=0.2) or refractive error causing more time in education (intercept=−0.002, SE=0.007, P=0.8), indicating that there was little evidence for directional genetic pleiotropy.” <SEP>Example—homogeneity<SEP>“We observed a J shaped relation between genetically predicted BMI and all-cause mortality. The curved shape of the relation was more pronounced in UK Biobank—with higher risk both in underweight participants and in overweight or obese participants. The lowest risk for the overall population was at a BMI of around 22-23 in the HUNT Study and around 25 in UK Biobank.” <SEP>Explanation<SEP>Authors should report the results from assessing the validity of the instrumental variable assumptions, as described under item 7 and box 3. These examples illustrate assessments of these assumptions, but do not represent an exhaustive list of possible assessments or assumptions.<SEP>Additional statistics (item 12b)<SEP>Report any additional statistics (eg, assessments of heterogeneity across genetic variants, such as I2, Q statistic, or E value).<SEP>Examples<SEP>“Cochran’s Q and I2 statistics were calculated to check for the presence of heterogeneity (dispersion of SNP effects) which can indicate pleiotropy. We found little evidence of heterogeneity for the association between body mass index and wellbeing (see supplementary table S8 for further information).” <SEP>“Chen et al. used a single variant in the ALDH2 gene to study the effects of alcohol intake on risk of hypertension. Among males, the variant-hypertension association was an odds ratio of 2.42. The E-value then is 4.27. The E-value for the lower limit of the confidence interval (1.66) is 2.71. As the analysis was conducted in an ethnically homogeneous Asian population, this E-value may be large enough to reasonably conclude that any residual confounding by ancestry is unlikely to explain away the effect.” <SEP>Explanation<SEP>Cochran’s Q and I2 statistics can be used to assess evidence of heterogeneity of causal effects estimated by each of the genetic variants. Evidence of heterogeneity suggests that there is at least one proposed instrument for which at least one of the instrumental variable assumptions fails to hold. The E value can be used to understand the degree to which unmeasured confounding might explain findings. A large E value could help support that confounding by ancestry is unlikely to explain a non-null effect.<SEP>Sensitivity analyses and additional analyses (item 13)<SEP>Sensitivity analyses for main results (item 13a)<SEP>Report any sensitivity analyses to assess the robustness of the main results to violations of the assumptions.<SEP>Example<SEP>“The fixed-effect inverse-variance weighted and Egger regression estimates suggest an inverse causal effect of CRP [C reactive protein] on CAD risk ​(table 1). However, the corresponding random-effects analyses imply that there is no convincing evidence for a causal effect. Moreover, the simple median estimate is in the opposite direction. This arises because, although the strongest genetic variants have negative causal estimates, the majority of genetic variants have positive causal estimates. The inconsistency of the estimates from different methods indicates that the genome-wide significant variants for CRP are not all valid instrumental variables, and that a causal conclusion based on these variants would be unreliable.” <SEP>Explanation<SEP>Authors should report on, and compare, results obtained from different approaches used to assess the robustness of conclusions to violation of the instrumental variable assumptions, as described in section 7 and box 3. If the results from all the approaches are largely consistent, author can have more confidence in drawing robust conclusions regarding the presence and magnitude of a causal effect.<SEP>Other analyses (item 13b)<SEP>Report results from other sensitivity analyses or additional analyses.<SEP>Example—independent replication<SEP>“The associations of genetically predicted body mass index and waist circumference with risk of being a smoker were replicated in the TAG [Tobacco and Genetics consortium] data (1.19 (1.06 to 1.33) and 1.32 (1.15 to 1.52), respectively.” <SEP>Example —validation of instruments<SEP>“The MR-PRESSO method identified one outlier SNP for heart failure, six outlier SNPs for coronary artery disease, and 11 outlier SNPs for arterial hypertension. Outlier-correction did not materially change the OR estimates for heart failure (1.13; 95% CI 1.08–1.17), coronary artery disease (1.08; 95% CI 1.06–1.10), or arterial hypertension (1.10; 95% CI 1.08–1.12). No outlier SNPs were identified in the MR-PRESSO analysis of the other outcomes.” <SEP>Example—simulations<SEP>“Figure 2 shows that TSLS [two-stage least squares regression methods] is positively biased when there is positive cross‐trait assortative mating on X and Y. The bias increased proportionally with increasing the degree of assortment. However, both TSLS (2) (i.e., adjusting for parent’s allele scores) and TSLS (3) (i.e., jointly modelling individual’s and parental effects, using nontransmitted allele scores as instruments of parental phenotype) were unbiased with false discovery rates close to 5%.” <SEP>Explanation<SEP>Results of sensitivity analyses or additional analyses, such as independent replication, validation of instruments, and simulation studies, should be presented if they have been performed, as described under item 8.<SEP>Direction of causality (item 13c)<SEP>Report any assessment of direction of causality (eg, bidirectional MR).<SEP>Example<SEP>“The BMI allele score was also associated with 25(OH)D concentrations (per unit increase −0.06%, [−0.10% to −0.02%], p=0.004) while no association with BMI was seen for either the vitamin D synthesis or metabolism allele scores (per allele in synthesis score: 0.01% [−0.17% to 0.20%], p = 0.88, metabolism allele score: 0.17% [−0.02% to 0.35%], p=0.08]).” <SEP>Explanation<SEP>Bidirectional MR can be used to orient the causal direction(s) of effect. This is done using two independent sets of genetic variants related to the exposure and outcome separately, and performing MR analyses to appraise causality in both directions. <SEP>Compare with non-MR analyses (item 13d)<SEP>When relevant, report and compare with estimates from non-MR analyses.<SEP>Example<SEP>“Using the Durbin-Wu-Hausman test for endogeneity, we found weak evidence that the instrumental variable estimate using the time spent in education allele score differed from the observational point estimate (Durbin-Wu-Hausman P=0.06), with the instrumental variable estimate suggesting a larger negative association.” <SEP>Explanation<SEP>Authors should describe important differences between MR estimates and estimates from non-MR analyses. Each study design has different types of biases and could have different degrees of statistical power. Putting the MR results in context will help readers understand if the strengths and weaknesses of MR allow for results that support or contradict previous evidence. In general, causal inference can be presented in a triangulation framework, evaluating the overall body of evidence from several different approaches.  <SEP>Additional visualisation of results (item 13e)<SEP>Consider additional plots to visualise results (eg, leave-one-out analyses).<SEP>Example<SEP>“Leave-one-out analysis: each row represents a two-sample MR analysis of BMI on subjective wellbeing using all of the genome-wide significant SNPs available from Locke et al. except for the SNP listed on the y-axis. The point represents the effect size with that SNP removed and the line represents the standard error. Leave-one-out analysis was conducted using MR Base to identify if any individual SNPs were driving the association between BMI and wellbeing . . . The SNP with the largest contribution to the effect is rs1421085 located on chromosome 16 in the second intron of the FTO (fat mass and obesity associated) gene. FTO has been repeatedly associated with obesity in different populations. However, the biological consequences of intronic FTO SNPs are still unknown. They are currently thought to play a regulatory role in FTO gene expression in the hypothalamus. Although research is not completely certain of the role of FTO, its large effect size and robust association with obesity suggest that this gene has the largest effect in the two-sample MR because of its BMI effect size rather than because of pleiotropic effects.” (See example in fig 4; further examples are available in web appendix 2.) <SEP>Explanation<SEP>Additional plots might also aid in the visualisation of results, assessing assumption violation, and detecting potential influential or outlier points. These include the leave-one-out plot, radial plot, and plots of each genetic variant against their studentised residuals or Cook’s distance for outlier assessment. ",
                  "Discussion<SEP>The discussion should look at the important issues pertaining to study interpretation and validity. Structured discussions can help authors avoid over-interpreting results, and act as a guide for readers.  <SEP>Key results (item 14)<SEP>Summarise key results with reference to study objectives.<SEP>Example<SEP>“Based on comprehensive genetic data from nearly 450 000 individuals, our study provides evidence that differences in body mass index and body fat distribution causally influence different aspects of smoking behaviour, including the risk of individuals taking up smoking, smoking intensity, and smoking cessation. These results highlight the role of obesity in influencing smoking initiation and cessation, which could have implications for public health interventions aiming to reduce the relevance of these important risk factors.” <SEP>Explanation<SEP>The discussion should begin with a summary of the main results and a statement of their importance. This section reminds readers of the study questions and its primary findings, and helps readers assess whether the interpretations that follow are consistent with the results. Good practice would be to keep the summary in the perspective of the main study objectives and focus on the prespecified hypothesis, reporting the estimates of the investigated causal association in the given population. <SEP>Limitations (item 15)<SEP>Discuss limitations of the study, taking into account the validity of the instrumental variable assumptions, other sources of potential bias, and imprecision. Discuss both direction and magnitude of any potential biases and your efforts to resolve them.<SEP>Example<SEP>“As in any Mendelian randomisation analysis, several assumptions were made, including that the genetic instruments were associated with the risk factor of interest, were independent of potential confounders, and could only affect the outcome through the risk factor and not through alternative pathways (that is, through pleiotropy). We note that the first assumption was satisfied because robustly associated gene variants were identified from the largest genome wide association study for each obesity parameter. Whether the other two assumptions held was not readily testable, although we conducted thorough sensitivity analyses that did not highlight any obvious violation of these assumptions. Secondly, a potential confounder of our results was population stratification by sociodemographic factors. Indeed, it was previously shown that the genetic instrument for body mass index was associated with various factors related to social class among women, including lower annual household income and level of deprivation. However, no such associations were seen in men. In our study, the associations between the genetic instruments of obesity and individuals taking up smoking and smoking intensity were consistently observed in both men and women, separately, and also when we excluded SNPs that were potentially linked to social deprivation. Therefore, apart from the inverse association between body fat percentage and smoking cessation observed in women only, population stratification by sociodemographic factors would not seem likely to explain those results.” (Further examples are available in web appendix 2.)<SEP>Explanation<SEP>Authors should address the plausibility of all the instrumental variable assumptions, which is especially important because many of the assumptions are not empirically verifiable. Authors could consider, for example, the possibility that (residual) genotype-phenotype confounders (such as population structure, genetic nurture, or assortative mating) could lead to a violation of the independence assumption. When evaluating a potential violation, authors should identify the sources of a violation that could affect results and discuss the relative importance of different violations, including the likely direction and magnitude of any bias they could induce.<SEP>Authors should also discuss the precision of the results. Imprecision can be due to several features of the study design. For example, an instrumental variable estimate’s precision from a meta-analysis of multiple SNPs will usually be greater than that for a single SNP. Suppose SNPs are chosen based on meeting a P value criterion in a discovery GWAS. In that case, authors should consider factors that affect this GWAS’s power to detect SNPs, such as sample size and measurement error. Instrumental variable estimates will also be more precise when estimated from larger datasets, because the standard error for the SNP effect estimates, used to calculate the instrumental variable estimate, will be smaller.<SEP>Interpretation (item 16)<SEP>Meaning (item 16a)<SEP>Give a cautious overall interpretation of results in the context of their limitations and in comparison to other studies.<SEP>Example<SEP>“These Mendelian randomization analyses suggest that the causal effect of CETP (cholesteryl ester transfer protein) inhibition on the risk of cardiovascular events appears to be determined by changes in the concentration of apoB-containing lipoproteins rather than changes in LDL-C or HDL-C level.” <SEP>Explanation<SEP>Provide a cautious interpretation of the overall results. When comparing with results from other studies, consider how the results might differ from previous estimates and discuss possible reasons for these differences. Such reasons could include violations of instrumental variable assumptions, imprecision, different estimation methods, and different studied populations. Consider that the overall results should be interpreted in the context of other studies that assessed the study question using different designs, allowing for triangulation of results (see item 13d). When interpreting the effect size, discuss assumptions underlying any extrapolations of effect size and how they may have influenced results.<SEP>Mechanism (item 16b)<SEP>Discuss underlying biological mechanisms that could drive a potential causal association between the investigated exposure and the outcome, and whether the gene-environment equivalence assumption is reasonable. Use causal language carefully, clarifying that instrumental variable estimates might provide causal effects only under certain assumptions.<SEP>Example<SEP>“The association between pubertal timing and weight status is complex and plausibly bidirectional. Increased adiposity in childhood has been linked to earlier pubertal maturation, although this relationship may be nonlinear in boys. Furthermore, several studies report evidence for an association between earlier age at puberty and later obesity. Therefore, we sought to control for both genetically predicted adult and childhood BMI, and we observed a similar magnitude of attenuation in the association between pubertal timing and risk of MS. However, there is a strong association between childhood and adult BMI, which limits the exploration of age-specific effects. Nonetheless, postpubertal rather than childhood obesity is most clearly related to MS susceptibility, making the association between pubertal timing and adult obesity the most likely mediator of the effect of age at puberty on risk of MS. Because it appears that BMI and pubertal timing are in the same causal biological pathway, the association of the selected genetic variants with both exposures represents an example of vertical pleiotropy due to shared biological underpinnings and thus does not bias the MR estimates.” <SEP>Explanation<SEP>While the biological mechanisms that allow genetic variants to be used as instrumental variables are often unknown, the discussion should consider possibilities. Doing so will enable readers to put the MR results in context about possible biological mechanisms, allowing for a better understanding of the plausibility of causal associations.<SEP>A common reason why MR estimates do not have a straightforward causal interpretation is because it is unclear whether the gene-environment equivalence assumption (table 2) is plausible (this assumption being that differences in the exposure between genetically defined subgroups of the population are equivalent to differences in the exposure due to an intervention—that is, the MR form of the consistency assumption). Effects of genetic variants can influence outcomes from conception onwards and in a manner that can be variable and complex. Various components of growth and development could be influenced by the variants, whilst their identification through genetic association studies is often from association with phenotype measured at one time point. The often lifelong influence of genetic variants can be different from the environmental influences investigated in conventional epidemiological studies that are generally experienced at defined stages of later life. Differing effects over the life course do not lessen the potential use of MR estimates. For example, it is possible to separate the effects of body mass index in childhood and adulthood on a variety of health outcomes using different instruments for the two exposure periods. Conversely, the long term effect of lowered low density lipoprotein (LDL) cholesterol can be estimated from early life onwards by MR and is double the effect estimate derived from randomised trials. This difference probably reflects MR providing an estimate of lifelong effects, whereas trials provide an effect estimate for period of randomisation, which lasts just a few years in cholesterol lowering randomised clinical trials. This difference seen between estimates relating to lifetime differences and shorter term cholesterol lowering is anticipated from the known cumulative effect of lipids on atherosclerotic coronary disease. The fact that similar estimates are seen in this regard for a wide variety of non-overlapping sets of instruments constructed to match pharmaceutical agents for LDL cholesterol lowering adds robustness to the interpretation. Time dependency of the effects of instruments is an important issue that should be considered when interpreting MR findings, and is well served by viewing this in the context of gene-environment equivalence.<SEP>Clinical relevance (item 16c)<SEP>Discuss whether the results have clinical or public policy relevance, and to what extent they inform effect sizes of possible interventions.<SEP>Example<SEP>“Although uncertainty remains around the precise function of each of the 162 SNPs, their degree of pleiotropy with cardiac traits, and the mechanisms by which these genetic variants exert their cardioprotective influence, conclusions can still be drawn . . . we note that interventions should be accompanied by careful monitoring for unforeseen side effects, especially in those people who may not thrive when forced into extended educational settings, which may otherwise aggravate health inequalities.” <SEP>Explanation<SEP>Investigators should describe the potential impact of the results on clinical practice or public policy, if any. Because many interventions cannot be tested in randomised clinical trials, MR evidence might help to better understand the possible causal effect of the exposure on the outcome (box 4). Such statements should be made with caution and in the light of evidence from other sources, such as other observational and experimental studies when available. Since clinical and policy interventions might have different effect sizes compared with the genetic variants included in the MR study, extrapolation of this evidence should be clearly described and cautious.<SEP>Interpretation of causal effect estimates<SEP>Various considerations are needed when interpreting causal estimates. If the homogeneity assumption is plausible, along with the other assumptions (box 2), then the causal estimate will represent the average causal effect of the exposure on the outcome in the studied population. If the homogeneity assumption cannot be made, but the monotonicity assumption is plausible, then the causal estimate can be used to represent the local average treatment effect. Caution is especially warranted when interpreting effect estimates with a binary exposure. In this instance, the homogeneity and monotonicity assumptions are less likely to be plausible. Also, if the exposure is a dichotomisation of a continuous risk factor, this poses a further threat to the violation of the exclusion restriction assumption. An additional consideration, which is particularly pertinent to the two sample MR setting, is whether the causal effect can truly be attributed to the binary exposure. For example, when two sample MR studies are carried out in exposure samples that contain only a small number of participants who have experienced the exposure in question, it would be misleading to interpret the effects as being those of the exposure itself. Instead, the causal effect estimate should be interpreted as reflecting the effects of the genetic liability to the exposure.  Finally, an important component of interpretation is defining the time period. Usually MR studies are interpreted as a so-called lifetime effect of the exposure, but some settings (eg, MR studies in pregnant women to study prenatal exposures) lend themselves to studies of period effects.<SEP>Generalisability (item 17)<SEP>Discuss the generalisability of the study results (a) to other populations, (b) across other exposure periods or timings, and (c) across other levels of exposure.<SEP>Example<SEP>“Our Mendelian randomisation work examined a linear relation between vitamin D levels and fracture risk. We did not test for the possibility of a threshold dependent relation—that is, effects that could be present only at very low levels of vitamin D . . . Finally, the non-significant trend observed for vitamin D towards having increased risk of fracture could be attributed to the selection of healthy people (that is, participants with very low levels of vitamin D and fracture, as well as those who are older, frail, and physically impaired, could have been under-represented in the studies included in the GWAS meta-analyses). Therefore, the vitamin D estimates of the current study cannot be generalised to these groups of older people.” <SEP>Explanation<SEP>The generalisability of a study is the extent to which the study’s results apply to circumstances different from the ones in which the study was conducted. For example, findings from a cohort of a specific age group collected in the past might not apply to people currently in the same age group. <SEP>MR studies can fail to generalise in other ways. For example, because genetic variants might not have a constant effect over the entire life course, authors should consider whether the effect estimate derived in the study would generalise to other exposure periods. For example, if the effect of the exposure on the outcome is time dependent, or only occurs during a critical period, MR estimates could be misleading if used to guide future interventions if they occur outside of this time frame. Likewise, if the effect of an exposure on an outcome is cumulative over many years, MR can overestimate the effect when compared to short term interventions.  <SEP>Also, MR estimates are directly calculated only for the exposure range caused by differences in alleles. Applying MR results, therefore, might not generalise to a wider exposure range. Further, if the MR estimate was derived from a population subgroup, it might not be generalisable beyond that subgroup.<SEP>Other information<SEP>Funding (item 18)<SEP>Describe sources of funding and the role of funders in the present study and, if applicable, sources of funding for the databases and original study or studies on which the present study is based.<SEP>Example<SEP>“Funding: The breast cancer genome-wide association analyses were supported by the Government of Canada through Genome Canada and the Canadian Institutes of Health Research, the Ministère de l’Économie, de la Science et de l’Innovation du Québec through Genome Québec and grant PSR-SIIRI-701, the National Institutes of Health (U19 CA148065, X01HG007492), Cancer Research UK (C1287/A10118, C1287/A16563, C1287/A10710), and the European Union (HEALTH-F2-2009-223175 and H2020 633784 and 634935). All studies and funders are listed in Michailidou et al.25 RCR, ELA, BMB, CLR, RMM, MM, DAL, and GDS are members of the MRC Integrative Epidemiology Unit at the University of Bristol funded by the Medical Research Council (grant Nos MM_UU_00011/1, MC_UU_00011/2, MC_UU_00011/5, MC_UU_00011/6, and MC_UU_00011/7). RCR is a de Pass VC research fellow at the University of Bristol. This study was supported by the NIHR Biomedical Research Centre at the University Hospitals Bristol NHS Foundation Trust and the University of Bristol. The views expressed in this publication are those of the authors and not necessarily those of the National Health Service, National Institute for Health Research, or Department of Health and Social Care. This work was also supported by Cancer Research UK (grant No C18281/A19169) and the Economic and Social Research Council (grant No ES/N000498/1). SEJ is funded by the Medical Research Council (grant No MR/M005070/1). TMF is supported by the European Research Council (grant No 323195:GLUCOSEGENES-FP7-IDEAS-ERC). MNW is supported by the Wellcome Trust Institutional Strategic Support Award (grant No WT097835MF).” <SEP>Explanation<SEP>The source of research funding can lead to bias or perceptions of bias in the design, conduct, or interpretation of research.  This bias is of special concern when research is funded by an entity that has an interest in outcomes that are favourable to its own commercial, academic, or other interests. Authors should disclose all funding sources and provide detailed information about the role of funders in developing the research question, collecting data, analysing data, selecting investigators, reviewing results, preparing the manuscript, or approving the manuscript for submission or publication. Other sources of influence can include employers, political appointees, and government researchers. Describing the source of funding allows readers to evaluate the work’s credibility and trustworthiness in the light of any potential influence from funders. Authors should disclose funding sources for biobanks or other repositories or databases used in their study because these entities also have commercial interests that might influence research integrity.  <SEP>Data and data sharing (item 19)<SEP>Provide the data used to perform all analyses or report where and how the data can be accessed, and reference these sources in the article. Provide the statistical code needed to reproduce the results in the article, or report whether the code is publicly accessible and if so, where.<SEP>Example<SEP>“Data sharing: The data reported in this paper are available by application directly to the UK Biobank. The genetic associations with the outcomes in the UK Biobank and CARDIoGRAMplusC4D consortium are provided in the supplementary data. Software code in R for implementing the mendelian randomisation analysis, including the principal components analysis, is provided in the supplementary note.” <SEP>Explanation<SEP>Original data are needed by readers or researchers who wish to evaluate or replicate analyses. Many funders and journals encourage or require data sharing, and provide guidance to authors about the content of any explicit data sharing statement required by their journal. Consensus is building that data sharing is “an inseparable part of the research process.” Ideally, a data sharing plan should be developed when a study is being organised and described in the study protocol and journal publications. The plan and any subsequent data sharing statement in the article should indicate, at a minimum, what data are available (eg, individual participant data, statistical analysis plan, documents related to the study, biobank or other database information) and how the data can be accessed. Contact information for the person or organisation holding the data and a description of the mechanism that will be used to share data should be provided. The statement should also describe any time limits on data availability, processes, and standards applied to requests for data (such as requirements for a research protocol or review of applications by a review board) and, if known, whether a charge is required to obtain the data. If the data are available only via federated analyses, the authors should consider making this clear to readers. When data come from multiple sources, and different conditions apply, consider the use of a table format instead of a text statement.<SEP>Conflicts of interest (item 20)<SEP>All authors should declare all potential conflicts of interest.<SEP>Example<SEP>“Competing interests: All authors have completed the International Committee of Medical Journal Editors (ICMJE) uniform disclosure form at www.icmje.org/coi_disclosure.pdf. ARC, DG, TT, JV, REW, GH, RM, SS, SB, GDS, MVH, IT, and AD declare no support from any organisation for the submitted work; no financial relationships with any organisations that might have an interest in the submitted work in the previous three years; no other relationships or activities that could appear to have influenced the submitted work. MRM reports grants from Pfizer and non-financial support from GlaxoSmithKline, outside the submitted work. NMD reports grants from ESRC, grants from MRC, during the conduct of the study; grants from GRAND/Pfizer for unrelated research, outside the submitted work. AET reports grants from Pfizer, outside the submitted work. LDH reports grants from MRC, during the conduct of the study. DW reports grants from NIH, during the conduct of the study.” <SEP>Explanation<SEP>Financial connections between researchers and commercial or other entities and firmly held ideological or intellectual views can lead to bias in the design, conduct, or reporting of study results. When such interests are not disclosed, public trust in the research enterprise is eroded. According to the International Committee of Medical Journal Editors, “conflict of interest exists when professional judgment concerning a primary interest (such as patients’ welfare or the validity of research) may be influenced by a secondary interest (such as financial gain). Perceptions of conflict of interest are as important as actual conflicts of interest.” Authors should err on the side of disclosing all matters that might be considered relevant by readers.",
                  "104787",
                  "['OBWCCE']"
                ],
                [
                  "45",
                  "30967578",
                  "Introduction<SEP>Studies focusing on the therapeutic potential of brown adipose tissue (BAT) against obesity have increased over the last decade. Two types of thermogenic cells, the classical brown and beige/brite adipocytes, are present in mammals. In mice, the classical brown adipocytes in BAT are constantly available in the interscapular area for heat production. The development and activity of beige adipocytes can be induced in white adipose tissue (WAT), as a result of cold, ß3-adrenergic stimulation or long-term treatment with peroxisome proliferator-activated receptor γ (PPARγ) agonists. Thermogenesis of both classical BAT and beige adipocytes is mediated by the mitochondrial uncoupling protein-1 (UCP1) located in the inner mitochondrial membrane. In these cells, the energy generated by the respiratory chain in the form of the proton gradient is dissipated, by the assistance of elongated fatty acids. This results in heat production instead of ATP synthesis.<SEP>Detailed studies in rodents revealed that, in spite of functional and morphological similarities, classical brown and beige adipocytes have distinct origins. Brown adipocytes derive from the same precursors as myocytes. On the other hand, white and beige adipocytes have different common mesenchymal precursors and their developmental programs are separated at a certain point of differentiation. When adrenergic stimulation, leading to the appearance of beige adipocyte population marked by high Tbx1 and Cited1 expression, is eliminated, beige adipocytes can be reversed into masked beige (white looking) cells. Genes, like zinc finger protein 1 (Zic1), have been described as classical brown adipocyte markers which are not expressed by the beige adipocytes.<SEP>Irisin is a myokine, cleaved from the FNDC5 transmembrane protein, which acts as a browning-inducer endocrine hormone, through integrin αVβ receptors and via the p38 MAPK and ERK pathways. In mice, physical exercise as well as shivering induced the secretion of irisin by skeletal myocytes which resulted in a marked beige differentiation in the subcutaneous WAT. The p38 MAPK signaling is also induced by distinct bone morphogenetic proteins (BMPs), including BMP7 which is an auto/paracrine mediator in mice that both drive classical brown and beige adipocyte differentiation. Another key endocrine hormone, atrial natriuretic peptide (ANP), which is secreted by cardiomyocytes and switches on p38 MAPK signaling, promotes UCP1-dependent thermogenesis and mitochondrial biogenesis of murine and human beige adipocytes. Besides UCP1-dependent thermogenesis, UCP1-independent heat-producing mechanisms were described as a beige specific feature. One example is a creatine-phosphate futile cycle, which requires coupled ATP synthesis. During this process, creatine kinase catalyzes the phosphorylation of creatine using ATP generating phospho-creatine and ADP. Then, phospho-creatine is immediately dephosphorylated which results in heat production. The importance of this cycle was demonstrated in mice which lack in adipose tissue the rate-limiting enzyme of creatine biosynthesis, glycine amidinotransferase developed diet-induced obesity due to the suppression of thermogenesis mediated mostly by beige adipocytes.<SEP>In contrast to detailed studies in rodents, there are only limited data about regulatory networks that drive human brown or beige adipocyte differentiation and activation. In adult humans, brown adipocytes are localized to the deep neck, supraclavicular or perispinal regions. However, these regions are difficult to access for sample collection. Human adipose-derived mesenchymal stem cells (hADMSCs) isolated from stromal vascular fractions (SVFs) of abdominal subcutaneous fat were able to differentiate into functional thermogenic adipocytes, presumed to be beige cells, in response to different stimuli. The creatine-driven futile cycle was also active in human beige adipocytes of different anatomical origins. However, when irisin was administered directly to differentiating human hADMSCs, inconsistent effects were observed.<SEP>It has been recently clarified that SNP rs1421085 T-to-C variant underlies the genetic association between the FTO locus and obesity. The presence of the risk-allele of the FTO locus results in a cell autonomous shift in the gene expression programs toward white instead of beige adipocyte with lipid-storage and a decrease in thermogenesis in response to cold stimulus. Of note, to our knowledge, the expression of the 2-oxoglutarate dependent dioxygenase enzyme encoded by the Fto gene is not influenced by the presence of the aforementioned risk-allele.<SEP>Human cell lines and cellular models are needed to explore more the key molecular elements of browning and targets of possible pharmacological treatments that can enhance browning. Shinoda et al. isolated SVF from supraclavicular and subcutaneous WAT from two healthy individuals, immortalized them, and found that brown adipocytes derived from single cell derived clones of preadipocytes from the supraclavicular regions displayed molecular signatures which resembled thermogenic beige adipocytes. Xue et al. generated human preadipocyte cell populations derived from the supraclavicular region of four healthy patients and immortalized them. Using similarly generated preadipocyte clones, genes which may determine the commitment to differentiation to brown adipocytes in the supraclavicular area have been identified. A non-immortalized human preadipocyte cell line, namely the Simpson-Golabi-Behmel syndrome (SGBS) cell strain was described by Wabitsch et al., in 2001 and it turned out to be a useful model for white adipocyte differentiation. SGBS cells behave very similarly to human primary preadipocytes and the differentiated white SGBS adipocytes cannot be distinguished from human primary adipocytes by morphology or in functional terms after culturing them in adipogenic medium. Furthermore, it has been also demonstrated that knockdown of 2-oxoglutarate dependent dioxygenase (encoded by the proposed Fto gene) or teneurine-2 in SGBS preadipocytes using siRNA induced both UCP1 mRNA and protein expression upon adipogenic differentiation raising the possibility that SGBS cells represent a preadipocyte population with a significant beige potential. In the present study, we investigated systematically how browning of SGBS cells can be induced by PPARγ, irisin and BMP7 stimuli, and found that browning differentiation results in functional and sustainable beige cells.",
                  "Materials and Methods<SEP>Cell culture- and differentiation of human SGBS preadipocytes into mature white and brown adipocytes<SEP>SGBS preadipocytes were seeded in 12-well plates (Costar) and cultured in DMEM-F12 (Dulbecco’s Modified Eagle’s Medium/Nutrient F-12 Ham) medium (Sigma, Germany) containing 33 µM biotin (Sigma), 17 µM pantothenic acid (Sigma), 100U/ml penicillin/streptomycin (Sigma) and 10% FBS (Gibco, UK) at 37 °C in 5% CO2 for overnight to attach. Cells were cultured in this medium until they reach complete confluency. The absence of mycoplasma was checked by polymerase chain reaction (PCR) analysis (PCR Mycoplasma Test Kit I/C, PromoKine, PromoCell France). White adipogenic differentiation was induced for four days using serum-free medium supplemented with 33 µM biotin, 17 µM pantothenic acid, 100U/ml penicillin/streptomycin, 2 µM rosiglitazone (Cayman Chemicals, USA), 10 µg/ml human apo-transferrin (Sigma), 20 nM human insulin (Sigma), 25 nM dexamethasone (Sigma), 500 µM 3-isobutyl-1-methylxantine (Sigma), 100 nM cortisol (Sigma) and 200pM triiodothyronine (Sigma). After the fourth day, the medium was changed; and rosiglitazone, 3-isobutyl-1-methylxantine and dexamethasone were removed during the remaining 10 days of differentiation. The differentiation medium was replaced in every fourth day.<SEP>PPARγ-driven browning differentiation was induced for four days using serum-free medium supplemented with 33 µM biotin, 17 µM pantothenic acid, 100U/ml penicillin/streptomycin, 10 µg/ml human apo-transferrin, 0,85 mM human insulin, 1 µM dexamethasone, 500 µM 3-isobutyl-1-methylxantine and 200pM triiodothyronine. After four days, the medium was changed for following ten days adding 500 nM rosiglitazone while dexamethasone and 3-isobutyl-1-methylxantine were omitted. In long-term experiments either the same culture conditions were applied for 21 and 28 days or it was replaced by the white differentiation medium without rosiglitazone, 3-isobutyl-1-methylxantine and dexamethasone. The differentiation medium was changed in every fourth day.<SEP>Human recombinant irisin (Cayman Chemicals) was used at 250 ng/ml concentration, and human recombinant BMP7 (R&D systems, Minneapolis, USA) was administered at 50 ng/ml concentration. Where indicated, the browning-inducers were added to the differentiating cells in increasing concentrations.<SEP>RNA isolation, RT-PCR, qPCR<SEP>Total RNA was isolated from differentiated SGBS adipocytes by using Trizol reagent (Invitrogen Life Technologies, USA). RNA concentrations were determined by spectrophotometry. To generate cDNA, TaqMan reverse transcription reagents kit (Applied Biosystems, USA) was applied according to the manufacturer’s instructions. Applied Biosystems designed the gene primers and probes. Quantitative real-time PCR (qPCR) was performed in a LightCycler 480 (Roche) with the program of 10 seconds at 94 °C, followed by 40 cycles of 12 seconds at 94 °C and 30 seconds at 60 °C. All samples were used in triplicates. Normalized gene expressions were calculated by ∆Ct method. Human Gapdh was used as an endogenous control.<SEP>Mitochondrial DNA isolation and quantification by qPCR<SEP>Total DNA was isolated from differentiated SGBS adipocytes by the phenol-chloroform extraction, using Trizol reagent (Invitrogen Life Technologies). During the qPCR we were using diluted samples, 10 µM from each primer (human mitochondrial DNA specific primers: forward 5′CTATGTCGCAGTATCTGTCTTTG-3′, reverse 5′-GTTATGATGTCTGTGTGGAAAG-3′, nuclear specific primers (SIRT1 gene): forward 5′CTTTGTGTGCTATAGATGATATGGTAAATTG-3′, reverse 5′GATTAAACAGTGTACAAAAGTAG-3′), and Maxima SYBR Green/ROX qPCR Master Mix (Thermo Scientific, USA). LightCycler 480 (Roche) was used with the program of 20 minutes at 95 °C, and 50 cycles of 15 seconds at 95 °C, 20 seconds at 58 °C, 20 seconds at 72 °C. Relative mitochondrial DNA content was calculated from the difference between the threshold cycle (CT) values for mitochondrial DNA and nuclear specific amplification. Data show mitochondrial genomes per diploid nuclei.<SEP>Antibodies and immunoblotting<SEP>Differentiated SGBS adipocytes and undifferentiated control cells were washed once with PBS, and then suspended in lysis buffer which consists of 50 nM Tris-HCl, 0,1% Triton X-100 (Sigma), 15 mM 2-mercaptoethanol (Sigma), 1 mM EDTA (Sigma) and protease inhibitor (Sigma). The lysates were suspended in 5x Laemmli loading buffer and boiled for 10 min at 100 °C. Equal amount of proteins were loaded onto a 12%-SDS-polyacrylamide gel, and transferred onto a PVDF Immobilon-P Transfer Membrane (Merck-Millipore, Germany). Then the membranes were blocked in 5% skimmed milk (Sigma) for 1 hour. For overnight, membranes were probed at 4 °C with primary antibodies: monoclonal anti-UCP1 (1:1000, R&D Systems, MAB6158), polyclonal anti-UCP1 (1:500, Sigma, U6382; 1:500, Thermo Scientific, PA1-24894), anti-OXPHOS (1:1000, Abcam, UK, ab110411), and anti-actin (1:10000, Sigma, A2066), in TBS-T containing 1% non-fat skimmed milk, followed by the incubation with horseradish-peroxidase-conjugated species corresponding secondary antibodies (Sigma) for 1 hour at room temperature. Immunoreactive proteins were visualized by Immobilion western chemiluminescence substrate (Merck-Millipore). ImageJ software was used to carry out the densitometry measurements.<SEP>Oxygen consumption and extracellular acidification measurements<SEP>Real-time oxygen consumption and extracellular acidification rates were measured by using an XF96 oximeter (Seahorse Biosciences, North Billerica, MA, USA). SGBS cells were seeded onto 96-well XF96 cell culture microplates. Cells were kept in growth medium at the longest for 24 hours and then the formerly-described differentiation process started. After recording the baseline oxygen consumption, cells received a single bolus of dibutyril-cAMP at 500 µM concentration to induce adrenergic stimulation. Then, stimulated oxygen consumption was measured in every 30 minutes. The final reading took place at 6 h post-treatment. Differentiated adipocytes were treated with 2 mM ß-guanidinopropionic acid (Sigma) to block the creatine-driven substrate cycle. In addition, proton leak respiration was measured by oligomycin (Enzo, USA) treatment at 2 µM concentration, which blocks the ATP synthase. For baseline correction, cells received a single bolus of Antimycin A (Sigma) treatment at 10 µM concentration. After the measurements, oxygen consumption rate was normalized to protein content.<SEP>Laser-scanning cytometry<SEP>SGBS cells were plated on eight-well ibidi micro slides (Ibidi GmbH, Germany) and differentiated as previously described. Immunofluorescence staining was carried out as described previously. Scanning was done by an iCys Research Imaging Cytometer (iCys, Thorlabs Imaging Systems, Sterling, VA, USA). The images were processed and analyzed (n = 3, 2000 cells per SGBS sample) by our high-throughput automatic cell-recognition protocol with some modifications using the iCys companion software (iNovator Application Development Toolkit version 7.0, CompuCyte Corporation, Westwood, MA, USA). As previuosly, cells were identified according to their Hoeschst 33342 nuclear staining. Then, the fluorescence signal intensity of the UCP1 immunostaining and the texture sum variance of the light scatter signal of lipid droplets were quantified in each cell within the 30-pixel immediate outward vicinity of the nucleus contour by the Cell Profiler software (The Broad Institute of MIT, MA, USA). Afterward, based on these fluorescence and light scatter signal of single cells, a semi-automated classification and enumeration of the differentiated white and brown adipocytes and undifferentiated preadipocytes was carried out applying the trained classification “Fast Gentle Boosting” of the Cell Profiler Analyst software (The Broad Institute of MIT, MA, USA).<SEP>Flow cytometry<SEP>To investigate the phenotype of the undifferentiated SGBS preadipocytes, a multiparametric analysis of surface antigen expression was performed by three-color flow cytometry using fluorochrome-conjugated antibodies with isotype matching controls. See reference ref. for further details about the analysis.<SEP>Genotyping<SEP>Genomic DNA was purified with GeneJET Genomic DNA Purification Kit (Thermo Scientific) according to the manufacturer’s protocol. Rs1421085 SNP (single nucleotide polymorphism) was genotyped by qPCR using TaqMan genotyping assays and by DNA sequencing. To amplify the corresponding genomic region, we designed the following primer pair: Forward: 5′GATGACACACACCATGAGCC, Reverse: 5′TAACAGTGGAGGTCAGCACA. Following PCR amplification, we purified the PCR product with NucleoSpin® Gel and PCR Clean-up kit (Macherey-Nagel, Germany). Then the quality of the product was investigated by 2% agarose gel electrophoresis. DNA was sequenced by Sanger sequencing method.<SEP>Statistical analysis<SEP>Results are expressed as the mean ± SD for the number of assays indicated. For multiple comparisons of groups statistical significance was calculated and evaluated by one-way ANOVA followed by Tukey post-hoc test. To compare two groups, two-tailed paired Student’s t-test was used.",
                  "Results<SEP>SGBS cells express surface markers similarly to primary preadipocytes and are heterozygous for the FTO risk allele rs1421085<SEP>Primarily, we examined undifferentiated SGBS cells by surface antigen expression analysis. We found that hematopoietic/monocyte markers (CD34, CD47), endothelial markers (CD54), fibroblast markers (CD73, CD90), integrins and CAMs (integrin ß1, CD44, CD325) were expressed on the surface of undifferentiated SGBS preadipocytes (Supplementary Fig. S1a). Then, we compared the surface antigen expression pattern of SGBS preadipocytes to SVF cells isolated from human abdominal subcutaneous fat. Most of the investigated markers were expressed similarly in SGBS and primary preadipocytes. However, CD34, CD44, CD146 and HLA-DR expression levels were higher in SGBS preadipocytes, while CD105, CD49a and CD31 antigens were expressed at a lower level compared to primary preadipocytes (Supplementary Fig. S1a). Next, we tested the presence of the C risk-allele of the rs1421085 locus; DNA sequencing (Supplementary Fig. S1b) and qPCR-based genotyping analysis (data not shown) determined that SGBS cells are heterozygous for the C risk allele.<SEP>SGBS preadipocytes respond to sustained PPARγ ligand and irisin or BMP7 treatment by inducing either beige or classical brown marker genes<SEP>We applied previously described white (initiated by four days treatment with the PPARγ-ligand rosiglitazone) and browning (with the continuous presence of rosiglitazone during differentiation) protocols to differentiate SGBS preadipocytes and compared the expression of selected thermo- and adipogenic marker genes in the two settings. The browning cocktail highly induced Ucp1 mRNA expression. Similarly, the presence of human recombinant irisin or BMP7 on the top of the white differentiation protocol resulted in enhanced Ucp1 mRNA expression; presence of irisin or BMP7 in the browning cocktail did not increase Ucp1 expression further (Fig. 1a). mRNA of brown-fat specific genes, like Cidea and Elovl3 were also enriched during the administration of the browning cocktail and when irisin was added to the white differentiation cocktail (Fig. 1b). In contrast, we observed decreased expression of Lep, a white adipocyte marker gene, in response to the browning differentiation (the effect of irisin and BMP7 was not significant in that respect). The expression of other adipogenic markers, Cebpb and Pparg, did not differ between white and browned SGBS adipocytes. Fabp4 was expressed at a significantly higher level in browned adipocytes compared to the white ones. Out of these markers, only the expression of Pparg was increased in response to irisin or BMP7 treatment (Supplementary Fig. S2).<SEP>The expression levels of the mitochondrial enrichment marker, Cyc1 and the master regulator of mitochondrial biogenesis, Pgc1a were significantly higher in browned SGBS cells compared to white adipocytes and irisin treatment had the same effect (Fig. 1c). In the undifferentiated SGBS preadipocytes we could detect high mitochondrial DNA content. Differentiated white adipocytes have relatively lower mitochondrial DNA content and irisin treatment resulted in significantly elevated mitochondrial DNA amount in them while the effect of BMP7 was moderate. The mitochondrial DNA amount was the highest in the case of browned cells after the application of the PPARγ-driven browning differentiation cocktail (Fig. 1d).<SEP>Next, we asked the question whether the beige-selective marker genes, including Tbx1 and Cited1 or classical brown adipocyte markers, like Zic1 were induced in SGBS cells. Irisin or the browning protocol resulted in marked upregulation of Tbx1 and Cited1 but no Zic1 induction. There was no further increase of Tbx1 and Cited1 expression when irisin was added on top of the browning protocol. BMP7, on the other hand, only upregulated Zic1 markedly and even prevented Tbx1 induction when it was combined with the browning differentiation cocktail (Fig. 1e). The expression of Tmem26 and Shox2, proposed beige-markers reported in mouse studies, was not elevated significantly in the browned SGBS adipocytes as compared to the white ones (Supplementary Fig. S3). Furthermore, Tnfrsf9 (CD137) was expressed markedly only in undifferentiated preadipocytes (data not shown). Interestingly, Pdk4, originally described as a classical brown marker in mice, showed a similar expression pattern as the beige-selective Tbx1 and Cited1, in SGBS adipocytes (Fig. 1e).<SEP>Then, we determined the concentration dependence of the irisin and BMP7 effects during white adipocyte differentiation in SGBS cells. Jedrychowski et al. found that irisin was present at 3–4 ng/ml concentration in human blood plasma in the case of sedentary life but increased to 4–5 ng/ml if individuals took part in aerobic training. Itgav, the gene which encodes the specific subunit of the recently identified irisin receptor is expressed both in undifferentiated SGBS preadipocytes and in differentiated adipocytes to the same extent (Supplementary Fig. S3). At 5 ng/ml concentration of irisin only a slight increase could be observed in Ucp1 expression compared to untreated cells. However, irisin was efficient above 50 ng/ml concentration. The same trend was observed when the expression of Tbx1, Cyc1, Elovl3 and Pgc1a was analyzed (Supplementary Fig. S4). The supplementation of BMP7 to the white adipogenic differentiation media in increasing concentrations resulted in upregulated expression of Ucp1 and Cidea brown marker genes compared to the untreated white adipocytes. In contrast, we observed downregulation of Lep at 100 ng/ml BMP7 concentration. However, we could not detect significant changes in the expression of Tbx1, Cyc1, and Elovl3 as a result of BMP7 treatment (Supplementary Fig. S5).<SEP>As a next step, we investigated the morphological characteristics of the white and browned SGBS cells by assessing the textural parameters and UCP1 protein content of the individual adipocytes using laser-scanning cytometry (Fig. 2a, Supplementary Fig. S6). We found higher UCP1 content in single SGBS adipocytes (Fig. 2b) and in total cell lysates (Fig. 2c) as a result of the 14 day-long PPARγ-driven browning differentiation, as compared to white. Moreover, we detected increased UCP1 protein content in response to irisin or BMP7 administration (Fig. 2b). In contrast to primary adipocytes the texture sum variance did not change significantly during the induction of browning marked by higher UCP1 expression (Fig. 2d).<SEP>Differentiated SGBS adipocytes respond to activation as functional beige cells using the creatine phosphate cycle<SEP>To test the functional capacity of differentiated SGBS adipocytes, mitochondrial oxygen consumption rate (OCR) was measured (Fig. 3a,b). We found higher basal OCR after the browning was induced by either PPARγ ligand, irisin or BMP7. When we stimulated adipocytes with cAMP, the OCR of browning adipocytes was elevated more robustly compared to the white adipocytes (Fig. 3b). A similar trend was observed as a result of oligomycin treatment which is an inhibitor of H+-ATP synthase and provides the measurement of proton leak respiration (Fig. 3b). In parallel, we detected significantly elevated extracellular acidification rate (ECAR) both in untreated and cAMP-stimulated browned adipocytes as compared to white cells (Fig. 3c).<SEP>We found it important to determine what proportion of cAMP induced OCR is related to the UCP1 independent creatine phosphate futile cycle, a characteristic feature of beige adipocytes, utilizing ß-guanidinopropionic acid (β-GPA) which is an inhibitor of this cycle. The creatine-cycle related OCR was markedly higher in case of the browned SGBS cells, as well as in irisin and BMP7 treated adipocytes, in contrast to untreated white adipocytes (Fig. 3d). Ckmt1a and Ckmt2 mitochondrial creatine kinases were expressed at a greater extent in the browned SGBS adipocytes as compared to those which were differentiated to white. The expression level of these genes was below the detection limit in undifferentiated preadipocytes (Fig. 3e). This suggests that the creatine kinase futile cycle is active in browned SGBS adipocytes induced either by PPARγ-driven browning differentiation or by the administration of irisin and BMP7 during white differentiation, demonstrating that the browned SGBS adipocytes functionally resemble beige cells.<SEP>Notably, Pm20d1 which encodes the secreted enzyme peptidase M20 domain containing 1 was induced in response to the administration of the browning cocktail and when irisin was added to the white differentiation cocktail (Supplementary Fig. S3). This means that browned SGBS adipocytes have the potential to produce N-acyl amino acids which can function as endogenous uncouplers of UCP1-independent thermogenesis.<SEP>The brown/beige adipocyte phenotype of differentiated SGBS cells is maintained in the absence of PPARγ-ligand<SEP>In order to determine whether the PPARγ-driven differentiation could maintain a beige phenotype on a longer time frame, we performed a long-term experiment after 2 weeks for 21 or 28 days; and in parallel samples the browning cocktail was replaced by the white differentiation cocktail for the third (B14,W7) and the fourth week (B14,W14). The highest Ucp1 mRNA expression was detectable at the end of the long-term browning differentiation program, at day 28 (Fig. 4a). Ucp1 mRNA level was increased when we replaced the browning cocktail to white, at least until day 21, then sank to the level of day 14. UCP1 protein expression was detected by Western blot at day 14 of PPARγ-driven differentiation and it was further enhanced by after one or two weeks of further stimulation that is long-term rosiglitazone induction resulted in robust upregulation of UCP1 at the protein level. When the PPARγ stimulus was removed after 2 weeks of browning differentiation program, the expression level of UCP1 protein was slightly elevated by the end of the third week, and it was still detectable one week later at a decreased level (Fig. 4a, Supplementary Fig. S7). This was verified with 3 independent anti-UCP1 antibodies (Supplementary Fig. S8).<SEP>Parallely, we investigated the expression of Cidea and Elovl3 brown-fat specific genes; similarly to Ucp1, further upregulation was observed when we continued the administration of the browning protocol; when PPARγ-stimulation was withdrawn, their expression levels remained elevated as compared to B14 (Fig. 4b). The replacement of the cocktails from browning to white without rosiglitazone did not induce de novo white adipogenesis, marked by the continuous low expression of Lep (Fig. 4c).<SEP>After four-weeks of differentiation, we observed by immunohistochemistry a six-fold higher UCP1 protein content as a result of the browning cocktail compared to white. When the PPARγ-agonist was replaced by the white differentiation protocol for the last 2 weeks the UCP1 protein content per cell did not change significantly (Fig. 4d) as compared to adipocytes maintained continuously in the presence of the PPARγ-driven differentiation regimen. In parallel, similar texture sum variance was detected in both cell populations (Fig. 4d).<SEP>Mitochondrial oxidative phosphorylation (OXPHOS) plays a fundamental role in energy production in brown/beige adipocytes. We found that protein expression of mitochondrial respiratory components in complex II, III and IV followed the pattern of UCP1 expression in SGBS adipocytes (Fig. 4e, Supplementary Fig. S7) that is higher expression of respiratory chain proteins were found in browned adipocytes compared to control white adipocytes indicating stimulated mitochondrial biogenesis. The withdrawal of PPARγ agonist resulted in only a slight reduction of the level these proteins suggesting the clearance of a small fraction of the mitochondria. These results suggest that the signaling pathways driving the beige phenotype in SGBS cells can be maintained at least for two consecutive weeks after the PPARγ agonist is removed.",
                  "Discussion<SEP>hADMSCs isolated from SVFs are the most widely used ex vivo system to study human adipocyte browning. Recently, we showed that hADMSCs from subcutaneous fat could be differentiated into beige adipocytes and we observed upregulation of brown-specific marker proteins (UCP1, CIDEA) in single adipocytes by using laser-scanning cytometry. Furthermore, we demonstrated that irisin could induce beige type of browning when added during white adipocyte differentiation as indicated by Tbx1 induction while BMP7 addition resulted in Zic1 induction pointing to the possibility of the classical brown adipocyte differentiation pathway. However, the major limitations of experiments using hADMSCs are the restricted availability of biopsy material and the potentially large variability between donors, which may greatly affect the reproducibility of the obtained data. Fat donors may vary in nutrition, metabolism, life-style and genetic background, e.g. the presence of the risk-allele of the FTO locus (rs1421085), which fundamentally affects adipocyte browning and represents robust genetic association with obesity.<SEP>Recently, several attempts were reported to overcome the limitations described above. Artificially immortalized cell lines, like PAZ6 cells, which were obtained from SVF of human infant BAT, and transformed with SV40 T and t antigens were able to differentiate into brown adipocytes in vitro. PAZ6 is considered to be a brown preadipocyte cell line, because high UCP1 expression was shown in differentiated PAZ6 cells which could be further induced by norepinephrine. Shinoda et al. compared the differentiation of 65 clonal preadipocyte lines originated from supraclavicular fat biopsies with 35 lines which were generated from the SVF of subcutaneous WAT. RNA sequencing was performed from three clonal brown and white adipocyte cultures, differentiated from supraclavicular and subcutaneous progenitor clones, respectively. They found that human browned adipocytes from supraclavicular regions display molecular signatures which resemble thermogenic beige adipocytes. In another study, clonally derived adipocyte progenitors from the deep neck area were capable of differentiating to functional thermogenic adipocytes and responded to BMP7 administration, in contrast to those which were isolated and generated from subcutaneous WAT.<SEP>The SGBS human preadipocyte cell line is often used in experiments to study human adipocyte biology and previously it served as a representative model of white adipocyte differentiation. The first results, showing the possibility that SGBS cells can differentiate into thermogenic adipocytes were described in 2013. 2-oxoglutarate dependent dioxygenase (encoded by the proposed Fto gene) deficient SGBS preadipocytes differentiated to white adipocytes had elevated UCP1 expression and uncoupled respiration, without any changes in mitochondrial mass or structure. Later, Tews et al. found that TENM2 (teneurin-2), which might inhibit the classical brown marker, Zic1, is enriched in white adipocyte progenitor cells from subcutaneous neck WAT compared to deep neck ones. They demonstrated that TENM2 knockdown with siRNA in SGBS cells resulted in increased UCP1 at mRNA and protein level. Furthermore, basal and proton leak mitochondrial respiration was enhanced. Interestingly, TENM2 deficient SGBS adipocytes had the same amount of mitochondria as the wild type ones and gained larger lipid droplets than the control cells. The aforementioned results suggest that TENM2 knockdown in SGBS adipocytes results in a beige gene expression program (the expression of Zic1 did not change) without enhanced mitochondrial biogenesis and accumulation of small lipid droplets in a multilocular arrangement. In these studies, SGBS cells differentiated to white adipocytes for two weeks expressed UCP1 at low, but detectable levels which we could confirm in the present study. In response to the knockdown of 2-oxoglutarate dependent dioxygenase or TENM2 or the administration of the atypical antipsychotic clozapine, the browning potential of the SGBS cells could be significantly enhanced.<SEP>In contrast to these studies and to results presented here, Guennoun et al. found that the SGBS adipocytes switch their phenotype during a four week-long differentiation program. Within two weeks, they noticed high UCP1 expression and a thermogenic phenotype even as a result of the white differentiation protocol without adding any browning stimuli. Then, the expression of UCP1 greatly declined in response to the continuation of the white differentiation for additional two weeks. Yeo et al. seemingly strengthened these results by comparing the gene expression pattern of the differentiated SGBS and primary hADMSC-derived subcutaneous adipocytes. Of note, hADMSCs in this case were obtained from obese patients and there might be a difference between the differentiation capacity of these and healthy hADMSCs and SGBS preadipocytes. They also observed that UCP1 protein expression was high in SGBS cells differentiated for 12 days even when either rosiglitazone or T3 were omitted from the media. In these experiments they considered a band on immunoblots with an apparent molecular mass of approximately 25 kDa as UCP1.<SEP>In our experiments, however, we could demonstrate UCP1 protein expression in response to long-term rosiglitazone and T3 administration (browning protocol) at ~33 kDa, but not when the white cocktail was applied. In order to confirm that we take the right band for UCP1, three kinds of anti-UCP1 (both monoclonal and polyclonal) antibodies were used to investigate the UCP1 protein expression. The application of polyclonal antibodies (U6382, PA1-24894) resulted in three distinct bands at ~36 kDa, ~33 kDa and below 28 kDa (Supplementary Fig. S8). However, when a monoclonal antibody (MAB6158) was applied, we could clearly demonstrate which band corresponds to the UCP1 protein, supported by its appearance at the predicted molecular weight of 33 kDa. Furthermore, UCP1 expressed in a mouse BAT lysate was detected at the very same molecular weight as the UCP1 band in the browning SGBS adipocyte samples (Supplementary Fig. S8).<SEP>Using laser-scanning cytometry and applying functional assays we aimed to investigate whether classical brown or beige adipocyte differentiation (browning) can be induced in SGBS cells and if these adipocytes maintain their morphology during long-term culturing. The browning of SGBS cells was induced by multiple approaches: sustained PPARγ stimulation, irisin and BMP7 treatment. Irisin induces browning of subcutaneous white adipocytes ex vivo, BMP7 drives brown fat cell differentiation of both mesenchymal progenitor cells and committed brown preadipocytes. In vivo, BMP7 is able to increase brown fat mass and thermogenic energy expenditure in mice. The present study demonstrates that the PPARγ-driven browning protocol (including rosiglitazone) and irisin treatment could be successfully used to induce browning of SGBS adipocytes, which resulted in a beige phenotype. BMP7 administration had a more moderate effect and induced a distinct gene expression program without the upregulation of the beige-selective markers, Tbx1 and Cited1. Both in primary subcutaneous and in SGBS adipocytes, BMP7 treatment enhanced the expression of the Pparg and Zic1 genes suggesting that this mediator rather induce a classical brown-like differentiation. This program, similarly to irisin administration, resulted in increased mitochondrial DNA and Ucp1 levels and elevated mitochondrial respiration. Of note, the majority of the browning marker genes were detectable in white SGBS adipocytes but their expression remained markedly lower compared to the browned cells.<SEP>The functional experiments detected UCP1-dependent proton leak (heat production) and high extracellular acidification in the case of beige adipocytes induced by either rosiglitazone or irisin. These parameters were significantly lower in the case of white adipocytes, however, could be stimulated in response to cAMP supporting the notion that some of the differentiated white adipocytes are masked beige cells. Besides UCP1-dependent thermogenesis, UCP1-independent heat-producing mechanisms were described as a beige specific feature. In our experiments, the involvement of the creatine-phosphate futile cycle was investigated in the heat production of SGBS beige adipocytes. Using a specific compound (β-GPA) which depletes creatine and inhibits the cycle, we demonstrated the induction of this futile cycle in response to a β-adrenergic cue in beige SGBS adipocytes.<SEP>Several single nucleotide polymorphisms located in the fat mass and obesity-associated gene, Fto were described by genome-wide association studies as a genetic trait predisposing for obesity. The T-to-C conversion at rs1421085 position affects a mesenchymal super-enhancer site and leads to increased IRX3 and IRX5 expression levels during early adipocyte differentiation, because of the lack of ARID5B binding and repression, which results in a developmental shift from energy-dissipating beige cells to energy-storing white adipocytes. Therefore, the potential of mitochondrial thermogenesis is reduced in those individuals who carry the C risk allele which occurs at a high frequency (approx. 44%) in the European population. So far no information has been available about the status of this allele in SGBS cells. A CC to TT rescue by genome editing of primary adipocytes resulted in an elevated Ucp1 expression by two and an increased stimulated oxygen consumption by seven fold. To our knowledge, no data is available how a heterozygous composition affects these parameters. In this study, we demonstrated that the SGBS cell line carries one copy of the C risk allele; still these preadipocytes were able to differentiate into functional beige adipocytes if browning stimuli (e.g. rosiglitazone or irisin treatment) were continuously present during their differentiation.<SEP>In summary, the induction of browning in SGBS cells followed the beige pathway, as a result of the PPARγ-driven differentiation (rosiglitazone) or irisin treatment. The beige phenomenon is supported by the elevated expression of brown and beige marker genes, multilocular morphology, enrichment of mitochondria, high oxygen consumption in response to cAMP and significant involvement of the creatine-phosphate futile cycle in heat production. In our experiments, when the browning-inducer, rosiglitazone was eliminated from the differentiation media, UCP1 expression and mitochondrial enrichment could be partially maintained. In summary, we propose that the human SGBS preadipocyte cells represent an uncommitted preadipocyte stage with one FTO risk allele, which are able to differentiate into both white and beige adipocytes and can be used as an easily applicable and valuable model to study human adipocyte browning in a reproducible as well as high-throughput manner contributing to the development of novel therapeutic approaches against obesity.",
                  "38998",
                  "['OBWCCE']"
                ],
                [
                  "46",
                  "30565958",
                  "INTRODUCTION<SEP>Elevated serum triglyceride (TG) level is an independent risk factor for T2DM incidence, even after controlling for body mass index (BMI) and all other conventional risk factors . From a clinical perspective, elevated TG levels may be observed as much as 10 years before diagnosis of T2DM . In conventional analysis, TG is substantially confined to the chylomicron and very low density lipoprotein (VLDL) fractions, collectively termed as triglyceride rich lipoproteins particles (TRLP). However, TRLPs are highly heterogeneous, and technological developments using nuclear magnetic resonance (NMR) allow detection of five discrete TRLP subfractions including very large, large, medium, small, and very small as defined by lipoprotein particle diameter , each of which may have distinct clinical properties. Therefore, further delineating associations between TG with T2DM according to TRLP subfractions may provide clinical insight.<SEP>Since 2007, through genome-wide association studies (GWAS), about 40 independent genetic loci have been discovered for association with TG levels . When combined as a genetic risk score (GRS), the TG-raising alleles at these loci provide a quantitative and continuous measure of genetic susceptibility to elevated TG levels. Recently, Klimentidis et al.,  reported a significant interaction between a TG-specific weighted GRS (TG-wGRS) and baseline TG for association with T2DM risk. The authors observed that TG was more strongly associated with T2DM incidence in low compared to high TG-wGRS individuals. Klimentidis et al.,  and Li et al.,  also reported that many of the TG-associated genetic variants at or near CILP2, GCKR, NAT2, PLA2G6, CETP and APOA1 genes were positively associated with TG but negatively associated with glycemic traits and T2DM. Meanwhile, formal Mendelian randomization analysis using TG-specific genetic instruments found no evidence of causal association between plasma TG and T2DM, and instead suggested a protective effect of genetically raised TG levels on T2DM .<SEP>In the current study, we seek to confirm and further explore these paradoxical findings by using the large, prospective population based sample of middle-aged women from the Women’s Genome Health Study (WGHS) to evaluate whether established TG-associated loci modify the association of baseline TG with the incidence of T2DM over a mean 18.6 years of follow-up. We also aimed to investigate whether the relationship between TG with T2DM risk can be refined to specific TRLP subfractions, potentially informing which biological aspects of lipid metabolism influence development of T2DM.",
                  "MATERIALS AND METHODS<SEP>Study Population<SEP>Individual level data for the WGHS is not currently publicly available but scripts for performing the analysis described in this report will be provided upon request. The study participants of the WGHS are derived from the Women’s Health Study (WHS) that was designed as a placebo-controlled randomized trial evaluating the impact of low-dose of aspirin and vitamin E for the primary prevention of CVD  among initially healthy middle age (45 years or older) health-care professionals. Subsequent to the conclusion of the trial, the study population has been followed in observational mode. The WGHS consists of the approximately 72% of WHS participants with a baseline blood sample. For the current analysis, we have 23,294 participants with available genotype data and self-reported European ancestry that was also confirmed through genetics . Klimentidis et al.,  performed analyses in the fasting sample, and considering this, for the current primary analyses we have 15,813 of 21,848 participants without diabetes at baseline who also had more than 8 hours of fasting status at baseline. Secondary analyses were performed among all participants (fasting and non-fasting). Analysis in the WGHS was approved by institutional review board of Brigham and Women’s Hospital, Boston.<SEP>At baseline, clinical characteristics were collected through self-reported questionnaire regarding age and body mass index (BMI), as described previously . All lipid levels including TG and HDL cholesterol (HDL-C) were enzymatically measured through Hitachi 917 analyzer (Roche Diagnostics) using baseline plasma. TGs were measured enzymatically with correction for endogenous glycerol.<SEP>Incident T2DM ascertainment<SEP>Incident T2DM cases were initially identified by self-report using annual questionnaires that asked if the participant had been diagnosed with diabetes since baseline . Self-reported incident T2DM cases were confirmed through physician-administered telephone interviews or the questionnaires that inquired about clinical manifestations of diabetes including diabetes medications, diagnostic diabetes testing and disease symptoms .<SEP>Measurement of Triglyceride-rich Lipoprotein (TRLP) Subfractions<SEP>NMR-based measures were performed for quantifying TRLPs, as has been reported previously . Briefly, H-NMR spectra (400 MHz) of plasma were deconvoluted with the LipoProfile IV assay (LipoScience) to resolve concentrations (nmol/L) of distinct TRLP subfractions, differentiated according to diameter, i.e. particle size. For the current analysis, we considered very large (90–240 nm), large (50–89 nm), medium (37–49 nm), small (30–36 nm) and very small (24–29 nm) TLRPs. Mean particles size (TRLZ) ranged from 30–100 nm. The TG and cholesterol content of the combined TRLPs, designated as TRLTG and TRLTC respectively, were also determined by NMR.<SEP>Genotyping and Genetic Risk Score<SEP>In the WGHS, genotyping was performed with the HumanHap300 Duo array using the Infinium II assay . Imputation for the SNPs that were not present in the genotyping array was performed using MACH (v. 1.0.16) using the 1000 genomes phase I v. 3 (March 2012) cosmopolitan reference panel. For the current analyses, we used 40-TG associated genetic variants that have been previously reported in association with TG , as our primary genetic predictor regarding analyses because 40-TG associated variants are similar to the 31-TG associated genetic variants in the publication by Klimentidis et al. In addition, we performed secondary analyses using recently published 127-TG associated genetic variants. Most of the SNPs were directly genotyped. The SNPs that were imputed had high imputation quality score (MACH r≥0.85). For the 40 TG-associated SNPs, effect allele frequencies, chromosomal position and gene names have been described previously and for the 127-TG associated SNPs, this information is shown in Supplemental Table I. Weighted genetic risk scores (TG-wGRSs) were created using published effect estimates from the 40-TG (TG-wGRS40) and 127-TG (TG-wGRS127) associated genetic variants separately and scaled to such that each unit reflects 10 alleles, as described previously.<SEP>Stastical Analysis<SEP>Cox proportional regression models were used to assess the association of TG, TRLPs, SNPs, and the TG-wGRSs with the incident T2DM. Interaction analyses were performed by additionally introducing a term of TG-associated SNPs or the TG-wGRSs × TG in the Cox models. For interaction analyses, the TG-wGRSs were used as continuous variables or as a tertiles (coded as 0, 1 and 2). TG was used as a continuous variable. The basic models were adjusted with principle components (PCs) (to control for European population substructure), age, and randomization to treatment in the WHS parent cohort (vitamin E and aspirin). The fully adjusted models included covariates from the basic model plus measures of HDL-C and BMI, similar to Klimentidis et al.,. All clinical covariates were determined at baseline. For the single SNP analysis, we considered a significant threshold p≤0.00125 (=0.05/40) after Bonferroni correction for the 40 single SNPs tests performed. Very large and large TRLP variables were not normally distributed as the distribution of these variables were left skewed. To assess the significance of associations with these variables, we used permutation procedures to derive empirical association P-values. We repeated the Cox regression over 10,000 iterations, each time resampling the TG-wGRS40 at random without replacement to generate a null distribution of beta coefficients. The empirical p-values were derived as the proportion of beta coefficients from the empirical null distribution with absolute value greater than the absolute value of the beta coefficient from the unpermuted model. Also, because many of the study participants have lower even undetectable concentrations of very large TRLP particles, we created variable combining the very large and large TRLP particles and tested that combined variable in interaction with TG-wGRS40 in predicting T2DM incidence. In sensitivity analyses, we generated trait residuals by regressing TG (and other TRLPs) against the TG-wGRS40 to eliminate collinearity, and used the residuals as an interactor variable with the TG-wGRS40 to assess association with T2DM. To investigate whether interactions with the TG-wGRS40 were consistent with the implicit assumption that the ratio of SNP effects between strata of TG was similar across all SNPs, we regressed SNP effects for incident T2DM (as beta coefficients) in the group with TG below or equal to median TG against the group with TG above median. SNPs with disproportionate influence on the regression relationship were identified using the R function “influence.measures” that flags extreme points for any of a series of regression diagnostics, including Cook’s D, leave-one-out analysis, covariance ratio, and leverage.",
                  "RESULTS<SEP>Clinical and general characteristics of the study participants are shown in Table 1. For the current analyses, there were N=15,813 with fasting status (and N=21,848 in total) female participants from the WGHS among whom 1,453 experienced incident cases for T2DM during follow-up (N=1,944 incident T2DM cases in total sample). The mean follow-up duration for T2DM incident was 18.6 (SD=5.3) years.<SEP>Associations of TG, TG-wGRS and incident T2DM<SEP>Although 40 SNPs have been identified through GWAS for TG, Klimentidis et al.  performed analysis using a TG-wGRS based on 31 TG-associated SNPs as they observed that this TG-wGRS was more strongly associated with TG. However, by reviewing the literature, we found that three SNPs, COBL1 rs10195252, APOB rs1042034, and APOE rs439401 were not primarily or secondarily associated with TG in the most recent GWAS . We therefore constructed a TG-wGRS and performed all of our primary analyses using 40-TG associated SNPs after excluding these three inconsistent SNPs but including 12 additional SNPs (PIGV-NR0B2 rs12748152, LRPAP1 rs6831256, VEGFA rs998584, RSPO3 rs719726, MIR148A rs4719841, MET rs38855, AKR1C4 rs1832007, PDXDC1 rs3198697, FTO rs1121980, MPP3 rs8077889, INSR rs7248104 and PEPD rs731839). All of the included genetic variants were primarily or secondarily associated with TG at genome-wide significance .<SEP>The TG-wGRS40 was positively associated with TG (β[SE]=0.38[0.01] mmol/L per 10-TG associated risk alleles, P-value≤0.0001) when the regression model was adjusted with age, BMI, HDL-C and PCs. TG was positively associated with T2DM incidence (HR [95% CI] =1.70[1.63, 1.77] per mmol/L, P-value≤0.0001) when the regression analysis was adjusted with age and PCs. The association of TG with T2DM was attenuated (HR [95% CI] = 1.37[1.30, 1.43] per 1mmol/L, P-value≤0.0001) when the regression analysis was additionally adjusted with BMI and HDL-C (Table 2). In a Cox model adjusted with age and PCs, we observed that the TG-wGRS40 was associated with a non-significant reduction in T2DM incidence (HR [95% CI] =0.94 [0.83, 1.06] per 10-TG associated risk alleles, P-value=0.30). However, when the model was additionally adjusted for baseline age, TG, BMI and HDL-C, the TG-wGRS40 was significantly associated with reduced T2DM incidence (HR [95% CI] =0.66 [0.58, 0.75] per 10-TG associated risk alleles, P-value ≤0.0001). Similarly, we observed that the TG-wGRS127 was negatively but non-significantly (HR [95% CI] =0.98 [0.93, 1.03] per mmol/L, P-value= 0.39) associated with the incidence of T2DM when the analyses were adjusted with basic model covariates. In the fully adjusted regression models, TG-wGRS127 was significantly negatively associated with T2DM incidence (HR [95% CI] =0.87 [0.83, 0.92] per mmol/L, P-value≤0.0001).<SEP>We observed a significant interaction of baseline TG levels and the TG-wGRS40 (Pinteraction= 0.002) for incident T2DM when the Cox model was adjusted for age and PCs. This interaction was also observed when the regression model was additionally adjusted for baseline BMI and HDL-C (Pinteraction=0.030) (Table 2). Through residual trait analysis, we observed that the TG-wGRS40 explained 4.99% trait variance in TG, similar to previously reported estimates . When we replaced the TG with GRS-residualized TG in the analysis, the TG-wGRS40 × trait-residual interactions for incident T2DM remained significant (Pinteraction= 0.005 for the basic model and Pinteraction= 0.05 for the fully adjusted model). In the Cox model adjusted for age and PCs, we observed that TG was associated with higher incidence of T2DM in the low TG-wGRS40 tertile (HR [95% CI] =1.98 [1.83, 2.14] per mmol/L) compared to the high TG-wGRS40 tertile (HR [95% CI] =1.68 [1.58, 1.80] per mmol/L), with a statistically significant interaction (Pinteraction=0.0007) (Figure S1). Similar interactions were observed when the regression models were additionally adjusted with BMI and HDL-C (Table 3). Similar to the finding with the TG-wGRS40, we observed significant interactions of TG-wGRS127 and TG levels in the risk of T2DM in the basic model (Pinteraction=0.0157) and in the fully adjusted model (Pinteraction=0.0432) (Table 2)<SEP>TG-rich lipoproteins, TG-wGRS and T2DM risk<SEP>In Cox models adjusted for basic covariates, total TRLP, very large, large, medium, very small TRLPs, TRLZ, TRLTG and TRLTC (except small TRLP) were significantly associated with T2DM (all P-values ≤0.0001), as has been published previously in a similar WHS sample . Regarding the TG-wGRS40, when the regression models were fully adjusted for BMI and HDL-C along-with total TG at baseline, we observed significant interactions between TG-wGRS40 and large TRLP (Pinteraction=0.016) and medium TRLP (Pinteraction=0.047) with T2DM (Table 2). No significant interactions were observed for very large (Pinteraction= 0.107), very small (Pinteraction=0.891) or small (Pinteraction=0.828) TRLPs. Similarly, a variable combining large and very large TRLP particles showed significant interaction with TG-wGRS40 in the incident T2DM (Pinteraction =0.0035). For both the large TRLPs and the combination of large and very large TRPLs, in models adjusted with basic covariates, we also observed significant interaction using a resampling approach to derive an empirical procedure to address the skew in the TRLP distribution (Pinteraction, empirical = 0.0024 and 0.0057, respectively). Similarly, we observed concordant interactions for the different TG-rich lipoprotein subfractions (as shown in Table 2).<SEP>Accordingly, we observed a stronger association of large TRLPs with incident of T2DM among individuals in the lower TG-wGRS40 tertile (HR [95% CI] =1.02 [0.97–1.06]) compared to the individuals in the higher TG-wGRS40 tertile (HR [95% CI] = 0.97 [0.93–1.01]) with a significant interaction (Pinteraction=0.013). Similar interactions were observed regarding medium TRLPs (Pinteraction= 0.012) (Table 3). However, the interactions with tertiles of TG-wGRS40 were non-significant for small and very small TRLPs (Pinteraction= 0.620) and (Pinteraction= 0.648), respectively. We observed materially similar results when the analyses were performed in the total WGHS sample, including both fasting and non-fasting participants (Supplemental Table I).<SEP>Justification of using a genetic risk score for testing interactions with TG<SEP>One assumption in using a genetic risk score (GRS) for gene-environment interaction studies, is that the ratio of individual SNP effects across environmental strata should be approximately similar for all SNPs. We tested this assumption by comparing SNP-T2DM association between two strata of TG (dichotomous variable was created based upon the median value: low ≤ 1.29 mmol/L vs high > 1.29 mmol/L). In a linear model of SNP effects in the two strata, we observed that four of the SNPs (KLHL8 rs442177, APOA1 rs964184, CAPN3 rs2412710 and CILP2 rs10401969) had disproportionate influence on the regression fit (Supplemental Table II and Figure S2). We performed sensitivity analysis of TG-wGRS40 × TG interactions in T2DM after removing the most influential SNPs (KLHL8 rs442177, APOA1 rs964184, CAPN3 rs2412710 and CILP2 rs10401969) from the TG-wGRS40 and observed significant interactions with TG (Pinteraction=0.016) for the basic model but not for the fully adjusted model (Pinteraction=0.27).<SEP>Pleiotropic effects<SEP>In order to reduce the potential influence of known pleiotropic effects on our conclusions, we removed genetic variants from the TG-wGRS40 that associate at genome-wide significance with T2DM related traits (GCKR rs1260326 [T2DM, fasting glucose, beta-cell function, fasting insulin], IRS1 rs2972146 [fasting glucose, T2DM, homeostatic model assessment of β-cell function, homeostatic model assessment of insulin resistance], FADS1–2-3 rs174546 [fasting glucose, T2DM, homeostatic model assessment of β-cell function], FTO rs1121980 [T2DM], CILP2 rs10401969 [T2DM] and PEPD rs731839 [T2DM]). As was found for the primary 40 SNP TG-wGRS, this 34 SNP TG-wGRS was associated with a non-significant reduction in T2DM incidence (HR [95% CI] =0.96 [0.84, 1.10] per 10-TG associated risk alleles, P-value=0.57). However, in models additionally adjusted for baseline TG, HDL-C and BMI, the 34 SNP TG-wGRS was negatively associated with T2DM (HR [95% CI] =0.67 [0.58, 0.77] per 10-TG associated risk alleles, P-value ≤0.0001). Similar to the 40 SNP TG-wGRS, we observed significant interactions of the 34 SNP TG-wGRS with very large TRLPs (Pinteraction=0.016) and large TRLPs (Pinteraction= 0.001) for association with T2DM outcomes (Supplemental Table III). In order to remove any of the potential pleiotropic effects from the TG-wGRS127, we excluded 15 genetic variants that associate with T2DM related traits. As with the TG-wGRS127, the 112 SNPs TG-wGRS was associated with non-significant decrease in T2DM (HR [95% CI] =0.97 [0.91, 1.02] per 10-TG associated risk alleles, P-value=0.26) but a significant decrease in the fully adjusted model (HR [95% CI] =0.86 [0.82, 0.91] per 10-TG associated risk alleles, P-value ≤0.0001). As was found for the TG-wGRS127, we observed significant interactions between 112 SNPs TG-wGRS and TG for the T2DM outcome in the basic model (Pinteraction=0.021) or and the fully adjusted model (Pinteraction=0.047).<SEP>Individual SNPs, TG and T2DM incidence<SEP>In single SNP analyses, we observed that 5 (out of 40) SNPs including FADS1–2-3 rs174546, APOA1 rs964184, CETP rs3764261, CILP2 rs10401969 and PLA2G6 rs5756931 showed significantly negative association (P-value≤0.05) with T2DM incidence but after correction for multiple testing, only APOA1 rs964184 remained statistically significant (Table 4). Moreover, 18 of the 40 SNPs were directionally negatively associated with T2DM in models adjusted for age, PCs, BMI and HDL. Regarding single SNPs × TG interactions on T2DM incidence, we observed that a significant excess (P≤0.001) of 9 (out of 40) variants showed nominally significant interactions (P≤0.05) but after correction for multiple testing none of the variants remained statistically significant (Table 4).",
                  "DISCUSSION<SEP>By using data from a well-powered prospective sample of initially healthy US females with homogeneous European ancestry, our study provides a compelling evidence that genetic risk for higher TG derived from a 40-SNP TG-wGRS is associated with lower T2DM risk, contrary to the positive observational associations with TG levels. Further, there was a significant interaction between the TG-wGRS40 and TG levels for predicting incident T2DM such that individuals with lower genetic predisposition to elevated TG had relatively stronger TG-based risk of T2DM than individuals with high genetic predisposition to elevated TG. We observed similar findings when the analyses were performed using a TG-wGRS based upon 127 TG-associated genetic variants. These results confirm findings from Klimentidis et al.,  who studied a TG-wGRS based on 31 SNPs and found both reduced risk of T2DM with increased genetic risk of TG and an interaction between the TG-wGRS and TG levels for T2DM. The 40 SNPs in our TG-wGRS excluded three SNPs from the TG-wGRS of Klimentidis that only had weak evidence of TG association while including 12 additional SNPs with primary association for TG.<SEP>The current study also extends the previously reported discordant effects of a TG-wGRS on TG and T2DM incidence and the interaction of TG with the TG-wGRS in Klimentidis et al., . Our investigation focuses on the interaction effects with TG to the large, and medium but not small or very small TRLPs, even after adjustment with TG. Given the significant interaction of medium and large TRLPs, it may have been surprising that there was no significant interaction with the very large TRLPs (findings are summarized in Figure 1). However, approximately 20 % of the sample had no detectable very large TRLPs leading to lower variability in very large TRLPs compared to the others. The combined variable of large and very large TRLP particles showed significant interaction with TG-wGRS40 for incident T2DM and the significance was affirmed by the empirically p-values.<SEP>Previously, Mora et al.,  using the same sample, reported that large TRLPs were associated with higher risk of T2DM compared to the smaller TRLPs, which was further confirmed in two independent cohorts of multiethnic men and women . Similarly, Garvey et al.,  reported that higher insulin resistance was associated with large but not small TRLPs. Festa et al.,  reported that larger but not smaller TRLPs were associated with increased risk of T2DM among 800 individuals of the Insulin Resistance Atherosclerosis Study who were followed-up for 5-years. Although the interactions persisted in TG adjusted analysis, the biochemical mechanism of our observed interactions may nevertheless be related to the larger amount of TG that is carried by larger compared with smaller TRLPs. Comparing individuals having insulin sensitivity to those with insulin resistant phenotypes, hepatic overproduction of larger TRLPs is a key feature of dyslipoproteinemia of insulin resistance and T2DM . It has been reported through in vivo lipoprotein kinetic experiments that increased larger TRLPs can change the composition of HDL particles through the action of cholesteryl ester transfer protein (CETP) and hepatic lipase, which targets TG, and this process leads to formation of small dense HDL particles and also increased catabolism of these particles . In turn, catabolism of HDL particles is associated with the formation of small density LDL (sdLDL) particles, which have been associated with insulin resistance and T2DM related phenotypes .<SEP>It is common in GRS interaction effects analysis to draw SNP weights in constructing GRS from single SNP marginal association effects. This approach assumes that the magnitude of main and interaction effects across SNPs are highly correlated, and indeed, genetic variants that show main effect association with a trait are generally good candidates for interaction effects . In a previous interaction analysis with adiposity, we have shown that the assumption was largely valid for the TG-wGRS40 in relation to TG as the outcome across normal compared to overweight/obese strata . By adopting that same approach, we demonstrate that the results here were largely valid also for interaction with TG in predicting T2DM, although the assumption of proportionality was challenged for four of the genetic variants (KLHL8 rs442177, APOA1 rs964184, CAPN3 rs2412710 and CILP2 rs10401969). Excluding these SNPs, the interaction of the GRS and TG-wGRS40 for T2DM remained significant.<SEP>The biological and pathophysiological mechanisms through which higher TG-wGRSs were protective for the risk of T2DM, after adjusting with baseline TG and HDL, are not clear. The effects of the TG-associated genetic variants on T2D may reflect their role in hepatic de-novo lipogenesis  through which they might regulate glucose uptake and glycolysis and may subsequently increase plasma TG levels . A potential explanation of the discordant effects of TG and T2DM is that higher glucose utilization and glycolytic flux leads to downregulation of glucose-6-phosphatase and upregulation of glucokinase (GCK), phosphofructokinase and fatty acid synthases. These changes lead to higher glycogen synthesis, increased concentration of malonyl CoA concentration and production of larger TRLP subfractions . Our observation that TG-wGRSs positively associate with TG but with lower risk to T2DM is consistent with this hypothesis.<SEP>Our study has many strengths, including a large number of incident T2DM cases, the homogenous study sample, a comprehensive collection of TG-associated genetic variants, longitudinal study design, and complete profiles of TRLP subfractions and standard lipids. The long follow-up duration for incident T2D also reduces the likelihood of reverse causation. Our study has several limitations including that our sample was based upon European ancestry women participants only; the generalizability of the current findings still needs to be established by testing among ancestrally diverse populations. TG, HDL and LDL are highly correlated lipids and it is difficult to distinguish the specific effects of one lipid fraction versus another. To limit such potential pleiotropic effects, TG-wGRS and T2DM incidence analyses were additionally adjusted with HDL and LDL and the results were materially similar. However, the findings from Klimentidis et al.,  were based upon both male and female participants and were similar to our findings suggesting that they are independent of sex.<SEP>In summary, our findings provide novel information understanding the complex relationship between TG and T2DM by focusing this relationship on the large TRLP component of TG. At the clinical level, the gene-based relationship between TG and T2DM related traits was opposite to the observational relationship, suggesting a complex gene regulation and cardiometabolic interplay between TG and T2DM related traits. One expects that the mechanisms underlying our associations most likely occur in tissues where lipids and T2DM related biomolecules are synthesized and stored. Our findings thus suggests future studies employing systematic biological approaches using liver, muscle and adipose tissues are warranted to help further understand the causal links between lipids, glycemic traits, and their consequences to T2DM risk.",
                  "27233",
                  "['OBWCCE']"
                ],
                [
                  "47",
                  "20028944",
                  "Variation in the transcription factor 7-like two (TCF7L2) gene strongly determines the risk of type 2 diabetes mainly via what is believed to be its effects on β-cell function, and the insulin secretion deficit of carriers of the diabetes risk alleles may be explained at least in part by incretin resistance.<SEP>The association of TCF7L2 with weight-related traits and body fat in humans is unclear. Conflicting data are reported on the issue of whether there is an influence of TCF7L2 variants on insulin resistance or liver fat, and a modulation of BMI and obesity was postulated. Helgason et al. defined two haplotypes, HapBtype 2 diabetes (rs7903146 efficiently tags HapB in all HapMap groups), which is mainly associated with type 2 diabetes, and HapA (rs7924080 efficiently tags HapA in all HapMap groups), which shows a suggestive association with BMI and altered concentrations of hunger-satiety hormones.<SEP>A role of TCF7L2 in adipocyte differentiation is hypothesized for the following reasons: 1) TCF7L2 is expressed in subcutaneous and visceral adipose tissue, and a decrease in TCF7L2 expression in adipose tissue could be demonstrated during caloric restriction; 2) TCF7L2 is part of the Wnt signaling cascade, and Wnt signaling inhibits adipogenesis; and 3) a polymorphism in the Wnt receptor LRP6 is associated with obesity, the metabolic syndrome, and bone loss.<SEP>In this study, we investigated whether TCF7L2 variants, first, are associated with body fat composition and ectopic lipid storage in our cross-sectional setting or, second, interfere with changes in body composition or weight loss during lifestyle intervention. To this end, we used a whole-body magnetic resonance imaging (MRI) approach enabling precise quantification of body fat stores and magnetic resonance spectroscopy (MRS) to measure ectopic fat deposition in the liver. In our analyses, we focused on single nucleotide polymorphisms (SNPs) rs7903146 and rs1255372, which have yielded the best association with type 2 diabetes across multiple Caucasian populations and tag haplotype HapBtype 2 diabetes. In addition, we also investigated SNPs rs1196205 and rs7895340, which were available in our cohort, are in near-complete linkage with SNP rs7924080 (both r2 ≥ 0.89, HapMap data), and therefore efficiently tag haplotype HapA.",
                  "RESEARCH DESIGN AND METHODS<SEP>We studied 309 nondiabetic German Caucasian subjects who participated in the ongoing Tuebingen Lifestyle Intervention Program (TULIP). The subjects were at increased risk for type 2 diabetes. Most of these subjects (∼70%) had a family history of diabetes. About 13% of the study subjects were related to each other. Oral glucose tolerance tests (OGTTs) revealed that 60% of the nondiabetic participants had normal glucose tolerance, 13% had impaired fasting glycemia, 14% had impaired glucose tolerance, and 13% had both impaired fasting glycemia and impaired glucose tolerance.<SEP>OGTT.<SEP>After an overnight fast, subjects ingested a solution containing 75-g glucose at 08:00 h. Plasma glucose, insulin, and C-peptide concentrations were determined at 0, 30, 60, 90, and 120 min thereafter.<SEP>Body composition.<SEP>Total body fat was measured by bioelectrical impedance (BIA-101; RJL Systems, Detroit, MI). In a subgroup of 210 subjects, an exact quantification of fat distribution was performed by MRI and MRS. Magnetic resonance examinations were performed on a 1.5-T whole-body imager (Magnetom Sonata; Siemens Medical Solutions, Erlangen, Germany). A whole-body imaging protocol was applied for recording a set of 90–120 parallel transverse slices. Slice thickness was 10 mm for the entire body. T1-weighted contrast was applied, allowing semiautomatic quantitative assessment of fatty tissue and other tissue types in each cross-section. MRI-derived visceral, nonvisceral, and total fat mass are given in kilograms. Hepatic fat content was determined by localized stimulated echo acquisition mode (STEAM) 1H-magnetic resonance spectroscopy (repetition time [TR] = 4 s, echo time [TE] = 10 ms; 32 scans). The lipid content was quantitatively assessed by analyzing the signal integral using the liver water signal integral as an internal reference. MRS-derived hepatic fat is given in percent of water signal.<SEP>Lifestyle intervention.<SEP>TULIP consists of exercise and dietary intervention for 2 years, which was adopted from the intervention used in the Finnish Diabetes Prevention Study (DPS) and was previously reported. Since this is an ongoing project and most data are available for the baseline and 9-month visit, we analyzed these data in the present study. The participants aimed at a weight loss of at least 5%, a reduction of caloric intake from fat to <30% and an increase of fiber intake to at least 15 g/1,000 kcal. Individuals were asked to perform at least 3 h of moderate exercise per week, which was monitored using a heart rate monitor (Polar, Büttelborn, Germany). All individuals had up to 10 sessions with a dietitian during the 9 months. Informed written consent was obtained from all participants, and the local ethics committee approved the protocol.<SEP>Genotyping.<SEP>DNA from whole blood was isolated using a commercial DNA isolation kit (NucleoSpin; Macherey & Nagel, Düren, Germany). Genotyping was performed using the TaqMan assay (Applied Biosystems, Forster City, CA). The TaqMan genotyping reaction was amplified on a GeneAmp PCR system 7000, and fluorescence was detected on an ABI PRISM 7000 sequence detector (Applied Biosystems). The genotyping success rates were 99.9% for SNP rs7903146, 99.6% for SNP rs12255372, 99.1% for rs11196205, and 98.9% for rs7895340. The genotypes were verified in 50 randomly selected subjects by bidirectional sequencing, and both methods yielded 100% identical results.<SEP>Analytical procedures.<SEP>Plasma insulin and C-peptide levels were determined with a commercial chemiluminescence system (ADVIA Centaur; Siemens Medical Solutions, Fernwald, Germany). Blood glucose was measured using a bedside glucose analyzer and the glucose oxidase method. (Yellow Springs Instruments, Yellow Springs, OH).<SEP>Calculations.<SEP>The area under the curve (AUC) of glucose and C-peptide during the OGTT was calculated according to the trapezoid method. Insulin sensitivity during the OGTT was estimated according to the formula: 10,000/(fasting glucose × fasting insulin × mean glucose [OGTT] × mean insulin [OGTT])1/2.<SEP>Statistical analyses.<SEP>Unless otherwise stated, data are given as means ± SD. Data that were not normally distributed were logarithmically transformed prior to statistical analysis. Hardy-Weinberg equilibrium was tested using the χ2 test. Differences in parameters between genotypes were tested using multivariate linear regression analysis. Differences with a P value <0.05 were considered statistically significant. The JMP 7.0 (SAS Institute, Cary, NC) statistical software package was used.<SEP>Power calculation.<SEP>The study was sufficiently powered (1 −β = 0.93) to detect effect sizes as small as SD/5 in the overall cohort for change in BMI.",
                  "RESULTS<SEP>The observed minor allele frequency for SNP rs7903146 was 34%, for rs12255372 it was 29%, for rs11196205 it was 48%, and for rs7895340 it was 46%. All SNPs were in Hardy-Weinberg equilibrium (P ≥ 0.4).<SEP>All data were adjusted for sex and age. Glucose concentrations, insulin secretion, and insulin sensitivity derived from the OGTT were additionally adjusted for BMI. The effect of the lifestyle intervention on glucose metabolism, body composition, and weight loss was reported previously. Subject characteristics and effect of lifestyle intervention are presented in Table 1.<SEP>In the cross-sectional setting, we found no association of the type 2 diabetes risk SNP rs7903146 with insulin sensitivity, BMI, bioimpedance-derived total body fat, visceral fat mass, nonvisceral fat mass, or liver fat (all P ≥ 0.2) (Table 2). However, we observed a trend for lower insulin secretion as measured by AUCC-Peptide/AUCGlucose (P ≤ 0.07) and higher 120-min glucose (P ≤ 0.1) in type 2 diabetes risk allele carriers (Table 2).<SEP>During lifestyle intervention, the risk allele carriers displayed lower reduction in BMI (P ≤ 0.0108) and total body fat (Pdom = 0.0155; Padd = 0.07) (Table 2). SNP rs7903146 explained 2.6% of the variation in the intervention-induced change of BMI. The variant was not associated with changes in fasting or 120-min glucose, insulin sensitivity, liver fat, or insulin secretion (all P ≥ 0.09). Because of the known effect of the variant on insulin secretion, we additionally adjusted the effect on change in BMI for insulin secretion and 120-min glucose. This however did not alter the results.<SEP>In the MRI/MRS cohort, we observed in the risk allele carriers a lower reduction in nonvisceral (Pdom = 0.0022; Padd = 0.06) and visceral fat mass (Pdom = 0.0165; Padd = 0.06) (Table 2). As adjustment for sex and age does not completely account for sex- and age-specific interaction effects of rs7903146 on body composition and weight loss, we performed separate analyses for male and female and old and young (separated by the median) study participants (supplemental Tables 1 and 2, available in an online appendix at http://diabetes.diabetesjournals.org/cgi/content/full/db09-1050/DC1). After stratification for sex, most of the associations observed in the overall cohort were also seen in the subgroups, and no clear sex effects could be detected. After stratification for age, however, the associations of the overall cohort were only reflected in the young subgroup.<SEP>To minimize the chance of statistical type 1 errors, we additionally analyzed SNP rs12255372, which is in incomplete linkage disequilibrium with rs7903146 (r2 = 0.77, HapMap data). Essentially the same results were obtained with rs12255372 (supplemental Table 3).<SEP>In contrast, SNPs rs11196205 and rs7895340 (tagging haplotype HapA) were not associated with body composition cross-sectionally or with weight loss during lifestyle intervention (all P > 0.5) (supplemental Tables 4 and 5).",
                  "DISCUSSION<SEP>In the present study, we found in a population at risk for type 2 diabetes that risk variants for type 2 diabetes of the TCF7L2 gene (rs7903146, rs12255372) influence lifestyle intervention-induced changes in BMI, total fat, and nonvisceral and visceral fat, while changes in insulin secretion, insulin sensitivity, or blood glucose are unaffected. Like our cross-sectional setting, most cross-sectional studies found no reliable association of variants in TCF7L2 gene with BMI. Nevertheless, Helgason et al. observed a moderately leaner phenotype in carriers of the haplotype HapBtype 2 diabetes (0.44% lower BMI per copy). The finding of a leaner phenotype is confirmed in the cross-sectional analyses of the Diabetes Prevention Program (DPP) and the DPS. The authors of the DPP concluded that their finding is unexpected and possibly artificial. Subjects with lower insulin secretion, like the TCF7L2 risk allele carriers, had a higher risk to develop diabetes even though they were leaner. Therefore, there may be a bias in the study population when enrolling only subjects at risk for type 2 diabetes. Although the HapA haplotype is reported to associate with BMI and altered concentrations of hunger-satiety hormones, the HapA-tagging SNPs tested in our cohort were not associated with weight loss in the longitudinal setting.<SEP>In the DPS as well as in the DPP, it was shown that the diabetogenic effect of the SNPs rs7903146 and rs12255372 in TCF7L2 is mitigated by lifestyle intervention. Changes in BMI and fat depots though were not reported in the DPS and the DPP; therefore, we cannot draw conclusions as to how changes in body weight influence the outcome in these studies. However, it may be speculated that smaller reductions in BMI and body fat due to the risk allele may not necessarily influence type 2 diabetes incidence, at least within the 3- to 4-year observation period.<SEP>We currently have no explanation for the conflicting findings with regard to the phenotype of TCF7L2 SNP carriers in cross-sectional settings versus our longitudinal setting. In the present study, the intervention was applied to all individuals so that each individual served as his/her own historic control subject. This approach may explain the different findings of studies that are fully randomized.<SEP>The effects of TCF7L2 on body fat found in this study need replication in other larger cohorts and longer-term lifestyle intervention studies to verify their clinical relevance. These studies should include a control group without weight loss and subjects not at high risk for diabetes. In addition, studies at the molecular level should also be undertaken to identify the possible mechanisms of how TCF7L2 influences body fat change.<SEP>In conclusion, we show that diabetes-associated variation in TCF7L2 influences changes in BMI and body fat and thus predicts the success of lifestyle intervention in terms of weight loss. Therefore, TCF7L2 represents a gene that influences individual susceptibility toward environmental factors. The mechanisms of how the variants interact with environmental factors to influence BMI are not understood and need further investigation.",
                  "13322",
                  "['OBWCCE', 'EATBTS', 'FOODIN']"
                ],
                [
                  "48",
                  "29180724",
                  "Introduction<SEP>Genome-wide association studies (GWASs) have successfully identified hundreds of SNPs associated with complex diseases or traits. However, to date, the SNPs identified fail to explain a large proportion of the variance of the traits/diseases under study. Previous studies have suggested that GWASs have the potential to explain a larger proportion of “missing heritability” mainly by using larger sample sizes. However, although acquiring larger sample sizes may increase statistical power, it is often not feasible since the recruiting and genotyping of additional participants is too costly. Therefore, there is a need for analytical methods that can better and more efficiently utilize the information contained in the existing pool of available data for the identification of trait-associated loci. Several of these types of methods have recently been developed and successfully applied to identify novel loci for various complex traits.<SEP>Pleiotropy is the phenomenon of a single gene or locus affecting two or more phenotypes. There is ample evidence to suggest that genetic pleiotropy exists in many correlated diseases and traits, such as bipolar disorder and schizophrenia, indicating that related traits may share overlapping genetic mechanisms. Through the incorporation of information regarding genetic pleiotropy, we can improve the detection power of common variants associated with complex diseases or traits by effectively increasing the sample sizes without the need to recruit more individuals. The joint analysis of related phenotypes may reveal novel insights into the common biological mechanisms and overlapping pathophysiological relationships between complex traits.<SEP>Andreassen et al. developed a genetic-pleiotropy-informed conditional false discovery rate (cFDR) method by leveraging two GWASs from associated traits in a conditional analysis. The method has been successfully applied to genetically associated diseases and phenotypes including schizophrenia and bipolar disorder, as well as blood pressure and other phenotypes. Our group has recently successfully applied the cFDR method to the joint analyses of bone mineral density (BMD) and breast cancer, BMD and coronary artery disease, femoral neck (FNK) BMD and height, and T2D and birth weight. All of these studies improved statistical power through the joint analysis of related traits, and unambiguously demonstrated the utility of the method for improving the identification of potentially novel trait-associated variants.<SEP>Obesity is a chronic metabolic disorder mainly characterized by excessive body fat. Body Mass Index (BMI) is widely used in obesity research and clinical diagnosis to quantify an individual’s tissue mass. Identification of the genetic determinants for BMI, a non-invasive measure of obesity that predicts the risk of related complications, could lead to a better understanding of the biological basis of obesity. Epidemiological studies estimate that the prevalence of overweight/obese individuals increased by >40% between 1980 and 2013, and that these elevated obesity levels are a driving force for the similarly rapid increase of Type II Diabetes (T2D). T2D is a chronic metabolic disorder characterized by high blood sugar, insulin resistance, and relative lack of insulin, all of which share some genetic susceptibility and functional mechanisms with obesity. Heritability studies have demonstrated a substantial genetic contribution to both obesity risk (h2~40–70%) and T2D (h2~26–69%). In 2014, an estimated 387 million people were living with diabetes, corresponding to a worldwide prevalence of 8.3%, and 90% of these individuals had T2D.<SEP>There has been substantial evidence to indicate an important relationship between obesity and T2D, along with strong support to suggest that obesity and T2D share some common genetic risk factors. The accumulation of body fat may be associated with several conditions related to T2D including insulin resistance, hyperinsulinemia, the reduced utilization of glucose in muscles and other tissues, and impaired glucose tolerance. Additionally, Corbin et al. used Mendelian Randomization (MR) Egger analysis to explore the complex relationship between these traits and demonstrated a true causal effect of BMI on T2D, as well as potential pleiotropy between the two phenotypes. Although dozens of genetic loci associated with BMI or T2D have been detected by GWASs, these loci can explain at best 10% of the genetic variance for either obesity or T2D. Considering the high degree of heritability and potential pleiotropy between these phenotypes, the two traits are ideal for the further analyses using the cFDR approach to improve the detection of loci associated with obesity and/or T2D.<SEP>In this study, we applied the genetic-pleiotropy-informed cFDR method on two large datasets of GWAS summary statistics for BMI and T2D to identify novel loci and pleiotropic relationships between these traits. These two GWASs have identified 97 and 62 loci associated with BMI and T2D respectively, but they only explain 2.7% and 5.7% of the total heritability for these traits. The purpose of our study is to improve SNP detection for obesity and T2D using these two existing GWASs, and gain some novel insights into the shared biological mechanisms and overlapping genetic heritability between them. The clarification of potentially shared genetic determinants may have significant implications for the identification of important biomarkers and development of novel therapeutic approaches for joint prediction, prevention, and intervention of the two related diseases/phenotypes.",
                  "Materials and Methods<SEP>GWAS Datasets<SEP>The dataset for T2D contains association summary statistics for a trans-ethnic T2D GWAS meta-analysis of 26,488 cases and 83,964 controls. Ancestry-specific meta-analyses were previously performed by component datasets from the full set of cohorts, including the DIAbetes Genetics Replication and Meta-analysis (DIAGRAM) Consortium (European descent), the Asian Genetic Epidemiology Network T2D (AGEN-T2D) Consortium (East Asian descent), the South Asian T2D (SAT2D) Consortium (south Asian descent), and the Mexican American T2D (MAT2D) Consortium (Mexican and Mexican American descent). Further details of the samples and methods employed within each ancestry group are presented in the corresponding consortium papers. Briefly, various genotyping products were applied in the individual study’s assay processes, with appropriate sample and SNP quality control (QC). Genotype imputation was conducted within each GWAS dataset using Phase II/III HapMap as the reference panels. Each SNP with MAF > 1% (except MAF > 5% in MAT2D GWAS due to a smaller sample size) and passing QC was analyzed for association with T2D using an adjusted additive model. Association summary statistics of each ancestry-specific meta-analysis were combined using a fixed-effect inverse-variance weighted meta-analysis. Genomic control (GC) was carried at the individual study level, after ethnicity-specific meta-analysis, and finally after trans-ethnic meta-analysis.<SEP>The GIANT dataset for BMI contains association summary statistics for the GWAS and Metabochips meta-analysis of 339,224 individuals of various ancestries, including 322,154 individuals of European descent and 17,072 individuals of African-American and Hispanic descent. The data contains the summary p-values from meta-analysis after correction for inflation due to potential population admixture. Two rounds of GC were separately applied both at the cohort level and after meta-analysis.<SEP>Data Preparation<SEP>Before the implementation of the cFDR method, several preparation steps were performed. First, we checked the European Ancestry cohorts for overlapping samples included in these two datasets (Table S1). Next, we combined the common SNPs included in these two datasets. Then we applied a linkage disequilibrium (LD) based SNP pruning method to remove large correlations between pairs of variants. The SNP pruning method begins using a window of 50 SNPs where LD is calculated between each pair of SNPs. The minor allele frequency (MAF) is the basis for the SNP pruning, where for pairs with r2 > 0.2 we removed the SNP with smaller MAF. Following this initial removal of SNPs, the window slides 5 SNPs forward and the process is repeated until there are no pairs of SNPs that are high in LD. The dataset was pruned using the HapMap 3 genotypes as a reference panel. Last, we performed gene annotation for the final set of 123,804 variants that were included in the analyses.<SEP>GC corrections were used in the GWASs to ensure that the variance estimates for each SNP are not inflated due to population structure and cryptic relatedness. Both of the original datasets we adopted in our study applied GC at the individual study level and again after meta-analysis, hence there was no need for us to reapply the GC in this analysis.<SEP>Statistical analysis<SEP>The cFDR approach is well-established now, which has been widely applied by many other groups and our group. We briefly summarized this cFDR approach as follows: after the data preparation processing, we computed the conditional empirical cumulative distribution functions (cdfs) of the corrected p-values for the x axis in conditional QQ plot. Empirical cdfs for BMI SNP p-values were conditioned on nominal p-values in T2D, and vice versa. For each nominal p-value, an estimate of the cFDR was obtained from the conditional empirical cdfs. Using this cFDR approach, we obtained two cFDR tables–cFDR result for BMI conditioned on T2D and vice versa. Using these tables we identified loci associated with BMI and T2D (cFDR < 0.05), respectively. Then a conjunction method was used to find SNPs significantly associated with both BMI and T2D. Specifically, we took the maximum of those two cFDR values above as our conjunction FDR.<SEP>Conditional QQ and enrichment plots for assessing pleiotropic enrichment<SEP>As an intuitive illustration, we presented conditional Q-Q plots to graphically assess the pleiotropic enrichment of SNPs of the principal phenotype successively conditioning on various strengths of associations with the conditional phenotype. We plotted the QQ curve for the quantiles of nominal -values obtained from GWAS summary statistics for association of the subset of SNPs that are below each significance threshold in the conditional trait. The nominal -values were plotted on the y-axis and the empirical quantiles (empirical cumulative distribution functions (cdfs)) of the nominal p-values were plotted on the x-axis. Under the global null hypothesis, the theoretical distribution of p-values is expected to lie approximately on the diagonal line of the Q-Q plots. Enrichment of genetic associations is indicated as a leftward deflection from the null line as the principal phenotype is successively conditioned on increasing strength of associations with the conditional phenotype. The degree of deflection between curves provides important information about the degree of pleiotropy between the two phenotypes. Larger deflections are considered to represent a greater enrichment of pleiotropic genes between the two phenotypes.<SEP>For the associated phenotypes BMI and T2D, pleiotropic “enrichment” of BMI with T2D exists if the proportion of SNPs or genes associated with BMI increases as a function of increased association with T2D. To confirm the pleiotropic enrichment effect, we presented fold-enrichment plots of nominal  values for BMI SNPs below the standard GWAS threshold of p < 5 × 10−8 and for subsets of SNPs determined by the significance of their association with T2D and vice versa. As the p values of the conditional phenotypes become more significant, lower upward shift from the null line will persist.<SEP>In order to check the pleiotropic enrichment and provide a baseline that can be used to confirm novel findings, we also generated conditional QQ plots for two control traits that are unlikely to be correlated with BMI and T2D, ADHD and MDD.<SEP>Conditional Manhattan plots for localizing genetic variants<SEP>To demonstrate the localization of the SNPs associated with BMI conditional on their significance on T2D, and the reverse, we present conditional Manhattan plots. The plots present the relationship between all SNPs within an LD block and their chromosomal locations. The 22 chromosomal locations are plotted on the x-axis, and the  BMI values conditional on T2D are plotted on the y-axis and vice versa for T2D. Any SNP with a  value greater than 1.3 (FDR < 0.05) was deemed to be significantly associated with the principal phenotype. We also present a conjunction Manhattan plot to demonstrate the locations of the common pleiotropic genetic variants associated with both phenotypes.<SEP>Functional annotation and gene enrichment analysis<SEP>In order to evaluate the biological functions of the individual trait associated loci identified by cFDR and pleiotropic loci identified by conjunction FDR, we performed functional annotation and gene enrichment analysis using the gene ontology (GO) terms database (http://geneontology.org/). All significant genes identified by cFDR and conjunction cFDR in our study were annotated and characterized based on three main categories: biological processes, cellular component and molecular functions. This analysis provided comprehensive biological information, allowing us to partially validate our findings by determining specific genes that are enriched in T2D- and obesity-related GO terms.<SEP>Protein-protein interaction network<SEP>In order to detect interactions and associations of the BMI-associated and T2D-associated genes respectively, protein-protein interaction analyses were conducted by searching the Search Tool for the Retrieval of Interacting Genes/Proteins (STRING) database (http://string-db.org/). The STRING database comprises known and predicted associations from curated databases or high-throughput experiments, and also with other associations derived from text mining, co-expression, and protein homology.",
                  "Results<SEP>Assessment of pleiotropic enrichment<SEP>The conditional Q-Q plot for BMI conditional on T2D (Upper Panel (left) in Fig. 1) showed some enrichment across varying significance thresholds for T2D. The presence of leftward shift when restricting the analysis to include the SNPs that have more significant associations with BMI indicates an increase in the number of true associations for a given T2D p-value. Similar enrichment is observed for T2D given BMI (Upper Panel (right) in Fig. 1), as there appears to be a similar departure pattern between the different curves. These leftward deflections from the null line indicate a greater proportion of true associations for any given BMI nominal p-value. <SEP>Based on the fold-enrichment plot (Lower Panel of Fig. 1), we observed SNP enrichment for BMI across different levels of significance with T2D and vice versa. For progressively stringent p-value thresholds for BMI SNPs, we observed about a 50-fold increase in the proportion of SNPs reaching the genome wide significance level of -log10 (p) > 7.3 when comparing the subset with the most stringent conditional association to the group with all SNPs. A 50-fold increase was also observed for T2D.<SEP>As negative controls, conditional Q-Q plots for BMI given nominal p-values of association with attention-deficit/hyperactivity disorder (ADHD) (Upper Panel (left) in Figure S1) and major depressive disorder (MDD) (Lower Panel (left) in Figure S1), and T2D conditional on ADHD (Upper Panel (left) in Figure S2) and MDD (Lower Panel (left) in Figure S2) all showed no enrichment and vice versa.<SEP>BMI loci identified with cFDR<SEP>Conditional on their association with T2D, we identified 287 significant SNPs (cFDR < 0.05) for BMI variation (Fig. 2 and Table S2), which were mapped to 21 different chromosomes (1–21) and annotated to 323 genes. In the original meta-analysis for BMI GWAS, 105 SNPs had p-values smaller than 1 × 10−5 while 36 of them reached the standard genome-wide significance of 5 × 10−8. We confirmed 43 SNPs that were reported in the original BMI GWAS analysis and previous BMI related GWASs. Another 40 SNPs that were reported to be associated with BMI-related traits were also confirmed in our analysis. The rest of the 204 SNPs were not previously reported in the original BMI GWAS or any other previous obesity studies. However, 26 of these 204 SNPs are in high linkage disequilibrium (LD) (r2 > 0.6) with other BMI-associated SNPs reported previously (Table S3). Among the 323 genes these 287 SNPs were annotated to, 146 of these genes were newly detected compared to the original BMI GWAS and previous obesity-related studies (Table S2). Among all the 287 detected loci for BMI, most of the genes were enriched in BMI-related terms such as “positive regulation of cellular metabolic process”, “positive regulation of metabolic process” and “regulation of protein metabolic process”. GO term enrichment analysis results were detailed in Table 1.  <SEP>T2D gene loci identified with cFDR<SEP>We identified 75 SNPs significantly (cFDR < 0.05) associated with T2D given their association with BMI (Fig. 3 and Table S4), which were located on 20 chromosomes (1–20) and annotated to 89 genes. In the original meta-analysis for T2D GWAS, 38 SNPs had p-values smaller than 1 × 10−5 while 12 of them reached the standard genome-wide significance of 5 × 10−8. We confirmed 17 SNPs that were reported in the original T2D GWAS analysis or previous T2D related GWASs. Another 18 SNPs that were reported to be associated with T2D-related traits were also confirmed in our analysis. The remaining 40 SNPs were not previously reported in the original T2D GWAS or any other T2D studies, although nine of these SNPs showed high LD (r2 > 0.6) with the T2D-associated SNPs reported previously (Table S5). For the 89 genes these 75 SNPs were annotated to, 42 of these genes were novel and not identified by the original T2D GWAS or previous T2D-related studies (Table S2). Of the detected loci for T2D, some of the genes were enriched in T2D-related terms such as “pancreas development”, “response to glucose” and “endocrine pancreas development”. GO term enrichment analysis were detailed in Table 1. <SEP>Pleiotropic gene loci for both BMI and T2D<SEP>The conjunction FDR analysis detected 23 independent pleiotropic loci that were significantly (conjunction FDR < 0.05) associated with both traits (Fig. 4 and Table 2). Of the 23 identified pleiotropic variants, one SNP rs9930506 (FTO) reached genome-wide significance in the original BMI and T2D GWASs. The SNPs rs7141420 (NRXN3), rs1996023 (GNPDA2 and GABRG1), rs16945088 (FTO), rs9540493 (LOC10272396 and LINC01052) and rs4238585 (GPR139 and GP2) reached genome-wide significance in only the original BMI GWAS. Six SNPs (rs10787472 (TCF7L2), rs2881654 (PPARG), rs849135 (JAZF1), rs4481184 (IGF2BP2), rs1783598 (FCHSD2) and rs12245680 (TCF7L2)) were reported to be significant for only T2D in the original or previous T2D GWAS. The two SNPs rs6795735 (ADAMTS9-AS2) and rs12454712 (BCL2) were previously reported to be associated with both obesity and T2D. The other five SNPs (rs17584208, rs11979110, rs10898868, rs1996023 and rs4474658) were previously reported to be associated with high density lipoprotein (HDL) and proinsulin. The final four SNPs were not previously reported in the original BMI and T2D GWASs or GWAS studies for any related traits. For the 30 genes the identified pleiotropic SNPs were annotated to, we found twelve of them (AKAP6, NPAS3, PSRC1, MYBPHL, MIR29A, GABRG1, ZNF664, FAM101A, LOC10272396, LINC01052, GPR139, and PUM1) were not identified by any BMI or T2D related GWASs. For the SNPs that were annotated to these 12 genes, two SNPs were located in the intronic regions of genes ZNF664 and PUM1 respectively, and the other five SNPs were all located in intergenic regions (Table 2). Of the detected 23 pleiotropic loci, most of the genes were enriched in BMI and T2D related terms such as “regulation of lipid storage”, “regulation of fat cell proliferation”, “fat cell differentiation”, and “fatty acid transport”. Detailed information of GO term analysis was given in Table 1.  <SEP>Protein-protein interaction network<SEP>The 323 identified BMI-associated genes were retrieved from the STRING database. Only 143 genes, including 46 novel genes, were annotated in this database. The 143 genes were clearly enriched in three clusters: TMEM18, PPARG and MAP2K5 (Figure S3). Two novel genes MSRA and PDILT, respectively encoding methionine sulfoxide reductase A and protein disulfide isomerase-like, were directly connected with the TMEM18 cluster. Another two novel genes, MED23 and ANPC4, respectively encoding mediator complex subunit 23 and anaphase promoting complex subunit 4, were involved in the PPARG cluster. Another three novel genes, MEF2D, RASL11A and PTPN12, respectively encoding myocyte enhancer factor 2D, RAS-like, family 11, member A and protein tyrosine phosphatase, were involved in the MAP2K5 cluster. (Figure S3).<SEP>The 89 identified T2D-associated genes were retrieved from the STRING database. Only 37 genes, including 7 novel genes, were annotated in this database. The 37 genes were clearly enriched into three clusters: HNF4A, MTNR1B and TCF7L2 (Figure S4). Three novel genes, ANXA11, BCL2L11 and NEUROG3, those respectively encoding Annexin A11, BCL2-like 11 and Neurogenin 3, were involved in the HNF4A cluster. Another two novel genes, NPBWR2 and PTHLH, encoding Neuropeptides B/W receptor 2 and Parathyroid hormone-like hormone, were involved in the MTNR1B cluster. The other novel gene MED30 was directly connected with TCF7L2 cluster (Figure S4)",
                  "Discussion<SEP>In our study, two GWASs with summary statistic p values were combined to explore the pleiotropic enrichment of SNPs that are associated with BMI and T2D. Compared to the conventional standard single phenotype GWASs, simultaneously analyzing multiple related traits allows for the increased discovery of trait-associated variants without requiring additional larger datasets for each individual trait. By leveraging the power of two different GWAS datasets from BMI and T2D, we discovered 287 loci for BMI and 75 loci for T2D. Using the standard GWAS significance threshold in the datasets, only 36 were significant for BMI. Most of the genes have not been reported to show borderline significance with BMI, as detailed in Table S2. Adopting the genetic pleiotropy-informed cFDR method, we found 12 additional novel loci associated with both BMI and T2D. These novel findings may enable us to further dissect the overlapping genetic mechanisms between these two related phenotypes. The improved detection of novel susceptibility loci with genetic pleiotropy may lead us to a better understanding of common etiology between disorders and have a significant impact on the clinical treatment and prevention of related complex human diseases.<SEP>The cFDR approach was adopted here to account for some of the missing heritability between traits or diseases. This method employs the idea that a variant with significant effects in two associated phenotypes is more likely to be a true effect, and therefore has a higher probability of being detected in multiple independent studies. This technique allows for an increase in effective sample size and therefore a subsequent increase in power to detect true associations for more variants with small to moderate effect sizes, which are often ignored in the standard single phenotype GWAS. In addition, the genetic enrichment presented in conditional Q-Q plots conveys that the decreased cFDR value for a given nominal p value greatly increases power to detect true association effects. When initially implementing the cFDR method, Andreassen et al. demonstrated one advantage of this model-free empirical cdf approach is for the avoidance of bias in cFDR estimates from model misspecification. Through a comparison of traditional unconditional FDR and cFDR methods, they found that the latter resulted in an increase of 15–20 times the number of SNPs under the same FDR threshold of 0.05.<SEP>Our cFDR analysis identified 23 pleiotropic signals annotated to 30 genes, providing evidence for the close relationship and shared genetic mechanisms between these two traits. These findings are consistent with the evidence from previous studies that have demonstrated a causal relationship between these two traits. The genes FTO, MC4R and TCF7L2 were frequently reported and replicated in previous BMI and T2D related studies. However, potential confounding factors and biases might coincidently be responsible for some of these associations. For the genes FTO and MC4R, their respective effects on T2D were found to be modest and previous studies showed that their effects on T2D disappeared after adjustment for BMI. In European populations, TCF7L2 was not reported as a risk factor for obesity although its effect on T2D risk is modulated by obesity because of the interaction between TCF7L2 polymorphisms (rs7903146) and BMI status. The implementation of the cFDR method in our study not only furnishes another empirical validation for the cFDR method to successfully detect novel and known disease associated genetic variants, but also shows the possibility of improved discovery of novel susceptibility loci using existing GWAS summary statistics. There were 14 genes (JAZF1, IGF2BP2, NRXN3, ADAMTS9-AS2, GIPR, FCHSD2, HHEX, EXOC6, KLF14, ARAP1, GNPDA2, GP2, C2CD4A and C2CD4B) that were associated with either BMI or T2D in previous studies but not with both that were detected as pleiotropic loci in this analysis. Furthermore, 12 novel genes are worth noting because no previous study has reported associations with either BMI or T2D for any of them. For the SNPs that were annotated to these 12 genes, two SNPs were located in the intronic regions of genes ZNF664 and PUM1 and the other five SNPs were all located in intergenic regions. As examples, we will discuss two of these genes ZNF664 and PUM1 for their potential functional relevance and significance.<SEP>The SNP rs825461 is located at the intronic region between gene ZNF664 and FAM101A. The ZNF664 gene encodes a protein named zinc finger protein 664, and one study reported that ZNF664 was involved in eye development and that the monogenic form may be associated with high risk of myopia. Furthermore, ZNF664 was previously reported to show suggestive association (P < 1E-4) with adiponectin, a protein involved in many metabolic processes including glucose regulation and fatty acid oxidation. The rs1473 SNP is located at the intronic region of the gene PUM1, a member of the PUF family of proteins that contains a sequence-specific RNA binding domain. One study reported that the protein may be involved in the regulation of embryogenesis, and cell development and differentiation. These genes may be involved in certain processes that are significant in the development of obesity and T2D, however future studies are needed to explore the exact mechanisms of those novel genes we identified.<SEP>Our study presents several strengths. First, the statistical power is increased through the cFDR method by leveraging two large GWAS datasets, providing an increase in the effective sample size. Although a meta-analysis of the same data would offer a similar gain, the meta-analysis approach only allows for more powerful detection of loci with the same direction of allelic effects in the phenotypes, whereas the cFDR method allows for detection of loci regardless of their effect directions. Second, we consider two traits that are unlikely to be correlated with BMI and T2D, ADHD and MDD, and generate conditional QQ plots with respect to these “control traits.” The “control trait” enrichment analysis provides an alternative way to examine pleiotropic enrichment and provides a baseline that can be used to statistically partially validate the novel findings in our study. We believe that the collider-stratification bias is unlikely in our analysis because, the GWAS datasets have undergone genomic control (GC) and we also carried out LD pruning with r2 > 0.2. In addition, our conditional analysis provides a model-free method to obtain conservative estimation.<SEP>Our study may also have some important limitations. First, we could not provide information about the effect estimates of pleiotropic loci on the phenotypes due to a lack of detailed individual-study-level data. However, we can infer this information from the summary effect sizes in the original GWAS study. This cFDR approach cannot distinguish between vertical and horizontal pleiotropy of the pleiotropic signals, although this question might be partially addressed in future Mendelian Randomization studies. Second, it is likely that some of our cFDR results may be  overestimated due to overlapping samples although the model-free approach is able to neutralize this overestimation of the conservative cFDR estimate. Alternative approaches may be applied to check whether novel loci could still be identified in order to further confirm novel findings in our study or to furnish an empirical comparison of the relative performance of alternative methods, a topic we wish to pursue in the future with comprehensive theoretical and simulation approaches.<SEP>In summary, by incorporating the shared genetic effects of two closely related traits into a conditional analysis framework, we observed significant pleiotropic enrichment between obesity and T2D. We identified several novel pleiotropic loci of potential functional significance for obesity and T2D in our analysis, and the results may provide us with novel insights into the shared genetic influences between these two disorders.",
                  "29968",
                  "['OBWCCE']"
                ],
                [
                  "49",
                  "31676834",
                  "Introduction<SEP>Type 2 diabetes (T2D) is a multifactorial disease, with a complex polygenic architecture. The association of the transcription factor 7-like 2 (TCF7L2) gene (chromosome 10q25.2-q25.3) with T2D is one of the most reproducible and robust finding in T2D genetics, as supported by Genome-wide association studies (GWAS), multiple replication studies and meta-analyses. Several single nucleotide polymorphisms (SNPs) within TCF7L2 were independently associated with T2D susceptibility and related traits (e.g. insulin secretion and blood glucose levels).<SEP>The protein encoded by TCF7L2 gene is a transcription factor, involved in the Wnt/beta-catenin signaling pathway, which plays a role in cell proliferation and differentiation. It is related to beta-cells and other pancreatic cells functions, as well as to the development and function of adipose tissue. In addition, TCF7L2 is expressed in multiple brain regions and characterized by existence of several alternative splicing variants in different species. In humans, a unique splice variant was found in the brain, pancreatic islets and gut and therefore named the “neuroendocrine form“. SNPs in TCF7L2 have been associated with psychiatric disorders, such as schizophrenia and bipolar disorder.<SEP>Individuals with T2D have higher risk for cognitive dysfunction and dementia than those without T2D. The most consistent findings in neuroimaging of T2D patients are higher number of infarcts and white matter lesions, as well as general cerebral and hippocampal atrophy. Few studies have found regional atrophy in different brain structures, but these results were not consistent. Greater atrophy of the amygdala and hippocampus had been associated not only with T2D, but also to high plasma glucose levels within the normal range. However, the underlying etiology of brain volume differences in T2D is still unknown. At the genetic level, previous studies in the Israel Diabetes and Cognitive Decline (IDCD) study found that apolipoprotein ε4 (APOE4) genotype and haptoglobin (Hp) 1–1 genotype, affect the relationship between neuroimaging phenotypes (White matter hyperintensities [WMHs] and Hippocampal volume, respectively) and glycemic control among T2D patients.<SEP>We examined the association of four TCF7L2 SNPs (rs7901695, rs7903146, rs11196205, and rs12255372) with WMH, gray matter, and regional volumes of the amygdala and hippocampus obtained from structural brain magnetic resonance imaging (MRI) in Jewish T2D elderly patients. These SNPs are reported in the literature as robustly associated with T2D. We hypothesized that TCF7L2 risk alleles for T2D would be associated with smaller regional brain volume and larger WMH volume among T2D patients.",
                  "Methods<SEP>Sample<SEP>Participants were recruited from the Israel Diabetes and Cognitive Decline (IDCD) study, a collaboration of the Icahn School of Medicine, Mount Sinai, NY, USA, Sheba Medical Center, Israel, and the Maccabi Health Services (MHS), Israel. The IDCD study design has been previously described in detail. Briefly, community-dwelling Israeli elderly individuals with T2D (≥65 years old) were recruited from the MHS diabetes registry. Criteria for enrolment into the IDCD study were: (1) having T2D (defined as any of the following- (A) HbA1c > 7.25%; (B) Glucose blood levels of 200 mg/dl on two examinations more than 3 months apart; (C) purchase of diabetic medication twice within 3 months; or (D) diagnosis of T2D (International Classification of Diseases [ICD9] code) by a general practitioner, internist, endocrinologist, ophthalmologist, or diabetes advisor, supported by a HbA1c > 6.5% or glucose > 125 mg/dl within half a year); (2) normal cognition at entry to the IDCD study; (3) being free of any neurological (e.g., Parkinson’s disease, stroke), psychiatric (e.g. schizophrenia) or other diseases (e.g., alcohol or drug abuse) that might affect cognition; (4) having an informant; (5) fluency in Hebrew; (6) living in the area of Tel Aviv. The Diabetes Registry has collected detailed laboratory, medication, and diagnoses information since 1998. Based on self-report, the IDCD individuals are unrelated to each other (at least at first- and second-degree level). The HbA1c and blood pressure (systolic and diastolic) values were calculated for each participant as the means of all measurements in the diabetes registry.<SEP>MRI acquisition<SEP>A randomly recruited sub-sample of the IDCD cohort underwent MRI scan, performed at the Diagnostic Imaging Department, Sheba Medical Center using a 3 Tesla scanner (GE, Signa HDxt, v16VO2). High-resolution (1 mm3) images were acquired by using a 3D inversion recovery prepared fast spoiled gradient-echo (FSPGR) T1-weighted sequence (TR/TE = 7.3/2.7 s, 20° flip angle, TI 450 ms). In addition, a T2-weighted fluid-attenuated inversion recovery (FLAIR) sequence was acquired with the following parameters: Repetition time/Echo time (TR/TE) 9500/123 ms, axial slices, slice-width/gap 3/0.4 mm, 22 cm FOV, 64 × 64 matrix, 90° flip-angle.<SEP>MRI analysis<SEP>For volumetric analysis, the voxel based morphometry (VBM) toolbox, (http://www.fil.ion.ucl.ac.uk/spm/ext/#VBMtools) implemented in Statistical Parametric Mapping (SPM8) software was used on the T1 weighted anatomical images. This procedure included automated iterative skull stripping, segmentation of the images into gray matter, white matter (WM), cerebrospinal fluid probability images, and spatial normalization of the gray matter images to a customized gray matter template in standard MNI (Montreal Neurological Institute) space. Finally, the gray matter maps were smoothed using an 8 mm Gaussian kernel. Gray matter probability maps were thresholded at 0.2 to minimize inclusion of incorrect tissue types. Total intracranial volume (TICV) was calculated by summing the segmented and thresholded images (TICV = gray matter + white matter + cerebrospinal fluid). Based on our a-priori hypothesis, we used a region of interest (ROI) approach centered on the amygdala and hippocampus, identified with the ‘Human Automated Anatomical Labelling (AAL) atlas’ within the Wake Forest University PickAtlas (http://www.rad.wfubmc.edu/fmri) and extracted using the MarsBaR ROI toolbox as implemented in SPM12. All reported volumes are total regional volumes.<SEP>For WMH quantification we used the Lesion segmentation toolbox (LST) (implemented in SPM8), following previously described methods. The default LST settings were used with the exception of κ (k), a value indicating the threshold for the initial lesion mask. Visual inspection of the probability maps across participants by using various k values, to maximize sensitivity while reducing false positive results, indicated that a k = 0.15 was the optimal value for our sample images. This procedure generated one binary lesion image per participant from which a total lesion volume (in milliliters) map was extracted.<SEP>SNPs selection and genotyping<SEP>Four intronic TCF7L2 SNPs (rs7901695, rs7903146, rs11196205, and rs12255372) were selected for this study (Table 2), based on ample evidence of their confirmed association with T2D and related traits. These SNPs were genotyped with the Sequenom MassARRAY system, at the Washington University Human Genetics Division Genotyping Core, St. Louis, USA. Quality control measures were implemented.<SEP>Statistical analysis<SEP>We employed hierarchical linear regression to study the association of the TCF7L2 SNPs with amygdalar, hippocampal, gray matter and WMH volumes, under three genetic models (additive, dominant and recessive – referring to the effect of the T2D risk allele). In the basic regression model (model A), we controlled for sex, age at IDCD baseline recruitment and TICV (this covariate was included in the models for gray matter, hippocampus and amygdala). In the second step (model B), we included in the regression model all the covariates from model A, in addition to a set of T2D related characteristics (time in the MHS diabetes registry [an approximation to T2D duration], mean HbA1C levels, use of T2D medication [yes/no]), mean body mass index (BMI), and ancestry (Ashkenazi vs. Non-Ashkenazi, based on self-report and land of birth data). In the third step (model C), we included in the regression model all the covariates from models A and B, in addition to mean systolic and mean diastolic blood pressure.<SEP>The analysis was conducted for each SNP and brain region separately. For each brain region, a two-sided p value of 0.0042 (0.05/12) was considered statistically significant following employment of Bonferroni correction for multiple testing (0.05/[4 SNPs included in the final analysis × 3 genetic models]). For statistical analysis, we used SPSS version 21.0 (SPSS Inc., Chicago, IL, USA). Hardy–Weinberg calculations, SNPs pairwise correlation and linkage disequilibrium (LD) values were obtained with PLINK (http://pngu.mgh.harvard.edu/purcell/plink). For the WMH, we applied square-root transformation to obtain normal distribution. Power calculation for TCF7L2 SNPs association with amygdalar volume was carried by Quanto v1.2.4 software. To assess a potential distinct contribution of the four TCF7L2 SNPs on amygdalar volume, we performed conditional analysis for each SNP (adjusting for a second SNPs within the regression model, coded recessively).<SEP>For haplotype analysis, we used PLINK to determine haplotypes blocks and frequencies. We employed hierarchical linear regression to study the association of the TCF7L2 haplotypes with amygdalar volume under recessive model (comparing carriers of two copies of haplotype of interest, to carriers of all other haplotypes combinations), adjusting for covariates.<SEP>Study approval and informed consent<SEP>All participants provided informed consent, and all experimental protocols were approved by the institutional review boards (IRBs) of all three collaborating institutions (Icahn School of Medicine, Mount Sinai, NY, USA, Sheba Medical Center, Israel, and MHS, Israel). In addition, all the methods were carried out in accordance with the relevant guidelines and regulations.",
                  "Results<SEP>Demographic and medical characteristics<SEP>The final analysis included 191 T2D individuals, all of them were IDCD participants. The demographic and clinical description of the sample is detailed in Table 1. All participants were both genotyped for the TCF7L2 SNPs (rs7901695, rs7903146, rs11196205, and rs12255372; Table 2) and had brain MRI scans. Genotyping success rate per SNP was 97.38–100%<SEP>Association of TCF7L2 SNPs with the volume of different brain regions<SEP>We found that two SNPs, rs7903146 and rs7901695 were highly correlated to each other (r2 = 0.94) and can be viewed essentially as a single signal (Table 3). From the analyzed SNPs, only rs7903146 showed slight deviation form Hardy-Weinberg equilibrium (p = 0.045), but nevertheless we included it in the analysis.<SEP>The mean volumes of the different brain regions, according to the four analyzed SNPs and genotypes are described in Table 4. As shown in Table 5, by employing linear regression, we found a significant association in all four TCF7L2 SNPs with amygdalar volume in the recessive genetic model (regression model B - adjusting for sex, age and total intracranial volume [TICV]): rs7901695- CC genotype vs. CT + TT genotypes, β = −0.21, p = 0.002; rs7903146-TT vs. TC + CC, β = −0.20, p = 0.003; rs11196205- CC vs. CG + GG, β = −0.247, p = 0.0003; and rs12255372- TT vs. TG + GG, β = −0.20, p = 0.003. These results withstood our threshold for multiple testing correction (p = 0.0042).<SEP>Controlling also for T2D-related covariates (time in the diabetes registry, mean hemoglobin A1c (HbA1C) levels and use of T2D medication [yes/no]), body mass index (BMI) and ancestry (regression model B) did not change the results substantively (rs7901695 β = −0.2, p = 0.003; rs7903146 β = −0.19, p = 0.005; rs11196205 β = −0.23, p = 0.001; and rs12255372 β = −0.19, p = 0.005), although rs12255372 and rs7903146 no longer withstood the threshold for multiple testing correction (Table 5). In all four SNPs, individuals who were homozygous of the T2D risk allele had ~9.5% smaller amygdalar volume compared to the carriers of the non-risk allele (Table 6). Adjusting also to systolic and diastolic blood pressure values did not change results (regression model C, Supplementary Table 1). In addition, TCF7L2 rs11196205 showed a significant association with amygdalar volume in the additive genetic model (regression model B- p = 0.005; model B- p = 0.008) but did not remain significant when implementing multiple testing correction (Table 5).<SEP>In order to analyze the potential distinct effect of the four TCF7L2 SNPs (under recessive model), we performed a conditional analysis, including a second SNP as a covariate in the regression model. In the joint analysis of rs11196205 and each of the other three SNP (separately), the association with amygdalar volume was still significant (regression model A- p = 0.039, p = 0.044, p = 0.033 – controlling for rs7901695, rs7903146 or rs12255372 respectively), or approaching significance (model B- p = 0.08, p = 0.086, p = 0.067 – controlling for rs7901695, rs7903146 or rs12255372 respectively) (Table 7). However, in the joint analysis of the highly correlated SNP rs7901695/rs7903146 with rs11196205 or rs12255372, the association of rs7901695 (model A, p = 0.61 and p = 0.3, respectively) or rs7903146 (model A, p = 0.624 and p = 0.322, respectively) with amygdalar volume became non- significant. Similarly, the joint analysis of rs12255372 with any of the other SNPs (Table 7) was not significant. Therefore, we assume that none of the three SNPs rs7901695, rs7903146 and rs12255372 contributed independently to the association with amygdalar volume, beyond the effect of the most highly significant SNP rs11196205.<SEP>Due to a variable levels of linkage disequilibrium (LD) between the four TCF7L2 SNPs, we performed haplotype analysis. Two haplotype blocks were found: 1. rs7901695 and rs7903146 (first block); 2. rs11196205 and rs12255372 (second block). Consistent with the recessive model, participants with two copies of the first block CT haplotype (rs7901695-C and rs7903146-T, N = 32) or participant with two copies of the second block CT haplotype (rs11196205-C and rs12255372-T, N = 35) had significantly smaller amygdalar volume compared to participants with other haplotype combinations in the same block (Supplementary Table 2). These results were essentially identical to the association of rs7903146 and rs12255372 alone, respectively (model A- p = 0.003; model B- p = 0.005). Combining all SNPs, carriers of two copies of the CTCT haplotype (rs7901695-C, rs7903146-T, rs11196205-C and rs12255372-T, N = 29) had significantly smaller amygdalar volume compared to participants with other haplotypes combinations (model A- p = 0.013; model B- p = 0.014). Information regarding the haplotype analysis, including haplotypes frequencies, is presented in Supplementary Tables 2 and 3.<SEP>A posteriori power estimates (based on the observed regression coefficients) for rs11196205 association with amygdalar volume in our sample (recessive model, minor allele frequency of 0.487, required p = 0.05), ranged from 94% (adjusting for only age, sex and TICV) to 89% (adjusting also for additional covariates). A posteriori power estimates for the other three SNPs (rs7901695, rs7903146 and rs12255372) association with amygdalar volume (recessive model, minor allele frequency of 0.382–0.403, required p = 0.05), were 77–83% (model A) and 68–78% (model B).<SEP>Interestingly, we also found a significant association of rs11196205 with hippocampal and gray matter volume under the recessive genetic model at a nominal significance level (β = −0.141, p = 0.035 and β = −0.085, p = 0.044, respectively), but these results became marginal when the second set of covariates was added (model B- β = −0.121, p = 0.064 and β = −0.075, p = 0.072, respectively) (Table 5). All other associations (amygdalar volume in the additive and dominant models, as well as all models for WMH, hippocampal and gray matter volumes) did not reach the required level of significance after correction for multiple testing.<SEP>Association of TCF7L2 SNPs with left versus right amygdalar volume<SEP>In a secondary analysis we applied similar linear regression for left and right amygdala separately, in accordance to previous evidence in the literature showing differences between the two sides. As shown in Table 8, association results of all four SNPS are stronger (at the significance level achieved) for the left amygdala under recessive model (Model B- rs7901695 β = −0.19, p = 0.005; rs7903146 β = −0.19, p = 0.006; rs11196205 β = −0.24, p = 0.0004; and rs12255372 β = −0.21, p = 0.002), while weaker for the right amygdala (Model B- rs7901695 β = −0.18, p = 0.012; rs7903146 β = −0.17, p = 0.018; rs11196205 β = −0.19, p = 0.006; and rs12255372 β = −0.16, p = 0.025). Looking at the left amygdala separately, results of conditional (Table 9) and haplotype (Supplementary Table 2) analyses are similar to that of the total amygdalar volume.",
                  "Discussion<SEP>The well-established association of TCF7L2 with T2D and the link between T2D and brain imaging changes, have motivated us study the association of this gene with neuroimaging phenotypes in our sample of elderly T2D Jewish patients. We have found a consistent association of TCF7L2 SNPs with amygdalar volume. In the four investigated SNPs (rs7901695, rs7903146, rs11196205, and rs12255372), carriers of two copies of the T2D risk allele had smaller amygdalar volume, compared to carriers of the non-risk allele (recessive model), while controlling for sex, age and TICV. Adjusting also for T2D related covariates, BMI, ancestry and blood pressure did not change the results substantially. Further examination of the left and right amygdala separately, revealed that the association is derived mainly due to the left amygdalar volume (p = 0.0004–0.006) than the right amygdalar volume (p = 0.006–0.025). On conditional analysis, we found that rs7901695, rs12255372 or rs7903146 SNPs associations with amygdalar volume were not independent of the most highly significant SNP rs11196205, and therefore only one association signal was detected in region. No associations of hippocampal, gray matter and WMH volumes with TCF7L2 SNPs withstood Bonferroni adjustment for multiple testing correction.<SEP>Several limitations of this study should be considered. Our sample size (N = 191 individuals) is considered small in the context of a genetic association study. Nevertheless, the sample is unique, since it includes only T2D elderly, a population at risk for cognitive decline and dementia. All participants had clinical (including measures of glycemic control), neuroimaging and genetic data. Some of the associations survived Bonferroni correction for multiple  testing, indicating robustness of the results and therefore reducing the likelihood of false positive results. The mere nominal level of association with gray matter and hippocampal volume, which did not withstand the Bonferroni correction, might be due to a small sample size, and larger size would have been an advantage in terms of statistical power. In addition, the cross-sectional design of the study impedes reaching conclusions of causality. The longitudinal component of the IDCD is ongoing and may assist in shedding light on functional effect of this association in the future.<SEP>Previous neuroimaging genetics studies did not find association of TCF7L2 with amygdalar volume. Of particular interest is a recent GWAS meta-analysis study of ~30,000 participants (mostly European origin) conducted by the Enhancing Neuro Imaging Genetics through Meta-Analysis (ENIGMA) consortium (which did not include specific cohorts of T2D patients in particular). No significant associations (additive model, p < 0.05) were found in this study between amygdalar volume and TCF7L2 SNPs rs7901695, rs7903146 and rs12255372. Although rs11196205 was not tested directly in the ENIGMA GWAS, its proxy SNP rs10885409 (D′ = 1, r2 > 0.95), found by using the Broad institute SNP proxy search (http://archive.broadinstitute.org/mpg/snap/ldsearch.php), was not significantly associated with amygdalar volume as well. Several explanations for the discrepancy in results are plausible in addition to different genetic model, including that the observed association of TCF7L2 with amygdalar volume is specific to the Jewish population, or alternatively is specific to T2D affected individuals. Our study did not include control participants without T2D, and therefore we cannot address the generalization of this finding to non-T2D individuals. It is also possible that the association is influenced by older age and potentially not found in younger population (this sample include individuals aged 65 years and older). Taken together, at the current stage, our findings should be considered as preliminary and caution is required in their interpretation. Further studies are required in various populations to validate it.<SEP>The amygdala, one of the limbic system’s components, has been implicated in several functions - mainly emotional processing and responses (e.g. fear, anxiety, and aggression), decision-making, associative learning and memory. Amygdalar aberrant function or structure is common in neurodevelopmental disorders. Previous studies in various populations have reported association of variation in several genes with amygdalar volume, including STMN1 and SLC6A4, CACNA1C and the oxytocin receptor OXTR.<SEP>Consistent with our results, previous reports have demonstrated an association between structural brain changes and T2D, e.g. lower brain volumes and greater brain atrophy in T2D patients, including amygdala. Indeed, greater amygdalar atrophy had been associated with high plasma glucose levels within the normal range.<SEP>TCF7L2 is expressed in many brain regions, including the amygdala in mice and at a relatively low level in human amygdala (Genotype Tissue expression portal, GTex, Broad Institute; https://gtexportal.org/home/gene/TCF7L2/). As part of the Wnt/beta-catenin signaling pathway, TCF7L2 plays role in the activation of lymphoid enhancer-binding factor 1/T cell factor (LEF1/TCF) transcription factors complexes. The Wnt/beta-catenin signaling is involved in neuroplasticity, adult neurogenesis and CNS development, as well as in amygdala-dependent learning and long-term memory formation. Decreased levels of beta‐catenin were found in the amygdala of rats that showed behavioral sensitization to administration of drugs of abuse. In humans, polymorphisms in TCF7L2 were associated with schizophrenia and bipolar disorder. At the behavioral level in animal models, TCF7L2 deficient mice demonstrated altered anxiety like behavior and fear learning, and this gene mediated cellular and behavioral response to lithium treatment in mice and zebrafish. Combined, these evidences implicate a role of TCF7L2 in brain function and behavioral phenotypes.<SEP>To conclude, our results in a sample of T2D elderly demonstrate for the first-time associations of four TCF7L2 SNPs with amygdalar volume. Confirmation of these results in additional cohorts is required in order to reach more definitive conclusions.",
                  "23441",
                  "['OBWCCE']"
                ]
              ],
              "shape": {
                "columns": 7,
                "rows": 1000
              }
            },
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pubmed_id</th>\n",
              "      <th>INTRO</th>\n",
              "      <th>METHODS</th>\n",
              "      <th>RESULTS</th>\n",
              "      <th>DISCUSS</th>\n",
              "      <th>text_length</th>\n",
              "      <th>TOPIC_LABEL</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>30419927</td>\n",
              "      <td>Background&lt;SEP&gt;The Mediterranean diet (MeD) is...</td>\n",
              "      <td>Methods&lt;SEP&gt;Study design and subjects&lt;SEP&gt;The ...</td>\n",
              "      <td>Results&lt;SEP&gt;Population characteristics&lt;SEP&gt;The...</td>\n",
              "      <td>Discussion&lt;SEP&gt;Among the different dietetic pa...</td>\n",
              "      <td>21979</td>\n",
              "      <td>['OBWCCE', 'EATBTS', 'FOODIN']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>30032228</td>\n",
              "      <td>Introduction&lt;SEP&gt;Overweight and obesity have b...</td>\n",
              "      <td>Methods&lt;SEP&gt;This systematic review was perform...</td>\n",
              "      <td>Results&lt;SEP&gt;Articles Identified&lt;SEP&gt;The initia...</td>\n",
              "      <td>Discussion&lt;SEP&gt;The purpose of this review was ...</td>\n",
              "      <td>32452</td>\n",
              "      <td>['OBWCCE', 'EATBTS']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>21799482</td>\n",
              "      <td>INTRODUCTION&lt;SEP&gt;In recent years, the prevalen...</td>\n",
              "      <td>METHODS AND PROCEDURES&lt;SEP&gt;Study population&lt;SE...</td>\n",
              "      <td>RESULTS&lt;SEP&gt;Characteristics of the study parti...</td>\n",
              "      <td>DISCUSSION&lt;SEP&gt;In our study, we found that var...</td>\n",
              "      <td>22444</td>\n",
              "      <td>['OBWCCE', 'EATBTS']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>24516669</td>\n",
              "      <td>Introduction&lt;SEP&gt;Osteoporosis and obesity are ...</td>\n",
              "      <td>Materials and Methods&lt;SEP&gt;Subjects&lt;SEP&gt;Two pop...</td>\n",
              "      <td>Results&lt;SEP&gt;The general and clinical character...</td>\n",
              "      <td>Discussion&lt;SEP&gt;Obesity is an established risk ...</td>\n",
              "      <td>21427</td>\n",
              "      <td>['OBWCCE']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>19079261</td>\n",
              "      <td>Obesity is a major public health problem, resu...</td>\n",
              "      <td>METHODS&lt;SEP&gt;Study design&lt;SEP&gt;This study is com...</td>\n",
              "      <td>RESULTS&lt;SEP&gt;Initial meta-analysis of GWAS stud...</td>\n",
              "      <td>DISCUSSION&lt;SEP&gt;Through meta-analysis of GWA da...</td>\n",
              "      <td>35778</td>\n",
              "      <td>['OBWCCE']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>995</th>\n",
              "      <td>24852944</td>\n",
              "      <td>Introduction&lt;SEP&gt;Adoptive immunotherapy is a p...</td>\n",
              "      <td>Materials and Methods&lt;SEP&gt;T cell culture&lt;SEP&gt;A...</td>\n",
              "      <td>Results and Discussions&lt;SEP&gt;The coupling of pr...</td>\n",
              "      <td>NaN</td>\n",
              "      <td>19075</td>\n",
              "      <td>['FOODAL']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>996</th>\n",
              "      <td>28231819</td>\n",
              "      <td>Background&lt;SEP&gt;Targeted cancer immunotherapies...</td>\n",
              "      <td>Methods&lt;SEP&gt;Tumor samples and sequencing&lt;SEP&gt;S...</td>\n",
              "      <td>Results&lt;SEP&gt;Tumor mutanomes are unique&lt;SEP&gt;We ...</td>\n",
              "      <td>Discussion&lt;SEP&gt;Our data reveal that inter-indi...</td>\n",
              "      <td>22078</td>\n",
              "      <td>['FOODAL']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>997</th>\n",
              "      <td>31676868</td>\n",
              "      <td>In type 1 diabetes (T1D), early inflammation o...</td>\n",
              "      <td>Methods:&lt;SEP&gt;Human Islets and EndoC-βH1&lt;SEP&gt;Hu...</td>\n",
              "      <td>Results&lt;SEP&gt;Pro-inflammatory cytokines impact ...</td>\n",
              "      <td>Discussion:&lt;SEP&gt;Our work illustrates the human...</td>\n",
              "      <td>43718</td>\n",
              "      <td>['FOODAL']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>998</th>\n",
              "      <td>31506281</td>\n",
              "      <td>Introduction&lt;SEP&gt;Germinal centers (GCs) form i...</td>\n",
              "      <td>Materials and methods&lt;SEP&gt;Animals and treatmen...</td>\n",
              "      <td>Results&lt;SEP&gt;TGFβ signaling occurs in GCBs and ...</td>\n",
              "      <td>Discussion&lt;SEP&gt;In this study, we showed that T...</td>\n",
              "      <td>42774</td>\n",
              "      <td>['FOODAL']</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>999</th>\n",
              "      <td>28415571</td>\n",
              "      <td>INTRODUCTION&lt;SEP&gt;Essential thrombocythemia (ET...</td>\n",
              "      <td>PATIENTS AND METHODS&lt;SEP&gt;Patient enrollment&lt;SE...</td>\n",
              "      <td>RESULTS&lt;SEP&gt;Mutational analysis&lt;SEP&gt;Among 54 E...</td>\n",
              "      <td>DISCUSSION&lt;SEP&gt;CALR mutations have been found ...</td>\n",
              "      <td>27830</td>\n",
              "      <td>['FOODAL']</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>1000 rows × 7 columns</p>\n",
              "</div>"
            ],
            "text/plain": [
              "     pubmed_id                                              INTRO  \\\n",
              "0     30419927  Background<SEP>The Mediterranean diet (MeD) is...   \n",
              "1     30032228  Introduction<SEP>Overweight and obesity have b...   \n",
              "2     21799482  INTRODUCTION<SEP>In recent years, the prevalen...   \n",
              "3     24516669  Introduction<SEP>Osteoporosis and obesity are ...   \n",
              "4     19079261  Obesity is a major public health problem, resu...   \n",
              "..         ...                                                ...   \n",
              "995   24852944  Introduction<SEP>Adoptive immunotherapy is a p...   \n",
              "996   28231819  Background<SEP>Targeted cancer immunotherapies...   \n",
              "997   31676868  In type 1 diabetes (T1D), early inflammation o...   \n",
              "998   31506281  Introduction<SEP>Germinal centers (GCs) form i...   \n",
              "999   28415571  INTRODUCTION<SEP>Essential thrombocythemia (ET...   \n",
              "\n",
              "                                               METHODS  \\\n",
              "0    Methods<SEP>Study design and subjects<SEP>The ...   \n",
              "1    Methods<SEP>This systematic review was perform...   \n",
              "2    METHODS AND PROCEDURES<SEP>Study population<SE...   \n",
              "3    Materials and Methods<SEP>Subjects<SEP>Two pop...   \n",
              "4    METHODS<SEP>Study design<SEP>This study is com...   \n",
              "..                                                 ...   \n",
              "995  Materials and Methods<SEP>T cell culture<SEP>A...   \n",
              "996  Methods<SEP>Tumor samples and sequencing<SEP>S...   \n",
              "997  Methods:<SEP>Human Islets and EndoC-βH1<SEP>Hu...   \n",
              "998  Materials and methods<SEP>Animals and treatmen...   \n",
              "999  PATIENTS AND METHODS<SEP>Patient enrollment<SE...   \n",
              "\n",
              "                                               RESULTS  \\\n",
              "0    Results<SEP>Population characteristics<SEP>The...   \n",
              "1    Results<SEP>Articles Identified<SEP>The initia...   \n",
              "2    RESULTS<SEP>Characteristics of the study parti...   \n",
              "3    Results<SEP>The general and clinical character...   \n",
              "4    RESULTS<SEP>Initial meta-analysis of GWAS stud...   \n",
              "..                                                 ...   \n",
              "995  Results and Discussions<SEP>The coupling of pr...   \n",
              "996  Results<SEP>Tumor mutanomes are unique<SEP>We ...   \n",
              "997  Results<SEP>Pro-inflammatory cytokines impact ...   \n",
              "998  Results<SEP>TGFβ signaling occurs in GCBs and ...   \n",
              "999  RESULTS<SEP>Mutational analysis<SEP>Among 54 E...   \n",
              "\n",
              "                                               DISCUSS  text_length  \\\n",
              "0    Discussion<SEP>Among the different dietetic pa...        21979   \n",
              "1    Discussion<SEP>The purpose of this review was ...        32452   \n",
              "2    DISCUSSION<SEP>In our study, we found that var...        22444   \n",
              "3    Discussion<SEP>Obesity is an established risk ...        21427   \n",
              "4    DISCUSSION<SEP>Through meta-analysis of GWA da...        35778   \n",
              "..                                                 ...          ...   \n",
              "995                                                NaN        19075   \n",
              "996  Discussion<SEP>Our data reveal that inter-indi...        22078   \n",
              "997  Discussion:<SEP>Our work illustrates the human...        43718   \n",
              "998  Discussion<SEP>In this study, we showed that T...        42774   \n",
              "999  DISCUSSION<SEP>CALR mutations have been found ...        27830   \n",
              "\n",
              "                        TOPIC_LABEL  \n",
              "0    ['OBWCCE', 'EATBTS', 'FOODIN']  \n",
              "1              ['OBWCCE', 'EATBTS']  \n",
              "2              ['OBWCCE', 'EATBTS']  \n",
              "3                        ['OBWCCE']  \n",
              "4                        ['OBWCCE']  \n",
              "..                              ...  \n",
              "995                      ['FOODAL']  \n",
              "996                      ['FOODAL']  \n",
              "997                      ['FOODAL']  \n",
              "998                      ['FOODAL']  \n",
              "999                      ['FOODAL']  \n",
              "\n",
              "[1000 rows x 7 columns]"
            ]
          },
          "execution_count": 49,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "import pandas as pd\n",
        "\n",
        "dataset_df = pd.read_csv(\"../datasets/working_dataset_1000.csv\")\n",
        "dataset_df"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 57,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "lT7cgCqkzAlK",
        "outputId": "7d729827-73a2-4339-cb2f-8f00c8d9b3d4"
      },
      "outputs": [],
      "source": [
        "dataset_df['TOPIC_LABEL'].unique()\n",
        "\n",
        "default_entity_types = [\n",
        "    \"Gene\",\n",
        "    \"Genetic Variant\",\n",
        "    \"Transcript\",\n",
        "    \"Protein\",\n",
        "    \"Nutrient\",\n",
        "    \"Food\",\n",
        "    \"Dietary Pattern\",\n",
        "    \"Physiological Process\",\n",
        "    \"Metabolic Pathway\",\n",
        "    \"Molecular Interaction\",\n",
        "    \"Environmental Factor\",\n",
        "    \"Disease\",\n",
        "]\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 50,
      "metadata": {
        "id": "jf7YePG1xZF1"
      },
      "outputs": [],
      "source": [
        "# df_FOODAL = dataset_df[dataset_df[\"TOPIC_LABEL\"] == [\"FOODAL\"]]\n",
        "# df_OBWCCE = dataset_df[dataset_df[\"TOPIC_LABEL\"] == [\"OBWCCE\"]]\n",
        "# df_EATBTS = dataset_df[dataset_df[\"TOPIC_LABEL\"] == [\"EATBTS\"]]\n",
        "\n",
        "df_FOODAL = dataset_df[dataset_df['TOPIC_LABEL'].apply(lambda x: 'FOODAL' in x)]\n",
        "df_OBWCCE = dataset_df[dataset_df['TOPIC_LABEL'].apply(lambda x: 'OBWCCE' in x)]\n",
        "df_EATBTS = dataset_df[dataset_df['TOPIC_LABEL'].apply(lambda x: 'EATBTS' in x)]\n",
        "df_VITMIN = dataset_df[dataset_df['TOPIC_LABEL'].apply(lambda x: 'VITMIN' in x)]\n",
        "df_FOODIN = dataset_df[dataset_df['TOPIC_LABEL'].apply(lambda x: 'FOODIN' in x)]"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "GB1f-m2LzsoI"
      },
      "outputs": [],
      "source": [
        "working_dir = \"/content/drive/MyDrive/workshop-ECAI/\"\n",
        "working_dir = os.getcwd() + \"/documents_from_working_dataset/\"\n",
        "df_FOODAL.sample(5)['RESULTS'].to_json(working_dir+\"foodal.json\", orient=\"records\")\n",
        "df_OBWCCE.sample(5)['RESULTS'].to_json(working_dir+\"obwcce.json\", orient=\"records\")\n",
        "df_EATBTS.sample(5)['RESULTS'].to_json(working_dir+\"eatbts.json\", orient=\"records\")\n",
        "df_VITMIN.sample(5)['RESULTS'].to_json(working_dir+\"vitmin.json\", orient=\"records\")\n",
        "df_FOODIN.sample(5)['RESULTS'].to_json(working_dir+\"foodin.json\", orient=\"records\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 51,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Hz3HMz7oz4qq",
        "outputId": "e27b5a45-8040-46b2-be44-bd2d52b2a96c"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "The JSON file contains 5 elements in the list.\n"
          ]
        }
      ],
      "source": [
        "# prompt: read a json file and count the element\n",
        "import json\n",
        "\n",
        "working_dir = os.getcwd() + \"/documents_from_working_dataset/\"\n",
        "file_path = working_dir+\"foodal.json\"  # Replace with the actual path to your JSON file\n",
        "\n",
        "with open(file_path, 'r') as file:\n",
        "  data = json.load(file)\n",
        "\n",
        "# Assuming the JSON file contains a list or a dictionary\n",
        "if isinstance(data, list):\n",
        "  element_count = len(data)\n",
        "  print(f\"The JSON file contains {element_count} elements in the list.\")\n",
        "elif isinstance(data, dict):\n",
        "  element_count = len(data)\n",
        "  print(f\"The JSON file contains {element_count} key-value pairs in the dictionary.\")\n",
        "else:\n",
        "  print(\"The JSON file does not contain a list or a dictionary at the top level.\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {},
      "outputs": [
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.openai.com/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "When biasing the question generation script toward biomedical entities, especially in the context of nutrigenetics, it's crucial to incorporate the intricate interactions between the specified entities such as genes, nutrients, diseases, and metabolic pathways. This targeted approach should reflect a deep understanding of how these entities interact within biological systems and their implications for nutritional genetics. Here's how the goal prompt can be adapted and elaborated to align with these considerations:\n",
            "\n",
            "---\n",
            "\n",
            "**Biomedical Goal Prompt:**\n",
            "\n",
            "In the context of nutrigenetics, where we analyze the interactions between genetic variants and nutritional elements, your task is to generate high-level questions focusing on the complex relationships and effects among the following entities: Gene, Genetic Variant, Transcript, Protein, Nutrient, Food, Dietary Pattern, Physiological Process, Metabolic Pathway, Molecular Interaction, Environmental Factor, and Disease. These questions should aim to deepen the global understanding of how nutritional factors interact with genetic makeup to influence health and disease outcomes.\n",
            "\n",
            "Given a corpus description and the parameters K (number of users), N (number of tasks per user), and M (number of high-level questions per (user, task) pair), your task is to craft questions that illuminate the interconnectedness and impact of these biomedical entities. Consider the following angles to guide your question generation:\n",
            "\n",
            "1. **Genomic Influence on Nutrition:** How do specific genetic variants influence individual responses to nutrients and dietary patterns? Explore effects on metabolic pathways and physiological processes.\n",
            "\n",
            "2. **Nutritional Impact on Gene Expression:** In what ways do particular nutrients or dietary patterns affect gene expression, transcript production, and protein synthesis? Assess the downstream impact on health and disease.\n",
            "\n",
            "3. **Disease and Genetic-Nutrient Interactions:** How do dietary factors interact with genes to modulate disease risk or progression? Consider molecular interactions and potential environmental influences.\n",
            "\n",
            "4. **Metabolic Pathway Insights:** What roles do metabolic pathways and molecular interactions play in mediating the effects of nutrition-genetic interactions on physiological processes and disease outcomes?\n",
            "\n",
            "5. **Environmental and Nutrient Modulation:** How do environmental factors alter the interactions between genes and nutrients, potentially influencing health outcomes and disease susceptibility? \n",
            "\n",
            "Your questions should endeavor to connect these entities within a comprehensive framework that reflects the complexity of nutrigenomic research, fostering a robust understanding of the underlying biological mechanisms.\n",
            "\n",
            "---\n",
            "\n",
            "This prompt capitalizes on the biomedical entities listed, encouraging question formation that not only evaluates the reader's comprehension of the interactions among these elements but also prompts further investigation and discussion about the implications in the field of nutrigenetics."
          ]
        }
      ],
      "source": [
        "from mychatgpt import bot\n",
        "men = bot(\"mendel\",model=4)\n",
        "m='''You have to bias this prompt, toward Biomedical entities and context on this entities relationship. \n",
        "This are the entities of our KNowledge Graph\n",
        "default_entity_types = [\n",
        "    \"Gene\",\n",
        "    \"Genetic Variant\",\n",
        "    \"Transcript\",\n",
        "    \"Protein\",\n",
        "    \"Nutrient\",\n",
        "    \"Food\",\n",
        "    \"Dietary Pattern\",\n",
        "    \"Physiological Process\",\n",
        "    \"Metabolic Pathway\",\n",
        "    \"Molecular Interaction\",\n",
        "    \"Environmental Factor\",\n",
        "    \"Disease\",\n",
        "]\n",
        "\n",
        "This is part of a script for question generation that will be used to generate high-level questions for a corpus of genetic  documents realted to nutrition (nutrigentics).\n",
        "\n",
        "goal_PROMPT = \"\"\" Given a corpus description and the parameters K (number of users), N (number of tasks per user), and M (number of high-level questions per (user, task) pair), your task is to generate high-level questions that assess global understanding of the corpus.\n",
        "\n",
        "\"\"\"\n",
        "\n",
        "'''\n",
        "men.c(m)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 10,
      "metadata": {
        "id": "Pu8Y_gpD3PiI"
      },
      "outputs": [],
      "source": [
        "role_PROMPT = \"\"\"\n",
        "      You are a helpful assistant responsible for generating evaluation questions\n",
        "      from a given corpus using a structured procedure.\n",
        "      \n",
        "      In the context of nutrigenetics, where we analyze the interactions between genetic variants and nutritional elements, your task is to generate high-level questions focusing on the complex relationships and effects among the following entities: Gene, Genetic Variant, Transcript, Protein, Nutrient, Food, Dietary Pattern, Physiological Process, Metabolic Pathway, Molecular Interaction, Environmental Factor, and Disease. These questions should aim to deepen the global understanding of how nutritional factors interact with genetic makeup to influence health and disease outcomes.\n",
        "      \"\"\"\n",
        "# goal_PROMPT = \"\"\"\n",
        "#       Given a corpus description and the parameters K (number of users), N (number of tasks per user),\n",
        "#       and M (number of high-level questions per (user, task) pair), your task is to generate high-level questions\n",
        "#       that assess global understanding of the corpus.\n",
        "#       \"\"\"\n",
        "goal_PROMPT = \"\"\"\n",
        "Given a corpus description and the parameters K (number of users), N (number of tasks per user), and M (number of high-level questions per (user, task) pair), your task is to craft questions that illuminate the interconnectedness and impact of these biomedical entities. Consider the following angles to guide your question generation:\n",
        "entities: Gene, Genetic Variant, Transcript, Protein, Nutrient, Food, Dietary Pattern, Physiological Process, Metabolic Pathway, Molecular Interaction, Environmental Factor, and Disease. These questions should aim to deepen the global understanding of how nutritional factors interact with genetic makeup to influence health and disease outcomes.\n",
        "\n",
        "Your questions should explore the following themes:\n",
        "1. **Genomic Influence on Nutrition:** How do specific genetic variants influence individual responses to nutrients and dietary patterns? Explore effects on metabolic pathways and physiological processes.\n",
        "\n",
        "2. **Nutritional Impact on Gene Expression:** In what ways do particular nutrients or dietary patterns affect gene expression, transcript production, and protein synthesis? Assess the downstream impact on health and disease.\n",
        "\n",
        "3. **Disease and Genetic-Nutrient Interactions:** How do dietary factors interact with genes to modulate disease risk or progression? Consider molecular interactions and potential environmental influences.\n",
        "\n",
        "4. **Metabolic Pathway Insights:** What roles do metabolic pathways and molecular interactions play in mediating the effects of nutrition-genetic interactions on physiological processes and disease outcomes?\n",
        "\n",
        "5. **Environmental and Nutrient Modulation:** How do environmental factors alter the interactions between genes and nutrients, potentially influencing health outcomes and disease susceptibility? \n",
        "\n",
        "Your questions should endeavor to connect these entities within a comprehensive framework that reflects the complexity of nutrigenomic research, fostering a robust understanding of the underlying biological mechanisms.ticError\n",
        "\n",
        "\"\"\"\n",
        "# generation_step_PROMPT = \"\"\"\n",
        "#       Your generation should follow these steps:\n",
        "#       1. Create K user personas based on the corpus, including their role/background and motivation for using the corpus.\n",
        "#       2. For each user, define N realistic and relevant tasks they would perform using the corpus.\n",
        "#       3. For each (user, task) pair, generate M high-level questions that:\n",
        "#           - Require comprehensive understanding of the whole corpus.\n",
        "#           - Do NOT depend on low-level facts or specific data points (e.g., names, dates, figures).\"\"\"\n",
        "\n",
        "generation_step_PROMPT = \"\"\"\n",
        "      Your generation should follow these steps:\n",
        "      1. Create 3 user personas based on the corpus, including their role/background and motivation for using the corpus.\n",
        "            user 1: a researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
        "            user 2: a healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
        "            user 3: a customer interested in personalized nutrition based on genetic information.\n",
        "      2. For each user, define N realistic and relevant tasks they would perform using the corpus.\n",
        "      3. For each (user, task) pair, generate M high-level questions that:\n",
        "          - Require comprehensive understanding of the whole corpus.\n",
        "          - Do NOT depend on low-level facts or specific data points (e.g., names, dates, figures).\"\"\"\n",
        "output_PROMPT = \"\"\"\n",
        "      User 1: [Persona description]\n",
        "      Task 1:\n",
        "      Q1. ...\n",
        "      Q2. ...\n",
        "      ...\n",
        "      Task 2:\n",
        "      Q1. ...\n",
        "      ...\n",
        "      User 2: ...\n",
        "      ...\n",
        "      User K: ...\"\"\"\n",
        "\n",
        "output_format_PROMPT = \"\"\"\n",
        "      {{\n",
        "        \"corpus_description\": \"<your summary of the corpus>\",\n",
        "        \"generated_questions\": \"<your structured output as described above>\"\n",
        "      }}\"\"\""
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 12,
      "metadata": {
        "id": "GWOHvw762McB"
      },
      "outputs": [],
      "source": [
        "def generate_questions(corpus_description, K, N, M):\n",
        "  PROMPT = f\"\"\"\n",
        "  ---Role--\n",
        "  {role_PROMPT}\n",
        "\n",
        "  ---Goal--\n",
        "  {goal_PROMPT}\n",
        "\n",
        "  {generation_step_PROMPT}\n",
        "\n",
        "  Your output should be structured as follows:\n",
        "  {output_PROMPT}\n",
        "\n",
        "  Format your response as a JSON object with the following structure:\n",
        "  {output_format_PROMPT}\n",
        "\n",
        "  ---Corpus Description--\n",
        "  {corpus_description}\n",
        "  ---Parameters--\n",
        "  Number of Users (K): {K}\n",
        "  Tasks per User (N): {N}\n",
        "  Questions per (User, Task) (M): {M}\n",
        "\n",
        "  Output:\n",
        "  \"\"\"\n",
        "  return PROMPT"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 101,
      "metadata": {
        "id": "mrSlx6kx9r0k"
      },
      "outputs": [],
      "source": [
        "import os\n",
        "from groq import Groq\n",
        "\n",
        "MODEL = \"meta-llama/llama-4-scout-17b-16e-instruct\"\n",
        "GROQ_API_KEY = \"***\"\n",
        "GROQ_API_KEY = api_keys[\"groq\"]\n",
        "\n",
        "client = Groq(\n",
        "    api_key=GROQ_API_KEY,\n",
        ")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "ZX-4q8NbaIgO"
      },
      "outputs": [],
      "source": [
        "with open('/content/drive/MyDrive/workshop-ECAI/groq_api.txt', 'w') as f:\n",
        "  f.write(GROQ_API_KEY)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "kuSS3c27Lx-P"
      },
      "outputs": [],
      "source": [
        "working_dir = \"/content/drive/MyDrive/workshop-ECAI/\"\n",
        "working_dir = os.getcwd() + \"/documents_from_working_dataset/\"\n",
        "foodal_docs = json.load(open(working_dir+\"foodal.json\"))\n",
        "obwcce_docs = json.load(open(working_dir+\"obwcce.json\"))\n",
        "eatbts_docs = json.load(open(working_dir+\"eatbts.json\"))\n",
        "vitmin_docs = json.load(open(working_dir+\"vitmin.json\"))\n",
        "foodin_docs = json.load(open(working_dir+\"foodin.json\"))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 68,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "mcZLpQPYMAy_",
        "outputId": "9b998198-7c31-4e35-bfde-6885b91ebb12"
      },
      "outputs": [
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\n",
            "\n",
            "The corpus provides a comprehensive overview of the complex relationships between genetic variants, nutritional elements, and disease outcomes. The papers highlight the importance of considering genetic variants in understanding responses to environmental factors and disease susceptibility. The studies demonstrate the use of various methodologies, including genome-wide association studies, genetic analysis, and multifactor dimensionality reduction analysis, to investigate gene-environment interactions. The corpus contributes to our understanding of the mechanisms underlying nutrigenetics and its applications in personalized medicine.\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\n",
            "\n",
            "The corpus provides a comprehensive overview of the current state of knowledge in nutrigenetics, highlighting the complex interactions between genetic variants, nutritional elements, and their effects on health and disease outcomes. The combined knowledge from these papers underscores the importance of personalized nutrition, inflammation, and disease susceptibility, and highlights the need for further research in this field.\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\n",
            "\n",
            "The corpus provides a comprehensive understanding of the complex interactions between genetic variants, nutritional elements, and disease outcomes. The papers collectively highlight the significance of genetics, nutrition, and pharmacogenetics in understanding disease susceptibility and response to treatment. The corpus covers various biomedical entities, including genes, nutrients, and diseases, and demonstrates the importance of considering these factors in disease prevention and management. Overall, the corpus offers valuable insights into the intricate relationships between genes, nutrients, and health, ultimately contributing to the advancement of personalized medicine and nutrigenetics.\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\n",
            "\n",
            "The corpus provides a comprehensive understanding of the complex relationships between genetic variants, nutritional elements, and their impact on health and disease outcomes. The papers highlight the importance of gene-nutrient interactions in modulating physiological processes, metabolic pathways, and disease susceptibility. The findings have implications for personalized nutrition and disease prevention strategies. The corpus also underscores the need for further research into the mechanisms underlying gene-nutrient interactions and their effects on human health.\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\n",
            "\n",
            "The corpus provides insights into the complex relationships between genetic variants, nutritional elements, and human health and disease outcomes. The papers highlight the importance of understanding the genetic basis of disease susceptibility and progression, as well as the role of environmental factors, such as nutrition and lifestyle interventions. The corpus covers a range of biomedical entities, including genes, genetic variants, proteins, and diseases, and demonstrates the utility of functional studies in elucidating the effects of genetic variants on protein activity and cellular processes. Overall, the corpus contributes to our understanding of the intricate relationships between genetic and environmental factors and their impact on human health and disease outcomes.\n"
          ]
        }
      ],
      "source": [
        "docs_list = [foodal_docs, obwcce_docs, eatbts_docs, vitmin_docs, foodin_docs]\n",
        "labels = [\"foodal\", \"obwcce\", \"eatbts\", \"vitmin\", \"foodin\"]\n",
        "\n",
        "for i, doc in enumerate(docs_list):\n",
        "    corpus_description_PROMPT = f\"\"\"\n",
        "    ---Role--\n",
        "    You are a helpful assistant tasked with generating a description of a corpus based on the results of 5 randomly selected academic papers.\n",
        "\n",
        "    ---Goal--\n",
        "    Given a set of 5 academic papers, your task is to summarize the key findings from each paper and combine this information into a coherent corpus description. The description should highlight common themes, methodologies used, and major conclusions drawn across the papers.\n",
        "\n",
        "    ---Context--\n",
        "    In the context of nutrigenetics, where we analyze the interactions between genetic variants and nutritional elements, your task is to generate high-level questions focusing on the complex relationships and effects among the following entities: Gene, Genetic Variant, Transcript, Protein, Nutrient, Food, Dietary Pattern, Physiological Process, Metabolic Pathway, Molecular Interaction, Environmental Factor, and Disease. These questions should aim to deepen the global understanding of how nutritional factors interact with genetic makeup to influence health and disease outcomes.\n",
        "\n",
        "    ---Papers---\n",
        "    1. {doc[0]}\n",
        "    2. {doc[1]}\n",
        "    3. {doc[2]}\n",
        "    4. {doc[3]}\n",
        "    5. {doc[4]}\n",
        "\n",
        "    ---Instructions---\n",
        "    For each paper:\n",
        "    - Briefly summarize the study's objective.\n",
        "    - Describe the methodology used.\n",
        "    - List the main findings or results highlighting biomedical entities.\n",
        "\n",
        "    After analyzing all 5 papers:\n",
        "    - Identify common themes across the papers.\n",
        "    - Highlight any significant differences in methodologies or results.\n",
        "    - Provide an overall description of the corpus that reflects the combined knowledge and insights from the papers.\n",
        "\n",
        "    ---Output Format---\n",
        "    Return your response as a single text object with the following structure:\n",
        "\n",
        "    <Detailed description of the corpus based on the 5 papers>\n",
        "\n",
        "    ---Output---\n",
        "    **Corpus Description**\n",
        "\n",
        "    \"\"\"\n",
        "\n",
        "\n",
        "    chat_completion = client.chat.completions.create(\n",
        "        messages=[\n",
        "            {\n",
        "                \"role\": \"user\",\n",
        "                \"content\": corpus_description_PROMPT\n",
        "            }\n",
        "        ],\n",
        "        model=MODEL,\n",
        "    )\n",
        "\n",
        "    answer = chat_completion.choices[0].message.content\n",
        "\n",
        "    # print(answer)\n",
        "\n",
        "\n",
        "    # prompt: write the foodal_answer in a file text in my folder in drive\n",
        "\n",
        "    corpus_description = answer.split(\"**\")[-1]\n",
        "    print(corpus_description)\n",
        "\n",
        "    with open(working_dir+f'{labels[i]}_answer.txt', 'w') as f:\n",
        "       f.write(corpus_description)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "XbgL0ElyQyAP"
      },
      "source": [
        "# Question Generation"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 70,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "P2O_-ffVwK43",
        "outputId": "b631ebbc-d9e2-4ba7-cbb2-2c0f23f9d4bf"
      },
      "outputs": [
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Here is the output:\n",
            "\n",
            "```\n",
            "{\n",
            "  \"corpus_description\": \"The corpus provides a comprehensive overview of the complex relationships between genetic variants, nutritional elements, and disease outcomes. The papers highlight the importance of considering genetic variants in understanding responses to environmental factors and disease susceptibility. The studies demonstrate the use of various methodologies, including genome-wide association studies, genetic analysis, and multifactor dimensionality reduction analysis, to investigate gene-environment interactions. The corpus contributes to our understanding of the mechanisms underlying nutrigenetics and its applications in personalized medicine.\",\n",
            "  \"generated_questions\": \"\n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Investigating the impact of genetic variants on nutrient metabolism\n",
            "  Q1. How do specific genetic variants in the genes involved in carbohydrate metabolism affect the response to dietary carbohydrates?\n",
            "  Q2. What are the effects of genetic variants on the expression of enzymes involved in fatty acid metabolism?\n",
            "  Q3. How do genetic variants in the genes encoding for nutrient transporters influence nutrient uptake and utilization?\n",
            "  Q4. Can genetic variants in the genes involved in epigenetic regulation modulate the response to dietary interventions?\n",
            "  Q5. How do genetic variants in the genes involved in inflammation affect the response to dietary patterns and disease risk?\n",
            "\n",
            "  Task2: Exploring the relationship between genetic variants and dietary patterns\n",
            "  Q1. What is the relationship between genetic variants in the genes involved in taste perception and dietary preferences?\n",
            "  Q2. How do genetic variants in the genes involved in satiety and appetite regulation influence food choices?\n",
            "  Q3. Can genetic variants in the genes involved in nutrient sensing affect the response to dietary restriction?\n",
            "  Q4. How do genetic variants in the genes involved in gut microbiota modulation influence the response to prebiotics and probiotics?\n",
            "  Q5. What are the effects of genetic variants on the response to different types of dietary interventions (e.g., low-carb, low-fat)?\n",
            "\n",
            "  Task3: Analyzing the impact of genetic variants on disease risk\n",
            "  Q1. How do genetic variants in the genes involved in insulin signaling affect the risk of developing type 2 diabetes?\n",
            "  Q2. What are the effects of genetic variants in the genes involved in lipid metabolism on cardiovascular disease risk?\n",
            "  Q3. Can genetic variants in the genes involved in inflammation modulate the risk of developing chronic diseases?\n",
            "  Q4. How do genetic variants in the genes involved in antioxidant defense affect the risk of developing age-related diseases?\n",
            "  Q5. What are the interactions between genetic variants and dietary patterns in modulating disease risk?\n",
            "\n",
            "  Task4: Investigating the role of epigenetics in nutrigenetics\n",
            "  Q1. How do epigenetic modifications affect the response to dietary interventions?\n",
            "  Q2. Can epigenetic marks be influenced by dietary patterns and nutrient exposures?\n",
            "  Q3. What are the effects of epigenetic modifications on gene expression and protein synthesis?\n",
            "  Q4. How do epigenetic mechanisms mediate the interactions between genes and environment?\n",
            "  Q5. Can epigenetic modifications be used as biomarkers for disease risk or response to dietary interventions?\n",
            "\n",
            "  Task5: Developing personalized nutrition recommendations\n",
            "  Q1. How can genetic variants be used to tailor dietary recommendations for optimal health outcomes?\n",
            "  Q2. What are the limitations and challenges of using genetic information for personalized nutrition?\n",
            "  Q3. Can genetic variants be used to predict responses to different types of dietary interventions?\n",
            "  Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of personalized nutrition recommendations?\n",
            "  Q5. What are the potential applications of nutrigenetics in public health policy and nutrition counseling?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Integrating nutrigenetics into clinical practice\n",
            "  Q1. What are the clinical implications of genetic variants in the genes involved in nutrient metabolism?\n",
            "  Q2. How can healthcare professionals use genetic information to tailor dietary advice for patients with chronic diseases?\n",
            "  Q3. What are the potential benefits and limitations of using genetic testing for nutritional purposes in clinical practice?\n",
            "  Q4. Can genetic variants be used to predict responses to nutritional interventions in patients with specific health conditions?\n",
            "  Q5. How do healthcare professionals communicate genetic information and dietary recommendations to patients?\n",
            "\n",
            "  Task2: Understanding the role of nutrigenetics in disease prevention\n",
            "  Q1. How do genetic variants in the genes involved in inflammation affect the risk of developing chronic diseases?\n",
            "  Q2. What are the effects of dietary patterns on gene expression and disease risk?\n",
            "  Q3. Can genetic variants be used to identify individuals at high risk of developing specific diseases?\n",
            "  Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of disease prevention strategies?\n",
            "  Q5. What are the potential applications of nutrigenetics in disease prevention and management?\n",
            "\n",
            "  Task3: Exploring the relationship between genetic variants and nutritional deficiencies\n",
            "  Q1. How do genetic variants in the genes involved in nutrient metabolism affect the risk of nutritional deficiencies?\n",
            "  Q2. What are the effects of genetic variants on the absorption and utilization of essential nutrients?\n",
            "  Q3. Can genetic variants be used to predict responses to nutritional supplements?\n",
            "  Q4. How do genetic variants in the genes involved in taste perception and food preferences influence nutritional deficiencies?\n",
            "  Q5. What are the implications of genetic variants for nutritional deficiency diagnosis and treatment?\n",
            "\n",
            "  Task4: Investigating the impact of epigenetics on clinical practice\n",
            "  Q1. How do epigenetic modifications affect the response to dietary interventions in patients with chronic diseases?\n",
            "  Q2. Can epigenetic marks be used as biomarkers for disease risk or response to dietary interventions?\n",
            "  Q3. What are the effects of epigenetic modifications on gene expression and protein synthesis in patients with specific health conditions?\n",
            "  Q4. How do epigenetic mechanisms mediate the interactions between genes and environment in clinical practice?\n",
            "  Q5. Can epigenetic modifications be used to monitor the effectiveness of nutritional interventions?\n",
            "\n",
            "  Task5: Developing evidence-based guidelines for nutrigenetic testing\n",
            "  Q1. What are the current evidence-based guidelines for using genetic testing for nutritional purposes in clinical practice?\n",
            "  Q2. How can healthcare professionals stay up-to-date with the rapidly evolving field of nutrigenetics?\n",
            "  Q3. What are the potential risks and benefits of using genetic testing for nutritional purposes in clinical practice?\n",
            "  Q4. How do healthcare professionals communicate genetic information and dietary recommendations to patients in a clear and actionable way?\n",
            "  Q5. What are the implications of genetic variants for clinical decision-making and patient care?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understanding the basics of nutrigenetics\n",
            "  Q1. What are the key concepts in nutrigenetics and how do they relate to personalized nutrition?\n",
            "  Q2. How do genetic variants affect my response to different nutrients and dietary patterns?\n",
            "  Q3. Can genetic testing help me understand my nutritional needs and develop a personalized diet plan?\n",
            "  Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of personalized nutrition recommendations?\n",
            "  Q5. What are the potential benefits and limitations of using genetic testing for nutritional purposes?\n",
            "\n",
            "  Task2: Exploring the role of genetic variants in dietary responses\n",
            "  Q1. How do genetic variants in the genes involved in carbohydrate metabolism affect my response to dietary carbohydrates?\n",
            "  Q2. What are the effects of genetic variants on my ability to lose weight or maintain weight loss?\n",
            "  Q3. Can genetic variants be used to predict my response to different types of dietary interventions (e.g., low-carb, low-fat)?\n",
            "  Q4. How do genetic variants in the genes involved in inflammation affect my risk of developing chronic diseases?\n",
            "  Q5. What are the implications of genetic variants for my nutritional needs and dietary recommendations?\n",
            "\n",
            "  Task3: Investigating the impact of epigenetics on nutrition\n",
            "  Q1. How do epigenetic modifications affect my response to dietary interventions?\n",
            "  Q2. Can epigenetic marks be influenced by my dietary patterns and nutrient exposures?\n",
            "  Q3. What are the effects of epigenetic modifications on my gene expression and protein synthesis?\n",
            "  Q4. How do epigenetic mechanisms mediate the interactions between my genes and environment?\n",
            "  Q5. Can epigenetic modifications be used to monitor the effectiveness of my nutritional interventions?\n",
            "\n",
            "  Task4: Understanding the relationship between genetic variants and disease risk\n",
            "  Q1. How do genetic variants in the genes involved in insulin signaling affect my risk of developing type 2 diabetes?\n",
            "  Q2. What are the effects of genetic variants in the genes involved in lipid metabolism on my cardiovascular disease risk?\n",
            "  Q3. Can genetic variants be used to predict my risk of developing specific diseases?\n",
            "  Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of disease prevention strategies?\n",
            "  Q5. What are the potential applications of nutrigenetics in disease prevention and management?\n",
            "\n",
            "  Task5: Applying nutrigenetic insights to daily life\n",
            "  Q1. How can I use genetic information to tailor my dietary recommendations for optimal health outcomes?\n",
            "  Q2. What are the practical implications of genetic variants for my daily food choices?\n",
            "  Q3. Can genetic variants be used to predict my response to different types of dietary interventions?\n",
            "  Q4. How do I incorporate genetic information into my overall health and wellness plan?\n",
            "  Q5. What are the potential benefits and limitations of using genetic testing for nutritional purposes in everyday life?\"\n",
            "}\n",
            "```\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Here is the output:\n",
            "\n",
            "```\n",
            "{\n",
            "  \"corpus_description\": \"The corpus provides a comprehensive overview of the current state of knowledge in nutrigenetics, highlighting the complex interactions between genetic variants, nutritional elements, and their effects on health and disease outcomes. The combined knowledge from these papers underscores the importance of personalized nutrition, inflammation, and disease susceptibility, and highlights the need for further research in this field.\",\n",
            "  \"generated_questions\": \"\n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "\n",
            "  Task1: Investigate the relationship between genetic variants and nutrient responses\n",
            "  Q1. How do specific genetic variants affect the metabolism of nutrients and influence physiological processes?\n",
            "  Q2. What are the downstream effects of genetic variants on metabolic pathways and disease susceptibility?\n",
            "  Q3. How do genetic variants interact with dietary patterns to modulate gene expression and protein synthesis?\n",
            "  Q4. What is the role of molecular interactions in mediating the effects of genetic variants on nutrient responses?\n",
            "  Q5. How do environmental factors influence the interactions between genetic variants and nutrients?\n",
            "\n",
            "  Task2: Examine the impact of nutritional elements on gene expression\n",
            "  Q1. How do particular nutrients or dietary patterns affect the expression of specific genes and transcripts?\n",
            "  Q2. What are the effects of nutritional elements on protein synthesis and metabolic pathways?\n",
            "  Q3. How do nutritional elements interact with genetic variants to modulate disease risk or progression?\n",
            "  Q4. What are the implications of nutritional elements on physiological processes and health outcomes?\n",
            "  Q5. How do environmental factors alter the interactions between nutritional elements and genes?\n",
            "\n",
            "  Task3: Analyze the relationship between genetic variants, nutrients, and disease susceptibility\n",
            "  Q1. How do genetic variants interact with nutrients to modulate disease risk or progression?\n",
            "  Q2. What are the effects of genetic variants on metabolic pathways and disease susceptibility?\n",
            "  Q3. How do dietary patterns influence the interactions between genetic variants and disease susceptibility?\n",
            "  Q4. What is the role of molecular interactions in mediating the effects of genetic variants and nutrients on disease outcomes?\n",
            "  Q5. How do environmental factors influence the interactions between genetic variants, nutrients, and disease susceptibility?\n",
            "\n",
            "  Task4: Investigate the role of metabolic pathways in nutrigenetics\n",
            "  Q1. What are the key metabolic pathways involved in mediating the effects of genetic variants and nutrients on physiological processes?\n",
            "  Q2. How do genetic variants and nutrients interact to modulate metabolic pathways and disease susceptibility?\n",
            "  Q3. What are the implications of metabolic pathways on health outcomes and disease risk?\n",
            "  Q4. How do environmental factors influence the interactions between genetic variants, nutrients, and metabolic pathways?\n",
            "  Q5. What are the potential therapeutic targets for modulating metabolic pathways and disease susceptibility?\n",
            "\n",
            "  Task5: Explore the applications of nutrigenetics in personalized nutrition\n",
            "  Q1. How can genetic information be used to tailor dietary recommendations for individuals?\n",
            "  Q2. What are the implications of genetic variants on nutritional requirements and dietary responses?\n",
            "  Q3. How can nutrigenetics be used to prevent or manage chronic diseases?\n",
            "  Q4. What are the challenges and limitations of implementing personalized nutrition based on genetic information?\n",
            "  Q5. How can healthcare professionals integrate nutrigenetics into clinical practice?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "\n",
            "  Task1: Apply nutrigenetic knowledge to patient care\n",
            "  Q1. How can genetic testing be used to inform dietary recommendations for patients?\n",
            "  Q2. What are the potential benefits and limitations of incorporating nutrigenetics into clinical practice?\n",
            "  Q3. How can healthcare professionals stay up-to-date with the latest research in nutrigenetics?\n",
            "  Q4. What are the implications of genetic variants on disease risk and progression in patients?\n",
            "  Q5. How can healthcare professionals communicate complex genetic information to patients?\n",
            "\n",
            "  Task2: Evaluate the role of nutrigenetics in disease prevention and management\n",
            "  Q1. How can nutrigenetics be used to prevent or manage chronic diseases such as diabetes and cardiovascular disease?\n",
            "  Q2. What are the key genetic variants associated with disease susceptibility and how can they be used to inform treatment decisions?\n",
            "  Q3. How do dietary patterns and nutritional elements interact with genetic variants to modulate disease risk or progression?\n",
            "  Q4. What are the potential therapeutic targets for modulating disease susceptibility and progression?\n",
            "  Q5. How can healthcare professionals integrate nutrigenetics into treatment plans?\n",
            "\n",
            "  Task3: Investigate the impact of environmental factors on nutrigenetic interactions\n",
            "  Q1. How do environmental factors such as lifestyle and exposure to toxins influence the interactions between genetic variants and nutrients?\n",
            "  Q2. What are the implications of environmental factors on disease susceptibility and progression?\n",
            "  Q3. How can healthcare professionals take into account environmental factors when making dietary recommendations?\n",
            "  Q4. What are the potential strategies for mitigating the effects of environmental factors on nutrigenetic interactions?\n",
            "  Q5. How can healthcare professionals stay informed about the latest research on environmental factors and nutrigenetic interactions?\n",
            "\n",
            "  Task4: Examine the role of nutrigenetics in personalized medicine\n",
            "  Q1. How can nutrigenetics be integrated into personalized medicine approaches?\n",
            "  Q2. What are the potential benefits and challenges of using nutrigenetics in personalized medicine?\n",
            "  Q3. How can healthcare professionals use genetic information to tailor treatment plans?\n",
            "  Q4. What are the implications of nutrigenetics on health outcomes and disease risk?\n",
            "  Q5. How can healthcare professionals communicate complex genetic information to patients in a personalized medicine context?\n",
            "\n",
            "  Task5: Explore the future of nutrigenetics in healthcare\n",
            "  Q1. What are the potential future directions for research in nutrigenetics?\n",
            "  Q2. How can healthcare professionals stay up-to-date with the latest advances in nutrigenetics?\n",
            "  Q3. What are the potential applications of nutrigenetics in healthcare and medicine?\n",
            "  Q4. How can nutrigenetics be used to improve health outcomes and disease prevention?\n",
            "  Q5. What are the challenges and limitations of implementing nutrigenetics in healthcare?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "\n",
            "  Task1: Understand the basics of nutrigenetics and genetic testing\n",
            "  Q1. What is nutrigenetics and how does it relate to genetic testing?\n",
            "  Q2. What are the benefits and limitations of genetic testing for nutritional purposes?\n",
            "  Q3. How can genetic information be used to inform dietary recommendations?\n",
            "  Q4. What are the key genetic variants associated with nutritional responses and disease susceptibility?\n",
            "  Q5. How can I use genetic information to improve my health and well-being?\n",
            "\n",
            "  Task2: Explore the relationship between genetic variants and nutrient responses\n",
            "  Q1. How do specific genetic variants affect my responses to different nutrients?\n",
            "  Q2. What are the implications of genetic variants on my dietary needs and responses?\n",
            "  Q3. How can I use genetic information to optimize my nutrition and health?\n",
            "  Q4. What are the potential benefits and limitations of using genetic information for personalized nutrition?\n",
            "  Q5. How can I stay informed about the latest research on nutrigenetics and genetic testing?\n",
            "\n",
            "  Task3: Investigate the role of nutrigenetics in disease prevention and management\n",
            "  Q1. How can nutrigenetics be used to prevent or manage chronic diseases?\n",
            "  Q2. What are the key genetic variants associated with disease susceptibility and how can they be used to inform treatment decisions?\n",
            "  Q3. How do dietary patterns and nutritional elements interact with genetic variants to modulate disease risk or progression?\n",
            "  Q4. What are the potential therapeutic targets for modulating disease susceptibility and progression?\n",
            "  Q5. How can I use genetic information to reduce my risk of chronic diseases?\n",
            "\n",
            "  Task4: Examine the impact of environmental factors on nutrigenetic interactions\n",
            "  Q1. How do environmental factors such as lifestyle and exposure to toxins influence the interactions between genetic variants and nutrients?\n",
            "  Q2. What are the implications of environmental factors on disease susceptibility and progression?\n",
            "  Q3. How can I take into account environmental factors when making dietary choices?\n",
            "  Q4. What are the potential strategies for mitigating the effects of environmental factors on nutrigenetic interactions?\n",
            "  Q5. How can I stay informed about the latest research on environmental factors and nutrigenetic interactions?\n",
            "\n",
            "  Task5: Explore the applications of nutrigenetics in personalized nutrition\n",
            "  Q1. How can I use genetic information to tailor my dietary recommendations?\n",
            "  Q2. What are the potential benefits and limitations of using genetic information for personalized nutrition?\n",
            "  Q3. How can I integrate nutrigenetics into my overall health and wellness plan?\n",
            "  Q4. What are the implications of nutrigenetics on my health outcomes and disease risk?\n",
            "  Q5. How can I stay up-to-date with the latest advances in nutrigenetics and personalized nutrition?\n",
            "\"\n",
            "}\n",
            "```\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Here is the output:\n",
            "\n",
            "```\n",
            "{\n",
            "  \"corpus_description\": \"The corpus provides a comprehensive understanding of the complex interactions between genetic variants, nutritional elements, and disease outcomes. The papers collectively highlight the significance of genetics, nutrition, and pharmacogenetics in understanding disease susceptibility and response to treatment. The corpus covers various biomedical entities, including genes, nutrients, and diseases, and demonstrates the importance of considering these factors in disease prevention and management. Overall, the corpus offers valuable insights into the intricate relationships between genes, nutrients, and health, ultimately contributing to the advancement of personalized medicine and nutrigenetics.\",\n",
            "  \"generated_questions\": \"\n",
            "  \n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Investigating the relationship between genetic variants and nutrient response\n",
            "  Q1. How do specific genetic variants affect the metabolism of nutrients and influence physiological processes?\n",
            "  Q2. What are the downstream effects of genetic variants on metabolic pathways and disease susceptibility?\n",
            "  Q3. How do genetic variants interact with dietary patterns to modulate gene expression and protein synthesis?\n",
            "  Q4. What is the role of molecular interactions in mediating the effects of genetic variants on nutrient response?\n",
            "  Q5. How do environmental factors influence the interactions between genetic variants and nutrients?\n",
            "\n",
            "  Task2: Exploring the impact of nutrigenetics on disease prevention\n",
            "  Q1. How do genetic variants contribute to disease susceptibility and progression in response to dietary factors?\n",
            "  Q2. What are the key metabolic pathways and physiological processes affected by genetic-nutrient interactions in disease development?\n",
            "  Q3. Can genetic variants be used to predict disease risk and tailor dietary interventions?\n",
            "  Q4. How do nutritional elements interact with genetic variants to modulate disease progression and treatment response?\n",
            "  Q5. What are the implications of nutrigenetics for personalized medicine and disease prevention?\n",
            "\n",
            "  Task3: Analyzing the effects of nutrients on gene expression\n",
            "  Q1. How do specific nutrients and dietary patterns affect gene expression and transcript production?\n",
            "  Q2. What are the mechanisms by which nutrients influence protein synthesis and modification?\n",
            "  Q3. How do nutrients interact with genetic variants to modulate gene expression and disease susceptibility?\n",
            "  Q4. What are the downstream effects of nutrient-gene interactions on metabolic pathways and physiological processes?\n",
            "  Q5. Can nutrient-gene interactions be used to develop targeted dietary interventions?\n",
            "\n",
            "  Task4: Understanding the role of environmental factors in nutrigenetics\n",
            "  Q1. How do environmental factors, such as lifestyle and exposure to toxins, influence genetic-nutrient interactions?\n",
            "  Q2. What are the effects of environmental factors on gene expression and protein synthesis in response to nutrients?\n",
            "  Q3. How do environmental factors modulate disease susceptibility and progression in response to genetic and nutritional factors?\n",
            "  Q4. Can environmental factors be used to tailor dietary interventions and disease prevention strategies?\n",
            "  Q5. What are the implications of environmental factors for personalized nutrigenetics and medicine?\n",
            "\n",
            "  Task5: Developing personalized nutrition recommendations\n",
            "  Q1. How can genetic information be used to tailor dietary recommendations and improve health outcomes?\n",
            "  Q2. What are the key genetic variants and nutritional elements to consider in personalized nutrition?\n",
            "  Q3. How do genetic variants interact with nutrients and dietary patterns to influence health outcomes?\n",
            "  Q4. Can personalized nutrition recommendations be used to prevent or manage disease?\n",
            "  Q5. What are the challenges and limitations of implementing personalized nutrition in clinical practice?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Applying nutrigenetics in patient care\n",
            "  Q1. How can healthcare professionals use genetic information to inform dietary recommendations and improve patient outcomes?\n",
            "  Q2. What are the key genetic variants and nutritional elements to consider in clinical practice?\n",
            "  Q3. How do genetic variants interact with nutrients and dietary patterns to influence disease susceptibility and progression?\n",
            "  Q4. Can nutrigenetic insights be used to tailor treatment strategies and improve patient outcomes?\n",
            "  Q5. What are the implications of nutrigenetics for healthcare policy and public health initiatives?\n",
            "\n",
            "  Task2: Understanding the clinical applications of nutrigenetics\n",
            "  Q1. What are the current and potential applications of nutrigenetics in clinical practice?\n",
            "  Q2. How can healthcare professionals stay up-to-date with the latest advances in nutrigenetics?\n",
            "  Q3. What are the challenges and limitations of implementing nutrigenetics in clinical practice?\n",
            "  Q4. How do genetic variants and nutritional elements interact to influence disease susceptibility and progression in clinical populations?\n",
            "  Q5. Can nutrigenetic insights be used to improve patient engagement and adherence to treatment?\n",
            "\n",
            "  Task3: Investigating the role of nutrigenetics in disease management\n",
            "  Q1. How do genetic variants and nutritional elements interact to influence disease progression and treatment response?\n",
            "  Q2. What are the key metabolic pathways and physiological processes affected by genetic-nutrient interactions in disease development?\n",
            "  Q3. Can nutrigenetic insights be used to tailor disease management strategies and improve patient outcomes?\n",
            "  Q4. How do environmental factors modulate the interactions between genetic variants and nutrients in disease management?\n",
            "  Q5. What are the implications of nutrigenetics for disease prevention and management in diverse populations?\n",
            "\n",
            "  Task4: Exploring the intersection of nutrigenetics and pharmacogenetics\n",
            "  Q1. How do genetic variants and nutritional elements interact to influence response to medication and treatment?\n",
            "  Q2. What are the implications of nutrigenetics for pharmacogenetics and personalized medicine?\n",
            "  Q3. Can nutrigenetic insights be used to tailor treatment strategies and improve patient outcomes?\n",
            "  Q4. How do genetic variants and nutritional elements interact to influence disease susceptibility and progression in response to treatment?\n",
            "  Q5. What are the challenges and limitations of integrating nutrigenetics and pharmacogenetics in clinical practice?\n",
            "\n",
            "  Task5: Communicating nutrigenetic information to patients\n",
            "  Q1. How can healthcare professionals effectively communicate nutrigenetic information to patients?\n",
            "  Q2. What are the key challenges and limitations of communicating complex genetic information to patients?\n",
            "  Q3. How can healthcare professionals use nutrigenetic insights to engage patients in disease prevention and management?\n",
            "  Q4. What are the implications of nutrigenetics for patient empowerment and health literacy?\n",
            "  Q5. Can nutrigenetic insights be used to improve patient-provider communication and health outcomes?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understanding personalized nutrition\n",
            "  Q1. How can genetic information be used to tailor dietary recommendations and improve health outcomes?\n",
            "  Q2. What are the key genetic variants and nutritional elements to consider in personalized nutrition?\n",
            "  Q3. How do genetic variants interact with nutrients and dietary patterns to influence health outcomes?\n",
            "  Q4. Can personalized nutrition recommendations be used to prevent or manage disease?\n",
            "  Q5. What are the benefits and limitations of using genetic information for personalized nutrition?\n",
            "\n",
            "  Task2: Exploring the role of genetic testing in personalized nutrition\n",
            "  Q1. What are the current and potential applications of genetic testing in personalized nutrition?\n",
            "  Q2. How can genetic testing be used to inform dietary recommendations and improve health outcomes?\n",
            "  Q3. What are the challenges and limitations of using genetic testing for personalized nutrition?\n",
            "  Q4. How do genetic variants and nutritional elements interact to influence disease susceptibility and progression?\n",
            "  Q5. Can genetic testing be used to tailor dietary interventions and improve health outcomes?\n",
            "\n",
            "  Task3: Investigating the impact of dietary patterns on health outcomes\n",
            "  Q1. How do dietary patterns and nutritional elements interact with genetic variants to influence health outcomes?\n",
            "  Q2. What are the key metabolic pathways and physiological processes affected by dietary patterns and nutritional elements?\n",
            "  Q3. Can dietary patterns and nutritional elements be used to modulate disease susceptibility and progression?\n",
            "  Q4. How do environmental factors influence the interactions between dietary patterns and genetic variants?\n",
            "  Q5. What are the implications of dietary patterns and nutritional elements for personalized nutrition and health?\n",
            "\n",
            "  Task4: Understanding the relationship between nutrigenetics and disease prevention\n",
            "  Q1. How do genetic variants and nutritional elements interact to influence disease susceptibility and progression?\n",
            "  Q2. What are the key genetic variants and nutritional elements to consider in disease prevention?\n",
            "  Q3. Can nutrigenetic insights be used to tailor disease prevention strategies and improve health outcomes?\n",
            "  Q4. How do environmental factors modulate the interactions between genetic variants and nutrients in disease prevention?\n",
            "  Q5. What are the implications of nutrigenetics for disease prevention and public health initiatives?\n",
            "\n",
            "  Task5: Evaluating the effectiveness of personalized nutrition\n",
            "  Q1. How can the effectiveness of personalized nutrition recommendations be evaluated and measured?\n",
            "  Q2. What are the key challenges and limitations of evaluating personalized nutrition?\n",
            "  Q3. How do genetic variants and nutritional elements interact to influence health outcomes in response to personalized nutrition?\n",
            "  Q4. Can personalized nutrition recommendations be used to improve health outcomes and reduce healthcare costs?\n",
            "  Q5. What are the implications of personalized nutrition for healthcare policy and public health initiatives?\n",
            "\"\n",
            "}\n",
            "```\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Here is the output:\n",
            "\n",
            "```\n",
            "{\n",
            "  \"corpus_description\": \"The corpus provides a comprehensive understanding of the complex relationships between genetic variants, nutritional elements, and their impact on health and disease outcomes. The papers highlight the importance of gene-nutrient interactions in modulating physiological processes, metabolic pathways, and disease susceptibility. The findings have implications for personalized nutrition and disease prevention strategies. The corpus also underscores the need for further research into the mechanisms underlying gene-nutrient interactions and their effects on human health.\",\n",
            "  \"generated_questions\": \"\n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Investigating the relationship between genetic variants and nutrient response.\n",
            "  Q1. How do specific genetic variants affect the metabolism of nutrients and their subsequent impact on physiological processes?\n",
            "  Q2. What are the underlying molecular mechanisms by which genetic variants influence individual responses to dietary patterns?\n",
            "  Q3. How do genetic variants interact with environmental factors to modulate nutrient effects on health outcomes?\n",
            "  Q4. What is the role of epigenetic modifications in mediating the effects of genetic variants on nutrient response?\n",
            "  Q5. How can genetic variant-nutrient interactions be leveraged to develop personalized nutrition recommendations?\n",
            "\n",
            "  Task2: Exploring the impact of nutrient-gene interactions on disease susceptibility.\n",
            "  Q1. How do specific nutrients or dietary patterns affect gene expression and contribute to disease risk or progression?\n",
            "  Q2. What are the key metabolic pathways and molecular interactions involved in the development of diseases influenced by gene-nutrient interactions?\n",
            "  Q3. Can genetic variants be used to predict an individual's susceptibility to diet-related diseases?\n",
            "  Q4. How do environmental factors influence the relationship between genetic variants, nutrient response, and disease susceptibility?\n",
            "  Q5. What are the implications of gene-nutrient interactions for the development of targeted disease prevention strategies?\n",
            "\n",
            "  Task3: Analyzing the effects of gene-nutrient interactions on physiological processes.\n",
            "  Q1. How do genetic variants affect the regulation of physiological processes, such as glucose metabolism or lipid metabolism, in response to dietary patterns?\n",
            "  Q2. What are the consequences of genetic variant-nutrient interactions on metabolic pathway flux and physiological outcomes?\n",
            "  Q3. Can nutrient-gene interactions be used to modulate physiological processes and improve health outcomes?\n",
            "  Q4. How do environmental factors, such as lifestyle or exposure to toxins, influence the effects of genetic variants on physiological processes?\n",
            "  Q5. What are the potential therapeutic applications of targeting gene-nutrient interactions to modulate physiological processes?\n",
            "\n",
            "  Task4: Investigating the role of nutrigenomics in personalized nutrition.\n",
            "  Q1. How can genetic information be used to tailor dietary recommendations to an individual's genetic profile?\n",
            "  Q2. What are the challenges and limitations of implementing personalized nutrition based on genetic information?\n",
            "  Q3. How do genetic variants affect the response to specific nutrients or dietary patterns, and what are the implications for personalized nutrition?\n",
            "  Q4. Can nutrigenomics be used to predict an individual's response to different dietary interventions?\n",
            "  Q5. What are the potential benefits and risks of using genetic information to guide dietary recommendations?\n",
            "\n",
            "  Task5: Examining the relationship between gene-nutrient interactions and environmental factors.\n",
            "  Q1. How do environmental factors, such as diet, lifestyle, or exposure to toxins, influence gene-nutrient interactions and health outcomes?\n",
            "  Q2. What are the mechanisms by which environmental factors modulate the effects of genetic variants on nutrient response and disease susceptibility?\n",
            "  Q3. Can environmental factors be used to mitigate or exacerbate the effects of genetic variants on health outcomes?\n",
            "  Q4. How do genetic variants interact with environmental factors to influence epigenetic modifications and gene expression?\n",
            "  Q5. What are the implications of gene-environment interactions for the development of targeted disease prevention strategies?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Applying nutrigenetic knowledge to patient care.\n",
            "  Q1. How can healthcare professionals use genetic information to inform dietary recommendations for patients with specific disease conditions?\n",
            "  Q2. What are the potential benefits and limitations of incorporating nutrigenetic testing into clinical practice?\n",
            "  Q3. How do genetic variants affect the response to different nutritional interventions, and what are the implications for patient care?\n",
            "  Q4. Can nutrigenomics be used to predict patient responses to different dietary patterns or nutritional supplements?\n",
            "  Q5. What are the challenges and opportunities for integrating nutrigenetic insights into clinical practice?\n",
            "\n",
            "  Task2: Exploring the role of nutrigenomics in disease prevention.\n",
            "  Q1. How can nutrigenetic insights be used to prevent or manage chronic diseases, such as cardiovascular disease or diabetes?\n",
            "  Q2. What are the key genetic variants and nutrient interactions involved in the development of diet-related diseases?\n",
            "  Q3. Can genetic testing be used to identify individuals at high risk for diet-related diseases, and what are the implications for disease prevention?\n",
            "  Q4. How do environmental factors influence the relationship between genetic variants and disease susceptibility?\n",
            "  Q5. What are the potential therapeutic applications of targeting gene-nutrient interactions to prevent or manage disease?\n",
            "\n",
            "  Task3: Understanding the limitations and challenges of nutrigenetic testing.\n",
            "  Q1. What are the limitations and challenges of using genetic testing to inform dietary recommendations or disease risk assessment?\n",
            "  Q2. How do genetic variants interact with environmental factors to influence health outcomes, and what are the implications for disease prevention?\n",
            "  Q3. Can genetic testing be used to predict an individual's response to specific nutritional interventions?\n",
            "  Q4. What are the potential risks and benefits of using genetic information to guide dietary recommendations?\n",
            "  Q5. How can healthcare professionals stay up-to-date with the rapidly evolving field of nutrigenomics?\n",
            "\n",
            "  Task4: Applying nutrigenetic insights to patient education.\n",
            "  Q1. How can healthcare professionals communicate complex nutrigenetic information to patients in a clear and actionable way?\n",
            "  Q2. What are the key concepts and principles that healthcare professionals should understand when applying nutrigenetic insights in clinical practice?\n",
            "  Q3. Can nutrigenomics be used to empower patients to take a more active role in managing their health through dietary choices?\n",
            "  Q4. How do genetic variants affect an individual's ability to respond to dietary advice or nutritional interventions?\n",
            "  Q5. What are the potential benefits and challenges of incorporating nutrigenetic testing into patient education and counseling?\n",
            "\n",
            "  Task5: Exploring the future of nutrigenomics in clinical practice.\n",
            "  Q1. What are the potential future directions for the application of nutrigenomics in clinical practice?\n",
            "  Q2. How will advances in genetic testing and nutrigenomics impact the delivery of healthcare services?\n",
            "  Q3. Can nutrigenomics be used to develop more effective and personalized treatment strategies for diet-related diseases?\n",
            "  Q4. What are the implications of nutrigenomics for public health policy and disease prevention initiatives?\n",
            "  Q5. How will the integration of nutrigenomics into clinical practice influence the role of healthcare professionals?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understanding the basics of nutrigenomics.\n",
            "  Q1. What are the key concepts and principles of nutrigenomics, and how do they relate to personalized nutrition?\n",
            "  Q2. How do genetic variants affect an individual's response to different nutrients and dietary patterns?\n",
            "  Q3. Can genetic testing be used to identify optimal dietary patterns or nutritional supplements for an individual?\n",
            "  Q4. What are the potential benefits and limitations of using genetic information to guide dietary choices?\n",
            "  Q5. How can I use nutrigenomic insights to take control of my health and well-being?\n",
            "\n",
            "  Task2: Exploring the role of genetic testing in personalized nutrition.\n",
            "  Q1. What types of genetic testing are available for nutrigenomics, and what do they entail?\n",
            "  Q2. How can I use genetic testing to inform my dietary choices and improve my health outcomes?\n",
            "  Q3. Can genetic testing be used to predict my susceptibility to diet-related diseases?\n",
            "  Q4. What are the potential risks and benefits of using genetic information to guide dietary recommendations?\n",
            "  Q5. How can I ensure that my genetic information is used responsibly and confidentially?\n",
            "\n",
            "  Task3: Understanding the implications of nutrigenomics for dietary choices.\n",
            "  Q1. How do genetic variants affect my response to different foods and nutrients, and what are the implications for my dietary choices?\n",
            "  Q2. Can nutrigenomics be used to identify optimal dietary patterns or nutritional supplements for my genetic profile?\n",
            "  Q3. How do environmental factors influence the relationship between genetic variants and nutrient response?\n",
            "  Q4. What are the potential benefits and limitations of using nutrigenomic insights to guide my dietary choices?\n",
            "  Q5. How can I stay up-to-date with the rapidly evolving field of nutrigenomics and its implications for my health?\n",
            "\n",
            "  Task4: Investigating the relationship between gene-nutrient interactions and disease risk.\n",
            "  Q1. How do genetic variants interact with nutrients and dietary patterns to influence disease risk or progression?\n",
            "  Q2. Can genetic testing be used to predict my susceptibility to diet-related diseases?\n",
            "  Q3. What are the key metabolic pathways and molecular interactions involved in the development of diseases influenced by gene-nutrient interactions?\n",
            "  Q4. How do environmental factors influence the relationship between genetic variants, nutrient response, and disease susceptibility?\n",
            "  Q5. What are the implications of gene-nutrient interactions for my health and well-being?\n",
            "\n",
            "  Task5: Exploring the future of personalized nutrition.\n",
            "  Q1. What are the potential future directions for the application of nutrigenomics in personalized nutrition?\n",
            "  Q2. How will advances in genetic testing and nutrigenomics impact the delivery of nutrition services?\n",
            "  Q3. Can nutrigenomics be used to develop more effective and personalized nutrition recommendations?\n",
            "  Q4. What are the implications of nutrigenomics for public health policy and disease prevention initiatives?\n",
            "  Q5. How will the integration of nutrigenomics into personalized nutrition influence the role of healthcare professionals and registered dietitians?\"\n",
            "}\n",
            "```\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Here is the output:\n",
            "\n",
            "```\n",
            "{\n",
            "  \"corpus_description\": \"The corpus provides insights into the complex relationships between genetic variants, nutritional elements, and human health and disease outcomes. The papers highlight the importance of understanding the genetic basis of disease susceptibility and progression, as well as the role of environmental factors, such as nutrition and lifestyle interventions. The corpus covers a range of biomedical entities, including genes, genetic variants, proteins, and diseases, and demonstrates the utility of functional studies in elucidating the effects of genetic variants on protein activity and cellular processes. Overall, the corpus contributes to our understanding of the intricate relationships between genetic and environmental factors and their impact on human health and disease outcomes.\",\n",
            "  \"generated_questions\": \"\n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Investigating the impact of genetic variants on nutrient metabolism\n",
            "  Q1. How do specific genetic variants affect the metabolism of nutrients, such as carbohydrates, proteins, and fats?\n",
            "  Q2. What are the underlying molecular mechanisms by which genetic variants influence nutrient metabolism?\n",
            "  Q3. How do genetic variants interact with dietary patterns to modulate metabolic pathways and physiological processes?\n",
            "  Q4. What are the implications of genetic variants on the development of metabolic disorders, such as obesity and diabetes?\n",
            "  Q5. How can functional studies be used to elucidate the effects of genetic variants on protein activity and cellular processes in the context of nutrient metabolism?\n",
            "\n",
            "  Task2: Exploring the relationship between genetic variants and dietary responses\n",
            "  Q1. How do genetic variants influence individual responses to different dietary patterns, such as low-carb or low-fat diets?\n",
            "  Q2. What are the key genetic variants associated with differential responses to dietary interventions?\n",
            "  Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk?\n",
            "  Q4. What are the potential applications of genetic testing in personalized nutrition and disease prevention?\n",
            "  Q5. How can systems biology approaches be used to integrate genetic, nutritional, and environmental factors and predict disease outcomes?\n",
            "\n",
            "  Task3: Analyzing the impact of nutrient-gene interactions on disease susceptibility\n",
            "  Q1. How do specific nutrients, such as folate and vitamin D, interact with genetic variants to modulate disease risk?\n",
            "  Q2. What are the underlying biological mechanisms by which nutrient-gene interactions influence disease susceptibility?\n",
            "  Q3. How do genetic variants affect the expression and activity of enzymes involved in nutrient metabolism?\n",
            "  Q4. What are the implications of nutrient-gene interactions for the prevention and treatment of chronic diseases, such as cardiovascular disease and cancer?\n",
            "  Q5. How can nutrigenomic studies inform the development of targeted dietary interventions for disease prevention and treatment?\n",
            "\n",
            "  Task4: Investigating the role of epigenetics in nutrigenomics\n",
            "  Q1. How do environmental factors, such as nutrition and lifestyle, influence epigenetic marks and gene expression?\n",
            "  Q2. What are the key epigenetic mechanisms by which nutrients and dietary patterns modulate gene expression and disease susceptibility?\n",
            "  Q3. How do epigenetic changes contribute to the development of chronic diseases, such as obesity and metabolic disorders?\n",
            "  Q4. What are the potential applications of epigenetic testing in personalized nutrition and disease prevention?\n",
            "  Q5. How can epigenomic studies inform the development of targeted dietary interventions for disease prevention and treatment?\n",
            "\n",
            "  Task5: Developing personalized nutrition recommendations based on genetic information\n",
            "  Q1. How can genetic testing be used to inform personalized nutrition recommendations?\n",
            "  Q2. What are the key genetic variants associated with differential responses to dietary interventions?\n",
            "  Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk?\n",
            "  Q4. What are the potential benefits and limitations of personalized nutrition based on genetic information?\n",
            "  Q5. How can systems biology approaches be used to integrate genetic, nutritional, and environmental factors and predict disease outcomes?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Applying nutrigenetic insights in clinical practice\n",
            "  Q1. How can healthcare professionals use genetic testing to inform dietary recommendations for patients?\n",
            "  Q2. What are the key genetic variants associated with differential responses to dietary interventions in clinical populations?\n",
            "  Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk in clinical populations?\n",
            "  Q4. What are the potential applications of nutrigenetic testing in clinical practice, such as disease prevention and treatment?\n",
            "  Q5. How can healthcare professionals stay up-to-date with the latest advances in nutrigenomics and apply them in clinical practice?\n",
            "\n",
            "  Task2: Investigating the role of nutrigenetics in disease prevention and treatment\n",
            "  Q1. How can nutrigenetic insights be used to prevent or treat chronic diseases, such as cardiovascular disease and cancer?\n",
            "  Q2. What are the key biological mechanisms by which nutrient-gene interactions influence disease susceptibility and progression?\n",
            "  Q3. How do genetic variants affect the expression and activity of enzymes involved in nutrient metabolism in clinical populations?\n",
            "  Q4. What are the implications of nutrient-gene interactions for the prevention and treatment of chronic diseases in clinical populations?\n",
            "  Q5. How can healthcare professionals use nutrigenetic testing to identify patients at high risk of disease and develop targeted interventions?\n",
            "\n",
            "  Task3: Analyzing the impact of nutrient-gene interactions on disease susceptibility\n",
            "  Q1. How do specific nutrients, such as folate and vitamin D, interact with genetic variants to modulate disease risk in clinical populations?\n",
            "  Q2. What are the underlying biological mechanisms by which nutrient-gene interactions influence disease susceptibility and progression?\n",
            "  Q3. How do genetic variants affect the expression and activity of enzymes involved in nutrient metabolism in clinical populations?\n",
            "  Q4. What are the implications of nutrient-gene interactions for the prevention and treatment of chronic diseases in clinical populations?\n",
            "  Q5. How can healthcare professionals use nutrigenetic testing to identify patients at high risk of disease and develop targeted interventions?\n",
            "\n",
            "  Task4: Exploring the relationship between genetic variants and dietary responses in clinical populations\n",
            "  Q1. How do genetic variants influence individual responses to different dietary patterns, such as low-carb or low-fat diets, in clinical populations?\n",
            "  Q2. What are the key genetic variants associated with differential responses to dietary interventions in clinical populations?\n",
            "  Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk in clinical populations?\n",
            "  Q4. What are the potential applications of genetic testing in personalized nutrition and disease prevention in clinical populations?\n",
            "  Q5. How can healthcare professionals stay up-to-date with the latest advances in nutrigenomics and apply them in clinical practice?\n",
            "\n",
            "  Task5: Developing targeted dietary interventions based on genetic information\n",
            "  Q1. How can healthcare professionals use genetic testing to inform personalized nutrition recommendations for patients?\n",
            "  Q2. What are the key genetic variants associated with differential responses to dietary interventions in clinical populations?\n",
            "  Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk in clinical populations?\n",
            "  Q4. What are the potential benefits and limitations of personalized nutrition based on genetic information in clinical populations?\n",
            "  Q5. How can healthcare professionals use nutrigenetic testing to identify patients at high risk of disease and develop targeted interventions?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understanding the basics of nutrigenomics\n",
            "  Q1. What are the key concepts in nutrigenomics and how do they relate to personalized nutrition?\n",
            "  Q2. How do genetic variants influence individual responses to different dietary patterns?\n",
            "  Q3. What are the potential benefits and limitations of personalized nutrition based on genetic information?\n",
            "  Q4. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk?\n",
            "  Q5. What are the current challenges and future directions in the field of nutrigenomics?\n",
            "\n",
            "  Task2: Exploring the relationship between genetic variants and dietary responses\n",
            "  Q1. How do genetic variants affect the metabolism of nutrients, such as carbohydrates, proteins, and fats?\n",
            "  Q2. What are the key genetic variants associated with differential responses to dietary interventions?\n",
            "  Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk?\n",
            "  Q4. What are the implications of genetic variants on the development of metabolic disorders, such as obesity and diabetes?\n",
            "  Q5. How can functional studies be used to elucidate the effects of genetic variants on protein activity and cellular processes in the context of nutrient metabolism?\n",
            "\n",
            "  Task3: Investigating the role of epigenetics in nutrigenomics\n",
            "  Q1. How do environmental factors, such as nutrition and lifestyle, influence epigenetic marks and gene expression?\n",
            "  Q2. What are the key epigenetic mechanisms by which nutrients and dietary patterns modulate gene expression and disease susceptibility?\n",
            "  Q3. How do epigenetic changes contribute to the development of chronic diseases, such as obesity and metabolic disorders?\n",
            "  Q4. What are the potential applications of epigenetic testing in personalized nutrition and disease prevention?\n",
            "  Q5. How can epigenomic studies inform the development of targeted dietary interventions for disease prevention and treatment?\n",
            "\n",
            "  Task4: Analyzing the impact of nutrient-gene interactions on disease susceptibility\n",
            "  Q1. How do specific nutrients, such as folate and vitamin D, interact with genetic variants to modulate disease risk?\n",
            "  Q2. What are the underlying biological mechanisms by which nutrient-gene interactions influence disease susceptibility and progression?\n",
            "  Q3. How do genetic variants affect the expression and activity of enzymes involved in nutrient metabolism?\n",
            "  Q4. What are the implications of nutrient-gene interactions for the prevention and treatment of chronic diseases?\n",
            "  Q5. How can nutrigenomic studies inform the development of targeted dietary interventions for disease prevention and treatment?\n",
            "\n",
            "  Task5: Developing a personalized nutrition plan based on genetic information\n",
            "  Q1. How can genetic testing be used to inform personalized nutrition recommendations?\n",
            "  Q2. What are the key genetic variants associated with differential responses to dietary interventions?\n",
            "  Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk?\n",
            "  Q4. What are the potential benefits and limitations of personalized nutrition based on genetic information?\n",
            "  Q5. How can individuals use nutrigenetic testing to take control of their health and make informed dietary choices?\"\n",
            "}\n",
            "```\n"
          ]
        }
      ],
      "source": [
        "K = 3\n",
        "N = 5\n",
        "M = 5\n",
        "# working_dir = \"/content/drive/MyDrive/workshop-ECAI/\"\n",
        "working_dir = os.getcwd() + \"/documents_from_working_dataset/\"\n",
        "\n",
        "docs_list = [foodal_docs, obwcce_docs, eatbts_docs, vitmin_docs, foodin_docs]\n",
        "labels = [\"foodal\", \"obwcce\", \"eatbts\", \"vitmin\", \"foodin\"]\n",
        "\n",
        "for i, doc in enumerate(docs_list):\n",
        "  with open(working_dir+f'{labels[i]}_answer.txt', 'r') as file:\n",
        "    corpus_description = file.read()\n",
        "\n",
        "  prompt = generate_questions(corpus_description, K, N, M)\n",
        "\n",
        "  chat_completion = client.chat.completions.create(\n",
        "      messages=[\n",
        "          {\n",
        "              \"role\": \"user\",\n",
        "              \"content\": prompt\n",
        "          }\n",
        "      ],\n",
        "      model=MODEL,\n",
        "  )\n",
        "\n",
        "  answer = chat_completion.choices[0].message.content\n",
        "\n",
        "  print(answer)\n",
        "\n",
        "  question_dir = os.getcwd() + \"/questions_files/\"\n",
        "  with open(question_dir+f'{labels[i]}_generated_question.txt', 'w') as f:\n",
        "    f.write(answer)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "fK5-N_u6SuSk"
      },
      "source": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "82eZDMxeSvn3"
      },
      "source": [
        "# Restructuring questions in a DataFrame\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 71,
      "metadata": {
        "id": "ykxVRxq9S1Nc"
      },
      "outputs": [],
      "source": [
        "question_dir = os.getcwd() + \"/questions_files/\"\n",
        "\n",
        "with open(question_dir+'foodal_generated_question.txt', 'r') as file:\n",
        "  foodal_questions = file.read()\n",
        "\n",
        "with open(question_dir+'obwcce_generated_question.txt', 'r') as file:\n",
        "  obwcce_questions = file.read()\n",
        "\n",
        "with open(question_dir+'eatbts_generated_question.txt', 'r') as file:\n",
        "  eatbts_questions = file.read()\n",
        "\n",
        "with open(question_dir+'vitmin_generated_question.txt', 'r') as file:\n",
        "  vitmin_questions = file.read()\n",
        "\n",
        "with open(question_dir+'foodin_generated_question.txt', 'r') as file:\n",
        "  foodin_questions = file.read()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "collapsed": true,
        "id": "toNX4nt_S9FK",
        "outputId": "89223218-1ff5-44a5-d5a1-4112724ef458"
      },
      "outputs": [],
      "source": [
        "print(foodal_questions)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 72,
      "metadata": {
        "id": "G0eSkhwcTX0c"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Processing category: FOODIN\n",
            "Processing category: FOODAL\n",
            "Processing category: OBWCCE\n",
            "Processing category: EATBTS\n",
            "Processing category: VITMIN\n"
          ]
        }
      ],
      "source": [
        "import pandas as pd\n",
        "import re\n",
        "\n",
        "# Step 1: Read file as plain text\n",
        "#-- Fatto nella cella sopra per tutti\n",
        "\n",
        "questions = [foodin_questions, foodal_questions, obwcce_questions, eatbts_questions, vitmin_questions]\n",
        "categories = [\"FOODIN\", \"FOODAL\", \"OBWCCE\", \"EATBTS\", \"VITMIN\"]\n",
        "\n",
        "data = {}\n",
        "\n",
        "for i, q in enumerate(questions):\n",
        "\n",
        "    # Step 2: Manually extract the 'generated_questions' content\n",
        "    match = re.search(r'\"generated_questions\"\\s*:\\s*\"(?P<content>.*)\"\\s*}', q, re.DOTALL)\n",
        "    if not match:\n",
        "        raise ValueError(\"Could not find 'generated_questions' content in the file.\")\n",
        "\n",
        "    content = match.group(\"content\")\n",
        "\n",
        "    # Step 3: Unescape escaped quotes and normalize the content\n",
        "    content = content.replace('\\\\\"', '\"')\n",
        "\n",
        "    # Step 4: Parse lines\n",
        "    lines = content.splitlines()\n",
        "\n",
        "\n",
        "    category = categories[i]  # or derive from file name\n",
        "    print(f\"Processing category: {category}\")\n",
        "\n",
        "    rows = []\n",
        "    current_user = None\n",
        "    current_task = None\n",
        "    questions = []\n",
        "\n",
        "    for line in lines:\n",
        "        line = line.strip()\n",
        "        if not line:\n",
        "            continue\n",
        "        if line.startswith(\"User\"):\n",
        "            current_user = line.split(\":\", 1)[1].strip()\n",
        "        elif line.startswith(\"Task\"):\n",
        "            if current_task and questions:\n",
        "                rows.append({\n",
        "                    \"Category\": category,\n",
        "                    \"USER Description\": current_user,\n",
        "                    \"TASK Description\": current_task,\n",
        "                    \"Questions\": questions\n",
        "                })\n",
        "                questions = []\n",
        "            current_task = line.split(\":\", 1)[1].strip()\n",
        "        elif re.match(r\"Q\\d+\\.\", line):\n",
        "            questions.append(line)\n",
        "\n",
        "    # Append the last block\n",
        "    if current_task and questions:\n",
        "        rows.append({\n",
        "            \"Category\": category,\n",
        "            \"USER Description\": current_user,\n",
        "            \"TASK Description\": current_task,\n",
        "            \"Questions\": questions\n",
        "        })\n",
        "    \n",
        "    data[category] = rows\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 73,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 519
        },
        "id": "bDD92uF_WmmB",
        "outputId": "a41b1874-02fd-46fa-f306-b7a29c263de1"
      },
      "outputs": [
        {
          "data": {
            "application/vnd.microsoft.datawrangler.viewer.v0+json": {
              "columns": [
                {
                  "name": "index",
                  "rawType": "int64",
                  "type": "integer"
                },
                {
                  "name": "Category",
                  "rawType": "object",
                  "type": "string"
                },
                {
                  "name": "USER Description",
                  "rawType": "object",
                  "type": "string"
                },
                {
                  "name": "TASK Description",
                  "rawType": "object",
                  "type": "string"
                },
                {
                  "name": "Questions",
                  "rawType": "object",
                  "type": "unknown"
                }
              ],
              "ref": "5653bddc-b0e4-4a17-8227-67dbcc16580d",
              "rows": [
                [
                  "0",
                  "FOODAL",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Investigating the impact of genetic variants on nutrient metabolism",
                  "['Q1. How do specific genetic variants in the genes involved in carbohydrate metabolism affect the response to dietary carbohydrates?', 'Q2. What are the effects of genetic variants on the expression of enzymes involved in fatty acid metabolism?', 'Q3. How do genetic variants in the genes encoding for nutrient transporters influence nutrient uptake and utilization?', 'Q4. Can genetic variants in the genes involved in epigenetic regulation modulate the response to dietary interventions?', 'Q5. How do genetic variants in the genes involved in inflammation affect the response to dietary patterns and disease risk?']"
                ],
                [
                  "1",
                  "FOODAL",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Exploring the relationship between genetic variants and dietary patterns",
                  "['Q1. What is the relationship between genetic variants in the genes involved in taste perception and dietary preferences?', 'Q2. How do genetic variants in the genes involved in satiety and appetite regulation influence food choices?', 'Q3. Can genetic variants in the genes involved in nutrient sensing affect the response to dietary restriction?', 'Q4. How do genetic variants in the genes involved in gut microbiota modulation influence the response to prebiotics and probiotics?', 'Q5. What are the effects of genetic variants on the response to different types of dietary interventions (e.g., low-carb, low-fat)?']"
                ],
                [
                  "2",
                  "FOODAL",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Analyzing the impact of genetic variants on disease risk",
                  "['Q1. How do genetic variants in the genes involved in insulin signaling affect the risk of developing type 2 diabetes?', 'Q2. What are the effects of genetic variants in the genes involved in lipid metabolism on cardiovascular disease risk?', 'Q3. Can genetic variants in the genes involved in inflammation modulate the risk of developing chronic diseases?', 'Q4. How do genetic variants in the genes involved in antioxidant defense affect the risk of developing age-related diseases?', 'Q5. What are the interactions between genetic variants and dietary patterns in modulating disease risk?']"
                ],
                [
                  "3",
                  "FOODAL",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Investigating the role of epigenetics in nutrigenetics",
                  "['Q1. How do epigenetic modifications affect the response to dietary interventions?', 'Q2. Can epigenetic marks be influenced by dietary patterns and nutrient exposures?', 'Q3. What are the effects of epigenetic modifications on gene expression and protein synthesis?', 'Q4. How do epigenetic mechanisms mediate the interactions between genes and environment?', 'Q5. Can epigenetic modifications be used as biomarkers for disease risk or response to dietary interventions?']"
                ],
                [
                  "4",
                  "FOODAL",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Developing personalized nutrition recommendations",
                  "['Q1. How can genetic variants be used to tailor dietary recommendations for optimal health outcomes?', 'Q2. What are the limitations and challenges of using genetic information for personalized nutrition?', 'Q3. Can genetic variants be used to predict responses to different types of dietary interventions?', 'Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of personalized nutrition recommendations?', 'Q5. What are the potential applications of nutrigenetics in public health policy and nutrition counseling?']"
                ],
                [
                  "5",
                  "FOODAL",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Integrating nutrigenetics into clinical practice",
                  "['Q1. What are the clinical implications of genetic variants in the genes involved in nutrient metabolism?', 'Q2. How can healthcare professionals use genetic information to tailor dietary advice for patients with chronic diseases?', 'Q3. What are the potential benefits and limitations of using genetic testing for nutritional purposes in clinical practice?', 'Q4. Can genetic variants be used to predict responses to nutritional interventions in patients with specific health conditions?', 'Q5. How do healthcare professionals communicate genetic information and dietary recommendations to patients?']"
                ],
                [
                  "6",
                  "FOODAL",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Understanding the role of nutrigenetics in disease prevention",
                  "['Q1. How do genetic variants in the genes involved in inflammation affect the risk of developing chronic diseases?', 'Q2. What are the effects of dietary patterns on gene expression and disease risk?', 'Q3. Can genetic variants be used to identify individuals at high risk of developing specific diseases?', 'Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of disease prevention strategies?', 'Q5. What are the potential applications of nutrigenetics in disease prevention and management?']"
                ],
                [
                  "7",
                  "FOODAL",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Exploring the relationship between genetic variants and nutritional deficiencies",
                  "['Q1. How do genetic variants in the genes involved in nutrient metabolism affect the risk of nutritional deficiencies?', 'Q2. What are the effects of genetic variants on the absorption and utilization of essential nutrients?', 'Q3. Can genetic variants be used to predict responses to nutritional supplements?', 'Q4. How do genetic variants in the genes involved in taste perception and food preferences influence nutritional deficiencies?', 'Q5. What are the implications of genetic variants for nutritional deficiency diagnosis and treatment?']"
                ],
                [
                  "8",
                  "FOODAL",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Investigating the impact of epigenetics on clinical practice",
                  "['Q1. How do epigenetic modifications affect the response to dietary interventions in patients with chronic diseases?', 'Q2. Can epigenetic marks be used as biomarkers for disease risk or response to dietary interventions?', 'Q3. What are the effects of epigenetic modifications on gene expression and protein synthesis in patients with specific health conditions?', 'Q4. How do epigenetic mechanisms mediate the interactions between genes and environment in clinical practice?', 'Q5. Can epigenetic modifications be used to monitor the effectiveness of nutritional interventions?']"
                ],
                [
                  "9",
                  "FOODAL",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Developing evidence-based guidelines for nutrigenetic testing",
                  "['Q1. What are the current evidence-based guidelines for using genetic testing for nutritional purposes in clinical practice?', 'Q2. How can healthcare professionals stay up-to-date with the rapidly evolving field of nutrigenetics?', 'Q3. What are the potential risks and benefits of using genetic testing for nutritional purposes in clinical practice?', 'Q4. How do healthcare professionals communicate genetic information and dietary recommendations to patients in a clear and actionable way?', 'Q5. What are the implications of genetic variants for clinical decision-making and patient care?']"
                ],
                [
                  "10",
                  "FOODAL",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Understanding the basics of nutrigenetics",
                  "['Q1. What are the key concepts in nutrigenetics and how do they relate to personalized nutrition?', 'Q2. How do genetic variants affect my response to different nutrients and dietary patterns?', 'Q3. Can genetic testing help me understand my nutritional needs and develop a personalized diet plan?', 'Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of personalized nutrition recommendations?', 'Q5. What are the potential benefits and limitations of using genetic testing for nutritional purposes?']"
                ],
                [
                  "11",
                  "FOODAL",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Exploring the role of genetic variants in dietary responses",
                  "['Q1. How do genetic variants in the genes involved in carbohydrate metabolism affect my response to dietary carbohydrates?', 'Q2. What are the effects of genetic variants on my ability to lose weight or maintain weight loss?', 'Q3. Can genetic variants be used to predict my response to different types of dietary interventions (e.g., low-carb, low-fat)?', 'Q4. How do genetic variants in the genes involved in inflammation affect my risk of developing chronic diseases?', 'Q5. What are the implications of genetic variants for my nutritional needs and dietary recommendations?']"
                ],
                [
                  "12",
                  "FOODAL",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Investigating the impact of epigenetics on nutrition",
                  "['Q1. How do epigenetic modifications affect my response to dietary interventions?', 'Q2. Can epigenetic marks be influenced by my dietary patterns and nutrient exposures?', 'Q3. What are the effects of epigenetic modifications on my gene expression and protein synthesis?', 'Q4. How do epigenetic mechanisms mediate the interactions between my genes and environment?', 'Q5. Can epigenetic modifications be used to monitor the effectiveness of my nutritional interventions?']"
                ],
                [
                  "13",
                  "FOODAL",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Understanding the relationship between genetic variants and disease risk",
                  "['Q1. How do genetic variants in the genes involved in insulin signaling affect my risk of developing type 2 diabetes?', 'Q2. What are the effects of genetic variants in the genes involved in lipid metabolism on my cardiovascular disease risk?', 'Q3. Can genetic variants be used to predict my risk of developing specific diseases?', 'Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of disease prevention strategies?', 'Q5. What are the potential applications of nutrigenetics in disease prevention and management?']"
                ],
                [
                  "14",
                  "FOODAL",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Applying nutrigenetic insights to daily life",
                  "['Q1. How can I use genetic information to tailor my dietary recommendations for optimal health outcomes?', 'Q2. What are the practical implications of genetic variants for my daily food choices?', 'Q3. Can genetic variants be used to predict my response to different types of dietary interventions?', 'Q4. How do I incorporate genetic information into my overall health and wellness plan?', 'Q5. What are the potential benefits and limitations of using genetic testing for nutritional purposes in everyday life?']"
                ]
              ],
              "shape": {
                "columns": 4,
                "rows": 15
              }
            },
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Category</th>\n",
              "      <th>USER Description</th>\n",
              "      <th>TASK Description</th>\n",
              "      <th>Questions</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A researcher in nutrigenetics, interested in u...</td>\n",
              "      <td>Investigating the impact of genetic variants o...</td>\n",
              "      <td>[Q1. How do specific genetic variants in the g...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A researcher in nutrigenetics, interested in u...</td>\n",
              "      <td>Exploring the relationship between genetic var...</td>\n",
              "      <td>[Q1. What is the relationship between genetic ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A researcher in nutrigenetics, interested in u...</td>\n",
              "      <td>Analyzing the impact of genetic variants on di...</td>\n",
              "      <td>[Q1. How do genetic variants in the genes invo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A researcher in nutrigenetics, interested in u...</td>\n",
              "      <td>Investigating the role of epigenetics in nutri...</td>\n",
              "      <td>[Q1. How do epigenetic modifications affect th...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A healthcare professional looking to apply nut...</td>\n",
              "      <td>Developing personalized nutrition recommendations</td>\n",
              "      <td>[Q1. How can genetic variants be used to tailo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A healthcare professional looking to apply nut...</td>\n",
              "      <td>Integrating nutrigenetics into clinical practice</td>\n",
              "      <td>[Q1. What are the clinical implications of gen...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A healthcare professional looking to apply nut...</td>\n",
              "      <td>Understanding the role of nutrigenetics in dis...</td>\n",
              "      <td>[Q1. How do genetic variants in the genes invo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A healthcare professional looking to apply nut...</td>\n",
              "      <td>Exploring the relationship between genetic var...</td>\n",
              "      <td>[Q1. How do genetic variants in the genes invo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A healthcare professional looking to apply nut...</td>\n",
              "      <td>Investigating the impact of epigenetics on cli...</td>\n",
              "      <td>[Q1. How do epigenetic modifications affect th...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A customer interested in personalized nutritio...</td>\n",
              "      <td>Developing evidence-based guidelines for nutri...</td>\n",
              "      <td>[Q1. What are the current evidence-based guide...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>10</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A customer interested in personalized nutritio...</td>\n",
              "      <td>Understanding the basics of nutrigenetics</td>\n",
              "      <td>[Q1. What are the key concepts in nutrigenetic...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>11</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A customer interested in personalized nutritio...</td>\n",
              "      <td>Exploring the role of genetic variants in diet...</td>\n",
              "      <td>[Q1. How do genetic variants in the genes invo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>12</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A customer interested in personalized nutritio...</td>\n",
              "      <td>Investigating the impact of epigenetics on nut...</td>\n",
              "      <td>[Q1. How do epigenetic modifications affect my...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>13</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A customer interested in personalized nutritio...</td>\n",
              "      <td>Understanding the relationship between genetic...</td>\n",
              "      <td>[Q1. How do genetic variants in the genes invo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>14</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A customer interested in personalized nutritio...</td>\n",
              "      <td>Applying nutrigenetic insights to daily life</td>\n",
              "      <td>[Q1. How can I use genetic information to tail...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "   Category                                   USER Description  \\\n",
              "0    FOODAL  A researcher in nutrigenetics, interested in u...   \n",
              "1    FOODAL  A researcher in nutrigenetics, interested in u...   \n",
              "2    FOODAL  A researcher in nutrigenetics, interested in u...   \n",
              "3    FOODAL  A researcher in nutrigenetics, interested in u...   \n",
              "4    FOODAL  A healthcare professional looking to apply nut...   \n",
              "5    FOODAL  A healthcare professional looking to apply nut...   \n",
              "6    FOODAL  A healthcare professional looking to apply nut...   \n",
              "7    FOODAL  A healthcare professional looking to apply nut...   \n",
              "8    FOODAL  A healthcare professional looking to apply nut...   \n",
              "9    FOODAL  A customer interested in personalized nutritio...   \n",
              "10   FOODAL  A customer interested in personalized nutritio...   \n",
              "11   FOODAL  A customer interested in personalized nutritio...   \n",
              "12   FOODAL  A customer interested in personalized nutritio...   \n",
              "13   FOODAL  A customer interested in personalized nutritio...   \n",
              "14   FOODAL  A customer interested in personalized nutritio...   \n",
              "\n",
              "                                     TASK Description  \\\n",
              "0   Investigating the impact of genetic variants o...   \n",
              "1   Exploring the relationship between genetic var...   \n",
              "2   Analyzing the impact of genetic variants on di...   \n",
              "3   Investigating the role of epigenetics in nutri...   \n",
              "4   Developing personalized nutrition recommendations   \n",
              "5    Integrating nutrigenetics into clinical practice   \n",
              "6   Understanding the role of nutrigenetics in dis...   \n",
              "7   Exploring the relationship between genetic var...   \n",
              "8   Investigating the impact of epigenetics on cli...   \n",
              "9   Developing evidence-based guidelines for nutri...   \n",
              "10          Understanding the basics of nutrigenetics   \n",
              "11  Exploring the role of genetic variants in diet...   \n",
              "12  Investigating the impact of epigenetics on nut...   \n",
              "13  Understanding the relationship between genetic...   \n",
              "14       Applying nutrigenetic insights to daily life   \n",
              "\n",
              "                                            Questions  \n",
              "0   [Q1. How do specific genetic variants in the g...  \n",
              "1   [Q1. What is the relationship between genetic ...  \n",
              "2   [Q1. How do genetic variants in the genes invo...  \n",
              "3   [Q1. How do epigenetic modifications affect th...  \n",
              "4   [Q1. How can genetic variants be used to tailo...  \n",
              "5   [Q1. What are the clinical implications of gen...  \n",
              "6   [Q1. How do genetic variants in the genes invo...  \n",
              "7   [Q1. How do genetic variants in the genes invo...  \n",
              "8   [Q1. How do epigenetic modifications affect th...  \n",
              "9   [Q1. What are the current evidence-based guide...  \n",
              "10  [Q1. What are the key concepts in nutrigenetic...  \n",
              "11  [Q1. How do genetic variants in the genes invo...  \n",
              "12  [Q1. How do epigenetic modifications affect my...  \n",
              "13  [Q1. How do genetic variants in the genes invo...  \n",
              "14  [Q1. How can I use genetic information to tail...  "
            ]
          },
          "execution_count": 73,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# Step 5: Create DataFrame\n",
        "df_foodal = pd.DataFrame(data[\"FOODAL\"])\n",
        "df_obwcce = pd.DataFrame(data[\"OBWCCE\"])\n",
        "df_eatbts = pd.DataFrame(data[\"EATBTS\"])\n",
        "df_vitmin = pd.DataFrame(data[\"VITMIN\"])\n",
        "df_foodin = pd.DataFrame(data[\"FOODIN\"])\n",
        "df_foodal"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 74,
      "metadata": {
        "id": "RW-UDhrIWnfD"
      },
      "outputs": [],
      "source": [
        "# prompt: append all these dataframe together: df_foodal\n",
        "\n",
        "df_combined = pd.concat([df_foodal, df_obwcce, df_eatbts, df_vitmin, df_foodin], ignore_index=True)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 84,
      "metadata": {
        "id": "n3RjtCnBXdco"
      },
      "outputs": [
        {
          "data": {
            "application/vnd.microsoft.datawrangler.viewer.v0+json": {
              "columns": [
                {
                  "name": "index",
                  "rawType": "int64",
                  "type": "integer"
                },
                {
                  "name": "Category",
                  "rawType": "object",
                  "type": "string"
                },
                {
                  "name": "USER Description",
                  "rawType": "object",
                  "type": "string"
                },
                {
                  "name": "TASK Description",
                  "rawType": "object",
                  "type": "string"
                },
                {
                  "name": "Questions",
                  "rawType": "object",
                  "type": "unknown"
                }
              ],
              "ref": "086b6670-5f68-42cf-b36a-2c7d009b3ae6",
              "rows": [
                [
                  "0",
                  "FOODAL",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Investigating the impact of genetic variants on nutrient metabolism",
                  "['Q1. How do specific genetic variants in the genes involved in carbohydrate metabolism affect the response to dietary carbohydrates?', 'Q2. What are the effects of genetic variants on the expression of enzymes involved in fatty acid metabolism?', 'Q3. How do genetic variants in the genes encoding for nutrient transporters influence nutrient uptake and utilization?', 'Q4. Can genetic variants in the genes involved in epigenetic regulation modulate the response to dietary interventions?', 'Q5. How do genetic variants in the genes involved in inflammation affect the response to dietary patterns and disease risk?']"
                ],
                [
                  "1",
                  "FOODAL",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Exploring the relationship between genetic variants and dietary patterns",
                  "['Q1. What is the relationship between genetic variants in the genes involved in taste perception and dietary preferences?', 'Q2. How do genetic variants in the genes involved in satiety and appetite regulation influence food choices?', 'Q3. Can genetic variants in the genes involved in nutrient sensing affect the response to dietary restriction?', 'Q4. How do genetic variants in the genes involved in gut microbiota modulation influence the response to prebiotics and probiotics?', 'Q5. What are the effects of genetic variants on the response to different types of dietary interventions (e.g., low-carb, low-fat)?']"
                ],
                [
                  "2",
                  "FOODAL",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Analyzing the impact of genetic variants on disease risk",
                  "['Q1. How do genetic variants in the genes involved in insulin signaling affect the risk of developing type 2 diabetes?', 'Q2. What are the effects of genetic variants in the genes involved in lipid metabolism on cardiovascular disease risk?', 'Q3. Can genetic variants in the genes involved in inflammation modulate the risk of developing chronic diseases?', 'Q4. How do genetic variants in the genes involved in antioxidant defense affect the risk of developing age-related diseases?', 'Q5. What are the interactions between genetic variants and dietary patterns in modulating disease risk?']"
                ],
                [
                  "3",
                  "FOODAL",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Investigating the role of epigenetics in nutrigenetics",
                  "['Q1. How do epigenetic modifications affect the response to dietary interventions?', 'Q2. Can epigenetic marks be influenced by dietary patterns and nutrient exposures?', 'Q3. What are the effects of epigenetic modifications on gene expression and protein synthesis?', 'Q4. How do epigenetic mechanisms mediate the interactions between genes and environment?', 'Q5. Can epigenetic modifications be used as biomarkers for disease risk or response to dietary interventions?']"
                ],
                [
                  "4",
                  "FOODAL",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Developing personalized nutrition recommendations",
                  "['Q1. How can genetic variants be used to tailor dietary recommendations for optimal health outcomes?', 'Q2. What are the limitations and challenges of using genetic information for personalized nutrition?', 'Q3. Can genetic variants be used to predict responses to different types of dietary interventions?', 'Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of personalized nutrition recommendations?', 'Q5. What are the potential applications of nutrigenetics in public health policy and nutrition counseling?']"
                ],
                [
                  "5",
                  "FOODAL",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Integrating nutrigenetics into clinical practice",
                  "['Q1. What are the clinical implications of genetic variants in the genes involved in nutrient metabolism?', 'Q2. How can healthcare professionals use genetic information to tailor dietary advice for patients with chronic diseases?', 'Q3. What are the potential benefits and limitations of using genetic testing for nutritional purposes in clinical practice?', 'Q4. Can genetic variants be used to predict responses to nutritional interventions in patients with specific health conditions?', 'Q5. How do healthcare professionals communicate genetic information and dietary recommendations to patients?']"
                ],
                [
                  "6",
                  "FOODAL",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Understanding the role of nutrigenetics in disease prevention",
                  "['Q1. How do genetic variants in the genes involved in inflammation affect the risk of developing chronic diseases?', 'Q2. What are the effects of dietary patterns on gene expression and disease risk?', 'Q3. Can genetic variants be used to identify individuals at high risk of developing specific diseases?', 'Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of disease prevention strategies?', 'Q5. What are the potential applications of nutrigenetics in disease prevention and management?']"
                ],
                [
                  "7",
                  "FOODAL",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Exploring the relationship between genetic variants and nutritional deficiencies",
                  "['Q1. How do genetic variants in the genes involved in nutrient metabolism affect the risk of nutritional deficiencies?', 'Q2. What are the effects of genetic variants on the absorption and utilization of essential nutrients?', 'Q3. Can genetic variants be used to predict responses to nutritional supplements?', 'Q4. How do genetic variants in the genes involved in taste perception and food preferences influence nutritional deficiencies?', 'Q5. What are the implications of genetic variants for nutritional deficiency diagnosis and treatment?']"
                ],
                [
                  "8",
                  "FOODAL",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Investigating the impact of epigenetics on clinical practice",
                  "['Q1. How do epigenetic modifications affect the response to dietary interventions in patients with chronic diseases?', 'Q2. Can epigenetic marks be used as biomarkers for disease risk or response to dietary interventions?', 'Q3. What are the effects of epigenetic modifications on gene expression and protein synthesis in patients with specific health conditions?', 'Q4. How do epigenetic mechanisms mediate the interactions between genes and environment in clinical practice?', 'Q5. Can epigenetic modifications be used to monitor the effectiveness of nutritional interventions?']"
                ],
                [
                  "9",
                  "FOODAL",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Developing evidence-based guidelines for nutrigenetic testing",
                  "['Q1. What are the current evidence-based guidelines for using genetic testing for nutritional purposes in clinical practice?', 'Q2. How can healthcare professionals stay up-to-date with the rapidly evolving field of nutrigenetics?', 'Q3. What are the potential risks and benefits of using genetic testing for nutritional purposes in clinical practice?', 'Q4. How do healthcare professionals communicate genetic information and dietary recommendations to patients in a clear and actionable way?', 'Q5. What are the implications of genetic variants for clinical decision-making and patient care?']"
                ],
                [
                  "10",
                  "FOODAL",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Understanding the basics of nutrigenetics",
                  "['Q1. What are the key concepts in nutrigenetics and how do they relate to personalized nutrition?', 'Q2. How do genetic variants affect my response to different nutrients and dietary patterns?', 'Q3. Can genetic testing help me understand my nutritional needs and develop a personalized diet plan?', 'Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of personalized nutrition recommendations?', 'Q5. What are the potential benefits and limitations of using genetic testing for nutritional purposes?']"
                ],
                [
                  "11",
                  "FOODAL",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Exploring the role of genetic variants in dietary responses",
                  "['Q1. How do genetic variants in the genes involved in carbohydrate metabolism affect my response to dietary carbohydrates?', 'Q2. What are the effects of genetic variants on my ability to lose weight or maintain weight loss?', 'Q3. Can genetic variants be used to predict my response to different types of dietary interventions (e.g., low-carb, low-fat)?', 'Q4. How do genetic variants in the genes involved in inflammation affect my risk of developing chronic diseases?', 'Q5. What are the implications of genetic variants for my nutritional needs and dietary recommendations?']"
                ],
                [
                  "12",
                  "FOODAL",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Investigating the impact of epigenetics on nutrition",
                  "['Q1. How do epigenetic modifications affect my response to dietary interventions?', 'Q2. Can epigenetic marks be influenced by my dietary patterns and nutrient exposures?', 'Q3. What are the effects of epigenetic modifications on my gene expression and protein synthesis?', 'Q4. How do epigenetic mechanisms mediate the interactions between my genes and environment?', 'Q5. Can epigenetic modifications be used to monitor the effectiveness of my nutritional interventions?']"
                ],
                [
                  "13",
                  "FOODAL",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Understanding the relationship between genetic variants and disease risk",
                  "['Q1. How do genetic variants in the genes involved in insulin signaling affect my risk of developing type 2 diabetes?', 'Q2. What are the effects of genetic variants in the genes involved in lipid metabolism on my cardiovascular disease risk?', 'Q3. Can genetic variants be used to predict my risk of developing specific diseases?', 'Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of disease prevention strategies?', 'Q5. What are the potential applications of nutrigenetics in disease prevention and management?']"
                ],
                [
                  "14",
                  "FOODAL",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Applying nutrigenetic insights to daily life",
                  "['Q1. How can I use genetic information to tailor my dietary recommendations for optimal health outcomes?', 'Q2. What are the practical implications of genetic variants for my daily food choices?', 'Q3. Can genetic variants be used to predict my response to different types of dietary interventions?', 'Q4. How do I incorporate genetic information into my overall health and wellness plan?', 'Q5. What are the potential benefits and limitations of using genetic testing for nutritional purposes in everyday life?']"
                ],
                [
                  "15",
                  "OBWCCE",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Investigate the relationship between genetic variants and nutrient responses",
                  "['Q1. How do specific genetic variants affect the metabolism of nutrients and influence physiological processes?', 'Q2. What are the downstream effects of genetic variants on metabolic pathways and disease susceptibility?', 'Q3. How do genetic variants interact with dietary patterns to modulate gene expression and protein synthesis?', 'Q4. What is the role of molecular interactions in mediating the effects of genetic variants on nutrient responses?', 'Q5. How do environmental factors influence the interactions between genetic variants and nutrients?']"
                ],
                [
                  "16",
                  "OBWCCE",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Examine the impact of nutritional elements on gene expression",
                  "['Q1. How do particular nutrients or dietary patterns affect the expression of specific genes and transcripts?', 'Q2. What are the effects of nutritional elements on protein synthesis and metabolic pathways?', 'Q3. How do nutritional elements interact with genetic variants to modulate disease risk or progression?', 'Q4. What are the implications of nutritional elements on physiological processes and health outcomes?', 'Q5. How do environmental factors alter the interactions between nutritional elements and genes?']"
                ],
                [
                  "17",
                  "OBWCCE",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Analyze the relationship between genetic variants, nutrients, and disease susceptibility",
                  "['Q1. How do genetic variants interact with nutrients to modulate disease risk or progression?', 'Q2. What are the effects of genetic variants on metabolic pathways and disease susceptibility?', 'Q3. How do dietary patterns influence the interactions between genetic variants and disease susceptibility?', 'Q4. What is the role of molecular interactions in mediating the effects of genetic variants and nutrients on disease outcomes?', 'Q5. How do environmental factors influence the interactions between genetic variants, nutrients, and disease susceptibility?']"
                ],
                [
                  "18",
                  "OBWCCE",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Investigate the role of metabolic pathways in nutrigenetics",
                  "['Q1. What are the key metabolic pathways involved in mediating the effects of genetic variants and nutrients on physiological processes?', 'Q2. How do genetic variants and nutrients interact to modulate metabolic pathways and disease susceptibility?', 'Q3. What are the implications of metabolic pathways on health outcomes and disease risk?', 'Q4. How do environmental factors influence the interactions between genetic variants, nutrients, and metabolic pathways?', 'Q5. What are the potential therapeutic targets for modulating metabolic pathways and disease susceptibility?']"
                ],
                [
                  "19",
                  "OBWCCE",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Explore the applications of nutrigenetics in personalized nutrition",
                  "['Q1. How can genetic information be used to tailor dietary recommendations for individuals?', 'Q2. What are the implications of genetic variants on nutritional requirements and dietary responses?', 'Q3. How can nutrigenetics be used to prevent or manage chronic diseases?', 'Q4. What are the challenges and limitations of implementing personalized nutrition based on genetic information?', 'Q5. How can healthcare professionals integrate nutrigenetics into clinical practice?']"
                ],
                [
                  "20",
                  "OBWCCE",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Apply nutrigenetic knowledge to patient care",
                  "['Q1. How can genetic testing be used to inform dietary recommendations for patients?', 'Q2. What are the potential benefits and limitations of incorporating nutrigenetics into clinical practice?', 'Q3. How can healthcare professionals stay up-to-date with the latest research in nutrigenetics?', 'Q4. What are the implications of genetic variants on disease risk and progression in patients?', 'Q5. How can healthcare professionals communicate complex genetic information to patients?']"
                ],
                [
                  "21",
                  "OBWCCE",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Evaluate the role of nutrigenetics in disease prevention and management",
                  "['Q1. How can nutrigenetics be used to prevent or manage chronic diseases such as diabetes and cardiovascular disease?', 'Q2. What are the key genetic variants associated with disease susceptibility and how can they be used to inform treatment decisions?', 'Q3. How do dietary patterns and nutritional elements interact with genetic variants to modulate disease risk or progression?', 'Q4. What are the potential therapeutic targets for modulating disease susceptibility and progression?', 'Q5. How can healthcare professionals integrate nutrigenetics into treatment plans?']"
                ],
                [
                  "22",
                  "OBWCCE",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Investigate the impact of environmental factors on nutrigenetic interactions",
                  "['Q1. How do environmental factors such as lifestyle and exposure to toxins influence the interactions between genetic variants and nutrients?', 'Q2. What are the implications of environmental factors on disease susceptibility and progression?', 'Q3. How can healthcare professionals take into account environmental factors when making dietary recommendations?', 'Q4. What are the potential strategies for mitigating the effects of environmental factors on nutrigenetic interactions?', 'Q5. How can healthcare professionals stay informed about the latest research on environmental factors and nutrigenetic interactions?']"
                ],
                [
                  "23",
                  "OBWCCE",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Examine the role of nutrigenetics in personalized medicine",
                  "['Q1. How can nutrigenetics be integrated into personalized medicine approaches?', 'Q2. What are the potential benefits and challenges of using nutrigenetics in personalized medicine?', 'Q3. How can healthcare professionals use genetic information to tailor treatment plans?', 'Q4. What are the implications of nutrigenetics on health outcomes and disease risk?', 'Q5. How can healthcare professionals communicate complex genetic information to patients in a personalized medicine context?']"
                ],
                [
                  "24",
                  "OBWCCE",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Explore the future of nutrigenetics in healthcare",
                  "['Q1. What are the potential future directions for research in nutrigenetics?', 'Q2. How can healthcare professionals stay up-to-date with the latest advances in nutrigenetics?', 'Q3. What are the potential applications of nutrigenetics in healthcare and medicine?', 'Q4. How can nutrigenetics be used to improve health outcomes and disease prevention?', 'Q5. What are the challenges and limitations of implementing nutrigenetics in healthcare?']"
                ],
                [
                  "25",
                  "OBWCCE",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Understand the basics of nutrigenetics and genetic testing",
                  "['Q1. What is nutrigenetics and how does it relate to genetic testing?', 'Q2. What are the benefits and limitations of genetic testing for nutritional purposes?', 'Q3. How can genetic information be used to inform dietary recommendations?', 'Q4. What are the key genetic variants associated with nutritional responses and disease susceptibility?', 'Q5. How can I use genetic information to improve my health and well-being?']"
                ],
                [
                  "26",
                  "OBWCCE",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Explore the relationship between genetic variants and nutrient responses",
                  "['Q1. How do specific genetic variants affect my responses to different nutrients?', 'Q2. What are the implications of genetic variants on my dietary needs and responses?', 'Q3. How can I use genetic information to optimize my nutrition and health?', 'Q4. What are the potential benefits and limitations of using genetic information for personalized nutrition?', 'Q5. How can I stay informed about the latest research on nutrigenetics and genetic testing?']"
                ],
                [
                  "27",
                  "OBWCCE",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Investigate the role of nutrigenetics in disease prevention and management",
                  "['Q1. How can nutrigenetics be used to prevent or manage chronic diseases?', 'Q2. What are the key genetic variants associated with disease susceptibility and how can they be used to inform treatment decisions?', 'Q3. How do dietary patterns and nutritional elements interact with genetic variants to modulate disease risk or progression?', 'Q4. What are the potential therapeutic targets for modulating disease susceptibility and progression?', 'Q5. How can I use genetic information to reduce my risk of chronic diseases?']"
                ],
                [
                  "28",
                  "OBWCCE",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Examine the impact of environmental factors on nutrigenetic interactions",
                  "['Q1. How do environmental factors such as lifestyle and exposure to toxins influence the interactions between genetic variants and nutrients?', 'Q2. What are the implications of environmental factors on disease susceptibility and progression?', 'Q3. How can I take into account environmental factors when making dietary choices?', 'Q4. What are the potential strategies for mitigating the effects of environmental factors on nutrigenetic interactions?', 'Q5. How can I stay informed about the latest research on environmental factors and nutrigenetic interactions?']"
                ],
                [
                  "29",
                  "OBWCCE",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Explore the applications of nutrigenetics in personalized nutrition",
                  "['Q1. How can I use genetic information to tailor my dietary recommendations?', 'Q2. What are the potential benefits and limitations of using genetic information for personalized nutrition?', 'Q3. How can I integrate nutrigenetics into my overall health and wellness plan?', 'Q4. What are the implications of nutrigenetics on my health outcomes and disease risk?', 'Q5. How can I stay up-to-date with the latest advances in nutrigenetics and personalized nutrition?']"
                ],
                [
                  "30",
                  "EATBTS",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Investigating the relationship between genetic variants and nutrient response",
                  "['Q1. How do specific genetic variants affect the metabolism of nutrients and influence physiological processes?', 'Q2. What are the downstream effects of genetic variants on metabolic pathways and disease susceptibility?', 'Q3. How do genetic variants interact with dietary patterns to modulate gene expression and protein synthesis?', 'Q4. What is the role of molecular interactions in mediating the effects of genetic variants on nutrient response?', 'Q5. How do environmental factors influence the interactions between genetic variants and nutrients?']"
                ],
                [
                  "31",
                  "EATBTS",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Exploring the impact of nutrigenetics on disease prevention",
                  "['Q1. How do genetic variants contribute to disease susceptibility and progression in response to dietary factors?', 'Q2. What are the key metabolic pathways and physiological processes affected by genetic-nutrient interactions in disease development?', 'Q3. Can genetic variants be used to predict disease risk and tailor dietary interventions?', 'Q4. How do nutritional elements interact with genetic variants to modulate disease progression and treatment response?', 'Q5. What are the implications of nutrigenetics for personalized medicine and disease prevention?']"
                ],
                [
                  "32",
                  "EATBTS",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Analyzing the effects of nutrients on gene expression",
                  "['Q1. How do specific nutrients and dietary patterns affect gene expression and transcript production?', 'Q2. What are the mechanisms by which nutrients influence protein synthesis and modification?', 'Q3. How do nutrients interact with genetic variants to modulate gene expression and disease susceptibility?', 'Q4. What are the downstream effects of nutrient-gene interactions on metabolic pathways and physiological processes?', 'Q5. Can nutrient-gene interactions be used to develop targeted dietary interventions?']"
                ],
                [
                  "33",
                  "EATBTS",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Understanding the role of environmental factors in nutrigenetics",
                  "['Q1. How do environmental factors, such as lifestyle and exposure to toxins, influence genetic-nutrient interactions?', 'Q2. What are the effects of environmental factors on gene expression and protein synthesis in response to nutrients?', 'Q3. How do environmental factors modulate disease susceptibility and progression in response to genetic and nutritional factors?', 'Q4. Can environmental factors be used to tailor dietary interventions and disease prevention strategies?', 'Q5. What are the implications of environmental factors for personalized nutrigenetics and medicine?']"
                ],
                [
                  "34",
                  "EATBTS",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Developing personalized nutrition recommendations",
                  "['Q1. How can genetic information be used to tailor dietary recommendations and improve health outcomes?', 'Q2. What are the key genetic variants and nutritional elements to consider in personalized nutrition?', 'Q3. How do genetic variants interact with nutrients and dietary patterns to influence health outcomes?', 'Q4. Can personalized nutrition recommendations be used to prevent or manage disease?', 'Q5. What are the challenges and limitations of implementing personalized nutrition in clinical practice?']"
                ],
                [
                  "35",
                  "EATBTS",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Applying nutrigenetics in patient care",
                  "['Q1. How can healthcare professionals use genetic information to inform dietary recommendations and improve patient outcomes?', 'Q2. What are the key genetic variants and nutritional elements to consider in clinical practice?', 'Q3. How do genetic variants interact with nutrients and dietary patterns to influence disease susceptibility and progression?', 'Q4. Can nutrigenetic insights be used to tailor treatment strategies and improve patient outcomes?', 'Q5. What are the implications of nutrigenetics for healthcare policy and public health initiatives?']"
                ],
                [
                  "36",
                  "EATBTS",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Understanding the clinical applications of nutrigenetics",
                  "['Q1. What are the current and potential applications of nutrigenetics in clinical practice?', 'Q2. How can healthcare professionals stay up-to-date with the latest advances in nutrigenetics?', 'Q3. What are the challenges and limitations of implementing nutrigenetics in clinical practice?', 'Q4. How do genetic variants and nutritional elements interact to influence disease susceptibility and progression in clinical populations?', 'Q5. Can nutrigenetic insights be used to improve patient engagement and adherence to treatment?']"
                ],
                [
                  "37",
                  "EATBTS",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Investigating the role of nutrigenetics in disease management",
                  "['Q1. How do genetic variants and nutritional elements interact to influence disease progression and treatment response?', 'Q2. What are the key metabolic pathways and physiological processes affected by genetic-nutrient interactions in disease development?', 'Q3. Can nutrigenetic insights be used to tailor disease management strategies and improve patient outcomes?', 'Q4. How do environmental factors modulate the interactions between genetic variants and nutrients in disease management?', 'Q5. What are the implications of nutrigenetics for disease prevention and management in diverse populations?']"
                ],
                [
                  "38",
                  "EATBTS",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Exploring the intersection of nutrigenetics and pharmacogenetics",
                  "['Q1. How do genetic variants and nutritional elements interact to influence response to medication and treatment?', 'Q2. What are the implications of nutrigenetics for pharmacogenetics and personalized medicine?', 'Q3. Can nutrigenetic insights be used to tailor treatment strategies and improve patient outcomes?', 'Q4. How do genetic variants and nutritional elements interact to influence disease susceptibility and progression in response to treatment?', 'Q5. What are the challenges and limitations of integrating nutrigenetics and pharmacogenetics in clinical practice?']"
                ],
                [
                  "39",
                  "EATBTS",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Communicating nutrigenetic information to patients",
                  "['Q1. How can healthcare professionals effectively communicate nutrigenetic information to patients?', 'Q2. What are the key challenges and limitations of communicating complex genetic information to patients?', 'Q3. How can healthcare professionals use nutrigenetic insights to engage patients in disease prevention and management?', 'Q4. What are the implications of nutrigenetics for patient empowerment and health literacy?', 'Q5. Can nutrigenetic insights be used to improve patient-provider communication and health outcomes?']"
                ],
                [
                  "40",
                  "EATBTS",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Understanding personalized nutrition",
                  "['Q1. How can genetic information be used to tailor dietary recommendations and improve health outcomes?', 'Q2. What are the key genetic variants and nutritional elements to consider in personalized nutrition?', 'Q3. How do genetic variants interact with nutrients and dietary patterns to influence health outcomes?', 'Q4. Can personalized nutrition recommendations be used to prevent or manage disease?', 'Q5. What are the benefits and limitations of using genetic information for personalized nutrition?']"
                ],
                [
                  "41",
                  "EATBTS",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Exploring the role of genetic testing in personalized nutrition",
                  "['Q1. What are the current and potential applications of genetic testing in personalized nutrition?', 'Q2. How can genetic testing be used to inform dietary recommendations and improve health outcomes?', 'Q3. What are the challenges and limitations of using genetic testing for personalized nutrition?', 'Q4. How do genetic variants and nutritional elements interact to influence disease susceptibility and progression?', 'Q5. Can genetic testing be used to tailor dietary interventions and improve health outcomes?']"
                ],
                [
                  "42",
                  "EATBTS",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Investigating the impact of dietary patterns on health outcomes",
                  "['Q1. How do dietary patterns and nutritional elements interact with genetic variants to influence health outcomes?', 'Q2. What are the key metabolic pathways and physiological processes affected by dietary patterns and nutritional elements?', 'Q3. Can dietary patterns and nutritional elements be used to modulate disease susceptibility and progression?', 'Q4. How do environmental factors influence the interactions between dietary patterns and genetic variants?', 'Q5. What are the implications of dietary patterns and nutritional elements for personalized nutrition and health?']"
                ],
                [
                  "43",
                  "EATBTS",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Understanding the relationship between nutrigenetics and disease prevention",
                  "['Q1. How do genetic variants and nutritional elements interact to influence disease susceptibility and progression?', 'Q2. What are the key genetic variants and nutritional elements to consider in disease prevention?', 'Q3. Can nutrigenetic insights be used to tailor disease prevention strategies and improve health outcomes?', 'Q4. How do environmental factors modulate the interactions between genetic variants and nutrients in disease prevention?', 'Q5. What are the implications of nutrigenetics for disease prevention and public health initiatives?']"
                ],
                [
                  "44",
                  "EATBTS",
                  "A customer interested in personalized nutrition based on genetic information.",
                  "Evaluating the effectiveness of personalized nutrition",
                  "['Q1. How can the effectiveness of personalized nutrition recommendations be evaluated and measured?', 'Q2. What are the key challenges and limitations of evaluating personalized nutrition?', 'Q3. How do genetic variants and nutritional elements interact to influence health outcomes in response to personalized nutrition?', 'Q4. Can personalized nutrition recommendations be used to improve health outcomes and reduce healthcare costs?', 'Q5. What are the implications of personalized nutrition for healthcare policy and public health initiatives?']"
                ],
                [
                  "45",
                  "VITMIN",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Investigating the relationship between genetic variants and nutrient response.",
                  "['Q1. How do specific genetic variants affect the metabolism of nutrients and their subsequent impact on physiological processes?', 'Q2. What are the underlying molecular mechanisms by which genetic variants influence individual responses to dietary patterns?', 'Q3. How do genetic variants interact with environmental factors to modulate nutrient effects on health outcomes?', 'Q4. What is the role of epigenetic modifications in mediating the effects of genetic variants on nutrient response?', 'Q5. How can genetic variant-nutrient interactions be leveraged to develop personalized nutrition recommendations?']"
                ],
                [
                  "46",
                  "VITMIN",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Exploring the impact of nutrient-gene interactions on disease susceptibility.",
                  "['Q1. How do specific nutrients or dietary patterns affect gene expression and contribute to disease risk or progression?', 'Q2. What are the key metabolic pathways and molecular interactions involved in the development of diseases influenced by gene-nutrient interactions?', \"Q3. Can genetic variants be used to predict an individual's susceptibility to diet-related diseases?\", 'Q4. How do environmental factors influence the relationship between genetic variants, nutrient response, and disease susceptibility?', 'Q5. What are the implications of gene-nutrient interactions for the development of targeted disease prevention strategies?']"
                ],
                [
                  "47",
                  "VITMIN",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Analyzing the effects of gene-nutrient interactions on physiological processes.",
                  "['Q1. How do genetic variants affect the regulation of physiological processes, such as glucose metabolism or lipid metabolism, in response to dietary patterns?', 'Q2. What are the consequences of genetic variant-nutrient interactions on metabolic pathway flux and physiological outcomes?', 'Q3. Can nutrient-gene interactions be used to modulate physiological processes and improve health outcomes?', 'Q4. How do environmental factors, such as lifestyle or exposure to toxins, influence the effects of genetic variants on physiological processes?', 'Q5. What are the potential therapeutic applications of targeting gene-nutrient interactions to modulate physiological processes?']"
                ],
                [
                  "48",
                  "VITMIN",
                  "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.",
                  "Investigating the role of nutrigenomics in personalized nutrition.",
                  "[\"Q1. How can genetic information be used to tailor dietary recommendations to an individual's genetic profile?\", 'Q2. What are the challenges and limitations of implementing personalized nutrition based on genetic information?', 'Q3. How do genetic variants affect the response to specific nutrients or dietary patterns, and what are the implications for personalized nutrition?', \"Q4. Can nutrigenomics be used to predict an individual's response to different dietary interventions?\", 'Q5. What are the potential benefits and risks of using genetic information to guide dietary recommendations?']"
                ],
                [
                  "49",
                  "VITMIN",
                  "A healthcare professional looking to apply nutrigenetic insights in clinical practice.",
                  "Examining the relationship between gene-nutrient interactions and environmental factors.",
                  "['Q1. How do environmental factors, such as diet, lifestyle, or exposure to toxins, influence gene-nutrient interactions and health outcomes?', 'Q2. What are the mechanisms by which environmental factors modulate the effects of genetic variants on nutrient response and disease susceptibility?', 'Q3. Can environmental factors be used to mitigate or exacerbate the effects of genetic variants on health outcomes?', 'Q4. How do genetic variants interact with environmental factors to influence epigenetic modifications and gene expression?', 'Q5. What are the implications of gene-environment interactions for the development of targeted disease prevention strategies?']"
                ]
              ],
              "shape": {
                "columns": 4,
                "rows": 75
              }
            },
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Category</th>\n",
              "      <th>USER Description</th>\n",
              "      <th>TASK Description</th>\n",
              "      <th>Questions</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A researcher in nutrigenetics, interested in u...</td>\n",
              "      <td>Investigating the impact of genetic variants o...</td>\n",
              "      <td>[Q1. How do specific genetic variants in the g...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A researcher in nutrigenetics, interested in u...</td>\n",
              "      <td>Exploring the relationship between genetic var...</td>\n",
              "      <td>[Q1. What is the relationship between genetic ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A researcher in nutrigenetics, interested in u...</td>\n",
              "      <td>Analyzing the impact of genetic variants on di...</td>\n",
              "      <td>[Q1. How do genetic variants in the genes invo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A researcher in nutrigenetics, interested in u...</td>\n",
              "      <td>Investigating the role of epigenetics in nutri...</td>\n",
              "      <td>[Q1. How do epigenetic modifications affect th...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>FOODAL</td>\n",
              "      <td>A healthcare professional looking to apply nut...</td>\n",
              "      <td>Developing personalized nutrition recommendations</td>\n",
              "      <td>[Q1. How can genetic variants be used to tailo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>70</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A customer interested in personalized nutritio...</td>\n",
              "      <td>Understanding the basics of nutrigenomics</td>\n",
              "      <td>[Q1. What are the key concepts in nutrigenomic...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>71</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A customer interested in personalized nutritio...</td>\n",
              "      <td>Exploring the relationship between genetic var...</td>\n",
              "      <td>[Q1. How do genetic variants affect the metabo...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>72</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A customer interested in personalized nutritio...</td>\n",
              "      <td>Investigating the role of epigenetics in nutri...</td>\n",
              "      <td>[Q1. How do environmental factors, such as nut...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>73</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A customer interested in personalized nutritio...</td>\n",
              "      <td>Analyzing the impact of nutrient-gene interact...</td>\n",
              "      <td>[Q1. How do specific nutrients, such as folate...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>74</th>\n",
              "      <td>FOODIN</td>\n",
              "      <td>A customer interested in personalized nutritio...</td>\n",
              "      <td>Developing a personalized nutrition plan based...</td>\n",
              "      <td>[Q1. How can genetic testing be used to inform...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>75 rows × 4 columns</p>\n",
              "</div>"
            ],
            "text/plain": [
              "   Category                                   USER Description  \\\n",
              "0    FOODAL  A researcher in nutrigenetics, interested in u...   \n",
              "1    FOODAL  A researcher in nutrigenetics, interested in u...   \n",
              "2    FOODAL  A researcher in nutrigenetics, interested in u...   \n",
              "3    FOODAL  A researcher in nutrigenetics, interested in u...   \n",
              "4    FOODAL  A healthcare professional looking to apply nut...   \n",
              "..      ...                                                ...   \n",
              "70   FOODIN  A customer interested in personalized nutritio...   \n",
              "71   FOODIN  A customer interested in personalized nutritio...   \n",
              "72   FOODIN  A customer interested in personalized nutritio...   \n",
              "73   FOODIN  A customer interested in personalized nutritio...   \n",
              "74   FOODIN  A customer interested in personalized nutritio...   \n",
              "\n",
              "                                     TASK Description  \\\n",
              "0   Investigating the impact of genetic variants o...   \n",
              "1   Exploring the relationship between genetic var...   \n",
              "2   Analyzing the impact of genetic variants on di...   \n",
              "3   Investigating the role of epigenetics in nutri...   \n",
              "4   Developing personalized nutrition recommendations   \n",
              "..                                                ...   \n",
              "70          Understanding the basics of nutrigenomics   \n",
              "71  Exploring the relationship between genetic var...   \n",
              "72  Investigating the role of epigenetics in nutri...   \n",
              "73  Analyzing the impact of nutrient-gene interact...   \n",
              "74  Developing a personalized nutrition plan based...   \n",
              "\n",
              "                                            Questions  \n",
              "0   [Q1. How do specific genetic variants in the g...  \n",
              "1   [Q1. What is the relationship between genetic ...  \n",
              "2   [Q1. How do genetic variants in the genes invo...  \n",
              "3   [Q1. How do epigenetic modifications affect th...  \n",
              "4   [Q1. How can genetic variants be used to tailo...  \n",
              "..                                                ...  \n",
              "70  [Q1. What are the key concepts in nutrigenomic...  \n",
              "71  [Q1. How do genetic variants affect the metabo...  \n",
              "72  [Q1. How do environmental factors, such as nut...  \n",
              "73  [Q1. How do specific nutrients, such as folate...  \n",
              "74  [Q1. How can genetic testing be used to inform...  \n",
              "\n",
              "[75 rows x 4 columns]"
            ]
          },
          "execution_count": 84,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "\n",
        "df_combined.to_pickle(\"questions_dataframe_v3.pkl\")\n",
        "df_combined"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 85,
      "metadata": {
        "id": "gDz9WfO9Xo3Q"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses. \n",
            "------\n",
            "A healthcare professional looking to apply nutrigenetic insights in clinical practice. \n",
            "------\n",
            "A customer interested in personalized nutrition based on genetic information. \n",
            "------\n"
          ]
        }
      ],
      "source": [
        "for desc in df_combined['USER Description'].drop_duplicates():\n",
        "    print(desc,\"\\n------\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 88,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "['Q1. How do specific genetic variants in the genes involved in carbohydrate metabolism affect the response to dietary carbohydrates?', 'Q2. What are the effects of genetic variants on the expression of enzymes involved in fatty acid metabolism?', 'Q3. How do genetic variants in the genes encoding for nutrient transporters influence nutrient uptake and utilization?', 'Q4. Can genetic variants in the genes involved in epigenetic regulation modulate the response to dietary interventions?', 'Q5. How do genetic variants in the genes involved in inflammation affect the response to dietary patterns and disease risk?'] \n",
            "------\n",
            "['Q1. What is the relationship between genetic variants in the genes involved in taste perception and dietary preferences?', 'Q2. How do genetic variants in the genes involved in satiety and appetite regulation influence food choices?', 'Q3. Can genetic variants in the genes involved in nutrient sensing affect the response to dietary restriction?', 'Q4. How do genetic variants in the genes involved in gut microbiota modulation influence the response to prebiotics and probiotics?', 'Q5. What are the effects of genetic variants on the response to different types of dietary interventions (e.g., low-carb, low-fat)?'] \n",
            "------\n",
            "['Q1. How do genetic variants in the genes involved in insulin signaling affect the risk of developing type 2 diabetes?', 'Q2. What are the effects of genetic variants in the genes involved in lipid metabolism on cardiovascular disease risk?', 'Q3. Can genetic variants in the genes involved in inflammation modulate the risk of developing chronic diseases?', 'Q4. How do genetic variants in the genes involved in antioxidant defense affect the risk of developing age-related diseases?', 'Q5. What are the interactions between genetic variants and dietary patterns in modulating disease risk?'] \n",
            "------\n",
            "['Q1. How do epigenetic modifications affect the response to dietary interventions?', 'Q2. Can epigenetic marks be influenced by dietary patterns and nutrient exposures?', 'Q3. What are the effects of epigenetic modifications on gene expression and protein synthesis?', 'Q4. How do epigenetic mechanisms mediate the interactions between genes and environment?', 'Q5. Can epigenetic modifications be used as biomarkers for disease risk or response to dietary interventions?'] \n",
            "------\n",
            "['Q1. How can genetic variants be used to tailor dietary recommendations for optimal health outcomes?', 'Q2. What are the limitations and challenges of using genetic information for personalized nutrition?', 'Q3. Can genetic variants be used to predict responses to different types of dietary interventions?', 'Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of personalized nutrition recommendations?', 'Q5. What are the potential applications of nutrigenetics in public health policy and nutrition counseling?'] \n",
            "------\n",
            "['Q1. What are the clinical implications of genetic variants in the genes involved in nutrient metabolism?', 'Q2. How can healthcare professionals use genetic information to tailor dietary advice for patients with chronic diseases?', 'Q3. What are the potential benefits and limitations of using genetic testing for nutritional purposes in clinical practice?', 'Q4. Can genetic variants be used to predict responses to nutritional interventions in patients with specific health conditions?', 'Q5. How do healthcare professionals communicate genetic information and dietary recommendations to patients?'] \n",
            "------\n",
            "['Q1. How do genetic variants in the genes involved in inflammation affect the risk of developing chronic diseases?', 'Q2. What are the effects of dietary patterns on gene expression and disease risk?', 'Q3. Can genetic variants be used to identify individuals at high risk of developing specific diseases?', 'Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of disease prevention strategies?', 'Q5. What are the potential applications of nutrigenetics in disease prevention and management?'] \n",
            "------\n",
            "['Q1. How do genetic variants in the genes involved in nutrient metabolism affect the risk of nutritional deficiencies?', 'Q2. What are the effects of genetic variants on the absorption and utilization of essential nutrients?', 'Q3. Can genetic variants be used to predict responses to nutritional supplements?', 'Q4. How do genetic variants in the genes involved in taste perception and food preferences influence nutritional deficiencies?', 'Q5. What are the implications of genetic variants for nutritional deficiency diagnosis and treatment?'] \n",
            "------\n",
            "['Q1. How do epigenetic modifications affect the response to dietary interventions in patients with chronic diseases?', 'Q2. Can epigenetic marks be used as biomarkers for disease risk or response to dietary interventions?', 'Q3. What are the effects of epigenetic modifications on gene expression and protein synthesis in patients with specific health conditions?', 'Q4. How do epigenetic mechanisms mediate the interactions between genes and environment in clinical practice?', 'Q5. Can epigenetic modifications be used to monitor the effectiveness of nutritional interventions?'] \n",
            "------\n",
            "['Q1. What are the current evidence-based guidelines for using genetic testing for nutritional purposes in clinical practice?', 'Q2. How can healthcare professionals stay up-to-date with the rapidly evolving field of nutrigenetics?', 'Q3. What are the potential risks and benefits of using genetic testing for nutritional purposes in clinical practice?', 'Q4. How do healthcare professionals communicate genetic information and dietary recommendations to patients in a clear and actionable way?', 'Q5. What are the implications of genetic variants for clinical decision-making and patient care?'] \n",
            "------\n",
            "['Q1. What are the key concepts in nutrigenetics and how do they relate to personalized nutrition?', 'Q2. How do genetic variants affect my response to different nutrients and dietary patterns?', 'Q3. Can genetic testing help me understand my nutritional needs and develop a personalized diet plan?', 'Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of personalized nutrition recommendations?', 'Q5. What are the potential benefits and limitations of using genetic testing for nutritional purposes?'] \n",
            "------\n",
            "['Q1. How do genetic variants in the genes involved in carbohydrate metabolism affect my response to dietary carbohydrates?', 'Q2. What are the effects of genetic variants on my ability to lose weight or maintain weight loss?', 'Q3. Can genetic variants be used to predict my response to different types of dietary interventions (e.g., low-carb, low-fat)?', 'Q4. How do genetic variants in the genes involved in inflammation affect my risk of developing chronic diseases?', 'Q5. What are the implications of genetic variants for my nutritional needs and dietary recommendations?'] \n",
            "------\n",
            "['Q1. How do epigenetic modifications affect my response to dietary interventions?', 'Q2. Can epigenetic marks be influenced by my dietary patterns and nutrient exposures?', 'Q3. What are the effects of epigenetic modifications on my gene expression and protein synthesis?', 'Q4. How do epigenetic mechanisms mediate the interactions between my genes and environment?', 'Q5. Can epigenetic modifications be used to monitor the effectiveness of my nutritional interventions?'] \n",
            "------\n",
            "['Q1. How do genetic variants in the genes involved in insulin signaling affect my risk of developing type 2 diabetes?', 'Q2. What are the effects of genetic variants in the genes involved in lipid metabolism on my cardiovascular disease risk?', 'Q3. Can genetic variants be used to predict my risk of developing specific diseases?', 'Q4. How do environmental factors (e.g., lifestyle, socioeconomic status) influence the effectiveness of disease prevention strategies?', 'Q5. What are the potential applications of nutrigenetics in disease prevention and management?'] \n",
            "------\n",
            "['Q1. How can I use genetic information to tailor my dietary recommendations for optimal health outcomes?', 'Q2. What are the practical implications of genetic variants for my daily food choices?', 'Q3. Can genetic variants be used to predict my response to different types of dietary interventions?', 'Q4. How do I incorporate genetic information into my overall health and wellness plan?', 'Q5. What are the potential benefits and limitations of using genetic testing for nutritional purposes in everyday life?'] \n",
            "------\n",
            "['Q1. How do specific genetic variants affect the metabolism of nutrients and influence physiological processes?', 'Q2. What are the downstream effects of genetic variants on metabolic pathways and disease susceptibility?', 'Q3. How do genetic variants interact with dietary patterns to modulate gene expression and protein synthesis?', 'Q4. What is the role of molecular interactions in mediating the effects of genetic variants on nutrient responses?', 'Q5. How do environmental factors influence the interactions between genetic variants and nutrients?'] \n",
            "------\n",
            "['Q1. How do particular nutrients or dietary patterns affect the expression of specific genes and transcripts?', 'Q2. What are the effects of nutritional elements on protein synthesis and metabolic pathways?', 'Q3. How do nutritional elements interact with genetic variants to modulate disease risk or progression?', 'Q4. What are the implications of nutritional elements on physiological processes and health outcomes?', 'Q5. How do environmental factors alter the interactions between nutritional elements and genes?'] \n",
            "------\n",
            "['Q1. How do genetic variants interact with nutrients to modulate disease risk or progression?', 'Q2. What are the effects of genetic variants on metabolic pathways and disease susceptibility?', 'Q3. How do dietary patterns influence the interactions between genetic variants and disease susceptibility?', 'Q4. What is the role of molecular interactions in mediating the effects of genetic variants and nutrients on disease outcomes?', 'Q5. How do environmental factors influence the interactions between genetic variants, nutrients, and disease susceptibility?'] \n",
            "------\n",
            "['Q1. What are the key metabolic pathways involved in mediating the effects of genetic variants and nutrients on physiological processes?', 'Q2. How do genetic variants and nutrients interact to modulate metabolic pathways and disease susceptibility?', 'Q3. What are the implications of metabolic pathways on health outcomes and disease risk?', 'Q4. How do environmental factors influence the interactions between genetic variants, nutrients, and metabolic pathways?', 'Q5. What are the potential therapeutic targets for modulating metabolic pathways and disease susceptibility?'] \n",
            "------\n",
            "['Q1. How can genetic information be used to tailor dietary recommendations for individuals?', 'Q2. What are the implications of genetic variants on nutritional requirements and dietary responses?', 'Q3. How can nutrigenetics be used to prevent or manage chronic diseases?', 'Q4. What are the challenges and limitations of implementing personalized nutrition based on genetic information?', 'Q5. How can healthcare professionals integrate nutrigenetics into clinical practice?'] \n",
            "------\n",
            "['Q1. How can genetic testing be used to inform dietary recommendations for patients?', 'Q2. What are the potential benefits and limitations of incorporating nutrigenetics into clinical practice?', 'Q3. How can healthcare professionals stay up-to-date with the latest research in nutrigenetics?', 'Q4. What are the implications of genetic variants on disease risk and progression in patients?', 'Q5. How can healthcare professionals communicate complex genetic information to patients?'] \n",
            "------\n",
            "['Q1. How can nutrigenetics be used to prevent or manage chronic diseases such as diabetes and cardiovascular disease?', 'Q2. What are the key genetic variants associated with disease susceptibility and how can they be used to inform treatment decisions?', 'Q3. How do dietary patterns and nutritional elements interact with genetic variants to modulate disease risk or progression?', 'Q4. What are the potential therapeutic targets for modulating disease susceptibility and progression?', 'Q5. How can healthcare professionals integrate nutrigenetics into treatment plans?'] \n",
            "------\n",
            "['Q1. How do environmental factors such as lifestyle and exposure to toxins influence the interactions between genetic variants and nutrients?', 'Q2. What are the implications of environmental factors on disease susceptibility and progression?', 'Q3. How can healthcare professionals take into account environmental factors when making dietary recommendations?', 'Q4. What are the potential strategies for mitigating the effects of environmental factors on nutrigenetic interactions?', 'Q5. How can healthcare professionals stay informed about the latest research on environmental factors and nutrigenetic interactions?'] \n",
            "------\n",
            "['Q1. How can nutrigenetics be integrated into personalized medicine approaches?', 'Q2. What are the potential benefits and challenges of using nutrigenetics in personalized medicine?', 'Q3. How can healthcare professionals use genetic information to tailor treatment plans?', 'Q4. What are the implications of nutrigenetics on health outcomes and disease risk?', 'Q5. How can healthcare professionals communicate complex genetic information to patients in a personalized medicine context?'] \n",
            "------\n",
            "['Q1. What are the potential future directions for research in nutrigenetics?', 'Q2. How can healthcare professionals stay up-to-date with the latest advances in nutrigenetics?', 'Q3. What are the potential applications of nutrigenetics in healthcare and medicine?', 'Q4. How can nutrigenetics be used to improve health outcomes and disease prevention?', 'Q5. What are the challenges and limitations of implementing nutrigenetics in healthcare?'] \n",
            "------\n",
            "['Q1. What is nutrigenetics and how does it relate to genetic testing?', 'Q2. What are the benefits and limitations of genetic testing for nutritional purposes?', 'Q3. How can genetic information be used to inform dietary recommendations?', 'Q4. What are the key genetic variants associated with nutritional responses and disease susceptibility?', 'Q5. How can I use genetic information to improve my health and well-being?'] \n",
            "------\n",
            "['Q1. How do specific genetic variants affect my responses to different nutrients?', 'Q2. What are the implications of genetic variants on my dietary needs and responses?', 'Q3. How can I use genetic information to optimize my nutrition and health?', 'Q4. What are the potential benefits and limitations of using genetic information for personalized nutrition?', 'Q5. How can I stay informed about the latest research on nutrigenetics and genetic testing?'] \n",
            "------\n",
            "['Q1. How can nutrigenetics be used to prevent or manage chronic diseases?', 'Q2. What are the key genetic variants associated with disease susceptibility and how can they be used to inform treatment decisions?', 'Q3. How do dietary patterns and nutritional elements interact with genetic variants to modulate disease risk or progression?', 'Q4. What are the potential therapeutic targets for modulating disease susceptibility and progression?', 'Q5. How can I use genetic information to reduce my risk of chronic diseases?'] \n",
            "------\n",
            "['Q1. How do environmental factors such as lifestyle and exposure to toxins influence the interactions between genetic variants and nutrients?', 'Q2. What are the implications of environmental factors on disease susceptibility and progression?', 'Q3. How can I take into account environmental factors when making dietary choices?', 'Q4. What are the potential strategies for mitigating the effects of environmental factors on nutrigenetic interactions?', 'Q5. How can I stay informed about the latest research on environmental factors and nutrigenetic interactions?'] \n",
            "------\n",
            "['Q1. How can I use genetic information to tailor my dietary recommendations?', 'Q2. What are the potential benefits and limitations of using genetic information for personalized nutrition?', 'Q3. How can I integrate nutrigenetics into my overall health and wellness plan?', 'Q4. What are the implications of nutrigenetics on my health outcomes and disease risk?', 'Q5. How can I stay up-to-date with the latest advances in nutrigenetics and personalized nutrition?'] \n",
            "------\n",
            "['Q1. How do specific genetic variants affect the metabolism of nutrients and influence physiological processes?', 'Q2. What are the downstream effects of genetic variants on metabolic pathways and disease susceptibility?', 'Q3. How do genetic variants interact with dietary patterns to modulate gene expression and protein synthesis?', 'Q4. What is the role of molecular interactions in mediating the effects of genetic variants on nutrient response?', 'Q5. How do environmental factors influence the interactions between genetic variants and nutrients?'] \n",
            "------\n",
            "['Q1. How do genetic variants contribute to disease susceptibility and progression in response to dietary factors?', 'Q2. What are the key metabolic pathways and physiological processes affected by genetic-nutrient interactions in disease development?', 'Q3. Can genetic variants be used to predict disease risk and tailor dietary interventions?', 'Q4. How do nutritional elements interact with genetic variants to modulate disease progression and treatment response?', 'Q5. What are the implications of nutrigenetics for personalized medicine and disease prevention?'] \n",
            "------\n",
            "['Q1. How do specific nutrients and dietary patterns affect gene expression and transcript production?', 'Q2. What are the mechanisms by which nutrients influence protein synthesis and modification?', 'Q3. How do nutrients interact with genetic variants to modulate gene expression and disease susceptibility?', 'Q4. What are the downstream effects of nutrient-gene interactions on metabolic pathways and physiological processes?', 'Q5. Can nutrient-gene interactions be used to develop targeted dietary interventions?'] \n",
            "------\n",
            "['Q1. How do environmental factors, such as lifestyle and exposure to toxins, influence genetic-nutrient interactions?', 'Q2. What are the effects of environmental factors on gene expression and protein synthesis in response to nutrients?', 'Q3. How do environmental factors modulate disease susceptibility and progression in response to genetic and nutritional factors?', 'Q4. Can environmental factors be used to tailor dietary interventions and disease prevention strategies?', 'Q5. What are the implications of environmental factors for personalized nutrigenetics and medicine?'] \n",
            "------\n",
            "['Q1. How can genetic information be used to tailor dietary recommendations and improve health outcomes?', 'Q2. What are the key genetic variants and nutritional elements to consider in personalized nutrition?', 'Q3. How do genetic variants interact with nutrients and dietary patterns to influence health outcomes?', 'Q4. Can personalized nutrition recommendations be used to prevent or manage disease?', 'Q5. What are the challenges and limitations of implementing personalized nutrition in clinical practice?'] \n",
            "------\n",
            "['Q1. How can healthcare professionals use genetic information to inform dietary recommendations and improve patient outcomes?', 'Q2. What are the key genetic variants and nutritional elements to consider in clinical practice?', 'Q3. How do genetic variants interact with nutrients and dietary patterns to influence disease susceptibility and progression?', 'Q4. Can nutrigenetic insights be used to tailor treatment strategies and improve patient outcomes?', 'Q5. What are the implications of nutrigenetics for healthcare policy and public health initiatives?'] \n",
            "------\n",
            "['Q1. What are the current and potential applications of nutrigenetics in clinical practice?', 'Q2. How can healthcare professionals stay up-to-date with the latest advances in nutrigenetics?', 'Q3. What are the challenges and limitations of implementing nutrigenetics in clinical practice?', 'Q4. How do genetic variants and nutritional elements interact to influence disease susceptibility and progression in clinical populations?', 'Q5. Can nutrigenetic insights be used to improve patient engagement and adherence to treatment?'] \n",
            "------\n",
            "['Q1. How do genetic variants and nutritional elements interact to influence disease progression and treatment response?', 'Q2. What are the key metabolic pathways and physiological processes affected by genetic-nutrient interactions in disease development?', 'Q3. Can nutrigenetic insights be used to tailor disease management strategies and improve patient outcomes?', 'Q4. How do environmental factors modulate the interactions between genetic variants and nutrients in disease management?', 'Q5. What are the implications of nutrigenetics for disease prevention and management in diverse populations?'] \n",
            "------\n",
            "['Q1. How do genetic variants and nutritional elements interact to influence response to medication and treatment?', 'Q2. What are the implications of nutrigenetics for pharmacogenetics and personalized medicine?', 'Q3. Can nutrigenetic insights be used to tailor treatment strategies and improve patient outcomes?', 'Q4. How do genetic variants and nutritional elements interact to influence disease susceptibility and progression in response to treatment?', 'Q5. What are the challenges and limitations of integrating nutrigenetics and pharmacogenetics in clinical practice?'] \n",
            "------\n",
            "['Q1. How can healthcare professionals effectively communicate nutrigenetic information to patients?', 'Q2. What are the key challenges and limitations of communicating complex genetic information to patients?', 'Q3. How can healthcare professionals use nutrigenetic insights to engage patients in disease prevention and management?', 'Q4. What are the implications of nutrigenetics for patient empowerment and health literacy?', 'Q5. Can nutrigenetic insights be used to improve patient-provider communication and health outcomes?'] \n",
            "------\n",
            "['Q1. How can genetic information be used to tailor dietary recommendations and improve health outcomes?', 'Q2. What are the key genetic variants and nutritional elements to consider in personalized nutrition?', 'Q3. How do genetic variants interact with nutrients and dietary patterns to influence health outcomes?', 'Q4. Can personalized nutrition recommendations be used to prevent or manage disease?', 'Q5. What are the benefits and limitations of using genetic information for personalized nutrition?'] \n",
            "------\n",
            "['Q1. What are the current and potential applications of genetic testing in personalized nutrition?', 'Q2. How can genetic testing be used to inform dietary recommendations and improve health outcomes?', 'Q3. What are the challenges and limitations of using genetic testing for personalized nutrition?', 'Q4. How do genetic variants and nutritional elements interact to influence disease susceptibility and progression?', 'Q5. Can genetic testing be used to tailor dietary interventions and improve health outcomes?'] \n",
            "------\n",
            "['Q1. How do dietary patterns and nutritional elements interact with genetic variants to influence health outcomes?', 'Q2. What are the key metabolic pathways and physiological processes affected by dietary patterns and nutritional elements?', 'Q3. Can dietary patterns and nutritional elements be used to modulate disease susceptibility and progression?', 'Q4. How do environmental factors influence the interactions between dietary patterns and genetic variants?', 'Q5. What are the implications of dietary patterns and nutritional elements for personalized nutrition and health?'] \n",
            "------\n",
            "['Q1. How do genetic variants and nutritional elements interact to influence disease susceptibility and progression?', 'Q2. What are the key genetic variants and nutritional elements to consider in disease prevention?', 'Q3. Can nutrigenetic insights be used to tailor disease prevention strategies and improve health outcomes?', 'Q4. How do environmental factors modulate the interactions between genetic variants and nutrients in disease prevention?', 'Q5. What are the implications of nutrigenetics for disease prevention and public health initiatives?'] \n",
            "------\n",
            "['Q1. How can the effectiveness of personalized nutrition recommendations be evaluated and measured?', 'Q2. What are the key challenges and limitations of evaluating personalized nutrition?', 'Q3. How do genetic variants and nutritional elements interact to influence health outcomes in response to personalized nutrition?', 'Q4. Can personalized nutrition recommendations be used to improve health outcomes and reduce healthcare costs?', 'Q5. What are the implications of personalized nutrition for healthcare policy and public health initiatives?'] \n",
            "------\n",
            "['Q1. How do specific genetic variants affect the metabolism of nutrients and their subsequent impact on physiological processes?', 'Q2. What are the underlying molecular mechanisms by which genetic variants influence individual responses to dietary patterns?', 'Q3. How do genetic variants interact with environmental factors to modulate nutrient effects on health outcomes?', 'Q4. What is the role of epigenetic modifications in mediating the effects of genetic variants on nutrient response?', 'Q5. How can genetic variant-nutrient interactions be leveraged to develop personalized nutrition recommendations?'] \n",
            "------\n",
            "['Q1. How do specific nutrients or dietary patterns affect gene expression and contribute to disease risk or progression?', 'Q2. What are the key metabolic pathways and molecular interactions involved in the development of diseases influenced by gene-nutrient interactions?', \"Q3. Can genetic variants be used to predict an individual's susceptibility to diet-related diseases?\", 'Q4. How do environmental factors influence the relationship between genetic variants, nutrient response, and disease susceptibility?', 'Q5. What are the implications of gene-nutrient interactions for the development of targeted disease prevention strategies?'] \n",
            "------\n",
            "['Q1. How do genetic variants affect the regulation of physiological processes, such as glucose metabolism or lipid metabolism, in response to dietary patterns?', 'Q2. What are the consequences of genetic variant-nutrient interactions on metabolic pathway flux and physiological outcomes?', 'Q3. Can nutrient-gene interactions be used to modulate physiological processes and improve health outcomes?', 'Q4. How do environmental factors, such as lifestyle or exposure to toxins, influence the effects of genetic variants on physiological processes?', 'Q5. What are the potential therapeutic applications of targeting gene-nutrient interactions to modulate physiological processes?'] \n",
            "------\n",
            "[\"Q1. How can genetic information be used to tailor dietary recommendations to an individual's genetic profile?\", 'Q2. What are the challenges and limitations of implementing personalized nutrition based on genetic information?', 'Q3. How do genetic variants affect the response to specific nutrients or dietary patterns, and what are the implications for personalized nutrition?', \"Q4. Can nutrigenomics be used to predict an individual's response to different dietary interventions?\", 'Q5. What are the potential benefits and risks of using genetic information to guide dietary recommendations?'] \n",
            "------\n",
            "['Q1. How do environmental factors, such as diet, lifestyle, or exposure to toxins, influence gene-nutrient interactions and health outcomes?', 'Q2. What are the mechanisms by which environmental factors modulate the effects of genetic variants on nutrient response and disease susceptibility?', 'Q3. Can environmental factors be used to mitigate or exacerbate the effects of genetic variants on health outcomes?', 'Q4. How do genetic variants interact with environmental factors to influence epigenetic modifications and gene expression?', 'Q5. What are the implications of gene-environment interactions for the development of targeted disease prevention strategies?'] \n",
            "------\n",
            "['Q1. How can healthcare professionals use genetic information to inform dietary recommendations for patients with specific disease conditions?', 'Q2. What are the potential benefits and limitations of incorporating nutrigenetic testing into clinical practice?', 'Q3. How do genetic variants affect the response to different nutritional interventions, and what are the implications for patient care?', 'Q4. Can nutrigenomics be used to predict patient responses to different dietary patterns or nutritional supplements?', 'Q5. What are the challenges and opportunities for integrating nutrigenetic insights into clinical practice?'] \n",
            "------\n",
            "['Q1. How can nutrigenetic insights be used to prevent or manage chronic diseases, such as cardiovascular disease or diabetes?', 'Q2. What are the key genetic variants and nutrient interactions involved in the development of diet-related diseases?', 'Q3. Can genetic testing be used to identify individuals at high risk for diet-related diseases, and what are the implications for disease prevention?', 'Q4. How do environmental factors influence the relationship between genetic variants and disease susceptibility?', 'Q5. What are the potential therapeutic applications of targeting gene-nutrient interactions to prevent or manage disease?'] \n",
            "------\n",
            "['Q1. What are the limitations and challenges of using genetic testing to inform dietary recommendations or disease risk assessment?', 'Q2. How do genetic variants interact with environmental factors to influence health outcomes, and what are the implications for disease prevention?', \"Q3. Can genetic testing be used to predict an individual's response to specific nutritional interventions?\", 'Q4. What are the potential risks and benefits of using genetic information to guide dietary recommendations?', 'Q5. How can healthcare professionals stay up-to-date with the rapidly evolving field of nutrigenomics?'] \n",
            "------\n",
            "['Q1. How can healthcare professionals communicate complex nutrigenetic information to patients in a clear and actionable way?', 'Q2. What are the key concepts and principles that healthcare professionals should understand when applying nutrigenetic insights in clinical practice?', 'Q3. Can nutrigenomics be used to empower patients to take a more active role in managing their health through dietary choices?', \"Q4. How do genetic variants affect an individual's ability to respond to dietary advice or nutritional interventions?\", 'Q5. What are the potential benefits and challenges of incorporating nutrigenetic testing into patient education and counseling?'] \n",
            "------\n",
            "['Q1. What are the potential future directions for the application of nutrigenomics in clinical practice?', 'Q2. How will advances in genetic testing and nutrigenomics impact the delivery of healthcare services?', 'Q3. Can nutrigenomics be used to develop more effective and personalized treatment strategies for diet-related diseases?', 'Q4. What are the implications of nutrigenomics for public health policy and disease prevention initiatives?', 'Q5. How will the integration of nutrigenomics into clinical practice influence the role of healthcare professionals?'] \n",
            "------\n",
            "['Q1. What are the key concepts and principles of nutrigenomics, and how do they relate to personalized nutrition?', \"Q2. How do genetic variants affect an individual's response to different nutrients and dietary patterns?\", 'Q3. Can genetic testing be used to identify optimal dietary patterns or nutritional supplements for an individual?', 'Q4. What are the potential benefits and limitations of using genetic information to guide dietary choices?', 'Q5. How can I use nutrigenomic insights to take control of my health and well-being?'] \n",
            "------\n",
            "['Q1. What types of genetic testing are available for nutrigenomics, and what do they entail?', 'Q2. How can I use genetic testing to inform my dietary choices and improve my health outcomes?', 'Q3. Can genetic testing be used to predict my susceptibility to diet-related diseases?', 'Q4. What are the potential risks and benefits of using genetic information to guide dietary recommendations?', 'Q5. How can I ensure that my genetic information is used responsibly and confidentially?'] \n",
            "------\n",
            "['Q1. How do genetic variants affect my response to different foods and nutrients, and what are the implications for my dietary choices?', 'Q2. Can nutrigenomics be used to identify optimal dietary patterns or nutritional supplements for my genetic profile?', 'Q3. How do environmental factors influence the relationship between genetic variants and nutrient response?', 'Q4. What are the potential benefits and limitations of using nutrigenomic insights to guide my dietary choices?', 'Q5. How can I stay up-to-date with the rapidly evolving field of nutrigenomics and its implications for my health?'] \n",
            "------\n",
            "['Q1. How do genetic variants interact with nutrients and dietary patterns to influence disease risk or progression?', 'Q2. Can genetic testing be used to predict my susceptibility to diet-related diseases?', 'Q3. What are the key metabolic pathways and molecular interactions involved in the development of diseases influenced by gene-nutrient interactions?', 'Q4. How do environmental factors influence the relationship between genetic variants, nutrient response, and disease susceptibility?', 'Q5. What are the implications of gene-nutrient interactions for my health and well-being?'] \n",
            "------\n",
            "['Q1. What are the potential future directions for the application of nutrigenomics in personalized nutrition?', 'Q2. How will advances in genetic testing and nutrigenomics impact the delivery of nutrition services?', 'Q3. Can nutrigenomics be used to develop more effective and personalized nutrition recommendations?', 'Q4. What are the implications of nutrigenomics for public health policy and disease prevention initiatives?', 'Q5. How will the integration of nutrigenomics into personalized nutrition influence the role of healthcare professionals and registered dietitians?'] \n",
            "------\n",
            "['Q1. How do specific genetic variants affect the metabolism of nutrients, such as carbohydrates, proteins, and fats?', 'Q2. What are the underlying molecular mechanisms by which genetic variants influence nutrient metabolism?', 'Q3. How do genetic variants interact with dietary patterns to modulate metabolic pathways and physiological processes?', 'Q4. What are the implications of genetic variants on the development of metabolic disorders, such as obesity and diabetes?', 'Q5. How can functional studies be used to elucidate the effects of genetic variants on protein activity and cellular processes in the context of nutrient metabolism?'] \n",
            "------\n",
            "['Q1. How do genetic variants influence individual responses to different dietary patterns, such as low-carb or low-fat diets?', 'Q2. What are the key genetic variants associated with differential responses to dietary interventions?', 'Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk?', 'Q4. What are the potential applications of genetic testing in personalized nutrition and disease prevention?', 'Q5. How can systems biology approaches be used to integrate genetic, nutritional, and environmental factors and predict disease outcomes?'] \n",
            "------\n",
            "['Q1. How do specific nutrients, such as folate and vitamin D, interact with genetic variants to modulate disease risk?', 'Q2. What are the underlying biological mechanisms by which nutrient-gene interactions influence disease susceptibility?', 'Q3. How do genetic variants affect the expression and activity of enzymes involved in nutrient metabolism?', 'Q4. What are the implications of nutrient-gene interactions for the prevention and treatment of chronic diseases, such as cardiovascular disease and cancer?', 'Q5. How can nutrigenomic studies inform the development of targeted dietary interventions for disease prevention and treatment?'] \n",
            "------\n",
            "['Q1. How do environmental factors, such as nutrition and lifestyle, influence epigenetic marks and gene expression?', 'Q2. What are the key epigenetic mechanisms by which nutrients and dietary patterns modulate gene expression and disease susceptibility?', 'Q3. How do epigenetic changes contribute to the development of chronic diseases, such as obesity and metabolic disorders?', 'Q4. What are the potential applications of epigenetic testing in personalized nutrition and disease prevention?', 'Q5. How can epigenomic studies inform the development of targeted dietary interventions for disease prevention and treatment?'] \n",
            "------\n",
            "['Q1. How can genetic testing be used to inform personalized nutrition recommendations?', 'Q2. What are the key genetic variants associated with differential responses to dietary interventions?', 'Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk?', 'Q4. What are the potential benefits and limitations of personalized nutrition based on genetic information?', 'Q5. How can systems biology approaches be used to integrate genetic, nutritional, and environmental factors and predict disease outcomes?'] \n",
            "------\n",
            "['Q1. How can healthcare professionals use genetic testing to inform dietary recommendations for patients?', 'Q2. What are the key genetic variants associated with differential responses to dietary interventions in clinical populations?', 'Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk in clinical populations?', 'Q4. What are the potential applications of nutrigenetic testing in clinical practice, such as disease prevention and treatment?', 'Q5. How can healthcare professionals stay up-to-date with the latest advances in nutrigenomics and apply them in clinical practice?'] \n",
            "------\n",
            "['Q1. How can nutrigenetic insights be used to prevent or treat chronic diseases, such as cardiovascular disease and cancer?', 'Q2. What are the key biological mechanisms by which nutrient-gene interactions influence disease susceptibility and progression?', 'Q3. How do genetic variants affect the expression and activity of enzymes involved in nutrient metabolism in clinical populations?', 'Q4. What are the implications of nutrient-gene interactions for the prevention and treatment of chronic diseases in clinical populations?', 'Q5. How can healthcare professionals use nutrigenetic testing to identify patients at high risk of disease and develop targeted interventions?'] \n",
            "------\n",
            "['Q1. How do specific nutrients, such as folate and vitamin D, interact with genetic variants to modulate disease risk in clinical populations?', 'Q2. What are the underlying biological mechanisms by which nutrient-gene interactions influence disease susceptibility and progression?', 'Q3. How do genetic variants affect the expression and activity of enzymes involved in nutrient metabolism in clinical populations?', 'Q4. What are the implications of nutrient-gene interactions for the prevention and treatment of chronic diseases in clinical populations?', 'Q5. How can healthcare professionals use nutrigenetic testing to identify patients at high risk of disease and develop targeted interventions?'] \n",
            "------\n",
            "['Q1. How do genetic variants influence individual responses to different dietary patterns, such as low-carb or low-fat diets, in clinical populations?', 'Q2. What are the key genetic variants associated with differential responses to dietary interventions in clinical populations?', 'Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk in clinical populations?', 'Q4. What are the potential applications of genetic testing in personalized nutrition and disease prevention in clinical populations?', 'Q5. How can healthcare professionals stay up-to-date with the latest advances in nutrigenomics and apply them in clinical practice?'] \n",
            "------\n",
            "['Q1. How can healthcare professionals use genetic testing to inform personalized nutrition recommendations for patients?', 'Q2. What are the key genetic variants associated with differential responses to dietary interventions in clinical populations?', 'Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk in clinical populations?', 'Q4. What are the potential benefits and limitations of personalized nutrition based on genetic information in clinical populations?', 'Q5. How can healthcare professionals use nutrigenetic testing to identify patients at high risk of disease and develop targeted interventions?'] \n",
            "------\n",
            "['Q1. What are the key concepts in nutrigenomics and how do they relate to personalized nutrition?', 'Q2. How do genetic variants influence individual responses to different dietary patterns?', 'Q3. What are the potential benefits and limitations of personalized nutrition based on genetic information?', 'Q4. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk?', 'Q5. What are the current challenges and future directions in the field of nutrigenomics?'] \n",
            "------\n",
            "['Q1. How do genetic variants affect the metabolism of nutrients, such as carbohydrates, proteins, and fats?', 'Q2. What are the key genetic variants associated with differential responses to dietary interventions?', 'Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk?', 'Q4. What are the implications of genetic variants on the development of metabolic disorders, such as obesity and diabetes?', 'Q5. How can functional studies be used to elucidate the effects of genetic variants on protein activity and cellular processes in the context of nutrient metabolism?'] \n",
            "------\n",
            "['Q1. How do specific nutrients, such as folate and vitamin D, interact with genetic variants to modulate disease risk?', 'Q2. What are the underlying biological mechanisms by which nutrient-gene interactions influence disease susceptibility and progression?', 'Q3. How do genetic variants affect the expression and activity of enzymes involved in nutrient metabolism?', 'Q4. What are the implications of nutrient-gene interactions for the prevention and treatment of chronic diseases?', 'Q5. How can nutrigenomic studies inform the development of targeted dietary interventions for disease prevention and treatment?'] \n",
            "------\n",
            "['Q1. How can genetic testing be used to inform personalized nutrition recommendations?', 'Q2. What are the key genetic variants associated with differential responses to dietary interventions?', 'Q3. How do environmental factors, such as lifestyle and nutritional exposures, interact with genetic variants to modulate disease risk?', 'Q4. What are the potential benefits and limitations of personalized nutrition based on genetic information?', 'Q5. How can individuals use nutrigenetic testing to take control of their health and make informed dietary choices?'] \n",
            "------\n"
          ]
        }
      ],
      "source": [
        "for desc in df_combined['Questions'].drop_duplicates():\n",
        "    print(desc,\"\\n------\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 86,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Investigating the impact of genetic variants on nutrient metabolism \n",
            "------\n",
            "Exploring the relationship between genetic variants and dietary patterns \n",
            "------\n",
            "Analyzing the impact of genetic variants on disease risk \n",
            "------\n",
            "Investigating the role of epigenetics in nutrigenetics \n",
            "------\n",
            "Developing personalized nutrition recommendations \n",
            "------\n",
            "Integrating nutrigenetics into clinical practice \n",
            "------\n",
            "Understanding the role of nutrigenetics in disease prevention \n",
            "------\n",
            "Exploring the relationship between genetic variants and nutritional deficiencies \n",
            "------\n",
            "Investigating the impact of epigenetics on clinical practice \n",
            "------\n",
            "Developing evidence-based guidelines for nutrigenetic testing \n",
            "------\n",
            "Understanding the basics of nutrigenetics \n",
            "------\n",
            "Exploring the role of genetic variants in dietary responses \n",
            "------\n",
            "Investigating the impact of epigenetics on nutrition \n",
            "------\n",
            "Understanding the relationship between genetic variants and disease risk \n",
            "------\n",
            "Applying nutrigenetic insights to daily life \n",
            "------\n",
            "Investigate the relationship between genetic variants and nutrient responses \n",
            "------\n",
            "Examine the impact of nutritional elements on gene expression \n",
            "------\n",
            "Analyze the relationship between genetic variants, nutrients, and disease susceptibility \n",
            "------\n",
            "Investigate the role of metabolic pathways in nutrigenetics \n",
            "------\n",
            "Explore the applications of nutrigenetics in personalized nutrition \n",
            "------\n",
            "Apply nutrigenetic knowledge to patient care \n",
            "------\n",
            "Evaluate the role of nutrigenetics in disease prevention and management \n",
            "------\n",
            "Investigate the impact of environmental factors on nutrigenetic interactions \n",
            "------\n",
            "Examine the role of nutrigenetics in personalized medicine \n",
            "------\n",
            "Explore the future of nutrigenetics in healthcare \n",
            "------\n",
            "Understand the basics of nutrigenetics and genetic testing \n",
            "------\n",
            "Explore the relationship between genetic variants and nutrient responses \n",
            "------\n",
            "Investigate the role of nutrigenetics in disease prevention and management \n",
            "------\n",
            "Examine the impact of environmental factors on nutrigenetic interactions \n",
            "------\n",
            "Investigating the relationship between genetic variants and nutrient response \n",
            "------\n",
            "Exploring the impact of nutrigenetics on disease prevention \n",
            "------\n",
            "Analyzing the effects of nutrients on gene expression \n",
            "------\n",
            "Understanding the role of environmental factors in nutrigenetics \n",
            "------\n",
            "Applying nutrigenetics in patient care \n",
            "------\n",
            "Understanding the clinical applications of nutrigenetics \n",
            "------\n",
            "Investigating the role of nutrigenetics in disease management \n",
            "------\n",
            "Exploring the intersection of nutrigenetics and pharmacogenetics \n",
            "------\n",
            "Communicating nutrigenetic information to patients \n",
            "------\n",
            "Understanding personalized nutrition \n",
            "------\n",
            "Exploring the role of genetic testing in personalized nutrition \n",
            "------\n",
            "Investigating the impact of dietary patterns on health outcomes \n",
            "------\n",
            "Understanding the relationship between nutrigenetics and disease prevention \n",
            "------\n",
            "Evaluating the effectiveness of personalized nutrition \n",
            "------\n",
            "Investigating the relationship between genetic variants and nutrient response. \n",
            "------\n",
            "Exploring the impact of nutrient-gene interactions on disease susceptibility. \n",
            "------\n",
            "Analyzing the effects of gene-nutrient interactions on physiological processes. \n",
            "------\n",
            "Investigating the role of nutrigenomics in personalized nutrition. \n",
            "------\n",
            "Examining the relationship between gene-nutrient interactions and environmental factors. \n",
            "------\n",
            "Applying nutrigenetic knowledge to patient care. \n",
            "------\n",
            "Exploring the role of nutrigenomics in disease prevention. \n",
            "------\n",
            "Understanding the limitations and challenges of nutrigenetic testing. \n",
            "------\n",
            "Applying nutrigenetic insights to patient education. \n",
            "------\n",
            "Exploring the future of nutrigenomics in clinical practice. \n",
            "------\n",
            "Understanding the basics of nutrigenomics. \n",
            "------\n",
            "Exploring the role of genetic testing in personalized nutrition. \n",
            "------\n",
            "Understanding the implications of nutrigenomics for dietary choices. \n",
            "------\n",
            "Investigating the relationship between gene-nutrient interactions and disease risk. \n",
            "------\n",
            "Exploring the future of personalized nutrition. \n",
            "------\n",
            "Exploring the relationship between genetic variants and dietary responses \n",
            "------\n",
            "Analyzing the impact of nutrient-gene interactions on disease susceptibility \n",
            "------\n",
            "Investigating the role of epigenetics in nutrigenomics \n",
            "------\n",
            "Developing personalized nutrition recommendations based on genetic information \n",
            "------\n",
            "Applying nutrigenetic insights in clinical practice \n",
            "------\n",
            "Investigating the role of nutrigenetics in disease prevention and treatment \n",
            "------\n",
            "Exploring the relationship between genetic variants and dietary responses in clinical populations \n",
            "------\n",
            "Developing targeted dietary interventions based on genetic information \n",
            "------\n",
            "Understanding the basics of nutrigenomics \n",
            "------\n",
            "Developing a personalized nutrition plan based on genetic information \n",
            "------\n"
          ]
        }
      ],
      "source": [
        "for desc in df_combined['TASK Description'].drop_duplicates():\n",
        "    print(desc,\"\\n------\")"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
        "# Use community report summary"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 7,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Loading graph from file: /root/projects/nano-graphrag/biomedical/ablation_study/cache_gemma2_all-mpnet-base-v2/kv_store_community_reports.json\n",
            "\n",
            "**Corpus Description**\n",
            "\n",
            "This community explores the relationships between genetic markers, immune system responses, and environmental triggers in Celiac Disease (CD). Key entities include genes like PTPN2 and HLA-DRB1*07:01, which are associated with increased CD risk. Immune cells like T-cells play a crucial role in the inflammatory response observed in CD.  The community highlights the complex interplay between genetic predisposition and environmental factors in the development of CD.\n",
            "\n",
            "This community explores the relationship between genetic variations, environmental factors, and asthma risk. Key entities include genes (like IL-1A -889C>T and TNF-A 238G>A),  SNPs (RS1898671, RS2227306, RS4073), and lifestyle factors like smoking status. The community reveals associations between certain genetic variations and increased susceptibility to asthma, particularly atopic asthma.\n",
            "\n",
            "This community centers on PTPN2 gene variants (specifically rs478582 and rs2476601) and their association with inflammatory autoimmune diseases, particularly Crohn's disease (CD).  Key relationships involve the influence of these variants on IFN-γ expression, CD average relative gene expression, and susceptibility to MAP infection. \n",
            "\n",
            "This community focuses on the development and application of CAR T cell therapy targeting the TAG72 protein, particularly in the context of ovarian cancer. Key entities include CAR T cells genetically engineered to express anti-TAG72 receptors (TAG72-BBζ CAR T cells), various ovarian cancer cell lines (e.g., OVCAR3, OV90), and immune markers like CD69 and IFNγ used to track CAR T cell activation and function.\n"
          ]
        }
      ],
      "source": [
        "\n",
        "import json\n",
        "cache_dict ={1: 'cache_llama3.1_8b_all-mpnet-base-v2',\n",
        "             2: 'cache_gemma2_all-mpnet-base-v2',\n",
        "             3: 'cache_qwen2.5_14b_all-mpnet-base-v2',\n",
        "             4: 'cache_qwen2_7b_dmis-lab_biobert-v1.1',\n",
        "             5: 'cache_qwen2_7b_all-mpnet-base-v2',\n",
        "             6: 'cache_qwen2.5_14b_dmis-lab_biobert-v1.1',\n",
        "             7: 'cache_llama3.1_8b_dmis-lab_biobert-v1.1',\n",
        "             8: 'cache_gemma2_dmis-lab_biobert-v1.1'}\n",
        "n = 2\n",
        "# community reports\n",
        "root = os.getcwd()\n",
        "root = \"/root/projects/nano-graphrag/biomedical/ablation_study\"\n",
        "file = os.path.join(root, cache_dict[n], 'kv_store_community_reports.json')\n",
        "print(f\"Loading graph from file: {file}\\n\")\n",
        "\n",
        "with open(file, 'r') as f:\n",
        "    community_reports = json.load(f)\n",
        "\n",
        "# Funzione per estrarre tutti i report con rating > 8 dal file json caricato\n",
        "def extract_high_score_reports(community_reports, threshold=8):\n",
        "    # Ritorna una lista dei valori report_json con rating maggiore della soglia\n",
        "    return [\n",
        "        report_data[\"report_json\"]\n",
        "        for report_data in community_reports.values()\n",
        "        if float(report_data[\"report_json\"][\"rating\"]) > threshold\n",
        "    ]\n",
        "\n",
        "\n",
        "reports = extract_high_score_reports(community_reports, threshold=7)\n",
        "\n",
        "high_score_reports_summary = [report[\"summary\"] for report in reports]\n",
        "len(high_score_reports_summary)\n",
        "\n",
        "# Make 4 list of the from high_score_reports_summary\n",
        "batch_size = 4\n",
        "batches = [high_score_reports_summary[i:i + batch_size] for i in range(0, len(high_score_reports_summary), batch_size)]\n",
        "# for i, batch in enumerate(batches):\n",
        "#     with open(f\"batch_{i+1}_high_score_reports.json\", 'w') as f:\n",
        "#         json.dump(batch, f, indent=4)\n",
        "\n",
        "batches[0]\n",
        "for i, batch in enumerate(batches):\n",
        "    text = \"\\n\\n\".join(batch)\n",
        "\n",
        "text = \"\"\"**Corpus Description**\\n\\n\"\"\"+text\n",
        "print(text)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {},
      "outputs": [
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Here is the output in the requested JSON format:\n",
            "\n",
            "```json\n",
            "{\n",
            "  \"corpus_description\": \"The corpus explores the complex interactions between genetic variants, nutritional elements, and their impact on health and disease outcomes, focusing on entities such as genes, genetic variants, transcripts, proteins, nutrients, food, dietary patterns, physiological processes, metabolic pathways, molecular interactions, environmental factors, and diseases.\",\n",
            "  \"generated_questions\": \"\n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Investigating the impact of genetic variants on nutrient metabolism\n",
            "  Q1. How do genetic variants in metabolic pathway genes influence the response to dietary interventions aimed at weight management?\n",
            "  Q2. What is the role of transcriptomics in understanding the effects of nutrient-gene interactions on metabolic health?\n",
            "  Q3. How do dietary patterns modulate the expression of genes involved in inflammation, and what are the implications for disease risk?\n",
            "  Q4. Can specific genetic variants predict responsiveness to nutritional interventions targeting cardiovascular health?\n",
            "  Q5. How do environmental factors, such as lifestyle, interact with genetic predispositions to affect nutrient metabolism and disease susceptibility?\n",
            "\n",
            "  Task2: Exploring gene-disease interactions\n",
            "  Q1. What is the relationship between genetic variants associated with type 2 diabetes and dietary factors that influence disease risk?\n",
            "  Q2. How do genetic predispositions to obesity interact with dietary patterns to affect metabolic health outcomes?\n",
            "  Q3. Can nutrigenetic insights into inflammatory bowel disease inform personalized dietary recommendations?\n",
            "  Q4. What are the implications of genetic variants in genes involved in immune response for dietary management of autoimmune diseases?\n",
            "  Q5. How do molecular interactions between nutrients and genetic variants influence disease progression in complex conditions like psoriasis?\n",
            "\n",
            "  Task3: Analyzing nutritional impact on gene expression\n",
            "  Q1. How do specific nutrients or dietary patterns affect the expression of genes involved in metabolic pathways?\n",
            "  Q2. What is the impact of vitamin D status on gene expression in immune cells, and how does this relate to disease susceptibility?\n",
            "  Q3. Can certain dietary components modulate the epigenetic regulation of genes involved in cancer development?\n",
            "  Q4. How do polyphenols in food interact with genetic variants to influence cardiovascular health outcomes?\n",
            "  Q5. What are the effects of dietary fiber on gut microbiome composition and host gene expression in the context of metabolic health?\n",
            "\n",
            "  Task4: Investigating metabolic pathway insights\n",
            "  Q1. What role do metabolic pathways play in mediating the effects of nutrition-genetic interactions on physiological processes?\n",
            "  Q2. How do genetic variants in metabolic pathway genes influence the response to dietary interventions aimed at weight management?\n",
            "  Q3. Can systems biology approaches reveal novel insights into the interplay between nutrition, genetics, and metabolic health?\n",
            "  Q4. What are the implications of genetic variants in genes involved in lipid metabolism for dietary management of cardiovascular disease?\n",
            "  Q5. How do molecular interactions between nutrients and genetic variants influence disease progression in complex conditions like metabolic syndrome?\n",
            "\n",
            "  Task5: Examining environmental and nutrient modulation\n",
            "  Q1. How do environmental factors, such as lifestyle, interact with genetic predispositions to affect nutrient metabolism and disease susceptibility?\n",
            "  Q2. What is the impact of environmental toxins on gene expression and nutrient metabolism in the context of disease risk?\n",
            "  Q3. Can nutritional interventions mitigate the effects of environmental pollutants on human health?\n",
            "  Q4. How do socioeconomic factors influence access to healthy dietary patterns and genetic testing for disease risk?\n",
            "  Q5. What are the implications of gene-environment interactions for personalized nutrition and disease prevention?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Applying nutrigenetic insights in clinical practice\n",
            "  Q1. How can genetic testing for nutrient metabolism inform personalized dietary recommendations in clinical settings?\n",
            "  Q2. What are the implications of genetic variants in genes involved in drug metabolism for nutritional interventions in patients?\n",
            "  Q3. Can nutrigenetic insights into disease risk inform targeted dietary interventions in high-risk populations?\n",
            "  Q4. How do healthcare professionals integrate genetic information into dietary counseling for patients with chronic diseases?\n",
            "  Q5. What are the challenges and opportunities for implementing nutrigenetic testing in clinical practice?\n",
            "\n",
            "  Task2: Investigating disease prevention and management\n",
            "  Q1. How can dietary patterns be tailored to prevent or manage chronic diseases based on genetic risk factors?\n",
            "  Q2. What are the implications of genetic variants in genes involved in immune response for dietary management of autoimmune diseases?\n",
            "  Q3. Can nutrigenetic insights into inflammatory bowel disease inform personalized dietary recommendations?\n",
            "  Q4. How do genetic predispositions to obesity interact with dietary patterns to affect metabolic health outcomes?\n",
            "  Q5. What are the implications of molecular interactions between nutrients and genetic variants for disease progression in complex conditions?\n",
            "\n",
            "  Task3: Exploring nutrigenetic testing and counseling\n",
            "  Q1. What are the current limitations and future directions of nutrigenetic testing for disease risk and dietary response?\n",
            "  Q2. How can healthcare professionals communicate complex genetic information to patients and tailor dietary recommendations?\n",
            "  Q3. Can genetic testing for nutrient metabolism inform personalized dietary recommendations in clinical settings?\n",
            "  Q4. What are the ethical considerations for using genetic information in dietary counseling?\n",
            "  Q5. How do healthcare professionals stay updated on the latest nutrigenetic research and its applications in clinical practice?\n",
            "\n",
            "  Task4: Analyzing case studies and applications\n",
            "  Q1. What are some successful examples of applying nutrigenetic insights in clinical practice to improve patient outcomes?\n",
            "  Q2. How do healthcare professionals integrate genetic information into dietary counseling for patients with complex conditions like metabolic syndrome?\n",
            "  Q3. Can case studies illustrate the benefits and challenges of implementing nutrigenetic testing in clinical settings?\n",
            "  Q4. What are the implications of genetic variants in genes involved in metabolic pathways for dietary management of disease?\n",
            "  Q5. How do healthcare professionals balance genetic risk factors with environmental influences when developing dietary recommendations?\n",
            "\n",
            "  Task5: Examining public health implications\n",
            "  Q1. What are the public health implications of integrating nutrigenetic insights into dietary guidelines and policy?\n",
            "  Q2. How can population-level genetic data inform targeted dietary interventions for disease prevention?\n",
            "  Q3. Can nutrigenetic research inform strategies for health disparities and unequal access to healthy dietary patterns?\n",
            "  Q4. What are the challenges and opportunities for implementing nutrigenetic testing in public health programs?\n",
            "  Q5. How do socioeconomic factors influence access to healthy dietary patterns and genetic testing for disease risk?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understanding genetic testing for nutrition\n",
            "  Q1. What genetic tests are available for assessing nutrient metabolism and dietary response?\n",
            "  Q2. How can genetic information inform personalized dietary recommendations for weight management?\n",
            "  Q3. Can genetic testing predict individual responses to specific nutrients or dietary patterns?\n",
            "  Q4. What are the limitations and potential biases of commercial genetic testing for nutrition and health?\n",
            "  Q5. How do I interpret genetic test results and apply them to my dietary choices?\n",
            "\n",
            "  Task2: Exploring dietary implications of genetic risk\n",
            "  Q1. If I have a genetic predisposition to type 2 diabetes, what dietary changes can I make to mitigate risk?\n",
            "  Q2. How can I tailor my diet to my genetic profile to optimize metabolic health?\n",
            "  Q3. Can genetic testing inform my decision on whether to follow a specific dietary pattern (e.g., vegan, gluten-free)?\n",
            "  Q4. What are the implications of genetic variants in genes involved in nutrient metabolism for my dietary choices?\n",
            "  Q5. How do I balance genetic risk factors with environmental influences when making dietary decisions?\n",
            "\n",
            "  Task3: Investigating nutrition and disease prevention\n",
            "  Q1. Can genetic testing predict my risk of developing certain diseases, and how can I use that information to prevent them through dietary changes?\n",
            "  Q2. What dietary patterns can help prevent or manage chronic diseases based on my genetic risk factors?\n",
            "  Q3. How do genetic predispositions to certain diseases interact with dietary factors to affect health outcomes?\n",
            "  Q4. Can nutrigenetic insights into disease risk inform personalized dietary recommendations for prevention and management?\n",
            "  Q5. What are the implications of molecular interactions between nutrients and genetic variants for my health and disease risk?\n",
            "\n",
            "  Task4: Examining the role of epigenetics\n",
            "  Q1. What is the relationship between epigenetic changes and dietary patterns in influencing disease risk?\n",
            "  Q2. Can nutritional interventions affect epigenetic marks and influence gene expression?\n",
            "  Q3. How do environmental factors interact with genetic predispositions to affect epigenetic regulation?\n",
            "  Q4. What are the implications of epigenetic changes for personalized nutrition and disease prevention?\n",
            "  Q5. Can epigenetic testing provide additional insights into my nutritional needs and health risks?\n",
            "\n",
            "  Task5: Evaluating commercial nutrigenetic services\n",
            "  Q1. What are the benefits and limitations of commercial nutrigenetic testing services for consumers?\n",
            "  Q2. How do I evaluate the scientific validity and clinical utility of commercial genetic tests for nutrition and health?\n",
            "  Q3. Can commercial nutrigenetic services provide actionable insights into my dietary needs and health risks?\n",
            "  Q4. What are the potential biases and conflicts of interest in commercial nutrigenetic testing?\n",
            "  Q5. How do I integrate commercial nutrigenetic testing into my overall health and wellness plan?\"\n",
            "  }\n",
            "}\n",
            "```\n",
            "<<< ----------------- >>>\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "```json\n",
            "{\n",
            "  \"corpus_description\": \"The corpus explores the interplay between genetic factors, nutritional elements, and disease risks, particularly focusing on obesity and type 2 diabetes. It includes key genes such as CDKAL1, DNAJC27, SEC16B, TFAP2B, TCF7L2, WFS1, and FTO, and their interactions with dietary factors, metabolic pathways, and environmental influences.\",\n",
            "  \"generated_questions\": \"\n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Investigating genetic influences on nutrient interactions\n",
            "  Q1. How do genetic variants in the CDKAL1 gene influence an individual's response to high-fiber diets in terms of obesity risk?\n",
            "  Q2. What is the role of the FTO gene in modulating body weight in response to different dietary patterns?\n",
            "  Q3. How do variations in the TCF7L2 gene affect glucose metabolism in the context of varying carbohydrate intake?\n",
            "  Q4. Can specific genetic variants in the SEC16B gene predict responsiveness to low-fat diets for weight management?\n",
            "  Q5. How do DNAJC27 gene variants interact with dietary factors to influence obesity risk?\n",
            "\n",
            "  Task2: Exploring nutritional impact on gene expression\n",
            "  Q1. How does fiber intake affect the expression of genes involved in metabolic pathways related to obesity?\n",
            "  Q2. What is the impact of different dietary fats on the transcription of genes associated with type 2 diabetes?\n",
            "  Q3. In what ways do nutrients influence the epigenetic regulation of genes like CDKAL1 and FTO?\n",
            "  Q4. Can certain dietary patterns alter the expression of genes involved in insulin sensitivity?\n",
            "  Q5. How do vitamins and minerals affect the regulation of genes associated with metabolic diseases?\n",
            "\n",
            "  Task3: Analyzing disease and genetic-nutrient interactions\n",
            "  Q1. How do interactions between genetic variants and dietary factors contribute to the risk of developing type 2 diabetes?\n",
            "  Q2. What role do genetic-nutrient interactions play in the progression of obesity-related complications?\n",
            "  Q3. Can dietary interventions mitigate the disease risk associated with specific genetic variants linked to obesity?\n",
            "  Q4. How do environmental factors influence the relationship between genetic predispositions and disease outcomes?\n",
            "  Q5. What are the implications of genetic-nutrient interactions for personalized nutrition and disease prevention?\n",
            "\n",
            "  Task4: Investigating metabolic pathway insights\n",
            "  Q1. What metabolic pathways are modulated by genetic variants in response to dietary factors?\n",
            "  Q2. How do molecular interactions within metabolic pathways mediate the effects of nutrition on disease risk?\n",
            "  Q3. Can understanding metabolic pathway interactions inform strategies for disease management through nutrition?\n",
            "  Q4. What is the role of key enzymes in mediating the impact of nutrients on metabolic diseases?\n",
            "  Q5. How do genetic variants affect the flux through metabolic pathways in response to dietary changes?\n",
            "\n",
            "  Task5: Examining environmental and nutrient modulation\n",
            "  Q1. How do environmental toxins interact with genetic predispositions to affect disease susceptibility?\n",
            "  Q2. What is the impact of lifestyle factors, such as physical activity, on genetic-nutrient interactions and health outcomes?\n",
            "  Q3. Can environmental factors modulate the epigenetic effects of nutritional exposures on gene expression?\n",
            "  Q4. How do socioeconomic factors influence access to nutrients and thereby affect genetic-nutrient interactions?\n",
            "  Q5. In what ways can policy interventions impact environmental factors to mitigate disease risks associated with genetic predispositions?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Applying genetic insights to dietary recommendations\n",
            "  Q1. How can genetic testing inform personalized dietary advice for obesity management?\n",
            "  Q2. What genetic factors should healthcare providers consider when recommending dietary interventions for type 2 diabetes?\n",
            "  Q3. Can genetic information improve the targeting of nutritional interventions for disease prevention?\n",
            "  Q4. How do healthcare professionals integrate genetic data into nutrition counseling?\n",
            "  Q5. What are the implications of genetic variability for the effectiveness of standard dietary guidelines?\n",
            "\n",
            "  Task2: Understanding disease risk and management\n",
            "  Q1. How can understanding genetic-nutrient interactions improve disease risk assessment and management?\n",
            "  Q2. What role do genetic factors play in the variability of patient responses to dietary interventions?\n",
            "  Q3. Can nutrigenetic insights enhance the development of targeted therapies for metabolic diseases?\n",
            "  Q4. How do environmental and lifestyle factors influence the implementation of nutrigenetic information in clinical practice?\n",
            "  Q5. What are the challenges and opportunities in translating nutrigenetic research into clinical practice?\n",
            "\n",
            "  Task3: Evaluating nutritional interventions\n",
            "  Q1. How effective are personalized nutrition interventions based on genetic information in improving health outcomes?\n",
            "  Q2. What are the key genetic markers for predicting responses to common dietary interventions?\n",
            "  Q3. Can genetic testing identify individuals at high risk of adverse effects from certain dietary components?\n",
            "  Q4. How do healthcare professionals evaluate the scientific evidence supporting nutrigenetic applications?\n",
            "  Q5. What are the ethical considerations in using genetic information for dietary recommendations?\n",
            "\n",
            "  Task4: Communicating with patients\n",
            "  Q1. How can healthcare providers effectively communicate complex genetic information to patients?\n",
            "  Q2. What are the best practices for discussing nutrigenetic testing results with patients?\n",
            "  Q3. How do patients' perceptions of genetic risk influence their adherence to dietary recommendations?\n",
            "  Q4. Can genetic counseling support patient understanding of nutrigenetic information?\n",
            "  Q5. How do healthcare professionals stay updated on the latest research in nutrigenetics to advise patients?\n",
            "\n",
            "  Task5: Implementing evidence-based practice\n",
            "  Q1. What are the current evidence-based guidelines for integrating nutrigenetics into clinical practice?\n",
            "  Q2. How can healthcare systems support the integration of nutrigenetic services?\n",
            "  Q3. What are the challenges in standardizing nutrigenetic testing and interpretation?\n",
            "  Q4. How do healthcare professionals assess the quality of genetic testing services for clinical use?\n",
            "  Q5. What are the future directions for research in translating nutrigenetics into clinical practice?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understanding personal genetic information\n",
            "  Q1. What does my genetic data reveal about my risk of developing obesity or type 2 diabetes?\n",
            "  Q2. How can I use my genetic information to make better dietary choices?\n",
            "  Q3. Can genetic testing help identify foods that I should avoid?\n",
            "  Q4. How do I interpret my genetic risk score for metabolic diseases?\n",
            "  Q5. What are the limitations of genetic testing for nutritional guidance?\n",
            "\n",
            "  Task2: Applying genetic insights to diet\n",
            "  Q1. How can I tailor my diet to my genetic predispositions for optimal health?\n",
            "  Q2. What dietary changes can I make to mitigate my genetic risk of obesity?\n",
            "  Q3. Can I use my genetic information to optimize my nutrient intake?\n",
            "  Q4. How do I incorporate genetic insights into my daily food choices?\n",
            "  Q5. What are the potential benefits and risks of following a genetically informed diet?\n",
            "\n",
            "  Task3: Exploring dietary options\n",
            "  Q1. What are the best dietary patterns for someone with a high genetic risk of type 2 diabetes?\n",
            "  Q2. How do different diets (e.g., keto, vegan) interact with my genetic makeup?\n",
            "  Q3. Can I use genetic information to guide my decisions about dietary supplements?\n",
            "  Q4. How do I balance my genetic predispositions with other factors like lifestyle and environment?\n",
            "  Q5. What are the implications of my genetic data for long-term health and disease prevention?\n",
            "\n",
            "  Task4: Considering environmental factors\n",
            "  Q1. How do environmental factors like stress and physical activity influence my genetic predispositions?\n",
            "  Q2. Can changing my environment or lifestyle mitigate my genetic risk of disease?\n",
            "  Q3. How do socioeconomic factors impact my access to healthy food choices and thereby my health?\n",
            "  Q4. What role do environmental toxins play in modulating genetic effects on health?\n",
            "  Q5. How can I use my genetic information to advocate for policies supporting healthier environments?\n",
            "\n",
            "  Task5: Engaging with healthcare providers\n",
            "  Q1. How can I work with my healthcare provider to integrate genetic insights into my healthcare plan?\n",
            "  Q2. What questions should I ask my healthcare provider about my genetic data and nutrition?\n",
            "  Q3. Can healthcare providers help me develop a personalized nutrition plan based on my genetic information?\n",
            "  Q4. How do I ensure that my genetic data is used responsibly in clinical practice?\n",
            "  Q5. What are the future possibilities for using genetic information in preventive healthcare?\"\n",
            "}\n",
            "```\n",
            "<<< ----------------- >>>\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Here is the output:\n",
            "\n",
            "```\n",
            "{\n",
            "  \"corpus_description\": \"The corpus explores the relationships between genetic variants, nutritional elements, and disease susceptibility, focusing on obesity, Type 2 Diabetes, and personalized nutrition.\",\n",
            "  \"generated_questions\": \"\n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Investigating the impact of genetic variants on obesity risk\n",
            "  Q1. How do specific genetic variants, such as the FTO RS9939609 polymorphism, influence individual responses to high-protein diets in terms of weight loss and waist circumference reduction?\n",
            "  Q2. What are the underlying metabolic pathways and molecular interactions that mediate the effects of genetic variants on obesity risk?\n",
            "  Q3. How do environmental factors, such as lifestyle and dietary patterns, modulate the interactions between genes and nutrients in influencing obesity susceptibility?\n",
            "  Q4. What is the relationship between genetic variants associated with obesity and the regulation of physiological processes, such as glucose homeostasis and lipid metabolism?\n",
            "  Q5. How do genetic-nutrient interactions influence the risk of developing obesity-related comorbidities?\n",
            "\n",
            "  Task2: Exploring the role of genetic variants in Type 2 Diabetes risk\n",
            "  Q1. How do genetic variants, such as rs10399931 and rs1044498, affect the expression of genes involved in glucose homeostasis and insulin sensitivity?\n",
            "  Q2. What are the downstream effects of these genetic variants on metabolic pathways and molecular interactions that contribute to Type 2 Diabetes risk?\n",
            "  Q3. How do dietary patterns, such as high-protein diets, interact with genetic variants to influence Type 2 Diabetes risk?\n",
            "  Q4. What is the relationship between genetic variants associated with Type 2 Diabetes and the regulation of physiological processes, such as insulin secretion and glucose uptake?\n",
            "  Q5. How do environmental factors, such as physical activity and lifestyle, modulate the interactions between genes and nutrients in influencing Type 2 Diabetes risk?\n",
            "\n",
            "  Task3: Analyzing the relationship between genetic variants and disease susceptibility\n",
            "  Q1. How do genetic variants, such as those in the TCF7L2 gene, influence disease susceptibility through their effects on molecular interactions and metabolic pathways?\n",
            "  Q2. What are the underlying biological mechanisms that link genetic variants to disease risk, and how do nutritional elements modulate these effects?\n",
            "  Q3. How do environmental factors, such as diet and lifestyle, interact with genetic variants to influence disease susceptibility?\n",
            "  Q4. What is the relationship between genetic variants associated with disease susceptibility and the regulation of physiological processes, such as inflammation and oxidative stress?\n",
            "  Q5. How do genetic-nutrient interactions influence the risk of developing complex diseases, such as obesity and Type 2 Diabetes?\n",
            "\n",
            "  Task4: Investigating the impact of nutritional elements on gene expression\n",
            "  Q1. How do specific nutrients, such as protein and fiber, affect gene expression and transcript production in the context of obesity and Type 2 Diabetes?\n",
            "  Q2. What are the downstream effects of nutritional elements on metabolic pathways and molecular interactions that contribute to disease risk?\n",
            "  Q3. How do dietary patterns, such as Mediterranean diets, influence gene expression and disease susceptibility?\n",
            "  Q4. What is the relationship between nutritional elements and the regulation of physiological processes, such as glucose homeostasis and lipid metabolism?\n",
            "  Q5. How do environmental factors, such as lifestyle and physical activity, modulate the interactions between nutrients and genes in influencing disease risk?\n",
            "\n",
            "  Task5: Exploring the applications of nutrigenetics in personalized nutrition\n",
            "  Q1. How can genetic information be used to tailor dietary recommendations for individuals with specific genetic variants associated with obesity and Type 2 Diabetes?\n",
            "  Q2. What are the implications of genetic-nutrient interactions for personalized nutrition and disease prevention?\n",
            "  Q3. How do environmental factors, such as lifestyle and dietary patterns, influence the effectiveness of personalized nutrition interventions?\n",
            "  Q4. What are the challenges and limitations of implementing personalized nutrition based on genetic information?\n",
            "  Q5. How can nutrigenetics be integrated into clinical practice to improve health outcomes and disease prevention?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Applying nutrigenetic insights in patient care\n",
            "  Q1. How can healthcare professionals use genetic information to inform dietary recommendations for patients with obesity and Type 2 Diabetes?\n",
            "  Q2. What are the potential benefits and limitations of using genetic testing to guide nutrition interventions in clinical practice?\n",
            "  Q3. How do environmental factors, such as lifestyle and dietary patterns, influence the effectiveness of nutrigenetic-based interventions?\n",
            "  Q4. What are the implications of genetic-nutrient interactions for patient care and disease management?\n",
            "  Q5. How can healthcare professionals stay up-to-date with the latest research in nutrigenetics and apply it in clinical practice?\n",
            "\n",
            "  Task2: Understanding the role of genetic variants in disease risk\n",
            "  Q1. How do genetic variants, such as the FTO RS9939609 polymorphism, influence disease risk and progression?\n",
            "  Q2. What are the underlying biological mechanisms that link genetic variants to disease risk, and how do nutritional elements modulate these effects?\n",
            "  Q3. How do environmental factors, such as diet and lifestyle, interact with genetic variants to influence disease susceptibility?\n",
            "  Q4. What is the relationship between genetic variants associated with disease susceptibility and the regulation of physiological processes, such as inflammation and oxidative stress?\n",
            "  Q5. How do genetic-nutrient interactions influence the risk of developing complex diseases, such as obesity and Type 2 Diabetes?\n",
            "\n",
            "  Task3: Exploring the applications of nutrigenetics in disease prevention\n",
            "  Q1. How can nutrigenetic insights be used to prevent or manage obesity and Type 2 Diabetes?\n",
            "  Q2. What are the potential benefits and limitations of using genetic testing to guide disease prevention strategies?\n",
            "  Q3. How do environmental factors, such as lifestyle and dietary patterns, influence the effectiveness of nutrigenetic-based interventions?\n",
            "  Q4. What are the implications of genetic-nutrient interactions for disease prevention and management?\n",
            "  Q5. How can healthcare professionals integrate nutrigenetics into their practice to improve patient outcomes and disease prevention?\n",
            "\n",
            "  Task4: Investigating the impact of nutritional elements on disease risk\n",
            "  Q1. How do specific nutrients, such as protein and fiber, influence disease risk and progression?\n",
            "  Q2. What are the downstream effects of nutritional elements on metabolic pathways and molecular interactions that contribute to disease risk?\n",
            "  Q3. How do dietary patterns, such as Mediterranean diets, influence disease susceptibility?\n",
            "  Q4. What is the relationship between nutritional elements and the regulation of physiological processes, such as glucose homeostasis and lipid metabolism?\n",
            "  Q5. How do environmental factors, such as lifestyle and physical activity, modulate the interactions between nutrients and genes in influencing disease risk?\n",
            "\n",
            "  Task5: Evaluating the evidence for nutrigenetic-based interventions\n",
            "  Q1. What is the current evidence for the effectiveness of nutrigenetic-based interventions in disease prevention and management?\n",
            "  Q2. How can healthcare professionals evaluate the quality of evidence for nutrigenetic-based interventions?\n",
            "  Q3. What are the potential risks and benefits of using genetic testing to guide nutrition interventions?\n",
            "  Q4. How do environmental factors, such as lifestyle and dietary patterns, influence the effectiveness of nutrigenetic-based interventions?\n",
            "  Q5. What are the implications of genetic-nutrient interactions for patient care and disease management?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understanding personalized nutrition based on genetic information\n",
            "  Q1. How can genetic information be used to tailor dietary recommendations for individuals with specific genetic variants associated with obesity and Type 2 Diabetes?\n",
            "  Q2. What are the potential benefits and limitations of using genetic testing to guide nutrition interventions?\n",
            "  Q3. How do environmental factors, such as lifestyle and dietary patterns, influence the effectiveness of personalized nutrition interventions?\n",
            "  Q4. What are the implications of genetic-nutrient interactions for personalized nutrition and disease prevention?\n",
            "  Q5. How can individuals use genetic information to make informed choices about their diet and lifestyle?\n",
            "\n",
            "  Task2: Exploring the role of genetic variants in nutritional responses\n",
            "  Q1. How do genetic variants, such as the FTO RS9939609 polymorphism, influence individual responses to nutrients and dietary patterns?\n",
            "  Q2. What are the underlying biological mechanisms that link genetic variants to nutritional responses?\n",
            "  Q3. How do environmental factors, such as diet and lifestyle, interact with genetic variants to influence nutritional responses?\n",
            "  Q4. What is the relationship between genetic variants associated with nutritional responses and the regulation of physiological processes, such as glucose homeostasis and lipid metabolism?\n",
            "  Q5. How do genetic-nutrient interactions influence the risk of developing complex diseases, such as obesity and Type 2 Diabetes?\n",
            "\n",
            "  Task3: Investigating the impact of nutritional elements on health outcomes\n",
            "  Q1. How do specific nutrients, such as protein and fiber, influence health outcomes and disease risk?\n",
            "  Q2. What are the downstream effects of nutritional elements on metabolic pathways and molecular interactions that contribute to health outcomes?\n",
            "  Q3. How do dietary patterns, such as Mediterranean diets, influence health outcomes and disease susceptibility?\n",
            "  Q4. What is the relationship between nutritional elements and the regulation of physiological processes, such as glucose homeostasis and lipid metabolism?\n",
            "  Q5. How do environmental factors, such as lifestyle and physical activity, modulate the interactions between nutrients and genes in influencing health outcomes?\n",
            "\n",
            "  Task4: Evaluating the evidence for personalized nutrition\n",
            "  Q1. What is the current evidence for the effectiveness of personalized nutrition interventions based on genetic information?\n",
            "  Q2. How can individuals evaluate the quality of evidence for personalized nutrition recommendations?\n",
            "  Q3. What are the potential risks and benefits of using genetic testing to guide nutrition interventions?\n",
            "  Q4. How do environmental factors, such as lifestyle and dietary patterns, influence the effectiveness of personalized nutrition interventions?\n",
            "  Q5. What are the implications of genetic-nutrient interactions for personalized nutrition and disease prevention?\n",
            "\n",
            "  Task5: Applying nutrigenetic insights in daily life\n",
            "  Q1. How can individuals use genetic information to make informed choices about their diet and lifestyle?\n",
            "  Q2. What are the practical implications of genetic-nutrient interactions for daily life and health outcomes?\n",
            "  Q3. How do environmental factors, such as lifestyle and dietary patterns, influence the effectiveness of nutrigenetic-based interventions?\n",
            "  Q4. What are the potential benefits and limitations of using genetic testing to guide nutrition interventions in daily life?\n",
            "  Q5. How can individuals integrate nutrigenetic insights into their daily life to improve health outcomes and disease prevention?\n",
            "\"\n",
            "}\n",
            "```\n",
            "<<< ----------------- >>>\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Here is the output:\n",
            "\n",
            "```\n",
            "{\n",
            "  \"corpus_description\": \"The corpus explores various biomedical topics, including the relationships between serum uric acid levels and cardiovascular risk factors, the role of IL-6 in inflammation and immune responses, the interplay between genes and ovarian cancer risk, and the interaction between T cells, TCRs, macrophages, and neoepitopes in immune responses.\",\n",
            "  \"generated_questions\": \"\n",
            "  \n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Investigating the impact of genetic variants on nutritional responses\n",
            "  Q1. How do genetic variants, such as rs3733951 and rs893006, influence individual responses to dietary patterns and nutrients, and what are the subsequent effects on metabolic pathways and physiological processes?\n",
            "  Q2. What is the relationship between genetic variants and the expression of genes involved in inflammation, such as IL1B and PTGS1, in response to different dietary patterns?\n",
            "  Q3. How do specific genetic variants affect the activity of enzymes involved in nutrient metabolism, such as ALOX5, and what are the resulting effects on health and disease outcomes?\n",
            "  Q4. Can genetic variants, such as IL1A RS17561, modulate the impact of nutrients on immune responses and disease susceptibility?\n",
            "  Q5. How do genetic variants influence the interaction between T cells, TCRs, macrophages, and neoepitopes in immune responses, and what are the implications for personalized nutrition?\n",
            "\n",
            "  Task2: Exploring the role of nutrients in gene expression\n",
            "  Q1. What are the effects of different nutrients and dietary patterns on gene expression, transcript production, and protein synthesis, and how do these effects impact health and disease outcomes?\n",
            "  Q2. How do particular nutrients, such as those involved in the ABO gene, affect the regulation of immune responses and disease susceptibility?\n",
            "  Q3. Can specific dietary patterns modulate the expression of genes involved in metabolic pathways, such as those related to serum uric acid levels and cardiovascular risk factors?\n",
            "  Q4. What is the relationship between nutrient intake and the expression of genes involved in inflammation, such as IL-6 and TNF?\n",
            "  Q5. How do nutrients and dietary patterns influence the interaction between genetic variants and disease risk or progression?\n",
            "\n",
            "  Task3: Analyzing disease and genetic-nutrient interactions\n",
            "  Q1. How do dietary factors interact with genes to modulate disease risk or progression, and what are the underlying molecular mechanisms?\n",
            "  Q2. What is the role of genetic variants in modulating the impact of nutrients on disease susceptibility and progression?\n",
            "  Q3. Can specific nutrients or dietary patterns affect the expression of genes involved in disease pathogenesis, such as those related to ovarian cancer risk?\n",
            "  Q4. How do environmental factors alter the interactions between genes and nutrients, potentially influencing health outcomes and disease susceptibility?\n",
            "  Q5. What are the implications of genetic-nutrient interactions for personalized nutrition and disease prevention?\n",
            "\n",
            "  Task4: Investigating metabolic pathway insights\n",
            "  Q1. What roles do metabolic pathways and molecular interactions play in mediating the effects of nutrition-genetic interactions on physiological processes and disease outcomes?\n",
            "  Q2. How do genetic variants affect the activity of enzymes involved in nutrient metabolism, and what are the resulting effects on metabolic pathways and disease risk?\n",
            "  Q3. Can specific dietary patterns modulate the expression of genes involved in metabolic pathways, and what are the implications for health and disease outcomes?\n",
            "  Q4. What is the relationship between genetic variants, nutrient intake, and the regulation of metabolic pathways?\n",
            "  Q5. How do molecular interactions between nutrients, genes, and environmental factors influence disease susceptibility and progression?\n",
            "\n",
            "  Task5: Examining environmental and nutrient modulation\n",
            "  Q1. How do environmental factors alter the interactions between genes and nutrients, potentially influencing health outcomes and disease susceptibility?\n",
            "  Q2. What are the effects of environmental factors on the expression of genes involved in nutrient metabolism and disease pathogenesis?\n",
            "  Q3. Can specific nutrients or dietary patterns modulate the impact of environmental factors on gene expression and disease risk?\n",
            "  Q4. How do genetic variants influence the interaction between environmental factors and nutrients, and what are the resulting effects on health and disease outcomes?\n",
            "  Q5. What are the implications of environmental and nutrient modulation for personalized nutrition and disease prevention?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Applying nutrigenetic insights in clinical practice\n",
            "  Q1. How can genetic testing be used to inform personalized nutrition recommendations for patients with specific genetic variants?\n",
            "  Q2. What are the implications of genetic-nutrient interactions for the prevention and treatment of chronic diseases, such as cardiovascular disease and ovarian cancer?\n",
            "  Q3. Can nutrigenetic insights be used to develop targeted dietary interventions for patients with specific genetic profiles?\n",
            "  Q4. How do healthcare professionals integrate nutrigenetic information into clinical practice, and what are the potential benefits and challenges?\n",
            "  Q5. What are the limitations and potential risks of applying nutrigenetic insights in clinical practice?\n",
            "\n",
            "  Task2: Exploring the role of nutrigenetics in disease prevention\n",
            "  Q1. What are the potential benefits of using nutrigenetic information for disease prevention, and how can it be integrated into public health initiatives?\n",
            "  Q2. How do genetic variants influence the effectiveness of dietary interventions for disease prevention and treatment?\n",
            "  Q3. Can nutrigenetic insights be used to identify high-risk populations and develop targeted prevention strategies?\n",
            "  Q4. What are the implications of nutrigenetic research for the development of personalized nutrition recommendations in clinical practice?\n",
            "  Q5. How do environmental factors interact with genetic variants to influence disease risk and progression?\n",
            "\n",
            "  Task3: Investigating the impact of nutrigenetics on patient outcomes\n",
            "  Q1. What are the potential benefits of using nutrigenetic information for improving patient outcomes, and how can it be integrated into clinical practice?\n",
            "  Q2. How do genetic variants influence patient responses to dietary interventions, and what are the implications for treatment outcomes?\n",
            "  Q3. Can nutrigenetic insights be used to develop personalized nutrition recommendations that improve patient outcomes?\n",
            "  Q4. What are the limitations and potential risks of applying nutrigenetic insights in clinical practice?\n",
            "  Q5. How do healthcare professionals communicate nutrigenetic information to patients, and what are the implications for patient education and empowerment?\n",
            "\n",
            "  Task4: Examining the role of nutrigenetics in public health\n",
            "  Q1. What are the potential benefits of using nutrigenetic information for public health initiatives, and how can it be integrated into policy and practice?\n",
            "  Q2. How do genetic variants influence the effectiveness of dietary interventions for public health initiatives?\n",
            "  Q3. Can nutrigenetic insights be used to develop targeted nutrition recommendations for specific populations?\n",
            "  Q4. What are the implications of nutrigenetic research for the development of public health policy and practice?\n",
            "  Q5. How do environmental factors interact with genetic variants to influence disease risk and progression in different populations?\n",
            "\n",
            "  Task5: Investigating the future of nutrigenetics\n",
            "  Q1. What are the potential future directions for nutrigenetic research, and how can it be integrated into clinical practice and public health initiatives?\n",
            "  Q2. How do genetic variants influence the interaction between nutrients and environmental factors, and what are the implications for health and disease outcomes?\n",
            "  Q3. Can nutrigenetic insights be used to develop personalized nutrition recommendations that take into account an individual's genetic profile and environmental factors?\n",
            "  Q4. What are the potential benefits and challenges of integrating nutrigenetic information into electronic health records and clinical decision support systems?\n",
            "  Q5. How do healthcare professionals stay up-to-date with the latest nutrigenetic research and its applications in clinical practice?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understanding personalized nutrition\n",
            "  Q1. How can genetic testing be used to inform personalized nutrition recommendations, and what are the potential benefits for health and well-being?\n",
            "  Q2. What are the key genetic variants that influence dietary responses, and how can they be used to develop personalized nutrition plans?\n",
            "  Q3. Can personalized nutrition recommendations based on genetic information help improve health outcomes and reduce disease risk?\n",
            "  Q4. What are the limitations and potential risks of using genetic information for personalized nutrition?\n",
            "  Q5. How do I interpret my genetic test results and apply them to my dietary choices?\n",
            "\n",
            "  Task2: Exploring the role of genetics in nutrition\n",
            "  Q1. What is the relationship between genetic variants and nutrient metabolism, and how do they influence dietary responses?\n",
            "  Q2. How do genetic variants affect the activity of enzymes involved in nutrient metabolism, and what are the resulting effects on health and disease outcomes?\n",
            "  Q3. Can genetic testing be used to identify genetic variants that influence dietary responses and disease risk?\n",
            "  Q4. What are the implications of genetic-nutrient interactions for personalized nutrition and disease prevention?\n",
            "  Q5. How do environmental factors interact with genetic variants to influence health outcomes and disease susceptibility?\n",
            "\n",
            "  Task3: Investigating the impact of nutrigenetics on health\n",
            "  Q1. What are the potential benefits of using nutrigenetic information for improving health outcomes, and how can it be integrated into daily life?\n",
            "  Q2. How do genetic variants influence the effectiveness of dietary interventions for improving health outcomes?\n",
            "  Q3. Can nutrigenetic insights be used to develop personalized nutrition recommendations that improve health outcomes and reduce disease risk?\n",
            "  Q4. What are the limitations and potential risks of applying nutrigenetic insights in daily life?\n",
            "  Q5. How do I use genetic information to make informed dietary choices and improve my health outcomes?\n",
            "\n",
            "  Task4: Examining the role of nutrigenetics in disease prevention\n",
            "  Q1. What are the potential benefits of using nutrigenetic information for disease prevention, and how can it be integrated into daily life?\n",
            "  Q2. How do genetic variants influence disease risk and progression, and what are the implications for personalized nutrition and disease prevention?\n",
            "  Q3. Can nutrigenetic insights be used to identify high-risk individuals and develop targeted prevention strategies?\n",
            "  Q4. What are the implications of nutrigenetic research for the development of personalized nutrition recommendations for disease prevention?\n",
            "  Q5. How do environmental factors interact with genetic variants to influence disease risk and progression?\n",
            "\n",
            "  Task5: Investigating the future of nutrigenetics\n",
            "  Q1. What are the potential future directions for nutrigenetic research, and how can it be integrated into daily life and public health initiatives?\n",
            "  Q2. How do genetic variants influence the interaction between nutrients and environmental factors, and what are the implications for health and disease outcomes?\n",
            "  Q3. Can nutrigenetic insights be used to develop personalized nutrition recommendations that take into account an individual's genetic profile and environmental factors?\n",
            "  Q4. What are the potential benefits and challenges of integrating nutrigenetic information into electronic health records and clinical decision support systems?\n",
            "  Q5. How do I stay up-to-date with the latest nutrigenetic research and its applications in personalized nutrition and disease prevention?\n",
            "\"\n",
            "}\n",
            "```\n",
            "<<< ----------------- >>>\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Here is the output:\n",
            "\n",
            "```\n",
            "{\n",
            "  \"corpus_description\": \"The corpus explores the complex relationships between genetic variants, nutritional elements, and disease outcomes, focusing on areas such as nutrigenetics, immune responses, and metabolic regulation.\",\n",
            "  \"generated_questions\": \"\n",
            "  \n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Investigating the impact of genetic variants on nutrient metabolism\n",
            "  Q1. How do specific genetic variants in genes involved in metabolic pathways affect the body's response to dietary carbohydrates?\n",
            "  Q2. What are the implications of genetic variation on the regulation of nutrient uptake and utilization in the body?\n",
            "  Q3. How do genetic variants influence the activity of enzymes involved in nutrient metabolism?\n",
            "  Q4. Can genetic variants in metabolic pathways predict an individual's response to specific dietary interventions?\n",
            "  Q5. How do environmental factors, such as diet, interact with genetic variants to influence metabolic health?\n",
            "\n",
            "  Task2: Exploring the role of nutrigenetics in disease prevention\n",
            "  Q1. What is the relationship between genetic variants, nutrient intake, and the risk of developing metabolic disorders?\n",
            "  Q2. How do dietary patterns, such as a high-fat diet, interact with genetic variants to influence disease susceptibility?\n",
            "  Q3. Can nutrigenetic insights be used to develop personalized dietary recommendations for disease prevention?\n",
            "  Q4. What are the potential therapeutic applications of nutrigenetics in the management of chronic diseases?\n",
            "  Q5. How do genetic variants in immune-related genes influence the response to dietary interventions?\n",
            "\n",
            "  Task3: Analyzing the effects of nutrients on gene expression\n",
            "  Q1. How do specific nutrients, such as omega-3 fatty acids, affect gene expression and protein synthesis?\n",
            "  Q2. What are the mechanisms by which dietary patterns, such as a Mediterranean diet, influence gene expression and health outcomes?\n",
            "  Q3. Can nutrients and dietary patterns modulate the expression of genes involved in disease susceptibility?\n",
            "  Q4. How do genetic variants affect the response to nutrient-induced changes in gene expression?\n",
            "  Q5. What are the implications of nutrient-gene interactions for our understanding of disease etiology?\n",
            "\n",
            "  Task4: Investigating the interplay between genetic and environmental factors\n",
            "  Q1. How do environmental factors, such as diet and lifestyle, interact with genetic variants to influence health outcomes?\n",
            "  Q2. What are the roles of epigenetic modifications in mediating the effects of nutrients and environmental factors on gene expression?\n",
            "  Q3. Can genetic variants influence an individual's susceptibility to environmental toxins and pollutants?\n",
            "  Q4. How do dietary patterns and lifestyle factors interact with genetic variants to modulate disease risk?\n",
            "  Q5. What are the implications of gene-environment interactions for the development of personalized medicine approaches?\n",
            "\n",
            "  Task5: Developing personalized nutrition recommendations\n",
            "  Q1. How can genetic information be used to develop tailored dietary recommendations for individuals?\n",
            "  Q2. What are the challenges and limitations of implementing personalized nutrition in clinical practice?\n",
            "  Q3. Can genetic variants in nutrient metabolism genes predict an individual's response to specific dietary interventions?\n",
            "  Q4. How do genetic variants influence the effectiveness of dietary therapies for disease management?\n",
            "  Q5. What are the potential benefits and risks of using genetic information to guide dietary advice?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Applying nutrigenetics in patient care\n",
            "  Q1. How can healthcare professionals use genetic information to inform dietary recommendations for patients?\n",
            "  Q2. What are the potential benefits and limitations of incorporating nutrigenetics into clinical practice?\n",
            "  Q3. Can genetic variants in nutrient metabolism genes predict patient responses to specific dietary interventions?\n",
            "  Q4. How do healthcare professionals communicate complex genetic information to patients?\n",
            "  Q5. What are the implications of nutrigenetics for patient education and counseling?\n",
            "\n",
            "  Task2: Understanding the role of nutrigenetics in disease management\n",
            "  Q1. How do genetic variants influence an individual's response to dietary therapies for disease management?\n",
            "  Q2. What are the potential therapeutic applications of nutrigenetics in the management of chronic diseases?\n",
            "  Q3. Can nutrigenetic insights be used to identify individuals at high risk of developing metabolic disorders?\n",
            "  Q4. How do dietary patterns and lifestyle factors interact with genetic variants to modulate disease risk?\n",
            "  Q5. What are the implications of nutrigenetics for the development of personalized medicine approaches?\n",
            "\n",
            "  Task3: Investigating the effects of nutrients on gene expression\n",
            "  Q1. How do specific nutrients, such as omega-3 fatty acids, affect gene expression and protein synthesis?\n",
            "  Q2. What are the mechanisms by which dietary patterns, such as a Mediterranean diet, influence gene expression and health outcomes?\n",
            "  Q3. Can nutrients and dietary patterns modulate the expression of genes involved in disease susceptibility?\n",
            "  Q4. How do genetic variants affect the response to nutrient-induced changes in gene expression?\n",
            "  Q5. What are the implications of nutrient-gene interactions for our understanding of disease etiology?\n",
            "\n",
            "  Task4: Exploring the interplay between genetic and environmental factors\n",
            "  Q1. How do environmental factors, such as diet and lifestyle, interact with genetic variants to influence health outcomes?\n",
            "  Q2. What are the roles of epigenetic modifications in mediating the effects of nutrients and environmental factors on gene expression?\n",
            "  Q3. Can genetic variants influence an individual's susceptibility to environmental toxins and pollutants?\n",
            "  Q4. How do dietary patterns and lifestyle factors interact with genetic variants to modulate disease risk?\n",
            "  Q5. What are the implications of gene-environment interactions for the development of personalized medicine approaches?\n",
            "\n",
            "  Task5: Staying up-to-date with the latest research\n",
            "  Q1. What are the most recent advances in the field of nutrigenetics and their implications for clinical practice?\n",
            "  Q2. How do healthcare professionals evaluate the quality and relevance of nutrigenetic research for clinical decision-making?\n",
            "  Q3. Can genetic variants influence an individual's response to specific dietary interventions?\n",
            "  Q4. How do healthcare professionals communicate complex genetic information to patients?\n",
            "  Q5. What are the potential benefits and limitations of incorporating nutrigenetics into clinical practice?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understanding personalized nutrition\n",
            "  Q1. How can genetic information be used to develop tailored dietary recommendations?\n",
            "  Q2. What are the benefits and limitations of using genetic information for personalized nutrition?\n",
            "  Q3. Can genetic variants in nutrient metabolism genes predict my response to specific dietary interventions?\n",
            "  Q4. How do genetic variants influence my nutritional needs and dietary requirements?\n",
            "  Q5. What are the implications of personalized nutrition for my health and well-being?\n",
            "\n",
            "  Task2: Exploring the role of genetic testing in nutrition\n",
            "  Q1. What types of genetic tests are available for assessing nutritional needs and dietary responses?\n",
            "  Q2. How do genetic tests for nutrition differ from those used in clinical settings?\n",
            "  Q3. Can genetic testing identify genetic variants that affect my response to specific nutrients?\n",
            "  Q4. How do I interpret the results of genetic tests for nutritional purposes?\n",
            "  Q5. What are the potential benefits and risks of using genetic testing for personalized nutrition?\n",
            "\n",
            "  Task3: Investigating the effects of diet on genetic expression\n",
            "  Q1. How do specific nutrients and dietary patterns affect gene expression and protein synthesis?\n",
            "  Q2. What are the mechanisms by which dietary patterns influence gene expression and health outcomes?\n",
            "  Q3. Can nutrients and dietary patterns modulate the expression of genes involved in disease susceptibility?\n",
            "  Q4. How do genetic variants affect the response to nutrient-induced changes in gene expression?\n",
            "  Q5. What are the implications of nutrient-gene interactions for my health and well-being?\n",
            "\n",
            "  Task4: Understanding gene-environment interactions\n",
            "  Q1. How do environmental factors, such as diet and lifestyle, interact with genetic variants to influence health outcomes?\n",
            "  Q2. What are the roles of epigenetic modifications in mediating the effects of nutrients and environmental factors on gene expression?\n",
            "  Q3. Can genetic variants influence my susceptibility to environmental toxins and pollutants?\n",
            "  Q4. How do dietary patterns and lifestyle factors interact with genetic variants to modulate disease risk?\n",
            "  Q5. What are the implications of gene-environment interactions for my health and well-being?\n",
            "\n",
            "  Task5: Making informed decisions about dietary choices\n",
            "  Q1. How can I use genetic information to inform my dietary choices?\n",
            "  Q2. What are the potential benefits and limitations of using genetic information for dietary decision-making?\n",
            "  Q3. Can genetic variants in nutrient metabolism genes predict my response to specific dietary interventions?\n",
            "  Q4. How do genetic variants influence my nutritional needs and dietary requirements?\n",
            "  Q5. What are the implications of nutrigenetics for my health and well-being?\"\n",
            "}\n",
            "```\n",
            "<<< ----------------- >>>\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "```json\n",
            "{\n",
            "  \"corpus_description\": \"The corpus encompasses various biomedical topics including Crohn's Disease, CAR T cell therapy, and genetic factors influencing obesity. It covers genetic variants, immune responses, clinical manifestations, and therapeutic interventions, providing a comprehensive view of complex biological interactions.\",\n",
            "  \"generated_questions\": \"\n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Exploring genetic influences on nutritional responses\n",
            "  Q1. How do genetic variants associated with Crohn's Disease affect the metabolism of specific nutrients?\n",
            "  Q2. Can genetic variations in metabolic pathways influence an individual's response to certain dietary patterns?\n",
            "  Q3. What is the role of the FTO gene in modulating physiological processes related to obesity and metabolic syndrome?\n",
            "  Q4. How do molecular interactions between nutrients and genetic variants impact disease susceptibility?\n",
            "  Q5. What are the implications of genetic-nutrient interactions on personalized nutrition and disease prevention?\n",
            "\n",
            "  Task2: Investigating nutritional impact on gene expression\n",
            "  Q1. How do specific nutrients or dietary patterns affect gene expression in the context of Crohn's Disease?\n",
            "  Q2. What are the effects of different dietary components on transcript production and protein synthesis in metabolic pathways?\n",
            "  Q3. Can certain nutrients modulate the expression of genes involved in obesity and metabolic syndrome?\n",
            "  Q4. How do environmental factors influence the relationship between nutrition and gene expression?\n",
            "  Q5. What are the downstream effects of nutritional influences on gene expression on physiological processes and disease outcomes?\n",
            "\n",
            "  Task3: Analyzing disease and genetic-nutrient interactions\n",
            "  Q1. How do dietary factors interact with genetic variants to modulate the risk of Crohn's Disease or obesity?\n",
            "  Q2. What is the role of molecular interactions in mediating the effects of nutrition-genetic interactions on disease susceptibility?\n",
            "  Q3. Can nutritional interventions be tailored to specific genetic profiles to prevent or manage disease?\n",
            "  Q4. How do environmental factors alter the interactions between genes and nutrients, influencing disease risk?\n",
            "  Q5. What are the implications of genetic-nutrient interactions on the development of personalized medicine approaches?\n",
            "\n",
            "  Task4: Understanding metabolic pathway insights\n",
            "  Q1. What are the key metabolic pathways involved in mediating the effects of nutrition-genetic interactions on physiological processes?\n",
            "  Q2. How do molecular interactions within metabolic pathways influence disease outcomes?\n",
            "  Q3. Can nutritional interventions target specific metabolic pathways to modulate disease risk?\n",
            "  Q4. What are the roles of genetic variants in metabolic pathways in influencing responses to dietary patterns?\n",
            "  Q5. How do environmental factors impact the functioning of metabolic pathways in the context of nutrition-genetic interactions?\n",
            "\n",
            "  Task5: Investigating environmental and nutrient modulation\n",
            "  Q1. How do environmental factors modulate the interactions between genes and nutrients, influencing health outcomes?\n",
            "  Q2. Can nutritional interventions mitigate the effects of environmental factors on disease susceptibility?\n",
            "  Q3. What are the implications of environmental-nutrient interactions on personalized nutrition and disease prevention?\n",
            "  Q4. How do genetic variants influence an individual's response to environmental-nutrient interactions?\n",
            "  Q5. What are the downstream effects of environmental-nutrient interactions on physiological processes and disease outcomes?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Applying nutrigenetic insights in clinical practice\n",
            "  Q1. How can genetic testing be used to tailor nutritional recommendations for patients with Crohn's Disease?\n",
            "  Q2. What are the clinical implications of genetic-nutrient interactions for disease prevention and management?\n",
            "  Q3. Can nutrigenetic insights inform the development of personalized nutrition plans for patients with obesity?\n",
            "  Q4. How do environmental factors impact the application of nutrigenetic insights in clinical practice?\n",
            "  Q5. What are the potential benefits and limitations of integrating nutrigenetic information into clinical decision-making?\n",
            "\n",
            "  Task2: Understanding disease risk and management\n",
            "  Q1. How do genetic variants associated with Crohn's Disease or obesity influence disease risk and progression?\n",
            "  Q2. What are the roles of nutritional factors in modulating disease risk and management?\n",
            "  Q3. Can nutrigenetic insights be used to identify high-risk individuals for targeted interventions?\n",
            "  Q4. How do molecular interactions between nutrients and genetic variants impact disease susceptibility and progression?\n",
            "  Q5. What are the implications of genetic-nutrient interactions on disease management and treatment outcomes?\n",
            "\n",
            "  Task3: Investigating nutritional influences on gene expression\n",
            "  Q1. How do specific nutrients or dietary patterns affect gene expression in the context of disease?\n",
            "  Q2. What are the effects of different dietary components on transcript production and protein synthesis in metabolic pathways?\n",
            "  Q3. Can certain nutrients modulate the expression of genes involved in disease susceptibility?\n",
            "  Q4. How do environmental factors influence the relationship between nutrition and gene expression?\n",
            "  Q5. What are the downstream effects of nutritional influences on gene expression on physiological processes and disease outcomes?\n",
            "\n",
            "  Task4: Exploring metabolic pathway insights\n",
            "  Q1. What are the key metabolic pathways involved in mediating the effects of nutrition-genetic interactions on physiological processes?\n",
            "  Q2. How do molecular interactions within metabolic pathways influence disease outcomes?\n",
            "  Q3. Can nutritional interventions target specific metabolic pathways to modulate disease risk?\n",
            "  Q4. What are the roles of genetic variants in metabolic pathways in influencing responses to dietary patterns?\n",
            "  Q5. How do environmental factors impact the functioning of metabolic pathways in the context of nutrition-genetic interactions?\n",
            "\n",
            "  Task5: Applying nutrigenetic insights in patient care\n",
            "  Q1. How can healthcare professionals use nutrigenetic information to inform patient care and nutrition counseling?\n",
            "  Q2. What are the potential benefits and limitations of integrating nutrigenetic insights into clinical practice?\n",
            "  Q3. Can nutrigenetic insights be used to develop personalized nutrition plans for patients with complex diseases?\n",
            "  Q4. How do environmental factors impact the application of nutrigenetic insights in patient care?\n",
            "  Q5. What are the implications of genetic-nutrient interactions on patient outcomes and disease management?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understanding personalized nutrition\n",
            "  Q1. How can genetic testing inform personalized nutrition recommendations for optimal health outcomes?\n",
            "  Q2. What are the benefits and limitations of using genetic information to tailor dietary advice?\n",
            "  Q3. Can genetic variants associated with disease susceptibility be used to inform nutritional interventions?\n",
            "  Q4. How do environmental factors impact the effectiveness of personalized nutrition plans?\n",
            "  Q5. What are the implications of genetic-nutrient interactions on individual health outcomes?\n",
            "\n",
            "  Task2: Exploring genetic influences on nutritional responses\n",
            "  Q1. How do genetic variants influence individual responses to specific nutrients or dietary patterns?\n",
            "  Q2. What are the effects of genetic variations on metabolic pathways and physiological processes?\n",
            "  Q3. Can genetic testing be used to identify individuals at risk for certain diseases based on nutritional responses?\n",
            "  Q4. How do molecular interactions between nutrients and genetic variants impact health outcomes?\n",
            "  Q5. What are the implications of genetic-nutrient interactions on personalized nutrition and disease prevention?\n",
            "\n",
            "  Task3: Investigating nutritional impact on gene expression\n",
            "  Q1. How do specific nutrients or dietary patterns affect gene expression in the context of disease?\n",
            "  Q2. What are the effects of different dietary components on transcript production and protein synthesis in metabolic pathways?\n",
            "  Q3. Can certain nutrients modulate the expression of genes involved in disease susceptibility?\n",
            "  Q4. How do environmental factors influence the relationship between nutrition and gene expression?\n",
            "  Q5. What are the downstream effects of nutritional influences on gene expression on physiological processes and disease outcomes?\n",
            "\n",
            "  Task4: Understanding disease risk and management\n",
            "  Q1. How do genetic variants associated with disease influence disease risk and progression?\n",
            "  Q2. What are the roles of nutritional factors in modulating disease risk and management?\n",
            "  Q3. Can nutrigenetic insights be used to identify high-risk individuals for targeted interventions?\n",
            "  Q4. How do molecular interactions between nutrients and genetic variants impact disease susceptibility and progression?\n",
            "  Q5. What are the implications of genetic-nutrient interactions on disease management and treatment outcomes?\n",
            "\n",
            "  Task5: Applying nutrigenetic insights in daily life\n",
            "  Q1. How can individuals use genetic information to inform their nutritional choices and lifestyle habits?\n",
            "  Q2. What are the potential benefits and limitations of integrating nutrigenetic insights into daily life?\n",
            "  Q3. Can nutrigenetic insights be used to develop personalized nutrition plans for optimal health outcomes?\n",
            "  Q4. How do environmental factors impact the application of nutrigenetic insights in daily life?\n",
            "  Q5. What are the implications of genetic-nutrient interactions on individual health outcomes and well-being?\"\n",
            "}\n",
            "```\n",
            "<<< ----------------- >>>\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Here is the output:\n",
            "\n",
            "```\n",
            "{\n",
            "  \"corpus_description\": \"The corpus explores the intricate relationships between genetic variations, lifestyle factors, and health outcomes, including type 2 diabetes, obesity, and other metabolic disorders. It highlights the interplay between genetic predispositions, environmental factors, and nutritional elements in influencing disease susceptibility and health outcomes.\",\n",
            "  \"generated_questions\": \"\n",
            "  \n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Investigating the impact of genetic variants on metabolic pathways\n",
            "  Q1. How do specific genetic variants, such as TCF7L2 and CDKAL1, influence the regulation of metabolic pathways, and what are the downstream effects on glucose metabolism?\n",
            "  Q2. What is the role of molecular interactions between genetic variants and nutrients in modulating physiological processes, such as insulin sensitivity?\n",
            "  Q3. How do dietary patterns, such as fiber intake, interact with genetic variants to affect metabolic markers, like FPG?\n",
            "  Q4. Can genetic variants in the OPRM1 gene influence the response to alcohol consumption and related health outcomes, such as hypertension?\n",
            "  Q5. How do environmental factors, like smoking and physical activity, modulate the interactions between genes and nutrients in influencing health outcomes?\n",
            "\n",
            "  Task2: Exploring the relationship between genetic variants and disease risk\n",
            "  Q1. What is the relationship between genetic variants, such as FTO and LEPR, and obesity risk, and how do dietary factors, like nutrient intake, influence this relationship?\n",
            "  Q2. How do genetic predispositions, such as SNPs associated with BMI, interact with lifestyle factors, like sleep duration, to affect weight management?\n",
            "  Q3. Can genetic variants in the WFS1 gene influence the risk of developing type 2 diabetes, and what is the role of environmental factors, like alcohol consumption?\n",
            "  Q4. What is the impact of genetic variants on the gut microbiome and its relationship to obesity risk?\n",
            "  Q5. How do molecular interactions between genetic variants and nutrients affect disease susceptibility and progression?\n",
            "\n",
            "  Task3: Analyzing the impact of nutrigenetic insights on personalized nutrition\n",
            "  Q1. How can genetic information be used to tailor dietary recommendations for individuals with specific genetic variants, such as TCF7L2?\n",
            "  Q2. What is the potential for nutrigenetic insights to inform the development of targeted interventions for obesity and metabolic disorders?\n",
            "  Q3. Can genetic testing for variants like FTO and LEPR inform personalized nutrition plans for weight management?\n",
            "  Q4. How do environmental factors, like physical activity, influence the effectiveness of nutrigenetic-based interventions?\n",
            "  Q5. What are the implications of nutrigenetic research for public health policy and dietary guidelines?\n",
            "\n",
            "  Task4: Investigating the role of epigenetics in nutrigenetics\n",
            "  Q1. What is the role of epigenetic modifications in mediating the effects of genetic variants on gene expression and protein synthesis?\n",
            "  Q2. How do environmental factors, like diet and lifestyle, influence epigenetic marks and subsequent gene expression?\n",
            "  Q3. Can epigenetic changes contribute to the development of metabolic disorders, such as type 2 diabetes?\n",
            "  Q4. What is the relationship between epigenetic modifications and the gut microbiome in influencing health outcomes?\n",
            "  Q5. How can epigenetic insights inform the development of targeted interventions for metabolic disorders?\n",
            "\n",
            "  Task5: Examining the intersection of nutrigenetics and environmental factors\n",
            "  Q1. How do environmental factors, like air pollution, interact with genetic variants to influence health outcomes?\n",
            "  Q2. What is the impact of lifestyle factors, like smoking, on the interactions between genes and nutrients?\n",
            "  Q3. Can environmental factors, like physical activity, modulate the effects of genetic variants on metabolic pathways?\n",
            "  Q4. How do socioeconomic factors influence access to healthy dietary patterns and subsequent health outcomes?\n",
            "  Q5. What are the implications of nutrigenetic research for environmental health policy and advocacy?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Integrating nutrigenetic insights into clinical practice\n",
            "  Q1. How can healthcare professionals use genetic testing to inform dietary recommendations for patients with specific genetic variants?\n",
            "  Q2. What are the potential benefits and limitations of incorporating nutrigenetic insights into clinical practice?\n",
            "  Q3. Can nutrigenetic testing help identify patients at high risk for metabolic disorders, such as type 2 diabetes?\n",
            "  Q4. How do healthcare professionals communicate complex nutrigenetic information to patients and their families?\n",
            "  Q5. What are the implications of nutrigenetic research for clinical guidelines and treatment protocols?\n",
            "\n",
            "  Task2: Applying nutrigenetic insights to disease prevention\n",
            "  Q1. How can nutrigenetic insights inform the development of targeted interventions for disease prevention?\n",
            "  Q2. What is the role of genetic testing in identifying individuals at high risk for metabolic disorders?\n",
            "  Q3. Can dietary patterns, such as a low-carb diet, be tailored to an individual's genetic profile to improve health outcomes?\n",
            "  Q4. How do environmental factors, like physical activity, influence the effectiveness of nutrigenetic-based interventions?\n",
            "  Q5. What are the potential benefits and limitations of using nutrigenetic insights for disease prevention?\n",
            "\n",
            "  Task3: Exploring the role of nutrigenetics in personalized medicine\n",
            "  Q1. How can nutrigenetic insights be used to tailor medical treatment to an individual's genetic profile?\n",
            "  Q2. What is the potential for nutrigenetic testing to inform personalized nutrition plans for patients with specific genetic variants?\n",
            "  Q3. Can genetic testing for variants like FTO and LEPR inform treatment decisions for obesity and related metabolic disorders?\n",
            "  Q4. How do healthcare professionals integrate nutrigenetic insights into comprehensive treatment plans?\n",
            "  Q5. What are the implications of nutrigenetic research for personalized medicine and targeted interventions?\n",
            "\n",
            "  Task4: Investigating the intersection of nutrigenetics and public health\n",
            "  Q1. What are the implications of nutrigenetic research for public health policy and dietary guidelines?\n",
            "  Q2. How can nutrigenetic insights inform the development of targeted interventions for disease prevention at the population level?\n",
            "  Q3. Can genetic testing for variants like TCF7L2 inform public health initiatives for type 2 diabetes prevention?\n",
            "  Q4. How do socioeconomic factors influence access to healthy dietary patterns and subsequent health outcomes?\n",
            "  Q5. What are the potential benefits and limitations of using nutrigenetic insights for public health initiatives?\n",
            "\n",
            "  Task5: Examining the role of nutrigenetics in health disparities\n",
            "  Q1. How do genetic variants contribute to health disparities in metabolic disorders, such as type 2 diabetes?\n",
            "  Q2. What is the impact of environmental factors, like access to healthy food, on health outcomes in diverse populations?\n",
            "  Q3. Can nutrigenetic insights inform targeted interventions for health disparities in metabolic disorders?\n",
            "  Q4. How do cultural and socioeconomic factors influence the effectiveness of nutrigenetic-based interventions?\n",
            "  Q5. What are the implications of nutrigenetic research for addressing health disparities and promoting health equity?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understanding personalized nutrition\n",
            "  Q1. How can genetic testing inform personalized nutrition plans for individuals with specific genetic variants?\n",
            "  Q2. What are the potential benefits and limitations of using genetic testing for personalized nutrition?\n",
            "  Q3. Can genetic testing for variants like FTO and LEPR inform dietary recommendations for weight management?\n",
            "  Q4. How do environmental factors, like physical activity, influence the effectiveness of personalized nutrition plans?\n",
            "  Q5. What are the implications of nutrigenetic research for personalized nutrition and health outcomes?\n",
            "\n",
            "  Task2: Exploring the role of genetic testing in nutrition\n",
            "  Q1. What is the role of genetic testing in identifying genetic variants that influence nutritional responses?\n",
            "  Q2. How can genetic testing inform dietary recommendations for individuals with specific genetic variants?\n",
            "  Q3. Can genetic testing for variants like TCF7L2 inform dietary recommendations for type 2 diabetes prevention?\n",
            "  Q4. How do healthcare professionals communicate complex genetic information to customers and their families?\n",
            "  Q5. What are the potential benefits and limitations of using genetic testing for nutrition and health outcomes?\n",
            "\n",
            "  Task3: Investigating the intersection of nutrigenetics and lifestyle factors\n",
            "  Q1. How do lifestyle factors, like smoking and physical activity, influence the interactions between genes and nutrients?\n",
            "  Q2. What is the impact of environmental factors, like air pollution, on health outcomes in diverse populations?\n",
            "  Q3. Can nutrigenetic insights inform targeted interventions for lifestyle-related health outcomes?\n",
            "  Q4. How do socioeconomic factors influence access to healthy dietary patterns and subsequent health outcomes?\n",
            "  Q5. What are the implications of nutrigenetic research for promoting healthy lifestyle choices?\n",
            "\n",
            "  Task4: Examining the role of nutrigenetics in disease prevention\n",
            "  Q1. How can nutrigenetic insights inform the development of targeted interventions for disease prevention?\n",
            "  Q2. What is the role of genetic testing in identifying individuals at high risk for metabolic disorders?\n",
            "  Q3. Can dietary patterns, such as a low-carb diet, be tailored to an individual's genetic profile to improve health outcomes?\n",
            "  Q4. How do environmental factors, like physical activity, influence the effectiveness of nutrigenetic-based interventions?\n",
            "  Q5. What are the potential benefits and limitations of using nutrigenetic insights for disease prevention?\n",
            "\n",
            "  Task5: Exploring the future of nutrigenetics and personalized nutrition\n",
            "  Q1. What are the potential applications of nutrigenetic research in precision medicine and targeted interventions?\n",
            "  Q2. How can nutrigenetic insights inform the development of personalized nutrition plans for diverse populations?\n",
            "  Q3. Can genetic testing for variants like FTO and LEPR inform personalized nutrition plans for weight management?\n",
            "  Q4. How do healthcare professionals integrate nutrigenetic insights into comprehensive treatment plans?\n",
            "  Q5. What are the implications of nutrigenetic research for promoting healthy lifestyle choices and disease prevention?\"\n",
            "}\n",
            "```\n",
            "<<< ----------------- >>>\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "```json\n",
            "{\n",
            "  \"corpus_description\": \"The corpus explores the intricate relationships between genetic variants, nutritional elements, and disease outcomes, with a focus on Age-Related Macular Degeneration (AMD), type 2 diabetes (T2D), and diabetes phenotypes. It highlights the influence of specific genes (e.g., ARMS2, CFH, TCF7L2, PPARΓ, CHI3LI, ENPP1, NOTCH2, GCK, TUFM, SULT1A1) and dietary factors (e.g., fiber and fat intake) on disease susceptibility and progression.\",\n",
            "  \"generated_questions\": \"\n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Investigate the role of genetic variants in metabolic pathways related to nutrient response.\n",
            "  Q1. How do genetic variants in the TCF7L2 gene influence an individual's response to dietary fiber in relation to T2D risk?\n",
            "  Q2. What is the impact of ARMS2 genetic variants on the metabolic pathway of fatty acid oxidation in the context of AMD?\n",
            "  Q3. How do variations in the PPARΓ gene affect protein synthesis in response to dietary fat intake?\n",
            "  Q4. Can specific genetic variants in the GCK gene modulate the physiological process of glucose metabolism in response to dietary patterns?\n",
            "  Q5. How do genetic variants in CHI3LI influence molecular interactions with nutrients to affect T2D susceptibility?\n",
            "\n",
            "  Task2: Examine the influence of nutrients on gene expression and disease risk.\n",
            "  Q1. What are the effects of omega-3 fatty acids on gene expression of inflammatory markers in the context of AMD risk?\n",
            "  Q2. How does dietary fiber intake influence the expression of genes involved in glucose metabolism, such as SULT1A1?\n",
            "  Q3. In what ways do antioxidants in food affect the expression of CFH and ARMS2 in relation to AMD progression?\n",
            "  Q4. Can certain amino acids in the diet impact the expression of ENPP1 and NOTCH2 genes in T2D?\n",
            "  Q5. How do polyphenols in food influence TUFM gene expression and diabetes phenotypes?\n",
            "\n",
            "  Task3: Explore genetic-nutrient interactions in disease progression.\n",
            "  Q1. How do interactions between genetic variants of TCF7L2 and dietary carbohydrates influence T2D progression?\n",
            "  Q2. What is the role of nutrient-gene interactions in modulating the risk of AMD, specifically with ARMS2 and CFH variants?\n",
            "  Q3. Can specific dietary patterns mitigate or exacerbate the risk of T2D in individuals with PPARΓ genetic variants?\n",
            "  Q4. How do genetic variants in GCK interact with dietary factors to influence diabetes phenotypes?\n",
            "  Q5. In what ways do environmental factors, like smoking, alter the nutrient-gene interactions affecting CHI3LI expression and T2D risk?\n",
            "\n",
            "  Task4: Understand the role of metabolic pathways in mediating nutrition-genetic interactions.\n",
            "  Q1. How do metabolic pathways involving fatty acid oxidation mediate the effects of genetic variants on T2D risk?\n",
            "  Q2. What role do insulin signaling pathways play in mediating the interaction between dietary factors and genetic variants in TCF7L2?\n",
            "  Q3. Can alterations in the metabolic pathway of glucose metabolism explain the interaction between GCK genetic variants and dietary carbohydrates?\n",
            "  Q4. How do molecular interactions within the complement pathway mediate the effects of ARMS2 and CFH variants on AMD risk?\n",
            "  Q5. In what ways do variations in metabolic pathways of lipid metabolism influence the response to dietary fats in individuals with PPARΓ variants?\n",
            "\n",
            "  Task5: Investigate the modulation of genetic-nutrient interactions by environmental factors.\n",
            "  Q1. How do environmental factors like physical activity level modulate the interaction between genetic variants and dietary patterns in T2D risk?\n",
            "  Q2. Can exposure to pollutants alter the nutrient-gene interactions affecting CHI3LI expression and T2D susceptibility?\n",
            "  Q3. What is the impact of stress on the interaction between genetic variants in CFH and dietary antioxidants in AMD progression?\n",
            "  Q4. How do socioeconomic factors influence access to nutrients that interact with genetic variants to affect disease outcomes?\n",
            "  Q5. In what ways do lifestyle factors, such as sleep quality, modulate the genetic-nutrient interactions affecting GCK expression and diabetes phenotypes?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Apply nutrigenetic knowledge to predict disease risk.\n",
            "  Q1. How can genetic testing for TCF7L2 variants inform dietary recommendations for T2D prevention?\n",
            "  Q2. What is the utility of genetic screening for ARMS2 and CFH in predicting AMD risk and guiding nutritional interventions?\n",
            "  Q3. Can genetic information on PPARΓ variants guide personalized dietary fat intake for T2D management?\n",
            "  Q4. How can knowledge of GCK genetic variants be used to tailor dietary advice for diabetes management?\n",
            "  Q5. What role can genetic testing for CHI3LI play in assessing T2D risk and developing targeted nutritional strategies?\n",
            "\n",
            "  Task2: Explore the practical application of nutrigenetics in patient care.\n",
            "  Q1. How can healthcare providers integrate genetic information into dietary counseling for patients with T2D?\n",
            "  Q2. What are the implications of genetic variants in nutrigenetic testing for personalized nutrition recommendations in AMD patients?\n",
            "  Q3. Can understanding genetic variants in CHI3LI inform targeted dietary interventions for T2D prevention in high-risk populations?\n",
            "  Q4. How do healthcare professionals stay updated with the evolving field of nutrigenetics to provide evidence-based dietary advice?\n",
            "  Q5. What are the ethical considerations in applying genetic information to provide personalized dietary recommendations?\n",
            "\n",
            "  Task3: Investigate the role of nutrigenetics in disease prevention.\n",
            "  Q1. How can early identification of genetic risk factors for T2D through nutrigenetic testing guide preventive dietary strategies?\n",
            "  Q2. Can nutrigenetic insights into ARMS2 and CFH variants contribute to preventive measures for AMD?\n",
            "  Q3. What is the potential of nutrigenetic testing for GCK in preventing diabetes phenotypes through targeted dietary interventions?\n",
            "  Q4. How do dietary patterns interact with genetic variants to prevent or delay the onset of T2D?\n",
            "  Q5. Can understanding the interaction between environmental factors and genetic variants inform preventive nutritional strategies for diseases like AMD and T2D?\n",
            "\n",
            "  Task4: Examine the integration of nutrigenetics into clinical guidelines.\n",
            "  Q1. How can nutrigenetic information be incorporated into clinical guidelines for T2D management?\n",
            "  Q2. What are the challenges in integrating genetic testing for AMD risk into clinical practice?\n",
            "  Q3. Can existing clinical guidelines for diabetes be improved with the inclusion of nutrigenetic testing for CHI3LI?\n",
            "  Q4. How do healthcare systems support the implementation of nutrigenetic testing for disease prevention and management?\n",
            "  Q5. What are the implications of nutrigenetic research findings for updating clinical guidelines on dietary management of T2D and AMD?\n",
            "\n",
            "  Task5: Evaluate the potential of nutrigenetics for personalized medicine.\n",
            "  Q1. How can nutrigenetic testing contribute to the development of personalized dietary plans for disease prevention?\n",
            "  Q2. What is the potential of integrating genetic information into nutritional therapy for T2D?\n",
            "  Q3. Can personalized nutrition based on genetic variants in ARMS2 and CFH improve AMD outcomes?\n",
            "  Q4. How do genetic variants in CHI3LI and PPARΓ influence the response to different types of dietary interventions?\n",
            "  Q5. What are the future directions for research in applying nutrigenetics to personalized medicine?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understand genetic influences on dietary responses.\n",
            "  Q1. How do genetic variants affect my response to dietary carbohydrates in terms of energy levels and weight management?\n",
            "  Q2. Can genetic testing reveal why some people benefit more from a low-fat diet than others?\n",
            "  Q3. How can I use genetic information to optimize my protein intake for better muscle synthesis?\n",
            "  Q4. What genetic factors might influence my sensitivity to caffeine and other dietary stimulants?\n",
            "  Q5. Can genetic information help tailor my dietary fiber intake for better digestive health?\n",
            "\n",
            "  Task2: Explore the application of genetic information in nutrition.\n",
            "  Q1. How can I apply genetic testing to determine the best dietary approach for my genetic predisposition to T2D?\n",
            "  Q2. What are the practical implications of having a genetic variant associated with AMD for my dietary choices?\n",
            "  Q3. Can genetic information guide me in choosing the right type of dietary fats for heart health?\n",
            "  Q4. How do I integrate genetic insights into my current dietary habits for improved health outcomes?\n",
            "  Q5. What are the limitations and benefits of using genetic information to inform my dietary decisions?\n",
            "\n",
            "  Task3: Investigate the interaction between genetics, diet, and disease risk.\n",
            "  Q1. How does my genetic risk for T2D interact with my dietary choices to influence disease onset?\n",
            "  Q2. Can understanding my genetic predisposition to AMD guide my antioxidant intake?\n",
            "  Q3. How do genetic factors influence my risk of metabolic syndrome in response to dietary patterns?\n",
            "  Q4. What role do genetic variants play in my response to dietary interventions for weight loss?\n",
            "  Q5. Can genetic testing help identify nutritional strategies to mitigate my risk of chronic diseases?\n",
            "\n",
            "  Task4: Understand the role of environmental factors in genetic-nutrient interactions.\n",
            "  Q1. How do environmental factors like pollution affect the interaction between my genetic makeup and dietary responses?\n",
            "  Q2. Can lifestyle factors such as exercise modify the genetic risk associated with certain dietary patterns?\n",
            "  Q3. How does stress influence the interaction between genetic variants and nutrient metabolism?\n",
            "  Q4. What is the impact of socioeconomic factors on access to nutrients that interact with my genetic background?\n",
            "  Q5. Can understanding the interaction between environmental factors and genetics inform my decisions on dietary supplements?\n",
            "\n",
            "  Task5: Apply nutrigenetic insights to improve health outcomes.\n",
            "  Q1. How can I use genetic information to improve my health outcomes through targeted dietary changes?\n",
            "  Q2. Can personalized nutrition based on genetic testing enhance the effectiveness of my current dietary regimen?\n",
            "  Q3. What are the potential health benefits of tailoring my diet based on genetic insights into nutrient metabolism?\n",
            "  Q4. How do I prioritize dietary changes based on my genetic risk for certain diseases?\n",
            "  Q5. Can applying nutrigenetic principles lead to better management of chronic diseases through diet?\"\n",
            "}\n",
            "```\n",
            "<<< ----------------- >>>\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Here is the output:\n",
            "\n",
            "```\n",
            "{\n",
            "  \"corpus_description\": \"The corpus explores the relationships between genes, proteins, and metabolic processes influencing various diseases and conditions, including kidney function, glucose regulation, immune system components, inflammatory mediators, and lipid metabolism.\",\n",
            "  \"generated_questions\": \"\n",
            "  \n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Investigating the impact of genetic variants on metabolic pathways\n",
            "  Q1. How do specific genetic variants in genes like GCKR and ALMS1 influence individual responses to dietary patterns, such as carbohydrate intake, and affect kidney function?\n",
            "  Q2. What are the effects of genetic variants on the regulation of metabolic pathways, such as those involved in glucose regulation and lipid metabolism?\n",
            "  Q3. How do genetic variants in immune system components, such as cytokines and transcription factors, impact the response to dietary factors and influence disease risk?\n",
            "  Q4. What is the relationship between genetic variants, nutrient sensing, and metabolic processes, and how do these interactions affect physiological outcomes?\n",
            "  Q5. How do genetic variants in genes involved in lipid metabolism, such as LPL, influence the response to dietary fatty acids and impact disease risk?\n",
            "\n",
            "  Task2: Exploring the role of nutrients in gene expression\n",
            "  Q1. How do specific nutrients, such as omega-3 fatty acids, influence gene expression and protein synthesis in the context of kidney function and glucose regulation?\n",
            "  Q2. What are the effects of dietary patterns, such as a high-fat diet, on gene expression and metabolic processes involved in immune system function?\n",
            "  Q3. How do nutrients, such as vitamins and minerals, impact the regulation of metabolic pathways and physiological processes?\n",
            "  Q4. What is the relationship between nutrient sensing, gene expression, and disease risk, and how do genetic variants influence this relationship?\n",
            "  Q5. How do dietary patterns, such as a Mediterranean diet, affect gene expression and protein synthesis in the context of lipid metabolism and disease risk?\n",
            "\n",
            "  Task3: Analyzing the relationship between genetic variants and disease risk\n",
            "  Q1. How do genetic variants in genes like NFATC1 and HDAC8 influence disease risk, and what is the role of dietary factors in modulating this risk?\n",
            "  Q2. What are the effects of genetic variants on the progression of diseases like diabetes and inflammatory bowel disease (IBD)?\n",
            "  Q3. How do genetic variants in immune system components impact the response to environmental factors and influence disease risk?\n",
            "  Q4. What is the relationship between genetic variants, metabolic pathways, and disease outcomes, and how do dietary factors influence this relationship?\n",
            "  Q5. How do genetic variants in genes involved in lipid metabolism impact the risk of conditions like hypertriglyceridemia and pancreatitis?\n",
            "\n",
            "  Task4: Investigating the impact of environmental factors on genetic-nutrient interactions\n",
            "  Q1. How do environmental factors, such as exposure to toxins, alter the interactions between genes and nutrients, influencing health outcomes and disease susceptibility?\n",
            "  Q2. What are the effects of lifestyle factors, such as physical activity, on genetic-nutrient interactions and disease risk?\n",
            "  Q3. How do environmental factors, such as diet and stress, impact the regulation of metabolic pathways and physiological processes?\n",
            "  Q4. What is the relationship between environmental factors, genetic variants, and nutrient sensing, and how do these interactions affect disease risk?\n",
            "  Q5. How do environmental factors, such as air pollution, influence the interactions between genes and nutrients, and impact health outcomes?\n",
            "\n",
            "  Task5: Developing personalized nutrition recommendations\n",
            "  Q1. How can genetic information be used to tailor dietary recommendations to an individual's specific genetic profile?\n",
            "  Q2. What are the implications of genetic variants on nutrient requirements and dietary recommendations?\n",
            "  Q3. How can nutrigenetic insights be integrated into clinical practice to improve patient outcomes?\n",
            "  Q4. What are the challenges and limitations of using genetic information to inform dietary recommendations?\n",
            "  Q5. How can genetic testing and nutrigenetic analysis be used to prevent or manage chronic diseases?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Understanding the genetic basis of disease\n",
            "  Q1. What are the key genetic variants associated with diseases like diabetes and kidney disease, and how do dietary factors influence disease risk?\n",
            "  Q2. How do genetic variants in immune system components impact disease susceptibility and progression?\n",
            "  Q3. What are the effects of genetic variants on metabolic pathways and physiological processes involved in lipid metabolism?\n",
            "  Q4. How do genetic variants in genes involved in glucose regulation impact disease risk and progression?\n",
            "  Q5. What are the implications of genetic variants on treatment strategies for diseases like hypertriglyceridemia?\n",
            "\n",
            "  Task2: Applying nutrigenetic insights in clinical practice\n",
            "  Q1. How can healthcare professionals use genetic information to inform dietary recommendations and improve patient outcomes?\n",
            "  Q2. What are the challenges and limitations of integrating nutrigenetic analysis into clinical practice?\n",
            "  Q3. How can healthcare professionals stay up-to-date with the latest research and developments in nutrigenetics?\n",
            "  Q4. What are the potential benefits and risks of using genetic testing and nutrigenetic analysis in clinical practice?\n",
            "  Q5. How can healthcare professionals communicate effectively with patients about the implications of genetic testing and nutrigenetic analysis?\n",
            "\n",
            "  Task3: Investigating the role of nutrients in disease prevention\n",
            "  Q1. What are the effects of specific nutrients, such as omega-3 fatty acids, on disease prevention and progression?\n",
            "  Q2. How do dietary patterns, such as a Mediterranean diet, impact disease risk and progression?\n",
            "  Q3. What are the implications of nutrient deficiencies on disease risk and progression?\n",
            "  Q4. How do nutrients, such as vitamins and minerals, impact the regulation of metabolic pathways and physiological processes?\n",
            "  Q5. What are the potential benefits and risks of using nutrient supplements for disease prevention and treatment?\n",
            "\n",
            "  Task4: Analyzing the relationship between genetic variants and environmental factors\n",
            "  Q1. How do environmental factors, such as exposure to toxins, alter the interactions between genes and nutrients, influencing health outcomes and disease susceptibility?\n",
            "  Q2. What are the effects of lifestyle factors, such as physical activity, on genetic-nutrient interactions and disease risk?\n",
            "  Q3. How do environmental factors, such as diet and stress, impact the regulation of metabolic pathways and physiological processes?\n",
            "  Q4. What is the relationship between environmental factors, genetic variants, and nutrient sensing, and how do these interactions affect disease risk?\n",
            "  Q5. How do environmental factors, such as air pollution, influence the interactions between genes and nutrients, and impact health outcomes?\n",
            "\n",
            "  Task5: Developing personalized treatment strategies\n",
            "  Q1. How can healthcare professionals use genetic information and nutrigenetic analysis to develop personalized treatment strategies?\n",
            "  Q2. What are the implications of genetic variants on treatment outcomes and disease progression?\n",
            "  Q3. How can healthcare professionals integrate nutrigenetic insights into treatment plans for patients with chronic diseases?\n",
            "  Q4. What are the potential benefits and risks of using genetic testing and nutrigenetic analysis to guide treatment decisions?\n",
            "  Q5. How can healthcare professionals communicate effectively with patients about the implications of genetic testing and nutrigenetic analysis for treatment decisions?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understanding the implications of genetic testing\n",
            "  Q1. What are the implications of genetic testing for my health and disease risk, and how can I use this information to inform my dietary choices?\n",
            "  Q2. How do genetic variants in genes like GCKR and ALMS1 impact my response to dietary patterns and nutrients?\n",
            "  Q3. What are the potential benefits and risks of using genetic testing and nutrigenetic analysis for personalized nutrition?\n",
            "  Q4. How can I use genetic information to optimize my nutrient intake and dietary patterns for better health outcomes?\n",
            "  Q5. What are the limitations and challenges of using genetic testing and nutrigenetic analysis for personalized nutrition?\n",
            "\n",
            "  Task2: Exploring the role of nutrients in health and disease\n",
            "  Q1. What are the effects of specific nutrients, such as omega-3 fatty acids, on health and disease risk?\n",
            "  Q2. How do dietary patterns, such as a Mediterranean diet, impact health outcomes and disease risk?\n",
            "  Q3. What are the implications of nutrient deficiencies on health and disease risk?\n",
            "  Q4. How do nutrients, such as vitamins and minerals, impact the regulation of metabolic pathways and physiological processes?\n",
            "  Q5. What are the potential benefits and risks of using nutrient supplements for health and disease prevention?\n",
            "\n",
            "  Task3: Investigating the relationship between genetic variants and nutrient responses\n",
            "  Q1. How do genetic variants in genes like LPL impact my response to dietary fatty acids and nutrients?\n",
            "  Q2. What are the effects of genetic variants on the regulation of metabolic pathways and physiological processes involved in lipid metabolism?\n",
            "  Q3. How do genetic variants in immune system components impact my response to dietary factors and disease risk?\n",
            "  Q4. What is the relationship between genetic variants, nutrient sensing, and metabolic processes, and how do these interactions affect health outcomes?\n",
            "  Q5. How do environmental factors, such as exposure to toxins, alter the interactions between genes and nutrients, influencing health outcomes and disease susceptibility?\n",
            "\n",
            "  Task4: Developing a personalized nutrition plan\n",
            "  Q1. How can I use genetic information and nutrigenetic analysis to develop a personalized nutrition plan?\n",
            "  Q2. What are the implications of genetic variants on nutrient requirements and dietary recommendations?\n",
            "  Q3. How can I integrate nutrigenetic insights into my daily dietary choices to optimize health outcomes?\n",
            "  Q4. What are the potential benefits and risks of using genetic testing and nutrigenetic analysis for personalized nutrition?\n",
            "  Q5. How can I communicate effectively with healthcare professionals about the implications of genetic testing and nutrigenetic analysis for personalized nutrition?\n",
            "\n",
            "  Task5: Staying up-to-date with the latest research and developments\n",
            "  Q1. How can I stay current with the latest research and developments in nutrigenetics and personalized nutrition?\n",
            "  Q2. What are the key challenges and limitations of using genetic testing and nutrigenetic analysis for personalized nutrition?\n",
            "  Q3. How can I evaluate the credibility and reliability of sources of information on nutrigenetics and personalized nutrition?\n",
            "  Q4. What are the potential applications and implications of nutrigenetic analysis for public health and disease prevention?\n",
            "  Q5. How can I contribute to the advancement of research and knowledge in the field of nutrigenetics and personalized nutrition?\"\n",
            "}\n",
            "```\n",
            "<<< ----------------- >>>\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:httpx:HTTP Request: POST https://api.groq.com/openai/v1/chat/completions \"HTTP/1.1 200 OK\"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "```json\n",
            "{\n",
            "  \"corpus_description\": \"The corpus explores the complex interactions between genetic variants, nutritional elements, and environmental factors in influencing health and disease outcomes, with a focus on autoimmune diseases, asthma, and cancer.\",\n",
            "  \"generated_questions\": \"\n",
            "  \n",
            "  User1: A researcher in nutrigenetics, interested in understanding how genetic variants influence dietary responses.\n",
            "  Task1: Investigating genetic influences on nutritional responses\n",
            "  Q1. How do specific genetic variants, such as those in the PTPN2 gene, affect an individual's response to gluten in the context of Celiac Disease?\n",
            "  Q2. What are the implications of genetic variations in metabolic pathways on the efficacy of dietary interventions for asthma?\n",
            "  Q3. How do genetic variants influence the expression of proteins involved in nutrient uptake and metabolism?\n",
            "  Q4. Can genetic testing identify individuals with a higher risk of adverse reactions to certain dietary components?\n",
            "  Q5. How do genetic variants in genes like HLA-DRB1*07:01 modulate the immune response to environmental triggers?\n",
            "\n",
            "  Task2: Analyzing nutritional impact on gene expression\n",
            "  Q1. What nutrients or dietary patterns can alter the expression of genes involved in inflammatory responses, such as IL-1A and TNF-A?\n",
            "  Q2. How do specific dietary components influence the production of transcripts and proteins involved in metabolic pathways?\n",
            "  Q3. What are the effects of different dietary patterns on gene expression in the context of autoimmune diseases like Crohn's Disease?\n",
            "  Q4. Can certain nutrients act as epigenetic modifiers, influencing gene expression and disease susceptibility?\n",
            "  Q5. How do dietary factors interact with genetic variants to affect the risk of developing asthma?\n",
            "\n",
            "  Task3: Exploring disease and genetic-nutrient interactions\n",
            "  Q1. How do genetic variants and dietary factors interact to modulate the risk of developing Celiac Disease?\n",
            "  Q2. What are the implications of genetic-nutrient interactions for the progression of autoimmune diseases like Crohn's Disease?\n",
            "  Q3. Can dietary interventions be tailored to mitigate the effects of genetic variants on disease susceptibility?\n",
            "  Q4. How do environmental factors, such as smoking status, influence the interaction between genetic variants and dietary factors in asthma risk?\n",
            "  Q5. What are the potential therapeutic applications of understanding genetic-nutrient interactions in disease management?\n",
            "\n",
            "  Task4: Investigating metabolic pathway insights\n",
            "  Q1. What roles do metabolic pathways play in mediating the effects of nutrition-genetic interactions on physiological processes?\n",
            "  Q2. How do genetic variants in metabolic pathways affect the response to dietary interventions?\n",
            "  Q3. Can molecular interactions between nutrients and genetic variants be targeted for therapeutic interventions?\n",
            "  Q4. What are the implications of genetic variations in metabolic pathways for the development of personalized nutrition strategies?\n",
            "  Q5. How do environmental factors influence the activity of metabolic pathways involved in nutrient metabolism?\n",
            "\n",
            "  Task5: Examining environmental and nutrient modulation\n",
            "  Q1. How do environmental factors, such as exposure to pollutants, alter the interactions between genes and nutrients?\n",
            "  Q2. What are the effects of lifestyle factors, such as smoking status, on the interaction between genetic variants and dietary factors in disease risk?\n",
            "  Q3. Can environmental factors modulate the expression of genes involved in nutrient metabolism?\n",
            "  Q4. How do genetic variants influence an individual's susceptibility to environmental toxins and nutrients?\n",
            "  Q5. What are the implications of environmental and nutrient modulation for the development of preventative strategies against chronic diseases?\n",
            "\n",
            "  User2: A healthcare professional looking to apply nutrigenetic insights in clinical practice.\n",
            "  Task1: Applying genetic insights to clinical practice\n",
            "  Q1. How can genetic testing be used to tailor dietary recommendations for patients with a high risk of Celiac Disease?\n",
            "  Q2. What are the clinical implications of genetic variations in genes like PTPN2 for the management of autoimmune diseases?\n",
            "  Q3. How can healthcare professionals integrate genetic information into personalized nutrition plans for patients with asthma?\n",
            "  Q4. What are the potential benefits and limitations of using genetic testing to guide dietary interventions in clinical practice?\n",
            "  Q5. How can healthcare professionals stay up-to-date with the latest research on genetic-nutrient interactions and its applications in clinical practice?\n",
            "\n",
            "  Task2: Understanding disease risk and progression\n",
            "  Q1. How do genetic variants and dietary factors interact to modulate the risk of developing autoimmune diseases like Crohn's Disease?\n",
            "  Q2. What are the implications of genetic-nutrient interactions for the progression of asthma?\n",
            "  Q3. Can dietary interventions be used to mitigate the effects of genetic variants on disease susceptibility?\n",
            "  Q4. How do environmental factors influence the interaction between genetic variants and dietary factors in disease risk?\n",
            "  Q5. What are the potential therapeutic applications of understanding genetic-nutrient interactions in disease management?\n",
            "\n",
            "  Task3: Investigating molecular mechanisms\n",
            "  Q1. What are the molecular mechanisms underlying the interaction between genetic variants and dietary factors in autoimmune diseases?\n",
            "  Q2. How do genetic variants affect the expression of proteins involved in nutrient uptake and metabolism?\n",
            "  Q3. Can molecular interactions between nutrients and genetic variants be targeted for therapeutic interventions?\n",
            "  Q4. What are the implications of genetic variations in metabolic pathways for the development of personalized nutrition strategies?\n",
            "  Q5. How do environmental factors influence the activity of metabolic pathways involved in nutrient metabolism?\n",
            "\n",
            "  Task4: Developing personalized nutrition strategies\n",
            "  Q1. How can genetic testing be used to tailor dietary recommendations for patients with a high risk of chronic diseases?\n",
            "  Q2. What are the benefits and limitations of using genetic information to guide dietary interventions in clinical practice?\n",
            "  Q3. Can dietary interventions be tailored to mitigate the effects of genetic variants on disease susceptibility?\n",
            "  Q4. How do genetic variants influence an individual's response to different dietary patterns?\n",
            "  Q5. What are the implications of genetic-nutrient interactions for the development of personalized nutrition strategies?\n",
            "\n",
            "  Task5: Exploring environmental influences\n",
            "  Q1. How do environmental factors, such as exposure to pollutants, alter the interactions between genes and nutrients?\n",
            "  Q2. What are the effects of lifestyle factors, such as smoking status, on the interaction between genetic variants and dietary factors in disease risk?\n",
            "  Q3. Can environmental factors modulate the expression of genes involved in nutrient metabolism?\n",
            "  Q4. How do genetic variants influence an individual's susceptibility to environmental toxins and nutrients?\n",
            "  Q5. What are the implications of environmental and nutrient modulation for the development of preventative strategies against chronic diseases?\n",
            "\n",
            "  User3: A customer interested in personalized nutrition based on genetic information.\n",
            "  Task1: Understanding personal genetic information\n",
            "  Q1. What do my genetic test results reveal about my risk of developing certain diseases?\n",
            "  Q2. How can I use my genetic information to make informed dietary choices?\n",
            "  Q3. What are the implications of my genetic variants for my nutritional needs and dietary recommendations?\n",
            "  Q4. Can I use genetic testing to identify genetic variants that affect my response to certain nutrients?\n",
            "  Q5. How can I incorporate genetic information into my daily dietary choices?\n",
            "\n",
            "  Task2: Exploring dietary implications\n",
            "  Q1. What dietary patterns or nutrients can help mitigate the effects of my genetic variants on disease susceptibility?\n",
            "  Q2. How can I tailor my diet to my genetic profile to optimize health outcomes?\n",
            "  Q3. What are the benefits and limitations of using genetic testing to guide dietary interventions?\n",
            "  Q4. Can I use genetic information to identify potential food sensitivities or intolerances?\n",
            "  Q5. How can I use genetic testing to monitor the effectiveness of dietary interventions?\n",
            "\n",
            "  Task3: Investigating gene-diet interactions\n",
            "  Q1. How do genetic variants influence my response to different dietary patterns?\n",
            "  Q2. What are the implications of genetic-nutrient interactions for my health and disease risk?\n",
            "  Q3. Can genetic testing identify genetic variants that affect my nutrient metabolism?\n",
            "  Q4. How do environmental factors influence the interaction between my genetic variants and dietary factors?\n",
            "  Q5. What are the potential therapeutic applications of understanding genetic-nutrient interactions for my health?\n",
            "\n",
            "  Task4: Understanding disease risk and prevention\n",
            "  Q1. What are the implications of my genetic test results for my risk of developing certain diseases?\n",
            "  Q2. How can I use genetic information to develop a preventative strategy against chronic diseases?\n",
            "  Q3. What are the benefits and limitations of using genetic testing to guide disease prevention strategies?\n",
            "  Q4. Can genetic testing identify genetic variants that affect my susceptibility to environmental toxins?\n",
            "  Q5. How can I use genetic information to monitor my disease risk and adjust my lifestyle accordingly?\n",
            "\n",
            "  Task5: Exploring nutrigenetic applications\n",
            "  Q1. What are the potential applications of nutrigenetics in personalized medicine?\n",
            "  Q2. How can I use genetic testing to guide my nutritional choices and optimize health outcomes?\n",
            "  Q3. What are the implications of genetic-nutrient interactions for my health and disease risk?\n",
            "  Q4. Can genetic testing identify genetic variants that affect my response to certain nutrients or dietary patterns?\n",
            "  Q5. How can I stay up-to-date with the latest research on genetic-nutrient interactions and its applications in personalized nutrition?\"\n",
            "}\n",
            "```\n",
            "<<< ----------------- >>>\n"
          ]
        }
      ],
      "source": [
        "K = 3\n",
        "N = 5\n",
        "M = 5\n",
        "# working_dir = \"/content/drive/MyDrive/workshop-ECAI/\"\n",
        "working_dir = os.getcwd() + \"/documents_from_working_dataset/\"\n",
        "\n",
        "answers =[]\n",
        "for i, batch in enumerate(batches):\n",
        "  for i, chunk in enumerate(batch):\n",
        "    text = \"\\n\\n\".join(chunk)\n",
        "\n",
        "  corpus_description = \"\"\"**Corpus Description**\\n\\n\"\"\"+text\n",
        "\n",
        "  corpus_description = batch\n",
        "\n",
        "  prompt = generate_questions(corpus_description, K, N, M)\n",
        "\n",
        "  chat_completion = client.chat.completions.create(\n",
        "      messages=[\n",
        "          {\n",
        "              \"role\": \"user\",\n",
        "              \"content\": prompt\n",
        "          }\n",
        "      ],\n",
        "      model=MODEL,\n",
        "  )\n",
        "\n",
        "  answer = chat_completion.choices[0].message.content\n",
        "\n",
        "  answers.append(answer)\n",
        "  print(answer)\n",
        "  print(\"<<< ----------------- >>>\")\n",
        "\n",
        "  # question_dir = os.getcwd() + \"/questions_files/\"\n",
        "  # with open(question_dir+f'{labels[i]}_generated_question.txt', 'w') as f:\n",
        "  #   f.write(answer)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 27,
      "metadata": {},
      "outputs": [],
      "source": [
        "import re\n",
        "import re\n",
        "text = \"\"\" \"\"\"\n",
        "\n",
        "# Extract User and Task information and question block (as a list ) from the input\n",
        "def extract_user_tasks(input_text):\n",
        "    user_tasks = []\n",
        "    lines = input_text.strip().split('\\n')\n",
        "    current_user = None\n",
        "    current_task = None\n",
        "    questions = []\n",
        "\n",
        "    for line in lines:\n",
        "        line = line.strip()\n",
        "        if line.startswith('User'):\n",
        "            if current_user and current_task and questions:\n",
        "                user_tasks.append((current_user, current_task, questions))\n",
        "            # Esxtract the User code (User1, User2, etc.)\n",
        "            current_user = line.split(':')[0].strip()\n",
        "            current_task = None\n",
        "            questions = []\n",
        "        elif line.startswith('Task'):\n",
        "            if current_task and questions:\n",
        "                user_tasks.append((current_user, current_task, questions))\n",
        "            current_task = line.split(':')[1].strip()\n",
        "            questions = []\n",
        "        elif line.startswith('Q'):\n",
        "            questions.append(line)\n",
        "\n",
        "    if current_user and current_task and questions:\n",
        "        user_tasks.append((current_user, current_task, questions))\n",
        "\n",
        "    return user_tasks\n",
        "extract_user_tasks(text)\n",
        "import pandas as pd\n",
        "df = pd.DataFrame(extract_user_tasks(text), columns=['User', 'Task', 'Questions'])\n",
        "df.to_pickle(f'questions_cache_gemma2_all-mpnet-base-v2.pkl')\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 25,
      "metadata": {},
      "outputs": [
        {
          "data": {
            "application/vnd.microsoft.datawrangler.viewer.v0+json": {
              "columns": [
                {
                  "name": "index",
                  "rawType": "int64",
                  "type": "integer"
                },
                {
                  "name": "User",
                  "rawType": "object",
                  "type": "string"
                },
                {
                  "name": "Task",
                  "rawType": "object",
                  "type": "string"
                },
                {
                  "name": "Questions",
                  "rawType": "object",
                  "type": "string"
                }
              ],
              "ref": "c9a0364d-05f7-453c-939e-dc2b36e1cb2e",
              "rows": [],
              "shape": {
                "columns": 3,
                "rows": 0
              }
            },
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>User</th>\n",
              "      <th>Task</th>\n",
              "      <th>Questions</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "Empty DataFrame\n",
              "Columns: [User, Task, Questions]\n",
              "Index: []"
            ]
          },
          "execution_count": 25,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "df"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {},
      "outputs": [],
      "source": []
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {},
      "outputs": [],
      "source": []
    }
  ],
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "display_name": "base",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.12.9"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}
