var graphJson = {"directed": false, "multigraph": false, "graph": {"node_default": {}, "edge_default": {}}, "nodes": [{"entity_type": "DEMOGRAPHIC", "description": "Men were older than women, while the obese women were older than the nonobese women.", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "id": "GENDER"}, {"entity_type": "DISEASE", "description": "Obesity risk is one of the health outcomes studied in relation to the presence of the FTO SNP and education level.<SEP>Obesity was associated more prevalently in carriers of A-allele compared to TT subjects.<SEP>The participants in the obese group had higher BMIs (approximately 27.1 kg/m2) and waist circumferences compared to those in the nonobese group.<SEP>The study focused on the relationship between PRS, dietary patterns, and obesity risk.<SEP>rs182549 is associated with obesity.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964<SEP>chunk-fa6293a0a10f175b4aebac8e885c0d20<SEP>chunk-63d0b5be485a69f44e2799ccc9510487<SEP>chunk-3ad8ef794841d749e48b3223741e91d3<SEP>chunk-3135befd0c3140ad097d96ace6de86a1", "id": "OBESITY"}, {"entity_type": "NON-OBESITY", "description": "The nonobese participants had lower BMI\'s approximately around 22.5 kg/m2", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "id": "NONOBESE GROUP"}, {"entity_type": "GENE", "description": "SEC16B is a gene that plays a role in endoplasmic reticulum export.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "SEC16 HOMOLOG B (SEC16B)"}, {"entity_type": "GENE", "description": "DNAJC27 is a gene involved in the hsp40 family that helps with cellular response to stress and heat shock proteins.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "DNAJ HEAT SHOCK PROTEIN FAMILY (HSP40) MEMBER C27 (DNAJC27)"}, {"entity_type": "GENE", "description": "CDKAL1 acts as a transcription factor that influences energy intake or expenditure by modulating related genes.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "CDK5 REGULATORY SUBUNIT-ASSOCIATED PROTEIN 1-LIKE 1 (CDKAL1)"}, {"entity_type": "GENE", "description": "TFAP2B is an important transcription factor involved in controlling metabolic pathways, especially for energy-related processes.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "TRANSCRIPTION FACTOR AP-2 BETA (TFAP2B)"}, {"entity_type": "PROTEIN", "description": "BDNF is a protein that plays various roles including modulating neuronal functions and affecting metabolism related to energy intake or expenditure.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "BDNF (BRAIN-DERIVED NEUROTROPHIC FACTOR)"}, {"entity_type": "PROTEIN", "description": "MYL2 facilitates muscle function by regulating myofilament sliding, involved in both physical activity and energy expenditure.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "MYOSIN LIGHT CHAIN-2 (MYL2)"}, {"entity_type": "GENE", "description": "OLFM4 is a gene that influences olfaction but also seems to impact metabolism or energy management based on its association with obesity risk.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "OLFACTOMEDIN-4 (OLFM4)"}, {"entity_type": "GENE", "description": "FTO plays multiple roles including regulating fat storage, metabolic pathways, and influencing obesity risks.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "FTO (FAT MASS AND OBESITY ASSOCIATED)"}, {"entity_type": "PROTEIN", "description": "MC4R is a receptor that regulates energy balance through its role in appetite control and metabolism.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "MC4R (MELANOCORTIN 4 RECEPTOR)"}, {"entity_type": "GENE", "description": "GIPR influences glucose metabolism, particularly insulin secretion or sensitivity in obesity risk scenarios.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "GIPR (GLUCAGON-LIKE PEPTIDE-1 RECEPTOR)"}, {"entity_type": "NUTRIENT/GENETIC VARIANT", "description": "FTO_rs1421085 is a genetic variant associated with obesity risk through its influence on energy intake or expenditure.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "FTO_RS1421085"}, {"entity_type": "DIETARY PATTERN/GENE", "description": "MC4R_rs6567160 influences appetite control which is part of the complex interaction between dietary patterns and obesity risk.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "MC4R_RS6567160"}, {"entity_type": "ENVIRONMENTAL FACTOR/COHORT", "description": "The Ansan/Ansung cohort provides insights into the genetic and environmental factors that contribute to obesity risk in specific populations.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "ANSAN/ANSUNG COHORT"}, {"entity_type": "GENE", "description": "The Polygenic Risk Score (PRS) was used in this study to assess the genetic risk for obesity.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "clusters": "[{level: 0, cluster: 0}]", "id": "PRS"}, {"entity_type": "DIETARY PATTERN", "description": "Initial menstruation age interacted with PRS for the obesity risk, suggesting a relationship between menarche and genetic predisposition.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "clusters": "[{level: 0, cluster: 0}]", "id": "MENARCHE AGE"}, {"entity_type": "DEMOGRAPHIC/ CLINICAL CHARACTERISTIC", "description": "A measure of body fat based on height and weight, used as a screening tool for obesity.<SEP>A measure used to evaluate obesity status related to genetic factors and glycemic status traits.<SEP>Body Mass Index (BMI) did not significantly differ between T2D-ESRD case subjects and nondiabetic control subjects.<SEP>Body Mass Index (BMI) was one of the physiological processes measured in relation to PRS and dietary patterns.<SEP>The BMI was affected by both menarche age and PRS in this context, indicating a physiological process related to body weight regulation.<SEP>Body Mass Index change associated with the FTO rs1558902 genotype and diet interventions.<SEP>Body Mass Index is a physiological process that was influenced by the FTO SNP in GP subjects.<SEP>Body Mass Index (BMI) has a positive correlation with snoring, affecting both males and females.<SEP>Mean BMI showed significant differences between PCOS cases and control subjects, highlighting its importance as a biomarker.<SEP>Body mass index is calculated based on weight and height, commonly used for screening of malnutrition and obesity.<SEP>BMI (Body Mass Index) varies according to the rs671 polymorphism significantly only in men.<SEP>BMI was associated with subgroup (overall P = .036) where the BMI in IBS-D was approximately 4 higher than in IBS-C and HV.<SEP>The average BMI was significantly higher in the cases compared to controls.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9<SEP>chunk-f9a60d3cf951d77afa115859755dac43<SEP>chunk-6f7ce1c71bd46f223756da44b0d0bb2c<SEP>chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6<SEP>chunk-2926e6216bf945b12ea4daa6345cad1f<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a<SEP>chunk-1f20dbdcc9de443dada1a2045dc7832b<SEP>chunk-3ad8ef794841d749e48b3223741e91d3<SEP>chunk-3135befd0c3140ad097d96ace6de86a1<SEP>chunk-9977706363cc368b3be503d07ed3ef62<SEP>chunk-2c178ae99eeffdddbea511cafbe4a7d5<SEP>chunk-1fde08e52a188ac6a549e7c11d59a092", "clusters": "[{level: 0, cluster: 1}]", "id": "BMI"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Menstruation age is an environmental factor that interacts with the Polygenic Risk Score for obesity risk.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "MENSTRUATION AGE"}, {"entity_type": "NUTRIENT", "description": "Plant-based diet intake had an interaction with the PRS in influencing obesity risk, suggesting a role of specific nutrients like plant foods.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "clusters": "[{level: 0, cluster: 0}]", "id": "PLANT-BASED DIET INTAKE"}, {"entity_type": "DIETARY PATTERN", "description": "Low consumption of plant-based diets showed an inverse impact on genetic influence for BMI compared to high plant-based dietary intake.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "LOW PLANT-BASED DIET INTAKE"}, {"entity_type": "DIETARY PATTERN", "description": "A high plant-based diet intake provided protection against obesity risk when combined with a high PRS, indicating its protective effect.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "HIGH PLANT-BASED DIET INTAKE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Fried food intake interacted with the PRS in obesity risk studies, suggesting that dietary factors can influence genetic predispositions for obesity.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "clusters": "[{level: 0, cluster: 0}]", "id": "FRIED-FOOD INTAKE"}, {"entity_type": "DIETARY PATTERN", "description": "High consumption of fried foods led to higher BMI but was offset by avoiding it when combined with a high PRS, implying the protective effect against obesogenic effects.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "HIGH FRIED-FOOD INTAKE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Adjusting for covariates did not relate PRS to menarche and menopausal age, suggesting other influencing factors were considered.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "COVARIATES"}, {"entity_type": "STATISTICAL MEASURE", "description": "Adjusted Odds Ratios (ORs) were used in statistical analysis, indicating the impact of PRS on obesity risk after considering certain variables.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "ADJUSTED ORS"}, {"entity_type": "CONFIDENCE INTERVAL", "description": "95% Confidence Interval was provided for adjusted ORs to give a range estimating the precision.<SEP>A statistical measure that gives a range within which we are 95% confident the true value lies, indicating precision of OR estimation.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3<SEP>chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "95% CI"}, {"entity_type": "RESEARCH DATA AND TABLES", "description": "Tables were used in the document to present statistical data, including ORs and their confidence intervals.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "TABLE 6"}, {"source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "description": "The genetic variant SEC16B_rs543874 interacts with MC4R_rs6567160 in influencing obesity risk.)", "entity_type": "UNKNOWN", "id": "SEC16B_RS543874"}, {"source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "description": "The genetic variant DNAJC27_rs713586 affects BDNF_rs6265 which contributes to the obesity risk.)", "entity_type": "UNKNOWN", "id": "BDNF_RS6265"}, {"source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "description": "The genetic variant DNAJC27_rs713586 affects BDNF_rs6265 which contributes to the obesity risk.)", "entity_type": "UNKNOWN", "id": "DNAJC27_RS713586"}, {"source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "description": "The genetic variant CDKAL1_rs9356744 is connected to GIPR_rs1444988703, impacting obesity risk.)", "entity_type": "UNKNOWN", "id": "CDKAL1_RS9356744"}, {"source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "description": "The genetic variant CDKAL1_rs9356744 is connected to GIPR_rs1444988703, impacting obesity risk.)", "entity_type": "UNKNOWN", "id": "GIPR_RS1444988703"}, {"source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "description": "FTO_rs1421085 contributes to obesity risk through its role in modulating fat mass and energy balance.)", "entity_type": "UNKNOWN", "id": "OBESITY RISK"}, {"source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "description": "Covariates were adjusted for in statistical analysis when relating PRS to other factors like menarche age.)", "entity_type": "UNKNOWN", "id": "ADJUSTMENT FACTOR"}, {"source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "description": "The impact of PRS on obesity was measured through BMI, showing a link between genetics and body weight regulation.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}]", "id": "OUTCOME MEASURE"}, {"source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "description": "Adjusted Odds Ratios were used to measure the effect of PRS in obesity risk analysis.)", "entity_type": "UNKNOWN", "id": "EFFECT MEASURE"}, {"source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "description": "Confidence intervals estimated the precision and reliability of adjusted odds ratios in statistical outcomes.)", "entity_type": "UNKNOWN", "id": "PRECISION ESTIMATE"}, {"source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "description": "Tables provided a structured way to present data from statistical analysis including key results like ORs and CI.)", "entity_type": "UNKNOWN", "id": "DATA PRESENTATION"}, {"entity_type": "GENOMIC VARIANT", "description": "The rs1558902 is a genetic variant that was examined for its association with weight changes in participants.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "RS1558902"}, {"entity_type": "OUTCOME", "description": "Weight loss or gain observed during the study period.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "WEIGHT"}, {"entity_type": "OUTCOME", "description": "Changes in Waist circumference related to genetic variation in the FTO gene and dietary protein.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "WC"}, {"entity_type": "OUTCOME", "description": "Fat is a macronutrient with several types like total fat, FM%, and subcutaneous adipose tissue (SAT).<SEP>Fat loss or retention measured through body composition analysis.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83<SEP>chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "FAT"}, {"entity_type": "BODY COMPOSITION", "description": "Free fat mass, which is associated with changes according to the FTO genotype and diet.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "FFM"}, {"entity_type": "BODY COMPOSITION", "description": "Percentage of Fat Mass related to genetic factors and protein intake.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "FM%"}, {"entity_type": "BODY COMPOSITION", "description": "Fat in abdominal area as a component of total body composition.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "TRUNK FAT"}, {"entity_type": "BODY FAT DISTRIBUTION", "description": "Total Abdominal Thickness measured by CT scan.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "TAT"}, {"entity_type": "BODY FAT DISTRIBUTION", "description": "Visceral Adipose Tissue, which shows different changes based on the FTO genotype and diet.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "VAT"}, {"entity_type": "BODY FAT DISTRIBUTION", "description": "Subcutaneous Adipose Tissue, which also varies with the genetic variant and dietary protein intake.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "SAT"}, {"entity_type": "NUTRIENT OR DIETARY PATTERN", "description": "The amount of dietary fat consumed by study participants.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "DIETARY FAT INTAKE"}, {"entity_type": "DIETARY PATTERN", "description": "Intervention involving high-protein diets compared to low-protein diets.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "PROTEIN DIET INTERVENTION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Age can influence fertility outcomes, such as affecting semen quality and vitamin status.<SEP>Age is associated with a higher likelihood of snoring, specifically increasing by 0.11 units per year.<SEP>Age of study participants.<SEP>Mean age was similar for all subjects involved.<SEP>Including age as a factor resulted in similar results for all genetic markers in relation to ovarian cancer.<SEP>When age was included as an additional covariate, results for all SNPs remained similar in relation to ovarian cancer risk.<SEP>Age played a role in the association between selenium levels and colorectal cancer/tumor risk.<SEP>Age is a demographic variable that differs significantly between ALDH2 rs671 genotypes in both males and females.<SEP>There was a significant positive relationship between age and increased risk of social disinhibition, with each year increase leading to an average score increase of 0.20 points.<SEP>The average age of cases was higher than that of controls.<SEP>Age is considered a factor that could influence treatment response and statistical analysis.<SEP>Age has been considered as a potential factor in the risk of HIV disease progression when correlated with TLR7 polymorphisms.", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403<SEP>chunk-6f7ce1c71bd46f223756da44b0d0bb2c<SEP>chunk-c9badb9a0fb813f42383cc8afc57ad45<SEP>chunk-2926e6216bf945b12ea4daa6345cad1f<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a<SEP>chunk-055177812e320344d131432b292111b0<SEP>chunk-d4e04dd6cd6ea28f04fd72d964574ec1<SEP>chunk-d311ac0182b37003c75c0005b4337fa6<SEP>chunk-2c91cdebb36ef064eb01a7021a961302<SEP>chunk-9977706363cc368b3be503d07ed3ef62<SEP>chunk-0d49552280a277a8a26456d06f8f26a8", "clusters": "[{level: 0, cluster: 0}]", "id": "AGE"}, {"entity_type": "DISEASE", "description": "Sex as a covariate was considered in the statistical analysis, indicating it may influence outcomes related to genetic risk scores or test performances.<SEP>Sex distribution among study participants.<SEP>Sex was considered as a factor that could influence BMI and obesity risk, especially when stratifying results.<SEP>Sex was noted among both groups, with more females in cases.<SEP>Sex has a significant association with snoring; males have an odds ratio of 2.264 compared to females.<SEP>Sex, as a demographic characteristic, showed no significant association with social disinhibition scores.<SEP>Sex is often associated with differences in how individuals respond to treatments, being part of lifestyle factors.<SEP>Sexual and gender differences can influence the course of HIV disease, including factors that may relate to TLR7 polymorphisms.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7<SEP>chunk-055177812e320344d131432b292111b0<SEP>chunk-d4e04dd6cd6ea28f04fd72d964574ec1<SEP>chunk-3135befd0c3140ad097d96ace6de86a1<SEP>chunk-2c91cdebb36ef064eb01a7021a961302<SEP>chunk-9977706363cc368b3be503d07ed3ef62<SEP>chunk-0d49552280a277a8a26456d06f8f26a8", "clusters": "[{level: 0, cluster: 0}]", "id": "SEX"}, {"entity_type": "DEMOGRAPHIC VARIABLE", "description": "Ethnic background of study participants.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "ETHNICITY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The AA genotype is linked to a specific response to diet.<SEP>The variant form of the glucokinase regulatory protein that is present in a certain percentage of individuals.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450<SEP>chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "AA GENOTYPE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Fat distribution changes in response to different protein diets.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "FAT DISTRIBUTION"}, {"entity_type": "DIETARY PATTERN", "description": "A low-protein diet affects fat distribution, particularly for those with the AA genotype.<SEP>A low-protein diet was prescribed for patients due to HI/HA syndrome.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83<SEP>chunk-2b81371954b99facae9a0056e70aa7bd", "id": "LOW-PROTEIN DIET"}, {"entity_type": "DIETARY PATTERN", "description": "The high-protein diet was associated with greater fat distribution changes in participants with the AA genotype compared to others.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "HIGH-PROTEIN DIET"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Participants with the AA genotype had less loss of TAT in response to low-protein diets.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "TAT (TOTAL ADIPOSE TISSUE)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The association between the AA genotype and less SAT loss was observed specifically for the low-protein diet.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "SAT (SUBCUTANEOUS ADIPOSE TISSUE)"}, {"entity_type": "DIETARY PATTERN", "description": "Genotype-time interactions were found with total fat, FM%, and percentage of trunk fat changes in response to a low-fat diet.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "LOW-FAT DIET"}, {"entity_type": "ENVIRONMENTAL FACTOR (RACE/ETHNICITY)", "description": "Similar trends as observed in the AA genotype participants were also seen in the white population but without detailed genotypic data provided.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "WHITE POPULATION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The term \'genotype-time interaction\' indicates how genetic factors influence responses over time.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "GENOTYPE-TIME INTERACTION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Total Adipose Tissue describes the overall fat storage in the body.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "TOTAL ADIPOSE TISSUE"}, {"entity_type": "NUTRIENT", "description": "FM%, or Fat Mass Percentage, is a measurement of total body fat as a percentage of total weight.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "FM% (FAT MASS PERCENTAGE)"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "The FTO genetic variant interacted with dietary protein to influence changes in fat mass, free-fat mass, and percentage of trunk fat.)", "entity_type": "UNKNOWN", "id": "CHANGES IN BODY COMPOSITION"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "The FTO genetic variant interacted with dietary protein to influence changes in fat mass, free-fat mass, and percentage of trunk fat.)", "entity_type": "UNKNOWN", "id": "FTO RS1558902 GENOTYPE"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "Age was adjusted for in the analysis, and interactions were found between age and FTO rs1558902 genotype on weight changes.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "ASSOCIATION WITH GENOTYPE"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "Sex differences influenced the association of the FTO genetic variant with body composition changes; sex was included in the analysis.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "EFFECT ON BODY COMPOSITION"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "Ethnicity interacted with the FTO rs1558902 genotype, potentially influencing weight and fat distribution responses to dietary interventions.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITH GENOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "The study focuses on association of this polymorphism with anthropometric variables.", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "clusters": "[{level: 0, cluster: 2}]", "id": "FTO RS9939609 POLYMORPHISM"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The waist circumference measurement followed a co-dominant model with BMI increase, reflecting association of the polymorphism with anthropometric variables.<SEP>Waist circumference measurements are used as a parameter to assess obesity and metabolic health.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9<SEP>chunk-3135befd0c3140ad097d96ace6de86a1", "id": "WAIST CIRCUMFERENCE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The study considers education level as a factor that interacts with the FTO SNP to influence BMI and obesity risk.", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "clusters": "[{level: 0, cluster: 2}]", "id": "EDUCATION LEVEL"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "In elderly HCR subjects, PA was considered an environmental factor influencing the FTO SNP\'s effect on anthropometric measurements.", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "id": "PHYSICAL ACTIVITY (PA)"}, {"entity_type": "EDUCATION CATEGORY", "description": "In stratified analyses by educational level, BMI association with A-allele varied among primary studies, secondary studies, and university level.", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "id": "UNIVERSITY LEVEL"}, {"entity_type": "EDUCATION CATEGORY", "description": "The FTO SNP was associated with higher BMI in non-university individuals compared to university level subjects.", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "id": "NON-UNIVERSITY INDIVIDUALS"}, {"entity_type": "GENOMIC VARIANT VARIANT TYPE", "description": "In GP subjects, AA variants of the FTO SNP were found to have a different effect on obesity risk compared to TT homozygotes.", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "id": "AA SUBJECTS"}, {"entity_type": "GENOMIC VARIANT", "description": "The FTO gene polymorphism, specifically a variant in this gene, was studied in relation to obesity risk.", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "id": "FTO SNP"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Education level is considered as an environmental factor that may affect the association between genetic polymorphisms and health outcomes.<SEP>Education level was considered as an environmental factor that may affect the association between the FTO SNP and obesity risk.", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "id": "EDUCATION"}, {"entity_type": "ENVIRONMENTAL FACTOR/", "description": "Physical activity was considered as a factor that could influence the relationship between genetic variations and obesity.<SEP>Lifestyle Factor", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487<SEP>chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "PA (PHYSICAL ACTIVITY)"}, {"source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "description": "The FTO SNP\'s influence on obesity risk was modulated by education level in GP subjects.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}]", "id": "INTERACTION WITH FTO SNPC"}, {"source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "description": "In elderly HCR subjects, PA was found to interact with the FTO SNP affecting anthropometric measurements.)", "entity_type": "UNKNOWN", "id": "IMPACT ON FTO SNP EFFECT"}, {"source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "description": "Sex influenced the relationship between the FTO SNP and BMI, as well as obesity risk.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "INTERACTION WITH BMI AND OBESITY"}, {"entity_type": "COHORT STUDY/RCT", "description": "None of the entities are directly associated with this text as it focuses on characteristics of cohort studies and RCTs rather than individual entities.", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "id": "PUBLICATION"}, {"entity_type": "GENE", "description": "NOS1 is a gene that plays a critical role in cardiovascular health by regulating nitric oxide production.", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "id": "NOS1"}, {"entity_type": "RS12441227", "description": "A genomic variant in the NOS1 gene has been linked to increased risk of hypertension.<SEP>Early MTX-related neurotoxicity may be influenced by certain genomic variations, potentially contributing to its manifestation.<SEP>A genomic variant specific to IL-6R\u03b1 gene influences the severity of colitis.<SEP>Specific allele of a genetic variant associated with increased risk for T2DM<SEP>The presence of specific genomic variants might influence AMD risk independently.<SEP>NA<SEP>rs8096658 has an effect size of \u03c0C = 0.877<SEP>rs8096658 overlaps with fetal kidney cell DHSs at the NFATC1 locus<SEP>A variant rs1099733 is highlighted as significant.<SEP>Specific genomic variants related to the context have been omitted.<SEP>SNP rs12441227 at the THSD4 locus explains 2.9% of the residual variance in FEV1/FVC lung function.", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6<SEP>chunk-45a965c3c3013c70d0fa0f1680283083<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-e8f81fdd25a22582c6ce3d1e15f74af3<SEP>chunk-0d8bc9a093c56fefc8b7af3fb9bc2862<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7<SEP>chunk-72f4505e43f67f3564607392cb178656<SEP>chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9<SEP>chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "clusters": "[{level: 0, cluster: 7}, {level: 1, cluster: 26}]", "id": "GENOMIC VARIANT"}, {"entity_type": "TRANSCRIPT", "description": "490 statistically significantly differentially expressed probes corresponding to 421 genes in DBA patients compared to controls.<SEP>A transcript related to the functions of GSTP1 protein was assessed for polymorphisms.<SEP>NA<SEP>The expression levels of specific transcripts associated with complement pathway genes were considered.<SEP>The genotyping results provide insights into specific transcripts associated with the detected genomic variants.<SEP>The transcript of the NOS1 gene is influenced by dietary patterns, particularly the intake of folate.<SEP>The transcript of IL-6R\u03b1 gene is related with disease protection seen after T cell transfers.<SEP>The transcript results from the process of transcription in which RNA molecules are synthesized by RNA polymerase using one or more strands of DNA as a template.<SEP>Transcripts are the primary output of gene expression and can be considered as a product of genetic information.<SEP>NFATC1 is a gene encoding a member of the NFAT family that affects kidney function and development.<SEP>The CASP3 transcript is considered in the analysis.<SEP>Specific transcripts related to the context have been omitted.<SEP>No transcript information provided for the associations identified.<SEP>Transcripts associated with these genetic markers were part of the study.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94<SEP>chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-a7d0bfa8b16262a1f313b7e3060969c6<SEP>chunk-45a965c3c3013c70d0fa0f1680283083<SEP>chunk-ab5011584b0230efbf80a98b2a693a0e<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-e8f81fdd25a22582c6ce3d1e15f74af3<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a<SEP>chunk-72f4505e43f67f3564607392cb178656<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea<SEP>chunk-0d8bc9a093c56fefc8b7af3fb9bc2862<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "clusters": "[{level: 0, cluster: 10}, {level: 1, cluster: 32}]", "id": "TRANSCRIPT"}, {"entity_type": "PROTEIN", "description": "The NOS1 protein is involved in the production of nitric oxide.<SEP>The NOS1 protein is involved in vascular function and interacts with folate metabolism.", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432<SEP>chunk-0d8bc9a093c56fefc8b7af3fb9bc2862<SEP>chunk-2b81371954b99facae9a0056e70aa7bd", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 44}]", "id": "NOS1 PROTEIN"}, {"entity_type": "NUTRIENT", "description": "Folate is a key nutrient found in leafy greens and plays a role in nitric oxide production.<SEP>Folate is a nutrient found in various foods that influences vitamin B12 levels.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "id": "FOLATE"}, {"entity_type": "FOOD", "description": "Leafy greens are a food source rich in folate, a nutrient critical for cardiovascular health.", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "id": "LEAFY GREENS"}, {"entity_type": "DIETARY PATTERN", "description": "A dietary pattern refers to a specific way of eating, which may involve certain types of foods and nutrients.<SEP>A dietary pattern refers to the overall composition and timing of food intake over a period of time.<SEP>Dietary patterns influence the transcript of the NOS1 gene and folate metabolism.<SEP>Dietary patterns might influence TAG levels through interaction with genotypes.<SEP>NA<SEP>The Mediterranean diet pattern might have been considered as relevant to health outcomes.<SEP>Specific dietary patterns related to the context have been omitted.<SEP>No dietary pattern information provided for the associations identified.<SEP>There was an examination of various dietary patterns that could impact genetic and metabolic outcomes.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6<SEP>chunk-e8f81fdd25a22582c6ce3d1e15f74af3<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a<SEP>chunk-72f4505e43f67f3564607392cb178656<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea<SEP>chunk-0d8bc9a093c56fefc8b7af3fb9bc2862<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "clusters": "[{level: 0, cluster: 10}, {level: 1, cluster: 31}]", "id": "DIETARY PATTERN"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Folate metabolism is a physiological process that interacts with the NOS1 protein to maintain vascular function.<SEP>Folate metabolism, influenced by dietary patterns and genetic factors like MTHFR variants, plays a critical role in cardiovascular health.", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb<SEP>chunk-0d8bc9a093c56fefc8b7af3fb9bc2862<SEP>chunk-2b81371954b99facae9a0056e70aa7bd", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 44}]", "id": "FOLATE METABOLISM"}, {"entity_type": "METABOLIC PATHWAY", "description": "The metabolic pathway of nitric oxide synthesis is central to the interaction between the NOS1 gene and cardiovascular health.", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 43}]", "id": "NITRIC OXIDE SYNTHESIS"}, {"entity_type": "DISEASE", "description": "Cardiovascular disease is a condition influenced by both genetic and environmental factors, including the NOS1 gene and dietary patterns.<SEP>Cardiovascular disease is influenced by factors such as genetic variants, dietary patterns, folate intake and environmental factors, affecting nitric oxide synthesis.", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862<SEP>chunk-2b81371954b99facae9a0056e70aa7bd", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 43}]", "id": "CARDIOVASCULAR DISEASE"}, {"entity_type": "GENETIC ALTERATION", "description": "Epigenetic modifications linked to poor dietary patterns can alter the metabolic pathway of nitric oxide synthesis.<SEP>These are changes to DNA without altering its sequence, influencing gene expression and cellular function.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "clusters": "[{level: 0, cluster: 10}, {level: 1, cluster: 31}]", "id": "EPIGENETIC MODIFICATION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Diet is an environmental factor that modulates the relationship between the NOS1 gene and cardiovascular health.", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "id": "DIET"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Physical activity can modulate the effects of alcohol consumption on cortisol levels.<SEP>Physical activity is a lifestyle factor that further modulates cardiovascular health.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a<SEP>chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 19}]", "id": "PHYSICAL ACTIVITY"}, {"entity_type": "DRUG CLASS/", "description": "Statins are a drug therapy used to manage cardiovascular disease risk.<SEP>Medication Type", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63<SEP>chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "id": "STATINS"}, {"entity_type": "THERAPY", "description": "Personalized therapies based on genetic and nutritional profiles are being explored to manage cardiovascular health.", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "id": "PERSONALIZED THERAPY"}, {"entity_type": "GENE", "description": "The TCF7L2 gene is associated with diabetes risk.<SEP>The TCF7L2 gene is of particular interest due to its strong association with type 2 diabetes (T2D) risk and known impact on incretin effect on insulin secretion.<SEP>The TCF7L2 gene\'s variants are of interest due to their potential interaction with dietary factors that could influence the risk of developing type 2 diabetes.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc<SEP>chunk-6a7c69534e11aa2fc4bd948484aa81c4", "id": "TCF7L2 GENE"}, {"entity_type": "GENOMIC VARIANT", "description": "The TCF7L2 variant rs7903146 interacts with both fiber and whole-grain intake, impacting insulin secretion and glucose metabolism.<SEP>The rs7903146 variant in the TCF7L2 gene is considered strongly associated with T2D risk.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "RS7903146 VARIANT"}, {"entity_type": "NUTRIENT", "description": "Increased cereal fiber intake was associated with lower T2D incidence among individuals carrying a specific TCF7L2 genotype (rs7903146 CC-variant).", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "FIBER INTAKE"}, {"entity_type": "NUTRIENT", "description": "Higher whole-grain consumption was linked to lower T2D risk in persons with the rs7903146 CC-variant.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "WHOLE GRAIN INTAKE"}, {"entity_type": "GENE", "description": "The NOTCH2 gene is involved in WNT activity, relevant for cellular growth and differentiation.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "NOTCH2 GENE"}, {"entity_type": "GENOMIC VARIANT", "description": "A specific variant of the NOTCH2 gene (rs10923931) interacts with dietary factors affecting T2D risk.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "RS10923931 VARIANT"}, {"entity_type": "GENE", "description": "The ZBED3 gene plays a role in WNT signaling pathways, contributing to cellular processes that influence T2D risk.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "ZBED3 GENE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "NOTCH2 gene and the ZBED3 gene interact with WNT activity, which is pertinent to glucose metabolism and insulin sensitivity.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "WNT ACTIVITY"}, {"entity_type": "GENE", "description": "The IRS1 gene is known for its role in insulin sensitivity, which is crucial for glucose homeostasis.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "IRS1 GENE"}, {"entity_type": "GENE", "description": "The GIPR gene impacts insulin secretion, contributing to glucose metabolism and diabetes risk.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "GIPR GENE"}, {"entity_type": "GENE", "description": "The PPAR\u03b3 gene regulates fatty acid storage, affecting lipid metabolism and glucose levels.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "PPAR\u0393 GENE"}, {"entity_type": "GENE", "description": "The FFAR4 gene is involved in G\u03b1q signaling pathways relevant for cellular responses to nutrients and hormones.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "FFAR4 GENE"}, {"entity_type": "GENE", "description": "The CAV2 gene affects total fat and saturated fatty acid metabolism, potentially influencing T2D risk.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "CAV2 GENE"}, {"entity_type": "GENOMIC VARIANT", "description": "Haemochromatosis-associated variant in the HFE gene linked to increased risk for Type 2 Diabetes (T2D) among women.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "HFE H63D"}, {"entity_type": "GENOMIC VARIANT", "description": "Another haemochromatosis-related variant in the HFE gene, also associated with higher T2D risk among women with this variant compared to those without.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "HFE C282Y"}, {"entity_type": "DISEASE", "description": "Increased likelihood of developing Type 2 Diabetes related to genetic variants like HFE H63D and C282Y.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "T2D RISK"}, {"entity_type": "NUTRIENT", "description": "Heme iron consumption interacts with the SLC40A1 rs744653 variant and shows borderline significance (P = .046) for association with T2D in total sample, particularly among men.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "HEME IRON INTAKE"}, {"entity_type": "NUTRIENT", "description": "Zinc intake has an interaction with the SLC30A8 zinc transporter variants like rs16889462 and rs16889462, but no clear association with T2D incidence in exposure strata.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "ZINC INTAKE"}, {"entity_type": "GENETIC VARIANT", "description": "A risk score related to multiple genetic variants that can influence the incidence of Type 2 Diabetes. Higher GRS correlates with increased T2D risk.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "T2D-GRS (DIABETES GENETICS RISK SCORE)"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant near the lactase persistence gene modifies the association of dairy products and Type 2 Diabetes, especially among milk consumers who carry the lactase persistence allele T-allele.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "LACTASE PERSISTENCE LCT RS4988235"}, {"entity_type": "NUTRIENT/DIETARY PATTERN", "description": "Coffee intake interacts with some genetic variants affecting its potential to lower Type 2 Diabetes risk. The strongest effect was observed for individuals carrying the TCF7L2 variant rs12255372.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "COFFEE INTAKE"}, {"entity_type": "DIETARY PATTERN", "description": "A score reflecting adherence to a Mediterranean diet does not show significant interaction with a Type 2 Diabetes risk score, indicating that the protective effect against T2D is consistent regardless of genetic predisposition.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "MEDITERRANEAN DIET SCORE"}, {"entity_type": "DIETARY PATTERN", "description": "This pattern associated with higher incidence of T2D is modified in its effects among individuals with high adherence to a Type 2 Diabetes risk score, implying that the increased risk does not apply uniformly to everyone.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "WESTERN DIETARY PATTERN"}, {"entity_type": "ENVIRONMENTAL FACTOR/", "description": "Lifestyle Factor", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "LTPA (LEISURE-TIME PHYSICAL ACTIVITY)"}, {"entity_type": "GENOMIC VARIANT", "description": "This allele in the ADRA2B gene interacts with changes in LTPA to show lower T2D incidence when carried alongside specific genotypes like the ABCC8 rs3758947 GG-genotype.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "ADRA2B GLU12 ALLELE"}, {"entity_type": "GENOMIC VARIANT/", "description": "Environmental Factor/", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "ABCC8 RS3758947 GG-GENOTYPE"}, {"entity_type": "PROTEIN/", "description": "Nutrient/Dietary Pattern/", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "TCF7L2 RS12255372 T-ALELE/"}, {"entity_type": "GENOMIC VARIANT/", "description": "Environmental Factor/", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "CDKAL1 RS7754840 G-ALELE/"}, {"entity_type": "GENOMIC VARIANT/", "description": "Environmental Factor/", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "KCNJ11 RS5215 CC-HAPLOTYPE/"}, {"entity_type": "GENOMIC VARIANT/", "description": "Environmental Factor/", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "INSULIN RECEPTOR SUBSTRATE 1 (IRS1) RS1522813 A-ALELE/"}, {"entity_type": "GENOMIC VARIANT/", "description": "Environmental Factor/", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "ADRA2B GLU12 ALLELE/"}, {"entity_type": "GENOMIC VARIANT/", "description": "Environmental Factor/", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "NON-RISK HAPLOTYPE OF SLC2A2 VARIANTS/"}, {"entity_type": "GENOMIC VARIANT", "description": "Persons with this genotype show specific response to change in LTPA.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "GG-GENOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "This haplotype of SLC2A2 variants seems to be protective against changes in LTPA.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "NONRISK HAPLOTYPE"}, {"entity_type": "GENE", "description": "These genetic variants were further investigated for their potential interactions with PA and change in LTPA.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "RS5393, RS5394, RS5400, RS5404"}, {"entity_type": "DIETARY PATTERN", "description": "Interventions aimed at reducing diabetes risk were studied to find out if they can modify the development of T2D.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "WEIGHT LOSS INTERVENTIONS"}, {"entity_type": "GENE", "description": "Participants with this gene variant showed lower T2D incidence in intervention groups compared to controls.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "TCF7L2 RS12255372"}, {"entity_type": "GENOMIC VARIANT", "description": "This allele was reported to have a similar finding as the TCF7L2 rs12255372, but without significant interaction test results.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "RS7903146 RISK T-ALLELE"}, {"entity_type": "GENE", "description": "This genetic variant showed an association with reduced T2D incidence during weight loss intervention in DPP study.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "ENPP1 RS1044498 DIABETOGENIC VARIANT"}, {"entity_type": "GENE", "description": "A trend for interaction was found for this variant and the weight loss intervention resulting in lower T2D incidence.<SEP>An interaction trend was observed for this MC4R gene variant when combined with weight loss interventions, as per findings from both studies.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "MC4R RS17066829"}, {"entity_type": "GENOMIC VARIANT", "description": "Persons with this genotype had a lower T2D incidence when combined with weight loss interventions, as reported by both DPP and DPS studies.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "PPAR\u0393 RS1152003 CC-GENOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "Weight loss intervention resulted in decreased T2D incidence among persons with this genotype compared to reference genotypes.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "TNF\u0391 RS1800629 GG-GENOTYPE"}, {"entity_type": "GENE", "description": "Persons with this allele had a lower risk of developing T2D after weight loss interventions, as indicated by the DPS study.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "ADRA2B GLU9-ALLELE"}, {"entity_type": "GENOMIC VARIANT", "description": "The LIPC variant was associated with a lower T2D incidence in persons who underwent a weight loss intervention according to the DPS study.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "LIPC RS2070895 A-ALLELE"}, {"entity_type": "GENOMIC VARIANT", "description": "GG-Genotype is one of several variants that was further investigated for its interaction with PA and change in LTPA.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "RS5393"}, {"entity_type": "GENOMIC VARIANT", "description": "GG-Genotype includes this variant among others that were studied regarding their response to physical activity changes.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "RS5394"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant is part of the non-risk haplotype, which showed protective effects against changes in LTPA.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "RS5400"}, {"entity_type": "GENOMIC VARIANT", "description": "GG-Genotype carries this variant among others that were analyzed for their interactions with PA or LTPA changes.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "RS5404"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Lifestyle Physical Activity (LTPA) refers to the moderate-to-vigorous intensity physical activity that participants engaged in, which was correlated with specific genotypes.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "LTPA"}, {"entity_type": "DIETARY PATTERN STUDY", "description": "Diet and Physical Activity Study (DPS) investigated the effects of weight loss interventions combined with various genetic variants on T2D risk, indicating potential interactions for certain genotypes.<SEP>Dietary Pattern Study (DPS) involved research focusing on dietary patterns, physical activity levels, and diabetes prevention among different populations.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "DPS"}, {"entity_type": "OTHER RESEARCH STUDY", "description": "A separate study alongside DPP that also looked at the impact of weight loss interventions and specific genetic variants on diabetes risk.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "ASTI STUDY"}, {"entity_type": "GENE", "description": "This gene variant was associated with reduced T2D incidence in individuals undergoing a weight loss intervention based on the DPP study results.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "ENPP1 RS1044498"}, {"entity_type": "RESEARCH STUDY", "description": "Diabetes Prevention Program (DPP) is the name of a major clinical trial that was referenced in studying the effects of weight loss interventions on T2D risk and genetic variants.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "DPP"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "All RCTs involve lifestyle interventions that aim to reduce weight and lower T2D risk, with variations in the methods and composition of these interventions.)", "entity_type": "UNKNOWN", "id": "CHARACTERISTICS OF RCTS/STUDY DESIGN"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f<SEP>chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "The lifestyle intervention focused on diet, physical activity, and healthy living choices.<SEP>The lifestyle intervention group had the largest decrease in fasting proinsulin levels at 1 year and was significantly lower than both metformin (p<0.001) or troglitazone groups.<SEP>All RCTs involve lifestyle interventions that aim to reduce weight and lower T2D risk, with variations in the methods and composition of these interventions.)", "entity_type": "INTERVENTION GROUP", "id": "LIFESTYLE INTERVENTION"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "In most studies, participants are over 40 years old at follow-up.)", "entity_type": "UNKNOWN", "id": "AGE OF PARTICIPANTS"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "In most studies, participants are over 40 years old at follow-up.)", "entity_type": "UNKNOWN", "id": "CHARACTERISTICS OF COHORT STUDIES/FOLLOW-UP TIME"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "The risk of cardiovascular disease varies among the different cohorts and RCT populations.)", "entity_type": "UNKNOWN", "id": "CARDIOVASCULAR RISK"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "The risk of cardiovascular disease varies among the different cohorts and RCT populations.)", "entity_type": "UNKNOWN", "id": "CHARACTERISTICS OF RCTS/BASELINE RISK"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "The dietary assessments are based on validated food frequency questionnaires or other methods like dietary history records or biomarker measurements, affecting the accuracy of findings.)", "entity_type": "UNKNOWN", "id": "DIETARY EXPOSURE/ASSESSMENT METHOD"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "The dietary assessments are based on validated food frequency questionnaires or other methods like dietary history records or biomarker measurements, affecting the accuracy of findings.)", "entity_type": "UNKNOWN", "id": "FOOD FREQUENCY QUESTIONNAIRE/"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "The physical activity levels were assessed using interviews or questionnaires which can introduce subjectivity and potential recall biases.)", "entity_type": "UNKNOWN", "id": "INTERVIEWS/QUESTIONNAIRES"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "The physical activity levels were assessed using interviews or questionnaires which can introduce subjectivity and potential recall biases.)", "entity_type": "UNKNOWN", "id": "PA EXPOSURE/ASSESSMENT METHOD"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Dietary intake can influence physical activity levels and vice versa, affecting overall health.)", "entity_type": "UNKNOWN", "id": "PA EXPOSURE"}, {"source_id": "chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Increased physical activity can reduce the risk of developing cardiovascular disease.)<SEP>Lifestyle factors include habits or choices made by individuals that impact health, such as diet and exercise.", "entity_type": "BEHAVIORAL CHOICE", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 19}]", "id": "LIFESTYLE FACTOR"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Statin medications help lower cholesterol levels and reduce the risk of heart disease.)", "entity_type": "UNKNOWN", "id": "DRUG THERAPY"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Personalized therapies consider genetic variations when planning treatment for cardiovascular conditions.)", "entity_type": "UNKNOWN", "id": "GENETIC PROFILES"}, {"entity_type": "GENOMIC VARIANT", "description": "The third one of the 23 obesity SNPs that reached a significant association with BMI at baseline.<SEP>rs7903146 in TCF7L2 gene has a frequency of association with type 2 diabetes.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43<SEP>chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "RS7903146"}, {"entity_type": "GENE", "description": "The TCF7L2 gene is associated with an increased risk of type 2 diabetes.<SEP>The TCF7L2 gene, associated with another one of the obesity predisposing SNPs.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43<SEP>chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "TCF7L2"}, {"entity_type": "GENOMIC VARIANT", "description": "rs10010131 in WFS1 gene shows a directionally consistent association with type 2 diabetes.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "id": "RS10010131"}, {"entity_type": "GENE", "description": "The WFS1 gene associates with an increased risk of type 2 diabetes, showing consistency with prior reports.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "id": "WFS1"}, {"entity_type": "GENOMIC VARIANT", "description": "rs10946398 in CDKAL1 gene is linked to the directionally consistent association with type 2 diabetes.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "id": "RS10946398"}, {"entity_type": "GENE", "description": "SNP (rs12252-C) is associated with type 2 diabetes risk on chromosome 6.<SEP>The CDKAL1 gene has a directionally consistent association with type 2 diabetes, similar to prior research.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-f9a60d3cf951d77afa115859755dac43", "id": "CDKAL1"}, {"entity_type": "GENOMIC VARIANT", "description": "rs864745 in JAZF1 gene is associated with an increased risk of type 2 diabetes.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "id": "RS864745"}, {"entity_type": "GENE", "description": "The JAZF1 gene associates with type 2 diabetes, supporting previous findings.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "id": "JAZF1"}, {"entity_type": "GENOMIC VARIANT", "description": "rs4607103 in ADAMTS9 gene is associated with an increased risk of type 2 diabetes but shows opposite directionality compared to a European sample.<SEP>rs4607103 variant in ADAMTS9 gene shows an association with type 2 diabetes, but its direction of effect is opposite to that observed in European populations.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "id": "RS4607103"}, {"entity_type": "GENE", "description": "The ADAMTS9 gene associates with type 2 diabetes, showing an opposing association when compared to findings from a European population.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "id": "ADAMTS9"}, {"entity_type": "GENOMIC VARIANT", "description": "A specific genetic variant associated with the TCF7L2 gene that influences diabetes risk.", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "id": "TCF7L2 RISK VARIANT RS7903146"}, {"source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "description": "The association between the TCF7L2 risk variant and the nominal analysis was significant (P = 0.011, unweighted OR 1.04 [95% CI 1.01\u20131.08]).)", "entity_type": "UNKNOWN", "id": "NOMINAL ANALYSIS RESULTS"}, {"source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "description": "The association between the TCF7L2 risk variant and the weighted analysis was also significant (P = 8.10 \u00d7 10\u22125, weighted OR 1.06 [1.03\u20131.10]).)", "entity_type": "UNKNOWN", "id": "WEIGHTED ANALYSIS RESULTS"}, {"source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "description": "The nominal analysis of the TCF7L2 risk variant did not show association with ordinal data (P = 0.33, unweighted OR 1.02 [0.98\u20131.05]).)", "entity_type": "UNKNOWN", "id": "ORDINAL ANALYSIS RESULTS (NOMINAL CATEGORIES)"}, {"source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "description": "The weighted ordinal analysis of the TCF7L2 risk variant also did not show association with ordinal data (P = 0.40, weighted OR 1.02 [0.98\u20131.06]).)", "entity_type": "UNKNOWN", "id": "ORDINAL ANALYSIS RESULTS (WEIGHTED)"}, {"source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e<SEP>chunk-6a7c69534e11aa2fc4bd948484aa81c4", "description": "In subjects of African ancestry, there was a decreasing percentage of IFG/IGT (62.8%) compared to other ethnic groups.<SEP>The analyses did not find evidence for interaction with African Ancestry when considering the proportions of ancestry (P = 0.74).)", "entity_type": "ENVIRONMENTAL FACTOR", "id": "AFRICAN ANCESTRY"}, {"source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "description": "There was no evidence for an interaction between the TCF7L2 risk variant and age in these analyses.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "INTERACTION"}, {"source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "description": "The count of risk alleles for the TCF7L2 risk variant was considered in nominal and ordinal analyses.)", "entity_type": "UNKNOWN", "id": "COUNT OF RISK ALLELES"}, {"source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "description": "The nominal analysis results showed a significant association (P = 0.011, unweighted OR 1.04 [95% CI 1.01\u20131.08]) for the TCF7L2 risk variant.)", "entity_type": "UNKNOWN", "id": "SIGNIFICANCE"}, {"source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7<SEP>chunk-6a7c69534e11aa2fc4bd948484aa81c4", "description": "The analysis revealed connections between specific variables, such as GRS and test performances, after adjusting for certain covariates.<SEP>The nominal ordinal analysis results did not show a significant association between the TCF7L2 risk variant and ordinal data (P = 0.33, unweighted OR 1.02 [0.98\u20131.05]).)", "entity_type": "STATISTICAL RELATIONSHIP", "id": "ASSOCIATION"}, {"entity_type": "DISEASE", "description": "The baseline clinical characteristic of 42.8% participants displayed normal glucose tolerance.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "NORMAL GLUCOSE TOLERANCE (NGT)"}, {"entity_type": "DISEASE", "description": "IFG/IGT is a glycemic status where subjects are classified as impaired fasting glucose/impaired glucose tolerance, comprising 42.5% of the participants.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "IMPAIRED FASTING GLUCOSE/IMPAIRED GLUCOSE TOLERANCE (IFG/IGT)"}, {"entity_type": "DISEASE", "description": "T2D accounts for 14.7% of the participants and is a condition where individuals are diagnosed with diabetes.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "TYPE 2 DIABETES (T2D)"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The ethnic distribution showed significant differences among the NGT, IFG/IGT, and T2D groups, especially in East Asians.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "EAST ASIAN ANCESTRY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Among European subjects, the percentage of T2D (14.4%) was lower than East Asians (22.7%).", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "EUROPEAN ANCESTRY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "In Native North American participants, there was a decreasing trend for T2D compared to other ethnic groups.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "NATIVE NORTH AMERICAN ANCESTRY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "For Latino Americans, the percentage of subjects with IFG/IGT was lower (55.5%) compared to East Asians.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "LATINO AMERICAN ANCESTRY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The ethnic composition of the cohort had significant differences across different glycemic statuses.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "ETHNIC DISTRIBUTION"}, {"entity_type": "GENOMIC VARIANT", "description": "One of the 23 obesity SNPs that reached a significant association with BMI at baseline.", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "RS9939609"}, {"entity_type": "GENE", "description": "FTO is associated with coffee consumption when considering its role in the development of snoring.<SEP>FTO is previously linked to alcohol consumption along with RNA5SP471 and SND1.<SEP>Located within or close to the FTO gene<SEP>The FTO gene, associated with one of the obesity predisposing SNPs.<SEP>rs9913297 variant is associated with BMI, fat-free mass, and waist circumference.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-ad609cb10e900d42a6901eea1591d8ba<SEP>chunk-13862d7ab74f5d4fa3f2261984d15d64<SEP>chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "FTO"}, {"entity_type": "GENOMIC VARIANT", "description": "Another one of the 23 obesity SNPs that reached a significant association with BMI at baseline.", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "RS2984618"}, {"entity_type": "GENE", "description": "The TAL1 gene, associated with another one of the obesity predisposing SNPs.", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "TAL1"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Evaluation of changes in BMI which is influenced by obesity predisposing SNPs, genetic risk score (GRS), and glycemic status traits.", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "BMI CHANGE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The accumulation of multiple obesity SNPs to determine an individual\'s obesity risk.", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "GENETIC RISK SCORE (GRS)"}, {"entity_type": "FASTING PLASMA GLUCOSE", "description": "A trait related to glycemic status that showed a significant interaction with one of the obesity SNPs on baseline BMI.<SEP>Fasting plasma glucose levels indicate insulin sensitivity and risk of diabetes.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9<SEP>chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "FPG"}, {"entity_type": "GENOMIC VARIANT", "description": "One of the 23 obesity SNPs associated with FPG and baseline BMI, but not observed as a main effect.<SEP>The SNP is associated with LEPR gene and has a significant interaction on baseline BMI.", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "RS1011527"}, {"entity_type": "GENE", "description": "This gene interacts with rs1011527 SNP to influence the glycemic status traits affecting BMI at baseline.", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "LEPR"}, {"entity_type": "TRAIT INTERACTION", "description": "These traits transitionally impact the relationship between obesity SNPs, GRS and BMI change.", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "GLYCEMIC STATUS TRAITS"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "Genetic risk score is positively associated with increased BMI at baseline, indicating a cumulative effect of multiple genetic factors on obesity.)<SEP>The genetic risk score is a cumulative measure influenced by multiple obesity predisposing SNPs and correlates significantly with BMI at baseline.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}]", "id": "GRS"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "There is an observed interaction between one obesity SNP and a trait related to glycemic status on baseline BMI.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 7}]", "id": "TRAIT INTERACTION"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "This SNP within the LEPR gene interacts significantly with FPG levels affecting baseline BMI.)", "entity_type": "UNKNOWN", "id": "FPG, BASELINE BMI"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "This SNP within the LEPR gene interacts significantly with FPG levels affecting baseline BMI.)", "entity_type": "UNKNOWN", "id": "RS1011527, LEPR GENE"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "The genetic variants, GRS of these SNPs interact with traits related to glycemic status to influence BMI.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}]", "id": "RS9939609, RS2984618, RS7903146, GRS, LEPR GENE, FPG"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "The genetic factors are not associated with transitions in BMI, suggesting no simultaneous main effect and interaction between genetic determinants and glycemic status change.)", "entity_type": "UNKNOWN", "id": "BMI TRANSITION"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "The genetic factors are not associated with transitions in BMI, suggesting no simultaneous main effect and interaction between genetic determinants and glycemic status change.)", "entity_type": "UNKNOWN", "id": "SNP(S), GRS"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "The interaction between these two SNPs with a trait related to glycemic status influences BMI at baseline.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 7}]", "id": "RS1011527, RS9939609"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "The SNP\'s interaction with fasting plasma glucose (FPG) levels affects BMI at baseline.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITH FPG"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "The SNP\'s combined interaction with FPG levels impacts baseline BMI.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITH FPG AND BASELINE BMI"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "This interaction between SNP, FPG levels, and baseline BMI showcases a complex relationship in influencing obesity markers.)", "entity_type": "UNKNOWN", "id": "FPG AND BASELINE BMI INTERACTION"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "This combination highlights a specific pathway by which obesity SNPs interact with metabolic markers to affect BMI at baseline and its transition.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITH FPG AND BASELINE BMI LEVEL"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "The SNP\'s involvement with FPG levels and baseline BMI demonstrates its impact on obesity markers through complex interplay.)", "entity_type": "UNKNOWN", "id": "FPG AND BASELINE BMI INTERACTION EFFECT"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "The SNP\'s interaction with fasting plasma glucose (FPG) directly affects baseline BMI.)", "entity_type": "UNKNOWN", "id": "BASELINE BMI INFLUENCE"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "The SNP\'s interaction with fasting plasma glucose (FPG) directly affects baseline BMI.)", "entity_type": "UNKNOWN", "id": "RS1011527, FPG"}, {"entity_type": "GENE", "description": "Impairment on the battery was related to two polymorphisms: MTHFR 1298A>C (p=0.03) and MS 2756A>G (p=0.05).<SEP>Survivors with the MTHFR 1298AC/CC genotypes had a 3.0-fold increased risk of impairment.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "clusters": "[{level: 0, cluster: 6}]", "id": "MTHFR 1298A>C"}, {"entity_type": "GENE", "description": "Impairment on the battery was related to two polymorphisms: MTHFR 1298A>C (p=0.03) and MS 2756A>G (p=0.05).<SEP>Survivors with the MS 2756AA genotype were 3.8 times as likely to meet criteria for impairment.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "clusters": "[{level: 0, cluster: 6}]", "id": "MS 2756A>G"}, {"entity_type": "CHILD PERFORMANCE MEASURE", "description": "Survivors with the MTHFR 1298AC/CC genotypes performed on average 7 points lower on TMTA.<SEP>TMTA (Trail Making Test A) is a measure for assessing attention and processing speed in cognitive function.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "clusters": "[{level: 0, cluster: 6}]", "id": "TMTA"}, {"entity_type": "CHILD PERFORMANCE MEASURE", "description": "Survivors with the MS 2756AA genotype scored on average 12.2 points lower on TMTA and 5.3 points lower on TMTB.<SEP>TMTB likely refers to another part or variant of the Trail Making Test, assessing distinct cognitive functions like attention and coordination.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "clusters": "[{level: 0, cluster: 6}]", "id": "TMTB"}, {"entity_type": "NEUROCOGNITIVE TEST", "description": "For survivors with the MS 2756AA genotype, the mean PEGNDH score was 11.7 points lower than that of individuals with the MS 2756AG/GG genotypes.<SEP>The PEGDH enzyme interacts with other metabolites or enzymes, indicating its role in a specific metabolic pathway.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "clusters": "[{level: 0, cluster: 5}]", "id": "PEGDH"}, {"entity_type": "NEUROCOGNITIVE TEST", "description": "Survivors with the MTRR 66AA genotype scored on average 10.7 points lower on PEGNDH.<SEP>The PEGNDH enzyme also interacts with molecules, contributing to the understanding of metabolic processes and pathways.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "clusters": "[{level: 0, cluster: 5}]", "id": "PEGNDH"}, {"entity_type": "NEUROCOGNITIVE TEST", "description": "CFL (Cellular Localization) is involved in the localization of molecules within cellular structures, impacting various biological functions.<SEP>Survivors with the MTRR 66AA genotype also scored 10.5 points lower on average on CFL, compared with survivors with the MTRR 66AG/GG genotypes.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "clusters": "[{level: 0, cluster: 1}]", "id": "CFL"}, {"entity_type": "DIETARY PATTERN/ENVIRONMENTAL FACTOR/DRUG/THERAPY", "description": "MTX (methotrexate) is a medication often used for cancer treatment.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "MTX"}, {"entity_type": "ASSESSMENT TOOL/TEST/MEASURE", "description": "The DIVERGT battery evaluates long-term cognitive outcomes in survivors of childhood cancer.<SEP>This battery consists of multiple tests designed to evaluate various dimensions of cognition and neurological function.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "DIVERGT BATTERY"}, {"entity_type": "SUBJECT POPULATION/STUDY PARTICIPANTS", "description": "The study population comprised of individuals who survived a diagnosis of ALL.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "SURVIVORS"}, {"entity_type": "GENETIC POLYMORPHISM", "description": "Specific genotypes related to the TSER (thymidylate synthase) gene were identified in this study.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "2R/3R AND 3R/3R GENOTYPES"}, {"entity_type": "GENETIC POLYMORPHISM/VARIANT", "description": "Polymorphisms in genes involved in methylation and folate metabolism (MTHFR and SHMT) were also assessed.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "677C>T, SHMT 1520C>T"}, {"entity_type": "GENETIC POLYMORPHISM/VARIANT", "description": "A variant within the SHMT gene was analyzed in relation to cognitive outcomes.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "1420C>T"}, {"entity_type": "INTERVENTION/TREATMENT", "description": "The number of intrathecal methotrexate (MTX) doses received by survivors was one covariate in the study.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "INTRATHECAL MTX DOSES"}, {"entity_type": "EXPOSURE/DOSE EXPOSURE", "description": "Cumulative exposure to intravenous methotrexate was another covariate examined.<SEP>IV MTX (Medication) exposure over time, specifically via intravenous administration, could influence neurological responses.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "CUMULATIVE IV MTX EXPOSURE"}, {"entity_type": "DISEASE", "description": "Early MTX-related neurotoxicity refers to a condition occurring before the full manifestation of adverse effects.<SEP>This indicates a potential environmental factor affecting neurological health in relation to medication use.", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "clusters": "[{level: 0, cluster: 7}, {level: 1, cluster: 26}]", "id": "EARLY MTX-RELATED NEUROTOXICITY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The DIG molecule engages in various interactions that might be relevant to certain physiological processes or diseases.", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "clusters": "[{level: 0, cluster: 1}]", "id": "DIG"}, {"entity_type": "DATA ANALYSIS", "description": "The data from the genotyping analysis provided insights into genetic predispositions for certain conditions or responses.", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "GENOTYPING RESULTS"}, {"entity_type": "DISEASE", "description": "The Genetic Risk Score is a measure used to assess an individual\'s risk of developing specific diseases based on their genetic makeup.", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "GRS (GENETIC RISK SCORE)"}, {"entity_type": "COVARIATE", "description": "Age at diagnosis is another factor taken into account, potentially impacting disease manifestation and response.", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "AGE AT DIAGNOSIS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Intrathecal exposure to MTX (Medication) could be an environmental or medical intervention affecting neurological outcomes.", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "INTRATHECAL MTX EXPOSURE"}, {"entity_type": "OUTCOME", "description": "Impaired performance on cognitive tests may indicate issues with attention, processing speed, or other neurological functions.", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "IMPAIRED PERFORMANCE"}, {"entity_type": "PROTEIN", "description": "A protein is a macromolecule consisting of amino acid sequences, produced through translation of messenger RNA (mRNA) and usually folded into a complex three-dimensional structure with numerous functional purposes in the body.<SEP>IL-6R\u03b1 protein interacts with genomic variants to influence disease severity in colitis.<SEP>Survivors with a GRS score of \u22656 showed impaired TMTA and TMTB performances, suggesting an association between these protein expression levels and cognitive function in this context.<SEP>The role of proteins encoded by complement pathway genes in influencing AMD risk was also explored.<SEP>NA<SEP>Proteins are large, complex molecules made up of amino acids that perform many vital functions in the body.<SEP>CASP3 protein\'s role is acknowledged in the text.<SEP>Specific proteins related to the context have been omitted.<SEP>The NOS1 protein is involved in cardiovascular function.<SEP>No protein information provided for the associations identified.<SEP>Specific proteins encoded by the aforementioned genes were examined in the analysis.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-a7d0bfa8b16262a1f313b7e3060969c6<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-e8f81fdd25a22582c6ce3d1e15f74af3<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7<SEP>chunk-6a657cd0a000b6df4db8f1305b795021<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-72f4505e43f67f3564607392cb178656<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea<SEP>chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 38}]", "id": "PROTEIN"}, {"entity_type": "NA", "description": "A nutrient is an essential substance that provides energy and sustains life. Nutrients are organic or inorganic substances required by living organisms for growth, maintenance of health, and normal physiological functions.<SEP>The nutritional status or specific intake patterns could affect the performance scores on various cognitive tests by influencing biological markers like GRS.<SEP>NA<SEP>Vitamin D nutrient intake or status may influence the outcomes discussed.<SEP>Specific nutrients related to the context have been omitted.<SEP>Folate is a key nutrient found in leafy greens and plays a role in cardiovascular health.<SEP>No nutrient information provided for the associations identified.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7<SEP>chunk-6a657cd0a000b6df4db8f1305b795021<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-72f4505e43f67f3564607392cb178656<SEP>chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "clusters": "[{level: 0, cluster: 10}, {level: 1, cluster: 33}]", "id": "NUTRIENT"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A molecular interaction is an interaction between two or more molecules, which can involve physical contact or biochemical interactions such as binding or enzymatic reactions.<SEP>IL-1R gene\'s interaction with IL-10R\u03b1Mdel and Rag1 plays a role in modulating both Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 disease severity.<SEP>Interactions between enzymes and their substrates, such as PEGDH and PEGNDH, play crucial roles in metabolic pathways that influence neurological functions.<SEP>This term refers to various interactions among proteins, cells, or other components within the context of disease development and response.<SEP>While M\u03d5 response to IL-10 is critical in attenuating disease, T cells are not a significant source of this IL-10 here.<SEP>The rs4988235 genotype is associated with dairy products including milk consumption.<SEP>The interactions between specific molecules in complement pathway and lifestyle were examined to understand AMD risk.<SEP>NA<SEP>These are interactions between molecules which can occur in biological systems and influence various cellular processes.<SEP>Increased DNA modifications consistently occur with age across LCT and MCM6 in lactase non-persistent CC individuals, while mostly decreasing DNA modification densities are observed for MCM6 and LCT with age in lactase persistent TT individuals.<SEP>Interaction between proteins is pointed out as crucial for certain biological functions.<SEP>Specific molecular interactions related to the context have been omitted.<SEP>The heritability of FVC and FEV1/FVC included both additive and non-additive genetic variance components, indicating molecular interactions between these factors.<SEP>There was an investigation of molecular interactions that could be impacted by genetic factors.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-a7d0bfa8b16262a1f313b7e3060969c6<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-e8f81fdd25a22582c6ce3d1e15f74af3<SEP>chunk-8fb20f873daf79a9b7d6e8d25c8889d6<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-ea45644d73f260cf365527dcc3af86e1<SEP>chunk-770a24a1a31cf671a7681e7c98a7ee91<SEP>chunk-72f4505e43f67f3564607392cb178656<SEP>chunk-35d8a4ea86cb92d18ee1c857a318d2d8<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea<SEP>chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 39}]", "id": "MOLECULAR INTERACTION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Age plays a role in lactase mRNA decreasing with age in lactase non-persistent individuals (CC) but not in intermediate (CT) or lactase persistent (TT) individuals.<SEP>An environmental factor is any aspect of the environment that influences the health and well-being of organisms. This includes both biotic (living) factors and abiotic (non-living) factors.<SEP>Exposure to MTX through both intrathecal and IV routes could have different impacts on early MTX-related neurotoxicity compared to other environmental factors like dietary patterns.<SEP>The environment, including genetic factors like IL-6R\u03b1 gene variants, influences colitis through its impact on T cell responses.<SEP>The effect of environmental factors like smoking or diet on the expression and function of complement pathway genes was analyzed.<SEP>NA<SEP>An environmental factor refers to conditions or influences that exist outside of the organism but affect its development, function, or health.<SEP>Exposure to environmental factor like smoking might impact the health outcomes mentioned.<SEP>Specific environmental factors related to the context have been omitted.<SEP>The evidence for association with SNPs at certain loci was present when the sample was stratified by age, suggesting environmental factor influence.<SEP>Environmental conditions were considered as potential modifiers in their relationship with genotypes.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-a7d0bfa8b16262a1f313b7e3060969c6<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-e8f81fdd25a22582c6ce3d1e15f74af3<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-ea45644d73f260cf365527dcc3af86e1<SEP>chunk-72f4505e43f67f3564607392cb178656<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea<SEP>chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 35}]", "id": "ENVIRONMENTAL FACTOR"}, {"source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45<SEP>chunk-4b6e8f78911cafc70959cfdfed786c29", "description": "Both polymorphisms in these genes were associated with performance on TMTA and TMTB.)<SEP>The MTHFR gene variant, specifically the 677C>T variant, was not associated with male infertility in this study population.", "entity_type": "GENE", "id": "MTHFR 677C>T"}, {"source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "description": "Both polymorphisms in these genes were associated with performance on TMTA and TMTB.)", "entity_type": "UNKNOWN", "id": "SHMT 1520C>T"}, {"source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "description": "Genotype comparison between MS 2756AA survivors and those with TSER 3R/3R or 2R/3R showed differences in certain neurocognitive outcomes.)", "entity_type": "UNKNOWN", "id": "MS 2756AA GENOTYPE"}, {"source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "description": "Genotype comparison between MS 2756AA survivors and those with TSER 3R/3R or 2R/3R showed differences in certain neurocognitive outcomes.)", "entity_type": "UNKNOWN", "id": "TSER 3R/3R GENOTYPES"}, {"source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-a7d0bfa8b16262a1f313b7e3060969c6<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-e8f81fdd25a22582c6ce3d1e15f74af3<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-72f4505e43f67f3564607392cb178656<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea<SEP>chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "description": "A disease is a complex state resulting from pathological changes in an individual\'s body that affects normal functions.<SEP>A disease refers to a state or condition in an individual involving a change from its usual normal structure, function, or process.)<SEP>Diseases such as colitis are influenced by both genetic and environmental factors that include IL-1R and IL-6R\u03b1 genes.<SEP>High GRS scores might indicate an overexpression or altered function of proteins that play key roles in cognitive processes related to MTX neurotoxicity.]<SEP>The exploration was centered around understanding how lifestyle affects susceptibility to AMD, considering genetic predispositions.<SEP>NA<SEP>Presence of cancer disease in patients or risk factors related to it are discussed.<SEP>Specific diseases related to the context have been omitted.<SEP>CD subjects with or without specific genotypes and/or MAPbacteremia show varying physiological responses compared to healthy controls.<SEP>The evidence for associations with SNPs at some loci increased to genome-wide levels of significance after excluding asthmatics, indicating disease association.<SEP>The outcomes related to various diseases or health conditions were studied based on the genetic and environmental factors.", "entity_type": "DISEASE", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 36}, {level: 2, cluster: 60}]", "id": "DISEASE"}, {"entity_type": "GENOMIC VARIANT", "description": "The C risk allele at SLC30A8 rs13266634 was significantly associated with higher fasting proinsulin levels.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "SLC30A8 RS13266634"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Fasting proinsulin levels were significantly decreased after 1 year of DPP intervention in all active groups compared to placebo.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "FASTING PROINSULIN LEVELS"}, {"entity_type": "POPULATION/STUDY GROUP", "description": "The C risk allele at SLC30A8 rs13266634 was associated with higher fasting proinsulin levels at baseline in the DPP cohort.<SEP>The DPP cohort involved participants undergoing a diabetes prevention program.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "clusters": "[{level: 0, cluster: 7}, {level: 1, cluster: 27}]", "id": "DPP COHORT"}, {"entity_type": "METABOLIC PATHWAY", "description": "Baseline fasting insulin was adjusted for, and this adjustment affected the association between SLC30A8 rs13266634 and fasting proinsulin levels.<SEP>Increased fasting insulin levels were noted in women with PCOS compared to control subjects.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f<SEP>chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "FASTING INSULIN"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Fasting glucose was another variable that was adjusted for in addition to baseline fasting insulin when assessing the association with fasting proinsulin levels.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "FASTING GLUCOSE"}, {"entity_type": "MEASURABLE OUTCOME", "description": "The DPP intervention significantly decreased fasting proinsulin levels after 1 year compared to placebo.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "PROINSULIN LEVELS"}, {"entity_type": "DRUG", "description": "At 1 year after DPP interventions, no significant association between SLC30A8 genotype and fasting proinsulin levels remained when adjusted for variables like insulin, age, sex, ethnicity (p=0.86).<SEP>Metformin was one DPP intervention that helped decrease fasting proinsulin levels among participants.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "METFORMIN"}, {"entity_type": "DRUG", "description": "The metformin intervention group had similar proinsulin levels to the troglitazone group at 1 year after DPP interventions.<SEP>The troglitazone group also experienced a decrease in fasting proinsulin levels, albeit not as significantly compared to lifestyle or metformin groups.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "TROGLITAZONE"}, {"entity_type": "STATISTICAL ANALYSIS", "description": "There was no significant genotype \u00d7 treatment interaction suggesting dosage of SLC30A8 risk alleles did not modify effectiveness of treatments.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "GENOTYPE \u00d7 TREATMENT INTERACTION"}, {"entity_type": "TREATMENT STRATEGY", "description": "Randomized DPP intervention included lifestyle, metformin, and troglitazone treatment groups.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "clusters": "[{level: 0, cluster: 7}, {level: 1, cluster: 27}]", "id": "INTERVENTION"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "After 1 year of DPP intervention, all active groups significantly decreased fasting proinsulin levels compared to placebo.)", "entity_type": "UNKNOWN", "id": "INTERVENTION GROUPS"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f<SEP>chunk-22216738ccbae0a6276bb4499018c434", "description": "Drugs are substances used for medical purposes and can have various effects on the body through biological interaction.<SEP>The lifestyle intervention led to the largest decrease in proinsulin levels at 1 year among the different treatment groups.)", "entity_type": "PHARMACEUTICAL COMPOUND", "id": "DRUG"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "At 1 year, there were no significant differences in proinsulin levels between the metformin and troglitazone intervention groups.)", "entity_type": "UNKNOWN", "id": "METFORMIN TREATMENT"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "No genotype-specific effectiveness was observed for any of the treatments evaluated based on SLC30A8 rs13266634 dosage.)", "entity_type": "UNKNOWN", "id": "STATISTICAL RESULT"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "The lifestyle group had a larger decrease in proinsulin levels than either metformin or troglitazone groups, indicating potential superiority.)", "entity_type": "UNKNOWN", "id": "COMPARISON WITH OTHER INTERVENTIONS"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "At 1 year after DPP interventions, the metformin and troglitazone had similar proinsulin levels, suggesting no significant difference in effectiveness between them for this study.)", "entity_type": "UNKNOWN", "id": "COMPARISON WITH TROGLITAZONE"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "The dosage of SLC30A8 risk alleles did not significantly modify the effectiveness of any treatment within the DPP cohort.)", "entity_type": "UNKNOWN", "id": "STATISTICAL FINDING"}, {"entity_type": "DIETARY PATTERN", "description": "A lipogenic diet composed of high carbohydrates that significantly increases PNPLA3 expression.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "HC DIET"}, {"entity_type": "NUTRIENT", "description": "An emulsified form of a lipid, included in HC diets to prevent the induction of PNPLA3 mRNA.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "LE (LIPID EMULSION)"}, {"entity_type": "GENE", "description": "PNPLA3 gene does not response to insulin and glucagon in HepG2 cells.<SEP>The gene whose expression was analyzed and is involved in lipogenesis; mRNA levels were upregulated by HC diet without LE but downregulated with LE inclusion.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a<SEP>chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "PNPLA3"}, {"entity_type": "TRANSCRIPT", "description": "A key transcriptional factor that regulates genes involved in lipid metabolism, showing a similar pattern to PNPLA3 expression.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "SREBP1C"}, {"entity_type": "PROTEIN", "description": "Expression patterns of FAS and ACC-1 were similar to that of PNPLA3.<SEP>Responsible for the formation of malonyl-CoA during de novo fatty acid synthesis and had similar expression patterns with SREBP1c and PNPLA3.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a<SEP>chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "ACC-1"}, {"entity_type": "GENE", "description": "A gene involved in lipolysis; no differences were found among diet groups for this gene.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "PNPLA2 (ATGL)"}, {"entity_type": "GENE", "description": "An enzyme responsible for synthesizing fatty acids during de novo synthesis pathway; showed no significant differences among diet groups.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "FAS (FATTY ACID SYNTHASE)"}, {"entity_type": "ENZYME", "description": "Involved in the transport of acyl-CoA molecules into mitochondria for \u03b2-oxidation and showed reduced activity under HC diet conditions.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "CARNITINE PALMITOYLTRANSFERASE I (CPT1)"}, {"entity_type": "ENZYME", "description": "Key enzyme in gluconeogenesis that converts glucose 6-phosphate to fructose 1,6-bisphosphate; expression was upregulated by HC diet but not significantly compared to LF or HF diet groups.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "G6PASE"}, {"entity_type": "TRANSCRIPTION FACTOR", "description": "A factor that regulates gene expression of glycolytic and gluconeogenic genes; showed a similar pattern of regulation with PNPLA3, SREBP1c, and ACC-1 in response to dietary carbohydrates.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "CHREBP (CYCLOPHILIN E RECEPTOR BINDING PROTEIN)"}, {"entity_type": "PROTEIN", "description": "Regulates energy metabolism by promoting the utilization of glucose and fatty acids; AMPK activity was increased under HC diet conditions compared to LF or HF diets.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "AMPK (ADENOSINE MONOPHOSPHATE ACTIVATED PROTEIN KINASE)"}, {"entity_type": "NUTRIENT", "description": "In studies conducted with 3T3-L1 adipocytes and human adipose tissue, PNPLA3 gene expression is regulated by insulin. However, our data suggests that hepatic PNPLA3 does not respond to insulin stimulation in HepG2 cells.<SEP>Insulin was administered to patient 7 after developing diabetes mellitus five years post-surgery due to ineffective diazoxide therapy.<SEP>Insulin is crucial for glucose metabolism and can be affected by SENP1\'s role.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd<SEP>chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f<SEP>chunk-fc33d6399ec56e65d007269407f1953d", "id": "INSULIN"}, {"entity_type": "HORMONE", "description": "We hypothesized that exercise may stimulate glucagon secretion and thereby decrease PNPLA3 gene expression. However, treatment of HepG2 cells with 100 nM glucagon had no effect on PNPLA3 mRNA expression.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "GLUCAGON"}, {"entity_type": "PHYSICAL ACTIVITY", "description": "Exercise was considered as a possible factor that may stimulate glucagon secretion to decrease PNPLA3 gene expression.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "EXERCISE"}, {"entity_type": "NUTRIENT", "description": "The major fatty acid components of the LE, including linoleic acid (LA), oleic acid (OA), and \u03b1-linolenic acid (ALA), were tested for their effects on PNPLA3 gene expression.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "FATTY ACIDS"}, {"entity_type": "FATTY ACID", "description": "Treatment with either 400 \u03bcM LA suppressed PNPLA3 gene expression in HepG2 cells under both low glucose and high glucose conditions.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "LINOLEIC ACID (LA)"}, {"entity_type": "FATTY ACID", "description": "Treatment with either 400 \u03bcM OA also suppressed PNPLA3 gene expression in HepG2 cells under both low glucose and high glucose conditions.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "OLEIC ACID (OA)"}, {"entity_type": "FATTY ACID", "description": "Treatment with ALA had no effect on PNPLA3 gene expression under either condition.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "\u0391-LINOLENIC ACID (ALA)"}, {"entity_type": "FATTY ACID", "description": "Both EPA and DHA suppressed PNPLA3 gene expression under both low and high glucose conditions.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "EICOSAPENTAENOIC ACID (EPA)"}, {"entity_type": "FATTY ACID", "description": "Both EPA and DHA suppressed PNPLA3 gene expression under both low and high glucose conditions.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "DOCOSAHEXAENOIC ACID (DHA)"}, {"entity_type": "DISEASE", "description": "Expression patterns of FAS and ACC-1 were similar to that of PNPLA3.<SEP>Fetal Alcohol Syndrome", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11<SEP>chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "FAS"}, {"source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "description": "Protein", "entity_type": "UNKNOWN", "id": "DEPLETION-REFEEDING CONDITIONS (SPECIFIC CONTEXT: HEPG2 CELL EXPERIMENTS)|PNPLA3 & CHREBP<|CYCLING BETWEEN GLUCOSE DEPLETION AND REFEEDING PHASES INFLUENCED PNPLA3 EXPRESSION, WITH MODERATE REDUCTIONS AFTER 8 HOURS OF DEPLETION; SHOWED REGULATION THROUGH TRANSCRIPTION FACTORS IN RESPONSE TO METABOLIC STATES.)(ENTITY"}, {"source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "description": "Protein", "entity_type": "UNKNOWN", "id": "LDL RECEPTOR"}, {"source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "description": "Our data suggests that hepatic PNPLA3 does not respond to insulin stimulation in HepG2 cells.)", "entity_type": "UNKNOWN", "id": "HEPG2 CELLS"}, {"entity_type": "GENE", "description": "A gene associated with breast cancer risk was studied, specifically rs1695.<SEP>A gene is a segment of DNA that carries specific genetic information, which when expressed through protein synthesis, encodes for a functional product.<SEP>A total of 916 genes were found to have significant modulation in the hippocampus of male adult rats exposed to PNS, using a fold change threshold of 1.4 and q-value <0.05.<SEP>Another gene analyzed for its role in breast cancer susceptibility was rs1138272.<SEP>SNP rs475862 is identified as part of genetic analysis.<SEP>The CASP3 gene plays a role in the study discussed.<SEP>Specific genes related to the context have been omitted.<SEP>The THSD4 gene is located on chromosome 15q23 and has genome-wide significant associations with FEV1/FVC lung function.<SEP>The power of testing for the ASAH1 gene effect was studied using different kernels.<SEP>The variable meta-regression model involved specific genes.", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e<SEP>chunk-e8f81fdd25a22582c6ce3d1e15f74af3<SEP>chunk-076f6aeeb369546413bc63c9f305553a<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-37088dc7c337929125b7fb31c9bf64cf<SEP>chunk-72f4505e43f67f3564607392cb178656<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea<SEP>chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "GENE"}, {"entity_type": "AXONAL GUIDANCE SIGNALING, PROTEIN KINASE A SIGNALING, GLUTAMATE RECEPTOR SIGNALING, WNT/\u0392-CATENIN SIGNALING, CREB SIGNALING IN NEURONS, SYNAPTIC LONG TERM DEPRESSION, SYNAPTIC LONG TERM POTENTIATION, GLUCOCORTICOID RECEPTOR SIGNALING", "description": "The PNS exposure led to modulation of 49 biological processes in the hippocampus.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "BIOLOGICAL PROCESS"}, {"entity_type": "IL-8 SIGNALING, STAT3 PATHWAY, CDK5 SIGNALING, IL-1 SIGNALING, IL-6 SIGNALING, BMP SIGNALING PATHWAY AND TGF-\u0392 SIGNALING", "description": "The PNS exposure was found to have significant influence on inflammation pathways such as IL-8 Signaling, STAT3 Pathway, etc.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "INFLAMMATION PATHWAY"}, {"entity_type": "68 MIRNAS (47 MATURE MIRNAS)", "description": "A total of 68 miRNAs were significantly modulated in animals exposed to prenatal stress, with the majority being mature miRNAs.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "MIRNAS"}, {"entity_type": "AXONAL GUIDANCE, PROTEIN KINASE-A SIGNALING, GLUCOCORTICOID RECEPTOR SIGNALING, TGF-\u0392 SIGNALING", "description": "The identified miRNAs targeted genes involved in neurodevelopment pathways like Axonal Guidance and Protein Kinase-A Signaling.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "NEURODEVELOPMENT PATHWAYS"}, {"entity_type": "STAT3 PATHWAY, PTEN SIGNALING, ILK SIGNALING, IL-8 SIGNALING", "description": "A combined analysis of mRNA and miRNAs revealed inflammation-related pathways such as STAT3 Pathway. Relationship(", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "INFLAMMATION PATHWAYS"}, {"entity_type": "TRANSCRIPT", "description": "An EPHA4 transcript is differentially modulated as part of neurodevelopment pathways affected by prenatal stress and childhood trauma.<SEP>EPHA4, when modulated, might have interactions with other genes contributing to depression.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182<SEP>chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "EPHA4"}, {"entity_type": "GENE", "description": "ARID5B is another gene that can be modulated in depression and has potential interactions with other genes.<SEP>The ARID5B gene shows a significant modulation in the hippocampus of rats exposed to PNS and in the blood samples from adults exposed to early life trauma.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182<SEP>chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "ARID5B"}, {"entity_type": "TRANSCRIPT", "description": "ISOC1 could potentially influence the modulation of depression-related genes and pathways.<SEP>The ISOC1 transcript is involved in pathways like Wnt/Ca+ pathway, cAMP signaling, CREB signaling that are modulated by prenatal stress and childhood trauma.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182<SEP>chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "ISOC1"}, {"entity_type": "GENE", "description": "HSF2 is associated with changes in gene expression related to depression.<SEP>The HSF2 gene shows significant modulation associated with neurodevelopmental pathways affected by early life events such as prenatal stress and childhood trauma.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182<SEP>chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "HSF2"}, {"entity_type": "TRANSCRIPT", "description": "LDLRAP1, when modulated, might have interactions with other depression-related genes.<SEP>The LDLRAP1 transcript is identified as a common marker that responds to prenatal stress in animals and childhood trauma in humans, indicating potential links between these developmental disruptions.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182<SEP>chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "LDLRAP1"}, {"entity_type": "TRANSCRIPT", "description": "LRRN3 could contribute to modulation of depression in the context of its expression changes.<SEP>LRRN3 transcript modulation implicates its involvement in pathways related to inflammation and neurodevelopment following early life adversities like prenatal stress and trauma.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182<SEP>chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "LRRN3"}, {"entity_type": "GENE", "description": "MYO1D\'s modulations can affect depression, potentially through interactions with other genes.<SEP>The MYO1D gene is differentially expressed, highlighting its role in pathways affected by the combined analysis of mRNA/miRNA in animals exposed to PNS and humans with a history of early life trauma.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182<SEP>chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "MYO1D"}, {"entity_type": "TRANSCRIPT", "description": "A PIK3C2B transcript response suggests its involvement in specific cellular signaling pathways disrupted by prenatal stress and childhood trauma, contributing to the common gene signature between these conditions.<SEP>PIK3C2B plays a role in the modulation of depression-related pathways and gene interactions.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182<SEP>chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "PIK3C2B"}, {"entity_type": "GENE", "description": "The Ferroptosis Factor 1 (Fezf1) shows significant modulation indicating its role in oxidative stress-related pathways that are affected by both prenatal stress and childhood trauma.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "FERROPTOSIS FACTOR 1 (FEZF1)"}, {"entity_type": "PROTEIN", "description": "Brain-Derived Neurotrophic Factor (BDNF), a key protein involved in neural development, exhibits differential modulation as part of neurodevelopmental pathways influenced by prenatal stress and early life adversity.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "NEUROTROPHIC FACTOR BDNF"}, {"entity_type": "TRANSCRIPT", "description": "A GRIK2 transcript response suggests its role in synaptic function and neurotransmission, which are disrupted following prenatal stress and childhood trauma.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "GLUTAMATE RECEPTOR 2 (GRIK2)"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "The Wnt/\u03b2-catenin signaling pathway is identified as modulated by both prenatal stress in animal models and childhood trauma in humans, indicating its importance in neurodevelopmental disorders.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "WNT/\u0392-CATENIN SIGNALING PATHWAY"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "The cAMP (cyclic Adenosine Monophosphate) signaling pathway shows modulation associated with neurodevelopment events such as prenatal stress and childhood trauma, highlighting its role in the regulation of cellular processes under these conditions.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "CAMP SIGNALING PATHWAY"}, {"entity_type": "PATHWAY", "description": "PTEN (Phosphatase and Tensin Homolog) signaling pathway is identified as being modulated by both prenatal stress exposure and early life events like childhood trauma, suggesting its involvement in inflammation and cell growth processes affected under such conditions.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "PTEN SIGNALING"}, {"entity_type": "PATHWAY", "description": "The Integrin-Linked Kinase (ILK) signaling pathway is involved in the modulation caused by prenatal stress and childhood trauma, indicating its role in cellular adhesion and regulation of cytoskeleton dynamics.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "ILK SIGNALING"}, {"entity_type": "GENETIC MARKER", "description": "A unique genetic signature associated with PTSD-like traits has been identified through the analysis of data from rats exposed to PNS and humans exposed to early life trauma, indicating shared genetic vulnerabilities between these groups.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "PTSD-SPECIFIC GENE SIGNATURE"}, {"entity_type": "INTEGRATION OF FINDINGS", "description": "The identification of common molecular mechanisms underlying neurodevelopmental disorders suggests a translational approach for understanding the impact of prenatal stress and childhood trauma on brain function and potentially guiding interventions.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "id": "COMMON MOLECULAR MECHANISMS"}, {"entity_type": "SNP (RS17592371)", "description": "A2M is a gene that can be modulated in depression and has interactions with other genes.<SEP>The SNP rs17592371 found within the FoxO1 gene demonstrates significant GxE interactions with both the Grady Trauma Project and Helsinki Birth cohort studies, linking genetic variation to depression risk through environmental stressors.)(entity<SEP>The SNP rs17592371 located within the FoxO1 gene was significantly associated with emotional stress predicting depressive symptoms, highlighting genetic variation in depression risk.<SEP>The SNP rs17592371 shows significant GxE interaction effects on both cohorts when tested for emotional stress predicting depressive symptoms, emphasizing its importance in depression risk.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "A2M"}, {"entity_type": "GENE", "description": "ARRDC4 is a gene that could possibly be involved in the complex disorder of depression.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "ARRDC4"}, {"entity_type": "GENE", "description": "FBXO32 could potentially play a role in the modulation of depression-related genes and their interactions.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "FBXO32"}, {"entity_type": "GENE", "description": "FoxO1 is activated in hippocampal progenitor stem cells following cortisol exposure, mediating the negative effect of stress on neurogenesis.<SEP>FoxO1 shows activation following cortisol exposure, affecting neurogenesis.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98<SEP>chunk-01cd90d27130aab1a096ad616e05f182", "id": "FOXO1"}, {"entity_type": "GENE", "description": "PPAP2A might be involved in depression through its modulations and potential gene interactions.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "PPAP2A"}, {"entity_type": "GENE", "description": "SAMD12 could have a role in depression, especially if its modulation is relevant.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "SAMD12"}, {"entity_type": "GENE", "description": "SERINC5 has been found to be modulated and potentially interacts with other genes in the context of depression.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "SERINC5"}, {"entity_type": "GENE", "description": "TGF-\u03b21 shows a significant genetic and environmental interaction with emotional stress predicting depressive symptoms, highlighting its role in depression risk through GxE effects.<SEP>TGF-\u03b21, when modulated, could play a role in depression through interactions with other genes.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "TGF-\u03921"}, {"entity_type": "DISEASE", "description": "TGF-\u03b21 modulation might indicate disease states relevant to depression.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "TRANSFORMING GROWTH FACTOR, BETA 1 (TGF-\u03921)"}, {"entity_type": "GENE", "description": "SLC24A3 is a gene that was found to be inversely modulated in both animals and humans, potentially impacting depression.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "SOLUTE CARRIER FAMILY 24 (SLC24A3)"}, {"entity_type": "GENE", "description": "BCL2, when downregulated or upregulated in rats compared to humans, might indicate different mechanisms related to depression.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "B-CELL CLL/LYMPHOMA 2 (BCL2)"}, {"entity_type": "GENE", "description": "BCL9 plays a role in the complex interactions between genes and their modulation across species.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "B-CELL CLL/LYMPHOMA 9 (BCL9)"}, {"entity_type": "GENE", "description": "LEF1 is involved in signaling pathways that might interact with other depression-related genes.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "LYMPHOID ENHANCER-BINDING FACTOR 1 (LEF1)"}, {"entity_type": "GENE", "description": "LIN54 could have interactions with other depression-related genes through its role in complex biological processes.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "LIN-54 DREAM MUVB CORE COMPLEX COMPONENT (LIN54)"}, {"entity_type": "GENE", "description": "PGAP1 might contribute to the modulation of depression, especially when upregulated or downregulated.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "POST-GPI ATTACHMENT TO PROTEINS 1 (PGAP1)"}, {"entity_type": "GENE", "description": "LRRK2 has been associated with the modulation of depression risk and could have potential gene-gene interactions involved in this process.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "LRRK2"}, {"entity_type": "GENE", "description": "MYH7, when modulated, might contribute to depression through its involvement in biological pathways.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "MYH7"}, {"entity_type": "GENE", "description": "NEK9 could potentially influence the modulation of depression-related genes and their interactions.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "NEK9"}, {"entity_type": "GENE", "description": "NR4A1 has been found to be involved in stress response and might have potential gene-gene interactions related to depression.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "NR4A1"}, {"entity_type": "GENE", "description": "PSMA7 could play a role in depression through its modulation, possibly interacting with other genes involved in the disease process.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "PSMA7"}, {"entity_type": "GENE", "description": "SAPKJIP1\'s modulations are associated with depression and might interact with other genes contributing to this condition.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "SAPKJIP1"}, {"entity_type": "GENE", "description": "SIRT3 is involved in redox balance and its modulation could have implications for the modulation of depression-related pathways.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "SIRT3"}, {"entity_type": "GENE", "description": "TIPRL shows upregulation under conditions related to depression, suggesting potential gene-gene interactions.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "TIPRL"}, {"entity_type": "GENE", "description": "TTC3\'s expression is modulated in the context of depression, potentially influencing other genes and pathways.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "TTC3"}, {"entity_type": "GENE", "description": "VPS4A could contribute to modulation patterns observed in depression through its role in cellular processes.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "VPS4A"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "A SNP was detected for respiratory quotient (RQ) during sleep.<SEP>A single nucleotide polymorphism with emotional abuse.<SEP>A variant found in the DNA sequence of a single base in the human genome which was identified as related to coffee consumption.<SEP>The minor allele of each Single Nucleotide Polymorphism (SNP) is related to an inverse association with type 2 diabetes.<SEP>The minor allele frequencies for rs1799853, rs1057910 and rs9923231 were low. (0.02, 0.02, and 0.06 respectively)<SEP>Single Nucleotide Polymorphism, variations in the DNA sequence at a specific position.<SEP>The study examined several Single Nucleotide Polymorphisms (SNPs) including TGF-\u00df1, IL-8, IL-10, and others.<SEP>Another SNP associated with FVC was also found in this gene cluster, at rs6966266.<SEP>One SNP associated with FEV1 was located near the CCL23-CCL18 gene cluster, at rs5740673.<SEP>One SNP from this set had a total possible number of \'low FEV1/FVC alleles\' associated with it.<SEP>SNP rs5649709 at the ASXL3 locus was associated with FVC in this study.<SEP>SNP rs5694874 was associated with FEV1 in this study, near the IL37 gene on chromosome 2q13.<SEP>SNP rs580662 at the PITPNC1 locus was associated with FEV1/FVC lung function.<SEP>SNP rs7658409 at the CHAF1B locus was associated with the FEV1/FVC ratio.<SEP>SNP rs9482635 downstream of the C10orf11 gene had a P-value < 10^-5 with FEV1/FVC lung function.<SEP>Each of the HapMap SNPs was considered as potential causal SNP one-at-a-time in relation to other SNPs and typed SNPs.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-704d1bc6fcda99ac888bd36c954fdb98<SEP>chunk-2e31819d63b80079cb6422efd6e684e0<SEP>chunk-f433e179cd5e32035a0303b9a14da3f1<SEP>chunk-bd81c38d10d8773bc4076cbb20bc289f<SEP>chunk-46227b557f62c59155b1b9cef71d0d17<SEP>chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a<SEP>chunk-4964b2db9e3ddddb4967fb7150e4404d", "clusters": "[{level: 0, cluster: 5}]", "id": "SNP"}, {"entity_type": "CELL TYPE", "description": "Activated by cortisol exposure and involves in mediation of stress-induced effects on neurogenesis.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "HIPPOCAMPAL PROGENITOR STEM CELL"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Stress hormone that activates FoxO1, has negative effects on neurogenesis and regulates glucocorticoid-induced gene expression.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "CORTISOL"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The process in which new neurons are produced from neural stem cells; negatively influenced by stress and cortisol exposure.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "NEUROGENESIS"}, {"entity_type": "STATISTICAL TEST", "description": "Tested the functional relevance of gene by examining its expression under genetic and environmental factors.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "GXE ANALYSIS"}, {"entity_type": "DISEASE STUDY POPULATION", "description": "Group of individuals studied for specific clinical conditions, relevant to understanding gene expression in neurogenesis.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "CLINICAL COHORT"}, {"entity_type": "DISEASE", "description": "Condition where the multipotent human fetal hippocampal progenitor cell line models certain aspects of depression symptoms and stress response.<SEP>There were no group differences in depression, which can be influenced by environmental factors like diet.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98<SEP>chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "DEPRESSION"}, {"entity_type": "CELL CULTURE", "description": "Stable, continuous cell culture used for research purposes including modeling effects of environmental factors on gene expression.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "IMMORTALIZED CELL LINE"}, {"entity_type": "CELL TYPE", "description": "Model system for studying stress-induced changes in gene expression and neurogenesis.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "HUMAN FETAL HIPPOCAMPAL PROGENITOR CELL LINE (HPC0A07/03C)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "FoxO1 mRNA levels increased upon cortisol treatment, indicating a stress response mechanism mediated by this protein.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "STRESS-INDUCED FOXO1 UPREGULATION"}, {"entity_type": "DRUG", "description": "Cell-permeable inhibitor of FoxO1 that counteracts the cortisol-induced upregulation and reduction in neuronal proliferation.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "AS1842856"}, {"entity_type": "MARKER", "description": "Used to measure cellular proliferation, specifically to quantify the number of BrdU+ cells indicating active cell division.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "BRDU"}, {"entity_type": "GENE", "description": "A stress-responsive gene that mediates glucocorticoid effects on brain function and neurogenesis; elevated under stress conditions.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "SGK-1"}, {"entity_type": "TABLE/INFORMATION SOURCE", "description": "Contains details about the single nucleotide polymorphism related to emotional abuse.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "SUPPLEMENTARY TABLE 9"}, {"entity_type": "GENE", "description": "A gene involved in stress response and glucocorticoid-induced effects on brain function, including neurogenesis; elevated under stress conditions.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "SGK-1 (SERUM GLUCOCORTICOID KINASE 1)"}, {"source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "description": "The genes identified through combined mRNA/miRNA analyses were mainly involved in neurodevelopment pathways.", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "NEURODEVELOPMENT"}, {"source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "description": "Out of the 68 miRNAs, 65 were downregulated while only 3 were upregulated after prenatal stress exposure.)", "entity_type": "UNKNOWN", "id": "DOWNREGULATION/UPREGULATION"}, {"source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "description": "Several pathways involved in neurodevelopment and inflammation were found to be similar between the PNS model and humans exposed to early life trauma.", "entity_type": "UNKNOWN", "id": "PATHWAY IN ANIMALS EXPOSED TO PNS"}, {"source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "description": "Several pathways involved in neurodevelopment and inflammation were found to be similar between the PNS model and humans exposed to early life trauma.", "entity_type": "UNKNOWN", "id": "PATHWAY IN HUMANS EXPOSED TO CHILDHOOD TRAUMA"}, {"source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "description": "22 genes were identified as common across both datasets (hippocampus of rats exposed to PNS and blood of adults exposed to childhood trauma). The probability that these genes didn\'t overlap by chance was extremely low, p-value = 1.8 \u00d7 10^-8.", "entity_type": "UNKNOWN", "id": "COMMON GENES"}, {"source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "description": "22 genes were identified as common across both datasets (hippocampus of rats exposed to PNS and blood of adults exposed to childhood trauma). The probability that these genes didn\'t overlap by chance was extremely low, p-value = 1.8 \u00d7 10^-8.", "entity_type": "UNKNOWN", "id": "INTEGRATION OF DATA"}, {"source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "description": "All 22 common genes were validated using real-time PCR both in the hippocampus of animals and in blood samples from humans exposed to childhood trauma. 16 out of these 22 genes showed consistent modulation direction, while 6 showed opposite modulation.", "entity_type": "UNKNOWN", "id": "GENE EXPRESSION IN HIPPOCAMPUS"}, {"source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "description": "All 22 common genes were validated using real-time PCR both in the hippocampus of animals and in blood samples from humans exposed to childhood trauma. 16 out of these 22 genes showed consistent modulation direction, while 6 showed opposite modulation.", "entity_type": "UNKNOWN", "id": "REAL-TIME PCR VALIDATION"}, {"source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "description": "FoxO1 shows significant GxE interactions across both cohorts when exposed to emotional stress, suggesting it plays a crucial role in the development of depressive symptoms in response to such stressors.)", "entity_type": "UNKNOWN", "id": "GXE INTERACTION WITH EMOTIONAL STRESS"}, {"source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "description": "FoxO1\'s gene expression is significantly affected by environmental exposures like emotional and physical abuse, indicating its role in the development of depressive symptoms through GxE interactions.)", "entity_type": "UNKNOWN", "id": "GXE INTERACTION WITH EMOTIONAL ABUSE AND SEXUAL ABUSE"}, {"source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "description": "The cluster formed by A2M, FoxO1 and TGF-\u03b21 indicates potential interactions that are involved in depression through overlapping pathways and signaling systems.)", "entity_type": "UNKNOWN", "id": "CLUSTER OF INTERACTING GENES (A2M, FOXO1, TGF-\u03921)"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98<SEP>chunk-5e709975097a773fb3d31e265d76a71b", "description": "Activation of FoxO1 in response to cortisol exposure, reflecting a mechanism involved in the negative impact of stress on neuronal proliferation.)<SEP>Stress was used as a condition in the experiment to induce anxiety and negative affect.", "entity_type": "ENVIRONMENTAL FACTOR", "id": "STRESS"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "FoxO1 is part of a molecular pathway that regulates the effect of stress on neurogenesis through its activation by cortisol exposure.)", "entity_type": "UNKNOWN", "id": "STRESS RESPONSE MECHANISM"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "AS1842856, as an inhibitor of FoxO1, is used to counteract the upregulation of FoxO1 and its effect on cortisol-induced reduction in neuronal proliferation.)", "entity_type": "UNKNOWN", "id": "AS1842856 (FOXO1 INHIBITOR)"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "AS1842856, as an inhibitor of FoxO1, is used to counteract the upregulation of FoxO1 and its effect on cortisol-induced reduction in neuronal proliferation.)", "entity_type": "UNKNOWN", "id": "DRUG INTERACTION"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "Cortisol plays a role in regulating neurogenesis, particularly under stress conditions, influencing the expression of key genes such as FoxO1 and SGK-1.)", "entity_type": "UNKNOWN", "id": "CORTISOL (STRESS HORMONE)"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "Cortisol plays a role in regulating neurogenesis, particularly under stress conditions, influencing the expression of key genes such as FoxO1 and SGK-1.)", "entity_type": "UNKNOWN", "id": "NEUROGENESIS REGULATION"}, {"entity_type": "GENOMIC VARIANT", "description": "The rs4771122 Genomic Variant was associated with long-term weight loss after bariatric surgery.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "RS4771122"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs477112 was significantly associated with trajectory group membership, nadir percent weight loss, and 2 year percent weight loss.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "RS477112"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Pre-surgical BMI adjustment was made when examining the association between rs4771122 and long-term weight loss.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "PRE-SURGICAL BMI"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Additional copies of the minor (G) allele were associated with a larger percentage weight lost after bariatric surgery.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "ADDITIONAL COPIES OF MINOR (G) ALLELE"}, {"entity_type": "DIETARY PATTERN", "description": "Pre-surgical weight was significantly associated with rs4771122 in sample.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "PRE-SURGICAL WEIGHT"}, {"entity_type": "OUTCOME MEASURE", "description": "The nadir percent weight loss was significantly associated with rs4771122 in the study.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "NADIR WEIGHT LOSS"}, {"entity_type": "OUTCOME MEASURE", "description": "rs4771122 had a significant association with 2-year weight loss in the dataset.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "2 YEAR WEIGHT LOSS"}, {"entity_type": "OUTCOME MEASURE", "description": "Long-term weight loss after bariatric surgery was associated with rs4771122.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "LONG TERM WEIGHT LOSS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Trajectory group membership and long-term weight loss outcomes were influenced by rs4771122.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "TRAJECTORIES"}, {"entity_type": "DISEASE POPULATION", "description": "The study included 52 self-identified black participants for the analysis of rs4771122 association.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "52 SELF-IDENTIFIED BLACKS"}, {"entity_type": "DATA SET ANALYSIS", "description": "Table 1 provided demographic data and weight observations from participants, which was used to analyze rs477112 genotype.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "TABLE 1 DEMOGRAPHICS AND WEIGHT OBSERVATIONS"}, {"entity_type": "TIME FRAME", "description": "Weight observations were made one year or longer after bariatric surgery for the analysis.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "ONE-YEAR OR MORE AFTER SURGERY"}, {"entity_type": "FOLLOW-UP DURATION", "description": "The average length of follow-up after surgery was 5.6 years with a standard deviation of 2.2 years.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "5.6 (2.2) YEARS OF FOLLOW-UP AFTER SURGERY"}, {"entity_type": "TOTAL OBSERVATIONS", "description": "A total of 1,634 weight observations were collected for analysis in the dataset.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "1634 WEIGHT OBSERVATIONS"}, {"entity_type": "STATISTICAL RESULTS", "description": "A table was provided that included statistical significance values (P-values) and effect sizes (beta estimates) from the analysis of SNP associations.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "TABLE INCLUDING P-VALUES AND BETA ESTIMATES"}, {"source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "description": "SNP rs4771122 achieved a low Bonferroni corrected P-value (.05/34 = .0017) for the association with long-term weight loss.)", "entity_type": "UNKNOWN", "id": "BONFERRONI CORRECTED P-VALUE"}, {"source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "description": "Additional copies of the minor (G) allele were significantly associated with higher linear mixed model weights after adjusting for several variables, including pre-surgical BMI.)", "entity_type": "UNKNOWN", "id": "BETA ESTIMATES"}, {"entity_type": "GENE", "description": "GCK gene plays a role in MODY and has variants that cause it.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "GCK"}, {"entity_type": "GENOMIC VARIANT", "description": "Variants of the GCK gene are tested for suspected or confirmed cases of MODY.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "GCK VARIANT"}, {"entity_type": "SEQUENCING METHOD", "description": "An alternative method used to confirm the results obtained from DHRMA.<SEP>Sanger sequencing is used to test for GCK variants in suspected or confirmed MODY cases.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5<SEP>chunk-658df2fc2f9361a65ff1d22464744763", "id": "SANGER SEQUENCING"}, {"entity_type": "DIAGNOSTIC TOOL", "description": "A MODY gene panel includes Sanger sequencing for testing GCK variants.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "MODY GENE PANEL"}, {"entity_type": "INSTITUTION", "description": "Testing at the University of Chicago contributes 72% of tested probands with GCK variants.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "UNIVERSITY OF CHICAGO"}, {"entity_type": "DATA SET", "description": "Clinical information is available for a subset of probands with GCK variants.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "CLINICAL INFORMATION"}, {"entity_type": "CLASSIFICATION", "description": "107 out of 134 identified GCK variants were previously reported as causing the condition.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "PATHOGENIC VARIANT"}, {"entity_type": "NEW ENTITY", "description": "27 out of the total identified variants were new to our knowledge and have not been previously described or reported.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "NOVEL VARIANT"}, {"entity_type": "TYPE OF VARIANT", "description": "17 missense, 1 nonsense, 2 splice sites, 1 in-frame deletion, 1 in-frame duplication, and 5 insertions/deletions were found.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "MISSENSE VARIANT"}, {"entity_type": "SPECIFIC VARIANT", "description": "This variant was identified except for Met210dup which occurred in a single family.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "MET210DUP"}, {"entity_type": "CLASSIFICATION CRITERIA", "description": "Of the novel variants, 26% were found to be pathogenic using ACMG guidelines, while 56% were likely pathogenic and 18% had uncertain significance.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "PATHOGENICITY CLASSIFICATION"}, {"entity_type": "GENETIC PHENOMENON", "description": "Parental or extended family DNA was available for 10 out of 28 probands, and co-segregation with the diabetes phenotype was established in those cases.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "CO-SEGREGATION"}, {"entity_type": "CLASSIFICATION", "description": "Out of five VUS, four were predicted to be damaging by in-silico tools; however, due to lack of segregation data or phenotypic data on family members, these variants did not meet pathogenic or likely pathogenic classification criteria.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "VUS (VARIANT OF UNCERTAIN SIGNIFICANCE)"}, {"entity_type": "EVENT", "description": "The p.Leu307Phe sequence change was presumed to be de novo in the proband due to it being identified only in one individual without a family history of diabetes and not detected in parent\'s DNA via targeted sequencing.<SEP>The spontaneous mutation not inherited from parents, resulting in a new genetic variation.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763<SEP>chunk-b930cdd3706af881cfa771b341af097d", "id": "DE NOVO MUTATION"}, {"entity_type": "GENETIC FEATURE", "description": "The p.His317Gln variant was predicted as tolerated by in-silico tools; however, it was found to have stable, mild fasting hyperglycemia levels and an HbA1c level that is below the expected range for GCK-MODY.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "MODERATELY CONSERVED SEQUENCE CHANGE"}, {"entity_type": "DIAGNOSTIC METHOD", "description": "Targeted sequencing of parents\' DNA did not detect certain variants indicating they might be de novo in probands, which was relevant in cases like p.Leu307Phe.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "TARGETED SEQUENCING"}, {"entity_type": "GENETIC TEST", "description": "Paternity testing was not performed for the proband with the variant p.Leu307Phe, potentially affecting its classification as a pathogenic variant.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "PATERNITY TESTING"}, {"entity_type": "VARIANT OF UNCERTAIN SIGNIFICANCE", "description": "One of five VUS was detected, which might be a damaging variant but lacks sufficient evidence for classification as pathogenic or likely pathogenic.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "P.SER127PRO"}, {"entity_type": "VARIANT OF UNCERTAIN SIGNIFICANCE", "description": "Another variant with uncertain significance due to lack of data on family members and its segregational patterns.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "P.HIS141PRO"}, {"entity_type": "NEW VARIANT", "description": "Identified only in one proband without a family history, it is presumed to be de novo mutation based on available data.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "P.LEU307PHE"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "The GCK gene plays a role in MODY and is associated with diabetes through genetic variants.)", "entity_type": "UNKNOWN", "id": "MODY"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Sanger sequencing serves as a diagnostic tool to test for GCK variants involved in MODY.)", "entity_type": "UNKNOWN", "id": "DIAGNOSTIC METHOD FOR GCK"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "A MODY gene panel includes Sanger sequencing and is used alongside other tools to diagnose MODY through testing for genetic variants.)", "entity_type": "UNKNOWN", "id": "COMPLEMENTING DIAGNOSTIC TOOL"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Testing conducted at the University of Chicago contributes significantly (72%) to the dataset of probands with GCK variants, indicating its importance in genetic research and healthcare.)", "entity_type": "UNKNOWN", "id": "PROVIDER OF TESTING"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "A subset of probands with GCK variants has clinical information available for analysis, providing insights into the manifestation of MODY.)", "entity_type": "UNKNOWN", "id": "DATA SET FOR ANALYSIS"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "ACMG guidelines are used to classify variants as pathogenic or likely pathogenic based on their impact on the GCK gene and its associated conditions.)", "entity_type": "UNKNOWN", "id": "CRITERIA FOR VARIANT EVALUATION"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Segregation analysis establishes co-segregation between tested probands\' DNA and diabetes phenotype, highlighting genetic linkage in MODY cases.)", "entity_type": "UNKNOWN", "id": "CO-SEGREGATION WITH DIABETES PHENOTYPE"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Segregation analysis establishes co-segregation between tested probands\' DNA and diabetes phenotype, highlighting genetic linkage in MODY cases.)", "entity_type": "UNKNOWN", "id": "GENETIC ANALYSIS"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "VUS are not clearly classified as pathogenic due to lack of evidence for their impact or segregation with the disease, requiring further research.)", "entity_type": "UNKNOWN", "id": "CLASSIFICATION CRITERIA"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Certain variants identified in probands might arise spontaneously (de novo), especially those without family history like p.Leu307Phe, impacting classification and understanding of genetic inheritance patterns.)", "entity_type": "UNKNOWN", "id": "GENETIC EVENT"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Targeted sequencing enables detection of specific variants in probands\' DNA that might be missed with whole-genome approaches; crucial for identifying new or rare variants like p.Leu307Phe.)", "entity_type": "UNKNOWN", "id": "DIAGNOSTIC METHOD FOR VARIANTS"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "In cases where genetic data is ambiguous, paternity testing can provide definitive answers on the origin of certain variants; it was not performed in this scenario affecting variant classification.)", "entity_type": "UNKNOWN", "id": "GENETIC ANALYSIS FOR PARENTAL VERIFICATION"}, {"entity_type": "DIETARY PATTERN", "description": "A measure of adherence to the Mediterranean diet.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "MD ADHERENCE"}, {"entity_type": "DIETARY PATTERN", "description": "The dietary pattern characterized by high intake of fruits, vegetables, whole grains, legumes, nuts, and olive oil; moderate consumption of fish and poultry; limited intake of red meat; and moderate alcohol intake.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "MEDITERRANEAN DIET"}, {"entity_type": "METABOLIC MARKER", "description": "Triglyceride concentration in the blood as an indicator of lipid metabolism.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "TAG CONCENTRATIONS"}, {"entity_type": "METABOLIC MARKER", "description": "The ratio between apolipoprotein B and apolipoprotein A-I, which is associated with risk for cardiovascular disease.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "APOB:APOA-1 RATIO"}, {"entity_type": "METABOLIC MARKER", "description": "High-density lipoprotein (HDL) cholesterol level, often referred to as \'good\' cholesterol because it helps remove excess cholesterol from the blood.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "HDL-C"}, {"entity_type": "PROTEIN", "description": "Apolipoprotein A-I is a protein component of HDL cholesterol that is involved in lipid metabolism and transport.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "APOA-1"}, {"entity_type": "METABOLIC MARKER", "description": "A measure of average blood glucose levels over the past 2-3 months, often used to assess long-term control of diabetes.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "HBA1C"}, {"entity_type": "METABOLIC MARKER", "description": "Low-density lipoprotein (LDL) cholesterol concentration in the blood; often referred to as \'bad\' cholesterol because it can contribute to plaque buildup in arteries.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "LDL-C"}, {"entity_type": "PROTEIN", "description": "Apolipoprotein B is a protein component of LDL cholesterol, which plays an important role in lipid metabolism and transport.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "APOB"}, {"entity_type": "GENOMIC VARIANT", "description": "A specific genetic variant found on chromosome 1p32 region, associated with glucokinase regulatory protein.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "GCKR RS780094 SNP"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The wild-type (reference) form of the glucokinase regulatory protein found in most individuals.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "GG GENOTYPE"}, {"entity_type": "DISEASE", "description": "A condition characterized by high levels of triglycerides, often leading to increased cardiovascular risk.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "HYPERTRIACYLGLYCEROLAEMIA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The combined effect of genetic factors (like SNP) and dietary habits on metabolic markers and health outcomes.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "GENOTYPE AND MEDITERRANEAN DIET ADHERENCE"}, {"entity_type": "STATISTICAL METHOD", "description": "A type of data analysis used to examine the effect of categorical variables when considering multiple strata or subpopulations.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "SUBGROUP ANALYSIS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Individual characteristics that may influence metabolic markers and diet adherence interaction.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "AGE, PHYSICAL ACTIVITY STATUS, AND SMOKING STATUS"}, {"entity_type": "GENOMIC VARIANT", "description": "Another genetic marker in high linkage disequilibrium with rs780094, used for subgroup analysis.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "id": "RS1260326 SNP"}, {"entity_type": "HABIT", "description": "Smoking status is a factor that influences various physiological processes including cortisol secretion and metabolism.<SEP>Smoking status refers to being a smoker or non-smoker.", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 21}]", "id": "SMOKING STATUS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "TAG levels are physiological in nature and can vary by genotype.", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 21}]", "id": "TAG LEVELS"}, {"entity_type": "DISEASE", "description": "Outcomes being analyzed are health outcomes which could be affected by smoking status.", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "id": "OUTCOMES"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Strata refer to specific subgroups of data, potentially related to genetic or environmental factors.", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 36}, {level: 2, cluster: 60}]", "id": "STRATA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A gene-diet interaction occurs when the effect of a diet on health outcomes is modified by a person\'s genetic makeup.", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 22}]", "id": "GENE\u2013DIET INTERACTION"}, {"source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "description": "The joint effect of specific genotypes (AA, GA, GG) with different levels of Mediterranean diet adherence influenced various metabolic markers like TAG, TC, apoB, and HbA1c.)", "entity_type": "UNKNOWN", "id": "METABOLIC PARAMETERS"}, {"source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "description": "An analysis that examines how outcomes such as lipid profiles or glucose control vary across subgroups defined by age, activity level, or smoking status.)", "entity_type": "UNKNOWN", "id": "OUTCOME VARIABLES"}, {"source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6<SEP>chunk-5e709975097a773fb3d31e265d76a71b", "description": "Genetic factors can influence how dietary patterns affect physiological processes like TAG levels.)<SEP>The term \'genotype\' refers to an individual\'s genetic makeup, specifically the combination of alleles they carry for certain genes.", "entity_type": "GENE", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 22}]", "id": "GENOTYPE"}, {"entity_type": "DISEASE", "description": "A patient initially misdiagnosed with seizure disorder before hypoglycemia was detected.<SEP>Cirrhosis of the Heart is diagnosed in 65 patients (47 males and 18 females) aged from birth to seven years old. It includes macrosomic individuals with a mean birth weight of 3,690 g.<SEP>The disease presentation varied by age: neonatal in 29 (44.6%), infancy up to one year in 26 patients (40%), and childhood over a year old in 10 patients (15.4%).", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "CHI"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Group 1 patients, 92.3% neonatal form, had no response to diazoxide treatment.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "DIAZOXIDE UNRESPONSIVE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Group 2 patients showed a significantly higher incidence of epilepsy compared with Group 1, and a longer duration from symptom manifestation to diagnosis.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "DIAZOXIDE RESPONSIVE"}, {"entity_type": "GENE", "description": "Mutations found in the ABCC8 gene were identified in 16 children (25% of cohort), none were homozygous, but five were compound heterozygous and ten were heterozygous.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "ABCC8 MUTATIONS"}, {"entity_type": "GENE", "description": "No mutations found on KATP channel gene mutation analysis, only in ABCC8.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "KCNJ11 MUTATIONS"}, {"entity_type": "GENE", "description": "Five novel ABCC8 mutations identified: p.Phe5SerfsX72, p.Gln273ArgfsX85, p.Leu724del, p.Asp1447Gly and IVS 25-1G>T.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "GLUD1 MUTATIONS"}, {"entity_type": "GENE", "description": "A novel heterozygous variant in ABCC8 gene was identified, which is pathogenic or likely pathogenic according to ACMG guidelines.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "HETEROZYGOUS VARIANT"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Among children with compound heterozygous mutations, three responded to diazoxide treatment, one showed no response and the fourth was regulated through diet.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "DIAZOXIDE TREATMENT EFFECTIVENESS"}, {"entity_type": "DISEASE", "description": "One case experienced hypoglycemic brain injury with global developmental delay after brief generalized convulsive periods.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "HYPOGLYCEMIA BRAIN INJURY"}, {"entity_type": "GENOMIC VARIANT", "description": "Five out of the sixteen parents had paternally inherited monoallelic mutations in CHI.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "PATERNALLY INHERITED MONOALLELIC MUTATION"}, {"entity_type": "DISEASE", "description": "One patient, case 3, was diazoxide-unresponsive and underwent surgery that confirmed diffuse pancreatic disease.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "DIFFUSE PANCREATIC DISEASE"}, {"entity_type": "DIAGNOSTIC METHOD", "description": "A focal lesion in the head of the pancreas was identified using an 18F-DOPA PET/CT scan. Histology showed atypical form with \u03b2-cell nuclei enlargement, including the head and part of the body and tail.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "18F-DOPA PET/CT SCAN"}, {"entity_type": "DISEASE", "description": "The patient experienced sustained hypoglycaemic brain injury resulting in global developmental delay.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "GLOBAL DEVELOPMENTAL DELAY"}, {"entity_type": "GENE", "description": "No variants were found in the GCK gene, which is involved in glucose metabolism.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "GCK GENE"}, {"entity_type": "GENE", "description": "Three different heterozygous mutations in the GLUD1 gene were identified in five patients with hypoglycaemia and mild HA.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "GLUD1 GENE"}, {"entity_type": "MUTATION", "description": "The most common mutation was c.3224-3226delACC ins CAGCCAGGAACTG, which resulted in p.Thr1042GlnfsX75.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "P.THR1042GLNFSX75 MUTATION"}, {"entity_type": "MUTATION", "description": "One of the novel mutations identified was a frameshift mutation resulting in p.Phe5SerfsX72.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "P.PHE5SERFSX72 MUTATION"}, {"entity_type": "MUTATION", "description": "Another novel mutation identified was a frameshift mutation leading to p.Gln273ArgfsX85.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "P.GLN273ARGFSX85 MUTATION"}, {"entity_type": "MUTATION", "description": "A deletion mutation was found resulting in p.Leu724del as one of the novel mutations.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "P.LEU724DEL MUTATION"}, {"entity_type": "MUTATION", "description": "Patient 3 carried the missense mutation c.314A>C (p.His105Pro) in exon 3, which was inherited from his father and was previously reported as diazoxide-unresponsive.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "P.HIS105PRO MISSENSE MUTATION"}, {"entity_type": "MUTATION", "description": "Patient 3 had an additional nonsense mutation c.2800C>T (p.Gln934X) in exon 23 inherited from his mother, also responsible for diazoxide-unresponsive phenotype.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "C.2800C>T NONSENSE MUTATION"}, {"entity_type": "GENE", "description": "The GLUD1 gene was detected in patients with hypoglycaemia, HI and mild HA.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "GLUD1"}, {"entity_type": "GENOMIC VARIANT", "description": "This mutation was found in patients 54, 55 and 56.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "P.ARG322 HIS"}, {"entity_type": "GENOMIC VARIANT", "description": "The p.Ser498Leu mutation was found in patient 52.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "P.SER498LEU"}, {"entity_type": "GENOMIC VARIANT", "description": "The p.Asn463Asp mutation was found in patient 53.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "P.ASN463ASP"}, {"entity_type": "DISEASE", "description": "These mutations were identified in patients with HI (Hypoglycaemia Induced by Gluconeogenesis Dysfunction).", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "HI"}, {"entity_type": "DISEASE", "description": "The GLUD1 genetic defect was associated with HA (Hyperammonaemia).", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "HA"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Monitoring serum ammonia concentration helps in assessing the effectiveness of treatment for HI/HA syndrome.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "SERUM AMMONIA CONCENTRATION"}, {"entity_type": "DRUG", "description": "Diazoxide was used as a treatment for patients unresponsive to low-protein diet.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "DIAZOXIDE"}, {"entity_type": "DISEASE", "description": "There were three diazoxide-unresponsive patients who required insulin or did not show improvement after treatment.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "DIAZOXIDE-UNRESPONSIVE"}, {"entity_type": "SURGICAL INTERVENTION", "description": "Subtotal pancreatectomy was performed on nine patients who showed no response to drugs.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": " SUBTOTAL PANCREATECTOMY"}, {"entity_type": "THERAPY", "description": "Diazoxide therapy successfully controlled hypoglycaemia in some cases; however, it had side effects like diarrhoea and hypertrichosis.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "DIAZOXIDE THERAPY"}, {"source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "description": "There were 47 males and 18 females diagnosed with CHI.)", "entity_type": "UNKNOWN", "id": "SEX RATIO"}, {"source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "description": "Group 1 patients, which made up 92.3% of the neonatal form, showed no response to diazoxide treatment.)", "entity_type": "UNKNOWN", "id": "CLINICAL SYMPTOM"}, {"source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "description": "Group 2 patients displayed a higher incidence of epilepsy compared with Group 1.)", "entity_type": "UNKNOWN", "id": "DIAZOXIDE-RESPONSIVE"}, {"source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "description": "Group 2 patients displayed a higher incidence of epilepsy compared with Group 1.)", "entity_type": "UNKNOWN", "id": "SYMPTOM INCIDENCE"}, {"source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "description": "The duration from symptom manifestation to diagnosis was longer in Group 2, particularly for neonatal form patients.)", "entity_type": "UNKNOWN", "id": "TIME DURATION BETWEEN SYMPTOM AND DIAGNOSIS"}, {"source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "description": "Symptoms included hypoglycemia and hypoglycaemic brain injury with global developmental delay.)", "entity_type": "UNKNOWN", "id": "HYPOGLYCEMIA"}, {"source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "description": "Symptoms included hypoglycemia and hypoglycaemic brain injury with global developmental delay.)", "entity_type": "UNKNOWN", "id": "SYMPTOM PRESENTATION"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "The p.Arg322 His mutation was found among patients presenting symptoms of HI.)", "entity_type": "UNKNOWN", "id": "P.ARG322 HIS MUTATION"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "The p.Ser498Leu mutation contributed to the risk of developing HI in one patient.)", "entity_type": "UNKNOWN", "id": "P.SER498LEU MUTATION"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "This mutation was associated with patients experiencing HA (Hyperammonaemia).)", "entity_type": "UNKNOWN", "id": "P.ASN463ASP MUTATION"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Diazoxide treatment for HI/HA syndrome had side effects, including hypertrichosis.)", "entity_type": "UNKNOWN", "id": "DIAZOXIDE TREATMENT"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Diazoxide treatment for HI/HA syndrome had side effects, including hypertrichosis.)", "entity_type": "UNKNOWN", "id": "HI/HA SYNDROME"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "A low-protein diet was used as a primary treatment before considering other interventions.)", "entity_type": "UNKNOWN", "id": "TREATMENT FOR HI/HA"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Patients with no response to diazoxide treatment required further interventions like insulin therapy or surgery.)", "entity_type": "UNKNOWN", "id": "DIAZOXIDE-UNRESPONSIVE PATIENTS"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Patients with no response to diazoxide treatment required further interventions like insulin therapy or surgery.)", "entity_type": "UNKNOWN", "id": "THERAPY OUTCOME"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "The NOS1 gene interacts with folate metabolism to regulate nitric oxide production in cardiovascular health.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 44}]", "id": "NOS1 GENE"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "A specific genetic variant in the NOS1 gene has been linked to an increased risk of developing cardiovascular disease.)", "entity_type": "UNKNOWN", "id": "CARDIOVASCULAR DISEASE RISK"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "A specific genetic variant in the NOS1 gene has been linked to an increased risk of developing cardiovascular disease.)", "entity_type": "UNKNOWN", "id": "GENETIC VARIANT IN NOS1 GENE"}, {"source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb<SEP>chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Dietary patterns influence folate intake, which plays a role in the nitric oxide synthesis pathway affecting cardiovascular health.)<SEP>Specific dietary patterns influence the expression of genes related to cardiovascular disease, including those involved in folate metabolism.", "entity_type": "DIETARY PATTERN", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 45}]", "id": "DIETARY PATTERNS"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Dietary patterns influence folate intake, which plays a role in the nitric oxide synthesis pathway affecting cardiovascular health.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 45}]", "id": "FOLATE INTAKE AND NITRIC OXIDE SYNTHESIS"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Environmental factors such as diet and physical activity modulate cardiovascular health through their effects on the NOS1 gene and folate metabolism.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 10}]", "id": "CARDIOVASCULAR HEALTH MODULATION"}, {"source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb<SEP>chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Environmental factors such as diet and physical activity modulate cardiovascular health through their effects on the NOS1 gene and folate metabolism.)<SEP>Lifestyle and environmental factors interact with genetic variations like MTHFR polymorphisms to impact cardiovascular health.", "entity_type": "ENVIRONMENTAL FACTOR", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 10}]", "id": "ENVIRONMENTAL FACTORS"}, {"entity_type": "GENE", "description": "Levels of KBs across the categories of glucose tolerance.<SEP>The KB metabolism pathway is involved in correlating with the expression of genes such as CPT1A, HMGCS1, HMGCS2, ACAT1, BDH1, OXCT1 and ACSS2.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f<SEP>chunk-bc2193bd869a06007022e1aa75877336", "id": "KBS"}, {"entity_type": "TRANSCRIPT", "description": "In the 2hPG category, AcAc level increased significantly in subjects with IGT by +21% as well as in newly diagnosed diabetes by +29%, compared to reference category.<SEP>In the FPG category, AcAc levels decreased significantly in individuals with IFG by \u22122% and increased significantly in individuals with newly detected diabetes by +64%.<SEP>The C allele of rs780094 had a nominally significant association (P = 0.003) with AcAc levels.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f<SEP>chunk-bc2193bd869a06007022e1aa75877336", "id": "ACAC"}, {"entity_type": "TRANSCRIPT", "description": "BHB level decreased significantly in subjects with IFG by \u22125% and increased significantly in the diabetic range by +99%.<SEP>BHB level increased nominally in IGT by +12% and in newly diagnosed type 2 diabetes by +52%, compared to the reference category.<SEP>The glucose-increasing C allele of rs780094 was associated with an increase of 5.6% per the C allele in BHB levels.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f<SEP>chunk-bc2193bd869a06007022e1aa75877336", "id": "BHB"}, {"entity_type": "NUTRIENT", "description": "Fasting AcAc and BHB levels correlated significantly with FPG and 2hPG levels.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "FASTING ACAC AND BHB"}, {"entity_type": "DISEASE", "description": "A total of 276 participants developed incident diabetes between the baseline and follow-up studies, most of whom had their FPG in the range of 7.0\u20137.5 mmol/L or their 2hPG level (52 of 80 participants) in the range of 11.1\u201312.0 mmol/L.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "HYPERGLYCEMIA"}, {"entity_type": "DISEASE", "description": "After further adjustment for confounding factors, AcAc predicted incident diabetes after the adjustment for confounding factors.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "INCIDENT DIABETES"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The highest quartile of glucose AUC versus the three lowest quartiles of glucose AUC was used to evaluate associations with development of hyperglycemia and conversion to type 2 diabetes.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "INSULIN AUC"}, {"entity_type": "HORMONE", "description": "The incretin hormone GLP-1 is secreted by L-cells in the intestine and plays a critical role in glucose homeostasis.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "GLP-1"}, {"entity_type": "TRANSPORTER", "description": "Sodium-glucose co-transporter 2 (SGLT2) is responsible for reabsorbing most of the filtered glucose back into the bloodstream, maintaining blood sugar levels.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "SGLT2"}, {"entity_type": "DISEASE", "description": "Type 2 diabetes mellitus (T2DM), characterized by insulin resistance and inadequate insulin secretion, results in hyperglycemia.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "DIABETES MELLITUS"}, {"entity_type": "PROCESS", "description": "The metabolic processes that regulate blood sugar levels involve the breakdown of glucose and its subsequent utilization or storage.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "METABOLISM"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "IRF1, involved in insulin resistance and susceptibility to type 2 diabetes, is proximal (220kb apart) from the block of SNPs.<SEP>Insulin resistance is a condition where cells fail to respond adequately to insulin, leading to elevated blood sugar levels.<SEP>Liver insulin resistance was detected under basal conditions and increased more pronouncedly under HFD in SENP1 knockout (KO) mice, a hallmark of obesity-linked type 2 diabetes development.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f<SEP>chunk-174831b97ed170039b374ba254d4f506<SEP>chunk-938d8bf2e56584de9dd556be5eb70964", "clusters": "[{level: 0, cluster: 6}]", "id": "INSULIN RESISTANCE"}, {"entity_type": "DIAGNOSTIC TEST", "description": "Fasting plasma glucose (FPG) and 2-hour postprandial plasma glucose (2hPG) tests are used to assess diabetes risk and glucose tolerance.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "GLUCOSE TOLERANCE TESTS"}, {"entity_type": "MEASURE", "description": "The Matsuda index calculates insulin sensitivity based on a person\'s fasting glucose and insulin levels, useful for assessing metabolic health.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "MATSUDA INDEX OF INSULIN SENSITIVITY"}, {"entity_type": "PARAMETER", "description": "The rate at which insulin is secreted by the pancreas in response to rising blood sugar levels, essential for glucose metabolism.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "INSULIN SECRETION RATE"}, {"entity_type": "GENETIC VARIANT", "description": "The glucose-increasing C allele of rs780094, located in the GCKR gene, showed a significant association with elevated levels of BHB.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "RS780094"}, {"entity_type": "GENE", "description": "Nominally significant associations were found for SNPs from the FADS1 gene and KB levels, both in terms of AcAc and BHB.<SEP>Polymorphisms in FADS1, including rs35357048 and rs174537, influence omega-6 and omega-3 fatty acid levels.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-bc2193bd869a06007022e1aa75877336", "id": "FADS1"}, {"entity_type": "GENE", "description": "SNPs from the ANK1 gene showed a nominally significant association with KB levels.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "ANK1"}, {"entity_type": "GENE", "description": "An exon 2 variant in GIPR on chromosome 16 was related to insulin sensitivity.<SEP>The GIPR gene was associated with KB levels, though not in a statistically significant manner.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-bc2193bd869a06007022e1aa75877336", "id": "GIPR"}, {"entity_type": "GENE", "description": "HMGA2 interacts with tooth eruption and number of teeth, influencing face width.<SEP>The HMGA2 gene showed an association with KB levels that was nominally significant.<SEP>The gene HMGA2 influences number of teeth at one year through its association with rs17101923.<SEP>Variants in the HMGA2 gene contribute significantly to the \'Age at first tooth\' SNP associations.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4<SEP>chunk-c63d60b991cd7442f690dcc364af47dd<SEP>chunk-bc2193bd869a06007022e1aa75877336<SEP>chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "HMGA2"}, {"entity_type": "GENE", "description": "SNPs from the SLC2A2 gene had a nominally significant association with KB levels.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "SLC2A2"}, {"entity_type": "GENE", "description": "GCKR encodes a hexokinase that plays a critical role in regulating glucose metabolism.<SEP>GCKR gene<SEP>Located within or close to the GCKR gene<SEP>The GCKR gene\'s C allele was associated with elevated BHB and AcAc levels in relation to type 2 diabetes or hyperglycemia risk SNPs.<SEP>SNPs (rs7903146, rs7903295) in GCKR on chromosome 3 are associated with fasting glucose and diabetes risk.<SEP>The GCKR gene has been linked to blood glucose levels, impacting kidney health through metabolic pathways.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-ad609cb10e900d42a6901eea1591d8ba<SEP>chunk-45a965c3c3013c70d0fa0f1680283083<SEP>chunk-372184452f1f9156514992455057f843<SEP>chunk-bc2193bd869a06007022e1aa75877336<SEP>chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "GCKR"}, {"entity_type": "GENE", "description": "Adipose tissue mRNA expression of the CPT1A gene (encoding carnitine palmitoyltransferase 1A) was positively correlated with glucose AUC and inversely with Matsuda ISI.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "CPT1A"}, {"entity_type": "GENE", "description": "The HMGCS1 gene, which encodes a protein involved in ketogenesis, did not correlate significantly with glucose metabolism parameters such as glucose AUC, insulin sensitivity or secretion.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "HMGCS1"}, {"entity_type": "GENE", "description": "Similarly, the HMGCS2 gene was also associated with ketogenesis but had no significant correlation with glucose AUC, insulin sensitivity or insulin secretion.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "HMGCS2"}, {"entity_type": "GENE", "description": "The ACAT1 gene (encoding acetyl-CoA acetyltransferase 1) showed significant correlations with glucose AUC, Matsuda ISI and insulin secretion in adipose tissue mRNA expression.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "ACAT1"}, {"entity_type": "GENE", "description": "The BDH1 gene was negatively correlated with glucose AUC and insulin secretion but positively correlated with Matsuda ISI when analyzing adipose tissue mRNA expression.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "BDH1"}, {"entity_type": "GENE", "description": "The OXCT1 gene showed a positive correlation with Matsuda ISI in the context of adipose tissue mRNA expression, suggesting its role in ketolysis and insulin sensitivity.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "OXCT1"}, {"entity_type": "GENE", "description": "The ACSS2 gene encoded by this gene was inversely correlated with glucose AUC and positively correlated with Matsuda ISI based on adipose tissue mRNA levels.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "ACSS2"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Glucose area under the curve (AUC) was negatively correlated with expression levels of genes like CPT1A, BDH1 and ACSS2 but positively correlated with ACAT1 and OXCT1 expression.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "GLUCOSE AUC"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Insulin sensitivity showed positive correlations with the expression of specific genes involved in ketolysis (ACAT1, BDH1, OXCT1) while being negatively correlated with other genes (CPT1A, ACSS2).", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "INSULIN SENSITIVITY"}, {"entity_type": "FUNCTION", "description": "Insulin secretion was inversely related to the expression of CPT1A and ACSS2 but positively correlated with BDH1 and OXCT1 in adipose tissue mRNA levels.<SEP>The glucose-stimulated insulin secretion from SENP1-deficient islets was significantly reduced compared to control groups.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336<SEP>chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "INSULIN SECRETION"}, {"entity_type": "GENE", "description": "SNPs from the PGC1A gene were associated with metabolic processes including glucose metabolism and lipid homeostasis.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "PGC1A"}, {"entity_type": "PROTEIN", "description": "Not affected by the deletion of SENP1 regarding mRNA or protein levels in adipocytes but plays a role in repressing inflammatory responses via SUMOylation.<SEP>The nuclear receptor peroxisome proliferator-activated receptor gamma (PPAR\u03b3) plays a crucial role in adipocyte differentiation and glucose metabolism.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4<SEP>chunk-bc2193bd869a06007022e1aa75877336", "id": "PPAR\u0393"}, {"entity_type": "COACTIVATOR PROTEIN", "description": "The coactivator protein, PGC1\u03b1 (Peroxisome proliferator-activated receptor \u03b3 coactivator 1 alpha), was shown to influence gene expression related to metabolic pathways.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "PGC1\u0391"}, {"entity_type": "PROTEIN", "description": "AMP-activated protein kinase (AMPK) is a key regulator of energy homeostasis and plays a role in glucose metabolism and insulin sensitivity.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "AMPK"}, {"entity_type": "COACTIVATOR PROTEIN", "description": "The coactivator PGC1\u03b2 interacts with several transcription factors, including PGC1\u03b1, to regulate genes involved in mitochondrial biogenesis and metabolic processes.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "PGC1\u0392"}, {"entity_type": "G-PROTEIN COUPLED RECEPTOR", "description": "The GLP-1 receptor (Glucagon-like Peptide 1) is implicated in glucose regulation and insulin secretion, often associated with type 2 diabetes management.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "GLP-1 RECEPTOR"}, {"entity_type": "G-PROTEIN COUPLED RECEPTOR", "description": "GLP-2 receptors play a role in gut health and can influence metabolism through interactions with the GLP-1 system.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "GLP-2 RECEPTOR"}, {"entity_type": "G-PROTEIN COUPLED RECEPTOR", "description": "The GIP receptor (Glucose-dependent insulinotropic polypeptide receptor) is involved in glucose metabolism and insulin secretion.", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "GIP RECEPTOR"}, {"source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "description": "Changes in the Matsuda index could suggest alterations in insulin sensitivity and insulin resistance, impacting diabetes risk assessment.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 6}]", "id": "MATSUDA INDEX"}, {"source_id": "chunk-bc2193bd869a06007022e1aa75877336", "description": "SNPs from the FADS1 gene were associated with KB levels, though not in a statistically significant manner.)", "entity_type": "UNKNOWN", "id": "KB LEVELS"}, {"source_id": "chunk-bc2193bd869a06007022e1aa75877336", "description": "The glucose-increasing C allele of rs780094 was associated with an increase of 5.6% in BHB levels per the C allele, after adjusting for age and BMI.)", "entity_type": "UNKNOWN", "id": "BHB LEVELS"}, {"source_id": "chunk-bc2193bd869a06007022e1aa75877336", "description": "The glucose-increasing C allele of rs780094 was associated with an increase of 5.6% in BHB levels per the C allele, after adjusting for age and BMI.)", "entity_type": "UNKNOWN", "id": "GCKR C ALLELE"}, {"source_id": "chunk-bc2193bd869a06007022e1aa75877336", "description": "The glucose-increasing C allele showed a nominally significant association (P = 0.003) with AcAc levels.)", "entity_type": "UNKNOWN", "id": "ACAC LEVELS"}, {"source_id": "chunk-bc2193bd869a06007022e1aa75877336", "description": "Adipose tissue mRNA expression of the CPT1A gene was positively correlated with glucose AUC, showing its role in fatty acid oxidation and insulin sensitivity.)", "entity_type": "UNKNOWN", "id": "CPT1A EXPRESSION"}, {"source_id": "chunk-bc2193bd869a06007022e1aa75877336", "description": "The BDH1 and OXCT1 genes had positive correlations with Matsuda ISI, reflecting their roles in ketolysis and insulin sensitivity respectively.)", "entity_type": "UNKNOWN", "id": "BDH1 AND OXCT1 EXPRESSION"}, {"source_id": "chunk-bc2193bd869a06007022e1aa75877336", "description": "The BDH1 and OXCT1 genes had positive correlations with Matsuda ISI, reflecting their roles in ketolysis and insulin sensitivity respectively.)", "entity_type": "UNKNOWN", "id": "MATSUDA ISI"}, {"source_id": "chunk-bc2193bd869a06007022e1aa75877336", "description": "The ACAT1 gene encoded acetyl-CoA acetyltransferase 1, which showed significant correlations with glucose AUC, Matsuda ISI and insulin secretion indicating its involvement in ketolysis.)", "entity_type": "UNKNOWN", "id": "ACAT1 EXPRESSION"}, {"source_id": "chunk-bc2193bd869a06007022e1aa75877336", "description": "The ACAT1 gene encoded acetyl-CoA acetyltransferase 1, which showed significant correlations with glucose AUC, Matsuda ISI and insulin secretion indicating its involvement in ketolysis.)", "entity_type": "UNKNOWN", "id": "GLUCOSE AUC AND INSULIN SECRETION"}, {"source_id": "chunk-bc2193bd869a06007022e1aa75877336", "description": "The expression of the CPT1A gene was negatively correlated with glucose AUC but positively related to Matsuda ISI, suggesting roles in fatty acid oxidation and insulin sensitivity respectively.)", "entity_type": "UNKNOWN", "id": "GLUCOSE AUC AND INSULIN SENSITIVITY"}, {"source_id": "chunk-bc2193bd869a06007022e1aa75877336", "description": "The ACSS2 gene showed an inverse correlation with glucose AUC but a positive relationship with Matsuda ISI, pointing to its role in adipose tissue metabolism.)", "entity_type": "UNKNOWN", "id": "ACSS2 EXPRESSION"}, {"entity_type": "GENOMIC VARIANT", "description": "19 SNPs out of 283 independent cardiometabolic SNPs (6.7%) were associated with CRP after correction for multiple testing.<SEP>Additional SNPs out of 283 independent cardiometabolic SNPs associated with CRP after correction for multiple testing.<SEP>SNPs are variations found within these genes that affect protein function.<SEP>Three minor alleles for the SNPs were associated with increased social disinhibition risk, while three others had protective effects.", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba<SEP>chunk-2fb95100a1290b27baae869c6560419a<SEP>chunk-36d70bb8409856300dd77669e7c1f9ed", "id": "SNPS"}, {"entity_type": "GENE", "description": "APOC1 gene<SEP>Located within or close to the APOC1 gene", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba<SEP>chunk-372184452f1f9156514992455057f843", "id": "APOC1"}, {"entity_type": "GENE", "description": "HNF1A gene<SEP>Located within or close to the HNF1A gene", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba<SEP>chunk-372184452f1f9156514992455057f843", "id": "HNF1A"}, {"entity_type": "GENE", "description": "Located within or close to the IL6R gene", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "IL6R"}, {"entity_type": "GENE", "description": "Located within or close to the PPP1R3B gene associated with CRP<SEP>Located within or close to the PPP1R3B gene<SEP>PPP1R3B gene", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba<SEP>chunk-372184452f1f9156514992455057f843", "id": "PPP1R3B"}, {"entity_type": "GENE", "description": "HNF4A gene<SEP>Located within or close to the HNF4A gene associated with CRP<SEP>Located within or close to the HNF4A gene", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba<SEP>chunk-372184452f1f9156514992455057f843", "id": "HNF4A"}, {"entity_type": "GENE", "description": "Located within or close to the PABPC4 gene", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "clusters": "[{level: 0, cluster: 3}]", "id": "PABPC4"}, {"entity_type": "GENE", "description": "BCL7B gene<SEP>Located within BCL7B gene associated with CRP<SEP>Located within or close to the BCL7B gene", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba<SEP>chunk-372184452f1f9156514992455057f843", "clusters": "[{level: 0, cluster: 3}]", "id": "BCL7B"}, {"entity_type": "GENE", "description": "Located near TMEM18", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "TMEM18"}, {"entity_type": "GENE", "description": "Located near PLTP and PCIF1 gene", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "PLTP"}, {"entity_type": "GENE", "description": "Located on chromosome 18, upstream of the MC4R gene", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "MC4R"}, {"entity_type": "GENE", "description": "IL1F10 gene<SEP>Located within or close to the IL1F10 gene and not identified in previous analysis", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba<SEP>chunk-372184452f1f9156514992455057f843", "id": "IL1F10"}, {"entity_type": "GENE", "description": "Located within PABPC gene associated with CRP<SEP>PABPC gene", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba<SEP>chunk-372184452f1f9156514992455057f843", "id": "PABPC"}, {"entity_type": "DISEASE", "description": "Coronary artery disease (CAD) is a large part of the heterogeneity explained by differences in percentage between patients and controls.<SEP>Coronary artery disease (CAD)", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea<SEP>chunk-372184452f1f9156514992455057f843", "id": "CAD"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs1183910", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "RS1183910"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs6734238", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "RS6734238"}, {"entity_type": "DISEASE", "description": "Total cholesterol is a cardiometabolic phenotype", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "TOTAL CHOLESTEROL"}, {"entity_type": "DISEASE", "description": "Type 2 Diabetes (T2D) is one mechanism for loss of renal function that was considered in studies on kidney function and genetic variants.<SEP>Type 2 diabetes (T2D)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391<SEP>chunk-372184452f1f9156514992455057f843", "id": "T2D"}, {"entity_type": "DIETARY PATTERN", "description": "CRP (C-Reactive Protein) gene results showed no significant association with ovarian cancer risk.<SEP>CRP is a biomarker for inflammation and cardiometabolic risk<SEP>CRP is associated with a range of cardiometabolic phenotypes through various biological mechanisms<SEP>SNPs in this gene showed some association with ovarian cancer risk, though not strong enough for a definitive conclusion.<SEP>There was no evidence of CRP gene\'s role in ovarian cancer risk.<SEP>CRP levels increase with diabetes, indicating metabolic disorders related to dietary patterns.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b<SEP>chunk-372184452f1f9156514992455057f843<SEP>chunk-b6ea16f7f7eb8bc6f36f2be03fe62403<SEP>chunk-fc33d6399ec56e65d007269407f1953d", "id": "CRP"}, {"entity_type": "METABOLIC PATHWAY/", "description": "Physiological Process", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "LDL-CHOLESTEROL"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Farnesoid X receptor (FXR) and retinoid X receptor (RXR", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "FXR/RXR ACTIVATION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Liver X receptor (LXR) and retinoid X receptor (RXR", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "LXR/RXR ACTIVATION"}, {"entity_type": "PATHWAY/", "description": "Physiological Process", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "MODY SIGNALING"}, {"source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "description": "The association p-values for 283 cardiometabolic SNPs and serum CRP from a GWAS are evaluated.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 36}, {level: 2, cluster: 59}]", "id": "CARDIOMETABOLIC SNPS"}, {"source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "description": "The association p-values for 283 cardiometabolic SNPs and serum CRP from a GWAS are evaluated.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 36}, {level: 2, cluster: 59}]", "id": "SNPS IN CRP GWAS"}, {"source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "description": "19 SNPs out of the 283 independent cardiometabolic SNPs were associated with CRP after correction for multiple testing.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 37}]", "id": "CRP GWAS SNPS"}, {"source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "description": "Some SNPs showed opposite effects on CRP levels and cardiometabolic phenotypes compared to observations in observational data.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 37}]", "id": "SNPS WITH DIFFERENT EFFECT DIRECTIONS"}, {"source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "description": "The lead SNPs of the 12 pleiotropic loci were mostly in high linkage disequilibrium (LD) with those found in CRP GWAS.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 37}]", "id": "GENOMIC LD"}, {"source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "description": "In some loci the lead SNPs from CRP and cardiometabolic GWAS were different but were in high LD.)", "entity_type": "UNKNOWN", "id": "PLEIOTROPIC LOCI"}, {"source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "description": "In some loci the lead SNPs from CRP and cardiometabolic GWAS were different but were in high LD.)", "entity_type": "UNKNOWN", "id": "SNPS IN CRP GWAS AND CARDIOMETABOLIC GWAS"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "SNP rs4420638 near the APOC1 gene influences T2D risk.)", "entity_type": "UNKNOWN", "id": "RS4420638"}, {"source_id": "chunk-7091ed039203b69ad83d281b22c89579<SEP>chunk-372184452f1f9156514992455057f843", "description": "Body\'s breakdown of fats into fatty acids that are used as energy sources or stored as triglycerides.<SEP>SNP PABPC4 may influence triglyceride levels via its effect on lipid metabolism.)", "entity_type": "METABOLIC PATHWAY", "clusters": "[{level: 0, cluster: 3}]", "id": "TRIGLYCERIDES"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "BCL7B\'s association with HDL cholesterol could be through its role in lipid metabolism.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "HDL-CHOLESTEROL"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "SNP rs4660293 near PABPC4 gene influences triglyceride levels via its impact on lipid metabolism>)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "RS4660293"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "The effect of BCL7B on certain cardiometabolic phenotypes may be mediated through BMI>)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "MEDIATED PLEIOTROPY/MEDIATION"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "SNP rs1260326 near GCKR gene affects CRP levels, and its association with CRP is robust even after adjustment for multiple testing>)", "entity_type": "UNKNOWN", "id": "RS1260326"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "mediation/Mediation", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "LIPIDS/"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "mediation/Mediation", "entity_type": "UNKNOWN", "id": "CRP/"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "mediation/Mediation", "entity_type": "UNKNOWN", "id": "PLTP/MC4R"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "mediated pleiotropy/Mediation", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "BCL7B/FTO/TMEM18"}, {"entity_type": "GENOMIC VARIANT", "description": "SNPs miR-34b/c rs4938723 were genotyped in patients with IA and controls.", "source_id": "chunk-055177812e320344d131432b292111b0", "id": "MIR-34B/C RS4938723"}, {"entity_type": "GENOMIC VARIANT", "description": "SNPs TP53 Arg72Pro was also genotyped in the study.", "source_id": "chunk-055177812e320344d131432b292111b0", "id": "TP53 ARG72PRO"}, {"entity_type": "PATIENTS", "description": "164 patients with IA were enrolled for genotyping of two SNPs.", "source_id": "chunk-055177812e320344d131432b292111b0", "id": "164 PATIENTS WITH IA"}, {"entity_type": "CONTROLS", "description": "A total of 426 controls were used for comparison.", "source_id": "chunk-055177812e320344d131432b292111b0", "id": "426 CONTROLS"}, {"entity_type": "NUMBER OF ANEURYSMS", "description": "The number of aneurysms was recorded for cases enrolled.", "source_id": "chunk-055177812e320344d131432b292111b0", "id": "ANEURYSMS"}, {"entity_type": "GENETIC TRAIT", "description": "CC genotype of miR-34b/c rs4938723 showed a decreased risk for IA compared to TT genotype.", "source_id": "chunk-055177812e320344d131432b292111b0", "id": "MIR-34B/C RS4938723 CC GENOTYPE"}, {"entity_type": "GENETIC TRAIT", "description": "No association was observed between TP53 Arg72Pro genotypes and IA risk.", "source_id": "chunk-055177812e320344d131432b292111b0", "id": "TP53 ARG72PRO GENOTYPES"}, {"entity_type": "GENETIC TRAIT", "description": "The combined genotype reduced the risk for IA when compared to other genotypes.", "source_id": "chunk-055177812e320344d131432b292111b0", "id": "COMBINED GENOTYPE OF MIR-34B/C RS4938723 CC AND TP53 ARG72PRO CG/CC/GG"}, {"entity_type": "GENE", "description": "rs1056513 (G1178S) is a nonsynonymous SNP in INADL on chromosome 1 that attained near genome-wide significance for weight, BMI, fat-free mass (FFM), fat mass (FM), trunk FM, and hip circumference.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "INADL"}, {"entity_type": "GENE", "description": "rs1056513 is a variant in the COL4A1 gene on chromosome 1 that was associated with weight z-score change.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "COL4A1"}, {"entity_type": "GENE", "description": "An intronic variant in MTNR1B on chromosome 11 was strongly associated with fasting glucose.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "MTNR1B"}, {"entity_type": "GENETIC REGION", "description": "Intronic and 3\u2032UTR variants in the APOA5-ZNF259 region on chromosome 11 were associated with triglycerides.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "clusters": "[{level: 0, cluster: 3}]", "id": "APOA5-ZNF259"}, {"entity_type": "GENE", "description": "An intronic variant in PCSK2 on chromosome 20 was associated with total antioxidants.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "PCSK2"}, {"entity_type": "GENETIC REGION", "description": "Variants in the flanking 3\u2032UTR regions of RNASE1 on chromosome 14 were associated with 24-h urinary nitrogen excretion.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "RNASE1"}, {"entity_type": "GENETIC REGION", "description": "An intronic variant in ASS1P11 on chromosome 7 was associated with creatinine excretion.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "ASS1P11"}, {"entity_type": "GENE", "description": "A nonsynonymous SNP (rs3733402; S143T) in GCH1 on chromosome 14 was associated with the 24-h urinary dopamine: creatinine ratio.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "GCH1"}, {"entity_type": "GENE", "description": "A nonsynonymous SNP (rs3733402; S143T) in KLKB1 on chromosome 4 was identified for serum free IGF-1.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "KLKB1"}, {"entity_type": "GENE", "description": "An intronic SNP in MPRIP on chromosome 17 was associated with serum IGFBP-3.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "MPRIP"}, {"entity_type": "GENETIC REGION", "description": "A block of twenty-three SNPs in the flanking 5\u2032UTR region of XPA/FOXE1 on chromosome 9 were highly associated with serum TSH.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "XPA/FOXE1 (TTF-2)"}, {"entity_type": "GENE", "description": "Highly significant associations for a nonsynonymous SNP (rs12075; G42D) and an intronic SNP in DARC on chromosome 1 were identified for MCP-1. Variants in GREB1 on chromosome 2 and DFNB31 on chromosome 9 were also identified for MCP-1.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "DARC"}, {"entity_type": "GENE", "description": "A variant in the 3\u2032UTR for CCR3 was highly associated with MCP1, as well as an intronic SNP in RASGEF1A (p=4.6-9.6E-08).<SEP>Analysis for associations with ovarian cancer did not suggest any risks linked to genotypes from these SNPs.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b<SEP>chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "CCR3"}, {"entity_type": "GENE", "description": "A variant was identified for IL-6 association in the 3\u2032UTR region of ABO.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 55}]", "id": "ABO"}, {"entity_type": "GENE", "description": "An intronic variant in TMEM229B on chromosome 14 was associated with ad libitum energy intake at dinner.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "TMEM229B"}, {"entity_type": "GENE", "description": "An intronic variant in ARHGAP11A on chromosome 15 was associated with sleep duration.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-46227b557f62c59155b1b9cef71d0d17", "id": "ARHGAP11A"}, {"entity_type": "GENE", "description": "A SNP in the intronic region of C21orf34 was detected for respiratory quotient (RQ) during sleep.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-46227b557f62c59155b1b9cef71d0d17", "id": "C21ORF34"}, {"entity_type": "GENE OR GENETIC REGION", "description": "No specific description given for RPL association.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "RPL"}, {"entity_type": "GENE", "description": "rs7289653 (G141C) is a nonsynonymous SNP in ABCA1 on chromosome 10 that was associated with serum cholesterol.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "ABCA1"}, {"entity_type": "GENE", "description": "An intronic variant in PPARG on chromosome 12 was strongly associated with fasting insulin levels.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "PPARG"}, {"entity_type": "GENE", "description": "A nonsynonymous SNP (rs1800541) and an intronic variant were associated with blood pressure.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "ADRB3"}, {"entity_type": "GENE", "description": "rs7041 is a highly significant SNP for HDL cholesterol and triglycerides on chromosome 3.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "LIPC"}, {"entity_type": "GENE", "description": "SNPs (rs429358, rs7412) in the APOE gene were associated with cognition and Alzheimer\'s disease risk.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "APOE"}, {"entity_type": "GENETIC REGION", "description": "Variants in the LIPC-RAPID region on chromosome 3 influenced HDL cholesterol levels.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "LIPC-RAPID 1/2"}, {"entity_type": "GENE", "description": "rs662799 variant is linked to triglycerides and LDL cholesterol.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "APOA4"}, {"entity_type": "GENE", "description": "A variant (rs6450187) in FABP7 on chromosome 1 was found for body weight.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "FABP7"}, {"entity_type": "GENETIC REGION", "description": "Variants near the FTO-RAPID region on chromosome 1 influenced BMI and waist circumference.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "FTO-RAPID 2.1"}, {"entity_type": "GENE", "description": "rs5888 variant is related to omega-6 to omega-3 fatty acid ratio in blood cells.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "FADS2"}, {"entity_type": "GENE", "description": "A variant in KCNQ1 on chromosome 18 was found for fasting glucose levels.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "KCNQ1"}, {"entity_type": "GENE", "description": "Variants near SHOX on chromosome 4 influence growth and skeletal development.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "SHOX"}, {"entity_type": "GENE", "description": "Encodes mu-class glutathione S-transferase, responsible for detoxifying electrophilic compounds by conjugation with glutathione.<SEP>rs3003659 variant is linked to smoking behavior and lung cancer risk.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-382f24e5db6941e2e3db379fe499b223", "id": "GSTM1"}, {"entity_type": "GENETIC REGION", "description": "Variants near TERT on chromosome 5 were associated with age-related diseases like frailty.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "TERT/TOUR"}, {"entity_type": "GENE", "description": "rs749833 variant is related to fasting triglycerides and insulin resistance.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "CPT1B"}, {"entity_type": "GENE", "description": "rs2068533 variant in CIRBP on chromosome 1 was found for BMI.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "CIRBP"}, {"entity_type": "GENE", "description": "SNPs (rs1048710, rs2929201) are associated with alcohol consumption and liver disease risk.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "SLC19A1"}, {"entity_type": "ALLELE OR VARIANT", "description": "The presence of B-allele in the ABO blood group system was found to be related to the risk of coronary artery disease.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "ABO<SUP>*</SUP>1B/F</SUP>B*"}, {"entity_type": "TRANSCRIPT", "description": "A variant was identified for accelerometer-measured light activity in the 5\u2032UTR region of RPL7P3 on chromosome 9.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "RPL7P3"}, {"entity_type": "TRANSCRIPT", "description": "Sedentary-light activity was measured in the 3\u2032UTR of CTCFL on chromosome 20.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "CTCFL"}, {"entity_type": "GENE", "description": "A variant identified for total energy expenditure (p\u200a=\u200a2.7E-08) is located on chromosome 19.<SEP>Variants identified for total energy expenditure are found in the MATK gene on chromosome 19.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "MATK"}, {"entity_type": "GENE", "description": "A highly significant variant for sleeping energy expenditure (p\u200a=\u200a6.0E-08) was identified on chromosome 15.<SEP>A variant that affects sleeping energy expenditure is located in the CHRNA3 gene on chromosome 15.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "CHRNA3"}, {"entity_type": "TRANSCRIPT", "description": "The 5\u2032UTR region of RPL7P3 is part of the transcript and its variants influence total energy expenditure.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "5\u2032UTR REGION OF RPL7P3"}, {"entity_type": "TRANSCRIPT", "description": "The 3\u2032UTR region of CTCFL is part of the transcript and its variants influence sedentary-light activity during sleep.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "3\u2032UTR OF CTCFL"}, {"entity_type": "PHYSIOLOGICAL PARAMETER", "description": "Adjusting for body weight was considered when identifying significant variants related to total energy and sleeping energy expenditure.<SEP>All SENP1-deficient mice had lower body weights after onset of diabetes compared to control mice.<SEP>HFD significantly increased body weight compared to standard chow in both Ctrl and SENP1-aP2KO mice.", "source_id": "chunk-174831b97ed170039b374ba254d4f506<SEP>chunk-a8f2b3b97a70df26efb35263714bf43c<SEP>chunk-46227b557f62c59155b1b9cef71d0d17", "id": "BODY WEIGHT"}, {"entity_type": "INSTRUMENT", "description": "An accelerometer was used to measure light activity during sleep, which influenced the identification of certain genetic variants.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "ACCELEROMETER"}, {"entity_type": "METABOLIC PATHWAY", "description": "Sleeping energy expenditure is a metabolic pathway that might be influenced by specific genetic and environmental factors.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "SLEEPING ENERGY EXPENDITURE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Total energy expenditure represents a physiological process affected by genetics, environment, and lifestyle factors.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "TOTAL ENERGY EXPENDITURE"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "description": "rs1056513 variant is associated with the change in body weight z-score.)", "entity_type": "UNKNOWN", "id": "WEIGHT Z-SCORE"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "description": "An intronic variant of PCSK2 was associated with total antioxidant content.)", "entity_type": "UNKNOWN", "id": "TOTAL ANTIOXIDANTS"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "description": "Variants in the flanking 3\u2032UTR regions of RNASE1 and ASS1P11 were related to nitrogen excretion and creatinine excretion.)", "entity_type": "UNKNOWN", "id": "EXCRETIONS"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "description": "Variants in the flanking 3\u2032UTR regions of RNASE1 and ASS1P11 were related to nitrogen excretion and creatinine excretion.)", "entity_type": "UNKNOWN", "id": "RNASE1, ASS1P11"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "description": "A nonsynonymous SNP (S143T) in GCH1 was associated with the dopamine: creatinine ratio.)", "entity_type": "UNKNOWN", "id": "DOPAMINE: CREATININE RATIO"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "description": "rs3733402 variant is linked to serum free insulin-like growth factor 1 (IGF-1).)", "entity_type": "UNKNOWN", "id": "SERUM FREE IGF-1"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "description": "An intronic SNP in MPRIP correlated with serum IGFBP-3 levels.)", "entity_type": "UNKNOWN", "id": "SERUM IGFBP-3"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "description": "A block of SNPs associated with high serum TSH levels was found near the XPA/FOXE1 gene on chromosome 9.)", "entity_type": "UNKNOWN", "id": "SERUM TSH"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "description": "SNPs in DARC are related to MCP-1 (Monocyte Chemoattractant Protein-1).)", "entity_type": "UNKNOWN", "id": "MCP-1"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "description": "A variant in the 3\u2032UTR region of CCR3 correlated with MCP-1.)", "entity_type": "UNKNOWN", "id": "MCP1"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-b971aa450a9b220806bdc000eb5efaad<SEP>chunk-174831b97ed170039b374ba254d4f506<SEP>chunk-8fb20f873daf79a9b7d6e8d25c8889d6<SEP>chunk-35d8a4ea86cb92d18ee1c857a318d2d8<SEP>chunk-0c1631ae2c195cb68139ea9d75a3b433", "description": "IL-6 (Interleukin-6) is a proinflammatory cytokine found at high levels in PATs of SENP1-aP2KO mice before the onset of diabetes.<SEP>IL-6 expression is significantly enhanced in PATs from SENP1-deficient mice.<SEP>IL-6 is a cytokine that plays crucial roles in inflammation, immune responses, and has been linked to several diseases including colitis.<SEP>In this context, IL-6 is a key protein that plays an essential role in the development and function of Th17 cells. It\'s also involved in mediating the severity of colitis.<SEP>Increased IL-1 and IL-6 from LPM\u03d5s can impact Th17 cell maturation.<SEP>Lesser elevations in IL-6 (3.4\u00b10.5), IL-23 (2.9\u00b10.3) and other pro-inflammatory cytokines were also identified...<SEP>Variants near ABO were significantly related to IL-6 (interleukin-6).)<SEP>Significantly elevated IL-6 was found in SENP1-deficient mice before onset of diabetes at age 7 weeks.", "entity_type": "CYTOKINE", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 55}]", "id": "IL-6"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "description": "An intronic variant in TMEM229B on chromosome 14 was associated with ad libitum energy intake at dinner.)", "entity_type": "UNKNOWN", "id": "ENERGY INTAKE"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-9977706363cc368b3be503d07ed3ef62", "description": "Sleep duration affects snoring prevalence, though specific details are not quantified.<SEP>rs1056513 variant is linked to sleep duration.)", "entity_type": "SLEEP PARAMETER", "id": "SLEEP DURATION"}, {"source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "description": "Intronic variants in C21orf34 were associated with RQ (respiratory quotient) levels during sleep.)", "entity_type": "UNKNOWN", "id": "RESPIRATORY QUOTIENT DURING SLEEP"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "A SNP in the intronic region of C21orf34 detected RQ for respiratory quotient (RQ) during sleep.)", "entity_type": "UNKNOWN", "id": "RESPIRATORY QUOTIENT (RQ) DURING SLEEP"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "A SNP in the intronic region of C21orf34 detected RQ for respiratory quotient (RQ) during sleep.)", "entity_type": "UNKNOWN", "id": "SNP IN C21ORF34"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "A variant identified for total energy expenditure was found in the 5\u2032UTR region of RPL7P3.)", "entity_type": "UNKNOWN", "id": "VARIANT IN 5\u2032UTR REGION OF RPL7P3"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Sedentary-light activity, measured by accelerometer, was detected in the 3\u2032UTR region of CTCFL.)", "entity_type": "UNKNOWN", "id": "ACCELEROMETER-MEASURED LIGHT ACTIVITY"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Sedentary-light activity, measured by accelerometer, was detected in the 3\u2032UTR region of CTCFL.)", "entity_type": "UNKNOWN", "id": "SEDENTARY-LIGHT ACTIVITY IN 3\u2032UTR OF CTCFL"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "A variant identified with highly significance for total energy expenditure is located on chromosome 19.)", "entity_type": "UNKNOWN", "id": "HIGHLY SIGNIFICANT"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "A variant identified with highly significance for total energy expenditure is located on chromosome 19.)", "entity_type": "UNKNOWN", "id": "VARIANT ON CHROMOSOME 19 FOR TOTAL ENERGY EXPENDITURE"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "A highly significant variant for sleeping energy expenditure was found on chromosome 15.)", "entity_type": "UNKNOWN", "id": "HIGHLY SIGNIFICANT VARIANT FOR SLEEPING ENERGY EXPENDITURE"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "A highly significant variant for sleeping energy expenditure was found on chromosome 15.)", "entity_type": "UNKNOWN", "id": "ON CHROMOSOME 15"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Variants in the 3\u2032UTR of CTCFL impact sedentary-light activity measured by an accelerometer.)", "entity_type": "UNKNOWN", "id": "SEDENTARY-LIGHT ACTIVITY DURING SLEEP"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "The variant associated with total energy expenditure is located within the MATK gene on chromosome 19.)", "entity_type": "UNKNOWN", "id": "MATK GENE ON CHROMOSOME 19"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "A significant variant for sleeping energy expenditure was found in the CHRNA3 gene on chromosome 15.)", "entity_type": "UNKNOWN", "id": "CHRNA3 GENE ON CHROMOSOME 15"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Body weight adjustment is crucial when identifying specific genetic variants related to energy expenditure.)", "entity_type": "UNKNOWN", "id": "ADJUSTMENT FOR BODY WEIGHT"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Body weight adjustment is crucial when identifying specific genetic variants related to energy expenditure.)", "entity_type": "UNKNOWN", "id": "IDENTIFICATION OF GENETIC VARIANTS"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "An accelerometer was used in measuring light activity, influencing the identification of genetic variants for different activities during sleep.)", "entity_type": "UNKNOWN", "id": "ACCELEROMETER USE"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "An accelerometer was used in measuring light activity, influencing the identification of genetic variants for different activities during sleep.)", "entity_type": "UNKNOWN", "id": "MEASUREMENT OF LIGHT ACTIVITY DURING SLEEP"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Sleeping energy expenditure is impacted by both genetic factors and environmental considerations.)", "entity_type": "UNKNOWN", "id": "INFLUENCED BY GENETICS AND ENVIRONMENT"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Total energy expenditure is influenced by genetic makeup, environmental conditions, and lifestyle choices.)", "entity_type": "UNKNOWN", "id": "AFFECTED BY GENETICS, ENVIRONMENT, LIFESTYLE"}, {"entity_type": "GENE", "description": "The APOBR gene was screened for mutations in 95 extremely obese children and adolescents.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "APOBR"}, {"entity_type": "GENOMIC VARIANT", "description": "13 sequence variants were found in the coding region of the APOBR gene.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS74949322"}, {"entity_type": "GENOMIC VARIANT", "description": "Variants rs74949322, rs151233, and others were identified in 95 obese children.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS151233"}, {"entity_type": "GENOMIC VARIANT", "description": "The variant p.Glu170 was synonymous and had no functional effect.<SEP>The variant p.Leu22 was synonymous and had no functional effect.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS149271"}, {"entity_type": "GENOMIC VARIANT", "description": "The variant p.Gln553 was synonymous and had no functional effect.<SEP>Variants rs180743 and rs180744 were close to deletion variants with potential effects on function.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS180744"}, {"entity_type": "GENOMIC VARIANT", "description": "The variant p.Pro1012 was synonymous and had no functional effect.<SEP>The variant was located in the un-translated region or intronic regions, changing transcription factor binding sites splicing enhancer and silencer binding sites.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS61738759"}, {"entity_type": "GENOMIC VARIANT", "description": "The variant c.*218C>G led to changes in transcription factor binding sites splicing enhancer and silencer binding sites.<SEP>Variants rs142786317 had no identified functional effects through in silico analysis.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS40833"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The APOBR gene deletion variant affects amino acid sequence, specifically p.Thr321_Gly329del9 and p.Gly369_Asp370del9, disrupting glutamate-rich region.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "APOBR DELETION VARIANT"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Non-synonymous variant rs180743: p.Pro428Ala may have a functional impact due to its position and conservation.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS180743"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The deletion variant of rs368546180 leads to an in-frame shortened amino acid sequence with no predicted function, affecting the repeat region of APOBR.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS368546180"}, {"entity_type": "GENOMIC VARIANT", "description": "The variant p.Gly560 was synonymous and had no functional effect.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS151174"}, {"entity_type": "GENOMIC VARIANT", "description": "The variant p.Ala686 was synonymous and had no functional effect.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS40831"}, {"entity_type": "S1 TABLE", "description": "Analysis results suggest this genomic variant has a \'non-neutral\' functional impact based on SNAP.<SEP>Association studies have linked this genomic variant to particular phenotypic characteristics, highlighting its potential role in genetic disease or trait manifestation.<SEP>Both of these variants are non-coding intronic variants found in the TUFM gene.<SEP>In silico analyses predict functional changes for the SNP, with some being classified as \'non-neutral\' or \'deleterious\'.<SEP>In silico predictions across multiple sources conclude that the variant is unrelated to obesity-related phenotypes.)(relationship<SEP>In-depth analysis across multiple tables suggests implications for transcription factors or splicing regulation within the TUFM gene based on SNPs.<SEP>SNP functional predictions include classifications like \'probably damaging\'.<SEP>The GO terms indicate that this variant might influence specific biological functions or pathways based on its genomic context.<SEP>The LD score analysis shows that this genomic variant contributes to genetic risk factors for diseases or traits based on its co-occurrence with other SNPs.<SEP>The analysis concludes that these variants may affect transcription factors or splicing elements within TUFM.<SEP>The genomic variant affects the transcription regulation of this gene according to computational analyses.<SEP>The genomic variant is located within this gene\'s sequence.<SEP>The in silico analysis shows changes to transcription factor binding sites.<SEP>The infrequent alleles at this variant cause changes of transcription factor binding sites, including splicing enhancer and silencer binding sites.<SEP>The table offers in-depth insights into the predicted functional effects of SNPs, including \'non-neutral\' or \'deleterious\' results.<SEP>The variant is associated with changes in gene expression levels as per eQTL analysis findings.<SEP>The variant is classified as \'deleterious\' by some computational tools, suggesting detrimental effects on function.<SEP>There are in silico predictions that the variant does not have an association with obesity.<SEP>This genomic variant is flagged as having clinical significance in the context of medical diagnostics or treatment strategies, based on its potential impact on health outcomes.<SEP>This genomic variant shows no significant association with obesity based on TDT analysis results.<SEP>This variant is an unknown genomic variant in the 3\' untranslated region of TUFM.<SEP>This variant was detected in a GWAS study, suggesting its possible relevance to specific health outcomes.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "RS142786317: C.*118_*119DELCA"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs61737565 leads to a change in splicing regulator or transcription factor binding sites.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "RS61737565"}, {"entity_type": "GENOMIC VARIANT", "description": "The variant c.922+29C>G results in the alteration of splicing regulator or transcription factor binding sites.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "C.922+29C>G"}, {"entity_type": "GENE", "description": "All five variants found in TUFM are predicted to cause changes in splicing regulator or transcription factor binding sites.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "TUFM"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs8061877 may disrupt a transcription factor binding site, affecting various factors like SRY, HFH 2, and HMG-IY.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "RS8061877"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant leads to changes in splice enhancer binding sites and splicing silencer sites by three programs, as well as alterations in splice sites despite being located in the non-coding 3\u2019 UTR of TUFM.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "clusters": "[{level: 0, cluster: 3}]", "id": "G.28854194C/G"}, {"entity_type": "IN-SILICO TOOL", "description": "TFSearch is used to predict disruptions in transcription factor binding sites for certain genomic variants.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "TFSEARCH"}, {"entity_type": "IN-SILICO TOOL", "description": "Consite also predicts disruptions in transcription factor binding sites, but potentially for different factors compared to TFSearch.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "CONSITE"}, {"entity_type": "IN-SILICO TOOL", "description": "Mutation Taster suggests alterations in splice sites for the variant g.28854194C/G despite its position in the non-coding 3\u2019 UTR of TUFM.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "MUTATION TASTER"}, {"entity_type": "GENE", "description": "Description of GeneX>", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "GENEX"}, {"entity_type": "GENOMIC VARIANT", "description": "Description of Genomic VariantY>", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "GENOMIC VARIANTY"}, {"entity_type": "TRANSCRIPT", "description": "Description of TranscriptZ>", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "TRANSCRIPTZ"}, {"entity_type": "PROTEIN", "description": "Description of ProteinA>", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "PROTEINA"}, {"entity_type": "NUTRIENT", "description": "Description of NutrientB>", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "NUTRIENTB"}, {"entity_type": "DIETARY PATTERN", "description": "Description of Dietary PatternC>", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "DIETARY PATTERNC"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Description of Physiological ProcessD>", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "PHYSIOLOGICAL PROCESSD"}, {"entity_type": "METABOLIC PATHWAY", "description": "Description of Metabolic PathwayE>", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "METABOLIC PATHWAYE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Description of Molecular InteractionF>", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "MOLECULAR INTERACTIONF"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Description of Environmental FactorG>", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "ENVIRONMENTAL FACTORG"}, {"entity_type": "DISEASE", "description": "Description of DiseaseH>", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "DISEASEH"}, {"source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "description": "Variants rs74949322, 151233 and others were found in the study sample.)", "entity_type": "UNKNOWN", "id": "APOBR GENE"}, {"source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "description": "The deletion variant is located near rs3833080, which is in high LD with this variant.)", "entity_type": "UNKNOWN", "id": "APOBR GENE DELETION VARIANT"}, {"source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "description": "The deletion variant is located near rs3833080, which is in high LD with this variant.)", "entity_type": "UNKNOWN", "id": "RS3833080"}, {"source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "description": "A synonymous variant was found within the SULT1A1 gene, having no impact on its function.)", "entity_type": "UNKNOWN", "id": "SULT1A1 GENE"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "All five variants in TUFM are predicted to lead to altered splicing regulator or transcription factor binding sites.)<SEP>In silico analysis is used to show that this variant changes transcription factor binding sites, including splicing enhancer and silencer binding sites.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "S1 TABLE"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "In silico analyses predict functional effects on SNPs, with one being described as \'non-neutral\' or \'deleterious\' for rs10797300: p.Pro19Leu.)<SEP>The analyses provide insights into possible functional impacts of SNPs, including \'non-neutral\' or \'deleterious\' predictions for some.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "TABLE 2"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "In silico predictions show no association between this variant and obesity.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "TABLE 3"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "These two variants are non-coding intronic variants within TUFM.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "RS8061877: C.248\u221218G>A"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "There is no association found between this variant and obesity in the trios tested using TDT analysis.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "TDT"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "Functional predictions for SNPs are given, including \'probably damaging\' for certain polymorphisms.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "POLYPHEN 2"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "In silico prediction tools like SNAP interpret the variant as having a \'non-neutral\' effect.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "SNAP"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "Some in silico tools classify the variant as being \'deleterious\', indicating potential harmful effects.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "SIFT"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "There is a consistent pattern across different analytical tools pointing towards potential functional impacts of the variant in TUFM.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "TABLE 2 AND S1 TABLE"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "Across different analytical data sets, predictions indicate lack of association between this variant and obesity traits.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "TABLE 3 AND S1 TABLE"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea<SEP>chunk-9977706363cc368b3be503d07ed3ef62<SEP>chunk-1f20dbdcc9de443dada1a2045dc7832b<SEP>chunk-1b40d071abac0ba432a9fb0096796f45", "description": "A Genome-wide association study (GWAS) was conducted to identify genetic variants associated with habitual snoring.<SEP>A genome-wide association study (GWAS) identified this variant as potentially linked to certain diseases or traits.)<SEP>A statistical method used to identify genetic variations associated with coffee consumption.<SEP>The GWAS SNP does not have an effect when regressed out from the trans-eQTL analysis of rs181206 on IRF1 and STAT1.", "entity_type": "GENOMIC VARIANT", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 8}]", "id": "GWAS"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "The linkage disequilibrium (LD) score analysis indicates that this variant has predictive power in genetic associations, based on its correlation with other SNPs.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "LD SCORE"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "The GO analysis reveals functional roles and biological pathways associated with this variant, providing insights into its potential impact on cellular processes.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "GENE ONTOLOGY (GO) TERMS"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "This variant has been associated with various phenotypic traits in association studies, contributing to our understanding of its functional effects.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "PHENOTYPE ASSOCIATION STUDIES"}, {"source_id": "chunk-607feda7df4db8151b99560074c91eea", "description": "An assessment of the clinical significance indicates that this variant could have implications for medical conditions, requiring further investigation into its pathogenicity.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 7}]", "id": "CLINICAL SIGNIFICANCE ASSESSMENT"}, {"source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "description": "Variants found in TUFM lead to changes in splicing processes related to alterations in transcription factor binding sites.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 9}]", "id": "GENOMIC VARIANTS"}, {"source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "description": "Disruption of a potential transcription factor binding site by rs8061877 affects various factors like SRY, HFH 2, and HMG-IY.)", "entity_type": "UNKNOWN", "id": "TRANSCRIPTION FACTOR BINDING SITE"}, {"source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "description": "Predicted changes in splice enhancer binding sites and splicing silencer sites for this variant by three programs.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "SPLICE ENHANCER/SPLICE SILENCER BINDING SITES"}, {"source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "description": "The variant g.28854194C/G is situated in the non-coding 3\u2019 UTR of TUFM, but it still affects splicing processes.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "NON-CODING 3\u2019 UTR"}, {"entity_type": "GENE", "description": "Analysis of 162 successfully genotyped SNPs in this gene suggested a risk association with ovarian cancer.<SEP>No significant associations were found at ALOX15 for ovarian cancer.<SEP>No significant associations were observed at SNPs in ALOX15 for ovarian cancer.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b<SEP>chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "id": "ALOX15"}, {"entity_type": "GENE", "description": "Incorporating both SNPs from this gene into regression modeling did not provide additional insight beyond the initial association found with rs1864414.<SEP>The ALOX5 gene was found to be associated with lower risk of ovarian cancer with an odds ratio of 0.77.<SEP>The ALOX5 gene was found to be associated with ovarian cancer risk, with a per-allele p-value of 0.004.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "ALOX5"}, {"entity_type": "GENE", "description": "Results on CCL2 did not show a clear association with ovarian cancer.<SEP>Results on CCL2 were null or inconsistent across study phases.<SEP>Twenty-one SNPs in this gene were found to have an association with the risk of ovarian cancer at a p<0.10 level.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b<SEP>chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "id": "CCL2"}, {"entity_type": "GENE", "description": "No significant associations for IL18 were found in relation to ovarian cancer risk.<SEP>No significant associations for IL18 were observed related to ovarian cancer.<SEP>This gene was one of the 21 SNPs found to have an association with ovarian cancer at p < 0.01 level during analysis.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b<SEP>chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "id": "IL18"}, {"entity_type": "GENE", "description": "SNPs in this gene were significantly associated with risk reduction of ovarian cancer, both in discovery and replication analyses.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL1A"}, {"entity_type": "GENE", "description": "SNPs in this gene were associated with a slightly reduced risk for ovarian cancer, as seen with the rs743507 SNP.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "NOS3"}, {"entity_type": "GENE", "description": "Analysis of PTGS1 showed an association that was marginally significant (p=0.04) during combined analysis.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "PTGS1"}, {"entity_type": "PROTEIN", "description": "This protein\'s SNPs showed a significant inverse association with ovarian cancer risk in specific histological subtypes.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "INTERLEUKIN 1, ALPHA (IL1A)"}, {"entity_type": "PROTEIN", "description": "The ALOX5 gene encodes this enzyme, which was found to be associated with reduced ovarian cancer risk.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "ARACHIDONATE 5-LIPOXYGENASE (ALOX5)"}, {"entity_type": "PROTEIN", "description": "SNPs in the NOS3 gene were found to have a potential association with lower ovarian cancer risk.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "NITRIC OXIDE SYNTHASE 3 (NOS3)"}, {"entity_type": "GENE", "description": "No significant associations were found between genotypes of SNPs in this gene and ovarian cancer risk.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "ALOX12"}, {"entity_type": "GENE", "description": "SNPs in this gene showed no evidence of association with ovarian cancer risk.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "CCL11"}, {"entity_type": "GENE", "description": "Results suggested no significant risk or protective effect of ovarian cancer associated with this gene\'s SNPs.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "CXCL16"}, {"entity_type": "GENE", "description": "Genotyping data did not suggest any strong associations between SNPs in this gene and ovarian cancer risk.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL10"}, {"entity_type": "GENE", "description": "SNPs of this gene were not associated with risk for ovarian cancer, based on the study data.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL15RA"}, {"entity_type": "GENE", "description": "No associations between genotypes of SNPs in this gene and ovarian cancer risk were found.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL1RN"}, {"entity_type": "GENE", "description": "There was no evidence to suggest that these SNPs confer a risk or protective effect for ovarian cancer.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL4R"}, {"entity_type": "GENE", "description": "This gene, more commonly known as COX-2, was considered but found to have no significant associations with ovarian cancer risk.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "PTGS2"}, {"entity_type": "PROTEIN", "description": "SNPs from this gene were marginally associated with ovarian cancer risk, though findings were not consistent across study phases.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "PROSTAGLANDIN-ENDOPEROXIDE SYNTHASE 1 (PTGS1)"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs7596684 in the IL1B gene suggests an association with ovarian cancer risk.", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "clusters": "[{level: 0, cluster: 4}, {level: 1, cluster: 25}]", "id": "IL1B RS7596684"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs2282169 in the PTGS1 gene suggests an association with ovarian cancer risk.", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "clusters": "[{level: 0, cluster: 4}, {level: 1, cluster: 24}]", "id": "PTGS1 RS2282169"}, {"entity_type": "GENE", "description": "Results on IL6 did not indicate a strong link with ovarian cancer risk.<SEP>Results on the IL6 gene were inconclusive or showed no association with ovarian cancer.", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "id": "IL6"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs7596684 in IL1B gene was associated with ovarian cancer.", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "clusters": "[{level: 0, cluster: 4}, {level: 1, cluster: 25}]", "id": "RS7596684"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs2282169 in PTGS1 gene was linked to ovarian cancer risk.", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "clusters": "[{level: 0, cluster: 4}, {level: 1, cluster: 25}]", "id": "RS2282169"}, {"source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "description": "This intronic SNP was associated with reduced ovarian cancer risk, as part of the ALOX5 gene\'s function.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 35}]", "id": "ALOX5 RS1864414"}, {"source_id": "chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-a7d0bfa8b16262a1f313b7e3060969c6<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-e8f81fdd25a22582c6ce3d1e15f74af3<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-72f4505e43f67f3564607392cb178656<SEP>chunk-089bcdf055eb2f007db49dbe2409429b<SEP>chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "description": "A physiological process is any process that occurs in living organisms as part of their functioning, such as metabolism, growth, response to stimuli, or reproduction.<SEP>IL-1R gene\'s response influences the physiological process affecting both Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 diseases.<SEP>This intronic SNP was associated with reduced ovarian cancer risk, as part of the ALOX5 gene\'s function.)<SEP>The impact of lifestyle on physiological processes that are part of the complement pathway\'s function was evaluated.<SEP>NA<SEP>These are processes that occur within organisms at the cellular or organismal level as part of their normal functioning.<SEP>Discussion of cellular responses that are pertinent to the biological context.<SEP>Specific physiological processes related to the context have been omitted.<SEP>The heritability of FEV1 was comprised entirely of additive genetic variance, indicating the role of this process in lung function.", "entity_type": "ADDITIVE GENETIC VARIANCE", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 41}]", "id": "PHYSIOLOGICAL PROCESS"}, {"source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "description": "The A114S missense change in this SNP was linked to a reduced risk for certain ovarian cancer subtypes.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 39}]", "id": "IL1A RS17561"}, {"source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "description": "SNPs in this gene showed a slight risk reduction for ovarian cancer, possibly influenced by dietary patterns.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 10}, {level: 1, cluster: 31}]", "id": "NOS3 RS743507"}, {"source_id": "chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-a7d0bfa8b16262a1f313b7e3060969c6<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-e8f81fdd25a22582c6ce3d1e15f74af3<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-72f4505e43f67f3564607392cb178656<SEP>chunk-089bcdf055eb2f007db49dbe2409429b<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea<SEP>chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "description": "A metabolic pathway consists of a series of chemical reactions that occur within cells and lead to the production of energy and other essential molecules required for life functions.<SEP>The metabolic pathway involving IL-6 responsiveness is clinically important only in Rag1\u2212/\u2212 colitis models, highlighting its role in disease severity.<SEP>The odds of ovarian cancer were marginally increased when considering this SNP, potentially related to metabolic pathways involved.)<SEP>The role of metabolic pathways related to complement proteins and lifestyle was studied.<SEP>NA<SEP>A metabolic pathway is a series of biochemical reactions that transform one set of molecules into another through enzymatic action.<SEP>Caspase metabolic pathway is mentioned in relation to cellular processes.<SEP>Specific metabolic pathways related to the context have been omitted.<SEP>No information provided about metabolic pathways related to the associations.<SEP>The study involved looking into specific metabolic pathways influenced by the genetic variants.", "entity_type": "METABOLIC PATHWAY", "clusters": "[{level: 0, cluster: 4}, {level: 1, cluster: 24}]", "id": "METABOLIC PATHWAY"}, {"source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "description": "A correlated SNP in the ALOX5 gene was also found to be associated with ovarian cancer risk, but its impact was less significant than that of rs1864414.)", "entity_type": "UNKNOWN", "id": "ALOX5 RS745986"}, {"source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "description": "A correlated SNP in the ALOX5 gene was also found to be associated with ovarian cancer risk, but its impact was less significant than that of rs1864414.)", "entity_type": "UNKNOWN", "id": "INTRAGENIC SNPS"}, {"source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "description": "This SNP in the IL1B gene showed association with ovarian cancer risk, though it was not as strong as some other findings.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}]", "id": "SNP ASSOCIATION"}, {"source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "description": "The odds of developing ovarian cancer were slightly elevated by this SNP, but the effect size was small and inconsistent.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 4}, {level: 1, cluster: 24}]", "id": "SNP EFFECT ON RISK"}, {"source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "description": "SNP rs7596684 in the IL1B gene shows a suggestive association with ovarian cancer risk.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 4}, {level: 1, cluster: 25}]", "id": "OVARIAN CANCER RISK"}, {"entity_type": "DISEASE", "description": "Childhood Acute Lymphoblastic Leukemia cases included in the study.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "ALL CASES"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Study recruitment center from which 118 childhood ALL cases were collected, from 2003 to 2010.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "TEXAS CHILDREN\u2019S CANCER CENTER"}, {"entity_type": "GENE", "description": "One of the folate metabolic gene analyzed for tagging SNPs in relation to childhood ALL.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "MTHFD2 GENE"}, {"entity_type": "GENE", "description": "Another folate metabolic gene with identified tagging SNPs associated with childhood ALL.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "BHMT2 GENE"}, {"entity_type": "GENE", "description": "Folate metabolic gene that included the SNP \'rs3918186\', which had a Bayes factor greater than 1 and was linked to an increased risk of childhood ALL.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "NOS3 GENE"}, {"entity_type": "GENE", "description": "Folate metabolic gene that includes the SNP \'rs1051266\', which also showed association with childhood ALL, albeit less evidence compared to NOS3 rs3918186.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "SLC19A1 GENE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "SNP \'rs3918186\' in the NOS3 gene was associated with childhood ALL risk, having a Bayes Factor of 7.38.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "MOLECULAR INTERACTION - NOS3 RS3918186 AND CHILDHOOD ALL RISK"}, {"entity_type": "MOLECULAR INTERACTION", "description": "SNP \'rs1051266\' in the SLC19A1 gene showed less evidence for association with childhood ALL, having a Bayes factor of 7.38.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "BAYES FACTOR - SLC19A1 RS1051266 AND CHILDHOOD ALL RISK"}, {"entity_type": "DISEASE CHARACTERISTIC", "description": "All cases included were under 14 years old when recruited into the study.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "CHILDHOOD AGE RANGE"}, {"entity_type": "GENOMIC VARIANT", "description": "Seven tagging SNPs across different folate metabolic genes that showed inclusion in the initial screening based on posterior probability of greater than 0.2.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "SNPS - INITIAL SCREENING SNPS"}, {"entity_type": "RELATIONSHIP STRENGTH MEASURE", "description": "The Bayes factor for SNP \'rs3918186\' in the NOS3 gene was greater than 1, indicating stronger support in data for its association with childhood ALL risk.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "SNP BAYES FACTOR - NOS3 RS3918186 AND CHILDHOOD ALL RISK"}, {"entity_type": "RELATIONSHIP STRENGTH MEASURE", "description": "The Bayes factor for SNP \'rs1051266\' in the SLC19A1 gene was slightly less than NOS3 rs3918186, indicating somewhat weaker support for its association with childhood ALL risk.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "SNP BAYES FACTOR - SLC19A1 RS1051266 AND CHILDHOOD ALL RISK"}, {"entity_type": "RELATIONSHIP STRENGTH MEASURE", "description": "The odds ratio (OR) of 2.16 for each copy of the minor allele of SNP \'rs3918186\' in the NOS3 gene suggests a nearly twofold increased risk of developing childhood ALL.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "OR - CHILDHOOD ALL RISK AND NOS3 RS3918186"}, {"entity_type": "RELATIONSHIP STRENGTH MEASURE", "description": "The 95% confidence interval (CI) for each copy of the minor allele of SNP \'rs1051266\' in the SLC19A1 gene suggests a range with an increased risk but not as strongly as NOS3 rs3918186.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "CI - CHILDHOOD ALL RISK AND SLC19A1 RS1051266"}, {"entity_type": "DISEASE STUDY CHARACTERISTIC", "description": "The participation rate was 85% for all families involved in the study.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "STUDY PARTICIPATION RATE"}, {"entity_type": "DISEASE STUDY CHARACTERISTIC", "description": "36% of families were complete triads which are unlikely to bias the results due to previous assessments.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "COMPLETE TRIADS PERCENTAGE"}, {"entity_type": "DEMOGRAPHIC - RACE/ETHNICITY", "description": "59 out of 118 cases were non-Hispanic whites, a significant portion of the study population.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "RACIAL/ETHNIC GROUPS - NON-HISPANIC WHITES"}, {"entity_type": "DEMOGRAPHIC - RACE/ETHNICITY", "description": "46 out of 118 cases were Hispanic, representing another sizeable subgroup in the study.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "RACIAL/ETHNIC GROUPS - HISPANICS"}, {"entity_type": "DEMOGRAPHIC - RACE/ETHNICITY", "description": "7 out of the total cases belonged to other racial or ethnic groups, contributing a smaller percentage to the overall population.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "RACIAL/ETHNIC GROUPS - OTHER RACES/ETHNICITIES"}, {"entity_type": "SAMPLE SIZE/POPULATION", "description": "A total of 118 childhood ALL cases from Texas Children\u2019s Cancer Center were included in this study.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "STUDY SAMPLE SIZE"}, {"entity_type": "GENOTYPING METHODOLOGY", "description": "Tagging SNPs for folate metabolic genes were used to identify potential genetic variants associated with childhood ALL.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "SNP IDENTIFICATION PROCESS - TAGGING SNPS"}, {"entity_type": "TEMPORAL CONTEXT", "description": "The study was conducted from 2003 to 2010 at Texas Children\u2019s Cancer Center.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "STUDY PERIOD - 2003-2010"}, {"entity_type": "STUDY PARTICIPATION RATE", "description": "The participation rate for the study period was 85%, with a total of 118 families involved.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "PARTICIPATION RATE - 85%"}, {"entity_type": "FAMILY COMPOSITION ANALYSIS", "description": "36% of the families were complete triads, which is a subgroup that researchers considered unlikely to introduce bias in their findings.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "COMPLETE TRIADS PERCENTAGE - 36%"}, {"entity_type": "DEMOGRAPHIC DIVERSITY", "description": "The study population included individuals from various racial/ethnic backgrounds including non-Hispanic whites, non-Hispanic blacks, Hispanics, and other races/ethnicities.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "RACIAL/ETHNIC DIVERSITY - MULTI-RACIAL POPULATION"}, {"entity_type": "CASE CHARACTERISTIC - AGE RANGE", "description": "All childhood ALL cases in the study were under 14 years old at the time of recruitment.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "AGE OF CASES - UNDER 14 YEARS"}, {"entity_type": "GENE FAMILY", "description": "Four folate metabolic genes (MTHFD2, BHMT2, NOS3, and SLC19A1) were analyzed for potential genetic variants associated with childhood ALL.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "FOLATE METABOLIC GENES - MTHFD2, BHMT2, NOS3, SLC19A1"}, {"entity_type": "GENETIC VARIANT ASSOCIATION", "description": "The SNP \'rs3918186\' in the NOS3 gene was found to have a significant association with an increased risk of developing childhood ALL.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "SNP ASSOCIATION - NOS3 RS3918186"}, {"entity_type": "GENETIC VARIANT ASSOCIATION", "description": "The SNP \'rs1051266\' in the SLC19A1 gene showed marginal association with an increased risk of childhood ALL.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "SNP ASSOCIATION - SLC19A1 RS1051266"}, {"entity_type": "STATISTICAL MEASURE FOR ASSOCIATION", "description": "The Bayes factor of 7.38 indicated strong statistical support for the association between \'rs3918186\' in the NOS3 gene and an increased risk of childhood ALL.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "BAYES FACTOR - NOS3 RS3918186 AND CHILDHOOD ALL RISK (B=7.38)"}, {"entity_type": "STATISTICAL MEASURE FOR ASSOCIATION", "description": "The odds ratio of 2.07 with a confidence interval from 1.25 to 3.46 indicated an association between the minor allele of \'rs1051266\' in the SLC19A1 gene and childhood ALL risk.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "CI INTERVAL - CHILDHOOD ALL RISK AND SLC19A1 RS1051266 (OR=2.07; 95% CI: 1.25-3.46)"}, {"entity_type": "STUDY TYPE", "description": "The study design involved comparing cases with childhood ALL to control subjects without the condition, facilitating the identification of genetic associations.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "CASE-CONTROL STUDY DESIGN"}, {"entity_type": "GENETIC RESEARCH TECHNIQUE", "description": "Tagging SNPs are used as genetic markers to represent a group of common variants across specific regions of chromosomes, aiding in association studies like this one on childhood ALL.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "TAGGING SNPS IN GENOMIC STUDIES"}, {"entity_type": "BIOLOGICAL FUNCTION", "description": "The SLC19A1 gene is involved in the transport of folate, and its tagging SNP (\'rs1051266\') showed evidence for a potential association with childhood ALL.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "SLC19A1 GENE - ROLE IN FOLATE TRANSPORT"}, {"entity_type": "ODDS RATIO INTERPRETATION", "description": "The odds ratio calculation demonstrated that each copy of the minor allele of \'rs3918186\' in the NOS3 gene increased the risk of developing childhood ALL by nearly two times.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "OR CALCULATION - CHILDHOOD ALL RISK AND NOS3 RS3918186 (OR=2.16)"}, {"entity_type": "CONFIDENCE INTERVAL INTERPRETATION", "description": "The confidence interval calculation for the odds ratio provided a range that suggested an increased risk of childhood ALL associated with the minor allele of \'rs1051266\' in the SLC19A1 gene.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "CI CALCULATION - CHILDHOOD ALL RISK AND SLC19A1 RS1051266 (95% CI: 1.51-3.15)"}, {"entity_type": "GENETIC RESEARCH OUTCOME", "description": "The study revealed that genetic variants within the folate metabolism pathway could be potential risk factors for childhood ALL, based on statistical measures like Bayes factor and odds ratio.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "STATISTICAL SIGNIFICANCE OF GENETIC VARIANTS IN FOLATE METABOLISM PATHWAY"}, {"entity_type": "SCIENTIFIC COMMUNITY CONTRIBUTION", "description": "Researchers involved in this study utilized genomic techniques to identify genetic associations with childhood ALL, contributing knowledge on the molecular basis of the disease.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "ROLE OF RESEARCHERS - STUDY CONDUCTERS"}, {"entity_type": "FUTURE INQUIRY FOCUS", "description": "The findings suggest that further research should explore gene-environment interactions to better understand how genetic factors and environmental exposures might jointly influence childhood ALL susceptibility.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "IMPLICATIONS FOR FUTURE RESEARCH - GENE-ENVIRONMENT INTERACTIONS"}, {"source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "description": "SNP \'rs3918186\' in the NOS3 gene was linked with an increased risk of childhood ALL, as indicated by a Bayes factor greater than 1.)", "entity_type": "UNKNOWN", "id": "RELATIONSHIP DESCRIPTION"}, {"source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "description": "SNP \'rs3918186\' in the NOS3 gene was linked with an increased risk of childhood ALL, as indicated by a Bayes factor greater than 1.)", "entity_type": "UNKNOWN", "id": "SNP RS3918186 IN NOS3 GENE AND CHILDHOOD ALL RISK"}, {"source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "description": "SNP \'rs1051266\' in the SLC19A1 gene showed evidence of association with childhood ALL, albeit less strength than NOS3 rs3918186.)", "entity_type": "UNKNOWN", "id": "SNP RS1051266 IN SLC19A1 GENE AND CHILDHOOD ALL RISK"}, {"entity_type": "DIETARY PATTERN", "description": "A dietary pattern consisting of reduced saturated fats and monounsaturated fats, with increased long chain n-3 PUFA intake.<SEP>The diet led to an increase in colon AA concentration among subjects with no minor alleles.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "HEALTHY EATING"}, {"entity_type": "DIETARY PATTERN", "description": "A diet that decreases in saturated fat and n-6 PUFA while increasing monounsaturated fat and long chain n-3 PUFA consumption.<SEP>Maintained fairly constant levels of AA concentrations, unlike the Healthy Eating arm.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "MEDITERRANEAN"}, {"entity_type": "NUTRIENT", "description": "A type of dietary fatty acid that is part of a dietary pattern, particularly decreased in the Healthy Eating group.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "SATURATED FAT (SFA)"}, {"entity_type": "NUTRIENT", "description": "Another type of dietary fatty acid, specifically increased in both diet groups after intervention.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "MONOUNSATURATED FAT (MUFA)"}, {"entity_type": "NUTRIENT", "description": "An omega-3 polyunsaturated fatty acid, significantly increased in the Healthy Eating group and in both groups after intervention.<SEP>Increased significantly (p=0.01) in both Healthy Eating and Mediterranean diets.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "LONG CHAIN N-3 PUFA"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "A specific fatty acid that shows a decrease when following the Mediterranean diet.<SEP>Significantly decreased (p=0.02), which may affect the body\'s nutrient status.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "SERUM 18:2 N-6"}, {"entity_type": "MOLECULAR INTERACTION", "description": "An indicator of dietary balance, increased in both intervention groups but not significantly in the Mediterranean group.<SEP>The ratio increased from 0.72 to 0.77, indicating changes in fat metabolism.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "POLYUNSATURATED FAT: SATURATED FAT RATIO"}, {"entity_type": "NUTRIENT", "description": "An omega-3 fatty acid concentration that increases after following a Mediterranean diet.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "SERUM 18:3 N-3"}, {"entity_type": "METABOLIC PATHWAY", "description": "A measure of omega-6 fatty acids in the colon, influenced by FADS gene cluster genotypes.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "COLONIC AA CONCENTRATION"}, {"entity_type": "DISEASE", "description": "An indicator for potential health issues; lower BMI was found among carriers with minor alleles at baseline.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "BMI (BODY MASS INDEX)"}, {"entity_type": "DISEASE", "description": "An acute-phase protein that may indicate inflammation or infection; it was not significantly affected by genotypes or diets.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "CRP (C-REACTIVE PROTEIN)"}, {"entity_type": "GENE", "description": "A genetic variant plays a role in adjusting counseling goals.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "GENETIC VARIANT"}, {"entity_type": "NUTRIENT", "description": "Increased significantly in the Mediterranean group, contributing to dietary changes.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "MUFA"}, {"entity_type": "NUTRIENT", "description": "A significant increase was observed (p=0.01), impacting fat composition.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "N-3 PUFA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "There were minimal changes, but long chain n-3 PUFA saw a significant increase.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "COLON FATTY ACID CONCENTRATIONS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Significantly increased for colon AA concentrations in the Mediterranean group (p=0.004).", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "20:4, N-6 (AA) CONCENTRATIONS"}, {"entity_type": "NUTRIENT", "description": "Significantly decreased in serum (p=0.02), impacting fatty acid profiles.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "18:2 N-6"}, {"entity_type": "NUTRIENT", "description": "Increased significantly only in the Mediterranean group, suggesting diet impacts on fat metabolism.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "MUFA (MONOUNSATURATED FAT)"}, {"entity_type": "NUTRIENT", "description": "Significantly increased in both dietary groups (p=0.01), reflecting changes associated with dietary patterns.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "N-3 PUFA (POLYUNSATURATED FATTY ACID)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Colon AA concentrations showed significant variation between diet types and genetic variant status.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "AA CONCENTRATIONS"}, {"entity_type": "DIETARY PATTERN", "description": "The intervention led to changes in colon AA concentration levels among subjects without minor alleles, highlighting diet-specific impacts.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "HEALTHY EATING DIET ARM"}, {"entity_type": "DIETARY PATTERN", "description": "Maintained a consistent profile for colon AA concentrations compared to the Healthy Eating arm, indicating differing dietary effects.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "MEDITERRANEAN DIET ARM"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The association between genotypes and serum fatty acid levels was significant for omega-6 fatty acids (20:4), with minor allele carriers showing a lower concentration.)<SEP>The presence of any minor alleles in the FADS gene cluster is significantly associated with a lower serum 20:4, n-6 concentration.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "GENOTYPE GROUP"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The association between genotypes and serum fatty acid levels was significant for omega-6 fatty acids (20:4), with minor allele carriers showing a lower concentration.)<SEP>The presence of any minor alleles in the FADS gene cluster is significantly associated with a lower serum 20:4, n-6 concentration.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "SERUM 20:4, N-6 CONCENTRATION"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "Mean serum 18:3, n-3 concentration for minor allele carriers were estimated to be 21% higher than those with all major alleles.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "SERUM 18:3, N-3 CONCENTRATION"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The presence of any minor alleles had a marginally significant association with the colonic 20:4, n-6 concentration (p = 0.07).)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "COLONIC 20:4, N-6 CONCENTRATION"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "Changes in dietary fat intake significantly impact serum fatty acid composition; reductions and increases were observed in both diet groups after intervention.)", "entity_type": "UNKNOWN", "id": "DIET ARM"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "Changes in dietary fat intake significantly impact serum fatty acid composition; reductions and increases were observed in both diet groups after intervention.)", "entity_type": "UNKNOWN", "id": "SERUM FATTY ACID CONCENTRATIONS"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "Both dietary intervention groups showed changes in serum fatty acid levels; the Healthy Eating group reduced SFA intake while increasing long chain n-3 PUFA, leading to a higher polyunsaturated fat: saturated fat ratio.)", "entity_type": "UNKNOWN", "id": "SERUM FATTY ACID CONCENTRATIONS AFTER INTERVENTION"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The Mediterranean diet resulted in changes towards a healthier fatty acid profile with increased n-3 PUFA and decreased 18:2 n-6 concentration, compared to baseline.)", "entity_type": "UNKNOWN", "id": "COLON FATTY ACID CONCENTRATIONS AFTER INTERVENTION"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The presence of any minor alleles significantly affected the serum fatty acid concentrations (20:4, n-6; 18:3, n-3) and marginally affected colonic 20:4, n-6 concentration.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "FATTY ACID CONCENTRATIONS AT BASELINE"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The dietary interventions led to significant changes in serum fatty acid concentrations; these changes were particularly notable between the two diet groups.)", "entity_type": "UNKNOWN", "id": "SERUM FATTY ACID CHANGES"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "Both dietary interventions led to changes in colon fatty acid concentrations; the Mediterranean group showed a particular decrease in 18:2 n-6 concentration.)", "entity_type": "UNKNOWN", "id": "COLON FATTY ACID CHANGES"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The Mediterranean diet resulted in a non-significant change in colon fatty acid composition compared to baseline, specifically with regards to the concentration of 18:2 n-6 fatty acids.)", "entity_type": "UNKNOWN", "id": "COLON 18:2 N-6 CONCENTRATION"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The Healthy Eating diet led to an increase in colon MUFA (monounsaturated fat) concentrations compared to baseline, indicating changes towards a healthier fatty acid profile.)", "entity_type": "UNKNOWN", "id": "COLON MUFA CONCENTRATION"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The Healthy Eating diet led to an increase in colon MUFA (monounsaturated fat) concentrations compared to baseline, indicating changes towards a healthier fatty acid profile.)", "entity_type": "UNKNOWN", "id": "HEALTHY EATING DIET"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "No significant effects of genotypes on colon insulin concentrations were observed, indicating the influence of diet rather than genetics in this aspect.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "COLON INSULIN CONCENTRATION"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "Both dietary interventions did not lead to significantly different serum glucose levels compared to baseline or between each other, suggesting that changes were minor or within normal variation.)", "entity_type": "UNKNOWN", "id": "SERUM GLUCOSE LEVELS"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "While both diet groups showed changes in colon fatty acid profiles after intervention, there was no significant difference between the Mediterranean and Healthy Eating diets regarding their polyunsaturated to saturated fat ratio.)", "entity_type": "UNKNOWN", "id": "COLON POLYUNSATURATED: SATURATED RATIO"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "Carriers of minor alleles responded differently to dietary interventions compared to those with all major alleles, in terms of changes in serum fatty acid concentrations.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "DIETARY INTERVENTION RESPONSE"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The Mediterranean diet showed a non-significant impact on serum insulin levels after the intervention period compared to baseline and other diets.)", "entity_type": "UNKNOWN", "id": "SERUM INSULIN LEVELS"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The Healthy Eating diet led to an increase in colon docosahexaenoic acid (DHA; a type of omega-3 fatty acid, also represented as 20:5, n-3), indicating beneficial effects on DHA levels.)<SEP>The Healthy Eating diet resulted in an increase in colon eicosapentaenoic acid (EPA; a type of omega-3 fatty acid, also represented as 20:5, n-6), indicating improved EPA levels after the intervention.)", "entity_type": "UNKNOWN", "id": "COLON 18:3, N-3 CONCENTRATION"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "No significant differences in response to genotypes were found for changes in serum triglyceride levels among the dietary intervention groups.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "SERUM TRIGLYCERIDE LEVELS"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "After the interventions, no diet group showed a significant effect on colon linoleic acid (20:4, n-6) concentration compared to baseline or each other.)", "entity_type": "UNKNOWN", "id": "COLON 20:4, N-6 CONCENTRATION"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The Mediterranean diet led to a decrease in serum low-density lipoprotein (LDL) cholesterol levels after the intervention period, indicating potential cardiovascular benefits from the dietary changes.)", "entity_type": "UNKNOWN", "id": "SERUM LDL LEVELS"}, {"source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "description": "A significant interaction was found for 20:4, n-6 (AA) in the colon between genotype and diet.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "GENETIC VARIANT AND DIET INTERVENTION"}, {"source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "description": "Mean colon AA concentrations were significantly lower in the Mediterranean arm compared to Healthy Eating by month 6.)", "entity_type": "UNKNOWN", "id": "DIET ARM EFFECT"}, {"source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "description": "Mean colon AA concentrations were significantly lower in the Mediterranean arm compared to Healthy Eating by month 6.)", "entity_type": "UNKNOWN", "id": "SUBJECT WITH NO MINOR ALLELES"}, {"source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "description": "Mean colon AA concentrations at month 6 differed significantly based on diet arm allocation, specifically for subjects without minor alleles.)", "entity_type": "UNKNOWN", "id": "DIET ARM SPECIFICITY"}, {"source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "description": "Mean colon AA concentrations at month 6 differed significantly based on diet arm allocation, specifically for subjects without minor alleles.)", "entity_type": "UNKNOWN", "id": "OUTCOME DIFFERENCE"}, {"entity_type": "DISEASE STUDY", "description": "The UK Biobank is a population-based study consisting of individuals with reported snoring and related factors.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "UK BIOBANK"}, {"entity_type": "SYMPTOM", "description": "Snoring is characterized by the loud or regular breathing sounds made when sleeping.<SEP>The prevalence of snoring was observed, with a higher rate noted among males compared to females.", "source_id": "chunk-a34b927ed42ff0917960122b4d203f83<SEP>chunk-9977706363cc368b3be503d07ed3ef62", "id": "SNORING"}, {"entity_type": "PARTICIPANT\'S RELATION TO REPORT SNORING", "description": "Participants in the sample were identified as snorers based on reports from their partners or housemates about their snoring.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "PARTNER OR HOUSEMATE"}, {"entity_type": "RELATED CONDITION", "description": "The prevalence of sleep apnoea is higher in the group identified as snorers.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "SLEEP APNOEA"}, {"entity_type": "SOCIAL ECONOMIC STATUS", "description": "A lower social economic status (measured by Townsend deprivation index and average household income) is associated with an increased likelihood of snoring among males only.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "SES"}, {"entity_type": "RISK FACTOR", "description": "Smoking frequency is correlated with snoring prevalence, more so in females than in males.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "SMOKING FREQUENCY"}, {"entity_type": "RISK FACTOR", "description": "Alcohol consumption frequency shows a correlation with snoring, particularly affecting males and to a lesser extent, females.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "ALCOHOL CONSUMPTION FREQUENCY"}, {"entity_type": "PHYSICAL TRAIT", "description": "Whole-body fat mass is correlated with snoring prevalence.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "WHOLE-BODY FAT MASS"}, {"entity_type": "STATISTICAL MEASURE", "description": "The overall SNP heritability on the liability scale for snoring is 9.9%, indicating the proportion of variance in snoring that can be attributed to genetic factors.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "H2SNP - SNP HERITABILITY ON THE LIABILITY SCALE"}, {"entity_type": "ASSOCIATION MEASURE", "description": "The highest genetic correlation with habitual snoring was found between self-reported sleep apnoea (rG = 0.78), suggesting a strong genetic overlap.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "RG - GENETIC CORRELATION COEFFICIENT"}, {"entity_type": "SLEEP DISORDERED BREATHING INDICATOR", "description": "Minimum SpO2 and Perc90, which are known proxies for sleep-disordered breathing (SDB) but not average SpO2, showed moderate significant genetic correlations with snoring.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "MINIMUM SPO2 AND PERC90 - TRAITS OF OVERNIGHT OXYHEMOGLOBIN SATURATION"}, {"entity_type": "TRAITS GENETICALLY CORRELATED TO SNORING", "description": "Several traits genetically correlated with snoring include BMI, obesity indicators, sleep apnoea, mood disorders, cardiovascular diseases, lung function, and personality traits related to stress and emotional health.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "BMI, WHOLE-BODY FAT MASS, SODIUM IN URINE, MOOD SWINGS, CORONARY ARTERY DISEASE, ALCOHOL INTAKE FREQUENCY, PULSE RATE, CURRENT TOBACCO SMOKING, HEART DISEASE, LUNG CANCER, FEV1/FVC RATIO, NEUROTICISM, SUBJECTIVE WELLBEING, HEART RATE"}, {"entity_type": "TRAITS GENETICALLY CORRELATED AFTER BMI ADJUSTMENT", "description": "After adjusting for BMI, snoring remained genetically correlated with psychiatric conditions like schizophrenia and traits related to sleep quality.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "SCHIZOPHRENIA, EDUCATIONAL ATTAINMENT, SODIUM IN URINE, DAYTIME DOZING, SLEEP APNOEA, EXCESSIVE DAYTIME SLEEPINESS"}, {"entity_type": "SLEEP DISORDERED BREATHING INDICATOR", "description": "Measures of nocturnal hypoxia showed increased genetic correlation with snoring after adjusting for BMI.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "MINIMUM SPO2, AVERAGE SPO2, PERC90 - MEASURES OF NOCTURNAL HYPOXIA"}, {"entity_type": "SLEEP PARAMETER", "description": "Sleep fragmentation, characterized by frequent awakenings during the night, affects the severity of snoring.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "SLEEP FRAGMENTATION"}, {"entity_type": "GENE", "description": "A genome-wide significant SNP mapping to the gene DLEU7 has been found in association with snoring.<SEP>The gene DLEU7 mapping to a genome-wide significant SNP on chromosome 13 is associated with snoring.<SEP>The gene DLEU7 interacts with rs9316505 in association with number of teeth at one year.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4<SEP>chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "DLEU7"}, {"entity_type": "MOLECULAR INTERACTION", "description": "MSRB3 affects the width of the nose and tooth eruption.<SEP>MSRB3 is another significant gene mapped with a genome-wide significant SNP associated with snoring.<SEP>The gene MSRB3 mapping to a genome-wide significant SNP on chromosome 12 is associated with snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64<SEP>chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "MSRB3"}, {"entity_type": "GENE", "description": "BCL11B, FTO, RNA5SP471 and SND1, NSUN3 have previously been linked to alcohol consumption in relation to snoring.<SEP>BCL11B, FTO, SMG6, ROBO2, NSUN3, SNAP91 and BCL2 are previously known to be associated with smoking.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "BCL11B"}, {"entity_type": "GENE", "description": "FTO, SND1 are also associated with coffee consumption in relation to snoring.<SEP>RNA5SP471\'s association with narcolepsy might have implications on the understanding of snoring mechanisms.<SEP>RNA5SP471, having connections with narcolepsy, might have implications for sleep disorders that can be associated with snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "RNA5SP471"}, {"entity_type": "GENE", "description": "LMO4 is related to insomnia which could be a contributing factor for snoring.<SEP>LMO4 is related to insomnia while RNA5SP471 shows association with narcolepsy.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "LMO4"}, {"entity_type": "GENE", "description": "ROBO2 was associated with chronotype and traits such as height and heel bone mineral density.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "ROBO2"}, {"entity_type": "GENE", "description": "Additional exploration of SNAP91\'s role in respiratory and cardiometabolic health might shed light on its relevance to snoring.<SEP>Prioritization of SNAP91 highlights its role in smoking, alcohol consumption, and coffee intake\'s influence on snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "SNAP91"}, {"entity_type": "GENE", "description": "ANAPC4 was identified through positional and eQTL mapping as a gene with implications for snoring and musculoskeletal traits like height and heel bone mineral density.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "ANAPC4"}, {"entity_type": "GENE", "description": "SNPs associated with SMG6 are found to have significant links with snoring, potentially related to respiratory or cardiometabolic conditions.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "SMG6"}, {"entity_type": "GENE", "description": "SND1 was previously linked to both alcohol and coffee consumption\'s effect on snoring through genome-wide gene-based association analyses.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "SND1"}, {"entity_type": "GENE", "description": "Genomic evidence connects NSUN3 with respiratory traits like sleep apnea and excessive daytime sleepiness, also involved in psychiatric conditions.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "NSUN3"}, {"entity_type": "GENE", "description": "While SIM1 is not associated with snoring through GWAS analysis alone, its connection to musculoskeletal traits might have indirect influences on snoring mechanisms.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "SIM1"}, {"entity_type": "GENE", "description": "The role of KCNQ5 in the genetic predisposition for insomnia could influence sleep patterns and, potentially, snoring frequency.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "KCNQ5"}, {"entity_type": "GENE", "description": "Positional mapping suggests CEP120\'s involvement in traits related to musculoskeletal health which might impact sleep quality and snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "CEP120"}, {"entity_type": "GENE", "description": "Genes like MACF1, prioritized based on their genomic associations with musculoskeletal traits, may indirectly affect snoring through sleep physiology.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "MACF1"}, {"source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "description": "BMI shows a moderate positive genetic correlation with snoring prevalence.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 4}]", "id": "GENETIC CORRELATION"}, {"source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "description": "Self-reported sleep apnoea has the highest genetic correlation with habitual snoring (rG = 0.78).)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 4}]", "id": "SLEEP APNOEA - SELF-REPORTED SLEEP APNOEA"}, {"source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "description": "Several traits with the strongest evidence of association were BMI, obesity indicators, sleep apnoea, mood disorders, cardiovascular diseases, lung function, and personality traits related to stress and emotional health.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 4}]", "id": "BMI, WHOLE-BODY FAT MASS, SODIUM IN URINE, MOOD SWINGS, CORONARY ARTERY DISEASE, ALCOHOL INTAKE FREQUENCY, PULSE RATE, CURRENT TOBACCO SMOKING, HEART DISEASE, LUNG CANCER, FEV1/FVC RATIO, NEUROTICISM, SUBJECTIVE WELLBEING, HEART RATE - TRAITS GENETICALLY CORRELATED TO SNORING"}, {"source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "description": "After adjusting for BMI, traits like schizophrenia, educational attainment, and sleep-related conditions remained genetically correlated with snoring.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 4}]", "id": "SCHIZOPHRENIA, EDUCATIONAL ATTAINMENT, SODIUM IN URINE, DAYTIME DOZING, SLEEP APNOEA, EXCESSIVE DAYTIME SLEEPINESS - TRAITS GENETICALLY CORRELATED AFTER BMI ADJUSTMENT"}, {"entity_type": "GENOMIC VARIANT", "description": "Single nucleotide polymorphisms (SNPs) in the VDBP gene were analyzed.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "VDBP SNPS"}, {"entity_type": "DISEASE", "description": "The vitamin D status of mothers and neonates was assessed based on VDBP polymorphisms.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "MATERNAL AND NEONATAL VITAMIN D STATUS"}, {"entity_type": "DISEASE", "description": "Vitamin D equilibrium in the maternal-neonate dyads was determined at different vitamin D concentrations (total 25(OH)D).", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "VITAMIN D STATUS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Maternal height was considered as a factor affecting vitamin D status.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "MATERNAL HEIGHT"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Pre-pregnancy BMI of mothers influenced the vitamin D equilibrium during delivery.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "BMI PRE-PREGNANCY"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Final (terminal) BMI of pregnant women affected vitamin D status.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "BMI TERMINAL"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Ultraviolet B (UVB) radiation exposure was considered as an environmental factor in relation to neonatal vitamin D levels.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "UVB"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The duration of pregnancy impacted the maternal-neonate vitamin D status.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "WEEKS OF GESTATION"}, {"entity_type": "GENOMIC VARIANT", "description": "The polymorphism rs2298850 of VDBP gene was analyzed.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "RS2298850"}, {"entity_type": "GENOMIC VARIANT", "description": "Another polymorphism, rs4588 in the VDBP gene, was considered for this analysis.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "RS4588"}, {"entity_type": "NUTRIENT", "description": "The level of maternal vitamin D (total 25(OH)D) was measured and compared with different genotypes.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "MATERNAL VITAMIN D CONCENTRATIONS"}, {"entity_type": "NUTRIENT", "description": "Levels of neonatal vitamin D status were quantified based on VDBP polymorphisms.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "NEONATAL 25(OH)D CONCENTRATIONS"}, {"entity_type": "DIETARY PATTERN", "description": "Cut-offs for maternal vitamin D levels at birth were established to categorize different genotypes.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "MATERNAL 25(OH)D CUT-OFFS AT BIRTH (\u226425 VS. \u226525 NMOL/L)"}, {"entity_type": "DIETARY PATTERN", "description": "Different thresholds for maternal vitamin D concentrations were set in relation to VDBP genotypes.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "MATERNAL VITAMIN D CUT-OFFS AT BIRTH (\u226450 VS. \u226550 NMOL/L AND \u226575 NMOL/L VS. \u226475 NMOL/L)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The interaction between VDBP polymorphisms and the delivery time concentration of maternal vitamin D was evaluated.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "MATERNAL VITAMIN D STATUS AT DELIVERY (\u226575 NMOL/L VS. \u226475 NMOL/L)"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "The polymorphisms in VDBP gene were associated with maternal vitamin D concentrations during delivery.)", "entity_type": "UNKNOWN", "id": "MATERNAL VITAMIN D STATUS"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "Neonatal vitamin D levels were associated with maternal VDBP polymorphisms. )", "entity_type": "UNKNOWN", "id": "MATERNAL VDBP GENOTYPES"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "Neonatal vitamin D levels were associated with maternal VDBP polymorphisms. )", "entity_type": "UNKNOWN", "id": "NEONATAL VITAMIN D STATUS AT BIRTH (CUT-OFFS OF 25 AND 50 NMOL/L)"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "The genetic variants in VDBP correlated with maternal vitamin D levels during childbirth.)", "entity_type": "UNKNOWN", "id": "MATERNAL VITAMIN D CONCENTRATION"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "The genetic variants in VDBP correlated with maternal vitamin D levels during childbirth.)", "entity_type": "UNKNOWN", "id": "VDBP POLYMORPHISMS"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "The cut-offs for maternal vitamin D were associated with specific genotypes of the VDBP gene.)", "entity_type": "UNKNOWN", "id": "MATERNAL VDBP POLYMORPHISMS"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "The cut-offs for maternal vitamin D were associated with specific genotypes of the VDBP gene.)", "entity_type": "UNKNOWN", "id": "MATERNAL VITAMIN D CUT-OFFS (\u226425 VS. \u226525 NMOL/L, \u226450 VS. \u226550 NMOL/L AND \u226575 NMOL/L VS. \u226475 NMOL/L)"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "The vitamin D status during delivery was linked to various genotypes of the VDBP gene.)", "entity_type": "UNKNOWN", "id": "VDBP GENOTYPES"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "The neonatal vitamin D levels correlated with the maternal VDBP genotypes at birth.)", "entity_type": "UNKNOWN", "id": "NEONATAL VITAMIN D LEVELS AT BIRTH (CUT-OFFS OF 25 AND 50 NMOL/L)"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "The polymorphisms rs2298850 and rs4588 were associated with maternal vitamin D concentrations during childbirth.)", "entity_type": "UNKNOWN", "id": "MATERNAL VITAMIN D CONCENTRATION DURING DELIVERY"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "The polymorphisms rs2298850 and rs4588 were associated with maternal vitamin D concentrations during childbirth.)", "entity_type": "UNKNOWN", "id": "VDBP SNPS (RS2298850 AND RS4588)"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "Maternal 25(OH)D Cut-Offs at Birth", "entity_type": "UNKNOWN", "id": "NEONATAL CUT-OFFS (\u226550 NMOL/L VS. \u226450 NMOL/L, \u226525 VS. \u226425 NMOL/L)"}, {"entity_type": "DISEASE", "description": "The biochemical and clinical characteristics of women with PCOS were presented, showing differences in BMI, free testosterone, bioactive testosterone, FAI, fasting insulin, and HOMA-IR.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "PCOS"}, {"entity_type": "DISEASE", "description": "Women who served as control subjects showed comparable age, menarche, waist-hip ratio, lipid profile, total testosterone, LH, and FSH levels.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "CONTROL SUBJECTS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The lipid profile was measured for both women with PCOS and control subjects.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "LIPID PROFILE"}, {"entity_type": "PROTEIN", "description": "Total testosterone level was determined in women with PCOS.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "TOTAL TESTOSTERONE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Luteinizing hormone (LH) levels were measured as part of the hormonal profile in both groups.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "LH"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Follicle-stimulating hormone (FSH) was also included in the hormonal analysis for women with PCOS and controls.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "FSH"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Levels of free testosterone were compared in women with PCOS and controls to identify differences.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "FREE TESTOSTERONE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Bioactive testosterone levels contributed to understanding hormone dynamics in both groups.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "BIOACTIVE TESTOSTERONE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "FAI was used as an indicator of the androgenic status, showing significant differences between PCOS cases and controls.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "FAI (FREE ANDROGEN INDEX)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "HOMA-IR was found to be significantly elevated in PCOS cases, indicating insulin resistance.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "HOMA-IR (HOMEOSTASIS MODEL ASSESSMENT OF INSULIN RESISTANCE)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Lower levels of SHBG were observed among women with PCOS compared to controls.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "SHBG (SEX HORMONE BINDING GLOBULIN)"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "While within the reference range, higher TSH was noted in women with PCOS, suggesting potential thyroid dysfunction.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "TSH (THYROID STIMULATING HORMONE)"}, {"entity_type": "GENE", "description": "Twelve VEGFA Single Nucleotide Polymorphisms (SNPs) were selected for analysis based on their established MAFs exceeding 5% in Caucasians.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "VEGFA (VASCULAR ENDOTHELIAL GROWTH FACTOR A)"}, {"entity_type": "GENOMIC VARIANT", "description": "This SNP had a higher frequency of the minor allele C in women with PCOS compared to control subjects (P = 9.0 \u00d7 10^-4).<SEP>This SNP was associated with an increased risk of PCOS (OR: 1.89; CI: 1.09\u20133.26).", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "RS3025020 (-583T>C)"}, {"entity_type": "GENOMIC VARIANT", "description": "A SNP that showed a significant difference in genotype distribution between PCOS cases and controls.<SEP>This SNP showed a protective nature against PCOS with reduced risk (OR: 0.65; CI: 0.44\u20130.97).", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "RS2010963 (-634C>G)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Mean serum VEGF levels were found to be significantly increased among rs3025020 genotype carriers.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "VEGF SERUM LEVELS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Increased fasting insulin, HOMA-IR and bioactive testosterone in rs3025020 carriers. Reduced fasting insulin, free testosterone, and bioactive testosterone in rs2010963 carriers were observed.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "PCOS FEATURES"}, {"entity_type": "RELATIONSHIP", "description": "Carriage of the minor (C) allele was positively associated with hs-CRP.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "RS3025020"}, {"source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "description": "The minor C allele of rs3025020 is associated with increased PCOS-related metabolic markers compared to the minor C allele of rs2010963.)", "entity_type": "UNKNOWN", "id": "RS3025020 AND RS2010963 GENOTYPES"}, {"entity_type": "GENOMIC VARIANT", "description": "Participants with different genotypes did not differ on self-report measures.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "GENOTYPE GROUPS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "A main effect of Block emerged for response bias, driven by higher response bias in later blocks compared to earlier blocks.<SEP>A main effect of Genotype due to higher response bias in val carriers relative to iso homozygotes indicates the potential link between specific genotypes and disease risk.<SEP>Increased cognitive load during the study led to higher response bias in participants regardless of their genotype.<SEP>Increased response bias was associated with specific patterns of neural activity detected through brain imaging techniques.<SEP>Response bias indicates how individuals tend to respond based on past rewards, often linked to certain genetic traits.<SEP>The cognitive process related to how individuals adapt their responses based on previous experiences and emotions.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647<SEP>chunk-5e709975097a773fb3d31e265d76a71b", "id": "RESPONSE BIAS"}, {"entity_type": "PROTEIN", "description": "Identified risk genes for certain diseases were found to influence response bias in participants carrying these variants.<SEP>Refers to individuals carrying one or two copies of the val allele at a specific gene locus.<SEP>Val carriers had significantly higher response bias than iso homozygotes.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647<SEP>chunk-5e709975097a773fb3d31e265d76a71b", "id": "VAL CARRIERS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Participants successfully adjusted their behavior based on reinforcement contingencies in the study.<SEP>Participants\' learning rate was affected by their ability to adjust behavior based on different reinforcement contingencies presented during the study.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "REINFORCEMENT CONTINGENCIES"}, {"entity_type": "POPULATION STRATIFICATION", "description": "The effect of genotypes varied significantly when only Caucasian subjects were considered, highlighting potential ethnic-specific impacts.<SEP>The main effect of Genotype was confirmed when considering Caucasian subjects only, suggesting that findings were not confounded by population stratification.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "CAUCASIAN SUBJECTS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Blocks 3 and 2 showed higher response bias compared to Block 1, suggesting that later blocks may be more influenced by environmental factors.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "BLOCK 3"}, {"entity_type": "DIETARY PATTERN", "description": "STAI scores were significantly elevated across all conditions, indicating increased anxiety levels among participants.<SEP>The ANOVA on STAI scores produced main effects of Condition and Time, with higher STAI scores during the stress condition relative to no-stress conditions.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "STAI SCORES"}, {"entity_type": "EFFECT SIZE MEASURE", "description": "A Cohen\'s d value exceeding 0.8 was considered indicative of a large effect size in comparing genotype groups.<SEP>Cohen\u2019s d was used to measure effect sizes for comparing genotypes and other variables.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "COHEN\'S D"}, {"entity_type": "INTERACTION EFFECT", "description": "A Condition \u00d7 Time interaction on STAI scores emerged, indicating that the effect of stress condition changed over time.<SEP>The condition \u00d7 time interaction on STAI scores revealed the dynamic effects of stress conditions changing over the course of the study.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "CONDITION X TIME INTERACTION"}, {"entity_type": "STATISTICAL TEST", "description": "Post-hoc tests were conducted to determine where differences occurred between specific groups or conditions.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "POST-HOC TESTS"}, {"entity_type": "AFFECT SCALE", "description": "Participants reported elevated Negative Affect (NA) scores during stress and no-stress conditions, suggesting increased emotional reactivity.<SEP>Participants with different genotypes showed varying levels of emotional reactivity, as measured by the Positive and Negative Affect Schedule (PANAS) scores.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "PANAS NA SCORES"}, {"entity_type": "STRESS RESPONSE", "description": "Increased non-specific skin conductance responses were observed during both stress conditions and no-stress periods for all genotypes.<SEP>More responses were observed in the stress condition compared to the no-stress condition for non-specific skin conductance responses.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "NON-SPECIFIC SC RESPONSES"}, {"entity_type": "DISEASE RISK FACTOR", "description": "Val carriers had more skin conductance responses, potentially indicating a different disease risk factor or response mechanism based on genotype.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "SC RESPONDER RATE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The no-stress condition was a control situation without any external stressors.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "NO-STRESS"}, {"entity_type": "INDIVIDUALS OR GENOTYPES", "description": "Refers to individuals who carry two identical alleles, often used in genetics to describe homozygous genotypes.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "ISOMORPHES (ISO HOMOZYGOTES)"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The stress manipulation condition in the experiment induced elevated anxiety and negative affect.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "CONDITION"}, {"entity_type": "EXPERIMENTAL DESIGN OR PROCEDURE", "description": "A task used in the experiment which requires participants to adapt their responses according to changing probabilities of reward.>", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "PROBABILISTIC REWARD TASK"}, {"entity_type": "CONDITION", "description": "The condition where the stress manipulation was applied, leading to different response biases compared to the no-stress condition.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "STRESS CONDITION"}, {"entity_type": "CONDITION", "description": "A control situation without any external stressors that serves as a reference point for assessing response biases in the presence of stress.<SEP>An opposite condition that contrasts with stressed conditions by absence of stressors.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "NO-STRESS CONDITION"}, {"entity_type": "STATISTICAL INTERACTION", "description": "An interaction observed between condition, block and genotype which indicates how these factors affect response bias differently.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "CONDITION X BLOCK X GENOTYPE INTERACTION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "A measure of how well individuals can distinguish between different levels or categories in a stimulus-response task.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "DISCRIMINABILITY"}, {"entity_type": "EXPERIMENTAL CONDITION OR SEQUENCE OF EVENTS", "description": "Refers to sequential stages or iterations within the experiment\'s experimental design.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "BLOCK 1, BLOCK 2, AND BLOCK 3"}, {"entity_type": "DISEASE OR DEFICIT", "description": "The phenomenon observed where response bias decreases under stress, a condition being investigated for its implications in genetic traits.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "STRESS INDUCED RESPONSE BIAS REDUCTIONS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Refers to the intensity of electric shock stimuli used during the experiment as one variable among many considered in hierarchical regressions.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "SHOCK INTENSITY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Changes in skin conductance, which can be influenced by stress levels and other factors, were also measured and controlled for.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "SKIN CONDUCTANCE RESPONSES"}, {"entity_type": "GENE VARIANT OR GENETIC MARKER", "description": "Circumstances where certain Single Nucleotide Polymorphisms were associated with stress-induced reward learning deficits.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "CRHR1 SNPS (RS12938031, RS10445364, RS4076452)"}, {"entity_type": "STATISTICAL METHOD", "description": "A type of regression analysis used in the study to control for potential confounding variables when assessing genetic effects.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "HIERARCHICAL REGRESSION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "A condition involving the induction of stress.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "STRESSED CONDITION"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Response bias showed a main effect for block, with higher bias in later blocks compared to earlier blocks during the study.)", "entity_type": "UNKNOWN", "id": "MAIN EFFECT OF BLOCK"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Val carriers exhibited significantly different response bias than iso homozygotes due to their genotype.)", "entity_type": "UNKNOWN", "id": "MAIN EFFECT OF GENOTYPE"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Participants adapted behavior based on reinforcement contingencies during the study, demonstrating the influence of external stimuli on behavior adjustment.)", "entity_type": "UNKNOWN", "id": "ADJUSTMENT OF BEHAVIOR"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Analysis confirmed the main effect of genotype when considering only Caucasian subjects to avoid confounding due to population stratification.)", "entity_type": "UNKNOWN", "id": "POPULATION STRATIFICATION INTERACTION EFFECT"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Increased response bias in val carriers relative to iso homozygotes suggests a potential link between specific genotypes and disease risk.)", "entity_type": "UNKNOWN", "id": "DISEASE RISK"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Later blocks showed higher response bias compared to earlier blocks, indicating that environment or task complexity influenced behavior later in the study.)", "entity_type": "UNKNOWN", "id": "ENVIRONMENTAL FACTOR INTERACTION"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Main effects of condition and time on STAI scores emerged, with higher scores during stress conditions relative to no-stress conditions.)", "entity_type": "UNKNOWN", "id": "CONDITION EFFECT"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Cohen\u2019s d was used to compare effect sizes between genotypes or different study variables, providing a measure of practical significance.)", "entity_type": "UNKNOWN", "id": "EFFECT SIZE COMPARISON"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "A significant interaction on STAI scores over time indicated that the stress condition influenced participants\' self-report measures differently at various points during the study.)", "entity_type": "UNKNOWN", "id": "STAI SCORE CHANGE"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Increased Negative Affect (NA) scores across conditions suggested heightened emotional responses to both stress and no-stress situations during the experiment.)", "entity_type": "UNKNOWN", "id": "AFFECT SCALE RESPONSE"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "More skin conductance responses were observed in response to the stress condition compared to the no-stress condition, suggesting a higher physiological arousal during stress.)", "entity_type": "UNKNOWN", "id": "STRESS RESPONSE RATE"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "A significant interaction effect was found between genotype groups and neuropsychological test performance, indicating different cognitive profiles across genotypes.)", "entity_type": "UNKNOWN", "id": "NEUROPSYCHOLOGICAL TEST PERFORMANCE INTERACTION EFFECT"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "There was a strong correlation between response bias and neural activity patterns in specific brain regions, as identified through functional MRI analysis.)", "entity_type": "UNKNOWN", "id": "NEURAL ACTIVITY CORRELATION"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "The presence of risk variant genes was correlated with enhanced response bias that may indicate heightened disease susceptibility.)", "entity_type": "UNKNOWN", "id": "GENETIC RISK TO DISEASE"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Participants\' adaptive behavior improved over time in response to reinforcement contingencies, showing plasticity and learning capabilities.)", "entity_type": "UNKNOWN", "id": "ADAPTIVE BEHAVIOR"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Analysis of specific genetic loci showed that their effects were more pronounced among Caucasian subjects than other ethnicities.)", "entity_type": "UNKNOWN", "id": "GENETIC INFLUENCE SPECIFICITY"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "The increase in response bias was moderated by the level of cognitive load, suggesting a threshold for overload conditions.)", "entity_type": "UNKNOWN", "id": "COGNITIVE LOAD EFFECT"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Anxiety levels influenced the magnitude of STAI scores, with high anxiety participants showing greater variability in their responses.)", "entity_type": "UNKNOWN", "id": "ANXIETY MODERATION FACTOR"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Cohen\'s d values were used to benchmark differences in mean scores across genotype groups, aiding in distinguishing true effects from random variation.)", "entity_type": "UNKNOWN", "id": "SIGNIFICANCE BENCHMARK COMPARISON"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Over time, the impact of stress conditions on STAI scores was more pronounced, indicating increased sensitivity to anxiety over longer periods.)", "entity_type": "UNKNOWN", "id": "TEMPORAL ANXIETY CHANGES"}, {"source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "description": "Differences in PANAS NA scores suggested that certain genotypes may predispose individuals to higher levels of negative affectivity.)", "entity_type": "UNKNOWN", "id": "GENETIC CONTRIBUTION TO EMOTION"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Genetic variant (val allele) had an effect on response bias specifically under conditions of stress.)", "entity_type": "UNKNOWN", "id": "CONDITION X GENOTYPE INTERACTION"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "val carriers showed a lower response bias in stress condition compared to no-stress condition.)", "entity_type": "UNKNOWN", "id": "COHENS D = 1.22, P = .056"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "val carriers showed a lower response bias in stress condition compared to no-stress condition.)", "entity_type": "UNKNOWN", "id": "SHOCK INTENSITY, SKIN CONDUCTANCE RESPONSES"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs17563 is associated with \'age at first tooth\' and is located near the BMP4 gene.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS17563"}, {"entity_type": "GENE", "description": "BMP4 is related to measures of craniofacial shape and size, showing pleiotropic effects.<SEP>The gene BMP4 interacts with rs17563 in association with age of onset of tooth development.<SEP>Variants in the BMP4 gene also contribute significantly to the \'Age at first tooth\' SNP associations.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4<SEP>chunk-c63d60b991cd7442f690dcc364af47dd<SEP>chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "BMP4"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs10740993 is associated with \'age at first tooth\' and resides near the CACNB2 gene.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS10740993"}, {"entity_type": "GENE", "description": "SNP rs1490384 near CACNB2 links to tooth eruption measures indicating genetic influence.<SEP>The gene CACNB2 interacts with rs10740993 in association with age of onset of tooth development.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4<SEP>chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "CACNB2"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs4937076 is associated with \'age at first tooth\' and influences expression of the CDON gene.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS4937076"}, {"entity_type": "GENE", "description": "The gene CDON interacts with rs4937076 in association with age of onset of tooth development.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "CDON"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs1799922 is associated with \'age at first tooth\' and influences expression of the CALU/OPN1SW gene.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS1799922"}, {"entity_type": "GENE", "description": "The gene CALU/OPN1SW interacts with rs1799922 in association with age of onset of tooth development.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "CALU/OPN1SW"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs997154 is associated with increased height and prominence of the mid-brow.<SEP>The genomic variant rs997154 is associated with \'age at first tooth\' and influences expression of the AJUBA/C14orf93 gene.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4<SEP>chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "RS997154"}, {"entity_type": "GENE", "description": "The genes AJUBA and C14orf93 interact with rs997154 in association with age of onset of tooth development.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "AJUBA/C14ORF93"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs7924176 affects nose width, and this is related to earlier tooth eruption.<SEP>SNP rs7924176 predisposes to wider nose widths, which is related to earlier tooth eruption.<SEP>The genomic variant rs7924176 is associated with \'age at first tooth\' and influences expression of the ADK gene.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4<SEP>chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "RS7924176"}, {"entity_type": "GENE", "description": "ADK\'s allele predisposes to wider noses which correlates with earlier tooth eruption.<SEP>The gene ADK interacts with rs7924176 in association with age of onset of tooth development.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4<SEP>chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "ADK"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs412000 is associated with \'age at first tooth\' and influences expression of the TEX14/RAD51C gene.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS412000"}, {"entity_type": "GENE", "description": "The genes TEX14 and RAD51C interact with rs412000 in association with age of onset of tooth development.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "TEX14/RAD51C"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs9316505 is associated with \'number of teeth\' and influences expression of the DLEU7 gene.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS9316505"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs17101923 influences face width, which correlates with number of teeth and earlier tooth eruption.<SEP>The genomic variant rs17101923 was identified as genome-wide significant for \'number of teeth\' and is located near the HMGA2 region.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4<SEP>chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "RS17101923"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs10932688 was identified as genome-wide significant for \'number of teeth\' and is located in the 2q35 region.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS10932688"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs6568401 was identified as suggestive level association for \'number of teeth\' and is located in the 6q21 region.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS6568401"}, {"entity_type": "GENOMIC VARIANT", "description": "Although not meeting genome-wide significance threshold, rs17105278 was still associated with \'number of teeth\' at this locus and is located near the RAD51L1 gene.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS17105278"}, {"entity_type": "GENOMIC VARIANT", "description": "Although not meeting genome-wide significance threshold, rs1956529 was still associated with \'number of teeth\' at this locus and influences expression of BMP4 genes.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS1956529"}, {"entity_type": "GENE", "description": "The gene KCNJ160 might have a role in tooth development based on the presence of independent secondary signals at rs11077486.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "KCNJ160"}, {"entity_type": "GENOMIC VARIANT", "description": "At the FAM155E-EDA region, there might be independent secondary signal for \'age at first tooth\', which suggests an additional genetic factor that contributes to tooth development.<SEP>SNP rs2520397 is part of a group of SNPs associated with \'age at first tooth\' and other phenotypes related to primary tooth and craniofacial development.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4<SEP>chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "RS2520397"}, {"entity_type": "GENE REGION", "description": "The region encompassing FAM155E and EDA genes might have roles in tooth development based on the presence of independent secondary signals at rs2520397.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "FAM155E\u2013EDA"}, {"entity_type": "GENOMIC VARIANT", "description": "At the HMGA2 region, there might be independent secondary signal for \'age at first tooth\', which suggests an additional genetic factor that contributes to tooth development.<SEP>SNP rs1472259 is related to the number of teeth and earlier tooth eruption.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4<SEP>chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "RS1472259"}, {"entity_type": "GENOMIC VARIANT", "description": "In the IGF2BP1 locus, there might be independent secondary signals influencing \'number of teeth\' and tooth development, indicating complex gene interactions.)(entity<SEP>SNP rs8069452 shows pleiotropic effects, associated with measures indexing height and prominence of mid-brow.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4<SEP>chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "RS8069452"}, {"entity_type": "GENOMIC VARIANT", "description": "In the RUNX2 locus, rs1061890 shows a significant association with \'number of teeth\' suggesting its role in controlling tooth number.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS1061890"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant, rs7365438, influences the expression levels of the SLC2A1 gene and correlates with age at first tooth development.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS7365438"}, {"entity_type": "GENE", "description": "The gene SNAI1 interacts with this variant rs7905307 which is associated with \'age at first tooth\' indicating its role in tooth development processes.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "SNAI1"}, {"entity_type": "GENOMIC VARIANT", "description": "This genomic variant shows a strong association with \'number of teeth\', suggesting it plays a crucial role in regulating the number of teeth in individuals.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS4647821"}, {"entity_type": "GENOMIC VARIANT", "description": "It influences tooth development and age at first eruption, making it an important factor for understanding tooth growth dynamics.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS352930"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant shows a significant association with \'age at first tooth\', suggesting its impact on the timing of tooth eruption.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS17684"}, {"entity_type": "GENOMIC VARIANT", "description": "A variant associated with \'number of teeth\' in the 7q11.21 region, highlighting its role in determining tooth count.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS903253"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant is associated with \'age at first tooth\', indicating its role in controlling the timing of primary tooth eruption.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS4668112"}, {"entity_type": "GENOMIC VARIANT", "description": "A variant that affects the expression levels of the ESR1 gene, influencing tooth development and age at first eruption.", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "RS3750934"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs1951867 is one of the SNPs associated with measures indexing width in the upper region of the face and nose.", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "RS1951867"}, {"entity_type": "MOLECULAR INTERACTION", "description": "AJUBA is associated with height increase, impacting the prominence of the mid-brow and tooth eruption.", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "AJUBA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "SNP in proximity of C6orf173 is associated with both age at first tooth and number of teeth, reflecting pleiotropic effects.", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "C6ORF173"}, {"entity_type": "MOLECULAR INTERACTION", "description": "SNP rs1570106 in RAD51L1 shows association with tooth eruption measures and height, linking genetic traits.", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "RAD51L1"}, {"entity_type": "ORGANISM/GROUP/ENTITY", "description": "A consortium that provides robustly associated height variants for further analysis of genetic influences on tooth eruption.", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "LANGO ALLEN ET AL. GIANT CONSORTIUM"}, {"entity_type": "STUDY/RESEARCH", "description": "The Northern Finland Birth Cohort 1966, a longitudinal study that contributes data on participants\' traits including tooth eruption and height.", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "NFBC1966"}, {"entity_type": "STUDY/RESEARCH", "description": "The Avon Longitudinal Study of Parents and Children, a large-scale birth cohort study providing data on various traits including dental development and height.", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "ALSPAC"}, {"entity_type": "GENETIC INFLUENCE", "description": "These specific SNPs play a significant role in both tooth eruption timing and overall height, indicating key genes involved in these developmental processes.", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "SNPS IN HMGA2 AND BMP4"}, {"entity_type": "TEMPORAL PHENOTYPE MEASURE", "description": "Measures of height taken at different ages (17 and 31 years) used in association studies with genetic factors influencing tooth development.", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "id": "SNPS FOR HEIGHT AT 17 YEARS AND HEIGHT AT 31 YEARS"}, {"entity_type": "DISEASE", "description": "Moderate evidence of association between \'Age at first tooth\' and height is observed.", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "id": "AGE AT FIRST TOOTH"}, {"entity_type": "DISEASE", "description": "Evidence of association with height appears to be driven by variants in HMGA2 and BMP4.", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "id": "NUMBER OF TEETH"}, {"entity_type": "DISEASE", "description": "The \'Age at first tooth\' is associated with height.", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "id": "HEIGHT"}, {"source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "description": "Several height-associated SNPs show strong evidence of association with tooth eruption in the expected direction (i.e., taller individuals have faster tooth eruption/more teeth).)", "entity_type": "UNKNOWN", "id": "HEIGHT-ASSOCIATED SNPS"}, {"source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "description": "Several height-associated SNPs show strong evidence of association with tooth eruption in the expected direction (i.e., taller individuals have faster tooth eruption/more teeth).)", "entity_type": "UNKNOWN", "id": "TOOTH ERUPTION"}, {"source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "description": "SNP rs1351394 shows strong evidence of association with both \'Age at first tooth\' and \'Number of teeth\', linking height with tooth eruption and number.)", "entity_type": "UNKNOWN", "id": "RS1351394"}, {"source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "description": "SNP rs12534093 has a positive effect on faster tooth eruption/more teeth linked to IGF2BP3 association.)", "entity_type": "UNKNOWN", "id": "IGF2BP3"}, {"source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "description": "SNP rs12534093 has a positive effect on faster tooth eruption/more teeth linked to IGF2BP3 association.)", "entity_type": "UNKNOWN", "id": "RS12534093"}, {"source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "description": "SNP rs1490384 near C6orf173 is associated with \'Age at first tooth\' and \'Number of teeth\', linking height to earlier tooth eruption.)", "entity_type": "UNKNOWN", "id": "RS1490384"}, {"source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "description": "SNP rs1570106 in RAD51L1 shows association with both \'Age at first tooth\' and \'Number of teeth\', indicating link between height and tooth eruption.)", "entity_type": "UNKNOWN", "id": "RS1570106"}, {"source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "description": "The allelic scores for \'Age at first tooth\' and \'Number of teeth\', derived from genome-wide significant SNPs, show moderate evidence of association with height.)", "entity_type": "UNKNOWN", "id": "GENOME-WIDE SIGNIFICANT SNPS FOR AGE AT FIRST TOOTH AND NUMBER OF TEETH"}, {"source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "description": "The allelic scores for \'Age at first tooth\' and \'Number of teeth\', derived from genome-wide significant SNPs, show moderate evidence of association with height.)", "entity_type": "UNKNOWN", "id": "HEIGHT AT 17 YEARS AND HEIGHT AT 31 YEARS"}, {"source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "description": "SNPs associated with \'Age at first tooth\' show moderate evidence of association with height.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION WITH HEIGHT"}, {"source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "description": "SNPs associated with \'Age at first tooth\' show moderate evidence of association with height.)", "entity_type": "UNKNOWN", "id": "SNPS FOR AGE AT FIRST TOOTH"}, {"source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "description": "Variants in the HMGA2 and BMP4 genes contribute significantly to associations between \'Age at first tooth\' SNPs and height.)", "entity_type": "UNKNOWN", "id": "CONTRIBUTION TO AGE AT FIRST TOOTH SNP ASSOCIATIONS"}, {"source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "description": "This threshold provides the most significant over-representation of genes in pathways for \'age at first tooth\' GWAS.)", "entity_type": "UNKNOWN", "id": "PATHWAY ANALYSIS"}, {"source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "description": "This threshold provides the most significant over-representation of genes in pathways for \'age at first tooth\' GWAS.)", "entity_type": "UNKNOWN", "id": "SNP ASSOCIATION P-VALUE THRESHOLD OF 0.005"}, {"entity_type": "GENE", "description": "IL-10R\u03b1 deletion restricted to T cells (IL-10R\u03b1Tdel) does not impact disease severity in this model of colitis.<SEP>IL-10R\u03b1Tdel mice produce selective deletion of the IL-10 receptor \u03b1 restricted to T cells.<SEP>IL-10R\u03b1Tdel mice show a genomic variant restricted to T cells with selective deletion of the IL-10 receptor \u03b1.<SEP>The deletion of IL-10 receptor \u03b1 restricted to T cells (IL-10R\u03b1Tdel) affects the transcripts in T cells, impacting their role in disease development.<SEP>The interaction between IL-10 and its receptor \u03b1 in T cells (IL-10R\u03b1Tdel) leads to disease development.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "IL-10R\u0391TDEL"}, {"entity_type": "GENE", "description": "Deletion of IL-10 receptor \u03b1 specifically in DCs (IL-10R\u03b1DCdel) does not affect disease quality or magnitude.<SEP>Deletion of IL-10 receptor \u03b1 specifically in DCs (IL-10R\u03b1DCdel) influences the transcripts within DCs, potentially altering disease manifestation.<SEP>IL-10 receptor \u03b1 deletion restricted to DCs (IL-10R\u03b1DCdel) affects disease magnitude and quality.<SEP>IL-10R\u03b1DCdel mice generate lineage-specific knockout of IL-10 receptor \u03b1 in DCs.<SEP>IL-10R\u03b1DCdel mice have a genomic variant that results in lineage-specific knockout of IL-10 receptor \u03b1 in DCs.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "IL-10R\u0391DCDEL"}, {"entity_type": "GENOMIC VARIANT", "description": "A genomic variant in IL-10R\u03b1Mdel modulates disease severity in Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 colitis models.<SEP>IL-10R\u03b1Mdel mice exhibit a genomic variant leading to selective deletion of IL-10 receptor \u03b1 restricted to macrophages.<SEP>IL-10R\u03b1Mdel mice produce lineage-specific deletion of the IL-10 receptor \u03b1 restricted to macrophages.<SEP>Selective deletion of IL-10 receptor \u03b1 in macrophages (IL-10R\u03b1Mdel) impacts colitis development.<SEP>Selective deletion of IL-10 receptor \u03b1 restricted to macrophages (IL-10R\u03b1Mdel) impacts transcripts in macrophages, contributing to the intensification of colitis development.<SEP>Selective deletion of the IL-10 receptor \u03b1 restricted to macrophages (IL-10R\u03b1Mdel) intensifies colitis development.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "IL-10R\u0391MDEL"}, {"entity_type": "GENE", "description": "Wild-type (WT) refers to non-modified or standard conditions in comparison to mutants.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "WT"}, {"entity_type": "GENE", "description": "Rag1\u2212/\u2212 mice have mutations that lack Rag-1, a lymphocyte-specific receptor gene.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "RAG1\u2212/\u2212"}, {"entity_type": "TRANSCRIPT", "description": "Naive CD4+CD25\u2212CD45RBhi T cell transcripts are influenced by lineage-specific IL-10 interactions in models of colitis.<SEP>Naive CD4+CD25\u2212CD45RBhi T cells refer to specific subsets of the immune system\'s T-cell population.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "NAIVE CD4+CD25\u2212CD45RBHI T CELLS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Th1, Th17, and FoxP3+ T cell subsets are involved in the lineage-specific effects of IL-10 on colitis development.<SEP>The transcripts of specific subsets of T cells are involved in lineage-specific interactions with IL-10 in models of colitis.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "TH1, TH17, FOXP3+ T CELL SUBSETS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Rag1\u2212/\u2212 recipients are the experimental subjects in which Rag1 mutations lead to no Rag-1, impacting T cell responses.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "RAG1\u2212/\u2212 RECIPIENTS"}, {"entity_type": "RELATIONSHIP WITH DISEASE DEVELOPMENT", "description": "The lack of IL-10 signaling into transferred T cells does not impact disease severity in this model of T cell transfer colitis.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "IL-10 SIGNALING INTO TRANSFERRED T CELLS"}, {"entity_type": "RELATIONSHIP WITH DISEASE DEVELOPMENT", "description": "Comparison between DC and T cell-specific IL-10 signaling reveals differential effects on disease quality and magnitude.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "DC AND T CELL IL-10 SIGNALING"}, {"entity_type": "DISEASE MODEL", "description": "Here too, no differences in population sizes were seen (Fig. 3d). To determine if T cell IL-10 production was itself functionally dispensable for the differential colitis...<SEP>IL-10R\u03b1MdelRag1\u2212/\u2212 mice represent a model of colitis development when IL-10 receptor \u03b1 is selectively deleted in macrophages.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51<SEP>chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "id": "IL-10R\u0391MDELRAG1\u2212/\u2212 MICE"}, {"entity_type": "GENETIC MARKER", "description": "The CX3CR1-driven Cre system expresses Cre recombinase under the control of a CX3CR1 promoter, affecting gene disruption specifically within granulocytes and macrophages.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "CX3CR1-DRIVEN CRE"}, {"entity_type": "GENOMIC VARIANT", "description": "A particularly prominent elevation in IL-1\u03b2 production was apparent...<SEP>LPM (Lung Pulmonary Macrophages) cells interact with IL-1\u03b2 to influence inflammation, specifically Th17 cell development during colitis.<SEP>LPM\u03d5s are lineage-specific cells that express the Cre transgene for IL-10R\u03b1 deletion.<SEP>LPM\u03d5s, derived from diseased IL-10R\u03b1MdelRag1\u2212/\u2212 mice or control Rag1\u2212/\u2212 mice, were used in co-culture with na\u00efve T cells.<SEP>Lamina propria macrophages (LPM\u03d5s) were significantly increased and activated in IL-10R\u03b1MdelRag1\u2212/\u2212 mice with colitis, as compared to Rag1\u2212/\u2212 controls.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51<SEP>chunk-b971aa450a9b220806bdc000eb5efaad<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-8fb20f873daf79a9b7d6e8d25c8889d6<SEP>chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "LPM\u03a6S"}, {"entity_type": "PATHOLOGICAL CONDITION", "description": "Spontaneous development of colitis without external stimulation or intervention.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "SPONTANEOUS COLITIS"}, {"entity_type": "RELATIONSHIP WITH PATHOLOGICAL CONDITIONS", "description": "Selective deletion of IL-10 receptor \u03b1 restricted to macrophages leads to spontaneous colitis development during specific aging periods.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "IL-10 RECEPTOR \u0391 DELETION IN MACROPHAGES (IL-10R\u0391MDEL) AND SPONTANEOUS COLITIS"}, {"entity_type": "THERAPEUTIC INTERVENTION", "description": "Depletion of neutrophils using specific antibodies aimed at Ly6G markers as a method for experimental control or intervention in disease models.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "NEUTROPHIL DEPLETION WITH ANTI-LY6G ANTIBODIES"}, {"entity_type": "COMPARISON WITH CONTROL CONDITIONS", "description": "Comparative analysis of clinical disease manifestation and histological pathology between IL-10R\u03b1MdelRag1\u2212/\u2212 mice and Rag1\u2212/\u2212 controls to understand the role of M\u03d5-specific IL-10 receptor \u03b1 deletion.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "CLINICAL DISEASE AND HISTOPATHOLOGY IN IL-10R\u0391MDELRAG1\u2212/\u2212 MICE VS. RAG1\u2212/\u2212 CONTROLS"}, {"entity_type": "PROTECTIVE MECHANISM", "description": "The protective effects of IL-10 against disease, as mediated by its interactions with specific immune cell types (e.g., macrophages and DCs), contribute to the overall immunoregulatory processes.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "IL-10-MEDIATED IMMUNOPROTECTION"}, {"entity_type": "CONTRIBUTING FACTOR FOR PROTECTIVE MECHANISM", "description": "The M\u03c6-specific responsiveness to IL-10 plays a critical role in its ability to protect against disease, accounting for much of its protective effects.)(entity", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "M\u03a6-SPECIFIC EFFECT OF IL-10 PROTECTION"}, {"entity_type": "PROTEIN", "description": "Myeloid differentiation primary response gene 88 (MyD88), an adaptor protein that plays a critical role in innate immune responses and downstream signaling of many pattern recognition receptors including Toll-like receptors (TLRs).", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "MYD88"}, {"entity_type": "PROTEIN FAMILY", "description": "A family of transmembrane proteins, the Toll-like receptors (TLRs), which recognize pathogen-associated molecular patterns (PAMPs) and initiate an immune response.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "TOLL-LIKE RECEPTORS (TLR)"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "The process by which cytokines transmit signals from cell to cell, influencing the activity of immune cells and modulating various biological responses.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "CYTOKINE SIGNALING"}, {"entity_type": "CELL TYPE", "description": "Cells that process antigens and present them on their surface for recognition by T cells during an immune response.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "ANTIGEN-PRESENTING CELLS (APCS)"}, {"entity_type": "TRANSCRIPT", "description": "Lymphocytes responsible for antibody production, playing a critical role in the adaptive immune system.<SEP>There was no signal detected in B cell data as re-analyzed from Fairfax et al.<SEP>The count of B cells is within the normal range with elevated levels of serum IgG, IgA, and IgM throughout development. However, serum IgE remains consistently elevated due to chronic eczema and reactive airway disease.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51<SEP>chunk-e323b0d4f82733a78f687e51f47b500a<SEP>chunk-1b40d071abac0ba432a9fb0096796f45", "id": "B CELLS"}, {"entity_type": "SUBTYPE OF T CELLS", "description": "A subtype of T-helper cells that are involved in cell-mediated immunity and play roles in activating other immune responses.>", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "T HELPER 1 (TH1) CELLS"}, {"entity_type": "DISEASE", "description": "Increased numbers of activated IL-10R\u03b1MdelRag1\u2212/\u2212 lamina propria macrophages were identified in mice with colitis compared to Rag1\u2212/\u2212 controls.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "IL-10R\u0391MDELRAG1\u2212/\u2212"}, {"entity_type": "SURFACE MARKER", "description": "The LPM\u03d5s displayed uniform expression of CD64, indicating activation and differentiation into a pro-inflammatory phenotype.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "CD64+ "}, {"entity_type": "CELL", "description": "A mixed population of T cells, responsible for either psoriasiform or eczematous reactions.<SEP>IL-10 production by transferred T cells did not influence disease magnitude clinically or histologically (Fig. 3e, f). Therefore, the M\u03d5 IL-10 response does not identifiably impact regulatory T cell presence.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a<SEP>chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "id": "T CELLS"}, {"entity_type": "TRANSCRIPT", "description": "Significantly increased percents and absolute numbers of CD4+IL-17A+ T cells were identified...", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "clusters": "[{level: 0, cluster: 16}]", "id": "TH17 CELLS"}, {"entity_type": "PROTEIN", "description": "Increased colonic infiltration by both Th1 and Th17 cells, and the Th17 response is required for disease development.", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "id": "CD4+IL-17A+ T CELLS"}, {"entity_type": "NUTRIENT", "description": "The elevated production of Th17-promoting cytokines was specific to the M\u03c6 population, indicating that M\u03c6s are the primary source for the increased Th17-associated cytokines.<SEP>The elevated production of Th17-promoting cytokines was specific to the M\u03d5 population, indicating that M\u03d5s are the primary source for the increased Th17-associated cytokines.", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "id": "TH17-PROMOTING CYTOKINES"}, {"entity_type": "DIETARY PATTERN", "description": "Elevated expression was specific to the M\u03c6s, indicating that M\u03c6s are the primary source for the increased Th17-associated cytokines and implying that IL-10 acts directly on these cells to suppress their production.<SEP>Macrophages are part of the innate immune system that can respond to various stimuli, including cytokines like IL-6 or IL-10; they\'re pivotal in understanding autoimmune inflammation and pathogenesis.", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6<SEP>chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 57}]", "id": "M\u03a6 (MACROPHAGES)"}, {"entity_type": "GENE", "description": "A particularly prominent elevation in IL-1\u03b2 production was apparent...<SEP>Interleukin 1 beta (IL-1\u03b2) is a pro-inflammatory cytokine that has essential functions in the steady-state development of intestinal Th17 cells and potentially contributes to the pathogenesis of colitis.<SEP>Treatment with IL-10 led to the complete suppression of IL-1\u03b2 maturation.<SEP>Interleukin-1 Beta (IL-1\u03b2) is expressed at high levels in PATs from SENP1-aP2KO mice under normal chow and HFD conditions.<SEP>The study also included examination of SNPs in the IL-1\u03b2 gene as they relate to CRC risk.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad<SEP>chunk-174831b97ed170039b374ba254d4f506<SEP>chunk-8fb20f873daf79a9b7d6e8d25c8889d6<SEP>chunk-35d8a4ea86cb92d18ee1c857a318d2d8<SEP>chunk-bd81c38d10d8773bc4076cbb20bc289f", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 56}]", "id": "IL-1\u0392"}, {"entity_type": "DISEASE", "description": "Here too, no differences in population sizes were seen (Fig. 3d). To determine if T cell IL-10 production was itself functionally dispensable for the differential colitis...", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "id": "RAG1\u2212/\u2212 MICE"}, {"entity_type": "TRANSCRIPT", "description": "IL-10 production by transferred T cells did not influence disease magnitude clinically or histologically (Fig. 3e, f). Therefore, the M\u03d5 IL-10 response does not identifiably impact regulatory T cell presence.", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "id": "TRANSFERRED T CELLS"}, {"entity_type": "INOS", "description": "For IL-1\u03b2, IL-6, IL-23p19, and iNOS, relative expression was compared in sorted colonic LPM\u03c6s, DCs, neutrophils, and epithelial cells.", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "id": "IL-23P19"}, {"entity_type": "TRANSCRIPT", "description": "ROR\u03b3T, and hence Th17 cells, proved essential for colitis in both recipient types.", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "id": "ROR\u0393T\u2212/\u2212 T CELLS"}, {"entity_type": "CELL", "description": "IL-10 can act on LPM\u03c6s to directly impair their support for Th17 production through downregulation of IL-1 and to a lesser extent IL-6.<SEP>Th17 cells are critical for colitis development and disease progression, depending on IL-6R\u03b1 signaling.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8<SEP>chunk-b971aa450a9b220806bdc000eb5efaad", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 58}]", "id": "TH17 CELL"}, {"entity_type": "PROTEIN", "description": "IL-6 Receptor Alpha (IL-6R\u03b1) is the receptor that binds to IL-6 protein. Its role in T cell responses was assessed in this context.<SEP>IL-6R\u03b1 gene plays a role in disease protection when T cell transfers are performed.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 58}]", "id": "IL-6R\u0391"}, {"entity_type": "CELL", "description": "Th1 cells are another type of immune response cells that were analyzed alongside Th17 cells for their role in colitis development.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 57}]", "id": "TH1 CELL"}, {"entity_type": "CELL", "description": "Regulatory T cells characterized by FoxP3 expression play a role in balancing immune responses; they\'re relevant to the study of autoimmune diseases and inflammation.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "FOXP3+ CD4+ T CELLS"}, {"entity_type": "CELL", "description": "T cells lacking IL-1 receptor (IL-1R) were used as a test for their impact on disease induction in the context of colitis.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "IL-1R\u2212/\u2212 T CELLS"}, {"entity_type": "PROTEIN", "description": "After treatment with IL-10, pro-IL-\u03b2 was diminished and mature IL-1\u03b2 undetectable in Rag1\u2212/\u2212 cultures.<SEP>IL-10 is a nutrient in the form of a dietary pattern that inhibits pro-IL-1\u03b2 production and modulates disease.<SEP>Interleukin 10 (IL-10) plays important roles in modulating immune responses, such as dampening inflammation and promoting M\u03c6 responses to IL-10, which may affect disease development.<SEP>Pre-treatment with IL-10 abrogated the induction caused by LPS and ATP stimulation in Rag1\u2212/\u2212 M\u03d5s.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-35d8a4ea86cb92d18ee1c857a318d2d8<SEP>chunk-b971aa450a9b220806bdc000eb5efaad", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 57}]", "id": "IL-10"}, {"entity_type": "DISEASE", "description": "Inflammatory bowel disease characterized by chronic inflammation of the colon; it was studied for its dependence on certain immune system components like Th17 cells and IL-6.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "COLITIS"}, {"entity_type": "DISEASE", "description": "Conditions that occur when the body\'s immune system mistakenly attacks healthy tissue, which could involve M\u03c6 responses or other immunological responses.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "AUTOIMMUNE DISEASE"}, {"entity_type": "RESPONSE", "description": "Th17 response refers to the activation of Th17 cells, which are essential in autoimmune disorders and inflammatory conditions like colitis.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "TH17 RESPONSE"}, {"entity_type": "STATE", "description": "The condition where the immune system mistakenly attacks healthy tissue, leading to various diseases including colitis.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 55}]", "id": "AUTOIMMUNITY"}, {"entity_type": "CELL", "description": "Macrophages are key components of innate immunity and play vital roles in inflammation and tissue repair processes.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 58}]", "id": "MACROPHAGE (M\u03a6)"}, {"entity_type": "T CELL TYPE", "description": "Regulatory T cells, characterized by their FoxP3 expression, control immune responses and maintain immune tolerance to prevent autoimmunity.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 57}]", "id": "REGULATORY T CELLS"}, {"entity_type": "SYSTEM", "description": "The part of the immune system that provides immediate defense against pathogens and damage, including macrophages and neutrophils.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 56}]", "id": "INNATE IMMUNE SYSTEM"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "Macrophages and dendritic cells process antigens for presentation to T cells, playing a critical role in adaptive immunity.<SEP>Process by which antigen-presenting cells (APCs) present antigen fragments to T cells for recognition and activation.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0<SEP>chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "ANTIGEN PRESENTATION"}, {"entity_type": "GENE", "description": "IL-1 receptor gene contributes to diminished clinical and histologic measures of colitis when T cell transfers are performed.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 56}]", "id": "IL-1R"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Caspase-1 cleavage was assessed in relation to its activation form under different treatments on macrophages.<SEP>Caspase-1 is a protein involved in the activation of pro-inflammatory cytokine production by cleaving inactive cytosolic precursor (pro-IL-1\u03b2) to its active form, supporting Th17 cell response and disease progression.<SEP>In the IL-10R\u03b1MdelRag1\u2212/\u2212 M\u03d5s, p20 formation was increased regardless of IL-10 or anti-IL-10R\u03b1 Ab treatment.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-b971aa450a9b220806bdc000eb5efaad", "clusters": "[{level: 0, cluster: 18}]", "id": "CASPASE-1"}, {"entity_type": "MOLECULAR INTERACTION", "description": "An inflammasome is a multiprotein complex that activates Caspase-1 in response to danger signals, leading to the processing of pro-inflammatory cytokines like IL-1\u03b2.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "INFLAMMASOME"}, {"entity_type": "CELLULAR COMPONENT", "description": "The CD11b surface marker on cells marks those that produce or are involved with pro-IL-1\u03b2 production and inflammation management in colitis models, associated with LPM\u03d5s.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "CD11B"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The process of colocalization refers to the overlapping location or function of molecules like IL-1\u03b2 and CD11b within cells or tissues during disease conditions.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "COLOCALIZE"}, {"entity_type": "ANALYTICAL METHOD", "description": "Flow cytometry is a technique used for measuring the physical and chemical properties of particles suspended in fluid, particularly useful for analyzing cell populations and quantifying protein expression like pro-IL-1\u03b2 levels.<SEP>Technique used for quantitative analysis of the phenotype of cells, including expression levels of specific markers or antigens.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "FLOW CYTOMETRY"}, {"entity_type": "METHOD", "description": "Western analysis or Western blotting is a method used to detect proteins by size using an antibody probe. It\'s applied here to study the presence of both pro-IL-1\u03b2 and IL-1\u03b2 in M\u03d5s (macrophages).", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "WESTERN ANALYSIS"}, {"entity_type": "STIMULATION AGENT", "description": "Lipopolysaccharide (LPS) is a component found on bacterial cell walls that can stimulate innate immune responses including the expression of pro-inflammatory cytokines like IL-1\u03b2.<SEP>Lipopolysaccharide stimulation led to a modest increase in activated caspase-1.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-b971aa450a9b220806bdc000eb5efaad", "clusters": "[{level: 0, cluster: 18}]", "id": "LPS"}, {"entity_type": "STIMULATION AGENT", "description": "Adenosine triphosphate (ATP), often present in many cellular sources, has been shown to induce inflammasome activation and promote inflammatory responses associated with cytokine production.<SEP>Adenosine triphosphate also stimulated the increase of activated caspase-1.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-b971aa450a9b220806bdc000eb5efaad", "clusters": "[{level: 0, cluster: 18}]", "id": "ATP"}, {"entity_type": "DRUG/CHEMICAL", "description": "An antibody that specifically binds to the IL-10 receptor can modulate signaling pathways by blocking IL-10\'s interaction, affecting pro-inflammatory cytokine expression like IL-1\u03b2 in cells.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "ANTI-IL-10R ANTIBODY"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "The process of protein expression refers to the production and secretion of proteins in cells, which is regulated and controlled during inflammation and disease states, impacting cytokine levels including IL-1\u03b2.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "PROTEIN EXPRESSION"}, {"entity_type": "TRANSCRIPT", "description": "The transcript of pro-IL-\u03b2 is involved in the regulation of IL-1\u03b2 maturation.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "clusters": "[{level: 0, cluster: 10}, {level: 1, cluster: 32}]", "id": "PRO-IL-\u0392"}, {"entity_type": "CELL", "description": "Rag1\u2212/\u2212 macrophages (M\u03d5s) respond differently under various treatments in comparison to other cell types.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "clusters": "[{level: 0, cluster: 18}]", "id": "RAG1\u2212/\u2212 M\u03a6S"}, {"entity_type": "CELL", "description": "The IL-10 receptor \u03b1 subunit knockout M\u03d5s of Rag1\u2212/\u2212 mice, which affects the signaling pathways related to IL-10.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "IL-10R\u0391MDELRAG1\u2212/\u2212 M\u03a6S"}, {"entity_type": "ANTIBODY", "description": "Blocking anti-IL-10R\u03b1 antibody markedly increased activated caspase-1, indicating normal restraint of caspase-1 activation by autocrine IL-10.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "ANTI-IL-10R\u0391 AB"}, {"entity_type": "CELL", "description": "These macrophages from IL-10R\u03b1MdelRag1\u2212/\u2212 mice showed elevated Th17 formation compared to Rag1\u2212/\u2212 LPM\u03d5s.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "IL-10R\u0391MDELRAG1\u2212/\u2212 LPM\u03a6S"}, {"entity_type": "EXPERIMENTAL TREATMENT", "description": "Th17 formation was found to be more modest after IL-6 receptor knockout T cell transfer.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "IL-6R\u2212/\u2212 T CELL TRANSFER"}, {"entity_type": "DRUG TREATMENT", "description": "Blocking the IL-6 receptor largely abrogated Th17 formation with Rag1\u2212/\u2212 LPM\u03d5s.<SEP>This specific type of antibody was used to block the effect of IL-6 on Th17 cell maturation when present in co-cultures with LPM\u03c6s from Rag1\u2212/\u2212 mice.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "\u0391IL-6R\u0391 BLOCKING ANTIBODY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Inhibition of IL-1 signaling led to more substantial inhibition of Th17 development in IL-10R\u03b1MdelRag1\u2212/\u2212-derived LPM\u03d5s.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "clusters": "[{level: 0, cluster: 17}, {level: 1, cluster: 58}]", "id": "IL-1 SIGNALING"}, {"entity_type": "PROTEIN", "description": "TGF\u03b2 was added to co-cultures of LPM\u03c6s with na\u00efve T cells but not IL-6.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "TGF\u0392"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "The maturation process of Th17 cells involves the interaction between different cell types and cytokines, including IL-10R\u03b1 signaling in LPM\u03c6s.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "TH17 CELL MATURATION"}, {"entity_type": "CELL LINE", "description": "T cell lines with a genetic defect that lack functional IL-6 receptors were used in some of the experiments to evaluate Th17 cell maturation.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "IL-6R\u2212/\u2212 T CELLS"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "IL-6 signaling pathways were inhibited using specific blocking antibodies to observe its effect on Th17 cell development.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "IL-6 SIGNALING"}, {"entity_type": "OUTCOME OF THE BIOLOGICAL PROCESS", "description": "The process of Th17 cell formation was essentially abrogated when both IL-1 and IL-6 signaling were blocked, demonstrating their roles in this cellular response.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "TH17 DEVELOPMENT"}, {"source_id": "chunk-29025895ac187b8e88dec5cea9478842<SEP>chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "IL-17 is involved in the immune response but no difference was observed regarding its production between experimental groups.<SEP>Increased colonic infiltration by both Th1 and Th17 cells, and the Th17 response is required for disease development.)", "entity_type": "PROTEIN", "clusters": "[{level: 0, cluster: 16}]", "id": "IL-17"}, {"source_id": "chunk-84ec948f22126cef7d93b1d7465b79db<SEP>chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-0c1631ae2c195cb68139ea9d75a3b433<SEP>chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "No differences in CD4+IFN-\u03b3+ T cell quantities were evident.)<SEP>No significant elevation of IFN-\u03b3 was observed in the experiment.<SEP>The expression of IFN-\u03b3 was higher in certain CD subjects compared to others without specific genotypes.<SEP>The protein encoded by PTPN2/PTPN22, involved in the regulation of immune responses.", "entity_type": "PROTEIN", "clusters": "[{level: 0, cluster: 16}]", "id": "IFN-\u0393"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "A particularly prominent elevation in IL-1\u03b2 production was apparent...)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 16}]", "id": "IL-23"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "For IL-1\u03b2, IL-6, IL-23p19, and iNOS, relative expression was compared in sorted colonic LPM\u03d5s, DCs, neutrophils, and epithelial cells.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 16}]", "id": "INOS"}, {"source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "description": "The complex interactions between different proteins like IL-6R\u03b1 and its ligand(s), which are crucial for processes including immune response regulation and disease development.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 39}]", "id": "PROTEIN-PROTEIN INTERACTION"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "A genomic variant specific to the IL-6R\u03b1 gene influences disease protection during T cell transfers.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 42}]", "id": "IL-6R\u0391 GENE"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "Treatment with an anti-IL-10 receptor antibody influences pro-IL-1\u03b2 expression by modulating signaling pathways in T cell responses and disease progression.)", "entity_type": "UNKNOWN", "id": "DRUG/CHEMICAL INTERACTION"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "The IL-6R\u03b1 gene\'s response is modified through interaction with genomic variants, impacting physiological processes that affect colitis models specifically in the Rag1\u2212/\u2212 context.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 42}]", "id": "PATHWAY MODIFICATION"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "Specific genotypic differences within the IL-6R\u03b1 gene and IL-10R\u03b1Mdel variants influence disease modulations across different genetic backgrounds like Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 7}, {level: 1, cluster: 26}]", "id": "DISEASE MODULATION"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "Caspase-1 is activated through an inflammasome response to danger signals or stimulation by agents like LPS and ATP, leading to the processing of pro-inflammatory cytokines such as IL-1\u03b2.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 18}]", "id": "PATHWAY ACTIVATION"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "The use of flow cytometry enables researchers to collect data on cell populations and protein expression levels like that of pro-IL-1\u03b2, aiding in understanding complex immune responses.)", "entity_type": "UNKNOWN", "id": "DATA COLLECTION METHOD"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "Western analysis serves as a method for visualizing proteins by size after they have been labeled with an antibody. It is crucial for confirming cytokine production levels like IL-1\u03b2 in response to different stimuli.)", "entity_type": "UNKNOWN", "id": "PROTEIN DETECTION METHOD"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "Exposure to LPS leads to activation of signaling pathways that impact pro-inflammatory responses and cytokine expression, such as those involved with IL-1\u03b2 production during colitis.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 18}]", "id": "SIGNALING PATHWAY ACTIVATION"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "The presence or release of ATP can stimulate cellular responses leading to inflammasome activation which impacts the expression levels of pro-inflammatory cytokines, like IL-1\u03b2, in inflammatory conditions such as colitis.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 18}]", "id": "STIMULATION EFFECT"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "Antibodies that bind to and disrupt IL-10 receptor function can alter signaling pathways that affect pro-inflammatory cytokine production, including IL-1\u03b2 levels in inflammatory contexts.)", "entity_type": "UNKNOWN", "id": "PATHWAY DISRUPTION"}, {"source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "description": "The transfer of IL-6 receptor knockout (IL-6R\u2212/\u2212) T cells into mice showed modest effects on Th17 formation, suggesting the importance of functional IL-6R for this process.)", "entity_type": "UNKNOWN", "id": "EXPERIMENTAL CONDITION"}, {"source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "description": "The transfer of IL-6 receptor knockout (IL-6R\u2212/\u2212) T cells into mice showed modest effects on Th17 formation, suggesting the importance of functional IL-6R for this process.)", "entity_type": "UNKNOWN", "id": "IL-6R\u2212/\u2212 T CELLS TRANSFER"}, {"source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "description": "Blocking the interaction with IL-6 receptor \u03b1 (IL-6R\u03b1), using specific antibodies, largely inhibited Th17 formation in cultures containing Rag1\u2212/\u2212 derived LPM\u03c6s.)", "entity_type": "UNKNOWN", "id": "EFFECT ON TH17 FORMATION"}, {"source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "description": "Inhibition of IL-6 signaling pathways led to a significant reduction or complete suppression of Th17 cell development, emphasizing the role of IL-6 in this process.)", "entity_type": "UNKNOWN", "id": "CONSEQUENCE ON TH17 DEVELOPMENT"}, {"source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "description": "Inhibition of IL-6 signaling pathways led to a significant reduction or complete suppression of Th17 cell development, emphasizing the role of IL-6 in this process.)", "entity_type": "UNKNOWN", "id": "IL-6 SIGNALING INHIBITION"}, {"entity_type": "COHORT", "description": "The TwinsUK cohort is composed of 1,176 samples including 169 monozygotic (MZ) and 419 dizygotic (DZ) twin pairs.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "TWINSUK"}, {"entity_type": "COHORT", "description": "The Netherlands Twin Registry (NTR) cohort includes only MZ twins, totaling N=312 individuals.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "NETHERLANDS TWIN REGISTRY (NTR)"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "DNA extraction method significantly contributes to heterogeneity in microbiome variation, accounting for 29% of the effect size on microbiome variation based on stepwise distance-based redundancy analysis (R2adjDNAext=0.27, Padj=7\u00d710\u22124).", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "DNA EXTRACTION METHOD"}, {"entity_type": "LANDSCAPE", "description": "The landscape of microbiome composition across different cohorts varies based on ethnic background, age, male/female ratio and microbiome analysis methodology.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "MICROBIAL COMPOSITION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Cohort ethnicity affects the heterogeneity in microbial composition.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "COHORT ETHNICITY"}, {"entity_type": "TECHNIQUE", "description": "Accounting for differences in sequencing depth through rarefaction allowed for taxonomic classification using direct taxonomic binning instead of OTU clustering methods.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "SEQUENCING DEPTH"}, {"entity_type": "METHODOLOGY", "description": "Direct taxonomic binning was performed for taxonomic classification instead of OTU clustering to analyze microbiome data.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "TAXONOMIC CLASSIFICATION"}, {"entity_type": "VARIANCE COMPONENT ANALYSIS", "description": "The 16S rRNA domain sequenced and DNA extraction methods used, together with cohort ethnicity, accounted for 32.74% of richness variance in the study.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "RICHNESS VARIANCE"}, {"entity_type": "FACTOR", "description": "The high heterogeneity of microbial composition across different cohorts is influenced by ethnic background, age distribution, sex ratio, and technical methodologies.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "COHORT HETEROGENEITY"}, {"entity_type": "METRIC", "description": "Alpha diversity metrics such as Shannon index, Simpson\'s index, and inverse Simpson index are important indicators of species richness and evenness in each cohort.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "ALPHA DIVERSITY METRICS"}, {"entity_type": "STUDY COMPONENT", "description": "The study includes twin cohorts to estimate heritability of gut microbiome composition.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "TWIN COHORTS"}, {"entity_type": "STATISTICAL METHOD", "description": "Heritability of bacterial taxa was estimated using SNP-based heritability analysis.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "SNP-BASED HERITABILITY"}, {"entity_type": "INFLUENCE", "description": "The additive genetic variance (A), shared environmental factors (C), and non-shared factors contribute to the variation in microbiome composition.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "GENETIC FACTORS"}, {"entity_type": "STATISTICAL VALIDATION", "description": "Twin-based analysis was replicated using another twin registry cohort for validation purposes.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "REPLICATION OF TWIN-BASED ANALYSIS"}, {"entity_type": "HERITABILITY MEASURE", "description": "Out of 159 bacterial taxa present in over 10% of twin pairs, 19 showed evidence of heritability based on the nominal P-value.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "TAXON HERITABILITY"}, {"entity_type": "STATISTICAL TOOL", "description": "The SNP-based heritability was calculated using LD score regression to account for linkage disequilibrium among genetic variants.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "LINKAGE DISEQUILIBRIUM (LD) SCORE REGRESSION"}, {"entity_type": "GENE", "description": "The LCT gene is located at 2q21.3 and includes a large block of about 1.5Mb with multiple protein-coding genes.<SEP>The LCT gene plays a role in lactase regulation.<SEP>The gene LCT encodes for the lactase enzyme which is responsible for breaking down lactose.<SEP>The gene encoding lactase, necessary for the digestion of lactose.<SEP>This transcript is associated with lactase persistence and can be regulated by the lncRNA in question.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c<SEP>chunk-af413a33fae8d27bdb15cd18447dfad4<SEP>chunk-ec104044b3aa65560be8d81dbb3940d6<SEP>chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "LCT"}, {"entity_type": "GENOMIC VARIANT", "description": "The rs182549 genotype showed a higher abundance of the genus Bifidobacterium in the adult group, but not in the younger group.<SEP>rs182549 is the strongest association signal identified near the LCT gene, which acts as a perfect proxy for functional LCT variant rs4988235.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46<SEP>chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "RS182549"}, {"entity_type": "DISEASE", "description": "Bifidobacterium abundance has been associated with variants located in the LCT locus through genome-wide association studies (GWAS).<SEP>The age-genotype interaction was significant in the GEM_v12 and GEM_ICHIP subcohorts, both comprising mostly European individuals<SEP>The genus Bifidobacterium showed a significant association with a protective effect against ulcerative colitis.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46<SEP>chunk-af413a33fae8d27bdb15cd18447dfad4<SEP>chunk-159d4c797dae6eb825c74067af703228", "id": "BIFIDOBACTERIUM"}, {"entity_type": "STATISTICAL SIGNIFICANCE", "description": "Cochran\'s Q test p-values indicated the statistical significance of heterogeneity across different cohorts in rs182549 related analysis.", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "COCHRAN\'S Q P"}, {"entity_type": "MICROBIAL ABUNDANCE", "description": "The abundance of Bifidobacterium has been positively correlated with milk product intake, especially when individuals are homozygous for the low-function LCT haplotype.", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "BIFIDOBACTERIUM ABUNDANCE"}, {"entity_type": "GENOMIC VARIANT", "description": "No significant association is observed between the rs4988235 genotype and risk of developing diabetes when considering baseline HbA1c levels or through multivariable adjustment.<SEP>The LP (rs4988235 T/T genotype) is associated with a range of 7.4% to 53.9% in prevalence across Europe.<SEP>The homozygous LP (T/T) genotype shows differences within Europe, with a range from 7.4% to 53.9% in Italy and Sweden respectively.<SEP>rs4988235 is a functional variant of rs182549 that has been associated with abundance levels of Bifidobacterium.<SEP>The minor allele frequencies for rs4988235 were observed in different populations.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91<SEP>chunk-af413a33fae8d27bdb15cd18447dfad4<SEP>chunk-7a423a439657efc1126fb6a075919ea5", "id": "RS4988235"}, {"entity_type": "ASSOCIATION SIGNAL", "description": "The LCT locus showed an association signal for Bifidobacterium abundance, identified through multi-ethnic genome-wide association studies (GWAS).", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "LCT MBQTL EFFECT"}, {"entity_type": "ETHNIC DIVERSITY", "description": "The GEM_v24 cohort includes individuals of diverse Israeli subethnicities as part of their composition.", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "GEM_V24 COHORT - ISRAELI SUBETHNICITIES"}, {"entity_type": "AGE COMPOSITION", "description": "The multi-ethnic GEM cohort comprises participants aged between 6 to 35, nearly half of whom are younger than 16 years old.", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "AGE RANGE BETWEEN 6 AND 35 YEARS"}, {"entity_type": "GENOTYPE VARIANT", "description": "This variant is part of the rs182549 locus and was found to have a significant interaction with age regarding Bifidobacterium abundance.", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "NC_000002.11:G.136616754CC"}, {"entity_type": "SUBGROUP COMPOSITION", "description": "These subcohorts are comprised mainly of European individuals when analyzing age-genotype interactions on Bifidobacterium abundance.", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "GEM_V12, GEM_ICHIP SUBCOHORTS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "While the GEM_v24 cohort, mainly composed of individuals of different Israeli subethnicities (see Online Methods) who live in Israel, showed neither an mbQTL effect.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "GEM_V12"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Neither did the GEM_ICHIP cohort", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "GEM_ICHIP"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The lack of an LCT mbQTL effect in adults was also observed in another Israel cohort (Personalized Nutrition Project (PNP), 481 adults, Beta = \u22120.20 [95%CI: \u22120.61, 0.20]).", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "GEM_V24"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The cohort that reported the lowest LCT effect sizes were the two cohorts of Israeli ethnicity volunteered in Israel (GEM_v24, PNP) and a child cohort.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "COPENHAGEN PROSPECTS STUDIES ON ASTHMA IN CHILDHOOD (COPSAC)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The tagging NC_000019.9:g.49218060C>T (rs35866622 non-secretor) allele was associated with a reduced abundance of the R. torques group and a decreased presence of the R. gnavus group.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "FUNCTIONAL FUT2 VARIANT RS601338"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Ruminococcus sp. are specialized in the degradation of complex carbohydrates, thereby supporting a link between genetic variation in the FUT2 gene, levels of mucus glycans and the abundance of this taxa.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "RUMINOCOCCUS TORQUES"}, {"entity_type": "GENOMIC VARIANT", "description": "Individuals homozygous for the stop-codon (rs601338*A/A, non-secretors) do not express ABO antigens on the intestinal mucosa.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "FUT2"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "FUT2 encodes the enzyme alpha-1,2-fucosyltransferase, which is responsible for the secretion of fucosylated mucus glycans in the gastrointestinal mucosa.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "R. TORQUES GROUP AND A DECREASED PRESENCE OF THE R. GNAVUS GROUP"}, {"entity_type": "MOLECULAR INTERACTION", "description": "We observed that the tagging NC_000019.9:g.49218060C>T (rs35866622 non-secretor) allele was associated with a reduced abundance of the R. torques group and a decreased presence of the R. gnavus group.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "RS35866622"}, {"entity_type": "DIETARY PATTERN", "description": "One of them includes three SNPs in 9q21 (top-SNP rs602075, P=3.57\u00d710\u22128) associated with abundance of Allisonella.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "ALLISONELLA"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The 9q21 locus includes the genes PCSK5, RFK and GCNT1, of which RFK encodes the enzyme that catalyzes the phosphorylation of riboflavin (vitamin B2) and GCNT1 encodes a glycosyltransferase involved in biosynthesis of mucin.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "PCSK5"}, {"entity_type": "DIETARY PATTERN", "description": "Cobalamin is required for host\u2013microbial interactions, and supplementation with cobalamin induced a substantial shift in the microbiota composition of an in vitro colon model. These associations suggest that some members of the gut microbiome community might be affected by genetic variants that regulate the absorption and metabolism of vitamins B2 and B12.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "COBALAMIN (VITAMIN B12)"}, {"entity_type": "DIETARY PATTERN", "description": "The strongest evidence for association was seen for a block of 10 SNPs (rs7574352, P=1.42\u00d710\u22129) associated with the family Peptococcaceae, a taxon negatively associated with stool levels of the gut inflammation markers chromogranin A", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "PEPTOCOCCACEAE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Differences in dietary patterns and gut microbiota can lead to variations in the abundance of this family of bacteria.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "ENTEROBACTERIACEAE"}, {"entity_type": "NUTRIENT", "description": "Dietary factors like fiber intake can modulate the effect of genetic variants on gut microbiota composition.<SEP>Insufficient dietary fiber intake may contribute to dysbiosis of gut microbiota, impacting tryptophan metabolism and AHR activation.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46<SEP>chunk-c791a4e35844d642ba0c69774072f7cd", "id": "DIETARY FIBER"}, {"entity_type": "GENOMIC VARIANT", "description": "A block of 10 SNPs (rs7574352) associated with family Peptococcaceae.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 46}]", "id": "FAMILY PEPTOCOCCACEAE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Stool levels of gut inflammation markers, chromogranin A is negatively correlated with family Peptococcaceae.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 46}]", "id": "CHROMOGRANIN A"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Stool levels of the gut inflammation markers calprotectin are negatively correlated with family Peptococcaceae.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 46}]", "id": "CALPROTECTIN"}, {"entity_type": "DISEASE", "description": "Family Peptococcaceae is negatively associated with ulcerative colitis.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 46}]", "id": "ULCERATIVE COLITIS"}, {"entity_type": "DISEASE", "description": "A chronic disease characterized by high blood glucose levels, which occurs when the body is unable to produce enough insulin or use it effectively.<SEP>The block of SNPs are associated with insulin resistance leading to type 2 diabetes.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b<SEP>chunk-938d8bf2e56584de9dd556be5eb70964", "clusters": "[{level: 0, cluster: 6}]", "id": "TYPE 2 DIABETES"}, {"entity_type": "GENOMIC VARIANT", "description": "Located in the intergenic region, associated with IRF1 involved in insulin resistance and type 2 diabetes.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 47}]", "id": "RS7574352"}, {"entity_type": "GENE", "description": "IRF1 gene expression levels are associated with T1D susceptibility and changes in its expression.<SEP>Involved in insulin resistance and susceptibility to type 2 diabetes.<SEP>The IRF1 gene encodes for an interferon regulatory factor which is crucial for antiviral defenses and was identified as affected by a specific trans-eQTL.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b<SEP>chunk-1b40d071abac0ba432a9fb0096796f45<SEP>chunk-938d8bf2e56584de9dd556be5eb70964", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 47}]", "id": "IRF1"}, {"entity_type": "DISEASE", "description": "rs182549 (LCT) predisposes individuals to lactose intolerance.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "LACTOSE INTOLERANCE"}, {"entity_type": "DISEASE", "description": "The rs182549 variant shows positive association with T2DM diagnosis.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "TYPE 2 DIABETES MELLITUS (T2DM)"}, {"entity_type": "TRAIT", "description": "rs182549 is associated with family history of T2DM (paternal, maternal and siblings).", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "FAMILY HISTORY OF T2DM"}, {"entity_type": "TRAIT", "description": "The variant rs182549 is associated with 1,5-anhydroglucitol, an indicator for glycemic variability.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "GLYCEMIC VARIABILITY"}, {"entity_type": "DIETARY PATTERN", "description": "NC_000019.9:g.49218060 (rs35866622) locus is positively associated with fish intake.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "FISH INTAKE"}, {"entity_type": "DISEASE", "description": "The secretor allele of the FUT1/FUT2 locus shows negative association with cholelithiasis.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "CHOLELITHIASIS"}, {"entity_type": "DISEASE", "description": "The secretor allele of the FUT1/FUT2 locus is negatively associated with Crohn\u2019s disease.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "CROHN\u2019S DISEASE"}, {"entity_type": "TRAIT", "description": "The secretor allele of the FUT1/FUT2 locus shows negative association with alcohol intake frequency.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "ALCOHOL INTAKE FREQUENCY"}, {"entity_type": "TRAIT", "description": "Bifidobacterium abundance is negatively correlated with high cholesterol, even though this correlation wasn\'t significant after correction for multiple testing.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "HIGH CHOLESTEROL"}, {"entity_type": "TRAIT", "description": "The secretor allele of the FUT1/FUT2 locus shows negative association with waist-to-hip ratio.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "WAIST-TO-HIP RATIO"}, {"entity_type": "METHODOLOGY", "description": "Bi-directional two-sample MR analyses were performed using the TwoSampleMR package to identify potential causal links between gut microbial taxa and phenotypes.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "MENDELIAN RANDOMIZATION ANALYSIS"}, {"entity_type": "DISEASE", "description": "Potential causal relationships were identified with autoimmune diseases as one group of phenotypes in our analysis.<SEP>Some autoimmune diseases show parallels to IBD due to similar dysregulations in immune responses, metabolism, and microbiota interactions.", "source_id": "chunk-159d4c797dae6eb825c74067af703228<SEP>chunk-c791a4e35844d642ba0c69774072f7cd", "id": "AUTOIMMUNE DISEASES"}, {"entity_type": "DISEASE", "description": "Potential causal relationships were considered for cardiovascular diseases as another group of phenotypes in the study.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "CARDIOVASCULAR DISEASES"}, {"entity_type": "DISEASE", "description": "We evaluated potential causal associations with metabolic diseases when examining phenotype correlations.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "METABOLIC DISEASES"}, {"entity_type": "DISEASE", "description": "Potential causal links were assessed for psychiatric disorders alongside other phenotype groups in this analysis.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "PSYCHIATRIC DISORDERS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The study focused on gut microbial taxa to identify potential causal relationships with various phenotypes.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "GUT MICROBIAL TAXA"}, {"entity_type": "MICROBIAL GROUP", "description": "A higher abundance of the class Actinobacteria was found to have a protective effect on ulcerative colitis.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "ACTINOBACTERIA"}, {"entity_type": "MICROBIAL GROUP", "description": "An increased abundance of the Oxalobacteraceae family was found to have a protective effect on rheumatoid arthritis.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "OXALOBACTERACEAE FAMILY"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "Differences in sequencing depth and DNA extraction methods are significant factors influencing alpha diversity metrics such as Shannon, Simpson, and inverse Simpson indices.)", "entity_type": "UNKNOWN", "id": "ALPHA DIVERSITY"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "Differences in sequencing depth and DNA extraction methods are significant factors influencing alpha diversity metrics such as Shannon, Simpson, and inverse Simpson indices.)", "entity_type": "UNKNOWN", "id": "SEQUENCING DEPTH AND DNA EXTRACTION METHODS"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "Cohort ethnicity along with the techniques used (e.g., DNA extraction methods) contributes to heterogeneity in microbial composition.)", "entity_type": "UNKNOWN", "id": "COHORT ETHNICITY AND TECHNIQUES USED"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "Differences in sequencing depth and DNA extraction methods impact richness by influencing the microbial composition\'s diversity across various samples.)", "entity_type": "UNKNOWN", "id": "RICHNESS"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "There is a significant correlation between the heterogeneity across cohorts and alpha diversity metrics, indicating how different factors contribute to microbial richness and evenness.)", "entity_type": "UNKNOWN", "id": "COHORT HETEROGENEITY AND ALPHA DIVERSITY"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "There is a significant correlation between the heterogeneity across cohorts and alpha diversity metrics, indicating how different factors contribute to microbial richness and evenness.)", "entity_type": "UNKNOWN", "id": "CORRELATION"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "An age-genotype interaction on Bifidobacterium abundance was found, which was significant in multi-ethnic cohorts but not evident in the GEM_v24 cohort.)", "entity_type": "UNKNOWN", "id": "AGE AND ETHNICITY INTERACTION"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "An age-genotype interaction on Bifidobacterium abundance was found, which was significant in multi-ethnic cohorts but not evident in the GEM_v24 cohort.)", "entity_type": "UNKNOWN", "id": "MBBTL EFFECT"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "The rs182549 genotype showed a significant SNP-age interaction on the level of Bifidobacterium abundance, with higher abundance observed in adults homozygous for the NC_000002.11:g.136616754CC variant.)", "entity_type": "UNKNOWN", "id": "P<0.05"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "The rs182549 genotype showed a significant SNP-age interaction on the level of Bifidobacterium abundance, with higher abundance observed in adults homozygous for the NC_000002.11:g.136616754CC variant.)", "entity_type": "UNKNOWN", "id": "RS182549 - BIFIDOBACTERIUM ABUNDANCE"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "The residual heterogeneity for rs182549 was low after including quadratic and cubic terms of age in the model, indicating that the majority of variation has been explained.)", "entity_type": "UNKNOWN", "id": "P=0.01"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "The residual heterogeneity for rs182549 was low after including quadratic and cubic terms of age in the model, indicating that the majority of variation has been explained.)", "entity_type": "UNKNOWN", "id": "RS182549 - MBQTL EFFECT HETEROGENEITY"}, {"source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "description": "The age-genotype interaction was significant in the GEM_v12 and GEM_ICHIP subcohorts, both comprising mostly European individuals.)", "entity_type": "UNKNOWN", "id": "AGE-GENOTYPE INTERACTION"}, {"source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "description": "rs182549 genotype showed a higher abundance of the genus Bifidobacterium in the adult group, but not in the younger group.)", "entity_type": "UNKNOWN", "id": "YOUNG VERSUS ADULT GROUP EFFECT"}, {"source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "description": "The lack of an LCT mbQTL effect was observed among individuals of Israeli ethnicity.)", "entity_type": "UNKNOWN", "id": "ISRAELI ETHNICITY"}, {"source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "description": "6)(entity", "entity_type": "UNKNOWN", "id": "PEPTOCOCCACEAE*|>GUT INFLAMMATION MARKERS"}, {"source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "description": "6)(entity", "entity_type": "UNKNOWN", "id": "SNPS ASSOCIATED WITH PEPTOCOCCACEAE WERE NEGATIVELY CORRELATED WITH LEVELS OF GUT INFLAMMATION MARKERS LIKE CHROMOGRANIN A AND CALPROTECTIN, SUGGESTING LINKS TO GASTROINTESTINAL HEALTH.)"}, {"source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "description": "The variant rs182549 predisposes to lactose intolerance.)", "entity_type": "UNKNOWN", "id": "RS182549 (LCT)"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9<SEP>chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9<SEP>chunk-938d8bf2e56584de9dd556be5eb70964", "description": "Diagnosis for individuals with high blood glucose levels and insulin resistance, commonly due to obesity or inactivity.<SEP>Type 2 Diabetes Mellitus group, indicating participants diagnosed with Type 2 diabetes.<SEP>Type 2 Diabetes Mellitus is the target disease being studied based on genetic associations and their role in disease manifestation.<SEP>Type 2 Diabetes Mellitus<SEP>rs182549 variant shows positive association with T2DM diagnosis.)", "entity_type": "DISEASE", "id": "T2DM"}, {"source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "description": "The variant NC_000019.9:g.49218060 (rs35866622) is positively associated with fish intake.)", "entity_type": "UNKNOWN", "id": "NC_000019.9:G.49218060 (RS35866622)"}, {"source_id": "chunk-159d4c797dae6eb825c74067af703228", "description": "Results suggest potential protective effects of specific gut microbial taxa on autoimmune disorders such as ulcerative colitis and rheumatoid arthritis.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 11}]", "id": "DISEASES"}, {"source_id": "chunk-159d4c797dae6eb825c74067af703228", "description": "Results suggest potential protective effects of specific gut microbial taxa on autoimmune disorders such as ulcerative colitis and rheumatoid arthritis.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 11}]", "id": "MICROBIAL TAXA"}, {"entity_type": "PROTEIN, NUTRIENT", "description": "Dairy products are related to the intake of milk, which is positively associated with one additional LP allele.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "DAIRY PRODUCTS"}, {"entity_type": "NUTRIENT", "description": "A positive association exists between an LP (T) allele and a higher intake of milk, as well as milk beverages.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "MILK INTAKE"}, {"entity_type": "DIETARY PATTERN", "description": "The intake of various dietary factors (potatoes, cereal products, fruits, poultry, non-alcoholic beverages, wine) show associations with rs4988235 genotype.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "DIETARY FACTORS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "rs4988235 is associated with a modestly lower HDL cholesterol level after multivariable adjustment.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "HDL CHOLESTEROL"}, {"entity_type": "ENVIRONMENTAL FACTOR, NUTRIENT", "description": "Dietary factors like potatoes, cereal products, fruits, poultry, non-alcoholic beverages and wine are related to rs4988235 in terms of diabetes risk.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "DIABETES RISK FACTORS"}, {"entity_type": "RELATIONSHIPS WITH DIABETES RISK FACTORS", "description": "Multivariate regression analysis suggests no association between dairy product intake and diabetes, as well as HDL cholesterol levels.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "MR ANALYSIS"}, {"entity_type": "RS4988235 AND DAIRY PRODUCTS & OTHER FOOD INTAKE", "description": "There is an association between rs4988235 genotype and dietary intake of dairy products including milk beverages.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "ENVIRONMENTAL FACTOR, NUTRIENT"}, {"entity_type": "RS4988235 AND RISK FACTORS FOR DIABETES", "description": "The rs4988235 genetic variant is associated with dietary factors potentially linked to diabetes risk, such as potatoes, cereals, fruits, poultry, non-alcoholic beverages, wine.<SEP>The rs4988235 genetic variant shows significant associations with specific nutrient intakes like carbohydrates, fats (C15:0 and C17:0), which could impact health outcomes.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "NUTRIENT, ENVIRONMENTAL FACTOR"}, {"entity_type": "RS4988235 AND HDL CHOLESTEROL LEVELS", "description": "The rs4988235 variant shows a minor relationship with HDL cholesterol levels after adjusting for other variables.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "PHYSIOLOGICAL PROCESS, ENVIRONMENTAL FACTOR"}, {"entity_type": "RS4988235 AND ASSOCIATIONS WITH DIABETES RISK FACTORS", "description": "The genetic variant rs4988235 correlates with various dietary patterns that could influence diabetes risk, as observed in the analysis.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "DIETARY PATTERN, ENVIRONMENTAL FACTOR"}, {"entity_type": "RS4988235 AND DAIRY INTAKE RELATIONSHIP", "description": "The relationship between rs4988235 and dairy product intake (milk, milk beverages) is mediated by environmental factors such as lifestyle or dietary habits.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "MOLECULAR INTERACTION, ENVIRONMENTAL FACTOR"}, {"entity_type": "RS4988235 AND DIABETES RISK THROUGH MULTIVARIABLE ADJUSTMENTS", "description": "Through multivariate regression analysis adjusting for multiple environmental factors (like genetic background), rs4988235\'s role in diabetes risk is assessed.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "DISEASE OUTCOMES, ENVIRONMENTAL FACTOR"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "The rs4988235 genetic variant does not show a statistically significant link with developing diabetes and baseline HbA1c levels after adjusting for other factors.)", "entity_type": "UNKNOWN", "id": "RISK OF DIABETES"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "An additional LP allele correlates with increased consumption of milk and milk-based beverages.)", "entity_type": "UNKNOWN", "id": "DAIRY PRODUCT INTAKE (MILK, MILK BEVERAGES)"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "An association exists between rs4988235 genotype and specific dietary patterns that could influence diabetes risk.)", "entity_type": "UNKNOWN", "id": "DIABETES RISK FACTORS (DIETARY)"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "There is a notable association between rs4988235 and certain dietary habits that may impact the risk of developing diabetes.)", "entity_type": "UNKNOWN", "id": "DIABETES RISK FACTORS (DIETARY PATTERNS)"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "Through the influence of environmental factors, rs4988235 affects interactions with dairy intake, particularly milk and milk beverages.)", "entity_type": "UNKNOWN", "id": "MOLECULAR INTERACTION (DAIRY CONSUMPTION)"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "rs4988235 is associated with particular nutrient intake levels such as total fats from specific fatty acids (C15:0, C17:0), impacting physiological processes and health outcomes.)", "entity_type": "UNKNOWN", "id": "NUTRIENT ASSOCIATIONS"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "rs4988235 plays a role in diabetes risk when considering multiple environmental factors, as indicated by multivariate analysis results.)", "entity_type": "UNKNOWN", "id": "DIABETES RISK (MULTIVARIABLE ADJUSTMENT)"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Treatment and placebo arms did not differ in social class.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "SOCIAL CLASS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Treatment and placebo arms did not differ in maternal education.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "MATERNAL EDUCATION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Treatment and placebo arms did not differ in verbal intellectual competence.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "VERBAL INTELLECTUAL COMPETENCE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Treatment and placebo arms did not differ in nonverbal intellectual competence.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "NONVERBAL INTELLECTUAL COMPETENCE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Treatment and placebo arms did not differ in perceived stress.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "PERCEIVED STRESS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Groups differed on nutritional status based on dietary caloric intake and choline intake.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "NUTRITIONAL STATUS"}, {"entity_type": "BEHAVIOR/", "description": "Both groups consumed alcohol heavily at conception, but frequency reduced during pregnancy.<SEP>Disease<|Increased alcohol consumption can lead to or exacerbate diseases, including addiction.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e<SEP>chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "ALCOHOL CONSUMPTION"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Average cigarettes/day consumption was similar among both groups.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "CIGARETTES/DAY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Use of illicit drugs other than marijuana was rare, but methamphetamine use was found in some women later in pregnancy.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "ILLICIT DRUGS OTHER THAN MARIJUANA"}, {"entity_type": "DRUG", "description": "None of the women reported using methaqualone, which is an environmental factor not related to alcohol or drug use.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "METHAQUALONE"}, {"entity_type": "DRUG", "description": "Cocaine was not used by any woman in this study.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "COCAINE"}, {"entity_type": "DRUG", "description": "No woman reported using heroin as it is a rare illicit drug usage.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "HEROIN"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "4 women (2 choline, 2 placebo) used methamphetamine later in pregnancy, which was an environmental factor not accounted for at recruitment.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "METHAMPHETAMINE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Groups did not differ on weeks gestation at the time of these assessments.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "WEEKS GESTATION AT TIME OF BOOKING, INITIAL SCREENING, OR RANDOMIZATION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Durations varied between 9 and 27 weeks among the groups.<SEP>The groups differed slightly in duration of choline supplementation.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "DURATION OF CHOLINE SUPPLEMENTATION"}, {"entity_type": "DISEASE", "description": "There were more girls in the choline group and more boys in the placebo group, which could indicate a gender bias.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "SEX OF INFANT"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Both groups consumed alcohol heavily during pregnancy, but frequency reduced to about 1-2 days/week.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "ALCOHOL CONSUMPTION DURING PREGNANCY"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Average cigarettes/day consumption was slightly higher among those in the choline group due to different factors influencing smoking habits.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "CIGARETTE CONSUMPTION DURING PREGNANCY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Use of illicit drugs other than marijuana was rare, and methamphetamine use was found later in pregnancy for some women.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "ILLICIT DRUG USE DURING PREGNANCY"}, {"entity_type": "DRUG", "description": "No one used methaqualone as it\'s not a common substance among the participants.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "METHAQUALONE USAGE"}, {"entity_type": "DRUG", "description": "No one used cocaine during this study.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "COCAINE USAGE"}, {"entity_type": "DRUG", "description": "No one used heroin during this study.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "HEROIN USAGE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Four women (2 choline, 2 placebo) used methamphetamine later in pregnancy.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "METHAMPHETAMINE USAGE DURING PREGNANCY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Groups differed slightly on weeks gestation at time of randomization.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "WEEKS GESTATION AT TIME OF RANDOMIZATION"}, {"entity_type": "LIFESTYLE FACTOR", "description": "No group differences in infant age at these assessments, indicating consistent monitoring over time.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "INFANT AGE AT THE 6.5-AND 12-MONTH ASSESSMENTS"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Groups did not differ significantly in physical activity levels during pregnancy.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "PHYSICAL ACTIVITY LEVEL DURING PREGNANCY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "There was no significant difference between the choline and placebo groups regarding prenatal stress levels.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "PRENATAL STRESS LEVELS"}, {"entity_type": "DISEASE", "description": "Anxiety symptoms did not differ significantly between the two treatment groups at any time point during pregnancy.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "PRENATAL ANXIETY SYMPTOMS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The frequency of antenatal care visits was similar in both treatment groups, indicating equal access to prenatal care.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "ANTENATAL CARE FREQUENCY"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Both groups showed comparable weight gain patterns throughout pregnancy, which could influence the outcomes.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "MATERNAL WEIGHT GAIN DURING PREGNANCY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "There was no significant difference in maternal calcium intake between the choline and placebo groups.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "PRENATAL CALCIUM INTAKE"}, {"entity_type": "PRE-PREGNANCY BEHAVIOR", "description": "A similar percentage of women in both arms had taken folic acid before pregnancy, which can influence neural tube development but does not affect study outcomes directly related to treatment.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "FOLIC ACID SUPPLEMENTATION BEFORE PREGNANCY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Both groups received the same standard of prenatal fetal monitoring throughout the study duration.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "PRENATAL FETAL MONITORING"}, {"entity_type": "LIFESTYLE FACTOR", "description": "There were no significant differences in sleep patterns between pregnant women receiving choline and those on placebo.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "PREGNANT WOMEN\'S SLEEP HABITS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The amount of prenatal vitamin supplementation with B-vitamins was similar across both groups, indicating comparable nutritional support for the mother.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "PRENATAL B VITAMINS SUPPLEMENTATION"}, {"entity_type": "DISEASE", "description": "Blood pressure levels were not significantly different between choline and placebo groups at any time during pregnancy or post-partum.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "MATERNAL BLOOD PRESSURE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Results of routine antenatal tests did not show significant differences that could affect the outcomes related to treatment groups.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "PRENATAL ANTENATAL TEST RESULTS"}, {"entity_type": "LIFESTYLE FACTOR", "description": "The choline and placebo groups showed similar initiation rates of breastfeeding following delivery.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "BREASTFEEDING PRACTICES AFTER DELIVERY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Scores on standardized mental health screening tools were not significantly different between the choline and placebo groups throughout pregnancy and post-partum period.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "MATERNAL MENTAL HEALTH SCREENING SCORES"}, {"entity_type": "DISEASE", "description": "No group differences in postpartum depression symptoms were found, indicating that both treatments might similarly affect mental health outcomes during this period.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "POSTPARTUM DEPRESSION SYMPTOMS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "There was no difference between the two groups in prenatal vitamin D levels which can influence bone density and immune function.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "PRENATAL VITAMIN D LEVELS"}, {"entity_type": "DEMOGRAPHIC FACTOR", "description": "Groups did not differ significantly by maternal age, suggesting similar age-related factors might affect both treatment arms equally.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "MATERNAL AGE AT PREGNANCY"}, {"entity_type": "DISEASE", "description": "No significant differences in the occurrence of pregnancy complications were observed between choline and placebo groups, which are influenced by various factors unrelated to the study intervention.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "PREGNANCY COMPLICATIONS OCCURRENCE"}, {"entity_type": "DISEASE", "description": "Fetal Alcohol Spectrum Disorders diagnosis", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "FASD"}, {"entity_type": "DISEASE", "description": "Possible Fetal Alcohol Spectrum (misinterpretation of data)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "PFAS"}, {"entity_type": "TRANSCRIPT", "description": "62 infants assessed in this study", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 40}]", "id": "INFANTS"}, {"entity_type": "GENOMIC VARIANT", "description": "Variant of the enzyme PEMT carrying this variant reduces endogenous choline synthesis", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "RS12325817"}, {"entity_type": "MOLECULAR INTERACTION", "description": "These outcomes were examined considering interactions within infants, choline groups, and placebo groups", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "6 OUTCOMES"}, {"entity_type": "NUTRIENT", "description": "Upper Limit for daily intake of Choline, not exceeded by any infant in this study", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "UL"}, {"entity_type": "NUTRIENT", "description": "Choline intake during pregnancy, considered in terms of adequacy", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "DIETARY CHOLINE INTAKE"}, {"entity_type": "NUTRIENT", "description": "Status during pregnancy influencing infant outcomes", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "MATERNAL DIETARY CHOLINE STATUS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Reduction of endogenous choline synthesis as a result of the variant", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "RS12325817 ALLELE REDUCTION"}, {"entity_type": "THERAPY", "description": "Group receiving dietary choline supplementation", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "CHOLINE TREATMENT GROUP"}, {"entity_type": "THERAPY EVALUATION", "description": "Statistical test to assess the impact of choline supplementation in managing Fetal Alcohol Spectrum Disorders outcomes.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "FASD MANAGEMENT TEST"}, {"source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "description": "Both groups showed heavy consumption, but frequency reduced during pregnancy, indicating potential effects on fetal development and infant growth.)(entity", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 19}]", "id": "ALCOHOL CONSUMPTION DURING PREGNANCY AND CIGARETTE CONSUMPTION DURING PREGNANCY, ILLICIT DRUG USE DURING PREGNANCY"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "8 infants met criteria for FAS and 2 for PFAS in the choline arm, compared with 5 who met criteria for FAS and 2 for PFAS in the placebo arm)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 40}]", "id": "CRITERIA"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "9.7% of infants were born to mothers whose dietary choline intake during pregnancy was adequate (450 mg/day))", "entity_type": "UNKNOWN", "id": "ADEQUATE"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "9.7% of infants were born to mothers whose dietary choline intake during pregnancy was adequate (450 mg/day))", "entity_type": "UNKNOWN", "id": "CHOLINE INTAKE"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "21.6% of the infants carried rs12325817 variant of the enzyme PEMT, reducing endogenous choline synthesis)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 40}]", "id": "MATERNAL PEMT ALLELE"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "The six outcomes differed between the choline group and placebo group based on maternal PEMT alleles)", "entity_type": "UNKNOWN", "id": "CHOLINE GROUP BY MATERNAL PEMT ALLELE"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "9.7% (3 choline-treated, 3 placebo) of the infants were born to mothers whose dietary choline intake was adequate.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 40}]", "id": "MATERNAL DIETARY STATUS"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "Significance tested for differences between treatment and placebo groups on outcomes relevant to FASD management)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 40}]", "id": "DISEASE MANAGEMENT APPROACH"}, {"entity_type": "GENOMIC VARIANT", "description": "The rs10886471 SNP is analyzed to study the short tandem repeat function in gene expression.<SEP>The single nucleotide polymorphism rs10886471 is studied for its association with blood glucose levels and prediabetes.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "RS10886471"}, {"entity_type": "POLYMERASE CHAIN REACTION", "description": "Polymerase chain reaction amplification of genomic DNA.<SEP>Used to confirm findings from microarray analysis and qRT-PCR methods, revealing two amplicons of 628 and 557 bp for patient with mutation c.1-1G>A affecting intron 1 acceptor splice site.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9<SEP>chunk-34ffb077c584b98555081f882f8f8b94", "id": "PCR"}, {"entity_type": "GENETIC PARAMETER", "description": "The distribution of allelic frequencies in different groups to determine the association with blood glucose levels.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "ALLELIC FREQUENCY"}, {"entity_type": "GROUP", "description": "Non-Genetic Factor Group used as a baseline to compare against case groups for disease association.<SEP>Normal Fasting Glucose group, representing participants with normal fasting glucose levels.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9<SEP>chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "NFG"}, {"entity_type": "GROUP", "description": "Classification for individuals with fasting glucose levels above normal but below prediabetes threshold.<SEP>Impaired Fasting Glucose group, including participants with impaired fasting glucose levels.<SEP>Impaired Fasting Glucose, a condition associated with increased risk of developing T2DM", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9<SEP>chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "IFG"}, {"entity_type": "GROUP", "description": "Diagnostic criterion indicating the body\'s decreased ability to regulate blood sugar during oral glucose tolerance tests.<SEP>Impaired Glucose Tolerance group, for individuals showing impaired glucose tolerance.<SEP>Impaired Glucose Tolerance, another condition associated with risk for T2DM development.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9<SEP>chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "IGT"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The levels of blood glucose in different groups as a measure to determine the presence of prediabetes or T2DM.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "BLOOD GLUCOSE"}, {"entity_type": "RESEARCH METHODOLOGY", "description": "Study conducted on the intronic (CA)n repeat polymorphisms for their association with prediabetes and Type 2 diabetes.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "ASSOCIATION STUDY"}, {"entity_type": "STATISTICAL ANALYSIS", "description": "Analysis method used to determine the strength of association between genetic polymorphism and disease risk.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "LOGISTIC REGRESSION ANALYSIS"}, {"entity_type": "SYSTOLIC BLOOD PRESSURE", "description": "Systolic blood pressure readings are taken as a measure of cardiovascular health.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "SBP"}, {"entity_type": "DIASTOLIC BLOOD PRESSURE", "description": "Diastolic blood pressure is measured to evaluate cardiovascular health and risk factors for hypertension.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "DBP"}, {"entity_type": "POSTPRANDIAL BLOOD GLUCOSE", "description": "Two-hour postprandial blood glucose measurements assess the body\'s ability to regulate blood sugar levels after meals.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "2 H PG"}, {"entity_type": "STATISTICAL MEASURE", "description": "Odds Ratio used to quantify the strength of association between genotypes and disease outcomes.", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "OR"}, {"entity_type": "STATISTICAL SIGNIFICANCE", "description": "Indicates very strong statistical significance for associations between genotypes and disease outcomes, where P is less than 0.001.", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "P<0.001"}, {"entity_type": "GENE", "description": "A group of different genetic variants or alleles, each potentially having varying effects on disease development", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "MULTIPLE ALLELES"}, {"entity_type": "ALLELE", "description": "One specific member of the Multiple Alleles group associated with certain disease outcomes", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "(CA)15"}, {"entity_type": "ALLELE", "description": "Another specific member of the Multiple Alleles group potentially influencing T2DM risk", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "(CA)17"}, {"entity_type": "ALLELE", "description": "A third variant within the Multiple Alleles set, possibly linked to disease outcomes", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "(CA)18"}, {"entity_type": "ALLELE", "description": "Final member of the Multiple Alleles group that might affect disease risk", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "(CA)19"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9<SEP>chunk-41c7131af28e2a745ef6b88b8601860f", "description": "Genetic polymorphisms involve variations in DNA sequence that may influence protein function and disease susceptibility.<SEP>The rs10886471 SNP shows a genetic polymorphism as indicated by the PIC values exceeding 0.5.)", "entity_type": "MOLECULAR INTERACTION", "id": "GENETIC POLYMORPHISM"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "The rs10886471 SNP shows a genetic polymorphism as indicated by the PIC values exceeding 0.5.)", "entity_type": "UNKNOWN", "id": "RS10886471 SNP"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "Intronic repeats could play roles in gene expression as enhancers or repressors based on studies.)", "entity_type": "UNKNOWN", "id": "ENHANCERS OR REPRESSORS"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "Intronic repeats could play roles in gene expression as enhancers or repressors based on studies.)", "entity_type": "UNKNOWN", "id": "INTRONIC (CA)N REPEATS"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "The allelic frequencies of (CA)n repeat polymorphisms vary across different groups to assess disease risk.)", "entity_type": "UNKNOWN", "id": "ALLELIC FREQUENCIES"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "The allelic frequencies of (CA)n repeat polymorphisms vary across different groups to assess disease risk.)", "entity_type": "UNKNOWN", "id": "GENETIC VARIATION"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "NFG group serves as a baseline/reference for comparison with other diabetes-related groups.)", "entity_type": "UNKNOWN", "id": "NFG GROUP"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "NFG group serves as a baseline/reference for comparison with other diabetes-related groups.)", "entity_type": "UNKNOWN", "id": "REFERENCE POPULATION"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "Groups of individuals showing elevated blood glucose levels, potentially leading to prediabetes or Type 2 diabetes.)", "entity_type": "UNKNOWN", "id": "DIABETES PREDIAGNOSIS"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "Groups of individuals showing elevated blood glucose levels, potentially leading to prediabetes or Type 2 diabetes.)", "entity_type": "UNKNOWN", "id": "IFG AND IGT GROUPS"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "Group representing participants with diagnosed Type 2 diabetes mellitus.)", "entity_type": "UNKNOWN", "id": "DIABETES DISEASE STATE"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "Group representing participants with diagnosed Type 2 diabetes mellitus.)", "entity_type": "UNKNOWN", "id": "T2DM GROUP"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "The allele (CA)16 is associated with increased risk of prediabetes and T2DM based on the logistic regression analysis.)", "entity_type": "UNKNOWN", "id": "ALLELE (CA)16"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "The allele (CA)16 is associated with increased risk of prediabetes and T2DM based on the logistic regression analysis.)", "entity_type": "UNKNOWN", "id": "INCREASED RISK FACTOR"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "Waist circumference is an indicator of excess abdominal fat, linked to metabolic syndrome and diabetes risk.)", "entity_type": "UNKNOWN", "id": "OBESITY INDICATOR"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "Specific allele of a genetic variant that serves as a reference for comparison with other alleles.)", "entity_type": "UNKNOWN", "id": "(CA)16"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "Specific allele of a genetic variant that serves as a reference for comparison with other alleles.)", "entity_type": "UNKNOWN", "id": "MULTIPLE ALLELE"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "Different disease stages or outcomes (Impaired Fasting Glucose, Impaired Glucose Tolerance, and Type 2 Diabetes Mellitus respectively), associated with the genetic variant and its alleles.)", "entity_type": "UNKNOWN", "id": "DISEASE GROUPS"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "Different disease stages or outcomes (Impaired Fasting Glucose, Impaired Glucose Tolerance, and Type 2 Diabetes Mellitus respectively), associated with the genetic variant and its alleles.)", "entity_type": "UNKNOWN", "id": "IFG, IGT, T2DM"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "The specific genotypes (variants/alleles) are linked to an increased risk for developing Type 2 Diabetes Mellitus.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION WITH T2DM"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "The specific genotypes (variants/alleles) are linked to an increased risk for developing Type 2 Diabetes Mellitus.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT AND MULTIPLE ALLELES"}, {"entity_type": "GENE", "description": "The GPX4 gene contains one SNP associated with colorectal cancer risk.<SEP>The GPx4 gene encodes glutathione peroxidase 4, which is vital in cellular defense against oxidative stress.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a<SEP>chunk-d311ac0182b37003c75c0005b4337fa6", "id": "GPX4 GENE"}, {"entity_type": "SNP", "description": "The SNP rs8178974 in the GPX4 gene was statistically significantly associated with colorectal cancer risk.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "RS8178974"}, {"entity_type": "NUTRIENT", "description": "Serum selenium concentration was not associated with colorectal cancer risk, but there was a wide range of concentrations and mean was relatively high.<SEP>Variation in serum selenium concentrations is associated with geographic distribution and local climate, among other factors.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "SERUM SELENIUM CONCENTRATION"}, {"entity_type": "DISEASE", "description": "Colorectal cancer is linked to factors such as age, BMI, physical activity level, smoking habits, and postmenopausal hormone use.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "COLORECTAL CANCER"}, {"entity_type": "DRUG", "description": "Use of NSAIDs was similar between cases (patients) and controls (healthy individuals).", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "NSAIDS"}, {"entity_type": "NUTRIENT", "description": "Intakes of calcium were lower in patients compared to healthy individuals.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "CALCIUM INTAKE"}, {"entity_type": "NUTRIENT", "description": "Dietary fiber intakes were also lower among patients versus controls.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "DIETARY FIBER INTAKE"}, {"entity_type": "DRUG", "description": "Supplement usage for selenium was similar between cases and control groups.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "SELENIUM SUPPLEMENT USE"}, {"entity_type": "DISEASE SUBTYPE", "description": "Most cases had colorectal cancer diagnosed specifically as colon cancer (81.1% of total cases).", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "CANCER IN THE COLON"}, {"entity_type": "DISEASE SUBTYPE", "description": "The association between selenium and colorectal tumor risk was statistically significant; however, results were heterogenous.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "COLORECTAL TUMOR RISK"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Smoking habits impacted the relationship with selenium intake and colorectal cancer risk, though not significantly.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "SMOKING HABIT"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Use of postmenopausal hormones did not show a significant association with selenium levels or colorectal cancer risk.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "POSTMENOPAUSAL HORMONE USE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Selenium measured in serum/plasma showed an inverse association with colorectal tumors; toenail measurements had no association.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "SERUM SELENIUM"}, {"entity_type": "DISEASE SUBTYPE", "description": "The protective association was stronger for studies measuring selenium levels after colorectal tumor diagnosis, compared to before diagnosis.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "COLORECTAL ADENOMA"}, {"entity_type": "NUTRIENT", "description": "Intakes of selenium, calcium, and dietary fiber were lower among patients compared to controls.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "SELENIUM INTAKE (CALCIUM, DIETARY FIBER)"}, {"entity_type": "DRUG", "description": "Selenium supplements and NSAID use were similar in patients versus healthy individuals.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "SELENIUM SUPPLEMENTATION (NSAIDS)"}, {"entity_type": "STATISTICAL ISSUE", "description": "Funnel plots suggested potential publication bias with more small studies showing negative associations.)", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "PUBLICATION BIAS (SMALL STUDIES)"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "Serum selenium concentration showed no significant association with overall colorectal cancer risk but showed a positive relationship specifically with colon cancer risk.)", "entity_type": "UNKNOWN", "id": "COLON CANCER RISK"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "The SNP rs8178974 in the GPX4 gene was associated with colorectal cancer risk, specifically showing significance.)", "entity_type": "UNKNOWN", "id": "COLORECTAL CANCER RISK"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "The SNP rs8178974 in the GPX4 gene was associated with colorectal cancer risk, specifically showing significance.)", "entity_type": "UNKNOWN", "id": "GPX4 GENE SNP (RS8178974)"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "There was potential evidence for publication bias in funnel plots indicating higher number of small studies reporting inverse associations.)", "entity_type": "UNKNOWN", "id": "PUBLICATION BIAS"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "Selenium showed a significant and inverse association with colorectal tumor risk specifically among men, but not significantly so among women.)", "entity_type": "UNKNOWN", "id": "GENDER BASED ANALYSIS"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "Results differed based on when (before or after) colorectal tumors were diagnosed; measurements taken post-diagnosis showed stronger associations.)", "entity_type": "UNKNOWN", "id": "DIAGNOSIS TIMING"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "Results differed based on when (before or after) colorectal tumors were diagnosed; measurements taken post-diagnosis showed stronger associations.)", "entity_type": "UNKNOWN", "id": "SERUM SELENIUM MEASUREMENT"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "The protective association between selenium levels was stronger for colorectal adenomas than cancer.)", "entity_type": "UNKNOWN", "id": "COLORECTAL ADENOMA RISK"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "The inverse association between selenium levels was stronger for adenomas compared to colorectal cancer.)", "entity_type": "UNKNOWN", "id": "CANCER RISK (ADENOMA VS. CANCER)"}, {"entity_type": "GENE", "description": "CARD9 is a gene involved in shaping the bacterial and fungal gut microbiota, and controlling fungal microbiota expansion in colitis.<SEP>Card9 deficiency impacts gut microbiota-induced susceptibility to colitis, affecting IL-22 signaling pathway via tryptophan metabolism.<SEP>Card9 plays a role in connection between IBD and ability of gut microbiota to produce AHR ligands, affected by genetic polymorphisms.<SEP>Card9 plays a role in inflammation and is involved in mediating recovery from DSS-induced colitis.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6<SEP>chunk-4d5e7ab61a7357d5106f4a8a355f40d0<SEP>chunk-c791a4e35844d642ba0c69774072f7cd", "id": "CARD9"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "DSS induces colitis and affects the mouse colon transcriptome, impacting healing processes.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "DEXTRAN SULFATE SODIUM (DSS)"}, {"entity_type": "PATHOLOGICAL PROCESS", "description": "Histopathologic alterations are observed in Card9\u2212/\u2212 mice during colitis induction.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "HISTOPATHOLOGIC ALTERATIONS"}, {"entity_type": "CELLULAR ACTIVITY", "description": "Reduced epithelial cell proliferation is associated with impaired intestinal healing in Card9\u2212/\u2212 mice.<SEP>The process of epithelial cells dividing and multiplying, which plays a crucial role in tissue regeneration and wound healing.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6<SEP>chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "EPITHELIAL CELL PROLIFERATION"}, {"entity_type": "CELL PROLIFERATION MARKER", "description": "A decrease in Ki67 staining is indicative of reduced cell proliferation in the intestine.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "KI67"}, {"entity_type": "MOLECULAR ACTIVITY", "description": "Increased cleavage of caspase 3 is associated with cell apoptosis and tissue damage.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "CLEAVED CASPASE 3"}, {"entity_type": "GENOMIC DATA SET", "description": "Comparing WT and Card9\u2212/\u2212 mouse transcriptomes during DSS-induced colitis reveals specific alterations.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "MOUSE TRANSCRIPTOME"}, {"entity_type": "GENOMIC VARIANT", "description": "Standard genetically normal mice used as a baseline for comparison with genetic variants or strains.<SEP>Wild-type (WT) mice have normal genotypes without any specific genomic variations linked to Card9 or microbiome effects.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6<SEP>chunk-29025895ac187b8e88dec5cea9478842", "id": "WT MICE"}, {"entity_type": "GENOMIC VARIANT", "description": "Card9 knockout (Card9\u2212/\u2212) mice lack the CARD9 gene, leading to differences in microbiota composition and susceptibility to colitis compared to WT mice.<SEP>Mice genetically engineered to lack the CARD9 gene which affects microbiota and colitis susceptibility.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6<SEP>chunk-29025895ac187b8e88dec5cea9478842", "id": "CARD9\u2212/\u2212 MICE"}, {"entity_type": "BACTERIAL COMMUNITY", "description": "Microbiota refers to the entire community of microorganisms that live within or on an organism. In this context, it specifically relates to the gut microbiome of mice.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "MICROBIOTA"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Germ-free mice are those kept in sterile environments and thus do not have any indigenous microorganisms, used here as a control for the impact of gut microbiota on certain traits.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "GERM-FREE (GF) MICE"}, {"entity_type": "DISEASE", "description": "A type of inflammatory bowel disease that was induced by treatment with DSS (dinitrobenzene sulfonic acid).<SEP>DSS (dinitrobenzene sulfonic acid) induced colitis is an animal model to study inflammation in the colon. This process helps researchers understand inflammatory bowel diseases like ulcerative colitis and Crohn\'s disease.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6<SEP>chunk-29025895ac187b8e88dec5cea9478842", "id": "DSS-INDUCED COLITIS"}, {"entity_type": "MOLECULAR ANALYSIS TOOL", "description": "A gene expression profiling tool used for quantifying the expression levels of genes in samples.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "NANOSTRING TECHNOLOGY"}, {"entity_type": "GENE", "description": "IL-22 (Interleukin 22) is a cytokine that has been implicated in intestinal homeostasis and antimicrobial resistance, particularly playing roles in epithelial cell responses to inflammation or injury.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "IL22"}, {"entity_type": "GENE", "description": "FOS is a transcription factor that interacts with other proteins to regulate gene expression, often found in pathways related to cell stress and response to stimuli including cytokine action.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "TRANSCRIPTION FACTOR FOS"}, {"entity_type": "GENE", "description": "IL-22 is essential for normalizing AHR ligand production and colitis susceptibility in Card9\u2212/\u2212 mice.<SEP>IL-22 plays a role in stimulating cells via IL-23 production and contributes to gut microbiota function.<SEP>Similar in T cells from psoriasiform lesions and classical psoriasis, reduced in peripheral blood mononuclear cells (PBMCs).<SEP>Polymorphisms in the IL-22 gene were studied with respect to risk for CRC.", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f<SEP>chunk-56c37d6045382dda8cc6037b2680282a<SEP>chunk-29025895ac187b8e88dec5cea9478842<SEP>chunk-c791a4e35844d642ba0c69774072f7cd", "id": "IL-22"}, {"entity_type": "CELL TYPE", "description": "T cells and innate lymphoid cells (ILCs) that play a role in colitis recovery after treatment.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "INTRAEPITHELIAL \u0393\u0394 T CELLS"}, {"entity_type": "IMMUNE CELLS", "description": "Cells involved in stimulating IL-22-producing cells through IL-23 production.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "INTESTINAL DENDRITIC CELLS AND MACROPHAGES"}, {"entity_type": "MICE STRAIN", "description": "Mice raised without microbiota for comparison experiments.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "GERM-FREE MICE"}, {"entity_type": "THERAPY METHOD", "description": "A medical approach that involves using pharmaceuticals and traditional medicine to treat diseases.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "ALLOPATHIC APPROACH"}, {"entity_type": "METABOLIC PROCESS", "description": "A process involving the metabolism of tryptophan by gut bacteria or host cells which can affect immune responses.<SEP>Alteration of tryptophan metabolism by gut microbiota contributes to AHR activation and colitis susceptibility.<SEP>Gut microbiota alters IL-22 signaling through tryptophan metabolism, leading to defective AHR activation.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842<SEP>chunk-c791a4e35844d642ba0c69774072f7cd", "id": "TRYPTOPHAN METABOLISM"}, {"entity_type": "SUBSTANCE TYPE", "description": "Byproducts of tryptophan metabolism, including indole-3-acetic acid (IAA) and kynurenine (Kyn), known to activate the AhR pathway.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "INDOLE DERIVATIVES"}, {"entity_type": "PATHWAY", "description": "Stimulation of AHR by indole derivatives leads to local IL-22 production and impacts immune cell function.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "AHR ACTIVATION"}, {"entity_type": "EVENT", "description": "Binding of AHR ligands to the Ah receptor, leading to specific gene expression.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "AHR LIGATION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Microbiota produces AHR ligands necessary for colitis susceptibility; lack of IL-22 impairs this interaction.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "AHR LIGANDS"}, {"entity_type": "NUTRIENT", "description": "Reduction in tryptophan correlates with decreased AHR activation and increased levels of Kyn in IBD subjects.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "TRYPTOPHAN"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Gut microbiota metabolism of tryptophan produces Kyn, impacting AHR signaling; reduced production in IBD subjects.<SEP>Increased levels of kynurenine (Kyn) are associated with reduced AHR activation and altered gut microbiota activity in IBD subjects.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "KYN"}, {"entity_type": "PROTEIN", "description": "Active immune cells in the gut of IBD patients use tryptophan to produce Kyn via IDO1; this process is hindered by microbiota dysbiosis.<SEP>IDO1 activity plays a role in the production of Kyn from tryptophan, impacting immune cell function in IBD patients.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "IDO1"}, {"entity_type": "GENOMIC VARIANT", "description": "Risk allele of this SNP within CARD9 is associated with reduced AHR activation by gut microbiota in IBD patients.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "IBD ASSOCIATED SNP (RS10781499)"}, {"entity_type": "GENE", "description": "No significant association found between major IBD-associated SNPs (NOD2, ATG16L1, LRRK2) and altered AHR activation by microbiota.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "NOD2"}, {"entity_type": "GENE", "description": "No significant association found between major IBD-associated SNPs (NOD2, ATG16L1, LRRK2) and altered AHR activation by microbiota.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "AUTOPHAGY-RELATED 16-LIKE 1 (ATG16L1)"}, {"entity_type": "GENE", "description": "No significant association found between major IBD-associated SNPs (NOD2, ATG16L1, LRRK2) and altered AHR activation by microbiota.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "LEUCINE-RICH REPEAT KINASE 2 (LRRK2)"}, {"entity_type": "DISEASE", "description": "Defective AHR is associated with decreased tryptophan metabolites in subjects with IBD, potentially due to dysregulated immune cell activity and microbiota metabolism.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "AHR"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Dysbiosis within the gut microbiota contributes to defective tryptophan metabolism and affects AHR activation in IBD.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "MICROBIOTA DYSBIOSIS"}, {"entity_type": "DISEASE", "description": "Chronic inflammation associated with reduced tryptophan metabolites and altered AHR pathway is characteristic of IBD.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "INFLAMMATORY BOWEL DISEASE (IBD)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Modulation of immune cell function by gut microbiota-derived metabolites impacts AHR activation in inflammatory diseases.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "IMMUNE CELL FUNCTION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Specific ligands produced by gut microbiota are crucial for proper AHR signaling; their production is compromised in IBD.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "GUT MICROBIOTA DERIVED LIGANDS"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "Altered AHR pathway due to genetic and environmental factors contributes to the development of inflammatory bowel disease (IBD).", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "AHR PATHWAY"}, {"entity_type": "SNP", "description": "Specific SNP rs10781499 in CARD9 gene influences AHR pathway and microbiota-derived ligand production; it is associated with IBD susceptibility.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "GENETIC POLYMORPHISM (RS10781499)"}, {"entity_type": "CONDITION", "description": "Autoinflammation is a condition characterized by chronic inflammation without an infection; it may involve alterations in AHR pathway and tryptophan metabolism.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "AUTOINFLAMMATION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Proper nutrient absorption, including tryptophan, is crucial for maintaining gut health and influencing the AHR pathway and microbiota function.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "NUTRIENT ABSORPTION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Differences in gut microbiota composition between individuals affects the ability of microbiota to produce tryptophan metabolites influencing AHR signaling.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "GUT MICROBIOTA COMPOSITION"}, {"entity_type": "THERAPY", "description": "Probiotics may improve gut health by modulating microbiota activity and supporting tryptophan metabolism, positively impacting AHR activation.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "PROBIOTICS"}, {"entity_type": "NUTRIENT", "description": "Prebiotics serve as a food source for beneficial gut bacteria promoting microbiota health; they may influence tryptophan metabolism and AHR signaling.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "PREBIOTICS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Metabolites from gut microbes can stimulate or inhibit the AHR pathway, influencing inflammatory processes in various diseases including IBD.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "MICROBIAL METABOLITES"}, {"source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "description": "Cytokine Storm", "entity_type": "UNKNOWN", "id": "ALTERATION<|ALTERED BACTERIAL MICROBIOTA COMPOSITION DURING COLITIS AFFECTS INFLAMMATORY RESPONSES AND HEALING PROCESSES.)(ENTITY"}, {"source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "description": "Cytokine Storm", "entity_type": "UNKNOWN", "id": "BACTERIAL MICROBIOTA"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Card9 deficiency contributes to gut microbiota-induced susceptibility to colitis by disrupting IL-22 signaling and tryptophan metabolism.)", "entity_type": "UNKNOWN", "id": "CARD9 DEFICIENCY"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Card9 deficiency contributes to gut microbiota-induced susceptibility to colitis by disrupting IL-22 signaling and tryptophan metabolism.)", "entity_type": "UNKNOWN", "id": "GUT MICROBIOTA"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Gut microbiota produces ligands that stimulate AHR signaling; reduction is observed in subjects with IBD.)", "entity_type": "UNKNOWN", "id": "MICROBIOTA DERIVED AHR LIGANDS"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "The risk allele of CARD9 SNP influences the ability of gut microbiota to produce AHR agonists, affecting AHR signaling in IBD patients.)", "entity_type": "UNKNOWN", "id": "RISK ALLELE (RS10781499)"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Major IBD-associated SNPs do not significantly affect the ability of gut microbiota to produce AHR agonists.)", "entity_type": "UNKNOWN", "id": "IBD"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Major IBD-associated SNPs do not significantly affect the ability of gut microbiota to produce AHR agonists.)", "entity_type": "UNKNOWN", "id": "NOD2, ATG16L1, LRRK2 SNPS"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Decreased tryptophan metabolites are associated with autoinflammatory conditions, impacting the balance of inflammation and immunological tolerance.)", "entity_type": "UNKNOWN", "id": "ALTERATIONS IN TRYPTOPHAN METABOLISM"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Probiotics can improve gut health by modulating microbiota composition and supporting AHR pathway regulation.)", "entity_type": "UNKNOWN", "id": "GUT HEALTH"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Probiotics can improve gut health by modulating microbiota composition and supporting AHR pathway regulation.)", "entity_type": "UNKNOWN", "id": "PROBIOTICS TREATMENT"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Prebiotics facilitate nutrient absorption including tryptophan, which impacts the synthesis of microbiota-derived metabolites affecting AHR signaling.)", "entity_type": "UNKNOWN", "id": "PREBIOTIC SUPPLEMENTATION"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Increased production of certain microbial metabolites by gut bacteria can stimulate or inhibit AHR activation, impacting immune responses and inflammation.)", "entity_type": "UNKNOWN", "id": "MICROBIAL METABOLITE PRODUCTION"}, {"entity_type": "PROTEIN", "description": "The study focused on baseline \u03b22M concentration as a biomarker for hypertension risk.<SEP>\u03b22-microglobulin (\u03b22m) is an auxiliary protein that binds to HLA class I molecules, like HLA-B*15:01, facilitating the presentation of peptides for T-cell recognition.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0<SEP>chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "\u03922M"}, {"entity_type": "DISEASE", "description": "Baseline \u03b22M concentration was positively associated with prevalent hypertension in the study sample.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "PREVALENT HYPERTENSION"}, {"entity_type": "STATISTICAL MEASURE", "description": "The OR of 1.14 and 1.29 for baseline \u03b22M concentration were used to assess its association with hypertension.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "ODDS RATIO (OR)"}, {"entity_type": "DISEASE", "description": "A total of 886 participants developed new-onset hypertension after a mean follow-up of six years.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "NEW-ONSET HYPERTENSION"}, {"entity_type": "STATISTICAL METHOD", "description": "GEE model was used to identify the association between baseline \u03b22M and risk of new-onset hypertension.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "GEE MODEL"}, {"entity_type": "DIETARY PATTERN", "description": "The relationship between baseline \u03b22M concentration and hypertension was analyzed using a GEE model adjusted for clinical covariates.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "BASELINE \u03922M"}, {"entity_type": "DISEASE", "description": "Age-dependent increase in glucose levels with early onset diabetes observed in SENP1-deficient mice compared to control mice.<SEP>Increased systemic cytokines indicate a prevalence of diabetic phenotype at age 14 weeks or older.<SEP>NOD develops diabetes as a result of Th1 dominant IFN-\u03b3-induced insulitis and leukocyte infiltrations of the pancreatic islets.<SEP>One of the baseline covariates used in statistical models to adjust for potential confounders was diabetes.<SEP>Diabetes is another disease that could impact the correlation between alcohol consumption, physical activity, and cortisol levels.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a<SEP>chunk-c29d575e156569afaa1ca3d6418a71b9<SEP>chunk-8382a49b61215482f9fc75e3a86cb987<SEP>chunk-0c1631ae2c195cb68139ea9d75a3b433", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 50}]", "id": "DIABETES"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Another baseline covariate considered was cigarette smoking, which is an environmental factor.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "CIGARETTE SMOKING"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Levels of systolic and diastolic blood pressure were among the factors used to assess baseline hypertension risk.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "SYSTOLIC AND DIASTOLIC BLOOD PRESSURE (BP)"}, {"entity_type": "STATISTICAL MEASURE", "description": "The IDI was used as a metric in reclassification analyses for predicting new-onset hypertension.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "INTEGRATED DISCRIMINATION IMPROVEMENT (IDI)"}, {"entity_type": "STATISTICAL MEASURE", "description": "The NRI revealed the improvement of \u03b22M prediction over clinical models for new-onset hypertension.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "CONTINUOUS NET RECLASSIFICATION INDEX (NRI)"}, {"entity_type": "GENOMIC VARIANT", "description": "This genetic variant is associated with plasma \u03b22M levels and SH2B3 expression.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "RS3184504 POLYMORPHISM"}, {"entity_type": "TRANSCRIPT", "description": "The association between the rs3184504 polymorphism and SH2B3 expression was tested using Mendelian randomization.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "SH2B3 EXPRESSION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Decreased SH2B3 expression was found to be causal for higher \u03b22M levels through Mendelian randomization.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "DECREASED SH2B3 EXPRESSION"}, {"entity_type": "ENVIRONMENTAL FACTOR (IN A MOUSE MODEL)", "description": "Testing effects of SH2B3 deficiency on \u03b22M expression in mice helped validate the hypothesis.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "SH2B3 DEFICIENCY"}, {"entity_type": "METABOLIC PATHWAY", "description": "In response to angiotensin II administration, Sh2b3\u2212/\u2212 mice showed a significant induction of \u03b22M expression, suggesting normal constraint by SH2B3 for \u03b22M in hypertensive conditions.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "\u03922M EXPRESSION"}, {"entity_type": "PROTEIN", "description": "SH2B3 is a protein that plays a role in constraining \u03b22M expression in response to hypertensive stimuli.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "SH2B3"}, {"entity_type": "MODEL", "description": "The Sh2b3\u2212/\u2212 mouse model was used to test the effects of SH2B3 deficiency on \u03b22M expression and hypertension.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "SH2B3\u2212/\u2212 MOUSE MODEL"}, {"entity_type": "EXPERIMENTAL PROCEDURE", "description": "Angiotensin II infusion in Sh2b3\u2212/\u2212 mice was performed to study their hypertensive response and renal/vascular dysfunction.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "ANGIOTENSIN II INFUSION"}, {"source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "description": "The GEE model was employed to analyze the relationship between baseline \u03b22M and risk for new-onset hypertension during follow-up.)", "entity_type": "UNKNOWN", "id": "LONGITUDINAL ANALYSES"}, {"source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "description": "SH2B3 normally regulates \u03b22M expression during hypertensive stimuli, as shown by the induction in Sh2b3\u2212/\u2212 mice.)", "entity_type": "UNKNOWN", "id": "REGULATION OF \u03922M EXPRESSION"}, {"entity_type": "RACE/ETHNICITY", "description": "Hispanic mothers were younger at delivery compared to non-Hispanic White mothers.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "HISPANIC"}, {"entity_type": "RACE/ETHNICITY", "description": "Non-Hispanic White mothers had a mean age of 31.8 years, while Hispanic mothers had a mean age of 27.0 years.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "NON-HISPANIC WHITE"}, {"entity_type": "AGE RANGE", "description": "Maternal age at delivery was consistent across cases and controls within maternal race/ethnicity.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "MATERNAL AGE"}, {"entity_type": "EDUCATION LEVEL", "description": "Case mothers had fewer years of formal education compared to control mothers, and Hispanic mothers had fewer years of formal education than non-Hispanic White mothers.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "YEARS OF FORMAL EDUCATION"}, {"entity_type": "NUTRIENT", "description": "Maternal periconceptional use of folic acid supplements was higher in non-Hispanic Whites than in Hispanic mothers.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "clusters": "[{level: 0, cluster: 10}, {level: 1, cluster: 33}]", "id": "FOLIC ACID SUPPLEMENTS"}, {"entity_type": "NUTRIENT", "description": "There were differences in daily dietary intake of folate across subgroups, but the variations were subtle.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "DAILY DIETARY INTAKE OF FOLATE"}, {"entity_type": "NUTRIENT", "description": "The composite folate intake variable showed more substantial variation within and across subgroups.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "COMPOSITE FOLATE INTAKE VARIABLE"}, {"entity_type": "GENE", "description": "There was no evidence of a nonrandom association between NTD-affected pregnancies and either maternal or infant variants in DHFR.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "DHFR"}, {"entity_type": "SNP", "description": "Infants with the CT genotype had an approximate 1.5-fold increase in risk compared to those with the CC genotype.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "clusters": "[{level: 0, cluster: 5}]", "id": "INFANT MTHFD1 SNPS RS2236224 AND RS2236225"}, {"entity_type": "NUTRIENT INTAKE", "description": "The OR[int CC vs. CT] for infants with the CT genotype was calculated based on folic acid intake categories.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "FOLIC ACID INTAKE CATEGORY"}, {"entity_type": "SNP", "description": "Infants with the AG genotype appeared to have a four-fold increase in risk compared to those with the GG genotype, particularly when maternal folate intake was low.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "clusters": "[{level: 0, cluster: 5}]", "id": "INFANT AG GENOTYPE IN MTHFD1 SNP RS11627387"}, {"entity_type": "SNP", "description": "Mothers had a decreased risk of an NTD-affected pregnancy when they had low intake of folates compared to common homozygous mothers in the high folate intake group.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "clusters": "[{level: 0, cluster: 5}]", "id": "MTHFR SNPS RS1476413, RS1801131 AND RS1801133"}, {"entity_type": "SNP", "description": "A maternal gene-only effect was associated with a reduced risk of an NTD-affected pregnancy for SHMT1, particularly when compared to mothers with the CC genotype.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "clusters": "[{level: 0, cluster: 5}]", "id": "SHMT1 SNP RS669340 CG GENOTYPE"}, {"entity_type": "SNP", "description": "An interaction effect was observed for infants with low folate intake and this AG genotype, which showed an association with increased risk compared to those with AA or CC genotypes.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "clusters": "[{level: 0, cluster: 5}]", "id": "INFANTS WITH THE SHMT1 SNP RS12939757 AG GENOTYPE"}, {"entity_type": "SNP", "description": "The gene-folate interaction was associated with a reduced risk of NTDs for mothers in the low folate intake group, particularly when compared to high-folate intake groups.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "TYMS SNPS RS502396 OR RS699517"}, {"entity_type": "SNP", "description": "An interaction effect on infants was also protective against risk for this SNP when they were in a low folate intake group, relative to those in high-folate intake groups.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "clusters": "[{level: 0, cluster: 5}]", "id": "INFANTS WITH THE TYMS SNP RS2847153 AG GENOTYPE"}, {"entity_type": "OUTCOME", "description": "Non-Hispanic White infants showed lower incidences of adverse outcomes related to folic acid intake and MTHFR, DHFR SNPs compared to Hispanic infants.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "NON-HISPANIC WHITE INFANTS"}, {"entity_type": "GENE MUTATIONS", "description": "Variations in SHMT1 genes across different race/ethnicity groups influenced the risk of having an NTD-affected pregnancy for mothers with low folate intake.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "SHMT1 SNPS"}, {"entity_type": "SNP IMPACT", "description": "Infants inheriting the homozygous variant for either of these two TYMS SNPs showed protective effects against NTDs when their mothers had low folate intake.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "INFANTS WITH TYMS SNPS RS502396 OR RS699517 HOMOZYGOUS GENOTYPE"}, {"entity_type": "GENETIC FACTORS", "description": "Maternal genetic variations in DHFR SNPs were associated with an increased risk of having an NTD-affected pregnancy, particularly among Hispanic mothers.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "MOTHERS WITH DHFR SNPS VARIANTS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Non-Hispanic White mothers had a mean age of 31.8 years, while Hispanic mothers had a mean age of 27.0 years.)", "entity_type": "UNKNOWN", "id": "NON-HISPANIC WHITE MOTHERS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Case infants were more likely to be female than control infants and had different patterns in maternal periconceptional use of folic acid supplements compared to non-Hispanic White mothers.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 10}, {level: 1, cluster: 33}]", "id": "CASE INFANTS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Hispanic mothers had fewer years of formal education compared to non-Hispanic White mothers.)", "entity_type": "UNKNOWN", "id": "HISPANIC MOTHERS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Maternal age at delivery was consistent across cases and controls within maternal race/ethnicity.)", "entity_type": "UNKNOWN", "id": "AGE RANGE"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Maternal age at delivery was consistent across cases and controls within maternal race/ethnicity.)", "entity_type": "UNKNOWN", "id": "MATERNAL AGE AT DELIVERY"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "There were differences in daily dietary intake of folate across subgroups, but the variations were subtle among different race/ethnicity groups.)", "entity_type": "UNKNOWN", "id": "NUTRIENT INTAKE"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "There was no evidence of a nonrandom association between NTD-affected pregnancies and either maternal or infant variants in DHFR.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "DHFR SNPS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "A gene-folate interaction was associated with a reduced risk of NTDs for mothers in low folate intake groups, compared to high-folate intake groups.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 5}]", "id": "INFANTS WITH THE TYMS SNPS RS502396 OR RS699517 HOMOZGOUS GENOTYPE"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Hispanic infants had a higher rate of Neural Tube Defects (NTDs) compared to non-Hispanic White infants.)", "entity_type": "UNKNOWN", "id": "HISPANIC INFANTS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Hispanic infants had a higher rate of Neural Tube Defects (NTDs) compared to non-Hispanic White infants.)", "entity_type": "UNKNOWN", "id": "NTD RATE"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Infants carrying the AG genotype displayed increased risks for NTDs when maternal folate intake was low, compared to GG genotype carriers.)", "entity_type": "UNKNOWN", "id": "HISPANIC INFANTS WITH AG GENOTYPE IN MTHFD1 SNP RS12939757"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Infants carrying the AG genotype displayed increased risks for NTDs when maternal folate intake was low, compared to GG genotype carriers.)", "entity_type": "UNKNOWN", "id": "SNP OUTCOMES"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Mothers carrying the CG genotype in SHMT1 had a reduced risk of NTDs when their folate intake was low, compared to CC and GG genotypes.)", "entity_type": "UNKNOWN", "id": "GENETIC RISK FACTOR"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Mothers carrying the CG genotype in SHMT1 had a reduced risk of NTDs when their folate intake was low, compared to CC and GG genotypes.)", "entity_type": "UNKNOWN", "id": "MOTHERS CARRYING RS669340 CG GENOTYPE"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "The presence of the CT genotype for these two specific MTHFD1 SNPs increased infant risk for NTDs when maternal folate intake was low.)", "entity_type": "UNKNOWN", "id": "GENETIC RISK"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "The presence of the CT genotype for these two specific MTHFD1 SNPs increased infant risk for NTDs when maternal folate intake was low.)", "entity_type": "UNKNOWN", "id": "INFANTS WITH AG GENOTYPE IN MTHFD1 SNPS RS2236224 AND RS2236225"}, {"entity_type": "GENOMIC VARIANT", "description": "No association found between rs10399931 and T2D after adjustment for age, sex, BMI and glucose tolerance.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "id": "RS10399931"}, {"entity_type": "GENOMIC VARIANT", "description": "No association found between rs4950928 and T2D after adjusting for age, sex, BMI, and glucose tolerance.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "id": "RS4950928"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The two identified haplotype blocks of CHI3LI did not show any association with T2D.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "id": "CHI3LI HAPLOTYPE BLOCK"}, {"entity_type": "DISEASE", "description": "None of the investigated SNPs showed significant associations with Type 2 diabetes.<SEP>T2D was assessed for its association with epigenetic variation at specific CpGs.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e<SEP>chunk-44a871da300abc9be525b468f502f5cb", "id": "T2D (TYPE 2 DIABETES)"}, {"entity_type": "DISEASE", "description": "A borderline protective association was detected for rs12123883 with IGT, but this is not a significant association.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "id": "IGT (IMPAIRED GLUCOSE TOLERANCE)"}, {"source_id": "chunk-44a871da300abc9be525b468f502f5cb", "description": "A significant association was found between rs10399931 and serum C-peptide concentration after 2 hours, p=0.03.)", "entity_type": "UNKNOWN", "id": "SERUM C-PEPTIDE CONCENTRATION AFTER 2 HOURS"}, {"source_id": "chunk-44a871da300abc9be525b468f502f5cb", "description": "No significant associations between rs10399931 or rs4950928 and any T2D quantitative traits were found.)", "entity_type": "UNKNOWN", "id": "RS10399931/4950928"}, {"source_id": "chunk-44a871da300abc9be525b468f502f5cb", "description": "No significant associations between rs10399931 or rs4950928 and any T2D quantitative traits were found.)", "entity_type": "UNKNOWN", "id": "T2D QUANTITATIVE TRAITS"}, {"source_id": "chunk-44a871da300abc9be525b468f502f5cb", "description": "The two identified haplotype blocks of CHI3LI did not show any association with T2D.)", "entity_type": "UNKNOWN", "id": "CHI3LI HAPLOTYPE BLOCKS"}, {"source_id": "chunk-44a871da300abc9be525b468f502f5cb", "description": "The results showed no significant associations between rs10399931 and quantitative traits related to T2D such as BMI or C-peptide concentration.)", "entity_type": "UNKNOWN", "id": "T2D QUANTITATIVE TRAITS (BMI, C-PEPTIDE CONCENTRATION)"}, {"entity_type": "GENETIC RISK FACTOR", "description": "Joint effects of lifestyle and variants in complement pathway genes were explored to understand their combined influence on AMD risk.<SEP>The presence of one or two A69S risk alleles and poor lifestyle leads to higher odds for AMD compared to no risk alleles and high HLS.<SEP>The presence of risk CFH alleles increases the odds for AMD.<SEP>There were 337 cases of AMD in the full sample, of which 91% were early stages of AMD.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5<SEP>chunk-a7d0bfa8b16262a1f313b7e3060969c6", "clusters": "[{level: 0, cluster: 0}]", "id": "AMD"}, {"entity_type": "DIETARY PATTERN", "description": "The HLS score indicates dietary lifestyle, with low (0\u20132) and high scores (4\u20136).", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "id": "HLS"}, {"entity_type": "GENOMIC VARIANT", "description": "The Y402H CFH risk allele is associated with increased odds for AMD.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "id": "Y402H CFH RISK ALLELE"}, {"entity_type": "GENOMIC VARIANT", "description": "Having at least one ARMS2 risk allele and poor lifestyle increases the odds for AMD compared to zero risk alleles and high HLS.<SEP>The A69S risk allele also increases the odds of AMD.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "clusters": "[{level: 0, cluster: 0}]", "id": "A69S RISK ALLELE"}, {"entity_type": "GENE", "description": "The G allele modifies the CFB gene function associated with increased odds of AMD.<SEP>rs641153 affects the CFB gene and is associated with increased odds of AMD.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "clusters": "[{level: 0, cluster: 0}]", "id": "G ALLELE OF RS641153 (CFB/C2)"}, {"entity_type": "GENOMIC VARIANT", "description": "No evidence for association between rs10033900 and AMD within CAREDS dataset.<SEP>No main effect was observed for rs10033900 in CAREDS when considering overall population.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "clusters": "[{level: 0, cluster: 0}]", "id": "RS10033900 (CFI)"}, {"entity_type": "GENOMIC VARIANT", "description": "No association was found with rs2230199 and AMD within the CAREDS dataset.<SEP>No statistical significance found with rs2230199 in relation to AMD in CAREDS data.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "clusters": "[{level: 0, cluster: 0}]", "id": "RS2230199 (C3)"}, {"entity_type": "GENE", "description": "The joint effect of poor lifestyle score (HLS) and a high-risk CFH genotype (CC) in women leads to greater odds of AMD compared to low genetic risk and healthy lifestyle.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "id": "CFH GENOTYPE"}, {"entity_type": "DIETARY PATTERN", "description": "In the full sample, having two high risk CFH alleles with a low HLS score increases the odds for AMD, compared to low genetic risk and healthy lifestyle.<SEP>In the full sample, having two high risk CFH alleles with a low HLS score increases the odds for AMD, compared to low risk genotype with high HLS.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "id": "HEALTHY LIFESTYLE SCORE (HLS)"}, {"entity_type": "NUTRIENT", "description": "The intake of omega-3 fatty acids does not influence the association between HLS and AMD in women with high risk CFH genotype.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "id": "\u03a9-3 FATTY ACIDS"}, {"entity_type": "GENE", "description": "In women with ARMS2 risk allele, having poor lifestyle has an increased but not clearly discernible joint effect when cross-classified due to low sample size.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "id": "ARMS2 GENOTYPE"}, {"entity_type": "DIETARY PATTERN", "description": "There was no evidence for a multiplicative interaction between lifestyle and AMD risk.", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 9}]", "id": "LIFESTYLE"}, {"entity_type": "GENE", "description": "Joint effects of lifestyle score and variants in additional complement pathway genes like CFB/C2 were explored.", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 9}]", "id": "CFB/C2"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP in the complement pathway genes like C3 were explored along with HLS, but their interaction was unreliable.", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 9}]", "id": "C3 RS2230199"}, {"entity_type": "GENOMIC VARIANT", "description": "Similar to C3, the SNP in complement pathway gene like CFI was also explored along with HLS for its interaction effects on AMD risk.", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 9}]", "id": "CFI RS10033900"}, {"source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "description": "Women carrying two copies of Y402H CFH risk allele had approximately 2.4 times greater odds for developing AMD compared to those with no such alleles.)<SEP>Women with two copies of the Y402H CFH allele had 2.4 times greater odds of AMD relative to women without any copies.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "Y402H CFH ALLELE"}, {"source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "description": "In women with high genetic risk for AMD, a lower HLS (indicating unhealthy dietary habits) was significantly associated with an increased likelihood of AMD compared to those with both low genetic risk and good dietary habits.)<SEP>In women with high genetic risk, HLS scores were significantly associated with an increased likelihood of AMD.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "HLS SCORE"}, {"source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "description": "A joint effect is observed between having both two high-risk CFH alleles and a low HLS score compared to low risk genotypes and high HLS, associated with higher odds for AMD.)<SEP>The combination of two high-risk CFH alleles and a poor HLS score was significantly associated with higher odds for AMD than individuals with low risk genotypes and healthy lifestyle scores.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "CFH GENOTYPE (CC)"}, {"source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "description": "For women carrying ARMS2 risk allele, poor lifestyle was associated with higher odds of AMD compared to those without the ARMS2 risk allele and maintaining a healthy diet.)<SEP>In the full sample, a significant interaction between dietary lifestyle scores and CFH genotype was observed in predicting higher odds for AMD compared to other genetic risk groups.)<SEP>In the full sample, there\'s an interaction between lifestyle factors and CFH genotype leading to increased odds of AMD compared to other genetic risk groups.)<SEP>In women carrying the ARMS2 risk allele, having poor lifestyle with HLS leads to higher odds for AMD compared to no risk alleles and high HLS.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "LIFESTYLE FACTORS (DIETARY LIFESTYLE SCORES)"}, {"source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "description": "The joint impact of poor lifestyle score and the high-risk CFH genotype was found to be more than additive for AMD risks, suggesting synergy in this relationship.)<SEP>The joint impact of poor lifestyle score on diet and high-risk CFH genotype was found to amplify the odds of AMD compared to having low-risk genotypes and a healthy lifestyle.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "INTERACTION EFFECT (CFH GENOTYPES AND HLS)"}, {"source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "description": "No influence on the association between HLS score and AMD risk, when adjusted for omega-3 fatty acid consumption.)<SEP>No significant influence on the association between HLS and AMD when adjusted for omega-3 fatty acid intake.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "\u03a9-3 FATTY ACIDS INTAKE"}, {"source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "description": "The combined effect of one or more ARMS2 risk alleles with poor lifestyle had an additive yet not significant 0.9 times increase in the odds for AMD compared to individuals without ARMS2 risk alleles and having a healthy lifestyle.)<SEP>The synergy of ARMS2 genetic risk with poor lifestyle was statistically significant but did not reach the level of additive effects, implying more than additive impact on AMD risks.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "SYNERGY INDEX (ARMS2 AND LIFESTYLE)"}, {"source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "description": "Interaction analysis revealed that while there were joint effects between lifestyle and genotypes in complement pathway genes, the estimates for synergy were unreliable.)<SEP>The combined effect of lifestyle and these genotypes was not reliable for estimating synergy due to uncertain odds distribution.)<SEP>The exploration focused on the joint effect of these genotypes along with lifestyle score in influencing AMD risk.)<SEP>The interaction analysis of these genotypes with lifestyle did not yield clear evidence for deviation from additive or multiplicative effects.)<SEP>The reliability of synergy estimates for these genotypes and HLS was low due to their combination leading to variable odds for AMD.)<SEP>There was no evidence to suggest that lifestyle significantly influences the effect of these genotypes on AMD risk.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 9}]", "id": "CFB/C2, C3 RS2230199, CFI RS10033900"}, {"source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "description": "Lifestyle factors could influence the expression levels of relevant transcripts, affecting AMD risk through this mechanism.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 9}]", "id": "TRANSCRIPTS"}, {"source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "description": "Lifestyle may impact the function or concentration of proteins associated with these genes, potentially affecting AMD risk.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 9}]", "id": "PROTEINS"}, {"source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "description": "Lifestyle factors could alter the dynamics of molecular interactions relevant to AMD risk.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 9}]", "id": "MOLECULAR INTERACTIONS"}, {"source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "description": "Lifestyle may modify or optimize these processes, which might influence AMD risk in a positive or negative way.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 9}]", "id": "PHYSIOLOGICAL PROCESSES"}, {"source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "description": "Lifestyle interventions might influence these pathways, potentially affecting AMD risk through such modifications.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}, {level: 1, cluster: 9}]", "id": "METABOLIC PATHWAYS"}, {"entity_type": "CHROMOSOME/GENOMIC REGION", "description": "A location on chromosome 1 where several SNPs associated with coffee consumption were identified.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "1Q24"}, {"entity_type": "HABITUAL BEHAVIOR/COFFEE DRINKING HABIT", "description": "The regular intake or consumption of coffee.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "COFFEE CONSUMPTION"}, {"entity_type": "SPECIFIC TYPE OF COFFEE/UNSWEETENED COFFEE", "description": "A type of coffee that is not sweetened, typically consumed as a beverage.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "BLACK COFFEE"}, {"entity_type": "SPECIFIC TYPE OF COFFEE/SWEETENED COFFEE", "description": "Coffee that has been sweetened with sugar or another sweetener.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "SUGARED COFFEE"}, {"entity_type": "CONDITION/STAGE BEFORE TYPE 2 DIABETES", "description": "Pre-diabetic condition where blood sugar levels are higher than normal but not yet high enough for a diagnosis of diabetes.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "PREDIABETES"}, {"entity_type": "VARIATION IN DNA SEQUENCE/GENETIC VARIATION", "description": "Alterations or changes to the sequence of DNA that occur at specific points in the genome.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "clusters": "[{level: 0, cluster: 4}]", "id": "GENETIC POLYMORPHISMS"}, {"entity_type": "GENE", "description": "A gene located on chromosome 1q24, one of the genes associated with coffee consumption.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "HECTD4 GENE"}, {"entity_type": "GENE", "description": "A gene involved in acyl-CoA dehydrogenase activity and was identified as related to coffee consumption.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "ACAD10 GENE"}, {"entity_type": "GENE", "description": "A muscle-specific myosin light chain 2 gene, one of the genes associated with coffee consumption.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "MYL2 GENE"}, {"entity_type": "GENE", "description": "A gene encoding a member of the CXC motif chemokine family and was identified as related to coffee consumption.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "CUX2 GENE"}, {"entity_type": "SELECTED GENETIC VARIANTS/SNPS USED FOR GENETIC RISK SCORES", "description": "Specific SNPs that were selected due to their association with habitual coffee consumption for further analysis.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "COFFEE-RELATED SNPS"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Genetic polymorphisms associated with coffee consumption were identified through a GWAS.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 8}]", "id": "IDENTIFIED SNPS"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Consumption of black or sugared coffee was assessed in relation to prediabetes and type 2 diabetes risk.)", "entity_type": "UNKNOWN", "id": "BLACK COFFEE/SUGARED COFFEE"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Five SNPs were selected based on their association with coffee consumption for use in calculating genetic risk scores.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 4}]", "id": "RISK SCORE CALCULATION"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Associations between habitual coffee consumption and prediabetes/type 2 diabetes varied based on the subgroup analysis of coffee types (black vs. sugared).)", "entity_type": "UNKNOWN", "id": "ASSOCIATIONS"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Associations between habitual coffee consumption and prediabetes/type 2 diabetes varied based on the subgroup analysis of coffee types (black vs. sugared).)", "entity_type": "UNKNOWN", "id": "GWAS/COFFEESUBGROUPS/"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "The relationship between coffee consumption and the risk of prediabetes or type 2 diabetes was modified by genetic polymorphisms.)", "entity_type": "UNKNOWN", "id": "GENETIC POLYMORPHISMS/COFFEE CONSUMPTION/>"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "The relationship between coffee consumption and the risk of prediabetes or type 2 diabetes was modified by genetic polymorphisms.)", "entity_type": "UNKNOWN", "id": "INFLUENCE"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "A measure of body fat based on height and weight, used as an indicator in this context.)", "entity_type": "UNKNOWN", "id": "BODY MEASUREMENT/BODY MASS INDEX"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Stage before type 2 diabetes where blood sugar levels are higher than normal but not yet high enough for a diagnosis of diabetes.)", "entity_type": "UNKNOWN", "id": "CONDITION/PRE-DIABETIC CONDITION"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Chronic disease characterized by high blood glucose levels, caused when the body fails to produce sufficient insulin or cannot use it effectively.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 6}]", "id": "DISEASE/"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "A tool used to calculate an individual\'s likelihood of developing prediabetes or type 2 diabetes based on their genetic makeup.)", "entity_type": "UNKNOWN", "id": "ASSESSMENT/EVALUATION OF GENETIC PREDISPOSITION"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "A tool used to calculate an individual\'s likelihood of developing prediabetes or type 2 diabetes based on their genetic makeup.)", "entity_type": "UNKNOWN", "id": "GENETIC RISK SCORES/"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "A variant of coffee that is not sweetened, typically consumed as a beverage.)", "entity_type": "UNKNOWN", "id": "SPECIFIC TYPE OF COFFEE/UNSWEETENED COFFEE"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Coffee that has been sweetened with sugar or another sweetener.)", "entity_type": "UNKNOWN", "id": "SPECIFIC TYPE OF COFFEE/SWEETENED COFFEE"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Coffee that has been sweetened with sugar or another sweetener.)", "entity_type": "UNKNOWN", "id": "SUGARED COFFEE/"}, {"entity_type": "GENE", "description": "An autoantigen NRP-V7-positive CD8+ T cells were detected in the pancreatic lymph nodes and spleen of SENP1-aP2KO mice.<SEP>Regulated by conditional media of SENP1-aP2KO pancreatic adipocytes and involved in NF-\u03baB dependent inflammation.<SEP>SENP1 deletion augments NEMO SUMOylation in adipocytes.<SEP>SENP1 is a gene that plays a role in diabetes development.<SEP>Senp1 is a gene that plays an important role in diabetes development.<SEP>The gene Senp1 was deficient in mice, leading to a T1DM phenotype.<SEP>We first investigated whether there is a correlation between SENP1 expression in adipocytes of non-obese diabetic (NOD) mice and human T1DM patients.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c<SEP>chunk-174831b97ed170039b374ba254d4f506<SEP>chunk-fc33d6399ec56e65d007269407f1953d<SEP>chunk-8382a49b61215482f9fc75e3a86cb987<SEP>chunk-eaf655c4dd9244523b048a122e232be4<SEP>chunk-0c1631ae2c195cb68139ea9d75a3b433<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 54}]", "id": "SENP1"}, {"entity_type": "GENE PROMOTERS", "description": "Three different gene promoters were used to drive Cre recombinase in adipocyte-specific deletion of Senp1: Adiponectin, PdgfR\u03b1, or aP2/Fabp4.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "ADIPONECTIN, PDGFR\u0391, AND AP2/FABP4"}, {"entity_type": "EXPRESSION", "description": "The expression of Senp1 mRNA and protein was diminished in isolated adipocytes from specific tissues like brown, white, and peri-pancreatic adipose tissue of SENP1-deficient mice.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "MRNA AND PROTEIN"}, {"entity_type": "ADIPOCYTE", "description": "The adipocytes isolated from interscapular brown, perigonadal white, and peri-pancreatic adipose tissue were used to study Senp1 expression in specific mice strains.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "ISOLATED ADIPOCYTES"}, {"entity_type": "CELL TYPES", "description": "SENP1 expression was intact in isolated macrophages but not in pancreatic islets from SENP1-deficient mice.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "MACROPHAGES AND PANCREATIC ISLETS"}, {"entity_type": "ADIPOSE TISSUE", "description": "Enhanced green fluorescent protein was detected in peri-pancreatic adipose tissue (PAT) but not in pancreatic islets when aP2-Cre deleter mice were mated with mT/mG reporter mice.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "PAT"}, {"entity_type": "HORMONE", "description": "Insulin levels in plasma were measured at various ages and showed significantly declined insulin secretion after ages 8 and 10 weeks, respectively, in SENP1-deficient mice compared to control mice.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "INSULIN LEVELS"}, {"entity_type": "TESTS", "description": "Glucose tolerance test (GTT) and insulin tolerance test (ITT) were performed to evaluate diabetes and insulin resistance in aged SENP1-deficient mice.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "GTT ANDITT"}, {"entity_type": "CELL TYPES", "description": "Specific deletion of Senp1 gene was conducted in smooth muscle cells/pericytes by SM22\u03b1-Cre for SENP1-smKO and vascular endothelial cells by VE-cadherin-Cre for SENP1-ecKO.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "SMOOTH MUSCLE CELLS/PERICYTES"}, {"entity_type": "LIPID PROFILES", "description": "Cholesterol and triglyceride levels were slightly reduced after age 18 weeks, while HDL and FFA showed normal profiles in SENP1-aP2KO mice compared to control.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "CHOLESTEROL AND TG"}, {"entity_type": "BEHAVIORAL PARAMETERS", "description": "SENP1-aP2KO mice had increased water consumption but similar dairy food uptake compared to control mice.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "WATER CONSUMPTION AND FOOD INTAKE"}, {"entity_type": "EXPERIMENTAL MODEL", "description": "SENP1-deficient mice were treated with streptozotocin to induce diabetes, which allowed researchers to study the impact of SENP1 deficiency on insulin-dependent conditions.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "STREPTOZOTOCIN (STZ)"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "The study suggested that SENP1 deficiency might contribute to cellular senescence through its effect on insulin and glucose metabolism.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "CELLULAR SENESCENCE"}, {"entity_type": "CELLULAR PROCESS", "description": "Research indicated that SENP1 plays a role in adipocyte differentiation, which is critical for energy storage and metabolism in fat cells.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "ADIPOCYTE DIFFERENTIATION"}, {"entity_type": "CELL TYPE", "description": "Experiments also explored how SENP1 affects hematopoietic stem cells, suggesting a broader impact beyond adipocytes and pancreatic islets.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "HEMATOPOIETIC STEM CELLS"}, {"entity_type": "ASSOCIATION WITH GHI", "description": "GHI is linked to metabolic syndrome, which includes obesity, insulin resistance, high blood pressure, and abnormal cholesterol levels.<SEP>The research demonstrated that Senp1 deficiency leads to metabolic syndrome traits including obesity, insulin resistance, and dyslipidemia in mice.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5<SEP>chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "METABOLIC SYNDROME"}, {"entity_type": "GENOMIC VARIANT", "description": "SENP1-deficient mice exhibit diabetic phenotypes, suggesting the variant influences diabetes.", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "id": "ADIPOCYTE-SPECIFIC SENP1-DEFICIENT MICE"}, {"entity_type": "DISEASE", "description": "Diabetes is characterized by T-cell-mediated destruction of pancreatic \u03b2 cells.", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "id": "INSULIN-DEPENDENT DIABETES MELLITUS"}, {"entity_type": "TRANSCRIPT", "description": "The transcript related to \u03b2 cell development shows changes in SENP1-deficient mice.", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "id": "\u0392 CELL"}, {"entity_type": "PROTEIN", "description": "Pancreatic \u03b2 cells are involved in insulin production and function.", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "id": "PANCREATIC \u0392 CELLS"}, {"entity_type": "METABOLIC PATHWAY", "description": "Ketone production is a response to the body\'s inability to absorb glucose efficiently.", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "id": "\u0392-HYDROXYBUTYRATE"}, {"entity_type": "CELL TYPE", "description": "CD8+ T cells also showed specific responses when compared with CD4+ T cells, indicating cell-type-specific effects.<SEP>These T cells target a peptide from islet-specific glucose-6-phosphatase catalytic subunit-related protein.<SEP>We also purified CD8+ T cells from the same PBMCs.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433<SEP>chunk-bec60ce5d500ee78c6792a2d92b048ca<SEP>chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "CD8+ T CELLS"}, {"entity_type": "RESPONSE", "description": "Autoimmune responses occur in the pancreases of SENP1-deficient mice, consistent with autoimmune diabetes.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "AUTOIMMUNE RESPONSE"}, {"entity_type": "DISEASE", "description": "T1DM (Type 1 Diabetes Mellitus) refers to the autoimmune condition in which insulin-producing cells are damaged and eventually destroyed, leading to inadequate production of insulin by the body.<SEP>T1DM is an autoimmune disease with immunological destruction of pancreatic \u03b2 cells resulting in insulin deficiency.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "T1DM"}, {"entity_type": "CHEMOKINE", "description": "PATs secret proinflammatory cytokines that overproduction may serve as a main reason for the break of immune tolerance and expansion of immune cells.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "CYTOKINE"}, {"entity_type": "CYTOKINE", "description": "Similar to IL-6, TNF-\u03b1 also shows increased expression in PATs of the SENP1-deficient mice.<SEP>TNF-\u03b1 levels were also significantly elevated in the same period.<SEP>Tumor Necrosis Factor Alpha (TNF-\u03b1), another proinflammatory cytokine, was also elevated in PAT tissue compared to other adipose depots.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433<SEP>chunk-174831b97ed170039b374ba254d4f506", "id": "TNF-\u0391"}, {"entity_type": "CYTOKINE", "description": "IL-I\u03b2 expression was significantly higher in PATs from SENP1-aP2KO mice compared with control mice.<SEP>IL-I\u03b2 was found to be significantly elevated in SENP1-deficient mice.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "IL-I\u0392"}, {"entity_type": "ANATOMICAL STRUCTURE", "description": "No abnormalities were found in cardiac function and vascular reactivity.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "CARDIAC FUNCTION"}, {"entity_type": "INFLAMMATORY CONDITION", "description": "Mice exhibit increased myocarditis which is indicative of the development of diabetes.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "MYOCARDITIS"}, {"entity_type": "ADIPOSE TISSUE", "description": "Paternal Adipose Tissue (PATs) include epididymal fat and are the focus of this study.<SEP>Peri-pancreatic adipocytes (PATs) express high levels of proinflammatory cytokines compared to other adipose depots.<SEP>Peri-pancreatic adipocytes show enhanced inflammation and a pro-inflammatory phenotype compared to control adipocytes.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433<SEP>chunk-eaf655c4dd9244523b048a122e232be4<SEP>chunk-174831b97ed170039b374ba254d4f506", "id": "PATS"}, {"entity_type": "PHENOTYPIC CHANGE", "description": "These changes suggest the development of a diabetic phenotype before onset of diabetes at age 7 weeks.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "DIABETIC PHENOTYPE"}, {"entity_type": "CELL", "description": "No significant differences were detected in macrophage numbers between control and SENP1-deficient mice at age 7 weeks.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "MACROPHAGE"}, {"entity_type": "CELL", "description": "Similar to macrophages, there was no significant difference in T cell numbers between the two groups at this time point.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "T CELL"}, {"entity_type": "MEASURE", "description": "Circulating cytokine levels were too low to induce systemic autoimmune responses.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "CYTOKINE CONCENTRATION"}, {"entity_type": "CYTOKINE GROUP", "description": "Increased systemic cytokines and cardiac myocarditis indicate the presence of a diabetic phenotype in older mice.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "SYSTEMIC CYTOKINES"}, {"entity_type": "PHENOTYPIC CHANGE", "description": "As the animals age, an increase in symptoms related to diabetes becomes evident.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "clusters": "[{level: 0, cluster: 9}]", "id": "DIABETES PHENOTYPE"}, {"entity_type": "ANATOMICAL SITE", "description": "There was no significant difference in cytokine expression between different adipose depots except PATs.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "ADIPOSE DEPOTS"}, {"entity_type": "CYTOKINE GROUP", "description": "Increased levels of pro-inflammatory cytokines like IFN-\u03b3 play a role in exacerbating diabetes conditions by causing inflammation and tissue damage.<SEP>Secreted from PATs, induce the upregulation of CCL5 expression leading to immune cell infiltration.<SEP>These cytokines were predominantly expressed at higher levels in PATs compared to other adipose tissues.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433<SEP>chunk-eaf655c4dd9244523b048a122e232be4<SEP>chunk-8382a49b61215482f9fc75e3a86cb987", "id": "PROINFLAMMATORY CYTOKINES"}, {"entity_type": "EVENT", "description": "The onset of diabetes is confirmed by the significant increase in circulating cytokines and cardiac myocarditis.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "clusters": "[{level: 0, cluster: 9}]", "id": "DIABETES ONSET"}, {"entity_type": "ACTIVITY", "description": "Peri-pancreatic adipocytes are suggested to be primarily responsible for the high concentrations of cytokines before diabetes onset.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "CYTOKINE PRODUCTION"}, {"entity_type": "MOLECULE", "description": "CCL5 expression is upregulated in pancreatic islets, suggesting an inflammatory response linked to type 1 diabetic development.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "CHEMOKINE"}, {"entity_type": "ANATOMICAL SITE", "description": "Islets show specific expression of CCL5 which may indicate recruitment of immune cells involved in diabetes progression.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "PANCREATIC ISLET"}, {"entity_type": "PATHOLOGICAL TARGET", "description": "CCL5 can potentially direct the recruitment of immune cells to the pancreas, contributing to autoimmune damage.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "AUTOIMMUNE TARGETS"}, {"entity_type": "CELL CULTURE SAMPLE", "description": "The media from pancreatic adipocytes of SENP1-aP2KO mice increases CCL5 expression in isolated islets.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "ISOLATED ISLETS"}, {"entity_type": "PATHOLOGICAL PROCESS", "description": "Degenerative changes occur in pancreatic \u03b2 cells of SENP1-deficient mice, leading to insulin deficiency.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "\u0392 CELL DEGENERATION"}, {"entity_type": "DISEASE", "description": "Mice develop a type 1 diabetes-like phenotype with autoimmune destruction of \u03b2 cells resulting in insulin scarcity.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "TYPE 1 DIABETES (T1D)"}, {"entity_type": "RESPONSE MECHANISM", "description": "Innate immune response is activated leading to inflammation and the recruitment of immune cells into pancreatic tissue.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "INNATE IMMUNITY"}, {"entity_type": "CELLULAR PROCESS", "description": "Macrophages become activated in pancreas of SENP1-deficient mice contributing to inflammatory responses.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "MACROPHAGE ACTIVATION"}, {"entity_type": "IMMUNE PATHWAY", "description": "A Th1 cytokine-driven immune response is involved, which typically targets and destroys \u03b2 cells in autoimmune diabetes.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "TH1 RESPONSE"}, {"entity_type": "RESPONSE TYPE", "description": "The mice exhibit signs of type 2 immunity response characterized by the recruitment of neutrophils and eosinophils.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "TYPE 2 IMMUNITY"}, {"entity_type": "CELL TYPE", "description": "Increased numbers of circulating and infiltrating neutrophils are found in SENP1-deficient mice before onset of diabetes.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "NEUTROPHIL"}, {"entity_type": "CELL TYPE", "description": "Elevated levels of eosinophils are observed in the peripheral blood of these mice, contributing to the inflammatory environment.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "EOSINOPHIL"}, {"entity_type": "IMMUNE RESPONSE", "description": "An antibody response is detected against pancreatic antigens specific for \u03b2 cells and other islet components.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "ANTIBODY RESPONSE"}, {"entity_type": "PATHOLOGICAL CONDITION", "description": "Inflammation in the pancreas is observed before clinical onset of diabetes, characterized by infiltration with immune cells.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "PANCREATIC INFLAMMATION"}, {"entity_type": "DISEASE FORM", "description": "Development of diabetic cardiomyopathy is evident as a result of prolonged hyperglycemia and metabolic stress in mice.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "DIABETIC CARDIOMYOPATHY"}, {"entity_type": "ANATOMICAL CHANGE", "description": "Increased cardiac fibrosis occurs due to chronic inflammation and altered metabolism.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "CARDIAC FIBROSIS"}, {"entity_type": "CELLULAR RESPONSE", "description": "Fibroblasts become activated and proliferate leading to cardiac fibrosis in the diabetic mice model.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "FIBROBLAST ACTIVATION"}, {"entity_type": "CELL TYPE", "description": "High local concentrations of cytokines secreted from these cells upregulate CCL5 expression, leading to immune cell infiltration and \u03b2 cell damage.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "AP2KO PANCREATIC ADIPOCYTE"}, {"entity_type": "MOLECULE", "description": "Upregulated in response to high local proinflammatory cytokine levels originating from adipocytes, involved in immune cell recruitment.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "CCL5"}, {"entity_type": "CELL TYPE", "description": "Infiltrate pancreas due to high local pro-inflammatory cytokine levels secreted by adipocytes.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "IMMUNE CELLS"}, {"entity_type": "EFFECT", "description": "Results from the direct cytotoxic effects of cytokines originating from peri-pancreatic adipocytes on pancreatic islets.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "CYTOTOXIC DAMAGE"}, {"entity_type": "PROTEIN", "description": "A critical activator of genes for inflammation and immune regulation, involved in pro-inflammatory responses in adipose tissue.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "NF-KB"}, {"entity_type": "PROTEIN", "description": "Part of the NF-kB signaling pathway that is upregulated upon SENP1 deletion, leading to increased NF-kB activity.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "IKK"}, {"entity_type": "PROTEIN", "description": "Decreased total levels and phosphorylation observed in adipose tissues of SENP1-aP2KO mice, contributing to enhanced NF-kB activation.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "I\u039aB\u0391"}, {"entity_type": "SUBUNIT", "description": "NF-kB subunit that is strongly upregulated upon SENP1 deletion, leading to increased pro-inflammatory gene expression and cytokine production.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "P65/RELA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A mechanism that contributes to the regulation of PPAR\u03b3 and its function in modulating inflammatory responses, not significantly affected by SENP1 deletion.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "SUMOYLATION"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "The process by which adipocytes differentiate under high-fat diet conditions can contribute to obesity and insulin resistance, potentially affecting T1DM progression.<SEP>The process of differentiation into mature adipocytes is reduced in some adipose depots of mice with SENP1 deletion, indicating an impact on cell development and maturation process.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4<SEP>chunk-8382a49b61215482f9fc75e3a86cb987", "id": "ADIPOGENESIS"}, {"entity_type": "PHYSICAL ATTRIBUTE", "description": "Cell sizes in PATs were larger under normal chow conditions, but smaller by 1.8-fold under HFD in SENP1-aP2KO mice.<SEP>Decreased in peri-pancreatic adipocytes from SENP1-aP2KO mice compared to control adipocytes, contributing to the overall phenotype.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4<SEP>chunk-174831b97ed170039b374ba254d4f506", "id": "CELL SIZE"}, {"entity_type": "QUANTITATIVE MEASURE", "description": "Increased number of cells observed in peri-pancreatic adipocytes from SENP1-aP2KO mice, suggesting a potentially enhanced proliferative activity or survival mechanism.<SEP>There were more cell numbers in PAT cells of the SENP1-aP2KO mice than Ctrl adipocytes under normal chow and this difference was amplified under HFD.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4<SEP>chunk-174831b97ed170039b374ba254d4f506", "id": "CELL NUMBERS"}, {"entity_type": "GENE/PRODUCT", "description": "Reduced expression levels of markers typically associated with adipocyte differentiation (fatty acid synthase, adipose triglyceride lipase and lipoprotein lipase) in peri-pancreatic adipocytes from SENP1-aP2KO mice.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "DIFFERENTIATION MARKERS"}, {"entity_type": "DISEASE/CONDITION", "description": "The functional diversity of adipose tissue depots can lead to different disease processes, with some depots showing enhanced pro-inflammatory phenotype due to changes like those observed in PATs.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "DEPOT-SPECIFIC PHENOTYPES"}, {"entity_type": "CELL SURFACE MARKER", "description": "Adipocytes expressing FABP4 are associated with a pro-inflammatory phenotype.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "ADIPOCYTE SPECIFIC MARKER (FABP4)"}, {"entity_type": "PATHOLOGICAL PROCESS", "description": "The result of strong induction by adipocyte culture supernatant from SENP1-aP2KO mice, involving the destruction of pancreatic tissues and \u03b2-cell death.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "PANCREATIC DISRUPTION"}, {"entity_type": "DIAGNOSTIC METHOD", "description": "A method used to monitor cell viability and cytotoxic effects on isolated pancreatic islets upon exposure to adipocyte culture supernatant.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "LIVE/DEAD VIABILITY/CYTOTOXICITY ASSAYS"}, {"entity_type": "MOLECULAR PROCESS", "description": "The process of cell death affecting \u03b2-cells in response to increased proinflammatory cytokines secreted by adipocytes.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "\u0392-CELL DEATH"}, {"entity_type": "CELL SURFACE MARKER", "description": "A marker used for identifying macrophages, which was not found colocalized with phosphor-p65/RelA in SENP1-aP2KO adipocytes.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "MACROPHAGE MARKER (F4/80)"}, {"entity_type": "MOLECULAR BIOLOGY TECHNIQUE", "description": "An assay used to identify the binding sites of p65/RelA on the promoters of pro-inflammatory cytokines like IL-6, TNF-\u03b1 and IL-1\u03b2.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "CHROMATIN IMMUNOPRECIPITATION (CHIP) ASSAY"}, {"entity_type": "ORGANISM TYPE", "description": "A cell line derived from white adipose tissue that can be differentiated into mature adipocytes for studying adipogenesis and related processes.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "PREADIPOCYTE CELL LINE (3T3-L1)"}, {"entity_type": "MOLECULAR TOOL", "description": "Used to knockdown the expression of a specific gene, in this case SENP1, in 3T3-L1 preadipocyte cells.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "SHORT INTERFERING RNA (SIRNA)"}, {"entity_type": "GENE/PROCESS", "description": "The phenomenon observed when genes encoding pro-inflammatory cytokines are upregulated in adipocytes due to increased NF-\u03baB activity following SENP1 deletion.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "PROINFLAMMATORY GENE EXPRESSION"}, {"entity_type": "MOLECULAR PROCESS", "description": "Resulting from enhanced NF-\u03baB activation, leading to elevated expression of cytokines like IL-6, TNF-\u03b1 and IL-1\u03b2 in adipose tissues.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "INCREASED PRO-INFLAMMATORY CYTOKINE LEVELS"}, {"entity_type": "DISEASE", "description": "The NOD mouse is a popular model used for studying type 1 diabetes (T1DM) due to its development of diabetes with late onset.<SEP>The NOD mouse is a popularly used model for T1DM, showing spontaneous development of diabetes.", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce<SEP>chunk-8382a49b61215482f9fc75e3a86cb987", "id": "NOD MOUSE"}, {"entity_type": "DISEASE", "description": "Consistent with reports, we observed the incidence of spontaneous diabetes in the NOD mouse was 60\u201380% in females and 20\u201330% in males.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 50}]", "id": "NON-OBESE DIABETIC (NOD) MOUSE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "NOD mice exhibit increased plasma IFN-\u03b3 with lower levels of proinflammatory cytokines compared with SENP1-aP2KO mice and serum from human T1DM patients.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "TH1 DOMINANT IFN-\u0393"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Increased plasma levels of proinflammatory cytokines were significantly increased in T1DM patients compared with the normal cohorts.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "PROINFLAMMATORY CYTOKINE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Isolated normal mouse pancreatic islets were cultured with serum collected from Ctrl, SENP1-aP2KO and NOD mice, or collected from normal human and T1DM patients. Islet structure and apoptosis was monitored by Live/Dead viability/Cytotoxicity assays.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "BETA-CELL DEATH"}, {"entity_type": "THERAPY", "description": "Our results indicated that prophylactic insulin therapy in SENP1-deficient mice at the age of 5 weeks for 9 weeks only partially ameliorates development of diabetes phenotypes including immune cell infiltration, \u03b2-cell apoptosis and islet disruptions.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "PROPHYLACTIC INSULIN THERAPY"}, {"entity_type": "DRUG", "description": "JSH-23 had no effects on body weight, blood glucose and lipid profile in the Ctrl mice. However, administration of JSH-23 in SENP1-aP2KO mice diminished SENP1 deletion-induced elevation of blood glucose and reduction of insulin levels in blood.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "NF-\u039aB INHIBITOR II JSH-23"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "HFD increased fat weight in all adipose depots including epididymal fat and PATs in Ctrl and SENP1-aP2KO mice. HFD may stimulate adipogenesis in", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "HIGH-FAT DIET (HFD)"}, {"entity_type": "MANIFESTATIONS", "description": "Diabetes phenotypes include immune cell infiltration, \u03b2-cell apoptosis and islet disruptions observed in diabetic mice.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "DIABETES PHENOTYPES"}, {"entity_type": "CELL DEATH", "description": "The process of \u03b2-cell death that occurs due to inflammation or autoimmune attack during the progression of diabetes.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "\u0392-CELL APOPTOSIS"}, {"entity_type": "STRUCTURAL CHANGES", "description": "Damage to islets in pancreatic tissue leading to malfunction and dysfunction, commonly seen in diabetic conditions.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "ISLET DISRUPTIONS"}, {"entity_type": "PATHOLOGICAL PROCESS", "description": "The autoimmune response against insulin-producing \u03b2-cells which leads to their destruction in diabetes.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "AUTOIMMUNE ATTACK"}, {"entity_type": "RESPONSE TO INJURY", "description": "Increased inflammatory responses are observed in pancreatic islets of diabetic mice, contributing to disease progression.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "INFLAMMATION"}, {"entity_type": "CELLULAR PHENOTYPE", "description": "Characteristics of the T-helper cells (Th1) that are dominant in NOD mice contributing to autoimmune response against \u03b2-cells.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "TH1 DOMINANT"}, {"entity_type": "GENE EXPRESSION", "description": "Induction of CCL5 gene expression is associated with inflammation and islet damages in diabetes, particularly under high-fat diet conditions.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "CCL5 EXPRESSION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "High-fat diet (HFD) was used to increase fat weight in all adipose depots, including epididymal fat and PATs.", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "id": "HFD"}, {"entity_type": "DISEASE", "description": "Hyperglycemia was accelerated under HFD conditions, starting at 8 weeks in SENP1-aP2KO mice instead of the usual onset at around 12 weeks on standard chow.", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "id": "HYPERGLYCAEMIA"}, {"entity_type": "DIETARY PATTERN", "description": "Under standard chow diet, basal insulin levels were not significantly different between SENP1-deficient and Ctrl mice. However, HFD significantly reduced these levels in both groups.", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "id": "BASAL INSULIN LEVELS"}, {"entity_type": "DIETARY PATTERN", "description": "Responses to glucose stimulation were similar under standard chow diet for both groups of mice, but HFD augmented the reduction of insulin levels in Ctrl mice and exacerbated it further in SENP1-aP2KO mice.", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "id": "GLUCOSE-STIMULATED INSULIN LEVELS"}, {"entity_type": "DISEASE", "description": "The SENP1 knockout (aP2KO) mice have genetic alterations related to diabetes, particularly affecting the Senp1 gene.", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "id": "SENP1-AP2KO"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "Creating genetically modified mice with adipocyte-specific deletion of Senp1 resulted in diabetes development.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 9}]", "id": "GENETIC MODIFICATION"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94<SEP>chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "Diminished expression of Senp1 was associated with diabetes development.)<SEP>The diabetes phenotype correlated with the alteration of Senp1 gene expression in adipocytes.)<SEP>The process of gene expression studied through microarray analysis and qRT-PCR methods.", "entity_type": "BIOLOGICAL PROCESS", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 53}]", "id": "GENE EXPRESSION"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "The specific cell types, such as macrophages and pancreatic islets, showed different expression patterns of Senp1 gene.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 53}]", "id": "SPECIFIC CELL TYPE"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "The expression of Senp1 gene localized to peri-pancreatic adipose tissue and was not present in pancreatic islets.)", "entity_type": "UNKNOWN", "id": "EXPRESSION"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "The expression of Senp1 gene localized to peri-pancreatic adipose tissue and was not present in pancreatic islets.)", "entity_type": "UNKNOWN", "id": "LOCALIZATION"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "Low insulin levels were observed as part of the diabetes phenotype in Senp1-deficient mice.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 50}]", "id": "HORMONAL RESPONSE"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "The tests confirmed the diabetes phenotype and insulin resistance in Senp1-deficient mice.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 50}]", "id": "ASSESSMENTS"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "Unlike the adipocyte-specific deletion models, Senp1-deficient mice created by SM22\u03b1-Cre or VE-cadherin-Cre did not develop a high-glucose phenotype.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 9}]", "id": "NORMAL PHENOTYPE"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "Reduced insulin secretion correlated with diabetes development in Senp1-deficient mice.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 50}]", "id": "HORMONAL FUNCTION"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "Lower body weight was a physical manifestation observed in the diabetic phenotype of Senp1-deficient mice.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 50}]", "id": "PHYSICAL CHARACTERISTICS"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "Alteration of cholesterol and triglyceride levels was observed as a metabolic change associated with diabetes development in Senp1-deficient mice.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 50}]", "id": "METABOLIC CHANGES"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "Increased water consumption was a behavioral indicator observed in the diabetes phenotype of Senp1-deficient mice, possibly due to osmotic diuresis.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 50}]", "id": "BEHAVIORAL INDICATORS"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "The regulation of PPAR\u03b3 by SENP1 impacts metabolic pathways involved in insulin sensitivity and glucose metabolism.)", "entity_type": "UNKNOWN", "id": "GENE REGULATION"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "The regulation of PPAR\u03b3 by SENP1 impacts metabolic pathways involved in insulin sensitivity and glucose metabolism.)", "entity_type": "UNKNOWN", "id": "METABOLIC FUNCTION"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "STZ was used as a method to induce diabetes in Senp1-deficient mice, providing insights into the role of SENP1 in maintaining glucose homeostasis.)", "entity_type": "UNKNOWN", "id": "CHEMICAL TREATMENT"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "STZ was used as a method to induce diabetes in Senp1-deficient mice, providing insights into the role of SENP1 in maintaining glucose homeostasis.)", "entity_type": "UNKNOWN", "id": "DIABETES INDUCTION"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "The impact of SENP1 deletion on the metabolic health of mice potentially leads to accelerated aging processes related to cellular senescence.)", "entity_type": "UNKNOWN", "id": "AGING PHENOMENA"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "The impact of SENP1 deletion on the metabolic health of mice potentially leads to accelerated aging processes related to cellular senescence.)", "entity_type": "UNKNOWN", "id": "GENE FUNCTION"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "SENP1\'s involvement in adipocyte differentiation highlights its importance during the development of metabolic traits like obesity and diabetes.)", "entity_type": "UNKNOWN", "id": "DEVELOPMENTAL BIOLOGY"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "SENP1\'s involvement in adipocyte differentiation highlights its importance during the development of metabolic traits like obesity and diabetes.)", "entity_type": "UNKNOWN", "id": "GENE ACTIVITY"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "The influence of SENP1 on hematopoietic stem cells could provide insights into its role in regenerative processes during diabetes development or treatment.)", "entity_type": "UNKNOWN", "id": "GENE INFLUENCE"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "The influence of SENP1 on hematopoietic stem cells could provide insights into its role in regenerative processes during diabetes development or treatment.)", "entity_type": "UNKNOWN", "id": "REGENERATIVE MEDICINE"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "Senp1 gene deletion results in a complex interplay of conditions characteristic of metabolic syndrome that affects multiple organ systems.)", "entity_type": "UNKNOWN", "id": "GENETIC PHENOTYPE"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "Senp1 gene deletion results in a complex interplay of conditions characteristic of metabolic syndrome that affects multiple organ systems.)", "entity_type": "UNKNOWN", "id": "HEALTH OUTCOMES"}, {"source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "description": "Increased cytokine concentration is a precursor event contributing to the development of diabetes and associated myocarditis symptoms in older mice.)(entity", "entity_type": "UNKNOWN", "id": "AFFECTS"}, {"source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "description": "Reduced SENP1 activity leads to increased local cytokine secretion that upregulates CCL5 expression.)", "entity_type": "UNKNOWN", "id": "REGULATION OF CCL5 EXPRESSION"}, {"source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "description": "Reduced SENP1 activity leads to increased local cytokine secretion that upregulates CCL5 expression.)", "entity_type": "UNKNOWN", "id": "SENP1 DELETION"}, {"source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "description": "Secreted from PATs, contribute to inflammation and immune cell infiltration in the pancreas.)", "entity_type": "UNKNOWN", "id": "LOCAL PRO-INFLAMMATORY CYTOKINES"}, {"source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "description": "Increased NF-\u03baB activity in adipose tissue affects adipogenesis through reduced levels of PPAR\u03b3 SUMOylation, influencing cell development and maturation process.)", "entity_type": "UNKNOWN", "id": "NF-\u039aB ACTIVATION"}, {"source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "description": "Increased NF-\u03baB activity in adipose tissue affects adipogenesis through reduced levels of PPAR\u03b3 SUMOylation, influencing cell development and maturation process.)", "entity_type": "UNKNOWN", "id": "REGULATION OF ADIPOCYTE DIFFERENTIATION"}, {"source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "description": "Deficiency reduces adipocyte differentiation while increasing pro-inflammatory markers like NF-\u03baB activity, contributing to enhanced inflammation in adipose depots.)", "entity_type": "UNKNOWN", "id": "ENHANCED PRO-INFLAMMATORY PHENOTYPE"}, {"source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "description": "Deletion of SENP1 in adipocytes enhances NEMO SUMOylation and NF-\u03baB activation, affecting cytokine expression and potentially impacting T1DM pathogenesis.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 54}]", "id": "NF-\u039aB SIGNALLING"}, {"source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "description": "Expression levels of transcripts are altered in adipocytes with SENP1 deletion, which may influence metabolic processes and inflammatory responses related to diabetes. )", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 10}, {level: 1, cluster: 32}]", "id": "ADIPOCYTE"}, {"source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "description": "SUMOylation of NEMO by SENP1 modulates the activation of NF-\u03baB and cytokine production in adipocytes, key aspects of inflammatory response linked to T1DM pathophysiology.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 38}]", "id": "INFLAMMATORY PROCESS"}, {"source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "description": "SENP1 deletion enhances NEMO SUMOylation even under stress conditions like DNA damage, which could contribute to inflammation and metabolic dysfunction in diabetes. )", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 39}]", "id": "DNA DAMAGE RESPONSE"}, {"source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "description": "Adipocyte-specific SENP1 deficiency may exacerbate the genetic predisposition towards T1DM by modulating multiple signaling pathways involved in insulin secretion and glucose metabolism. )", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 35}]", "id": "GENETIC PREDISPOSITION"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "A critical role of SENP1-dependent NF-\u03baB activity in T1DM progression in SENP1-aP2KO mice.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 54}]", "id": "NF-\u039aB ACTIVITY"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Increased plasma levels of proinflammatory cytokines were observed in T1DM patients compared to normal subjects.)", "entity_type": "UNKNOWN", "id": "INCREASED LEVELS"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Serum from SENP1-aP2KO mice, NOD mice and human T1DM all induced disruption and pancreatic cell death.)", "entity_type": "UNKNOWN", "id": "INDUCTION"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Prophylactic insulin therapy partially ameliorates development of diabetes phenotypes in SENP1-deficient mice.)", "entity_type": "UNKNOWN", "id": "AMELIORATION OF DIABETIC PHENOTYPES"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Administration of JSH-23 to SENP1-aP2KO mice diminished the effects of SENP1 deletion on blood glucose levels and insulin in blood.)", "entity_type": "UNKNOWN", "id": "DIMINISHED ELEVATION OF BLOOD GLUCOSE AND REDUCTION OF INSULIN LEVELS"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "HFD increased fat weight in all adipose depots including epididymal fat and PATs.)", "entity_type": "UNKNOWN", "id": "INCREASED FAT WEIGHT"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "SENP1-dependent NF-\u03baB activity regulates the expression of genes involved in inflammation, apoptosis, and islet damage during diabetes progression.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 54}]", "id": "REGULATION OF GENE EXPRESSION"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Prophylactic insulin therapy helps partially ameliorate some diabetic phenotypes by supporting insulin levels and immune cell infiltration in mice.)", "entity_type": "UNKNOWN", "id": "PREVENTATIVE EFFECT"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "JSH-23 administration reduces blood glucose elevation and insulin reduction, potentially impacting the course of T1DM in SENP1-aP2KO mice.)", "entity_type": "UNKNOWN", "id": "DISEASE MODIFYING EFFECT"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "HFD may stimulate adipogenesis and increase fat weight, which could contribute indirectly to inflammation and potentially exacerbate diabetic phenotypes.)", "entity_type": "UNKNOWN", "id": "CONTRIBUTION TO DISEASE PROGRESSION"}, {"source_id": "chunk-174831b97ed170039b374ba254d4f506", "description": "High-fat diet significantly increased the production of proinflammatory cytokines like Interleukin-6 (IL-6), Tumor Necrosis Factor Alpha (TNF-\u03b1), and Interleukin-1 Beta (IL-1\u03b2) in PATs.)", "entity_type": "UNKNOWN", "id": "IL-6, TNF-\u0391, IL-1\u0392"}, {"source_id": "chunk-174831b97ed170039b374ba254d4f506", "description": "Under basal conditions, liver insulin resistance was detected under HFD conditions, which was more pronounced than under normal chow diet in Senp1 knockout (KO) mice. This is a hallmark of obesity-linked type 2 diabetes development.)", "entity_type": "UNKNOWN", "id": "LIVER INSULIN RESISTANCE"}, {"entity_type": "GENOMIC VARIANT", "description": "ApaI (rs7975232) is a variant evaluated in association with risk for ovarian cancer.", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "id": "APAI"}, {"entity_type": "GENOMIC VARIANT", "description": "FokI (rs2228570 or rs10735810) is a variant associated with an increased risk of ovarian cancer.", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "id": "FOKI"}, {"entity_type": "GENE", "description": "Cdx-2 gene was found to be associated with colorectal cancer risk.<SEP>Cdx-2 gene was the focus in some studies evaluated for association with ovarian cancer.<SEP>Cdx-2 is a gene involved in ovarian cancer risk assessment.", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c<SEP>chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "CDX-2"}, {"entity_type": "GENOMIC VARIANT", "description": "TaqI is another gene related to ovarian cancer studies.<SEP>TaqI is considered as a transcript related to ovarian cancer risk assessment.<SEP>TaqI variant was studied as it related to ovarian cancer risk.<SEP>The TaqI variant is known for its association in the context of various disease outcomes.", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c<SEP>chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "TAQI"}, {"entity_type": "DISEASE", "description": "Carriers of the A allele showed significantly increased risk for ovarian cancer.<SEP>No association was found for the TaqI variant and the risk of ovarian cancer in both subgroup studies and Caucasians.<SEP>This disease is central as several genetic variants were associated with the risk of developing ovarian cancer in this meta-analysis.", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c<SEP>chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "OVARIAN CANCER"}, {"entity_type": "STATISTICAL TEST", "description": "Hardy-Weinberg equilibrium is a concept in population genetics that suggests the genetic makeup of a population will remain constant from generation to generation in the absence of external influences; it was not maintained among controls due to certain factors.<SEP>The equilibrium process was utilized to assess distribution uniformity for genotypes among control populations.<SEP>No strong evidence was found against Hardy-Weinberg equilibrium except in Romania.", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c<SEP>chunk-c9badb9a0fb813f42383cc8afc57ad45<SEP>chunk-7a423a439657efc1126fb6a075919ea5", "id": "HARDY-WEINBERG EQUILIBRIUM"}, {"entity_type": "GENOMIC VARIANT", "description": "BsmI (rs1544410) has been implicated as a biomarker in several studies.", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "id": "RS1544410"}, {"entity_type": "GENOMIC VARIANT", "description": "Cdx-2 is a genomic variant that shows an association with ovarian cancer risk.<SEP>Cdx-2\'s genomic variant rs11568820 had specific implications in ovarian cancer risk.<SEP>The variant was evaluated in subgroup studies conducted with a Japanese subgroup and the NECC subgroup.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "RS11568820"}, {"entity_type": "GENOMIC VARIANT", "description": "TaqI is a genomic variant that shows no significant effect on ovarian cancer risk.<SEP>rs731236 is one of the four analyzed VDR variants.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4<SEP>chunk-6600322cc75094b1c57707e4b9c85954", "id": "RS731236"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The Northern Sweden subgroup refers to geographical location of study participants involved.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "NORTHERN SWEDEN SUBGROUP"}, {"entity_type": "SCIENTIFIC STUDY", "description": "Study conducted by Clendenen et al. was one of the subgroup studies evaluated.<SEP>Study performed by Clendenen et al. was part of the literature search on Cdx-2 and ovarian cancer risk.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "CLENDENEN ET AL."}, {"entity_type": "SUBGROUP OF PARTICIPANTS", "description": "The NECC subgroup refers to a subset of participants involved in a study, potentially based on shared characteristics or location.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "NECC SUBGROUP"}, {"entity_type": "RESEARCH TEAM", "description": "A team led by Tworoger et al. contributed to subgroup studies evaluating variants like Cdx-2 related to ovarian cancer.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "TWOROGER ET AL."}, {"source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "description": "No significant association between BsmI variant and risk of ovarian cancer was found under the dominant genetic model.)", "entity_type": "UNKNOWN", "id": "BSMI"}, {"source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "description": "The African American subgroup study contributed significantly to the heterogeneity between the studies evaluated for BsmI variant.)", "entity_type": "UNKNOWN", "id": "AFRICAN AMERICAN POPULATION"}, {"source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "description": "The African American subgroup study contributed significantly to the heterogeneity between the studies evaluated for BsmI variant.)", "entity_type": "UNKNOWN", "id": "GENETIC VARIANTS"}, {"source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "description": "The genomic variant rs11568820 in Cdx-2 gene was associated with an increased risk of ovarian cancer.)", "entity_type": "UNKNOWN", "id": "CDX-2 GENE VARIANT AND OVARIAN CANCER"}, {"source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "description": "The TaqI gene variant showed no association with ovarian cancer based on subgroup studies.)", "entity_type": "UNKNOWN", "id": "TAQI GENE VARIANT AND OVARIAN CANCER"}, {"entity_type": "GENOMIC VARIANT", "description": "rs7975232 is one of the four analyzed VDR variants.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "RS7975232"}, {"entity_type": "GENOMIC VARIANT", "description": "rs1544410 is one of the four analyzed VDR variants.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "RS154410"}, {"entity_type": "GENOMIC VARIANT", "description": "rs2228570 is one of the four analyzed VDR variants.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "RS2228570"}, {"entity_type": "GENE", "description": "The VDR gene contains the four analyzed variants: rs731236, rs7975232, rs1544410 and rs2228570.<SEP>The VDR gene encodes vitamin D receptor, which plays a role in calcium and phosphate metabolism and regulates gene transcription.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "VDR"}, {"entity_type": "DISEASE", "description": "T. cruzi is a term used to refer to the infection studied in this text.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "T. CRUZI"}, {"entity_type": "DISEASE STATE", "description": "Seronegative refers to individuals without antibodies against T. cruzi, indicating no previous exposure or current non-infection by T. cruzi.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "SERONEGATIVE"}, {"entity_type": "DISEASE STATE", "description": "Seropositive refers to individuals with antibodies against T. cruzi, indicating a history of infection by this parasite.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "SEROPOSITIVE"}, {"entity_type": "DISEASE STATE", "description": "Asymptomatic refers to individuals without symptoms or clinical signs of Chagas disease (CCC), despite being infected.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "ASYMPTOMATIC"}, {"entity_type": "DISEASE", "description": "Chagas disease (CCC) is a term used for the disease studied in this text, characterized by various manifestations depending on infection stage and individual immunity.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "CHAGAS DISEASE (CCC)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Linkage disequilibrium (LD) is the non-random association between genetic variants that are inherited together.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "LD"}, {"entity_type": "DISEASE", "description": "Chagas disease, caused by the parasite Trypanosoma cruzi, results in chronic infection and can lead to various complications like heart failure.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "CHAGAS DISEASE"}, {"entity_type": "MOLECULAR PRODUCT", "description": "Antibodies are proteins produced by immune cells that recognize and bind to specific antigens.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "ANTIBODIES"}, {"entity_type": "SAMPLE TYPE", "description": "Serum is a biological sample that contains various proteins, including cytokines like IL-8.<SEP>Serum is the liquid part of blood after clotting factors have been removed.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68<SEP>chunk-6600322cc75094b1c57707e4b9c85954", "id": "SERUM"}, {"entity_type": "DISEASE STATE", "description": "Asymptomatic individuals are those who carry a disease but do not exhibit any symptoms.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "ASYMPTOMATIC INDIVIDUALS"}, {"entity_type": "DISEASE STATE", "description": "Symptomatic individuals display one or more signs or symptoms of Chagas disease.", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "id": "SYMPTOMATIC INDIVIDUALS"}, {"source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "description": "The minor allele, *A of rs2228570 is overrepresented in the seronegative subset.)", "entity_type": "UNKNOWN", "id": "RS2228570*"}, {"source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "description": "Specific VDR SNP haplotypes correlate with susceptibility or resistance to T. cruzi infection.)<SEP>The VDR polymorphisms may influence the risk of infection by T. cruzi, but only rs2228570 shows a protective effect against infection.)", "entity_type": "UNKNOWN", "id": "VDR SNPS"}, {"source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "description": "The three variants form three different haplotypes (ACC, GAT and AAC).)", "entity_type": "UNKNOWN", "id": "RS731236, RS7975232, RS1544410"}, {"source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "description": "Minor allele *A of rs2228570 is overrepresented among seropositive individuals compared to seronegative and asymptomatic individuals.)", "entity_type": "UNKNOWN", "id": "SEROPOSITIVE INDIVIDUALS"}, {"entity_type": "GENE", "description": "The TT genotype of IL-1A -889C>T was found to be more frequent in HNP patients than controls.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "IL-1A -889C>T"}, {"entity_type": "GENE", "description": "CT and T alleles of TNF-A 238G>A had frequencies more than healthy controls compared to HNP cases.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "TNF-A 238G>A"}, {"entity_type": "GENE", "description": "The distribution of VDR TaqI genotypes did not show any significant difference between HNP patients and control individuals.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "VDR TAQI"}, {"entity_type": "DISEASE", "description": "HNPs (healthier neck pain) were studied for associations with the genotypes of IL-1A, TNF-A, and VDR.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "HNP"}, {"entity_type": "POOLED ESTIMATE", "description": "The odds ratio for IL-1A -889C>T genotype was calculated at the risk level of HNPs in comparison to CC vs. CT+TT genotypes.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "OR: 0.82"}, {"entity_type": "POOLED ESTIMATE", "description": "No association between TNF-A 238G>A and HNP susceptibility was observed when comparing GG vs. GA+AA genotypes.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "OR: 0.63"}, {"entity_type": "GENOMIC VARIANT", "description": "IL-1A rs1800587 was included in studies for HNP.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "RS1800587"}, {"entity_type": "GENOMIC VARIANT", "description": "TNF-A rs361525 polymorphism was considered among the factors studied for HNP.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "RS361525"}, {"entity_type": "GENE", "description": "IL-1A, TNF-A, and VDR were genes that were examined in relation to HNP.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "GENES"}, {"entity_type": "GENE", "description": "Assumed to influence offspring birthweight through maternal health.<SEP>Explained variance in serum B12 concentrations in GWAS.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "MATERNAL SERUM B12 LEVELS"}, {"entity_type": "STUDY DESIGN", "description": "Performed a GWAS to find variants related to B12 levels.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "GWAS (GENOME-WIDE ASSOCIATION STUDY)"}, {"entity_type": "GENE", "description": "Explained variance in serum folate concentration in GWAS.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "MATERNAL FOLATE LEVELS"}, {"entity_type": "VARIANT TYPE", "description": "Close to 6 million SNPs were identified for association with gene expression.<SEP>Included in MR analysis to explain variance.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca<SEP>chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "SNPS (SINGLE NUCLEOTIDE POLYMORPHISMS)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Demographic, clinical, and patient-reported risk factors explained 13% of the variance in acute nausea; ABCB1 genotype contributed an additional 5% to this variability (p=0.10).<SEP>Used F statistics for individual SNPs and IVW analysis for causal effect estimates.<SEP>The process of statistical testing to evaluate genetic associations with hematological parameters.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8<SEP>chunk-3ae06e9a3330573da28ca0a76aa685ce<SEP>chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "STATISTICAL ANALYSIS"}, {"entity_type": "NUTRIENT", "description": "Measured concentrations of serum B12 in pregnant women.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "SERUM B12 CONCENTRATIONS"}, {"entity_type": "NUTRIENT", "description": "Measured concentrations of serum folate in pregnant women.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "SERUM FOLATE CONCENTRATION"}, {"entity_type": "OUTCOME", "description": "Birthweight is a measure often used in maternal and child health studies to assess potential effects on offspring.<SEP>Measured outcome influenced by maternal and fetal nutrient levels.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "OFFSPRING BIRTHWEIGHT"}, {"entity_type": "NUTRIENT", "description": "Measured levels of B12 in the fetus, not associated with offspring birthweight.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "FETAL B12 LEVELS"}, {"entity_type": "NUTRIENT", "description": "Measured levels of folate in the fetus, negatively associated with their own birthweight.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "FETAL FOLATE LEVELS"}, {"entity_type": "POPULATION", "description": "Participants of the study on maternal and fetal nutrient levels.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "PREGNANT WOMEN FROM EFSOCH STUDY"}, {"entity_type": "STUDY DESIGN", "description": "Conducted observational research focusing on pregnant women\'s nutrient levels and birth outcomes.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "EFSOCH STUDY"}, {"entity_type": "METHODOLOGY", "description": "Used for causal effect estimation in the presence of potential confounders.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "IVW ANALYSIS (INVERSE VARIANCE WEIGHTED)"}, {"entity_type": "STATISTICAL APPROACH", "description": "Conducted to assess robustness of results under different assumptions or scenarios.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "SENSITIVITY ANALYSES"}, {"entity_type": "STATISTICAL TEST", "description": "Used to identify variability in the causal effect estimates across individual SNPs.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "HETEROGENEITY TESTS"}, {"entity_type": "GENOMIC VARIANT", "description": "Appeared to have little effect on birthweight of pregnant women.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "MMACHC VARIANT (RS12272669)"}, {"entity_type": "GENOMIC VARIANT", "description": "A priori expected to have pleiotropic effects, but found no strong evidence for it.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "FUT2 VARIANT (RS602662)"}, {"entity_type": "GENOMIC VARIANT", "description": "Caused attenuation in the causal effect of maternal folate levels on birthweight when excluded from analysis.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "MTHFR SNP (RS1801133)"}, {"entity_type": "GENOMIC VARIANT", "description": "Appeared to have little effect on birthweight of pregnant women.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "MMACHC SNP (RS12272669)"}, {"entity_type": "GENOMIC VARIANT", "description": "A priori expected to have pleiotropic effects, but found no strong evidence for it.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "FUT2 SNP (RS602662)"}, {"entity_type": "VITAMIN", "description": "Vitamin B12 levels are influenced by folate intake and may affect birthweight according to some studies.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "id": "B12"}, {"entity_type": "NUTRIENT", "description": "Maternal consumption of folate is a factor in the relationship between maternal vitamin B12 levels and offspring birthweight.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "id": "MATERNAL FOLATE"}, {"entity_type": "VITAMIN", "description": "Maternal intake of vitamin B12 may influence offspring birthweight, but not evidence for this was found.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "id": "MATERNAL VITAMIN B12"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Gestational duration refers to the length of time from conception until birth, which was not found to be influenced by B12 or folate levels.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "id": "GESTATIONAL DURATION"}, {"entity_type": "ANATOMICAL SEQUENCE", "description": "This refers to pathways or sequences of events that occur within specific organs or systems in the human body.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "id": "BIOLOGICAL PATHWAY"}, {"entity_type": "MEDICAL TREATMENT", "description": "A therapy refers to methods or procedures used in medicine to alleviate symptoms, cure diseases, or improve health.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "id": "THERAPY"}, {"source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "description": "No evidence for a causal effect of maternal B12 levels on offspring birthweight.)", "entity_type": "UNKNOWN", "id": "MATERNAL B12 LEVELS"}, {"source_id": "chunk-22216738ccbae0a6276bb4499018c434", "description": "There is no indication that levels of B12 or folate have an effect on gestational duration based on the data.)", "entity_type": "UNKNOWN", "id": "B12/FOLATE LEVELS"}, {"entity_type": "GENE", "description": "A single nucleotide polymorphism in the MTHFR gene, with a carrier rate of 55.04% among LV patients.<SEP>The MTHFR C677T gene variant carrier rate was 51.15% in Korea and Afghanistan.", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb<SEP>chunk-d0476488384aadcae4630e94d982d2e0", "id": "MTHFR C677T"}, {"entity_type": "GENE", "description": "A different single nucleotide polymorphism in the MTHFR gene, with a carrier rate of 43.90% among LV patients.<SEP>The MTHFR A1298C gene variant carrier rate was 43.86% in the population studied.", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb<SEP>chunk-d0476488384aadcae4630e94d982d2e0", "id": "MTHFR A1298C"}, {"entity_type": "GENE", "description": "The PAI-1 -675 4G/5G gene variant carrier rate was 82.9% in Korea and Afghanistan.<SEP>The PAI-1 gene variant with a homozygous carrier rate of 29.47%, and the highest total carrier rate (85.26%) among assessed SNPs.", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb<SEP>chunk-d0476488384aadcae4630e94d982d2e0", "id": "PAI-1 -675 4G/5G"}, {"entity_type": "GENE", "description": "A relatively rare polymorphism in PAI-1 gene, with a carrier rate of 55.56%.<SEP>The PAI-1 A844G polymorphism has not been tested yet in North America.", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb<SEP>chunk-d0476488384aadcae4630e94d982d2e0", "id": "PAI-1 A844G"}, {"entity_type": "GENE", "description": "The Prothrombin G20210A gene variant carrier rate was 6.45% in Korea and Afghanistan.<SEP>The Prothrombin G20210A gene variant carrier rate was 75% in North America.<SEP>The Prothrombin gene variant, with the lowest total carrier rate (10.59%) among assessed SNPs.", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb<SEP>chunk-d0476488384aadcae4630e94d982d2e0", "id": "PROTHROMBIN G20210A"}, {"entity_type": "GENE", "description": "A relatively low carrier rate of 14.07% in LV patients, ranked second lowest among tested SNPs.<SEP>The Factor V G1691A gene variant carrier rate was 28.57% in North America.<SEP>The Factor V G1691A gene variant carrier rate was 36.45% in Korea and Afghanistan.", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb<SEP>chunk-d0476488384aadcae4630e94d982d2e0", "id": "FACTOR V G1691A"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Activities such as physical exercise modify the impact of genetic predispositions on cardiovascular disease, in conjunction with dietary habits.", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "id": "LIFESTYLE FACTORS"}, {"entity_type": "DRUG", "description": "Statin medications are used to manage cholesterol levels and reduce the risk associated with certain genetic variants linked to cardiovascular disease.", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 43}]", "id": "STATINS THERAPY"}, {"entity_type": "PATIENT", "description": "115 LV patients were evaluated, with a total of male:female ratio as 31.30%:68.70%, mean age being 27.26 years.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "LV PATIENTS"}, {"entity_type": "GEOGRAPHICAL REGION", "description": "In Europe, there were eight articles involving 49 LV patients from five countries.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "EUROPE"}, {"entity_type": "COUNTRY", "description": "Portugal is one of the European countries where studies were conducted.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "PORTUGAL"}, {"entity_type": "COUNTRY", "description": "Spain also had involvement in these studies within Europe.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "SPAIN"}, {"entity_type": "COUNTRY", "description": "The British population was included in European studies on LV patients.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "BRITAIN"}, {"entity_type": "COUNTRY", "description": "German participants were part of the evaluation in Europe.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "GERMANY"}, {"entity_type": "COUNTRY", "description": "French individuals contributed to the studies conducted within Europe.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "FRANCE"}, {"source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "description": "The MTHFR gene\'s different variant types, like C677T and A1298C, are associated with various aspects of cardiovascular health.)", "entity_type": "UNKNOWN", "id": "MTHFR C677T/A1298C"}, {"source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "description": "Polymorphisms in the PAI-1 gene contribute to cardiovascular disease risk, as seen with variants such as -675 4G/5G and A844G.)", "entity_type": "UNKNOWN", "id": "PAI-1 4G/5G/A844G"}, {"source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "description": "SNPs in prothrombin gene (G20210A) and factor V gene (G1691A) are related to thrombosis risk and contribute to cardiovascular health.)", "entity_type": "UNKNOWN", "id": "PROTHROMBIN G20210A/FACTOR V G1691A"}, {"source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "description": "The variant carrier rate for MTHFR C677T was 64.28% in Europe.)", "entity_type": "UNKNOWN", "id": "VARIANT CARRIER RATE"}, {"entity_type": "NUTRIENT", "description": "Folic acid is a vitamin that contains B9 and it plays an important role in DNA synthesis, cell division, and the creation of new blood cells.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "FOLIC ACID"}, {"entity_type": "DIETARY PATTERN", "description": "The vitamin status of individuals may have effects on fertility, as seen with folic acid containing vitamins and semen quality.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "VITAMIN STATUS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Semen quality is a physiological process that can be affected by factors such as vitamin usage and folate intake.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "SEMEN QUALITY"}, {"entity_type": "DISEASE", "description": "Infertile men are those who have difficulties with conception, which may relate to their semen quality and vitamin status.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "INFERTILE MEN"}, {"entity_type": "DISEASE", "description": "Fertility controls are individuals without fertility issues or problems with conception, used as a comparison group in studies.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "FERTILITY CONTROLS"}, {"entity_type": "NUTRIENT", "description": "Lower serum folate concentration is associated with poorer semen quality in infertile men compared to fertile controls.<SEP>Serum folate is the concentration of folate present in blood serum and it influences semen quality according to vitamin usage.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "SERUM FOLATE"}, {"entity_type": "NUTRIENT", "description": "RCF stands for reduced cofactor function, which can be influenced by serum folate levels and associated with infertility.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "RCF"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Homocysteine (tHcy) is a metabolic product that may have implications on semen quality, but it was not significantly different between groups in this study.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "THCY"}, {"entity_type": "DIETARY PATTERN", "description": "Vitamin supplements can increase the levels of nutrients like folate and B12 while reducing tHcy concentrations, which shows benefits for semen parameters.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "VITAMIN SUPPLEMENTS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "In the Czech Republic, smoking status was associated with milk drinking habits along with educational attainment.<SEP>Smoking is a behavior that may negatively impact reproductive health and sperm quality in men.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45<SEP>chunk-7a423a439657efc1126fb6a075919ea5", "id": "SMOKING"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The period of abstinence before semen collection can influence semen parameters like concentration and volume.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "ABSTINENCE TIME"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The use of folic acid-containing vitamins might not have a significant impact on semen quality, especially in infertile men compared to fertile controls.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "FOLIC ACID-CONTAINING VITAMINS USAGE"}, {"entity_type": "DIETARY PATTERN", "description": "Individuals who use vitamin supplements tend to have higher folate and B12 levels and lower tHcy concentrations than non-users.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "VITAMIN SUPPLEMENT USERS"}, {"entity_type": "DISEASE", "description": "Being diagnosed with infertility may be associated with certain genetic factors like the choline pathway enzyme gene PEMT M175V minor V allele or variant rs173665 in TCblR.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "INFERTILITY STATUS"}, {"entity_type": "GENE", "description": "The B12 transcobalamin receptor gene has a variant rs173665 that may influence fertility status in males, showing negative association.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "B12 TRANSCOBALAMIN RECEPTOR GENE (TCBLR) VARIANT RS173665"}, {"entity_type": "GENE", "description": "The PEMT enzyme gene has a minor V allele at position 175 that may have a positive association with male infertility status.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "PHOSPHATIDYLETHANOLAMINE N-METHYLTRANSFERASE (PEMT) M175V MINOR V ALLELE"}, {"entity_type": "DISEASE", "description": "Male infertility is a condition where a man has difficulty with conception or producing viable sperm, potentially related to genetic and environmental factors.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "MALE INFERTILITY"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Semen concentration refers to the number of sperm cells per milliliter in semen, an important factor for fertility that can be influenced by vitamin usage.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "SEMEN CONCENTRATION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The total count of sperm produced or present in a sample of semen, which is crucial for fertility and can be impacted by various factors including vitamin intake and age.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "SPERM NUMBER"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Semen volume measures the amount of liquid produced during ejaculation and it plays a role in fertility; its parameters may differ between fertile men and those with male infertility.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "SEMINAL VOLUME"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Sperm motility refers to the ability of sperm cells to move and swim towards an egg, which is essential for successful fertilization; it can be influenced by vitamin status and age.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "SPERM MOTILITY"}, {"entity_type": "DISEASE COMPARISON", "description": "Fertile individuals used as a comparative group in studies assessing factors like semen quality or genetic influences on fertility.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "FERTILE CONTROLS"}, {"entity_type": "STATISTICAL ANALYSIS", "description": "Bonferroni correction is a statistical procedure used to adjust p-values for multiple comparisons, to control the false discovery rate when analyzing data; it was applied here to avoid false positives in genetic association studies.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "BONFERRONI METHOD"}, {"source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "description": "Vitamin supplementation has been shown to have beneficial effects on seminal parameters such as sperm concentration and motility in this study.)", "entity_type": "UNKNOWN", "id": "SEMINAL PARAMETERS"}, {"source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "description": "Lower serum folate levels are correlated with poorer semen quality specifically among infertile men, compared to those with fertility issues.)", "entity_type": "UNKNOWN", "id": "INFERTILE MEN VS. FERTILE CONTROLS"}, {"source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "description": "Age is a factor in determining fertility status; older individuals tend to have differences in reproductive health metrics compared to younger individuals.)", "entity_type": "UNKNOWN", "id": "FERTILITY STATUS"}, {"entity_type": "GENE", "description": "This enzyme catalyzes the reversible conversion of serine to glycine, transferring methyl groups to tetrahydrofolate (THF), which is transformed into 5,10-methylenetetrahydrofolate (5,10-CH2-THF).", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "SHMT1"}, {"entity_type": "GENOMIC VARIANT", "description": "A significant increase was observed of the G allele when compared with controls.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "RS1979277"}, {"entity_type": "MOLECULAR INTERACTION", "description": "This polymorphism results in an alteration of Leu435 to Phe.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "LEU435PHE"}, {"entity_type": "GENOMIC VARIANT", "description": "A very significant increase was observed of the C allele and CC genotype in TMD patients.<SEP>Promoter region variation (C allele) that is significantly more common in TMD patients compared with controls.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "SHMT1 RS638416"}, {"entity_type": "PROMOTER REGION", "description": "This refers to the promoter region of the SHMT1 gene.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "5\'-UTR"}, {"entity_type": "GENE", "description": "The MTHFD1 gene encodes a protein with three distinct enzyme activities: 5,10-methylenetrahydrofolate dehydrogenase, 5,10-methylenetrahydrofolate cyclohydrolase, and 10-formyltetrahydrofolate-synthetase.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "MTHFD1"}, {"entity_type": "GENOMIC VARIANT", "description": "A transition from G to A resulting in an alteration of ARG-653-GLN.<SEP>Transition variant leading to GLN change at amino acid position 653, associated with susceptibility to neural tube defects sensitive to folates.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "RS2236225"}, {"entity_type": "MOLECULAR INTERACTION", "description": "This leads to an alteration of the amino acid Arginine to Glutamine at position 653.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "RS2236225 (G TO A)"}, {"entity_type": "GENOMIC VARIANT", "description": "Deletion of the GSTM1 gene results in a null variant.<SEP>Deletion", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "GSTM1-NULL"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "This variant may increase susceptibility to environmental toxins and carcinogens.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "GSTM1-NULL (NULL VARIANT)"}, {"entity_type": "GENE", "description": "Responsible for regenerating Methionine Synthase by reductive methylation using SAM as a methyl donor.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "MTRR"}, {"entity_type": "GENOMIC VARIANT", "description": "Polymorphism affecting MTRR enzyme leading to altered methionine regeneration and Ile22Met substitution.<SEP>Substitution of A by G results in Ile22Met, producing an enzyme with less affinity for the substrate.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "RS1801394"}, {"entity_type": "GENOMIC VARIANT", "description": "This is a VNTR polymorphism that leads to differences in the number of repeats (S-DRD4 and L-DRD4).<SEP>Polymorphism", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "DRD4-48BPTR"}, {"entity_type": "MOLECULAR INTERACTION/", "description": "Variation", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "SHORT ALLELE S-DRD4/LONG ALLELE L-DRD4"}, {"entity_type": "METHOD", "description": "The CDC (Centers for Disease Control and Prevention) Measurement by Absorbance (MBA) method is used for measuring serum folate.", "source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "id": "CDC MBA"}, {"entity_type": "METHOD", "description": "The CDC High Performance Liquid Chromatography-Mass Spectrometry/Mass Spectrometry (HPLC-MS/MS) method is another technique utilized for measuring serum folate.", "source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "id": "CDC HPLC-MS/MS"}, {"entity_type": "QUALITY METRIC<|A METRIC USED TO ASSESS THE CONSISTENCY OF RESULTS WITHIN AN INDIVIDUAL LABORATORY ACROSS REPEATED MEASUREMENTS.)(ENTITY", "description": "CDC QC Procedures", "source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "id": "WITHIN-LABORATORY IMPRECISION"}, {"entity_type": "TRANSCRIPT", "description": "Results showed that the mean concentration of 30.2 nmol/L compared to 35.3 nmol/L for S-FOL with IH-CAL and 428 nmol/L compared to 469 nmol/L with WB-FOL<SEP>The mean difference to the CDC MBA target value was 30% under IH reagents.<SEP>When both reagents were harmonized, an intermediate mean difference and SD was observed.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704<SEP>chunk-7c25217961fba65d15f20beda8a58f1c", "id": "S-FOL"}, {"entity_type": "TRANSCRIPT", "description": "Results showed that the mean concentration of 30.7 nmol/L compared to 35.3 nmol/L for S-FOL and 418 nmol/L compared to 469 nmol/L with WB-FOL<SEP>Similarly to S-FOL, an intermediate mean difference and SD was estimated for WB-FOL under the harmonized condition.<SEP>The mean difference to the CDC MBA target value decreased from 25% \u00b1 19% with IH reagents to 13% \u00b1 13% with CDC reagents.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704<SEP>chunk-7c25217961fba65d15f20beda8a58f1c", "id": "WB-FOL"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Conditions related to CDC-MO showed mean results of 27.0 nmol/L for S-FOL and 358 nmol/L for WB-FOL compared to IH reagent conditions.<SEP>Microorganism provided for round-robin kit.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704<SEP>chunk-7c25217961fba65d15f20beda8a58f1c", "id": "CDC-MO"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "IH-CAL condition showed mean results that were closer to the CDC MBA target concentrations compared to IH reagents.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "IH-CAL"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The model estimated among-laboratory variability for conditions affecting microorganism and calibrator results.<SEP>The model estimated among-laboratory variability for microorganism and calibrator conditions", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "COVARIANCE"}, {"entity_type": "OVERALL MICROORGANISM EFFECT", "description": "A small effect was observed with ~5% for S-FOL and ~2% for WB-FOL when averaged across days.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "MICROORGANISM EFFECT"}, {"entity_type": "OVERALL CALIBRATOR EFFECT", "description": "Large differences were seen in the overall calibrator effects", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "CALIBRATOR EFFECT"}, {"entity_type": "DISEASE", "description": "Labs under CDC-MO & CDC-CAL condition achieved results within \u00b120% of the target value except for lab #6.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "CDC MBA TARGET VALUES"}, {"entity_type": "DIETARY PATTERN", "description": "Mean difference to CDC MBA target was estimated at 21% for S-FOL and 18% for WB-FOL with IH reagents", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "GROUP 1 LABORATORIES"}, {"entity_type": "DIETARY PATTERN", "description": "Mean difference to CDC MBA target was estimated at 30% for both S-FOL and WB-FOL with IH reagents", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "GROUP 2 LABORATORIES"}, {"entity_type": "DISEASE", "description": "An estimated small day effect of ~5% was found for S-FOL and ~2% for WB-FOL.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 36}, {level: 2, cluster: 60}]", "id": "DAY EFFECT"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The large 95% Confidence Intervals (CIs) suggested underlying variability within and among laboratories.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "95% CIS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The analysis involved transforming raw data using a logarithmic function to normalize the distribution.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "LOG TRANSFORMED RESULTS"}, {"entity_type": "STATISTICAL PROCESS", "description": "Variance was evaluated through geometric mean, which is useful for averaging rates and ratios.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "GEOMETRIC MEANS"}, {"entity_type": "VARIABILITY MEASURE", "description": "The standard deviation of 5.38 nmol/L for S-FOL and 67.5 nmol/L for WB-FOL indicated the spread of data points.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "SD (STANDARD DEVIATION)"}, {"entity_type": "DIETARY PATTERN", "description": "Calibrator provided for round-robin kit.", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "id": "CDC-CAL"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Results under CDC-MO were closer to the target values than IH reagents results in IH-CAL conditions.)", "entity_type": "UNKNOWN", "id": "CDC-MO & CDC-CAL"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Results under CDC-MO were closer to the target values than IH reagents results in IH-CAL conditions.)", "entity_type": "UNKNOWN", "id": "IH-CAL & IH-CAL"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "A small day effect was observed for S-FOL and WB-FOL averaged across days.)", "entity_type": "UNKNOWN", "id": "OVERALL MICROORGANISM EFFECT"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Large differences in calibrator effects were observed, averaging across microorganisms.)", "entity_type": "UNKNOWN", "id": "OVERALL CALIBRATOR EFFECT"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Estimations showed variation in mean difference to CDC MBA target values based on dietary pattern (IH reagents vs. CDC reagents).)", "entity_type": "UNKNOWN", "id": "GROUP 1 LABORATORIES & GROUP 2 LABORATORIES"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "The analysis considered covariance across different microorganisms and calibrators.)", "entity_type": "UNKNOWN", "id": "MICROORGANISM & CALIBRATOR"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "CIs highlighted the wide range of variation among different laboratories.)", "entity_type": "UNKNOWN", "id": "VARIABILITY WITHIN & AMONG LABORATORIES"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Logarithmic transformation was used to normalize data distributions.)", "entity_type": "UNKNOWN", "id": "NORMALIZATION PROCESS"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "High SD values indicated a larger spread or variability in S-FOL and WB-FOL concentrations.)", "entity_type": "UNKNOWN", "id": "SPREAD OF DATA POINTS"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "95% of the laboratory results were expected to be within these limits under the harmonized condition when CDC reagents were used.)", "entity_type": "UNKNOWN", "id": "CDC MBA TARGET VALUE"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "95% of the laboratory results were expected to be within these limits under the harmonized condition when CDC reagents were used.)", "entity_type": "UNKNOWN", "id": "LIMITS OF AGREEMENT"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "With the harmonized condition, it can be anticipated that the differences would amount to approximately \u00b110\u201330% for S-FOL and approximately \u00b120\u201340% for WB-FOL.)", "entity_type": "UNKNOWN", "id": "DIFFERENCES BETWEEN DAY 1 AND DAY 2"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "With the harmonized condition, it can be anticipated that the differences would amount to approximately \u00b110\u201330% for S-FOL and approximately \u00b120\u201340% for WB-FOL.)", "entity_type": "UNKNOWN", "id": "S-FOL & WB-FOL"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "Day 1 vs Day 2 analysis showed a lack of agreement in WB-FOL data, unlike other laboratories.)", "entity_type": "UNKNOWN", "id": "LAB #8 RESULTS (WB-FOL)"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "Day 1 vs Day 2 analysis showed a lack of agreement in WB-FOL data, unlike other laboratories.)", "entity_type": "UNKNOWN", "id": "LACK OF AGREEMENT"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "Exhibited less reasonable agreement compared to other labs under harmonized condition.)", "entity_type": "UNKNOWN", "id": "LABS #6 & 9"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "Observed overall good agreement except for labs #6 & 9 when using CDC round-robin kit.)", "entity_type": "UNKNOWN", "id": "AGREEMENT UNDER HARMONIZED CONDITION"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "Observed overall good agreement except for labs #6 & 9 when using CDC round-robin kit.)", "entity_type": "UNKNOWN", "id": "ALL LABS (WITH SAME MO & CAL)"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "Estimates under harmonized condition for days 1 vs. day 2 analysis.)", "entity_type": "UNKNOWN", "id": "DIFFERENCES EXPECTED FOR S-FOL & WB-FOL"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "Estimates under harmonized condition for days 1 vs. day 2 analysis.)", "entity_type": "UNKNOWN", "id": "LIMITS IN RANGE OF +/-10% TO +/-30%/<BR/> +/-20% TO +/-40%"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "Lab #8 showed inconsistency between day-wise analysis.)", "entity_type": "UNKNOWN", "id": "DAY 1 DAY 2 COMPARISON"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "Lab #8 showed inconsistency between day-wise analysis.)", "entity_type": "UNKNOWN", "id": "LACK OF AGREEMENT IN WB-FOL"}, {"entity_type": "GENOMIC VARIANT", "description": "GSTP1 (Glutathione S-transferase P1), rs1695 (A>G) is correlated with CSF use during chemotherapy.<SEP>The rs1695 polymorphism was analyzed in our study, revealing its association with breast cancer risk.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404<SEP>chunk-ab5011584b0230efbf80a98b2a693a0e", "id": "RS1695"}, {"entity_type": "GENOMIC VARIANT", "description": "The rs1138272 polymorphism\'s analysis also showed no deviation from Hardy-Weinberg equilibrium; however, it was not associated with the risk of breast cancer.", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "id": "RS1138272"}, {"entity_type": "DISEASE", "description": "Breast cancer risk was analyzed using genetic data and polymorphisms such as rs1695 and rs1138272.", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "id": "BREAST CANCER"}, {"entity_type": "PROTEIN", "description": "The function of the GSTP1 protein was assessed for a specific single nucleotide polymorphism (Ile105Val), with results showing reduced amyloid propensity.", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "id": "GSTP1 PROTEIN"}, {"source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "description": "A specific single nucleotide polymorphism (Ile105Val) in GSTP1 shows reduced amyloid propensity according to WALTZ output.)", "entity_type": "UNKNOWN", "id": "ILE105VAL"}, {"entity_type": "GENE", "description": "The most common pathogenic variant among WD patients, especially prevalent in haplogroup A.<SEP>The most prevalent pathogenic variant among WD patients in China.<SEP>The most prevalent pathogenic variant in China associated with three haplogroups (A, B, C).<SEP>The most prevalent variant associated with haplogroups A, B, C.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450<SEP>chunk-fa2d834930246b534181c3d6d2c049e1", "id": "P.R778L"}, {"entity_type": "GENE", "description": "Complex haplotype spectrum including haplogroups D through I.<SEP>More complex haplotype spectrum including six haplogroups (D, E, F, G, H, I) associated with WD chromosomes.<SEP>The second-most common variant, more frequent than expected in Fujian province compared to other regions.<SEP>The second-most prevalent variant, with a slightly higher prevalence in Fujian province compared to Zhejiang.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450<SEP>chunk-fa2d834930246b534181c3d6d2c049e1", "id": "P.P992L"}, {"entity_type": "GENE", "description": "The third most prevalent variant with highest distribution observed in Fujian province.<SEP>The third-most prevalent variant, showing the highest prevalence in Fujian province.<SEP>Uniform state presented with only haplogroup J.<SEP>Uniform state presented, only one kind of haplogroup on WD chromosomes (haplogroup J).", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450<SEP>chunk-fa2d834930246b534181c3d6d2c049e1", "id": "P.T935M"}, {"entity_type": "PATIENT POPULATION", "description": "The target population of this study, comprising 715 unrelated WD patients.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "WD PATIENTS"}, {"entity_type": "SPATIAL ATTRIBUTE", "description": "Distribution of WD cases across southeastern China.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "GEOGRAPHICAL DISTRIBUTION"}, {"entity_type": "GEOGRAPHICAL ENTITY", "description": "A region with a high prevalence of p.T935M in the WD patient population.<SEP>A specific region noted for having high prevalence of certain variants compared to other provinces.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "FUJIAN PROVINCE"}, {"entity_type": "GEOGRAPHICAL ENTITY", "description": "A region where p.P992L has higher prevalence than expected.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "ZHEJIANG"}, {"entity_type": "GEOGRAPHICAL ENTITY", "description": "A region with a relatively low prevalence of p.T935M compared to other regions.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "JIANGSU"}, {"entity_type": "GEOGRAPHICAL ENTITY", "description": "A region with very limited cases for all three variants, indicating lower prevalence.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "SHANGHAI"}, {"entity_type": "GEOGRAPHICAL ENTITY", "description": "A region that shows higher prevalence of p.R778L than Zhejiang and Jiangsu but less pronounced compared to Fujian.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "JIANGXI"}, {"entity_type": "MOLECULAR INTERACTION", "description": "These two variants are often present together in some patients, suggesting co-occurrence.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "P.R778L AND P.P992L"}, {"entity_type": "MOLECULAR INTERACTION", "description": "These two variants also show a tendency to occur together, as evident from certain patient data.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "P.R778L AND P.T935M"}, {"entity_type": "VARIANTS GROUP", "description": "The three prevalent pathogenic variants associated with WD patients.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "P.R778L, P.P992L, AND P.T935M"}, {"entity_type": "SPATIAL DISTRIBUTION ATTRIBUTE", "description": "Analysis on the prevalence of pathogenic variants across different regions within China.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "GEOGRAPHICAL ANALYSIS"}, {"entity_type": "GENETIC GROUPING", "description": "The dominant genetic grouping to which p.R778L variant is strongly associated.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "HAPLOGROUP A"}, {"entity_type": "GEOGRAPHICAL ENTITY", "description": "One of the provinces with less pronounced prevalence of p.P992L compared to other variants and regions.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "ZHEJIANG PROVINCE"}, {"entity_type": "GEOGRAPHICAL ENTITY", "description": "A province showing low presence for all three variants, indicating lower prevalence.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "JIANGSU PROVINCE"}, {"entity_type": "GEOGRAPHICAL ENTITY", "description": "A city noted for having very limited cases for the pathogenic variants among WD patients.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "SHANGHAI CITY"}, {"entity_type": "GEOGRAPHICAL ENTITY", "description": "A region demonstrating high prevalence of p.R778L variant than Zhejiang but not as much as Fujian province.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "JIANGXI PROVINCE"}, {"entity_type": "GENETIC COMPOSITION ATTRIBUTE", "description": "The variation in frequency and distribution patterns of the pathogenic variants across different genetic markers.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "ALLELE DISTRIBUTION"}, {"entity_type": "HAPLOGROUP", "description": "Overwhelmingly common haplogroup associated with WD chromosomes (97.0%). Can be subdivided into haplotype variants A1\u2013A4.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "A"}, {"entity_type": "HAPLOGROUP", "description": "Relatively scarce, found in WD chromosomes at 1.0% compared to control ones. C", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "B"}, {"entity_type": "HAPLOGROUP", "description": "Appeared sporadically on WD chromosomes at 3.4%.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "D"}, {"entity_type": "HAPLOGROUP", "description": "More common on WD chromosomes than control ones, containing two variants (E1, E2), with variant E1 occupying the main part.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "E"}, {"entity_type": "HAPLOGROUP", "description": "Appeared sporadically, found at 6.9% on WD chromosomes.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "F"}, {"entity_type": "HAPLOGROUP", "description": "Sporadically appeared on WD chromosomes at 3.4%.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "G"}, {"entity_type": "HAPLOGROUP", "description": "Appeared sporadically, found at 10.3% on WD chromosomes.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "H"}, {"entity_type": "HAPLOGROUP", "description": "Appeared sporadically, found at 3.4% on WD chromosomes.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "I"}, {"entity_type": "HAPLOGROUP", "description": "Only one kind of haplogroup on WD chromosomes, consisting of two variants J1 (12-4-8) and J2 (12-5-8), showing great significance vs. control ones.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "J"}, {"entity_type": "HAPLOTYPE VARIANTS A1\u2013A4", "description": "Variants of haplogroup A.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "10-7-7; 10-7-8; 10-8-7; 10-8-8"}, {"entity_type": "HAPLOTYPE VARIANTS E1, E2", "description": "Variants associated with haplogroup E.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "10-9-5; 10-9-6"}, {"entity_type": "HAPLOTYPES J1, J2", "description": "Variants associated with haplogroup J.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "10-4-8; 12-4-8"}, {"entity_type": "3.4%", "description": "Frequencies of Haplogroups A and B in Control Chromosomes", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "97.0%/"}, {"entity_type": "NUMBER OF HAPLOTYPES E/", "description": "E variants found on WD chromosomes", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "21/29; 6/177"}, {"source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "description": "Significant linkage disequilibrium found between D13S314 marker and p.R778L and p.P992L variants.)", "entity_type": "UNKNOWN", "id": "D13S314"}, {"source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "description": "Significant linkage disequilibrium found between D13S314 marker and p.R778L and p.P992L variants.)", "entity_type": "UNKNOWN", "id": "LINKAGE DISEQUILIBRIUM"}, {"source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "description": "Significant association of D13S301 with p.R778L, p.P992L and p.T935M at specific alleles.)", "entity_type": "UNKNOWN", "id": "D13S301"}, {"source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "description": "Significant linkage disequilibrium detected between D13S316 marker and the three prevalent pathogenic variants, showing strong associations at different alleles.)", "entity_type": "UNKNOWN", "id": "D13S316"}, {"source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "description": "Significantly higher frequency of p.P992L in Fujian province compared to Zhejiang.)", "entity_type": "UNKNOWN", "id": "ALLELE FREQUENCY COMPARISON"}, {"source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "description": "Highest prevalence of p.T935M variant found in Fujian province, compared to other regions.)", "entity_type": "UNKNOWN", "id": "REGIONAL PREVALENCE"}, {"source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "description": "Tendency for these two variants to occur together among a subset of patients.)", "entity_type": "UNKNOWN", "id": "CO-OCCURRENCE"}, {"source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "description": "A1", "entity_type": "UNKNOWN", "id": "HAPLOTYPE VARIANTS A1\u2013A4/"}, {"source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "description": "E2", "entity_type": "UNKNOWN", "id": "HAPLOTYPE VARIANT E1/"}, {"source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "description": "Frequencies of Haplogroups A/B in Control Chromosomes", "entity_type": "UNKNOWN", "id": "3.4%"}, {"entity_type": "GENOMIC VARIANT", "description": "The minor allele frequency is low (0.02), consistent with previous studies.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "RS1799853"}, {"entity_type": "GENE", "description": "We combined rs1799853, rs1057910, rs28371686, rs7900194, rs28371685,(CYP2C9*2, *3, *5, *8, and *11) into a CYP2C9 star variant group due to low frequency of the individual SNPs.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "CYP2C9 STAR VARIANT GROUP"}, {"entity_type": "GENOMIC VARIANT", "description": "The minor allele frequency is also low (0.02), consistent with previous studies.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "RS1057910"}, {"entity_type": "GENOMIC VARIANT", "description": "This SNP has a lower minor allele frequency compared to rs1799853 and rs1057910 (0.06 respectively).", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "clusters": "[{level: 0, cluster: 7}, {level: 1, cluster: 26}]", "id": "RS9923231"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The individual SNPs reduce CYP2C9 activity.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "CYP2C9 ACTIVITY"}, {"entity_type": "STATISTICAL PROCESS", "description": "The SNPs did not show deviations from Hardy Weinberg equilibrium at P<0.001.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "HWE"}, {"entity_type": "DRUG CONCENTRATION OR DOSE", "description": "The mean therapeutic warfarin dose was 46.9 mg/week.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "WARFARIN DOSE"}, {"entity_type": "MEDICAL CONDITION", "description": "More than half of the patients were started on warfarin due to Deep Vein Thrombosis (DVT) or Pulmonary Embolism (PE).", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "DVT/PE"}, {"entity_type": "CLINICAL TRIAL", "description": "The algorithms were tested in an African American (AA) validation cohort.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "AA VALIDATION COHORT"}, {"entity_type": "PERFORMANCE METRIC", "description": "Comparing predicted dose requirements with actual maintenance dose.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "ALGORITHM ACCURACY"}, {"entity_type": "GENOMIC VARIANT", "description": "One of several SNPs combined into a CYP2C9 star variant group.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "RS28371686"}, {"entity_type": "GENOMIC VARIANT", "description": "Another SNP in the combination for creating the CYP2C9 star variant group.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "RS7900194"}, {"entity_type": "GENOMIC VARIANT", "description": "Part of a set of SNPs forming a CYP2C9 star variant group.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "RS28371685"}, {"entity_type": "ALGORITHM", "description": "A machine learning algorithm used to predict drug dosages.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "NOVEL AA ALGORITHM"}, {"entity_type": "GROUP", "description": "A patient cohort that receives a high dosage of the medication.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "HIGH DOSE GROUP"}, {"entity_type": "ALGORITHM", "description": "Two other pharmacogenomics-based algorithms for predicting drug dosages.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "IWPC ALGORITHMS"}, {"entity_type": "PERCENTAGE", "description": "The percentage of patients correctly assigned to the high-dose group by one of the IWPC algorithms.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "26%"}, {"entity_type": "PERCENTAGE", "description": "The percentage of patients correctly assigned to the high-dose group by another IWPC algorithm.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "2%"}, {"entity_type": "QUALITY MEASURE", "description": "A measure indicating how close predicted doses are to observed doses, often represented as Mean Absolute Error (MAE).", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "PRECISION"}, {"entity_type": "CLINICAL BENEFIT INDEX", "description": "An index to determine the clinical benefit of integrating a novel algorithm into practice.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "NNG"}, {"entity_type": "GENOMIC VARIANT", "description": "Genotyping using the TaqMan method assessed associations with different SNPs.<SEP>Genotyping was conducted on 93 patients as part of the study.<SEP>The process of determining an individual\'s genetic makeup, particularly their admixture in this context.", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f<SEP>chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8<SEP>chunk-b2730f3087aee6d9cff6e4939b229323", "id": "GENOTYPING"}, {"entity_type": "POPULATION GROUP", "description": "Admixed African Americans, individuals with a mixed ancestry containing African and European genes.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "AAS"}, {"entity_type": "GENETIC PARAMETER", "description": "The proportion of an individual\'s genetic makeup that originates from the admixed African American population.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "WAA"}, {"entity_type": "ALGORITHM", "description": "A specific pharmacogenomics-based prediction tool for drug dosages, compared with our novel AA algorithm.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "IWPC PHARMACOGENOMICS ALGORITHM"}, {"entity_type": "DOSE RANGE", "description": "A defined limit for drug dosages that ensure effectiveness and safety according to medical standards.<SEP>The accepted or optimal dosage limit for a medication within which it is considered to be effective and safe.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "20% THERAPEUTIC DOSE RANGE"}, {"entity_type": "DOSAGE ESTIMATION", "description": "The calculated amount of drug to be administered based on genetic factors and other variables.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "PREDICTED DOSE"}, {"entity_type": "STATISTICAL MEASURE", "description": "Average proportion of an individual\'s admixed African American ancestry, used as a measure in the IWPC algorithm.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "MEAN PERCENT WAA"}, {"entity_type": "BIOLOGICAL TRAIT", "description": "The inherited genetic information that determines an individual\'s traits and influences their susceptibility to diseases.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "GENETIC BACKGROUND"}, {"entity_type": "GENETIC COMPOSITION", "description": "A mix of different ancestries within a population or among individuals.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "ADMIXTURE"}, {"entity_type": "MEDICAL TREATMENT", "description": "Quantities of medication given as part of therapeutic interventions.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "DRUG DOSAGES"}, {"entity_type": "CULTURAL AND BIOLOGICAL FACTOR", "description": "Unique factors related to a specific group that can influence drug effectiveness, including genetic makeup.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "POPULATION-SPECIFIC VARIABLES"}, {"entity_type": "DOSAGE CALCULATION", "description": "Predicted amounts of medication administered based on computational models or algorithms.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "PREDICTED DOSES DERIVED"}, {"entity_type": "PERFORMANCE METRIC", "description": "A measure indicating the average magnitude of errors in a set of predictions, without considering their direction. It\'s used here to evaluate predictive accuracy.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "MAE (MEAN ABSOLUTE ERROR)"}, {"entity_type": "DOSAGE PRECISION", "description": "Drug dosages that fall within an acceptable and defined percentage difference from the target dose.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "DOSES PREDICTED WITHIN 20% RANGE"}, {"entity_type": "PHARMACEUTICAL MODEL", "description": "A comprehensive model or framework used to optimize drug dosage recommendations based on genetic information.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "IWPC (INTEGRATED WORKFLOW FOR PHARMACOGENOMICS)"}, {"entity_type": "CLINICAL EVALUATION METHOD", "description": "An evaluation process that compares the benefit of using a new algorithm against existing methods in clinical practice.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "COMPUTING THE NNG"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "The minor allele frequency for rs1057910 indicates reduced CYP2C9 activity.)", "entity_type": "UNKNOWN", "id": "CYP2C9*3"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "The minor allele frequency for rs9923231 indicates a lower minor allele frequency than other SNPs, consistent with previous studies.)", "entity_type": "UNKNOWN", "id": "CYP2C9*8"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "SNPs did not show deviations from Hardy Weinberg equilibrium at P<0.001)", "entity_type": "UNKNOWN", "id": "STATISTICAL PROCESS"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Accuracy of predicting dose requirements compared with actual maintenance dose.)", "entity_type": "UNKNOWN", "id": "PERFORMANCE METRIC"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "The algorithms were tested using African American (AA) patients in a clinical trial.)", "entity_type": "UNKNOWN", "id": "CLINICAL TRIAL"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Together, they form part of the CYP2C9 star variant group indicating reduced activity.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 39}]", "id": "RS1799853 AND RS1057910"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "These are other individual SNPs that were part of the combined CYP2C9 star variant group.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 7}, {level: 1, cluster: 26}]", "id": "CYP2C9*5, CYP2C9*8, AND CYP2C9*11"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "This set of SNPs collectively represents the CYP2C9 star variant group that was considered.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "SNP SET COMBINING RS1799853, RS1057910, RS28371686, RS7900194, RS28371685, CYP2C9*2, CYP2C9*3, CYP2C9*5, CYP2C9*8 AND CYP2C9*11"}, {"entity_type": "GENE", "description": "CYP27B1 gene plays a role in vitamin D metabolism.", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "id": "CYP27B1"}, {"entity_type": "GENE", "description": "CYP24A1 gene is involved in the degradation of 1,25(OH)2D and affects vitamin D levels.", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "id": "CYP24A1"}, {"entity_type": "SNP OF CYP24A1", "description": "CYP24A1 SNP rs927650 is associated with concentrations of 25(OH)D, impacting vitamin D levels.<SEP>There is a suggestive relationship between the CYP24A1 SNP rs927650 and concentrations of 25(OH)D.", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "id": "RS927650"}, {"entity_type": "SNP OF CYP24A1", "description": "CYP24A1 SNP rs35051736 shows a statistically significant relationship with odds for advanced adenoma recurrence.", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "id": "RS35051736"}, {"entity_type": "SNP OF CYP24A1", "description": "The G allele in CYP24A1 rs2296241 is associated with higher odds for recurrent adenomas with villous histology.", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "id": "CYP24A1 RS2296241"}, {"entity_type": "SNP OF CYP27B1", "description": "CYP27B1 SNP rs2762939 has a relationship with concentrations of 1,25(OH)2D.", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "id": "CYP27B1 RS2762939"}, {"entity_type": "METABOLIC PATHWAY", "description": "The metabolism and levels of vitamin D metabolites such as 25(OH)D and 1,25(OH)2D are impacted by genetic variations in CYP27B1 and CYP24A1.", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "id": "VITAMIN D METABOLITE CONCENTRATIONS"}, {"entity_type": "DISEASE", "description": "There is a statistically significant association between certain SNPs of the CYP24A1 gene and odds for adenoma recurrence.", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "id": "ADENOMA RECURRENCE"}, {"entity_type": "DISEASE", "description": "Increasing number of copies of the G allele in CYP24A1 rs2296241 is associated with higher odds for recurrent adenomas with villous histology.", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "id": "VILLOUS HISTOLOGY ADENOMAS"}, {"entity_type": "PROTEIN", "description": "Apolipoprotein B-440 plays a role in lipid metabolism.", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "id": "APOLIPOPROTEIN B-440"}, {"entity_type": "GENOMIC VARIANT", "description": "The minor allele is a variant found on one copy of a gene.", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 36}, {level: 2, cluster: 60}]", "id": "MINOR ALLELE"}, {"source_id": "chunk-2fb95100a1290b27baae869c6560419a", "description": "SNP rs927650 in CYP24A1 shows a relationship with vitamin D metabolite concentration 25(OH)D.)", "entity_type": "UNKNOWN", "id": "25(OH)D CONCENTRATION"}, {"source_id": "chunk-2fb95100a1290b27baae869c6560419a", "description": "SNP rs927650 in CYP24A1 shows a relationship with vitamin D metabolite concentration 25(OH)D.)", "entity_type": "UNKNOWN", "id": "CYP24A1 SNP RS927650"}, {"source_id": "chunk-2fb95100a1290b27baae869c6560419a", "description": "SNP rs35051736 has a statistically significant relationship with odds for advanced adenoma recurrence.)", "entity_type": "UNKNOWN", "id": "ADVANCED ADENOMA RECURRENCE"}, {"source_id": "chunk-2fb95100a1290b27baae869c6560419a", "description": "SNP rs35051736 has a statistically significant relationship with odds for advanced adenoma recurrence.)", "entity_type": "UNKNOWN", "id": "CYP24A1 SNP RS35051736"}, {"source_id": "chunk-2fb95100a1290b27baae869c6560419a", "description": "CYP24A1 SNPs like rs927650 and rs35051736 have relationships with vitamin D metabolites 25(OH)D and 1,25(OH)2D.)", "entity_type": "UNKNOWN", "id": "CYP24A1 SNPS"}, {"source_id": "chunk-2fb95100a1290b27baae869c6560419a", "description": "SNPs like rs2296241 and rs2762939 in CYP27B1 show relationships with vitamin D metabolites 25(OH)D and 1,25(OH)2D.)", "entity_type": "UNKNOWN", "id": "CYP27B1 SNPS"}, {"source_id": "chunk-2fb95100a1290b27baae869c6560419a", "description": "SNPs like rs2296241 and rs35051736 affect the odds of recurrent adenomas with villous histology.)", "entity_type": "UNKNOWN", "id": "CYP24A1 SNPS (RS2296241, RS35051736)"}, {"source_id": "chunk-2fb95100a1290b27baae869c6560419a", "description": "SNPs like rs2296241 and rs35051736 affect the odds of recurrent adenomas with villous histology.)", "entity_type": "UNKNOWN", "id": "VILLOUS HISTOLOGY ADENOMAS RECURRENCE"}, {"source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "description": "The NOS1 protein catalyzes the production of nitric oxide, playing a role in its synthesis.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 44}]", "id": "NITRIC OXIDE"}, {"source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "description": "Dietary factors can impact cardiovascular health through various molecular interactions.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 45}]", "id": "CARDIOVASCULAR HEALTH"}, {"source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "description": "Dietary choices can interact with genetic factors to impact health outcomes.)<SEP>Dietary factors can interact with genetic factors to impact health outcomes.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 45}]", "id": "GENETIC INFLUENCE"}, {"entity_type": "GENOMIC VARIANT", "description": "The c.428_430delAAG variant is known in CYP24A1 gene.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "C.428_430DELAAG VARIANT"}, {"entity_type": "GENOMIC VARIANT", "description": "The c.1186C\u2009>\u2009T variant is also known in CYP24A1 gene.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "C.1186C\u2009>\u2009T VARIANT"}, {"entity_type": "GENE", "description": "Samples with wild-type status are used as positive controls.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "WT (WILD-TYPE)"}, {"entity_type": "TRANSCRIPT", "description": "The wt amplicons represent sequences that do not carry any variant in CYP24A1 gene.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "WT AMPLICON"}, {"entity_type": "TRANSCRIPT", "description": "Amplicon corresponding to the c.428_430delAAG variant.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "C.428_430DELAAG VARIANT AMPLICON"}, {"entity_type": "TRANSCRIPT", "description": "Amplicon corresponding to the c.1186C\u2009>\u2009T variant.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "C.1186C\u2009>\u2009T VARIANT AMPLICON"}, {"entity_type": "SNP (SINGLE NUCLEOTIDE POLYMORPHISM)", "description": "The SNP rs777676129 is located in CYP24A1 gene, with different melting temperatures for wt, heterozygous and homozygous genotypes.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "RS777676129"}, {"entity_type": "SNP (SINGLE NUCLEOTIDE POLYMORPHISM)", "description": "The SNP rs114368325 is another polymorphism located in CYP24A1 gene with specific melting behavior for different genotypes.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "RS114368325"}, {"entity_type": "INDIVIDUAL/GENOTYPE", "description": "Sample carrying both tested variants in CYP24A1 gene.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "COMPOUND HETEROZYGOUS SAMPLE"}, {"entity_type": "QUANTITATIVE MEASURE", "description": "Tm value obtained from HRMA analysis, indicating the melting point of DNA samples.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "NORMALIZED TM (TM VALUE)"}, {"entity_type": "VISUAL ANALYSIS TOOL", "description": "Tool used to view differences in melting profiles that are not visible under normal Tm conditions.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "TEMPERATURE-SHIFTED AND DIFFERENT PLOT"}, {"entity_type": "ANALYTICAL TECHNIQUE", "description": "A technique for accurate genotyping of CYP24A1 gene variants using HRMA analysis.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "DHRMA (DEEP HIGH RESOLUTION MELTING ASSAY)"}, {"entity_type": "EXPERIMENTAL SETUP", "description": "A setup performed simultaneously for both amplicons using HRMA analysis.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "DUPLEX ASSAY"}, {"entity_type": "ANALYTICAL PERFORMANCE", "description": "Precision achieved in repeated experiments with the same sample using the assay.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "INTRA-ASSAY PRECISION"}, {"entity_type": "ANALYTICAL CONSISTENCY", "description": "Consistency observed across multiple experiments with different samples.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "INTER-ASSAY PRECISION"}, {"entity_type": "SAMPLE PREPARATION TECHNIQUE", "description": "Method used to test sensitivity of HRMA by reducing DNA concentration and still distinguishably identifying variants.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "SERIAL DILUTIONS"}, {"entity_type": "SAMPLE", "description": "Samples with wild-type CYP24A1 gene status used as positive controls.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "WT SAMPLE"}, {"entity_type": "GENOTYPE/", "description": "Samples with homozygous genotypes for the variant in question.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "HOMOZYGOUS SAMPLE"}, {"entity_type": "GENOTYPE/", "description": "Samples with heterozygous genotypes for the variant in question.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "HETEROZYGOUS SAMPLE"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "Amplicon corresponding to the c.1186C\u2009>\u2009T variant is genotyped using SNP rs114368325.)", "entity_type": "UNKNOWN", "id": "RS114368325 SNPS"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "The Tm values from the HRMA analysis are characteristic of different genotypes for SNPs rs777676129 and rs114368325.)", "entity_type": "UNKNOWN", "id": "NORMALIZED TM VALUES"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "The Tm values from the HRMA analysis are characteristic of different genotypes for SNPs rs777676129 and rs114368325.)", "entity_type": "UNKNOWN", "id": "RS777676129 SNPS AND RS114368325 SNPS"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "The Tm differences allow distinguishing the two amplicons corresponding to different variants.)", "entity_type": "UNKNOWN", "id": "C.428_430DELAAG VARIANT AMPLICON AND C.1186C\u2009>\u2009T VARIANT AMPLICON"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "Across multiple experiments, inter-assay precision is ensured by achieving reproducible results using different samples.)", "entity_type": "UNKNOWN", "id": "TRIPLE PCR/HRM EXPERIMENTS"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "The sensitivity of the HRMA technique allows distinguishing variants even at lower frequencies.)", "entity_type": "UNKNOWN", "id": "VARIANT ALLELE FREQUENCY"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "Homozygous samples for c.428_430delAAG variant are identified through HRMA analysis.)", "entity_type": "UNKNOWN", "id": "HOMOZYGOUS SAMPLE/"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "Heterozygous samples for c.428_430delAAG variant are identified through HRMA analysis.)", "entity_type": "UNKNOWN", "id": "HETEROZYGOUS SAMPLE/"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "The protocol is validated on further 10 DNA samples having the suspicion of idiopathic intracranial hypertension (IIH).)", "entity_type": "UNKNOWN", "id": "10 DNA SAMPLES WITH IIH SUSPICIONS"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "The protocol is validated on further 10 DNA samples having the suspicion of idiopathic intracranial hypertension (IIH).)", "entity_type": "UNKNOWN", "id": "DUPLEX-PCR/HRMA PROTOCOL/"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "Specific heterozygous sample for c.428_430delAAG variant is identified through the protocol.)", "entity_type": "UNKNOWN", "id": "1 HETEROZYGOUS C.428_430DELAAG SAMPLE/"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "Specific heterozygous sample for c.1186C\u2009>\u2009T variant is identified through the protocol.)", "entity_type": "UNKNOWN", "id": "1 HETEROZYGOUS C.1186C\u2009>\u2009T SAMPLE/"}, {"entity_type": "CLINICAL OUTCOME", "description": "The patient\u2019s developmental skill levels, as assessed by the Denver Developmental Screening Test II.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "DENVER DEVELOPMENTAL SCREENING TEST II"}, {"entity_type": "DIAGNOSTIC TESTS", "description": "A brain MRI at 20 months of age showed atrophy and delayed myelination.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "BRAIN MRI"}, {"entity_type": "CLINICAL NEUROPHYSIOLOGY STUDIES", "description": "Indicators of motor function that showed normal values.<SEP>Median nerve compound motor action potentials were measured, showing values that were within the normal range for infants.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "MEDIAN NERVE COMPOUND MOTOR ACTION POTENTIALS"}, {"entity_type": "CLINICAL NEUROPHYSIOLOGY STUDIES", "description": "Median nerve conduction velocity was also evaluated and showed a value that was within the normal range.<SEP>Neural conduction speed was measured and fell within the expected range.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "MEDIAN NERVE CONDUCTION VELOCITY"}, {"entity_type": "CLINICAL NEUROPHYSIOLOGY STUDIES", "description": "Median nerve sensory action potential was measured, showing a value that was above the normal range for infants.<SEP>Sensory response levels were analyzed, falling above the normal threshold.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "MEDIAN NERVE SENSORY ACTION POTENTIAL"}, {"entity_type": "BIOCHEMICAL FINDINGS", "description": "The cerebrospinal fluid copper and catecholamine ratios were analyzed and fell within the established ranges for classical Menkes disease.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "CEREBROSPINAL FLUID COPPER AND CATECHOLAMINE RATIOS"}, {"entity_type": "MOLECULAR ANALYSES", "description": "Confirmation of mosaicism for the P1001L mutation was carried out using a BsmAI restriction site.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "P1001L MUTATION IN GENOMIC DNA"}, {"entity_type": "MOLECULAR ANALYSES", "description": "The relative proportions of fragments associated with the wild type and mutant alleles were estimated through densitometry and Q-RT-PCR experiments.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "WILD TYPE: MUTANT ALLELE RATIOS"}, {"entity_type": "TISSUE SAMPLES", "description": "A primary fibroblast culture obtained from the skin biopsy showed a marked difference in the abundance of the wild type allele compared to blood DNA.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "BLOOD DNA"}, {"entity_type": "TISSUE SAMPLES", "description": "In comparison to whole blood DNA, primary fibroblasts showed an 12:88 ratio of wild type to mutant alleles.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "FIBROBLAST DNA"}, {"entity_type": "TISSUE COMPARISON", "description": "Quantitative real-time PCR experiments demonstrated a similar comparison between the wild type and mutant allele ratios in both tissues.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "WILD TYPE: MUTANT RATIO IN BLOOD VS. PRIMARY FIBROBLASTS"}, {"entity_type": "GENOMIC VARIANT", "description": "The patient\'s development levels and skills as assessed by Denver Developmental Screening Test II.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "CLINICAL OUTCOME"}, {"entity_type": "TRANSCRIPT", "description": "Tests such as MRI and clinical neurophysiology studies were conducted.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "DIAGNOSTIC TESTS"}, {"entity_type": "DISEASE", "description": "Levels of cerebrospinal fluid copper did not indicate copper overload or Menkes disease.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "CEREBROSPINAL FLUID COPPER"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The balance of catecholamines was assessed, showing inconsistency with Menkes disease criteria.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "CATECHOLAMINE RATIOS"}, {"entity_type": "GENE", "description": "This specific genetic variation was confirmed through molecular analysis.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "P1001L MUTATION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A new BsmAI recognition site was created due to the P1001L mutation.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "BSMAI RESTRICTION SITE"}, {"entity_type": "RELATIONSHIPS AMONG ENTITIES", "description": "The relative abundance of wild type vs. mutant alleles differed between whole blood and fibroblast samples.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "WILD TYPE: MUTANT ALLELE RATIO IN BLOOD DNA"}, {"entity_type": "COMPARATIVE ANALYSIS", "description": "A lower level of the wild type allele was detected compared to blood DNA and control samples.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "WILD TYPE: MUTANT ALLELE RATIO IN PRIMARY FIBROBLASTS"}, {"source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "description": "The P1001L mutation was confirmed by creating a novel BsmAI restriction site.)", "entity_type": "UNKNOWN", "id": "MOLECULAR ANALYSES"}, {"source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "description": "BsmAI digestion levels were compared between whole blood DNA and fibroblast DNA, showing differences in mosaicism.)", "entity_type": "UNKNOWN", "id": "BSMAI DIGESTION LEVEL"}, {"source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "description": "The wild type: mutant allele ratio was estimated using densitometry and Q-RT-PCR methods, with different levels found in blood compared to fibroblast DNA.)", "entity_type": "UNKNOWN", "id": "COMPARATIVE ANALYSIS"}, {"source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "description": "The assessments showed normal levels indicative of healthy motor function.)", "entity_type": "UNKNOWN", "id": "CLINICAL NEUROPHYSIOLOGY STUDIES"}, {"source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "description": "Levels in cerebrospinal fluid indicated no copper overload or Menkes disease characteristics based on these ratios.)", "entity_type": "UNKNOWN", "id": "BIOCHEMICAL FINDINGS"}, {"entity_type": "CELL CULTURE MODEL", "description": "Caco-2 cells do not express deiodinase at a level detectable by enzyme assay.<SEP>Human intestinal epithelial cells cultured as Caco-2 were used in studies regarding the expression of LCT mRNA levels under specific genetic manipulations.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c<SEP>chunk-c7872499c2a84b16c506a135d8033bb7", "id": "CACO-2 CELLS"}, {"entity_type": "PROMOTER OR REGULATORY SEQUENCE", "description": "The GPX4 3\u2032UTR was used to link rat IDI coding sequences in transfected Caco-2 cells.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "GPX4 3\u2032UTR"}, {"entity_type": "GENE", "description": "Transfection of Caco-2 cells with rat IDI coding sequences linked to the GPX4 3\u2032UTR resulted in the detection of IDI transcripts.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "RAT IDI"}, {"entity_type": "SNP/SNP VARIANT", "description": "The SNP rs713041 corresponds to a variant that differs between the T (IDI-T) and C (IDI-C) variants.)", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "RS713041"}, {"entity_type": "GENE", "description": "Transfected cells expressed transgenes corresponding to either IDI-T or IDI-C.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "TRANSGENE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Conditions where cells are grown in media lacking sufficient selenium supply.<SEP>Conditions where cells are grown without sufficient selenium supply, affecting cell viability.<SEP>Conditions where cells are grown without sufficient selenium supply.<SEP>|>Supplement", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a<SEP>chunk-c7872499c2a84b16c506a135d8033bb7", "id": "SE-DEPLETED MEDIUM"}, {"entity_type": "CHEMICAL/", "description": "|>Reagent", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "TERT-BUTYLHYDROPEROXIDE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Cell growth conditions with additional sources of selenium added to the media.<SEP>Conditions with an added source of selenium to cell growth media.<SEP>|>Supplement", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a<SEP>chunk-c7872499c2a84b16c506a135d8033bb7", "id": "SE-SUPPLEMENTED MEDIUM"}, {"entity_type": "ENZYME ACTIVITY", "description": "In both sets of transfected cells, under different Se supply conditions, the GPx activity was lower in Se-depleted compared to Se-supplemented cells.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "GPX ACTIVITY"}, {"entity_type": "PROTEIN/", "description": "|>Selenoprotein", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "TR1 PROTEIN"}, {"entity_type": "PROTEIN/", "description": "|>Selenium-Dependent Enzyme", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "GPX4 PROTEIN"}, {"entity_type": "MESSENGER RNA/", "description": "The mRNA transcript encoding selenoprotein H, a protein involved in the selenium metabolism process.<SEP>|>Gene Expression", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a<SEP>chunk-c7872499c2a84b16c506a135d8033bb7", "id": "SELENOPROTEIN H MRNA"}, {"entity_type": "ENZYME/", "description": "|>Hormone Metabolism", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "DEIODINASE"}, {"entity_type": "SNP/SNP VARIANT/", "description": "|>Genetic Variation", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "RS713041 VARIANT"}, {"entity_type": "MOLECULAR COMPONENT/", "description": "|>Selenium Metabolism<|The availability of Se and selenocysteine incorporation machinery is considered in the study regarding its impact on antioxidant selenoproteins synthesis compared to the IDI transgene (which has no antioxidant function).", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "SE INCORPORATION MACHINERY"}, {"entity_type": "CULTURE CONDITION/", "description": "|>Selenium Concentration<|Experiments were carried out under varying Se supply conditions in culture media, affecting cellular responses.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "SE SUPPLY CONDITIONS/EXPOSURE"}, {"entity_type": "QUANTIFIABLE MEASURE/", "description": "|>Oxidative Stress<|Measuring ROS levels following tert-butylhydroperoxide treatment is crucial to understand cell stress response.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "ROS LEVELS/REACTIVE SPECIES"}, {"entity_type": "NUTRIENT", "description": "Selenium (Se) is an essential nutrient required for the synthesis of selenoproteins.<SEP>Selenium plays a crucial role as an essential nutrient for cells.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "SE"}, {"entity_type": "GENE", "description": "The gene GPx4 encodes glutathione peroxidase 4, which plays a role in antioxidant defense against oxidative stress.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "GPX4"}, {"entity_type": "CELL LINE", "description": "Cells from IDI-C line were shown to have differences compared to IDI-T in selenoprotein expression and oxidative stress sensitivity.<SEP>Cells from cell line IDI-C show changes in selenoprotein expression and sensitivity to oxidative stress.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "IDI-C CELLS"}, {"entity_type": "GENE", "description": "The gene TR1 encodes thioredoxin, a redox-regulatory protein that interacts with selenium.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "TR1"}, {"entity_type": "GENE", "description": "The gene GPx1 encodes glutathione peroxidase 1, which also plays an antioxidant role.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "GPX1"}, {"entity_type": "TRANSCRIPT", "description": "mRNA transcripts of the genes coding for glutathione peroxides are detected in the study.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "GPX4 MRNAS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The level of selenium present in the growth medium of cells.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "SE CONTENT OF THE MEDIUM"}, {"entity_type": "CELL LINE", "description": "Conversely, cells of IDI-T line had distinct characteristics when compared with IDI-C in response to selenium conditions and oxidative challenges.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "IDI-T CELLS"}, {"entity_type": "DIETARY PATTERN", "description": "An oxidant agent used in experiments to induce oxidative stress on cells.<SEP>The chemical agent used in the oxidative stress challenge to cells.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "0.5 MM TERT-BUTYLHYDROPEROXIDE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The measure of cell health and function, affected by various environmental factors including selenium supply.<SEP>The measure of cell health is used to infer the potential development or impact on conditions that could be described under \'disease\' context, in this case referring to oxidative stress-related cellular damage.<SEP>The measure of cellular health and functional capacity under different conditions and treatments.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "CELL VIABILITY"}, {"entity_type": "DIETARY PATTERN", "description": "Exposure of cells to oxidative stress, simulating conditions that may occur in an environment with high levels of reactive oxygen species (ROS).", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "0.5 MM TERT-BUTYLHYDROPEROXIDE CHALLENGE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The availability and concentration of selenium impacting cell function and viability.<SEP>The availability and concentration of selenium in cell culture conditions, crucial for many biochemical processes.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "SE SUPPLY"}, {"entity_type": "NUTRIENT", "description": "Low concentration of selenium added to media for experimental purposes.<SEP>Low concentration of selenium added to the media.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "0.5 NM SE"}, {"entity_type": "NUTRIENT", "description": "An even lower concentration of selenium, used in experimental conditions to assess its impact on cell behavior.<SEP>An even lower level of selenium supplementation for certain experiments to assess cellular responses.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "1 NM SE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The element selenium plays a role as both nutrient and environmental factor influencing cellular processes and responses.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "SELENIUM (SE)"}, {"entity_type": "GENE", "description": "The TR1 gene produces thioredoxin, a protein involved in redox regulation related to selenium metabolism.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "TR1 GENE"}, {"entity_type": "GENE", "description": "The GPx1 gene codes for glutathione peroxidase 1, which contributes to the antioxidant defense system.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "GPX1 GENE"}, {"entity_type": "DISEASE CONTEXT", "description": "Cells with lower levels of selenium show signs resembling disease states due to inadequate antioxidant defenses.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "CELL VIABILITY UNDER LOW SE"}, {"source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "description": "In Se-supplemented medium, the GPx4 protein levels increased significantly in IDI-C cells compared to those expressing the IDI-T transgene, suggesting a stronger antioxidant response.)", "entity_type": "UNKNOWN", "id": "EFFECT ON CELL VIABILITY/IMPACT ON ANTIOXIDANT ENZYME ACTIVITY"}, {"source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "description": "The transfected cells showed lower viability over a range of tert-butylhydroperoxide concentrations with 40\u201360% viability when challenged with 1\u20135 mM tert-butylhydroperoxide.)", "entity_type": "UNKNOWN", "id": "SE-DEPLETED MEDIUM/"}, {"source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "description": "The transfected cells showed lower viability over a range of tert-butylhydroperoxide concentrations with 40\u201360% viability when challenged with 1\u20135 mM tert-butylhydroperoxide.)", "entity_type": "UNKNOWN", "id": "|>OXIDATIVE CHALLENGE/SE SUPPLY CONDITIONS"}, {"source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "description": "In Se-depleted conditions, GPx activity was lower compared to that in Se-supplemented cells.)", "entity_type": "UNKNOWN", "id": "|>GPX ACTIVITY/IMPACT ON SELENOPROTEINS"}, {"source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "description": "When sufficient Se is present, the IDI-C cell line showed a greater increase in GPx4 content than the IDI-T cell line when grown under Se-supplemented conditions.)", "entity_type": "UNKNOWN", "id": "SE SUPPLY CONDITIONS/"}, {"source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "description": "When sufficient Se is present, the IDI-C cell line showed a greater increase in GPx4 content than the IDI-T cell line when grown under Se-supplemented conditions.)", "entity_type": "UNKNOWN", "id": "|>GPX4 PROTEIN LEVELS"}, {"source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "description": "Different levels of selenium availability affect the behavior and response of IDI-C and IDI-T cell lines, impacting viability and sensitivity to oxidative challenges.)", "entity_type": "UNKNOWN", "id": "IDI-C CELLS AND IDI-T CELLS"}, {"source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "description": "Cells exposed to very low concentrations of selenium show decreased viability compared to those grown in conditions with adequate selenium supply.)", "entity_type": "UNKNOWN", "id": "0.5 NM SE OR 1 NM SE"}, {"source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "description": "Cells exposed to very low concentrations of selenium show decreased viability compared to those grown in conditions with adequate selenium supply.)", "entity_type": "UNKNOWN", "id": "CELL VIABILITY UNDER LOW SELENIUM CONDITIONS"}, {"source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "description": "Indications of cell damage or dysfunction that could be related to disease states when cells are exposed to high levels of reactive oxygen species (ROS).)", "entity_type": "UNKNOWN", "id": "DISEASE CONTEXT IN TERMS OF OXIDATIVE STRESS"}, {"source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "description": "Cells treated with selenium supplementation show improved viability compared to standard growth conditions.)", "entity_type": "UNKNOWN", "id": "CELL VIABILITY UNDER HIGH SE"}, {"source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "description": "The oxidative stress induced by tert-butylhydroperoxide affects cell health, impacting factors like apoptosis or cell cycle progression.)", "entity_type": "UNKNOWN", "id": "RESPONSE OF CELLS"}, {"source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "description": "Lower concentrations of selenium have distinct effects on cells, often leading to different outcomes compared to adequate selenium levels.)", "entity_type": "UNKNOWN", "id": "IMPACT ON CELLULAR RESPONSES"}, {"source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "description": "The presence and supply of selenium influence the effectiveness of various cellular defense systems against oxidative stress.)", "entity_type": "UNKNOWN", "id": "CELLULAR DEFENSE MECHANISMS"}, {"entity_type": "GENE", "description": "We identified 12 missense and one nonsense mutations in CHAT in 11 kinships.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "CHAT"}, {"entity_type": "MUTATION", "description": "Nine mutations are novel; p.Thr553Asn and p.Ser704Pro detected in this laboratory were published but not functionally characterized by. p.Ala557Thr was previously reported by.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "MISSENSE MUTATION"}, {"entity_type": "MUTATION", "description": "p.Ala557Thr also determines the phenotype in patient 10 who carries a nonsense mutation on her second allele.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "NONSENSE MUTATION"}, {"entity_type": "MISSENSE MUTATION", "description": "This mutation results in a substitution of Thr to Asn at residue 553, which is conserved across several species.<SEP>This mutation was detected but not functionally characterized by previous studies.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "P.THR553ASN"}, {"entity_type": "MISSENSE MUTATION", "description": "Another missense mutation involving Serine to Proline substitution at residue 704 with conservation across different organisms.<SEP>This mutation was previously identified but needs further functional characterization.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "P.SER704PRO"}, {"entity_type": "MISSENSE MUTATION", "description": "A novel mutation with a phenotypic impact that was confirmed in multiple patients.<SEP>Previously reported and confirmed in this study for its impact on phenotype.<SEP>This mutation involves Alanine to Threonine substitution which is conserved in multiple species including dog, mouse, rat and chicken.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "P.ALA557THR"}, {"entity_type": "RESIDUE", "description": "Conserved residue among canine, murine, rat, and chicken ChAT proteins.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "MET202"}, {"entity_type": "RESIDUE", "description": "This conserved residue is also present in Drosophila and C. elegans.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "TRP421"}, {"entity_type": "RESIDUE", "description": "Conserved among canine, murine, rat, chicken, Drosophila and C. elegans ChAT proteins.<SEP>Conserved among canine, murine, rat, chicken, Drosophila, and C. elegans ChAT proteins.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "THR553"}, {"entity_type": "RESIDUE", "description": "Conservation of this residue is shared across canine, murine, rat, chicken, Drosophila and C. elegans.<SEP>Conservation of this residue is shared across canine, murine, rat, chicken, Drosophila, and C. elegans.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "ALA557"}, {"entity_type": "SUBJECT", "description": "Homozygous for p.Ala557Thr, showing apparent homozygosity.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "PATIENT 8"}, {"entity_type": "SUBJECT", "description": "Heterozygous for p.Al557Thr and inherited the mutation from the unaffected mother.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "PATIENT 5"}, {"entity_type": "SUBJECT", "description": "Homozygous for p.Ile689Ser, born to consanguineous parents.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "PATIENT 11"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Eight mutants expressed at significantly lower levels than wild-type and five of these (p.Trp421Ser, p.Ser498Pro, p.Thr553Asn, p.Ala557Thr, p.Ser572Trp) expressed at <50% of wild-type.)", "entity_type": "UNKNOWN", "id": "CHAT MUTATIONS"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Eight mutants expressed at significantly lower levels than wild-type and five of these (p.Trp421Ser, p.Ser498Pro, p.Thr553Asn, p.Ala557Thr, p.Ser572Trp) expressed at <50% of wild-type.)", "entity_type": "UNKNOWN", "id": "EXPRESSION LEVEL"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Ten mutations alter one or more rate constants of ChAT activation and two mutations compromise the thermal stability of the mutant protein.)", "entity_type": "UNKNOWN", "id": "KINETIC PARAMETERS"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Ten mutations alter one or more rate constants of ChAT activation and two mutations compromise the thermal stability of the mutant protein.)", "entity_type": "UNKNOWN", "id": "MISSENSE MUTATIONS"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Compared the expression levels of wild-type CHAT versus mutant CHATs.)", "entity_type": "UNKNOWN", "id": "COMPARISON WITH MUTANT CHATS"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Compared the expression levels of wild-type CHAT versus mutant CHATs.)", "entity_type": "UNKNOWN", "id": "WILD-TYPE CHAT"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Mutations like p.Ala557Thr compromise the thermal stability of ChAT.)", "entity_type": "UNKNOWN", "id": "IMPACT ON PROTEIN STABILITY"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "The study highlights the importance of conserved residues like Met202, Trp421, Thr553, Ala557 among canine, murine, rat, chicken, Drosophila, and C. elegans ChAT proteins.)", "entity_type": "UNKNOWN", "id": "CONSERVATION ACROSS SPECIES"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "The study highlights the importance of conserved residues like Met202, Trp421, Thr553, Ala557 among canine, murine, rat, chicken, Drosophila, and C. elegans ChAT proteins.)", "entity_type": "UNKNOWN", "id": "RESIDUES"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "The presence of specific mutations like p.Ala557Thr in multiple patients indicates their influence on the phenotype.)", "entity_type": "UNKNOWN", "id": "MUTATIONS"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "The presence of specific mutations like p.Ala557Thr in multiple patients indicates their influence on the phenotype.)", "entity_type": "UNKNOWN", "id": "PHENOTYPIC IMPACT"}, {"entity_type": "GENE", "description": "Annotation: BCL2-like 11 [apoptosis facilitator]<SEP>We found nominal associations for SNPs associated with methylation at CpG sites annotated to BCL2L11.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e<SEP>chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "BCL2L11"}, {"entity_type": "GENE", "description": "Annotation: NADH dehydrogenase [ubiquinone] Fe-S protein 7, 20 kDa", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "NDUFS7"}, {"entity_type": "GENE", "description": "Annotation: eukaryotic translation initiation factor 3, subunit J", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "EIF3J"}, {"entity_type": "GENE", "description": "Annotation: zinc finger protein 761", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "ZNF761"}, {"entity_type": "GENE", "description": "Annotation: proteasome assembly chaperone 3", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "PSMG3"}, {"entity_type": "GENE", "description": "Annotation: sema domain, transmembrane domain [TM], and cytoplasmic domain ([semaphorin] 6A)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "ENTHD1"}, {"entity_type": "GENE", "description": "Annotation: LRIG1", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "LRIG1"}, {"entity_type": "GENE", "description": "Annotation: RIPK3", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "RIPK3"}, {"entity_type": "GENE", "description": "Annotation: RPS6KC1", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "RPS6KC1"}, {"entity_type": "GENE", "description": "Annotation: ATP2B4", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "ATP2B4"}, {"entity_type": "GENE", "description": "Annotation: sema domain, transmembrane domain [TM], and cytoplasmic domain ([semaphorin] 6A)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "SEMA6A"}, {"entity_type": "GENE", "description": "Annotation: TAP binding protein [tapasin]", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "TAPBP"}, {"entity_type": "GENE", "description": "Annotation: ral guanine nucleotide dissociation stimulator-like 2 (FDR\u2009<\u20090.05)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "RGL2"}, {"entity_type": "GENE", "description": "Annotation: cytoskeleton associated protein 5", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "CKAP5"}, {"entity_type": "GENE", "description": "Annotation: FOS-like antigen 2", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "FOSL2"}, {"entity_type": "GENE", "description": "Annotation: family with sequence similarity 20, member C", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "FAM20C"}, {"entity_type": "GENE", "description": "Annotation: FERM Domain containing 6", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "FRMD6"}, {"entity_type": "GENE", "description": "Annotation: family with sequence similarity 216, member A", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "FAM216A"}, {"entity_type": "GENE", "description": "Annotation: chromosome 1 open reading frame 98", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "C1ORF98"}, {"entity_type": "GENE", "description": "Annotation: DEAD (Asp-Asn) box polypeptide 11 like 1, also known as X-linked SET domain containing 3", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "DDX11L1"}, {"entity_type": "GENE", "description": "Annotation: chromosome 1 open reading frame 20", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "C1ORF20"}, {"entity_type": "GENE", "description": "Annotation: DEAD (Asp-Asn) box polypeptide 57", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "DDX57"}, {"entity_type": "GENE", "description": "Annotation: chromosome 9 open reading frame 38", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "C9ORF38"}, {"entity_type": "GENE", "description": "Annotation: DEAD (Asp-Asn) box polypeptide 20 like 1, also known as SET domain containing 5", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "DDX20L1"}, {"entity_type": "GENE", "description": "Annotation: chromosome 7 open reading frame 46", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "C7ORF46"}, {"entity_type": "GENE", "description": "Annotation: DEAD (Asp-Asn) box polypeptide H, also known as KIAA0528", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "DDX3H"}, {"entity_type": "GENE", "description": "Annotation: chromosome 1 open reading frame 149", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "C1ORF149"}, {"entity_type": "GENE", "description": "Annotation: DEAD (Asp-Asn) box polypeptide 6, also known as Ddx6", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "DDX6"}, {"entity_type": "GENE", "description": "Annotation: chromosome 7 open reading frame 30", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "C7ORF30"}, {"entity_type": "GENE", "description": "Annotation: DEAD (Asp-Asn) box polypeptide 1, also known as DExD-box protein 1", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "DDX1"}, {"entity_type": "GENE", "description": "Annotation: chromosome 5 open reading frame 49", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "C5ORF49"}, {"entity_type": "GENE", "description": "Annotation: DEAD (Asp-Asn) box polypeptide X, also known as DDX3H", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "DDX3X"}, {"entity_type": "FUNCTIONAL REGULATORY ELEMENT", "description": "Enrichment in CpGs associated with oxidative markers compared to all CpGs on the array suggests enrichment in functional regulatory elements.<SEP>Enrichment of CpGs associated with oxidative markers in CpG island shores suggests potential impact on transcription and regulation.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "CPG ISLAND SHORES"}, {"entity_type": "GENE", "description": "45% of phenotype-associated CpGs are located in promoters, indicating potential influence on gene expression.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "PROMOTER"}, {"entity_type": "FUNCTIONAL REGULATORY REGION", "description": "Twenty-three percent of sites showing strong evidence of being part of functional regulatory regions by RegulomeDB analysis.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "REGULOMEDB SCORE 1A-2C"}, {"entity_type": "TRANSCRIPTION FACTOR", "description": "Enrichment of TF binding sites (E2F1) by PSCAN analysis indicates regulation of genes annotated to phenotype-associated CpGs.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "TF BINDING SITES"}, {"entity_type": "GENE SET ENRICHMENT ANALYSIS", "description": "DAVID\'s clustering tool shows evidence of genes with functional terms such as phosphorylation and phosphate metabolic processes, indicating biological pathways related to CpG annotated genes.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "FUNCTIONAL ANNOTATION CLUSTERING TOOL"}, {"entity_type": "SNP", "description": "41.8% of phenotype-CpGs have significant cis-meQTLs, suggesting genetic influences on DNA methylation variation associated with oxidative markers.<SEP>Investigation of 1599 significant meQTL SNPs found no evidence for overrepresentation among CHD or T2D associations, suggesting independent genetic and epigenetic influences.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "CIS-MEQTL SNPS"}, {"entity_type": "ASSOCIATION BETWEEN GENOTYPE AND PHENOTYPE", "description": "Modest evidence for association between genotype-phenotype, genotype-CpG and CpG-phenotype overlap suggests mediation by epigenetics in certain cases.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "GENOTYPE-PHENOTYPE ASSOCIATION"}, {"entity_type": "INTERGENIC SITES", "description": "Sigificant meQTL SNPs annotated to intergenic sites on chromosomes 5 (rs17057846) and chromosome 12 (rs1354156), indicating roles in CpG methylation patterns.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "CHROMOSOME REGIONS"}, {"entity_type": "GENE ANNOTATION", "description": "CpG sites annotated to various genes like BCL2L11, AMDHD1, etc., showing associations with CHD and T2D in GWAS data.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "BCL2L11, AMDHD1, SEMA6A, FAM20C, HSP90AB1, WNK2"}, {"entity_type": "AMDHD1 CPG SITE ANNOTATION", "description": "CpG site cg15351688 associated with AMDHD1 shows significant associations in CHD GWAS data.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "CG15351688, RS7955450, RS7486703"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP associated with CpG site annotated to BCL2L11.<SEP>SNP associated with the CpG site in BCL2L11.<SEP>SNP rs726430 at BCL2L11 CpG site cg07547695 shows association in CHD and T2D GWAS data.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e<SEP>chunk-b05610f4a9a1033449b92655bfb67b28", "id": "RS726430"}, {"entity_type": "FAM20C CPG SITES ANNOTATION", "description": "CpG sites cg05969038 and cg21913519 annotated to FAM20C show association in CHD GWAS data.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "CG05969038, RS7786461; CG21913519: RS7786461"}, {"entity_type": "HSP90AB1 CPG SITES ANNOTATION", "description": "CpGs cg26241416 with SNPs rs6905285, rs7758726, and rs666462 show association in CHD GWAS data.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "RS6905285, RS7758726, RS666462"}, {"entity_type": "AMDHD1 CPG SITE ANNOTATIONS", "description": "CpG site cg15351688 with SNPs rs10777751, rs7955450, and rs7486703 shows association in CHD GWAS data.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "RS10777751, RS7955450, RS7486703"}, {"entity_type": "SEMA6A CPG SITES ANNOTATIONS", "description": "CpG site cg14396800 with SNPs rs2991377, rs10992689, and rs10821105 shows association in CHD GWAS data.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "RS2991377, RS10992689, RS10821105"}, {"entity_type": "CPB1 CPG SITE ANNOTATION", "description": "CpG site cg04072156 with SNP rs16861015 associated in T2D GWAS data.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "RS17139825"}, {"entity_type": "INTERGENIC SITE CPG ANNOTATION", "description": "CpG site cg14532755 with SNPs rs1665364, rs2236786, and rs3752749 shows association in T2D GWAS data.)(entity", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "RS1665364, RS2236786, RS3752749"}, {"entity_type": "PROMOTER REGION", "description": "CpGs located within the TSS200 region show significant enrichment, suggesting potential regulation of genes involved in oxidative stress processes.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "TSS200"}, {"entity_type": "FUNCTIONAL REGULATORY ELEMENT", "description": "Studies indicate that enhancer regions are also enriched with CpG sites associated with markers for oxidative stress, potentially impacting gene expression modulation.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "ENHANCERS"}, {"entity_type": "REGULATORY DNA FEATURES", "description": "Research reveals these shores have a unique pattern of CpGs related to oxidative stress, indicating their role in transcriptional regulation.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "CPG ISLANDS SHORES (50 BP AWAY FROM TSS AND CDS)"}, {"entity_type": "PROMOTER ELEMENTS", "description": "CpG islands found near TSS show a significant correlation with markers for oxidative damage, implicating them as key regulatory elements.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "TRANSCRIPTION START SITES (TSS) REGIONS"}, {"entity_type": "TRANSCRIPTOMIC FEATURES", "description": "CpGs in this region are enriched with those related to oxidative markers, highlighting their impact on gene expression and protein function.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "CODING SEQUENCE (CDS) 50 BP AWAY"}, {"entity_type": "PROMOTER NEIGHBORHOODS", "description": "Analysis of CpG patterns around TSS up to 2 kilobases away reveals correlations with oxidative stress indicators, suggesting broader promoter influence.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "TSS2K REGIONS"}, {"entity_type": "GENOMIC LANDSCAPE FEATURES", "description": "High-density CpG islands in specific genomic regions are associated with increased markers for oxidative damage, indicating complex regulatory networks.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "CPG ISLANDS DENSITY"}, {"entity_type": "EPIGENETIC LANDMARKS", "description": "Certain CpG sites near island boundaries show enrichment related to oxidative stress biomarkers, potentially influencing gene expression and chromatin structure dynamics.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "CPG ISLAND BOUNDARY MARKERS"}, {"entity_type": "REGULATORY INTERACTIONS", "description": "CpGs with strong evidence of TF binding are significantly correlated with markers for oxidative damage, indicating their role in regulatory networks associated with these processes.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "TRANSCRIPTION FACTORS BINDING SITES"}, {"entity_type": "GENETIC AND EPIGENETIC CORRELATION", "description": "Unique CpG sites annotated to specific phenotypes show differential patterns compared to general CpG distributions, suggesting involvement in disease or trait-specific mechanisms related to oxidative stress.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "PHENOTYPE-SPECIFIC CPG ANNOTATIONS"}, {"entity_type": "GENE", "description": "CpG sites annotated to CEP55 were also considered.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CEP55"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP associated with the CpG site in CEP55.<SEP>SNP linked to CpG site annotated to CEP55.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "RS12782691"}, {"entity_type": "GENE", "description": "CpG sites annotated to CPB1 were included as well.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CPB1"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP associated with the CpG site in CPB1.<SEP>SNP connected to CpG annotated to CPB1.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "RS16861015"}, {"entity_type": "INTERGENIC SITE", "description": "A CpG site was annotated to an intergenic region on chromosome 4.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CHR. 4 INTERGENIC SITE"}, {"entity_type": "GENOMIC VARIANTS", "description": "These SNPs were associated with the CpG sites mentioned above.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "RS726430, RS12782691, RS16861015, ETC."}, {"entity_type": "GENE", "description": "Methylation levels at HIST1H4D\'s 3\'-UTR was found to be associated with T2D.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "HIST1H4D"}, {"entity_type": "CPG SITE", "description": "This CpG site in the 3\'-UTR of HIST1H4D showed an association with T2D.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CG08170869"}, {"entity_type": "GENE", "description": "Methylation levels at DVL1\'s body were linked to T2D.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "DISHEVELLED-1 (DVL1)"}, {"entity_type": "CPG SITE", "description": "This CpG site in the body of DVL1 was associated with T2D.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CG03465880"}, {"entity_type": "GENES", "description": "The genes HIST1H4D and DVL1 were found to have a role in disease through their methylation levels.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "HIST1H4D, DVL1"}, {"entity_type": "DISEASE", "description": "CVD events were not significantly associated with the methylation levels at phenotype-associated CpGs, except for cg11842944 in ZNF341 and its association with heart failure.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CVD (CARDIOVASCULAR DISEASE)"}, {"entity_type": "GENE", "description": "Methylation levels at a CpG in this gene showed the strongest association to heart failure.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "ZNF341"}, {"entity_type": "CPG SITE", "description": "This CpG site was associated with both T2D and heart failure, showing differential associations.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CG11842944"}, {"entity_type": "CONDITIONS", "description": "These cardiovascular conditions were separately analyzed for their association with methylation at cg11842944 in ZNF341.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "MYOCARDIAL INFARCTION, HEART FAILURE, ISCHAEMIC STROKE"}, {"entity_type": "CPG SITE", "description": "The CpG site in the gene BCL2L11 was found associated with rs726430.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "BCL2L11 CG SITE"}, {"entity_type": "CPG SITE", "description": "The CpG site in the gene CEP55 was associated with rs12782691.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CEP55 CG SITE"}, {"entity_type": "CPG SITE", "description": "The CpG site in the gene CPB1 was related to rs16861015 SNP.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CPB1 CG SITE"}, {"entity_type": "GENOMIC VARIANTS", "description": "Multiple SNPs were associated with an intergenic CpG site on chromosome 4.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "RS732754, RS744658, ETC."}, {"entity_type": "INTERGENIC CPG SITE", "description": "An intergenic region on chromosome 4 was found to be associated with several SNPs mentioned.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CHR. 4 INTERGENIC CG SITE"}, {"entity_type": "CPG SITE (CG08170869)", "description": "Methylation at the CpG site in HIST1H4D\'s 3\' UTR was associated with T2D.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "HIST1H4D 3\'-UTR"}, {"entity_type": "CPG SITE (CG03465880)", "description": "Methylation levels at this CpG site within DVL1 were related to the risk of developing Type 2 Diabetes.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "DVL1 BODY"}, {"entity_type": "CPG SITE (CG11842944)", "description": "The methylation level at this CpG site in ZNF341 showed a link with heart failure.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "ZNF341 CG SITE"}, {"source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "description": "The CpG site in BCL2L11 was associated with the SNP rs726430.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT (RS726430)"}, {"source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "description": "The CpG site in CEP55 was linked to the SNP rs12782691.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT (RS12782691)"}, {"source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "description": "The CpG site associated with CPB1 had a connection with rs16861015 SNP.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT (RS16861015)"}, {"source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "description": "Methylation at this CpG site was associated with T2D.)", "entity_type": "UNKNOWN", "id": "CPG SITE (CG08170869)"}, {"source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "description": "This CpG site within DVL1 showed an association with the risk of developing T2D.)", "entity_type": "UNKNOWN", "id": "CPG SITE (CG03465880)"}, {"source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "description": "Methylation levels at this CpG were linked to heart failure.)", "entity_type": "UNKNOWN", "id": "CPG SITE (CG11842944)"}, {"entity_type": "PROBAND", "description": "The subject of the study, sequenced for exome enrichment and sequencing.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "CMH000067"}, {"entity_type": "PARENT", "description": "One of CMH000067\'s healthy parents.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "CMH000068"}, {"entity_type": "OTHER PARENT", "description": "The other parent of CMH000067, also healthy.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "CMH000069"}, {"entity_type": "TECHNOLOGY", "description": "The sequencer used for exome enrichment and sequencing.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "ILLUMINA HISEQ 2000"}, {"entity_type": "PROCESS", "description": "The process of capturing and analyzing genetic information from the samples.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "SEQUENCING"}, {"entity_type": "DATA SIZE", "description": "Total sequence data generated during the study.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "7.7 GIGABASES"}, {"entity_type": "COVERAGE", "description": "Average nuclear exome coverage.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "59.6\u00d7"}, {"entity_type": "TARGET COVERAGE", "description": "The percentage of targeted exome sites covered at least 16\u00d7.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "86.0%"}, {"entity_type": "GENE", "description": "Specific gene identified with a mutation.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "ATP SYNTHASE 6 GENE"}, {"entity_type": "GENE SYMBOL", "description": "Alternate notation for ATP synthase 6 gene, as it is mitochondrial.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MT-ATP6"}, {"entity_type": "DISEASE", "description": "A disease characterized by the deficiency of complex V activity due to a mutation in MT-ATP6.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "LEIGH SYNDROME"}, {"entity_type": "DATABASE ID", "description": "Identification number from Online Mendelian Inheritance in Man for Leigh syndrome.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "OMIM #516060"}, {"entity_type": "VARIANT", "description": "Specific variant identified as c.467T>G substitution leading to p.Leu156Arg amino acid change.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "C.467T>G"}, {"entity_type": "TERM", "description": "The situation where a single genetic variation is almost universally present in all cells of an individual.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "HOMOPLASMIC"}, {"entity_type": "READS COUNT", "description": "Number of times the base was sequenced during the process.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "21 SEQUENCING READS"}, {"entity_type": "CLASSIFICATION", "description": "A mutation known to cause disease due to its impact on specific protein function.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "KNOWN DISEASE CAUSING VARIANT"}, {"entity_type": "ASSAY", "description": "The method used for confirming findings in a controlled and standardized way, based on the size variation of DNA fragments.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "RESTRICTION FRAGMENT LENGTH POLYMORPHISM (RFLP)"}, {"entity_type": "METHOD", "description": "A type of sequencing that allows precise measurement of nucleotides at each position in the sequence.<SEP>The process used to confirm the heteroplasmic nature of certain mitochondrial mutations.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d<SEP>chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "CAPILLARY SEQUENCING"}, {"entity_type": "SAMPLE TYPE", "description": "Body fluid collected for genetic analysis.<SEP>The origin from which biological samples are taken, usually for medical testing purposes.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "BLOOD"}, {"entity_type": "SAMPLE TYPE", "description": "Another body fluid, this time excretory, tested for genetic material.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "URINE"}, {"entity_type": "TECHNIQUE", "description": "Advanced method of DNA sequencing that targets and analyzes all the protein-coding regions of the genome.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "NEXT-GENERATION EXOME SEQUENCING"}, {"entity_type": "OTHER SAMPLE", "description": "An individual who has passed away but was also sequenced as part of this study.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "DECEASED SIBLING"}, {"entity_type": "ANALYSIS METHOD", "description": "Exome sequencing specifically to detect genetic variations within the coding regions of genes, not the entire genome.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "EXOME SEQUENCING"}, {"entity_type": "SEQUENCING DEPTH", "description": "Amount of data generated from each sample.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "GREATER THAN 8 GIGABASES"}, {"entity_type": "TARGET COVERAGE RATE", "description": "Percentage of targeted genomic regions with at least 10\u00d7 coverage.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "87%"}, {"entity_type": "VARIANT COUNT", "description": "Total number of genetic variations identified per sample using RUNES for initial classification.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "3500 VARIANTS"}, {"entity_type": "CLASSIFICATION LEVEL", "description": "Three tiers that categorize the significance of the detected genetic variation based on severity and evidence.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "CATEGORY 1\u20133"}, {"entity_type": "ALLELE FREQUENCY THRESHOLD", "description": "Minimum frequency required for variants to be included in analysis, excluding common variations.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "< 1%"}, {"entity_type": "INHERITANCE MODEL", "description": "A theory that a genetic trait requires two copies of the variant (one from each parent) to manifest.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "RECESSIVE INHERITANCE MODEL"}, {"entity_type": "VARIANT TYPE", "description": "Variants present on both alleles for the same gene, indicating inheritance of the same mutation from both parents.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "HOMOZYGOUS VARIANTS"}, {"entity_type": "INHERITANCE PATTERN", "description": "Individuals having at least two genetic variations spread across their chromosomes from both parents.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "TWO OR MORE VARIANTS INHERITED ON DIFFERENT ALLELES"}, {"entity_type": "GENE SET", "description": "Collection of genes selected for further investigation due to shared pathogenic mutations among affected siblings.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "18 GENES"}, {"entity_type": "DISEASE-ASSOCIATED GENE", "description": "Specific gene identified as associated with mitochondrial diseases, particularly relevant here in relation to complex I dysfunction.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "NDUFV1"}, {"entity_type": "GENETIC STATE", "description": "Condition where individuals carry different mutations on each copy of the same gene, often seen in cases requiring a recessive mode of inheritance.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "COMPOUND HETEROZYGOTES"}, {"entity_type": "VARIANT IN NDUFV1 GENE", "description": "Specific substitution found within the NDUFV1 gene sequence leading to p.Glu246Lys amino acid change.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "C.736G>A"}, {"entity_type": "AMINO ACID CHANGE", "description": "The resulting amino acid sequence after a mutation occurs, here affecting Glutamic acid to Lysine at position 246 of the protein sequence.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "P.GLU246LYS"}, {"entity_type": "SECOND NDUFV1 VARIANT", "description": "Another variant in the same gene leading to p.Ala117Thr amino acid change.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "C.349G>A"}, {"entity_type": "AMINO ACID CHANGE (NDUFV1)", "description": "The outcome of the mutation at position 349, affecting Alanine to Threonine at position 117 of NDUFV1 protein sequence.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "P.ALA117THR"}, {"entity_type": "VARIANT TYPE (GENETIC MUTATIONS)", "description": "Variants that alter one or more amino acids in a protein due to mutations but not altering the reading frame, often associated with disease.(entity", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MISSENSE VARIANTS"}, {"entity_type": "METHOD", "description": "A method used to identify genetic mutations using advanced sequencing technologies.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MUTATION SCREENING BY NEXT-GENERATION SEQUENCING"}, {"entity_type": "FUNCTION", "description": "The enzymatic activity responsible for the final steps of oxidative phosphorylation in mitochondria.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MITOCHONDRIAL COMPLEX V ACTIVITY"}, {"entity_type": "SPECIFIC FORM", "description": "A variant form of Leigh syndrome characterized by additional neurological symptoms involving the brainstem and spinal cord.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "LEIGH SYNDROME WITH BRAINSTEM AND SPINAL CORD INVOLVEMENT"}, {"entity_type": "POPULATION GENETICS", "description": "A genetic lineage traced through maternal lineage, specifically in mitochondrial DNA studies.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MITOCHONDRIAL DNA HAPLOGROUP"}, {"entity_type": "STATISTICAL METHOD", "description": "The process of estimating how often specific mutations occur within a population or an individual over time.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MUTATION RATE ESTIMATION"}, {"entity_type": "COMPREHENSIVE ANALYSIS", "description": "A complete analysis method that sequences the entire human genome to identify all types of genetic variations.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "WHOLE GENOME SEQUENCING"}, {"entity_type": "CLINICAL HISTORY", "description": "The medical history within a family that includes instances or risk for developing a disease like Leigh syndrome.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "FAMILY HISTORY OF LEIGH SYNDROME"}, {"entity_type": "GENETIC VARIATION TYPE", "description": "A type of genetic variant affecting the splicing process in pre-mRNA, leading to altered gene function.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "EXONIC SPLICING VARIATION (ESV)"}, {"entity_type": "CLINICAL OBSERVATION", "description": "The range of symptoms and signs that individuals with the same underlying condition may exhibit during clinical evaluation.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "CLINICAL PRESENTATION VARIABILITY"}, {"entity_type": "DISEASE CATEGORY", "description": "A group of disorders affecting the nervous system and muscles, including Leigh syndrome.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "NEUROMUSCULAR DISEASE"}, {"entity_type": "MOLECULAR COMPONENT", "description": "The enzyme responsible for synthesizing mitochondrial RNA from DNA templates in mitochondria.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MITOCHONDRIAL RNA POLYMERASE"}, {"entity_type": "PATHOLOGICAL FEATURE", "description": "A condition where the initial step of oxidative phosphorylation is impaired, as seen in some cases of Leigh syndrome.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "COMPLEX I ACTIVITY DEFECT"}, {"entity_type": "SPECIFIC MUTATION ESTIMATION", "description": "The process to estimate how often new mutations occur without parental contribution during an individual\'s development.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "DE NOVO MUTATION RATE ESTIMATION"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "A cellular process by which mitochondria are produced and maintained in sufficient quantity for metabolic demands.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MITOCHONDRIAL BIOGENESIS"}, {"entity_type": "BIOCHEMICAL PATHWAY", "description": "The series of enzyme-catalyzed reactions that produce ATP from adenosine diphosphate (ADP) using oxygen as the final electron acceptor.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "RESPIRATORY CHAIN"}, {"entity_type": "PATHOPHYSIOLOGICAL CONDITION", "description": "A condition characterized by impaired function of mitochondria, affecting cellular energy production and other essential processes.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MITOCHONDRIAL DYSFUNCTION"}, {"entity_type": "INHERITANCE PATTERN", "description": "The way in which mitochondrial DNA traits are passed from mother to offspring, differing from nuclear DNA inheritance.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MITOCHONDRIAL INHERITANCE PATTERN"}, {"entity_type": "SPECIFIC MODE OF INHERITANCE", "description": "A type of genetic trait or disease that is transmitted directly from the mother to her offspring without involving the father\'s contribution.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MATERNAL TRANSMISSION"}, {"entity_type": "PROTEIN", "description": "The protein functions to release electrons from NADH via complex 1 of the electron transport chain.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "ACID LONG PROTEIN"}, {"entity_type": "NUTRIENT", "description": "NADH is part of a protein that binds FMN and iron\u2013sulfur clusters for proper functioning of complex I.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "NADH"}, {"entity_type": "VITAMIN", "description": "FMN, also known as flavin mononucleotide (FMN), binds to proteins in the electron transport chain.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "FMN"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Iron\u2013sulfur clusters are part of several enzymes and play a role in the protein\'s function.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "IRON\u2013SULFUR CLUSTER"}, {"entity_type": "AMINO ACID", "description": "Glu246Lys mutation occurs in the FMN binding site which could potentially impact NADH binding.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "GLUTAMIC ACID (GLU)"}, {"entity_type": "GENOMIC VARIANT", "description": "Some identified variants are categorized as 1\u20133 by RUNES, which may be associated with disease causing mutations.<SEP>Variants were found through sequencing of DNA from peripheral blood cells.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "VARIANT"}, {"entity_type": "DIETARY PATTERN", "description": "The nuclear exome was enriched and sequenced to identify genetic variants.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "NUCLEAR EXOME"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The mean target nuclear exome coverage of 71.9\u00d7 indicates the thoroughness of sequencing.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MEAN TARGET NUCLEAR EXOME COVERAGE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Nuclear and mtDNA variants were found through enrichment for the nuclear exome.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MTDNA"}, {"entity_type": "AMINO ACID", "description": "The mutation Ser146Asn is caused by c.437G>A in the COQ2 gene.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "SERINE (SER)"}, {"entity_type": "GENE", "description": "COQ2 mutations have been reported to cause coenzyme Q10 deficiency (OMIM #609825).", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "COENZYME Q2 HOMOLOG, PRENYLTRANSFERASE"}, {"entity_type": "GENOMIC VARIANT", "description": "The mutation c.437G>A changes serine (Ser) to aspartic acid (Asp) at position 146.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "C.437G>A"}, {"entity_type": "AMINO ACID", "description": "Mutation Arg387X causes a premature stop codon following the change from arginine to a stop codon.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "ARGININE (ARG)"}, {"entity_type": "GENE", "description": "The variant found in MT-ND1 is novel and implies ~22% heteroplasmy in peripheral blood.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MT-ND1 GENE"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant was confirmed through subsequent capillary sequencing.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "3754C>A"}, {"entity_type": "DISEASE", "description": "Mutations in MT-ND1 can cause Leber optic atrophy (OMIM #535000).<SEP>Mutations in MT-ND1 can lead to Leber optic atrophy (OMIM #535000).", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "LEBER OPTIC ATROPHY"}, {"entity_type": "DISEASE", "description": "Mutations in MT-ND1 are linked to mitochondrial complex I deficiency (OMIM #252010).<SEP>Mutations in MT-ND1 can lead to mitochondrial complex I deficiency (OMIM #252010).", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MITOCHONDRIAL COMPLEX I DEFICIENCY"}, {"entity_type": "DISEASE", "description": "Mutations in MT-ND1 can also result in MELAS syndrome (OMIM #540000).", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MELAS SYNDROME"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The acid long protein functions as part of complex 1, which aids in electron transfer.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "COMPLEX 1 OF ELECTRON TRANSPORT CHAIN"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The process by which the electron is released from NADH to ubiquinone.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "ELECTRON TRANSPORT CHAIN"}, {"entity_type": "VITAMIN", "description": "Ubiquinone plays a role in electron transfer within complex 1.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "UBIQUINONE"}, {"entity_type": "GENOMIC VARIANT", "description": "Previously reported disease-causing mutations include c.437G>A changing serine to aspartic acid at position 146 in the COQ2 gene.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "C.437G>A (SER146ASN) MUTATION"}, {"entity_type": "GENOMIC VARIANT", "description": "A second variant changes arginine at amino acid position 387 to a premature stop codon through c.1159C>T.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "C.1159C>T (ARG387X) MUTATION"}, {"entity_type": "DISEASE", "description": "COQ2 mutations are associated with coenzyme Q10 deficiency (OMIM #609825).", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "COENZYME Q10 DEFICIENCY"}, {"entity_type": "QUANTITATIVE MEASURE", "description": "Indicators used in capillary sequencing for estimating variant frequencies.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "DIDEOXYNUCLEOTIDE PEAKS"}, {"entity_type": "GENETIC PHENOMENON", "description": "The presence of multiple variants within a single individual, as observed with the MT-ND1 gene variant.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "HETEROPLASMY"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "NADH is a key component in the electron transport chain.)", "entity_type": "UNKNOWN", "id": "FUNCTIONING"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "FMN binds to proteins like iron\u2013sulfur clusters for proper functioning.)", "entity_type": "UNKNOWN", "id": "BINDING"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "Iron\u2013sulfur clusters are part of several enzymes, including the protein mentioned.)", "entity_type": "UNKNOWN", "id": "PROTEIN FUNCTION"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "The mutation could impact NADH binding to proteins.)", "entity_type": "UNKNOWN", "id": "GLU246LYS MUTATION"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "Mutation changes serine (Ser) to aspartic acid (Asp) at position 146 in COQ2 gene.)", "entity_type": "UNKNOWN", "id": "MUTATION"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "Mutation causes arginine (Arg) change to a premature stop codon at amino acid position 387.)", "entity_type": "UNKNOWN", "id": "C.1159C>T"}, {"entity_type": "GENE", "description": "Results revealed significant genotype group differences across drinking variables, specifically AUDIT score, number of drinking days, and drinks per drinking day in the past 30 days.<SEP>The intronic region of the opioid receptor gene OPRM1 contained genomic variant rs1998220, which was related to social disinhibition.<SEP>rs1998220 in the intronic region of OPRM1 showed associations with social disinhibition.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544<SEP>chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "OPRM1"}, {"entity_type": "DRINKING VARIABLE", "description": "The AUDIT score was used to measure alcohol consumption among participants grouped by their OPRM1 genotype.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "AUDIT SCORE"}, {"entity_type": "DRINKING VARIABLE", "description": "Number of days when drinking occurred varied significantly across different OPRM1 genotypes.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "NUMBER OF DRINKING DAYS"}, {"entity_type": "BEHAVIORAL METRIC", "description": "The number of drinks consumed on a drinking day was measured for alcohol-dependent individuals.<SEP>The quantity of alcohol consumed on drinking days differed among participants based on their OPRM1 genotype.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544<SEP>chunk-7542fe4e6d1c727325a6be02a71f8e59", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 29}]", "id": "DRINKS PER DRINKING DAY"}, {"entity_type": "ENVIRONMENTAL FACTOR/", "description": "Dietary Pattern/>Analyses controlled for ALDH2 (rs671) and ADH1B (rs1229984) markers to validate main results.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "ALDH2"}, {"entity_type": "ENVIRONMENTAL FACTOR/", "description": "Dietary Pattern/>Analyses controlled for ADH1B marker to further validate the results related to OPRM1 genotype and alcohol effects.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "ADH1B"}, {"entity_type": "ENVIRONMENTAL FACTOR/", "description": "Dietary Pattern/>Medications (naltrexone and placebo) were tested for their effects on alcohol craving, sedation, or stimulation along with OPRM1 genotypes.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "MEDICATION"}, {"entity_type": "ENVIRONMENTAL FACTOR/", "description": "Dietary Pattern/>The combination of specific medication and OPRM1 genotype influenced the outcomes related to alcohol craving, sedation or stimulation.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "MEDICATION \u00d7 GENOTYPE INTERACTION"}, {"entity_type": "ENVIRONMENTAL FACTOR/", "description": "Dietary Pattern/>Blood Alcohol Concentration (BrAC) was controlled to examine the effect of alcohol craving, sedation or stimulation.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "BRAC"}, {"entity_type": "GENETIC FACTOR", "description": "ALDH2 genotype is a genetic factor that was found to have significant covariates in the analysis.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "ALDH2 GENOTYPE"}, {"entity_type": "PHYSIOLOGICAL MEASUREMENT", "description": "Baseline stimulation values were identified as significant covariates.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "BASELINE STIMULATION VALUES"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Smoking status, considered as an environmental factor, was not a significant moderator of the effects in any analysis.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "NICOTINE SMOKING STATUS"}, {"entity_type": "GENOTYPE", "description": "Asp40 carrier was one genotype group that showed significantly fewer drinks consumption compared to Asn40 homozygotes.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "ASP40 CARRIER"}, {"entity_type": "GENOTYPE", "description": "Asn40 homozygotes were another genotype group in the study.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "ASN40 HOMOZYGOTE"}, {"entity_type": "DRUG/SUBSTANCE", "description": "Nicotine was considered as one of the environmental factors that did not significantly moderate the effects of medication or genotype.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "NICOTINE"}, {"entity_type": "GENETIC FACTOR", "description": "When OPRM1 genotype was treated as a three-level variable, it did not significantly alter the previously reported pharmacogenetic results.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "OPRM1 GENOTYPE"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "The combined effect of medications, OPRM1 genotypes, and BrAC in influencing alcohol-induced subjective responses was analyzed.)<SEP>The complex interactions between medications, OPRM1 gene variants, and BrAC influenced evaluations related to alcohol-induced sedative responses.)<SEP>The complex interactions between medications, OPRM1 gene variants, and BrAC levels influenced evaluations related to alcohol cravings.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "MEDICATION \u00d7 GENOTYPE \u00d7 BRAC INTERACTION"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "The impact of OPRM1 genotype in conjunction with medication influenced alcohol-induced responses was assessed by considering current cigarette smoking status.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "CIGARETTE SMOKING STATUS"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "The influence of OPRM1 gene variations and naltrexone were studied to understand their mutual impact on alcohol effects.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "NALTREXONE"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836<SEP>chunk-62d8719735e6c6eaa899d9ea64143544", "description": "A control substance given to the comparison group in the study.<SEP>The role of the OPRM1 gene on alcohol responses under placebo conditions was analyzed during studies employing various medications.)", "entity_type": "DRUG", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 30}]", "id": "PLACEBO"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "The impact of the OPRM1 gene on alcohol craving was evaluated in relation to medication types and BrAC levels.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "ALCOHOL CRAVING"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "The OPRM1 gene\'s effect on alcohol-induced sedation was compared across different medication types and BrAC levels.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "ALCOHOL-INDUCED SEDATION"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "The OPRM1 gene\'s effect on alcohol-induced stimulant responses was studied considering different medication types and BrAC levels.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "ALCOHOL-INDUCED STIMULATION/"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "Stimulation/>The combined effects of various medications with OPRM1 gene variations on assessing alcohol-induced stimulant responses were explored.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "MEDICATION \u00d7 GENOTYPE INTERACTION/"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "Stimulation/>The complex interactions between medications, OPRM1 gene variants, and BrAC influenced evaluations related to alcohol-induced stimulant responses.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 28}]", "id": "MEDICATION \u00d7 GENOTYPE \u00d7 BRAC INTERACTION/"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "The Asn40 homozygote group, when administered the placebo, demonstrated unique responses to alcohol-induced stimulation.)", "entity_type": "UNKNOWN", "id": "GENE/"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "The Asn40 homozygote group, when administered the placebo, demonstrated unique responses to alcohol-induced stimulation.)", "entity_type": "UNKNOWN", "id": "ASN40 HOMOZYGOTES"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "Environmental Factors/>Medications\' combined effects with OPRM1 gene variants on alcohol-induced responses were considered when looking at external factors.)<SEP>Environmental Factors/>The combined effect of certain medications with OPRM1 gene variants played a key role in evaluating alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The combined effect of certain medications with OPRM1 gene variants significantly influenced evaluations of alcohol responses while taking into account external influences.)<SEP>Environmental Factors/>The combined impact of certain medications with OPRM1 gene variants played a key role in evaluating alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The combined influence of certain medications with OPRM1 gene variants played an important role in evaluating alcohol responses while taking into account environmental influences.)<SEP>Environmental Factors/>The combined influence of certain medications with OPRM1 gene variants significantly shaped evaluations of alcohol responses while taking into account environmental influences.)<SEP>Environmental Factors/>The combined influence of medications with OPRM1 gene variants shaped the analysis of alcohol responses when considering external factors.)", "entity_type": "UNKNOWN", "id": "MEDICATION \u00d7 GENOTYPE INTERACTION/"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "Environmental Factors/>The combined effect of BrAC levels with different medications and OPRM1 gene variants influenced assessments related to alcohol responses when examining environmental influences.)<SEP>Environmental Factors/>The combined effect of differing BrAC levels with specific medications and OPRM1 gene variants significantly influenced evaluations of alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The combined effect of differing BrAC levels with specific medications and OPRM1 gene variants was crucial for assessing alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The combined effect of varying BrAC levels with specific medications and OPRM1 gene variants significantly influenced evaluations of alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The combined influence of varying BrAC levels with specific medications and OPRM1 gene variants significantly influenced evaluations of alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The combined influence of varying BrAC levels with specific medications and OPRM1 gene variants was pivotal in assessing alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The intricate relationship among BrAC levels, various medications, and OPRM1 gene variants influenced evaluations related to alcohol responses.)", "entity_type": "UNKNOWN", "id": "BRAC \u00d7 MEDICATION \u00d7 GENOTYPE INTERACTION/"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "Stimulation/>The Asn40 homozygote group, when receiving the placebo, showed distinct responses related to alcohol-induced stimulation compared to others.)<SEP>Stimulation/>The effect of being an Asn40 homozygote with exposure to the placebo led to differentiated responses in alcohol-induced stimulation when compared to others.)<SEP>Stimulation/>The effect of being an Asn40 homozygote, when exposed to a placebo condition, led to differentiated responses in alcohol-induced stimulation relative to others.)<SEP>Stimulation/>The effect of being an Asn40 homozygote, when exposed to a placebo condition, led to distinctive responses in alcohol-induced stimulation relative to others.)<SEP>Stimulation/>The influence of being an Asn40 homozygote, when exposed to a placebo condition, led to distinctive responses in alcohol-induced stimulation relative to others.)<SEP>Stimulation/>The influence of being an Asn40 homozygote, when in the placebo condition, led to distinctive alcohol-induced stimulation responses relative to others.)", "entity_type": "UNKNOWN", "id": "ASN40 HOMOZYGOTES + PLACEBO GROUP/"}, {"source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "description": "There was a significant main effect of ALDH2 genotype on latency to first drink, with Asn40 homozygotes having shorter latencies.)", "entity_type": "UNKNOWN", "id": "LATENCY TO FIRST DRINK"}, {"source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "description": "The analysis found no significant medication \u00d7 genotype interaction regarding the effects on alcohol self-administration or latency to first drink consumption.)", "entity_type": "UNKNOWN", "id": "GENETIC FACTOR"}, {"source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "description": "The analysis found no significant medication \u00d7 genotype interaction regarding the effects on alcohol self-administration or latency to first drink consumption.)", "entity_type": "UNKNOWN", "id": "MEDICATION EFFECTS"}, {"source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "description": "Treatments of OPRM1 genotype as a three-level variable did not significantly affect the reported pharmacogenetic outcomes related to alcohol self-administration and latency to drink.)", "entity_type": "UNKNOWN", "id": "PHARMACOGENETIC RESULTS"}, {"entity_type": "GENE", "description": "The ALDH2 rs671 genotypes are variants present in the human genome that influence alcohol metabolism.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 21}]", "id": "ALDH2 RS671 GENOTYPES"}, {"entity_type": "DIETARY PATTERN", "description": "Daily drinks refer to the amount of alcohol consumed per day, classified based on ALDH2 rs671 genotypes.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 21}]", "id": "DAILY DRINKS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Cortisol levels are a measure of the body\'s stress hormone that varies according to ALDH2 rs671 genotypes and sex.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 21}]", "id": "CORTISOL LEVELS"}, {"entity_type": "DISEASE", "description": "Hypertension is a condition that affects blood pressure and can influence cortisol levels when considered as a confounding factor.<SEP>Hypertension, or high blood pressure, can lead to serious health issues if not managed.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a<SEP>chunk-41c7131af28e2a745ef6b88b8601860f", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 20}]", "id": "HYPERTENSION"}, {"entity_type": "SOCIOECONOMIC FACTOR", "description": "Education status can influence lifestyle factors like physical activity and smoking, impacting cortisol levels in both men and women.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 19}]", "id": "EDUCATION STATUS"}, {"entity_type": "GENOMIC VARIANT", "description": "The rs671 polymorphism is a single nucleotide variation found within the ALDH2 gene which influences alcohol metabolism.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 21}]", "id": "RS671 POLYMORPHISM"}, {"entity_type": "STATISTICAL METHOD", "description": "Mendelian randomization is a technique used to infer causality between an exposure and an outcome by using genetic variants as instrumental variables, as applied in Table 4 for alcohol consumption and cortisol levels.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "MR (MENDELIAN RANDOMIZATION)"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "ALDH2 genotypes are associated with age differences in both males and females.)", "entity_type": "UNKNOWN", "id": "GENOTYPES"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "Genotypes associated with the rs671 variant may result in age differences among genotypic groups.)", "entity_type": "UNKNOWN", "id": "RS671 GENOTYPES"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "The MR technique was used to assess potential causal links between genetically predicted alcohol intake levels and changes in blood cortisol levels.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION BETWEEN ALCOHOL CONSUMPTION AND BLOOD CORTISOL LEVEL"}, {"entity_type": "DISEASE", "description": "Hyperuricemia, characterized by elevated uric acid levels in the blood.<SEP>The prevalence of Hyperuricemia (HUA) was 14.71% in the study population, with 151 HUA cases and 1055 controls.", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd<SEP>chunk-7091ed039203b69ad83d281b22c89579", "id": "HUA"}, {"entity_type": "SNP<|THE DISTRIBUTION OF SLC22A12 RS893006 WAS NOT SIGNIFICANTLY ASSOCIATED WITH HYPERURICEMIA (HUA) IN BOTH MALES AND FEMALES.)", "description": "The G allele of SLC22A12 rs893006 is a genetic variant that contributes to the generation of a genetic risk score for sUA levels.<SEP>14", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd<SEP>chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "id": "SLC22A12 RS893006"}, {"entity_type": "GENETIC VARIANT", "description": "The C allele of SLC2A9 rs3733951 is a genetic variant that affects sUA levels.", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "id": "SLC2A9 RS3733951"}, {"entity_type": "GENETIC VARIANT", "description": "The T allele of ABCG2 rs2231142 influences the risk of Hyperuricemia (HUA) with or without cardiovascular risk factors.", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "id": "ABCG2 RS2231142"}, {"entity_type": "METABOLIC PATHWAY/PHYSIOLOGICAL PROCESS", "description": "Serum uric acid (sUA) level is affected by genetic variants and can be influenced by cardiovascular risk factors.<SEP>Uric acid production regulation in the body, related to cardiovascular disease risk.", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00<SEP>chunk-7091ed039203b69ad83d281b22c89579", "id": "SUA LEVEL"}, {"entity_type": "ENVIRONMENTAL FACTOR/LIFESTYLE FACTOR", "description": "Factors such as obesity, hypertension, high triglyceride levels, low HDL cholesterol levels, etc., are cardiovascular risk factors that influence the association between genetic variants and HUA risk/sUA levels.", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "id": "CARDIOVASCULAR RISK FACTOR"}, {"entity_type": "DISEASE", "description": "Cardiovascular disease risk factor associated with uric acid levels.", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "id": "SUA"}, {"entity_type": "GENOMIC VARIANT", "description": "A specific genomic variant identified with the rs2231142 tag.", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "id": "RS2231142 T ALLELE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "A blood test that measures the percentage of red blood cells in whole blood.", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "id": "HEMATOCRIT"}, {"entity_type": "METABOLIC PATHWAY", "description": "The process by which the body produces cholesterol, essential for cell membranes and hormone synthesis.", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "clusters": "[{level: 0, cluster: 2}]", "id": "CHOLESTEROL"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Individuals who carry the T variant at the rs2231142 locus, impacting uric acid metabolism and cardiovascular disease risk.", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "clusters": "[{level: 0, cluster: 5}]", "id": "RS2231142 T ALLELE CARRIERS"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd<SEP>chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "Increased triglyceride levels can lead to lipemia retinalis, abdominal pain and gastrointestinal hemorrhage.<SEP>Increased triglyceride levels were observed in the HUA group.)", "entity_type": "NUTRIENT", "id": "TG"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "Participants with Hyperuricemia showed higher LDL levels than those without.)", "entity_type": "UNKNOWN", "id": "LDL"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "A higher percentage of hypertensive patients were identified among those with Hyperuricemia (HUA).)", "entity_type": "UNKNOWN", "id": "HYPERTENSIVE PATIENTS"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "The usage ratio of diuretics was not significantly different between the HUA cases and controls.)", "entity_type": "UNKNOWN", "id": "DIURETICS"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "No significant difference in Fasting Blood Glucose (FBG) levels was observed between HUA cases and controls.)", "entity_type": "UNKNOWN", "id": "FBG"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "The study found that HUA cases showed a significant association with the risk alleles of ABC-related genes, as well as an increased sUA level and HUA risk.)", "entity_type": "UNKNOWN", "id": "ABC"}, {"source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "description": "Genetic Risk Score", "entity_type": "UNKNOWN", "id": "GENETIC VARIANT/"}, {"source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "description": "Genetic Risk Score", "entity_type": "UNKNOWN", "id": "SLC2A9 RS3733951/ABCG2 RS2231142/SLC22A12 RS893006"}, {"source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "description": "Metabolic Pathway/", "entity_type": "UNKNOWN", "id": "SUA LEVEL/SLC2A9 RS3733951/ABCG2 RS2231142/"}, {"source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "description": "Metabolic Pathway/", "entity_type": "UNKNOWN", "id": "SLC22A12 RS893006"}, {"source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "description": "Environmental Factor/Lifestyle Factor/<SEP>Environmental Factor/", "entity_type": "UNKNOWN", "id": "ENVIRONMENTAL FACTOR/CARDIOVASCULAR RISK FACTOR"}, {"source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "description": "sUA level/", "entity_type": "UNKNOWN", "id": "ENVIRONMENTAL FACTOR/CARDIOVASCULAR RISK FACTOR/"}, {"source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "description": "Physiological Process", "entity_type": "UNKNOWN", "id": "METABOLIC PATHWAY/"}, {"source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "description": "Physiological Process", "entity_type": "UNKNOWN", "id": "SUA LEVEL/RS13206634"}, {"source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "description": "Physiological Process/", "entity_type": "UNKNOWN", "id": "SUA LEVEL/RS7509837"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5<SEP>chunk-7091ed039203b69ad83d281b22c89579", "description": "Carriers of the rs2231142 T allele are associated with cardiovascular risk factors, including sUA levels and HUA risk.)<SEP>GHI increases the risk of developing cardiovascular diseases later in life, partly due to insulin resistance and other metabolic abnormalities.", "entity_type": "LINKED TO GHI", "id": "CARDIOVASCULAR RISK FACTORS"}, {"entity_type": "GENE", "description": "The LRP2 gene is associated with kidney function and has been identified in trans-ethnic meta-analysis.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "LRP2"}, {"entity_type": "GENE", "description": "NFATC1 plays a central role in inducible gene transcription during immune response and is a downstream target of the transplant immunosuppression drug cyclosporine A.<SEP>The NFATC1 gene was discovered in a recently published meta-analysis of European ancestry GWASs.<SEP>The mRNA expression of Nfatc1 was significantly reduced during high-salt exposure and tended to decrease in CKD models.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391<SEP>chunk-45a965c3c3013c70d0fa0f1680283083<SEP>chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "NFATC1"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "eGFR (estimated glomerular filtration rate) is an important indicator for kidney function and has been studied across ancestry groups.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "EGFR"}, {"entity_type": "DISEASE", "description": "End-stage renal disease (ESRD) is a condition defined by eGFR < 60 mL/min/1.73 m2, or the occurrence of ESRD.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "ESRD"}, {"entity_type": "DISEASE", "description": "T1D (Type 1 Diabetes) is associated with the region containing rs181206 and changes in expression of IRF1 and STAT1.<SEP>The identified GWAS SNPs were associated with T1D, highlighting their role in disease susceptibility.<SEP>Type 1 Diabetes (T1D), another condition associated with diabetic nephropathy, was also considered in studies related to kidney function and genetic variants.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391<SEP>chunk-5b68770e1b446d6cba0d4e1ecabf497b<SEP>chunk-1b40d071abac0ba432a9fb0096796f45", "id": "T1D"}, {"entity_type": "GENE", "description": "The PDILT-UMOD gene has been reported as driving association signals for CKD and shows homogeneous effects on eGFR across diverse populations.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "PDILT-UMOD"}, {"entity_type": "GENE", "description": "The HFE gene, involved in iron metabolism and related to kidney function.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "HFE"}, {"entity_type": "GENE", "description": "The SLC34A1 gene is associated with phosphate reabsorption in the kidney.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "SLC34A1"}, {"entity_type": "GENE", "description": "DCDC5 and MPPED2 are interacting proteins involved in the regulation of cell motility and adhesion.<SEP>The DCDC5-MPPED2 gene is implicated in retinal development and function, but its role in kidney function requires further investigation.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391<SEP>chunk-45a965c3c3013c70d0fa0f1680283083", "id": "DCDC5-MPPED2"}, {"entity_type": "NUTRIENT", "description": "Nutrient X plays a role in various physiological processes that influence kidney health.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "NUTRIENT X"}, {"entity_type": "DIETARY PATTERN", "description": "Dietary pattern Y has been associated with improved kidney function or increased risk, depending on its components.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "DIETARY PATTERN Y"}, {"entity_type": "GENE", "description": "RGS14 encodes a member of the regulator of G protein signaling family, which modulates downstream effects of G\u03b1 subunits and has unknown function in kidneys.<SEP>The mRNA expression of Rgs14 was decreased by about 75% during high-salt exposure and approached statistical significance in the CKD model.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083<SEP>chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "RGS14"}, {"entity_type": "REGULATORY CHROMATIN STATE", "description": "The lead SNPs at certain loci overlap with annotated regulatory chromatin states in specific cells.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "DHSS"}, {"entity_type": "TRANSCRIPTION FACTOR BINDING SITES", "description": "Significant effects on posterior probability of driving association signals are observed for variants near ChIP-seq binding sites, notably for HDAC8", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "CHIP-SEQ BINDING SITES"}, {"entity_type": "HISTONE DEACETYLASE", "description": "HDACs play a role in kidney function and development.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "HDAC8"}, {"entity_type": "DRUG", "description": "The potential use of histone deacetylase inhibitors as treatments for kidney disease", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "HISTONE DEACETYLASE INHIBITORS"}, {"entity_type": "GENE", "description": "PIP5K1B is a phosphatidylinositol 4,5-bisphosphate (PtdIns(4,5)P2) kinase that catalyzes the production of PtdIns(4,5)P2.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "PIP5K1B"}, {"entity_type": "REGULATORY CHROMATIN STATES", "description": "DHSs refer to regions of deoxyribonucleoprotein (DNAP) in chromatin that are accessible for transcription factors binding.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "DHSS IN RPTECS AND HRCES"}, {"entity_type": "TRANSCRIPTION FACTOR BINDING SITES", "description": "These sites represent the locations on the DNA where specific transcription factors bind and regulate gene expression.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "CHIP-SEQ BINDING SITES FOR HDAC8"}, {"entity_type": "GENE", "description": "loco mutants with locod06164/+ exhibit improved survival on NaCl-supplemented food.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "LOCOD06164"}, {"entity_type": "GENE", "description": "loco mutants with locoEY-P283/+ show enhanced survival on NaCl-supplemented food compared to controls.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "LOCOEY-P283/"}, {"entity_type": "DISEASE", "description": "There is minimal mortality for all genotypes on non-NaCl-supplemented food during the tested time period, indicating similar viability.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "Y1W1"}, {"entity_type": "DISEASE", "description": "This strain is salt-sensitive and prone to glomerulosclerosis when subjected to high-salt diets.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "129S6 MOUSE STRAIN"}, {"entity_type": "ENVIRONMENTAL FACTOR/", "description": "High salt intake leads to a decrease in both Nfatc1 and Rgs14 mRNA expression.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "HIGH-SALT DIET"}, {"entity_type": "DISEASE/", "description": "The expression of Nfatc1 and Rgs14 were altered during CKD development, with Nfatc1 showing significant reduction.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "CKD (CHRONIC KIDNEY DISEASE)"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "The eGFR decreasing allele is associated with increased risk of CKD in individuals.)", "entity_type": "UNKNOWN", "id": "CKD"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "Type 2 Diabetes impacts kidney function through mechanisms such as diabetic nephropathy, influencing eGFR.)", "entity_type": "UNKNOWN", "id": "KIDNEY FUNCTION"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "The SNP Z is significantly associated with a decrease in eGFR and an increased risk of kidney disease.)", "entity_type": "UNKNOWN", "id": "SNP Z"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "Variants in the HFE gene affect iron metabolism, which can impact kidney health through oxidative stress and inflammation.)", "entity_type": "UNKNOWN", "id": "HFE GENE"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "Variants in the HFE gene affect iron metabolism, which can impact kidney health through oxidative stress and inflammation.)", "entity_type": "UNKNOWN", "id": "IRON METABOLISM"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "Mutations or variations in SLC34A1 may alter phosphate reabsorption, impacting kidney function.)", "entity_type": "UNKNOWN", "id": "PHOSPHATE REABSORPTION"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "While primarily associated with vision, changes in DCDC5-MPPED2 may indirectly influence kidney health via shared signaling pathways or genetic networks.)", "entity_type": "UNKNOWN", "id": "RETINAL DEVELOPMENT"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "The dietary pattern Y may provide protective benefits against kidney disease through specific nutrients and antioxidants.)", "entity_type": "UNKNOWN", "id": "RISK OR PROTECTION"}, {"source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "description": "lead SNPs at certain loci contribute significantly to the posterior probability of driving association signals.)", "entity_type": "UNKNOWN", "id": "LEAD SNPS"}, {"source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "description": "lead SNPs at certain loci contribute significantly to the posterior probability of driving association signals.)", "entity_type": "UNKNOWN", "id": "POSTERIOR PROBABILITY"}, {"source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "description": "Ancestral orthologs for both genes are present in Drosophila melanogaster)", "entity_type": "UNKNOWN", "id": "NFATC1 GENOMIC VARIANT"}, {"source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "description": "Ancestral orthologs for both genes are present in Drosophila melanogaster)", "entity_type": "UNKNOWN", "id": "RGS14 GENOMIC VARIANT"}, {"source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "description": "Variants mapping to non-coding sequence at these loci account for more than 99.4% of the probability of driving eGFR association.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION SIGNALS"}, {"source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "description": "Variants mapping to non-coding sequence at these loci account for more than 99.4% of the probability of driving eGFR association.)", "entity_type": "UNKNOWN", "id": "KIDNEY DHSS"}, {"source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "description": "There are significant effects on posterior probability for variants near HDAC8 binding sites)", "entity_type": "UNKNOWN", "id": "SIGNIFICANT EFFECTS"}, {"source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "description": "The potential use of histone deacetylase inhibitors as treatments for kidney disease)", "entity_type": "UNKNOWN", "id": "POTENTIAL TREATMENT"}, {"source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "description": "The lead SNPs at NFATC1 locus overlaps with DHSs in renal proximal tubular epithelial cells (RPTECs) and renal cortical epithelial cells (HRCEs))", "entity_type": "UNKNOWN", "id": "REGULATORY CHROMATIN STATE IN RPTECS&HRCES"}, {"source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "description": "Reduction in expression of the RGS14 ortholog loco results in longer lifespan and stress resistance)", "entity_type": "UNKNOWN", "id": "LONGEVITY & STRESS RESISTANCE"}, {"entity_type": "DISEASE", "description": "In this study, 975 culture-confirmed Campylobacter infections were reported, with some infected by quinolone-resistant isolates.", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "id": "CAMPYLOBACTER"}, {"entity_type": "GENE", "description": "study identified 12 quinolone-resistant cases that were domestically acquired.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "QUINOLONE-RESISTANT CASES"}, {"entity_type": "DISEASE", "description": "An unmatched subanalysis on domestically acquired infections (quinolone-resistant versus quinolone-sensitive) was performed.<SEP>Infections treated with fluoroquinolones before specimen collection were excluded.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "clusters": "[{level: 0, cluster: 10}, {level: 1, cluster: 34}]", "id": "DOMESTICALLY ACQUIRED INFECTIONS"}, {"entity_type": "DRUG", "description": "We found information on antimicrobial drugs received by 122 of 126 patients.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "ANTIMICROBIAL DRUGS"}, {"entity_type": "DISEASE", "description": "Forty patients (32.8%) were treated with antimicrobial agents for their campylobacteriosis; of these, 33 patients (27%) received a fluoroquinolone, 6 patients (4.9%) received a macrolide, and 1 patient (1%) received both a fluoroquinolone and a macrolide for the C. jejuni infection.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "CAMPYLOBACTERIOSIS"}, {"entity_type": "DIETARY PATTERN", "description": "The human isolates were included in a database and compared with 180 Campylobacter isolates obtained from retail food products (chicken [n = 139], turkey [n = 39], and pork [n = 2]).", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "RETAIL FOOD PRODUCTS"}, {"entity_type": "DIETARY PATTERN", "description": "Of the human isolates, most (63%) were from Danish-bred food animals; the remaining were from imported food from France, Italy, or the United Kingdom.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "BROILER CHICKENS"}, {"entity_type": "GENOMIC VARIANT", "description": "A subanalysis of resistance status by origin of 139 retail chicken products (domestic versus imported) showed that 7 (8.0%) of 87 C. jejuni isolates and three (60%) of five C. coli isolates from domestic raised chicken products were resistant to nalidixic acid.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "FLUOROQUINOLONE-RESISTANT ISOLATES"}, {"entity_type": "PHENOTYPE", "description": "Of isolates from imported chicken products, 5 (14.7%) (3 isolates from France and 2 isolates from the United Kingdom) of 34 C. jejuni isolates and 1 (10%) (from France) of 10 C. coli isolates were resistant to nalidixic acid.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "NALIDIXIC ACID RESISTANCE"}, {"entity_type": "PHENOTYPE", "description": "Of the isolates from chicken products, two isolates (one C. jejuni and one C. coli) from domestic chicken products were resistant to both antimicrobial agents. A subanalysis of resistance status by origin showed that 7 (8.0%) of 87 C. jejuni isolates and three (60%) of five C. coli isolates from domestic raised chicken products were resistant to nalidixic acid.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "ERYTHROMYCIN RESISTANCE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "We found information on antimicrobial drugs received by 122 of 126 patients.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "ANTIMICROBIAL DRUG TREATMENT"}, {"entity_type": "SPECIES", "description": "A type of bacteria known for causing campylobacteriosis.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "CAMPYLOBACTER JEJUNI"}, {"entity_type": "SPECIES", "description": "Another type of bacteria related to Campylobacter jejuni that can cause similar infections.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "CAMPYLOBACTER COLI"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "Foreign travel was associated with a higher proportion of quinolone-resistant Campylobacter infections compared to domestic infections.)", "entity_type": "UNKNOWN", "id": "QUINOLONE RESISTANCE"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "Foreign travel was associated with a higher proportion of quinolone-resistant Campylobacter infections compared to domestic infections.)", "entity_type": "UNKNOWN", "id": "TRAVEL"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "Of the three erythromycin-resistant isolates, two were also quinolone-resistant and originated from travelers returning to Denmark from Spain and Portugal.)", "entity_type": "UNKNOWN", "id": "C. COLI"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "Travel to certain regions like southern Europe and Southeast Asia was significantly associated with high prevalence of quinolone-resistant Campylobacter infections.)", "entity_type": "UNKNOWN", "id": "DESTINATION"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "The mean duration of illness was longer for patients infected by quinolone-resistant C. jejuni compared to those infected by quinolone-sensitive strains.)", "entity_type": "UNKNOWN", "id": "C. JEJUNI"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "The mean duration of illness was longer for patients infected by quinolone-resistant C. jejuni compared to those infected by quinolone-sensitive strains.)", "entity_type": "UNKNOWN", "id": "DURATION OF ILLNESS"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "Foreign travel, especially to specific destinations like Spain and Portugal, significantly increased the risk of acquiring a quinolone-resistant Campylobacter infection.)", "entity_type": "UNKNOWN", "id": "FOREIGN TRAVEL"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "C. jejuni infections did not differ in severity when comparing resistant to sensitive strains, but there was a difference in the mean duration of illness.)", "entity_type": "UNKNOWN", "id": "QUINOLONE SENSITIVITY/RESISTANCE"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "Certain destinations like southern Europe and Southeast Asia were associated with higher proportions of quinolone-resistant Campylobacter infections than others.)", "entity_type": "UNKNOWN", "id": "TRAVEL DESTINATIONS"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "Swimming was identified as an independent risk factor for acquiring a quinolone-resistant C. jejuni infection, though the exact mechanism is unclear.)", "entity_type": "UNKNOWN", "id": "SWIMMING"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "Eating fresh poultry other than chicken and turkey was associated with an increased risk of quinolone-resistant infections.)", "entity_type": "UNKNOWN", "id": "FRESH POULTRY"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "The RiboGroups of C. coli isolates play a role in identifying the different susceptibilities to quinolones among the strains.)", "entity_type": "UNKNOWN", "id": "C. COLI STRAINS"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "The RiboGroups of C. coli isolates play a role in identifying the different susceptibilities to quinolones among the strains.)", "entity_type": "UNKNOWN", "id": "QUINOLONE SUSCEPTIBILITY"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "The serotypes of C. jejuni isolates can influence their sensitivity patterns against quinolones.)", "entity_type": "UNKNOWN", "id": "C. JEJUNI STRAINS"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "A patient was found to have two C. jejuni strains with identical serotype and RiboGroup, but different susceptibility patterns towards quinolones due to a Thr-86 to Ala-86 mutation in gyrA.)", "entity_type": "UNKNOWN", "id": "C. JEJUNI STRAIN CO-INFECTION"}, {"source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "description": "Quinolone-resistant isolates showed decreased susceptibility to fluoroquinolones compared to primary model, but sample size was lower resulting in increased confidence intervals.)", "entity_type": "UNKNOWN", "id": "QUINOLONE-RESISTANT ISOLATES"}, {"source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "description": "The origin of food isolates was traced back to various sources including Danish-bred food animals, imported foods from France, Italy, and the United Kingdom.)", "entity_type": "UNKNOWN", "id": "FOOD ISOLATES"}, {"source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "description": "The origin of food isolates was traced back to various sources including Danish-bred food animals, imported foods from France, Italy, and the United Kingdom.)", "entity_type": "UNKNOWN", "id": "ORIGIN"}, {"source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "description": "Patients with domestically acquired campylobacteriosis were more likely to receive antimicrobial treatment compared to those with travel-related infections.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 10}, {level: 1, cluster: 34}]", "id": "TREATMENT"}, {"source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "description": "Human isolates of Campylobacter bacteria were compared with isolates obtained from retail food products (chicken, turkey, pork), chicken fecal samples, and broiler chickens.)", "entity_type": "UNKNOWN", "id": "RETAIL FOOD PRODUCTS AND BROILER CHICKENS"}, {"source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "description": "Human isolates of Campylobacter bacteria were compared with isolates obtained from retail food products (chicken, turkey, pork), chicken fecal samples, and broiler chickens.)", "entity_type": "UNKNOWN", "id": "SOURCE"}, {"entity_type": "GENOMIC VARIANT", "description": "Minor allele of rs9534511 was associated with an increase in social disinhibition score by 0.32 points (95% CI: 0.14-0.50, P < 0.001).<SEP>The Gencall score of less than 95% for rs9534511 on HTR2A was indicative of a potential association with social disinhibition.", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed<SEP>chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS9534511"}, {"entity_type": "GENOMIC VARIANT", "description": "Minor allele of rs4938056 was associated with an increase in social disinhibition score.<SEP>The genomic variant rs4938056 on HTR3B had a Gencall score below 95%, which was related to social disinhibition risk.", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed<SEP>chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS4938056"}, {"entity_type": "GENOMIC VARIANT", "description": "Minor allele of rs1998220 was associated with an increase in social disinhibition score by 0.19 points (95% CI: 0.06-0.32, P = 0.003).<SEP>The intronic region of the opioid receptor gene OPRM1 contained the genomic variant rs1998220, which showed associations with social disinhibition.", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed<SEP>chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS1998220"}, {"entity_type": "GENOMIC VARIANT", "description": "Minor allele of rs1003921 was associated with a decrease (negative effect) in social disinhibition score by -0.64 points (95% CI: -1.13 to -0.15, P = 0.011).<SEP>Residing in an intronic region of KCNC1, the genomic variant rs1003921 was related to social disinhibition risk.", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed<SEP>chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS1003921"}, {"entity_type": "GENOMIC VARIANT", "description": "Minor allele of rs16116 was associated with a decrease (negative effect) in social disinhibition score by -0.29 points (95% CI: -0.48 to -0.09, P = 0.004).<SEP>The intergenic region downstream of NPY contained the genomic variant rs16116 and showed an association with increased social disinhibition scores.", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed<SEP>chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS16116"}, {"entity_type": "SNP", "description": "Minor allele of rs16870286 was associated with a protective effect against social disinhibition.<SEP>Minor allele of rs16870286 was associated with a protective effect.<SEP>The genomic variant rs16870286 was part of LINC00518, which codes for an uncharacterized protein and showed associations with social disinhibition.", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed<SEP>chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS16870286"}, {"entity_type": "GENE", "description": "SNP rs9534511 on the HTR2A gene pathway was significant in the final multivariable model associated with social disinhibition.<SEP>rs9534511 on HTR2A showed significant association with social disinhibition.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "HTR2A"}, {"entity_type": "GENE", "description": "Genomic variant rs4938056 on HTR3B was related to increased risk of social disinhibition.<SEP>Genomic variant rs4938056 on the HTR3B gene pathway showed a potential association with increased risk of social disinhibition.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "HTR3B"}, {"entity_type": "GENE", "description": "Intronic genomic variant rs1003921 on KCNC1 gene showed associations with increased risk of social disinhibition.<SEP>The genomic variant rs1003921 on KCNC1 was associated with increased risk of social disinhibition.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "KCNC1"}, {"entity_type": "GENE", "description": "The intergenic region downstream of NPY contained the genomic variant rs16116, which was associated with social disinhibition scores.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "NPY"}, {"entity_type": "GENE", "description": "The genomic variant rs16870286 was part of LINC00518 gene, coding for an uncharacterized protein, which showed associations with increased risk of social disinhibition.<SEP>The genomic variant rs16870286 was part of LINC00518 gene, coding for an uncharacterized protein, which showed associations with social disinhibition.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "LINC00518"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs263781 was associated with social disinhibition risk.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS263781"}, {"entity_type": "GENOMIC VARIANT", "description": "Genomic variant rs12646596 showed associations with increased social disinhibition scores.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS12646596"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs3807492 was related to risk of social disinhibition.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS3807492"}, {"entity_type": "PSYCHOSOCIAL RISK FACTOR", "description": "TAS score, a measure of psychosocial risk factor, was positively associated with increased social disinhibition scores.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "TAS"}, {"entity_type": "PSYCHOSOCIAL RISK FACTOR", "description": "DAS score, another psychosocial risk factor, was positively correlated with social disinhibition.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "DAS"}, {"entity_type": "SOCIAL DISINHIBITION DETERMINANT", "description": "Participants whose parents had completed high school or higher showed no statistically significant difference in social disinhibition scores compared to those with less education.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "PARENTAL EDUCATION LEVEL"}, {"entity_type": "SOCIAL DISINHIBITION DETERMINANT", "description": "The overall linguistic acculturation score was found to be positively associated with increased risk of social disinhibition, leading to a higher score increase.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "LINGUISTIC ACCULTURATION"}, {"entity_type": "NON-CODING RNA INTERACTIONS", "description": "rs16116 in the intergenic region downstream of NPY showed associations with increased social disinhibition scores.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "NPY GENE INTERGENIC REGION"}, {"entity_type": "PRINCIPAL COMPONENT ANALYSIS COMPONENT", "description": "The top eigenvalue from the principal components analysis indicated significant variation in participants related to social disinhibition.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "EIGENVALUE 1"}, {"entity_type": "PRINCIPAL COMPONENT ANALYSIS COMPONENT", "description": "The second largest eigenvalue revealed substantial contribution to differences in social disinhibition.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "EIGENVALUE 2"}, {"entity_type": "PRINCIPAL COMPONENT ANALYSIS COMPONENT", "description": "The third largest eigenvalue showed further variation related to the social disinhibition outcome.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "EIGENVALUE 3"}, {"entity_type": "DISEASE", "description": "Social disinhibition score increased by 0.20 points (95% CI: 0.17-0.23) and 0.45 points (95% CI: 0.40-0.51), respectively.", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "id": "SOCIAL DISINHIBITION SCORE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "These were not significantly associated with social disinhibition score.", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "id": "GENDER AND PARENTAL EDUCATION"}, {"entity_type": "GENOMIC VARIANTS", "description": "These SNPs had positive associations with the increased risk of social disinhibition.", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "id": "SNPS (RS1998220, RS9534511, AND RS4938056)"}, {"source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "description": "A 1-point increase in linguistic acculturation score leads to a 0.30 point increase in social disinhibition scores.)", "entity_type": "UNKNOWN", "id": "SOCIAL DISINHIBITION"}, {"source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "description": "PC3 accounted for additional variance related to social disinhibition among study participants.)", "entity_type": "UNKNOWN", "id": "PRINCIPAL COMPONENT 3"}, {"source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "description": "Three SNPs\' minor alleles led to an increase in the social disinhibition score, while three others had protective effects against it.)", "entity_type": "UNKNOWN", "id": "INCREASED SOCIAL DISINHIBITION SCORE"}, {"source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "description": "Negative Effects on Social Disinhibition", "entity_type": "UNKNOWN", "id": "DECREASED SOCIAL DISINHIBITION SCORE/"}, {"source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "description": "Negative Effects on Social Disinhibition", "entity_type": "UNKNOWN", "id": "RS16116 & RS1003921"}, {"entity_type": "GENE", "description": "The CNR1 gene is based on studies by Zhang et al and National Center for Biotechnology Information.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "clusters": "[{level: 0, cluster: 7}, {level: 1, cluster: 26}]", "id": "CNR1"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs2023239, found upstream of exon 3 of the CNR1 gene, has been described as an alternative promoter.<SEP>The genotypic variant rs2023239 was examined in relation to the effects of alcohol.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e<SEP>chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "RS2023239"}, {"entity_type": "SAMPLE/", "description": "Disease<|Samples from post-mortem human prefrontal cortex (PFC) were used in association studies.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "POSTMORTEM HUMAN PFC"}, {"entity_type": "MOLECULAR INTERACTION/", "description": "Disease<|CNR1 receptors modulate drug-seeking behavior, which can lead to conditions like addiction.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "CNR1 RECEPTORS"}, {"entity_type": "ENVIRONMENTAL FACTOR/", "description": "Disease<|Exposure to alcohol cues activates neurocircuitry related to drug-seeking behavior in individuals with certain genetic variations.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "ALCOHOL CUE"}, {"entity_type": "ASSESSMENT TOOL/", "description": "Disease<|The Alcohol Use Disorder Identification Test (AUDIT) measures severity of alcohol use disorders.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "AUDIT"}, {"entity_type": "BEHAVIOR/", "description": "Disease<|Anxiety can influence behavior patterns and contribute to other health conditions.<SEP>Risk Factor/", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "ANXIETY"}, {"entity_type": "BEHAVIORS/", "description": "Risk Factor<|Tobacco use is a risk factor for many diseases, including those related to alcohol consumption.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "TOBACCO USE"}, {"entity_type": "BEHAVIOR/", "description": "Disease<|Certain genotypes may influence how individuals perceive rewarding effects of alcohol, potentially affecting their behavior and health.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "ALCOHOL REWARDING EFFECTS"}, {"entity_type": "ASSESSMENT TOOL/", "description": "Disease<|The Profile of Mood States (POMS) is used to assess emotional states like reward and positive affect.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "PROFILE OF MOOD STATES"}, {"entity_type": "CONTROL MEASURE/", "description": "Risk Factor<|Using placebo beverages helps in determining the effects without direct influence from substances.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "PLACEBO BEVERAGE"}, {"entity_type": "MOLECULAR INTERACTION/", "description": "CNR1 Gene<|The rs2023239 genomic variant acts as an alternative promoter for the CNR1 gene, influencing its expression.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "ALTERNATIVE PROMOTER"}, {"entity_type": "MOLECULAR INTERACTION/", "description": "CB1 Receptor<|Cannabinoid receptors are involved in various biological functions, and CB1 is a specific type of cannabinoid receptor.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "CANNABINOID RECEPTOR 1/"}, {"entity_type": "ENVIRONMENTAL FACTOR/", "description": "Disease/Exposure to alcohol cues can activate neurocircuitry related to drug-seeking behavior in certain individuals, potentially leading to addictive disorders.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "ALCOHOL CUE EXPOSURE/"}, {"entity_type": "RISK FACTOR/", "description": "Disease/Increased alcohol consumption is linked to several health risks, including addiction.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "ALCOHOL CONSUMPTION/"}, {"entity_type": "RISK FACTOR/", "description": "Disease/Tobacco use is considered a risk factor contributing to multiple diseases, including addiction.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "TOBACCO USE/BEHAVIORS/"}, {"entity_type": "BEHAVIOR/", "description": "Anxiety/Disease/Genetic variations like rs2023239 may influence how individuals perceive rewarding effects of alcohol, which can vary based on anxiety levels or predispositions to certain diseases.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "ALCOHOL REWARDING EFFECTS/"}, {"entity_type": "ASSESSMENT TOOL/", "description": "Risk Factor/", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "PROFILE OF MOOD STATES/"}, {"entity_type": "CONTROL MEASURE/", "description": "Risk Factor/", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "PLACEBO BEVERAGE/"}, {"entity_type": "VARIANT", "description": "Individuals with the CT genotype were identified in the studies.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "CT GENOTYPE"}, {"entity_type": "VARIANT", "description": "The TT genotype refers to individuals without rs2023239 variant present.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "TT GENOTYPE"}, {"entity_type": "DRUG", "description": "Medications used were olanzapine and cyproheptadine hydrochloride.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "MEDICATION: OLANZAPINE VS CYPROHEPTADINE HYDROCHLORIDE"}, {"entity_type": "GENOTYPE", "description": "The genotypes included were TT, CT, and CC for the study.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "GENOTYPE: TT VS CT AND CC"}, {"entity_type": "DOSE LEVEL", "description": "The drink levels considered were 1, 2, and 3 during the study.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "DRINK: 1 VS 2 VS 3"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Reward was measured in relation to alcohol consumption for individuals with different genotypes.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "REWARD"}, {"entity_type": "EMOTIONAL STATE", "description": "Positive emotional states were observed after alcohol intake for individuals with specific genotypes.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "POSITIVE AFFECT"}, {"entity_type": "SUBJECTIVE EXPERIENCE", "description": "Self-reported feelings of euphoria were examined in relation to different medications and drinks.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "SELF-REPORTED HIGH"}, {"entity_type": "DRUG TYPE", "description": "Two medications, cyproheptadine hydrochloride and olanzapine, were compared in the study.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "ACTIVE CONTROL MEDICATION (CYPROHEPTADINE) VS OLANZAPINE"}, {"entity_type": "GENETIC VARIANT", "description": "The genetic variant CNR1 was considered for its potential interaction with olanzapine.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "CNR1 VARIATION"}, {"entity_type": "PSYCHOLOGICAL PROCESS", "description": "Cravings elicited by cues were assessed in individuals treated with olanzapine and placebo.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "CUE-ELICITED CRAVING"}, {"entity_type": "TREATMENT COMPARISON", "description": "Comparisons were made between treatment with placebo and treatment with olanzapine.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 29}]", "id": "PLACEBO VS OLANZAPINE"}, {"entity_type": "DRUG", "description": "Olanzapine was used as a medication in studies examining its effect on alcohol consumption and craving.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "MEDICATION: OLANZAPINE"}, {"entity_type": "DRUG", "description": "Cyproheptadine hydrochloride served as an active control medication.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "CYPROHEPTADINE HYDROCHLORIDE"}, {"entity_type": "GENETIC VARIANT", "description": "The genetic variant CNR1 was also considered in studies looking at olanzapine\'s impact on craving.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "CNR1 VARIATION (RS2035618)"}, {"entity_type": "STATISTICAL INTERACTION", "description": "A statistical analysis revealed an interaction between medication, genotype and cue in relation to craving response.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "MEDICATION \u00d7 GENOTYPE \u00d7 CUE INTERACTION"}, {"entity_type": "STATISTICAL INTERACTION", "description": "A statistical analysis found a link between medications, genotypes and the number of drinks consumed.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "MEDICATION \u00d7 GENOTYPE \u00d7 DRINK INTERACTION"}, {"entity_type": "STATISTICAL ANALYSIS", "description": "The analysis showed significant differences in craving response based on medication taken, individual\'s genetic profile, and presence of cues.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "MEDICATION \u00d7 GENOTYPE \u00d7 CUE INTERACTION ON CRAVING"}, {"entity_type": "STATISTICAL INTERACTION", "description": "A statistical analysis indicated differences in self-reported feelings of euphoria related to medications used, genotypes, and the number of drinks consumed.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "MEDICATION \u00d7 GENOTYPE \u00d7 DRINK INTERACTION ON SELF-REPORTED HIGH"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "The rs2023239 variant influences CB1 receptor levels, affecting neurocircuitry related to drug-seeking behavior.)", "entity_type": "UNKNOWN", "id": "CB1 RECEPTOR"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Disease<|Genetic variations in CNR1 can affect CB1 receptor levels, which are associated with addiction and other diseases.)", "entity_type": "UNKNOWN", "id": "CB1 RECEPTOR LEVELS/"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Disease<|Genetic variations in CNR1 can affect CB1 receptor levels, which are associated with addiction and other diseases.)", "entity_type": "UNKNOWN", "id": "CNR1 GENE"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Disease<|Exposure to alcohol cues can activate neurocircuitry that modulates drug-seeking behavior, possibly related to addictive disorders.)", "entity_type": "UNKNOWN", "id": "ALCOHOL CUE EXPOSURE"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Disease<|Exposure to alcohol cues can activate neurocircuitry that modulates drug-seeking behavior, possibly related to addictive disorders.)", "entity_type": "UNKNOWN", "id": "NEUROCIRCUITRY ACTIVATION/"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Alcohol Use<|The rs2023239 variant influences CB1 receptor function and its sensitivity to alcohol.)", "entity_type": "UNKNOWN", "id": "CB1 RECEPTOR REGULATION/"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Alcohol Use<|The rs2023239 variant influences CB1 receptor function and its sensitivity to alcohol.)", "entity_type": "UNKNOWN", "id": "RS2023239 VARIANT"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Anxiety<|Genetic variations in CNR1 may impact responses to alcohol, influencing anxiety levels.)", "entity_type": "UNKNOWN", "id": "CNR1 GENOTYPE"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Anxiety/", "entity_type": "UNKNOWN", "id": "RS2023239 VARIANT/"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Anxiety/", "entity_type": "UNKNOWN", "id": "CB1 RECEPTOR REGULATION/"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Anxiety/", "entity_type": "UNKNOWN", "id": "CB1 RECEPTOR/"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Alcohol Consumption/", "entity_type": "UNKNOWN", "id": "EXON 3/"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Alcohol Consumption/", "entity_type": "UNKNOWN", "id": "CANNABINOID RECEPTOR REGULATION/"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Risk Factor/", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT/"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "Risk Factor/", "entity_type": "UNKNOWN", "id": "ALTERNATIVE PROMOTER/"}, {"source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "description": "The genotypic variant rs2023239 was related to the response to different medications during alcohol consumption studies.)", "entity_type": "UNKNOWN", "id": "MEDICATION: OLANZAPINE AND CYPROHEPTADINE HYDROCHLORIDE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Involves packaging of dietary fat into chylomicrons in the small intestine, which are later transported to the bloodstream.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "EXOGENOUS LIPID METABOLISM"}, {"entity_type": "METABOLIC PATHWAY", "description": "Initial step involves hydrolysis of chylomicrons by lipoprotein lipase (LPL).", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "ENDOGENOUS LIPID METABOLISM"}, {"entity_type": "PROTEIN", "description": "An enzyme that hydrolyzes triglycerides in lipoproteins.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "LIPOPROTEIN LIPASE (LPL)"}, {"entity_type": "DISEASE", "description": "A rare autosomal recessive disorder with a prevalence of 1/1,000,000 in the United States and higher in other world regions due to a founder effect.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "LPL DEFICIENCY"}, {"entity_type": "PATHOLOGICAL CONDITION", "description": "Can be present in affected homozygotes or compound heterozygotes with LPL deficiency.<SEP>Patients homozygous or compound heterozygotes of the LPL gene can present with marked hypertriglyceridemia.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63<SEP>chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "HYPERTRIGLYCERIDEMIA"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Often have normal lipids, even when carrying one copy of the mutated gene.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "SIMPLE HETEROZYGOTE CARRIERS"}, {"entity_type": "CLINICAL SIGN", "description": "Characterized by yellowish, papular eruptions on the skin, often seen in individuals with elevated levels of lipid metabolism-related disorders.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "ERUPTIVE XANTHOMAS"}, {"entity_type": "SYMPTOM", "description": "One of the symptoms that can be experienced by patients with LPL deficiency or other related conditions.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "ABDOMINAL PAIN"}, {"entity_type": "ANATOMICAL ABNORMALITY", "description": "Enlargement of liver and spleen, often observed in individuals with hypertriglyceridemia or LPL deficiency.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "HEPATO-SPLENOMEGALY"}, {"entity_type": "COMPLICATION", "description": "A complication that can occur due to severe hypertriglyceridemia affecting the digestive system.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "LOWER GASTROINTESTINAL BLEEDING"}, {"entity_type": "RECURRING CONDITION", "description": "An inflammatory condition of the pancreas, which might recur in patients with LPL deficiency or related lipid metabolism disorders.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "RECURRENT PANCREATITIS"}, {"entity_type": "OCULAR ABNORMALITY", "description": "A condition characterized by excessive blood leakage into the eye, often related to high levels of lipids in circulation.<SEP>The condition where fat deposits accumulate in blood vessels supplying the retina, leading to vision problems.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63<SEP>chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "LIPEMIA RETINALIS"}, {"entity_type": "TREATMENT", "description": "The cornerstone of treatment for patients with LPL deficiency or other lipid metabolism disorders.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "DIETARY FAT RESTRICTION"}, {"entity_type": "FORMULA", "description": "An infant formula with a high fat content from medium-chain triglycerides, used for treatment purposes in individuals with certain lipid metabolism disorders.<SEP>Formula with a high content of medium-chain triglycerides and lower long-chain fats, used as part of the treatment regimen for affected individuals.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "ENFAPORT LIPIL"}, {"entity_type": "GENE", "description": "The gene responsible for encoding lipoprotein lipase (LPL), which plays a role in multiple stages of lipid metabolism.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "LPL GENE"}, {"entity_type": "GENETIC CONDITION", "description": "A disorder that requires the defective gene to be inherited from both parents in order to express clinical symptoms.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "AUTOSOMAL RECESSIVE DISORDER"}, {"entity_type": "ETHNICITY/", "description": "Population Subgroup", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "PUERTO RICAN DESCENT"}, {"entity_type": "SYMPTOM/CLINICAL SIGN", "description": "Non-bloody, often vomited material that resembles coffee grounds, seen in patients with gastrointestinal issues or certain diseases.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "COFFEE-GROUND EMESIS"}, {"entity_type": "GASTROINTESTINAL SYMPTOM", "description": "Dark, tarry stool resulting from bleeding in the upper gastrointestinal tract, possibly due to severe hypertriglyceridemia.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "MELENA"}, {"entity_type": "SKIN CONDITION/CLINICAL SIGN", "description": "A rash characterized by small, red, raised lesions on the skin surface that can be a symptom of various health issues including disorders related to lipid metabolism.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "PAPULAR RASH"}, {"entity_type": "ANATOMICAL ABNORMALITY", "description": "Blood with visible excess fat droplets due to high levels of lipids in circulation.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "GROSSLY LIPEMIC BLOOD"}, {"entity_type": "DIAGNOSTIC TEST", "description": "A laboratory test used to evaluate the level of different lipids (such as cholesterol and triglycerides) in the blood after an overnight fast.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "FASTING LIPID PROFILE"}, {"entity_type": "ENZYME/", "description": "Protein", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "PANCREATIC LIPASE"}, {"entity_type": "GENETIC STATUS/LIFESTYLE FACTOR", "description": "Carriers who have one normal copy of the LPL gene and one mutated copy but do not exhibit symptoms.)", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "HETEROZYGOTE INDIVIDUALS"}, {"entity_type": "CLINICAL SIGN/", "description": "Symptom", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "REFLUX"}, {"entity_type": "MEDICAL INTERVENTION/THERAPY", "description": "Pharmacological treatments aimed at reducing levels of cholesterol and triglycerides in the blood.)", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "LIPID-LOWERING DRUGS"}, {"entity_type": "NUTRIENT/VITAMIN/BIOACTIVE COMPOUND", "description": "Often used as a dietary supplement or medication to lower triglycerides and increase HDL (good) cholesterol levels.)", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "NIACIN"}, {"entity_type": "COLESTIPOL", "description": "Drug Class/", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "CHOLESTYRAMINE/"}, {"entity_type": "XENICAL", "description": "Weight Loss Drug/Medication", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "ORLISTAT/"}, {"entity_type": "ENZYME/", "description": "Protein", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "GASTROINTESTINAL LIPASE"}, {"entity_type": "MEDICAL CONDITION/ANATOMICAL ABNORMALITY", "description": "A disorder where the pancreas fails to produce enough digestive enzymes for proper digestion of food, often affecting fat breakdown.)", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "PANCREATIC EXOCRINE INSUFFICIENCY"}, {"entity_type": "JEJUNOSTOMY", "description": "Intervention/", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "GASTROSTOMY/"}, {"entity_type": "GENOMIC VARIANT", "description": "The p.Q240H mutation in exon 5 of the LPL gene causes severe hypertriglyceridemia.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "P.Q240H"}, {"entity_type": "GENE", "description": "The LPL deficiency has clinical manifestations that include eruptive xanthomas, lipemia retinalis, abdominal pain, gastrointestinal hemorrhage, and pancreatitis.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "LPL"}, {"entity_type": "METABOLIC PATHWAY", "description": "The damage from increased TG levels can extend to the posterior pole of retina and risk increases with higher TG levels.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "TG LEVELS"}, {"entity_type": "DIETARY PATTERN", "description": "Dietary restriction is considered as a cornerstone for treating LPL deficiency.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "FAT RESTRICTION"}, {"entity_type": "NUTRIENT", "description": "Long-chain triglycerides (LCTs) are replaced with medium-chain triglycerides (MCTs) in dietary formulas for patients with LPL deficiency.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "MCT"}, {"entity_type": "DRUG", "description": "Combination of drug therapies like gemfibrozil, omega-3 fatty acids, and orlistat can decrease TG levels when used with a fat-restricted diet.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "GEMFIBROZIL"}, {"entity_type": "DRUG", "description": "Currently under investigation for treating LPL deficiency, DGAT1 inhibitors target the synthesis of triglycerides during dietary fat absorption.<SEP>Drugs under investigation that specifically target the DGAT1 enzyme to inhibit triglyceride synthesis during fat absorption.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "DGAT1 INHIBITORS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A gain-of-function variant of the LPL gene is approved in Europe for treatment of patients with LPL deficiency.<SEP>A genetic variant that enhances the function of a gene product, in this case enhancing LPL activity.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "GAIN-OF-FUNCTION VARIANT"}, {"entity_type": "THERAPY", "description": "Introduction of a gain-of-function variant of the LPL gene through intramuscular therapy is considered but limited in duration and cost.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "INTRAMUSCULAR INTRODUCTION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The role of DGAT1 mediates triglyceride synthesis during dietary fat absorption, an area under investigation for treatment.)(entity", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "DIACYLGLYCEROL O-ACYLTRANSFERASE 1 (DGAT1)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A complex consisting of triglycerides, proteins and phospholipids that transport fats through the bloodstream.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "TRIGLYCERIDE-RICH LIPOPROTEINS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Increased viscosity of body fluids due to high levels of triglycerides that can cause blood flow issues.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "HYPERVISCOSITY"}, {"entity_type": "DISEASE", "description": "Inflammation of the pancreas often associated with high levels of TG and is a common issue in patients with LPL deficiency.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "PANCREATITIS"}, {"entity_type": "DRUG", "description": "Drugs used as part of therapy for LPL deficiency to decrease triglyceride levels when combined with dietary changes.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "GEMFIBROZIL, OMEGA-3 FATTY ACIDS, ORLISTAT"}, {"entity_type": "THERAPY", "description": "A medical procedure where drugs or other substances are injected into muscle tissue, as part of treatment for LPL deficiency.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "INTRAMUSCULAR THERAPY"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Function", "entity_type": "UNKNOWN", "id": "ROLE/"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "An effective treatment strategy to manage symptoms and complications associated with this genetic disorder.)", "entity_type": "UNKNOWN", "id": "TREATMENT FOR LPL DEFICIENCY"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Medical Intervention", "entity_type": "UNKNOWN", "id": "FORMULA/"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Genetic Connection", "entity_type": "UNKNOWN", "id": "AUTOSOMAL RECESSIVE DISORDER/"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Genetic Connection", "entity_type": "UNKNOWN", "id": "LPL DEFICIENCY"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Diagnostic Method", "entity_type": "UNKNOWN", "id": "MUTATION ANALYSIS/"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Diagnostic Method", "entity_type": "UNKNOWN", "id": "GENOMIC DNA"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Association between symptoms and conditions", "entity_type": "UNKNOWN", "id": "CLINICAL SIGNS/"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Association between symptoms and conditions", "entity_type": "UNKNOWN", "id": "PATHOLOGICAL MANIFESTATIONS"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "High triglyceride levels", "entity_type": "UNKNOWN", "id": "GASTROINTESTINAL COMPLICATIONS"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "High triglyceride levels", "entity_type": "UNKNOWN", "id": "OUTCOME OF LPL DEFICIENCY/"}, {"entity_type": "DRUG", "description": "A drug used for treating patients during the study.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 30}]", "id": "TOPIRAMATE"}, {"entity_type": "GENOMIC VARIANT", "description": "A genetic variant that influences the effects of topiramate on HDDs in European-American patients.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "RS2832407"}, {"entity_type": "DISEASE", "description": "The Disease under Study, referring to the condition being treated with topiramate and placebo.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "HDDS"}, {"entity_type": "PATIENT SUBGROUP", "description": "European-American patients included in the study.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 36}, {level: 2, cluster: 60}]", "id": "EA PATIENTS"}, {"entity_type": "GENOMIC VARIANT", "description": "Individuals carrying at least one copy of the A allele of rs2832407.<SEP>Individuals carrying the A allele of rs2832407 that respond differently to topiramate.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "A-ALLELE CARRIERS"}, {"entity_type": "GENOMIC VARIANT", "description": "Individuals who are homozygous for the C allele of rs2832407.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "C-HOMOZYGOTES"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836", "description": "The study aims to determine the effectiveness of topiramate and placebo in managing HDDs.)", "entity_type": "UNKNOWN", "id": "EFFICACY"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836", "description": "The rs2832407 variant moderates the effect of topiramate on HDDs, showing a significant difference between C-homozygotes and A-allele carriers.)", "entity_type": "UNKNOWN", "id": "TOPIRAMATE EFFECT"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836<SEP>chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "description": "A-allele carriers showed a significantly higher mean severity rating for adverse events compared to those with other genotypes when taking topiramate.)<SEP>Unintended negative responses or harmful effects from medical procedures or treatments.", "entity_type": "SAFETY OUTCOME", "id": "ADVERSE EVENTS"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836", "description": "C-homozygotes showed a significant reduction in HDDs when taking topiramate, with an NNT of 2.28.)", "entity_type": "UNKNOWN", "id": "EFFECT"}, {"entity_type": "PERSON", "description": "The study involved a total of 120 eligible patients who were approached, with 90% signing consent and 89% completing the study.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "PATIENT"}, {"entity_type": "INFORMATION GATHERING", "description": "Complete self-report and genetic data were collected from 88 patients who participated in the current analyses.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "DATA COLLECTION"}, {"entity_type": "STATISTICAL ANALYSIS", "description": "The demographic and clinical characteristics of the sample were analyzed, with a mean age of 60 years (SD=10).", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "SAMPLE ANALYSIS"}, {"entity_type": "RESEARCH SUBJECTS", "description": "Patients in the study were predominantly female (55%), white (93%), non-Hispanic (97%) and had an annual household income of at least $40,000 per year (60%).", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "STUDY PARTICIPANTS"}, {"entity_type": "MEDICAL CONDITION", "description": "Patients diagnosed with gastrointestinal (34%), thoracic (27%), or breast malignancies (26%); most received highly-emetogenic chemotherapy regimens (65%).", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "MALIGNANCY TYPES"}, {"entity_type": "PHARMACEUTICAL PRODUCT", "description": "Commonly used anti-emetics included palonosetron (99%) and dexamethasone (98%). Additional medications like prochlorperazine (88%), ondansetron (46%), and fosaprepitant (34%) were also prescribed.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "ANTI-EMETIC MEDICATIONS"}, {"entity_type": "SYMPTOM EVALUATION", "description": "12% of patients experienced CTCAE grade 2 or above acute nausea, while 30% experienced delayed nausea. Vomiting was less frequent, with 3% experiencing it in the acute period and 11% in the delayed period.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "CINV SEVERITY"}, {"entity_type": "MEDICAL PROCEDURE", "description": "Anti-emetic regimen switching occurred in 14% of patients based on their response to chemotherapy.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "CHEMOTHERAPY REGIMEN"}, {"entity_type": "SUBJECTIVE ASSESSMENT", "description": "Risk factors for CINV included patient-reported severity of acute nausea associated with younger age and reduced hours of sleep the night before chemotherapy. Severity of CTCAE grade of acute nausea was linked to age, sleep hours, and history of morning sickness.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "PATIENT-REPORTED RISK FACTORS FOR CINV"}, {"entity_type": "BIOLOGICAL FACTOR", "description": "Variation in ABCB1 (rs1045642) was found associated with more severe acute nausea compared to other genotypes. No associations were found between genetic factors such as HTR3B (rs45460698), CYP2D6 phenotype, or activity score and acute or delayed CINV.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "GENETIC RISK FACTORS"}, {"entity_type": "DIAGNOSTIC PROCEDURE", "description": "The genotyping process was conducted as part of a comprehensive approach to analyze genetic data for participants.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "GENOTYPING PROCESS"}, {"entity_type": "RESEARCH TOOL", "description": "Complete self-report and genetic data were collected using validated questionnaires and standardized tools.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "SELF-REPORT DATA"}, {"entity_type": "STATISTICAL METHOD", "description": "A multivariate statistical analysis method was employed to identify significant associations between patient characteristics, clinical variables, and reported symptoms.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "DATA ANALYSIS METHOD"}, {"entity_type": "CLINICAL ASSESSMENT SCALE", "description": "The Common Terminology Criteria for Adverse Events (CTCAE) grading system was used to assess the severity of chemotherapy-induced nausea and vomiting in patients.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "CTCAE GRADING SYSTEM"}, {"entity_type": "DEMOGRAPHIC VARIABLE", "description": "Younger age was identified as an independent risk factor for increased acute nausea severity, based on self-report data.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "AGE AS A RISK FACTOR"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Reduced hours of sleep the night before chemotherapy was found to be associated with more severe acute nausea symptoms.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "SLEEP HOURS AS A RISK FACTOR"}, {"entity_type": "SYMPTOM HISTORY", "description": "A history of nausea and vomiting due to morning sickness was linked to higher severity in CTCAE grade for both acute and delayed nausea.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "MORNING SICKNESS HISTORY AS A RISK FACTOR"}, {"entity_type": "CHEMOTHERAPEUTIC AGENT", "description": "The use of highly emetogenic chemotherapy regimens was significantly associated with increased risk of severe delayed nausea symptoms.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "HIGHLY EMETOGENIC CHEMOTHERAPY"}, {"entity_type": "MEDICAL DIAGNOSIS", "description": "Cancer stage, particularly advanced stages, correlated with more severe delayed nausea symptoms based on CTCAE grading.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "STAGE OF CANCER AS A RISK FACTOR"}, {"entity_type": "GENETIC VARIANT", "description": "Variation in the ABCB1 gene (specifically rs1045642) was found to influence acute nausea severity, with different genotypes showing varying levels of symptom intensity.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "ABCB1 GENOTYPE AND ACUTE NAUSEA"}, {"entity_type": "GENETIC VARIANT EXPRESSION", "description": "Patients carrying the heterozygous ABCB1 genotype reported more severe acute nausea compared to those with homozygous wild-type or variant genotypes.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "HETEROZYGOTE GENOTYPE AND ACUTE NAUSEA SEVERITY"}, {"entity_type": "MEDICAL SIGN", "description": "No risk factors for vomiting were identified, but it was documented based on self-reports and medical monitoring in the study participants.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "VOMITING AS A SYMPTOM"}, {"entity_type": "STATISTICAL METHODOLOGY", "description": "Multivariable regression analysis was used to quantify the effects of demographic, clinical, and patient-reported variables on acute nausea severity.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "REGRESSION ANALYSIS"}, {"entity_type": "ANALYTICAL INSIGHT", "description": "The identified risk factors explained a certain percentage of variance in acute nausea symptoms; genetic factor added an incremental amount of explanation beyond demographic and clinical factors.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "VARIANCE EXPLAINED BY RISK FACTORS"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Genotyping was conducted as part of data collection for the study with 93 participants.)", "entity_type": "UNKNOWN", "id": "GENOTYPING AND DATA COLLECTION"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Genotyping was conducted as part of data collection for the study with 93 participants.)", "entity_type": "UNKNOWN", "id": "STUDY METHODS"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "The sample\'s characteristics were analyzed, revealing a mean age of 60 years (SD=10).)", "entity_type": "UNKNOWN", "id": "SAMPLE ANALYSIS AND DESCRIPTIVE STATISTICS"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Patients in the study were predominantly female (55%) and had various malignancies.)", "entity_type": "UNKNOWN", "id": "PATIENT CHARACTERISTICS"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Patients in the study were predominantly female (55%) and had various malignancies.)", "entity_type": "UNKNOWN", "id": "POPULATION DESCRIPTION"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "The study involved patients diagnosed with gastrointestinal, thoracic, or breast cancers who received highly-emetogenic chemotherapy.)", "entity_type": "UNKNOWN", "id": "HEALTH CONDITION AND TREATMENT"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "The study involved patients diagnosed with gastrointestinal, thoracic, or breast cancers who received highly-emetogenic chemotherapy.)", "entity_type": "UNKNOWN", "id": "MALIGNANCY TYPES AND CHEMOTHERAPY"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Commonly used anti-emetics included palonosetron, dexamethasone, prochlorperazine, ondansetron, and fosaprepitant.)", "entity_type": "UNKNOWN", "id": "ANTI-EMETIC MEDICATIONS AND TREATMENT"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Risk factors for CINV such as age, sleep hours, and morning sickness were linked to the severity of nausea.)", "entity_type": "UNKNOWN", "id": "CINV SEVERITY AND RISK FACTORS"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Risk factors for CINV such as age, sleep hours, and morning sickness were linked to the severity of nausea.)", "entity_type": "UNKNOWN", "id": "SYMPTOM ASSESSMENT AND FACTORS"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "The genetic variation in ABCB1 was found to be associated with more severe acute nausea compared to other genotypes.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION WITH SYMPTOM"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "The genetic variation in ABCB1 was found to be associated with more severe acute nausea compared to other genotypes.)", "entity_type": "UNKNOWN", "id": "GENETIC RISK FACTOR (ABCB1 RS1045642)"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Demographic, clinical, and patient-reported factors explained 13% of the variance in acute nausea; ABCB1 genotype contributed an additional variance (p=0.10).)", "entity_type": "UNKNOWN", "id": "ANALYSIS RESULTS"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Demographic, clinical, and patient-reported factors explained 13% of the variance in acute nausea; ABCB1 genotype contributed an additional variance (p=0.10).)", "entity_type": "UNKNOWN", "id": "STATISTICAL SIGNIFICANCE AND VARIANCE"}, {"entity_type": "GENE", "description": "A gene involved in DNA replication and repair processes.<SEP>MCM6 is another gene involved in the regulation of LCT.<SEP>The gene MCM6 remains unchanged in expression when silencing of LOC100507600 occurs, acting as a control in this context.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c<SEP>chunk-ec104044b3aa65560be8d81dbb3940d6<SEP>chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "MCM6"}, {"entity_type": "EPIGENETIC MODIFICATION", "description": "CpG sites are critical for epigenetic regulation and correlate with mRNA levels.<SEP>Specific locations along DNA that are analyzed to understand modification patterns.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6<SEP>chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "CPG SITE"}, {"entity_type": "SNP", "description": "A specific Single Nucleotide Polymorphism (SNP) in the LCT gene is linked to lactase persistence or non-persistence.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "LACTASE PERSISTENCE SNP"}, {"entity_type": "EPIGENETIC PROCESS", "description": "Across the genome, there are chromosome-wide patterns of DNA modification that play a role in gene regulation.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "CHROMOSOME-WIDE DNA MODIFICATION"}, {"entity_type": "BIOLOGICAL TOOL", "description": "High-density arrays used for interrogation of DNA modifications across human chromosomes.<SEP>Specific high-resolution genomic microarrays used to survey DNA modifications.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "AFFYMETRIX HIGH DENSITY HUMAN TILING ARRAYS"}, {"entity_type": "CELL TYPE", "description": "Enterocytes are specific intestinal cells that express the LCT gene.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "ENTEROCYTES"}, {"entity_type": "CELL TYPE", "description": "These cells lining the intestine play a role in lactase expression.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "INTESTINAL EPITHELIAL CELLS"}, {"entity_type": "BODY PART", "description": "The inferior part of the small intestine where enterocytes are found.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "INFERIOR JEJUNUM"}, {"entity_type": "MODEL ORGANISM", "description": "Mice, especially as infants, exhibit high intestinal lactase activity followed by a gradual decrease.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "INFANT MICE"}, {"entity_type": "MODEL ORGANISM", "description": "In contrast to infant mice, adult mice have lower intestinal lactase activity.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "ADULT MICE"}, {"entity_type": "TIME POINT", "description": "This refers to the age of mice at which lactase activity begins to decline.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "POSTNATAL DAY 60"}, {"entity_type": "CHROMOSOME", "description": "Mouse chromosome that contains the Lct gene, responsible for lactose digestion.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "MOUSE CHROMOSOME 1"}, {"entity_type": "BIOLOGICAL TOOL", "description": "A method used to enrich unmodified DNA fractions.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "MTAG-CLICK METHOD"}, {"entity_type": "BIOLOGICAL TOOL", "description": "Microarrays used for detection of differential DNA modification in different age groups.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "TILING MICROARRAYS"}, {"entity_type": "SPECIFIC TOOL", "description": "These are the specific arrays used to analyze the epigenetic modifications on mouse chromosomes.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "MOUSE TILING MICROARRAYS"}, {"entity_type": "METHYLATION SITES", "description": "CpG sites identified where DNA modification changes were detected, impacting gene expression.<SEP>We determined DNA methylation levels at certain CpG sites for comparison between CD4+ and CD8+ T cells.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca<SEP>chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "CPG SITES"}, {"entity_type": "CHROMOSOME", "description": "The human chromosome that contains the LCT gene.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "CHROMOSOME 2"}, {"entity_type": "BODY PART SAMPLE", "description": "Samples isolated from the inferior part of the small intestine for analysis.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "INFERIOR JEJUNUM SAMPLES"}, {"entity_type": "CELL TYPE SAMPLE", "description": "Specific intestinal cells lining the villi, analyzed for DNA modification patterns.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "VILLI ENTEROCYTES"}, {"entity_type": "EPIGENETIC MEASUREMENT", "description": "Measurements of cytosine methylation levels in CpG sites across different genomic regions.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "CPG SITE DENSITIES"}, {"entity_type": "CONTROL SAMPLE", "description": "The control group representing the jejunum without enterocytes for comparison study.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "ENTEROCYTE-DEFICIENT JEJUNUM"}, {"entity_type": "AGE POINT", "description": "Specific age point at birth in mice where lactase activity is high.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "POSTNATAL DAY 6"}, {"entity_type": "METHOD OF ANALYSIS", "description": "Comprehensive analysis across the entire length of a chromosome to identify correlated DNA modifications and gene expression.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "CHROMOSOME-WIDE ANALYSIS"}, {"entity_type": "GENOMIC SEQUENCING METHOD", "description": "High-resolution genomic sequencing technique used for CpG site-specific epigenetic modifications.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "BISULFITE PADLOCK PROBE SEQUENCING"}, {"entity_type": "SPECIFIC CPG LOCATION", "description": "A specific CpG site of interest in the MCM6 gene linked to lactase persistence or non-persistence.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "CPG SITE 13910"}, {"entity_type": "BIOLOGICAL FUNCTION", "description": "The process by which LCT facilitates the breakdown and absorption of lactose in the small intestine.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "LACTOSE DIGESTION"}, {"entity_type": "GENE REGION", "description": "A specific region within MCM6 gene that influences LCT mRNA levels through epigenetic modifications.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "MCM6 INTRON 13-EXON 13"}, {"entity_type": "PHENOTYPIC TRAIT", "description": "The traits distinguishing individuals who can or cannot digest lactose, influenced by genetic factors.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "LACTASE PERSISTENCE AND NON-PERSISTENCE"}, {"entity_type": "CELL TYPE", "description": "Cells lining the small intestine responsible for nutrient absorption.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "ENTEROCYTE"}, {"entity_type": "STATISTICAL METHOD", "description": "A method used to adjust p-values in multiple hypothesis testing to control the family-wise error rate.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "BONFERRONI CORRECTION"}, {"entity_type": "DNA MODIFICATION ANALYSIS", "description": "Identified areas on DNA with specific modifications after comparison between different samples.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "DISTINCT REGION"}, {"entity_type": "DISEASE", "description": "A hypothesis that DNA modifications are erased at embryogenesis, enabling high postnatal LCT activity followed by gradual regain with age in a haplotype-specific manner.<SEP>Converging evidence indicates that LCT intron 2 is an evolutionarily old regulatory element common in mice and humans for the development of lactase non-persistence.<SEP>The inability of adults to digest lactose due to lack or reduced activity of LCT.", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1<SEP>chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "LACTASE NON-PERSISTENCE"}, {"entity_type": "DISEASE", "description": "Adults\' ability to continue digesting lactose, often inherited from parents.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "LACTASE PERSISTENCE"}, {"entity_type": "CHROMATIN REFERENCE MAPS", "description": "Regions on chromatin that are sensitive to DNase I enzyme due to accessible or less compact structure.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "DNASE I HYPERSENSITIVITY SITES"}, {"entity_type": "HISTONE MODIFICATION", "description": "A specific histone mark that was found to have peaks at certain regulatory sites within the LD block of rs35923643.<SEP>Post-translational modification of histones, often associated with active transcription start sites.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a<SEP>chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "H3K4ME1"}, {"entity_type": "HISTONE MARK", "description": "Another type of post-translational histone modification linked to enhancer activity and gene expression.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "H3K27AC"}, {"entity_type": "HISTONE MARK", "description": "Significant histone modification at promoter regions indicating active transcription start sites.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "H3K4ME3"}, {"entity_type": "DNA MODIFICATION ANALYSIS", "description": "Differential DNA modifications observed within LCT introns that may influence gene expression.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "LCT INTRONS 2 AND 3"}, {"entity_type": "CELL TYPE SPECIFIC EXPRESSION", "description": "Specific regions of MCM6 exhibit distinct DNA modification patterns in enterocytes compared to other tissues, affecting its activity.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "MCM6 INTRON 13 AND EXON 16/17"}, {"entity_type": "GENETIC ASSOCIATION", "description": "A specific SNP (C/T-13910) is part of haplotypes associated with lactase non-persistence or persistence in adults.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "HAPLOTYPES C/T-13910"}, {"entity_type": "AGE AND DEVELOPMENTAL CHANGES", "description": "Investigation of sperm samples reveals no DNA modification differences between genotypes, suggesting age-related changes start after conception.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "SPERM SAMPLES"}, {"entity_type": "DEVELOPMENTAL REGULATION", "description": "Modifications at MCM6 intron 13 complete most of their age-dependent regulation during the first two decades postnatally.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "POSTNATAL MODIFICATIONS"}, {"entity_type": "GENOMIC VARIANT", "description": "Changes at MCM6 intron 13 for haplotypes containing C-13910 must occur after conception.<SEP>Investigation of DNA modification changes within the MCM6 and LCT regions stratified for C/T-13910 genotypes showed increased DNA modifications consistently with age in lactase non-persistent CC individuals.", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "id": "MCM6 INTRON 13"}, {"entity_type": "GENOMIC VARIANT", "description": "Investigation of chromatin marks using ENCODE data identified an enhancer element (H3K27ac and H3K4me1 peaks) at Lct intron 2, along with an active promoter at LCT exon 1\u2013intron 1.<SEP>Significant differences between C-13910 and T-13910 carrying haplotypes, showing more gradual DNA modification changes compared to MCM6 intron 13.", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "id": "MCM6 EXON 16"}, {"entity_type": "DIETARY PATTERN", "description": "In our cohort of 21\u201372-year-old individuals, LCT mRNA was significantly decreasing with age in lactase non-persistent CC individuals.", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "id": "LACTASE NON-PERSISTENCE (CC)"}, {"entity_type": "DIETARY PATTERN", "description": "Lactase persistence showed no aging changes in intermediate CT individuals.", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "id": "INTERMEDIATE (CT) INDIVIDUAL"}, {"entity_type": "DIETARY PATTERN", "description": "A non-significant increase in lactase persistent TT individuals, as shown by R2 = 0.13 and p = 0.2.", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "id": "LACTASE PERSISTENT (TT)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "This lncRNA, transcribed from the opposite DNA strand than LCT, follows that it is a natural antisense transcript which can alter target gene expression through DNA modification.", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "id": "LNCRNA LOC100507600"}, {"entity_type": "NATURAL ANTISENSE TRANSCRIPT", "description": "The lncRNA LOC100507600 is a natural antisense transcript to LCT, suggesting it may influence target gene expression through DNA modification.", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "id": "LACTASE MRNA"}, {"entity_type": "CELL LINE", "description": "Caco-2 is a human intestinal epithelial cell line.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "CACO-2"}, {"entity_type": "GENOMIC VARIANT", "description": "A single nucleotide polymorphism at position -13910 has the TT genotype in this context.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "SNP -13910"}, {"entity_type": "GENE", "description": "The gene LOC100507600 expression was reduced by small-interfering RNAs directed towards it, resulting in a 20% decrease and a 25% reduction in LCT mRNA.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "LOC100507600"}, {"entity_type": "PROTEIN", "description": "CTCF is an insulator protein binding site that overlaps with the lncRNA promoter and was nearly ubiquitous across human tissues based on ENCODE datasets.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "CTCF"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Increased DNA methylation can inhibit CTCF binding at insulator sites, leading to tissue-specific gene expression activation.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "DNA METHYLATION"}, {"entity_type": "GENETIC MANIPULATION", "description": "CRISPR-Cas9 technology was utilized to manipulate differentially modified DNA regions contributing to LCT regulation.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "LACTASE REGULATORY ELEMENTS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The deletion of intron 2 in the Lct gene showed evidence of cross-species lactase gene regulation.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "INTRON 2 OF LCT"}, {"entity_type": "GENETIC MODIFICATION TOOL", "description": "CRISPR-Cas9 technology was used for genome editing, specifically targeting intronic regions without compromising splicing sites and exons involved in Lct mRNA processing and stability.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "CRISPR-CAS9 GENOME EDITING TECHNOLOGY"}, {"entity_type": "EXPERIMENTAL MODEL", "description": "Genetic manipulation studies were conducted on C57BL/6N mice.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "C57BL/6N MICE"}, {"entity_type": "GENE", "description": "The LCT gene plays a role in lactose metabolism, as it encodes for the lactase enzyme.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "LACTASE (LCT)"}, {"entity_type": "GENETIC MANIPULATION RESULT", "description": "A deletion of Mcm6 intron 13 yielded less pronounced reductions in Lct mRNA levels compared to other deletions.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "MCM6 INTRONIC DELETION"}, {"entity_type": "BIOLOGICAL MODEL", "description": "Studies were conducted on both adult and infant mice, with differing outcomes for the LCT mRNA level reductions related to specific deletions.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "ADULT AND INFANT MICE"}, {"entity_type": "GENETIC MODIFICATION", "description": "CRISPR-Cas9n-induced deletions in regulatory elements of MCM6 intron 13 or LCT intron 2 were performed, with notable effects observed when the cells reached a differentiated state.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "CRISPR-CAS9 INDUCED DELETIONS"}, {"entity_type": "QUANTITATIVE MEASURE", "description": "The deletion in Lct Introns significantly affected LCT mRNA levels during adult and infant mouse studies, with a greater reduction noted in adults compared to infants for intron 2.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "LCT MRNA LEVELS"}, {"entity_type": "GENE INTRANUCLEAR COMPONENTS", "description": "Deletions within MCM6 introns were investigated for their effect on LCT mRNA levels across different experimental models, such as adult and infant mice.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "MCM6 INTRONS"}, {"entity_type": "CELLULAR CONDITION PRIOR TO CONFLUENCE", "description": "Cultured human intestinal epithelial cells were examined prior to confluence (undifferentiated state) as a control for further analysis of LCT expression under differentiation conditions.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "CACO-2 CELLS - UNDIFFERENTIATED STATE"}, {"entity_type": "AGE-RELATED EFFECTS ON LCT EXPRESSION", "description": "Studies conducted with adult versus infant mice demonstrated age-specific differences in how certain deletions affect LCT mRNA levels and expression patterns in their intestinal tissues.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "ADULT VS INFANT MOUSE STUDIES"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Epigenetic regulation involves interactions between proteins and DNA modifying enzymes, impacting gene expression.<SEP>The study of epigenetic regulation was pivotal in understanding how certain genomic regions can influence the expression levels of the lactase gene (LCT) under various experimental conditions.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c<SEP>chunk-41c7131af28e2a745ef6b88b8601860f", "id": "EPIGENETIC REGULATION"}, {"entity_type": "CELLULAR STATES AND EXPRESSION ANALYSIS", "description": "Examination of LCT expression under different cellular states, including cancer-like conditions prior to confluence and fully differentiated intestinal epithelial cells post-confluence, provided insights into the effects of specific genomic manipulations.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "CANCER-LIKE STATE VS DIFFERENTIATED CELLS"}, {"entity_type": "COMPARATIVE EXPRESSION STUDIES", "description": "Detailed quantitative data analyses were performed to compare and contrast the effects of different deletions on LCT mRNA expression across various conditions, including age groups (adults vs infants) and experimental models.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "QUANTITATIVE DATA ANALYSIS"}, {"entity_type": "LCT GENE EXPRESSION ANALYSIS ACROSS AGES", "description": "Analysis of genomic responses to specific modifications or deletions revealed age-specific differences in LCT gene expression patterns, highlighting developmental influences and potential health impacts associated with these variations.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "AGE-SPECIFIC DIFFERENCES IN GENOMIC RESPONSE"}, {"source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "description": "Increased DNA modification at specific CpG sites in the LCT gene correlates with decreased mRNA levels.)", "entity_type": "UNKNOWN", "id": "DNA MODIFICATION"}, {"source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "description": "CpGs clusters around specific regions of MCM6 suggest epigenetic control influencing LCT expression.)", "entity_type": "UNKNOWN", "id": "MCM6 INTRONS AND EXON 16"}, {"source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "description": "A particular SNP in the MCM6 gene is correlated with lactase persistence or non-persistence via epigenetic mechanisms.)", "entity_type": "UNKNOWN", "id": "LACTASE PERSISTENCE SNP C/T-13910"}, {"source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "description": "A particular SNP in the MCM6 gene is correlated with lactase persistence or non-persistence via epigenetic mechanisms.)", "entity_type": "UNKNOWN", "id": "SNP-LCT RELATIONSHIP"}, {"source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "description": "Variation in DNA modifications between enterocytes and enterocyte-deficient jejunum highlights regulation specific to cell types.)", "entity_type": "UNKNOWN", "id": "CELL TYPE SPECIFIC EXPRESSION"}, {"source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "description": "Variation in DNA modifications between enterocytes and enterocyte-deficient jejunum highlights regulation specific to cell types.)", "entity_type": "UNKNOWN", "id": "REGULATION BY CELL TYPE"}, {"source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "description": "Age-specific differential DNA modifications detected on the mouse chromosome containing the Lct gene indicate age-related changes in lactase expression.)", "entity_type": "UNKNOWN", "id": "AGE GROUP COMPARISON - MOUSE LCT"}, {"source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "description": "Age-specific differential DNA modifications detected on the mouse chromosome containing the Lct gene indicate age-related changes in lactase expression.)", "entity_type": "UNKNOWN", "id": "AGE-DEPENDENT CHANGES"}, {"source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "description": "CpG sites identified as having varying DNA modifications across individuals and cell types, affecting LCT expression.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIATION IMPACT"}, {"source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "description": "CpG sites identified as having varying DNA modifications across individuals and cell types, affecting LCT expression.)", "entity_type": "UNKNOWN", "id": "INTER-INDIVIDUAL AND CELL-TYPE SPECIFIC SITES"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "Using Bonferroni correction when analyzing multiple CpG sites to prevent false positives in identifying significant differences.)", "entity_type": "UNKNOWN", "id": "DNA MODIFICATION PROFILE"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "DNA modifications in specific regions of MCM6, such as introns and exon 16/17, may impact its role in transcriptional regulation.)", "entity_type": "UNKNOWN", "id": "MCM6 INTRONS AND EXON 16/17"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "DNA modifications in specific regions of MCM6, such as introns and exon 16/17, may impact its role in transcriptional regulation.)", "entity_type": "UNKNOWN", "id": "REGULATORY FUNCTION"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "The maps showing DNase sensitivity sites overlap with regions of interest for LCT gene modifications, suggesting active regulatory elements.)", "entity_type": "UNKNOWN", "id": "CHROMATIN REFERENCE MAPS"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "Histone marks indicating activity at the enhancer region contribute to cell-type-specific regulation of genes like LCT and MCM6.)", "entity_type": "UNKNOWN", "id": "ENHANCER ACTIVITY"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "Histone marks indicating activity at the enhancer region contribute to cell-type-specific regulation of genes like LCT and MCM6.)", "entity_type": "UNKNOWN", "id": "H3K4ME1 AND H3K27AC"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "The promoter region of gene encoding for LCT shows high H3K4me3 levels indicating the potential for transcription initiation.)", "entity_type": "UNKNOWN", "id": "ACTIVE PROMOTER"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "The promoter region of gene encoding for LCT shows high H3K4me3 levels indicating the potential for transcription initiation.)", "entity_type": "UNKNOWN", "id": "LCT EXON 1"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "Different genetic backgrounds exhibit divergent DNA modifications that change over age, influencing LCT expression in enterocytes.)", "entity_type": "UNKNOWN", "id": "AGE-DEPENDENT REGULATION"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "Specific SNP C/T-13910 is associated with significant changes in DNA modification at MCM6 locus during postnatal development.)", "entity_type": "UNKNOWN", "id": "C/T-13910 SNP"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "Specific SNP C/T-13910 is associated with significant changes in DNA modification at MCM6 locus during postnatal development.)", "entity_type": "UNKNOWN", "id": "DNA MODIFICATION CHANGES"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "During embryonic development, LCT activity might be high due to absence of postnatal modifications; they gradually regain with age.)", "entity_type": "UNKNOWN", "id": "AGE-RELATED EPIGENETIC MODIFICATIONS"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "During embryonic development, LCT activity might be high due to absence of postnatal modifications; they gradually regain with age.)", "entity_type": "UNKNOWN", "id": "EMBRYOGENESIS AND POSTNATAL ACTIVITY"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "DNase I hypersensitivity sites on chromatin indicate regions that are more accessible, suggesting active regulatory processes in LCT and MCM6 genes.)", "entity_type": "UNKNOWN", "id": "DNA ACCESSIBILITY"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "DNase I hypersensitivity sites on chromatin indicate regions that are more accessible, suggesting active regulatory processes in LCT and MCM6 genes.)", "entity_type": "UNKNOWN", "id": "DNASE I ENZYME"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "The presence of histone mark H3K27ac at certain DNA regions suggests their involvement as enhancers for gene expression, particularly in LCT and MCM6 genes.)", "entity_type": "UNKNOWN", "id": "HISTONE MODIFICATION H3K27AC"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "H3K4me1 and H3K4me3 are post-translational modifications that play crucial roles in regulating gene expression, affecting transcriptional start sites in LCT and MCM6.)", "entity_type": "UNKNOWN", "id": "GENE EXPRESSION REGULATION"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "H3K4me1 and H3K4me3 are post-translational modifications that play crucial roles in regulating gene expression, affecting transcriptional start sites in LCT and MCM6.)", "entity_type": "UNKNOWN", "id": "POST-TRANSLATIONAL HISTONE MODIFICATION"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "Epigenetic modifications occurring from conception onwards gradually influence the age-dependent regulation of lactase activity in adults.)", "entity_type": "UNKNOWN", "id": "AGE-DEPENDENT EPIGENETIC CHANGES"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "Epigenetic modifications occurring from conception onwards gradually influence the age-dependent regulation of lactase activity in adults.)", "entity_type": "UNKNOWN", "id": "LACTASE ACTIVITY"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "High D\' value indicates strong correlation between certain SNPs, suggesting they are often inherited together and affect lactase gene expression or persistence.)", "entity_type": "UNKNOWN", "id": "GENETIC ASSOCIATION"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "High D\' value indicates strong correlation between certain SNPs, suggesting they are often inherited together and affect lactase gene expression or persistence.)", "entity_type": "UNKNOWN", "id": "PAIRWISE LINKAGE DISEQUILIBRIUM"}, {"source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "description": "The situation with MCM6 exon 16 shows significant differences between C-13910 and T-13910 carrying haplotypes, likely similar to the situation of MCM6 intron 13 in that DNA modifications are erased at embryogenesis.)", "entity_type": "UNKNOWN", "id": "DNA MODIFICATIONS"}, {"source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "description": "In lactase non-persistent CC individuals, LCT mRNA was significantly decreasing with age, whereas there were no aging changes in intermediate CT individuals and a non-significant increase in lactase persistent TT individuals.)", "entity_type": "UNKNOWN", "id": "LCT MRNA"}, {"source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "description": "The large degree of mammalian sequence conservation found in the LCT intron 2 region was not found in any other intron of LCT or MCM6, suggesting that LCT intron 2 is an evolutionarily old regulatory element.)", "entity_type": "UNKNOWN", "id": "MAMMALIAN SEQUENCE CONSERVATION"}, {"source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "description": "Our findings support the concept of age-related processes contributing to lactase regulation across species, as evidenced by evolutionary conservation of epigenetic factors and large degree of mammalian sequence conservation.)", "entity_type": "UNKNOWN", "id": "AGE-DEPENDENT DOWNREGULATION"}, {"source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "description": "Our findings support the concept of age-related processes contributing to lactase regulation across species, as evidenced by evolutionary conservation of epigenetic factors and large degree of mammalian sequence conservation.)", "entity_type": "UNKNOWN", "id": "EVOLUTIONARY CONSERVATION"}, {"source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "description": "Mammalian intronic conserved elements found near the LCT intron 2 region suggest its involvement in development, possibly alongside other regulatory processes such as age-related changes and evolutionary conservation.)", "entity_type": "UNKNOWN", "id": "DEVELOPMENTAL PROCESSES"}, {"source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "description": "Mammalian intronic conserved elements found near the LCT intron 2 region suggest its involvement in development, possibly alongside other regulatory processes such as age-related changes and evolutionary conservation.)", "entity_type": "UNKNOWN", "id": "LACTASE REGULATION"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "The TT genotype at SNP -13910 impacts LCT mRNA levels, which is relevant for lactase persistence.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT -13910"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "The TT genotype at SNP -13910 impacts LCT mRNA levels, which is relevant for lactase persistence.)", "entity_type": "UNKNOWN", "id": "LCT EXPRESSION"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Silencing of LOC100507600 leads to a 20% reduction in expression and concomitant 25% decrease in LCT mRNA, demonstrating regulation.)", "entity_type": "UNKNOWN", "id": "LCT MRNA REDUCTION"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Silencing of LOC100507600 leads to a 20% reduction in expression and concomitant 25% decrease in LCT mRNA, demonstrating regulation.)", "entity_type": "UNKNOWN", "id": "LOC100507600 SILENCING"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Increased DNA methylation inhibits CTCF binding at insulator sites, affecting tissue-specific gene expression.)", "entity_type": "UNKNOWN", "id": "CTCF BINDING INHIBITION"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Increased DNA methylation inhibits CTCF binding at insulator sites, affecting tissue-specific gene expression.)", "entity_type": "UNKNOWN", "id": "DNA METHYLATION INCREASE"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "CRISPR-Cas9 technology was utilized to manipulate lactase regulatory elements in genes like Lct and Mcm6 for understanding their role in regulation.)", "entity_type": "UNKNOWN", "id": "CRISPR-CAS9 GENOME EDITING"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "CRISPR-Cas9 technology was utilized to manipulate lactase regulatory elements in genes like Lct and Mcm6 for understanding their role in regulation.)", "entity_type": "UNKNOWN", "id": "LACTASE REGULATORY ELEMENT MANIPULATION"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Deletion of intron 2 in the LCT gene resulted in a widespread downregulation of LCT mRNA throughout the duodenum and jejunum.)", "entity_type": "UNKNOWN", "id": "INTRON 2 DELETION OF LCT"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Deletion of intron 2 in the LCT gene resulted in a widespread downregulation of LCT mRNA throughout the duodenum and jejunum.)", "entity_type": "UNKNOWN", "id": "LCT MRNA WIDESPREAD DOWNREGULATION"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Deletions using CRISPR-Cas9 technology showed varying impacts on LCT mRNA expression across different stages and models, including effects in adult and infant mice.)", "entity_type": "UNKNOWN", "id": "CRISPR-CAS9N-INDUCED DELETIONS"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Deletions using CRISPR-Cas9 technology showed varying impacts on LCT mRNA expression across different stages and models, including effects in adult and infant mice.)", "entity_type": "UNKNOWN", "id": "DIFFERENTIAL EFFECTS ON LCT EXPRESSION"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "The epigenetically-regulated regions in the MCM6 intron 13 or LCT intron 2 were found to impact human LCT expression in cultured cells, particularly after differentiation into an epithelial-like state.)", "entity_type": "UNKNOWN", "id": "EPIGENETIC REGULATION SITES"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "The epigenetically-regulated regions in the MCM6 intron 13 or LCT intron 2 were found to impact human LCT expression in cultured cells, particularly after differentiation into an epithelial-like state.)", "entity_type": "UNKNOWN", "id": "LCT EXPRESSION IMPACT"}, {"entity_type": "DISEASE", "description": "The study controlled the minor allele frequencies of rs4988235 within the population.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "CONTROLS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The \'T\' (persistence) allele at rs4988235 is associated with lactase persistence.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "T (PERSISTENCE) ALLELE"}, {"entity_type": "GENOMIC VARIANT", "description": "The \'C\' allele was found in the UK populations.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "C ALLELE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": " Romania showed nominal evidence for departure of genotype frequencies from Hardy-Weinberg equilibrium.<SEP>In Romania, there was nominal evidence for departure from Hardy-Weinberg equilibrium among rs4988235 genotypes.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "ROMANIA"}, {"entity_type": "DIETARY PATTERN", "description": "The milk drinking patterns differed between different rs4988235 genotype groups.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "MILK DRINKING PROFILES"}, {"entity_type": "STATISTICAL MEASURE", "description": "Genotype was associated with milk drinking in the controls group.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "OR 2.39 [1.81, 3.15], P=6.9*10\u221210"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A higher proportion of individuals with lactase non-persistent CC genotype reported never consuming milk.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "LACTASE NON-PERSISTENT CC GENOTYPE AT RS4988235"}, {"entity_type": "COUNTRY SPECIFIC DATA", "description": "In the Czech Republic, both milk consumption and educational attainment showed associations with other factors (smoking).<SEP>The Czech Republic showed distinct patterns in the frequency and distribution of the \'T\' (persistence) and \'C\' alleles.<SEP>The frequency of the lactase persistence driving allele was highest in the Czech Republic.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "CZECH REPUBLIC"}, {"entity_type": "DISEASE", "description": "Milk consumption was associated with increased risk of RCC in the Czech Republic.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "RISK OF RCC (RENAL CELL CARCINOMA)"}, {"entity_type": "STATISTICAL MEASURE", "description": "A strong relationship between milk consumption and cancer risk was observed in the Czech Republic.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "OR=1.68, 95% CI 1.13, 2.49 P=0.01"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Education level correlated with milk consumption patterns in all countries studied.<SEP>Educational attainment showed association with milk consumption.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "EDUCATIONAL ATTAINMENT"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Differences were observed in milk consumption patterns related to different lactase genotypes.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "MILK CONSUMPTION"}, {"entity_type": "GENETIC FACTOR", "description": "Analysis of rs4988235 genotype did not show substantial differences in RCC risk compared to milk consumption.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "RS4988235 GENOTYPE"}, {"entity_type": "ASSOCIATION STUDY", "description": "There was nominal evidence for association of rs4988235 genotype with education.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "GENOTYPE ASSOCIATION WITH EDUCATION"}, {"entity_type": "COUNTRY SPECIFIC DATA", "description": "Minor allele frequencies for rs4988235 showed diversity across different Polish populations.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "POLAND"}, {"entity_type": "COUNTRY SPECIFIC DATA", "description": "In Russia, minor allele frequencies of rs4988235 were observed to be intermediate between other Eastern European populations.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "RUSSIA"}, {"source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "description": "Increased milk consumption was linked to an elevated risk of RCC in the Czech Republic.)", "entity_type": "UNKNOWN", "id": "RISK OF RCC"}, {"source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "description": "Results for country-specific data showed varied associations, with no systematic pattern observed.)", "entity_type": "UNKNOWN", "id": "COUNTRY SPECIFIC DATA"}, {"source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "description": "Results for country-specific data showed varied associations, with no systematic pattern observed.)", "entity_type": "UNKNOWN", "id": "DIFFERENCES BETWEEN MILK CONSUMPTION AND CANCER RISK"}, {"source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "description": "Different country-specific minor allele frequency (MAF) estimates reflected intermediate frequencies as expected in Eastern European populations.)", "entity_type": "UNKNOWN", "id": "COUNTRY SPECIFIC MAF ESTIMATES"}, {"source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "description": "Different country-specific minor allele frequency (MAF) estimates reflected intermediate frequencies as expected in Eastern European populations.)", "entity_type": "UNKNOWN", "id": "FREQUENCY OF GENOTYPES"}, {"entity_type": "TREATMENT", "description": "HU treatment was administered to 111 patients, resulting in statistically significant improvements in several hematological parameters.", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "id": "HU TREATMENT"}, {"entity_type": "GENOMIC VARIANT", "description": "The HMIP-2 SNP is associated with MTD HbF in our observations.", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "id": "HMIP-2 SNP"}, {"entity_type": "GENOMIC VARIANT", "description": "The C-A haplotype of HMIP-2 is related to MTD HbF.", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "id": "C-A HAPLOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "Various HMIP-2 haplotypes are being analyzed in relation to HbF levels.", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "id": "HMIP-2 HAPLOTYPE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Baseline Hemoglobin F levels are a physiological state that was considered when analyzing HMIP-2 haplotype associations.", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "id": "BASELINE HBF"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Change in Hemoglobin F level over time is a physiological process related to the study of HMIP-2 variants and haplotypes.", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "id": "DELTA HBF"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Mean Total Direct Hemoglobin F is a physiological measurement that\'s relevant to the study of HMIP-2 genetic factors.", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "id": "MTD HBF"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "HU exposure led to statistically significant improvements in multiple hematological parameters among the patients.)", "entity_type": "UNKNOWN", "id": "IMPROVEMENTS"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf<SEP>chunk-34ffb077c584b98555081f882f8f8b94", "description": "HU exposure led to statistically significant improvements in multiple hematological parameters among the patients.)<SEP>Statistically significantly differentially expressed probes and genes identified in the study.", "entity_type": "STATISTICAL MEASURE", "id": "STATISTICAL SIGNIFICANCE"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "The correlation of baseline HbF with MTD HbF indicates that patients with higher baseline HbF tended to have higher values of MTD HbF.)", "entity_type": "UNKNOWN", "id": "COMPARISON"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "A decrease in baseline HbF values correlated with an increase in delta HbF, indicating a potentially compensatory mechanism.)", "entity_type": "UNKNOWN", "id": "DECREASE INCREASE"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf<SEP>chunk-814fe155e9cb953da61f6a79d1d25784", "description": "Due to its extremely low frequency, this specific SNP could not be included in further analysis within the cohort.)<SEP>The proportion of individuals carrying each allele at a given locus in a population, which was analyzed in studies on BDNF and migraine.", "entity_type": "GENETIC DATA", "id": "ALLELE FREQUENCY"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "Due to its extremely low frequency, this specific SNP could not be included in further analysis within the cohort.)", "entity_type": "UNKNOWN", "id": "EXCLUSION"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "These SNPs demonstrated adherence to HWE principles, thus facilitating their inclusion in subsequent regression analysis.)", "entity_type": "UNKNOWN", "id": "CONSISTENCY"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "These SNPs demonstrated adherence to HWE principles, thus facilitating their inclusion in subsequent regression analysis.)", "entity_type": "UNKNOWN", "id": "EQUILIBRIUM TEST"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "SNP analysis through regression highlighted correlations with several hematological values before and after hydroxyurea treatment.)", "entity_type": "UNKNOWN", "id": "HEMATOLOGICAL PARAMETERS"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "These specific variants of the BCL11A gene correlated positively with elevated levels of both baseline hemoglobin (Hb) and maximum tolerated dose hemoglobin (MTD Hb).)", "entity_type": "UNKNOWN", "id": "HEMOGLOBIN PARAMETERS"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "These specific variants of the BCL11A gene correlated positively with elevated levels of both baseline hemoglobin (Hb) and maximum tolerated dose hemoglobin (MTD Hb).)", "entity_type": "UNKNOWN", "id": "INCREASED LEVEL"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "The association of these BCL11A SNPs included changes in hemoglobin levels following hydroxyurea exposure.)", "entity_type": "UNKNOWN", "id": "CHANGE MEASURE"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "This strong linkage suggests that HMIP-2 SNPs are closely linked and likely inherited together, affecting shared genetic functions related to hemoglobin levels.)", "entity_type": "UNKNOWN", "id": "CHROMOSOMAL LOCATION"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "This strong linkage suggests that HMIP-2 SNPs are closely linked and likely inherited together, affecting shared genetic functions related to hemoglobin levels.)", "entity_type": "UNKNOWN", "id": "PROXIMITY"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "Regular assessment of ANC provided insights into the effects of HU treatment on the immune system\'s granulocyte component.)", "entity_type": "UNKNOWN", "id": "MONITORING"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "Regular assessment of ANC provided insights into the effects of HU treatment on the immune system\'s granulocyte component.)", "entity_type": "UNKNOWN", "id": "PARAMETER EVALUATION"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "This comprehensive genetic association testing aimed to identify specific genetic markers influencing hematological responses.)", "entity_type": "UNKNOWN", "id": "GENE-SPECIFIC ANALYSIS"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "This comprehensive genetic association testing aimed to identify specific genetic markers influencing hematological responses.)", "entity_type": "UNKNOWN", "id": "MULTIPLE TESTING"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "Strong LD suggests these genetic variants share a recent evolutionary history, highlighting their significance in hematological traits.)", "entity_type": "UNKNOWN", "id": "ANCESTRAL ORIGIN"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "Strong LD suggests these genetic variants share a recent evolutionary history, highlighting their significance in hematological traits.)", "entity_type": "UNKNOWN", "id": "GENETIC CONFIGURATION"}, {"source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "description": "Haplotype analysis allowed for the identification of specific genetic combinations that significantly influence hemoglobin levels in response to hydroxyurea treatment.)", "entity_type": "UNKNOWN", "id": "ENHANCED/REDUCED EFFECT"}, {"source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "description": "The significance of HMIP-2 SNP or haplotype association with MTD HbF dropped upon considering baseline HbF levels.)", "entity_type": "UNKNOWN", "id": "HMIP-2 SNP/HAPLOTYPE"}, {"source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "description": "No link was found between specific HMIP-2 haplotypes or SNPs and Delta HbF based on our analysis.)", "entity_type": "UNKNOWN", "id": "HMIP-2 HAPLOTYPE/SNP"}, {"entity_type": "DISEASE", "description": "Fanconi anemia is a genetic disorder characterized by increased sensitivity to certain chemicals, including diethylnitrosourea (DEN) and mitomycin C (MMC), which cause DNA damage and result in chromosomal breakage.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "FA"}, {"entity_type": "GENE", "description": "The expression of wild type UBE2T may be due to recombination of the maternally derived allele.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "clusters": "[{level: 0, cluster: 6}]", "id": "WILD TYPE UBE2T"}, {"entity_type": "CELLULAR SAMPLE", "description": "RA2627 fibroblasts were used to study the effects of UBE2T deficiency on cellular defects related to Fanconi anemia.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "id": "RA2627 FIBROBLASTS"}, {"entity_type": "BIOLOGICAL PHENOMENON/DISEASE/CONDITION", "description": "RA2627 cellular defects were characterized by hypersensitivity to various crosslinking agents, which were associated with Fanconi anemia.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "id": "FA PHENOTYPES"}, {"entity_type": "PROTEIN/MOLECULE", "description": "These proteins were involved in the monoubiquitination process, which was restored by UBE2T overexpression.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "id": "FANCD2 AND FANCI"}, {"entity_type": "BIOLOGICAL STRUCTURE/MOLECULAR COMPLEX<|THE FORMATION OF FANCD2 FOCI WAS INFLUENCED BY UBE2T EXPRESSION, INDICATING ITS ROLE IN THE DNA DAMAGE REPAIR PATHWAY WHEN ASSOCIATED WITH FANCONI ANEMIA.)", "description": "6", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "clusters": "[{level: 0, cluster: 6}]", "id": "FANCD2 FOCI"}, {"entity_type": "PHYSIOLOGICAL STATE/CONDITION", "description": "The increased sensitivity of RA2627 cells to crosslinking agents was a manifestation of FA phenotype.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "id": "CELLULAR HYPERSENSITIVITY"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "The overexpression of wild type UBE2T in RA2627 fibroblasts rescued the cellular hypersensitivity to crosslinking agents like MMC, DEB and cisplatin.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 6}]", "id": "RESCUE"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "UBE2T expression restored cell cycle distribution in G2 phase after treatment with MMC.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 6}]", "id": "CELL CYCLE"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "The overexpression of wild type UBE2T restored monoubiquitination of FANCD2 and FANCI proteins following treatment with MMC.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 6}]", "id": "MONOUBIQUITINATION OF PROTEINS"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "The overexpression of wild type UBE2T restored sensitivity of RA2627 cells to crosslinking agents following treatment.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 6}]", "id": "CROSSLINKING AGENTS"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "The overexpression of wild type UBE2T in RA2627 cells was able to rescue or \'rescue\' FA phenotypes due to its deficiency.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 6}]", "id": "FANCONI ANEMIA"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "The expression of wild type UBE2T facilitated the restoration of the monoubiquitination activity on these proteins after treatment with MMC.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 6}]", "id": "MONOUBIQUITINATION OF FANCD2 AND FANCI"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "The overexpression of wild type UBE2T improved or \'rescued\' the cellular hypersensitivity characteristic of Fanconi anemia cells.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 6}]", "id": "CELLULAR SENSITIVITY"}, {"entity_type": "DISEASE", "description": "Results from whole genome expression profiling study using Affymetrix GeneChip Human Genome U133A 2.0 Arrays on dermal fibroblasts isolated from four DBA patients carrying mutations in RPS19.<SEP>The condition being studied, with differential gene expression patterns identified via both qRT-PCR and microarray analysis.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94<SEP>chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "DBA"}, {"entity_type": "GENE", "description": "Defects associated with whole genome expression profiling study in DBA patients.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "RPS19"}, {"entity_type": "MICROARRAY", "description": "Used for the whole genome expression profiling study on DBA patients\' dermal fibroblasts.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "AFFYMETRIX GENECHIP HUMAN GENOME U133A 2.0 ARRAYS"}, {"entity_type": "CELL TYPE", "description": "Cells isolated from DBA patients used for gene expression analysis.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "DERMAL FIBROBLASTS"}, {"entity_type": "MICROARRAY PROBES", "description": "13,396 \'Present\' detection calls with intensity values more than 100 used for data analysis from Affymetrix GeneChip Human Genome U133A 2.0 Arrays.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "PROBE"}, {"entity_type": "QUANTITATIVE RT-PCR", "description": "A quantitative real-time reverse transcriptase-polymerase chain reaction technique for validating microarray data results.<SEP>Method used for confirmation of reduced RPS19 levels in patients carrying the frameshift mutation c.58delG.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94<SEP>chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "QRT-PCR"}, {"entity_type": "GENOMIC VARIANT OR MOLECULAR INTERACTION", "description": "Analysis performed on samples from DBA patients carrying various mutations in RPS19, including missense (p.Arg62Trp and p.Arg101His), frameshift mutation (c.58delG), and splice site mutation (c.1-1G>A).", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "MUTATION ANALYSIS"}, {"entity_type": "METABOLIC PATHWAY OR BIOLOGICAL PROCESS", "description": "Pathway identified as being enriched with genes in the differential expression dataset.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "AMINOACYL-TRNA BIOSYNTHESIS"}, {"entity_type": "METABOLIC PATHWAY OR BIOLOGICAL PROCESS", "description": "Another pathway significantly associated with DBA patients through gene enrichment analysis.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "GLYCINE, SERINE AND THREONINE METABOLISM"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Pathway identified in the context of impaired biological processes detected in DBA patients.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "DEATH RECEPTOR SIGNALING"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "Role in interferon induction and antiviral response found to be involved through analysis of molecular functions of annotated genes.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "PKR ACTIVITY"}, {"entity_type": "MOLECULAR FUNCTION OR BIOLOGICAL PROCESS", "description": "Category identified through pathway enrichment analysis related to biological function in DBA patients.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "RAR ACTIVATION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Enrichment cluster of gene clusters with roles in protein synthesis, including a large cluster of RP genes.<SEP>Protein synthesis is a fundamental physiological process for creating proteins necessary for cell function and growth.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94<SEP>chunk-41c7131af28e2a745ef6b88b8601860f", "id": "PROTEIN SYNTHESIS"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "A process involving cell death is one of the biological functions mentioned.<SEP>Role identified through molecular function analysis of annotated genes related to cellular development and biological pathways involving cell death.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94<SEP>chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "CELL DEATH"}, {"entity_type": "BIOLOGICAL PROCESS OR METABOLIC PATHWAY", "description": "Enrichment cluster found in the context of molecular functions and biological processes associated with DBA patients.<SEP>Lipid metabolism is a biological pathway within the \'Top Bio Functions\' category.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94<SEP>chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "LIPID METABOLISM"}, {"entity_type": "BIOLOGICAL PROCESS OR MOLECULAR FUNCTION", "description": "Cluster identified as part of enriched gene functions related to transport mechanisms in DBA patients.<SEP>Molecular transport activities are part of the functions described.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94<SEP>chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "MOLECULAR TRANSPORT"}, {"entity_type": "BIOLOGICAL PATHWAY OR PROCESS", "description": "Category identified through pathway enrichment analysis, encompassing multiple biological functions involved in DBA pathogenesis.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "TOP BIO FUNCTIONS"}, {"entity_type": "RESEARCH APPROACH OR FIELD", "description": "The study employs functional genomics to understand the molecular basis of disease states like DBA, focusing on RPS19 mutations and their impact.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "FUNCTIONAL GENOMICS"}, {"entity_type": "HIGH-THROUGHPUT SEQUENCING TECHNOLOGY", "description": "Method used for transcriptome profiling which provides comprehensive information about gene expression in DBA patients.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "RNA SEQUENCING"}, {"entity_type": "STATISTICAL METHOD OR DATA INTERPRETATION TOOL", "description": "Analytical tool applied to identify significantly overrepresented biological pathways, functions, and molecular interactions from the dataset of differentially expressed genes.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "ENRICHMENT ANALYSIS"}, {"entity_type": "BIOLOGICAL PROCESS IDENTIFICATION", "description": "Process identifying pathways that are significantly enriched with a large number of genes differentially expressed in DBA patients compared to controls.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "PATHWAY ENRICHMENT"}, {"entity_type": "GROUP OR CATEGORY OF GENES", "description": "Cluster of RiboSomes (RP) related genes identified as having significant roles in protein synthesis, potentially affected by mutations in RPS19.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "RP GENES CLUSTER"}, {"entity_type": "BIOLOGICAL PROCESS OR PATHWAY", "description": "Development processes are part of the \'Top Bio Functions\' category.<SEP>Function associated with cell division, growth, and differentiation that might be disrupted in DBA patients according to the study findings.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94<SEP>chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "CELLULAR DEVELOPMENT"}, {"entity_type": "PATHWAYS OR FUNCTIONS AFFECTED", "description": "Specific biological processes or pathways found to be impaired due to gene expression alterations in DBA patients as identified through pathway analysis.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "IMPAIRED BIOLOGICAL PROCESSES"}, {"entity_type": "GENE", "description": "A cluster of genes, including RPL22, involved in synthesis related to RP genes.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL22"}, {"entity_type": "GENE", "description": "Another gene within the same cluster as RPL22 and other RP genes.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL27A"}, {"entity_type": "GENE", "description": "Included in a cluster of RP genes with roles in synthesis processes.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL29"}, {"entity_type": "GENE", "description": "A member of an RP gene cluster, part of the synthesis process.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL31"}, {"entity_type": "GENE", "description": "Clustered with other RP genes, contributing to synthesis and cellular functions.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL14"}, {"entity_type": "GENE", "description": "Included in a group of RP genes known for their roles in synthesis processes.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL18A"}, {"entity_type": "GENE", "description": "Part of a cluster of RP genes with involvement in synthesis related activities.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPS3"}, {"entity_type": "GENE", "description": "Clustered with other RP genes involved in cellular development and lipid metabolism.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL18"}, {"entity_type": "GENE", "description": "A gene included in clusters related to molecular functions and metabolic processes.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL13"}, {"entity_type": "GENE", "description": "Part of an RP gene cluster, associated with roles including synthesis and transport processes.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL3"}, {"entity_type": "GENE", "description": "Included within a group of RP genes with potential involvement in synthesis-related biological pathways.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL34"}, {"entity_type": "GENE", "description": "Clustered with other RP genes that play roles in cellular development, lipid metabolism and molecular transport.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPS2"}, {"entity_type": "GENE", "description": "A member of a group of RP genes known for their functions related to synthesis processes.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL15"}, {"entity_type": "GENE", "description": "Included within clusters associated with RP genes that are involved in synthesis and cellular metabolism.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL28"}, {"entity_type": "DISEASE", "description": "Identified as one of the diseases and disorders in the \'Top Bio Functions\' category.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "CANCER"}, {"entity_type": "DISEASE", "description": "A haematological disorder is among those listed under diseases associated with molecular functions.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "HAEMATOLOGICAL DISORDER"}, {"entity_type": "DISEASE", "description": "An immunological disorder was found in the list of identified diseases and disorders.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "IMMUNOLOGICAL DISORDER"}, {"entity_type": "DISEASE", "description": "A renal disorder is one of those linked to the \'Top Bio Functions\' category.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RENAL DISORDER"}, {"entity_type": "DISEASE", "description": "A skeletal disorder was part of the diseases listed in relation to molecular and cellular functions.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "SKELETAL DISORDER"}, {"entity_type": "DEVELOPMENT PROCESS", "description": "Processes related to embryonic development were observed in gene clusters.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "EMBRYONIC DEVELOPMENT"}, {"entity_type": "TISSUE FUNCTION", "description": "Functions associated with skeletal tissues are mentioned within the \'Top Bio Functions\' category.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "SKELETAL TISSUE FUNCTION"}, {"entity_type": "TISSUE FUNCTION", "description": "Function of connective tissue was observed in the identified gene clusters and biological pathways.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "CONNECTIVE TISSUE FUNCTION"}, {"entity_type": "SUBJECT", "description": "Patients with a condition studied for differential gene expression patterns.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "DBA PATIENTS"}, {"entity_type": "COMPARISON GROUP", "description": "The control group used for comparison with DBA patients to validate microarray data results.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "CONTROL GROUP"}, {"entity_type": "GENE", "description": "One of the genes tested via qRT-PCR that was found differentially expressed in DBA patients.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "AMPD3"}, {"entity_type": "GENE", "description": "Another gene from a differential expression study, up-regulated in DBA patients by qRT-PCR and microarray analysis.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "CCND2"}, {"entity_type": "GENE", "description": "Gene found to be up-regulated in DBA patients through both qRT-PCR validation and microarray data.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "SOD2"}, {"entity_type": "GENE", "description": "Up-regulated gene in DBA patients, confirmed by qRT-PCR and corroborated with microarray expression results.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "TNFAIP3"}, {"entity_type": "GENE", "description": "Down-regulated gene in DBA patients validated through qRT-PCR and microarray analysis for differential expression.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "COMP"}, {"entity_type": "GENE", "description": "Another down-regulated gene in DBA patients, confirmed with qRT-PCR and microarray data as part of the study\'s validation process.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "WARS"}, {"entity_type": "GENE", "description": "Down-regulated in DBA patients by both qRT-PCR and microarray analysis, verified for differential expression.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "ZIC1"}, {"entity_type": "ANALYSIS METHOD", "description": "The method used to assess gene expression patterns before qRT-PCR validation.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "MICROARRAY DATA"}, {"entity_type": "CONDITION", "description": "The condition of interest being studied through gene expression analysis.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "DBA CONDITION"}, {"entity_type": "ANALYSIS METHOD", "description": "The primary method used to determine differential gene expression in DBA patients.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "MICROARRAY ANALYSIS"}, {"entity_type": "VALIDATION TECHNIQUE", "description": "An additional technique employed to confirm the findings from microarray data.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "QRT-PCR VALIDATION"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "The process of identifying statistically significantly differentially expressed genes/probes from the microarray study.)", "entity_type": "UNKNOWN", "id": "DIFFERENTIAL EXPRESSION"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Comparative analysis between DBA patients and controls in gene expression profiling.)", "entity_type": "UNKNOWN", "id": "COMPARISON GROUP OR STATISTICAL TEST"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Comparative analysis between DBA patients and controls in gene expression profiling.)", "entity_type": "UNKNOWN", "id": "DBA PATIENTS VS. CONTROLS"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Methodology used to identify differentially expressed genes through the study of gene expression across the entire genome.)", "entity_type": "UNKNOWN", "id": "GENE EXPRESSION PROFILING"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Methodology used to identify differentially expressed genes through the study of gene expression across the entire genome.)", "entity_type": "UNKNOWN", "id": "WHOLE GENOME TECHNOLOGY"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Technology platform used for high-throughput screening and analysis of gene expression from the DBA patients\' samples.)", "entity_type": "UNKNOWN", "id": "MICROARRAY PLATFORM"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "The sample source from which gene expression data was collected and analyzed through microarray and qRT-PCR methods.)", "entity_type": "UNKNOWN", "id": "DERMAL FIBROBLAST SAMPLES"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "The sample source from which gene expression data was collected and analyzed through microarray and qRT-PCR methods.)", "entity_type": "UNKNOWN", "id": "STUDY SUBJECT OR TISSUE SAMPLE"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Method employed to validate findings of reduced RPS19 levels in DBA patients carrying the frameshift mutation, alongside other genomic variants.)", "entity_type": "UNKNOWN", "id": "QRT-PCR CONFIRMATION"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Method employed to validate findings of reduced RPS19 levels in DBA patients carrying the frameshift mutation, alongside other genomic variants.)", "entity_type": "UNKNOWN", "id": "VALIDATION TECHNIQUE"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Specific mutation analyzed through PCR and sequencing methods to elucidate effects on gene expression and protein synthesis pathways in DBA patients.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT OR MOLECULAR INTERACTION"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Specific mutation analyzed through PCR and sequencing methods to elucidate effects on gene expression and protein synthesis pathways in DBA patients.)", "entity_type": "UNKNOWN", "id": "INTRON 1 SPLICE SITE MUTATION C.1-1G>A"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Pathway identified as significantly enriched with genes from the differential expression dataset, highlighting its role in DBA pathogenesis.)", "entity_type": "UNKNOWN", "id": "AMINOACYL-TRNA BIOSYNTHESIS PATHWAY"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Pathway identified as significantly enriched with genes from the differential expression dataset, highlighting its role in DBA pathogenesis.)", "entity_type": "UNKNOWN", "id": "ENRICHMENT CLUSTER OF GENES"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Further pathway enrichment identified through analysis of molecular functions related to metabolic pathways involved in DBA patients.)", "entity_type": "UNKNOWN", "id": "BIOLOGICAL PATHWAY OR ENRICHMENT CLUSTER"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Specific pathway impacted and analyzed as part of impaired biological processes detected in the study on DBA patients.)", "entity_type": "UNKNOWN", "id": "PATHWAY OR BIOLOGICAL PROCESS"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Analysis revealed the involvement of PKR activity in antiviral responses through enriched gene functions identified in DBA patients.)", "entity_type": "UNKNOWN", "id": "ROLE IN INTERFERON INDUCTION"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Category associated with biological processes and pathways related to retinoid acid receptor activation found significant through enrichment analysis in the study on DBA patients.)", "entity_type": "UNKNOWN", "id": "BIOLOGICAL FUNCTION OR PATHWAY ENRICHMENT"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Cluster identified for gene functions involved in protein synthesis, including a large group of RP genes contributing to this pathway\'s significance in DBA.)", "entity_type": "UNKNOWN", "id": "GENE ENRICHMENT CATEGORY"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Cluster identified for gene functions involved in protein synthesis, including a large group of RP genes contributing to this pathway\'s significance in DBA.)", "entity_type": "UNKNOWN", "id": "PROTEIN SYNTHESIS CLUSTER"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Enrichment cluster found related to roles in cellular development and death pathways identified through molecular function analysis for DBA patients.)", "entity_type": "UNKNOWN", "id": "BIOLOGICAL PROCESS OR FUNCTION"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Enrichment cluster found related to roles in cellular development and death pathways identified through molecular function analysis for DBA patients.)", "entity_type": "UNKNOWN", "id": "CELL DEATH ROLE"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Cluster associated with lipid metabolism processes as part of the biological mechanisms significantly impacted in the study on DBA patients.)", "entity_type": "UNKNOWN", "id": "LIPID METABOLISM ENRICHMENT"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Cluster associated with lipid metabolism processes as part of the biological mechanisms significantly impacted in the study on DBA patients.)", "entity_type": "UNKNOWN", "id": "METABOLIC PATHWAY OR BIOLOGICAL PROCESS"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Enriched category related to molecular transport functions contributing to biological pathways involved in DBA pathogenesis, as identified through analysis of gene functions and annotations.)", "entity_type": "UNKNOWN", "id": "BIOLOGICAL FUNCTION OR PATHWAY"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Enriched category related to molecular transport functions contributing to biological pathways involved in DBA pathogenesis, as identified through analysis of gene functions and annotations.)", "entity_type": "UNKNOWN", "id": "MOLECULAR TRANSPORT CLUSTER"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Category encompassing multiple biological functions identified as critical for the pathogenesis of DBA patients through pathway enrichment analysis performed on the dataset of differentially expressed genes.)(entity", "entity_type": "UNKNOWN", "id": "PATHWAY ENRICHMENT OR GROUP OF BIOLOGICAL FUNCTIONS"}, {"source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "description": "Category encompassing multiple biological functions identified as critical for the pathogenesis of DBA patients through pathway enrichment analysis performed on the dataset of differentially expressed genes.)(entity", "entity_type": "UNKNOWN", "id": "TOP BIO FUNCTIONS CATEGORY"}, {"source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "description": "Part of a cluster including other RP genes involved in synthesis related processes.)", "entity_type": "UNKNOWN", "id": "CLUSTER OF GENES"}, {"source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "description": "Up-regulated gene identified as differentially expressed by qRT-PCR and microarray analysis in DBA patients.)", "entity_type": "UNKNOWN", "id": "DIFFERENTIALLY EXPRESSED GENE"}, {"source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "description": "Method used to validate the results of microarray analysis for gene expression in DBA patients.)", "entity_type": "UNKNOWN", "id": "VALIDATION METHOD"}, {"entity_type": "GENOMIC VARIANT", "description": "The BDNF rs6265 (G/A) polymorphism is a genetic variant analyzed in studies on migraine.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "BDNF RS6265 (G/A) POLYMORPHISM"}, {"entity_type": "DISEASE", "description": "Migraine is a neurological disorder characterized by recurrent headaches.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "MIGRAINE"}, {"entity_type": "GENE", "description": "The BDNF gene encodes the brain-derived neurotrophic factor protein and was studied for its role in migraine.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "BDNF GENE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The rs6265 polymorphism occurs within the BDNF gene, specifically affecting the G/A variant.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "RS6265 (G/A) POLYMORPHISM"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Research on migraine and BDNF polymorphisms focuses on understanding risk factors for the disease.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "MIGRAINE RISK"}, {"entity_type": "STATISTICAL METHOD", "description": "The meta-analysis used fixed-effects models to account for heterogeneity among studies.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "HETEROGENEITY IN STUDIES"}, {"entity_type": "META-ANALYSIS APPROACH", "description": "A statistical method employed when studies are homogeneous, as determined by a non-significant heterogeneity test.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "FIXED-EFFECTS MODEL"}, {"entity_type": "ASSOCIATION ANALYSIS", "description": "Studies were analyzed for associations between the BDNF rs6265 polymorphism and migraine risk.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "MIGRAINE ASSOCIATION"}, {"entity_type": "STATISTICAL MODEL", "description": "An analysis method used to assess the relationship between a genetic variant and an outcome by considering all genotypes in one group.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "ALLELE MODEL"}, {"entity_type": "GENETIC MODEL", "description": "A model that evaluates whether having at least two copies of the recessive allele affects disease risk.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "RECESSIVE MODEL"}, {"entity_type": "STATISTICAL METHODOLOGY", "description": "A model used to evaluate if there\'s an additive effect of alleles on the outcome, considering all possible genotypes.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "ADDITIVE MODEL"}, {"entity_type": "GENETIC ANALYSIS", "description": "An analysis assessing whether having at least one copy of a specific allele confers risk.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "DOMINANT MODEL"}, {"entity_type": "STUDY POPULATION", "description": "Cohorts from Australia that contributed to the studies on BDNF gene.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "AUSTRALIAN-BASED INDEPENDENT COHORTS"}, {"entity_type": "STATISTICAL TEST", "description": "A measure of population genetics used in genetic studies, indicating if the observed genotype frequencies are consistent with those expected under random mating and no other forces acting on the population.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "HARDY-WEINBERG EQUILIBRIUM (HWE)"}, {"entity_type": "QUALITY MEASURE", "description": "An indicator assessing the quality or risk of bias in research articles; scores \u22656 indicate high-quality studies.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "NOS SCORE"}, {"entity_type": "GENETIC INFORMATION", "description": "Specific genetic configurations found within the BDNF gene that were studied for their association with migraine.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "BDNF GENOTYPE"}, {"entity_type": "STUDY DESIGN", "description": "A type of observational study where cases (people with the disease) are compared to controls (people without the disease), used here for understanding genetic risk factors like BDNF polymorphisms.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "CASE-CONTROL STUDIES"}, {"entity_type": "DATABASE SOURCES", "description": "Academic databases including peer-reviewed articles and clinical evidence, utilized in literature searches to find studies on BDNF gene-polymorphism relation with migraine.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "COCHRANE DATABASES"}, {"entity_type": "DATABASE SOURCES", "description": "An interdisciplinary citation indexing service used for scholarly research that was searched alongside PubMed and Cochrane to identify relevant studies.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "WEB OF SCIENCE"}, {"source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "description": "The rs2049046 polymorphism was significantly linked to migraine.)", "entity_type": "UNKNOWN", "id": "RS2049046 (A/T) POLYMORPHISM"}, {"source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "description": "Polymorphisms in the BDNF gene were studied as potential risk factors for developing migraine.)", "entity_type": "UNKNOWN", "id": "BDNF GENE POLYMORPHISMS"}, {"source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "description": "Polymorphisms in the BDNF gene were studied as potential risk factors for developing migraine.)", "entity_type": "UNKNOWN", "id": "MIGRAINE RISK FACTOR"}, {"source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "description": "The specific G/A variant of the polymorphism was studied within its genotype context, linking it to migraine risk.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION WITH BDNF GENOTYPE"}, {"source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "description": "Another genetic variant that may interact or have an additive effect alongside the rs6265 polymorphism in influencing migraine.)", "entity_type": "UNKNOWN", "id": "GENETIC INTERACTION WITH RS6265"}, {"entity_type": "GENE", "description": "The genetic disposition of Patient 1 is influenced by a positive family history for psoriasis and HS among first-degree relatives.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "PATIENT 1"}, {"entity_type": "GENE", "description": "Likewise, the genetic background of Patient 2 includes a positive family history for psoriasis.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "PATIENT 2"}, {"entity_type": "GENE", "description": "Patient 3\'s genetic makeup also carries a positive family history for HS and psoriasis, as seen in his first-degree relatives.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "PATIENT 3"}, {"entity_type": "DISEASE", "description": "Patients 1, 2, and 3 are affected by severe hidradenitis suppurativa (HS), which is associated with the development of paradoxical psoriasiform reactions following treatment with adalimumab.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "HS"}, {"entity_type": "DRUG", "description": "Adalimumab, an anti-TNF-\u03b1 therapy, was used for patients 1 and 2 but led to paradoxical psoriasiform reactions in these HS patients.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "ADALIMUMAB"}, {"entity_type": "THERAPY", "description": "The use of the anti-TNF-\u03b1 treatment is related to the development of paradoxical psoriasiform reactions as seen in patients undergoing this therapy.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "ANTI-TNF-\u0391"}, {"entity_type": "DISEASE", "description": "Patients 1 and 2 developed psoriasiform eruptions after starting adalimumab, indicating a paradoxical reaction.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "PSORIASIFORM ERUPTIONS"}, {"entity_type": "DISEASE SCORE", "description": "Patient 1\'s psoriasis severity score (PASI) was at 6.8 after the onset of psoriasiform eruptions following treatment.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "PASI 6.8"}, {"entity_type": "DISEASE SCORE", "description": "The PASI score for patient 2 was slightly lower than that of Patient 1, yet still indicative of a paradoxical psoriasiform reaction and dermatitis.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "PASI 5.2"}, {"entity_type": "DISEASE MEASURE", "description": "Patient 1\'s hidradenitis suppurativa was scored at Hurley III, indicating significant disease severity.<SEP>Patient 2 showed a similar pattern of HS severity as Patient 1, scoring the same Hurley III Sartorius score.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "HURLEY III, SARTORIUS SCORE: 41.5"}, {"entity_type": "DISEASE MEASURE", "description": "Patient 3\'s severe hidradenitis suppurativa was characterized by a high Hurley III Sartorius score of 61.5.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "HURLEY III, SARTORIUS SCORE: 61.5"}, {"entity_type": "CELL", "description": "Enriched CD8+, but not CD4+, compared to PBMCs from the same patients.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "CD3+ CELLS"}, {"entity_type": "DISEASE", "description": "Showed an enrichment of both CD4+ and CD8+ subpopulations in skin T cells.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "CLASSICAL PSORIASIS"}, {"entity_type": "CYTOKINE", "description": "Expressed significantly more in paradoxical skin lesions than classical psoriasis, mainly in keratinocytes.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "TYPE I IFN-\u0392"}, {"entity_type": "CELL", "description": "Express IFN-\u03ba at levels comparable to classical and nonclassical psoriasis.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "KERATINOCYTES"}, {"entity_type": "CYTOKINE", "description": "Cytoplasmic expression in psoriasiform lesions, membrane-bound in classical psoriasis.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "IFN-\u039a"}, {"entity_type": "CYTOKINE", "description": "Deregulated by anti-TNF-\u03b1 therapy and strongly overexpressed in paradoxical skin reactions.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "LYMPHOTOXINS"}, {"entity_type": "GENE", "description": "All HS patients showed variants, either homozygosis or heterozygosis.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "ERAP1"}, {"entity_type": "ALLELE", "description": "HS patients mostly show genetic polymorphisms related to HLA-Cw6.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "HLA-C REGION"}, {"entity_type": "SNP", "description": "Three specific SNPs found in all patients, with distribution varying among them.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "ERAP1 SNPS"}, {"entity_type": "SNP", "description": "Encoding A20 protein, interestingly, patient 3 shared multiple SNPs with others.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "TNFAIP SNPS"}, {"entity_type": "SNP", "description": "Polymorphisms related to skin-barrier proteins.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "CDSN AND CCHCR1 SNPS"}, {"entity_type": "CYTOKINE", "description": "One of several cytokines analyzed for genetic variants associated with psoriasis.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "IL-17F"}, {"entity_type": "RECEPTOR", "description": "Cytokine receptor potentially involved in psoriasis pathogenesis; showed genetic variants among HS patients.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "IL-17RA"}, {"entity_type": "GENE", "description": "SNP rs11209026 previously associated with paradoxical psoriasiform reactions to anti-TNFs, but not found in all HS patients.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "IL-23R"}, {"entity_type": "GENE", "description": "Pathogenic cytokine gene involved in immune responses and inflammatory conditions; showed genetic polymorphisms among HS patients.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "ILB12B"}, {"entity_type": "PROTEIN", "description": "Encoded by the IKB\u03b6 gene, involved in signal transduction pathways related to inflammation; shared SNPs with other psoriasis-related genes.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "IKB\u0396"}, {"entity_type": "PROTEIN", "description": "Part of the A20 protein complex; genetic variants present among HS patients associated with paradoxical reactions.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "A20 BINDING PROTEIN"}, {"entity_type": "GENE", "description": "Catalytic subunit of the Janus kinase (JAK) family involved in signal transduction pathways, showed polymorphisms among HS patients.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "TYK2"}, {"entity_type": "GENE", "description": "Coding for a skin-barrier protein; two specific SNPs were identified among the analyzed HS patients.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "CDSN"}, {"entity_type": "GENE", "description": "Protein involved in skin physiology and barrier function; three specific SNPs identified, potentially related to paradoxical reactions.<SEP>Three SNPs (rs1576, rs130079, rs746647) are present in the CCHCR1 gene as mentioned in Table 1.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a<SEP>chunk-72f4505e43f67f3564607392cb178656", "id": "CCHCR1"}, {"entity_type": "GENE", "description": "The IL23R gene is discussed with two SNPs associated with paradoxical psoriasiform reactions to anti-TNFs and identified in specific patients.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "IL23R"}, {"entity_type": "GENOMIC VARIANT", "description": "Previously, rs11209026 in the IL23R gene has been linked to paradoxical psoriasiform reactions with anti-TNFs as stated.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 36}, {level: 2, cluster: 60}]", "id": "RS11209026"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs72676067 was detected in patient 2 in the context of IL23R gene study.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "RS72676067"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs1004819 was found associated with the IL23R gene and identified in patient 1.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "RS1004819"}, {"entity_type": "DISEASE", "description": "Referenced table provides information on SNPs of interest within the CCHCR1 and IL23R genes.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "TABLE 1"}, {"source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "description": "The development of paradoxical psoriasiform reactions occurred following treatment with adalimumab in patients affected by severe HS.)", "entity_type": "UNKNOWN", "id": "PARADOXICAL PSORIASIFORM REACTIONS"}, {"source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "description": "Each patient displayed unique disease responses including erythematous-pustular lesions, pustular lesions, and erythemato-scaly plaques respectively after therapy.)", "entity_type": "UNKNOWN", "id": "DISEASE RESPONSE"}, {"source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "description": "Each patient displayed unique disease responses including erythematous-pustular lesions, pustular lesions, and erythemato-scaly plaques respectively after therapy.)", "entity_type": "UNKNOWN", "id": "PATIENT 1, PATIENT 2, AND PATIENT 3"}, {"source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "description": "The anti-TNF-\u03b1 treatment led to distinct paradoxical reactions of varying severity in HS patients as measured by PASI scores.)", "entity_type": "UNKNOWN", "id": "REACTIONS"}, {"source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "description": "The psoriasis severity index (PASI) indicated a range of disease severity in the patients with paradoxical reactions.)", "entity_type": "UNKNOWN", "id": "DISEASE SEVERITY SCALE"}, {"source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "description": "The psoriasis severity index (PASI) indicated a range of disease severity in the patients with paradoxical reactions.)", "entity_type": "UNKNOWN", "id": "PASI 6.8, PASI 5.2, AND PASI 5.6"}, {"source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "description": "The Hurley III Sartorius scoring system was used to measure the severity of hidradenitis suppurativa in each patient.)", "entity_type": "UNKNOWN", "id": "DISEASE SEVERITY MEASURE"}, {"source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "description": "The Hurley III Sartorius scoring system was used to measure the severity of hidradenitis suppurativa in each patient.)", "entity_type": "UNKNOWN", "id": "HURLEY III, SARTORIUS SCORE: 41.5, HURLEY III, SARTORIUS SCORE: 41.5, AND HURLEY III, SARTORIUS SCORE: 61.5"}, {"source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "description": "In Patient 2, a mixed population of T cells responsible for both psoriasiform and eczematous reactions was likely present.)", "entity_type": "UNKNOWN", "id": "DISEASE SUBTYPE"}, {"source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "description": "In Patient 2, a mixed population of T cells responsible for both psoriasiform and eczematous reactions was likely present.)", "entity_type": "UNKNOWN", "id": "PATIENT 2\'S ECZEMATOUS REACTIONS"}, {"source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "description": "Overexpression of type I IFNs is characteristic in paradoxical psoriasis, more than classical or nonclassical.)", "entity_type": "UNKNOWN", "id": "PARADOXICAL PSORIASIS"}, {"source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "description": "Deregulated by anti-TNF-\u03b1 therapy and overexpressed specifically in paradoxical skin reactions.)", "entity_type": "UNKNOWN", "id": "ANTI-TNF-\u0391 THERAPY"}, {"source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "description": "All HS patients showed variants of ERAP1, with specific SNPs found across individuals.)", "entity_type": "UNKNOWN", "id": "ERAP1 VARIANTS"}, {"source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "description": "All HS patients showed variants of ERAP1, with specific SNPs found across individuals.)", "entity_type": "UNKNOWN", "id": "HS PATIENTS"}, {"source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "description": "Three specific ERAP1 SNPs were found in all HS patients with differing distributions across individuals.)", "entity_type": "UNKNOWN", "id": "SPECIFIC SNPS"}, {"source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "description": "Shared multiple TNFAIP-related SNPs among patients, indicating genetic predispositions to paradoxical reactions.)", "entity_type": "UNKNOWN", "id": "SNP DISTRIBUTION"}, {"entity_type": "STRUCTURE", "description": "The human FANCL central domain has a fused bilobal shape with specific dimensions consisting of four \u03b1-helices and nine \u03b2-strands.", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "FANCL CENTRAL DOMAIN"}, {"entity_type": "STRUCTURE", "description": "The Drosophila melanogaster (Dm) DRWD domain shares similarities in structure, particularly in the fusion of two lobes, containing a bilobal shape with similar dimensions as human FANCL central domain.", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "DRWD DOMAIN"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A hydrophobic groove exists between the last \u03b2-strand of the four-stranded \u03b2-meander and \u03b1-helix 3 in DmDRWD corresponding to residues Ile-243, Met-246, Leu-248, Leu-264, Leu-268, and Trp-271.<SEP>A hydrophobic groove formed by residues Phe-253, Val-260, and Leu-268 exists on the human URD but is extended compared to DmFANCL.<SEP>A small hydrophobic patch resides on the N-terminal lobe of the URD, involved in interactions between residues like Leu-149, Tyr-165, Phe-166, and Val-167.<SEP>An additional small hydrophobic patch is present on the N-terminal region of the URD corresponding to residues Tyr-112, Tyr-113, and Leu-116.<SEP>The URD has distinct electrostatic potential differences compared to DmDRWD which wraps around like a ring on the backside of central FANCL domain.<SEP>There is an additional small hydrophobic patch consisting of Trp-212, Val-213, and Leu-214 on the backside of the central FANCL domain.", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "URD"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "In Drosophila DRWD, the corresponding residues are His-208, Val-209, and Leu-210. This patch is partly occluded by the DRWD-RING loop in the full-length FANCL structure.<SEP>The C-terminal domain of DmDRWD consists of a hydrophobic groove between the last \u03b2-strand of the four-stranded \u03b2-meander and \u03b1-helix 3 which follows (residues Ile-243, Met-246, Leu-248, Leu-264, Leu-268, and Trp-271.", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "DRWD"}, {"entity_type": "MOLECULAR INTERACTION", "description": "An 11-fold decrease in binding affinity was observed between mutant Ube2T-F63A and RING as seen in ITC analysis.<SEP>ICL analysis of the RING domain reveals equilibrium binding dissociation constant (Kd) of 454 nm.<SEP>Initial size-exclusion chromatography results showed no change suggesting lack of complex formation between URD and Ube2T.<SEP>Purified RING domain shows a 1:1 stoichiometric interaction with Ube2T as judged by size-exclusion chromatography.", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "RING DOMAIN"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Mutation of Phe-63 in Ube2T is predicted to interact with the RING and no complex shift was observed when titrating it into RING using analytical gel filtration.", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "PHE-63"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Exposure of hydrophobic patches on the surface of URD is required for substrate binding.", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "FANCI, FANCD2"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Reduction in binding was observed with URD mutants compared to wild type when pulldown experiments were performed with FANCI and FANCD2.", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "MUTANTS L149A/F166A, L248A/F252A/L254A/I265A, V127A/Y128A, AND W212A/L214A"}, {"entity_type": "REGULATORY MECHANISM", "description": "Autophosphorylation at Thr-67 plays a critical role in the activation of FAM159A.<SEP>FAM159A exhibits tissue-specific expression profiles, which affect its function in different biological processes across various organs.<SEP>FAM159A forms a complex with components of the RING-H2 E3 ubiquitin ligase to regulate signaling pathways.<SEP>FAM159A is part of a pathway involved in the regulation of cell cycle checkpoints and DNA repair mechanisms.", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "FAM159A"}, {"entity_type": "MOLECULAR INTERACTION", "description": "An interaction site on PDPH7 is involved in its binding to FAM159A, characterized by residues Glu-46, Val-82, and Leu-83.<SEP>PDPH7 has a catalytic activity that facilitates the interaction with FAM159A under certain cellular conditions.", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "PHD FINGER PROTEIN 7 (PDPH7)"}, {"entity_type": "REGULATORY ROLE", "description": "PDPH7 is primarily localized within the nucleus, indicating its role in chromatin regulation and gene control mechanisms.<SEP>The interaction with FAM159A modulates PDPH7\'s ability to regulate histone modifications, influencing gene expression patterns.", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "PDPH7"}, {"entity_type": "GENETIC VARIANT", "description": "Mutations L248A/F252A/L254A/I265A reduce the binding of mutant proteins with FANCI and FANCD2, indicating that exposed hydrophobic patches are crucial for substrate interaction.", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "id": "L248A/F252A/L254A/I265A"}, {"entity_type": "GENETIC VARIANT", "description": "Mutations W212A/L214A result in reduced binding compared to wild type URD, suggesting these mutations affect the ability of hydrophobic patches on the C-terminal lobe.", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "id": "W212A/L214A"}, {"entity_type": "GENETIC VARIANT", "description": "Mutation L254A leads to a decrease in substrate-binding capacity with FANCI and FANCD2 proteins.", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "id": "L254A"}, {"entity_type": "GENETIC VARIANT", "description": "Mutation I265A influences the binding affinity of URD with FANCI and FANCD2, particularly when compared to wild type proteins.", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "id": "I265A"}, {"entity_type": "GENETIC VARIANT", "description": "Mutation W212A impacts the interaction with FANCI and FANCD2 proteins, affecting substrate binding as seen with L214A.", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "id": "W212A"}, {"entity_type": "PROTEIN", "description": "FANCI is a protein that interacts with FANCD2 in repair of DNA damage, particularly in the Fanconi Anemia pathway.", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "id": "FANCI"}, {"entity_type": "PROTEIN", "description": "FANCD2 is associated with FANCI as part of the core complex involved in homologous recombination for repairing chromosomal breakage.", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "id": "FANCD2"}, {"source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "description": "No change in migration was observed by size-exclusion chromatography suggesting no complex formation between URD and Ube2T.)", "entity_type": "UNKNOWN", "id": "UBE2T"}, {"source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "description": "A 11-fold decrease in binding affinity was observed between mutant Ube2T-F63A and RING as seen in ITC analysis.)", "entity_type": "UNKNOWN", "id": "PHE-63 MUTATION"}, {"source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "description": "L254A/", "entity_type": "UNKNOWN", "id": "L248A/"}, {"source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "description": "L254A/", "entity_type": "UNKNOWN", "id": "252A/"}, {"source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "description": "Hydrophobic Patches", "entity_type": "UNKNOWN", "id": "W212A/"}, {"source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "description": "Hydrophobic Patches", "entity_type": "UNKNOWN", "id": "L214A"}, {"source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "description": "The exposed hydrophobic patches on the C-terminal lobe are required for substrate binding as indicated by lower affinity with L254A mutation.)", "entity_type": "UNKNOWN", "id": "HYDROPHOBIC PATCHES"}, {"source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "description": "Binding affinity of URD proteins to FANCI and FANCD2 decreases with specific mutations (L248A/F252A/L254A/I265A, W212A/L214A), emphasizing the role of hydrophobic patches.)<SEP>Interaction of URD with FANCI/FANCD2 proteins depends on the exposed hydrophobic patches on its C-terminal lobe, which are impacted by certain genetic variants.", "entity_type": "UNKNOWN", "id": "FANCI/FANCD2"}, {"source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "description": "The mutations influence substrate binding affinity, with reduced interaction observed for URD proteins.", "entity_type": "UNKNOWN", "id": "BINDING AFFINITY"}, {"source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "description": "The mutations influence substrate binding affinity, with reduced interaction observed for URD proteins.", "entity_type": "UNKNOWN", "id": "MUTATIONS L248A/F252A/L254A/I265A/W212A/L214A"}, {"source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "description": "Exposure of hydrophobic patches on the C-terminal lobe is necessary for efficient interaction between URD and FANCI/FANCD2, as indicated by decreased binding affinity with specific mutations.", "entity_type": "UNKNOWN", "id": "SUBSTRATE INTERACTION"}, {"entity_type": "GENOMIC VARIANT", "description": "This genomic variant was assessed for Hardy-Weinberg equilibrium in healthy control patients. It did not show significant differential treatment effects associated with the study.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "FGFR4 RS376618"}, {"entity_type": "GENE", "description": "This functional genetic variant (Gly388Arg) showed association with differential colesevelam treatment effect on AC t1/2 and GC24.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "FGFR4 RS351855"}, {"entity_type": "DRUG", "description": "A medication that was tested for its differential treatment effects based on the genotypes of FGFR4 rs351855.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "COLESEVELAM"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The functional variant, FGFR4 rs351855 (Gly388Arg), affects colesevelam\'s treatment effect on specific outcomes in this study.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "FGFR4 GENOTYPE VARIATION"}, {"entity_type": "GENOMIC VARIANT", "description": "This genomic variant showed a differential treatment effect on CT compared to placebo, affecting GC24 values and AC t1/2.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "KLB RS4975017"}, {"entity_type": "GENE", "description": "A functional variant of this gene (Arg728Gln) was associated with numerical but not statistically significant differential colesevelam treatment effects on specific outcomes.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "KLB RS17618244"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "This physiological process was influenced by the FGFR4 rs351855 genetic variant and colesevelam treatment.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "AC T1/2"}, {"entity_type": "METABOLIC PATHWAY", "description": "This metabolic pathway was impacted by the KLB rs4975017 and rs17618244 genetic variants, as well as colesevelam treatment.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "GC24"}, {"entity_type": "NUTRIENT", "description": "Serum FGF19 and stool weight are also shown in Table 1.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "FGF19"}, {"entity_type": "METABOLIC PATHWAY", "description": "Overall univariate association of serum C4 with subgroup was identified (overall P = .02) with higher serum C4 in IBS-D than in IBS-C and HV.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "BA SYNTHESIS"}, {"entity_type": "FOOD CONTENT", "description": "Numerical differences between subgroups in the 48-hour stool weight and 48-hour stool fat that were not statistically significant, with higher stool weight and stool fat in IBS-D compared with HV, and lower values in IBS-C compared with HV.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "STOOL FAT"}, {"entity_type": "MEASURE OF BODY WEIGHT", "description": "Numerical differences between subgroups in the 48-hour stool weight and 48-hour stool fat that were not statistically significant, with higher stool weight and stool fat in IBS-D compared with HV, and lower values in IBS-C compared with HV.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "STOOL WEIGHT"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Previous cholecystectomy was recorded in 23% of IBS-D, approximately 4% of IBS-C, and approximately 4% of HV (P = .05). All patients\u2019 symptoms anteceded the cholecystectomy.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "CHOLECYSTECTOMY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "There were numerical differences between subgroups in the 48-hour stool weight and 48-hour stool fat that were not statistically significant, with higher stool weight and stool fat in IBS-D compared with HV, and lower values in IBS-C compared with HV.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "clusters": "[{level: 0, cluster: 2}]", "id": "BA EXCRETION"}, {"entity_type": "RANK CORRELATION", "description": "There was a significant Spearman correlation between serum C4 and FGF19 (rs = \u22120.324, P = .007), stool weight (rs = 0.366, P = .003), and total stool BA (rs = 0.606, P < .001).", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "FGF19 CORRELATION"}, {"entity_type": "RANK CORRELATION", "description": "There was a significant Spearman correlation between serum C4 and FGF19 (rs = \u22120.324, P = .007), stool weight (rs = 0.366, P = .003), and total stool BA (rs = 0.606, P < .001).", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "STOOL WEIGHT CORRELATION"}, {"entity_type": "RANK CORRELATION", "description": "There was a significant Spearman correlation between serum C4 and FGF19 (rs = \u22120.324, P = .007), stool weight (rs = 0.366, P = .003), and total stool BA (rs = 0.606, P < .001).", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "TOTAL STOOL BA CORRELATION"}, {"entity_type": "RANK CORRELATION", "description": "BMI correlated with serum C4 (rs = 0.423, P < .001) and total stool BA (rs = 0.507, P < .001).", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "BMI CORRELATION"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "In IBS-D subgroup, the BMI was approximately 4 units higher than in other subgroups (P = .036).)", "entity_type": "UNKNOWN", "id": "IBS-D"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "There were numerical differences between subgroups with respect to stool fat content with higher values found in IBS-D compared to HV and lower values found in IBS-C compared to HV.)", "entity_type": "UNKNOWN", "id": "SUBGROUP COMPARISON"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "IBS-D had significantly higher levels of stool BA content compared to IBS-C but not compared to HV, indicating subgroup differences in the synthesis and excretion of bile acids.)", "entity_type": "UNKNOWN", "id": "BA SYNTHESIS (STOOL EXCRETION)"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "No association was observed between fasting serum FGF19 levels with subgroups indicated no direct impact on BA synthesis or stool excretion.)", "entity_type": "UNKNOWN", "id": "LACK OF ASSOCIATION"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "No association was observed between fasting serum FGF19 levels with subgroups indicated no direct impact on BA synthesis or stool excretion.)", "entity_type": "UNKNOWN", "id": "SERUM FGF19"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "The FGFR4 rs1966265 variant was marginally associated with total stool BA content in IBS-D compared to other subgroups.)", "entity_type": "UNKNOWN", "id": "BA SYNTHESIS COMPARISON"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "The FGFR4 rs1966265 variant was marginally associated with total stool BA content in IBS-D compared to other subgroups.)", "entity_type": "UNKNOWN", "id": "FGF14 RS1966265"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "Spearman correlation coefficients indicated that serum C4 and stool weight were positively correlated indicating a relationship between these two parameters.)", "entity_type": "UNKNOWN", "id": "STOOL WEIGHT (SERUM C4)"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "A significant Spearman correlation was found between serum C4 levels and total stool BA suggesting a possible link in the metabolism of bile acids.)", "entity_type": "UNKNOWN", "id": "TOTAL STOOL BA (SERUM C4)"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "Spearman correlation coefficients suggested that BMI correlated with higher levels of serum C4 indicating potential influence on metabolic processes related to bile acid synthesis or excretion.)", "entity_type": "UNKNOWN", "id": "BMI (SERUM C4)"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "Cholecystectomy had a statistically significant impact on IBS symptoms in the IBS-D subgroup compared to IBS-C and HV groups.)", "entity_type": "UNKNOWN", "id": "SYMPTOM IMPACT"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "BMI correlated positively with serum C4 levels, indicating a possible influence of obesity on markers related to bile acid metabolism or other metabolic processes.)", "entity_type": "UNKNOWN", "id": "SERUM C4"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "There was a significant Spearman correlation between stool weight and the amount of fat in stools within different subgroup comparisons.)", "entity_type": "UNKNOWN", "id": "STOOL FAT CONTENT"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "The total content of bile acids in feces showed a strong positive correlation with serum cholesterol levels, suggesting a relationship between dietary factors and bile acid metabolism.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}]", "id": "TOTAL STOOL BA"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "FGF19 showed significant correlations with both stool weight and total stool BA content implying its role might be involved in fecal properties or bile acid synthesis and excretion.)", "entity_type": "UNKNOWN", "id": "STOOL WEIGHT (BA SYNTHESIS)"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "The genetic variant was marginally associated with total stool BA content suggesting a possible link between the gene variant and metabolic processes related to bile acid synthesis.)", "entity_type": "UNKNOWN", "id": "FGFR4 RS1966265 CORRELATION"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "The genetic variation FGFR4 rs351855 was not significantly associated with serum C4 or stool BA suggesting it did not influence bile acid metabolism related markers.)", "entity_type": "UNKNOWN", "id": "FGFR4 RS351855 CORRELATION"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "The genetic variation FGFR4 rs351855 was not significantly associated with serum C4 or stool BA suggesting it did not influence bile acid metabolism related markers.)", "entity_type": "UNKNOWN", "id": "NO ASSOCIATION"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "The genetic variant FGFR4 rs376618 was also found without significant association with serum C4 or stool BA indicating no influence on bile acid pathways.)", "entity_type": "UNKNOWN", "id": "FGFR4 RS376618 CORRELATION"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "Spearman correlations showed that serum C4 levels were positively correlated with total cholesterol levels in blood, suggesting potential links to metabolic processes related to lipids or bile acid metabolism.)", "entity_type": "UNKNOWN", "id": "CHOLESTEROL CORRELATION"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "A significant Spearman correlation between FGF19 and total stool BA content suggested that this factor might play a role in the metabolic pathways related to bile acid synthesis or excretion.)", "entity_type": "UNKNOWN", "id": "BILE ACID METABOLISM"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "A significant Spearman correlation between FGF19 and total stool BA content suggested that this factor might play a role in the metabolic pathways related to bile acid synthesis or excretion.)", "entity_type": "UNKNOWN", "id": "FGF19 (BA SYNTHESIS)"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "The presence of cholecystectomy had a statistically significant effect on IBS symptoms specifically within the IBS-D subgroup, suggesting differences in symptom profiles compared to other subgroups.)", "entity_type": "UNKNOWN", "id": "IBS-D (CHOLECYSTECTOMY)"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "The presence of cholecystectomy had a statistically significant effect on IBS symptoms specifically within the IBS-D subgroup, suggesting differences in symptom profiles compared to other subgroups.)", "entity_type": "UNKNOWN", "id": "SYMPTOM COMPARISON"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "A significant correlation between serum C4 and BA synthesis indicated that changes in this inflammatory marker could influence bile acid metabolic pathways or associated health outcomes.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 4}, {level: 1, cluster: 24}]", "id": "SERUM C4 (BA SYNTHESIS CORRELATION)"}, {"entity_type": "PROTEIN", "description": "FAAP20 is a protein that binds to FANCA through its N-terminal region and contains a UBZ domain in its C terminal extremity.<SEP>FAAP20 protein interacts with UBZ domains and is involved inUb binding.<SEP>The in vivo accumulation of FAAP20 at DNA damage sites is essential for proper recruitment to repair processes, which can influence disease states.<SEP>The protein FAAP20 hydrophobically interacts with the canonical Ile44-centered hydrophobic patch of Ub.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591<SEP>chunk-cc528af98598e1adb4684c0166430cf4<SEP>chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "FAAP20"}, {"entity_type": "GENE", "description": "FANCA gene plays an important role as part of the FA core complex, which is involved in Fanconi Anemia pathway.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "FANCA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The UBZ domain interacts with Ub chains to facilitate the function of FAAP20.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "UBZ DOMAIN"}, {"entity_type": "UBIQUITIN SPECIES", "description": "K63-linked ubiquitins are part of a type of modified ubiquitin which regulates cellular processes like DNA repair, cell cycle control, and protein degradation.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "K63-LINKED UBIQUITIN (K63-UB2)"}, {"entity_type": "PROTEIN", "description": "FAAP20-UBZ binds Ub through specific residues in the \u03b1-helix and \u03b2-loop, facilitating its function.<SEP>FAAP20-UBZ is the region of FAAP20 protein that interacts with K63-Ubiquitins.<SEP>The FAAP20-UBZ domain plays a role in the process of Ub binding, as mutations affecting its structure hinder this process.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591<SEP>chunk-cc528af98598e1adb4684c0166430cf4<SEP>chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "FAAP20-UBZ"}, {"entity_type": "METAL IONS", "description": "A zinc ion plays a crucial role in stabilizing the UBZ domain structure and mediating interaction with ubiquitin chains.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "ZINC ION"}, {"entity_type": "PROTEIN REGION", "description": "This central alpha-helix interacts with the distal Ubiquitin, playing a role in binding affinity.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "CENTRAL \u0391-HELIX (RESIDUES 160\u2013173)"}, {"entity_type": "PROTEIN STRUCTURE", "description": "The second loop of FAAP20-UBZ interacts with the distal Ubiquitin, facilitating its specificity towards certain ubiquitin species.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "SECOND LOOP (RESIDUES 143\u2013159 AND 174\u2013180)"}, {"entity_type": "PROTEIN INTERACTIONS", "description": "These specific regions allow for unique interactions with different types of ubiquitins (K63-, K48- or M1-linked), indicating the role in ubiquitin chain specificity.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "\u0391-HELIX AND SECOND LOOP"}, {"entity_type": "CELLULAR COMPONENT", "description": "Ubiquitins play crucial roles as post-translational modification components, which can be linked together to form chains on proteins for various cellular processes.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "UBIQUITINS (UB)"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "FAAP20 functions within the Fanconi Anemia pathway where it plays a role in DNA repair and cell cycle regulation through its interaction with ubiquitin species.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "FANCONI ANEMIA PATHWAY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The FA complex involves multiple components including FANCA, which are crucial for the cellular response to DNA damage.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "FANCONI ANEMIA (FA) COMPLEX"}, {"entity_type": "POST-TRANSLATIONAL MODIFICATION", "description": "The process of ubiquitination involving ubiquitin (Ub) is a key molecular interaction influenced by FAAP20, WRNIP1, RAD18, Pol\u03b7, NEMO, and FAAP20-UBZ mutants which plays crucial roles in cellular processes related to disease prevention or progression.<SEP>Ubiquitination is a process that occurs on proteins where ubiquitin molecules are attached, affecting protein stability and function.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4<SEP>chunk-fa32a208a2f004c10c1b6777c3be13f2", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 35}]", "id": "UBIQUITINATION"}, {"entity_type": "ANALYTICAL TECHNIQUE", "description": "SPR technique used for studying the binding affinity of FAAP20-UBZ towards different types of ubiquitins.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "SURFACE PLASMON RESONANCE (SPR)"}, {"entity_type": "QUANTITATIVE ANALYSIS/", "description": "FAAP20-UBZ shows efficient interaction with Ub, with lower Kd values compared to other UBDs and Pol\u03b7 UBZ.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "AFFINITY CONSTANTS (KD)/RESONANCE UNITS (RU)"}, {"entity_type": "GENE", "description": "FAAP19 is a gene that plays an auxiliary role in Fanconi Anemia pathway, potentially interacting with FAAP20 and contributing to DNA repair processes.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "FAAP19"}, {"entity_type": "GENE", "description": "FANCL is a core component of the FA core complex involved in modulating the activity of other FA proteins including FAAP20.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "FANCL"}, {"entity_type": "GENE", "description": "UVRAG interacts with FAAP20 and contributes to the formation of the FA core complex, participating in ubiquitin-dependent processes.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "UVRAG"}, {"entity_type": "MOLECULAR ROLE", "description": "FAAP20 could potentially function as an E3 ubiquitin ligase through its UBZ domain interaction with ubiquitins.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "UBIQUITIN E3 LIGASE"}, {"entity_type": "CELLULAR PROCESS", "description": "The Fanconi Anemia pathway, including FAAP20, contributes to the cellular response against DNA damage by facilitating efficient repair mechanisms.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "DNA REPAIR"}, {"entity_type": "REGULATORY FUNCTION", "description": "FAAP20 interacts with other FA proteins to regulate cell cycle progression, ensuring proper mitotic processes.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "CELL CYCLE REGULATION"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "Ubiquitination of proteins through FAAP20 and its UBZ domain is a PTM crucial for maintaining cellular homeostasis.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "POST-TRANSLATIONAL MODIFICATION (PTM)"}, {"entity_type": "INTER-PROTEIN ASSOCIATION", "description": "The FA complex involves multiple components like FANCA, FANCL, UVRAG etc., which are integrated through various interactions including with FAAP20 and its UBZ domain.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "FA COMPLEX COMPONENTS INTERACTION"}, {"entity_type": "DISEASE", "description": "Deficiencies or malfunctions of proteins in the FA complex like FAAP20 can lead to Fanconi Anemia, a disorder affecting DNA repair processes.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "FANCONI ANEMIA (FA) DISORDER"}, {"entity_type": "REGULATORY EFFECT", "description": "FAAP20 through its UBZ domain may influence protein stability by mediating ubiquitination and subsequent degradation or modification of other proteins.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "PROTEIN STABILITY"}, {"entity_type": "PATHOPHYSIOLOGICAL PROCESS", "description": "The FA pathway including FAAP20 influences the regulation of cell death in response to DNA damage, influencing apoptosis or necrosis pathways.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "CELL DEATH REGULATION"}, {"entity_type": "PROTEIN", "description": "Ub interacts with FAAP20 through a series of hydrogen bonds and hydrophobic interactions.<SEP>Ubiquitin (Ub) serves as a substrate for interaction with various UBZ domains.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591<SEP>chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "UB"}, {"entity_type": "TRANSCRIPT", "description": "FAAP20\'s localization and accumulation at ICL sites suggests an involvement in the response to double-strand breaks.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "ICL"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "A mutation affecting a zinc-coordinating residue likely destabilizes FAAP20-UBZ structure, impacting its functionality as it suggests dependencies on cellular conditions.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 41}]", "id": "ZINC-COORDINATING RESIDUE CYS150"}, {"entity_type": "MOLECULAR INTERACTION", "description": "These mutations significantly reduce binding affinity of FAAP20 to Ub, suggesting the importance of these interactions in proper FAAP20 function.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 39}]", "id": "FAAP20-UBZ MUTANTS D164A, A172L AND D177A"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The FAAP20-UBZ domain is essential for physiological processes such as recruitment to DNA damage sites.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 41}]", "id": "FAAP20-UBZ DOMAIN"}, {"entity_type": "DISEASE", "description": "The UBZ domain of WRNIP1 interacts with ubiquitin (Ub) which is involved in repair mechanisms crucial for maintaining cellular health and preventing disease.<SEP>The \u03b2-loop of WRNIP1 facilitates interactions with Ub by aligning Ile44 to hydrophobic residues on the \u03b1-helix or \u03b2-loop.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591<SEP>chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "WRNIP1-UBZ"}, {"entity_type": "MOLECULAR INTERACTION", "description": "RAD18 binds Ub through its conserved sequence motif, allowing for specific interaction mechanisms.<SEP>RAD18-UBZ domain binds ubiquitin through specific interactions, differing from FAAP20-UBZ and WRNIP1-UBZ interactions.<SEP>The UBZ domain of RAD18 plays a role similar to FAAP20, interacting with Ub and participating in cellular processes essential for DNA damage response and potentially impacting disease states.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591<SEP>chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "RAD18-UBZ"}, {"entity_type": "TRANSCRIPT", "description": "Pol\u03b7-UBZ interacts with Ub similarly to FAAP20-UBZ, despite sequence differences.<SEP>Similar to FAAP20, Pol\u03b7 interacts with ubiquitin through its UBZ domain which is involved in repair mechanisms that could influence disease prevention or progression.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591<SEP>chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "POL\u0397-UBZ"}, {"entity_type": "DIETARY PATTERN", "description": "NEMO\'s UBZ domain interacts with Ub and while it does not have a dietary pattern context, it can be inferred to play a role similar to FAAP20 in cellular processes that may influence health and disease through its interactions with ubiquitin.<SEP>NEMO-UBZ interacts with Ub in a manner different from RAD18-UBZ and WRNIP1-UBZ.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591<SEP>chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "NEMO-UBZ"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Mutations affecting FAAP20\'s UBZ domain could lead to disruptions in cellular processes influenced by ubiquitin, which can be considered as an environmental factor affecting health and disease.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "FAAP20-UBZ MUTANTS C150S, D164A, A168L, W180A"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "These FAAP20-UBZ mutations affect the affinity to ubiquitin leading to potential disruptions in physiological processes essential for cell maintenance and repair.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "FAAP20-UBZ MUTANTS L161A, A172L, D177A"}, {"entity_type": "DISEASE", "description": "These proteins interact with ubiquitin (Ub) through their UBZ domains influencing cellular repair mechanisms which can impact the development and progression of various diseases by regulating cell survival and division.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 36}, {level: 2, cluster: 60}]", "id": "FAAP20, WRNIP1, RAD18, POL\u0397, NEMO"}, {"entity_type": "TRANSCRIPT", "description": "The transcript of Pol\u03b7 gene plays a role in the interaction with UBZ domains.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "id": "POL\u0397"}, {"entity_type": "GENE", "description": "The RAD18 gene contributes to the interaction betweenUBZ domains and ubiquitin through specific binding motifs.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "id": "RAD18"}, {"entity_type": "GENOMIC VARIANT", "description": "A genomic variant in WRNIP1 gene affects its interaction with UBZ domains.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "id": "WRNIP1"}, {"source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "description": "FAAP20-UBZ interacts with K63-linked ubiquitin.)", "entity_type": "UNKNOWN", "id": "UBIQUITINS"}, {"source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "description": "These regions of FAAP20-UBZ specifically interact with ubiquitins, affecting the binding specificity.)", "entity_type": "UNKNOWN", "id": "CENTRAL \u0391-HELIX AND SECOND LOOP"}, {"source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "description": "These regions of FAAP20-UBZ specifically interact with ubiquitins, affecting the binding specificity.)", "entity_type": "UNKNOWN", "id": "UBIQUITIN"}, {"source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "description": "FAAP20-UBZ shows strong interaction with different ubiquitin species, as measured by SPR.)", "entity_type": "UNKNOWN", "id": "AFFINITY TOWARDS UBIQUITINS"}, {"source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "description": "FANCL acts as a modulatory component that interacts with FAAP20 and other FA proteins to maintain the activity of ubiquitin-dependent pathways.)", "entity_type": "UNKNOWN", "id": "ROLE IN FA COMPLEX"}, {"source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "description": "UVRAG integrates into the FA core complex alongside FAAP20, participating in the coordination of ubiquitin-related processes.)", "entity_type": "UNKNOWN", "id": "CONTRIBUTION TO FA CORE COMPLEX"}, {"source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "description": "FAAP19 may interact with FANCL for enhanced DNA repair capacity through coordinated activities within Fanconi Anemia pathway.)", "entity_type": "UNKNOWN", "id": "FAAP19 AND FANCL"}, {"source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "description": "FAAP19 may interact with FANCL for enhanced DNA repair capacity through coordinated activities within Fanconi Anemia pathway.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITHIN FA PATHWAY"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "The UBZ domain of FAAP20 is involved in a metabolic pathway related to the interaction with ubiquitin (Ub), crucial for cell repair mechanisms.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 4}, {level: 1, cluster: 24}]", "id": "UBZ-DOMAIN"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "Mutations in key residues necessary for Ub binding may lead to dysfunctions and potentially contribute to disease states.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 39}]", "id": "FAAP20-UBZ MUTANTS"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "FAAP20\'s localization to ICL sites is essential for proper response to DNA damage repair, which could impact disease prevention or progression.)", "entity_type": "UNKNOWN", "id": "ICL SITES"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "Mutations affecting FAAP20\'s UBZ domain can influence the cellular environment by disrupting interactions with ubiquitin (Ub), thereby impacting various biological processes and health outcomes.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 35}]", "id": "FAAP20-UBZ MUTANTS D164A, A172L, D177A"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "The interaction between proteins like FAAP20, WRNIP1, RAD18, Pol\u03b7, NEMO and ubiquitin (Ub) through their UBZ domains is crucial for various cellular processes that play a role in disease states.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 39}]", "id": "UBZ-DOMAIN PROTEINS"}, {"entity_type": "DISEASE", "description": "The present study included a total of 101 Crohn\'s Disease (CD) patients with clinical symptoms such as chronic diarrhea, abdominal pain, anorexia, and abdominal distension.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "CD PATIENTS"}, {"entity_type": "POPULATION GROUP", "description": "A control group consisting of healthy individuals was included for comparison with the HIV-infected participants.<SEP>A total of 103 healthy controls were recruited for comparison with the CD patients.<SEP>Healthy controls are individuals without any diagnosis related to the study.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292<SEP>chunk-6a657cd0a000b6df4db8f1305b795021<SEP>chunk-1418daebca32811d9530c70d8e9bf4cc", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 52}]", "id": "HEALTHY CONTROLS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "30% of the CD patients and 50% of the controls had parents who were consanguineous (related by blood) which could influence their health.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "CONSANGUINEOUS PARENTS"}, {"entity_type": "DISEASE", "description": "Three autoimmune conditions - type 1 diabetes mellitus, autoimmune thyroiditis, and systemic lupus erythematosus, were observed among the patients. Non-autoimmune diseases like osteomalacia, seizure disorders, and Down syndrome were also noted.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "AUTOIMMUNE MANIFESTATIONS"}, {"entity_type": "GENE", "description": "Four HLA-tag Single Nucleotide Polymorphisms (SNPs) were analyzed for their association with CD patients.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "HLA TAGSNPS"}, {"entity_type": "GENOMIC VARIANT", "description": "Genotypes for the SNP rs2395182 were available for all participants involved in the study (n=233).<SEP>The \'G\' minor allele of rs2395182 was more prevalent in healthy controls than in the CD group. This difference is statistically significant (P=0.015, OR = 2.01, 95% CI = 1.138\u20133.562).", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2<SEP>chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "RS2395182"}, {"entity_type": "GENOMIC VARIANT", "description": "The \'C\' minor allele of rs7775228 showed a significant difference in frequency between controls (21%) and CD patients (14%). The statistical results were P=0.0497; OR = 1.67; 95% CI = 0.998\u20132.787.<SEP>The SNP rs7775228 was also successfully genotyped among the participants (n=233).", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2<SEP>chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "RS7775228"}, {"entity_type": "GENOMIC VARIANT", "description": "The minor \'T\' allele was more prevalent in the CD group compared to healthy controls (10% vs 3%). This difference is statistically significant (P=0.001; OR = 4.074; 95% CI = 1.615\u201310.273.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "RS2187668"}, {"entity_type": "GENOMIC VARIANT", "description": "There was no statistically significant difference in the frequency of minor alleles between case and control groups for rs7454108 (P-value > 0.5).", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "RS7454108"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The risk classification of HLA-DQ genotypes was based on previous studies in Saudi Arabia.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "HLA-DQ HAPLOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "The homozygous HLA-DQ2.5 haplotype showed significantly higher frequency among CD patients than controls (10.89% vs 2.91%; p = 0.024).", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "HLA-DQ2.5 HAPLOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "The DQ8 heterozygous haplotype was more frequent among CD patients (33.66%) compared to controls, with a significant difference in the DQ8/DQX haplotype between groups (32.67% vs 14.56%, p = 0.014).", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "DQ8 HETEROZYGOUS HAPLOTYPES"}, {"entity_type": "GENOMIC VARIANT", "description": "The homozygous DQ8 high risk haplotype was more frequent among controls (7.76%) than in patients (1.98%), but this difference wasn\'t statistically significant.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "HOMOZYGOUS HLA-DQ HAPLOTYPES"}, {"entity_type": "GENOMIC VARIANT", "description": "The predicted homozygous HLA-DQ2.2 haplotype was more frequent in controls (5.82%) compared to CD patients (0.99%).", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "HOMOZYGOUS HLA-DQ2.2 HAPLOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "A total of 29.13% of the control group and 24.75% of the CD patient group were found to be heterozygous for DQ2.2.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "HETEROZYGOUS DQ2.2 HAPLOTYPES"}, {"entity_type": "VALIDATION METHOD", "description": "Sanger sequencing analysis confirmed the accuracy (100%) of the Real-time PCR based tagSNPs approach in determining HLA-DQ haplotypes among randomly selected individuals from both study groups.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "HLA-DQ HAPLOTYPE ACCURACY"}, {"entity_type": "DISEASE", "description": "4% of CD patients had Down syndrome, which is a genetic disorder affecting development.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "DOWN SYNDROME"}, {"entity_type": "DISEASE", "description": "4% of CD patients experienced seizure disorders, an electrical disturbance in the brain leading to abnormal behavior or sensations.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "SEIZURE DISORDERS"}, {"entity_type": "DISEASE", "description": "8% of CD patients had autoimmune thyroiditis, an autoimmune condition affecting the thyroid gland.<SEP>Diagnosis by treating with levothyroxine, a thyroid hormone replacement therapy.<SEP>The development of autoimmune thyroid disorders is indicated by the presence of anti-thyroglobulin and anti-thyroid peroxidase antibodies in STAT5B deficient patients.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5<SEP>chunk-e323b0d4f82733a78f687e51f47b500a<SEP>chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "AUTOIMMUNE THYROIDITIS"}, {"entity_type": "DISEASE", "description": "32% of CD patients were diagnosed with type 1 diabetes mellitus, a metabolic disorder where the body can\'t produce insulin.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "TYPE 1 DIABETES MELLITUS"}, {"entity_type": "GENOMIC VARIANT", "description": "ABCB1 (P-glycoprotein), rs1128503 (C>T) is associated with grade 3-4 anemia.<SEP>rs1128503 interacts with ABCB1 (P-glycoprotein), which can affect drug efflux and potentially influence chemotherapy toxicity.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS1128503"}, {"entity_type": "GENOMIC VARIANT", "description": "ABCC2 (Multidrug Resistance Protein 2), rs12762549 (C>G) is associated with grade 3-4 anemia.<SEP>Variant rs12762549 (ABCC2 A\u2009>\u2009G) was significantly associated with PFI in the fully-adjusted model, but not in the age-only adjusted model.<SEP>rs12762549 affects ABCC2\'s function, potentially leading to altered cellular transport mechanisms and influencing anemia risk.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404<SEP>chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "RS12762549"}, {"entity_type": "GENOMIC VARIANT", "description": "ABCA1 (ATP-binding cassette transporter A1), rs363717 (A>G) is associated with grade 3-4 anemia.<SEP>rs363717 impacts ABCA1 activity, which could alter lipid metabolism and contribute to anemia development.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS363717"}, {"entity_type": "GENOMIC VARIANT", "description": "ERCC1 (Excision repair cross-complementation group 1), rs11615 (T>C) is associated with grade 3-4 anemia.<SEP>rs11615\'s influence on ERCC1 suggests a role in DNA repair mechanisms, which might affect cell survival under chemotherapy stress.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS11615"}, {"entity_type": "GENOMIC VARIANT", "description": "CYP3A4, rs4986910 (T>C) is associated with thrombocytopenia grade 3-4.<SEP>rs4986910 modulates CYP3A4 function, potentially affecting drug metabolism and toxicity related to chemotherapy.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS4986910"}, {"entity_type": "GENOMIC VARIANT", "description": "ABCC2 (Multidrug Resistance Protein 2), rs2073337 (A>G) is correlated with CSF use during chemotherapy.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS2073337"}, {"entity_type": "GENOMIC VARIANT", "description": "RASSF1A (Ras association domain family 1 member A), rs6763398 (C>T) is associated with an increased risk of developing grade 2 or higher toxicity.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS6763398"}, {"entity_type": "GENOMIC VARIANT", "description": "GSTM1 (Glutathione S-transferase M1), rs4607905 (A>C) is correlated with the need for hematopoietic growth factors during chemotherapy.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS4607905"}, {"entity_type": "GENOMIC VARIANT", "description": "BCPY2C9, rs1800569 (C>T) impacts drug metabolism and its influence on chemotherapy response.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS1800569"}, {"entity_type": "GENOMIC VARIANT", "description": "The variant rs1799793 in ERCC2 was significantly correlated with PFI, with a lower hazard ratio and confidence interval compared to the AA or GA carriers.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "RS1799793"}, {"entity_type": "GENOMIC VARIANT", "description": "Variant rs6104 (SERPINB2 C\u2009>\u2009G) also showed association with PFI in the fully-adjusted model, yet it was not statistically significant when age adjustment alone was applied.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "RS6104"}, {"entity_type": "GENOMIC VARIANT", "description": "The variant rs1799793 is located in ERCC2 and its presence (GG carriers) showed a statistically significant advantage for PFI.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "ERCC2 G\u2009>\u2009A"}, {"entity_type": "GENOMIC VARIANT", "description": "Variant rs12762549 was associated with PFI when both age at diagnosis, FIGO stage, tumor grade, tumor histology, and residual disease after debulking surgery were considered.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "ABCC2 A\u2009>\u2009G"}, {"entity_type": "GENOMIC VARIANT", "description": "Variant rs6104 was found to be related to PFI in the fully-adjusted model but not significant when age adjustment alone was used.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "SERPINB2 C\u2009>\u2009G"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The presence of the rs12762549 variant is associated with an increased risk of developing grade 3-4 anemia.)", "entity_type": "UNKNOWN", "id": "GRADE 3-4 ANEMIA"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The presence of the rs4986910 variant is associated with an increased risk of developing thrombocytopenia grade 3-4.)", "entity_type": "UNKNOWN", "id": "THROMBOCYTOPENIA GRADE 3-4"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The presence of the rs2074087 variant shows an association with ESA usage during chemotherapy, potentially influencing treatment efficacy and managing side effects.)", "entity_type": "UNKNOWN", "id": "GROWTH FACTORS USE (ESAS)"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The presence of the rs2074087 variant shows an association with ESA usage during chemotherapy, potentially influencing treatment efficacy and managing side effects.)", "entity_type": "UNKNOWN", "id": "RS2074087"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The presence of the rs2073337 variant shows an association with CSF usage during chemotherapy, indicating potential need for supplementary treatments to manage side effects.)", "entity_type": "UNKNOWN", "id": "GROWTH FACTORS USE (CSFS)"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs1799793 variant shows significant association with PFI, indicating potential genetic influences on treatment outcome.)", "entity_type": "UNKNOWN", "id": "PROGRESSION-FREE INTERVAL (PFI)"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs434074 variant shows an association with neutropenia during chemotherapy treatment, indicating a genetic predisposition to this side effect.)", "entity_type": "UNKNOWN", "id": "NEUTROPENIA"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs434074 variant shows an association with neutropenia during chemotherapy treatment, indicating a genetic predisposition to this side effect.)", "entity_type": "UNKNOWN", "id": "RS434074"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs1862219 variant is associated with an increased risk of developing grade 2 or higher toxicity during chemotherapy, highlighting a genetic factor in adverse reactions.)", "entity_type": "UNKNOWN", "id": "GRADE 2 OR HIGHER TOXICITY"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs1862219 variant is associated with an increased risk of developing grade 2 or higher toxicity during chemotherapy, highlighting a genetic factor in adverse reactions.)", "entity_type": "UNKNOWN", "id": "RS1862219"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs737645 variant correlates with the need for hematopoietic growth factors during chemotherapy, indicating genetic influences on supportive care requirements.)", "entity_type": "UNKNOWN", "id": "HEMATOPOIETIC GROWTH FACTORS USE"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs737645 variant correlates with the need for hematopoietic growth factors during chemotherapy, indicating genetic influences on supportive care requirements.)", "entity_type": "UNKNOWN", "id": "RS737645"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs653980 variant shows an association with the development of grade 3 or higher peripheral neuropathy, suggesting a genetic predisposition to this specific side effect.)", "entity_type": "UNKNOWN", "id": "PERIPHERAL NEUROPATHY"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs653980 variant shows an association with the development of grade 3 or higher peripheral neuropathy, suggesting a genetic predisposition to this specific side effect.)", "entity_type": "UNKNOWN", "id": "RS653980"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs10866159 variant correlates with the development of hematological toxicity, highlighting a genetic factor influencing this side effect.)", "entity_type": "UNKNOWN", "id": "HEMATOLOGICAL TOXICITY"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs10866159 variant correlates with the development of hematological toxicity, highlighting a genetic factor influencing this side effect.)", "entity_type": "UNKNOWN", "id": "RS10866159"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs9095462 variant shows an association with the development of grade 2 or higher nausea and vomiting, indicating a genetic predisposition to these symptoms.)", "entity_type": "UNKNOWN", "id": "NAUSEA AND VOMITING"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs9095462 variant shows an association with the development of grade 2 or higher nausea and vomiting, indicating a genetic predisposition to these symptoms.)", "entity_type": "UNKNOWN", "id": "RS9095462"}, {"source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "description": "The genomic variant rs6104 showed an association with PFI in the fully-adjusted model but did not meet statistical significance when age adjustment alone was considered.)", "entity_type": "UNKNOWN", "id": "PFI"}, {"entity_type": "GENE", "description": "SNP array of I1 showed a de novo 6.4 Mb deletion on chromosome 3 containing more than 40 RefSeq genes, including UMPS.", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "UMPS"}, {"entity_type": "PROTEIN", "description": "c.302delA variant found in I4", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "LYS101ARGFS*9"}, {"entity_type": "DISEASE", "description": "The c.385G>T (p.Gly129*) variant in UMPS is associated with agenesis of the corpus callosum", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 36}, {level: 2, cluster: 60}]", "id": "P.GLY129*"}, {"entity_type": "MISSENSE ALTERATION", "description": "The c.224C>A variant was found in pedigree 2 among twelve family members, with nine being maternally inherited.", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "P.PRO75HIS"}, {"entity_type": "PROTEIN", "description": "In I3 and his mother M3, two variants c.286A>G (p.Arg96Gly) and c.1285G>C were found", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "ARG96GLY"}, {"source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "description": "In I1, there was a de novo 6.4 Mb deletion on chromosome 3 that contained the UMPS gene.)", "entity_type": "UNKNOWN", "id": "DELETION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The TaqMan method was used for genotyping in the study.", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "TAQMAN VIIA\u2122 7 REAL-TIME PCR"}, {"entity_type": "SNP", "description": "This refers to the specific SNP in the TGF-\u00df1 gene that was genotyped.", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "RS1800469 C/T"}, {"entity_type": "SNP", "description": "This is another SNP, specifically within the IL-10 gene, which was genotyped for association with CRC.", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "RS1800896 A/G"}, {"entity_type": "GENE", "description": "The IL-8 gene\'s rs4073 polymorphism in its promoter region was analyzed.", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "INTERLEUKIN-8 (IL-8)"}, {"entity_type": "GENE", "description": "The promoter region of the IL-10 gene underwent genotyping to identify a specific SNP (rs1800896 A/G).", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "INTERLEUKIN-10 (IL-10)"}, {"entity_type": "GENE", "description": "The promoter region of the IL-27 gene was analyzed, focusing on a SNP (rs17855750T/G).", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "INTERLEUKIN-27 (IL-27)"}, {"entity_type": "GENE", "description": "This is one of the genes examined for its promoter region SNPs related to CRC.", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "TRANSFORMING GROWTH FACTOR-BETA 1 (TGF-SS1)"}, {"entity_type": "SNP", "description": "This refers to a polymorphism within the IL-8 gene\'s promoter region.", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "RS4073 T/A"}, {"entity_type": "SNP", "description": "A specific SNP within the IL-1\u03b2 gene was analyzed for association with CRC.", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "RS5493062"}, {"entity_type": "GENE", "description": "Polymorphisms in the IL-1\u03b1 gene were also considered as part of this study.", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "IL-1\u0391"}, {"entity_type": "SNP", "description": "This is a SNP found within the IL-1\u03b1 gene that was evaluated for its relevance to CRC risk.", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "RS3093422"}, {"entity_type": "GENOMIC VARIANT", "description": "The CC genotype is associated with the occurrence of Colorectal Cancer (CRC) and is slightly lower in patients compared to control groups.", "source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "id": "CC GENOTYPE"}, {"entity_type": "VARIANT", "description": "A T allele has a slight association with CRC risk, though it did not reach statistical significance.", "source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "id": "T ALLELE"}, {"entity_type": "DISEASE", "description": "CRC patients showed a slightly lower presence of the T allele compared to control groups.", "source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "id": "CRC PATIENTS"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "The TaqMan method was utilized to assess the association between SNPs and CRC.)", "entity_type": "UNKNOWN", "id": "TAQMAN"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "Genotyping assessed the TGF-\u00df1 gene for specific SNPs that might be associated with CRC risk.)", "entity_type": "UNKNOWN", "id": "TGF-SS1 SNPS"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "The polymorphism rs4073 in the IL-8 gene was genotyped to assess its association with CRC.)", "entity_type": "UNKNOWN", "id": "IL-8 GENOTYPE"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "SNP analysis of IL-10 gene\'s rs1800896 polymorphism was performed for CRC risk assessment.)", "entity_type": "UNKNOWN", "id": "IL-10 GENOTYPE"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "Polymorphisms in the IL-22 gene were studied to understand their link with CRC susceptibility.)", "entity_type": "UNKNOWN", "id": "IL-22 GENOTYPES"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "Polymorphisms in the IL-22 gene were studied to understand their link with CRC susceptibility.)", "entity_type": "UNKNOWN", "id": "RS1179251 C/G AND RS2227485 C/T"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "Genotyping of this SNP in IL-27\'s promoter region was performed for its role in CRC development.)", "entity_type": "UNKNOWN", "id": "IL-27 GENOTYPE"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "Genotyping of this SNP in IL-27\'s promoter region was performed for its role in CRC development.)", "entity_type": "UNKNOWN", "id": "RS17855750T/G"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "The polymorphism of IL-8\'s rs4073 was assessed to determine if it influences the risk of developing colorectal cancer.)", "entity_type": "UNKNOWN", "id": "CRC RISK"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "The polymorphism of IL-8\'s rs4073 was assessed to determine if it influences the risk of developing colorectal cancer.)", "entity_type": "UNKNOWN", "id": "IL-8 RS4073 T/A"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "Genotyping for this specific SNP in the IL-1\u03b2 gene provided insights into its potential association with CRC.)", "entity_type": "UNKNOWN", "id": "IL-1\u0392 SNPS"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "This polymorphism was analyzed as it relates to changes in IL-1\u03b1 that may impact the risk of colorectal cancer development.)", "entity_type": "UNKNOWN", "id": "CRC SUSCEPTIBILITY"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "This polymorphism was analyzed as it relates to changes in IL-1\u03b1 that may impact the risk of colorectal cancer development.)", "entity_type": "UNKNOWN", "id": "IL-1\u0391 RS3093422"}, {"source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "description": "While the T allele showed a slight association with CRC, this was not statistically significant.)", "entity_type": "UNKNOWN", "id": "NON-SIGNIFICANT ASSOCIATION"}, {"entity_type": "PATIENT", "description": "A total of 387 unrelated asthmatic patients were included in the analysis.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "ASTHMATIC PATIENT"}, {"entity_type": "PATIENT", "description": "A total of 212 unrelated healthy controls were included in the analysis.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "HEALTHY CONTROL"}, {"entity_type": "POPULATION SUBGROUP", "description": "The non-Hispanic white group showed good separation from other groups based on PCA.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "NON-HISPANIC WHITE GROUP"}, {"entity_type": "POPULATION SUBGROUP", "description": "The non-Hispanic black group was clearly separated from the non-Hispanic white group.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "NON-HISPANIC BLACK GROUP"}, {"entity_type": "POPULATION SUBGROUP", "description": "The Hispanic group\'s principal component scores were in-between those of the other groups.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "HISPANIC GROUP"}, {"entity_type": "ANCESTRY MARKER", "description": "Scores of more than 80% variability were explained by the first five principal components.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "PRINCIPAL COMPONENT SCORE"}, {"entity_type": "POPULATION SUBGROUP", "description": "The asthma population had a higher average age and BMI compared to controls.<SEP>The second most frequent haplotype was associated with asthma.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "ASTHMA POPULATION"}, {"entity_type": "LUNG FUNCTION PARAMETER", "description": "Reduced % predicted FEV1 was observed in asthmatic patients compared to healthy controls.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "FEV1% PREDICTION"}, {"entity_type": "LUNG FUNCTION PARAMETER", "description": "Reduced % predicted FVC was also observed in the asthma population.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "FVC% PREDICTION"}, {"entity_type": "LUNG FUNCTION PARAMETER", "description": "Lower FEV1/FVC ratios were noted in asthmatic patients.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "FEV1/FVC RATIO"}, {"entity_type": "IMMUNOLOGICAL PARAMETER", "description": "Increased total IgE was observed in asthmatic patients.<SEP>Increased total IgE was observed in the asthma population.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "TOTAL IGE"}, {"entity_type": "CLINICAL FEATURE", "description": "A significantly higher proportion of cases compared to controls were found to be atopic.<SEP>More cases than controls were found to be atopic.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "ATOPY"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs1898671 showed nominal association with asthma.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "TSLP SNPS"}, {"entity_type": "POPULATION SUBGROUP", "description": "Increased risk of asthma was noted for ex-smokers carrying SNP rs1898671.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "EX-SMOKER SUBGROUP"}, {"entity_type": "POPULATION SUBGROUP", "description": "SNP rs1898671 did not show significant association with asthma in never-smokers.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "NEVER-SMOKER SUBGROUP"}, {"entity_type": "COMBINED DATA SET", "description": "Analysis of the large APCAT cohort supported the association of SNP rs1898671 and asthma.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "REPLICATION COHORT"}, {"entity_type": "POPULATION SUBGROUP", "description": "SNP rs1898671 validated its effect on asthma risk in ex-smokers.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "EX-SMOKER SUBGROUP REPLICATION"}, {"entity_type": "POPULATION SUBGROUP", "description": "No significant association found for SNP rs1898671 in never-smokers.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "NEVER-SMOKER SUBGROUP REPLICATION"}, {"entity_type": "HAPLOTYPE", "description": "This haplotype showed a significant increased risk with asthma after adjustment.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "TSLP HAPLOTYPE GAGGGCAAAG"}, {"entity_type": "DEMOGRAPHIC/ CLINICAL FEATURE", "description": "There was a difference in race/ethnicity between cases and controls, with p=0.01.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "RACE/ETHNICITY"}, {"entity_type": "PCA DATA POINT", "description": "The first principal component scores showed separation between the non-Hispanic white group and the non-Hispanic black group.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "PRINCIPAL COMPONENT SCORES"}, {"entity_type": "POPULATION STRATIFICATION METHOD OUTPUT", "description": "The STRUCTURE method estimated that each subject\'s ancestry is a blend of multiple underlying populations based on their principal component scores.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "STRUCTURAL POPULATION INFORMATION"}, {"entity_type": "ANCESTRY DATA POINT", "description": "More than 80% of variability in ancestry markers was explained by the first five principal components, accounting for over 80% of total variance.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "ANCESTRY MARKERS"}, {"source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68<SEP>chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "Asthma is a chronic inflammatory disease affecting the airways.<SEP>SNP rs1898671 showed a nominal association with asthma susceptibility.)", "entity_type": "DISEASE", "id": "ASTHMA"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "The SNP\'s risk for asthma was increased in ex-smoker subgroups.)", "entity_type": "UNKNOWN", "id": "EX-SMOKERS"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "SNP rs1898671 did not show significant association with asthma in never-smokers.)", "entity_type": "UNKNOWN", "id": "NEVER-SMOKERS"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "The replication cohort validated the association of SNP rs1898671 with asthma.)", "entity_type": "UNKNOWN", "id": "LARGE DATA SET"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "The replication cohort validated the association of SNP rs1898671 with asthma.)", "entity_type": "UNKNOWN", "id": "REPLICATION COHORT 6 COAHS"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "SNP rs1898671\'s effect on asthma risk was confirmed in ex-smoker subgroups.)", "entity_type": "UNKNOWN", "id": "EX-SMOKERS IN REPLICATION COHORT"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "No significant association found for SNP rs1898671 in never-smokers.)", "entity_type": "UNKNOWN", "id": "NEVER-SMOKERS IN REPLICATION COHORT"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "Cases showed differences such as age and BMI compared to healthy control group.)", "entity_type": "UNKNOWN", "id": "ASTHMATIC PATIENT SUBGROUP"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "Cases showed differences such as age and BMI compared to healthy control group.)", "entity_type": "UNKNOWN", "id": "CLINICAL POPULATION GROUP"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "PCA revealed clear separation between the non-Hispanic white group and other groups.)", "entity_type": "UNKNOWN", "id": "POPULATION STRATIFICATION COMPONENT"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "The asthma population had a higher average age compared to healthy controls.)", "entity_type": "UNKNOWN", "id": "AGE DIFFERENCE"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "The asthma population had a higher average age compared to healthy controls.)", "entity_type": "UNKNOWN", "id": "COMPARISON BETWEEN GROUPS"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "Asthmatic patients had significantly higher BMI values than healthy control participants.)", "entity_type": "UNKNOWN", "id": "BMI COMPARISON"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "There was a statistically significant difference in race/ethnicity between the asthmatic patient group and the healthy controls, with p=0.01.)", "entity_type": "UNKNOWN", "id": "CLINICAL FEATURE COMPARISON"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "There was a statistically significant difference in race/ethnicity between the asthmatic patient group and the healthy controls, with p=0.01.)", "entity_type": "UNKNOWN", "id": "RACE/ETHNICITY DIFFERENCE"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "Reduced FEV1%, FVC% and FEV1/FVC ratios were noted specifically in asthma patients compared to healthy individuals.)", "entity_type": "UNKNOWN", "id": "DIFFERENCES BETWEEN GROUPS"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "Reduced FEV1%, FVC% and FEV1/FVC ratios were noted specifically in asthma patients compared to healthy individuals.)", "entity_type": "UNKNOWN", "id": "LUNG FUNCTION PARAMETER REDUCTIONS"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "Increased total IgE levels were a characteristic feature seen exclusively among asthmatic patients.)", "entity_type": "UNKNOWN", "id": "COMPARISON OF IMMOBILE PARAMETERS"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "Increased total IgE levels were a characteristic feature seen exclusively among asthmatic patients.)", "entity_type": "UNKNOWN", "id": "TOTAL IGE INCREASE"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "A significantly greater proportion of asthmatic patients demonstrated atopic characteristics compared to healthy controls, with p=0.01.)", "entity_type": "UNKNOWN", "id": "ATOPY PROPORTION"}, {"entity_type": "GENOMIC VARIANT", "description": "One of the four SNPs found in PTPN2, not significant in CD compared to healthy controls.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS1893217"}, {"entity_type": "GENOMIC VARIANT", "description": "Another one of the four SNPs found in PTPN2, not significant in CD compared to healthy controls.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS2542151"}, {"entity_type": "GENOMIC VARIANT", "description": "One of the four SNPs found in PTPN2, not significant in CD compared to healthy controls.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS7234029"}, {"entity_type": "GENOMIC VARIANT", "description": "Significant SNP found in both PTPN2 and CD subjects compared to healthy controls.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS478582"}, {"entity_type": "GENOMIC VARIANT", "description": "One of the five SNPs specific to PTPN22, not considered significant in CD compared to healthy controls.<SEP>The genomic variant rs2476601 is associated with several autoimmune disorders.<SEP>One of the genome-wide significant SNPs for general IgM identified during our analysis of IgM anti-PC GWAS.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db<SEP>chunk-0dc1ddaa86432d222da7fc75090f700a<SEP>chunk-dee0a8a550579bff19f48d41446e842d", "id": "RS2476601"}, {"entity_type": "GENE", "description": "Not considered in the analysis due to lack of significance.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS2488457"}, {"entity_type": "GENOMIC VARIANT", "description": "Not considered in the analysis due to lack of significance.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS33996649"}, {"entity_type": "GENE", "description": "Not considered in the analysis due to lack of significance.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS34209542"}, {"entity_type": "GENOMIC VARIANT", "description": "Not considered in the analysis due to lack of significance.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS2476599"}, {"entity_type": "GENOMIC VARIANT", "description": "Specific SNPs of PTPN2:rs478582 associated with CD subjects.<SEP>The rs478582 variant in the PTPN2 gene was studied for its impact on gene expression and disease susceptibility.<SEP>This SNP in PTPN2 is associated with increased IFN-\u03b3 expression in CD subjects with certain genotypes.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568<SEP>chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-6a657cd0a000b6df4db8f1305b795021", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 53}]", "id": "PTPN2:RS478582"}, {"entity_type": "ALLELE", "description": "Having either heterozygous (TC) or minor (CC) alleles of rs478582 was associated with lower CD average relative gene expression.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "TC ALLELES"}, {"entity_type": "STATISTICAL MEASURE", "description": "Comparisons with a P-value < 0.05 indicated significant differences in relative gene expression.<SEP>P-values < 0.05 indicated significant differences in relative gene expression of IFN-\u03b3 between CD and healthy control groups.<SEP>The P-values were < 0.05, indicating statistical significance in comparisons.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "P-VALUE"}, {"entity_type": "ALLELE", "description": "Having homozygous (CC) alleles of rs478582 was associated with a lower average relative gene expression compared to the healthy controls.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "CC ALLELES"}, {"entity_type": "VARIANT IN PTPN2:RS2476601", "description": "The rs2476601 variant in the PTPN2 gene did not change expression when compared with normal (GG) subjects, but showed lower average relative gene expression in CD subjects.<SEP>The variant did not show change when comparing to normal (GG) genotypes, but exhibited lower average relative gene expression in CD subjects.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "GA OR AA ALLELES"}, {"entity_type": "GENOTYPE", "description": "The minor (AA) genotypes for rs2476601 were very uncommon among all subjects.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "AA ALLELES RARE"}, {"entity_type": "GENOMIC VARIANT PTPN2:RS478582", "description": "These genotypes were associated with a higher rate of MAPbacteremia presence among the CD group compared to healthy controls.<SEP>These variants were associated with significantly higher average relative gene expression in IFN-\u03b3 among the CD group compared to normal subjects.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "HETEROZYGOUS (TC) OR MINOR (CC) ALLELES"}, {"entity_type": "GENOMIC VARIANT PTPN22:RS2476601", "description": "These variants were uncommon among all subjects and showed no significant change in IFN-\u03b3 gene expression in CD subjects.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "HETEROZYGOUS (GA) OR MINOR (AA) ALLELES"}, {"entity_type": "ODDS RATIO AND CONFIDENCE INTERVAL", "description": "An odds ratio greater than 1 indicates an increased risk for MAP infection in individuals with PTPN2:rs478582.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "OR = 17.5, CI: 5.65-54.3"}, {"entity_type": "ALLELE", "description": "Having AA alleles for rs6980683 was linked to lower average relative gene expression of IFN-\u03b3 compared to other genotypes.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "AA ALLELES"}, {"entity_type": "DISEASE", "description": "CD (Celiac Disease) is linked to SNPs in the PTPN2 gene.<SEP>Condition characterized by the symptoms and signs mentioned, with a genetic predisposition indicated by SNPs in PTPN2 and PTPN22.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-6a657cd0a000b6df4db8f1305b795021", "id": "CD"}, {"entity_type": "GENOMIC VARIANT", "description": "SNPs of PTPN22:rs2476601 associated with presence of MAPbacteremia.<SEP>This SNP in PTPN22 affects T-cell functionality and response when combined with MAPbacteremia.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-6a657cd0a000b6df4db8f1305b795021", "id": "PTPN22:RS2476601"}, {"entity_type": "DISEASE", "description": "Presence of MAPbacteremia correlated with specific SNPs in the PTPN2 gene.<SEP>Presence of MAPbacteremia was noted to influence T-cell proliferation response.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-6a657cd0a000b6df4db8f1305b795021", "id": "MAPBACTEREMIA"}, {"entity_type": "SUBJECT", "description": "CD subjects are individuals diagnosed with Celiac Disease.", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 51}]", "id": "CD SUBJECTS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Treatment with PHA resulted in a fold change increase in T-cell proliferation compared to healthy controls.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "id": "PHA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "When treated with MAP PPD-like, the average fold change in CD subjects was higher than in healthy controls.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "id": "MAP PPD-LIKE"}, {"entity_type": "DISEASE", "description": "The T-cell response of these subjects showed less fold change compared to those with specific genotypes.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "id": "CD SUBJECTS WITHOUT MAPBACTEREMIA AND PTPN2:RS478582/PTPN22:RS2476601"}, {"entity_type": "CONDITION", "description": "These subjects had a higher average overall fold change compared to CD subjects without MAPbacteremia presence.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "id": "CD SUBJECTS WITH MAPBACTEREMIA AND SNPS IN PTPN2 OR PTPN22"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The function and response of T-cells, influenced by SNPs, MAPbacteremia presence, and molecular interactions.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "id": "T-CELLS"}, {"source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "description": "The Haplotype combination of T-G (major/major) was more significantly present among healthy controls compared to CD subjects.)", "entity_type": "UNKNOWN", "id": "HAPLOTYPE T-G"}, {"source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "description": "The Haplotype combination of C-G (heterozygous or minor/major) and C-A (heterozygous or minor/heterozygous or minor) was more significantly present among CD subjects compared to healthy controls.)", "entity_type": "UNKNOWN", "id": "HAPLOTYPE C-G"}, {"source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "description": "The presence of rs2476601 was not significantly associated with differences in expression levels of PTPN2 or the cytokine IFN-\u03b3 in CD subjects compared to healthy controls. )", "entity_type": "UNKNOWN", "id": "EXPRESSION OF PTPN2 AND IFN-\u0393"}, {"source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "description": "1(entity", "entity_type": "UNKNOWN", "id": "OR = 17.5, CI: 5.65-54.3&RISK FACTOR COMPARISON&INCREASED RISK OF DISEASE"}, {"source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "description": "1(entity", "entity_type": "UNKNOWN", "id": "THE ODDS RATIO COMPARISONS SHOWED A STATISTICALLY SIGNIFICANT INCREASE IN DISEASE RISK ASSOCIATED WITH PTPN2:RS478582.)"}, {"source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "description": "SNPs of PTPN2:rs478582 are associated with the risk and manifestation of Celiac Disease.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 51}]", "id": "CELIAC DISEASE"}, {"source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "description": "Specific SNPs show higher presence in individuals with heterozygous (TC) alleles.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 52}]", "id": "HETEROZYGOUS ALLELES"}, {"source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "description": "CD subjects are more likely to exhibit MAPbacteremia when specific PTPN2:rs478582 SNPs are present.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 51}]", "id": "PRESENCE OF MAPBACTEREMIA"}, {"source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "description": "Healthy controls exhibit none or lower levels of both MAPbacteremia and the studied SNPs.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}, {level: 1, cluster: 52}]", "id": "NO MAPBACTEREMIA OR SPECIFIC SNPS"}, {"source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "description": "These SNPs influence T-cell proliferation response when exposed to PHA or MAP PPD-like molecules.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 39}]", "id": "SNPS IN PTPN2 AND PTPN22"}, {"source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "description": "Alteration of T-cell response when specific pathogens are present in the environment, affecting physiological processes.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 35}]", "id": "MAPBACTEREMIA PRESENCE"}, {"entity_type": "GENE", "description": "The PTPN22 gene is associated with lymphocyte subsets and their activation states.<SEP>The PTPN22 gene plays a crucial role in modulating T-cell function and differentiation.<SEP>The expression of PTPN22 is shown to differ between Treg and non-regulatory (Tconv) cells, impacting their roles in autoimmune disorders.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d<SEP>chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "PTPN22"}, {"entity_type": "TRANSCRIPT", "description": "Naive Tconv cells express significantly less PTPN22 transcripts compared to memory Tconv and Tregs.<SEP>The naive Tconv cells increased 12.00\u00b15.45 fold after activation.", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "NAIVE TCONV"}, {"entity_type": "TRANSCRIPT", "description": "Memory Tconv cells increased by 4.01\u00b10.89 fold compared to their resting state, showing a significantly lower increase than naive Tconv.<SEP>Memory Tconv cells show higher expression of PTPN22 transcripts than naive Tconv and Tregs.", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "MEMORY TCONV"}, {"entity_type": "TRANSCRIPT", "description": "Data highlight the role of regulatory T cells (Tregs) in modulating immune responses.<SEP>The Treg cells showed an increase of 2.85\u00b11.31 fold in PTPN22 expression following activation (significantly less than naive and memory Tcells).<SEP>Tregs exhibit the highest PTPN22 transcript levels compared to naive Tconv and memory Tconv.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d<SEP>chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "TREG"}, {"entity_type": "ACTIVATION STATE", "description": "In vitro polyclonal activation leads to a rapid increase in PTPN22 expression in all three subsets (naive Tconv, memory Tconv, and Treg).", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "POLYCLONAL ACTIVATION"}, {"entity_type": "PROTEIN VARIANT", "description": "LYP-620R is the non-risk variant of PTPN22 encoding amino acid 620.<SEP>Overexpression of LYP-620R results in reduced APC-mediated proliferation of both Tconv and Treg cells compared to controls.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7<SEP>chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "LYP-620R"}, {"entity_type": "PROTEIN VARIANT", "description": "LYP-620W represents the risk variant of PTPN22 encoding amino acid 620, associated with T-cell dysfunction.<SEP>LYP-620W shows a differential impact on Treg proliferation, being able to block it to the same degree as LYP-620R.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7<SEP>chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "LYP-620W"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Calcium signaling is a critical pathway for downstream events following TCR ligation and affects the expression of several genes including PTPN22.", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "CALCIUM SIGNALING"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The MAPK/ERK (mitogen-activated protein kinase/extracellular signal-regulated kinase) pathway is interconnected with calcium signaling in modulating T-cell responses and the expression of PTPN22.", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "MAPK/ERK PATHWAY"}, {"entity_type": "SIGNAL MARKER", "description": "The level of phosphorylated ERK (pERK) is a measure of activity within the MAPK/ERK pathway, which correlates with downstream TCR signaling events.", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "PERK"}, {"entity_type": "CALCIUM MEASUREMENT", "description": "Calcium flux quantifies changes in intracellular calcium levels as a response to T-cell receptor (TCR) activation, impacting PTPN22 expression and function.", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "CA2+ FLUX"}, {"entity_type": "ACTIVATION MARKER", "description": "CD69 is dynamically expressed following Tconv activation.<SEP>The CD69 molecule is an early marker of T-cell activation which is upregulated shortly after TCR ligation and correlates with the activity of downstream signaling cascades like MAPK/ERK and PTPN22-related pathways.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7<SEP>chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "CD69"}, {"entity_type": "ACTIVATION MARKER", "description": "ICOS expression kinetics are involved in Treg activation.<SEP>Inducible costimulatory (ICOS) molecule serves as a co-stimulatory receptor on T-cells following activation and modulates several functions, including PTPN22 expression.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7<SEP>chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "ICOS"}, {"entity_type": "ACTIVATION MARKER", "description": "CD25, an integral part of the IL-2 receptor, is dramatically upregulated after activation.<SEP>The CD25 molecule, also known as IL-2 receptor alpha-chain (IL2R\u03b1), is another early marker for T-cell activation which can influence the levels of PTPN22.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7<SEP>chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "CD25"}, {"entity_type": "ACTIVATION MARKER", "description": "Also known as NKG2D or NKp46, CD226 acts as a co-stimulatory receptor on T-cells and is involved in modulating gene expression patterns including that of PTPN22.<SEP>CD226 expression kinetics involve intensification followed by a reduction in surface expression during Tconv activation.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7<SEP>chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "CD226"}, {"entity_type": "CO-INHIBITORY RECEPTOR", "description": "Like CD69, ICOS, and CD226, PD-1 expression kinetics involve an intensification followed by a reduction in surface expression following Tconv activation.<SEP>Programmable death protein 1 (PD-1) is a checkpoint receptor whose interaction with ligands can inhibit T-cell activity and, thus, may influence the expression or function of genes like PTPN22.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7<SEP>chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "PD-1"}, {"entity_type": "CELL TYPE", "description": "Antigen presenting cells are critical for initiating immune responses by presenting antigens to T-cells and can modulate gene expression patterns like that of PTPN22.", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "APCS (ANTIGEN PRESENTING CELLS)"}, {"entity_type": "PATHOLOGICAL FACTOR", "description": "The presence of the risk variant (LYP-620W) in the PTPN22 gene is linked with an increased risk for autoimmune diseases through its impact on T-cell function and regulation.", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "AUTOIMMUNE DISEASE RISK"}, {"entity_type": "PROTEIN", "description": "CTLA-4 is also observed to be dynamically expressed after Treg activation.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "CTLA-4"}, {"entity_type": "DISEASE", "description": "Tconv cells are mentioned as contrast to Treg cells, indicating different immune cell types.<SEP>Tconv cells exhibit blunted activation marker kinetics upon overexpression of LYP-620R compared to mock-transfected controls.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d<SEP>chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "TCONV"}, {"entity_type": "DISEASE", "description": "Treg cells also show blunted activation marker kinetics when overexpressing LYP-620R, except for CD226 on Tconv.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "TREGS"}, {"entity_type": "ACTIVATION MARKER KINETICS", "description": "The kinetics of activation marker expressions were observed in response to anti-CD3 and anti-CD28 stimulation.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "CD3+ AND CD28+ CELLS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "PD-L1, an interaction partner of PD-1, plays a role in inhibiting the immune response.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "PD-L1"}, {"entity_type": "PROTEIN", "description": "CD28 is co-stimulatory receptor that promotes T-cell activation and survival with CD3 stimulation.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "CD28"}, {"entity_type": "MOLECULAR INTERACTION", "description": "TAP1 protein facilitates the translocation of antigen peptides into the endoplasmic reticulum for presentation on MHC class I molecules.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "TAP1"}, {"entity_type": "PROTEIN", "description": "MHC class II molecules present antigens to CD4+ T cells for recognition during adaptive immune responses.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "MHC CLASS II"}, {"entity_type": "PROTEIN", "description": "DC-SIGN is a C-type lectin that plays a role in antigen presentation and adhesion of dendritic cells with B cells, monocytes or other leukocytes.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "CD208 (DC-SIGN)"}, {"entity_type": "GENE", "description": "Data suggest that LYP gene variants may influence immune responses.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "LYP"}, {"entity_type": "GENOMIC VARIANT", "description": "Variants of the LYP gene show different effects on immune response regulation.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "LYP VARIANT 620R"}, {"entity_type": "GENOMIC VARIANT", "description": "Similar to variant 620R, but with reduced efficacy in certain contexts.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "LYP VARIANT 620W"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A Th1 cell signature is implicated in several autoimmune disorders associated with a particular genetic variant, but this was not observed when examining LYP variants.<SEP>A Th1 cell signature is implicated in the autoimmune disorders associated with a specific risk variant.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "TH1 SIGNATURE"}, {"entity_type": "CYTOKINE", "description": "Interferon-gamma (IFN\u03b3) is produced by CAR T cells in response to OAS4 and OAS7 cells, but not OAS3 cells.<SEP>The secretion of interferon-gamma (IFN\u03b3) by T cells, which is affected by the expression of PTPN22.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d<SEP>chunk-183db5c908df609bb5289896b4ba09d0", "id": "IFN\u0393"}, {"entity_type": "ANTIGEN PRESENTING CELLS", "description": "Autologous APCs play a role in immune responses as they present antigens to T cell receptors.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "APCS"}, {"entity_type": "T CELL RESPONSE", "description": "Transduced T cell response is influenced by the expression of LYP variants, with 620R showing greater repression than 620W.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "TRESP"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The capacity of regulatory T cells to suppress immune responses, which is altered when PTPN22 expression is influenced by LYP variant.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "TREG-MEDIATED SUPPRESSION"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "The ability of Tregs to modulate or inhibit the function of other T cells was assessed under different genetic conditions.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "SUPPRESSION CAPACITY"}, {"entity_type": "CYTOKINE PROCESS", "description": "The secretion of interferon-gamma (IFN\u03b3) by T cells plays a role in autoimmune disorders.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "IFN\u0393 SECRETION"}, {"entity_type": "DISEASE", "description": "Autoimmune disorders are linked to certain genetic variants and immune cell signatures, such as the Th1 signature.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "AUTOMIMMUNE DISORDERS"}, {"entity_type": "CELLULAR PROCESS", "description": "The proliferation of regulatory T (Treg) cells may be modulated by genetic factors like LYP gene variants.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "TREG PROLIFERATION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Antigen presenting cells (APCs) and regulatory T cell interactions are influenced in autoimmune disorders.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "APCS SUPPRESSION"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "The ability to modulate or inhibit the function of T cells is a critical process in immune regulation, which is influenced by genetic variants like rs2476601 and LYP gene variants.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "T CELL RESPONSE MODULATION"}, {"source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca<SEP>chunk-5b68770e1b446d6cba0d4e1ecabf497b<SEP>chunk-daf8d8b6d90cc3f92b645357f7c1294b<SEP>chunk-1b40d071abac0ba432a9fb0096796f45", "description": "CD4+ T cells were specifically analyzed for their differential response to the genomic variant and gene expression.<SEP>Endogenous PTPN22 expression varies across different subsets of CD4+ T cells (naive, memory, and regulatory) at rest and post-activation.)<SEP>The effect on trans-eQTL analysis was specific to CD4+ T cells, not detected in CD8+ T cells.<SEP>We purified CD4+ T cells from peripheral blood mononuclear cells (PBMCs) of European individuals.", "entity_type": "CELL TYPE", "id": "CD4+ T CELLS"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The level of PTPN22 expression is higher in activated compared to quiescent states for all analyzed CD4+ T cell subsets.)", "entity_type": "UNKNOWN", "id": "ACTIVATION STATE"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The level of PTPN22 expression is higher in activated compared to quiescent states for all analyzed CD4+ T cell subsets.)", "entity_type": "UNKNOWN", "id": "PTPN22 EXPRESSION"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The PTPN22 gene, encoded by different variants like LYP-620R or LYP-620W, plays a role in modulating immune responses including T-cell function and differentiation.)", "entity_type": "UNKNOWN", "id": "PTPN22 GENE"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The PTPN22 gene, encoded by different variants like LYP-620R or LYP-620W, plays a role in modulating immune responses including T-cell function and differentiation.)", "entity_type": "UNKNOWN", "id": "ROLE IN IMMUNE REGULATION"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "Calcium signaling pathways affect the expression of PTPN22 by impacting key processes like MAPK/ERK pathway activity, influencing downstream effects in T-cell function and autoimmune disease risk.)", "entity_type": "UNKNOWN", "id": "DOWNSTREAM EFFECTS ON PTPN22"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The MAPK/ERK pathway is interconnected with calcium signaling pathways, both impacting the expression of PTPN22 through coordinated regulatory mechanisms in T-cell function and autoimmune disease risk.)", "entity_type": "UNKNOWN", "id": "CONNECTION TO CALCIUM SIGNALING"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The level of pERK serves as a marker for activity within the MAPK/ERK pathway, which has downstream effects on several processes including PTPN22 expression and T-cell function.)", "entity_type": "UNKNOWN", "id": "PERK MARKER"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The level of pERK serves as a marker for activity within the MAPK/ERK pathway, which has downstream effects on several processes including PTPN22 expression and T-cell function.)", "entity_type": "UNKNOWN", "id": "SIGNIFICANCE IN DOWNSTREAM EVENTS"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "Calcium flux quantifies changes in intracellular calcium levels as a response to T-cell receptor (TCR) ligation, affecting downstream signaling cascades including those involved with PTPN22 expression and function.)", "entity_type": "UNKNOWN", "id": "RESPONSE TO TCR LIGATION"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The CD69 molecule serves as an early marker for T-cell activation which correlates with the expression levels of genes like PTPN22, impacting downstream signaling pathways and immune responses.)", "entity_type": "UNKNOWN", "id": "CD69 MARKER"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The CD69 molecule serves as an early marker for T-cell activation which correlates with the expression levels of genes like PTPN22, impacting downstream signaling pathways and immune responses.)", "entity_type": "UNKNOWN", "id": "EARLY T-CELL ACTIVATION SIGN"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The inducible costimulatory (ICOS) molecule acts as a co-stimulatory receptor on T-cells during activation processes, influencing gene expression patterns including PTPN22.)", "entity_type": "UNKNOWN", "id": "ICOS RECEPTOR"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The inducible costimulatory (ICOS) molecule acts as a co-stimulatory receptor on T-cells during activation processes, influencing gene expression patterns including PTPN22.)", "entity_type": "UNKNOWN", "id": "MODULATION BY ACTIVATED T-CELLS"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The CD25 molecule, also known as IL2R\u03b1, serves as an indicator of early T-cell activation and modulates the expression of genes such as PTPN22 through its regulatory activities.)", "entity_type": "UNKNOWN", "id": "CD25 MARKER"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The CD25 molecule, also known as IL2R\u03b1, serves as an indicator of early T-cell activation and modulates the expression of genes such as PTPN22 through its regulatory activities.)", "entity_type": "UNKNOWN", "id": "T-CELL ACTIVATION INDICATORS"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The CD226 receptor, also known as NKG2D or NKp46, plays a role in co-stimulating immune responses on T-cells and influences downstream regulatory mechanisms involving PTPN22 expression.)", "entity_type": "UNKNOWN", "id": "CD226 RECEPTOR"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The CD226 receptor, also known as NKG2D or NKp46, plays a role in co-stimulating immune responses on T-cells and influences downstream regulatory mechanisms involving PTPN22 expression.)", "entity_type": "UNKNOWN", "id": "CO-STIMULATION ROLE IN IMMUNE REGULATION"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The PD-1 receptor, alongside its ligands, contributes to immune suppression through inhibiting T-cell activity. This can have indirect effects on the regulation of genes like PTPN22 and T-cell function.)", "entity_type": "UNKNOWN", "id": "PD-1 CO-INHIBITORY RECEPTOR"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The PD-1 receptor, alongside its ligands, contributes to immune suppression through inhibiting T-cell activity. This can have indirect effects on the regulation of genes like PTPN22 and T-cell function.)", "entity_type": "UNKNOWN", "id": "ROLE IN IMMUNE SUPPRESSION"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "Antigen-presenting cells (APCs) play a crucial role in initiating immune responses by presenting antigens to T-cells, which can modulate the expression and activity of genes like PTPN22 involved in T-cell regulation.)", "entity_type": "UNKNOWN", "id": "APC CELL TYPE"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "Antigen-presenting cells (APCs) play a crucial role in initiating immune responses by presenting antigens to T-cells, which can modulate the expression and activity of genes like PTPN22 involved in T-cell regulation.)", "entity_type": "UNKNOWN", "id": "ROLE IN IMMUNE RESPONSE INITIATION"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "The presence of risk variants such as LYP-620W in the PTPN22 gene is linked with an increased risk for autoimmune diseases through its impact on modulating T-cell function and regulatory pathways.)", "entity_type": "UNKNOWN", "id": "CONNECTION VIA GENETIC VARIANTS"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "Transduced Tconv cells show reduced APC-mediated proliferation upon overexpression of LYP-620R compared to mock-transfected controls.)", "entity_type": "UNKNOWN", "id": "APC"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "Transduced Tconv cells show reduced APC-mediated proliferation upon overexpression of LYP-620R compared to mock-transfected controls.)", "entity_type": "UNKNOWN", "id": "TCONVE"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "CD4+ effector T cells modulate host immune responses through cytokine production after encountering cognate antigens.)", "entity_type": "UNKNOWN", "id": "CD4+ EFFECTOR T CELLS"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7<SEP>chunk-183db5c908df609bb5289896b4ba09d0", "description": "Interleukin-2 (IL-2) is produced by TAG72-BB\u03b6 CAR T cells against OAS4 cells, but not OAS3 and OAS7 cells.<SEP>Overexpression of PTPN22 influences IL-2 secretion; no alteration by LYP-620R, increase at the 24-hour time point by LYP-620W.)", "entity_type": "CYTOKINE", "id": "IL-2"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "Overexpression of PTPN22 influences IL-2 secretion; no alteration by LYP-620R, increase at the 24-hour time point by LYP-620W.)", "entity_type": "UNKNOWN", "id": "INFLUENCE BY LYP VARIANTS"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "The secretion of these cytokines was robustly inhibited by expression of either LYP variant for the first 48 hours.)", "entity_type": "UNKNOWN", "id": "IL-4, IL-5, IL-9, IL-10, AND IL-13"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "The secretion of these cytokines was robustly inhibited by expression of either LYP variant for the first 48 hours.)", "entity_type": "UNKNOWN", "id": "INHIBITION BY PTPN22 VARIANTS"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "IFN\u03b3 secretion is not altered by overexpression of either variant of LYP, despite its association with autoimmune disorders.)", "entity_type": "UNKNOWN", "id": "NO ALTERATION BY PTPN22 VARIANTS"}, {"source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "description": "The LYP variant 620W affects the Tcell intrinsic susceptibility and modulates activation, proliferation, and effector mechanisms rather than impacting directly on suppressive functions of Tregs.)", "entity_type": "UNKNOWN", "id": "TCONV INTRINSIC SUSCEPTIBILITY"}, {"source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "description": "Data suggest that both LYP variants may influence immune responses but show different effects when compared to PTPN22 expression in Treg cells.)", "entity_type": "UNKNOWN", "id": "LYP VARIANT 620R/LYP VARIANT 620W"}, {"source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "description": "While the suppression capacity of Tregs was slightly increased with variant 620W, no significant difference was observed in Treg proliferation compared to variants 620R or eGFP reporter Tregs.)", "entity_type": "UNKNOWN", "id": "SUPPRESSION CAPACITY AND PROLIFERATION"}, {"source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "description": "The genomic variant rs2476601 affects the risk for several autoimmune disorders through its association with a Th1 cell signature and IFN\u03b3 secretion.)", "entity_type": "UNKNOWN", "id": "AUTOIMMUNE DISORDERS"}, {"source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "description": "The Th1 signature associated with rs2476601 is involved in the development of autoimmune disorders.)", "entity_type": "UNKNOWN", "id": "DISEASE ASSOCIATION"}, {"source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "description": "The Th1 signature associated with rs2476601 is involved in the development of autoimmune disorders.)", "entity_type": "UNKNOWN", "id": "TH1 SIGNATURE/RS2476601"}, {"entity_type": "CELL", "description": "TAG72-CAR T cells containing a 4-1BB intracellular co-stimulatory domain demonstrate in vitro activation against purified TAG72.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "TAG72-CAR T CELLS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The humanized scFv CC49, the human IgG4 Fc extracellular spacer lacking a CH2 domain (\u0394CH2), and CD4 transmembrane domain are linked to the 4-1BB intracellular co-stimulatory domain for cell activation.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "4-1BB INTRACELLULAR CO-STIMULATORY DOMAIN"}, {"entity_type": "CELLULAR COMPONENT", "description": "The CD3\u03b6 cytolytic domain is a part of the TAG72-BB\u03b6 CAR construct, which contributes to cell proliferation and killing ability.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CD3\u0396 CYTOLYTIC DOMAIN"}, {"entity_type": "ANTIBODY", "description": "A humanized antibody fragment that binds to TAG72 antigen, which is attached to the 4-1BB co-stimulatory domain.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "HUMANIZED SCFV CC49"}, {"entity_type": "PROTEIN", "description": "This part of the Fc region provides a linker between scFv CC49 and CD3\u03b6 cytolytic domain, without a CH2 domain.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "HUMAN IGG4 FC EXTRACELLULAR SPACER LACKING A CH2 DOMAIN (\u0394CH2)"}, {"entity_type": "CELLULAR COMPONENT", "description": "Part of the T-cell receptor complex that is involved in signal transduction for activation.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CD4 TRANSMEMBRANE DOMAIN"}, {"entity_type": "MOLECULAR CONSTRUCTION", "description": "A genetic modification to create T cells capable of specifically recognizing and killing cells expressing TAG72 antigen, using 4-1BB co-stimulatory domain and CD3\u03b6 cytolytic domain.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "TAG72-BB\u0396 CAR CONSTRUCT"}, {"entity_type": "MARKER", "description": "This truncated form of CD19 helps in tracking the CAR T cells during their manufacturing process.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CD19T FOR CELL TRACKING"}, {"entity_type": "CELL", "description": "T-cells that were not transduced with the TAG72-BB\u03b6 CAR construct for comparison.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "MOCK (UNTRANSDUCED) T CELLS"}, {"entity_type": "CYTOKINE AND CELL SURFACE MARKER", "description": "Expression of this co-stimulatory molecule is a measure of T cell activation when cultured with plate-bound, but not soluble, purified TAG72.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CD137 EXPRESSION"}, {"entity_type": "CYTOKINE AND CELL SURFACE MARKER", "description": "Another marker for T cell activation, which is induced by plate-bound TAG72.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CELL-SURFACE CD69 EXPRESSION"}, {"entity_type": "CYTOKINE", "description": "Interferon gamma is a pro-inflammatory cytokine that indicates an active immune response against pathogens or tumor cells when induced by antigen-presenting cells (APCs).", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "IFN\u0393 RELEASE"}, {"entity_type": "CELL LINE", "description": "Human ovarian cancer cell line expressing TAG72, used in various functional assays.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "OVCAR3 CELLS"}, {"entity_type": "CELL LINE", "description": "Human ovarian cancer cell line expressing a higher level of TAG72 compared to OVCAR3.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "OV90 CELLS"}, {"entity_type": "CELL LINE", "description": "An ovarian cancer cell line with very low levels detected for TAG72 expression, used as negative control in the experiments.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "COV362.4 CELLS"}, {"entity_type": "CELL LINE", "description": "Cells that do not express TAG72 antigen or express it at a lower level, used for comparison to TAG72-expressing cell lines.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "SKOV3 AND OVCAR8 CELLS"}, {"entity_type": "CELL LINE", "description": "A colon cancer line expressing TAG72, used in some experiments to confirm the specificity of CAR T cells against ovarian cancer targets.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "LS174T"}, {"entity_type": "TUMOR SAMPLE", "description": "Cells isolated directly from tumor-bearing animals for in vitro evaluation of T cell activity against ovarian cancer.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "OVCAR3 AND OV90 CELLS HARVESTED FROM ASCITES"}, {"entity_type": "BIOLOGICAL FUNCTION", "description": "Ability to kill ovarian cancer cells that express the TAG72 antigen, as assessed by co-culture assays with E:T ratios ranging from 1:1 to 1:2.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "TAG72-BB\u0396 CAR T CELLS KILLING POTENTIAL"}, {"entity_type": "FUNCTIONAL PROPERTY", "description": "Killing ability of CAR T cells towards targets is dependent on the presence of a specific antigen (TAG72) that binds to the CAR construct.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "ANTIGEN-DEPENDENT ACTIVITY"}, {"entity_type": "FUNCTIONALITY ASSESSMENT", "description": "Confirmation that CAR T cells are targeted against the antigen-specific cell lines expressing TAG72 and not against non-expressing or low-expression cell lines.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "SPECIFICITY OF TAG72-CAR T CELLS"}, {"entity_type": "BIOLOGICAL FUNCTION", "description": "The capacity of TAG72-BB\u03b6 CAR T cells to eliminate ovarian cancer cells through direct cytotoxicity, as tested against various cell lines at different time points (24 h and 72 h).", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "TUMOR KILLING ABILITY"}, {"entity_type": "RESPONSE MEASURE", "description": "Indicators of an active immune response or T cell activation in response to antigen-presenting cells (APCs) and target cells expressing TAG72.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CYTOKINE PRODUCTION FROM CAR T CELLS"}, {"entity_type": "CYTOKINE", "description": "Sustained production over 72 h indicates a prolonged anti-tumor immune response, possibly correlating with the persistence of CAR T cells against tumor targets.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "IFN\u0393 LEVELS"}, {"entity_type": "CYTOKINE RESPONSE", "description": "Initial phase of cytokine production by T-cells that is detectable and peaks at earlier times (up to 24 h) when co-cultured with antigen-positive ovarian cancer targets.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "IL-2 PRODUCTION PEAK AT EARLY TIME POINTS"}, {"entity_type": "QUANTITATIVE MEASURE", "description": "The amount of TAG72 antigen present on tumor cells that are isolated from the ovarian cancer ascites fluid.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "TAG72 EXPRESSION LEVELS IN ASCITES FROM DIFFERENT PATIENTS"}, {"entity_type": "TEMPORAL CHANGE", "description": "Downregulation of TAG72 surface expression by cancer cells during culturing, potentially due to mechanisms such as shedding or loss of antigenic expression.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "REDUCTION IN TAG72 EXPRESSION OVER TIME"}, {"entity_type": "ASSAY TYPE", "description": "Methods used to measure killing ability of T cells against target cells.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CYTOTOXICITY ASSAYS"}, {"entity_type": "RATIO MEASUREMENT", "description": "The effector-to-target cell ratio during co-culture experiments, indicating how many T cells are provided for every target cell.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "E:T RATIOS (1:1, 1:2)"}, {"entity_type": "ASSAY TYPE", "description": "Methods used to monitor the growth and division of cells over time.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CELL PROLIFERATION ASSAYS"}, {"entity_type": "ANALYTICAL METHOD", "description": "Method used to detect and quantify proteins at the molecular level in a sample.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "WESTERN BLOTTING"}, {"entity_type": "ASSAY TYPE", "description": "A method for detecting or quantifying molecules like cytokines, antibodies, and other proteins.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "ELISA (ENZYME-LINKED IMMUNOSORBENT ASSAY)"}, {"entity_type": "STORAGE METHOD", "description": "Preservation of biological samples at extremely low temperatures (-196\u00b0C) to maintain cell viability for future use.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CRYOPRESERVATION OF T CELLS"}, {"entity_type": "CLINICAL TRIAL STAGE", "description": "Early-stage trials conducted in a laboratory or animal model, aimed at assessing safety and feasibility.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "PRECLINICAL TRIALS"}, {"entity_type": "CLINICAL TRIAL STAGE", "description": "Initial human study to evaluate the immediate side effects of new treatments.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "PHASE I CLINICAL TRIAL"}, {"entity_type": "PHARMACOLOGICAL ANALYSIS", "description": "Study of how the body affects a drug, including absorption, distribution, metabolism, and excretion.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "PHARMACOKINETICS AND PHARMACODYNAMICS (PK/PD)"}, {"entity_type": "OUTCOME MEASURE", "description": "Criteria that are used to assess the effectiveness or impact of a treatment in clinical trials.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CLINICAL ENDPOINTS"}, {"entity_type": "OUTCOME ASSESSMENT", "description": "Experiments designed to measure how well a particular therapy works in achieving its intended outcomes.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "EFFICACY STUDIES"}, {"entity_type": "STUDY DESIGN", "description": "Ongoing monitoring and assessment of the results and impacts after an intervention has been applied.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "LONG-TERM FOLLOW-UP"}, {"entity_type": "STATISTICAL METHOD", "description": "Analysis used to estimate the length of time before a certain event occurs, typically death or disease recurrence.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "SURVIVAL ANALYSIS"}, {"entity_type": "OUTCOME MEASURE", "description": "Percentage of patients responding positively to treatment as defined by predefined criteria.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "RESPONSE RATE"}, {"entity_type": "CLINICAL OUTCOME", "description": "Decrease in the size or mass of a tumor following treatment.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "TUMOR BURDEN REDUCTION"}, {"entity_type": "PATIENT EVALUATION", "description": "Measurements that assess how patients\' daily activities and personal well-being are affected by their condition or treatment.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "QUALITY-OF-LIFE ASSESSMENTS"}, {"entity_type": "TOXICITY ASSESSMENT", "description": "Comprehensive evaluations of the adverse effects associated with drug usage.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "DRUG SAFETY PROFILES"}, {"entity_type": "PROTEIN", "description": "TAG72 is a protein expressed in ovarian cancer ascites.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "TAG72"}, {"entity_type": "TRANSCRIPT", "description": "The transcripts of the OAS3, OAS4, and OAS7 cells express TAG72 by flow cytometry.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "OAS3, OAS4, OAS7"}, {"entity_type": "CELL", "description": "CAR T cells are used for cancer treatment through ex vivo culturing and subsequent intravenous or intraperitoneal delivery.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "CAR T CELLS"}, {"entity_type": "CONTROL CELL", "description": "Mock T cells are a control group without CAR, compared with CAR T cells in cancer treatments.<SEP>Mock T cells lack CAR, serving as comparison to active CAR T cells.<SEP>Mock T cells lack the Chimeric Antigen Receptor (CAR), serving as a comparison to active CAR T cells in cancer treatments.<SEP>T cells without CAR modification, used as a control for treatment response.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf<SEP>chunk-183db5c908df609bb5289896b4ba09d0", "id": "MOCK T CELLS"}, {"entity_type": "CAR T CELL VARIANT", "description": "Protein derived from combined gene therapy of T cells and BB\u03b6 antibodies to target ovarian cancer.<SEP>TAG72-BB\u03b6 CAR T cells include an antibody against TAG72 and the BzIL-1B\u03b6, which have shown potent anti-tumor activity.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf<SEP>chunk-183db5c908df609bb5289896b4ba09d0", "id": "TAG72-BB\u0396 CAR T CELLS"}, {"entity_type": "CELL LINE", "description": "OVCAR3 cells are a type of ovarian cancer cell line used for in vitro studies.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "OVCAR3"}, {"entity_type": "CELL LINE", "description": "OV90 is another type of ovarian cancer cell line used for in vitro studies with uniform TAG72 expression.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "OV90"}, {"entity_type": "DISEASE", "description": "These cancers are represented by the modelled mice and human conditions, where tumors grow within the peritoneum.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "INVASIVE PERITONEAL OVARIAN CANCERS"}, {"entity_type": "TRANSCRIPT", "description": "The transcript of cell line OAS3 expresses TAG72 proteins.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "OAS3"}, {"entity_type": "TRANSCRIPT", "description": "The transcript of cell line OAS4 expresses TAG72 proteins.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "OAS4"}, {"entity_type": "TRANSCRIPT", "description": "The transcript of cell line OAS7 expresses TAG72 proteins.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "OAS7"}, {"entity_type": "TREATMENT METHOD", "description": "TAG72-BB\u03b6 CAR T cells are administered by regional intraperitoneal delivery, showing potent anti-tumor activity against peritoneal ovarian tumors in mice.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "REGIONAL INTRAPERITONEAL DELIVERY"}, {"entity_type": "DELIVERY ROUTE", "description": "TAG72-BB\u03b6 CAR T cells are also delivered systemically through intravenous injection, but with limited therapeutic efficacy compared to regional intraperitoneal delivery.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "SYSTEMIC INTRAVENOUS DELIVERY"}, {"entity_type": "DIAGNOSTIC TOOL", "description": "Optical imaging is used non-invasively to track the growth of transduced OVCAR3 tumor cells expressing eGFP/ffluc in mice.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "NON-INVASIVE OPTICAL IMAGING"}, {"entity_type": "DISEASE", "description": "Models used to study the effects of TAG72-BB\u03b6 CAR T Cells in ovarian cancer.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "OVARIAN CANCER XENOGRAFT MODELS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The rate at which tumors grow compared with different dosing regimens.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "RELATIVE TUMOR GROWTH KINETICS"}, {"entity_type": "DISEASE", "description": "Duration of survival after treatment, as measured by the time difference between single and repeat doses.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "OVERALL SURVIVAL"}, {"entity_type": "TRANSCRIPT", "description": "Quantitative measure of T cells within specific sites in the body after CAR T cell therapy.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "T CELL NUMBERS"}, {"entity_type": "THERAPY", "description": "Type of treatment that delivers CAR T cells to a specific location in the body for localized cancer control.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "REGIONALLY DELIVERED CAR T CELLS"}, {"entity_type": "DISEASE", "description": "Reemergence of tumors after initial therapy, often due to antigen loss or escape.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "TUMOR RECURRENCES"}, {"entity_type": "TREATMENT", "description": "The process of introducing CAR T cells into the body to combat cancer.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "T CELL INFUSION"}, {"entity_type": "CONTROL TREATMENT", "description": "A treatment without active components for comparison with experimental treatments.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "MOCK TREATMENT"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The reduction or disappearance of specific antigens on cell surfaces following CAR T cell therapy, leading to tumor resistance.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "ANTIGEN LOSS"}, {"entity_type": "CELLULAR PROPERTY", "description": "Variation within the cells composing a solid tumor that can influence treatment responses and recurrence.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "HETEROGENEITY IN SOLID TUMORS"}, {"entity_type": "ANTIGEN MARKER", "description": "Potential target antigen identified in ovarian cancer, often found alongside MUC1 and TAG72.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "MUC16"}, {"entity_type": "ANTIGEN MARKER", "description": "An established tumor antigen that may be expressed on cells within ovarian tumors.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "MUC1"}, {"entity_type": "EXPERIMENTAL SETUP", "description": "The process of growing cancer cells alongside CAR T cells to study their interactions and outcomes.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "TUMOR CELL CO-CULTURE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Decrease in the strength or efficiency of antigen recognition by CAR T cells due to reduced affinity for specific antigens.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "AVIDITY REDUCTION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Characteristics of a particular tumor type that may impact treatment response and outcome.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "AGGRESSION OF A TUMOR MODEL"}, {"entity_type": "ANTIGEN MARKER", "description": "Presence of this ovarian cancer antigen on the cell surface that may influence T cell recognition and efficacy.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "MUC1 EXPRESSION"}, {"entity_type": "OVARIAN CANCER SUBTYPE", "description": "Specific subtype or variant of ovarian cancer used in experiments to study CAR T cell therapy effects.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "OV90 MODEL"}, {"entity_type": "DISEASE MANAGEMENT OUTCOME", "description": "Signs of tumor shrinkage following treatment, indicating successful intervention.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "TUMOR REGRESSIONS"}, {"entity_type": "HOST CONDITION", "description": "Mice with OV90 tumors used to study the impact of CAR T cell therapy under specific tumor conditions.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "OV90-BEARING MICE"}, {"entity_type": "CELLULAR COMPONENT", "description": "Cells that remain after treatment, potentially contributing to recurrence if not fully eliminated.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "VIABLE RESIDUAL TUMOR CELLS"}, {"entity_type": "TREATMENT SCHEDULE", "description": "The administration of TAG72-BB\u03b6 CAR T Cells at one point in time.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "SINGLE DOSING REGIMEN"}, {"entity_type": "TEMPORAL METRIC", "description": "Time period shortly after CAR T cell infusion that can influence therapeutic outcomes.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "EARLY TIME POINTS"}, {"entity_type": "ORGANIZATIONAL ENVIRONMENT", "description": "The local environment surrounding the tumor including stromal cells, blood vessels, and other factors that impact treatment.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "TUMOR MICROENVIRONMENT"}, {"entity_type": "RESISTANCE FACTOR", "description": "Mechanisms by which tumors evade CAR T cell therapy independent of antigen-specific mechanisms.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "AGNOSTIC RESISTANCE MECHANISMS"}, {"entity_type": "EXPERIMENTAL DESIGN", "description": "Model used to assess interactions between CAR T cells and tumor cells in vitro.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "CAR T CELL CO-CULTURE MODEL"}, {"entity_type": "CELLULAR COMPONENT", "description": "Cancerous cells remaining after CAR T cell treatment, potentially contributing to recurrence.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "RESIDUAL TUMOR CELLS"}, {"entity_type": "MOLECULE", "description": "Nucleic acid-based molecules that bind specifically to antigens and can enhance CAR T cell targeting or functionality.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "APTAMERS"}, {"entity_type": "PROTEIN", "description": "Antibody fragment used in combination with T cells for enhanced targeting of ovarian cancer tumors.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "BB\u0396 ANTIBODIES"}, {"entity_type": "DISEASE", "description": "Bone condition characterized by the loss of blood supply to bone tissue, potentially related to CAR T cell treatment effects.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "AVASCULAR NECROSIS"}, {"entity_type": "THERAPY", "description": "Treatment strategy using genetically engineered T cells that express chimeric antigen receptors for cancer immunotherapy.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "CAR T CELL THERAPY"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "Process by which tumors regenerate after treatment, possibly involving stem cell-like cells or other factors.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "TUMOR REGENERATION FACTOR"}, {"entity_type": "QUANTITATIVE MEASURE", "description": "Number of infectious viral particles present in the body, potentially related to immune response or therapy side effects.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "VIRAL LOAD"}, {"source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "description": "Mock T cells do not exhibit detectable anti-tumor activity towards TAG72-negative cell lines such as OAS3, unlike CAR T cells.)", "entity_type": "UNKNOWN", "id": "ANTI-TUMOR ACTIVITY"}, {"source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "description": "Interferon-gamma production by CAR T cells is specific to TAG72-positive cell types like OAS4 and OAS7, but not OAS3.)", "entity_type": "UNKNOWN", "id": "RESPONSE TO CELL TYPE"}, {"source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "description": "Interleukin-2 secretion by TAG72-BB\u03b6 CAR T cells specifically targets and kills OAS4 cells while sparing other cell types like OAS3 and OAS7.)", "entity_type": "UNKNOWN", "id": "CAR RESPONSE"}, {"source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "description": "TAG72-BB\u03b6 CAR T cells treated by regional intraperitoneal delivery demonstrate more effective anti-tumor activity than those administered systemically through intravenous injection.)", "entity_type": "UNKNOWN", "id": "THERAPEUTIC EFFICACY"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Multiple dosing of TAG72-BB\u03b6 CAR T Cells demonstrated more extensive tumor regression and durable control compared with a single dose.)", "entity_type": "UNKNOWN", "id": "DOSING REGIMENS"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Multiple dosing of TAG72-BB\u03b6 CAR T Cells demonstrated more extensive tumor regression and durable control compared with a single dose.)", "entity_type": "UNKNOWN", "id": "THERAPEUTIC STRATEGY"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "A significantly extended survival benefit was observed in mice receiving repeat doses of TAG72-BB\u03b6 CAR T Cells over a single dose.)", "entity_type": "UNKNOWN", "id": "OVERALL SURVIVAL BENEFIT"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Increased numbers of T cells were noted within peritoneal tumors in mice with repeat treatment for TAG72-BB\u03b6 CAR T Cells compared to a single dose.)", "entity_type": "UNKNOWN", "id": "DOSAGE INFLUENCE"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "The efficacy of CAR T cells can be affected by lifestyle factors like diet and physical activity, which modulate the tumor microenvironment.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 19}]", "id": "CAR T CELL EFFICACY"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Tumor recurrences following TAG72-BB\u03b6 CAR T cell therapy are often due to antigen escape or loss.)", "entity_type": "UNKNOWN", "id": "RESISTANCE MECHANISM"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Tumor recurrences following TAG72-BB\u03b6 CAR T cell therapy are often due to antigen escape or loss.)", "entity_type": "UNKNOWN", "id": "TUMOR RECURRENCE"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Alteration of TAG72 expression levels over time, particularly during tumor recurrence and post-TAG72-BB\u03b6 CAR T Cell therapy.)", "entity_type": "UNKNOWN", "id": "ANTIGEN EXPRESSION DYNAMICS"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Alteration of TAG72 expression levels over time, particularly during tumor recurrence and post-TAG72-BB\u03b6 CAR T Cell therapy.)", "entity_type": "UNKNOWN", "id": "TEMPORAL CHANGE IN ANTIGENS"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Tumor recurrences can be predicted based on antigen dynamics following CAR T cell treatment in different models.)", "entity_type": "UNKNOWN", "id": "OUTCOME PREDICTION"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Tumor recurrences can be predicted based on antigen dynamics following CAR T cell treatment in different models.)", "entity_type": "UNKNOWN", "id": "RECURRENT TUMOR MODELS"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Interaction between aptamers and antigens on tumor cells to enhance CAR T cell efficacy.)", "entity_type": "UNKNOWN", "id": "APTAMER BINDING"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Interaction between aptamers and antigens on tumor cells to enhance CAR T cell efficacy.)", "entity_type": "UNKNOWN", "id": "INTERPLAY BETWEEN MOLECULES"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Unpredictable mechanisms that prevent CAR T cell effectiveness, beyond antigen-specific resistance.)", "entity_type": "UNKNOWN", "id": "AGNOSTIC RESISTANCE"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Unpredictable mechanisms that prevent CAR T cell effectiveness, beyond antigen-specific resistance.)", "entity_type": "UNKNOWN", "id": "CONSEQUENCE OF T CELL TREATMENT"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Evaluating how long CAR T cells remain viable and active within the body during treatment.)", "entity_type": "UNKNOWN", "id": "CAR T CELL SURVIVAL"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Evaluating how long CAR T cells remain viable and active within the body during treatment.)", "entity_type": "UNKNOWN", "id": "OUTCOME MEASUREMENT"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "The duration needed for bone tissue to recover from potential side effects of CAR T cell therapy or associated conditions like avascular necrosis.)", "entity_type": "UNKNOWN", "id": "RECOVERY TIME FOR BONE TISSUE"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "The duration needed for bone tissue to recover from potential side effects of CAR T cell therapy or associated conditions like avascular necrosis.)", "entity_type": "UNKNOWN", "id": "TIMEFRAME"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Potential negative consequences of CAR T cell treatment that need to be managed or mitigated.)", "entity_type": "UNKNOWN", "id": "ADVERSE EFFECTS"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Potential negative consequences of CAR T cell treatment that need to be managed or mitigated.)", "entity_type": "UNKNOWN", "id": "THERAPEUTIC LIMITATION"}, {"entity_type": "GENE", "description": "The C10orf11 gene is located on chromosome 10q22.3 and was associated with FEV1/FVC lung function.", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "id": "C10ORF11 GENE"}, {"entity_type": "GENE", "description": "The CHAF1B gene is located on chromosome 21q22.13 and was associated with the FEV1/FVC ratio.", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "id": "CHAF1B GENE"}, {"entity_type": "GENE CLUSTER", "description": "The CCL23-CCL18 gene cluster on chromosome 17q12 had associations with two phenotypes in this study.", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "id": "CCL23-CCL18 GENE CLUSTER"}, {"entity_type": "GENE", "description": "The PITPNC1 gene is located on chromosome 17q24.2 and had an association with FEV1/FVC lung function.", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "id": "PITPNC1 GENE"}, {"entity_type": "GENE", "description": "The IL37 gene is located on chromosome 2q13 and had an association with FEV1 lung function.", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "id": "IL37 GENE"}, {"entity_type": "GENE", "description": "The ASXL3 gene is located on chromosome 18q12.1 and had an association with FVC lung function.", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "id": "ASXL3 GENE"}, {"entity_type": "STATISTICAL MODEL", "description": "A multimarker model using LASSO regression included 80 SNPs that provided the smallest mean square error of FEV1/FVC lung function.", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "id": "LASSO REGRESSION MODEL"}, {"entity_type": "GENETIC METRIC", "description": "Total number of \'low FEV1/FVC alleles\' was used to assess the phenotypic effects in the multimarker model analysis.", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "id": "TOTAL NUMBER OF \'LOW FEV1/FVC ALLELES\'"}, {"entity_type": "FUNCTIONAL ANALYSIS TOOL", "description": "The GRAIL algorithm is used here to analyze connections between genes based on data.<SEP>The GRAIL algorithm was used to mine Gene Ontology databases for functional connections between genes near the predictive SNPs.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2<SEP>chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "id": "GRAIL ALGORITHM"}, {"entity_type": "GENOMIC VARIANT", "description": "The \'low FEV1/FVC alleles\' refer to specific variants present in a bin.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "LOW FEV1/FVC ALLELES"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The mean residual FEV1/FVC is being monitored and found to decrease with an increasing number of \'low FEV1/FVC alleles\', implying genetic influence on lung function.<SEP>This process is the average measurement of forced expiratory volume against time, used here as an indicator of lung function.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "MEAN RESIDUAL FEV1/FVC"}, {"entity_type": "GENOMIC VARIANT", "description": "The number indicates how many specific genetic variants are present related to lower than normal FEV1/FVC.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "NUMBER OF \'LOW FEV1/FVC ALLELES\'"}, {"entity_type": "MOLECULAR INTERACTION", "description": "An additive pattern means that the effect of each variant adds up, contributing to overall changes in a trait.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "ADDITIVE GENETIC ARCHITECTURE"}, {"entity_type": "DATABASE", "description": "A comprehensive resource for information about gene functions and biological processes.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "GENE ONTOLOGY DATABASE"}, {"entity_type": "RELATIONSHIP", "description": "Refers to the association found between genes near predictive SNPs and other genes through GRAIL analysis.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "FUNCTIONAL CONNECTION"}, {"entity_type": "GENE", "description": "Genes identified with significant functional connections to predictive SNPs.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "CLUSTERS OF \u0392-DEFENSIN GENES, TWO CHEMOKINE GENES (CCL18, CXCL12), TNFRSF13B"}, {"entity_type": "GENOMIC VARIANT", "description": "GWAS loci refer to genomic locations where genetic variations associated with a trait are found.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "THSD4-UACA-TLE3, C10ORF11, CCL23-CCL18 LOCUS"}, {"entity_type": "GENOMIC VARIANT", "description": "This specific variant is related to \u03b2-defensin genes, as well as to CXCL12 and TNFRSF13B based on GRAIL analysis.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "SNP AT THE CCL23-CCL18 LOCUS"}, {"entity_type": "MOLECULAR INTERACTION/", "description": "Explained Variance", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "5.8% OF THE RESIDUAL VARIANCE IN FEV1/FVC"}, {"entity_type": "GENOMIC VARIANT/", "description": "SNP", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "80 PREDICTIVE SNPS"}, {"entity_type": "GENOMIC VARIANT/GWAS LOCUS/", "description": "Locus", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "REPLICATED LOCI, THSD4-UACA-TLE3 AND C10ORF11"}, {"entity_type": "EXPLAINED VARIANCE/EXPLAINED VARIANCE/", "description": "Statistical Component", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "6.8% OF THE RESIDUAL VARIANCE IN FEV1/FVC"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "SNP rs12441227 at the THSD4 locus has a genome-wide significant association with FEV1/FVC lung function.)", "entity_type": "UNKNOWN", "id": "RS12441227 SNP"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "SNP rs12441227 at the THSD4 locus has a genome-wide significant association with FEV1/FVC lung function.)", "entity_type": "UNKNOWN", "id": "THSD4 GENE"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "The heritability of FVC and FEV1/FVC included both additive and non-additive genetic variance components, indicating complex molecular interactions.)", "entity_type": "UNKNOWN", "id": "GENETIC VARIANCE"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "The heritability of FVC and FEV1/FVC included both additive and non-additive genetic variance components, indicating complex molecular interactions.)", "entity_type": "UNKNOWN", "id": "NON-ADDITIVE GENETIC VARIANCE"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "Excluding individuals with asthma led to increased evidence for association at some loci, suggesting that disease status may influence genetic associations.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}]", "id": "ASTHMA EXCLUSION"}, {"source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "description": "Not found to be functionally connected with other genes defined by the 80 SNPs, based on the analysis.)", "entity_type": "UNKNOWN", "id": "THSD4-UACA-TLE3, C10ORF11"}, {"source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "description": "Explains a proportion of variability in forced expiratory volume against time due to genetic factors.)<SEP>The statistical connection of this specific SNP variant to FEV1/FVC is a contributing factor explaining part of its variability.)", "entity_type": "UNKNOWN", "id": "EXPLAINED VARIANCE IN FEV1/FVC"}, {"source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "description": "association", "entity_type": "UNKNOWN", "id": "FUNCTIONAL CONNECTION/"}, {"entity_type": "GENOMIC VARIANT", "description": "This SNP was reported in 14 studies out of the total 16.", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "id": "RS231775"}, {"entity_type": "GENOMIC VARIANT", "description": "This SNP was reported in 6 studies out of the total 16.", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "id": "RS231725"}, {"entity_type": "GENOMIC VARIANT", "description": "This SNP was reported in 9 studies out of the total 16.", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "id": "RS3087243"}, {"entity_type": "GENOMIC VARIANT", "description": "This SNP was reported in 5 studies out of the total 16.", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "id": "RS5742909"}, {"entity_type": "GENE", "description": "The gene associated with polymorphisms rs231775, rs231725, rs3087243, and rs5742909", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "id": "CTLA4"}, {"entity_type": "DISEASE", "description": "The disease which susceptibility was being studied through aforementioned SNPs of the gene CTLA4.", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "id": "PBC"}, {"source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "description": "The G versus A comparison for rs231775 polymorphism showed significant heterogeneity, and thus a random-effect model was used to generate extensive CIs in these genetic models.)", "entity_type": "UNKNOWN", "id": "G VERSUS A"}, {"source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "description": "The A versus G comparison for rs231725 polymorphism showed significant heterogeneity, leading to the use of a random-effect model.)", "entity_type": "UNKNOWN", "id": "A VERSUS G"}, {"source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "description": "For rs3087243 polymorphism, while GA versus GG comparison showed some heterogeneity, no significant association with PBC susceptibility was identified.)", "entity_type": "UNKNOWN", "id": "GA VERSUS GG"}, {"source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "description": "The TT + TC versus CC comparison for rs5742909 polymorphism revealed significant heterogeneity and linkage disequilibrium among the four SNPs.)", "entity_type": "UNKNOWN", "id": "TC VERSUS CC"}, {"source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "description": "Both polymorphisms showed significant association with PBC susceptibility for both Asians and Caucasians in subgroup analyses.)", "entity_type": "UNKNOWN", "id": "RS231775 AND RS231725"}, {"source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "description": "No obvious publication bias was identified through these methods, suggesting the reliability of our results.)", "entity_type": "UNKNOWN", "id": "EGGER\'S REGRESSION AND BEGG\'S FUNNEL PLOT"}, {"entity_type": "SAMPLE TYPE", "description": "The samples were isolated as peripheral blood mononuclear cells from the individuals.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "PERIPHERAL BLOOD MONONUCLEAR CELLS (PBMCS)"}, {"entity_type": "METHODOLOGY", "description": "We performed genome-wide gene expression analysis to identify genetic control of gene expression in T cells.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "GENOME-WIDE GENE EXPRESSION ANALYSIS"}, {"entity_type": "TYPE OF REGULATORY SNPS", "description": "We identified cis-eQTLs for regulation within 1Mb intervals for CD4+ and CD8+ T cells.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "CIS-EQTLS (CIS-REGULATORY SINGLE NUCLEOTIDE POLYMORPHISMS)"}, {"entity_type": "GENE", "description": "The IKBKE gene plays an essential role in regulating inflammatory responses to viral infections.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "IKBKE GENE"}, {"entity_type": "GENE", "description": "STAT6 is involved in T cell differentiation as a transcriptional activator.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "STAT6 TRANSCRIPTIONAL ACTIVATOR"}, {"entity_type": "GENE", "description": "STX2 is associated with intracellular transport of vesicles.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "STX2 (SYNTAXIN 2) INVOLVEMENT IN INTRACELLULAR TRANSPORT"}, {"entity_type": "RNA", "description": "Long coding RNA located near the IKBKE gene.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "LONG CODING RNA RP11\u2212534L20.5"}, {"entity_type": "GENE", "description": "CTLA4 is associated with T1D and other endocrine autoimmune diseases.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "INHIBITORY IMMUNE CHECKPOINT GENE CTLA4"}, {"entity_type": "ANALYTICAL METHOD", "description": "Standard tissue-by-tissue analysis was conducted to assess eQTL effects.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "TISSUE-BY-TISSUE ANALYSIS"}, {"entity_type": "ANALYTICAL METHOD", "description": "A multi-tissue joint cis-eQTL analysis accounts for incomplete power and provides a more comprehensive view.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "MULTI-TISSUE JOINT CIS-EQTL ANALYSIS"}, {"entity_type": "STATISTICAL APPROACH", "description": "These Bayesian methods were used to combine information across genes in a joint framework.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "BAYESIAN MODEL AVERAGING AND HIERARCHICAL MODELLING"}, {"entity_type": "GENETIC LOCATION", "description": "The genetic control of the expression within the CTLA4 gene locus was studied.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "CTLA4 GENE LOCUS"}, {"entity_type": "EXON REGULATION", "description": "The expression regulation was investigated through the alternative splicing of exons in CTLA4.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "ALTERNATIVE SPLICING OF EXONS 3RD AND 4TH"}, {"entity_type": "METHYLATION VARIATION", "description": "Divergent DNA methylation between CD4+ and CD8+ T cells was detected for certain genes.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "DIFFERENTIAL METHYLATION"}, {"entity_type": "SHARED REGULATORY EFFECTS", "description": "All 3871 genes with eQTLs showed strong posterior probability of being shared in both cell types.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "CIS-EQTLS SHARED BETWEEN CD4+ AND CD8+ T CELLS"}, {"entity_type": "DISEASE AND GENETIC LINK", "description": "The CT60 polymorphism was associated with T1D and RA, linking genetic variants to autoimmune diseases.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "CTLA4 AND DISEASE ASSOCIATION"}, {"entity_type": "PROTEIN", "description": "CTLA4\u662f\u4e0e\u81ea\u8eab\u514d\u75ab\u6027\u75be\u75c5\u59821\u578b\u7cd6\u5c3f\u75c5(T1D)\u76f8\u5173\u7684\u514d\u75ab\u68c0\u67e5\u70b9\u86cb\u767d\u3002", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "\u514d\u75ab\u68c0\u67e5\u70b9\u5206\u5b50CTLA4"}, {"entity_type": "EPIGENETIC MODIFICATION", "description": "\u6211\u4eec\u5bf9\u7279\u5b9a\u7684CpG\u5c9b\u8fdb\u884c\u4e86DNA\u7532\u57fa\u5316\u7684\u6d4b\u5b9a\uff0c\u4ee5\u6bd4\u8f83CD4+ T\u7ec6\u80de\u548cCD8+ T\u7ec6\u80de\u4e4b\u95f4\u7684\u5dee\u5f02\u3002", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "CPG\u5c9b\u7532\u57fa\u5316"}, {"entity_type": "CELLULAR PROCESS GENE", "description": "STX2\u4e0e\u56ca\u6ce1\u5728\u7ec6\u80de\u5185\u7684\u8fd0\u8f93\u76f8\u5173\u3002", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "\u7ec6\u80de\u5185\u8fd0\u8f93\u57fa\u56e0STX2"}, {"entity_type": "NON-CODING RNA", "description": "\u4f4d\u4e8eIKBKE\u57fa\u56e0\u9644\u8fd1\u7684\u957f\u7f16\u7801RNA\u8fdb\u884c\u4e86\u7814\u7a76\u3002", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "\u957f\u7f16\u7801RNA RP11\u2212534L20.5"}, {"entity_type": "GENE REGULATORY SNPS", "description": "\u6211\u4eec\u8bc6\u522b\u4e86\u5f71\u54cdCD4+ T\u7ec6\u80de\u548cCD8+ T\u7ec6\u80de\u4e2d\u57fa\u56e0\u8868\u8fbe\u7684cis-eQTL\u3002", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "CIS-EQTLS (\u8c03\u63a7\u57fa\u56e0\u8868\u8fbe\u7684\u7b49\u4f4d\u53d8\u5f02\u4f53)"}, {"entity_type": "TRANSCRIPTOME-WIDE ASSOCIATION STUDY METHODOLOGY", "description": "\u6211\u4eec\u8fd0\u7528\u4e86\u591a\u7ec4\u7ec7\u8054\u5408cis-eQTL\u5206\u6790\uff0c\u4ee5\u5229\u7528\u6240\u6709\u7ec4\u7ec7\u7684\u6570\u636e\u6765\u589e\u5f3a\u7edf\u8ba1\u6548\u529b\u548c\u5168\u9762\u6027\u3002", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "\u591a\u7ec4\u7ec7\u8054\u5408CIS-EQTL\u5206\u6790"}, {"entity_type": "STATISTICAL TECHNIQUE", "description": "\u6211\u4eec\u4f7f\u7528\u8d1d\u53f6\u65af\u6a21\u578b\u5e73\u5747\u5316\u4e0e\u5206\u5c42\u5efa\u6a21\u6280\u672f\u6765\u6574\u5408\u57fa\u56e0\u8868\u8fbe\u6570\u636e\u4e2d\u7684\u4fe1\u606f\u3002", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "\u8d1d\u53f6\u65af\u6a21\u578b\u5e73\u5747\u5316\u4e0e\u5206\u5c42\u5efa\u6a21"}, {"entity_type": "GENE", "description": "The CTLA4 gene was analyzed to determine its relationship with other genes and their expression levels.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "CGTAL4"}, {"entity_type": "GENOMIC VARIANT", "description": "A specific genomic variant of the CTLA4 gene was inversely correlated with gene expression.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "CG26091609"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The inverse correlation suggests that changes in expression levels might influence physiological processes.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "EXPRESSION LEVELS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "eQTLs were detected between the genomic variant and gene expression levels, indicating genetic regulation of expression.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "clusters": "[{level: 0, cluster: 7}, {level: 1, cluster: 26}]", "id": "EQTL EFFECTS"}, {"entity_type": "ANALYSIS TYPE", "description": "A multi-tissue approach was taken to identify shared eQTL effects across different cell types.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 48}]", "id": "MULTI-TISSUE ANALYSIS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The analysis revealed trans-acting regulatory elements influencing gene expression, particularly in the context of autoimmune diseases like T1D.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 48}]", "id": "TRANS-ACTING REGULATORY LOCUS"}, {"entity_type": "GENOMIC VARIANT", "description": "Genome-wide association study (GWAS) SNPs were selected for their potential impact on gene expression.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 47}]", "id": "GWAS SNPS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "SNPs affecting distant genes indicated long-range interactions within the genome.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "DISTANT GENES"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "A region across chromosome 12 showed significant impact on hundreds of gene expression levels, indicating a broad effect on multiple biological pathways.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "BROAD-RANGE IMPACT"}, {"entity_type": "GENE", "description": "The RPS26 gene plays a role in protein synthesis and its expression levels are influenced by the trans-acting regulatory locus.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 48}]", "id": "RPS26"}, {"entity_type": "GENE", "description": "The CDK2 gene encodes cyclin-dependent kinase 2, which is involved in cell cycle regulation.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "CDK2"}, {"entity_type": "GENE", "description": "The RAB5B gene encodes a member of the Ras superfamily of GTPases and participates in vesicle trafficking processes.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 48}]", "id": "RAB5B"}, {"entity_type": "GENE", "description": "The SUOX gene is related to urea cycle metabolism and its expression levels were identified through trans-eQTL analysis.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "SUOX"}, {"entity_type": "GENE", "description": "The IKZF4 gene is associated with immune responses and was found in the region affecting expression levels of multiple genes.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 48}]", "id": "IKZF4"}, {"entity_type": "GENE", "description": "The ERBB3 gene encodes for a receptor tyrosine kinase which interacts with growth factors and participates in signaling pathways.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "ERBB3"}, {"entity_type": "GENE", "description": "The IL27 gene codes for a cytokine that plays an important role in immune responses and inflammation, showing significant trans-eQTL effects on expression levels.<SEP>Transcription of the IL-27 cytokine is not affected by the trans-eQTL locus.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b<SEP>chunk-1b40d071abac0ba432a9fb0096796f45", "id": "IL27"}, {"entity_type": "GENE", "description": "The STAT1 gene codes for a transcription factor that plays a key role in the immune response and inflammation, particularly influenced by SNPs associated with T1D risk.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 49}]", "id": "STAT1"}, {"entity_type": "GENE", "description": "The REC8 gene is related to meiosis and was one of multiple genes identified as affected through trans-eQTL analysis.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 45}]", "id": "REC8"}, {"entity_type": "GENOMIC VARIANT", "description": "The missense SNP rs181206 on IRF1 and STAT1 has an effect that remains after regressing out the GWAS SNP, but not vice versa.<SEP>The missense SNP rs181206 on IRF1 interacts with STAT1 in trans-eQTL effects.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "RS181206"}, {"entity_type": "STATISTICAL METHOD", "description": "A Bayesian test supports the hypothesis that T1D susceptibility is linked to changes in expression of IRF1 and STAT1, sharing a single causal variant with rs181206G.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "BAYESIAN TEST FOR COLOCALISATION"}, {"entity_type": "CELL TYPE", "description": "The trans-eQTL locus had no effects on IL27 expression in cis analysis.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "MONOCYTES"}, {"entity_type": "MOLECULES", "description": "These molecules together act as a receptor for the IL-27 cytokine in CD4+ T cells.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "IL27RA AND IL6ST (GP130)"}, {"entity_type": "OBSERVATION", "description": "Higher expression levels of IRF1 and STAT1 genes were found in CD4+ compared to CD8+ T cells.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "CELL TYPE-SPECIFIC EFFECT"}, {"entity_type": "CYTOKINE", "description": "IL-27 is produced by innate immune cells, interacts with its receptor IL27RA and activates the STAT1/STAT3 pathway in T cells after forming a heterodimer with EBI3.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "IL-27"}, {"entity_type": "SEQUENCE", "description": "Interferon stimulated response elements (ISRE) are activated by the STAT1/STAT3 pathway which induces transcription of several interferon-induced genes, including IRF1 and STAT1 itself.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "ISRE"}, {"entity_type": "CELL LINE", "description": "Human embryonic kidney 293 cells were used for transfection with IL-27 wild-type or missense variants along with EBI3 protein to study their effect on IRF1 and STAT1 expression.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "HEK293 CELLS"}, {"entity_type": "SAMPLE TYPE", "description": "Peripheral blood mononuclear cells (PBMCs) from healthy individuals were used for real-time PCR studies of IRF1 and STAT1 transcript levels after IL-27 exposure.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "PBMCS"}, {"entity_type": "GENOMIC VARIANT", "description": "The missense SNP rs181206G in the IL27 gene was found to have a causal effect on decreased expression of IRF1 and STAT1.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "RS181206[G]"}, {"entity_type": "GENOMIC VARIANT", "description": "A GWAS SNP which does not have an effect when regressed out from trans-eQTL analysis of rs181206 on IRF1 and STAT1.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "RS3135402"}, {"entity_type": "PROTEIN", "description": "The protein that forms a heterodimer with IL-27 to interact with its receptor and activate the STAT1/STAT3 pathway in T cells after producing IL-27.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "EBI3"}, {"entity_type": "LOCATION", "description": "A genetic locus identified by trans-eQTL analysis which affects expression of IRF1, STAT1, and IL27 but not IL27RA or IL6ST genes.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "TRANS-EQTL LOCUS"}, {"source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "description": "The multi-tissue analysis was advantageous for identifying genes with lower effect sizes or incomplete power due to its capacity for accounting for such scenarios.)(entity", "entity_type": "UNKNOWN", "id": "METHODOLOGICAL ADVANTAGES"}, {"source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "description": "The multi-tissue analysis was advantageous for identifying genes with lower effect sizes or incomplete power due to its capacity for accounting for such scenarios.)(entity", "entity_type": "UNKNOWN", "id": "MULTI-TISSUE JOINT CIS-EQTL ANALYSIS STRENGTHS"}, {"source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "description": "CDK2 is one of the genes influenced through trans-expression quantitative trait loci analysis.)", "entity_type": "UNKNOWN", "id": "TRANS-EQTLS"}, {"source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "description": "The SUOX gene expression was analyzed through trans-eQTL studies indicating its association with T1D genetic risk factors.)", "entity_type": "UNKNOWN", "id": "TRANS-EQTL ANALYSIS"}, {"source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "description": "ERBB3 gene expression was associated with specific SNPs identified in the T1D susceptibility analysis.)", "entity_type": "UNKNOWN", "id": "SNPS ASSOCIATION"}, {"source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "description": "The STAT1 gene expression was influenced through both trans-eQTL effects and GWAS, indicating multiple genetic influences on its activity.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 49}]", "id": "TRANS-EQTLS AND GWAS SNPS"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "The rs181206[G] allele results in lower transcript levels for IRF1 compared to the wild-type Leu119 variant.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 47}]", "id": "RS181206[G]"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "The rs181206[G] allele in the IL27 gene is associated with T1D (Type 1 Diabetes) susceptibility.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 14}, {level: 1, cluster: 47}]", "id": "T1D SUSCEPTIBILITY"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "T1D development involves changes in expression of IRF1 and STAT1, which might be linked through a common causal variant with rs181206G.)", "entity_type": "UNKNOWN", "id": "CELLULAR MECHANISM"}, {"entity_type": "PROTEIN", "description": "IL-8 protein is a cytokine that plays a role in the immune response and inflammation.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "IL-8 PROTEIN"}, {"entity_type": "TRANSCRIPT", "description": "The IL-8 mRNA transcript is involved in the expression of the IL-8 protein.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "IL-8 MRNA"}, {"entity_type": "CELL LINE", "description": "The PBMC composition refers to the types of cells found in peripheral blood samples.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "PBMCS (PERIPHERAL BLOOD MONONUCLEAR CELLS)"}, {"entity_type": "TYPE OF DISEASE", "description": "Moderate asthma is a less severe form of asthma than severe asthma.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "MODERATE ASTHMA"}, {"entity_type": "TYPE OF DISEASE", "description": "Severe asthma requires more intensive management due to its severity.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "SEVERE ASTHMA"}, {"entity_type": "TYPE OF DISEASE", "description": "Mild asthma usually responds well to standard treatments.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "MILD ASTHMA"}, {"entity_type": "GENETIC VARIANT", "description": "The rs4073 genetic variant is a polymorphism in the IL-8 gene associated with asthma susceptibility.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "RS4073"}, {"entity_type": "GENETIC VARIANT", "description": "The rs2227306 genetic variant is another polymorphism in the IL-8 gene related to asthma.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "RS2227306"}, {"entity_type": "COMBINED GENETIC VARIANT", "description": "A haplotype is a combination of two or more nearby variants that are inherited together.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "HAPLOTYPE"}, {"entity_type": "STATISTICAL MEASURE", "description": "The LD test measures the statistical dependence between genetic variants.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "LD (LINKAGE DISEQUILIBRIUM)"}, {"entity_type": "NUTRIENT", "description": "Serine is an amino acid that plays a role in the metabolism of IL-8 proteins.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "IL-8 SERINE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The inflammatory response mediated by IL-8 and other cytokines helps defend against pathogens.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "INFLAMMATORY RESPONSE"}, {"entity_type": "ORGANISM COMPONENT", "description": "The immune system includes various cells, molecules, tissues, and organs working together to protect the body from infection.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "IMMUNE SYSTEM"}, {"entity_type": "COMBINED GENETIC VARIANT", "description": "A genetic haplotype is a group of genes or variants that are inherited together on the same chromosome.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "GENETIC HAPLOTYPE"}, {"entity_type": "GENE", "description": "The ASAH1 gene is on 8p22-p21.3 and consists of 86 HapMap SNPs and 14 typed SNPs on the Illumina HumanHap550 array.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "ASAH1 GENE"}, {"entity_type": "TRANSCRIPT", "description": "A SNP-set was formed by including only these 14 typed SNPs for the ASAH1 gene.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "SNP-SET"}, {"entity_type": "GENOMIC VARIANT", "description": "There are 86 HapMap SNPs associated with the ASAH1 gene.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "HAPMAP SNPS"}, {"entity_type": "PROTEIN", "description": "The Illumina HumanHap550 array contains 14 typed SNPs for the ASAH1 gene.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "ILLUMINA HUMANHAP550 ARRAY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The ASAH1 gene region on chromosome 8 is characterized by a good correlation structure among its SNPs and typed SNPs.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "8P22-P21.3 REGION"}, {"entity_type": "DRUG", "description": "The survival kernel machine SNP-set test evaluates genomic data for GWAS, where \'kernel\' is used as the underlying statistical model.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "SURVIVAL KERNEL MACHINE SNP-SET TEST"}, {"entity_type": "THERAPY", "description": "The min test serves as a benchmark in this context; it also gave correct size indicating its effectiveness alongside other methods like survival kernel machine test.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "MIN TEST"}, {"entity_type": "MOLECULAR INTERACTION", "description": "When the median R2 of the causal SNP with typed SNPs is moderate, the kernel machine method has higher power than the min test.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "KERNEL MACHINE METHOD"}, {"entity_type": "METABOLIC PATHWAY", "description": "A scenario where survival time is simulated assuming a linear model benefits both methods equally or shows slight preference to certain methods like IBS and weighted IBS kernel.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "LINEAR KERNEL"}, {"entity_type": "BIOLOGICAL PATHWAY", "description": "The IBS (Identity By State) kernel offers an alternative method for evaluating genomic data alongside the survival kernel machine test, showing comparable power.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "IBS KERNEL"}, {"entity_type": "DIETARY PATTERN", "description": "Using MAF as weights in weighted IBS kernel can upweight rare alleles in GWAS settings, leading to varied results depending on causal SNP characteristics.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "WEIGHTED IBS KERNEL"}, {"entity_type": "GENOMIC VARIANT", "description": "One of the typed SNPs associated with the NAT2 gene, located on chromosome 8p22.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "RS1390358"}, {"entity_type": "GENOMIC VARIANT", "description": "Another typed SNP associated with the NAT2 gene, located close to the transcription start of the gene.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "RS1112005"}, {"entity_type": "GENOMIC VARIANT", "description": "A typed SNP found near the transcription start region of the NAT2 gene on chromosome 8p22.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "RS7832071"}, {"entity_type": "GENOMIC VARIANT", "description": "One of the four typed SNPs in the NAT2 gene, located towards the end of the gene.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "RS1208"}, {"entity_type": "TRANSCRIPT", "description": "A specific SNP among the four typed SNPs (rs1390358, rs1112005, rs7832071, and rs1208) in NAT2.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "SNP 7"}, {"entity_type": "TRANSCRIPT", "description": "A specific SNP among the four typed SNPs that is closely related to SNPs 2, 5, 9, and 20 in terms of linkage disequilibrium (LD).", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "SNP 11"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "An SNP with a high LD score to SNP 11 but not well-correlated with other typed SNPs.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "SNP 2"}, {"entity_type": "METABOLIC PATHWAY", "description": "An SNP showing high correlation (R2 = 0.91) with SNP 11 and potentially other typed SNPs.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "SNP 5"}, {"entity_type": "BIOLOGICAL PATHWAY", "description": "An SNP demonstrating perfect LD with SNP 11 (R2 = 0.99), suggesting strong genetic linkage to the causal SNP when tested alone or in high LD with it.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "SNP 9"}, {"entity_type": "GENETIC MUTATION", "description": "A specific typed SNP in NAT2 that is closely correlated with SNP 11 (R2 = 0.99) and may exhibit similar results to this SNP due to their high linkage disequilibrium (LD).", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "SNP 20"}, {"entity_type": "PROTEIN", "description": "Part of the human leukocyte antigen (HLA) complex on chromosome 6p21.3.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "HLA-B GENE"}, {"entity_type": "GENOMIC VARIANT", "description": "One of the two typed SNPs associated with the HLA-B gene, with a lower minor allele frequency (MAF) than rs2523608.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "RS1058026"}, {"entity_type": "GENOMIC VARIANT", "description": "Another typed SNP in the HLA-B gene with higher minor allele frequency compared to rs1058026.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "RS2523608"}, {"entity_type": "GENE", "description": "The power of testing for the ASAH1 gene effect using different kernels was studied.", "source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "id": "ASAH1"}, {"source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "description": "Median R2 represents a commonly used measure of linkage disequilibrium (LD), which describes genetic correlation among SNPs.)", "entity_type": "UNKNOWN", "id": "LD MEASURE"}, {"source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "description": "Median R2 represents a commonly used measure of linkage disequilibrium (LD), which describes genetic correlation among SNPs.)", "entity_type": "UNKNOWN", "id": "MEDAN R2"}, {"source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "description": "In GWAS contexts, the weighted IBS kernel emphasizes rare alleles by incorporating their importance through MAF (minor allele frequency), impacting test outcomes when rare variants are causal.)", "entity_type": "UNKNOWN", "id": "RARE ALLELES"}, {"source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "description": "In the context of GWAS, causal SNPs may influence genetic outcomes depending on their LD score and MAF when evaluated using different statistical methods such as survival kernel machine test or weighted IBS kernel.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}, {level: 1, cluster: 39}]", "id": "CAUSAL SNPS"}, {"source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "description": "Both min test and linear kernel suffered from substantial loss of power as they mis-specified the model compared to the IBS and weighted IBS kernels.)", "entity_type": "UNKNOWN", "id": "LINEAR KERNEL& STANDARD IBS KERNEL"}, {"source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "description": "The kernel machine SNP-set test had better performance than the min test, especially when there was good LD structure similar to that of the ASAH1 gene.)<SEP>The kernel machine SNP-set test outperformed the min test for studying gene effects based on imputed data, especially when there was good linkage disequilibrium (LD) structure like in ASAH1.)", "entity_type": "UNKNOWN", "id": "KERNEL MACHINE SNP-SET TEST"}, {"source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "description": "The linear kernel and weighted IBS kernel had lower power compared to the standard IBS kernel, especially when causal SNPs have higher MAFs.)", "entity_type": "UNKNOWN", "id": "IBS KERNEL& WEIGHTED IBS KERNEL"}, {"source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "description": "The min test suffered from substantial loss of power compared to the other kernels due to mis-specifying the model.)", "entity_type": "UNKNOWN", "id": "LINEAR KERNEL& IBS KERNEL& WEIGHTED IBS KERNEL"}, {"entity_type": "GENOMIC VARIANT", "description": "A total of 22 SNPs that passed QC (quality control) were tested for their association with response in 213 anti-CTLA-4 treated patients.<SEP>Anti-CTLA-4 is an intervention used in clinical trials, not directly classified into environmental factors but related to treatment conditions.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1<SEP>chunk-0d49552280a277a8a26456d06f8f26a8", "id": "ANTI-CTLA-4"}, {"entity_type": "GENE", "description": "The analysis included autoimmune risk SNPs associated with celiac disease, inflammatory bowel disease, rheumatoid arthritis and type 1 diabetes.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "AUTOIMMUNE RISK SNPS"}, {"entity_type": "GENOMIC VARIANT", "description": "Anti-PD-1 is another intervention technique, associated with lifestyle and medical treatment rather than daily lifestyle choices.<SEP>For anti-PD-1 treatment, 24 SNPs passing QC were tested in 169 patients", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1<SEP>chunk-0d49552280a277a8a26456d06f8f26a8", "id": "ANTI-PD-1"}, {"entity_type": "GENE", "description": "Under the dominant regression model, consistent with the additive model, rs17388568 was found to have the strongest association with response.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "RS17388568"}, {"entity_type": "MOLECULAR INTERACTION", "description": "rs17388568 was mapped to the genomic region containing IL2, ADAD1 and IL21, which is associated with allergy, colitis and type 1 diabetes.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "IL2, ADAD1 AND IL21"}, {"entity_type": "DISEASE", "description": "The response rate for combination therapy was the highest.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "ANTI-CTLA-4 TREATED PATIENTS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "A larger proportion of our patients were males, consistent with the sex distribution of metastatic melanoma.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "DISEASE DISTRIBUTION OF METASTATIC MELANOMA"}, {"entity_type": "DIETARY PATTERN", "description": "A combination of anti-CTLA-4 and/or anti-PD-1 treatments were applied to patients.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "COMBINATION THERAPIES"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "There was a statistically significant difference in the age distribution between treatment cohorts.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "TREATMENT COHORTS"}, {"entity_type": "OUTCOME", "description": "A subset of patients showed positive response to their respective treatments.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "RESPONDERS"}, {"entity_type": "OUTCOME", "description": "A subset of patients did not show a positive response to their respective treatments.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "NON-RESPONDERS"}, {"entity_type": "SUBGROUP ANALYSIS", "description": "A separate analysis was conducted for a younger age group, which revealed stronger effects of certain SNPs.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "AGE GROUP"}, {"entity_type": "GENE", "description": "No specific gene information provided.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "N RESPONDERS"}, {"entity_type": "GENOMIC VARIANT", "description": "No specific genomic variant information provided.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "CONTROLS=33"}, {"entity_type": "TRANSCRIPT", "description": "No specific transcript information provided.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "N NON-RESPONDERS"}, {"entity_type": "PROTEIN", "description": "No specific protein information provided.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "CASES=10"}, {"entity_type": "NUTRIENT", "description": "No explicit nutrient information associated with the 23 SNPs.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "23 SNPS"}, {"entity_type": "DIETARY PATTERN", "description": "The Bonferroni method is a statistical approach, not related to dietary patterns.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "BONFERRONI"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The Holm-Bonferroni correction is a procedure used in statistics for adjusting the significance threshold, related to data analysis and error handling.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "HOLM-BONFERRONI METHODS"}, {"entity_type": "METABOLIC PATHWAY", "description": "Small sample sizes may limit statistical power, not related to metabolic pathways directly but affects statistical reliability.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "SMALL SAMPLE SIZE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "No specific molecular interaction provided in the text.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "COMBINATION-THERAPY"}, {"entity_type": "DISEASE", "description": "Treatment response could be indicative of disease management or outcome but isn\'t explicitly defined as a specific disease type in the provided text.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "TREATMENT RESPONSE"}, {"entity_type": "DRUG", "description": "The type of treatment drug used can affect the response to a combination therapy or individual therapies.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "TREATMENT DRUGS"}, {"source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "description": "Several associations were observed between autoimmune risk SNPs and response to both anti-CTLA-4 and anti-PD-1 treatments.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION WITH RESPONSE"}, {"source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "description": "A small sample size limits the power of an analysis to detect true associations, indicating that findings might not be reliable due to lower statistical confidence.)", "entity_type": "UNKNOWN", "id": "POWER OF ANALYSIS"}, {"source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "description": "Combination therapy is being grouped with other treatments for association analysis; this does not establish a causal relationship but reflects the experimental design.)", "entity_type": "UNKNOWN", "id": "GROUPING OF TREATMENTS"}, {"source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "description": "Sex can influence differences in how individuals respond to treatments, reflecting a lifestyle factor.)", "entity_type": "UNKNOWN", "id": "DIFFERENCE IN RESPONSE"}, {"source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "description": "The specific treatment drug used has an impact on the patient\'s response to therapy.)", "entity_type": "UNKNOWN", "id": "IMPACT ON RESPONSE"}, {"entity_type": "GENE", "description": "EBNA2 binds to the MS risk loci and affects their expression.<SEP>Results indicate that EBNA2 binds chromatin at MS risk loci dependent on the presence of a specific risk allele.<SEP>The results suggest that EBNA2 binds chromatin at MS risk loci, influencing transcript expression.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-bb351b97d99fce93d17df522db92dda7", "id": "EBNA2"}, {"entity_type": "GENOMIC VARIANT", "description": "Of the six identified MS risk SNPs, EBNA2 was found to bind to these loci dependent on the MS risk allele.<SEP>The MS risk loci were associated with altered gene expression.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-bb351b97d99fce93d17df522db92dda7", "id": "MS RISK LOCI"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Results show that EBNA2 interacts with the risk allele to bind chromatin and influence gene expression.", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "ALLELE"}, {"entity_type": "METABOLIC PATHWAY", "description": "EBNA2-TAT inhibits EBV latency III growth program through dose-response experiments.", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "EBV LATENCY III GROWTH PROGRAM"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "LCL proliferation is associated with expression of MS risk genes, which can be influenced by EBNA2-TAT treatment.", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "LCL PROLIFERATION"}, {"entity_type": "DIETARY PATTERN", "description": "The effect of EBNA2 inhibition was tested in a dose-response experiment with varying concentrations.", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "EBNA2-TAT"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Risk alleles were identified for MS SNPs, influencing gene expression and EBV DNA copy number.", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "RISK ALLELE"}, {"entity_type": "PROTEIN", "description": "Expression of the EBV gene LMP1 was reduced significantly by EBNA2-TAT at 50 \u00b5M concentration.<SEP>Latency protein LMP1 interacts with various proteins and signaling pathways, potentially contributing to MS through its function on the latency III infection cycle of EBV.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-bb351b97d99fce93d17df522db92dda7", "id": "LMP1"}, {"entity_type": "MOLECULAR INTERACTION", "description": "EBNA2-TAT specifically binds to the RBP-J\u03ba site on EBNA2 through GST affinity assay, indicating specific interaction and effect.", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "RBP-J\u039a SITE"}, {"entity_type": "DISEASE", "description": "A mutant peptide of EBNA2-TAT had no inhibitory effects on EBNA2 function or cell viability, suggesting specificity to RBP-J\u03ba interaction for observed effects.", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "MUTANT PEPTIDE OF EBNA2-TAT"}, {"entity_type": "THERAPY", "description": "Inhibition of EBNA2 alters MS risk gene expression in LCLs through specific molecular interactions and pathways.", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "EBNA2 INHIBITION"}, {"entity_type": "GENE", "description": "EBNA2 was demonstrated to bind RCOR1, one of the MS risk loci, dependent on the presence of a specific risk allele.<SEP>RCOR1 is another gene that was identified in association with MS risk loci under the effect of EBNA2.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-bb351b97d99fce93d17df522db92dda7", "id": "RCOR1"}, {"entity_type": "GENE", "description": "The effect of EBNA2 inhibition suggested that EBNA2 inhibits expression of TNFSF1A in LCLs.", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "TNFSF1A"}, {"entity_type": "GENE", "description": "The results suggest that EBNA2 inhibits expression of TBX6 in LCLs, indicating a potential role in modulating MS risk genes.", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "TBX6"}, {"entity_type": "GENE", "description": "CLECL1, a gene related to MS risk loci, showed no significant effect from EBNA2 binding preference.<SEP>EBNA2 inhibition showed reduced risk gene expression on CLECL1 suggesting an effect from EBNA2.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-bb351b97d99fce93d17df522db92dda7", "id": "CLECL1"}, {"entity_type": "GENE", "description": "Genetic associations found that EBV DNA copy number is influenced by the TNFAIP8 locus, and these effects were altered upon EBNA2-TAT treatment.<SEP>TNFAIP8 is an immune system gene that was affected by EBNA2 in the context of MS risk loci.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-bb351b97d99fce93d17df522db92dda7", "id": "TNFAIP8"}, {"entity_type": "GENE", "description": "TNFRSF1A, or Tumor Necrosis Factor Receptor Superfamily Member 1A, plays a role in immune response and is one of the MS risk genes impacted by EBNA2 interaction with the risk loci.<SEP>The expression of TNFRSF1A was significantly affected by EBNA2 inhibition, suggesting a role in modulating MS risk genes through EBNA2 interaction.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-bb351b97d99fce93d17df522db92dda7", "id": "TNFRSF1A"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The allelic association between genotypes at risk loci was altered by EBNA2 binding.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "ALLELIC ASSOCIATION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "EBNA2 changed the expression levels of specific genes like TRAF3, CD40, TNFAIP8, TNFRSF1A, TBX6 and RCOR1 at MS risk loci.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "EXPRESSION OF GENES"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The direction of gene expression alteration depended on the MS risk allele preference of EBNA2 binding.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "RISK ALLELE PREFERENCE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "LMP2 is another latency protein that plays a role in EBV\'s latency III phase alongside LMP1, which can influence gene expression and MS risk factors.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "LMP2"}, {"entity_type": "METABOLIC PATHWAY", "description": "The cycle of EBV latency III involves the regulation of viral proteins including EBNA2 and LMP1/LMP2, influencing immune evasion and cell transformation processes.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "LATENCY III INFECTION CYCLE"}, {"entity_type": "BIOLOGICAL ENTITY", "description": "LCLs are established when a virus like EBV infects B-cells that have undergone permanent growth arrest due to Epstein-Barr viral infection.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "LCLS (B-CELL LINE ESTABLISHED BY EBV-LATENT INFECTION)"}, {"entity_type": "COMPARATIVE BIOLOGICAL STATE", "description": "Refers to normal, uninfected B cells as opposed to those infected with Epstein-Barr Virus (EBV), impacting expression and function.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "UNINFECTED B CELLS"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "EBNA2 was found to bind chromatin at MS risk loci dependent on the presence of a specific risk allele, influencing gene expression.)", "entity_type": "UNKNOWN", "id": "MS RISK GENES"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "EBV DNA copy number is associated with several MS risk SNPs and genes in LCLs, which can be influenced by EBNA2-TAT treatment.)", "entity_type": "UNKNOWN", "id": "EBV DNA COPY NUMBER"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "EBV DNA copy number is associated with several MS risk SNPs and genes in LCLs, which can be influenced by EBNA2-TAT treatment.)", "entity_type": "UNKNOWN", "id": "LCLS"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "EBNA2 interacts specifically with the RBP-J\u03ba site on EBNA2, affecting gene expression through this interaction.)", "entity_type": "UNKNOWN", "id": "RBP-J\u039a INTERACTION"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Expression of MS risk genes is associated with LCL proliferation, which can be modulated by EBNA2 inhibition.)", "entity_type": "UNKNOWN", "id": "MS RISK SNPS AND GENES"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "A dose-response experiment was conducted to determine the optimal concentration of EBNA2-TAT that inhibits EBNA2 transcriptional function without excess cytotoxicity.)", "entity_type": "UNKNOWN", "id": "DOSAGE RESPONSE"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "The data suggest that TBX6 and RCOR1 are modulated by EBNA2 inhibition, providing insights into how EBNA2 might influence genetic risk factors for MS.)", "entity_type": "UNKNOWN", "id": "GENETIC MODULATION"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "The data suggest that TBX6 and RCOR1 are modulated by EBNA2 inhibition, providing insights into how EBNA2 might influence genetic risk factors for MS.)", "entity_type": "UNKNOWN", "id": "TBX6 AND RCOR1"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "EBNA2 effect on gene expression at MS risk loci could be ablated by EBNA2-TAT treatment.)", "entity_type": "UNKNOWN", "id": "EBNA2 TREATMENT"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "EBNA2 effect on gene expression at MS risk loci could be ablated by EBNA2-TAT treatment.)", "entity_type": "UNKNOWN", "id": "GENOTYPE EFFECT"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "The effect on gene expression due to EBNA2 affects the program of EBV latency III infection cycle.)", "entity_type": "UNKNOWN", "id": "PROGRAM OF EBV LATENCY III"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "The effect on gene expression due to EBNA2 affects the program of EBV latency III infection cycle.)", "entity_type": "UNKNOWN", "id": "RISK GENE EXPRESSION"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "EBNA2 interaction with MS risk alleles may increase host immune evasion by enhancing its function.)", "entity_type": "UNKNOWN", "id": "EBNA2 BINDING"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "EBNA2 interaction with MS risk alleles may increase host immune evasion by enhancing its function.)", "entity_type": "UNKNOWN", "id": "HOST IMMUNE EVASION"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "The EBNA2 interaction on MS risk genes led to changes in their expression levels, affecting immune evasion processes.)", "entity_type": "UNKNOWN", "id": "EBNA2 INTERACTION EFFECT"}, {"entity_type": "GENE", "description": "The initial search retrieved 154 potentially relevant references<SEP>The initial search retrieved 154 potentially relevant references.", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "id": "RS3842787"}, {"entity_type": "GENE", "description": "A flow diagram schematizing the selection process of identified articles with specific reasons, and the baseline characteristics of all qualified articles are presented in Figure 1<SEP>rs20417 has an associated P-value of 0.034, indicating its significance in explaining part of the heterogeneity.", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea<SEP>chunk-54c4fede216a6c4183b95ee1a2edcb17", "id": "RS20417"}, {"entity_type": "GENE", "description": "There were a total of 4, 6, 10 and 7 qualified articles and 353/595, 344/698, 588/878", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "id": "RS20118426"}, {"entity_type": "GENE", "description": "The retrieved articles were published between 2003 and 2012, and 11 of them were written in Chinese<SEP>rs1126643 shows a significant source of heterogeneity with a P-value of 0.012, suggesting its importance in study outcomes.", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea<SEP>chunk-54c4fede216a6c4183b95ee1a2edcb17", "id": "RS1126643"}, {"entity_type": "GENOMIC VARIANT", "description": "Taking all available comparisons together for each polymorphism observed significant association of COX-2", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "id": "COX-2 GENE RS20417"}, {"entity_type": "GENOMIC VARIANT", "description": "Subgroup analyses<SEP>Whereas no significance was found for COX-1 gene rs3842787 and ITGA2B gene rs20118426", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "id": "ITGA2 GENE RS1126643"}, {"entity_type": "GENOMIC VARIANT", "description": "Overall analyses", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "id": "COX-1 GENE RS3842787"}, {"entity_type": "DIETARY PATTERN", "description": "To estimate the influence of categorical confounders, separate analyses were performed within strata involving two or more comparisons", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "id": "CHINESE POPULATION"}, {"entity_type": "DIETARY PATTERN", "description": "Further grouping articles by ethnicity of study populations (mainly Chinese and Caucasian)", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "id": "CAUCASIAN POPULATION"}, {"entity_type": "GENOMIC VARIANT", "description": "Subgroup analyses", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "id": "ITGA2B GENE RS20118426"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Averaged platelet number serves as an important factor contributing to the heterogeneity in rs1126643.", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea", "id": "AVERAGED PLATELET NUMBER"}, {"source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "description": "There was a significant association between rs20417 in the COX-2 gene and aspirin insensitivity)", "entity_type": "UNKNOWN", "id": "COX-2 GENE"}, {"source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "description": "Risk estimates by rs1126643-T allele were more potentiated among patients with aspirin semi-resistance compared with those with aspirin resistance)", "entity_type": "UNKNOWN", "id": "ITGA2 GENE"}, {"source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "description": "Differences in the percentage of CAD between patients and controls explained a large part of heterogeneity for rs20417)", "entity_type": "UNKNOWN", "id": "CAD PERCENTAGE"}, {"source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "description": "Averaged platelet number was a significant source of heterogeneity for rs1126643)", "entity_type": "UNKNOWN", "id": "PLATELET NUMBER"}, {"source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "description": "The association of COX-1 gene polymorphism rs3842787 with aspirin insensitivity was not significant)", "entity_type": "UNKNOWN", "id": "COX-1 GENE"}, {"source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "description": "The influence of ITGA2B gene polymorphism rs20118426 on the response to aspirin could be assessed under subgroup analyses)", "entity_type": "UNKNOWN", "id": "ITGA2B GENE"}, {"entity_type": "CLINICAL PRESENTATION", "description": "The GHI phenotype encompasses severe short stature, centripetal fat distribution, a prominent forehead, and a depressed nasal bridge.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "GHI PHENOTYPE"}, {"entity_type": "CLINICAL PRESENTATION", "description": "Severe short stature was present in all three siblings with height standard deviation scores (height SDS) of -6.5, -5.06, and -4.7 respectively.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "SHORT STATURE"}, {"entity_type": "GHI PHENOTYPE", "description": "This refers to the accumulation of fat near the center of the body as part of the GHI phenotype.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "CENTRIPETAL FAT DISTRIBUTION"}, {"entity_type": "GHI PHENOTYPE", "description": "Characterized by an increased prominence or size of the forehead, as seen in the GHI phenotype of three siblings.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "PROMINENT FOREHEAD"}, {"entity_type": "GHI PHENOTYPE", "description": "The nasal bridge being lower than normal is one feature of the GHI phenotype observed among these patients.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "DEPRESSED NASAL BRIDGE"}, {"entity_type": "PRESENTATION", "description": "This includes eczema and reactive airway disease which require corticosteroids for management.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "ATOPIC DISEASE"}, {"entity_type": "ATOPIC DISEASE", "description": "A skin disorder characterized by itchy, red, inflamed patches on the skin, often seen in atopy cases.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "ECZEMA"}, {"entity_type": "ATOPIC DISEASE", "description": "An inflammatory condition affecting the airways that can lead to asthma-like symptoms and is part of atopy.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "REACTIVE AIRWAY DISEASE"}, {"entity_type": "AUTOIMMUNE THYROIDITIS", "description": "Levothyroxine is used to replace missing or underproducing thyroid hormones in autoimmune thyroiditis.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "THYROID HORMONE REPLACEMENT THERAPY"}, {"entity_type": "PRESENTATION", "description": "Developed as a part of their condition requiring corticosteroids for management.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "CHRONIC ECZEMA"}, {"entity_type": "PRESENTATION", "description": "Involving skin, urinary tract, and lungs necessitating antibiotics and supplemental oxygen.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "RECURRENT INFECTIONS"}, {"entity_type": "PRESENTATION", "description": "Noted during clinical evaluations of all three siblings.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "GENERAL LYMPHADENOPATHY"}, {"entity_type": "PRESENTATION", "description": "Observed in patient 3 requiring medical attention for symptoms.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "GASTROESOPHAGEAL REFLUX DISEASE (GERD)"}, {"entity_type": "PRESENTATION", "description": "The levels were within normal range in each sibling at presentation, consistent with GHI.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "NORMAL SERUM GH LEVELS"}, {"entity_type": "GHI PRESENTATION", "description": "These low values are indicative of Growth Hormone Insensitivity Syndrome (GHI), alongside clinical symptoms.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "BELOW-NORMAL IGF-1, IGF-BP3, AND ACID LABILE SUBUNIT (ALS) CONCENTRATIONS"}, {"entity_type": "CLINICAL PRESENTATION", "description": "Characterized by late onset of secondary sexual characteristics in adolescents.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "DELAYED PUBERTY"}, {"entity_type": "CONSEQUENCE OF GROWTH HORMONE INSENSITIVITY SYNDROME (GHI)", "description": "A decrease in bone density, increasing the risk for osteoporosis.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "BONE DENSITY REDUCTION"}, {"entity_type": "CLINICAL PRESENTATION", "description": "Common symptom in patients with GHI due to lack of growth hormone which affects muscle mass and strength.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "MUSCLE WEAKNESS"}, {"entity_type": "CONSEQUENCE OF GHI", "description": "Some studies suggest that children with GHI may have learning difficulties and cognitive impairments.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "LEARNING DIFFICULTIES"}, {"entity_type": "CONSEQUENCES IN PATIENTS", "description": "GHI patients might exhibit behavioral issues like irritability or hyperactivity, which are possibly related to neurotransmitter imbalances.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "BEHAVIORAL PROBLEMS"}, {"entity_type": "COMMON IN GHI POPULATION", "description": "Difficulty sleeping is another common complaint among individuals with growth hormone insensitivity.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "SLEEP DISTURBANCES"}, {"entity_type": "PRESENT IN GHI CASES", "description": "This includes issues such as constipation or diarrhea due to hormonal imbalances and metabolic effects.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "GASTROINTESTINAL PROBLEMS"}, {"entity_type": "IN MALES WITH GHI", "description": "Men may experience reduced fertility or erectile dysfunction as a result of inadequate growth hormone levels.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "REPRODUCTIVE ISSUES"}, {"entity_type": "GHI CONSEQUENCE", "description": "The condition can affect the psychological well-being, leading to anxiety or depression due to physical and social changes.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "PSYCHOLOGICAL IMPACT"}, {"entity_type": "GENOMIC VARIANT", "description": "The STAT5B gene harbors a loss-of-function variant, leading to reduced expression and inability for phosphorylation or nuclear trafficking upon growth hormone (GH) stimulation.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "STAT5B P.W631* VARIANT"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Sustained B cell overactivity is associated with atopy and autoimmunity in STAT5B deficient siblings.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "SUSTAINED B CELL HYPERACTIVITY"}, {"entity_type": "GENOMIC VARIANT", "description": "A loss-of-function (LOF) variant of the STAT5B gene was identified, which impacts immune profiles in patients.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "LOF STAT5B VARIANT"}, {"entity_type": "TRANSCRIPT", "description": "The number of natural killer (NK) cells remains normal during infancy through early childhood in STAT5B deficient siblings.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "NK CELLS"}, {"entity_type": "DISEASE", "description": "High levels of IgE are associated with an atopic phenotype often linked to STAT5B deficiency.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "IGE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Moderate to severe T cell lymphopenia has been observed in STAT5B deficient patients at diagnosis, indicating a deficiency or destruction of peripheral T cells.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "T CELL LYMPHOPENIA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The assessment of recent thymic emigrants (RTEs) by quantifying CD4+CD45+CD31+ T cells reveals that while P3 displays reduced RTEs which normalize over time, this normalization varies among siblings.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "THYMIC OUTPUT"}, {"entity_type": "TRANSCRIPT", "description": "The numbers and percentages of regulatory T (Treg) cells identified as CD4+CD25highCD127low or CD25+FOXP3+ are within normal ranges in P2 and P3, with reductions identified only later for P1.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "ABSOLUTE NUMBER AND TOTAL PERCENTAGE OF TREGS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Young STAT5B deficient siblings exhibit pronounced changes in memory CD8+ T cells, characterized by reduced na\u00efve cells and elevations in memory/activation phenotype, impacting various subtypes including central memory (TCM), effector memory (TEM), and effector memory re-expressing CD45RA (TEMRA) over time.)(entity", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "MEMORY CD8+ T CELL DYSREGULATION"}, {"entity_type": "DISEASE", "description": "STAT5B contributes to maintaining self-tolerance, which is disrupted with loss-of-function variants leading to autoimmune responses.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "SELF-TOLERANCE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Reductions in IL-2 signaling due to STAT5B variant results in decreased FOXP3 induction and impaired Treg function.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "IL-2 SIGNALING REDUCTION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "STAT5B affects memory T cell subsets, impacting their development and function during infancy through childhood.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "MEMORY T CELL SUBSETS"}, {"entity_type": "TRANSCRIPT", "description": "The TCM (T-cell memory) cells remained within normal ranges.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "TCM CELLS"}, {"entity_type": "CELL TYPE", "description": "No alterations in the proportion of CD4+ TEM (terminally differentiated effector memory) cells were identified.<SEP>No changes were identified in the proportion of CD4+ TEM (terminally differentiated effector memory) cells.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "CD4+ TEM CELLS"}, {"entity_type": "CELL TYPE", "description": "Increased proportions of CD8+ TEM (terminally differentiated effector memory) cells became clinically significant with age, specifically noticeable in each sibling.<SEP>Increases in proportions of CD8+ TEM (terminally differentiated effector memory) cells became clinically significant with age, specifically in each sibling.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "CD8+ TEM CELL PROPORTIONS"}, {"entity_type": "CELL TYPE", "description": "An increase in the percentage of CD8+ TEMRA (recently activated T helper) cells was detected during infancy and then normalized around age 4.<SEP>Elevations in the percentage of CD8+ TEMRA (recently activated T helper) cells were detected during infancy and normalized around age 4.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "INFANT CD8+ TEMRA CELLS"}, {"entity_type": "TIME POINT", "description": "CD8+ memory T cells began to be elevated, suggesting a critical role for the LOF (loss-of-function) STAT5B variant in CD8+ T cell activation and homeostasis.<SEP>CD8+ memory T cells showed elevated proportions starting from infancy, indicating a crucial role of the LOF STAT5B variant in CD8+ T cell activation and homeostasis.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "AGE 4 CD8+ MEMORY T CELLS"}, {"entity_type": "GENETIC CONDITION", "description": "Our patients carried a variant of STAT5B, which despite normal CD4+ memory T cell subsets, was associated with dysregulation of CD8+ T cells.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "STAT5B DEFICIENCY"}, {"entity_type": "CELL TYPE", "description": "Elevated proportions of CD8+ memory T cells began in infancy, suggesting a role for the LOF STAT5B variant in CD8+ T cell activation and homeostasis.)(entity", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "CD8+ MEMORY T CELLS"}, {"entity_type": "GENE", "description": "Our patients carried a variant of the STAT5B gene, which despite normal levels of CD4+ memory T cells, was associated with altered regulation of CD8+ T cells.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "STAT5B"}, {"entity_type": "GENETIC VARIANT", "description": "The presence of a LOF STAT5B variant in our patients led to dysregulation of CD8+ T cells despite normal CD4+ memory T cell subsets.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "LOF (LOSS-OF-FUNCTION) STAT5B VARIANT"}, {"entity_type": "CELL TYPE", "description": "The proportions of CD4+ memory T cells remained normal across the study, despite the presence of the LOF STAT5B variant and dysregulation in other parts of the immune system.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "CD4+ MEMORY T CELLS"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "Whole Exome Sequencing (WES) was performed on patient 1 at age 3.9, revealing a novel homozygous nonsense STAT5B mutation.)", "entity_type": "UNKNOWN", "id": "IDENTIFICATION OF NOVEL MUTATION"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "Whole Exome Sequencing (WES) was performed on patient 1 at age 3.9, revealing a novel homozygous nonsense STAT5B mutation.)", "entity_type": "UNKNOWN", "id": "WES ANALYSIS"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "The WES analysis showed heterozygosity in the STAT5B locus in both parents, implying they are carriers of the variant.)", "entity_type": "UNKNOWN", "id": "PARENTAL HETEROZYGOSITY"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "The WES analysis showed heterozygosity in the STAT5B locus in both parents, implying they are carriers of the variant.)", "entity_type": "UNKNOWN", "id": "TRIO CLINICAL WHOLE EXOME SEQUENCING"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "Sanger sequencing confirmed the presence of the mutation in the fibroblast cultures from each sibling.)", "entity_type": "UNKNOWN", "id": "CONFIRMATION OF MUTATION"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "The new mutation is adjacent to the first identified mutation in STAT5B, p.A630P.)", "entity_type": "UNKNOWN", "id": "COMPARISON WITH IDENTIFIED MUTATIONS"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "The new mutation is adjacent to the first identified mutation in STAT5B, p.A630P.)", "entity_type": "UNKNOWN", "id": "NOVEL STAT5B MUTATION LOCATION"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "The truncated STAT5B protein lacks ~50% of the Src Homology 2 (SH2) domain and the critical phosphorylated tyrosine at position 699.)", "entity_type": "UNKNOWN", "id": "ROLE OF SH2 DOMAIN"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "The truncated STAT5B protein lacks ~50% of the Src Homology 2 (SH2) domain and the critical phosphorylated tyrosine at position 699.)", "entity_type": "UNKNOWN", "id": "SH2 DOMAIN IMPAIRMENT"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "The mutation results in the absence of the entire transactivation domain (TAD).)", "entity_type": "UNKNOWN", "id": "ROLE OF TAD DOMAIN"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "The mutation results in the absence of the entire transactivation domain (TAD).)", "entity_type": "UNKNOWN", "id": "TAD DOMAIN ABSENCE"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "In vitro reconstitution analyses showed significantly reduced expression of the truncated STAT5B protein.)", "entity_type": "UNKNOWN", "id": "EXPRESSION ANALYSIS"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "In vitro reconstitution analyses showed significantly reduced expression of the truncated STAT5B protein.)", "entity_type": "UNKNOWN", "id": "REDUCED EXPRESSION OF P.W631* PROTEIN"}, {"entity_type": "GENOMIC VARIANT", "description": "The genotyping results of rs28383172 was successfully done in 98% of the patients. Two individuals were excluded due to poor DNA quality.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS28383172"}, {"entity_type": "GENOMIC VARIANT", "description": "Three SNPs, including rs4713586, had genotypes available for all 233 patients.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS4713586"}, {"entity_type": "TECHNIQUE", "description": "A Custom TaqMan SNP Genotyping Assay was used for the genotyping of rs28383172.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "SNP GENOTYPING ASSAY"}, {"entity_type": "TECHNIQUE", "description": "The KASPar-on-Demand prevalidated assays were utilized to obtain genotypes for all 3 SNPs (rs2395182, rs4713586 and rs7775228).", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "KASPAR-ON-DEMAND PREVALIDATED ASSAYS"}, {"entity_type": "GENE", "description": "The study involved HLA-DRB1 gene analysis for the genotype carrier status of DRB1*07:01.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "HLA DRB1*07:01"}, {"entity_type": "GENE", "description": "Haplotype carrier status includes DQA1*02:01 allele.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "DQA1*02:01"}, {"entity_type": "GENE", "description": "Haplotype carrier status also involves the presence of DQB1*02:02 allele.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "DQB1*02:02"}, {"entity_type": "GROUP", "description": "A group was identified based on genotyping results using SNPs rs28383172, rs2395182, rs4713586 and rs7775228.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "SNP-GENOTYPED SUBPOPULATION"}, {"entity_type": "POPULATION SUPER TYPE", "description": "The Hungarian population showed most similarity to the EUR super population in terms of linkage disequilibrium coefficients between investigated SNPs.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "HUNGARIAN POPULATION"}, {"entity_type": "POPULATION SUPER TYPE", "description": "Algorithm V performed with sensitivity and >95% specificity in the AMR (Admixed American) population.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "AMR"}, {"entity_type": "POPULATION SUPER TYPE", "description": "Algorithm V also achieved 100% sensitivity and over 95% specificity in the East Asian (EAS) population.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "EAS"}, {"entity_type": "POPULATION SUPER TYPE", "description": "In the EUR super population, algorithm V had significantly higher values for sensitivity and negative predictive value compared to algorithms III and IV. It also showed a significant increase in specificity and positive predictive value when compared to algorithm II.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "EUR"}, {"entity_type": "GENOMIC VARIANT", "description": "The genotypes for both SNPs, including rs6031, are available among all study subjects (n=250).", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS6031"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs9738479 was successfully genotyped in the cohort of patients.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS9738479"}, {"entity_type": "GENOMIC VARIANT", "description": "The genotyping process for rs1045641 yielded results for all participants involved (n=273).", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS1045641"}, {"entity_type": "TECHNOLOGY", "description": "High-throughput automated SNP genotyping platform was employed to determine the rs5742931 status.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "SNP GENOTYPING PLATFORM"}, {"entity_type": "PAIRWISE RELATIONSHIP", "description": "The linkage disequilibrium (LD) analysis revealed that rs6031 was in strong LD with rs9738479, but not rs5742931.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS6031 AND RS9738479"}, {"entity_type": "PAIRWISE RELATIONSHIP", "description": "The SNP rs1045641 was found to be in moderate linkage disequilibrium with rs9738479, but not with rs5742931 or rs6031.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS1045641 AND RS9738479"}, {"entity_type": "PAIRWISE RELATIONSHIP", "description": "The genotypic frequencies for rs5742931 showed a significantly different distribution compared to rs6031, indicating a low degree of linkage disequilibrium.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS5742931 AND RS6031"}, {"entity_type": "PAIRWISE RELATIONSHIP", "description": "A significant statistical difference was found in the genotypic frequencies between rs1045641 and rs6031, indicating minimal LD.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS1045641 AND RS6031"}, {"entity_type": "PAIRWISE RELATIONSHIP", "description": "The haplotype analysis revealed that rs9738479 was in weak linkage disequilibrium with rs6031, but not with rs5742931 or rs1045641.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS9738479 AND RS6031"}, {"entity_type": "PAIRWISE RELATIONSHIP", "description": "The association analysis did not find a significant LD between rs1045641 and rs5742931, suggesting they are independently distributed.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS1045641 AND RS5742931"}, {"entity_type": "PAIRWISE RELATIONSHIP", "description": "The statistical analysis showed no evidence of linkage disequilibrium between rs9738479 and rs5742931, indicating they might be in different recombination hotspots.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS9738479 AND RS5742931"}, {"entity_type": "PAIRWISE RELATIONSHIP", "description": "Genetic association studies did not detect any strong linkage disequilibrium between rs6031 and rs1045641, but showed a slight correlation.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS6031 AND RS1045641"}, {"entity_type": "DATASET", "description": "A large-scale genetic database used for evaluating algorithms and comparing parameters.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "1000 GENOMES DATASET"}, {"entity_type": "SUPER POPULATION", "description": "A subset of the 1000 Genomes dataset that specifically represents European populations.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "EUR SUPER POPULATION"}, {"entity_type": "ALGORITHM", "description": "One of several machine learning models or statistical methods tested for their performance.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM III"}, {"entity_type": "ALGORITHM", "description": "Another model used in comparison to algorithm III and V in the analysis.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM IV"}, {"entity_type": "ALGORITHM", "description": "A second testing model used alongside algorithms III, IV, and V.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM II"}, {"entity_type": "ALGORITHM", "description": "The best performing model among those tested on the 1000 Genomes dataset.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM V"}, {"entity_type": "SUBPOPULATION", "description": "A subset of individuals within the broader dataset who have their SNPs (single nucleotide polymorphisms) genotyped.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "SNP GENOTYPED SUBPOPULATION"}, {"entity_type": "HAPLOTYPE", "description": "A combination of several DNA variations found together on a single chromosome, in this case related to the HLA system.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "HLA-DRB1*07:01\u2013DQA1*02:01\u2013DQB1*02:02 HAPLOTYPE"}, {"entity_type": "MARKER", "description": "Alternative genetic indicators that can be used as proxies for certain traits or conditions when direct testing is unavailable or impractical.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "SURROGATE MARKERS"}, {"entity_type": "DISEASE", "description": "A sensitivity to the antibiotic E. coli ASP, typically an allergy or adverse reaction.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "E. COLI ASP HYPERSENSITIVITY"}, {"entity_type": "CARRIER STATUS", "description": "The presence of a specific combination of genetic variants in an individual.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "HAPLOTYPE CARRIER STATUS"}, {"entity_type": "TECHNIQUE", "description": "A method of genetic tagging that uses two single nucleotide polymorphisms (SNPs) for correlation.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "2-SNP TAGGING"}, {"entity_type": "DRUG OR COMPOUND", "description": "An antibiotic, specifically Erythromycin, often causing hypersensitivity reactions.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "E. COLI ASP"}, {"entity_type": "GENETIC INFORMATION", "description": "The state where a person carries the specific combination of alleles for HLA genes mentioned.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "HAPLOTYPE CARRIER STATUS OF HLA-DRB1*07:01\u2013DQA1*02:01\u2013DQB1*02:02"}, {"entity_type": "TESTING MODEL", "description": "Other computational models tested against algorithm V in performance comparisons.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM II, III, IV"}, {"entity_type": "DATA COLLECTION", "description": "A comprehensive collection of human genetic data used for population genetic studies.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "DATASET FROM 1000 GENOMES PROJECT"}, {"entity_type": "STATISTICAL MEASURE", "description": "Quantitative indicators of the accuracy of a test or algorithm, with sensitivity measuring true positives and specificity true negatives.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "SENSITIVITY AND SPECIFICITY"}, {"entity_type": "STATISTICAL MEASURE", "description": "The probability that an individual who tests negative truly does not have the condition being tested for.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "NEGATIVE PREDICTIVE VALUE (NPV)"}, {"entity_type": "STATISTICAL MEASURE", "description": "The probability that an individual who tests positive truly has the condition being tested for.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "POSITIVE PREDICTIVE VALUE (PPV)"}, {"entity_type": "SUBSTITUTE INDICATOR", "description": "A genetic variant or combination of variants used as a proxy when direct markers are unavailable or challenging to obtain.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "SURROGATE MARKER"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "Algorithm V had a better positive predictive value than algorithms II, III and IV when tested on the 1000 Genomes dataset.)", "entity_type": "UNKNOWN", "id": "POSITIVE PREDICTIVE VALUE"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "The performance of the HLA-DRB1*07:01-DQA1*02:01-DQB1*02:02 haplotype as a screening test was evaluated against E. coli ASP hypersensitivity.)", "entity_type": "UNKNOWN", "id": "SCREENING TEST PERFORMANCE"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "The sensitivity of the HLA haplotype to correctly identify affected patients was 23.4%.)", "entity_type": "UNKNOWN", "id": "SENSITIVITY VALUE"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "The specificity of the HLA haplotype was high at 88.5%, indicating a low rate of false positives.)", "entity_type": "UNKNOWN", "id": "SPECIFICITY"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "The negative predictive value of the HLA haplotype was 63.1%, indicating that among those not identified as having the haplotype, a high percentage did not have E. coli ASP hypersensitivity.)", "entity_type": "UNKNOWN", "id": "NEGATIVE PREDICTIVE VALUE"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "The HLA haplotype and its best tagger algorithm (V) had the same performance parameters.)", "entity_type": "UNKNOWN", "id": "IDENTICAL PERFORMANCE MEASURES"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "Algorithm V did not perfectly predict the haplotype carrier status, as evidenced by a sensitivity and specificity of less than 100% when tagging the haplotype.)", "entity_type": "UNKNOWN", "id": "TAGGER ACCURACY"}, {"entity_type": "GENE", "description": "The TLR7 gene is involved in the response to pathogens and inflammation.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "id": "TLR7"}, {"entity_type": "GENOMIC VARIANT", "description": "Polymorphism rs179008 is located within the TLR7 gene.<SEP>Polymorphism rs179008 of TLR7 was studied for its correlation with HIV disease progression.", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302<SEP>chunk-05d583f893b703816ad8de9561aa5292", "id": "RS179008"}, {"entity_type": "GENOMIC VARIANT", "description": "Another polymorphism, rs179009 in the TLR7 gene, was also examined for HIV disease risk in this context.<SEP>Polymorphism rs179009 is another variant found in the TLR7 gene.", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302<SEP>chunk-05d583f893b703816ad8de9561aa5292", "id": "RS179009"}, {"entity_type": "DISEASE", "description": "The study enrolled HIV-infected individuals to examine associations between genetic variations and disease progression.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "id": "HIV-INFECTED INDIVIDUALS"}, {"entity_type": "DEMOGRAPHIC", "description": "Participants\' mean age during enrollment ranged from 36 to 33 years, with SDs provided.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "id": "AGE (YEARS)"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The baseline CD4 cell count is a measure of an individual\'s immune system function and health status.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "id": "CD4 COUNTS"}, {"entity_type": "GENE", "description": "TLR7 gene polymorphisms have been studied in relation to HIV disease progression.", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "id": "TLR7 (RS179008)"}, {"entity_type": "GENOMIC VARIANT", "description": "The rs179009 variant of the TLR7 gene was also examined for its correlation with HIV disease progression.", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "id": "TLR7 (RS179009)"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Tobacco use is an environmental factor potentially impacting disease progression when examined alongside genetic markers like TLR7.", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "id": "TOBACCO"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Alcohol consumption is another environmental factor that was analyzed for its potential interaction with the rs179008 and rs179009 polymorphisms in relation to HIV disease progression.", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "id": "ALCOHOL"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The starting levels of CD4+ T cells, a key immune system component, were measured as part of the analysis for its interaction with the studied genetic markers and environmental factors in HIV disease progression.", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "id": "BASELINE CD4 COUNTS"}, {"entity_type": "DISEASE", "description": "HIV disease progression was the outcome variable being analyzed through multivariate logistic regression using various risk factors including TLR7 gene polymorphisms.", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "id": "HIV DISEASE PROGRESSION"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "Polymorphisms rs179008 and rs179009 in the TLR7 gene were significantly reduced or associated with HIV-infected individuals compared to healthy controls.)", "entity_type": "UNKNOWN", "id": "TLR7 POLYMORPHISMS"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "The rs179009 AG genotype was less frequent among HIV-infected individuals than in the control group, with a reduced risk of HIV infection.)", "entity_type": "UNKNOWN", "id": "TLR7 POLYMORPHISM (RS179009 AG GENOTYPE)"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "The distribution of haplotype AG was found higher in HIV-infected individuals compared to healthy controls, indicating a possible association with the disease.)", "entity_type": "UNKNOWN", "id": "TLR7 HAPLOTYPES (RS179008 AND RS179009)"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "Polymorphism rs179009 showed decreased frequency in individuals across early, intermediate, and advanced HIV stages compared to the healthy control group.)", "entity_type": "UNKNOWN", "id": "HIV DISEASE STAGES"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "Polymorphism rs179009 showed decreased frequency in individuals across early, intermediate, and advanced HIV stages compared to the healthy control group.)", "entity_type": "UNKNOWN", "id": "TLR7 POLYMORPHISMS (RS179009)"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "The frequency of rs179009 GG genotype was lower in individuals with advanced HIV disease stages compared to the healthy control group, suggesting a possible correlation.)", "entity_type": "UNKNOWN", "id": "ADVANCED HIV DISEASE STAGE"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "The frequency of rs179009 GG genotype was lower in individuals with advanced HIV disease stages compared to the healthy control group, suggesting a possible correlation.)", "entity_type": "UNKNOWN", "id": "TLR7 POLYMORPHISMS (RS179009 GG GENOTYPE)"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "Polymorphism distributions did not significantly differ between HIV-infected individuals with or without tobacco use; however, a higher prevalence of TLR7 rs179009AG genotype was noted among those who consumed tobacco.)", "entity_type": "UNKNOWN", "id": "ENVIRONMENTAL FACTORS (TOBACCO CONSUMPTION, ALCOHOL USE)"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "Polymorphism distributions did not significantly differ between HIV-infected individuals with or without tobacco use; however, a higher prevalence of TLR7 rs179009AG genotype was noted among those who consumed tobacco.)", "entity_type": "UNKNOWN", "id": "TLR7 POLYMORPHISMS (RS179008 AND RS179009)"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "The distributions of rs179008 CC, CT, and TT genotypes were nearly equivalent between tobacco-consuming and nonconsuming HIV-infected individuals.)", "entity_type": "UNKNOWN", "id": "TLR7 POLYMORPHISMS (RS179008 CC, CT, TT GENOTYPES)"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "The distributions of rs179008 CC, CT, and TT genotypes were nearly equivalent between tobacco-consuming and nonconsuming HIV-infected individuals.)", "entity_type": "UNKNOWN", "id": "TOBACCO-CONSUMING HIV-INFECTED INDIVIDUALS VS. NON-USERS"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "Similar distributions of rs179008 AA, AT, and TT and rs179009 AA, AG, and GG genotypes were observed between alcohol-taking HIV-infected individuals and non-takers.)", "entity_type": "UNKNOWN", "id": "ALCOHOL-TAKING HIV-INFECTED INDIVIDUALS VS. NON-USERS"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "Similar distributions of rs179008 AA, AT, and TT and rs179009 AA, AG, and GG genotypes were observed between alcohol-taking HIV-infected individuals and non-takers.)", "entity_type": "UNKNOWN", "id": "TLR7 POLYMORPHISMS (RS179009 AA, AG, GG GENOTYPES)"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "These genetic variations were analyzed through multivariate logistic regression for associations with various factors including age, sex, tobacco use, alcohol consumption, and baseline CD4 counts.)", "entity_type": "UNKNOWN", "id": "MULTIVARIATE LOGISTIC REGRESSION ANALYSIS"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "Associations were sought between specific TLR7 polymorphisms and HIV-infected status versus health in the study population.)", "entity_type": "UNKNOWN", "id": "HIV-INFECTED INDIVIDUALS VS. HEALTHY CONTROLS"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "The relationship between baseline CD4 cell count levels and different TLR7 polymorphisms was explored through this statistical method, aiming to uncover any significant correlations.)", "entity_type": "UNKNOWN", "id": "BASELINE CD4 COUNTS AND TLR7 POLYMORPHISMS"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "Assessments were made between various stages of HIV progression and the control group in terms of polymorphism frequencies.)", "entity_type": "UNKNOWN", "id": "EARLY, INTERMEDIATE, ADVANCED HIV DISEASE STAGES VS. HEALTHY CONTROLS"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "Comparative analysis was conducted between individuals who consumed tobacco and those who did not, with regard to TLR7 polymorphism distribution.)", "entity_type": "UNKNOWN", "id": "COMPARISONS"}, {"source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "description": "Analysis of correlation between genetic variants in the TLR7 gene with HIV disease progression through multivariate logistic regression.)", "entity_type": "UNKNOWN", "id": "TLR7 (RS179008 AND RS179009)"}, {"source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "description": "These factors were considered as potential independent risk factors for HIV disease progression alongside rs179008 and rs179009 variants of the TLR7 gene.)", "entity_type": "UNKNOWN", "id": "AGE, SEX, TOBACCO, ALCOHOL, BASELINE CD4 COUNTS"}, {"source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "description": "These factors were considered as potential independent risk factors for HIV disease progression alongside rs179008 and rs179009 variants of the TLR7 gene.)", "entity_type": "UNKNOWN", "id": "RISK FACTOR"}, {"source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "description": "The results from this correlation analysis were not directly presented, implying they supported or refuted the role of studied factors in HIV disease progression.)", "entity_type": "UNKNOWN", "id": "CORRELATION ANALYSIS RESULTS"}, {"source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "description": "The results from this correlation analysis were not directly presented, implying they supported or refuted the role of studied factors in HIV disease progression.)", "entity_type": "UNKNOWN", "id": "TABLE NOT SHOWN"}, {"source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "description": "Both genetic variations were included in multivariate logistic regression analysis to assess their influence on HIV disease progression.)", "entity_type": "UNKNOWN", "id": "RS179008 AND RS179009"}, {"source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "description": "These demographic and health indicators were considered as potential risk factors for HIV disease progression together with rs179008 and rs179009.)", "entity_type": "UNKNOWN", "id": "AGE, SEX, TOBACCO USE, ALCOHOL CONSUMPTION, BASELINE CD4 COUNTS"}, {"source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "description": "The analysis was aimed at identifying significant predictors of HIV disease progression including genetic markers, environmental factors, and health status parameters.)", "entity_type": "UNKNOWN", "id": "OUTCOME OF DISEASE PROGRESSION STUDY"}, {"entity_type": "GENOMIC VARIANT", "description": "ALK R1275Q is a well-known hotspot mutation in the ALK gene.<SEP>ALK gene mutation plays a major role in determining outcomes of targeted cancer immunotherapies.<SEP>The ALK gene contains a variant at position 1275 which introduces an arginine (Q) mutation.<SEP>The ALK gene mutation ALK R1275Q plays a critical role in cancer immunotherapies.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2<SEP>chunk-de413ae6100407d8dadd5bdee87c1074<SEP>chunk-614333487e4b92705503909d34155179", "id": "ALK R1275Q"}, {"entity_type": "GENOMIC VARIANT", "description": "NRAS Q61K is an activating mutation commonly found in melanoma, thyroid, and colorectal cancers.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "NRAS Q61K"}, {"entity_type": "NONAMER", "description": "ARDIYRASY is a predicted neoepitope derived from the ALK R1275Q mutation.<SEP>ARDIYRASY is the self-antigen peptide used as a control in the HLA-B*15:01 binding affinity study.<SEP>The self-antigen ARDIYRASY has a reduced binding affinity for HLA, leading to formation of only aggregated complexes.<SEP>The self-peptide sequence that is naturally presented by HLA-B*15:01 and serves as a comparison to the neoepitopes introduced.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2<SEP>chunk-de413ae6100407d8dadd5bdee87c1074", "id": "ARDIYRASY"}, {"entity_type": "NONAMER", "description": "GMAQDIYRA is an ALK R1275Q nonamer that binds HLA-A*02:01.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "GMAQDIYRA"}, {"entity_type": "NONAMER", "description": "ILDTAGKEEY is a neoepitope derived from the NRAS Q61K mutation.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "ILDTAGKEEY"}, {"entity_type": "DECAMER", "description": "MAQDIYRASY is another ALK R1275Q derived decamer neoepitope.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "MAQDIYRASY"}, {"entity_type": "DECAMER", "description": "AQDIYRASYY is a predicted HLA-B*15:01-restricted neoepitope arising from the ALK R1275Q mutation.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "AQDIYRASYY"}, {"entity_type": "MHC CLASS I", "description": "A major histocompatibility complex molecule on the surface of cells, specifically HLA-B*15:01, used to present peptides for T cell recognition.<SEP>HLA-B*15:01 is a major histocompatibility complex molecule that interacts with specific peptides, including ALK neoepitopes.<SEP>HLA-B*15:01 is the major histocompatibility complex class I molecule used in the study.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2<SEP>chunk-830d8819004960d1d149ac70149c72a0<SEP>chunk-de413ae6100407d8dadd5bdee87c1074", "id": "HLA-B*15:01"}, {"entity_type": "MHC CLASS I", "description": "HLA-A*02:01 is a common HLA allele that binds ALK R1275Q nonamer GMAQDIYRA.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "HLA-A*02:01"}, {"entity_type": "INTERACTION", "description": "The heterodimerization of \u03b1 and \u03b2 subunits forms stable p/MHC complexes with distinct stabilities in vitro.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "HETERODIMERIZATION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "ALK tumor neoantigens form p/MHC complexes with distinct stabilities, as demonstrated by SEC.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "ALK TUMOR NEOANTIGENS"}, {"entity_type": "GENOMIC VARIANT", "description": "The ZNF717 mutation (Q716) is present in three NBL samples.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "ZNF717 Q716"}, {"entity_type": "SAMPLE COLLECTION", "description": "The study includes a set of 100 primary neuroblastoma (NBL) sample pairs from the TARGET cohort.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "TARGET COHORT SAMPLES"}, {"entity_type": "BIOINFORMATICS METHOD", "description": "ProTECT analysis provides a range of therapeutically relevant neoantigen/HLA interactions for validation and characterization.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "PROTECT ANALYSIS"}, {"entity_type": "MOLECULAR CONSTRUCTION", "description": "The recombinant HLA-B*15:01 molecule is prepared using standard methods.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "RECOMBINANT HLA-B*15:01"}, {"entity_type": "PROTEIN REFOLDING MEDIUM", "description": "Purification of the protein from Escherichia coli inclusion bodies involves standard refolding and purification techniques.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "ESCHERICHIA COLI INCLUSION BODIES"}, {"entity_type": "ANALYTICAL TECHNIQUE", "description": "The SEC method separates and characterizes the different forms of complex molecules, such as p/MHC complexes and free \u03b22m.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "SIZE EXCLUSION CHROMATOGRAPHY (SEC)"}, {"entity_type": "ANALYTICAL METHODOLOGY", "description": "LC\u2013MS is used to confirm the presence of the correct peptide in each sample by analyzing their masses.", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "LIQUID CHROMATOGRAPHY\u2013MASS SPECTROSCOPY (LC\u2013MS)"}, {"entity_type": "GENE", "description": "The ALK gene encodes a protein that plays a role in cell signaling and is associated with cancer. The presence of specific amino acid sequences (AQDIYRASY) within the gene is denoted as the nonamer neoepitope.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "id": "ALK NONAMER NEOEPITOPE"}, {"entity_type": "PEPTIDE", "description": "A nine-amino acid sequence within the ALK protein that adopts an extended conformation.<SEP>Nonamers yield a stable p/MHC complex.<SEP>The ALK neoepitopes adopt a short 310 helix conformation, suggesting their interaction with the groove of MHC molecule.<SEP>The ALK neoepitopes are considered as a potential target for recognition by CD8+ T cells in the context of disease.<SEP>The nonamer complex provides insight into its unique conformation in the MHC groove, which impacts TCR interaction properties.<SEP>The nonamer neoepitope has comparable recognition frequency to an immunodominant epitope when tested using MHC tetramer staining.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803<SEP>chunk-830d8819004960d1d149ac70149c72a0", "id": "NONAMER"}, {"entity_type": "GENE", "description": "The ALK gene encodes a protein with specific amino acid sequences (AQDIYRASYY) denoted as the decamer neoepitope, which forms a compact 310 helical backbone.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "id": "ALK DECAMER NEOEPITOPE"}, {"entity_type": "PROTEIN", "description": "A ten-amino acid sequence within the ALK protein that adopts a short \'zig-zag\' conformation and fits compactly in the MHC groove.<SEP>Certain HLA and H2 MHC alleles yield partially folded, peptide-free molecules that can have measurable thermal stabilities.<SEP>Decamers result in a stable p/MHC complex.<SEP>Similar to the nonamer, the decamer neoepitope also shows recognition by CD8+ T cells as determined through the MHC tetramer staining assay.<SEP>The ALK neoepitopes display a conformation that resembles the 310 helix when bound to HLA-A*01:01.<SEP>The ALK neoepitopes in the decamer complex display polar contacts within both A- and F-pockets of MHC groove.<SEP>The decamer complex provides valuable insights into the peptide conformation when bound to HLA-A*01:01.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803<SEP>chunk-830d8819004960d1d149ac70149c72a0", "id": "DECAMER"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The process in which recombinant proteins are refolded into their native conformation outside of a living cell.<SEP>We performed in vitro refolding of ALK-derived peptides with HLA-A*01:01 to validate our structure-based simulations.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0<SEP>chunk-614333487e4b92705503909d34155179", "id": "IN VITRO REFOLDING"}, {"entity_type": "TECHNIQUE", "description": "A combination technique that separates and identifies molecules based on size and mass, used to confirm binding of neoepitopes to HLA-B*15:01.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "id": "LIQUID CHROMATOGRAPHY\u2013MASS SPECTROMETRY (LC\u2013MS)"}, {"entity_type": "TECHNIQUE", "description": "An assay for measuring kinetic stability or unfolding transition of proteins, used here to assess the thermal stability of p/MHC complexes.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "id": "DIFFERENTIAL SCANNING FLUORIMETRY (DSF)"}, {"entity_type": "BIOLOGICAL STRUCTURE", "description": "The recess in MHC molecules where peptides bind and are presented to T cells.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "id": "EPITOPE BINDING GROOVE"}, {"entity_type": "SEQUENCE FEATURE", "description": "Consensus sequence patterns found within MHC-restricted peptides.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "id": "PEPTIDE MOTIF"}, {"entity_type": "PHYSICAL PROPERTY", "description": "Further characterization of purified complexes reveals thermal stability at 47.9\u00b0C for decamer peptides refolded with HLA-A*01:01.<SEP>Measure of how well a molecule maintains its shape at varying temperatures.<SEP>The purified p/MHC complex shows a thermal stability of 47.9\u00b0C for decamer peptides.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0<SEP>chunk-614333487e4b92705503909d34155179", "id": "THERMAL STABILITY"}, {"entity_type": "CELL SURFACE PROTEIN", "description": "Receptors found on the surface of T lymphocytes that bind to peptides presented by MHC molecules.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "id": "T CELL RECEPTOR (TCR)"}, {"entity_type": "PEPTIDE", "description": "A peptide sequence consisting of nine amino acids that binds to HLA-B*15:01.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "NONAMER PEPTIDE"}, {"entity_type": "PEPTIDE", "description": "A peptide sequence consisting of ten amino acids that binds to HLA-B*15:01.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "DECAMER PEPTIDE"}, {"entity_type": "PROTEIN", "description": "The protein naturally expressed by the body that is recognized as self and does not elicit an immune response.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "SELF-ANTIGEN"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A type of chemical bond formed between atoms due to partial positive or negative charges on hydrogen, commonly seen in the interaction between residues and HLA molecules.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "HYDROGEN BONDS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Electrostatic interactions between positively charged (amino acids like lysine and arginine) and negatively charged (amino acids like aspartic acid and glutamic acid) side chains.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "SALT BRIDGES"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A type of interaction occurring between charged molecules, which in this context is the energy exchange when charges are attracted or repelled by the surface of HLA molecules and peptides.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "ELECTROSTATIC INTERACTIONS"}, {"entity_type": "STRUCTURAL ENTITY", "description": "The configuration where a nine-amino-acid peptide binds to HLA-B*15:01 with its specific orientation towards solvent.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "NONAMER STRUCTURE"}, {"entity_type": "STRUCTURAL ENTITY", "description": "The configuration where a ten-amino-acid peptide binds to HLA-B*15:01, showing altered conformation compared to the nonamer structure.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "DECAMER STRUCTURE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A specific type of secondary structure in proteins characterized by hydrogen bonds between backbone atoms that give rise to a 3.6 \u00e5ngstrom spacing along the helical axis.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "310 HELIX CONFORMATION"}, {"entity_type": "RESIDUE", "description": "The tyrosine residue at position nine in the decamer peptide which forms an additional hydrogen bond with the HLA heavy chain, stabilizing the complex.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "TYR9"}, {"entity_type": "GEOMETRIC ALIGNMENT", "description": "A method to compare two molecular structures by aligning them based on their backbone atoms and assessing the differences between them.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "STRUCTURAL SUPERPOSITION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Certain residues in proteins are positioned towards solvent during structural analysis, which can influence protein-protein or protein-ligand interactions.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "RESIDUES ORIENTED TOWARD SOLVENT"}, {"entity_type": "RESIDUE", "description": "The serine residue at position sixty-seven on \u03b11 helix of MHC molecule which is involved in potential clashes during self-peptide binding.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "SER67"}, {"entity_type": "RESIDUE", "description": "Alanine residue at position twenty-four on the floor of the MHC groove contributing to a clash with the self-decamer peptide.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "ALA24"}, {"entity_type": "RESIDUE", "description": "Methionine residue at position forty-five on the floor of the MHC groove which is involved in clashes during self-peptide binding.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "MET45"}, {"entity_type": "RESIDUE", "description": "Tyrosine residue nine in the decamer peptide, known for forming additional hydrogen bonds with HLA heavy chain and contributing to complex stability.", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "Y9 (TYR9)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "We evaluated non-redundant sets of 2,904 HLA alleles from the EMBL-EBI database to predict peptide/HLA interactions.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "HLA ALLELES"}, {"entity_type": "PROTEIN", "description": "We developed high-throughput approaches using X-ray structures for neoepitopes to predict peptide/HLA interactions and surface features of peptide residues poised for TCR interactions.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "NONAMER ALK PEPTIDE"}, {"entity_type": "PROTEIN", "description": "The approach was motivated by the observation that the peptide backbone conformation shows minimal variance in all nonamer/HLA-B*15:01 structures reported in the PDB.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "DECAMER ALK PEPTIDE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Our selection of a non-redundant set of 2,904 HLA alleles allowed us to carry out detailed Rosetta comparative modeling calculations for each allele.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "NON-REDUNDANT HLA ALLELES"}, {"entity_type": "STRUCTURAL MODEL", "description": "We used the experimentally determined structures of HLA-B*15:01 as templates in our calculations.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "HLA-B*15:01 STRUCTURE"}, {"entity_type": "PREVIOUS METHODOLOGY", "description": "In contrast to previous structure-based peptide/HLA modeling methods, we employed a fixed-peptide backbone threading approach.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "FLEXIBLE PEPTIDE DOCKING APPROACH"}, {"entity_type": "METHODOLOGY", "description": "Our strategy involved energy minimization of the interacting peptide and HLA residues to drastically confine the docking degrees of freedom.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "FIXED-PEPTIDE BACKBONE THREADING APPROACH"}, {"entity_type": "PREDICTIVE ANALYSIS", "description": "Using this method, we extracted highly reproducible binding energies for both nonamer and decamer peptides.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "PEPTIDE/HLA INTERACTION PREDICTIONS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Among the selected 116 HLA alleles from all A, B, and C types, those show lower binding energies for both nonamer and decamer peptides compared to our initial structural templates.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "LOWER BINDING ENERGY HLA ALLELES"}, {"entity_type": "EXPERIMENTAL COMPARISON", "description": "As a control, we computed the binding energy for a mock HLA allele with mutated residues, which exhibited high-binding energy indicating low affinity.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "NEGATIVE CONTROL MOCK HLA ALLELE"}, {"entity_type": "ANALYSIS RESULT", "description": "An evident correlation was observed between computed binding energies and sequence similarity to the best binding allele.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "RESIDUE SIMILARITY TO TOP BINDER"}, {"entity_type": "CHARACTERIZATION METRIC", "description": "We derived a sequence profile among good binders for neoepitopes using Kullback\u2013Leibler sequence logo, revealing preferred residues in the MHC-binding groove.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "SEQUENCE PROFILE OF GOOD BINDERS"}, {"entity_type": "IDENTIFICATION METRIC", "description": "Residues with high affinity binding consistently observed were identified through sequence profiles.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "KEY SEQUENCE FEATURES"}, {"entity_type": "CONTACT PATTERN", "description": "Inspection of structural models revealed similar polar contacts primarily in the A-, B-, and F-pockets, correlating well with positions of invariant MHC residues.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "POLAR CONTACTS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Our method predicts peptide/HLA binding energies and the sequence similarity to top binders.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "PEPTIDE BINDING ENERGY"}, {"entity_type": "ANALYTICAL TOOL", "description": "We used the Kullback\u2013Leibler sequence logo to derive a sequence profile among good binders.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "KULLBACK\u2013LEIBLER SEQUENCE LOGO"}, {"entity_type": "GENE", "description": "HLA-A*01:01 is an MHC allele.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "HLA-A*01:01"}, {"entity_type": "PROTEIN", "description": "The ALK neoepitopes have lower affinity for HLA-A*01:01 compared to HLA-B*15:01.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "ALK NEOEPITOPES"}, {"entity_type": "METABOLIC PATHWAY", "description": "Both complexes crystallize at high resolution, validating their atomic features as predicted by simulations.<SEP>Both decamer and nonamer complexes exhibit thermal stability, indicating their interaction with HLA-A*01:01.<SEP>Both decamer and nonamer complexes show thermal stability, suggesting the neoepitopes have lower affinity.<SEP>Their unique conformations suggest they might interact differently with T-cell receptors (TCRs).<SEP>These peptides form stable complexes with HLA-A*01:01, which allows for detailed structural characterization.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "DECAMER AND NONAMER"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Peptides form stable MHC complexes with HLA alleles.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "PEPTIDE"}, {"entity_type": "DISEASE", "description": "A known immunodominant epitope is associated with certain disease conditions related to immune response.", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "clusters": "[{level: 0, cluster: 11}, {level: 1, cluster: 36}, {level: 2, cluster: 60}]", "id": "KNOWN IMMUNODOMINANT EPITOPE"}, {"entity_type": "NUTRIENT", "description": "Vitamin D is a nutrient essential for bone health, immune system function, and more.", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "clusters": "[{level: 0, cluster: 4}, {level: 1, cluster: 23}]", "id": "VITAMIN D"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Regular physical training contributes to overall health by improving fitness levels.", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 20}]", "id": "PHYSICAL TRAINING"}, {"entity_type": "DISEASE", "description": "Cardiomyopathy is a disease affecting the heart muscle that can lead to heart failure.", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "clusters": "[{level: 0, cluster: 4}, {level: 1, cluster: 23}]", "id": "CARDIOMYOPATHY"}, {"entity_type": "METABOLIC PATHWAY", "description": "Calcium metabolism involves the absorption, distribution, and excretion of calcium in the body.", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "clusters": "[{level: 0, cluster: 4}, {level: 1, cluster: 23}]", "id": "CALCIUM METABOLISM"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Oxidative stress occurs when there is an imbalance between oxidants and antioxidants in the body.", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "clusters": "[{level: 0, cluster: 0}, {level: 1, cluster: 20}]", "id": "OXIDATIVE STRESS"}, {"entity_type": "METABOLIC PATHWAY", "description": "Glycolysis is a metabolic pathway involved in the breakdown of glucose to produce energy.", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "clusters": "[{level: 0, cluster: 4}, {level: 1, cluster: 23}]", "id": "GLYCOLYSIS"}, {"source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "description": "The ALK R1275Q mutation leads to the generation of neoepitopes like ARDIYRASY, which is predicted to bind HLA-B*15:01.)", "entity_type": "UNKNOWN", "id": "ALYRASY"}, {"source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "description": "The ALK R1275Q mutation leads to the generation of two distinct neoepitopes, MAQDIYRASY and AQDIYRASYY.)", "entity_type": "UNKNOWN", "id": "MAQDIYRASY/AQDIYRASYY"}, {"source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "description": "Nonamer GMAQDIYRA, derived from ALK R1275Q mutation, forms a complex that exhibits distinct stability compared to other peptides.)", "entity_type": "UNKNOWN", "id": "STABILITY"}, {"source_id": "chunk-830d8819004960d1d149ac70149c72a0", "description": "The ALK nonamer neoepitope binds to HLA-B*15:01 through in vitro refolding.)", "entity_type": "UNKNOWN", "id": "BOUND BY HLA-B*15:01"}, {"source_id": "chunk-830d8819004960d1d149ac70149c72a0", "description": "The ALK nonamer and decamer neoepitopes adopt distinct conformations within the MHC groove: extended for the nonamer and compact 310 helix for the decamer.)", "entity_type": "UNKNOWN", "id": "ALK NONAMER NEOEPITOPE/DECAMER NEOEPITOPE"}, {"source_id": "chunk-830d8819004960d1d149ac70149c72a0", "description": "The ALK nonamer and decamer neoepitopes adopt distinct conformations within the MHC groove: extended for the nonamer and compact 310 helix for the decamer.)", "entity_type": "UNKNOWN", "id": "CONFORMATION"}, {"source_id": "chunk-830d8819004960d1d149ac70149c72a0", "description": "A comparison between the ALK nonamer (nine amino acids) and decamer (ten amino acids), showing how peptide length influences conformation within a fixed MHC groove.)", "entity_type": "UNKNOWN", "id": "LENGTH"}, {"source_id": "chunk-830d8819004960d1d149ac70149c72a0", "description": "A comparison between the ALK nonamer (nine amino acids) and decamer (ten amino acids), showing how peptide length influences conformation within a fixed MHC groove.)", "entity_type": "UNKNOWN", "id": "NONAMER/DECAMER"}, {"source_id": "chunk-830d8819004960d1d149ac70149c72a0", "description": "Epitopes", "entity_type": "UNKNOWN", "id": "BIND TO"}, {"source_id": "chunk-830d8819004960d1d149ac70149c72a0", "description": "Epitopes", "entity_type": "UNKNOWN", "id": "MHC MOLECULES"}, {"source_id": "chunk-830d8819004960d1d149ac70149c72a0", "description": "Antigen Presentation", "entity_type": "UNKNOWN", "id": "ANTIGEN-PRESENTING CELLS (APCS)"}, {"source_id": "chunk-830d8819004960d1d149ac70149c72a0", "description": "Antigen Presentation", "entity_type": "UNKNOWN", "id": "PROCESS OF"}, {"source_id": "chunk-830d8819004960d1d149ac70149c72a0", "description": "Peptides Present On MHC molecules", "entity_type": "UNKNOWN", "id": "RECOGNIZE AND BIND TO"}, {"source_id": "chunk-830d8819004960d1d149ac70149c72a0", "description": "Peptides Present On MHC molecules", "entity_type": "UNKNOWN", "id": "TCR"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "Longer arginine side chain causes steric hindrance between peptide and MHC-binding groove, affecting affinity for self-peptide binding to HLA-B*15:01.)", "entity_type": "UNKNOWN", "id": "ARG2 SIDE CHAIN"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "Longer arginine side chain causes steric hindrance between peptide and MHC-binding groove, affecting affinity for self-peptide binding to HLA-B*15:01.)", "entity_type": "UNKNOWN", "id": "STRUCTURAL HINDRANCE"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "Increase in intra-peptide hydrogen bonds contribute to the stability of the decamer complex compared to nonamer structure.)", "entity_type": "UNKNOWN", "id": "STABILIZING INTERACTION"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "Polar interactions such as hydrogen bonds and salt bridges enhance stability of HLA complexes with peptides.)", "entity_type": "UNKNOWN", "id": "POLAR INTERACTIONS"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "Polar interactions such as hydrogen bonds and salt bridges enhance stability of HLA complexes with peptides.)", "entity_type": "UNKNOWN", "id": "STABILIZING FORCES"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "Additional hydrophobic packing interactions further stabilize the decamer complex by reducing electrostatic repulsion between hydrophobic side chains.)", "entity_type": "UNKNOWN", "id": "HYDROPHOBIC PACKING"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "The self-antigen\'s structure results in clashes with MHC groove, leading to its exclusion from forming a stable HLA complex compared to neoepitopes.)", "entity_type": "UNKNOWN", "id": "STRUCTURAL EXCLUSION"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "The potential of other HLAs to display ALK R1275Q neoepitopes depends on the individual\'s HLA haplotype, playing a role in immune response.)", "entity_type": "UNKNOWN", "id": "HAPLOTYPE-SPECIFIC BINDING"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "Comparison of affinities between self-peptide and neoepitopes for HLA-B*15:01 highlights potential differences in immunogenicity and therapeutic relevance.)", "entity_type": "UNKNOWN", "id": "AFFINITY DIFFERENCES"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "Comparison of affinities between self-peptide and neoepitopes for HLA-B*15:01 highlights potential differences in immunogenicity and therapeutic relevance.)", "entity_type": "UNKNOWN", "id": "P/MHC COMPLEXES"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "The affinity of nonamer peptides towards HLA-B*15:01 is lower compared to other neoepitopes due to structural compatibility.)", "entity_type": "UNKNOWN", "id": "NONAMER BINDING"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "Increase in binding energy and structural stability provided by the decamer configuration results from a more compact conformation and enhanced interactions with HLA-B*15:01.)", "entity_type": "UNKNOWN", "id": "DECAMER PRESENTATION"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "Increase in binding energy and structural stability provided by the decamer configuration results from a more compact conformation and enhanced interactions with HLA-B*15:01.)", "entity_type": "UNKNOWN", "id": "ENHANCED STABILITY"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "There\'s a clear sequence bias for specific residues in the MHC-binding groove yielding more favorable interactions with two peptides.)", "entity_type": "UNKNOWN", "id": "SEQUENCE BIAS"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "The mock polyAla HLA exhibits considerably higher binding energy across the entire peptide length compared to top binders.)", "entity_type": "UNKNOWN", "id": "HIGH BINDING ENERGY"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "The mock polyAla HLA exhibits considerably higher binding energy across the entire peptide length compared to top binders.)", "entity_type": "UNKNOWN", "id": "MOCK HLA ALLELE"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "The ALK R1275Q genomic variant is associated with cancer immunotherapies outcomes.)", "entity_type": "UNKNOWN", "id": "CANCER IMMUNOTHERAPIES"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "Our simulations accurately predict peptide/HLA binding energies and interactions based on sequence similarity.)", "entity_type": "UNKNOWN", "id": "PREDICTION ACCURACY"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "The thermal stability measurement reflects the stability of the p/MHC complex in vitro.)", "entity_type": "UNKNOWN", "id": "PHYSICAL PROPERTY"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "The sequence logo provides insights into preferred residues for effective peptide/HLA binding.)", "entity_type": "UNKNOWN", "id": "SEQUENCE ANALYSIS"}, {"source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "description": "The binding energy calculations suggest that both neoepitopes have a lower affinity for HLA-A*01:01 compared to HLA-B*15:01.)", "entity_type": "UNKNOWN", "id": "BINDING ENERGY"}, {"source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "description": "The ALK-derived neoepitopes can be recognized by CD8+ T cells, as evidenced by MHC tetramer staining assay and multichannel flow cytometry analysis.)", "entity_type": "UNKNOWN", "id": "RECOGNITION BY CD8+ T CELLS"}, {"source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "description": "When bound to HLA-A*01:01, the side chain of Arg6 flips outward in comparison to its position when bound to HLA-B*15:01, altering TCR interaction properties.)", "entity_type": "UNKNOWN", "id": "CONFORMATIONAL COMPARISON"}, {"source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "description": "The recognition frequencies for the nonamer and a known immunodominant SARS-CoV-2 epitope are comparable when tested using MHC tetramers.)", "entity_type": "UNKNOWN", "id": "COMPARISON WITH SARS-COV-2 DERIVED EPITOPES"}, {"source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "description": "Both neoepitopes display nominal recognition frequencies by CD8+ T cells, highlighting their potential as targets for immune response study.)", "entity_type": "UNKNOWN", "id": "RECOGNITION EFFICIENCY COMPARISON"}, {"entity_type": "GENOMIC VARIANT", "description": "One single nucleotide polymorphism (SNP) rs74420772 in 3p14.1 achieved genome-wide significance in TwinGene.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "RS74420772"}, {"entity_type": "DISEASE", "description": "IgM anti-PC levels were lower in prevalent CLL cases compared to matched controls, but not different between incident cases and their matches.<SEP>Individuals with IgM anti-PC measurements and genotypes were used in the study.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a<SEP>chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "IGM ANTI-PC"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs74420772 achieved genome-wide significance in TwinGene (P = 2.7 \u00d7 10\u22128).", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "SNP RS74420772"}, {"entity_type": "GENOMIC VARIANT", "description": "The allele rs35923643-G or its proxy SNP rs735665-A was recognized as the top risk allele for CLL.<SEP>The leading SNP identified in our GWAS meta-analysis for IgM anti-PC, located on chromosome 11q24.1.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "RS735665"}, {"entity_type": "GENOMIC VARIANT", "description": "rs35923643-G has been identified as a genome-wide significant SNP associated with IgM anti-PC and is part of the LD block that overlaps with the GRAM domain containing 1B (GRAMD1B) gene.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "RS35923643-G"}, {"entity_type": "GENOMIC REGION", "description": "Two SNPs in 1p31.3 and six SNPs in 11q24.1 achieved genome-wide significance with P < 5.0 \u00d7 10\u22128.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "1P31.3 AND 11Q24.1"}, {"entity_type": "GENOMIC VARIANT", "description": "rs735665-A was identified as a main genetic variant specific for general IgM, rather than general IgA and general IgG.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "RS735665-A"}, {"entity_type": "DISEASE/CONDITION", "description": "Two disease conditions that were not the main focus of analysis but were compared with the main condition (general IgM) in our study.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "IGA AND GENERAL IGG"}, {"entity_type": "CELL LINE", "description": "One of several cell lines used for scanning potential regulatory marks associated with genomic variants in the top locus 11q24.1.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "GM12878"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs35923643 is located on locus 11q24.1.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "RS35923643"}, {"entity_type": "DISEASE", "description": "rs35923643 is associated with Chronic Lymphocytic Leukemia (CLL), both prevalent and incident cases.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "CLL"}, {"entity_type": "CELL LINE", "description": "This is a cell line used for ChIP-seq experiments and analysis of epigenetic marks.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "GM12878 CELL LINE"}, {"entity_type": "CELL TYPE", "description": "These are hematopoietic stem cells or progenitor cells that were analyzed using DNaseI hypersensitivity data.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "CD34+ CELLS"}, {"entity_type": "CELL LINE", "description": "Another cell line used for analysis of DNaseI hypersensitivity.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "GM12865 CELL LINE"}, {"entity_type": "CELL TYPE", "description": "A specific type of white blood cell analyzed using ChIP-seq data to determine RUNX3 binding affinity.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "B-CELL LINE"}, {"source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "description": "rs35923643-G is part of the LD block overlapping with GRAM domain containing 1B (GRAMD1B) gene.)", "entity_type": "UNKNOWN", "id": "11Q24.1"}, {"source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "description": "rs735665-A was identified as a genome-wide significant SNP associated with general IgM increasing alleles related to IgM anti-PC.)", "entity_type": "UNKNOWN", "id": "GENERAL IGM AND IGM ANTI-PC"}, {"source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "description": "The allele rs735665-A was associated not only with CLL but also showed a negative association with the risk of certain disease states involving general IgM, general IgA, or general IgG levels.)", "entity_type": "UNKNOWN", "id": "GENERAL IGM AND GENERAL IGA/IGG"}, {"source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "description": "rs35923643 was found to have peaks of histone mark monomethylated histone H3 lysine 4 (H3K4Me1) and DNaseI hypersensitivity in leukemia or immune-cell lines.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 7}, {level: 1, cluster: 26}]", "id": "CHROMATIN"}, {"source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "description": "ChIP-seq experiments suggested binding sites for 22 different transcription factors within the region around rs35923643.)", "entity_type": "UNKNOWN", "id": "TF BINDING"}, {"source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "description": "ChIP-seq experiments suggested binding sites for 22 different transcription factors within the region around rs35923643.)", "entity_type": "UNKNOWN", "id": "TRANSCRIPTION FACTORS"}, {"source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "description": "The strongest signals were observed for RUNX3 and SPI1 from FIMO tool, with the RUNX3 binding motif including SNP rs35923643.)", "entity_type": "UNKNOWN", "id": "BINDING MOTIFS"}, {"source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "description": "The strongest signals were observed for RUNX3 and SPI1 from FIMO tool, with the RUNX3 binding motif including SNP rs35923643.)", "entity_type": "UNKNOWN", "id": "RUNX3 AND SPI1 TFS"}, {"source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "description": "The RegulomeDB database gave support for rs35923643 affecting the binding of transcription factors in general.)", "entity_type": "UNKNOWN", "id": "REGULOMEDB SUPPORT"}, {"source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "description": "IgM anti-PC was lower in prevalent CLL cases compared to matched controls, with a declining trend between sampling and diagnosis but not statistically significant.)", "entity_type": "UNKNOWN", "id": "CLL CASE ASSOCIATION"}, {"source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "description": "The Encyclopedia of DNA Elements (ENCODE) provided data on transcription factor binding within the vicinity of rs35923643, identifying suggested binding sites for 22 different TFs.)", "entity_type": "UNKNOWN", "id": "ENCODE DATA"}, {"source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "description": "The Encyclopedia of DNA Elements (ENCODE) provided data on transcription factor binding within the vicinity of rs35923643, identifying suggested binding sites for 22 different TFs.)", "entity_type": "UNKNOWN", "id": "TFS"}, {"source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "description": "A small nested case-control study found IgM anti-PC to be lower in prevalent cases of CLL compared to matched controls, but not significantly associated with incident cases using stratified Cox proportional hazards model.)", "entity_type": "UNKNOWN", "id": "PREVALENT CLL ASSOCIATION"}, {"source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "description": "An analysis revealed a non-statistically significant declining trend in IgM anti-PC levels between CLL cases and matched controls as time from sampling to diagnosis increased.)", "entity_type": "UNKNOWN", "id": "STATISTICAL TREND"}, {"source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "description": "An analysis revealed a non-statistically significant declining trend in IgM anti-PC levels between CLL cases and matched controls as time from sampling to diagnosis increased.)", "entity_type": "UNKNOWN", "id": "TIME BETWEEN SAMPLING AND DIAGNOSIS"}], "links": [{"weight": 0.046, "description": "Education level interacts with the presence of the FTO SNP to affect the risk of developing obesity.)<SEP>The effect of the FTO SNP on obesity risk can vary depending on educational background.)", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "order": 1, "source": "OBESITY", "target": "EDUCATION"}, {"weight": 2.01, "description": "An association was investigated between a variant in the FTO gene (the FTO SNP) and obesity risk.)", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "order": 1, "source": "OBESITY", "target": "FTO SNP"}, {"weight": 8.0, "description": "rs182549 is associated with obesity.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "OBESITY", "target": "RS182549"}, {"weight": 2.0, "description": "FTO_rs1421085 contributes to obesity risk through its role in modulating fat mass and energy balance.)", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "order": 1, "source": "FTO_RS1421085", "target": "OBESITY RISK"}, {"weight": 2.0, "description": "The genetic variant SEC16B_rs543874 interacts with MC4R_rs6567160 in influencing obesity risk.)", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "order": 1, "source": "MC4R_RS6567160", "target": "SEC16B_RS543874"}, {"weight": 1.0, "description": "The initial menstruation age interacted with the Polygenic Risk Score for obesity risk, indicating an environmental factor\'s influence on genetic predispositions.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "PRS", "target": "MENARCHE AGE"}, {"weight": 2.0, "description": "A plant-based diet intake showed an interaction with PRS in influencing obesity risk, highlighting a relationship between dietary choices and genetic predisposition.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "PRS", "target": "PLANT-BASED DIET INTAKE"}, {"weight": 3.0, "description": "Fried food intake exhibited an interaction with the PRS for obesity risk, indicating that dietary factors can modify genetic influences on obesity.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "PRS", "target": "FRIED-FOOD INTAKE"}, {"weight": 2.0, "description": "The impact of PRS on obesity was measured through BMI, showing a link between genetics and body weight regulation.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "BMI", "target": "OUTCOME MEASURE"}, {"weight": 1.0, "description": "The FTO SNP showed strong association with BMI in GP subjects.)", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "order": 1, "source": "BMI", "target": "FTO RS9939609 POLYMORPHISM"}, {"weight": 8.0, "description": "The analyses did not find evidence for an interaction between the TCF7L2 risk variant and BMI.)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "BMI", "target": "INTERACTION"}, {"weight": 10.0, "description": "Genetic risk score is positively associated with increased BMI at baseline, indicating a cumulative effect of multiple genetic factors on obesity.)<SEP>The genetic risk score is a cumulative measure influenced by multiple obesity predisposing SNPs and correlates significantly with BMI at baseline.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "BMI", "target": "GRS"}, {"weight": 5.0, "description": "The genetic variants, GRS of these SNPs interact with traits related to glycemic status to influence BMI.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "BMI", "target": "RS9939609, RS2984618, RS7903146, GRS, LEPR GENE, FPG"}, {"weight": 1.0, "description": "Carriers of any minor alleles had a lower mean BMI compared to those with all major alleles at baseline (mean = 27.8 SD 3.7 vs. 26.1 SD 3.6, p=0.02).)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "BMI", "target": "GENOTYPE GROUP"}, {"weight": 0.78, "description": "BMI shows a moderate positive genetic correlation with snoring prevalence.)", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "order": 1, "source": "BMI", "target": "GENETIC CORRELATION"}, {"weight": 2.0, "description": "A measure of body fat based on height and weight, used as an indicator in this context.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "BMI", "target": "BODY MEASUREMENT/BODY MASS INDEX"}, {"weight": 4.0, "description": "The rs671 polymorphism impacts BMI significantly only among men, suggesting an influence of the genetic factor on body mass.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "BMI", "target": "GENOTYPES"}, {"weight": 6.0, "description": "The BMI was lower in the control group compared to participants with HUA.)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "BMI", "target": "HUA"}, {"weight": 2.5, "description": "In IBS-D subgroup, the BMI was approximately 4 units higher than in other subgroups (P = .036).)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "BMI", "target": "IBS-D"}, {"weight": 8.0, "description": "BMI correlated positively with serum C4 levels, indicating a possible influence of obesity on markers related to bile acid metabolism or other metabolic processes.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "BMI", "target": "SERUM C4"}, {"weight": 1.0, "description": "Covariates were adjusted for in statistical analysis when relating PRS to other factors like menarche age.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "COVARIATES", "target": "ADJUSTMENT FACTOR"}, {"weight": 3.0, "description": "Adjusted Odds Ratios were used to measure the effect of PRS in obesity risk analysis.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "ADJUSTED ORS", "target": "EFFECT MEASURE"}, {"weight": 4.0, "description": "Confidence intervals estimated the precision and reliability of adjusted odds ratios in statistical outcomes.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "95% CI", "target": "PRECISION ESTIMATE"}, {"weight": 5.0, "description": "Tables provided a structured way to present data from statistical analysis including key results like ORs and CI.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "TABLE 6", "target": "DATA PRESENTATION"}, {"weight": 2.0, "description": "The genetic variant DNAJC27_rs713586 affects BDNF_rs6265 which contributes to the obesity risk.)", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "order": 1, "source": "BDNF_RS6265", "target": "DNAJC27_RS713586"}, {"weight": 2.0, "description": "The genetic variant CDKAL1_rs9356744 is connected to GIPR_rs1444988703, impacting obesity risk.)", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "order": 1, "source": "CDKAL1_RS9356744", "target": "GIPR_RS1444988703"}, {"weight": 4.0, "description": "Age is often an adjustment factor in statistical models when analyzing treatment response.)", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "order": 1, "source": "ADJUSTMENT FACTOR", "target": "AGE"}, {"weight": 5.0, "description": "A significantly extended survival benefit was observed in mice receiving repeat doses of TAG72-BB\u03b6 CAR T Cells over a single dose.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "OUTCOME MEASURE", "target": "OVERALL SURVIVAL BENEFIT"}, {"weight": 1.0, "description": "Genetic variant rs1558902 was associated with weight loss, particularly in the context of a high-protein diet.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "RS1558902", "target": "WEIGHT"}, {"weight": 1.0, "description": "A nonsynonymous SNP in INADL was associated with weight, BMI, FFM, FM, trunk FM and hip circumference.)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "WEIGHT", "target": "INADL"}, {"weight": 3.0, "description": "A high-protein diet influenced fat distribution differently based on the FTO rs1558902 genotype, affecting total abdominal thickness (TAT), visceral adipose tissue (VAT), and subcutaneous adipose tissue (SAT).)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "PROTEIN DIET INTERVENTION", "target": "FAT DISTRIBUTION"}, {"weight": 4.0, "description": "Age was adjusted for in the analysis, and interactions were found between age and FTO rs1558902 genotype on weight changes.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "AGE", "target": "ASSOCIATION WITH GENOTYPE"}, {"weight": 6.0, "description": "There was no evidence for an interaction between the TCF7L2 risk variant and age in these analyses.)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "AGE", "target": "INTERACTION"}, {"weight": 2.0, "description": "Non-Hispanic White mothers had a mean age of 31.8 years, while Hispanic mothers had a mean age of 27.0 years.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "AGE", "target": "NON-HISPANIC WHITE MOTHERS"}, {"weight": 3.0, "description": "Age influences serum folate levels and semen quality; older individuals may show decreased folate concentrations.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "AGE", "target": "SERUM FOLATE"}, {"weight": 9.0, "description": "Age is a factor in determining fertility status; older individuals tend to have differences in reproductive health metrics compared to younger individuals.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "AGE", "target": "FERTILITY STATUS"}, {"weight": 3.0, "description": "ALDH2 genotypes are associated with age differences in both males and females.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "AGE", "target": "GENOTYPES"}, {"weight": 4.0, "description": "Genotypes associated with the rs671 variant may result in age differences among genotypic groups.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "AGE", "target": "RS671 GENOTYPES"}, {"weight": 3.0, "description": "In lactase non-persistent CC individuals, LCT mRNA was significantly decreasing with age, whereas there were no aging changes in intermediate CT individuals and a non-significant increase in lactase persistent TT individuals.)", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "order": 1, "source": "AGE", "target": "LCT MRNA"}, {"weight": 5.0, "description": "Sex differences influenced the association of the FTO genetic variant with body composition changes; sex was included in the analysis.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "SEX", "target": "EFFECT ON BODY COMPOSITION"}, {"weight": 5.0, "description": "Sex influenced the relationship between the FTO SNP and BMI, as well as obesity risk.)", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "order": 1, "source": "SEX", "target": "INTERACTION WITH BMI AND OBESITY"}, {"weight": 7.0, "description": "There was no evidence for an interaction between the TCF7L2 risk variant and sex in these analyses.)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "SEX", "target": "INTERACTION"}, {"weight": 5.0, "description": "Sex can influence differences in how individuals respond to treatments, reflecting a lifestyle factor.)", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "order": 1, "source": "SEX", "target": "DIFFERENCE IN RESPONSE"}, {"weight": 6.0, "description": "Ethnicity interacted with the FTO rs1558902 genotype, potentially influencing weight and fat distribution responses to dietary interventions.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "ETHNICITY", "target": "INTERACTION WITH GENOTYPE"}, {"weight": 1.0, "description": "The age-genotype interaction was significant in multi-ethnic cohorts, but not observed in the GEM_v24 cohort which contains individuals of different Israeli subethnicities.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "ETHNICITY", "target": "RS182549"}, {"weight": 2.0, "description": "Participants with the AA genotype responded differently to low-protein diets compared to those without this specific genetic variant.)", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "order": 1, "source": "AA GENOTYPE", "target": "LOW-PROTEIN DIET"}, {"weight": 3.0, "description": "The AA genotype showed a distinct response pattern when subjected to high-protein diets, with greater fat distribution changes in comparison to participants lacking the AA variant.)", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "order": 1, "source": "AA GENOTYPE", "target": "HIGH-PROTEIN DIET"}, {"weight": 1.0, "description": "The AA variant of the GCKR gene is associated with higher TAG concentrations compared to the GG genotype.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "AA GENOTYPE", "target": "GCKR RS780094 SNP"}, {"weight": 5.0, "description": "Similar to rs780094, this SNP also exhibits significant association with hypertriacylglycerolaemia when combined with the AA genotype.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "AA GENOTYPE", "target": "RS1260326 SNP"}, {"weight": 7.0, "description": "A low-protein diet was used as a primary treatment before considering other interventions.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "LOW-PROTEIN DIET", "target": "TREATMENT FOR HI/HA"}, {"weight": 4.0, "description": "A specific relationship was observed between genotypes and dietary responses on total fat, FM%, and trunk fat changes when exposed to low-fat diets.)", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "order": 1, "source": "LOW-FAT DIET", "target": "GENOTYPE-TIME INTERACTION"}, {"weight": 2.0, "description": "The FTO genetic variant interacted with dietary protein to influence changes in fat mass, free-fat mass, and percentage of trunk fat.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "CHANGES IN BODY COMPOSITION", "target": "FTO RS1558902 GENOTYPE"}, {"weight": 4.0, "description": "In the GP subjects, there was an interaction between educational level and the FTO SNP in determining BMI.)", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "order": 1, "source": "FTO RS9939609 POLYMORPHISM", "target": "EDUCATION LEVEL"}, {"weight": 9.0, "description": "Waist circumference is an indicator of excess abdominal fat, linked to metabolic syndrome and diabetes risk.)", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "order": 1, "source": "WAIST CIRCUMFERENCE", "target": "OBESITY INDICATOR"}, {"weight": 2.0, "description": "The FTO SNP\'s influence on obesity risk was modulated by education level in GP subjects.)", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "order": 1, "source": "EDUCATION LEVEL", "target": "INTERACTION WITH FTO SNPC"}, {"weight": 3.0, "description": "In elderly HCR subjects, PA was found to interact with the FTO SNP affecting anthropometric measurements.)", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "order": 1, "source": "PHYSICAL ACTIVITY (PA)", "target": "IMPACT ON FTO SNP EFFECT"}, {"weight": 0.023, "description": "An interaction term between the FTO gene polymorphism and education level was found when determining obesity risk.)", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "order": 1, "source": "FTO SNP", "target": "EDUCATION"}, {"weight": 0.957, "description": "The effect of education on the association between the FTO SNP and obesity risk was adjusted for physical activity.)", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "order": 1, "source": "EDUCATION", "target": "PA (PHYSICAL ACTIVITY)"}, {"weight": 1.0, "description": "The NOS1 gene interacts with folate metabolism to regulate nitric oxide production.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "NOS1", "target": "FOLATE"}, {"weight": 3.0, "description": "A genomic variant in the NOS1 gene is linked to an increased risk of cardiovascular disease.)<SEP>A specific genomic variant may increase the risk of developing cardiovascular disease.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862<SEP>chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "GENOMIC VARIANT", "target": "CARDIOVASCULAR DISEASE"}, {"weight": 1.0, "description": "Genomic variations might predispose individuals to a quicker manifestation of MTX-induced neurotoxic effects.)", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "order": 1, "source": "GENOMIC VARIANT", "target": "EARLY MTX-RELATED NEUROTOXICITY"}, {"weight": 2.0, "description": "The transcript analysis could highlight how specific genomic variants affect the expression levels of certain genes, thereby influencing disease risks or responses.)", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "order": 1, "source": "GENOMIC VARIANT", "target": "TRANSCRIPT"}, {"weight": 2.0, "description": "The association between SLC30A8 rs13266634 and fasting proinsulin levels was observed in the DPP cohort.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "GENOMIC VARIANT", "target": "DPP COHORT"}, {"weight": 2.0, "description": "A genomic variant specific to the IL-6R\u03b1 gene influences disease protection during T cell transfers.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "GENOMIC VARIANT", "target": "IL-6R\u0391 GENE"}, {"weight": 6.0, "description": "Correlation between low SENP1 expression and onset of diabetes in NOD mice suggests a potential link to genetic predisposition towards autoimmune diseases like T1DM.)<SEP>NA, no specific genomic variant related disease mentioned)<SEP>Specific genomic variants associated with IL-10R\u03b1Mdel and Rag1 affect both Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 disease severity.)<SEP>The rs1138272 genetic variant did not exhibit a significant link to the development of breast cancer.)", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "order": 1, "source": "GENOMIC VARIANT", "target": "DISEASE"}, {"weight": 9.0, "description": "Specific genotypic differences within the IL-6R\u03b1 gene and IL-10R\u03b1Mdel variants influence disease modulations across different genetic backgrounds like Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "GENOMIC VARIANT", "target": "DISEASE MODULATION"}, {"weight": 2.0, "description": "21.6% of the infants carried rs12325817 variant of the enzyme PEMT, which reduces endogenous choline synthesis)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "GENOMIC VARIANT", "target": "INFANTS"}, {"weight": 8.0, "description": "Contributes to a low allele frequency pattern consistent with previous studies.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "GENOMIC VARIANT", "target": "RS9923231"}, {"weight": 9.0, "description": "These are other individual SNPs that were part of the combined CYP2C9 star variant group.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "GENOMIC VARIANT", "target": "CYP2C9*5, CYP2C9*8, AND CYP2C9*11"}, {"weight": 6.0, "description": "The genomic variant rs2023239 is part of the CNR1 gene and affects its expression and function.)", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "GENOMIC VARIANT", "target": "CNR1"}, {"weight": 3.0, "description": "The gene and its variants may have complex interactions that affect genetic susceptibility to certain diseases.)", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "GENOMIC VARIANT", "target": "GENE"}, {"weight": 1.0, "description": "Increased IFN-\u03b3 expression was associated with specific genotypes in certain CD subjects.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "GENOMIC VARIANT", "target": "IFN-\u0393"}, {"weight": 4.0, "description": "Genetic variations had effects on expression levels through eQTLs.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "GENOMIC VARIANT", "target": "EQTL EFFECTS"}, {"weight": 2.0, "description": "rs35923643 was found to have peaks of histone mark monomethylated histone H3 lysine 4 (H3K4Me1) and DNaseI hypersensitivity in leukemia or immune-cell lines.)", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "order": 1, "source": "GENOMIC VARIANT", "target": "CHROMATIN"}, {"weight": 4.0, "description": "NA, no specific transcript-dietary pattern relationship mentioned)<SEP>The transcript of the NOS1 gene is influenced by dietary patterns, particularly folate intake.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "order": 1, "source": "TRANSCRIPT", "target": "DIETARY PATTERN"}, {"weight": 3.0, "description": "The transcript of pro-IL-\u03b2 is involved in regulating the maturation of IL-1\u03b2.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "TRANSCRIPT", "target": "PRO-IL-\u0392"}, {"weight": 1.0, "description": "Expression levels of transcripts are altered in adipocytes with SENP1 deletion, which may influence metabolic processes and inflammatory responses related to diabetes. )", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "order": 1, "source": "TRANSCRIPT", "target": "ADIPOCYTE"}, {"weight": 1.0, "description": "A transcript connected with GSTP1 gene functions was investigated for single nucleotide polymorphisms (SNPs).)", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "order": 1, "source": "TRANSCRIPT", "target": "GENE"}, {"weight": 4.0, "description": "Transcripts can influence protein expression which in turn affects metabolic pathways.)<SEP>Transcripts of genes can influence the expression level or function of proteins, impacting biological processes.)", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea<SEP>chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "TRANSCRIPT", "target": "PROTEIN"}, {"weight": 12.0, "description": "The NOS1 protein interacts with folate metabolism to maintain vascular function.)<SEP>The NOS1 protein interacts with folate metabolism to modulate its effects on cardiovascular health.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432<SEP>chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "NOS1 PROTEIN", "target": "FOLATE METABOLISM"}, {"weight": 4.0, "description": "Alterations in the NOS1 protein can contribute to cardiovascular disease.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "NOS1 PROTEIN", "target": "CARDIOVASCULAR DISEASE"}, {"weight": 6.0, "description": "The NOS1 protein catalyzes the production of nitric oxide, playing a role in its synthesis.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "NOS1 PROTEIN", "target": "NITRIC OXIDE"}, {"weight": 10.0, "description": "Nutritional factors from dietary patterns can affect the expression and function of the NOS1 protein.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "NOS1 PROTEIN", "target": "DIETARY PATTERNS"}, {"weight": 5.0, "description": "Folate is a nutrient found in leafy greens, which are important for cardiovascular health.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "FOLATE", "target": "LEAFY GREENS"}, {"weight": 0.5, "description": "There\'s a potential causal relationship between folate and vitamin B12 levels, but it might not impact offspring birthweight directly.)", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "order": 1, "source": "FOLATE", "target": "B12"}, {"weight": 0.3, "description": "The impact of maternal consumption of folate on offspring birthweight was not found to have a causal effect based on the data.)", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "order": 1, "source": "FOLATE", "target": "OFFSPRING BIRTHWEIGHT"}, {"weight": 6.0, "description": "Poor dietary patterns can lead to epigenetic modifications that alter nitric oxide synthesis.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "DIETARY PATTERN", "target": "EPIGENETIC MODIFICATION"}, {"weight": 10.0, "description": "Certain dietary patterns may improve or impair nutrient absorption and overall health.)<SEP>Individuals with specific dietary patterns may exhibit altered levels of certain nutrients that could influence their GRS scores and cognitive test performances.)<SEP>Specific nutrients may correlate with certain dietary patterns that impact health outcomes.)", "source_id": "chunk-72f4505e43f67f3564607392cb178656<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7<SEP>chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "DIETARY PATTERN", "target": "NUTRIENT"}, {"weight": 2.0, "description": "Genetic factors can influence how dietary patterns affect physiological processes like TAG levels.)", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "order": 1, "source": "DIETARY PATTERN", "target": "GENOTYPE"}, {"weight": 1.689, "description": "SNPs in this gene showed a slight risk reduction for ovarian cancer, possibly influenced by dietary patterns.)", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "order": 1, "source": "DIETARY PATTERN", "target": "NOS3 RS743507"}, {"weight": 2.0, "description": "Only 9.7% of infants were born to mothers whose dietary choline intake during pregnancy was adequate (450 mg/day))", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "DIETARY PATTERN", "target": "INFANTS"}, {"weight": 1.0, "description": "NA, no disease-dietary pattern relationship explicitly mentioned)", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "order": 1, "source": "DIETARY PATTERN", "target": "DISEASE"}, {"weight": 5.0, "description": "Domestically acquired infections were associated with eating fresh poultry other than chicken and turkey.)", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "order": 1, "source": "DIETARY PATTERN", "target": "DOMESTICALLY ACQUIRED INFECTIONS"}, {"weight": 6.0, "description": "Specific dietary patterns may influence gene expression through molecular interactions.)", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea", "order": 1, "source": "DIETARY PATTERN", "target": "MOLECULAR INTERACTION"}, {"weight": 4.0, "description": "The NOS1 gene interacts with folate metabolism to regulate nitric oxide production in cardiovascular health.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "FOLATE METABOLISM", "target": "NOS1 GENE"}, {"weight": 7.0, "description": "Alterations in the nitric oxide synthesis pathway are associated with cardiovascular disease.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "NITRIC OXIDE SYNTHESIS", "target": "CARDIOVASCULAR DISEASE"}, {"weight": 15.0, "description": "Statin therapy is used to manage risk factors and improve outcomes in cardiovascular disease patients.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "CARDIOVASCULAR DISEASE", "target": "STATINS THERAPY"}, {"weight": 0.0, "description": "Individuals carrying the rs2231142 T allele may be predisposed to higher risks for certain types of cardiovascular diseases, as indicated by their uric acid levels and associated biomarkers.)", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "order": 1, "source": "CARDIOVASCULAR DISEASE", "target": "RS2231142 T ALLELE CARRIERS"}, {"weight": 8.0, "description": "Dietary intake can influence physical activity levels and vice versa, affecting overall health.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "DIET", "target": "PA EXPOSURE"}, {"weight": 9.0, "description": "Increased physical activity can reduce the risk of developing cardiovascular disease.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "PHYSICAL ACTIVITY", "target": "LIFESTYLE FACTOR"}, {"weight": 7.0, "description": "Education status might correlate with physical activity levels, which in turn affects cortisol levels influenced by alcohol intake.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "PHYSICAL ACTIVITY", "target": "EDUCATION STATUS"}, {"weight": 8.0, "description": "Physical activity could modify the impact of smoking status on cortisol levels through lifestyle and health behavior.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "PHYSICAL ACTIVITY", "target": "SMOKING STATUS"}, {"weight": 10.0, "description": "Statin medications help lower cholesterol levels and reduce the risk of heart disease.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "STATINS", "target": "DRUG THERAPY"}, {"weight": 11.0, "description": "Personalized therapies consider genetic variations when planning treatment for cardiovascular conditions.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "PERSONALIZED THERAPY", "target": "GENETIC PROFILES"}, {"weight": 3.0, "description": "All RCTs involve lifestyle interventions that aim to reduce weight and lower T2D risk, with variations in the methods and composition of these interventions.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "CHARACTERISTICS OF RCTS/STUDY DESIGN", "target": "LIFESTYLE INTERVENTION"}, {"weight": 4.0, "description": "The lifestyle intervention led to the largest decrease in proinsulin levels at 1 year among the different treatment groups.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "LIFESTYLE INTERVENTION", "target": "DRUG"}, {"weight": 8.0, "description": "The lifestyle group had a larger decrease in proinsulin levels than either metformin or troglitazone groups, indicating potential superiority.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "LIFESTYLE INTERVENTION", "target": "COMPARISON WITH OTHER INTERVENTIONS"}, {"weight": 4.0, "description": "In most studies, participants are over 40 years old at follow-up.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "AGE OF PARTICIPANTS", "target": "CHARACTERISTICS OF COHORT STUDIES/FOLLOW-UP TIME"}, {"weight": 5.0, "description": "The risk of cardiovascular disease varies among the different cohorts and RCT populations.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "CARDIOVASCULAR RISK", "target": "CHARACTERISTICS OF RCTS/BASELINE RISK"}, {"weight": 6.0, "description": "The dietary assessments are based on validated food frequency questionnaires or other methods like dietary history records or biomarker measurements, affecting the accuracy of findings.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "DIETARY EXPOSURE/ASSESSMENT METHOD", "target": "FOOD FREQUENCY QUESTIONNAIRE/"}, {"weight": 7.0, "description": "The physical activity levels were assessed using interviews or questionnaires which can introduce subjectivity and potential recall biases.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "INTERVIEWS/QUESTIONNAIRES", "target": "PA EXPOSURE/ASSESSMENT METHOD"}, {"weight": 1.0, "description": "Both groups showed heavy consumption, but frequency reduced during pregnancy, indicating potential effects on fetal development and infant growth.)(entity", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "order": 1, "source": "LIFESTYLE FACTOR", "target": "ALCOHOL CONSUMPTION DURING PREGNANCY AND CIGARETTE CONSUMPTION DURING PREGNANCY, ILLICIT DRUG USE DURING PREGNANCY"}, {"weight": 7.0, "description": "The efficacy of CAR T cells can be affected by lifestyle factors like diet and physical activity, which modulate the tumor microenvironment.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "LIFESTYLE FACTOR", "target": "CAR T CELL EFFICACY"}, {"weight": 6.0, "description": "Commonly used anti-emetics included palonosetron, dexamethasone, prochlorperazine, ondansetron, and fosaprepitant.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "DRUG THERAPY", "target": "ANTI-EMETIC MEDICATIONS AND TREATMENT"}, {"weight": 11.0, "description": "This specific SNP, part of the TCF7L2 gene, was associated with another one of the 23 obesity SNPs that showed significant association with BMI at baseline.)<SEP>rs7903146 is a variant in the TCF7L2 gene that has an increased association with type 2 diabetes.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43<SEP>chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "RS7903146", "target": "TCF7L2"}, {"weight": 3.0, "description": "The FTO gene is associated with one of these obesity SNPs that have a direction of effect consistent with published GWAS data, showing significant association with BMI at baseline.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "RS7903146", "target": "RS9939609"}, {"weight": 9.0, "description": "rs10010131 is associated with WFS1, which shows directionally consistent risk for type 2 diabetes.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "RS10010131", "target": "WFS1"}, {"weight": 10.0, "description": "rs10946398 associates with CDKAL1 gene showing a directionally consistent association in relation to type 2 diabetes.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "RS10946398", "target": "CDKAL1"}, {"weight": 11.0, "description": "rs864745 variant is associated with JAZF1 gene which has an increased risk for type 2 diabetes.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "RS864745", "target": "JAZF1"}, {"weight": 25.0, "description": "The rs4607103 variant associated with ADAMTS9 exhibits a differing directionality of association when compared to findings from European studies on type 2 diabetes.)<SEP>rs4607103 in ADAMTS9 shows association but opposite directionality compared to European data on type 2 diabetes.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "RS4607103", "target": "ADAMTS9"}, {"weight": 1.0, "description": "The association between the TCF7L2 risk variant and the nominal analysis was significant (P = 0.011, unweighted OR 1.04 [95% CI 1.01\u20131.08]).)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "TCF7L2 RISK VARIANT RS7903146", "target": "NOMINAL ANALYSIS RESULTS"}, {"weight": 2.0, "description": "The association between the TCF7L2 risk variant and the weighted analysis was also significant (P = 8.10 \u00d7 10\u22125, weighted OR 1.06 [1.03\u20131.10]).)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "TCF7L2 RISK VARIANT RS7903146", "target": "WEIGHTED ANALYSIS RESULTS"}, {"weight": 3.0, "description": "The nominal analysis of the TCF7L2 risk variant did not show association with ordinal data (P = 0.33, unweighted OR 1.02 [0.98\u20131.05]).)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "TCF7L2 RISK VARIANT RS7903146", "target": "ORDINAL ANALYSIS RESULTS (NOMINAL CATEGORIES)"}, {"weight": 4.0, "description": "The weighted ordinal analysis of the TCF7L2 risk variant also did not show association with ordinal data (P = 0.40, weighted OR 1.02 [0.98\u20131.06]).)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "TCF7L2 RISK VARIANT RS7903146", "target": "ORDINAL ANALYSIS RESULTS (WEIGHTED)"}, {"weight": 5.0, "description": "The analyses did not find evidence for interaction with African Ancestry when considering the proportions of ancestry (P = 0.74).)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "TCF7L2 RISK VARIANT RS7903146", "target": "AFRICAN ANCESTRY"}, {"weight": 9.0, "description": "The count of risk alleles for the TCF7L2 risk variant was considered in nominal and ordinal analyses.)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "TCF7L2 RISK VARIANT RS7903146", "target": "COUNT OF RISK ALLELES"}, {"weight": 10.0, "description": "The nominal analysis results showed a significant association (P = 0.011, unweighted OR 1.04 [95% CI 1.01\u20131.08]) for the TCF7L2 risk variant.)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "NOMINAL ANALYSIS RESULTS", "target": "SIGNIFICANCE"}, {"weight": 11.0, "description": "The weighted analysis results showed a more significant association (P = 8.10 \u00d7 10\u22125, weighted OR 1.06 [1.03\u20131.10]) for the TCF7L2 risk variant.)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "WEIGHTED ANALYSIS RESULTS", "target": "SIGNIFICANCE"}, {"weight": 12.0, "description": "The nominal ordinal analysis results did not show a significant association between the TCF7L2 risk variant and ordinal data (P = 0.33, unweighted OR 1.02 [0.98\u20131.05]).)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "ORDINAL ANALYSIS RESULTS (NOMINAL CATEGORIES)", "target": "ASSOCIATION"}, {"weight": 13.0, "description": "The weighted ordinal analysis results also did not show a significant association between the TCF7L2 risk variant and ordinal data (P = 0.40, weighted OR 1.02 [0.98\u20131.06]).)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "ORDINAL ANALYSIS RESULTS (WEIGHTED)", "target": "ASSOCIATION"}, {"weight": 2.0, "description": "There is an observed interaction between one obesity SNP and a trait related to glycemic status on baseline BMI.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "INTERACTION", "target": "TRAIT INTERACTION"}, {"weight": 1.0, "description": "A significant interaction was found for 20:4, n-6 (AA) in the colon between genotype and diet.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "INTERACTION", "target": "GENETIC VARIANT AND DIET INTERVENTION"}, {"weight": 2.0, "description": "The six outcomes differed between the choline group and placebo group based on maternal PEMT alleles)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "INTERACTION", "target": "CHOLINE GROUP BY MATERNAL PEMT ALLELE"}, {"weight": 1.0, "description": "cg18894781 associated with FRMD6 may suggest interaction in cell polarization, adhesion dynamics, or motility)(entity", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "order": 1, "source": "INTERACTION", "target": "FRMD6"}, {"weight": 6.0, "description": "SNP analysis through regression highlighted correlations with several hematological values before and after hydroxyurea treatment.)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "ASSOCIATION", "target": "HEMATOLOGICAL PARAMETERS"}, {"weight": 8.0, "description": "The association of these BCL11A SNPs included changes in hemoglobin levels following hydroxyurea exposure.)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "ASSOCIATION", "target": "CHANGE MEASURE"}, {"weight": 2.0, "description": "The CC genotype is associated with an increased risk of developing Colorectal Cancer based on statistical significance (p=0.009).)", "source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "order": 1, "source": "ASSOCIATION", "target": "CC GENOTYPE"}, {"weight": 10.0, "description": "Associations were sought between specific TLR7 polymorphisms and HIV-infected status versus health in the study population.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "ASSOCIATION", "target": "HIV-INFECTED INDIVIDUALS VS. HEALTHY CONTROLS"}, {"weight": 3.0, "description": "These two genetic variants are among the top three that reached significant associations with BMI at baseline.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "RS9939609", "target": "RS2984618"}, {"weight": 5.0, "description": "The association between one of the 23 obesity SNPs and BMI at baseline has direction consistent with previous Genome-Wide Association Study data.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "RS9939609", "target": "FTO"}, {"weight": 1.0, "description": "mediated pleiotropy/Mediation", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "FTO", "target": "CRP/"}, {"weight": 3.0, "description": "The TAL1 gene is linked to this SNP that reached significance in its association with BMI at baseline.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "RS2984618", "target": "TAL1"}, {"weight": 3.0, "description": "This SNP in the LEPR gene has significant interaction on BMI level when combined with FPG levels, indicating a specific pathway relating obesity genetics to glycemic status.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "FPG", "target": "RS1011527"}, {"weight": 3.0, "description": "The SNP\'s interaction with fasting plasma glucose (FPG) levels affects BMI at baseline.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "RS1011527", "target": "INTERACTION WITH FPG"}, {"weight": 3.0, "description": "The SNP\'s combined interaction with FPG levels impacts baseline BMI.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "RS1011527", "target": "INTERACTION WITH FPG AND BASELINE BMI"}, {"weight": 3.0, "description": "This interaction between SNP, FPG levels, and baseline BMI showcases a complex relationship in influencing obesity markers.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "RS1011527", "target": "FPG AND BASELINE BMI INTERACTION"}, {"weight": 3.0, "description": "This combination highlights a specific pathway by which obesity SNPs interact with metabolic markers to affect BMI at baseline and its transition.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "RS1011527", "target": "INTERACTION WITH FPG AND BASELINE BMI LEVEL"}, {"weight": 3.0, "description": "The SNP\'s involvement with FPG levels and baseline BMI demonstrates its impact on obesity markers through complex interplay.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "RS1011527", "target": "FPG AND BASELINE BMI INTERACTION EFFECT"}, {"weight": 0.06, "description": "The GRS was associated with lower scores on TMTA (Trail Making Test A), suggesting a potential link between genetic risk and cognitive function.)", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "order": 1, "source": "GRS", "target": "TMTA"}, {"weight": 0.002, "description": "The GRS correlated significantly with reduced performance on TMTB, indicating an impact of genetic predispositions on attention measures.)", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "order": 1, "source": "GRS", "target": "TMTB"}, {"weight": 1.0, "description": "No significant difference was observed between the GRS groups for PEGDH, suggesting a lack of association in specific metabolic processes.)", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "order": 1, "source": "GRS", "target": "PEGDH"}, {"weight": 1.0, "description": "An analogous relationship to PEGDH, indicating no observable link between the Genetic Risk Score and this other metabolite interaction.)", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "order": 1, "source": "GRS", "target": "PEGNDH"}, {"weight": 1.0, "description": "Similar observations for DIG: GRS groups did not show significant differences in mean scores, suggesting minimal impact on cognitive measures related to its function.)", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "order": 1, "source": "GRS", "target": "DIG"}, {"weight": 1.0, "description": "No association was found between the Genetic Risk Score and impairment on CFL or other specific tests within the DIVERGT battery.)", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "order": 1, "source": "GRS", "target": "CFL"}, {"weight": 3.0, "description": "The interaction between these two SNPs with a trait related to glycemic status influences BMI at baseline.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "TRAIT INTERACTION", "target": "RS1011527, RS9939609"}, {"weight": 3.0, "description": "This SNP within the LEPR gene interacts significantly with FPG levels affecting baseline BMI.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "FPG, BASELINE BMI", "target": "RS1011527, LEPR GENE"}, {"weight": 2.0, "description": "The genetic factors are not associated with transitions in BMI, suggesting no simultaneous main effect and interaction between genetic determinants and glycemic status change.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "BMI TRANSITION", "target": "SNP(S), GRS"}, {"weight": 3.0, "description": "The SNP\'s interaction with fasting plasma glucose (FPG) directly affects baseline BMI.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "BASELINE BMI INFLUENCE", "target": "RS1011527, FPG"}, {"weight": 2.0, "description": "Two polymorphisms (MTHFR 1298A>C and MS 2756A>G) were related to performance on TMTA and TMTB.)", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "order": 1, "source": "MTHFR 1298A>C", "target": "MS 2756A>G"}, {"weight": 4.0, "description": "Survivors with the MS 2756AA genotype scored significantly lower than survivors with other genotypes on TMTB.)", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "order": 1, "source": "MS 2756A>G", "target": "TMTB"}, {"weight": 3.0, "description": "Survivors with the MTHFR 1298AC/CC genotypes performed worse than those with the MTHFR 1298AA genotype on both tests.)", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "order": 1, "source": "TMTA", "target": "TMTB"}, {"weight": 5.0, "description": "The mean PEGNDH score of survivors with the MS 2756AA genotype was significantly lower than those with different genotypes.)", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "order": 1, "source": "PEGDH", "target": "PEGNDH"}, {"weight": 1.0, "description": "Age ranged from birth to seven years old for patients with CHI diagnosis.)", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "order": 1, "source": "AGE AT DIAGNOSIS", "target": "CHI"}, {"weight": 5.0, "description": "The duration from symptom manifestation to diagnosis was longer in Group 2, particularly for neonatal form patients.)", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "order": 1, "source": "AGE AT DIAGNOSIS", "target": "TIME DURATION BETWEEN SYMPTOM AND DIAGNOSIS"}, {"weight": 3.0, "description": "High GRS scores might indicate an overexpression or altered function of proteins that play key roles in cognitive processes related to MTX neurotoxicity.]", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "order": 1, "source": "PROTEIN", "target": "DISEASE"}, {"weight": 1.0, "description": "SENP1 deletion augments NEMO SUMOylation in adipocytes, which is thought to regulate the activation of NF-\u03baB and cytokine production.)", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "order": 1, "source": "PROTEIN", "target": "SENP1"}, {"weight": 1.0, "description": "NA, no protein-nutrient relationship mentioned)", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "order": 1, "source": "PROTEIN", "target": "NUTRIENT"}, {"weight": 1.0, "description": "SUMOylation of NEMO by SENP1 modulates the activation of NF-\u03baB and cytokine production in adipocytes, key aspects of inflammatory response linked to T1DM pathophysiology.)", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "order": 1, "source": "PROTEIN", "target": "INFLAMMATORY PROCESS"}, {"weight": 3.0, "description": "Exposure to environmental factors can modify the function and stability of proteins.)", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea", "order": 1, "source": "PROTEIN", "target": "ENVIRONMENTAL FACTOR"}, {"weight": 7.0, "description": "Maternal periconceptional use of folic acid supplements was higher in non-Hispanic Whites than in Hispanic mothers.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "NUTRIENT", "target": "FOLIC ACID SUPPLEMENTS"}, {"weight": 3.0, "description": "Certain nutrients might interact with environmental factors to influence health outcomes.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "NUTRIENT", "target": "ENVIRONMENTAL FACTOR"}, {"weight": 0.005, "description": "The A114S missense change in this SNP was linked to a reduced risk for certain ovarian cancer subtypes.)", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "order": 1, "source": "MOLECULAR INTERACTION", "target": "IL1A RS17561"}, {"weight": 6.0, "description": "The complex interactions between different proteins like IL-6R\u03b1 and its ligand(s), which are crucial for processes including immune response regulation and disease development.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "MOLECULAR INTERACTION", "target": "PROTEIN-PROTEIN INTERACTION"}, {"weight": 5.0, "description": "Both the IL-1R and IL-6R\u03b1 genes interact with other molecular components to modulate both Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 disease severity.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "MOLECULAR INTERACTION", "target": "GENE"}, {"weight": 2.0, "description": "The six outcomes on which the choline group and placebo group differed were examined)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "MOLECULAR INTERACTION", "target": "INFANTS"}, {"weight": 6.0, "description": "NA, no specific physiological process-molecular interaction mentioned)<SEP>Understanding how specific physiological processes are affected by molecular interactions is crucial for disease prevention and treatment.)", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce<SEP>chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "MOLECULAR INTERACTION", "target": "PHYSIOLOGICAL PROCESS"}, {"weight": 1.0, "description": "SENP1 deletion enhances NEMO SUMOylation even under stress conditions like DNA damage, which could contribute to inflammation and metabolic dysfunction in diabetes. )", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "order": 1, "source": "MOLECULAR INTERACTION", "target": "DNA DAMAGE RESPONSE"}, {"weight": 7.0, "description": "Together, they form part of the CYP2C9 star variant group indicating reduced activity.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "MOLECULAR INTERACTION", "target": "RS1799853 AND RS1057910"}, {"weight": 7.0, "description": "Changes in molecular interactions might lead to the development or progression of diseases, as mentioned.)", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "MOLECULAR INTERACTION", "target": "DISEASE"}, {"weight": 8.0, "description": "Reduction of binding affinity due to these mutations suggests a disruption in specific interactions that are key for FAAP20\'s function.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "MOLECULAR INTERACTION", "target": "FAAP20-UBZ MUTANTS D164A, A172L AND D177A"}, {"weight": 10.0, "description": "Mutations in FAAP20\'s UBZ domain affect its ability to interact with ubiquitin (Ub) leading to disruptions in cellular functions that can have implications for disease states.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "MOLECULAR INTERACTION", "target": "FAAP20-UBZ MUTANTS"}, {"weight": 14.0, "description": "The interaction between proteins like FAAP20, WRNIP1, RAD18, Pol\u03b7, NEMO and ubiquitin (Ub) through their UBZ domains is crucial for various cellular processes that play a role in disease states.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "MOLECULAR INTERACTION", "target": "UBZ-DOMAIN PROTEINS"}, {"weight": 4.0, "description": "These SNPs influence T-cell proliferation response when exposed to PHA or MAP PPD-like molecules.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "MOLECULAR INTERACTION", "target": "SNPS IN PTPN2 AND PTPN22"}, {"weight": 7.0, "description": "In the context of GWAS, causal SNPs may influence genetic outcomes depending on their LD score and MAF when evaluated using different statistical methods such as survival kernel machine test or weighted IBS kernel.)", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "order": 1, "source": "MOLECULAR INTERACTION", "target": "CAUSAL SNPS"}, {"weight": 4.0, "description": "Certain molecular interactions are key in metabolic pathways, affecting how they function or respond to stimuli.)", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea", "order": 1, "source": "MOLECULAR INTERACTION", "target": "METABOLIC PATHWAY"}, {"weight": 2.376, "description": "This genetic variant influences the odds of developing ovarian cancer.)", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "ALOX5 RS1864414"}, {"weight": 11.0, "description": "Environmental factors can contribute to the onset or exacerbation of diseases.)<SEP>The environment, including genetic factors such as specific gene variants, influences colitis through impacts on T cell responses.)", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "DISEASE"}, {"weight": 7.0, "description": "Differential effects of environmental factors on metabolic pathways can influence health outcomes.)<SEP>NA, no specific metabolic pathway-environmental factor relationship mentioned)", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce<SEP>chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "METABOLIC PATHWAY"}, {"weight": 1.0, "description": "Adipocyte-specific SENP1 deficiency may exacerbate the genetic predisposition towards T1DM by modulating multiple signaling pathways involved in insulin secretion and glucose metabolism. )", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "GENETIC PREDISPOSITION"}, {"weight": 3.0, "description": "The study investigated genetic variations that influence ChAT expression and function, leading to altered kinetic parameters and potential phenotypic outcomes.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "GENETIC INFLUENCE"}, {"weight": 7.0, "description": "Non-smokers showed marginally shorter latency to drink compared to smokers but did not significantly moderate medication or genotype effects.)<SEP>There was an association between smoking status and latency to drink, but it did not significantly moderate the effects of genotype or medication.)", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "SMOKING STATUS"}, {"weight": 12.0, "description": "The process of ubiquitination, influenced by proteins like FAAP20, WRNIP1, RAD18, Pol\u03b7 and NEMO through their UBZ domains, is an environmental factor that plays a significant role in cellular processes linked to health and disease.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "UBIQUITINATION"}, {"weight": 13.0, "description": "Mutations affecting FAAP20\'s UBZ domain can influence the cellular environment by disrupting interactions with ubiquitin (Ub), thereby impacting various biological processes and health outcomes.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "FAAP20-UBZ MUTANTS D164A, A172L, D177A"}, {"weight": 5.0, "description": "Alteration of T-cell response when specific pathogens are present in the environment, affecting physiological processes.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "MAPBACTEREMIA PRESENCE"}, {"weight": 7.0, "description": "Both polymorphisms in these genes were associated with performance on TMTA and TMTB.)", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "order": 1, "source": "MTHFR 677C>T", "target": "SHMT 1520C>T"}, {"weight": 8.0, "description": "Genotype comparison between MS 2756AA survivors and those with TSER 3R/3R or 2R/3R showed differences in certain neurocognitive outcomes.)", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "order": 1, "source": "MS 2756AA GENOTYPE", "target": "TSER 3R/3R GENOTYPES"}, {"weight": 4.0, "description": "Different subgroups within the population could have varying disease profiles based on environmental or genetic factors.)", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "order": 1, "source": "DISEASE", "target": "STRATA"}, {"weight": 2.0, "description": "19 SNPs out of the 283 independent cardiometabolic SNPs were associated with CRP after correction for multiple testing.)", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "order": 1, "source": "DISEASE", "target": "CRP GWAS SNPS"}, {"weight": 6.0, "description": "Additional SNPs out of 283 cardiometabolic SNPs were associated with CRP after correction for multiple testing.)", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "order": 1, "source": "DISEASE", "target": "SNPS IN CRP GWAS"}, {"weight": 2.0, "description": "Day-to-day variations influenced the S-FOL and WB-FOL results.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "DISEASE", "target": "DAY EFFECT"}, {"weight": 3.1532, "description": "The rs1695 gene variant showed an association with increased risk of developing breast cancer.)", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "order": 1, "source": "DISEASE", "target": "GENE"}, {"weight": 1.0, "description": "The presence of the minor allele may be associated with increased risk or severity of certain diseases.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "DISEASE", "target": "MINOR ALLELE"}, {"weight": 2.0, "description": "Elevated cholesterol levels are a significant risk factor for developing cardiovascular diseases, including heart attack and stroke.)", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "order": 1, "source": "DISEASE", "target": "CHOLESTEROL"}, {"weight": 2.0, "description": "High triglyceride levels increase the risk of developing conditions such as pancreatitis and cardiovascular disease.)", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "order": 1, "source": "DISEASE", "target": "TRIGLYCERIDES"}, {"weight": 4.0, "description": "European-American patients are part of the study population with HDDs under consideration.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "DISEASE", "target": "EA PATIENTS"}, {"weight": 2.0, "description": "The deficiency of wild type UBE2T results in Fanconi anemia-T subtype.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "DISEASE", "target": "WILD TYPE UBE2T"}, {"weight": 3.0, "description": "A previously associated SNP rs11209026 is mentioned with relation to paradoxical psoriasiform reactions and anti-TNFs treatment.)", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "DISEASE", "target": "RS11209026"}, {"weight": 5.0, "description": "Mutations in key residues necessary for Ub binding may lead to dysfunctions and potentially contribute to disease states.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "DISEASE", "target": "FAAP20-UBZ MUTANTS"}, {"weight": 11.0, "description": "Proteins with UBZ domains involved in ubiquitination pathways are implicated in the regulation of cell survival and division processes which can impact disease progression based on their interactions with ubiquitin (Ub).)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "DISEASE", "target": "FAAP20, WRNIP1, RAD18, POL\u0397, NEMO"}, {"weight": 0.15, "description": "Diseases related to the context and their interactions with other entities are yet to be determined.)", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "order": 1, "source": "DISEASE", "target": "DISEASE"}, {"weight": 3.0, "description": "The c.385G>T variant (p.Gly129*) is linked to agenesis of the corpus callosum)", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "order": 1, "source": "DISEASE", "target": "P.GLY129*"}, {"weight": 7.0, "description": "GWAS SNPs were associated with T1D through identified trans-acting regulatory effects and eQTLs.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "DISEASE", "target": "GWAS SNPS"}, {"weight": 1.0, "description": "The presence or identification of a known immunodominant epitope may indicate or contribute to specific diseases linked to antigenic presentation and recognition processes.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "DISEASE", "target": "KNOWN IMMUNODOMINANT EPITOPE"}, {"weight": 1.0, "description": "In the DPP cohort, the C risk allele at this genomic variant was associated with higher fasting proinsulin levels.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "SLC30A8 RS13266634", "target": "FASTING PROINSULIN LEVELS"}, {"weight": 3.0, "description": "After 1 year of DPP intervention, all active groups significantly decreased fasting proinsulin levels compared to placebo.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "FASTING PROINSULIN LEVELS", "target": "INTERVENTION GROUPS"}, {"weight": 7.0, "description": "Participants in the DPP cohort were randomized into intervention groups for 1 year.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "DPP COHORT", "target": "INTERVENTION"}, {"weight": 3.0, "description": "An intronic variant of MTNR1B was strongly linked to fasting glucose levels.)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "FASTING GLUCOSE", "target": "MTNR1B"}, {"weight": 5.0, "description": "At 1 year, there were no significant differences in proinsulin levels between the metformin and troglitazone intervention groups.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "PROINSULIN LEVELS", "target": "METFORMIN TREATMENT"}, {"weight": 9.0, "description": "At 1 year after DPP interventions, the metformin and troglitazone had similar proinsulin levels, suggesting no significant difference in effectiveness between them for this study.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "METFORMIN", "target": "COMPARISON WITH TROGLITAZONE"}, {"weight": 6.0, "description": "No genotype-specific effectiveness was observed for any of the treatments evaluated based on SLC30A8 rs13266634 dosage.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "GENOTYPE \u00d7 TREATMENT INTERACTION", "target": "STATISTICAL RESULT"}, {"weight": 10.0, "description": "The dosage of SLC30A8 risk alleles did not significantly modify the effectiveness of any treatment within the DPP cohort.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "GENOTYPE \u00d7 TREATMENT INTERACTION", "target": "STATISTICAL FINDING"}, {"weight": 1.0, "description": "HC diet increases the expression of PNPLA3 mRNA, which is inhibited with the addition of lipid emulsion (LE).)", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "order": 1, "source": "HC DIET", "target": "PNPLA3"}, {"weight": 5.0, "description": "Correlation between SREBP1c and ACC-1 mRNA levels in mouse liver tissues; strong positive correlation observed.)<SEP>SREBP1c and ACC-1 show similar expression patterns to PNPLA3; high correlations were observed between their mRNA levels.)", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "order": 1, "source": "SREBP1C", "target": "ACC-1"}, {"weight": 4.0, "description": "The same treatment also had no effect on ACC-1 mRNA expression under either condition.)", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "order": 1, "source": "ACC-1", "target": "INSULIN"}, {"weight": 6.0, "description": "Treatment with glucagon also had no effect on ACC-1 mRNA expression in HepG2 cells, but increased PC expression.)", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "order": 1, "source": "ACC-1", "target": "GLUCAGON"}, {"weight": 1.0, "description": "Our data suggests that hepatic PNPLA3 does not respond to insulin stimulation in HepG2 cells.)", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "order": 1, "source": "INSULIN", "target": "HEPG2 CELLS"}, {"weight": 3.0, "description": "Incubation of HepG2 cells with 10 nM insulin for 24 h did not increase the expression of FAS, but induced the expression of pyruvate carboxylase (PC), a known target of insulin action.)", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "order": 1, "source": "INSULIN", "target": "FAS"}, {"weight": 2.0, "description": "Treatment of HepG2 cells with glucagon had no effect on PNPLA3 mRNA expression, but stimulated pyruvate carboxylase (PC), a known target of glucagon action.)", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "order": 1, "source": "GLUCAGON", "target": "HEPG2 CELLS"}, {"weight": 5.0, "description": "Treatment of HepG2 cells with 100 nM glucagon in low glucose medium for 24 hours did not affect FAS gene expression, but stimulated the expression of pyruvate carboxylase (PC), a known target of glucagon action.)", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "order": 1, "source": "GLUCAGON", "target": "FAS"}, {"weight": 3.0, "description": "Different fatty acids tested had varying effects on PNPLA3 gene expression in HepG2 cells under low and high glucose conditions.)", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "order": 1, "source": "FATTY ACIDS", "target": "HEPG2 CELLS"}, {"weight": 1.0, "description": "Protein", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "order": 1, "source": "DEPLETION-REFEEDING CONDITIONS (SPECIFIC CONTEXT: HEPG2 CELL EXPERIMENTS)|PNPLA3 & CHREBP<|CYCLING BETWEEN GLUCOSE DEPLETION AND REFEEDING PHASES INFLUENCED PNPLA3 EXPRESSION, WITH MODERATE REDUCTIONS AFTER 8 HOURS OF DEPLETION; SHOWED REGULATION THROUGH TRANSCRIPTION FACTORS IN RESPONSE TO METABOLIC STATES.)(ENTITY", "target": "LDL RECEPTOR"}, {"weight": 2.0, "description": "The genes identified through combined mRNA/miRNA analyses were mainly involved in neurodevelopment pathways.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "order": 1, "source": "GENE", "target": "NEURODEVELOPMENT"}, {"weight": 3.0, "description": "The top hits were located within or close to various genes like APOC1, HNF1A, GCKR etc.)", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "order": 1, "source": "GENE", "target": "SNPS IN CRP GWAS"}, {"weight": 10.0, "description": "There was no evidence of a nonrandom association between NTD-affected pregnancies and either maternal or infant variants in DHFR.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "GENE", "target": "DHFR SNPS"}, {"weight": 10.0, "description": "This set of SNPs collectively represents the CYP2C9 star variant group that was considered.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "GENE", "target": "SNP SET COMBINING RS1799853, RS1057910, RS28371686, RS7900194, RS28371685, CYP2C9*2, CYP2C9*3, CYP2C9*5, CYP2C9*8 AND CYP2C9*11"}, {"weight": 1.0, "description": "The OPRM1 gene was associated with differences in the AUDIT (Alcohol Use Disorders Identification Test) score across genotype groups.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "AUDIT SCORE"}, {"weight": 1.0, "description": "There were significant differences in the number of drinking days among OPRM1 genotype groups.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "NUMBER OF DRINKING DAYS"}, {"weight": 1.0, "description": "Participants with different OPRM1 genotypes had varying drinks consumed on drinking days.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "DRINKS PER DRINKING DAY"}, {"weight": 1.0, "description": "The effects of OPRM1 genotype on alcohol consumption were assessed after controlling for ALDH2 genetic variations.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "ALDH2"}, {"weight": 1.0, "description": "The impact of OPRM1 genotype on alcohol responses was adjusted with ADH1B genetic information in the analysis.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "ADH1B"}, {"weight": 2.0, "description": "OPRM1 genotype was compared against different medications to understand its effect on alcohol-induced responses.)<SEP>The actions of the OPRM1 gene on alcohol-related behaviors were examined using different medications.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "MEDICATION"}, {"weight": 3.0, "description": "The combined effects of different medications with OPRM1 gene variations on alcohol craving were studied.)<SEP>The combined effects of various medications with OPRM1 gene variations on assessing alcohol-induced sedative responses were studied.)<SEP>The interaction between a drug and OPRM1 genetic variation impacted the observed effects on different alcohol-induced responses.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "MEDICATION \u00d7 GENOTYPE INTERACTION"}, {"weight": 3.0, "description": "The OPRM1 gene\'s influence on alcohol responses was adjusted for BrAC levels during assessments.)<SEP>The influence of BrAC levels on alcohol craving assessment was explored in relation to OPRM1 genotypes and medications.)<SEP>The influence of BrAC levels on assessing alcohol-induced sedation was analyzed in relation to OPRM1 genotypes and medications.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "BRAC"}, {"weight": 3.0, "description": "The combined effect of medications, OPRM1 genotypes, and BrAC in influencing alcohol-induced subjective responses was analyzed.)<SEP>The complex interactions between medications, OPRM1 gene variants, and BrAC influenced evaluations related to alcohol-induced sedative responses.)<SEP>The complex interactions between medications, OPRM1 gene variants, and BrAC levels influenced evaluations related to alcohol cravings.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "MEDICATION \u00d7 GENOTYPE \u00d7 BRAC INTERACTION"}, {"weight": 1.0, "description": "The impact of OPRM1 genotype in conjunction with medication influenced alcohol-induced responses was assessed by considering current cigarette smoking status.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "CIGARETTE SMOKING STATUS"}, {"weight": 1.0, "description": "The influence of OPRM1 gene in combination with medications on responses related to alcohol intake was evaluated after taking into account nicotine smoking status.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "NICOTINE SMOKING STATUS"}, {"weight": 1.0, "description": "The influence of OPRM1 gene variations and naltrexone were studied to understand their mutual impact on alcohol effects.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "NALTREXONE"}, {"weight": 1.0, "description": "The role of the OPRM1 gene on alcohol responses under placebo conditions was analyzed during studies employing various medications.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "PLACEBO"}, {"weight": 1.0, "description": "The impact of the OPRM1 gene on alcohol craving was evaluated in relation to medication types and BrAC levels.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "ALCOHOL CRAVING"}, {"weight": 1.0, "description": "The OPRM1 gene\'s effect on alcohol-induced sedation was compared across different medication types and BrAC levels.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "ALCOHOL-INDUCED SEDATION"}, {"weight": 1.0, "description": "The OPRM1 gene\'s effect on alcohol-induced stimulant responses was studied considering different medication types and BrAC levels.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "ALCOHOL-INDUCED STIMULATION/"}, {"weight": 1.0, "description": "Stimulation/>The combined effects of various medications with OPRM1 gene variations on assessing alcohol-induced stimulant responses were explored.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "MEDICATION \u00d7 GENOTYPE INTERACTION/"}, {"weight": 1.0, "description": "Stimulation/>The complex interactions between medications, OPRM1 gene variants, and BrAC influenced evaluations related to alcohol-induced stimulant responses.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE", "target": "MEDICATION \u00d7 GENOTYPE \u00d7 BRAC INTERACTION/"}, {"weight": 3.0, "description": "Out of the 68 miRNAs, 65 were downregulated while only 3 were upregulated after prenatal stress exposure.)", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "order": 1, "source": "MIRNAS", "target": "DOWNREGULATION/UPREGULATION"}, {"weight": 2.0, "description": "EPHA4 was found to have potential interactions with LDLRAP1 through modulation patterns, implying possible biological pathways involved in depression.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "EPHA4", "target": "LDLRAP1"}, {"weight": 3.0, "description": "ARID5B was observed in a single cluster with TGF-\u03b21, implying potential interactions or overlapping regulatory mechanisms.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "ARID5B", "target": "TGF-\u03921"}, {"weight": 5.0, "description": "ARRDC4 and HSF2 show similarities in modulation patterns across different conditions, suggesting they might interact or share regulatory roles.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "HSF2", "target": "ARRDC4"}, {"weight": 7.0, "description": "Homo sapiens stress factor 2 might interact with SAMD12 based on their modulation patterns and roles in complex disorders like depression.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "HSF2", "target": "SAMD12"}, {"weight": 4.0, "description": "FBXO32 might interact with LRRN3 based on their modulation patterns and roles in biological processes related to depression.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "LRRN3", "target": "FBXO32"}, {"weight": 6.0, "description": "Both A2M and FoxO1 were modulated similarly in animals and humans, suggesting they might interact or share signaling pathways related to depression.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "A2M", "target": "FOXO1"}, {"weight": 1.0, "description": "Both FoxO1 and TGF-\u03b21 were part of a cluster that showed interactions through physical interaction, co-expression or involvement in common pathways, indicating potential biological crosstalk related to depression.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "FOXO1", "target": "TGF-\u03921"}, {"weight": 8.0, "description": "FoxO1 shows significant GxE interactions across both cohorts when exposed to emotional stress, suggesting it plays a crucial role in the development of depressive symptoms in response to such stressors.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "FOXO1", "target": "GXE INTERACTION WITH EMOTIONAL STRESS"}, {"weight": 9.0, "description": "FoxO1\'s gene expression is significantly affected by environmental exposures like emotional and physical abuse, indicating its role in the development of depressive symptoms through GxE interactions.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "FOXO1", "target": "GXE INTERACTION WITH EMOTIONAL ABUSE AND SEXUAL ABUSE"}, {"weight": 6.0, "description": "The cluster formed by A2M, FoxO1 and TGF-\u03b21 indicates potential interactions that are involved in depression through overlapping pathways and signaling systems.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "FOXO1", "target": "CLUSTER OF INTERACTING GENES (A2M, FOXO1, TGF-\u03921)"}, {"weight": 3.0, "description": "Activation of FoxO1 in response to cortisol exposure, reflecting a mechanism involved in the negative impact of stress on neuronal proliferation.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "FOXO1", "target": "STRESS"}, {"weight": 3.0, "description": "FoxO1 is part of a molecular pathway that regulates the effect of stress on neurogenesis through its activation by cortisol exposure.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "FOXO1", "target": "STRESS RESPONSE MECHANISM"}, {"weight": 1.0, "description": "The Genome-Wide association studies identified specific SNPs significantly associated with snoring.)", "source_id": "chunk-a34b927ed42ff0917960122b4d203f83", "order": 1, "source": "SNP", "target": "GWAS"}, {"weight": 11.0, "description": "Infants with the CT genotype had an approximate 1.5-fold increase risk compared to those with the CC genotype based on folic acid intake categories.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "SNP", "target": "INFANT MTHFD1 SNPS RS2236224 AND RS2236225"}, {"weight": 12.0, "description": "Infants with the AG genotype had a four-fold increase risk compared to those with GG genotype when maternal folate intake was low.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "SNP", "target": "INFANT AG GENOTYPE IN MTHFD1 SNP RS11627387"}, {"weight": 13.0, "description": "Mothers had a decreased risk of an NTD-affected pregnancy when they had low intake of folates compared to common homozygous mothers in the high folate intake group.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "SNP", "target": "MTHFR SNPS RS1476413, RS1801131 AND RS1801133"}, {"weight": 14.0, "description": "A maternal gene-only effect was associated with a reduced risk of an NTD-affected pregnancy compared to mothers with CC genotype.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "SNP", "target": "SHMT1 SNP RS669340 CG GENOTYPE"}, {"weight": 15.0, "description": "An interaction effect was observed for infants with low folate intake and this AG genotype, which showed an association with increased risk compared to those with AA or CC genotypes.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "SNP", "target": "INFANTS WITH THE SHMT1 SNP RS12939757 AG GENOTYPE"}, {"weight": 16.0, "description": "A gene-folate interaction was associated with a reduced risk of NTDs for mothers in low folate intake groups, compared to high-folate intake groups.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "SNP", "target": "INFANTS WITH THE TYMS SNPS RS502396 OR RS699517 HOMOZGOUS GENOTYPE"}, {"weight": 1.0, "description": "An interaction effect on infants was also protective against risk for this SNP when they were in a low folate intake group, relative to those in high-folate intake groups.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "SNP", "target": "INFANTS WITH THE TYMS SNP RS2847153 AG GENOTYPE"}, {"weight": 1.0, "description": "Identified genetic variations associated with coffee consumption through GWAS.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "SNP", "target": "GENETIC POLYMORPHISMS"}, {"weight": 1.0, "description": "There\'s an inverse relationship between the minor allele of SNPs and the risk of developing Type 2 Diabetes.)", "source_id": "chunk-b803830d07bff8ae14550f6458048707", "order": 1, "source": "SNP", "target": "TYPE 2 DIABETES"}, {"weight": 1.0, "description": "Lead SNPs at the loci of interest have common effects on eGFR across ancestry groups, indicating shared genetic influences.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "SNP", "target": "EGFR"}, {"weight": 1.0, "description": "The TaqMan method was utilized to assess the association between SNPs and CRC.)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "SNP", "target": "TAQMAN"}, {"weight": 4.0, "description": "Elevated mRNA levels of SGK-1 following cortisol treatment; indicates involvement in glucocorticoid-induced changes related to neurogenesis.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "CORTISOL", "target": "SGK-1"}, {"weight": 4.0, "description": "Several pathways involved in neurodevelopment and inflammation were found to be similar between the PNS model and humans exposed to early life trauma.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "order": 1, "source": "PATHWAY IN ANIMALS EXPOSED TO PNS", "target": "PATHWAY IN HUMANS EXPOSED TO CHILDHOOD TRAUMA"}, {"weight": 5.0, "description": "22 genes were identified as common across both datasets (hippocampus of rats exposed to PNS and blood of adults exposed to childhood trauma). The probability that these genes didn\'t overlap by chance was extremely low, p-value = 1.8 \u00d7 10^-8.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "order": 1, "source": "COMMON GENES", "target": "INTEGRATION OF DATA"}, {"weight": 6.0, "description": "All 22 common genes were validated using real-time PCR both in the hippocampus of animals and in blood samples from humans exposed to childhood trauma. 16 out of these 22 genes showed consistent modulation direction, while 6 showed opposite modulation.", "source_id": "chunk-37088dc7c337929125b7fb31c9bf64cf", "order": 1, "source": "GENE EXPRESSION IN HIPPOCAMPUS", "target": "REAL-TIME PCR VALIDATION"}, {"weight": 4.0, "description": "AS1842856, as an inhibitor of FoxO1, is used to counteract the upregulation of FoxO1 and its effect on cortisol-induced reduction in neuronal proliferation.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "AS1842856 (FOXO1 INHIBITOR)", "target": "DRUG INTERACTION"}, {"weight": 3.0, "description": "Cortisol plays a role in regulating neurogenesis, particularly under stress conditions, influencing the expression of key genes such as FoxO1 and SGK-1.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "CORTISOL (STRESS HORMONE)", "target": "NEUROGENESIS REGULATION"}, {"weight": 1.0, "description": "SNP rs4771122 achieved a low Bonferroni corrected P-value (.05/34 = .0017) for the association with long-term weight loss.)", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "order": 1, "source": "RS4771122", "target": "BONFERRONI CORRECTED P-VALUE"}, {"weight": 1.0, "description": "Additional copies of the minor (G) allele were significantly associated with higher linear mixed model weights after adjusting for several variables, including pre-surgical BMI.)", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "order": 1, "source": "RS4771122", "target": "BETA ESTIMATES"}, {"weight": 1.0, "description": "The GCK gene plays a role in MODY and is associated with diabetes through genetic variants.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "GCK", "target": "MODY"}, {"weight": 2.0, "description": "Sanger sequencing serves as a diagnostic tool to test for GCK variants involved in MODY.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "SANGER SEQUENCING", "target": "DIAGNOSTIC METHOD FOR GCK"}, {"weight": 20.0, "description": "Sanger sequencing confirmed the presence of the mutation in the fibroblast cultures from each sibling.)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "SANGER SEQUENCING", "target": "CONFIRMATION OF MUTATION"}, {"weight": 3.0, "description": "A MODY gene panel includes Sanger sequencing and is used alongside other tools to diagnose MODY through testing for genetic variants.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "MODY GENE PANEL", "target": "COMPLEMENTING DIAGNOSTIC TOOL"}, {"weight": 4.0, "description": "Testing conducted at the University of Chicago contributes significantly (72%) to the dataset of probands with GCK variants, indicating its importance in genetic research and healthcare.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "UNIVERSITY OF CHICAGO", "target": "PROVIDER OF TESTING"}, {"weight": 5.0, "description": "A subset of probands with GCK variants has clinical information available for analysis, providing insights into the manifestation of MODY.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "CLINICAL INFORMATION", "target": "DATA SET FOR ANALYSIS"}, {"weight": 8.0, "description": "The variant was found through subsequent capillary sequencing and implies heteroplasmy.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "NOVEL VARIANT", "target": "3754C>A"}, {"weight": 6.0, "description": "ACMG guidelines are used to classify variants as pathogenic or likely pathogenic based on their impact on the GCK gene and its associated conditions.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "PATHOGENICITY CLASSIFICATION", "target": "CRITERIA FOR VARIANT EVALUATION"}, {"weight": 8.0, "description": "VUS are not clearly classified as pathogenic due to lack of evidence for their impact or segregation with the disease, requiring further research.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "VUS (VARIANT OF UNCERTAIN SIGNIFICANCE)", "target": "CLASSIFICATION CRITERIA"}, {"weight": 9.0, "description": "Certain variants identified in probands might arise spontaneously (de novo), especially those without family history like p.Leu307Phe, impacting classification and understanding of genetic inheritance patterns.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "DE NOVO MUTATION", "target": "GENETIC EVENT"}, {"weight": 10.0, "description": "Targeted sequencing enables detection of specific variants in probands\' DNA that might be missed with whole-genome approaches; crucial for identifying new or rare variants like p.Leu307Phe.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "TARGETED SEQUENCING", "target": "DIAGNOSTIC METHOD FOR VARIANTS"}, {"weight": 11.0, "description": "In cases where genetic data is ambiguous, paternity testing can provide definitive answers on the origin of certain variants; it was not performed in this scenario affecting variant classification.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "PATERNITY TESTING", "target": "GENETIC ANALYSIS FOR PARENTAL VERIFICATION"}, {"weight": 7.0, "description": "Segregation analysis establishes co-segregation between tested probands\' DNA and diabetes phenotype, highlighting genetic linkage in MODY cases.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "CO-SEGREGATION WITH DIABETES PHENOTYPE", "target": "GENETIC ANALYSIS"}, {"weight": 1.0, "description": "A higher adherence to the Mediterranean diet was observed to be significantly associated with lower TAG concentrations.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "TAG CONCENTRATIONS"}, {"weight": 2.0, "description": "Increased MD adherence was related to a lower apoB:apoA-1 ratio, which is an indicator of reduced cardiovascular risk.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "APOB:APOA-1 RATIO"}, {"weight": 3.0, "description": "Adherence to the Mediterranean diet was associated with higher HDL cholesterol levels, suggesting better lipid metabolism.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "HDL-C"}, {"weight": 4.0, "description": "Increased adherence to the Mediterranean diet led to an increase in apoA-I protein concentration, which is a component of HDL and is involved in lipid transport.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "APOA-1"}, {"weight": 5.0, "description": "An increased level of adherence to the MD was associated with lower HbA1c levels, indicating better blood glucose control.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "HBA1C"}, {"weight": 6.0, "description": "The Mediterranean diet adherence was inversely related to LDL cholesterol concentrations; people adhering more closely had lower LDL levels.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "LDL-C"}, {"weight": 7.0, "description": "Adherence to the Mediterranean diet showed an inverse association with apolipoprotein B concentration, potentially indicating reduced risk for cardiovascular disease.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "APOB"}, {"weight": 2.0, "description": "Lower MD adherence was inversely related to the risk of hypertriacylglycerolaemia.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "HYPERTRIACYLGLYCEROLAEMIA"}, {"weight": 1.0, "description": "The Mediterranean diet resulted in a non-significant change in colon fatty acid composition compared to baseline, specifically with regards to the concentration of 18:2 n-6 fatty acids.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "MEDITERRANEAN DIET", "target": "COLON 18:2 N-6 CONCENTRATION"}, {"weight": 1.0, "description": "The Mediterranean diet showed a non-significant impact on serum insulin levels after the intervention period compared to baseline and other diets.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "MEDITERRANEAN DIET", "target": "SERUM INSULIN LEVELS"}, {"weight": 1.0, "description": "The Mediterranean diet led to a decrease in serum low-density lipoprotein (LDL) cholesterol levels after the intervention period, indicating potential cardiovascular benefits from the dietary changes.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "MEDITERRANEAN DIET", "target": "SERUM LDL LEVELS"}, {"weight": 1.0, "description": "The association between APOC1 and HbA1C levels suggests a possible pleiotropic effect on metabolic traits.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "HBA1C", "target": "APOC1"}, {"weight": 3.0, "description": "The joint effect of specific genotypes (AA, GA, GG) with different levels of Mediterranean diet adherence influenced various metabolic markers like TAG, TC, apoB, and HbA1c.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "GENOTYPE AND MEDITERRANEAN DIET ADHERENCE", "target": "METABOLIC PARAMETERS"}, {"weight": 4.0, "description": "An analysis that examines how outcomes such as lipid profiles or glucose control vary across subgroups defined by age, activity level, or smoking status.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "SUBGROUP ANALYSIS", "target": "OUTCOME VARIABLES"}, {"weight": 0.5, "description": "Both polymorphisms showed significant association with PBC susceptibility for both Asians and Caucasians in subgroup analyses.)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "SUBGROUP ANALYSIS", "target": "RS231775 AND RS231725"}, {"weight": 1.0, "description": "There could be a difference in TAG levels based on smoking status, potentially influenced by genotypes.)", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "order": 1, "source": "SMOKING STATUS", "target": "TAG LEVELS"}, {"weight": 3.0, "description": "Smoking status may modify the effect of certain diets on health outcomes based on individual genetics.)", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "order": 1, "source": "SMOKING STATUS", "target": "GENE\u2013DIET INTERACTION"}, {"weight": 9.0, "description": "Smoking might influence cortisol levels independently or in conjunction with other factors like alcohol consumption.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "SMOKING STATUS", "target": "CORTISOL LEVELS"}, {"weight": 5.0, "description": "Genetic variations can influence how dietary choices impact health outcomes.)", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "order": 1, "source": "GENE\u2013DIET INTERACTION", "target": "GENOTYPE"}, {"weight": 2.0, "description": "There were 47 males and 18 females diagnosed with CHI.)", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "order": 1, "source": "CHI", "target": "SEX RATIO"}, {"weight": 2.0, "description": "Mutations in the GLUD1 gene were associated with patients experiencing HI (Hypoglycaemia Induced by Gluconeogenesis Dysfunction).)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "GLUD1 GENE", "target": "HI"}, {"weight": 3.0, "description": "The p.Arg322 His mutation was found among patients presenting symptoms of HI.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "HI", "target": "P.ARG322 HIS MUTATION"}, {"weight": 4.0, "description": "The p.Ser498Leu mutation contributed to the risk of developing HI in one patient.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "HI", "target": "P.SER498LEU MUTATION"}, {"weight": 5.0, "description": "This mutation was associated with patients experiencing HA (Hyperammonaemia).)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "HA", "target": "P.ASN463ASP MUTATION"}, {"weight": 3.0, "description": "Group 1 patients, which made up 92.3% of the neonatal form, showed no response to diazoxide treatment.)", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "order": 1, "source": "DIAZOXIDE-UNRESPONSIVE", "target": "CLINICAL SYMPTOM"}, {"weight": 4.0, "description": "Group 2 patients displayed a higher incidence of epilepsy compared with Group 1.)", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "order": 1, "source": "DIAZOXIDE-RESPONSIVE", "target": "SYMPTOM INCIDENCE"}, {"weight": 6.0, "description": "Symptoms included hypoglycemia and hypoglycaemic brain injury with global developmental delay.)", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "order": 1, "source": "HYPOGLYCEMIA", "target": "SYMPTOM PRESENTATION"}, {"weight": 6.0, "description": "Diazoxide treatment for HI/HA syndrome had side effects, including hypertrichosis.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "DIAZOXIDE TREATMENT", "target": "HI/HA SYNDROME"}, {"weight": 8.0, "description": "Patients with no response to diazoxide treatment required further interventions like insulin therapy or surgery.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "DIAZOXIDE-UNRESPONSIVE PATIENTS", "target": "THERAPY OUTCOME"}, {"weight": 5.0, "description": "A specific genetic variant in the NOS1 gene has been linked to an increased risk of developing cardiovascular disease.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "CARDIOVASCULAR DISEASE RISK", "target": "GENETIC VARIANT IN NOS1 GENE"}, {"weight": 6.0, "description": "Dietary patterns influence folate intake, which plays a role in the nitric oxide synthesis pathway affecting cardiovascular health.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "DIETARY PATTERNS", "target": "FOLATE INTAKE AND NITRIC OXIDE SYNTHESIS"}, {"weight": 6.0, "description": "Dietary factors can impact cardiovascular health through various molecular interactions.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "DIETARY PATTERNS", "target": "CARDIOVASCULAR HEALTH"}, {"weight": 13.0, "description": "Dietary choices can interact with genetic factors to impact health outcomes.)<SEP>Dietary factors can interact with genetic factors to impact health outcomes.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "DIETARY PATTERNS", "target": "GENETIC INFLUENCE"}, {"weight": 7.0, "description": "Environmental factors such as diet and physical activity modulate cardiovascular health through their effects on the NOS1 gene and folate metabolism.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "CARDIOVASCULAR HEALTH MODULATION", "target": "ENVIRONMENTAL FACTORS"}, {"weight": 1.0, "description": "Lifestyle is an environmental factor that can potentially alter the impact of other environmental conditions on AMD risk through its influence on complement pathway genes.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "ENVIRONMENTAL FACTORS", "target": "LIFESTYLE"}, {"weight": 1.0, "description": "Levels of AcAc increased significantly in the highest quintile of Matsuda ISI up to 29%.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "KBS", "target": "ACAC"}, {"weight": 1.0, "description": "Levels of BHB also increased significantly in the highest quintile of Matsuda ISI up to 41%, respectively.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "KBS", "target": "BHB"}, {"weight": 2.0, "description": "In a multivariate linear regression model including insulin sensitivity and insulin secretion as independent variables, insulin sensitivity remained inversely associated with AcAc levels and BHB levels.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "KBS", "target": "INSULIN SENSITIVITY"}, {"weight": 5.0, "description": "In individuals with diabetes mellitus, specific ketone body (KB) levels like AcAc and BHB may indicate metabolic changes and increase risk factors for hyperglycemia.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "KBS", "target": "DIABETES MELLITUS"}, {"weight": 3.0, "description": "Changes in AcAc (acetoacetate) level may also influence GLP-1 secretion through their impact on metabolic pathways related to glucose homeostasis.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "ACAC", "target": "GLP-1"}, {"weight": 2.0, "description": "The C allele of rs780094 showed a nominally significant association (P = 0.003) with AcAc levels.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "ACAC", "target": "RS780094"}, {"weight": 4.0, "description": "Alterations in BHB (beta-hydroxybutyrate) levels might affect SGLT2\'s function, impacting glucose reabsorption and blood sugar control.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "BHB", "target": "SGLT2"}, {"weight": 1.0, "description": "The glucose-increasing C allele of rs780094 was associated with a significant increase of BHB levels.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "BHB", "target": "RS780094"}, {"weight": 6.0, "description": "Area under the curve (AUC) of insulin represents total insulin response over time, reflecting metabolic processes involved in glucose regulation.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "INSULIN AUC", "target": "METABOLISM"}, {"weight": 8.0, "description": "The rate of insulin secretion might be influenced by GLP-1 levels through their mutual impact on glucose homeostasis and metabolic pathways.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "GLP-1", "target": "INSULIN SECRETION RATE"}, {"weight": 7.0, "description": "Changes in the Matsuda index could suggest alterations in insulin sensitivity and insulin resistance, impacting diabetes risk assessment.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "INSULIN RESISTANCE", "target": "MATSUDA INDEX"}, {"weight": 5.0, "description": "The block of SNPs are associated with insulin resistance leading to type 2 diabetes.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "INSULIN RESISTANCE", "target": "TYPE 2 DIABETES"}, {"weight": 9.0, "description": "The glucose-increasing C allele showed a nominally significant association (P = 0.003) with AcAc levels.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "RS780094", "target": "ACAC LEVELS"}, {"weight": 3.0, "description": "SNPs from the FADS1 gene were associated with KB levels, though not in a statistically significant manner.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "FADS1", "target": "KB LEVELS"}, {"weight": 4.0, "description": "SNPs from the ANK1 gene showed an association with KB levels that was nominally significant.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "ANK1", "target": "KB LEVELS"}, {"weight": 5.0, "description": "The GIPR gene was associated with KB levels, though not in a statistically significant manner.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "GIPR", "target": "KB LEVELS"}, {"weight": 6.0, "description": "The HMGA2 gene showed an association with KB levels that was nominally significant.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "HMGA2", "target": "KB LEVELS"}, {"weight": 2.0, "description": "SNP rs1351394 shows strong evidence of association with both \'Age at first tooth\' and \'Number of teeth\', linking height with tooth eruption and number.)", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "order": 1, "source": "HMGA2", "target": "RS1351394"}, {"weight": 7.0, "description": "SNPs from the SLC2A2 gene had a nominally significant association with KB levels.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "SLC2A2", "target": "KB LEVELS"}, {"weight": 1.0, "description": "SNP rs1260326 near GCKR gene affects CRP levels, and its association with CRP is robust even after adjustment for multiple testing>)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "GCKR", "target": "RS1260326"}, {"weight": 9.0, "description": "The GCKR gene influences blood glucose levels, which can affect the risk of developing kidney disease through metabolic stress.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "GCKR", "target": "BLOOD GLUCOSE"}, {"weight": 10.0, "description": "Adipose tissue mRNA expression of the CPT1A gene was positively correlated with glucose AUC, showing its role in fatty acid oxidation and insulin sensitivity.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "GLUCOSE AUC", "target": "CPT1A EXPRESSION"}, {"weight": 1.0, "description": "The expression of BDH1 and OXCT1 genes were positively correlated with insulin sensitivity (Matsuda ISI), highlighting their roles in metabolic processes.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "INSULIN SENSITIVITY", "target": "BDH1 AND OXCT1 EXPRESSION"}, {"weight": 8.0, "description": "The glucose-increasing C allele of rs780094 was associated with an increase of 5.6% in BHB levels per the C allele, after adjusting for age and BMI.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "BHB LEVELS", "target": "GCKR C ALLELE"}, {"weight": 13.0, "description": "The expression of the CPT1A gene was negatively correlated with glucose AUC but positively related to Matsuda ISI, suggesting roles in fatty acid oxidation and insulin sensitivity respectively.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "CPT1A EXPRESSION", "target": "GLUCOSE AUC AND INSULIN SENSITIVITY"}, {"weight": 11.0, "description": "The BDH1 and OXCT1 genes had positive correlations with Matsuda ISI, reflecting their roles in ketolysis and insulin sensitivity respectively.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "BDH1 AND OXCT1 EXPRESSION", "target": "MATSUDA ISI"}, {"weight": 12.0, "description": "The ACAT1 gene encoded acetyl-CoA acetyltransferase 1, which showed significant correlations with glucose AUC, Matsuda ISI and insulin secretion indicating its involvement in ketolysis.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "ACAT1 EXPRESSION", "target": "GLUCOSE AUC AND INSULIN SECRETION"}, {"weight": 14.0, "description": "The ACSS2 gene showed an inverse correlation with glucose AUC but a positive relationship with Matsuda ISI, pointing to its role in adipose tissue metabolism.)", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "order": 1, "source": "GLUCOSE AUC AND INSULIN SENSITIVITY", "target": "ACSS2 EXPRESSION"}, {"weight": 1.0, "description": "Three SNPs\' minor alleles led to an increase in the social disinhibition score, while three others had protective effects against it.)", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "order": 1, "source": "SNPS", "target": "INCREASED SOCIAL DISINHIBITION SCORE"}, {"weight": 2.0, "description": "The T-cell functionality and response were influenced by SNPs in PTPN2 and PTPN22.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "SNPS", "target": "CD"}, {"weight": 1.0, "description": "SNP APOC1 is associated with CRP levels in the context of its role in triglyceride metabolism>)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "APOC1", "target": "CRP"}, {"weight": 1.0, "description": "SNP rs1183910 near HNF1A gene influences CRP levels through pathways related to bile acid metabolism and inflammation>)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "HNF1A", "target": "CRP"}, {"weight": 1.0, "description": "Interleukin-6 receptor (IL6R) pathway could modulate CRP production or clearance, potentially influencing inflammation markers like CRP>)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "IL6R", "target": "CRP"}, {"weight": 1.0, "description": "SNP PABPC4 may influence triglyceride levels via its effect on lipid metabolism.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "PABPC4", "target": "TRIGLYCERIDES"}, {"weight": 1.0, "description": "SNP rs4660293 near PABPC4 gene influences triglyceride levels via its impact on lipid metabolism>)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "PABPC4", "target": "RS4660293"}, {"weight": 1.0, "description": "BCL7B\'s association with HDL cholesterol could be through its role in lipid metabolism.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "BCL7B", "target": "HDL-CHOLESTEROL"}, {"weight": 1.0, "description": "The effect of BCL7B on certain cardiometabolic phenotypes may be mediated through BMI>)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "BCL7B", "target": "MEDIATED PLEIOTROPY/MEDIATION"}, {"weight": 1.0, "description": "mediation/Mediation", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "BCL7B", "target": "LIPIDS/"}, {"weight": 1.0, "description": "mediated pleiotropy/Mediation", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "TMEM18", "target": "CRP/"}, {"weight": 1.0, "description": "SNP rs1183910 in the HNF1A gene is associated with CAD.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "CAD", "target": "RS1183910"}, {"weight": 0.034, "description": "rs20417 is significantly associated with explaining a part of the heterogeneity for CAD.)", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea", "order": 1, "source": "CAD", "target": "RS20417"}, {"weight": 0.012, "description": "rs1126643 contributes as a significant source of heterogeneity in relation to CAD.)", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea", "order": 1, "source": "CAD", "target": "RS1126643"}, {"weight": 1.0, "description": "SNP rs6734238 near IL1F10 and other interleukin 1 family genes affects total cholesterol levels.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "RS6734238", "target": "TOTAL CHOLESTEROL"}, {"weight": 1.0, "description": "SNP rs4420638 near the APOC1 gene influences T2D risk.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "T2D", "target": "RS4420638"}, {"weight": 0.25, "description": "After adjusting for age, sex, BMI, and glucose tolerance, there was no association between rs10399931 and T2D.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "T2D", "target": "RS10399931"}, {"weight": 1.0, "description": "After adjustment for age, sex, BMI, and glucose tolerance, none of the other tgSNPs were associated with T2D or IFG or IGT except rs12123883 for borderline protective effect.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "T2D", "target": "RS4950928"}, {"weight": 1.0, "description": "The two identified haplotype blocks of CHI3LI did not show any association with T2D.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "T2D", "target": "CHI3LI HAPLOTYPE BLOCKS"}, {"weight": 4.0, "description": "Type 2 Diabetes impacts kidney function through mechanisms such as diabetic nephropathy, influencing eGFR.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "T2D", "target": "KIDNEY FUNCTION"}, {"weight": 1.0, "description": "The association p-values for 283 cardiometabolic SNPs and serum CRP from a GWAS are evaluated.)", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "order": 1, "source": "CARDIOMETABOLIC SNPS", "target": "SNPS IN CRP GWAS"}, {"weight": 4.0, "description": "Some SNPs showed opposite effects on CRP levels and cardiometabolic phenotypes compared to observations in observational data.)", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "order": 1, "source": "CRP GWAS SNPS", "target": "SNPS WITH DIFFERENT EFFECT DIRECTIONS"}, {"weight": 5.0, "description": "The lead SNPs of the 12 pleiotropic loci were mostly in high linkage disequilibrium (LD) with those found in CRP GWAS.)", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "order": 1, "source": "CRP GWAS SNPS", "target": "GENOMIC LD"}, {"weight": 7.0, "description": "In some loci the lead SNPs from CRP and cardiometabolic GWAS were different but were in high LD.)", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "order": 1, "source": "PLEIOTROPIC LOCI", "target": "SNPS IN CRP GWAS AND CARDIOMETABOLIC GWAS"}, {"weight": 4.0, "description": "Intronic and 3\u2032UTR variants in the APOA5-ZNF259 region are related to triglyceride levels.)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "TRIGLYCERIDES", "target": "APOA5-ZNF259"}, {"weight": 1.0, "description": "SNP rs4660293 near PABPC4 may influence triglyceride levels through mediation by BMI>)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "RS4660293", "target": "MEDIATED PLEIOTROPY/MEDIATION"}, {"weight": 1.0, "description": "mediated pleiotropy/Mediation", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "LIPIDS/", "target": "BCL7B/FTO/TMEM18"}, {"weight": 1.0, "description": "mediation/Mediation", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "CRP/", "target": "PLTP/MC4R"}, {"weight": 2.0, "description": "rs1056513 variant is associated with the change in body weight z-score.)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "COL4A1", "target": "WEIGHT Z-SCORE"}, {"weight": 5.0, "description": "An intronic variant of PCSK2 was associated with total antioxidant content.)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "PCSK2", "target": "TOTAL ANTIOXIDANTS"}, {"weight": 7.0, "description": "A nonsynonymous SNP (S143T) in GCH1 was associated with the dopamine: creatinine ratio.)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "GCH1", "target": "DOPAMINE: CREATININE RATIO"}, {"weight": 8.0, "description": "rs3733402 variant is linked to serum free insulin-like growth factor 1 (IGF-1).)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "KLKB1", "target": "SERUM FREE IGF-1"}, {"weight": 9.0, "description": "An intronic SNP in MPRIP correlated with serum IGFBP-3 levels.)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "MPRIP", "target": "SERUM IGFBP-3"}, {"weight": 10.0, "description": "A block of SNPs associated with high serum TSH levels was found near the XPA/FOXE1 gene on chromosome 9.)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "XPA/FOXE1 (TTF-2)", "target": "SERUM TSH"}, {"weight": 11.0, "description": "SNPs in DARC are related to MCP-1 (Monocyte Chemoattractant Protein-1).)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "DARC", "target": "MCP-1"}, {"weight": 12.0, "description": "A variant in the 3\u2032UTR region of CCR3 correlated with MCP-1.)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "CCR3", "target": "MCP1"}, {"weight": 13.0, "description": "Variants near ABO were significantly related to IL-6 (interleukin-6).)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "ABO", "target": "IL-6"}, {"weight": 14.0, "description": "An intronic variant in TMEM229B on chromosome 14 was associated with ad libitum energy intake at dinner.)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "TMEM229B", "target": "ENERGY INTAKE"}, {"weight": 15.0, "description": "rs1056513 variant is linked to sleep duration.)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "ARHGAP11A", "target": "SLEEP DURATION"}, {"weight": 16.0, "description": "Intronic variants in C21orf34 were associated with RQ (respiratory quotient) levels during sleep.)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "C21ORF34", "target": "RESPIRATORY QUOTIENT DURING SLEEP"}, {"weight": 15.0, "description": "The 5\u2032UTR region variant of RPL7P3 influences total energy expenditure.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "5\u2032UTR REGION OF RPL7P3", "target": "TOTAL ENERGY EXPENDITURE"}, {"weight": 16.0, "description": "Variants in the 3\u2032UTR of CTCFL impact sedentary-light activity measured by an accelerometer.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "3\u2032UTR OF CTCFL", "target": "SEDENTARY-LIGHT ACTIVITY DURING SLEEP"}, {"weight": 5.0, "description": "The high-fat diet significantly increased body weight in both wild-type (Ctrl) and Senp1 knock-out (SENP1-aP2KO) mice.)", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "BODY WEIGHT", "target": "HFD"}, {"weight": 18.0, "description": "A significant variant for sleeping energy expenditure was found in the CHRNA3 gene on chromosome 15.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "SLEEPING ENERGY EXPENDITURE", "target": "CHRNA3 GENE ON CHROMOSOME 15"}, {"weight": 21.0, "description": "Sleeping energy expenditure is impacted by both genetic factors and environmental considerations.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "SLEEPING ENERGY EXPENDITURE", "target": "INFLUENCED BY GENETICS AND ENVIRONMENT"}, {"weight": 10.0, "description": "A variant identified for total energy expenditure was found in the 5\u2032UTR region of RPL7P3.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "TOTAL ENERGY EXPENDITURE", "target": "VARIANT IN 5\u2032UTR REGION OF RPL7P3"}, {"weight": 17.0, "description": "The variant associated with total energy expenditure is located within the MATK gene on chromosome 19.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "TOTAL ENERGY EXPENDITURE", "target": "MATK GENE ON CHROMOSOME 19"}, {"weight": 22.0, "description": "Total energy expenditure is influenced by genetic makeup, environmental conditions, and lifestyle choices.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "TOTAL ENERGY EXPENDITURE", "target": "AFFECTED BY GENETICS, ENVIRONMENT, LIFESTYLE"}, {"weight": 6.0, "description": "Variants in the flanking 3\u2032UTR regions of RNASE1 and ASS1P11 were related to nitrogen excretion and creatinine excretion.)", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "order": 1, "source": "EXCRETIONS", "target": "RNASE1, ASS1P11"}, {"weight": 6.0, "description": "A particularly prominent elevation in IL-1\u03b2 production was apparent in IL-10R\u03b1MdelRag1\u2212/\u2212 compared with Rag1\u2212/\u2212 LPM\u03d5s (mRNA expression ratio: 7.5\u00b11.2).)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "IL-6", "target": "IL-1\u0392"}, {"weight": 7.0, "description": "IL-6 supports Th17 cell formation and plays a role in exacerbating colitis depending on the recipient\'s genetics (Rag1\u2212/\u2212 vs IL-10R\u03b1MdelRag1\u2212/\u2212).)<SEP>Increased production of IL-1 and IL-6 by LPM\u03c6s can directly influence Th17 cell maturation; this impact was more pronounced in IL-10R\u03b1MdelRag1\u2212/\u2212-derived LPM\u03c6s.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8<SEP>chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "IL-6", "target": "TH17 CELL"}, {"weight": 1.0, "description": "IL-6 interacts with various components of the innate immune system to modulate inflammation and promote survival responses.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "IL-6", "target": "INNATE IMMUNE SYSTEM"}, {"weight": 8.0, "description": "Increased levels of IL-6 are associated with several autoimmune diseases, including colitis, due to its role in pro-inflammatory responses.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "IL-6", "target": "AUTOIMMUNITY"}, {"weight": 9.0, "description": "A SNP in the intronic region of C21orf34 detected RQ for respiratory quotient (RQ) during sleep.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "RESPIRATORY QUOTIENT (RQ) DURING SLEEP", "target": "SNP IN C21ORF34"}, {"weight": 11.0, "description": "Sedentary-light activity, measured by accelerometer, was detected in the 3\u2032UTR region of CTCFL.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "ACCELEROMETER-MEASURED LIGHT ACTIVITY", "target": "SEDENTARY-LIGHT ACTIVITY IN 3\u2032UTR OF CTCFL"}, {"weight": 12.0, "description": "A variant identified with highly significance for total energy expenditure is located on chromosome 19.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "HIGHLY SIGNIFICANT", "target": "VARIANT ON CHROMOSOME 19 FOR TOTAL ENERGY EXPENDITURE"}, {"weight": 13.0, "description": "A highly significant variant for sleeping energy expenditure was found on chromosome 15.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "HIGHLY SIGNIFICANT VARIANT FOR SLEEPING ENERGY EXPENDITURE", "target": "ON CHROMOSOME 15"}, {"weight": 19.0, "description": "Body weight adjustment is crucial when identifying specific genetic variants related to energy expenditure.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "ADJUSTMENT FOR BODY WEIGHT", "target": "IDENTIFICATION OF GENETIC VARIANTS"}, {"weight": 20.0, "description": "An accelerometer was used in measuring light activity, influencing the identification of genetic variants for different activities during sleep.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "ACCELEROMETER USE", "target": "MEASUREMENT OF LIGHT ACTIVITY DURING SLEEP"}, {"weight": 2.0, "description": "Variants rs74949322, 151233 and others were found in the study sample.)", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "order": 1, "source": "RS151233", "target": "APOBR GENE"}, {"weight": 2.0, "description": "A synonymous variant was found within the SULT1A1 gene, having no impact on its function.)", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "order": 1, "source": "RS151233", "target": "SULT1A1 GENE"}, {"weight": 3.0, "description": "This variant resulted in a glutamic acid change at position 170, but it was synonymous and had no functional effect.)", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "order": 1, "source": "RS149271", "target": "SULT1A1 GENE"}, {"weight": 4.0, "description": "The variant led to an arginine change at position 553, being another synonymous variant with no functional impact.)", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "order": 1, "source": "RS180744", "target": "SULT1A1 GENE"}, {"weight": 7.0, "description": "The variant resulted in a proline change at position 1012, which was synonymous and carried no functional consequences.)", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "order": 1, "source": "RS61738759", "target": "SULT1A1 GENE"}, {"weight": 8.0, "description": "This variant led to the creation of new transcription factor binding sites splicing enhancer and silencer binding sites, altering its function.)", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "order": 1, "source": "RS40833", "target": "SULT1A1 GENE"}, {"weight": 4.0, "description": "Both variants were carried by the individual hetero- or homozygously, due to their high LD relationship.)", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "order": 1, "source": "RS180743", "target": "APOBR GENE DELETION VARIANT"}, {"weight": 5.0, "description": "Variants rs180743 and rs368546180 are in high LD, leading to their co-occurrence.)", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "order": 1, "source": "RS180743", "target": "RS368546180"}, {"weight": 6.0, "description": "High linkage disequilibrium between the variant (rs2008514) and both rs180743 and rs3833080 results in their co-carrying.)", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "order": 1, "source": "RS180743", "target": "RS3833080"}, {"weight": 7.0, "description": "The deletion variant was identified once heterozygously in an obese individual, while the other variant did not exhibit this pattern.)", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "order": 1, "source": "RS368546180", "target": "APOBR GENE DELETION VARIANT"}, {"weight": 5.0, "description": "This variant resulted in a glycine change at position 560, which was also a synonymous variant without any functional effect.)", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "order": 1, "source": "RS151174", "target": "SULT1A1 GENE"}, {"weight": 6.0, "description": "The variant led to an alanine change at position 686, but it had no impact on the function due to being synonymous.)", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "order": 1, "source": "RS40831", "target": "SULT1A1 GENE"}, {"weight": 7.0, "description": "All five variants in TUFM are predicted to lead to altered splicing regulator or transcription factor binding sites.)<SEP>In silico analysis is used to show that this variant changes transcription factor binding sites, including splicing enhancer and silencer binding sites.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "S1 TABLE"}, {"weight": 10.0, "description": "In silico analyses predict functional effects on SNPs, with one being described as \'non-neutral\' or \'deleterious\' for rs10797300: p.Pro19Leu.)<SEP>The analyses provide insights into possible functional impacts of SNPs, including \'non-neutral\' or \'deleterious\' predictions for some.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "TABLE 2"}, {"weight": 3.0, "description": "In silico predictions show no association between this variant and obesity.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "TABLE 3"}, {"weight": 17.0, "description": "This genomic variant influences aspects of gene TUFM\'s functionality or expression based on the provided data.)<SEP>This genomic variant is found in the 3\' untranslated region of TUFM.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "TUFM"}, {"weight": 5.0, "description": "These two variants are non-coding intronic variants within TUFM.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "RS8061877: C.248\u221218G>A"}, {"weight": 7.0, "description": "This variant is an unknown variant located in the 3\' untranslated region of TUFM.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "G.28854194C/G"}, {"weight": 9.0, "description": "There is no association found between this variant and obesity in the trios tested using TDT analysis.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "TDT"}, {"weight": 10.0, "description": "Functional predictions for SNPs are given, including \'probably damaging\' for certain polymorphisms.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "POLYPHEN 2"}, {"weight": 11.0, "description": "In silico prediction tools like SNAP interpret the variant as having a \'non-neutral\' effect.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "SNAP"}, {"weight": 12.0, "description": "Some in silico tools classify the variant as being \'deleterious\', indicating potential harmful effects.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "SIFT"}, {"weight": 14.0, "description": "There is a consistent pattern across different analytical tools pointing towards potential functional impacts of the variant in TUFM.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "TABLE 2 AND S1 TABLE"}, {"weight": 15.0, "description": "Across different analytical data sets, predictions indicate lack of association between this variant and obesity traits.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "TABLE 3 AND S1 TABLE"}, {"weight": 17.0, "description": "A genome-wide association study (GWAS) identified this variant as potentially linked to certain diseases or traits.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "GWAS"}, {"weight": 18.0, "description": "The linkage disequilibrium (LD) score analysis indicates that this variant has predictive power in genetic associations, based on its correlation with other SNPs.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "LD SCORE"}, {"weight": 19.0, "description": "The GO analysis reveals functional roles and biological pathways associated with this variant, providing insights into its potential impact on cellular processes.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "GENE ONTOLOGY (GO) TERMS"}, {"weight": 20.0, "description": "This variant has been associated with various phenotypic traits in association studies, contributing to our understanding of its functional effects.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "PHENOTYPE ASSOCIATION STUDIES"}, {"weight": 21.0, "description": "An assessment of the clinical significance indicates that this variant could have implications for medical conditions, requiring further investigation into its pathogenicity.)", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "order": 1, "source": "RS142786317: C.*118_*119DELCA", "target": "CLINICAL SIGNIFICANCE ASSESSMENT"}, {"weight": 1.0, "description": "Variants found in TUFM lead to changes in splicing processes related to alterations in transcription factor binding sites.)", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "order": 1, "source": "TUFM", "target": "GENOMIC VARIANTS"}, {"weight": 2.0, "description": "Disruption of a potential transcription factor binding site by rs8061877 affects various factors like SRY, HFH 2, and HMG-IY.)", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "order": 1, "source": "RS8061877", "target": "TRANSCRIPTION FACTOR BINDING SITE"}, {"weight": 3.0, "description": "Predicted changes in splice enhancer binding sites and splicing silencer sites for this variant by three programs.)", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "order": 1, "source": "G.28854194C/G", "target": "SPLICE ENHANCER/SPLICE SILENCER BINDING SITES"}, {"weight": 4.0, "description": "The variant g.28854194C/G is situated in the non-coding 3\u2019 UTR of TUFM, but it still affects splicing processes.)", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "order": 1, "source": "G.28854194C/G", "target": "NON-CODING 3\u2019 UTR"}, {"weight": 3.0, "description": "The deletion variant is located near rs3833080, which is in high LD with this variant.)", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "order": 1, "source": "APOBR GENE DELETION VARIANT", "target": "RS3833080"}, {"weight": 1.0, "description": "Genetic polymorphisms associated with coffee consumption were identified through a GWAS.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "GWAS", "target": "IDENTIFIED SNPS"}, {"weight": 1.0, "description": "Lifestyle may have differing effects depending on the presence or absence of certain genomic variants.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "GENOMIC VARIANTS", "target": "LIFESTYLE"}, {"weight": 0.02, "description": "This SNP in the IL1B gene showed association with ovarian cancer risk, though it was not as strong as some other findings.)", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "order": 1, "source": "IL1B RS7596684", "target": "SNP ASSOCIATION"}, {"weight": 0.02, "description": "SNP rs7596684 in the IL1B gene shows a suggestive association with ovarian cancer risk.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "IL1B RS7596684", "target": "OVARIAN CANCER RISK"}, {"weight": 0.347, "description": "The odds of ovarian cancer were marginally increased when considering this SNP, potentially related to metabolic pathways involved.)", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "order": 1, "source": "PTGS1 RS2282169", "target": "METABOLIC PATHWAY"}, {"weight": 0.04, "description": "The odds of developing ovarian cancer were slightly elevated by this SNP, but the effect size was small and inconsistent.)", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "order": 1, "source": "PTGS1 RS2282169", "target": "SNP EFFECT ON RISK"}, {"weight": 0.04, "description": "SNP rs2282169 in the PTGS1 gene suggests an association with ovarian cancer risk.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "PTGS1 RS2282169", "target": "OVARIAN CANCER RISK"}, {"weight": 0.02, "description": "SNP rs7596684 in IL1B gene was associated with increased ovarian cancer risk.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "RS7596684", "target": "OVARIAN CANCER RISK"}, {"weight": 0.04, "description": "SNP rs2282169 in PTGS1 gene suggests a possible link to ovarian cancer.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "RS2282169", "target": "OVARIAN CANCER RISK"}, {"weight": 0.002, "description": "This intronic SNP was associated with reduced ovarian cancer risk, as part of the ALOX5 gene\'s function.)", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "order": 1, "source": "ALOX5 RS1864414", "target": "PHYSIOLOGICAL PROCESS"}, {"weight": 4.0, "description": "The IL-6R\u03b1 gene\'s response modulates a specific physiological process affecting only Rag1\u2212/\u2212 colitis model, highlighting its role in disease management.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "PHYSIOLOGICAL PROCESS", "target": "IL-6R\u0391 GENE"}, {"weight": 2.0, "description": "Endogenous choline synthesis reduced due to the rs12325817 variant in 21.6% of infants)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "PHYSIOLOGICAL PROCESS", "target": "INFANTS"}, {"weight": 1.0, "description": "Alterations in metabolic pathways related to fatty acid metabolism in adipose tissue are influenced by SENP1 expression and SUMOylation status, impacting insulin sensitivity and glucose homeostasis. )", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "order": 1, "source": "PHYSIOLOGICAL PROCESS", "target": "METABOLIC PATHWAY"}, {"weight": 9.0, "description": "The UBZ domain is vital for physiological processes such as recruitment to sites of DNA damage, highlighting its role in cellular maintenance and repair.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "PHYSIOLOGICAL PROCESS", "target": "FAAP20-UBZ DOMAIN"}, {"weight": 2.0, "description": "The impact on outcomes based on differences in the choline and placebo groups considering maternal status and genetic variants.)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "METABOLIC PATHWAY", "target": "INFANTS"}, {"weight": 9.0, "description": "A significant correlation between serum C4 and BA synthesis indicated that changes in this inflammatory marker could influence bile acid metabolic pathways or associated health outcomes.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "METABOLIC PATHWAY", "target": "SERUM C4 (BA SYNTHESIS CORRELATION)"}, {"weight": 4.0, "description": "The UBZ domain of FAAP20 is involved in a metabolic pathway related to the interaction with ubiquitin (Ub), crucial for cell repair mechanisms.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "METABOLIC PATHWAY", "target": "UBZ-DOMAIN"}, {"weight": 5.0, "description": "Alterations in glycolysis pathways can influence cellular energy production, impacting various physiological functions.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "METABOLIC PATHWAY", "target": "GLYCOLYSIS"}, {"weight": 0.01, "description": "A correlated SNP in the ALOX5 gene was also found to be associated with ovarian cancer risk, but its impact was less significant than that of rs1864414.)", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "order": 1, "source": "ALOX5 RS745986", "target": "INTRAGENIC SNPS"}, {"weight": 4.0, "description": "Excluding individuals with asthma led to increased evidence for association at some loci, suggesting that disease status may influence genetic associations.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "SNP ASSOCIATION", "target": "ASTHMA EXCLUSION"}, {"weight": 1.0, "description": "SNP \'rs3918186\' in the NOS3 gene was linked with an increased risk of childhood ALL, as indicated by a Bayes factor greater than 1.)", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "order": 1, "source": "RELATIONSHIP DESCRIPTION", "target": "SNP RS3918186 IN NOS3 GENE AND CHILDHOOD ALL RISK"}, {"weight": 2.0, "description": "SNP \'rs1051266\' in the SLC19A1 gene showed evidence of association with childhood ALL, albeit less strength than NOS3 rs3918186.)", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "order": 1, "source": "RELATIONSHIP DESCRIPTION", "target": "SNP RS1051266 IN SLC19A1 GENE AND CHILDHOOD ALL RISK"}, {"weight": 1.0, "description": "After dietary intervention, the Mediterranean group showed an increase in MUFA and n-3 PUFA consumption while experiencing a decrease in 18:2 n-6 concentration.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "COLON FATTY ACID CONCENTRATIONS", "target": "DIET ARM"}, {"weight": 2.0, "description": "The association between genotypes and serum fatty acid levels was significant for omega-6 fatty acids (20:4), with minor allele carriers showing a lower concentration.)<SEP>The presence of any minor alleles in the FADS gene cluster is significantly associated with a lower serum 20:4, n-6 concentration.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "GENOTYPE GROUP", "target": "SERUM 20:4, N-6 CONCENTRATION"}, {"weight": 1.0, "description": "Mean serum 18:3, n-3 concentration for minor allele carriers were estimated to be 21% higher than those with all major alleles.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "GENOTYPE GROUP", "target": "SERUM 18:3, N-3 CONCENTRATION"}, {"weight": 1.0, "description": "The presence of any minor alleles had a marginally significant association with the colonic 20:4, n-6 concentration (p = 0.07).)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "GENOTYPE GROUP", "target": "COLONIC 20:4, N-6 CONCENTRATION"}, {"weight": 1.0, "description": "The presence of any minor alleles significantly affected the serum fatty acid concentrations (20:4, n-6; 18:3, n-3) and marginally affected colonic 20:4, n-6 concentration.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "GENOTYPE GROUP", "target": "FATTY ACID CONCENTRATIONS AT BASELINE"}, {"weight": 1.0, "description": "No significant effects of genotypes on colon insulin concentrations were observed, indicating the influence of diet rather than genetics in this aspect.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "GENOTYPE GROUP", "target": "COLON INSULIN CONCENTRATION"}, {"weight": 1.0, "description": "Carriers of minor alleles responded differently to dietary interventions compared to those with all major alleles, in terms of changes in serum fatty acid concentrations.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "GENOTYPE GROUP", "target": "DIETARY INTERVENTION RESPONSE"}, {"weight": 1.0, "description": "No significant differences in response to genotypes were found for changes in serum triglyceride levels among the dietary intervention groups.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "GENOTYPE GROUP", "target": "SERUM TRIGLYCERIDE LEVELS"}, {"weight": 1.0, "description": "Changes in dietary fat intake significantly impact serum fatty acid composition; reductions and increases were observed in both diet groups after intervention.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "DIET ARM", "target": "SERUM FATTY ACID CONCENTRATIONS"}, {"weight": 1.0, "description": "Both dietary intervention groups showed changes in serum fatty acid levels; the Healthy Eating group reduced SFA intake while increasing long chain n-3 PUFA, leading to a higher polyunsaturated fat: saturated fat ratio.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "DIET ARM", "target": "SERUM FATTY ACID CONCENTRATIONS AFTER INTERVENTION"}, {"weight": 1.0, "description": "The Mediterranean diet resulted in changes towards a healthier fatty acid profile with increased n-3 PUFA and decreased 18:2 n-6 concentration, compared to baseline.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "DIET ARM", "target": "COLON FATTY ACID CONCENTRATIONS AFTER INTERVENTION"}, {"weight": 1.0, "description": "The dietary interventions led to significant changes in serum fatty acid concentrations; these changes were particularly notable between the two diet groups.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "DIET ARM", "target": "SERUM FATTY ACID CHANGES"}, {"weight": 1.0, "description": "Both dietary interventions led to changes in colon fatty acid concentrations; the Mediterranean group showed a particular decrease in 18:2 n-6 concentration.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "DIET ARM", "target": "COLON FATTY ACID CHANGES"}, {"weight": 1.0, "description": "Both dietary interventions did not lead to significantly different serum glucose levels compared to baseline or between each other, suggesting that changes were minor or within normal variation.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "DIET ARM", "target": "SERUM GLUCOSE LEVELS"}, {"weight": 1.0, "description": "While both diet groups showed changes in colon fatty acid profiles after intervention, there was no significant difference between the Mediterranean and Healthy Eating diets regarding their polyunsaturated to saturated fat ratio.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "DIET ARM", "target": "COLON POLYUNSATURATED: SATURATED RATIO"}, {"weight": 1.0, "description": "After the interventions, no diet group showed a significant effect on colon linoleic acid (20:4, n-6) concentration compared to baseline or each other.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "DIET ARM", "target": "COLON 20:4, N-6 CONCENTRATION"}, {"weight": 1.0, "description": "The Healthy Eating diet led to an increase in colon MUFA (monounsaturated fat) concentrations compared to baseline, indicating changes towards a healthier fatty acid profile.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "COLON MUFA CONCENTRATION", "target": "HEALTHY EATING DIET"}, {"weight": 2.0, "description": "The Healthy Eating diet led to an increase in colon docosahexaenoic acid (DHA; a type of omega-3 fatty acid, also represented as 20:5, n-3), indicating beneficial effects on DHA levels.)<SEP>The Healthy Eating diet resulted in an increase in colon eicosapentaenoic acid (EPA; a type of omega-3 fatty acid, also represented as 20:5, n-6), indicating improved EPA levels after the intervention.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "HEALTHY EATING DIET", "target": "COLON 18:3, N-3 CONCENTRATION"}, {"weight": 2.0, "description": "Mean colon AA concentrations were significantly lower in the Mediterranean arm compared to Healthy Eating by month 6.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "DIET ARM EFFECT", "target": "SUBJECT WITH NO MINOR ALLELES"}, {"weight": 2.0, "description": "Mean colon AA concentrations at month 6 differed significantly based on diet arm allocation, specifically for subjects without minor alleles.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "DIET ARM SPECIFICITY", "target": "OUTCOME DIFFERENCE"}, {"weight": 1.0, "description": "Self-reported sleep apnoea has the highest genetic correlation with habitual snoring (rG = 0.78).)", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "order": 1, "source": "GENETIC CORRELATION", "target": "SLEEP APNOEA - SELF-REPORTED SLEEP APNOEA"}, {"weight": 1.0, "description": "Several traits with the strongest evidence of association were BMI, obesity indicators, sleep apnoea, mood disorders, cardiovascular diseases, lung function, and personality traits related to stress and emotional health.)", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "order": 1, "source": "GENETIC CORRELATION", "target": "BMI, WHOLE-BODY FAT MASS, SODIUM IN URINE, MOOD SWINGS, CORONARY ARTERY DISEASE, ALCOHOL INTAKE FREQUENCY, PULSE RATE, CURRENT TOBACCO SMOKING, HEART DISEASE, LUNG CANCER, FEV1/FVC RATIO, NEUROTICISM, SUBJECTIVE WELLBEING, HEART RATE - TRAITS GENETICALLY CORRELATED TO SNORING"}, {"weight": 1.0, "description": "After adjusting for BMI, traits like schizophrenia, educational attainment, and sleep-related conditions remained genetically correlated with snoring.)", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "order": 1, "source": "GENETIC CORRELATION", "target": "SCHIZOPHRENIA, EDUCATIONAL ATTAINMENT, SODIUM IN URINE, DAYTIME DOZING, SLEEP APNOEA, EXCESSIVE DAYTIME SLEEPINESS - TRAITS GENETICALLY CORRELATED AFTER BMI ADJUSTMENT"}, {"weight": 0.05, "description": "The polymorphisms in VDBP gene were associated with maternal vitamin D concentrations during delivery.)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "VDBP SNPS", "target": "MATERNAL VITAMIN D STATUS"}, {"weight": 1.0, "description": "The vitamin D status during delivery was linked to various genotypes of the VDBP gene.)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "MATERNAL VITAMIN D STATUS AT DELIVERY (\u226575 NMOL/L VS. \u226475 NMOL/L)", "target": "VDBP GENOTYPES"}, {"weight": 1.0, "description": "Neonatal vitamin D levels were associated with maternal VDBP polymorphisms. )", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "MATERNAL VDBP GENOTYPES", "target": "NEONATAL VITAMIN D STATUS AT BIRTH (CUT-OFFS OF 25 AND 50 NMOL/L)"}, {"weight": 1.0, "description": "The genetic variants in VDBP correlated with maternal vitamin D levels during childbirth.)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "MATERNAL VITAMIN D CONCENTRATION", "target": "VDBP POLYMORPHISMS"}, {"weight": 1.0, "description": "Maternal 25(OH)D Cut-Offs at Birth", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "VDBP POLYMORPHISMS", "target": "NEONATAL CUT-OFFS (\u226550 NMOL/L VS. \u226450 NMOL/L, \u226525 VS. \u226425 NMOL/L)"}, {"weight": 1.0, "description": "The cut-offs for maternal vitamin D were associated with specific genotypes of the VDBP gene.)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "MATERNAL VDBP POLYMORPHISMS", "target": "MATERNAL VITAMIN D CUT-OFFS (\u226425 VS. \u226525 NMOL/L, \u226450 VS. \u226550 NMOL/L AND \u226575 NMOL/L VS. \u226475 NMOL/L)"}, {"weight": 1.0, "description": "The neonatal vitamin D levels correlated with the maternal VDBP genotypes at birth.)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "MATERNAL VDBP POLYMORPHISMS", "target": "NEONATAL VITAMIN D LEVELS AT BIRTH (CUT-OFFS OF 25 AND 50 NMOL/L)"}, {"weight": 1.0, "description": "The polymorphisms rs2298850 and rs4588 were associated with maternal vitamin D concentrations during childbirth.)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "MATERNAL VITAMIN D CONCENTRATION DURING DELIVERY", "target": "VDBP SNPS (RS2298850 AND RS4588)"}, {"weight": 1.0, "description": "The minor C allele of rs3025020 is associated with increased PCOS-related metabolic markers compared to the minor C allele of rs2010963.)", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "order": 1, "source": "PCOS FEATURES", "target": "RS3025020 AND RS2010963 GENOTYPES"}, {"weight": 1.0, "description": "The main effect of Genotype was found due to differences in behavior adjustment across reinforcement contingencies (Condition) and time.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "GENOTYPE GROUPS", "target": "CONDITION X TIME INTERACTION"}, {"weight": 1.0, "description": "A significant interaction effect was found between genotype groups and neuropsychological test performance, indicating different cognitive profiles across genotypes.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "GENOTYPE GROUPS", "target": "NEUROPSYCHOLOGICAL TEST PERFORMANCE INTERACTION EFFECT"}, {"weight": 2.0, "description": "Response bias showed a main effect for block, with higher bias in later blocks compared to earlier blocks during the study.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "RESPONSE BIAS", "target": "MAIN EFFECT OF BLOCK"}, {"weight": 6.0, "description": "Increased response bias in val carriers relative to iso homozygotes suggests a potential link between specific genotypes and disease risk.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "RESPONSE BIAS", "target": "DISEASE RISK"}, {"weight": 2.0, "description": "There was a strong correlation between response bias and neural activity patterns in specific brain regions, as identified through functional MRI analysis.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "RESPONSE BIAS", "target": "NEURAL ACTIVITY CORRELATION"}, {"weight": 6.0, "description": "The increase in response bias was moderated by the level of cognitive load, suggesting a threshold for overload conditions.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "RESPONSE BIAS", "target": "COGNITIVE LOAD EFFECT"}, {"weight": 2.0, "description": "Genetic variant (val allele) had an effect on response bias specifically under conditions of stress.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "RESPONSE BIAS", "target": "CONDITION X GENOTYPE INTERACTION"}, {"weight": 4.0, "description": "The val allele explained unique variance for reductions in stress-induced response bias after accounting for factors like discriminability and other genotypes.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "RESPONSE BIAS", "target": "STRESS INDUCED RESPONSE BIAS REDUCTIONS"}, {"weight": 2.0, "description": "Under non-stressed conditions, response bias tends to remain stable or follow baseline patterns.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "RESPONSE BIAS", "target": "NO-STRESS CONDITION"}, {"weight": 3.0, "description": "Val carriers exhibited significantly different response bias than iso homozygotes due to their genotype.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "VAL CARRIERS", "target": "MAIN EFFECT OF GENOTYPE"}, {"weight": 3.0, "description": "The presence of risk variant genes was correlated with enhanced response bias that may indicate heightened disease susceptibility.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "VAL CARRIERS", "target": "GENETIC RISK TO DISEASE"}, {"weight": 1.0, "description": "Val carriers had more non-specific Skin Conductance Responses than iso homozygotes in stress condition.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "VAL CARRIERS", "target": "STRESS CONDITION"}, {"weight": 4.0, "description": "Participants adapted behavior based on reinforcement contingencies during the study, demonstrating the influence of external stimuli on behavior adjustment.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "REINFORCEMENT CONTINGENCIES", "target": "ADJUSTMENT OF BEHAVIOR"}, {"weight": 4.0, "description": "Participants\' adaptive behavior improved over time in response to reinforcement contingencies, showing plasticity and learning capabilities.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "REINFORCEMENT CONTINGENCIES", "target": "ADAPTIVE BEHAVIOR"}, {"weight": 5.0, "description": "Analysis confirmed the main effect of genotype when considering only Caucasian subjects to avoid confounding due to population stratification.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "CAUCASIAN SUBJECTS", "target": "POPULATION STRATIFICATION INTERACTION EFFECT"}, {"weight": 5.0, "description": "Analysis of specific genetic loci showed that their effects were more pronounced among Caucasian subjects than other ethnicities.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "CAUCASIAN SUBJECTS", "target": "GENETIC INFLUENCE SPECIFICITY"}, {"weight": 7.0, "description": "Later blocks showed higher response bias compared to earlier blocks, indicating that environment or task complexity influenced behavior later in the study.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "BLOCK 3", "target": "ENVIRONMENTAL FACTOR INTERACTION"}, {"weight": 8.0, "description": "Main effects of condition and time on STAI scores emerged, with higher scores during stress conditions relative to no-stress conditions.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "STAI SCORES", "target": "CONDITION EFFECT"}, {"weight": 7.0, "description": "Anxiety levels influenced the magnitude of STAI scores, with high anxiety participants showing greater variability in their responses.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "STAI SCORES", "target": "ANXIETY MODERATION FACTOR"}, {"weight": 9.0, "description": "Cohen\u2019s d was used to compare effect sizes between genotypes or different study variables, providing a measure of practical significance.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "COHEN\'S D", "target": "EFFECT SIZE COMPARISON"}, {"weight": 8.0, "description": "Cohen\'s d values were used to benchmark differences in mean scores across genotype groups, aiding in distinguishing true effects from random variation.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "COHEN\'S D", "target": "SIGNIFICANCE BENCHMARK COMPARISON"}, {"weight": 10.0, "description": "A significant interaction on STAI scores over time indicated that the stress condition influenced participants\' self-report measures differently at various points during the study.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "CONDITION X TIME INTERACTION", "target": "STAI SCORE CHANGE"}, {"weight": 9.0, "description": "Over time, the impact of stress conditions on STAI scores was more pronounced, indicating increased sensitivity to anxiety over longer periods.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "CONDITION X TIME INTERACTION", "target": "TEMPORAL ANXIETY CHANGES"}, {"weight": 11.0, "description": "Increased Negative Affect (NA) scores across conditions suggested heightened emotional responses to both stress and no-stress situations during the experiment.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "PANAS NA SCORES", "target": "AFFECT SCALE RESPONSE"}, {"weight": 10.0, "description": "Differences in PANAS NA scores suggested that certain genotypes may predispose individuals to higher levels of negative affectivity.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "PANAS NA SCORES", "target": "GENETIC CONTRIBUTION TO EMOTION"}, {"weight": 1.0, "description": "More skin conductance responses were observed in response to the stress condition compared to the no-stress condition, suggesting a higher physiological arousal during stress.)", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "order": 1, "source": "NON-SPECIFIC SC RESPONSES", "target": "STRESS RESPONSE RATE"}, {"weight": 9.0, "description": "Subject group studied for differential gene expression patterns via qRT-PCR and microarray data verification.)", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "order": 1, "source": "CONDITION", "target": "DBA PATIENTS"}, {"weight": 3.0, "description": "Elevated discriminability was observed in Blocks 3 and 2 compared to Block 1, but no significant differences were found between genotype groups across blocks.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "STRESS CONDITION", "target": "DISCRIMINABILITY"}, {"weight": 5.0, "description": "val carriers showed a lower response bias in stress condition compared to no-stress condition.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "COHENS D = 1.22, P = .056", "target": "SHOCK INTENSITY, SKIN CONDUCTANCE RESPONSES"}, {"weight": 4.0, "description": "SNP rs1490384 near C6orf173 is associated with \'Age at first tooth\' and \'Number of teeth\', linking height to earlier tooth eruption.)", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "order": 1, "source": "CACNB2", "target": "RS1490384"}, {"weight": 5.0, "description": "SNP rs1570106 in RAD51L1 shows association with both \'Age at first tooth\' and \'Number of teeth\', indicating link between height and tooth eruption.)", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "order": 1, "source": "RAD51L1", "target": "RS1570106"}, {"weight": 0.8, "description": "Variants in the HMGA2 and BMP4 genes contribute significantly to associations between \'Age at first tooth\' SNPs and height.)", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "order": 1, "source": "SNPS IN HMGA2 AND BMP4", "target": "CONTRIBUTION TO AGE AT FIRST TOOTH SNP ASSOCIATIONS"}, {"weight": 1.0, "description": "Several height-associated SNPs show strong evidence of association with tooth eruption in the expected direction (i.e., taller individuals have faster tooth eruption/more teeth).)", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "order": 1, "source": "HEIGHT-ASSOCIATED SNPS", "target": "TOOTH ERUPTION"}, {"weight": 3.0, "description": "SNP rs12534093 has a positive effect on faster tooth eruption/more teeth linked to IGF2BP3 association.)", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "order": 1, "source": "IGF2BP3", "target": "RS12534093"}, {"weight": 6.0, "description": "The allelic scores for \'Age at first tooth\' and \'Number of teeth\', derived from genome-wide significant SNPs, show moderate evidence of association with height.)", "source_id": "chunk-29b3ac5f9dbe166f36559a272d7bcba1", "order": 1, "source": "GENOME-WIDE SIGNIFICANT SNPS FOR AGE AT FIRST TOOTH AND NUMBER OF TEETH", "target": "HEIGHT AT 17 YEARS AND HEIGHT AT 31 YEARS"}, {"weight": 1.0, "description": "SNPs associated with \'Age at first tooth\' show moderate evidence of association with height.)", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "order": 1, "source": "ASSOCIATION WITH HEIGHT", "target": "SNPS FOR AGE AT FIRST TOOTH"}, {"weight": 0.9, "description": "This threshold provides the most significant over-representation of genes in pathways for \'age at first tooth\' GWAS.)", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "order": 1, "source": "PATHWAY ANALYSIS", "target": "SNP ASSOCIATION P-VALUE THRESHOLD OF 0.005"}, {"weight": 7.0, "description": "Treatment with an anti-IL-10 receptor antibody influences pro-IL-1\u03b2 expression by modulating signaling pathways in T cell responses and disease progression.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "IL-10R\u0391MDEL", "target": "DRUG/CHEMICAL INTERACTION"}, {"weight": 3.0, "description": "Increased formation of Th17 cells was observed when LPM\u03c6s were derived from IL-10R\u03b1MdelRag1\u2212/\u2212 mice compared to those from Rag1\u2212/\u2212 mice.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "LPM\u03a6S", "target": "IL-10R\u0391MDELRAG1\u2212/\u2212 M\u03a6S"}, {"weight": 5.0, "description": "Significantly increased percents and absolute numbers of CD4+IL-17A+ T cells were identified in IL-10R\u03b1Mdel Rag1\u2212/\u2212 compared with Rag1\u2212/\u2212 colons.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "TH17 CELLS", "target": "IL-1\u0392"}, {"weight": 7.0, "description": "Increased colonic infiltration by both Th1 and Th17 cells, and the Th17 response is required for disease development.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "TH17 CELLS", "target": "IL-17"}, {"weight": 8.0, "description": "No differences in CD4+IFN-\u03b3+ T cell quantities were evident.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "TH17 CELLS", "target": "IFN-\u0393"}, {"weight": 9.0, "description": "A particularly prominent elevation in IL-1\u03b2 production was apparent...)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "TH17 CELLS", "target": "IL-23"}, {"weight": 10.0, "description": "For IL-1\u03b2, IL-6, IL-23p19, and iNOS, relative expression was compared in sorted colonic LPM\u03d5s, DCs, neutrophils, and epithelial cells.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "TH17 CELLS", "target": "INOS"}, {"weight": 7.0, "description": "IL-10 influences the function of M\u03c6s and their responses to other stimuli like IL-6, impacting disease pathogenesis in Rag1\u2212/\u2212 mice.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "M\u03a6 (MACROPHAGES)", "target": "IL-10"}, {"weight": 6.0, "description": "IL-1\u03b2 plays an essential role in Th17 cell development during healthy steady-state conditions, but its importance is downplayed by IL-6R\u03b1 signaling suppression.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "IL-1\u0392", "target": "TH1 CELL"}, {"weight": 8.0, "description": "Interleukin 1 beta (IL-1\u03b2) binds to the IL-1 receptor (IL-1R), activating pro-inflammatory responses that can exacerbate colitis, especially in the presence of elevated IL-1\u03b2 expression.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "IL-1\u0392", "target": "IL-1R"}, {"weight": 4.0, "description": "IL-1\u03b2 has a dual role: it contributes to the activation of Th17 cells and potentially suppresses regulatory T cell function, influencing autoimmune diseases like colitis.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "IL-1\u0392", "target": "REGULATORY T CELLS"}, {"weight": 10.0, "description": "IL-1\u03b2 acts as a pro-inflammatory cytokine in the innate immune system, playing a critical role in conditions such as colitis through its impact on inflammation processes.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "IL-1\u0392", "target": "INNATE IMMUNE SYSTEM"}, {"weight": 6.0, "description": "IL-10 led to complete suppression of IL-1\u03b2 maturation following treatment with it.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "IL-1\u0392", "target": "IL-10"}, {"weight": 4.0, "description": "No significant differences were found between Th1 and Th17 cell percentages during T-cell transfer experiments, suggesting distinct roles in colitis development.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "TH17 CELL", "target": "TH1 CELL"}, {"weight": 14.0, "description": "IL-6R\u03b1 signaling supports but is not essential for Th17 formation; it influences the response of both IL-10R\u03b1MdelRag1\u2212/\u2212 and Rag1\u2212/\u2212 mice.)<SEP>The presence and activity of IL-6 Receptor Alpha (IL-6R\u03b1), a key factor for Th17 cell development, are demonstrated to influence disease progression in Rag1\u2212/\u2212 mice.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "TH17 CELL", "target": "IL-6R\u0391"}, {"weight": 2.0, "description": "Th17 cells contribute to autoimmunity through their pro-inflammatory cytokine production, particularly in conditions like colitis.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "TH17 CELL", "target": "AUTOIMMUNITY"}, {"weight": 7.0, "description": "Th17 cells interact with macrophages, leading to a cascade of events that can either exacerbate or mitigate inflammation in conditions like colitis.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "TH17 CELL", "target": "MACROPHAGE (M\u03a6)"}, {"weight": 6.0, "description": "IL-10 can act on LPM\u03c6s to directly suppress Th17 production, especially in the context of downregulated IL-1 and partially regulated IL-6 expression.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "TH17 CELL", "target": "IL-10"}, {"weight": 5.0, "description": "Inhibition of IL-1 signaling led to a substantial reduction in Th17 cell development, highlighting its role in supporting Th17 production by LPM\u03c6s.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "TH17 CELL", "target": "IL-1 SIGNALING"}, {"weight": 3.0, "description": "In the presence of IL-6R\u03b1 signaling pathway, macrophages can be influenced to either promote or limit inflammation depending on the context.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "IL-6R\u0391", "target": "MACROPHAGE (M\u03a6)"}, {"weight": 15.0, "description": "IL-10 modulates immune responses by inhibiting pro-inflammatory cytokines including IL-6, affecting Th1 cell functions and potentially impacting the development of colitis.)<SEP>Interleukin 10 (IL-10) has the ability to modulate immune responses by reducing inflammation; it influences T cell responses, particularly Th1 cells, in the context of colitis development.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "TH1 CELL", "target": "IL-10"}, {"weight": 9.0, "description": "The relationship between Th1 cells and regulatory T cells is crucial for balancing immune responses; an imbalance can lead to autoimmune diseases like colitis.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "TH1 CELL", "target": "REGULATORY T CELLS"}, {"weight": 10.0, "description": "Autocrine IL-10 normally restrains caspase-1 activation and this function was disrupted by blocking IL-10R\u03b1 Ab treatment in Rag1\u2212/\u2212 M\u03d5s.)<SEP>In the IL-10R\u03b1MdelRag1\u2212/\u2212 M\u03d5s, IL-10 normally restrains caspase-1 activation through autocrine signaling; this was not affected by pre-treatment with anti-IL-10R\u03b1 Ab.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "IL-10", "target": "CASPASE-1"}, {"weight": 10.0, "description": "Caspase-1 is activated through an inflammasome response to danger signals or stimulation by agents like LPS and ATP, leading to the processing of pro-inflammatory cytokines such as IL-1\u03b2.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "CASPASE-1", "target": "PATHWAY ACTIVATION"}, {"weight": 2.0, "description": "Lipopolysaccharide stimulation leads to an increase in the level of activated caspase-1.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "CASPASE-1", "target": "LPS"}, {"weight": 3.0, "description": "Stimulation by ATP also leads to an increase in the activation form of caspase-1.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "CASPASE-1", "target": "ATP"}, {"weight": 4.0, "description": "Rag1\u2212/\u2212 macrophages showed increased caspase cleavage upon LPS and ATP stimulation.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "CASPASE-1", "target": "RAG1\u2212/\u2212 M\u03a6S"}, {"weight": 11.0, "description": "The use of flow cytometry enables researchers to collect data on cell populations and protein expression levels like that of pro-IL-1\u03b2, aiding in understanding complex immune responses.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "FLOW CYTOMETRY", "target": "DATA COLLECTION METHOD"}, {"weight": 12.0, "description": "Western analysis serves as a method for visualizing proteins by size after they have been labeled with an antibody. It is crucial for confirming cytokine production levels like IL-1\u03b2 in response to different stimuli.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "WESTERN ANALYSIS", "target": "PROTEIN DETECTION METHOD"}, {"weight": 13.0, "description": "Exposure to LPS leads to activation of signaling pathways that impact pro-inflammatory responses and cytokine expression, such as those involved with IL-1\u03b2 production during colitis.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "LPS", "target": "SIGNALING PATHWAY ACTIVATION"}, {"weight": 14.0, "description": "The presence or release of ATP can stimulate cellular responses leading to inflammasome activation which impacts the expression levels of pro-inflammatory cytokines, like IL-1\u03b2, in inflammatory conditions such as colitis.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "ATP", "target": "STIMULATION EFFECT"}, {"weight": 15.0, "description": "Antibodies that bind to and disrupt IL-10 receptor function can alter signaling pathways that affect pro-inflammatory cytokine production, including IL-1\u03b2 levels in inflammatory contexts.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "ANTI-IL-10R ANTIBODY", "target": "PATHWAY DISRUPTION"}, {"weight": 4.0, "description": "Blocking the interaction with IL-6 receptor \u03b1 (IL-6R\u03b1), using specific antibodies, largely inhibited Th17 formation in cultures containing Rag1\u2212/\u2212 derived LPM\u03c6s.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "\u0391IL-6R\u0391 BLOCKING ANTIBODY", "target": "EFFECT ON TH17 FORMATION"}, {"weight": 8.0, "description": "The IL-6R\u03b1 gene\'s response is modified through interaction with genomic variants, impacting physiological processes that affect colitis models specifically in the Rag1\u2212/\u2212 context.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "IL-6R\u0391 GENE", "target": "PATHWAY MODIFICATION"}, {"weight": 3.0, "description": "The transfer of IL-6 receptor knockout (IL-6R\u2212/\u2212) T cells into mice showed modest effects on Th17 formation, suggesting the importance of functional IL-6R for this process.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "EXPERIMENTAL CONDITION", "target": "IL-6R\u2212/\u2212 T CELLS TRANSFER"}, {"weight": 5.0, "description": "Inhibition of IL-6 signaling pathways led to a significant reduction or complete suppression of Th17 cell development, emphasizing the role of IL-6 in this process.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "CONSEQUENCE ON TH17 DEVELOPMENT", "target": "IL-6 SIGNALING INHIBITION"}, {"weight": 3.5, "description": "The DNA extraction method significantly influences microbiome variation based on stepwise distance-based redundancy analysis (R2adj=0.27) with a P-value adjustment of Padj=7\u00d710\u22124.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "DNA EXTRACTION METHOD", "target": "MICROBIAL COMPOSITION"}, {"weight": 3.0, "description": "The DNA extraction method is a major factor contributing to richness variance, explaining 32.74% of the variation based on a variance component analysis.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "DNA EXTRACTION METHOD", "target": "RICHNESS VARIANCE"}, {"weight": 2.8, "description": "Ethnic background significantly contributes to heterogeneity in microbial composition, affecting alpha diversity and richness across different cohorts.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "MICROBIAL COMPOSITION", "target": "COHORT ETHNICITY"}, {"weight": 2.6, "description": "Cohort ethnicity along with the techniques used (e.g., DNA extraction methods) contributes to heterogeneity in microbial composition.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "MICROBIAL COMPOSITION", "target": "COHORT ETHNICITY AND TECHNIQUES USED"}, {"weight": 2.6, "description": "Ethnic background and techniques used in studies affect alpha diversity metrics such as Shannon, Simpson, and inverse Simpson indices, contributing to heterogeneity among cohorts.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "ALPHA DIVERSITY METRICS", "target": "COHORT ETHNICITY AND TECHNIQUES USED"}, {"weight": 1.0, "description": "Increased DNA modification at specific CpG sites in the LCT gene correlates with decreased mRNA levels.)", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "order": 1, "source": "LCT", "target": "DNA MODIFICATION"}, {"weight": 1.0, "description": "The LCT gene and MCM6 share overlapping regions with differential DNA modifications that influence cell type-specific expression.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "LCT", "target": "MCM6"}, {"weight": 3.0, "description": "The age-genotype interaction was significant in the GEM_v12 and GEM_ICHIP subcohorts, both comprising mostly European individuals.)", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "order": 1, "source": "RS182549", "target": "AGE-GENOTYPE INTERACTION"}, {"weight": 9.0, "description": "rs182549 variant shows positive association with T2DM diagnosis.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "RS182549", "target": "T2DM"}, {"weight": 10.0, "description": "The rs182549 variant is associated with family history of T2DM (paternal, maternal and siblings).)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "RS182549", "target": "FAMILY HISTORY OF T2DM"}, {"weight": 11.0, "description": "The rs182549 variant shows association with 1,5-anhydroglucitol as an indicator for glycemic variability.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "RS182549", "target": "GLYCEMIC VARIABILITY"}, {"weight": 6.0, "description": "rs182549 genotype showed a higher abundance of the genus Bifidobacterium in the adult group, but not in the younger group.)", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "order": 1, "source": "BIFIDOBACTERIUM", "target": "YOUNG VERSUS ADULT GROUP EFFECT"}, {"weight": 16.0, "description": "Bifidobacterium abundance shows negative correlation with high cholesterol, even though this correlation wasn\'t significant after correction for multiple testing.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "BIFIDOBACTERIUM", "target": "HIGH CHOLESTEROL"}, {"weight": 1.0, "description": "The rs4988235 genotype is associated with higher consumption of dairy products like milk and milk beverages.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "RS4988235", "target": "DAIRY PRODUCTS"}, {"weight": 2.0, "description": "With an additional LP (T) allele, there\'s a positive association between the rs4988235 genotype and increased milk intake.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "RS4988235", "target": "MILK INTAKE"}, {"weight": 3.0, "description": "The rs4988235 genetic variant is related to dietary factors such as potato consumption, cereal products intake, fruits consumption, poultry eating habits, non-alcoholic beverage consumption, and wine drinking.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "RS4988235", "target": "DIETARY FACTORS"}, {"weight": 7.0, "description": "A minor decrease in HDL cholesterol levels is associated with rs4988235 genotype after adjusting for other variables.)<SEP>A statistically significant, albeit slight decrease in HDL cholesterol is associated with the rs4988235 genotype.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "RS4988235", "target": "HDL CHOLESTEROL"}, {"weight": 6.0, "description": "The rs4988235 genetic variant does not show a statistically significant link with developing diabetes and baseline HbA1c levels after adjusting for other factors.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "RS4988235", "target": "RISK OF DIABETES"}, {"weight": 1.0, "description": "An additional LP allele correlates with increased consumption of milk and milk-based beverages.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "RS4988235", "target": "DAIRY PRODUCT INTAKE (MILK, MILK BEVERAGES)"}, {"weight": 2.0, "description": "An association exists between rs4988235 genotype and specific dietary patterns that could influence diabetes risk.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "RS4988235", "target": "DIABETES RISK FACTORS (DIETARY)"}, {"weight": 4.0, "description": "There is a notable association between rs4988235 and certain dietary habits that may impact the risk of developing diabetes.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "RS4988235", "target": "DIABETES RISK FACTORS (DIETARY PATTERNS)"}, {"weight": 5.0, "description": "Through the influence of environmental factors, rs4988235 affects interactions with dairy intake, particularly milk and milk beverages.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "RS4988235", "target": "MOLECULAR INTERACTION (DAIRY CONSUMPTION)"}, {"weight": 6.0, "description": "rs4988235 is associated with particular nutrient intake levels such as total fats from specific fatty acids (C15:0, C17:0), impacting physiological processes and health outcomes.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "RS4988235", "target": "NUTRIENT ASSOCIATIONS"}, {"weight": 7.0, "description": "rs4988235 plays a role in diabetes risk when considering multiple environmental factors, as indicated by multivariate analysis results.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "RS4988235", "target": "DIABETES RISK (MULTIVARIABLE ADJUSTMENT)"}, {"weight": 0.03, "description": "In the Czech Republic, there is a weak association between rs4988235 genotype and educational attainment.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "RS4988235", "target": "EDUCATIONAL ATTAINMENT"}, {"weight": 3.0, "description": "The lack of an LCT mbQTL effect was observed among individuals of Israeli ethnicity.)", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "order": 1, "source": "GEM_V24", "target": "ISRAELI ETHNICITY"}, {"weight": 1.0, "description": "A genomic variant (rs7574352) is associated with family Peptococcaceae.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "FAMILY PEPTOCOCCACEAE", "target": "RS7574352"}, {"weight": 2.0, "description": "Stool levels of gut inflammation markers chromogranin A are negatively correlated with family Peptococcaceae.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "FAMILY PEPTOCOCCACEAE", "target": "CHROMOGRANIN A"}, {"weight": 3.0, "description": "Stool levels of the gut inflammation markers calprotectin are negatively correlated with family Peptococcaceae.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "FAMILY PEPTOCOCCACEAE", "target": "CALPROTECTIN"}, {"weight": 4.0, "description": "Family Peptococcaceae is negatively associated with ulcerative colitis.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "FAMILY PEPTOCOCCACEAE", "target": "ULCERATIVE COLITIS"}, {"weight": 4.0, "description": "Chronic disease characterized by high blood glucose levels, caused when the body fails to produce sufficient insulin or cannot use it effectively.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "TYPE 2 DIABETES", "target": "DISEASE/"}, {"weight": 6.0, "description": "Located near IRF1 gene involved in insulin resistance and type 2 diabetes.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "RS7574352", "target": "IRF1"}, {"weight": 15.0, "description": "The IRF1 gene was affected by GWAS SNPs that are associated with T1D and modulate its expression levels.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "IRF1", "target": "GWAS SNPS"}, {"weight": 5.0, "description": "The rs181206[G] allele results in lower transcript levels for IRF1 compared to the wild-type Leu119 variant.)", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "order": 1, "source": "IRF1", "target": "RS181206[G]"}, {"weight": 7.0, "description": "The variant rs182549 predisposes to lactose intolerance.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "LACTOSE INTOLERANCE", "target": "RS182549 (LCT)"}, {"weight": 12.0, "description": "The variant NC_000019.9:g.49218060 (rs35866622) is positively associated with fish intake.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "FISH INTAKE", "target": "NC_000019.9:G.49218060 (RS35866622)"}, {"weight": 13.0, "description": "The secretor allele of the FUT1/FUT2 locus is negatively associated with cholelithiasis.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "CHOLELITHIASIS", "target": "NC_000019.9:G.49218060 (RS35866622)"}, {"weight": 14.0, "description": "The secretor allele of the FUT1/FUT2 locus shows negative association with Crohn\'s disease.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "CROHN\u2019S DISEASE", "target": "NC_000019.9:G.49218060 (RS35866622)"}, {"weight": 15.0, "description": "The secretor allele of the FUT1/FUT2 locus shows negative association with alcohol intake frequency.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "ALCOHOL INTAKE FREQUENCY", "target": "NC_000019.9:G.49218060 (RS35866622)"}, {"weight": 17.0, "description": "The secretor allele of the FUT1/FUT2 locus shows negative association with waist-to-hip ratio.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "WAIST-TO-HIP RATIO", "target": "NC_000019.9:G.49218060 (RS35866622)"}, {"weight": 3.0, "description": "Differences in sequencing depth and DNA extraction methods are significant factors influencing alpha diversity metrics such as Shannon, Simpson, and inverse Simpson indices.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "ALPHA DIVERSITY", "target": "SEQUENCING DEPTH AND DNA EXTRACTION METHODS"}, {"weight": 3.0, "description": "Differences in sequencing depth and DNA extraction methods impact richness by influencing the microbial composition\'s diversity across various samples.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "SEQUENCING DEPTH AND DNA EXTRACTION METHODS", "target": "RICHNESS"}, {"weight": 2.5, "description": "There is a significant correlation between the heterogeneity across cohorts and alpha diversity metrics, indicating how different factors contribute to microbial richness and evenness.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "COHORT HETEROGENEITY AND ALPHA DIVERSITY", "target": "CORRELATION"}, {"weight": 3.0, "description": "A decrease in baseline HbF values correlated with an increase in delta HbF, indicating a potentially compensatory mechanism.)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "CORRELATION", "target": "DECREASE INCREASE"}, {"weight": 9.5, "description": "Spearman correlation coefficients indicated that serum C4 and stool weight were positively correlated indicating a relationship between these two parameters.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "CORRELATION", "target": "STOOL WEIGHT (SERUM C4)"}, {"weight": 10.0, "description": "A significant Spearman correlation was found between serum C4 levels and total stool BA suggesting a possible link in the metabolism of bile acids.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "CORRELATION", "target": "TOTAL STOOL BA (SERUM C4)"}, {"weight": 11.0, "description": "Spearman correlation coefficients suggested that BMI correlated with higher levels of serum C4 indicating potential influence on metabolic processes related to bile acid synthesis or excretion.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "CORRELATION", "target": "BMI (SERUM C4)"}, {"weight": 1.0, "description": "An age-genotype interaction on Bifidobacterium abundance was found, which was significant in multi-ethnic cohorts but not evident in the GEM_v24 cohort.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "AGE AND ETHNICITY INTERACTION", "target": "MBBTL EFFECT"}, {"weight": 1.0, "description": "The rs182549 genotype showed a significant SNP-age interaction on the level of Bifidobacterium abundance, with higher abundance observed in adults homozygous for the NC_000002.11:g.136616754CC variant.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "P<0.05", "target": "RS182549 - BIFIDOBACTERIUM ABUNDANCE"}, {"weight": 1.0, "description": "The residual heterogeneity for rs182549 was low after including quadratic and cubic terms of age in the model, indicating that the majority of variation has been explained.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "P=0.01", "target": "RS182549 - MBQTL EFFECT HETEROGENEITY"}, {"weight": 1.0, "description": "6)(entity", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "order": 1, "source": "PEPTOCOCCACEAE*|>GUT INFLAMMATION MARKERS", "target": "SNPS ASSOCIATED WITH PEPTOCOCCACEAE WERE NEGATIVELY CORRELATED WITH LEVELS OF GUT INFLAMMATION MARKERS LIKE CHROMOGRANIN A AND CALPROTECTIN, SUGGESTING LINKS TO GASTROINTESTINAL HEALTH.)"}, {"weight": 1.0, "description": "Results suggest potential protective effects of specific gut microbial taxa on autoimmune disorders such as ulcerative colitis and rheumatoid arthritis.)", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "order": 1, "source": "DISEASES", "target": "MICROBIAL TAXA"}, {"weight": 1.0, "description": "Lifestyle may have differing impacts on the development of AMD across different diseases and health states.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "DISEASES", "target": "LIFESTYLE"}, {"weight": 5.0, "description": "There are associations between certain dietary factors like potatoes, cereal products, fruits, poultry, non-alcoholic beverages and wine and diabetes risk when linked to the rs4988235 genotype.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "DIETARY FACTORS", "target": "DIABETES RISK FACTORS"}, {"weight": 1.0, "description": "8 infants met criteria for FAS and 2 for PFAS in the choline arm, compared with 5 who met criteria for FAS and 2 for PFAS in the placebo arm)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "INFANTS", "target": "CRITERIA"}, {"weight": 2.0, "description": "21.6% of the infants carried rs12325817 variant of the enzyme PEMT, reducing endogenous choline synthesis)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "INFANTS", "target": "MATERNAL PEMT ALLELE"}, {"weight": 2.0, "description": "9.7% (3 choline-treated, 3 placebo) of the infants were born to mothers whose dietary choline intake was adequate.)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "INFANTS", "target": "MATERNAL DIETARY STATUS"}, {"weight": 2.0, "description": "Significance tested for differences between treatment and placebo groups on outcomes relevant to FASD management)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "INFANTS", "target": "DISEASE MANAGEMENT APPROACH"}, {"weight": 2.0, "description": "9.7% of infants were born to mothers whose dietary choline intake during pregnancy was adequate (450 mg/day))", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "ADEQUATE", "target": "CHOLINE INTAKE"}, {"weight": 1.0, "description": "No associations were found between rs10399931 and impaired fasting glucose after adjusting for covariates.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "IFG", "target": "RS10399931"}, {"weight": 1.0, "description": "There was no evidence suggesting an association between rs4950928 and impaired glucose tolerance, although a borderline protective effect was observed for rs12123883.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "IGT", "target": "RS4950928"}, {"weight": 1.0, "description": "The rs10886471 SNP shows a genetic polymorphism as indicated by the PIC values exceeding 0.5.)", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "order": 1, "source": "GENETIC POLYMORPHISM", "target": "RS10886471 SNP"}, {"weight": 2.0, "description": "CT and T alleles were more common in healthy control subjects than those affected by HNP.)", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "order": 1, "source": "GENETIC POLYMORPHISM", "target": "TNF-A 238G>A"}, {"weight": 2.0, "description": "Intronic repeats could play roles in gene expression as enhancers or repressors based on studies.)", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "order": 1, "source": "ENHANCERS OR REPRESSORS", "target": "INTRONIC (CA)N REPEATS"}, {"weight": 3.0, "description": "The allelic frequencies of (CA)n repeat polymorphisms vary across different groups to assess disease risk.)", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "order": 1, "source": "ALLELIC FREQUENCIES", "target": "GENETIC VARIATION"}, {"weight": 1.0, "description": "The TT genotype of IL-1A was more frequent among patients with HNP compared to controls.)", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "order": 1, "source": "GENETIC VARIATION", "target": "IL-1A -889C>T"}, {"weight": 4.0, "description": "NFG group serves as a baseline/reference for comparison with other diabetes-related groups.)", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "order": 1, "source": "NFG GROUP", "target": "REFERENCE POPULATION"}, {"weight": 5.0, "description": "Groups of individuals showing elevated blood glucose levels, potentially leading to prediabetes or Type 2 diabetes.)", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "order": 1, "source": "DIABETES PREDIAGNOSIS", "target": "IFG AND IGT GROUPS"}, {"weight": 6.0, "description": "Group representing participants with diagnosed Type 2 diabetes mellitus.)", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "order": 1, "source": "DIABETES DISEASE STATE", "target": "T2DM GROUP"}, {"weight": 7.0, "description": "The allele (CA)16 is associated with increased risk of prediabetes and T2DM based on the logistic regression analysis.)", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "order": 1, "source": "ALLELE (CA)16", "target": "INCREASED RISK FACTOR"}, {"weight": 1.0, "description": "Specific allele of a genetic variant that serves as a reference for comparison with other alleles.)", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "order": 1, "source": "(CA)16", "target": "MULTIPLE ALLELE"}, {"weight": 3.0, "description": "Different disease stages or outcomes (Impaired Fasting Glucose, Impaired Glucose Tolerance, and Type 2 Diabetes Mellitus respectively), associated with the genetic variant and its alleles.)", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "order": 1, "source": "DISEASE GROUPS", "target": "IFG, IGT, T2DM"}, {"weight": 4.0, "description": "The specific genotypes (variants/alleles) are linked to an increased risk for developing Type 2 Diabetes Mellitus.)", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "order": 1, "source": "ASSOCIATION WITH T2DM", "target": "GENOMIC VARIANT AND MULTIPLE ALLELES"}, {"weight": 1.0, "description": "Serum selenium concentration showed no significant association with overall colorectal cancer risk but showed a positive relationship specifically with colon cancer risk.)", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "order": 1, "source": "SERUM SELENIUM CONCENTRATION", "target": "COLON CANCER RISK"}, {"weight": 3.0, "description": "There was potential evidence for publication bias in funnel plots indicating higher number of small studies reporting inverse associations.)", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "order": 1, "source": "SERUM SELENIUM CONCENTRATION", "target": "PUBLICATION BIAS"}, {"weight": 6.0, "description": "The protective association between selenium levels was stronger for colorectal adenomas than cancer.)", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "order": 1, "source": "SERUM SELENIUM CONCENTRATION", "target": "COLORECTAL ADENOMA RISK"}, {"weight": 7.0, "description": "The inverse association between selenium levels was stronger for adenomas compared to colorectal cancer.)", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "order": 1, "source": "SERUM SELENIUM CONCENTRATION", "target": "CANCER RISK (ADENOMA VS. CANCER)"}, {"weight": 4.0, "description": "Selenium showed a significant and inverse association with colorectal tumor risk specifically among men, but not significantly so among women.)", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "order": 1, "source": "COLORECTAL TUMOR RISK", "target": "GENDER BASED ANALYSIS"}, {"weight": 2.0, "description": "The SNP rs8178974 in the GPX4 gene was associated with colorectal cancer risk, specifically showing significance.)", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "order": 1, "source": "COLORECTAL CANCER RISK", "target": "GPX4 GENE SNP (RS8178974)"}, {"weight": 1.2, "description": "No obvious publication bias was identified through these methods, suggesting the reliability of our results.)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "PUBLICATION BIAS", "target": "EGGER\'S REGRESSION AND BEGG\'S FUNNEL PLOT"}, {"weight": 5.0, "description": "Results differed based on when (before or after) colorectal tumors were diagnosed; measurements taken post-diagnosis showed stronger associations.)", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "order": 1, "source": "DIAGNOSIS TIMING", "target": "SERUM SELENIUM MEASUREMENT"}, {"weight": 5.0, "description": "The risk allele of CARD9 SNP influences the ability of gut microbiota to produce AHR agonists, affecting AHR signaling in IBD patients.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "CARD9", "target": "RISK ALLELE (RS10781499)"}, {"weight": 3.0, "description": "AHR activation is influenced by gut microbiota-mediated tryptophan metabolism, which is altered in IBD patients.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "TRYPTOPHAN METABOLISM", "target": "AHR ACTIVATION"}, {"weight": 2.0, "description": "Increased production of certain microbial metabolites by gut bacteria can stimulate or inhibit AHR activation, impacting immune responses and inflammation.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "AHR ACTIVATION", "target": "MICROBIAL METABOLITE PRODUCTION"}, {"weight": 4.0, "description": "Gut microbiota produces ligands that stimulate AHR signaling; reduction is observed in subjects with IBD.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "AHR", "target": "MICROBIOTA DERIVED AHR LIGANDS"}, {"weight": 5.0, "description": "Decreased tryptophan metabolites are associated with autoinflammatory conditions, impacting the balance of inflammation and immunological tolerance.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "AUTOINFLAMMATION", "target": "ALTERATIONS IN TRYPTOPHAN METABOLISM"}, {"weight": 3.0, "description": "Prebiotics facilitate nutrient absorption including tryptophan, which impacts the synthesis of microbiota-derived metabolites affecting AHR signaling.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "NUTRIENT ABSORPTION", "target": "PREBIOTIC SUPPLEMENTATION"}, {"weight": 1.0, "description": "Cytokine Storm", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "order": 1, "source": "ALTERATION<|ALTERED BACTERIAL MICROBIOTA COMPOSITION DURING COLITIS AFFECTS INFLAMMATORY RESPONSES AND HEALING PROCESSES.)(ENTITY", "target": "BACTERIAL MICROBIOTA"}, {"weight": 2.0, "description": "Card9 deficiency contributes to gut microbiota-induced susceptibility to colitis by disrupting IL-22 signaling and tryptophan metabolism.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "CARD9 DEFICIENCY", "target": "GUT MICROBIOTA"}, {"weight": 6.0, "description": "Major IBD-associated SNPs do not significantly affect the ability of gut microbiota to produce AHR agonists.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "IBD", "target": "NOD2, ATG16L1, LRRK2 SNPS"}, {"weight": 4.0, "description": "Probiotics can improve gut health by modulating microbiota composition and supporting AHR pathway regulation.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "GUT HEALTH", "target": "PROBIOTICS TREATMENT"}, {"weight": 1.0, "description": "A positive association between baseline \u03b22M concentration and prevalent hypertension risk was observed after adjusting for clinical covariates.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "\u03922M", "target": "PREVALENT HYPERTENSION"}, {"weight": 2.0, "description": "Baseline \u03b22M concentration was significantly associated with the development of new-onset hypertension in the study sample.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "\u03922M", "target": "NEW-ONSET HYPERTENSION"}, {"weight": 3.0, "description": "The GEE model was employed to analyze the relationship between baseline \u03b22M and risk for new-onset hypertension during follow-up.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "GEE MODEL", "target": "LONGITUDINAL ANALYSES"}, {"weight": 1.0, "description": "Creating genetically modified mice with adipocyte-specific deletion of Senp1 resulted in diabetes development.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "DIABETES", "target": "GENETIC MODIFICATION"}, {"weight": 7.0, "description": "Diminished expression of Senp1 was associated with diabetes development.)<SEP>The diabetes phenotype correlated with the alteration of Senp1 gene expression in adipocytes.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "DIABETES", "target": "GENE EXPRESSION"}, {"weight": 8.0, "description": "Low insulin levels were observed as part of the diabetes phenotype in Senp1-deficient mice.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "DIABETES", "target": "HORMONAL RESPONSE"}, {"weight": 9.0, "description": "The tests confirmed the diabetes phenotype and insulin resistance in Senp1-deficient mice.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "DIABETES", "target": "ASSESSMENTS"}, {"weight": 11.0, "description": "Reduced insulin secretion correlated with diabetes development in Senp1-deficient mice.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "DIABETES", "target": "HORMONAL FUNCTION"}, {"weight": 12.0, "description": "Lower body weight was a physical manifestation observed in the diabetic phenotype of Senp1-deficient mice.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "DIABETES", "target": "PHYSICAL CHARACTERISTICS"}, {"weight": 13.0, "description": "Alteration of cholesterol and triglyceride levels was observed as a metabolic change associated with diabetes development in Senp1-deficient mice.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "DIABETES", "target": "METABOLIC CHANGES"}, {"weight": 1.0, "description": "Increased water consumption was a behavioral indicator observed in the diabetes phenotype of Senp1-deficient mice, possibly due to osmotic diuresis.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "DIABETES", "target": "BEHAVIORAL INDICATORS"}, {"weight": 8.0, "description": "The SENP1 gene contributes to diabetes development in mice.)", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "order": 1, "source": "DIABETES", "target": "SENP1"}, {"weight": 3.0, "description": "There is a correlation between SENP1 expression and diabetes in NOD mice.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "DIABETES", "target": "NON-OBESE DIABETIC (NOD) MOUSE"}, {"weight": 6.0, "description": "Diabetes could influence cortisol levels and the relationship with alcohol consumption based on different models analyzed.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "DIABETES", "target": "CORTISOL LEVELS"}, {"weight": 4.0, "description": "The rs3184504 polymorphism affects SH2B3 expression, which in turn impacts \u03b22M levels.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "RS3184504 POLYMORPHISM", "target": "SH2B3 EXPRESSION"}, {"weight": 5.0, "description": "SH2B3 normally regulates \u03b22M expression during hypertensive stimuli, as shown by the induction in Sh2b3\u2212/\u2212 mice.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "SH2B3", "target": "REGULATION OF \u03922M EXPRESSION"}, {"weight": 4.0, "description": "Hispanic mothers had fewer years of formal education compared to non-Hispanic White mothers.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "YEARS OF FORMAL EDUCATION", "target": "HISPANIC MOTHERS"}, {"weight": 6.0, "description": "Non-Hispanic White mothers had a lower mean years of formal education compared to Hispanic mothers and non-Hispanic White mothers.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "YEARS OF FORMAL EDUCATION", "target": "NON-HISPANIC WHITE MOTHERS"}, {"weight": 3.0, "description": "Case infants were more likely to be female than control infants and had different patterns in maternal periconceptional use of folic acid supplements compared to non-Hispanic White mothers.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "FOLIC ACID SUPPLEMENTS", "target": "CASE INFANTS"}, {"weight": 8.0, "description": "There were differences in daily dietary intake of folate across subgroups, but the variations were subtle among different race/ethnicity groups.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "DAILY DIETARY INTAKE OF FOLATE", "target": "NUTRIENT INTAKE"}, {"weight": 9.0, "description": "The composite folate intake variable showed more substantial variation within and across race/ethnicity groups.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "COMPOSITE FOLATE INTAKE VARIABLE", "target": "NUTRIENT INTAKE"}, {"weight": 5.0, "description": "Maternal age at delivery was consistent across cases and controls within maternal race/ethnicity.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "AGE RANGE", "target": "MATERNAL AGE AT DELIVERY"}, {"weight": 18.0, "description": "Hispanic infants had a higher rate of Neural Tube Defects (NTDs) compared to non-Hispanic White infants.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "HISPANIC INFANTS", "target": "NTD RATE"}, {"weight": 20.0, "description": "Infants carrying the AG genotype displayed increased risks for NTDs when maternal folate intake was low, compared to GG genotype carriers.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "HISPANIC INFANTS WITH AG GENOTYPE IN MTHFD1 SNP RS12939757", "target": "SNP OUTCOMES"}, {"weight": 22.0, "description": "Mothers carrying the CG genotype in SHMT1 had a reduced risk of NTDs when their folate intake was low, compared to CC and GG genotypes.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "GENETIC RISK FACTOR", "target": "MOTHERS CARRYING RS669340 CG GENOTYPE"}, {"weight": 24.0, "description": "The presence of the CT genotype for these two specific MTHFD1 SNPs increased infant risk for NTDs when maternal folate intake was low.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "GENETIC RISK", "target": "INFANTS WITH AG GENOTYPE IN MTHFD1 SNPS RS2236224 AND RS2236225"}, {"weight": 1.0, "description": "A significant association was found between rs10399931 and serum C-peptide concentration after 2 hours, p=0.03.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "RS10399931", "target": "SERUM C-PEPTIDE CONCENTRATION AFTER 2 HOURS"}, {"weight": 1.0, "description": "The results showed no significant associations between rs10399931 and quantitative traits related to T2D such as BMI or C-peptide concentration.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "RS10399931", "target": "T2D QUANTITATIVE TRAITS (BMI, C-PEPTIDE CONCENTRATION)"}, {"weight": 1.0, "description": "No associations were found between rs4950928 and the quantitative traits of interest in relation to T2D.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "RS4950928", "target": "T2D QUANTITATIVE TRAITS (BMI, C-PEPTIDE CONCENTRATION)"}, {"weight": 1.0, "description": "No significant associations between rs10399931 or rs4950928 and any T2D quantitative traits were found.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "RS10399931/4950928", "target": "T2D QUANTITATIVE TRAITS"}, {"weight": 2.0, "description": "Women carrying two copies of Y402H CFH risk allele had approximately 2.4 times greater odds for developing AMD compared to those with no such alleles.)<SEP>Women with two copies of the Y402H CFH allele had 2.4 times greater odds of AMD relative to women without any copies.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "Y402H CFH ALLELE"}, {"weight": 4.0, "description": "Having two copies of the A69S risk allele leads to a 2.2 times increased odds of AMD compared to zero copies.)<SEP>In the full sample, a combined effect of one or more ARMS2 risk alleles with poor lifestyle has a 1.97 times increased odds for AMD compared to zero risk alleles and high HLS.)<SEP>In women with ARMS2 risk allele, poor lifestyle significantly increased the odds of AMD up to about 1.97 times compared to individuals without any ARMS2 risk alleles and having a healthy lifestyle.)<SEP>Possessing at least one A69S risk allele was associated with about 2.2 times increased odds of AMD, relative to individuals without the risk allele.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "A69S RISK ALLELE"}, {"weight": 1.02, "description": "The G allele was associated with an increased probability of AMD.)<SEP>The G allele was found to increase the probability of developing AMD by approximately 1.08 times compared to those with other alleles.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "G ALLELE OF RS641153 (CFB/C2)"}, {"weight": 2.0, "description": "No main effect was observed for rs10033900 in relation to AMD within CAREDS.)<SEP>No significant association was observed between rs10033900 and AMD within the CAREDS dataset.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "RS10033900 (CFI)"}, {"weight": 2.0, "description": "No observation of association with rs2230199 and AMD in the CAREDS dataset.)<SEP>No statistically significant link between rs2230199 and AMD was found in the CAREDS dataset.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "RS2230199 (C3)"}, {"weight": 2.0, "description": "In women with high genetic risk for AMD, a lower HLS (indicating unhealthy dietary habits) was significantly associated with an increased likelihood of AMD compared to those with both low genetic risk and good dietary habits.)<SEP>In women with high genetic risk, HLS scores were significantly associated with an increased likelihood of AMD.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "HLS SCORE"}, {"weight": 6.6, "description": "A joint effect is observed between having both two high-risk CFH alleles and a low HLS score compared to low risk genotypes and high HLS, associated with higher odds for AMD.)<SEP>The combination of two high-risk CFH alleles and a poor HLS score was significantly associated with higher odds for AMD than individuals with low risk genotypes and healthy lifestyle scores.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "CFH GENOTYPE (CC)"}, {"weight": 4.16, "description": "For women carrying ARMS2 risk allele, poor lifestyle was associated with higher odds of AMD compared to those without the ARMS2 risk allele and maintaining a healthy diet.)<SEP>In the full sample, a significant interaction between dietary lifestyle scores and CFH genotype was observed in predicting higher odds for AMD compared to other genetic risk groups.)<SEP>In the full sample, there\'s an interaction between lifestyle factors and CFH genotype leading to increased odds of AMD compared to other genetic risk groups.)<SEP>In women carrying the ARMS2 risk allele, having poor lifestyle with HLS leads to higher odds for AMD compared to no risk alleles and high HLS.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "LIFESTYLE FACTORS (DIETARY LIFESTYLE SCORES)"}, {"weight": 2.0, "description": "The joint impact of poor lifestyle score and the high-risk CFH genotype was found to be more than additive for AMD risks, suggesting synergy in this relationship.)<SEP>The joint impact of poor lifestyle score on diet and high-risk CFH genotype was found to amplify the odds of AMD compared to having low-risk genotypes and a healthy lifestyle.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "INTERACTION EFFECT (CFH GENOTYPES AND HLS)"}, {"weight": 2.0, "description": "No influence on the association between HLS score and AMD risk, when adjusted for omega-3 fatty acid consumption.)<SEP>No significant influence on the association between HLS and AMD when adjusted for omega-3 fatty acid intake.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "\u03a9-3 FATTY ACIDS INTAKE"}, {"weight": 2.0, "description": "The combined effect of one or more ARMS2 risk alleles with poor lifestyle had an additive yet not significant 0.9 times increase in the odds for AMD compared to individuals without ARMS2 risk alleles and having a healthy lifestyle.)<SEP>The synergy of ARMS2 genetic risk with poor lifestyle was statistically significant but did not reach the level of additive effects, implying more than additive impact on AMD risks.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "SYNERGY INDEX (ARMS2 AND LIFESTYLE)"}, {"weight": 1.0, "description": "In the subsample of women with stable diets, odds for AMD were higher for those with both risk factors compared to neither.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "AMD", "target": "LIFESTYLE"}, {"weight": 1.0, "description": "The joint effect of these genotypes along with lifestyle was explored to understand if they synergize in increasing AMD risk.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "AMD", "target": "CFB/C2, C3 RS2230199, CFI RS10033900"}, {"weight": 0.5, "description": "Absence of main effects for SNPs in CFB/C2 indicates that lifestyle does not strongly interact with this gene in relation to AMD risk.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "LIFESTYLE", "target": "CFB/C2"}, {"weight": 0.5, "description": "The combination of C3 variant and HLS does not lend towards a clear evidence for interaction effect on AMD risk.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "LIFESTYLE", "target": "C3 RS2230199"}, {"weight": 0.5, "description": "SNP in this gene along with HLS had a variable effect on odds of AMD, not strong enough for clear interaction evidence.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "LIFESTYLE", "target": "CFI RS10033900"}, {"weight": 4.5, "description": "Interaction analysis revealed that while there were joint effects between lifestyle and genotypes in complement pathway genes, the estimates for synergy were unreliable.)<SEP>The combined effect of lifestyle and these genotypes was not reliable for estimating synergy due to uncertain odds distribution.)<SEP>The exploration focused on the joint effect of these genotypes along with lifestyle score in influencing AMD risk.)<SEP>The interaction analysis of these genotypes with lifestyle did not yield clear evidence for deviation from additive or multiplicative effects.)<SEP>The reliability of synergy estimates for these genotypes and HLS was low due to their combination leading to variable odds for AMD.)<SEP>There was no evidence to suggest that lifestyle significantly influences the effect of these genotypes on AMD risk.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "LIFESTYLE", "target": "CFB/C2, C3 RS2230199, CFI RS10033900"}, {"weight": 1.0, "description": "Lifestyle factors could influence the expression levels of relevant transcripts, affecting AMD risk through this mechanism.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "LIFESTYLE", "target": "TRANSCRIPTS"}, {"weight": 1.0, "description": "Lifestyle may impact the function or concentration of proteins associated with these genes, potentially affecting AMD risk.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "LIFESTYLE", "target": "PROTEINS"}, {"weight": 1.0, "description": "Lifestyle factors could alter the dynamics of molecular interactions relevant to AMD risk.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "LIFESTYLE", "target": "MOLECULAR INTERACTIONS"}, {"weight": 1.0, "description": "Lifestyle may modify or optimize these processes, which might influence AMD risk in a positive or negative way.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "LIFESTYLE", "target": "PHYSIOLOGICAL PROCESSES"}, {"weight": 1.0, "description": "Lifestyle interventions might influence these pathways, potentially affecting AMD risk through such modifications.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "LIFESTYLE", "target": "METABOLIC PATHWAYS"}, {"weight": 11.0, "description": "Nutrient X is involved in multiple physiological processes that can impact kidney function and disease risk.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "PHYSIOLOGICAL PROCESSES", "target": "NUTRIENT X"}, {"weight": 2.0, "description": "Consumption of black or sugared coffee was assessed in relation to prediabetes and type 2 diabetes risk.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "COFFEE CONSUMPTION", "target": "BLACK COFFEE/SUGARED COFFEE"}, {"weight": 6.0, "description": "A variant of coffee that is not sweetened, typically consumed as a beverage.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "BLACK COFFEE", "target": "SPECIFIC TYPE OF COFFEE/UNSWEETENED COFFEE"}, {"weight": 3.0, "description": "Stage before type 2 diabetes where blood sugar levels are higher than normal but not yet high enough for a diagnosis of diabetes.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "PREDIABETES", "target": "CONDITION/PRE-DIABETIC CONDITION"}, {"weight": 3.0, "description": "Five SNPs were selected based on their association with coffee consumption for use in calculating genetic risk scores.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "GENETIC POLYMORPHISMS", "target": "RISK SCORE CALCULATION"}, {"weight": 6.0, "description": "HS patients mostly carried genetic polymorphisms near or in the HLA-Cw6 region, except for certain patterns.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "GENETIC POLYMORPHISMS", "target": "HLA-C REGION"}, {"weight": 4.0, "description": "Associations between habitual coffee consumption and prediabetes/type 2 diabetes varied based on the subgroup analysis of coffee types (black vs. sugared).)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "ASSOCIATIONS", "target": "GWAS/COFFEESUBGROUPS/"}, {"weight": 13.0, "description": "Assessments were made between various stages of HIV progression and the control group in terms of polymorphism frequencies.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "ASSOCIATIONS", "target": "EARLY, INTERMEDIATE, ADVANCED HIV DISEASE STAGES VS. HEALTHY CONTROLS"}, {"weight": 5.0, "description": "The relationship between coffee consumption and the risk of prediabetes or type 2 diabetes was modified by genetic polymorphisms.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "GENETIC POLYMORPHISMS/COFFEE CONSUMPTION/>", "target": "INFLUENCE"}, {"weight": 5.0, "description": "A tool used to calculate an individual\'s likelihood of developing prediabetes or type 2 diabetes based on their genetic makeup.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "ASSESSMENT/EVALUATION OF GENETIC PREDISPOSITION", "target": "GENETIC RISK SCORES/"}, {"weight": 7.0, "description": "Coffee that has been sweetened with sugar or another sweetener.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "SPECIFIC TYPE OF COFFEE/SWEETENED COFFEE", "target": "SUGARED COFFEE/"}, {"weight": 1.0, "description": "Deletion of SENP1 in adipocytes enhances NEMO SUMOylation and NF-\u03baB activation, affecting cytokine expression and potentially impacting T1DM pathogenesis.)", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "order": 1, "source": "SENP1", "target": "NF-\u039aB SIGNALLING"}, {"weight": 2.0, "description": "A critical role of SENP1-dependent NF-\u03baB activity in T1DM progression in SENP1-aP2KO mice.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "SENP1", "target": "NF-\u039aB ACTIVITY"}, {"weight": 7.0, "description": "The high-fat diet caused a significant reduction of insulin levels, and this was more pronounced in Senp1 knockout (KO) mice compared to wild-type (Ctrl) mice.)", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "INSULIN LEVELS", "target": "HFD"}, {"weight": 1.0, "description": "The high-fat diet augmented fat weight in PAT depots including epididymal fat and paternal adipose tissue (PATs).)", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "PATS", "target": "HFD"}, {"weight": 3.0, "description": "The cell size of PATs was larger under normal chow conditions but smaller by a factor of 1.8 under high-fat diet (HFD).)", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "PATS", "target": "CELL SIZE"}, {"weight": 4.0, "description": "Cell numbers in PAT cells were higher in the SENP1-aP2KO mice compared to Ctrl adipocytes under normal chow and this was more pronounced under HFD conditions.)", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "PATS", "target": "CELL NUMBERS"}, {"weight": 1.0, "description": "Increased cytokine concentration is a precursor event contributing to the development of diabetes and associated myocarditis symptoms in older mice.)(entity", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "order": 1, "source": "CYTOKINE CONCENTRATION", "target": "AFFECTS"}, {"weight": 2.0, "description": "SENP1-deficient mice created with Adipoq-Cre showed a diabetic phenotype compared to control mice.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "DIABETES PHENOTYPE", "target": "GENETIC MODIFICATION"}, {"weight": 7.0, "description": "Early onset of diabetes was associated with specific deletion of Senp1 gene via different genetic modification methods.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "DIABETES ONSET", "target": "GENETIC MODIFICATION"}, {"weight": 1.0, "description": "Secreted from PATs, contribute to inflammation and immune cell infiltration in the pancreas.)", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "order": 1, "source": "CYTOKINE PRODUCTION", "target": "LOCAL PRO-INFLAMMATORY CYTOKINES"}, {"weight": 59.82, "description": "CD4+ effector T cells modulate host immune responses through cytokine production after encountering cognate antigens.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CYTOKINE PRODUCTION", "target": "CD4+ EFFECTOR T CELLS"}, {"weight": 4.0, "description": "Increased plasma levels of proinflammatory cytokines were observed in T1DM patients compared to normal subjects.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "PROINFLAMMATORY CYTOKINE", "target": "INCREASED LEVELS"}, {"weight": 5.0, "description": "Serum from SENP1-aP2KO mice, NOD mice and human T1DM all induced disruption and pancreatic cell death.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "BETA-CELL DEATH", "target": "INDUCTION"}, {"weight": 6.0, "description": "Prophylactic insulin therapy partially ameliorates development of diabetes phenotypes in SENP1-deficient mice.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "PROPHYLACTIC INSULIN THERAPY", "target": "AMELIORATION OF DIABETIC PHENOTYPES"}, {"weight": 3.0, "description": "Prophylactic insulin therapy helps partially ameliorate some diabetic phenotypes by supporting insulin levels and immune cell infiltration in mice.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "PROPHYLACTIC INSULIN THERAPY", "target": "PREVENTATIVE EFFECT"}, {"weight": 7.0, "description": "Administration of JSH-23 to SENP1-aP2KO mice diminished the effects of SENP1 deletion on blood glucose levels and insulin in blood.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "NF-\u039aB INHIBITOR II JSH-23", "target": "DIMINISHED ELEVATION OF BLOOD GLUCOSE AND REDUCTION OF INSULIN LEVELS"}, {"weight": 4.0, "description": "JSH-23 administration reduces blood glucose elevation and insulin reduction, potentially impacting the course of T1DM in SENP1-aP2KO mice.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "NF-\u039aB INHIBITOR II JSH-23", "target": "DISEASE MODIFYING EFFECT"}, {"weight": 8.0, "description": "HFD increased fat weight in all adipose depots including epididymal fat and PATs.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "HIGH-FAT DIET (HFD)", "target": "INCREASED FAT WEIGHT"}, {"weight": 5.0, "description": "HFD may stimulate adipogenesis and increase fat weight, which could contribute indirectly to inflammation and potentially exacerbate diabetic phenotypes.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "HIGH-FAT DIET (HFD)", "target": "CONTRIBUTION TO DISEASE PROGRESSION"}, {"weight": 2.0, "description": "High-fat diet significantly increased the production of proinflammatory cytokines like Interleukin-6 (IL-6), Tumor Necrosis Factor Alpha (TNF-\u03b1), and Interleukin-1 Beta (IL-1\u03b2) in PATs.)", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "HFD", "target": "IL-6, TNF-\u0391, IL-1\u0392"}, {"weight": 6.0, "description": "The onset of hyperglycemia was accelerated under HFD conditions, starting at the age of 8 weeks in SENP1-aP2KO mice instead of around 12 weeks under standard chow diet.)", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "HFD", "target": "HYPERGLYCAEMIA"}, {"weight": 8.0, "description": "Under basal conditions, liver insulin resistance was detected under HFD conditions, which was more pronounced than under normal chow diet in Senp1 knockout (KO) mice. This is a hallmark of obesity-linked type 2 diabetes development.)", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "HFD", "target": "LIVER INSULIN RESISTANCE"}, {"weight": 10.0, "description": "Unlike the adipocyte-specific deletion models, Senp1-deficient mice created by SM22\u03b1-Cre or VE-cadherin-Cre did not develop a high-glucose phenotype.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "GENETIC MODIFICATION", "target": "NORMAL PHENOTYPE"}, {"weight": 5.0, "description": "The specific cell types, such as macrophages and pancreatic islets, showed different expression patterns of Senp1 gene.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "GENE EXPRESSION", "target": "SPECIFIC CELL TYPE"}, {"weight": 5.0, "description": "The expression of PTPN2 gene was lower in individuals with heterozygous or minor allele (TC or CC) compared to those with the normal TT genotype. )", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "order": 1, "source": "GENE EXPRESSION", "target": "PTPN2:RS478582"}, {"weight": 6.0, "description": "The expression of Senp1 gene localized to peri-pancreatic adipose tissue and was not present in pancreatic islets.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "EXPRESSION", "target": "LOCALIZATION"}, {"weight": 15.0, "description": "The regulation of PPAR\u03b3 by SENP1 impacts metabolic pathways involved in insulin sensitivity and glucose metabolism.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "GENE REGULATION", "target": "METABOLIC FUNCTION"}, {"weight": 16.0, "description": "STZ was used as a method to induce diabetes in Senp1-deficient mice, providing insights into the role of SENP1 in maintaining glucose homeostasis.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "CHEMICAL TREATMENT", "target": "DIABETES INDUCTION"}, {"weight": 17.0, "description": "The impact of SENP1 deletion on the metabolic health of mice potentially leads to accelerated aging processes related to cellular senescence.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "AGING PHENOMENA", "target": "GENE FUNCTION"}, {"weight": 18.0, "description": "SENP1\'s involvement in adipocyte differentiation highlights its importance during the development of metabolic traits like obesity and diabetes.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "DEVELOPMENTAL BIOLOGY", "target": "GENE ACTIVITY"}, {"weight": 19.0, "description": "The influence of SENP1 on hematopoietic stem cells could provide insights into its role in regenerative processes during diabetes development or treatment.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "GENE INFLUENCE", "target": "REGENERATIVE MEDICINE"}, {"weight": 20.0, "description": "Senp1 gene deletion results in a complex interplay of conditions characteristic of metabolic syndrome that affects multiple organ systems.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "GENETIC PHENOTYPE", "target": "HEALTH OUTCOMES"}, {"weight": 2.0, "description": "Reduced SENP1 activity leads to increased local cytokine secretion that upregulates CCL5 expression.)", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "order": 1, "source": "REGULATION OF CCL5 EXPRESSION", "target": "SENP1 DELETION"}, {"weight": 4.0, "description": "Deficiency reduces adipocyte differentiation while increasing pro-inflammatory markers like NF-\u03baB activity, contributing to enhanced inflammation in adipose depots.)", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "order": 1, "source": "SENP1 DELETION", "target": "ENHANCED PRO-INFLAMMATORY PHENOTYPE"}, {"weight": 2.0, "description": "Increased NF-\u03baB activity in adipose tissue affects adipogenesis through reduced levels of PPAR\u03b3 SUMOylation, influencing cell development and maturation process.)", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "order": 1, "source": "NF-\u039aB ACTIVATION", "target": "REGULATION OF ADIPOCYTE DIFFERENTIATION"}, {"weight": 2.0, "description": "SENP1-dependent NF-\u03baB activity regulates the expression of genes involved in inflammation, apoptosis, and islet damage during diabetes progression.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "NF-\u039aB ACTIVITY", "target": "REGULATION OF GENE EXPRESSION"}, {"weight": 0.5, "description": "No significant association was found with ApaI (rs7975232) and risk of ovarian cancer under fixed-effect model.)", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "order": 1, "source": "APAI", "target": "OVARIAN CANCER"}, {"weight": 1.0, "description": "There was a significant association between the FokI variant and an increased risk for ovarian cancer.)", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "order": 1, "source": "FOKI", "target": "OVARIAN CANCER"}, {"weight": 1.2, "description": "These two genetic markers were found to have synergistic effects on cancer risk.)", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "order": 1, "source": "CDX-2", "target": "TAQI"}, {"weight": 3.0, "description": "The gene Cdx-2 was involved in evaluating the risk for ovarian cancer through genomic variant rs11568820.)", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "order": 1, "source": "CDX-2", "target": "OVARIAN CANCER"}, {"weight": 6.0, "description": "Gene TaqI showed no significant effect on ovarian cancer risk as per the meta-analysis and subgroup studies.)<SEP>No association was found for the TaqI variant and the risk of ovarian cancer in both subgroup studies and Caucasians.)", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "order": 1, "source": "TAQI", "target": "OVARIAN CANCER"}, {"weight": 0.7, "description": "No significant association between BsmI variant and risk of ovarian cancer was found under the dominant genetic model.)", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "order": 1, "source": "OVARIAN CANCER", "target": "BSMI"}, {"weight": 1.0, "description": "Carriers of the A allele showed significantly increased risk for ovarian cancer based on subgroup studies conducted with Japanese and NECC subgroups.)", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "order": 1, "source": "OVARIAN CANCER", "target": "RS11568820"}, {"weight": 1.3, "description": "The same genomic variant is associated with similar disease outcomes in different studies.)", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "order": 1, "source": "RS1544410", "target": "BSMI"}, {"weight": 0.93, "description": "rs731236 and rs1544410 have high linkage disequilibrium (LD) within our study population.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "RS1544410", "target": "RS731236"}, {"weight": 0.1, "description": "There is low linkage disequilibrium (LD) between rs7975232 and rs1544410 within our study population.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "RS1544410", "target": "RS7975232"}, {"weight": 5.0, "description": "The genomic variant rs11568820 in Cdx-2 gene was associated with an increased risk of ovarian cancer.)", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "order": 1, "source": "RS11568820", "target": "CDX-2 GENE VARIANT AND OVARIAN CANCER"}, {"weight": 6.0, "description": "The TaqI gene variant showed no association with ovarian cancer based on subgroup studies.)", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "order": 1, "source": "RS731236", "target": "TAQI GENE VARIANT AND OVARIAN CANCER"}, {"weight": 0.93, "description": "These two SNPs have relatively high linkage disequilibrium (LD) within our study population.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "RS731236", "target": "RS7975232"}, {"weight": 0.1, "description": "There is low linkage disequilibrium (LD) between rs731236 and rs2228570 within our study population.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "RS731236", "target": "RS2228570"}, {"weight": 2.0, "description": "The African American subgroup study contributed significantly to the heterogeneity between the studies evaluated for BsmI variant.)", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "order": 1, "source": "AFRICAN AMERICAN POPULATION", "target": "GENETIC VARIANTS"}, {"weight": 1.0, "description": "The three variants form three different haplotypes (ACC, GAT and AAC).)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "GENETIC VARIANTS", "target": "RS731236, RS7975232, RS1544410"}, {"weight": 60.0, "description": "The MTHFR gene\'s different variant types, like C677T and A1298C, are associated with various aspects of cardiovascular health.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "GENETIC VARIANTS", "target": "MTHFR C677T/A1298C"}, {"weight": 60.0, "description": "Polymorphisms in the PAI-1 gene contribute to cardiovascular disease risk, as seen with variants such as -675 4G/5G and A844G.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "GENETIC VARIANTS", "target": "PAI-1 4G/5G/A844G"}, {"weight": 60.0, "description": "SNPs in prothrombin gene (G20210A) and factor V gene (G1691A) are related to thrombosis risk and contribute to cardiovascular health.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "GENETIC VARIANTS", "target": "PROTHROMBIN G20210A/FACTOR V G1691A"}, {"weight": 2.0, "description": "Specific VDR SNP haplotypes correlate with susceptibility or resistance to T. cruzi infection.)<SEP>The VDR polymorphisms may influence the risk of infection by T. cruzi, but only rs2228570 shows a protective effect against infection.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "T. CRUZI", "target": "VDR SNPS"}, {"weight": 1.0, "description": "The minor allele, *A of rs2228570 is overrepresented in the seronegative subset.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "SERONEGATIVE", "target": "RS2228570*"}, {"weight": 1.0, "description": "The minor allele, *A of rs2228570 is underrepresented in the seropositive subset.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "SEROPOSITIVE", "target": "RS2228570*"}, {"weight": 1.0, "description": "The minor allele, *A of rs2228570 is underrepresented in the asymptomatic subset.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "ASYMPTOMATIC", "target": "RS2228570*"}, {"weight": 1.0, "description": "The VDR polymorphisms may influence the susceptibility of asymptomatic infection, rs2228570 shows a protective effect against symptomatic infection.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "ASYMPTOMATIC", "target": "VDR SNPS"}, {"weight": 1.0, "description": "The minor allele, *A of rs2228570 is overrepresented in CCC patients.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "CHAGAS DISEASE (CCC)", "target": "RS2228570*"}, {"weight": 1.0, "description": "The VDR polymorphisms may influence the susceptibility to develop CCC, and rs2228570 shows an association with resistance to CCC.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "CHAGAS DISEASE (CCC)", "target": "VDR SNPS"}, {"weight": 1.0, "description": "The single nucleotide polymorphisms (SNPs) in the VDR gene show linkage disequilibrium with each other.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "LD", "target": "VDR SNPS"}, {"weight": 1.0, "description": "Minor allele *A of rs2228570 is underrepresented among asymptomatic individuals compared to seronegative individuals.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "ASYMPTOMATIC INDIVIDUALS", "target": "RS2228570*"}, {"weight": 1.0, "description": "Minor allele *A of rs2228570 is overrepresented among seropositive individuals compared to seronegative and asymptomatic individuals.)", "source_id": "chunk-6600322cc75094b1c57707e4b9c85954", "order": 1, "source": "RS2228570*", "target": "SEROPOSITIVE INDIVIDUALS"}, {"weight": 1.0, "description": "Evidence of a positive causal effect of maternal folate levels on offspring birthweight.)", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "order": 1, "source": "MATERNAL FOLATE LEVELS", "target": "OFFSPRING BIRTHWEIGHT"}, {"weight": 5.0, "description": "Geometric mean was employed as part of the statistical analysis for better average representation.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "STATISTICAL ANALYSIS", "target": "GEOMETRIC MEANS"}, {"weight": 1.0, "description": "SNPs or haplotypes of HMIP-2 were assessed through statistical analysis for their correlations with various hemoglobin measurements.)", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "order": 1, "source": "STATISTICAL ANALYSIS", "target": "HMIP-2 SNP/HAPLOTYPE"}, {"weight": 0.469, "description": "No evidence for a causal effect of maternal B12 levels on offspring birthweight.)", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "order": 1, "source": "OFFSPRING BIRTHWEIGHT", "target": "MATERNAL B12 LEVELS"}, {"weight": 0.478, "description": "No evidence for a causal effect of fetal B12 levels on their own birthweight.)", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "order": 1, "source": "OFFSPRING BIRTHWEIGHT", "target": "FETAL B12 LEVELS"}, {"weight": 0.037, "description": "Evidence of a negative effect of fetal folate levels on their own birthweight, but the effect size was smaller than that for maternal folate.)", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "order": 1, "source": "OFFSPRING BIRTHWEIGHT", "target": "FETAL FOLATE LEVELS"}, {"weight": 0.8, "description": "No evidence of a causal relationship between maternal intake of vitamin B12 and offspring birthweight was detected.)", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "order": 1, "source": "OFFSPRING BIRTHWEIGHT", "target": "MATERNAL VITAMIN B12"}, {"weight": 0.0, "description": "Assumed no clear relation since they were not found to influence each other\'s effects on offspring birthweight.)", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "order": 1, "source": "FETAL B12 LEVELS", "target": "MATERNAL B12 LEVELS"}, {"weight": 0.7, "description": "A possible influence of maternal folate on maternal vitamin B12 levels, which could affect health outcomes including birthweight.)", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "order": 1, "source": "MATERNAL FOLATE", "target": "MATERNAL VITAMIN B12"}, {"weight": 0.4, "description": "There is no indication that levels of B12 or folate have an effect on gestational duration based on the data.)", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "order": 1, "source": "GESTATIONAL DURATION", "target": "B12/FOLATE LEVELS"}, {"weight": 53.0, "description": "Carrier rates for the MTHFR C677T variant among LV patient populations.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "MTHFR C677T", "target": "LV PATIENTS"}, {"weight": 1.0, "description": "The variant carrier rate for MTHFR C677T was 64.28% in Europe.)", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "order": 1, "source": "MTHFR C677T", "target": "VARIANT CARRIER RATE"}, {"weight": 56.0, "description": "Carrier rate of MTHFR A1298C polymorphism in LV patients.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "MTHFR A1298C", "target": "LV PATIENTS"}, {"weight": 2.0, "description": "The variant carrier rate for MTHFR A1298C was 44% in Europe.)", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "order": 1, "source": "MTHFR A1298C", "target": "VARIANT CARRIER RATE"}, {"weight": 95.0, "description": "Total carrier rate for PAI-1 variant among LV patient populations.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "PAI-1 -675 4G/5G", "target": "LV PATIENTS"}, {"weight": 3.0, "description": "The variant carrier rate for PAI-1 -675 4G/5G was 94.45% in Europe.)", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "order": 1, "source": "PAI-1 -675 4G/5G", "target": "VARIANT CARRIER RATE"}, {"weight": 18.0, "description": "Carrier rates for the PAI-1 A844G variant in LV patients.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "PAI-1 A844G", "target": "LV PATIENTS"}, {"weight": 85.0, "description": "Low carrier rate of Prothrombin G20210A among LV patients.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "PROTHROMBIN G20210A", "target": "LV PATIENTS"}, {"weight": 10.0, "description": "The variant carrier rate for Prothrombin G20210A was 55.56% in Europe.)<SEP>The variant carrier rate for Prothrombin G20210A was 75% in studies conducted in North America.)", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "order": 1, "source": "PROTHROMBIN G20210A", "target": "VARIANT CARRIER RATE"}, {"weight": 135.0, "description": "Carrier rate for Factor V G1691A variant in LV patient populations.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "FACTOR V G1691A", "target": "LV PATIENTS"}, {"weight": 12.0, "description": "The variant carrier rate for Factor V G1691A was 21.74% in Europe.)<SEP>The variant carrier rate for Factor V G1691A was 28.57% in studies carried out in North America.)", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "order": 1, "source": "FACTOR V G1691A", "target": "VARIANT CARRIER RATE"}, {"weight": 6.0, "description": "In infertile men, the use of folic acid-containing vitamins does not significantly impact semen quality compared to fertile controls.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "INFERTILE MEN", "target": "FOLIC ACID-CONTAINING VITAMINS USAGE"}, {"weight": 10.0, "description": "Smoking habits are linked with reduced semen quality and may contribute to poorer fertility outcomes among infertile men.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "INFERTILE MEN", "target": "SMOKING"}, {"weight": 1.0, "description": "Use of folic acid-containing vitamins correlates with serum folate concentration and semen quality parameters.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "SERUM FOLATE", "target": "FOLIC ACID-CONTAINING VITAMINS USAGE"}, {"weight": 5.0, "description": "The duration of abstinence prior to semen collection affects serum folate concentrations, impacting fertility and semen parameters.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "SERUM FOLATE", "target": "ABSTINENCE TIME"}, {"weight": 8.0, "description": "Lower serum folate levels are correlated with poorer semen quality specifically among infertile men, compared to those with fertility issues.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "SERUM FOLATE", "target": "INFERTILE MEN VS. FERTILE CONTROLS"}, {"weight": 12.0, "description": "Lower serum folate levels may be associated with a higher risk for infertility or poorer semen quality in men.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "SERUM FOLATE", "target": "INFERTILITY STATUS"}, {"weight": 4.0, "description": "Smoking habits can impact RCF levels, which in turn may affect the overall semen quality and fertility status.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "RCF", "target": "SMOKING"}, {"weight": 2.0, "description": "Individuals taking vitamin supplements tend to have lower tHcy levels compared to non-users, suggesting a possible role in sperm health.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "THCY", "target": "VITAMIN SUPPLEMENT USERS"}, {"weight": 11.0, "description": "The length of abstinence before semen collection can influence seminal parameters like sperm count and motility, impacting fertility assessments.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "ABSTINENCE TIME", "target": "SEMINAL PARAMETERS"}, {"weight": 7.0, "description": "Vitamin supplementation has been shown to have beneficial effects on seminal parameters such as sperm concentration and motility in this study.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "VITAMIN SUPPLEMENT USERS", "target": "SEMINAL PARAMETERS"}, {"weight": 1.0, "description": "The model-estimated relative differences compared for S-FOL and WB-FOL varied by laboratory and condition.)<SEP>The results of both indicators showed comparable behavior in terms of differences when all laboratories used CDC reagents or under a harmonized condition.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704<SEP>chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "S-FOL", "target": "WB-FOL"}, {"weight": 1.0, "description": "The analysis considered covariance across different microorganisms and calibrators.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "COVARIANCE", "target": "MICROORGANISM & CALIBRATOR"}, {"weight": 3.0, "description": "A small day effect was observed for S-FOL and WB-FOL averaged across days.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "MICROORGANISM EFFECT", "target": "OVERALL MICROORGANISM EFFECT"}, {"weight": 4.0, "description": "Large differences in calibrator effects were observed, averaging across microorganisms.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "CALIBRATOR EFFECT", "target": "OVERALL CALIBRATOR EFFECT"}, {"weight": 5.0, "description": "Estimations showed variation in mean difference to CDC MBA target values based on dietary pattern (IH reagents vs. CDC reagents).)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "CDC MBA TARGET VALUES", "target": "GROUP 1 LABORATORIES & GROUP 2 LABORATORIES"}, {"weight": 3.0, "description": "CIs highlighted the wide range of variation among different laboratories.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "95% CIS", "target": "VARIABILITY WITHIN & AMONG LABORATORIES"}, {"weight": 4.0, "description": "Logarithmic transformation was used to normalize data distributions.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "LOG TRANSFORMED RESULTS", "target": "NORMALIZATION PROCESS"}, {"weight": 6.0, "description": "High SD values indicated a larger spread or variability in S-FOL and WB-FOL concentrations.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "SD (STANDARD DEVIATION)", "target": "SPREAD OF DATA POINTS"}, {"weight": 2.0, "description": "Results under CDC-MO were closer to the target values than IH reagents results in IH-CAL conditions.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "CDC-MO & CDC-CAL", "target": "IH-CAL & IH-CAL"}, {"weight": 1.0, "description": "95% of the laboratory results were expected to be within these limits under the harmonized condition when CDC reagents were used.)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "CDC MBA TARGET VALUE", "target": "LIMITS OF AGREEMENT"}, {"weight": 1.0, "description": "Exhibited less reasonable agreement compared to other labs under harmonized condition.)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "LIMITS OF AGREEMENT", "target": "LABS #6 & 9"}, {"weight": 2.0, "description": "With the harmonized condition, it can be anticipated that the differences would amount to approximately \u00b110\u201330% for S-FOL and approximately \u00b120\u201340% for WB-FOL.)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "DIFFERENCES BETWEEN DAY 1 AND DAY 2", "target": "S-FOL & WB-FOL"}, {"weight": 3.0, "description": "Day 1 vs Day 2 analysis showed a lack of agreement in WB-FOL data, unlike other laboratories.)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "LAB #8 RESULTS (WB-FOL)", "target": "LACK OF AGREEMENT"}, {"weight": 2.0, "description": "Observed overall good agreement except for labs #6 & 9 when using CDC round-robin kit.)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "AGREEMENT UNDER HARMONIZED CONDITION", "target": "ALL LABS (WITH SAME MO & CAL)"}, {"weight": 3.0, "description": "Estimates under harmonized condition for days 1 vs. day 2 analysis.)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "DIFFERENCES EXPECTED FOR S-FOL & WB-FOL", "target": "LIMITS IN RANGE OF +/-10% TO +/-30%/<BR/> +/-20% TO +/-40%"}, {"weight": 4.0, "description": "Lab #8 showed inconsistency between day-wise analysis.)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "DAY 1 DAY 2 COMPARISON", "target": "LACK OF AGREEMENT IN WB-FOL"}, {"weight": 3.1532, "description": "Genetic analysis found that the rs1695 polymorphism is associated with an increased risk of breast cancer.)", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "order": 1, "source": "RS1695", "target": "BREAST CANCER"}, {"weight": 1.0, "description": "The presence of the rs1695 variant shows an association with CSF usage during chemotherapy, suggesting potential genetic influences on treatment responsiveness and need for supportive care.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "RS1695", "target": "GROWTH FACTORS USE (CSFS)"}, {"weight": 1.0, "description": "The rs1138272 polymorphism was not found to be associated with the risk of breast cancer in our study.)", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "order": 1, "source": "RS1138272", "target": "BREAST CANCER"}, {"weight": -228.0, "description": "A specific single nucleotide polymorphism (Ile105Val) in GSTP1 shows reduced amyloid propensity according to WALTZ output.)", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "order": 1, "source": "GSTP1 PROTEIN", "target": "ILE105VAL"}, {"weight": 1.0, "description": "Association of p.R778L with haplogroup A.)", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "order": 1, "source": "P.R778L", "target": "HAPLOGROUP A"}, {"weight": 2.0, "description": "Significantly higher frequency of p.P992L in Fujian province compared to Zhejiang.)", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "order": 1, "source": "P.P992L", "target": "ALLELE FREQUENCY COMPARISON"}, {"weight": 1.0, "description": "Highest prevalence of p.T935M variant found in Fujian province, compared to other regions.)", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "order": 1, "source": "P.T935M", "target": "REGIONAL PREVALENCE"}, {"weight": 3.0, "description": "Tendency for these two variants to occur together among a subset of patients.)", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "order": 1, "source": "P.R778L AND P.P992L", "target": "CO-OCCURRENCE"}, {"weight": 3.0, "description": "Likely association of these two variants in some WD patients, as evident from certain data.)", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "order": 1, "source": "P.R778L AND P.T935M", "target": "CO-OCCURRENCE"}, {"weight": 1.0, "description": "A1", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "order": 1, "source": "10-7-7; 10-7-8; 10-8-7; 10-8-8", "target": "HAPLOTYPE VARIANTS A1\u2013A4/"}, {"weight": 1.0, "description": "E2", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "order": 1, "source": "10-9-5; 10-9-6", "target": "HAPLOTYPE VARIANT E1/"}, {"weight": 1.0, "description": "Frequencies of Haplogroups A/B in Control Chromosomes", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "order": 1, "source": "97.0%/", "target": "3.4%"}, {"weight": 1.0, "description": "Significant linkage disequilibrium found between D13S314 marker and p.R778L and p.P992L variants.)", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "order": 1, "source": "D13S314", "target": "LINKAGE DISEQUILIBRIUM"}, {"weight": 1.0, "description": "Significant association of D13S301 with p.R778L, p.P992L and p.T935M at specific alleles.)", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "order": 1, "source": "LINKAGE DISEQUILIBRIUM", "target": "D13S301"}, {"weight": 1.0, "description": "Significant linkage disequilibrium detected between D13S316 marker and the three prevalent pathogenic variants, showing strong associations at different alleles.)", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "order": 1, "source": "LINKAGE DISEQUILIBRIUM", "target": "D13S316"}, {"weight": 1.0, "description": "rs1799853 is one of the SNPs combined into a CYP2C9 star variant group.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "RS1799853", "target": "CYP2C9 STAR VARIANT GROUP"}, {"weight": 2.0, "description": "The minor allele frequency for rs1057910 indicates reduced CYP2C9 activity.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "CYP2C9 ACTIVITY", "target": "CYP2C9*3"}, {"weight": 3.0, "description": "The minor allele frequency for rs9923231 indicates a lower minor allele frequency than other SNPs, consistent with previous studies.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "CYP2C9 ACTIVITY", "target": "CYP2C9*8"}, {"weight": 4.0, "description": "SNPs did not show deviations from Hardy Weinberg equilibrium at P<0.001)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "HWE", "target": "STATISTICAL PROCESS"}, {"weight": 6.0, "description": "The algorithms were tested using African American (AA) patients in a clinical trial.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "AA VALIDATION COHORT", "target": "CLINICAL TRIAL"}, {"weight": 5.0, "description": "Accuracy of predicting dose requirements compared with actual maintenance dose.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "ALGORITHM ACCURACY", "target": "PERFORMANCE METRIC"}, {"weight": 2.0, "description": "Genotyping assessed the TGF-\u00df1 gene for specific SNPs that might be associated with CRC risk.)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "GENOTYPING", "target": "TGF-SS1 SNPS"}, {"weight": 8.0, "description": "Different genetic backgrounds exhibit divergent DNA modifications that change over age, influencing LCT expression in enterocytes.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "GENETIC BACKGROUND", "target": "AGE-DEPENDENT REGULATION"}, {"weight": 3.0, "description": "The sample\'s characteristics were analyzed, revealing a mean age of 60 years (SD=10).)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "STATISTICAL PROCESS", "target": "SAMPLE ANALYSIS AND DESCRIPTIVE STATISTICS"}, {"weight": 1.0, "description": "CYP24A1 SNPs like rs927650 and rs35051736 have relationships with vitamin D metabolites 25(OH)D and 1,25(OH)2D.)", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "order": 1, "source": "VITAMIN D METABOLITE CONCENTRATIONS", "target": "CYP24A1 SNPS"}, {"weight": 1.0, "description": "SNPs like rs2296241 and rs2762939 in CYP27B1 show relationships with vitamin D metabolites 25(OH)D and 1,25(OH)2D.)", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "order": 1, "source": "VITAMIN D METABOLITE CONCENTRATIONS", "target": "CYP27B1 SNPS"}, {"weight": 1.0, "description": "SNPs like rs927650 and rs35051736 in CYP24A1 are associated with odds for adenoma recurrence.)", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "order": 1, "source": "ADENOMA RECURRENCE", "target": "CYP24A1 SNPS"}, {"weight": 1.0, "description": "SNP rs927650 in CYP24A1 shows a relationship with vitamin D metabolite concentration 25(OH)D.)", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "order": 1, "source": "25(OH)D CONCENTRATION", "target": "CYP24A1 SNP RS927650"}, {"weight": 1.0, "description": "SNP rs35051736 has a statistically significant relationship with odds for advanced adenoma recurrence.)", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "order": 1, "source": "ADVANCED ADENOMA RECURRENCE", "target": "CYP24A1 SNP RS35051736"}, {"weight": 1.0, "description": "SNPs like rs2296241 and rs35051736 affect the odds of recurrent adenomas with villous histology.)", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "order": 1, "source": "CYP24A1 SNPS (RS2296241, RS35051736)", "target": "VILLOUS HISTOLOGY ADENOMAS RECURRENCE"}, {"weight": 17.0, "description": "The REC8 gene\'s role in meiosis is impacted by genetic variations identified in the analysis of T1D susceptibility.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "GENETIC INFLUENCE", "target": "REC8"}, {"weight": 1.0, "description": "The wt sample is differentiated from variant amplicons using HRMA analysis.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "C.428_430DELAAG VARIANT AMPLICON", "target": "WT SAMPLE"}, {"weight": 1.0, "description": "Homozygous samples for c.428_430delAAG variant are identified through HRMA analysis.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "C.428_430DELAAG VARIANT AMPLICON", "target": "HOMOZYGOUS SAMPLE/"}, {"weight": 1.0, "description": "Heterozygous samples for c.428_430delAAG variant are identified through HRMA analysis.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "C.428_430DELAAG VARIANT AMPLICON", "target": "HETEROZYGOUS SAMPLE/"}, {"weight": 1.0, "description": "Amplicon corresponding to the c.1186C\u2009>\u2009T variant is genotyped using SNP rs114368325.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "C.1186C\u2009>\u2009T VARIANT AMPLICON", "target": "RS114368325 SNPS"}, {"weight": 1.0, "description": "The wt sample is also differentiated from variant amplicons using HRMA analysis.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "C.1186C\u2009>\u2009T VARIANT AMPLICON", "target": "WT SAMPLE"}, {"weight": 1.0, "description": "Homozygous samples for c.1186C\u2009>\u2009T variant are identified through HRMA analysis.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "C.1186C\u2009>\u2009T VARIANT AMPLICON", "target": "HOMOZYGOUS SAMPLE/"}, {"weight": 1.0, "description": "Heterozygous samples for c.1186C\u2009>\u2009T variant are identified through HRMA analysis.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "C.1186C\u2009>\u2009T VARIANT AMPLICON", "target": "HETEROZYGOUS SAMPLE/"}, {"weight": 2.0, "description": "The duplex assay performed in one test ensures full superimposability and reproducibility for genotyping samples.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "DUPLEX ASSAY", "target": "INTRA-ASSAY PRECISION"}, {"weight": 3.0, "description": "Across multiple experiments, inter-assay precision is ensured by achieving reproducible results using different samples.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "INTER-ASSAY PRECISION", "target": "TRIPLE PCR/HRM EXPERIMENTS"}, {"weight": 4.0, "description": "The sensitivity of the HRMA technique allows distinguishing variants even at lower frequencies.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "SERIAL DILUTIONS", "target": "VARIANT ALLELE FREQUENCY"}, {"weight": 1.0, "description": "The Tm values from the HRMA analysis are characteristic of different genotypes for SNPs rs777676129 and rs114368325.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "NORMALIZED TM VALUES", "target": "RS777676129 SNPS AND RS114368325 SNPS"}, {"weight": 1.0, "description": "The Tm differences allow distinguishing the two amplicons corresponding to different variants.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "NORMALIZED TM VALUES", "target": "C.428_430DELAAG VARIANT AMPLICON AND C.1186C\u2009>\u2009T VARIANT AMPLICON"}, {"weight": 1.0, "description": "The protocol is validated on further 10 DNA samples having the suspicion of idiopathic intracranial hypertension (IIH).)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "10 DNA SAMPLES WITH IIH SUSPICIONS", "target": "DUPLEX-PCR/HRMA PROTOCOL/"}, {"weight": 1.0, "description": "Specific heterozygous sample for c.428_430delAAG variant is identified through the protocol.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "DUPLEX-PCR/HRMA PROTOCOL/", "target": "1 HETEROZYGOUS C.428_430DELAAG SAMPLE/"}, {"weight": 1.0, "description": "Specific heterozygous sample for c.1186C\u2009>\u2009T variant is identified through the protocol.)", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "DUPLEX-PCR/HRMA PROTOCOL/", "target": "1 HETEROZYGOUS C.1186C\u2009>\u2009T SAMPLE/"}, {"weight": 7.0, "description": "The assessments showed normal levels indicative of healthy motor function.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "MEDIAN NERVE COMPOUND MOTOR ACTION POTENTIALS", "target": "CLINICAL NEUROPHYSIOLOGY STUDIES"}, {"weight": 8.0, "description": "This measurement was within the norm, suggesting normal neural transmission speed.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "MEDIAN NERVE CONDUCTION VELOCITY", "target": "CLINICAL NEUROPHYSIOLOGY STUDIES"}, {"weight": 9.0, "description": "Sensory assessment revealed an elevated response compared to typical infant levels.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "MEDIAN NERVE SENSORY ACTION POTENTIAL", "target": "CLINICAL NEUROPHYSIOLOGY STUDIES"}, {"weight": 10.0, "description": "Levels in cerebrospinal fluid indicated no copper overload or Menkes disease characteristics based on these ratios.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "CEREBROSPINAL FLUID COPPER AND CATECHOLAMINE RATIOS", "target": "BIOCHEMICAL FINDINGS"}, {"weight": 7.0, "description": "The wild type: mutant allele ratio was estimated using densitometry and Q-RT-PCR methods, with different levels found in blood compared to fibroblast DNA.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "WILD TYPE: MUTANT ALLELE RATIOS", "target": "COMPARATIVE ANALYSIS"}, {"weight": 5.0, "description": "The development assessment results were influenced by the presence of the P1001L genetic variant.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "CLINICAL OUTCOME", "target": "P1001L MUTATION"}, {"weight": 6.0, "description": "Tests like MRI and clinical neurophysiology were part of the diagnostic process that was affected by molecular findings.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "DIAGNOSTIC TESTS", "target": "MOLECULAR ANALYSES"}, {"weight": 5.0, "description": "The P1001L mutation was confirmed by creating a novel BsmAI restriction site.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "P1001L MUTATION", "target": "MOLECULAR ANALYSES"}, {"weight": 6.0, "description": "BsmAI digestion levels were compared between whole blood DNA and fibroblast DNA, showing differences in mosaicism.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "MOLECULAR ANALYSES", "target": "BSMAI DIGESTION LEVEL"}, {"weight": 2.0, "description": "The presence of TR1 gene is influenced by conditions with depleted selenium supply.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "SE-DEPLETED MEDIUM", "target": "TR1"}, {"weight": 5.0, "description": "Cell line IDI-C exhibits altered behavior or performance when grown in conditions deficient in selenium supply.)<SEP>Cell line IDI-C shows differences in response to selenium conditions compared to other cell types or states when grown under Se-depleted conditions.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "SE-DEPLETED MEDIUM", "target": "IDI-C CELLS"}, {"weight": 1.0, "description": "In Se-supplemented medium, the GPx4 protein levels increased significantly in IDI-C cells compared to those expressing the IDI-T transgene, suggesting a stronger antioxidant response.)", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "order": 1, "source": "SE-SUPPLEMENTED MEDIUM", "target": "EFFECT ON CELL VIABILITY/IMPACT ON ANTIOXIDANT ENZYME ACTIVITY"}, {"weight": 5.0, "description": "Selenium supplementation can have an impact on cellular viability under certain conditions, such as oxidative stress exposure.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "SE-SUPPLEMENTED MEDIUM", "target": "CELL VIABILITY"}, {"weight": 2.0, "description": "Cells treated with selenium supplementation show improved viability compared to standard growth conditions.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "SE-SUPPLEMENTED MEDIUM", "target": "CELL VIABILITY UNDER HIGH SE"}, {"weight": 3.0, "description": "There seems no direct relationship based on the provided context. GPx1 and selenoprotein H are encoded by different genes, hence they might be related in terms of selenium metabolism but not specifically indicated by the text.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "SELENOPROTEIN H MRNA", "target": "GPX1"}, {"weight": 1.0, "description": "Selenium (Se) affects GPx4 expression and function, impacting antioxidant defense mechanisms.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "SE", "target": "GPX4"}, {"weight": 6.0, "description": "The application of this chemical agent to cells may induce a state resembling cellular \'disease\' by causing oxidative damage and stress.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "CELL VIABILITY", "target": "0.5 MM TERT-BUTYLHYDROPEROXIDE CHALLENGE"}, {"weight": 9.0, "description": "The availability and concentration of selenium influence the cellular health, as evidenced by contrasting outcomes under different conditions.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "CELL VIABILITY", "target": "SELENIUM (SE)"}, {"weight": 10.0, "description": "Indications of cell damage or dysfunction that could be related to disease states when cells are exposed to high levels of reactive oxygen species (ROS).)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "CELL VIABILITY", "target": "DISEASE CONTEXT IN TERMS OF OXIDATIVE STRESS"}, {"weight": 4.0, "description": "Selenium availability is directly correlated with the level of cell viability and functional capacity under various experimental conditions.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "CELL VIABILITY", "target": "SE SUPPLY"}, {"weight": 3.0, "description": "The oxidative stress induced by tert-butylhydroperoxide affects cell health, impacting factors like apoptosis or cell cycle progression.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "0.5 MM TERT-BUTYLHYDROPEROXIDE CHALLENGE", "target": "RESPONSE OF CELLS"}, {"weight": 7.0, "description": "Different levels of selenium availability affect the behavior and response of IDI-C and IDI-T cell lines, impacting viability and sensitivity to oxidative challenges.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "SE SUPPLY", "target": "IDI-C CELLS AND IDI-T CELLS"}, {"weight": 6.0, "description": "The presence and supply of selenium influence the effectiveness of various cellular defense systems against oxidative stress.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "SELENIUM (SE)", "target": "CELLULAR DEFENSE MECHANISMS"}, {"weight": 2.0, "description": "The transfected cells showed lower viability over a range of tert-butylhydroperoxide concentrations with 40\u201360% viability when challenged with 1\u20135 mM tert-butylhydroperoxide.)", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "order": 1, "source": "SE-DEPLETED MEDIUM/", "target": "|>OXIDATIVE CHALLENGE/SE SUPPLY CONDITIONS"}, {"weight": 4.0, "description": "In Se-depleted conditions, GPx activity was lower compared to that in Se-supplemented cells.)", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "order": 1, "source": "SE-DEPLETED MEDIUM/", "target": "|>GPX ACTIVITY/IMPACT ON SELENOPROTEINS"}, {"weight": 5.0, "description": "When sufficient Se is present, the IDI-C cell line showed a greater increase in GPx4 content than the IDI-T cell line when grown under Se-supplemented conditions.)", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "order": 1, "source": "SE SUPPLY CONDITIONS/", "target": "|>GPX4 PROTEIN LEVELS"}, {"weight": 8.0, "description": "Cells exposed to very low concentrations of selenium show decreased viability compared to those grown in conditions with adequate selenium supply.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "0.5 NM SE OR 1 NM SE", "target": "CELL VIABILITY UNDER LOW SELENIUM CONDITIONS"}, {"weight": 5.0, "description": "Lower concentrations of selenium have distinct effects on cells, often leading to different outcomes compared to adequate selenium levels.)", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "order": 1, "source": "0.5 NM SE OR 1 NM SE", "target": "IMPACT ON CELLULAR RESPONSES"}, {"weight": 1.0, "description": "Eight mutants expressed at significantly lower levels than wild-type and five of these (p.Trp421Ser, p.Ser498Pro, p.Thr553Asn, p.Ala557Thr, p.Ser572Trp) expressed at <50% of wild-type.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "CHAT MUTATIONS", "target": "EXPRESSION LEVEL"}, {"weight": 3.0, "description": "EBNA2 altered the allelic association with expression compared to that found in B cells.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "EXPRESSION LEVEL", "target": "ALLELIC ASSOCIATION"}, {"weight": 2.0, "description": "Ten mutations alter one or more rate constants of ChAT activation and two mutations compromise the thermal stability of the mutant protein.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "KINETIC PARAMETERS", "target": "MISSENSE MUTATIONS"}, {"weight": 5.0, "description": "Mutations like p.Ala557Thr compromise the thermal stability of ChAT.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "MISSENSE MUTATIONS", "target": "IMPACT ON PROTEIN STABILITY"}, {"weight": 4.0, "description": "Compared the expression levels of wild-type CHAT versus mutant CHATs.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "COMPARISON WITH MUTANT CHATS", "target": "WILD-TYPE CHAT"}, {"weight": 6.0, "description": "The study highlights the importance of conserved residues like Met202, Trp421, Thr553, Ala557 among canine, murine, rat, chicken, Drosophila, and C. elegans ChAT proteins.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "CONSERVATION ACROSS SPECIES", "target": "RESIDUES"}, {"weight": 7.0, "description": "The presence of specific mutations like p.Ala557Thr in multiple patients indicates their influence on the phenotype.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "MUTATIONS", "target": "PHENOTYPIC IMPACT"}, {"weight": 1.0, "description": "The CpG site in BCL2L11 was associated with the SNP rs726430.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "BCL2L11", "target": "GENOMIC VARIANT (RS726430)"}, {"weight": 2.0, "description": "The CpG site in CEP55 was linked to the SNP rs12782691.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "CEP55", "target": "GENOMIC VARIANT (RS12782691)"}, {"weight": 3.0, "description": "The CpG site associated with CPB1 had a connection with rs16861015 SNP.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "CPB1", "target": "GENOMIC VARIANT (RS16861015)"}, {"weight": 6.0, "description": "Methylation levels at this CpG were linked to heart failure.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "ZNF341", "target": "CPG SITE (CG11842944)"}, {"weight": 4.0, "description": "Methylation at this CpG site was associated with T2D.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "HIST1H4D 3\'-UTR", "target": "CPG SITE (CG08170869)"}, {"weight": 5.0, "description": "This CpG site within DVL1 showed an association with the risk of developing T2D.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "DVL1 BODY", "target": "CPG SITE (CG03465880)"}, {"weight": 2.0, "description": "NADH is a key component in the electron transport chain.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "NADH", "target": "FUNCTIONING"}, {"weight": 3.0, "description": "FMN binds to proteins like iron\u2013sulfur clusters for proper functioning.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "FMN", "target": "BINDING"}, {"weight": 4.0, "description": "Iron\u2013sulfur clusters are part of several enzymes, including the protein mentioned.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "IRON\u2013SULFUR CLUSTER", "target": "PROTEIN FUNCTION"}, {"weight": 6.0, "description": "Mutation changes serine (Ser) to aspartic acid (Asp) at position 146 in COQ2 gene.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "C.437G>A", "target": "MUTATION"}, {"weight": 9.0, "description": "Mutations in MT-ND1 can lead to disorders like Leber optic atrophy, mitochondrial complex I deficiency, or MELAS.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "MT-ND1 GENE", "target": "MUTATION"}, {"weight": 5.0, "description": "The mutation could impact NADH binding to proteins.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "BINDING", "target": "GLU246LYS MUTATION"}, {"weight": 7.0, "description": "Mutation causes arginine (Arg) change to a premature stop codon at amino acid position 387.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "MUTATION", "target": "C.1159C>T"}, {"weight": 6.0, "description": "Individuals with the CT genotype consumed more drinks on a drinking day when treated with placebo compared to olanzapine.)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "DRINKS PER DRINKING DAY", "target": "PLACEBO VS OLANZAPINE"}, {"weight": 1.0, "description": "There was a significant main effect of ALDH2 genotype on latency to first drink, with Asn40 homozygotes having shorter latencies.)", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "order": 1, "source": "ALDH2 GENOTYPE", "target": "LATENCY TO FIRST DRINK"}, {"weight": 1.0, "description": "Topiramate and placebo were used in the study, with topiramate showing more efficacy against HDDs compared to placebo (NNT = 5.29).)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "PLACEBO", "target": "TOPIRAMATE"}, {"weight": 1.0, "description": "The Asn40 homozygote group, when administered the placebo, demonstrated unique responses to alcohol-induced stimulation.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE/", "target": "ASN40 HOMOZYGOTES"}, {"weight": 70.0, "description": "Environmental Factors/>Medications\' combined effects with OPRM1 gene variants on alcohol-induced responses were considered when looking at external factors.)<SEP>Environmental Factors/>The combined effect of certain medications with OPRM1 gene variants played a key role in evaluating alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The combined effect of certain medications with OPRM1 gene variants significantly influenced evaluations of alcohol responses while taking into account external influences.)<SEP>Environmental Factors/>The combined impact of certain medications with OPRM1 gene variants played a key role in evaluating alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The combined influence of certain medications with OPRM1 gene variants played an important role in evaluating alcohol responses while taking into account environmental influences.)<SEP>Environmental Factors/>The combined influence of certain medications with OPRM1 gene variants significantly shaped evaluations of alcohol responses while taking into account environmental influences.)<SEP>Environmental Factors/>The combined influence of medications with OPRM1 gene variants shaped the analysis of alcohol responses when considering external factors.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE/", "target": "MEDICATION \u00d7 GENOTYPE INTERACTION/"}, {"weight": 70.0, "description": "Environmental Factors/>The combined effect of BrAC levels with different medications and OPRM1 gene variants influenced assessments related to alcohol responses when examining environmental influences.)<SEP>Environmental Factors/>The combined effect of differing BrAC levels with specific medications and OPRM1 gene variants significantly influenced evaluations of alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The combined effect of differing BrAC levels with specific medications and OPRM1 gene variants was crucial for assessing alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The combined effect of varying BrAC levels with specific medications and OPRM1 gene variants significantly influenced evaluations of alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The combined influence of varying BrAC levels with specific medications and OPRM1 gene variants significantly influenced evaluations of alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The combined influence of varying BrAC levels with specific medications and OPRM1 gene variants was pivotal in assessing alcohol responses while considering environmental influences.)<SEP>Environmental Factors/>The intricate relationship among BrAC levels, various medications, and OPRM1 gene variants influenced evaluations related to alcohol responses.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE/", "target": "BRAC \u00d7 MEDICATION \u00d7 GENOTYPE INTERACTION/"}, {"weight": 70.0, "description": "Stimulation/>The Asn40 homozygote group, when receiving the placebo, showed distinct responses related to alcohol-induced stimulation compared to others.)<SEP>Stimulation/>The effect of being an Asn40 homozygote with exposure to the placebo led to differentiated responses in alcohol-induced stimulation when compared to others.)<SEP>Stimulation/>The effect of being an Asn40 homozygote, when exposed to a placebo condition, led to differentiated responses in alcohol-induced stimulation relative to others.)<SEP>Stimulation/>The effect of being an Asn40 homozygote, when exposed to a placebo condition, led to distinctive responses in alcohol-induced stimulation relative to others.)<SEP>Stimulation/>The influence of being an Asn40 homozygote, when exposed to a placebo condition, led to distinctive responses in alcohol-induced stimulation relative to others.)<SEP>Stimulation/>The influence of being an Asn40 homozygote, when in the placebo condition, led to distinctive alcohol-induced stimulation responses relative to others.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "GENE/", "target": "ASN40 HOMOZYGOTES + PLACEBO GROUP/"}, {"weight": 2.0, "description": "The analysis found no significant medication \u00d7 genotype interaction regarding the effects on alcohol self-administration or latency to first drink consumption.)", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "order": 1, "source": "GENETIC FACTOR", "target": "MEDICATION EFFECTS"}, {"weight": 5.0, "description": "Treatments of OPRM1 genotype as a three-level variable did not significantly affect the reported pharmacogenetic outcomes related to alcohol self-administration and latency to drink.)", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "order": 1, "source": "GENETIC FACTOR", "target": "PHARMACOGENETIC RESULTS"}, {"weight": 1.0, "description": "There\'s an association between ALDH2 rs671 genotypes and daily alcohol consumption, impacting the pattern of drinks per day.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "ALDH2 RS671 GENOTYPES", "target": "DAILY DRINKS"}, {"weight": 2.0, "description": "The ALDH2 rs671 genotypes affect cortisol levels differently between men and women, influenced by the sex-specific patterns of genotypes.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "ALDH2 RS671 GENOTYPES", "target": "CORTISOL LEVELS"}, {"weight": 5.0, "description": "Hypertension, as a confounding variable, may affect cortisol levels when considering alcohol consumption in linear regression models.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "CORTISOL LEVELS", "target": "HYPERTENSION"}, {"weight": 3.0, "description": "The rs671 polymorphism affects cortisol levels due to its influence on alcohol metabolism, which can impact the body\'s stress response.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "CORTISOL LEVELS", "target": "RS671 POLYMORPHISM"}, {"weight": 7.0, "description": "Oxidative stress contributes to the development and progression of hypertension through various cellular mechanisms.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "HYPERTENSION", "target": "OXIDATIVE STRESS"}, {"weight": 5.0, "description": "The MR technique was used to assess potential causal links between genetically predicted alcohol intake levels and changes in blood cortisol levels.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "MR (MENDELIAN RANDOMIZATION)", "target": "ASSOCIATION BETWEEN ALCOHOL CONSUMPTION AND BLOOD CORTISOL LEVEL"}, {"weight": 4.0, "description": "Increased triglyceride levels were observed in the HUA group.)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "HUA", "target": "TG"}, {"weight": 5.0, "description": "Participants with Hyperuricemia showed higher LDL levels than those without.)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "HUA", "target": "LDL"}, {"weight": 7.0, "description": "A higher percentage of hypertensive patients were identified among those with Hyperuricemia (HUA).)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "HUA", "target": "HYPERTENSIVE PATIENTS"}, {"weight": 8.0, "description": "The usage ratio of diuretics was not significantly different between the HUA cases and controls.)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "HUA", "target": "DIURETICS"}, {"weight": 9.0, "description": "No significant difference in Fasting Blood Glucose (FBG) levels was observed between HUA cases and controls.)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "HUA", "target": "FBG"}, {"weight": 13.0, "description": "The study found that HUA cases showed a significant association with the risk alleles of ABC-related genes, as well as an increased sUA level and HUA risk.)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "HUA", "target": "ABC"}, {"weight": 0.0, "description": "Increased sUA level is associated with HUA risk among rs2231142 T allele carriers.)", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "order": 1, "source": "HUA", "target": "SUA"}, {"weight": 0.0, "description": "Increased sUA levels are associated with enhanced cardiovascular risks among rs2231142 T allele carriers.)", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "order": 1, "source": "SUA LEVEL", "target": "CARDIOVASCULAR RISK FACTORS"}, {"weight": 0.0, "description": "Carriers of the rs2231142 T allele are associated with cardiovascular risk factors, including sUA levels and HUA risk.)", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "order": 1, "source": "RS2231142 T ALLELE", "target": "CARDIOVASCULAR RISK FACTORS"}, {"weight": 0.0, "description": "Abnormal hematocrit may indicate underlying conditions that can influence cardiovascular health and risk factors.)", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "order": 1, "source": "HEMATOCRIT", "target": "CARDIOVASCULAR RISK FACTORS"}, {"weight": 14.0, "description": "The total content of bile acids in feces showed a strong positive correlation with serum cholesterol levels, suggesting a relationship between dietary factors and bile acid metabolism.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "CHOLESTEROL", "target": "TOTAL STOOL BA"}, {"weight": 1.0, "description": "Genetic Risk Score", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "order": 1, "source": "GENETIC VARIANT/", "target": "SLC2A9 RS3733951/ABCG2 RS2231142/SLC22A12 RS893006"}, {"weight": 2.0, "description": "Environmental Factor/Lifestyle Factor/<SEP>Environmental Factor/", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "order": 1, "source": "GENETIC VARIANT/", "target": "ENVIRONMENTAL FACTOR/CARDIOVASCULAR RISK FACTOR"}, {"weight": 5.0, "description": "sUA level/", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "order": 1, "source": "GENETIC VARIANT/", "target": "ENVIRONMENTAL FACTOR/CARDIOVASCULAR RISK FACTOR/"}, {"weight": 1.0, "description": "Metabolic Pathway/", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "order": 1, "source": "SUA LEVEL/SLC2A9 RS3733951/ABCG2 RS2231142/", "target": "SLC22A12 RS893006"}, {"weight": 1.0, "description": "Physiological Process", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "order": 1, "source": "METABOLIC PATHWAY/", "target": "SUA LEVEL/RS13206634"}, {"weight": 1.0, "description": "Physiological Process/", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "order": 1, "source": "METABOLIC PATHWAY/", "target": "SUA LEVEL/RS7509837"}, {"weight": 2.0, "description": "The eGFR decreasing allele is associated with increased risk of CKD in individuals.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "EGFR", "target": "CKD"}, {"weight": 3.0, "description": "A decrease in eGFR is also a defining characteristic for the occurrence of ESRD, linking kidney function measures to disease outcomes.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "EGFR", "target": "ESRD"}, {"weight": 5.0, "description": "Type 1 Diabetes similarly affects kidney health through diabetic nephropathy, which can be reflected in changes to eGFR.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "T1D", "target": "KIDNEY FUNCTION"}, {"weight": 4.0, "description": "T1D development involves changes in expression of IRF1 and STAT1, which might be linked through a common causal variant with rs181206G.)", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "order": 1, "source": "T1D", "target": "CELLULAR MECHANISM"}, {"weight": 8.0, "description": "Mutations or variations in SLC34A1 may alter phosphate reabsorption, impacting kidney function.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "SLC34A1", "target": "PHOSPHATE REABSORPTION"}, {"weight": 10.0, "description": "While primarily associated with vision, changes in DCDC5-MPPED2 may indirectly influence kidney health via shared signaling pathways or genetic networks.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "DCDC5-MPPED2", "target": "RETINAL DEVELOPMENT"}, {"weight": 12.0, "description": "The dietary pattern Y may provide protective benefits against kidney disease through specific nutrients and antioxidants.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "DIETARY PATTERN Y", "target": "RISK OR PROTECTION"}, {"weight": 3.0, "description": "The genomic variant rs35923643 was found to have potential binding sites for transcription factors in a 700bp region as determined by ChIP-seq experiments.)", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "order": 1, "source": "CHIP-SEQ BINDING SITES", "target": "RS35923643"}, {"weight": 5.0, "description": "The potential use of histone deacetylase inhibitors as treatments for kidney disease)", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "order": 1, "source": "HISTONE DEACETYLASE INHIBITORS", "target": "POTENTIAL TREATMENT"}, {"weight": 4.0, "description": "There are significant effects on posterior probability for variants near HDAC8 binding sites)", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "order": 1, "source": "CHIP-SEQ BINDING SITES FOR HDAC8", "target": "SIGNIFICANT EFFECTS"}, {"weight": 6.0, "description": "The SNP Z is significantly associated with a decrease in eGFR and an increased risk of kidney disease.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "KIDNEY FUNCTION", "target": "SNP Z"}, {"weight": 7.0, "description": "Variants in the HFE gene affect iron metabolism, which can impact kidney health through oxidative stress and inflammation.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "HFE GENE", "target": "IRON METABOLISM"}, {"weight": 1.0, "description": "lead SNPs at certain loci contribute significantly to the posterior probability of driving association signals.)", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "order": 1, "source": "LEAD SNPS", "target": "POSTERIOR PROBABILITY"}, {"weight": 2.0, "description": "Ancestral orthologs for both genes are present in Drosophila melanogaster)", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "order": 1, "source": "NFATC1 GENOMIC VARIANT", "target": "RGS14 GENOMIC VARIANT"}, {"weight": 6.0, "description": "The lead SNPs at NFATC1 locus overlaps with DHSs in renal proximal tubular epithelial cells (RPTECs) and renal cortical epithelial cells (HRCEs))", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "order": 1, "source": "NFATC1 GENOMIC VARIANT", "target": "REGULATORY CHROMATIN STATE IN RPTECS&HRCES"}, {"weight": 7.0, "description": "Reduction in expression of the RGS14 ortholog loco results in longer lifespan and stress resistance)", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "order": 1, "source": "RGS14 GENOMIC VARIANT", "target": "LONGEVITY & STRESS RESISTANCE"}, {"weight": 3.0, "description": "Variants mapping to non-coding sequence at these loci account for more than 99.4% of the probability of driving eGFR association.)", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "order": 1, "source": "ASSOCIATION SIGNALS", "target": "KIDNEY DHSS"}, {"weight": 8.0, "description": "Patients with domestically acquired campylobacteriosis were more likely to receive antimicrobial treatment compared to those with travel-related infections.)", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "order": 1, "source": "DOMESTICALLY ACQUIRED INFECTIONS", "target": "TREATMENT"}, {"weight": 6.0, "description": "Quinolone-resistant isolates showed decreased susceptibility to fluoroquinolones compared to primary model, but sample size was lower resulting in increased confidence intervals.)", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "order": 1, "source": "ANTIMICROBIAL DRUG TREATMENT", "target": "QUINOLONE-RESISTANT ISOLATES"}, {"weight": 2.0, "description": "Foreign travel was associated with a higher proportion of quinolone-resistant Campylobacter infections compared to domestic infections.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "QUINOLONE RESISTANCE", "target": "TRAVEL"}, {"weight": 3.0, "description": "Of the three erythromycin-resistant isolates, two were also quinolone-resistant and originated from travelers returning to Denmark from Spain and Portugal.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "QUINOLONE RESISTANCE", "target": "C. COLI"}, {"weight": 4.0, "description": "Travel to certain regions like southern Europe and Southeast Asia was significantly associated with high prevalence of quinolone-resistant Campylobacter infections.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "QUINOLONE RESISTANCE", "target": "DESTINATION"}, {"weight": 6.0, "description": "Foreign travel, especially to specific destinations like Spain and Portugal, significantly increased the risk of acquiring a quinolone-resistant Campylobacter infection.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "QUINOLONE RESISTANCE", "target": "FOREIGN TRAVEL"}, {"weight": 8.0, "description": "Certain destinations like southern Europe and Southeast Asia were associated with higher proportions of quinolone-resistant Campylobacter infections than others.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "QUINOLONE RESISTANCE", "target": "TRAVEL DESTINATIONS"}, {"weight": 9.0, "description": "Swimming was identified as an independent risk factor for acquiring a quinolone-resistant C. jejuni infection, though the exact mechanism is unclear.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "QUINOLONE RESISTANCE", "target": "SWIMMING"}, {"weight": 3.0, "description": "A patient was found to have two C. jejuni strains with identical serotype and RiboGroup, but different susceptibility patterns towards quinolones due to a Thr-86 to Ala-86 mutation in gyrA.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "QUINOLONE RESISTANCE", "target": "C. JEJUNI STRAIN CO-INFECTION"}, {"weight": 5.0, "description": "The mean duration of illness was longer for patients infected by quinolone-resistant C. jejuni compared to those infected by quinolone-sensitive strains.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "C. JEJUNI", "target": "DURATION OF ILLNESS"}, {"weight": 7.0, "description": "C. jejuni infections did not differ in severity when comparing resistant to sensitive strains, but there was a difference in the mean duration of illness.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "C. JEJUNI", "target": "QUINOLONE SENSITIVITY/RESISTANCE"}, {"weight": 10.0, "description": "Eating fresh poultry other than chicken and turkey was associated with an increased risk of quinolone-resistant infections.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "QUINOLONE SENSITIVITY/RESISTANCE", "target": "FRESH POULTRY"}, {"weight": 1.0, "description": "The RiboGroups of C. coli isolates play a role in identifying the different susceptibilities to quinolones among the strains.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "C. COLI STRAINS", "target": "QUINOLONE SUSCEPTIBILITY"}, {"weight": 2.0, "description": "The serotypes of C. jejuni isolates can influence their sensitivity patterns against quinolones.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "QUINOLONE SUSCEPTIBILITY", "target": "C. JEJUNI STRAINS"}, {"weight": 7.0, "description": "The origin of food isolates was traced back to various sources including Danish-bred food animals, imported foods from France, Italy, and the United Kingdom.)", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "order": 1, "source": "FOOD ISOLATES", "target": "ORIGIN"}, {"weight": 9.0, "description": "Human isolates of Campylobacter bacteria were compared with isolates obtained from retail food products (chicken, turkey, pork), chicken fecal samples, and broiler chickens.)", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "order": 1, "source": "RETAIL FOOD PRODUCTS AND BROILER CHICKENS", "target": "SOURCE"}, {"weight": 3.0, "description": "Each unit increase in TAS score is associated with an estimated 0.20 points increase in social disinhibition scores.)", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "order": 1, "source": "TAS", "target": "SOCIAL DISINHIBITION"}, {"weight": 4.0, "description": "A participant\'s overall DAS score increases their social disinhibition score by 0.45 points for each unit increase.)", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "order": 1, "source": "DAS", "target": "SOCIAL DISINHIBITION"}, {"weight": 2.0, "description": "A 1-point increase in linguistic acculturation score leads to a 0.30 point increase in social disinhibition scores.)", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "order": 1, "source": "LINGUISTIC ACCULTURATION", "target": "SOCIAL DISINHIBITION"}, {"weight": 5.0, "description": "PC3 accounted for additional variance related to social disinhibition among study participants.)", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "order": 1, "source": "EIGENVALUE 3", "target": "PRINCIPAL COMPONENT 3"}, {"weight": 2.0, "description": "Gender and parental education were not significantly associated with social disinhibition score.)", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "order": 1, "source": "SOCIAL DISINHIBITION SCORE", "target": "GENDER AND PARENTAL EDUCATION"}, {"weight": 3.0, "description": "The minor alleles for these SNPs were found to contribute positively to the risk of developing a higher social disinhibition score.)", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "order": 1, "source": "SNPS (RS1998220, RS9534511, AND RS4938056)", "target": "INCREASED SOCIAL DISINHIBITION SCORE"}, {"weight": 4.0, "description": "Negative Effects on Social Disinhibition", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "order": 1, "source": "DECREASED SOCIAL DISINHIBITION SCORE/", "target": "RS16116 & RS1003921"}, {"weight": 5.0, "description": "The rs2023239 variant influences CB1 receptor levels, affecting neurocircuitry related to drug-seeking behavior.)", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "RS2023239", "target": "CB1 RECEPTOR"}, {"weight": 1.0, "description": "The genotypic variant rs2023239 was related to the response to different medications during alcohol consumption studies.)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "RS2023239", "target": "MEDICATION: OLANZAPINE AND CYPROHEPTADINE HYDROCHLORIDE"}, {"weight": 1.0, "description": "Anxiety/", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "CANNABINOID RECEPTOR 1/", "target": "CB1 RECEPTOR/"}, {"weight": 10.0, "description": "Anxiety<|Genetic variations in CNR1 may impact responses to alcohol, influencing anxiety levels.)", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "ALCOHOL CONSUMPTION/", "target": "CNR1 GENOTYPE"}, {"weight": 2.0, "description": "Individuals with the CT genotype were compared against those without the variant (TT) in relation to their responses to alcohol and medications.)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "CT GENOTYPE", "target": "GENOTYPE: TT VS CT AND CC"}, {"weight": 3.0, "description": "The response of individuals with the CT genotype was shown to be more rewarding under olanzapine treatment compared to a control medication.)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "REWARD", "target": "MEDICATION: OLANZAPINE"}, {"weight": 4.0, "description": "Individuals with the CT genotype reported significantly higher feelings of euphoria when treated with cyproheptadine hydrochloride compared to olanzapine.)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "SELF-REPORTED HIGH", "target": "ACTIVE CONTROL MEDICATION (CYPROHEPTADINE) VS OLANZAPINE"}, {"weight": 3.0, "description": "The CNR1 gene variant influenced the response to medication when combined with specific genotypes and cues in relation to craving.)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "CNR1 VARIATION (RS2035618)", "target": "MEDICATION \u00d7 GENOTYPE \u00d7 CUE INTERACTION"}, {"weight": 7.0, "description": "Disease<|Genetic variations in CNR1 can affect CB1 receptor levels, which are associated with addiction and other diseases.)", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "CB1 RECEPTOR LEVELS/", "target": "CNR1 GENE"}, {"weight": 8.0, "description": "Disease<|Exposure to alcohol cues can activate neurocircuitry that modulates drug-seeking behavior, possibly related to addictive disorders.)", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "ALCOHOL CUE EXPOSURE", "target": "NEUROCIRCUITRY ACTIVATION/"}, {"weight": 9.0, "description": "Alcohol Use<|The rs2023239 variant influences CB1 receptor function and its sensitivity to alcohol.)", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "CB1 RECEPTOR REGULATION/", "target": "RS2023239 VARIANT"}, {"weight": 1.0, "description": "Anxiety/", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "RS2023239 VARIANT/", "target": "CB1 RECEPTOR REGULATION/"}, {"weight": 1.0, "description": "Alcohol Consumption/", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "EXON 3/", "target": "CANNABINOID RECEPTOR REGULATION/"}, {"weight": 1.0, "description": "Risk Factor/", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "GENOMIC VARIANT/", "target": "ALTERNATIVE PROMOTER/"}, {"weight": 2.0, "description": "Function", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "LIPOPROTEIN LIPASE (LPL)", "target": "ROLE/"}, {"weight": 3.0, "description": "An effective treatment strategy to manage symptoms and complications associated with this genetic disorder.)", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "DIETARY FAT RESTRICTION", "target": "TREATMENT FOR LPL DEFICIENCY"}, {"weight": 4.0, "description": "Medical Intervention", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "ENFAPORT LIPIL", "target": "FORMULA/"}, {"weight": 5.0, "description": "Genetic Connection", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "AUTOSOMAL RECESSIVE DISORDER/", "target": "LPL DEFICIENCY"}, {"weight": 6.0, "description": "Diagnostic Method", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "MUTATION ANALYSIS/", "target": "GENOMIC DNA"}, {"weight": 7.0, "description": "Association between symptoms and conditions", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "CLINICAL SIGNS/", "target": "PATHOLOGICAL MANIFESTATIONS"}, {"weight": 8.0, "description": "High triglyceride levels", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "GASTROINTESTINAL COMPLICATIONS", "target": "OUTCOME OF LPL DEFICIENCY/"}, {"weight": 3.0, "description": "The rs2832407 variant moderates the effect of topiramate on HDDs, showing a significant difference between C-homozygotes and A-allele carriers.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "RS2832407", "target": "TOPIRAMATE EFFECT"}, {"weight": 3.0, "description": "Individuals carrying at least one copy of the A allele show different responses to topiramate compared to those who are homozygous for the C allele (higher severity of adverse events).)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "RS2832407", "target": "A-ALLELE CARRIERS"}, {"weight": 2.0, "description": "The study aims to determine the effectiveness of topiramate and placebo in managing HDDs.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "HDDS", "target": "EFFICACY"}, {"weight": 5.0, "description": "A-allele carriers showed a significantly higher mean severity rating for adverse events compared to those with other genotypes when taking topiramate.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "A-ALLELE CARRIERS", "target": "ADVERSE EVENTS"}, {"weight": 4.0, "description": "C-homozygotes showed a significant reduction in HDDs when taking topiramate, with an NNT of 2.28.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "C-HOMOZYGOTES", "target": "EFFECT"}, {"weight": 2.0, "description": "Genotyping was conducted as part of data collection for the study with 93 participants.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "GENOTYPING AND DATA COLLECTION", "target": "STUDY METHODS"}, {"weight": 4.0, "description": "Patients in the study were predominantly female (55%) and had various malignancies.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "PATIENT CHARACTERISTICS", "target": "POPULATION DESCRIPTION"}, {"weight": 5.0, "description": "The study involved patients diagnosed with gastrointestinal, thoracic, or breast cancers who received highly-emetogenic chemotherapy.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "HEALTH CONDITION AND TREATMENT", "target": "MALIGNANCY TYPES AND CHEMOTHERAPY"}, {"weight": 7.0, "description": "Risk factors for CINV such as age, sleep hours, and morning sickness were linked to the severity of nausea.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "CINV SEVERITY AND RISK FACTORS", "target": "SYMPTOM ASSESSMENT AND FACTORS"}, {"weight": 8.0, "description": "The genetic variation in ABCB1 was found to be associated with more severe acute nausea compared to other genotypes.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "ASSOCIATION WITH SYMPTOM", "target": "GENETIC RISK FACTOR (ABCB1 RS1045642)"}, {"weight": 9.0, "description": "Demographic, clinical, and patient-reported factors explained 13% of the variance in acute nausea; ABCB1 genotype contributed an additional variance (p=0.10).)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "ANALYSIS RESULTS", "target": "STATISTICAL SIGNIFICANCE AND VARIANCE"}, {"weight": 2.0, "description": "Using Bonferroni correction when analyzing multiple CpG sites to prevent false positives in identifying significant differences.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "BONFERRONI CORRECTION", "target": "DNA MODIFICATION PROFILE"}, {"weight": 3.0, "description": "Certain DNA modifications may influence the regulatory functions of LCT gene leading to lactase non-persistence.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "DISTINCT REGION", "target": "LACTASE NON-PERSISTENCE"}, {"weight": 2.0, "description": "The situation with MCM6 exon 16 shows significant differences between C-13910 and T-13910 carrying haplotypes, likely similar to the situation of MCM6 intron 13 in that DNA modifications are erased at embryogenesis.)", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "order": 1, "source": "LACTASE NON-PERSISTENCE", "target": "DNA MODIFICATIONS"}, {"weight": 5.0, "description": "The maps showing DNase sensitivity sites overlap with regions of interest for LCT gene modifications, suggesting active regulatory elements.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "DNASE I HYPERSENSITIVITY SITES", "target": "CHROMATIN REFERENCE MAPS"}, {"weight": 5.0, "description": "The correlation between the human jejunum cohort\'s expression of the lncRNA LOC100507600 and LCT mRNA was strong, suggesting that this lncRNA could modulate target gene expression.)", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "order": 1, "source": "LNCRNA LOC100507600", "target": "LCT MRNA"}, {"weight": 6.0, "description": "Deletion in MCM6 intron 13 led to segment-specific reductions in LCT mRNA, but not affecting Mcm6 mRNA levels, indicating specificity of effect.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "MCM6 INTRONIC DELETION", "target": "LCT MRNA REDUCTION"}, {"weight": 2.0, "description": "CpGs clusters around specific regions of MCM6 suggest epigenetic control influencing LCT expression.)", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "order": 1, "source": "EPIGENETIC REGULATION", "target": "MCM6 INTRONS AND EXON 16"}, {"weight": 3.0, "description": "A particular SNP in the MCM6 gene is correlated with lactase persistence or non-persistence via epigenetic mechanisms.)", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "order": 1, "source": "LACTASE PERSISTENCE SNP C/T-13910", "target": "SNP-LCT RELATIONSHIP"}, {"weight": 4.0, "description": "Variation in DNA modifications between enterocytes and enterocyte-deficient jejunum highlights regulation specific to cell types.)", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "order": 1, "source": "CELL TYPE SPECIFIC EXPRESSION", "target": "REGULATION BY CELL TYPE"}, {"weight": 5.0, "description": "Age-specific differential DNA modifications detected on the mouse chromosome containing the Lct gene indicate age-related changes in lactase expression.)", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "order": 1, "source": "AGE GROUP COMPARISON - MOUSE LCT", "target": "AGE-DEPENDENT CHANGES"}, {"weight": 6.0, "description": "CpG sites identified as having varying DNA modifications across individuals and cell types, affecting LCT expression.)", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "order": 1, "source": "GENOMIC VARIATION IMPACT", "target": "INTER-INDIVIDUAL AND CELL-TYPE SPECIFIC SITES"}, {"weight": 4.0, "description": "DNA modifications in specific regions of MCM6, such as introns and exon 16/17, may impact its role in transcriptional regulation.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "MCM6 INTRONS AND EXON 16/17", "target": "REGULATORY FUNCTION"}, {"weight": 6.0, "description": "Histone marks indicating activity at the enhancer region contribute to cell-type-specific regulation of genes like LCT and MCM6.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "ENHANCER ACTIVITY", "target": "H3K4ME1 AND H3K27AC"}, {"weight": 2.0, "description": "The presence of histone mark H3K27ac at certain DNA regions suggests their involvement as enhancers for gene expression, particularly in LCT and MCM6 genes.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "ENHANCER ACTIVITY", "target": "HISTONE MODIFICATION H3K27AC"}, {"weight": 7.0, "description": "The promoter region of gene encoding for LCT shows high H3K4me3 levels indicating the potential for transcription initiation.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "ACTIVE PROMOTER", "target": "LCT EXON 1"}, {"weight": 9.0, "description": "Specific SNP C/T-13910 is associated with significant changes in DNA modification at MCM6 locus during postnatal development.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "C/T-13910 SNP", "target": "DNA MODIFICATION CHANGES"}, {"weight": 10.0, "description": "During embryonic development, LCT activity might be high due to absence of postnatal modifications; they gradually regain with age.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "AGE-RELATED EPIGENETIC MODIFICATIONS", "target": "EMBRYOGENESIS AND POSTNATAL ACTIVITY"}, {"weight": 1.0, "description": "DNase I hypersensitivity sites on chromatin indicate regions that are more accessible, suggesting active regulatory processes in LCT and MCM6 genes.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "DNA ACCESSIBILITY", "target": "DNASE I ENZYME"}, {"weight": 3.0, "description": "H3K4me1 and H3K4me3 are post-translational modifications that play crucial roles in regulating gene expression, affecting transcriptional start sites in LCT and MCM6.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "GENE EXPRESSION REGULATION", "target": "POST-TRANSLATIONAL HISTONE MODIFICATION"}, {"weight": 4.0, "description": "Epigenetic modifications occurring from conception onwards gradually influence the age-dependent regulation of lactase activity in adults.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "AGE-DEPENDENT EPIGENETIC CHANGES", "target": "LACTASE ACTIVITY"}, {"weight": 5.0, "description": "High D\' value indicates strong correlation between certain SNPs, suggesting they are often inherited together and affect lactase gene expression or persistence.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "GENETIC ASSOCIATION", "target": "PAIRWISE LINKAGE DISEQUILIBRIUM"}, {"weight": 4.0, "description": "The large degree of mammalian sequence conservation found in the LCT intron 2 region was not found in any other intron of LCT or MCM6, suggesting that LCT intron 2 is an evolutionarily old regulatory element.)", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "order": 1, "source": "DNA MODIFICATIONS", "target": "MAMMALIAN SEQUENCE CONSERVATION"}, {"weight": 6.0, "description": "Our findings support the concept of age-related processes contributing to lactase regulation across species, as evidenced by evolutionary conservation of epigenetic factors and large degree of mammalian sequence conservation.)", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "order": 1, "source": "AGE-DEPENDENT DOWNREGULATION", "target": "EVOLUTIONARY CONSERVATION"}, {"weight": 7.0, "description": "Mammalian intronic conserved elements found near the LCT intron 2 region suggest its involvement in development, possibly alongside other regulatory processes such as age-related changes and evolutionary conservation.)", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "order": 1, "source": "DEVELOPMENTAL PROCESSES", "target": "LACTASE REGULATION"}, {"weight": 1.0, "description": "The TT genotype at SNP -13910 impacts LCT mRNA levels, which is relevant for lactase persistence.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "GENOMIC VARIANT -13910", "target": "LCT EXPRESSION"}, {"weight": 2.0, "description": "Silencing of LOC100507600 leads to a 20% reduction in expression and concomitant 25% decrease in LCT mRNA, demonstrating regulation.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "LCT MRNA REDUCTION", "target": "LOC100507600 SILENCING"}, {"weight": 3.0, "description": "Increased DNA methylation inhibits CTCF binding at insulator sites, affecting tissue-specific gene expression.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "CTCF BINDING INHIBITION", "target": "DNA METHYLATION INCREASE"}, {"weight": 4.0, "description": "CRISPR-Cas9 technology was utilized to manipulate lactase regulatory elements in genes like Lct and Mcm6 for understanding their role in regulation.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "CRISPR-CAS9 GENOME EDITING", "target": "LACTASE REGULATORY ELEMENT MANIPULATION"}, {"weight": 5.0, "description": "Deletion of intron 2 in the LCT gene resulted in a widespread downregulation of LCT mRNA throughout the duodenum and jejunum.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "INTRON 2 DELETION OF LCT", "target": "LCT MRNA WIDESPREAD DOWNREGULATION"}, {"weight": 7.0, "description": "Deletions using CRISPR-Cas9 technology showed varying impacts on LCT mRNA expression across different stages and models, including effects in adult and infant mice.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "CRISPR-CAS9N-INDUCED DELETIONS", "target": "DIFFERENTIAL EFFECTS ON LCT EXPRESSION"}, {"weight": 8.0, "description": "The epigenetically-regulated regions in the MCM6 intron 13 or LCT intron 2 were found to impact human LCT expression in cultured cells, particularly after differentiation into an epithelial-like state.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "EPIGENETIC REGULATION SITES", "target": "LCT EXPRESSION IMPACT"}, {"weight": 1.68, "description": "Increased milk consumption was linked to an elevated risk of RCC in the Czech Republic.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "MILK CONSUMPTION", "target": "RISK OF RCC"}, {"weight": 0.0, "description": "Results for country-specific data showed varied associations, with no systematic pattern observed.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "COUNTRY SPECIFIC DATA", "target": "DIFFERENCES BETWEEN MILK CONSUMPTION AND CANCER RISK"}, {"weight": 0.0, "description": "Different country-specific minor allele frequency (MAF) estimates reflected intermediate frequencies as expected in Eastern European populations.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "COUNTRY SPECIFIC MAF ESTIMATES", "target": "FREQUENCY OF GENOTYPES"}, {"weight": 1.0, "description": "There\'s an association between the HMIP-2 SNP and Mean Total Direct HbF, although significance was lost upon correction for baseline HbF.)", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "order": 1, "source": "HMIP-2 SNP", "target": "MTD HBF"}, {"weight": 0.0, "description": "There was no association found between the C-A haplotype of HMIP-2 and Delta HbF when analyzing the data.)", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "order": 1, "source": "C-A HAPLOTYPE", "target": "MTD HBF"}, {"weight": 0.0, "description": "No association was found between any specific HMIP-2 haplotype and Delta HbF, as determined from our observations.)", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "order": 1, "source": "HMIP-2 HAPLOTYPE", "target": "DELTA HBF"}, {"weight": 1.0, "description": "The significance of HMIP-2 SNP or haplotype association with MTD HbF dropped upon considering baseline HbF levels.)", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "order": 1, "source": "BASELINE HBF", "target": "HMIP-2 SNP/HAPLOTYPE"}, {"weight": 0.0, "description": "No link was found between specific HMIP-2 haplotypes or SNPs and Delta HbF based on our analysis.)", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "order": 1, "source": "DELTA HBF", "target": "HMIP-2 HAPLOTYPE/SNP"}, {"weight": 2.0, "description": "The correlation of baseline HbF with MTD HbF indicates that patients with higher baseline HbF tended to have higher values of MTD HbF.)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "MTD HBF", "target": "COMPARISON"}, {"weight": 1.0, "description": "Our analysis shows an initial association between certain HMIP-2 SNPs/haps and MTD HbF before adjustment for baseline levels.)", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "order": 1, "source": "MTD HBF", "target": "HMIP-2 SNP/HAPLOTYPE"}, {"weight": 1.0, "description": "HU exposure led to statistically significant improvements in multiple hematological parameters among the patients.)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "IMPROVEMENTS", "target": "STATISTICAL SIGNIFICANCE"}, {"weight": 1.0, "description": "The process of identifying statistically significantly differentially expressed genes/probes from the microarray study.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "STATISTICAL SIGNIFICANCE", "target": "DIFFERENTIAL EXPRESSION"}, {"weight": 4.0, "description": "Due to its extremely low frequency, this specific SNP could not be included in further analysis within the cohort.)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "ALLELE FREQUENCY", "target": "EXCLUSION"}, {"weight": 5.0, "description": "These SNPs demonstrated adherence to HWE principles, thus facilitating their inclusion in subsequent regression analysis.)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "CONSISTENCY", "target": "EQUILIBRIUM TEST"}, {"weight": 7.0, "description": "These specific variants of the BCL11A gene correlated positively with elevated levels of both baseline hemoglobin (Hb) and maximum tolerated dose hemoglobin (MTD Hb).)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "HEMOGLOBIN PARAMETERS", "target": "INCREASED LEVEL"}, {"weight": 11.0, "description": "This strong linkage suggests that HMIP-2 SNPs are closely linked and likely inherited together, affecting shared genetic functions related to hemoglobin levels.)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "CHROMOSOMAL LOCATION", "target": "PROXIMITY"}, {"weight": 12.0, "description": "Regular assessment of ANC provided insights into the effects of HU treatment on the immune system\'s granulocyte component.)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "MONITORING", "target": "PARAMETER EVALUATION"}, {"weight": 13.0, "description": "This comprehensive genetic association testing aimed to identify specific genetic markers influencing hematological responses.)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "GENE-SPECIFIC ANALYSIS", "target": "MULTIPLE TESTING"}, {"weight": 14.0, "description": "Strong LD suggests these genetic variants share a recent evolutionary history, highlighting their significance in hematological traits.)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "ANCESTRAL ORIGIN", "target": "GENETIC CONFIGURATION"}, {"weight": 15.0, "description": "Haplotype analysis allowed for the identification of specific genetic combinations that significantly influence hemoglobin levels in response to hydroxyurea treatment.)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "GENETIC CONFIGURATION", "target": "ENHANCED/REDUCED EFFECT"}, {"weight": 6.0, "description": "The overexpression of wild type UBE2T in RA2627 fibroblasts rescued the cellular hypersensitivity to crosslinking agents like MMC, DEB and cisplatin.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "WILD TYPE UBE2T", "target": "RESCUE"}, {"weight": 8.0, "description": "UBE2T expression restored cell cycle distribution in G2 phase after treatment with MMC.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "WILD TYPE UBE2T", "target": "CELL CYCLE"}, {"weight": 10.0, "description": "The overexpression of wild type UBE2T restored monoubiquitination of FANCD2 and FANCI proteins following treatment with MMC.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "WILD TYPE UBE2T", "target": "MONOUBIQUITINATION OF PROTEINS"}, {"weight": 12.0, "description": "The expression of wild type UBE2T facilitated the formation of FANCD2 foci after treatment with MMC.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "WILD TYPE UBE2T", "target": "FANCD2 FOCI"}, {"weight": 3.0, "description": "The overexpression of wild type UBE2T restored sensitivity of RA2627 cells to crosslinking agents following treatment.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "WILD TYPE UBE2T", "target": "CROSSLINKING AGENTS"}, {"weight": 4.0, "description": "The overexpression of wild type UBE2T in RA2627 cells was able to rescue or \'rescue\' FA phenotypes due to its deficiency.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "WILD TYPE UBE2T", "target": "FANCONI ANEMIA"}, {"weight": 5.0, "description": "The expression of wild type UBE2T facilitated the restoration of the monoubiquitination activity on these proteins after treatment with MMC.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "WILD TYPE UBE2T", "target": "MONOUBIQUITINATION OF FANCD2 AND FANCI"}, {"weight": 7.0, "description": "The overexpression of wild type UBE2T improved or \'rescued\' the cellular hypersensitivity characteristic of Fanconi anemia cells.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "WILD TYPE UBE2T", "target": "CELLULAR SENSITIVITY"}, {"weight": 4.0, "description": "Technology platform used for high-throughput screening and analysis of gene expression from the DBA patients\' samples.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "AFFYMETRIX GENECHIP HUMAN GENOME U133A 2.0 ARRAYS", "target": "MICROARRAY PLATFORM"}, {"weight": 10.0, "description": "Method used to validate the results of microarray analysis for gene expression in DBA patients.)", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "order": 1, "source": "QRT-PCR", "target": "VALIDATION METHOD"}, {"weight": 9.0, "description": "Further pathway enrichment identified through analysis of molecular functions related to metabolic pathways involved in DBA patients.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "GLYCINE, SERINE AND THREONINE METABOLISM", "target": "BIOLOGICAL PATHWAY OR ENRICHMENT CLUSTER"}, {"weight": 10.0, "description": "Specific pathway impacted and analyzed as part of impaired biological processes detected in the study on DBA patients.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DEATH RECEPTOR SIGNALING", "target": "PATHWAY OR BIOLOGICAL PROCESS"}, {"weight": 11.0, "description": "Analysis revealed the involvement of PKR activity in antiviral responses through enriched gene functions identified in DBA patients.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "PKR ACTIVITY", "target": "ROLE IN INTERFERON INDUCTION"}, {"weight": 12.0, "description": "Category associated with biological processes and pathways related to retinoid acid receptor activation found significant through enrichment analysis in the study on DBA patients.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "RAR ACTIVATION", "target": "BIOLOGICAL FUNCTION OR PATHWAY ENRICHMENT"}, {"weight": 1.0, "description": "Part of a cluster including other RP genes involved in synthesis related processes.)", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "order": 1, "source": "RPL22", "target": "CLUSTER OF GENES"}, {"weight": 2.0, "description": "Up-regulated gene identified as differentially expressed by qRT-PCR and microarray analysis in DBA patients.)", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "order": 1, "source": "AMPD3", "target": "DIFFERENTIALLY EXPRESSED GENE"}, {"weight": 3.0, "description": "An up-regulated gene in DBA patients confirmed through both qRT-PCR validation and microarray data.)", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "order": 1, "source": "CCND2", "target": "DIFFERENTIALLY EXPRESSED GENE"}, {"weight": 4.0, "description": "Gene found to be up-regulated in DBA patients based on both qRT-PCR results and microarray expression patterns.)", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "order": 1, "source": "SOD2", "target": "DIFFERENTIALLY EXPRESSED GENE"}, {"weight": 5.0, "description": "Up-regulation confirmed in DBA patients by both qRT-PCR validation and microarray data analysis.)", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "order": 1, "source": "TNFAIP3", "target": "DIFFERENTIALLY EXPRESSED GENE"}, {"weight": 6.0, "description": "Down-regulated gene validated through qRT-PCR and microarray analysis for differential expression patterns in DBA patients.)", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "order": 1, "source": "COMP", "target": "DIFFERENTIALLY EXPRESSED GENE"}, {"weight": 7.0, "description": "An identified down-regulated gene in DBA patients supported by qRT-PCR results and microarray data validation.)", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "order": 1, "source": "WARS", "target": "DIFFERENTIALLY EXPRESSED GENE"}, {"weight": 8.0, "description": "Down-regulated in DBA patients, confirmed with both qRT-PCR and microarray analysis as part of the study\'s gene expression patterns validation process.)", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "order": 1, "source": "ZIC1", "target": "DIFFERENTIALLY EXPRESSED GENE"}, {"weight": 2.0, "description": "Comparative analysis between DBA patients and controls in gene expression profiling.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "COMPARISON GROUP OR STATISTICAL TEST", "target": "DBA PATIENTS VS. CONTROLS"}, {"weight": 3.0, "description": "Methodology used to identify differentially expressed genes through the study of gene expression across the entire genome.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "GENE EXPRESSION PROFILING", "target": "WHOLE GENOME TECHNOLOGY"}, {"weight": 5.0, "description": "The sample source from which gene expression data was collected and analyzed through microarray and qRT-PCR methods.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DERMAL FIBROBLAST SAMPLES", "target": "STUDY SUBJECT OR TISSUE SAMPLE"}, {"weight": 6.0, "description": "Method employed to validate findings of reduced RPS19 levels in DBA patients carrying the frameshift mutation, alongside other genomic variants.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "QRT-PCR CONFIRMATION", "target": "VALIDATION TECHNIQUE"}, {"weight": 7.0, "description": "Specific mutation analyzed through PCR and sequencing methods to elucidate effects on gene expression and protein synthesis pathways in DBA patients.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "GENOMIC VARIANT OR MOLECULAR INTERACTION", "target": "INTRON 1 SPLICE SITE MUTATION C.1-1G>A"}, {"weight": 8.0, "description": "Pathway identified as significantly enriched with genes from the differential expression dataset, highlighting its role in DBA pathogenesis.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "AMINOACYL-TRNA BIOSYNTHESIS PATHWAY", "target": "ENRICHMENT CLUSTER OF GENES"}, {"weight": 13.0, "description": "Cluster identified for gene functions involved in protein synthesis, including a large group of RP genes contributing to this pathway\'s significance in DBA.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "GENE ENRICHMENT CATEGORY", "target": "PROTEIN SYNTHESIS CLUSTER"}, {"weight": 14.0, "description": "Enrichment cluster found related to roles in cellular development and death pathways identified through molecular function analysis for DBA patients.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "BIOLOGICAL PROCESS OR FUNCTION", "target": "CELL DEATH ROLE"}, {"weight": 15.0, "description": "Cluster associated with lipid metabolism processes as part of the biological mechanisms significantly impacted in the study on DBA patients.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "LIPID METABOLISM ENRICHMENT", "target": "METABOLIC PATHWAY OR BIOLOGICAL PROCESS"}, {"weight": 16.0, "description": "Enriched category related to molecular transport functions contributing to biological pathways involved in DBA pathogenesis, as identified through analysis of gene functions and annotations.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "BIOLOGICAL FUNCTION OR PATHWAY", "target": "MOLECULAR TRANSPORT CLUSTER"}, {"weight": 1.0, "description": "Category encompassing multiple biological functions identified as critical for the pathogenesis of DBA patients through pathway enrichment analysis performed on the dataset of differentially expressed genes.)(entity", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "PATHWAY ENRICHMENT OR GROUP OF BIOLOGICAL FUNCTIONS", "target": "TOP BIO FUNCTIONS CATEGORY"}, {"weight": 8.0, "description": "The refolding process validates the structural models through the formation of stable p/MHC complexes.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "VALIDATION METHOD", "target": "IN VITRO REFOLDING"}, {"weight": 1.0, "description": "The specific G/A variant of the polymorphism was studied within its genotype context, linking it to migraine risk.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "BDNF RS6265 (G/A) POLYMORPHISM", "target": "ASSOCIATION WITH BDNF GENOTYPE"}, {"weight": 0.03, "description": "The rs6265 polymorphism in the BDNF gene was significantly associated with migraine.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "MIGRAINE", "target": "RS6265 (G/A) POLYMORPHISM"}, {"weight": 0.02, "description": "The rs2049046 polymorphism was significantly linked to migraine.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "MIGRAINE", "target": "RS2049046 (A/T) POLYMORPHISM"}, {"weight": 1.0, "description": "Another genetic variant that may interact or have an additive effect alongside the rs6265 polymorphism in influencing migraine.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "RS2049046 (A/T) POLYMORPHISM", "target": "GENETIC INTERACTION WITH RS6265"}, {"weight": 1.0, "description": "Polymorphisms in the BDNF gene were studied as potential risk factors for developing migraine.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "BDNF GENE POLYMORPHISMS", "target": "MIGRAINE RISK FACTOR"}, {"weight": 1.0, "description": "SNP rs1004819 was identified in patient 1 within the context of IL23R gene investigation.)", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "PATIENT 1", "target": "RS1004819"}, {"weight": 1.0, "description": "SNP rs72676067 was detected in patient 2 during IL23R gene study.)", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "PATIENT 2", "target": "RS72676067"}, {"weight": 6.0, "description": "The development of paradoxical psoriasiform reactions occurred following treatment with adalimumab in patients affected by severe HS.)", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "order": 1, "source": "HS", "target": "PARADOXICAL PSORIASIFORM REACTIONS"}, {"weight": 10.0, "description": "The anti-TNF-\u03b1 treatment led to distinct paradoxical reactions of varying severity in HS patients as measured by PASI scores.)", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "order": 1, "source": "ADALIMUMAB", "target": "REACTIONS"}, {"weight": 2.0, "description": "Enrichment of both CD4+ and CD8+ subpopulations compared to skin T cells from HS patients with paradoxical reactions.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "CD3+ CELLS", "target": "CLASSICAL PSORIASIS"}, {"weight": 1.0, "description": "Overexpression of type I IFNs is characteristic in paradoxical psoriasis, more than classical or nonclassical.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "TYPE I IFN-\u0392", "target": "PARADOXICAL PSORIASIS"}, {"weight": 3.0, "description": "Cytoplasmic expression in psoriasiform lesions, membrane-bound in classical psoriasis; comparable levels across conditions.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "KERATINOCYTES", "target": "IFN-\u039a"}, {"weight": 4.0, "description": "Deregulated by anti-TNF-\u03b1 therapy and overexpressed specifically in paradoxical skin reactions.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "LYMPHOTOXINS", "target": "ANTI-TNF-\u0391 THERAPY"}, {"weight": 7.0, "description": "Three specific ERAP1 SNPs were found in all HS patients with differing distributions across individuals.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "ERAP1 SNPS", "target": "SPECIFIC SNPS"}, {"weight": 8.0, "description": "Shared multiple TNFAIP-related SNPs among patients, indicating genetic predispositions to paradoxical reactions.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "TNFAIP SNPS", "target": "SNP DISTRIBUTION"}, {"weight": 5.0, "description": "The text discusses both CCHCR1 and IL23R gene, indicating potential comparisons or overlaps in genetic studies.)", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "CCHCR1", "target": "IL23R"}, {"weight": 8.0, "description": "Each patient displayed unique disease responses including erythematous-pustular lesions, pustular lesions, and erythemato-scaly plaques respectively after therapy.)", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "order": 1, "source": "DISEASE RESPONSE", "target": "PATIENT 1, PATIENT 2, AND PATIENT 3"}, {"weight": 12.0, "description": "The psoriasis severity index (PASI) indicated a range of disease severity in the patients with paradoxical reactions.)", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "order": 1, "source": "DISEASE SEVERITY SCALE", "target": "PASI 6.8, PASI 5.2, AND PASI 5.6"}, {"weight": 14.0, "description": "The Hurley III Sartorius scoring system was used to measure the severity of hidradenitis suppurativa in each patient.)", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "order": 1, "source": "DISEASE SEVERITY MEASURE", "target": "HURLEY III, SARTORIUS SCORE: 41.5, HURLEY III, SARTORIUS SCORE: 41.5, AND HURLEY III, SARTORIUS SCORE: 61.5"}, {"weight": 8.0, "description": "In Patient 2, a mixed population of T cells responsible for both psoriasiform and eczematous reactions was likely present.)", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "order": 1, "source": "DISEASE SUBTYPE", "target": "PATIENT 2\'S ECZEMATOUS REACTIONS"}, {"weight": 5.0, "description": "All HS patients showed variants of ERAP1, with specific SNPs found across individuals.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "ERAP1 VARIANTS", "target": "HS PATIENTS"}, {"weight": 1.0, "description": "The RING domain\'s electrostatic potential is distinct from the DmDRWD, which wraps around like a ring on the backside of central FANCL.)", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "order": 1, "source": "URD", "target": "RING DOMAIN"}, {"weight": 0.5, "description": "No change in migration was observed by size-exclusion chromatography suggesting no complex formation between URD and Ube2T.)", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "order": 1, "source": "URD", "target": "UBE2T"}, {"weight": 0.6, "description": "Reduction in binding was observed with URD mutants compared to wild type when pulldown experiments were performed with FANCI and FANCD2.)", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "order": 1, "source": "URD", "target": "FANCI, FANCD2"}, {"weight": 6.0, "description": "Exposing hydrophobic residues on its surface facilitates interaction with FANCI and FANCD2 proteins, as implied by URD\'s binding properties.)<SEP>Exposing hydrophobic residues on its surface facilitates interaction with FANCI and FANCD2, as demonstrated by the URD proteins L248A/F252A/L254A/I265A, W212A/L214A.)", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "URD", "target": "HYDROPHOBIC PATCHES"}, {"weight": 6.0, "description": "Binding affinity of URD proteins to FANCI and FANCD2 decreases with specific mutations (L248A/F252A/L254A/I265A, W212A/L214A), emphasizing the role of hydrophobic patches.)<SEP>Interaction of URD with FANCI/FANCD2 proteins depends on the exposed hydrophobic patches on its C-terminal lobe, which are impacted by certain genetic variants.", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "URD", "target": "FANCI/FANCD2"}, {"weight": 0.8, "description": "A 1:1 stoichiometric interaction with Ube2T as shown in size-exclusion chromatography, resulting in an equilibrium binding dissociation constant (Kd) of 454 nm.)", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "order": 1, "source": "RING DOMAIN", "target": "UBE2T"}, {"weight": 0.7, "description": "A 11-fold decrease in binding affinity was observed between mutant Ube2T-F63A and RING as seen in ITC analysis.)", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "order": 1, "source": "RING DOMAIN", "target": "PHE-63 MUTATION"}, {"weight": 3.0, "description": "The mutations at positions 248/252/254 and I265, when compared to mutations at positions W212/A214, both reduce substrate binding capacity.)", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "L248A/F252A/L254A/I265A", "target": "W212A/L214A"}, {"weight": 3.0, "description": "The L248A/F252A/L254A/I265A mutations decrease the binding of URD with FANCI.)", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "L248A/F252A/L254A/I265A", "target": "FANCI"}, {"weight": 3.0, "description": "These mutations reduce the binding of URD with FANCD2 proteins, as observed in comparison to wild type URD.)", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "L248A/F252A/L254A/I265A", "target": "FANCD2"}, {"weight": 3.0, "description": "Mutations W212/A214 in URD affect its interaction capacity with FANCI proteins, as seen with L248A/F252A/L254A/I265A.)", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "W212A/L214A", "target": "FANCI"}, {"weight": 3.0, "description": "Mutations W212/A214 affect substrate interaction with FANCD2, similar to the L248A/F252A/L254A/I265A mutations.)", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "W212A/L214A", "target": "FANCD2"}, {"weight": 3.0, "description": "Similar to L248A/F252A/L254A/I265A, mutations W212/A214 influence hydrophobic patches on the C-terminal lobe to impact substrate binding.)", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "W212A/L214A", "target": "HYDROPHOBIC PATCHES"}, {"weight": 3.0, "description": "The exposed hydrophobic patches on the C-terminal lobe are required for substrate binding as indicated by lower affinity with L254A mutation.)", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "L254A", "target": "HYDROPHOBIC PATCHES"}, {"weight": 3.0, "description": "Exposing hydrophobic patches in the C-terminal lobe is necessary for efficient interaction, shown by the reduced binding capacity of URD with FANCI and FANCD2 with I265A mutation.)", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "I265A", "target": "HYDROPHOBIC PATCHES"}, {"weight": 3.0, "description": "L254A/", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "L248A/", "target": "252A/"}, {"weight": 3.0, "description": "Hydrophobic Patches", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "W212A/", "target": "L214A"}, {"weight": 3.0, "description": "Exposure of hydrophobic patches on the C-terminal lobe is necessary for efficient interaction between URD and FANCI/FANCD2, as indicated by decreased binding affinity with specific mutations.", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "HYDROPHOBIC PATCHES", "target": "SUBSTRATE INTERACTION"}, {"weight": 3.0, "description": "The presence of exposed hydrophobic patches on these proteins is essential for their interactions with URD and thereby for efficient DNA repair processes.)", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "HYDROPHOBIC PATCHES", "target": "FANCI/FANCD2"}, {"weight": 3.0, "description": "The mutations influence substrate binding affinity, with reduced interaction observed for URD proteins.", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "BINDING AFFINITY", "target": "MUTATIONS L248A/F252A/L254A/I265A/W212A/L214A"}, {"weight": 8.0, "description": "The binding affinity for self-antigen ARDIYRASY is insufficient to form stable complexes with HLA-B*15:01.)", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "order": 1, "source": "BINDING AFFINITY", "target": "ALYRASY"}, {"weight": 9.0, "description": "The affinity of nonamer peptides towards HLA-B*15:01 is lower compared to other neoepitopes due to structural compatibility.)", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "order": 1, "source": "BINDING AFFINITY", "target": "NONAMER BINDING"}, {"weight": 0.0, "description": "The FGFR4 SNP (rs376618) did not show significant differential effects when used with colesevelam.)<SEP>The FGFR4 SNP (rs376618) did not show significant differential treatment effects associated with colesevelam in this study.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "FGFR4 RS376618", "target": "COLESEVELAM"}, {"weight": 1.0, "description": "The FGFR4 genetic variant (rs351855) has an impact on the response to the drug colesevelam, influencing parameters like AC t1/2 and GC24.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "FGFR4 RS351855", "target": "COLESEVELAM"}, {"weight": 0.9, "description": "The functional variant (Gly388Arg) in FGFR4 is associated with altered AC t1/2 values during colesevelam treatment.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "FGFR4 RS351855", "target": "AC T1/2"}, {"weight": 0.8, "description": "The KLB genetic variant (rs4975017) was found to have a differential treatment effect with colesevelam, affecting GC24 value and AC t1/2.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "COLESEVELAM", "target": "KLB RS4975017"}, {"weight": 0.5, "description": "The KLB genetic variant (rs17618244) had numerical though not statistically significant differential treatment effects with colesevelam on AC t1/2 and GC24.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "COLESEVELAM", "target": "KLB RS17618244"}, {"weight": 0.8, "description": "The KLB genetic variant shows influence on GC24 levels when treated with colesevelam.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "KLB RS4975017", "target": "GC24"}, {"weight": 0.5, "description": "This functional KLB variant has an impact on GC24 values under treatment with colesevelam, though not statistically significant.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "KLB RS17618244", "target": "GC24"}, {"weight": 13.0, "description": "Significant Spearman correlation between serum FGF19 and total stool BA indicates that FGF19 levels may influence the metabolic pathways related to bile acid synthesis.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "BA SYNTHESIS", "target": "FGF19 CORRELATION"}, {"weight": 7.0, "description": "The genetic variant was marginally associated with total stool BA content suggesting a possible link between the gene variant and metabolic processes related to bile acid synthesis.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "BA SYNTHESIS", "target": "FGFR4 RS1966265 CORRELATION"}, {"weight": 3.0, "description": "There were numerical differences between subgroups with respect to stool fat content with higher values found in IBS-D compared to HV and lower values found in IBS-C compared to HV.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "STOOL FAT", "target": "SUBGROUP COMPARISON"}, {"weight": 3.5, "description": "Similar to stool fat analysis, there were numerical differences between subgroups with respect to stool weight content. Higher weights were observed in IBS-D vs. HV and lower values found in IBS-C compared to HV.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "STOOL WEIGHT", "target": "SUBGROUP COMPARISON"}, {"weight": 12.0, "description": "Significant Spearman correlation between serum FGF19 and stool weight suggests a possible impact of FGF19 levels on the physical properties of feces, such as bulk or consistency.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "STOOL WEIGHT", "target": "FGF19 CORRELATION"}, {"weight": 4.5, "description": "There was a significant Spearman correlation between stool weight and the amount of fat in stools within different subgroup comparisons.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "STOOL WEIGHT", "target": "STOOL FAT CONTENT"}, {"weight": 5.5, "description": "Total BA content in feces was shown to correlate with stool weight, suggesting that increased bulk or volume could impact the overall excreted bile acids.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "BA EXCRETION", "target": "TOTAL STOOL BA"}, {"weight": 8.5, "description": "FGF19 showed significant correlations with both stool weight and total stool BA content implying its role might be involved in fecal properties or bile acid synthesis and excretion.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "FGF19 CORRELATION", "target": "STOOL WEIGHT (BA SYNTHESIS)"}, {"weight": 6.5, "description": "Cholecystectomy had a statistically significant impact on IBS symptoms in the IBS-D subgroup compared to IBS-C and HV groups.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "IBS-D", "target": "SYMPTOM IMPACT"}, {"weight": 4.0, "description": "IBS-D had significantly higher levels of stool BA content compared to IBS-C but not compared to HV, indicating subgroup differences in the synthesis and excretion of bile acids.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "SUBGROUP COMPARISON", "target": "BA SYNTHESIS (STOOL EXCRETION)"}, {"weight": 5.5, "description": "No association was observed between fasting serum FGF19 levels with subgroups indicated no direct impact on BA synthesis or stool excretion.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "LACK OF ASSOCIATION", "target": "SERUM FGF19"}, {"weight": 7.5, "description": "The FGFR4 rs1966265 variant was marginally associated with total stool BA content in IBS-D compared to other subgroups.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "BA SYNTHESIS COMPARISON", "target": "FGF14 RS1966265"}, {"weight": 7.0, "description": "Spearman correlations showed that serum C4 levels were positively correlated with total cholesterol levels in blood, suggesting potential links to metabolic processes related to lipids or bile acid metabolism.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "SERUM C4", "target": "CHOLESTEROL CORRELATION"}, {"weight": 6.0, "description": "The genetic variation FGFR4 rs351855 was not significantly associated with serum C4 or stool BA suggesting it did not influence bile acid metabolism related markers.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "FGFR4 RS351855 CORRELATION", "target": "NO ASSOCIATION"}, {"weight": 5.5, "description": "The genetic variant FGFR4 rs376618 was also found without significant association with serum C4 or stool BA indicating no influence on bile acid pathways.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "NO ASSOCIATION", "target": "FGFR4 RS376618 CORRELATION"}, {"weight": 9.0, "description": "A significant Spearman correlation between FGF19 and total stool BA content suggested that this factor might play a role in the metabolic pathways related to bile acid synthesis or excretion.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "BILE ACID METABOLISM", "target": "FGF19 (BA SYNTHESIS)"}, {"weight": 6.5, "description": "The presence of cholecystectomy had a statistically significant effect on IBS symptoms specifically within the IBS-D subgroup, suggesting differences in symptom profiles compared to other subgroups.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "IBS-D (CHOLECYSTECTOMY)", "target": "SYMPTOM COMPARISON"}, {"weight": 3.0, "description": "The zinc ion coordinates with FAAP20 to maintain its structure and function.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "FAAP20", "target": "ZINC ION"}, {"weight": 5.0, "description": "FAAP20 plays a role within this pathway for DNA repair and cell cycle regulation.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "FAAP20", "target": "FANCONI ANEMIA PATHWAY"}, {"weight": 6.0, "description": "FAAP20 interacts with Ub through hydrophobic and hydrogen bonding interactions, which are essential for its proper function.)<SEP>The FAAP20 protein shows a specific interaction pattern with Ub compared to other proteins.)", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591<SEP>chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "FAAP20", "target": "UB"}, {"weight": 7.0, "description": "FAAP20\'s localization to ICL sites is essential for proper response to DNA damage repair, which could impact disease prevention or progression.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "FAAP20", "target": "ICL SITES"}, {"weight": 2.0, "description": "FAAP20-UBZ interacts with K63-linked ubiquitin.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "FAAP20-UBZ", "target": "UBIQUITINS"}, {"weight": 6.0, "description": "FAAP20-UBZ shows strong interaction with different ubiquitin species, as measured by SPR.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "FAAP20-UBZ", "target": "AFFINITY TOWARDS UBIQUITINS"}, {"weight": 1.0, "description": "While FAAP20-UBZ shares similarities in the mode of interaction with ubiquitin as Pol\u03b7-UBZ, they differ from WRNIP1 and RAD18 UBZs.)", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "FAAP20-UBZ", "target": "POL\u0397-UBZ"}, {"weight": 3.0, "description": "FAAP20-UBZ binds ubiquitin via specific \u03b1-helix residues, establishing functional interactions.)", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "FAAP20-UBZ", "target": "UB"}, {"weight": 8.0, "description": "FANCL acts as a modulatory component that interacts with FAAP20 and other FA proteins to maintain the activity of ubiquitin-dependent pathways.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "FANCL", "target": "ROLE IN FA COMPLEX"}, {"weight": 9.0, "description": "UVRAG integrates into the FA core complex alongside FAAP20, participating in the coordination of ubiquitin-related processes.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "UVRAG", "target": "CONTRIBUTION TO FA CORE COMPLEX"}, {"weight": 3.0, "description": "Similar to FAAP20-UBZ, Pol\u03b7-UBZ binds ubiquitin by interacting with its \u03b1-helix, suggesting a convergent evolutionary mechanism in binding Ub.)", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "UB", "target": "POL\u0397-UBZ"}, {"weight": 4.0, "description": "NEMO-UBZ binds ubiquitin differently due to sequence differences compared to other UBZs like FAAP20-UBZ and WRNIP1-UBZ.)", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "UB", "target": "NEMO-UBZ"}, {"weight": 6.0, "description": "The zinc-coordinating residue plays a structural role in maintaining the stability of the UBZ domain, influencing its functionality and consequently cellular processes.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "ZINC-COORDINATING RESIDUE CYS150", "target": "FAAP20-UBZ DOMAIN"}, {"weight": 2.0, "description": "Both WRNIP1-UBZ and RAD18-UBZ interact with ubiquitin through their \u03b2-loop, suggesting similarities in the mechanism of interaction between these UBZs.)", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "WRNIP1-UBZ", "target": "RAD18-UBZ"}, {"weight": 1.0, "description": "NEMO-UBZ and Pol\u03b7-UBZ exhibit different modes of interaction with ubiquitin compared to WRNIP1-UBZ and RAD18-UBZ, based on sequence conservation.)", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "POL\u0397-UBZ", "target": "NEMO-UBZ"}, {"weight": 4.0, "description": "Despite sequence differences, Pol\u03b7 and RAD18 show similar interaction patterns with Ub through their UBZ domains.)", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "POL\u0397", "target": "RAD18"}, {"weight": 5.0, "description": "WRNIP1-UBZ and RAD18-UBZ interact with ubiquitin through their \u03b2-loop residues, indicating a common interaction mode between these UBZ domains.)", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "RAD18", "target": "WRNIP1"}, {"weight": 4.0, "description": "These regions of FAAP20-UBZ specifically interact with ubiquitins, affecting the binding specificity.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "CENTRAL \u0391-HELIX AND SECOND LOOP", "target": "UBIQUITIN"}, {"weight": 10.0, "description": "FAAP19 may interact with FANCL for enhanced DNA repair capacity through coordinated activities within Fanconi Anemia pathway.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "FAAP19 AND FANCL", "target": "INTERACTION WITHIN FA PATHWAY"}, {"weight": 2.0, "description": "The study compares CD subjects to healthy controls to identify genetic markers.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "HEALTHY CONTROLS", "target": "CD SUBJECTS"}, {"weight": 5.0, "description": "Healthy controls show less presence of the heterozygous (TC) alleles for the study SNPs.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "HEALTHY CONTROLS", "target": "HETEROZYGOUS ALLELES"}, {"weight": 7.0, "description": "Healthy controls exhibit none or lower levels of both MAPbacteremia and the studied SNPs.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "HEALTHY CONTROLS", "target": "NO MAPBACTEREMIA OR SPECIFIC SNPS"}, {"weight": 1.0, "description": "The presence of the rs12762549 variant is associated with an increased risk of developing grade 3-4 anemia.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "RS12762549", "target": "GRADE 3-4 ANEMIA"}, {"weight": 1.0, "description": "The presence of the rs363717 variant is associated with an increased risk of developing grade 3-4 anemia.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "RS363717", "target": "GRADE 3-4 ANEMIA"}, {"weight": 1.0, "description": "The presence of the rs11615 variant is associated with an increased risk of developing grade 3-4 anemia.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "RS11615", "target": "GRADE 3-4 ANEMIA"}, {"weight": 1.0, "description": "The presence of the rs4986910 variant is associated with an increased risk of developing thrombocytopenia grade 3-4.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "RS4986910", "target": "THROMBOCYTOPENIA GRADE 3-4"}, {"weight": 1.0, "description": "The presence of the rs2073337 variant shows an association with CSF usage during chemotherapy, indicating potential need for supplementary treatments to manage side effects.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "RS2073337", "target": "GROWTH FACTORS USE (CSFS)"}, {"weight": 1.0, "description": "The rs1799793 variant shows significant association with PFI, indicating potential genetic influences on treatment outcome.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "RS1799793", "target": "PROGRESSION-FREE INTERVAL (PFI)"}, {"weight": 0.04, "description": "The genomic variant rs6104 showed an association with PFI in the fully-adjusted model but did not meet statistical significance when age adjustment alone was considered.)", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "order": 1, "source": "RS6104", "target": "PFI"}, {"weight": 1.0, "description": "The presence of the rs2074087 variant shows an association with ESA usage during chemotherapy, potentially influencing treatment efficacy and managing side effects.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "GROWTH FACTORS USE (ESAS)", "target": "RS2074087"}, {"weight": 1.0, "description": "The rs434074 variant shows an association with neutropenia during chemotherapy treatment, indicating a genetic predisposition to this side effect.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "NEUTROPENIA", "target": "RS434074"}, {"weight": 1.0, "description": "The rs1862219 variant is associated with an increased risk of developing grade 2 or higher toxicity during chemotherapy, highlighting a genetic factor in adverse reactions.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "GRADE 2 OR HIGHER TOXICITY", "target": "RS1862219"}, {"weight": 1.0, "description": "The rs737645 variant correlates with the need for hematopoietic growth factors during chemotherapy, indicating genetic influences on supportive care requirements.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "HEMATOPOIETIC GROWTH FACTORS USE", "target": "RS737645"}, {"weight": 1.0, "description": "The rs653980 variant shows an association with the development of grade 3 or higher peripheral neuropathy, suggesting a genetic predisposition to this specific side effect.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "PERIPHERAL NEUROPATHY", "target": "RS653980"}, {"weight": 1.0, "description": "The rs10866159 variant correlates with the development of hematological toxicity, highlighting a genetic factor influencing this side effect.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "HEMATOLOGICAL TOXICITY", "target": "RS10866159"}, {"weight": 1.0, "description": "The rs9095462 variant shows an association with the development of grade 2 or higher nausea and vomiting, indicating a genetic predisposition to these symptoms.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "NAUSEA AND VOMITING", "target": "RS9095462"}, {"weight": 1.0, "description": "In I1, there was a de novo 6.4 Mb deletion on chromosome 3 that contained the UMPS gene.)", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "order": 1, "source": "UMPS", "target": "DELETION"}, {"weight": 7.0, "description": "The TGF-\u00df1 gene\'s promoter region SNPs were genotyped to evaluate their potential association with CRC.)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "RS1800469 C/T", "target": "TGF-SS1 SNPS"}, {"weight": 4.0, "description": "SNP analysis of IL-10 gene\'s rs1800896 polymorphism was performed for CRC risk assessment.)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "RS1800896 A/G", "target": "IL-10 GENOTYPE"}, {"weight": 3.0, "description": "The polymorphism rs4073 in the IL-8 gene was genotyped to assess its association with CRC.)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "RS4073 T/A", "target": "IL-8 GENOTYPE"}, {"weight": 2.0, "description": "Genotyping for this specific SNP in the IL-1\u03b2 gene provided insights into its potential association with CRC.)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "RS5493062", "target": "IL-1\u0392 SNPS"}, {"weight": 4.0, "description": "While the T allele showed a slight association with CRC, this was not statistically significant.)", "source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "order": 1, "source": "T ALLELE", "target": "NON-SIGNIFICANT ASSOCIATION"}, {"weight": 5.0, "description": "Polymorphisms in the IL-22 gene were studied to understand their link with CRC susceptibility.)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "IL-22 GENOTYPES", "target": "RS1179251 C/G AND RS2227485 C/T"}, {"weight": 6.0, "description": "Genotyping of this SNP in IL-27\'s promoter region was performed for its role in CRC development.)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "IL-27 GENOTYPE", "target": "RS17855750T/G"}, {"weight": 1.0, "description": "The polymorphism of IL-8\'s rs4073 was assessed to determine if it influences the risk of developing colorectal cancer.)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "CRC RISK", "target": "IL-8 RS4073 T/A"}, {"weight": 3.0, "description": "This polymorphism was analyzed as it relates to changes in IL-1\u03b1 that may impact the risk of colorectal cancer development.)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "CRC SUSCEPTIBILITY", "target": "IL-1\u0391 RS3093422"}, {"weight": 2.0, "description": "rs2476601 was found with less frequency in CD subjects than in healthy controls.)", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "order": 1, "source": "HEALTHY CONTROL", "target": "RS2476601"}, {"weight": 8.0, "description": "PCA revealed clear separation between the non-Hispanic white group and other groups.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "NON-HISPANIC WHITE GROUP", "target": "POPULATION STRATIFICATION COMPONENT"}, {"weight": 9.0, "description": "The non-Hispanic black group showed distinct scores on PC1 compared to others, indicating clear separation from the non-Hispanic white group.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "NON-HISPANIC BLACK GROUP", "target": "POPULATION STRATIFICATION COMPONENT"}, {"weight": 10.0, "description": "PC1 scores for the Hispanic group were in-between those of other groups, suggesting overlapping ancestry with others.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "HISPANIC GROUP", "target": "POPULATION STRATIFICATION COMPONENT"}, {"weight": 2.0, "description": "The SNP\'s risk for asthma was increased in ex-smoker subgroups.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "ASTHMA POPULATION", "target": "EX-SMOKERS"}, {"weight": 3.0, "description": "SNP rs1898671 did not show significant association with asthma in never-smokers.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "ASTHMA POPULATION", "target": "NEVER-SMOKERS"}, {"weight": 5.0, "description": "SNP rs1898671\'s effect on asthma risk was confirmed in ex-smoker subgroups.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "ASTHMA POPULATION", "target": "EX-SMOKERS IN REPLICATION COHORT"}, {"weight": 6.0, "description": "No significant association found for SNP rs1898671 in never-smokers.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "ASTHMA POPULATION", "target": "NEVER-SMOKERS IN REPLICATION COHORT"}, {"weight": 1.0, "description": "SNP rs1898671 showed a nominal association with asthma susceptibility.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "TSLP SNPS", "target": "ASTHMA"}, {"weight": 1.0, "description": "Increased levels of IL-8 protein in serum and mRNA were observed in asthmatic patients compared to controls.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "ASTHMA", "target": "IL-8 PROTEIN"}, {"weight": 2.0, "description": "The mRNA expression level of IL-8 was significantly higher in asthmatic patients than in healthy controls.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "ASTHMA", "target": "IL-8 MRNA"}, {"weight": 3.0, "description": "The rs4073 genetic variant showed a significant association with asthma susceptibility.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "ASTHMA", "target": "RS4073"}, {"weight": 4.0, "description": "The rs2227306 genetic variant was significantly associated with asthma risk.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "ASTHMA", "target": "RS2227306"}, {"weight": 4.0, "description": "The replication cohort validated the association of SNP rs1898671 with asthma.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "LARGE DATA SET", "target": "REPLICATION COHORT 6 COAHS"}, {"weight": 7.0, "description": "Cases showed differences such as age and BMI compared to healthy control group.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "ASTHMATIC PATIENT SUBGROUP", "target": "CLINICAL POPULATION GROUP"}, {"weight": 11.0, "description": "The asthma population had a higher average age compared to healthy controls.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "AGE DIFFERENCE", "target": "COMPARISON BETWEEN GROUPS"}, {"weight": 12.0, "description": "Asthmatic patients had significantly higher BMI values than healthy control participants.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "COMPARISON BETWEEN GROUPS", "target": "BMI COMPARISON"}, {"weight": 13.0, "description": "There was a statistically significant difference in race/ethnicity between the asthmatic patient group and the healthy controls, with p=0.01.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "CLINICAL FEATURE COMPARISON", "target": "RACE/ETHNICITY DIFFERENCE"}, {"weight": 16.0, "description": "A significantly greater proportion of asthmatic patients demonstrated atopic characteristics compared to healthy controls, with p=0.01.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "CLINICAL FEATURE COMPARISON", "target": "ATOPY PROPORTION"}, {"weight": 14.0, "description": "Reduced FEV1%, FVC% and FEV1/FVC ratios were noted specifically in asthma patients compared to healthy individuals.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "DIFFERENCES BETWEEN GROUPS", "target": "LUNG FUNCTION PARAMETER REDUCTIONS"}, {"weight": 15.0, "description": "Increased total IgE levels were a characteristic feature seen exclusively among asthmatic patients.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "COMPARISON OF IMMOBILE PARAMETERS", "target": "TOTAL IGE INCREASE"}, {"weight": 1.0, "description": "rs478582 was detected significantly more often among CD subjects compared to healthy controls.)", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "order": 1, "source": "RS478582", "target": "CD"}, {"weight": 3.0, "description": "The Haplotype combination of T-G (major/major) was more significantly present among healthy controls compared to CD subjects.)", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "order": 1, "source": "RS478582", "target": "HAPLOTYPE T-G"}, {"weight": 4.0, "description": "The Haplotype combination of C-G (heterozygous or minor/major) and C-A (heterozygous or minor/heterozygous or minor) was more significantly present among CD subjects compared to healthy controls.)", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "order": 1, "source": "RS478582", "target": "HAPLOTYPE C-G"}, {"weight": 6.0, "description": "The presence of rs2476601 was not significantly associated with differences in expression levels of PTPN2 or the cytokine IFN-\u03b3 in CD subjects compared to healthy controls. )", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "order": 1, "source": "RS2476601", "target": "EXPRESSION OF PTPN2 AND IFN-\u0393"}, {"weight": 5.0, "description": "The genomic variant rs2476601 affects the risk for several autoimmune disorders through its association with a Th1 cell signature and IFN\u03b3 secretion.)", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "order": 1, "source": "RS2476601", "target": "AUTOIMMUNE DISORDERS"}, {"weight": 2.0, "description": "The SNP rs2476601 was identified as one of the 8 genome-wide significant SNPs for general IgM that were also associated with IgM anti-PC.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "RS2476601", "target": "GENERAL IGM AND IGM ANTI-PC"}, {"weight": 1.0, "description": "SNPs of PTPN2:rs478582 are associated with the risk and manifestation of Celiac Disease.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "PTPN2:RS478582", "target": "CELIAC DISEASE"}, {"weight": 3.0, "description": "Specific SNPs show higher presence in individuals with heterozygous (TC) alleles.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "PTPN2:RS478582", "target": "HETEROZYGOUS ALLELES"}, {"weight": 3.0, "description": "The presence of MAPbacteremia affected the T-cell proliferation response.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "MAPBACTEREMIA", "target": "T-CELLS"}, {"weight": 4.0, "description": "CD subjects are more likely to exhibit MAPbacteremia when specific PTPN2:rs478582 SNPs are present.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "CD SUBJECTS", "target": "PRESENCE OF MAPBACTEREMIA"}, {"weight": 1.0, "description": "1(entity", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "order": 1, "source": "OR = 17.5, CI: 5.65-54.3&RISK FACTOR COMPARISON&INCREASED RISK OF DISEASE", "target": "THE ODDS RATIO COMPARISONS SHOWED A STATISTICALLY SIGNIFICANT INCREASE IN DISEASE RISK ASSOCIATED WITH PTPN2:RS478582.)"}, {"weight": 6.0, "description": "The presence of MAPbacteremia in CD subjects is linked to specific genetic markers.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "CELIAC DISEASE", "target": "PRESENCE OF MAPBACTEREMIA"}, {"weight": 1.0, "description": "Endogenous PTPN22 expression varies across different subsets of CD4+ T cells (naive, memory, and regulatory) at rest and post-activation.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "PTPN22", "target": "CD4+ T CELLS"}, {"weight": 3.0, "description": "The increase in PTPN22 expression was significantly lower in memory Tconv cells compared to naive Tconv after activation.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "NAIVE TCONV", "target": "MEMORY TCONV"}, {"weight": 4.0, "description": "Treg cells displayed the highest PTPN22 expression among all subsets both at rest and following activation.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "TREG", "target": "PTPN22 EXPRESSION"}, {"weight": 31.87, "description": "Overexpression of LYP-620R blunts activation marker kinetics in Treg cells, except for CD226 on Tconv.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "TREG", "target": "LYP-620R"}, {"weight": 75.49, "description": "Overexpression of LYP-620R reduces the APC-mediated proliferation of Tconv cells.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "LYP-620R", "target": "TCONV"}, {"weight": 71.38, "description": "LYP-620W reduces Treg proliferation by a similar degree as LYP-620R when overexpressed.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "LYP-620W", "target": "TREGS"}, {"weight": 2.0, "description": "Calcium signaling pathways affect the expression of PTPN22 by impacting key processes like MAPK/ERK pathway activity, influencing downstream effects in T-cell function and autoimmune disease risk.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "CALCIUM SIGNALING", "target": "DOWNSTREAM EFFECTS ON PTPN22"}, {"weight": 3.0, "description": "The MAPK/ERK pathway is interconnected with calcium signaling pathways, both impacting the expression of PTPN22 through coordinated regulatory mechanisms in T-cell function and autoimmune disease risk.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "MAPK/ERK PATHWAY", "target": "CONNECTION TO CALCIUM SIGNALING"}, {"weight": 5.0, "description": "Calcium flux quantifies changes in intracellular calcium levels as a response to T-cell receptor (TCR) ligation, affecting downstream signaling cascades including those involved with PTPN22 expression and function.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "CA2+ FLUX", "target": "RESPONSE TO TCR LIGATION"}, {"weight": 12.0, "description": "The presence of risk variants such as LYP-620W in the PTPN22 gene is linked with an increased risk for autoimmune diseases through its impact on modulating T-cell function and regulatory pathways.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "AUTOIMMUNE DISEASE RISK", "target": "CONNECTION VIA GENETIC VARIANTS"}, {"weight": 4.0, "description": "While the suppression capacity of Tregs was slightly increased with variant 620W, no significant difference was observed in Treg proliferation compared to variants 620R or eGFP reporter Tregs.)", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "order": 1, "source": "TREGS", "target": "SUPPRESSION CAPACITY AND PROLIFERATION"}, {"weight": 1.0, "description": "Overexpression of LYP variant 620R leads to a similar level of effector cytokine secretion as compared to eGFP reporter Tregs, indicating no significant impact on Treg activity.)", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "order": 1, "source": "LYP VARIANT 620R", "target": "TREG-MEDIATED SUPPRESSION"}, {"weight": 2.0, "description": "The LYP variant 620W affects the Tcell intrinsic susceptibility and modulates activation, proliferation, and effector mechanisms rather than impacting directly on suppressive functions of Tregs.)", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "order": 1, "source": "LYP VARIANT 620W", "target": "TCONV INTRINSIC SUSCEPTIBILITY"}, {"weight": 59.08, "description": "IFN\u03b3 secretion is not altered by overexpression of either variant of LYP, despite its association with autoimmune disorders.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "IFN\u0393", "target": "NO ALTERATION BY PTPN22 VARIANTS"}, {"weight": 2.0, "description": "Interferon-gamma production by CAR T cells is specific to TAG72-positive cell types like OAS4 and OAS7, but not OAS3.)", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "order": 1, "source": "IFN\u0393", "target": "RESPONSE TO CELL TYPE"}, {"weight": 2.0, "description": "The level of PTPN22 expression is higher in activated compared to quiescent states for all analyzed CD4+ T cell subsets.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "ACTIVATION STATE", "target": "PTPN22 EXPRESSION"}, {"weight": 3.0, "description": "Data suggest that both LYP variants may influence immune responses but show different effects when compared to PTPN22 expression in Treg cells.)", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "order": 1, "source": "PTPN22 EXPRESSION", "target": "LYP VARIANT 620R/LYP VARIANT 620W"}, {"weight": 1.0, "description": "The PTPN22 gene, encoded by different variants like LYP-620R or LYP-620W, plays a role in modulating immune responses including T-cell function and differentiation.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "PTPN22 GENE", "target": "ROLE IN IMMUNE REGULATION"}, {"weight": 4.0, "description": "The level of pERK serves as a marker for activity within the MAPK/ERK pathway, which has downstream effects on several processes including PTPN22 expression and T-cell function.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "PERK MARKER", "target": "SIGNIFICANCE IN DOWNSTREAM EVENTS"}, {"weight": 6.0, "description": "The CD69 molecule serves as an early marker for T-cell activation which correlates with the expression levels of genes like PTPN22, impacting downstream signaling pathways and immune responses.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "CD69 MARKER", "target": "EARLY T-CELL ACTIVATION SIGN"}, {"weight": 7.0, "description": "The inducible costimulatory (ICOS) molecule acts as a co-stimulatory receptor on T-cells during activation processes, influencing gene expression patterns including PTPN22.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "ICOS RECEPTOR", "target": "MODULATION BY ACTIVATED T-CELLS"}, {"weight": 8.0, "description": "The CD25 molecule, also known as IL2R\u03b1, serves as an indicator of early T-cell activation and modulates the expression of genes such as PTPN22 through its regulatory activities.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "CD25 MARKER", "target": "T-CELL ACTIVATION INDICATORS"}, {"weight": 9.0, "description": "The CD226 receptor, also known as NKG2D or NKp46, plays a role in co-stimulating immune responses on T-cells and influences downstream regulatory mechanisms involving PTPN22 expression.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "CD226 RECEPTOR", "target": "CO-STIMULATION ROLE IN IMMUNE REGULATION"}, {"weight": 10.0, "description": "The PD-1 receptor, alongside its ligands, contributes to immune suppression through inhibiting T-cell activity. This can have indirect effects on the regulation of genes like PTPN22 and T-cell function.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "PD-1 CO-INHIBITORY RECEPTOR", "target": "ROLE IN IMMUNE SUPPRESSION"}, {"weight": 11.0, "description": "Antigen-presenting cells (APCs) play a crucial role in initiating immune responses by presenting antigens to T-cells, which can modulate the expression and activity of genes like PTPN22 involved in T-cell regulation.)", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "APC CELL TYPE", "target": "ROLE IN IMMUNE RESPONSE INITIATION"}, {"weight": 49.53, "description": "Transduced Tconv cells show reduced APC-mediated proliferation upon overexpression of LYP-620R compared to mock-transfected controls.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "APC", "target": "TCONVE"}, {"weight": 51.37, "description": "Overexpression of PTPN22 influences IL-2 secretion; no alteration by LYP-620R, increase at the 24-hour time point by LYP-620W.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "IL-2", "target": "INFLUENCE BY LYP VARIANTS"}, {"weight": 3.0, "description": "Interleukin-2 secretion by TAG72-BB\u03b6 CAR T cells specifically targets and kills OAS4 cells while sparing other cell types like OAS3 and OAS7.)", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "order": 1, "source": "IL-2", "target": "CAR RESPONSE"}, {"weight": 67.43, "description": "The secretion of these cytokines was robustly inhibited by expression of either LYP variant for the first 48 hours.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "IL-4, IL-5, IL-9, IL-10, AND IL-13", "target": "INHIBITION BY PTPN22 VARIANTS"}, {"weight": 6.0, "description": "The Th1 signature associated with rs2476601 is involved in the development of autoimmune disorders.)", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "order": 1, "source": "DISEASE ASSOCIATION", "target": "TH1 SIGNATURE/RS2476601"}, {"weight": 1.0, "description": "A linear kernel performs well when survival time is simulated based on a linear genetic effect, showing similar or slightly preferred power over other kernels and tests.)", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "order": 1, "source": "SURVIVAL ANALYSIS", "target": "LINEAR KERNEL"}, {"weight": 3.0, "description": "TAG72-BB\u03b6 CAR T cells have potent anti-tumor activity against TAG72-positive cells.)", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "order": 1, "source": "CAR T CELLS", "target": "TAG72-BB\u0396 CAR T CELLS"}, {"weight": 1.0, "description": "Mock T cells show no detectable anti-tumor activity towards the TAG72-negative OVCAR3 cells.)", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "order": 1, "source": "MOCK T CELLS", "target": "OVCAR3"}, {"weight": 2.0, "description": "No therapeutic benefit is observed in mice treated with mock T cells against peritoneal ovarian cancers.)", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "order": 1, "source": "MOCK T CELLS", "target": "INVASIVE PERITONEAL OVARIAN CANCERS"}, {"weight": 1.0, "description": "Mock T cells do not exhibit detectable anti-tumor activity towards TAG72-negative cell lines such as OAS3, unlike CAR T cells.)", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "order": 1, "source": "MOCK T CELLS", "target": "ANTI-TUMOR ACTIVITY"}, {"weight": 4.0, "description": "Despite aggressive nature, repeated delivery of TAG72-BB\u03b6 CAR T cells demonstrates more durable anti-tumor responses compared to single dose.)", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "order": 1, "source": "TAG72-BB\u0396 CAR T CELLS", "target": "OV90"}, {"weight": 4.0, "description": "TAG72-BB\u03b6 CAR T cells treated by regional intraperitoneal delivery demonstrate more effective anti-tumor activity than those administered systemically through intravenous injection.)", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "order": 1, "source": "REGIONAL INTRAPERITONEAL DELIVERY", "target": "THERAPEUTIC EFFICACY"}, {"weight": 6.0, "description": "Increased numbers of T cells were noted within peritoneal tumors in mice with repeat treatment for TAG72-BB\u03b6 CAR T Cells compared to a single dose.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "T CELL NUMBERS", "target": "DOSAGE INFLUENCE"}, {"weight": 4.0, "description": "Multiple dosing of TAG72-BB\u03b6 CAR T Cells demonstrated more extensive tumor regression and durable control compared with a single dose.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "DOSING REGIMENS", "target": "THERAPEUTIC STRATEGY"}, {"weight": 8.0, "description": "Tumor recurrences following TAG72-BB\u03b6 CAR T cell therapy are often due to antigen escape or loss.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "RESISTANCE MECHANISM", "target": "TUMOR RECURRENCE"}, {"weight": 9.0, "description": "Alteration of TAG72 expression levels over time, particularly during tumor recurrence and post-TAG72-BB\u03b6 CAR T Cell therapy.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "ANTIGEN EXPRESSION DYNAMICS", "target": "TEMPORAL CHANGE IN ANTIGENS"}, {"weight": 10.0, "description": "Tumor recurrences can be predicted based on antigen dynamics following CAR T cell treatment in different models.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "OUTCOME PREDICTION", "target": "RECURRENT TUMOR MODELS"}, {"weight": 13.0, "description": "Interaction between aptamers and antigens on tumor cells to enhance CAR T cell efficacy.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "APTAMER BINDING", "target": "INTERPLAY BETWEEN MOLECULES"}, {"weight": 14.0, "description": "Unpredictable mechanisms that prevent CAR T cell effectiveness, beyond antigen-specific resistance.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "AGNOSTIC RESISTANCE", "target": "CONSEQUENCE OF T CELL TREATMENT"}, {"weight": 15.0, "description": "Evaluating how long CAR T cells remain viable and active within the body during treatment.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "CAR T CELL SURVIVAL", "target": "OUTCOME MEASUREMENT"}, {"weight": 16.0, "description": "The duration needed for bone tissue to recover from potential side effects of CAR T cell therapy or associated conditions like avascular necrosis.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "RECOVERY TIME FOR BONE TISSUE", "target": "TIMEFRAME"}, {"weight": 17.0, "description": "Potential negative consequences of CAR T cell treatment that need to be managed or mitigated.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "ADVERSE EFFECTS", "target": "THERAPEUTIC LIMITATION"}, {"weight": 1.0, "description": "Significantly connected to \u03b2-defensin genes, as well as to CXCL12 and TNFRSF13B according to GRAIL analysis.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "FUNCTIONAL CONNECTION", "target": "SNP AT THE CCL23-CCL18 LOCUS"}, {"weight": 0.0, "description": "Not found to be functionally connected with other genes defined by the 80 SNPs, based on the analysis.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "FUNCTIONAL CONNECTION", "target": "THSD4-UACA-TLE3, C10ORF11"}, {"weight": 11.6, "description": "Explains a proportion of variability in forced expiratory volume against time due to genetic factors.)<SEP>The statistical connection of this specific SNP variant to FEV1/FVC is a contributing factor explaining part of its variability.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "SNP AT THE CCL23-CCL18 LOCUS", "target": "EXPLAINED VARIANCE IN FEV1/FVC"}, {"weight": 0.0, "description": "association", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "REPLICATED LOCI, THSD4-UACA-TLE3 AND C10ORF11", "target": "FUNCTIONAL CONNECTION/"}, {"weight": 3.0, "description": "SNP rs12441227 at the THSD4 locus has a genome-wide significant association with FEV1/FVC lung function.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "RS12441227 SNP", "target": "THSD4 GENE"}, {"weight": 2.0, "description": "The heritability of FVC and FEV1/FVC included both additive and non-additive genetic variance components, indicating complex molecular interactions.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "GENETIC VARIANCE", "target": "NON-ADDITIVE GENETIC VARIANCE"}, {"weight": 0.8, "description": "The G versus A comparison for rs231775 polymorphism showed significant heterogeneity, and thus a random-effect model was used to generate extensive CIs in these genetic models.)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "RS231775", "target": "G VERSUS A"}, {"weight": 0.8, "description": "SNP rs231775 is associated with the gene CTLA4)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "RS231775", "target": "CTLA4"}, {"weight": 0.7, "description": "The A versus G comparison for rs231725 polymorphism showed significant heterogeneity, leading to the use of a random-effect model.)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "RS231725", "target": "A VERSUS G"}, {"weight": 0.7, "description": "SNP rs231725 is also associated with the gene CTLA4)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "RS231725", "target": "CTLA4"}, {"weight": 1.9, "description": "For rs3087243 polymorphism, while GA versus GG comparison showed some heterogeneity, no significant association with PBC susceptibility was identified.)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "RS3087243", "target": "GA VERSUS GG"}, {"weight": 1.9, "description": "The SNP rs3087243 shows association with the gene CTLA4)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "RS3087243", "target": "CTLA4"}, {"weight": 3.6, "description": "The TT + TC versus CC comparison for rs5742909 polymorphism revealed significant heterogeneity and linkage disequilibrium among the four SNPs.)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "RS5742909", "target": "TC VERSUS CC"}, {"weight": 3.6, "description": "SNP rs5742909 is linked to the gene CTLA4)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "RS5742909", "target": "CTLA4"}, {"weight": 1.0, "description": "Gene CTLA4 and its SNPs are related in studies to susceptibility for PBC disease)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "CTLA4", "target": "PBC"}, {"weight": 2.0, "description": "Shared trans-acting regulatory elements were identified across different cell types in multi-tissue analysis.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "MULTI-TISSUE ANALYSIS", "target": "TRANS-ACTING REGULATORY LOCUS"}, {"weight": 6.0, "description": "A specific region was influenced by GWAS SNPs, contributing to the regulation of gene expression.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "TRANS-ACTING REGULATORY LOCUS", "target": "GWAS SNPS"}, {"weight": 8.0, "description": "The RPS26 gene expression levels are regulated by a trans-acting region on chromosome 12 with broad impact.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "TRANS-ACTING REGULATORY LOCUS", "target": "RPS26"}, {"weight": 10.0, "description": "The RAB5B gene expression levels are affected by a shared regulatory locus across multiple cell types.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "TRANS-ACTING REGULATORY LOCUS", "target": "RAB5B"}, {"weight": 12.0, "description": "The IKZF4 gene is part of a region showing significant effect on multiple gene expressions due to regulatory elements.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "TRANS-ACTING REGULATORY LOCUS", "target": "IKZF4"}, {"weight": 9.0, "description": "CDK2 is one of the genes influenced through trans-expression quantitative trait loci analysis.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "CDK2", "target": "TRANS-EQTLS"}, {"weight": 11.0, "description": "The SUOX gene expression was analyzed through trans-eQTL studies indicating its association with T1D genetic risk factors.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "SUOX", "target": "TRANS-EQTL ANALYSIS"}, {"weight": 13.0, "description": "ERBB3 gene expression was associated with specific SNPs identified in the T1D susceptibility analysis.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "ERBB3", "target": "SNPS ASSOCIATION"}, {"weight": 14.0, "description": "The IL27 gene expression levels were determined through trans-eQTL effects, linking it to genetic variation influencing immune response.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "IL27", "target": "TRANS-EQTLS"}, {"weight": 16.0, "description": "The STAT1 gene expression was influenced through both trans-eQTL effects and GWAS, indicating multiple genetic influences on its activity.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "STAT1", "target": "TRANS-EQTLS AND GWAS SNPS"}, {"weight": 5.0, "description": "The rs181206[G] allele also affects STAT1 expression, leading to lower transcript levels in comparison to the Leu119 variant of IL-27.)", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "order": 1, "source": "STAT1", "target": "RS181206[G]"}, {"weight": 1.0, "description": "The multi-tissue analysis was advantageous for identifying genes with lower effect sizes or incomplete power due to its capacity for accounting for such scenarios.)(entity", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "order": 1, "source": "METHODOLOGICAL ADVANTAGES", "target": "MULTI-TISSUE JOINT CIS-EQTL ANALYSIS STRENGTHS"}, {"weight": 3.0, "description": "The rs181206[G] allele in the IL27 gene is associated with T1D (Type 1 Diabetes) susceptibility.)", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "order": 1, "source": "RS181206[G]", "target": "T1D SUSCEPTIBILITY"}, {"weight": 2.0, "description": "IL-8 protein is involved in the immune system\'s function by attracting immune cells and activating them.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "IL-8 PROTEIN", "target": "IMMUNE SYSTEM"}, {"weight": 4.0, "description": "Increased IL-8 mRNA expression is part of the inflammatory response mechanism in asthmatic patients.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "IL-8 MRNA", "target": "INFLAMMATORY RESPONSE"}, {"weight": 3.0, "description": "The rs4073 variant may contribute to a specific genetic haplotype associated with asthma risk.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "RS4073", "target": "GENETIC HAPLOTYPE"}, {"weight": 1.0, "description": "Serine metabolism contributes to the inflammatory response, including IL-8 release.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "IL-8 SERINE", "target": "INFLAMMATORY RESPONSE"}, {"weight": 5.0, "description": "The 86 HapMap SNPs are part of the ASAH1 gene region on chromosome 8p22-p21.3.)", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "order": 1, "source": "ASAH1 GENE", "target": "HAPMAP SNPS"}, {"weight": 4.0, "description": "A SNP-set was created using the kernel machine method for assessing potential causal SNPs in the context of GWAS studies.)", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "order": 1, "source": "SNP-SET", "target": "KERNEL MACHINE METHOD"}, {"weight": 2.0, "description": "The kernel machine method outperforms the min test in scenarios where causal SNPs are moderately correlated with typed SNPs.)", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "order": 1, "source": "MIN TEST", "target": "KERNEL MACHINE METHOD"}, {"weight": -0.0566, "description": "Both min test and linear kernel suffered from substantial loss of power as they mis-specified the model compared to the IBS and weighted IBS kernels.)", "source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "order": 1, "source": "MIN TEST", "target": "LINEAR KERNEL& STANDARD IBS KERNEL"}, {"weight": 0.7524, "description": "The kernel machine SNP-set test had better performance than the min test, especially when there was good LD structure similar to that of the ASAH1 gene.)<SEP>The kernel machine SNP-set test outperformed the min test for studying gene effects based on imputed data, especially when there was good linkage disequilibrium (LD) structure like in ASAH1.)", "source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "order": 1, "source": "MIN TEST", "target": "KERNEL MACHINE SNP-SET TEST"}, {"weight": 0.634, "description": "The min test suffered from substantial loss of power compared to the other kernels due to mis-specifying the model.)", "source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "order": 1, "source": "MIN TEST", "target": "LINEAR KERNEL& IBS KERNEL& WEIGHTED IBS KERNEL"}, {"weight": 0.634, "description": "When the SNP with higher Minor Allele Frequency (MAF) was causal, the weighted IBS kernel showed lower power than the IBS kernel as it downweighted the effect of more common alleles.)", "source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "order": 1, "source": "LINEAR KERNEL", "target": "IBS KERNEL"}, {"weight": -0.214, "description": "The linear kernel and weighted IBS kernel had lower power compared to the standard IBS kernel, especially when causal SNPs have higher MAFs.)", "source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "order": 1, "source": "LINEAR KERNEL", "target": "IBS KERNEL& WEIGHTED IBS KERNEL"}, {"weight": 8.780000000000001, "description": "The weighted IBS kernel had higher power than the standard IBS kernel when incorporating information on rare variants.)<SEP>The weighted IBS kernel had much greater power compared to the linear kernel and standard IBS kernel due to incorporating information on rare variants.)<SEP>The weighted IBS kernel had similar power to the standard IBS kernel in most cases.)", "source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "order": 1, "source": "IBS KERNEL", "target": "WEIGHTED IBS KERNEL"}, {"weight": 6.0, "description": "In GWAS contexts, the weighted IBS kernel emphasizes rare alleles by incorporating their importance through MAF (minor allele frequency), impacting test outcomes when rare variants are causal.)", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "order": 1, "source": "WEIGHTED IBS KERNEL", "target": "RARE ALLELES"}, {"weight": 3.0, "description": "Median R2 represents a commonly used measure of linkage disequilibrium (LD), which describes genetic correlation among SNPs.)", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "order": 1, "source": "LD MEASURE", "target": "MEDAN R2"}, {"weight": 16.0, "description": "Several associations were observed between autoimmune risk SNPs and response to both anti-CTLA-4 and anti-PD-1 treatments.)", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "order": 1, "source": "AUTOIMMUNE RISK SNPS", "target": "ASSOCIATION WITH RESPONSE"}, {"weight": 18.0, "description": "rs17388568 had the strongest association with response under the dominant regression model in the context of anti-PD-1 treatment.)", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "order": 1, "source": "RS17388568", "target": "ASSOCIATION WITH RESPONSE"}, {"weight": 2.0, "description": "A small sample size limits the power of an analysis to detect true associations, indicating that findings might not be reliable due to lower statistical confidence.)", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "order": 1, "source": "SMALL SAMPLE SIZE", "target": "POWER OF ANALYSIS"}, {"weight": 3.0, "description": "Combination therapy is being grouped with other treatments for association analysis; this does not establish a causal relationship but reflects the experimental design.)", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "order": 1, "source": "COMBINATION-THERAPY", "target": "GROUPING OF TREATMENTS"}, {"weight": 6.0, "description": "The specific treatment drug used has an impact on the patient\'s response to therapy.)", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "order": 1, "source": "TREATMENT DRUGS", "target": "IMPACT ON RESPONSE"}, {"weight": 1.0, "description": "EBNA2 was found to bind chromatin at MS risk loci dependent on the presence of a specific risk allele, influencing gene expression.)", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "EBNA2", "target": "MS RISK GENES"}, {"weight": 1.0, "description": "Since EBNA2 binds one allele more than the other at MS risk loci, it influences allelic associations with expression compared to that found in B cells.)", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "EBNA2", "target": "ALLELIC ASSOCIATION"}, {"weight": 1.0, "description": "The EBNA2 effect on gene expression was dependent upon MS risk loci genotypes.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "EBNA2", "target": "MS RISK LOCI"}, {"weight": 4.0, "description": "Expression of MS risk genes is associated with LCL proliferation, which can be modulated by EBNA2 inhibition.)", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "LCL PROLIFERATION", "target": "MS RISK SNPS AND GENES"}, {"weight": 5.0, "description": "A dose-response experiment was conducted to determine the optimal concentration of EBNA2-TAT that inhibits EBNA2 transcriptional function without excess cytotoxicity.)", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "EBNA2-TAT", "target": "DOSAGE RESPONSE"}, {"weight": 3.0, "description": "EBNA2 interacts specifically with the RBP-J\u03ba site on EBNA2, affecting gene expression through this interaction.)", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "MS RISK GENES", "target": "RBP-J\u039a INTERACTION"}, {"weight": 6.0, "description": "The EBNA2 interaction on MS risk genes led to changes in their expression levels, affecting immune evasion processes.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "MS RISK GENES", "target": "EBNA2 INTERACTION EFFECT"}, {"weight": 2.0, "description": "EBV DNA copy number is associated with several MS risk SNPs and genes in LCLs, which can be influenced by EBNA2-TAT treatment.)", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "EBV DNA COPY NUMBER", "target": "LCLS"}, {"weight": 2.0, "description": "The data suggest that TBX6 and RCOR1 are modulated by EBNA2 inhibition, providing insights into how EBNA2 might influence genetic risk factors for MS.)", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "GENETIC MODULATION", "target": "TBX6 AND RCOR1"}, {"weight": 2.0, "description": "EBNA2 effect on gene expression at MS risk loci could be ablated by EBNA2-TAT treatment.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "EBNA2 TREATMENT", "target": "GENOTYPE EFFECT"}, {"weight": 4.0, "description": "The effect on gene expression due to EBNA2 affects the program of EBV latency III infection cycle.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "PROGRAM OF EBV LATENCY III", "target": "RISK GENE EXPRESSION"}, {"weight": 1.0, "description": "EBNA2 interaction with MS risk alleles may increase host immune evasion by enhancing its function.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "EBNA2 BINDING", "target": "HOST IMMUNE EVASION"}, {"weight": 6.0, "description": "The association of COX-1 gene polymorphism rs3842787 with aspirin insensitivity was not significant)", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "order": 1, "source": "RS3842787", "target": "COX-1 GENE"}, {"weight": 1.0, "description": "There was a significant association between rs20417 in the COX-2 gene and aspirin insensitivity)", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "order": 1, "source": "RS20417", "target": "COX-2 GENE"}, {"weight": 4.0, "description": "Differences in the percentage of CAD between patients and controls explained a large part of heterogeneity for rs20417)", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "order": 1, "source": "RS20417", "target": "CAD PERCENTAGE"}, {"weight": 7.0, "description": "The influence of ITGA2B gene polymorphism rs20118426 on the response to aspirin could be assessed under subgroup analyses)", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "order": 1, "source": "RS20118426", "target": "ITGA2B GENE"}, {"weight": 2.0, "description": "Risk estimates by rs1126643-T allele were more potentiated among patients with aspirin semi-resistance compared with those with aspirin resistance)", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "order": 1, "source": "RS1126643", "target": "ITGA2 GENE"}, {"weight": 3.0, "description": "The odds of aspirin insensitivity in Chinese was nearly threefold relative to Caucasians under both allelic and dominant models)", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "order": 1, "source": "RS1126643", "target": "CHINESE POPULATION"}, {"weight": 5.0, "description": "Averaged platelet number was a significant source of heterogeneity for rs1126643)", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "order": 1, "source": "RS1126643", "target": "PLATELET NUMBER"}, {"weight": 0.012, "description": "The averaged platelet number is an important factor for the significance of rs1126643 in explaining heterogeneity.)", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea", "order": 1, "source": "RS1126643", "target": "AVERAGED PLATELET NUMBER"}, {"weight": 20.0, "description": "Whole Exome Sequencing (WES) was performed on patient 1 at age 3.9, revealing a novel homozygous nonsense STAT5B mutation.)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "IDENTIFICATION OF NOVEL MUTATION", "target": "WES ANALYSIS"}, {"weight": 15.0, "description": "The WES analysis showed heterozygosity in the STAT5B locus in both parents, implying they are carriers of the variant.)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "PARENTAL HETEROZYGOSITY", "target": "TRIO CLINICAL WHOLE EXOME SEQUENCING"}, {"weight": 10.0, "description": "The new mutation is adjacent to the first identified mutation in STAT5B, p.A630P.)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "COMPARISON WITH IDENTIFIED MUTATIONS", "target": "NOVEL STAT5B MUTATION LOCATION"}, {"weight": 10.0, "description": "The truncated STAT5B protein lacks ~50% of the Src Homology 2 (SH2) domain and the critical phosphorylated tyrosine at position 699.)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "ROLE OF SH2 DOMAIN", "target": "SH2 DOMAIN IMPAIRMENT"}, {"weight": 10.0, "description": "The mutation results in the absence of the entire transactivation domain (TAD).)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "ROLE OF TAD DOMAIN", "target": "TAD DOMAIN ABSENCE"}, {"weight": 20.0, "description": "In vitro reconstitution analyses showed significantly reduced expression of the truncated STAT5B protein.)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "EXPRESSION ANALYSIS", "target": "REDUCED EXPRESSION OF P.W631* PROTEIN"}, {"weight": 4.0, "description": "Algorithm V had a better positive predictive value than algorithms II, III and IV when tested on the 1000 Genomes dataset.)", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "ALGORITHM V", "target": "POSITIVE PREDICTIVE VALUE"}, {"weight": 11.0, "description": "Algorithm V did not perfectly predict the haplotype carrier status, as evidenced by a sensitivity and specificity of less than 100% when tagging the haplotype.)", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "ALGORITHM V", "target": "TAGGER ACCURACY"}, {"weight": 5.0, "description": "The performance of the HLA-DRB1*07:01-DQA1*02:01-DQB1*02:02 haplotype as a screening test was evaluated against E. coli ASP hypersensitivity.)", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "HLA-DRB1*07:01\u2013DQA1*02:01\u2013DQB1*02:02 HAPLOTYPE", "target": "SCREENING TEST PERFORMANCE"}, {"weight": 6.0, "description": "The sensitivity of the HLA haplotype to correctly identify affected patients was 23.4%.)", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "HLA-DRB1*07:01\u2013DQA1*02:01\u2013DQB1*02:02 HAPLOTYPE", "target": "SENSITIVITY VALUE"}, {"weight": 7.0, "description": "The specificity of the HLA haplotype was high at 88.5%, indicating a low rate of false positives.)", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "HLA-DRB1*07:01\u2013DQA1*02:01\u2013DQB1*02:02 HAPLOTYPE", "target": "SPECIFICITY"}, {"weight": 8.0, "description": "The positive predictive value for the HLA haplotype was 57.9%, indicating that among those identified as having the haplotype, a high percentage were indeed affected by E. coli ASP hypersensitivity.)", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "HLA-DRB1*07:01\u2013DQA1*02:01\u2013DQB1*02:02 HAPLOTYPE", "target": "POSITIVE PREDICTIVE VALUE"}, {"weight": 9.0, "description": "The negative predictive value of the HLA haplotype was 63.1%, indicating that among those not identified as having the haplotype, a high percentage did not have E. coli ASP hypersensitivity.)", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "HLA-DRB1*07:01\u2013DQA1*02:01\u2013DQB1*02:02 HAPLOTYPE", "target": "NEGATIVE PREDICTIVE VALUE"}, {"weight": 10.0, "description": "The HLA haplotype and its best tagger algorithm (V) had the same performance parameters.)", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "HLA-DRB1*07:01\u2013DQA1*02:01\u2013DQB1*02:02 HAPLOTYPE", "target": "IDENTICAL PERFORMANCE MEASURES"}, {"weight": 1.0, "description": "Polymorphisms rs179008 and rs179009 in the TLR7 gene were significantly reduced or associated with HIV-infected individuals compared to healthy controls.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "HIV-INFECTED INDIVIDUALS", "target": "TLR7 POLYMORPHISMS"}, {"weight": 2.0, "description": "The rs179009 AG genotype was less frequent among HIV-infected individuals than in the control group, with a reduced risk of HIV infection.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "HIV-INFECTED INDIVIDUALS", "target": "TLR7 POLYMORPHISM (RS179009 AG GENOTYPE)"}, {"weight": 3.0, "description": "The distribution of haplotype AG was found higher in HIV-infected individuals compared to healthy controls, indicating a possible association with the disease.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "HIV-INFECTED INDIVIDUALS", "target": "TLR7 HAPLOTYPES (RS179008 AND RS179009)"}, {"weight": 4.0, "description": "Polymorphism rs179009 showed decreased frequency in individuals across early, intermediate, and advanced HIV stages compared to the healthy control group.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "HIV DISEASE STAGES", "target": "TLR7 POLYMORPHISMS (RS179009)"}, {"weight": 5.0, "description": "The frequency of rs179009 GG genotype was lower in individuals with advanced HIV disease stages compared to the healthy control group, suggesting a possible correlation.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "ADVANCED HIV DISEASE STAGE", "target": "TLR7 POLYMORPHISMS (RS179009 GG GENOTYPE)"}, {"weight": 6.0, "description": "Polymorphism distributions did not significantly differ between HIV-infected individuals with or without tobacco use; however, a higher prevalence of TLR7 rs179009AG genotype was noted among those who consumed tobacco.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "ENVIRONMENTAL FACTORS (TOBACCO CONSUMPTION, ALCOHOL USE)", "target": "TLR7 POLYMORPHISMS (RS179008 AND RS179009)"}, {"weight": 11.0, "description": "The effects of environmental factors like tobacco use or alcohol intake on TLR7 polymorphisms were analyzed using multivariate logistic regression to identify potential interactions.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "ENVIRONMENTAL FACTORS (TOBACCO CONSUMPTION, ALCOHOL USE)", "target": "MULTIVARIATE LOGISTIC REGRESSION ANALYSIS"}, {"weight": 9.0, "description": "These genetic variations were analyzed through multivariate logistic regression for associations with various factors including age, sex, tobacco use, alcohol consumption, and baseline CD4 counts.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "TLR7 POLYMORPHISMS (RS179008 AND RS179009)", "target": "MULTIVARIATE LOGISTIC REGRESSION ANALYSIS"}, {"weight": 7.0, "description": "The distributions of rs179008 CC, CT, and TT genotypes were nearly equivalent between tobacco-consuming and nonconsuming HIV-infected individuals.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "TLR7 POLYMORPHISMS (RS179008 CC, CT, TT GENOTYPES)", "target": "TOBACCO-CONSUMING HIV-INFECTED INDIVIDUALS VS. NON-USERS"}, {"weight": 14.0, "description": "Comparative analysis was conducted between individuals who consumed tobacco and those who did not, with regard to TLR7 polymorphism distribution.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "TOBACCO-CONSUMING HIV-INFECTED INDIVIDUALS VS. NON-USERS", "target": "COMPARISONS"}, {"weight": 8.0, "description": "Similar distributions of rs179008 AA, AT, and TT and rs179009 AA, AG, and GG genotypes were observed between alcohol-taking HIV-infected individuals and non-takers.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "ALCOHOL-TAKING HIV-INFECTED INDIVIDUALS VS. NON-USERS", "target": "TLR7 POLYMORPHISMS (RS179009 AA, AG, GG GENOTYPES)"}, {"weight": 15.0, "description": "Similar comparisons were made for alcohol consumption in terms of its association with the distribution of specific TLR7 genotypes among HIV-infected individuals.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "ALCOHOL-TAKING HIV-INFECTED INDIVIDUALS VS. NON-USERS", "target": "COMPARISONS"}, {"weight": 12.0, "description": "The relationship between baseline CD4 cell count levels and different TLR7 polymorphisms was explored through this statistical method, aiming to uncover any significant correlations.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "MULTIVARIATE LOGISTIC REGRESSION ANALYSIS", "target": "BASELINE CD4 COUNTS AND TLR7 POLYMORPHISMS"}, {"weight": 1.0, "description": "Analysis of correlation between genetic variants in the TLR7 gene with HIV disease progression through multivariate logistic regression.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "MULTIVARIATE LOGISTIC REGRESSION ANALYSIS", "target": "TLR7 (RS179008 AND RS179009)"}, {"weight": 2.0, "description": "Both genetic variations were included in multivariate logistic regression analysis to assess their influence on HIV disease progression.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "MULTIVARIATE LOGISTIC REGRESSION ANALYSIS", "target": "RS179008 AND RS179009"}, {"weight": 4.0, "description": "The analysis was aimed at identifying significant predictors of HIV disease progression including genetic markers, environmental factors, and health status parameters.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "MULTIVARIATE LOGISTIC REGRESSION ANALYSIS", "target": "OUTCOME OF DISEASE PROGRESSION STUDY"}, {"weight": 2.0, "description": "These factors were considered as potential independent risk factors for HIV disease progression alongside rs179008 and rs179009 variants of the TLR7 gene.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "AGE, SEX, TOBACCO, ALCOHOL, BASELINE CD4 COUNTS", "target": "RISK FACTOR"}, {"weight": 3.0, "description": "These demographic and health indicators were considered as potential risk factors for HIV disease progression together with rs179008 and rs179009.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "RISK FACTOR", "target": "AGE, SEX, TOBACCO USE, ALCOHOL CONSUMPTION, BASELINE CD4 COUNTS"}, {"weight": 3.0, "description": "The results from this correlation analysis were not directly presented, implying they supported or refuted the role of studied factors in HIV disease progression.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "CORRELATION ANALYSIS RESULTS", "target": "TABLE NOT SHOWN"}, {"weight": 3.0, "description": "Both ALK and NRAS mutations are recurrent in various cancer types, including NBL.)", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "order": 1, "source": "ALK R1275Q", "target": "NRAS Q61K"}, {"weight": 4.0, "description": "The ALK R1275Q mutation leads to the generation of neoepitopes like ARDIYRASY, which is predicted to bind HLA-B*15:01.)", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "order": 1, "source": "ALK R1275Q", "target": "ALYRASY"}, {"weight": 5.0, "description": "The ALK R1275Q mutation generates the nonamer GMAQDIYRA that binds HLA-A*02:01.)", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "order": 1, "source": "ALK R1275Q", "target": "GMAQDIYRA"}, {"weight": 7.0, "description": "The ALK R1275Q mutation leads to the generation of two distinct neoepitopes, MAQDIYRASY and AQDIYRASYY.)", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "order": 1, "source": "ALK R1275Q", "target": "MAQDIYRASY/AQDIYRASYY"}, {"weight": 7.0, "description": "The potential of other HLAs to display ALK R1275Q neoepitopes depends on the individual\'s HLA haplotype, playing a role in immune response.)", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "order": 1, "source": "ALK R1275Q", "target": "HAPLOTYPE-SPECIFIC BINDING"}, {"weight": 6.0, "description": "The ALK R1275Q genomic variant is associated with cancer immunotherapies outcomes.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "ALK R1275Q", "target": "CANCER IMMUNOTHERAPIES"}, {"weight": 6.0, "description": "The NRAS Q61K activating mutation results in neoepitope ILDTAGKEEY, which is predicted to bind HLA-A*01:01.)", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "order": 1, "source": "NRAS Q61K", "target": "ILDTAGKEEY"}, {"weight": 9.0, "description": "Nonamer GMAQDIYRA, derived from ALK R1275Q mutation, forms a complex that exhibits distinct stability compared to other peptides.)", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "order": 1, "source": "GMAQDIYRA", "target": "STABILITY"}, {"weight": 3.0, "description": "The ALK nonamer neoepitope binds to HLA-B*15:01 through in vitro refolding.)", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "order": 1, "source": "ALK NONAMER NEOEPITOPE", "target": "BOUND BY HLA-B*15:01"}, {"weight": 12.0, "description": "The recognition frequencies for the nonamer and a known immunodominant SARS-CoV-2 epitope are comparable when tested using MHC tetramers.)", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "order": 1, "source": "NONAMER", "target": "COMPARISON WITH SARS-COV-2 DERIVED EPITOPES"}, {"weight": 4.0, "description": "The ALK decamer neoepitope binds to HLA-B*15:01 via a compact 310 helical conformation.)", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "order": 1, "source": "ALK DECAMER NEOEPITOPE", "target": "BOUND BY HLA-B*15:01"}, {"weight": 9.0, "description": "The thermal stability measurement reflects the stability of the p/MHC complex in vitro.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "THERMAL STABILITY", "target": "PHYSICAL PROPERTY"}, {"weight": 1.0, "description": "The addition of a C-terminal tyrosine residue alters the tertiary structure of HLA complex through conformational plasticity within the MHC groove.)", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "order": 1, "source": "NONAMER PEPTIDE", "target": "DECAMER STRUCTURE"}, {"weight": 6.0, "description": "The self-antigen\'s structure results in clashes with MHC groove, leading to its exclusion from forming a stable HLA complex compared to neoepitopes.)", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "order": 1, "source": "SELF-ANTIGEN", "target": "STRUCTURAL EXCLUSION"}, {"weight": 3.0, "description": "Increase in intra-peptide hydrogen bonds contribute to the stability of the decamer complex compared to nonamer structure.)", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "order": 1, "source": "HYDROGEN BONDS", "target": "STABILIZING INTERACTION"}, {"weight": 3.0, "description": "There\'s a clear sequence bias for specific residues in the MHC-binding groove yielding more favorable interactions with two peptides.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "HLA ALLELES", "target": "SEQUENCE BIAS"}, {"weight": 4.0, "description": "We find a clear trend for both nonamer and decamer peptides that bulk of binding energy provided by \'anchor\' positions.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "NONAMER ALK PEPTIDE", "target": "DECAMER ALK PEPTIDE"}, {"weight": 7.0, "description": "Our simulations accurately predict peptide/HLA binding energies and interactions based on sequence similarity.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "PEPTIDE BINDING ENERGY", "target": "PREDICTION ACCURACY"}, {"weight": 10.0, "description": "The sequence logo provides insights into preferred residues for effective peptide/HLA binding.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "KULLBACK\u2013LEIBLER SEQUENCE LOGO", "target": "SEQUENCE ANALYSIS"}, {"weight": 10.0, "description": "The ALK-derived neoepitopes can be recognized by CD8+ T cells, as evidenced by MHC tetramer staining assay and multichannel flow cytometry analysis.)", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "order": 1, "source": "ALK NEOEPITOPES", "target": "RECOGNITION BY CD8+ T CELLS"}, {"weight": 9.0, "description": "The binding energy calculations suggest that both neoepitopes have a lower affinity for HLA-A*01:01 compared to HLA-B*15:01.)", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "order": 1, "source": "DECAMER AND NONAMER", "target": "BINDING ENERGY"}, {"weight": 11.0, "description": "When bound to HLA-A*01:01, the side chain of Arg6 flips outward in comparison to its position when bound to HLA-B*15:01, altering TCR interaction properties.)", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "order": 1, "source": "DECAMER AND NONAMER", "target": "CONFORMATIONAL COMPARISON"}, {"weight": 13.0, "description": "Both neoepitopes display nominal recognition frequencies by CD8+ T cells, highlighting their potential as targets for immune response study.)", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "order": 1, "source": "DECAMER AND NONAMER", "target": "RECOGNITION EFFICIENCY COMPARISON"}, {"weight": 3.0, "description": "Vitamin D plays a role in calcium metabolism by enhancing the absorption of calcium in the gut.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "VITAMIN D", "target": "CALCIUM METABOLISM"}, {"weight": 4.0, "description": "Regular physical training helps reduce oxidative stress by increasing antioxidant defenses and improving overall metabolic health.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "PHYSICAL TRAINING", "target": "OXIDATIVE STRESS"}, {"weight": 6.0, "description": "Disruptions in calcium metabolism are associated with cardiomyopathy due to its critical role in heart muscle function.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "CARDIOMYOPATHY", "target": "CALCIUM METABOLISM"}, {"weight": 8.0, "description": "Alterations in glycolysis pathways can affect heart function, potentially leading to cardiomyopathy or exacerbating its symptoms.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "CARDIOMYOPATHY", "target": "GLYCOLYSIS"}, {"weight": 2.0, "description": "The ALK nonamer and decamer neoepitopes adopt distinct conformations within the MHC groove: extended for the nonamer and compact 310 helix for the decamer.)", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "order": 1, "source": "ALK NONAMER NEOEPITOPE/DECAMER NEOEPITOPE", "target": "CONFORMATION"}, {"weight": 1.0, "description": "A comparison between the ALK nonamer (nine amino acids) and decamer (ten amino acids), showing how peptide length influences conformation within a fixed MHC groove.)", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "order": 1, "source": "LENGTH", "target": "NONAMER/DECAMER"}, {"weight": 3.0, "description": "Epitopes", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "order": 1, "source": "BIND TO", "target": "MHC MOLECULES"}, {"weight": 2.0, "description": "Antigen Presentation", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "order": 1, "source": "ANTIGEN-PRESENTING CELLS (APCS)", "target": "PROCESS OF"}, {"weight": 4.0, "description": "Peptides Present On MHC molecules", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "order": 1, "source": "RECOGNIZE AND BIND TO", "target": "TCR"}, {"weight": 2.0, "description": "Longer arginine side chain causes steric hindrance between peptide and MHC-binding groove, affecting affinity for self-peptide binding to HLA-B*15:01.)", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "order": 1, "source": "ARG2 SIDE CHAIN", "target": "STRUCTURAL HINDRANCE"}, {"weight": 4.0, "description": "Polar interactions such as hydrogen bonds and salt bridges enhance stability of HLA complexes with peptides.)", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "order": 1, "source": "POLAR INTERACTIONS", "target": "STABILIZING FORCES"}, {"weight": 5.0, "description": "Additional hydrophobic packing interactions further stabilize the decamer complex by reducing electrostatic repulsion between hydrophobic side chains.)", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "order": 1, "source": "STABILIZING FORCES", "target": "HYDROPHOBIC PACKING"}, {"weight": 8.0, "description": "Comparison of affinities between self-peptide and neoepitopes for HLA-B*15:01 highlights potential differences in immunogenicity and therapeutic relevance.)", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "order": 1, "source": "AFFINITY DIFFERENCES", "target": "P/MHC COMPLEXES"}, {"weight": 10.0, "description": "Increase in binding energy and structural stability provided by the decamer configuration results from a more compact conformation and enhanced interactions with HLA-B*15:01.)", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "order": 1, "source": "DECAMER PRESENTATION", "target": "ENHANCED STABILITY"}, {"weight": 5.0, "description": "The mock polyAla HLA exhibits considerably higher binding energy across the entire peptide length compared to top binders.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "HIGH BINDING ENERGY", "target": "MOCK HLA ALLELE"}, {"weight": 1.0, "description": "SNP rs74420772 was genome-wide significant in TwinGene associated with IgM anti-PC measurements.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "RS74420772", "target": "IGM ANTI-PC"}, {"weight": 6.0, "description": "IgM anti-PC was lower in prevalent CLL cases compared to matched controls, with a declining trend between sampling and diagnosis but not statistically significant.)", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "order": 1, "source": "IGM ANTI-PC", "target": "CLL CASE ASSOCIATION"}, {"weight": 5.0, "description": "A small nested case-control study found IgM anti-PC to be lower in prevalent cases of CLL compared to matched controls, but not significantly associated with incident cases using stratified Cox proportional hazards model.)", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "order": 1, "source": "IGM ANTI-PC", "target": "PREVALENT CLL ASSOCIATION"}, {"weight": 3.0, "description": "rs35923643-G is part of the LD block overlapping with GRAM domain containing 1B (GRAMD1B) gene.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "RS35923643-G", "target": "11Q24.1"}, {"weight": 7.0, "description": "The SNP rs735665 was best-fitted and explained the largest variance of CLL risk in polygenic risk score analysis.)<SEP>The allele rs35923643-G or proxy SNP rs735665-A was recognized as the top risk allele for CLL.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "RS735665-A", "target": "CLL"}, {"weight": 4.0, "description": "rs735665-A was identified as a genome-wide significant SNP associated with general IgM increasing alleles related to IgM anti-PC.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "RS735665-A", "target": "GENERAL IGM AND IGM ANTI-PC"}, {"weight": 1.0, "description": "The allele rs735665-A was associated not only with CLL but also showed a negative association with the risk of certain disease states involving general IgM, general IgA, or general IgG levels.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "RS735665-A", "target": "GENERAL IGM AND GENERAL IGA/IGG"}, {"weight": 5.0, "description": "The RegulomeDB database gave support for rs35923643 affecting the binding of transcription factors in general.)", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "order": 1, "source": "RS35923643", "target": "REGULOMEDB SUPPORT"}, {"weight": 3.0, "description": "ChIP-seq experiments suggested binding sites for 22 different transcription factors within the region around rs35923643.)", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "order": 1, "source": "TF BINDING", "target": "TRANSCRIPTION FACTORS"}, {"weight": 4.0, "description": "The strongest signals were observed for RUNX3 and SPI1 from FIMO tool, with the RUNX3 binding motif including SNP rs35923643.)", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "order": 1, "source": "BINDING MOTIFS", "target": "RUNX3 AND SPI1 TFS"}, {"weight": 4.0, "description": "The Encyclopedia of DNA Elements (ENCODE) provided data on transcription factor binding within the vicinity of rs35923643, identifying suggested binding sites for 22 different TFs.)", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "order": 1, "source": "ENCODE DATA", "target": "TFS"}, {"weight": 6.0, "description": "An analysis revealed a non-statistically significant declining trend in IgM anti-PC levels between CLL cases and matched controls as time from sampling to diagnosis increased.)", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "order": 1, "source": "STATISTICAL TREND", "target": "TIME BETWEEN SAMPLING AND DIAGNOSIS"}]}