var graphJson = {"directed": false, "multigraph": false, "graph": {"node_default": {}, "edge_default": {}}, "nodes": [{"entity_type": "GENE", "description": "The SEC16B gene is involved in endoplasmic reticulum export.", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "id": "SEC16 HOMOLOG B"}, {"entity_type": "GENE", "description": "DNAJC27, a member of the DnaJ heat shock protein family, plays a role in cellular processes.", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "id": "DNAJ HEAT SHOCK PROTEIN FAMILY (HSP40) MEMBER C27"}, {"entity_type": "GENE", "description": "CDKAL1 is associated with CDK5 and its function in metabolic regulation.", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "id": "CDK5 REGULATORY SUBUNIT-ASSOCIATED PROTEIN 1-LIKE 1"}, {"entity_type": "GENE", "description": "TFAP2B acts as a transcription factor influencing gene expression.", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "id": "TRANSCRIPTION FACTOR AP-2 BETA"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs543874 affects the function of SEC16B gene.", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "id": "RS543874"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs713586 is related to the expression levels of DNAJC27 gene.", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "id": "RS713586"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs9356744 influences the activity of CDKAL1 transcription factor.", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "id": "RS9356744"}, {"entity_type": "GENOMIC VARIANT", "description": "All mentioned identifiers are Single Nucleotide Polymorphisms (SNPs) associated with different genes or gene functions related to obesity risk.", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "id": "RS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Endoplasmic reticulum is a key organelle involved in protein synthesis and transport.", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 43}]", "id": "ENDOPLASMIC RETICULUM"}, {"entity_type": "BIOLOGICAL FUNCTION", "description": "The DnaJ heat shock protein family contributes to various cellular processes including stress response and protein folding.", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "id": "CELLULAR PROCESSES"}, {"entity_type": "MOLECULAR FUNCTION", "description": "CDKAL1 influences metabolic pathways by regulating gene expression.", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "id": "METABOLIC REGULATION"}, {"entity_type": "FUNCTIONAL ROLE/", "description": "TFAP2B is responsible for regulating the transcription of specific genes.<SEP>Response to Stimuli/", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20<SEP>chunk-bb351b97d99fce93d17df522db92dda7", "id": "TRANSCRIPTION FACTOR ACTIVITY"}, {"entity_type": "GENOMIC VARIANT", "description": "These SNPs are variations found within the DNA sequence that can affect gene function and contribute to obesity risk.", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "id": "SINGLE NUCLEOTIDE POLYMORPHISM (SNP)"}, {"entity_type": "GENE", "description": "SEC16 Homolog B plays a role as an endoplasmic reticulum export factor.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "SEC16B (SEC16B)"}, {"entity_type": "GENE", "description": "DnaJ heat shock protein family member C27 is involved in the process.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "DNAJC27 (DNAJC27)"}, {"entity_type": "GENE", "description": "CDK5 regulatory subunit-associated protein 1-like 1 modulates transcription factors related to obesity risk.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "CDKAL1 (CDKAL1)"}, {"entity_type": "GENE", "description": "Transcription factor AP-2 Beta is involved in gene regulation and impacts obesity.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "TFAP2B (TFAP2B)"}, {"entity_type": "GENE", "description": "Brain-Derived Neurotrophic Factor has a role in neural development and affects the obesity risk through SNPs.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "BDNF (BDNF)"}, {"entity_type": "GENE", "description": "Myosin light chain-2 is related to energy expenditure, impacting obesity.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "MYL2 (MYL2)"}, {"entity_type": "GENE", "description": "Olfactomedin-4 influences obesity through its gene variants.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "OLFM4 (OLFM4)"}, {"entity_type": "GENE", "description": "Fat mass- and obesity-associated protein affects metabolism related to obesity risk.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "FTO (FTO)"}, {"entity_type": "GENE", "description": "Melanocortin 4 receptor gene plays a role in energy intake regulation and obesity.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "MC4R (MC4R)"}, {"entity_type": "GENE", "description": "Glucose-dependent insulinotropic polypeptide receptor contributes to glucose metabolism, impacting obesity risk.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "GIPR (GIPR)"}, {"entity_type": "COVARIATE FACTOR", "description": "Not directly associated with obesity risk but interacts with PRS through gender.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "MENOPAUSE AGE"}, {"entity_type": "GENE", "description": "Solute Carrier Family 27 Member 5 is involved in lipid metabolism impacting obesity.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "SLC27A5 (SLC27A5)"}, {"entity_type": "MICRORNA", "description": "MicroRNA-1308 regulates gene expression related to energy balance and obesity risk.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "MIR1308 (MIR1308)"}, {"entity_type": "GENE", "description": "Tetraspanin 9, also known as CD82, plays a role in cell signaling pathways relevant to obesity.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "TSPAN9 (TSPAN9)"}, {"entity_type": "GENE", "description": "Adiponectin Receptor 1 modulates glucose metabolism influencing obesity risk.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "ADIPOR1 (ADIPOR1)"}, {"entity_type": "ASSOCIATION WITH OBESITY-SPECIFIC GENES", "description": "SNP rs9653047 is linked to regulation of genes like CDH13, SLC27A5, ADIPOR1, and others through its role in metabolic pathways.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "SNP RS9653047"}, {"entity_type": "GENE", "description": "Polyhomeotic 1 gene affects chromatin structure contributing to obesity risk via epigenetic mechanisms.", "source_id": "chunk-eec0b4c60ebeb28599428b3552de2d1b", "id": "PHC1 (PHC1)"}, {"entity_type": "GENOMIC VARIANT", "description": "PRS was associated with obesity risk.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 35}]", "id": "PRS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Early menarche age interacted with PRS for the obesity risk.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 35}]", "id": "MENARCHE AGE"}, {"entity_type": "BODY MEASUREMENT", "description": "A measure that calculates the body weight divided by height squared to assess if a person is underweight, normal weight, overweight or obese.<SEP>BMI was higher in participants with an early menarche age compared to those with late menarche age when adjusting for covariates.<SEP>Baseline characteristics had no association with genotype.<SEP>Increased BMI was observed in subjects carrying the A-allele of the FTO rs9939609 polymorphism compared to TT subjects.<SEP>Mean BMI was not significantly different between T2D-ESRD case subjects and nondiabetic control subjects (P = 0.19), suggesting that BMI may not affect their respective conditions.<SEP>No evidence for interaction with BMI in relation to the association between TCF7L2 gene and the number of risk alleles.<SEP>Basal BMI level and its change are influenced by obesity predisposing SNPs, GRS, and glycemic status traits.<SEP>Body mass index is associated with baseline glycemic status and changes over time, influencing health outcomes.<SEP>Individuals with lower BMI were observed to adhere more to the Mediterranean diet.<SEP>Mediator in the pathway analysis of genetic loci affecting cardiometabolic phenotypes after adjustment.<SEP>Body Mass Index (BMI), a measure of adiposity, was assessed and analyzed in the study.<SEP>BMI had a moderate genetic correlation with snoring, reflecting obesity as a risk factor for both traits.<SEP>Body mass index (BMI), calculated as weight in kilograms divided by height squared, was significantly lower among women with PCOS compared to controls.<SEP>Body Mass Index, a measure of body weight relative to height.<SEP>The association between rs10399931 and BMI was found to be significant (p = 0.03), indicating the genetic variant\'s potential impact on body mass index.<SEP>Participants who consumed black or sugared coffee were more likely to have a higher BMI.<SEP>BMI differed according to the rs671 polymorphism significantly among men.<SEP>Body Mass Index (BMI) is significantly associated with higher sUA levels and increased HUA risk in both males and females.<SEP>Asthmatic patients had a significantly higher BMI than the healthy control group.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43<SEP>chunk-1f20dbdcc9de443dada1a2045dc7832b<SEP>chunk-9977706363cc368b3be503d07ed3ef62<SEP>chunk-c6fdc5ed25c4e17b8c421a952bb05450<SEP>chunk-2926e6216bf945b12ea4daa6345cad1f<SEP>chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00<SEP>chunk-6f7ce1c71bd46f223756da44b0d0bb2c<SEP>chunk-6a7c69534e11aa2fc4bd948484aa81c4<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d<SEP>chunk-2c178ae99eeffdddbea511cafbe4a7d5<SEP>chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-bb507a90a1f5d5ddd01a93200ff2d45e<SEP>chunk-3135befd0c3140ad097d96ace6de86a1<SEP>chunk-372184452f1f9156514992455057f843<SEP>chunk-3ad8ef794841d749e48b3223741e91d3<SEP>chunk-c29d575e156569afaa1ca3d6418a71b9<SEP>chunk-44a871da300abc9be525b468f502f5cb<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a<SEP>chunk-1fde08e52a188ac6a549e7c11d59a092", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 34}]", "id": "BMI"}, {"entity_type": "NUTRIENT INTAKE AND DIETARY PATTERN", "description": "There was an interaction between PRS and a plant-based diet intake affecting obesity risk.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "PLANT-BASED DIET INTAKE"}, {"entity_type": "DIETARY PATTERN", "description": "In low plant-based diet intake, the genetic impact on BMI was lower compared to participants with high PRS and high plant-based dietary intake.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "LOW PLANT-BASED DIET INTAKE"}, {"entity_type": "DISEASE RISK", "description": "Adults with a high PRS were protected against obesity risk when following a high plant-based diet.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "HIGH PRS ADULTS"}, {"entity_type": "OBESITY RISK", "description": "Participants with low plant-based dietary intake and high PRS had higher BMI compared to those with low PRS and high plant-based dietary intake.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "clusters": "[{level: 0, cluster: 7}]", "id": "LOW PLANT-BASED DIET INTAKE & HIGH PRS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "High fried-food intake was associated with a higher BMI, but the PRS impact was offset in participants with high fried-food intake.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "HIGH FRIED-FOOD INTAKE"}, {"entity_type": "LIFESTYLE CHOICE", "description": "Fried-food intake should be avoided to protect against obesity risk.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "AVOIDING FRIED-FOOD INTAKE"}, {"entity_type": "OUTCOME", "description": "The PRS, diet type (specifically plant-based and fried-food intake), influenced obesity risk.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "clusters": "[{level: 0, cluster: 7}]", "id": "OBESITY RISK"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Early menarche is a stage of human development characterized by the onset of menstruation.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "EARLY MENARCHE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Late menarche refers to delayed onset of menstruation, often associated with various health conditions.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "LATE MENARCHE"}, {"entity_type": "STATISTICAL METHODOLOGY COMPONENT", "description": "BMI, age, sex, and other factors like smoking and alcohol intake were included as covariates in GWAS analysis to adjust for their potential effects on snoring.<SEP>Covariates are other variables considered in statistical analyses that could influence the relationship being studied.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62<SEP>chunk-3ad8ef794841d749e48b3223741e91d3", "id": "COVARIATES"}, {"entity_type": "RISK FACTOR", "description": "A measure of body size that\'s calculated using weight and height.<SEP>BMI is a measure of body fat based on height and weight, used to assess obesity or underweight conditions.<SEP>A measure that calculates body weight divided by height squared used for assessing if a person is underweight, normal weight, overweight or obese.<SEP>BMI showed a positive correlation with snoring, with increased BMI associated with increased snoring risk.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d<SEP>chunk-3135befd0c3140ad097d96ace6de86a1<SEP>chunk-3ad8ef794841d749e48b3223741e91d3", "clusters": "[{level: 0, cluster: 7}]", "id": "BMI (BODY MASS INDEX)"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "A complex condition characterized by the accumulation of excess body fat leading to health risks such as heart disease and type 2 diabetes.<SEP>A higher prevalence of obesity was noted among subjects with an A-allele (compared to TT subjects) based on the FTO rs9939609 polymorphism in the GP.<SEP>Obesity is a condition that may be influenced by genetic variations like the FTO SNP and education level.<SEP>Obesity is a medical condition characterized by an excess of body fat that may impact health negatively.<SEP>Excess body weight is a risk factor for developing IFG/IGT and T2D, impacting glucose metabolism processes.<SEP>Obesity is associated with higher serum uric acid (sUA) levels and increased HUA risk.", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1<SEP>chunk-3ad8ef794841d749e48b3223741e91d3<SEP>chunk-bb507a90a1f5d5ddd01a93200ff2d45e<SEP>chunk-d0476488384aadcae4630e94d982d2e0<SEP>chunk-63d0b5be485a69f44e2799ccc9510487<SEP>chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "id": "OBESITY"}, {"entity_type": "DIETARY PATTERN", "description": "A plant-based diet primarily consists of foods derived from plants such as fruits, vegetables, legumes, and grains.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "PLANT-BASED DIET"}, {"entity_type": "NUTRIENT INTAKE AND DIETARY PATTERN", "description": "Consumption of a high amount of plant-based food items in the diet.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "HIGH PLANT-BASED DIET INTAKE"}, {"entity_type": "GENOMIC VARIANT", "description": "A low PRS indicates a lower genetic predisposition towards certain health outcomes or diseases.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "LOW PRS (POLYGENIC RISK SCORE)"}, {"entity_type": "GENOMIC VARIANT", "description": "A high PRS suggests a higher genetic predisposition to certain health outcomes or disease susceptibility.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "HIGH PRS"}, {"entity_type": "NUTRIENT INTAKE AND DIETARY PATTERN", "description": "Consumption of a low amount of fried food in the diet, often associated with healthier eating habits.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "LOW FRIED-FOOD INTAKE"}, {"entity_type": "OUTCOME", "description": "Obesogenic effects refer to actions or factors that contribute to obesity or weight gain.", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "id": "OBESOGENIC EFFECTS"}, {"source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-46227b557f62c59155b1b9cef71d0d17<SEP>chunk-fa6293a0a10f175b4aebac8e885c0d20<SEP>chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "description": "A gene\'s different variants (genotypes) showed varied levels of TAGs.<SEP>SNPs within these genes may influence their function and impact the obesity risk.)<SEP>A highly significant variant for total energy expenditure (p\u200a=\u200a2.7E-08) was identified for MATK on chromosome 19.<SEP>Sleeping energy expenditure (p\u200a=\u200a6.0E-08) variant was identified for CHRNA3 on chromosome 15.<SEP>The Gene is involved in various biological processes and can influence disease development.<SEP>The OPRM1 gene plays a crucial role in mediating pain perception and reward pathways.", "entity_type": "GENE", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 43}]", "id": "GENE"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00<SEP>chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9<SEP>chunk-2fb95100a1290b27baae869c6560419a<SEP>chunk-5b0cc96debf110f3544685d3b227ff27<SEP>chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-fa6293a0a10f175b4aebac8e885c0d20<SEP>chunk-45a965c3c3013c70d0fa0f1680283083<SEP>chunk-b2730f3087aee6d9cff6e4939b229323", "description": "A genomic variant related to B12 metabolism that may influence its levels.<SEP>SNPs within these genes may influence their function and impact the obesity risk.)<SEP>The text doesn\'t provide details on the genomic variant specific to this context.<SEP>The specific genotypes or variants that are associated with abnormal blood glucose levels.<SEP>A specific variant found in this gene has been linked to certain disease susceptibility.<SEP>The CYP24A1 SNP rs927650 shows suggestive relationship with 25(OH)D levels.<SEP>The T/T genotype variant is associated with higher sUA levels compared to C/C carriers of the SLC2A9 gene.<SEP>rs8096658 at the NFATC1 locus has an effect on posterior probability (\u03c0C = 0.877) for driving association signals with eGFR.<SEP>The CC genotype is associated with an increased risk of Colorectal Cancer (CRC).<SEP>This variant has been linked to increased risk of hypertension.", "entity_type": "GENOMIC VARIANT", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 38}]", "id": "GENOMIC VARIANT"}, {"source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "description": "SNP rs713586 affects the activity levels of DNAJC27 involved in cellular stress response and protein folding.)", "entity_type": "UNKNOWN", "id": "DNAJ HEAT SHOCK PROTEIN FAMILY (HSP40)"}, {"source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "description": "SNP rs9356744 impacts the function of CDKAL1, influencing metabolic pathways and energy balance.)", "entity_type": "UNKNOWN", "id": "CDK5 REGULATORY SUBUNIT-ASSOCIATED PROTEIN 1-LIKE 1 (CDKAL1)"}, {"source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "description": "The SNP variants affect TFAP2B\'s ability to regulate gene expression related to obesity risk.)", "entity_type": "UNKNOWN", "id": "TRANSCRIPTION FACTOR AP-2 BETA (TFAP2B)"}, {"source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "description": "Plant-based diet intake and PRS have an interaction influencing obesity risk.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 7}]", "id": "PLANT-BASED DIET INTAKE& PRS"}, {"source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "description": "Fried-food intake and PRS show interaction where PRS offset the obesity risk effect of high fried-food intake.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 7}]", "id": "HIGH FRIED-FOOD INTAKE& PRS"}, {"source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "description": "A plant-based diet may mitigate obesity risk for individuals with low PRS.)", "entity_type": "UNKNOWN", "id": "PLANT-BASED DIET INTAKE& LOW PRS"}, {"source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "description": "A plant-based diet may mitigate obesity risk for individuals with low PRS.)", "entity_type": "UNKNOWN", "id": "REDUCTION IN OBESITY RISK"}, {"source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "description": "Despite high PRS, high fried-food intake might not significantly increase obesity risk due to counterbalancing factors.)", "entity_type": "UNKNOWN", "id": "HIGH FRIED-FOOD INTAKE& HIGH PRS"}, {"source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "description": "Despite high PRS, high fried-food intake might not significantly increase obesity risk due to counterbalancing factors.)", "entity_type": "UNKNOWN", "id": "OFFSET OF OBESOGENIC EFFECTS"}, {"entity_type": "GENOMIC VARIANT", "description": "The minor allele frequency was 0.402 in the total population.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "clusters": "[{level: 0, cluster: 0}]", "id": "FTO RS1558902 GENOTYPE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Baseline characteristics had no association with genotype.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "clusters": "[{level: 0, cluster: 0}]", "id": "WEIGHT"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Baseline characteristics had no association with genotype.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "WC"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Baseline characteristics had no association with genotype.", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "BODY COMPOSITION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Changes in fat distribution were observed as a result of different dietary interventions.<SEP>No associations between the FTO genotypes and these measures were observed.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83<SEP>chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "FAT DISTRIBUTION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Changes observed in trunk fat percentage are part of overall body composition dynamics in response to diet.<SEP>No significant genetic effects on changes in fat distribution were found at baseline.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83<SEP>chunk-2926e6216bf945b12ea4daa6345cad1f", "id": "TRUNK FAT"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The AA genotype is a genetic variation that affects fat distribution and response to certain diets.<SEP>The reference genotype and its interaction with diet<SEP>Val allele carriers showed higher response bias compared to iso homozygotes in Study 1.<SEP>The genotype effect of EBNA2 is dependent on specific genotypes at MS risk loci.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83<SEP>chunk-e1b4f4dac52e806f19c580640a50c729<SEP>chunk-4b4f7d54511e2d8bbf5af6073d25b647<SEP>chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "GENOTYPE"}, {"entity_type": "DIETARY PATTERN", "description": "All patients were started on a low-protein diet (1.5 g/kg/day of natural protein intake).<SEP>The low-protein diet led to less loss in TAT and SAT compared with participants having the AA genotype.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83<SEP>chunk-2b81371954b99facae9a0056e70aa7bd", "id": "LOW-PROTEIN DIET"}, {"entity_type": "DIETARY PATTERN", "description": "Participants with the AA genotype had a greater decrease in fat distribution when responding to the high-prorotein diet.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "HIGH-PROTEIN DIET"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The total fat was observed to change in response to the low-fat diet in our study.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "TOTAL FAT"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Changes in fat mass percentage were noted in relation to the low-fat diet, although no genotype-time interactions on changes in fat distribution with the low- or high-fat diets.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "FM% (FAT MASS PERCENTAGE)"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Our study also monitored the changes in percentage of trunk fat in response to dietary interventions.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "PERCENTAGE OF TRUNK FAT"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The changes in TAT and SAT were observed as part of physiological processes in response to different diets.", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "id": "TAT (TOTAL ABDOMINAL TRANSIENT) AND SAT (SUBCUTANEOUS ABDOMINAL TISSUE)"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "The risk allele (A) was significantly associated with a smaller change in waist circumference compared to other groups.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "WAIST"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d<SEP>chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "High-protein intake was associated with greater weight loss when the AA genotype of the FTO rs1558902 variant is present.)<SEP>The quantity and quality of proteins included in the daily diet.", "entity_type": "PROTEIN CONSUMPTION", "clusters": "[{level: 0, cluster: 1}]", "id": "PROTEIN INTAKE"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "High-protein intake was associated with greater weight loss when the AA genotype of the FTO rs1558902 variant is present.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}]", "id": "WEIGHT LOSS"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "The risk allele carriers in the high-protein group had greater fat loss compared to non-carriers of the AA genotype.)", "entity_type": "UNKNOWN", "id": "AA GENOTYPE (FTO RS1558902)"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "The risk allele carriers in the high-protein group had greater fat loss compared to non-carriers of the AA genotype.)", "entity_type": "UNKNOWN", "id": "FAT LOSS IN HIGH-PROTEIN DIET INTAKE"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "Participants with the AA genotype had less fat loss when assigned to the low-protein diet compared to those without this genotype.)<SEP>Participants with the AA genotype had less loss of trunk fat and SAT when assigned to the low-protein diet compared to those without this genotype.)", "entity_type": "UNKNOWN", "id": "FAT LOSS IN LOW-PROTEIN DIET INTAKE"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "The risk allele carriers in the high-protein group had greater fat loss in TAT and VAT compared to non-carriers.)", "entity_type": "UNKNOWN", "id": "FAT DISTRIBUTION IN HIGH-PROTEIN DIET INTAKE"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "Participants with the AA genotype were associated with less fat loss of trunk fat when assigned to the low-protein diet compared to those without this genotype.)<SEP>Participants with the AA genotype were associated with less loss of SAT when assigned to the low-protein diet compared to those without this genotype.)", "entity_type": "UNKNOWN", "id": "FAT DISTRIBUTION IN LOW-PROTEIN DIET INTAKE"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "The risk allele carriers who followed a high-protein diet had greater loss in body composition over time.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "BODY COMPOSITION CHANGE OVER TIME"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "Participants with the AA genotype experienced greater fat loss when consuming a high-protein diet compared to those without this variant.)", "entity_type": "UNKNOWN", "id": "FAT LOSS RESPONSE TO HIGH-PROTEIN DIET"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "Individuals with the AA genotype experienced less fat loss when consuming a low-protein diet compared to those without this variant.)", "entity_type": "UNKNOWN", "id": "FAT LOSS RESPONSE TO LOW-PROTEIN DIET"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "When following a high-protein diet, participants carrying the AA genotype of the FTO rs1558902 showed greater fat loss compared to other genotypes.)", "entity_type": "UNKNOWN", "id": "FAT LOSS"}, {"source_id": "chunk-3135befd0c3140ad097d96ace6de86a1<SEP>chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "An indicator used in assessing obesity risk along with BMI; influenced by genetic factors like the FTO SNP.<SEP>The risk allele (A) was significantly associated with a smaller change in waist circumference compared to other groups.)", "entity_type": "ANTHROPOMETRIC MEASUREMENT", "clusters": "[{level: 0, cluster: 0}]", "id": "WAIST CIRCUMFERENCE"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "High-protein intake was associated with greater weight loss when the AA genotype of the FTO rs1558902 variant is present.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}]", "id": "WEIGHT LOSS RESPONSE"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "The risk allele carriers who followed a high-protein diet showed greater loss in body composition over time.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 0}]", "id": "BODY COMPOSITION CHANGE OVER TIME RESPONSE"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "Participants with the AA genotype experienced greater fat loss when consuming a high-protein diet compared to those without this variant.)", "entity_type": "UNKNOWN", "id": "FAT LOSS IN HIGH-PROTEIN DIET RESPONSE"}, {"source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "description": "Individuals with the AA genotype experienced less fat loss when consuming a low-protein diet compared to those carrying the variant.)", "entity_type": "UNKNOWN", "id": "FAT LOSS IN LOW-PROTEIN DIET RESPONSE"}, {"source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "Among women, the AA genotype group had lower levels of cortisol compared to other genotypes.<SEP>Individuals having the AA genotype showed greater decrease in fat distribution upon consumption of high-protein diets compared to others.)", "entity_type": "GENOTYPE", "id": "AA GENOTYPE"}, {"source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "description": "Genotype did not show significant interactions with changes in fat distribution when following low-fat diet, as opposed to other dietary patterns.)", "entity_type": "UNKNOWN", "id": "LOW-FAT DIET"}, {"source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "description": "No significant genotype-time interactions on changes in fat distribution were found when following low-fat diets, highlighting the role of genetic background in dietary responses.)", "entity_type": "UNKNOWN", "id": "LOW-FAT DIET INTERACTION"}, {"source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83<SEP>chunk-e1b4f4dac52e806f19c580640a50c729<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-cc528af98598e1adb4684c0166430cf4<SEP>chunk-6264e17a8290bc0a9e24c68c21c1f15c<SEP>chunk-af413a33fae8d27bdb15cd18447dfad4<SEP>chunk-2fb95100a1290b27baae869c6560419a<SEP>chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-6a657cd0a000b6df4db8f1305b795021<SEP>chunk-b2730f3087aee6d9cff6e4939b229323", "description": "A diet consisting of olive oil, fish, and nuts that was compared with a Healthy Eating group<SEP>The response to different diets is influenced by trunk fat percentage dynamics, which are a measure of body composition changes.)<SEP>No dietary patterns are discussed in the provided context.<SEP>Host genetics through mbQTLs and mbBTLs interacted with dietary patterns influencing bacterial taxa abundance.<SEP>A set of eating habits or dietary practices associated with specific health outcomes.<SEP>This dietary pattern could influence both B12 and folate levels through food intake and might be related to gestational duration.<SEP>A unique way of eating or consuming food that is characteristic of a particular culture or geographical area.<SEP>No specific dietary patterns mentioned in the provided text.<SEP>Eating fresh poultry other than chicken and turkey was identified as a risk factor through the univariate analysis, suggesting a potential dietary pattern affecting Campylobacter infections.<SEP>Dietary patterns influence the intake of nutrients like folate and can modulate physiological processes related to cardiovascular health.<SEP>Certain dietary patterns can influence the expression of genomic variants associated with disease risk.<SEP>The pattern that influences gene expression and nutrient metabolism.<SEP>Dietary patterns that include certain foods could affect the expression levels of these MS risk genes.", "entity_type": "DIETARY PATTERN", "id": "DIETARY PATTERN"}, {"entity_type": "GENOMIC VARIANT", "description": "The FTO rs9939609 polymorphism is associated with anthropometric variables such as weight, BMI, and waist circumference.", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "clusters": "[{level: 0, cluster: 7}]", "id": "FTO RS9939609 POLYMORPHISM"}, {"entity_type": "DIETARY PATTERN", "description": "General population dietary patterns were not specifically described but are implied to influence obesity risk.", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "id": "GP"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Overweight and obese population, indicating environmental factors impacting obesity such as lifestyle or diet.", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "id": "OHP"}, {"entity_type": "POPULATION GROUP", "description": "Elderly individuals with high BMI; relevant for studying age-related changes in obesity risk or dietary effects.<SEP>HCR subjects might refer to another specific group of individuals mentioned in the text, distinct from university study attendees.", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487<SEP>chunk-3135befd0c3140ad097d96ace6de86a1", "id": "HCR SUBJECTS"}, {"entity_type": "HEALTH INDICATOR", "description": "Percentage of individuals within a population who have obesity, which is influenced by genetic polymorphisms and environmental factors like education level and physical activity (PA).", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "id": "OBESITY PREVALENCE"}, {"entity_type": "GENOMIC VARIANT", "description": "The FTO SNP is a variant found in genetic studies related to obesity risk.", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "clusters": "[{level: 0, cluster: 22}]", "id": "FTO SNP"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Education level influences the relationship between the FTO SNP and obesity risk.", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "clusters": "[{level: 0, cluster: 7}]", "id": "EDUCATION"}, {"entity_type": "EDUCATIONAL LEVEL", "description": "University studies are a measure of educational attainment and influence certain health risk associations.", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "id": "UNIVERSITY STUDIES"}, {"entity_type": "STATISTICAL ANALYSIS", "description": "The statistical analysis found an association between education level, FTO SNP, and obesity risk.", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "id": "ASSOCIATION"}, {"entity_type": "STATISTICAL MEASURE", "description": "The P-value of 0.023 indicates statistical significance in the interaction term.", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "id": "P FOR INTERACTION"}, {"entity_type": "STATISTICAL ADJUSTMENT", "description": "Additional adjustment for physical activity was considered but not shown in the provided text.", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "id": "ADJUSTMENT FOR PHYSICAL ACTIVITY (PA)"}, {"source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "description": "The interaction between the FTO SNP and education significantly impacted BMI in GP subjects.)", "entity_type": "UNKNOWN", "id": "EDUCATION LEVEL AND FTO RS9939609 POLYMORPHISM"}, {"source_id": "chunk-3135befd0c3140ad097d96ace6de86a1<SEP>chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "description": "An interaction effect occurs when the impact of one independent variable on the dependent variable differs at different levels of another independent variable.<SEP>The interaction between the FTO SNP and education significantly impacted BMI in GP subjects.)", "entity_type": "STATISTICAL TERM", "id": "INTERACTION EFFECT"}, {"source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450<SEP>chunk-3135befd0c3140ad097d96ace6de86a1", "description": "Influence of educational attainment on obesity risk, as it interacts with the presence of certain genotypes (such as FTO rs9939609) to impact health outcomes like BMI and obesity prevalence.)<SEP>Participants with a higher level of education had more adherence to Mediterranean diet.", "entity_type": "DEMOGRAPHIC", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 32}]", "id": "EDUCATION LEVEL"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-5bfe051a79e5e215dd3fc65296825591<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-d53a9f0fba772a7b52b906f476f0aef7<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7<SEP>chunk-fa32a208a2f004c10c1b6777c3be13f2<SEP>chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-2fb95100a1290b27baae869c6560419a<SEP>chunk-63d0b5be485a69f44e2799ccc9510487<SEP>chunk-b2730f3087aee6d9cff6e4939b229323", "description": "A condition where the body\'s normal functions are disrupted, leading to symptoms and potential health risks.)<SEP>IL-1R and IL-6R\u03b1 impacts disease severity as mentioned in the text.<SEP>The determination of obesity risk involves multiple factors including genetic variations and education levels.)<SEP>The study does not specify the disease directly but implies the involvement of conditions requiring MTX or related to its side effects<SEP>A particular illness or medical condition characterized by specific symptoms and pathological changes in the body.<SEP>This is a condition characterized by abnormal growth of tissue in the lining of the colon or rectum, which could recur during study follow-up.<SEP>FAAP20 protein accumulation at DNA damage sites depends on a functional UBZ domain and is required for proper recruitment to ICL (interstrand cross-link) sites.<SEP>The presence of charged residues at positions corresponding to FAAP20-UBZ might suggest that such mutations could lead to certain diseases if they disrupt normal function.<SEP>Disease was observed to be associated with LYP variants and their impact on T cell proliferation and cytokine production.<SEP>These genes are associated with or involved in conditions such as multiple sclerosis (MS) and other autoimmune disorders related to immune evasion.", "entity_type": "DISEASE", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 40}]", "id": "DISEASE"}, {"source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "description": "The FTO SNP plays a role in determining obesity risk, considering factors like education and physical activity.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 22}]", "id": "OBESITY RISK DETERMINATION"}, {"source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "description": "Education level is associated with the relationship between genetic variations like the FTO SNP and obesity risk.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 32}]", "id": "ASSOCIATION WITH OBESITY RISK"}, {"source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "description": "The statistical estimations regarding disease association do not change after certain adjustments, indicating robustness of findings.)", "entity_type": "UNKNOWN", "id": "DISEASE ASSOCIATION"}, {"source_id": "chunk-63d0b5be485a69f44e2799ccc9510487<SEP>chunk-0d49552280a277a8a26456d06f8f26a8<SEP>chunk-bdeb2c0c1ba80048874f6dedb2c2b619<SEP>chunk-dcbd547f8218e7097805720fd6ea43ba", "description": "A measure used in research to determine if results are likely due to chance or a genuine effect.<SEP>The qRT-PCR and microarray data were statistically significant, indicating reliable results.<SEP>The statistical estimations regarding disease association do not change after certain adjustments, indicating robustness of findings.)<SEP>The analysis did not find any statistically significant associations between the factors tested and treatment outcomes after adjustments for multiple testing methods.", "entity_type": "OUTCOME/ENDPOINT", "id": "STATISTICAL SIGNIFICANCE"}, {"entity_type": "GENE", "description": "The TCF7L2 gene is associated with type 2 diabetes risk and modifies the effect of incretins on insulin secretion.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "TCF7L2 GENE"}, {"entity_type": "GENOMIC VARIANT", "description": "The rs7903146 genetic variant in the TCF7L2 gene has been widely studied for its potential interaction with dietary factors and type 2 diabetes risk.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "RS7903146 VARIANT"}, {"entity_type": "DISEASE", "description": "T2D is a condition influenced by genetic variants such as those found in the TCF7L2 gene, as well as dietary factors like fiber intake.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "T2D (TYPE 2 DIABETES)"}, {"entity_type": "NUTRIENT/", "description": "Dietary Pattern", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "INCREASED CEREAL FIBER INTAKE"}, {"entity_type": "GENOMIC VARIANT/", "description": "Molecular Interaction/", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "RS10923931 VARIANT"}, {"entity_type": "GENOMIC VARIANT/", "description": "Molecular Interaction/", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "IRS1 RS2943641 VARIANT"}, {"entity_type": "GENOMIC VARIANT/", "description": "Dietary Factor/", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "GIPR RS10423928 VARIANT"}, {"entity_type": "GENOMIC VARIANT/", "description": "Molecular Interaction/", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "PPAR\u0393 VARIANTS RS1801282 AND RS3856806/"}, {"entity_type": "GENOMIC VARIANT/", "description": "Alcohol Consumption/", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "ADH1C *1/*2 VARIANT"}, {"entity_type": "NUTRIENT/", "description": "The association of heme iron with T2D incidence varied depending on genetic factors like the HFE gene variants, showing variable interactions across different populations.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "HEME IRON INTAKE"}, {"entity_type": "GENOMIC VARIANT/", "description": "heme Iron Intake/", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "HFE VARIANTS RS1799945 AND RS1800562/"}, {"entity_type": "GENOMIC VARIANT/", "description": "micronutrient Consumption/", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "HECTD4 VARIANTS/"}, {"entity_type": "GENOMIC VARIANT", "description": "A specific variant in the GLUCOSE TRANSPORTER 5 (GLUT5) gene has been found to be linked to an increased risk for type 2 diabetes.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "GLUT5 VARIANT"}, {"entity_type": "GENE", "description": "Research suggests that alterations in the Fatty Acid Binding Protein 3 (FABP3) gene could impact the development of insulin resistance and, consequently, increase the likelihood of developing type 2 diabetes.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "FABP3 GENE"}, {"entity_type": "GENE", "description": "Studies have linked genetic variations in the CDKAL1 gene to a heightened risk for both type 1 and type 2 diabetes.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "CDKAL1 GENE"}, {"entity_type": "GENOMIC VARIANT/", "description": "Disease/", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "PPARGC1A VARIANT"}, {"entity_type": "GENOMIC VARIANTS", "description": "Variants in the MTHFR gene have shown mixed associations with increased or decreased risks for diabetes depending on environmental factors like diet and lifestyle.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "MTHFR VARIANTS"}, {"entity_type": "PHYSIOLOGICAL PROCESS/", "description": "Disease/", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "GLUT4 ACTIVITY"}, {"entity_type": "BIOLOGICAL MARKER", "description": "The measurement of C-peptide levels can be indicative of pancreatic beta-cell function and is used to assess insulin secretion capacity; it\u2019s important for diagnosing both type 1 and type 2 diabetes.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "C-PEPTIDE"}, {"entity_type": "MOLECULAR ACTIVITY/", "description": "Disease/", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "SGLT1 EXPRESSION"}, {"entity_type": "GENOMIC VARIANT/", "description": "Disease/", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "PPARG VARIANT RS3856806"}, {"entity_type": "GENE", "description": "Mutations or polymorphisms in the KCNJ11 gene encoding the insulin receptor are linked with both type 1 and type 2 diabetes, affecting glucose homeostasis.", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "KCNJ11 GENE"}, {"entity_type": "GENETIC HAPLOTYPE/", "description": "Disease/", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "id": "PPARGC1A HAPLOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "The H63D variant in the HFE gene has been associated with increased risk of type 2 diabetes (T2D) among women.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "HFE RS1799945 VARIANT"}, {"entity_type": "GENOMIC VARIANT", "description": "The C282Y variant in the HFE gene was also observed in association studies, but its interaction with T2D risk in EPIC-InterAct wasn\'t significant.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "HFE RS1800562 VARIANT"}, {"entity_type": "DIETARY STUDY", "description": "Observational studies conducted through EPIC-InterAct have been used to assess the interactions between specific HFE variants and dietary factors on T2D risk.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "EPIC-INTERACT STUDY"}, {"entity_type": "GENE", "description": "The HFE gene is associated with iron metabolism and has variants linked to T2D risk.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "HOMO SAPIENS HFE GENE"}, {"entity_type": "SAMPLE POPULATION", "description": "Male and female populations were studied for genetic interactions with dietary factors affecting T2D risk.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "MEN AND WOMEN POPULATION GROUPS"}, {"entity_type": "BIOLOGICAL MEASURE", "description": "A measure of how quickly carbohydrates in foods raise blood sugar levels.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "GLYCEMIC INDEX"}, {"entity_type": "LABORATORY TEST", "description": "A test that provides a measure of average blood glucose control over the previous 2-3 months.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "GLYCATED HEMOGLOBIN (HBA1C)"}, {"entity_type": "MEDICAL CONDITION", "description": "A metabolic disorder characterized by high levels of blood sugar due to insulin resistance or insufficient production of insulin.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "TYPE 2 DIABETES MELLITUS (T2D)"}, {"entity_type": "STATISTICAL MEASURE", "description": "The proportion of variation in a trait that can be attributed to genetic factors.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "HERITABILITY"}, {"entity_type": "FIELD OF STUDY", "description": "A field which studies the genetic structure and dynamics of populations across large geographic areas.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "POPULATION GENOMICS"}, {"entity_type": "BIOLOGICAL TRAIT", "description": "The presence of certain genes that may increase a person\'s risk for developing specific diseases or conditions.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "GENETIC PREDISPOSITION"}, {"entity_type": "FIELD OF STUDY", "description": "A subfield of biology that studies changes in gene function and expression that do not involve changes to the DNA sequence itself.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "EPIGENETICS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Close to 6 million SNPs, and expression data from 38,839 probes within 23,704 genes were included in the analysis.<SEP>The changes in mRNA levels during high-salt exposure and in CKD models provide insights into how environmental stressors affect gene regulation.<SEP>The process by which information from a gene is translated into a functional product, usually a protein.<SEP>EBNA2 affects the expression level of several genes that are at risk for MS, impacting their physiological processes.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf<SEP>chunk-bec60ce5d500ee78c6792a2d92b048ca<SEP>chunk-f7bf38c7a21b86ad85fa360a21bfb447<SEP>chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "GENE EXPRESSION"}, {"entity_type": "STATISTICAL TOOL", "description": "A tool used in genetics to assess the combined effect of multiple genetic variants on disease risk.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "POLYGENIC RISK SCORES (PRS)"}, {"entity_type": "BIOLOGICAL VARIANT", "description": "Variants in DNA sequences that occur frequently within a population and are not due to errors during replication.", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "id": "GENETIC POLYMORPHISM"}, {"entity_type": "GENOMIC VARIANT", "description": "Persons with this genotype and non-risk haplotype seem to be unresponsive to change in LTPA.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "GG-GENOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "Non-risk haplotype of the four SLC2A2 variants is associated with responsiveness to PA or changes in LTPA.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "NON-RISK HAPLOTYPE"}, {"entity_type": "PHYSICAL ACTIVITY", "description": "Change in LTPA affects persons with GG-genotype and non-risk haplotype.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "LTPA"}, {"entity_type": "GENOMIC VARIANT", "description": "Persons with this allele seem to be unresponsive to change in LTPA, in contrast to those with GG-genotype.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "RISK ALLELE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Weight loss interventions can reduce T2D incidence among persons with specific genetic variants.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "WEIGHT LOSS INTERVENTIONS"}, {"entity_type": "DIETARY PATTERN", "description": "DPP study indicates that weight loss intervention may modify diabetes risk development.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "DIABETES PREVENTION PROGRAM (DPP)"}, {"entity_type": "DIETARY PATTERN", "description": "DPS study suggests similar findings for genetic variants and reduced T2D incidence post-intervention.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "DOUBLE PODOCYTE STUDY (DPS)"}, {"entity_type": "GENOMIC VARIANT", "description": "Participants with this genotype showed lower T2D incidence in intervention group compared to control, albeit interaction test not significant.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "TCF7L2 RS12255372 RISK TT-GENOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "Similar findings were reported for persons with this allele by DPP and Asti study but the corresponding interaction tests weren\'t significant.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "RS7903146 RISK T-ALLELE"}, {"entity_type": "GENOMIC VARIANT", "description": "Weight loss intervention was associated with reduced T2D incidence among persons with this variant, though a trend for interaction was found.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "ENPP1 RS1044498 DIABETOGENIC VARIANT"}, {"entity_type": "GENOMIC VARIANT", "description": "An interaction between this genetic variant and weight loss interventions resulted in a lower T2D incidence among participants.<SEP>Trend for interaction between weight loss intervention and this variant was observed in certain studies.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "MC4R RS17066829"}, {"entity_type": "GENOMIC VARIANT", "description": "Persons with this genotype showed lower T2D incidence after weight loss intervention compared to reference genotypes.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "PPAR\u0393 RS1152003 CC-GENOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "The weight loss intervention resulted in a lower T2D incidence among persons with this genotype.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "TNF\u0391 RS1800629 GG-GENOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "Trend for interaction between weight loss intervention and this allele was found in DPS study.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "ADRA2B GLU9-ALLELE"}, {"entity_type": "GENOMIC VARIANT", "description": "The weight loss intervention resulted in a lower T2D incidence among persons with this allele compared to reference genotypes.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "LIPC RS2070895 A-ALLELE"}, {"entity_type": "GENOMIC VARIANT", "description": "Persons with this variant showed a reduced T2D incidence due to weight loss intervention, although an interaction trend was detected.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "RS1044498 DIABETOGENIC VARIANT"}, {"entity_type": "GENOMIC VARIANT", "description": "Participants with the CC-genotype showed lower T2D risk after undergoing a weight loss intervention, as indicated by both DPP and DPS studies.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "RS1152003 CC-GENOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "The weight loss intervention resulted in reduced T2D incidence for persons with this variant, as shown in the DPS study.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "RS1800629 GG-VARIANT"}, {"entity_type": "GENOMIC VARIANT", "description": "A trend indicating an interaction between ADRA2B Glu9-allele and weight loss interventions was found in the DPS analysis.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "GLU9-ALLELE"}, {"entity_type": "GENOMIC VARIANT", "description": "Participants with this variant showed a lower T2D incidence after participating in a weight loss intervention, according to both DPP and DPS findings.", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "id": "RS2070895 A-VARIANT"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "19 out of 66 eligible publications reported statistically significant G \u00d7 E interactions for incident T2D risk.)", "entity_type": "UNKNOWN", "id": "DIETARY EXPOSURE"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862<SEP>chunk-1b40d071abac0ba432a9fb0096796f45", "description": "Potential G \u00d7 D interaction was found with genetic variants in the TCF7L2 gene.)<SEP>Variation detected in association studies with T1D susceptibility.", "entity_type": "GENOMIC VARIANT", "id": "GENETIC VARIANT"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Potential G \u00d7 D interaction was found with genetic variants in the TCF7L2 gene.)", "entity_type": "UNKNOWN", "id": "T2D RISK"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Interaction of fiber and whole grain with genetic variants was frequently investigated.)", "entity_type": "UNKNOWN", "id": "G \u00d7 E INTERACTION"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Interaction of fiber and whole grain with genetic variants was frequently investigated.)", "entity_type": "UNKNOWN", "id": "WHOLE GRAIN"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Mainly European descendants were included in NHS and HPFS.)", "entity_type": "UNKNOWN", "id": "EUROPEAN DESCENDANTS"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Mainly European descendants were included in NHS and HPFS.)", "entity_type": "UNKNOWN", "id": "SAMPLE RECRUITMENT"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Most studies had participants aged over 40 years.)", "entity_type": "UNKNOWN", "id": "PARTICIPANTS AGE"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Most studies had participants aged over 40 years.)", "entity_type": "UNKNOWN", "id": "SAMPLE SIZE"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Follow-up time ranged from four to 26 years, with more than eight years in most studies.)", "entity_type": "UNKNOWN", "id": "FOLLOW-UP TIME"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Follow-up time ranged from four to 26 years, with more than eight years in most studies.)", "entity_type": "UNKNOWN", "id": "YEARS RANGE"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f<SEP>chunk-af413a33fae8d27bdb15cd18447dfad4<SEP>chunk-bb507a90a1f5d5ddd01a93200ff2d45e<SEP>chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8<SEP>chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Cultural and genetic background factored into the analysis.<SEP>Studies were conducted on populations of mainly or only European ethnicity.)<SEP>The ethnic distribution significantly affects the prevalence and characteristics of glucose tolerance in different populations.<SEP>Ethnicity also interacts with rs182549 and affects the abundance of Bifidobacterium.<SEP>White (93%), non-Hispanic (97%)", "entity_type": "DEMOGRAPHIC", "id": "ETHNICITY"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "Studies were conducted on populations of mainly or only European ethnicity.)", "entity_type": "UNKNOWN", "id": "POPULATIONS"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "In DPP, participants received individual advice to reduce weight through lower fat and calorie intake along with higher physical activity.)", "entity_type": "UNKNOWN", "id": "INTERVENTION IN DPP"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "In DPP, participants received individual advice to reduce weight through lower fat and calorie intake along with higher physical activity.)", "entity_type": "UNKNOWN", "id": "LIFESTYLE-INTERVENTION"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "In DPS, the intervention guided participants to reduce weight by increasing physical activity and following a recommended diet.)", "entity_type": "UNKNOWN", "id": "INTERVENTION IN DPS"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "In DPP, participants were advised on Mediterranean dietary patterns with extra-virgin olive oil or mixed nuts as alternatives to low-fat diets.)", "entity_type": "UNKNOWN", "id": "DIETARY RECOMMENDATIONS"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "In DPP, participants were advised on Mediterranean dietary patterns with extra-virgin olive oil or mixed nuts as alternatives to low-fat diets.)", "entity_type": "UNKNOWN", "id": "LIFESTYLE FACTORS - DPP"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "PREDIMED participants were randomized into two Mediterranean-diet groups which consumed extra virgin olive oil (1 L/week) or mixed nuts (30 g/day).)", "entity_type": "UNKNOWN", "id": "LIFESTYLE FACTORS - PREDIMED"}, {"source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "description": "PREDIMED participants were randomized into two Mediterranean-diet groups which consumed extra virgin olive oil (1 L/week) or mixed nuts (30 g/day).)", "entity_type": "UNKNOWN", "id": "MEDITERRANEAN-DIET GROUPS"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "Higher intake of cereal fiber is associated with lower T2D incidence among individuals carrying the rs7903146 CC-genotype but not for those with the risk T-allele.)", "entity_type": "UNKNOWN", "id": "FIBER INTAKE/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "Higher intake of cereal fiber is associated with lower T2D incidence among individuals carrying the rs7903146 CC-genotype but not for those with the risk T-allele.)", "entity_type": "UNKNOWN", "id": "RS7903146 VARIANT/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "The variants rs10923931 in NOTCH2 and rs4457053 in ZBED3 have been associated with interactions involving fibre intake but were not consistently replicated across studies.)", "entity_type": "UNKNOWN", "id": "FIBRE INTAKE/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "The variants rs10923931 in NOTCH2 and rs4457053 in ZBED3 have been associated with interactions involving fibre intake but were not consistently replicated across studies.)", "entity_type": "UNKNOWN", "id": "RS10923931 VARIANT/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "The IRS1 rs2943641 genetic variant has been associated with insulin sensitivity and type 2 diabetes, though its findings have not been validated in other studies.)", "entity_type": "UNKNOWN", "id": "INSULIN SENSITIVITY/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "The IRS1 rs2943641 genetic variant has been associated with insulin sensitivity and type 2 diabetes, though its findings have not been validated in other studies.)", "entity_type": "UNKNOWN", "id": "RS2943641 VARIANT/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "The GIPR rs10423928 variant has shown interaction with insulin secretion and type 2 diabetes but could not be replicated in all studies.)", "entity_type": "UNKNOWN", "id": "INSULIN SECRETION/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "The GIPR rs10423928 variant has shown interaction with insulin secretion and type 2 diabetes but could not be replicated in all studies.)", "entity_type": "UNKNOWN", "id": "RS10423928 VARIANT/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "The PPAR\u03b3 variants were linked to a trend for interaction with fat intake but could not be consistently replicated.)", "entity_type": "UNKNOWN", "id": "FAT INTAKE/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "The PPAR\u03b3 variants were linked to a trend for interaction with fat intake but could not be consistently replicated.)", "entity_type": "UNKNOWN", "id": "RS1801282 AND RS3856806 VARIANTS/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "The ADH1C*2-allele was associated with lower diabetes risk among alcohol-consuming US women, contrasting its role in men.)", "entity_type": "UNKNOWN", "id": "ADH1C *1/*2 VARIANT/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "The ADH1C*2-allele was associated with lower diabetes risk among alcohol-consuming US women, contrasting its role in men.)", "entity_type": "UNKNOWN", "id": "ALCOHOL CONSUMPTION/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "Higher heme iron intake was associated with increased T2D risk among women carrying specific HFE gene variants but not for men.)", "entity_type": "UNKNOWN", "id": "HEME IRON INTAKE/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "Higher heme iron intake was associated with increased T2D risk among women carrying specific HFE gene variants but not for men.)", "entity_type": "UNKNOWN", "id": "T2D (TYPE 2 DIABETES)/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "Environmental Factor/", "entity_type": "UNKNOWN", "id": "GENETIC FACTOR INTERACTION/"}, {"source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "description": "Environmental Factor/", "entity_type": "UNKNOWN", "id": "PPARGC1A VARIANT/"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "The H63D variant was found to increase the risk of T2D specifically among women, based on data from EPIC-InterAct and DPP studies.)", "entity_type": "UNKNOWN", "id": "T2D RISK IN WOMEN"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "A similar trend to increased risk of T2D was observed for women with the H63D variant, though not statistically significant.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITH HFE RS1799945 VARIANT IN WOMEN"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "While a similar trend was noted for women and the C282Y variant, the interaction did not reach significance.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITH HFE RS1800562 VARIANT IN WOMEN (TREND BUT NOT SIGNIFICANT)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "In EPIC-InterAct study for men population group, an interaction was found to be significant between the H63D variant and T2D risk.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITH HFE RS1799945 VARIANT IN MEN"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "In EPIC-InterAct study for men population group, an interaction was found to be significant between the H63D variant and T2D risk.)", "entity_type": "UNKNOWN", "id": "MEN POPULATION GROUP (EPIC-INTERACT)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "A similar but not confirmed trend of increased T2D risk was found among men carrying the H63D variant through DPP study.)", "entity_type": "UNKNOWN", "id": "MEN POPULATION GROUP (DPP)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "Heme iron intake showed interactions with the SLC40A1 variant rs744653 for men, and with TMPRSS6 variant rs855791 across both sexes when not reaching a strong significance.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITH SLC40A1 RS744653 VARIANT IN MEN AND TMPRSS6 RS855791 IN TOTAL SAMPLE (BORDERLINE SIGNIFICANCE)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "Dietary patterns that reflect either Western or Mediterranean diet were associated differently with T2D risk, influenced by specific genetic variants.)", "entity_type": "UNKNOWN", "id": "DIETARY PATTERNS: WESTERN & MEDITERRANEAN"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "Dietary patterns that reflect either Western or Mediterranean diet were associated differently with T2D risk, influenced by specific genetic variants.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITH GENETIC VARIANTS IN T2D RISK"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "High intake of red and processed meat increased T2D risk among men with high genetic risk scores.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITH T2D-GRS IN MEN"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "High intake of red and processed meat increased T2D risk among men with high genetic risk scores.)", "entity_type": "UNKNOWN", "id": "RED AND PROCESSED MEAT INTAKE (HPFS)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "A marginally nonsignificant interaction was found between olive oil intake and the GIPR variant rs10423928.)", "entity_type": "UNKNOWN", "id": "MARGINALLY NONSIGNIFICANT INTERACTION WITH GIPR VARIANT RS10423928 IN MEN"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "A marginally nonsignificant interaction was found between olive oil intake and the GIPR variant rs10423928.)", "entity_type": "UNKNOWN", "id": "OLIVE OIL INTAKE (EPIC-INTERACT)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "Carriers of the lactase persistence LCT rs4988235 T-allele had higher T2D risk than lactose nonpersistence CC-genotype carriers, with differences observed based on milk consumption.)", "entity_type": "UNKNOWN", "id": "LACTASE PERSISTENCE LCT GENE VARIANTS (DANISH STUDY)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "Carriers of the lactase persistence LCT rs4988235 T-allele had higher T2D risk than lactose nonpersistence CC-genotype carriers, with differences observed based on milk consumption.)", "entity_type": "UNKNOWN", "id": "MILK CONSUMPTION IN T2D RISK"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "Findings from EPIC-InterAct and KARE suggest that coffee intake interacts differently with certain genetic variants for T2D risk, either reducing or increasing the risk.)", "entity_type": "UNKNOWN", "id": "COFFEE INTAKE (EPIC-INTERACT & KARE)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "Findings from EPIC-InterAct and KARE suggest that coffee intake interacts differently with certain genetic variants for T2D risk, either reducing or increasing the risk.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITH SOME GENETIC VARIANTS IN T2D RISK"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "A stronger T2D risk-lowering effect of coffee intake was observed among individuals carrying the T allele of the TCF7L2 variant.)", "entity_type": "UNKNOWN", "id": "COFFEE INTAKE (EPIC-INTERACT)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "A stronger T2D risk-lowering effect of coffee intake was observed among individuals carrying the T allele of the TCF7L2 variant.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITH TCF7L2 VARIANT RS12255372 IN MEN"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "Coffee consumers with specific genetic variants (rs4402960 T-allele, rs7754840 G-allele, rs5215 CC-genotype) showed a reduced combined prediabetes and T2D risk compared to non-consumers.)", "entity_type": "UNKNOWN", "id": "MILK CONSUMPTION (KARE)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "Coffee consumers with specific genetic variants (rs4402960 T-allele, rs7754840 G-allele, rs5215 CC-genotype) showed a reduced combined prediabetes and T2D risk compared to non-consumers.)", "entity_type": "UNKNOWN", "id": "REDUCTION IN COMBINED PREDIABETES AND T2D RISK FOR COFFEE CONSUMERS"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "A Western dietary pattern was associated with higher T2D risk only among individuals with a high genetic score for T2D, but not among those with a low score.)", "entity_type": "UNKNOWN", "id": "INTERACTION WITH HIGH T2D-GRS IN MEN"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "A Western dietary pattern was associated with higher T2D risk only among individuals with a high genetic score for T2D, but not among those with a low score.)", "entity_type": "UNKNOWN", "id": "WESTERN DIETARY PATTERN (HPFS)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "A Mediterranean diet score showed no interaction with the risk of developing type 2 diabetes according to genetic scores.)", "entity_type": "UNKNOWN", "id": "MEDITERRANEAN DIET (EPIC-INTERACT)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "A Mediterranean diet score showed no interaction with the risk of developing type 2 diabetes according to genetic scores.)", "entity_type": "UNKNOWN", "id": "NO INTERACTION WITH T2D-GRS"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "The protective effect of physical activity against T2D was modified by certain genetic variants, with stronger effects observed in individuals not adhering to a high genetic risk score for T2D.)", "entity_type": "UNKNOWN", "id": "MODIFYING EFFECT ON T2D RISK BY GENETIC VARIANTS"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "The protective effect of physical activity against T2D was modified by certain genetic variants, with stronger effects observed in individuals not adhering to a high genetic risk score for T2D.)", "entity_type": "UNKNOWN", "id": "PA (PHYSICAL ACTIVITY)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "In the Swedish Malm\u00f6 preventive program, lower protective effects of higher physical activity against T2D incidence were observed among individuals with a specific genetic variant for HNF1B.)", "entity_type": "UNKNOWN", "id": "WEAKENED PROTECTIVE EFFECT OF HIGHER PA ON T2D INCIDENCE (SWEDISH MALM\u00d6 PROGRAM)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "An increase in leisure-time physical activity was associated with lower T2D incidence among individuals with certain genetic variants, such as the Glu12 allele of ADRA2B, GG genotype of ABCC8 rs3758947, and non-risk haplotype of four SLC2A2 variants.)", "entity_type": "UNKNOWN", "id": "INCREASED LTPA RESULTING IN LOWER T2D INCIDENCE"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "Decreased leisure-time physical activity was associated with higher T2D incidence among individuals carrying the PPAR\u03b3 rs1801282 Ala- and rs17036314 C-alleles, GG genotype of PPAR\u03b3, or non-risk haplotype of four SLC2A2 variants.)", "entity_type": "UNKNOWN", "id": "REDUCED LEISURE-TIME PHYSICAL ACTIVITY ASSOCIATED WITH HIGHER T2D INCIDENCE"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "In NHS, women carrying the IRS1 rs1522813 A-allele who were physically inactive had a higher risk of developing T2D compared to those with GG-genotype but active.)", "entity_type": "UNKNOWN", "id": "GLYCOGEN STORAGE DISEASE PROTEIN 1 (IRS1)"}, {"source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "description": "In NHS, women carrying the IRS1 rs1522813 A-allele who were physically inactive had a higher risk of developing T2D compared to those with GG-genotype but active.)", "entity_type": "UNKNOWN", "id": "INCREASED RISK WITH LOW PA AND T2D RISK A1 ALLELE IN WOMEN"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e<SEP>chunk-180ca3d6d27d84b30ebf5d20f135de18<SEP>chunk-159d4c797dae6eb825c74067af703228", "description": "Interaction between specific genetic variants and weight loss interventions can influence diabetes risk.)<SEP>The genetic variants played a role in influencing microbiome composition, affecting causality estimation during Mendelian Randomization.<SEP>Several genetic variants within the CNR1 gene were found to influence CB1 receptor levels and, subsequently, affect alcohol seeking behavior.", "entity_type": "BIOCHEMICAL COMPONENT", "id": "GENETIC VARIANTS"}, {"source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "description": "Participants with certain risk genotypes showed reduced T2D incidence after weight loss intervention.)", "entity_type": "UNKNOWN", "id": "DIABETES INCIDENCE"}, {"source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "description": "Participants with certain risk genotypes showed reduced T2D incidence after weight loss intervention.)", "entity_type": "UNKNOWN", "id": "RISK GENOTYPES"}, {"source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "description": "The DPP study indicates that certain genetic variants influence the effectiveness of weight loss interventions in diabetes risk reduction.)", "entity_type": "UNKNOWN", "id": "GENETIC VARIANTS & WEIGHT LOSS INTERVENTIONS"}, {"entity_type": "GENE", "description": "SNP rs7903146 discovered within the TCF7L2 gene has a consistent direction of effect with published GWAS data on BMI at baseline.<SEP>The TCF7L2 gene is associated with the count of risk alleles.<SEP>The study reports an association with the rs7903146 variant in TCF7L2 gene, having a high odds ratio (OR 1.38 [95% CI 1.23\u20131.54], P = 1.25 \u00d7 10\u22128).", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092<SEP>chunk-f9a60d3cf951d77afa115859755dac43<SEP>chunk-6a7c69534e11aa2fc4bd948484aa81c4", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 50}]", "id": "TCF7L2"}, {"entity_type": "SNP", "description": "SNP rs7903146 discovered within the TCF7L2 gene has a consistent direction of effect with published GWAS data on BMI at baseline.<SEP>The genomic variant rs7903146 in the TCF7L2 gene is analyzed in this study.<SEP>This variant in TCF7L2 gene shows a significant association with type 2 diabetes (T2D) as per the analysis.", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092<SEP>chunk-f9a60d3cf951d77afa115859755dac43<SEP>chunk-6a7c69534e11aa2fc4bd948484aa81c4", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 50}]", "id": "RS7903146"}, {"entity_type": "GENE", "description": "The rs10010131 variant in WFS1 gene was found to be associated, although not consistently as reported in previous European studies.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "id": "WFS1"}, {"entity_type": "GENOMIC VARIANT", "description": "A single nucleotide polymorphism (SNP) in the WFS1 gene showed an association with T2D.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "id": "RS10010131"}, {"entity_type": "GENE", "description": "The rs10946398 variant in CDKAL1 gene was associated with T2D, consistent with previous findings.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 52}]", "id": "CDKAL1"}, {"entity_type": "GENOMIC VARIANT", "description": "This SNP of the CDKAL1 gene was found to associate with risk for T2D in this study.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 52}]", "id": "RS10946398"}, {"entity_type": "GENE", "description": "The rs864745 variant in JAZF1 gene showed association, consistent with earlier reports.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 52}]", "id": "JAZF1"}, {"entity_type": "GENOMIC VARIANT", "description": "This particular genomic variant of the JAZF1 gene was found associated with risk for T2D based on this study\'s findings.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 52}]", "id": "RS864745"}, {"entity_type": "GENE", "description": "SNP rs4607103 in ADAMTS9 was associated but had an opposite direction of effect compared to European studies, pointing towards a unique genetic interaction.<SEP>The rs4607103 variant in ADAMTS9 gene exhibited association; however, its direction of effect was opposite to that reported in European samples.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 49}]", "id": "ADAMTS9"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant found in the ADAMTS9 gene had an opposing direction of association compared to earlier European findings.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 49}]", "id": "RS4607103"}, {"entity_type": "DISEASE", "description": "Set 1 comprised a higher percentage of women compared to the control group, with T2D-ESRD (type 2 diabetes and end-stage renal disease) case subjects having an older mean age at enrollment than controls.", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "clusters": "[{level: 0, cluster: 2}]", "id": "T2D-ESRD"}, {"entity_type": "DISEASE", "description": "T2D patients were identified in the cohort, and certain genetic variants were tested against this disease.<SEP>The meta-analysis showed consistent results across sets 1 and 2 in relation to rs7903146 variant of TCF7L2 gene, indicating association with risk for type 2 diabetes.<SEP>Type 2 diabetes is a metabolic disorder characterized by high blood sugar due to insulin resistance or insufficient production of insulin.<SEP>Type 2 diabetes associated with the G-allele of rs1183910 in HNF1A and increased CRP levels.<SEP>Type 2 diabetes has been linked to certain lead SNPs through analysis of individuals contributing to trans-ethnic meta-analysis.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb<SEP>chunk-f9a60d3cf951d77afa115859755dac43<SEP>chunk-372184452f1f9156514992455057f843<SEP>chunk-bb507a90a1f5d5ddd01a93200ff2d45e<SEP>chunk-3edc72730d68b4c7fd6ba397aa087391", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 51}]", "id": "T2D"}, {"entity_type": "COUNT OF RISK ALLELES", "description": "The count of risk alleles associated with the TCF7L2 gene\'s genetic variants was considered for analysis.", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 50}]", "id": "NUMBER OF TCF7L2 RISK VARIANTS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Age (years) is a variable that characterizes individuals enrolled in the study.<SEP>Age had a significant association with snoring, with each year increasing odds of snoring by 1.011 (95% CI: 1.009-1.012).<SEP>Demographic factor adjusted when analyzing results.<SEP>Evidence for interaction with age in relation to the association between TCF7L2 gene and the number of risk alleles is not found.<SEP>There is a positive association between age and the prevalence of IFG/IGT and T2D, with older individuals being more prone to these conditions.<SEP>Younger individuals showed a tendency towards adhering more to the Mediterranean diet compared to older individuals.<SEP>The age of women was considered an environmental factor when analyzing the data, which could influence the results due to its impact on various biological processes related to cancer risk.<SEP>Measure of age in years.<SEP>Age shows an interaction with rs182549, affecting Bifidobacterium abundance differently in different age groups.<SEP>No statistically significant interaction was found between age and selenium\'s impact on colorectal cancer risk, emphasizing the importance of other factors.<SEP>No statistically significant interaction was found between age and selenium\'s impact on colorectal cancer risk.<SEP>Participants who consumed black coffee or sugared coffee were more likely to be younger.<SEP>There was a significant difference in age distribution between different genotypes.<SEP>Patients were a mean age of 60 years (SD=10)<SEP>Symptoms and vital signs were measured in the participants during the study.<SEP>Average age was slightly higher for asthmatic patients compared to healthy controls in this study.<SEP>The age variable was considered as a covariate during pooled association analysis.<SEP>The entity \'age\' has a relationship with other entities indicating its influence on HIV disease progression.<SEP>The entity \'age\' has a relationship with other entities indicating the influence of age in HIV disease progression.", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4<SEP>chunk-8f23bd741f82dc8ca1e2600d790b690f<SEP>chunk-1f20dbdcc9de443dada1a2045dc7832b<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d<SEP>chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8<SEP>chunk-b6ea16f7f7eb8bc6f36f2be03fe62403<SEP>chunk-2c91cdebb36ef064eb01a7021a961302<SEP>chunk-d311ac0182b37003c75c0005b4337fa6<SEP>chunk-9977706363cc368b3be503d07ed3ef62<SEP>chunk-0d49552280a277a8a26456d06f8f26a8<SEP>chunk-af413a33fae8d27bdb15cd18447dfad4<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a<SEP>chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6<SEP>chunk-bb507a90a1f5d5ddd01a93200ff2d45e<SEP>chunk-c6fdc5ed25c4e17b8c421a952bb05450<SEP>chunk-05d583f893b703816ad8de9561aa5292<SEP>chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 30}]", "id": "AGE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Gender influences glycemic status; females tend to have a higher frequency of NGT compared to males.<SEP>Interaction with sex regarding the relationship between the TCF7L2 gene\'s genetic variants and count of risk alleles was also explored but found insignificant.<SEP>Gender was considered a variable in adjustment factors.<SEP>Women tended to adhere more to the Mediterranean diet than men.<SEP>Sex showed an even stronger association with snoring, with males having a relative risk ratio of 2.264 compared to females.<SEP>Sex was considered as a covariate in the adjusted analyses for treatment response.<SEP>Sex is another demographic characteristic of the study participants.<SEP>The entity \'sex\' has a relation to HIV disease progression, suggesting its role is considered in disease risk assessment.", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4<SEP>chunk-8f23bd741f82dc8ca1e2600d790b690f<SEP>chunk-9977706363cc368b3be503d07ed3ef62<SEP>chunk-2c91cdebb36ef064eb01a7021a961302<SEP>chunk-0d49552280a277a8a26456d06f8f26a8<SEP>chunk-bb507a90a1f5d5ddd01a93200ff2d45e<SEP>chunk-c6fdc5ed25c4e17b8c421a952bb05450<SEP>chunk-05d583f893b703816ad8de9561aa5292", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 31}]", "id": "SEX"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The degree of African ancestry was considered but found not to have an effect on the relationship between the TCF7L2 gene\'s genetic variants and count of risk alleles.", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "clusters": "[{level: 0, cluster: 3}]", "id": "DEGREE OF AFRICAN ANCESTRY"}, {"source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "description": "Set 1 had a higher percentage of women compared to controls, indicating potential sex-based disparities in T2D and ESRD associations.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}]", "id": "SEX RATIO"}, {"source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "description": "The enrollment age difference between T2D-ESRD case subjects and controls was observed, potentially influencing study outcomes or the progression of conditions.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}]", "id": "AGE AT ENROLLMENT"}, {"source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "description": "Diagnosis age disparity between T2D and ESRD subjects was observed, possibly influencing outcomes or disease management.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}]", "id": "AGE AT DIAGNOSIS"}, {"source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "description": "The combined variants rs7903146 in TCF7L2, ADAMTS9, CDKAL1 and JAZF1 genes showed association with type 2 diabetes (T2D), highlighting genetic contributions to multiple risk factors.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 51}]", "id": "RS7903146+ADAMTS9+CDKAL1+JAZF1"}, {"source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "description": "The combined variants rs864745 in JAZF1 gene showed association with T2D through this study, emphasizing genetic factors in diabetes risk assessment.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 51}]", "id": "RS864745+JAZF1"}, {"source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "description": "This study\'s results highlighted contributions of various risk factors to T2D and ESRD, underscoring interplay between conditions.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}]", "id": "RISK FACTOR CONTRIBUTION"}, {"source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "description": "The combined genetic variants rs7903146 in TCF7L2, ADAMTS9, CDKAL1, JAZF1 and rs864745 and rs10946398 in other genes were associated with type 2 diabetes (T2D), emphasizing the role of genetic polymorphisms.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 51}]", "id": "RS7903146+ADAMTS9+CDKAL1+JAZF1+RS864745+RS10946398"}, {"source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "description": "SNPs rs4607103 and rs10946398 were found associated with T2D through this study, indicating genetic variability in diabetes risk.)", "entity_type": "UNKNOWN", "id": "ADAMTS9+CDKAL1+JAZF1"}, {"source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "description": "SNPs rs4607103 and rs10946398 were found associated with T2D through this study, indicating genetic variability in diabetes risk.)", "entity_type": "UNKNOWN", "id": "RS4607103+RS10946398"}, {"source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "description": "This study\'s findings underscored the importance of clinical parameters in assessing T2D and ESRD, emphasizing their interrelation.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}]", "id": "CLINICAL PARAMETERS"}, {"source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "description": "SNPs rs864745, JAZF1, rs4607103 and rs10946398 were found associated with T2D and ESRD through this study, highlighting the importance of genetic factors in disease assessments.)", "entity_type": "UNKNOWN", "id": "ADAMTS9+CDKAL1+T2D-ESRD CLINICAL PARAMETERS"}, {"source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "description": "SNPs rs864745, JAZF1, rs4607103 and rs10946398 were found associated with T2D and ESRD through this study, highlighting the importance of genetic factors in disease assessments.)", "entity_type": "UNKNOWN", "id": "RS864745+JAZF1+RS4607103+RS10946398"}, {"source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "description": "The combined genetic variants rs7903146 in TCF7L2, ADAMTS9, CDKAL1, JAZF1, rs864745 and rs10946398 showed association with type 2 diabetes (T2D), indicating complex genetic interplay.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 2}]", "id": "RS7903146+ADAMTS9+CDKAL1+JAZF1+RS864745+RS10946398+RS4607103+RS10946398+RS864745"}, {"source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "description": "No evidence for interaction between age and the association between TCF7L2 gene and the number of risk alleles was found.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "INTERACTION"}, {"entity_type": "DISEASE", "description": "Normal glucose tolerance is a condition where individuals have normal fasting plasma glucose levels.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "NGT"}, {"entity_type": "DISEASE", "description": "Impaired fasting glucose/impaired glucose tolerance indicates abnormal glucose metabolism.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "IFG/IGT"}, {"entity_type": "NUTRIENT", "description": "FPG levels are directly related to glucose metabolism and can indicate metabolic health.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "FASTING PLASMA GLUCOSE (FPG)"}, {"entity_type": "METABOLIC PATHWAY", "description": "Increased 2hPG levels suggest abnormalities in post-meal glycemic control, indicative of insulin sensitivity issues.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "TWO-HOUR PLASMA GLUCOSE (2HPG)"}, {"entity_type": "DISEASE", "description": "Transitioning from NGT to IFG/IGT or T2D is associated with changes in BMI and metabolic health parameters.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "GLYCEMIC STATUS TRANSITION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Changes in FPG levels are linked to BMI change, possibly through effects on insulin sensitivity and glucose metabolism processes.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "CHANGE IN FPG LEVELS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Alterations in post-meal glucose levels can affect BMI and metabolic health outcomes, reflecting dietary or lifestyle impacts.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 34}]", "id": "CHANGE IN 2HPG LEVELS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Dietary habits, physical activity levels, sleep patterns, and stress management affect the development of glucose tolerance disorders.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "LIFESTYLE FACTORS"}, {"entity_type": "TEMPORAL PROCESS", "description": "The age at which individuals develop NGT, IFG/IGT, or T2D can influence the progression of metabolic health issues.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "id": "AGE OF ONSET"}, {"entity_type": "MOLECULAR INTERACTION", "description": "IRF1\'s involvement suggests a potential molecular interaction between the SNP block and insulin resistance mechanisms.<SEP>Impaired insulin sensitivity contributes to the development of IFG/IGT and progression towards T2D.<SEP>Insulin resistance was indicated by higher fasting insulin levels and HOMA-IR scores among women with PCOS.<SEP>The proximity of rs7574352 to the IRF1 gene might suggest a link between this variant and insulin resistance or metabolic disorders.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5<SEP>chunk-bb507a90a1f5d5ddd01a93200ff2d45e<SEP>chunk-938d8bf2e56584de9dd556be5eb70964", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 45}]", "id": "INSULIN RESISTANCE"}, {"entity_type": "INDICATORS OF METABOLIC SYNDROME", "description": "A cluster of conditions\u2014high blood pressure, high blood sugar, excess body fat around the waist, and abnormal cholesterol levels\u2014that increase a person\'s risk for heart disease, stroke, and diabetes.<SEP>The presence of metabolic syndrome increases the likelihood of developing IFG/IGT and T2D due to shared pathophysiological mechanisms.", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "METABOLIC SYNDROME"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Alcohol consumption was very heavy for both groups at time of conception and continued to be so across pregnancy.<SEP>Excessive alcohol intake is linked to impaired glucose tolerance and can influence the progression towards diabetes.<SEP>The quantity and frequency of alcohol consumption in the diet.<SEP>There was no significant difference in alcohol consumption between the case and control groups.", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd<SEP>chunk-bb507a90a1f5d5ddd01a93200ff2d45e<SEP>chunk-a9cd8765fac3ebb933c3fade561bc42e<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "ALCOHOL CONSUMPTION"}, {"entity_type": "DISEASE", "description": "A higher rate of hypertension was noted among HUA cases compared to controls.<SEP>A state of high blood pressure that was considered as one of the covariates.<SEP>Coexisting hypertension can exacerbate glucose intolerance and increase the risk of developing diabetes.<SEP>Hypertension contributes to sUA level elevation and increases the risk of hyperuricemia (HUA).", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd<SEP>chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00<SEP>chunk-bb507a90a1f5d5ddd01a93200ff2d45e<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "HYPERTENSION"}, {"entity_type": "GENE", "description": "Genes among the 12 identified pleiotropic loci associated with CRP.<SEP>SNP rs9939609 located in the FTO gene is associated with BMI at baseline.<SEP>Linked to alcohol consumption, which can affect sleep patterns including snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64<SEP>chunk-1fde08e52a188ac6a549e7c11d59a092<SEP>chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "FTO"}, {"entity_type": "GENE", "description": "SNP rs2984618 found in the TAL1 gene shows a significant association with BMI at baseline.", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "TAL1"}, {"entity_type": "GENETIC RISK SCORE", "description": "A Genetic Risk Score (GRS) that ranged from 0 to 9 was calculated based on multiple polymorphisms<SEP>The Genotype Risk Score (GRS) is significantly associated with increased BMI at baseline.", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "GRS"}, {"entity_type": "SNP", "description": "The SNP rs9939609 located in the FTO gene is associated with BMI at baseline.", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "RS9939609"}, {"entity_type": "SNP", "description": "SNP rs2984618 found in the TAL1 gene shows a significant association with BMI at baseline.", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "id": "RS2984618"}, {"source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "description": "An increase in fasting plasma glucose levels shows a positive association with BMI changes, possibly through metabolic pathways affecting fat accumulation.)", "entity_type": "UNKNOWN", "id": "BMIC"}, {"source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "description": "Transitioning from NGT to IFG/IGT or T2D is associated with disease progression and increased health risks.)", "entity_type": "UNKNOWN", "id": "DISEASE PROGRESSION"}, {"source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "description": "Fasting plasma glucose and two-hour post-meal glucose levels are negatively correlated with BMI at baseline but positively related to BMI change, indicating complex metabolic interactions.)", "entity_type": "UNKNOWN", "id": "BMI CHANGE"}, {"source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "description": "Fasting plasma glucose and two-hour post-meal glucose levels are negatively correlated with BMI at baseline but positively related to BMI change, indicating complex metabolic interactions.)", "entity_type": "UNKNOWN", "id": "FPG AND 2HPG LEVELS"}, {"source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "description": "Participants with NGT display lower changes in BMI compared to IFG/IGT or T2D subjects, highlighting protective effects of normal glucose tolerance on body weight maintenance.)", "entity_type": "UNKNOWN", "id": "BASELINE GLYCEMIC STATUS"}, {"source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "description": "Participants with NGT display lower changes in BMI compared to IFG/IGT or T2D subjects, highlighting protective effects of normal glucose tolerance on body weight maintenance.)", "entity_type": "UNKNOWN", "id": "CHANGE IN BMI"}, {"source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "description": "Transitioning from NGT to IFG/IGT increases the risk of developing obesity and metabolic disorders due to higher baseline BMI and BMI change.)", "entity_type": "UNKNOWN", "id": "IFG/IGT TRANSITION"}, {"source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "description": "Transitioning from NGT to IFG/IGT increases the risk of developing obesity and metabolic disorders due to higher baseline BMI and BMI change.)", "entity_type": "UNKNOWN", "id": "INCREASED RISK"}, {"source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "description": "Maintaining normal glucose tolerance over time has a protective effect on body weight management compared to transitioning stages of impaired glycemic status.)", "entity_type": "UNKNOWN", "id": "PREVENTIVE EFFECT"}, {"source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "description": "Maintaining normal glucose tolerance over time has a protective effect on body weight management compared to transitioning stages of impaired glycemic status.)", "entity_type": "UNKNOWN", "id": "STABLE NGT"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "The rs1011527 SNP in the LEPR gene interacts significantly with FPG on baseline BMI.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 36}]", "id": "FPG (FASTING PLASMA GLUCOSE)"}, {"source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "description": "The rs1011527 SNP in the LEPR gene interacts significantly with FPG on baseline BMI.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 36}]", "id": "RS1011527 SNP IN LEPR"}, {"entity_type": "GENE", "description": "The MTHFR gene carries a polymorphism at position 1298 where the nucleotide A is replaced by C.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "MTHFR 1298A>C"}, {"entity_type": "GENE", "description": "The MS (Methylenetetrahydrofolate dehydrogenase, NADH-dependent) gene carries a polymorphism at position 2756 where the nucleotide A is replaced by G.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "MS 2756A>G"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "MTX (Methotrexate) related neurotoxicity is an adverse effect observed in patients undergoing treatment with Methotrexate, which occurred in 7 out of the study participants.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "MTX-RELATED NEUROTOXICITY"}, {"entity_type": "TEST BATTERY", "description": "DIVERGT battery is a comprehensive neuropsychological test battery used for assessing cognitive functions such as attention and processing speed.<SEP>The DIVERGT battery might have been considered as part of assessing cognitive function but was not associated with GRS", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "DIVERGT BATTERY"}, {"entity_type": "PROTOCOL", "description": "POG protocols refer to the clinical trial protocols conducted by the Pediatric Oncology Group, which were primarily utilized in the study\'s treatment of patients.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "PEDIATRIC ONCOLOGY GROUP (POG) PROTOCOLS"}, {"entity_type": "TEST MEASURE", "description": "PEGDH activity was not related to the GRS suggesting it may not be involved in the pathway related to MTX-related neurotoxicity or disease impact<SEP>PEGDH is a component part of PEGNDH test that assesses working memory and attention span.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "PEGDH"}, {"entity_type": "TEST MEASURE", "description": "This measures an individual\'s cognitive ability in the domain of Working Memory, particularly focusing on encoding and retrieval processes.<SEP>9", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "DIG"}, {"entity_type": "TEST MEASURE", "description": "CFL (Cognitive Functioning Level) test evaluates executive functions which include cognitive flexibility, inhibition control, planning, working memory, attention, and cognitive processing speed.<SEP>10", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "CFL"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Impaired TMTA scores were observed in patients who had MTX-related neurotoxicity<SEP>TMTA (Trail Making Test A) assesses focused attention and processing speed.<SEP>TMTA performance may be negatively influenced by MTX-related neurotoxic events<SEP>TMTA seems to have been negatively impacted by MTX-related neurotoxicity, as indicated in the study", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "TMTA"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Low TMTB performance was associated with GRS \u2265 6, indicating potential disease impact<SEP>TMTB (Trail Making Test B) evaluates shifting attention and processing speed.<SEP>TMTB scores were significantly different based on GRS, suggesting potential involvement in neurodegenerative processes related to MTX use<SEP>TMTB was significantly lower in patients with a GRS \u2265 6 compared to those with a GRS < 6", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29<SEP>chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "TMTB"}, {"entity_type": "POLYMORPHISM", "description": "The SHMT 1420C>T is a genetic variant found within the Serine Hydroxymethyltransferase (SHMT) gene, influencing its activity and potentially impacting folate metabolism.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "SHMT 1420C>T POLYMORPHISM"}, {"entity_type": "GENE", "description": "The MTHFR Trapping Reaction Reductase (MTRR) gene is essential for methylcobalamin-dependent reactions in the body\'s folate cycle.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "MTRR GENE"}, {"entity_type": "TEST", "description": "Trail Making Test A assesses attention and cognitive processing speed.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "TMTA TEST"}, {"entity_type": "TEST", "description": "Trail Making Test B evaluates executive functions, particularly working memory and cognitive flexibility.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "TMTB TEST"}, {"entity_type": "TEST", "description": "The Cognitive Functioning Level (CFL) test is designed to measure various components of cognitive function including attention span, memory retention, and processing speed.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "COGNITIVE FUNCTIONING LEVEL (CFL) TEST"}, {"entity_type": "TEST", "description": "This test evaluates the performance in certain areas related to working memory and attention processes, particularly sensitive to the impact of MTHFR polymorphisms.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "PEGNDH TEST"}, {"entity_type": "SKILL/FUNCTION", "description": "Cognitive flexibility involves the ability to adapt thinking styles or responses according to different situations or tasks.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "COGNITIVE FLEXIBILITY"}, {"entity_type": "SKILL/FUNCTION", "description": "This skill involves suppressing impulses and reactions, particularly in response to distractions or when switching between tasks.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "INHIBITION CONTROL"}, {"entity_type": "SKILL/FUNCTION", "description": "Planning is the ability to think ahead, set objectives, and organize actions towards achieving these goals.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "PLANNING SKILLS"}, {"entity_type": "MEMORY TYPE", "description": "The capacity to temporarily hold and manipulate information in mind while performing tasks.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "WORKING MEMORY"}, {"entity_type": "COGNITIVE PROCESS", "description": "The ability to focus on specific stimuli or tasks while filtering out distractions.", "source_id": "chunk-4b6e8f78911cafc70959cfdfed786c29", "id": "ATTENTION"}, {"entity_type": "DRUG", "description": "MTX is a drug that may be associated with early neurotoxicity<SEP>MTX is associated with neurotoxicity and has been used to calculate genetic risk score", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "MTX"}, {"entity_type": "METABOLIC PATHWAY<|PEGNDH ACTIVITY DID NOT SHOW A SIGNIFICANT ASSOCIATION WITH TMT PERFORMANCE, IMPLYING IT MIGHT BE SEPARATE FROM MTX\'S IMPACT ON COGNITION", "description": "8", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "id": "PEGNDH"}, {"entity_type": "GENOMIC VARIANT", "description": "The C risk allele at SLC30A8 rs13266634 was significantly associated with higher fasting proinsulin levels.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "SLC30A8 RS13266634"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The association between the SLC30A8 risk allele and fasting proinsulin levels was observed.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "FASTING PROINSULIN"}, {"entity_type": "DIETARY PATTERN", "description": "The study population included individuals from DPP (Diabetes Prevention Program) cohort.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "DPP COHORT"}, {"entity_type": "METABOLIC PATHWAY", "description": "An alternative term for fasting insulin metabolism that was part of the analysis context.<SEP>Fasting insulin is a metabolic pathway involved in glucose regulation.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "SLIMFAST INSULIN"}, {"entity_type": "NUTRIENT", "description": "A baseline measurement used for adjusting variables related to insulin levels.<SEP>Baseline fasting glucose levels were considered as one of the covariates for adjusting variables.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "FASTING GLUCOSE"}, {"entity_type": "GENE", "description": "The study focused on the SLC30A8 gene and its risk allele at rs13266634.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "SLC30A8"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "One of the active intervention groups was a lifestyle intervention group.<SEP>Type of treatment group that involved changes in lifestyle habits.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "LIFESTYLE INTERVENTION"}, {"entity_type": "DRUG", "description": "One of the treatment modalities tested for lowering proinsulin levels.<SEP>The metformin intervention group had similar proinsulin levels compared to the troglitazone group at year 1.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "METFORMIN"}, {"entity_type": "DRUG", "description": "Another active intervention group was the troglitazone group, which also showed a decrease in proinsulin levels.<SEP>Another medication used as a control or alternative in the study.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "TROGLITAZONE"}, {"entity_type": "THERAPY", "description": "After 1 year of randomised DPP intervention, fasting proinsulin levels were significantly decreased.<SEP>The primary intervention conducted over one year.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "DPP INTERVENTIONS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Exploration of whether genetic predispositions modify treatment response.<SEP>There was no significant interaction between genotype and the effectiveness of any treatment.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "GENOTYPE \u00d7 TREATMENT INTERACTION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Result or change observed post-intervention application.<SEP>The effectiveness of different treatments on reducing proinsulin levels after DPP interventions.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "TREATMENT EFFECT"}, {"entity_type": "GENOMIC VARIANT", "description": "The locus was considered when analyzing the association between SLC30A8 risk alleles and fasting proinsulin levels after adjusting for other variables.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "SLC30A8 LOCUS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Fasting proinsulin levels were studied for associations with genetic factors.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "PROINSULIN"}, {"entity_type": "DIETARY PATTERN", "description": "A major study cohort involving dietary intervention in diabetes prevention.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "DPP (DIABETES PREVENTION PROGRAM)"}, {"entity_type": "THERAPY", "description": "Groups that underwent different treatments as part of the DPP program.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "INTERVENTION GROUPS"}, {"entity_type": "CONTROL TREATMENT", "description": "The group receiving no active treatment, serving as a baseline for comparison.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "PLACEBO GROUP"}, {"entity_type": "TIME PERIOD", "description": "A significant time duration considered in assessing changes post-intervention.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "1 YEAR"}, {"entity_type": "NUTRIENT", "description": "Fasting insulin levels were notably higher in women with PCOS compared to control subjects, indicating insulin resistance.<SEP>Part of the body\'s response to maintain glucose levels, used for adjusting analysis.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5<SEP>chunk-8f23bd741f82dc8ca1e2600d790b690f", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 45}]", "id": "FASTING INSULIN"}, {"entity_type": "CONTROL EFFECT", "description": "Reference point to measure against treatment effects in intervention groups.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "PLACEBO GROUP EFFECT"}, {"entity_type": "STATISTICAL ADJUSTMENT FACTORS", "description": "Common demographic variables considered for statistical analyses.", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "id": "AGE, SEX AND ETHNICITY ADJUSTMENT"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "The DPP cohort was exposed to certain dietary patterns.)", "entity_type": "UNKNOWN", "id": "DIETARY PATTERN EXPOSURE"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "The DPP cohort was exposed to certain dietary patterns.)", "entity_type": "UNKNOWN", "id": "DPP COHORT INDIVIDUALS"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "Baseline fasting insulin was used as an adjustment factor for statistical analysis.)", "entity_type": "UNKNOWN", "id": "ADJUSTMENT FACTOR"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "Baseline fasting insulin was used as an adjustment factor for statistical analysis.)", "entity_type": "UNKNOWN", "id": "BASELINE FASTING INSULIN"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "The active interventions had a significant impact on lowering fasting proinsulin levels.)", "entity_type": "UNKNOWN", "id": "ACTIVE INTERVENTION GROUPS (LIFESTYLE, METFORMIN, TROGLITAZONE)"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "The metformin and troglitazone intervention groups showed similar but distinct drug responses in terms of decreasing proinsulin levels.)", "entity_type": "UNKNOWN", "id": "DRUG RESPONSE COMPARISON"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "The metformin and troglitazone intervention groups showed similar but distinct drug responses in terms of decreasing proinsulin levels.)", "entity_type": "UNKNOWN", "id": "METFORMIN AND TROGLITAZONE INTERVENTION GROUPS"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "After DPP interventions, the association between SLC30A8 risk alleles and fasting proinsulin levels was no longer statistically significant.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION AFTER TREATMENT"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "After DPP interventions, the association between SLC30A8 risk alleles and fasting proinsulin levels was no longer statistically significant.)", "entity_type": "UNKNOWN", "id": "GENOTYPE AT SLC30A8 LOCUS"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "Impact of lifestyle changes on fasting proinsulin levels following DPP interventions.)", "entity_type": "UNKNOWN", "id": "IMPACT"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "Impact of lifestyle changes on fasting proinsulin levels following DPP interventions.)", "entity_type": "UNKNOWN", "id": "LIFESTYLE INTERVENTION EFFECTS"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "Comparative analysis between the drug effects of metformin and troglitazone on proinsulin levels.)", "entity_type": "UNKNOWN", "id": "COMPARISON"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "Comparative analysis between the drug effects of metformin and troglitazone on proinsulin levels.)", "entity_type": "UNKNOWN", "id": "METFORMIN VS. TROGLITAZONE"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "The influence of specific genotypes on the effectiveness of diabetes interventions over time.)", "entity_type": "UNKNOWN", "id": "GENOTYPE AT SLC30A8 LOCUS INTERACTION"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "The influence of specific genotypes on the effectiveness of diabetes interventions over time.)", "entity_type": "UNKNOWN", "id": "MODULATION"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "How different treatment responses are influenced by varying age levels within study participants.)", "entity_type": "UNKNOWN", "id": "AGE-DEPENDENCY"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "How different treatment responses are influenced by varying age levels within study participants.)", "entity_type": "UNKNOWN", "id": "TREATMENT RESPONSES WITH AGE ADJUSTMENT"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "The influence of sex on the body\'s response to proinsulin levels post-intervention.)", "entity_type": "UNKNOWN", "id": "GENDER-EFFECT"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "The influence of sex on the body\'s response to proinsulin levels post-intervention.)", "entity_type": "UNKNOWN", "id": "SEX DIFFERENCES IN PROINSULIN RESPONSE"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "How ethnicity impacts the effectiveness of diabetes prevention interventions and outcomes.)", "entity_type": "UNKNOWN", "id": "CULTURAL-EFFECT"}, {"source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "description": "How ethnicity impacts the effectiveness of diabetes prevention interventions and outcomes.)", "entity_type": "UNKNOWN", "id": "ETHNICITY-BASED TREATMENT OUTCOMES"}, {"entity_type": "DIETARY PATTERN", "description": "The high carbohydrate diet increases the expression of PNPLA3 in mouse liver.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "clusters": "[{level: 0, cluster: 19}]", "id": "HC DIET"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Inclusion of lipid emulsion in the HC diet prevents the induction of PNPLA3 mRNA dose-dependently.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "LE (LIPID EMULSION)"}, {"entity_type": "GENE", "description": "The PNPLA3 gene does not respond to insulin and glucagon in HepG2 cells.<SEP>The expression of PNPLA3 is increased by the high carbohydrate diet and attenuated by lipid emulsions.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f<SEP>chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "clusters": "[{level: 0, cluster: 19}]", "id": "PNPLA3"}, {"entity_type": "TRANSCRIPT", "description": "The transcript levels of PNPLA3 correlate with liver total TG content in mice fed a high carbohydrate diet.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "PNPLA3 MRNA"}, {"entity_type": "GENE", "description": "SREBP1c shows a similar expression pattern to that of PNPLA3, and it is correlated with ACC-1 mRNA and PNPLA3 mRNA.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "SREBP1C"}, {"entity_type": "GENE", "description": "Acetyl-CoA Carboxylase 1 (ACC-1), a fatty acid synthase involved in lipid synthesis, had similar expression patterns to PNPLA3.<SEP>Both SREBP1c and ACC-1 mRNA show a similar expression pattern and correlation to PNPLA3, indicating they are involved in the lipogenic pathway.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f<SEP>chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "ACC-1"}, {"entity_type": "GENE", "description": "The expression of PNPLA2 (ATGL) is not different among diets, as it can break down TG.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "PNPLA2 (ATGL)"}, {"entity_type": "DIETARY PATTERN", "description": "Mice fed HF diet developed fatty liver with increased hepatic TG content and suppressed PNPLA3 mRNA levels compared to the LF diet group.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "clusters": "[{level: 0, cluster: 19}]", "id": "HF DIET"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Our in vivo data suggested that exercise may stimulate glucagon secretion, which could potentially decrease PNPLA3 gene expression.<SEP>Voluntary exercise reduces PNPLA3 mRNA levels in mice fed HF diet, decreasing both SREBP1c and ACC-1 gene expression as well.<SEP>Voluntary exercise reduces SREBP1c and ACC-1 mRNA levels in mice fed HF diet, suggesting a role in modulating metabolic pathways related to energy expenditure and fat metabolism.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f<SEP>chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "clusters": "[{level: 0, cluster: 19}]", "id": "EXERCISE"}, {"entity_type": "BLOOD GLUCOSE LEVEL", "description": "Glucose regulates PNPLA3 gene expression in HepG2 cells, where it can induce or suppress the level of expression based on depletion-refeeding conditions.<SEP>The amount of glucose present in the blood, which is essential for providing energy to the body\'s cells. High levels can indicate diabetes or other metabolic disorders.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d", "clusters": "[{level: 0, cluster: 19}]", "id": "GLUCOSE"}, {"entity_type": "TRANSCRIPT", "description": "SREBP1c transcription is regulated by diet, with increased expression on HC diet compared to NC and controlled for by LE.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "clusters": "[{level: 0, cluster: 19}]", "id": "SREBP1C MRNA"}, {"entity_type": "TRANSCRIPT", "description": "ACC-1 mRNA levels are correlated with PNPLA3 and SREBP1c mRNA in mouse liver, indicating coordinated regulation of lipogenic pathways.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "id": "ACC-1 MRNA"}, {"entity_type": "GENE", "description": "FAS gene expression does not differ between LF and HF diet groups but may influence lipid synthesis as part of the lipogenic pathway.", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "clusters": "[{level: 0, cluster: 19}]", "id": "FAS (FATTY ACID SYNTHASE)"}, {"entity_type": "HORMONE/", "description": "Hormone produced by the pancreas that helps regulate blood sugar levels.<SEP>Studies conducted with adipocytes have shown that the PNPLA3 gene expression is regulated by insulin, which increased promoter activity of PNPLA3 gene in Chinese hamster ovary cells.<SEP>SNPs were associated with insulin levels, related to metabolic syndrome development.<SEP>The level of insulin produced and/or circulating in the bloodstream, often used as an indicator of metabolic health or the risk for developing diabetes.<SEP>|>Metabolic Pathway/Cellular Process/Environmental Factor/", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f<SEP>chunk-174831b97ed170039b374ba254d4f506<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d<SEP>chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-f2e122f964bade37f487049bd69a3f9f", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 47}]", "id": "INSULIN"}, {"entity_type": "HORMONE", "description": "It was hypothesized that exercise may stimulate glucagon secretion and decrease PNPLA3 gene expression but our data showed that treatment of HepG2 cells with 100 nM glucagon in low glucose medium had no effect on PNPLA3 mRNA expression.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 46}]", "id": "GLUCAGON"}, {"entity_type": "NUTRIENT", "description": "Several unsaturated fatty acids were investigated for their effect on PNPLA3 gene expression. Linoleic acid (LA), oleic acid (OA), and \u03b1-linolenic acid (ALA) were the major components of Intralipid used in our studies. The results showed that treatment with fatty acids such as OA, LA, EPA, DHA or their combination significantly suppressed PNPLA3 gene expression.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 46}]", "id": "FATTY ACIDS"}, {"entity_type": "FATTY ACID", "description": "EPA and DHA were two long-chain polyunsaturated fatty acids found to suppress PNPLA3 gene expression under both low and high glucose conditions.<SEP>Long-chain polyunsaturated fatty acid EPA suppressed PNPLA3 gene expression under both low and high glucose conditions similar to docosahexaenoic acid (DHA).", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "EICOSAPENTAENOIC ACID"}, {"entity_type": "GENE", "description": "Fatty Acid Synthase (FAS) is a key enzyme in lipogenesis, which was found to be upregulated by insulin stimulation but showed no change when PNPLA3 mRNA expression was decreased.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "FAS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "FAS is an enzyme that converts acetyl-CoA into malonyl-CoA and then further to palmitoyl-CoA during fatty acid synthesis; its expression was observed to be upregulated by insulin but unaffected when PNPLA3 mRNA level was reduced.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "FATTY ACID SYNTHASE"}, {"entity_type": "PROTEIN", "description": "ACC-1 is a protein that catalyzes the first committed step in fatty acid synthesis and its expression patterns were similar to those of PNPLA3.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "ACETYL-COA CARBOXYLASE 1"}, {"entity_type": "NUTRIENT", "description": "Directly added exogenous fatty acids like oleic acid (OA) had effects on PNPLA3 gene expression, decreasing it when co-administered with PNPLA3 siRNA but not alone or under high glucose conditions.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "clusters": "[{level: 0, cluster: 19}]", "id": "EXOGENOUS FATTY ACID"}, {"entity_type": "DIETARY PATTERN", "description": "The diet containing Intralipid\u00ae was used in our studies to induce a condition that completely reversed the induction of PNPLA3 by HC-diet, indicating its role in modulating fatty acid metabolism and potentially affecting PNPLA3 gene expression.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "INTRALIPID\u00ae"}, {"entity_type": "FATTY ACID", "description": "One of the components of Intralipid\u00ae (and major unsaturated fatty acids), Linoleic Acid (LA) when treated directly with HepG2 cells, suppressed PNPLA3 gene expression under both low and high glucose conditions.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "LINOLEIC ACID"}, {"entity_type": "FATTY ACID", "description": "Another component of Intralipid\u00ae, \u03b1-linolenic Acid (ALA), did not affect PNPLA3 gene expression regardless of glucose concentration, differentiating it from other fatty acids that suppressed the expression of PNPLA3.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "\u0391-LINOLENIC ACID"}, {"entity_type": "FATTY ACID", "description": "Another long-chain polyunsaturated fatty acid, DHA also suppressed PNPLA3 gene expression under the same conditions as EPA.", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "id": "DOCOSAHEXAENOIC ACID"}, {"source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "description": "Studies indicated that insulin increased promoter activity of PNPLA3 gene in Chinese hamster ovary cells but incubation with 10 nM insulin for 24 h did not increase PNPLA3 mRNA compared to control cells.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 46}]", "id": "PNPLA3 GENE"}, {"entity_type": "GENE", "description": "A2M is a gene that plays roles in multiple molecular biological processes. Its expression can be influenced by environmental factors.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "A2M"}, {"entity_type": "GENE", "description": "ARID5B is another gene with roles and interactions similar to those of other genes in the list, potentially participating in signaling pathways and cellular functions.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "ARID5B"}, {"entity_type": "GENE", "description": "ARRDC4 is a gene involved in processes like A2M, ARID5B, and others, possibly contributing to biological systems or functions related to cell metabolism or signal transduction.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "ARRDC4"}, {"entity_type": "GENE", "description": "EPHA4 shares characteristics with other genes listed, likely playing roles similar to those described for ISOC1, LDLRAP1, LRRN3, etc., in different cellular and biological contexts.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "EPHA4"}, {"entity_type": "GENE", "description": "FBXO32 is a gene that interacts with others on the list, possibly involved in processes related to protein degradation or cell regulation.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "FBXO32"}, {"entity_type": "GENE", "description": "FoxO1 has been found to be up-regulated and plays significant roles in depression-related pathways when combined with certain environmental factors.<SEP>FoxO1 is a gene involved in the activation of hippocampal progenitor stem cells following cortisol exposure.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182<SEP>chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "FOXO1"}, {"entity_type": "GENE", "description": "HSF2 is a gene that might contribute to stress response or repair mechanisms, similar to other listed genes but distinct by its role in specific cellular processes.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "HSF2"}, {"entity_type": "GENE", "description": "ISOC1 has been modulated with changes in different conditions and interacts with other listed genes as part of complex biological pathways.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "ISOC1"}, {"entity_type": "GENE", "description": "LDLRAP1 is involved in lipid metabolism processes, possibly interacting with others to affect cellular functions or response to stimuli.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "LDLRAP1"}, {"entity_type": "GENE", "description": "LRRN3 shows similarities in biological roles as other genes listed but contributes uniquely through its specific interactions and functions.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "LRRN3"}, {"entity_type": "GENE", "description": "MYO1D interacts with others, contributing to processes related to muscle function or cellular dynamics, based on the context provided.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "MYO1D"}, {"entity_type": "GENE", "description": "PIK3C2B is a gene that plays roles similar to other listed genes in signaling pathways and potentially cell regulation, with interactions dependent on environmental factors.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "PIK3C2B"}, {"entity_type": "GENE", "description": "PPAP2A is another gene whose expression or functions might be influenced by various conditions and can interact with others for specific biological outcomes.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "PPAP2A"}, {"entity_type": "GENE", "description": "SAMD12, like other genes listed, likely has roles in molecular processes but contributes uniquely through its interactions and biological functions.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "SAMD12"}, {"entity_type": "GENE", "description": "SERINC5 is a gene that interacts with others on the list, potentially participating in diverse cellular pathways or biological mechanisms.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "SERINC5"}, {"entity_type": "GENE", "description": "TGF-\u03b21 plays roles in multiple biological processes, often in response to environmental stimuli and interacting with other genes for various outcomes.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "TGF-\u03921"}, {"entity_type": "GENE", "description": "BCL2 is a gene involved in apoptosis regulation, possibly interacting with other genes through signaling pathways.<SEP>Related to snoring, insomnia, and sleep-related traits like daytime dozing or excessive sleepiness.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64<SEP>chunk-01cd90d27130aab1a096ad616e05f182", "id": "BCL2"}, {"entity_type": "GENE", "description": "BCL9 contributes to various cellular processes similar to BCL2 and other listed genes but uniquely participates in specific interactions or biological mechanisms.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "BCL9"}, {"entity_type": "GENE", "description": "LEF1 acts as a transcriptional co-factor influencing gene expression, showing roles similar to ARID5B and others in complex signaling pathways.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "LEF1"}, {"entity_type": "GENE", "description": "LIN54 is involved in the regulation of cell cycle progression or other biological processes related to cell dynamics through interactions with proteins like MYO1D.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "LIN54"}, {"entity_type": "GENE", "description": "PGAP1 interacts with others, possibly contributing to glycosylation processes or affecting protein function and interaction networks.", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "id": "PGAP1"}, {"entity_type": "GENOMIC VARIANT", "description": "283 cardiometabolic SNPs with p-values smaller than expected under the null hypothesis in CRP GWAS.<SEP>The text mentions an 1 SNP with emotional abuse.<SEP>A SNP in the intronic region of C21orf34 detected for respiratory quotient (RQ) during sleep.<SEP>Single nucleotide polymorphisms (SNPs) are variations at a specific position of the genome that may interact with genes to affect ovarian cancer risk.<SEP>7 SNPs were initially identified in the 128 tagging SNPs, including specific SNPs from MTHFD2, BHMT2, NOS3, and SLC19A1 genes.<SEP>A variation in a single DNA building block at a specific position in the genome.<SEP>Single Nucleotide Polymorphism used to construct allelic scores for \'Age at first tooth\' and \'Number of teeth\'.<SEP>The minor allele of each Single Nucleotide Polymorphism (SNP) is associated with a lower risk when combined with type 2 diabetes.<SEP>Single Nucleotide Polymorphisms that were studied for their causal effects on birthweight and B12/folate levels.<SEP>Single Nucleotide Polymorphisms were identified as functionally related through the GRAIL analysis.<SEP>We investigated six MS risk SNPs (all previously identified as eQTLs in LCLs, with evidence implicating EBNA2 binding) and found that EBNA2 binds dependent on the MS risk allele at five loci.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d<SEP>chunk-c63d60b991cd7442f690dcc364af47dd<SEP>chunk-7efd4f037425eeac77e329347b16beb2<SEP>chunk-b6ea16f7f7eb8bc6f36f2be03fe62403<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-704d1bc6fcda99ac888bd36c954fdb98<SEP>chunk-46227b557f62c59155b1b9cef71d0d17<SEP>chunk-0ce38a2d49e01ec790464306e01de9b4<SEP>chunk-ad609cb10e900d42a6901eea1591d8ba<SEP>chunk-bb351b97d99fce93d17df522db92dda7<SEP>chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 42}]", "id": "SNP"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Emotional abuse is an environmental factor that affects the activation of FoxO1 in hippocampal progenitor cells following cortisol exposure.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "EMOTIONAL ABUSE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Cortisol, a stress hormone, is involved in the negative effect of stress on neurogenesis and activates FoxO1 gene expression.<SEP>The pattern of cortisol levels varied by sex based on the ALDH2 rs671 genotypes.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "CORTISOL"}, {"entity_type": "TRANSCRIPT", "description": "The level of FoxO1 transcript was significantly increased following treatment with cortisol.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "FOXO1 MRNA"}, {"entity_type": "PROTEIN", "description": "AS1842856 is a cell-permeable inhibitor of FoxO1 which can completely inhibit the cortisol-induced upregulation of FoxO1 and the reduction in neuronal proliferation.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "AS1842856"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The inhibition of AS1842856 counteracts the cortisol-induced reduction in neuronal proliferation, indicating a role for FoxO1 signaling.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "NEURONAL PROLIFERATION"}, {"entity_type": "GENE", "description": "SGK-1 is a stress-responsive gene that mediates glucocorticoid effects on brain function and the negative impact of cortisol on neurogenesis.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "SERUM GLUCOCORTICOID KINASE 1 (SGK-1)"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The effect of glucocorticoids like cortisol on brain function is mediated by SGK-1, which was found to be elevated following stress and cortisol exposure.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "GLUCOCORTICOID EFFECTS"}, {"entity_type": "DISEASE", "description": "Stress in animal models (stressed rats) led to the elevation of SGK1 mRNA levels.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "STRESSED RATS"}, {"entity_type": "DISEASE", "description": "Similar to stressed rats, depressed patients showed elevated SGK1 levels when exposed to cortisol, highlighting the relevance of this pathway in neurological disorders.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "DEPRESSED PATIENTS"}, {"entity_type": "DIETARY PATTERN", "description": "Table not shown, but it may contain details on dietary patterns related to emotional abuse.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "SUPPLEMENTARY TABLE 9"}, {"entity_type": "PROTEIN", "description": "The established model for testing effects of stress and cortisol exposure.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "IMMORTALIZED, MULTIPOTENT HUMAN FETAL HIPPOCAMPAL PROGENITOR CELL LINE (HPC0A07/03C)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Represents the testing of depression-related changes using an in vitro cell line model.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "DEPRESSION IN A DISH"}, {"entity_type": "TEMPORAL PROCESS", "description": "Duration for which cells were treated with cortisol, indicating time frame of study.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "24\u2009H TREATMENT"}, {"entity_type": "DOSE/QUANTITY", "description": "Cortisol concentration used during the experiment.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "100\u2009\u039cM CORTISOL"}, {"entity_type": "PROTEIN", "description": "Concentration of the FoxO1 inhibitor in the cell culture experiments.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "100\u2009NM AS1842856"}, {"entity_type": "STATISTICAL METHOD", "description": "A statistical test used to determine if there are differences among group means.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "1-WAY ANOVA"}, {"entity_type": "STATISTICAL SIGNIFICANCE", "description": "Indicates a significant increase in FoxO1 mRNA levels with a very low probability of the result occurring by chance.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "FC\u2009=\u2009+\u20093.9 AND P\u2009<\u20090.0001"}, {"entity_type": "CELL TYPE", "description": "Cells that have incorporated BrdU, used to measure proliferation rates.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "BRDU\u2009+\u2009CELLS"}, {"entity_type": "STATISTICAL SIGNIFICANCE", "description": "Indicates a significant increase in SGK-1 mRNA levels with low probability of chance occurrence, as per previous studies.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "FC\u2009=\u2009+\u20093.7 AND P\u2009<\u20090.0001"}, {"entity_type": "OUTCOME/RESULT", "description": "No effect observed when AS1842856 was used without cortisol exposure.", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "id": "AS1842856 ALONE DID NOT EXERT ANY EFFECTS"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "The FoxO1 gene is activated following stress exposure and cortisol treatment, indicating a role in mediating negative effects on neurogenesis.)", "entity_type": "UNKNOWN", "id": "STRESS"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "AS1842856 is able to counteract cortisol-induced reduction in neuronal proliferation, indicating its role in modulating glucocorticoid effects.)", "entity_type": "UNKNOWN", "id": "CORTISOL-INDUCED CHANGES"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "SGK-1 is involved in mediating glucocorticoid effects, specifically cortisol\'s negative impact on neurogenesis.)", "entity_type": "UNKNOWN", "id": "CORTISOL-INDUCED CHANGES IN NEUROGENESIS"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "SGK-1 is involved in mediating glucocorticoid effects, specifically cortisol\'s negative impact on neurogenesis.)", "entity_type": "UNKNOWN", "id": "SGK-1"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "The table may provide details about dietary patterns associated with emotional abuse, although not explicitly mentioned.)", "entity_type": "UNKNOWN", "id": "EMOTIONAL ABUSE (SEE SUPPLEMENTARY TABLE 9)"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "The mRNA levels of FoxO1 are upregulated following stress and cortisol exposure.)", "entity_type": "UNKNOWN", "id": "STRESS-INDUCED UPREGULATION"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "Cortisol was found to reduce neuronal proliferation, as indicated by decreased BrdU+ cells.)", "entity_type": "UNKNOWN", "id": "CORTISOL-INDUCED REDUCTION IN NEURONAL PROLIFERATION (BRDU\u2009+\u2009CELLS)"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "Cortisol was found to reduce neuronal proliferation, as indicated by decreased BrdU+ cells.)", "entity_type": "UNKNOWN", "id": "OUTCOME/EFFECT"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "A 24-hour exposure to cortisol led to an increase in the expression of SGK1 mRNA.)", "entity_type": "UNKNOWN", "id": "24\u2009H CORTISOL TREATMENT"}, {"source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "description": "A 24-hour exposure to cortisol led to an increase in the expression of SGK1 mRNA.)", "entity_type": "UNKNOWN", "id": "EFFECT ON SGK1 MRNA"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs4771122 had the lowest p-value (.0035) among all SNPs for association with long-term weight loss after bariatric surgery.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "RS4771122"}, {"entity_type": "DISEASE", "description": "The primary outcome of interest in this study, which examines the relationship between selected genetic variants and long-term weight loss after bariatric surgery.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "LONG TERM WEIGHT LOSS"}, {"entity_type": "DATA POINT", "description": "Quantitative data points collected one-year or more after surgical intervention to assess changes in weight.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "WEIGHT OBSERVATIONS"}, {"entity_type": "DURATION", "description": "Time period since the surgery during which measurements were taken, averaging around 5.6 years with some as long as 9.5 years.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "FOLLOW-UP LENGTH"}, {"entity_type": "DEMOGRAPHIC VARIABLE", "description": "Proportion of female participants reflecting typical demographics of bariatric surgery patients.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "81 (85%) PARTICIPANTS WERE FEMALE"}, {"entity_type": "GENETIC VARIANT LIST", "description": "Total number of Single Nucleotide Polymorphisms analyzed in the study, with rs4771122 being one that showed significant association.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "34 SNPS"}, {"entity_type": "GENOMIC VARIANT", "description": "Specific genetic variants associated with more weight loss in this study; each additional copy correlated with a larger percentage weight lost.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "ADDITIONAL COPIES OF THE MINOR (G) ALLELE"}, {"entity_type": "BODY MASS INDEX", "description": "A measure indicating participants\' weight-to-height ratio before surgery, considered as a covariate when assessing associations.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "PRE-SURGICAL BMI"}, {"entity_type": "OUTCOME MEASURE", "description": "Minimum percentage of weight lost by each participant during their post-surgery journey.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "NADIR PERCENT WEIGHT LOSS"}, {"entity_type": "GENETIC SAMPLE GROUP", "description": "Subgroup of participants used to assess the association between rs4771122 and outcomes in this demographic, with no significant associations found.", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "id": "SELF-IDENTIFIED BLACKS GENOTYPED"}, {"source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "description": "Additional copies were associated with increased pre-surgical weight in this study.)", "entity_type": "UNKNOWN", "id": "PRE-SURGICAL WEIGHT"}, {"source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "description": "rs4771122 was nominally significantly associated with nadir percent weight loss.)", "entity_type": "UNKNOWN", "id": "SNP RS4771122"}, {"source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "description": "SNP rs4771122 showed association with 2-year percent weight loss.)", "entity_type": "UNKNOWN", "id": "2 YEAR PERCENT WEIGHT LOSS"}, {"entity_type": "GENE", "description": "GCK plays a role in MODY (Maturity-Onset Diabetes of the Young) by carrying variants associated with diabetes.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "GCK"}, {"entity_type": "DISEASE", "description": "GCK-MODY is a type of diabetes that can be diagnosed through genetic variants in GCK gene.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "id": "GCK-MODY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A specific interaction between two or more biological macromolecules involved in various cellular functions.<SEP>ACMG guidelines were used for assessing pathogenicity of novel GCK variants.<SEP>This molecular interaction is implied in the text but not fully detailed.<SEP>The interaction between two or more molecules that can affect biological function.<SEP>An interaction between genetic variants that affect nutrient metabolism and environmental factors such as diet.<SEP>The proteins interact as part of vitamin D metabolism pathway to regulate hormone levels in the body.<SEP>The text describes multiple SNPs associated with changes in baseline and MTD hemoglobin F levels; however, these associations are separate entities rather than a molecular interaction for Delta HbF.<SEP>Lifestyle choices and environmental exposures can impact the interaction between folate metabolism and NOS1 protein activity.<SEP>These molecules interact with EBNA2 or other components in the EBV latency III program, influencing gene expression and immune evasion.", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-2fb95100a1290b27baae869c6560419a<SEP>chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-3cf4e094a5312845a1cc4da4304d4bdf<SEP>chunk-6a657cd0a000b6df4db8f1305b795021<SEP>chunk-b2730f3087aee6d9cff6e4939b229323", "id": "MOLECULAR INTERACTION"}, {"entity_type": "PROTEIN", "description": "A missense mutation was one of the types identified through sequence analysis.<SEP>The p.His317Gln variant was predicted as a moderately conserved sequence change through in-silico tools.<SEP>These proteins are mentioned in the text. However, their specific types (e.g., cytokine receptors, phagocytic markers) aren\'t fully specified here.<SEP>The Protein encoded by the Gene functions in metabolic pathways and interacts with other biological entities.<SEP>SENP1 expression (messenger RNA (mRNA) and protein)<SEP>A specific protein produced from the gene\'s mRNA transcript, involved in various cellular processes.<SEP>These are enzymes involved in vitamin D metabolism, specifically 1\u03b1-hydroxylation of vitamin D3 (CYP27B1) and hydroxylation steps that regulate the concentration of 1,25(OH)2D (CYP24A1).<SEP>These MS risk genes produce proteins that may interact with EBNA2 and be involved in immune evasion mechanisms.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c<SEP>chunk-658df2fc2f9361a65ff1d22464744763<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-2fb95100a1290b27baae869c6560419a<SEP>chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-b2730f3087aee6d9cff6e4939b229323", "id": "PROTEIN"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Clinical findings were similar for probands who carry GCK variants, compared to those described previously.)", "entity_type": "UNKNOWN", "id": "CLINICAL INFORMATION"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Clinical findings were similar for probands who carry GCK variants, compared to those described previously.)", "entity_type": "UNKNOWN", "id": "PROBANDS"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Sequence analysis identified 134 GCK variants in probands from the year 2008 to 2018.)", "entity_type": "UNKNOWN", "id": "GCK VARIANTS"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Sequence analysis identified 134 GCK variants in probands from the year 2008 to 2018.)", "entity_type": "UNKNOWN", "id": "SEQUENCE ANALYSIS"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Establishing co-segregation confirmed that the diabetes phenotype is indeed associated with specific genetic variants.)", "entity_type": "UNKNOWN", "id": "CO-SEGREGATION WITH THE DIABETES PHENOTYPE"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Establishing co-segregation confirmed that the diabetes phenotype is indeed associated with specific genetic variants.)", "entity_type": "UNKNOWN", "id": "PARENTAL OR EXTENDED FAMILY DNA"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Seven VUS were predicted to be damaging by in-silico tools, but they did not satisfy pathogenic or likely pathogenic classification requirements due to lack of segregation and/or phenotypic data.)", "entity_type": "UNKNOWN", "id": "PREDICTIVE TOOLS"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "Seven VUS were predicted to be damaging by in-silico tools, but they did not satisfy pathogenic or likely pathogenic classification requirements due to lack of segregation and/or phenotypic data.)", "entity_type": "UNKNOWN", "id": "VUS (VARIANTS OF UNCERTAIN SIGNIFICANCE)"}, {"source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "description": "One proband had mild fasting hyperglycemia and stable HbA1c level despite being identified with this genetic change, indicating that the clinical manifestation might not always match genetic data.)", "entity_type": "UNKNOWN", "id": "CLINICAL PHENOTYPE"}, {"entity_type": "DIETARY PATTERN", "description": "The Mediterranean diet adherence categories categorized 33%, 51%, and 16% individuals.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 32}]", "id": "MD ADHERENCE"}, {"entity_type": "DIETARY PATTERN", "description": "The Mediterranean diet emphasizes plant-based foods, olive oil, and moderate consumption of fish and dairy products with lower red meat intake.<SEP>The adherence to the Mediterranean diet was generally higher in women, younger individuals with lower BMI and WC, more physically active people, and those with a higher level of education.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450<SEP>chunk-d0476488384aadcae4630e94d982d2e0", "id": "MEDITERRANEAN DIET"}, {"entity_type": "METABOLIC MARKER", "description": "In medium and high MD adherence categories, TAG concentrations were significantly lower.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 32}]", "id": "TAG CONCENTRATIONS"}, {"entity_type": "METABOLIC MARKER", "description": "HDL-C (high-density lipoprotein cholesterol) concentrations were higher in medium and high MD adherence compared to the lowest category.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 32}]", "id": "HDL-C CONCENTRATIONS"}, {"entity_type": "METABOLIC MARKER", "description": "The apoB:apoA-1 ratio was significantly lower in medium and high MD adherence categories.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 32}]", "id": "APOB:APOA-1 RATIO"}, {"entity_type": "METABOLIC MARKER", "description": "HbA1c (glycated hemoglobin) concentration was lower with an increased Mediterranean diet adherence.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 32}]", "id": "HBA1C CONCENTRATION"}, {"entity_type": "LIFESTYLE FACTOR", "description": "A lifestyle factor that further modulates cardiovascular health in relation to genetic factors.<SEP>Physical activity levels were controlled for in the statistical analysis.<SEP>Physical activity was found to be lower among cases compared to controls, potentially affecting health outcomes.<SEP>Physical activity was related to higher adherence of MD among individuals.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-c6fdc5ed25c4e17b8c421a952bb05450<SEP>chunk-d311ac0182b37003c75c0005b4337fa6<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a", "clusters": "[{level: 0, cluster: 4}]", "id": "PHYSICAL ACTIVITY"}, {"entity_type": "BODY MEASUREMENT", "description": "Lower WC was associated with higher adherence to Mediterranean diet.", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 32}]", "id": "WC (WAIST CIRCUMFERENCE)"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Non-smokers tend to consume fewer drinks compared to smokers but doesn\'t significantly moderate medication, genotype effects related to alcohol self-administration.<SEP>Smoking status was considered as an environmental factor when analyzing outcomes.<SEP>Whether or not a person is currently smoking tobacco products.", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d<SEP>chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "SMOKING STATUS"}, {"entity_type": "CATEGORY OF ENVIRONMENTAL FACTOR", "description": "The category \'smoker\' represents a specific instance of smoking status in the environmental factor context.", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "id": "SMOKER"}, {"entity_type": "CATEGORY OF ENVIRONMENTAL FACTOR", "description": "The category \'non-smoker\' represents another instance of smoking status in the environmental factor context.", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "id": "NON-SMOKER"}, {"entity_type": "BIOLOGICAL PARAMETER/PHYSIOLOGICAL PROCESS/METABOLIC PATHWAY/OUTCOME/RESPONSE VARIABLE/MEASURE OF HEALTH STATUS", "description": "There were differences by genotype in TAG levels, which could indicate differences in lipid metabolism or health status.", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "id": "TAG LEVELS"}, {"source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "description": "A gene-diet interaction was examined to see if there were significant interactions between genetics and diet, however no such interactions were found.)", "entity_type": "UNKNOWN", "id": "GENE-DIET INTERACTION"}, {"source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "description": "Smoking status was considered alongside dietary patterns to assess potential interactions on health outcomes.)", "entity_type": "UNKNOWN", "id": "DIETARY PATTERN AND SMOKING STATUS"}, {"source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "description": "Smoking status was considered alongside dietary patterns to assess potential interactions on health outcomes.)", "entity_type": "UNKNOWN", "id": "LIFESTYLE/ENVIRONMENTAL FACTOR INTERACTION"}, {"entity_type": "DISEASE", "description": "Congenital Hyperinsulinism (CHI) is a disorder characterized by excessive insulin secretion from pancreatic beta cells, typically associated with hypoglycemia.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "CHI"}, {"entity_type": "DISEASE", "description": "Macrosomia refers to a newborn that is significantly larger than average for its gestational age or postconceptional age.", "source_id": "chunk-f921f48a406901d82ec2e32af99485bf", "id": "MACROSOMIA"}, {"entity_type": "GENE", "description": "The GLUD1 gene was detected in patients with hypoglycaemia, HI and mild HA.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "GLUD1"}, {"entity_type": "GENOMIC VARIANT", "description": "Three different heterozygous mutations in the GLUD1 gene were identified in five patients.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "HETEROZYGOUS MUTATION"}, {"entity_type": "MUTATION", "description": "The p.Arg322 His mutation was found in patients 54, 55 and 56.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "P.ARG322 HIS"}, {"entity_type": "INHERITANCE PATTERN", "description": "The p.Arg322 His mutation is autosomal and dominantly inherited from their father.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "AUTOSOMAL DOMINANT INHERITANCE"}, {"entity_type": "MUTATION", "description": "The p.Ser498Leu mutation was found in patient 52.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "P.SER498LEU"}, {"entity_type": "MUTATION", "description": "The p.Asn463Asp mutation was found in patient 53.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "P.ASN463ASP"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The serum ammonia concentration of this group of patients was 85-184 \u00b5mol/l.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "SERUM AMMONIA CONCENTRATION"}, {"entity_type": "DISEASE", "description": "HI/HA syndrome due to a GLUD1 genetic defect", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "HI/HA SYNDROME"}, {"entity_type": "OUTCOME", "description": "Of 65 patients, 40 (61.5%) achieved long-term stable glycaemic control by diazoxide alone.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "LONG-TERM STABLE GLYCAEMIC CONTROL"}, {"entity_type": "THERAPY", "description": "Side effects of the diazoxide treatment were observed in 32 (80%) patients.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "DIAZOXIDE TREATMENT"}, {"entity_type": "SIDE EFFECTS", "description": "Gastrointestinal disturbances such as nausea, vomiting, severe gastrointestinal upset and poor appetite occurred in 69%", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "GASTROINTESTINAL DISTURBANCES"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Different degrees of hypertrichosis occurred in 55% (22/40) of patients during clinical follow-up.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "HYPERTRICHOSIS"}, {"entity_type": "SIDE EFFECT", "description": "Effective diazoxide therapy had to be stopped because of thrombocytopenic purpura.<SEP>Thrombocytopenic purpura necessitated stopping diazoxide treatment due to severe blood cell counts issues.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "THROMBOCYTOPENIC PURPURA"}, {"entity_type": "SURGERY", "description": "Nine patients were treated with subtotal pancreatectomy, and one patient underwent pancreatectomy twice.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "SUBTOTAL PANCREATECTOMY"}, {"entity_type": "SURGERY", "description": "Pancreatectomy was performed in 10 patients who were unresponsive to drugs.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "PANCREATECTOMY"}, {"entity_type": "DISEASE", "description": "One patient (patient 7) who underwent surgery at two months of age developed diabetes mellitus at five years of age and was treated with insulin.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "DIABETES MELLITUS"}, {"entity_type": "THERAPY", "description": "Two cases still had mild hypoglycaemia after surgery; one (patient 4) was successfully managed with regular daytime and overnight feedings, the other (patient 9) was treated with diazoxide.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "INSULIN TREATMENT"}, {"entity_type": "OUTCOME", "description": "There were three diazoxide-unresponsive patients: one patient died of multiorgan failure and two patients abandoned the treatment and died of severe hypoglycemia after three to seven days at home.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "SEVERE HYPOGLYCAEMIA"}, {"entity_type": "GENE", "description": "The NOS2 gene plays a role in immune responses and is involved in oxidative stress.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "NOS2"}, {"entity_type": "METABOLIC PATHWAY", "description": "Dysregulation of the NOS1 gene can impact catecholamine synthesis, impacting cardiovascular health.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "CATECHOLAMINE SYNTHESIS"}, {"entity_type": "CELL TYPE", "description": "The genomic variant in the NOS1 gene may affect neural cells specifically, influencing cognitive function and hypertension risk.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "NEURAL CELLS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Cognition is a broad term encompassing mental abilities like memory, learning, attention span, processing speed, and problem-solving.<SEP>NOS1 protein interactions with neurotransmitters contribute to cognitive processes and are influenced by dietary patterns.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd<SEP>chunk-d0476488384aadcae4630e94d982d2e0", "id": "COGNITIVE FUNCTION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Lifestyle factors, such as smoking and stress, can exacerbate the relationship between NOS1 gene variants and cardiovascular disease risk.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "CARDIOVASCULAR DISEASE RISK FACTORS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Certain epigenetic modifications of the GLUD1 gene may lead to drug resistance in patients with HI/HA syndrome.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "DRUG RESISTANCE"}, {"entity_type": "DISEASE", "description": "Hypoglycemia resulting from GLUD1 mutations can be managed through a low-protein diet or medications like diazoxide.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "HYPOGLYCEMIA"}, {"entity_type": "THERAPY RESPONSE", "description": "The development of diabetes mellitus after surgery might require insulin therapy for long-term management.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "INSULIN DEPENDENCY"}, {"entity_type": "OUTCOME", "description": "One patient who experienced multiorgan failure required intensive care and may have had other underlying health conditions.", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "id": "MULTIORGAN FAILURE"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Subtotal pancreatectomy is effective in patients unresponsive to drug therapy, such as diazoxide.)", "entity_type": "UNKNOWN", "id": "SURGERY RESPONSE"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Some patients developed diabetes mellitus after surgery and required insulin treatment.)", "entity_type": "UNKNOWN", "id": "OUTCOME OF SURGERY"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Three patients were unresponsive to diazoxide therapy, leading to severe hypoglycemia and subsequent death.)", "entity_type": "UNKNOWN", "id": "DIAZOXIDE UNRESPONSIVENESS"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Genetic variations in NOS1 may exacerbate or mitigate risk for cardiovascular diseases through interaction with lifestyle factors.)", "entity_type": "UNKNOWN", "id": "NOS1 GENE VARIANTS"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Dietary choices and intake of certain nutrients can influence how NOS1 functions, affecting cognitive health.)", "entity_type": "UNKNOWN", "id": "DIETARY PATTERNS"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Changes in GLUD1 gene expression through epigenetic mechanisms can impact the effectiveness of diazoxide treatment for HI/HA syndrome.)", "entity_type": "UNKNOWN", "id": "DRUG THERAPY EFFECTIVENESS"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Changes in GLUD1 gene expression through epigenetic mechanisms can impact the effectiveness of diazoxide treatment for HI/HA syndrome.)", "entity_type": "UNKNOWN", "id": "EPIGENETIC MODIFICATIONS"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Implementing a low-protein diet is an effective management strategy for patients experiencing hypoglycemia due to GLUD1 mutations.)", "entity_type": "UNKNOWN", "id": "DIETARY INTERVENTIONS"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Implementing a low-protein diet is an effective management strategy for patients experiencing hypoglycemia due to GLUD1 mutations.)", "entity_type": "UNKNOWN", "id": "HYPOGLYCEMIA MANAGEMENT"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "The development of diabetes mellitus after surgery indicates a possible long-term complication requiring specific therapeutic interventions.)", "entity_type": "UNKNOWN", "id": "DIABETES MELLITUS DEVELOPMENT"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "The development of diabetes mellitus after surgery indicates a possible long-term complication requiring specific therapeutic interventions.)", "entity_type": "UNKNOWN", "id": "SURGERY OUTCOME"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Multiorgan failure might be linked to other underlying health conditions or complications in patients undergoing complex surgical procedures.)", "entity_type": "UNKNOWN", "id": "COMORBIDITY"}, {"source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "description": "Treatment outcomes can involve insulin dependency as a result of the management strategies for diabetes mellitus post-surgery.)", "entity_type": "UNKNOWN", "id": "THERAPY RESPONSE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Levels of AcAc across glucose tolerance categories.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "ACAC"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Levels of BHB across glucose tolerance categories.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "BHB"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Evaluation in the Fasting Plasma Glucose (FPG) category.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "FPG"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Evaluation in the 2-hour plasma glucose (2hPG) category.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "2HPG"}, {"entity_type": "DISEASE", "description": "Indicates individuals who have impaired fasting glucose, one of the stages leading to T2DM.<SEP>Individuals with Impaired Fasting Glycemia (IFG).", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f<SEP>chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "IFG"}, {"entity_type": "DISEASE", "description": "Nondiabetic individuals. The reference category for glucose tolerance tests.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "NONDIABETIC"}, {"entity_type": "DISEASE", "description": "Indicates individuals who have impaired glucose tolerance, another stage leading to T2DM.<SEP>Individuals with Impaired Glucose Tolerance (IGT).", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f<SEP>chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "IGT"}, {"entity_type": "DISEASE", "description": "Diabetes diagnosed after baseline, not previously known.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "NEWLY DETECTED DIABETES"}, {"entity_type": "PROTEIN", "description": "The NOS1 protein is involved in vascular function and interacts with folate metabolism.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "NOS1 PROTEIN"}, {"entity_type": "NUTRIENT", "description": "A key nutrient found in leafy greens that affects nitric oxide production.<SEP>Folate is a nutrient found in leafy greens, which are important for cardiovascular health.<SEP>Folate is another nutrient closely related to B12, particularly in the context of maternal health and newborn\'s birthweight.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-f2e122f964bade37f487049bd69a3f9f", "id": "FOLATE"}, {"entity_type": "FOOD", "description": "Folate is abundant in leafy green vegetables.<SEP>Leafy greens are a food source rich in folate, contributing to cardiovascular health through its impact on metabolic pathways.<SEP>This food source provides a rich intake of the above-mentioned nutrient.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-cc528af98598e1adb4684c0166430cf4<SEP>chunk-f2e122f964bade37f487049bd69a3f9f", "id": "LEAFY GREENS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "An external condition that affects the health status of an organism, including climate and pollutants.<SEP>An external factor, such as temperature, pollution, or lifestyle choices, which impacts health outcomes.<SEP>Dietary patterns and physical activity that modulate glucose metabolism and affect the relationship between KBs, glucose tolerance.<SEP>The context doesn\'t suggest any environmental factors are relevant to this activity.<SEP>The interaction of an environmental factor with maternal nutrition could affect the length of pregnancy.<SEP>No specific environmental factors mentioned within this context.<SEP>The expression levels of CHAT may be influenced by environmental factors.<SEP>Exposure to environmental factors can impact these MS risk genes through various mechanisms, including EBNA2 interaction.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-2fb95100a1290b27baae869c6560419a<SEP>chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-f2e122f964bade37f487049bd69a3f9f<SEP>chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-b2730f3087aee6d9cff6e4939b229323<SEP>chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "ENVIRONMENTAL FACTOR"}, {"entity_type": "DRUG", "description": "Drug therapy used for managing cardiovascular disease risk.<SEP>Drugs like statins may be used to manage the risk of diseases related to poor gene and dietary interactions.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-f2e122f964bade37f487049bd69a3f9f", "id": "STATINS"}, {"entity_type": "THERAPY", "description": "Personalized therapies based on genetic factors.<SEP>Therapies designed based on genetic profiles and nutritional data are being explored for cardiovascular health management.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-f2e122f964bade37f487049bd69a3f9f", "id": "PERSONALIZED THERAPY"}, {"entity_type": "BIOMEDICAL PARAMETER", "description": "Poor insulin sensitivity, often seen in metabolic disorders like diabetes, can affect fat metabolism and lead to high triglycerides.<SEP>Refers to how well your body uses insulin.<SEP>The association of rs7574352 with IRF1 could potentially influence insulin sensitivity through its impact on metabolic pathways.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86<SEP>chunk-f2e122f964bade37f487049bd69a3f9f<SEP>chunk-938d8bf2e56584de9dd556be5eb70964", "id": "INSULIN SENSITIVITY"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Process by which the pancreas secretes insulin into the bloodstream.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "INSULIN SECRETION"}, {"entity_type": "STATISTICAL MEASURE", "description": "Measure of insulin sensitivity based on glucose and insulin levels.", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "id": "MATSUDA INSULIN SENSITIVITY INDEX"}, {"entity_type": "GENETIC VARIANT", "description": "The C allele of rs780094 in GCKR gene showed association with increased levels of BHB and AcAc.), (relationship", "source_id": "chunk-bc2193bd869a06007022e1aa75877336", "id": "RS780094"}, {"source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "description": "AcAc levels are inversely associated with insulin sensitivity.)", "entity_type": "UNKNOWN", "id": "ACAC& INSULIN SENSITIVITY"}, {"source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "description": "BHB levels remain inversely associated with insulin sensitivity.)", "entity_type": "UNKNOWN", "id": "BHB& INSULIN SENSITIVITY"}, {"source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "description": "High AcAc levels could weaken or eliminate the association of insulin secretion and development of hyperglycemia and diabetes.)", "entity_type": "UNKNOWN", "id": "ACAC& INSULIN SECRETION"}, {"source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "description": "Similar results are obtained when comparing highest quartile of glucose AUC to three lowest quartiles, indicating an inverse relationship with BHB levels.)", "entity_type": "UNKNOWN", "id": "BHB& INSULIN SECRETION"}, {"source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "description": "Highest quartile AcAc predicts conversion to type 2 diabetes (OR 1.32 [95% CI 1.00\u20131.74], P = 0.047).)", "entity_type": "UNKNOWN", "id": "ACAC& DEVELOPMENT OF HYPERGLYCEMIA"}, {"source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "description": "Highest quartile AcAc predicts conversion to type 2 diabetes (OR 1.32 [95% CI 1.00\u20131.74], P = 0.047).)", "entity_type": "UNKNOWN", "id": "PREDICTOR"}, {"source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "description": "Highest quartile BHB also predicts conversion to type 2 diabetes, albeit with less statistical significance.)", "entity_type": "UNKNOWN", "id": "BHB& DEVELOPMENT OF HYPERGLYCEMIA"}, {"entity_type": "GENE", "description": "Gene location for associations involving T2D and CRP.<SEP>Location for SNPs related to T2D, CRP, and CAD.<SEP>One of 12 genes located within or close to pleiotropic loci associated with CRP.", "source_id": "chunk-372184452f1f9156514992455057f843<SEP>chunk-ad609cb10e900d42a6901eea1591d8ba", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 29}]", "id": "APOC1"}, {"entity_type": "GENE", "description": "Another one of the 12 genes related to pleiotropic loci associated with CRP.<SEP>SNPs associated with CRP, CAD risk reduction for G-allele of rs1183910, and T2D.", "source_id": "chunk-372184452f1f9156514992455057f843<SEP>chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "HNF1A"}, {"entity_type": "GENE", "description": "GCKR is associated with kidney function through its genetic variations.<SEP>Involved in fructose-2,6-bisphosphate aldolase regulation, associated with risk of T2D and CAD.<SEP>Located within or close to a pleiotropic locus associated with CRP.<SEP>One of the genes involved in associations with cardiometabolic SNPs.<SEP>The gene GCKR is associated with increased risk of CKD, and its variant p.Pro446Leu has been shown to result in altered de novo triglyceride and cholesterol synthesis and export as well as decreased plasma glucose concentrations.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083<SEP>chunk-372184452f1f9156514992455057f843<SEP>chunk-3edc72730d68b4c7fd6ba397aa087391<SEP>chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "GCKR"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer.<SEP>Another gene identified as part of pleiotropic loci related to CRP.<SEP>Part of genetic regions with pleiotropic effects on CRP and cardiometabolic phenotypes.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b<SEP>chunk-372184452f1f9156514992455057f843<SEP>chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "IL6R"}, {"entity_type": "GENE", "description": "Located within one of the 12 pleiotropic loci for CRP association.", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "PP1R3B"}, {"entity_type": "GENE", "description": "Associations found with T2D and possibly other cardiometabolic phenotypes.<SEP>Part of a pleiotropic locus associated with CRP.<SEP>Part of multiple associations including T2D, CAD, and CRP.", "source_id": "chunk-372184452f1f9156514992455057f843<SEP>chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "HNF4A"}, {"entity_type": "GENE", "description": "Located near a pleiotropic locus related to CRP.", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "PABPC4"}, {"entity_type": "GENE", "description": "Identified in multiple associations with different outcomes.<SEP>One of the genes within or close to pleiotropic loci for CRP association.", "source_id": "chunk-372184452f1f9156514992455057f843<SEP>chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "BCL7B"}, {"entity_type": "GENE", "description": "Another gene part of one of the pleiotropic loci found to be linked to CRP.", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "TMEM18"}, {"entity_type": "GENE", "description": "Located near a pleiotropic locus related to CRP association.<SEP>One of the genetic regions without significant association with CRP after correction for multiple testing.", "source_id": "chunk-372184452f1f9156514992455057f843<SEP>chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "PLTP"}, {"entity_type": "GENE", "description": "Among 12 identified pleiotropic loci associated with CRP.<SEP>Another gene with pleiotropic effects on CRP but not significantly associated with it after adjustment.", "source_id": "chunk-372184452f1f9156514992455057f843<SEP>chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "MC4R"}, {"entity_type": "SNP", "description": "Near IL1F10 gene on chromosome 6, identified with p-value of 4.0\u00d710^-6 as a novel association.", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "IL1F10 RS11513913"}, {"entity_type": "SNP", "description": "Located near HNF4A gene on chromosome 3, this SNP was found with p-value = 5.6\u00d710^-6 as an association.", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "HNF4A RS2781790"}, {"entity_type": "SNP", "description": "Close to BCL7B gene on chromosome 9, identified with p-value of 7.3\u00d710^-6 as a novel signal.", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "BCL7B RS2824302"}, {"entity_type": "SNP", "description": "In the first intron of FTO gene on chromosome 16, this SNP shows an association with CRP with p-value of 3.0\u00d710^-7.", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "FTO RS12544258"}, {"entity_type": "SNP", "description": "A variant located within APOC1 locus identified with a p-value below 5.0\u00d710^-8 as a significant association.", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "APOC1 RS2999767"}, {"entity_type": "SNP", "description": "In the GCKR gene region on chromosome 9, this SNP has an association with CRP at a p-value of 3.4\u00d710^-6.", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "GCKR RS3747261"}, {"entity_type": "SNP", "description": "This variant near HNF1A locus shows significance in CRP GWAS with p-value = 2.9\u00d710^-6.", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "HNF1A RS10811655"}, {"entity_type": "SNP", "description": "Located near GCKR gene on chromosome 10, this SNP was identified as associated with CRP with a p-value of 4.8\u00d710^-7.", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "GCKR RS732022"}, {"entity_type": "SNP", "description": "Identified in the IL6R locus on chromosome 2 with a p-value below 1.0\u00d710^-5 as an association for CRP.", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "id": "IL6R RS5739889"}, {"entity_type": "GENE", "description": "Identified in associations with CRP and CAD.<SEP>Not identified in previous analysis but associated with CRP and CAD.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "IL1F10"}, {"entity_type": "GENE", "description": "Identified in associations with CRP and cardiometabolic phenotypes.<SEP>Identified in multiple associations including T2D, CAD, and CRP.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "PPP1R3B"}, {"entity_type": "GENE", "description": "Involved in pleiotropic effects on CRP and cardiometabolic phenotypes.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "PABPC"}, {"entity_type": "DISEASE", "description": "Coronary Artery Disease is a condition where CAD represents significant clinical interest.<SEP>Coronary artery disease (CAD) was influenced by the differences explained by the genomic variant rs20417.<SEP>Coronary artery disease associated with certain genetic SNPs.", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17<SEP>chunk-372184452f1f9156514992455057f843<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea", "id": "CAD"}, {"entity_type": "NUTRIENT/CLINICAL PARAMETER", "description": "C-reactive protein (CRP) is an acute-phase reactant that increases during inflammation and can be elevated in conditions like cancer.<SEP>C-reactive protein (CRP), as an acute-phase reactant that increases during inflammation, showed no clear or consistent association with ovarian cancer in the combined analyses.<SEP>CRP is the subject of analysis regarding its pleiotropic relationships with other cardiometabolic phenotypes and SNPs.<SEP>Increased CRP levels are linked to the G-allele of rs1183910 in HNF1A and decreased risk of CAD for A-allele of rs4420638 in APOC1.<SEP>Increased CRP levels showed associations with several SNPs, potentially reflecting inflammation status.<SEP>One of the genes with SNPs associated with risk of ovarian cancer.<SEP>Individuals who develop diabetes typically have higher CRP levels.", "source_id": "chunk-372184452f1f9156514992455057f843<SEP>chunk-b6ea16f7f7eb8bc6f36f2be03fe62403<SEP>chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-fc33d6399ec56e65d007269407f1953d<SEP>chunk-089bcdf055eb2f007db49dbe2409429b", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 29}]", "id": "CRP"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Mediator in the pathway analysis of genetic loci affecting cardiometabolic phenotypes after adjustment.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "HBA1C"}, {"entity_type": "NUTRIENT/DIETARY PATTERN", "description": "High levels of cholesterol is linked to increased sUA level and HUA risk.<SEP>One of several factors that could mediate pleiotropic effects, especially when adjusted for other variables.<SEP>The cholesterol levels were not specified but were likely measured", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729<SEP>chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00<SEP>chunk-372184452f1f9156514992455057f843", "id": "CHOLESTEROL"}, {"entity_type": "PATHWAY", "description": "One of several significant pathways identified in pathway analysis involving pleiotropic genes.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "FXR/RXR ACTIVATION"}, {"entity_type": "PATHWAY", "description": "Significantly enriched canonical pathway in the pathway analysis involving pleiotropic genes.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "LXR/RXR ACTIVATION"}, {"entity_type": "PATHWAY", "description": "Another significantly enriched canonical pathway identified from the pathway analysis.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "MODY SIGNALING"}, {"entity_type": "PATHWAY", "description": "Enriched pathway indicating an association between certain genetic variants and hepatic dysfunction.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "HEPATIC CHOLESTASIS"}, {"entity_type": "PATHWAY", "description": "Significantly enriched pathway related to inflammation and immune system response in the context of pleiotropic effects.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "ACUTE PHASE RESPONSE SIGNALING"}, {"entity_type": "GENE", "description": "Linked to increased risk of CAD and dyslipidemia through pleiotropic effects.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "FABP2"}, {"entity_type": "GENE", "description": "Low-density lipoprotein receptor-related protein 1, involved in lipid metabolism and associated with CRP levels.<SEP>The LPL gene codes for lipoprotein lipase, which plays critical roles in fat metabolism and triglyceride levels.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86<SEP>chunk-372184452f1f9156514992455057f843", "id": "LPL"}, {"entity_type": "GENE", "description": "Plays a role in angiogenesis and inflammation, associated with CAD risk through pleiotropic mechanisms.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "ADAMTS7"}, {"entity_type": "GENE", "description": "Enzyme involved in folate metabolism, linked to CRP levels and T2D.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "MTHFD1L"}, {"entity_type": "GENE", "description": "Important for lipoprotein metabolism, with SNPs showing associations with increased CAD risk.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "APOE"}, {"entity_type": "GENE", "description": "Cytokine gene associated with inflammation response and pleiotropic effects on CRP and HDL-cholesterol.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "IL17F"}, {"entity_type": "GENE", "description": "Involved in docosahexaenoic acid synthesis, linked to lower CRP levels and improved cardiometabolic health.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "AHCY"}, {"entity_type": "GENE", "description": "Regulates genes involved in cholesterol synthesis, showing associations with T2D risk.", "source_id": "chunk-372184452f1f9156514992455057f843", "id": "SREBF1"}, {"source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "description": "The lead SNP in CAD GWAS shows low LD (r^2 = 0.52) with top hit in CRP GWAS, indicating potential pleiotropic effects.)(entity", "entity_type": "UNKNOWN", "id": "CRP SNPS IN CAD GWAS"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "SNP rs4420638, located near APOC1 gene, shows a strong association with type 2 diabetes.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 51}]", "id": "RS4420638"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "The G-allele of the SNP rs1183910 in HNF1A gene is associated with lower risk of coronary artery disease, while its A-allele increases serum CRP levels.)", "entity_type": "UNKNOWN", "id": "G-ALLELE OF RS1183910"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "After adjusting for CRP, the effect of SNP associated with FTO on cardiometabolic phenotypes mainly goes through BMI, suggesting mediated pleiotropy.)", "entity_type": "UNKNOWN", "id": "MEDIATED PLEIOTROPY"}, {"source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9<SEP>chunk-372184452f1f9156514992455057f843", "description": "After adjusting for multiple factors, the effect of BCL7B on HDL-cholesterol is still noticeable but mostly mediated through lipid variables.)<SEP>For BCL7B and HDL-cholesterol levels, lipid variables appear to be the most important mediators after adjustment.)<SEP>High-density lipoprotein cholesterol levels, part of the lipid profile test.", "entity_type": "LABORATORY TEST", "id": "HDL-CHOLESTEROL"}, {"source_id": "chunk-372184452f1f9156514992455057f843<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "Adjustments for CRP led to a considerable decrease in the estimate of association between PABPC4 and triglyceride levels.)<SEP>The amount of triglycerides present in the blood. Triglycerides are a type of fat stored in adipose tissue that can be converted into energy by the body.", "entity_type": "BLOOD TRIGLYCERIDE LEVELS", "id": "TRIGLYCERIDES"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "SNP rs3764260 near IL1F10 gene shows significant association with CRP levels and inflammation.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 29}]", "id": "RS3764260"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "SNP rs12581299 in FABP2 gene is associated with increased risk of T2D.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 51}]", "id": "RS12581299"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "The G-allele of SNP rs7041 in LPL gene shows association with CAD susceptibility.)", "entity_type": "UNKNOWN", "id": "G-ALLELE OF RS7041"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "SNP rs5698321 near MTHFD1L gene affects HDL-cholesterol levels.)", "entity_type": "UNKNOWN", "id": "RS5698321"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "SNP rs17429505 in ADAMTS7 gene is associated with CAD risk through pleiotropic mechanisms.)", "entity_type": "UNKNOWN", "id": "RS17429505"}, {"source_id": "chunk-372184452f1f9156514992455057f843", "description": "SNP rs638982 near SREBF1 gene shows association with T2D susceptibility.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 51}]", "id": "RS638982"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP miR-34b/c rs4938723 was successfully genotyped for patients with IA and controls.", "source_id": "chunk-055177812e320344d131432b292111b0", "id": "MIR-34B/C RS4938723"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP TP53 Arg72Pro was also genotyped along with miR-34b/c rs4938723.", "source_id": "chunk-055177812e320344d131432b292111b0", "id": "TP53 ARG72PRO"}, {"entity_type": "DISEASE", "description": "The study focused on 164 patients with IA and a control group of 426 individuals.", "source_id": "chunk-055177812e320344d131432b292111b0", "id": "IA"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Carriers with the combined genotypes of miR-34b/c rs4938723CC and TP53 Arg72Pro CG/CC/GG had a decreased risk of IA compared to other combined genotypes.)", "entity_type": "UNKNOWN", "id": "MIR-34B/C RS4938723 AND TP53 ARG72PRO"}, {"source_id": "chunk-055177812e320344d131432b292111b0<SEP>chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8<SEP>chunk-d311ac0182b37003c75c0005b4337fa6<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "A gender identifier, typically male or female.<SEP>Stratification analysis by gender did not find any association between the two SNPs and IA risk.)<SEP>Selenium\'s protective effect against colorectal tumors is stronger in men compared to women.<SEP>The majority of patients were female (55%)", "entity_type": "DEMOGRAPHIC", "id": "GENDER"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "No association between the number of aneurysms and miR-34b/c rs4938723 genotypes was observed.)<SEP>No significant association between miR-34b/c rs4938723 and the number of aneurysms was observed.)", "entity_type": "UNKNOWN", "id": "NUMBER OF ANEURYSMS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Association between miR-34b/c rs4938723 genotypes and IA, but not with the number of aneurysms.)", "entity_type": "UNKNOWN", "id": "NUMBER OF ANEURYSMS IN CASES"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Genotype frequencies of miR-34b/c rs4938723 met the Hardy Weinberg equilibrium requirements.)", "entity_type": "UNKNOWN", "id": "HARDY WEINBERG EQUILIBRIUM"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The sex ratio differed, with a higher proportion of female cases relative to male participants.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 31}]", "id": "RATIO BETWEEN GROUPS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The CC genotype of miR-34b/c rs4938723 was associated with a lower risk of IA compared to TT genotype.)", "entity_type": "UNKNOWN", "id": "DECREASE RISK OF IA"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Carriers with the combined genotypes rs4938723CC and TP53 Arg72Pro CG/CC/GG had a reduced risk of IA.)", "entity_type": "UNKNOWN", "id": "COMBINED GENOTYPES"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Gender was used as a variable for stratified analysis.)", "entity_type": "UNKNOWN", "id": "ANALYSIS METHOD"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "P-values for significant associations ranged from 0.004 to 0.006.)", "entity_type": "UNKNOWN", "id": "LEVEL OF SIGNIFICANCE"}, {"source_id": "chunk-055177812e320344d131432b292111b0<SEP>chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "P-values for significant associations ranged from 0.004 to 0.006.)<SEP>The statistical measure that evaluates the strength of association between genotypes and disease risk.", "entity_type": "STATISTICAL MEASURE", "id": "P-VALUE"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Not specified in the text, no relationship mentioned.)", "entity_type": "UNKNOWN", "id": "PERSONALIZATION OF THERAPY"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Total sample size was comprised of cases and controls amounting to 590 participants.)", "entity_type": "UNKNOWN", "id": "SUBJECTS NUMBER"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Two SNPs\u2014miR-34b/c rs4938723 and TP53 Arg72Pro were analyzed for their potential association with IA.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION WITH DISEASE"}, {"source_id": "chunk-159d4c797dae6eb825c74067af703228<SEP>chunk-0d49552280a277a8a26456d06f8f26a8<SEP>chunk-055177812e320344d131432b292111b0<SEP>chunk-50e07db177a6a08799261934e8e71ac4", "description": "Common SNPs were tested for association with age at first tooth and number of teeth.<SEP>Two SNPs\u2014miR-34b/c rs4938723 and TP53 Arg72Pro were analyzed for their potential association with IA.)<SEP>These were crucial in identifying causal relationships as they supported the estimation of causality.<SEP>The associations with the 23 SNPs tested did not reach statistical significance after adjustments for multiple testing.", "entity_type": "GENOMIC VARIANT", "id": "SNPS"}, {"source_id": "chunk-7a423a439657efc1126fb6a075919ea5<SEP>chunk-055177812e320344d131432b292111b0<SEP>chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "description": "Assessment of genotypic frequencies to confirm random mating and no evolutionary pressures.)<SEP>Studies had to be in Hardy-Weinberg equilibrium for statistical analyses, with some subgroups not fulfilling this condition, thus affecting the results.<SEP>No strong evidence for departure from Hardy-Weinberg equilibrium was found except nominally in Romania.", "entity_type": "PHYSIOLOGICAL PROCESS", "id": "HARDY-WEINBERG EQUILIBRIUM"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Assessment of genotypic frequencies to confirm random mating and no evolutionary pressures.)", "entity_type": "UNKNOWN", "id": "TEST TYPE"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "SNPs miR-34b/c rs4938723 and TP53 Arg72Pro were considered in the IA risk assessment.)", "entity_type": "UNKNOWN", "id": "RISK FACTOR FOR IA"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "A p-value less than 0.05 was considered statistically significant for some associations, as seen in miR-34b/c rs4938723 association with IA risk.)", "entity_type": "UNKNOWN", "id": "SIGNIFICANCE LEVEL"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "No specific emphasis was placed on ancestry groups, with the study including participants from diverse backgrounds.)", "entity_type": "UNKNOWN", "id": "GROUP FOCUS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "No specific emphasis was placed on ancestry groups, with the study including participants from diverse backgrounds.)", "entity_type": "UNKNOWN", "id": "INDIVIDUALS ANCESTRY"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The control group was substantially larger than the case group (164 vs. 426 participants).)", "entity_type": "UNKNOWN", "id": "NUMBER OF CONTROLS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The control group was substantially larger than the case group (164 vs. 426 participants).)", "entity_type": "UNKNOWN", "id": "SAMPLE SIZE COMPARISON"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study involved individuals from different demographic groups, though no specific diversity measures were stated.)", "entity_type": "UNKNOWN", "id": "STUDY POPULATION DIVERSITY"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study involved individuals from different demographic groups, though no specific diversity measures were stated.)", "entity_type": "UNKNOWN", "id": "VARIETY OF PARTICIPANTS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study did not specify the age distribution among either cases or controls.)", "entity_type": "UNKNOWN", "id": "AGES RANGE"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study did not specify the age distribution among either cases or controls.)", "entity_type": "UNKNOWN", "id": "CHARACTERISTICS OF SAMPLE POPULATION"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The sex ratio showed an imbalance, with more females than males among the study participants.)", "entity_type": "UNKNOWN", "id": "GENDER PROPORTION"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The number of cases with IA was 164, which constituted a smaller subgroup compared to the control population (426).)", "entity_type": "UNKNOWN", "id": "NUMBER OF CASES"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The number of cases with IA was 164, which constituted a smaller subgroup compared to the control population (426).)", "entity_type": "UNKNOWN", "id": "SPECIFIC GROUP SIZE"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study did not include any data regarding the ancestral backgrounds of participants.)", "entity_type": "UNKNOWN", "id": "ANCESTRY BACKGROUND"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study utilized a case-control approach to compare individuals with IA against those without it.)", "entity_type": "UNKNOWN", "id": "RESEARCH METHODOLOGY"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study utilized a case-control approach to compare individuals with IA against those without it.)", "entity_type": "UNKNOWN", "id": "STUDY DESIGN"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study likely involved statistical tools including Hardy Weinberg equilibrium test, P-value computation for assessing significance, etc., although specific details were not given.)", "entity_type": "UNKNOWN", "id": "RESEARCH ANALYTICAL PROCESS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study likely involved statistical tools including Hardy Weinberg equilibrium test, P-value computation for assessing significance, etc., although specific details were not given.)", "entity_type": "UNKNOWN", "id": "STATISTICAL METHODOLOGY"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study aimed at determining if genetic variants could predict or influence the incidence of IA.)", "entity_type": "UNKNOWN", "id": "RESEARCH OBJECTIVE"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study aimed at determining if genetic variants could predict or influence the incidence of IA.)", "entity_type": "UNKNOWN", "id": "SCIENTIFIC INQUIRY"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study included genetic variables like miR-34b/c rs4938723 and TP53 Arg72Pro, alongside other potential risk factors.)", "entity_type": "UNKNOWN", "id": "RISK DETERMINANTS FOR DISEASE"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study included genetic variables like miR-34b/c rs4938723 and TP53 Arg72Pro, alongside other potential risk factors.)", "entity_type": "UNKNOWN", "id": "SNP VARIABLES"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The study did not specify the ethnic or ancestral diversity among participants.)", "entity_type": "UNKNOWN", "id": "HERITAGE VARIABILITY"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Findings were interpreted statistically, comparing case and control groups for identifying potential risk factors.)", "entity_type": "UNKNOWN", "id": "DATA ANALYSIS PROCESS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Findings were interpreted statistically, comparing case and control groups for identifying potential risk factors.)", "entity_type": "UNKNOWN", "id": "STATISTICAL EVALUATION"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "While the study included gender as an analysis variable, it found no consistent sex-specific associations with IA risk based on genetic markers.)", "entity_type": "UNKNOWN", "id": "INFLUENCE ON RESEARCH OUTCOMES"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "While the study included gender as an analysis variable, it found no consistent sex-specific associations with IA risk based on genetic markers.)", "entity_type": "UNKNOWN", "id": "SEX FACTOR"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The sample encompassed both case and control groups that differed in terms of genetic risk factors, sex ratio, age range, etc., though exact details were not provided.)", "entity_type": "UNKNOWN", "id": "CHARACTERISTICS OF PARTICIPANTS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The sample encompassed both case and control groups that differed in terms of genetic risk factors, sex ratio, age range, etc., though exact details were not provided.)", "entity_type": "UNKNOWN", "id": "STUDY SAMPLE PROFILES"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Results with P-values less than 0.05 were considered statistically significant and worthy of further investigation.)", "entity_type": "UNKNOWN", "id": "DETERMINING STATISTICAL RELEVANCE"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Results with P-values less than 0.05 were considered statistically significant and worthy of further investigation.)", "entity_type": "UNKNOWN", "id": "SIGNIFICANCE THRESHOLD FOR P-VALUES"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Rigorous approaches were taken with genetic data validation and statistical significance determination to ensure the robustness of findings.)", "entity_type": "UNKNOWN", "id": "GENETIC DATA ANALYSIS RIGOR"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Rigorous approaches were taken with genetic data validation and statistical significance determination to ensure the robustness of findings.)", "entity_type": "UNKNOWN", "id": "QUALITY ASSURANCE IN RESEARCH METHODOLOGY"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "While ancestry was indirectly relevant through the sample\'s diversity, its direct impact on genetic findings for IA was not specifically addressed or analyzed.)", "entity_type": "UNKNOWN", "id": "HERITAGE INFLUENCE"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "While ancestry was indirectly relevant through the sample\'s diversity, its direct impact on genetic findings for IA was not specifically addressed or analyzed.)", "entity_type": "UNKNOWN", "id": "OUTCOME DETERMINANTS IN GENETIC RESEARCH"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "In the context of IA risk, no distinct gender-specific effects were discerned in relation to genetic markers such as miR-34b/c rs4938723 or TP53 Arg72Pro.)", "entity_type": "UNKNOWN", "id": "GENDER-SPECIFIC ANALYSIS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "In the context of IA risk, no distinct gender-specific effects were discerned in relation to genetic markers such as miR-34b/c rs4938723 or TP53 Arg72Pro.)", "entity_type": "UNKNOWN", "id": "SNP INFLUENCE ON DISEASE RISK"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "A mix of statistical analyses was employed to identify patterns or associations in the data related to genetic markers and IA disease prevalence.)", "entity_type": "UNKNOWN", "id": "EXAMINATION TECHNIQUES FOR VARIABLES"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "A mix of statistical analyses was employed to identify patterns or associations in the data related to genetic markers and IA disease prevalence.)", "entity_type": "UNKNOWN", "id": "STATISTICAL TOOLS USED"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "While the impact of specific genes was explored through SNPs like miR-34b/c rs4938723 and TP53 Arg72Pro, ancestral genetic influences were not a focal point.)", "entity_type": "UNKNOWN", "id": "HERITAGE GENETICS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "While the impact of specific genes was explored through SNPs like miR-34b/c rs4938723 and TP53 Arg72Pro, ancestral genetic influences were not a focal point.)", "entity_type": "UNKNOWN", "id": "INFLUENCE ON DISEASE SUSCEPTIBILITY"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Analysis of genetic markers did not highlight any sex-specific patterns or differences that could significantly influence IA risk based on the data provided.)", "entity_type": "UNKNOWN", "id": "SEX DISTRIBUTION IMPACT"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Genetic links identified required a statistical significance ascertained through P-values below the established threshold to be considered reliable.)", "entity_type": "UNKNOWN", "id": "DETERMINING GENETIC ASSOCIATIONS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Genetic links identified required a statistical significance ascertained through P-values below the established threshold to be considered reliable.)", "entity_type": "UNKNOWN", "id": "STATISTICAL SIGNIFICANCE CRITERION"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "While sex could indirectly affect how genetic heritage influences IA risk, its specific effect was not a central focus given the study\'s concentration on SNPs like miR-34b/c rs4938723 and TP53 Arg72Pro.)", "entity_type": "UNKNOWN", "id": "GENDER ROLE IN HEREDITARY IMPACT"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "While sex could indirectly affect how genetic heritage influences IA risk, its specific effect was not a central focus given the study\'s concentration on SNPs like miR-34b/c rs4938723 and TP53 Arg72Pro.)", "entity_type": "UNKNOWN", "id": "MEDIATION OF GENETIC INFLUENCE ON DISEASE RISK"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Ancestry diversity was not considered in the assessment of genetic markers\' relevance to IA risk, potentially missing nuanced insights into how different populations may differently respond based on their unique genetic makeup.)", "entity_type": "UNKNOWN", "id": "ASSESSING GENETIC MARKER ASSOCIATIONS WITH DISEASE RISK"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Ancestry diversity was not considered in the assessment of genetic markers\' relevance to IA risk, potentially missing nuanced insights into how different populations may differently respond based on their unique genetic makeup.)", "entity_type": "UNKNOWN", "id": "HERITAGE DIVERSITY INTERPRETATION"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Finding a correlation between genomic factors and IA disease states required statistical evidence with P-values below the specified threshold for reliability.)", "entity_type": "UNKNOWN", "id": "DETERMINING CORRELATION WITH DISEASE"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Finding a correlation between genomic factors and IA disease states required statistical evidence with P-values below the specified threshold for reliability.)", "entity_type": "UNKNOWN", "id": "STATISTICAL SIGNIFICANCE CRITERIA FOR GENOMIC FACTORS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "SNPs were considered possible influencers of IA risk through statistical methods, seeking evidence that they could be linked to disease prevalence or predisposition within the sample populations.)", "entity_type": "UNKNOWN", "id": "GENETIC MARKERS FOR RISK ASSESSMENT"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "SNPs were considered possible influencers of IA risk through statistical methods, seeking evidence that they could be linked to disease prevalence or predisposition within the sample populations.)", "entity_type": "UNKNOWN", "id": "POTENTIAL INFLUENCERS IN DISEASE STATE DETERMINATION"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "While sex could theoretically affect the correlations between specific SNPs and IA, its impact was not a primary focus given the study\'s emphasis on establishing associations rather than exploring nuanced gender-related dynamics.)", "entity_type": "UNKNOWN", "id": "GENDER INFLUENCE ON GENETIC MARKER CORRELATIONS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "While sex could theoretically affect the correlations between specific SNPs and IA, its impact was not a primary focus given the study\'s emphasis on establishing associations rather than exploring nuanced gender-related dynamics.)", "entity_type": "UNKNOWN", "id": "POTENTIAL MODIFIERS IN SNP-AIA RISK ASSOCIATION"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Given the primary focus on SNPs, no explicit analysis was conducted into how ancestral backgrounds might influence the relevance or predictive power of genetic markers in IA risk determination.)", "entity_type": "UNKNOWN", "id": "ANCESTRY CONSIDERATION IN GENETIC MARKER CORRELATION"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Given the primary focus on SNPs, no explicit analysis was conducted into how ancestral backgrounds might influence the relevance or predictive power of genetic markers in IA risk determination.)", "entity_type": "UNKNOWN", "id": "RELEVANCE TO DISEASE RISK ASSESSMENT"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Genomic factors were evaluated for their correlation with disease prevalence using statistical methods like P-values and the Hardy Weinberg equilibrium test, seeking evidence that they could be predictive of IA risk based on data from case versus control groups.)", "entity_type": "UNKNOWN", "id": "CORRELATION WITH DISEASE PREVALENCE DETERMINATION"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Genomic factors were evaluated for their correlation with disease prevalence using statistical methods like P-values and the Hardy Weinberg equilibrium test, seeking evidence that they could be predictive of IA risk based on data from case versus control groups.)", "entity_type": "UNKNOWN", "id": "STATISTICAL ASSESSMENT IN GENOMIC FACTOR EVALUATION"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "While sex was acknowledged as part of the population characteristics, its role did not feature prominently in the direct correlation assessments between genetic markers and IA risk based on the methodology employed by the study.)", "entity_type": "UNKNOWN", "id": "INFLUENCE IN DISEASE RISK ASSESSMENT THROUGH SNPS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "While sex was acknowledged as part of the population characteristics, its role did not feature prominently in the direct correlation assessments between genetic markers and IA risk based on the methodology employed by the study.)", "entity_type": "UNKNOWN", "id": "SEX ROLE IN GENETIC MARKER CORRELATION EVALUATION"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Given the study\'s focus on establishing correlations rather than exploring nuanced population dynamics, heritage diversity was not a primary factor considered when assessing the relevance of genetic markers like miR-34b/c rs4938723 and TP53 Arg72Pro in IA risk determinations through statistical means.)", "entity_type": "UNKNOWN", "id": "HERITAGE DIVERSITY\'S ROLE IN GENETIC MARKER CORRELATION EVALUATION"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "Given the study\'s focus on establishing correlations rather than exploring nuanced population dynamics, heritage diversity was not a primary factor considered when assessing the relevance of genetic markers like miR-34b/c rs4938723 and TP53 Arg72Pro in IA risk determinations through statistical means.)", "entity_type": "UNKNOWN", "id": "RELEVANCE TO DISEASE RISK THROUGH STATISTICAL ASSESSMENT"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "To assess the meaningful impact of genetic factors on IA disease states, P-value thresholds were applied during comparative case-control analyses involving SNPs like miR-34b/c rs4938723 and TP53 Arg72Pro to determine if they significantly correlated with disease prevalence or predisposition.)", "entity_type": "UNKNOWN", "id": "SIGNIFICANCE IN DISEASE STATE EVALUATION THROUGH COMPARATIVE ANALYSIS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "To assess the meaningful impact of genetic factors on IA disease states, P-value thresholds were applied during comparative case-control analyses involving SNPs like miR-34b/c rs4938723 and TP53 Arg72Pro to determine if they significantly correlated with disease prevalence or predisposition.)", "entity_type": "UNKNOWN", "id": "STATISTICAL CRITERIA FOR GENETIC FACTOR IMPACT DETERMINATION"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The evaluation process sought to ascertain if specific genetic markers could contribute meaningfully to IA risk determinations by analyzing their correlations with case versus control populations, potentially factoring in sex or heritage diversity as secondary considerations.)", "entity_type": "UNKNOWN", "id": "CORRELATION DETERMINATION FOR DISEASE PREVALENCE THROUGH STATISTICAL ANALYSIS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The evaluation process sought to ascertain if specific genetic markers could contribute meaningfully to IA risk determinations by analyzing their correlations with case versus control populations, potentially factoring in sex or heritage diversity as secondary considerations.)", "entity_type": "UNKNOWN", "id": "GENETIC MARKER EVALUATION IN RISK ASSESSMENT"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The process involved applying statistical tools like P-values and Hardy Weinberg equilibrium tests to assess if correlations exist between specific SNPs and IA disease states, with sex or heritage diversity factored into secondary analyses rather than being the primary focus in determining genetic marker significance.)", "entity_type": "UNKNOWN", "id": "CORRELATION ASSESSMENT FOR DISEASE STATE DETERMINATION THROUGH STATISTICAL ANALYSIS"}, {"source_id": "chunk-055177812e320344d131432b292111b0", "description": "The process involved applying statistical tools like P-values and Hardy Weinberg equilibrium tests to assess if correlations exist between specific SNPs and IA disease states, with sex or heritage diversity factored into secondary analyses rather than being the primary focus in determining genetic marker significance.)", "entity_type": "UNKNOWN", "id": "STATISTICAL SIGNIFICANCE IN GENETIC MARKER CORRELATION EVALUATION"}, {"entity_type": "DISEASE", "description": "A Genome-Wide Association Study that investigates the relationship between genetic variants and birthweight.<SEP>A genome-wide association study (GWAS) was conducted on children from Hispanic families to identify obesity-related traits associated with genetic variants.<SEP>Genome-wide association study conducted for \'Age at first tooth\', resulting in allelic score construction.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-c63d60b991cd7442f690dcc364af47dd<SEP>chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "GWAS"}, {"entity_type": "TRANSCRIPT", "description": "Fat-free mass (FFM) was measured as part of body composition assessment.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "FFM"}, {"entity_type": "TRANSCRIPT", "description": "Body fat mass (FM) was measured to assess adiposity and body composition.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "FM"}, {"entity_type": "GENE", "description": "A nonsynonymous single nucleotide polymorphism (SNP; rs1056513, G1178S (NP_005790.2)) in the INADL gene on chromosome 1 was associated with body weight and body composition.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "INADL"}, {"entity_type": "GENE", "description": "An intronic variant of COL4A1 on chromosome 13 was related to weight z-score change during the study.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "COL4A1"}, {"entity_type": "GENE", "description": "An intronic variant of MTNR1B on chromosome 11 showed strong association with fasting glucose levels.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "MTNR1B"}, {"entity_type": "GENOMIC REGION", "description": "Intronic and 3\u2032UTR variants in the APOA5-ZNF259 region on chromosome 11 were associated with triglyceride levels.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "APOA5-ZNF259"}, {"entity_type": "GENE", "description": "An intronic variant of PCSK2 on chromosome 20 was linked to total antioxidants.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "PCSK2"}, {"entity_type": "GENOMIC REGION", "description": "Variants in the flanking 3\u2032UTR regions of RNASE1 on chromosome 14 were associated with urinary nitrogen excretion.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "RNASE1"}, {"entity_type": "GENOMIC REGION", "description": "An intronic variant of ASS1P11 on chromosome 7 was related to creatinine excretion.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "ASS1P11"}, {"entity_type": "GENE", "description": "A variant in the intron of GCH1 on chromosome 14 associated with urinary dopamine: creatinine ratio.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "GCH1"}, {"entity_type": "GENE", "description": "A nonsynonymous SNP (rs3733402, S143T (NP_000883.2)) in KLKB1 on chromosome 4 was related to serum free IGF-1 levels.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "KLKB1"}, {"entity_type": "GENE", "description": "An intronic SNP of MPRIP on chromosome 17 was associated with serum IGFBP-3 levels.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "MPRIP"}, {"entity_type": "GENOMIC REGION", "description": "A block of twenty-three SNPs in the flanking 5\u2032UTR region of XPA/FOXE1 (TTF-2) on chromosome 9 was highly associated with serum TSH levels.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "XPA/FOXE1 (TTF-2)"}, {"entity_type": "GENE", "description": "A nonsynonymous SNP (rs12075, G42D (NP_002027.2)) and an intronic variant of DARC on chromosome 1 were found to be strongly associated with MCP-1 levels.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "DARC"}, {"entity_type": "GENE", "description": "Coding variants in GREB1 on chromosome 2 were identified as related to MCP-1 levels.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "GREB1"}, {"entity_type": "GENE", "description": "An intronic variant of DFNB31 on chromosome 9 was linked with MCP-1 levels, among other inflammation markers.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "DFNB31"}, {"entity_type": "GENE", "description": "A variant in the 3\u2032UTR region for CCR3 showed high association with MCP1 and related inflammation markers.<SEP>Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-089bcdf055eb2f007db49dbe2409429b", "id": "CCR3"}, {"entity_type": "GENE", "description": "Variants in the intronic region of RASGEF1A on chromosome 9 were associated with MCP-1, among other inflammation-related SNPs.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "RASGEF1A"}, {"entity_type": "GENE", "description": "An intronic SNP of ABO was strongly related to IL-6 levels.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "ABO"}, {"entity_type": "GENE", "description": "An intronic variant in TMEM229B on chromosome 14 showed significance for ad libitum energy intake at dinner.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "TMEM229B"}, {"entity_type": "GENE", "description": "An intronic variant in ARHGAP11A on chromosome 15 was associated with sleep duration.<SEP>An intronic variant of ARHGAP11A on chromosome 15 was related to sleep duration, which affects energy expenditure and substrate utilization.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-46227b557f62c59155b1b9cef71d0d17", "id": "ARHGAP11A"}, {"entity_type": "GENOMIC REGION", "description": "A SNP in the intronic region of C21orf34 was detected as associated with respiratory quotient (RQ) during sleep, impacting energy metabolism.<SEP>A SNP in the intronic region of C21orf34 was detected for respiratory quotient (RQ) during sleep.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-46227b557f62c59155b1b9cef71d0d17", "id": "C21ORF34"}, {"entity_type": "DISEASE", "description": "Polycystic ovary syndrome (PCOS) is a common endocrine disorder characterized by hyperandrogenism, chronic anovulation, and polycystic ovaries.<SEP>Polycystic ovary syndrome (PCOS), a common endocrine disorder in women, showed a link with specific SNPs.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5<SEP>chunk-6794be47bcde46ad4a07e0cc372ea3d2", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 45}]", "id": "PCOS"}, {"entity_type": "DISEASE", "description": "A chronic metabolic disease marked by elevated blood sugar levels due to insufficient production or use of insulin.<SEP>Autoimmune responses in the pancreas can lead to diabetes, characterized by insulin deficiency due to immunological destruction of pancreatic \u03b2 cells.<SEP>SNPs were found to be associated with type 2 diabetes and insulin resistance.<SEP>The presence or absence of diabetes was also included as a covariate in the models.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433<SEP>chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a<SEP>chunk-d0476488384aadcae4630e94d982d2e0", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 28}]", "id": "DIABETES"}, {"entity_type": "CONDITION", "description": "An intronic variant of the ELK1 gene was linked to obstructive sleep apnea.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "OBSTRUCTIVE SLEEP APNEA"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "High-density lipoprotein (HDL) cholesterol is a type of cholesterol that helps remove excess cholesterol from the body.<SEP>SNPs were associated with HDL (high-density lipoprotein) cholesterol levels, impacting cardiovascular risk.<SEP>rs4988235 showed a modestly lower HDL cholesterol level, suggesting an impact on lipid metabolism.<SEP>rs4988235 was associated with lower HDL cholesterol, suggesting an impact on lipid metabolism.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "HDL CHOLESTEROL"}, {"entity_type": "BIOMARKER", "description": "Certain SNPs were related to LDL (low-density lipoprotein) cholesterol, influencing heart disease risk.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "LDL CHOLESTEROL"}, {"entity_type": "ENZYME", "description": "Variants were found to be linked with AST (aspartate aminotransferase), indicating liver health.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "AST"}, {"entity_type": "ENZYME", "description": "SNPs influenced ALT (alanine transaminase) levels, relevant for liver function assessment.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "ALT"}, {"entity_type": "BIOMARKER", "description": "An intron variant of the PCSK9 gene was related to LDL-particle size distribution (LDL-P), impacting atherogenicity.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "LDL-P"}, {"entity_type": "PROTEIN", "description": "SNPs correlated with CRP levels were found, indicating inflammation and risk for chronic diseases.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "C-REACTIVE PROTEIN"}, {"entity_type": "ENZYME", "description": "Variants influenced Gamma-glutamyl transferase (GGT) activity, reflecting liver health and metabolism.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "GAMMA-GLUTAMYL TRANSFERASE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A variant in KLKB1 was found linked to serum IGF-1 (insulin-like growth factor 1), indicating growth and health status.<SEP>Insulin-like growth factor 1 (IGF-1) is a peptide hormone that plays a role in cell proliferation and survival.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2<SEP>chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "IGF-1"}, {"entity_type": "HORMONE", "description": "SNPs affecting MPRIP influenced IGFBP-3 levels, impacting metabolism of insulin-like growth factors.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "IGFBP-3"}, {"entity_type": "HORMONE", "description": "Twenty-three SNPs in the flanking 5\'UTR region of XPA/FOXE1 were associated with serum TSH (thyroid-stimulating hormone) levels.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 45}]", "id": "TSH"}, {"entity_type": "CHEMOKINE", "description": "MCP-1 plays a role in recruiting leukocytes into the inflamed colonic tissue.<SEP>SNPs related to MCP-1 (monocyte chemoattractant protein 1), a key cytokine in inflammation and immune response.", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6<SEP>chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "MCP-1"}, {"entity_type": "CYTOKINE", "description": "An intron variant of ABO was associated with IL-6 levels, indicating inflammation status and health risks.<SEP>IL-6 is a pro-inflammatory cytokine that impacts T cell function and disease severity.<SEP>IL-6, like IL-1\u03b2 and TNF-\u03b1, is increased in colitis and contributes to the intensification of the Th17 response.<SEP>Increased levels of IL-1 and IL-6 from LPM\u03c6s can impact Th17 cell maturation in diseased mice.<SEP>The gene expression levels of IL-6, TNF-\u03b1 and IL-1\u03b2 were increased in the SENP1 knockdown cells.<SEP>|>Disease/Inflammation", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6<SEP>chunk-b971aa450a9b220806bdc000eb5efaad<SEP>chunk-174831b97ed170039b374ba254d4f506<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce<SEP>chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "IL-6"}, {"entity_type": "MARKER", "description": "Multiple genetic variants were linked to various inflammation markers, highlighting their role in disease risk.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "INFLAMMATION MARKER"}, {"entity_type": "PHYSIOLOGICAL MEASUREMENT", "description": "A SNP within the intron of C21orf34 was associated with RQ during sleep, reflecting metabolic rate and substrate utilization.", "source_id": "chunk-6794be47bcde46ad4a07e0cc372ea3d2", "id": "RESPIRATORY QUOTIENT (RQ)"}, {"entity_type": "TRANSCRIPT", "description": "A variant was identified for accelerometer-measured light activity in the 5\u2032UTR region of RPL7P3 on chromosome 9.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "RPL7P3"}, {"entity_type": "TRANSCRIPT", "description": "Sedentary-light activity was measured in the 3\u2032UTR of CTCFL on chromosome 20.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "CTCFL"}, {"entity_type": "GENE", "description": "A highly significant variant for total energy expenditure (p\u200a=\u200a2.7E-08) was identified for MATK on chromosome 19.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "MATK"}, {"entity_type": "GENE", "description": "Sleeping energy expenditure (p\u200a=\u200a6.0E-08) variant was identified for CHRNA3 on chromosome 15.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "CHRNA3"}, {"entity_type": "ARHGAP11A 15TH CHROMOSOME", "description": "An intronic variant in ARHGAP11A on chromosome 15 was associated with sleep duration.", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "id": "INTRONIC VARIANT"}, {"entity_type": "GENOMIC VARIANT", "description": "A variant identified for accelerometer-measured light activity in the 5\u2032UTR region of RPL7P3 on chromosome 9.<SEP>Sedentary-light activity was measured in the 3\u2032UTR of CTCFL on chromosome 20.<SEP>Data demonstrate LYP variants, 620R and 620W, result in similar levels of effector cytokine secretion.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d<SEP>chunk-46227b557f62c59155b1b9cef71d0d17", "id": "VARIANT"}, {"source_id": "chunk-9977706363cc368b3be503d07ed3ef62<SEP>chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Shorter sleep duration was associated with higher snoring prevalence in the sample.<SEP>The intronic variant in ARHGAP11A on chromosome 15 was associated with sleep duration.)", "entity_type": "RISK FACTOR", "id": "SLEEP DURATION"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "A SNP in the intronic region of C21orf34 detected for RQ during sleep.)", "entity_type": "UNKNOWN", "id": "RQ (RESPIRATORY QUOTIENT)"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Accelerometer-measured light activity variant was found in the 5\u2032UTR region of RPL7P3.)", "entity_type": "UNKNOWN", "id": "LIGHT ACTIVITY"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Accelerometer-measured light activity variant was found in the 5\u2032UTR region of RPL7P3.)", "entity_type": "UNKNOWN", "id": "RPL7P3 5\u2032UTR"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Sedentary-light activity was measured in the 3\u2032UTR of CTCFL on chromosome 20.)<SEP>Sedentary-light activity was measured using CTCFL\'s 3\u2032UTR sequence on chromosome 20.)", "entity_type": "UNKNOWN", "id": "CTCFL 3\u2032UTR"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Sedentary-light activity was measured in the 3\u2032UTR of CTCFL on chromosome 20.)<SEP>Sedentary-light activity was measured using CTCFL\'s 3\u2032UTR sequence on chromosome 20.)", "entity_type": "UNKNOWN", "id": "SEDENTARY-LIGHT ACTIVITY"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "A highly significant variant identified for total energy expenditure (p\u200a=\u200a2.7E-08) was associated with MATK on chromosome 19.)", "entity_type": "UNKNOWN", "id": "MATK CHROMOSOME 19"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "A highly significant variant identified for total energy expenditure (p\u200a=\u200a2.7E-08) was associated with MATK on chromosome 19.)", "entity_type": "UNKNOWN", "id": "TOTAL ENERGY EXPENDITURE"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Sleeping energy expenditure variant identified for CHRNA3 on chromosome 15.)", "entity_type": "UNKNOWN", "id": "CHRNA3 CHROMOSOME 15"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Sleeping energy expenditure variant identified for CHRNA3 on chromosome 15.)", "entity_type": "UNKNOWN", "id": "SLEEPING ENERGY EXPENDITURE"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "A highly significant variant associated with total energy expenditure (p\u200a=\u200a2.7E-08) was identified for MATK on chromosome 19.)", "entity_type": "UNKNOWN", "id": "MATK 19TH CHROMOSOME"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Sleeping energy expenditure variant identified for CHRNA3 on chromosome 15.)", "entity_type": "UNKNOWN", "id": "CHRNA3 15TH CHROMOSOME"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "A SNP in the C21orf34 intronic region detected for RQ during sleep.)", "entity_type": "UNKNOWN", "id": "C21ORF34 INTRONIC REGION"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "An intronic variant in ARHGAP11A on chromosome 15 was associated with sleep duration.)", "entity_type": "UNKNOWN", "id": "ARHGAP11A 15TH CHROMOSOME"}, {"source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "description": "Accelerometer-measured light activity variant found in the RPL7P3 5\u2032UTR region.)", "entity_type": "UNKNOWN", "id": "RPL7P3 5\u2032UTR REGION"}, {"entity_type": "GENE", "description": "The APOBR gene encodes a protein that plays a role in lipid metabolism.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "APOBR"}, {"entity_type": "GENOMIC VARIANT", "description": "A single nucleotide variant located in the coding region of the APOBR gene.<SEP>Located in the un-translated region (UTR) or intronic regions and has no significant functional impact.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS74949322"}, {"entity_type": "GENOMIC VARIANT", "description": "Another variant found within the APOBR gene, affecting the amino acid sequence of its protein product.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS151233"}, {"entity_type": "GENOMIC VARIANT", "description": "A synonymous variant in the coding region of the APOBR gene, with no impact on the resulting amino acid sequence.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS149271"}, {"entity_type": "GENOMIC VARIANT", "description": "Mutates the amino acid sequence by deleting several residues and is located within a repeat region.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS3833080"}, {"entity_type": "GENOMIC VARIANT", "description": "A non-synonymous variant that causes an amino acid change, creating a new amino acid residue (Pro428Ala).", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS180743"}, {"entity_type": "GENOMIC VARIANT", "description": "Synonymous variant found in the APOBR gene.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS151174"}, {"entity_type": "GENOMIC VARIANT", "description": "A synonymous variant found in the APOBR gene, with a minor allele frequency of 0.023.<SEP>A synonymous variant within the coding region of the APOBR gene.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS40831"}, {"entity_type": "GENOMIC VARIANT", "description": "Another synonymous variant identified in the APOBR gene.<SEP>Located at the 1.3kb intron of APOBR gene.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS61738759"}, {"entity_type": "GENOMIC VARIANT", "description": "A synonymous variant present within the coding sequence of the APOBR gene.<SEP>Located close to rs368546180 with high linkage disequilibrium (LD) and associated with obesity.<SEP>This variant has been observed in multiple studies near or overlapping with rs149271, showing commonalities in association with obesity.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS40833"}, {"entity_type": "GENOMIC VARIANT", "description": "Identified as a variant in the un-translated region or intronic part of the APOBR gene.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS142786317"}, {"entity_type": "GENOMIC VARIANT", "description": "Mutates the amino acid sequence by introducing an 9bp deletion, disrupting three residues, and is located near rs3833080.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS368546180"}, {"entity_type": "GENOMIC VARIANT", "description": "Located in the coding region of APOBR gene but details not specified.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS200751685"}, {"entity_type": "GENOMIC VARIANT", "description": "Another variant associated with obesity, showing high LD with rs3833080 and rs149271.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS200743"}, {"entity_type": "GENOMIC VARIANT", "description": "Located in the 3\' untranslated region (UTR) of the APOBR gene, its function is unclear but associated with obesity.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS26905922"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant is found within an exon of the APOBR gene and has been linked to obesity.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS149782"}, {"entity_type": "GENOMIC VARIANT", "description": "A variant that resides in a CpG island close to the start codon of the APOBR gene, potentially influencing promoter activity.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS61365071"}, {"entity_type": "GENOMIC VARIANT", "description": "Located near rs368546180 and rs149271 on chromosome 1p34.1 with low minor allele frequency in lean individuals.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS40827"}, {"entity_type": "GENOMIC VARIANT", "description": "A synonymous variant found in the coding region of APOBR gene with high linkage disequilibrium (LD) with rs151233.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS1807642"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant is identified within an intron near exon 10 of the APOBR gene and has implications for splicing regulation.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS2993674"}, {"entity_type": "GENOMIC VARIANT", "description": "Located in a non-coding region adjacent to the APOBR gene with unclear functional relevance but related obesity risk.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS8111918"}, {"entity_type": "GENOMIC VARIANT", "description": "A synonymous variant within coding sequence of APOBR gene that is associated with high LD with rs151174.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS40832"}, {"entity_type": "GENOMIC VARIANT", "description": "Another variant near the 5\' UTR region of APOBR gene showing low minor allele frequency in lean subjects.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS61738760"}, {"entity_type": "GENOMIC VARIANT", "description": "Located close to the 3\' end of APOBR gene but specific details about its function are not specified.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS200751691"}, {"entity_type": "GENOMIC VARIANT", "description": "Another variant found within the APOBR gene\'s intronic region, potentially impacting gene regulation mechanisms.", "source_id": "chunk-7ca64f92964b051cfb1c43f967f243a5", "id": "RS61738762"}, {"entity_type": "GENOMIC VARIANT", "description": "The rs142786317 variant results in the deletion of the nucleotides CA at positions *118 and *119, leading to changes in transcription factor binding sites, splicing enhancer and silencer binding sites, or both.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS142786317: C.*118_*119DELCA"}, {"entity_type": "GENE", "description": "SULT1A1 is a gene involved in the detection of variants that may affect its function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "SULT1A1"}, {"entity_type": "TRANSCRIPT", "description": "Seven non-synonymous Single Nucleotide Polymorphisms (SNPs) were found in the coding region of SULT1A1, which could potentially change amino acids and influence protein function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "NON-SYNONYMOUS SNPS"}, {"entity_type": "PROTEIN", "description": "The SNP rs4149403 leads to an isoleucine (Ile) to threonine (Thr) substitution at position 7, which might affect protein stability and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS4149403: P.ILE7THR"}, {"entity_type": "PROTEIN", "description": "The SNP rs10797300 results in a proline (Pro) to leucine (Leu) substitution at position 19, potentially impacting protein structure and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS10797300: P.PRO19LEU"}, {"entity_type": "PROTEIN", "description": "The SNP rs145008170 leads to a serine (Ser) to aspartic acid (Asn) change at position 44, which might affect protein conformation and activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS145008170: P.SER44ASN"}, {"entity_type": "PROTEIN", "description": "The SNP rs4987024 results in a tyrosine (Tyr) to phenylalanine (Phe) substitution at position 62, potentially influencing protein stability and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS4987024: P.TYR62PHE"}, {"entity_type": "PROTEIN", "description": "The SNP rs142241142 leads to an alanine (Ala) to valine (Val) change at position 164, which could impact protein activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS142241142: P.ALA164VAL"}, {"entity_type": "PROTEIN", "description": "The SNP rs1059491 causes a asparagine (Asn) to threonine (Thr) substitution at position 235, potentially affecting protein structure and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS1059491: P.ASN235THR"}, {"entity_type": "PROTEIN", "description": "The SNP rs75191166 results in a lysine (Lys) to glycine (Gly) change at position 282, which might impact protein stability and activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS75191166: P.LYS282GLY"}, {"entity_type": "NON-CODING VARIANT", "description": "The variant rs4149406 alters the nucleotide composition (+34 relative to position 148) in the 3\' untranslated region.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS4149406: C.148+34T/C"}, {"entity_type": "NON-CODING VARIANT", "description": "The variant rs3743963 changes nucleotides at a position near to exon 2, affecting splice sites or transcription factor binding.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS3743963: C.500\u221219T/C"}, {"entity_type": "NON-CODING VARIANT", "description": "The variant rs710410 introduces a substitution (+7 relative to the first codon) impacting possible transcription factor binding or splicing enhancer sites.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS710410: C.*7T/C"}, {"entity_type": "NON-CODING VARIANT", "description": "The variant rs762634 alters the nucleotides near the start of an exon, which might influence splice sites or transcription factor binding.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS762634: C.*14A>G"}, {"entity_type": "NON-CODING VARIANT", "description": "The variant rs145790611 changes the nucleotides (+39 relative to exon 5) potentially impacting splicing or transcription factor binding.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS145790611: C.241+39G/A"}, {"entity_type": "UNKNOWN VARIANT", "description": "An unknown variant detected in the 3\' untranslated region of TUFM, previously not documented.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "C.3536C>G"}, {"entity_type": "INTRONIC VARIANT", "description": "The variant rs7187776 introduces a substitution near -55 relative to exon 2\'s beginning, potentially affecting splicing or transcription factor binding.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS7187776: C.-55T>C"}, {"entity_type": "INTRONIC VARIANT", "description": "The variant rs4788099 results in a substitution near +13 relative to exon 6\'s start, potentially impacting splicing or transcription factor binding.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS4788099: C.817+13T>C"}, {"entity_type": "INTRONIC VARIANT", "description": "The variant rs8061877 changes nucleotides at position -18 relative to exon 3\'s end, potentially affecting transcription factor binding.<SEP>The variant rs8061877 results in a substitution at position -18 relative to exon 3, potentially affecting transcription factor binding.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS8061877: C.248\u221218G>A"}, {"entity_type": "INTRONIC VARIANT", "description": "The variant rs61737565 introduces a substitution near +29 relative to exon 8\'s start, possibly affecting transcription factor binding or splicing sites.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS61737565: C.922+29C>G"}, {"entity_type": "GENOMIC VARIANT", "description": "An unknown variant introduced by g.28854194C/G, not previously documented.)(entity<SEP>The genomic variant g.28854194C/G causes changes in splice enhancer binding sites and splicing silencer sites as predicted by three programs (S1 Table).", "source_id": "chunk-607feda7df4db8151b99560074c91eea<SEP>chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "G.28854194C/G"}, {"entity_type": "PROTEIN", "description": "The SNP rs228540 results in a proline (Pro) to alanine (Ala) substitution at position 116, which might affect protein stability and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS228540: P.PRO116ALA"}, {"entity_type": "PROTEIN", "description": "The SNP rs739728 leads to a threonine (Thr) to isoleucine (Ile) substitution at position 162, which could impact protein conformation and activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS739728: P.THR162ILE"}, {"entity_type": "PROTEIN", "description": "The SNP rs141420 results in a serine (Ser) to glycine (Gly) substitution at position 19, potentially influencing protein stability and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS141420: P.SER19GLY"}, {"entity_type": "PROTEIN", "description": "The SNP rs157783 leads to a leucine (Leu) to methionine (Met) change at position 26, which could affect protein activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS157783: P.LEU26MET"}, {"entity_type": "PROTEIN", "description": "The SNP rs49170 results in an glutamic acid (Glu) to glycine (Gly) substitution at position 108, potentially impacting protein conformation and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS49170: P.GLU108GLY"}, {"entity_type": "PROTEIN", "description": "The SNP rs236589 causes an alanine (Ala) to threonine (Thr) substitution at position 31, potentially affecting protein structure and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS236589: P.ALA31THR"}, {"entity_type": "PROTEIN", "description": "The SNP rs76301 results in a glutamic acid (Glu) to glycine (Gly) substitution at position 67, which might impact protein stability and activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS76301: P.GLU67GLY"}, {"entity_type": "PROTEIN", "description": "The SNP rs80492 leads to a proline (Pro) to leucine (Leu) change at position 85, potentially influencing protein structure and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS80492: P.PRO85LEU"}, {"entity_type": "PROTEIN", "description": "The SNP rs73681 causes a tyrosine (Tyr) to cysteine (Cys) substitution at position 142, which could affect protein stability and activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS73681: P.TYR142CYS"}, {"entity_type": "PROTEIN", "description": "The SNP rs695 results in a serine (Ser) to phenylalanine (Phe) substitution at position 27, potentially impacting protein conformation and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS695: P.SER27PHE"}, {"entity_type": "PROTEIN", "description": "The SNP rs14306 leads to a leucine (Leu) to methionine (Met) change at position 38, which could affect protein activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS14306: P.LEU38MET"}, {"entity_type": "PROTEIN", "description": "The SNP rs75981 results in an alanine (Ala) to threonine (Thr) substitution at position 52, potentially influencing protein structure and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS75981: P.ALA52THR"}, {"entity_type": "PROTEIN", "description": "The SNP rs66340 leads to a proline (Pro) to serine (Ser) change at position 60, which could impact protein stability and activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS66340: P.PRO60SER"}, {"entity_type": "PROTEIN", "description": "The SNP rs75982 results in a leucine (Leu) to glutamine (Gln) substitution at position 61, potentially affecting protein conformation and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS75982: P.LEU61GLN"}, {"entity_type": "PROTEIN", "description": "The SNP rs640 leads to a valine (Val) to isoleucine (Ile) change at position 71, which could influence protein activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS640: P.VAL71ILE"}, {"entity_type": "PROTEIN", "description": "The SNP rs5238 results in a tyrosine (Tyr) to cysteine (Cys) substitution at position 98, potentially impacting protein stability and activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS5238: P.TYR98CYS"}, {"entity_type": "PROTEIN", "description": "The SNP rs764 leads to a serine (Ser) to phenylalanine (Phe) change at position 100, which could affect protein conformation and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS764: P.SER100PHE"}, {"entity_type": "PROTEIN", "description": "The SNP rs23659 causes a proline (Pro) to alanine (Ala) substitution at position 102, potentially affecting protein stability and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS23659: P.PRO102ALA"}, {"entity_type": "PROTEIN", "description": "The SNP rs740 results in a leucine (Leu) to glutamine (Gln) substitution at position 113, which might impact protein conformation and activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS740: P.LEU113GLN"}, {"entity_type": "PROTEIN", "description": "The SNP rs665 leads to a threonine (Thr) to isoleucine (Ile) change at position 123, potentially influencing protein structure and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS665: P.THR123ILE"}, {"entity_type": "PROTEIN", "description": "The SNP rs742 results in a phenylalanine (Phe) to glutamic acid (Glu) substitution at position 129, potentially impacting protein conformation and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS742: P.PHE129GLU"}, {"entity_type": "PROTEIN", "description": "The SNP rs653 leads to a tyrosine (Tyr) to serine (Ser) change at position 130, which could affect protein stability and activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS653: P.TYR130SER"}, {"entity_type": "PROTEIN", "description": "The SNP rs743 results in a leucine (Leu) to glycine (Gly) substitution at position 132, potentially affecting protein conformation and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS743: P.LEU132GLY"}, {"entity_type": "PROTEIN", "description": "The SNP rs649 leads to a valine (Val) to methionine (Met) change at position 140, which could influence protein activity.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS649: P.VAL140MET"}, {"entity_type": "PROTEIN", "description": "The SNP rs738 causes a proline (Pro) to leucine (Leu) substitution at position 152, potentially affecting protein structure and function.", "source_id": "chunk-607feda7df4db8151b99560074c91eea", "id": "RS738: P.PRO152LEU"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs61737565 is located at position c.922+29C>G in a TUFM gene.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "RS61737565"}, {"entity_type": "GENE", "description": "Five variants are found in the TUFM gene, which may affect splicing regulator or transcription factor binding sites.<SEP>The TUFM gene includes the non-coding 3\u2019 UTR region which contains a genomic variant g.28854194C/G.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "TUFM"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs8061877 disrupts a transcription factor binding site for SRY, HFH 2, and HMG-IY (TFSearch and Consite).", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "RS8061877"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Splicing regulators are involved in altering splicing sites due to specific genomic variants like rs61737565.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "SPLICING REGULATOR"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Transcription factors can be affected by certain genomic variants leading to altered binding sites such as rs8061877.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "TRANSCRIPTION FACTOR BINDING SITE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Changes in splice enhancer sites are caused by specific genomic variants like g.28854194C/G within a gene.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "SPLICE ENHANCER BINDING SITES"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Splicing silencer sites can be altered due to the presence of certain genomic variants, as seen with rs61737565 and g.28854194C/G in TUFM.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "SPLICING SILENCER SITES"}, {"entity_type": "GENE", "description": "Although it is non-coding, the 3\u2019 UTR can still influence gene expression, which is impacted by genomic variants such as g.28854194C/G within TUFM.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "CODING REGION"}, {"entity_type": "GENE", "description": "The presence of specific genomic variants can affect non-coding parts of the gene like the 3\u2019 UTR, which plays a role in regulating gene expression.", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "id": "NON-CODING REGIONS"}, {"source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "description": "The genomic variant rs61737565 affects the TUFM gene by altering splicing regulator or transcription factor binding sites.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT RS61737565"}, {"source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "description": "The genomic variant rs61737565 affects the TUFM gene by altering splicing regulator or transcription factor binding sites.)", "entity_type": "UNKNOWN", "id": "TUFM GENE"}, {"source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "description": "The genomic variant rs8061877 disrupts the binding of transcription factors SRY, HFH 2, and HMG-IY in the TUFM gene.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT RS8061877"}, {"source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "description": "The genomic variant g.28854194C/G causes changes in splice enhancer and splicing silencer sites within the TUFM gene.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT G.28854194C/G"}, {"source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "description": "The non-coding 3\u2019 UTR region of TUFM contains genomic variant g.28854194C/G which affects the gene function.)", "entity_type": "UNKNOWN", "id": "NON-CODING 3\u2019 UTR OF TUFM"}, {"source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "description": "Disruption of transcription factor binding sites occurs due to rs8061877 variant, influencing gene regulation.)", "entity_type": "UNKNOWN", "id": "TRANSCRIPTION FACTOR BINDING SITES"}, {"source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "description": "g.28854194C/G affects splice sites via changes in enhancer and silencer regions.)", "entity_type": "UNKNOWN", "id": "SPLICE ENHANCER AND SPLICING SILENCER SITES"}, {"source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "description": "The non-coding 3\u2019 UTR can impact gene expression independently or through interactions with genomic variants like g.28854194C/G.)", "entity_type": "UNKNOWN", "id": "GENE EXPRESSION REGULATION"}, {"entity_type": "GENE", "description": "ALOX15, a gene involved in arachidonic acid metabolism and immune response, was not significantly associated with ovarian cancer risk in the combined analysis.<SEP>Analysis of 162 successfully genotyped SNPs in approximately 900 epithelial ovarian cancer cases and 1,000 controls.<SEP>The ALOX15 gene encodes an arachidonate 5-lipoxygenase, which is involved in various physiological processes related to inflammation and metabolism.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b<SEP>chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "id": "ALOX15"}, {"entity_type": "GENE", "description": "Association between rs1864414 and ovarian cancer was confirmed in combined analysis.<SEP>Association with risk of ovarian cancer found for rs1864414 (OR 0.77, p=0.004) and rs743507 (OR 0.79, p=0.003).", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "ALOX5"}, {"entity_type": "GENE", "description": "Included in analysis of 21 SNPs associated with risk of ovarian cancer.<SEP>The IL18 gene encodes interleukin 1 alpha, which plays a role in the inflammatory response and cancer development.<SEP>The IL18 gene encoding interleukin 1 alpha plays a role in the inflammatory response but showed null or inconsistent associations with ovarian cancer based on the phase of studies.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b<SEP>chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "id": "IL18"}, {"entity_type": "GENE", "description": "Suggests associations with decreased risk of ovarian cancer (OR 0.90, p=0.005).", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL1A"}, {"entity_type": "GENE", "description": "NOS3 is one of the genes being studied for its role in folate metabolism and its association with childhood ALL.<SEP>NOS3 rs3918186 is a SNP in the NOS3 gene that was analyzed for its association with childhood acute lymphoblastic leukemia (ALL).<SEP>rs743507 (OR 0.79, p=0.003) associated with risk of ovarian cancer.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b<SEP>chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "NOS3"}, {"entity_type": "GENE", "description": "Association suggested for rs2282169 in combined analyses.<SEP>The PTGS1 gene encodes prostaglandin-endoperoxide synthase 1.<SEP>The PTGS1 gene was associated with ovarian cancer risk through its variant rs2282169, suggesting a potential link between prostaglandin-endoperoxide synthase 1 and ovarian cancer predisposition.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b<SEP>chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "id": "PTGS1"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer (p\u2019s > 0.10).", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "ALOX12"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "CCL11"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer (p\u2019s > 0.10).", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "CCL3"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer (p\u2019s > 0.10).", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "CXCL16"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer (p\u2019s > 0.10).", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL10"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL15RA"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer (p\u2019s > 0.10).", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL1RN"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL4R"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer (p\u2019s > 0.10).", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL7R"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL8RA"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer (p\u2019s > 0.10).", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL8RB"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer (p\u2019s > 0.10).", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "IL9"}, {"entity_type": "GENE", "description": "No association was observed in analysis of SNPs in this gene.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "PTGS2"}, {"entity_type": "GENE", "description": "Analysis within this gene did not suggest associations between genotype and risk of ovarian cancer (p\u2019s > 0.10).", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "TLR2"}, {"entity_type": "GENE", "description": "No association was observed in analysis of SNPs in this gene (p\u2019s > 0.10).", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "TNF"}, {"entity_type": "GENE", "description": "Variants in BRCA1 have been shown to increase the risk of ovarian cancer.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "BRCA1"}, {"entity_type": "GENE", "description": "SNPs in BRCA2 are associated with an increased risk of ovarian cancer, especially in patients with a family history of the disease.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "BRCA2"}, {"entity_type": "GENE", "description": "Mutations or variants in PTEN have been linked to a higher risk of ovarian cancer, particularly among BRCA1 carriers.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "PTEN"}, {"entity_type": "GENE", "description": "SNPs in SMAD4 have been implicated as potential genetic markers for the predisposition to ovarian cancer.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "SMAD4"}, {"entity_type": "GENE", "description": "Mutations or variants in CDH1 have been found to increase the likelihood of ovarian cancer, particularly when present alongside other genetic markers like BRCA2 mutations.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "CDH1"}, {"entity_type": "GENE", "description": "Variants in BARD1 have been associated with a higher risk of developing ovarian cancer, especially among women carrying BRCA2 mutations.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "BARD1"}, {"entity_type": "GENE", "description": "Polymorphisms in CHEK2 have been linked to an elevated risk for developing ovarian cancer, especially among women with inherited BRCA1 mutations.", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "id": "CHEK2"}, {"entity_type": "GENE", "description": "IL1B is a gene that encodes interleukin 1, beta.", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "id": "IL1B"}, {"entity_type": "GENOMIC VARIANT", "description": "The genomic variant rs7596684 in IL1B gene may be associated with increased risk of ovarian cancer, based on some preliminary findings.<SEP>The genomic variant rs7596684 in the IL1B gene has been associated with ovarian cancer risk.<SEP>The genomic variant rs7596684 within IL1B gene shows a potential association with ovarian cancer, which was partly identified in the text.<SEP>The genomic variant rs7596684, located in the IL1B gene, was found to have a significant association with ovarian cancer risk.", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "id": "RS7596684"}, {"entity_type": "GENOMIC VARIANT", "description": "Although rs2282169 in PTGS1 has been associated with ovarian cancer, this information was not directly mentioned; it is an implication of the initial genomic variant\'s significance.<SEP>The association between rs2282169 variant in PTGS1 gene and ovarian cancer risk is a partially identified linkage that was mentioned only implicitly.<SEP>The genomic variant rs2282169 in the PTGS1 gene has been suggested to associate with ovarian cancer risk.", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "id": "RS2282169"}, {"entity_type": "GENE", "description": "The CCL2 gene codes for a chemokine that plays roles in immune response, inflammation, and cancer progression.<SEP>The CCL2 gene encoding chemokine, which is involved in immune response and inflammation, showed null or inconsistent results for its association with ovarian cancer across different study phases.", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "id": "CCL2"}, {"entity_type": "GENE", "description": "Interleukin 6 (IL6), an important cytokine that influences inflammation, was not found to have a consistent or significant association with ovarian cancer in the combined analyses.<SEP>The IL6 gene codes for interleukin-6, an important cytokine involved in the immune response and inflammation that can influence cancer progression.", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 29}]", "id": "IL6"}, {"entity_type": "DISEASE", "description": "A disease under investigation related to the genetic variants and their association with cancer risk.<SEP>Ovarian cancer is a type of malignancy that affects the ovaries and has multiple risk factors including genetic predispositions, age, and environmental exposures like smoking or diet.", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c<SEP>chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "id": "OVARIAN CANCER"}, {"source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "description": "Significant associations were present in clear cell, mucinous, and endometrioid subtypes, suggesting different risk profiles.)(entity", "entity_type": "UNKNOWN", "id": "HETEROGENEITY FOR CLEAR CELL, MUCINOUS, AND ENDOMETRIOID SUBTYPES (IL1A RS17561)"}, {"source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "description": "Significant associations were present in clear cell, mucinous, and endometrioid subtypes, suggesting different risk profiles.)(entity", "entity_type": "UNKNOWN", "id": "P=0.03"}, {"source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "description": "No strong evidence suggests that ALOX15 plays a significant role in increasing the risk of ovarian cancer as per the study phases.)", "entity_type": "UNKNOWN", "id": "LACK OF ASSOCIATION"}, {"source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "description": "Results on the relationship between CCL2 and ovarian cancer risk were mixed throughout different study phases, indicating uncertain associations.)", "entity_type": "UNKNOWN", "id": "STUDY PHASES"}, {"source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "description": "The relationship between CRP and ovarian cancer risk was ambiguous due to inconsistent findings across study phases.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 29}]", "id": "LACK OF CONSISTENT ASSOCIATION"}, {"source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "description": "The relationship between IL18 and ovarian cancer risk was unclear due to varying results across different stages of research phases.)", "entity_type": "UNKNOWN", "id": "PHASE OF STUDIES"}, {"source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "description": "The variant rs2282169 in PTGS1 gene appears to be linked with ovarian cancer risk, although this was not explicitly stated in the text.)", "entity_type": "UNKNOWN", "id": "RS2282169& OVARIAN CANCER RISK"}, {"source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "description": "A possible link exists between rs7596684 variant within IL1B gene and increased risk of ovarian cancer based on some findings, though this was not fully elucidated.)", "entity_type": "UNKNOWN", "id": "RS7596684&OVARIAN CANCER RISK"}, {"source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "description": "A potential connection exists between rs2282169 variant in PTGS1 gene and ovarian cancer, though this was not extensively detailed.)", "entity_type": "UNKNOWN", "id": "RS2282169&POTENTIAL OVARIAN CANCER LINKAGE"}, {"source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "description": "A preliminary linkage between rs7596684 variant in IL1B gene and potential increase in risk of ovarian cancer is suggested by some observations, though these were not thoroughly explained.)", "entity_type": "UNKNOWN", "id": "RS7596684&POTENTIAL INCREASED OVARIAN CANCER RISK"}, {"entity_type": "GENE", "description": "MTHFD2 gene plays a role in the folate metabolic pathway.<SEP>MTHFD2 genes were among the folate metabolic genes included in the analysis.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "MTHFD2"}, {"entity_type": "GENE", "description": "BHMT2 gene also contributes to the folate metabolic pathway analyzed for childhood ALL risk association.<SEP>BHMT2 genes were among the folate metabolic genes included in the analysis.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "BHMT2"}, {"entity_type": "GENE", "description": "SLC19A1 gene was among the folate metabolic genes with a tagging SNP analyzed for association with childhood ALL.<SEP>SLC19A1 is another gene involved in the folate metabolic pathways and was evaluated in relation to childhood ALL.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "SLC19A1"}, {"entity_type": "SNP", "description": "NOS3 rs3918186 had a Bayes Factor of 7.38, indicating it was significantly associated with childhood ALL risk.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "RS3918186"}, {"entity_type": "SNP", "description": "For each copy of the minor allele of SLC19A1 rs1051266, there was a 2.07 times risk of developing childhood ALL.<SEP>SLC19A1 rs1051266 showed less evidence for association with childhood ALL compared to NOS3 rs3918186, as its Bayes Factor was lower.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "RS1051266"}, {"entity_type": "METABOLIC PATHWAY", "description": "The study focused on tagging SNPs of 16 folate metabolic genes for associations with childhood acute lymphoblastic leukemia (ALL).", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "FOLATE METABOLIC PATHWAY"}, {"entity_type": "DISEASE", "description": "The research aimed to identify genetic factors, including SNPs in folate metabolism-related genes, associated with the risk of developing childhood ALL.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "CHILDHOOD ACUTE LYMPHOBLASTIC LEUKEMIA"}, {"entity_type": "DATA TABLE", "description": "This table summarizes the population characteristics of childhood ALL cases, including gender, participation rate, triad completeness, and racial/ethnic group distribution.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "TABLE 2"}, {"entity_type": "DATA TABLE", "description": "Table 3 includes information on the initial screening of SNPs from folate metabolic genes like MTHFD2, BHMT2, NOS3, and SLC19A1.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "TABLE 3"}, {"entity_type": "DATA TABLE", "description": "Table 4 presents the results of the final analysis, including Bayes Factors for SNPs associated with childhood ALL risk.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "TABLE 4"}, {"entity_type": "STATISTICAL MEASURE", "description": "The odds ratio is used to quantify the strength of association between specific SNPs and the risk of developing childhood ALL based on the data.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "OR (ODDS RATIO)"}, {"entity_type": "STATISTICAL INTERVAL", "description": "A 95% confidence interval provides a range within which we can be confident that the true value lies for statistical measures such as odds ratios or Bayes Factors.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "95% CI (CONFIDENCE INTERVAL)"}, {"entity_type": "STATISTICAL MEASURE", "description": "This measure reflects the updated probability of an SNP being included in the model after considering all available evidence and data.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "POSTERIOR PROBABILITY"}, {"entity_type": "DISEASE", "description": "The initial screening process involved analyzing tagging SNPs from folate metabolic genes to identify those with a high posterior probability for further examination.<SEP>There were no group differences in initial screening or randomization.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e<SEP>chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "INITIAL SCREENING"}, {"entity_type": "STUDY PROCESS", "description": "The final analysis consisted of more in-depth evaluation of selected SNPs that passed the initial screening stage based on their statistical significance and relevance.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "FINAL ANALYSIS"}, {"entity_type": "GENOMIC VARIANT", "description": "SNPs are variations in a single DNA building block at specific positions in the genome, which were analyzed for associations with childhood ALL risk.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "SNP (SINGLE NUCLEOTIDE POLYMORPHISM)"}, {"entity_type": "STATISTICAL MEASURE", "description": "Prior odds represent the likelihood of an event occurring before considering any new evidence or data.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "PRIOR ODDS"}, {"entity_type": "STATISTICAL TERM", "description": "In this context, association refers to the relationship between specific SNPs in folate metabolic genes and the risk of developing childhood ALL based on statistical analysis.", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "id": "ASSOCATION"}, {"source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "description": "NOS3 rs3918186 showed a significant association with an increased risk of childhood ALL based on Bayes Factors and odds ratios.)", "entity_type": "UNKNOWN", "id": "NOS3 RS3918186"}, {"source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "description": "While SLC19A1 rs1051266 had some evidence for association, the strength was weaker than that of NOS3 rs3918186.)", "entity_type": "UNKNOWN", "id": "SLC19A1 RS1051266"}, {"source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "description": "The data from Table 2 were used as criteria for selecting which SNPs would proceed to initial screening based on their relevance and completeness.)", "entity_type": "UNKNOWN", "id": "INITIAL SCREENING CRITERIA"}, {"source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "description": "The data from Table 2 were used as criteria for selecting which SNPs would proceed to initial screening based on their relevance and completeness.)", "entity_type": "UNKNOWN", "id": "TABLE 2 POPULATION CHARACTERISTICS"}, {"source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "description": "SNPs initially identified in the study underwent an initial screening process, where those with a high posterior probability were selected for further analysis.)", "entity_type": "UNKNOWN", "id": "INITIAL SCREENING PROCESS"}, {"source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "description": "SNPs initially identified in the study underwent an initial screening process, where those with a high posterior probability were selected for further analysis.)", "entity_type": "UNKNOWN", "id": "SNP INITIAL IDENTIFICATION"}, {"entity_type": "GENETIC HAPLOTYPE", "description": "A set of genetic variations located together on a chromosome that are usually inherited together.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "FADS1/2 HAPLOTYPE"}, {"entity_type": "FREQUENCY OF MINOR ALLELE", "description": "The proportion of times the less common form (minor allele) is found at a particular genetic locus in a population.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "MINOR ALLELE FREQUENCY"}, {"entity_type": "CATEGORICAL DEMOGRAPHIC VARIABLE", "description": "Identifying variable for different racial groups.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "RACE"}, {"entity_type": "LEVELS OF ALPHA-LINOLENIC ACID", "description": "The level of alpha-linolenic acid (ALA) in the blood serum or colon tissue.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "AA CONCENTRATION"}, {"entity_type": "EICOSAPENTAENOIC ACID", "description": "A long-chain omega-3 fatty acid found primarily in oily fish and krill oil.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "EPA"}, {"entity_type": "LONG CHAIN OMEGA-3 FATTY ACIDS", "description": "Fatty acids that are longer than 18 carbon atoms, with an odd number of carbon atoms greater than or equal to 20, and have three double bonds in the trans configuration.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "LONG CHAIN N-3 FATTY ACIDS"}, {"entity_type": "ALPHA LINOLENIC ACID INTAKE", "description": "The amount of alpha-linolenic acid (ALA) included in the diet.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "DIETARY AA INTAKE"}, {"entity_type": "INTAKE OF LONG CHAIN OMEGA 3 FATTY ACIDS", "description": "The amount of long-chain omega-3 fatty acids, specifically EPA and DHA, consumed through diet.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "LONG CHAIN N-3 FATTY ACIDS INTAKE"}, {"entity_type": "NUTRITIONAL INTERVENTION PROGRAM", "description": "A structured program designed to modify dietary habits through education or supplementation to improve health outcomes.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "DIETARY INTERVENTION"}, {"entity_type": "ALPHA LINOLENIC ACID CONCENTRATION IN BLOOD SERUM", "description": "The level of alpha-linolenic acid present in blood serum.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "SERUM AA CONCENTRATION"}, {"entity_type": "FATTY ACIDS LEVELS IN COLON", "description": "Changes observed in colon fatty acid concentrations due to diet<SEP>Levels of fatty acids found within the colon tissue, specifically 20:4n-6, 18:3n-3 and 20:4n-6 concentration.", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "COLON FATTY ACID CONCENTRATIONS"}, {"entity_type": "TOTAL CHOLESTEROL LEVEL", "description": "The total amount of cholesterol in the blood, which includes both high-density lipoprotein (HDL) and low-density lipoprotein (LDL) cholesterol levels.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "TOTAL CHOLESTEROL"}, {"entity_type": "LOW-DENSITY LIPOPROTEIN (LDL)", "description": "A type of lipoprotein that transports cholesterol from the liver to cells throughout the body, often referred to as \'bad\' cholesterol due to its potential role in heart disease.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "LOW DENSITY LIPOPROTEINS"}, {"entity_type": "HIGH-DENSITY LIPOPROTEIN (HDL)", "description": "A type of lipoprotein that helps remove cholesterol from cells and transport it back to the liver, often referred to as \'good\' cholesterol.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "HIGH DENSITY LIPOPROTEINS"}, {"entity_type": "C-REACTIVE PROTEIN", "description": "A protein found in the bloodstream that increases during inflammation and infection. Elevated CRP levels may indicate an increased risk of heart disease or other inflammatory conditions.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "CRP (C-REACTIVE PROTEIN)"}, {"entity_type": "COLON FATTY ACID PROFILE", "description": "The types and amounts of fatty acids present in colon tissue.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "FATTY ACID COMPOSITION"}, {"entity_type": "LEVELS OF N-3 FATTY ACIDS IN THE RED BLOOD CELL MEMBRANE PHOSPHOLIPIDS", "description": "The ratio of EPA (Eicosapentaenoic acid) + DHA (Docosahexaenoic acid) to total n-6 and n-3 fatty acids present in red blood cells.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "OMEGA-3 INDEX"}, {"entity_type": "FA/TLC1 PROTEIN EXPRESSION", "description": "The level of expression of the Fatty Acid Translocase 1 protein, which is involved in transport of long chain fatty acids into cells.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "FATTY ACID TRANSLOCASE 1"}, {"entity_type": "EFFICIENCY OF EPA SYNTHESIS", "description": "The proportionate ability to convert dietary EPA into longer-chain n-3 fatty acids like DHA.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "EPA CONVERSION EFFICIENCY"}, {"entity_type": "FIBER INTAKE", "description": "The total amount of dietary fiber consumed in the diet.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "DIETARY FIBER INTAKE"}, {"entity_type": "NUTRIENT", "description": "Lower calcium intake was found in cases compared to controls, potentially affecting nutritional status.<SEP>The level or amount of calcium taken from food sources.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6<SEP>chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "CALCIUM INTAKE"}, {"entity_type": "SE INTAKE", "description": "The dietary intake of selenium, an essential mineral for human health.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "SELENIUM INTAKE"}, {"entity_type": "VC INTAKE", "description": "The consumption of Vitamin C through dietary sources.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "VITAMIN C INTAKE"}, {"entity_type": "ADHERENCE TO MEDITERRANEAN DIET", "description": "The degree to which a person follows the principles and components of the traditional Mediterranean diet.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "MEDITERRANEAN DIET ADHERENCE"}, {"entity_type": "PHYSICAL ACTIVITY LEVEL", "description": "The frequency, intensity, duration, and type of regular physical activity performed by an individual.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "REGULAR EXERCISE"}, {"entity_type": "CAFFEINE CONSUMPTION", "description": "The amount of caffeine consumed through coffee, tea or other sources.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "COFFEE INTAKE"}, {"entity_type": "PHARMACOLOGICAL USE", "description": "The prescription drugs or supplements being taken by an individual on a regular basis.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "MEDICATION USE"}, {"entity_type": "PSYCHOLOGICAL STRESS LEVEL", "description": "The subjective and objective measures of stress experienced by individuals.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "STRESS LEVELS"}, {"entity_type": "QUALITY OF SLEEP", "description": "The adequacy, efficiency, continuity, and restfulness of sleep in an individual.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "SLEEP QUALITY"}, {"entity_type": "MARKERS OF INFLAMMATION", "description": "Levels or presence of various substances in the blood that indicate an inflammatory response within the body.", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "id": "INFLAMMATORY MARKERS"}, {"entity_type": "NUTRIENT", "description": "A type of polyunsaturated fatty acid that significantly decreased in serum", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "20:4, N-6 (AA)"}, {"entity_type": "NUTRIENT", "description": "Monounsaturated fatty acids that increased significantly in both groups", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "MUFA"}, {"entity_type": "NUTRIENT", "description": "A type of polyunsaturated fatty acid that significantly increased in both groups", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "N-3 PUFA"}, {"entity_type": "NUTRIENT", "description": "A type of polyunsaturated fatty acid that significantly decreased in serum", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "SERUM 18:2 N-6"}, {"entity_type": "DIETARY PATTERN", "description": "A diet with a focus on fruits, vegetables, whole grains, and lean proteins that was compared to the Mediterranean group", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "id": "HEALTHY EATING GROUP"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The relationship between the presence of a specific SNP and various baseline characteristics (gender, race, age, BMI).)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 42}]", "id": "BASELINE CHARACTERISTICS"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The correlation between successful genotyping and minor allele frequency in SNPs.)", "entity_type": "UNKNOWN", "id": "GENOTYPING SUCCESS RATE"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The correlation between successful genotyping and minor allele frequency in SNPs.)", "entity_type": "UNKNOWN", "id": "MINOR ALLELE FREQUENCIES"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The predictive relationship of dietary intake on serum levels of specific fatty acids (AA, EPA).)", "entity_type": "UNKNOWN", "id": "DIETARY INTAKES"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The predictive relationship of dietary intake on serum levels of specific fatty acids (AA, EPA).)", "entity_type": "UNKNOWN", "id": "SERUM FATTY ACID CONCENTRATIONS"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "The association between genotype and BMI in the baseline demographics.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 34}]", "id": "GENOTYPE GROUP"}, {"source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "description": "How changes in dietary intake during an intervention affect serum levels of various fatty acids.)", "entity_type": "UNKNOWN", "id": "DIETARY INTERVENTION EFFECTS"}, {"source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "description": "Mean colon AA concentrations differed significantly between diet arms at 6 months for subjects without minor alleles in the FADS1/2 gene cluster.)", "entity_type": "UNKNOWN", "id": "COLON AA CONCENTRATIONS"}, {"source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "description": "Mean colon AA concentrations differed significantly between diet arms at 6 months for subjects without minor alleles in the FADS1/2 gene cluster.)", "entity_type": "UNKNOWN", "id": "DIET ARM"}, {"source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "description": "There was a significant interaction of genotype by diet for AA concentrations in the colon.)", "entity_type": "UNKNOWN", "id": "COLON CONCENTRATIONS"}, {"source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "description": "No significant genotype-by-diet interactions were found for EPA concentration.)", "entity_type": "UNKNOWN", "id": "NO GENOTYPE-DIET INTERACTION"}, {"source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "description": "Serum 20:4, n-6 (AA) concentration significantly decreased in the Mediterranean group.)", "entity_type": "UNKNOWN", "id": "MEDITERRANEAN GROUP"}, {"source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "description": "Mean colon AA concentrations were estimated to be 16% (95% CI = [5%, 26%]) lower for the Healthy Eating arm than the Mediterranean arm at 6 months.)", "entity_type": "UNKNOWN", "id": "NO MINOR ALLELES"}, {"entity_type": "POPULATION-BASED DISCOVERY SAMPLE", "description": "408,317 individuals of white British ancestry were included in this study.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "UK BIOBANK"}, {"entity_type": "OUTCOME MEASURE", "description": "This population-based study used self-reported snoring as the main outcome measure based on partner or housemate complaints.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 33}]", "id": "SNORING"}, {"entity_type": "COMORBID CONDITION", "description": "The prevalence of sleep apnoea was higher among those reporting snoring.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "SLEEP APNOEA"}, {"entity_type": "RISK FACTOR", "description": "A lower SES was linked to an increased risk of snoring in males only.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "SOCIOECONOMIC STATUS (SES)"}, {"entity_type": "RISK FACTOR", "description": "Smoking frequency showed a positive correlation with snoring prevalence, particularly in females and to some extent in males.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "clusters": "[{level: 0, cluster: 26}]", "id": "SMOKING FREQUENCY"}, {"entity_type": "RISK FACTOR", "description": "Alcohol consumption was positively correlated with snoring, though this association was weaker compared to smoking.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "ALCOHOL CONSUMPTION FREQUENCY"}, {"entity_type": "RISK FACTOR", "description": "There was a correlation between whole-body fat mass and snoring prevalence, indicating obesity as a risk factor.<SEP>Whole-body fat mass was another trait with a moderate genetic correlation to snoring, related to obesity.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 34}]", "id": "WHOLE-BODY FAT MASS"}, {"entity_type": "STATISTICAL METHOD", "description": "A GWAS was conducted on 408,317 individuals to identify genetic variants related to snoring.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "GWAS (GENOME-WIDE ASSOCIATION STUDY)"}, {"entity_type": "MEASURE OF GENETIC CONTRIBUTION", "description": "The SNP heritability for snoring was estimated at 9.9% (SE=0.39%) indicating the genetic influence on the trait.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "clusters": "[{level: 0, cluster: 10}]", "id": "SNP HERITABILITY"}, {"entity_type": "ASSOCIATION MEASURE", "description": "GWAS results showed specific genetic correlations between snoring and various traits beyond sleep apnea, including measures of BMI, heart disease, lung function, psychiatric conditions, and more.<SEP>Sleep apnoea showed a high genetic correlation of 0.78 with snoring, suggesting shared genetic factors between the two conditions.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "GENETIC CORRELATION"}, {"entity_type": "CORRELATED TRAIT", "description": "Alcohol intake frequency had a weak positive genetic correlation with snoring, indicating potential influence.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "ALCOHOL INTAKE FREQUENCY"}, {"entity_type": "CORRELATED TRAITS", "description": "Traits such as sleep apnoea and excessive daytime sleepiness showed moderate to high genetic correlations with snoring, reflecting common underlying mechanisms.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "SLEEP DISORDERS"}, {"entity_type": "CORRELATED TRAIT", "description": "The trait of neuroticism was negatively correlated with snoring, possibly indicating a protective effect against snoring.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "NEUROTICISM"}, {"entity_type": "CORRELATED TRAIT", "description": "There was no genetic correlation between heart rate and snoring in this study, suggesting no shared genetic basis.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "HEART RATE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Partners or housemates\' complaints were used as an indicator for snoring in this study.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "PARTNER"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Complaints from housemates were also considered when assessing snoring prevalence.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "HOUSEMATE"}, {"entity_type": "STATISTICAL METHODOLOGY COMPONENT", "description": "The Genome-Wide Association Study adjusted for basic covariates such as BMI, sex, age, and other risk factors like smoking and alcohol consumption when analyzing genetic associations with snoring.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "GWAS ANALYSIS"}, {"entity_type": "SUPPLEMENTARY METHODOLOGY COMPONENT", "description": "This analysis was performed to examine the impact of adjusting for BMI on associations with previously identified genetic variants related to snoring.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "FOLLOW-UP SENSITIVITY ANALYSIS"}, {"entity_type": "IMPACT OF METHODOLOGICAL INTERVENTION", "description": "Adjusting for BMI changed the signal at some genetic loci associated with snoring, highlighting potential confounding effects in GWAS analysis.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "BMI ADJUSTMENT EFFECTS"}, {"entity_type": "OUTCOME OF METHODOLOGICAL INTERVENTION", "description": "After adjusting for BMI, the genetic correlation between snoring and traits like sleep apnea showed small increases, suggesting that confounding by BMI was reduced.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "ADJUSTED GENETIC CORRELATIONS"}, {"entity_type": "METHODOLOGICAL COMPARISON OUTCOME", "description": "Comparing the unadjusted to adjusted GWAS revealed a high degree of similarity in results, supporting the robustness of findings despite accounting for BMI.", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "id": "UNADJUSTED VS. ADJUSTED GWAS"}, {"entity_type": "GENE", "description": "13 chromosome gene associated with snoring and nocturnal hypoxia.<SEP>Closely linked with MSRB3 in genome-wide significant SNPs; 13 chromosome gene.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "DLEU7"}, {"entity_type": "GENE", "description": "12 chromosome gene associated with snoring and nocturnal hypoxia.<SEP>Another 12 chromosome gene identified from positional and eQTL mapping related to snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "MSRB3"}, {"entity_type": "GENE", "description": "Previously linked to smoking, another risk factor for snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "BCL11B"}, {"entity_type": "GENE", "description": "Previously associated with coffee consumption; caffeine can influence sleep and snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "SMG6"}, {"entity_type": "GENE", "description": "Related to chronotype, impacting sleep schedules and snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "ROBO2"}, {"entity_type": "GENE", "description": "One of the prioritized genes for snoring, associated with smoking and alcohol consumption.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "NSUN3"}, {"entity_type": "GENE", "description": "Another gene from the prioritization list related to snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "SNAP91"}, {"entity_type": "GENE", "description": "Previously associated with smoking, a factor that can increase snoring risk.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "BLC11B"}, {"entity_type": "GENE", "description": "Linked with alcohol consumption and possibly affects sleep patterns, including snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "RNA5SP471"}, {"entity_type": "GENE", "description": "Also found to be associated with alcohol consumption and related to snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "SND1"}, {"entity_type": "GENE", "description": "Related to insomnia, a condition often linked to snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "LMO4"}, {"entity_type": "DISEASE", "description": "A neurological disorder that might influence sleep patterns and snoring.", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "id": "NARCOLEPSY"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant was identified as genome-wide significant SNP for snoring, has an effect size that differs significantly between sexes, and correlates highly with cross-sex genetic correlation.", "source_id": "chunk-a34b927ed42ff0917960122b4d203f83", "id": "RS199797821"}, {"entity_type": "GENOMIC VARIANT", "description": "Another variant identified as genome-wide significant for snoring, also shows a significantly different effect size between sexes but in the same direction.", "source_id": "chunk-a34b927ed42ff0917960122b4d203f83", "id": "RS200391180"}, {"entity_type": "SNP HERITABILITY ON THE LIABILITY SCALE FOR MALES", "description": "Heritability quantifies the proportion of variation in a trait that is due to genetic factors, here calculated with standard error.", "source_id": "chunk-a34b927ed42ff0917960122b4d203f83", "id": "H2SNP: 8.77% (SE = 0.54%)"}, {"entity_type": "SNP HERITABILITY ON THE LIABILITY SCALE FOR FEMALES", "description": "Heritability quantifies the proportion of variation in a trait that is due to genetic factors, here calculated with standard error.", "source_id": "chunk-a34b927ed42ff0917960122b4d203f83", "id": "H2SNP: 12.42% (SE = 0.57%)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The lead SNPs are part of molecular interactions within the genome which influence snoring traits.", "source_id": "chunk-a34b927ed42ff0917960122b4d203f83", "id": "RS199797821 AND RS200391180"}, {"entity_type": "GENETIC CORRELATION", "description": "High level of similarity in the effect sizes between males and females for these SNPs.", "source_id": "chunk-a34b927ed42ff0917960122b4d203f83", "id": "CROSS-SEX GENETIC CORRELATION: RG = 0.914 (SE = 0.033, P-VALUE = 1.91 \u00d7 10\u2212160)"}, {"entity_type": "DISEASE", "description": "The prevalence of self-reported recent snoring is calculated here, with a higher occurrence observed in males compared to females.", "source_id": "chunk-a34b927ed42ff0917960122b4d203f83", "id": "SELF-REPORTED RECENT SNORING (PREVALENCE: 32%)"}, {"entity_type": "PROGNOSTIC SCORE", "description": "This score was associated with increased risk for reporting recent snoring and potential obstructive sleep apnea based on sample prevalence.", "source_id": "chunk-a34b927ed42ff0917960122b4d203f83", "id": "RECENT SNORING PGS DECILE (HIGHEST VS. LOWEST)"}, {"entity_type": "VARIABLE", "description": "An indicator reflecting the different levels or intensity of snoring.", "source_id": "chunk-a34b927ed42ff0917960122b4d203f83", "id": "FREQUENCY OF SNORING SEVERITY"}, {"entity_type": "VARIANCE", "description": "The predictive power of the PGS for recent snoring was assessed, contributing to understanding genetic influence on this trait.)(entity", "source_id": "chunk-a34b927ed42ff0917960122b4d203f83", "id": "RECENT SNORING VARIANCE EXPLAINED BY PGS (FIG. 6C)"}, {"entity_type": "SNP", "description": "Key single nucleotide polymorphisms identified as significant contributors to snoring traits.", "source_id": "chunk-a34b927ed42ff0917960122b4d203f83", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 53}]", "id": "LEAD SNPS"}, {"source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "description": "Sleep apnoea showed a high genetic correlation (rG = 0.78) with snoring, indicating shared genetic factors.)", "entity_type": "UNKNOWN", "id": "GENETIC CORRELATION WITH SNORING"}, {"source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "description": "The SNP heritability for sleep apnoea was positively correlated with BMI\'s heritability (rG = 0.683), reflecting obesity as a risk factor.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 9}]", "id": "SNP HERITABILITY FOR SLEEP APNOEA"}, {"source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "description": "While alcohol consumption was not significantly genetically correlated with snoring, it might have had an indirect effect on sleep apnoea through shared risk factors.)", "entity_type": "UNKNOWN", "id": "GENETIC CORRELATION WITH SLEEP APNOEA"}, {"source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "description": "The genetic correlation between the adjusted genome-wide association study (GWAS) and snoring shows high consistency.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 33}]", "id": "ADJUSTED GWAS"}, {"source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "description": "Genetic correlation between nocturnal hypoxia measures such as minimum SpO2 and average SpO2 shows small increases in relation with snoring after adjusting for BMI.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 33}]", "id": "NOCTURNAL HYPOXIA"}, {"source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "description": "GWAS results suggest a possible bidirectional causal relationship between BMI and heart attack; BMI causes an increase in the risk of heart attack, which could be linked to snoring through sleep disruption or other factors.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 34}]", "id": "HEART ATTACK"}, {"entity_type": "GENOMIC VARIANT", "description": "SNPs are genetic variations in VDBP that may influence vitamin D metabolism.<SEP>SNPs were analyzed for their distribution among mothers and neonates.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "VDBP SINGLE NUCLEOTIDE POLYMORPHISMS (SNPS)"}, {"entity_type": "MOTHER AND NEONATE", "description": "Demographic, dietary and biochemical data from mothers and neonates are presented in Table 1.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "MOTHERS AND NEONATES"}, {"entity_type": "GENOMIC VARIANT", "description": "Maternal VDBP polymorphisms were considered for different genotypes (Table 2).", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "VDBP POLYMORPHISMS"}, {"entity_type": "MOTHER AND NEONATE DYAD", "description": "The study involves comparing mothers and their respective neonates to analyze the relationship between maternal genotypes and neonatal outcomes.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "MOTHERS AND NEONATES DYADS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Exposure to UVB radiation affects vitamin D synthesis in skin, which is influenced by VDBP polymorphisms.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "UVB RADIATION EXPOSURE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Gestation length can impact vitamin D status due to differences in maternal metabolism and sunlight exposure.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "WEEKS OF GESTATION"}, {"entity_type": "PHYSICAL MEASUREMENT", "description": "Height is a physical characteristic that might influence vitamin D synthesis or requirement.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "MATERNAL HEIGHT (CM)"}, {"entity_type": "BODY MASS INDEX BEFORE PREGNANCY", "description": "Pre-pregnancy BMI affects metabolic health and may influence vitamin D status.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "BMI PRE-PREGNANCY (KG/M2)"}, {"entity_type": "BODY MASS INDEX AFTER PREGNANCY", "description": "Post-pregnancy BMI reflects changes in metabolism that could impact vitamin D levels.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "BMI TERMINAL (KG/M2)"}, {"entity_type": "NUTRIENT CONCENTRATION", "description": "Levels of maternal serum 25-hydroxyvitamin D reflect her vitamin D status.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "MATERNAL 25(OH)D CONCENTRATIONS"}, {"entity_type": "NUTRIENT CONCENTRATION", "description": "Levels of neonatal serum 25-hydroxyvitamin D indicate their vitamin D status.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "NEONATAL 25(OH)D CONCENTRATIONS"}, {"entity_type": "GENOMIC VARIANT", "description": "Specific maternal VDBP genotypes may correlate with higher maternal vitamin D levels during delivery, as compared to others.<SEP>Specific maternal VDBP genotypes showed an association with higher vitamin D levels during delivery, based on p-values.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "MATERNAL VDBP GENOTYPES (CC FOR RS2298850 AND CC FOR RS4588)"}, {"entity_type": "THRESHOLDS FOR VITAMIN D STATUS", "description": "Cut-off points of neonatal vitamin D concentrations define different states of vitamin D equilibrium.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "NEONATAL 25(OH)D CUT-OFFS AT BIRTH (\u226550 NMOL/L VS. \u226450 NMOL/L AND \u226525 VS. \u226425 NMOL/L)"}, {"entity_type": "THRESHOLDS FOR VITAMIN D STATUS", "description": "Different maternal vitamin D statuses were compared with VDBP genotypes using various cut-off points.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "MATERNAL VITAMIN D STATUS AT BIRTH (CUT-OFFS AT BIRTH \u226425 VS. \u226525 NMOL/L, \u226450 VS. \u226550 NMOL/L AND \u226575 NMOL/L VS. \u226475 NMOL/L)"}, {"entity_type": "ASSOCIATION BETWEEN GENOTYPE AND OUTCOME", "description": "The neonatal vitamin D status varied based on the genetic makeup of their mother\'s VDBP polymorphisms, using cut-offs of 25 and 50 nmol/L.", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "id": "NEONATAL 25(OH)D CONCENTRATIONS, WITH RESPECT TO MATERNAL VDBP GENOTYPE DISTRIBUTION"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "Maternal vitamin D concentrations were analyzed in relation to her genotypes for VDBP polymorphisms.)<SEP>The vitamin D status of mothers was analyzed in relation to their VDBP polymorphisms.)", "entity_type": "UNKNOWN", "id": "MATERNAL VITAMIN D STATUS"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "Maternal vitamin D concentrations were analyzed in relation to her genotypes for VDBP polymorphisms.)<SEP>The vitamin D status of mothers was analyzed in relation to their VDBP polymorphisms.)", "entity_type": "UNKNOWN", "id": "MOTHERS"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "The health and developmental outcomes of neonates were compared with their mothers\' VDBP genotypes, looking at cut-offs of 25 and 50 nmol/L for vitamin D.)<SEP>Vitamin D status of neonates was also assessed with respect to their VDBP polymorphisms.)", "entity_type": "UNKNOWN", "id": "NEONATAL VITAMIN D STATUS"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "The health and developmental outcomes of neonates were compared with their mothers\' VDBP genotypes, looking at cut-offs of 25 and 50 nmol/L for vitamin D.)<SEP>Vitamin D status of neonates was also assessed with respect to their VDBP polymorphisms.)", "entity_type": "UNKNOWN", "id": "NEONATES"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "Percentage distribution of genotypes based on VDBP polymorphisms for mothers and neonates are presented (Table 2).)", "entity_type": "UNKNOWN", "id": "GENOTYPE PERCENTAGE DISTRIBUTIONS"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "Vitamin D cut-offs at birth were compared with maternal VDBP polymorphisms (Table 7).)", "entity_type": "UNKNOWN", "id": "CUT-OFFS AT BIRTH (\u226550 NMOL/L VS. \u226450 NMOL/L AND \u226525 VS. \u226425 NMOL/L)"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "Mothers with specific VDBP genotypes tended to have higher vitamin D concentrations at birth compared to other genotypes (p = 0.05 and p = 0.04 respectively, as seen in Table 8).)", "entity_type": "UNKNOWN", "id": "HIGHER VITAMIN D CONCENTRATIONS DURING DELIVERY"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "Neonatal vitamin D status was not significantly different based on maternal VDBP genotypes with cut-offs of 25 and 50 nmol/L at birth (Table 9 and Table 10).)", "entity_type": "UNKNOWN", "id": "MATERNAL VDBP POLYMORPHISMS"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "Certain maternal VDBP genotypes were associated with higher vitamin D levels when the mother delivered (Table 8).)", "entity_type": "UNKNOWN", "id": "MATERNAL VITAMIN D STATUS DURING DELIVERY"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "Neonatal vitamin D statuses were compared with different maternal VDBP genotypes based on birth cut-offs for vitamin D.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION BETWEEN GENOTYPE AND OUTCOME"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "Neonatal vitamin D statuses were compared with different maternal VDBP genotypes based on birth cut-offs for vitamin D.)", "entity_type": "UNKNOWN", "id": "NEONATAL VITAMIN D CUT-OFFS AT BIRTH (25 VS. 50 NMOL/L)"}, {"source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "description": "Specific maternal VDBP genotypes tended to have higher vitamin D levels at the time of delivery compared to other genotypes.)", "entity_type": "UNKNOWN", "id": "MATERNAL VITAMIN D CONCENTRATIONS DURING DELIVERY"}, {"entity_type": "NUTRIENT", "description": "Free testosterone is a form of circulating androgen hormone that may influence metabolism and reproductive function. Higher levels were observed among women with PCOS.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 45}]", "id": "FREE TESTOSTERONE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Bioactive testosterone, an active form of the androgenic hormone, showed significant differences between women with PCOS and controls.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 45}]", "id": "BIOACTIVE TESTOSTERONE"}, {"entity_type": "METABOLIC PATHWAY", "description": "The homeostatic model assessment of insulin resistance (HOMA-IR) was found significantly elevated among women with PCOS, an indicator of insulin resistance.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "HOMA-IR (HOMEOSTATIC MODEL ASSESSMENT OF INSULIN RESISTANCE)"}, {"entity_type": "PROTEIN", "description": "Lower levels of SHBG, a protein that binds to androgen hormones, were observed in women with PCOS compared to controls.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "SHBG (SEX HORMONE BINDING GLOBULIN)"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Though within normal range, TSH was higher among women with PCOS compared to control subjects.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "TSH (THYROID STIMULATING HORMONE)"}, {"entity_type": "GENOMIC VARIANT", "description": "Vascular Endothelial Growth Factor A (VEGFA) Single Nucleotide Polymorphisms were analyzed in relation to the presence of PCOS.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "VEGFA SNPS"}, {"entity_type": "TRANSCRIPT", "description": "The VEGFA SNP rs3025020 showed a higher frequency of the C allele among women with PCOS, potentially affecting expression levels.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "RS3025020 (-583T>C)"}, {"entity_type": "PROTEIN", "description": "Another VEGFA SNP, rs2010963, had differential genotype distribution between cases and controls, indicating possible influence on protein function.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "RS2010963 (-634C>G)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "There was no correlation between either VEGFA SNP (rs3025020 or rs2010963) and the levels of anti-inflammatory cytokine IL-10 among PCOS women.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "ANTI-INFLAMMATORY IL-10"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Correlation analysis did not reveal any correlation between either VEGFA SNP (rs3025020 or rs2010963) and pro-inflammatory cytokine TNF\u03b1 among PCOS women.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "PRO-INFLAMMATORY TNF\u0391"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "In the model including rs2010963, there was a negative association between the presence of the minor allele (C) and ESR.", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "id": "ERYTHROCYTE SEDIMENTATION RATE (ESR)"}, {"source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "description": "Elevated HOMA-IR was seen in women with PCOS, reflecting insulin resistance.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 45}]", "id": "HOMA-IR"}, {"source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "description": "Lower levels of SHBG were noted among women with PCOS compared to controls.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 45}]", "id": "SHBG"}, {"source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "description": "Among the model including rs2010963, women with the minor allele (C) showed lower erythrocyte sedimentation rate (ESR).)", "entity_type": "UNKNOWN", "id": "ESR"}, {"entity_type": "DISEASE", "description": "No significant difference found on self-report measures among genotype groups.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "STUDY 1"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A main effect of Condition on response bias was observed; participants showed reduced bias towards frequently rewarded stimuli under stress conditions.<SEP>A significant main effect of Condition for response bias due to reduced response bias toward the more frequently rewarded stimulus in the stress condition.<SEP>Study 1 found a main effect of Genotype on response bias during different blocks.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647<SEP>chunk-5e709975097a773fb3d31e265d76a71b", "id": "RESPONSE BIAS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Blocks influenced participants\' response bias and behavior according to reinforcement contingencies in Study 1.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "BLOCK"}, {"entity_type": "DISEASE", "description": "Genotype groups did not differ significantly on any demographic or self-report variables, but shock intensity was lower for val carriers compared to iso homozygotes.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "STUDY 2"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "A significant main effect of Condition and Time was observed for STAI scores, indicating heightened stress perception during the stress condition.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "STAI SCORES"}, {"entity_type": "DISEASE", "description": "No significant difference found on self-report measures among genotype groups.", "source_id": "chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "STUDY 1/"}, {"entity_type": "SNP/ASSOCIATION", "description": "The CT60A/G variant was linked with lower CTLA4 mRNA levels in individuals, indicating a role in genetic risk for autoimmune disorders.<SEP>Val allele carriers showed higher response bias compared to iso homozygotes in Study 1.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca<SEP>chunk-4b4f7d54511e2d8bbf5af6073d25b647", "id": "GENOTYPE/"}, {"entity_type": "DISEASE", "description": "Stress manipulation was successful; participants reported elevated anxiety and negative affect, and had more non-specific SC responses in the stress relative to no-stress condition.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "CONDITION"}, {"entity_type": "INTERACTION", "description": "The val allele explained unique variance in stress-induced response bias reductions, suggesting genetic predisposition influences adaptation to stress.<SEP>This effect was qualified by a Genotype \u00d7 Condition interaction, indicating that val carriers had significantly lower response bias in the stress condition compared to iso homozygotes.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "GENOTYPE \u00d7 CONDITION INTERACTION"}, {"entity_type": "CONDITION", "description": "Val carriers exhibited lower response bias specifically under the stress condition compared to no-stress conditions; this was not statistically significant for iso homozygotes.<SEP>Val carriers tended to have lower response bias in the stress relative to no-stress condition; the correlation between stress and no-stress response bias was not significant in either group.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "STRESS CONDITION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Elevated discriminability emerged due to elevated discriminability in Blocks 3 and 2 relative to Block 1, both p < .02.<SEP>Enhanced discriminability in Blocks 3 and 2 relative to Block 1 indicated improved sensitivity to stimuli differences during stressed states.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "DISCRIMINABILITY"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Differences in discriminability occurred between Blocks 1, 2, and 3; Blocks 2 and 3 had elevated discriminability compared to Block 1.", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "id": "BLOCK 1, BLOCK 2, AND BLOCK 3"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Differences in discriminability were evident between Conditions 3 and 2, both of which showed elevated values compared to Condition 1.)<SEP>Enhanced performance was noted in Conditions/Blocks with elevated discriminability compared to the baseline condition.)", "entity_type": "UNKNOWN", "id": "BLOCK DISCRIMINABILITY"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Differences in discriminability were evident between Conditions 3 and 2, both of which showed elevated values compared to Condition 1.)<SEP>Enhanced performance was noted in Conditions/Blocks with elevated discriminability compared to the baseline condition.)", "entity_type": "UNKNOWN", "id": "CONDITION 3 AND CONDITION 2"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Val carriers tended to have lower response bias in the stress condition compared to no-stress conditions; this difference was not statistically significant for iso homozygotes.)", "entity_type": "UNKNOWN", "id": "COHORT COMPARISON"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Val carriers tended to have lower response bias in the stress condition compared to no-stress conditions; this difference was not statistically significant for iso homozygotes.)", "entity_type": "UNKNOWN", "id": "STRESS CONDITION AND NO-STRESS CONDITION"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "There were differences between Blocks 1, 2, and 3 in terms of discriminability; Block 2 and Block 3 showed improved performance compared to Block 1.)", "entity_type": "UNKNOWN", "id": "BLOCK SEQUENCE EFFECTS"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "There were differences between Blocks 1, 2, and 3 in terms of discriminability; Block 2 and Block 3 showed improved performance compared to Block 1.)", "entity_type": "UNKNOWN", "id": "CONDITION 1, CONDITION 2 AND CONDITION 3"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "For discriminability, a main effect of Block emerged due to elevated discriminability in Blocks 3 and 2 relative to Block 1, both p < .02.)", "entity_type": "UNKNOWN", "id": "CONDITION 1"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "These SNPs were recently associated with stress-induced reward learning deficits, which influenced the findings on response bias.)<SEP>Three CRHR1 SNPs were associated with stress-induced reward learning deficits, which were taken into account as part of the control analyses.)<SEP>Three CRHR1 SNPs were associated with stress-induced reward learning deficits; these were considered as covariates.)", "entity_type": "UNKNOWN", "id": "CRHR1 SNPS"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "These SNPs were recently associated with stress-induced reward learning deficits, which influenced the findings on response bias.)<SEP>Three CRHR1 SNPs were associated with stress-induced reward learning deficits, which were taken into account as part of the control analyses.)<SEP>Three CRHR1 SNPs were associated with stress-induced reward learning deficits; these were considered as covariates.)", "entity_type": "UNKNOWN", "id": "STRESS-INDUCED REWARD LEARNING DEFICITS"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Val carriers had a distinct pattern of response bias reduction in the stress condition relative to no-stress conditions; this relationship held even after accounting for various factors like discriminability, shock intensity, skin conductance responses, and CRHR1 SNPs.)<SEP>Val carriers tended to have lower response bias in the stress condition compared to no-stress conditions; this difference was not statistically significant for iso homozygotes.)", "entity_type": "UNKNOWN", "id": "NO-STRESS CONDITION"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "An elevated discriminability emerged specifically in Conditions 3 and 2; performance was better than in Condition 1.)", "entity_type": "UNKNOWN", "id": "ELEVATED DISCRIMINABILITY"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Improved performance was noted specifically in Conditions/Blocks where discriminability was elevated.)", "entity_type": "UNKNOWN", "id": "BLOCK 2 AND BLOCK 3"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Unique variance in response bias reductions was determined after accounting for discriminability, shock intensity, and skin conductance responses.)", "entity_type": "UNKNOWN", "id": "DISCRIMINABILITY, SHOCK INTENSITY, SKIN CONDUCTANCE RESPONSES"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Unique variance in response bias reductions was determined after accounting for discriminability, shock intensity, and skin conductance responses.)", "entity_type": "UNKNOWN", "id": "RESPONSE BIAS REDUCTIONS"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Performance varied between Conditions/Blocks due to differences in discriminability; Blocks showed enhanced performance compared to the first block.)", "entity_type": "UNKNOWN", "id": "CONDITION 1, CONDITION 2, AND CONDITION 3"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Unique variance was explained by response bias reductions even after factoring in the effects of discriminability and skin conductance responses along with shock intensity.)", "entity_type": "UNKNOWN", "id": "DISCRIMINABILITY AND SHOCK INTENSITY, SKIN CONDUCTANCE RESPONSES"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "The sequence of conditions influenced performance outcomes; improvements were noted specifically in Conditions/Blocks where performance was enhanced due to increased discriminability.)", "entity_type": "UNKNOWN", "id": "CONDITION 3, CONDITION 2, AND CONDITION 1"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "The sequence of conditions influenced performance outcomes; improvements were noted specifically in Conditions/Blocks where performance was enhanced due to increased discriminability.)", "entity_type": "UNKNOWN", "id": "SEQUENCE OF CONDITIONS EFFECTS"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Val carriers showed a reduced response bias in the stress condition relative to no-stress conditions; this was observed even after considering various factors like discriminability, shock intensity, skin conductance responses, and CRHR1 SNPs.)", "entity_type": "UNKNOWN", "id": "GENOTYPE-SPECIFIC RESPONSE BIAS"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Enhanced performance was noted specifically in Conditions/Blocks where the discriminability metric was elevated; these blocks showed better performance compared to baseline conditions.)", "entity_type": "UNKNOWN", "id": "IMPROVED PERFORMANCE"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Val carriers had reduced response bias specifically in the stress condition when compared to no-stress conditions; this relationship held after factoring in discriminability, shock intensity, skin conductance responses, and CRHR1 SNPs.)", "entity_type": "UNKNOWN", "id": "COHORT COMPARISON OF RESPONSE BIAS"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "After considering the association of specific CRHR1 SNPs with stress-induced changes, unique variance was identified in response bias reductions during stressed conditions.)", "entity_type": "UNKNOWN", "id": "CRHR1 SNPS AND STRESSED CONDITION RESPONSE BIAS REDUCTIONS"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "After considering the association of specific CRHR1 SNPs with stress-induced changes, unique variance was identified in response bias reductions during stressed conditions.)", "entity_type": "UNKNOWN", "id": "UNIQUE VARIANCE IN STRESS-INDUCED CHANGES"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Val carriers showed reduced response bias specifically in the stress condition compared to no-stress conditions; this relationship was observed even after considering factors like discriminability, shock intensity, skin conductance responses, and CRHR1 SNPs for both groups.)", "entity_type": "UNKNOWN", "id": "STRESS CONDITION COMPARISON BETWEEN GROUPS"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Val carriers showed reduced response bias specifically in the stress condition compared to no-stress conditions; this relationship was observed even after considering factors like discriminability, shock intensity, skin conductance responses, and CRHR1 SNPs for both groups.)", "entity_type": "UNKNOWN", "id": "VAL AND ISO HETEROZYGOTES RESPONSE BIAS"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Within Conditions/Blocks where performance was enhanced due to elevated discriminability, improvements were observed; these conditions showed superior performance compared to baseline states after adjusting for various factors.)", "entity_type": "UNKNOWN", "id": "CONDITION 3 AND CONDITION 2 EFFECTS COMPARISON"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Within Conditions/Blocks where performance was enhanced due to elevated discriminability, improvements were observed; these conditions showed superior performance compared to baseline states after adjusting for various factors.)", "entity_type": "UNKNOWN", "id": "PERFORMANCE ENHANCEMENT IN ELEVATED DISCRIMINABILITY CONDITIONS"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Val carriers exhibited reduced response bias specifically in the stress condition relative to no-stress conditions, a relationship that was maintained after adjusting for factors like discriminability, shock intensity, skin conductance responses, and CRHR1 SNPs.)", "entity_type": "UNKNOWN", "id": "GENOTYPE-SPECIFIC RESPONSE BIAS REDUCTIONS DURING STRESSED CONDITIONS"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Val carriers exhibited reduced response bias specifically in the stress condition relative to no-stress conditions, a relationship that was maintained after adjusting for factors like discriminability, shock intensity, skin conductance responses, and CRHR1 SNPs.)", "entity_type": "UNKNOWN", "id": "STRESS CONDITION AND NO-STRESS CONDITION COMPARISON"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Elevated performance within Conditions/Blocks that featured elevated discriminability indicated enhanced outcomes; these instances showed superior performance compared to baseline states, adjusting for various factors.)", "entity_type": "UNKNOWN", "id": "PERFORMANCE ENHANCEMENT IN SPECIFIC BLOCKS/DAYS/"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Val carriers\' reduced response bias specifically in the stress condition relative to no-stress conditions, which maintained after adjusting for factors like discriminability, shock intensity, skin conductance responses, was further informed by considering specific CRHR1 SNPs.)", "entity_type": "UNKNOWN", "id": "GENOTYPE-SPECIFIC RESPONSE BIAS REDUCTIONS DURING STRESSED CONDITIONS AND CRHR1 SNPS ASSOCIATION"}, {"source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "description": "Elevated performance within Conditions/Blocks that featured elevated discriminability indicated enhanced outcomes; these instances showed superior performance compared to baseline states, adjusted for various factors including CRHR1 SNPs.)", "entity_type": "UNKNOWN", "id": "PERFORMANCE ENHANCEMENT IN SPECIFIC BLOCKS/CONDITIONS AND DISCRIMINABILITY ASSOCIATION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Age at first tooth is a physiological process that can be influenced by genetic factors.)<SEP>Moderate evidence of association with height, driven largely by variants in HMGA2 and BMP4.", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd<SEP>chunk-50e07db177a6a08799261934e8e71ac4", "id": "AGE AT FIRST TOOTH"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Moderate evidence of association with height, driven largely by variants in HMGA2 and BMP4.<SEP>Number of teeth is a physiological process that can be influenced by genetic factors.)", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd<SEP>chunk-50e07db177a6a08799261934e8e71ac4", "id": "NUMBER OF TEETH"}, {"entity_type": "GENE/", "description": "Genomic Variant", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "LOCI"}, {"entity_type": "GENE/", "description": "Genomic Variant", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "NOVEL ASSOCIATIONS"}, {"entity_type": "GENE/", "description": "Genomic Variant", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "PREVIOUSLY REPORTED GENES/REGIONS"}, {"entity_type": "DISEASE/", "description": "Phenotype", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "GENOME-WIDE SIGNIFICANCE"}, {"entity_type": "MOLECULAR INTERACTION/", "description": "An allelic score constructed from genome-wide significant SNPs for \'Age at first tooth\' and \'Number of teeth\'.<SEP>Genomic Variant<SEP>Therapy", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd<SEP>chunk-50e07db177a6a08799261934e8e71ac4", "id": "ALLELIC SCORE"}, {"entity_type": "GENOMIC VARIANT/", "description": "Phenotype", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "SUGGESTIVE LEVELS OF ASSOCIATION"}, {"entity_type": "STATISTICAL PROCESS/", "description": "Analysis", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "FAMILY-WISE ERROR RATE"}, {"entity_type": "GENE/", "description": "Genomic Variant", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "SECONDARY SIGNALS"}, {"entity_type": "SNP", "description": "Locus/", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "SECONDARY ANALYSIS/"}, {"entity_type": "MOLECULAR MECHANISM/", "description": "Phenotype", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "BIOLOGICAL PATHWAYS"}, {"entity_type": "DEVELOPMENT/", "description": "Genomic Variant", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "PHENOTYPIC PLASTICITY"}, {"entity_type": "STATISTICAL MEASURE/", "description": "Trait", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "HERITABILITY ESTIMATION"}, {"entity_type": "DEVELOPMENT/", "description": "Phenotype", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "ENVIRONMENTAL INFLUENCE"}, {"entity_type": "DENTAL HEALTH/", "description": "Trait", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "CLINICAL RELEVANCE"}, {"entity_type": "COMPLEX TRAIT/", "description": "Phenotype", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "GENETIC ARCHITECTURE"}, {"entity_type": "STATISTICAL ANALYSIS METHOD", "description": "Used PCA (Principal Component Analysis) and STRUCTURE method for identifying population substructure before association tests.<SEP>Study Design", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4<SEP>chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "POPULATION STRATIFICATION"}, {"entity_type": "STATISTICAL METHOD/", "description": "Multiple Studies", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "META-ANALYSIS"}, {"entity_type": "DEVELOPMENT/", "description": "Time-Series Data", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "LONGITUDINAL DYNAMICS"}, {"entity_type": "POLICY/", "description": "Dental Health", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "id": "PUBLIC HEALTH IMPLICATIONS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The physiological process moderated by \'Age at first tooth\' and \'Number of teeth\' allelic scores.", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "id": "HEIGHT"}, {"entity_type": "GENE", "description": "HMGA2 is associated with the moderate evidence of association between \'Age at first tooth\' and height.", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "id": "HMGA2"}, {"entity_type": "GENE", "description": "BMP4 is associated with the moderate evidence of association between \'Number of teeth\' and height.", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "id": "BMP4"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Analysis using the ALLIGATOR software to determine over-representation of genes in pathways.", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 42}]", "id": "PATHWAY ANALYSIS"}, {"source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "description": "Analysis", "entity_type": "UNKNOWN", "id": "EFFECTIVE NUMBER OF STATISTICAL TESTS"}, {"source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "description": "Analysis", "entity_type": "UNKNOWN", "id": "LOCUS/"}, {"source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "description": "SNPs underwent additional analyses to discern underlying genetic influences beyond the primary associations.)", "entity_type": "UNKNOWN", "id": "SECONDARY ANALYSIS"}, {"source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "description": "SNP association P-value threshold of 0.005 gave significant over-representation of genes in pathways for \'Age at first tooth\' GWAS.)", "entity_type": "UNKNOWN", "id": "PATHWAY"}, {"entity_type": "GENE", "description": "IL-10R\u03b1Tdel mice are generated with a selective deletion of the specific IL-10 receptor in T cells.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 40}]", "id": "IL-10R\u0391TDEL"}, {"entity_type": "GENE", "description": "IL-10R\u03b1DCdel mice have a selective deletion of the IL-10 receptor restricted to dendritic cells.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "IL-10R\u0391DCDEL"}, {"entity_type": "GENE", "description": "IL-10R\u03b1Mdel mice lack IL-10 receptor specifically in macrophages.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "IL-10R\u0391MDEL"}, {"entity_type": "GENOMIC VARIANT/DISEASE/NUTRIENT/PROTEIN/TRANSCRIPT/DIETARY PATTERN/PHYSIOLOGICAL PROCESS/METABOLIC PATHWAY/MOLECULAR INTERACTION/ENVIRONMENTAL FACTOR/DISEASE", "description": "Undefined information provided in the text, no direct relation identified.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "WT"}, {"entity_type": "GENOMIC VARIANT/DISEASE/NUTRIENT/PROTEIN/TRANSCRIPT/DIETARY PATTERN/PHYSIOLOGICAL PROCESS/METABOLIC PATHWAY/MOLECULAR INTERACTION/ENVIRONMENTAL FACTOR/DISEASE", "description": "No significant change in disease severity was apparent with neutrophil depletion in IL-10R\u03b1MdelRag1\u2212/\u2212 mice.<SEP>Rag1\u2212/\u2212 mice are the recipients used in this experiment for inducing and monitoring T cell transfer-mediated colitis.<SEP>Undefined information provided in the text, no direct relation identified.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51<SEP>chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "RAG1\u2212/\u2212"}, {"entity_type": "DIETARY PATTERN/PHYSIOLOGICAL PROCESS/METABOLIC PATHWAY/MOLECULAR INTERACTION/", "description": "No clear relationship among entities identified in this context.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "IL-10R\u0391DCDELRAG1\u2212/\u2212"}, {"entity_type": "PROTEIN/TRANSCRIPT/ENVIRONMENTAL FACTOR", "description": "Increased numbers and activation of IL-10R\u03b1MdelRag1-/- LPM\u03c6s.<SEP>Macrophage-specific deletion of IL-10 receptor leads to intensified colitis.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51<SEP>chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "IL-10R\u0391MDELRAG1\u2212/\u2212"}, {"entity_type": "GENE/PROTEIN/TRANSCRIPT", "description": "Germline deletion of IL-10R\u03b1 does not affect disease outcomes in Rag1\u2212/\u2212 mice, suggesting that M\u03d5-specific effects are crucial for IL-10\u2019s protective role.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "IL-10R\u0391FL/FLRAG1\u2212/\u2212"}, {"entity_type": "DISEASE/ENVIRONMENTAL FACTOR/", "description": "No direct relationship identified among entities mentioned.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "IL-10R\u0391MDELRAG1+/+"}, {"entity_type": "MOLECULAR INTERACTION/PHYSIOLOGICAL PROCESS/METABOLIC PATHWAY/MOLECULAR INTERACTION/", "description": "Depleting neutrophils does not significantly affect histopathology in Rag1\u2212/\u2212 recipients, indicating they have a minor protective role.", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "id": "NEUTROPHIL DEPLETION ABSOLUTY AB TREATMENT"}, {"entity_type": "MACROPHAGE", "description": "To isolate the M\u03c6-specific effect, we depleted neutrophils with specific Ab beginning prior to T cell transfer.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "M\u03a6"}, {"entity_type": "CELL", "description": "Undetectable in the peripheral blood from \u03b1Ly6G Ab but not control Ab treated animals throughout the experimental interval.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "NEUTROPHIL"}, {"entity_type": "ANTIBODY", "description": "To identify causes of the enhanced IL-10R\u03b1MdelRag1-/- disease, we first analyzed lamina propria macrophages (LPM\u03c6s).", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "\u0391LY6G AB"}, {"entity_type": "MACROPHAGE", "description": "Significantly increased in IL-10R\u03b1MdelRag1-/- mice with colitis compared with Rag1-/- controls.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "LAMINA PROPRIA MACROPHAGE (LPM\u03a6)"}, {"entity_type": "MACROPHAGE", "description": "These cells showed a uniformly elevated surface expression of activation markers like CD40, CD80 and CD86.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "IL-10R\u0391MDELRAG1\u2212/\u2212 LPM\u03a6S"}, {"entity_type": "CELL PROCESS", "description": "Unlike for LPM\u03c6s, no differences in BrdU incorporation were evident in colonic CD4 T cells, DCs, and neutrophils, or in M\u03c6s from other locations.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "BRDU INCORPORATION"}, {"entity_type": "REGULATORY CELL", "description": "Treg co-transferred with na\u00efve T cells even at a 1:1 ratio were incapable of preventing disease.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "T REGULATORY CELLS (TREG)"}, {"entity_type": "DISEASE STATE COMPARISON", "description": "No differences were observed in Foxp3+ iTreg forming.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "IL-10R\u0391MDELRAG1\u2212/\u2212 MICE AND CONTROL RAG1\u2212/\u2212 COLITIC MICE"}, {"entity_type": "REGULATORY CELL TRANSFER", "description": "The transferred cells were able to fully suppress disease development in both IL-10R\u03b1MdelRag1\u2212/\u2212 and Rag1\u2212/\u2212 recipients.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "TREG TRANSFERS"}, {"entity_type": "ORGAN", "description": "A >2-fold increase in Treg in the spleens of recipient mice was seen.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "SPLEEN"}, {"entity_type": "PROTEIN", "description": "This population showed a uniformly elevated surface expression of CD40, which is an activation marker.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "CD40"}, {"entity_type": "PROTEIN", "description": "The cells also exhibited enhanced levels of CD80 and CD86 on their surface.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "CD80"}, {"entity_type": "DISEASE MODEL", "description": "Recent findings have identified defective iTreg formation and Treg function using this model.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "IL-10R\u0392\u2212/\u2212RAG2\u2212/\u2212 MICE"}, {"entity_type": "REGULATORY CELL", "description": "These cells play a role in the control of immune responses, which was assessed through their ability to prevent disease.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "T REGULATORY CELLS (TREGS)"}, {"entity_type": "CELL", "description": "Transferred naive T cells were used for inducing colitis and assessing Treg function.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "NA\u00cfVE T CELLS"}, {"entity_type": "GENETIC MODEL", "description": "This mouse line allowed us to analyze IL-10 production during disease development.", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "id": "IL-10-GFP KNOCK-IN REPORTER MICE"}, {"entity_type": "PROTEIN", "description": "Pre-treatment with IL-10 abrogates induction of activated caspase-1 by LPS and ATP in Rag1\u2212/\u2212 M\u03c6s.<SEP>The IL-10 protein plays a critical role in suppressing pro-inflammatory cytokine production by macrophages and is involved in modulating the Th17 response.<SEP>Treatment with IL-10 diminished pro-IL-\u03b2, undetectable mature IL-1\u03b2 in Rag1\u2212/\u2212 cultures.", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6<SEP>chunk-b971aa450a9b220806bdc000eb5efaad", "id": "IL-10"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Macrophages produce IL-1\u03b2, IL-6, TNF-\u03b1, MCP-1, and other cytokines that are increased during colitis. This heightened production is specific to M1-like macrophages in IL-10R\u03b1MdelRag1\u2212/\u2212 mice.", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "id": "MACROPHAGE"}, {"entity_type": "CELL", "description": "Formation of some Th17 cells with Rag1\u2212/\u2212 LPM\u03c6s, but markedly elevated in IL-10R\u03b1MdelRag1\u2212/\u2212 M\u03c6s.<SEP>Th17 cells are critical for the development of colitis, as they increase during disease progression and their absence prevents or attenuates disease severity.", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6<SEP>chunk-b971aa450a9b220806bdc000eb5efaad", "id": "TH17 CELL"}, {"entity_type": "PROINFLAMMATORY CYTOKINES/MOLECULAR INTERACTION/CELLULAR PROCESS/ENVIRONMENTAL FACTOR/", "description": "A pro-inflammatory cytokine whose production is modulated by IL-10.<SEP>IL-1\u03b2 is a pro-inflammatory cytokine that is upregulated in colonic inflammation. It plays an essential role in the induction of Th17 cells.<SEP>Increased IL-1\u03b2 levels are associated with inflammation and disease in the context of LPM\u03c6s\' role.<SEP>|>Disease/Inflammation", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6<SEP>chunk-174831b97ed170039b374ba254d4f506<SEP>chunk-b971aa450a9b220806bdc000eb5efaad<SEP>chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "IL-1\u0392"}, {"entity_type": "CYTOKINE", "description": "<SEP>TNF-\u03b1 is involved in inflammation and may be produced by M\u03d5s during disease, contributing to colonic pathology.<SEP>|>Disease/Inflammation<SEP>Tumor Necrosis Factor-alpha (TNF-\u03b1) is a pro-inflammatory cytokine that participates in various inflammatory processes.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c<SEP>chunk-8fb20f873daf79a9b7d6e8d25c8889d6<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce<SEP>chunk-174831b97ed170039b374ba254d4f506", "id": "TNF-\u0391"}, {"entity_type": "RECEPTOR", "description": "The IL-10 receptor (IL-10R\u03b1) on macrophages is critical for the ability of IL-10 to suppress inflammatory responses and modulate immune cell function, especially during colitis.", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "id": "IL-10R\u0391"}, {"entity_type": "PROTEIN", "description": "IL-17, along with IL-22, helps regulate inflammation during the recovery from colitis.<SEP>Increased IL-17 production by T cells has been linked with Th17-mediated inflammation, including in inflamed colons.", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6<SEP>chunk-29025895ac187b8e88dec5cea9478842", "id": "IL-17"}, {"entity_type": "PROTEIN", "description": "IFN-\u03b3 is typically associated with the TH1 (helper T) response and can contribute to inflammation but does not seem as significantly involved as IL-17 during colitis.<SEP>Interferon gamma (IFN-\u03b3) plays a role in Th1 dominant insulitis and leukocyte infiltrations in pancreatic islets of NOD mice.<SEP>The gene expression of IFN-\u03b3 was not affected.<SEP>Interferon-gamma is a cytokine primarily produced by T helper (Th)1 cells, essential for immune response against pathogens.<SEP>The expression levels of IFN-\u03b3 increased when examining the combined effects of MAPbacteremia presence and specific SNPs on PTPN2/PTPN22 expression in CD subjects compared to non-infected subjects without these SNPs.<SEP>The expression of IFN-\u03b3 was higher in CD subjects with certain genetic variants compared to those without.", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce<SEP>chunk-0380fd029b3eda680c87c1b8bcbe7f4c<SEP>chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-8382a49b61215482f9fc75e3a86cb987<SEP>chunk-6a657cd0a000b6df4db8f1305b795021", "id": "IFN-\u0393"}, {"entity_type": "GENE", "description": "The delRag1 gene was involved in the colitis response, but alternative pathologic pathways may have also been involved.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "DELRAG1\u2212/\u2212 MICE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The Th17 response is fundamental to T cell transfer colitis and involves colonic inflammation.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "TH17 RESPONSE"}, {"entity_type": "PROTEIN", "description": "ROR\u03b3T, essential for the development of Th17 cells, was proven important in both Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 recipients.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "ROR\u0393T\u2212/\u2212 T CELLS"}, {"entity_type": "DISEASE", "description": "IL-6-independent colitis is found in IL-10R\u03b1MdelRag1\u2212/\u2212 mice, which indicates a different pathogenic mechanism.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "IL-6 INDEPENDENT COLITIS"}, {"entity_type": "PROTEIN", "description": "The lack of IL-6 receptor alpha (IL-6R\u03b1) on T cells proved effective in inducing disease pathogenesis.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "IL-6R\u0391\u2212/\u2212 T CELLS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Th17 cell percentages among CD4+TCR+ T cells were reduced when IL-6R\u03b1\u2212/\u2212 T cells are transferred into recipients.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "TH17 CELL PERCENTAGES"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The representation of Th17 cells was significantly decreased specifically in IL-10R\u03b1MdelRag1\u2212/\u2212 mice\'s colon.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "TH17 CELLS IN THE COLON"}, {"entity_type": "TISSUE", "description": "Analysis of T cell responses leading to colonic injury was carried out at the MLNs of recipient mice.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "MLN (MESENTERIC LYMPH NODE)"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Differences were identified in absolute numbers of Th17 cells across different organs including spleens, MLNs, and colons.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "TH17 CELLS IN SPLEENS, MLNS, AND COLONS"}, {"entity_type": "TISSUE", "description": "The total T cell number was affected by the transfer combination IL-6R\u03b1\u2212/\u2212\u2192Rag1\u2212/\u2212, leading to decreased numbers in certain organs.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "TOTAL T CELL NUMBER"}, {"entity_type": "MOLECULAR INTERACTION", "description": "IL-6 is essential for T cell expansion and colitis development specifically in Rag1\u2212/\u2212 mice.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "IL-6 RESPONSE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "IL-10\'s actions on macrophages (M\u03c6) generate an IL-6 requirement for autoinflammatory disease and robust T cell expansion.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "IL-10 ACTIONS ON M\u03a6S"}, {"entity_type": "DISEASE", "description": "No differences were identified in the proportions of Th1 cells or Foxp3+CD4+ cells during comparison.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "TH17 DEVELOPMENT"}, {"entity_type": "GENE", "description": "The IL-6\u03b2 receptor played an essential role in mediating disease pathogenesis.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "IL-6\u0392 RECEPTOR"}, {"entity_type": "DISEASE VARIANT", "description": "This specific disease variant of mice exhibited a dramatic elevation in IL-1\u03b2 expression, suggesting it contributes to the disease.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "INTERLEUKIN (IL)-10R\u0391MDELRAG1 MICE"}, {"entity_type": "CELL TYPE", "description": "The actions on macrophages generated an IL-6 requirement for autoinflammatory disease and robust T cell expansion.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "MACROPHAGES (M\u03a6)"}, {"entity_type": "PROTEIN", "description": "Naive T cells were transferred into Rag1\u2212/\u2212 mice to induce colitis, highlighting their role in the disease process.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "TRANSFERRED NAIVE T CELLS"}, {"entity_type": "CELL SURFACE RECEPTOR", "description": "The absence of IL-6R\u03b1 on T cells was found effective in inducing less severe clinical and histologic measures of colitis.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "IL-6 RECEPTOR \u0391 (IL-6R\u0391)"}, {"entity_type": "CELL SURFACE RECEPTOR", "description": "The role of this receptor in the impact of IL-1\u03b2 during induction of colitis was tested and proven essential.", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "id": "INTERLEUKIN (IL)-1 RECEPTOR"}, {"entity_type": "GENE", "description": "IL-6R\u03b1 plays a role in disease protection when transferred into recipients.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "IL-6R\u0391"}, {"entity_type": "TRANSCRIPT", "description": "A Transcript of a gene, which plays a role in protein synthesis and can be influenced by environmental factors.<SEP>The text doesn\'t specify any transcripts related to this context.<SEP>The expression of this transcript in maternal tissues could be involved in the regulation or response to B12 or folate.<SEP>The protein-coding sequence of a gene that is expressed during development and in normal cellular functions.<SEP>No transcript information provided for this context.<SEP>NFATC1 is a transcript that can be affected by genomic variants mapping to its region, impacting gene regulation and disease.<SEP>The transcript is influenced by dietary patterns and folate intake.<SEP>The transcript of these MS risk genes play a role in the EBV latency III program.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-2fb95100a1290b27baae869c6560419a<SEP>chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-45a965c3c3013c70d0fa0f1680283083<SEP>chunk-b2730f3087aee6d9cff6e4939b229323", "id": "TRANSCRIPT"}, {"entity_type": "NUTRIENT", "description": "A Nutrient that is essential for health and can be part of a Dietary Pattern influencing physiological processes.<SEP>The text doesn\'t mention any nutrients related to this context.<SEP>A key nutrient found in leafy greens, playing a role in nitric oxide production and cardiovascular health.<SEP>An essential substance required for maintaining proper physiological function in organisms, including vitamins and minerals.<SEP>The nutrient vitamin D is metabolized by these enzymes to maintain calcium and phosphate levels in blood.<SEP>The text does not provide information on any specific nutrient that interacts with a gene.<SEP>Folate is a key nutrient found in leafy greens, which plays a role in cardiovascular health and nitric oxide production.<SEP>Folate metabolism is influenced by genetic variants in genes like NOS1, which plays a critical role in cardiovascular health.<SEP>Certain nutrients may influence the expression of these MS risk genes through dietary patterns or other means.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-cc528af98598e1adb4684c0166430cf4<SEP>chunk-2fb95100a1290b27baae869c6560419a<SEP>chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-3cf4e094a5312845a1cc4da4304d4bdf<SEP>chunk-6a657cd0a000b6df4db8f1305b795021<SEP>chunk-b2730f3087aee6d9cff6e4939b229323<SEP>chunk-382f24e5db6941e2e3db379fe499b223", "id": "NUTRIENT"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "An internal process occurring in the body that maintains homeostasis and supports life functions.<SEP>There\'s no clear reference to a specific physiological process mentioned in the text for this context.<SEP>The process of B12 absorption and metabolism might have been considered here.<SEP>The body\'s functions related to maintaining homeostasis and supporting life activities.<SEP>The process of converting vitamin D into its active form (1,25(OH)2D) involves CYP27B1 and CYP24A1 proteins.<SEP>The text mentions that folate metabolism interacts with the NOS1 protein to maintain vascular function; however, this interaction is not fully detailed in the provided information.<SEP>The interaction between folate metabolism and NOS1 gene is a key physiological process that modulates cardiovascular health.<SEP>These MS risk genes are involved in various physiological processes relevant to immune response and disease.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-2fb95100a1290b27baae869c6560419a<SEP>chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-22216738ccbae0a6276bb4499018c434<SEP>chunk-3cf4e094a5312845a1cc4da4304d4bdf<SEP>chunk-6a657cd0a000b6df4db8f1305b795021<SEP>chunk-b2730f3087aee6d9cff6e4939b229323", "id": "PHYSIOLOGICAL PROCESS"}, {"entity_type": "METABOLIC PATHWAY", "description": "A sequence of chemical reactions within a cell where molecules are transformed into other molecules.<SEP>The metabolic pathways aren\'t explicitly stated here, but IL-1\u03b2 production could be part of an inflammatory response pathway.<SEP>A series of biochemical reactions that occur within cells and are responsible for energy production, synthesis of molecules, and degradation of substances.<SEP>This pathway includes the steps from 25(OH)D to 1,25(OH)2D synthesis using enzymes CYP27B1 and CYP24A1.<SEP>Disruptions in the nitric oxide synthesis metabolic pathway are associated with an increased risk of developing cardiovascular disease.<SEP>The metabolic pathways involving these proteins or metabolites may affect the activity of MS risk genes or EBNA2 interaction.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c<SEP>chunk-b803830d07bff8ae14550f6458048707<SEP>chunk-2fb95100a1290b27baae869c6560419a<SEP>chunk-03962ffc6066ee5da582bdc89310b1d6<SEP>chunk-6a657cd0a000b6df4db8f1305b795021<SEP>chunk-b2730f3087aee6d9cff6e4939b229323", "id": "METABOLIC PATHWAY"}, {"entity_type": "LYMPHOCYTE SUBSET", "description": "Th17 cells are key in orchestrating the immune response to inflammation.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "TH17 CELLS"}, {"entity_type": "CELL SURFACE MARKER", "description": "A marker for monocytes and macrophages, indicative of inflammatory processes.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "CD11B"}, {"entity_type": "MACROPHAGE SUBTYPE", "description": "Lung progenitor macrophages are crucial in the lung\'s immune response and inflammation management.<SEP>Specific type of macrophages that contribute to lung inflammation in autoimmune diseases.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "LPM\u03a6S (LUNG PROGENITOR MACROPHAGES)"}, {"entity_type": "PROTEIN", "description": "Caspase-1 is part of the inflammasome, involved in processing pro-inflammatory cytokines.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "CASPASE-1"}, {"entity_type": "CELLULAR COMPLEX", "description": "Multi-protein complexes that lead to the activation and maturation of cytokines like IL-1\u03b2.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "INFLAMMASOMES"}, {"entity_type": "LYMPHOID ORGAN", "description": "Site of T cell priming where antigen processing occurs before entering the bloodstream.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "MLNS (MESENTERIC LYMPH NODE)"}, {"entity_type": "BONE STRUCTURE", "description": "Hematopoietic stem cells develop into various immune cells here, including macrophages and T cells.", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "id": "BONE MARROW"}, {"entity_type": "TRANSCRIPT/PROTEIN", "description": "Decreased expression of pro-IL-\u03b2 by IL-10 treatment in Rag1\u2212/\u2212 cultures.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "PRO-IL-\u0392"}, {"entity_type": "DISEASE/PROTEIN", "description": "Undetectable mature IL-1\u03b2 levels after IL-10 treatment in Rag1\u2212/\u2212 M\u03c6s.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "MATURE IL-1\u0392"}, {"entity_type": "MOLECULAR INTERACTION/CYTOKINE RECEPTOR", "description": "M\u03c6s lacking IL-10R\u03b1 maintain normal levels of pro-IL-\u03b2 and mature IL-1\u03b2, unlike Rag1\u2212/\u2212 M\u03c6s.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "IL-10R\u0391MDELRAG1\u2212/\u2212 M\u03a6S"}, {"entity_type": "DRUG/NEUTRALIZING ANTIBODY", "description": "Blockade of the IL-10 receptor increases activated caspase-1 (p20) in Rag1\u2212/\u2212 M\u03c6s.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "ANTI-IL-10R ANTIBODY"}, {"entity_type": "PHYSIOLOGICAL PROCESS/STIMULATION", "description": "LPS stimulation leads to a modest increase in activated caspase-1 in Rag1\u2212/\u2212 M\u03c6s.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "LPS"}, {"entity_type": "PHYSIOLOGICAL PROCESS/STIMULATION", "description": "ATP also stimulates the activation of caspase-1 by LPS-stimulated Rag1\u2212/\u2212 M\u03c6s.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "ATP"}, {"entity_type": "MOLECULAR INTERACTION/ACTIVATION", "description": "Pre-treatment with anti-IL-10R\u03b1 Ab increases activated caspase-1, indicating a normal restrain by IL-10 on caspase-1 activation in Rag1\u2212/\u2212 M\u03c6s.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "CASPASE-1 CLEAVAGE"}, {"entity_type": "MOLECULAR INTERACTION/CELLULAR PROCESS", "description": "Increased IL-1 and IL-6 from LPM\u03c6s can impact Th17 cell maturation in the presence of na\u00efve T cells with added TGF\u03b2.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "TH17 CELL MATURATION"}, {"entity_type": "DRUG/ANTIBODY", "description": "Blocking IL-6 receptor partially inhibits Th17 formation from Rag1\u2212/\u2212 LPM\u03c6s, but less so from IL-10R\u03b1MdelRag1\u2212/\u2212-derived LPM\u03c6s.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "\u0391IL-6R\u0391 BLOCKING ANTIBODY"}, {"entity_type": "MOLECULAR INTERACTION/INHIBITION/", "description": "Blocking IL-1 signaling leads to more substantial inhibition of Th17 development in IL-10R\u03b1Mdel Rag1\u2212/\u2212-derived LPM\u03c6s.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "IL-1 SIGNALING"}, {"entity_type": "MOLECULAR INTERACTION/CYTOKINE RECEPTOR", "description": "Transfer of IL-6R\u2212/\u2212 T cells results in modest effect on Th17 formation with IL-10R\u03b1MdelRag1\u2212/\u2212-derived LPM\u03c6s.", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "id": "IL-6R\u2212/\u2212 T CELLS"}, {"source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "description": "These cells showed a uniformly elevated surface expression of activation markers like CD40, CD80 and CD86.)", "entity_type": "UNKNOWN", "id": "INCREASED NUMBERS AND ACTIVATION"}, {"source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "description": "Though no significant difference in histopathology was identified (Fig. 1e, f).)", "entity_type": "UNKNOWN", "id": "EFFECT ON IL-10R\u0391MDELRAG1\u2212/\u2212 MICE"}, {"source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "description": "Though no significant difference in histopathology was identified (Fig. 1e, f).)", "entity_type": "UNKNOWN", "id": "NEUTROPHIL DEPLETION"}, {"source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "description": "Treg transfers led to a >2-fold increase in Treg in the spleens of recipient mice and Treg percentages did not differ between IL-10R\u03b1MdelRag1\u2212/\u2212 and Rag1\u2212/\u2212 recipients.)", "entity_type": "UNKNOWN", "id": "DISEASE SUPPRESSION"}, {"source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "description": "Treg transfers led to a >2-fold increase in Treg in the spleens of recipient mice and Treg percentages did not differ between IL-10R\u03b1MdelRag1\u2212/\u2212 and Rag1\u2212/\u2212 recipients.)", "entity_type": "UNKNOWN", "id": "TREG"}, {"source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "description": "No influence on disease magnitude when T cell IL-10 production was performed.)", "entity_type": "UNKNOWN", "id": "DISEASE MAGNITUDE"}, {"source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "description": "No influence on disease magnitude when T cell IL-10 production was performed.)", "entity_type": "UNKNOWN", "id": "IL-10 PRODUCTION BY TRANSFERRED T CELLS"}, {"source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "description": "The cells had a heightened expression of CD40, CD80 and CD86 on their surface.)", "entity_type": "UNKNOWN", "id": "INCREASED ACTIVATION MARKERS"}, {"source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "description": "Significant increase in LPM\u03c6 numbers was observed in IL-10R\u03b1MdelRag1\u2212/\u2212 mice compared to Rag1\u2212/\u2212 controls.)", "entity_type": "UNKNOWN", "id": "DISEASE SEVERITY COMPARISON"}, {"source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "description": "Significant increase in LPM\u03c6 numbers was observed in IL-10R\u03b1MdelRag1\u2212/\u2212 mice compared to Rag1\u2212/\u2212 controls.)", "entity_type": "UNKNOWN", "id": "IL-10R\u0391MDELRAG1\u2212/\u2212 MICE VS. RAG1\u2212/\u2212 CONTROLS"}, {"source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "description": "Transferred Treg were able to fully suppress disease development regardless of whether the transferred T cells produced IL-10 or not.)", "entity_type": "UNKNOWN", "id": "DISEASE SUPPRESSION CAPABILITY"}, {"source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "description": "Transferred Treg were able to fully suppress disease development regardless of whether the transferred T cells produced IL-10 or not.)", "entity_type": "UNKNOWN", "id": "TREGS VS. T CELLS WITH/WITHOUT IL-10 PRODUCTION"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "Transferring Th17 cells into Rag1\u2212/\u2212 mice results in colitis development. It also implies a crucial role for Th17 cells in disease pathogenesis, as they are necessary and sufficient for disease progression.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 40}]", "id": "TH17 CELL TRANSFER"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "The IL-10R\u03b1MdelRag1\u2212/\u2212 mice exhibit exacerbated colitis due to an elevated Th17 response and increased levels of pro-inflammatory cytokines, indicating that the loss of IL-10R\u03b1 function influences disease severity.)", "entity_type": "UNKNOWN", "id": "INCREASED DISEASE SEVERITY"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "Increased Th17 cell presence in inflamed colonic tissues correlates with the development and progression of colitis, highlighting their role in inflammatory bowel disease.)", "entity_type": "UNKNOWN", "id": "INFLAMED COLONS"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "IL-1\u03b2 acts as a pro-inflammatory cytokine that is upregulated during colitis, contributing to inflammation by inducing other pro-inflammatory cytokines and cell recruitment.)", "entity_type": "UNKNOWN", "id": "PRO-INFLAMMATORY CYTOKINE"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "Increased levels of IL-23 are identified alongside IL-1\u03b2 and IL-6, suggesting a significant role in intensifying the Th17 response during colitis.)", "entity_type": "UNKNOWN", "id": "CYTOKINE"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "Increased levels of IL-23 are identified alongside IL-1\u03b2 and IL-6, suggesting a significant role in intensifying the Th17 response during colitis.)", "entity_type": "UNKNOWN", "id": "IL-23"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "The percent and absolute numbers of CD4+IL-17A+ T cells are significantly increased in IL-10R\u03b1Mdel Rag1\u2212/\u2212 compared to Rag1\u2212/\u2212 mice, indicating an enhanced Th17 response.)", "entity_type": "UNKNOWN", "id": "CD4+IL-17A+ T CELLS"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "The percent and absolute numbers of CD4+IL-17A+ T cells are significantly increased in IL-10R\u03b1Mdel Rag1\u2212/\u2212 compared to Rag1\u2212/\u2212 mice, indicating an enhanced Th17 response.)", "entity_type": "UNKNOWN", "id": "INCREASE"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "After induction of disease with CD4+CD25\u2212CD45RBhi ROR\u03b3T-GFP reporter T cells, there is an increased identification of GFP+ T cells in colons at weeks 4 and 8, suggesting the involvement of Th17 cells in the disease process.)", "entity_type": "UNKNOWN", "id": "GFP+ T CELLS"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "Elevated production of Th17-promoting cytokines (IL-1\u03b2, IL-6, IL-23p19), and other pro-inflammatory markers like iNOS indicates a heightened Th17 response in colitis.)", "entity_type": "UNKNOWN", "id": "INCREASED EXPRESSION"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "Elevated production of Th17-promoting cytokines (IL-1\u03b2, IL-6, IL-23p19), and other pro-inflammatory markers like iNOS indicates a heightened Th17 response in colitis.)", "entity_type": "UNKNOWN", "id": "TH17-ASSOCIATED CYTOKINES"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "Increased expression of IL-1\u03b2, IL-6, IL-23p19, and iNOS specifically within macrophages suggests they are the primary source for increased Th17-associated cytokines and implicates a suppressive role played by IL-10 on these cells.)", "entity_type": "UNKNOWN", "id": "M\u03a6S"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "Increased expression of IL-1\u03b2, IL-6, IL-23p19, and iNOS specifically within macrophages suggests they are the primary source for increased Th17-associated cytokines and implicates a suppressive role played by IL-10 on these cells.)", "entity_type": "UNKNOWN", "id": "SPECIFIC EXPRESSION"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "The transfer of ROR\u03b3T\u2212/\u2212 (Th17 cells deficient) T cells into Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 recipients prevents or attenuates colitis, indicating that Th17 cell involvement is essential for disease development.)", "entity_type": "UNKNOWN", "id": "PREVENT OR ATTENUATE DISEASE"}, {"source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "description": "The transfer of ROR\u03b3T\u2212/\u2212 (Th17 cells deficient) T cells into Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 recipients prevents or attenuates colitis, indicating that Th17 cell involvement is essential for disease development.)", "entity_type": "UNKNOWN", "id": "ROR\u0393T\u2212/\u2212 T CELLS TRANSFER"}, {"source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "description": "There was a difference in disease development between Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 recipients.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 40}]", "id": "RAG1\u2212/\u2212 AND IL-10R\u0391MDELRAG1\u2212/\u2212 RECIPIENTS"}, {"source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "description": "The Th17 response was found to be reduced in the MLNs of Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 recipients.)", "entity_type": "UNKNOWN", "id": "TH17 CELL PERCENTAGES AMONG CD4+TCR+ T CELLS"}, {"source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "description": "IL-6R\u03b1 signaling supports but is not essential for Th17 formation specifically in the MLNs of both recipient lines.)", "entity_type": "UNKNOWN", "id": "RESPONSE"}, {"source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "description": "IL-6R\u03b1 signaling supports but is not essential for Th17 formation specifically in the MLNs of both recipient lines.)", "entity_type": "UNKNOWN", "id": "TH17 CELL DEVELOPMENT"}, {"source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "description": "There was a difference in absolute numbers of Th17 cells present between Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 recipients.)", "entity_type": "UNKNOWN", "id": "ABSOLUTE NUMBERS OF TH17 CELLS ACROSS ORGANS"}, {"source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "description": "Rag1\u2212/\u2212 mice demonstrated a broad requirement for IL-6 for T cell expansion and development of colitis.)", "entity_type": "UNKNOWN", "id": "RAG1\u2212/\u2212 MICE"}, {"source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "description": "In contrast, IL-10R\u03b1MdelRag1\u2212/\u2212 mice showed dispensable IL-6 for the development of colitis.)", "entity_type": "UNKNOWN", "id": "IL-10R\u0391MDELRAG1\u2212/\u2212 MICE"}, {"source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "description": "The absence of IL-6R\u03b1 signaling in T cells induced impaired expansion and lesser disease effect compared to WT T cell transfers.)", "entity_type": "UNKNOWN", "id": "IL-6R\u0391\u2212/\u2212 T CELLS AND RAG1\u2212/\u2212 COMBINATION"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "Transfer of IL-6R\u03b1 deficient T cells shows virtually complete disease protection.)", "entity_type": "UNKNOWN", "id": "DISEASE PROTECTION"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "Transfer of IL-6R\u03b1 deficient T cells shows virtually complete disease protection.)", "entity_type": "UNKNOWN", "id": "IL-6R\u0391 TRANSFER"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "Transfers of IL-1R deficient T cells show diminished clinical measures compared to WT transfers in Rag1\u2212/\u2212 colitis models.)", "entity_type": "UNKNOWN", "id": "IL-1R TRANSFER"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "Transfers of IL-1R deficient T cells show diminished clinical measures compared to WT transfers in Rag1\u2212/\u2212 colitis models.)", "entity_type": "UNKNOWN", "id": "REDUCED DISEASE SEVERITY"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "While IL-6 responsiveness affects disease severity in Rag1\u2212/\u2212, IL-1 response impacts both Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 models.)", "entity_type": "UNKNOWN", "id": "DISTINCT DOSE RESPONSE"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "While IL-6 responsiveness affects disease severity in Rag1\u2212/\u2212, IL-1 response impacts both Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 models.)", "entity_type": "UNKNOWN", "id": "IL-6R\u0391 VS. IL-1R TRANSFER IMPACT"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "Transfers of IL-6R\u03b1 deficient T cells show reduced Th17 cell proportions compared to WT T cells transfers, specifically in colonic environment.)", "entity_type": "UNKNOWN", "id": "REDUCED IN T CELL TRANSFER MODELS"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "Transfers of IL-6R\u03b1 deficient T cells show reduced Th17 cell proportions compared to WT T cells transfers, specifically in colonic environment.)", "entity_type": "UNKNOWN", "id": "TH17 CELL PROPORTION"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "In diseased IL-10R\u03b1MdelRag1\u2212/\u2212 mice, there\'s a higher amount of pro-IL-1\u03b2 compared to Rag1\u2212/\u2212 mice in colons.)", "entity_type": "UNKNOWN", "id": "DIFFERENCES BETWEEN IL-10R\u0391MDELRAG1\u2212/\u2212 AND RAG1\u2212/\u2212 MICE"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "In diseased IL-10R\u03b1MdelRag1\u2212/\u2212 mice, there\'s a higher amount of pro-IL-1\u03b2 compared to Rag1\u2212/\u2212 mice in colons.)", "entity_type": "UNKNOWN", "id": "IL-1\u0392 PRODUCTION IMPACT"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "IL-10 reduces the levels of pro-IL-1\u03b2 and prevents mature IL-1\u03b2 formation, as shown in cultured Rag1\u2212/\u2212 macrophages.)", "entity_type": "UNKNOWN", "id": "IL-10\'S EFFECT ON PRO-IL-1\u0392 LEVELS"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "IL-10 reduces the levels of pro-IL-1\u03b2 and prevents mature IL-1\u03b2 formation, as shown in cultured Rag1\u2212/\u2212 macrophages.)", "entity_type": "UNKNOWN", "id": "REGULATION BY IL-10 IN M\u03a6S"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "In both Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 mice models, IL-6 plays a more critical role in colitis when compared to IL-10 suppression of LPM\u03c6s\' IL-1\u03b2 production.)", "entity_type": "UNKNOWN", "id": "CYTOKINE INFLUENCE"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "In both Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 mice models, IL-6 plays a more critical role in colitis when compared to IL-10 suppression of LPM\u03c6s\' IL-1\u03b2 production.)", "entity_type": "UNKNOWN", "id": "IL-6 AND DISEASE SEVERITY"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "IL-10 inhibits the production and maturation of pro-inflammatory cytokines like IL-1\u03b2, reducing disease severity in colitis models.)", "entity_type": "UNKNOWN", "id": "ANTI-INFLAMMATORY ACTION"}, {"source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "description": "IL-10 inhibits the production and maturation of pro-inflammatory cytokines like IL-1\u03b2, reducing disease severity in colitis models.)", "entity_type": "UNKNOWN", "id": "IL-10\'S ROLE"}, {"source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "description": "Pre-treatment with IL-10 abrogates induction of activated caspase-1 in Rag1\u2212/\u2212 M\u03c6s.)", "entity_type": "UNKNOWN", "id": "IL-10 TREATMENT"}, {"source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "description": "Anti-IL-10R\u03b1 Ab markedly increases activated caspase-1, indicating normal restraint by IL-10 in Rag1\u2212/\u2212 M\u03c6s.)", "entity_type": "UNKNOWN", "id": "ANTI-IL-10R\u0391 AB"}, {"source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "description": "Increased IL-1 and IL-6 from LPM\u03c6s can impact Th17 cell maturation in the presence of na\u00efve T cells with added TGF\u03b2.)", "entity_type": "UNKNOWN", "id": "LPM\u03a6S"}, {"source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "description": "Blocking IL-6 receptor partially inhibits Th17 formation from Rag1\u2212/\u2212 LPM\u03c6s, but less so from IL-10R\u03b1MdelRag1\u2212/\u2212-derived LPM\u03c6s.)", "entity_type": "UNKNOWN", "id": "TH17 FORMATION"}, {"source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "description": "Blocking IL-1 signaling leads to more substantial inhibition of Th17 development in IL-10R\u03b1Mdel Rag1\u2212/\u2212-derived LPM\u03c6s.)", "entity_type": "UNKNOWN", "id": "TH17 CELL MATURATION/"}, {"source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "description": "Blocking IL-6 receptor substantially inhibits Th17 development in diseased mice, as shown by IL-6R\u2212/\u2212 T cell transfer results.)", "entity_type": "UNKNOWN", "id": "ANTI-IL-6R\u0391 AB"}, {"source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "description": "Blockage of IL-1 receptor pathway leads to more substantial inhibition of Th17 cell development in diseased mice, particularly with IL-10R\u03b1MdelRag1\u2212/\u2212 LPM\u03c6s.)", "entity_type": "UNKNOWN", "id": "TH17 CELL FORMATION/"}, {"entity_type": "COHORT", "description": "The TwinsUK cohort was composed of 1,176 samples including 169 monozygotic (MZ) and 419 dizygotic (DZ) twin pairs used for heritability estimation.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "TWINSUK"}, {"entity_type": "COHORT", "description": "The Netherlands Twin Registry (NTR) cohort was composed of only MZ twins, N=312 individuals, and used to replicate the MZ intraclass correlation coefficient (ICC).", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "NETHERLANDS TWIN REGISTRY (NTR)"}, {"entity_type": "METRIC", "description": "Richness and Shannon diversity differed significantly across cohorts.<SEP>The lack of heritability for alpha diversity indices suggests that they might not be influenced by genetics alone.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae<SEP>chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "ALPHA DIVERSITY"}, {"entity_type": "COHORT", "description": "One of the twenty cohorts included samples from European ancestry, with a total of N=13,266 participants.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "EUROPEAN COHORT"}, {"entity_type": "COHORT", "description": "One cohort comprised individuals from Middle-Eastern ancestry, including N=481 participants.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "MIDDLE-EASTERN COHORT"}, {"entity_type": "COHORT", "description": "A cohort consisted of East Asian individuals with a total of N=811 participants.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "EAST ASIAN COHORT"}, {"entity_type": "COHORT", "description": "One cohort was composed of American Hispanic or Latin individuals, including N=1,097 participants.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "AMERICAN HISPANIC/LATIN COHORT"}, {"entity_type": "COHORT", "description": "The final cohort comprised African-American individuals with a total of N=114 participants.", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "id": "AFRICAN AMERICAN COHORT"}, {"entity_type": "GENE", "description": "LCT locus \u2013 Bifidobacterium is associated with the abundance of Bifidobacterium.<SEP>The LCT gene is involved in cell-type specific expression and identified regions associated with DNA modifications.<SEP>The lactase gene is associated with lactose digestion and metabolism.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d<SEP>chunk-af413a33fae8d27bdb15cd18447dfad4<SEP>chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "LCT"}, {"entity_type": "SNP", "description": "The rs182549 genotype showed a higher abundance of the genus Bifidobacterium in the adult group, but not in the younger group.<SEP>rs182549, a functional variant for LCT gene.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46<SEP>chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "RS182549"}, {"entity_type": "PROTEIN", "description": "A higher abundance of the genus Bifidobacterium was also found to potentially have a protective effect on ulcerative colitis.<SEP>Association of rs182549 with abundance level of Bifidobacterium.<SEP>The age-genotype interaction was significant in the GEM_v12 and GEM_ICHIP subcohorts, both comprising mostly European individuals.", "source_id": "chunk-159d4c797dae6eb825c74067af703228<SEP>chunk-fed46d4ab6d22b3530c4c06a4b7def46<SEP>chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "BIFIDOBACTERIUM"}, {"entity_type": "GENOMIC VARIANT", "description": "A large genomic variant located in the LCT gene and contains several protein-coding genes.", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "LCT 2Q21.3 BLOCK"}, {"entity_type": "PROTEIN", "description": "A functional variant of the LCT gene, which shows a strong association with Bifidobacterium abundance.", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "FUNCTIONAL LCT VARIANT RS4988235"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The COPSAC2010 cohort contributes to heterogeneity in rs182549 association signal.", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "COPSAC2010 COHORT"}, {"entity_type": "DIETARY PATTERN", "description": "The TwinsUK cohort showed genome-wide significant mbTLs for several taxa, including high heritability for Bifidobacterium.", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "TWINSUK COHORT"}, {"entity_type": "GENOMIC VARIANT", "description": "Family Peptococcaceae showed heterogeneity across taxa tested.<SEP>The genomic region rs61841503 affects the abundance of Peptococcaceae, influencing vitamin B12 metabolism and potentially impacting gut inflammation markers.<SEP>These associations suggest that some members of the gut microbiome community might be affected by genetic variants that regulate the absorption and metabolism of vitamins B2 and B12.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46<SEP>chunk-af413a33fae8d27bdb15cd18447dfad4", "id": "PEPTOCOCCACEAE"}, {"entity_type": "DIETARY PATTERN", "description": "The lack of an LCT mbQTL effect in adults was also observed in another Israel cohort in the study (Personalized Nutrition Project (PNP), 481 adults, Beta = \u22120.20 [95%CI: \u22120.61, 0.20]).", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "GEM_V12"}, {"entity_type": "DISEASE", "description": "The cohorts that reported the lowest LCT effect sizes were the two cohorts of Israeli ethnicity volunteered in Israel (GEM_v24, PNP) and a child cohort.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "COPENHAGEN PROSPECTS STUDIES ON ASTHMA IN CHILDHOOD (COPSAC)"}, {"entity_type": "SNP", "description": "This variant was associated with a reduced abundance of the R. torques group and a decreased presence of the R. gnavus group.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "RS35866622"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Ruminococcus sp are specialized in the degradation of complex carbohydrates, thereby supporting a link between genetic variation in the FUT2 gene, levels of mucus glycans and the abundance of this taxa.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "RUMINOCOCCUS TORQUES"}, {"entity_type": "DISEASE", "description": "The tagging NC_000019.9:g.49218060C>T (rs35866622 non-secretor) allele was associated with a reduced abundance of the R. torques group and a decreased presence of the R. gnavus group.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "R2=0.8; D\u2019=0.9"}, {"entity_type": "GENE", "description": "FUT2 encodes the enzyme alpha-1,2-fucosyltransferase, which is responsible for the secretion of fucosylated mucus glycans in the gastrointestinal mucosa.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "FUT2"}, {"entity_type": "SNP", "description": "Individuals homozygous for the stop-codon (rs601338*A/A, non-secretors) do not express ABO antigens on the intestinal mucosa.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "RS601338"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The tagging NC_000019.9:g.49218060C>T (rs35866622 non-secretor) allele was associated with a reduced abundance of the R. torques group and a decreased presence of the R. gnavus group.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "R. TORQUES GROUP"}, {"entity_type": "SNP", "description": "The 9q21 locus includes the genes PCSK5, RFK and GCNT1, of which RFK encodes the enzyme that catalyzes the phosphorylation of riboflavin (vitamin B2) and GCNT1 encodes a glycosyltransferase involved in biosynthesis of mucin.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "RS602075"}, {"entity_type": "GENE", "description": "These products play major roles in the host-microbiota interactions within the intestine, where they are used by bacteria for their metabolism and involved in the regulation of the host immune defense.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "RFK"}, {"entity_type": "SNP", "description": "This association signal affects the abundance of the heritable family Peptostreptococcaceae, which is located in the CUBN gene, the receptor for the complexes of cobalamin (vitamin B12) with gastric intrinsic factor.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "RS61841503"}, {"entity_type": "GENE", "description": "CUBN is expressed in the kidneys and the intestinal epithelium and is associated with B12-deficient anemia and albuminuria. Cobalamin is required for host-microbial interactions, and supplementation with cobalamin induced a substantial shift in the microbiota composition of an in vitro colon model.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "CUBN"}, {"entity_type": "SNP", "description": "The genomic variant rs7574352 was associated with the family Peptococcaceae, a taxon negatively associated with gut inflammation markers and ulcerative colitis.<SEP>The strongest evidence for association was seen for a block of 10 SNPs associated with the family Peptococcaceae, a taxon negatively associated with stool levels of gut inflammation markers.<SEP>The strongest evidence for association was seen for a block of 10 SNPs associated with the family Peptococcaceae, a taxon negatively associated with stool levels of the gut inflammation markers chromogranin A and calprotectin.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46<SEP>chunk-938d8bf2e56584de9dd556be5eb70964", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 42}]", "id": "RS7574352"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The tagging NC_000019.9:g.49218060C>T (rs35866622 non-secretor) allele was associated with a reduced abundance of the R. torques group and a decreased presence of the R. gnavus group.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "RS601338*RS35866622"}, {"entity_type": "GENOMIC REGION", "description": "This association signal affects the abundance of both Peptococcaceae and Peptostreptococcaceae, which are families of bacteria involved in vitamin B12 metabolism.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "RS61841503*RS7574352"}, {"entity_type": "BACTERIAL FAMILIES", "description": "The genomic regions rs61841503 and rs7574352 affect the abundance of these bacterial families, which are involved in vitamin B12 metabolism.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "PEPTOCOCCACEAE*PEPTOSTREPTOCOCCACEAE"}, {"entity_type": "GENOMIC REGION*ASSOCIATION SIGNAL", "description": "The genomic region rs61841503 affects the abundance of Peptostreptococcaceae, which are involved in vitamin B12 metabolism and potentially related to changes in gut inflammation markers.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "RS61841503*PEPTOSTREPTOCOCCACEAE"}, {"entity_type": "ASSOCIATION SIGNAL*STOOL INFLAMMATION MARKERS", "description": "The genomic region rs7574352 has a strong association with the abundance of Peptococcaceae bacteria, which can negatively impact stool inflammation markers.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "RS7574352*PEPTOCOCCACEAE"}, {"entity_type": "DISEASE*RS61841503*ASSOCIATION", "description": "The genomic region rs61841503 has been associated with chronic obstructive pulmonary disease (COPD) risk.", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "id": "COPD"}, {"entity_type": "DIETARY PATTERN", "description": "A block of 10 SNPs including rs7574352 was associated with the family Peptococcaceae, suggesting a potential dietary pattern related to gut health.<SEP>The association of rs7574352 with the family Peptococcaceae suggests a dietary pattern that might influence gut health.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "FAMILY PEPTOCOCCACEAE"}, {"entity_type": "PROTEIN", "description": "The LLD phenotype for chromogranin A is negatively correlated with the presence of the association block for rs7574352.<SEP>rs7574352 was negatively correlated with chromogranin A levels, possibly indicating its role in inflammatory responses.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "CHROMOGRANIN A"}, {"entity_type": "PROTEIN", "description": "The LLD phenotype for calprotectin is negatively correlated with the presence of the association block for rs7574352.<SEP>The association of rs7574352 with calprotectin suggests a link between this variant and gut inflammation markers.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "CALPROTECTIN"}, {"entity_type": "DISEASE", "description": "rs7574352 was associated with ulcerative colitis, a disease linked to gut inflammation markers like chromogranin A and calprotectin.<SEP>rs7574352 was associated with ulcerative colitis, highlighting its potential role in inflammatory bowel diseases.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 42}]", "id": "ULCERATIVE COLITIS"}, {"entity_type": "GENE", "description": "IRF1 and STAT1 expression changes are associated with T1D susceptibility.<SEP>The association block of 10 SNPs including rs7574352 is located near the gene IRF1 involved in insulin resistance and type 2 diabetes susceptibility.<SEP>The genomic region including rs7574352 is near the IRF1 gene, which may influence metabolism and inflammatory processes.<SEP>The expression levels of this protein were significantly affected by the rs4788084[T] SNP in CD4+ T cells.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b<SEP>chunk-1b40d071abac0ba432a9fb0096796f45<SEP>chunk-938d8bf2e56584de9dd556be5eb70964", "id": "IRF1"}, {"entity_type": "DISEASE", "description": "rs7574352 is associated with type 2 diabetes, linked to IRF1 gene and insulin resistance.<SEP>rs7574352 was linked with type 2 diabetes, underscoring its potential role in genetic predispositions for metabolic diseases.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 42}]", "id": "TYPE 2 DIABETES"}, {"entity_type": "ANALYSIS METHOD", "description": "GSEA analysis on the genes of mbTLs suggests enrichment for expression in small intestine, brain regions.<SEP>Gene Set Enrichment Analysis (GSEA) revealed enrichment of genes related to small intestine and brain regions associated with rs7574352.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "GSEA"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Small intestine is indicated as a primary physiological process by GSEA.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "TERMINAL ILEUM"}, {"entity_type": "BRAIN REGIONS", "description": "Enrichment for expression in these brain regions indicates potential neurological processes linked to mbTLs.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "SUBSTANIA NIGRA AND PUTAMEN BASAL GANGLIA"}, {"entity_type": "ANALYSIS METHOD", "description": "FUMA analysis of 20 mbQTL loci returns positional and eQTL genes related to small intestine, brain regions, and insulin resistance.<SEP>Fine Mapping using the Unified Meta-Analytic approach (FUMA) on rs7574352 suggested associations with various complex traits and diseases.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "FUMA"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "GSEA analysis shows enrichment for Bifidobacterium abundance linked to various complex traits and diseases.<SEP>The analysis suggests that the presence of Bifidobacterium might be influenced by genetic factors related to rs7574352.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "BIFIDOBACTERIUM ABUNDANCE"}, {"entity_type": "STATISTICAL SIGNIFICANCE", "description": "GWAS ATLAS identified SNPs associated with multiple dietary and metabolic phenotypes below the 5\u00d710\u22128 threshold for significance.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "5\u00d710\u22128 SIGNIFICANCE THRESHOLD"}, {"entity_type": "GENOMIC VARIANT", "description": "The most pleiotropic variant, rs182549 is associated with obesity, T2DM diagnosis and several nutritional phenotypes in UK Biobank cohort.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "RS182549 (LCT)"}, {"entity_type": "CONDITION", "description": "The genetic variation at rs7574352 might predispose individuals to lactose intolerance through its impact on gut microbiota and metabolism.<SEP>rs182549 predisposes individuals to lactose intolerance which is negatively correlated with obesity and positively associated with T2DM.<SEP>The results of the study provided insights into mechanisms related to lactose intolerance, linking DNA modifications with LCT expression.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d<SEP>chunk-938d8bf2e56584de9dd556be5eb70964", "id": "LACTOSE INTOLERANCE"}, {"entity_type": "GENE", "description": "The functional variant rs4988235 of the LCT gene is associated with glycemic variability, indicating a link between galactose metabolism and insulin sensitivity.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "LCT (GALACTOSE-1-PHOSPHATE URIDYL TRANSFERASE)"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The GSEA results highlight the involvement of small intestinal processes in mechanisms related to rs7574352.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "SMALL INTESTINE"}, {"entity_type": "BRAIN REGIONS", "description": "GSEA indicated enrichment for gene expression in these brain regions linked with rs7574352, possibly implicating neurological pathways.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "SUBSTANTIA NIGRA AND PUTAMEN BASAL GANGLIA"}, {"entity_type": "PATHWAY ENRICHMENT", "description": "The GSEA results showed significant enrichment in biological pathways related to small intestine functions and neurobiological processes linked with rs7574352.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "GSEA ENRICHMENT FOR SMALL INTESTINE AND BRAIN REGIONS"}, {"entity_type": "OUTCOME MEASURE", "description": "rs7574352 predisposes individuals to T2DM, which is indicated by a higher likelihood of diagnosis in populations carrying the variant.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "T2DM DIAGNOSIS"}, {"entity_type": "PHYSIOLOGICAL PARAMETER", "description": "rs7574352 may affect glycemic variability, which is a key indicator of insulin responsiveness in metabolic health.", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "id": "GLYCEMIC VARIABILITY"}, {"entity_type": "MICROBIAL TAXA", "description": "The gut microbial taxa were analyzed using Mendelian Randomization to identify potential causal links with phenotypes.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "GUT MICROBIAL TAXA"}, {"entity_type": "DISEASE", "description": "We focused on diseases including autoimmune, cardiovascular, metabolic and psychiatric in bi-directional two-sample MR analyses.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "DISEASES"}, {"entity_type": "AUTOIMMUNE DISEASE", "description": "Autoimmune diseases were part of the group for phenotypes analyzed using Mendelian Randomization.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "AUTOIMMUNE"}, {"entity_type": "CARDIOVASCULAR DISEASE", "description": "Cardiovascular disease was also included in the group of diseases analyzed.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "CARDIOVASCULAR"}, {"entity_type": "METABOLIC DISEASE", "description": "We performed bi-directional two-sample MR analyses on metabolic diseases along with autoimmune and psychiatric conditions.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "METABOLIC"}, {"entity_type": "PSYCHIATRIC DISORDER", "description": "The analysis was extended to include psychiatric disorders as part of the phenotypes examined.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "PSYCHIATRIC"}, {"entity_type": "NUTRITIONAL PHENOTYPE", "description": "Nutritional phenotypes were also a focus in our bi-directional two-sample MR analyses using Mendelian Randomization.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "NUTRITIONAL PHENOTYPES"}, {"entity_type": "GENETICS", "description": "Host genetics had an impact on microbiome composition, and this was considered when performing causality estimation using the MR analysis.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "HOST GENETICS"}, {"entity_type": "MICROBIOME COMPOSITION", "description": "The composition of the microbiome was influenced by host genetics as part of the study parameters.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "MICROBIOME COMPOSITION"}, {"entity_type": "GENETIC VARIANTS IMPACT", "description": "Our analysis took into account that genetic variants had a limited effect on microbial taxa variation in microbiome composition.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "LIMITED IMPACT OF GENETIC VARIANTS ON MICROBIAL TAXA VARIABILITY"}, {"entity_type": "CAUSALITY ESTIMATION", "description": "We conducted several sensitivity analyses to estimate causality using the MR analysis, ensuring to exclude any results confounded by pleiotropy.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "CAUSALITY ESTIMATION"}, {"entity_type": "PLEIOTROPY", "description": "We excluded results that showed evidence of being influenced by pleiotropy when estimating causality in our MR analysis.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "PLEIOTROPY"}, {"entity_type": "SNPS", "description": "Only pairs supported by three or more Single Nucleotide Polymorphisms (SNPs) were considered for the final analysis results.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "RESULTS WITH THREE OR MORE SNPS"}, {"entity_type": "CAUSAL RELATIONSHIPS", "description": "We did not find any evidence of causal relationships between microbiome taxa and dietary preferences based on our criteria.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "CAUSAL RELATIONSHIPS"}, {"entity_type": "PHYLUM", "description": "A higher abundance of the class Actinobacteria may have a protective effect on ulcerative colitis.<SEP>Decreases in Actinobacteria phylum were revealed by LEfSe analysis.<SEP>There\'s a decrease in the phylum Actinobacteria at baseline, suggesting an alteration in bacterial composition.", "source_id": "chunk-159d4c797dae6eb825c74067af703228<SEP>chunk-e025ebd5580ff5f1a76bedbcaed2a0b6<SEP>chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "ACTINOBACTERIA"}, {"entity_type": "MICROBIAL TAXA", "description": "The higher abundance of the Oxalobacteraceae family may be protective against rheumatoid arthritis based on our analysis.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "OXALOBACTERACEAE FAMILY"}, {"entity_type": "AUTOIMMUNE DISEASES", "description": "In our analyses, we found potential protective effects of certain microbial taxa on specific autoimmune conditions like ulcerative colitis.", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "id": "ULCERATIVE COLITIS AND AUTOIMMUNE CONDITIONS"}, {"entity_type": "DISEASE", "description": "The analysis revealed a protective effect potentially linked to the abundance of certain bacterial taxa against rheumatoid arthritis.<SEP>rs1893217 is associated with rheumatoid arthritis.", "source_id": "chunk-159d4c797dae6eb825c74067af703228<SEP>chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "RHEUMATOID ARTHRITIS"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "The SNP-based heritability calculated from mbQTL summary statistics using linkage disequilibrium (LD) score regression showed two bacterial taxa, genus Ruminiclostridium 9 and family Peptostreptococcaeae, passing the significance threshold given the number of 211 taxa tested.)", "entity_type": "UNKNOWN", "id": "LINKAGE DISEQUILIBRIUM (LD) SCORE REGRESSION"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "The SNP-based heritability calculated from mbQTL summary statistics using linkage disequilibrium (LD) score regression showed two bacterial taxa, genus Ruminiclostridium 9 and family Peptostreptococcaeae, passing the significance threshold given the number of 211 taxa tested.)", "entity_type": "UNKNOWN", "id": "SNP-BASED HERITABILITY"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "The estimation of heritability and SNP-based heritability showed significant correlation across the tested taxa, with correlation coefficient R=0.244.)", "entity_type": "UNKNOWN", "id": "GENUS RUMINICLOSTRIDIUM 9&FAMILY PEPTOSTREPTOCOCCAEAE"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "In TwinsUK, heritability estimation was performed using the ACE model, which includes additive genetic variance (A), shared environmental factors (C), and non-shared factors plus error (E).)", "entity_type": "UNKNOWN", "id": "TWINSUK COHORT&MOLECULAR INTERACTION"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "The Netherlands Twin Registry cohort was used to replicate the MZ intraclass correlation coefficient (ICC) for heritability estimation.)", "entity_type": "UNKNOWN", "id": "NETHERLANDS TWIN REGISTRY (NTR)&HERITABILITY ESTIMATION"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "The Netherlands Twin Registry cohort was used to replicate the MZ intraclass correlation coefficient (ICC) for heritability estimation.)", "entity_type": "UNKNOWN", "id": "REPLICATION"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "SNP-based heritability estimation from mbQTL summary statistics was performed using linkage disequilibrium (LD) score regression on the TwinsUK cohort.)", "entity_type": "UNKNOWN", "id": "TWINSUK COHORT&SNP-BASED HERITABILITY"}, {"source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "description": "The NTR cohort provided replication for the SNP-based heritability estimation done on the TwinsUK cohort.)", "entity_type": "UNKNOWN", "id": "NETHERLANDS TWIN REGISTRY (NTR)&SNP-BASED HERITABILITY"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "An LCT locus associated with abundance of Bifidobacterium.)", "entity_type": "UNKNOWN", "id": "LCT LOCUS"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "rs182549 is a proxy for the functional LCT variant.)", "entity_type": "UNKNOWN", "id": "FUNCTIONAL LCT VARIANT"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea", "description": "The COPSAC2010 cohort contributed to the heterogeneity in rs182549 association signal.)<SEP>The variation or diversity found among the data due to different factors like genetic variants or environmental influences.", "entity_type": "METABOLIC PATHWAY", "id": "HETEROGENEITY"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "Age shows an interaction with rs182549, affecting Bifidobacterium abundance differently in different age groups.)", "entity_type": "UNKNOWN", "id": "AGE AND GENOTYPE INTERACTION"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "Age shows an interaction with rs182549, affecting Bifidobacterium abundance differently in different age groups.)", "entity_type": "UNKNOWN", "id": "AGE& RS182549"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "Both ethnicity and rs182549 are related to the abundance of Bifidobacterium.)", "entity_type": "UNKNOWN", "id": "RS182549& BIFIDOBACTERIUM"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "The LCT locus, associated with abundance of Bifidobacterium through mbQTL mapping.)", "entity_type": "UNKNOWN", "id": "LCT LOCUS - BIFIDOBACTERIUM"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "The LCT locus, associated with abundance of Bifidobacterium through mbQTL mapping.)", "entity_type": "UNKNOWN", "id": "MBQTL"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "mbBTL mapping identified loci associated with the presence or absence of specific bacterial taxa in the gut microbiota.)", "entity_type": "UNKNOWN", "id": "MBBTL"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "mbBTL mapping identified loci associated with the presence or absence of specific bacterial taxa in the gut microbiota.)", "entity_type": "UNKNOWN", "id": "PRESENCE/ABSENCE OF BACTERIAL TAXA"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "No significant hits were found in the meta-GWAS analysis for alpha diversity indices, indicating low genetic influence.)", "entity_type": "UNKNOWN", "id": "META-GWAS"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "Cochran\'s Q test showed evidence of heterogeneity across cohorts related to rs182549 association with Bifidobacterium abundance.)", "entity_type": "UNKNOWN", "id": "COCHRAN\u2019S Q"}, {"source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "description": "Cochran\'s Q test showed evidence of heterogeneity across cohorts related to rs182549 association with Bifidobacterium abundance.)", "entity_type": "UNKNOWN", "id": "RS182549 HETEROGENEITY"}, {"source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "description": "SNP rs182549, associated with Bifidobacterium abundance, has been positively linked to T2DM diagnosis and a family history of the disease.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 34}]", "id": "LCT (BIFIDOBACTERIUM)"}, {"source_id": "chunk-174831b97ed170039b374ba254d4f506<SEP>chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9<SEP>chunk-938d8bf2e56584de9dd556be5eb70964", "description": "SNP rs182549, associated with Bifidobacterium abundance, has been positively linked to T2DM diagnosis and a family history of the disease.)<SEP>T2DM group was designated as one of the case groups in relation to blood glucose levels and genetic variants.<SEP>Type 2 Diabetes Mellitus is the ultimate disease state associated with genetic variants and abnormal blood glucose levels.<SEP>Type 2 Diabetes Mellitus is a metabolic disorder characterized by high blood sugar levels due to insulin resistance or insufficient insulin production.", "entity_type": "DISEASE/INFLAMMATION", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 47}]", "id": "T2DM"}, {"source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "description": "The secretor allele at the FUT1/FUT2 locus is associated with fish intake and height in dietary patterns.)", "entity_type": "UNKNOWN", "id": "FISH INTAKE HEIGHT"}, {"source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "description": "The secretor allele at the FUT1/FUT2 locus is associated with fish intake and height in dietary patterns.)", "entity_type": "UNKNOWN", "id": "FUT1/FUT2 LOCUS"}, {"source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "description": "Functional LCT variant rs4988235 is associated with glycemic variability, an indicator of insulin sensitivity.)", "entity_type": "UNKNOWN", "id": "RS4988235 VARIANT"}, {"source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "description": "A nominally significant correlation between Bifidobacterium abundance and raw vegetable intake was observed before multiple testing correction.)", "entity_type": "UNKNOWN", "id": "RAW VEGETABLE INTAKE"}, {"source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "description": "The secretor allele at the FUT1/FUT2 locus is negatively associated with risks of cholelithiasis.)", "entity_type": "UNKNOWN", "id": "CHOLELITHIASIS"}, {"source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "description": "The secretor allele at the FUT1/FUT2 locus is negatively associated with risks of cholelithiasis.)", "entity_type": "UNKNOWN", "id": "RS35866622, FUT1/FUT2 LOCUS"}, {"source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "description": "Gene-based PheWAS enrichment analysis indicated strong links between the LCT locus and metabolic traits, including BMI.)", "entity_type": "UNKNOWN", "id": "METABOLIC TRAITS"}, {"source_id": "chunk-159d4c797dae6eb825c74067af703228", "description": "The limited effect of genetic variants on microbial taxa variability was considered in sensitivity analyses during our causality estimation process.)", "entity_type": "UNKNOWN", "id": "LIMITED IMPACT OF GENETIC VARIANTS"}, {"source_id": "chunk-159d4c797dae6eb825c74067af703228", "description": "The limited effect of genetic variants on microbial taxa variability was considered in sensitivity analyses during our causality estimation process.)", "entity_type": "UNKNOWN", "id": "MICROBIAL TAXA VARIABILITY"}, {"source_id": "chunk-159d4c797dae6eb825c74067af703228", "description": "Causal relationships were only considered if supported by three or more Single Nucleotide Polymorphisms (SNPs).)", "entity_type": "UNKNOWN", "id": "CAUSAL RELATIONSHIPS IDENTIFIED"}, {"source_id": "chunk-159d4c797dae6eb825c74067af703228", "description": "Causal relationships were only considered if supported by three or more Single Nucleotide Polymorphisms (SNPs).)", "entity_type": "UNKNOWN", "id": "THREE OR MORE SNPS"}, {"source_id": "chunk-159d4c797dae6eb825c74067af703228", "description": "Actinobacteria and the genus Bifidobacterium may have a protective effect against ulcerative colitis.)", "entity_type": "UNKNOWN", "id": "HIGHER ABUNDANCE OF ACTINOBACTERIA AND BIFIDOBACTERIUM"}, {"source_id": "chunk-159d4c797dae6eb825c74067af703228", "description": "Actinobacteria and the genus Bifidobacterium may have a protective effect against ulcerative colitis.)", "entity_type": "UNKNOWN", "id": "PROTECTIVE EFFECT ON ULCERATIVE COLITIS"}, {"source_id": "chunk-159d4c797dae6eb825c74067af703228", "description": "The Oxalobacteraceae family\'s higher abundance may be protective against rheumatoid arthritis, according to our study results.)", "entity_type": "UNKNOWN", "id": "HIGHER ABUNDANCE OF OXALOBACTERACEAE FAMILY"}, {"source_id": "chunk-159d4c797dae6eb825c74067af703228", "description": "The Oxalobacteraceae family\'s higher abundance may be protective against rheumatoid arthritis, according to our study results.)", "entity_type": "UNKNOWN", "id": "PROTECTIVE EFFECT ON RHEUMATOID ARTHRITIS"}, {"entity_type": "GENOMIC VARIANT", "description": "LP is a genetic variant with rs4988235 genotype, which affects various cardiovascular health biomarkers.<SEP>Minor allele frequencies for rs4988235 within controls were observed in different populations.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5<SEP>chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "RS4988235"}, {"entity_type": "SAMPLE SIZE", "description": "The study included 12,722 participants, with an average age of 52 years.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "12,722 PARTICIPANTS"}, {"entity_type": "NUTRIENT", "description": "Average milk intake was 162 g/day and non-consumption was rare (7.1%) in EPIC-InterAct.<SEP>Increased intake of milk was positively associated with one additional LP allele.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "MILK INTAKE"}, {"entity_type": "FOOD", "description": "Consumption of dairy products varied between countries, with lower intake observed in France.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "DAIRY PRODUCTS"}, {"entity_type": "COUNTRY", "description": "In Italy, the prevalence of homozygous LP variant was 7.4%, whereas it reached up to 53.9% in Sweden.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "ITALY"}, {"entity_type": "METABOLIC PATHWAY", "description": "rs4988235 genotype influenced levels of total fats, with a moderate increase observed for both C15:0 and C17:0.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "C15:0 AND C17:0"}, {"entity_type": "DIETARY PATTERN", "description": "Consumption of potatoes was higher in participants with a higher number of LP alleles.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "POTATOES"}, {"entity_type": "DIETARY PATTERN", "description": "Participants with LP variant showed lower consumption of cereals compared to those without.<SEP>The term \'cereal\' refers to a broad group of foods that can include grains like wheat, barley, and oats.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "CEREALS"}, {"entity_type": "DIETARY PATTERN", "description": "Fruit intake was observed to be influenced by the presence of the LP variant, indicating potential genetic-diet interactions.<SEP>In contrast, fruit intake was higher among participants with LP variant.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "FRUITS"}, {"entity_type": "FOOD", "description": "Higher intake of poultry was associated with the presence of rs4988235 genotype.<SEP>The consumption of poultry showed a specific pattern among participants with certain genotypes like rs4988235.<SEP>In 10 (11.9%) of 84 domestically acquired infections, patients reported eating fresh poultry other than chicken and turkey compared with 4 (9.5%) of 42 infections in persons with travel-related infections.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9<SEP>chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "POULTRY"}, {"entity_type": "FOOD", "description": "These beverages were found to have varying associations with the LP variant, potentially affecting health outcomes.<SEP>rs4988235 was linked to higher consumption of non-alcoholic beverages.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "NON-ALCOHOLIC BEVERAGES"}, {"entity_type": "FOOD", "description": "Consumption of wine showed a modest negative association with rs4988235 genotype.<SEP>Wine intake was noted in relation to rs4988235 genotype, suggesting complex interactions between genetics and diet.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "WINE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "rs4988235 did not show significant association with HbA1c levels at baseline.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "HBA1C LEVELS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Dietary intakes were associated with rs4988235, including potatoes, cereals, fruits, poultry, non-alcoholic beverages, and wine.<SEP>Dietary patterns associated with rs4988235 variant also affected the risk of developing diabetes.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "DIABETES RISK FACTORS"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant plays a role in cardiovascular health through its influence on metabolic pathways and diet.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "LP (RS4988235 T/T GENOTYPE)"}, {"entity_type": "STATISTICAL MEASURE", "description": "The principal components method was used to adjust the analysis for genetic structure, reducing bias.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "PRINCIPAL COMPONENTS"}, {"entity_type": "LOCATION FACTOR", "description": "The EPIC-InterAct study involved multiple centers across different countries, contributing to the sample size and diversity.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "STUDY CENTRE"}, {"entity_type": "CLINICAL OUTCOME", "description": "A combined outcome was used for milk and milk beverages intake to enhance statistical power.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "COMPOSITE ENDPOINT"}, {"entity_type": "STATISTICAL MEASURE", "description": "A measure of goodness-of-fit indicating the strength of association between variables in regression analysis.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "F STATISTIC"}, {"entity_type": "NUTRIENT", "description": "Increased potato consumption was associated with higher potassium intake, impacting cardiovascular health.", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "id": "POTASSIUM INTAKE"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "The LP genetic variant was positively related to milk consumption, affecting the F-statistic for predicting milk intake.)", "entity_type": "UNKNOWN", "id": "RS4988235 AND MILK INTAKE"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "There was a minor association between rs4988235 and increased levels of saturated fats (C15:0 and C17:0).)", "entity_type": "UNKNOWN", "id": "RS4988235 AND PLASMA C15:0/C17:0"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "Consumption of potatoes was higher in individuals with the LP genetic variant.)", "entity_type": "UNKNOWN", "id": "RS4988235 AND POTATO INTAKE"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "The intake of cereals was inversely related to the presence of rs4988235 genotype.)", "entity_type": "UNKNOWN", "id": "RS4988235 AND CEREAL INTAKE"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "Fruit consumption showed a positive association with LP variant.)", "entity_type": "UNKNOWN", "id": "RS4988235 AND FRUIT INTAKE"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "Poultry intake was associated with the presence of rs4988235 genotype.)", "entity_type": "UNKNOWN", "id": "FOOD"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "Poultry intake was associated with the presence of rs4988235 genotype.)", "entity_type": "UNKNOWN", "id": "RS4988235 AND POULTRY INTAKE"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "The consumption of non-alcoholic beverages was higher in individuals carrying the LP variant.)", "entity_type": "UNKNOWN", "id": "RS4988235 AND NON-ALCOHOLIC BEVERAGE INTAKE"}, {"source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "description": "Wine consumption showed a negative relationship with rs4988235 genotype.)", "entity_type": "UNKNOWN", "id": "RS4988235 AND WINE INTAKE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "There were no group differences in social class or maternal education.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "SOCIAL CLASS"}, {"entity_type": "FORMAL EDUCATION LEVEL", "description": "Case mothers had fewer years of formal education compared to control mothers.<SEP>There were no group differences in social class or maternal education.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914<SEP>chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "MATERNAL EDUCATION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "There were no group differences in perceived stress,", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "PERCEIVED STRESS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "There were no group differences in depression,", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "DEPRESSION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "There were no group differences in nutritional status,", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "NUTRITIONAL STATUS"}, {"entity_type": "NUTRIENT", "description": "About 30% of both groups were poorly nourished based on their dietary caloric intake.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "DIETARY CALORIC INTAKE"}, {"entity_type": "NUTRIENT", "description": "More than 70% did not meet the AI for dietary choline intake.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "AI FOR DIETARY CHOLINE INTAKE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Both groups smoked about 1/4 pack/day on average,", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "CIGARETTE SMOKING"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Illicit drug use was rare, none of the women reported using cocaine, heroin, or methaqualone.<SEP>Use of illicit drugs other than marijuana was rare.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "ILLICIT DRUG USE"}, {"entity_type": "DRUG", "description": "Although women who used methamphetamine were excluded at time of recruitment, 4 reported using it later in pregnancy (2 choline, 2 placebo).", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "METHAMPHETAMINE USE"}, {"entity_type": "DISEASE", "description": "There were no group differences in infant age at the 6.5- and 12-month assessments.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "INFANT AGE"}, {"entity_type": "DISEASE", "description": "There were no group differences in weeks gestation at time of booking,", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "WEEKS GESTATION AT TIME OF BOOKING"}, {"entity_type": "DISEASE", "description": "There were no group differences in duration of choline supplementation.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "RANDOMIZATION DURATION"}, {"entity_type": "DISEASE", "description": "Infants differed on sex, with more girls in the choline and more boys in the placebo arm.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "SEX OF INFANT"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Women continued to drink heavily across pregnancy (\u22488-9 drinks/occasion) but reduced their frequency.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "ALCOHOL CONSUMPTION ACROSS PREGNANCY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Both groups reduced smoking frequency to about 1\u20132 days/week.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "CIGARETTE SMOKING FREQUENCY"}, {"entity_type": "DRUG", "description": "Although women who used methamphetamine were excluded at time of recruitment, 4 reported using it later in pregnancy (2 choline, 2 placebo).", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "METHAMPHETAMINE USE LATER IN PREGNANCY"}, {"entity_type": "DISEASE", "description": "Weeks gestation at randomization ranged from 8.6\u201326.0 (Median=20.0)).(entity", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "INFANT GA AT TIME OF BOOKING AND RANDOMIZATION"}, {"entity_type": "DISEASE", "description": "More girls were born to mothers in the choline group.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "SEX OF INFANTS IN CHOLINE ARM"}, {"entity_type": "DISEASE", "description": "Assessments were conducted at 6.5 and 12 months.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "NUMBER OF INFANT ASSESSMENTS"}, {"entity_type": "OUTCOME MEASURE", "description": "Choline-treated infants showed a significantly greater increase in percent correct responses across three sessions than the placebo group.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "INFANTS\' EYEBLINK CONDITIONING SCORES"}, {"entity_type": "RELATIONSHIP", "description": "There was a clear linear relation between maternal adherence to choline supplementation and eyeblink conditioning, with better adherence leading to higher conditioning scores.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "LINEAR RELATION BETWEEN ADHERENCE AND CONDITIONING"}, {"entity_type": "OUTCOME MEASURE", "description": "Birth measures showed marked underpercentile values across all groups, reflecting fetal alcohol growth restriction.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "Z-SCORES AND PERCENTILES FOR BIRTH SIZE MEASURES"}, {"entity_type": "OUTCOME MEASURE", "description": "Infants in the choline group demonstrated significant catch-up growth in weight and head circumference by 6.5 months, which continued at 12 months.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "CATCH-UP GROWTH IN CHOLINE GROUP"}, {"entity_type": "DISEASE", "description": "Choline-treated infants had a higher incidence of Fetal Alcohol Spectrum Disorders (8 infants) compared to the placebo group (5 infants).", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "FASD DIAGNOSIS CRITERIA"}, {"entity_type": "STATISTICAL MEASURE", "description": "Indicated moderate-to-large effect size for FTII processing speed measure in choline treatment arm.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "COHEN\'S D MEASURE"}, {"entity_type": "DISEASE DISTRIBUTION", "description": "In the placebo group, 5 infants met criteria for FAS and 2 for PFAS compared to 8 infants with FAS and 2 with PFAS in the choline treatment group.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "FAS DIAGNOSIS INCIDENCE BY ARM"}, {"entity_type": "OUTLYING DATA POINT", "description": "One maternal outlier was identified with very poor adherence and high percentage correct responses, which had a bivariate effect on the analysis.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "METHAMPHETAMINE OUTLIER"}, {"entity_type": "STATISTICAL MEASURE", "description": "Mahalanobis distance test identified one bivariate outlier that influenced the linear relation between adherence and eyeblink conditioning scores.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "STATISTICAL TEST FOR ADHERENCE-CONDITIONING RELATION"}, {"entity_type": "OUTCOME MEASURE", "description": "Infants in the choline group showed significantly greater increases across age periods compared to the placebo group for weight and head circumference.", "source_id": "chunk-a9cd8765fac3ebb933c3fade561bc42e", "id": "SIGNIFICANCE OF WEIGHT AND HEAD CIRCUMFERENCE PERCENTILES IN CHOLINE GROUP"}, {"entity_type": "DISEASE", "description": "Fetal alcohol spectrum disorder is a condition that affects infants and can be diagnosed based on specific criteria.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "FASD"}, {"entity_type": "NUTRIENT", "description": "A nutrient important for pregnancy health, which has an adequate intake standard (>450 mg/day).<SEP>An essential nutrient that plays a critical role in pregnancy and fetal development.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "CHOLINE"}, {"entity_type": "GENOMIC VARIANT", "description": "A genetic variant affecting the enzyme PEMT, reducing endogenous choline synthesis.<SEP>This genomic variant in the enzyme PEMT reduces endogenous choline synthesis.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "RS12325817 VARIANT"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The enzyme PEMT interacts with choline metabolism during pregnancy, influencing dietary intake standards and outcomes.<SEP>The enzyme that affects choline metabolism and interactions with maternal diet during pregnancy.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "PEMT"}, {"entity_type": "SUBJECT POPULATION", "description": "A demographic group composed of individuals who are infants, typically ranging in age from newborns up to around 2 years old.<SEP>The infants assessed in this study were part of the research sample that met specific criteria for diagnosis or nutritional status.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11<SEP>chunk-b930cdd3706af881cfa771b341af097d", "id": "INFANTS"}, {"entity_type": "DIETARY PATTERN", "description": "A group given a placebo, one component being their usual dietary intake which did not include choline supplementation.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "PLACEBO ARM"}, {"entity_type": "DIETARY PATTERN", "description": "A group that received choline supplements as part of their diet during pregnancy.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "CHOLINE-TREATED GROUP"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The physiological process involving the intake and metabolism of choline in mothers during pregnancy.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "MATERNAL DIETARY CHOLINE STATUS"}, {"entity_type": "NUTRIENT INTAKE", "description": "Mothers whose intake of choline was adequate (>450 mg/day) during pregnancy.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "CHOLINE INTAKE DURING PREGNANCY"}, {"entity_type": "DIETARY PATTERN", "description": "The Upper Limit for dietary choline, which is not exceeded by any infants in this study.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "UL (UPPER LIMIT)"}, {"entity_type": "TRANSCRIPT", "description": "All the infants assessed were part of this research study, and their outcomes were analyzed based on various criteria.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "62 INFANTS"}, {"entity_type": "TRANSCRIPT", "description": "The transcripts of the infants involved in this study were used to assess various health outcomes related to their exposure to choline.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "INFANTS\' HEALTH OUTCOMES"}, {"entity_type": "DIETARY PATTERN", "description": "This refers to the typical dietary intake of participants in the placebo arm, which was not supplemented with choline.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "PLACEBO GROUP INTAKE"}, {"entity_type": "DIETARY PATTERN", "description": "The dietary pattern followed by this group included additional choline supplements during pregnancy, which influenced maternal and fetal outcomes.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "CHOLINE-SUPPLEMENTED GROUP INTAKE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "This encompasses the physical health conditions related to a mother\'s diet and its impact on her ability to meet the essential nutrient requirements.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "MATERNAL NUTRITIONAL STATUS"}, {"entity_type": "NUTRIENT INTAKE", "description": "The standard amount of choline that is considered adequate for pregnant women, which was not achieved by all in this study.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "CHOLINE INTAKE THRESHOLD"}, {"entity_type": "DIETARY PATTERN", "description": "A dietary limit set to prevent potential harmful effects from excessive intake of a specific nutrient; here referring to choline.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "UPPER LIMIT (UL)"}, {"entity_type": "TRANSCRIPT", "description": "The study focused on assessing various developmental milestones in infants related to their exposure to choline.", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "id": "INFANTS\' DEVELOPMENT STAGES"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "The diagnosis of FASD was applied to 32.3% of infants in the choline group compared to 22.6% in the placebo group.)", "entity_type": "UNKNOWN", "id": "FASD DIAGNOSIS"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "Influence of maternal dietary choline status on endogenous choline synthesis when carrying rs12325817 variant.)", "entity_type": "UNKNOWN", "id": "MATERNAL PEMT ENZYME INTERACTION WITH CHOLINE INTAKE"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "Differences in infants\' outcomes between those given choline supplements during their mothers\' pregnancies versus placebo.)", "entity_type": "UNKNOWN", "id": "CHOLINE SUPPLEMENTATION IN PREGNANCY"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "Differences in infants\' outcomes between those given choline supplements during their mothers\' pregnancies versus placebo.)", "entity_type": "UNKNOWN", "id": "INFANTS\' NUTRITIONAL AND HEALTH OUTCOMES"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "The impact of different dietary standards for choline on the health and development of infants.)", "entity_type": "UNKNOWN", "id": "INSUFFICIENT VS. ADEQUATE MATERNAL CHOLINE INTAKE DURING PREGNANCY"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "The relationship between choline intake during pregnancy and the prevalence of fetal alcohol spectrum disorders among infants.)", "entity_type": "UNKNOWN", "id": "CHOLINE SUPPLEMENTATION AND FASD DIAGNOSIS"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "How maternal dietary choline status affects outcomes for their infants, including growth, development, and disease risk.)", "entity_type": "UNKNOWN", "id": "MATERNAL CHOLINE STATUS IMPACT ON INFANT HEALTH"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "The impact of different levels of choline consumption compared to other nutrients in the diet on pregnancy health and infant development.)", "entity_type": "UNKNOWN", "id": "CHOLINE INTAKE VS. MATERNAL NUTRITIONAL STATUS"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "The impact of different levels of choline consumption compared to other nutrients in the diet on pregnancy health and infant development.)", "entity_type": "UNKNOWN", "id": "NUTRIENT INTAKE"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "How genetic differences influence how a mother\'s body produces choline internally during pregnancy, affecting dietary requirements.)", "entity_type": "UNKNOWN", "id": "INFLUENCE OF GENOMIC VARIANT ON ENDOGENOUS CHOLINE SYNTHESIS"}, {"source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "description": "The study explores the synergistic effects of choline supplementation on enzyme function related to choline metabolism in pregnant mothers carrying a specific genetic variant.)", "entity_type": "UNKNOWN", "id": "CHOLINE SUPPLEMENTATION AND MATERNAL ENZYME INTERACTIONS"}, {"entity_type": "GENE", "description": "This polymorphism is a genetic marker with a CA repeat sequence and has been studied in relation to splicing enhancers or repressors during gene expression.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "RS10886471"}, {"entity_type": "TRANSCRIPT", "description": "Short tandem repeats (STR) of the \'CA\' nucleotide sequence were analyzed for their function.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "(CA)N"}, {"entity_type": "GENOMIC VARIANT", "description": "The single nucleotide polymorphism at this position in rs10886471 was studied.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "RS10886471 SNP"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Polymerase Chain Reaction (PCR) was used to amplify the genomic DNA for sequencing analysis.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "PCR"}, {"entity_type": "PROTEIN", "description": "Primers specific to the rs10886471 region were designed and used in PCR.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "SPECIFIC PRIMERS"}, {"entity_type": "PROTEIN", "description": "The GLP-1 receptor plays a critical role in glucose metabolism by facilitating insulin secretion.", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "id": "GLP-1 RECEPTOR"}, {"entity_type": "STATISTICAL MEASURE", "description": "The odds ratio with 95% confidence intervals for the association between the genetic variant and T2DM, IFG, or IGT.", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "OR (95% CI)"}, {"entity_type": "SIGNIFICANCE LEVEL", "description": "Statistically significant p-values indicating strong evidence against null hypothesis of no association between genetic variants and blood glucose groups.", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "id": "P\u200a=\u200a0.001"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "The genetic variation at rs10886471 influences the frequency of different \'CA\' repeat alleles based on glucose levels.)", "entity_type": "UNKNOWN", "id": "ALLELE FREQUENCIES"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "The CA16 allele shows an increased risk associated with prediabetes and type 2 diabetes compared to other alleles.)", "entity_type": "UNKNOWN", "id": "ALLELE (CA)16"}, {"source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "description": "The CA16 allele shows an increased risk associated with prediabetes and type 2 diabetes compared to other alleles.)", "entity_type": "UNKNOWN", "id": "RISK FOR PREDIABETES AND T2DM"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "The NFG group was designated as the control group in relation to comparing with case groups (IFG, IGT, T2DM).)", "entity_type": "UNKNOWN", "id": "CONTROL GROUP"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "The NFG group was designated as the control group in relation to comparing with case groups (IFG, IGT, T2DM).)", "entity_type": "UNKNOWN", "id": "NFG GROUP"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "The (CA)16 allele was used as a reference for comparison when assessing the association of multiple alleles with different blood glucose groups.)", "entity_type": "UNKNOWN", "id": "(CA)16 ALLELE"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "The (CA)16 allele was used as a reference for comparison when assessing the association of multiple alleles with different blood glucose groups.)", "entity_type": "UNKNOWN", "id": "REFERENCE ALLELE"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "These groups were identified based on their association with genetic variants in relation to abnormal blood glucose levels.)", "entity_type": "UNKNOWN", "id": "CASE GROUPS"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "These groups were identified based on their association with genetic variants in relation to abnormal blood glucose levels.)", "entity_type": "UNKNOWN", "id": "IFG, IGT, AND T2DM GROUPS"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "Specific genotypes are negatively correlated with the IFG group based on their association with abnormal blood glucose levels.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANTS"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "Specific genotypes are negatively correlated with the IFG group based on their association with abnormal blood glucose levels.)", "entity_type": "UNKNOWN", "id": "IFG GROUP"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "Specific genotypes show a negative correlation with the IGT group, relating to genetic variants and abnormal blood glucose levels.)", "entity_type": "UNKNOWN", "id": "IGT GROUP"}, {"source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "description": "Genetic variants are significantly associated with T2DM, as identified in statistical analyses.)", "entity_type": "UNKNOWN", "id": "T2DM GROUP"}, {"entity_type": "NUTRIENT", "description": "Serum selenium concentration affects colorectal cancer risk.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "SERUM SELENIUM"}, {"entity_type": "DISEASE", "description": "Cases of colorectal cancer were diagnosed in the study participants, with some developing cancer in their colon.<SEP>There is an association observed between the CC genotype and the occurrence of Colorectal Cancer (CRC).", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6<SEP>chunk-5b0cc96debf110f3544685d3b227ff27", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 38}]", "id": "COLORECTAL CANCER"}, {"entity_type": "GENE", "description": "The single polymorphism rs8178974 in GPX4 gene is associated with increased colorectal cancer risk.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "GPX4 GENE"}, {"entity_type": "DRUG", "description": "Cases and controls had similar use of non-steroidal anti-inflammatory drugs (NSAIDs), which may influence the results.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "NSAIDS"}, {"entity_type": "NUTRIENT", "description": "Cases had lower dietary fiber intake than controls, possibly related to lifestyle or disease risk factors.<SEP>Dietary fiber comes from plant-based foods and is important for digestive health, including the prevention of certain chronic diseases.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6<SEP>chunk-d0476488384aadcae4630e94d982d2e0", "id": "DIETARY FIBER"}, {"entity_type": "DISEASE", "description": "Tumors of the colon and rectum were classified in studies that measured selenium concentration.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "COLORECTAL TUMOR"}, {"entity_type": "NUTRIENT", "description": "Serum selenium levels showed an inverse association with colorectal tumor risk.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "SELENIUM"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs8178974 in GPX4 gene is linked to colorectal cancer, but overall gene variation isn\'t.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "GPX4 SNPS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Smoking was found to be slightly higher among cases compared to controls.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "SMOKING"}, {"entity_type": "LIFESTYLE FACTOR", "description": "The use of postmenopausal hormones is lower in cases, indicating a potential role in hormone-sensitive conditions.<SEP>The use of postmenopausal hormones is lower in cases, potentially affecting hormone-sensitive cancers.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "POSTMENOPAUSAL HORMONE USE"}, {"entity_type": "DISEASE", "description": "A strong inverse association exists between serum selenium concentration and the risk of developing colorectal adenomas, highlighting its potential role as a biomarker or preventive factor.<SEP>Colorectal adenomas were more strongly associated with selenium than cancer, indicating a potential role in early stages of the disease.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "COLORECTAL ADENOMA"}, {"entity_type": "STATISTICAL ANALYSIS", "description": "A meta-analysis test found no significant publication bias for some variants and ovarian cancer risk, while it showed a marginal bias for one variant.<SEP>There was evidence for potential publication bias towards finding inverse associations between selenium and colorectal tumors.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6<SEP>chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "PUBLICATION BIAS"}, {"entity_type": "STATISTICAL FACTOR", "description": "Selenium\'s protective effect against colorectal tumors is stronger in men compared to women, indicating sex-specific differences.", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "id": "GENDER EFFECT"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "SNP rs8178974 in GPX4 gene is linked to increased colorectal cancer risk.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 38}]", "id": "GPX4 GENE POLYMORPHISM"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "An inverse association exists between serum selenium concentration and the risk of colorectal tumors.)", "entity_type": "UNKNOWN", "id": "SERUM SELENIUM CONCENTRATION"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "An increased protective effect against colorectal tumors exists for men compared to women, indicating a potential gender-specific differential response to selenium intake.)<SEP>Selenium\'s protective effect against colorectal tumors is stronger in men compared to women.)", "entity_type": "UNKNOWN", "id": "SELENIUM CONCENTRATION"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "SNP rs8178974 in GPX4 gene interacts with the risk of colorectal cancer, suggesting genetic predisposition.)", "entity_type": "UNKNOWN", "id": "GENETIC INTERACTION"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "SNP rs8178974 in GPX4 gene interacts with the risk of colorectal cancer, suggesting genetic predisposition.)", "entity_type": "UNKNOWN", "id": "GPX4 SNPS AND COLORECTAL CANCER"}, {"source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "description": "There is an inverse relationship between serum selenium concentrations and lifestyle factors such as smoking and postmenopausal hormone use.)", "entity_type": "UNKNOWN", "id": "LIFESTYLE FACTORS (INCLUDING SMOKING AND HORMONE USE)"}, {"entity_type": "GENE", "description": "CARD9 has a role in shaping the bacterial and fungal gut microbiota and is required to control fungal microbiota expansion in colitis.<SEP>The Card9 gene plays a role in colitis recovery and immune responses.<SEP>The gut microbiota of Card9\u2212/\u2212 mice contributes to the susceptibility of the mice to colitis via impaired tryptophan metabolism leading to defective AHR activation.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6<SEP>chunk-4d5e7ab61a7357d5106f4a8a355f40d0<SEP>chunk-c791a4e35844d642ba0c69774072f7cd", "id": "CARD9"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "DSS is an environmental factor that induces colitis.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "DEXTRAN SULFATE SODIUM (DSS)"}, {"entity_type": "DISEASE", "description": "Card9\u2212/\u2212 mice are more susceptible to colitis, as compared to wild-type (WT) mice.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "COLITIS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Recovery from colitis is impaired in Card9\u2212/\u2212 mice with delayed weight gain.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "WEIGHT GAIN"}, {"entity_type": "PATHOLOGICAL PROCESS", "description": "Histopathologic alterations refer to changes observed at the microscopic level, suggesting impaired intestinal healing.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "HISTOPATHOLOGIC ALTERATIONS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Decreased epithelial cell proliferation was observed during the recovery period of Card9\u2212/\u2212\u2192GF mice.<SEP>Epithelial cells show reduced proliferation in Card9\u2212/\u2212 mice during colitis recovery.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6<SEP>chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "EPITHELIAL CELL PROLIFERATION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Increased apoptosis was noted in Card9\u2212/\u2212\u2192GF mice as compared to WT\u2192GF mice.<SEP>Increased apoptosis, shown by increased staining for cleaved caspase 3, confirms impaired intestinal healing.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6<SEP>chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "APOPTOSIS"}, {"entity_type": "PROTEIN", "description": "The proliferation marker Ki67 is used to assess cell proliferation in tissues.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "KI67"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Increased staining for cleaved caspase 3 indicates increased apoptosis, which is not beneficial for recovery.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "CLEAVED CASPASE 3"}, {"entity_type": "GENOMIC DATA", "description": "Comparison of colon transcriptomes between WT and Card9\u2212/\u2212 mice during DSS-induced colitis.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "COLONIC TRANSCRIPTOMES"}, {"entity_type": "TRANSCRIPTOMICS", "description": "The number of upregulated genes is higher in Card9\u2212/\u2212 mice compared to WT on day 7, suggesting an imbalance in gene expression.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "UPREGULATED GENES"}, {"entity_type": "PATHWAY", "description": "Pathways related to immune response show dominance, which corresponds to stronger signals in Card9\u2212/\u2212 mice during colitis recovery.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "IMMUNE-RELATED PATHWAYS"}, {"entity_type": "PATHWAY", "description": "During the recovery period on day 12, cell proliferation pathways were more activated in WT mice than in Card9\u2212/\u2212 mice.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "CELL PROLIFERATION AND REPLICATION"}, {"entity_type": "GENE", "description": "Reg3g gene encoding REGIII-\u03b3 is highly induced and differentially expressed between Card9\u2212/\u2212 and WT mice on days 7 and 12.<SEP>Reg3g gene plays a role in colonic healing and regeneration.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "REG3G"}, {"entity_type": "GENE", "description": "Reg3b gene contributes to the protection against bacterial infections.<SEP>Reg3b gene encoding REGIII-\u03b2 shows similar expression patterns as Reg3g in response to colitis.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "REG3B"}, {"entity_type": "GENE", "description": "Il1b gene is also highly induced and differentially expressed between Card9\u2212/\u2212 and WT mice on days 7 and 12 during colitis.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "INTERLEUKIN 1 BETA (IL1B)"}, {"entity_type": "CYTOKINE", "description": "Administration of exogenous IL-22 was sufficient to normalize AHR ligand production and colitis susceptibility in Card9\u2212/\u2212 mice.<SEP>IL-22 plays a major role in mediating recovery from DSS-induced colitis.<SEP>Interleukin 22 (IL-22) is a cytokine that promotes tissue repair and regeneration.<SEP>The protein IL-22 plays a role in immune responses and is produced by intraepithelial \u03b3\u03b4 T cells.<SEP>IL-22 was similar in T cells of psoriasiform lesions and classical psoriasis.<SEP>Interleukin-22 promotes skin repair and regeneration; it can also contribute to inflammation in various diseases.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842<SEP>chunk-4d5e7ab61a7357d5106f4a8a355f40d0<SEP>chunk-0380fd029b3eda680c87c1b8bcbe7f4c<SEP>chunk-56c37d6045382dda8cc6037b2680282a<SEP>chunk-c791a4e35844d642ba0c69774072f7cd", "id": "IL-22"}, {"entity_type": "BIOLOGICAL SYSTEM", "description": "The composition of the gut microbiota is altered at both fungal and bacterial levels in Card9\u2212/\u2212 mice compared to WT mice.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "GUT MICROBIOTA"}, {"entity_type": "FUNGAL LOAD", "description": "The colonic fungal load peaks earlier and reaches higher values at day 7 in Card9\u2212/\u2212 mice during DSS-induced colitis.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "FUNGAL LOAD"}, {"entity_type": "DIVERSITY METRIC", "description": "Increased resilience of the fungal microbiota is observed in WT mice compared to Card9\u2212/\u2212 mice, suggesting stability and adaptability.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "FUNGAL DIVERSITY"}, {"entity_type": "PHYLUM", "description": "The gut fungal microbiota of WT and Card9\u2212/\u2212 mice is dominated by members of Ascomycota, Basidiomycota, and Zygomycota phyla.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "ASCOMYCOTA"}, {"entity_type": "CYTOKINE", "description": "Interleukin 17A has a protective role in colitis by enhancing the response to IL-22.<SEP>Interleukin-17A is pro-inflammatory cytokine produced by Th17 cells, involved in autoimmune and inflammatory conditions.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c<SEP>chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "IL-17A"}, {"entity_type": "PROTEIN", "description": "Reg3g, encoded by the Reg3g gene, functions as a regenerative and repair factor in the gastrointestinal tract.", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "REGENERATING ISLET-DERIVED PROTEIN III GAMMA (REGIII-\u0393)"}, {"entity_type": "GENUS", "description": "Decreased Adlercreutzia is observed at baseline in Card9\u2212/\u2212 mice microbiota.<SEP>Differences were observed at baseline including decreases in Adlercreutzia in the Card9\u2212/\u2212 mouse microbiota.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6<SEP>chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "id": "ADLERCREUTZIA"}, {"entity_type": "SPECIES", "description": "Decreases in Lactobacillus reuteri were found in the Card9\u2212/\u2212 mouse microbiota according to LEfSe analysis.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "LACTOBACILLUS REUTERI"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Bacterial biodiversity was observed between WT and Card9\u2212/\u2212 mice at various time points.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "BACTERIAL BIODIVERSITY"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "No differences were noted regarding fungal biodiversity between WT and Card9\u2212/\u2212 mice at any of the time points.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "FUNGAL BIODIVERSITY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A negative correlation was observed between bacterial and fungal biodiversity in WT mice, suggesting an inter-kingdom relationship.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "INTER-KINGDOM RELATIONSHIP"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A positive correlation was observed in Card9\u2212/\u2212 mice, indicating abnormal interactions within the gut ecosystem between bacteria and fungi.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "ABNORMAL INTERACTIONS"}, {"entity_type": "DISEASE", "description": "Possible consequences on gut homeostasis were suggested by the correlation patterns of gut microbiota and host\'s immune response.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "GUT HOMEOSTASIS"}, {"entity_type": "MICROBIAL COMPONENT", "description": "The bacterial component was involved in the phenotype observed in Card9\u2212/\u2212 mice, but not the fungal component.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "BACTERIAL MICROBIOTA"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Colonization of WT germ-free mice with microbiota from either WT or Card9\u2212/\u2212 mice recaptures the phenotype seen in Card9\u2212/\u2212 mice.<SEP>Microbiota plays a role in colitis susceptibility by influencing tryptophan metabolism and IL-22 production.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842<SEP>chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "GERM-FREE (GF) MICE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "DSS was used to induce injury and assess recovery response in both models.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "DSS INDUCED INJURY"}, {"entity_type": "TECHNIQUE", "description": "179 inflammation-associated genes were assessed using NanoString technology for expression levels.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "NANOSTRING TECHNOLOGY"}, {"entity_type": "GENE", "description": "Il22 was one of the most highly downregulated genes in Card9\u2212/\u2212\u2192GF mice as compared to WT\u2192GF mice.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "IL22"}, {"entity_type": "GENES", "description": "Cxcl1 and Cxcl5, known IL-22 target genes, were negatively correlated with Il22 expression in Card9\u2212/\u2212\u2192GF mice.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "CYTOKINE ENCODING GENES"}, {"entity_type": "GENE", "description": "Fos gene encoding a transcription factor was known as an IL-22 target gene and had decreased expression levels in Card9\u2212/\u2212\u2192GF mice.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "TRANSCRIPTION FACTOR ENCODING GENE"}, {"entity_type": "DISEASE", "description": "IL-22 has been implicated in intestinal homeostasis, with implications for innate antimicrobial resistance in mice.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "INTESTINAL HOMEOSTASIS"}, {"entity_type": "MODEL", "description": "Il22\u2212/\u2212 mice exhibit susceptibility to DSS-induced colitis and impaired healing during recovery periods similar to Card9\u2212/\u2212 mice.", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "id": "IL22 KNOCKOUT MICE"}, {"entity_type": "CELL", "description": "Dendritic cells stimulate IL-22-producing cells via IL-23 production.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "DENDRITIC CELLS"}, {"entity_type": "CELL", "description": "Macrophages are involved in the process of stimulating IL-22-producing cells, alongside dendritic cells.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "MACROPHAGES"}, {"entity_type": "DIETARY PATTERN", "description": "The gut microbiota contributes to colitis susceptibility and its ability to produce AHR ligands is associated with CARD9 and IBD.<SEP>The gut microbiota contributes to immune responses and affects the production of cytokines like IL-22.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842<SEP>chunk-c791a4e35844d642ba0c69774072f7cd", "id": "MICROBIOTA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Alterations in tryptophan metabolism can influence AHR activation, which is critical for IL-22 production.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "ALTERED TRYPTOPHAN METABOLISM"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Tryptophan derivatives like IAA promote local IL-22 production and are ligands for the AHR.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "INDOLE DERIVATIVES"}, {"entity_type": "MOLECULAR INTERACTION", "description": "AHR is involved in the regulation of local IL-22 production, with its activity being modulated by gut microbiota components.<SEP>The gut microbiota alters the IL-22 signaling pathway via the impairment of tryptophan metabolism which leads to defective AHR activation.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842<SEP>chunk-c791a4e35844d642ba0c69774072f7cd", "id": "AHR"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The underdevelopment of the gut immune system can lead to decreased levels of AHR ligands like IAA.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "GUT IMMUNE SYSTEM"}, {"entity_type": "DISEASE", "description": "A deficient microbiota contributes to increased colitis sensitivity in Card9\u2212/\u2212 mice through impaired tryptophan metabolism.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "CARD9\u2212/\u2212 MOUSE MICROBIOTA"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The processes involving the metabolism of L-tryptophan, particularly by gut bacteria and host cells like IDO1, are altered in Card9\u2212/\u2212 mice.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "TRYPTOPHAN METABOLISM DEFECTS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "IDO1 plays a role in tryptophan catabolism leading to kynurenine production, which affects IL-22 induction and AHR activity.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "INDOLEAMINE 2,3-DIOXYGENASE (IDO1)"}, {"entity_type": "CHEMICAL SUBSTANCE", "description": "Indole derivatives like IAA are ligands for the AHR receptor that promote local IL-22 production and modulate immune responses.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "AHR LIGANDS"}, {"entity_type": "DRUG", "description": "Ficz is an agonist of the AHR receptor used to assess its role in mitigating colitis and restoring IL-22 expression.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "6-FORMYLINDOLO(3,2-B)CARBAZOLE (FICZ)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Changes in gut microbiota composition, like those present in Card9\u2212/\u2212 mice, can influence host immune responses and inflammatory pathways.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "GUT MICROBIOTA COMPOSITION"}, {"entity_type": "BIOLOGICAL FUNCTION", "description": "The activity of the AHR receptor is crucial for the regulation of local cytokine production and immune responses to gut microbiota stimuli.", "source_id": "chunk-29025895ac187b8e88dec5cea9478842", "id": "AHR ACTIVITY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Alteration in tryptophan metabolism by the gut microbiota affects AHR activation.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "TRYPTOPHAN METABOLISM"}, {"entity_type": "METABOLIC PATHWAY", "description": "Subjects with IBD use tryptophan to produce Kyn via IDO1, but their gut microbiota\'s metabolism of tryptophan is impaired leading to defective AHR activation.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "KYN"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Activated immune cells in the gut produce Kyn through IDO1 activity when they utilize tryptophan.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "IDO1"}, {"entity_type": "DISEASE", "description": "Defects in AHR activation and changes in tryptophan metabolism are associated with IBD.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "IBD(INFLAMMATORY BOWEL DISEASE)"}, {"entity_type": "DIETARY PATTERN", "description": "The gut microbiota\'s derived metabolites play a role in AHR activation and IBD.", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "id": "MICROBIOTA DERIVED METABOLITES"}, {"source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "description": "Alterations in the gut fungal microbiota are observed at both baseline and during colitis, with Card9\u2212/\u2212 mice showing a less resilient microbiota compared to WT mice.)", "entity_type": "UNKNOWN", "id": "FUNGAL MICROBIOTA"}, {"source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "description": "Alterations in the gut fungal microbiota are observed at both baseline and during colitis, with Card9\u2212/\u2212 mice showing a less resilient microbiota compared to WT mice.)", "entity_type": "UNKNOWN", "id": "GUT HEALTH"}, {"source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea", "description": "Decreases in Adlercreutzia were revealed between baseline Card9\u2212/\u2212 mouse microbiota compared to WT microbiota.)<SEP>Variations observed between patients and controls, contributing to the heterogeneity in results.", "entity_type": "MOLECULAR INTERACTION", "id": "DIFFERENCES"}, {"source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "description": "Decreased levels of Lactobacillus reuteri were identified in the Card9\u2212/\u2212 mouse microbiota based on LEfSe results.)", "entity_type": "UNKNOWN", "id": "DECREASES"}, {"source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "description": "A negative correlation between bacterial biodiversity was observed between WT and Card9\u2212/\u2212 mice across various time points.)", "entity_type": "UNKNOWN", "id": "CORRELATION"}, {"source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "description": "No notable differences were found in fungal biodiversity between baseline WT and Card9\u2212/\u2212 mouse microbiota at any time point.)", "entity_type": "UNKNOWN", "id": "NO DIFFERENCES"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Impaired tryptophan metabolism leading to defective AHR activation contributes to colitis susceptibility in Card9\u2212/\u2212 mice.)", "entity_type": "UNKNOWN", "id": "CARD9\u2212/\u2212 MICE"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Alteration of the gut microbiota\'s function affects AHR activation through tryptophan metabolism and IL-22 pathway.)", "entity_type": "UNKNOWN", "id": "AHR ACTIVATION"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Subjects with IBD use tryptophan to produce Kyn via IDO1 activity in their gut immune cells.)", "entity_type": "UNKNOWN", "id": "TRYPTOPHAN"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "The ability of the microbiota to produce AHR ligands is associated with CARD9, IBD and their metabolic pathways.)", "entity_type": "UNKNOWN", "id": "AHR AGONISTS"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Variants in CARD9 are linked to reduced AHR activation by microbiota-derived metabolites extracted from fecal samples in individuals with IBD.)", "entity_type": "UNKNOWN", "id": "IBD"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Impaired tryptophan metabolism in Card9\u2212/\u2212 mice affects AHR ligand production, thus impacting colitis susceptibility.)", "entity_type": "UNKNOWN", "id": "AHR LIGAND PRODUCTION"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Gut immune cells utilize tryptophan via IDO1 to produce Kyn when it comes to AHR activation in individuals with IBD.)", "entity_type": "UNKNOWN", "id": "GUT IMMUNE CELLS"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "Healthy subjects\' fecal samples induce higher levels of AHR activation compared to those from patients with IBD.)", "entity_type": "UNKNOWN", "id": "FECAL SAMPLES"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "The microbiota\'s tryptophan metabolites are required for the proper AHR ligation process in individuals without IBD.)", "entity_type": "UNKNOWN", "id": "AHR LIGATION"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "The microbiota\'s tryptophan metabolites are required for the proper AHR ligation process in individuals without IBD.)", "entity_type": "UNKNOWN", "id": "TRYPTOPHAN METABOLITES"}, {"source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "description": "The genetic variant rs10781499 within CARD9 impacts the interaction with the gut microbiota, affecting AHR activation in IBD patients.)", "entity_type": "UNKNOWN", "id": "RS10781499"}, {"entity_type": "GENE", "description": "A gene associated with prevalent hypertension and new-onset hypertension.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "\u03922M"}, {"entity_type": "DISEASE", "description": "Prevalent hypertension is a condition associated with baseline \u03b22M concentration.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "PREVALENT HYPERTENSION"}, {"entity_type": "DISEASE", "description": "New-onset hypertension risk is positively associated with baseline \u03b22M concentration.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "NEW-ONSET HYPERTENSION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The physiological process of measuring beta-2 microglobulin levels in blood.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "\u03922M CONCENTRATION"}, {"entity_type": "STATISTICAL MEASURE", "description": "A statistical measure for the association between baseline \u03b22M concentration and hypertension risk.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "ODDS RATIO (OR)"}, {"entity_type": "COVARIATE", "description": "Variables such as age, sex, BMI, diabetes, smoking status, cholesterol levels, statin use, eGFR, and blood pressure that are considered in multivariable models.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "CLINICAL COVARIATES"}, {"entity_type": "CLASSIFICATION", "description": "Statistical quartile classification of \u03b22M concentration into four groups.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "QUARTILES"}, {"entity_type": "DEMOGRAPHIC VARIABLE", "description": "Average age of participants at baseline.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "MEAN AGE"}, {"entity_type": "DEMOGRAPHIC PROPORTION", "description": "Proportion of women in the study sample.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "WOMEN RATIO (54%)"}, {"entity_type": "STATISTICAL MODEL", "description": "Generalized Estimating Equation model used for longitudinal analysis.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "GEE MODEL"}, {"entity_type": "MODEL EVALUATION METRIC", "description": "Measures how well a statistical model predicts new onset hypertension.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "DISCRIMINATION"}, {"entity_type": "RECLASSIFICATION METRIC", "description": "Improvement in the prediction of new onset hypertension after adding \u03b22M to the clinical model.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "INTEGRATED DISCRIMINATION IMPROVEMENT (IDI)"}, {"entity_type": "RECLASSIFICATION METRIC", "description": "Measures how a model\'s predictions change when using additional variables like \u03b22M.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "NET RECLASSIFICATION INDEX (NRI)"}, {"entity_type": "PREDICTIVE MODEL", "description": "A predictive model consisting of various baseline covariates for hypertension risk assessment.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "CLINICAL MODEL"}, {"entity_type": "CONDITION", "description": "Presence of albumin in urine as an indicator for kidney damage or dysfunction.<SEP>Presence of albumin in urine, which was considered as a confounding factor when estimating \u03b22M\'s association with new-onset hypertension.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "ALBUMINURIA"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Estimated glomerular filtration rate, used to assess kidney function and adjust for baseline conditions.<SEP>The process of estimating glomerular filtration rate (eGFR) across different ancestry groups was found to have genetic associations.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9<SEP>chunk-3edc72730d68b4c7fd6ba397aa087391", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 53}]", "id": "EGFR"}, {"entity_type": "GENETIC TERM", "description": "The effect of a genetic variant on the gene it is located near within its immediate genomic context.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "CIS-EFFECTS (WITHIN 1 MEGABASE)"}, {"entity_type": "GENETIC TERM", "description": "The effect of a genetic variant that affects a gene located far from where it lies in the genome (greater than 1 mega base distance).", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "TRANS-EFFECTS (>1 MEGABASE AWAY)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Expression level of SH2B3 gene, influenced by the rs3184504 polymorphism and related to changes in blood \u03b22M levels.<SEP>Expression level of SH2B3, which is influenced by \u03b22M levels.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "SH2B3 EXPRESSION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A reduction in SH2B3 expression as a result of rs3184504 polymorphism.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "DECREASED SH2B3 EXPRESSION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Variants that impact gene function within the same chromosome and relatively close to their target genes.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "CIS-GENETIC VARIATION (WITHIN SH2B3)"}, {"entity_type": "GENOMIC VARIANT", "description": "A genetic variant causing a deficiency in SH2B3 expression by deleting its corresponding gene.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "SH2B3 GENE DEFICIENCY (SH2B3\u2212/\u2212 MICE)"}, {"entity_type": "EXPERIMENTAL CONDITION", "description": "Hypertensive response induced through the administration of angiotensin II to test effects.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "ANGIOTENSIN II-INDUCED HYPERTENSION"}, {"entity_type": "BIOLOGICAL RESPONSE", "description": "Increased \u03b22M expression in response to hypertensive stimuli in Sh2b3\u2212/\u2212 mice compared to wild-type ones.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "\u03922M EXPRESSION INDUCTION"}, {"entity_type": "MEDICATION STATUS", "description": "Use of statin medications by participants in relation to \u03b22M concentration and hypertension risk.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "STATINS USE"}, {"entity_type": "LABORATORY MEASUREMENT", "description": "The upper number in blood pressure measurement, indicating the arterial pressure during heart contraction.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "SYSTOLIC BLOOD PRESSURE (BP)"}, {"entity_type": "LABORATORY MEASUREMENT", "description": "The lower number in blood pressure measurement, indicating the arterial pressure during rest between heartbeats.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "DIASTOLIC BLOOD PRESSURE (BP)"}, {"entity_type": "CLINICAL MEASURE", "description": "An estimate of the glomerular filtration rate, measuring how well kidneys filter waste products from blood.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "EGFR (ESTIMATED GLOMERULAR FILTRATION RATE)"}, {"entity_type": "PHARMACOLOGICAL AGENT", "description": "A vasoactive hormone that increases blood pressure by causing arteriolar constriction and stimulating aldosterone secretion.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "ANGIOTENSIN II"}, {"entity_type": "GENETIC VARIATION", "description": "Specific genetic variation in the human genome, associated with plasma \u03b22M levels and SH2B3 expression.", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "id": "RS3184504 POLYMORPHISM"}, {"source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "description": "The clinical model adjusts for various confounding factors to assess the independent association between \u03b22M and hypertension.)", "entity_type": "UNKNOWN", "id": "ADJUSTMENT FOR CONFOUNDING FACTORS"}, {"source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "description": "In a longitudinal study, increased baseline \u03b22M concentration is associated with an elevated risk of new-onset hypertension.)", "entity_type": "UNKNOWN", "id": "BASELINE \u03922M CONCENTRATION"}, {"source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "description": "In a longitudinal study, increased baseline \u03b22M concentration is associated with an elevated risk of new-onset hypertension.)", "entity_type": "UNKNOWN", "id": "LONGITUDINAL ASSOCIATION WITH NEW-ONSET HYPERTENSION"}, {"source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "description": "The addition of \u03b22M to the clinical model improves its ability to discriminate between individuals who will develop new onset hypertension.)", "entity_type": "UNKNOWN", "id": "DISCRIMINATION FOR NEW-ONSET HYPERTENSION"}, {"source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "description": "Decreased SH2B3 expression leads to higher \u03b22M levels, as indicated by the inverse variance weighted MR analysis.)", "entity_type": "UNKNOWN", "id": "CAUSAL EFFECT ON \u03922M LEVELS"}, {"source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "description": "The rs3184504 genetic variant may influence \u03b22M levels through its impact on SH2B3 expression.)", "entity_type": "UNKNOWN", "id": "MEDIATION THROUGH SH2B3 EXPRESSION"}, {"source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "description": "Deficiency in SH2B3 function is associated with a more severe hypertensive response and renal dysfunction.)", "entity_type": "UNKNOWN", "id": "INCREASED HYPERTENSIVE RESPONSE"}, {"source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "description": "Deficiency in SH2B3 function is associated with a more severe hypertensive response and renal dysfunction.)", "entity_type": "UNKNOWN", "id": "SH2B3\u2212/\u2212 MOUSE MODEL"}, {"source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "description": "The rs3184504 polymorphism influences SH2B3 expression, as indicated by the association between it and SH2B3 gene function in the Mendelian randomization study.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION WITH SH2B3 EXPRESSION"}, {"entity_type": "INFANT", "description": "Case infants were more likely to be female than control infants.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "CASE INFANT"}, {"entity_type": "INFANT", "description": "Control infants had a different sex distribution compared to case infants.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "CONTROL INFANT"}, {"entity_type": "MOTHER", "description": "Non-Hispanic White mothers tended to be older at delivery compared to Hispanic mothers.<SEP>The racial/ethnic makeup of the study population was consistent within cases and controls for non-Hispanic Whites but varied for Hispanics.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "NON-HISPANIC WHITE MOTHER"}, {"entity_type": "MOTHER", "description": "Hispanic mothers received less formal education than non-Hispanic White mothers.<SEP>Hispanic mothers were younger than non-Hispanic White mothers but consistent within maternal race/ethnicity.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "HISPANIC MOTHER"}, {"entity_type": "NUTRIENT SUPPLEMENT", "description": "Periconceptional use of folic acid supplements was higher among non-Hispanic White mothers compared to Hispanic mothers.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "FOLIC ACID SUPPLEMENTS"}, {"entity_type": "DIETARY PATTERN/FOLATE CONSUMPTION", "description": "Amounts and sources of dietary folate consumed daily by mothers were also evaluated.<SEP>The differences in daily dietary intake of folate were subtle between the studied subgroups.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "DAILY DIETARY FOLATE INTAKE"}, {"entity_type": "DIETARY ASSESSMENT", "description": "A composite measure capturing overall folate intake was more similar across groups than single nutrient intake measures.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "COMPOSITE FOLATE INTAKE VARIABLE"}, {"entity_type": "GENETIC POLYMORPHISM", "description": "There was no evidence of association between DHFR gene variants and NTD-affected pregnancies in the studied population.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "DHFR GENE VARIANT"}, {"entity_type": "SNP", "description": "The CT genotype for MTHFD1 SNPs was associated with a risk of 1.58 (CI:0.99,2.51) in infants from the lowest folate intake group compared to those with CC genotypes.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "MTHFD1 SNPS RS2236224 AND RS2236225"}, {"entity_type": "SNP", "description": "The AG genotype for MTHFD1 SNP was associated with a risk of four-fold increase (CI: 2.33,7.75) in NTD risk when maternal folate intake was low compared to GG genotypes.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "MTHFD1 SNP RS11627387 AG GENOTYPE"}, {"entity_type": "SNP", "description": "Mothers with low folate intake had a decreased risk of an NTD-affected pregnancy if they carried heterozygous MTHFR SNPs compared to common homozygous mothers.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "MTHFR SNPS RS1476413, RS1801131 AND RS1801133"}, {"entity_type": "SNP", "description": "The CG genotype for SHMT1 was associated with a 31% risk reduction (CI: 0.49,0.96) in NTD-affected pregnancies compared to CC genotypes.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "SHMT1 SNP RS669340 CG GENOTYPE"}, {"entity_type": "SNP INTERACTION", "description": "There was an interaction effect for SHMT1 SNPs associated with risk of NTDs in infants depending on maternal folate intake.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "SHMT1 SNP RS12939757 AG/AA GENOTYPES AND LOW VS HIGH FOLATE INTAKE"}, {"entity_type": "SNP INTERACTION", "description": "A gene-folate interaction was associated with a reduced risk of NTDs for mothers in the low folate intake group compared to those in the high intake group.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "TYMS SNP RS502396 OR RS699517 HETEROZYGOUS GENOTYPES AND LOW VS HIGH FOLATE INTAKE"}, {"entity_type": "SNP INTERACTION", "description": "The gene-folate interaction also had a protective effect on infants with AG genotypes compared to AA genotypes when mothers were on low folate intake.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "GENE-FOLATE INTERACTION FOR TYMS SNPS RS2847153 AND LOW VS HIGH FOLATE INTAKE IN INFANTS"}, {"entity_type": "STUDY DESIGN", "description": "A case-control design was employed to compare characteristics of infants with NTDs to those without.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "CASE-CONTROL DESIGN"}, {"entity_type": "COVARIATES OR EXPLANATORY VARIABLES", "description": "The study considered maternal demographics including age, race/ethnicity, and education level as potential confounding factors.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "MATERNAL DEMOGRAPHICS"}, {"entity_type": "INTERVENTION OR EXPOSURE", "description": "Maternal intake of folic acid supplements before pregnancy was assessed as an intervention or exposure factor.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "PERICONCEPTIONAL FOLIC ACID SUPPLEMENTATION"}, {"entity_type": "OUTCOME MEASURE", "description": "A composite measure combining intake from dietary sources and supplements was used to evaluate overall maternal folate status.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "COMPOSITE MEASURE FOR FOLATE INTAKE"}, {"entity_type": "GENETIC MARKER OR POLYMORPHISM", "description": "The genotypes of specific polymorphisms in the dihydrofolate reductase (DHFR) gene were examined for their association with NTD risk.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "DHFR GENE VARIANT GENOTYPES"}, {"entity_type": "SNP OR GENETIC MARKER", "description": "The genotypes of two specific single nucleotide polymorphisms (SNPs) in the MTHFD1 gene were analyzed.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "MTHFD1 SNPS RS2236224 AND RS2236225 GENOTYPES"}, {"entity_type": "SNP INTERACTION WITH FOLATE INTAKE", "description": "Interaction between genotypes of a specific SNP and maternal folate intake on NTD risk was studied.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "MTHFD1 SNP RS11627387 AG/AA GENOTYPES"}, {"entity_type": "SNP OR GENETIC POLYMORPHISM INTERACTION", "description": "Combinations of genotypes from three specific MTHFR SNPs were investigated for their effect on NTD risk.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "MTHFR SNPS RS1476413, RS1801131 AND RS1801133 GENOTYPES"}, {"entity_type": "GENETIC MARKER OR POLYMORPHISM", "description": "The genotypes of a specific polymorphism in the serine hydroxymethyltransferase 1 (SHMT1) gene were considered as an additional genetic factor.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "SHMT1 SNP RS669340 CG/CC GENOTYPES"}, {"entity_type": "SNP OR GENETIC MARKER INTERACTION WITH FOLATE INTAKE", "description": "Combinations of genotypes from specific TYMS SNPs were analyzed for their impact on NTD risk, especially in the context of low maternal folate intake.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "TYMS SNP RS2847153 AND RS699517 GENOTYPES"}, {"entity_type": "INTERACTION EFFECT BETWEEN GENES AND NUTRIENT INTAKE", "description": "The study aimed to uncover whether certain genotypes might interact differently with maternal folic acid intake, affecting NTD risk.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "GENE-FOLATE INTERACTION FOR DHFR AND MTHFD1 SNPS"}, {"entity_type": "OUTCOME COMPARISON", "description": "Comparisons were made between infants classified as cases (with NTDs) and those in the control group without.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "CASE INFANT VS. CONTROL INFANT COMPARISON"}, {"entity_type": "COHORT COMPOSITION OR STRATIFICATION FACTOR", "description": "The racial/ethnic composition was considered, with separate analyses for non-Hispanic White vs. Hispanic mothers and infants.", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "id": "RACIAL/ETHNIC SUBGROUPS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "There was an age difference between non-Hispanic White and Hispanic mothers based on supplement use.)", "entity_type": "UNKNOWN", "id": "MATERNAL AGE AT DELIVERY"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "There was a disparity in formal education attainment between Hispanic and non-Hispanic White mothers.)", "entity_type": "UNKNOWN", "id": "HISPANIC MOTHER EDUCATION LEVEL"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "There was a disparity in formal education attainment between Hispanic and non-Hispanic White mothers.)", "entity_type": "UNKNOWN", "id": "NON-HISPANIC WHITE MOTHER EDUCATION LEVEL"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "The composite measure showed subtle differences but more consistency compared to single nutrient intake measures.)", "entity_type": "UNKNOWN", "id": "DIFFERENCES ACROSS SUBGROUPS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "The composite measure showed subtle differences but more consistency compared to single nutrient intake measures.)", "entity_type": "UNKNOWN", "id": "FOLATE INTAKE VARIABLE COMPOSITE MEASURE"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "No evidence of association between DHFR gene variants and risk for NTD pregnancies in studied population.)", "entity_type": "UNKNOWN", "id": "DHFR GENE VARIANT AND NTD-AFFECTED PREGNANCIES"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "No evidence of association between DHFR gene variants and risk for NTD pregnancies in studied population.)", "entity_type": "UNKNOWN", "id": "GENETIC POLYMORPHISM ASSOCIATION"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "CT genotype was associated with increased risk compared to CC genotypes for infants in lower folate intake groups.)", "entity_type": "UNKNOWN", "id": "INFANT RISK FACTOR"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "CT genotype was associated with increased risk compared to CC genotypes for infants in lower folate intake groups.)", "entity_type": "UNKNOWN", "id": "MTHFD1 SNPS RS2236224 AND RS2236225 CT GENOTYPE"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "AG genotype was associated with four-fold increase risk of NTDs when maternal folate intake was low compared to GG genotypes.)", "entity_type": "UNKNOWN", "id": "MTHFD1 SNP RS11627387 AG/AA GENOTYPES AND LOW FOLATE INTAKE"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "AG genotype was associated with four-fold increase risk of NTDs when maternal folate intake was low compared to GG genotypes.)", "entity_type": "UNKNOWN", "id": "NTD RISK FACTOR"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Low folate intake was associated with a decreased risk of an NTD-affected pregnancy if mothers carried heterozygous MTHFR SNPs compared to common homozygous mothers.)", "entity_type": "UNKNOWN", "id": "FOLATE INTAKE AND RISK OF NTDS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Low folate intake was associated with a decreased risk of an NTD-affected pregnancy if mothers carried heterozygous MTHFR SNPs compared to common homozygous mothers.)", "entity_type": "UNKNOWN", "id": "MTHFR SNPS RS1476413, RS1801131 AND RS1801133 HETEROZYGOUS VS COMMON HOMOZYGOUS MOTHERS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "CG genotype was associated with a 31% risk reduction of NTD-affected pregnancies compared to CC genotypes among mothers.)", "entity_type": "UNKNOWN", "id": "RISK REDUCTION FOR NTDS IN MOTHERS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "An interaction effect was observed with AG vs AA genotypes in infants depending on maternal folate intake.)", "entity_type": "UNKNOWN", "id": "RISK FOR NTDS INTERACTION EFFECT"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "An interaction effect was observed with AG vs AA genotypes in infants depending on maternal folate intake.)", "entity_type": "UNKNOWN", "id": "SHMT1 SNP RS12939757 AG/AA GENOTYPES AND LOW FOLATE INTAKE IN INFANTS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Gene-folate interaction was associated with reduced risk of NTDs among mothers on low folate intake compared to high intake group.)", "entity_type": "UNKNOWN", "id": "RISK REDUCTION FOR NTDS INTERACTION EFFECT"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "Gene-folate interaction was associated with reduced risk of NTDs among mothers on low folate intake compared to high intake group.)", "entity_type": "UNKNOWN", "id": "TYMS SNP RS502396 OR RS699517 HETEROZYGOUS GENOTYPES AND LOW FOLATE INTAKE IN MOTHERS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "An interaction effect was protective for infants with AG genotypes compared to AA genotypes when mothers were on low folate intake.)", "entity_type": "UNKNOWN", "id": "RISK FOR NTDS INTERACTION EFFECT IN INFANTS"}, {"source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "description": "An interaction effect was protective for infants with AG genotypes compared to AA genotypes when mothers were on low folate intake.)", "entity_type": "UNKNOWN", "id": "TYMS SNPS RS2847153 AND LOW FOLATE INTAKE IN INFANTS, INTERACTION EFFECT FOR AG GENOTYPES"}, {"entity_type": "GENOMIC VARIANT", "description": "rs10399931 is a genetic variant without significant association with Type 2 Diabetes after adjusting for age, sex, BMI and glucose tolerance.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 37}]", "id": "RS10399931"}, {"entity_type": "GENOMIC VARIANT", "description": "rs4950928 is another genomic variant that was not associated with T2D after adjustment for covariates.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 51}]", "id": "RS4950928"}, {"entity_type": "DISEASE", "description": "IFG was tested with genetic variants but showed no significant associations.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "id": "IMPAIRED FASTING GLUCOSE (IFG)"}, {"entity_type": "DISEASE", "description": "IGT was investigated for associations with certain genetic variants, though a borderline protective effect was detected for one specific variant.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "id": "IMPAIRED GLUCOSE TOLERANCE (IGT)"}, {"entity_type": "QUANTITATIVE TRAIT", "description": "An association was discovered between rs10399931 and serum C-peptide concentration after a 2-hour mark, suggesting its influence on this specific trait.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 37}]", "id": "SERUM C-PEPTIDE CONCENTRATION"}, {"entity_type": "GENOMIC VARIANT", "description": "rs10546972, another genetic variant of interest, showed no significant association with T2D in our analysis.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 51}]", "id": "RS10546972"}, {"entity_type": "CONDITION", "description": "A condition including both IFG and IGT was not associated with specific SNPs except for a borderline protective effect detected for rs12123883.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "id": "IMPAIRED GLUCOSE REGULATION (IGR)"}, {"entity_type": "GENOMIC REGION", "description": "Haplotype block 1 of the CHI3LI gene did not show any significant associations with T2D.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 51}]", "id": "CHI3LI HAPLOTYPE BLOCK 1"}, {"entity_type": "GENOMIC REGION", "description": "Another genomic region of the CHI3LI gene similarly showed no association with Type 2 Diabetes.", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 51}]", "id": "CHI3LI HAPLOTYPE BLOCK 2"}, {"entity_type": "DISEASE", "description": "Aerosolized microdust pneumonitis (AMD) is a condition characterized by lung inflammation and fibrosis, often resulting from exposure to dust particles.<SEP>Age-related macular degeneration (AMD) is a condition that can be influenced by both genetic factors and lifestyle choices.", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6<SEP>chunk-23a090455f50ab98c53e8b90941c6da5", "clusters": "[{level: 0, cluster: 0}]", "id": "AMD"}, {"entity_type": "SCORE", "description": "HLS scores are low when dietary intake does not meet recommended guidelines.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "id": "LOW HLS SCORES"}, {"entity_type": "SCORE", "description": "HLS scores are high when dietary intake meets or exceeds recommended guidelines.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "id": "HIGH HLS SCORES"}, {"entity_type": "DIETARY PATTERN", "description": "A stable diet is characterized by consistent food and nutrient intake over time.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "id": "STABLE DIETS"}, {"entity_type": "GENE", "description": "The Y402H variant of the complement factor H gene (CFH) has been linked to increased risk for AMD.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "clusters": "[{level: 0, cluster: 0}]", "id": "Y402H CFH RISK ALLELE"}, {"entity_type": "GENE", "description": "An additional copy of the A69S CFH risk allele increases odds of AMD.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "clusters": "[{level: 0, cluster: 0}]", "id": "A69S RISK ALLELE"}, {"entity_type": "GENE", "description": "The G allele for the single nucleotide polymorphism in the complement factor CFB/C2 gene is associated with increased AMD risk.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "clusters": "[{level: 0, cluster: 0}]", "id": "G ALLELE OF RS641153 (CFB/C2)"}, {"entity_type": "GENE", "description": "No main effect was observed within CAREDS for rs10033900, a variant of CFH related to AMD risk.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "clusters": "[{level: 0, cluster: 0}]", "id": "RS10033900"}, {"entity_type": "GENE", "description": "Similarly, no main effect was found for rs2230199, another CFH variant linked to AMD.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "clusters": "[{level: 0, cluster: 0}]", "id": "RS2230199"}, {"entity_type": "GENOMIC VARIANT", "description": "CFH genotypes refer to the different forms of the CFH gene which can affect susceptibility to AMD.", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "id": "CFH GENOTYPE"}, {"entity_type": "DIETARY PATTERN", "description": "Lifestyle is a factor that could potentially interact with genetic factors to influence the risk of age-related macular degeneration (AMD).", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "clusters": "[{level: 0, cluster: 6}]", "id": "LIFESTYLE"}, {"entity_type": "SYNERGY INDEX", "description": "The synergy index is a measure used in this context, indicating the combined effect of two or more factors on disease risk.", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "id": "SI"}, {"entity_type": "GENE", "description": "The ARMS2 gene carries risk alleles associated with AMD.", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "id": "ARMS2 GENOTYPE"}, {"entity_type": "HEALTHY LIFESTYLE SCORE", "description": "The HLS score was used to evaluate adherence to a healthy lifestyle, which could potentially interact with ARMS2 genotypes on AMD risk.", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "id": "HLS"}, {"source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "description": "There was no evidence for joint effects between lifestyle and variants in the CFB/C2 gene on AMD risk.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 15}]", "id": "CFB/C2"}, {"source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "description": "The main effects for SNPs in C3 were absent when examining interactions with lifestyle, indicating limited influence on AMD risk.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 12}]", "id": "C3 RS2230199"}, {"source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "description": "Similar to the C3 SNP, there were no strong indications of interaction between lifestyle and this variant in CFI for AMD risk.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 16}]", "id": "CFI RS10033900"}, {"source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "description": "The CFB/C2 gene variants were also explored in relation to AMD, but their main effects did not significantly influence AMD risk in combination with HLS.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 14}]", "id": "CFB/C2 GENE"}, {"entity_type": "GENOMIC VARIANT", "description": "A total of 18 SNPs located in 1q24 achieved a suggestive significance.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "1Q24"}, {"entity_type": "GENOMIC VARIANT", "description": "A total of 18 SNPs located in 12q24 also achieved a suggestive significance.<SEP>When we additionally adjusted for baseline BMI, the same 18 significant SNPs were identified as well.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "12Q24"}, {"entity_type": "SNP", "description": "One of the genotyped SNPs considered coffee-related.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "RS2074356"}, {"entity_type": "SNP", "description": "Another genotyped SNP considered coffee-related.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "RS11066015"}, {"entity_type": "SNP", "description": "Yet another genotyped SNP considered coffee-related.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "RS12229654"}, {"entity_type": "SNP", "description": "One of the imputed SNPs considered coffee-related.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "RS11065828"}, {"entity_type": "SNP", "description": "The other imputed SNP considered coffee-related.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "RS79105258"}, {"entity_type": "GENE", "description": "Located in gene HECTD4, one of the identified genes associated with coffee consumption.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "HECTD4"}, {"entity_type": "GENE", "description": "Located in gene ACAD10, another one of the identified genes associated with coffee consumption.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "ACAD10"}, {"entity_type": "GENE", "description": "Located in gene MYL2, yet another identified gene associated with coffee consumption.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "MYL2"}, {"entity_type": "GENE", "description": "Located in gene CUX2, one of the genes associated with coffee consumption.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "CUX2"}, {"entity_type": "POPULATION GROUP", "description": "Participants who were men and had been consuming \u22652 cups/day of black coffee had a lower risk for prediabetes and type 2 diabetes.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "MEN"}, {"entity_type": "POPULATION GROUP", "description": "Although the association was not statistically significant among women, an inverse trend was observed for sugared coffee consumption.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "WOMEN"}, {"entity_type": "QUANTITATIVE MEASURE", "description": "The information on genetic risk scores for participants based on their SNPs associated with coffee consumption.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "GENETIC RISK SCORES"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Participants who consumed habitual coffee were also more likely to drink alcohol.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "ALCOHOL INTAKE"}, {"entity_type": "DIETARY PATTERN", "description": "Participants who preferred sugared coffee over black coffee were part of the sugared coffee dietary pattern.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "SUGARED COFFEE"}, {"entity_type": "STUDY POPULATION", "description": "The baseline characteristics provided a comprehensive overview of the study population\'s demographics, behaviors, and health status.", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "id": "PARTICIPANTS"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "A Genome-Wide Association Study was conducted to find the genetic polymorphisms associated with habitual coffee consumption.)", "entity_type": "UNKNOWN", "id": "STATISTICAL ANALYSIS"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Suggestive significance (p < 1 \u00d7 10\u22125) for genotyped SNPs in GWAS.)", "entity_type": "UNKNOWN", "id": "STATISTICAL MEASURE"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Genetic polymorphisms identified by GWAS were associated with habitual coffee consumption.)", "entity_type": "UNKNOWN", "id": "SNP AND COFFEE CONSUMPTION"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86<SEP>chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "A total of 18 SNPs located in different genomic regions achieved suggestive significance.)<SEP>Variations within genes related to lipid metabolism can influence an individual\'s susceptibility to hypertriglyceridemia.", "entity_type": "BIOMEDICAL VARIATION", "id": "GENETIC POLYMORPHISMS"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "A total of 18 SNPs located in different genomic regions achieved suggestive significance.)", "entity_type": "UNKNOWN", "id": "GWAS RESULTS"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "An inverse association was more pronounced among individuals with high genetic risk scores for black coffee consumption, but the interaction was not statistically significant.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 3}]", "id": "GENETIC RISK SCORES AND COFFEE CONSUMPTION"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Participants who consumed \u22652 cups/day of black or sugared coffee had a lower risk of prediabetes and type 2 diabetes.)", "entity_type": "UNKNOWN", "id": "COFFEE CONSUMPTION AND DIABETES RISK"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Participants who consumed \u22652 cups/day of black or sugared coffee had a lower risk of prediabetes and type 2 diabetes.)", "entity_type": "UNKNOWN", "id": "PREVENTION EFFECT"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "There was a correlation between age and coffee consumption, with younger participants being more likely to consume black or sugared coffee.)", "entity_type": "UNKNOWN", "id": "AGE AND COFFEE CONSUMPTION"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Coffee consumers tended to have higher rates of alcohol consumption.)", "entity_type": "UNKNOWN", "id": "COFFEE CONSUMPTION AND ALCOHOL INTAKE"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Coffee preferences could indicate different dietary patterns or habits.)", "entity_type": "UNKNOWN", "id": "COFFEE TYPE AND DIETARY PREFERENCE"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Coffee preferences could indicate different dietary patterns or habits.)", "entity_type": "UNKNOWN", "id": "PREFERENCE"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Detailed examination was conducted on the baseline characteristics of the participants for various analyses.)", "entity_type": "UNKNOWN", "id": "CHARACTERISTICS ANALYSIS"}, {"source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "description": "Detailed examination was conducted on the baseline characteristics of the participants for various analyses.)", "entity_type": "UNKNOWN", "id": "STUDY POPULATION ANALYSIS"}, {"entity_type": "GENE", "description": "SENP1 deficient mice show autoimmune mediated damages.<SEP>SENP1-deficient mice exhibit T1DM phenotype<SEP>The SENP1 gene is involved in the development of spontaneous diabetes in NOD mice and human T1DM patients.<SEP>The gene SENP1 plays a role in autoimmune responses occurring in the pancreases of mice.", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c<SEP>chunk-0c1631ae2c195cb68139ea9d75a3b433<SEP>chunk-8382a49b61215482f9fc75e3a86cb987<SEP>chunk-fc33d6399ec56e65d007269407f1953d", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 28}]", "id": "SENP1"}, {"entity_type": "TRANSCRIPT", "description": "SENP1 expression (messenger RNA (mRNA) and protein)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "MRNA"}, {"entity_type": "DIETARY PATTERN", "description": "Genetically modified mice with an adipocyte-specific deletion of SENP1<SEP>we created genetically modified mice with an adipocyte-specific deletion of SENP1", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "ADIPONECTIN, PDGFR\u0391 OR AP2/FABP4"}, {"entity_type": "MOLECULAR INTERACTION", "description": "we created genetically modified mice with an adipocyte-specific deletion of SENP1", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "CRE RECOMBINASE"}, {"entity_type": "DISEASE", "description": "SENP1-deficient mice were viable after birth. However, these adipocyte-specific SENP1-deficient mice all developed a diabetic phenotype (Fig.<SEP>|>Adipocyte Cells", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c<SEP>chunk-174831b97ed170039b374ba254d4f506", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 47}]", "id": "PATS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "the aP2-Cre has a moderate efficiency in adipocytes with variable degrees of target gene recombination", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "AP2-CRE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "in a floxed gene locus-dependent manner", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "MACROPHAGES OR CAPILLARY ENDOTHELIUM"}, {"entity_type": "DIETARY PATTERN", "description": "we focused on male mice for further studies", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "FASTING"}, {"entity_type": "DISEASE", "description": "SENP1-smKO and SENP1-ecKO mice did not develop a high-glucose phenotype even at the age of 14 weeks", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "SENP1-SMKO"}, {"entity_type": "MOLECULAR INTERACTION", "description": "specifically deleted in vascular endothelial cells", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "VE-CADHERIN-CRE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "the GTT was prolonged and similar insulin responses were observed", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "GTT AND ITT"}, {"entity_type": "DISEASE", "description": "glucose-stimulated insulin secretion from SENP1-deficient islets was significantly reduced", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "id": "ISOLATED ISLETS"}, {"entity_type": "DISEASE", "description": "Insulin-dependent diabetes mellitus results from T-cell-mediated destruction of insulin-producing pancreatic islet \u03b2 cells.", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "id": "INSULIN-DEPENDENT DIABETES MELLITUS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Immune cell-specific effector T-cell invasion in the pancreatic tissue attacks the islets and induces cell death.", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "id": "T-CELL"}, {"entity_type": "MOLECULAR INTERACTION/KETONE BODY", "description": "Beta-hydroxybutyrate is classified as ketones that are produced in the body when the body loses its ability to absorb glucose properly.", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 28}]", "id": "\u0392-HYDROXYBUTYRATE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "T cells, which are part of the immune system, interact with autoantigens (NRP-V7) from a specific glucose-6-phosphatase catalytic subunit-related protein in pancreatic lymph nodes and spleen.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "AUTOANTIGEN NRP-V7-POSITIVE CD8+ T CELLS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A peptide derived from the islet-specific glucose-6-phosphatase catalytic subunit interacts with NRP-V7-positive CD8+ T cells.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "PEPTIDE FROM ISLET-SPECIFIC GLUCOSE-6-PHOSPHATASE CATALYTIC SUBUNIT-RELATED PROTEIN"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "A consequence of autoimmune damage to pancreatic \u03b2 cells is the reduction in the production and release of insulin, leading to insulin deficiency.<SEP>|>Disease/Type 1 Diabetes", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433<SEP>chunk-174831b97ed170039b374ba254d4f506", "id": "INSULIN DEFICIENCY"}, {"entity_type": "CYTOKINE", "description": "SENP1-deficient mice show elevated levels of proinflammatory cytokines such as IL-6, TNF-\u03b1 and IL-I\u03b2 before the onset of diabetes.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "PROINFLAMMATORY CYTOKINES IL-6, TNF-\u0391, IL-I\u0392"}, {"entity_type": "AUTOIMMUNE RESPONSE", "description": "Increased systemic cytokines in SENP1-deficient mice suggest a potential for broader autoimmune reactions.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "SYSTEMIC AUTOIMMUNE RESPONSES"}, {"entity_type": "DISEASE", "description": "The combination of high proinflammatory cytokine levels and altered adipose tissue metabolism can result in the development of diabetes, manifested by diabetic phenotypes such as cardiac function changes and vascular reactivity issues.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "DIABETIC PHENOTYPE"}, {"entity_type": "BIOMEDICAL STRUCTURE", "description": "Alterations in adipose tissue function can impact lipid metabolism and triglyceride levels.<SEP>SENP1 deletion affects the expression of certain cytokines specifically in peri-pancreatic adipocytes compared to other types of adipose tissue.<SEP>|>Disease/Diabetes", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86<SEP>chunk-0c1631ae2c195cb68139ea9d75a3b433<SEP>chunk-174831b97ed170039b374ba254d4f506", "id": "ADIPOSE TISSUE"}, {"entity_type": "CYTOKINE", "description": "The production of proinflammatory cytokines is higher in peri-pancreatic adipocytes than in other adipose tissues due to SENP1 deletion.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "PROINFLAMMATORY CYTOKINES IL-6, TNF-\u0391 AND IL-I\u0392"}, {"entity_type": "ORGAN/TISSUE", "description": "SENP1-deficient mice exhibit increased cytokine levels and myocarditis specifically in their pancreases, which are indicative of a prevalent diabetic phenotype.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "PANCREAS"}, {"entity_type": "CHEMOKINE", "description": "CCL5 plays a role in directing recruitment of immune cells to inflammation sites or autoimmune targets contributing to type-1 diabetes development.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "CCL5 (RANTES)"}, {"entity_type": "RECEPTOR TYPE", "description": "CCL5 binds to CCR5 receptors, playing an important part in recruiting immune cells that contribute to autoimmune responses and disease onset.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "CCR5 LIGAND"}, {"entity_type": "PATHOLOGICAL PROCESS", "description": "The upregulation of specific chemokines like CCL5 in pancreatic islets could lead to a cascade of events leading to the development of type-1 diabetes.", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "id": "TYPE-1 DIABETIC DEVELOPMENT"}, {"entity_type": "ADIPOCYTE", "description": "The adipocytes isolated from mice lacking the SENP1 gene at the adipose tissue of pancreas (SENP1-aP2KO). These adipocytes are characterized by smaller cell size and higher number of cells compared to Ctrl adipocytes.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "SENP1-AP2KO PANCREATIC ADIPOCYTES"}, {"entity_type": "ADIPOCYTE", "description": "Adipocytes isolated from normal mice, used as a control for the SENP1-aP2KO mice experiments.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "CONTROL ADIPOCYTES"}, {"entity_type": "ANTIBODY", "description": "Neutralizing antibody against IL-6. Used to reduce upregulation of CCL5 expression.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "IL-6 ANTIBODY"}, {"entity_type": "ANTIBODY", "description": "Neutralizing antibody against TNF, used together with the IL-6 antibody in experiments to reduce upregulation of CCL5 expression.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "TNF ANTIBODY"}, {"entity_type": "ANTIBODY", "description": "Neutralizing antibody against IL-1\u03b2, used together with the IL-6 and TNF antibodies in experiments to reduce upregulation of CCL5 expression.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "IL-1\u0392 ANTIBODY"}, {"entity_type": "CYTOKINE", "description": "High levels of local pro-inflammatory cytokines secreted from peripancreatic adipocytes (PATs) induces the expression of CCL5, a chemokine that promotes infiltration of immune cells into pancreas.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "PROINFLAMMATORY CYTOKINES"}, {"entity_type": "CHEMOKINE", "description": "CCL5 is expressed in response to high local concentrations of pro-inflammatory cytokines from PATs. It recruits immune cells which cause damages on \u03b2-cells in pancreas.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "CCL5"}, {"entity_type": "ADIPOCYTE", "description": "The adipocytes located around the pancreas that are involved in the production and secretion of pro-inflammatory cytokines such as IL-6, TNF or IL-1\u03b2. Their augmented release contributes to damages on \u03b2 cells.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "PERI-PANCREATIC ADIPOCYTES (PATS)"}, {"entity_type": "CYTOKINE", "description": "High local concentrations of cytokines secreted by PATs upregulate CCL5 expression and leads to infiltration of immune cells into pancreas, causing damages on \u03b2 cells.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "LOCAL PROINFLAMMATORY CYTOKINES"}, {"entity_type": "PATHOLOGY", "description": "The process where pancreatic tissues are damaged due to the direct effect of adipocyte-derived cytokines on pancreatic islets.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "PANCREATIC DISRUPTION"}, {"entity_type": "ADIPOCYTE", "description": "Adipocytes isolated from mice with a deletion in the SENP1 gene at the peripheric pancreas (SENP1-aP2KO). Show characteristics of reduced adipogenesis and increased cytokine production.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "PANCREATIC ADIPOCYTES FROM SENP1-AP2KO MICE"}, {"entity_type": "PROTEIN", "description": "The protein form of PPAR\u03b3, crucial for repressing inflammatory responses via SUMOylation pathway.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "PPAR\u0393 PROTEIN"}, {"entity_type": "KINASE", "description": "In the adipocytes with the SENP1 knockdown, phosphorylated IKK levels were increased markedly.<SEP>The activated form of I\u03baB Kinase which gets phosphorylated during the activation process of NF-\u03baB pathway.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "PHOSPHORYLATED IKK"}, {"entity_type": "SUBUNIT OF NF-\u039aB COMPLEX", "description": "The RelA subunit of the NF-\u03baB protein complex that binds to DNA and activates gene transcription during immune response and inflammation processes.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4", "id": "NF-\u039aB SUBUNIT P65/RELA"}, {"entity_type": "PROTEIN", "description": "The RelA subunit of the NF-\u03baB protein complex that binds to DNA and activates gene transcription during immune response and inflammation processes.<SEP>The gene expressions of proinflammatory cytokines from adipocytes were significantly higher in the SENP1 knockdown cells.", "source_id": "chunk-eaf655c4dd9244523b048a122e232be4<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "NF-\u039aB P65/RELA"}, {"entity_type": "GENE", "description": "The adipocytes were transfected with a Ctrl or SENP1 siRNA.", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "SENP1 SHORT INTERFERING RNA (SIRNA)"}, {"entity_type": "PROTEIN", "description": "IKK, I\u03baB\u03b1 and NF-\u03baB p65/RelA levels were all significantly upregulated in the SENP1 knockdown cells.", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "I\u039aB\u0391"}, {"entity_type": "CYTOKINE", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "IL-1 \u0392"}, {"entity_type": "GENE", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "SENP2"}, {"entity_type": "GENE", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "SENP3"}, {"entity_type": "GENE", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "SENP5"}, {"entity_type": "GENE", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "SENP6"}, {"entity_type": "GENE", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "SENP7"}, {"entity_type": "PROTEIN", "description": "The NEMO protein binds Ub through a \u03b2\u03b2\u03b1 structure with specific interactions involving Pro21-Val22 residues in the \u03b2-loop replaced by Pro398-Lys399 and Glu636-Lys637 respectively, indicating distinct modes of interaction compared to other UBZs.<SEP>(", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2<SEP>chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "NEMO"}, {"entity_type": "PROTEIN", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "IKK"}, {"entity_type": "PROTEIN", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "P65/RELA"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "DNA DAMAGE AGENT ETOPOSIDE VP16"}, {"entity_type": "CELL", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "ADIPOCYTES ISOLATED FROM SENP1-AP2KO MICE"}, {"entity_type": "PROTEIN", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "PHOSPHORYLATED IKK AND P65/RELA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "CYTOKINE PRODUCTIONS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "DNA DAMAGE INDUCED BY ETOPOSIDE VP16"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "SUMOYLATIONS OF NEMO AND P65/RELA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "INCREASED SUMOYLATION IN ADIPOCYTES WITH SENP1 DELETION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "STRESS RESPONSES SUCH AS OXIDATIVE STRESS, INFLAMMATORY STIMULI AND DNA DAMAGE"}, {"entity_type": "PROTEIN", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "PHOSPHORYLATION OF IKK AND P65/RELA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "BASAL AND STRESS-INDUCED NEMO SUMOYLATION IN ADIPOCYTES WITH SENP1 DELETION"}, {"entity_type": "PROTEIN", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "FLAG-NEMO MUTANTS (NEMO K277R, K309R OR THE DOUBLE MUTANT)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "SUMO-DEFICIENT NEMO MUTANTS"}, {"entity_type": "ROLE IN REGULATING NF-\u039aB SIGNALLING", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "DOMINANT NEGATIVE FORMS OF SUMOYLATED NEMO"}, {"entity_type": "DISEASE", "description": "(", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "id": "CLINICAL RELEVANCE OF ADIPOCYTE-SPECIFIC SENP1-DEFICIENT MICE TO HUMAN T1DM"}, {"entity_type": "MODEL", "description": "The NOD mouse model is widely used for studying type 1 diabetes.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "NON-OBESE DIABETIC (NOD) MOUSE"}, {"entity_type": "DISEASE", "description": "T1DM is an autoimmune disease characterized by the destruction of insulin-producing beta cells in the pancreas.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "TYPE 1 DIABETES MELLITUS (T1DM)"}, {"entity_type": "IMMUNOLOGICAL RESPONSE", "description": "The predominant T-cell response is TH1, leading to inflammation and autoimmunity against beta cells.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "TH1 DOMINANT"}, {"entity_type": "BODY FLUID", "description": "Blood serum collected from NOD mice with diabetes or human T1DM patients contains increased levels of proinflammatory cytokines.<SEP>Serum is a liquid component of blood excluding cells, where levels of IL-8 can be measured.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68<SEP>chunk-8382a49b61215482f9fc75e3a86cb987", "id": "SERUM"}, {"entity_type": "ADIPOSE TISSUE", "description": "PAT stands for perivascular adipose tissue, an important site of inflammation and insulin resistance.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "PAT"}, {"entity_type": "PROTEIN", "description": "Pre-proinsulin is a precursor protein to the production of insulin in beta cells.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "PRE-PROINSULIN"}, {"entity_type": "DISEASE", "description": "An autoimmune disease, like T1DM, occurs when the immune system mistakenly attacks and destroys healthy tissue.", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "id": "AUTOIMMUNE DISEASE"}, {"entity_type": "DIETARY PATTERN", "description": "High Fat Diet increased fat weight in all adipose depots, including epididymal fat and PATs.<SEP>|>Metabolic Pathway/", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 47}]", "id": "HFD"}, {"entity_type": "GENOMIC VARIANT/MOLECULAR INTERACTION/ENVIRONMENTAL FACTOR", "description": "Control condition under normal chow diet.", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "id": "CTRL"}, {"entity_type": "DISEASE/GENOME/GENE/PROTEIN/MOLECULAR INTERACTION/CELL", "description": "SENP1 knockout mice with a patern-specific deletion of the SEPP1 gene, impacting adipose tissue development and function.", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "id": "SENP1-AP2KO"}, {"entity_type": "ADIPOSE TISSUE", "description": "A type of adipose tissue found around the testes and bladder in vertebrates.", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "id": "EPIDIDYMAL FAT"}, {"entity_type": "ASSAY/", "description": "|>Cellular Process/", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "id": "ITT"}, {"entity_type": "CELLULAR PROCESS/ENVIRONMENTAL FACTOR/", "description": "|>Disease/Diabetes", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "id": "BASAL INSULIN"}, {"entity_type": "CELLULAR PROCESS/MOLECULAR INTERACTION/EXPERIMENTAL METHOD/", "description": "|>Functional Test/", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "id": "GLUCOSE-INSULIN COEFFICIENT"}, {"entity_type": "DISEASE/INFLAMMATION/", "description": "|>Metabolic Pathway/Cellular Process/Environmental Factor/", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 47}]", "id": "HEPATIC STEATOSIS"}, {"entity_type": "DISEASE/INFLAMMATION/", "description": "|>Metabolic Pathway/Cellular Process/Environmental Factor/", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "id": "FATTY LIVER DISEASE"}, {"entity_type": "DISEASE", "description": "The SNP is linked to type 1 diabetes.<SEP>|>Inflammation/Cellular Process", "source_id": "chunk-174831b97ed170039b374ba254d4f506<SEP>chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "TYPE 1 DIABETES"}, {"entity_type": "CELLULAR PHENOMENON/PATHOLOGICAL PROCESS/", "description": "|>Disease/Type 1 Diabetes", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "id": "BETA CELL LOSS"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "the adipocyte-specific deletion of SENP1)", "entity_type": "UNKNOWN", "id": "SENP1 DEFICIENCY"}, {"source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "description": "developed a diabetic phenotype (Fig.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 48}]", "id": "T1DM"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7<SEP>chunk-fc33d6399ec56e65d007269407f1953d", "description": "Immune cell-specific effector T-cell invasion in the pancreatic tissue attacks the islets and induces cell death.)<SEP>Pathway/Mechanism of Disease/Condition", "entity_type": "BIOLOGICAL PROCESS/", "id": "AUTOIMMUNE RESPONSE"}, {"source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "description": "There was a sharp decline in insulin staining and CD31+ vascular area in the islets in SENP1-aP2KO mice (Fig. 3d with quantifications in Fig. 3e), indicating a loss of function in the \u03b2 cell with disrupted vasculatures.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 28}]", "id": "\u0392 CELLS"}, {"source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "description": "The distribution of immune cells in the pancreases was determined by fluorescence-activated cell sorting (FACS) analyses with specific surface markers. MHCII+CD11b+CD11c\u2212 macrophages, CD3e+ T cells and CD19+ B cells were markedly increased in the pancreases of SENP1-aP2KO versus the Ctrl mice.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 28}]", "id": "MHCII+CD11B+CD11C\u2212 MACROPHAGES"}, {"source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "description": "We observed that Th1 (IFN-\u03b3+CD4+) and Th17 (IL-17+CD4+) effector T-cell subsets expanded concomitant with reduction of regulatory T-cell subset (CD4+CD25+Foxp3+) in the pancreatic lymph nodes of SENP1-aP2KO mice.)", "entity_type": "UNKNOWN", "id": "TH1 (IFN-\u0393+CD4+) AND TH17 (IL-17+CD4+) EFFECTOR T-CELL SUBSETS"}, {"source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "description": "We also observed an increase in activated dendritic cells in the lymph node of pancreases.)", "entity_type": "UNKNOWN", "id": "ACTIVATED DENDRITIC CELLS"}, {"source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "description": "Proinflammatory cytokine overproduction serves as the main reason for immunologic tolerance break leading to diabetes.)", "entity_type": "UNKNOWN", "id": "CYTOKINES IL-6, TNF-\u0391 AND IL-I\u0392"}, {"source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "description": "Proinflammatory cytokine overproduction serves as the main reason for immunologic tolerance break leading to diabetes.)", "entity_type": "UNKNOWN", "id": "EFFECT ON"}, {"source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "description": "Deletion of SENP1 gene strongly enhances the expression of pro-inflammatory cytokines in peri-pancreatic adipocytes compared to other adipose depots.)", "entity_type": "UNKNOWN", "id": "ENHANCEMENT OF"}, {"source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "description": "Deletion of SENP1 gene strongly enhances the expression of pro-inflammatory cytokines in peri-pancreatic adipocytes compared to other adipose depots.)", "entity_type": "UNKNOWN", "id": "SENP1 DELETION"}, {"source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "description": "Increased proinflammatory cytokines, cardiac function changes and vascular reactivity issues are potential causes leading to diabetes in SENP1-deficient mice.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 1}, {level: 1, cluster: 28}]", "id": "CAUSE OF"}, {"source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "description": "The expression of CCL5 mRNA significantly increases in islets due to secreted cytokines from peri-pancreatic adipocytes, specifically before the onset of diabetes in SENP1-deficient mice.)", "entity_type": "UNKNOWN", "id": "UPREGULATION AND LOCALIZATION"}, {"source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "description": "SENP1 siRNA increased phosphorylation of IKK in adipocytes.)", "entity_type": "UNKNOWN", "id": "REGULATE PHOSPHO-IKK"}, {"source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "description": "SENP1 siRNA increased phosphorylation of IKK in adipocytes.)", "entity_type": "UNKNOWN", "id": "SENP1 SIRNA"}, {"source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "description": "The SENP1 knockdown cells showed significantly higher gene expressions of IL-6, TNF-\u03b1 and IL-1\u03b2 compared to Ctrl cells.)", "entity_type": "UNKNOWN", "id": "INCREASE PROINFLAMMATORY CYTOKINE EXPRESSION"}, {"source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "description": "The SENP1 knockdown cells showed significantly higher gene expressions of IL-6, TNF-\u03b1 and IL-1\u03b2 compared to Ctrl cells.)", "entity_type": "UNKNOWN", "id": "SENP1 KNOCKDOWN CELLS"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Age-dependent reduction in SENP1 expression correlates with the progression of spontaneous diabetes in NOD mice.)", "entity_type": "UNKNOWN", "id": "HYPERGLYCAEMIA"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Age-dependent reduction in SENP1 expression correlates with the progression of spontaneous diabetes in NOD mice.)", "entity_type": "UNKNOWN", "id": "SENP1 EXPRESSION"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Increased plasma levels of proinflammatory cytokines are significantly higher in T1DM patients compared to healthy controls.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 48}]", "id": "PRO-INFLAMMATORY CYTOKINE LEVELS"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "The Th1 dominant response due to IFN-\u03b3-induced insulitis is associated with diabetes progression in NOD mice.)", "entity_type": "UNKNOWN", "id": "DIABETES PROGRESSION"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "The Th1 dominant response due to IFN-\u03b3-induced insulitis is associated with diabetes progression in NOD mice.)", "entity_type": "UNKNOWN", "id": "TH1 DOMINANT RESPONSE"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Serum from human T1DM patients and NOD mice induces disruption and cell death in pancreatic islets.)", "entity_type": "UNKNOWN", "id": "PANCREATIC CELL DEATH"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Serum from human T1DM patients and NOD mice induces disruption and cell death in pancreatic islets.)", "entity_type": "UNKNOWN", "id": "SERUM FROM T1DM PATIENTS"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Prophylactic insulin therapy only partially ameliorates the development of diabetes phenotypes in SENP1-deficient mice.)", "entity_type": "UNKNOWN", "id": "DIABETES PHENOTYPES"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Prophylactic insulin therapy only partially ameliorates the development of diabetes phenotypes in SENP1-deficient mice.)", "entity_type": "UNKNOWN", "id": "PROPHYLACTIC INSULIN THERAPY"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Administration of an NF-\u03baB activation inhibitor reduces blood glucose levels and attenuates islet damages in SENP1-aP2KO mice.)", "entity_type": "UNKNOWN", "id": "BLOOD GLUCOSE"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "Administration of an NF-\u03baB activation inhibitor reduces blood glucose levels and attenuates islet damages in SENP1-aP2KO mice.)", "entity_type": "UNKNOWN", "id": "NF-\u039aB INHIBITORS"}, {"source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "description": "HFD conditions augment the diabetic phenotype in SENP1 deficient mice, affecting fat weight, cell size and number of adipocytes.)", "entity_type": "UNKNOWN", "id": "HIGH-FAT DIET (HFD)"}, {"source_id": "chunk-174831b97ed170039b374ba254d4f506", "description": "|>T2DM", "entity_type": "UNKNOWN", "id": "DISEASE/T1DM/"}, {"source_id": "chunk-174831b97ed170039b374ba254d4f506", "description": "|>SENP1-aP2KO", "entity_type": "UNKNOWN", "id": "PROINFLAMMATORY CYTOKINES/PATS/"}, {"source_id": "chunk-174831b97ed170039b374ba254d4f506", "description": "|>SENP1-aP2KO", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 47}]", "id": "PATS/"}, {"source_id": "chunk-174831b97ed170039b374ba254d4f506", "description": "|>HFD/", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 24}, {level: 1, cluster: 47}]", "id": "LIVER INSULIN RESISTANCE"}, {"source_id": "chunk-174831b97ed170039b374ba254d4f506", "description": "|>Hyperglycaemia", "entity_type": "UNKNOWN", "id": "INSULIN RESISTANCE/"}, {"source_id": "chunk-174831b97ed170039b374ba254d4f506", "description": "|>SENP1-aP2KO", "entity_type": "UNKNOWN", "id": "HFD/"}, {"source_id": "chunk-174831b97ed170039b374ba254d4f506", "description": "|>Comorbidity", "entity_type": "UNKNOWN", "id": "GENETIC TRAIT/T2DM/"}, {"source_id": "chunk-174831b97ed170039b374ba254d4f506", "description": "|>Insulin Deficiency", "entity_type": "UNKNOWN", "id": "TYPE 1 DIABETES/"}, {"entity_type": "GENOMIC VARIANT", "description": "The ApaI variant (rs7975232) evaluated for association with ovarian cancer risk.", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "id": "APAI"}, {"entity_type": "GENOMIC VARIANT", "description": "The BsmI variant (rs1544410) associated with ovarian cancer risk.", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "id": "BSMI"}, {"entity_type": "POPULATION", "description": "Studies conducted on the Caucasian population.<SEP>When the studies were restricted to Caucasians, no significant association between specific genetic variants and ovarian cancer was found.", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c<SEP>chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "CAUCASIANS"}, {"entity_type": "GENOMIC VARIANT", "description": "The FokI variant (rs2228570 or rs10735810) showed an increased risk of ovarian cancer.", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "id": "FOKI"}, {"entity_type": "GENOMIC VARIANT", "description": "Cdx-2 is a gene that influences the risk of ovarian cancer based on genetic variations.<SEP>The Cdx-2 variant evaluated for association with ovarian cancer risk.", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c<SEP>chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "CDX-2"}, {"entity_type": "GENOMIC VARIANT", "description": "A specific genomic variant rs11568820 in Cdx-2 has been evaluated for its association with ovarian cancer risk.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "RS11568820"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "A Japanese subgroup was part of the study, which showed that this variant may not be associated with risk when conducting the meta-analysis for a Japanese population.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "JAPANESE SUBGROUP"}, {"entity_type": "SUBGROUP STUDY", "description": "The NECC subgroup indicated no significant association between Cdx-2 and ovarian cancer risk based on their genetic data analysis.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "NECC SUBGROUP"}, {"entity_type": "GENE", "description": "TaqI is a gene that was studied for its association with ovarian cancer risk based on genetic variations.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "TAQI"}, {"entity_type": "GENOMIC VARIANT", "description": "A specific genomic variant rs731236 in TaqI has been evaluated for its relation to the risk of ovarian cancer.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "RS731236"}, {"entity_type": "SUBGROUP STUDY", "description": "The Northern Sweden subgroup showed that the variant had no significant effect on the risk of ovarian cancer when conducting a separate analysis.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "NORTHERN SWEDEN SUBGROUP"}, {"entity_type": "RESEARCHER/STUDY", "description": "This subgroup study by Clendenen et al. in Northern Sweden contributed to the overall data, but did not show any association with the variant and ovarian cancer risk.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "CLENDENEN ET AL."}, {"entity_type": "ANALYSIS METHODOLOGY", "description": "The dominant genetic model was used to evaluate if there\'s an increase in risk associated with the presence of specific variants.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "GENETIC MODEL (DOMINANT)"}, {"entity_type": "STATISTICAL TEST", "description": "The sensitivity tests showed that no single study had a considerable influence on the overall risk estimates for the variants in question.", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "id": "SENSITIVITY TESTS"}, {"source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "description": "The FokI variant was significantly associated with an increased risk of ovarian cancer.)", "entity_type": "UNKNOWN", "id": "OVARIAN CANCER RISK"}, {"entity_type": "GENOMIC VARIANT", "description": "The TT genotype of IL-1A -889C>T had a frequency of 1.37 in HNP patients compared to controls.<SEP>The TT genotype of IL-1A -889C>T was more frequent in HNP patients than controls.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "IL-1A -889C>T"}, {"entity_type": "GENOMIC VARIANT", "description": "The CT genotype and T allele of TNF-A 238G>A were more frequent in healthy controls compared to HNP cases.<SEP>The GG genotype, GA genotype and AA genotype of TNF-A 238G>A did not show any significant association with the risk for HNP.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "TNF-A 238G>A"}, {"entity_type": "GENOMIC VARIANT", "description": "No difference was observed in the distribution of VDR TaqI genotypes between HNP patients and healthy controls.<SEP>TT vs. TC+CC, TC vs. TT+CC and OR: 0.94; 95%CI: 0.69, 1.28 were not associated with susceptibility to degenerative disc disease HNP.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "VDR TAQI"}, {"entity_type": "DISEASE", "description": "The results did not show a significant association between the polymorphisms of IL-1A, TNF-A and VDR and the risk of HNPs.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "HNP"}, {"entity_type": "GENOTYPE MODELS", "description": "No association was observed between genotypes models and the risk of HNP for IL-1A -889C>T polymorphism.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "CC VS. CT+TT, CT VS. CC+TT AND TT VS. CT+CC"}, {"entity_type": "GENOTYPE MODELS", "description": "Pooled estimates for allele distribution and genotype models of TNF-A 238G>A also had no significant association with the risk for HNP.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "GG VS. GA+AA, GA VS. GG+AA AND AA VS. GG+GA"}, {"entity_type": "GENOTYPE MODELS", "description": "None of the genotypes for VDR TaqI (rs731236) were associated with susceptibility to degenerative disc disease HNP.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "TT VS. TC+CC, TC VS. TT+CC AND OR: 2.65; 95%CI: 0.60, 11.85"}, {"entity_type": "ALLELE", "description": "No significant association was observed between the frequency of C allele and T allele in relation to the risk of HNPs for IL-1A -889C>T polymorphism.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "C ALLELE"}, {"entity_type": "GENOMIC VARIANT", "description": "A specific genomic variant named rs1800587 was studied for its association with IL-1A.<SEP>The study mentioned that a particular genetic variant, rs1800587, was associated with IL-1A.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "RS1800587"}, {"entity_type": "GENOMIC VARIANT", "description": "Two studies on TNF-A gene used genomic variation rs361525 to analyze its role in health conditions.<SEP>Two studies were included that investigated the TNF-A gene using a genomic variant denoted as rs361525.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "RS361525"}, {"entity_type": "GENOMIC VARIANT", "description": "Nine research articles utilized the VDR gene polymorphism, referred to as rs731236, for studying its association with different diseases.<SEP>Nine studies considered variations in VDR using this specific polymorphism, labeled as rs731236.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "VDR (RS731236)"}, {"entity_type": "GENE", "description": "The research focused on a genetic variant in IL-1A, which is related to inflammatory responses.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "IL-1A"}, {"entity_type": "GENE", "description": "Studies included genomic variations in TNF-A gene, which are involved in immune response and inflammation.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "TNF-A"}, {"entity_type": "GENE", "description": "Nine articles investigated variations within the vitamin D receptor (VDR) gene, denoted as rs731236, to understand its link with degenerative disc disease.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "VDR"}, {"entity_type": "GENOTYPE", "description": "The TT genotype of IL-1A -889C>T was examined for its frequency in HNP patients.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "IL-1A -889C>T TT GENOTYPE"}, {"entity_type": "GENOTYPES", "description": "The study evaluated the association between these genotypes of TNF-A 238G>A variant and HNP risk.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "TNF-A 238G>A GG GENOTYPE, GA GENOTYPE AND AA GENOTYPE"}, {"entity_type": "GENOTYPES COMPARISON", "description": "Comparison was made between different genotypes for VDR TaqI to understand their impact on susceptibility to degenerative disc disease HNP.", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "id": "VDR TAQI TT VS. TC+CC, TC VS. TT+CC"}, {"source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "description": "The TT genotype of IL-1A -889C>T was found to be more frequent in HNP patients.)", "entity_type": "UNKNOWN", "id": "HNP PATIENT FREQUENCY"}, {"source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "description": "CT and T allele of TNF-A 238G>A were found more frequent among healthy controls compared to HNP cases.)", "entity_type": "UNKNOWN", "id": "HEALTHY CONTROL FREQUENCY"}, {"source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "description": "CT and T allele of TNF-A 238G>A were found more frequent among healthy controls compared to HNP cases.)", "entity_type": "UNKNOWN", "id": "TNF-A 238G>A CT GENOTYPE AND T ALLELE"}, {"source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "description": "Genotypes of VDR TaqI were not associated with susceptibility to degenerative disc disease HNP.)", "entity_type": "UNKNOWN", "id": "NO ASSOCIATION WITH HNP"}, {"source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "description": "Genotypes of VDR TaqI were not associated with susceptibility to degenerative disc disease HNP.)", "entity_type": "UNKNOWN", "id": "VDR TAQI TT VS. TC+CC, TC VS. TT+CC AND OR: 2.65; 95%CI: 0.60, 11.85"}, {"source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "description": "The genotypes models of IL-1A -889C>T were not associated with the risk for HNP.)", "entity_type": "UNKNOWN", "id": "IL-1A -889C>T GENOTYPES CC VS. CT+TT, CT VS. CC+TT AND TT VS. CT+CC"}, {"source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "description": "The genotypes models of IL-1A -889C>T were not associated with the risk for HNP.)", "entity_type": "UNKNOWN", "id": "NO ASSOCIATION WITH RISK"}, {"source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "description": "Pooled estimates of TNF-A 238G>A genotypes were not associated with the risk for HNP.)", "entity_type": "UNKNOWN", "id": "TNF-A 238G>A GENOTYPES GG VS. GA+AA, GA VS. GG+AA AND AA VS. GG+GA"}, {"source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "description": "None of the genotypes for VDR TaqI were associated with susceptibility to degenerative disc disease HNP.)", "entity_type": "UNKNOWN", "id": "VDR TAQI GENOTYPES TT VS. TC+CC, TC VS. TT+CC AND OR: 1.01; 95%CI: 0.55, 1.85"}, {"source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "description": "There was no significant association between the C allele and T allele in relation to the risk of HNPs for IL-1A -889C>T.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION WITH RISK"}, {"source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "description": "There was no significant association between the C allele and T allele in relation to the risk of HNPs for IL-1A -889C>T.)", "entity_type": "UNKNOWN", "id": "C ALLELE OF IL-1A -889C>T"}, {"entity_type": "STATISTICAL METHOD", "description": "A method used in the study to combine multiple independent studies on maternal and fetal B12/folate levels and offspring birthweight.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "IVW ANALYSIS"}, {"entity_type": "STATISTICAL ANALYSIS", "description": "Analyses performed to test for robustness of results against unmeasured confounding factors.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "SENSITIVITY ANALYSES"}, {"entity_type": "STATISTICAL METHOD", "description": "A technique used in the study to assess how sensitive the causal effect estimates are to individual SNPs.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "LEAVE-ONE-OUT ANALYSIS"}, {"entity_type": "STATISTICAL TEST", "description": "A test conducted to check for variation among results from different studies or groups within a study.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "HETEROGENEITY TEST"}, {"entity_type": "GENETIC VARIANT", "description": "The MMACHC variant (rs12272669) which had little effect in pregnant women at 28 weeks of gestation.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "MMACHC VARIANT"}, {"entity_type": "GENETIC VARIANT", "description": "The FUT2 variant (rs602662), expected to have pleiotropic effects based on prior knowledge.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "FUT2 VARIANT"}, {"entity_type": "GENETIC VARIANT", "description": "A single nucleotide polymorphism in the MTHFR gene used for assessing potential heterogeneity and its effect.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "MTHFR SNP RS1801133"}, {"entity_type": "OUTCOME MEASURE", "description": "The weight of the newborn, measured as standard deviations (SD) from the mean weight.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "BIRTHWEIGHT"}, {"entity_type": "OUTCOME MEASURE", "description": "Measures of blood serum levels of vitamin B12, which are assessed for their causal effect on birthweight.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "SERUM B12 CONCENTRATIONS"}, {"entity_type": "OUTCOME MEASURE", "description": "Measures of blood serum folate levels, which are associated with offspring birthweight through MR analysis.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "SERUM FOLATE CONCENTRATIONS"}, {"entity_type": "STATISTICAL ANALYSIS", "description": "Finding no significant evidence for pleiotropic effects from MR Egger regression analysis for both B12 and folate levels on birthweight.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "PLEIOTROPY TEST RESULTS"}, {"entity_type": "CRITICAL STATISTICAL ANALYSIS", "description": "Concern over potential bias due to weak instruments, particularly noted for folate analysis results.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "WEAK INSTRUMENT BIAS"}, {"entity_type": "STUDY FINDING", "description": "Recognition of variability among different study outcomes for B12/folate levels impacting birthweight.", "source_id": "chunk-1d2a8d3ce4f9baae7e1c9c14410045d0", "id": "OUTCOME VARIABILITY"}, {"entity_type": "NUTRIENT", "description": "B12 is a nutrient that affects folate levels and may influence offspring birthweight.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "id": "B12"}, {"entity_type": "DISEASE", "description": "The level of B12 in a mother might influence the offspring\u2019s birthweight, but no causal effect was detected.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "id": "MATERNAL B12 LEVELS"}, {"entity_type": "DISEASE", "description": "Offspring birthweight can be potentially influenced by maternal health conditions and nutrient levels such as folate and B12.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "id": "OFFSPRING BIRTHWEIGHT"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The study attempts to detect a causal effect between maternal and offspring\'s nutritional status (B12 and Folate) and certain health outcomes like birthweight, but found no strong evidence of such effects.", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "id": "CAUSAL EFFECT"}, {"source_id": "chunk-22216738ccbae0a6276bb4499018c434", "description": "B12 and Folate are nutrients that might have a molecular interaction affecting maternal health and offspring development.)", "entity_type": "UNKNOWN", "id": "B12&FOLATE"}, {"source_id": "chunk-22216738ccbae0a6276bb4499018c434", "description": "There is no evidence suggesting a causal effect of maternal B12 levels on offspring birthweight, which indicates that other factors might be more influential in this relationship.)", "entity_type": "UNKNOWN", "id": "CAUSAL RELATIONSHIP"}, {"source_id": "chunk-22216738ccbae0a6276bb4499018c434", "description": "There is no evidence suggesting a causal effect of maternal B12 levels on offspring birthweight, which indicates that other factors might be more influential in this relationship.)", "entity_type": "UNKNOWN", "id": "MATERNAL B12 LEVELS&OFFSPRING BIRTHWEIGHT"}, {"source_id": "chunk-22216738ccbae0a6276bb4499018c434", "description": "There was a small causal effect detected between folate levels and B12 levels suggesting their interplay in maternal health.)", "entity_type": "UNKNOWN", "id": "FOLATE LEVELS&B12 LEVELS"}, {"source_id": "chunk-22216738ccbae0a6276bb4499018c434", "description": "A genomic variant may influence B12 metabolism, affecting its physiological process and levels.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT&B12 METABOLISM"}, {"source_id": "chunk-22216738ccbae0a6276bb4499018c434", "description": "The expression of this transcript could be related to the levels or regulation of nutrients like B12 and folate.)", "entity_type": "UNKNOWN", "id": "TRANSCRIPT&NUTRIENT LEVELS"}, {"source_id": "chunk-22216738ccbae0a6276bb4499018c434", "description": "This environmental factor might correlate with certain dietary patterns that influence nutrient intake.)", "entity_type": "UNKNOWN", "id": "ENVIRONMENTAL FACTOR&DIETARY PATTERN"}, {"source_id": "chunk-22216738ccbae0a6276bb4499018c434", "description": "Alterations in nutrient levels, such as B12 and folate, could potentially affect gestational duration or have been studied for this relationship.)", "entity_type": "UNKNOWN", "id": "NUTRIENT LEVELS&GESTATIONAL DURATION"}, {"entity_type": "GENOMIC VARIANT", "description": "MTHFR C677T is a genetic variant involved in the methionine and folate metabolism pathway.<SEP>Thirteen articles presented LV patients that were examined for MTHFR C677T.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0<SEP>chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "id": "MTHFR C677T"}, {"entity_type": "GENOMIC VARIANT", "description": "The PAI-1 -675 4G/5G polymorphism influences the activity of tissue plasminogen activator (tPA) and fibrinolysis.<SEP>Three studies and two case reports presented assessment of PAI-1 -675 4G/5G in a total of 95 LV patients.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0<SEP>chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "id": "PAI-1 -675 4G/5G"}, {"entity_type": "GENOMIC VARIANT", "description": "Eighty-five LV patients from 15 studies or case reports were evaluated for Prothrombin G20210A.<SEP>Prothrombin G20210A is a common variant in the PROTHO2 gene, influencing blood coagulation.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0<SEP>chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "id": "PROTHROMBIN G20210A"}, {"entity_type": "GENOMIC VARIANT", "description": "Factor V G1691A affects the function of factor V protein involved in the blood clotting process.<SEP>The total carrier rate of Factor V G1691A (14.07%) in LV patients was the second lowest among the tested SNPs.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0<SEP>chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "id": "FACTOR V G1691A"}, {"entity_type": "GENOMIC VARIANT", "description": "Compared to PAI-1 -675 4G/5G, the variant PAI-1 A844G has been rarely assessed in LV patients.", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "id": "PAI-1 A844G"}, {"entity_type": "GENE", "description": "MTHFR A1298C is another genetic variant affecting the MTHFR enzyme function.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "MTHFR A1298C"}, {"entity_type": "NUTRIENT", "description": "DHA, or docosahexaenoic acid, another omega-3 fatty acid critical for brain function and development, particularly in the retina of the eye.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "DHA"}, {"entity_type": "NUTRIENT", "description": "Common components like arachidonic acid are part of a class of essential polyunsaturated fatty acids that may compete with omega-3s for enzymatic activities involved in inflammatory pathways.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "OMEGA-6 FATTY ACIDS"}, {"entity_type": "DIETARY PATTERN", "description": "Consumption of whole grains as part of a dietary pattern provides beneficial nutrients such as fiber, B vitamins, and minerals.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "WHOLE GRAINS"}, {"entity_type": "DIETARY PATTERN", "description": "A vegetarian diet excludes animal-derived foods and relies on fruits, vegetables, legumes, nuts, seeds, and grains for nutrition.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "VEGETARIAN DIET"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Communication between neurons through the release of neurotransmitters at synapses is crucial for cognitive function.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "NEUROTRANSMISSION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The body\'s response to stressful situations can be studied in terms of its physiological and biochemical reactions, including hormonal and inflammatory responses.", "source_id": "chunk-d0476488384aadcae4630e94d982d2e0", "id": "STRESS RESPONSE"}, {"entity_type": "DISEASE", "description": "A condition that can be influenced by both genetic variants and dietary patterns.<SEP>A group of disorders affecting the heart and blood vessels, which can lead to conditions like coronary artery disease, stroke, or congestive heart failure.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-d0476488384aadcae4630e94d982d2e0", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 39}]", "id": "CARDIOVASCULAR DISEASE"}, {"entity_type": "GENE", "description": "MTHFR 677C>T is a gene associated with folate/B12/tHcy metabolism.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "MTHFR 677C>T"}, {"entity_type": "GENE", "description": "Male infertility was positively associated with being homozygous for the choline pathway enzyme gene phosphatidylethanolamine N-methyltransferase (PEMT) M175V minor V allele.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "PEMT M175V"}, {"entity_type": "GENE", "description": "Male infertility was negatively associated with the B12 transcobalamin receptor gene TCblR variant rs173665 in heterozygous subjects.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "TCBLR (CD320)"}, {"entity_type": "NUTRIENT", "description": "Serum folate concentrations were significantly lower among infertile men compared to fertile controls, particularly among non-users of vitamin supplements.<SEP>The study compared serum folate measured by two methods: CDC MBA and CDC HPLC-MS/MS.", "source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8<SEP>chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "SERUM FOLATE"}, {"entity_type": "NUTRIENT", "description": "Red cell folate (RCF) concentrations in non-users of vitamins were also found to be significantly lower among infertile than fertile men, even after adjusting for age and other factors.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "RCF (RED CELL FOLATE)"}, {"entity_type": "NUTRIENT", "description": "While vitamin B12 levels did not show a significant difference between infertile and fertile groups in the non-user cohort, they were lower among all infertile men compared to fertile controls.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "B12 (VITAMIN B12)"}, {"entity_type": "METABOLIC PATHWAY", "description": "Homocysteine concentrations showed an increase in infertile men compared to fertile controls but did not reach statistical significance.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "THCY (HOMOCYSTEINE)"}, {"entity_type": "GENOMIC VARIANT", "description": "The analysis focused on the genotypes of chromosome 12 genes associated with folate/B12/tHcy metabolism.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "CHROMOSOME 12 GENES"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Homozygous for the choline pathway enzyme gene PEMT M175V minor V allele was associated with male infertility.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "CHOLINE PATHWAY ENZYME GENE PHOSPHATIDYLETHANOLAMINE N-METHYLTRANSFERASE (PEMT) M175V MINOR V ALLELE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The heterozygous subjects of the B12 transcobalamin receptor gene TCblR variant rs173665 were negatively associated with male infertility.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "B12 TRANSCOBALAMIN RECEPTOR GENE TCBLR (CD320) VARIANT RS173665"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Serum folate concentrations were analyzed in individuals whose levels were below the study population median to assess their impact on male fertility.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "SERUM FOLATE CONCENTRATION BELOW STUDY POPULATION MEDIAN (14.70 NMOL L\u22121)"}, {"entity_type": "NUTRIENT", "description": "Vitamin B12 levels did not significantly differ between infertile and fertile groups among vitamin non-users.", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "id": "SERUM B12 CONCENTRATION"}, {"source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "description": "The PEMT M175V minor V allele was associated with male infertility status among homozygous individuals.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 43}]", "id": "PEMT M175V MINOR V ALLELE"}, {"source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "description": "The TCblR variant rs173665 was negatively associated with male infertility status among heterozygous subjects.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 43}]", "id": "TCBLR (CD320) VARIANT RS173665 IN HETEROZYGOUS SUBJECTS"}, {"source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "description": "An analysis focusing on individuals with serum folate concentrations below the median revealed no association between these levels and male infertility.)", "entity_type": "UNKNOWN", "id": "SERUM FOLATE CONCENTRATION BELOW STUDY POPULATION MEDIAN"}, {"source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "description": "When focusing on the MTHFR 677C>T variant, individuals with serum folate levels below the study population median showed no association with male infertility.)", "entity_type": "UNKNOWN", "id": "GENE-SPECIFIC NUTRIENT INTERACTION"}, {"source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "description": "When focusing on the MTHFR 677C>T variant, individuals with serum folate levels below the study population median showed no association with male infertility.)", "entity_type": "UNKNOWN", "id": "MTHFR 677C>T VARIANT IN INDIVIDUALS WITH SERUM FOLATE BELOW MEDIAN"}, {"source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "description": "In individuals who did not take vitamin supplements, the PEMT M175V minor V allele and the TCblR rs173665 genetic variants showed significant associations with infertility status.)", "entity_type": "UNKNOWN", "id": "MOLECULAR INTERACTION AND GENETIC POLYMORPHISM"}, {"source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "description": "In individuals who did not take vitamin supplements, the PEMT M175V minor V allele and the TCblR rs173665 genetic variants showed significant associations with infertility status.)", "entity_type": "UNKNOWN", "id": "PEMT M175V MINOR V ALLELE AND TCBLR RS173665 VARIANT IN NON-USERS OF VITAMINS TO FERTILITY STATUS"}, {"entity_type": "GENE", "description": "This enzyme catalyzes the reversible conversion of serine to glycine, transferring methyl groups to tetrahydrofolate (THF), which is transformed into 5,10-methylenetetrahydrofolate (5,10-CH2-THF). During cellular proliferation, this is the predominant pathway for pyrimidine synthesis.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "SHMT1"}, {"entity_type": "GENOMIC VARIANT", "description": "The G allele of this polymorphism shows a significant increase when compared with controls in TMD patients (OR = 3.99; 95% CI 1.72, 9.25; p = 0,002).", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "RS1979277"}, {"entity_type": "GENOMIC VARIANT", "description": "The C allele and the CC genotype show a very significant increase in TMD patients (OR = 2.80; 95% CI 1.51, 5.21; p = 0,013).", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "SHMT1 RS638416"}, {"entity_type": "GENE", "description": "MTHFD1 encodes a protein that catalyzes three interconversion reactions of one-carbon derivatives of THF for the synthesis of methionine, thymidylate, and purines.<SEP>This gene encodes a protein with three distinct enzyme activities: catalyzing reactions of one-carbon derivatives of THF for the synthesis of methionine, thymidylate, and purines. It has a trifunctional enzymatic activity conferred by two larger domains.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "MTHFD1"}, {"entity_type": "GENOMIC VARIANT", "description": "A transition from G to A results in an alteration of ARG-653-GLN associated with susceptibility to neural tube defects sensitive to folates. The T allele was more frequent among TMD patients (OR = 3.09; 95% CI 1.27, 7.50; p = 0,016).", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "RS2236225"}, {"entity_type": "GENOMIC VARIANT", "description": "The null variant of this gene has been associated with an increase in cancer and male infertility possibly due to greater susceptibility to environmental toxins and carcinogens as well as alterations in toxicity and efficiency of certain substances.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "GSTM1 NULL POLYMORPHISM"}, {"entity_type": "GENOMIC VARIANT", "description": "The substitution of A by G results in the enzyme with less affinity for the substrate. An increase of the A allele (OR = 2.35; 95% CI 1.10, 5.00; p = 0.037) was observed in TMD patients.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "RS1801394"}, {"entity_type": "GENOMIC VARIANT", "description": "The VNTR polymorphism presents between 2 to 11 repeats in exon 3 of DRD4, which encodes the third intracellular receptor loop and is characterized as short (S-DRD4) or long (L-DRD4) alleles based on the number of repetitions. Statistical differences were observed in genotype frequencies but no significant increase was found for S-DRD4 allele.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "DRD4-48BPTR"}, {"entity_type": "GENE", "description": "MTRR is responsible for the regeneration of Methionine Synthase (MTR) using SAM as a methyl donor.", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "id": "MTRR"}, {"entity_type": "NUTRIENT", "description": "The study compared the concentrations of 5-methylTHF under different laboratory conditions.<SEP>Two types of IH-CAL were used by the laboratories: one using 5-methylTHF and another using folic acid.", "source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8<SEP>chunk-257a38a8ce6582c434f31b023b4a9704", "id": "5-METHYLTHF"}, {"entity_type": "CULTURE MEDIUM", "description": "All laboratories used a standardized Folic Acid Casei Medium as their critical reagent.)", "source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "id": "FOLIC ACID CASEI MEDIUM"}, {"entity_type": "NUTRIENT", "description": "Comparison was also done for folic acid as a nutrient in laboratory settings.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "FOLIC ACID"}, {"entity_type": "DISEASE/PHYSIOLOGICAL PROCESS", "description": "The study aimed to compare results with the CDC MBA target concentrations as a reference physiological process or health standard.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "CDC MBA TARGET CONCENTRATION"}, {"entity_type": "TRANSCRIPT", "description": "<SEP>S-FOL is one of the two types of folic acid used in laboratory analyses, along with WB-FOL.", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c<SEP>chunk-257a38a8ce6582c434f31b023b4a9704", "id": "S-FOL"}, {"entity_type": "TRANSCRIPT", "description": "<SEP>WB-FOL represents another type of folic acid utilized for similar assessments as S-FOL.", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c<SEP>chunk-257a38a8ce6582c434f31b023b4a9704", "id": "WB-FOL"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The CDC-MO condition is an environment-related factor in laboratory settings, affecting results compared to other conditions.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "CDC MO CONDITION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The CDC-CAL condition represents another environmental influence on laboratory test results when compared to the CDC MO condition.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "CDC CAL CONDITION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The IH-MO condition signifies yet another specific environmental context in which tests were conducted.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "IH MO CONDITION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The IH-CAL condition is a different environmental factor tested, distinct from CDC conditions.", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "id": "IH CAL CONDITION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "<SEP>Microorganisms play a role in the variability of test results and could influence outcomes when comparing different reagents.", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c<SEP>chunk-257a38a8ce6582c434f31b023b4a9704", "id": "MICROORGANISM"}, {"entity_type": "DISEASE", "description": "", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "id": "CDC MBA TARGET VALUE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "id": "GROUP 2 LABORATORIES"}, {"entity_type": "DIETARY PATTERN", "description": "", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "id": "FOLIC ACID CALIBRATOR"}, {"entity_type": "REAGENT", "description": "", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "id": "IH REAGENTS"}, {"entity_type": "REAGENT", "description": "", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "id": "CDC REAGENTS"}, {"entity_type": "CONTROL SUBSTANCE", "description": "", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "id": "CALIBRATOR"}, {"entity_type": "EXPERIMENTAL SETUP", "description": "", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "id": "HARMONIZED CONDITION"}, {"source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "description": "A high Pearson correlation (r = 0.999) was observed between the results of the CDC MBA and CDC HPLC-MS/MS methods for serum folate measurement.)", "entity_type": "UNKNOWN", "id": "CDC HPLC-MS/MS"}, {"source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "description": "A high Pearson correlation (r = 0.999) was observed between the results of the CDC MBA and CDC HPLC-MS/MS methods for serum folate measurement.)", "entity_type": "UNKNOWN", "id": "CDC MBA"}, {"source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "description": "All participating laboratories reported their results using different procedures, such as sample dilution techniques and incubation periods.)", "entity_type": "UNKNOWN", "id": "LABORATORY PROCEDURES"}, {"source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "description": "All participating laboratories reported their results using different procedures, such as sample dilution techniques and incubation periods.)", "entity_type": "UNKNOWN", "id": "SERUM SAMPLES"}, {"source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "description": "The use of different sample dilutions (1:100 for serum and 1:140 for whole-blood hemolysate by CDC recommended, or lab-specific dilution) influenced the within-laboratory imprecision.)", "entity_type": "UNKNOWN", "id": "LABORATORY VARIABILITY"}, {"source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "description": "The use of different sample dilutions (1:100 for serum and 1:140 for whole-blood hemolysate by CDC recommended, or lab-specific dilution) influenced the within-laboratory imprecision.)", "entity_type": "UNKNOWN", "id": "SAMPLE DILUTIONS"}, {"source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "description": "Labs incubated their assay plates at 37\u00b0C for 42\u201345 h; an exception was Lab #3 which used only an 18-h incubation period.)", "entity_type": "UNKNOWN", "id": "37\u00b0C INCUBATION"}, {"source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "description": "Labs incubated their assay plates at 37\u00b0C for 42\u201345 h; an exception was Lab #3 which used only an 18-h incubation period.)", "entity_type": "UNKNOWN", "id": "ASSAY STABILITY"}, {"source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "description": "Most laboratories read their plates at 590 nm and carefully controlled timing to ensure consistency across all plates.)", "entity_type": "UNKNOWN", "id": "DATA CONSISTENCY"}, {"source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "description": "Most laboratories read their plates at 590 nm and carefully controlled timing to ensure consistency across all plates.)", "entity_type": "UNKNOWN", "id": "READ PLATES AT 590 NM"}, {"source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "description": "A third-order polynomial regression was most commonly used for calibration curve fitting, but other methods were reported as well.)", "entity_type": "UNKNOWN", "id": "CALIBRATION ACCURACY"}, {"source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "description": "A third-order polynomial regression was most commonly used for calibration curve fitting, but other methods were reported as well.)", "entity_type": "UNKNOWN", "id": "POLYNOMIAL REGRESSION FIT"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Comparison between two types of nutrients (5-methylTHF vs. folic acid) to evaluate their impact on laboratory test results.)", "entity_type": "UNKNOWN", "id": "5-METHYLTHF/FOLIC ACID"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Comparison between two types of nutrients (5-methylTHF vs. folic acid) to evaluate their impact on laboratory test results.)", "entity_type": "UNKNOWN", "id": "NUTRIENT/NUTRIENT RELATIONSHIP"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "S-FOL\'s comparison against the CDC MBA target value as a reference physiological or health standard.)", "entity_type": "UNKNOWN", "id": "CDC MBA TARGET CONCENTRATION/S-FOL"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "S-FOL\'s comparison against the CDC MBA target value as a reference physiological or health standard.)", "entity_type": "UNKNOWN", "id": "DISEASE/PHYSIOLOGICAL PROCESS-NUTRIENT RELATIONSHIP"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Similar relationship for WB-FOL in comparison with the CDC MBA target.)", "entity_type": "UNKNOWN", "id": "CDC MBA TARGET CONCENTRATION/WB-FOL"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Comparison of two transcript forms (S-FOL and WB-FOL) in laboratory conditions.)", "entity_type": "UNKNOWN", "id": "S-FOL/WB-FOL"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Comparison of two transcript forms (S-FOL and WB-FOL) in laboratory conditions.)", "entity_type": "UNKNOWN", "id": "TRANSCRIPT/TRANSCRIPT RELATIONSHIP"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Relation between two different environmental contexts tested for their impact on laboratory results (CDC MO vs CDC CAL).)", "entity_type": "UNKNOWN", "id": "CDC MO CONDITION/CDC CAL CONDITION"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Relation between two different environmental contexts tested for their impact on laboratory results (CDC MO vs CDC CAL).)", "entity_type": "UNKNOWN", "id": "ENVIRONMENTAL FACTOR/ENVIRONMENTAL FACTOR RELATIONSHIP"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Comparison of outcomes influenced by distinct environmental conditions (IH MO and IH CAL). )", "entity_type": "UNKNOWN", "id": "IH MO CONDITION/IH CAL CONDITION"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Role of microorganisms in influencing S-FOL results when compared against the CDC MBA target.)", "entity_type": "UNKNOWN", "id": "MICROORGANISM/S-FOL"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Role of microorganisms in influencing S-FOL results when compared against the CDC MBA target.)", "entity_type": "UNKNOWN", "id": "MOLECULAR INTERACTION-NUTRIENT RELATIONSHIP"}, {"source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "description": "Similar interaction for WB-FOL and microorganisms during comparisons.)", "entity_type": "UNKNOWN", "id": "MICROORGANISM/WB-FOL"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "1. When all labs used IH reagents, the mean difference to CDC MBA target decreased.)", "entity_type": "UNKNOWN", "id": "USE OF IH REAGENTS"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "When CDC reagents were used by all laboratories, the mean \u00b1 SD difference to the CDC MBA target decreased.)", "entity_type": "UNKNOWN", "id": "USE OF CDC REAGENTS"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "The limits indicate where 95% of lab results are expected compared with CDC MBA when using CDC reagents.)", "entity_type": "UNKNOWN", "id": "LIMITS OF AGREEMENT"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "The limits indicate where 95% of lab results are expected compared with CDC MBA when using CDC reagents.)", "entity_type": "UNKNOWN", "id": "RANGE WITHIN WHICH 95% OF RESULTS EXPECTED"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "All labs, except #6 and #9 showed good agreement under harmonized conditions.)", "entity_type": "UNKNOWN", "id": "HARMONIZED CONDITION AND SAME MICROORGANISM/CALIBRATOR"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "All labs, except #6 and #9 showed good agreement under harmonized conditions.)", "entity_type": "UNKNOWN", "id": "REASONABLE AGREEMENT"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "Differences were expected to be approximately \u00b110\u201330% for S-FOL and approximately \u00b120\u201340% for WB-FOL with harmonized conditions.)", "entity_type": "UNKNOWN", "id": "DIFFERENCES BETWEEN DAY 1 AND DAY 2"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "Lab #8 showed discrepancies in WB-FOL results between day 1 and day 2.)", "entity_type": "UNKNOWN", "id": "LAB #8 - WB-FOL"}, {"source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "description": "Lab #8 showed discrepancies in WB-FOL results between day 1 and day 2.)", "entity_type": "UNKNOWN", "id": "LACK OF AGREEMENT"}, {"entity_type": "GENOMIC VARIANT", "description": "GSTP1 gene variant A>G showed a significant link to CSF use during chemotherapy.<SEP>The frequency of genotypes AA, AG, and GG for controls is 50.00%, 44.00%, and 06.00% respectively.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404<SEP>chunk-ab5011584b0230efbf80a98b2a693a0e", "id": "RS1695"}, {"entity_type": "GENOMIC VARIANT", "description": "The distribution of alleles and genotypes of rs1138272 polymorphism are summarized in Table 2.", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "id": "RS1138272"}, {"entity_type": "ALLELE", "description": "The frequency of genotypes CC, CT, and TT for controls is 92.00%, 08.00%, and 00.00% respectively.", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "id": "CC"}, {"entity_type": "ALLELE", "description": "While these ratios were calculated 88.00%, 11.00%, and 01.00%, respectively.", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "id": "CT"}, {"entity_type": "ALLELE", "description": "The frequency of genotypes CC, CT, and TT for cases is 88.00%, 11.00%, and 01.00% respectively.", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "id": "TT"}, {"entity_type": "PARAMETER", "description": "In our analysis, dWALTZ was calculated -228.00 which means that the polymorphism reduces the amyloid propensity of the GSTP1 enzyme.<SEP>The dWALTZ calculation for the Ile105Val polymorphism was -228.00.", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "id": "DWALTZ"}, {"entity_type": "POLYMORPHISM/SNP", "description": "Based on WALTZ output, the Ile105Val polymorphism decreases the amyloid propensity of the GSTP1 enzyme.", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "id": "ILE105VAL"}, {"entity_type": "PROTEIN", "description": "We evaluated effects of Ile105Val SNP on the function of GSTP1 protein using SNPeffects online software.", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "id": "GSTP1"}, {"source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "description": "The frequency ratios were found as 50.00%, 44.00% and 6.00% respectively.)", "entity_type": "UNKNOWN", "id": "GENOTYPE AA, AG, AND GG FOR CONTROLS"}, {"source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "description": "The frequency ratios were found as 92.00%, 08.00% and 00.00% respectively for control group whereas these ratios for cases were calculated 88.00%, 11.00% and 01.00%)", "entity_type": "UNKNOWN", "id": "GENOTYPE CC, CT AND TT FOR CONTROLS AND CASES"}, {"source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "description": "The Ile105Val polymorphism decreases the amyloid propensity of the GSTP1 enzyme (dWALTZ was -228.00)<SEP>The Ile105Val polymorphism reduces the amyloid propensity of the GSTP1 enzyme (dWALTZ was -228.00)", "entity_type": "UNKNOWN", "id": "WALTZ OUTPUT"}, {"source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "description": "We evaluated effects of Ile105Val SNP on the GSTP1 function using SNPeffects online software.)", "entity_type": "UNKNOWN", "id": "FUNCTION"}, {"source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "description": "The WALTZ algorithm specifically predicts that Ile105Val polymorphism decreases the amyloid propensity of GSTP1 enzyme.)", "entity_type": "UNKNOWN", "id": "FUNCTION AND ENZYME ACTIVITY"}, {"entity_type": "GENOMIC VARIANT", "description": "The most prevalent pathogenic variant in China, p.R778L, was found to associate with three haplogroups (A, B, C).<SEP>p.R778L is one of the most prevalent pathogenic variants found in WD patients, with an allelic frequency of 31.7%.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1<SEP>chunk-047f259db19cbb4de945706a1bd1b450", "id": "P.R778L"}, {"entity_type": "GENOMIC VARIANT", "description": "The haplotype spectrum of p.P992L was more complex, including six haplogroups (D, E, F, G, H, I).<SEP>p.P992L has a close allelic frequency to p.R778L and is the second most prevalent variant with an allelic frequency of 15.7%.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1<SEP>chunk-047f259db19cbb4de945706a1bd1b450", "id": "P.P992L"}, {"entity_type": "GENOMIC VARIANT", "description": "The allelic frequency of p.T935M in Fujian province was significantly higher than other regions, indicating a tendency for aggregation.<SEP>p.T935M presented a uniform state with only one kind of haplogroup on WD chromosomes, haplogroup J.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1<SEP>chunk-047f259db19cbb4de945706a1bd1b450", "id": "P.T935M"}, {"entity_type": "GEOGRAPHICAL FACTOR", "description": "Fujian province contributes the largest proportion (261, 36.8%) among five major regions with high p.T935M frequency.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "FUJIAN PROVINCE"}, {"entity_type": "GEOGRAPHICAL FACTOR", "description": "Zhejiang has a moderate contribution to the distribution of p.T935M variants.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "ZHEJIANG"}, {"entity_type": "GEOGRAPHICAL FACTOR", "description": "The number of cases in Jiangsu is significantly lower than other regions, with an allelic frequency of 4.5% for p.T935M.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "JIANGSU"}, {"entity_type": "GEOGRAPHICAL FACTOR", "description": "With a small population base and limited data available, Shanghai\'s contribution to p.T935M prevalence is minimal.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "SHANGHAI"}, {"entity_type": "GEOGRAPHICAL FACTOR", "description": "Jiangxi contributes the smallest proportion among five major regions with an allelic frequency of 2.3% for p.T935M.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "JIANGXI"}, {"entity_type": "GENOMIC VARIANT", "description": "The variant p.N1015K has an allelic frequency of 2.4%, which indicates its relative rarity among WD patients.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "P.N1015K"}, {"entity_type": "GEOGRAPHICAL FACTOR", "description": "These provinces have a higher prevalence of p.T935M than Jiangsu and Shanghai due to their unique demographic characteristics and potential historical genetic influences.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "FUJIAN PROVINCE, ZHEJIANG"}, {"entity_type": "HAPLOTYPE SPECIFICITY", "description": "The specific haplotype associated with allele 8 of p.T935M is predominant in the Fujian population, possibly linked to regional ancestry or selective pressures.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "P.T935M HAPLOTYPE 8 IN FUJIAN PROVINCE"}, {"entity_type": "POLYGENIC RISK SCORE", "description": "These variants contribute significantly to the polygenic risk score for WD in Chinese populations, reflecting their cumulative impact on disease susceptibility.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "P.R778L, P.P992L, AND P.S1014P"}, {"entity_type": "GWAS RESULTS", "description": "Significant genetic loci identified through genome-wide association studies (GWAS) include the regions harboring p.R778L, p.P992L, and p.T935M, providing insights into genetic risk factors for WD.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "DISEASE SUSCEPTIBILITY LOCI"}, {"entity_type": "REGIONAL FREQUENCY VARIANCE", "description": "Despite its minor contribution compared to other regions, the allelic frequency of p.T935M in Jiangxi is 2.3%, highlighting regional variations.", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "id": "P.T935M IN JIANGXI"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Haplogroup A was overwhelmingly common on WD chromosomes (97/100, 97.0%) vs. control ones (17/177, 9.6%, p < 0.01).", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "HAPLOGROUP A"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Haplogroup B was found to be relatively scarce (1.0%).", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "HAPLOGROUP B"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Haplogroup C appeared with a frequency of 2.0% when compared with haplogroup A.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "HAPLOGROUP C"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Haplogroup D appeared sporadically, accounting for 3.4% on WD chromosomes.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "HAPLOGROUP D"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "It was more common than control ones (72.4% vs. 3.4%, p < 0.01).", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "HAPLOGROUP E"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Haplogroup F appeared with a frequency of 6.9% on WD chromosomes.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "HAPLOGROUP F"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Haplogroup G appeared sporadically, accounting for 3.4% on WD chromosomes.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "HAPLOGROUP G"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Haplogroup H appeared with a frequency of 10.3% and 12.5% on WD chromosomes respectively.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "HAPLOGROUP H"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Haplogroup I appeared sporadically, accounting for 3.4% on WD chromosomes.", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "HAPLOGROUP I"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Haplogroup J was the only kind of haplogroup on WD chromosomes with a great significance vs. control ones (100 vs. 15.3%, p < 0.01).", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "id": "HAPLOGROUP J"}, {"source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "description": "Associations between specific alleles at different markers and various haplogroups underscore the interplay between genetic drift, natural selection, and demographic history in shaping genetic diversity within the WD population.)(entity", "entity_type": "UNKNOWN", "id": "GENETIC MARKER"}, {"source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "description": "Associations between specific alleles at different markers and various haplogroups underscore the interplay between genetic drift, natural selection, and demographic history in shaping genetic diversity within the WD population.)(entity", "entity_type": "UNKNOWN", "id": "HAPLOGROUP ASSOCIATION"}, {"entity_type": "GENE", "description": "CYP2C9 gene polymorphism that reduces enzyme activity.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "CYP2C9 STAR VARIANT"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP in CYP2C9 gene with low minor allele frequency, associated with reduced enzyme activity.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "RS1799853"}, {"entity_type": "GENOMIC VARIANT", "description": "Another SNP in the CYP2C9 gene with low minor allele frequency that impacts enzyme function.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "RS1057910"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP found to have a low minor allele frequency and its relevance is consistent with previous studies.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "RS9923231"}, {"entity_type": "GENOMIC VARIANT", "description": "Additional SNPs in the CYP2C9 gene that reduce enzyme activity and were combined into a group due to their low frequency.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "RS28371686, RS7900194, RS28371685"}, {"entity_type": "GENE", "description": "CYP2C9 is an enzyme that plays role in processing vitamin K, affecting the metabolism of drugs like warfarin.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "CYP2C9"}, {"entity_type": "DRUG", "description": "Vitamin K antagonist used for anticoagulation therapy.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "WARFARIN"}, {"entity_type": "DISEASE", "description": "Deep vein thrombosis (DVT) and pulmonary embolism (PE), conditions that prompted warfarin use in over half of patients.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "DVT/PE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The algorithms used to predict the required dose of warfarin based on genetic variants.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "ALGORITHM"}, {"entity_type": "DIETARY PATTERN", "description": "International Warfarin Pharmacogenetics Consortium; group that developed a set of algorithms for predicting warfarin doses based on genetics.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "IWPC"}, {"entity_type": "VALIDATION COHORT", "description": "Specific subset of patients with African American ancestry used to validate the accuracy of new algorithms.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "AA VALIDATION COHORT"}, {"entity_type": "DIETARY PATTERN", "description": "Group that developed a set of algorithms for predicting warfarin doses based on genetics.", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "id": "INTERNATIONAL WARFARIN PHARMACOGENETICS CONSORTIUM (IWPC)"}, {"entity_type": "ALGORITHM", "description": "The novel AA algorithm significantly outperforms both of the IWPC algorithms in patients with high doses.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "NOVEL AA ALGORITHM"}, {"entity_type": "ALGORITHM", "description": "This IWPC algorithm correctly assigns 26% of high-dose patients.<SEP>This algorithm shows a strong inverse correlation between percent WAA and predicted dose within the 20% therapeutic dose range for patients requiring doses \u226560 mg/week, but our novel AA algorithm does not exhibit such an association.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "IWPC PHARMACOGENOMICS ALGORITHM"}, {"entity_type": "ALGORITHM", "description": "This IWPC clinical algorithm correctly assigns only 2% of high-dose patients.<SEP>This algorithm has a lower correlation compared to the IWPC pharmacogenomics algorithm when dealing with percent WAA and predicted doses within the 20% therapeutic dose range for high-dose patients (\u226560 mg/week).", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "IWPC CLINICAL ALGORITHM"}, {"entity_type": "ALGORITHM OUTPUT", "description": "The novel AA algorithm was found to provide closer and more precise predicted doses compared to observed doses, as evidenced by lower MAE values.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "PREDICTED DOSE"}, {"entity_type": "MEASURE OF CLINICAL BENEFIT", "description": "The NNG was used to compare the clinical benefit of incorporating our novel AA algorithm into practice versus using the IWPC clinical algorithm.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "NNG (NET NUMERICAL GAIN)"}, {"entity_type": "MEASUREMENT TOOL", "description": "The availability and usage of AIMs have enabled accurate measurement of admixture in AAs, enhancing the performance evaluation of algorithms.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "AIMS (ADMIXTURE INFORMATION METHODS)"}, {"entity_type": "PERFORMANCE METRIC", "description": "Our novel AA algorithm shows no significant association with percent WAA, indicating its ability to incorporate population-specific variables effectively.", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "id": "PERCENT WAA (WITHIN ADMIXTURE ACCURACY)"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "CYP2C9 genetic variants were grouped due to low frequency, impacting the prediction models.)", "entity_type": "UNKNOWN", "id": "CYP2C9 STAR VARIANT GROUP"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Algorithms considered the impact of low-frequency SNPs on predicting warfarin doses.)", "entity_type": "UNKNOWN", "id": "RS1799853, RS1057910, RS9923231 SNPS"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "The CYP2C9 gene polymorphisms influence how algorithms predict drug metabolism.)", "entity_type": "UNKNOWN", "id": "CYP2C9 ENZYME"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Studies previously observed low minor allele frequencies for these SNPs.)", "entity_type": "UNKNOWN", "id": "PREVIOUS STUDIES"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Studies previously observed low minor allele frequencies for these SNPs.)", "entity_type": "UNKNOWN", "id": "RS1799853, RS1057910, AND RS9923231 SNPS AND LOW FREQUENCIES"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Comparisons of algorithm performance against IWPC methods to assess its validity in predicting warfarin doses.)", "entity_type": "UNKNOWN", "id": "ALGORITHM ACCURACY"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Comparisons of algorithm performance against IWPC methods to assess its validity in predicting warfarin doses.)", "entity_type": "UNKNOWN", "id": "IWPC ALGORITHMS"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Evaluation of how well each algorithm predicts dosing requirements within the target range.)", "entity_type": "UNKNOWN", "id": "ALGORITHM PREDICTIVE POWER"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Evaluation of how well each algorithm predicts dosing requirements within the target range.)", "entity_type": "UNKNOWN", "id": "NOVEL AA ALGORITHM, IWPC PHARMACOGENOMICS AND CLINICAL ALGORITHMS"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Analysis performed for different groups based on warfarin dosage needs.)", "entity_type": "UNKNOWN", "id": "LOW-, INTERMEDIATE- AND HIGH-DOSE PATIENTS"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Analysis performed for different groups based on warfarin dosage needs.)", "entity_type": "UNKNOWN", "id": "PREDICTIVE ACCURACY IN DOSE SUBGROUPS"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Specific comparison of accuracy in predicting lower doses among the algorithms.)", "entity_type": "UNKNOWN", "id": "ALGORITHM PERFORMANCE FOR LOW-DOSE PATIENTS"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Similar focus on assessing accuracy with regards to mid-range warfarin dosages.)", "entity_type": "UNKNOWN", "id": "ALGORITHM PERFORMANCE FOR INTERMEDIATE-DOSE GROUP"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "A detailed comparison of how well each algorithm performs in predicting higher doses.)", "entity_type": "UNKNOWN", "id": "ALGORITHM PERFORMANCE FOR HIGH-DOSE PATIENTS"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "A detailed comparison of how well each algorithm performs in predicting higher doses.)", "entity_type": "UNKNOWN", "id": "NOVEL AA ALGORITHM COMPARED AGAINST IWPC METHODS"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "The algorithms considered the impact of CYP2C9 genetic variants on predicting appropriate warfarin dosing.)", "entity_type": "UNKNOWN", "id": "ALGORITHM, CYP2C9 GENE"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "The algorithms considered the impact of CYP2C9 genetic variants on predicting appropriate warfarin dosing.)", "entity_type": "UNKNOWN", "id": "CYP2C9 ENZYME POLYMORPHISMS AND WARFARIN DOSE PREDICTION"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Some algorithms were more accurate when considering lower-frequency SNPs associated with reduced CYP2C9 activity.)", "entity_type": "UNKNOWN", "id": "ALGORITHM, SNPS"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "Some algorithms were more accurate when considering lower-frequency SNPs associated with reduced CYP2C9 activity.)", "entity_type": "UNKNOWN", "id": "SNP FREQUENCIES AND ALGORITHM PERFORMANCE"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "The performance of the AA novel algorithm was compared to that of IWPC algorithms in terms of dose prediction precision.)", "entity_type": "UNKNOWN", "id": "ALGORITHM 1 VS. ALGORITHM 2 (IWPC, AA NOVEL)"}, {"source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "description": "The performance of the AA novel algorithm was compared to that of IWPC algorithms in terms of dose prediction precision.)", "entity_type": "UNKNOWN", "id": "ALGORITHM ACCURACY COMPARISON"}, {"source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "description": "The novel AA algorithm provides higher precision in predicting doses.)", "entity_type": "UNKNOWN", "id": "PRECISION"}, {"source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "description": "The novel AA algorithm performs well when evaluated with accurate measurements of admixture provided by AIMs.)", "entity_type": "UNKNOWN", "id": "ACCURATE MEASUREMENT OF ADMIXTURE"}, {"source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "description": "The novel AA algorithm accounts for and incorporates population-specific variables, which results in better performance when dealing with the broad range of WAA among AAs.)", "entity_type": "UNKNOWN", "id": "POPULATION-SPECIFIC VARIABLES"}, {"source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "description": "The IWPC pharmacogenomics algorithm exhibits inversely correlated performance with percent WAA in predicting doses for high-dose patients (\u226560 mg/week), unlike our novel AA algorithm.)", "entity_type": "UNKNOWN", "id": "INVERSELY CORRELATED PERFORMANCE"}, {"source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "description": "Compared to the IWPC pharmacogenomics algorithm, our novel AA algorithm has a higher correlation with percent WAA and predicted doses for high-dose patients (\u226560 mg/week), indicating superior performance.)", "entity_type": "UNKNOWN", "id": "LOWER CORRELATION"}, {"entity_type": "GENE", "description": "CYP27B1 is a gene associated with vitamin D metabolism.", "source_id": "chunk-2fb95100a1290b27baae869c6560419a", "id": "CYP27B1"}, {"entity_type": "GENOMIC VARIANT", "description": "The text does not provide a comprehensive description for this entity.", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 38}]", "id": "MINOR ALLELE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The molecule involved in interactions with several bodily functions.", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "id": "NITRIC OXIDE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The group of processes that occur within living organisms, including growth and metabolism.", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "id": "PHYSIOLOGICAL PROCESSES"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Conditions that affect organisms externally, including diet and physical activity.", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "id": "ENVIRONMENTAL FACTORS"}, {"source_id": "chunk-cc528af98598e1adb4684c0166430cf4<SEP>chunk-f05ecc979eea31a1ab844c2c7de12432", "description": "Nitric oxide plays a critical role in regulating blood pressure and the health of cardiovascular systems.)<SEP>Optimal cardiovascular health is influenced by adequate folate intake and proper metabolic pathways involving one-carbon metabolism.", "entity_type": "OUTCOME OF BIOLOGICAL PROCESS", "id": "CARDIOVASCULAR HEALTH"}, {"source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "description": "Statins can help manage nitric oxide levels to reduce cardiovascular risks.)", "entity_type": "UNKNOWN", "id": "DRUG THERAPY"}, {"source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "description": "A balanced nutrient profile can influence lifestyle choices in an individual.)", "entity_type": "UNKNOWN", "id": "LIFESTYLE FACTOR"}, {"source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "description": "A balanced nutrient profile can influence lifestyle choices in an individual.)", "entity_type": "UNKNOWN", "id": "NUTRIENT PROFILE"}, {"entity_type": "GENOMIC VARIANT", "description": "This is a known CYP24A1 variant represented by n=2 samples.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "C.428_430DELAAG VARIANT"}, {"entity_type": "GENOMIC VARIANT", "description": "This is another variant of the CYP24A1 gene with n=2 samples in homozygous and heterozygous forms.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "C.1186C\u2009>\u2009T VARIANT"}, {"entity_type": "GENE", "description": "Samples with known CYP24A1 status were used as positive controls and wt represents their common reference form.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "WILD-TYPE (WT)"}, {"entity_type": "TRANSCRIPT", "description": "This refers to a single sample carrying both tested variants in the CYP24A1 gene.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "N\u2009=\u20091 COMPOUND HETEROZYGOUS SAMPLE"}, {"entity_type": "MOLECULAR INTERACTION/METABOLIC PATHWAY", "description": "These are two different target sequences that amplify common mutant sites, and their Tms were used to differentiate genotypes.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "RS777676129 AMPLICONS"}, {"entity_type": "MOLECULAR INTERACTION/METABOLIC PATHWAY", "description": "Also, these represent another set of target sequences for amplification which is indicative of different genotypes in the CYP24A1 gene.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "RS114368325 AMPLICONS"}, {"entity_type": "PHYSIOLOGICAL PROCESS/DISEASE", "description": "These refer to the analysis method used for identifying specific melting behavior related to different CYP24A1 genotypes which may be associated with disease conditions.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "NORMALIZED AND TEMPERATURE-SHIFTED AND DIFFERENT PLOT"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The dual nature of this assay is utilized for analyzing both amplicons and their Tm profiles together.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "DUPLEX ASSAY"}, {"entity_type": "METHOD/DIAGNOSTICS", "description": "This stands for dually hybridization melting analysis that determines the genotype based on melt curve patterns.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "DHRMA"}, {"entity_type": "METHOD/DIAGNOSTICS", "description": "A singleplex method used to check genotypes but in this context, it\'s less specific than the duplex assay.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "SINGLE PLEX HRMA"}, {"entity_type": "METHOD/DIAGNOSTICS", "description": "This technique confirms or validates the results obtained from dHRMA and HRMA.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "SANGER SEQUENCING"}, {"entity_type": "METHOD/DIAGNOSTICS", "description": "A combination method that combines PCR and HRMA for more efficient detection of genetic variants.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "DUPLEX-PCR/HRMA"}, {"entity_type": "GENOTYPE ASSIGNMENT/VALIDATION AND SENSITIVITY STUDY", "description": "The comparison between dHRMA outcomes with Sanger sequencing for genotype validation and sensitivity tests.", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "id": "DHRMA RESULTS/SANGER SEQUENCING RESULTS/"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "metabolic pathway", "entity_type": "UNKNOWN", "id": "MOLECULAR INTERACTION/TM DIFFERENCE/"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "metabolic pathway", "entity_type": "UNKNOWN", "id": "RS777676129 AMPLICONS/RS114368325 AMPLICONS"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "physiological process", "entity_type": "UNKNOWN", "id": "DISEASE ASSOCIATION/"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "physiological process", "entity_type": "UNKNOWN", "id": "NORMALIZED AND TEMPERATURE-SHIFTED AND DIFFERENT PLOT/CYP24A1 GENOTYPES"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "sensitivity study", "entity_type": "UNKNOWN", "id": "VALIDATION/"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "confirmation", "entity_type": "UNKNOWN", "id": "DIAGNOSTIC VALIDATION/"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "confirmation", "entity_type": "UNKNOWN", "id": "DUPLEX-PCR/HRMA/10 DNA SAMPLES WITH THE SUSPICION OF IIH"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "amplicon differentiation", "entity_type": "UNKNOWN", "id": "SPECIFIC GENOTYPING/"}, {"source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "description": "sequencing results", "entity_type": "UNKNOWN", "id": "DUPLEX-PCR/HRMA RESULTS/"}, {"entity_type": "DISEASE", "description": "At 20 months of age, the patient\u2019s developmental skill levels, as assessed by the Denver Developmental Screening Test II, included gross motor, 5 months; fine motor, 7 months; language, 7 months; and personal-social, 11 months.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "CLINICAL OUTCOME"}, {"entity_type": "DISEASE", "description": "A brain MRI at 20 months of age showed atrophy and delayed myelination.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "DIAGNOSTIC TESTS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Median nerve compound motor action potentials were measured, with normal values for a 1- to 2-year-old infant being \u22653.5 mV at the L wrist and \u22653.5 mV at the L elbow.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "MEDIAN NERVE COMPOUND MOTOR ACTION POTENTIAL"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Median nerve conduction velocity was measured, with normal values for a 1- to 2-year-old infant being \u226535 m/s.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "MEDIAN NERVE CONDUCTION VELOCITY"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Median nerve sensory action potential measurement yielded a value of 36 \u03bcV, where normal values for a 1- to 2-year-old infant are \u226515 \u03bcV.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "MEDIAN NERVE SENSORY ACTION POTENTIAL"}, {"entity_type": "DISEASE", "description": "Table I summarizes biochemical findings in the proband. His serum copper level was normal; urine b2-microglobulin and plasma catecholamine ratios were also evaluated, but did not match Menkes disease characteristics.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 40}]", "id": "BIOCHEMICAL FINDINGS"}, {"entity_type": "GENOMIC VARIANT", "description": "We confirmed mosaicism for the P1001L mutation in the proband\u2019s genomic DNA. The mutant allele was detected; however, it wasn\'t found in the mother. This mutation created a novel BsmAI restriction site.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 38}]", "id": "MOLECULAR ANALYSES"}, {"entity_type": "GENE", "description": "The P1001L mutation indicates mosaicism which is associated with Menkes disease and is present on the patient\'s DNA; it was not detected in the mother\'s genomic DNA.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 43}]", "id": "P1001L MUTATION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The P1001L mutation creates a new BsmAI restriction site, which affects the processing of DNA.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "BSMAI RESTRICTION SITE"}, {"entity_type": "BIOCHEMICAL PROCESS", "description": "PCR and BsmAI digestion were performed on the patient\'s blood and skin fibroblast culture samples to assess mosaicism levels. The wild type allele was less abundant in the fibroblasts compared to the blood.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "PCR AND BSMAI DIGESTION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Densitometry results showed a wild type:mutant allele ratio of 77:23 for whole blood DNA and 12:88 in fibroblast DNA.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "WILD TYPE: MUTANT ALLELE RATIO"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Q-RT-PCR experiments showed similar wild type:mutant ratios, with a ratio of 69:31 for blood versus a 12:88 ratio in primary fibroblasts.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "QUANTITATIVE REAL-TIME PCR (Q-RT-PCR)"}, {"entity_type": "QUALITY ASSURANCE", "description": "The control for completeness of BsmA1 digestion (ATP7A exon 7) was validated using Q-RT-PCR, indicating a 99% digestion efficiency in these experiments.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "CONTROL FOR COMPLETENESS OF BSMAI DIGESTION"}, {"entity_type": "NUTRIENT", "description": "Serum copper level was normal.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "COPPER LEVEL"}, {"entity_type": "NUTRIENT", "description": "Urine b2-microglobulin, a marker of renal tubular injury due to copper overload, was also normal.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "B2-MICROGLOBULIN"}, {"entity_type": "METABOLIC PATHWAY", "description": "Plasma catecholamine ratios reflect the activity of dopamine-b-hydroxylase, a copper-dependent enzyme.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "CATECHOLAMINE RATIOS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Dopamine-b-hydroxylase is a copper-dependent enzyme that contributes to neurotransmitter synthesis.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "DOPAMINE-B-HYDROXYLASE"}, {"entity_type": "DISEASE", "description": "Classical Menkes disease is characterized by specific biochemical findings and molecular genetic changes, including elevated ceruloplasmin levels and mutations in the ATP7A gene.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "MENKES DISEASE"}, {"entity_type": "GENE", "description": "The ATP7A gene plays a role in copper transport and metabolism; mutations or deletions can lead to Menkes disease.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "ATP7A GENE"}, {"entity_type": "GENOMIC VARIANT", "description": "The presence of mosaicism means that the individual carries two different cellular populations, each with a different genetic constitution.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "MOSAICISM"}, {"entity_type": "EXON", "description": "This is a specific part of the ATP7A gene sequence which was used as an indicator for BsmAI restriction site analysis during molecular tests.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "ATP7A EXON 7"}, {"entity_type": "CELL TYPE", "description": "Primary fibroblast cultures are obtained from skin biopsies and used in various biological assays to study cellular functions and responses.", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "id": "FIBROBLAST CULTURE"}, {"source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "description": "Differences were observed in PCR and BsmAI digestion results between blood and skin fibroblast samples, which may indicate variations in mosaicism levels for the P1001L mutation.)", "entity_type": "UNKNOWN", "id": "BIOCHEMICAL PROCESS"}, {"source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "description": "A control test was used to ensure the accuracy and completeness of BsmAI digestion through Q-RT-PCR, providing quality assurance in molecular analysis results.)", "entity_type": "UNKNOWN", "id": "QUALITY ASSURANCE"}, {"entity_type": "CELL LINE", "description": "Caco-2 cells are human epithelial-like intestinal cell lines used for studying drug permeability and metabolism.<SEP>Caco-2 cells are used in drug metabolism studies due to their intestinal-like characteristics.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "CACO-2 CELLS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Deiodinase is an enzyme that converts inactive thyroid hormones to active forms, playing a critical role in thyroid hormone regulation.<SEP>This gene is involved in the conversion of inactive thyroid hormones into active forms.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "DEIODINASE"}, {"entity_type": "GENE", "description": "The human form of the deiodinase gene (hID1) does not express at detectable levels under certain conditions.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "HUMAN IDI"}, {"entity_type": "GENE", "description": "Coding sequences for rat deiodinase are transfected into Caco-2 cells for study.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "RAT IDI CODING SEQUENCES"}, {"entity_type": "TRANSCRIPT", "description": "The 3\' untranslated region (UTR) of the GPX4 gene is used to control the expression of rat IDI sequences in transfected Caco-2 cells.<SEP>The specific 3\' UTR found in the GPX4 gene which affects mRNA stability and translation efficiency.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "GPX4 3\u2032UTR"}, {"entity_type": "NUTRIENT", "description": "An essential nutrient for human health, involved in numerous enzymatic reactions including those involving deiodinase.<SEP>Selenium is essential for proper thyroid function and plays a role in deiodinase expression.<SEP>Selenium, an essential nutrient for human health, influences deiodinase expression and function.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "SE (SELENIUM)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Exposure to ROS leads to increases in total reactive oxygen species production.<SEP>Oxidative stress markers indicating cellular damage from uncontrolled production of reactive oxygen species.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "ROS (REACTIVE OXYGEN SPECIES)"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Condition where the environment lacks sufficient selenium, affecting cellular functions including thyroid hormone metabolism.<SEP>Depletion of selenium from the culture medium affects cellular responses and deiodinase expression.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "SE DEPLETION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Selenium is used for synthesis of selenoproteins like glutathione peroxidase (GPx), thioredoxin reductase 1 (TR1), and others.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "SELENOPROTEINS"}, {"entity_type": "DIETARY PATTERN", "description": "Supplementation with selenium in the culture medium can affect deiodinase expression differently from regular cell growth conditions.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "SE-SUPPLEMENTATION MEDIUM"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Depletion of selenium affects cellular responses, including oxidative stress and GPx activity, under different experimental conditions.<SEP>Growth conditions under low Se supply have different effects on cell viability compared to supplemented medium.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7<SEP>chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "SE-DEPLETED MEDIUM"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Se-supply levels influence cell viability under oxidative stress, with IDI-C cells being more sensitive.<SEP>The ability of cells to survive after an oxidative challenge is influenced by deiodinase expression levels and selenoprotein content.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7<SEP>chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "CELL VIABILITY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "An oxidant used as a challenge agent in cellular experiments, affecting cell viability and ROS production.<SEP>Tert-butylhydroperoxide was used as an oxidant to challenge the cells and induce a response.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7<SEP>chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "TERT-BUTYLHYDROPEROXIDE"}, {"entity_type": "THEORETICAL CONCEPT", "description": "A hypothesis suggesting that differing 3\'UTRs (T or C variant) may compete for Se during oxidative stress conditions.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "SE COMPETITION THEORY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Machinery involved in the synthesis of selenoproteins, including deiodinase proteins, using selenium.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "SELENOCYSTEINE INCORPORATION MACHINERY"}, {"entity_type": "CELL LINE", "description": "Cells that have been genetically modified by introducing exogenous DNA, specifically rat IDI coding sequences in this context.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "TRANSFECTED CACO-2 CELLS"}, {"entity_type": "TRANSCRIPT FEATURE", "description": "The region at the 3\' end of a gene\'s mRNA transcript that can influence gene expression and stability.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "3\u2032UTR (THREE PRIME UNTRANSLATED REGION)"}, {"entity_type": "DIETARY INTERVENTION", "description": "Administering selenium to improve or restore levels in cases of deficiency.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "SE SUPPLEMENTATION"}, {"entity_type": "PATHOLOGICAL CONDITION", "description": "Increased levels of reactive oxygen species leading to cellular damage and dysfunction.", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "id": "OXIDATIVE STRESS"}, {"entity_type": "TRANSCRIPT", "description": "Increased mean GPx4 mRNA levels are not significantly different in IDI-T compared to IDI-C cells when grown with Se.<SEP>The increase in GPx4 mRNA levels correlates with sufficient Se availability.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "GPX4"}, {"entity_type": "TRANSCRIPT", "description": "Mean TR1 mRNA levels are similar in IDI-T and IDI-C cells under different Se conditions.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "TR1"}, {"entity_type": "TRANSCRIPT", "description": "Mean GPx1 mRNA levels show no difference between IDI-T and IDI-C cells under Se-depleted or supplemented medium.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "GPX1"}, {"entity_type": "PROTEIN", "description": "The mRNA for selenoprotein H was found to be significantly lower in IDI-C cells, regardless of the Se supply conditions.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "SELENOPROTEIN H"}, {"entity_type": "NUTRIENT", "description": "Selenium (Se) is present in the medium and affects cellular viability when supplied at different concentrations.", "source_id": "chunk-a908a3115a9120ca2623513fe4a5e13a", "id": "SE"}, {"source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "description": "Cells under Se-depleted conditions show a greater reduction in viability when challenged with tert-butylhydroperoxide compared to normal medium conditions.)", "entity_type": "UNKNOWN", "id": "EFFECT ON CELL VIABILITY"}, {"source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "description": "Cell lines over-expressing either the IDI-T or IDI-C transcripts show lower viability after tert-butylhydroperoxide challenge compared to untransfected cells.)", "entity_type": "UNKNOWN", "id": "CELLULAR RESPONSE"}, {"source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "description": "Cell lines over-expressing either the IDI-T or IDI-C transcripts show lower viability after tert-butylhydroperoxide challenge compared to untransfected cells.)", "entity_type": "UNKNOWN", "id": "TRANSFECTED CACO-2 CELLS EXPRESSING IDI-T AND IDI-C TRANSCRIPTS"}, {"source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "description": "Exposure to tert-butylhydroperoxide leads to an increase in ROS levels, particularly under Se-depleted conditions and differing 3\'UTR variants.)", "entity_type": "UNKNOWN", "id": "TERT-BUTYLHYDROPEROXIDE EXPOSURE"}, {"source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "description": "The expression level of the deiodinase gene (both human IDI and rat IDI) affects selenoprotein H mRNA levels under varying selenium supply conditions.)", "entity_type": "UNKNOWN", "id": "DEIODINASE EXPRESSION LEVELS"}, {"source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "description": "The expression level of the deiodinase gene (both human IDI and rat IDI) affects selenoprotein H mRNA levels under varying selenium supply conditions.)", "entity_type": "UNKNOWN", "id": "INFLUENCE ON SELENOPROTEIN H MRNA"}, {"source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "description": "Hypothesis suggesting that differing expression levels of rat IDI under se-depleted conditions could influence selenoprotein production differently compared to normal conditions.)", "entity_type": "UNKNOWN", "id": "THEORETICAL MODEL"}, {"entity_type": "GENE", "description": "Quantitative RT-PCR revealed the patient, her father and one brother are hemizygous for CHAT.<SEP>We identified 12 missense and one nonsense mutations in CHAT in 11 kinships.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "CHAT"}, {"entity_type": "MUTATION", "description": "Nine mutations are novel; p.Thr553Asn and p.Ser704Pro were published but not functionally characterized by. p.Ala557Thr was previously reported.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "MISSENSE MUTATION"}, {"entity_type": "MUTATION", "description": "None of the mutations was detected in 200 to 400 alleles of unrelated control subjects.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "NONSENSE MUTATION"}, {"entity_type": "CONSERVED RESIDUE", "description": "All mutated residues are conserved in dog, mouse, rat, and chicken.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "RESIDUES"}, {"entity_type": "CONSERVED INVERTEBRATE", "description": "Four residues (Met202, Trp421, Thr553, and Ala557) are also conserved in Drosophila and C. elegans.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "DROSOPHILA"}, {"entity_type": "MOLECULAR TECHNIQUE", "description": "Genetically engineered mutant and wild-type CHAT cDNAs were analyzed using immunoblots of cell lysates.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "IMMUNOBLOTS"}, {"entity_type": "TRANSCRIPT VARIANT", "description": "The spinal cord (S-type) ChAT transcript has two alternative start sites that yield 70 kDa and 74 kDa proteins at a constant ratio.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "START SITES"}, {"entity_type": "FUNCTIONAL STUDY", "description": "Eight mutants expressed at significantly lower levels than wild-type, and five of these (p.Trp421Ser, p.Ser498Pro, p.Thr553Asn, p.Ala557Thr, p.Ser572Trp) expressed at <50% of wild-type.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "MUTATION ANALYSIS"}, {"entity_type": "KINETIC PARAMETER", "description": "Ten mutations alter one or more rate constants of ChAT activation and two mutations compromise the thermal stability of the mutant protein.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "RATE CONSTANTS"}, {"entity_type": "MOLECULAR PROPERTY", "description": "The Discussion analyzes the effects of each mutation and interprets them in the context of the atomic structural model of ChAT.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "THERMAL STABILITY"}, {"entity_type": "MUTATION PREVALENCE", "description": "Mutations were not found in control subjects, implying low prevalence among the general population.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "CONTROL SUBJECTS"}, {"entity_type": "CONSERVED RESIDUES ACROSS SPECIES", "description": "The conserved residues across species like Drosophila and C. elegans suggest evolutionary conservation.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "EVOLUTIONARY CONSERVATION"}, {"entity_type": "THERMAL STABILITY OF MUTANT PROTEINS", "description": "Two mutations compromise the thermal stability of CHAT proteins, indicating potential instability under varying temperatures.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "PROTEIN STABILITY"}, {"entity_type": "WILD-TYPE EXPRESSION COMPARISON", "description": "The expression levels of wild-type CHAT were compared with mutated forms to evaluate genetic engineering impact.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "GENETIC ENGINEERING"}, {"entity_type": "KINETIC PARAMETERS AND ACTIVATION", "description": "Ten mutations changed one or more kinetic parameters for ChAT activation, demonstrating altered function.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "FUNCTIONAL CHARACTERIZATION"}, {"entity_type": "ATOMIC STRUCTURAL MODEL OF CHAT", "description": "The atomic structural model provided insights into mutation effects and biological mechanisms.", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "id": "STRUCTURAL MODEL"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "The expression of eight mutants was significantly lower than wild-type; five expressed at less than 50% of wild-type levels.)", "entity_type": "UNKNOWN", "id": "FUNCTIONAL IMPACT"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "The expression of eight mutants was significantly lower than wild-type; five expressed at less than 50% of wild-type levels.)", "entity_type": "UNKNOWN", "id": "MISSENSE MUTATIONS"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "None of the mutations were detected in control subjects, suggesting a low prevalence.)", "entity_type": "UNKNOWN", "id": "DISEASE PREVALENCE"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Conserved residues across species like Drosophila and C. elegans indicate that certain functions are evolutionarily preserved.)<SEP>The conserved residues are present across species, including Drosophila and C. elegans, indicating evolutionary conservation.)", "entity_type": "UNKNOWN", "id": "CONSERVED RESIDUES"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Eight mutants expressed at significantly lower levels than wild-type; five of these (p.Trp421Ser, p.Ser498Pro, p.Thr553Asn, p.Ala557Thr, p.Ser572Trp) expressed at <50% of wild-type.)", "entity_type": "UNKNOWN", "id": "CHAT EXPRESSION"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Eight mutants expressed at significantly lower levels than wild-type; five of these (p.Trp421Ser, p.Ser498Pro, p.Thr553Asn, p.Ala557Thr, p.Ser572Trp) expressed at <50% of wild-type.)", "entity_type": "UNKNOWN", "id": "PROTEIN LEVEL IMPACT"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Ten mutations alter rate constants for ChAT activation; two compromise thermal stability of mutant proteins.)", "entity_type": "UNKNOWN", "id": "KINETIC PARAMETER EFFECT"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Ten mutations alter rate constants for ChAT activation; two compromise thermal stability of mutant proteins.)", "entity_type": "UNKNOWN", "id": "MUTATION RATE CONSTANTS"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "The absence of mutations in control subjects suggests a limited genetic predisposition towards these mutations.)", "entity_type": "UNKNOWN", "id": "CONTROL SUBJECT MUTATIONS"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "The absence of mutations in control subjects suggests a limited genetic predisposition towards these mutations.)", "entity_type": "UNKNOWN", "id": "MUTATION PREVALENCE"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Mutation-induced changes might lead to compromised thermal stability under different temperatures.)", "entity_type": "UNKNOWN", "id": "THERMAL STABILITY OF MUTANTS"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Comparisons were made between wild-type and mutated CHAT proteins to evaluate the impact on expression levels.)", "entity_type": "UNKNOWN", "id": "WILD-TYPE COMPARISON"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Mutations altered kinetic parameters involved in ChAT activation, affecting its catalytic efficiency.)", "entity_type": "UNKNOWN", "id": "FUNCTIONAL PARAMETERS"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "Mutations altered kinetic parameters involved in ChAT activation, affecting its catalytic efficiency.)", "entity_type": "UNKNOWN", "id": "KINETIC ACTIVATION CHANGES"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "The atomic structural model offered insights into how mutations affect the protein\'s 3D structure and function.)", "entity_type": "UNKNOWN", "id": "ATOMIC STRUCTURAL MODEL"}, {"source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "description": "The atomic structural model offered insights into how mutations affect the protein\'s 3D structure and function.)", "entity_type": "UNKNOWN", "id": "STRUCTURAL INSIGHTS"}, {"entity_type": "GENE", "description": "A gene annotated to CpG sites which was found to have nominal associations with methylation.<SEP>BCL2-like 11 [apoptosis facilitator]", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7<SEP>chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "BCL2L11"}, {"entity_type": "GENE", "description": "NADH dehydrogenase [ubiquinone] Fe-S protein 7, 20 kDa", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "NDUFS7"}, {"entity_type": "GENE", "description": "eukaryotic translation initiation factor 3, subunit J", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "EIF3J"}, {"entity_type": "GENE", "description": "zinc finger protein 761", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "ZNF761"}, {"entity_type": "GENE", "description": "proteasome assembly chaperone 3", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "PSMG3"}, {"entity_type": "GENE", "description": "sema domain, transmembrane domain [TM], and cytoplasmic domain ([semaphorin] 6A", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "SEMFA6A"}, {"entity_type": "GENE", "description": "TAP binding protein [tapasin]", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "TAPBP"}, {"entity_type": "GENE", "description": "ral guanine nucleotide dissociation stimulator-like 2 (FDR <0.05)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "RGL2"}, {"entity_type": "GENE", "description": "cytoskeleton associated protein 5", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "CKAP5"}, {"entity_type": "GENE", "description": "FOS-like antigen 2", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "FOSL2"}, {"entity_type": "GENE", "description": "family with sequence similarity 20, member C", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "FAM20C"}, {"entity_type": "GENE", "description": "FERM Domain containing 6", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "FRMD6"}, {"entity_type": "GENE", "description": "Glutamyltransferase", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "id": "GGT"}, {"entity_type": "FUNCTIONAL REGULATORY ELEMENT", "description": "The CpGs associated with oxidative stress are enriched in CpG island shores compared to all CpGs on the array.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "CPG ISLAND SHORE"}, {"entity_type": "GENE", "description": "A larger proportion of phenotype-associated CpGs were located in promoters of genes, thus having the potential to affect transcription from adjacent genes.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "PROMOTER OF GENE"}, {"entity_type": "FUNCTIONAL REGULATORY REGION", "description": "Twenty-three percentages of sites showed strong evidence being located in a functional regulatory region (RegulomeDB score 1a-2c) based on the analysis using RegulomeDB.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "REGULOMEDB SCORE 1A-2C"}, {"entity_type": "GENE/PROTEIN", "description": "We found significant enrichment of four TF binding sites, including E2F1 which is associated with atherosclerosis and coronary heart disease (CHD).", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "E2F1"}, {"entity_type": "FUNCTIONAL TERM", "description": "Genes annotated to phenotype-associated CpGs are clustered into functional terms relating to phosphorylation and phosphate/phosphorous metabolic processes based on the analysis using DAVID.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "PHOSPHORYLATION"}, {"entity_type": "FUNCTIONAL TERM", "description": "The clustering of genes includes annotations related to pleckstrin domains, which may indicate pathways involved in oxidative stress.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "PLECKSTRIN DOMAIN"}, {"entity_type": "GENETIC SEQUENCE VARIANT", "description": "There are significant cis-meQTL SNPs among CpGs associated with oxidative markers, suggesting genetic influence on methylation patterns.<SEP>We found 28 CpGs associated with oxidative markers that had a significant cis-meQTL (FDR <0.05), suggesting genetic sequence variants contribute to DNA methylation variation.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "CIS-MEQTL SNPS"}, {"entity_type": "SUGGESTIVE EVIDENCE OF ASSOCIATION", "description": "We found two instances where associations of genotype-phenotype, genotype-CpG and CpG-phenotype overlapped with nominal p-value <0.001.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "ASSOCIATION BETWEEN GENOTYPE-PHENOTYPE, GENOTYPE-CPG, AND CPG-PHENOTYPE"}, {"entity_type": "GENETIC DISEASES", "description": "GWAS data from the CARDIoGRAM consortium for CHD and the DIAGRAM consortium for T2D did not show overrepresentation of nominally significant associations among significant meQTL SNPs, with p-value <0.05.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "CHD AND T2D IN GWAS DATA"}, {"entity_type": "FUNCTIONAL ELEMENT", "description": "CpG islands associated with oxidative stress markers are enriched compared to all CpGs.<SEP>Promoter regions where CpG dinucleotides are clustered and often associated with high levels of transcriptional activity.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28<SEP>chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "CPG ISLAND"}, {"entity_type": "GENE", "description": "Phenotype-associated CpGs have a higher proportion located in gene promoters.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "PROMOTER"}, {"entity_type": "FUNCTIONAL REGULATORY REGION", "description": "A significant percentage of CpGs is identified as being located in functional regulatory regions based on RegulomeDB score.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "REGULOMEDB SCORE 1-2"}, {"entity_type": "TRANSCRIPTION FACTOR BINDING SITE", "description": "Four transcription factor binding sites, including that of E2F1, are significantly enriched among the CpGs associated with oxidative markers.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "E2F1 BINDING SITE"}, {"entity_type": "CELLULAR PROCESS", "description": "Genes annotated to oxidative stress-associated CpGs cluster into functional terms related to phosphorylation processes.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "PHOSPHORYLATION PROCESS"}, {"entity_type": "FUNCTIONAL TERM ANNOTATIONS", "description": "The clustering of genes includes annotations associated with pleckstrin domains relevant to oxidative stress responses.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "PLECKSTRIN DOMAIN ANNOTATIONS"}, {"entity_type": "STATISTICAL TEST", "description": "Analysis of GWAS data for diseases like CHD and T2D in relation to CpG sites does not show significant overrepresentation among the cis-meQTL SNPs associated with oxidative markers.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "GENOME-WIDE ASSOCIATION STUDY (GWAS)"}, {"entity_type": "BIOLOGICAL COMPONENT ANNOTATION", "description": "The genes annotated to phenotype-associated CpGs cover a range of biological components including, but not limited to, regulatory elements and cellular processes related to oxidative stress.", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "id": "GENE ANNOTATION"}, {"entity_type": "CPG SITE", "description": "CpG site cg07547695 associated with rs726430 and BCL2L11 gene.<SEP>CpG site cg07547695 associated with rs726430, related to the BCL2L11 gene.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CG07547695"}, {"entity_type": "GENE", "description": "A gene annotated to CpG sites which was found to have nominal associations with methylation.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CEP55"}, {"entity_type": "CPG SITE", "description": "CpG site cg24371383 associated with rs12782691, related to the CEP55 gene.<SEP>CpG site cg24371383 related to rs12782691 and CEP55 gene.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CG24371383"}, {"entity_type": "GENE", "description": "A gene annotated to CpG sites which was found to have nominal associations with methylation.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CPB1"}, {"entity_type": "CPG SITE", "description": "CpG site cg04072156 associated with rs16861015, related to the CPB1 gene.<SEP>CpG site cg04072156 connected with rs16861015, CPB1 gene, and BCL2L11.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CG04072156"}, {"entity_type": "CPG SITE", "description": "An intergenic CpG site was found in the analysis.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "INTERGENIC SITE ON CHR. 4"}, {"entity_type": "CPG SITE", "description": "CpG site cg14532755 associated with multiple SNPs on chr. 4.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "CG14532755"}, {"entity_type": "SNP", "description": "SNP rs726430 associated with BCL2L11 and CpG site cg07547695.<SEP>SNP rs726430 linked to BCL2L11 gene and CpG site cg07547695.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "RS726430"}, {"entity_type": "SNP", "description": "SNP rs12782691 related to CEP55 gene via CpG site cg24371383.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "RS12782691"}, {"entity_type": "SNP", "description": "SNP rs16861015 linked with CPB1 and CpG site cg04072156.<SEP>SNP rs16861015 linked with CPB1 gene and CpG site cg04072156.", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "id": "RS16861015"}, {"source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "description": "A CpG site in BCL2L11 was associated with TGSH levels.)", "entity_type": "UNKNOWN", "id": "TGSH"}, {"source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "description": "One CpG site in NDUFS7 was significantly linked to GSH levels.)", "entity_type": "UNKNOWN", "id": "GSH"}, {"source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "description": "The methylation at a CpG annotated to EIF3J was found associated with the GSSG/GSH ratio.)", "entity_type": "UNKNOWN", "id": "GSSG/GSH RATIO"}, {"source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "description": "CpG methylation in SEMFA6A was related to oxLDL levels.)", "entity_type": "UNKNOWN", "id": "OXLDL"}, {"source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-6a657cd0a000b6df4db8f1305b795021<SEP>chunk-491fec1ffc43ca0e67c1080702f038a7<SEP>chunk-6cb127717713bd0ab5be9436fc0c7568", "description": "Compared to healthy controls, CD subjects had a higher presence of MAPbacteremia.<SEP>Higher methylation at TAPBP CpGs was found associated with CD marker levels.)<SEP>Inflammatory bowel disease, specifically Crohn\'s Disease (CD), is the context under which studies have observed altered PTPN2:rs478582 gene expression.<SEP>The relative gene expression of transcripts related to PTPN2:rs478582 is investigated in individuals with different allele combinations for this SNP.<SEP>The study involved patients with Celiac Disease (CD) and examined the relation between their condition and gene expressions.", "entity_type": "DISEASE", "id": "CD"}, {"source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "description": "Methylation in the promoter region of RGL2 was found related to BCD-LDL levels.)", "entity_type": "UNKNOWN", "id": "BCD-LDL"}, {"source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "description": "CpGs associated with oxidative stress were enriched in CpG island shores and functional regulatory regions based on the analysis using RegulomeDB.)", "entity_type": "UNKNOWN", "id": "CPGS ASSOCIATED WITH OXIDATIVE STRESS"}, {"source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "description": "CpGs associated with oxidative stress were enriched in CpG island shores and functional regulatory regions based on the analysis using RegulomeDB.)", "entity_type": "UNKNOWN", "id": "FUNCTIONAL REGULATORY ELEMENTS"}, {"source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "description": "There was modest suggestive evidence of overlap between genotype-phenotype, genotype-CpG, and CpG-phenotype associations for certain CpGs.)", "entity_type": "UNKNOWN", "id": "GENOTYPE-PHENOTYPE, GENOTYPE-CPG, AND CPG-PHENOTYPE ASSOCIATIONS"}, {"source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "description": "There was modest suggestive evidence of overlap between genotype-phenotype, genotype-CpG, and CpG-phenotype associations for certain CpGs.)", "entity_type": "UNKNOWN", "id": "OVERLAP BETWEEN ASSOCIATIONS"}, {"source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e<SEP>chunk-ec104044b3aa65560be8d81dbb3940d6", "description": "CpG sites are related when they share SNPs, such as cg07547695 associated with rs726430 and cg04072156 linked to rs16861015.)<SEP>DNA modification differences at specific CpG sites within the LCT gene suggest epigenetic regulation influencing lactose metabolism and persistence.<SEP>Significant DNA modification differences at CpG sites suggest regulatory roles affecting gene expression and the establishment of lactase persistence or non-persistence.<SEP>Specific CpG sites within MCM6 are found to mediate individual differences in LCT mRNA levels, implicating MCM6 as a regulator of lactase expression.<SEP>Specific CpG sites within the genes exhibit correlations between DNA modification density and LCT mRNA levels, suggesting epigenetic regulation of gene expression.<SEP>The location on a chromosome where certain nucleotides pair form CpG dinucleotides that play crucial roles in gene regulation and function.", "entity_type": "GENOMIC VARIANT", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 44}]", "id": "CPG SITE"}, {"source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "description": "Increased risk of diseases is associated with specific methylation levels at CpG sites; e.g., higher T2D risk related to hypomethylation in DVL1 and HIST1H4D.)", "entity_type": "UNKNOWN", "id": "DISEASE INCIDENCE"}, {"source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "description": "Increased risk of diseases is associated with specific methylation levels at CpG sites; e.g., higher T2D risk related to hypomethylation in DVL1 and HIST1H4D.)", "entity_type": "UNKNOWN", "id": "METHYLATION LEVEL"}, {"source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "description": "No significant association was found between methylation levels and incident CVD events.)", "entity_type": "UNKNOWN", "id": "CVD EVENT"}, {"source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "description": "The body of ZNF341 gene is associated with incident T2D, shown by its CpG site cg11842944 methylation level.)", "entity_type": "UNKNOWN", "id": "CG11842944"}, {"source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "description": "The body of ZNF341 gene is associated with incident T2D, shown by its CpG site cg11842944 methylation level.)", "entity_type": "UNKNOWN", "id": "ZNF341"}, {"source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "description": "Methylation at a specific CpG site (cg11842944 in ZNF341 gene) showed the strongest association to heart failure among CVD events.)", "entity_type": "UNKNOWN", "id": "HEART FAILURE"}, {"entity_type": "GENE", "description": "The ATP synthase 6 gene is part of the mitochondrial genome responsible for complex V (ATP synthase) activity in the electron transport chain.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "ATP SYNTHASE 6"}, {"entity_type": "GENOMIC VARIANT", "description": "An apparently homoplasmic de novo mutation in mtDNA 8993T>G causing an alteration at p.Leu156Arg, responsible for Leigh syndrome due to deficient activity of complex V (ATP synthase).", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MTDNA 8993T>G"}, {"entity_type": "DISEASE", "description": "A neurological disorder caused by mitochondrial enzyme deficiencies that lead to progressive brain degeneration.<SEP>An inherited disorder resulting from the deficiency of complex V (ATP synthase) activity, associated with mitochondrial DNA variant 8993T>G.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "LEIGH SYNDROME"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A validated RFLP assay was used for confirmation on both blood and urine samples from the child.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "RESTRICTION FRAGMENT LENGTH POLYMORPHISM (RFLP)"}, {"entity_type": "THERAPY", "description": "Capillary sequencing methods were utilized for testing DNA isolated from maternal urine and blood to detect mutations, though it may fail in detecting heteroplasmy of less than 10%.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "CAPILLARY SEQUENCING"}, {"entity_type": "THERAPY", "description": "Exome enrichment and sequencing was performed on Illumina HiSeq 2000 platform for accurate detection of variants.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "EXOME SEQUENCING"}, {"entity_type": "TRANSCRIPT", "description": "The MT-ATP6 transcript is part of the mitochondrial genome encoding ATP synthase activity necessary for proper energy metabolism.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MT-ATP6"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Exclusion of maternal DNA coverage may not be sufficient to detect low levels of heteroplasmy due to insufficient sequencing depth.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "OMITTING MATERNAL DNA"}, {"entity_type": "GENE", "description": "The NDUFV1 gene encodes a protein involved in releasing electrons from NADH via complex 1 of the electron transport chain and functions related to proper mitochondrial energy metabolism.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "NDUFV1 GENE"}, {"entity_type": "GENOMIC VARIANT", "description": "A missense variant in NDUFV1 gene causing a change at codon Glu246 to Lys, potentially affecting protein function and mitochondrial energy metabolism.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "C.736G>A, P.GLU246LYS"}, {"entity_type": "GENOMIC VARIANT", "description": "Another missense variant found in NDUFV1 gene causing a change at codon Ala117 to Thr, predicted to be damaging based on in silico analysis.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "C.349G>A, P.ALA117THR"}, {"entity_type": "PROTEIN", "description": "The NDUFV1 protein functions as part of the electron transport chain complex 1 and contains sites for NADH binding and iron-sulfur cluster binding necessary for proper protein activity.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "NDUFV1 PROTEIN"}, {"entity_type": "RELATIONSHIP STRENGTH", "description": "Applying a recessive inheritance model to filter variants helped identify two missense variants in NDUFV1 gene shared by affected siblings, suggesting potential disease causation.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "RECESSIVE INHERITANCE MODEL"}, {"entity_type": "RELATIONSHIP TYPE", "description": "Identifying both infants as compound heterozygotes for the missense variants in NDUFV1 gene suggests they each carry one variant on opposite alleles, further supporting the pathogenicity of these mutations.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "COMPOUND HETEROZYGOTES"}, {"entity_type": "MOLECULAR COMPONENT", "description": "A circular DNA molecule containing both protein-coding and non-coding regions, which is specific to mitochondria.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MITOCHONDRIAL GENOME"}, {"entity_type": "PATHWAY", "description": "A series of protein complexes in the inner mitochondrial membrane that facilitate electron transfer during oxidative phosphorylation.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "ELECTRON TRANSPORT CHAIN"}, {"entity_type": "SUBSTRATE", "description": "A high-energy molecule used as a carrier of hydrogen atoms to drive ATP synthesis in the electron transport chain.<SEP>NADH is a molecule that functions in the electron transport chain and is involved in metabolic processes.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33<SEP>chunk-b930cdd3706af881cfa771b341af097d", "id": "NADH"}, {"entity_type": "COMPUTATIONAL TOOL", "description": "An in-depth computer-based method for predicting protein function and identifying potential disease-related mutations.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "IN SILICO ANALYSIS"}, {"entity_type": "GENETIC PHENOMENON", "description": "The coexistence of two or more different DNA sequences within the mitochondrial genome of a single cell.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "HETEROPLASMY"}, {"entity_type": "MOLECULAR TECHNIQUE", "description": "A method used for determining the complete sequence of nucleotides in a DNA molecule.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "DNA SEQUENCING"}, {"entity_type": "SEQUENCING PLATFORM", "description": "A high-throughput sequencing technology platform produced by Illumina, capable of generating large volumes of sequence data efficiently.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "ILLUMINA HISEQ 2000"}, {"entity_type": "SAMPLE TYPES", "description": "Biological samples collected for genetic testing, where maternal urine offers a non-invasive alternative to blood for some analyses.", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "id": "MATERNAL URINE AND BLOOD"}, {"entity_type": "GENOMIC VARIANT", "description": "The process of identifying specific genotypes or alleles in an individual\'s DNA sequence, often used in genetic research and diagnostics.<SEP>The purified cells were subjected to genome-wide gene expression analysis, genotyping and imputation using the 1000 Genomes reference panel.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca<SEP>chunk-b930cdd3706af881cfa771b341af097d", "id": "GENOTYPING"}, {"entity_type": "NUTRIENT", "description": "FMN (Flavin Mononucleotide) is a cofactor that works with NADH and iron-sulfur clusters for proper functioning of complex I.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "FMN"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Iron\u2013sulfur clusters are components that help in the proper function of the NADH binding site and FMN.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "IRON\u2013SULFUR CLUSTER"}, {"entity_type": "GENOMIC VARIANT", "description": "The Glu246Lys mutation occurs within the FMN-binding site and could be involved in the interaction with NADH.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "GLU246LYS MUTATION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "This is a protein complex that releases electrons from NADH and facilitates their movement to ubiquinone.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "COMPLEX 1 OF ELECTRON TRANSPORT CHAIN"}, {"entity_type": "GENE", "description": "COQ2\'s mutations have been associated with coenzyme Q10 deficiency (OMIM #609825).", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "COENZYME Q2 HOMOLOG, PRENYLTRANSFERASE (COQ2)"}, {"entity_type": "DIETARY PATTERN", "description": "The nuclear exome refers to the part of DNA that can be affected by environmental factors, including diet.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "NUCLEAR EXOME"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "This process involves sequencing depth and affects how well specific parts of the genome are analyzed.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MEAN TARGET NUCLEAR EXOME COVERAGE"}, {"entity_type": "DIETARY PATTERN", "description": "The mtDNA can be influenced by factors like diet, affecting the number and type of variants found.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MITOCHONDRIAL DNA (MTDNA)"}, {"entity_type": "VARIANT CLASSIFICATION", "description": "Variants classified as category 1-3 are considered potentially disease-causing or significant in genetic analysis.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "CATEGORY 1\u20133 VARIANTS"}, {"entity_type": "INHERITANCE TYPE", "description": "This pattern refers to the mode of inheritance where an individual needs two copies of a variant to express a trait or disorder.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "RECESSIVE INHERITANCE PATTERN"}, {"entity_type": "GENETIC CONDITION", "description": "Genes with these variants are typically associated with recessive genetic disorders.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "HOMOZYGOUS VARIANTS& TWO OR MORE VARIANTS"}, {"entity_type": "DISEASE", "description": "A disorder characterized by a deficiency in coenzyme Q10, affecting various biological processes.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "COENZYME Q10 DEFICIENCY (OMIM #609825)"}, {"entity_type": "DISEASE", "description": "This is a specific type of Leber\'s hereditary optic neuropathy, characterized by rapid vision loss in the young adult population.<SEP>This is one of the disorders that can result from mutations in MT-ND1.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "LEBER OPTIC ATROPHY (OMIM #535000)"}, {"entity_type": "DISEASE", "description": "A condition where individuals have defects or deficiencies affecting multiple subunits of respiratory complex I found in both nuclear and mitochondrial DNA.<SEP>Mutations in MT-ND1 may also lead to deficiencies affecting mitochondrial complexes involved in respiration.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MITOCHONDRIAL COMPLEX I DEFICIENCY (OMIM #252010)"}, {"entity_type": "DISEASE", "description": "Another disorder that can be associated with mutations in genes like MT-ND1, including muscle abnormalities and stroke-like episodes.<SEP>This specific disorder involves mutations within mtDNA that manifest with a diverse range of symptoms, including neurological issues and muscle abnormalities.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MELAS SYNDROME (OMIM #540000)"}, {"entity_type": "GENE", "description": "This gene encodes a subunit of respiratory complex I involved in mitochondrial energy production.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MT-ND1 GENE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "This is a protein complex that includes NADH, FMN and iron\u2013sulfur clusters for electron transport.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "COMPLEX I OF ELECTRON TRANSPORT CHAIN"}, {"entity_type": "ANALYTICAL DATA", "description": "These are used to estimate the heteroplasmy level of mitochondrial DNA variants.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "DIDEOXYNUCLEOTIDE PEAKS"}, {"entity_type": "VARIANT TYPE", "description": "This variant causes an arginine codon at position 387 to be replaced by a premature stop codon (X).", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "3754C>A MUTATION IN MT-ND1"}, {"entity_type": "DISEASE", "description": "A hereditary disease affecting the retina of the eye, often with progressive loss of vision.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "LEBER OPTIC ATROPHY"}, {"entity_type": "DISEASE", "description": "A condition involving deficiencies in multiple subunits encoded by mitochondrial DNA and nuclear genes which affects respiratory chain complexes.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MITOCHONDRIAL COMPLEX I DEFICIENCY"}, {"entity_type": "DISEASE", "description": "A multisystem disorder caused by mutations in mtDNA with symptoms that vary widely among affected individuals, including neurological deficits.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MELAS SYNDROME"}, {"entity_type": "SYSTEM", "description": "These are part of the mitochondrial electron transport chain involved in cellular respiration.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "RESPIRATORY CHAIN COMPLEXES"}, {"entity_type": "SYMPTOM GROUP", "description": "MELAS can present with symptoms including muscle weakness and atrophy.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MUSCLE ABNORMALITIES"}, {"entity_type": "SYMPTOM GROUP", "description": "In MELAS, affected individuals may experience episodic strokes or transient ischemic attacks.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "STROKE-LIKE EPISODES"}, {"entity_type": "GENE PRODUCT", "description": "These are components of respiratory complexes I and other mitochondrial enzymes that require correct protein expression for normal function.", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "id": "MITOCHONDRIAL DNA ENCODED SUBUNITS"}, {"source_id": "chunk-b930cdd3706af881cfa771b341af097d", "description": "A validated RFLP assay confirmed the findings on DNA from blood and urine samples from the child.)", "entity_type": "UNKNOWN", "id": "VALIDATION"}, {"source_id": "chunk-b930cdd3706af881cfa771b341af097d", "description": "Capillary sequencing methods were used to confirm the presence of mutations in both maternal urine and blood, though it may fail for heteroplasmy detection at low levels.)", "entity_type": "UNKNOWN", "id": "CONFIRMATION"}, {"source_id": "chunk-b930cdd3706af881cfa771b341af097d", "description": "Exome enrichment and sequencing on Illumina HiSeq 2000 was performed to generate comprehensive variant data for accurate genotyping.)", "entity_type": "UNKNOWN", "id": "DATA GENERATION"}, {"source_id": "chunk-b930cdd3706af881cfa771b341af097d", "description": "Capillary sequencing methods might fail to detect heteroplasmy below 10% even when mutations are present in blood samples.)", "entity_type": "UNKNOWN", "id": "DETECTION LIMIT"}, {"source_id": "chunk-b930cdd3706af881cfa771b341af097d", "description": "Application of a recessive inheritance model helped filter variants to identify two missense mutations in NDUFV1 gene shared by affected siblings, suggesting potential disease causation.)", "entity_type": "UNKNOWN", "id": "VARIANTS FILTERING"}, {"source_id": "chunk-b930cdd3706af881cfa771b341af097d", "description": "Identifying both infants as compound heterozygotes for the missense variants in NDUFV1 gene suggests they each carry one variant on opposite alleles, further supporting the pathogenicity of these mutations.)", "entity_type": "UNKNOWN", "id": "GENOTYPE INTERPRETATION"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "The Glu246Lys mutation in the FMN binding site could potentially affect COQ2 function, leading to potential disease.)", "entity_type": "UNKNOWN", "id": "COQ2"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "Variants in COQ2 have been linked to coenzyme Q10 deficiency, affecting the body\'s ability to produce essential compounds.)", "entity_type": "UNKNOWN", "id": "COQ2 GENE"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "The depth of nuclear exome sequencing determines the comprehensiveness and accuracy of genetic data analysis.)", "entity_type": "UNKNOWN", "id": "NUCLEAR DNA SEQUENCING DEPTH"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "Changes in dietary patterns can influence the number and type of mitochondrial variants found.)", "entity_type": "UNKNOWN", "id": "ENVIRONMENTAL FACTORS (INCLUDING DIET)"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "Changes in dietary patterns can influence the number and type of mitochondrial variants found.)", "entity_type": "UNKNOWN", "id": "MITOCHONDRIAL VARIANTS"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "Mutations in this gene lead to disorders like Leber optic atrophy, mitochondrial complex I deficiency, and MELAS.)", "entity_type": "UNKNOWN", "id": "MUTATIONS IN MT-ND1"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "Mutations in this gene lead to disorders like Leber optic atrophy, mitochondrial complex I deficiency, and MELAS.)", "entity_type": "UNKNOWN", "id": "NADH DEHYDROGENASE SUBUNIT 1 (MT-ND1)"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "Poor dietary choices may contribute to conditions involving defects in the respiratory chain components of mitochondria.)", "entity_type": "UNKNOWN", "id": "DIETARY PATTERN (SPECIFICALLY DIET AFFECTING MTDNA)"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "These genetic disorders are associated with mutations within the MT-ND1 gene that is a part of complex I.)", "entity_type": "UNKNOWN", "id": "LEBER OPTIC ATROPHY, MITOCHONDRIAL COMPLEX I DEFICIENCY, MELAS"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "Genes carrying two homozygous variants or two or more variants are often linked to recessive genetic disorders.)", "entity_type": "UNKNOWN", "id": "GENETIC DISORDERS (RECESSIVE INHERITANCE)"}, {"source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "description": "The heteroplasmic level indicates the mixture of normal and mutant mtDNA, affecting its expression within cells.)", "entity_type": "UNKNOWN", "id": "HETEROPLASMY LEVEL"}, {"entity_type": "GENE", "description": "OPRM1 encodes a mu-opioid receptor.<SEP>OPRM1 is a gene that plays a role in drug response and alcohol metabolism.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1<SEP>chunk-62d8719735e6c6eaa899d9ea64143544", "id": "OPRM1"}, {"entity_type": "DIETARY PATTERN", "description": "AUDIT score is a measure of alcohol use, classified under drinking variables.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "id": "AUDIT SCORE"}, {"entity_type": "DIETARY PATTERN", "description": "The number of drinking days is another measure of alcohol consumption, categorized as drinking variables.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "id": "NUMBER OF DRINKING DAYS"}, {"entity_type": "DIETARY PATTERN", "description": "Drinks consumed per drinking day is a metric for assessing alcohol intake frequency and quantity.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "id": "DRINKS PER DRINKING DAY"}, {"entity_type": "GENOMIC VARIANT", "description": "The ALDH2 variant (rs671) impacts the activity of aldehyde dehydrogenase, affecting alcohol metabolism.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "id": "ALDH2"}, {"entity_type": "GENOMIC VARIANT", "description": "The ADH1B variant (rs1229984) influences the breakdown of alcohol and other alcohols in the body.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "id": "ADH1B"}, {"entity_type": "SIDE EFFECT", "description": "A side effect experienced when taking naltrexone compared to placebo.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "id": "DROWSINESS"}, {"entity_type": "SIDE EFFECT", "description": "This auditory disturbance was observed with naltrexone use compared to placebo.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "id": "RINGING IN THE EARS"}, {"entity_type": "SIDE EFFECT", "description": "No significant medication associations were found for nausea among side effects measured by SAFTEE.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "id": "NAUSEA"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Alcohol Craving is influenced by Alcohol Breath Concentration (BrAC) and other factors.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "id": "AUDAC"}, {"entity_type": "METABOLIC PATHWAY", "description": "Alcohol Breath Concentration reflects the metabolic pathway of alcohol in the body.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "id": "BRAC"}, {"entity_type": "LIFESTYLE FACTOR", "description": "A binary variable indicating whether a participant is a current smoker.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "id": "CURRENT CIGARETTE SMOKING STATUS"}, {"entity_type": "COVARIATE", "description": "Used to represent current cigarette smoking status in statistical models.", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "id": "BINARY VARIABLE (0=NON-SMOKER, 1=SMOKER)"}, {"entity_type": "GENOTYPE", "description": "The Asn40 homozygotes in the placebo group showed effects on alcohol self-administration and consumption compared to other genotypes.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "ASN40 HOMOZYGOTES + PLACEBO GROUP"}, {"entity_type": "GENOTYPE", "description": "Carriers of Asp40 genotype consume significantly fewer drinks as compared to Asn40 homozygotes in alcohol self-administration experiments.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "ASP40 CARRIERS"}, {"entity_type": "GENETIC FACTOR", "description": "The ALDH2 genotype plays a role in the pharmacogenetic effects, specifically regarding nicotine smoking status and alcohol consumption.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "ALDH2 GENOTYPE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Nicotine smoking status is not significantly associated with any of the effects examined such as medication impact on alcohol self-administration or latency to drink.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "NICOTINE SMOKING STATUS"}, {"entity_type": "OUTCOME VARIABLE", "description": "The number of drinks consumed by participants does not have a significant main effect of medication as per Poisson regression analysis.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "TOTAL NUMBER OF DRINKS CONSUMED"}, {"entity_type": "OUTCOME VARIABLE", "description": "The time taken for the first drink was significantly influenced by genotype but not by medication, as shown in Cox proportional regression models.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "LATENCY TO FIRST DRINK"}, {"entity_type": "STATISTICAL INTERACTION", "description": "There is no significant interaction between medication and genotype regarding the effects on alcohol self-administration or latency to first drink as per presented analysis.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "MEDICATION \u00d7 GENOTYPE INTERACTION"}, {"entity_type": "GENETIC VARIANT", "description": "Carriers of different OPRM1 genotypes may exhibit varied responses to medication or alcohol based on their biological mechanisms.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "OPRM1 GENOTYPE"}, {"entity_type": "STATISTICAL METHODOLOGY", "description": "Poisson regression is utilized for the statistical analysis of count data such as total number of drinks consumed.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "POISSON REGRESSION ANALYSIS"}, {"entity_type": "STATISTICAL METHODOLOGY", "description": "A Cox proportional hazards regression model helps assess time-to-event outcomes like latency to drink.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "COX PROPORTIONAL REGRESSION MODEL"}, {"entity_type": "STATISTICAL TEST", "description": "The Wald chi-square test evaluates the significance of independent variables in logistic and Poisson regression models.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "WALD CHI-SQUARE TEST"}, {"entity_type": "STATISTICAL MEASURE", "description": "A Hazard Ratio represents the risk of an event occurring over time when comparing different groups, such as smokers vs non-smokers.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "HAZARD RATIO (HR)"}, {"entity_type": "DATA PROPERTY", "description": "The presence of a non-normal data distribution indicates that certain assumptions for certain statistical tests may not be met.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "NON-NORMAL DISTRIBUTION"}, {"entity_type": "STATISTICAL TERM", "description": "A main effect in ANOVA refers to the direct influence of one independent variable on the dependent variable, regardless of other variables.", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "id": "MAIN EFFECT"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "Two genetic variants related in influencing alcohol metabolism, drug response.)", "entity_type": "UNKNOWN", "id": "ADH1B GENOTYPE"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "Differences in side effects experienced when taking naltrexone compared to placebo.)", "entity_type": "UNKNOWN", "id": "MEDICATION (NALTREXONE/PLACEBO)"}, {"source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "description": "Differences in side effects experienced when taking naltrexone compared to placebo.)", "entity_type": "UNKNOWN", "id": "SIDE EFFECTS"}, {"source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "description": "The ALDH2 genotype\'s influence extends into environmental factors like smoking status in determining alcohol consumption patterns.)", "entity_type": "UNKNOWN", "id": "GENETIC FACTOR"}, {"source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "description": "In the context of the experiments, medication \u00d7 genotype interaction is not a significant factor for all outcomes as evidenced by analysis results.)", "entity_type": "UNKNOWN", "id": "STATISTICAL INTERACTION"}, {"source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "description": "The ALDH2 genotype is a statistical covariate affecting alcohol consumption in the models.)", "entity_type": "UNKNOWN", "id": "COVARIATE"}, {"source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "description": "The medication\'s impact on various outcomes was analyzed using chi-square tests for significance.)", "entity_type": "UNKNOWN", "id": "MEDICATION EFFECTS"}, {"source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "description": "The medication\'s impact on various outcomes was analyzed using chi-square tests for significance.)", "entity_type": "UNKNOWN", "id": "STATISTICAL ANALYSIS OUTCOME"}, {"source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "description": "Latency to first drink is a key outcome measure in evaluating the timing of alcohol consumption patterns.)", "entity_type": "UNKNOWN", "id": "LATENCY TO DRINK"}, {"source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "description": "Latency to first drink is a key outcome measure in evaluating the timing of alcohol consumption patterns.)", "entity_type": "UNKNOWN", "id": "OUTCOME MEASURE"}, {"entity_type": "GENOTYPE", "description": "Moderate treatment effect observed only in the GG genotype for both AC t1/2 and GC24.<SEP>The GG genotype had the highest daily alcohol consumption among men.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "GG GENOTYPE"}, {"entity_type": "GENOTYPE", "description": "Among men, the GA genotype group had lower levels of cortisol compared to the GG and AA genotypes.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "GA GENOTYPE"}, {"entity_type": "EXPOSURE", "description": "Daily alcohol consumption was positively associated with blood cortisol levels in both sexes.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "DAILY ALCOHOL CONSUMPTION"}, {"entity_type": "STATISTICAL MODEL", "description": "In univariate model, daily alcohol consumption was positively associated with blood cortisol levels in both sexes.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "UNIVARIATE MODEL"}, {"entity_type": "STATISTICAL MODEL", "description": "After adjustment for covariates like age, physical activity, smoking status, hypertension, diabetes, and education status, the association between daily alcohol consumption and blood cortisol levels remained similar or stronger in men compared to women.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "MULTIVARIATE LINEAR REGRESSION MODELS"}, {"entity_type": "CAUSAL ESTIMATE", "description": "In MR analysis for men, one standard drink/day of genetically predicted alcohol consumption was associated with an increase of 0.36 (95% CI: 0.21-0.51) in cortisol levels.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "GENETICALLY PREDICTED ALCOHOL CONSUMPTION"}, {"entity_type": "QUALITY CONTROL", "description": "A statistical method that evaluates how robust results are to removal or inclusion of individual data points.<SEP>In sensitivity analysis for men, one standard drink/day elevates cortisol by 0.27 (95% CI: 0.14-0.41).", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "SENSITIVITY ANALYSIS"}, {"entity_type": "DIETARY PATTERN", "description": "Ex-drinkers were included in sensitivity analysis.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "EX-DRINKER"}, {"entity_type": "NUTRIENT", "description": "A standard drink is a measure used to quantify alcohol consumption, equivalent to about 14 grams of pure alcohol.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "STANDARD DRINK/DAY"}, {"entity_type": "GENOMIC VARIANT", "description": "The ALDH2 rs671 polymorphism affects the activity of the aldehyde dehydrogenase enzyme and impacts alcohol metabolism.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "RS671 POLYMORPHISM"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Results from polysomnography were not explicitly mentioned but could be related to sleep patterns or quality, potentially affected by alcohol consumption.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "POLYSOMNOGRAPHY"}, {"entity_type": "VARIABLE", "description": "Education status was included as a covariate in the multivariate models.", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "id": "EDUCATION STATUS"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "The pattern of cortisol levels varied based on the ALDH2 rs671 genotypes by sex.)", "entity_type": "UNKNOWN", "id": "ALDH2 RS671 GENOTYPES"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "The pattern of cortisol levels varied based on the ALDH2 rs671 genotypes by sex.)", "entity_type": "UNKNOWN", "id": "CORTISOL LEVEL"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "There was a significant difference in age distribution among different genotypes.)", "entity_type": "UNKNOWN", "id": "AGE DISTRIBUTION"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "There was a significant difference in age distribution among different genotypes.)", "entity_type": "UNKNOWN", "id": "GENOTYPIC VARIANCE"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd<SEP>chunk-9e14f9bdee02839e4a6b12cff2020a5a<SEP>chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "BMI was associated with subgroups as shown by Table 1, where IBS-D had a BMI approximately 4 higher than IBS-C and HV.<SEP>HUA participants had higher BMI, suggesting possible obesity or overweight conditions.<SEP>The BMI differed according to the rs671 polymorphism significantly among men.)", "entity_type": "PHYSIOLOGICAL PROCESS", "id": "BODY MASS INDEX (BMI)"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "In both sexes, daily alcohol consumption was positively associated with blood cortisol levels.)", "entity_type": "UNKNOWN", "id": "BLOOD CORTISOL LEVELS"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "In MR analysis for men, one standard drink/day of genetically predicted alcohol consumption was associated with an increase of 0.36 in cortisol levels.)", "entity_type": "UNKNOWN", "id": "CORTISOL LEVEL (MR ANALYSIS)"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "In sensitivity analysis for men, one standard drink/day elevates cortisol by 0.27.)", "entity_type": "UNKNOWN", "id": "CORTISOL LEVEL (GENETICALLY PREDICTED ALCOHOL CONSUMPTION)"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "Ex-drinkers could be included in sensitivity analyses considering their genotypic status.)", "entity_type": "UNKNOWN", "id": "EX-DRINKERS"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "Ex-drinkers could be included in sensitivity analyses considering their genotypic status.)", "entity_type": "UNKNOWN", "id": "RS671 GENOTYPES"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "The genetically predicted alcohol consumption likely refers to the difference between the actual reported intake and a statistical estimate based on genetic factors.)", "entity_type": "UNKNOWN", "id": "GENETIC PREDISPOSITION AND ALCOHOL CONSUMPTION"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "Both terms refer to different aspects of the same ALDH2 gene polymorphism, with \'ALDH2\' being more common in literature but \'rs671\' specifying a particular SNP.)", "entity_type": "UNKNOWN", "id": "ALDH2 RS671"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "The inclusion of these covariates is to adjust for confounding factors when assessing the association between alcohol consumption and cortisol levels.)", "entity_type": "UNKNOWN", "id": "EDUCATION STATUS, HYPERTENSION, DIABETES"}, {"source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "description": "Genetic predisposition may influence how individuals engage in lifestyle behaviors such as physical activity.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 4}]", "id": "GENETIC FACTORS AND LIFESTYLE"}, {"entity_type": "DISEASE", "description": "The study population comprised a total of 1026 individuals, including males and females.<SEP>The term \'individuals\' refers to the study participants without genetic data, known as \'controls\'.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5<SEP>chunk-befd5ea20007b1d02d987897c798dadd", "id": "INDIVIDUALS"}, {"entity_type": "DISEASE", "description": "HUA (Hyperuricemia) prevalence was found to be 14.71% among the participants.", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "id": "PREVALENCE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Increased serum creatinine levels were observed in individuals with HUA, indicating renal function alteration.", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "id": "SERUM CREATININE"}, {"entity_type": "METABOLIC PATHWAY", "description": "High levels of TG were found among participants with HUA, potentially related to dyslipidemia.", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "id": "TRIGLYCERIDE (TG)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Increased LDL levels in HUA cases compared to controls may indicate a disrupted lipid profile.", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "id": "LOW-DENSITY LIPOPROTEIN (LDL)"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The usage ratio of diuretics showed no significant difference between the two groups.", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "id": "DIURETICS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "FBG levels were not significantly different in HUA cases and controls, suggesting similar insulin sensitivity.", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "id": "FASTING BLOOD GLUCOSE (FBG)"}, {"entity_type": "LIFESTYLE FACTOR", "description": "No significant difference in tobacco smoking history was found between the HUA cases and controls.", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "id": "TOBACCO SMOKING"}, {"entity_type": "DISEASE", "description": "There were no significant differences in diabetes history between HUA cases and control participants.", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "id": "HISTORY OF DIABETES"}, {"entity_type": "DISEASE/CONDITION/PARAMETER/FACTOR/LIFESTYLE/ENVIRONMENTAL FACTOR)", "description": "Gender/<Age Range>/Ethnicity", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "id": "SUBJECT"}, {"entity_type": "GENE", "description": "Individuals with the C allele in the SLC2A9 gene have lower serum uric acid (sUA) levels.<SEP>The C allele of the SLC2A9 gene is linked to lower serum uric acid (sUA) levels in G/G genotypes compared with T/T genotypes.", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "id": "SLC2A9 RS3733951 C"}, {"entity_type": "GENOMIC VARIANT", "description": "The T allele in rs2231142 of the ABCG2 gene increases the risk of hyperuricemia (HUA) especially in males with obesity, hypertension, higher TG levels and lower HDL levels.<SEP>The T allele of the rs2231142 variant of the ABCG2 gene is associated with higher sUA and HUA risk in males when accounting for cardiovascular risks.", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "id": "RS2231142 ABCG2 T"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Environmental factors such as obesity, hypertension, high TG levels, low HDL levels are associated with increased sUA level and HUA risk.", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "id": "CARDIOVASCULAR RISK FACTORS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Triglyceride levels are associated with high sUA level and increase the risk of HUA, especially in males and females with obesity and hypertension.", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "id": "TG LEVELS"}, {"entity_type": "GENOMIC VARIANT", "description": "The C allele of HMGCR rs3849267 is associated with lower serum uric acid (sUA) levels.", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "id": "HMGCR RS3849267 C"}, {"entity_type": "GENOMIC VARIANT", "description": "The C allele of rs1051730 variant is linked to lower sUA levels and HUA risk.", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "id": "RS1051730 C"}, {"entity_type": "GENOMIC VARIANT", "description": "The T allele in ABCG2 rs2231142 polymorphism increases the risk of hyperuricemia (HUA) especially when accounting for obesity, hypertension and other cardiovascular risks.", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "id": "ABCG2 RS2231142 T"}, {"entity_type": "METABOLIC PATHWAY", "description": "The levels of sUA (serum uric acid) are a part of the metabolic pathway.", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "id": "SUA LEVEL"}, {"entity_type": "GENETIC VARIANT", "description": "The rs2231142 genetic variant is associated with cardiovascular risk.", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "id": "RS2231142 T ALLELE"}, {"entity_type": "DISEASE", "description": "High uric acid levels (HUA) are a disease risk factor.", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "id": "HUA RISK"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "Increased sUA levels showed a significant association with the Beta coefficient of each copy of risk allele (p=0.002).)", "entity_type": "UNKNOWN", "id": "SLC2A9 RS3733591"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "Increased sUA levels showed a significant association with the Beta coefficient of each copy of risk allele (p=0.002).)", "entity_type": "UNKNOWN", "id": "URIC ACID LEVEL"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "There was a strong association between the ABCG2 variant\'s T allele and sUA levels and HUA risk. )", "entity_type": "UNKNOWN", "id": "ABCG2 RS2231142"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "The distribution of SLC22A12 variants showed no significant association with HUA risk.)", "entity_type": "UNKNOWN", "id": "SLC22A12 RS893006"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "The T allele of rs2231142 ABCG2 was associated with sUA levels and HUA in both males and females, while the C risk allele of SLC2A9 was only related to females. The G risk allele of SLC22A12 was associated with sUA level only in males.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION BETWEEN GENOTYPES AND SUA LEVEL"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "The association between different genotypes and HUA risk differed by gender: the T allele of rs2231142 ABCG2 showed significance, while there was no significant association with SLC22A12 rs893006 variants.)", "entity_type": "UNKNOWN", "id": "ASSOCIATION BETWEEN GENOTYPES AND HUA RISK"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "The polymorphism of ABCG2 rs2231142 was significantly associated with sUA levels (p=6.98e-8).)", "entity_type": "UNKNOWN", "id": "ABCOR2 RS2231142 POLYMORPHISM"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "In males, the G/G genotypes of the ABCG2 rs2231142 polymorphism had significantly lower sUA levels compared to T/T genotypes (p=4.40e-6).)", "entity_type": "UNKNOWN", "id": "G/G GENOTYPES"}, {"source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "description": "The G/G genotypes of the ABCG2 rs2231142 polymorphism had significantly lower sUA levels compared to T/T genotypes across all populations (p=6.98e-8).)", "entity_type": "UNKNOWN", "id": "T/T GENOTYPES"}, {"source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "description": "High blood pressure heightens the risk for hyperuricemia, which might be due in part to increased serum uric acid concentrations.)", "entity_type": "UNKNOWN", "id": "HYPERTENSION - ENVIRONMENTAL FACTOR"}, {"entity_type": "GENE", "description": "The LRP2 gene is associated with kidney function.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "LRP2"}, {"entity_type": "GENE", "description": "NFATC1 plays a central role in inducible gene transcription during immune response and is a downstream target of transplant immunosuppression drug cyclosporine A.<SEP>The NFATC1 gene has been discovered in a meta-analysis of European ancestry GWASs for eGFR.<SEP>The renal mRNA expression of NFATC1 was significantly decreased under high-salt diet exposure and showed a trend towards reduction in a CKD model.<SEP>This gene shows strong signals for association with eGFR and CKD.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083<SEP>chunk-3edc72730d68b4c7fd6ba397aa087391<SEP>chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "NFATC1"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant at the LRP2 locus was found to be associated with eGFR and leads to kidney function impairment.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "RS57989581"}, {"entity_type": "GENOMIC VARIANT", "description": "A new variant in NFATC1 was identified that is also significant for eGFR, indicating its role in kidney function.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "RS8096658"}, {"entity_type": "DISEASE", "description": "End-stage renal disease (ESRD) is associated with the eGFR-decreasing alleles at various genetic loci.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "ESRD"}, {"entity_type": "DISEASE", "description": "The associations between T1D susceptibility, changes in expression for IRF1 and STAT1 are associated with a specific genetic region.<SEP>Type 1 diabetes was also found associated with eGFR in GWASs, though not as strongly as type 2 diabetes.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45<SEP>chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "T1D"}, {"entity_type": "GENE", "description": "The PDILT-UMOD gene locus demonstrates strong associations with both CKD and kidney function, indicating a critical role in these functions.<SEP>The PDILT-UMOD locus is found to have distinct genetic signals associated with both kidney function and CKD risk, indicating a critical role in these processes.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "PDILT-UMOD"}, {"entity_type": "GENE", "description": "Another identified genetic variant (rs62435145) associated with increased risk of CKD.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "UNCX"}, {"entity_type": "GENE", "description": "The ALMS1 gene shows nominal evidence of association with CKD, indicating its potential role in renal function related diseases.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "ALMS1"}, {"entity_type": "GENOMIC VARIANT", "description": "A variant at the ALMS1 gene shows association with CKD risk, contributing to our understanding of genetic factors influencing renal health.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "RS7587577"}, {"entity_type": "GENOMIC VARIANT", "description": "The variant rs77924615 at the PDILT-UMOD locus demonstrates strong associations with both kidney function and CKD, highlighting its genetic impact on these processes.<SEP>This variant at the PDILT-UMOD locus has been implicated in driving association signals for both eGFR and CKD, emphasizing its importance.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 53}]", "id": "RS77924615"}, {"entity_type": "GENE", "description": "The SLC34A1 gene contributes to eGFR associations.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "SLC34A1"}, {"entity_type": "GENE", "description": "The gene DCDC5-MPPED2 is involved at four variants mapping to 27.9kb, which might have implications for disease susceptibility or function.<SEP>This genetic locus has significant association signals for both kidney function and related diseases, indicating potential mechanisms in renal health.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083<SEP>chunk-3edc72730d68b4c7fd6ba397aa087391", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 53}]", "id": "DCDC5-MPPED2"}, {"entity_type": "GENE", "description": "The PRKAG2 gene shows evidence of association with eGFR in diabetic individuals, suggesting a role in diabetes-linked kidney dysfunction.", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "id": "PRKAG2"}, {"entity_type": "GENE", "description": "RGS14 encodes a member of the regulator of G protein signaling family which modulates downstream effects of G\u03b1 subunits in kidneys.<SEP>Rgs14\'s renal mRNA expression was notably reduced during high salt exposure and approached statistical significance in the CKD model, suggesting altered gene expression under stress conditions.)(entity", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083<SEP>chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "RGS14"}, {"entity_type": "TRANSCRIPT", "description": "In Drosophila, loco is the closest RGS14 ortholog with reduced expression associated with longer lifespan and stress resistance.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "DROSOPHILA MELANOGASTER (LOCUS)"}, {"entity_type": "PROTEIN", "description": "HDAC inhibitors are involved in kidney function and development, which may suggest their role in kidney disease treatments as they affect histone deacetylases (HDACs).", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 40}]", "id": "RSK8"}, {"entity_type": "GENE", "description": "The gene PIP5K1B has five variants mapping to 3.5kb and could be associated with mechanisms impacting eGFR in the context of genetic fine-mapping and genomic annotation studies.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "PIPK1B"}, {"entity_type": "DISEASE/GENE", "description": "NFATC1 is a gene that plays a central role in inducible gene transcription during immune response, which might be related to transplant immunosuppression drug cyclosporine A.<SEP>NFATC1 plays a central role in inducible gene transcription during immune response and is a downstream target of the transplant immunosuppression drug cyclosporine A.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "NFATC1 (MIM: 602513)"}, {"entity_type": "DISEASE/GENE", "description": "RGS14 encodes a member of the regulator of G protein signaling family which modulates downstream effects of G\u03b1 subunits and has unknown function in kidneys.<SEP>RGS14 is a gene encoding a member of the regulator of G protein signaling family, which modulates downstream effects of G\u03b1 subunits in kidneys.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "RGS14 (MIM: 602513)"}, {"entity_type": "DISEASE/GENE", "description": "SLC34A1 is a gene involved at two variants mapping to 0.6kb, which has been associated with risk of CKD and can influence eGFR through its role in the kidney.<SEP>SLC34A1 is a gene with variants involved that might have implications for understanding mechanisms related to kidney diseases and functions.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "SLC34A1 (MIM: 182309)"}, {"entity_type": "DISEASE/GENE", "description": "GCKR is a gene with three variants mapping to 11.7kb that has been linked to increased risk of CKD and its p.Pro446Leu variant specifically.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "GCKR (MIM: 600842)"}, {"entity_type": "DISEASE/GENE", "description": "This gene is associated with four variants mapping to 27.9kb that could impact the mechanisms involved in genetic fine-mapping and genomic annotation studies.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "DCDC5-MPPED2 (MIM: 612321, 600911)"}, {"entity_type": "DISEASE/GENE", "description": "The gene PIP5K1B has five variants mapping to 3.5kb that are of interest in the context of understanding genetic associations with eGFR.", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "id": "PIPK1B (MIM: 602745)"}, {"entity_type": "GENE", "description": "loco mutants with this gene showed improved survival when fed NaCl-supplemented food.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "LOCOD06164"}, {"entity_type": "GENOMIC VARIANT", "description": "The y1w1 genomic variant had little effect on viability but was associated with resistance to salt stress in loco mutants.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "Y1W1"}, {"entity_type": "GENE", "description": "The locoEY-P283 gene showed similar properties as the locod06164 gene in terms of salt tolerance when fed NaCl-supplemented food.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "LOCOEY-P283"}, {"entity_type": "NUTRIENT", "description": "The supplementation of diet with varying NaCl concentrations helped to assess the stress resistance of genotypes with the loco mutation.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "NACL"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "There was no baseline difference in viability among genotypes fed non-NaCl supplemented food during testing period.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "NON-NACL-SUPPLEMENTED FOOD"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The survival of heterozygous loco mutants was improved on NaCl-supplemented food compared to controls, indicating a role for the gene in resistance to salt stress.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "SURVIVAL"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Experiments with hybrid heterogenic backgrounds were conducted to exclude effects of inbreeding depression on findings.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "INBREEDING DEPRESSION"}, {"entity_type": "DISEASE", "description": "These hybrid genotypes were less susceptible to salt stress than their isogenic counterparts, suggesting a role for genetic diversity in adaptation to environmental pressures.<SEP>These strains were less salt-susceptible than isogenic strains, showing that genetic background impacts salt stress resistance.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "HYBRID HETEROGENIC STRAINS"}, {"entity_type": "DIETARY PATTERN", "description": "The 129S6 mouse strain\'s physiological responses were studied under high-salt diet conditions and during CKD models.<SEP>This strain was used in studies to examine roles of genes NFATC1 and RGS14 under high-salt diet conditions.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "129S6 MOUSE STRAIN"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "A two-week exposure to a high-salt diet influenced renal mRNA expression of Nfatc1 and Rgs14.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "HIGH-SALT DIET"}, {"entity_type": "TIME PERIOD", "description": "The 2-week duration allowed us to assess changes in gene expression under high-salt conditions.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "2-WEEK EXPOSURE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A CKD model using sub-total nephrectomy induced a renal disease state that was associated with altered gene expression of NFATC1 and RGS14.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "SUB-TOTAL NEPHRECTOMY"}, {"entity_type": "DISEASE", "description": "Renal disease susceptibility increased in this model as indicated by the decrease in Rgs14 and Nfatc1 mRNA expressions during CKD conditions.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "CKD MODEL"}, {"entity_type": "MOLECULAR INTERACTION", "description": "High salt intake was found to affect mRNA expression levels of genes like Nfatc1 and Rgs14, suggesting a complex relationship between dietary factors and gene regulation.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "HIGH-SALT EXPOSURE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Sub-total nephrectomy as part of the CKD model introduced a chronic kidney disease state which altered gene expressions related to renal function.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "CKD-INDUCED NEPHRECTOMY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "These terms refer to the varying responses of different genotypes towards dietary salt intake, indicating potential genetic determinants for resistance or sensitivity.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "SALT SUSCEPTIBILITY"}, {"entity_type": "DISEASE", "description": "The improved survival observed in certain genotypes on NaCl-supplemented food suggests they have natural defenses against environmental stressors like salt.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "RESISTANCE TO STRESS"}, {"entity_type": "RELATIONSHIP STRENGTH", "description": "Although not definitively established, the data supports a potential link between dietary factors, gene expression changes, and disease states, implying these elements are interrelated in complex biological pathways.", "source_id": "chunk-5ac437b050552f5db2abf19f1bf897cf", "id": "CAUSE AND EFFECT"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "Variability in allele frequencies between different ancestry groups does not affect the allelic effects on eGFR, suggesting a shared genetic basis for this measurement across populations.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 27}, {level: 1, cluster: 53}]", "id": "ANCESTRY GROUPS"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "SNP rs62435145 associated with increased risk of CKD through its influence on kidney function.)", "entity_type": "UNKNOWN", "id": "CKD"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "SNP rs62435145 associated with increased risk of CKD through its influence on kidney function.)", "entity_type": "UNKNOWN", "id": "RS62435145"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "The NFATC1 gene shows evidence of association with diabetic nephropathy and altered eGFR in both type 1 and type 2 diabetes.)", "entity_type": "UNKNOWN", "id": "T2D, T1D"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "A collective of genes and specific genetic variations contribute to the complex interplay between CKD risk, eGFR measurements, and underlying pathologies.)", "entity_type": "UNKNOWN", "id": "CKD, EGFR"}, {"source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "description": "A collective of genes and specific genetic variations contribute to the complex interplay between CKD risk, eGFR measurements, and underlying pathologies.)", "entity_type": "UNKNOWN", "id": "UNCX, ALMS1, PDILT-UMOD, NFATC1"}, {"entity_type": "DISEASE", "description": "Campylobacter infections were confirmed in 975 patients; out of these, 18.2% (177 cases) had quinolone-resistant isolates and 0.3% (3 cases) had erythromycin-resistant strains.", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "id": "CAMPYLOBACTER"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Quinolone resistance was significantly associated with the origin of infection; among travelers returning to Denmark, 50.0% (76 out of 152) were infected by quinolone-resistant strains, while domestically acquired infections showed a lower proportion (9.9%, 52 out of 526).", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "id": "QUINOLONE RESISTANCE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Foreign travel was linked to higher proportions of quinolone resistance; travel destinations such as Spain and Portugal were specifically mentioned.", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "id": "TRAVEL"}, {"entity_type": "DRUG", "description": "Quinolones were the drug class related to the study, with specific emphasis on their efficacy against Campylobacter infections.", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "id": "QUINOLONE"}, {"entity_type": "GENOMIC VARIANT", "description": "Of the three erythromycin-resistant isolates, two were also quinolone-resistant and originated from travelers returning to Denmark from Spain and Portugal. This indicates possible genetic variations among strains.", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "id": "CAMPYLOBACTER COLI"}, {"entity_type": "ACTIVITY", "description": "Swimming was another activity associated with an increased risk for quinolone-resistant C. jejuni infection.", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "id": "SWIMMING"}, {"entity_type": "DEMOGRAPHIC FACTOR", "description": "There were no significant age-related differences in the findings regarding quinolone resistance, though data were collected across different age groups.", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "id": "AGE GROUP"}, {"entity_type": "SPECIES", "description": "C. jejuni is a type of Campylobacter bacterium that was frequently found as resistant to fluoroquinolones.<SEP>Most of the human isolates were C. jejuni strains.", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c<SEP>chunk-0d155c005072e55d94179ce7952bf5a9", "id": "CAMPYLOBACTER JEJUNI"}, {"entity_type": "ANTIBIOTIC SENSITIVITY", "description": "The majority (574 out of 975) of Campylobacter infections were sensitive to fluoroquinolones, indicating resistance was more prevalent.", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "id": "QUINOLONE-SENSITIVE"}, {"entity_type": "DRUG CLASS", "description": "Fluoroquinolones are a class of antibiotics commonly used in the treatment and prevention of C. jejuni infections.<SEP>Infections treated with fluoroquinolones before specimen collection were excluded.", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c<SEP>chunk-0d155c005072e55d94179ce7952bf5a9", "id": "FLUOROQUINOLONES"}, {"entity_type": "GENETIC GROUPING", "description": "The RiboGroups are used for classifying and grouping strains together based on their ribosomal RNA sequences.", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "id": "RIBOGROUP"}, {"entity_type": "PHENOTYPIC CHARACTERISTIC", "description": "Quinolone-resistant isolates were characterized by specific patterns indicating the mechanism of resistance.", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "id": "RESISTANCE PATTERN"}, {"entity_type": "GENETIC CHARACTERISTIC", "description": "Several silent mutations were identified within the gyrA gene of quinolone-resistant Campylobacter strains, contributing to resistance without altering protein function.", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "id": "SILENT MUTATIONS"}, {"entity_type": "DISEASE", "description": "The study identified 12 quinolone-resistant cases that were domestically acquired.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "QUINOLONE-RESISTANT CASES"}, {"entity_type": "ANTIMICROBIAL DRUGS", "description": "The study found information on antimicrobial drugs received by 122 of 126 patients.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "AMOXICILLIN-CLAVULANATE, AZITHROMYCIN, DOXYCYCLINE, CLARITHROMYCIN, LEVOFLOXACIN, COTRIMOXAZOLE, SULFAMETHOXAZOLE/TRIMETHOPRIM, TETRACYCLINE"}, {"entity_type": "DISEASE", "description": "Of the human isolates, most (63%) were from Danish-bred food animals; the remaining were imported from France (n=48), Italy (n=7), and the United Kingdom (n=9).", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "C. JEJUNI"}, {"entity_type": "DRUG", "description": "The human isolates were included in a database, compared with Campylobacter isolates obtained from retail food products (chicken [n=139], turkey [n=39], and pork [n=2])", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "FLAVOXATE, CLARITHROMYCIN"}, {"entity_type": "FOOD", "description": "Most of the isolates were from Danish-bred food animals.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "RETAIL FOOD PRODUCTS"}, {"entity_type": "FOOD", "description": "Three chicken isolates were unknown in origin.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "BROILER CHICKENS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Of 49 isolates from broiler chicken fecal samples, two isolates (4.1%) were nalidixic acid-resistant; one was also erythromycin-resistant.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "NONDOMESTIC"}, {"entity_type": "DRUG RESISTANCE", "description": "Five (2.0%) of the 153 C. jejuni isolates and five (18.5%) of the 27 C. coli isolates were resistant to erythromycin.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "NALIDIXIC ACID-RESISTANCE"}, {"entity_type": "DRUG RESISTANCE", "description": "A subanalysis showed that seven (8.0%) of eighty-seven C. jejuni isolates and three (60%) of five C. coli isolates from domestic-raised chicken products were resistant to nalidixic acid.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "ERYTHROMYCIN RESISTANCE"}, {"entity_type": "DRUG RESISTANCE", "description": "Of the 49 isolates from broiler chicken fecal samples, seven were erythromycin-resistant; one was also nalidixic acid-resistant.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "C. COLI RESISTANCE"}, {"entity_type": "BACTERIAL PATHOGEN", "description": "The human isolates were included in a database and compared with Campylobacter isolates obtained from retail food products (chicken [n = 139], turkey [n = 39], and pork [n=2])", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "CAMPYLOBACTER ISOLATES"}, {"entity_type": "BACTERIAL STRAIN", "description": "The remaining isolates were C. coli strains.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "C. COLI"}, {"entity_type": "DRUG", "description": "Five (2.0%) of the 153 C. jejuni isolates and one (18.5%) of the 27 C. coli isolates were resistant to quinalenoxacin.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "QUINALENOXACIN"}, {"entity_type": "DRUG", "description": "One isolate (a C. jejuni) from domestic chicken products was resistant to both nalidixic acid and ciprofloxacin.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "CIPROFLOXACIN"}, {"entity_type": "DRUG", "description": "Two isolates were erythromycin-resistant; one of which was also nalidixic acid-resistant.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "ERYTHROMYCIN"}, {"entity_type": "DRUG", "description": "Five (10.2%) isolates (four C. jejuni, one C. coli) were erythromycin-resistant.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "CLARITHROMYCIN"}, {"entity_type": "DRUG RESISTANCE PHENOTYPE", "description": "Eighteen subtypes were identified exclusively among quinolone-resistant isolates; five of these 18 subtypes matched those found in retail food products and broiler chickens.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "QUINOLONE-RESISTANT ISOLATES"}, {"entity_type": "DRUG SENSITIVITY PHENOTYPE", "description": "One hundred and two (76.7%) subtypes were exclusively among quinolone-sensitive isolates; 34 of these types also matched human isolates, indicating a link between dietary sources and drug sensitivity.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "QUINOLONE-SENSITIVE ISOLATES"}, {"entity_type": "DRUG RESISTANCE PHENOTYPE", "description": "Five (20.8%) of the quinolone-resistant isolates were identified among retail food products, broiler chickens, or both; three (14.7%) of these from chicken products.", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "QUINOLONE-RESISTANT C. JEJUNI"}, {"entity_type": "CONTEXTUAL LOCATION", "description": "The likelihood that domestically acquired C. jejuni strains had subtypes found in retail food products and broiler chickens was higher than those with quinolone-resistant isolates (270 of 444 vs. 15 of 51, RR = 2.07, CI 1.34 to 3.18, p < 0.001).", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "id": "DOMESTICALLY ACQUIRED C. JEJUNI ISOLATES"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "Eating specific types of poultry (excluding chicken and turkey) was identified as a risk factor for quinolone-resistant C. jejuni infection based on the case-comparison study.)", "entity_type": "UNKNOWN", "id": "RISK FACTOR"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "Fluoroquinolones were recommended as a primary treatment for C. jejuni infections until their ineffectiveness against resistant strains was recognized.)", "entity_type": "UNKNOWN", "id": "TREATMENT APPROACH"}, {"source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "description": "The prevalence of fluoroquinolone resistance was notably higher among travelers compared to domestically acquired cases, as indicated by the study\'s data analysis.)", "entity_type": "UNKNOWN", "id": "PREVALENCE COMPARISON"}, {"source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "description": "Infections treated with fluoroquinolones before specimen collection were excluded, indicating a distinction between domestically acquired and travel-related infections.)", "entity_type": "UNKNOWN", "id": "DOMESTICALLY ACQUIRED"}, {"source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "description": "The majority of human isolates came from Danish-bred food animals, suggesting a connection to foodborne transmission.)", "entity_type": "UNKNOWN", "id": "FOODBORNE"}, {"source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "description": "Of the retail food products compared with broiler chickens, most were from domestic sources, which was different than imported isolates.)", "entity_type": "UNKNOWN", "id": "ORIGIN"}, {"source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "description": "Several of the isolates identified as quinolone-resistant or -sensitive matched those found in human patients, indicating a potential shared source.)", "entity_type": "UNKNOWN", "id": "C. JEJUNI ISOLATES FROM RETAIL FOOD PRODUCTS AND BROILER CHICKENS"}, {"source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "description": "Several of the isolates identified as quinolone-resistant or -sensitive matched those found in human patients, indicating a potential shared source.)", "entity_type": "UNKNOWN", "id": "SIMILARITY OR COHORT"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant is located in HTR2A and affects serotonin receptor activity.<SEP>This variant is located in HTR2A and is part of the serotonin pathway.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS9534511"}, {"entity_type": "GENE", "description": "HTR2A encodes a serotonin receptor.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "HTR2A"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant is located in HTR3B and is part of the serotonin pathway.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS4938056"}, {"entity_type": "GENE", "description": "HTR3B encodes a serotonin receptor.<SEP>The rs4938056 variant impacts HTR3B, also a serotonin receptor gene.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "HTR3B"}, {"entity_type": "GENOMIC VARIANT", "description": "Located in an intronic region of OPRM1, which influences opioid receptor function.<SEP>This variant is located in an intronic region of OPRM1, which is part of the opioid receptor gene.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS1998220"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant resides in an intronic region of KCNC1, which is related to ion channel activity.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS1003921"}, {"entity_type": "GENE", "description": "KCNC1 encodes a voltage-gated potassium channel protein 1.<SEP>The rs1003921 variant is associated with voltage-gated potassium channel activity.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "KCNC1"}, {"entity_type": "GENOMIC VARIANT", "description": "This intergenic variant affects NPY neurotransmission regulation pathways.<SEP>This variant is in an intergenic region downstream of NPY, which is related to neurotransmission regulation.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS16116"}, {"entity_type": "GENE", "description": "NPY encodes a neuropeptide Y.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "NPY"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant is part of LINC00518, an uncharacterized RNA gene which potentially influences cellular processes.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "RS16870286"}, {"entity_type": "GENE", "description": "LINC00518 encodes for an uncharacterized protein or functions as a long non-coding RNA.<SEP>The rs16870286 variant impacts long non-coding RNA LINC00518, possibly affecting gene expression or other cellular processes.", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "id": "LINC00518"}, {"entity_type": "DISEASE", "description": "The social disinhibition score was increased by 0.20 points (95% CI: 0.17\u20130.23) and by 0.45 points (95% CI: 0.40\u20130.51), respectively, in association with three SNPs.<SEP>The social disinhibition score was increased by a certain amount associated with specific SNPs.", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 40}]", "id": "SOCIAL DISINHIBITION SCORE"}, {"source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "description": "An increase in age is correlated with a higher social disinhibition score.)<SEP>Increasing age is positively associated with increased social disinhibition score.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 18}]", "id": "SOCIAL DISINHIBITION"}, {"source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "description": "Higher linguistic acculturation scores are positively related to higher mean social disinhibition scores.)<SEP>Increased linguistic acculturation positively correlates with social disinhibition scores.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 18}]", "id": "LINGUISTIC ACCULTURATION"}, {"source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "description": "The total anxiety score (TAS) is significantly and positively associated with social disinhibition.)<SEP>Total Anxiety Scale (TAS) score is significantly correlated to higher social disinhibition scores.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 18}]", "id": "TAS"}, {"source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "description": "Depression Anxiety Scale (DAS) score shows a positive correlation with the social disinhibition measure.)<SEP>Depression anxiety score (DAS) is significantly related to increased risk of social disinhibition.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 18}]", "id": "DAS"}, {"source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "description": "Three of the SNPs were associated with increased risk while three had protective effects on the social disinhibition score.)", "entity_type": "UNKNOWN", "id": "MINOR ALLELES"}, {"entity_type": "GENE", "description": "CNR1 gene is located in a region focused on the rs2023239 variant, which was found near exon 3 as an alternative promoter.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "CNR1"}, {"entity_type": "GENOMIC VARIANT", "description": "The rs2023239 genetic variant was examined in relation to the rewarding effects of alcohol in a previous study.<SEP>The rs2023239 genomic variant is located upstream of exon 3 and has been associated with CB1 receptor levels.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e<SEP>chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "RS2023239"}, {"entity_type": "PROTEIN", "description": "CB1 receptors modulate the mesocorticolimbic neurocircuitry related to drug-seeking behavior in animals, influenced by genetic variations like rs2023239.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "CB1 RECEPTOR"}, {"entity_type": "TISSUE SAMPLE", "description": "Post-mortem human prefrontal cortex (PFC) tissue was analyzed for the association between rs2023239 and CB1 receptor levels.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "POSTMORTEM PFC"}, {"entity_type": "STATISTICAL METHOD", "description": "ANOVA was used to compare the relationship of the CT genotype with CB1 binding in Brodmann areas 9 and 10.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "ANOVA"}, {"entity_type": "MOLECULAR COMPONENT", "description": "CNR1 receptors modulate the mesocorticolimbic neurocircuitry involved in drug-seeking behavior.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "CNR1 RECEPTORS"}, {"entity_type": "BEHAVIORAL TRAIT", "description": "The genetic variation affecting CB1 receptor levels was expected to associate with differential activation during alcohol exposure.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "ALCOHOL SEEKING BEHAVIOR"}, {"entity_type": "RESEARCH METHOD", "description": "Neuroimaging data from a recent study were reanalyzed to explore the functional significance of rs2023239 on brain activation.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "NEUROIMAGING STUDY"}, {"entity_type": "STATISTICAL METHOD", "description": "The ANOVA analysis revealed significant differences in CB1 receptor binding between genotypes.", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "id": "ANOVA ANALYSIS"}, {"entity_type": "DRUG", "description": "The study analyzed an effect of the medication olanzapine.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "MEDICATION: OLANZAPINE"}, {"entity_type": "DRUG", "description": "The study compared the effect of cyproheptadine to olanzapine in its analysis.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "CYPROHEPTADINE HYDROCHLORIDE"}, {"entity_type": "THERAPY", "description": "The recent clinical trial analyzed the effects of olanzapine on cue-elicited craving and drinks per drinking day.", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "id": "CLINICAL TRIAL"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "The genetic variant rs2023239 is linked to CB1 receptor levels.)", "entity_type": "UNKNOWN", "id": "CB1 RECEPTOR LEVEL"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "The rs2023239 variant has been associated with altered brain activation patterns during alcohol exposure.)", "entity_type": "UNKNOWN", "id": "BRAIN ACTIVATION"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "The CT group had a significant correlation between alcohol use disorder scores and certain neural activations.)", "entity_type": "UNKNOWN", "id": "ALCOHOL USE DISORDER SCORE"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "The CT group had a significant correlation between alcohol use disorder scores and certain neural activations.)", "entity_type": "UNKNOWN", "id": "CT GROUP"}, {"source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "description": "14)(entity", "entity_type": "UNKNOWN", "id": "REWARD RESPONSE<|THE CT GROUP DEMONSTRATED A SIGNIFICANT INCREASE IN REWARD RESPONSE COMPARED TO THE TT GROUP AFTER CONSUMING ALCOHOL.)"}, {"source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "description": "Individuals with the CT genotype reported greater reward from alcoholic beverages.)", "entity_type": "UNKNOWN", "id": "GENOTYPE CT"}, {"source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "description": "Individuals with the CT genotype reported greater reward from alcoholic beverages.)", "entity_type": "UNKNOWN", "id": "REWARDING EFFECTS"}, {"source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "description": "In an earlier study, individuals with genotypes CT or CC showed this interaction when receiving active control medication (cyproheptadine).)", "entity_type": "UNKNOWN", "id": "GENOTYPE CT AND CC"}, {"source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "description": "In an earlier study, individuals with genotypes CT or CC showed this interaction when receiving active control medication (cyproheptadine).)", "entity_type": "UNKNOWN", "id": "MEDICATION \u00d7 GENOTYPE \u00d7 DRINK INTERACTION"}, {"source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "description": "There might be an interaction between CNR1 variation and the effect of olanzapine on alcohol\'s rewarding impact.)", "entity_type": "UNKNOWN", "id": "CNR1 VARIATION AND OLANZAPINE"}, {"source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "description": "There might be an interaction between CNR1 variation and the effect of olanzapine on alcohol\'s rewarding impact.)", "entity_type": "UNKNOWN", "id": "INTERACTION ON REWARDING EFFECTS OF ALCOHOL"}, {"source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "description": "In a recent clinical trial, the CT genotype showed greater cue-elicited craving when treated with placebo compared to olanzapine.)", "entity_type": "UNKNOWN", "id": "MEDICATION \u00d7 GENOTYPE \u00d7 CUE INTERACTION ON CRAVING"}, {"source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "description": "In the recent clinical trial, individuals with the CT genotype reported more drinks per drinking day than those with the TT genotype when they were treated with both placebo vs. olanzapine.)<SEP>Individuals with CT genotype reported more drinks per drinking day than those with TT genotype when treated with placebo versus olanzapine.)", "entity_type": "UNKNOWN", "id": "CT GENOTYPE AND TT GENOTYPE"}, {"source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "description": "The analyses revealed a genotype-drink interaction when individuals were treated with olanzapine.)", "entity_type": "UNKNOWN", "id": "GENOTYPE \u00d7 DRINK INTERACTION"}, {"source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "description": "There were no significant genotype x medication interactions on reward from alcohol beverages when cyproheptadine was compared to olanzapine.)", "entity_type": "UNKNOWN", "id": "GENOTYPE \u00d7 MEDICATION INTERACTIONS"}, {"source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "description": "There were no significant genotype x medication interactions on reward from alcohol beverages when cyproheptadine was compared to olanzapine.)", "entity_type": "UNKNOWN", "id": "MEDICATION: OLANZAPINE AND CYPROHEPTADINE HYDROCHLORIDE"}, {"source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "description": "In a recent study, there was a significant medication-genotype-cue interaction for craving when individuals were given different medications (cyproheptadine vs. placebo) compared to olanzapine.)", "entity_type": "UNKNOWN", "id": "MEDICATION \u00d7 GENOTYPE \u00d7 CUE INTERACTION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Involves packaging of dietary fat into chylomicrons in the small intestine, which are later transported to the bloodstream.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "EXOGENOUS LIPID METABOLISM"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Initial step involves hydrolysis of chylomicrons by lipoprotein lipase (LPL).", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "ENDOGENOUS LIPID METABOLISM"}, {"entity_type": "PROTEIN", "description": "The primary protein involved in breaking down cholesterol and triglycerides in lipoproteins.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "LIPOPROTEIN LPL"}, {"entity_type": "DISEASE", "description": "Rare autosomal recessive disorder with low prevalence due to a founder effect.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "LPL DEFICIENCY"}, {"entity_type": "CONDITION", "description": "Present in affected individuals, particularly homozygotes or compound heterozygotes of LPL deficiency.<SEP>This disease is characterized by abnormally high levels of triglycerides in the blood, which can lead to various health complications.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63<SEP>chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "HYPERTRIGLYCERIDEMIA"}, {"entity_type": "CLINICAL FEATURE", "description": "Reddish-yellow papular rash that appears during the disease progression.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "ERUPTIVE XANTHOMAS"}, {"entity_type": "CLINICAL FEATURE", "description": "A common manifestation of high triglyceride levels, indicating potential health risks or discomfort.<SEP>Symptom experienced by individuals affected by LPL deficiency.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63<SEP>chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "ABDOMINAL PAIN"}, {"entity_type": "CLINICAL FEATURE", "description": "Enlargement of liver and spleen, often observed in LPL deficiency patients.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "HEPATO-SPLENOMEGALY"}, {"entity_type": "CONDITION", "description": "Repeated episodes of inflammation within the pancreas due to LPL deficiency.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "RECURRENT PANCREATITIS"}, {"entity_type": "CONDITION", "description": "Eye condition characterized by cloudy, white vitreous humor caused by high lipid levels in the blood.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "LIPEMIA RETINALIS"}, {"entity_type": "TREATMENT", "description": "Fundamental aspect of therapy for managing LPL deficiency.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "DIETARY FAT RESTRICTION"}, {"entity_type": "MEDICATION", "description": "Formula with a higher proportion of medium-chain triglycerides compared to long-chain fats.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "ENFAPORT LIPIL FORMULA"}, {"entity_type": "GENETIC MUTATION", "description": "Autosomal recessive mutation within exon 5 of the LPL gene causes deficiency.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "LPL GENE MUTATION"}, {"entity_type": "MUTATION", "description": "Transition from G to A at codon position 644 results in substitution of glycine for glutamic acid.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "P.G215E"}, {"entity_type": "MUTATION", "description": "Transversion from G to T at codon position 721 causes an amino acid change from glutamine to histidine.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "LPL P.Q240H"}, {"entity_type": "LOCATION", "description": "Located on chromosome 8p21.3, spanning approximately 30 kb with a mature protein length of 448 amino acids.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "LPL LOCUS"}, {"entity_type": "CLASSIFICATION", "description": "Over 150 mutations identified as pathogenic in the LPL gene, including missense, nonsense, and other types.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "PATHOGENIC LPL MUTATIONS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Identified for transporting LPL to capillary lumen by recent studies.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "GPIHBP1"}, {"entity_type": "CLASSIFICATION", "description": "Mutation in this patient expands known mutations to over 150, including involvement of residue 240 highlighting functional importance.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "LPL DEFICIENCY MUTATION SPECTRUM"}, {"entity_type": "LIFESTYLE FACTOR", "description": "Method of infant feeding which may influence lipid profiles.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "BREAST-FEEDING"}, {"entity_type": "FORMULA", "description": "Special formula with higher fat content from medium-chain triglycerides and lower long-chain fats.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "ENFAPORT LIPIL"}, {"entity_type": "PROPORTION OF FAT CONTENT IN ENFAPORT LIPIL", "description": "Specific composition for dietary intervention in LPL deficiency treatment.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "84% MEDIUM-CHAIN TRIGLYCERIDES"}, {"entity_type": "PROPORTION OF FAT CONTENT IN ENFAPORT LIPIL", "description": "Particular type of fat to manage lipid metabolism disorders like LPL deficiency.<SEP>Specific composition for dietary intervention in LPL deficiency treatment.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "16% LONG-CHAIN FATS"}, {"entity_type": "FAT TYPE IN ENFAPORT LIPIL", "description": "Particular type of fat to manage lipid metabolism disorders like LPL deficiency.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "84% MCT (MEDIUM-CHAIN TRIGLYCERIDES)"}, {"entity_type": "TECHNIQUE", "description": "Kit used for extracting DNA from blood samples during genetic analysis.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "PUREGENE DNA ISOLATION KIT"}, {"entity_type": "MANUFACTURER", "description": "Company that produces the Puregene DNA isolation kit.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "GENTRA SYSTEMS, QIAGEN INC"}, {"entity_type": "METHOD", "description": "Polymerase Chain Reaction process for amplifying DNA sequences during genetic studies.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "PCR AMPLIFICATIONS"}, {"entity_type": "REAGENTS", "description": "Specific DNA sequences used in PCR to amplify particular parts of the genome.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "PRIMERS"}, {"entity_type": "GENETIC ANALYSIS AREAS", "description": "Targeted regions of LPL gene for mutation detection through Sanger sequencing.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "CODING REGIONS AND INTRON-EXON BOUNDARIES"}, {"entity_type": "ANALYTICAL TECHNIQUE", "description": "Method used to determine the DNA sequence of PCR products.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "BI-DIRECTIONAL SANGER SEQUENCING"}, {"entity_type": "SOFTWARE", "description": "Used software tool for analyzing genetic data obtained from sequencing experiments.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "SEQSCAPE V2.6"}, {"entity_type": "ETHICAL AND REGULATORY APPROVAL", "description": "Protocol approved by the Western University Ethics Review Board for genetic study.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "GENETIC ANALYSIS BOARD PROTOCOL 07920E"}, {"entity_type": "CONSENT PROCESS", "description": "Procedure to ensure participants\' agreement before their data is used in research.", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "id": "INFORMED CONSENT"}, {"entity_type": "GENOMIC VARIANT", "description": "This is a novel mutation in exon 5 of the LPL gene causing severe hypertriglyceridemia.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "P.Q240H"}, {"entity_type": "GENOMIC VARIANT", "description": "Exon 5 of the LPL gene contains the p.Q240H mutation causing hypertriglyceridemia.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "EXON 5"}, {"entity_type": "PROTEIN", "description": "LPL performs essential functions including lipolysis and fat metabolism; its malfunction due to mutations leads to hypertriglyceridemia.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "LIPOPROTEIN LIPASE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The p.Q240H mutation in exon 5 affects the protein structure/function, influencing LPL activity.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "EXON 5 MUTATION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "This genetic mutation is an inherent factor that predisposes individuals to hypertriglyceridemia.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "P.Q240H MUTATION"}, {"entity_type": "DRUG", "description": "A drug used for treating high triglyceride levels by inhibiting the synthesis of cholesterol and triglycerides.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "GEMFIBROZIL"}, {"entity_type": "NUTRIENT", "description": "These are recommended in dietary therapy for reducing triglyceride levels due to their beneficial effects on lipid metabolism.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "OMEGA-3 FATTY ACIDS"}, {"entity_type": "DRUG", "description": "An over-the-counter medication that aids weight loss by blocking the absorption of fats from food, which can help manage hypertriglyceridemia.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "ORLISTAT"}, {"entity_type": "MOLECULAR INTERACTION", "description": "DGAT1 inhibitors target a key enzyme involved in triglyceride synthesis during fat absorption; this approach aims to reduce triglyceride levels.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "DGAT1 INHIBITORS"}, {"entity_type": "GENOMIC VARIANT", "description": "A variant of the LPL gene that can confer beneficial effects, though it\'s costly and has limited long-term effectiveness.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "GAIN-OF-FUNCTION VARIANT"}, {"entity_type": "THERAPY", "description": "This procedure involves introducing a therapeutic agent into muscle tissue to treat specific diseases, used here for the LPL gene in Europe.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "INTRAMUSCULAR INTRODUCTION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "DGAT1 facilitates triglyceride synthesis; inhibiting it can be a strategy for managing hypertriglyceridemia.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "DIACYLGLYCEROL O-ACYLTRANSFERASE 1 (DGAT1)"}, {"entity_type": "SYMPTOM", "description": "A potentially severe complication associated with elevated triglycerides, possibly due to hyperviscosity or other factors.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "GASTROINTESTINAL HEMORRHAGE"}, {"entity_type": "DISEASE", "description": "An inflammation of the pancreas that can occur in individuals with high triglyceride levels; it often requires medical intervention.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "PANCREATITIS"}, {"entity_type": "DEVELOPMENTAL OUTCOME", "description": "In individuals with LPL deficiency, dietary fat restriction may not impede linear growth despite potential health challenges.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "LINEAR GROWTH"}, {"entity_type": "DEVELOPMENTAL PROCESS", "description": "Children and adolescents with LPL deficiency can experience normal puberty despite restrictions on fatty foods.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "PUBERTY"}, {"entity_type": "DIETARY INTERVENTION", "description": "A dietary approach to manage hypertriglyceridemia by reducing fat intake; however, it may have limitations for growing individuals.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "FAT RESTRICTION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Exposure to sunlight affects vitamin D synthesis, which influences lipid metabolism.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "SUNLIGHT EXPOSURE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Vitamin D impacts fat absorption and metabolism, playing a role in managing hypertriglyceridemia.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "VITAMIN D SYNTHESIS"}, {"entity_type": "BIOMEDICAL PROCESS", "description": "Vitamin D modulates lipid metabolism through interactions with specific enzymes and proteins involved in fat handling.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "LIPID METABOLISM"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The actions of these biological catalysts are crucial for vitamin D-mediated effects on lipid metabolism.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "ENZYMES AND PROTEINS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Healthy pancreatic function is essential, as it affects fat absorption and other metabolic processes that influence triglyceride levels.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "PANCREATIC FUNCTION"}, {"entity_type": "BIOMEDICAL FUNCTION", "description": "The process by which the body absorbs dietary fats plays a role in lipid metabolism and can affect hypertriglyceridemia.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "FAT ABSORPTION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Enzyme activities influence fat metabolism, including those that may be modulated by vitamin D or impacted by hypertriglyceridemia.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "ENZYME ACTIVITY"}, {"entity_type": "PHYSIOLOGICAL SYSTEM", "description": "The gastrointestinal tract\'s health affects fat absorption and subsequent lipid levels.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "GASTROINTESTINAL TRACT"}, {"entity_type": "MOLECULAR COMPONENTS", "description": "These proteins play roles in transporting, storing, and managing fatty acids throughout the body, impacting triglyceride metabolism.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "FATTY ACID BINDING PROTEINS"}, {"entity_type": "DRUG EFFECT", "description": "The way drugs like gemfibrozil or orlistat affect the body and manage triglycerides is part of their pharmacodynamic profile.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "PHARMACODYNAMICS"}, {"entity_type": "BIOMEDICAL OUTCOME", "description": "The effectiveness of various therapeutic interventions in managing hypertriglyceridemia is a measure of their therapeutic efficacy.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "THERAPEUTIC EFFICACY"}, {"entity_type": "BIOMEDICAL RESPONSE", "description": "Potential side effects from drug therapies or treatments for hypertriglyceridemia are biomedical responses that need monitoring.", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "id": "ADVERSE EFFECTS"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Linking genetic disorder to symptoms such as hypertriglyceridemia, xanthomas, abdominal pain, etc.)", "entity_type": "UNKNOWN", "id": "CLINICAL FEATURES"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Direct link between lifestyle change and managing the disease.)", "entity_type": "UNKNOWN", "id": "TREATMENT FOR LPL DEFICIENCY"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Understanding spectrum of mutations and expanding known cases.)", "entity_type": "UNKNOWN", "id": "AUTOSOMAL RECESSIVE LPL DEFICIENCY MUTATION"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Understanding spectrum of mutations and expanding known cases.)", "entity_type": "UNKNOWN", "id": "PATHOGENIC MUTATIONS IN LPL GENE"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Role played by GPIHBP1 in transporting the enzyme to capillary lumen.)", "entity_type": "UNKNOWN", "id": "MOLECULAR INTERACTION WITH LPL"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Link between genetic mutation causing LPL deficiency and its clinical symptoms such as hypertriglyceridemia, xanthomas, etc.)", "entity_type": "UNKNOWN", "id": "AUTOSOMAL RECESSIVE LPL DEFICIENCY MUTATION AND CLINICAL MANIFESTATIONS"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Association between the specific fat content composition in Enfaport Lipil formula and improvement in lipid profiles as a treatment for LPL deficiency.)", "entity_type": "UNKNOWN", "id": "FORMULA COMPOSITION (ENFAPORT LIPIL) AND TREATMENT EFFICACY"}, {"source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "description": "Association between the specific fat content composition in Enfaport Lipil formula and improvement in lipid profiles as a treatment for LPL deficiency.)", "entity_type": "UNKNOWN", "id": "OUTCOME RELATIONSHIP"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "The genomic DNA analysis was performed on the subjects to identify genetic variants like p.Q240H.)", "entity_type": "UNKNOWN", "id": "GENOMIC DNA ANALYSIS"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "Long-term dietary fat restriction is a cornerstone of treatment for LPL deficiency, helping manage hypertriglyceridemia.)", "entity_type": "UNKNOWN", "id": "TREATMENT"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "MCT-based formulas are used as fat supplements because they\'re absorbed directly into the portal vein, aiding in fat management.)", "entity_type": "UNKNOWN", "id": "FAT SUPPLEMENT"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "MCT-based formulas are used as fat supplements because they\'re absorbed directly into the portal vein, aiding in fat management.)", "entity_type": "UNKNOWN", "id": "MCT-BASED FORMULA"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "The use of DGAT1 inhibitors to target triglyceride synthesis is an investigational treatment approach for LPL deficiency.)", "entity_type": "UNKNOWN", "id": "DGAT1 INHIBITORS AND THERAPY"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "Combined with a fat-restricted diet, drugs like gemfibrozil and omega-3 fatty acids help manage high triglyceride levels in individuals with LPL deficiency.)", "entity_type": "UNKNOWN", "id": "GEMFIBROZIL AND OMEGA-3 FATTY ACIDS"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "An approved gene therapy procedure for managing hypertriglyceridemia involves introducing a gain-of-function variant of the LPL gene.)", "entity_type": "UNKNOWN", "id": "GENE THERAPY"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "An approved gene therapy procedure for managing hypertriglyceridemia involves introducing a gain-of-function variant of the LPL gene.)", "entity_type": "UNKNOWN", "id": "INTRAMUSCULAR INTRODUCTION OF GAIN-OF-FUNCTION VARIANT"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "Orlistat is used as an over-the-counter weight loss aid, which may also help manage hypertriglyceridemia by blocking fat absorption.)", "entity_type": "UNKNOWN", "id": "DRUG MANAGEMENT"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "Orlistat is used as an over-the-counter weight loss aid, which may also help manage hypertriglyceridemia by blocking fat absorption.)", "entity_type": "UNKNOWN", "id": "ORLISTAT AND OVER-THE-COUNTER USE"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "Elevated triglyceride levels can lead to abdominal pain and hyperviscosity, which may cause gastrointestinal issues.)", "entity_type": "UNKNOWN", "id": "ABDOMINAL PAIN AND HYPERVISCOSITY"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "Elevated triglyceride levels can lead to abdominal pain and hyperviscosity, which may cause gastrointestinal issues.)", "entity_type": "UNKNOWN", "id": "SYMPTOM AND COMPLICATION RELATIONSHIP"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "An elevated risk of pancreatitis exists in individuals with high triglyceride levels due to factors like fat metabolism dysfunction.)", "entity_type": "UNKNOWN", "id": "DISEASE AND CAUSE RELATIONSHIP"}, {"source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "description": "An elevated risk of pancreatitis exists in individuals with high triglyceride levels due to factors like fat metabolism dysfunction.)", "entity_type": "UNKNOWN", "id": "PANCREATITIS AND HIGH TRIGLYCERIDES"}, {"entity_type": "DRUG", "description": "A medication used for treating patients during treatment.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "TOPIRAMATE"}, {"entity_type": "DRUG", "description": "A control group in the study which did not receive Topiramate.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "PLACEBO"}, {"entity_type": "GENOMIC VARIANT", "description": "A genetic variant associated with the response to topiramate treatment.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "RS2832407"}, {"entity_type": "DISEASE", "description": "The ethnic group being studied, which experiences a range of responses and adverse events related to treatments.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "EUROPEAN-AMERICAN (EA)"}, {"entity_type": "OUTCOME MEASURE", "description": "A metric used for measuring efficacy in the treatment outcomes.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "HDDS (HEADACHE DAILY DIARY SCORES)"}, {"entity_type": "STATISTICAL MEASURE", "description": "The number of patients who need to be treated with topiramate to prevent one additional headache daily diary score.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "NNT (NUMBER NEEDED TO TREAT)"}, {"entity_type": "OUTCOME MEASURE", "description": "Events or symptoms that occur as side effects of treatments.<SEP>Unwanted effects or symptoms that occur as a result of treatment, including severity and frequency.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "ADVERSE EVENTS"}, {"entity_type": "ADVERSE EVENT LEVEL", "description": "The least severe type of adverse event in the study.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "MILD ADVERSE EVENT"}, {"entity_type": "ADVERSE EVENT LEVEL", "description": "An adverse event that is more significant than mild but not as severe as severe.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "MODERATE ADVERSE EVENT"}, {"entity_type": "ADVERSE EVENT LEVEL", "description": "A serious and potentially life-threatening adverse effect resulting from treatment.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "SEVERE ADVERSE EVENT"}, {"entity_type": "STATISTICAL MEASURE", "description": "The adjusted number needed to treat considering the presence of moderate or greater adverse events.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "NNT-AEMOD+ (ADJUSTED NNT FOR MODERATE EVENTS)"}, {"entity_type": "STATISTICAL MEASURE", "description": "The adjusted number needed to treat considering severe adverse events only.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "NNT-AESEV+"}, {"entity_type": "OUTCOME MEASURE", "description": "The measure used to determine how well a treatment is working.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "EFFICACY"}, {"entity_type": "OUTCOME MEASURE", "description": "A subset of adverse events that are considered more severe and potentially life-threatening.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "SEVERE OR GREATER ADVERSE EVENTS"}, {"entity_type": "STUDY GROUP", "description": "The subgroup being studied with a defined demographic.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "EUROPEAN-AMERICAN SUBGROUP (N=122)"}, {"entity_type": "GENOMIC VARIANT*EA PATIENTS", "description": "A genetic variant found in European-American patients.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "RS2832407*C-HOMOZYGOTES"}, {"entity_type": "GENOMIC VARIANT*EA PATIENTS", "description": "Individuals carrying the A-allele for rs2832407 genetic variant.", "source_id": "chunk-0c7ab28db433187759e760725a105836", "id": "A-ALLELE CARRIERS"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836", "description": "Number Needed to Treat (NNT) quantifies how many patients need treatment with Topiramate to prevent one additional headache daily diary score.)", "entity_type": "UNKNOWN", "id": "NNT"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836", "description": "The EA ethnicity shows differences in response to Topiramate based on the genetic variant rs2832407.)", "entity_type": "UNKNOWN", "id": "EUROPEAN-AMERICAN (EA)"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836", "description": "The EA ethnicity shows differences in response to Topiramate based on the genetic variant rs2832407.)", "entity_type": "UNKNOWN", "id": "RS2832407*TOPIRAMATE"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836", "description": "A high majority of EA patients experience mild adverse events, which is not statistically different between treatment groups.)", "entity_type": "UNKNOWN", "id": "92% OF EA PATIENTS"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836", "description": "For rs2832407*C-homozygotes, moderate adverse events are more frequent with topiramate compared to placebo.)", "entity_type": "UNKNOWN", "id": "MODERATE OR GREATER ADVERSE EVENTS"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836", "description": "For rs2832407*C-homozygotes, moderate adverse events are more frequent with topiramate compared to placebo.)", "entity_type": "UNKNOWN", "id": "RS2832407*C-HOMOZYGOTES*TOPIRAMATE VS. PLACEBO"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836", "description": "For rs2832407*A-allele carriers, there is a significant difference in the frequency of severe adverse events between topiramate and placebo.)", "entity_type": "UNKNOWN", "id": "RS2832407*A-ALLELE CARRIERS*TOPIRAMATE VS. PLACEBO"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836", "description": "For rs2832407*A-allele carriers, there is a significant difference in the frequency of severe adverse events between topiramate and placebo.)", "entity_type": "UNKNOWN", "id": "SEVERE ADVERSE EVENTS"}, {"source_id": "chunk-0c7ab28db433187759e760725a105836", "description": "For rs2832407*C-homozygotes, the adjusted number needed to treat considering moderate adverse events is lower for Topiramate.)", "entity_type": "UNKNOWN", "id": "NNT-AEMOD+"}, {"entity_type": "CLINICAL", "description": "A total of 120 eligible patients who were approached signed consent (90%). Of these, 107 (89%) completed the study, genotyping was conducted on 93 patients, and 88 patients had complete self-report and genetic data. Therefore, 88 patients were included in the current analyses.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "PATIENT CHARACTERISTICS"}, {"entity_type": "DEMOGRAPHIC", "description": "had an annual household income of at least $40,000 per year (60%)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "INCOME"}, {"entity_type": "CLINICAL", "description": "Most patients were diagnosed with gastrointestinal (34%), thoracic (27%) or breast malignancies (26%)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "DIAGNOSIS"}, {"entity_type": "CLINICAL", "description": "had advanced stage disease (74%)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "STAGE OF DISEASE"}, {"entity_type": "CLINICAL", "description": "received highly-emetogenic chemotherapy regimens (65%)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "CHEMOTHERAPY REGIMEN"}, {"entity_type": "DRUG", "description": "Prescribed anti-emetic medications are shown in Table II. Nearly all participants received palonosetron (99%) and dexamethasone (98%)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "ANTI-EMETIC MEDICATIONS"}, {"entity_type": "PATIENT-REPORTED", "description": "In the univariate analyses examining patient-reported risk factors for CINV, patient-reported severity of acute nausea was associated with younger age (r=\u22120.32, p<0.01) and fewer hours of sleep the night before chemotherapy (r=\u22120.34, p<0.01).", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "SELF-REPORTED NAUSEA"}, {"entity_type": "OUTCOME", "description": "3% in the acute period and 11% in the delayed period experienced CTCAE grade of vomiting.<SEP>A total 12% of patients experienced CTCAE grade 2 or above acute nausea.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "CTCAE GRADE 2 OR ABOVE ACUTE NAUSEA"}, {"entity_type": "OUTCOME", "description": "30% experienced delayed nausea.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "DELAYED NAUSEA"}, {"entity_type": "OUTCOME", "description": "The severity of CTCAE grade of acute nausea was associated with younger age (r=\u22120.24, p=0.02), fewer hours of sleep the night before chemotherapy (r=0.29, p<0.01), and history of nausea and vomiting due to morning sickness (r=0.22, p=0.04).", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "CTCAE GRADE 3 OR ABOVE NAUSEA"}, {"entity_type": "CLINICAL", "description": "Severity of CTCAE grade of delayed nausea was associated with advanced stage cancer (r=\u22120.22, p=0.04)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "ADVANCED STAGE CANCER"}, {"entity_type": "CLINICAL", "description": "Receipt of highly emetogenic chemotherapy (65%)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "RECEIPT OF HIGHLY EMETOGENIC CHEMOTHERAPY"}, {"entity_type": "GENETIC", "description": "There was an association between variation in ABCB1 (rs1045642), which was associated with acute nausea, and the severity of acute nausea.", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "VARIATION IN ABCB1 (RS1045642)"}, {"entity_type": "STATISTICAL", "description": "In multivariable analysis, demographic, clinical, and patient-reported risk factors accounted for 13% of variance in acute nausea (p=0.03), while ABCB1 genotype added an additional 5% of variance (p=0.10).", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "id": "MULTIVARIABLE ANALYSIS OF ACUTE NAUSEA"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Almost all patients were prescribed palonosetron, a common anti-emetic medication.)", "entity_type": "UNKNOWN", "id": "ANTI-EMETIC"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Almost all patients were prescribed palonosetron, a common anti-emetic medication.)", "entity_type": "UNKNOWN", "id": "PALONOSETRON"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "98% of the participants received dexamethasone, another commonly used anti-emetic drug.)", "entity_type": "UNKNOWN", "id": "DEXAMETHASONE"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Younger age was associated with increased severity of patient-reported acute nausea.)", "entity_type": "UNKNOWN", "id": "RISK FACTOR FOR ACUTE NAUSEA"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Younger age was associated with increased severity of patient-reported acute nausea.)", "entity_type": "UNKNOWN", "id": "YOUNGER AGE"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Reduced sleep prior to chemotherapy was significantly linked to higher severity of patient-reported acute nausea.)", "entity_type": "UNKNOWN", "id": "FEWER HOURS OF SLEEP"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "The younger age was not associated with acute vomiting, based on CTCAE grading.)", "entity_type": "UNKNOWN", "id": "RISK FACTOR FOR ACUTE VOMITING"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "The association between advanced stage cancer and the severity of delayed nausea.)", "entity_type": "UNKNOWN", "id": "RISK FACTOR FOR DELAYED NAUSEA"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "The receipt of highly emetogenic chemotherapy was significantly linked to increased risk factors for nausea, both acute and delayed.)", "entity_type": "UNKNOWN", "id": "HIGHLY EMETOGENIC CHEMOTHERAPY"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "The receipt of highly emetogenic chemotherapy was significantly linked to increased risk factors for nausea, both acute and delayed.)", "entity_type": "UNKNOWN", "id": "RISK FACTOR FOR NAUSEA"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "Genetic variation in ABCB1 (rs1045642) was linked to more severe acute nausea, with heterozygotes reporting more severe symptoms than homozygous wild type or variant genotypes.)", "entity_type": "UNKNOWN", "id": "ABCB1 RS1045642 GENOTYPE"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "These factors together explained a significant portion of the variance in acute nausea severity.)", "entity_type": "UNKNOWN", "id": "DEMOGRAPHIC, CLINICAL, PATIENT-REPORTED VARIABLES"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "These factors together explained a significant portion of the variance in acute nausea severity.)", "entity_type": "UNKNOWN", "id": "RISK FACTORS FOR ACUTE NAUSEA"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "The ABCB1 genotype significantly enhanced the risk factors for acute nausea in conjunction with other demographic, clinical, and patient-reported variables.)", "entity_type": "UNKNOWN", "id": "ABCB1 GENOTYPE"}, {"source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "description": "The ABCB1 genotype significantly enhanced the risk factors for acute nausea in conjunction with other demographic, clinical, and patient-reported variables.)", "entity_type": "UNKNOWN", "id": "ENHANCER OF ACUTE NAUSEA RISK"}, {"entity_type": "GENE", "description": "A gene related to the study, possibly involved in regulatory interactions with LCT.<SEP>The MCM6 gene exhibits similar regions of DNA modification when compared to LCT intron 5 and the introns/exon identified in inter-individual analysis.<SEP>The MCM6 gene interacts with LCT, potentially affecting lactose metabolism and persistence.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d<SEP>chunk-512b044cf65a4e519d519c6239959c6c<SEP>chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "MCM6"}, {"entity_type": "DIETARY PATTERN", "description": "Lactase activity is crucial for lactose digestion, a dietary process relevant to human health.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "LACTOSE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Changes in DNA modification patterns are associated with changes in gene expression and may influence lactase persistence.<SEP>Such DNA modifications occur within MCM6 intron 13 during embryogenesis.", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1<SEP>chunk-ec104044b3aa65560be8d81dbb3940d6", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 44}]", "id": "DNA MODIFICATION"}, {"entity_type": "CELL TYPE", "description": "Intestinal epithelial cells play a role in the regulation of gene expression, including that of LCT, affecting lactose metabolism.<SEP>Regions identified were specific to Enterocytes, showing differences in DNA modifications compared to the enterocyte-deficient jejunum.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d<SEP>chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "ENTEROCYTES"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Surgical jejunum samples are obtained from unrelated human adults to study genetic and epigenetic factors influencing lactase persistence.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "JEJUNUM SAMPLE"}, {"entity_type": "TECHNOLOGICAL TOOL", "description": "High-throughput genotyping platforms used for chromosome-wide investigation of gene regulation and expression.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "AFFYMETRIX HIGH DENSITY HUMAN TILING ARRAYS"}, {"entity_type": "TECHNOLOGICAL TOOL", "description": "Method for enriching unmodified DNA fractions to study epigenetic changes in gene expression.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "MTAG-CLICK METHOD"}, {"entity_type": "TECHNOLOGICAL TOOL", "description": "Microarray-based technology allowing for chromosome-wide investigations in mouse genomes.<SEP>Microarray-based technology utilized for epigenetic analysis of mouse chromosomes, including Lct and related regions.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "MOUSE TILING MICROARRAYS"}, {"entity_type": "DISEASE", "description": "Variability among individuals affects lactase expression and may contribute to lactose intolerance or persistence.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "INTER-INDIVIDUAL DIFFERENCES"}, {"entity_type": "DISEASE", "description": "Cell-type specific differences in DNA modifications may indicate mechanisms underlying cell-specific expression of genes involved in lactase activity and metabolism, potentially impacting lactase non-persistence or persistence.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "CELL TYPE SPECIFIC EXPRESSION"}, {"entity_type": "TRANSCRIPT", "description": "Messenger RNA from the Lactase gene which is involved in lactose metabolism.<SEP>The mRNA transcript of the LCT gene is crucial for lactase activity and metabolism.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c<SEP>chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "LCT MRNA"}, {"entity_type": "CELL TYPE", "description": "Cells lining the digestive tract, primarily involved in nutrient absorption and immune responses.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "INTESTINAL EPITHELIAL CELLS"}, {"entity_type": "GENETIC BACKGROUND", "description": "A specific mouse strain commonly used in biological research due to its genetic stability and consistency.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "INBRED C57BL/6N MOUSE LINE"}, {"entity_type": "BIOMEDICAL MEASURE", "description": "Quantitative assessment of DNA modification patterns within a genome or region, relevant for gene regulation studies.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "DNA MODIFICATION DENSITIES"}, {"entity_type": "MOLECULAR TECHNIQUE", "description": "A method for determining DNA methylation patterns, utilized to fine-map specific regions of the genome.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "BISULFITE PADLOCK PROBE SEQUENCING"}, {"entity_type": "VARIATION TYPE", "description": "Alterations in DNA sequence or structure within a genetic locus that can affect gene expression, protein function, or phenotypic traits.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "GENETIC VARIATION"}, {"entity_type": "BIOLOGICAL COMPONENT", "description": "A segment of DNA, RNA or protein that controls the initiation, rate, and/or specificity of gene expression.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "REGULATORY REGION"}, {"entity_type": "OBSERVATION", "description": "The observable characteristics of a cell resulting from its genetic makeup and environmental interactions.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "CELLULAR PHENOTYPE"}, {"entity_type": "BIOLOGICAL MATERIAL", "description": "A segment of small intestine tissue obtained for analysis, often used to study gastrointestinal function or disease states.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "HUMAN JEJUNUM SAMPLE"}, {"entity_type": "STATISTICAL MEASURE", "description": "The comparison of gene expression levels between different biological conditions or samples, indicating altered transcriptional activity.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "DIFFERENTIAL EXPRESSION"}, {"entity_type": "GENETIC PHENOMENON", "description": "The variation in genetic characteristics among individuals within a species, contributing to phenotypic diversity.", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "id": "INTER-INDIVIDUAL VARIABILITY"}, {"entity_type": "GENOMIC VARIANT", "description": "Significant DNA modification differences were observed at 35 CpG sites when comparing enterocytes with enterocyte-deficient jejunum tissues.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "CPG SITES"}, {"entity_type": "DNA MODIFICATION", "description": "Significant DNA modifications observed at CpG sites associated with the LCT gene and MCM6.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "CPG SITE MODIFICATION"}, {"entity_type": "TISSUE", "description": "The jejunum tissue did not show the same DNA modifications as enterocytes, indicating cell-type specific regulation of DNA modifications.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "JEJUNUM TISSUE"}, {"entity_type": "LNCRNA", "description": "The identified region in LCT\'s intron 5 maps to this long non-coding RNA, contributing to its regulation.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "LONG NON-CODING RNA (LNCRNA) LO100507600"}, {"entity_type": "REGULATORY REGION", "description": "The exon and intron regions of the MCM6 gene were found modified and regulatory for chromatin features like DNase I hypersensitivity sites, H3K4me1, and H3K27ac.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "3\u2032-UTR OF MCM6"}, {"entity_type": "CHROMATIN FEATURES", "description": "The identified regions in LCT introns 2 and 3 and MCM6 intron 13, exon 16, and 3\u2032-UTR show DNase I hypersensitivity.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "DNASE I HYPERSENSITIVITY SITES"}, {"entity_type": "CHROMATIN FEATURES", "description": "These histone marks indicate active enhancers at the LCT introns 2 and 3, as well as MCM6 intron 13 and exon 16 to 3\u2032-UTR.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "HISTONE MARKS H3K4ME1 AND H3K27AC"}, {"entity_type": "ENHANCER SITES", "description": "No other enhancer sites were observed elsewhere in a 100 kb region centered over LCT and MCM6, indicating specificity of these identified regions.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "ACTIVE ENHANCER SITES"}, {"entity_type": "GENETIC BACKGROUND", "description": "The LCT-MCM6 haplotypes exhibit strong associations with lactase status across genetic backgrounds.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "HAPLOTYPES ASSOCIATED WITH LACTASE NON-PERSISTENCE AND PERSISTENCE"}, {"entity_type": "TEMPORAL CHANGES", "description": "Differential epigenetic modifications occur in different haplotypes leading to age-dependent regulation of LCT expression.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "AGE-DEPENDENT REGULATION OF LCT"}, {"entity_type": "SINGLE NUCLEOTIDE POLYMORPHISM (SNP)", "description": "This SNP is associated with the lactase persistence and non-persistence genotypes, and its presence leads to age-related DNA modification changes.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "SNP C/T-13910"}, {"entity_type": "INTERACTING GENES", "description": "A study found an interplay between TERT and INT1 genes that affects the expression of LCT, suggesting a network regulatory mechanism.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "TERT-INT1 INTERPLAY"}, {"entity_type": "METHODOLOGY", "description": "The approach used to identify genes associated with defects in RPS19 in DBA patients.<SEP>The researchers used gene expression profiling to understand the impact of DNA modifications on different cell types like enterocytes.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d<SEP>chunk-34ffb077c584b98555081f882f8f8b94", "id": "GENE EXPRESSION PROFILING"}, {"entity_type": "METHYLATION TYPE", "description": "CpG methylation patterns were specifically analyzed at 35 CpG sites, indicating a link between specific DNA modifications and enterocyte differentiation.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "CPG METHYLATION"}, {"entity_type": "EXPERIMENTAL MODEL", "description": "The use of enterocyte-deficient jejunum tissues in comparison to normal tissues provided insights into cell-type specific DNA modification effects on LCT expression.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "ENTEROCYTE DEFICIENT TISSUE"}, {"entity_type": "ANALYTICAL METHOD", "description": "This method was used to identify the regions where DNA modifications occurred consistently within individuals (intra-individual) or across different individuals (inter-individual).", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "INTRA-INDIVIDUAL AND INTER-INDIVIDUAL ANALYSIS"}, {"entity_type": "SPECIFIC CPGS", "description": "These were key sites identified for differences in DNA modifications between enterocytes and enterocyte-deficient jejunum tissues.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "CPG SITES 35"}, {"entity_type": "EXPRESSION PATTERN", "description": "Researchers observed that age-dependent changes in expression patterns were associated with epigenetic modifications at specific genomic regions like LCT and MCM6.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "id": "AGE-DEPENDENT EXPRESSION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Four selected SNPs for analysis in a follow-up cohort related to HLA carrier status.<SEP>The investigation into SNPs like C/T-13910 highlighted genetic variations that could influence the DNA modification patterns observed in different individuals or tissues.", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d<SEP>chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "SNPS (SINGLE NUCLEOTIDE POLYMORPHISMS)"}, {"entity_type": "GENOMIC REGION", "description": "A specific intronic region in the MCM6 gene, potentially affecting LCT regulation.<SEP>Changes at MCM6 intron 13 for haplotypes containing C-13910 must occur after conception.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c<SEP>chunk-ea45644d73f260cf365527dcc3af86e1", "id": "MCM6 INTRON 13"}, {"entity_type": "NUTRIENT/GENETIC MARKER", "description": "A SNP in this region indicates the presence of a specific genetic variant.", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "id": "C-13910"}, {"entity_type": "DIETARY PATTERN/EPIGENETIC FACTOR", "description": "The genotypes CT and TT influence DNA modifications at this region.", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 44}]", "id": "CT AND TT GENOTYPES"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The process of embryonic development where DNA modifications are erased.", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "id": "EMBRYOGENESIS"}, {"entity_type": "MOLECULAR INTERACTION/EVOLUTIONARY CONSERVATION", "description": "DNA modification densities change with age, suggesting an evolutionary factor.", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "id": "AGE-RELATED CHANGES"}, {"entity_type": "CELL LINE", "description": "A human intestinal epithelial cell line used for studying gene regulation.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "CACO-2"}, {"entity_type": "GENOMIC VARIANT", "description": "A specific genetic variant at the SNP -13910 location.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "SNP -13910"}, {"entity_type": "TRANSCRIPT", "description": "The transcript of an lncRNA (long non-coding RNA) with a specific gene identifier.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "LOC100507600"}, {"entity_type": "PROTEIN", "description": "The enzyme that breaks down lactose into glucose and galactose.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "LACTASE"}, {"entity_type": "TRANSCRIPTION FACTOR", "description": "A protein involved in binding to insulator sites on DNA which can regulate transcription.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "CTCF"}, {"entity_type": "MOLECULAR INTERACTION", "description": "DNA methylation of 450,000 CpG sites was determined in the same purified cells of 100 of the individuals to add explanations to observed differential expression between cell types.<SEP>The process of adding a methyl group to DNA, affecting gene expression through chromatin modification.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c<SEP>chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "DNA METHYLATION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A genome editing tool that uses RNA-guided Cas9 nuclease for precise deletion or addition of genetic material.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "CRISPR-CAS9 GENOME EDITING TECHNOLOGY"}, {"entity_type": "GENOMIC REGION", "description": "Specific areas of the DNA sequence studied, including introns and regulatory elements.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "DNA REGIONS"}, {"entity_type": "DISEASE", "description": "A condition in which lactose digestion ability is maintained into adulthood.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "LACTASE PERSISTENCE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Specific deletions induced by CRISPR-Cas9 technology to study gene functions.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "CRISPR-CAS9N-INDUCED DELETION"}, {"entity_type": "GENOMIC REGION", "description": "A specific intronic region in the Lactase gene, with evolutionary conservation.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "LCT INTRON 2"}, {"entity_type": "GENOMIC REGION", "description": "DNA regions that impact gene expression through modifications such as methylation and histone modification.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "EPIGENETICALLY-REGULATED REGIONS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Interactions between regulatory molecules, such as transcription factors or non-coding RNAs with specific DNA sequences.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "REGULATORY INTERACTIONS"}, {"entity_type": "CELLULAR PROCESS", "description": "The process of Caco-2 cells transforming into more specialized intestinal cell types after confluence.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "DIFFERENTIATION PROCESS"}, {"entity_type": "ANATOMICAL LOCATION", "description": "Different developmental stages or age groups in the context of gut physiology and gene expression.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "ADULT MICE/INFANT MICE"}, {"entity_type": "CELL TYPE", "description": "Cells derived from human intestine that are cultured for experimental studies.", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "id": "CULTURED HUMAN INTESTINAL EPITHELIAL CELLS"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "The LCT gene exhibits modifications at specific genomic regions contributing to cell-type specific expression and epigenetic regulation.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 44}]", "id": "LCT GENE"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "Significant DNA modification differences occur at CpG sites, linking them as important genetic variants in LCT\'s expression control.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT ASSOCIATION"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "Differences in DNA modifications highlight the cell-type specific regulation of LCT expression and DNA modifications.)", "entity_type": "UNKNOWN", "id": "CELL-TYPE SPECIFICITY"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "Differences in DNA modifications highlight the cell-type specific regulation of LCT expression and DNA modifications.)", "entity_type": "UNKNOWN", "id": "ENTEROCYTES AND JEJUNUM TISSUES"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "MCM6 gene shows similar regions to LCT\'s for DNA modification studies, highlighting the genetic linkage between these genes.)", "entity_type": "UNKNOWN", "id": "HAPLOTYPE ASSOCIATION"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "MCM6 gene shows similar regions to LCT\'s for DNA modification studies, highlighting the genetic linkage between these genes.)", "entity_type": "UNKNOWN", "id": "MCM6 GENE"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "The lncRNA is linked with LCT intron 5 region and plays a role in its regulation, showing interplay between different genomic elements.)", "entity_type": "UNKNOWN", "id": "FUNCTIONAL LINKAGE"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "The MCM6\'s 3\u2032-UTR is associated with chromatin features that indicate active regulatory functions like enhancer sites and histone marks, demonstrating the gene\'s role in regulation.)", "entity_type": "UNKNOWN", "id": "MCM6 GENE 3\u2032-UTR"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "The MCM6\'s 3\u2032-UTR is associated with chromatin features that indicate active regulatory functions like enhancer sites and histone marks, demonstrating the gene\'s role in regulation.)", "entity_type": "UNKNOWN", "id": "REGULATORY FUNCTION"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "The DNase I hypersensitivity sites at LCT introns 2 and 3, as well as MCM6\'s regions, indicate regulatory significance for gene expression.)", "entity_type": "UNKNOWN", "id": "REGULATORY SIGNIFICANCE"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "These histone marks are characteristic of active enhancers and play a role in gene regulation at the identified chromatin reference map sites.)", "entity_type": "UNKNOWN", "id": "ENHANCER ROLES"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "The LCT-MCM6 haplotypes exhibit strong associations with lactase persistence and non-persistence across genetic backgrounds, highlighting the role of these haplotypes in age-dependent regulation.)", "entity_type": "UNKNOWN", "id": "GENETIC LINKAGE"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "The LCT-MCM6 haplotypes exhibit strong associations with lactase persistence and non-persistence across genetic backgrounds, highlighting the role of these haplotypes in age-dependent regulation.)", "entity_type": "UNKNOWN", "id": "LCT-MCM6 HAPLOTYPES"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "Age-dependent epigenetic modifications occur at SNP C/T-13910, indicating temporal genomic responses to genetic variations over life stages.)(entity", "entity_type": "UNKNOWN", "id": "SNP C/T-13910 AND AGE CHANGES"}, {"source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "description": "Age-dependent epigenetic modifications occur at SNP C/T-13910, indicating temporal genomic responses to genetic variations over life stages.)(entity", "entity_type": "UNKNOWN", "id": "TEMPORAL GENOMIC RESPONSE"}, {"source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "description": "High conservation of the LCT intron 2 element is observed across mammals, including humans.)", "entity_type": "UNKNOWN", "id": "MAMMALS"}, {"source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "description": "The lncRNA regulates LCT mRNA levels, suggesting a potential mechanism for modulating enzyme expression.)", "entity_type": "UNKNOWN", "id": "LNCRNA LOC100507600"}, {"source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "description": "SNPs at this position influence the variation in haplotypes associated with lactase non-persistence or persistence.)", "entity_type": "UNKNOWN", "id": "HAPLOTYPES"}, {"source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "description": "SNPs at this position influence the variation in haplotypes associated with lactase non-persistence or persistence.)", "entity_type": "UNKNOWN", "id": "SNP-13910"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Increased DNA methylation inhibits CTCF binding at insulator sites, potentially activating gene expression.)", "entity_type": "UNKNOWN", "id": "CTCF BINDING"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "The deletion in MCM6 intron 13 did not alter Mcm6 mRNA levels when compared to the negative control.)", "entity_type": "UNKNOWN", "id": "MCM6 INTRON 13 DELETION"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Evolutionarily conserved LCT intron 2 establishes stronger control of LCT mRNA in adulthood.)", "entity_type": "UNKNOWN", "id": "AGE-SPECIFIC REGULATION"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Deletions affecting MCM6 intron 13 or LCT intron 2 resulted in significant decreases in LCT mRNA levels.)", "entity_type": "UNKNOWN", "id": "DOWNREGULATION/DECREASE IN LCT MRNA"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Deletions affecting MCM6 intron 13 or LCT intron 2 resulted in significant decreases in LCT mRNA levels.)", "entity_type": "UNKNOWN", "id": "MCM6 INTRON 13/LCT INTRON 2"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Deletions of epigenetically regulated DNA regions impacted human LCT expression, supporting their importance in regulation.)", "entity_type": "UNKNOWN", "id": "CRISPR-CAS9N-INDUCED DELETION/"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "The mutation in Lct intron 2 produced a significantly greater loss of LCT mRNA expression in adult mice compared to infants.)", "entity_type": "UNKNOWN", "id": "ADULT INTESTINE/INFANT GUT RESPONSE"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "The mutation in Lct intron 2 produced a significantly greater loss of LCT mRNA expression in adult mice compared to infants.)", "entity_type": "UNKNOWN", "id": "LACTASE PERSISTENCE/CRISPR-CAS9 DELETIONS"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Deletions affecting MCM6 or LCT regulatory regions resulted in a significant decrease in LCT mRNA levels in differentiated human intestinal epithelial cells.)", "entity_type": "UNKNOWN", "id": "MCM6 INTRON 13/LCT INTRON 2/"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Deletions affecting MCM6 or LCT regulatory regions resulted in a significant decrease in LCT mRNA levels in differentiated human intestinal epithelial cells.)", "entity_type": "UNKNOWN", "id": "REGULATORY IMPACT"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "The deletion effects on LCT mRNA varied between undifferentiated and differentiated states of cultured Caco-2 cells, supporting the importance of regulatory regions in human LCT expression.)", "entity_type": "UNKNOWN", "id": "CELLULAR STATE/EXPRESSION LEVELS"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "The deletion effects on LCT mRNA varied between undifferentiated and differentiated states of cultured Caco-2 cells, supporting the importance of regulatory regions in human LCT expression.)", "entity_type": "UNKNOWN", "id": "CRISPR-CAS9 DELETIONS/LCT EXPRESSION"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Deletions affecting MCM6 or LCT regulatory regions resulted in a significant decrease in LCT mRNA levels in differentiated human intestinal epithelial cells.)", "entity_type": "UNKNOWN", "id": "DECREASE IN LCT MRNA"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "Deletions affecting MCM6 or LCT regulatory regions resulted in a significant decrease in LCT mRNA levels in differentiated human intestinal epithelial cells.)", "entity_type": "UNKNOWN", "id": "MCM6/LCT REGULATORY REGIONS/"}, {"source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "description": "The deletion effects on LCT mRNA varied between undifferentiated and differentiated states of cultured Caco-2 cells, supporting the importance of regulatory regions in human LCT expression.)", "entity_type": "UNKNOWN", "id": "CRISPR-CAS9 DELETIONS/LCT EXPRESSION/"}, {"entity_type": "COMPARISON GROUP", "description": "Group of individuals without the specific condition or characteristic being compared to the subject group.<SEP>The study participants without genetic data are referred to as \'controls\'.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea", "id": "CONTROLS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Population frequency of the persistence allele at rs4988235 is 0.28 in Romania.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "ROMANIA"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The population frequency for \'T\' (persistence) allele at rs4988235 in Poland is observed to be 0.40.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "POLAND"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Frequency of the \'T\' (persistence) allele at rs4988235 for all populations was consistent with southern and eastern Europe.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "RUSSIA"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The frequency of lactase persistence driving allele is the highest in the Czech Republic, showing a relationship between genotype and milk consumption.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "CZECH REPUBLIC"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "\'C\' (non-persistence) allele was found at a frequency of 0.26 in the UK.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "UK"}, {"entity_type": "DIETARY PATTERN", "description": "Results include descriptive statistics and milk drinking patterns of all study participants.<SEP>The descriptive statistics and milk consumption patterns of study participants are presented.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "MILK DRINKING PROFILES"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Genotype rs4988235 was associated with a higher milk drinking rate in controls.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "OR 2.39"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Differences were observed in the consumption patterns of milk based on different rs4988235 genotypes.<SEP>The association between education and milk consumption was mostly observed in the Czech Republic.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "MILK CONSUMPTION"}, {"entity_type": "DISEASE", "description": "Risk of RCC (Renal Cell Carcinoma) was found to be elevated among those consuming milk compared to non-consumers.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "RCC"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A strong, observed relationship between milk consumption and cancer risk in the Czech Republic.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "OR 1.68"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Odds ratio of RCC was driven by milk consumption.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "RCC OR=1.35"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Educational attainment had an association with milk consumption across all countries.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "EDUCATION ATTAINMENT"}, {"entity_type": "GENOMIC VARIANT", "description": "Differences were not found between rs4988235 genotype and RCC risk either in analyses by country or as a whole.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "RS4988235 GENOTYPE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Overall odds of RCC did not show substantial differences based on genotypic groups.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "GENOTYPIC GROUP OR=1.01"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Variations in milk drinking habits were observed based on the genetic variations at rs4988235.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "MILK CONSUMPTION PATTERNS FOR DIFFERENT RS4988235 GENOTYPES"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The study found differences in milk intake between people with different lactase persistence genotypes.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "CONSUMPTION PATTERNS OF MILK"}, {"entity_type": "MOLECULAR INTERACTION", "description": "There was a relationship identified between milk consumption and RCC (Renal Cell Carcinoma) risk.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "MILK AND CANCER RISK ASSOCIATION"}, {"entity_type": "DISEASE", "description": "The genetic variant rs4988235 was linked with variations in milk drinking profiles.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "ASSOCIATION BETWEEN GENOTYPE RS4988235"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The RCC risk showed an elevated risk when consuming milk, which was mainly found in the Czech Republic.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "CANCER RISK ASSOCIATION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The study considered genetic data from individuals without genetic data to be \'controls\'.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "GENETIC DATA FOR CONTROLS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "There wasn\'t a consistent association found between genotype and milk consumption across all countries.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "MILK DRINKING TENDENCY ASSOCIATION"}, {"entity_type": "DISEASE", "description": "The analysis did not find significant differences in RCC risk based on the genotypes of rs4988235 within individual groups.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "RISK OF RCC BY GENOTYPIC GROUP OR=1.01"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Certain countries showed nominal but not systematic representation of relationships between variables like education and milk consumption.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "COUNTRY SPECIFIC DATA ASSOCIATIONS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Genetic variation at rs4988235 had a nominal evidence for association with educational attainment.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "GENOTYPE WITH EDUCATION ASSOCIATION"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The Czech Republic showed stronger associations between certain variables and milk consumption patterns compared to other countries.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "CZECH REPUBLIC STRONGEST ASSOCIATIONS"}, {"entity_type": "DISEASE", "description": "Differences in milk consumption patterns were associated with the risk of developing RCC.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "MILK CONSUMPTION PATTERN DIFFERENCES"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The study explored whether different genotypes related to lactase persistence had varying impacts on milk consumption and health outcomes.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "ASSOCIATION BETWEEN LACTASE PERSISTENCE GENOTYPES"}, {"entity_type": "DISEASE", "description": "Several factors could potentially have influenced the outcomes of the study, including education attainment.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "VARIABLES POTENTIALLY CONFOUNDING RESULTS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The Czech Republic showed significant associations between different variables and RCC risk when analyzing milk consumption patterns.", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "id": "CZECH REPUBLIC STRONGEST ASSOCIATIONS IN RCC RISK ANALYSIS"}, {"source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "description": "Differences were observed based on different rs4988235 genotypes related to milk consumption.)", "entity_type": "UNKNOWN", "id": "MILK CONSUMPTION PATTERNS"}, {"source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "description": "Strong relationship found specifically in the Czech Republic between milk consumption and RCC risk.)", "entity_type": "UNKNOWN", "id": "RCC OR=1.68"}, {"source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "description": "The strongest representation of associations between variables like education and smoking was found in the Czech Republic.)", "entity_type": "UNKNOWN", "id": "ASSOCIATIONS"}, {"entity_type": "TREATMENT", "description": "We observed statistically significant improvements in several hematological parameters following exposure to Hydroxyurea (HU) in 111 patients.", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "id": "HU TREATMENT"}, {"entity_type": "GENETIC VARIANT", "description": "This variant was not included in the analysis due to its very low frequency.", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "id": "SNP RS7482144"}, {"entity_type": "GENETIC VARIANT", "description": "One of the BCL11A SNPs associated with increased baseline and MTD HbF levels.", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "id": "RS1427407"}, {"entity_type": "GENETIC VARIANT", "description": "Another BCL11A SNP associated with increased baseline and MTD HbF levels, also showing association with delta HbF.", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "id": "RS4671393"}, {"entity_type": "HAPLOTYPE", "description": "Compared to the G-G-T reference, patients carrying this haplotype had higher baseline and MTD HbF levels.", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "id": "T-A-C HAPLOTYPE"}, {"entity_type": "HAPLOTYPE", "description": "The most frequent haplotype contained the wild-type alleles for both HMIP-2 SNPs, showing higher baseline HbF levels.", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "id": "T-G HAPLOTYPE"}, {"entity_type": "DISEASE", "description": "The text discusses associations of specific BCL11A SNPs and haplotypes with baseline, MTD HbF levels; however, it does not provide information about Delta HbF solely due to these genetic factors.<SEP>The text discusses associations of specific haplotypes with baseline and MTD HbF levels; however, these are not the same as discussing an interaction between a haplotype and Delta HbF.<SEP>The text discusses associations of the BCL11A SNPs with baseline and MTD HbF levels; however, it does not explicitly discuss their relationship to Delta HbF.<SEP>There was no association of any HMIP-2 haplotype with delta HbF, indicating that the SNPs did not influence changes in HbF levels.)<SEP>There was no statistically significant association between any HMIP-2 haplotype and changes in HbF levels (delta HbF), suggesting that the SNPs do not influence the rate of change of HbF in this context.", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce<SEP>chunk-3cf4e094a5312845a1cc4da4304d4bdf", "id": "DELTA HBF"}, {"entity_type": "GENOMIC VARIANT", "description": "The HMIP-2 SNP is associated with MTD HbF levels.", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "id": "HMIP-2 SNP"}, {"entity_type": "GENOMIC VARIANT", "description": "The C-A haplotype of the HMIP-2 SNP is related to MTD HbF levels, but loses significance after adjusting for baseline HbF.<SEP>The C-A haplotype of the HMIP-2 SNP loses statistical significance when adjusting for baseline HbF levels, indicating that its association with MTD HbF is not independent of pre-existing HbF concentration.", "source_id": "chunk-3ae06e9a3330573da28ca0a76aa685ce", "id": "C-A HAPLOTYPE"}, {"entity_type": "DISEASE", "description": "Fanconi anemia (FA) is being considered as a potential condition due to UBE2T deficiency.<SEP>Subject presented at birth with symptoms like bilateral radial aplasia, absent thumbs, microcephaly, micrognathia, caf\u00e9 au lait spots, and an absent left kidney.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f<SEP>chunk-ffd1276c89bcac745a7e58082e97236f", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 38}]", "id": "FA"}, {"entity_type": "GENE", "description": "Cellular phenotype of Fanconi anemia cell line of unknown complementation group was observed in this subject.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "UNKNOWN COMPLEMENTATION GROUP OF FANCONI ANEMIA"}, {"entity_type": "SYMPTOM", "description": "Subject presented with bilateral radial aplasia as a symptom.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "BILATERAL RADIAL APLASIA"}, {"entity_type": "SYMPTOM", "description": "Subject had absent thumbs as another symptom.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "ABSENT THUMBS"}, {"entity_type": "SYMPTOM", "description": "Microcephaly was observed in the subject, a condition characterized by an abnormally small head size.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "MICROCEPHALY"}, {"entity_type": "SYMPTOM", "description": "Micrognathia refers to a small jawbone or underdeveloped lower jaw of the subject.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "MICROGNATHIA"}, {"entity_type": "SYMPTOM", "description": "Subject had caf\u00e9 au lait spots, which are brownish-blue macules on the skin.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "CAF\u00c9 AU LAIT SPOTS"}, {"entity_type": "SYMPTOM", "description": "The absence of a left kidney was another symptom noted in the subject.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "ABSENT LEFT KIDNEY"}, {"entity_type": "SAMPLE TYPE", "description": "Peripheral blood samples were collected and used for various analyses.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "PERIPHERAL BLOOD SAMPLES"}, {"entity_type": "CROSSLINKING AGENT", "description": "RA2627 fibroblasts are hypersensitive to crosslinking agents like DEB in survival assays.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "DIEXPOXYBUTANE (DEB)"}, {"entity_type": "EVENT", "description": "Increased chromosomal breakage levels were observed in RA2627 fibroblasts compared to BJ wild type fibroblast and FA-A patient cells (RA3087).", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "CHROMOSOMAL BREAKAGE"}, {"entity_type": "ABNORMALITY", "description": "RA2627 cells are deficient for FANCD2 monoubiquitination, which is a hallmark of Fanconi anemia.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "FANCD2 MONOUBIQUITINATION DEFICIENCY"}, {"entity_type": "OBSERVATION", "description": "No FANCD2 foci were observed in RA2627 cells following 24-hour treatment with MMC.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "FANCD2 FOCUS ABSENCE"}, {"entity_type": "OBSERVATION", "description": "The subject\u2019s fibroblasts display a deficiency of FA pathway activation, consistent with the deficiency of the FA core complex or ID2 complex.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "FA PATHWAY ACTIVATION DEFICIENCY"}, {"entity_type": "MUTATION", "description": "A single FANCA mutation, c.2574C>G/p.Ser858Arg was identified previously in Fanconi anemia, but it did not lead to normal levels of FANCA and rescue the monoubiquitination defect of FANCD2.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 40}]", "id": "FANCA MUTATION"}, {"entity_type": "WHOLE EXOME SEQUENCING DATA ANALYSIS", "description": "Whole exome sequencing data analysis revealed a single FANCA mutation in the subject.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "WES DATA ANALYSIS"}, {"entity_type": "GENOMIC ANALYSIS METHOD", "description": "Analysis of high-resolution array comparative genomic hybridization did not detect deletions or duplications in known FA genes for this subject\u2019s fibroblasts and peripheral blood samples.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "HIGH RESOLUTION ARRAY COMPARATIVE GENOMIC HYBRIDIZATION (ACGH)"}, {"entity_type": "TRANSCRIPT LEVEL ABNORMALITY", "description": "RNA sequencing revealed a marked reduction in UBE2T transcript levels compared to a non-FA patient cell line (RA3380).", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "UBE2T DECREASED TRANSCRIPT LEVELS"}, {"entity_type": "VALIDATION METHOD", "description": "Decreased UBE2T transcript levels were confirmed by RT-qPCR analysis.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "RT-QPCR VALIDATION"}, {"entity_type": "PROTEIN LEVEL ABNORMALITY", "description": "No detectable UBE2T protein was found in RA2627 fibroblast lysates based on western blot analysis.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "UBE2T PROTEIN UNDETECTABLE"}, {"entity_type": "MUTATION COMBINATION", "description": "Sanger sequencing of genomic DNA and cDNA from the subject revealed compound heterozygous mutations in UBE2T. The paternally derived deletion, g.202332626_202341295del resulted from recombination of two AluYa5 repeats.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "COMPOUND HETEROZYGOUS MUTATIONS IN UBE2T"}, {"entity_type": "MUTATION ORIGIN", "description": "The maternally derived mutation consisted of a large duplication of the genomic region between two AluYa5 repeats.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "MATERNALLY DERIVED DUPLICATION"}, {"entity_type": "MUTANT TRANSCRIPT", "description": "Cloning of proband cDNA revealed a transcript containing the hypothesized duplication c.-64_468dup (dupEx2_6) that leads to frameshift and premature stop codon due to inclusion of non-coding region from exon 2.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "TRANSCRIPT WITH HAPLOID MUTATIONS"}, {"entity_type": "PATHOLOGY", "description": "The pathology analysis showed reduced hematopoietic stem cells in the bone marrow with dysplasia, indicative of bone marrow failure syndrome.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "BONE MARROW FAILURE SYNDROME"}, {"entity_type": "SYMPTOM", "description": "Pancytopenia was a symptom noted in this case, which included low counts of red blood cells, white blood cells, and platelets.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "PANCYTOPENIA"}, {"entity_type": "SYMPTOM", "description": "A skin rash with varying degrees of severity was another presented symptom.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "SKIN RASH"}, {"entity_type": "SYMPTOM", "description": "Enlarged lymph nodes were observed as part of the clinical presentation.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "LYMPH NODE ENLARGEMENT"}, {"entity_type": "PATHOLOGICAL FINDINGS", "description": "Reduction in numbers and quality of hematopoietic stem cells was found in bone marrow biopsies.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "HEMATOPOIETIC STEM CELL REDUCTION"}, {"entity_type": "PATHOLOGICAL FINDING", "description": "Morphological abnormalities indicating dysplasia were observed in the bone marrow samples.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "DYSPLASIA"}, {"entity_type": "DIAGNOSTIC METHOD", "description": "Whole-genome sequencing was performed to identify genetic mutations related to the syndrome.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "GENOMIC SEQUENCING"}, {"entity_type": "GENETIC MUTATION", "description": "A novel mutation, c.823G>T/p.Val275Met in CIRH1A gene was identified through genomic analysis.", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "id": "CIRH1A MUTATION"}, {"entity_type": "PROTEIN", "description": "The expression of wild type UBE2T gene is being considered as a potential cause for the observed phenotypes.<SEP>Ube2T interacts with the RING domain of FANCL but not directly with the URD according to preliminary size-exclusion chromatography analysis.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f<SEP>chunk-e097891654194982dfc0ef87401d3aa9", "id": "UBE2T"}, {"entity_type": "TRANSCRIPT", "description": "Wild type UBE2T transcript has been introduced into RA2627 fibroblasts.<SEP>Wild type UBE2T transcript has been introduced into the proband blood sample.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "id": "WILD TYPE UBE2T"}, {"entity_type": "CELLULAR", "description": "RA2627 cells are used in the study as they show certain cellular defects associated with FA when UBE2T expression is deficient.<SEP>RA2627 cellular defects are associated with the FA phenotype.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "id": "RA2627 FIBROBLASTS"}, {"entity_type": "CHEMICAL COMPOUND", "description": "MMC, DEB and cisplatin are chemical agents used in assays to test UBE2T\'s effect on the cell.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "id": "MMC"}, {"entity_type": "PHENOMENON", "description": "RA2627 cells show hypersensitivity to crosslinking agents like MMC when UBE2T is deficient.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "id": "CELLULAR HYPERSENSITIVITY"}, {"entity_type": "CELL CYCLE", "description": "Accumulation of RA2627 cells in G2 phase was observed following treatment with MMC and was rescued by UBE2T overexpression.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "id": "G2 PHASE ACCUMULATION"}, {"entity_type": "MOLECULAR ACTIVITY", "description": "Monoubiquitination of FANCD2 and FANCI proteins plays a role in FA phenotypes and is restored with UBE2T expression.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "id": "FANCD2 AND FANCI MONOUBIQUITINATION"}, {"entity_type": "CELLULAR STRUCTURE", "description": "Formation of FANCD2 foci occurs after treatment with MMC, which is crucial for the FA phenotype.<SEP>Formation of FANCD2 foci was observed after treatment with MMC, which was rescued by UBE2T expression.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "id": "FANCD2 FOCI"}, {"entity_type": "CHEMICAL COMPOUND", "description": "These chemical agents were used to test the effects of UBE2T overexpression on RA2627 cells and their resistance to crosslinking agents.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "id": "MMC, DEB AND CISPLATIN"}, {"entity_type": "CELL CYCLE", "description": "Accumulation of RA2627 cells in G1 phase was observed following treatment with MMC and was not rescued by UBE2T overexpression, indicating cell cycle regulation is altered.", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "id": "G1 PHASE ACCUMULATION"}, {"source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "description": "Decreased transcript levels of the UBE2T gene were detected, which is a consequence of mutations identified in this subject.)", "entity_type": "UNKNOWN", "id": "CONSEQUENCE OF MUTATION IN UBE2T"}, {"source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "description": "The combination of maternally derived duplication and paternally derived deletion resulted in genetic reversion phenomena observed in the proband\'s LCLs, supporting normal UBE2T expression.)", "entity_type": "UNKNOWN", "id": "COMPOUND HETEROZYGOUS MUTATIONS"}, {"source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "description": "The combination of maternally derived duplication and paternally derived deletion resulted in genetic reversion phenomena observed in the proband\'s LCLs, supporting normal UBE2T expression.)", "entity_type": "UNKNOWN", "id": "GENETIC REVERSION PHENOMENON"}, {"source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "description": "Wild type UBE2T overexpression was able to rescue cellular defects associated with Fanconi anemia in RA2627 fibroblasts, demonstrating the causality of UBE2T deficiency.)", "entity_type": "UNKNOWN", "id": "CELLULAR PHENOTYPE RESCUED BY UBE2T"}, {"source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "description": "Wild type UBE2T overexpression was able to rescue cellular defects associated with Fanconi anemia in RA2627 fibroblasts, demonstrating the causality of UBE2T deficiency.)", "entity_type": "UNKNOWN", "id": "OVEREXPRESSION OF WILD TYPE UBE2T"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "Overexpression of wild type UBE2T gene rescues FA phenotypes in RA2627 fibroblasts.)", "entity_type": "UNKNOWN", "id": "OVEREXPRESSION"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "Insufficient expression or mutation in UBE2T leads to Fanconi anemia.)", "entity_type": "UNKNOWN", "id": "FANCONI ANEMIA (FA)"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "Insufficient expression or mutation in UBE2T leads to Fanconi anemia.)", "entity_type": "UNKNOWN", "id": "UBE2T DEFICIENCY"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "The expression of wild type UBE2T restores the monoubiquitination status of FANCD2 and FANCI proteins following FA phenotype induction by treatment with MMC.)", "entity_type": "UNKNOWN", "id": "FANCD2 MONOUBIQUITINATION"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "UBE2T overexpression rescues the formation of FANCD2 foci observed after treatment with MMC, associated with FA phenotype rescue.)", "entity_type": "UNKNOWN", "id": "FANCD2 FOCI FORMATION"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "The overexpression of wild type UBE2T rescues cellular defects associated with Fanconi Anemia (FA) in RA2627 fibroblasts, as shown by restored cell cycle distribution and FANCD2 foci formation.)", "entity_type": "UNKNOWN", "id": "FA PHENOTYPE RESCUE"}, {"source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "description": "The overexpression of wild type UBE2T rescues cellular defects associated with Fanconi Anemia (FA) in RA2627 fibroblasts, as shown by restored cell cycle distribution and FANCD2 foci formation.)", "entity_type": "UNKNOWN", "id": "WILD TYPE UBE2T OVEREXPRESSION"}, {"entity_type": "PATIENT SAMPLE", "description": "Results from gene expression profiling study on cells from DBA patients using Affymetrix GeneChip Human Genome U133A 2.0 Arrays.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "DBA PATIENTS"}, {"entity_type": "MICROARRAY", "description": "The microarrays used for the genome-wide expression profiling of DBA patient\'s dermal fibroblasts.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "AFFYMETRIX GENECHIP"}, {"entity_type": "CELL TYPE", "description": "Cells obtained from DBA patients for gene expression analysis.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "DERMAL FIBROBLASTS"}, {"entity_type": "GENE", "description": "The focus gene identified in the study, which has mutations linked to the DBA condition.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "RPS19"}, {"entity_type": "GENE", "description": "One of several genes involved in aminoacyl-tRNA biosynthesis.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "LARS"}, {"entity_type": "GENE", "description": "Another gene related to aminoacyl-tRNA biosynthesis.<SEP>WARS (Valine tRNA Synthetase) plays a role in amino acid metabolism and was identified as differentially expressed in the study.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94<SEP>chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "WARS"}, {"entity_type": "GENE", "description": "Part of the group of genes associated with metabolic pathways like glycine, serine, and threonine metabolism.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "GARS"}, {"entity_type": "GENE", "description": "Also involved in the same set of metabolic processes.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "SARS"}, {"entity_type": "GENE", "description": "One of multiple enzymes related to aminoacyl-tRNA biosynthesis.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "QARS"}, {"entity_type": "GENE", "description": "Involved in a variety of biological pathways and functions.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "EPRS"}, {"entity_type": "GENE", "description": "Associated with glycine, serine, and threonine metabolism.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "PSAT1"}, {"entity_type": "GENE", "description": "An enzyme involved in glycolysis and gluconeogenesis.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "PHGDH"}, {"entity_type": "GENE", "description": "Part of the set of genes related to aminoacyl-tRNA biosynthesis and metabolic processes.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "CTH"}, {"entity_type": "GENE", "description": "Related to the regulation or processes within the cell, potentially involved in metabolism.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "SHMT2"}, {"entity_type": "GENE", "description": "Another key gene involved in cell death processes, specifically apoptosis.<SEP>Involved in apoptosis signaling pathway, specifically death receptor signaling.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "CASP9"}, {"entity_type": "GENE", "description": "A component of the nuclear factor kappa-B (NF-\u03baB) complex.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "NFKBIA"}, {"entity_type": "GENE", "description": "Another component of NF-\u03baB involved in inflammation and immune response.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "NFKBIE"}, {"entity_type": "GENE", "description": "Involved in apoptosis regulation, particularly through the caspase cascade.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "APAF1"}, {"entity_type": "GENE", "description": "Part of the complex that regulates the NF-\u03baB pathway.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "NFKB2"}, {"entity_type": "GENE", "description": "A critical component of the NF-\u03baB signaling pathways.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "NFKB1"}, {"entity_type": "GENE", "description": "Involved in apoptosis regulation and anti-apoptosis mechanisms.", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "id": "BIRC3"}, {"entity_type": "GENE", "description": "RPL22 is a gene involved in protein synthesis. It is part of the large cluster of RP genes that includes several other ribosomal proteins like RPL27A, RPL29, and others.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL22"}, {"entity_type": "GENE", "description": "RPS3 is another gene from the same cluster as RPL22, involved in protein synthesis and cell development.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPS3"}, {"entity_type": "GENE", "description": "RPL18 is part of a large cluster of RP genes that play roles in cellular development, lipid metabolism, and molecular transport.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL18"}, {"entity_type": "GENE", "description": "RPS2 belongs to the same cluster of genes involved in protein synthesis and other cellular functions.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPS2"}, {"entity_type": "DISEASE CATEGORY", "description": "A category that includes several cancer-related genes has been identified through this study along with other disease categories.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "CANCER RELATED GENES"}, {"entity_type": "DISORDER CATEGORY", "description": "The study found clusters of genes involved in haematological disorders, among others.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "HAEMATOLOGICAL DISORDERS"}, {"entity_type": "DISORDER CATEGORY", "description": "Genes related to immunological disorders are part of the identified disease categories.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "IMMUNOLOGICAL DISORDERS"}, {"entity_type": "DISORDER CATEGORY", "description": "A cluster of genes associated with renal disorders has been found.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RENAL DISORDERS"}, {"entity_type": "DISORDER CATEGORY", "description": "Genes involved in the development and function of skeletal tissues are part of this study\'s findings.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "SKELETAL DISORDERS"}, {"entity_type": "DEVELOPMENTAL FUNCTION", "description": "The cluster includes genes related to embryonic development processes.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "EMBRYONIC DEVELOPMENT"}, {"entity_type": "VALIDATION METHOD", "description": "Validation was performed by real-time RT-PCR on microarray data for selected genes. This method showed a high degree of correlation with the initial microarray results.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "GENE EXPRESSION VALIDATION"}, {"entity_type": "QUANTITATIVE TECHNIQUE", "description": "Microarrays were used to measure gene expression levels and identify differentially expressed genes in DBA patients compared to controls.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "MICROARRAY DATA"}, {"entity_type": "VALIDATION TECHNIQUE", "description": "Quantitative real-time PCR was utilized as a validation method for microarray data, demonstrating consistency in gene expression results.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "QRT-PCR"}, {"entity_type": "COMPARISON RESULT", "description": "A correlation coefficient of 0.7 was found between the two methods, suggesting a strong agreement.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "CORRELATION BETWEEN MICROARRAY AND QRT-PCR"}, {"entity_type": "ANALYSIS OUTCOME", "description": "The microarray data showed consistent trends with qRT-PCR results when comparing fold changes for different genes in each sample.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "FOLD CHANGE ANALYSIS"}, {"entity_type": "GENE", "description": "RPL27A is a ribosomal protein gene involved in protein synthesis and cell development.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL27A"}, {"entity_type": "GENE", "description": "RPL29 is part of the large cluster of RP genes, contributing to functions like cellular development and lipid metabolism.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL29"}, {"entity_type": "GENE", "description": "RPL31 plays a role in protein synthesis with contributions to metabolic pathways related to cell growth and maintenance.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL31"}, {"entity_type": "GENE", "description": "RPL14 is part of the cluster of ribosomal proteins known for their involvement in cellular processes including development and metabolism.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL14"}, {"entity_type": "GENE", "description": "RPL18A contributes to protein synthesis alongside other RP genes, particularly impacting lipid metabolism.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL18A"}, {"entity_type": "GENE", "description": "RPL28 is a ribosomal protein that helps in protein synthesis and may have roles in cellular development and function.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "RPL28"}, {"entity_type": "GENE", "description": "AMPD3 gene expression was found to be up-regulated in DBA patients, suggesting potential involvement in metabolic pathways or disease mechanisms.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "AMPD3"}, {"entity_type": "GENE", "description": "CCND2 (Cyclin D2) is a cancer-related gene involved in cell cycle regulation and has been detected as differentially expressed.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "CCND2"}, {"entity_type": "GENE", "description": "SOD2, or Superoxide Dismutase 2, is a key enzyme involved in antioxidant defense and was identified with increased expression in DBA patients.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "SOD2"}, {"entity_type": "GENE", "description": "Tumor Necrosis Factor Alpha-Inducible Protein 3 (TNFAIP3) is an immunological gene found to be up-regulated in DBA patients.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "TNFAIP3"}, {"entity_type": "GENE", "description": "Comp, or Colloidosomal Matrix Protein, was down-regulated in DBA patients, potentially impacting tissue development and matrix function.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "COMP"}, {"entity_type": "GENE", "description": "ZIC1, or Zeste-Waist Homolog 1, is involved in embryonic development and was found to be down-regulated in DBA patients.", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "id": "ZIC1"}, {"source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "description": "Differential expression was identified for specific genes associated with DBA patients compared to controls.)", "entity_type": "UNKNOWN", "id": "GENE EXPRESSION DIFFERENCES"}, {"source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "description": "Differential expression was identified for specific genes associated with DBA patients compared to controls.)", "entity_type": "UNKNOWN", "id": "OUTCOME OF STUDY"}, {"source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "description": "The microarray data was validated using qRT-PCR, providing a cross-check on gene expression levels.)", "entity_type": "UNKNOWN", "id": "METHOD COMPARISON"}, {"source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "description": "The microarray data was validated using qRT-PCR, providing a cross-check on gene expression levels.)", "entity_type": "UNKNOWN", "id": "MICROARRAY DATA VALIDATION"}, {"source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "description": "Both methods showed consistent trends for differentially expressed genes across the samples.)", "entity_type": "UNKNOWN", "id": "OUTCOME OF CROSS-VALIDATION"}, {"source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "description": "Both methods showed consistent trends for differentially expressed genes across the samples.)", "entity_type": "UNKNOWN", "id": "QRT-PCR AND MICROARRAY RESULTS AGREEMENT"}, {"entity_type": "GENOMIC VARIANT", "description": "The BDNF rs6265 polymorphism analysis of the gene is included in studies on migraine.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "BDNF RS6265"}, {"entity_type": "PROTEIN", "description": "The BDNF rs6265 (G/A) polymorphism refers to a protein variant associated with migraine.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "RS6265 G/A"}, {"entity_type": "DISEASE", "description": "The focus of the studies includes assessing the association between specific genotypes and the development of Migraine headaches.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "MIGRAINE"}, {"entity_type": "GENOMIC VARIANT", "description": "The BDNF rs2049046 polymorphism assessment is part of migraine studies.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "BDNF RS2049046"}, {"entity_type": "PROTEIN", "description": "The BDNF rs2049046 (A/T) polymorphism refers to a protein variant in association with Migraine.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "RS2049046 A/T"}, {"entity_type": "MOLECULAR INTERACTION", "description": "All SNPs included studies were in Hardy-Weinberg equilibrium, implying the frequency of genotypes is consistent with theoretical expectations based on allele frequencies.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "HARDY-WEINBERG EQUILIBRIUM (HWE)"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The NOS score indicates quality of the studies, suggesting good performance in terms of study design and analysis.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "NOS SCORE"}, {"entity_type": "NUTRIENT", "description": "One of the nucleobases found in DNA, the G allele in BDNF rs6265.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "GUANINE"}, {"entity_type": "NUTRIENT", "description": "One of the nucleobases found in DNA, the T allele in BDNF rs2049046.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "THYMINE"}, {"entity_type": "POPULATION", "description": "Refers to groups from Western European ancestry that were included in studies on migraine.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "CAUCASIAN POPULATION GROUPS"}, {"entity_type": "STUDY DESIGN", "description": "Indicates the use of Australian populations for certain analyses or data collection.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "AUSTRALIAN-BASED COHORTS"}, {"entity_type": "RELATIONSHIPS", "description": "The interaction between single nucleotide polymorphisms (SNPs) in BDNF gene and migraine development.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "MOLECULAR INTERACTION BETWEEN SNPS AND MIGRAINE"}, {"entity_type": "STATISTICAL METHOD", "description": "A method used for meta-analysis to combine results of different studies when there is little heterogeneity.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "FIXED-EFFECTS MODEL"}, {"entity_type": "VARIABILITY", "description": "The degree of difference among the studies included in the meta-analysis; no significant differences were found.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "HETEROGENEITY DIFFERENCE"}, {"entity_type": "STATISTICAL TESTS", "description": "Used in meta-analysis to assess publication bias by examining funnel plot asymmetry.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "BEGG\u2019S TEST"}, {"entity_type": "STATISTICAL TESTS", "description": "Another method for detecting publication bias by analyzing the relationship between effect size and precision of studies.", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "id": "EGGER\u2019S TEST"}, {"source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "description": "The criterion for selecting studies which were conducted on individuals from a specific geographical region and ancestry group.)", "entity_type": "UNKNOWN", "id": "STUDY SELECTION CRITERIA"}, {"source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "description": "A component of the study design involving cohorts or participants from Australia.)", "entity_type": "UNKNOWN", "id": "STUDY COMPONENT"}, {"source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "description": "The process by which specific genetic variations affect the risk or development of migraine headaches.)", "entity_type": "UNKNOWN", "id": "BIOLOGICAL PROCESS"}, {"source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "description": "A statistical model used to analyze combined data from multiple studies with minimal heterogeneity.)", "entity_type": "UNKNOWN", "id": "METHODOLOGY IN META-ANALYSIS"}, {"source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "description": "The comparison of variance among different studies contributing to the meta-analysis when no significant differences are present.)", "entity_type": "UNKNOWN", "id": "STATISTICAL ANALYSIS IN META-ANALYSIS"}, {"source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "description": "An analysis that tests how sensitive results are to individual data point inclusion or exclusion, often used in meta-analyses.)", "entity_type": "UNKNOWN", "id": "ROBUSTNESS CHECK"}, {"source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "description": "A statistical test utilized for identifying potential bias in publication, specifically by examining asymmetry in funnel plots.)", "entity_type": "UNKNOWN", "id": "DETECTION OF PUBLICATION BIAS"}, {"source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "description": "An approach used to identify publication bias through the correlation between effect size and precision across studies.)", "entity_type": "UNKNOWN", "id": "BIAS DETECTION METHOD"}, {"entity_type": "DISEASE", "description": "Severe hidradenitis suppurativa (HS) is a chronic inflammatory skin condition characterized by recurring abscesses and\u7aa6\u9053 in affected individuals.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "HS"}, {"entity_type": "DISEASE", "description": "Paradoxical psoriasis refers to the development of psoriasiform reactions as a response to certain treatments, such as adalimumab.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "PSORIASIS"}, {"entity_type": "DRUG", "description": "Anti-TNF-\u03b1 therapy (adalimumab) is used for managing inflammatory diseases like HS and psoriasis; however, paradoxical reactions can occur following its administration.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "ADALIMUMAB"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Hurley III scale is a grading system used to evaluate the severity of hidradenitis suppurativa based on clinical features like abscesses, sinus tracts, and inflammation.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "HURLEY III SCALE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Sartorius score measures the severity of hidradenitis suppurativa by assessing clinical signs such as abscesses,\u7aa6\u9053,\u7aa6\u7ba1\u708e, and erythema.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "SARTORIUS SCORE"}, {"entity_type": "DISEASE", "description": "Psoriasis Area and Severity Index (PASI) is a measure used to quantify the severity of psoriasis on different body parts, aiding in treatment evaluation.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "PASI"}, {"entity_type": "PATHOLOGICAL CONDITION", "description": "Characterized by redness (erythema) and scaling on affected skin areas.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "ERYTHEMATO-SCALY LESIONS"}, {"entity_type": "PATHOLOGICAL CONDITION", "description": "Bullae-like formations filled with pus, often seen in various skin conditions including psoriasis.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "PUSTULAR LESIONS"}, {"entity_type": "PATHOLOGICAL FEATURE", "description": "Solid, usually non-tender lumps found beneath the skin.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "NODULES"}, {"entity_type": "PATHOLOGICAL STRUCTURE", "description": "Abnormal openings connecting cavities or passages within tissues or organs.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "FISTULAS AND SINUS TRACTS"}, {"entity_type": "DISEASE", "description": "An autoimmune condition characterized by hair loss on the scalp, which can also occur in other parts of the body.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "ALOPECIA AREATA"}, {"entity_type": "PATHOLOGICAL CONDITION", "description": "Skin lesions resembling eczema, characterized by redness and itching, often with varying degrees of scaling or oozing.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "ECZEMATOUS-LIKE SKIN LESIONS"}, {"entity_type": "CELL TYPE", "description": "Neutrophils are part of the innate immune system and can play a role in inflammation and tissue damage during skin reactions.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "CD15+ NEUTROPHILS"}, {"entity_type": "CELL TYPE", "description": "Plasmacytoid dendritic cells (pDC) are a type of myeloid cell found in the skin, with roles in immune responses.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "BDCA2+ PDCS"}, {"entity_type": "CELL TYPE", "description": "Mast cells are involved in allergic reactions and play a role in inflammation through the release of various mediators.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "C-KIT/CD117+ MAST CELLS"}, {"entity_type": "CELL TYPE", "description": "Macrophages are phagocytic cells that help remove debris, pathogens, and other substances from the body; they can be involved in inflammatory responses.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "CD68+ MACROPHAGES"}, {"entity_type": "CELL TYPE", "description": "A type of white blood cell that circulates in the bloodstream and migrates to sites of inflammation, where it matures into macrophages.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "MONOCYTES"}, {"entity_type": "CELL TYPE", "description": "CD3+ cells from skin biopsies of HS patients were enriched in CD8+, but not in CD4+ cells.<SEP>T cells are a class of lymphocytes involved in adaptive immunity; they play various roles depending on their subtype (e.g., CD4+ T-helper cells, CD8+ cytotoxic T-cells).", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c<SEP>chunk-56c37d6045382dda8cc6037b2680282a", "id": "CD3+ CELLS"}, {"entity_type": "CELL TYPE", "description": "Dendritic cells (DCs) are antigen-presenting cells that activate the immune response; BDCA2+ refers to a specific subset with certain functions.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "BDCA2+ DCS"}, {"entity_type": "CYTOKINE", "description": "Interleukin-36\u03b3 is a pro-inflammatory cytokine that can be produced by neutrophils, contributing to immune responses and potentially promoting skin disorders.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "IL-36\u0393"}, {"entity_type": "RECOGNITION", "description": "CD8+ T cells also participated in the analysis of differential expressions and eQTLs related to the CTLA4 gene.<SEP>CD8+ T cells are cytotoxic and play roles in immune responses against viruses, cancer, and other pathogens; they may also contribute to skin inflammation.<SEP>No significant trans-eQTL effects were found in this region for CD8+ T cells.<SEP>The frequencies of recognizing neoantigens are nominal, comparable to known immunodominant epitopes.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b<SEP>chunk-d3a9c93928447522ce3df4209f8ac803<SEP>chunk-1b40d071abac0ba432a9fb0096796f45<SEP>chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "CD8+ T CELLS"}, {"entity_type": "BIOLOGICAL SAMPLE", "description": "Peripheral Blood Mononuclear Cells are a type of blood cell found in the bloodstream that includes lymphocytes and monocytes.<SEP>Peripheral Blood Mononuclear Cells are a type of blood cell found in the bloodstream that includes lymphocytes and monocytes.)(entity", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "PBMCS"}, {"entity_type": "CELL TYPE", "description": "T-regulatory cells (Tregs) play an important role in immune tolerance by suppressing inflammatory responses, particularly in autoimmune diseases like HS and psoriasis.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "TREG CELLS"}, {"entity_type": "SUBTYPE OF CELL TYPE", "description": "Foxp3 is a transcription factor that is essential for the development and function of regulatory T cells (Tregs), which help maintain immune tolerance and prevent autoimmunity.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "FOXP3+ TREGS"}, {"entity_type": "CLINICAL CONCEPT", "description": "Identifying HS or psoriasis from other inflammatory skin conditions requires a thorough differential diagnosis process based on symptoms, clinical features, and patient history.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "DIFFERENTIAL DIAGNOSIS"}, {"entity_type": "PATHOLOGY CONCEPT", "description": "Histopathological examination of skin biopsies is crucial for confirming the diagnosis of HS and distinguishing it from similar disorders with overlapping symptoms.", "source_id": "chunk-0380fd029b3eda680c87c1b8bcbe7f4c", "id": "HISTOPATHOLOGICAL FEATURES"}, {"entity_type": "CELL TYPE", "description": "A mixed population of T cells, responsible for either the psoriasiform or eczematous reactions.<SEP>Cells targeted by IL-27 and its receptor to induce a STAT1/STAT3 signaling pathway.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a<SEP>chunk-1b40d071abac0ba432a9fb0096796f45", "id": "T CELLS"}, {"entity_type": "DISEASE", "description": "CD3+ skin T cells of classical psoriasis showed instead an enrichment of both CD4+ and CD8+ subpopulations.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "CLASSICAL PSORIASIS"}, {"entity_type": "CYTOKINE", "description": "The type I IFN-\u03b2 was expressed in paradoxical skin lesions, mainly in keratinocytes.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "TYPE I IFN-\u0392"}, {"entity_type": "CYTOKINE", "description": "The epidermis of psoriasiform reactions was also immunoreactive for IFN-\u03ba, another keratinocyte-derived type I IFN.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "IFN-\u039a"}, {"entity_type": "PROTEIN", "description": "We finally investigated LT-\u03b1 and LT-\u03b2, two members of the cytokine TNF family, also known as TNF-\u03b2 and TNF-C 35, possibly deregulated by anti-TNF-\u03b1 therapy.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "LL-37"}, {"entity_type": "PROTEIN", "description": "Both lymphotoxins were strongly overexpressed in paradoxical skin reactions, especially in keratinocytes of the basal layer epidermis.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "LYMPHOTOXINS (LT-\u0391 AND LT-\u0392)"}, {"entity_type": "GENE", "description": "Among them, we studied SNPs frequent in the psoriatic population, such as polymorphisms of the HLA-C and HLA-B regions and the ERAP1 gene.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "ERAP1"}, {"entity_type": "LOCATION", "description": "We also analyzed genetic variants of pathogenic cytokines, receptors and signal transducers (i.e. TNF-\u03b1, IL-17F, IL-17RA, IL-23R, IL-12B, IKB\u03b6, A20, A20 binding protein, Tyk2), and of skin-barrier proteins (i.e. CDSN, CCHCR1)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "HLA-C REGION"}, {"entity_type": "SNP", "description": "Three SNPs in ERAP1 (rs30187/rs30186/rs26653) and nine variants in HLA-C were found in all patients.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "ERAP1 SNPS"}, {"entity_type": "GENE", "description": "Two other SNPs in CDSN (rs1062470/rs707913) and three SNPs in CCHCR1 (rs1576/rs130079/rs746647) were carried by patient 3.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "CDSN"}, {"entity_type": "BIOLOGICAL MOLECULE", "description": "In patients with paradoxical psoriasis, the presence of autoantibodies against TNF-\u03b1 or other cytokines was detected.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "ANTIBODY"}, {"entity_type": "PHARMACEUTICAL INTERVENTION", "description": "The overexpression of innate immunity molecules and genetic variants in HS patients might suggest a potential mechanism for paradoxical psoriasis to some anti-TNF-\u03b1 therapies.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "ANTI-TNF-\u0391 THERAPY"}, {"entity_type": "LOCATION", "description": "A SNP (rs11209026) in the IL23R gene has been previously associated with paradoxical psoriasiform reactions to anti-TNFs.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "TNF-\u0391 GENE"}, {"entity_type": "SNP", "description": "We could not find this SNP (rs11209026) in any of the HS patients, but two other SNPs (rs72676067 and rs1004819) were instead detected in patients 2 and 1.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "IL23R SNPS"}, {"entity_type": "GENE", "description": "Genetic variants of pathogenic cytokines, such as TNF-\u03b1, IL-17F, IL-17RA, and IL-23R were analyzed.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "TNF-\u0391, IL-17F, IL-17RA, IL-23R GENES"}, {"entity_type": "GENE", "description": "SNPs in the skin-barrier protein CDSN were also investigated.", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "id": "SKIN-BARRIER PROTEIN CDSN GENE"}, {"entity_type": "GENE", "description": "SNPs (rs1576, rs130079, rs746647) in this gene are mentioned.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "CCHCR1"}, {"entity_type": "GENE", "description": "Previous studies have associated SNPs in this gene with paradoxical psoriasiform reactions to anti-TNFs and it was found in the provided dataset as well.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "IL23R"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP identified in the IL23R gene that has been linked with certain conditions before but could not be found among HS patients.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "RS11209026"}, {"entity_type": "GENOMIC VARIANT", "description": "This SNP was detected instead of rs11209026 in patient 2, showcasing genetic diversity.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "RS72676067"}, {"entity_type": "GENOMIC VARIANT", "description": "Another variant found specifically in one of the patients, indicating individual differences.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "RS1004819"}, {"entity_type": "DISEASE", "description": "This term suggests a condition related to skin disorders that can occur as an adverse reaction to certain treatments.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "PARADOXICAL PSORIASIFORM REACTIONS"}, {"entity_type": "DRUG", "description": "A group of drugs used for treating inflammatory conditions, which may sometimes result in paradoxical psoriasiform reactions.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "ANTI-TNFS"}, {"entity_type": "SUBJECT GROUP", "description": "Highly specific group of individuals under study whose genetic profiles were compared to others with different SNPs.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "HS PATIENTS"}, {"entity_type": "RS130079/", "description": "rs746647)", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "SNP (RS1576/"}, {"entity_type": "GENE", "description": "The gene IL23R is mentioned and studied through its different SNPs (rs72676067, rs1004819), highlighting its role or associations in the context.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "IL23R"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant was mentioned alongside other SNPs associated with CCHCR1.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "RS130079"}, {"entity_type": "GENOMIC VARIANT", "description": "Another SNP in the same gene, CCHCR1, as previously identified.", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "id": "RS746647"}, {"source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "description": "The type I IFN-\u03b2 was expressed in paradoxical skin lesions, mainly in keratinocytes.)", "entity_type": "UNKNOWN", "id": "PARADOXICAL PSORIASIS"}, {"source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "description": "The epidermis of psoriasiform reactions was also immunoreactive for IFN-\u03ba, another keratinocyte-derived type I IFN.)", "entity_type": "UNKNOWN", "id": "PSORIASIFORM REACTIONS"}, {"source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "description": "Two other SNPs in CDSN and three SNPs in CCHCR1 were carried by patient 3.)", "entity_type": "UNKNOWN", "id": "CDSN SNPS"}, {"source_id": "chunk-72f4505e43f67f3564607392cb178656", "description": "rs1004819", "entity_type": "UNKNOWN", "id": "RS72676067/"}, {"source_id": "chunk-72f4505e43f67f3564607392cb178656", "description": "The term indicates possible outcomes related to specific SNPs or genes involved, such as adverse effects of certain treatments.)", "entity_type": "UNKNOWN", "id": "DISEASE/OUTCOME"}, {"source_id": "chunk-72f4505e43f67f3564607392cb178656", "description": "Connection to the previously mentioned conditions when discussing alternative treatments or outcomes.)<SEP>The class of drugs is mentioned in relation to potential side effects observed in the context of genetic variations.)", "entity_type": "UNKNOWN", "id": "THERAPY/DRUG"}, {"source_id": "chunk-72f4505e43f67f3564607392cb178656", "description": "A defined group of subjects whose data contributes to the comparative analysis.)<SEP>Focus on genetic analysis involving a specific group of subjects, in this context related to SNPs and potential health outcomes.)", "entity_type": "UNKNOWN", "id": "SUBJECT GROUP/SAMPLE"}, {"source_id": "chunk-72f4505e43f67f3564607392cb178656", "description": "Relevance to certain SNPs and potential side effects of treatment, highlighting risks in genetic profiling.)", "entity_type": "UNKNOWN", "id": "OUTCOME/ADVERSE EFFECT"}, {"entity_type": "GENE", "description": "FANCL is a gene that plays an important role in nucleotide excision repair and is composed of two lobes with four \u03b1-helices and nine \u03b2-strands.", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "FANCL"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The DRWD domain consists of both N-terminal and C-terminal lobes, each having a four-stranded \u03b2-meander followed by an \u03b1-helix in the terminal lobe.", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "DRWD DOMAIN"}, {"entity_type": "GENOMIC VARIANT/", "description": "DmFANCL is a gene that corresponds to Drosophila FANCL, which plays a role in DNA repair processes.<SEP>Transcript/", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8<SEP>chunk-e097891654194982dfc0ef87401d3aa9", "id": "DMFANCL"}, {"entity_type": "\u0392-STRANDS/", "description": "\u03b2-Meanders", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "\u0391-HELICES/"}, {"entity_type": "STRANDS/", "description": "Meanders", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "HELICES/"}, {"entity_type": "\u0391-HELIX/", "description": "The N-terminal lobe\'s structure is compared with DmDRWD domain, revealing differences and similarities in secondary structures like \u03b2-meanders and \u03b1-helices.<SEP>\u03b2-Element/", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "N-TERMINAL LOBE/"}, {"entity_type": "COMPARISON/", "description": "When comparing DmDRWD with known structures, E2 ubiquitin-conjugating enzymes are found to be structurally similar to both N-terminal and C-terminal lobes of human central domain.<SEP>Disease", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "E2 UBC ENZYMES/"}, {"entity_type": "MOLECULAR INTERACTION/", "description": "Disease", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "CYSTEINE/"}, {"entity_type": "DISEASE/", "description": "Results of this assay indicate that URD lacks catalytic function by failing to form a thioester linkage with ubiquitin.<SEP>Test Result", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "THIOESTER CHARGE ASSAY/"}, {"entity_type": "C-TERMINAL LOBES/", "description": "Comparison/", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "N-"}, {"entity_type": "DISEASE/", "description": "Stability Requirement", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "CONSERVATION/"}, {"entity_type": "COMPARISON/", "description": "A comparison of structures using the DALI server shows similarities between DmFANCL and human FANCL central domain, but notable differences exist in specific regions.", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "DRWD DOMAIN/"}, {"entity_type": "MOLECULAR FUNCTION/", "description": "The cysteine residue in the N-terminal lobe is positioned similarly as catalytic residues in E2 enzymes, suggesting potential for forming a thioester bond with ubiquitin.", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "CATALYTIC CYSTEINE/"}, {"entity_type": "\u0392-ELEMENT/", "description": "Structural Feature/", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "\u0391-HELIX/"}, {"entity_type": "STABILITY PARAMETER/", "description": "The conservation levels on different parts of URD indicate potential for stability differences between vertebrate and invertebrate versions of the protein.", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "SURFACE CONSERVATION/"}, {"entity_type": "COMPOSITION ANALYSIS/", "description": "Hydrophobic patches are identified as part of surface composition, contributing to the protein\'s interaction dynamics with its environment and other molecules.", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "id": "EXTENSIVE HYDROPHOBIC RESIDUES/"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The URD (ubiquitin-like domain) interacts with various components including FANCI and FANCD2 through specific hydrophobic patches.<SEP>Wild type URD was purified to homogeneity and subjected to pulldown arrays with FANCI and FANCD2.", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac<SEP>chunk-e097891654194982dfc0ef87401d3aa9", "id": "URD"}, {"entity_type": "GENE", "description": "DmDRWD is a gene that corresponds to the Drosophila DRWD (DNA repair domain).", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "DMDRWD"}, {"entity_type": "GENE", "description": "The Human URD refers to the ubiquitin-like domain found in humans.", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "HUMAN URD"}, {"entity_type": "GENE", "description": "Drosophila DRWD stands for the DNA repair domain found in Drosophila species.", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "DROSOPHILA DRWD"}, {"entity_type": "PROTEIN", "description": "FANCI protein was used in the pulldown arrays with URD<SEP>The protein FANCI is one of the proteins that interacts with the URD during DNA repair processes.", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac<SEP>chunk-e097891654194982dfc0ef87401d3aa9", "id": "FANCI"}, {"entity_type": "PROTEIN", "description": "FANCD2 is another protein involved in DNA repair and binds to the URD, as evidenced by pulldown experiments.<SEP>FANCD2 protein was also used in the pulldown arrays with URD", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac<SEP>chunk-e097891654194982dfc0ef87401d3aa9", "id": "FANCD2"}, {"entity_type": "PROTEIN", "description": "Mutation at Phe-63 in Ube2T affects its interaction affinity with RING domain of FANCL, as seen through ITC studies.", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "PHE-63 IN UBE2T"}, {"entity_type": "GENE", "description": "DRWD refers to DNA repair domain found in organisms, like Drosophila.", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "id": "DRWD"}, {"entity_type": "GENOMIC VARIANT", "description": "The mutations L248A, F252A, L254A and I265A were present on the C-terminal lobe", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "id": "L248A/F252A/L254A/I265A"}, {"entity_type": "GENOMIC VARIANT", "description": "The mutations W212A and L214A were present on a different area, the N-terminal lobe", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "id": "W212A/L214A"}, {"entity_type": "TYPE OF COMPONENT", "description": "Description of Component B", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "id": "COMPONENT B"}, {"entity_type": "ENZYME", "description": "Citrate synthase is an enzyme that catalyzes the condensation of acetyl-CoA and oxaloacetate to form citrate.", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "id": "CITRATE SYNTHASE"}, {"source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "description": "Structure Comparison/", "entity_type": "UNKNOWN", "id": "FANCL/"}, {"source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "description": "Structure Comparison/", "entity_type": "UNKNOWN", "id": "URD/"}, {"source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "description": "DALI Comparison/", "entity_type": "UNKNOWN", "id": "STRUCTURAL SIMILARITY/"}, {"source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "description": "Thioester Charge Assay/", "entity_type": "UNKNOWN", "id": "THIOESTER BOND FORMATION/"}, {"source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "description": "Comparative Analysis/", "entity_type": "UNKNOWN", "id": "N-TERMINAL/ C-TERMINAL LOBES/"}, {"source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "description": "Comparative Analysis/", "entity_type": "UNKNOWN", "id": "SURFACE COMPOSITION/"}, {"source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "description": "The URD plays a critical role in facilitating the interaction between FANCI and FANCD2 during DNA repair processes by serving as a binding site.)", "entity_type": "UNKNOWN", "id": "FANCI AND FANCD2"}, {"entity_type": "GENOMIC VARIANT", "description": "The exception to Hardy-Weinberg equilibrium satisfaction for five candidate SNPs.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "FGFR4 RS376618"}, {"entity_type": "FUNCTIONAL GENETIC VARIANT", "description": "Associated with differential colesevelam treatment effect on AC t1/2 and GC24.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "FGFR4 RS351855"}, {"entity_type": "GENOTYPE", "description": "Increased Colesevelam effect on CT, with increased AC t1/2 and decreased GC24.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "RS351855 GA/AA GENOTYPE GROUP"}, {"entity_type": "GENOTYPE", "description": "No significant effect of Colesevelam treatment on CT with either AC t1/2 or GC24.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "RS351855 GG GENOTYPE GROUP"}, {"entity_type": "SNP", "description": "Four out of five candidate SNPs evaluated for Hardy-Weinberg equilibrium satisfaction.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "FGFR4 SNPS"}, {"entity_type": "GENOMIC VARIANT", "description": "Differential treatment effect on CT, associated with lower GC24 value and potentially longer AC t1/2 compared to placebo.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "KLB RS4975017"}, {"entity_type": "GENOTYPE", "description": "Lower GC24 value in this group when treated with Colesevelam versus placebo.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "CA/AA GENOTYPE GROUP"}, {"entity_type": "GENOTYPE", "description": "No significant treatment effects observed in the CC genotype for GC 24 and AC t1/2.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "CC GENOTYPE GROUP"}, {"entity_type": "FUNCTIONAL VARIANT", "description": "Numerical but not statistically significant differential Colesevelam treatment effects on AC t1/2 and GC24.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "KLB RS17618244"}, {"entity_type": "GENOTYPE", "description": "No significant effect of Colesevelam treatment on CT or respective parameters, regardless of genotype.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "GA/AA GENOTYPE GROUP"}, {"entity_type": "FUNCTIONAL VARIANT OF KLB GENE", "description": "The variant was numerically associated with differential treatment effects on AC t1/2 and GC24, though not statistically significant.", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "id": "RS17618244"}, {"source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "description": "Significantly delayed CT and numerically decreased GC24 in the GA/AA genotype group, compared to placebo.)", "entity_type": "UNKNOWN", "id": "COLESEVELAM TREATMENT EFFECT"}, {"source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "description": "Satisfied except for FGFR4 rs376618; no significant differential treatment effects associated with this SNP.)", "entity_type": "UNKNOWN", "id": "RS376618"}, {"source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "description": "The rs351855 GA/AA genotype group showed a significantly delayed CT, with increased AC t1/2 compared to placebo, while the GG genotype group did not show any significant effect of Colesevelam treatment.)", "entity_type": "UNKNOWN", "id": "RS351855 GA/AA GENOTYPE GROUP AND GG GENOTYPE GROUP"}, {"source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "description": "No significant effects of Colesevelam treatment on CT were observed in the CC genotype group, compared to placebo.)", "entity_type": "UNKNOWN", "id": "RS351855 GA/AA GENOTYPE GROUP AND CC GENOTYPE GROUP"}, {"source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "description": "Both were genetic variants that showed association with differential colesevelam treatment effect.)", "entity_type": "UNKNOWN", "id": "KLB GENOTYPE VARIATION"}, {"entity_type": "NUTRIENT", "description": "Participants\' dietary patterns were recorded for daily fat intake during the study.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "ESTIMATED DAILY FAT INTAKE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Previous cholecystectomy was recorded in 23% of IBS-D, approximately 4% of IBS-C, and approximately 4% of HV (P = .05).<SEP>Prior cholecystectomy was recorded in participants, differing across subgroups (IBS-D, IBS-C, and HV).", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "CHOLECYSTECTOMY"}, {"entity_type": "DISEASE", "description": "Participants\' symptoms were noted and considered to be related to a diagnosis of either IBS-D, IBS-C or were healthy volunteers (HV), with those in IBS-D not showing significant change after cholecystectomy.<SEP>Participants\' symptoms were noted as part of diagnosis for IBS-D, IBS-C or being healthy volunteers (HV), with those in IBS-D not showing significant change after cholecystectomy.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "SYMPTOMS"}, {"entity_type": "NUTRIENT", "description": "FGF19 is a protein involved in the regulation of bile acid synthesis and metabolism. Measurements showed no significant association with subgroups, although it can play a role in various metabolic pathways.<SEP>Serum levels of FGF19 were measured as part of the study but did not show an association with subgroups.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "SERUM FGF19"}, {"entity_type": "NUTRIENT", "description": "48-hour stool fat was recorded and compared between IBS-D, IBS-C, HV, with IBS-D having lower values than HV, and IBS-C having higher values compared to HV.<SEP>48-hour stool fat was recorded as part of the study to assess digestive processes and fat absorption.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "STOOL FAT"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The weight of the 48-hour stool sample provides information on fecal output and gastrointestinal transit times.<SEP>The weight of the 48-hour stool sample was also measured in each participant.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "clusters": "[{level: 0, cluster: 25}]", "id": "STOOL WEIGHT"}, {"entity_type": "METABOLIC PATHWAY", "description": "The synthesis and excretion of bile acids are part of digestive processes involving the liver and gallbladder, especially in relation to diet and health conditions like IBS-D.<SEP>The synthesis and excretion of bile acids were monitored using various tests on stool samples, with IBS-D patients showing increased synthesis or excretion compared to HV.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "STOOL BILE ACIDS (BA) SYNTHESIS/EXCRETION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The C4 enzyme is involved in this process of bile acid biosynthesis through the CYP7A1 pathway. Correlation with serum C4 levels suggests a role in bile acid metabolism and potentially IBD.<SEP>The C4 enzyme was associated with the synthesis of bile acids via the cholesterol 7\u03b1-hydroxylase pathway, indicating its role in bile acid production.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "CHOLESTEROL 7\u0391-HYDROXYLASE (CYP7A1) PATHWAY"}, {"entity_type": "NUTRIENT", "description": "Stool BA levels were found to be significantly higher in IBS-D patients compared to IBS-C but not compared to HV.<SEP>Stool BA was used as an indicator for the health status of participants, with differences observed between IBS-D, IBS-C, and HV.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "clusters": "[{level: 0, cluster: 25}]", "id": "STOOL BA"}, {"entity_type": "NUTRIENT", "description": "Serum levels of fibroblast growth factor 19 (FGF19) were recorded as part of the study, which did not show an association with subgroups.", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "id": "FASTING SERUM FGF19"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "The BMI in IBS-D subgroup was approximately 4 units higher than that in IBS-C and HV, indicating a potential link to body weight regulation and digestive conditions.)<SEP>The BMI in IBS-D subgroup was approximately 4 units higher than that in IBS-C and HV.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 8}, {level: 1, cluster: 34}]", "id": "IBS-D"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "A significantly higher serum C4 level correlated with a higher stool BA synthesis or excretion.)<SEP>Increased serum C4 levels were associated with higher stool BA synthesis or excretion, suggesting the role of this protein in bile acid dynamics.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 25}]", "id": "SERUM C4"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "The total content of stool bile acids correlated positively and strongly with the weight of stool samples.)<SEP>The total content of stool bile acids was positively correlated with stool weight samples, reflecting digestion and fat absorption.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 25}]", "id": "TOTAL STOOL BA"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "A correlation between the weight of stools and fat content indicated differences in digestive efficiency among participants.)<SEP>A positive correlation between the weight of stools and the fat content was observed.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 25}]", "id": "FAT CONTENT"}, {"source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "description": "A negative correlation between fasting serum FGF19 levels and stool weight suggested possible metabolic differences influencing fecal output.)<SEP>A negative correlation between fasting serum FGF19 levels and stool weight was observed, indicating less weight could be associated with higher FGF19.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 21}]", "id": "FGF19"}, {"entity_type": "PROTEIN", "description": "FAAP20 is a 20 kDa protein that binds to FANCA through its N-terminal region and contains a conserved UBZ domain in its C terminal extremity.<SEP>In vivo accumulation of FAAP20 at ICL sites is dependent on UBZ domain functionality.<SEP>The FAAP20 protein hydrophobically interacts with the canonical Ile44-centered hydrophobic patch of Ub, and its second loop participates in the hydrophobic interaction with Ile44. The Asp164 O\u03b4 atoms hydrogen bond with Ala46 and Gly47 side chains of Ub. Trp180 at the C-terminal extremity of FAAP20-UBZ interacts with Ile44 through a hydrophobic interaction, also hydrogen bonding with main-chain CO groups of Leu171, Ser174, Thr175, and Val178 of Ub.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4<SEP>chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "FAAP20"}, {"entity_type": "GENE", "description": "FANCA gene encodes for the FAAP20 protein, which plays an important role in the interaction with UBZ domain.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "FANCA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The UBZ domain at C-terminal extremity of FAAP20 facilitates its specific interaction with Ub chains.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "UBZ DOMAIN"}, {"entity_type": "METABOLIC PATHWAY", "description": "FAAP20 interacts and binds to the K63-linked Ub2 in a complex, highlighting its role in the ubiquitination pathway.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "UB CHAINS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The crystal structure of FAAP20-UBZ with K63-linked Ub2 demonstrates its interaction within the ubiquitination process.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "K63-LINKED UB2"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "FAAP20, through its UBZ domain, is involved in the ubiquitination pathway which is critical for cell cycle progression and maintenance of cellular homeostasis.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "UBIQUITINATION PATHWAY"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The interaction with Ub chains suggests FAAP20\'s role in maintaining cellular stability under environmental changes.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "CELLULAR HOMEOSTASIS"}, {"entity_type": "RELATIONSHIP STRENGTH (1:1 STOICHIOMETRY)", "description": "FAAP20-UBZ interacts with one K63-Ub2 molecule, which could be an artifact of crystallization process based on SPR spectroscopy data.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "SPECIFIC INTERACTION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "A process that involves folate\'s interaction with the NOS1 protein to maintain vascular function.<SEP>Folate metabolism, which involves various enzymes including those found in the one-carbon metabolism pathway, interacts with NOS1 protein to maintain vascular function.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-cc528af98598e1adb4684c0166430cf4", "id": "FOLATE METABOLISM"}, {"entity_type": "METABOLIC PATHWAY", "description": "The one-carbon metabolism pathway is crucial for folate utilization and impacts on cardiovascular health by supporting DNA synthesis and methylation processes.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "ONE-CARBON METABOLISM PATHWAY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Folate influences DNA synthesis through its role in providing methyl groups, impacting gene expression related to heart function.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "DNA SYNTHESIS"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Proper methylation processes, supported by sufficient folate, are essential for maintaining normal heart function and preventing cardiovascular disease.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "METHYLATION"}, {"entity_type": "BIOLOGICAL ATTRIBUTE", "description": "Variants in genes encoding enzymes involved in the one-carbon metabolism pathway can influence cardiovascular health, including those related to folate metabolism.", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "id": "GENETIC FACTORS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "FAAP20-UBZ binds Ub through its \u03b1-helix motif. The interaction is closely related to MIU domain of Rabex5 and Vps27-UIM domains, suggesting convergent evolution of these three distinct Ub-binding motifs.<SEP>The UBDs (Ubiquitin Binding Domains) such as UIM, MIU, and FAAP20-UBZ bind Ub with close resemblance suggesting convergent evolution of these distinct Ub-binding motifs.<SEP>Ub (ubiquitin) is bound to FAAP20 through interactions including a hydrophobic interaction with Ile44-centered patch formed by Leu8, Ile44, Val70 side chains and aliphatic portions of Thr66, His68 side chains. The Asp164 O\u03b4 atoms hydrogen bond with Ala46 and Gly47 main-chain amide groups respectively.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591<SEP>chunk-fa32a208a2f004c10c1b6777c3be13f2", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 41}]", "id": "UB"}, {"entity_type": "PROTEIN", "description": "Site-directed mutations D164A, A168L and W180A in FAAP20-UBZ do not allow binding to Ub similar to mutation Cys150 which disables bridging between the first loop and central \u03b1-helix thus likely destabilizing FAAP20-UBZ structure. Binding affinities of mutants L161A, A172L, D177A are significantly reduced but still measurable.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "FAAP20-UBZ MUTANTS"}, {"entity_type": "PROCESS", "description": "After intermolecular interactions with Ub, FAAP20 proteins are recruited to DNA damage sites formed by interstrand cross-links (ICLs).", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "ICL FORMATION"}, {"entity_type": "PROCESS", "description": "Wild-type FAAP20 accumulates at DNA damage sites following UVA irradiation in cells pre-incubated with psoralen. Mutant forms like C150S, D164A, A168L and W180A have diffuse localization throughout the cell.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "CELLULAR LOCALIZATION"}, {"entity_type": "DISEASE", "description": "FAAP20 protein accumulation at ICL sites is investigated in cells irradiated with UVA after psoralen addition. Localization of GFP-tagged FAAP20 proteins is monitored by confocal imaging.<SEP>The study involves in vivo treatment of mice with TAG72-BB\u03b6 CAR T cells to evaluate their anti-tumor activity.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2<SEP>chunk-183db5c908df609bb5289896b4ba09d0", "id": "IN VIVO STUDY"}, {"entity_type": "CATEGORY", "description": "The WD-repeat protein FAAP20 contains a UBZ domain which interacts with Ub, influencing its cellular localization and accumulation at DNA damage sites.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "WD-REPEAT PROTEIN"}, {"entity_type": "COMPARISON", "description": "Conservation or replacement of residues involved in Ub interactions is observed among human, mouse, rat, chicken, and pufferfish FAAP20 proteins. The exact sequence conservation cannot be assessed due to high divergence between UBZs.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "CONSERVATION OF RESIDUES"}, {"entity_type": "METHODOLOGY", "description": "The 3D structures of WRNIP1-, RAD18-, Pol\u03b7- and NEMO-UBZ are compared with FAAP20-UBZ in complex with Ub. WRNIP1 and RAD18 display the canonical \u03b2\u03b2\u03b1 fold whereas Pol\u03b7 and NEMO interact differently.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "3D STRUCTURAL ANALYSIS"}, {"entity_type": "PROTEIN", "description": "Has a \u03b2-loop that interacts with Ile44 of Ub similar to FAAP20-UBZ.<SEP>The Pol\u03b7 protein hydrophobically interacts with Ub and binds through a \u03b2\u03b2\u03b1 structure, with interactions involving Pro398-Lys399 in NEMO-UBZ and Glu636-Lys637 in Pol\u03b7 UBZ.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591<SEP>chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "POL\u0397"}, {"entity_type": "PROTEIN", "description": "The RAD18 protein also adopts the canonical \u03b2\u03b2\u03b1 fold for binding to Ub, involving specific residues in both antiparallel \u03b2-strands of the \u03b2-loop that interact with corresponding residues on Ub.", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "id": "RAD18"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Does not fit in the hydrophobic patch of Ub centered on Ile44 due to the larger side chain of Met415 compared to Ala168 found in FAAP20-UBZ, suggesting a different mode of interaction with Ub.<SEP>Interacts with Ub in a manner different from RAD18-UBZ and WRNIP1-UBZ, having highly conserved Pro21-Val22 replaced by Pro398-Lys399 and Glu636-Lys637 in Pol\u03b7-UBZ.<SEP>The side chain of Met415 is much larger than that of Ala168 of FAAP20-UBZ causing it to fit poorly in the hydrophobic patch centered on Ile44, suggesting different interaction with Ub.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "id": "NEMO-UBZ"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Has charged residues at positions corresponding to Asp164, Leu167, Ala168, and Leu171 of FAAP20-UBZ which help in binding Ub similarly to FAAP20-UBZ.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "id": "POL\u0397-UBZ"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Has sequence conservation suggesting that Pol\u03b7-UBZ binds Ub similar to FAAP20-UBZ, but this is not observed for NEMO-UBZ.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "id": "WRNIP1-UBZ"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The \u03b2-loop region lacks or diverges in FAAP20-UBZ and Pol\u03b7-UBZ compared to WRNIP1/Rad18 group, which interact with Ile44 of Ub through their \u03b2-loop.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 41}]", "id": "FAAP20-UBZ"}, {"entity_type": "PROTEIN", "description": "Has UIM (Ub-interacting motif) domain which binds Ub with a polarity opposite to FAAP20-UBZ and Rabex5-MIU but the majority of the Ub-interacting residues on the \u03b1-helix are identical or functionally equivalent among Vps27-UIM, FAAP20-UBZ, and Rabex5-MIU.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "id": "VPS27"}, {"entity_type": "PROTEIN", "description": "Has MIU (motif interacting with Ub) domain which binds Ub in the same orientation as FAAP20-UBZ. The majority of residues involved in Ub binding are identical or functionally equivalent among Vps27-UIM, FAAP20-UBZ, and Rabex5-MIU.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "id": "RABEX5"}, {"entity_type": "MOLECULAR INTERACTION", "description": "All UBDs including NEMO-UBZ, Pol\u03b7-UBZ, WRNIP1-UBZ and FAAP20-UBZ show the capacity to bind with Ub through their respective \u03b1-helix motifs indicating ubiquitination pathway involvement.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "id": "UBIQUITIN (UB)"}, {"source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "description": "FAAP20 interacts specifically with K63-Ub2 through its UBZ domain, indicating specific targeting of this type of ubiquitin chain in the cell.)", "entity_type": "UNKNOWN", "id": "FAAP20-UBZ COMPLEX"}, {"source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "description": "SPR data suggests that FAAP20-UBZ might interact with K63-Ub2 in a 1:1 stoichiometry, although this may represent an experimental artifact based on the calculated Rmax.)", "entity_type": "UNKNOWN", "id": "1:1 STOICHIOMETRY INTERACTION"}, {"source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "description": "The interaction between specific dietary nutrients, like folate from leafy greens, is crucial for maintaining optimal cardiovascular health based on lifestyle factors and genetic predispositions.)", "entity_type": "UNKNOWN", "id": "NUTRIENT INTERACTION"}, {"source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "description": "The interaction between specific dietary nutrients, like folate from leafy greens, is crucial for maintaining optimal cardiovascular health based on lifestyle factors and genetic predispositions.)", "entity_type": "UNKNOWN", "id": "RELATIONSHIP STRENGTH (5/6)"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "Mutations D164A, A168L and W180A in FAAP20-UBZ disable binding to Ub and C150S mutation destabilizes the structure leading to a lack of functional recruitment of FAAP20 proteins.)", "entity_type": "UNKNOWN", "id": "EFFECT"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "Mutations D164A, A168L and W180A in FAAP20-UBZ disable binding to Ub and C150S mutation destabilizes the structure leading to a lack of functional recruitment of FAAP20 proteins.)", "entity_type": "UNKNOWN", "id": "FAAP20-UBZ MUTATIONS"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "The intermolecular interactions between FAAP20 protein domain UBZ and Ub are essential for proper cellular localization and accumulation at DNA damage sites in response to intermolecular binding events and ICL formation processes.)", "entity_type": "UNKNOWN", "id": "SIGNIFICANCE"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "The intermolecular interactions between FAAP20 protein domain UBZ and Ub are essential for proper cellular localization and accumulation at DNA damage sites in response to intermolecular binding events and ICL formation processes.)", "entity_type": "UNKNOWN", "id": "UB INTERACTION"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "Conservation of residues involved in Ub interactions is observed among various species but sequence variations are noted due to high divergence between UBZ structures, requiring detailed 3D structural analysis for precise comparisons.)", "entity_type": "UNKNOWN", "id": "EVOLUTIONARY INSIGHT"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "Conservation of residues involved in Ub interactions is observed among various species but sequence variations are noted due to high divergence between UBZ structures, requiring detailed 3D structural analysis for precise comparisons.)", "entity_type": "UNKNOWN", "id": "RESIDUE CONSERVATION"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "Comparative analysis reveals differences in the modes and strength of interactions between various UBZs (FAAP20-UBZ, WRNIP1-UBZ, RAD18-UBZ) and Ub based on conserved or replaced key residues.)", "entity_type": "UNKNOWN", "id": "OBSERVATION"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "Comparative analysis reveals differences in the modes and strength of interactions between various UBZs (FAAP20-UBZ, WRNIP1-UBZ, RAD18-UBZ) and Ub based on conserved or replaced key residues.)", "entity_type": "UNKNOWN", "id": "UB BINDING COMPARISON"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "The variations in key interacting residues among UBZ proteins suggest evolutionary adaptations for binding efficiency or specific roles in cellular processes involving Ub.)", "entity_type": "UNKNOWN", "id": "IMPLICATION"}, {"source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "description": "The variations in key interacting residues among UBZ proteins suggest evolutionary adaptations for binding efficiency or specific roles in cellular processes involving Ub.)", "entity_type": "UNKNOWN", "id": "RESIDUE VARIATIONS"}, {"source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "description": "Interaction with Ile44 of Ub through their \u03b2-loop suggests similarities between WRNIP1, Rad18 and Pol\u03b7-UBZ binding to Ub.", "entity_type": "UNKNOWN", "id": "WRNIP1/RAD18 GROUP"}, {"source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "description": "Similar residues are involved in Ub-binding by Vps27-UIM and FAAP20-UBZ with opposite polarity towards the bound Ub showing convergent evolution among UBDs.", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 41}]", "id": "VPS27-UIM"}, {"source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "description": "The same critical residues involved in Ub binding are identical between Rabex5-MIU and FAAP20-UBZ suggesting closely related interaction mechanisms.", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 41}]", "id": "RABEX5-MIU"}, {"source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "description": "While FAAP20-UBZ shows interactions with Ub similar to MIU domain of Rabex5 and Vps27-UIM, it differs in its mode of interaction compared to RAD18-UBZ indicating structural diversity among UBDs.", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 41}]", "id": "RAD18-UBZ"}, {"entity_type": "GENOMIC VARIANT", "description": "Minor \'G\' allele is more prevalent in healthy controls than CD group.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "RS2395182"}, {"entity_type": "GENOMIC VARIANT", "description": "Minor \'C\' allele frequency among controls is significantly different from that in CD patients.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "RS7775228"}, {"entity_type": "GENOMIC VARIANT", "description": "Minor \'T\' allele shows a significant difference between 10% of the CD group and 3% of healthy controls.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "RS2187668"}, {"entity_type": "GENOMIC VARIANT", "description": "There was no statistically significant difference between case and control groups.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "RS7454108"}, {"entity_type": "MOLECULAR INTERACTION", "description": "HLA-DQ2.5 haplotype is significantly higher in CD patients compared to healthy controls (10.89 vs 2.91%; p = 0.024).", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "HLA-DQ2.5 HAPLOTYPE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The DQ8 heterozygous haplotypes are highly frequent among CD patients (33.66%), with significant differences between the patient and control group (p = 0.014).", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "DQ8 HAPLOTYPE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "There were no heterozygous DQ2.5 carriers in either patients or controls.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "DQ8/DQ2.5 HAPLOTYPE"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The predicted homozygous DQ2.2 haplotype was frequent among control (5.82%) compared to CD patients (0.99%).", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "DQ2.2 HAPLOTYPE"}, {"entity_type": "DISEASE", "description": "SNP results show significant differences in the minor allele frequency distribution between CD patients and healthy controls, particularly for rs2395182, rs7775228, and rs2187668.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "HLA TAG SNPS"}, {"entity_type": "CLINICAL SYMPTOM", "description": "Common symptom seen among CD patients characterized by loss of appetite.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "ANOREXIA"}, {"entity_type": "CLINICAL SYMPTOM", "description": "A common manifestation in CD patients noted as a feeling of fullness or bloating in the abdomen.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "ABDOMINAL DISTENSION"}, {"entity_type": "DISEASE", "description": "An autoimmune disease affecting insulin production found among some CD patients, making up about 32% of cases.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "TYPE 1 DIABETES MELLITUS"}, {"entity_type": "DISEASE", "description": "An inflammatory condition targeting the thyroid gland often associated with other autoimmune diseases like CD, seen in about 8% of our patient group.<SEP>The presence of antibodies against thyroid-related proteins indicates the autoimmune response affecting thyroid tissue.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc<SEP>chunk-e323b0d4f82733a78f687e51f47b500a", "id": "AUTOIMMUNE THYROIDITIS"}, {"entity_type": "DISEASE", "description": "A chronic autoimmune disease characterized by inflammation and damage to various parts of the body found among about 3% of our patients.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "SYSTEMIC LUPUS ERYTHEMATOSUS"}, {"entity_type": "NON-AUTOIMMUNE DISEASE MANIFESTATION", "description": "A bone softening condition often due to vitamin D deficiency observed in less than 5% of participants.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "OSTEOMALACIA"}, {"entity_type": "NON-AUTOIMMUNE DISEASE MANIFESTATION", "description": "Neurological issue characterized by recurrent seizures found among about 4% of patients.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "SEIZURE DISORDERS"}, {"entity_type": "NON-AUTOIMMUNE DISEASE MANIFESTATION", "description": "A genetic disorder typically associated with cognitive and physical development delay seen in around 4% of our participants.", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "id": "DOWN SYNDROME"}, {"source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "description": "HLA-DQ genotypes show a significant difference between CD patients compared to healthy controls.)", "entity_type": "UNKNOWN", "id": "CD PATIENTS"}, {"source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "description": "HLA-DQ genotypes show a significant difference between CD patients compared to healthy controls.)", "entity_type": "UNKNOWN", "id": "HLA-DQ HAPLOTYPE"}, {"source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc<SEP>chunk-05d583f893b703816ad8de9561aa5292<SEP>chunk-6f7ce1c71bd46f223756da44b0d0bb2c<SEP>chunk-6a657cd0a000b6df4db8f1305b795021", "description": "The distribution of rs179009 genotypes varies between HIV-infected individuals and healthy controls.<SEP>The group of individuals without any medical condition as a comparison to asthmatic patients.<SEP>There are significant differences in the minor allele frequency distribution among CD patients compared to healthy controls for these SNPs.)<SEP>When comparing healthy controls to CD subjects, there were differences in the presence of MAPbacteremia and specific allele combinations.", "entity_type": "DISEASE", "id": "HEALTHY CONTROLS"}, {"source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "description": "There are significant differences in the minor allele frequency distribution among CD patients compared to healthy controls for these SNPs.)", "entity_type": "UNKNOWN", "id": "RS2395182, RS7775228, AND RS2187668"}, {"source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "description": "No difference exists between the tag SNP predicted homozygous HLA-DQ2.5 haplotype in CD patients and healthy controls, though not statistically significant.)", "entity_type": "UNKNOWN", "id": "DQ2.5 HAPLOTYPE"}, {"source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "description": "The sequencing analysis with sanger method confirmed the accuracy of Real-time PCR based tagSNPs approach in determining the HLA-DQ haplotype.)", "entity_type": "UNKNOWN", "id": "HLA TAG SNPS AND HAPLOTYPE RESULTS"}, {"source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "description": "The sequencing analysis with sanger method confirmed the accuracy of Real-time PCR based tagSNPs approach in determining the HLA-DQ haplotype.)", "entity_type": "UNKNOWN", "id": "VALIDATION BY SANGER SEQUENCING"}, {"source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "description": "An autoimmune disease that affects insulin production; a part of our disease manifestation group.)", "entity_type": "UNKNOWN", "id": "DISEASE GROUP"}, {"source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "description": "Condition characterized by weak bones often due to vitamin D deficiency; identified among a small percentage of participants.)", "entity_type": "UNKNOWN", "id": "NON-AUTOIMMUNE DISEASE GROUP"}, {"source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "description": "Conditions characterized by recurrent seizures, sometimes associated with autoimmune diseases like CD; observed in a subset of patients.)", "entity_type": "UNKNOWN", "id": "NEUROLOGICAL CONDITION GROUP"}, {"source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "description": "Genetic disorder leading to cognitive and physical development challenges; identified among approximately 4% of participants.)", "entity_type": "UNKNOWN", "id": "DEVELOPMENTAL DISORDER GROUP"}, {"entity_type": "GENOMIC VARIANT", "description": "CABCB1 gene variant C> T was associated with grade 3-4 anemia.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS1128503"}, {"entity_type": "GENOMIC VARIANT", "description": "ABCC2 gene variant C>G was significantly related to grade 3-4 anemia.<SEP>This variant, rs12762549 (ABCC2 A\u2009>\u2009G), was significantly associated with PFI in the fully-adjusted model.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404<SEP>chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "RS12762549"}, {"entity_type": "GENOMIC VARIANT", "description": "ABCA1 gene variant A>G showed a significant link with grade 3-4 anemia.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS363717"}, {"entity_type": "GENOMIC VARIANT", "description": "ERCC1 gene variant T>C was associated with grade 3-4 anemia.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS11615"}, {"entity_type": "GENOMIC VARIANT", "description": "CYP3A4 gene variant T>C correlated with grade 3-4 thrombocytopenia.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS4986910"}, {"entity_type": "GENOMIC VARIANT", "description": "ABCC2 gene variant A>G was significantly associated with CSF use.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS2073337"}, {"entity_type": "GENOMIC VARIANT", "description": "UGT1A1 gene variant C>T showed a link to grade 3-4 thrombocytopenia.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS671"}, {"entity_type": "GENOMIC VARIANT", "description": "ABCB1 ABCG2 gene variant T>G correlated with grade 4 neutropenia.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS20425"}, {"entity_type": "GENOMIC VARIANT", "description": "TERT promoter variant C>T was associated with an increased risk of anemia and thrombocytopenia in ovarian cancer patients.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS769108"}, {"entity_type": "GENOMIC VARIANT", "description": "GSTP1 gene variant A>G showed a link to an increased use of ESA during chemotherapy.", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "id": "RS352746"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant, rs1799793 (ERCC2 G\u2009>\u2009A), was significantly correlated with PFI in both cases.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "RS1799793"}, {"entity_type": "DISEASE", "description": "It refers to Platinum-free interval which survived correction for multiple testing", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "PFI"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant, rs6104 (SERPINB2 C\u2009>\u2009G), was significantly associated with PFI in the fully-adjusted model.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "RS6104"}, {"entity_type": "GENE", "description": "ABCC2 gene encodes for a transporter that is involved in drug resistance and other cellular processes.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "ABCC2"}, {"entity_type": "PROTEIN", "description": "The ABCC2 protein functions as an efflux pump, which removes certain substances like drugs from cells.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "GTPASE"}, {"entity_type": "GENOMIC VARIANT", "description": "rs12762549 variant is a genetic mutation characterized by the substitution of nucleotide at position 12762549 leading to an amino acid change in ABCC2 protein.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "A/G"}, {"entity_type": "GENOMIC VARIANT", "description": "rs6104 variant indicates another genetic alteration with a substitution of nucleotide at position 6104 influencing SERPINB2 function.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "G/T"}, {"entity_type": "GENE", "description": "Another important gene for drug metabolism and resistance, ABCB1 encodes for the B-cell leukemia/lymphoma 1 (BCL-1) multidrug resistance protein.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "ABCB1"}, {"entity_type": "MOLECULAR INTERACTION", "description": "ABCB1 interacts with various proteins to form a transmembrane channel that facilitates drug export from cells.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "ATP-BINDING CASSETTE TRANSPORTER"}, {"entity_type": "DISEASE", "description": "A measure of the time duration after chemotherapy when no further treatment is required, reflecting tumor control without recurrence.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "PLATINUM-FREE INTERVAL"}, {"entity_type": "DISEASE", "description": "A system for classifying cancer stages based on tumor characteristics and extent of spread.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "FIGO STAGE"}, {"entity_type": "DISEASE", "description": "An assessment of the degree to which a tumor cell differs from normal cells, reflecting the malignancy level.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "TUMOR GRADE"}, {"entity_type": "DISEASE", "description": "A detailed microscopic examination that categorizes cancer based on its origin and cellular structure.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "TUMOR HISTOLOGY"}, {"entity_type": "DISEASE", "description": "The presence of cancer cells remaining in the body post-surgery, indicating incomplete removal or residual burden.", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "id": "RESIDUAL DISEASE AFTER DEBULKING SURGERY"}, {"source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "description": "The rs1799793 ERCC2 G>A variant showed a significant correlation with progression-free interval (PFI) and overall survival (OS) in ovarian cancer.)", "entity_type": "UNKNOWN", "id": "PFI AND OS"}, {"source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "description": "Correction for multiple testing is used when several statistical tests are performed simultaneously, which helps to reduce type I errors.)<SEP>This refers to the application of statistical methods used when multiple hypotheses are tested simultaneously, adjusting p-values to control the family-wise error rate.)", "entity_type": "UNKNOWN", "id": "CORRECTION FOR MULTIPLE TESTING"}, {"entity_type": "GENOMIC VARIANT", "description": "De novo 6.4 Mb deletion on chromosome 3 containing more than 40 RefSeq genes, including UMPS", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "CHR3:121233326\u2013127618464"}, {"entity_type": "GENE/TRANSCRIPT", "description": "Additionally copy number variations were excluded in individuals 4, 6, and 7<SEP>Additionally, in-depth analysis of UMPS variants in the database<SEP>Has a de novo 6.4 Mb deletion and contains UMPS gene<SEP>Karyotyping, SNP array, and gene-specific deletion/duplication testing did not reveal abnormalities", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "UMPS"}, {"entity_type": "DISEASE/CONDITION", "description": "Not causing agenesis when deleted", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "AGENESIS OF THE CORPUS CALLOSUM"}, {"entity_type": "VARIANT", "description": "Found in I4; Premature stop leading to loss of normal protein function via protein truncation or NMD<SEP>Found in I4; deletion leading to loss of normal protein function", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "C.302DELA (P.LYS101ARGFS*9)"}, {"entity_type": "INDIVIDUAL/FAMILY MEMBER", "description": "Deletion contains UMPS gene and may contribute to the patient\u2019s phenotype", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "I1"}, {"entity_type": "VARIANT", "description": "Maternally inherited and also found in healthy brother S7<SEP>UMPS variant, maternally inherited, also found in healthy family member S7", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "C.385G>T (P.GLY129*)"}, {"entity_type": "INDIVIDUAL/FAMILY MEMBER", "description": "C.385G>T (p.Gly129*) variant found; maternally inherited and present in healthy brother<SEP>c.688C>T (p.Arg230Cys) variant found; maternally inherited", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "I7"}, {"entity_type": "VARIANT", "description": "Detected in I10; maternally inherited<SEP>Found in I10 and I11; maternally and paternally inherited respectively", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "C.889G>T (P.GLU297*)"}, {"entity_type": "INDIVIDUAL/FAMILY MEMBER", "description": "C.889G>T (p.Glu297*) variant found, paternally inherited", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "I10"}, {"entity_type": "VARIANT", "description": "Found in I3 and his healthy mother M3, may be on the same allele", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "C.224C>A (P.PRO75HIS)"}, {"entity_type": "FAMILY", "description": "12 family members tested positive for c.224C>A variant", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "PEDIGREE 2"}, {"entity_type": "VARIANT/MUTATION", "description": "Found in I3 and mother M3, may be on the same allele", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "C.286A>G (P.ARG96GLY)"}, {"entity_type": "INDIVIDUAL/FAMILY MEMBER", "description": "c.310G>A (Gly104Arg) variant detected; maternally inherited", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "I5"}, {"entity_type": "VARIANT", "description": "Detected in I6, maternally inherited", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "C.689G>A (P.ARG230HIS)"}, {"entity_type": "INDIVIDUAL/FAMILY MEMBER", "description": "Detected c.857T>A (p.Ile286Asn) and c.451G>A (p.Val151Met)<SEP>c.386T>C (p.Tyr130Ser) variant detected", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "I9"}, {"entity_type": "INDIVIDUAL/FAMILY MEMBER", "description": "c.857T>A (p.Ile286Asn) and c.451G>A (p.Val151Met) found", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "I11"}, {"entity_type": "VARIANT/MUTATION", "description": "Found in pedigree 2, tested positive by family members", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "P.PRO75HIS"}, {"entity_type": "VARIANT/MUTATION", "description": "Synonymous change, heterozygous and maternally inherited in individual 6", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "C.360A>G (P.LEU120LEU)"}, {"entity_type": "VARIANT", "description": "Detected in I9; benign single nucleotide polymorphism", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "C.386T>C (P.TYR130SER)"}, {"entity_type": "VARIANT", "description": "Detected in I12; benign single nucleotide polymorphism", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "C.457C>T (P.ALA153THR)"}, {"entity_type": "INDIVIDUAL/FAMILY MEMBER", "description": "c.457C>T (p.Ala153Thr) variant detected", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "I12"}, {"entity_type": "UMPS VARIANTS", "description": "Not identified through exome sequencing", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "EXON SKIPPING MUTATIONS"}, {"entity_type": "UMPS", "description": "Excluded in individuals 4, 6, and 7", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "COPY NUMBER VARIATIONS"}, {"entity_type": "IN UMPS GENE", "description": "Not detected by the next generation sequencing panel", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "SPLICE SITE VARIANTS"}, {"entity_type": "UMPS GENE", "description": "Identified through targeted sequencing", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "INTRONIC MUTATIONS"}, {"entity_type": "UMPS GENE", "description": "Detected in I3; potentially affecting mRNA stability and translation", "source_id": "chunk-b5a1267b747ee0cf908d98fa3fc26199", "id": "UTR (UNTRANSLATED REGION) VARIANT"}, {"entity_type": "GENOTYPING METHOD", "description": "The TaqMan method used for genotyping by an allele discrimination assay.", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "TAQMAN (VIIA\u2122 7 REAL-TIME PCR)"}, {"entity_type": "GENOMIC VARIANT", "description": "This is a specific variant of TGF\u00df1 gene, with alleles \'C\' and \'T\'", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "TGF-SS1 (RS1800469 C/T)"}, {"entity_type": "GENE", "description": "Interleukin 8 gene with two alleles, \'T\' and \'A\', which were studied", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "IL-8 (RS4073)"}, {"entity_type": "GENE", "description": "Interleukin 10 gene having two alleles, \'A\' and \'G\', was evaluated for association with CRC.", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "IL-10 (RS1800896)"}, {"entity_type": "GENE", "description": "Two polymorphisms of IL-22 gene were studied, one by C/G and another by C/T", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "IL-22 (RS1179251 C/G & RS2227485 C/T)"}, {"entity_type": "GENE", "description": "Promoter region of IL-27 gene was analyzed with two alleles \'T\' and \'G\'<SEP>The variant of IL-27 gene has two alleles \'T\' and \'G\', that may influence inflammation or immune responses", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "IL-27 (RS17855750 T/G)"}, {"entity_type": "PROTEIN", "description": "A family of proteins involved in cell signaling, inflammation, and immune responses", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "INTERLEUKIN"}, {"entity_type": "SNP", "description": "A single nucleotide polymorphism (SNP) within the IL-8 gene", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "RS4073 T/A"}, {"entity_type": "SNP", "description": "A SNP in the IL-10 promoter region", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "RS1800896 A/G"}, {"entity_type": "SNP", "description": "One of two SNPs found in the IL-22 gene, one by C/G", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "RS1179251 C/G"}, {"entity_type": "SNP", "description": "The second IL-22 polymorphism analyzed with alleles \'C\' and \'T\'", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "RS2227485 C/T"}, {"entity_type": "SNP", "description": "A promoter region SNP in the IL-27 gene", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "RS17855750 T/G"}, {"entity_type": "SNP (TRANSFORMING GROWTH FACTOR-SS1)", "description": "A genetic variation at this position affects the TGF\u03b21 gene", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "id": "RS1800469 C/T"}, {"entity_type": "GENOMIC VARIANT", "description": "The CC genotype is associated with an increased risk of Colorectal Cancer (CRC).", "source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "id": "CC GENOTYPE"}, {"entity_type": "GENOMIC VARIANT", "description": "The T allele has a slightly lower frequency in the CRC patients compared to the control group, but this does not reach statistical significance.<SEP>The T allele has a slightly lower frequency in the CRC patients compared to the control group.", "source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "id": "T ALLELE"}, {"entity_type": "GENOMIC VARIANT RELATIONSHIP", "description": "The comparison between CC genotype and T allele shows a different frequency in CRC patients.", "source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "id": "CC GENOTYPE VS T ALLELE"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "The TGF\u00df1 gene polymorphism which was studied in relation to CRC)", "entity_type": "UNKNOWN", "id": "TGFSS1 (RS1800469 C/T)"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "The TGF\u00df1 gene polymorphism which was studied in relation to CRC)", "entity_type": "UNKNOWN", "id": "TRANSFORMING GROWTH FACTOR \u03921"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "A variant of the IL-8 gene and its promoter region SNP were analyzed for its role in CRC)", "entity_type": "UNKNOWN", "id": "IL-8 (RS4073 T/A)"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "A variant of the IL-8 gene and its promoter region SNP were analyzed for its role in CRC)", "entity_type": "UNKNOWN", "id": "INTERLEUKIN 8"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "SNP analysis conducted on the promoter region of IL-10 gene and its correlation with CRC)", "entity_type": "UNKNOWN", "id": "IL-10 (RS1800896 A/G)"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "SNP analysis conducted on the promoter region of IL-10 gene and its correlation with CRC)", "entity_type": "UNKNOWN", "id": "INTERLEUKIN 10"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "Two SNPs within the IL-22 gene were studied in association with CRC)", "entity_type": "UNKNOWN", "id": "INTERLEUKIN 22"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "SNP analysis of the promoter region of IL-27 gene and its potential role in CRC)", "entity_type": "UNKNOWN", "id": "INTERLEUKIN 27"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "Decreased CT genotype in the TGF\u00df1 gene was observed to be associated with a lower risk of developing CRC)", "entity_type": "UNKNOWN", "id": "DISEASE RISK"}, {"source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "description": "The variant GG genotype and G allele were slightly associated with a decreased risk of CRC development)", "entity_type": "UNKNOWN", "id": "CRC RISK"}, {"source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "description": "Association between specific genotypes (CC) and the risk of developing Colorectal Cancer)", "entity_type": "UNKNOWN", "id": "DISEASE: COLORECTAL CANCER"}, {"source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "description": "Association between specific genotypes (CC) and the risk of developing Colorectal Cancer)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT: CC"}, {"source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "description": "The frequency difference comparison between T allele and other genotypes in CRC patients.)", "entity_type": "UNKNOWN", "id": "FREQUENCY DIFFERENCE"}, {"source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "description": "The frequency difference comparison between T allele and other genotypes in CRC patients.)", "entity_type": "UNKNOWN", "id": "GENOMIC VARIANT: T ALLELE"}, {"entity_type": "PATIENT", "description": "The group of individuals with asthma included in the analysis.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "ASTHMATIC PATIENTS"}, {"entity_type": "POPULATION SAMPLE", "description": "The total sample that includes both asthmatic patients and healthy controls for the analysis.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "NYUBAR POPULATION"}, {"entity_type": "SEXUAL DEMOGRAPHIC", "description": "A majority of the participants were women in this study.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "FEMALE"}, {"entity_type": "SMOKING STATUS", "description": "All cases and controls were non-smokers, as they were never smokers.<SEP>Another subgroup within the study, which did not show a significant association with asthma risk despite the presence of SNPs.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "NEVER-SMOKERS"}, {"entity_type": "CULTURAL OR GENETIC DEMOGRAPHIC", "description": "Differences were noted between asthmatic cases and controls in terms of self-reported race/ethnicity, which could influence genetic studies.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "SELF-REPORTED RACE/ETHNICITY"}, {"entity_type": "MEDICAL CONDITION", "description": "The primary diagnosis was asthma for all participants classified as cases.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "ASTHMA DIAGNOSIS"}, {"entity_type": "FUNCTIONAL HEALTH MEASUREMENT", "description": "Reduced lung function parameters like FEV1, FVC, and FEV1/FVC were noted in asthmatic patients compared to controls.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "LUNG FUNCTION"}, {"entity_type": "BLOOD TEST INDICATOR", "description": "Asthmatic patients had significantly higher total IgE levels than the control group.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "TOTAL IGE"}, {"entity_type": "DISEASE", "description": "More cases were determined as atopic based on their symptoms and testing results, indicating a potential genetic predisposition to allergies.<SEP>The frequency of the IL-8 genotypes differing significantly indicated a role in atopic asthma development.<SEP>The development of allergic symptoms, including chronic eczema and reactive airway disease, is indicative of an atopic phenotype often seen in people with STAT5B deficiency.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68<SEP>chunk-e323b0d4f82733a78f687e51f47b500a<SEP>chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "ATOPY"}, {"entity_type": "GENETIC TRAITS", "description": "Data on ancestry markers that help differentiate between self-reported racial groups, useful in genetic studies.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "ANCESTRY MARKERS"}, {"entity_type": "SINGLE NUCLEOTIDE POLYMORPHISM", "description": "Specific genetic variations analyzed for association with asthma susceptibility.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "TSLP SNPS"}, {"entity_type": "GENETIC VARIANT", "description": "One SNP showing nominal association with asthma, especially in ex-smokers, but not significant in never-smokers.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "SNP RS1898671"}, {"entity_type": "COMBINED DATASET OF MULTIPLE STUDIES", "description": "A large cohort combining six cohorts for replication analysis to validate the findings from NYUBAR.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "APCAT COHORTS"}, {"entity_type": "SMOKING HISTORY CATEGORY", "description": "Subgroup analyzed separately, showing significant association with SNP rs1898671 and asthma risk.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "EX-SMOKERS"}, {"entity_type": "HEALTH RISK INDICATOR", "description": "The possibility to develop asthma based on genetic and environmental factors, influenced by specific SNPs like rs1898671.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "ASTHMA SUSCEPTIBILITY"}, {"entity_type": "GENETIC SEQUENCE CONFIGURATION", "description": "This haplotype had an estimated frequency of 24% and showed a significant increased risk associated with asthma after adjusting for age, BMI, and population stratification.", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "id": "HAPLOTYPE GAGGGCAAAG"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "Confirmation of findings in a larger combined cohort, validating the associations made with NYUBAR population.)", "entity_type": "UNKNOWN", "id": "REPLICATION COHORT VALIDATION"}, {"source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "description": "Increased risk of asthma was observed when taking into account population structure differences in the analysis.)", "entity_type": "UNKNOWN", "id": "POPULATION STRATIFICATION ADJUSTMENT"}, {"entity_type": "GENOMIC VARIANT", "description": "The nine SNPs examined in PTPN2/22 found in both CD subjects and healthy controls are analyzed.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "PTPN2/22 SNP"}, {"entity_type": "GENOMIC VARIANT", "description": "One of the four SNPs found in PTPN2, rs1893217 was not significant in CD.<SEP>The most significant association was found for rs1893217 under a dominant logistic model. Carriers of at least one copy of a minor allele G (AG or GG) were 2.79 times more likely to be non-responders, compared to those with the homozygous reference genotype (AA).", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db<SEP>chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "RS1893217"}, {"entity_type": "GENOMIC VARIANT", "description": "One of the four SNPs found in PTPN2, rs2542151 was not significant in CD.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS2542151"}, {"entity_type": "GENOMIC VARIANT", "description": "One of the four SNPs found in PTPN2, rs7234029 was not significant in CD.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS7234029"}, {"entity_type": "GENOMIC VARIANT", "description": "A genomic variant detected in 57/69 (82.6%) of CD compared to 36/59 (61.0%) of healthy controls (OR = 3.03, 95%CI: 1.35-6.84, P-values < 0.05).", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS478582"}, {"entity_type": "GENOMIC VARIANT", "description": "Achieved nominal significance in association with IgM anti-PC GWAS.<SEP>Five SNPs specific to PTPN22 were considered but none was significant in CD compared to the healthy controls.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db<SEP>chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "RS2476601"}, {"entity_type": "GENOMIC VARIANT", "description": "None of these five SNPs were considered significant for PTPN22, though rs2476601 is found significantly in various inflammatory autoimmune diseases.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS2488457"}, {"entity_type": "GENOMIC VARIANT", "description": "None of the five SNPs specific to PTPN22 were considered significant for CD compared to healthy controls.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS33996649"}, {"entity_type": "GENOMIC VARIANT", "description": "One of the five SNPs specific to PTPN22, but was not found to be significant in CD versus healthy controls.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS34209542"}, {"entity_type": "GENOMIC VARIANT", "description": "Another one out of these five SNPs for PTPN22 that was not considered significant in CD compared to the healthy controls.", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "id": "RS2476599"}, {"entity_type": "GENOMIC VARIANT", "description": "Certain genotypes at the PTPN2 locus were associated with increased IFN-\u03b3 gene expression in CD subjects.<SEP>Heterozygous (TC) and minor (CC) alleles in PTPN2:rs478582 were associated with increased risk of MAPbacteremia.<SEP>The PTPN2:rs478582 gene has two types of alleles (TC) or minor (CC), and studies have shown that these can impact relative gene expression.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-6a657cd0a000b6df4db8f1305b795021<SEP>chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "PTPN2:RS478582"}, {"entity_type": "DISEASE", "description": "Healthy controls refer to individuals without the specific SNPs mentioned in PTPN2:rs478582, allowing comparisons with those who carry the gene variants.<SEP>Healthy controls refer to individuals without the specific SNPs mentioned in PTPN2:rs478582, used as a baseline for comparisons against those carrying gene variants.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "HEALTHY CONTROL"}, {"entity_type": "GENOMIC VARIANT", "description": "Individuals carrying heterozygous TC alleles for PTPN2:rs478582 show a significantly lower average relative gene expression of the CD compared to healthy controls.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "HETEROZYGOUS (TC) ALLELES"}, {"entity_type": "GENOMIC VARIANT", "description": "Similar to heterozygous TC, individuals with minor CC alleles in PTPN2:rs478582 also display notably decreased gene expression in the CD versus healthy control subjects.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "MINOR (CC) ALLELES"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "This metric measures the average level of gene expression under comparison between subjects carrying specific PTPN2:rs478582 alleles and healthy controls.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "AVERAGE RELATIVE GENE EXPRESSION"}, {"entity_type": "RELATIONSHIP STRENGTH", "description": "Studies have found statistically significant differences in average relative gene expression for both heterozygous (TC) and minor (CC) alleles of PTPN2:rs478582 when compared to the corresponding healthy control groups.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "SIGNIFICANT DIFFERENCE"}, {"entity_type": "DISEASE", "description": "The detection rate of MAP bacteremia is notably higher in individuals with heterozygous TC or minor CC alleles for PTPN2:rs478582, indicating a potential link between specific gene variants and susceptibility to infection.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "CD BACTEREMIA PRESENCE"}, {"entity_type": "COMPARISON GROUP", "description": "When comparing CD subjects carrying certain genotypes with healthy controls sharing similar genotypes (like TC or CC for PTPN2:rs478582), distinct differences in MAP bacteremia presence are observed, highlighting the impact of genetic factors.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "HEALTHY CONTROLS WITH SIMILAR SNPS"}, {"entity_type": "EFFECT STRENGTH", "description": "The odds ratio (17.5 and 21.6) quantifies the strength of association between specific PTPN2:rs478582 genotypes and increased susceptibility to MAP infection in CD subjects, as compared to healthy controls.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "OR VALUE (ODDS RATIO)"}, {"entity_type": "GENOMIC VARIANT", "description": "Individuals carrying PTPN2:rs478582 TC (heterozygous) alleles show a notably decreased gene expression in the CD compared to healthy controls.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "PTPN2:RS478582:TC ALLELES"}, {"entity_type": "GENOMIC VARIANT", "description": "Individuals with minor CC (homozygous) PTPN2:rs478582 alleles also display significantly lower gene expression in the CD versus healthy control subjects.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "PTPN2:RS478582:CC ALLELES"}, {"entity_type": "MEDICAL CONDITION", "description": "The presence of bacteremia due to Mycobacterium avium subspecies paratuberculosis (MAP) infection is notably higher in CD subjects with certain genotypes for PTPN2:rs478582.", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "id": "MAP BACTEREMIA"}, {"entity_type": "GENOMIC VARIANT", "description": "Heterozygous (GA) or minor (AA) alleles in PTPN22:rs2476601 were rare and associated with low risk of MAPbacteremia.<SEP>SNPs (Single Nucleotide Polymorphisms) at this locus were found to influence T-cell functionality and response.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-6a657cd0a000b6df4db8f1305b795021", "id": "PTPN22:RS2476601"}, {"entity_type": "GENE", "description": "The expression levels of PTPN2 gene were significantly lower in CD subjects compared to healthy controls regardless of MAPbacteremia status.", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "id": "PTPN2"}, {"entity_type": "DISEASE", "description": "Presence of MAPbacteremia was associated with different allele combinations in PTPN2:rs478582 and PTPN22:rs2476601, which influenced the expression levels of PTPN2/PTPN22 and IFN-\u03b3.<SEP>The presence of MAPbacteremia was considered as an environmental factor that affects T-cell response in CD subjects.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-6a657cd0a000b6df4db8f1305b795021", "id": "MAPBACTEREMIA"}, {"entity_type": "MOLECULAR MECHANISM/", "description": "Long-term dietary patterns may lead to epigenetic modifications that affect gene expression related to cardiovascular health.<SEP>Modifications linked to poor dietary patterns that alter the metabolic pathway of nitric oxide synthesis.<SEP>Alteration/Factor/Context/", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-6a657cd0a000b6df4db8f1305b795021<SEP>chunk-bb351b97d99fce93d17df522db92dda7", "id": "EPIGENETIC MODIFICATION"}, {"entity_type": "GENE", "description": "The TP53 gene was mentioned as part of the study\'s analysis on p53 expression and its relation to CD.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "id": "TP53"}, {"entity_type": "PROTEIN", "description": "p53 protein levels were measured in T-cells from both healthy controls and CD subjects, with a focus on its role in response.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "id": "P53"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Phytohemagglutinin (PHA) was used as an activator of T-cells for measuring their proliferation response.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "id": "PHA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "A modified form of Mycobacterium avium subspecies paratuberculosis (MAP PPD-like) was utilized in the study to test T-cell response.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "id": "MAP PPD-LIKE"}, {"entity_type": "GENE", "description": "This entity plays a critical role in cardiovascular health by regulating nitric oxide production.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "id": "NOS1"}, {"entity_type": "METABOLIC PATHWAY", "description": "This central pathway is influenced by genetic and environmental factors, affecting cardiovascular health.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 39}]", "id": "NITRIC OXIDE SYNTHESIS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The diet is a significant environmental factor modulating the relationship between genes and cardiovascular health.", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "clusters": "[{level: 0, cluster: 4}]", "id": "DIET"}, {"source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "description": "The analysis of SNPs in both PTPN2/22 shows significant differences between CD subjects and healthy controls, suggesting a potential role of these genetic variations in the development or manifestation of CD.)(entity", "entity_type": "UNKNOWN", "id": "SNPS COMPARISON"}, {"source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "description": "The odds ratio quantifies how much more likely it is for CD subjects with specific PTPN2:rs478582 genotypes to develop MAP infection, indicating the strength of their association.)", "entity_type": "UNKNOWN", "id": "STRENGTH OF ASSOCIATION"}, {"source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "description": "Decrease in PTPN2 expression level between healthy controls and CD subjects regardless of MAPbacteremia presence.)", "entity_type": "UNKNOWN", "id": "PTPN2 EXPRESSION"}, {"source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "description": "Increased IFN-\u03b3 expression upon combined effects of specific SNPs, MAPbacteremia presence in CD subjects compared to non-infected controls.)", "entity_type": "UNKNOWN", "id": "IFN-\u0393 EXPRESSION"}, {"source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "description": "Variants influencing folate metabolism pathways are associated with increased risk for certain diseases, including cardiovascular disease.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 40}]", "id": "GENOMIC VARIANT IN FOLATE METABOLISM PATHWAY"}, {"source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "description": "Interactions between nutrients and specific molecular processes in the body can have significant effects on physiological functions like metabolism and disease risk.)", "entity_type": "UNKNOWN", "id": "MOLECULAR PROCESS"}, {"source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0<SEP>chunk-614333487e4b92705503909d34155179", "description": "T cells respond to neoepitopes when presented by MHC molecules on the surface of cancer cells.<SEP>The presence of MAPbacteremia was found to impact T-cell response and proliferation in CD patients.)", "entity_type": "BIOLOGICAL PROCESS", "id": "T-CELL RESPONSE"}, {"source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "description": "PHA stimulation led to an increased T-cell proliferation response in both CD subjects and healthy controls.)", "entity_type": "UNKNOWN", "id": "T-CELL PROLIFERATION"}, {"entity_type": "GENE", "description": "Data demonstrate higher expression of PTPN22 in Treg compared to Tconv cells, suggesting potential influence on disease risk via Treg function.<SEP>PTPN22 is a gene that encodes the LYP protein and plays a role in regulating the activation state of lymphocytes.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d<SEP>chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "PTPN22"}, {"entity_type": "CELL TYPE", "description": "CD4+ T cells are a type of immune cell that can be divided into different subsets based on their differentiation state, including na\u00efve, memory, and regulatory cells.", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "CD4+ T CELL SUBSET"}, {"entity_type": "T CELL SUBSET", "description": "Na\u00efve CD4+ Tconv (central memory) cells are at rest and have not been previously exposed to an antigen.", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "NA\u00cfVE TCONV"}, {"entity_type": "T CELL SUBSET", "description": "Memory CD4+ Tcells were sorted from leukopacks determined to be homozygous for the T1D non-risk allele, which encodes LYP-620R.", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "MEMORY TCONV"}, {"entity_type": "T CELL SUBSET", "description": "Tregs are a subset of CD4+ T cells with immunosuppressive functions that play a role in maintaining immune tolerance and controlling autoimmune responses.", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "id": "REGULATORY T (TREG)"}, {"entity_type": "TRANSCRIPT", "description": "CD69 is dynamically expressed following Tconv activation.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "CD69"}, {"entity_type": "TRANSCRIPT", "description": "ICOS expression kinetics involve an intensification followed by a reduction of surface expression.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "ICOS"}, {"entity_type": "TRANSCRIPT", "description": "CD25 is upregulated in this time period.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "CD25"}, {"entity_type": "TRANSCRIPT", "description": "CD226 exhibits activation-induced expression kinetics.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "CD226"}, {"entity_type": "TRANSCRIPT", "description": "CTLA-4 is involved in the Tregs\' activation-induced expression kinetics.", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "CTLA-4"}, {"entity_type": "GENE", "description": "Data suggest LYP gene variants influence cytokine secretion and T-cell regulation.<SEP>LYP is a gene that encodes tyrosine phosphatase.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d<SEP>chunk-d53a9f0fba772a7b52b906f476f0aef7", "id": "LYP"}, {"entity_type": "CYTOKINE", "description": "Studies show IFN\u03b3 secretion was not altered by overexpression of LYP variants, agreeing with Ptpn22 knockout mice studies.<SEP>Molecular Interaction", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "IFN\u0393"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Data imply Th1 signature in autoimmune disorders associated with the rs2476601 risk variant; IFN\u03b3 secretion was not altered by overexpression of either LYP variant.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "TH1 SIGNATURE"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Hypothesis that PTPN22\'s risk variant affects disease risk through altering Treg-mediated suppression is tested and supported.<SEP>Molecular Interaction/Environmental Factor/", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "TREG SUPPRESSION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Transduced Treg showed altered capacity for Tresp suppression when overexpressing LYP variants; 620W variant showed reduced effect compared to 620R.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "TCELL SUPPRESSION"}, {"entity_type": "ENVIRONMENTAL FACTOR/", "description": "Lifestyle Factor", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "TCONV INTRINSIC SUSCEPTIBILITY"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Overexpression of LYP variants on transduced Tconv led to differences in Tresp proliferation; 620R showed significant repression while 620W showed dramatically reduced effect across the range.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "TRESP PROLIFERATION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "LYP variant\'s impact was assessed on Treg suppressive capacity; LYP-620W slightly enhanced suppressive function compared to LYP-620R and eGFP reporter Tregs, but not due to differences in Treg proliferation.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "SUPPRESSION BY TREGS"}, {"entity_type": "PHYSIOLOGICAL PROCESS/", "description": "Molecular Interaction", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "TREG PROLIFERATION"}, {"entity_type": "GENOMIC VARIANT", "description": "Evidence shows that variant 620R results in similar levels of effector cytokine secretion as compared to the other variant.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "620R VARIANT"}, {"entity_type": "GENOMIC VARIANT", "description": "Findings indicate that variant 620W leads to a somewhat lesser degree of inhibition compared to variant 620R, but this difference was not significant between variants.", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "620W VARIANT"}, {"entity_type": "AUTOIMMUNE DISORDERS", "description": "Molecular Interaction/Environmental Factor/", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "TH1 SIGNATURE/"}, {"entity_type": "GENE/", "description": "Genomic Variant/", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "PTPN22/LYP"}, {"entity_type": "CELL TYPE/", "description": "Physiological Process/", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "TRESP (T-RESPONDING CELLS)"}, {"entity_type": "CELL TYPE/", "description": "Physiological Process/", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "TREG (REGULATORY T CELLS)"}, {"entity_type": "TRESP PROLIFERATION/", "description": "Molecular Interaction/Environmental Factor/", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "id": "SUPPRESSION BY TREGS/"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "CD8+ T cell subset", "entity_type": "UNKNOWN", "id": "BIOLOGICAL FUNCTION<|THE PTPN22 GENE CONTRIBUTES TO THE REGULATION OF LYMPHOCYTE ACTIVATION STATE THROUGH MODULATION OF TCR SIGNALING PATHWAYS LIKE CA2+ FLUX AND PERK SIGNALING.](ENTITY"}, {"source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "description": "CD8+ T cell subset", "entity_type": "UNKNOWN", "id": "PTPN22 GENE"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "The kinetics of CD69 involve an intensification followed by a reduction of surface expression.)", "entity_type": "UNKNOWN", "id": "ACTIVATION MARKER KINETICS"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "The expression of CD25 was observed when normalized to internal mock-transfected controls.)", "entity_type": "UNKNOWN", "id": "INTERNAL MOCK-TRANSFECTED CONTROLS"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "CD226 on Tregs exhibited a unique kinetics where it waned before increasing to a level higher than baseline.)", "entity_type": "UNKNOWN", "id": "TREGS ACTIVATED KINETICS"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "The PTPN22 risk variant permits proliferation in Tconv, but not in Treg.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 40}]", "id": "LYP VARIANTS IMPACT"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "Overexpression of LYP-620R reduced APC-mediated proliferation of Tconv and Treg.)", "entity_type": "UNKNOWN", "id": "ACTIVATION INDUCED PROLIFERATION"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "Overexpression of LYP-620R reduced APC-mediated proliferation of Tconv and Treg.)", "entity_type": "UNKNOWN", "id": "LYP-620R IMPACT"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "The LYP-620W variant was capable of blocking Treg proliferation to a similar degree as the LYP-620R variant.)", "entity_type": "UNKNOWN", "id": "LYP-620W IMPACT"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "Overexpression of PTPN22 influences T cell cytokine production, but mitogenic activation by PMA and ionomycin was not altered.)", "entity_type": "UNKNOWN", "id": "CELL ACTIVATION BY PMA AND IONOMYCIN"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "Overexpression of PTPN22 influences T cell cytokine production, but mitogenic activation by PMA and ionomycin was not altered.)", "entity_type": "UNKNOWN", "id": "LYP VARIANTS"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7<SEP>chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "description": "The net accumulation of IL-2 was not altered by overexpression of the LYP-620R variant, but increased at the 24-hour time point by the LYP-620W variant.)<SEP>The secretion of IFN\u03b3 and IL-2 cytokines by TAG72-BB\u03b6 CAR T cells is indicative of their anti-tumor immune response.", "entity_type": "ACTIVITY", "id": "CYTOKINE PRODUCTION"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "The net accumulation of IL-2 was not altered by overexpression of the LYP-620R variant, but increased at the 24-hour time point by the LYP-620W variant.)", "entity_type": "UNKNOWN", "id": "IL-2 SECRETION"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "The secretion of IL-4 was robustly inhibited for the first 48 hours, but to a somewhat lesser degree by LYP-620W at the 72-hour time point.)", "entity_type": "UNKNOWN", "id": "IL-4 SECRETION"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "The secretion of IL-5 was not altered by either variant of LYP.)", "entity_type": "UNKNOWN", "id": "IL-5 SECRETION"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "The secretion of IL-9 was robustly inhibited for the first 48 hours and to a somewhat lesser degree by LYP-620W at the 72-hour time point.)", "entity_type": "UNKNOWN", "id": "IL-9 SECRETION"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "The secretion of IL-10 was robustly inhibited for the first 48 hours, but to a somewhat lesser degree by LYP-620W at the 72-hour time point.)", "entity_type": "UNKNOWN", "id": "IL-10 SECRETION"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "The secretion of IL-13 was robustly inhibited for the first 48 hours and to a somewhat lesser degree by LYP-620W at the 72-hour time point.)", "entity_type": "UNKNOWN", "id": "IL-13 SECRETION"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "IFN\u03b3 secretion was not altered by overexpression of either variant of LYP.)", "entity_type": "UNKNOWN", "id": "IFN\u0393 SECRETION"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "CD69 is an activation marker expressed on T cells.)", "entity_type": "UNKNOWN", "id": "ACTIVATION MARKER"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "CD25, also known as IL-2 receptor alpha chain (CD152), is upregulated on activated T cells and plays a role in costimulation during immune responses.)", "entity_type": "UNKNOWN", "id": "CO-STIMULATORY LIGAND"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "CTLA-4 is a co-inhibitory receptor that downregulates the activity of T lymphocytes during autoimmune diseases and transplantation.)", "entity_type": "UNKNOWN", "id": "CO-INHIBITORY LIGAND"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "The association between PTPN22 (LYP gene) variations and certain autoimmune disorders, such as ankylosing spondylitis or inflammatory bowel disease.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 40}]", "id": "AUTOIMMUNE DISORDER"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "LYP variants influence T cell proliferation during activation.)", "entity_type": "UNKNOWN", "id": "REGULATED BY LYP VARIANTS"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7<SEP>chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "description": "LYP variants influence T cell proliferation during activation.)<SEP>The process by which T cells increase in number, often seen during immune responses or cancer treatments.", "entity_type": "CELLULAR PROCESS", "id": "T CELL PROLIFERATION"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "Lymphoproteins (LYP, PTPN22) impact the secretion of various cytokines, modulating Th1 and Th2 responses.)", "entity_type": "UNKNOWN", "id": "CYTOKINE SECRETION"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "Lymphoproteins (LYP, PTPN22) impact the secretion of various cytokines, modulating Th1 and Th2 responses.)", "entity_type": "UNKNOWN", "id": "MODULATED BY LYP VARIANTS"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "CD4+ effector T cells play a crucial role in regulating host immune responses through the production of cytokines and other effectors.)", "entity_type": "UNKNOWN", "id": "CD4+ EFFECTOR T CELLS"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "CD4+ effector T cells play a crucial role in regulating host immune responses through the production of cytokines and other effectors.)", "entity_type": "UNKNOWN", "id": "REGULATE HOST IMMUNE RESPONSE"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "Antigen-presenting cells (APCs) are essential for the activation and proliferation of T cells during immune response stimulation.)", "entity_type": "UNKNOWN", "id": "APCS"}, {"source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "description": "Antigen-presenting cells (APCs) are essential for the activation and proliferation of T cells during immune response stimulation.)", "entity_type": "UNKNOWN", "id": "MITOGENIC ACTIVATION"}, {"source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "description": "LYP Variants", "entity_type": "UNKNOWN", "id": "TCONV INTRINSIC SUSCEPTIBILITY/"}, {"source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "description": "LYP Variants/Molecular Interaction/<SEP>LYP Variants", "entity_type": "UNKNOWN", "id": "TRESP PROLIFERATION/"}, {"source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "description": "Environmental Factor/", "entity_type": "UNKNOWN", "id": "LYP VARIANTS/TH1 SIGNATURE/AUTOIMMUNE DISORDERS"}, {"source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "description": "Environmental Factor/", "entity_type": "UNKNOWN", "id": "MOLECULAR INTERACTION/"}, {"source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "description": "LYP Variants/Molecular Interaction/", "entity_type": "UNKNOWN", "id": "TREG SUPPRESSION/"}, {"source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "description": "LYP Variants/Molecular Interaction/", "entity_type": "UNKNOWN", "id": "TCONV INTRINSIC SUSCEPTIBILITY/"}, {"entity_type": "GENE", "description": "The TAG72 gene encodes a protein associated with ovarian cancer.<SEP>The TAG72 gene expresses TAG72 protein in human ovarian cancer ascites.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0<SEP>chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "TAG72"}, {"entity_type": "TRANSCRIPT", "description": "CAR T cells expressing the 4-1BB co-stimulatory domain were developed for targeting ovarian cancer cells.<SEP>Chimeric Antigen Receptor (CAR) T cells are used for CAR-mediated killing of tumor cells.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0<SEP>chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CAR T CELLS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The 4-1BB intracellular co-stimulatory domain enhances CAR T cell activation and anti-tumor activity.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "4-1BB INTRACELLULAR CO-STIMULATORY DOMAIN"}, {"entity_type": "PROTEIN", "description": "The humanized scFv CC49 component of the CAR provides a specific affinity to the TAG72 antigen on cancer cells.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "HUMANIZED SCFV CC49"}, {"entity_type": "PROTEIN", "description": "This spacer is part of the engineered CAR structure, facilitating its binding and signaling properties without affecting cell surface expression.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "HUMAN IGG4 FC EXTRACELLULAR SPACER"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The CD4 transmembrane domain contributes to the proper surface expression and stability of the CAR on T cells.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CD4 TRANSMEMBRANE DOMAIN"}, {"entity_type": "PROTEIN", "description": "CD19t is used for tracking TAG72-BB\u03b6 CAR T cells during manufacturing and clinical trials.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CD19T (CD19 TRUNCATED) FOR CELL TRACKING"}, {"entity_type": "TRANSCRIPT", "description": "These are the control T cells that have not been transduced with the CAR construct, used to compare against CAR T cells.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "MOCK (UNTRANSDUCED) T CELLS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The induction of CD137 expression on TAG72-BB\u03b6 CAR T cells is a measure of their activation against the TAG72 antigen.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CD137 EXPRESSION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "CD69 expression on the surface of T cells is an early indicator of T cell activation and response to stimulation.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CELL SURFACE CD69 EXPRESSION"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Interferon-gamma (IFN\u03b3) release by T cells is a key marker for their cytotoxic function in immune responses.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "IFN\u0393 RELEASE"}, {"entity_type": "PARAMETER", "description": "The E:T ratio determines the effectiveness of CAR T cell killing activity; higher ratios indicate reduced killing efficiency.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "E:T RATIO (EFFECTOR:TARGET RATIO)"}, {"entity_type": "ACTIVITY", "description": "TAG72-BB\u03b6 CAR T cells show potent killing against ovarian cancer cells expressing the TAG72 antigen.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CELL KILING ACTIVITY AGAINST TAG72-POSITIVE CELLS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Human ovarian cancer ascites provides a source of cells that are used to assess the in vitro activity and specificity of TAG72-BB\u03b6 CAR T cells.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "OVARIAN CANCER ASCITES"}, {"entity_type": "DISEASE", "description": "Tumor cells are the targets of TAG72-BB\u03b6 CAR T cells, specifically those expressing the TAG72 antigen.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "TUMOR CELLS"}, {"entity_type": "ACTIVITY", "description": "The activity of TAG72-BB\u03b6 CAR T cells against ovarian cancer cell lines in vitro.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "A-SPECIFIC ACTIVITY"}, {"entity_type": "BIOLOGICAL PROCESS", "description": "The process by which living organisms build proteins using genetic information from DNA or RNA.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "PROTEIN SYNTHESIS"}, {"entity_type": "CANCER TYPE", "description": "A specific type of cancer that originates in the cells lining the ovaries.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "OVARIAN CANCER CELLS"}, {"entity_type": "RESEARCH STAGE", "description": "The phase of research conducted prior to clinical trials, where safety and efficacy are tested in laboratory settings.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "PRECLINICAL STUDIES"}, {"entity_type": "CLINICAL TRIAL PHASES", "description": "The first stage of human testing for new drugs or treatments, focusing on safety and dosage.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "PHASE I CLINICAL TRIALS"}, {"entity_type": "CLINICAL TRIAL PHASES", "description": "Evaluates the efficacy and side effects of a drug in a larger patient group than Phase I.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "PHASE II CLINICAL TRIALS"}, {"entity_type": "CLINICAL TRIAL PHASES", "description": "Large-scale studies that confirm clinical benefit, monitor adverse reactions, and assess overall effectiveness.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "PHASE III CLINICAL TRIALS"}, {"entity_type": "SAFETY CONCERN", "description": "Potential harmful effects that may occur when taking medications, including at therapeutic doses.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "DRUG TOXICITY"}, {"entity_type": "IMMUNE RESPONSE", "description": "The body\'s defense mechanism against pathogens or foreign substances using antibodies.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "ANTIBODY-MEDIATED RESPONSE"}, {"entity_type": "MOLECULAR COMPONENT", "description": "Artificial receptors engineered to recognize and bind to specific antigens on cancer cells.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CHIMERIC ANTIGEN RECEPTORS (CARS)"}, {"entity_type": "IMMUNE SYSTEM COMPONENT", "description": "A part of the immune system that develops memory after exposure to pathogens, enhancing future responses.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "ADAPTIVE IMMUNITY"}, {"entity_type": "OUTCOME MEASURE", "description": "The reduction or elimination of tumor size following treatment, indicating therapeutic effectiveness.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "TUMOR REGRESSION"}, {"entity_type": "DISEASE", "description": "A set of statistical techniques used to estimate the duration of time until an event occurs (e.g., death, disease recurrence).<SEP>The survival of mice treated with TAG72-BB\u03b6 CAR T cells was analyzed to evaluate the efficacy of therapy.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0<SEP>chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "SURVIVAL ANALYSIS"}, {"entity_type": "OUTCOME MEASURE", "description": "Assessments conducted to measure patients\' subjective well-being and functional status.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "QUALITY OF LIFE SURVEYS"}, {"entity_type": "OUTCOME MEASURE", "description": "The proportion of patients who respond favorably to a particular treatment regimen.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "CLINICAL RESPONSE RATE"}, {"entity_type": "DISEASE STATUS INDICATOR", "description": "A worsening condition characterized by tumor growth and spread in the body.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "TUMOR PROGRESSION"}, {"entity_type": "THERAPEUTIC MODALITY", "description": "Drugs that target immune checkpoints, enhancing T cell activity to combat cancer.", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "id": "IMMUNE CHECKPOINT INHIBITORS"}, {"entity_type": "GENOMIC VARIANT", "description": "Three patients with ovarian cancer have genetic variants that influence TAG72 expression.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "OAS3, OAS4, OAS7"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Flow cytometry is used to quantify the percentage of TAG72 protein in ovarian cancer cells.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "FLOW CYTOMETRY"}, {"entity_type": "NUTRIENT", "description": "Freshly thawed ascites from patients with ovarian cancer are used for vitro assays.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "FRESHLY THAWED ASCITES"}, {"entity_type": "MOLECULAR INTERACTION", "description": "CAR T cells exhibit cytolytic activity against TAG72-positive ovarian cancer cells.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "CYTOLYTIC ACTIVITY"}, {"entity_type": "DISEASE", "description": "OVCAR3 is a type of ovarian cancer cell line used in vitro assays.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "OVCAR3"}, {"entity_type": "TRANSCRIPT", "description": "The OVCAR3 cells were lentivirally transduced to express eGFP/ffluc transcripts for tracking tumor growth via non-invasive optical imaging.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "EGFP/FFLUC"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Intraperitoneal injection of TAG72-BB\u03b6 CAR T cells into mice for the treatment of peritoneal ovarian tumors.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "INTRAPERITONEAL INJECTION"}, {"entity_type": "REGIONAL INTRAPERITONEAL DELIVERY", "description": "Regional intraperitoneal delivery of TAG72-BB\u03b6 CAR T cells exhibits potent anti-tumor activity in mice with peritoneal ovarian tumors.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "I.P."}, {"entity_type": "I.V.", "description": "Limited anti-tumor responses were observed when TAG72-BB\u03b6 CAR T cells were delivered via intravenous injection.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "INTRAVENOUS INJECTION"}, {"entity_type": "THERAPY", "description": "Mice treated with Mock showed limited to no therapeutic benefits.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "MOCK TREATMENT"}, {"entity_type": "GENOMIC VARIANT", "description": "Immune compromised NSG (NOD/SCID gamma) mice were used for establishing tumor xenografts.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "NSG MICE"}, {"entity_type": "TRANSCRIPT", "description": "Mice treated with Mock cells did not show any therapeutic benefits, which contrasts the response of CAR T cell treatment.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "MOCK CELLS"}, {"entity_type": "THERAPY", "description": "Regional intraperitoneal delivery of TAG72-BB\u03b6 CAR T cells was effective against peritoneal ovarian tumors.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "REGIONAL I.P. DELIVERY"}, {"entity_type": "THERAPY", "description": "Intravenous administration of CAR T cells showed limited efficacy compared to regional i.p. delivery.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "I.V."}, {"entity_type": "MOLECULAR INTERACTION", "description": "CAR T cells produced IFN\u03b3 and IL-2 against cancerous cells, but mock T cells did not show similar activity.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "CAR T CELLS AND MOCK T CELL"}, {"entity_type": "DISEASE", "description": "OVCAR3 tumors were established by intraperitoneal injection of OVCAR3 cells in NSG mice to mimic peritoneal ovarian tumors.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "OVCAR3 TUMOR"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Non-invasive optical imaging was used for tracking tumor growth after treatment with CAR T cells.", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "id": "OPTICAL IMAGING"}, {"entity_type": "THERAPY", "description": "Compared with a single dose of TAG72-BB\u03b6 CAR T cells, repeat dosing over the course of 1 month demonstrated more durable anti-tumor responses.<SEP>Repeating dosing of TAG72-BB\u03b6 CAR T cells over a month showed more durable anti-tumor responses compared to single dosing.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf<SEP>chunk-183db5c908df609bb5289896b4ba09d0", "id": "REPEAT DOSING"}, {"entity_type": "TRANSCRIPT", "description": "In this study, the overall survival was extended significantly in mice that received repeat doses of TAG72-BB\u03b6 CAR T cells.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "TAG72-BB\u0396 CAR T CELLS"}, {"entity_type": "DRUG", "description": "The study evaluated the therapeutic efficacy of Antibody-Drug Conjugates (ADCs) in ovarian cancer xenograft models.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "ANTIBODY-DRUG CONJUGATES (ADCS)"}, {"entity_type": "DISEASE", "description": "This model showed reduced numbers, expansion and persistence of CAR T cells following repeat dosing compared to the OVCAR3 model, suggesting potential suppressive mechanisms affecting T cell function.<SEP>When plotted as relative tumor growth kinetics, repeat dosing promoted more extensive tumor regression as well as more durable control of tumors compared with single dosing.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "OV90 MODEL"}, {"entity_type": "DISEASE", "description": "The study indicated that in this model, higher survival benefits were associated with repeated TAG72-BB\u03b6 CAR T cell dosing over single dose compared to the OV90 model.<SEP>The study showed that TAG72-BB\u03b6 CAR T cells exhibit potent anti-tumor activity in ovarian cancer xenograft models.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "OVCAR3 MODEL"}, {"entity_type": "TRANSCRIPT", "description": "The study evaluated the therapeutic efficacy of Antibody-Drug Conjugates (ADCs) and compared it with mock treatment in ovarian cancer xenograft models.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "MOCK"}, {"entity_type": "TRANSCRIPT", "description": "The expression of antigens on tumors was heterogeneous following mock treatment versus infusion with TAG72-BB\u03b6 CAR T cells, impacting the efficacy in different time points.", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "id": "T CELL INFUSION"}, {"source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "description": "CD69 expression on the surface of T cells correlates with early stages of T cell activation and response to stimulation.)", "entity_type": "UNKNOWN", "id": "CAR T CELL ACTIVATION"}, {"source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "description": "Cytotoxic T cells, such as those expressing TAG72-BB\u03b6 CARs, release IFN\u03b3 in response to antigen presentation by tumor cells.)", "entity_type": "UNKNOWN", "id": "ANTIGEN-SPECIFIC RESPONSE"}, {"source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "description": "The E:T ratio impacts the efficiency of cell killing activity; higher ratios can lead to reduced killing effectiveness.)", "entity_type": "UNKNOWN", "id": "KILLING EFFICIENCY"}, {"source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "description": "TAG72-BB\u03b6 CAR T cells show high efficacy against ovarian cancer cells expressing the TAG72 antigen in vitro.)", "entity_type": "UNKNOWN", "id": "IN VITRO ASSAYS"}, {"source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "description": "The secretion of IFN\u03b3 and IL-2 by CAR T cells is indicative of their robust cytotoxic function against tumor targets.)", "entity_type": "UNKNOWN", "id": "ANTIBODY CYTOTOXIC FUNCTION"}, {"source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "description": "Ovarian cancer cells expressing the TAG72 antigen are targeted by TAG72-BB\u03b6 CAR T cells to demonstrate their specificity and potential clinical application.)", "entity_type": "UNKNOWN", "id": "DISEASE TARGET"}, {"source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "description": "Repeating the dose of TAG72-BB\u03b6 CAR T cells over a month improved therapeutic responses compared to single dosing.)", "entity_type": "UNKNOWN", "id": "ANTI-TUMOR EFFICACY"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "The study demonstrated that repeat dosing of TAG72-BB\u03b6 CAR T cells provided greater control of tumors compared to a single dose, based on comparison between the OVCAR3 model and the OV90 model.)", "entity_type": "UNKNOWN", "id": "OVC90 MODEL"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Antigen loss or escape following mock treatment was observed in tumor recurrences at early time points from mice treated with TAG72-BB\u03b6 CAR T cells.)", "entity_type": "UNKNOWN", "id": "TUMOR RECURRENCES"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "The expression of antigens on tumors varied following mock treatment versus the infusion of TAG72-BB\u03b6 CAR T cells in mice, with different patterns noted at 12 weeks post-infusion.)", "entity_type": "UNKNOWN", "id": "MOCK AND TAG72-BB\u0396 CAR T CELL TREATED MICE"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Reduced numbers, expansion and persistence of CAR T cells were observed in the blood of OV90-bearing mice compared to OVCAR3 model, suggesting potential suppressive mechanisms affecting T cell function and overall CAR T cell efficacy.)", "entity_type": "UNKNOWN", "id": "T CELL PERSISTENCE"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "The major resistance mechanism against CAR T cell therapy is tumor antigen heterogeneity that promotes eventual antigen loss or escape.)", "entity_type": "UNKNOWN", "id": "ANTIGEN HETEROGENEITY"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "Antigen loss or escape from tumor cells was observed more frequently at early time points after mock treatment compared to repeated TAG72-BB\u03b6 CAR T cell dosing.)", "entity_type": "UNKNOWN", "id": "DISEASE RESISTANCE"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "The survival benefit associated with repeated TAG72-BB\u03b6 CAR T cell infusion over the OV90 model was more pronounced compared to OVCAR3 model due to differences in treatment timing and tumor dynamics.)", "entity_type": "UNKNOWN", "id": "SURVIVAL BENEFIT"}, {"source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "description": "The survival benefit associated with repeated TAG72-BB\u03b6 CAR T cell infusion over the OV90 model was more pronounced compared to OVCAR3 model due to differences in treatment timing and tumor dynamics.)", "entity_type": "UNKNOWN", "id": "T CELL INFUSION TIMING"}, {"entity_type": "POPULATION", "description": "A population sample of individuals used for genetic analysis.<SEP>A total of 1,144 Hutterites (613 females, 53.6%) aged 6\u201389 years were included.", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a<SEP>chunk-7efd4f037425eeac77e329347b16beb2", "id": "HUTTERITES"}, {"entity_type": "GENE", "description": "An association was found with a SNP located downstream of the IL37 gene on chromosome 2q13 related to FEV1.", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "id": "INTERLEUKIN-37 GENE"}, {"entity_type": "GENE", "description": "An intron of the ASXL3 gene on chromosome 18q12.1 was found to be associated with FVC.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "id": "ASXL3 GENE"}, {"entity_type": "GENOMIC VARIANT", "description": "The mean residual FEV1/FVC decreased with increasing number of \'low FEV1/FVC alleles\', consistent with an additive genetic architecture.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 38}]", "id": "LOW FEV1/FVC ALLELES"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "The mean residual FEV1/FVC was calculated for Hutterites in each bin to measure lung function.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "FEV1/FVC"}, {"entity_type": "MOLECULAR INTERACTION", "description": "An additive genetic architecture was consistent with the decreasing mean residual FEV1/FVC.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "ADDITIVE GENETIC ARCHITECTURE"}, {"entity_type": "SOFTWARE TOOL", "description": "The GRAIL algorithm was used to mine the Gene Ontology database for functional connections between genes.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "GRAIL ALGORITHM"}, {"entity_type": "DATABASE", "description": "The Gene Ontology database was mined using the GRAIL algorithm to find functional connections.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "GENE ONTOLOGY"}, {"entity_type": "TRANSCRIPT", "description": "Three clusters of \u03b2-defensin genes were identified in the subset of significantly related genes.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "\u0392-DEFENSIN GENES"}, {"entity_type": "TRANSCRIPT", "description": "One chemokine gene (CCL18) was part of the identified functional connections with SNPs.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "CCL18"}, {"entity_type": "TRANSCRIPT", "description": "Another chemokine gene (CXCL12) was connected to the \u03b2-defensin genes and TNFRSF13B in the GRAIL analysis.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "CXCL12"}, {"entity_type": "TRANSCRIPT", "description": "The gene TNFRSF13B was identified as functionally related through the GRAIL algorithm.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "TNFRSF13B"}, {"entity_type": "GENOMIC VARIANT", "description": "A replicated locus (THSD4-UACA-TLE3) was not functionally connected to any other genes defined by the 80 SNPs.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 38}]", "id": "THSD4-UACA-TLE3"}, {"entity_type": "GENOMIC VARIANT", "description": "Similarly, another replicated locus (C10orf11) did not show functional connections with other identified genes.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 38}]", "id": "C10ORF11"}, {"entity_type": "GENOMIC VARIANT", "description": "A SNP at the CCL23-CCL18 locus was significantly connected to \u03b2-defensin genes, as well as CXCL12 and TNFRSF13B.<SEP>A specific SNP at this genomic locus was significantly connected to \u03b2-defensin genes and TNFRSF13B, as well as CXCL12.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "clusters": "[{level: 0, cluster: 11}]", "id": "CCL23-CCL18 LOCUS"}, {"entity_type": "EXPLAINED VARIANCE", "description": "The six SNPs together explained 5.8% of the residual variance in FEV1/FVC in the Hutterites.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "5.8% OF RESIDUAL VARIANCE"}, {"entity_type": "GENETIC STUDY", "description": "A genetic study conducted on a population sample, represented by Fig. E3b in the Online Repository.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "HUTTERITE GWAS"}, {"entity_type": "TRANSCRIPT", "description": "Three clusters of \u03b2-defensin genes were part of the functional connections found in the analysis.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "\u0392-DEFENSIN GENE CLUSTER"}, {"entity_type": "TRANSCRIPT", "description": "Chemokine genes CCL18 and CXCL12 were identified among functionally related transcripts.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "CHEMOKINE GENE"}, {"entity_type": "TRANSCRIPT", "description": "The TNFRSF13B gene showed functional links with \u03b2-defensin genes and CCL18 through the GRAIL analysis.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "id": "TNFRSF13B GENE"}, {"entity_type": "GENOMIC VARIANT", "description": "One replicated genetic locus (THSD4-UACA-TLE3) was examined but did not show functional connections to other genes identified by the 80 SNPs.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "clusters": "[{level: 0, cluster: 11}]", "id": "THSD4-UACA-TLE3 LOCUS"}, {"entity_type": "GENOMIC VARIANT", "description": "Another replicated locus (C10orf11), despite being one of two replicated loci, did not display functional connections with other genes defined by the 80 SNPs.", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 38}]", "id": "C10ORF11 LOCUS"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "A genome-wide significant association was found at the THSD4-UACA-TLE3 locus on chromosome 15q23.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 11}]", "id": "GENETIC LOCATION"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "The FEV1/FVC ratio showed associations with SNPs at several loci, including the THSD4-UACA-TLE3 and C10orf11 loci.)", "entity_type": "UNKNOWN", "id": "LUNG FUNCTION TRAIT"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "SNPs associated with FEV1/FVC were found downstream of the C10orf11 gene on chromosome 10q22.3.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 43}]", "id": "C10ORF11 GENE"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "The CHAF1B locus on chromosome 21q22.13 showed evidence for association with SNPs related to lung function.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 11}]", "id": "CHAF1B LOCUS"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "The PITPNC1 locus on chromosome 17q24.2 showed evidence for association with SNPs related to lung function.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 11}]", "id": "PITPNC1 LOCUS"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "A SNP in this gene was associated with FEV1 and FVC.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 23}, {level: 1, cluster: 43}]", "id": "IL37 GENE"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "A multimarker model including all SNPs with P < 10\u22123 was performed using LASSO regression.)", "entity_type": "UNKNOWN", "id": "MULTIMARKER MODELING"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "The minimum set of 80 SNPs providing the best predictive value for FEV1/FVC were identified in the GWAS.)", "entity_type": "UNKNOWN", "id": "80 SNPS"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "The minimum set of 80 SNPs providing the best predictive value for FEV1/FVC were identified in the GWAS.)", "entity_type": "UNKNOWN", "id": "GENETIC VARIANT SET"}, {"source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "description": "The mean square error of FEV1/FVC was assessed using LASSO regression with a set of 80 SNPs.)", "entity_type": "UNKNOWN", "id": "MEAN SQUARE ERROR"}, {"source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "description": "The population sample used for genetic analysis provided insights into the genetic architecture and its impact on lung function.)", "entity_type": "UNKNOWN", "id": "POPULATION SAMPLE"}, {"source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "description": "The GRAIL algorithm was utilized to discover functional connections between genes, revealing links among \u03b2-defensin genes and other identified transcripts.)", "entity_type": "UNKNOWN", "id": "SOFTWARE TOOL"}, {"source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "description": "The 5.8% explained variance in FEV1/FVC by the six SNPs indicated their contribution to understanding the genetic factors influencing lung function.)", "entity_type": "UNKNOWN", "id": "EXPLAINED VARIANCE"}, {"source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "description": "A genetic study on Hutterites, represented by Fig. E3b in the Online Repository, helped validate findings related to specific loci and functional gene interactions.)", "entity_type": "UNKNOWN", "id": "GENETIC STUDY"}, {"source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "description": "The Gene Ontology database provided context for functional connections identified by the GRAIL algorithm.)", "entity_type": "UNKNOWN", "id": "DATABASE"}, {"source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "description": "The Hutterite genome-wide association study provided insights into the genetic basis for variations in lung function through its connections to specific loci and transcripts.)", "entity_type": "UNKNOWN", "id": "HUTTERITES GWAS"}, {"entity_type": "GENOMIC VARIANT", "description": "The SNPs which were mostly consulted were rs231775.", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "clusters": "[{level: 0, cluster: 17}]", "id": "RS231775"}, {"entity_type": "GENOMIC VARIANT", "description": "This SNP was reported in 6 studies.", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "clusters": "[{level: 0, cluster: 17}]", "id": "RS231725"}, {"entity_type": "GENOMIC VARIANT", "description": "The SNPs which were mostly consulted were rs3087243.", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 40}]", "id": "RS3087243"}, {"entity_type": "GENOMIC VARIANT", "description": "This SNP was reported in 5 studies.", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "clusters": "[{level: 0, cluster: 17}]", "id": "RS5742909"}, {"entity_type": "GENE", "description": "SNPs of CTLA4 gene were the focus of this study.<SEP>The CTLA4 gene was studied in relation to its expression levels and epigenetic DNA methylation.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b<SEP>chunk-f02653cb486dbf68cc259f74ea8b2b2e", "clusters": "[{level: 0, cluster: 17}]", "id": "CTLA4"}, {"entity_type": "PROTEIN", "description": "The GG genotype refers to this allele.", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "id": "G"}, {"entity_type": "PROTEIN", "description": "This allele is denoted by GA or AA genotypes.", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "id": "A"}, {"entity_type": "DISEASE", "description": "Primary Biliary Cholangitis (PBC) was the disease focus of this study.", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "clusters": "[{level: 0, cluster: 17}]", "id": "PBC"}, {"entity_type": "GENE", "description": "We identified cis regulatory SNPs for 2605 genes in CD4+ T cells and 2056 genes in CD8+ T cells at probe-level false discovery rate (FDR) < 0.05.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "CIS REGULATORY SNP"}, {"entity_type": "GENOMIC VARIANT", "description": "The purified cells were subjected to genome-wide gene expression analysis, genotyping and imputation using the 1000 Genomes reference panel.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "IMPUTATION"}, {"entity_type": "CPG SITE", "description": "DNA methylation of these sites was determined for added insights on gene expression differences.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "450K CPG SITES"}, {"entity_type": "GENOMIC RESOURCE", "description": "A reference panel used to perform genotyping and imputation in the analysis.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "1000 GENOMES REFERENCE PANEL"}, {"entity_type": "CELL TYPE", "description": "We purified CD4+ and CD8+ T cells from peripheral blood mononuclear cells (PBMCs) of healthy individuals.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "CD4+ T CELLS/CD8+ T CELLS"}, {"entity_type": "CELL TYPE/CLONAL POPULATION", "description": "The genetic association was extended to polyclonal populations of CD4+ and CD8+ T cells.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "AUTOIMMUNE T CELL CLONES/"}, {"entity_type": "SNP DISTRIBUTION/POPULATION GENETICS", "description": "The frequency of specific genotypes across the population was assessed for its association with CTLA4 expression levels.", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "id": "GENOTYPE FREQUENCIES/"}, {"entity_type": "GENOMIC VARIANT", "description": "This genomic variant in the CTLA4 gene showed an inverse correlation with the expression levels of three probes.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "CG26091609"}, {"entity_type": "TRANSCRIPT", "description": "Three different transcripts of the CTLA4 gene were related to its expression differences.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "PROBES"}, {"entity_type": "CELL TYPE", "description": "CD4+ T cells were one of the cell types involved in the study of differential expressions and eQTLs.<SEP>Trans-eQTL effect detected specifically in CD4+ T cell subtypes.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b<SEP>chunk-1b40d071abac0ba432a9fb0096796f45", "id": "CD4+ T CELLS"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Epigenetic DNA methylation influences the splicing effects, affecting gene expression levels.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "DIFFERENTIAL SPLICING EFFECTS"}, {"entity_type": "TRANSCRIPT-SPECIFIC", "description": "Specific eQTL effects were observed in trans-acting regulatory loci of certain genes.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "EQTL EFFECT"}, {"entity_type": "PROXIMITY", "description": "GWAS SNPs affecting the expression levels of distantly located genes were identified.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "DISTAL GENES"}, {"entity_type": "CHROMOSOME 12Q13.2", "description": "A trans-acting regulatory locus with broad-range impact on gene expression was found in this region.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "TRANS-EQTL REGION"}, {"entity_type": "SINGLE NUCLEOTIDE POLYMORPHISM", "description": "There were genome-wide significant SNPs implicated in T1D, vitiligo, and autoimmune diseases.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "GWAS SNPS"}, {"entity_type": "GENES", "description": "This trans-acting region influenced the expression levels of hundreds of genes.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "BROAD RANGE IMPACT"}, {"entity_type": "GENE", "description": "The lead SNP was located in this gene\'s 5\'UTR, part of the ribosomal small subunit protein.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "RPS26 GENE"}, {"entity_type": "PROTEIN", "description": "This trans-eQTL region contained several genes related to T-cell function and immunity.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "CD127L (CTLA4) MOLECULE"}, {"entity_type": "DISEASE ASSOCIATION", "description": "The identified SNPs had associations with autoimmune and inflammatory diseases.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "GWAS SNPS IMPLICATED IN DISEASES"}, {"entity_type": "DISEASE", "description": "A trans-acting regulatory locus was found to influence gene expression linked to T1D risk.<SEP>This disease was shown to have a genetic association with the identified SNPs and their effects on gene expressions.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "T1D (TYPE 1 DIABETES)"}, {"entity_type": "GENOMIC VARIANT/SNP", "description": "This specific SNP had associations with several GWAS SNPs related to autoimmune diseases.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "RS1131017 SNP"}, {"entity_type": "STATISTICAL MEASURE", "description": "Strong linkage disequilibrium was present between the top trans-eQTL SNP and GWAS SNPs.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "LD (LINKAGE DISEQUILIBRIUM)"}, {"entity_type": "GENES", "description": "A shared overlap in genes associated with both trans-acting and GWAS SNPs was identified.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "OVERLAP OF GENES"}, {"entity_type": "MEASUREMENT", "description": "Changes in expression levels were observed for certain genes in association with SNPs.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "EXPRESSION LEVELS"}, {"entity_type": "DATABASE", "description": "GWAS SNPs analyzed came from this catalog, indicating genetic associations related to diseases.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "GWAS CATALOG"}, {"entity_type": "STATISTICAL SIGNIFICANCE", "description": "SNPs were selected based on a stringent statistical threshold for significance.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "P-VALUE < 5X10-8"}, {"entity_type": "GENOMIC VARIANT/SNP", "description": "This GWAS SNP associated with lower expression of IRF1 and STAT1 genes specifically in CD4+ T cells.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "RS4788084[T]"}, {"entity_type": "CHROMOSOMAL LOCATION", "description": "The IL27 gene region is located on chromosome 16 at this specific location, close to rs4788084[T].", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "CHR16P11.2"}, {"entity_type": "GENE", "description": "This gene was analyzed for trans-eQTL effects related to its expression.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "IL27 GENE"}, {"entity_type": "PROTEIN", "description": "Expression levels of this protein showed a trans-eQTL effect for rs4788084[T], specifically in CD4+ T cells.<SEP>IRF1 and STAT1 expression changes are associated with T1D susceptibility.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b<SEP>chunk-1b40d071abac0ba432a9fb0096796f45", "id": "STAT1"}, {"entity_type": "GENE", "description": "This gene was also found to be related to trans-eQTL effects associated with the IL27 region and GWAS SNPs.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "REC8"}, {"entity_type": "GENOMIC VARIANT/SNP", "description": "This SNP caused an amino acid change (Leu119Pro) in the alpha-helical domain of IL-27.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "RS181206[G]"}, {"entity_type": "STATISTICAL METHOD", "description": "A Bayesian test was used to assess the hypothesis that multiple factors (T1D, IRF1 expression changes) are influenced by a single causal variant in this region.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "BAYESIAN TEST FOR COLOCALISATION"}, {"entity_type": "POPULATION SAMPLE", "description": "Genetic data analyzed came from this population sample for LD assessment and frequency analysis.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "1000 GENOMES PHASE 3 EUROPEAN POPULATION"}, {"entity_type": "CHROMOSOMAL REGION", "description": "This genomic location was extensively analyzed for trans-eQTL effects using all available SNPs within this area.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "CHROMOSOME 16:28,2\u201329,1 MB REGION"}, {"entity_type": "POPULATION SAMPLE", "description": "This specific European population sample was used for LD analysis between rs181206[G] and other SNPs.", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "id": "ESTONIAN POPULATION"}, {"entity_type": "GENOMIC VARIANT", "description": "The effect of the missense SNP on IRF1 and STAT1 remains after regressing out the effect of this genomic variant.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "GWAS SNP"}, {"entity_type": "TRANSCRIPT", "description": "No signal was detected in B cells and monocytes when re-analyzing Fairfax et al.\'s data.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "B CELLS"}, {"entity_type": "TRANSCRIPT", "description": "The IL27 SNP rs181206G affects both IRF1 and STAT1 expression, supporting the hypothesis that T1D susceptibility is related to this variant.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "IL27 SNP RS181206G"}, {"entity_type": "MOLECULAR INTERACTION", "description": "IL-27 interacts with its receptor IL27RA and activates the STAT1/STAT3 pathway in T cells.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "IL-27 CYTOKINE"}, {"entity_type": "SEQUENCE MOTIF", "description": "Key elements in the genome recognized by the transcription factors STAT1 and STAT3 upon interferon stimulation.<SEP>The binding of the STAT1/STAT3 pathway to ISRE induces transcription of several interferon-induced genes.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "INTERFERON STIMULATED RESPONSE ELEMENTS (ISRE)"}, {"entity_type": "GENE", "description": "Together, IL27RA and IL6ST genes act as a receptor for the IL-27 cytokine in CD4+ T cells.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "IL27RA AND IL6ST (GP130)"}, {"entity_type": "CELL LINE", "description": "Cells used for transfection to study functional effects of the IL27 missense SNP.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "HEK293 CELLS"}, {"entity_type": "CELL TYPE", "description": "Human peripheral blood mononuclear cells were analyzed for expression levels after IL-27 exposure.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "HUMAN PBMCS"}, {"entity_type": "METHODOLOGY", "description": "Statistical method used to identify genetic variants influencing gene expression.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "TRANS-EQTL ANALYSIS"}, {"entity_type": "ANALYSIS MODEL", "description": "Best-fitting model used to determine the relationship between IL27 SNP rs181206G, IRF1 and STAT1.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "MODEL 1 OF STRUCTURAL EQUATION MODELING"}, {"entity_type": "EXPERIMENTAL TECHNIQUE", "description": "Simulation conducted to support findings of structural equation modeling.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "SIMULATION EXPERIMENT"}, {"entity_type": "MOLECULAR MECHANISM", "description": "Proposed pathway through which IL-27 influences the expression levels of IRF1 and STAT1.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "MECHANISM FOR EFFECT OF IL-27 ON IRF1 AND STAT1 EXPRESSION"}, {"entity_type": "SAMPLE POPULATION", "description": "Healthy donors providing peripheral blood mononuclear cells for the study.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "HUMAN PBMCS FROM FOUR HEALTHY INDIVIDUALS"}, {"entity_type": "GENE CLASS", "description": "Genes whose expression is induced by interferons, including IRF1 and STAT1.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "INTERFERON-INDUCED GENES"}, {"entity_type": "MOLECULAR PATHWAY", "description": "Activation of the STAT1/STAT3 signaling pathway by IL-27 binding to its receptor on T cells.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "PATHWAY ACTIVATION"}, {"entity_type": "CELL CULTURE TECHNIQUE", "description": "Process of introducing foreign DNA into HEK293 cells for studying genetic effects.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "HEK293 CELL TRANSFECTION"}, {"entity_type": "STATISTICAL MEASURE", "description": "Measure comparing the fixed effects of the rs181206[G] allele on IRF1 and STAT1 transcript levels.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "FIXED EFFECTS ESTIMATE"}, {"entity_type": "MOLECULAR BIOLOGY TECHNIQUE", "description": "Method used to quantify gene expression in human PBMCs after IL-27 exposure.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "REAL-TIME PCR"}, {"entity_type": "BIOLOGICAL SAMPLE", "description": "Type of biological material used for molecular studies in this research, including leukocytes from blood.", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "id": "HUMAN PERIPHERAL BLOOD MONONUCLEAR CELLS (PBMCS)"}, {"source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "description": "Correlation with expression levels", "entity_type": "UNKNOWN", "id": "CPG SITE/"}, {"source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "description": "Correlation with expression levels", "entity_type": "UNKNOWN", "id": "METHYLATED SITES"}, {"source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "description": "Gene expression levels were corrected before identifying trans-eQTL associations.)", "entity_type": "UNKNOWN", "id": "CIS-EQTL EFFECTS"}, {"source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "description": "Gene expression levels were corrected before identifying trans-eQTL associations.)", "entity_type": "UNKNOWN", "id": "CORRECTION FACTOR"}, {"source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "description": "SNPs related to gene expressions in distant genes had a broad impact on multiple genes.)", "entity_type": "UNKNOWN", "id": "GWAS SNPS AND TRANS-ACTING GWAS SNPS"}, {"source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "description": "A region influenced the expression levels of several overlapping genes in CD4+ and CD8+ T cells.)", "entity_type": "UNKNOWN", "id": "BROAD-RANGE IMPACT AND OVERLAP OF GENES"}, {"source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "description": "A region influenced the expression levels of several overlapping genes in CD4+ and CD8+ T cells.)", "entity_type": "UNKNOWN", "id": "IMPACT SHARED AMONG GENES"}, {"source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "description": "The lead trans-eQTL SNP was in LD with several GWAS SNPs related to autoimmune diseases.)", "entity_type": "UNKNOWN", "id": "RS1131017 SNP AND GWAS SNPS"}, {"source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "description": "This test supported the hypothesis of a shared causal variant associated with T1D susceptibility and gene expression changes.)", "entity_type": "UNKNOWN", "id": "STATISTICAL METHOD"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "T1D susceptibility is associated with changes in IRF1 and STAT1 expression, which may be influenced by a specific genomic variant.)", "entity_type": "UNKNOWN", "id": "CHANGES IN IRF1 AND STAT1 EXPRESSION"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "T1D susceptibility is associated with changes in IRF1 and STAT1 expression, which may be influenced by a specific genomic variant.)", "entity_type": "UNKNOWN", "id": "T1D SUSCEPTIBILITY"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "Specific trans-eQTL effect detected only in CD4+ T cells but not in CD8+ T cells.)", "entity_type": "UNKNOWN", "id": "CD4+ T CELLS AND CD8+ T CELLS"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "Specific trans-eQTL effect detected only in CD4+ T cells but not in CD8+ T cells.)", "entity_type": "UNKNOWN", "id": "TRANS-EQTL EFFECTS"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "No signal present in B cells or monocytes when re-analyzing Fairfax et al.\'s data.)", "entity_type": "UNKNOWN", "id": "B CELLS AND MONOCYTES"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "No signal present in B cells or monocytes when re-analyzing Fairfax et al.\'s data.)", "entity_type": "UNKNOWN", "id": "SIGNAL PRESENCE"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "IL27 SNP rs181206G has effects on IRF1 and STAT1 expression, confirmed by trans-eQTL analysis.)", "entity_type": "UNKNOWN", "id": "TRANS-EQTL LOCUS EFFECTS"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "The IL-27 cytokine interacts with the IL27RA gene (together with gp130, also known as IL6ST) to activate STAT1/STAT3 pathway.)", "entity_type": "UNKNOWN", "id": "IL-27 CYTOKINE AND IL27RA/GP130"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "The IL-27 cytokine interacts with the IL27RA gene (together with gp130, also known as IL6ST) to activate STAT1/STAT3 pathway.)", "entity_type": "UNKNOWN", "id": "RECEPTOR INTERACTION"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "Interferon stimulated response elements are bound by STAT1 and STAT3 pathways leading to transcriptional activation.)", "entity_type": "UNKNOWN", "id": "BINDING"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "Interferon stimulated response elements are bound by STAT1 and STAT3 pathways leading to transcriptional activation.)", "entity_type": "UNKNOWN", "id": "ISRE"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "The Pro119 variant of the IL27 gene resulted in lower expression levels of IRF1 and STAT1 after transfection experiments.)", "entity_type": "UNKNOWN", "id": "IL27 GENE PRO119 ALLELE"}, {"source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "description": "The Pro119 variant of the IL27 gene resulted in lower expression levels of IRF1 and STAT1 after transfection experiments.)", "entity_type": "UNKNOWN", "id": "TRANSFECTION EFFECTS"}, {"entity_type": "PROTEIN", "description": "IL-8 protein plays a role in immune response, being involved in the production of inflammatory cytokines.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "IL-8 PROTEIN"}, {"entity_type": "TRANSCRIPT", "description": "The transcript of IL-8 gene encodes for the IL-8 protein and plays an essential role in regulating its expression.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "IL-8 MRNA"}, {"entity_type": "CELL COMPOSITION", "description": "PBMC stands for Peripheral Blood Mononuclear Cells, which were claimed not to differ between asthma patients and healthy controls.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "PBMC COMPOSITION"}, {"entity_type": "DISEASE", "description": "Asthma is a chronic inflammatory disease of the airways that can result in wheezing, chest tightness, and shortness of breath.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "ASTHMA"}, {"entity_type": "GENOMIC VARIANT", "description": "rs4073 TT genotype was associated with an increased risk of asthma development.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "IL-8 GENOTYPES: RS4073 TT"}, {"entity_type": "GENOMIC VARIANT", "description": "rs2227306 CC or CT genotypes were significantly different between case and control groups, suggesting its association with asthma susceptibility.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "IL-8 GENOTYPES: RS2227306 CC/CT"}, {"entity_type": "DISEASE SUBTYPE", "description": "Subject with rs4073 TT had more risk to be diagnosed with advanced stage asthma.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "ADVANCED STAGE ASTHMA"}, {"entity_type": "HAPLOTYPE", "description": "The frequency of the rare diplotype AT was significantly associated with a lower asthma childhood risk.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "RARE DIPLOTYPE: AT"}, {"entity_type": "GENOMIC VARIANT", "description": "The rs4073 T allele is associated with increased asthma risk and severity.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "RS4073 T ALLELE"}, {"entity_type": "STATISTICAL ANALYSIS", "description": "The frequencies of IL-8 genotypes were analyzed using the Hardy-Weinberg Equilibrium (HWE).", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "IL-8 GENOTYPE FREQUENCIES"}, {"entity_type": "GENETIC RELATIONSHIP MEASURE", "description": "The Linkage Disequilibrium test evaluated for genetic associations between different IL-8 polymorphisms.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "LD TEST (D\' VALUE)"}, {"entity_type": "DISEASE DYNAMICS", "description": "Analysis of disease progression over time aimed to understand patterns and triggers of asthma exacerbations.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "ASTHMA DISEASE PROGRESSION OVER TIME"}, {"entity_type": "TREATMENT AND MANAGEMENT", "description": "Ongoing clinical assessments help in tailoring treatment plans based on individual responses and disease progression.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "CLINICAL ASSESSMENT AND INTERVENTION"}, {"entity_type": "INTERPLAY IN ASTHMA MANAGEMENT", "description": "The interplay between clinical outcomes, such as disease severity, and genetic factors influences the management strategies for asthma patients.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "CLINICAL OUTCOME AND GENETIC FACTORS"}, {"entity_type": "DIAGNOSTIC AND PROGNOSTIC CRITERIA", "description": "Development of tools that combine genetic markers with clinical data aids in personalized asthma management plans.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "GENETIC RISK ASSESSMENT TOOLS"}, {"entity_type": "NON-GENETIC INFLUENCES", "description": "Consideration of lifestyle and environmental factors plays a crucial role in asthma management alongside genetic considerations.", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "id": "LIFESTYLE AND ENVIRONMENTAL FACTORS"}, {"source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "description": "Significantly increased IL-8 mRNA levels were found in asthmatic patients compared to controls, especially those with moderate-to-severe asthma.)", "entity_type": "UNKNOWN", "id": "IL-8 MRNA LEVELS"}, {"source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "description": "rs4073 TT genotype was significantly associated with an increased risk of asthma development.)", "entity_type": "UNKNOWN", "id": "ASTHMA RISK"}, {"source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "description": "rs4073 TT genotype was significantly associated with an increased risk of asthma development.)", "entity_type": "UNKNOWN", "id": "RS4073 TT GENOTYPE"}, {"source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "description": "The frequencies of rs2227306 CC and CT genotypes were found to be significantly different between case and control groups, suggesting its association with asthma susceptibility.)", "entity_type": "UNKNOWN", "id": "RS2227306 CC/CT GENOTYPES"}, {"source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "description": "Increased IL-8 mRNA expression was linked with the frequency of atopic asthma among children.)", "entity_type": "UNKNOWN", "id": "IL-8 PRO-INFLAMMATORY RESPONSE"}, {"source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "description": "The rs4073 T allele was considered as a risk factor for asthma severity.)", "entity_type": "UNKNOWN", "id": "ASTHMA SEVERITY IN TUNISIAN CHILDREN"}, {"entity_type": "GENE", "description": "The ASAH1 gene consists of 86 HapMap SNPs and 14 typed SNPs on the Illumina HumanHap550 array, making it a complex genetic entity.<SEP>The ASAH1 gene has an effect on certain traits, and its power is tested using different kernels.<SEP>The ASAH1 gene is on 8p22-p21.3 and consists of 86 HapMap SNPs and 14 typed SNPs on the Illumina HumanHap550 array.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0<SEP>chunk-076f6aeeb369546413bc63c9f305553a", "id": "ASAH1 GENE"}, {"entity_type": "GENE", "description": "NAT2 lies on 8p22 and has 22 HapMap SNPs and only 4 typed SNPs from the transcription start to end of the gene.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "NAT2 GENE"}, {"entity_type": "GENE", "description": "For a much larger gene like FGFR2 (Figure 5), in regions of high LD, the kernel machine method again outperforms the min test. In regions of weak LD, the two methods have comparable power.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "FGFR2 GENE"}, {"entity_type": "GENE", "description": "The HLA-B gene is a part of a family of genes known as the human leukocyte antigen (HLA) complex and lies on 6p21.3.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "HLA-B GENE"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs1390358 has moderate R2 correlation with the typed SNPs.<SEP>SNP rs1390358 is one of the HapMap SNPs included in the NAT2 gene region.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "RS1390358"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs1112005 displays a high degree of R2 correlation with other typed SNPs.<SEP>SNP rs1112005 is another HapMap SNP found within the NAT2 gene region.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "RS1112005"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs7832071 is one of the typed SNPs from the transcription start to end of the NAT2 gene.<SEP>SNP rs7832071 shares similar LD properties to rs1112005, showing a 99% R2 correlation.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "RS7832071"}, {"entity_type": "GENOMIC VARIANT", "description": "SNP rs1208 is also among those typed SNPs with moderate R2 correlations.<SEP>SNP rs1208 is among the typed SNPs found in the NAT2 gene region.", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "id": "RS1208"}, {"source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "description": "The SNP-set for the ASAH1 gene includes only these 14 typed SNPs that are in good LD with each other and with the causal SNPs.)", "entity_type": "UNKNOWN", "id": "SNP-SET"}, {"source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "description": "SNPs 7, 11, 13, and 21 are in good LD with each other and when tested as causal, the power of kernel machine method is higher than min test.)", "entity_type": "UNKNOWN", "id": "TYPED SNPS"}, {"source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "description": "In regions of high LD, the kernel machine method outperforms the min test.)", "entity_type": "UNKNOWN", "id": "HIGH LD REGIONS"}, {"source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "description": "In regions of weak LD, the two methods have comparable power.)", "entity_type": "UNKNOWN", "id": "WEAK LD REGIONS"}, {"source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "description": "The empirical power for HLA-B gene shows that using MAF as weights in weighted IBS kernel can upweight rare alleles.)", "entity_type": "UNKNOWN", "id": "SNPS 1058026 AND RS2523608"}, {"source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "description": "The weighted IBS kernel outperforms the linear kernel in terms of power when rare variants are important.)", "entity_type": "UNKNOWN", "id": "IBS KERNEL"}, {"source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "description": "The weighted IBS kernel outperforms the linear kernel in terms of power when rare variants are important.)", "entity_type": "UNKNOWN", "id": "LINEAR KERNEL"}, {"source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "description": "When rs2523608 has higher minor allele frequency, it\'s not as effective as when the more common allele (which is also causal) gets downweighted by a certain kernel function.)", "entity_type": "UNKNOWN", "id": "HIGHER MAF"}, {"source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "description": "When rs2523608 has higher minor allele frequency, it\'s not as effective as when the more common allele (which is also causal) gets downweighted by a certain kernel function.)", "entity_type": "UNKNOWN", "id": "RS2523608 SNV/CAUSAL SNP"}, {"entity_type": "GENE", "description": "rs1893217 was mapped to the PTPN2 gene.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "PTPN2 GENE"}, {"entity_type": "DISEASE", "description": "The SNP is associated with autoimmune diseases such as celiac disease.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "CELIAC DISEASE"}, {"entity_type": "DISEASE", "description": "The SNP is also associated with inflammatory bowel disease.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "INFLAMMATORY BOWEL DISEASE"}, {"entity_type": "GENOMIC VARIANT", "description": "Under an additive model, rs17388568 showed a significant association with response rate for anti-PD-1 treatment. Carriers of at least one minor allele (GA or AA) are 74% less likely to resist the treatment compared to those with GG genotype.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "RS17388568"}, {"entity_type": "GENE", "description": "The SNP is mapped near IL2, a gene involved in immunity and inflammation.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "IL2 GENE"}, {"entity_type": "GENE", "description": "Another gene closely associated with rs17388568 through its location on the same locus as IL2 and IL21.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "ADAD1 GENE"}, {"entity_type": "GENE", "description": "The locus containing IL21 has been previously linked to allergic conditions, colitis, and type 1 diabetes.", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "id": "IL21 GENE"}, {"entity_type": "DRUG", "description": "anti-CTLA-4 is a drug used in treatment, but no significant association was observed with treatment response.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "ANTI-CTLA-4"}, {"entity_type": "DRUG", "description": "anti-PD-1 is another drug tested that did not show a significant association with treatment response.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "ANTI-PD-1"}, {"entity_type": "TREATMENT", "description": "The combination of different therapies was performed on patients, but no significant association with the treatment response was observed after adjustments for age, sex and treatment drugs.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "COMBINATION THERAPY"}, {"entity_type": "ANALYSIS", "description": "A pooled analysis across treatments was conducted to assess associations without any significant findings.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "POOLED ASSOCIATION ANALYSIS"}, {"entity_type": "MEDICATION/DRUG", "description": "The different types of treatments (including anti-CTLA-4, anti-PD-1 and combination therapy) were adjusted for during pooled analysis to assess their impact on treatment response.", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "id": "TREATMENT DRUGS"}, {"source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "description": "The association of rs1893217 and response rate was tested in anti-CTLA-4 treated patients, with an OR value of approximately 2.79 for non-responders.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 30}]", "id": "RESPONSE RATE"}, {"source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "description": "rs17388568 shows a strong association with response rate specifically in anti-PD-1 treatment.)", "entity_type": "UNKNOWN", "clusters": "[{level: 0, cluster: 13}, {level: 1, cluster: 38}]", "id": "RESPONSE RATE FOR ANTI-PD-1"}, {"source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "description": "The effect of rs17388568 on response rate was stronger in younger female patients, although the interactions were not statistically significant.)", "entity_type": "UNKNOWN", "id": "AGE AND SEX"}, {"source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "description": "The effect of rs17388568 on response rate was stronger in younger female patients, although the interactions were not statistically significant.)", "entity_type": "UNKNOWN", "id": "INTERACTION EFFECT ON RESPONSE"}, {"source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "description": "No significant association was observed between anti-CTLA-4 or anti-PD-1 drugs and treatment response across different treatments, after adjustments for age, sex, and drug types.)", "entity_type": "UNKNOWN", "id": "ANTI-CTLA-4/ANTI-PD-1"}, {"source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "description": "A pooled analysis of combination therapy outcomes did not yield significant associations with treatment response when adjusting for various covariates.)", "entity_type": "UNKNOWN", "id": "COMBINATION THERAPY/POOLED ASSOCIATION ANALYSIS"}, {"source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "description": "A pooled analysis of combination therapy outcomes did not yield significant associations with treatment response when adjusting for various covariates.)", "entity_type": "UNKNOWN", "id": "TREATMENT RESPONSE"}, {"source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "description": "After adjustments for age, sex, treatment drugs, and other covariates, no associations were found with SNPs related to treatment response.)", "entity_type": "UNKNOWN", "id": "COVARIATE IMPACT"}, {"source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "description": "After adjustments for age, sex, treatment drugs, and other covariates, no associations were found with SNPs related to treatment response.)", "entity_type": "UNKNOWN", "id": "SNPS/AGE/SEX/TREATMENT DRUGS"}, {"source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "description": "No significant association was observed between the type of drug (anti-CTLA-4, anti-PD-1 or combination therapy) and treatment response after adjusting for age, sex and different types of treatment.)", "entity_type": "UNKNOWN", "id": "ANTI-CTLA-4/ANTI-PD-1/COMBINATION THERAPY/"}, {"source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "description": "No significant association was observed between the type of drug (anti-CTLA-4, anti-PD-1 or combination therapy) and treatment response after adjusting for age, sex and different types of treatment.)", "entity_type": "UNKNOWN", "id": "TREATMENT TYPE IMPACT"}, {"source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "description": "Adjustments made to account for covariates like age, sex and specific treatments did not reveal significant associations with factors being tested (SNPs, drug therapies) related to response or outcomes.)", "entity_type": "UNKNOWN", "id": "AGE/SEX/TREATMENT DRUGS/"}, {"source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "description": "Adjustments made to account for covariates like age, sex and specific treatments did not reveal significant associations with factors being tested (SNPs, drug therapies) related to response or outcomes.)", "entity_type": "UNKNOWN", "id": "COVARIATE ADJUSTMENT"}, {"entity_type": "GENE", "description": "EBNA2 binds chromatin at MS risk loci dependent on the risk allele.<SEP>EBNA2 binds one allele more than the other and interacts with MS risk loci.", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7<SEP>chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "EBNA2"}, {"entity_type": "MOLECULAR INTERACTION/ENVIRONMENTAL FACTOR/DISEASE", "description": "Of the six MS risk SNPs investigated, EBNA2 was found to bind dependent on MS risk allele at five loci (TRAF3/RCOR1 rs1258869, CD40 rs1883832, TNFAIP8 locus rs32658, TNFRSF1A locus rs180069, TBX6 rs3809627).", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "SNP/RISK ALLELE/LOCUS/EBV LATENCY III"}, {"entity_type": "PROTEIN/DRUG", "description": "We undertook ChIP for EBNA2 in 33 samples and investigated the effects of EBNA2-TAT on LCLs.", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "EBNA2-TAT"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "LCL proliferation was reduced significantly by all concentrations tested with increasing doses of EBNA2-TAT (40% at 50\u00b5M, approximately 50% at 100\u00b5M, over 50% at 200\u00b5M).", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "LCL PROLIFERATION"}, {"entity_type": "GENE/MOLECULAR INTERACTION", "description": "Expression of the EBNA2 regulated EBV gene LMP1 was reduced significantly by EBNA2-TAT (p < 0\u00b705) at concentrations of 50\u00b5M.", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "LMP1 EXPRESSION/EBV LMP1"}, {"entity_type": "GENES/", "description": "Disease/EBNA2 Inhibition Effect", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "SEVEN MS-ASSOCIATED GENES"}, {"entity_type": "MOLECULAR INTERACTION/GENETIC TRAIT/", "description": "Disease/Environmental Factor/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "EBV DNA COPY NUMBER"}, {"entity_type": "ALLELIC ASSOCIATION/EBNA2 INHIBITION/", "description": "Genotype/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "GENOTYPE EFFECT/"}, {"entity_type": "MOLECULAR COMPLEX/", "description": "Activity/Environmental Factor/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "RNA POL II"}, {"entity_type": "BIOLOGICAL PROCESS/", "description": "Molecular Mechanism/Pathway", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "CHROMATIN REMODELING"}, {"entity_type": "VIRAL FUNCTION/", "description": "Effect on Cells/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "HIV-1 REPLICATION"}, {"entity_type": "PHYSIOLOGICAL PROCESS/", "description": "Pathway/Molecular Interaction/Disease/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "IMMUNE CELL PROLIFERATION"}, {"entity_type": "INTERACTION/", "description": "Relationship/Evolutionary Dynamics/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "VIRUS/HOST INTERACTIONS"}, {"entity_type": "BIOLOGICAL PROCESS/", "description": "Pathway/Response to Stimuli/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "B-CELL ACTIVATION"}, {"entity_type": "MOLECULAR FUNCTION/", "description": "Process/Control Mechanism/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "TRANSCRIPTIONAL REGULATION"}, {"entity_type": "BIOLOGICAL PROCESS/", "description": "Pathway/Response to Injury/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "NEUROINFLAMMATION"}, {"entity_type": "STATE OF VIRUS/", "description": "Status/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "VIRAL LATENCY"}, {"entity_type": "OUTCOME OF DISEASE/", "description": "Damage/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "NEURODEGENERATION"}, {"entity_type": "BIOLOGICAL PROCESS/", "description": "Pathway/Control Mechanism/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "CELL CYCLE REGULATION"}, {"entity_type": "QUANTITATIVE MEASURE/", "description": "Attribute/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "GENE EXPRESSION PROFILE"}, {"entity_type": "STATISTICAL MEASURE/", "description": "Data Type/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "RISK ALLELE FREQUENCIES"}, {"entity_type": "BIOLOGICAL PROCESS/", "description": "Pathway/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "NEUROINFLAMMATORY RESPONSE"}, {"entity_type": "STATE OF VIRUS/", "description": "Infection Stage/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "LATENT INFECTION DYNAMICS"}, {"entity_type": "DISEASE", "description": "STAT5B deficiency can lead to the autoimmune response where immune cells attack healthy tissues.<SEP>Disease Subcategory/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7<SEP>chunk-e323b0d4f82733a78f687e51f47b500a", "id": "AUTOIMMUNITY"}, {"entity_type": "RISK FACTOR/", "description": "Variable/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "id": "GENETIC SUSCEPTIBILITY"}, {"entity_type": "DISEASE", "description": "MS risk loci are regions in the genome that increase the risk of developing multiple sclerosis when there is a certain genotype.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "MS RISK LOCI"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The allelic association with expression was altered by EBNA2, potentially impacting genetic predispositions to disease.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "ALLELIC ASSOCIATION"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "EBNA2 changes the expression levels of certain genes in lymphoblastoid cell lines (LCLs) compared to uninfected B cells.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "EXPRESSION LEVEL"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The direction of effect on risk gene expression is inferred based on relative allelic expression in LCLs.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "RELATIVE ALLELIC EXPRESSION"}, {"entity_type": "MOLECULAR PROCESS", "description": "Some MS risk genes are involved in the EBV latency III program, which interacts with EBNA2 and may influence immune evasion processes.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "EBV LATENCY III PROGRAM"}, {"entity_type": "SIGNALING PATHWAY", "description": "These signaling pathways are part of the EBV latency III program that can be affected by EBNA2\'s interaction with MS risk loci.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "LMP1 AND LMP2 SIGNALLING"}, {"entity_type": "MOLECULAR FUNCTION", "description": "Enhanced EBNA2 function in relation to MS risk genes could lead to evasion of the host immune response.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "IMMUNE EVASION"}, {"entity_type": "GENETIC FACTOR", "description": "EBNA2\'s preference for certain risk alleles suggests a genetic predisposition factor in its impact on gene expression.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "RISK ALLELE PREFERENCE"}, {"entity_type": "BIOLOGICAL PATHWAY", "description": "The biological pathways these MS risk genes are part of can be disrupted by EBNA2, affecting cellular functions and disease susceptibility.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "BIOLOGICAL PATHWAY"}, {"entity_type": "MOLECULAR FUNCTION", "description": "Certain molecular functions related to immune response or disease are influenced by the expression levels or activities of these MS risk genes.", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "id": "MOLECULAR FUNCTION"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Therapy", "entity_type": "UNKNOWN", "id": "MS RISK ALLELES/SNPS/GENOMIC VARIANT/DISEASE/ENVIRONMENTAL FACTOR/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Disease/Environmental Factor/", "entity_type": "UNKNOWN", "id": "EBNA2-TAT/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Disease/Environmental Factor/", "entity_type": "UNKNOWN", "id": "DRUG/EFFECT ON EBV DNA COPY NUMBER/MOLECULAR INTERACTION/GENETIC TRAIT/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Therapy", "entity_type": "UNKNOWN", "id": "EBNA2 BINDING/ALLELIC ASSOCIATION/MS RISK ALLELES/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Therapy", "entity_type": "UNKNOWN", "id": "EXPRESSION/GENETIC TRAIT/DISEASE/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Disease/Environmental Factor/", "entity_type": "UNKNOWN", "id": "LCL PROLIFERATION/EBNA2-TAT/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Disease/Environmental Factor/", "entity_type": "UNKNOWN", "id": "EFFECT ON GENE EXPRESSION/MOLECULAR INTERACTION/GENETIC TRAIT/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Therapy", "entity_type": "UNKNOWN", "id": "TNFAIP8/EXPRESSION/EBNA2 INHIBITION/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Therapy", "entity_type": "UNKNOWN", "id": "EBV DNA COPY NUMBER/MOLECULAR INTERACTION/GENETIC TRAIT/DISEASE/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Disease/Therapy", "entity_type": "UNKNOWN", "id": "GENOTYPE EFFECT/ALLELIC ASSOCIATION/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Disease/Therapy", "entity_type": "UNKNOWN", "id": "EBNA2 BINDING/EXPRESSION/EBNA2 INHIBITION/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Disease/Therapy", "entity_type": "UNKNOWN", "id": "EBNA2 BINDING/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Disease/Therapy", "entity_type": "UNKNOWN", "id": "ALLELIC ASSOCIATION/EXPRESSION/GENETIC TRAIT/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Genetic Trait/Disease/", "entity_type": "UNKNOWN", "id": "EBNA2 INHIBITION/EBNA2-TAT/"}, {"source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "description": "Genetic Trait/Disease/", "entity_type": "UNKNOWN", "id": "EFFECT ON LCLS/MOLECULAR INTERACTION/"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "The genotype effect of EBNA2 involves altering allelic association with expression compared to that found in uninfected B cells.)", "entity_type": "UNKNOWN", "id": "EBNA2 EFFECT"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "EBNA2\'s impact on gene expression correlates with specific MS risk genes and their role in the EBV latency III program.)", "entity_type": "UNKNOWN", "id": "MS RISK GENES INTERACTION"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "The alteration of allelic expression based on EBNA2 interaction is inferred from comparison to uninfected B cells and has implications for gene regulation.)", "entity_type": "UNKNOWN", "id": "ALLELIC EXPRESSION"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "The alteration of allelic expression based on EBNA2 interaction is inferred from comparison to uninfected B cells and has implications for gene regulation.)", "entity_type": "UNKNOWN", "id": "INFERRED EFFECT"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "The program involving LMP1 and LMP2 signaling can be influenced by EBNA2\'s interaction with MS risk loci, potentially impacting immune evasion through alteration of gene expression.)", "entity_type": "UNKNOWN", "id": "EFFECT ON IMMUNE EVASION"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "EBNA2\'s preference for certain risk alleles has implications on altering the direction of allelic association with expression in specific MS risk genes.)", "entity_type": "UNKNOWN", "id": "INFLUENCE ON GENE EXPRESSION"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "The expression levels of these genes are affected by EBV infection, which can be influenced by factors such as EBNA2 interaction and genotype.)", "entity_type": "UNKNOWN", "id": "EBV INFECTION"}, {"source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "description": "The relationship between the risk allele preference and genetic drift could impact the overall expression pattern of MS risk genes.)", "entity_type": "UNKNOWN", "id": "GENETIC DRIFT"}, {"entity_type": "GENETIC POLYMORPHISM", "description": "The initial search revealed rs3842787 genetic polymorphism in association studies.", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "id": "RS3842787"}, {"entity_type": "GENETIC POLYMORPHISM", "description": "The genomic variant rs20417 showed significant differences in percentage of CAD between patients and controls, contributing to a large part of heterogeneity.<SEP>This polymorphism was found to be associated with aspirin insensitivity.", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea", "id": "RS20417"}, {"entity_type": "GENETIC POLYMORPHISM", "description": "The examination of this genetic variant also revealed its association with aspirin insensitivity.", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "id": "RS20118426"}, {"entity_type": "GENETIC POLYMORPHISM", "description": "The genomic variant rs1126643 was another significant source of heterogeneity related to averaged platelet number.<SEP>This polymorphism was significantly associated with aspirin insensitivity in the analysis.", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17<SEP>chunk-734cac289b42d37dad32bd51ea24a1ea", "id": "RS1126643"}, {"entity_type": "DISEASE", "description": "The association of genetic variants with aspirin insensitivity was a central topic of investigation.", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "id": "ASPIRIN INSENSITIVITY"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Averaged platelet number was a significant factor in the heterogeneity observed for the genomic variant rs1126643.)<SEP>The mean number of platelets in a sample, reflecting normal or abnormal platelet counts which may affect blood clotting and thus be relevant to cardiovascular health.", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea", "id": "AVERAGED PLATELET NUMBER"}, {"entity_type": "GENOMIC VARIANTS", "description": "Single nucleotide polymorphisms that were studied in relation to the meta-regression model\'s outcomes.", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea", "id": "RS20417 AND RS1126643"}, {"entity_type": "DISEASE MEASURE", "description": "A quantifiable indicator measuring coronary artery disease.", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea", "id": "PERCENTAGE OF CAD"}, {"entity_type": "SUBJECT GROUP", "description": "Individuals under examination, usually those with a condition or characteristic being studied.", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea", "id": "PATIENTS"}, {"entity_type": "STATISTICAL MEASURE", "description": "Results from statistical tests indicating significance, with lower values suggesting stronger evidence against the null hypothesis.", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea", "id": "P-VALUE (0.034 AND 0.012)"}, {"entity_type": "BIOLOGICAL SIGNIFICANCE", "description": "Statistical indication that observed differences are not due to chance, implying importance in the context of the study\'s biological or clinical question.", "source_id": "chunk-734cac289b42d37dad32bd51ea24a1ea", "id": "SIGNIFICANCE (P-VALUES OF RS20417 AND RS1126643)"}, {"entity_type": "PROTEIN", "description": "This variant converts a tryptophan residue at position 631 into an alternative amino acid causing premature termination (nonsense).", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "P.W631*"}, {"entity_type": "GENE", "description": "The gene coding for this protein and is associated with growth disorders.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "STAT5B"}, {"entity_type": "GENOMIC VARIANT", "description": "This variant affects codon 1892 of the STAT5B gene, resulting in a premature stop codon (p.W631*).", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "C.1892G>A"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The p.W631* variant results in an altered protein structure that affects its function.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "W631*"}, {"entity_type": "EXISTING VARIANT", "description": "This is another known mutation in STAT5B, located nearby the new mutation.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "P.A630P"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Growth Hormone, produced by the pituitary gland, regulates growth and metabolism.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "GH"}, {"entity_type": "PROTEIN", "description": "Insulin-like growth factor binding protein 3 (IGFBP3), which binds IGFs, modulating their effects on cells.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "IGFBP3"}, {"entity_type": "PROTEIN", "description": "Acid labile subunit (ALS) is a component of the complex that regulates IGF activity.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "ALS"}, {"entity_type": "DRUG", "description": "Recombinant human insulin-like growth factor 1 used for treatment in GHI phenotype.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "RECOMBINANT HUMAN INSULIN-LIKE GROWTH FACTOR 1 (RHIGF-1)"}, {"entity_type": "CLINICAL TEST", "description": "Whole Exome Sequencing, used to identify genetic variants linked to the clinical presentation.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "WES"}, {"entity_type": "CELL MODEL", "description": "Human embryonic kidney cells transfected with growth hormone receptors used for reconstitution analyses.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "HEK-GHRS"}, {"entity_type": "TRANSCRIPTION FACTOR", "description": "The transactivation domain (TAD) of STAT proteins regulates transcriptional activation in response to cytokine signals.", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "id": "TAD"}, {"entity_type": "GENOMIC VARIANT", "description": "The STAT5B gene has a variant that leads to the production of an altered protein, resulting in reduced expression.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "STAT5B P.W631* VARIANT"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Sustained activation and proliferation of B cells are associated with atopy (a type of allergic reaction) and autoimmunity in individuals with STAT5B deficiency.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "SUSTAINED B CELL HYPERACTIVITY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "There is an increase in immunoglobulins, which are antibodies produced by B cells, especially IgE, indicating heightened B cell activity.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "IMMUNOGLOBULIN PRODUCTION"}, {"entity_type": "NUTRIENT", "description": "Increased levels of Immunoglobulin E (IgE) can be considered a type of nutrient that contributes to allergic reactions and immune hyperactivity.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "IGE"}, {"entity_type": "DISEASE", "description": "A chronic skin condition characterized by inflammation, itching, and redness, which is frequently associated with STAT5B deficiency.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "CHRONIC ECZEMA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "An interaction between increased IgE levels and the development of respiratory symptoms that lead to conditions such as asthma.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "REACTIVE AIRWAY DISEASE"}, {"entity_type": "DISEASE", "description": "Decreased platelet count (thrombocytopenia) may occur in patients with STAT5B deficiency, affecting blood clotting abilities.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "THROMBOCYTOPENIA"}, {"entity_type": "MOLECULAR INTERACTION", "description": "An elevated level of neutrophils (white blood cells), known as neutrophilia, can be observed as a characteristic immunological feature in individuals with STAT5B deficiency.", "source_id": "chunk-e323b0d4f82733a78f687e51f47b500a", "id": "NEUTROPHILIA"}, {"entity_type": "TRANSCRIPT", "description": "TCM cells remained within normal ranges.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "TCM CELLS"}, {"entity_type": "TRANSCRIPT", "description": "No alterations in the proportion CD4+ TEM cells were identified.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "CD4+ TEM CELLS"}, {"entity_type": "TRANSCRIPT", "description": "Increases in CD8+ TEM cell proportions became clinically significant with age in each sibling.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "CD8+ TEM CELLS"}, {"entity_type": "DIETARY PATTERN", "description": "Elevated proportions of CD8+ memory T cells beginning in infancy suggest a role in CD8+ T cell activation and homeostasis associated with this LOF STAT5B variant.)(entity<SEP>Elevations in the percentage of CD8+ TEMRA cells detected during infancy.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "INFANCY"}, {"entity_type": "DIETARY PATTERN", "description": "Elevations in the percentage of CD8+ TEMRA cells detected specifically in P2 and P3.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "P2 AND P3"}, {"entity_type": "DIETARY PATTERN", "description": "A normalization point around age 4 for elevated percentages of CD8+ TEMRA cells.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "AGE 4"}, {"entity_type": "DIETARY PATTERN", "description": "Remained normal in P1 regarding elevation of CD8+ TEMRA cell percentages.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "P1"}, {"entity_type": "DISEASE", "description": "An observation not previously reported for STAT5B deficiency related to dysregulation of CD8+ T cells.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "STAT5B DEFICIENCY"}, {"entity_type": "TRANSCRIPT", "description": "Increases in CD8+ TEM cell proportions showed clinical significance as age progressed in siblings (Figures not specifically mentioned but implied from context).", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "AGE SIGNIFICANCE"}, {"entity_type": "TRANSCRIPT", "description": "Elevations in the percentage of CD8+ TEMRA cells were detected during infancy, as specified for P2 and P3, but normalized around age 4.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "INFANCY OBSERVATION"}, {"entity_type": "DIETARY PATTERN", "description": "Emphasizes that some variables remained normal over time (P1), while others showed significant deviations (P2, P3) as described by Figures S4C, Figure 3C and their corresponding text.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "NORMALITY"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Noted alongside normal CD4+ memory T cell subsets, indicating a unique aspect of STAT5B deficiency (context implies dysregulated process but specific figure or reference not provided).", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "CD8+ T CELL DYSREGULATION"}, {"entity_type": "GENOMIC VARIANT", "description": "Suggests that an LOF (loss-of-function) variant in STAT5B could influence CD8+ T cell activation and homeostasis.", "source_id": "chunk-4389e37ac64d551cdf89e7f4230a3589", "id": "LOF STAT5B VARIANT"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "This existing mutation is in proximity to the newly identified p.W631* variant in STAT5B.)", "entity_type": "UNKNOWN", "id": "EXISTING VARIANT"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "A homozygous variant in STAT5B gene results in a loss-of-function mutation leading to altered protein function.)", "entity_type": "UNKNOWN", "id": "GENE MUTATION"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "Src Homology 2 (SH2) domain is critical for STAT5B activation and homodimer formation.)", "entity_type": "UNKNOWN", "id": "DOMAIN"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "Src Homology 2 (SH2) domain is critical for STAT5B activation and homodimer formation.)", "entity_type": "UNKNOWN", "id": "SH2"}, {"source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "description": "The p.W631* mutation results in reduced expression, inability for phosphorylation and nuclear localization impairment.)", "entity_type": "UNKNOWN", "id": "MUTATION EFFECT"}, {"entity_type": "GENE", "description": "The HLA-DRB1 gene was sequenced based on genotyping results.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "HLA-DRB1"}, {"entity_type": "GENOMIC VARIANT", "description": "Exon 2 of the HLA-DRB1 gene was used in a previous investigation for genotyping.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "EXON 2 HLA-DRB1"}, {"entity_type": "GENE", "description": "Exons 2 and 3 of the HLA-DQB1 gene were sequenced based on previous genotyping results.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "HLA-DQB1"}, {"entity_type": "ASSAY", "description": "The success rate of genotyping rs28383172 using this assay was 98%.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "CUSTOM TAQMAN SNP GENOTYPING ASSAY"}, {"entity_type": "ASSAY", "description": "Three SNPs (rs2395182, rs4713586 and rs7775228) had genotypes available for all 233 patients using these assays.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "KASPAR-ON-DEMAND PREVALIDATED ASSAYS"}, {"entity_type": "STATISTICAL MEASURE", "description": "These were calculated between the four SNPs and several super populations based on the 1000 Genomes dataset.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "LINKED DISEQUILIBRIUM (LD) COEFFICIENTS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The LD coefficients in the Hungarian population matched most closely with those of the EUR super population.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "HUNGARIAN POPULATION"}, {"entity_type": "PREDICTOR", "description": "This was associated with HLA-DRB1*07:01 carrier status and DQA1*02:01\u2013DQB1*02:02 haplotype carrier status.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS28383172 AG OR GG GENOTYPES"}, {"entity_type": "PREDICTOR", "description": "This was associated with the rs4713586 TT and rs7775228 TC or CC genotypes in relation to DQA1*02:01\u2013DQB1*02:02 haplotype carrier status.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS2395182 TT OR TG GENOTYPES"}, {"entity_type": "PREDICTOR", "description": "This was associated with HLA-DRB1*07:01\u2013DQA1*02:01\u2013DQB1*02:02 haplotype carrier status in patients with rs28383172 AG or GG genotypes.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS7775228 TC OR CC GENOTYPES"}, {"entity_type": "ASSOCIATION", "description": "Carrying the G allele of rs28383172 was associated with HLA-DRB1*07:01 allele carrier status.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "HLA-DRB1*07:01 ALLELE CARRIER STATUS"}, {"entity_type": "ASSOCIATION", "description": "The presence of the rest of the patients without carrying the HLA-DQB1*02:02 allele was observed.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "HLA-DQB1*02:02 ALLELE"}, {"entity_type": "GENOMIC REGION", "description": "Single nucleotide polymorphism (SNP) analysis was performed at this genomic location, which is part of the Human Leukocyte Antigen (HLA) complex.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "HLA-DRB5/DRB6 LOCUS"}, {"entity_type": "SNPS", "description": "These two SNPs were genotyped using the Custom TaqMan SNP Genotyping Assay in this study.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS28383172 AND RS2274697"}, {"entity_type": "HAPLOTYPE", "description": "This specific combination of DQA1 and DQB1 alleles was found among the patients with certain genotypes.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "DQA1*02:01\u2013DQB1*03:02 HAPLOTYPE"}, {"entity_type": "GENE", "description": "The HLA-DRB7 gene was also part of the analysis, although its exact exons or SNPs were not specified in this context.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "HLA-DRB7/"}, {"entity_type": "SNPS", "description": "These three specific SNPs from the HLA region underwent genotyping using KASPar-on-Demand prevalidated assays.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS2395182, RS4713586, AND RS7775228"}, {"entity_type": "POPULATION GROUPING", "description": "The LD coefficients were calculated based on comparisons among super-populations, including the Hungarian population and EUR (European) super population.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "SUPER POPULATIONS"}, {"entity_type": "GENOTYPE DISTRIBUTION", "description": "Specific distributions of these genotypes were observed in relation to certain patient characteristics or statuses.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS28383172 AG OR GG GENOTYPES, RS2395182 TT OR TG GENOTYPES, RS4713586 TT GENOTYPE, RS7775228 TC OR CC GENOTYPES"}, {"entity_type": "HAPLOTYPE", "description": "This particular HLA haplotype was identified among the patients with specific genotype distributions.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "HLA-DRB1*07:01\u2013DQA1*03:01\u2013DQB1*02:02 HAPLOTYPE"}, {"entity_type": "PROTEIN PRODUCTS", "description": "The study also involved analysis of protein products encoded by these HLA genes, which are important for antigen presentation in immune responses.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "HLA-DRB1 AND HLA-DPB1 PROTEINS"}, {"entity_type": "ALLELE TYPE", "description": "The genetic variants identified on the rs28383172 locus include this specific G allele.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "RS28383172 G ALLELE"}, {"entity_type": "ALLELE COMPOSITION", "description": "These specific alleles were found together in certain patient samples, indicating specific HLA types or haplotypes.", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "id": "HLA-DRB1*07:01 AND HLA-DQB1*02:02 ALLELES"}, {"entity_type": "DATASET/", "description": "Genomic Data", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "1000 GENOMES DATASET"}, {"entity_type": "POPULATION SAMPLE", "description": "The EUR super population had a significantly higher sensitivity and negative predictive value of algorithm V compared to other algorithms.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "EUR SUPER POPULATION"}, {"entity_type": "ALGORITHM/", "description": "Statistical Algorithm", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM III"}, {"entity_type": "ALGORITHM/", "description": "Statistical Algorithm", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM IV"}, {"entity_type": "ALGORITHM/", "description": "Statistical Algorithm", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM II"}, {"entity_type": "ALGORITHM/", "description": "Statistical Algorithm", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM V"}, {"entity_type": "SUBPOPULATION SAMPLE/", "description": "Genetic Data", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "SNP-GENOTYPED SUBPOPULATION"}, {"entity_type": "GENOMIC VARIANT/", "description": "SNP Haplotype", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "HLA-DRB1*07:01-DQA1*02:01-DQB1*02:02 HAPLOTYPE"}, {"entity_type": "GENOMIC VARIANT/", "description": "SNP Markers", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "SURROGATE MARKERS"}, {"entity_type": "DISEASE/", "description": "Disease<SEP>Medical Condition", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "E. COLI ASP HYPERSENSITIVITY"}, {"entity_type": "STATISTICAL MEASURE/", "description": "Performance Metric", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "SENSITIVITY VALUE"}, {"entity_type": "STATISTICAL MEASURE/", "description": "Performance Metric", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "SPECIFICITY VALUE"}, {"entity_type": "STATISTICAL MEASURE/", "description": "Performance Metric", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "POSITIVE PREDICTIVE VALUE"}, {"entity_type": "STATISTICAL MEASURE/", "description": "Performance Metric", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "NEGATIVE PREDICTIVE VALUE"}, {"entity_type": "STATISTICAL MEASURE/", "description": "Performance Metric", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM V PERFORMANCE"}, {"entity_type": "GENOMIC DATA/", "description": "Individual Data", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "HAPLOTYPE CARRIER STATUS"}, {"entity_type": "STATISTICAL MEASURE/", "description": "Performance Metric", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM V MISSES"}, {"entity_type": "STATISTICAL MEASURE/", "description": "Classification Error", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "FALSE POSITIVES AND NEGATIVES"}, {"entity_type": "STATISTICAL MEASURE/", "description": "Performance Metric", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM V SENSITIVITY AND SPECIFICITY"}, {"entity_type": "STATISTICAL MEASURE/", "description": "Performance Metric", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "FALSE HAPLOTYPE CARRIERS"}, {"entity_type": "STATISTICAL MEASURE/", "description": "Performance Metric", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "FALSE NEGATIVES AND FALSE POSITIVES"}, {"entity_type": "STATISTICAL MEASURE/", "description": "Comparative Analysis", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "PERFORMANCE MEASURES COMPARISON"}, {"entity_type": "GENOMIC DATA SAMPLE", "description": "The large-scale genomic dataset used for comparisons and analyses.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "1000 GENOMES DATASET/"}, {"entity_type": "GEOGRAPHIC POPULATION", "description": "A subset of the European genetic population analyzed for its unique genetic markers.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "EUR SUPER POPULATION/"}, {"entity_type": "COMPARATIVE ALGORITHM", "description": "One of several algorithms being evaluated for performance in comparison to others.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM III/"}, {"entity_type": "COMPARATIVE ALGORITHM", "description": "Another algorithm used alongside algorithms II and V for comparative analysis.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM IV/"}, {"entity_type": "COMPARATIVE ALGORITHM", "description": "An additional statistical algorithm part of the study to compare against algorithm V.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM II/"}, {"entity_type": "GENOMIC VARIANT", "description": "A specific genetic sequence variant associated with E. coli ASP hypersensitivity.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "HLA-DRB1*07:01-DQA1*02:01-DQB1*02:02 HAPLOTYPE/"}, {"entity_type": "GENOMIC MARKERS", "description": "Additional genomic markers used alongside the primary SNP haplotype in screening tests for hypersensitivity identification.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "SURROGATE MARKERS/"}, {"entity_type": "MEDICAL CONDITION", "description": "A medical condition being screened for using genetic data and markers.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "E. COLI ASP HYPERSENSITIVITY/"}, {"entity_type": "STATISTICAL MEASURE", "description": "The percentage indicating how well the test correctly identifies those with E. coli ASP hypersensitivity.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "SENSITIVITY VALUE/"}, {"entity_type": "STATISTICAL MEASURE", "description": "The percentage indicating how well the test correctly identifies individuals without E. coli ASP hypersensitivity.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "SPECIFICITY VALUE/"}, {"entity_type": "STATISTICAL MEASURE", "description": "The ratio of true positives to all positive results in a screening for E. coli ASP hypersensitivity.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "POSITIVE PREDICTIVE VALUE/"}, {"entity_type": "STATISTICAL MEASURE", "description": "The probability that those identified as negative truly do not have E. coli ASP hypersensitivity.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "NEGATIVE PREDICTIVE VALUE/"}, {"entity_type": "STATISTICAL MEASURE", "description": "An evaluation of algorithm V\'s performance against benchmarks and other algorithms in the study.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "ALGORITHM V PERFORMANCE/"}, {"entity_type": "GENOMIC DATA ANALYSIS", "description": "The status identifying individuals who carry the specific HLA haplotype.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "HAPLOTYPE CARRIER STATUS/"}, {"entity_type": "DIAGNOSTIC TEST", "description": "A process for diagnosing E. coli ASP hypersensitivity using genetic information and markers.", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "id": "E. COLI ASP HYPERSENSITIVITY SCREENING/"}, {"source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "description": "were associated with DQA1*02:01\u2013DQB1*02:02 haplotype carrier status.)", "entity_type": "UNKNOWN", "id": "RS2395182 TT OR TG GENOTYPES&RS4713586 TT AND RS7775228 TC OR CC GENOTYPES"}, {"source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "description": "had perfect concordance with HLA-DRB1*07:01 allele carrier status.)", "entity_type": "UNKNOWN", "id": "G ALLELE OF RS28383172"}, {"source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "description": "had perfect concordance with HLA-DRB1*07:01 allele carrier status.)", "entity_type": "UNKNOWN", "id": "HLA TYPE"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "Statistical Analysis", "entity_type": "UNKNOWN", "id": "ALGORITHM V VS. ALGORITHMS II, III, IV"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "Statistical Analysis", "entity_type": "UNKNOWN", "id": "PERFORMANCE COMPARISON/"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "Statistical Analysis", "entity_type": "UNKNOWN", "id": "ALGORITHM V VS. HLA-DRB1*07:01-DQA1*02:01-DQB1*02:02 HAPLOTYPE"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "Correlation Analysis", "entity_type": "UNKNOWN", "id": "SENSITIVITY VALUE AND E. COLI ASP HYPERSENSITIVITY"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "Correlation Analysis", "entity_type": "UNKNOWN", "id": "STATISTICAL MEASURE/"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "Error Rate", "entity_type": "UNKNOWN", "id": "SPECIFICITY VALUE AND FALSE POSITIVES"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "Accuracy Rate", "entity_type": "UNKNOWN", "id": "POSITIVE PREDICTIVE VALUE AND TRUE CASES"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "Error Rate", "entity_type": "UNKNOWN", "id": "NEGATIVE PREDICTIVE VALUE AND FALSE NEGATIVES"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "Statistical Analysis", "entity_type": "UNKNOWN", "id": "ALGORITHM V AND ALGORITHM V HAPLOTYPE ACCURACY"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "Performance Metric", "entity_type": "UNKNOWN", "id": "ALGORITHM V AND FALSE HAPLOTYPE CARRIERS"}, {"source_id": "chunk-1f50436a36721971a0a256131d7fca62", "description": "Performance Metric", "entity_type": "UNKNOWN", "id": "MISCLASSIFICATION/"}, {"entity_type": "GENE", "description": "The TLR7 gene plays a role in the response to pathogens by producing Toll-like receptors that mediate innate immune responses.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "id": "TLR7"}, {"entity_type": "GENOMIC VARIANT", "description": "Polymorphism rs179008 in the TLR7 gene is associated with HIV-infected individuals.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "id": "RS179008"}, {"entity_type": "GENOMIC VARIANT", "description": "Polymorphisms rs179009 are found in the TLR7 gene, which follows Hardy-Weinberg equilibrium.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "id": "RS179009"}, {"entity_type": "DISEASE", "description": "The distribution of rs179009 AG genotype is significantly reduced in HIV-infected individuals compared to healthy controls.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "id": "HIV-INFECTED INDIVIDUALS"}, {"entity_type": "DISEASE", "description": "The prevalence of the TLR7 rs179009 AG genotype is lower in early, intermediate, and advanced HIV disease stages compared to healthy controls.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "id": "HIV DISEASE STAGES"}, {"entity_type": "DISEASE", "description": "In early HIV disease stage individuals, the prevalence of the TLR7 rs179009 AG genotype shows reduced frequency as compared to healthy controls.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "id": "EARLY HIV DISEASE STAGE"}, {"entity_type": "DISEASE", "description": "The frequency of the TLR7 rs179009 AG genotype is decreased in intermediate HIV disease stage individuals compared with healthy controls.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "id": "INTERMEDIATE HIV DISEASE STAGE"}, {"entity_type": "DISEASE", "description": "In advanced HIV disease stages, the prevalence of the TLR7 rs179009 GG genotype decreases as compared to healthy controls.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "id": "ADVANCED HIV DISEASE STAGE"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "The distribution of TLR7 polymorphisms was not different between HIV-infected individuals with or without tobacco and alcohol use.", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "id": "TOBACCO AND ALCOHOL USERS"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Tobacco usage is identified as an environmental factor potentially influencing HIV disease progression.", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 31}]", "id": "TOBACCO"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Alcohol use is considered another environmental factor that may impact the progression of HIV disease.", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 31}]", "id": "ALCOHOL"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Baseline CD4 cell count was found to have a correlation with HIV disease progression.", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 30}]", "id": "BASELINE CD4 COUNTS"}, {"entity_type": "GENOMIC VARIANT", "description": "Polymorphisms in the TLR7 gene were considered as genetic factors related to HIV disease progression.", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "clusters": "[{level: 0, cluster: 5}, {level: 1, cluster: 30}]", "id": "TLR7 (RS179008 AND RS179009)"}, {"entity_type": "DISEASE", "description": "HIV disease progression is the outcome being analyzed using various factors and genetic polymorphisms.", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "id": "HIV DISEASE PROGRESSION"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "Polymorphisms in the TLR7 gene are associated with decreased AG genotype prevalence in HIV-infected individuals compared to healthy controls.)", "entity_type": "UNKNOWN", "id": "TLR7 (RS179008, RS179009)"}, {"source_id": "chunk-05d583f893b703816ad8de9561aa5292", "description": "The prevalence of specific TLR7 rs179009 genotypes decreases across different stages of HIV disease compared to healthy controls.)", "entity_type": "UNKNOWN", "id": "TLR7 POLYMORPHISM"}, {"entity_type": "GENOMIC VARIANT", "description": "ALK R1275Q is a gene that plays a major role in determining the outcome of targeted cancer immunotherapies.<SEP>Among these are two well-known hotspot mutations, NRAS Q61K and ALK R1275Q.><SEP>We find at least one neoepitope-generating mutation persisting in the relapsed tumor for five of six patients (Table S1 and Supplementary Data S1 in Supplementary Material).>", "source_id": "chunk-614333487e4b92705503909d34155179<SEP>chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "ALK R1275Q"}, {"entity_type": "GENOMIC VARIANT", "description": "A reduced version of our software, ProTECT (Figure 1), was initially run on a batch of six primary:relapsed NBL sample pairs from the TARGET cohort.>", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "NRAS Q61K"}, {"entity_type": "GENOMIC VARIANT", "description": "A reduced version of our software, ProTECT (Figure 1), was initially run on a batch of six primary:relapsed NBL sample pairs from the TARGET cohort.>", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "NRAS Q61K MUTATION"}, {"entity_type": "DISEASE", "description": "We find at least one neoepitope-generating mutation persisting in the relapsed tumor for five of six patients (Table S1 and Supplementary Data S1 in Supplementary Material).>", "source_id": "chunk-52a8c1d4352526a90e26deb6f06682d2", "id": "NEUROBLASTOMA (NBL)"}, {"entity_type": "GENOMIC VARIANT", "description": "Lower affinity indicates potential for ALK neoepitopes to be involved in diseases related to HLA-A*01:01 interaction.<SEP>The presence of the ALK neoepitopes was confirmed after performing liquid chromatography\u2013mass spectroscopy (LC\u2013MS).", "source_id": "chunk-830d8819004960d1d149ac70149c72a0<SEP>chunk-d3a9c93928447522ce3df4209f8ac803", "id": "ALK NEOEPITOPE"}, {"entity_type": "TRANSCRIPT", "description": "The correct peptide with observed masses of 1,086.70 and 1,249.71 Da corresponds to the nonamer and decamer peptides.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "id": "PEPTIDE"}, {"entity_type": "DECAMER", "description": "Both ALK neoepitopes have a lower affinity for HLA-A*01:01 as indicated by the binding energy calculations.<SEP>The nonamer peptide adopts a canonical extended conformation promoted by specific anchors within the HLA groove, resulting in a \'curved\' conformation.<SEP>The thermal stability of both is different, with decamer having higher stability (47.9\u00b0C) than nonamer (46.7\u00b0C).", "source_id": "chunk-830d8819004960d1d149ac70149c72a0<SEP>chunk-d3a9c93928447522ce3df4209f8ac803", "id": "NONAMER"}, {"entity_type": "PHYSIOLOGICAL PROCESS/MOLECULAR INTERACTION", "description": "The binding affinity and stability of the decamer complex indicate it forms a higher affinity complex with HLA-B*15:01, differing from the nonamer\'s characteristics.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "clusters": "[{level: 0, cluster: 20}, {level: 1, cluster: 40}]", "id": "ALK NEOEPITOPES"}, {"entity_type": "MOLECULAR INTERACTION/ENVIRONMENTAL FACTOR", "description": "The structural analysis of the X-ray structures suggests a trend where peptide backbone conformation is influenced by its length and anchor motifs.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "id": "HLA-B*15:01/\u03922M/AQDIYRASY (NONAMER)"}, {"entity_type": "DISEASE", "description": "Comparisons with previously deposited HLA-B*15:01-restricted antigens indicate that nonamer epitopes are generally in extended conformations, consistent with the ALK-derived nonamer neoepitope\'s structure.", "source_id": "chunk-830d8819004960d1d149ac70149c72a0", "id": "PEPTIDE AMINO ACID SEQUENCE"}, {"entity_type": "PROTEIN", "description": "Ile4 residue", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "ILE4"}, {"entity_type": "PROTEIN", "description": "Y5 is a Tyrine residue", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "TYR5"}, {"entity_type": "PROTEIN", "description": "S8 represents Serine", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "SER8"}, {"entity_type": "PROTEIN", "description": "Y9 indicates Tyrine", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "id": "TYR9"}, {"entity_type": "MOLECULAR INTERACTION", "description": "2,904 HLA alleles were selected and their interactions with the ALK peptides were predicted using Rosetta comparative modeling calculations.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "HLA"}, {"entity_type": "INTERACTION AMONG MOLECULES", "description": "The approach exploits high-resolution X-ray structures to predict peptide/HLA interactions.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "PEPTIDE/HLA INTERACTIONS"}, {"entity_type": "CALCULATION METHOD", "description": "Detailed calculations were carried out using Rosetta comparative modeling for predicting interactions.<SEP>The Rosetta comparative modeling method was used for the simulation and prediction of peptide/HLA interactions.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "ROSETTA COMPARATIVE MODELING"}, {"entity_type": "MODELING APPROACH", "description": "The fixed-peptide backbone threading approach was used to confine the docking degrees of freedom.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "PEPTIDE BACKBONE THREADING"}, {"entity_type": "STRUCTURAL DATA SOURCE", "description": "The nonamer and decamer ALK peptides were modeled based on high-resolution X-ray structures.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "X-RAY STRUCTURES"}, {"entity_type": "QUANTITATIVE METRIC", "description": "Minimal variance in peptide backbone conformation was observed across all nonamer/HLA-B*15:01 structures.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "RMSD (ROOT MEAN SQUARE DEVIATION)"}, {"entity_type": "ENERGY MEASURE", "description": "Reproducible binding energies were extracted for both the nonamer and decamer peptides.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "BINDING ENERGIES"}, {"entity_type": "MOLECULAR INTERACTION", "description": "116 HLA alleles showed lower binding energies compared to initial templates, suggesting broader display potential.<SEP>The HLA alleles selected play an essential role in presenting peptides derived from mutated oncogenes.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "HLA ALLELES"}, {"entity_type": "RESIDUE SEQUENCE ANALYSIS", "description": "A sequence bias for specific residues in the HLA-binding groove was identified based on sequence identity score.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "SEQUENCE BIAS"}, {"entity_type": "DATA VISUALIZATION", "description": "The Kullback-Leibler sequence logo was used to highlight preferred residues in the MHC-binding groove.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "KULLBACK\u2013LEIBLER SEQUENCE LOGO"}, {"entity_type": "MOLECULAR ASSEMBLY", "description": "A stable p/MHC complex formed upon refolding of HLA-A*01:01 with ALK-derived peptides.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "P/MHC COMPLEX"}, {"entity_type": "GENE", "description": "ALK is an oncogene that, when mutated, plays a significant role in cancer development.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "ALK"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The ALK R1275Q neoepitopes are derived from this mutation and interact with HLA molecules to trigger T-cell responses.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "NEOPEPITOPES"}, {"entity_type": "AFFINITY MEASURE", "description": "The affinity between peptides and HLA molecules affects their interaction stability and biological function.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "PEPTIDE BINDING AFFINITY"}, {"entity_type": "MOLECULAR STRUCTURE", "description": "The decamer ALK peptide consists of 10 amino acids involved in cancer immunotherapy.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "DECAMER PEPTIDE"}, {"entity_type": "MOLECULAR STRUCTURE", "description": "The nonamer ALK peptide is a shorter, nine-amino-acid version of the ALK-derived neoepitopes.", "source_id": "chunk-614333487e4b92705503909d34155179", "id": "NONAMER PEPTIDE"}, {"entity_type": "GENE", "description": "Refolding of HLA-A*01:01 with decamer or nonamer peptide results in a stable p/MHC complex.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "HLA-A*01:01"}, {"entity_type": "NONAMER", "description": "The decamer complex crystallizes at 2.9\u2009\u00c5 resolution and shows a short 310 helix conformation of the peptide backbone when bound to HLA-A*01:01.<SEP>The thermal stability is 47.9\u00b0C for the decamer and 46.7\u00b0C for the nonamer, indicating lower affinity of both ALK neoepitopes for HLA-A*01:01 compared to HLA-B*15:01.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "DECAMER"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The conformation is indicative of peptide interaction with groove of MHC molecule.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "310 HELIX CONFORMATION"}, {"entity_type": "ALLOPHYCOCYANIN (APC) OR PHYCOERYTHRIN (PE)", "description": "Double staining assay using HLA-matched tetramers reveals reactive CD8+ T cells to both nonamer and decamer peptides.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "X-RAY STRUCTURE"}, {"entity_type": "DOUBLE STAINING", "description": "Simultaneous staining experiments with nonamer and decamer tetramers did not identify CD8+ T cells recognizing both epitopes simultaneously.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "NONAMER/DECAMER"}, {"entity_type": "GENE", "description": "The HLA and MHC allele has been previously reported to yield partially folded, peptide-free molecules with measurable thermal stabilities.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "HLA-B*15:01"}, {"entity_type": "X-RAY STRUCTURE", "description": "Crystallizes in the P32221 space group at a resolution of 2.9\u2009\u00c5 and reveals that the peptide backbone adopts a short 310 helix conformation when bound to HLA-A*01:01.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "DECAMER COMPLEX"}, {"entity_type": "MOLECULAR INTERACTION", "description": "The side-chain hydroxyl group of the peptide Tyr10 is in contact with Trp, Lys, and Tyr side-chain atoms from the F-pocket of the MHC groove.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "TRYPTOPHAN (TRP)"}, {"entity_type": "MOLECULAR INTERACTION", "description": "When bound to HLA-A*01:01, the side chain of Arg6 flips outwards altering the peptide surface displayed to TCRs compared to its binding when it binds with HLA-B*15:01.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "ARGININE (ARG)"}, {"entity_type": "RECEPTOR", "description": "The CD8+ T cells interact with the altered-self neoepitopes through their TCRs as a part of immune response.", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "id": "T CELL RECEPTOR (TCR)"}, {"entity_type": "DISEASE", "description": "A disease associated with immune response and antigen presentation.", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "id": "KNOWN IMMUNODOMINANT EPITOPE"}, {"entity_type": "GENE", "description": "Description of unknown gene X", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "id": "UNKNOWN GENE X"}, {"entity_type": "GENOMIC VARIANT", "description": "Description of genomic variant Y", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "id": "UNKNOWN VARIANT Y"}, {"entity_type": "TRANSCRIPT", "description": "Description of transcript Z", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "id": "TRANSCRIPT Z"}, {"entity_type": "PROTEIN", "description": "Description of protein A", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "id": "PROTEIN A"}, {"entity_type": "NUTRIENT", "description": "Description of nutrient B", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "id": "NUTRIENT B"}, {"entity_type": "DIETARY PATTERN", "description": "Description of dietary pattern C", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "id": "DIETARY PATTERN C"}, {"entity_type": "PHYSIOLOGICAL PROCESS", "description": "Description of physiological process D", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "id": "PHYSIOLOGICAL PROCESS D"}, {"entity_type": "METABOLIC PATHWAY", "description": "Description of metabolic pathway E", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "id": "METABOLIC PATHWAY E"}, {"entity_type": "MOLECULAR INTERACTION", "description": "Description of molecular interaction F", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "id": "MOLECULAR INTERACTION F"}, {"entity_type": "ENVIRONMENTAL FACTOR", "description": "Description of environmental factor G", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "id": "ENVIRONMENTAL FACTOR G"}, {"entity_type": "DISEASE", "description": "Description of disease H", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "id": "DISEASE H"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "Asp46", "entity_type": "UNKNOWN", "id": "SELF-PEPTIDE/HLA-B*15:01 COMPLEX"}, {"source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "description": "Asp46", "entity_type": "UNKNOWN", "id": "STRUCTURAL EXCLUSION AND IMMUNE TOLERANCE<|INABILITY OF SELF-PEPTIDE TO FORM STABLE COMPLEXES WITH HLA DUE TO STRUCTURAL CONSTRAINTS LEADING TO IMMUNE TOLERANCE.)(ENTITY"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "Structure-based modeling methods were previously applied to predict peptide/HLA interactions using this template.)", "entity_type": "UNKNOWN", "id": "PEPTIDE/HLA MODELING"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "The peptide backbone conformation showed minimal variance in all nonamer/HLA-B*15:01 structures.)", "entity_type": "UNKNOWN", "id": "PEPTIDE BACKBONE CONFORMATION"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "The peptide backbone conformation showed minimal variance in all nonamer/HLA-B*15:01 structures.)", "entity_type": "UNKNOWN", "id": "VARIANCE ANALYSIS"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "An allele with mutated residues exhibited a high binding energy and was distant from the best binder.)", "entity_type": "UNKNOWN", "id": "BINDING AFFINITY"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "An allele with mutated residues exhibited a high binding energy and was distant from the best binder.)", "entity_type": "UNKNOWN", "id": "MOCK POLYALA HLA"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "A trend for specific residues contributing to overall binding energy was observed.)", "entity_type": "UNKNOWN", "id": "RESIDUES IN MHC BINDING GROOVE"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "A trend for specific residues contributing to overall binding energy was observed.)", "entity_type": "UNKNOWN", "id": "SEQUENCE BIAS EVALUATION"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "Highly invariant residues were consistently found in HLA alleles with high affinity binding.)", "entity_type": "UNKNOWN", "id": "INVARIANT RESIDUES"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "Highly invariant residues were consistently found in HLA alleles with high affinity binding.)", "entity_type": "UNKNOWN", "id": "KEY FEATURES"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "Similar polar contacts, primarily in the F-pocket of MHC, correlated well with positions of invariant MHC residues.)", "entity_type": "UNKNOWN", "id": "CONTACT PATTERN"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "Similar polar contacts, primarily in the F-pocket of MHC, correlated well with positions of invariant MHC residues.)", "entity_type": "UNKNOWN", "id": "F-POCKET"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "A stable p/MHC complex formed after refolding nonamer and decamer peptides with HLA-A*01:01.)", "entity_type": "UNKNOWN", "id": "STABILITY MEASURE"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "This mutation in the ALK gene serves as an indicator for potential cancer development or progression.)", "entity_type": "UNKNOWN", "id": "ALK R1275Q MUTATION"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "This mutation in the ALK gene serves as an indicator for potential cancer development or progression.)", "entity_type": "UNKNOWN", "id": "CANCER MARKER"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "The selection of HLA alleles aims to broaden the display repertoire that can accommodate neoepitopes and enhance immune response.)", "entity_type": "UNKNOWN", "id": "HLA ALLELES SELECTION"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "The selection of HLA alleles aims to broaden the display repertoire that can accommodate neoepitopes and enhance immune response.)", "entity_type": "UNKNOWN", "id": "REPERTOIRE EXPANSION"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "Comparative analysis of peptide sequences helps identify those with higher affinity or potential for stronger T-cell responses.)", "entity_type": "UNKNOWN", "id": "PEPTIDE SEQUENCE SIMILARITY"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "The Kullback-Leibler divergence measure is used to quantify sequence diversity among HLA alleles involved in peptide binding.)", "entity_type": "UNKNOWN", "id": "KULLBACK-LEIBLER DIVERGENCE"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "The Kullback-Leibler divergence measure is used to quantify sequence diversity among HLA alleles involved in peptide binding.)", "entity_type": "UNKNOWN", "id": "SEQUENCE DIVERSITY METRIC"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "Analyze the nature and specificity of polar contacts between peptides, MHC molecules, and T cells during immune response interactions.)", "entity_type": "UNKNOWN", "id": "MOLECULAR INTERACTION DETAIL"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "Analyze the nature and specificity of polar contacts between peptides, MHC molecules, and T cells during immune response interactions.)", "entity_type": "UNKNOWN", "id": "POLAR CONTACTS ANALYSIS"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "The thermal stability test confirms the robustness of p/MHC complexes formed with selected HLA alleles under physiological conditions.)", "entity_type": "UNKNOWN", "id": "BIOPHYSICAL PROPERTY"}, {"source_id": "chunk-614333487e4b92705503909d34155179", "description": "The thermal stability test confirms the robustness of p/MHC complexes formed with selected HLA alleles under physiological conditions.)", "entity_type": "UNKNOWN", "id": "THERMAL STABILITY ASSESSMENT"}, {"source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "description": "Frequency of recognition is comparable for neoepitopes and an immunodominant epitope, indicating similar interaction potential.)", "entity_type": "UNKNOWN", "id": "RECOGNITION"}, {"source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "description": "Both peptides have nominal frequencies comparable to known immunodominant epitopes when recognized by CD8+ T cells.)", "entity_type": "UNKNOWN", "id": "T CELL RECOGNITION"}, {"source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "description": "The lower affinity for HLA-A*01:01 might imply involvement in disease mechanisms related to these neoepitopes.)", "entity_type": "UNKNOWN", "id": "ALK NEOEPITOPE (NONAMER AND DECAMER)"}, {"source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "description": "The lower affinity for HLA-A*01:01 might imply involvement in disease mechanisms related to these neoepitopes.)", "entity_type": "UNKNOWN", "id": "DISEASE MECHANISMS"}, {"source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "description": "This epitope is present on the surface of epithelial cells, participating in the immune response.)", "entity_type": "UNKNOWN", "id": "EPITHELIAL CELLS"}, {"source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "description": "Gene X is involved in a known immune response process.)", "entity_type": "UNKNOWN", "id": "KNOWN IMMUNE RESPONSE I"}, {"source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "description": "Genomic variant Y may contribute to an unknown genetic condition.)", "entity_type": "UNKNOWN", "id": "UNKNOWN GENETIC CONDITION J"}, {"source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "description": "Transcript Z codes for multiple proteins, including K and L.)", "entity_type": "UNKNOWN", "id": "PROTEINS K, L"}, {"source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "description": "Dietary pattern C influences metabolic pathway D through nutrient intake.)", "entity_type": "UNKNOWN", "id": "METABOLIC PATHWAY D"}, {"source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "description": "Physiological process D depends on molecular interaction E for its function.)", "entity_type": "UNKNOWN", "id": "MOLECULAR INTERACTION E"}, {"entity_type": "GENOMIC VARIANT", "description": "A single nucleotide polymorphism (SNP) that showed genome-wide significance in TwinGene GWAS.<SEP>A single nucleotide polymorphism in 3p14.1 achieved genome-wide significance in the TwinGene study.<SEP>The expression level of this genomic variant is associated with the risk of cardiovascular disease.<SEP>This variant may influence protein function and potentially lead to cardiovascular disease.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "RS74420772"}, {"entity_type": "DISEASE", "description": "A small nested case\u2013control study found IgM anti-PC to be lower in prevalent CLL cases than in matched controls (P = 0.006).<SEP>Inflammatory response against PC (Phosphatidylcholine) which is associated with various diseases including autoimmune disorders.<SEP>Polygenic risk scores for IgM levels may inform the choice of therapies targeting autoimmunity or inflammatory conditions.<SEP>The expression levels of IgM antibodies against PC are potentially influenced by genetic variants such as rs735665 and rs9392504.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b<SEP>chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "IGM ANTI-PC"}, {"entity_type": "GENOMIC VARIANT", "description": "Identified as a top risk allele for Chronic Lymphocytic Leukemia (CLL) and significant for general IgM levels.<SEP>Recognized as a top risk allele for CLL (Chronic lymphocytic leukemia) and specific for general IgM.<SEP>The variant rs735665 has been found to be linked with specific dietary patterns affecting general IgM levels.<SEP>This genomic variant has regulatory effects on transcription factors and histone modifications in leukemia or immune-cell lines.<SEP>This variant plays a significant role in polygenic risk score calculations related to immunoglobulin levels.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "RS735665"}, {"entity_type": "GENOMIC VARIANT", "description": "One of the 33 independent genome-wide significant SNPs identified in the study related to CLL.<SEP>One of the genome-wide significant SNPs for CLL identified in this study.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "RS9392504"}, {"entity_type": "GENE", "description": "Switching from the major allele T/A at rs35923643 to the minor C/G allele impeded the predicted binding affinity of RUNX3, making the P-value increase 10-fold.<SEP>The genomic variant located at the LD block was scanned for regulatory marks in leukemia- or immune-cell lines.<SEP>This variant is located within the GRAM domain containing 1B (GRAMD1B) gene.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b<SEP>chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "RS35923643"}, {"entity_type": "IMMUNE SYSTEM COMPONENT", "description": "A type of immunoglobulin that plays a role in various immune responses, including autoimmune diseases and cancer risk.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "IGM"}, {"entity_type": "EPIGENETIC LANDMARK", "description": "Indicates regions with increased accessibility to DNase I enzyme, which is associated with transcriptional regulation, including regulatory functions of genomic variants like rs735665.", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "id": "DNASEI HYPERSENSITIVITY SITES"}, {"entity_type": "HISTONE MARK", "description": "Peaks of histone mark monomethylated histone H3 lysine 4 (H3K4Me1) were found at the position of rs35923643.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "H3K4ME1"}, {"entity_type": "DNASEI HYPERSENSITIVITY", "description": "Deoxyribonuclease I (DNaseI) hypersensitivity was also detected at that location.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "DNASEI HYPERSENSITIVITY"}, {"entity_type": "TRANSCRIPTION FACTORS", "description": "There were suggested binding sites for 22 different transcription factors within the ~700bp region around rs35923643.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "TFS"}, {"entity_type": "TRANSCRIPTION FACTOR", "description": "The strongest signals among matched sequences from Find Individual Motif Occurrences (FIMO) tool were observed for Runt-related TF 3 (RUNX3).", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "RUNX3"}, {"entity_type": "TRANSCRIPTION FACTOR", "description": "Another transcription factor was SPI1.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "SPI1"}, {"entity_type": "MOTIF", "description": "RUNX3 binding motifs included the SNP rs35923643 (at position 391 bp).", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "RUNX3 BINDING MOTIFS"}, {"entity_type": "TRANSCRIPTION FACTOR BINDING", "description": "Haib_RUNX3_GM12878_Motif1_fw_ic0 was found from a B-cell line when binding RUNX3.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "HAIB_RUNX3_GM12878_MOTIF1_FW_IC0"}, {"entity_type": "DATABASE", "description": "The RegulomeDB database gave support for rs35923643 affecting the binding of transcription factors in general.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "REGULOMEDB DATABASE"}, {"entity_type": "CASES", "description": "The difference of IgM anti-PC levels between CLL cases and matched controls was dependent on the time between sampling and diagnosis.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "PREVALENT CLL CASES"}, {"entity_type": "CASES", "description": "No association between IgM anti-PC and incident CLL risk estimated from stratified Cox proportional hazards model (P = 0.354).", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "INCIDENT CLL CASES"}, {"entity_type": "TEMPORAL FACTOR", "description": "The difference of IgM anti-PC levels showed a declining trend but was not statistically significant (\u03b2 = \u20130.08, P = 0.10).", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "TIME BETWEEN SAMPLING AND DIAGNOSIS"}, {"entity_type": "CELL LINE", "description": "The cell line GM12878 was used for histone mark monomethylated histone H3 lysine 4 (H3K4Me1) and deoxyribonuclease I (DNaseI) hypersensitivity scans.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "GM12878"}, {"entity_type": "CELL LINE", "description": "The cell line CD34+ was used for deoxyribonuclease I (DNaseI) hypersensitivity scan.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "CD34+ CELLS"}, {"entity_type": "CELL LINE", "description": "The cell line GM12865 was also used for deoxyribonuclease I (DNaseI) hypersensitivity scan.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "GM12865"}, {"entity_type": "TECHNIQUE", "description": "Chromatin immunoprecipitation-sequencing (ChIP-seq) experiments from Encyclopedia of DNA Elements (ENCODE) were used to detect suggested binding sites for transcription factors.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "CHROMATIN IMMUNOPRECIPITATION-SEQUENCING (CHIP-SEQ)"}, {"entity_type": "DATABASE", "description": "The Chromatin immunoprecipitation-sequencing (ChIP-seq) data came from the Encyclopedia of DNA Elements (ENCODE) database.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "ENCYCLOPEDIA OF DNA ELEMENTS (ENCODE)"}, {"entity_type": "TOOL", "description": "The tool used for finding individual transcription factor motifs was Find Individual Motif Occurrences (FIMO).", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "FIND INDIVIDUAL MOTIF OCCURRENCES (FIMO)"}, {"entity_type": "STUDY DESIGN", "description": "A small nested case-control study design was utilized to compare IgM anti-PC levels between prevalent CLL cases and matched controls.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "NESTED CASE-CONTROL STUDY"}, {"entity_type": "STATISTICAL METHOD", "description": "The stratified Cox proportional hazards model was used to estimate the association between IgM anti-PC and incident CLL risk.", "source_id": "chunk-83ee45fa1acb6546ff72e454c7880f7b", "id": "STRATIFIED COX PROPORTIONAL HAZARDS MODEL"}, {"source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "description": "rs735665 is significantly associated with the risk of CLL, indicating its role in this disease.)", "entity_type": "UNKNOWN", "id": "CLL"}, {"source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "description": "rs735665 affects the expression level of general IgM, suggesting a genetic basis for this immune response.)", "entity_type": "UNKNOWN", "id": "GENERAL IGM"}, {"source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "description": "rs735665 contributes substantially to the polygenic risk scores for increasing IgM, highlighting its genetic influence on this measure.)", "entity_type": "UNKNOWN", "id": "POLYGENIC RISK SCORE FOR IGM"}, {"source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "description": "Genetic variations like rs735665 and rs9392504 can influence the expression of IgM antibodies against PC.)", "entity_type": "UNKNOWN", "id": "RS735665, RS9392504"}, {"source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "description": "The genetic variation rs35923643 resides in the GRAMD1B gene, indicating a possible role of this variant within the gene\'s function.)", "entity_type": "UNKNOWN", "id": "GRAMD1B"}, {"source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "description": "The variant rs735665 is linked to changes in epigenetic marks like H3K4Me1, indicating its potential role in gene regulation.)", "entity_type": "UNKNOWN", "id": "REGULATORY MARK"}, {"source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "description": "The polygenic risk scores associated with IgM levels can guide personalized treatment strategies for autoimmune disorders like CLL.)", "entity_type": "UNKNOWN", "id": "POLYGENIC RISK SCORE"}, {"source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "description": "The polygenic risk scores associated with IgM levels can guide personalized treatment strategies for autoimmune disorders like CLL.)", "entity_type": "UNKNOWN", "id": "THERAPIES FOR AUTOIMMUNE DISEASES"}], "links": [{"weight": 2.0, "description": "The expression of SEC16B gene may be influenced by its interaction with endoplasmic reticulum processes.)", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "order": 1, "source": "ENDOPLASMIC RETICULUM", "target": "GENE"}, {"weight": 3.0, "description": "SNP rs713586 affects the activity levels of DNAJC27 involved in cellular stress response and protein folding.)", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "order": 1, "source": "CELLULAR PROCESSES", "target": "DNAJ HEAT SHOCK PROTEIN FAMILY (HSP40)"}, {"weight": 4.0, "description": "SNP rs9356744 impacts the function of CDKAL1, influencing metabolic pathways and energy balance.)", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "order": 1, "source": "METABOLIC REGULATION", "target": "CDK5 REGULATORY SUBUNIT-ASSOCIATED PROTEIN 1-LIKE 1 (CDKAL1)"}, {"weight": 5.0, "description": "The SNP variants affect TFAP2B\'s ability to regulate gene expression related to obesity risk.)", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "order": 1, "source": "TRANSCRIPTION FACTOR ACTIVITY", "target": "TRANSCRIPTION FACTOR AP-2 BETA (TFAP2B)"}, {"weight": 2.0, "description": "PRS interacts with menarche age to influence obesity risk.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "PRS", "target": "MENARCHE AGE"}, {"weight": 3.0, "description": "BMI varies by menarche age in participants, influenced by PRS.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "MENARCHE AGE", "target": "BMI"}, {"weight": 2.0, "description": "In individuals without a university education, the FTO rs9939609 polymorphism was linked with higher BMI. Conversely, in those with university studies, no such association or even an inverse effect of A-allele was noted.)", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "order": 1, "source": "BMI", "target": "EDUCATION"}, {"weight": 1.0, "description": "Type 2 diabetes and end-stage renal disease (T2D-ESRD) case subjects showed different mean BMI levels compared to nondiabetic control subjects, indicating potential influence or interaction.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "BMI", "target": "T2D-ESRD"}, {"weight": 1.0, "description": "Evidence for interaction with BMI in relation to the association between TCF7L2 gene and the number of risk alleles was not found.)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "BMI", "target": "INTERACTION"}, {"weight": 0.7, "description": "Increased two-hour post-meal glucose levels are negatively correlated with baseline BMI but positively related to BMI change over time, suggesting different impacts on obesity risk factors.)", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "order": 1, "source": "BMI", "target": "CHANGE IN 2HPG LEVELS"}, {"weight": 1.44, "description": "No significant main effect was observed for the rs1011527 SNP in LEPR on BMI without considering interaction terms.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "BMI", "target": "RS1011527 SNP IN LEPR"}, {"weight": 7.0, "description": "Lower BMI was linked to higher adherence to the Mediterranean diet.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "BMI", "target": "MD ADHERENCE"}, {"weight": 5.0, "description": "The association between genotype and BMI in the baseline demographics.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "BMI", "target": "GENOTYPE GROUP"}, {"weight": 0.0, "description": "The SNP heritability for BMI showed a positive association with the trait of snoring, though not statistically significant (rG=0.695, SE=0.184, p-value=2*10^(-3)[ldsc \u03c72-test]).)", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "order": 1, "source": "BMI", "target": "SNP HERITABILITY"}, {"weight": 0.0, "description": "The SNP heritability for sleep apnoea was positively correlated with BMI\'s heritability (rG = 0.683), reflecting obesity as a risk factor.)", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "order": 1, "source": "BMI", "target": "SNP HERITABILITY FOR SLEEP APNOEA"}, {"weight": 4.0, "description": "After adjusting for body mass index (BMI), a significant amount of predisposition to snoring is not fully explained by BMI.)", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "order": 1, "source": "BMI", "target": "SNORING"}, {"weight": 8.0, "description": "BMI was found to exert a causal effect on snoring, and whole-body fat mass also showed bidirectional relationships.)", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "order": 1, "source": "BMI", "target": "WHOLE-BODY FAT MASS"}, {"weight": 10.0, "description": "GWAS results suggest a possible bidirectional causal relationship between BMI and heart attack; BMI causes an increase in the risk of heart attack, which could be linked to snoring through sleep disruption or other factors.)", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "order": 1, "source": "BMI", "target": "HEART ATTACK"}, {"weight": 1.0, "description": "Increased BMI was observed among women with PCOS compared to control subjects.)", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "order": 1, "source": "BMI", "target": "PCOS"}, {"weight": 4.0, "description": "Gene-based PheWAS analysis for LCT locus shows strong links with BMI, suggesting potential impact on metabolic traits.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "BMI", "target": "LCT (BIFIDOBACTERIUM)"}, {"weight": 2.0, "description": "rs10399931 showed a significant association (p = 0.03) with BMI, indicating its relevance to body mass index.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "BMI", "target": "RS10399931"}, {"weight": 0.722, "description": "The BMI in IBS-D subgroup was approximately 4 units higher than that in IBS-C and HV, indicating a potential link to body weight regulation and digestive conditions.)<SEP>The BMI in IBS-D subgroup was approximately 4 units higher than that in IBS-C and HV.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "BMI", "target": "IBS-D"}, {"weight": 0.0, "description": "There was a positive association between BMI and serum C4 levels, though not statistically significant.)<SEP>While there was no significant association between BMI and serum C4 levels, they may still play roles independently in body weight regulation and metabolic processes.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "BMI", "target": "SERUM C4"}, {"weight": 5.0, "description": "In low plant-based diet, genetic impact on BMI is higher in those with high PRS compared to others.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "LOW PLANT-BASED DIET INTAKE & HIGH PRS", "target": "OBESITY RISK"}, {"weight": 4.0, "description": "Plant-based diet intake and PRS have an interaction influencing obesity risk.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "OBESITY RISK", "target": "PLANT-BASED DIET INTAKE& PRS"}, {"weight": 6.0, "description": "Fried-food intake and PRS show interaction where PRS offset the obesity risk effect of high fried-food intake.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "OBESITY RISK", "target": "HIGH FRIED-FOOD INTAKE& PRS"}, {"weight": 3.0, "description": "In GP subjects, obesity risk was inversely proportional to educational level; university-educated individuals had a lower obesity risk.)", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "order": 1, "source": "OBESITY RISK", "target": "EDUCATION"}, {"weight": 2.0, "description": "Influence of educational attainment on obesity risk, as it interacts with the presence of certain genotypes (such as FTO rs9939609) to impact health outcomes like BMI and obesity prevalence.)", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "order": 1, "source": "OBESITY RISK", "target": "EDUCATION LEVEL"}, {"weight": 1.0, "description": "The determination of obesity risk involves multiple factors including genetic variations and education levels.)", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "order": 1, "source": "OBESITY RISK", "target": "DISEASE"}, {"weight": 4.0, "description": "The inclusion of these covariates is to adjust for confounding factors when assessing the association between alcohol consumption and cortisol levels.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "COVARIATES", "target": "EDUCATION STATUS, HYPERTENSION, DIABETES"}, {"weight": 1.0, "description": "The FTO SNP shows significant association with BMI; higher BMI values are observed among individuals carrying specific allele variations of the polymorphism.)", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "order": 1, "source": "BMI (BODY MASS INDEX)", "target": "FTO RS9939609 POLYMORPHISM"}, {"weight": 1.0, "description": "SNPs within these genes may influence their function and impact the obesity risk.)", "source_id": "chunk-fa6293a0a10f175b4aebac8e885c0d20", "order": 1, "source": "GENE", "target": "GENOMIC VARIANT"}, {"weight": 0.0, "description": "The CFB/C2 gene variants were also explored in relation to AMD, but their main effects did not significantly influence AMD risk in combination with HLS.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "GENE", "target": "CFB/C2 GENE"}, {"weight": 2.0, "description": "The PEMT M175V minor V allele was associated with male infertility status among homozygous individuals.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "GENE", "target": "PEMT M175V MINOR V ALLELE"}, {"weight": 3.0, "description": "The TCblR variant rs173665 was negatively associated with male infertility status among heterozygous subjects.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "GENE", "target": "TCBLR (CD320) VARIANT RS173665 IN HETEROZYGOUS SUBJECTS"}, {"weight": 7.0, "description": "Molecular analyses identified mosaicism for a specific P1001L mutation affecting the ATP7A gene, which might indicate Menkes disease in the child.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "GENE", "target": "P1001L MUTATION"}, {"weight": 2.0, "description": "SNPs can be related to genes when they influence their methylation, as in the case of rs726430 with BCL2L11 and rs16861015 with CPB1.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "GENE", "target": "SNP"}, {"weight": 3.0, "description": "A CpG site can be related to a gene if it\'s annotated to that gene, as seen with cg07547695 and BCL2L11.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "GENE", "target": "CPG SITE"}, {"weight": 7.0, "description": "SNPs associated with FEV1/FVC were found downstream of the C10orf11 gene on chromosome 10q22.3.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "GENE", "target": "C10ORF11 GENE"}, {"weight": 11.0, "description": "A SNP in this gene was associated with FEV1 and FVC.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "GENE", "target": "IL37 GENE"}, {"weight": 1.0, "description": "The text seems to be discussing one genomic variant and not describing relationships between variants.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "GENOMIC VARIANT", "target": "MINOR ALLELE"}, {"weight": 6.0, "description": "Mosaicism for a P1001L mutation was confirmed in genomic DNA, but not detected in the mother\'s DNA; this mutation is associated with Menkes disease.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "GENOMIC VARIANT", "target": "MOLECULAR ANALYSES"}, {"weight": 10.0, "description": "Subject\'s Fanconi anemia phenotype can be attributed to the combination of genomic variants, including mutations in UBE2T.)", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "order": 1, "source": "GENOMIC VARIANT", "target": "FA"}, {"weight": 2.0, "description": "There is an association between specific genotypes and the occurrence of Colorectal Cancer (CRC).)", "source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "order": 1, "source": "GENOMIC VARIANT", "target": "COLORECTAL CANCER"}, {"weight": 2.0, "description": "A genomic variant in the NOS1 gene is linked to an increased risk of cardiovascular disease.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "GENOMIC VARIANT", "target": "CARDIOVASCULAR DISEASE"}, {"weight": 1.0, "description": "The \'low FEV1/FVC alleles\' showed an additive genetic architecture that affected lung function measurements.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "GENOMIC VARIANT", "target": "LOW FEV1/FVC ALLELES"}, {"weight": 1.0, "description": "A replicated locus (THSD4-UACA-TLE3), despite being one of two replicated loci, did not show functional connections with other genes defined by the 80 SNPs.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "GENOMIC VARIANT", "target": "THSD4-UACA-TLE3"}, {"weight": 1.0, "description": "Replicated locus (C10orf11) was another example where no functional connection to other identified genes existed.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "GENOMIC VARIANT", "target": "C10ORF11"}, {"weight": 2.0, "description": "A specific genetic locus, CCL23-CCL18 SNP, demonstrated significant functional interactions with \u03b2-defensin genes and TNFRSF13B along with CXCL12.)<SEP>The SNP at the CCL23-CCL18 locus demonstrated significant connections with \u03b2-defensin genes and TNFRSF13B, as well as CXCL12.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "GENOMIC VARIANT", "target": "CCL23-CCL18 LOCUS"}, {"weight": 1.0, "description": "Single Nucleotide Polymorphisms were analyzed to identify specific genetic variations connected through functional gene interactions.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "GENOMIC VARIANT", "target": "SNP"}, {"weight": 1.0, "description": "Although a replicated genetic locus, THSD4-UACA-TLE3 was found not to be functionally connected with other genes identified by the 80 SNPs.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "GENOMIC VARIANT", "target": "THSD4-UACA-TLE3 LOCUS"}, {"weight": 1.0, "description": "Another replicated genomic region (C10orf11) did not show any functional connections to the other genes defined through the analysis of the 80 SNPs.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "GENOMIC VARIANT", "target": "C10ORF11 LOCUS"}, {"weight": 0.005, "description": "The association of rs1893217 and response rate was tested in anti-CTLA-4 treated patients, with an OR value of approximately 2.79 for non-responders.)", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "order": 1, "source": "GENOMIC VARIANT", "target": "RESPONSE RATE"}, {"weight": 0.0002, "description": "rs17388568 shows a strong association with response rate specifically in anti-PD-1 treatment.)", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "order": 1, "source": "GENOMIC VARIANT", "target": "RESPONSE RATE FOR ANTI-PD-1"}, {"weight": 3.0, "description": "A plant-based diet may mitigate obesity risk for individuals with low PRS.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "PLANT-BASED DIET INTAKE& LOW PRS", "target": "REDUCTION IN OBESITY RISK"}, {"weight": 4.0, "description": "Despite high PRS, high fried-food intake might not significantly increase obesity risk due to counterbalancing factors.)", "source_id": "chunk-3ad8ef794841d749e48b3223741e91d3", "order": 1, "source": "HIGH FRIED-FOOD INTAKE& HIGH PRS", "target": "OFFSET OF OBESOGENIC EFFECTS"}, {"weight": 3.64, "description": "The risk allele (A) was significantly associated with a 1.51-kg greater weight loss in the high-protein group.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "FTO RS1558902 GENOTYPE", "target": "WEIGHT"}, {"weight": 2.73, "description": "The risk allele (A) was significantly associated with a smaller change in waist circumference compared to other groups.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "FTO RS1558902 GENOTYPE", "target": "WAIST"}, {"weight": 4.6, "description": "The risk allele carriers who followed a high-protein diet had greater loss in body composition over time.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "FTO RS1558902 GENOTYPE", "target": "BODY COMPOSITION CHANGE OVER TIME"}, {"weight": 3.64, "description": "The risk allele (A) was significantly associated with a 1.51-kg greater weight loss in the high-protein group.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "FTO RS1558902 GENOTYPE", "target": "WEIGHT LOSS"}, {"weight": 2.73, "description": "The risk allele (A) was significantly associated with a smaller change in waist circumference compared to other groups.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "FTO RS1558902 GENOTYPE", "target": "WAIST CIRCUMFERENCE"}, {"weight": 4.6, "description": "The risk allele carriers who followed a high-protein diet showed greater loss in body composition over time.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "FTO RS1558902 GENOTYPE", "target": "BODY COMPOSITION CHANGE OVER TIME RESPONSE"}, {"weight": 6.0, "description": "The response to different diets is influenced by trunk fat percentage dynamics, which are a measure of body composition changes.)", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "order": 1, "source": "TRUNK FAT", "target": "DIETARY PATTERN"}, {"weight": 2.0, "description": "Participants with the AA genotype experienced less loss in TAT and SAT when following a low-protein diet.)", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "order": 1, "source": "GENOTYPE", "target": "LOW-PROTEIN DIET"}, {"weight": 4.0, "description": "Genotype did not show significant interactions with changes in fat distribution when following low-fat diet, as opposed to other dietary patterns.)", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "order": 1, "source": "GENOTYPE", "target": "LOW-FAT DIET"}, {"weight": 5.0, "description": "No significant genotype-time interactions on changes in fat distribution were found when following low-fat diets, highlighting the role of genetic background in dietary responses.)", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "order": 1, "source": "GENOTYPE", "target": "LOW-FAT DIET INTERACTION"}, {"weight": 2.0, "description": "There were differences by genotype in TAG levels in both strata for each of the three factors mentioned.)", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "order": 1, "source": "GENOTYPE", "target": "MOLECULAR INTERACTION"}, {"weight": 0.89, "description": "A Genotype \u00d7 Condition interaction was observed for response bias, with val carriers having significantly lower response bias in the stress condition.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "GENOTYPE", "target": "RESPONSE BIAS"}, {"weight": 2.0, "description": "Genotype rs4988235 was associated with milk drinking in controls.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "GENOTYPE", "target": "CONTROLS"}, {"weight": 12.0, "description": "No substantial differences were observed between rs4988235 genotype and RCC risk based on milk consumption.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "GENOTYPE", "target": "MILK CONSUMPTION"}, {"weight": 2.0, "description": "The genotype effect of EBNA2 involves altering allelic association with expression compared to that found in uninfected B cells.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "GENOTYPE", "target": "EBNA2 EFFECT"}, {"weight": 8.0, "description": "The specific genotypes at these MS risk loci increase the individual\'s susceptibility to diseases like multiple sclerosis (MS), including those related to immune evasion or other disorders.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "GENOTYPE", "target": "DISEASE RISK"}, {"weight": 4.6, "description": "Individuals with the AA genotype experienced less fat loss when consuming a low-protein diet compared to those carrying the variant.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "LOW-PROTEIN DIET", "target": "FAT LOSS"}, {"weight": 2.0, "description": "Diazoxide treatment was used in combination with a low-protein diet for patients with HI/HA syndrome.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "LOW-PROTEIN DIET", "target": "DIAZOXIDE TREATMENT"}, {"weight": 4.7, "description": "When following a high-protein diet, participants carrying the AA genotype of the FTO rs1558902 showed greater fat loss compared to other genotypes.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "HIGH-PROTEIN DIET", "target": "FAT LOSS"}, {"weight": 3.0, "description": "Individuals having the AA genotype showed greater decrease in fat distribution upon consumption of high-protein diets compared to others.)", "source_id": "chunk-a321a6d63755366a1fc3eafbd9fbce83", "order": 1, "source": "HIGH-PROTEIN DIET", "target": "AA GENOTYPE"}, {"weight": 4.56, "description": "High-protein intake was associated with greater weight loss when the AA genotype of the FTO rs1558902 variant is present.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "PROTEIN INTAKE", "target": "WEIGHT LOSS"}, {"weight": 4.56, "description": "High-protein intake was associated with greater weight loss when the AA genotype of the FTO rs1558902 variant is present.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "PROTEIN INTAKE", "target": "WEIGHT LOSS RESPONSE"}, {"weight": 8.6, "description": "The risk allele carriers in the high-protein group had greater fat loss compared to non-carriers of the AA genotype.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "AA GENOTYPE (FTO RS1558902)", "target": "FAT LOSS IN HIGH-PROTEIN DIET INTAKE"}, {"weight": 8.4, "description": "Participants with the AA genotype had less fat loss when assigned to the low-protein diet compared to those without this genotype.)<SEP>Participants with the AA genotype had less loss of trunk fat and SAT when assigned to the low-protein diet compared to those without this genotype.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "AA GENOTYPE (FTO RS1558902)", "target": "FAT LOSS IN LOW-PROTEIN DIET INTAKE"}, {"weight": 8.8, "description": "The risk allele carriers in the high-protein group had greater fat loss in TAT and VAT compared to non-carriers.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "AA GENOTYPE (FTO RS1558902)", "target": "FAT DISTRIBUTION IN HIGH-PROTEIN DIET INTAKE"}, {"weight": 8.6, "description": "Participants with the AA genotype were associated with less fat loss of trunk fat when assigned to the low-protein diet compared to those without this genotype.)<SEP>Participants with the AA genotype were associated with less loss of SAT when assigned to the low-protein diet compared to those without this genotype.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "AA GENOTYPE (FTO RS1558902)", "target": "FAT DISTRIBUTION IN LOW-PROTEIN DIET INTAKE"}, {"weight": 4.7, "description": "Participants with the AA genotype experienced greater fat loss when consuming a high-protein diet compared to those without this variant.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "AA GENOTYPE (FTO RS1558902)", "target": "FAT LOSS RESPONSE TO HIGH-PROTEIN DIET"}, {"weight": 4.6, "description": "Individuals with the AA genotype experienced less fat loss when consuming a low-protein diet compared to those without this variant.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "AA GENOTYPE (FTO RS1558902)", "target": "FAT LOSS RESPONSE TO LOW-PROTEIN DIET"}, {"weight": 4.7, "description": "Participants with the AA genotype experienced greater fat loss when consuming a high-protein diet compared to those without this variant.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "AA GENOTYPE (FTO RS1558902)", "target": "FAT LOSS IN HIGH-PROTEIN DIET RESPONSE"}, {"weight": 4.6, "description": "Individuals with the AA genotype experienced less fat loss when consuming a low-protein diet compared to those carrying the variant.)", "source_id": "chunk-2926e6216bf945b12ea4daa6345cad1f", "order": 1, "source": "AA GENOTYPE (FTO RS1558902)", "target": "FAT LOSS IN LOW-PROTEIN DIET RESPONSE"}, {"weight": 3.0, "description": "Consumption of potatoes was higher in individuals with the LP genetic variant.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "DIETARY PATTERN", "target": "RS4988235 AND POTATO INTAKE"}, {"weight": 4.0, "description": "The intake of cereals was inversely related to the presence of rs4988235 genotype.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "DIETARY PATTERN", "target": "RS4988235 AND CEREAL INTAKE"}, {"weight": 7.0, "description": "The consumption of non-alcoholic beverages was higher in individuals carrying the LP variant.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "DIETARY PATTERN", "target": "RS4988235 AND NON-ALCOHOLIC BEVERAGE INTAKE"}, {"weight": 3.0, "description": "Improper dietary patterns can lead to nutrient deficiencies or excesses influencing health.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "DIETARY PATTERN", "target": "NUTRIENT INTAKE"}, {"weight": 5.0, "description": "Certain dietary patterns may interact with specific molecules like nitric oxide to impact health.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "DIETARY PATTERN", "target": "MOLECULAR INTERACTION"}, {"weight": 2.0, "description": "Eating specific types of poultry (excluding chicken and turkey) was identified as a risk factor for quinolone-resistant C. jejuni infection based on the case-comparison study.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "DIETARY PATTERN", "target": "RISK FACTOR"}, {"weight": 9.0, "description": "The influence of dietary patterns on environmental factors, such as exposure to certain chemicals, affects overall health outcomes including cardiovascular disease risk.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "DIETARY PATTERN", "target": "ENVIRONMENTAL FACTOR"}, {"weight": 3.0, "description": "The transcript of the NOS1 gene is influenced by dietary patterns, particularly folate intake.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "DIETARY PATTERN", "target": "TRANSCRIPT"}, {"weight": 6.0, "description": "Poor dietary patterns can lead to epigenetic modifications that alter nitric oxide synthesis.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "DIETARY PATTERN", "target": "EPIGENETIC MODIFICATION"}, {"weight": 1.0, "description": "The association between the FTO SNP and anthropometric measurements like BMI was different across educational levels, with stronger associations observed among those with primary or secondary studies compared to university-level subjects.)", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "order": 1, "source": "FTO RS9939609 POLYMORPHISM", "target": "EDUCATION"}, {"weight": 0.023, "description": "An interaction term was found between the FTO SNP and education in determining obesity risk, as per statistical analysis.)", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "order": 1, "source": "FTO SNP", "target": "EDUCATION"}, {"weight": 1.0, "description": "The FTO SNP plays a role in determining obesity risk, considering factors like education and physical activity.)", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "order": 1, "source": "FTO SNP", "target": "OBESITY RISK DETERMINATION"}, {"weight": 1.0, "description": "The lead SNP in CAD GWAS shows low LD (r^2 = 0.52) with top hit in CRP GWAS, indicating potential pleiotropic effects.)(entity", "source_id": "chunk-ad609cb10e900d42a6901eea1591d8ba", "order": 1, "source": "ASSOCIATION", "target": "CRP SNPS IN CAD GWAS"}, {"weight": 3.0, "description": "Genetic polymorphisms identified by GWAS were associated with habitual coffee consumption.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "ASSOCIATION", "target": "SNP AND COFFEE CONSUMPTION"}, {"weight": 9.0, "description": "Coffee consumers tended to have higher rates of alcohol consumption.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "ASSOCIATION", "target": "COFFEE CONSUMPTION AND ALCOHOL INTAKE"}, {"weight": 8.0, "description": "In the Czech Republic, an association between genotype and milk consumption was seen.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "ASSOCIATION", "target": "CZECH REPUBLIC"}, {"weight": 10.0, "description": "Educational attainment showed association with milk consumption across all countries.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "ASSOCIATION", "target": "EDUCATION ATTAINMENT"}, {"weight": 2.0, "description": "SNPs related to gene expressions in distant genes had a broad impact on multiple genes.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "ASSOCIATION", "target": "GWAS SNPS AND TRANS-ACTING GWAS SNPS"}, {"weight": 4.0, "description": "The lead trans-eQTL SNP was in LD with several GWAS SNPs related to autoimmune diseases.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "ASSOCIATION", "target": "RS1131017 SNP AND GWAS SNPS"}, {"weight": 95.0, "description": "were associated with DQA1*02:01\u2013DQB1*02:02 haplotype carrier status)", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "order": 1, "source": "ASSOCIATION", "target": "RS28383172 AG OR GG GENOTYPES"}, {"weight": 100.0, "description": "was in perfect concordance with HLA-DRB1*07:01 allele carrier status.)", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "order": 1, "source": "ASSOCIATION", "target": "RS28383172 G ALLELE"}, {"weight": 95.0, "description": "were associated with DQA1*02:01\u2013DQB1*02:02 haplotype carrier status.)", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "order": 1, "source": "ASSOCIATION", "target": "RS2395182 TT OR TG GENOTYPES&RS4713586 TT AND RS7775228 TC OR CC GENOTYPES"}, {"weight": 95.0, "description": "was associated with HLA-DRB1*07:01\u2013DQA1*02:01\u2013DQB1*02:02 haplotype carrier status.)", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "order": 1, "source": "ASSOCIATION", "target": "RS7775228 TC OR CC GENOTYPES"}, {"weight": 100.0, "description": "had perfect concordance with HLA-DRB1*07:01 allele carrier status.)", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "order": 1, "source": "ASSOCIATION", "target": "G ALLELE OF RS28383172"}, {"weight": 4.0, "description": "The interaction between the FTO SNP and education significantly impacted BMI in GP subjects.)", "source_id": "chunk-3135befd0c3140ad097d96ace6de86a1", "order": 1, "source": "EDUCATION LEVEL AND FTO RS9939609 POLYMORPHISM", "target": "INTERACTION EFFECT"}, {"weight": 0.957, "description": "Education level is associated with the relationship between genetic variations like the FTO SNP and obesity risk.)", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "order": 1, "source": "EDUCATION LEVEL", "target": "ASSOCIATION WITH OBESITY RISK"}, {"weight": 6.0, "description": "Higher education level correlated with increased adherence to Mediterranean diet.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "EDUCATION LEVEL", "target": "MD ADHERENCE"}, {"weight": 1.0, "description": "Selective deletion of IL-10 receptor restricted to T cells has no impact on colitis development.)", "source_id": "chunk-bcd248f7a9b2453bfc563b9439fbdd51", "order": 1, "source": "DISEASE", "target": "IL-10R\u0391TDEL"}, {"weight": 3.0, "description": "Transferring Th17 cells into Rag1\u2212/\u2212 mice results in colitis development. It also implies a crucial role for Th17 cells in disease pathogenesis, as they are necessary and sufficient for disease progression.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "DISEASE", "target": "TH17 CELL TRANSFER"}, {"weight": 2.0, "description": "There was a difference in disease development between Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 recipients.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "DISEASE", "target": "RAG1\u2212/\u2212 AND IL-10R\u0391MDELRAG1\u2212/\u2212 RECIPIENTS"}, {"weight": 1.0, "description": "No specific relationship with diseases mentioned in the given text.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "DISEASE", "target": "MINOR ALLELE"}, {"weight": 5.0, "description": "The patient\'s biochemical tests did not align with Menkes disease characteristics, indicating discrepancies between serum copper levels and catecholamine ratios compared to the typical profile of Menkes disease.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "DISEASE", "target": "BIOCHEMICAL FINDINGS"}, {"weight": 0.6, "description": "Variants of RGS14/locus, which modulates downstream effects of G\u03b1 subunits in kidneys, might influence disease risk and potentially have a role in conditions associated with longer lifespan and stress resistance.)", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "order": 1, "source": "DISEASE", "target": "RSK8"}, {"weight": 10.0, "description": "Social disinhibition was positively and negatively associated with specific SNPs, indicating disease risk or protection.)", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "order": 1, "source": "DISEASE", "target": "SOCIAL DISINHIBITION SCORE"}, {"weight": 11.0, "description": "The FANCA mutation, c.2574C>G/p.Ser858Arg identified by whole exome sequencing was previously associated with Fanconi anemia, but it did not lead to normal FANCA levels or monoubiquitination of FANCD2.)", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "order": 1, "source": "DISEASE", "target": "FANCA MUTATION"}, {"weight": 5.0, "description": "The interaction between different UBDs like NEMO-UBZ, Pol\u03b7-UBZ etc. and Ub may have implications for disease development due to altered protein functions.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "DISEASE", "target": "UB"}, {"weight": 7.0, "description": "Variants influencing folate metabolism pathways are associated with increased risk for certain diseases, including cardiovascular disease.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "DISEASE", "target": "GENOMIC VARIANT IN FOLATE METABOLISM PATHWAY"}, {"weight": 6.0, "description": "The PTPN22 risk variant permits proliferation in Tconv, but not in Treg.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "DISEASE", "target": "LYP VARIANTS IMPACT"}, {"weight": 6.0, "description": "The association between PTPN22 (LYP gene) variations and certain autoimmune disorders, such as ankylosing spondylitis or inflammatory bowel disease.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "DISEASE", "target": "AUTOIMMUNE DISORDER"}, {"weight": 1.0, "description": "No association was identified between rs3087243 and PBC susceptibility.)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "DISEASE", "target": "RS3087243"}, {"weight": 4.0, "description": "The lower affinity to HLA-A*01:01 suggests potential involvement in disease related to neoepitope interaction mechanisms.)", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "order": 1, "source": "DISEASE", "target": "ALK NEOEPITOPES"}, {"weight": 1.0, "description": "The statistical estimations regarding disease association do not change after certain adjustments, indicating robustness of findings.)", "source_id": "chunk-63d0b5be485a69f44e2799ccc9510487", "order": 1, "source": "DISEASE ASSOCIATION", "target": "STATISTICAL SIGNIFICANCE"}, {"weight": 1.0, "description": "19 out of 66 eligible publications reported statistically significant G \u00d7 E interactions for incident T2D risk.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "STATISTICAL SIGNIFICANCE", "target": "DIETARY EXPOSURE"}, {"weight": 7.0, "description": "Heme iron intake showed interactions with the SLC40A1 variant rs744653 for men, and with TMPRSS6 variant rs855791 across both sexes when not reaching a strong significance.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "HEME IRON INTAKE", "target": "INTERACTION WITH SLC40A1 RS744653 VARIANT IN MEN AND TMPRSS6 RS855791 IN TOTAL SAMPLE (BORDERLINE SIGNIFICANCE)"}, {"weight": 13.0, "description": "Higher heme iron intake was associated with increased T2D risk among women carrying specific HFE gene variants but not for men.)", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "order": 1, "source": "HFE VARIANTS RS1799945 AND RS1800562/", "target": "HEME IRON INTAKE/"}, {"weight": 15.0, "description": "The T2D gene risk score and two HECTD4 variants showed no interaction with alcohol consumption.)", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "order": 1, "source": "HECTD4 VARIANTS/", "target": "ALCOHOL CONSUMPTION/"}, {"weight": 2.0, "description": "The H63D variant was found to increase the risk of T2D specifically among women, based on data from EPIC-InterAct and DPP studies.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "HFE RS1799945 VARIANT", "target": "T2D RISK IN WOMEN"}, {"weight": 3.0, "description": "A similar trend to increased risk of T2D was observed for women with the H63D variant, though not statistically significant.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "EPIC-INTERACT STUDY", "target": "INTERACTION WITH HFE RS1799945 VARIANT IN WOMEN"}, {"weight": 4.0, "description": "While a similar trend was noted for women and the C282Y variant, the interaction did not reach significance.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "EPIC-INTERACT STUDY", "target": "INTERACTION WITH HFE RS1800562 VARIANT IN WOMEN (TREND BUT NOT SIGNIFICANT)"}, {"weight": 0.4, "description": "Estimation of heritability (H2) shows that among the 159 bacterial taxa present in over 10% of twin pairs, 19 taxa showed evidence for heritability with Pnominal<0.05.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "HERITABILITY", "target": "MOLECULAR INTERACTION"}, {"weight": 3.0, "description": "The heritabilities of lung function measures indicated that genetic variation contributed to phenotypic variance.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "HERITABILITY", "target": "STATISTICAL MEASURE"}, {"weight": 6.0, "description": "SNPs are markers for genetic predispositions influencing age at first tooth and number of teeth.)", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "GENETIC PREDISPOSITION", "target": "SNPS"}, {"weight": 1.0, "description": "The transfection process leads to an up to 10-fold range of expression of the rat IDI gene between various cell clones.)", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "order": 1, "source": "GENE EXPRESSION", "target": "TRANSFECTED CACO-2 CELLS"}, {"weight": 3.0, "description": "EBNA2\'s impact on gene expression correlates with specific MS risk genes and their role in the EBV latency III program.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "GENE EXPRESSION", "target": "MS RISK GENES INTERACTION"}, {"weight": 9.0, "description": "The expression levels of these genes are affected by EBV infection, which can be influenced by factors such as EBNA2 interaction and genotype.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "GENE EXPRESSION", "target": "EBV INFECTION"}, {"weight": 3.0, "description": "Certain genetic variants are associated with LTPA responsiveness.)", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "order": 1, "source": "LTPA", "target": "GENETIC VARIANTS"}, {"weight": 7.0, "description": "The risk allele within CARD9 associated with rs10781499 is connected to reduced AHR activation by microbiota-derived metabolites in patients with IBD.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "RISK ALLELE", "target": "AHR ACTIVATION"}, {"weight": 1.0, "description": "Interaction between specific genetic variants and weight loss interventions can influence diabetes risk.)", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "order": 1, "source": "WEIGHT LOSS INTERVENTIONS", "target": "GENETIC VARIANTS"}, {"weight": 4.0, "description": "Weight loss intervention affects diabetes risk according to DPP findings.)", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "order": 1, "source": "WEIGHT LOSS INTERVENTIONS", "target": "DIABETES PREVENTION PROGRAM (DPP)"}, {"weight": 5.0, "description": "DPS study indicates that weight loss interventions can reduce T2D incidence for specific genetic variants.)", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "order": 1, "source": "WEIGHT LOSS INTERVENTIONS", "target": "DOUBLE PODOCYTE STUDY (DPS)"}, {"weight": 8.0, "description": "Certain weight loss interventions impact diabetes risk among individuals with different genotypes, as observed in multiple studies.)", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "order": 1, "source": "WEIGHT LOSS INTERVENTIONS", "target": "RISK GENOTYPES"}, {"weight": 6.0, "description": "The DPP study indicates that certain genetic variants influence the effectiveness of weight loss interventions in diabetes risk reduction.)", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "order": 1, "source": "DIABETES PREVENTION PROGRAM (DPP)", "target": "GENETIC VARIANTS & WEIGHT LOSS INTERVENTIONS"}, {"weight": 7.0, "description": "The DPS analysis found that specific genetic variants and weight loss interventions have a combined effect on reducing T2D incidence.)", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "order": 1, "source": "DOUBLE PODOCYTE STUDY (DPS)", "target": "GENETIC VARIANTS & WEIGHT LOSS INTERVENTIONS"}, {"weight": 2.0, "description": "Potential G \u00d7 D interaction was found with genetic variants in the TCF7L2 gene.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "GENETIC VARIANT", "target": "T2D RISK"}, {"weight": 4.0, "description": "Genome-wide significant associations were identified with Single Nucleotide Polymorphisms (SNPs) at various loci.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "GENETIC VARIANT", "target": "SNPS"}, {"weight": 3.0, "description": "Interaction of fiber and whole grain with genetic variants was frequently investigated.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "G \u00d7 E INTERACTION", "target": "WHOLE GRAIN"}, {"weight": 4.0, "description": "Mainly European descendants were included in NHS and HPFS.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "EUROPEAN DESCENDANTS", "target": "SAMPLE RECRUITMENT"}, {"weight": 5.0, "description": "Most studies had participants aged over 40 years.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "PARTICIPANTS AGE", "target": "SAMPLE SIZE"}, {"weight": 1.0, "description": "Total sample size was comprised of cases and controls amounting to 590 participants.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "SAMPLE SIZE", "target": "SUBJECTS NUMBER"}, {"weight": 6.0, "description": "Follow-up time ranged from four to 26 years, with more than eight years in most studies.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "FOLLOW-UP TIME", "target": "YEARS RANGE"}, {"weight": 7.0, "description": "Studies were conducted on populations of mainly or only European ethnicity.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "ETHNICITY", "target": "POPULATIONS"}, {"weight": 5.0, "description": "Both ethnicity and rs182549 are related to the abundance of Bifidobacterium.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "ETHNICITY", "target": "RS182549& BIFIDOBACTERIUM"}, {"weight": 8.0, "description": "In DPP, participants received individual advice to reduce weight through lower fat and calorie intake along with higher physical activity.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "INTERVENTION IN DPP", "target": "LIFESTYLE-INTERVENTION"}, {"weight": 9.0, "description": "In DPS, the intervention guided participants to reduce weight by increasing physical activity and following a recommended diet.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "LIFESTYLE-INTERVENTION", "target": "INTERVENTION IN DPS"}, {"weight": 10.0, "description": "In DPP, participants were advised on Mediterranean dietary patterns with extra-virgin olive oil or mixed nuts as alternatives to low-fat diets.)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "DIETARY RECOMMENDATIONS", "target": "LIFESTYLE FACTORS - DPP"}, {"weight": 11.0, "description": "PREDIMED participants were randomized into two Mediterranean-diet groups which consumed extra virgin olive oil (1 L/week) or mixed nuts (30 g/day).)", "source_id": "chunk-0d8bc9a093c56fefc8b7af3fb9bc2862", "order": 1, "source": "LIFESTYLE FACTORS - PREDIMED", "target": "MEDITERRANEAN-DIET GROUPS"}, {"weight": 3.0, "description": "Higher intake of cereal fiber is associated with lower T2D incidence among individuals carrying the rs7903146 CC-genotype but not for those with the risk T-allele.)", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "order": 1, "source": "FIBER INTAKE/", "target": "RS7903146 VARIANT/"}, {"weight": 6.0, "description": "The variants rs10923931 in NOTCH2 and rs4457053 in ZBED3 have been associated with interactions involving fibre intake but were not consistently replicated across studies.)", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "order": 1, "source": "FIBRE INTAKE/", "target": "RS10923931 VARIANT/"}, {"weight": 7.0, "description": "The IRS1 rs2943641 genetic variant has been associated with insulin sensitivity and type 2 diabetes, though its findings have not been validated in other studies.)", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "order": 1, "source": "INSULIN SENSITIVITY/", "target": "RS2943641 VARIANT/"}, {"weight": 8.0, "description": "The GIPR rs10423928 variant has shown interaction with insulin secretion and type 2 diabetes but could not be replicated in all studies.)", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "order": 1, "source": "INSULIN SECRETION/", "target": "RS10423928 VARIANT/"}, {"weight": 9.0, "description": "The PPAR\u03b3 variants were linked to a trend for interaction with fat intake but could not be consistently replicated.)", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "order": 1, "source": "FAT INTAKE/", "target": "RS1801282 AND RS3856806 VARIANTS/"}, {"weight": 10.0, "description": "The ADH1C*2-allele was associated with lower diabetes risk among alcohol-consuming US women, contrasting its role in men.)", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "order": 1, "source": "ADH1C *1/*2 VARIANT/", "target": "ALCOHOL CONSUMPTION/"}, {"weight": 11.0, "description": "Higher heme iron intake was associated with increased T2D risk among women carrying specific HFE gene variants but not for men.)", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "order": 1, "source": "HEME IRON INTAKE/", "target": "T2D (TYPE 2 DIABETES)/"}, {"weight": 1.0, "description": "Environmental Factor/", "source_id": "chunk-ab449d4ec9f3baa290d63f09fdbd3fbc", "order": 1, "source": "GENETIC FACTOR INTERACTION/", "target": "PPARGC1A VARIANT/"}, {"weight": 5.0, "description": "In EPIC-InterAct study for men population group, an interaction was found to be significant between the H63D variant and T2D risk.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "INTERACTION WITH HFE RS1799945 VARIANT IN MEN", "target": "MEN POPULATION GROUP (EPIC-INTERACT)"}, {"weight": 6.0, "description": "A similar but not confirmed trend of increased T2D risk was found among men carrying the H63D variant through DPP study.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "INTERACTION WITH HFE RS1799945 VARIANT IN MEN", "target": "MEN POPULATION GROUP (DPP)"}, {"weight": 8.0, "description": "Dietary patterns that reflect either Western or Mediterranean diet were associated differently with T2D risk, influenced by specific genetic variants.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "DIETARY PATTERNS: WESTERN & MEDITERRANEAN", "target": "INTERACTION WITH GENETIC VARIANTS IN T2D RISK"}, {"weight": 9.0, "description": "High intake of red and processed meat increased T2D risk among men with high genetic risk scores.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "INTERACTION WITH T2D-GRS IN MEN", "target": "RED AND PROCESSED MEAT INTAKE (HPFS)"}, {"weight": 10.0, "description": "A marginally nonsignificant interaction was found between olive oil intake and the GIPR variant rs10423928.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "MARGINALLY NONSIGNIFICANT INTERACTION WITH GIPR VARIANT RS10423928 IN MEN", "target": "OLIVE OIL INTAKE (EPIC-INTERACT)"}, {"weight": 11.0, "description": "Carriers of the lactase persistence LCT rs4988235 T-allele had higher T2D risk than lactose nonpersistence CC-genotype carriers, with differences observed based on milk consumption.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "LACTASE PERSISTENCE LCT GENE VARIANTS (DANISH STUDY)", "target": "MILK CONSUMPTION IN T2D RISK"}, {"weight": 12.0, "description": "Findings from EPIC-InterAct and KARE suggest that coffee intake interacts differently with certain genetic variants for T2D risk, either reducing or increasing the risk.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "COFFEE INTAKE (EPIC-INTERACT & KARE)", "target": "INTERACTION WITH SOME GENETIC VARIANTS IN T2D RISK"}, {"weight": 13.0, "description": "A stronger T2D risk-lowering effect of coffee intake was observed among individuals carrying the T allele of the TCF7L2 variant.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "COFFEE INTAKE (EPIC-INTERACT)", "target": "INTERACTION WITH TCF7L2 VARIANT RS12255372 IN MEN"}, {"weight": 14.0, "description": "Coffee consumers with specific genetic variants (rs4402960 T-allele, rs7754840 G-allele, rs5215 CC-genotype) showed a reduced combined prediabetes and T2D risk compared to non-consumers.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "MILK CONSUMPTION (KARE)", "target": "REDUCTION IN COMBINED PREDIABETES AND T2D RISK FOR COFFEE CONSUMERS"}, {"weight": 15.0, "description": "A Western dietary pattern was associated with higher T2D risk only among individuals with a high genetic score for T2D, but not among those with a low score.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "INTERACTION WITH HIGH T2D-GRS IN MEN", "target": "WESTERN DIETARY PATTERN (HPFS)"}, {"weight": 16.0, "description": "A Mediterranean diet score showed no interaction with the risk of developing type 2 diabetes according to genetic scores.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "MEDITERRANEAN DIET (EPIC-INTERACT)", "target": "NO INTERACTION WITH T2D-GRS"}, {"weight": 17.0, "description": "The protective effect of physical activity against T2D was modified by certain genetic variants, with stronger effects observed in individuals not adhering to a high genetic risk score for T2D.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "MODIFYING EFFECT ON T2D RISK BY GENETIC VARIANTS", "target": "PA (PHYSICAL ACTIVITY)"}, {"weight": 18.0, "description": "In the Swedish Malm\u00f6 preventive program, lower protective effects of higher physical activity against T2D incidence were observed among individuals with a specific genetic variant for HNF1B.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "PA (PHYSICAL ACTIVITY)", "target": "WEAKENED PROTECTIVE EFFECT OF HIGHER PA ON T2D INCIDENCE (SWEDISH MALM\u00d6 PROGRAM)"}, {"weight": 19.0, "description": "An increase in leisure-time physical activity was associated with lower T2D incidence among individuals with certain genetic variants, such as the Glu12 allele of ADRA2B, GG genotype of ABCC8 rs3758947, and non-risk haplotype of four SLC2A2 variants.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "PA (PHYSICAL ACTIVITY)", "target": "INCREASED LTPA RESULTING IN LOWER T2D INCIDENCE"}, {"weight": 20.0, "description": "Decreased leisure-time physical activity was associated with higher T2D incidence among individuals carrying the PPAR\u03b3 rs1801282 Ala- and rs17036314 C-alleles, GG genotype of PPAR\u03b3, or non-risk haplotype of four SLC2A2 variants.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "PA (PHYSICAL ACTIVITY)", "target": "REDUCED LEISURE-TIME PHYSICAL ACTIVITY ASSOCIATED WITH HIGHER T2D INCIDENCE"}, {"weight": 21.0, "description": "In NHS, women carrying the IRS1 rs1522813 A-allele who were physically inactive had a higher risk of developing T2D compared to those with GG-genotype but active.)", "source_id": "chunk-f7bf38c7a21b86ad85fa360a21bfb447", "order": 1, "source": "GLYCOGEN STORAGE DISEASE PROTEIN 1 (IRS1)", "target": "INCREASED RISK WITH LOW PA AND T2D RISK A1 ALLELE IN WOMEN"}, {"weight": 1.0, "description": "SNPs miR-34b/c rs4938723 and TP53 Arg72Pro were considered in the IA risk assessment.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "GENETIC VARIANTS", "target": "RISK FACTOR FOR IA"}, {"weight": 5.0, "description": "Three SNPs in ERAP1 were found in all patients.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "GENETIC VARIANTS", "target": "ERAP1 SNPS"}, {"weight": 6.0, "description": "Two other SNPs in CDSN and three SNPs in CCHCR1 were carried by patient 3.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "GENETIC VARIANTS", "target": "CDSN SNPS"}, {"weight": 2.0, "description": "Participants with certain risk genotypes showed reduced T2D incidence after weight loss intervention.)", "source_id": "chunk-180ca3d6d27d84b30ebf5d20f135de18", "order": 1, "source": "DIABETES INCIDENCE", "target": "RISK GENOTYPES"}, {"weight": 5.0, "description": "The TCF7L2 gene includes the genomic variant rs7903146 that was analyzed in this study for its association with risk alleles.)<SEP>The rs7903146 variant found in TCF7L2 gene showed a significant association with type 2 diabetes (T2D) through this study\'s analysis.)<SEP>The rs7903146 variant found within the TCF7L2 gene is associated with type 2 diabetes (T2D), as indicated in this analysis.)<SEP>The rs7903146 variant in TCF7L2 gene showed consistent association with type 2 diabetes (T2D), as underlined by this study\'s findings.)<SEP>The rs7903146 variant was found associated with T2D through this study\'s analyses and meta-analysis, underlining its significance in diabetes risk assessment.)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4<SEP>chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "TCF7L2", "target": "RS7903146"}, {"weight": 1.0, "description": "The study\'s meta-analysis showed consistent results across sets 1 and 2 for rs7903146 variant in TCF7L2 gene, indicating a robust association with type 2 diabetes (T2D).)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "TCF7L2", "target": "T2D"}, {"weight": 4.0, "description": "The rs7903146 variant found in TCF7L2 gene showed association with type 2 diabetes (T2D), underlining genetic contributions to diabetes risk assessment.)<SEP>The rs7903146 variant found in TCF7L2 gene was associated with type 2 diabetes (T2D), as confirmed by this study\'s analyses and findings.)<SEP>The rs7903146 variant found in TCF7L2 gene was associated with type 2 diabetes (T2D), as shown through the study\'s meta-analysis and analyses.)<SEP>The rs7903146 variant within TCF7L2 gene was associated with type 2 diabetes (T2D), as confirmed by the study\'s analyses and meta-analysis results.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "RS7903146", "target": "T2D"}, {"weight": 2.0, "description": "The rs7903146 variant in ADAMTS9 gene showed association with T2D, indicating genetic interactions influencing diabetes risk.)<SEP>This variant within ADAMTS9 gene was found associated with type 2 diabetes (T2D), though its effect direction was at variance from European studies.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "RS7903146", "target": "ADAMTS9"}, {"weight": 1.0, "description": "The rs7903146 variant found within CDKAL1 gene showed association with type 2 diabetes (T2D), as indicated by this study\'s results.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "RS7903146", "target": "CDKAL1"}, {"weight": 1.0, "description": "The count of risk alleles associated with the TCF7L2 gene\'s genetic variants, including rs7903146, was part of the study\'s analysis.)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "RS7903146", "target": "NUMBER OF TCF7L2 RISK VARIANTS"}, {"weight": 1.0, "description": "SNP rs10010131 in the WFS1 gene was associated with type 2 diabetes (T2D), showing a consistent association despite variations compared to European studies.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "WFS1", "target": "RS10010131"}, {"weight": 3.0, "description": "SNP rs10946398 in CDKAL1 gene was found associated with T2D, replicating results from previous studies and analyses.)<SEP>SNP rs10946398 was found associated with T2D through this study, underlining genetic factors in diabetes risk.)<SEP>This SNP in the CDKAL1 gene was found associated with T2D, indicating an alignment of findings consistent with previous research.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "CDKAL1", "target": "RS10946398"}, {"weight": 1.0, "description": "The rs10946398 variant in CDKAL1 gene was associated with T2D, supporting the existing body of research on this connection.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "CDKAL1", "target": "T2D"}, {"weight": 1.0, "description": "SNP rs10946398 in CDKAL1 gene was found associated with T2D, aligning with previous research findings.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "RS10946398", "target": "T2D"}, {"weight": 1.0, "description": "SNP rs10946398 found in JAZF1 gene was associated with T2D, reinforcing findings from previous research.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "RS10946398", "target": "JAZF1"}, {"weight": 3.0, "description": "The rs864745 variant found in JAZF1 gene was associated with T2D, as underlined by this study\'s results and analyses.)<SEP>The rs864745 variant found within JAZF1 gene showed association with type 2 diabetes (T2D), as confirmed by this study\'s findings.)<SEP>The rs864745 variant in JAZF1 gene showed association with T2D, replicating results from prior studies and analyses.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "JAZF1", "target": "RS864745"}, {"weight": 2.0, "description": "The rs864745 variant in JAZF1 gene was associated with type 2 diabetes (T2D), as indicated by the study\'s findings and analyses.)<SEP>The rs864745 variant of JAZF1 gene showed association with type 2 diabetes (T2D), replicating results from earlier studies and analyses.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "JAZF1", "target": "T2D"}, {"weight": 8.0, "description": "A variant in the ADAMTS9 gene was associated with T2D, but its effect direction was opposite to previous European findings.)<SEP>SNP rs4607103 found in ADAMTS9 gene was associated with type 2 diabetes (T2D), but showed an opposing effect direction compared to European studies.)<SEP>SNP rs4607103 in ADAMTS9 gene was associated with T2D, but its effect direction was opposite to that observed in European studies.)<SEP>SNP rs4607103 in ADAMTS9 gene was associated with T2D, though its effect direction was opposite that reported in European studies.)<SEP>SNP rs4607103 was found associated with T2D through this study\'s findings, highlighting genetic linkages.)<SEP>SNP rs4607103 was found associated with T2D through this study, indicating genetic overlap in diabetes risk.)<SEP>The rs4607103 variant found in ADAMTS9 gene showed association with T2D, but its effect direction differed from European studies.)<SEP>The rs4607103 variant of ADAMTS9 gene was associated with T2D, though its direction of effect differed from European reports.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "ADAMTS9", "target": "RS4607103"}, {"weight": 2.0, "description": "A single nucleotide polymorphism found in ADAMTS9 gene was associated with type 2 diabetes (T2D), with its effect direction differing from European studies.)<SEP>Type 2 diabetes (T2D) was associated with a variant in the ADAMTS9 gene, highlighting genetic factors contributing to T2D risk.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "ADAMTS9", "target": "T2D"}, {"weight": 1.0, "description": "Type 2 diabetes and end-stage renal disease (T2D-ESRD) case subjects had an older mean age at enrollment compared to controls, highlighting a possible temporal or health condition correlation.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "T2D-ESRD", "target": "AGE"}, {"weight": 1.0, "description": "Set 1 had a higher percentage of women compared to controls, indicating potential sex-based disparities in T2D and ESRD associations.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "T2D-ESRD", "target": "SEX RATIO"}, {"weight": 1.0, "description": "The enrollment age difference between T2D-ESRD case subjects and controls was observed, potentially influencing study outcomes or the progression of conditions.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "T2D-ESRD", "target": "AGE AT ENROLLMENT"}, {"weight": 1.0, "description": "Diagnosis age disparity between T2D and ESRD subjects was observed, possibly influencing outcomes or disease management.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "T2D-ESRD", "target": "AGE AT DIAGNOSIS"}, {"weight": 1.0, "description": "This study\'s results highlighted contributions of various risk factors to T2D and ESRD, underscoring interplay between conditions.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "T2D-ESRD", "target": "RISK FACTOR CONTRIBUTION"}, {"weight": 1.0, "description": "This study\'s findings underscored the importance of clinical parameters in assessing T2D and ESRD, emphasizing their interrelation.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "T2D-ESRD", "target": "CLINICAL PARAMETERS"}, {"weight": 1.0, "description": "The combined genetic variants rs7903146 in TCF7L2, ADAMTS9, CDKAL1, JAZF1, rs864745 and rs10946398 showed association with type 2 diabetes (T2D), indicating complex genetic interplay.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "T2D-ESRD", "target": "RS7903146+ADAMTS9+CDKAL1+JAZF1+RS864745+RS10946398+RS4607103+RS10946398+RS864745"}, {"weight": 1.0, "description": "The combined variants rs7903146 in TCF7L2, ADAMTS9, CDKAL1 and JAZF1 genes showed association with type 2 diabetes (T2D), highlighting genetic contributions to multiple risk factors.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "T2D", "target": "RS7903146+ADAMTS9+CDKAL1+JAZF1"}, {"weight": 1.0, "description": "The combined variants rs864745 in JAZF1 gene showed association with T2D through this study, emphasizing genetic factors in diabetes risk assessment.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "T2D", "target": "RS864745+JAZF1"}, {"weight": 1.0, "description": "The combined genetic variants rs7903146 in TCF7L2, ADAMTS9, CDKAL1, JAZF1 and rs864745 and rs10946398 in other genes were associated with type 2 diabetes (T2D), emphasizing the role of genetic polymorphisms.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "T2D", "target": "RS7903146+ADAMTS9+CDKAL1+JAZF1+RS864745+RS10946398"}, {"weight": 1.0, "description": "SNP rs4420638, located near APOC1 gene, shows a strong association with type 2 diabetes.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "T2D", "target": "RS4420638"}, {"weight": 5.0, "description": "Type 2 diabetes shows associations that may involve CRP and pleiotropic SNPs, with some estimates decreasing after adjustment.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "T2D", "target": "CRP"}, {"weight": 2.0, "description": "SNP rs12581299 in FABP2 gene is associated with increased risk of T2D.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "T2D", "target": "RS12581299"}, {"weight": 6.0, "description": "SNP rs638982 near SREBF1 gene shows association with T2D susceptibility.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "T2D", "target": "RS638982"}, {"weight": 1.0, "description": "The genomic variant rs10399931 did not show significant associations with T2D after adjustment for covariates.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "T2D", "target": "RS10399931"}, {"weight": 1.0, "description": "Similar to rs10399931, rs4950928 was also not associated with Type 2 Diabetes.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "T2D", "target": "RS4950928"}, {"weight": 1.0, "description": "The association between rs10546972 and T2D was not significant after adjustment for covariates.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "T2D", "target": "RS10546972"}, {"weight": 1.0, "description": "Haplotype block 1 of CHI3LI did not show any association with Type 2 Diabetes.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "T2D", "target": "CHI3LI HAPLOTYPE BLOCK 1"}, {"weight": 1.0, "description": "Haplotype block 2 of the CHI3LI gene also showed no significant associations with T2D.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "T2D", "target": "CHI3LI HAPLOTYPE BLOCK 2"}, {"weight": 8.0, "description": "Diabetic nephropathy in type 2 diabetes impacts eGFR levels through a combination of genetic and metabolic factors.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "T2D", "target": "EGFR"}, {"weight": 1.0, "description": "No evidence for interaction between age and the association between TCF7L2 gene and the number of risk alleles was found.)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "AGE", "target": "INTERACTION"}, {"weight": 9.0, "description": "Younger age correlated with higher adherence to the Mediterranean diet.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "AGE", "target": "MD ADHERENCE"}, {"weight": 1.001, "description": "An increase in age is correlated with a higher social disinhibition score.)<SEP>Increasing age is positively associated with increased social disinhibition score.)", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "order": 1, "source": "AGE", "target": "SOCIAL DISINHIBITION"}, {"weight": 6.43, "description": "The effect of rs1893217 on response rate was stronger in the younger age group.)", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "order": 1, "source": "AGE", "target": "RESPONSE RATE"}, {"weight": 4.0, "description": "Age and sex are related entities considered in multivariate logistic regression analysis for HIV disease progression.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "AGE", "target": "SEX"}, {"weight": 6.0, "description": "Age is correlated with tobacco usage, both being factors analyzed for their impact on HIV disease progression.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "AGE", "target": "TOBACCO"}, {"weight": 8.0, "description": "Age and alcohol consumption are related entities that might influence the risk of HIV disease progression.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "AGE", "target": "ALCOHOL"}, {"weight": 10.0, "description": "Age is correlated with baseline CD4 counts, which play a role in assessing HIV disease progression.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "AGE", "target": "BASELINE CD4 COUNTS"}, {"weight": 12.0, "description": "The entity \'age\' correlates with the genetic polymorphisms of TLR7, which are factors studied for HIV disease progression.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "AGE", "target": "TLR7 (RS179008 AND RS179009)"}, {"weight": 1.0, "description": "There is no evidence of interaction between sex and the relationship between the TCF7L2 gene\'s genetic variants and count of risk alleles.)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "SEX", "target": "INTERACTION"}, {"weight": 10.0, "description": "Women were observed to adhere more to the Mediterranean diet compared to men.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "SEX", "target": "MD ADHERENCE"}, {"weight": 1.0, "description": "The sex ratio differed, with a higher proportion of female cases relative to male participants.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "SEX", "target": "RATIO BETWEEN GROUPS"}, {"weight": 1.0, "description": "None of the interactions with age or sex were statistically significant for rs1893217, indicating that there is no evidence suggesting gender-specific associations.)", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "order": 1, "source": "SEX", "target": "RESPONSE RATE"}, {"weight": 14.0, "description": "Sex is related to tobacco usage in terms of their combined influence on HIV disease progression.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "SEX", "target": "TOBACCO"}, {"weight": 16.0, "description": "Sex and alcohol consumption have a connection that might affect the risk associated with HIV disease progression.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "SEX", "target": "ALCOHOL"}, {"weight": 1.0, "description": "The degree of African ancestry did not show an effect on the relationship between the TCF7L2 gene\'s genetic variants and count of risk alleles.)", "source_id": "chunk-6a7c69534e11aa2fc4bd948484aa81c4", "order": 1, "source": "DEGREE OF AFRICAN ANCESTRY", "target": "INTERACTION"}, {"weight": 1.0, "description": "SNPs rs4607103 and rs10946398 were found associated with T2D through this study, indicating genetic variability in diabetes risk.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "ADAMTS9+CDKAL1+JAZF1", "target": "RS4607103+RS10946398"}, {"weight": 1.0, "description": "SNPs rs864745, JAZF1, rs4607103 and rs10946398 were found associated with T2D and ESRD through this study, highlighting the importance of genetic factors in disease assessments.)", "source_id": "chunk-f9a60d3cf951d77afa115859755dac43", "order": 1, "source": "ADAMTS9+CDKAL1+T2D-ESRD CLINICAL PARAMETERS", "target": "RS864745+JAZF1+RS4607103+RS10946398"}, {"weight": 5.0, "description": "An inverse association was more pronounced among individuals with high genetic risk scores for black coffee consumption, but the interaction was not statistically significant.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "INTERACTION", "target": "GENETIC RISK SCORES AND COFFEE CONSUMPTION"}, {"weight": 1.0, "description": "Transitioning from NGT to IFG/IGT or T2D is associated with disease progression and increased health risks.)", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "order": 1, "source": "GLYCEMIC STATUS TRANSITION", "target": "DISEASE PROGRESSION"}, {"weight": 1.0, "description": "An increase in fasting plasma glucose levels shows a positive association with BMI changes, possibly through metabolic pathways affecting fat accumulation.)", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "order": 1, "source": "CHANGE IN FPG LEVELS", "target": "BMIC"}, {"weight": 8.0, "description": "Lifestyle choices like physical activity can modulate the molecular interaction between dietary components and gene expression related to cardiovascular health.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "LIFESTYLE FACTORS", "target": "MOLECULAR INTERACTION"}, {"weight": 8.0, "description": "Increased insulin resistance as indicated by HOMA-IR score was present among women with PCOS compared to controls.)", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "order": 1, "source": "INSULIN RESISTANCE", "target": "PCOS"}, {"weight": 4.0, "description": "|>HFD/", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "INSULIN RESISTANCE", "target": "PATS/"}, {"weight": 6.0, "description": "After adjusting for CRP, the effect of SNP associated with FTO on cardiometabolic phenotypes mainly goes through BMI, suggesting mediated pleiotropy.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "FTO", "target": "MEDIATED PLEIOTROPY"}, {"weight": 2.0, "description": "A GRS \u2265 6 was associated with lower mean scores on TMTB compared to a GRS < 6", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "order": 1, "source": "GRS", "target": "TMTB"}, {"weight": -0.1, "description": "Fasting plasma glucose and two-hour post-meal glucose levels are negatively correlated with BMI at baseline but positively related to BMI change, indicating complex metabolic interactions.)", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "order": 1, "source": "BMI CHANGE", "target": "FPG AND 2HPG LEVELS"}, {"weight": 3.0, "description": "Participants with NGT display lower changes in BMI compared to IFG/IGT or T2D subjects, highlighting protective effects of normal glucose tolerance on body weight maintenance.)", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "order": 1, "source": "BASELINE GLYCEMIC STATUS", "target": "CHANGE IN BMI"}, {"weight": 2.5, "description": "Transitioning from NGT to IFG/IGT increases the risk of developing obesity and metabolic disorders due to higher baseline BMI and BMI change.)", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "order": 1, "source": "IFG/IGT TRANSITION", "target": "INCREASED RISK"}, {"weight": 4.0, "description": "Maintaining normal glucose tolerance over time has a protective effect on body weight management compared to transitioning stages of impaired glycemic status.)", "source_id": "chunk-bb507a90a1f5d5ddd01a93200ff2d45e", "order": 1, "source": "PREVENTIVE EFFECT", "target": "STABLE NGT"}, {"weight": 2.0, "description": "The rs1011527 SNP in the LEPR gene interacts significantly with FPG on baseline BMI.)", "source_id": "chunk-1fde08e52a188ac6a549e7c11d59a092", "order": 1, "source": "FPG (FASTING PLASMA GLUCOSE)", "target": "RS1011527 SNP IN LEPR"}, {"weight": 1.0, "description": "Early MTX-related neurotoxicity may have affected performance on TMTA as per the study)", "source_id": "chunk-ea6355aaf22d7cdd3331602638122ed7", "order": 1, "source": "TMTA", "target": "MTX"}, {"weight": 4.0, "description": "The active interventions had a significant impact on lowering fasting proinsulin levels.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "TREATMENT EFFECT", "target": "ACTIVE INTERVENTION GROUPS (LIFESTYLE, METFORMIN, TROGLITAZONE)"}, {"weight": 4.0, "description": "Increased fasting insulin levels were found among women with PCOS, indicative of insulin resistance.)", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "order": 1, "source": "FASTING INSULIN", "target": "PCOS"}, {"weight": 2.0, "description": "The DPP cohort was exposed to certain dietary patterns.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "DIETARY PATTERN EXPOSURE", "target": "DPP COHORT INDIVIDUALS"}, {"weight": 3.0, "description": "Baseline fasting insulin was used as an adjustment factor for statistical analysis.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "ADJUSTMENT FACTOR", "target": "BASELINE FASTING INSULIN"}, {"weight": 5.0, "description": "The metformin and troglitazone intervention groups showed similar but distinct drug responses in terms of decreasing proinsulin levels.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "DRUG RESPONSE COMPARISON", "target": "METFORMIN AND TROGLITAZONE INTERVENTION GROUPS"}, {"weight": 6.0, "description": "After DPP interventions, the association between SLC30A8 risk alleles and fasting proinsulin levels was no longer statistically significant.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "ASSOCIATION AFTER TREATMENT", "target": "GENOTYPE AT SLC30A8 LOCUS"}, {"weight": 2.0, "description": "Impact of lifestyle changes on fasting proinsulin levels following DPP interventions.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "IMPACT", "target": "LIFESTYLE INTERVENTION EFFECTS"}, {"weight": 3.0, "description": "Comparative analysis between the drug effects of metformin and troglitazone on proinsulin levels.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "COMPARISON", "target": "METFORMIN VS. TROGLITAZONE"}, {"weight": 4.0, "description": "The influence of specific genotypes on the effectiveness of diabetes interventions over time.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "GENOTYPE AT SLC30A8 LOCUS INTERACTION", "target": "MODULATION"}, {"weight": 5.0, "description": "How different treatment responses are influenced by varying age levels within study participants.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "AGE-DEPENDENCY", "target": "TREATMENT RESPONSES WITH AGE ADJUSTMENT"}, {"weight": 6.0, "description": "The influence of sex on the body\'s response to proinsulin levels post-intervention.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "GENDER-EFFECT", "target": "SEX DIFFERENCES IN PROINSULIN RESPONSE"}, {"weight": 7.0, "description": "How ethnicity impacts the effectiveness of diabetes prevention interventions and outcomes.)", "source_id": "chunk-8f23bd741f82dc8ca1e2600d790b690f", "order": 1, "source": "CULTURAL-EFFECT", "target": "ETHNICITY-BASED TREATMENT OUTCOMES"}, {"weight": 1.0, "description": "The high carbohydrate diet increases PNPLA3 mRNA and protein levels in mouse liver, which is attenuated by lipid emulsions.)", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "order": 1, "source": "HC DIET", "target": "PNPLA3"}, {"weight": 3.0, "description": "High carbohydrate diet increases expression of SREBP1c transcription compared to NC and controlled by LE.)", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "order": 1, "source": "HC DIET", "target": "SREBP1C MRNA"}, {"weight": 2.0, "description": "Mice fed HF diet have elevated hepatic TG content but suppressed PNPLA3 gene expression compared to mice on LF diet.)", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "order": 1, "source": "PNPLA3", "target": "HF DIET"}, {"weight": 3.0, "description": "Voluntary exercise reduces PNPLA3 mRNA levels in mice fed HF diet, suggesting a relationship between physical activity and gene expression in lipogenesis.)", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "order": 1, "source": "PNPLA3", "target": "EXERCISE"}, {"weight": 4.0, "description": "Glucose regulates PNPLA3 gene expression through depletion-refeeding conditions in HepG2 cells, indicating metabolic pathways influencing the expression of this gene.)", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "order": 1, "source": "PNPLA3", "target": "GLUCOSE"}, {"weight": 5.0, "description": "The treatment of HepG2 cells with exogenous fatty acids like OA showed no effect on lipid content when oil red O staining was used, but a significant increase in FAS mRNA expression was observed instead, highlighting the dual regulation effects of fatty acids on both PNPLA3 and other genes involved in lipid metabolism.)", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "order": 1, "source": "PNPLA3", "target": "EXOGENOUS FATTY ACID"}, {"weight": 4.0, "description": "Mice fed HF diet have similar FAS gene expression compared to LF, but other genes like SREBP1c and ACC-1 are modulated by diet.)", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "order": 1, "source": "HF DIET", "target": "FAS (FATTY ACID SYNTHASE)"}, {"weight": 5.0, "description": "Physical activity reduces the transcription of SREBP1c on HF diet in mice, indicating a metabolic pathway influenced by lifestyle activities.)", "source_id": "chunk-c27a7b63e450810a5d9f9e0f70b73a7a", "order": 1, "source": "EXERCISE", "target": "SREBP1C MRNA"}, {"weight": 1.0, "description": "The hypothesis suggested that exercise may stimulate glucagon secretion, thereby decreasing the expression of PNPLA3 gene but our data showed no effect of 100 nM glucagon on PNPLA3 mRNA expression under low glucose conditions.)", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "order": 1, "source": "EXERCISE", "target": "GLUCAGON"}, {"weight": 2.0, "description": "Studies indicated that insulin increased promoter activity of PNPLA3 gene in Chinese hamster ovary cells but incubation with 10 nM insulin for 24 h did not increase PNPLA3 mRNA compared to control cells.)", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "order": 1, "source": "INSULIN", "target": "PNPLA3 GENE"}, {"weight": 3.0, "description": "|>SENP1-aP2KO", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "INSULIN", "target": "PATS/"}, {"weight": 3.0, "description": "Exercise was hypothesized to stimulate glucagon secretion which could decrease PNPLA3 gene expression but our data showed no effect of 100 nM glucagon on PNPLA3 mRNA under low glucose conditions.)", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "order": 1, "source": "GLUCAGON", "target": "FATTY ACIDS"}, {"weight": 4.0, "description": "Several unsaturated fatty acids investigated showed effects on PNPLA3 gene expression when added directly to HepG2 cells, with some significantly suppressing its expression.)", "source_id": "chunk-b89ac7c0c243e0d9dcdcf4ebf07b538f", "order": 1, "source": "FATTY ACIDS", "target": "PNPLA3 GENE"}, {"weight": 4.0, "description": "The FoxO1 gene is activated following stress exposure and cortisol treatment, indicating a role in mediating negative effects on neurogenesis.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "FOXO1", "target": "STRESS"}, {"weight": 3.0, "description": "The cell-permeable inhibitor AS1842856 can inhibit the upregulation of FoxO1 by cortisol, suggesting it mediates the effect of stress on this gene.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "FOXO1", "target": "AS1842856"}, {"weight": 4.0, "description": "FoxO1 signaling mediates the effect of stress on neuronal proliferation.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "FOXO1", "target": "NEURONAL PROLIFERATION"}, {"weight": 2.0, "description": "The gene ISOC1 interacts with LDLRAP1 through physical interaction or co-expression, contributing possibly to lipid metabolism or other cellular functions.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "ISOC1", "target": "LDLRAP1"}, {"weight": 3.0, "description": "LRRN3 may show interactions with MYO1D as part of common pathways or biological processes related to cell regulation and dynamics.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "LRRN3", "target": "MYO1D"}, {"weight": 4.0, "description": "MYO1D interacts with PIK3C2B, possibly influencing signaling pathways or cellular responses based on their combined expression or functions.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "MYO1D", "target": "PIK3C2B"}, {"weight": 5.0, "description": "The gene PPAP2A might interact with SAMD12 in specific cellular processes or pathways related to biochemical reactions or regulatory mechanisms.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "PPAP2A", "target": "SAMD12"}, {"weight": 6.0, "description": "SERINC5 interacts with TGF-\u03b21, possibly contributing to cytokine signaling or other biological responses associated with this pathway.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "SERINC5", "target": "TGF-\u03921"}, {"weight": 9.0, "description": "The glycosylation process facilitated by PGAP1 might interact with TGF-\u03b21 to regulate extracellular matrix dynamics or other biological responses.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "TGF-\u03921", "target": "PGAP1"}, {"weight": 8.0, "description": "Transcriptional factors like LEF1 could interact with LIN54, influencing gene expression patterns in various cellular contexts.)", "source_id": "chunk-01cd90d27130aab1a096ad616e05f182", "order": 1, "source": "LEF1", "target": "LIN54"}, {"weight": 2.0, "description": "The relationship between the presence of a specific SNP and various baseline characteristics (gender, race, age, BMI).)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "SNP", "target": "BASELINE CHARACTERISTICS"}, {"weight": 1.0, "description": "Evaluation of the over-representation of genes in pathways associated with variants related to \'Age at first tooth\'.)", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "order": 1, "source": "SNP", "target": "PATHWAY ANALYSIS"}, {"weight": 0.0, "description": "Variants like C3 rs2230199 were investigated for potential interactions with lifestyle on AMD risk, but they showed no strong main effects.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "SNP", "target": "C3 RS2230199"}, {"weight": 0.0, "description": "The CFI rs10033900 SNP was considered in the study of joint effects with lifestyle factors on AMD, but its combined effect with HLS did not yield reliable estimates.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "SNP", "target": "CFI RS10033900"}, {"weight": 1.0, "description": "A specific SNP has an inverse relationship with the development or severity of type 2 diabetes.)", "source_id": "chunk-b803830d07bff8ae14550f6458048707", "order": 1, "source": "SNP", "target": "TYPE 2 DIABETES"}, {"weight": 3.0, "description": "The mRNA levels of FoxO1 are upregulated following stress and cortisol exposure.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "FOXO1 MRNA", "target": "STRESS-INDUCED UPREGULATION"}, {"weight": 2.0, "description": "AS1842856 is able to counteract cortisol-induced reduction in neuronal proliferation, indicating its role in modulating glucocorticoid effects.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "AS1842856", "target": "CORTISOL-INDUCED CHANGES"}, {"weight": 1.0, "description": "The table may provide details about dietary patterns associated with emotional abuse, although not explicitly mentioned.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "SUPPLEMENTARY TABLE 9", "target": "EMOTIONAL ABUSE (SEE SUPPLEMENTARY TABLE 9)"}, {"weight": 1.0, "description": "The stress manipulation had a significant effect on reducing response bias toward the more frequently rewarded stimulus.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "STRESS", "target": "CONDITION"}, {"weight": 1.41, "description": "An elevated discriminability was found in Conditions 3 and 2, indicating better performance than in Condition 1.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "STRESS", "target": "CONDITION 3 AND CONDITION 2"}, {"weight": 2.0, "description": "SGK-1 is involved in mediating glucocorticoid effects, specifically cortisol\'s negative impact on neurogenesis.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "CORTISOL-INDUCED CHANGES IN NEUROGENESIS", "target": "SGK-1"}, {"weight": 4.0, "description": "Cortisol was found to reduce neuronal proliferation, as indicated by decreased BrdU+ cells.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "CORTISOL-INDUCED REDUCTION IN NEURONAL PROLIFERATION (BRDU\u2009+\u2009CELLS)", "target": "OUTCOME/EFFECT"}, {"weight": 3.0, "description": "A 24-hour exposure to cortisol led to an increase in the expression of SGK1 mRNA.)", "source_id": "chunk-704d1bc6fcda99ac888bd36c954fdb98", "order": 1, "source": "24\u2009H CORTISOL TREATMENT", "target": "EFFECT ON SGK1 MRNA"}, {"weight": 2.0, "description": "Additional copies were associated with increased pre-surgical weight in this study.)", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "order": 1, "source": "ADDITIONAL COPIES OF THE MINOR (G) ALLELE", "target": "PRE-SURGICAL WEIGHT"}, {"weight": 3.0, "description": "rs4771122 was nominally significantly associated with nadir percent weight loss.)", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "order": 1, "source": "NADIR PERCENT WEIGHT LOSS", "target": "SNP RS4771122"}, {"weight": 4.0, "description": "SNP rs4771122 showed association with 2-year percent weight loss.)", "source_id": "chunk-b3d775642c43fa3524a531a3f6ec1e41", "order": 1, "source": "SNP RS4771122", "target": "2 YEAR PERCENT WEIGHT LOSS"}, {"weight": 1.0, "description": "A gene-diet interaction was examined to see if there were significant interactions between genetics and diet, however no such interactions were found.)", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "order": 1, "source": "MOLECULAR INTERACTION", "target": "GENE-DIET INTERACTION"}, {"weight": 1.0, "description": "AcAc levels are inversely associated with insulin sensitivity.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "MOLECULAR INTERACTION", "target": "ACAC& INSULIN SENSITIVITY"}, {"weight": 2.0, "description": "BHB levels remain inversely associated with insulin sensitivity.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "MOLECULAR INTERACTION", "target": "BHB& INSULIN SENSITIVITY"}, {"weight": 3.0, "description": "High AcAc levels could weaken or eliminate the association of insulin secretion and development of hyperglycemia and diabetes.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "MOLECULAR INTERACTION", "target": "ACAC& INSULIN SECRETION"}, {"weight": 4.0, "description": "Similar results are obtained when comparing highest quartile of glucose AUC to three lowest quartiles, indicating an inverse relationship with BHB levels.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "MOLECULAR INTERACTION", "target": "BHB& INSULIN SECRETION"}, {"weight": 0.8, "description": "The estimation of heritability and SNP-based heritability showed significant correlation across the tested taxa, with correlation coefficient R=0.244.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "MOLECULAR INTERACTION", "target": "GENUS RUMINICLOSTRIDIUM 9&FAMILY PEPTOSTREPTOCOCCAEAE"}, {"weight": 1.0, "description": "The LP genetic variant was positively related to milk consumption, affecting the F-statistic for predicting milk intake.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "MOLECULAR INTERACTION", "target": "RS4988235 AND MILK INTAKE"}, {"weight": 6.0, "description": "B12 and Folate are nutrients that might have a molecular interaction affecting maternal health and offspring development.)", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "order": 1, "source": "MOLECULAR INTERACTION", "target": "B12&FOLATE"}, {"weight": 9.0, "description": "A genomic variant may influence B12 metabolism, affecting its physiological process and levels.)", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "order": 1, "source": "MOLECULAR INTERACTION", "target": "GENOMIC VARIANT&B12 METABOLISM"}, {"weight": 8.0, "description": "The presence of a novel BsmAI restriction site was confirmed through molecular analysis as a result of the P1001L mutation.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "MOLECULAR INTERACTION", "target": "BSMAI RESTRICTION SITE"}, {"weight": 10.0, "description": "The analysis of DNA from different tissues showed distinct wild type:mutant allele ratios that help to assess genetic variations and mosaicism in the patient.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "MOLECULAR INTERACTION", "target": "WILD TYPE: MUTANT ALLELE RATIO"}, {"weight": 11.0, "description": "The results of Q-RT-PCR experiments demonstrated consistent wild type:mutant allele ratios across different tissues, contributing to a clearer understanding of genetic mosaicism.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "MOLECULAR INTERACTION", "target": "QUANTITATIVE REAL-TIME PCR (Q-RT-PCR)"}, {"weight": 1.0, "description": "An additive genetic architecture was consistent with the observation that FEV1/FVC decreased with increasing number of \'low FEV1/FVC alleles\'.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "MOLECULAR INTERACTION", "target": "ADDITIVE GENETIC ARCHITECTURE"}, {"weight": 3.0, "description": "The W631* variant results in a truncated protein that affects its ability to function correctly.)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "MOLECULAR INTERACTION", "target": "W631*"}, {"weight": 5.0, "description": "The X-ray structure confirms the conformation of peptide bound to MHC groove and supports the fixed-backbone modeling approach.)", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "order": 1, "source": "MOLECULAR INTERACTION", "target": "X-RAY STRUCTURE"}, {"weight": 3.0, "description": "Clinical findings were similar for probands who carry GCK variants, compared to those described previously.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "CLINICAL INFORMATION", "target": "PROBANDS"}, {"weight": 8.0, "description": "One proband had mild fasting hyperglycemia and stable HbA1c level despite being identified with this genetic change, indicating that the clinical manifestation might not always match genetic data.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "PROBANDS", "target": "CLINICAL PHENOTYPE"}, {"weight": 5.0, "description": "Sequence analysis identified 134 GCK variants in probands from the year 2008 to 2018.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "GCK VARIANTS", "target": "SEQUENCE ANALYSIS"}, {"weight": 5.0, "description": "Comparative analysis of peptide sequences helps identify those with higher affinity or potential for stronger T-cell responses.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "SEQUENCE ANALYSIS", "target": "PEPTIDE SEQUENCE SIMILARITY"}, {"weight": 6.0, "description": "Establishing co-segregation confirmed that the diabetes phenotype is indeed associated with specific genetic variants.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "CO-SEGREGATION WITH THE DIABETES PHENOTYPE", "target": "PARENTAL OR EXTENDED FAMILY DNA"}, {"weight": 7.0, "description": "Seven VUS were predicted to be damaging by in-silico tools, but they did not satisfy pathogenic or likely pathogenic classification requirements due to lack of segregation and/or phenotypic data.)", "source_id": "chunk-658df2fc2f9361a65ff1d22464744763", "order": 1, "source": "PREDICTIVE TOOLS", "target": "VUS (VARIANTS OF UNCERTAIN SIGNIFICANCE)"}, {"weight": 1.0, "description": "In medium and high MD adherence, TAG concentrations were significantly lower.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "TAG CONCENTRATIONS"}, {"weight": 2.0, "description": "HDL-C (high-density lipoprotein cholesterol) concentrations were higher with increased Mediterranean diet adherence.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "HDL-C CONCENTRATIONS"}, {"weight": 3.0, "description": "The apoB:apoA-1 ratio was significantly lower in medium and high MD adherence categories.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "APOB:APOA-1 RATIO"}, {"weight": 4.0, "description": "HbA1c concentration was lower as the Mediterranean diet adherence increased.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "HBA1C CONCENTRATION"}, {"weight": 5.0, "description": "Physical activity was associated with higher adherence to the Mediterranean diet.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "PHYSICAL ACTIVITY"}, {"weight": 8.0, "description": "Lower waist circumference was associated with increased adherence to Mediterranean diet.)", "source_id": "chunk-c6fdc5ed25c4e17b8c421a952bb05450", "order": 1, "source": "MD ADHERENCE", "target": "WC (WAIST CIRCUMFERENCE)"}, {"weight": 5.0, "description": "Genetic predisposition may influence how individuals engage in lifestyle behaviors such as physical activity.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "PHYSICAL ACTIVITY", "target": "GENETIC FACTORS AND LIFESTYLE"}, {"weight": 8.0, "description": "Diet and physical activity are lifestyle and environmental factors that modulate cardiovascular health.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "PHYSICAL ACTIVITY", "target": "DIET"}, {"weight": 3.0, "description": "Smoking status was considered alongside dietary patterns to assess potential interactions on health outcomes.)", "source_id": "chunk-e0d9c2e8ae2160bad4ae90ab6e8a12f6", "order": 1, "source": "DIETARY PATTERN AND SMOKING STATUS", "target": "LIFESTYLE/ENVIRONMENTAL FACTOR INTERACTION"}, {"weight": 4.0, "description": "A specific heterozygous mutation follows an autosomal dominant inheritance pattern.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "HETEROZYGOUS MUTATION", "target": "AUTOSOMAL DOMINANT INHERITANCE"}, {"weight": 3.0, "description": "The p.Arg322 His mutation is associated with a response to diazoxide treatment in patients.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "P.ARG322 HIS", "target": "DIAZOXIDE TREATMENT"}, {"weight": 6.0, "description": "The p.Ser498Leu mutation contributes to the HI/HA syndrome in patient 52.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "P.SER498LEU", "target": "HI/HA SYNDROME"}, {"weight": 3.0, "description": "Increased serum ammonia concentration may be associated with gastrointestinal disturbances and poor appetite.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "SERUM AMMONIA CONCENTRATION", "target": "GASTROINTESTINAL DISTURBANCES"}, {"weight": 7.0, "description": "Subtotal pancreatectomy is effective in patients unresponsive to drug therapy, such as diazoxide.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "SUBTOTAL PANCREATECTOMY", "target": "SURGERY RESPONSE"}, {"weight": 5.0, "description": "Some patients developed diabetes mellitus after surgery and required insulin treatment.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "DIABETES MELLITUS", "target": "OUTCOME OF SURGERY"}, {"weight": 1.0, "description": "Three patients were unresponsive to diazoxide therapy, leading to severe hypoglycemia and subsequent death.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "SEVERE HYPOGLYCAEMIA", "target": "DIAZOXIDE UNRESPONSIVENESS"}, {"weight": 3.0, "description": "Dietary choices and intake of certain nutrients can influence how NOS1 functions, affecting cognitive health.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "COGNITIVE FUNCTION", "target": "DIETARY PATTERNS"}, {"weight": 2.0, "description": "Genetic variations in NOS1 may exacerbate or mitigate risk for cardiovascular diseases through interaction with lifestyle factors.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "CARDIOVASCULAR DISEASE RISK FACTORS", "target": "NOS1 GENE VARIANTS"}, {"weight": 6.0, "description": "Five of the quinolone-resistant C. jejuni isolates found among retail food products or broiler chickens were also found in human isolates, highlighting possible environmental sources for drug resistance.)", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "order": 1, "source": "DRUG RESISTANCE", "target": "C. JEJUNI"}, {"weight": 9.0, "description": "Treatment outcomes can involve insulin dependency as a result of the management strategies for diabetes mellitus post-surgery.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "INSULIN DEPENDENCY", "target": "THERAPY RESPONSE"}, {"weight": 8.0, "description": "Multiorgan failure might be linked to other underlying health conditions or complications in patients undergoing complex surgical procedures.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "MULTIORGAN FAILURE", "target": "COMORBIDITY"}, {"weight": 4.0, "description": "Changes in GLUD1 gene expression through epigenetic mechanisms can impact the effectiveness of diazoxide treatment for HI/HA syndrome.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "DRUG THERAPY EFFECTIVENESS", "target": "EPIGENETIC MODIFICATIONS"}, {"weight": 5.0, "description": "Implementing a low-protein diet is an effective management strategy for patients experiencing hypoglycemia due to GLUD1 mutations.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "DIETARY INTERVENTIONS", "target": "HYPOGLYCEMIA MANAGEMENT"}, {"weight": 6.0, "description": "The development of diabetes mellitus after surgery indicates a possible long-term complication requiring specific therapeutic interventions.)", "source_id": "chunk-2b81371954b99facae9a0056e70aa7bd", "order": 1, "source": "DIABETES MELLITUS DEVELOPMENT", "target": "SURGERY OUTCOME"}, {"weight": 2.0, "description": "AcAc levels decrease significantly in IFG and increase significantly in diabetes compared to the reference category (FPG \u22645.4 mmol/L).)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "ACAC", "target": "FPG"}, {"weight": 4.0, "description": "AcAc level increases significantly in IGT and newly diagnosed diabetes as compared with the reference category (2hPG \u22645.9 mmol/L).)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "ACAC", "target": "2HPG"}, {"weight": 3.0, "description": "BHB level decreases significantly in IFG and increases significantly in diabetes compared to the reference category (FPG \u22645.4 mmol/L).)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "BHB", "target": "FPG"}, {"weight": 5.0, "description": "BHB level nominally increases slightly in IGT but significantly in newly diagnosed diabetes, compared to the reference category (2hPG \u22645.9 mmol/L).)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "BHB", "target": "2HPG"}, {"weight": 4.0, "description": "The NOS1 protein interacts with folate metabolism to maintain vascular function.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "NOS1 PROTEIN", "target": "FOLATE METABOLISM"}, {"weight": 3.0, "description": "The NOS1 gene interacts with folate metabolism to regulate nitric oxide production.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "FOLATE", "target": "NOS1"}, {"weight": 5.0, "description": "Folate is a nutrient found in leafy greens, which are important for cardiovascular health.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "FOLATE", "target": "LEAFY GREENS"}, {"weight": 1.0, "description": "Stratification analysis by gender did not find any association between the two SNPs and IA risk.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "GENDER"}, {"weight": 2.0, "description": "There was a minor association between rs4988235 and increased levels of saturated fats (C15:0 and C17:0).)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "RS4988235 AND PLASMA C15:0/C17:0"}, {"weight": 4.0, "description": "An analysis focusing on individuals with serum folate concentrations below the median revealed no association between these levels and male infertility.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "SERUM FOLATE CONCENTRATION BELOW STUDY POPULATION MEDIAN"}, {"weight": 1.0, "description": "The ALDH2 genotype\'s influence extends into environmental factors like smoking status in determining alcohol consumption patterns.)", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "GENETIC FACTOR"}, {"weight": 4.0, "description": "Frequency of non-persistence allele was observed differently in comparison to other populations.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "UK"}, {"weight": 6.0, "description": "Differences were observed based on different rs4988235 genotypes related to milk consumption.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "MILK CONSUMPTION PATTERNS"}, {"weight": 6.0, "description": "The relationship between the NOS1 gene and environmental factors can influence its activity in maintaining vascular function.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "NOS1"}, {"weight": 13.0, "description": "Human ovarian cancer ascites provides an in vitro model for testing the specificity, efficacy, and activity of TAG72-BB\u03b6 CAR T cells.)", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "order": 1, "source": "ENVIRONMENTAL FACTOR", "target": "OVARIAN CANCER ASCITES"}, {"weight": 1.0, "description": "Not specified in the text, no relationship mentioned.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "STATINS", "target": "PERSONALIZATION OF THERAPY"}, {"weight": 9.0, "description": "Statins are a drug therapy that may be part of personalized therapies for managing cardiovascular disease risk.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "STATINS", "target": "PERSONALIZED THERAPY"}, {"weight": 5.0, "description": "Highest quartile AcAc predicts conversion to type 2 diabetes (OR 1.32 [95% CI 1.00\u20131.74], P = 0.047).)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "ACAC& DEVELOPMENT OF HYPERGLYCEMIA", "target": "PREDICTOR"}, {"weight": 6.0, "description": "Highest quartile BHB also predicts conversion to type 2 diabetes, albeit with less statistical significance.)", "source_id": "chunk-f2e122f964bade37f487049bd69a3f9f", "order": 1, "source": "PREDICTOR", "target": "BHB& DEVELOPMENT OF HYPERGLYCEMIA"}, {"weight": 3.0, "description": "SNPs located near APOC1 show associations with CRP and other cardiometabolic phenotypes including T2D and CAD.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "APOC1", "target": "CRP"}, {"weight": 4.0, "description": "SNPs in the HNF1A gene are associated with risk reduction for CAD but increased serum CRP levels when carrying G-allele of rs1183910.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "HNF1A", "target": "CAD"}, {"weight": 9.0, "description": "Adjustments for CRP led to a considerable decrease in the estimate of association between PABPC4 and triglyceride levels.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "PABPC4", "target": "TRIGLYCERIDES"}, {"weight": 9.0, "description": "After adjusting for multiple factors, the effect of BCL7B on HDL-cholesterol is still noticeable but mostly mediated through lipid variables.)<SEP>For BCL7B and HDL-cholesterol levels, lipid variables appear to be the most important mediators after adjustment.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "BCL7B", "target": "HDL-CHOLESTEROL"}, {"weight": 7.0, "description": "Similarly to FTO, adjusted effects of SNP related to TMEM18 for pleiotropic outcomes are mainly mediated by BMI.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "TMEM18", "target": "MEDIATED PLEIOTROPY"}, {"weight": 2.0, "description": "The G-allele of the SNP rs1183910 in HNF1A gene is associated with lower risk of coronary artery disease, while its A-allele increases serum CRP levels.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "CAD", "target": "G-ALLELE OF RS1183910"}, {"weight": 3.0, "description": "The G-allele of SNP rs7041 in LPL gene shows association with CAD susceptibility.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "CAD", "target": "G-ALLELE OF RS7041"}, {"weight": 5.0, "description": "SNP rs17429505 in ADAMTS7 gene is associated with CAD risk through pleiotropic mechanisms.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "CAD", "target": "RS17429505"}, {"weight": 3.0, "description": "A statistically significant association was observed between rs20417 polymorphism and CAD.)", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "order": 1, "source": "CAD", "target": "RS20417"}, {"weight": 4.0, "description": "The study identified a strong link between rs20118426 polymorphism and the presence of CAD.)", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "order": 1, "source": "CAD", "target": "RS20118426"}, {"weight": 1.0, "description": "SNP rs3764260 near IL1F10 gene shows significant association with CRP levels and inflammation.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "CRP", "target": "RS3764260"}, {"weight": 3.6, "description": "The relationship between CRP and ovarian cancer risk was ambiguous due to inconsistent findings across study phases.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "CRP", "target": "LACK OF CONSISTENT ASSOCIATION"}, {"weight": 5.0, "description": "Individuals who develop diabetes typically have higher CRP levels.)", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "order": 1, "source": "CRP", "target": "DIABETES"}, {"weight": 0.72, "description": "Mean serum 18:2 n-6 significantly decreased (p=0.02), and both MUFA and n-3 PUFA significantly increased in the Mediterranean group.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "CHOLESTEROL", "target": "MEDITERRANEAN GROUP"}, {"weight": 4.0, "description": "SNP rs5698321 near MTHFD1L gene affects HDL-cholesterol levels.)", "source_id": "chunk-372184452f1f9156514992455057f843", "order": 1, "source": "HDL-CHOLESTEROL", "target": "RS5698321"}, {"weight": 0.006, "description": "The CC genotype of miR-34b/c rs4938723 was associated with a decreased risk of IA compared to the TT genotype.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "MIR-34B/C RS4938723", "target": "IA"}, {"weight": 2.0, "description": "No association between the number of aneurysms and miR-34b/c rs4938723 genotypes was observed.)<SEP>No significant association between miR-34b/c rs4938723 and the number of aneurysms was observed.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "MIR-34B/C RS4938723", "target": "NUMBER OF ANEURYSMS"}, {"weight": 1.0, "description": "Association between miR-34b/c rs4938723 genotypes and IA, but not with the number of aneurysms.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "MIR-34B/C RS4938723", "target": "NUMBER OF ANEURYSMS IN CASES"}, {"weight": 1.0, "description": "Genotype frequencies of miR-34b/c rs4938723 met the Hardy Weinberg equilibrium requirements.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "MIR-34B/C RS4938723", "target": "HARDY WEINBERG EQUILIBRIUM"}, {"weight": 0.006, "description": "The CC genotype of miR-34b/c rs4938723 was associated with a lower risk of IA compared to TT genotype.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "MIR-34B/C RS4938723", "target": "DECREASE RISK OF IA"}, {"weight": 1.0, "description": "No association between TP53 Arg72Pro and IA risk was observed in genotypic or allelic association analysis.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "TP53 ARG72PRO", "target": "IA"}, {"weight": 2.0, "description": "No significant association between TP53 Arg72Pro and the number of aneurysms was found.)<SEP>No significant link found between TP53 Arg72Pro and the number of aneurysms in the study.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "TP53 ARG72PRO", "target": "NUMBER OF ANEURYSMS"}, {"weight": 1.0, "description": "No association found between TP53 Arg72Pro and either IA risk or the number of aneurysms.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "TP53 ARG72PRO", "target": "NUMBER OF ANEURYSMS IN CASES"}, {"weight": 1.0, "description": "The genotype frequency distribution for TP53 Arg72Pro also met the Hardy-Weinberg equilibrium criteria.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "TP53 ARG72PRO", "target": "HARDY WEINBERG EQUILIBRIUM"}, {"weight": 0.004, "description": "Carriers with the combined genotypes of miR-34b/c rs4938723CC and TP53 Arg72Pro CG/CC/GG had a decreased risk of IA compared to other combined genotypes.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "IA", "target": "MIR-34B/C RS4938723 AND TP53 ARG72PRO"}, {"weight": 0.004, "description": "Carriers with the combined genotypes rs4938723CC and TP53 Arg72Pro CG/CC/GG had a reduced risk of IA.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "IA", "target": "COMBINED GENOTYPES"}, {"weight": 1.0, "description": "Gender was used as a variable for stratified analysis.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "GENDER", "target": "ANALYSIS METHOD"}, {"weight": 1.0, "description": "The T allele of rs2231142 ABCG2 was associated with sUA levels and HUA in both males and females, while the C risk allele of SLC2A9 was only related to females. The G risk allele of SLC22A12 was associated with sUA level only in males.)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "GENDER", "target": "ASSOCIATION BETWEEN GENOTYPES AND SUA LEVEL"}, {"weight": 1.0, "description": "The association between different genotypes and HUA risk differed by gender: the T allele of rs2231142 ABCG2 showed significance, while there was no significant association with SLC22A12 rs893006 variants.)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "GENDER", "target": "ASSOCIATION BETWEEN GENOTYPES AND HUA RISK"}, {"weight": 1.0, "description": "P-values for significant associations ranged from 0.004 to 0.006.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "LEVEL OF SIGNIFICANCE", "target": "P-VALUE"}, {"weight": 1.0, "description": "A p-value less than 0.05 was considered statistically significant for some associations, as seen in miR-34b/c rs4938723 association with IA risk.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "P-VALUE", "target": "SIGNIFICANCE LEVEL"}, {"weight": 2.0, "description": "Suggestive significance (p < 1 \u00d7 10\u22125) for genotyped SNPs in GWAS.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "P-VALUE", "target": "STATISTICAL MEASURE"}, {"weight": 1.0, "description": "Two SNPs\u2014miR-34b/c rs4938723 and TP53 Arg72Pro were analyzed for their potential association with IA.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "ASSOCIATION WITH DISEASE", "target": "SNPS"}, {"weight": 2.0, "description": "Association was identified between common SNPs and age at first tooth.)<SEP>Genetic variation at SNPs was associated with age at first tooth.)", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "AGE AT FIRST TOOTH"}, {"weight": 2.0, "description": "Association was identified between common SNPs and number of teeth.)<SEP>Genetic variation at SNPs was associated with number of teeth.)", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "NUMBER OF TEETH"}, {"weight": 5.0, "description": "SNPs associated with the identification of genome-wide significant loci.)", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "LOCI"}, {"weight": 8.0, "description": "SNPs were associated with novel gene/region for age at first tooth.)", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "NOVEL ASSOCIATIONS"}, {"weight": 4.0, "description": "SNPs associated with the confirmation of previously reported genes/regions.)", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "PREVIOUSLY REPORTED GENES/REGIONS"}, {"weight": 1.0, "description": "SNPs were associated with genome-wide significance in the phenotype.)", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "GENOME-WIDE SIGNIFICANCE"}, {"weight": 12.0, "description": "SNPs used in the creation of an allelic score that could inform therapeutic intervention.)<SEP>SNPs were used to create an allelic score that explained variation in phenotype.)", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "ALLELIC SCORE"}, {"weight": 5.0, "description": "SNPs showed suggestive associations beyond the genome-wide significance threshold.)", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "SUGGESTIVE LEVELS OF ASSOCIATION"}, {"weight": 9.0, "description": "SNPs were associated with the identification of secondary genomic variants within each locus.)", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "SECONDARY SIGNALS"}, {"weight": 1.0, "description": "SNPs underwent additional analyses to discern underlying genetic influences beyond the primary associations.)", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "SECONDARY ANALYSIS"}, {"weight": 8.0, "description": "SNPs implicated specific pathways related to tooth development and maturation.)", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "BIOLOGICAL PATHWAYS"}, {"weight": 7.0, "description": "SNPs were linked with changes in phenotype due to environmental exposure influencing development of dentition.)", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "PHENOTYPIC PLASTICITY"}, {"weight": 6.0, "description": "Estimations were based on correlations between SNPs and trait variability. )", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "HERITABILITY ESTIMATION"}, {"weight": 7.0, "description": "SNPs showed responsiveness to environmental factors that modify dentition development timelines. )", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "ENVIRONMENTAL INFLUENCE"}, {"weight": 5.0, "description": "SNPs were indicative of potential clinical applications in assessing risk or influencing prognosis of dental health issues related to age at first tooth and number of teeth. )", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "CLINICAL RELEVANCE"}, {"weight": 6.0, "description": "SNPs provided insights into the complex genetic structure influencing dental development timelines across diverse populations. )", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "GENETIC ARCHITECTURE"}, {"weight": 5.0, "description": "SNPs were analyzed considering population-specific genetic variations that might contribute to or mask associations with dentition development. )", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "POPULATION STRATIFICATION"}, {"weight": 6.0, "description": "SNPs were identified through a comprehensive analysis across various datasets, increasing the robustness of findings related to dentition development traits. )", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "META-ANALYSIS"}, {"weight": 6.0, "description": "SNPs were tracked across different developmental stages to uncover temporal patterns influencing dentition development. )", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "LONGITUDINAL DYNAMICS"}, {"weight": 5.0, "description": "SNPs were used as a basis for developing policies and guidelines that consider genetic predispositions in dental health management and public health initiatives. )", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "SNPS", "target": "PUBLIC HEALTH IMPLICATIONS"}, {"weight": 4.0, "description": "Three of the SNPs were associated with increased risk while three had protective effects on the social disinhibition score.)", "source_id": "chunk-36d70bb8409856300dd77669e7c1f9ed", "order": 1, "source": "SNPS", "target": "MINOR ALLELES"}, {"weight": 1.0, "description": "There were no associations found between the 23 SNPs and combination therapy outcomes, even after adjustments for multiple testing methods and covariates.)", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "order": 1, "source": "SNPS", "target": "COMBINATION THERAPY"}, {"weight": 1.0, "description": "Assessment of genotypic frequencies to confirm random mating and no evolutionary pressures.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "HARDY-WEINBERG EQUILIBRIUM", "target": "TEST TYPE"}, {"weight": 1.0, "description": "The statistical analysis for Hardy-Weinberg equilibrium was performed to check the validity of certain subgroup results in relation to genetic studies on ovarian cancer risk.)", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "order": 1, "source": "HARDY-WEINBERG EQUILIBRIUM", "target": "STATISTICAL ANALYSIS"}, {"weight": 5.0, "description": "Nominal evidence for departure from Hardy-Weinberg equilibrium was found only in Romania.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "HARDY-WEINBERG EQUILIBRIUM", "target": "ROMANIA"}, {"weight": 0.0, "description": "Satisfied except for FGFR4 rs376618; no significant differential treatment effects associated with this SNP.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "HARDY-WEINBERG EQUILIBRIUM", "target": "RS376618"}, {"weight": 1.0, "description": "No specific emphasis was placed on ancestry groups, with the study including participants from diverse backgrounds.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "GROUP FOCUS", "target": "INDIVIDUALS ANCESTRY"}, {"weight": 1.0, "description": "The control group was substantially larger than the case group (164 vs. 426 participants).)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "NUMBER OF CONTROLS", "target": "SAMPLE SIZE COMPARISON"}, {"weight": 1.0, "description": "The study involved individuals from different demographic groups, though no specific diversity measures were stated.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "STUDY POPULATION DIVERSITY", "target": "VARIETY OF PARTICIPANTS"}, {"weight": 1.0, "description": "The study did not specify the age distribution among either cases or controls.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "AGES RANGE", "target": "CHARACTERISTICS OF SAMPLE POPULATION"}, {"weight": 1.0, "description": "The sex ratio showed an imbalance, with more females than males among the study participants.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "CHARACTERISTICS OF SAMPLE POPULATION", "target": "GENDER PROPORTION"}, {"weight": 1.0, "description": "The study did not include any data regarding the ancestral backgrounds of participants.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "CHARACTERISTICS OF SAMPLE POPULATION", "target": "ANCESTRY BACKGROUND"}, {"weight": 1.0, "description": "The study did not specify the ethnic or ancestral diversity among participants.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "CHARACTERISTICS OF SAMPLE POPULATION", "target": "HERITAGE VARIABILITY"}, {"weight": 1.0, "description": "The number of cases with IA was 164, which constituted a smaller subgroup compared to the control population (426).)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "NUMBER OF CASES", "target": "SPECIFIC GROUP SIZE"}, {"weight": 1.0, "description": "The study utilized a case-control approach to compare individuals with IA against those without it.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "RESEARCH METHODOLOGY", "target": "STUDY DESIGN"}, {"weight": 1.0, "description": "The study included genome-wide genotyping and spirometry phenotypes in a GWAS.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "STUDY DESIGN", "target": "GWAS"}, {"weight": 1.0, "description": "The study likely involved statistical tools including Hardy Weinberg equilibrium test, P-value computation for assessing significance, etc., although specific details were not given.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "RESEARCH ANALYTICAL PROCESS", "target": "STATISTICAL METHODOLOGY"}, {"weight": 13.0, "description": "A multimarker model including all SNPs with P < 10\u22123 was performed using LASSO regression.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "STATISTICAL METHODOLOGY", "target": "MULTIMARKER MODELING"}, {"weight": 1.0, "description": "The study aimed at determining if genetic variants could predict or influence the incidence of IA.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "RESEARCH OBJECTIVE", "target": "SCIENTIFIC INQUIRY"}, {"weight": 1.0, "description": "The study included genetic variables like miR-34b/c rs4938723 and TP53 Arg72Pro, alongside other potential risk factors.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "RISK DETERMINANTS FOR DISEASE", "target": "SNP VARIABLES"}, {"weight": 1.0, "description": "Findings were interpreted statistically, comparing case and control groups for identifying potential risk factors.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "DATA ANALYSIS PROCESS", "target": "STATISTICAL EVALUATION"}, {"weight": 1.0, "description": "While the study included gender as an analysis variable, it found no consistent sex-specific associations with IA risk based on genetic markers.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "INFLUENCE ON RESEARCH OUTCOMES", "target": "SEX FACTOR"}, {"weight": 1.0, "description": "The sample encompassed both case and control groups that differed in terms of genetic risk factors, sex ratio, age range, etc., though exact details were not provided.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "CHARACTERISTICS OF PARTICIPANTS", "target": "STUDY SAMPLE PROFILES"}, {"weight": 1.0, "description": "Results with P-values less than 0.05 were considered statistically significant and worthy of further investigation.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "DETERMINING STATISTICAL RELEVANCE", "target": "SIGNIFICANCE THRESHOLD FOR P-VALUES"}, {"weight": 1.0, "description": "Rigorous approaches were taken with genetic data validation and statistical significance determination to ensure the robustness of findings.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "GENETIC DATA ANALYSIS RIGOR", "target": "QUALITY ASSURANCE IN RESEARCH METHODOLOGY"}, {"weight": 1.0, "description": "While ancestry was indirectly relevant through the sample\'s diversity, its direct impact on genetic findings for IA was not specifically addressed or analyzed.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "HERITAGE INFLUENCE", "target": "OUTCOME DETERMINANTS IN GENETIC RESEARCH"}, {"weight": 1.0, "description": "In the context of IA risk, no distinct gender-specific effects were discerned in relation to genetic markers such as miR-34b/c rs4938723 or TP53 Arg72Pro.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "GENDER-SPECIFIC ANALYSIS", "target": "SNP INFLUENCE ON DISEASE RISK"}, {"weight": 1.0, "description": "Analysis of genetic markers did not highlight any sex-specific patterns or differences that could significantly influence IA risk based on the data provided.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "SNP INFLUENCE ON DISEASE RISK", "target": "SEX DISTRIBUTION IMPACT"}, {"weight": 1.0, "description": "A mix of statistical analyses was employed to identify patterns or associations in the data related to genetic markers and IA disease prevalence.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "EXAMINATION TECHNIQUES FOR VARIABLES", "target": "STATISTICAL TOOLS USED"}, {"weight": 1.0, "description": "While the impact of specific genes was explored through SNPs like miR-34b/c rs4938723 and TP53 Arg72Pro, ancestral genetic influences were not a focal point.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "HERITAGE GENETICS", "target": "INFLUENCE ON DISEASE SUSCEPTIBILITY"}, {"weight": 1.0, "description": "Genetic links identified required a statistical significance ascertained through P-values below the established threshold to be considered reliable.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "DETERMINING GENETIC ASSOCIATIONS", "target": "STATISTICAL SIGNIFICANCE CRITERION"}, {"weight": 1.0, "description": "While sex could indirectly affect how genetic heritage influences IA risk, its specific effect was not a central focus given the study\'s concentration on SNPs like miR-34b/c rs4938723 and TP53 Arg72Pro.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "GENDER ROLE IN HEREDITARY IMPACT", "target": "MEDIATION OF GENETIC INFLUENCE ON DISEASE RISK"}, {"weight": 1.0, "description": "Ancestry diversity was not considered in the assessment of genetic markers\' relevance to IA risk, potentially missing nuanced insights into how different populations may differently respond based on their unique genetic makeup.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "ASSESSING GENETIC MARKER ASSOCIATIONS WITH DISEASE RISK", "target": "HERITAGE DIVERSITY INTERPRETATION"}, {"weight": 1.0, "description": "Finding a correlation between genomic factors and IA disease states required statistical evidence with P-values below the specified threshold for reliability.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "DETERMINING CORRELATION WITH DISEASE", "target": "STATISTICAL SIGNIFICANCE CRITERIA FOR GENOMIC FACTORS"}, {"weight": 1.0, "description": "SNPs were considered possible influencers of IA risk through statistical methods, seeking evidence that they could be linked to disease prevalence or predisposition within the sample populations.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "GENETIC MARKERS FOR RISK ASSESSMENT", "target": "POTENTIAL INFLUENCERS IN DISEASE STATE DETERMINATION"}, {"weight": 1.0, "description": "While sex could theoretically affect the correlations between specific SNPs and IA, its impact was not a primary focus given the study\'s emphasis on establishing associations rather than exploring nuanced gender-related dynamics.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "GENDER INFLUENCE ON GENETIC MARKER CORRELATIONS", "target": "POTENTIAL MODIFIERS IN SNP-AIA RISK ASSOCIATION"}, {"weight": 1.0, "description": "Given the primary focus on SNPs, no explicit analysis was conducted into how ancestral backgrounds might influence the relevance or predictive power of genetic markers in IA risk determination.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "ANCESTRY CONSIDERATION IN GENETIC MARKER CORRELATION", "target": "RELEVANCE TO DISEASE RISK ASSESSMENT"}, {"weight": 1.0, "description": "Genomic factors were evaluated for their correlation with disease prevalence using statistical methods like P-values and the Hardy Weinberg equilibrium test, seeking evidence that they could be predictive of IA risk based on data from case versus control groups.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "CORRELATION WITH DISEASE PREVALENCE DETERMINATION", "target": "STATISTICAL ASSESSMENT IN GENOMIC FACTOR EVALUATION"}, {"weight": 1.0, "description": "While sex was acknowledged as part of the population characteristics, its role did not feature prominently in the direct correlation assessments between genetic markers and IA risk based on the methodology employed by the study.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "INFLUENCE IN DISEASE RISK ASSESSMENT THROUGH SNPS", "target": "SEX ROLE IN GENETIC MARKER CORRELATION EVALUATION"}, {"weight": 1.0, "description": "Given the study\'s focus on establishing correlations rather than exploring nuanced population dynamics, heritage diversity was not a primary factor considered when assessing the relevance of genetic markers like miR-34b/c rs4938723 and TP53 Arg72Pro in IA risk determinations through statistical means.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "HERITAGE DIVERSITY\'S ROLE IN GENETIC MARKER CORRELATION EVALUATION", "target": "RELEVANCE TO DISEASE RISK THROUGH STATISTICAL ASSESSMENT"}, {"weight": 1.0, "description": "To assess the meaningful impact of genetic factors on IA disease states, P-value thresholds were applied during comparative case-control analyses involving SNPs like miR-34b/c rs4938723 and TP53 Arg72Pro to determine if they significantly correlated with disease prevalence or predisposition.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "SIGNIFICANCE IN DISEASE STATE EVALUATION THROUGH COMPARATIVE ANALYSIS", "target": "STATISTICAL CRITERIA FOR GENETIC FACTOR IMPACT DETERMINATION"}, {"weight": 1.0, "description": "The evaluation process sought to ascertain if specific genetic markers could contribute meaningfully to IA risk determinations by analyzing their correlations with case versus control populations, potentially factoring in sex or heritage diversity as secondary considerations.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "CORRELATION DETERMINATION FOR DISEASE PREVALENCE THROUGH STATISTICAL ANALYSIS", "target": "GENETIC MARKER EVALUATION IN RISK ASSESSMENT"}, {"weight": 1.0, "description": "The process involved applying statistical tools like P-values and Hardy Weinberg equilibrium tests to assess if correlations exist between specific SNPs and IA disease states, with sex or heritage diversity factored into secondary analyses rather than being the primary focus in determining genetic marker significance.)", "source_id": "chunk-055177812e320344d131432b292111b0", "order": 1, "source": "CORRELATION ASSESSMENT FOR DISEASE STATE DETERMINATION THROUGH STATISTICAL ANALYSIS", "target": "STATISTICAL SIGNIFICANCE IN GENETIC MARKER CORRELATION EVALUATION"}, {"weight": 1.0, "description": "Construction of allelic scores from genome-wide significant SNPs for \'Age at first tooth\' and \'Number of teeth\' as part of GWAS.)", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "order": 1, "source": "GWAS", "target": "ALLELIC SCORE"}, {"weight": 1.0, "description": "A Genome-Wide Association Study was conducted to find the genetic polymorphisms associated with habitual coffee consumption.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "GWAS", "target": "STATISTICAL ANALYSIS"}, {"weight": 8.0, "description": "The intronic variant in ARHGAP11A on chromosome 15 was associated with sleep duration.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "ARHGAP11A", "target": "SLEEP DURATION"}, {"weight": 7.0, "description": "A SNP in the intronic region of C21orf34 detected for RQ during sleep.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "C21ORF34", "target": "RQ (RESPIRATORY QUOTIENT)"}, {"weight": 2.0, "description": "Significantly higher free testosterone levels were noted in women with PCOS.)", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "order": 1, "source": "PCOS", "target": "FREE TESTOSTERONE"}, {"weight": 3.0, "description": "Women with PCOS showed differences in bioactive testosterone compared to control subjects.)", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "order": 1, "source": "PCOS", "target": "BIOACTIVE TESTOSTERONE"}, {"weight": 5.0, "description": "Elevated HOMA-IR was seen in women with PCOS, reflecting insulin resistance.)", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "order": 1, "source": "PCOS", "target": "HOMA-IR"}, {"weight": 6.0, "description": "Lower levels of SHBG were noted among women with PCOS compared to controls.)", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "order": 1, "source": "PCOS", "target": "SHBG"}, {"weight": 7.0, "description": "Although within the normal range, TSH was higher among women with PCOS compared to control subjects.)", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "order": 1, "source": "PCOS", "target": "TSH"}, {"weight": 2.0, "description": "The assays described above suggest that type-1-like diabetic phenotypes developed in the adipocyte-specific SENP1-deficient mice.)", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "order": 1, "source": "DIABETES", "target": "SENP1"}, {"weight": 4.0, "description": "There was a sharp decline in insulin staining and CD31+ vascular area in the islets in SENP1-aP2KO mice (Fig. 3d with quantifications in Fig. 3e), indicating a loss of function in the \u03b2 cell with disrupted vasculatures.)", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "order": 1, "source": "DIABETES", "target": "\u0392 CELLS"}, {"weight": 6.0, "description": "Beta-hydroxybutyrate is classified as ketones that are produced in the body when the body loses its ability to absorb glucose properly.)", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "order": 1, "source": "DIABETES", "target": "\u0392-HYDROXYBUTYRATE"}, {"weight": 7.0, "description": "The distribution of immune cells in the pancreases was determined by fluorescence-activated cell sorting (FACS) analyses with specific surface markers. MHCII+CD11b+CD11c\u2212 macrophages, CD3e+ T cells and CD19+ B cells were markedly increased in the pancreases of SENP1-aP2KO versus the Ctrl mice.)", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "order": 1, "source": "DIABETES", "target": "MHCII+CD11B+CD11C\u2212 MACROPHAGES"}, {"weight": 5.0, "description": "Increased proinflammatory cytokines, cardiac function changes and vascular reactivity issues are potential causes leading to diabetes in SENP1-deficient mice.)", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "order": 1, "source": "DIABETES", "target": "CAUSE OF"}, {"weight": 7.0, "description": "IL-6, like IL-1\u03b2, plays an important role in exacerbating inflammation through its ability to stimulate the production of other pro-inflammatory cytokines such as TNF-\u03b1.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "IL-6", "target": "PRO-INFLAMMATORY CYTOKINE"}, {"weight": 8.0, "description": "Inhibition of IL-6 activity by blocking its receptor partially abrogates Th17 formation with Rag1\u2212/\u2212 LPM\u03c6s.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "IL-6", "target": "IL-10 TREATMENT"}, {"weight": 5.0, "description": "|>SENP1-aP2KO", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "IL-6", "target": "PROINFLAMMATORY CYTOKINES/PATS/"}, {"weight": 4.0, "description": "A SNP in the C21orf34 intronic region detected for RQ during sleep.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "RESPIRATORY QUOTIENT (RQ)", "target": "C21ORF34 INTRONIC REGION"}, {"weight": 5.0, "description": "An intronic variant in ARHGAP11A on chromosome 15 was associated with sleep duration.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "SLEEP DURATION", "target": "ARHGAP11A 15TH CHROMOSOME"}, {"weight": 6.0, "description": "Accelerometer-measured light activity variant was found in the 5\u2032UTR region of RPL7P3.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "LIGHT ACTIVITY", "target": "RPL7P3 5\u2032UTR"}, {"weight": 6.0, "description": "Accelerometer-measured light activity variant found in the RPL7P3 5\u2032UTR region.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "LIGHT ACTIVITY", "target": "RPL7P3 5\u2032UTR REGION"}, {"weight": 12.0, "description": "Sedentary-light activity was measured in the 3\u2032UTR of CTCFL on chromosome 20.)<SEP>Sedentary-light activity was measured using CTCFL\'s 3\u2032UTR sequence on chromosome 20.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "CTCFL 3\u2032UTR", "target": "SEDENTARY-LIGHT ACTIVITY"}, {"weight": 4.0, "description": "A highly significant variant identified for total energy expenditure (p\u200a=\u200a2.7E-08) was associated with MATK on chromosome 19.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "MATK CHROMOSOME 19", "target": "TOTAL ENERGY EXPENDITURE"}, {"weight": 3.0, "description": "A highly significant variant associated with total energy expenditure (p\u200a=\u200a2.7E-08) was identified for MATK on chromosome 19.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "TOTAL ENERGY EXPENDITURE", "target": "MATK 19TH CHROMOSOME"}, {"weight": 3.0, "description": "Sleeping energy expenditure variant identified for CHRNA3 on chromosome 15.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "CHRNA3 CHROMOSOME 15", "target": "SLEEPING ENERGY EXPENDITURE"}, {"weight": 2.0, "description": "Sleeping energy expenditure variant identified for CHRNA3 on chromosome 15.)", "source_id": "chunk-46227b557f62c59155b1b9cef71d0d17", "order": 1, "source": "SLEEPING ENERGY EXPENDITURE", "target": "CHRNA3 15TH CHROMOSOME"}, {"weight": 1.0, "description": "rs61737565 variant affects splicing by potentially altering binding sites for splicing regulators.)", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "order": 1, "source": "SPLICING REGULATOR", "target": "GENOMIC VARIANT RS61737565"}, {"weight": 1.0, "description": "The genomic variant rs61737565 affects the TUFM gene by altering splicing regulator or transcription factor binding sites.)", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "order": 1, "source": "GENOMIC VARIANT RS61737565", "target": "TUFM GENE"}, {"weight": 1.0, "description": "The genomic variant rs8061877 disrupts the binding of transcription factors SRY, HFH 2, and HMG-IY in the TUFM gene.)", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "order": 1, "source": "TUFM GENE", "target": "GENOMIC VARIANT RS8061877"}, {"weight": 1.0, "description": "The genomic variant g.28854194C/G causes changes in splice enhancer and splicing silencer sites within the TUFM gene.)", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "order": 1, "source": "TUFM GENE", "target": "GENOMIC VARIANT G.28854194C/G"}, {"weight": 1.0, "description": "The non-coding 3\u2019 UTR region of TUFM contains genomic variant g.28854194C/G which affects the gene function.)", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "order": 1, "source": "TUFM GENE", "target": "NON-CODING 3\u2019 UTR OF TUFM"}, {"weight": 1.0, "description": "Disruption of transcription factor binding sites occurs due to rs8061877 variant, influencing gene regulation.)", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "order": 1, "source": "GENOMIC VARIANT RS8061877", "target": "TRANSCRIPTION FACTOR BINDING SITES"}, {"weight": 1.0, "description": "g.28854194C/G affects splice sites via changes in enhancer and silencer regions.)", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "order": 1, "source": "GENOMIC VARIANT G.28854194C/G", "target": "SPLICE ENHANCER AND SPLICING SILENCER SITES"}, {"weight": 1.0, "description": "The non-coding 3\u2019 UTR can impact gene expression independently or through interactions with genomic variants like g.28854194C/G.)", "source_id": "chunk-fde6653b9d9013ed054505cfe4cd1455", "order": 1, "source": "NON-CODING 3\u2019 UTR OF TUFM", "target": "GENE EXPRESSION REGULATION"}, {"weight": 3.4, "description": "No strong evidence suggests that ALOX15 plays a significant role in increasing the risk of ovarian cancer as per the study phases.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "ALOX15", "target": "LACK OF ASSOCIATION"}, {"weight": 3.7, "description": "The relationship between IL18 and ovarian cancer risk was unclear due to varying results across different stages of research phases.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "IL18", "target": "PHASE OF STUDIES"}, {"weight": 3.3, "description": "The variant rs2282169 within the PTGS1 gene is linked to an association with increased risk for ovarian cancer, indicating its role in this disease\'s etiology.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "PTGS1", "target": "RS2282169"}, {"weight": 3.9, "description": "The variant rs2282169 in PTGS1 gene appears to be linked with ovarian cancer risk, although this was not explicitly stated in the text.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "PTGS1", "target": "RS2282169& OVARIAN CANCER RISK"}, {"weight": 3.11, "description": "A potential connection exists between rs2282169 variant in PTGS1 gene and ovarian cancer, though this was not extensively detailed.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "PTGS1", "target": "RS2282169&POTENTIAL OVARIAN CANCER LINKAGE"}, {"weight": 3.2, "description": "A strong genetic relationship exists between the IL1B gene and ovarian cancer risk due to the specific genomic variant rs7596684.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "IL1B", "target": "RS7596684"}, {"weight": 3.1, "description": "A possible link exists between rs7596684 variant within IL1B gene and increased risk of ovarian cancer based on some findings, though this was not fully elucidated.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "IL1B", "target": "RS7596684&OVARIAN CANCER RISK"}, {"weight": 3.12, "description": "A preliminary linkage between rs7596684 variant in IL1B gene and potential increase in risk of ovarian cancer is suggested by some observations, though these were not thoroughly explained.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "IL1B", "target": "RS7596684&POTENTIAL INCREASED OVARIAN CANCER RISK"}, {"weight": 3.5, "description": "Results on the relationship between CCL2 and ovarian cancer risk were mixed throughout different study phases, indicating uncertain associations.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "CCL2", "target": "STUDY PHASES"}, {"weight": 3.8, "description": "The relationship between IL6 and ovarian cancer risk was uncertain due to inconsistent findings throughout different research phases.)", "source_id": "chunk-b6ea16f7f7eb8bc6f36f2be03fe62403", "order": 1, "source": "IL6", "target": "LACK OF CONSISTENT ASSOCIATION"}, {"weight": 1.0, "description": "Significant associations were present in clear cell, mucinous, and endometrioid subtypes, suggesting different risk profiles.)(entity", "source_id": "chunk-089bcdf055eb2f007db49dbe2409429b", "order": 1, "source": "HETEROGENEITY FOR CLEAR CELL, MUCINOUS, AND ENDOMETRIOID SUBTYPES (IL1A RS17561)", "target": "P=0.03"}, {"weight": 1.0, "description": "MTHFD2 gene is part of the folate metabolic pathway being analyzed in relation to childhood ALL risk.)", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "order": 1, "source": "MTHFD2", "target": "FOLATE METABOLIC PATHWAY"}, {"weight": 1.0, "description": "BHMT2 gene contributes to the overall folate metabolic pathway evaluated for its association with childhood ALL.)", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "order": 1, "source": "BHMT2", "target": "FOLATE METABOLIC PATHWAY"}, {"weight": 1.0, "description": "SLC19A1 is involved in a component of the folate metabolic pathway that was assessed for its role in childhood ALL risk.)", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "order": 1, "source": "SLC19A1", "target": "FOLATE METABOLIC PATHWAY"}, {"weight": 7.38, "description": "NOS3 rs3918186 showed a significant association with an increased risk of childhood ALL based on Bayes Factors and odds ratios.)", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "order": 1, "source": "CHILDHOOD ACUTE LYMPHOBLASTIC LEUKEMIA", "target": "NOS3 RS3918186"}, {"weight": 1.0, "description": "While SLC19A1 rs1051266 had some evidence for association, the strength was weaker than that of NOS3 rs3918186.)", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "order": 1, "source": "CHILDHOOD ACUTE LYMPHOBLASTIC LEUKEMIA", "target": "SLC19A1 RS1051266"}, {"weight": 1.0, "description": "The data from Table 2 were used as criteria for selecting which SNPs would proceed to initial screening based on their relevance and completeness.)", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "order": 1, "source": "INITIAL SCREENING CRITERIA", "target": "TABLE 2 POPULATION CHARACTERISTICS"}, {"weight": 2.0, "description": "SNPs initially identified in the study underwent an initial screening process, where those with a high posterior probability were selected for further analysis.)", "source_id": "chunk-0ce38a2d49e01ec790464306e01de9b4", "order": 1, "source": "INITIAL SCREENING PROCESS", "target": "SNP INITIAL IDENTIFICATION"}, {"weight": 1.0, "description": "The distribution of minor alleles within the FADS gene cluster defined by the haplotype.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "FADS1/2 HAPLOTYPE", "target": "MINOR ALLELE FREQUENCY"}, {"weight": 1.0, "description": "No significant genotype-by-diet interactions were found for EPA concentration.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "EPA", "target": "NO GENOTYPE-DIET INTERACTION"}, {"weight": 0.01, "description": "Long chain n-3 PUFA significantly increased in both Healthy Eating and Mediterranean groups.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "COLON FATTY ACID CONCENTRATIONS", "target": "HEALTHY EATING GROUP"}, {"weight": 0.77, "description": "Mean serum 18:2 n-6 significantly decreased (p=0.02), and both MUFA and n-3 PUFA significantly increased in the Mediterranean group.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "COLON FATTY ACID CONCENTRATIONS", "target": "MEDITERRANEAN GROUP"}, {"weight": 0.004, "description": "There was a significant interaction of genotype by diet for AA concentrations in the colon.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "20:4, N-6 (AA)", "target": "COLON CONCENTRATIONS"}, {"weight": 0.02, "description": "Serum 20:4, n-6 (AA) concentration significantly decreased in the Mediterranean group.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "20:4, N-6 (AA)", "target": "MEDITERRANEAN GROUP"}, {"weight": 0.77, "description": "Both MUFA and n-3 PUFA significantly increased (p=0.0005 and p=0.01, respectively) in the Mediterranean arm only.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "MUFA", "target": "MEDITERRANEAN GROUP"}, {"weight": 0.72, "description": "Both MUFA and n-3 PUFA significantly increased (p=0.0005 and p=0.01, respectively) in the Mediterranean arm only.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "N-3 PUFA", "target": "HEALTHY EATING GROUP"}, {"weight": 0.72, "description": "Mean serum 18:2 n-6 significantly decreased (p=0.02) in the Mediterranean group.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "SERUM 18:2 N-6", "target": "MEDITERRANEAN GROUP"}, {"weight": 3.0, "description": "The correlation between successful genotyping and minor allele frequency in SNPs.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "GENOTYPING SUCCESS RATE", "target": "MINOR ALLELE FREQUENCIES"}, {"weight": 4.0, "description": "The predictive relationship of dietary intake on serum levels of specific fatty acids (AA, EPA).)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "DIETARY INTAKES", "target": "SERUM FATTY ACID CONCENTRATIONS"}, {"weight": 6.0, "description": "How changes in dietary intake during an intervention affect serum levels of various fatty acids.)", "source_id": "chunk-8b9890cf10c4a751a1d0ec7e9843002d", "order": 1, "source": "SERUM FATTY ACID CONCENTRATIONS", "target": "DIETARY INTERVENTION EFFECTS"}, {"weight": 0.004, "description": "Mean colon AA concentrations differed significantly between diet arms at 6 months for subjects without minor alleles in the FADS1/2 gene cluster.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "COLON AA CONCENTRATIONS", "target": "DIET ARM"}, {"weight": 1.0, "description": "Mean colon AA concentrations were estimated to be 16% lower in the Mediterranean arm than the Healthy Eating arm at 6 months.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "COLON AA CONCENTRATIONS", "target": "MEDITERRANEAN GROUP"}, {"weight": 0.01, "description": "Mean colon AA concentrations were estimated to be 16% (95% CI = [5%, 26%]) lower for the Healthy Eating arm than the Mediterranean arm at 6 months.)", "source_id": "chunk-e1b4f4dac52e806f19c580640a50c729", "order": 1, "source": "COLON AA CONCENTRATIONS", "target": "NO MINOR ALLELES"}, {"weight": 2.0, "description": "The genetic correlation between the adjusted genome-wide association study (GWAS) and snoring shows high consistency.)", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "order": 1, "source": "SNORING", "target": "ADJUSTED GWAS"}, {"weight": 6.0, "description": "Genetic correlation between nocturnal hypoxia measures such as minimum SpO2 and average SpO2 shows small increases in relation with snoring after adjusting for BMI.)", "source_id": "chunk-13862d7ab74f5d4fa3f2261984d15d64", "order": 1, "source": "SNORING", "target": "NOCTURNAL HYPOXIA"}, {"weight": 1.0, "description": "Sleep apnoea showed a high genetic correlation (rG = 0.78) with snoring, indicating shared genetic factors.)", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "order": 1, "source": "SLEEP APNOEA", "target": "GENETIC CORRELATION WITH SNORING"}, {"weight": 0.0, "description": "There was no significant genetic correlation between smoking frequency and sleep apnoea, indicating different etiologies.)", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "order": 1, "source": "SMOKING FREQUENCY", "target": "SNP HERITABILITY FOR SLEEP APNOEA"}, {"weight": 0.0, "description": "While alcohol consumption was not significantly genetically correlated with snoring, it might have had an indirect effect on sleep apnoea through shared risk factors.)", "source_id": "chunk-9977706363cc368b3be503d07ed3ef62", "order": 1, "source": "ALCOHOL CONSUMPTION FREQUENCY", "target": "GENETIC CORRELATION WITH SLEEP APNOEA"}, {"weight": 9.0, "description": "Lead single nucleotide polymorphisms (SNPs) at eGFR loci are significantly associated with kidney function and disease.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "LEAD SNPS", "target": "EGFR"}, {"weight": 3.0, "description": "Percentage distribution of genotypes based on VDBP polymorphisms for mothers and neonates are presented (Table 2).)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "VDBP POLYMORPHISMS", "target": "GENOTYPE PERCENTAGE DISTRIBUTIONS"}, {"weight": 3.0, "description": "Certain maternal VDBP genotypes were associated with higher vitamin D levels when the mother delivered (Table 8).)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "VDBP POLYMORPHISMS", "target": "MATERNAL VITAMIN D STATUS DURING DELIVERY"}, {"weight": 5.0, "description": "Mothers with specific VDBP genotypes tended to have higher vitamin D concentrations at birth compared to other genotypes (p = 0.05 and p = 0.04 respectively, as seen in Table 8).)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "MATERNAL VDBP GENOTYPES (CC FOR RS2298850 AND CC FOR RS4588)", "target": "HIGHER VITAMIN D CONCENTRATIONS DURING DELIVERY"}, {"weight": 5.0, "description": "Specific maternal VDBP genotypes tended to have higher vitamin D levels at the time of delivery compared to other genotypes.)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "MATERNAL VDBP GENOTYPES (CC FOR RS2298850 AND CC FOR RS4588)", "target": "MATERNAL VITAMIN D CONCENTRATIONS DURING DELIVERY"}, {"weight": 6.0, "description": "Neonatal vitamin D statuses varied based on maternal VDBP polymorphisms when compared using cut-offs of 25 and 50 nmol/L (Tables 9 and 10).)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "NEONATAL 25(OH)D CONCENTRATIONS, WITH RESPECT TO MATERNAL VDBP GENOTYPE DISTRIBUTION", "target": "ASSOCIATION BETWEEN GENOTYPE AND OUTCOME"}, {"weight": 2.0, "description": "Maternal vitamin D concentrations were analyzed in relation to her genotypes for VDBP polymorphisms.)<SEP>The vitamin D status of mothers was analyzed in relation to their VDBP polymorphisms.)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "MATERNAL VITAMIN D STATUS", "target": "MOTHERS"}, {"weight": 4.0, "description": "Vitamin D cut-offs at birth were compared with maternal VDBP polymorphisms (Table 7).)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "MATERNAL VITAMIN D STATUS", "target": "CUT-OFFS AT BIRTH (\u226550 NMOL/L VS. \u226450 NMOL/L AND \u226525 VS. \u226425 NMOL/L)"}, {"weight": 4.0, "description": "The health and developmental outcomes of neonates were compared with their mothers\' VDBP genotypes, looking at cut-offs of 25 and 50 nmol/L for vitamin D.)<SEP>Vitamin D status of neonates was also assessed with respect to their VDBP polymorphisms.)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "NEONATAL VITAMIN D STATUS", "target": "NEONATES"}, {"weight": 6.0, "description": "Neonatal vitamin D status was not significantly different based on maternal VDBP genotypes with cut-offs of 25 and 50 nmol/L at birth (Table 9 and Table 10).)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "NEONATAL VITAMIN D STATUS", "target": "MATERNAL VDBP POLYMORPHISMS"}, {"weight": 4.0, "description": "Neonatal vitamin D statuses were compared with different maternal VDBP genotypes based on birth cut-offs for vitamin D.)", "source_id": "chunk-617cab4d8c10048a1335bf565693bf97", "order": 1, "source": "ASSOCIATION BETWEEN GENOTYPE AND OUTCOME", "target": "NEONATAL VITAMIN D CUT-OFFS AT BIRTH (25 VS. 50 NMOL/L)"}, {"weight": 9.0, "description": "No correlation was found between the VEGFA SNP rs3025020 and levels of anti-inflammatory cytokine IL-10 among women with PCOS.)", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "order": 1, "source": "RS3025020 (-583T>C)", "target": "ANTI-INFLAMMATORY IL-10"}, {"weight": 10.0, "description": "Correlation analysis revealed no link between the VEGFA SNP rs2010963 and levels of pro-inflammatory cytokine TNF\u03b1 among PCOS women.)", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "order": 1, "source": "RS2010963 (-634C>G)", "target": "PRO-INFLAMMATORY TNF\u0391"}, {"weight": 11.0, "description": "Among the model including rs2010963, women with the minor allele (C) showed lower erythrocyte sedimentation rate (ESR).)", "source_id": "chunk-2c178ae99eeffdddbea511cafbe4a7d5", "order": 1, "source": "RS2010963 (-634C>G)", "target": "ESR"}, {"weight": 2.3200000000000003, "description": "Val carriers had a distinct pattern of response bias reduction in the stress condition relative to no-stress conditions; this relationship held even after accounting for various factors like discriminability, shock intensity, skin conductance responses, and CRHR1 SNPs.)<SEP>Val carriers tended to have lower response bias in the stress condition compared to no-stress conditions; this difference was not statistically significant for iso homozygotes.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "STRESS CONDITION", "target": "NO-STRESS CONDITION"}, {"weight": 1.62, "description": "Differences in discriminability were evident between Conditions 3 and 2, both of which showed elevated values compared to Condition 1.)<SEP>Enhanced performance was noted in Conditions/Blocks with elevated discriminability compared to the baseline condition.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "BLOCK DISCRIMINABILITY", "target": "CONDITION 3 AND CONDITION 2"}, {"weight": 0.8, "description": "An elevated discriminability emerged specifically in Conditions 3 and 2; performance was better than in Condition 1.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "CONDITION 3 AND CONDITION 2", "target": "ELEVATED DISCRIMINABILITY"}, {"weight": 0.78, "description": "Improved performance was noted specifically in Conditions/Blocks where discriminability was elevated.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "CONDITION 3 AND CONDITION 2", "target": "BLOCK 2 AND BLOCK 3"}, {"weight": 0.9, "description": "Enhanced performance was noted specifically in Conditions/Blocks where the discriminability metric was elevated; these blocks showed better performance compared to baseline conditions.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "CONDITION 3 AND CONDITION 2", "target": "IMPROVED PERFORMANCE"}, {"weight": 1.22, "description": "Val carriers tended to have lower response bias in the stress condition compared to no-stress conditions; this difference was not statistically significant for iso homozygotes.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "COHORT COMPARISON", "target": "STRESS CONDITION AND NO-STRESS CONDITION"}, {"weight": 1.0, "description": "Val carriers showed a reduced response bias in the stress condition relative to no-stress conditions; this was observed even after considering various factors like discriminability, shock intensity, skin conductance responses, and CRHR1 SNPs.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "STRESS CONDITION AND NO-STRESS CONDITION", "target": "GENOTYPE-SPECIFIC RESPONSE BIAS"}, {"weight": 1.0, "description": "Val carriers had reduced response bias specifically in the stress condition when compared to no-stress conditions; this relationship held after factoring in discriminability, shock intensity, skin conductance responses, and CRHR1 SNPs.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "STRESS CONDITION AND NO-STRESS CONDITION", "target": "COHORT COMPARISON OF RESPONSE BIAS"}, {"weight": 0.55, "description": "There were differences between Blocks 1, 2, and 3 in terms of discriminability; Block 2 and Block 3 showed improved performance compared to Block 1.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "BLOCK SEQUENCE EFFECTS", "target": "CONDITION 1, CONDITION 2 AND CONDITION 3"}, {"weight": 0.8, "description": "Performance varied between Conditions/Blocks due to differences in discriminability; Blocks showed enhanced performance compared to the first block.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "BLOCK SEQUENCE EFFECTS", "target": "CONDITION 1, CONDITION 2, AND CONDITION 3"}, {"weight": 1.63, "description": "For discriminability, a main effect of Block emerged due to elevated discriminability in Blocks 3 and 2 relative to Block 1, both p < .02.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "CONDITION 1", "target": "CONDITION 1"}, {"weight": 1.66, "description": "These SNPs were recently associated with stress-induced reward learning deficits, which influenced the findings on response bias.)<SEP>Three CRHR1 SNPs were associated with stress-induced reward learning deficits, which were taken into account as part of the control analyses.)<SEP>Three CRHR1 SNPs were associated with stress-induced reward learning deficits; these were considered as covariates.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "CRHR1 SNPS", "target": "STRESS-INDUCED REWARD LEARNING DEFICITS"}, {"weight": 1.0, "description": "Unique variance in response bias reductions was determined after accounting for discriminability, shock intensity, and skin conductance responses.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "DISCRIMINABILITY, SHOCK INTENSITY, SKIN CONDUCTANCE RESPONSES", "target": "RESPONSE BIAS REDUCTIONS"}, {"weight": 1.0, "description": "Unique variance was explained by response bias reductions even after factoring in the effects of discriminability and skin conductance responses along with shock intensity.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "RESPONSE BIAS REDUCTIONS", "target": "DISCRIMINABILITY AND SHOCK INTENSITY, SKIN CONDUCTANCE RESPONSES"}, {"weight": 0.9, "description": "The sequence of conditions influenced performance outcomes; improvements were noted specifically in Conditions/Blocks where performance was enhanced due to increased discriminability.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "CONDITION 3, CONDITION 2, AND CONDITION 1", "target": "SEQUENCE OF CONDITIONS EFFECTS"}, {"weight": 0.6, "description": "After considering the association of specific CRHR1 SNPs with stress-induced changes, unique variance was identified in response bias reductions during stressed conditions.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "CRHR1 SNPS AND STRESSED CONDITION RESPONSE BIAS REDUCTIONS", "target": "UNIQUE VARIANCE IN STRESS-INDUCED CHANGES"}, {"weight": 1.0, "description": "Val carriers showed reduced response bias specifically in the stress condition compared to no-stress conditions; this relationship was observed even after considering factors like discriminability, shock intensity, skin conductance responses, and CRHR1 SNPs for both groups.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "STRESS CONDITION COMPARISON BETWEEN GROUPS", "target": "VAL AND ISO HETEROZYGOTES RESPONSE BIAS"}, {"weight": 0.9, "description": "Within Conditions/Blocks where performance was enhanced due to elevated discriminability, improvements were observed; these conditions showed superior performance compared to baseline states after adjusting for various factors.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "CONDITION 3 AND CONDITION 2 EFFECTS COMPARISON", "target": "PERFORMANCE ENHANCEMENT IN ELEVATED DISCRIMINABILITY CONDITIONS"}, {"weight": 0.9, "description": "Elevated performance within Conditions/Blocks that featured elevated discriminability indicated enhanced outcomes; these instances showed superior performance compared to baseline states, adjusting for various factors.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "CONDITION 3 AND CONDITION 2 EFFECTS COMPARISON", "target": "PERFORMANCE ENHANCEMENT IN SPECIFIC BLOCKS/DAYS/"}, {"weight": 1.0, "description": "Elevated performance within Conditions/Blocks that featured elevated discriminability indicated enhanced outcomes; these instances showed superior performance compared to baseline states, adjusted for various factors including CRHR1 SNPs.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "CONDITION 3 AND CONDITION 2 EFFECTS COMPARISON", "target": "PERFORMANCE ENHANCEMENT IN SPECIFIC BLOCKS/CONDITIONS AND DISCRIMINABILITY ASSOCIATION"}, {"weight": 0.9, "description": "Val carriers exhibited reduced response bias specifically in the stress condition relative to no-stress conditions, a relationship that was maintained after adjusting for factors like discriminability, shock intensity, skin conductance responses, and CRHR1 SNPs.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "GENOTYPE-SPECIFIC RESPONSE BIAS REDUCTIONS DURING STRESSED CONDITIONS", "target": "STRESS CONDITION AND NO-STRESS CONDITION COMPARISON"}, {"weight": 0.9, "description": "Val carriers\' reduced response bias specifically in the stress condition relative to no-stress conditions, which maintained after adjusting for factors like discriminability, shock intensity, skin conductance responses, was further informed by considering specific CRHR1 SNPs.)", "source_id": "chunk-5e709975097a773fb3d31e265d76a71b", "order": 1, "source": "STRESS CONDITION AND NO-STRESS CONDITION COMPARISON", "target": "GENOTYPE-SPECIFIC RESPONSE BIAS REDUCTIONS DURING STRESSED CONDITIONS AND CRHR1 SNPS ASSOCIATION"}, {"weight": 1.0, "description": "SNP associated with pathway analysis using the ALLIGATOR software.)", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "order": 1, "source": "AGE AT FIRST TOOTH", "target": "PATHWAY"}, {"weight": 1.0, "description": "No significant over-representation of pathways was found for \'number of teeth\' GWAS, despite SNP association P-values.)", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "order": 1, "source": "NUMBER OF TEETH", "target": "PATHWAY"}, {"weight": 0.004, "description": "SNP association P-value threshold of 0.005 gave significant over-representation of genes in pathways for \'Age at first tooth\' GWAS.)", "source_id": "chunk-c63d60b991cd7442f690dcc364af47dd", "order": 1, "source": "ALLELIC SCORE", "target": "PATHWAY"}, {"weight": 0.7, "description": "In TwinsUK, heritability estimation was performed using the ACE model, which includes additive genetic variance (A), shared environmental factors (C), and non-shared factors plus error (E).)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "HERITABILITY ESTIMATION", "target": "TWINSUK COHORT&MOLECULAR INTERACTION"}, {"weight": 0.8, "description": "SNP-based heritability estimation from mbQTL summary statistics was performed using linkage disequilibrium (LD) score regression on the TwinsUK cohort.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "HERITABILITY ESTIMATION", "target": "TWINSUK COHORT&SNP-BASED HERITABILITY"}, {"weight": 0.7, "description": "The NTR cohort provided replication for the SNP-based heritability estimation done on the TwinsUK cohort.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "HERITABILITY ESTIMATION", "target": "NETHERLANDS TWIN REGISTRY (NTR)&SNP-BASED HERITABILITY"}, {"weight": 6.0, "description": "Analysis", "source_id": "chunk-50e07db177a6a08799261934e8e71ac4", "order": 1, "source": "EFFECTIVE NUMBER OF STATISTICAL TESTS", "target": "LOCUS/"}, {"weight": 3.0, "description": "The Th17 response was found to be reduced in the MLNs of Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 recipients.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "PATHWAY", "target": "TH17 CELL PERCENTAGES AMONG CD4+TCR+ T CELLS"}, {"weight": 5.0, "description": "There was a difference in absolute numbers of Th17 cells present between Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 recipients.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "PATHWAY", "target": "ABSOLUTE NUMBERS OF TH17 CELLS ACROSS ORGANS"}, {"weight": 6.0, "description": "Rag1\u2212/\u2212 mice demonstrated a broad requirement for IL-6 for T cell expansion and development of colitis.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "PATHWAY", "target": "RAG1\u2212/\u2212 MICE"}, {"weight": 7.0, "description": "In contrast, IL-10R\u03b1MdelRag1\u2212/\u2212 mice showed dispensable IL-6 for the development of colitis.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "PATHWAY", "target": "IL-10R\u0391MDELRAG1\u2212/\u2212 MICE"}, {"weight": 8.0, "description": "The absence of IL-6R\u03b1 signaling in T cells induced impaired expansion and lesser disease effect compared to WT T cell transfers.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "PATHWAY", "target": "IL-6R\u0391\u2212/\u2212 T CELLS AND RAG1\u2212/\u2212 COMBINATION"}, {"weight": 4.0, "description": "The IL-10R\u03b1MdelRag1\u2212/\u2212 mice exhibit exacerbated colitis due to an elevated Th17 response and increased levels of pro-inflammatory cytokines, indicating that the loss of IL-10R\u03b1 function influences disease severity.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "IL-10R\u0391MDELRAG1\u2212/\u2212", "target": "INCREASED DISEASE SEVERITY"}, {"weight": 4.0, "description": "These cells showed a uniformly elevated surface expression of activation markers like CD40, CD80 and CD86.)", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "order": 1, "source": "IL-10R\u0391MDELRAG1\u2212/\u2212 LPM\u03a6S", "target": "INCREASED NUMBERS AND ACTIVATION"}, {"weight": 5.0, "description": "The cells had a heightened expression of CD40, CD80 and CD86 on their surface.)", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "order": 1, "source": "IL-10R\u0391MDELRAG1\u2212/\u2212 LPM\u03a6S", "target": "INCREASED ACTIVATION MARKERS"}, {"weight": 2.0, "description": "The presence of IL-10 on macrophages suppresses their pro-inflammatory cytokine production, such as IL-1\u03b2 and IL-6, which is critical in the regulation of Th17 cell response.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "IL-10", "target": "MACROPHAGE"}, {"weight": 2.0, "description": "IL-10 suppresses caspase-1 activation by Rag1\u2212/\u2212 M\u03c6s.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "IL-10", "target": "CASPASE-1"}, {"weight": 6.0, "description": "IL-1\u03b2 acts as a pro-inflammatory cytokine that is upregulated during colitis, contributing to inflammation by inducing other pro-inflammatory cytokines and cell recruitment.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "IL-1\u0392", "target": "PRO-INFLAMMATORY CYTOKINE"}, {"weight": 5.0, "description": "|>SENP1-aP2KO", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "IL-1\u0392", "target": "PROINFLAMMATORY CYTOKINES/PATS/"}, {"weight": 5.0, "description": "|>SENP1-aP2KO", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "TNF-\u0391", "target": "PROINFLAMMATORY CYTOKINES/PATS/"}, {"weight": 3.0, "description": "The gene expression of IFN-\u03b3 is influenced by genotypes at the PTPN2:rs478582 locus.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "IFN-\u0393", "target": "PTPN2:RS478582"}, {"weight": 5.0, "description": "Increased Th17 cell presence in inflamed colonic tissues correlates with the development and progression of colitis, highlighting their role in inflammatory bowel disease.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "TH17 RESPONSE", "target": "INFLAMED COLONS"}, {"weight": 6.0, "description": "The comparison with untransduced T cells helps to assess the enhanced function of CAR T cells expressing the 4-1BB domain and TAG72-specific components.)", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "order": 1, "source": "TRANSCRIPT", "target": "MOCK (UNTRANSDUCED) T CELLS"}, {"weight": 1.0, "description": "Three clusters of \u03b2-defensin genes were found to be associated with SNPs through the GRAIL analysis.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "TRANSCRIPT", "target": "\u0392-DEFENSIN GENES"}, {"weight": 1.0, "description": "The CCL18 gene was one among several chemokine genes identified by the GRAIL algorithm and linked to other specific transcripts.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "TRANSCRIPT", "target": "CCL18"}, {"weight": 1.0, "description": "The CXCL12 gene showed connections with \u03b2-defensin genes, TNFRSF13B, and CCL18 according to the analysis performed by GRAIL.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "TRANSCRIPT", "target": "CXCL12"}, {"weight": 1.0, "description": "The TNFRSF13B gene was among those identified as functionally connected via the GRAIL algorithm.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "TRANSCRIPT", "target": "TNFRSF13B"}, {"weight": 1.0, "description": "The clusters of \u03b2-defensin genes were found to be functionally related to other transcripts via the GRAIL algorithm.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "TRANSCRIPT", "target": "\u0392-DEFENSIN GENE CLUSTER"}, {"weight": 1.0, "description": "Chemokine genes such as CCL18 and CXCL12 were connected with \u03b2-defensin genes through the analysis.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "TRANSCRIPT", "target": "CHEMOKINE GENE"}, {"weight": 1.0, "description": "The TNFRSF13B transcript was found to interact functionally with \u03b2-defensin genes and CCL18 as per the GRAIL analysis results.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "TRANSCRIPT", "target": "TNFRSF13B GENE"}, {"weight": 7.0, "description": "The transcriptional regulation of these MS risk genes by EBNA2 influences their expression and potential for immune evasion.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "TRANSCRIPT", "target": "EBNA2"}, {"weight": 5.0, "description": "Fruit consumption showed a positive association with LP variant.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "NUTRIENT", "target": "RS4988235 AND FRUIT INTAKE"}, {"weight": 1.0, "description": "There was no association between MTHFR 677C>T and male infertility; however, this result remained unchanged when analyzing individuals with serum folate concentrations below the study population median.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "NUTRIENT", "target": "MTHFR 677C>T"}, {"weight": 2.0, "description": "Normal values for median nerve compound motor action potential were measured, indicating a physiological process within the patient\'s nervous system.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "PHYSIOLOGICAL PROCESS", "target": "MEDIAN NERVE COMPOUND MOTOR ACTION POTENTIAL"}, {"weight": 3.0, "description": "The conduction velocity of the median nerve was also assessed, demonstrating normal values in comparison to infants aged 1-2 years.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "PHYSIOLOGICAL PROCESS", "target": "MEDIAN NERVE CONDUCTION VELOCITY"}, {"weight": 4.0, "description": "A measurement of median nerve sensory action potential fell within expected ranges for a child aged 1-2 years old.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "PHYSIOLOGICAL PROCESS", "target": "MEDIAN NERVE SENSORY ACTION POTENTIAL"}, {"weight": 1.0, "description": "A measure of forced expiratory volume in one second to the forced vital capacity was taken for each individual.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "PHYSIOLOGICAL PROCESS", "target": "FEV1/FVC"}, {"weight": 3.0, "description": "Anti-IL-10R\u03b1 Ab markedly increases activated caspase-1 in Rag1\u2212/\u2212 M\u03c6s, indicating normal restraint by IL-10.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "CASPASE-1", "target": "ANTI-IL-10R ANTIBODY"}, {"weight": 4.0, "description": "LPS stimulation modestly increases activated caspase-1 in Rag1\u2212/\u2212 M\u03c6s.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "CASPASE-1", "target": "LPS"}, {"weight": 5.0, "description": "ATP also stimulates activation of caspase-1 by LPS-stimulated Rag1\u2212/\u2212 M\u03c6s.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "CASPASE-1", "target": "ATP"}, {"weight": 6.0, "description": "Pre-treatment with IL-10 abrogates induction of activated caspase-1 in Rag1\u2212/\u2212 M\u03c6s.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "CASPASE-1", "target": "IL-10 TREATMENT"}, {"weight": 7.0, "description": "Anti-IL-10R\u03b1 Ab markedly increases activated caspase-1, indicating normal restraint by IL-10 in Rag1\u2212/\u2212 M\u03c6s.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "CASPASE-1", "target": "ANTI-IL-10R\u0391 AB"}, {"weight": 8.0, "description": "Increased IL-1 and IL-6 from LPM\u03c6s can impact Th17 cell maturation in the presence of na\u00efve T cells with added TGF\u03b2.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "TH17 CELL MATURATION", "target": "LPM\u03a6S"}, {"weight": 9.0, "description": "Blocking IL-6 receptor substantially inhibits Th17 development in diseased mice, as shown by IL-6R\u2212/\u2212 T cell transfer results.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "TH17 CELL MATURATION", "target": "ANTI-IL-6R\u0391 AB"}, {"weight": 9.0, "description": "Blocking IL-6 receptor partially inhibits Th17 formation from Rag1\u2212/\u2212 LPM\u03c6s, but less so from IL-10R\u03b1MdelRag1\u2212/\u2212-derived LPM\u03c6s.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "\u0391IL-6R\u0391 BLOCKING ANTIBODY", "target": "TH17 FORMATION"}, {"weight": 10.0, "description": "Blocking IL-1 signaling leads to more substantial inhibition of Th17 development in IL-10R\u03b1Mdel Rag1\u2212/\u2212-derived LPM\u03c6s.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "IL-1 SIGNALING", "target": "TH17 CELL MATURATION/"}, {"weight": 10.0, "description": "Blockage of IL-1 receptor pathway leads to more substantial inhibition of Th17 cell development in diseased mice, particularly with IL-10R\u03b1MdelRag1\u2212/\u2212 LPM\u03c6s.)", "source_id": "chunk-b971aa450a9b220806bdc000eb5efaad", "order": 1, "source": "IL-1 SIGNALING", "target": "TH17 CELL FORMATION/"}, {"weight": 3.0, "description": "Though no significant difference in histopathology was identified (Fig. 1e, f).)", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "order": 1, "source": "EFFECT ON IL-10R\u0391MDELRAG1\u2212/\u2212 MICE", "target": "NEUTROPHIL DEPLETION"}, {"weight": 5.0, "description": "Treg transfers led to a >2-fold increase in Treg in the spleens of recipient mice and Treg percentages did not differ between IL-10R\u03b1MdelRag1\u2212/\u2212 and Rag1\u2212/\u2212 recipients.)", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "order": 1, "source": "DISEASE SUPPRESSION", "target": "TREG"}, {"weight": 6.0, "description": "No influence on disease magnitude when T cell IL-10 production was performed.)", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "order": 1, "source": "DISEASE MAGNITUDE", "target": "IL-10 PRODUCTION BY TRANSFERRED T CELLS"}, {"weight": 6.0, "description": "Significant increase in LPM\u03c6 numbers was observed in IL-10R\u03b1MdelRag1\u2212/\u2212 mice compared to Rag1\u2212/\u2212 controls.)", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "order": 1, "source": "DISEASE SEVERITY COMPARISON", "target": "IL-10R\u0391MDELRAG1\u2212/\u2212 MICE VS. RAG1\u2212/\u2212 CONTROLS"}, {"weight": 7.0, "description": "Transferred Treg were able to fully suppress disease development regardless of whether the transferred T cells produced IL-10 or not.)", "source_id": "chunk-380a8a6e0d08cd28068a146ae5e00ce4", "order": 1, "source": "DISEASE SUPPRESSION CAPABILITY", "target": "TREGS VS. T CELLS WITH/WITHOUT IL-10 PRODUCTION"}, {"weight": 8.0, "description": "Increased levels of IL-23 are identified alongside IL-1\u03b2 and IL-6, suggesting a significant role in intensifying the Th17 response during colitis.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "CYTOKINE", "target": "IL-23"}, {"weight": 9.0, "description": "The percent and absolute numbers of CD4+IL-17A+ T cells are significantly increased in IL-10R\u03b1Mdel Rag1\u2212/\u2212 compared to Rag1\u2212/\u2212 mice, indicating an enhanced Th17 response.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "CD4+IL-17A+ T CELLS", "target": "INCREASE"}, {"weight": 10.0, "description": "After induction of disease with CD4+CD25\u2212CD45RBhi ROR\u03b3T-GFP reporter T cells, there is an increased identification of GFP+ T cells in colons at weeks 4 and 8, suggesting the involvement of Th17 cells in the disease process.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "INCREASE", "target": "GFP+ T CELLS"}, {"weight": 11.0, "description": "Elevated production of Th17-promoting cytokines (IL-1\u03b2, IL-6, IL-23p19), and other pro-inflammatory markers like iNOS indicates a heightened Th17 response in colitis.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "INCREASED EXPRESSION", "target": "TH17-ASSOCIATED CYTOKINES"}, {"weight": 12.0, "description": "Increased expression of IL-1\u03b2, IL-6, IL-23p19, and iNOS specifically within macrophages suggests they are the primary source for increased Th17-associated cytokines and implicates a suppressive role played by IL-10 on these cells.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "M\u03a6S", "target": "SPECIFIC EXPRESSION"}, {"weight": 13.0, "description": "The transfer of ROR\u03b3T\u2212/\u2212 (Th17 cells deficient) T cells into Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 recipients prevents or attenuates colitis, indicating that Th17 cell involvement is essential for disease development.)", "source_id": "chunk-8fb20f873daf79a9b7d6e8d25c8889d6", "order": 1, "source": "PREVENT OR ATTENUATE DISEASE", "target": "ROR\u0393T\u2212/\u2212 T CELLS TRANSFER"}, {"weight": 4.0, "description": "IL-6R\u03b1 signaling supports but is not essential for Th17 formation specifically in the MLNs of both recipient lines.)", "source_id": "chunk-35d8a4ea86cb92d18ee1c857a318d2d8", "order": 1, "source": "RESPONSE", "target": "TH17 CELL DEVELOPMENT"}, {"weight": 2.0, "description": "Transfer of IL-6R\u03b1 deficient T cells shows virtually complete disease protection.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "DISEASE PROTECTION", "target": "IL-6R\u0391 TRANSFER"}, {"weight": 3.0, "description": "Transfers of IL-1R deficient T cells show diminished clinical measures compared to WT transfers in Rag1\u2212/\u2212 colitis models.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "IL-1R TRANSFER", "target": "REDUCED DISEASE SEVERITY"}, {"weight": 4.0, "description": "While IL-6 responsiveness affects disease severity in Rag1\u2212/\u2212, IL-1 response impacts both Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 models.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "DISTINCT DOSE RESPONSE", "target": "IL-6R\u0391 VS. IL-1R TRANSFER IMPACT"}, {"weight": 5.0, "description": "Transfers of IL-6R\u03b1 deficient T cells show reduced Th17 cell proportions compared to WT T cells transfers, specifically in colonic environment.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "REDUCED IN T CELL TRANSFER MODELS", "target": "TH17 CELL PROPORTION"}, {"weight": 6.0, "description": "In diseased IL-10R\u03b1MdelRag1\u2212/\u2212 mice, there\'s a higher amount of pro-IL-1\u03b2 compared to Rag1\u2212/\u2212 mice in colons.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "DIFFERENCES BETWEEN IL-10R\u0391MDELRAG1\u2212/\u2212 AND RAG1\u2212/\u2212 MICE", "target": "IL-1\u0392 PRODUCTION IMPACT"}, {"weight": 7.0, "description": "IL-10 reduces the levels of pro-IL-1\u03b2 and prevents mature IL-1\u03b2 formation, as shown in cultured Rag1\u2212/\u2212 macrophages.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "IL-10\'S EFFECT ON PRO-IL-1\u0392 LEVELS", "target": "REGULATION BY IL-10 IN M\u03a6S"}, {"weight": 8.0, "description": "In both Rag1\u2212/\u2212 and IL-10R\u03b1MdelRag1\u2212/\u2212 mice models, IL-6 plays a more critical role in colitis when compared to IL-10 suppression of LPM\u03c6s\' IL-1\u03b2 production.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "CYTOKINE INFLUENCE", "target": "IL-6 AND DISEASE SEVERITY"}, {"weight": 9.0, "description": "IL-10 inhibits the production and maturation of pro-inflammatory cytokines like IL-1\u03b2, reducing disease severity in colitis models.)", "source_id": "chunk-f2c864ef2d87119a8129e011b1c42e3c", "order": 1, "source": "ANTI-INFLAMMATORY ACTION", "target": "IL-10\'S ROLE"}, {"weight": 0.5, "description": "The TwinsUK cohort (1,176 samples) was used to estimate heritability using the ACE model and NTR cohort (312 MZ twins) was used for replication of MZ intraclass correlation coefficient (ICC).)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "TWINSUK", "target": "NETHERLANDS TWIN REGISTRY (NTR)"}, {"weight": 8.0, "description": "No significant hits were found in the meta-GWAS analysis for alpha diversity indices, indicating low genetic influence.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "ALPHA DIVERSITY", "target": "META-GWAS"}, {"weight": 1.0, "description": "The LCT and MCM6 genes exhibit inter-relationships, with MCM6 influencing regulation of LCT expression related to lactose metabolism and persistence.)", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "order": 1, "source": "LCT", "target": "MCM6"}, {"weight": 2.0, "description": "rs182549 is a proxy for the functional LCT variant.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "RS182549", "target": "FUNCTIONAL LCT VARIANT"}, {"weight": 9.0, "description": "The rs182549 SNP showed a significant impact on Bifidobacterium abundance in individuals homozygous for the low-function LCT haplotype.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "RS182549", "target": "BIFIDOBACTERIUM ABUNDANCE"}, {"weight": 1.0, "description": "An LCT locus associated with abundance of Bifidobacterium.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "BIFIDOBACTERIUM", "target": "LCT LOCUS"}, {"weight": 3.0, "description": "The COPSAC2010 cohort contributed to the heterogeneity in rs182549 association signal.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "COPSAC2010 COHORT", "target": "HETEROGENEITY"}, {"weight": 1.0, "description": "The tagging NC_000019.9:g.49218060C>T (rs35866622 non-secretor) allele was associated with a reduced abundance of the R. torques group and a decreased presence of the R. gnavus group.)", "source_id": "chunk-fed46d4ab6d22b3530c4c06a4b7def46", "order": 1, "source": "R. TORQUES GROUP", "target": "RS601338*RS35866622"}, {"weight": 1.0, "description": "SNP rs7574352 has been shown to be associated with ulcerative colitis, potentially linking gut inflammation markers chromogranin A and calprotectin.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "RS7574352", "target": "ULCERATIVE COLITIS"}, {"weight": 2.0, "description": "The genomic variant rs7574352 is linked to type 2 diabetes susceptibility through its proximity to the gene IRF1 involved in insulin resistance.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "RS7574352", "target": "TYPE 2 DIABETES"}, {"weight": 9.0, "description": "A nominally significant correlation between Bifidobacterium abundance and raw vegetable intake was observed before multiple testing correction.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "BIFIDOBACTERIUM ABUNDANCE", "target": "RAW VEGETABLE INTAKE"}, {"weight": 6.0, "description": "rs182549 variant predisposes individuals to lactose intolerance, negatively correlated with obesity and positively associated with T2DM.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "RS182549 (LCT)", "target": "LACTOSE INTOLERANCE"}, {"weight": 7.0, "description": "The SNP rs182549 was found to be significantly associated with multiple dietary and metabolic phenotypes including T2DM diagnosis.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "RS182549 (LCT)", "target": "T2DM DIAGNOSIS"}, {"weight": 8.0, "description": "Functional LCT variant rs4988235 is associated with glycemic variability, an indicator of insulin sensitivity.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "GLYCEMIC VARIABILITY", "target": "RS4988235 VARIANT"}, {"weight": 3.0, "description": "Gut microbial taxa were analyzed for potential causal links with autoimmune, cardiovascular, metabolic and psychiatric diseases using Mendelian Randomization.)", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "order": 1, "source": "GUT MICROBIAL TAXA", "target": "DISEASES"}, {"weight": 2.0, "description": "Host genetics influenced microbiome composition during the analysis of gut microbial taxa using MR.)", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "order": 1, "source": "HOST GENETICS", "target": "MICROBIOME COMPOSITION"}, {"weight": 3.0, "description": "Card9\u2212/\u2212 mouse microbiota showed decreased Actinobacteria phylum according to LEfSe analysis.)", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "order": 1, "source": "ACTINOBACTERIA", "target": "DIFFERENCES"}, {"weight": 0.6, "description": "The SNP-based heritability calculated from mbQTL summary statistics using linkage disequilibrium (LD) score regression showed two bacterial taxa, genus Ruminiclostridium 9 and family Peptostreptococcaeae, passing the significance threshold given the number of 211 taxa tested.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "LINKAGE DISEQUILIBRIUM (LD) SCORE REGRESSION", "target": "SNP-BASED HERITABILITY"}, {"weight": 1.0, "description": "The Netherlands Twin Registry cohort was used to replicate the MZ intraclass correlation coefficient (ICC) for heritability estimation.)", "source_id": "chunk-2c395e2fed68f9267c06e9fcee853aae", "order": 1, "source": "NETHERLANDS TWIN REGISTRY (NTR)&HERITABILITY ESTIMATION", "target": "REPLICATION"}, {"weight": 1.0, "description": "Gene-based PheWAS enrichment analysis indicated strong links between the LCT locus and metabolic traits, including BMI.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "LCT LOCUS", "target": "METABOLIC TRAITS"}, {"weight": 4.0, "description": "Age shows an interaction with rs182549, affecting Bifidobacterium abundance differently in different age groups.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "AGE AND GENOTYPE INTERACTION", "target": "AGE& RS182549"}, {"weight": 6.0, "description": "The LCT locus, associated with abundance of Bifidobacterium through mbQTL mapping.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "LCT LOCUS - BIFIDOBACTERIUM", "target": "MBQTL"}, {"weight": 7.0, "description": "mbBTL mapping identified loci associated with the presence or absence of specific bacterial taxa in the gut microbiota.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "MBBTL", "target": "PRESENCE/ABSENCE OF BACTERIAL TAXA"}, {"weight": 10.0, "description": "Cochran\'s Q test showed evidence of heterogeneity across cohorts related to rs182549 association with Bifidobacterium abundance.)", "source_id": "chunk-af413a33fae8d27bdb15cd18447dfad4", "order": 1, "source": "COCHRAN\u2019S Q", "target": "RS182549 HETEROGENEITY"}, {"weight": 3.0, "description": "SNP rs182549, associated with Bifidobacterium abundance, has been positively linked to T2DM diagnosis and a family history of the disease.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "LCT (BIFIDOBACTERIUM)", "target": "T2DM"}, {"weight": 4.0, "description": "|>HFD/", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "T2DM", "target": "PATS/"}, {"weight": 5.0, "description": "Hepatic fat accumulation contributes to the development and exacerbation of Type 2 Diabetes Mellitus.)", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "T2DM", "target": "HEPATIC STEATOSIS"}, {"weight": 5.0, "description": "The secretor allele at the FUT1/FUT2 locus is associated with fish intake and height in dietary patterns.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "FISH INTAKE HEIGHT", "target": "FUT1/FUT2 LOCUS"}, {"weight": 10.0, "description": "The secretor allele at the FUT1/FUT2 locus is negatively associated with risks of cholelithiasis.)", "source_id": "chunk-938d8bf2e56584de9dd556be5eb70964", "order": 1, "source": "CHOLELITHIASIS", "target": "RS35866622, FUT1/FUT2 LOCUS"}, {"weight": 3.0, "description": "The limited effect of genetic variants on microbial taxa variability was considered in sensitivity analyses during our causality estimation process.)", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "order": 1, "source": "LIMITED IMPACT OF GENETIC VARIANTS", "target": "MICROBIAL TAXA VARIABILITY"}, {"weight": 2.0, "description": "Causal relationships were only considered if supported by three or more Single Nucleotide Polymorphisms (SNPs).)", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "order": 1, "source": "CAUSAL RELATIONSHIPS IDENTIFIED", "target": "THREE OR MORE SNPS"}, {"weight": 3.0, "description": "Actinobacteria and the genus Bifidobacterium may have a protective effect against ulcerative colitis.)", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "order": 1, "source": "HIGHER ABUNDANCE OF ACTINOBACTERIA AND BIFIDOBACTERIUM", "target": "PROTECTIVE EFFECT ON ULCERATIVE COLITIS"}, {"weight": 2.0, "description": "The Oxalobacteraceae family\'s higher abundance may be protective against rheumatoid arthritis, according to our study results.)", "source_id": "chunk-159d4c797dae6eb825c74067af703228", "order": 1, "source": "HIGHER ABUNDANCE OF OXALOBACTERACEAE FAMILY", "target": "PROTECTIVE EFFECT ON RHEUMATOID ARTHRITIS"}, {"weight": 6.0, "description": "Poultry intake was associated with the presence of rs4988235 genotype.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "FOOD", "target": "RS4988235 AND POULTRY INTAKE"}, {"weight": 8.0, "description": "Wine consumption showed a negative relationship with rs4988235 genotype.)", "source_id": "chunk-770a24a1a31cf671a7681e7c98a7ee91", "order": 1, "source": "FOOD", "target": "RS4988235 AND WINE INTAKE"}, {"weight": 2.0, "description": "Influence of maternal dietary choline status on endogenous choline synthesis when carrying rs12325817 variant.)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "CHOLINE", "target": "MATERNAL PEMT ENZYME INTERACTION WITH CHOLINE INTAKE"}, {"weight": 4.0, "description": "How genetic differences influence how a mother\'s body produces choline internally during pregnancy, affecting dietary requirements.)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "RS12325817 VARIANT", "target": "INFLUENCE OF GENOMIC VARIANT ON ENDOGENOUS CHOLINE SYNTHESIS"}, {"weight": 1.0, "description": "The diagnosis of FASD was applied to 32.3% of infants in the choline group compared to 22.6% in the placebo group.)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "INFANTS", "target": "FASD DIAGNOSIS"}, {"weight": 4.0, "description": "The impact of different dietary standards for choline on the health and development of infants.)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "MATERNAL DIETARY CHOLINE STATUS", "target": "INSUFFICIENT VS. ADEQUATE MATERNAL CHOLINE INTAKE DURING PREGNANCY"}, {"weight": 2.0, "description": "How maternal dietary choline status affects outcomes for their infants, including growth, development, and disease risk.)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "INFANTS\' HEALTH OUTCOMES", "target": "MATERNAL CHOLINE STATUS IMPACT ON INFANT HEALTH"}, {"weight": 1.0, "description": "The relationship between choline intake during pregnancy and the prevalence of fetal alcohol spectrum disorders among infants.)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "FASD DIAGNOSIS", "target": "CHOLINE SUPPLEMENTATION AND FASD DIAGNOSIS"}, {"weight": 5.0, "description": "The study explores the synergistic effects of choline supplementation on enzyme function related to choline metabolism in pregnant mothers carrying a specific genetic variant.)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "MATERNAL PEMT ENZYME INTERACTION WITH CHOLINE INTAKE", "target": "CHOLINE SUPPLEMENTATION AND MATERNAL ENZYME INTERACTIONS"}, {"weight": 3.0, "description": "Differences in infants\' outcomes between those given choline supplements during their mothers\' pregnancies versus placebo.)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "CHOLINE SUPPLEMENTATION IN PREGNANCY", "target": "INFANTS\' NUTRITIONAL AND HEALTH OUTCOMES"}, {"weight": 3.0, "description": "The impact of different levels of choline consumption compared to other nutrients in the diet on pregnancy health and infant development.)", "source_id": "chunk-d594d57f0043cce2b3d29c80323b1a11", "order": 1, "source": "CHOLINE INTAKE VS. MATERNAL NUTRITIONAL STATUS", "target": "NUTRIENT INTAKE"}, {"weight": 1.0, "description": "The genetic variation at rs10886471 influences the frequency of different \'CA\' repeat alleles based on glucose levels.)", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "order": 1, "source": "RS10886471 SNP", "target": "ALLELE FREQUENCIES"}, {"weight": 2.0, "description": "The CA16 allele shows an increased risk associated with prediabetes and type 2 diabetes compared to other alleles.)", "source_id": "chunk-66f614368ca7f1bd578d9eb259083dd9", "order": 1, "source": "ALLELE (CA)16", "target": "RISK FOR PREDIABETES AND T2DM"}, {"weight": 1.0, "description": "The NFG group was designated as the control group in relation to comparing with case groups (IFG, IGT, T2DM).)", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "order": 1, "source": "CONTROL GROUP", "target": "NFG GROUP"}, {"weight": 2.0, "description": "The (CA)16 allele was used as a reference for comparison when assessing the association of multiple alleles with different blood glucose groups.)", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "order": 1, "source": "(CA)16 ALLELE", "target": "REFERENCE ALLELE"}, {"weight": 3.0, "description": "These groups were identified based on their association with genetic variants in relation to abnormal blood glucose levels.)", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "order": 1, "source": "CASE GROUPS", "target": "IFG, IGT, AND T2DM GROUPS"}, {"weight": 3.0, "description": "Specific genotypes are negatively correlated with the IFG group based on their association with abnormal blood glucose levels.)", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "order": 1, "source": "GENOMIC VARIANTS", "target": "IFG GROUP"}, {"weight": 4.0, "description": "Specific genotypes show a negative correlation with the IGT group, relating to genetic variants and abnormal blood glucose levels.)", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "order": 1, "source": "GENOMIC VARIANTS", "target": "IGT GROUP"}, {"weight": 5.0, "description": "Genetic variants are significantly associated with T2DM, as identified in statistical analyses.)", "source_id": "chunk-63f7d78d4c6afd6ff9eb7b59d8c046b9", "order": 1, "source": "GENOMIC VARIANTS", "target": "T2DM GROUP"}, {"weight": 0.12, "description": "There is an inverse relationship between serum selenium concentrations and lifestyle factors such as smoking and postmenopausal hormone use.)", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "order": 1, "source": "SERUM SELENIUM", "target": "LIFESTYLE FACTORS (INCLUDING SMOKING AND HORMONE USE)"}, {"weight": 0.02, "description": "SNP rs8178974 in GPX4 gene is linked to increased colorectal cancer risk.)", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "order": 1, "source": "COLORECTAL CANCER", "target": "GPX4 GENE POLYMORPHISM"}, {"weight": 0.595, "description": "An inverse association exists between serum selenium concentration and the risk of colorectal tumors.)", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "order": 1, "source": "COLORECTAL TUMOR", "target": "SERUM SELENIUM CONCENTRATION"}, {"weight": 1.0, "description": "Begg\u2019s test and Egger\u2019s test indicate no publication bias in the meta-analysis.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "PUBLICATION BIAS", "target": "STATISTICAL ANALYSIS"}, {"weight": 0.58, "description": "An increased protective effect against colorectal tumors exists for men compared to women, indicating a potential gender-specific differential response to selenium intake.)<SEP>Selenium\'s protective effect against colorectal tumors is stronger in men compared to women.)", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "order": 1, "source": "GENDER EFFECT", "target": "SELENIUM CONCENTRATION"}, {"weight": 0.29, "description": "SNP rs8178974 in GPX4 gene interacts with the risk of colorectal cancer, suggesting genetic predisposition.)", "source_id": "chunk-d311ac0182b37003c75c0005b4337fa6", "order": 1, "source": "GENETIC INTERACTION", "target": "GPX4 SNPS AND COLORECTAL CANCER"}, {"weight": 1.0, "description": "Card9 plays a role in modulating the response to DSS-induced colitis, impacting recovery mechanisms.)", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "order": 1, "source": "CARD9", "target": "DEXTRAN SULFATE SODIUM (DSS)"}, {"weight": 1.0, "description": "CARD9 has a role in shaping the bacterial microbiota and controlling fungal expansion during colitis.)", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "order": 1, "source": "CARD9", "target": "BACTERIAL MICROBIOTA"}, {"weight": 6.0, "description": "Variants in CARD9 are linked to reduced AHR activation by microbiota-derived metabolites extracted from fecal samples in individuals with IBD.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "CARD9", "target": "IBD"}, {"weight": 13.0, "description": "There is a connection between CARD9, gut microbiota function, and the ability of the microbiota to produce AHR agonists.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "CARD9", "target": "MICROBIOTA"}, {"weight": 2.0, "description": "Impaired tryptophan metabolism leading to defective AHR activation contributes to colitis susceptibility in Card9\u2212/\u2212 mice.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "COLITIS", "target": "CARD9\u2212/\u2212 MICE"}, {"weight": 1.0, "description": "IL-22 is required for the production of AHR ligands by the microbiota.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "IL-22", "target": "AHR"}, {"weight": 3.0, "description": "Alteration of the gut microbiota\'s function affects AHR activation through tryptophan metabolism and IL-22 pathway.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "GUT MICROBIOTA", "target": "AHR ACTIVATION"}, {"weight": 2.0, "description": "Decreases in Adlercreutzia were revealed between baseline Card9\u2212/\u2212 mouse microbiota compared to WT microbiota.)", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "order": 1, "source": "ADLERCREUTZIA", "target": "DIFFERENCES"}, {"weight": 4.0, "description": "Decreased levels of Lactobacillus reuteri were identified in the Card9\u2212/\u2212 mouse microbiota based on LEfSe results.)", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "order": 1, "source": "LACTOBACILLUS REUTERI", "target": "DECREASES"}, {"weight": 5.0, "description": "A negative correlation between bacterial biodiversity was observed between WT and Card9\u2212/\u2212 mice across various time points.)", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "order": 1, "source": "BACTERIAL BIODIVERSITY", "target": "CORRELATION"}, {"weight": 6.0, "description": "No notable differences were found in fungal biodiversity between baseline WT and Card9\u2212/\u2212 mouse microbiota at any time point.)", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "order": 1, "source": "FUNGAL BIODIVERSITY", "target": "NO DIFFERENCES"}, {"weight": 7.0, "description": "A negative correlation was noted between bacterial and fungal biodiversity in WT mice, implying an inter-kingdom relationship.)", "source_id": "chunk-e025ebd5580ff5f1a76bedbcaed2a0b6", "order": 1, "source": "INTER-KINGDOM RELATIONSHIP", "target": "CORRELATION"}, {"weight": 5.0, "description": "The ability of the microbiota to produce AHR ligands is associated with CARD9, IBD and their metabolic pathways.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "MICROBIOTA", "target": "AHR AGONISTS"}, {"weight": 8.0, "description": "The gut microbiota alters tryptophan metabolism, impacting the production of Kyn and subsequent AHR activation.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "MICROBIOTA", "target": "TRYPTOPHAN"}, {"weight": 4.0, "description": "Subjects with IBD use tryptophan to produce Kyn via IDO1 activity in their gut immune cells.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "KYN", "target": "TRYPTOPHAN"}, {"weight": 10.0, "description": "Gut immune cells utilize tryptophan via IDO1 to produce Kyn when it comes to AHR activation in individuals with IBD.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "IDO1", "target": "GUT IMMUNE CELLS"}, {"weight": 2.0, "description": "Alterations in the gut fungal microbiota are observed at both baseline and during colitis, with Card9\u2212/\u2212 mice showing a less resilient microbiota compared to WT mice.)", "source_id": "chunk-4d5e7ab61a7357d5106f4a8a355f40d0", "order": 1, "source": "FUNGAL MICROBIOTA", "target": "GUT HEALTH"}, {"weight": 8.0, "description": "There was a correlation between age and coffee consumption, with younger participants being more likely to consume black or sugared coffee.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "CORRELATION", "target": "AGE AND COFFEE CONSUMPTION"}, {"weight": 11.0, "description": "This environmental factor might correlate with certain dietary patterns that influence nutrient intake.)", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "order": 1, "source": "CORRELATION", "target": "ENVIRONMENTAL FACTOR&DIETARY PATTERN"}, {"weight": 7.0, "description": "Link between genetic mutation causing LPL deficiency and its clinical symptoms such as hypertriglyceridemia, xanthomas, etc.)", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "CORRELATION", "target": "AUTOSOMAL RECESSIVE LPL DEFICIENCY MUTATION AND CLINICAL MANIFESTATIONS"}, {"weight": 9.0, "description": "Impaired tryptophan metabolism in Card9\u2212/\u2212 mice affects AHR ligand production, thus impacting colitis susceptibility.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "CARD9\u2212/\u2212 MICE", "target": "AHR LIGAND PRODUCTION"}, {"weight": 11.0, "description": "Healthy subjects\' fecal samples induce higher levels of AHR activation compared to those from patients with IBD.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "AHR ACTIVATION", "target": "FECAL SAMPLES"}, {"weight": 14.0, "description": "The genetic variant rs10781499 within CARD9 impacts the interaction with the gut microbiota, affecting AHR activation in IBD patients.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "AHR ACTIVATION", "target": "RS10781499"}, {"weight": 12.0, "description": "The microbiota\'s tryptophan metabolites are required for the proper AHR ligation process in individuals without IBD.)", "source_id": "chunk-c791a4e35844d642ba0c69774072f7cd", "order": 1, "source": "AHR LIGATION", "target": "TRYPTOPHAN METABOLITES"}, {"weight": 1.0, "description": "Baseline \u03b22M concentration is positively associated with prevalent hypertension, as shown by the odds ratio.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "\u03922M", "target": "PREVALENT HYPERTENSION"}, {"weight": 2.0, "description": "A higher baseline \u03b22M concentration predicts a greater risk of new-onset hypertension over time.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "\u03922M", "target": "NEW-ONSET HYPERTENSION"}, {"weight": 5.0, "description": "The addition of \u03b22M to the clinical model improves its ability to discriminate between individuals who will develop new onset hypertension.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "\u03922M", "target": "DISCRIMINATION FOR NEW-ONSET HYPERTENSION"}, {"weight": 3.0, "description": "The clinical model adjusts for various confounding factors to assess the independent association between \u03b22M and hypertension.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "CLINICAL COVARIATES", "target": "ADJUSTMENT FOR CONFOUNDING FACTORS"}, {"weight": 10.0, "description": "Variability in allele frequencies between different ancestry groups does not affect the allelic effects on eGFR, suggesting a shared genetic basis for this measurement across populations.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "EGFR", "target": "ANCESTRY GROUPS"}, {"weight": 4.0, "description": "This variant at the PDILT-UMOD locus is crucial for eGFR measurements across diverse populations.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "EGFR", "target": "RS77924615"}, {"weight": 6.0, "description": "Associations at the DCDC5-MPPED2 locus suggest mechanisms for genetic influences on kidney function.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "EGFR", "target": "DCDC5-MPPED2"}, {"weight": 6.0, "description": "Decreased SH2B3 expression leads to higher \u03b22M levels, as indicated by the inverse variance weighted MR analysis.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "SH2B3 EXPRESSION", "target": "CAUSAL EFFECT ON \u03922M LEVELS"}, {"weight": 7.0, "description": "The rs3184504 genetic variant may influence \u03b22M levels through its impact on SH2B3 expression.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "RS3184504 POLYMORPHISM", "target": "MEDIATION THROUGH SH2B3 EXPRESSION"}, {"weight": 9.0, "description": "The rs3184504 polymorphism influences SH2B3 expression, as indicated by the association between it and SH2B3 gene function in the Mendelian randomization study.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "RS3184504 POLYMORPHISM", "target": "ASSOCIATION WITH SH2B3 EXPRESSION"}, {"weight": 4.0, "description": "In a longitudinal study, increased baseline \u03b22M concentration is associated with an elevated risk of new-onset hypertension.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "BASELINE \u03922M CONCENTRATION", "target": "LONGITUDINAL ASSOCIATION WITH NEW-ONSET HYPERTENSION"}, {"weight": 8.0, "description": "Deficiency in SH2B3 function is associated with a more severe hypertensive response and renal dysfunction.)", "source_id": "chunk-c29d575e156569afaa1ca3d6418a71b9", "order": 1, "source": "INCREASED HYPERTENSIVE RESPONSE", "target": "SH2B3\u2212/\u2212 MOUSE MODEL"}, {"weight": 2.0, "description": "There was an age difference between non-Hispanic White and Hispanic mothers based on supplement use.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "FOLIC ACID SUPPLEMENTS", "target": "MATERNAL AGE AT DELIVERY"}, {"weight": 9.0, "description": "CG genotype was associated with a 31% risk reduction of NTD-affected pregnancies compared to CC genotypes among mothers.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "SHMT1 SNP RS669340 CG GENOTYPE", "target": "RISK REDUCTION FOR NTDS IN MOTHERS"}, {"weight": 3.0, "description": "There was a disparity in formal education attainment between Hispanic and non-Hispanic White mothers.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "HISPANIC MOTHER EDUCATION LEVEL", "target": "NON-HISPANIC WHITE MOTHER EDUCATION LEVEL"}, {"weight": 4.0, "description": "The composite measure showed subtle differences but more consistency compared to single nutrient intake measures.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "DIFFERENCES ACROSS SUBGROUPS", "target": "FOLATE INTAKE VARIABLE COMPOSITE MEASURE"}, {"weight": 5.0, "description": "No evidence of association between DHFR gene variants and risk for NTD pregnancies in studied population.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "DHFR GENE VARIANT AND NTD-AFFECTED PREGNANCIES", "target": "GENETIC POLYMORPHISM ASSOCIATION"}, {"weight": 6.0, "description": "CT genotype was associated with increased risk compared to CC genotypes for infants in lower folate intake groups.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "INFANT RISK FACTOR", "target": "MTHFD1 SNPS RS2236224 AND RS2236225 CT GENOTYPE"}, {"weight": 7.0, "description": "AG genotype was associated with four-fold increase risk of NTDs when maternal folate intake was low compared to GG genotypes.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "MTHFD1 SNP RS11627387 AG/AA GENOTYPES AND LOW FOLATE INTAKE", "target": "NTD RISK FACTOR"}, {"weight": 8.0, "description": "Low folate intake was associated with a decreased risk of an NTD-affected pregnancy if mothers carried heterozygous MTHFR SNPs compared to common homozygous mothers.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "FOLATE INTAKE AND RISK OF NTDS", "target": "MTHFR SNPS RS1476413, RS1801131 AND RS1801133 HETEROZYGOUS VS COMMON HOMOZYGOUS MOTHERS"}, {"weight": 10.0, "description": "An interaction effect was observed with AG vs AA genotypes in infants depending on maternal folate intake.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "RISK FOR NTDS INTERACTION EFFECT", "target": "SHMT1 SNP RS12939757 AG/AA GENOTYPES AND LOW FOLATE INTAKE IN INFANTS"}, {"weight": 11.0, "description": "Gene-folate interaction was associated with reduced risk of NTDs among mothers on low folate intake compared to high intake group.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "RISK REDUCTION FOR NTDS INTERACTION EFFECT", "target": "TYMS SNP RS502396 OR RS699517 HETEROZYGOUS GENOTYPES AND LOW FOLATE INTAKE IN MOTHERS"}, {"weight": 1.0, "description": "An interaction effect was protective for infants with AG genotypes compared to AA genotypes when mothers were on low folate intake.)", "source_id": "chunk-9a35876dc1aaf7342f023cab4f5b9914", "order": 1, "source": "RISK FOR NTDS INTERACTION EFFECT IN INFANTS", "target": "TYMS SNPS RS2847153 AND LOW FOLATE INTAKE IN INFANTS, INTERACTION EFFECT FOR AG GENOTYPES"}, {"weight": 3.0, "description": "A significant association was found between rs10399931 and serum C-peptide concentration after a 2-hour mark, indicating its impact on insulin response.)", "source_id": "chunk-44a871da300abc9be525b468f502f5cb", "order": 1, "source": "RS10399931", "target": "SERUM C-PEPTIDE CONCENTRATION"}, {"weight": 1.0, "description": "Having the Y402H variant of CFH gene increases the odds for AMD.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "Y402H CFH RISK ALLELE"}, {"weight": 2.0, "description": "An additional copy of the A69S risk allele increases the odds of AMD relative to women with zero copies.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "A69S RISK ALLELE"}, {"weight": 3.0, "description": "The G allele for the rs641153 variant in CFB/C2 gene is associated with increased AMD risk.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "G ALLELE OF RS641153 (CFB/C2)"}, {"weight": 4.0, "description": "No main effect was observed within CAREDS for rs10033900, related to AMD risk.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "RS10033900"}, {"weight": 5.0, "description": "Similarly, no main effect was found for rs2230199 in relation to AMD.)", "source_id": "chunk-23a090455f50ab98c53e8b90941c6da5", "order": 1, "source": "AMD", "target": "RS2230199"}, {"weight": 0.0, "description": "There was no evidence for multiplicative interactions between lifestyle factors and genetic risks like the ARMS2 genotype on AMD development.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "AMD", "target": "LIFESTYLE"}, {"weight": 0.0, "description": "There was no evidence for joint effects between lifestyle and variants in the CFB/C2 gene on AMD risk.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "LIFESTYLE", "target": "CFB/C2"}, {"weight": 0.0, "description": "The main effects for SNPs in C3 were absent when examining interactions with lifestyle, indicating limited influence on AMD risk.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "LIFESTYLE", "target": "C3 RS2230199"}, {"weight": 0.0, "description": "Similar to the C3 SNP, there were no strong indications of interaction between lifestyle and this variant in CFI for AMD risk.)", "source_id": "chunk-a7d0bfa8b16262a1f313b7e3060969c6", "order": 1, "source": "LIFESTYLE", "target": "CFI RS10033900"}, {"weight": 2.0, "description": "Correction for multiple testing is used when several statistical tests are performed simultaneously, which helps to reduce type I errors.)<SEP>This refers to the application of statistical methods used when multiple hypotheses are tested simultaneously, adjusting p-values to control the family-wise error rate.)", "source_id": "chunk-e8f81fdd25a22582c6ce3d1e15f74af3", "order": 1, "source": "STATISTICAL ANALYSIS", "target": "CORRECTION FOR MULTIPLE TESTING"}, {"weight": 15.0, "description": "The mean square error of FEV1/FVC was assessed using LASSO regression with a set of 80 SNPs.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "STATISTICAL MEASURE", "target": "MEAN SQUARE ERROR"}, {"weight": 4.0, "description": "A total of 18 SNPs located in different genomic regions achieved suggestive significance.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "GENETIC POLYMORPHISMS", "target": "GWAS RESULTS"}, {"weight": 6.0, "description": "Participants who consumed \u22652 cups/day of black or sugared coffee had a lower risk of prediabetes and type 2 diabetes.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "COFFEE CONSUMPTION AND DIABETES RISK", "target": "PREVENTION EFFECT"}, {"weight": 10.0, "description": "Coffee preferences could indicate different dietary patterns or habits.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "COFFEE TYPE AND DIETARY PREFERENCE", "target": "PREFERENCE"}, {"weight": 11.0, "description": "Detailed examination was conducted on the baseline characteristics of the participants for various analyses.)", "source_id": "chunk-1f20dbdcc9de443dada1a2045dc7832b", "order": 1, "source": "CHARACTERISTICS ANALYSIS", "target": "STUDY POPULATION ANALYSIS"}, {"weight": 1.0, "description": "the adipocyte-specific deletion of SENP1)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "ADIPONECTIN, PDGFR\u0391 OR AP2/FABP4", "target": "SENP1 DEFICIENCY"}, {"weight": 2.0, "description": "developed a diabetic phenotype (Fig.)", "source_id": "chunk-a8f2b3b97a70df26efb35263714bf43c", "order": 1, "source": "PATS", "target": "T1DM"}, {"weight": 5.0, "description": "High Fat Diet leads to increased fat weight in PAT adipose depots of SENP1-aP2KO mice under normal chow diet.)", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "PATS", "target": "HFD"}, {"weight": 3.0, "description": "Immune cell-specific effector T-cell invasion in the pancreatic tissue attacks the islets and induces cell death.)", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "order": 1, "source": "T-CELL", "target": "AUTOIMMUNE RESPONSE"}, {"weight": 1.0, "description": "An autoantigen NRP-V7-positive CD8+ T cells, targeting a peptide from islet-specific glucose-6-phosphatase catalytic subunit-related protein, were detected in the pancreatic lymph nodes and the spleen of SENP1-aP2KO mice at the age of 12 weeks.)", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "order": 1, "source": "AUTOANTIGEN NRP-V7-POSITIVE CD8+ T CELLS", "target": "AUTOIMMUNE RESPONSE"}, {"weight": 6.0, "description": "|>Hepatic Steatosis/T2DM/", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "INSULIN DEFICIENCY", "target": "TYPE 1 DIABETES/"}, {"weight": 13.0, "description": "HFD conditions augment the diabetic phenotype in SENP1 deficient mice, affecting fat weight, cell size and number of adipocytes.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "DIABETIC PHENOTYPE", "target": "HIGH-FAT DIET (HFD)"}, {"weight": 8.0, "description": "|>SENP1-aP2KO", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "ADIPOSE TISSUE", "target": "HFD/"}, {"weight": 7.0, "description": "High levels of pro-inflammatory cytokines secreted from peri-pancreatic adipocytes contribute directly to the upregulation of CCL5 expression in pancreatic islets.)", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "order": 1, "source": "PROINFLAMMATORY CYTOKINES IL-6, TNF-\u0391 AND IL-I\u0392", "target": "EFFECT ON"}, {"weight": 6.0, "description": "CCL5 contributes to autoimmune responses by recruiting immune cells that damage pancreatic \u03b2 cells, leading to insulin deficiency and diabetes.)", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "order": 1, "source": "CCL5", "target": "AUTOIMMUNE RESPONSE"}, {"weight": 8.0, "description": "The expression of CCL5 mRNA significantly increases in islets due to secreted cytokines from peri-pancreatic adipocytes, specifically before the onset of diabetes in SENP1-deficient mice.)", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "order": 1, "source": "CCL5", "target": "UPREGULATION AND LOCALIZATION"}, {"weight": 1.0, "description": "IL-8 protein levels in serum were significantly higher in patients compared to controls.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "SERUM", "target": "IL-8 PROTEIN"}, {"weight": 4.0, "description": "|>Control Condition/", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "HFD", "target": "PATS/"}, {"weight": 5.0, "description": "|>T2DM", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "SENP1-AP2KO", "target": "DISEASE/T1DM/"}, {"weight": 3.0, "description": "|>Hyperglycaemia", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "FATTY LIVER DISEASE", "target": "INSULIN RESISTANCE/"}, {"weight": 5.0, "description": "|>Comorbidity", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "TYPE 1 DIABETES", "target": "GENETIC TRAIT/T2DM/"}, {"weight": 3.0, "description": "|>Insulin Deficiency", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "BETA CELL LOSS", "target": "TYPE 1 DIABETES/"}, {"weight": 8.0, "description": "Increased plasma levels of proinflammatory cytokines are significantly higher in T1DM patients compared to healthy controls.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "T1DM", "target": "PRO-INFLAMMATORY CYTOKINE LEVELS"}, {"weight": 8.0, "description": "We observed that Th1 (IFN-\u03b3+CD4+) and Th17 (IL-17+CD4+) effector T-cell subsets expanded concomitant with reduction of regulatory T-cell subset (CD4+CD25+Foxp3+) in the pancreatic lymph nodes of SENP1-aP2KO mice.)", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "order": 1, "source": "AUTOIMMUNE RESPONSE", "target": "TH1 (IFN-\u0393+CD4+) AND TH17 (IL-17+CD4+) EFFECTOR T-CELL SUBSETS"}, {"weight": 9.0, "description": "We also observed an increase in activated dendritic cells in the lymph node of pancreases.)", "source_id": "chunk-fc33d6399ec56e65d007269407f1953d", "order": 1, "source": "AUTOIMMUNE RESPONSE", "target": "ACTIVATED DENDRITIC CELLS"}, {"weight": 3.0, "description": "Proinflammatory cytokine overproduction serves as the main reason for immunologic tolerance break leading to diabetes.)", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "order": 1, "source": "CYTOKINES IL-6, TNF-\u0391 AND IL-I\u0392", "target": "EFFECT ON"}, {"weight": 4.0, "description": "Deletion of SENP1 gene strongly enhances the expression of pro-inflammatory cytokines in peri-pancreatic adipocytes compared to other adipose depots.)", "source_id": "chunk-0c1631ae2c195cb68139ea9d75a3b433", "order": 1, "source": "ENHANCEMENT OF", "target": "SENP1 DELETION"}, {"weight": 4.0, "description": "SENP1 siRNA increased phosphorylation of IKK in adipocytes.)", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "order": 1, "source": "REGULATE PHOSPHO-IKK", "target": "SENP1 SIRNA"}, {"weight": 6.0, "description": "The SENP1 knockdown cells showed significantly higher gene expressions of IL-6, TNF-\u03b1 and IL-1\u03b2 compared to Ctrl cells.)", "source_id": "chunk-f290cff8dac7b522b4f56d03e8df48ce", "order": 1, "source": "INCREASE PROINFLAMMATORY CYTOKINE EXPRESSION", "target": "SENP1 KNOCKDOWN CELLS"}, {"weight": 7.0, "description": "Age-dependent reduction in SENP1 expression correlates with the progression of spontaneous diabetes in NOD mice.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "HYPERGLYCAEMIA", "target": "SENP1 EXPRESSION"}, {"weight": 9.0, "description": "The Th1 dominant response due to IFN-\u03b3-induced insulitis is associated with diabetes progression in NOD mice.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "DIABETES PROGRESSION", "target": "TH1 DOMINANT RESPONSE"}, {"weight": 10.0, "description": "Serum from human T1DM patients and NOD mice induces disruption and cell death in pancreatic islets.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "PANCREATIC CELL DEATH", "target": "SERUM FROM T1DM PATIENTS"}, {"weight": 11.0, "description": "Prophylactic insulin therapy only partially ameliorates the development of diabetes phenotypes in SENP1-deficient mice.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "DIABETES PHENOTYPES", "target": "PROPHYLACTIC INSULIN THERAPY"}, {"weight": 12.0, "description": "Administration of an NF-\u03baB activation inhibitor reduces blood glucose levels and attenuates islet damages in SENP1-aP2KO mice.)", "source_id": "chunk-8382a49b61215482f9fc75e3a86cb987", "order": 1, "source": "BLOOD GLUCOSE", "target": "NF-\u039aB INHIBITORS"}, {"weight": 4.0, "description": "|>HFD/", "source_id": "chunk-174831b97ed170039b374ba254d4f506", "order": 1, "source": "PATS/", "target": "LIVER INSULIN RESISTANCE"}, {"weight": 2.0, "description": "The ApaI and BsmI variants were not significantly associated with ovarian cancer risk based on the meta-analysis.)", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "order": 1, "source": "APAI", "target": "OVARIAN CANCER RISK"}, {"weight": 1.5, "description": "Both FokI and ApaI variants were assessed in the same studies regarding their association with ovarian cancer, indicating potential genetic interaction.)", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "order": 1, "source": "APAI", "target": "FOKI"}, {"weight": 3.0, "description": "There was no significant association for the variant (rs1544410) and ovarian cancer risk as determined by both fixed-effect model and exclusion of heterogeneity-contributing subgroup.)", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "order": 1, "source": "BSMI", "target": "OVARIAN CANCER RISK"}, {"weight": 2.5, "description": "The meta-analysis restricted to Caucasian population yielded different results for BsmI compared to other ethnic groups, suggesting a possible influence of ethnicity on the variant\'s association with ovarian cancer.)", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "order": 1, "source": "BSMI", "target": "CAUCASIANS"}, {"weight": 1.0, "description": "The FokI variant was significantly associated with an increased risk of ovarian cancer.)", "source_id": "chunk-1c470c93632f3704d6f3566de7a1dd0c", "order": 1, "source": "FOKI", "target": "OVARIAN CANCER RISK"}, {"weight": 1.0, "description": "The genomic variant rs11568820 was evaluated as part of genetic studies related to Cdx-2 and its potential impact on ovarian cancer risk.)", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "order": 1, "source": "CDX-2", "target": "RS11568820"}, {"weight": 1.0, "description": "The genomic variant rs731236 was studied for its relation with the TaqI gene and the risk of developing ovarian cancer.)", "source_id": "chunk-dc5e8cc04a1b16b1e00f3a4120e004c4", "order": 1, "source": "TAQI", "target": "RS731236"}, {"weight": 3.0, "description": "The TT genotype of IL-1A -889C>T was found to be more frequent in HNP patients.)", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "order": 1, "source": "IL-1A -889C>T TT GENOTYPE", "target": "HNP PATIENT FREQUENCY"}, {"weight": 5.0, "description": "CT and T allele of TNF-A 238G>A were found more frequent among healthy controls compared to HNP cases.)", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "order": 1, "source": "HEALTHY CONTROL FREQUENCY", "target": "TNF-A 238G>A CT GENOTYPE AND T ALLELE"}, {"weight": 7.0, "description": "Genotypes of VDR TaqI were not associated with susceptibility to degenerative disc disease HNP.)", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "order": 1, "source": "NO ASSOCIATION WITH HNP", "target": "VDR TAQI TT VS. TC+CC, TC VS. TT+CC AND OR: 2.65; 95%CI: 0.60, 11.85"}, {"weight": 3.0, "description": "The genotypes models of IL-1A -889C>T were not associated with the risk for HNP.)", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "order": 1, "source": "IL-1A -889C>T GENOTYPES CC VS. CT+TT, CT VS. CC+TT AND TT VS. CT+CC", "target": "NO ASSOCIATION WITH RISK"}, {"weight": 5.0, "description": "Pooled estimates of TNF-A 238G>A genotypes were not associated with the risk for HNP.)", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "order": 1, "source": "NO ASSOCIATION WITH RISK", "target": "TNF-A 238G>A GENOTYPES GG VS. GA+AA, GA VS. GG+AA AND AA VS. GG+GA"}, {"weight": 7.0, "description": "None of the genotypes for VDR TaqI were associated with susceptibility to degenerative disc disease HNP.)", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "order": 1, "source": "NO ASSOCIATION WITH RISK", "target": "VDR TAQI GENOTYPES TT VS. TC+CC, TC VS. TT+CC AND OR: 1.01; 95%CI: 0.55, 1.85"}, {"weight": 3.0, "description": "There was no significant association between the C allele and T allele in relation to the risk of HNPs for IL-1A -889C>T.)", "source_id": "chunk-ba82b7eb40cf9eea561d839b20584f91", "order": 1, "source": "ASSOCIATION WITH RISK", "target": "C ALLELE OF IL-1A -889C>T"}, {"weight": 7.0, "description": "There is no evidence suggesting a causal effect of maternal B12 levels on offspring birthweight, which indicates that other factors might be more influential in this relationship.)", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "order": 1, "source": "CAUSAL RELATIONSHIP", "target": "MATERNAL B12 LEVELS&OFFSPRING BIRTHWEIGHT"}, {"weight": 8.0, "description": "There was a small causal effect detected between folate levels and B12 levels suggesting their interplay in maternal health.)", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "order": 1, "source": "CAUSAL RELATIONSHIP", "target": "FOLATE LEVELS&B12 LEVELS"}, {"weight": 10.0, "description": "The expression of this transcript could be related to the levels or regulation of nutrients like B12 and folate.)", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "order": 1, "source": "CAUSAL RELATIONSHIP", "target": "TRANSCRIPT&NUTRIENT LEVELS"}, {"weight": 12.0, "description": "Alterations in nutrient levels, such as B12 and folate, could potentially affect gestational duration or have been studied for this relationship.)", "source_id": "chunk-22216738ccbae0a6276bb4499018c434", "order": 1, "source": "CAUSAL RELATIONSHIP", "target": "NUTRIENT LEVELS&GESTATIONAL DURATION"}, {"weight": 99.5, "description": "The study assessed MTHFR C677T alongside Prothrombin G20210A in LV patients.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "MTHFR C677T", "target": "PROTHROMBIN G20210A"}, {"weight": 189.60000000000002, "description": "The study evaluated MTHFR C677T and Factor V G1691A together in LV patients, alongside other SNPs.)<SEP>The study evaluated both MTHFR C677T and Factor V G1691A together in LV patients.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "MTHFR C677T", "target": "FACTOR V G1691A"}, {"weight": 95.3, "description": "Both MTHFR C677T and PAI-1 A844G were assessed for their prevalence in LV patients.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "MTHFR C677T", "target": "PAI-1 A844G"}, {"weight": 189.2, "description": "Both PAI-1 -675 4G/5G and Factor V G1691A were assessed for their prevalence in LV patients.)<SEP>The assessment of PAI-1 -675 4G/5G was compared to Factor V G1691A.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "PAI-1 -675 4G/5G", "target": "FACTOR V G1691A"}, {"weight": 190.1, "description": "Both PAI-1 -675 4G/5G and Prothrombin G20210A were assessed in the same studies.)<SEP>The study evaluated the prevalence of PAI-1 -675 4G/5G and Prothrombin G20210A in LV patients.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "PAI-1 -675 4G/5G", "target": "PROTHROMBIN G20210A"}, {"weight": 93.1, "description": "Compared to other SNPs, PAI-1 A844G and Prothrombin G20210A were both assessed for their prevalence in LV patients.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "PROTHROMBIN G20210A", "target": "PAI-1 A844G"}, {"weight": 89.5, "description": "The study compared the carriers of Prothrombin G20210A and Factor V G1691A in LV patients.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "PROTHROMBIN G20210A", "target": "FACTOR V G1691A"}, {"weight": 91.7, "description": "The study compared the prevalence of MTHFR A1298C and Factor V G1691A in LV patients.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "FACTOR V G1691A", "target": "MTHFR A1298C"}, {"weight": 90.2, "description": "The study evaluated PAI-1 A844G and Factor V G1691A together for their prevalence in LV patients.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "FACTOR V G1691A", "target": "PAI-1 A844G"}, {"weight": 94.5, "description": "The study compared the carriers of MTHFR A1298C and PAI-1 A844G in LV patients.)", "source_id": "chunk-b025a32e74a29cbbd5a0ce3d051f3fdb", "order": 1, "source": "PAI-1 A844G", "target": "MTHFR A1298C"}, {"weight": 7.0, "description": "Alterations in the nitric oxide synthesis pathway are associated with cardiovascular disease.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "CARDIOVASCULAR DISEASE", "target": "NITRIC OXIDE SYNTHESIS"}, {"weight": 1.0, "description": "When focusing on the MTHFR 677C>T variant, individuals with serum folate levels below the study population median showed no association with male infertility.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "GENE-SPECIFIC NUTRIENT INTERACTION", "target": "MTHFR 677C>T VARIANT IN INDIVIDUALS WITH SERUM FOLATE BELOW MEDIAN"}, {"weight": 2.0, "description": "In individuals who did not take vitamin supplements, the PEMT M175V minor V allele and the TCblR rs173665 genetic variants showed significant associations with infertility status.)", "source_id": "chunk-c9badb9a0fb813f42383cc8afc57ad45", "order": 1, "source": "MOLECULAR INTERACTION AND GENETIC POLYMORPHISM", "target": "PEMT M175V MINOR V ALLELE AND TCBLR RS173665 VARIANT IN NON-USERS OF VITAMINS TO FERTILITY STATUS"}, {"weight": 1.0, "description": "Substitution of A by G results in an enzyme with less affinity for the substrate. The frequency of the A allele increased among TMD patients.)", "source_id": "chunk-382f24e5db6941e2e3db379fe499b223", "order": 1, "source": "RS1801394", "target": "MTRR"}, {"weight": 6.0, "description": "Labs that used folic acid calibrator)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "GROUP 2 LABORATORIES", "target": "S-FOL/WB-FOL"}, {"weight": 16.0, "description": "Differences were expected to be approximately \u00b110\u201330% for S-FOL and approximately \u00b120\u201340% for WB-FOL with harmonized conditions.)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "HARMONIZED CONDITION", "target": "DIFFERENCES BETWEEN DAY 1 AND DAY 2"}, {"weight": 2.0, "description": "A high Pearson correlation (r = 0.999) was observed between the results of the CDC MBA and CDC HPLC-MS/MS methods for serum folate measurement.)", "source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "order": 1, "source": "CDC HPLC-MS/MS", "target": "CDC MBA"}, {"weight": 3.0, "description": "All participating laboratories reported their results using different procedures, such as sample dilution techniques and incubation periods.)", "source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "order": 1, "source": "LABORATORY PROCEDURES", "target": "SERUM SAMPLES"}, {"weight": 5.0, "description": "The use of different sample dilutions (1:100 for serum and 1:140 for whole-blood hemolysate by CDC recommended, or lab-specific dilution) influenced the within-laboratory imprecision.)", "source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "order": 1, "source": "LABORATORY VARIABILITY", "target": "SAMPLE DILUTIONS"}, {"weight": 6.0, "description": "Labs incubated their assay plates at 37\u00b0C for 42\u201345 h; an exception was Lab #3 which used only an 18-h incubation period.)", "source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "order": 1, "source": "37\u00b0C INCUBATION", "target": "ASSAY STABILITY"}, {"weight": 7.0, "description": "Most laboratories read their plates at 590 nm and carefully controlled timing to ensure consistency across all plates.)", "source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "order": 1, "source": "DATA CONSISTENCY", "target": "READ PLATES AT 590 NM"}, {"weight": 8.0, "description": "A third-order polynomial regression was most commonly used for calibration curve fitting, but other methods were reported as well.)", "source_id": "chunk-dd3c1834035af8b4436c8058609b6fd8", "order": 1, "source": "CALIBRATION ACCURACY", "target": "POLYNOMIAL REGRESSION FIT"}, {"weight": 2.0, "description": "Comparison between two types of nutrients (5-methylTHF vs. folic acid) to evaluate their impact on laboratory test results.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "5-METHYLTHF/FOLIC ACID", "target": "NUTRIENT/NUTRIENT RELATIONSHIP"}, {"weight": 4.0, "description": "S-FOL\'s comparison against the CDC MBA target value as a reference physiological or health standard.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "CDC MBA TARGET CONCENTRATION/S-FOL", "target": "DISEASE/PHYSIOLOGICAL PROCESS-NUTRIENT RELATIONSHIP"}, {"weight": 6.0, "description": "Similar relationship for WB-FOL in comparison with the CDC MBA target.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "DISEASE/PHYSIOLOGICAL PROCESS-NUTRIENT RELATIONSHIP", "target": "CDC MBA TARGET CONCENTRATION/WB-FOL"}, {"weight": 8.0, "description": "Comparison of two transcript forms (S-FOL and WB-FOL) in laboratory conditions.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "S-FOL/WB-FOL", "target": "TRANSCRIPT/TRANSCRIPT RELATIONSHIP"}, {"weight": 8.0, "description": "1. When all labs used IH reagents, the mean difference to CDC MBA target decreased.)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "S-FOL/WB-FOL", "target": "USE OF IH REAGENTS"}, {"weight": 10.0, "description": "When CDC reagents were used by all laboratories, the mean \u00b1 SD difference to the CDC MBA target decreased.)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "S-FOL/WB-FOL", "target": "USE OF CDC REAGENTS"}, {"weight": 10.0, "description": "Relation between two different environmental contexts tested for their impact on laboratory results (CDC MO vs CDC CAL).)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "CDC MO CONDITION/CDC CAL CONDITION", "target": "ENVIRONMENTAL FACTOR/ENVIRONMENTAL FACTOR RELATIONSHIP"}, {"weight": 12.0, "description": "Comparison of outcomes influenced by distinct environmental conditions (IH MO and IH CAL). )", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "ENVIRONMENTAL FACTOR/ENVIRONMENTAL FACTOR RELATIONSHIP", "target": "IH MO CONDITION/IH CAL CONDITION"}, {"weight": 14.0, "description": "Role of microorganisms in influencing S-FOL results when compared against the CDC MBA target.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "MICROORGANISM/S-FOL", "target": "MOLECULAR INTERACTION-NUTRIENT RELATIONSHIP"}, {"weight": 16.0, "description": "Similar interaction for WB-FOL and microorganisms during comparisons.)", "source_id": "chunk-257a38a8ce6582c434f31b023b4a9704", "order": 1, "source": "MOLECULAR INTERACTION-NUTRIENT RELATIONSHIP", "target": "MICROORGANISM/WB-FOL"}, {"weight": 12.0, "description": "The limits indicate where 95% of lab results are expected compared with CDC MBA when using CDC reagents.)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "LIMITS OF AGREEMENT", "target": "RANGE WITHIN WHICH 95% OF RESULTS EXPECTED"}, {"weight": 14.0, "description": "All labs, except #6 and #9 showed good agreement under harmonized conditions.)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "HARMONIZED CONDITION AND SAME MICROORGANISM/CALIBRATOR", "target": "REASONABLE AGREEMENT"}, {"weight": 18.0, "description": "Lab #8 showed discrepancies in WB-FOL results between day 1 and day 2.)", "source_id": "chunk-7c25217961fba65d15f20beda8a58f1c", "order": 1, "source": "LAB #8 - WB-FOL", "target": "LACK OF AGREEMENT"}, {"weight": 2.0, "description": "The frequency ratios were found as 50.00%, 44.00% and 6.00% respectively.)", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "order": 1, "source": "RS1695", "target": "GENOTYPE AA, AG, AND GG FOR CONTROLS"}, {"weight": 4.0, "description": "The frequency ratios were found as 92.00%, 08.00% and 00.00% respectively for control group whereas these ratios for cases were calculated 88.00%, 11.00% and 01.00%)", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "order": 1, "source": "RS1138272", "target": "GENOTYPE CC, CT AND TT FOR CONTROLS AND CASES"}, {"weight": -456.0, "description": "The Ile105Val polymorphism decreases the amyloid propensity of the GSTP1 enzyme (dWALTZ was -228.00)<SEP>The Ile105Val polymorphism reduces the amyloid propensity of the GSTP1 enzyme (dWALTZ was -228.00)", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "order": 1, "source": "ILE105VAL", "target": "WALTZ OUTPUT"}, {"weight": 3.0, "description": "We evaluated effects of Ile105Val SNP on the GSTP1 function using SNPeffects online software.)", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "order": 1, "source": "GSTP1", "target": "FUNCTION"}, {"weight": 4.0, "description": "The WALTZ algorithm specifically predicts that Ile105Val polymorphism decreases the amyloid propensity of GSTP1 enzyme.)", "source_id": "chunk-ab5011584b0230efbf80a98b2a693a0e", "order": 1, "source": "GSTP1", "target": "FUNCTION AND ENZYME ACTIVITY"}, {"weight": 1.0, "description": "The genetic analysis identified a uniform association of p.T935M with haplogroup J on WD chromosomes.)(entity", "source_id": "chunk-fa2d834930246b534181c3d6d2c049e1", "order": 1, "source": "P.T935M", "target": "HAPLOGROUP J"}, {"weight": 1.0, "description": "Associations between specific alleles at different markers and various haplogroups underscore the interplay between genetic drift, natural selection, and demographic history in shaping genetic diversity within the WD population.)(entity", "source_id": "chunk-047f259db19cbb4de945706a1bd1b450", "order": 1, "source": "GENETIC MARKER", "target": "HAPLOGROUP ASSOCIATION"}, {"weight": 1.0, "description": "CYP2C9 genetic variants were grouped due to low frequency, impacting the prediction models.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "ALGORITHM", "target": "CYP2C9 STAR VARIANT GROUP"}, {"weight": 2.0, "description": "Algorithms considered the impact of low-frequency SNPs on predicting warfarin doses.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "ALGORITHM", "target": "RS1799853, RS1057910, RS9923231 SNPS"}, {"weight": 3.0, "description": "The CYP2C9 gene polymorphisms influence how algorithms predict drug metabolism.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "ALGORITHM", "target": "CYP2C9 ENZYME"}, {"weight": 5.0, "description": "The novel AA algorithm outperforms the IWPC pharmacogenomics algorithm by assigning a significantly greater percentage of high-dose patients correctly.)", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "order": 1, "source": "NOVEL AA ALGORITHM", "target": "IWPC PHARMACOGENOMICS ALGORITHM"}, {"weight": 16.0, "description": "The incorporation of our novel AA algorithm into practice provides a higher net numerical gain compared to the use of the IWPC clinical algorithm.)<SEP>The novel AA algorithm outperforms the IWPC clinical algorithm by assigning a much higher percentage of high-dose patients correctly.)", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "order": 1, "source": "NOVEL AA ALGORITHM", "target": "IWPC CLINICAL ALGORITHM"}, {"weight": 6.0, "description": "The novel AA algorithm provides higher precision in predicting doses.)", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "order": 1, "source": "NOVEL AA ALGORITHM", "target": "PRECISION"}, {"weight": 4.0, "description": "The novel AA algorithm performs well when evaluated with accurate measurements of admixture provided by AIMs.)", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "order": 1, "source": "NOVEL AA ALGORITHM", "target": "ACCURATE MEASUREMENT OF ADMIXTURE"}, {"weight": 8.0, "description": "The novel AA algorithm accounts for and incorporates population-specific variables, which results in better performance when dealing with the broad range of WAA among AAs.)", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "order": 1, "source": "NOVEL AA ALGORITHM", "target": "POPULATION-SPECIFIC VARIABLES"}, {"weight": 2.0, "description": "The IWPC pharmacogenomics algorithm exhibits inversely correlated performance with percent WAA in predicting doses for high-dose patients (\u226560 mg/week), unlike our novel AA algorithm.)", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "order": 1, "source": "IWPC PHARMACOGENOMICS ALGORITHM", "target": "INVERSELY CORRELATED PERFORMANCE"}, {"weight": 3.0, "description": "Compared to the IWPC pharmacogenomics algorithm, our novel AA algorithm has a higher correlation with percent WAA and predicted doses for high-dose patients (\u226560 mg/week), indicating superior performance.)", "source_id": "chunk-b2730f3087aee6d9cff6e4939b229323", "order": 1, "source": "IWPC CLINICAL ALGORITHM", "target": "LOWER CORRELATION"}, {"weight": 4.0, "description": "Studies previously observed low minor allele frequencies for these SNPs.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "PREVIOUS STUDIES", "target": "RS1799853, RS1057910, AND RS9923231 SNPS AND LOW FREQUENCIES"}, {"weight": 5.0, "description": "Comparisons of algorithm performance against IWPC methods to assess its validity in predicting warfarin doses.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "ALGORITHM ACCURACY", "target": "IWPC ALGORITHMS"}, {"weight": 6.0, "description": "Evaluation of how well each algorithm predicts dosing requirements within the target range.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "ALGORITHM PREDICTIVE POWER", "target": "NOVEL AA ALGORITHM, IWPC PHARMACOGENOMICS AND CLINICAL ALGORITHMS"}, {"weight": 8.0, "description": "Specific comparison of accuracy in predicting lower doses among the algorithms.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "NOVEL AA ALGORITHM, IWPC PHARMACOGENOMICS AND CLINICAL ALGORITHMS", "target": "ALGORITHM PERFORMANCE FOR LOW-DOSE PATIENTS"}, {"weight": 9.0, "description": "Similar focus on assessing accuracy with regards to mid-range warfarin dosages.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "NOVEL AA ALGORITHM, IWPC PHARMACOGENOMICS AND CLINICAL ALGORITHMS", "target": "ALGORITHM PERFORMANCE FOR INTERMEDIATE-DOSE GROUP"}, {"weight": 7.0, "description": "Analysis performed for different groups based on warfarin dosage needs.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "LOW-, INTERMEDIATE- AND HIGH-DOSE PATIENTS", "target": "PREDICTIVE ACCURACY IN DOSE SUBGROUPS"}, {"weight": 10.0, "description": "A detailed comparison of how well each algorithm performs in predicting higher doses.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "ALGORITHM PERFORMANCE FOR HIGH-DOSE PATIENTS", "target": "NOVEL AA ALGORITHM COMPARED AGAINST IWPC METHODS"}, {"weight": 1.0, "description": "The algorithms considered the impact of CYP2C9 genetic variants on predicting appropriate warfarin dosing.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "ALGORITHM, CYP2C9 GENE", "target": "CYP2C9 ENZYME POLYMORPHISMS AND WARFARIN DOSE PREDICTION"}, {"weight": 2.0, "description": "Some algorithms were more accurate when considering lower-frequency SNPs associated with reduced CYP2C9 activity.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "ALGORITHM, SNPS", "target": "SNP FREQUENCIES AND ALGORITHM PERFORMANCE"}, {"weight": 3.0, "description": "The performance of the AA novel algorithm was compared to that of IWPC algorithms in terms of dose prediction precision.)", "source_id": "chunk-f433e179cd5e32035a0303b9a14da3f1", "order": 1, "source": "ALGORITHM 1 VS. ALGORITHM 2 (IWPC, AA NOVEL)", "target": "ALGORITHM ACCURACY COMPARISON"}, {"weight": 1.0, "description": "Nitric oxide plays a critical role in regulating blood pressure and the health of cardiovascular systems.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "NITRIC OXIDE", "target": "CARDIOVASCULAR HEALTH"}, {"weight": 2.0, "description": "Statins can help manage nitric oxide levels to reduce cardiovascular risks.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "NITRIC OXIDE", "target": "DRUG THERAPY"}, {"weight": 7.0, "description": "Changes in environmental conditions could impact the expression levels of CHAT proteins.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "ENVIRONMENTAL FACTORS", "target": "CHAT EXPRESSION"}, {"weight": 6.0, "description": "Combined with a fat-restricted diet, drugs like gemfibrozil and omega-3 fatty acids help manage high triglyceride levels in individuals with LPL deficiency.)", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "order": 1, "source": "DRUG THERAPY", "target": "GEMFIBROZIL AND OMEGA-3 FATTY ACIDS"}, {"weight": 4.0, "description": "A balanced nutrient profile can influence lifestyle choices in an individual.)", "source_id": "chunk-f05ecc979eea31a1ab844c2c7de12432", "order": 1, "source": "LIFESTYLE FACTOR", "target": "NUTRIENT PROFILE"}, {"weight": 1.0, "description": "sensitivity study", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "WILD-TYPE (WT)", "target": "VALIDATION/"}, {"weight": 1.0, "description": "sequencing results", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "WILD-TYPE (WT)", "target": "DUPLEX-PCR/HRMA RESULTS/"}, {"weight": 1.0, "description": "metabolic pathway", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "MOLECULAR INTERACTION/TM DIFFERENCE/", "target": "RS777676129 AMPLICONS/RS114368325 AMPLICONS"}, {"weight": 1.0, "description": "amplicon differentiation", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "RS777676129 AMPLICONS/RS114368325 AMPLICONS", "target": "SPECIFIC GENOTYPING/"}, {"weight": 1.0, "description": "physiological process", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "DISEASE ASSOCIATION/", "target": "NORMALIZED AND TEMPERATURE-SHIFTED AND DIFFERENT PLOT/CYP24A1 GENOTYPES"}, {"weight": 1.0, "description": "confirmation", "source_id": "chunk-1327975b50e4a0aac1792cb595041ee5", "order": 1, "source": "DIAGNOSTIC VALIDATION/", "target": "DUPLEX-PCR/HRMA/10 DNA SAMPLES WITH THE SUSPICION OF IIH"}, {"weight": 9.0, "description": "Differences were observed in PCR and BsmAI digestion results between blood and skin fibroblast samples, which may indicate variations in mosaicism levels for the P1001L mutation.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "PCR AND BSMAI DIGESTION", "target": "BIOCHEMICAL PROCESS"}, {"weight": 12.0, "description": "A control test was used to ensure the accuracy and completeness of BsmAI digestion through Q-RT-PCR, providing quality assurance in molecular analysis results.)", "source_id": "chunk-c5f88943265b25ad302b08fd04a602d7", "order": 1, "source": "CONTROL FOR COMPLETENESS OF BSMAI DIGESTION", "target": "QUALITY ASSURANCE"}, {"weight": 2.0, "description": "Cells under Se-depleted conditions show a greater reduction in viability when challenged with tert-butylhydroperoxide compared to normal medium conditions.)", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "order": 1, "source": "SE-DEPLETED MEDIUM", "target": "EFFECT ON CELL VIABILITY"}, {"weight": 6.0, "description": "Hypothesis suggesting that differing expression levels of rat IDI under se-depleted conditions could influence selenoprotein production differently compared to normal conditions.)", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "order": 1, "source": "SE COMPETITION THEORY", "target": "THEORETICAL MODEL"}, {"weight": 4.0, "description": "Exposure to tert-butylhydroperoxide leads to an increase in ROS levels, particularly under Se-depleted conditions and differing 3\'UTR variants.)", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "order": 1, "source": "OXIDATIVE STRESS", "target": "TERT-BUTYLHYDROPEROXIDE EXPOSURE"}, {"weight": 3.0, "description": "Cell lines over-expressing either the IDI-T or IDI-C transcripts show lower viability after tert-butylhydroperoxide challenge compared to untransfected cells.)", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "order": 1, "source": "CELLULAR RESPONSE", "target": "TRANSFECTED CACO-2 CELLS EXPRESSING IDI-T AND IDI-C TRANSCRIPTS"}, {"weight": 5.0, "description": "The expression level of the deiodinase gene (both human IDI and rat IDI) affects selenoprotein H mRNA levels under varying selenium supply conditions.)", "source_id": "chunk-c7872499c2a84b16c506a135d8033bb7", "order": 1, "source": "DEIODINASE EXPRESSION LEVELS", "target": "INFLUENCE ON SELENOPROTEIN H MRNA"}, {"weight": 2.0, "description": "None of the mutations were detected in control subjects, suggesting a low prevalence.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "NONSENSE MUTATION", "target": "DISEASE PREVALENCE"}, {"weight": 8.0, "description": "A stable p/MHC complex formed after refolding nonamer and decamer peptides with HLA-A*01:01.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "THERMAL STABILITY", "target": "STABILITY MEASURE"}, {"weight": 11.0, "description": "Conserved residues across species like Drosophila and C. elegans indicate that certain functions are evolutionarily preserved.)<SEP>The conserved residues are present across species, including Drosophila and C. elegans, indicating evolutionary conservation.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "EVOLUTIONARY CONSERVATION", "target": "CONSERVED RESIDUES"}, {"weight": 9.0, "description": "Mutation-induced changes might lead to compromised thermal stability under different temperatures.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "PROTEIN STABILITY", "target": "THERMAL STABILITY OF MUTANTS"}, {"weight": 10.0, "description": "Comparisons were made between wild-type and mutated CHAT proteins to evaluate the impact on expression levels.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "GENETIC ENGINEERING", "target": "WILD-TYPE COMPARISON"}, {"weight": 4.0, "description": "The expression of eight mutants was significantly lower than wild-type; five expressed at less than 50% of wild-type levels.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "FUNCTIONAL IMPACT", "target": "MISSENSE MUTATIONS"}, {"weight": 5.0, "description": "Eight mutants expressed at significantly lower levels than wild-type; five of these (p.Trp421Ser, p.Ser498Pro, p.Thr553Asn, p.Ala557Thr, p.Ser572Trp) expressed at <50% of wild-type.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "CHAT EXPRESSION", "target": "PROTEIN LEVEL IMPACT"}, {"weight": 6.0, "description": "Ten mutations alter rate constants for ChAT activation; two compromise thermal stability of mutant proteins.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "KINETIC PARAMETER EFFECT", "target": "MUTATION RATE CONSTANTS"}, {"weight": 3.0, "description": "The absence of mutations in control subjects suggests a limited genetic predisposition towards these mutations.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "CONTROL SUBJECT MUTATIONS", "target": "MUTATION PREVALENCE"}, {"weight": 11.0, "description": "Mutations altered kinetic parameters involved in ChAT activation, affecting its catalytic efficiency.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "FUNCTIONAL PARAMETERS", "target": "KINETIC ACTIVATION CHANGES"}, {"weight": 12.0, "description": "The atomic structural model offered insights into how mutations affect the protein\'s 3D structure and function.)", "source_id": "chunk-2266e03ff38ff200390d9e9e50efbd74", "order": 1, "source": "ATOMIC STRUCTURAL MODEL", "target": "STRUCTURAL INSIGHTS"}, {"weight": 1.0, "description": "A CpG site in BCL2L11 was associated with TGSH levels.)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "order": 1, "source": "BCL2L11", "target": "TGSH"}, {"weight": 1.0, "description": "SNP rs726430 linked to the BCL2L11 gene via CpG site cg07547695.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "BCL2L11", "target": "RS726430"}, {"weight": 2.0, "description": "One CpG site in NDUFS7 was significantly linked to GSH levels.)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "order": 1, "source": "NDUFS7", "target": "GSH"}, {"weight": 3.0, "description": "The methylation at a CpG annotated to EIF3J was found associated with the GSSG/GSH ratio.)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "order": 1, "source": "EIF3J", "target": "GSSG/GSH RATIO"}, {"weight": 4.0, "description": "Methylation in the promoter region of ZNF761 showed the strongest association with GSSG/GSH ratio.)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "order": 1, "source": "ZNF761", "target": "GSSG/GSH RATIO"}, {"weight": 5.0, "description": "Psmg3 CpGs were associated with the GSSG/GSH ratio.)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "order": 1, "source": "PSMG3", "target": "GSSG/GSH RATIO"}, {"weight": 6.0, "description": "CpG methylation in SEMFA6A was related to oxLDL levels.)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "order": 1, "source": "SEMFA6A", "target": "OXLDL"}, {"weight": 7.0, "description": "Higher methylation at TAPBP CpGs was found associated with CD marker levels.)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "order": 1, "source": "TAPBP", "target": "CD"}, {"weight": 8.0, "description": "Methylation in the promoter region of RGL2 was found related to BCD-LDL levels.)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "order": 1, "source": "RGL2", "target": "BCD-LDL"}, {"weight": 9.0, "description": "Methylation at six CpGs linked with BCD-LDL was annotated to CKAP5.)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "order": 1, "source": "CKAP5", "target": "BCD-LDL"}, {"weight": 10.0, "description": "Methylation in FOSL2 CpG sites showed a strong association with BCD-LDL levels.)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "order": 1, "source": "FOSL2", "target": "BCD-LDL"}, {"weight": 11.0, "description": "CpGs annotated to FAM20C were found significant for BCD-LDL methylation.)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "order": 1, "source": "FAM20C", "target": "BCD-LDL"}, {"weight": 12.0, "description": "Methylation sites in FRMD6 showed association with BCD-LDL levels.)", "source_id": "chunk-491fec1ffc43ca0e67c1080702f038a7", "order": 1, "source": "FRMD6", "target": "BCD-LDL"}, {"weight": 2.0, "description": "SNP rs12782691 is associated with CEP55 and CpG site cg24371383.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "CEP55", "target": "RS12782691"}, {"weight": 3.0, "description": "SNP rs16861015 linked to CPB1 gene through CpG site cg04072156.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "CPB1", "target": "RS16861015"}, {"weight": 1.0, "description": "The analysis of SNPs in both PTPN2/22 shows significant differences between CD subjects and healthy controls, suggesting a potential role of these genetic variations in the development or manifestation of CD.)(entity", "source_id": "chunk-84ec948f22126cef7d93b1d7465b79db", "order": 1, "source": "CD", "target": "SNPS COMPARISON"}, {"weight": 1.0, "description": "Subjects with specific genotypes at PTPN2:rs478582 have a higher probability of experiencing MAPbacteremia in CD cases.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "CD", "target": "PTPN2:RS478582"}, {"weight": 3.0, "description": "Increased probability of having MAPbacteremia in CD cases compared to healthy controls.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "CD", "target": "MAPBACTEREMIA"}, {"weight": 5.0, "description": "Increased IFN-\u03b3 expression upon combined effects of specific SNPs, MAPbacteremia presence in CD subjects compared to non-infected controls.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "CD", "target": "IFN-\u0393 EXPRESSION"}, {"weight": 3.0, "description": "Celiac Disease (CD) subjects showed variations in TP53 expression compared to healthy individuals.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "CD", "target": "TP53"}, {"weight": 2.0, "description": "CpGs associated with oxidative stress were enriched in CpG island shores and functional regulatory regions based on the analysis using RegulomeDB.)", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "order": 1, "source": "CPGS ASSOCIATED WITH OXIDATIVE STRESS", "target": "FUNCTIONAL REGULATORY ELEMENTS"}, {"weight": 3.0, "description": "There was modest suggestive evidence of overlap between genotype-phenotype, genotype-CpG, and CpG-phenotype associations for certain CpGs.)", "source_id": "chunk-b05610f4a9a1033449b92655bfb67b28", "order": 1, "source": "GENOTYPE-PHENOTYPE, GENOTYPE-CPG, AND CPG-PHENOTYPE ASSOCIATIONS", "target": "OVERLAP BETWEEN ASSOCIATIONS"}, {"weight": 1.0, "description": "CpG sites are related when they share SNPs, such as cg07547695 associated with rs726430 and cg04072156 linked to rs16861015.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "CPG SITE", "target": "CPG SITE"}, {"weight": 2.0, "description": "Specific CpG sites influence DNA modification levels, thereby impacting gene expression patterns associated with lactase activity and lactose tolerance.)", "source_id": "chunk-ec104044b3aa65560be8d81dbb3940d6", "order": 1, "source": "CPG SITE", "target": "DNA MODIFICATION"}, {"weight": 4.0, "description": "Increased risk of diseases is associated with specific methylation levels at CpG sites; e.g., higher T2D risk related to hypomethylation in DVL1 and HIST1H4D.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "DISEASE INCIDENCE", "target": "METHYLATION LEVEL"}, {"weight": 5.0, "description": "No significant association was found between methylation levels and incident CVD events.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "METHYLATION LEVEL", "target": "CVD EVENT"}, {"weight": 7.0, "description": "Methylation at a specific CpG site (cg11842944 in ZNF341 gene) showed the strongest association to heart failure among CVD events.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "CVD EVENT", "target": "HEART FAILURE"}, {"weight": 6.0, "description": "The body of ZNF341 gene is associated with incident T2D, shown by its CpG site cg11842944 methylation level.)", "source_id": "chunk-8276370dea9f12d2b305c355f74c7f6e", "order": 1, "source": "CG11842944", "target": "ZNF341"}, {"weight": 5.0, "description": "The MT-ATP6 transcript is associated with Leigh syndrome due to the presence of a known disease-causing mutation 8993T>G in ATP synthase activity.)", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "order": 1, "source": "LEIGH SYNDROME", "target": "MT-ATP6"}, {"weight": 2.0, "description": "A validated RFLP assay confirmed the findings on DNA from blood and urine samples from the child.)", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "order": 1, "source": "RESTRICTION FRAGMENT LENGTH POLYMORPHISM (RFLP)", "target": "VALIDATION"}, {"weight": 3.0, "description": "Capillary sequencing methods were used to confirm the presence of mutations in both maternal urine and blood, though it may fail for heteroplasmy detection at low levels.)", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "order": 1, "source": "CAPILLARY SEQUENCING", "target": "CONFIRMATION"}, {"weight": 6.0, "description": "Capillary sequencing methods might fail to detect heteroplasmy below 10% even when mutations are present in blood samples.)", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "order": 1, "source": "CAPILLARY SEQUENCING", "target": "DETECTION LIMIT"}, {"weight": 4.0, "description": "Exome enrichment and sequencing on Illumina HiSeq 2000 was performed to generate comprehensive variant data for accurate genotyping.)", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "order": 1, "source": "EXOME SEQUENCING", "target": "DATA GENERATION"}, {"weight": 7.0, "description": "The NDUFV1 gene is responsible for functions related to electron transport chain complex 1 and energy metabolism within the mitochondria.)", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "order": 1, "source": "NDUFV1 GENE", "target": "ELECTRON TRANSPORT CHAIN"}, {"weight": 8.0, "description": "A missense variant in NDUFV1 gene resulting in a change at codon Glu246 to Lys potentially affecting the function of the protein.)", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "order": 1, "source": "C.736G>A, P.GLU246LYS", "target": "NDUFV1 PROTEIN"}, {"weight": 9.0, "description": "A second missense variant in NDUFV1 gene leading to a change at codon Ala117 to Thr, which may also be damaging based on in silico analysis.)", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "order": 1, "source": "C.349G>A, P.ALA117THR", "target": "NDUFV1 PROTEIN"}, {"weight": 10.0, "description": "The NDUFV1 protein functions as part of complex 1 of the electron transport chain and contains sites for NADH binding and iron-sulfur cluster binding essential for proper energy metabolism in mitochondria.)", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "order": 1, "source": "NDUFV1 PROTEIN", "target": "ELECTRON TRANSPORT CHAIN"}, {"weight": 11.0, "description": "Application of a recessive inheritance model helped filter variants to identify two missense mutations in NDUFV1 gene shared by affected siblings, suggesting potential disease causation.)", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "order": 1, "source": "RECESSIVE INHERITANCE MODEL", "target": "VARIANTS FILTERING"}, {"weight": 1.0, "description": "Identifying both infants as compound heterozygotes for the missense variants in NDUFV1 gene suggests they each carry one variant on opposite alleles, further supporting the pathogenicity of these mutations.)", "source_id": "chunk-b930cdd3706af881cfa771b341af097d", "order": 1, "source": "COMPOUND HETEROZYGOTES", "target": "GENOTYPE INTERPRETATION"}, {"weight": 4.0, "description": "Complex I functions by releasing electrons from NADH and is critical for energy metabolism.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "NADH", "target": "COMPLEX I OF ELECTRON TRANSPORT CHAIN"}, {"weight": 5.0, "description": "The Glu246Lys mutation in the FMN binding site could potentially affect COQ2 function, leading to potential disease.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "GLU246LYS MUTATION", "target": "COQ2"}, {"weight": 10.0, "description": "The depth of nuclear exome sequencing determines the comprehensiveness and accuracy of genetic data analysis.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "MEAN TARGET NUCLEAR EXOME COVERAGE", "target": "NUCLEAR DNA SEQUENCING DEPTH"}, {"weight": 2.0, "description": "Variants classified as category 1-3 are often associated with recessive inheritance patterns leading to genetic diseases.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "CATEGORY 1\u20133 VARIANTS", "target": "RECESSIVE INHERITANCE PATTERN"}, {"weight": 11.0, "description": "Genes carrying two homozygous variants or two or more variants are often linked to recessive genetic disorders.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "HOMOZYGOUS VARIANTS& TWO OR MORE VARIANTS", "target": "GENETIC DISORDERS (RECESSIVE INHERITANCE)"}, {"weight": 3.0, "description": "Variants in COQ2 have been linked to coenzyme Q10 deficiency, affecting the body\'s ability to produce essential compounds.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "COENZYME Q10 DEFICIENCY (OMIM #609825)", "target": "COQ2 GENE"}, {"weight": 12.0, "description": "The heteroplasmic level indicates the mixture of normal and mutant mtDNA, affecting its expression within cells.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "3754C>A MUTATION IN MT-ND1", "target": "HETEROPLASMY LEVEL"}, {"weight": 7.0, "description": "Poor dietary choices may contribute to conditions involving defects in the respiratory chain components of mitochondria.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "MITOCHONDRIAL COMPLEX I DEFICIENCY", "target": "DIETARY PATTERN (SPECIFICALLY DIET AFFECTING MTDNA)"}, {"weight": 9.0, "description": "Changes in dietary patterns can influence the number and type of mitochondrial variants found.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "ENVIRONMENTAL FACTORS (INCLUDING DIET)", "target": "MITOCHONDRIAL VARIANTS"}, {"weight": 8.0, "description": "Mutations in this gene lead to disorders like Leber optic atrophy, mitochondrial complex I deficiency, and MELAS.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "MUTATIONS IN MT-ND1", "target": "NADH DEHYDROGENASE SUBUNIT 1 (MT-ND1)"}, {"weight": 1.0, "description": "These genetic disorders are associated with mutations within the MT-ND1 gene that is a part of complex I.)", "source_id": "chunk-5d67fc9ddf960da009981b77d8d37f33", "order": 1, "source": "MUTATIONS IN MT-ND1", "target": "LEBER OPTIC ATROPHY, MITOCHONDRIAL COMPLEX I DEFICIENCY, MELAS"}, {"weight": 1.0, "description": "Both AUDIT score and number of drinking days are measures related to alcohol use.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "AUDIT SCORE", "target": "NUMBER OF DRINKING DAYS"}, {"weight": 2.0, "description": "The quantity consumed per drinking day affects the rising BrAC levels.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "DRINKS PER DRINKING DAY", "target": "BRAC"}, {"weight": 39.0, "description": "In the recent clinical trial, individuals with the CT genotype reported more drinks per drinking day than those with the TT genotype when they were treated with both placebo vs. olanzapine.)<SEP>Individuals with CT genotype reported more drinks per drinking day than those with TT genotype when treated with placebo versus olanzapine.)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "DRINKS PER DRINKING DAY", "target": "CT GENOTYPE AND TT GENOTYPE"}, {"weight": 3.0, "description": "Two genetic variants related in influencing alcohol metabolism, drug response.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "ALDH2 GENOTYPE", "target": "ADH1B GENOTYPE"}, {"weight": 3.0, "description": "The ALDH2 genotype is a statistical covariate affecting alcohol consumption in the models.)", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "order": 1, "source": "ALDH2 GENOTYPE", "target": "COVARIATE"}, {"weight": 2.0, "description": "In the context of the experiments, medication \u00d7 genotype interaction is not a significant factor for all outcomes as evidenced by analysis results.)", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "order": 1, "source": "MAIN EFFECT", "target": "STATISTICAL INTERACTION"}, {"weight": 4.0, "description": "Differences in side effects experienced when taking naltrexone compared to placebo.)", "source_id": "chunk-62d8719735e6c6eaa899d9ea64143544", "order": 1, "source": "MEDICATION (NALTREXONE/PLACEBO)", "target": "SIDE EFFECTS"}, {"weight": 4.0, "description": "The medication\'s impact on various outcomes was analyzed using chi-square tests for significance.)", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "order": 1, "source": "MEDICATION EFFECTS", "target": "STATISTICAL ANALYSIS OUTCOME"}, {"weight": 5.0, "description": "Latency to first drink is a key outcome measure in evaluating the timing of alcohol consumption patterns.)", "source_id": "chunk-27037cc5f35c77d6a57fa2f0fa1dc4a6", "order": 1, "source": "LATENCY TO DRINK", "target": "OUTCOME MEASURE"}, {"weight": 2.0, "description": "Lung function measures, including FEV1, FVC, and the FEV1/FVC ratio, were used as outcome measures.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "OUTCOME MEASURE", "target": "LUNG FUNCTION"}, {"weight": 4.0, "description": "In both sexes, daily alcohol consumption was positively associated with blood cortisol levels.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "DAILY ALCOHOL CONSUMPTION", "target": "BLOOD CORTISOL LEVELS"}, {"weight": 5.0, "description": "After adjusting for covariates like age, physical activity, smoking status, hypertension, diabetes, and education status, the association between daily alcohol consumption and blood cortisol levels remained similar in men but was higher in women.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "UNIVARIATE MODEL", "target": "MULTIVARIATE LINEAR REGRESSION MODELS"}, {"weight": 6.0, "description": "In MR analysis for men, one standard drink/day of genetically predicted alcohol consumption was associated with an increase of 0.36 in cortisol levels.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "GENETICALLY PREDICTED ALCOHOL CONSUMPTION", "target": "CORTISOL LEVEL (MR ANALYSIS)"}, {"weight": 7.0, "description": "In sensitivity analysis for men, one standard drink/day elevates cortisol by 0.27.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "SENSITIVITY ANALYSIS", "target": "CORTISOL LEVEL (GENETICALLY PREDICTED ALCOHOL CONSUMPTION)"}, {"weight": 1.0, "description": "An analysis that tests how sensitive results are to individual data point inclusion or exclusion, often used in meta-analyses.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "SENSITIVITY ANALYSIS", "target": "ROBUSTNESS CHECK"}, {"weight": 2.0, "description": "The genetically predicted alcohol consumption likely refers to the difference between the actual reported intake and a statistical estimate based on genetic factors.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "STANDARD DRINK/DAY", "target": "GENETIC PREDISPOSITION AND ALCOHOL CONSUMPTION"}, {"weight": 3.0, "description": "The BMI differed according to the rs671 polymorphism significantly among men.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "RS671 POLYMORPHISM", "target": "BODY MASS INDEX (BMI)"}, {"weight": 3.0, "description": "Both terms refer to different aspects of the same ALDH2 gene polymorphism, with \'ALDH2\' being more common in literature but \'rs671\' specifying a particular SNP.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "RS671 POLYMORPHISM", "target": "ALDH2 RS671"}, {"weight": 1.0, "description": "The pattern of cortisol levels varied based on the ALDH2 rs671 genotypes by sex.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "ALDH2 RS671 GENOTYPES", "target": "CORTISOL LEVEL"}, {"weight": 2.0, "description": "There was a significant difference in age distribution among different genotypes.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "AGE DISTRIBUTION", "target": "GENOTYPIC VARIANCE"}, {"weight": 1.0, "description": "Ex-drinkers could be included in sensitivity analyses considering their genotypic status.)", "source_id": "chunk-9e14f9bdee02839e4a6b12cff2020a5a", "order": 1, "source": "EX-DRINKERS", "target": "RS671 GENOTYPES"}, {"weight": 2.0, "description": "The genomic DNA analysis was performed on the subjects to identify genetic variants like p.Q240H.)", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "order": 1, "source": "SUBJECT", "target": "GENOMIC DNA ANALYSIS"}, {"weight": 1.0, "description": "Cardiovascular risk factors such as obesity and hypertension increase the risk of hyperuricemia (HUA).)", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "order": 1, "source": "CARDIOVASCULAR RISK FACTORS", "target": "HUA RISK"}, {"weight": 3.0, "description": "Higher sUA level may be associated with increased cardiovascular risk factors among rs2231142 T allele carriers.)", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "order": 1, "source": "CARDIOVASCULAR RISK FACTORS", "target": "SUA LEVEL"}, {"weight": 6.0, "description": "The rs2231142 T allele is associated with other cardiovascular risk factors besides sUA level.)", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "order": 1, "source": "CARDIOVASCULAR RISK FACTORS", "target": "RS2231142 T ALLELE"}, {"weight": 1.0, "description": "There was a strong association between the ABCG2 variant\'s T allele and sUA levels and HUA risk. )", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "SUA LEVEL", "target": "ABCG2 RS2231142"}, {"weight": 1.0, "description": "The polymorphism of ABCG2 rs2231142 was significantly associated with sUA levels (p=6.98e-8).)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "SUA LEVEL", "target": "ABCOR2 RS2231142 POLYMORPHISM"}, {"weight": 1.0, "description": "In males, the G/G genotypes of the ABCG2 rs2231142 polymorphism had significantly lower sUA levels compared to T/T genotypes (p=4.40e-6).)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "SUA LEVEL", "target": "G/G GENOTYPES"}, {"weight": 1.0, "description": "The G/G genotypes of the ABCG2 rs2231142 polymorphism had significantly lower sUA levels compared to T/T genotypes across all populations (p=6.98e-8).)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "SUA LEVEL", "target": "T/T GENOTYPES"}, {"weight": 4.0, "description": "The sUA level is influenced by the rs2231142 T allele in certain individuals.)", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "order": 1, "source": "SUA LEVEL", "target": "RS2231142 T ALLELE"}, {"weight": 5.0, "description": "Individuals with the rs2231142 T allele may have an increased risk for HUA.)", "source_id": "chunk-7091ed039203b69ad83d281b22c89579", "order": 1, "source": "RS2231142 T ALLELE", "target": "HUA RISK"}, {"weight": 1.0, "description": "The distribution of SLC22A12 variants showed no significant association with HUA risk.)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "HUA RISK", "target": "SLC22A12 RS893006"}, {"weight": 1.0, "description": "High blood pressure heightens the risk for hyperuricemia, which might be due in part to increased serum uric acid concentrations.)", "source_id": "chunk-e3ddbe8dd84aeaf2dbe30550c6c32d00", "order": 1, "source": "HUA RISK", "target": "HYPERTENSION - ENVIRONMENTAL FACTOR"}, {"weight": 0.0, "description": "Increased sUA levels showed a significant association with the Beta coefficient of each copy of risk allele (p=0.002).)", "source_id": "chunk-befd5ea20007b1d02d987897c798dadd", "order": 1, "source": "SLC2A9 RS3733591", "target": "URIC ACID LEVEL"}, {"weight": 5.0, "description": "The NFATC1 gene shows evidence of association with diabetic nephropathy and altered eGFR in both type 1 and type 2 diabetes.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "NFATC1", "target": "T2D, T1D"}, {"weight": 0.7, "description": "The single NFAT family member in Drosophila plays a potential role in ionic or osmotic regulation, similar to the role suggested for human NFATC1.)", "source_id": "chunk-45a965c3c3013c70d0fa0f1680283083", "order": 1, "source": "NFATC1", "target": "DROSOPHILA MELANOGASTER (LOCUS)"}, {"weight": 2.0, "description": "The variant at the ALMS1 gene may lead to ESRD when present, indicating genetic predisposition for end-stage renal disease.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "ESRD", "target": "RS7587577"}, {"weight": 3.0, "description": "The PDILT-UMOD locus, containing rs77924615, contributes significantly to CKD risk through multiple associated SNPs.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "PDILT-UMOD", "target": "CKD"}, {"weight": 6.0, "description": "Mice treated with Mock showed no significant improvement in overall survival.)", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "order": 1, "source": "SURVIVAL", "target": "MOCK TREATMENT"}, {"weight": 1.0, "description": "SNP rs62435145 associated with increased risk of CKD through its influence on kidney function.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "CKD", "target": "RS62435145"}, {"weight": 7.0, "description": "A collective of genes and specific genetic variations contribute to the complex interplay between CKD risk, eGFR measurements, and underlying pathologies.)", "source_id": "chunk-3edc72730d68b4c7fd6ba397aa087391", "order": 1, "source": "CKD, EGFR", "target": "UNCX, ALMS1, PDILT-UMOD, NFATC1"}, {"weight": 1.0, "description": "Foreign travel was found to be significantly associated with a higher prevalence of quinolone-resistant Campylobacter infections, as evidenced by statistical analysis.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "QUINOLONE RESISTANCE", "target": "TRAVEL"}, {"weight": 5.0, "description": "The prevalence of fluoroquinolone resistance was notably higher among travelers compared to domestically acquired cases, as indicated by the study\'s data analysis.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "QUINOLONE RESISTANCE", "target": "PREVALENCE COMPARISON"}, {"weight": 3.0, "description": "Swimming was recognized as another activity associated with an increased risk of acquiring quinolone-resistant C. jejuni infections.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "SWIMMING", "target": "RISK FACTOR"}, {"weight": 4.0, "description": "Fluoroquinolones were recommended as a primary treatment for C. jejuni infections until their ineffectiveness against resistant strains was recognized.)", "source_id": "chunk-6264e17a8290bc0a9e24c68c21c1f15c", "order": 1, "source": "FLUOROQUINOLONES", "target": "TREATMENT APPROACH"}, {"weight": 1.0, "description": "Infections treated with fluoroquinolones before specimen collection were excluded, indicating a distinction between domestically acquired and travel-related infections.)", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "order": 1, "source": "QUINOLONE-RESISTANT CASES", "target": "DOMESTICALLY ACQUIRED"}, {"weight": 5.0, "description": "The study compared antimicrobial drug information from patients with domestically acquired C. jejuni infections to those without travel-related infections.)", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "order": 1, "source": "AMOXICILLIN-CLAVULANATE, AZITHROMYCIN, DOXYCYCLINE, CLARITHROMYCIN, LEVOFLOXACIN, COTRIMOXAZOLE, SULFAMETHOXAZOLE/TRIMETHOPRIM, TETRACYCLINE", "target": "DOMESTICALLY ACQUIRED"}, {"weight": 2.0, "description": "The majority of human isolates came from Danish-bred food animals, suggesting a connection to foodborne transmission.)", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "order": 1, "source": "C. JEJUNI", "target": "FOODBORNE"}, {"weight": 3.0, "description": "Of the retail food products compared with broiler chickens, most were from domestic sources, which was different than imported isolates.)", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "order": 1, "source": "RETAIL FOOD PRODUCTS", "target": "ORIGIN"}, {"weight": 7.0, "description": "The study compared isolates from both retail food products and broiler chickens with those from human patients to assess similarities in serotypes and RiboGroups (subtypes).)", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "order": 1, "source": "RETAIL FOOD PRODUCTS", "target": "BROILER CHICKENS"}, {"weight": 4.0, "description": "A comparison of isolates from broiler chicken fecal samples and human isolates showed differences in resistance patterns, indicating a potential source for domestically acquired infections.)", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "order": 1, "source": "BROILER CHICKENS", "target": "DOMESTICALLY ACQUIRED"}, {"weight": 1.0, "description": "Several of the isolates identified as quinolone-resistant or -sensitive matched those found in human patients, indicating a potential shared source.)", "source_id": "chunk-0d155c005072e55d94179ce7952bf5a9", "order": 1, "source": "C. JEJUNI ISOLATES FROM RETAIL FOOD PRODUCTS AND BROILER CHICKENS", "target": "SIMILARITY OR COHORT"}, {"weight": 1.001, "description": "Higher linguistic acculturation scores are positively related to higher mean social disinhibition scores.)<SEP>Increased linguistic acculturation positively correlates with social disinhibition scores.)", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "order": 1, "source": "SOCIAL DISINHIBITION", "target": "LINGUISTIC ACCULTURATION"}, {"weight": 1.001, "description": "The total anxiety score (TAS) is significantly and positively associated with social disinhibition.)<SEP>Total Anxiety Scale (TAS) score is significantly correlated to higher social disinhibition scores.)", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "order": 1, "source": "SOCIAL DISINHIBITION", "target": "TAS"}, {"weight": 1.001, "description": "Depression Anxiety Scale (DAS) score shows a positive correlation with the social disinhibition measure.)<SEP>Depression anxiety score (DAS) is significantly related to increased risk of social disinhibition.)", "source_id": "chunk-d4e04dd6cd6ea28f04fd72d964574ec1", "order": 1, "source": "SOCIAL DISINHIBITION", "target": "DAS"}, {"weight": 4.0, "description": "Variants of CNR1 receptors may influence alcohol seeking behavior and neurocircuitry.)", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "CNR1", "target": "ALCOHOL SEEKING BEHAVIOR"}, {"weight": 5.0, "description": "The genetic variant rs2023239 is linked to CB1 receptor levels.)", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "RS2023239", "target": "CB1 RECEPTOR LEVEL"}, {"weight": 6.0, "description": "The rs2023239 variant has been associated with altered brain activation patterns during alcohol exposure.)", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "RS2023239", "target": "BRAIN ACTIVATION"}, {"weight": 16.0, "description": "In a recent clinical trial, the CT genotype showed greater cue-elicited craving when treated with placebo compared to olanzapine.)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "RS2023239", "target": "MEDICATION \u00d7 GENOTYPE \u00d7 CUE INTERACTION ON CRAVING"}, {"weight": 5.0, "description": "The analyses revealed a genotype-drink interaction when individuals were treated with olanzapine.)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "MEDICATION: OLANZAPINE", "target": "GENOTYPE \u00d7 DRINK INTERACTION"}, {"weight": 8.0, "description": "The CT group had a significant correlation between alcohol use disorder scores and certain neural activations.)", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "ALCOHOL USE DISORDER SCORE", "target": "CT GROUP"}, {"weight": 1.0, "description": "14)(entity", "source_id": "chunk-27c6611dceafae880176ce8a7e7fc07e", "order": 1, "source": "CT GROUP", "target": "REWARD RESPONSE<|THE CT GROUP DEMONSTRATED A SIGNIFICANT INCREASE IN REWARD RESPONSE COMPARED TO THE TT GROUP AFTER CONSUMING ALCOHOL.)"}, {"weight": 4.0, "description": "Individuals with the CT genotype reported greater reward from alcoholic beverages.)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "GENOTYPE CT", "target": "REWARDING EFFECTS"}, {"weight": 6.0, "description": "In an earlier study, individuals with genotypes CT or CC showed this interaction when receiving active control medication (cyproheptadine).)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "GENOTYPE CT AND CC", "target": "MEDICATION \u00d7 GENOTYPE \u00d7 DRINK INTERACTION"}, {"weight": 8.0, "description": "There might be an interaction between CNR1 variation and the effect of olanzapine on alcohol\'s rewarding impact.)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "CNR1 VARIATION AND OLANZAPINE", "target": "INTERACTION ON REWARDING EFFECTS OF ALCOHOL"}, {"weight": 7.0, "description": "There were no significant genotype x medication interactions on reward from alcohol beverages when cyproheptadine was compared to olanzapine.)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "GENOTYPE \u00d7 MEDICATION INTERACTIONS", "target": "MEDICATION: OLANZAPINE AND CYPROHEPTADINE HYDROCHLORIDE"}, {"weight": 9.0, "description": "In a recent study, there was a significant medication-genotype-cue interaction for craving when individuals were given different medications (cyproheptadine vs. placebo) compared to olanzapine.)", "source_id": "chunk-7542fe4e6d1c727325a6be02a71f8e59", "order": 1, "source": "MEDICATION: OLANZAPINE AND CYPROHEPTADINE HYDROCHLORIDE", "target": "MEDICATION \u00d7 GENOTYPE \u00d7 CUE INTERACTION"}, {"weight": 2.0, "description": "Linking genetic disorder to symptoms such as hypertriglyceridemia, xanthomas, abdominal pain, etc.)", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "LPL DEFICIENCY", "target": "CLINICAL FEATURES"}, {"weight": 5.0, "description": "The use of DGAT1 inhibitors to target triglyceride synthesis is an investigational treatment approach for LPL deficiency.)", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "order": 1, "source": "LPL DEFICIENCY", "target": "DGAT1 INHIBITORS AND THERAPY"}, {"weight": 3.0, "description": "Direct link between lifestyle change and managing the disease.)", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "DIETARY FAT RESTRICTION", "target": "TREATMENT FOR LPL DEFICIENCY"}, {"weight": 3.0, "description": "Long-term dietary fat restriction is a cornerstone of treatment for LPL deficiency, helping manage hypertriglyceridemia.)", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "order": 1, "source": "DIETARY FAT RESTRICTION", "target": "TREATMENT"}, {"weight": 4.0, "description": "Formula choice based on understanding of genetic factors in treatment)", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "ENFAPORT LIPIL FORMULA", "target": "LPL GENE MUTATION"}, {"weight": 6.0, "description": "Role played by GPIHBP1 in transporting the enzyme to capillary lumen.)", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "GPIHBP1", "target": "MOLECULAR INTERACTION WITH LPL"}, {"weight": 5.0, "description": "Understanding spectrum of mutations and expanding known cases.)", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "AUTOSOMAL RECESSIVE LPL DEFICIENCY MUTATION", "target": "PATHOGENIC MUTATIONS IN LPL GENE"}, {"weight": 8.0, "description": "Association between the specific fat content composition in Enfaport Lipil formula and improvement in lipid profiles as a treatment for LPL deficiency.)", "source_id": "chunk-bbdb97da6903bfbdae42d4c3cb9c9c63", "order": 1, "source": "FORMULA COMPOSITION (ENFAPORT LIPIL) AND TREATMENT EFFICACY", "target": "OUTCOME RELATIONSHIP"}, {"weight": 4.0, "description": "rhIGF-1 is a medical intervention for growth disorders associated with GHI phenotype.)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "TREATMENT", "target": "RECOMBINANT HUMAN INSULIN-LIKE GROWTH FACTOR 1 (RHIGF-1)"}, {"weight": 4.0, "description": "MCT-based formulas are used as fat supplements because they\'re absorbed directly into the portal vein, aiding in fat management.)", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "order": 1, "source": "FAT SUPPLEMENT", "target": "MCT-BASED FORMULA"}, {"weight": 7.0, "description": "An approved gene therapy procedure for managing hypertriglyceridemia involves introducing a gain-of-function variant of the LPL gene.)", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "order": 1, "source": "GENE THERAPY", "target": "INTRAMUSCULAR INTRODUCTION OF GAIN-OF-FUNCTION VARIANT"}, {"weight": 8.0, "description": "Orlistat is used as an over-the-counter weight loss aid, which may also help manage hypertriglyceridemia by blocking fat absorption.)", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "order": 1, "source": "DRUG MANAGEMENT", "target": "ORLISTAT AND OVER-THE-COUNTER USE"}, {"weight": 9.0, "description": "Elevated triglyceride levels can lead to abdominal pain and hyperviscosity, which may cause gastrointestinal issues.)", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "order": 1, "source": "ABDOMINAL PAIN AND HYPERVISCOSITY", "target": "SYMPTOM AND COMPLICATION RELATIONSHIP"}, {"weight": 10.0, "description": "An elevated risk of pancreatitis exists in individuals with high triglyceride levels due to factors like fat metabolism dysfunction.)", "source_id": "chunk-1e1db26cd046378fc4ac9fd4c4d35c86", "order": 1, "source": "DISEASE AND CAUSE RELATIONSHIP", "target": "PANCREATITIS AND HIGH TRIGLYCERIDES"}, {"weight": 1.0, "description": "Genetic variant rs2832407 moderates the effect of topiramate on efficacy for EA patients.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "TOPIRAMATE", "target": "RS2832407"}, {"weight": 2.0, "description": "Number Needed to Treat (NNT) quantifies how many patients need treatment with Topiramate to prevent one additional headache daily diary score.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "TOPIRAMATE", "target": "NNT"}, {"weight": 4.0, "description": "For rs2832407*C-homozygotes, Topiramate significantly reduces the risk of HDDs compared to placebo group.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "TOPIRAMATE", "target": "RS2832407*C-HOMOZYGOTES"}, {"weight": 5.0, "description": "There is a small difference in the likelihood of no HDDs between topiramate and placebo groups for A-allele carriers.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "TOPIRAMATE", "target": "A-ALLELE CARRIERS"}, {"weight": 1.0, "description": "Topiramate is effective in reducing HDDs based on the study results.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "TOPIRAMATE", "target": "EFFICACY"}, {"weight": 3.0, "description": "Number Needed to Treat (NNT) for the placebo group, which experiences a different response rate compared to topiramate.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "PLACEBO", "target": "NNT"}, {"weight": 2.0, "description": "The placebo group experiences adverse events that differ in frequency and severity from Topiramate group.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "PLACEBO", "target": "ADVERSE EVENTS"}, {"weight": 7.0, "description": "A high majority of EA patients experience mild adverse events, which is not statistically different between treatment groups.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "MILD ADVERSE EVENT", "target": "92% OF EA PATIENTS"}, {"weight": 10.0, "description": "The adjusted number needed to treat considering moderate adverse events shows a different efficacy between topiramate and placebo.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "NNT-AEMOD+ (ADJUSTED NNT FOR MODERATE EVENTS)", "target": "RS2832407*C-HOMOZYGOTES*TOPIRAMATE VS. PLACEBO"}, {"weight": 11.0, "description": "The adjusted number needed to treat considering severe adverse events highlights a distinct difference in efficacy between topiramate and placebo for rs2832407*A-allele carriers.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "NNT-AESEV+", "target": "RS2832407*A-ALLELE CARRIERS*TOPIRAMATE VS. PLACEBO"}, {"weight": 4.0, "description": "The adjusted number needed to treat considering severe adverse events is higher for rs2832407*A-allele carriers on topiramate compared to placebo.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "NNT-AESEV+", "target": "A-ALLELE CARRIERS"}, {"weight": 3.0, "description": "For rs2832407*C-homozygotes, the adjusted number needed to treat considering moderate adverse events is lower for Topiramate.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "RS2832407*C-HOMOZYGOTES", "target": "NNT-AEMOD+"}, {"weight": 6.0, "description": "The EA ethnicity shows differences in response to Topiramate based on the genetic variant rs2832407.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "EUROPEAN-AMERICAN (EA)", "target": "RS2832407*TOPIRAMATE"}, {"weight": 8.0, "description": "For rs2832407*C-homozygotes, moderate adverse events are more frequent with topiramate compared to placebo.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "MODERATE OR GREATER ADVERSE EVENTS", "target": "RS2832407*C-HOMOZYGOTES*TOPIRAMATE VS. PLACEBO"}, {"weight": 9.0, "description": "For rs2832407*A-allele carriers, there is a significant difference in the frequency of severe adverse events between topiramate and placebo.)", "source_id": "chunk-0c7ab28db433187759e760725a105836", "order": 1, "source": "RS2832407*A-ALLELE CARRIERS*TOPIRAMATE VS. PLACEBO", "target": "SEVERE ADVERSE EVENTS"}, {"weight": 2.0, "description": "The association between advanced stage cancer and the severity of delayed nausea.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "ADVANCED STAGE CANCER", "target": "RISK FACTOR FOR DELAYED NAUSEA"}, {"weight": 2.0, "description": "Almost all patients were prescribed palonosetron, a common anti-emetic medication.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "ANTI-EMETIC", "target": "PALONOSETRON"}, {"weight": 4.0, "description": "98% of the participants received dexamethasone, another commonly used anti-emetic drug.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "ANTI-EMETIC", "target": "DEXAMETHASONE"}, {"weight": 2.0, "description": "Younger age was associated with increased severity of patient-reported acute nausea.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "RISK FACTOR FOR ACUTE NAUSEA", "target": "YOUNGER AGE"}, {"weight": 4.0, "description": "Reduced sleep prior to chemotherapy was significantly linked to higher severity of patient-reported acute nausea.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "RISK FACTOR FOR ACUTE NAUSEA", "target": "FEWER HOURS OF SLEEP"}, {"weight": 2.0, "description": "Genetic variation in ABCB1 (rs1045642) was linked to more severe acute nausea, with heterozygotes reporting more severe symptoms than homozygous wild type or variant genotypes.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "RISK FACTOR FOR ACUTE NAUSEA", "target": "ABCB1 RS1045642 GENOTYPE"}, {"weight": 6.0, "description": "The younger age was not associated with acute vomiting, based on CTCAE grading.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "YOUNGER AGE", "target": "RISK FACTOR FOR ACUTE VOMITING"}, {"weight": 4.0, "description": "The receipt of highly emetogenic chemotherapy was significantly linked to increased risk factors for nausea, both acute and delayed.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "HIGHLY EMETOGENIC CHEMOTHERAPY", "target": "RISK FACTOR FOR NAUSEA"}, {"weight": 4.0, "description": "These factors together explained a significant portion of the variance in acute nausea severity.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "DEMOGRAPHIC, CLINICAL, PATIENT-REPORTED VARIABLES", "target": "RISK FACTORS FOR ACUTE NAUSEA"}, {"weight": 6.0, "description": "The ABCB1 genotype significantly enhanced the risk factors for acute nausea in conjunction with other demographic, clinical, and patient-reported variables.)", "source_id": "chunk-1ddde2b1bbb4b6e6dc1fbd9d0a8701a8", "order": 1, "source": "ABCB1 GENOTYPE", "target": "ENHANCER OF ACUTE NAUSEA RISK"}, {"weight": 1.0, "description": "The LCT gene exhibits modifications at specific genomic regions contributing to cell-type specific expression and epigenetic regulation.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "DNA MODIFICATION", "target": "LCT GENE"}, {"weight": 4.0, "description": "The CT and TT genotypes are associated with specific changes in DNA modifications within the analyzed region.)", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "order": 1, "source": "DNA MODIFICATION", "target": "CT AND TT GENOTYPES"}, {"weight": 3.0, "description": "Age-dependent changes affect LCT mRNA levels, indicating a dynamic regulatory process.)", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "order": 1, "source": "LCT MRNA", "target": "AGE-RELATED CHANGES"}, {"weight": 6.0, "description": "The lncRNA regulates LCT mRNA levels, suggesting a potential mechanism for modulating enzyme expression.)", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "order": 1, "source": "LCT MRNA", "target": "LNCRNA LOC100507600"}, {"weight": 2.0, "description": "Significant DNA modification differences occur at CpG sites, linking them as important genetic variants in LCT\'s expression control.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "CPG SITES", "target": "GENOMIC VARIANT ASSOCIATION"}, {"weight": 5.0, "description": "The lncRNA is linked with LCT intron 5 region and plays a role in its regulation, showing interplay between different genomic elements.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "LONG NON-CODING RNA (LNCRNA) LO100507600", "target": "FUNCTIONAL LINKAGE"}, {"weight": 7.0, "description": "The DNase I hypersensitivity sites at LCT introns 2 and 3, as well as MCM6\'s regions, indicate regulatory significance for gene expression.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "DNASE I HYPERSENSITIVITY SITES", "target": "REGULATORY SIGNIFICANCE"}, {"weight": 8.0, "description": "These histone marks are characteristic of active enhancers and play a role in gene regulation at the identified chromatin reference map sites.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "HISTONE MARKS H3K4ME1 AND H3K27AC", "target": "ENHANCER ROLES"}, {"weight": 2.0, "description": "The Affymetrix GeneChips used for gene profiling are specific and allow screening of thousands of transcripts.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "GENE EXPRESSION PROFILING", "target": "AFFYMETRIX GENECHIP"}, {"weight": 2.0, "description": "The haplotypes containing C-13910 in MCM6 intron 13 influence DNA modifications after conception.)", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "order": 1, "source": "MCM6 INTRON 13", "target": "C-13910"}, {"weight": 2.0, "description": "Increased DNA methylation inhibits CTCF binding at insulator sites, potentially activating gene expression.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "DNA METHYLATION", "target": "CTCF BINDING"}, {"weight": 2.0, "description": "The deletion in MCM6 intron 13 did not alter Mcm6 mRNA levels when compared to the negative control.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "CRISPR-CAS9 GENOME EDITING TECHNOLOGY", "target": "MCM6 INTRON 13 DELETION"}, {"weight": 5.0, "description": "High conservation of the LCT intron 2 element is observed across mammals, including humans.)", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "order": 1, "source": "LCT INTRON 2", "target": "MAMMALS"}, {"weight": 2.0, "description": "Evolutionarily conserved LCT intron 2 establishes stronger control of LCT mRNA in adulthood.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "LCT INTRON 2", "target": "AGE-SPECIFIC REGULATION"}, {"weight": 2.0, "description": "Deletions of epigenetically regulated DNA regions impacted human LCT expression, supporting their importance in regulation.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "EPIGENETICALLY-REGULATED REGIONS", "target": "CRISPR-CAS9N-INDUCED DELETION/"}, {"weight": 3.0, "description": "Differences in DNA modifications highlight the cell-type specific regulation of LCT expression and DNA modifications.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "CELL-TYPE SPECIFICITY", "target": "ENTEROCYTES AND JEJUNUM TISSUES"}, {"weight": 4.0, "description": "MCM6 gene shows similar regions to LCT\'s for DNA modification studies, highlighting the genetic linkage between these genes.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "HAPLOTYPE ASSOCIATION", "target": "MCM6 GENE"}, {"weight": 6.0, "description": "The MCM6\'s 3\u2032-UTR is associated with chromatin features that indicate active regulatory functions like enhancer sites and histone marks, demonstrating the gene\'s role in regulation.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "MCM6 GENE 3\u2032-UTR", "target": "REGULATORY FUNCTION"}, {"weight": 9.0, "description": "The LCT-MCM6 haplotypes exhibit strong associations with lactase persistence and non-persistence across genetic backgrounds, highlighting the role of these haplotypes in age-dependent regulation.)", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "GENETIC LINKAGE", "target": "LCT-MCM6 HAPLOTYPES"}, {"weight": 1.0, "description": "Age-dependent epigenetic modifications occur at SNP C/T-13910, indicating temporal genomic responses to genetic variations over life stages.)(entity", "source_id": "chunk-4964b2db9e3ddddb4967fb7150e4404d", "order": 1, "source": "SNP C/T-13910 AND AGE CHANGES", "target": "TEMPORAL GENOMIC RESPONSE"}, {"weight": 4.0, "description": "SNPs at this position influence the variation in haplotypes associated with lactase non-persistence or persistence.)", "source_id": "chunk-ea45644d73f260cf365527dcc3af86e1", "order": 1, "source": "HAPLOTYPES", "target": "SNP-13910"}, {"weight": 1.0, "description": "Deletions affecting MCM6 intron 13 or LCT intron 2 resulted in significant decreases in LCT mRNA levels.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "DOWNREGULATION/DECREASE IN LCT MRNA", "target": "MCM6 INTRON 13/LCT INTRON 2"}, {"weight": 1.0, "description": "Deletions affecting MCM6 intron 13 or LCT intron 2 resulted in significant decreases in LCT mRNA levels.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "DOWNREGULATION/DECREASE IN LCT MRNA", "target": "MCM6 INTRON 13/LCT INTRON 2/"}, {"weight": 2.0, "description": "The mutation in Lct intron 2 produced a significantly greater loss of LCT mRNA expression in adult mice compared to infants.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "ADULT INTESTINE/INFANT GUT RESPONSE", "target": "LACTASE PERSISTENCE/CRISPR-CAS9 DELETIONS"}, {"weight": 1.0, "description": "Deletions affecting MCM6 or LCT regulatory regions resulted in a significant decrease in LCT mRNA levels in differentiated human intestinal epithelial cells.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "MCM6 INTRON 13/LCT INTRON 2/", "target": "REGULATORY IMPACT"}, {"weight": 1.0, "description": "The deletion effects on LCT mRNA varied between undifferentiated and differentiated states of cultured Caco-2 cells, supporting the importance of regulatory regions in human LCT expression.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "CELLULAR STATE/EXPRESSION LEVELS", "target": "CRISPR-CAS9 DELETIONS/LCT EXPRESSION"}, {"weight": 1.0, "description": "The deletion effects on LCT mRNA varied between undifferentiated and differentiated states of cultured Caco-2 cells, supporting the importance of regulatory regions in human LCT expression.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "CELLULAR STATE/EXPRESSION LEVELS", "target": "CRISPR-CAS9 DELETIONS/LCT EXPRESSION/"}, {"weight": 1.0, "description": "Deletions affecting MCM6 or LCT regulatory regions resulted in a significant decrease in LCT mRNA levels in differentiated human intestinal epithelial cells.)", "source_id": "chunk-512b044cf65a4e519d519c6239959c6c", "order": 1, "source": "DECREASE IN LCT MRNA", "target": "MCM6/LCT REGULATORY REGIONS/"}, {"weight": 3.0, "description": "The highest frequency of lactase persistence allele showed the strongest association between genotype and milk consumption.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "CZECH REPUBLIC", "target": "MILK CONSUMPTION"}, {"weight": 11.0, "description": "The strongest representation of associations between variables like education and smoking was found in the Czech Republic.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "CZECH REPUBLIC", "target": "ASSOCIATIONS"}, {"weight": 7.0, "description": "Elevated risk of RCC was found associated with milk consumption.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "MILK CONSUMPTION", "target": "RCC"}, {"weight": 9.0, "description": "Strong relationship found specifically in the Czech Republic between milk consumption and RCC risk.)", "source_id": "chunk-7a423a439657efc1126fb6a075919ea5", "order": 1, "source": "MILK CONSUMPTION", "target": "RCC OR=1.68"}, {"weight": 1.0, "description": "Also associated with increased delta HbF compared to the T-G reference haplotype.)", "source_id": "chunk-3cf4e094a5312845a1cc4da4304d4bdf", "order": 1, "source": "T-A-C HAPLOTYPE", "target": "DELTA HBF"}, {"weight": 12.0, "description": "Decreased transcript levels of the UBE2T gene were detected, which is a consequence of mutations identified in this subject.)", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "order": 1, "source": "UBE2T DECREASED TRANSCRIPT LEVELS", "target": "CONSEQUENCE OF MUTATION IN UBE2T"}, {"weight": 4.0, "description": "While Ube2T interacts with the RING domain of FANCL in a 1:1 stoichiometry, it does not interact directly with URD as confirmed by size-exclusion chromatography.)", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "order": 1, "source": "UBE2T", "target": "URD"}, {"weight": 1.0, "description": "Overexpression of wild type UBE2T gene rescues FA phenotypes in RA2627 fibroblasts.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "WILD TYPE UBE2T", "target": "OVEREXPRESSION"}, {"weight": 3.0, "description": "The overexpression of wild type UBE2T rescues the cellular hypersensitivity shown by RA2627 cells to crosslinking agents like MMC.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "WILD TYPE UBE2T", "target": "CELLULAR HYPERSENSITIVITY"}, {"weight": 4.0, "description": "UBE2T overexpression in RA2627 cells reduces G2 phase cell cycle accumulation induced by MMC treatment.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "WILD TYPE UBE2T", "target": "G2 PHASE ACCUMULATION"}, {"weight": 5.0, "description": "The expression of wild type UBE2T restores the monoubiquitination status of FANCD2 and FANCI proteins following FA phenotype induction by treatment with MMC.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "WILD TYPE UBE2T", "target": "FANCD2 MONOUBIQUITINATION"}, {"weight": 6.0, "description": "UBE2T overexpression rescues the formation of FANCD2 foci observed after treatment with MMC, associated with FA phenotype rescue.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "WILD TYPE UBE2T", "target": "FANCD2 FOCI FORMATION"}, {"weight": 13.0, "description": "The combination of maternally derived duplication and paternally derived deletion resulted in genetic reversion phenomena observed in the proband\'s LCLs, supporting normal UBE2T expression.)", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "order": 1, "source": "COMPOUND HETEROZYGOUS MUTATIONS", "target": "GENETIC REVERSION PHENOMENON"}, {"weight": 1.0, "description": "Wild type UBE2T overexpression was able to rescue cellular defects associated with Fanconi anemia in RA2627 fibroblasts, demonstrating the causality of UBE2T deficiency.)", "source_id": "chunk-b3b792ab4e6f6251de68ef6e959aee1f", "order": 1, "source": "CELLULAR PHENOTYPE RESCUED BY UBE2T", "target": "OVEREXPRESSION OF WILD TYPE UBE2T"}, {"weight": 2.0, "description": "Insufficient expression or mutation in UBE2T leads to Fanconi anemia.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "FANCONI ANEMIA (FA)", "target": "UBE2T DEFICIENCY"}, {"weight": 1.0, "description": "The overexpression of wild type UBE2T rescues cellular defects associated with Fanconi Anemia (FA) in RA2627 fibroblasts, as shown by restored cell cycle distribution and FANCD2 foci formation.)", "source_id": "chunk-ffd1276c89bcac745a7e58082e97236f", "order": 1, "source": "FA PHENOTYPE RESCUE", "target": "WILD TYPE UBE2T OVEREXPRESSION"}, {"weight": 4.0, "description": "DBA patients carrying mutations in RPS19 were the focus of the gene expression profiling study.)<SEP>Mutations in RPS19 were identified as significant in DBA patients through the gene expression study.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "RPS19"}, {"weight": 1.0, "description": "DBA patients showed differentially expressed LARS compared to controls.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "LARS"}, {"weight": 2.0, "description": "WARS was found differently expressed in DBA patients\' gene expression profiles.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "WARS"}, {"weight": 3.0, "description": "Variations in GARS expression were identified among DBA patient samples.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "GARS"}, {"weight": 4.0, "description": "Significant differential expression of SARS was noted in DBA patients compared to healthy individuals.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "SARS"}, {"weight": 5.0, "description": "Differences in QARS mRNA levels were detected between DBA patient groups and controls.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "QARS"}, {"weight": 6.0, "description": "Alterations in EPRS expression was observed specifically in DBA patients through the study.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "EPRS"}, {"weight": 7.0, "description": "Statistically significant differential PSAT1 expression in DBA patients.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "PSAT1"}, {"weight": 8.0, "description": "DBA patients showed differences in PHGDH gene expression compared to controls.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "PHGDH"}, {"weight": 9.0, "description": "CTH expression varied between DBA patient and control samples, indicating possible involvement.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "CTH"}, {"weight": 10.0, "description": "SHMT2 was differentially expressed in the gene profiling of DBA patients.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "SHMT2"}, {"weight": 11.0, "description": "Differences in CASP9 expression were found specifically among DBA patient groups compared to controls.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "CASP9"}, {"weight": 12.0, "description": "Variations in NFKBIA mRNA levels were identified between DBA patients and healthy individuals.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "NFKBIA"}, {"weight": 13.0, "description": "Differential expression of NFKBIE was found among DBA patient samples compared to controls.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "NFKBIE"}, {"weight": 14.0, "description": "Alterations in APAF1 mRNA levels were detected specifically in DBA patients.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "APAF1"}, {"weight": 15.0, "description": "Differences in NFKB2 expression were noted among DBA patient samples compared to control individuals.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "NFKB2"}, {"weight": 16.0, "description": "NFKB1 showed variations in expression levels specifically within the DBA patient group.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "NFKB1"}, {"weight": 17.0, "description": "Differential BIRC3 expression was detected between DBA patients and controls during gene profiling.)", "source_id": "chunk-34ffb077c584b98555081f882f8f8b94", "order": 1, "source": "DBA PATIENTS", "target": "BIRC3"}, {"weight": 1.0, "description": "Differential expression was identified for specific genes associated with DBA patients compared to controls.)", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "order": 1, "source": "GENE EXPRESSION DIFFERENCES", "target": "OUTCOME OF STUDY"}, {"weight": 2.0, "description": "The microarray data was validated using qRT-PCR, providing a cross-check on gene expression levels.)", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "order": 1, "source": "METHOD COMPARISON", "target": "MICROARRAY DATA VALIDATION"}, {"weight": 3.0, "description": "Both methods showed consistent trends for differentially expressed genes across the samples.)", "source_id": "chunk-dcbd547f8218e7097805720fd6ea43ba", "order": 1, "source": "OUTCOME OF CROSS-VALIDATION", "target": "QRT-PCR AND MICROARRAY RESULTS AGREEMENT"}, {"weight": 0.03, "description": "A significant association between rs6265 polymorphism and migraine is observed using the fixed-effects model.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "BDNF RS6265", "target": "MIGRAINE"}, {"weight": 1.0, "description": "The guanine nucleobase component of the rs6265 polymorphism.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "BDNF RS6265", "target": "GUANINE"}, {"weight": 0.02, "description": "The data suggests a significant association of rs2049046 polymorphism with migraine in allele, recessive and additive models.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "MIGRAINE", "target": "RS2049046 A/T"}, {"weight": 1.0, "description": "The thymine nucleobase component of the rs2049046 polymorphism.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "BDNF RS2049046", "target": "THYMINE"}, {"weight": 1.0, "description": "The criterion for selecting studies which were conducted on individuals from a specific geographical region and ancestry group.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "CAUCASIAN POPULATION GROUPS", "target": "STUDY SELECTION CRITERIA"}, {"weight": 1.0, "description": "A component of the study design involving cohorts or participants from Australia.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "AUSTRALIAN-BASED COHORTS", "target": "STUDY COMPONENT"}, {"weight": 1.0, "description": "The process by which specific genetic variations affect the risk or development of migraine headaches.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "MOLECULAR INTERACTION BETWEEN SNPS AND MIGRAINE", "target": "BIOLOGICAL PROCESS"}, {"weight": 1.0, "description": "A statistical model used to analyze combined data from multiple studies with minimal heterogeneity.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "FIXED-EFFECTS MODEL", "target": "METHODOLOGY IN META-ANALYSIS"}, {"weight": 1.0, "description": "The comparison of variance among different studies contributing to the meta-analysis when no significant differences are present.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "HETEROGENEITY DIFFERENCE", "target": "STATISTICAL ANALYSIS IN META-ANALYSIS"}, {"weight": 1.0, "description": "A statistical test utilized for identifying potential bias in publication, specifically by examining asymmetry in funnel plots.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "BEGG\u2019S TEST", "target": "DETECTION OF PUBLICATION BIAS"}, {"weight": 1.0, "description": "An approach used to identify publication bias through the correlation between effect size and precision across studies.)", "source_id": "chunk-814fe155e9cb953da61f6a79d1d25784", "order": 1, "source": "EGGER\u2019S TEST", "target": "BIAS DETECTION METHOD"}, {"weight": 6.0, "description": "Frequency of recognition is comparable for neoepitopes and an immunodominant epitope, indicating similar interaction potential.)", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "order": 1, "source": "CD8+ T CELLS", "target": "RECOGNITION"}, {"weight": 1.0, "description": "CD3+ skin T cells of classical psoriasis showed instead an enrichment of both CD4+ and CD8+ subpopulations.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "T CELLS", "target": "CLASSICAL PSORIASIS"}, {"weight": 2.0, "description": "The type I IFN-\u03b2 was expressed in paradoxical skin lesions, mainly in keratinocytes.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "TYPE I IFN-\u0392", "target": "PARADOXICAL PSORIASIS"}, {"weight": 3.0, "description": "The epidermis of psoriasiform reactions was also immunoreactive for IFN-\u03ba, another keratinocyte-derived type I IFN.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "IFN-\u039a", "target": "PSORIASIFORM REACTIONS"}, {"weight": 4.0, "description": "We finally investigated LT-\u03b1 and LT-\u03b2, two members of the cytokine TNF family, also known as TNF-\u03b2 and TNF-C 35, possibly deregulated by anti-TNF-\u03b1 therapy.)", "source_id": "chunk-56c37d6045382dda8cc6037b2680282a", "order": 1, "source": "LL-37", "target": "PARADOXICAL PSORIASIS"}, {"weight": 2.0, "description": "A comparison between genes is performed here with respect to their SNPs, indicating a possible genetic linkage or functional association.)<SEP>Comparison between these two genes emphasizes their role or potential interactions related to specific SNPs.)", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "CCHCR1", "target": "IL23R"}, {"weight": 1.0, "description": "rs1004819", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "RS11209026", "target": "RS72676067/"}, {"weight": 1.0, "description": "Illustrates genetic diversity among HS patients through comparison of SNPs found in the IL23R gene.)", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "RS72676067", "target": "RS1004819"}, {"weight": 1.0, "description": "The term indicates possible outcomes related to specific SNPs or genes involved, such as adverse effects of certain treatments.)", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "PARADOXICAL PSORIASIFORM REACTIONS", "target": "DISEASE/OUTCOME"}, {"weight": 1.0, "description": "Relevance to certain SNPs and potential side effects of treatment, highlighting risks in genetic profiling.)", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "PARADOXICAL PSORIASIFORM REACTIONS", "target": "OUTCOME/ADVERSE EFFECT"}, {"weight": 2.0, "description": "Connection to the previously mentioned conditions when discussing alternative treatments or outcomes.)<SEP>The class of drugs is mentioned in relation to potential side effects observed in the context of genetic variations.)", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "ANTI-TNFS", "target": "THERAPY/DRUG"}, {"weight": 2.0, "description": "A defined group of subjects whose data contributes to the comparative analysis.)<SEP>Focus on genetic analysis involving a specific group of subjects, in this context related to SNPs and potential health outcomes.)", "source_id": "chunk-72f4505e43f67f3564607392cb178656", "order": 1, "source": "HS PATIENTS", "target": "SUBJECT GROUP/SAMPLE"}, {"weight": 3.0, "description": "The DmDRWD is associated with the function of DmFANCL, although they are not directly interacting proteins.)", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "order": 1, "source": "DMFANCL", "target": "DMDRWD"}, {"weight": 1.0, "description": "DALI Comparison/", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "order": 1, "source": "E2 UBC ENZYMES/", "target": "STRUCTURAL SIMILARITY/"}, {"weight": 1.0, "description": "Thioester Charge Assay/", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "order": 1, "source": "CATALYTIC CYSTEINE/", "target": "THIOESTER BOND FORMATION/"}, {"weight": 1.0, "description": "The URD interacts with FANCI to facilitate substrate binding during DNA repair processes.)", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "order": 1, "source": "URD", "target": "FANCI"}, {"weight": 2.0, "description": "The URD binds with FANCD2 in a similar mechanism as it does for FANCI, supporting its role in DNA repair and recombination.)", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "order": 1, "source": "URD", "target": "FANCD2"}, {"weight": 5.0, "description": "The mutation at Phe-63 in the Ube2T protein affects its binding affinity to the RING domain of FANCL, impacting interactions indirectly through the URD.)", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "order": 1, "source": "URD", "target": "PHE-63 IN UBE2T"}, {"weight": 6.0, "description": "The URD plays a critical role in facilitating the interaction between FANCI and FANCD2 during DNA repair processes by serving as a binding site.)", "source_id": "chunk-e097891654194982dfc0ef87401d3aa9", "order": 1, "source": "URD", "target": "FANCI AND FANCD2"}, {"weight": 1.0, "description": "Mutations in this variant of URD showed reduced binding with FANCI and FANCD2.)", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "URD", "target": "L248A/F252A/L254A/I265A"}, {"weight": 1.0, "description": "Mutations in this variant of URD behaved as wild type when tested against pulldown arrays.)", "source_id": "chunk-6135f8baf39fe26dcea3d987aa6100ac", "order": 1, "source": "URD", "target": "W212A/L214A"}, {"weight": 1.0, "description": "Structure Comparison/", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "order": 1, "source": "FANCL/", "target": "URD/"}, {"weight": 1.0, "description": "Comparative Analysis/", "source_id": "chunk-6b6599f2e169a7aa2a4822b71605cbc8", "order": 1, "source": "N-TERMINAL/ C-TERMINAL LOBES/", "target": "SURFACE COMPOSITION/"}, {"weight": 0.46, "description": "Significantly delayed CT and numerically decreased GC24 in the GA/AA genotype group, compared to placebo.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "FGFR4 RS351855", "target": "COLESEVELAM TREATMENT EFFECT"}, {"weight": 1.0, "description": "Both were genetic variants that showed association with differential colesevelam treatment effect.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "FGFR4 RS351855", "target": "KLB GENOTYPE VARIATION"}, {"weight": 0.048, "description": "Lower GC24 value and potentially longer AC t1/2 were observed in the CA/AA genotype group, compared to placebo.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "KLB RS4975017", "target": "COLESEVELAM TREATMENT EFFECT"}, {"weight": 1.0, "description": "Numerical but not statistically significant differential Colesevelam treatment effects on AC t1/2 and GC24, with moderate effects in the GG genotype.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "KLB RS17618244", "target": "COLESEVELAM TREATMENT EFFECT"}, {"weight": 1.0, "description": "This variant showed numerical, though not statistically significant association with differential colesevelam treatment effects on AC t1/2 and GC24.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "RS17618244", "target": "COLESEVELAM TREATMENT EFFECT"}, {"weight": 1.0, "description": "The rs351855 GA/AA genotype group showed a significantly delayed CT, with increased AC t1/2 compared to placebo, while the GG genotype group did not show any significant effect of Colesevelam treatment.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "COLESEVELAM TREATMENT EFFECT", "target": "RS351855 GA/AA GENOTYPE GROUP AND GG GENOTYPE GROUP"}, {"weight": 1.0, "description": "No significant effects of Colesevelam treatment on CT were observed in the CC genotype group, compared to placebo.)", "source_id": "chunk-5c78814fb7c198ee95071ee5a8462070", "order": 1, "source": "COLESEVELAM TREATMENT EFFECT", "target": "RS351855 GA/AA GENOTYPE GROUP AND CC GENOTYPE GROUP"}, {"weight": 1.842, "description": "The total content of stool bile acids correlated positively and strongly with the weight of stool samples.)<SEP>The total content of stool bile acids was positively correlated with stool weight samples, reflecting digestion and fat absorption.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "STOOL WEIGHT", "target": "TOTAL STOOL BA"}, {"weight": 1.712, "description": "A correlation between the weight of stools and fat content indicated differences in digestive efficiency among participants.)<SEP>A positive correlation between the weight of stools and the fat content was observed.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "STOOL WEIGHT", "target": "FAT CONTENT"}, {"weight": -0.732, "description": "A negative correlation between fasting serum FGF19 levels and stool weight suggested possible metabolic differences influencing fecal output.)<SEP>A negative correlation between fasting serum FGF19 levels and stool weight was observed, indicating less weight could be associated with higher FGF19.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "STOOL WEIGHT", "target": "FGF19"}, {"weight": 1.474, "description": "A significantly higher serum C4 level correlated with a higher stool BA synthesis or excretion.)<SEP>Increased serum C4 levels were associated with higher stool BA synthesis or excretion, suggesting the role of this protein in bile acid dynamics.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "STOOL BA", "target": "SERUM C4"}, {"weight": 1.056, "description": "Bile acid synthesis was moderately correlated with serum C4 levels, indicating its influence on bile acid metabolism in participants.)<SEP>The total content of bile acids in stools correlated positively but moderately with serum C4 levels.)", "source_id": "chunk-7aea4201f8f5f090ffa8d3bfe2fcdff6", "order": 1, "source": "SERUM C4", "target": "TOTAL STOOL BA"}, {"weight": 2.0, "description": "The FAAP20 protein binds to FANCA through its N-terminal region, demonstrating their interplay in cellular processes.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "FAAP20", "target": "FANCA"}, {"weight": 3.0, "description": "The UBZ domain of FAAP20 plays a critical role in the binding and interaction with Ub chains.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "FAAP20", "target": "UBZ DOMAIN"}, {"weight": 6.0, "description": "SPR data suggests that FAAP20-UBZ might interact with K63-Ub2 in a 1:1 stoichiometry, although this may represent an experimental artifact based on the calculated Rmax.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "FAAP20", "target": "1:1 STOICHIOMETRY INTERACTION"}, {"weight": 4.0, "description": "FAAP20 interacts specifically with K63-Ub2 through its UBZ domain, indicating specific targeting of this type of ubiquitin chain in the cell.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "K63-LINKED UB2", "target": "FAAP20-UBZ COMPLEX"}, {"weight": 5.0, "description": "The role of FAAP20 in the ubiquitination pathway highlights its importance for maintaining cellular stability and homeostasis, which is a fundamental process influenced by environmental factors.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "UBIQUITINATION PATHWAY", "target": "CELLULAR HOMEOSTASIS"}, {"weight": 4.0, "description": "\u03b1-helix mediated Ub interaction is closely related to MIU domain of Rabex5 which shows convergent evolution with FAAP20-UBZ among different UBDs.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "UB", "target": "FAAP20-UBZ"}, {"weight": 3.0, "description": "The interaction mode between NEMO-UBZ and Ub is different from Pol\u03b7-UBZ due to differences in the \u03b2-loop sequence.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "NEMO-UBZ", "target": "POL\u0397-UBZ"}, {"weight": 5.0, "description": "Interaction with Ile44 of Ub through their \u03b2-loop suggests similarities between WRNIP1, Rad18 and Pol\u03b7-UBZ binding to Ub.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "POL\u0397-UBZ", "target": "WRNIP1/RAD18 GROUP"}, {"weight": 5.0, "description": "The \u03b2-loop region of Pol\u03b7-UBZ shares similarities with WRNIP1/Rad18 group proteins in interaction with Ile44 of Ub, highlighting common mechanisms among these UBDs.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "POL\u0397-UBZ", "target": "WRNIP1-UBZ"}, {"weight": 5.0, "description": "Similar residues are involved in Ub-binding by Vps27-UIM and FAAP20-UBZ with opposite polarity towards the bound Ub showing convergent evolution among UBDs.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "FAAP20-UBZ", "target": "VPS27-UIM"}, {"weight": 1.0, "description": "The same critical residues involved in Ub binding are identical between Rabex5-MIU and FAAP20-UBZ suggesting closely related interaction mechanisms.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "FAAP20-UBZ", "target": "RABEX5-MIU"}, {"weight": 4.0, "description": "While FAAP20-UBZ shows interactions with Ub similar to MIU domain of Rabex5 and Vps27-UIM, it differs in its mode of interaction compared to RAD18-UBZ indicating structural diversity among UBDs.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "FAAP20-UBZ", "target": "RAD18-UBZ"}, {"weight": 7.0, "description": "The interaction between specific dietary nutrients, like folate from leafy greens, is crucial for maintaining optimal cardiovascular health based on lifestyle factors and genetic predispositions.)", "source_id": "chunk-cc528af98598e1adb4684c0166430cf4", "order": 1, "source": "NUTRIENT INTERACTION", "target": "RELATIONSHIP STRENGTH (5/6)"}, {"weight": 10.0, "description": "Interactions between nutrients and specific molecular processes in the body can have significant effects on physiological functions like metabolism and disease risk.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "NUTRIENT INTERACTION", "target": "MOLECULAR PROCESS"}, {"weight": 7.0, "description": "Mutations D164A, A168L and W180A in FAAP20-UBZ disable binding to Ub and C150S mutation destabilizes the structure leading to a lack of functional recruitment of FAAP20 proteins.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "EFFECT", "target": "FAAP20-UBZ MUTATIONS"}, {"weight": 8.0, "description": "The intermolecular interactions between FAAP20 protein domain UBZ and Ub are essential for proper cellular localization and accumulation at DNA damage sites in response to intermolecular binding events and ICL formation processes.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "SIGNIFICANCE", "target": "UB INTERACTION"}, {"weight": 9.0, "description": "Conservation of residues involved in Ub interactions is observed among various species but sequence variations are noted due to high divergence between UBZ structures, requiring detailed 3D structural analysis for precise comparisons.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "EVOLUTIONARY INSIGHT", "target": "RESIDUE CONSERVATION"}, {"weight": 10.0, "description": "Comparative analysis reveals differences in the modes and strength of interactions between various UBZs (FAAP20-UBZ, WRNIP1-UBZ, RAD18-UBZ) and Ub based on conserved or replaced key residues.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "OBSERVATION", "target": "UB BINDING COMPARISON"}, {"weight": 11.0, "description": "The variations in key interacting residues among UBZ proteins suggest evolutionary adaptations for binding efficiency or specific roles in cellular processes involving Ub.)", "source_id": "chunk-fa32a208a2f004c10c1b6777c3be13f2", "order": 1, "source": "IMPLICATION", "target": "RESIDUE VARIATIONS"}, {"weight": 4.0, "description": "While Vps27-UIM has a different polarity towards the bound Ub compared to FAAP20-UBZ and Rabex5-MIU, it shares some identical residues involved in Ub-binding suggesting possible functional similarities.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "VPS27-UIM", "target": "RAD18-UBZ"}, {"weight": 4.0, "description": "Interaction with Ub by Rabex5-MIU closely resembles FAAP20-UBZ despite having opposite polarity towards bound Ub, indicating evolutionary conservation of interaction motifs among UBDs.", "source_id": "chunk-5bfe051a79e5e215dd3fc65296825591", "order": 1, "source": "RABEX5-MIU", "target": "RAD18-UBZ"}, {"weight": 0.014, "description": "The DQ8 heterozygous haplotypes are highly frequent among CD patients compared to control group.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "DQ8 HAPLOTYPE", "target": "CD PATIENTS"}, {"weight": 1.0, "description": "The predicted homozygous DQ2.2 haplotype was found more frequently among control group compared to CD patients.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "DQ2.2 HAPLOTYPE", "target": "CD PATIENTS"}, {"weight": 1.0, "description": "Commonly observed as a clinical symptom among CD patients, often linked to weight loss or malnutrition.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "ANOREXIA", "target": "CD PATIENTS"}, {"weight": 1.0, "description": "Often reported by CD patients, reflecting the body\'s response to inflammation in the gut.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "ABDOMINAL DISTENSION", "target": "CD PATIENTS"}, {"weight": 1.0, "description": "An autoimmune disease that affects insulin production; a part of our disease manifestation group.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "TYPE 1 DIABETES MELLITUS", "target": "DISEASE GROUP"}, {"weight": 1.0, "description": "A chronic condition involving the thyroid gland; another autoimmune condition observed among CD patients.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "AUTOIMMUNE THYROIDITIS", "target": "DISEASE GROUP"}, {"weight": 1.0, "description": "Chronic inflammatory disease affecting multiple organs; one of several autoimmune disorders in our patient group.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "SYSTEMIC LUPUS ERYTHEMATOSUS", "target": "DISEASE GROUP"}, {"weight": 1.0, "description": "Condition characterized by weak bones often due to vitamin D deficiency; identified among a small percentage of participants.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "OSTEOMALACIA", "target": "NON-AUTOIMMUNE DISEASE GROUP"}, {"weight": 1.0, "description": "Conditions characterized by recurrent seizures, sometimes associated with autoimmune diseases like CD; observed in a subset of patients.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "SEIZURE DISORDERS", "target": "NEUROLOGICAL CONDITION GROUP"}, {"weight": 1.0, "description": "Genetic disorder leading to cognitive and physical development challenges; identified among approximately 4% of participants.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "DOWN SYNDROME", "target": "DEVELOPMENTAL DISORDER GROUP"}, {"weight": 0.024, "description": "HLA-DQ genotypes show a significant difference between CD patients compared to healthy controls.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "CD PATIENTS", "target": "HLA-DQ HAPLOTYPE"}, {"weight": 1.0, "description": "There are significant differences in the minor allele frequency distribution among CD patients compared to healthy controls for these SNPs.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "HEALTHY CONTROLS", "target": "RS2395182, RS7775228, AND RS2187668"}, {"weight": 1.0, "description": "No difference exists between the tag SNP predicted homozygous HLA-DQ2.5 haplotype in CD patients and healthy controls, though not statistically significant.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "HEALTHY CONTROLS", "target": "DQ2.5 HAPLOTYPE"}, {"weight": 2.0, "description": "Rare presence of specific genotypes at PTPN22:rs2476601 among healthy controls implies lower risk for MAPbacteremia.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "HEALTHY CONTROLS", "target": "PTPN22:RS2476601"}, {"weight": 4.0, "description": "Decrease in PTPN2 expression level between healthy controls and CD subjects regardless of MAPbacteremia presence.)", "source_id": "chunk-6a657cd0a000b6df4db8f1305b795021", "order": 1, "source": "HEALTHY CONTROLS", "target": "PTPN2 EXPRESSION"}, {"weight": 1.0, "description": "The sequencing analysis with sanger method confirmed the accuracy of Real-time PCR based tagSNPs approach in determining the HLA-DQ haplotype.)", "source_id": "chunk-1418daebca32811d9530c70d8e9bf4cc", "order": 1, "source": "HLA TAG SNPS AND HAPLOTYPE RESULTS", "target": "VALIDATION BY SANGER SEQUENCING"}, {"weight": 1.0, "description": "The rs1799793 ERCC2 G>A variant showed a significant correlation with progression-free interval (PFI) and overall survival (OS) in ovarian cancer.)", "source_id": "chunk-561eacb5d916258d3ed4f08f53587404", "order": 1, "source": "RS1799793", "target": "PFI AND OS"}, {"weight": 6.0, "description": "Two SNPs within the IL-22 gene were studied in association with CRC)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "IL-22 (RS1179251 C/G & RS2227485 C/T)", "target": "INTERLEUKIN 22"}, {"weight": 7.0, "description": "SNP analysis of the promoter region of IL-27 gene and its potential role in CRC)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "IL-27 (RS17855750 T/G)", "target": "INTERLEUKIN 27"}, {"weight": 3.0, "description": "The variant GG genotype and G allele were slightly associated with a decreased risk of CRC development)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "IL-27 (RS17855750 T/G)", "target": "CRC RISK"}, {"weight": 2.0, "description": "Decreased CT genotype in the TGF\u00df1 gene was observed to be associated with a lower risk of developing CRC)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "RS1800469 C/T", "target": "DISEASE RISK"}, {"weight": 1.0, "description": "The T allele has a slightly lower frequency compared to the CC genotype in CRC patients.)", "source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "order": 1, "source": "CC GENOTYPE", "target": "T ALLELE"}, {"weight": 3.0, "description": "The TGF\u00df1 gene polymorphism which was studied in relation to CRC)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "TGFSS1 (RS1800469 C/T)", "target": "TRANSFORMING GROWTH FACTOR \u03921"}, {"weight": 4.0, "description": "A variant of the IL-8 gene and its promoter region SNP were analyzed for its role in CRC)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "IL-8 (RS4073 T/A)", "target": "INTERLEUKIN 8"}, {"weight": 5.0, "description": "SNP analysis conducted on the promoter region of IL-10 gene and its correlation with CRC)", "source_id": "chunk-bd81c38d10d8773bc4076cbb20bc289f", "order": 1, "source": "IL-10 (RS1800896 A/G)", "target": "INTERLEUKIN 10"}, {"weight": 2.0, "description": "Association between specific genotypes (CC) and the risk of developing Colorectal Cancer)", "source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "order": 1, "source": "DISEASE: COLORECTAL CANCER", "target": "GENOMIC VARIANT: CC"}, {"weight": 3.0, "description": "The frequency difference comparison between T allele and other genotypes in CRC patients.)", "source_id": "chunk-5b0cc96debf110f3544685d3b227ff27", "order": 1, "source": "FREQUENCY DIFFERENCE", "target": "GENOMIC VARIANT: T ALLELE"}, {"weight": 8.0, "description": "No significant association between this SNP and asthma risk was found among never-smokers.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "NEVER-SMOKERS", "target": "SNP RS1898671"}, {"weight": 6.0, "description": "Increased IL-8 mRNA expression was linked with the frequency of atopic asthma among children.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "ATOPY", "target": "IL-8 PRO-INFLAMMATORY RESPONSE"}, {"weight": 6.0, "description": "Increased risk association with asthma specifically observed when analyzing ex-smokers.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "SNP RS1898671", "target": "EX-SMOKERS"}, {"weight": 10.0, "description": "Confirmation of findings in a larger combined cohort, validating the associations made with NYUBAR population.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "ASTHMA SUSCEPTIBILITY", "target": "REPLICATION COHORT VALIDATION"}, {"weight": 5.0, "description": "The frequencies of rs2227306 CC and CT genotypes were found to be significantly different between case and control groups, suggesting its association with asthma susceptibility.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "ASTHMA SUSCEPTIBILITY", "target": "RS2227306 CC/CT GENOTYPES"}, {"weight": 12.0, "description": "Increased risk of asthma was observed when taking into account population structure differences in the analysis.)", "source_id": "chunk-6f7ce1c71bd46f223756da44b0d0bb2c", "order": 1, "source": "HAPLOTYPE GAGGGCAAAG", "target": "POPULATION STRATIFICATION ADJUSTMENT"}, {"weight": 1.0, "description": "The odds ratio quantifies how much more likely it is for CD subjects with specific PTPN2:rs478582 genotypes to develop MAP infection, indicating the strength of their association.)", "source_id": "chunk-6cb127717713bd0ab5be9436fc0c7568", "order": 1, "source": "OR VALUE (ODDS RATIO)", "target": "STRENGTH OF ASSOCIATION"}, {"weight": 4.0, "description": "The presence of MAPbacteremia was found to impact T-cell response and proliferation in CD patients.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "MAPBACTEREMIA", "target": "T-CELL RESPONSE"}, {"weight": 3.0, "description": "PHA stimulation led to an increased T-cell proliferation response in both CD subjects and healthy controls.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "PHA", "target": "T-CELL PROLIFERATION"}, {"weight": 4.0, "description": "The use of MAP PPD-like as a stimulus showed different effects on the T-cell response across various subject groups.)", "source_id": "chunk-883c371ea1f500a1ba8af8366328e8b0", "order": 1, "source": "MAP PPD-LIKE", "target": "T-CELL RESPONSE"}, {"weight": 1.0, "description": "The kinetics of CD69 involve an intensification followed by a reduction of surface expression.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CD69", "target": "ACTIVATION MARKER KINETICS"}, {"weight": 1.0, "description": "CD69 is an activation marker expressed on T cells.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CD69", "target": "ACTIVATION MARKER"}, {"weight": 2.0, "description": "ICOS expression kinetics involve an intensification followed by a reduction of surface expression.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "ICOS", "target": "ACTIVATION MARKER KINETICS"}, {"weight": 2.0, "description": "ICOS is a co-stimulatory molecule involved in the activation of T lymphocytes.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "ICOS", "target": "ACTIVATION MARKER"}, {"weight": 3.0, "description": "The expression of CD25 was observed when normalized to internal mock-transfected controls.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CD25", "target": "INTERNAL MOCK-TRANSFECTED CONTROLS"}, {"weight": 3.0, "description": "CD25, also known as IL-2 receptor alpha chain (CD152), is upregulated on activated T cells and plays a role in costimulation during immune responses.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CD25", "target": "CO-STIMULATORY LIGAND"}, {"weight": 4.0, "description": "CD226 on Tregs exhibited a unique kinetics where it waned before increasing to a level higher than baseline.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CD226", "target": "TREGS ACTIVATED KINETICS"}, {"weight": 4.0, "description": "CD226 (also known as KLRB1 or NKG2D) is an activation receptor expressed on various types of immune cells.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CD226", "target": "ACTIVATION MARKER"}, {"weight": 5.0, "description": "Tregs also exhibited activation-induced expression kinetics for CTLA-4.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CTLA-4", "target": "TREGS ACTIVATED KINETICS"}, {"weight": 5.0, "description": "CTLA-4 is a co-inhibitory receptor that downregulates the activity of T lymphocytes during autoimmune diseases and transplantation.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CTLA-4", "target": "CO-INHIBITORY LIGAND"}, {"weight": 7.0, "description": "LYP Variants", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "order": 1, "source": "TREG SUPPRESSION", "target": "TCONV INTRINSIC SUSCEPTIBILITY/"}, {"weight": 5.0, "description": "Autoimmune Disorders", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "order": 1, "source": "TH1 SIGNATURE/", "target": "LYP VARIANTS"}, {"weight": 16.0, "description": "LYP Variants/Molecular Interaction/<SEP>LYP Variants", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "order": 1, "source": "SUPPRESSION BY TREGS/", "target": "TRESP PROLIFERATION/"}, {"weight": 1.0, "description": "CD8+ T cell subset", "source_id": "chunk-daf8d8b6d90cc3f92b645357f7c1294b", "order": 1, "source": "BIOLOGICAL FUNCTION<|THE PTPN22 GENE CONTRIBUTES TO THE REGULATION OF LYMPHOCYTE ACTIVATION STATE THROUGH MODULATION OF TCR SIGNALING PATHWAYS LIKE CA2+ FLUX AND PERK SIGNALING.](ENTITY", "target": "PTPN22 GENE"}, {"weight": 7.0, "description": "Overexpression of LYP-620R reduced APC-mediated proliferation of Tconv and Treg.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "ACTIVATION INDUCED PROLIFERATION", "target": "LYP-620R IMPACT"}, {"weight": 8.0, "description": "The LYP-620W variant was capable of blocking Treg proliferation to a similar degree as the LYP-620R variant.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "ACTIVATION INDUCED PROLIFERATION", "target": "LYP-620W IMPACT"}, {"weight": 9.0, "description": "Overexpression of PTPN22 influences T cell cytokine production, but mitogenic activation by PMA and ionomycin was not altered.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CELL ACTIVATION BY PMA AND IONOMYCIN", "target": "LYP VARIANTS"}, {"weight": 10.0, "description": "The net accumulation of IL-2 was not altered by overexpression of the LYP-620R variant, but increased at the 24-hour time point by the LYP-620W variant.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CYTOKINE PRODUCTION", "target": "IL-2 SECRETION"}, {"weight": 11.0, "description": "The secretion of IL-4 was robustly inhibited for the first 48 hours, but to a somewhat lesser degree by LYP-620W at the 72-hour time point.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CYTOKINE PRODUCTION", "target": "IL-4 SECRETION"}, {"weight": 12.0, "description": "The secretion of IL-5 was not altered by either variant of LYP.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CYTOKINE PRODUCTION", "target": "IL-5 SECRETION"}, {"weight": 13.0, "description": "The secretion of IL-9 was robustly inhibited for the first 48 hours and to a somewhat lesser degree by LYP-620W at the 72-hour time point.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CYTOKINE PRODUCTION", "target": "IL-9 SECRETION"}, {"weight": 14.0, "description": "The secretion of IL-10 was robustly inhibited for the first 48 hours, but to a somewhat lesser degree by LYP-620W at the 72-hour time point.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CYTOKINE PRODUCTION", "target": "IL-10 SECRETION"}, {"weight": 15.0, "description": "The secretion of IL-13 was robustly inhibited for the first 48 hours and to a somewhat lesser degree by LYP-620W at the 72-hour time point.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CYTOKINE PRODUCTION", "target": "IL-13 SECRETION"}, {"weight": 16.0, "description": "IFN\u03b3 secretion was not altered by overexpression of either variant of LYP.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CYTOKINE PRODUCTION", "target": "IFN\u0393 SECRETION"}, {"weight": 12.0, "description": "The secretion of IFN\u03b3 and IL-2 by CAR T cells is indicative of their robust cytotoxic function against tumor targets.)", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "order": 1, "source": "CYTOKINE PRODUCTION", "target": "ANTIBODY CYTOTOXIC FUNCTION"}, {"weight": 7.0, "description": "LYP variants influence T cell proliferation during activation.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "REGULATED BY LYP VARIANTS", "target": "T CELL PROLIFERATION"}, {"weight": 8.0, "description": "Lymphoproteins (LYP, PTPN22) impact the secretion of various cytokines, modulating Th1 and Th2 responses.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CYTOKINE SECRETION", "target": "MODULATED BY LYP VARIANTS"}, {"weight": 9.0, "description": "CD4+ effector T cells play a crucial role in regulating host immune responses through the production of cytokines and other effectors.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "CD4+ EFFECTOR T CELLS", "target": "REGULATE HOST IMMUNE RESPONSE"}, {"weight": 10.0, "description": "Antigen-presenting cells (APCs) are essential for the activation and proliferation of T cells during immune response stimulation.)", "source_id": "chunk-d53a9f0fba772a7b52b906f476f0aef7", "order": 1, "source": "APCS", "target": "MITOGENIC ACTIVATION"}, {"weight": 5.0, "description": "Environmental Factor/", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "order": 1, "source": "LYP VARIANTS/TH1 SIGNATURE/AUTOIMMUNE DISORDERS", "target": "MOLECULAR INTERACTION/"}, {"weight": 7.0, "description": "LYP Variants/Molecular Interaction/", "source_id": "chunk-dee0a8a550579bff19f48d41446e842d", "order": 1, "source": "TREG SUPPRESSION/", "target": "TCONV INTRINSIC SUSCEPTIBILITY/"}, {"weight": 3.0, "description": "The 4-1BB co-stimulatory domain enhances the activation and anti-tumor efficacy of CAR T cells expressing TAG72.)", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "order": 1, "source": "TAG72", "target": "4-1BB INTRACELLULAR CO-STIMULATORY DOMAIN"}, {"weight": 2.0, "description": "The TAG72 gene influences the cytolytic activity against OVCAR3 ovarian cancer cells through CAR T cell treatment.)", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "order": 1, "source": "TAG72", "target": "OVCAR3"}, {"weight": 4.0, "description": "The humanized scFv CC49 contributes to specificity by binding to the TAG72 antigen on cancer cells.)", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "order": 1, "source": "CAR T CELLS", "target": "HUMANIZED SCFV CC49"}, {"weight": 5.0, "description": "CD19t is a component of the CAR that facilitates the tracking and monitoring of TAG72-BB\u03b6 CAR T cells during treatment.)", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "order": 1, "source": "CAR T CELLS", "target": "CD19T (CD19 TRUNCATED) FOR CELL TRACKING"}, {"weight": 4.0, "description": "Regional intraperitoneal delivery of CAR T cells demonstrated more potent anti-tumor effects compared to intravenous injection.)", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "order": 1, "source": "CAR T CELLS", "target": "INTRAPERITONEAL INJECTION"}, {"weight": 7.0, "description": "Induction of CD137 expression on TAG72-BB\u03b6 CAR T cells is a measure of their activation against the TAG72 antigen.)", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "order": 1, "source": "CD137 EXPRESSION", "target": "TAG72-BB\u0396 CAR T CELLS"}, {"weight": 8.0, "description": "CD69 expression on the surface of T cells correlates with early stages of T cell activation and response to stimulation.)", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "order": 1, "source": "CELL SURFACE CD69 EXPRESSION", "target": "CAR T CELL ACTIVATION"}, {"weight": 9.0, "description": "Cytotoxic T cells, such as those expressing TAG72-BB\u03b6 CARs, release IFN\u03b3 in response to antigen presentation by tumor cells.)", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "order": 1, "source": "IFN\u0393 RELEASE", "target": "ANTIGEN-SPECIFIC RESPONSE"}, {"weight": 10.0, "description": "The E:T ratio impacts the efficiency of cell killing activity; higher ratios can lead to reduced killing effectiveness.)", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "order": 1, "source": "E:T RATIO (EFFECTOR:TARGET RATIO)", "target": "KILLING EFFICIENCY"}, {"weight": 11.0, "description": "TAG72-BB\u03b6 CAR T cells show high efficacy against ovarian cancer cells expressing the TAG72 antigen in vitro.)", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "order": 1, "source": "CELL KILING ACTIVITY AGAINST TAG72-POSITIVE CELLS", "target": "IN VITRO ASSAYS"}, {"weight": 1.0, "description": "Ovarian cancer cells expressing the TAG72 antigen are targeted by TAG72-BB\u03b6 CAR T cells to demonstrate their specificity and potential clinical application.)", "source_id": "chunk-bdeb2c0c1ba80048874f6dedb2c2b619", "order": 1, "source": "TUMOR CELLS", "target": "DISEASE TARGET"}, {"weight": 4.0, "description": "Antigen loss or escape following mock treatment was observed in tumor recurrences at early time points from mice treated with TAG72-BB\u03b6 CAR T cells.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "MOCK TREATMENT", "target": "TUMOR RECURRENCES"}, {"weight": 8.0, "description": "Antigen loss or escape from tumor cells was observed more frequently at early time points after mock treatment compared to repeated TAG72-BB\u03b6 CAR T cell dosing.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "MOCK TREATMENT", "target": "DISEASE RESISTANCE"}, {"weight": 8.0, "description": "Repeating the dose of TAG72-BB\u03b6 CAR T cells over a month improved therapeutic responses compared to single dosing.)", "source_id": "chunk-183db5c908df609bb5289896b4ba09d0", "order": 1, "source": "REPEAT DOSING", "target": "ANTI-TUMOR EFFICACY"}, {"weight": 2.0, "description": "Repeating the dose course over 1 month led to more durable anti-tumor responses compared to single dosing with TAG72-BB\u03b6 CAR T cells in the OVCAR3 model.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "REPEAT DOSING", "target": "OVCAR3 MODEL"}, {"weight": 1.0, "description": "The study compared the therapeutic responses of repeat dosing of TAG72-BB\u03b6 CAR T cells with a single dose of Antibody-Drug Conjugates in ovarian cancer xenograft models.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "TAG72-BB\u0396 CAR T CELLS", "target": "ANTIBODY-DRUG CONJUGATES (ADCS)"}, {"weight": 3.0, "description": "The study demonstrated that repeat dosing of TAG72-BB\u03b6 CAR T cells provided greater control of tumors compared to a single dose, based on comparison between the OVCAR3 model and the OV90 model.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "OV90 MODEL", "target": "OVC90 MODEL"}, {"weight": 5.0, "description": "The expression of antigens on tumors varied following mock treatment versus the infusion of TAG72-BB\u03b6 CAR T cells in mice, with different patterns noted at 12 weeks post-infusion.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "T CELL INFUSION", "target": "MOCK AND TAG72-BB\u0396 CAR T CELL TREATED MICE"}, {"weight": 6.0, "description": "Reduced numbers, expansion and persistence of CAR T cells were observed in the blood of OV90-bearing mice compared to OVCAR3 model, suggesting potential suppressive mechanisms affecting T cell function and overall CAR T cell efficacy.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "TUMOR RECURRENCES", "target": "T CELL PERSISTENCE"}, {"weight": 7.0, "description": "The major resistance mechanism against CAR T cell therapy is tumor antigen heterogeneity that promotes eventual antigen loss or escape.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "TUMOR RECURRENCES", "target": "ANTIGEN HETEROGENEITY"}, {"weight": 9.0, "description": "The survival benefit associated with repeated TAG72-BB\u03b6 CAR T cell infusion over the OV90 model was more pronounced compared to OVCAR3 model due to differences in treatment timing and tumor dynamics.)", "source_id": "chunk-7fe5e7a40fe11afeb152e614de7ef1cf", "order": 1, "source": "SURVIVAL BENEFIT", "target": "T CELL INFUSION TIMING"}, {"weight": 1.0, "description": "The population sample used for genetic analysis provided insights into the genetic architecture and its impact on lung function.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "HUTTERITES", "target": "POPULATION SAMPLE"}, {"weight": 6.0, "description": "The FEV1/FVC ratio showed associations with SNPs at several loci, including the THSD4-UACA-TLE3 and C10orf11 loci.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "FEV1/FVC", "target": "LUNG FUNCTION TRAIT"}, {"weight": 1.0, "description": "The GRAIL algorithm was utilized to discover functional connections between genes, revealing links among \u03b2-defensin genes and other identified transcripts.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "GRAIL ALGORITHM", "target": "SOFTWARE TOOL"}, {"weight": 1.0, "description": "The Gene Ontology database provided context for functional connections identified by the GRAIL algorithm.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "GENE ONTOLOGY", "target": "DATABASE"}, {"weight": 9.0, "description": "SNPs at the CCL23-CCL18 locus on chromosome 17q12 had associations with FEV1/FVC.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "CCL23-CCL18 LOCUS", "target": "GENETIC LOCATION"}, {"weight": 1.0, "description": "The 5.8% explained variance in FEV1/FVC by the six SNPs indicated their contribution to understanding the genetic factors influencing lung function.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "5.8% OF RESIDUAL VARIANCE", "target": "EXPLAINED VARIANCE"}, {"weight": 1.0, "description": "A genetic study on Hutterites, represented by Fig. E3b in the Online Repository, helped validate findings related to specific loci and functional gene interactions.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "HUTTERITE GWAS", "target": "GENETIC STUDY"}, {"weight": 5.0, "description": "A genome-wide significant association was found at the THSD4-UACA-TLE3 locus on chromosome 15q23.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "THSD4-UACA-TLE3 LOCUS", "target": "GENETIC LOCATION"}, {"weight": 8.0, "description": "The CHAF1B locus on chromosome 21q22.13 showed evidence for association with SNPs related to lung function.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "GENETIC LOCATION", "target": "CHAF1B LOCUS"}, {"weight": 10.0, "description": "The PITPNC1 locus on chromosome 17q24.2 showed evidence for association with SNPs related to lung function.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "GENETIC LOCATION", "target": "PITPNC1 LOCUS"}, {"weight": 14.0, "description": "The minimum set of 80 SNPs providing the best predictive value for FEV1/FVC were identified in the GWAS.)", "source_id": "chunk-7cf8f1ee14e4f57553bbe5b4ab0c664a", "order": 1, "source": "80 SNPS", "target": "GENETIC VARIANT SET"}, {"weight": 1.0, "description": "The Hutterite genome-wide association study provided insights into the genetic basis for variations in lung function through its connections to specific loci and transcripts.)", "source_id": "chunk-7efd4f037425eeac77e329347b16beb2", "order": 1, "source": "GENETIC STUDY", "target": "HUTTERITES GWAS"}, {"weight": 0.0002, "description": "rs231775 polymorphism of CTLA4 showed significant association with PBC susceptibility.)<SEP>rs231775 polymorphism of CTLA4 was significantly associated with PBC susceptibility.)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "RS231775", "target": "CTLA4"}, {"weight": 0.1, "description": "The rs231725 polymorphism also showed significant association with PBC susceptibility.)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "RS231725", "target": "CTLA4"}, {"weight": -1.0, "description": "No association was identified between rs3087243 and PBC susceptibility.)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "RS3087243", "target": "PBC"}, {"weight": 0.06, "description": "The study found no association between rs5742909 polymorphisms and PBC susceptibility.)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "RS5742909", "target": "PBC"}, {"weight": 0.05, "description": "The results showed that the CTLA4 gene was significantly associated with susceptibility to Primary Biliary Cholangitis (PBC).)", "source_id": "chunk-f02653cb486dbf68cc259f74ea8b2b2e", "order": 1, "source": "CTLA4", "target": "PBC"}, {"weight": 5.0, "description": "This test supported the hypothesis of a shared causal variant associated with T1D susceptibility and gene expression changes.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "BAYESIAN TEST FOR COLOCALISATION", "target": "STATISTICAL METHOD"}, {"weight": 5.0, "description": "IL27 SNP rs181206G has effects on IRF1 and STAT1 expression, confirmed by trans-eQTL analysis.)", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "order": 1, "source": "IL27 SNP RS181206G", "target": "TRANS-EQTL LOCUS EFFECTS"}, {"weight": 1.0, "description": "Correlation with expression levels", "source_id": "chunk-bec60ce5d500ee78c6792a2d92b048ca", "order": 1, "source": "CPG SITE/", "target": "METHYLATED SITES"}, {"weight": 1.0, "description": "Gene expression levels were corrected before identifying trans-eQTL associations.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "CIS-EQTL EFFECTS", "target": "CORRECTION FACTOR"}, {"weight": 3.0, "description": "A region influenced the expression levels of several overlapping genes in CD4+ and CD8+ T cells.)", "source_id": "chunk-5b68770e1b446d6cba0d4e1ecabf497b", "order": 1, "source": "BROAD-RANGE IMPACT AND OVERLAP OF GENES", "target": "IMPACT SHARED AMONG GENES"}, {"weight": 2.0, "description": "T1D susceptibility is associated with changes in IRF1 and STAT1 expression, which may be influenced by a specific genomic variant.)", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "order": 1, "source": "CHANGES IN IRF1 AND STAT1 EXPRESSION", "target": "T1D SUSCEPTIBILITY"}, {"weight": 3.0, "description": "Specific trans-eQTL effect detected only in CD4+ T cells but not in CD8+ T cells.)", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "order": 1, "source": "CD4+ T CELLS AND CD8+ T CELLS", "target": "TRANS-EQTL EFFECTS"}, {"weight": 4.0, "description": "No signal present in B cells or monocytes when re-analyzing Fairfax et al.\'s data.)", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "order": 1, "source": "B CELLS AND MONOCYTES", "target": "SIGNAL PRESENCE"}, {"weight": 6.0, "description": "The IL-27 cytokine interacts with the IL27RA gene (together with gp130, also known as IL6ST) to activate STAT1/STAT3 pathway.)", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "order": 1, "source": "IL-27 CYTOKINE AND IL27RA/GP130", "target": "RECEPTOR INTERACTION"}, {"weight": 7.0, "description": "Interferon stimulated response elements are bound by STAT1 and STAT3 pathways leading to transcriptional activation.)", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "order": 1, "source": "BINDING", "target": "ISRE"}, {"weight": 8.0, "description": "The Pro119 variant of the IL27 gene resulted in lower expression levels of IRF1 and STAT1 after transfection experiments.)", "source_id": "chunk-1b40d071abac0ba432a9fb0096796f45", "order": 1, "source": "IL27 GENE PRO119 ALLELE", "target": "TRANSFECTION EFFECTS"}, {"weight": 2.0, "description": "No significant difference in PBMC composition was claimed between asthma patients and healthy controls.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "PBMC COMPOSITION", "target": "ASTHMA"}, {"weight": 3.0, "description": "Significantly increased IL-8 mRNA levels were found in asthmatic patients compared to controls, especially those with moderate-to-severe asthma.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "ASTHMA", "target": "IL-8 MRNA LEVELS"}, {"weight": 7.0, "description": "The rs4073 T allele was considered as a risk factor for asthma severity.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "RS4073 T ALLELE", "target": "ASTHMA SEVERITY IN TUNISIAN CHILDREN"}, {"weight": 4.0, "description": "rs4073 TT genotype was significantly associated with an increased risk of asthma development.)", "source_id": "chunk-3c66ee37333cba1e51a2bdc729038b68", "order": 1, "source": "ASTHMA RISK", "target": "RS4073 TT GENOTYPE"}, {"weight": 5.0, "description": "The SNP-set for the ASAH1 gene includes only these 14 typed SNPs that are in good LD with each other and with the causal SNPs.)", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "order": 1, "source": "ASAH1 GENE", "target": "SNP-SET"}, {"weight": 5.0, "description": "SNPs 7, 11, 13, and 21 are in good LD with each other and when tested as causal, the power of kernel machine method is higher than min test.)", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "order": 1, "source": "NAT2 GENE", "target": "TYPED SNPS"}, {"weight": 4.0, "description": "In regions of high LD, the kernel machine method outperforms the min test.)", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "order": 1, "source": "FGFR2 GENE", "target": "HIGH LD REGIONS"}, {"weight": 4.0, "description": "In regions of weak LD, the two methods have comparable power.)", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "order": 1, "source": "FGFR2 GENE", "target": "WEAK LD REGIONS"}, {"weight": 4.0, "description": "The empirical power for HLA-B gene shows that using MAF as weights in weighted IBS kernel can upweight rare alleles.)", "source_id": "chunk-2e31819d63b80079cb6422efd6e684e0", "order": 1, "source": "HLA-B GENE", "target": "SNPS 1058026 AND RS2523608"}, {"weight": 1.0, "description": "The weighted IBS kernel outperforms the linear kernel in terms of power when rare variants are important.)", "source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "order": 1, "source": "IBS KERNEL", "target": "LINEAR KERNEL"}, {"weight": 1.0, "description": "When rs2523608 has higher minor allele frequency, it\'s not as effective as when the more common allele (which is also causal) gets downweighted by a certain kernel function.)", "source_id": "chunk-076f6aeeb369546413bc63c9f305553a", "order": 1, "source": "HIGHER MAF", "target": "RS2523608 SNV/CAUSAL SNP"}, {"weight": 2.0, "description": "No significant association was observed between anti-CTLA-4 or anti-PD-1 drugs and treatment response across different treatments, after adjustments for age, sex, and drug types.)", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "order": 1, "source": "POOLED ASSOCIATION ANALYSIS", "target": "ANTI-CTLA-4/ANTI-PD-1"}, {"weight": 1.0, "description": "The effect of rs17388568 on response rate was stronger in younger female patients, although the interactions were not statistically significant.)", "source_id": "chunk-2b5346dfc62015294aa57aeab3960cd1", "order": 1, "source": "AGE AND SEX", "target": "INTERACTION EFFECT ON RESPONSE"}, {"weight": 3.0, "description": "A pooled analysis of combination therapy outcomes did not yield significant associations with treatment response when adjusting for various covariates.)", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "order": 1, "source": "COMBINATION THERAPY/POOLED ASSOCIATION ANALYSIS", "target": "TREATMENT RESPONSE"}, {"weight": 1.0, "description": "After adjustments for age, sex, treatment drugs, and other covariates, no associations were found with SNPs related to treatment response.)", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "order": 1, "source": "COVARIATE IMPACT", "target": "SNPS/AGE/SEX/TREATMENT DRUGS"}, {"weight": 2.0, "description": "No significant association was observed between the type of drug (anti-CTLA-4, anti-PD-1 or combination therapy) and treatment response after adjusting for age, sex and different types of treatment.)", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "order": 1, "source": "ANTI-CTLA-4/ANTI-PD-1/COMBINATION THERAPY/", "target": "TREATMENT TYPE IMPACT"}, {"weight": 3.0, "description": "Adjustments made to account for covariates like age, sex and specific treatments did not reveal significant associations with factors being tested (SNPs, drug therapies) related to response or outcomes.)", "source_id": "chunk-0d49552280a277a8a26456d06f8f26a8", "order": 1, "source": "AGE/SEX/TREATMENT DRUGS/", "target": "COVARIATE ADJUSTMENT"}, {"weight": 1.0, "description": "Therapy", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "EBNA2", "target": "MS RISK ALLELES/SNPS/GENOMIC VARIANT/DISEASE/ENVIRONMENTAL FACTOR/"}, {"weight": 1.0, "description": "The EBNA2 effect on gene expression is dependent on specific genotypes at MS risk loci and interacts with the direction of allelic expression.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "EBNA2", "target": "MS RISK LOCI"}, {"weight": 10.0, "description": "The relationship between the risk allele preference and genetic drift could impact the overall expression pattern of MS risk genes.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "ALLELIC ASSOCIATION", "target": "GENETIC DRIFT"}, {"weight": 5.0, "description": "The program involving LMP1 and LMP2 signaling can be influenced by EBNA2\'s interaction with MS risk loci, potentially impacting immune evasion through alteration of gene expression.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "EBV LATENCY III PROGRAM", "target": "EFFECT ON IMMUNE EVASION"}, {"weight": 6.0, "description": "EBNA2\'s preference for certain risk alleles has implications on altering the direction of allelic association with expression in specific MS risk genes.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "RISK ALLELE PREFERENCE", "target": "INFLUENCE ON GENE EXPRESSION"}, {"weight": 1.0, "description": "Disease/Environmental Factor/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "EBNA2-TAT/", "target": "DRUG/EFFECT ON EBV DNA COPY NUMBER/MOLECULAR INTERACTION/GENETIC TRAIT/"}, {"weight": 1.0, "description": "Therapy", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "EBNA2 BINDING/ALLELIC ASSOCIATION/MS RISK ALLELES/", "target": "EXPRESSION/GENETIC TRAIT/DISEASE/"}, {"weight": 1.0, "description": "Disease/Environmental Factor/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "LCL PROLIFERATION/EBNA2-TAT/", "target": "EFFECT ON GENE EXPRESSION/MOLECULAR INTERACTION/GENETIC TRAIT/"}, {"weight": 1.0, "description": "Therapy", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "TNFAIP8/EXPRESSION/EBNA2 INHIBITION/", "target": "EBV DNA COPY NUMBER/MOLECULAR INTERACTION/GENETIC TRAIT/DISEASE/"}, {"weight": 1.0, "description": "Disease/Therapy", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "GENOTYPE EFFECT/ALLELIC ASSOCIATION/", "target": "EBNA2 BINDING/EXPRESSION/EBNA2 INHIBITION/"}, {"weight": 1.0, "description": "Disease/Therapy", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "EBNA2 BINDING/", "target": "ALLELIC ASSOCIATION/EXPRESSION/GENETIC TRAIT/"}, {"weight": 1.0, "description": "Genetic Trait/Disease/", "source_id": "chunk-bb351b97d99fce93d17df522db92dda7", "order": 1, "source": "EBNA2 INHIBITION/EBNA2-TAT/", "target": "EFFECT ON LCLS/MOLECULAR INTERACTION/"}, {"weight": 4.0, "description": "The alteration of allelic expression based on EBNA2 interaction is inferred from comparison to uninfected B cells and has implications for gene regulation.)", "source_id": "chunk-03962ffc6066ee5da582bdc89310b1d6", "order": 1, "source": "ALLELIC EXPRESSION", "target": "INFERRED EFFECT"}, {"weight": 1.0, "description": "Polymorphism rs20417 was found to be significantly associated with aspirin insensitivity.)", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "order": 1, "source": "RS20417", "target": "ASPIRIN INSENSITIVITY"}, {"weight": 2.0, "description": "The risk allele of rs1126643 is strongly linked to aspirin insensitivity, as evident from the analysis.)", "source_id": "chunk-54c4fede216a6c4183b95ee1a2edcb17", "order": 1, "source": "RS1126643", "target": "ASPIRIN INSENSITIVITY"}, {"weight": 7.0, "description": "The p.W631* mutation results in reduced expression, inability for phosphorylation and nuclear localization impairment.)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "P.W631*", "target": "MUTATION EFFECT"}, {"weight": 1.0, "description": "The W631* variant occurs in the gene coding for STAT5B protein.)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "STAT5B", "target": "W631*"}, {"weight": 5.0, "description": "A homozygous variant in STAT5B gene results in a loss-of-function mutation leading to altered protein function.)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "STAT5B", "target": "GENE MUTATION"}, {"weight": 2.0, "description": "This existing mutation is in proximity to the newly identified p.W631* variant in STAT5B.)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "P.A630P", "target": "EXISTING VARIANT"}, {"weight": 6.0, "description": "Src Homology 2 (SH2) domain is critical for STAT5B activation and homodimer formation.)", "source_id": "chunk-825b9a3dd613f4c55dbf49fa3998d0d5", "order": 1, "source": "DOMAIN", "target": "SH2"}, {"weight": 95.0, "description": "were associated with DQA1*02:01\u2013DQB1*02:02 haplotype carrier status.)", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "order": 1, "source": "RS28383172 AG OR GG GENOTYPES", "target": "HAPLOTYPE CARRIER STATUS"}, {"weight": 100.0, "description": "had perfect concordance with HLA-DRB1*07:01 allele carrier status.)", "source_id": "chunk-0dd0781947ec4a92007eb1736b1888c2", "order": 1, "source": "RS28383172 G ALLELE", "target": "HLA TYPE"}, {"weight": 2.0, "description": "Statistical Analysis", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "ALGORITHM V VS. ALGORITHMS II, III, IV", "target": "PERFORMANCE COMPARISON/"}, {"weight": 3.0, "description": "Statistical Analysis", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "PERFORMANCE COMPARISON/", "target": "ALGORITHM V VS. HLA-DRB1*07:01-DQA1*02:01-DQB1*02:02 HAPLOTYPE"}, {"weight": 1.0, "description": "Statistical Analysis", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "PERFORMANCE COMPARISON/", "target": "ALGORITHM V AND ALGORITHM V HAPLOTYPE ACCURACY"}, {"weight": 1.0, "description": "Correlation Analysis", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "SENSITIVITY VALUE AND E. COLI ASP HYPERSENSITIVITY", "target": "STATISTICAL MEASURE/"}, {"weight": 2.0, "description": "Error Rate", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "STATISTICAL MEASURE/", "target": "SPECIFICITY VALUE AND FALSE POSITIVES"}, {"weight": 3.0, "description": "Accuracy Rate", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "STATISTICAL MEASURE/", "target": "POSITIVE PREDICTIVE VALUE AND TRUE CASES"}, {"weight": 2.0, "description": "Error Rate", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "STATISTICAL MEASURE/", "target": "NEGATIVE PREDICTIVE VALUE AND FALSE NEGATIVES"}, {"weight": 3.0, "description": "Performance Metric", "source_id": "chunk-1f50436a36721971a0a256131d7fca62", "order": 1, "source": "ALGORITHM V AND FALSE HAPLOTYPE CARRIERS", "target": "MISCLASSIFICATION/"}, {"weight": 2.0, "description": "Polymorphisms in the TLR7 gene are associated with decreased AG genotype prevalence in HIV-infected individuals compared to healthy controls.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "HIV-INFECTED INDIVIDUALS", "target": "TLR7 (RS179008, RS179009)"}, {"weight": 3.0, "description": "The prevalence of specific TLR7 rs179009 genotypes decreases across different stages of HIV disease compared to healthy controls.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "HIV DISEASE STAGES", "target": "TLR7 POLYMORPHISM"}, {"weight": 4.0, "description": "Polymorphisms in the TLR7 gene are not affected by tobacco or alcohol use among HIV-infected individuals, but distribution of AG genotype is higher among those who consume tobacco.)", "source_id": "chunk-05d583f893b703816ad8de9561aa5292", "order": 1, "source": "TOBACCO AND ALCOHOL USERS", "target": "TLR7 POLYMORPHISM"}, {"weight": 18.0, "description": "Tobacco usage is related to alcohol use, both being potential factors influencing HIV disease progression.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "TOBACCO", "target": "ALCOHOL"}, {"weight": 20.0, "description": "The baseline CD4 cell count correlates with genetic polymorphisms of TLR7, which could influence HIV disease progression.)", "source_id": "chunk-2c91cdebb36ef064eb01a7021a961302", "order": 1, "source": "BASELINE CD4 COUNTS", "target": "TLR7 (RS179008 AND RS179009)"}, {"weight": 3.0, "description": "Comparing frequencies of CD8+ T cell recognition indicates similarity between both peptides and an immunodominant epitope.)", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "order": 1, "source": "NONAMER", "target": "DECAMER"}, {"weight": 2.0, "description": "The thermal stability suggests lower affinity for decamer compared to HLA-B*15:01, supporting the potential role in disease mechanisms.)", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "order": 1, "source": "HLA-A*01:01", "target": "DECAMER"}, {"weight": 7.0, "description": "Both peptides have nominal frequencies comparable to known immunodominant epitopes when recognized by CD8+ T cells.)", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "order": 1, "source": "NONAMER/DECAMER", "target": "T CELL RECOGNITION"}, {"weight": 2.0, "description": "Structure-based modeling methods were previously applied to predict peptide/HLA interactions using this template.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "HLA-B*15:01", "target": "PEPTIDE/HLA MODELING"}, {"weight": 1.0, "description": "This epitope is present on the surface of epithelial cells, participating in the immune response.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "KNOWN IMMUNODOMINANT EPITOPE", "target": "EPITHELIAL CELLS"}, {"weight": 2.0, "description": "Gene X is involved in a known immune response process.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "UNKNOWN GENE X", "target": "KNOWN IMMUNE RESPONSE I"}, {"weight": 3.0, "description": "Genomic variant Y may contribute to an unknown genetic condition.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "UNKNOWN VARIANT Y", "target": "UNKNOWN GENETIC CONDITION J"}, {"weight": 4.0, "description": "Transcript Z codes for multiple proteins, including K and L.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "TRANSCRIPT Z", "target": "PROTEINS K, L"}, {"weight": 5.0, "description": "Protein A interacts with nutrient B in biological processes.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "PROTEIN A", "target": "NUTRIENT B"}, {"weight": 6.0, "description": "Nutrient B is abundant in dietary pattern C.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "NUTRIENT B", "target": "DIETARY PATTERN C"}, {"weight": 7.0, "description": "Dietary pattern C influences metabolic pathway D through nutrient intake.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "DIETARY PATTERN C", "target": "METABOLIC PATHWAY D"}, {"weight": 8.0, "description": "Physiological process D depends on molecular interaction E for its function.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "PHYSIOLOGICAL PROCESS D", "target": "MOLECULAR INTERACTION E"}, {"weight": 9.0, "description": "Environmental factor G is a risk factor for disease H.)", "source_id": "chunk-41c7131af28e2a745ef6b88b8601860f", "order": 1, "source": "ENVIRONMENTAL FACTOR G", "target": "DISEASE H"}, {"weight": 1.0, "description": "Asp46", "source_id": "chunk-de413ae6100407d8dadd5bdee87c1074", "order": 1, "source": "SELF-PEPTIDE/HLA-B*15:01 COMPLEX", "target": "STRUCTURAL EXCLUSION AND IMMUNE TOLERANCE<|INABILITY OF SELF-PEPTIDE TO FORM STABLE COMPLEXES WITH HLA DUE TO STRUCTURAL CONSTRAINTS LEADING TO IMMUNE TOLERANCE.)(ENTITY"}, {"weight": 3.0, "description": "The peptide backbone conformation showed minimal variance in all nonamer/HLA-B*15:01 structures.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "PEPTIDE BACKBONE CONFORMATION", "target": "VARIANCE ANALYSIS"}, {"weight": 4.0, "description": "An allele with mutated residues exhibited a high binding energy and was distant from the best binder.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "BINDING AFFINITY", "target": "MOCK POLYALA HLA"}, {"weight": 5.0, "description": "A trend for specific residues contributing to overall binding energy was observed.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "RESIDUES IN MHC BINDING GROOVE", "target": "SEQUENCE BIAS EVALUATION"}, {"weight": 6.0, "description": "Highly invariant residues were consistently found in HLA alleles with high affinity binding.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "INVARIANT RESIDUES", "target": "KEY FEATURES"}, {"weight": 7.0, "description": "Similar polar contacts, primarily in the F-pocket of MHC, correlated well with positions of invariant MHC residues.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "CONTACT PATTERN", "target": "F-POCKET"}, {"weight": 3.0, "description": "This mutation in the ALK gene serves as an indicator for potential cancer development or progression.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "ALK R1275Q MUTATION", "target": "CANCER MARKER"}, {"weight": 4.0, "description": "The selection of HLA alleles aims to broaden the display repertoire that can accommodate neoepitopes and enhance immune response.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "HLA ALLELES SELECTION", "target": "REPERTOIRE EXPANSION"}, {"weight": 6.0, "description": "The Kullback-Leibler divergence measure is used to quantify sequence diversity among HLA alleles involved in peptide binding.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "KULLBACK-LEIBLER DIVERGENCE", "target": "SEQUENCE DIVERSITY METRIC"}, {"weight": 7.0, "description": "Analyze the nature and specificity of polar contacts between peptides, MHC molecules, and T cells during immune response interactions.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "MOLECULAR INTERACTION DETAIL", "target": "POLAR CONTACTS ANALYSIS"}, {"weight": 8.0, "description": "The thermal stability test confirms the robustness of p/MHC complexes formed with selected HLA alleles under physiological conditions.)", "source_id": "chunk-614333487e4b92705503909d34155179", "order": 1, "source": "BIOPHYSICAL PROPERTY", "target": "THERMAL STABILITY ASSESSMENT"}, {"weight": 8.0, "description": "The lower affinity for HLA-A*01:01 might imply involvement in disease mechanisms related to these neoepitopes.)", "source_id": "chunk-d3a9c93928447522ce3df4209f8ac803", "order": 1, "source": "ALK NEOEPITOPE (NONAMER AND DECAMER)", "target": "DISEASE MECHANISMS"}, {"weight": 3.1, "description": "The genomic variant rs74420772 is significantly associated with the expression level of IgM antibodies against PC, potentially influencing disease risk.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "RS74420772", "target": "IGM ANTI-PC"}, {"weight": 1.6, "description": "Genetic variations like rs735665 and rs9392504 can influence the expression of IgM antibodies against PC.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "IGM ANTI-PC", "target": "RS735665, RS9392504"}, {"weight": 1.2, "description": "rs735665 is significantly associated with the risk of CLL, indicating its role in this disease.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "RS735665", "target": "CLL"}, {"weight": 1.3, "description": "rs735665 affects the expression level of general IgM, suggesting a genetic basis for this immune response.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "RS735665", "target": "GENERAL IGM"}, {"weight": 1.4, "description": "rs735665 contributes substantially to the polygenic risk scores for increasing IgM, highlighting its genetic influence on this measure.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "RS735665", "target": "POLYGENIC RISK SCORE FOR IGM"}, {"weight": 2.2, "description": "The variant rs735665 is linked to changes in epigenetic marks like H3K4Me1, indicating its potential role in gene regulation.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "RS735665", "target": "REGULATORY MARK"}, {"weight": 1.5, "description": "rs9392504 has a nominal significance level associated with CLL, contributing to its risk prediction.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "RS9392504", "target": "CLL"}, {"weight": 3.2, "description": "The genetic variation rs35923643 resides in the GRAMD1B gene, indicating a possible role of this variant within the gene\'s function.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "RS35923643", "target": "GRAMD1B"}, {"weight": 2.3, "description": "The polygenic risk scores associated with IgM levels can guide personalized treatment strategies for autoimmune disorders like CLL.)", "source_id": "chunk-0dc1ddaa86432d222da7fc75090f700a", "order": 1, "source": "POLYGENIC RISK SCORE", "target": "THERAPIES FOR AUTOIMMUNE DISEASES"}]}